<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0035)https://www.ncbi.nlm.nih.gov/pubmed -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"></head><body><pre>PMID- 19085332
OWN - NLM
STAT- MEDLINE
DCOM- 20090202
LR  - 20101118
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 6
DP  - 2008 Dec
TI  - The influence of model averaging on clade posteriors: an example using the
      triggerfishes (Family Balistidae).
PG  - 905-19
LID - 10.1080/10635150802562392 [doi]
AB  - Although substantial uncertainty typically surrounds the choice of the best model
      in most phylogenetic analyses, little is known about how accommodating this
      uncertainty affects phylogenetic inference. Here we explore the influence of
      Bayesian model averaging on the phylogenetic inference of the triggerfishes
      (Family: Balistidae), a charismatic group of reef fishes. We focus on clade
      support as this area has received little attention and is typically one of the
      most important outcomes of phylogenetic studies. We present a novel phylogenetic 
      hypothesis for the family Balistidae based on an analysis of two mitochondrial
      (12S, 16S) and three nuclear genes (TMO-4C4, Rhodopsin, RAG1) sampled from 26
      ingroup species. Despite the presence of substantial model uncertainty in almost 
      all partitions of our data, we found model-averaged topologies and clade
      posteriors to be nearly identical to those conditioned on a single model.
      Furthermore, statistical comparison of clade posteriors using the Wilcoxon
      signed-rank test revealed no significant differences. Our results suggest that
      although current model-selection approaches are likely to lead to
      overparameterization of the substitution model, the consequences of conditioning 
      on this overparameterized model are likely to be mild. Our phylogenetic results
      strongly support the monophyly of the triggerfishes but suggest that the genera
      Balistoides and Pseudobalistes are polyphyletic. Divergence time estimation
      supports a Miocene origin of the crown group. Despite the presence of several
      young species-rich subclades, statistical analysis of temporal diversification
      patterns reveals no significant increase in the rate of cladogenesis across
      geologic time intervals.
FAU - Dornburg, Alex
AU  - Dornburg A
AD  - School of Biological Sciences, Washington State University, Pullman, Washington
      99164, USA. dornburgalex@yale.edu
FAU - Santini, Francesco
AU  - Santini F
FAU - Alfaro, Michael E
AU  - Alfaro ME
LA  - eng
SI  - GENBANK/AY308790
SI  - GENBANK/AY679622
SI  - GENBANK/AY679623
SI  - GENBANK/AY679624
SI  - GENBANK/AY679625
SI  - GENBANK/AY679626
SI  - GENBANK/AY679627
SI  - GENBANK/AY679628
SI  - GENBANK/AY679629
SI  - GENBANK/AY679630
SI  - GENBANK/AY679631
SI  - GENBANK/AY679632
SI  - GENBANK/AY679633
SI  - GENBANK/AY679634
SI  - GENBANK/AY700238
SI  - GENBANK/AY700239
SI  - GENBANK/AY700240
SI  - GENBANK/AY700241
SI  - GENBANK/AY700242
SI  - GENBANK/AY700243
SI  - GENBANK/AY700244
SI  - GENBANK/AY700245
SI  - GENBANK/AY700246
SI  - GENBANK/AY700247
SI  - GENBANK/AY700248
SI  - GENBANK/AY700249
SI  - GENBANK/AY700250
SI  - GENBANK/AY700251
SI  - GENBANK/AY700308
SI  - GENBANK/AY700310
SI  - GENBANK/AY700311
SI  - GENBANK/AY700312
SI  - GENBANK/AY700313
SI  - GENBANK/AY700314
SI  - GENBANK/AY700315
SI  - GENBANK/AY700316
SI  - GENBANK/AY700317
SI  - GENBANK/AY700318
SI  - GENBANK/AY700319
SI  - GENBANK/AY700320
SI  - GENBANK/AY700321
SI  - GENBANK/EU108801
SI  - GENBANK/EU108802
SI  - GENBANK/EU108803
SI  - GENBANK/EU108804
SI  - GENBANK/EU108805
SI  - GENBANK/EU108806
SI  - GENBANK/EU108807
SI  - GENBANK/EU108808
SI  - GENBANK/EU108809
SI  - GENBANK/EU108810
SI  - GENBANK/EU108811
SI  - GENBANK/EU108812
SI  - GENBANK/EU108813
SI  - GENBANK/EU108814
SI  - GENBANK/EU108815
SI  - GENBANK/EU108816
SI  - GENBANK/EU108817
SI  - GENBANK/EU108818
SI  - GENBANK/EU108819
SI  - GENBANK/EU108820
SI  - GENBANK/EU108821
SI  - GENBANK/EU108822
SI  - GENBANK/EU108823
SI  - GENBANK/EU108824
SI  - GENBANK/EU108825
SI  - GENBANK/EU108826
SI  - GENBANK/EU108827
SI  - GENBANK/EU108828
SI  - GENBANK/EU108829
SI  - GENBANK/EU108830
SI  - GENBANK/EU108831
SI  - GENBANK/EU108832
SI  - GENBANK/EU108833
SI  - GENBANK/EU108834
SI  - GENBANK/EU108835
SI  - GENBANK/EU108836
SI  - GENBANK/EU108837
SI  - GENBANK/EU108838
SI  - GENBANK/EU108839
SI  - GENBANK/EU108840
SI  - GENBANK/EU108841
SI  - GENBANK/EU108842
SI  - GENBANK/EU108843
SI  - GENBANK/EU108844
SI  - GENBANK/EU108845
SI  - GENBANK/EU108846
SI  - GENBANK/EU108847
SI  - GENBANK/EU108848
SI  - GENBANK/EU108849
SI  - GENBANK/EU108850
SI  - GENBANK/EU108851
SI  - GENBANK/EU108852
SI  - GENBANK/EU108853
SI  - GENBANK/EU108854
SI  - GENBANK/EU108855
SI  - GENBANK/EU108856
SI  - GENBANK/EU108857
SI  - GENBANK/EU108858
SI  - GENBANK/EU108859
SI  - GENBANK/EU108860
SI  - GENBANK/EU108861
SI  - GENBANK/EU108862
SI  - GENBANK/EU108863
SI  - GENBANK/EU108864
SI  - GENBANK/EU108865
SI  - GENBANK/EU108866
SI  - GENBANK/EU108867
SI  - GENBANK/EU108868
SI  - GENBANK/EU108869
SI  - GENBANK/EU108870
SI  - GENBANK/EU108871
SI  - GENBANK/EU108872
SI  - GENBANK/EU108873
SI  - GENBANK/EU108874
SI  - GENBANK/EU108875
SI  - GENBANK/EU108876
SI  - GENBANK/EU108877
SI  - GENBANK/EU108878
SI  - GENBANK/NC004416
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Fishes/*classification/genetics
MH  - Linear Models
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2008/12/17 09:00
MHDA- 2009/02/03 09:00
CRDT- 2008/12/17 09:00
PHST- 2008/12/17 09:00 [entrez]
PHST- 2008/12/17 09:00 [pubmed]
PHST- 2009/02/03 09:00 [medline]
AID - 906545059 [pii]
AID - 10.1080/10635150802562392 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Dec;57(6):905-19. doi: 10.1080/10635150802562392.

PMID- 19085331
OWN - NLM
STAT- MEDLINE
DCOM- 20090202
LR  - 20081216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 6
DP  - 2008 Dec
TI  - The modified gap excess ratio (GER*) and the stratigraphic congruence of dinosaur
      phylogenies.
PG  - 891-904
LID - 10.1080/10635150802570809 [doi]
AB  - Palaeontologists routinely map their cladograms onto what is known of the fossil 
      record. Where sister taxa first appear as fossils at different times, a ghost
      range is inferred to bridge the gap between these dates. Some measure of the
      total extent of ghost ranges across the tree underlies several indices of
      cladistic/stratigraphic congruence. We investigate this congruence for 19
      independent, published cladograms of major dinosaur groups and report exceptional
      agreement between the phylogenetic and stratigraphic patterns, evidenced by sums 
      of ghost ranges near the theoretical minima. This implies that both phylogenetic 
      and stratigraphic data reflect faithfully the evolutionary history of dinosaurs, 
      at least for the taxa included in this study. We formally propose modifications
      to an existing index of congruence (the gap excess ratio; GER), designed to
      remove a bias in the range of values possible with trees of different shapes. We 
      also propose a more informative index of congruence--GER*--that takes account of 
      the underlying distribution of sums of ghost ranges possible when permuting
      stratigraphic range data across the tree. Finally, we incorporate data on the
      range of possible first occurrence dates into our estimates of congruence,
      extending a procedure originally implemented with the modified Manhattan
      stratigraphic measure and GER to our new indices. Most dinosaur data sets
      maintain extremely high congruence despite such modifications.
FAU - Wills, Matthew A
AU  - Wills MA
AD  - Department of Biology and Biochemistry, The University of Bath, Claverton Down,
      Bath, UK. m.a.wills@bath.ac.uk
FAU - Barrett, Paul M
AU  - Barrett PM
FAU - Heathcote, Julia F
AU  - Heathcote JF
LA  - eng
GR  - BB/C006682/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Dinosaurs/*classification
MH  - *Phylogeny
EDAT- 2008/12/17 09:00
MHDA- 2009/02/03 09:00
CRDT- 2008/12/17 09:00
PHST- 2008/12/17 09:00 [entrez]
PHST- 2008/12/17 09:00 [pubmed]
PHST- 2009/02/03 09:00 [medline]
AID - 906549706 [pii]
AID - 10.1080/10635150802570809 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Dec;57(6):891-904. doi: 10.1080/10635150802570809.

PMID- 19085330
OWN - NLM
STAT- MEDLINE
DCOM- 20090202
LR  - 20081216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 6
DP  - 2008 Dec
TI  - Comparison of methods for species-tree inference in the sawfly genus Neodiprion
      (Hymenoptera: Diprionidae).
PG  - 876-90
LID - 10.1080/10635150802580949 [doi]
AB  - Conifer-feeding sawflies in the genus Neodiprion provide an excellent opportunity
      to investigate the origin and maintenance of barriers to reproduction, but
      obtaining a phylogenetic estimate for comparative studies of Neodiprion
      speciation has proved difficult. Specifically, nonmonophyly within and
      discordance between individual gene trees, both of which are common in groups
      that diverged recently and/or rapidly, make it impossible to infer a species tree
      using methods that are designed to estimate gene trees. Therefore, in this study,
      we estimate relationships between members of the lecontei species group using
      four approaches that are intended to estimate species, not gene, trees: (1)
      minimize deep coalescences (MDC), (2) shallowest divergences (SD), (3) Bayesian
      estimation of species trees (BEST), and (4) a novel approach that combines
      concatenation with monophyly constraints (CMC). Multiple populations are sampled 
      for most species and all four methods incorporate this intraspecific variation
      into estimates of interspecific relationships. We investigate the sensitivity of 
      each method to taxonomic sampling, and, for the BEST method, we assess the impact
      of prior choice on species-tree inference. We also compare species-tree estimates
      to one another and to a morphologically based hypothesis to identify clades that 
      are supported by multiple analyses and lines of evidence. We find that both
      taxonomic sampling and method choice impact species-tree estimates and that, for 
      these data, the BEST method is strongly influenced by Theta and branch-length
      priors. We also find that the CMC method is the least sensitive to taxonomic
      sampling. Finally, although interspecific genetic variation is low due to the
      recent divergence of the lecontei group, our results to date suggest that
      incomplete lineage sorting and interspecific gene flow are the main factors
      complicating species-tree inference in Neodiprion. Based on these analyses, we
      propose a phylogenetic hypothesis for the lecontei group. Finally, our results
      suggest that, even for very challenging groups like Neodiprion, an underlying
      species-tree signal can be extracted from multi-locus data as long as
      intraspecific variation is adequately sampled and methods that focus on the
      estimation of species trees are used.
FAU - Linnen, Catherine R
AU  - Linnen CR
AD  - Museum of Comparative Zoology, Harvard University, Cambridge, MA 02138, USA.
      clinnen@oeb.harvard.edu
FAU - Farrell, Brian D
AU  - Farrell BD
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Genetic Variation
MH  - Hymenoptera/*classification/genetics
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2008/12/17 09:00
MHDA- 2009/02/03 09:00
CRDT- 2008/12/17 09:00
PHST- 2008/12/17 09:00 [entrez]
PHST- 2008/12/17 09:00 [pubmed]
PHST- 2009/02/03 09:00 [medline]
AID - 906550332 [pii]
AID - 10.1080/10635150802580949 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Dec;57(6):876-90. doi: 10.1080/10635150802580949.

PMID- 19085329
OWN - NLM
STAT- MEDLINE
DCOM- 20090202
LR  - 20101118
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 6
DP  - 2008 Dec
TI  - Phylogeny of the beaked whale genus Mesoplodon (Ziphiidae: Cetacea) revealed by
      nuclear introns: implications for the evolution of male tusks.
PG  - 857-75
LID - 10.1080/10635150802559257 [doi]
AB  - With 14 species currently recognized, the beaked whale genus Mesoplodon (family
      Ziphiidae) is the most speciose in the order Cetacea. Beaked whales are widely
      distributed but are rarely seen at sea due to their oceanic distribution,
      deep-diving capacity, and apparent low abundance. Morphological differentiation
      among Mesoplodon species is relatively limited, with the exception of tooth form 
      in adult males. Based on scarring patterns, males appear to use their tusk-like
      teeth as weapons in aggressive encounters with other males. Females are
      effectively toothless. We used sequences from seven nuclear introns (3348 base
      pairs) to construct a robust and highly resolved phylogeny, which was then used
      as a framework to test predictions from four hypotheses seeking to explain
      patterns of Mesoplodon tusk morphology and/or the processes that have driven the 
      diversification of this genus: (1) linear progression of tusk form; (2)
      allopatric speciation through isolation in adjacent deep-sea canyons; (3)
      sympatric speciation through sexual selection on tusks; and (4) selection for
      species-recognition cues. Maximum-likelihood and Bayesian reconstructions
      confirmed the monophyly of the genus and revealed that what were considered
      ancestral and derived tusk forms have in fact arisen independently on several
      occasions, contrary to predictions from the linear-progression hypothesis.
      Further, none of the three well-supported species clades was confined to a single
      ocean basin, as might have been expected from the deep-sea canyon-isolation or
      sexual-selection hypotheses, and some species with similar tusks have overlapping
      distributions, contrary to predictions from the species-recognition hypothesis.
      However, the divergent tusk forms and sympatric distributions of three of the
      four sister-species pairs identified suggest that sexual selection on male tusks 
      has likely played an important role in this unique radiation, although other
      forces are clearly also involved. To our knowledge, this is the first time that
      sexual selection has been explicitly implicated in the radiation of a mammalian
      group outside terrestrial ungulates.
FAU - Dalebout, Merel L
AU  - Dalebout ML
AD  - School of Biological, Earth, and Environmental Sciences (BEES), University of New
      South Wales, Kensington, NSW, Australia. m.dalebout@unsw.edu.au
FAU - Steel, Debbie
AU  - Steel D
FAU - Baker, C Scott
AU  - Baker CS
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Cell Nucleus/*genetics
MH  - Female
MH  - Introns/*genetics
MH  - Male
MH  - *Phylogeny
MH  - Species Specificity
MH  - Tooth/anatomy &amp; histology
MH  - Whales/anatomy &amp; histology/*classification/*genetics
EDAT- 2008/12/17 09:00
MHDA- 2009/02/03 09:00
CRDT- 2008/12/17 09:00
PHST- 2008/12/17 09:00 [entrez]
PHST- 2008/12/17 09:00 [pubmed]
PHST- 2009/02/03 09:00 [medline]
AID - 906592768 [pii]
AID - 10.1080/10635150802559257 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Dec;57(6):857-75. doi: 10.1080/10635150802559257.

PMID- 19085328
OWN - NLM
STAT- MEDLINE
DCOM- 20090202
LR  - 20081216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 6
DP  - 2008 Dec
TI  - The impact of reticulate evolution on genome phylogeny.
PG  - 844-56
LID - 10.1080/10635150802559265 [doi]
AB  - Genome phylogenies are used to build tree-like representations of evolutionary
      relationships among genomes. However, in condensing the phylogenetic signals
      within a set of genomes down to a single tree, these methods generally do not
      explicitly take into account discordant signals arising due to lateral genetic
      transfer. Because conflicting vertical and horizontal signals can produce
      compromise trees that do not reflect either type of history, it is essential to
      understand the sensitivity of inferred genome phylogenies to these confounding
      effects. Using replicated simulations of genome evolution, we show that different
      scenarios of lateral genetic transfer have significant impacts on the ability to 
      recover the "true" tree of genomes, even when corrections for phylogenetically
      discordant signals are used.
FAU - Beiko, Robert G
AU  - Beiko RG
AD  - Faculty of Computer Science, Dalhousie University, Institute for Molecular
      Bioscience/ARC Centre for Bioinformatics, Brisbane, Australia. beiko@cs.dal.ca
FAU - Doolittle, W Ford
AU  - Doolittle WF
FAU - Charlebois, Robert L
AU  - Charlebois RL
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Gene Transfer, Horizontal
MH  - Genetic Speciation
MH  - Genetic Variation
MH  - *Genome
MH  - *Phylogeny
EDAT- 2008/12/17 09:00
MHDA- 2009/02/03 09:00
CRDT- 2008/12/17 09:00
PHST- 2008/12/17 09:00 [entrez]
PHST- 2008/12/17 09:00 [pubmed]
PHST- 2009/02/03 09:00 [medline]
AID - 906593303 [pii]
AID - 10.1080/10635150802559265 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Dec;57(6):844-56. doi: 10.1080/10635150802559265.

PMID- 19085327
OWN - NLM
STAT- MEDLINE
DCOM- 20090202
LR  - 20190302
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 6
DP  - 2008 Dec
TI  - Evidence for a new root of the tree of life.
PG  - 835-43
LID - 10.1080/10635150802555933 [doi]
AB  - Directed indels, insertions or deletions within paralogous genes, have the
      potential to root the tree of life. Here we apply the top-down rooting algorithm 
      to indels found in PyrD (dihydroorotate dehydrogenase), a key enzyme involved in 
      the de novo biosynthesis of pyrimidines, and HisA
      (P-ribosylformimino-AICAR-P-isomerase), an essential enzyme in the histidine
      biosynthesis pathway. Through the comparison of each indel with its two
      paralogous outgroups, we exclude the root of the tree of life from the clade that
      encompasses the Actinobacteria, the double-membrane prokaryotes, and their last
      common ancestor. In combination with previous indel rooting studies excluding the
      root from a clade consisting of the Firmicutes, the Archaea, and their last
      common ancestor, this provides evidence for a unique eubacterial root for the
      tree of life located between the actinobacterial-double-membrane clade and the
      archaeal-firmicute clade. Mapping the phylogenetic distributions of genes
      involved in peptidoglycan and lipid synthesis onto this rooted tree
      parsimoniously implies that the cenancestral prokaryotic population consisted of 
      organisms enclosed by a single, ester-linked lipid membrane, covered by a
      peptidoglycan layer.
FAU - Lake, James A
AU  - Lake JA
AD  - Department of Molecular, Cellular, and Developmental Biology, University of
      California, Los Angeles, California 90095, USA. Lake@mbi.ucla.edu
FAU - Servin, Jacqueline A
AU  - Servin JA
FAU - Herbold, Craig W
AU  - Herbold CW
FAU - Skophammer, Ryan G
AU  - Skophammer RG
LA  - eng
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 1.3.- (Oxidoreductases Acting on CH-CH Group Donors)
RN  - EC 1.3.5.2 (dihydroorotate dehydrogenase)
RN  - EC 5.- (Isomerases)
SB  - IM
MH  - Archaea/*classification
MH  - Bacteria/*classification
MH  - *Evolution, Molecular
MH  - INDEL Mutation/genetics
MH  - Isomerases/genetics
MH  - Oxidoreductases Acting on CH-CH Group Donors/genetics
MH  - *Phylogeny
EDAT- 2008/12/17 09:00
MHDA- 2009/02/03 09:00
CRDT- 2008/12/17 09:00
PHST- 2008/12/17 09:00 [entrez]
PHST- 2008/12/17 09:00 [pubmed]
PHST- 2009/02/03 09:00 [medline]
AID - 906543685 [pii]
AID - 10.1080/10635150802555933 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Dec;57(6):835-43. doi: 10.1080/10635150802555933.

PMID- 19085326
OWN - NLM
STAT- MEDLINE
DCOM- 20090202
LR  - 20081216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 6
DP  - 2008 Dec
TI  - Selecting taxa to save or sequence: desirable criteria and a greedy solution.
PG  - 825-34
LID - 10.1080/10635150802552831 [doi]
AB  - Three desirable properties for any method of selecting a subset of evolutionary
      units (EUs) for conservation or for genomic sequencing are discussed. These
      properties are spread, stability, and applicability. We are motivated by a
      practical case in which the maximization of phylogenetic diversity (PD), which
      has been suggested as a suitable method, appears to lead to counterintuitive
      collections of EUs and does not meet these three criteria. We define a simple
      greedy algorithm (GREEDYMMD) as a close approximation to choosing the subset that
      maximizes the minimum pairwise distance (MMD) between EUs. GREEDYMMD satisfies
      our three criteria and may be a useful alternative to PD in real-world
      situations. In particular, we show that this method of selection is suitable
      under a model of biodiversity in which features arise and/or disappear during
      evolution. We also show that if distances between EUs satisfy the ultrametric
      condition, then GREEDYMMD delivers an optimal subset of EUs that maximizes both
      the minimum pairwise distance and the PD. Finally, because GREEDYMMD works with
      distances and does not require a tree, it is readily applicable to many data
      sets.
FAU - Bordewich, Magnus
AU  - Bordewich M
AD  - Department of Computer Science, Durham University, Durham, United Kingdom.
      m.j.r.bordewich@durham.ac.uk
FAU - Rodrigo, Allen G
AU  - Rodrigo AG
FAU - Semple, Charles
AU  - Semple C
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Classification/*methods
MH  - Genomics/*methods/standards
MH  - *Phylogeny
EDAT- 2008/12/17 09:00
MHDA- 2009/02/03 09:00
CRDT- 2008/12/17 09:00
PHST- 2008/12/17 09:00 [entrez]
PHST- 2008/12/17 09:00 [pubmed]
PHST- 2009/02/03 09:00 [medline]
AID - 906547611 [pii]
AID - 10.1080/10635150802552831 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Dec;57(6):825-34. doi: 10.1080/10635150802552831.

PMID- 18853367
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20081014
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - A justification for reporting the majority-rule consensus tree in Bayesian
      phylogenetics.
PG  - 814-21
LID - 10.1080/10635150802422308 [doi]
FAU - Holder, Mark T
AU  - Holder MT
AD  - Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence,
      Kansas 66045, USA. mtholder@ku.edu
FAU - Sukumaran, Jeet
AU  - Sukumaran J
FAU - Lewis, Paul O
AU  - Lewis PO
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Decision Theory
MH  - Evolution, Molecular
MH  - *Genetic Variation
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904008159 [pii]
AID - 10.1080/10635150802422308 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):814-21. doi: 10.1080/10635150802422308.

PMID- 18853366
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20181201
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - The use of mean instead of smallest interspecific distances exaggerates the size 
      of the "barcoding gap" and leads to misidentification.
PG  - 809-13
LID - 10.1080/10635150802406343 [doi]
FAU - Meier, Rudolf
AU  - Meier R
AD  - Department of Biological Sciences, University Scholars Programme, National
      University of Singapore, Singapore, Singapore. dbsmr@nus.edu.sg
FAU - Zhang, Guanyang
AU  - Zhang G
FAU - Ali, Farhan
AU  - Ali F
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Base Sequence
MH  - DNA/*genetics
MH  - Electronic Data Processing/*methods
MH  - Genetic Speciation
MH  - Genetic Variation/genetics
MH  - Phylogeny
MH  - Research Design
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904008883 [pii]
AID - 10.1080/10635150802406343 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):809-13. doi: 10.1080/10635150802406343.

PMID- 18853365
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20081014
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - Rooting and dating maples (Acer) with an uncorrelated-rates molecular clock:
      implications for north American/Asian disjunctions.
PG  - 795-808
LID - 10.1080/10635150802422282 [doi]
AB  - Simulations suggest that molecular clock analyses can correctly identify the root
      of a tree even when the clock assumption is severely violated. Clock-based
      rooting of phylogenies may be particularly useful when outgroup rooting is
      problematic. Here, we explore relaxed-clock rooting in the Acer/Dipteronia clade 
      of Sapindaceae, which comprises genera of highly uneven species richness and
      problematic mutual monophyly. Using an approach that does not presuppose rate
      autocorrelation between ancestral and descendant branches and hence does not
      require a rooted a priori topology, we analyzed data from up to seven chloroplast
      loci for some 50 ingroup species. For comparison, we used midpoint and outgroup
      rooting and dating methods that rely on rooted input trees, namely penalized
      likelihood, a Bayesian autocorrelated-rates model, and a strict clock. The
      chloroplast sequences used here reject a single global substitution rate, and the
      assumption of autocorrelated rates was also rejected. The root was placed between
      Acer and Dipteronia by all three rooting methods, albeit with low statistical
      support. Analyses of Acer diversification with a lineage-through-time plot and
      different survival models, although sensitive to missing data, suggest a gradual 
      decrease in the average diversification rate. The nine North American species of 
      Acer diverged from their nearest relatives at widely different times: eastern
      American Acer diverged in the Oligocene and Late Miocene; western American
      species in the Late Eocene and Mid Miocene; and the Acer core clade, including A.
      saccharum, dates to the Miocene. Recent diversification in North America is
      strikingly rare compared to diversification in eastern Asia.
FAU - Renner, Susanne S
AU  - Renner SS
AD  - Department of Biology, University of Munich, Munich, Germany.
      renner@lrz.uni-muenchen.de
FAU - Grimm, Guido W
AU  - Grimm GW
FAU - Schneeweiss, Gerald M
AU  - Schneeweiss GM
FAU - Stuessy, Tod F
AU  - Stuessy TF
FAU - Ricklefs, Robert E
AU  - Ricklefs RE
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Acer/*genetics
MH  - Asia
MH  - Demography
MH  - *Evolution, Molecular
MH  - Fossils
MH  - Models, Genetic
MH  - North America
MH  - Software
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904007721 [pii]
AID - 10.1080/10635150802422282 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):795-808. doi: 10.1080/10635150802422282.

PMID- 18853364
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20081014
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - Trees of trees: an approach to comparing multiple alternative phylogenies.
PG  - 785-94
LID - 10.1080/10635150802424072 [doi]
AB  - Phylogenetic analysis very commonly produces several alternative trees for a
      given fixed set of taxa. For example, different sets of orthologous genes may be 
      analyzed, or the analysis may sample from a distribution of probable trees. This 
      article describes an approach to comparing and visualizing multiple alternative
      phylogenies via the idea of a "tree of trees" or "meta-tree." A meta-tree
      clusters phylogenies with similar topologies together in the same way that a
      phylogeny clusters species with similar DNA sequences. Leaf nodes on a meta-tree 
      correspond to the original set of phylogenies given by some analysis, whereas
      interior nodes correspond to certain consensus topologies. The construction of
      meta-trees is motivated by analogy with construction of a most parsimonious tree 
      for DNA data, but instead of using DNA letters, in a meta-tree the characters are
      partitions or splits of the set of taxa. An efficient algorithm for meta-tree
      construction is described that makes use of a known relationship between the
      majority consensus and parsimony in terms of gain and loss of splits. To
      illustrate these ideas meta-trees are constructed for two datasets: a set of gene
      trees for species of yeast and trees from a bootstrap analysis of a set of gene
      trees in ray-finned fish. A software tool for constructing meta-trees and
      comparing alternative phylogenies is available online, and the source code can be
      obtained from the author.
FAU - Nye, Tom M W
AU  - Nye TM
AD  - School of Mathematics and Statistics, Newcastle University, Newcastle, UK.
      tom.nye@ncl.ac.uk
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Classification/methods
MH  - DNA/genetics
MH  - *Evolution, Molecular
MH  - Fishes/genetics
MH  - *Genetic Variation
MH  - *Phylogeny
MH  - Plants/genetics
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904007528 [pii]
AID - 10.1080/10635150802424072 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):785-94. doi: 10.1080/10635150802424072.

PMID- 18853363
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - Wagner and Dollo: a stochastic duet by composing two parsimonious solos.
PG  - 772-84
LID - 10.1080/10635150802434394 [doi]
AB  - New contributions toward generalizing evolutionary models expand greatly our
      ability to analyze complex evolutionary characters and advance phylogeny
      reconstruction. In this article, we extend the binary stochastic Dollo model to
      allow for multi-state characters. In doing so, we align previously incompatible
      Wagner and Dollo parsimony principles under a common probabilistic framework by
      embedding arbitrary continuous-time Markov chains into the binary stochastic
      Dollo model. This approach enables us to analyze character traits that exhibit
      both Dollo and Wagner characteristics throughout their evolutionary histories.
      Utilizing Bayesian inference, we apply our novel model to analyze intron
      conservation patterns and the evolution of alternatively spliced exons. The
      generalized framework we develop demonstrates potential in distinguishing between
      phylogenetic hypotheses and providing robust estimates of evolutionary rates.
      Moreover, for the two applications analyzed here, our framework is the first to
      provide an adequate stochastic process for the data. We discuss possible
      extensions to the framework from both theoretical and applied perspectives.
FAU - Alekseyenko, Alexander V
AU  - Alekseyenko AV
AD  - Department of Biomathematics, David Geffen School of Medicine at UCLA, Los
      Angeles, California 90095, USA. alexander.alekseyenko@ucla.edu
FAU - Lee, Christopher J
AU  - Lee CJ
FAU - Suchard, Marc A
AU  - Suchard MA
LA  - eng
GR  - R01 GM086887/GM/NIGMS NIH HHS/United States
GR  - U54-RR021813/RR/NCRR NIH HHS/United States
GR  - GM008185)/GM/NIGMS NIH HHS/United States
GR  - T32 GM008185/GM/NIGMS NIH HHS/United States
GR  - U54 RR021813/RR/NCRR NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Alternative Splicing
MH  - Animals
MH  - Anopheles
MH  - *Biological Evolution
MH  - Caenorhabditis elegans
MH  - Drosophila melanogaster
MH  - Exons
MH  - Humans
MH  - *Models, Genetic
MH  - *Stochastic Processes
PMC - PMC4677801
MID - NIHMS741975
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904009007 [pii]
AID - 10.1080/10635150802434394 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):772-84. doi: 10.1080/10635150802434394.

PMID- 18853362
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20081014
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - A rapid bootstrap algorithm for the RAxML Web servers.
PG  - 758-71
LID - 10.1080/10635150802429642 [doi]
AB  - Despite recent advances achieved by application of high-performance computing
      methods and novel algorithmic techniques to maximum likelihood (ML)-based
      inference programs, the major computational bottleneck still consists in the
      computation of bootstrap support values. Conducting a probably insufficient
      number of 100 bootstrap (BS) analyses with current ML programs on large
      datasets-either with respect to the number of taxa or base pairs-can easily
      require a month of run time. Therefore, we have developed, implemented, and
      thoroughly tested rapid bootstrap heuristics in RAxML (Randomized Axelerated
      Maximum Likelihood) that are more than an order of magnitude faster than current 
      algorithms. These new heuristics can contribute to resolving the computational
      bottleneck and improve current methodology in phylogenetic analyses.
      Computational experiments to assess the performance and relative accuracy of
      these heuristics were conducted on 22 diverse DNA and AA (amino acid), single
      gene as well as multigene, real-world alignments containing 125 up to 7764
      sequences. The standard BS (SBS) and rapid BS (RBS) values drawn on the
      best-scoring ML tree are highly correlated and show almost identical average
      support values. The weighted RF (Robinson-Foulds) distance between SBS- and
      RBS-based consensus trees was smaller than 6% in all cases (average 4%). More
      importantly, RBS inferences are between 8 and 20 times faster (average 14.73)
      than SBS analyses with RAxML and between 18 and 495 times faster than BS analyses
      with competing programs, such as PHYML or GARLI. Moreover, this performance
      improvement increases with alignment size. Finally, we have set up two freely
      accessible Web servers for this significantly improved version of RAxML that
      provide access to the 200-CPU cluster of the Vital-IT unit at the Swiss Institute
      of Bioinformatics and the 128-CPU cluster of the CIPRES project at the San Diego 
      Supercomputer Center. These Web servers offer the possibility to conduct
      large-scale phylogenetic inferences to a large part of the community that does
      not have access to, or the expertise to use, high-performance computing
      resources.
FAU - Stamatakis, Alexandros
AU  - Stamatakis A
AD  - Exelixis Lab, Teaching and Research Unit Bioinformatics, Department of Computer
      Science, Ludwig-Maximilians-University Munich, Munich, Germany.
      stamatakis@bio.ifi.lmu.de
FAU - Hoover, Paul
AU  - Hoover P
FAU - Rougemont, Jacques
AU  - Rougemont J
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Computers
MH  - Evolution, Molecular
MH  - *Internet
MH  - Likelihood Functions
MH  - Phylogeny
MH  - Plants/genetics
MH  - *Software
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904007647 [pii]
AID - 10.1080/10635150802429642 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):758-71. doi: 10.1080/10635150802429642.

PMID- 18853361
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - Statistical assignment of DNA sequences using Bayesian phylogenetics.
PG  - 750-7
LID - 10.1080/10635150802422316 [doi]
AB  - We provide a new automated statistical method for DNA barcoding based on a
      Bayesian phylogenetic analysis. The method is based on automated database
      sequence retrieval, alignment, and phylogenetic analysis using a custom-built
      program for Bayesian phylogenetic analysis. We show on real data that the method 
      outperforms Blast searches as a measure of confidence and can help eliminate 80% 
      of all false assignment based on best Blast hit. However, the most important
      advance of the method is that it provides statistically meaningful measures of
      confidence. We apply the method to a re-analysis of previously published ancient 
      DNA data and show that, with high statistical confidence, most of the published
      sequences are in fact of Neanderthal origin. However, there are several cases of 
      chimeric sequences that are comprised of a combination of both Neanderthal and
      modern human DNA.
FAU - Munch, Kasper
AU  - Munch K
AD  - Department of Integrative Biology, University of California, Berkeley, California
      94720-3140, USA.
FAU - Boomsma, Wouter
AU  - Boomsma W
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
FAU - Willerslev, Eske
AU  - Willerslev E
FAU - Nielsen, Rasmus
AU  - Nielsen R
LA  - eng
GR  - R01 GM069801/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Computer Simulation
MH  - DNA/*genetics
MH  - Insecta/genetics
MH  - *Models, Genetic
MH  - *Models, Statistical
MH  - *Phylogeny
MH  - Plants/genetics
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904007579 [pii]
AID - 10.1080/10635150802422316 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):750-7. doi: 10.1080/10635150802422316.

PMID- 18853360
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20081014
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - Multiple colonizations, in situ speciation, and volcanism-associated
      stepping-stone dispersals shaped the phylogeography of the Macaronesian red
      fescues (Festuca L., Gramineae).
PG  - 732-49
LID - 10.1080/10635150802302450 [doi]
AB  - Whereas examples of insular speciation within the endemic-rich Macaronesian
      hotspot flora have been documented, the phylogeography of recently evolved plants
      in the region has received little attention. The Macaronesian red fescues
      constitute a narrow and recent radiation of four closely related diploid species 
      distributed in the Canary Islands (F. agustinii), Madeira (F. jubata), and the
      Azores (F. francoi and F. petraea), with a single extant relative distributed in 
      mainland southwest Europe (F. rivularis). Bayesian structure and priority
      consensus tree approaches and population spatial correlations between genetic,
      geographical, and dispersal distances were used to elucidate the
      phylogeographical patterns of these grasses. Independent versus related origins
      and dispersal versus isolation by distance (IBD) hypotheses were tested to
      explain the genetic differentiation of species and populations, respectively.
      Genetic structure was found to be geographically distributed among the
      archipelagos and the islands endemics. The high number of shared AFLP fragments
      in all four species suggests a recent single origin from a continental Pliocene
      ancestor. However, the strong allelic structure detected among the Canarian,
      Madeiran, and Azorean endemics and the significant standardized residual values
      obtained from structured Bayesian analysis for pairwise related origin hypotheses
      strongly supported the existence of three independent continental-oceanic
      colonization events. The Canarian F. agustinii, the Madeiran F. jubata, and the
      two sister F. francoi and F. petraea Azorean species likely evolved from
      different continental founders in their respective archipelagos. Despite the
      short span of time elapsed since colonization, the two sympatric Azorean species 
      probably diverged in situ, following ecological adaptation, from a common
      ancestor that arrived from the near mainland. Simple dispersal hypotheses
      explained most of the genetic variation at the species level better than IBD
      models. The optimal dispersal model for F. agustinii was a bidirectional
      centripetal stepping-stone colonization pattern, an eastern-to-western
      volcanism-associated dispersion was favored for F. francoi, whereas for the
      recently derived F. petraea a counterintuitive direction of colonization
      (west-to-east) was suggested. The population-based phylogeographical trends
      deduced from our study could be used as predictive models for other Macaronesian 
      plant endemics with similar distribution areas and dispersal abilities.
FAU - Diaz-Perez, Antonio
AU  - Diaz-Perez A
AD  - Department of Agriculture (Botany), High Polytechnic School of Huesca, University
      of Zaragoza, Huesca, Spain. adiape@cantv.net
FAU - Sequeira, Miguel
AU  - Sequeira M
FAU - Santos-Guerra, Arnoldo
AU  - Santos-Guerra A
FAU - Catalan, Pilar
AU  - Catalan P
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Demography
MH  - Festuca/*genetics/*microbiology
MH  - *Genetic Speciation
MH  - Genetic Variation
MH  - *Phylogeny
MH  - Spain
MH  - Volcanic Eruptions
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904007884 [pii]
AID - 10.1080/10635150802302450 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):732-49. doi: 10.1080/10635150802302450.

PMID- 18853359
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20081014
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - Comparison of species tree methods for reconstructing the phylogeny of bearded
      manakins (Aves: Pipridae, Manacus) from multilocus sequence data.
PG  - 719-31
LID - 10.1080/10635150802422290 [doi]
AB  - Although the power of multi-locus data in estimating species trees is apparent,
      it is also clear that the analytical methodologies for doing so are still
      maturing. For example, of the methods currently available for estimating species 
      trees from multilocus data, the Bayesian method introduced by Liu and Pearl
      (2007; BEST) is the only one that provides nodal support values. Using gene
      sequences from five nuclear loci, we explored two analytical methods (deep
      coalescence and BEST) to reconstruct the species tree of the five primary Manacus
      OTUs: M. aurantiacus, M. candei, M. vitellinus, populations of M. manacus from
      west of the Andes (M. manacus (w)), and populations of M. manacus from east of
      the Andes (M. manacus (e)). Both BEST and deep coalescence supported a sister
      relationship between M. vitellinus and M. manacus (w). A lower probability tree
      from the BEST analysis and one of the most parsimonious deep coalescence trees
      also supported a sister relationship between M. candei and M. aurantiacus.
      Because hybrid zones connect the distributions of most Manacus species, we
      examined the potential influence of post-divergence gene flow on the sister
      relationship of parapatrically distributed M. vitellinus and M. manacus (w). An
      isolation-with-migration (IM) analysis found relatively high levels of gene flow 
      between M. vitellinus and M. manacus (w). Whether the gene flow is obscuring a
      true sister relationship between M. manacus (w) and M. manacus (e) remained
      unclear, pointing to the need for more detailed models accommodating
      multispecies, multilocus DNA sequence data.
FAU - Brumfield, Robb T
AU  - Brumfield RT
AD  - Museum of Natural Science, Baton Rouge, Louisiana 70803, USA. brumfld@lsu.edu
FAU - Liu, Liang
AU  - Liu L
FAU - Lum, David E
AU  - Lum DE
FAU - Edwards, Scott V
AU  - Edwards SV
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Birds/*classification/*genetics
MH  - Demography
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Recombination, Genetic
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904007372 [pii]
AID - 10.1080/10635150802422290 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):719-31. doi: 10.1080/10635150802422290.

PMID- 18853358
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20081014
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - Statistical tests between competing hypotheses of Hox cluster evolution.
PG  - 708-18
LID - 10.1080/10635150802430079 [doi]
AB  - The Hox genes encode transcription factors that play vital roles in the
      anterior-posterior patterning of all bilaterian phyla studied to date.
      Additionally, the gain of Hox genes by duplication has been widely implicated as 
      a driving force in the evolution of animal body plans. Because of this,
      reconstructing the evolution of the Hox cluster has been the focus of intense
      research interest. It has been commonly assumed that an ancestral four-gene
      ProtoHox cluster was duplicated early in animal evolution to give rise to the Hox
      and ParaHox clusters. However, this hypothesis has recently been called into
      question, and a number of alternative hypotheses of Hox and ParaHox gene
      evolution have been proposed. Here, we present the first statistical comparisons 
      of current hypotheses of Hox and ParaHox gene evolution. We use two statistical
      methods that represent two different approaches to the treatment of phylogenetic 
      uncertainty. In the first method, we estimate the maximum-likelihood tree for
      each hypothesis and compare these trees to one another using a parametric
      bootstrapping approach. In the second method, we use Bayesian phylogenetics to
      estimate the posterior distribution of trees, then we calculate the support for
      each hypothesis from this distribution. The results of both methods are largely
      congruent. We find that we are able to reject five out of the eight current
      hypotheses of Hox and ParaHox gene evolution that we consider. We conclude that
      the ProtoHox cluster is likely to have contained either three or four genes but
      that there is insufficient phylogenetic signal in the homeodomains to distinguish
      between these alternatives.
FAU - Lanfear, Robert
AU  - Lanfear R
AD  - Centre for the Study of Evolution, School of Life Sciences, University of Sussex,
      Brighton, UK. Rob.Lanfear@anu.edu.au
FAU - Bromham, Lindell
AU  - Bromham L
LA  - eng
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Homeodomain Proteins)
RN  - 0 (Transcription Factors)
SB  - IM
MH  - Animals
MH  - Cluster Analysis
MH  - *Evolution, Molecular
MH  - Homeodomain Proteins/*genetics
MH  - Invertebrates/genetics
MH  - Mice
MH  - Models, Genetic
MH  - Multigene Family/*genetics
MH  - Phylogeny
MH  - Plants/genetics
MH  - Transcription Factors/*genetics
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904008019 [pii]
AID - 10.1080/10635150802430079 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):708-18. doi: 10.1080/10635150802430079.

PMID- 18853357
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - A comparative study in ancestral range reconstruction methods: retracing the
      uncertain histories of insular lineages.
PG  - 693-707
LID - 10.1080/10635150802426473 [doi]
AB  - Island systems have long been useful models for understanding lineage
      diversification in a geographic context, especially pertaining to the importance 
      of dispersal in the origin of new clades. Here we use a well-resolved phylogeny
      of the flowering plant genus Cyrtandra (Gesneriaceae) from the Pacific Islands to
      compare four methods of inferring ancestral geographic ranges in islands: two
      developed for character-state reconstruction that allow only single-island ranges
      and do not explicitly associate speciation with range evolution (Fitch parsimony 
      [FP; parsimony-based] and stochastic mapping [SM; likelihood-based]) and two
      methods developed specifically for ancestral range reconstruction, in which
      widespread ranges (spanning islands) are integral to inferences about speciation 
      scenarios (dispersal-vicariance analysis [DIVA; parsimony-based] and
      dispersal-extinction-cladogenesis [DEC; likelihood-based]). The methods yield
      conflicting results, which we interpret in light of their respective assumptions.
      FP exhibits the least power to unequivocally reconstruct ranges, likely due to a 
      combination of having flat (uninformative) transition costs and not using branch 
      length information. SM reconstructions generally agree with a prior hypothesis
      about dispersal-driven speciation across the Pacific, despite the conceptual
      mismatch between its character-based model and this mode of range evolution. In
      contrast with narrow extant ranges for species of Cyrtandra, DIVA reconstructs
      broad ancestral ranges at many nodes. DIVA results also conflict with geological 
      information on island ages; we attribute these conflicts to the parsimony
      criterion not considering branch lengths or time, as well as vicariance being the
      sole means of divergence for widespread ancestors. DEC analyses incorporated
      geological information on island ages and allowed prior hypotheses about range
      size and dispersal rates to be evaluated in a likelihood framework and gave more 
      nuanced inferences about range evolution and the geography of speciation than
      other methods tested. However, ancestral ranges at several nodes could not be
      conclusively resolved, due possibly to uncertainty in the phylogeny or the
      relative complexity of the underlying model. Of the methods tested, SM and DEC
      both converge on plausible hypotheses for area range histories in Cyrtandra, due 
      in part to the consideration of branch lengths and/or timing of events. We
      suggest that DEC model-based methods for ancestral range inference could be
      improved by adopting a Bayesian SM approach, in which stochastic sampling of
      complete geographic histories could be integrated over alternative phylogenetic
      topologies. Likelihood-based estimates of ancestral ranges for Cyrtandra suggest 
      a major dispersal route into the Pacific through the islands of Fiji and Samoa,
      motivating future biogeographic investigation of this poorly known region.
FAU - Clark, John R
AU  - Clark JR
AD  - School of Biological Sciences, Washington State University, P.O. Box 644236,
      Pullman, Washington, USA. johnrobertclark@gmail.com
FAU - Ree, Richard H
AU  - Ree RH
FAU - Alfaro, Michael E
AU  - Alfaro ME
FAU - King, Matthew G
AU  - King MG
FAU - Wagner, Warren L
AU  - Wagner WL
FAU - Roalson, Eric H
AU  - Roalson EH
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Computer Simulation
MH  - DNA, Plant/genetics
MH  - Ecosystem
MH  - *Evolution, Molecular
MH  - Geography
MH  - Likelihood Functions
MH  - Magnoliopsida/*genetics
MH  - Models, Genetic
MH  - Pacific Ocean
MH  - Phylogeny
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904008219 [pii]
AID - 10.1080/10635150802426473 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):693-707. doi: 10.1080/10635150802426473.

PMID- 18853356
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20081014
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - A model-based approach to study nearest-neighbor influences reveals complex
      substitution patterns in non-coding sequences.
PG  - 675-92
LID - 10.1080/10635150802422324 [doi]
AB  - In this article, we present a likelihood-based framework for modeling site
      dependencies. Our approach builds upon standard evolutionary models but
      incorporates site dependencies across the entire tree by letting the evolutionary
      parameters in these models depend upon the ancestral states at the neighboring
      sites. It thus avoids the need for introducing new and high-dimensional
      evolutionary models for site-dependent evolution. We propose a Markov chain Monte
      Carlo approach with data augmentation to infer the evolutionary parameters under 
      our model. Although our approach allows for wide-ranging site dependencies, we
      illustrate its use, in two non-coding datasets, in the case of nearest-neighbor
      dependencies (i.e., evolution directly depending only upon the immediate flanking
      sites). The results reveal that the general time-reversible model with
      nearest-neighbor dependencies substantially improves the fit to the data as
      compared to the corresponding model with site independence. Using the parameter
      estimates from our model, we elaborate on the importance of the 5-methylcytosine 
      deamination process (i.e., the CpG effect) and show that this process also
      depends upon the 5' neighboring base identity. We hint at the possibility of a
      so-called TpA effect and show that the observed substitution behavior is very
      complex in the light of dinucleotide estimates. We also discuss the presence of
      CpG effects in a nuclear small subunit dataset and find significant evidence that
      evolutionary models incorporating context-dependent effects perform substantially
      better than independent-site models and in some cases even outperform models that
      incorporate varying rates across sites.
FAU - Baele, Guy
AU  - Baele G
AD  - Department of Applied Mathematics and Computer Science, Ghent University, Ghent, 
      Belgium.
FAU - Van de Peer, Yves
AU  - Van de Peer Y
FAU - Vansteelandt, Stijn
AU  - Vansteelandt S
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Untranslated)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Base Sequence
MH  - DNA/*genetics
MH  - Evolution, Molecular
MH  - Gene Expression Regulation
MH  - Likelihood Functions
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - RNA, Untranslated/*genetics
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904008488 [pii]
AID - 10.1080/10635150802422324 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):675-92. doi: 10.1080/10635150802422324.

PMID- 18853355
OWN - NLM
STAT- MEDLINE
DCOM- 20081217
LR  - 20081014
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 5
DP  - 2008 Oct
TI  - Penalized likelihood phylogenetic inference: bridging the parsimony-likelihood
      gap.
PG  - 665-74
LID - 10.1080/10635150802422274 [doi]
AB  - The increasing diversity and heterogeneity of molecular data for phylogeny
      estimation has led to development of complex models and model-based estimators.
      Here, we propose a penalized likelihood (PL) framework in which the levels of
      complexity in the underlying model can be smoothly controlled. We demonstrate the
      PL framework for a four-taxon tree case and investigate its properties. The PL
      framework yields an estimator in which the majority of currently employed
      estimators such as the maximum-parsimony estimator, homogeneous likelihood
      estimator, gamma mixture likelihood estimator, etc., become special cases of a
      single family of PL estimators. Furthermore, using the appropriate penalty
      function, the complexity of the underlying models can be partitioned into
      separately controlled classes allowing flexible control of model complexity.
FAU - Kim, Junhyong
AU  - Kim J
AD  - Department of Biology and Penn Genome Frontiers Institute, University of
      Pennsylvania, Philadelphia, Pennsylvania, USA. junhyong@sas.upenn.edu
FAU - Sanderson, Michael J
AU  - Sanderson MJ
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Drosophila/genetics
MH  - Evolution, Molecular
MH  - Genetic Variation
MH  - *Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2008/10/15 09:00
MHDA- 2008/12/18 09:00
CRDT- 2008/10/15 09:00
PHST- 2008/10/15 09:00 [pubmed]
PHST- 2008/12/18 09:00 [medline]
PHST- 2008/10/15 09:00 [entrez]
AID - 904008679 [pii]
AID - 10.1080/10635150802422274 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Oct;57(5):665-74. doi: 10.1080/10635150802422274.

PMID- 18709601
OWN - NLM
STAT- MEDLINE
DCOM- 20100930
LR  - 20080818
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 4
DP  - 2008 Aug
TI  - Phylogeny estimation and alignment via POY versus Clustal + PAUP*: a response to 
      Ogden and Rosenberg (2007).
PG  - 653-7
LID - 10.1080/10635150802302476 [doi]
FAU - Lehtonen, Samuli
AU  - Lehtonen S
AD  - Department of Biology, Section of Biodiversity and Environmental Science,
      University of Turku, Turku, Finland. samile@utu.fi
LA  - eng
PT  - Comment
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2007 Apr;56(2):182-93. PMID: 17454974
MH  - Classification/methods
MH  - Computer Simulation
MH  - *Phylogeny
MH  - *Sequence Alignment
MH  - *Software
EDAT- 2008/08/19 09:00
MHDA- 2010/10/01 06:00
CRDT- 2008/08/19 09:00
PHST- 2008/08/19 09:00 [pubmed]
PHST- 2010/10/01 06:00 [medline]
PHST- 2008/08/19 09:00 [entrez]
AID - 901677676 [pii]
AID - 10.1080/10635150802302476 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Aug;57(4):653-7. doi: 10.1080/10635150802302476.

PMID- 18709600
OWN - NLM
STAT- MEDLINE
DCOM- 20100930
LR  - 20080818
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 4
DP  - 2008 Aug
TI  - More taxa are not necessarily better for the reconstruction of ancestral
      character states.
PG  - 647-53
LID - 10.1080/10635150802203898 [doi]
FAU - Li, Guoliang
AU  - Li G
AD  - Department of Computer Science, National University of Singapore, Singapore.
      ligl@comp.nus.edu.sg
FAU - Steel, Mike
AU  - Steel M
FAU - Zhang, Louxin
AU  - Zhang L
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Models, Biological
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2008/08/19 09:00
MHDA- 2010/10/01 06:00
CRDT- 2008/08/19 09:00
PHST- 2008/08/19 09:00 [pubmed]
PHST- 2010/10/01 06:00 [medline]
PHST- 2008/08/19 09:00 [entrez]
AID - 901677598 [pii]
AID - 10.1080/10635150802203898 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Aug;57(4):647-53. doi: 10.1080/10635150802203898.

PMID- 18709599
OWN - NLM
STAT- MEDLINE
DCOM- 20100930
LR  - 20080818
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 4
DP  - 2008 Aug
TI  - Assessing the performance of single-copy genes for recovering robust phylogenies.
PG  - 613-27
LID - 10.1080/10635150802306527 [doi]
AB  - Phylogenies involving nonmodel species are based on a few genes, mostly chosen
      following historical or practical criteria. Because gene trees are sometimes
      incongruent with species trees, the resulting phylogenies may not accurately
      reflect the evolutionary relationships among species. The increase in
      availability of genome sequences now provides large numbers of genes that could
      be used for building phylogenies. However, for practical reasons only a few genes
      can be sequenced for a wide range of species. Here we asked whether we can
      identify a few genes, among the single-copy genes common to most fungal genomes, 
      that are sufficient for recovering accurate and well-supported phylogenies. Fungi
      represent a model group for phylogenomics because many complete fungal genomes
      are available. An automated procedure was developed to extract single-copy
      orthologous genes from complete fungal genomes using a Markov Clustering
      Algorithm (Tribe-MCL). Using 21 complete, publicly available fungal genomes with 
      reliable protein predictions, 246 single-copy orthologous gene clusters were
      identified. We inferred the maximum likelihood trees using the individual
      orthologous sequences and constructed a reference tree from concatenated protein 
      alignments. The topologies of the individual gene trees were compared to that of 
      the reference tree using three different methods. The performance of individual
      genes in recovering the reference tree was highly variable. Gene size and the
      number of variable sites were highly correlated and significantly affected the
      performance of the genes, but the average substitution rate did not. Two genes
      recovered exactly the same topology as the reference tree, and when concatenated 
      provided high bootstrap values. The genes typically used for fungal phylogenies
      did not perform well, which suggests that current fungal phylogenies based on
      these genes may not accurately reflect the evolutionary relationships among
      species. Analyses on subsets of species showed that the phylogenetic performance 
      did not seem to depend strongly on the sample. We expect that the best-performing
      genes identified here will be very useful for phylogenetic studies of fungi, at
      least at a large taxonomic scale. Furthermore, we compare the method developed
      here for finding genes for building robust phylogenies with previous ones and we 
      advocate that our method could be applied to other groups of organisms when more 
      complete genomes are available.
FAU - Aguileta, G
AU  - Aguileta G
AD  - Laboratoire Ecologie, Systematique et Evolution, Universite Paris-Sud, Orsay,
      UMR8079, Orsay, Cedex, France. gabriela.aguileta@u-psud.fr
FAU - Marthey, S
AU  - Marthey S
FAU - Chiapello, H
AU  - Chiapello H
FAU - Lebrun, M-H
AU  - Lebrun MH
FAU - Rodolphe, F
AU  - Rodolphe F
FAU - Fournier, E
AU  - Fournier E
FAU - Gendrault-Jacquemard, A
AU  - Gendrault-Jacquemard A
FAU - Giraud, T
AU  - Giraud T
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Fungi/classification/genetics
MH  - Genes, Fungal/genetics
MH  - Likelihood Functions
MH  - Multigene Family
MH  - *Phylogeny
EDAT- 2008/08/19 09:00
MHDA- 2010/10/01 06:00
CRDT- 2008/08/19 09:00
PHST- 2008/08/19 09:00 [pubmed]
PHST- 2010/10/01 06:00 [medline]
PHST- 2008/08/19 09:00 [entrez]
AID - 901677523 [pii]
AID - 10.1080/10635150802306527 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Aug;57(4):613-27. doi: 10.1080/10635150802306527.

PMID- 18709598
OWN - NLM
STAT- MEDLINE
DCOM- 20100930
LR  - 20080818
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 4
DP  - 2008 Aug
TI  - On the distributions of bootstrap support and posterior distributions for a star 
      tree.
PG  - 602-12
LID - 10.1080/10635150802302468 [doi]
AB  - Several authors have recently noted that when data are generated from a star
      topology, posterior probabilities can often be very large, even with arbitrarily 
      large sequence lengths. This is counter to intuition, which suggests convergence 
      to the limit of equal probability for each topology. Here the limiting
      distributions of bootstrap support and posterior probabilities are obtained for a
      four-taxon star tree. Theoretical results are given, providing confirmation that 
      this counterintuitive phenomenon holds for both posterior probabilities and
      bootstrap support. For large samples the limiting results for posterior
      probabilities are the same regardless of the prior. With equal-length terminal
      edges, the limiting distribution is similar but not the same across different
      choices for the lengths of the edges. In contrast to previous results, the case
      of unequal lengths of terminal edges is considered. With two long edges, the
      posterior probability of the tree with long edges together tends to be much
      larger. Using the neighbor-joining algorithm, with equal edge lengths, the
      distribution of bootstrap support tends to be qualitatively comparable to
      posterior probabilities. As with posterior probabilities, when two of the edges
      are long, bootstrap support for the tree with long branches together tends to be 
      large. The bias is less pronounced, however, as the distribution of bootstrap
      support gets close to uniform for this tree, whereas posterior probabilities are 
      much more likely to be large. Our findings for maximum likelihood estimation are 
      based entirely on simulation and in contrast suggest that bootstrap support tends
      to be fairly constant across edge-length choices.
FAU - Susko, Edward
AU  - Susko E
AD  - Department of Mathematics and Statistics, Dalhousie University, Halifax, Nova
      Scotia, Canada. susko@mathstat.dal.ca
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Likelihood Functions
MH  - Phylogeny
EDAT- 2008/08/19 09:00
MHDA- 2010/10/01 06:00
CRDT- 2008/08/19 09:00
PHST- 2008/08/19 09:00 [pubmed]
PHST- 2010/10/01 06:00 [medline]
PHST- 2008/08/19 09:00 [entrez]
AID - 901677716 [pii]
AID - 10.1080/10635150802302468 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Aug;57(4):602-12. doi: 10.1080/10635150802302468.

PMID- 18709597
OWN - NLM
STAT- MEDLINE
DCOM- 20100930
LR  - 20080818
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 4
DP  - 2008 Aug
TI  - Phylogenetic signal, evolutionary process, and rate.
PG  - 591-601
LID - 10.1080/10635150802302427 [doi]
AB  - A recent advance in the phylogenetic comparative analysis of continuous traits
      has been explicit, model-based measurement of "phylogenetic signal" in data sets 
      composed of observations collected from species related by a phylogenetic tree.
      Phylogenetic signal is a measure of the statistical dependence among species'
      trait values due to their phylogenetic relationships. Although phylogenetic
      signal is a measure of pattern (statistical dependence), there has nonetheless
      been a widespread propensity in the literature to attribute this pattern to
      aspects of the evolutionary process or rate. This may be due, in part, to the
      perception that high evolutionary rate necessarily results in low phylogenetic
      signal; and, conversely, that low evolutionary rate or stabilizing selection
      results in high phylogenetic signal (due to the resulting high resemblance
      between related species). In this study, we use individual-based numerical
      simulations on stochastic phylogenetic trees to clarify the relationship between 
      phylogenetic signal, rate, and evolutionary process. Under the simplest model for
      quantitative trait evolution, homogeneous rate genetic drift, there is no
      relation between evolutionary rate and phylogenetic signal. For other
      circumstances, such as functional constraint, fluctuating selection, niche
      conservatism, and evolutionary heterogeneity, the relationship between process,
      rate, and phylogenetic signal is complex. For these reasons, we recommend against
      interpretations of evolutionary process or rate based on estimates of
      phylogenetic signal.
FAU - Revell, Liam J
AU  - Revell LJ
AD  - Department of Organismic and Evolutionary Biology, Harvard University, Cambridge,
      Massachusetts 02138, USA. lrevell@fas.harvard.edu
FAU - Harmon, Luke J
AU  - Harmon LJ
FAU - Collar, David C
AU  - Collar DC
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Genetic Drift
MH  - Models, Genetic
MH  - Mutation
MH  - *Phylogeny
EDAT- 2008/08/19 09:00
MHDA- 2010/10/01 06:00
CRDT- 2008/08/19 09:00
PHST- 2008/08/19 09:00 [pubmed]
PHST- 2010/10/01 06:00 [medline]
PHST- 2008/08/19 09:00 [entrez]
AID - 901677765 [pii]
AID - 10.1080/10635150802302427 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Aug;57(4):591-601. doi: 10.1080/10635150802302427.

PMID- 18686196
OWN - NLM
STAT- MEDLINE
DCOM- 20100930
LR  - 20080807
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 4
DP  - 2008 Aug
TI  - An integrative method for delimiting cohesion species: finding the
      population-species interface in a group of Californian trapdoor spiders with
      extreme genetic divergence and geographic structuring.
PG  - 628-46
LID - 10.1080/10635150802302443 [doi]
AB  - Here we present an objective, repeatable approach to delineating species when
      populations are divergent and highly structured geographically using the
      Californian trapdoor spider species complex Aptostichus atomarius Simon as a
      model system. This system is particularly difficult because under strict criteria
      of geographical concordance coupled with estimates of genetic divergence, an
      unrealistic number of population lineages would qualify as species (20 to 60).
      Our novel phylogeographic approach, which is generally applicable but
      particularly relevant to highly structured systems, uses genealogical exclusivity
      to establish a topological framework to examine lineages for genetic and
      ecological exchangeability in an effort to delimit cohesion species. Both
      qualitative assessments of habitat and niche-based distribution modeling are
      employed to evaluate selective regime and ecological interchangeability among
      genetic lineages; adaptive divergence among populations is weighted more heavily 
      than simple geographical concordance. Based on these analyses we conclude that
      five cohesion species should be recognized, three of which are new to science.
FAU - Bond, Jason E
AU  - Bond JE
AD  - Department of Biology, East Carolina University, Greenville, North Carolina
      27858, USA. bondja@ecu.edu
FAU - Stockman, Amy K
AU  - Stockman AK
LA  - eng
SI  - GENBANK/EU569876
SI  - GENBANK/EU569877
SI  - GENBANK/EU569878
SI  - GENBANK/EU569879
SI  - GENBANK/EU569880
SI  - GENBANK/EU569881
SI  - GENBANK/EU569882
SI  - GENBANK/EU569883
SI  - GENBANK/EU569884
SI  - GENBANK/EU569885
SI  - GENBANK/EU569886
SI  - GENBANK/EU569887
SI  - GENBANK/EU569888
SI  - GENBANK/EU569889
SI  - GENBANK/EU569890
SI  - GENBANK/EU569891
SI  - GENBANK/EU569892
SI  - GENBANK/EU569893
SI  - GENBANK/EU569894
SI  - GENBANK/EU569895
SI  - GENBANK/EU569896
SI  - GENBANK/EU569897
SI  - GENBANK/EU569898
SI  - GENBANK/EU569899
SI  - GENBANK/EU569900
SI  - GENBANK/EU569901
SI  - GENBANK/EU569902
SI  - GENBANK/EU569903
SI  - GENBANK/EU569904
SI  - GENBANK/EU569905
SI  - GENBANK/EU569906
SI  - GENBANK/EU569907
SI  - GENBANK/EU569908
SI  - GENBANK/EU569909
SI  - GENBANK/EU569910
SI  - GENBANK/EU569911
SI  - GENBANK/EU569912
SI  - GENBANK/EU569913
SI  - GENBANK/EU569914
SI  - GENBANK/EU569915
SI  - GENBANK/EU569916
SI  - GENBANK/EU569917
SI  - GENBANK/EU569918
SI  - GENBANK/EU569919
SI  - GENBANK/EU569920
SI  - GENBANK/EU569921
SI  - GENBANK/EU569922
SI  - GENBANK/EU569923
SI  - GENBANK/EU569924
SI  - GENBANK/EU569925
SI  - GENBANK/EU569926
SI  - GENBANK/EU569927
SI  - GENBANK/EU569928
SI  - GENBANK/EU569929
SI  - GENBANK/EU569930
SI  - GENBANK/EU569931
SI  - GENBANK/EU569932
SI  - GENBANK/EU569933
SI  - GENBANK/EU569934
SI  - GENBANK/EU569935
SI  - GENBANK/EU569936
SI  - GENBANK/EU569937
SI  - GENBANK/EU569938
SI  - GENBANK/EU569939
SI  - GENBANK/EU569940
SI  - GENBANK/EU569941
SI  - GENBANK/EU569942
SI  - GENBANK/EU569943
SI  - GENBANK/EU569944
SI  - GENBANK/EU569945
SI  - GENBANK/EU569946
SI  - GENBANK/EU569947
SI  - GENBANK/EU569948
SI  - GENBANK/EU569949
SI  - GENBANK/EU569950
SI  - GENBANK/EU569951
SI  - GENBANK/EU569952
SI  - GENBANK/EU569953
SI  - GENBANK/EU569954
SI  - GENBANK/EU569955
SI  - GENBANK/EU569956
SI  - GENBANK/EU569957
SI  - GENBANK/EU569958
SI  - GENBANK/EU569959
SI  - GENBANK/EU569960
SI  - GENBANK/EU569961
SI  - GENBANK/EU569962
SI  - GENBANK/EU569963
SI  - GENBANK/EU569964
SI  - GENBANK/EU569965
SI  - GENBANK/EU569966
SI  - GENBANK/EU569967
SI  - GENBANK/EU569968
SI  - GENBANK/EU569969
SI  - GENBANK/EU569970
SI  - GENBANK/EU569971
SI  - GENBANK/EU569972
SI  - GENBANK/EU569973
SI  - GENBANK/EU569974
SI  - GENBANK/EU569975
SI  - GENBANK/EU569976
SI  - GENBANK/EU569977
SI  - GENBANK/EU569978
SI  - GENBANK/EU569979
SI  - GENBANK/EU569980
SI  - GENBANK/EU569981
SI  - GENBANK/EU569982
SI  - GENBANK/EU569983
SI  - GENBANK/EU569984
SI  - GENBANK/EU569985
SI  - GENBANK/EU569986
SI  - GENBANK/EU569987
SI  - GENBANK/EU569988
SI  - GENBANK/EU569989
SI  - GENBANK/EU569990
SI  - GENBANK/EU569991
SI  - GENBANK/EU569992
SI  - GENBANK/EU569993
SI  - GENBANK/EU569994
SI  - GENBANK/EU569995
SI  - GENBANK/EU569996
SI  - GENBANK/EU569997
SI  - GENBANK/EU569998
SI  - GENBANK/EU569999
SI  - GENBANK/EU570000
SI  - GENBANK/EU570001
SI  - GENBANK/EU570002
SI  - GENBANK/EU570003
SI  - GENBANK/EU570004
SI  - GENBANK/EU570005
SI  - GENBANK/EU570006
SI  - GENBANK/EU570007
SI  - GENBANK/EU570008
SI  - GENBANK/EU570009
SI  - GENBANK/EU570010
SI  - GENBANK/EU570011
SI  - GENBANK/EU570012
SI  - GENBANK/EU570013
SI  - GENBANK/EU570014
SI  - GENBANK/EU570015
SI  - GENBANK/EU570016
SI  - GENBANK/EU570017
SI  - GENBANK/EU570018
SI  - GENBANK/EU570019
SI  - GENBANK/EU570020
SI  - GENBANK/EU570021
SI  - GENBANK/EU570022
SI  - GENBANK/EU570023
SI  - GENBANK/EU570024
SI  - GENBANK/EU570025
SI  - GENBANK/EU570026
SI  - GENBANK/EU570027
SI  - GENBANK/EU570028
SI  - GENBANK/EU570029
SI  - GENBANK/EU570030
SI  - GENBANK/EU570031
SI  - GENBANK/EU570032
SI  - GENBANK/EU570033
SI  - GENBANK/EU570034
SI  - GENBANK/EU570035
SI  - GENBANK/EU570036
SI  - GENBANK/EU570037
SI  - GENBANK/EU570038
SI  - GENBANK/EU570039
SI  - GENBANK/EU570040
SI  - GENBANK/EU570041
SI  - GENBANK/EU570042
SI  - GENBANK/EU570043
SI  - GENBANK/EU570044
SI  - GENBANK/EU570045
SI  - GENBANK/EU570046
SI  - GENBANK/EU570047
SI  - GENBANK/EU570048
SI  - GENBANK/EU570049
SI  - GENBANK/EU570050
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - California
MH  - Female
MH  - *Genetic Variation
MH  - Genitalia, Male/anatomy &amp; histology
MH  - *Geography
MH  - Male
MH  - Models, Biological
MH  - Molecular Sequence Data
MH  - Species Specificity
MH  - Spiders/*classification/*genetics
EDAT- 2008/08/08 09:00
MHDA- 2010/10/01 06:00
CRDT- 2008/08/08 09:00
PHST- 2008/08/08 09:00 [pubmed]
PHST- 2010/10/01 06:00 [medline]
PHST- 2008/08/08 09:00 [entrez]
AID - 901420969 [pii]
AID - 10.1080/10635150802302443 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Aug;57(4):628-46. doi: 10.1080/10635150802302443.

PMID- 18686195
OWN - NLM
STAT- MEDLINE
DCOM- 20100930
LR  - 20080807
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 4
DP  - 2008 Aug
TI  - The vestigial olfactory receptor subgenome of odontocete whales: phylogenetic
      congruence between gene-tree reconciliation and supermatrix methods.
PG  - 574-90
LID - 10.1080/10635150802304787 [doi]
AB  - The macroevolutionary transition of whales (cetaceans) from a terrestrial
      quadruped to an obligate aquatic form involved major changes in sensory
      abilities. Compared to terrestrial mammals, the olfactory system of baleen whales
      is dramatically reduced, and in toothed whales is completely absent. We sampled
      the olfactory receptor (OR) subgenomes of eight cetacean species from four
      families. A multigene tree of 115 newly characterized OR sequences from these
      eight species and published data for Bos taurus revealed a diverse array of class
      II OR paralogues in Cetacea. Evolution of the OR gene superfamily in toothed
      whales (Odontoceti) featured a multitude of independent pseudogenization events, 
      supporting anatomical evidence that odontocetes have lost their olfactory sense. 
      We explored the phylogenetic utility of OR pseudogenes in Cetacea, concentrating 
      on delphinids (oceanic dolphins), the product of a rapid evolutionary radiation
      that has been difficult to resolve in previous studies of mitochondrial DNA
      sequences. Phylogenetic analyses of OR pseudogenes using both gene-tree
      reconciliation and supermatrix methods yielded fully resolved, consistently
      supported relationships among members of four delphinid subfamilies. Alternative 
      minimizations of gene duplications, gene duplications plus gene losses, deep
      coalescence events, and nucleotide substitutions plus indels returned highly
      congruent phylogenetic hypotheses. Novel DNA sequence data for six single-copy
      nuclear loci and three mitochondrial genes (&gt; 5000 aligned nucleotides) provided 
      an independent test of the OR trees. Nucleotide substitutions and indels in OR
      pseudogenes showed a very low degree of homoplasy in comparison to mitochondrial 
      DNA and, on average, provided more variation than single-copy nuclear DNA. Our
      results suggest that phylogenetic analysis of the large OR superfamily will be
      effective for resolving relationships within Cetacea whether supermatrix or
      gene-tree reconciliation procedures are used.
FAU - McGowen, Michael R
AU  - McGowen MR
AD  - Department of Biology, University of California, Riverside, California 92521,
      USA. mmcgo002@student.ucr.edu
FAU - Clark, Clay
AU  - Clark C
FAU - Gatesy, John
AU  - Gatesy J
LA  - eng
SI  - GENBANK/EU684973
SI  - GENBANK/EU684974
SI  - GENBANK/EU684975
SI  - GENBANK/EU684976
SI  - GENBANK/EU684977
SI  - GENBANK/EU684978
SI  - GENBANK/EU684979
SI  - GENBANK/EU684980
SI  - GENBANK/EU684981
SI  - GENBANK/EU684982
SI  - GENBANK/EU684983
SI  - GENBANK/EU684984
SI  - GENBANK/EU684985
SI  - GENBANK/EU684986
SI  - GENBANK/EU684987
SI  - GENBANK/EU684988
SI  - GENBANK/EU684989
SI  - GENBANK/EU684990
SI  - GENBANK/EU684991
SI  - GENBANK/EU684992
SI  - GENBANK/EU684993
SI  - GENBANK/EU684994
SI  - GENBANK/EU684995
SI  - GENBANK/EU684996
SI  - GENBANK/EU684997
SI  - GENBANK/EU684998
SI  - GENBANK/EU684999
SI  - GENBANK/EU685000
SI  - GENBANK/EU685001
SI  - GENBANK/EU685002
SI  - GENBANK/EU685003
SI  - GENBANK/EU685004
SI  - GENBANK/EU685005
SI  - GENBANK/EU685006
SI  - GENBANK/EU685007
SI  - GENBANK/EU685008
SI  - GENBANK/EU685009
SI  - GENBANK/EU685010
SI  - GENBANK/EU685011
SI  - GENBANK/EU685012
SI  - GENBANK/EU685013
SI  - GENBANK/EU685014
SI  - GENBANK/EU685015
SI  - GENBANK/EU685016
SI  - GENBANK/EU685017
SI  - GENBANK/EU685018
SI  - GENBANK/EU685019
SI  - GENBANK/EU685020
SI  - GENBANK/EU685021
SI  - GENBANK/EU685022
SI  - GENBANK/EU685023
SI  - GENBANK/EU685024
SI  - GENBANK/EU685025
SI  - GENBANK/EU685026
SI  - GENBANK/EU685027
SI  - GENBANK/EU685028
SI  - GENBANK/EU685029
SI  - GENBANK/EU685030
SI  - GENBANK/EU685031
SI  - GENBANK/EU685032
SI  - GENBANK/EU685033
SI  - GENBANK/EU685034
SI  - GENBANK/EU685035
SI  - GENBANK/EU685036
SI  - GENBANK/EU685037
SI  - GENBANK/EU685038
SI  - GENBANK/EU685039
SI  - GENBANK/EU685040
SI  - GENBANK/EU685041
SI  - GENBANK/EU685042
SI  - GENBANK/EU685043
SI  - GENBANK/EU685044
SI  - GENBANK/EU685045
SI  - GENBANK/EU685046
SI  - GENBANK/EU685047
SI  - GENBANK/EU685048
SI  - GENBANK/EU685049
SI  - GENBANK/EU685050
SI  - GENBANK/EU685051
SI  - GENBANK/EU685052
SI  - GENBANK/EU685053
SI  - GENBANK/EU685054
SI  - GENBANK/EU685055
SI  - GENBANK/EU685056
SI  - GENBANK/EU685057
SI  - GENBANK/EU685058
SI  - GENBANK/EU685059
SI  - GENBANK/EU685060
SI  - GENBANK/EU685061
SI  - GENBANK/EU685062
SI  - GENBANK/EU685063
SI  - GENBANK/EU685064
SI  - GENBANK/EU685065
SI  - GENBANK/EU685066
SI  - GENBANK/EU685067
SI  - GENBANK/EU685068
SI  - GENBANK/EU685069
SI  - GENBANK/EU685070
SI  - GENBANK/EU685071
SI  - GENBANK/EU685072
SI  - GENBANK/EU685073
SI  - GENBANK/EU685074
SI  - GENBANK/EU685075
SI  - GENBANK/EU685076
SI  - GENBANK/EU685077
SI  - GENBANK/EU685078
SI  - GENBANK/EU685079
SI  - GENBANK/EU685080
SI  - GENBANK/EU685081
SI  - GENBANK/EU685082
SI  - GENBANK/EU685083
SI  - GENBANK/EU685084
SI  - GENBANK/EU685085
SI  - GENBANK/EU685086
SI  - GENBANK/EU685087
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Receptors, Odorant)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Cattle
MH  - Classification/*methods
MH  - DNA/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Evolution, Molecular
MH  - Genetic Variation
MH  - Genome/*genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Receptors, Odorant/*genetics
MH  - Whales/*classification/*genetics
EDAT- 2008/08/08 09:00
MHDA- 2010/10/01 06:00
CRDT- 2008/08/08 09:00
PHST- 2008/08/08 09:00 [pubmed]
PHST- 2010/10/01 06:00 [medline]
PHST- 2008/08/08 09:00 [entrez]
AID - 901406507 [pii]
AID - 10.1080/10635150802304787 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Aug;57(4):574-90. doi: 10.1080/10635150802304787.

PMID- 18686194
OWN - NLM
STAT- MEDLINE
DCOM- 20100930
LR  - 20080807
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 4
DP  - 2008 Aug
TI  - The role of geography and ecological opportunity in the diversification of day
      geckos (Phelsuma).
PG  - 562-73
LID - 10.1080/10635150802304779 [doi]
AB  - We examine the effects of ecological opportunity and geographic area on rates of 
      species accumulation and morphological evolution following archipelago
      colonization in day geckos (genus Phelsuma) in the Indian Ocean. Using a newly
      generated molecular phylogeny for the genus, we present evidence that these
      geckos likely originated on Madagascar, whereas colonization of three
      archipelagos in the Indian Ocean, the Seychelles, Mascarene, and Comoros Islands 
      has produced three independent monophyletic radiations. We find that rates of
      species accumulation are not elevated following colonization but are roughly
      equivalent on all three isolated archipelagos and on the larger island of
      Madagascar. However, rates of species accumulation have slowed through time on
      Madagascar. Rates of morphological evolution are higher in both the Mascarene and
      Seychelles archipelagos compared to rates on Madagascar. This negative
      relationship between rate of morphological evolution and island area suggests
      that ecological opportunity is an important factor in diversification of day
      gecko species.
FAU - Harmon, Luke J
AU  - Harmon LJ
AD  - Department of Biological Sciences, University of Idaho, Moscow, Idaho 83844-3051,
      USA. lukeh@uidaho.edu
FAU - Melville, Jane
AU  - Melville J
FAU - Larson, Allan
AU  - Larson A
FAU - Losos, Jonathan B
AU  - Losos JB
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - *Genetic Speciation
MH  - *Geography
MH  - Indian Ocean Islands
MH  - Lizards/*anatomy &amp; histology/*classification
MH  - Models, Biological
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Principal Component Analysis
MH  - Sequence Alignment
EDAT- 2008/08/08 09:00
MHDA- 2010/10/01 06:00
CRDT- 2008/08/08 09:00
PHST- 2008/08/08 09:00 [pubmed]
PHST- 2010/10/01 06:00 [medline]
PHST- 2008/08/08 09:00 [entrez]
AID - 901406303 [pii]
AID - 10.1080/10635150802304779 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Aug;57(4):562-73. doi: 10.1080/10635150802304779.

PMID- 18686193
OWN - NLM
STAT- MEDLINE
DCOM- 20100930
LR  - 20080807
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 4
DP  - 2008 Aug
TI  - A likelihood framework for estimating phylogeographic history on a continuous
      landscape.
PG  - 544-61
LID - 10.1080/10635150802304761 [doi]
AB  - Due to lack of an adequate statistical framework, biologists studying
      phylogeography are abandoning traditional methods of estimating phylogeographic
      history in favor of statistical methods designed to test a priori hypotheses.
      These new methods may, however, have limited descriptive utility. Here, we
      develop a new statistical framework that can be used to both test a priori
      hypotheses and estimate phylogeographic history of a gene (and the statistical
      confidence in that history) in the absence of such hypotheses. The statistical
      approach concentrates on estimation of geographic locations of the ancestors of a
      set of sampled organisms. Now we use (2) to derive the likelihood of the
      ancestral geographic coordinates and the value of the scaled dispersal parameter,
      given the observed geographic coordinates (assuming known topology and branch
      lengths). Using a maximum likelihood approach, which is implemented in the new
      program PhyloMapper, we apply this statistical framework to a 246-taxon
      mitochondrial genealogy of North American chorus frogs, focusing in detail on one
      of these species. We demonstrate three lines of evidence for recent northward
      expansion of the mitochondrion of the coastal clade of Pseudacris feriarum:
      higher per-generation dispersal distance in the recently colonized region, a
      noncentral ancestral location, and directional migration. After illustrating one 
      method of accommodating phylogenetic uncertainty, we conclude by discussing how
      extensions of this framework could function to incorporate a priori ecological
      and geological information into phylogeographic analyses.
FAU - Lemmon, Alan R
AU  - Lemmon AR
AD  - Section of Integrative Biology, University of Texas-Austin, Austin, Texas 78712, 
      USA. alemmon@evotutor.org
FAU - Lemmon, Emily Moriarty
AU  - Lemmon EM
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Anura/classification/genetics
MH  - Classification/*methods
MH  - DNA, Mitochondrial/genetics
MH  - Data Interpretation, Statistical
MH  - Emigration and Immigration
MH  - Geography
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2008/08/08 09:00
MHDA- 2010/10/01 06:00
CRDT- 2008/08/08 09:00
PHST- 2008/08/08 09:00 [pubmed]
PHST- 2010/10/01 06:00 [medline]
PHST- 2008/08/08 09:00 [entrez]
AID - 901333126 [pii]
AID - 10.1080/10635150802304761 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Aug;57(4):544-61. doi: 10.1080/10635150802304761.

PMID- 18629738
OWN - NLM
STAT- MEDLINE
DCOM- 20100930
LR  - 20080716
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 4
DP  - 2008 Aug
TI  - Perfectly misleading distances from ternary characters.
PG  - 540-3
LID - 10.1080/10635150802203880 [doi]
AB  - D. Huson and M. Steel showed that for any two binary phylogenetic trees on the
      same set of n taxa, there exists a sequence of multistate characters that is
      homoplasy-free only on the first tree but perfectly additive only on the second
      one. The original construction of such a sequence required n - 1 character states
      and it remained an open question whether a sequence using fewer character states 
      can always be found. In the present note we will answer this question by showing 
      that three character states suffice to construct such misleading sequences--even 
      if we insist that they conform to an ultrametric (i.e., fit a molecular clock).
FAU - Bandelt, Hans-Jurgen
AU  - Bandelt HJ
AD  - Department of Mathematics, University of Hamburg, Hamburg, Germany.
      bandelt@math.uni-hamburg.de
FAU - Fischer, Mareike
AU  - Fischer M
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - Models, Genetic
MH  - Phylogeny
EDAT- 2008/07/17 09:00
MHDA- 2010/10/01 06:00
CRDT- 2008/07/17 09:00
PHST- 2008/07/17 09:00 [pubmed]
PHST- 2010/10/01 06:00 [medline]
PHST- 2008/07/17 09:00 [entrez]
AID - 794974401 [pii]
AID - 10.1080/10635150802203880 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Aug;57(4):540-3. doi: 10.1080/10635150802203880.

PMID- 18622808
OWN - NLM
STAT- MEDLINE
DCOM- 20100930
LR  - 20080714
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 4
DP  - 2008 Aug
TI  - Optimal data partitioning and a test case for ray-finned fishes (Actinopterygii) 
      based on ten nuclear loci.
PG  - 519-39
LID - 10.1080/10635150802206883 [doi]
AB  - Data partitioning, the combined phylogenetic analysis of homogeneous blocks of
      data, is a common strategy used to accommodate heterogeneities in complex
      multilocus data sets. Variation in evolutionary rates and substitution patterns
      among sites are typically addressed by partitioning data by gene, codon position,
      or both. Excessive partitioning of the data, however, could lead to
      overparameterization; therefore, it seems critical to define the minimum numbers 
      of partitions necessary to improve the overall fit of the model. We propose a new
      method, based on cluster analysis, to find an optimal partitioning strategy for
      multilocus protein-coding data sets. A heuristic exploration of alternative
      partitioning schemes, based on Bayesian and maximum likelihood (ML) criteria, is 
      shown here to produce an optimal number of partitions. We tested this method
      using sequence data of 10 nuclear genes collected from 52 ray-finned fish
      (Actinopterygii) and four tetrapods. The concatenated sequences included 7995
      nucleotide sites maximally split into 30 partitions defined a priori based on
      gene and codon position. Our results show that a model based on only 10
      partitions defined by cluster analysis performed better than partitioning by both
      gene and codon position. Alternative data partitioning schemes also are shown to 
      affect the topologies resulting from phylogenetic analysis, especially when
      Bayesian methods are used, suggesting that overpartitioning may be of major
      concern. The phylogenetic relationships among the major clades of ray-finned fish
      were assessed using the best data-partitioning schemes under ML and Bayesian
      methods. Some significant results include the monophyly of "Holostei" (Amia and
      Lepisosteus), the sister-group relationships between (1) esociforms and
      salmoniforms and (2) osmeriforms and stomiiforms, the polyphyly of Perciformes,
      and a close relationship of cichlids and atherinomorphs.
FAU - Li, Chenhong
AU  - Li C
AD  - School of Biological Sciences, University of Nebraska, Lincoln, NE 68588, USA.
      cli@unlserve.unl.edu
FAU - Lu, Guoqing
AU  - Lu G
FAU - Orti, Guillermo
AU  - Orti G
LA  - eng
SI  - GENBANK/EU001863
SI  - GENBANK/EU001864
SI  - GENBANK/EU001865
SI  - GENBANK/EU001866
SI  - GENBANK/EU001867
SI  - GENBANK/EU001868
SI  - GENBANK/EU001869
SI  - GENBANK/EU001870
SI  - GENBANK/EU001871
SI  - GENBANK/EU001872
SI  - GENBANK/EU001873
SI  - GENBANK/EU001874
SI  - GENBANK/EU001875
SI  - GENBANK/EU001878
SI  - GENBANK/EU001879
SI  - GENBANK/EU001880
SI  - GENBANK/EU001881
SI  - GENBANK/EU001882
SI  - GENBANK/EU001883
SI  - GENBANK/EU001884
SI  - GENBANK/EU001885
SI  - GENBANK/EU001886
SI  - GENBANK/EU001887
SI  - GENBANK/EU001888
SI  - GENBANK/EU001889
SI  - GENBANK/EU001890
SI  - GENBANK/EU001891
SI  - GENBANK/EU001892
SI  - GENBANK/EU001893
SI  - GENBANK/EU001894
SI  - GENBANK/EU001895
SI  - GENBANK/EU001896
SI  - GENBANK/EU001897
SI  - GENBANK/EU001900
SI  - GENBANK/EU001901
SI  - GENBANK/EU001902
SI  - GENBANK/EU001903
SI  - GENBANK/EU001904
SI  - GENBANK/EU001905
SI  - GENBANK/EU001906
SI  - GENBANK/EU001907
SI  - GENBANK/EU001909
SI  - GENBANK/EU001910
SI  - GENBANK/EU001911
SI  - GENBANK/EU001912
SI  - GENBANK/EU001913
SI  - GENBANK/EU001914
SI  - GENBANK/EU001915
SI  - GENBANK/EU001916
SI  - GENBANK/EU001917
SI  - GENBANK/EU001918
SI  - GENBANK/EU001919
SI  - GENBANK/EU001920
SI  - GENBANK/EU001921
SI  - GENBANK/EU001922
SI  - GENBANK/EU001923
SI  - GENBANK/EU001924
SI  - GENBANK/EU001925
SI  - GENBANK/EU001926
SI  - GENBANK/EU001927
SI  - GENBANK/EU001928
SI  - GENBANK/EU001929
SI  - GENBANK/EU001930
SI  - GENBANK/EU001931
SI  - GENBANK/EU001932
SI  - GENBANK/EU001933
SI  - GENBANK/EU001934
SI  - GENBANK/EU001935
SI  - GENBANK/EU001936
SI  - GENBANK/EU001937
SI  - GENBANK/EU001938
SI  - GENBANK/EU001939
SI  - GENBANK/EU001940
SI  - GENBANK/EU001941
SI  - GENBANK/EU001942
SI  - GENBANK/EU001943
SI  - GENBANK/EU001944
SI  - GENBANK/EU001945
SI  - GENBANK/EU001946
SI  - GENBANK/EU001947
SI  - GENBANK/EU001948
SI  - GENBANK/EU001949
SI  - GENBANK/EU001950
SI  - GENBANK/EU001951
SI  - GENBANK/EU001952
SI  - GENBANK/EU001953
SI  - GENBANK/EU001954
SI  - GENBANK/EU001955
SI  - GENBANK/EU001956
SI  - GENBANK/EU001957
SI  - GENBANK/EU001958
SI  - GENBANK/EU001959
SI  - GENBANK/EU001960
SI  - GENBANK/EU001961
SI  - GENBANK/EU001962
SI  - GENBANK/EU001963
SI  - GENBANK/EU001964
SI  - GENBANK/EU001966
SI  - GENBANK/EU001967
SI  - GENBANK/EU001968
SI  - GENBANK/EU001969
SI  - GENBANK/EU001971
SI  - GENBANK/EU001972
SI  - GENBANK/EU001973
SI  - GENBANK/EU001974
SI  - GENBANK/EU001975
SI  - GENBANK/EU001976
SI  - GENBANK/EU001977
SI  - GENBANK/EU001978
SI  - GENBANK/EU001979
SI  - GENBANK/EU001980
SI  - GENBANK/EU001981
SI  - GENBANK/EU001982
SI  - GENBANK/EU001983
SI  - GENBANK/EU001984
SI  - GENBANK/EU001985
SI  - GENBANK/EU001986
SI  - GENBANK/EU001987
SI  - GENBANK/EU001988
SI  - GENBANK/EU001989
SI  - GENBANK/EU001990
SI  - GENBANK/EU001991
SI  - GENBANK/EU001992
SI  - GENBANK/EU001993
SI  - GENBANK/EU001994
SI  - GENBANK/EU001995
SI  - GENBANK/EU001996
SI  - GENBANK/EU001997
SI  - GENBANK/EU001998
SI  - GENBANK/EU001999
SI  - GENBANK/EU002000
SI  - GENBANK/EU002001
SI  - GENBANK/EU002002
SI  - GENBANK/EU002003
SI  - GENBANK/EU002004
SI  - GENBANK/EU002005
SI  - GENBANK/EU002006
SI  - GENBANK/EU002007
SI  - GENBANK/EU002008
SI  - GENBANK/EU002009
SI  - GENBANK/EU002010
SI  - GENBANK/EU002011
SI  - GENBANK/EU002012
SI  - GENBANK/EU002013
SI  - GENBANK/EU002014
SI  - GENBANK/EU002015
SI  - GENBANK/EU002016
SI  - GENBANK/EU002017
SI  - GENBANK/EU002018
SI  - GENBANK/EU002019
SI  - GENBANK/EU002020
SI  - GENBANK/EU002021
SI  - GENBANK/EU002022
SI  - GENBANK/EU002023
SI  - GENBANK/EU002024
SI  - GENBANK/EU002025
SI  - GENBANK/EU002026
SI  - GENBANK/EU002027
SI  - GENBANK/EU002028
SI  - GENBANK/EU002029
SI  - GENBANK/EU002030
SI  - GENBANK/EU002031
SI  - GENBANK/EU002032
SI  - GENBANK/EU002033
SI  - GENBANK/EU002034
SI  - GENBANK/EU002035
SI  - GENBANK/EU002036
SI  - GENBANK/EU002037
SI  - GENBANK/EU002038
SI  - GENBANK/EU002039
SI  - GENBANK/EU002040
SI  - GENBANK/EU002041
SI  - GENBANK/EU002042
SI  - GENBANK/EU002043
SI  - GENBANK/EU002044
SI  - GENBANK/EU002045
SI  - GENBANK/EU002046
SI  - GENBANK/EU002047
SI  - GENBANK/EU002048
SI  - GENBANK/EU002049
SI  - GENBANK/EU002050
SI  - GENBANK/EU002051
SI  - GENBANK/EU002052
SI  - GENBANK/EU002053
SI  - GENBANK/EU002054
SI  - GENBANK/EU002055
SI  - GENBANK/EU002056
SI  - GENBANK/EU002057
SI  - GENBANK/EU002058
SI  - GENBANK/EU002059
SI  - GENBANK/EU002060
SI  - GENBANK/EU002061
SI  - GENBANK/EU002062
SI  - GENBANK/EU002063
SI  - GENBANK/EU002064
SI  - GENBANK/EU002065
SI  - GENBANK/EU002066
SI  - GENBANK/EU002067
SI  - GENBANK/EU002069
SI  - GENBANK/EU002071
SI  - GENBANK/EU002072
SI  - GENBANK/EU002073
SI  - GENBANK/EU002074
SI  - GENBANK/EU002075
SI  - GENBANK/EU002076
SI  - GENBANK/EU002077
SI  - GENBANK/EU002078
SI  - GENBANK/EU002079
SI  - GENBANK/EU002080
SI  - GENBANK/EU002081
SI  - GENBANK/EU002082
SI  - GENBANK/EU002083
SI  - GENBANK/EU002084
SI  - GENBANK/EU002085
SI  - GENBANK/EU002086
SI  - GENBANK/EU002087
SI  - GENBANK/EU002088
SI  - GENBANK/EU002089
SI  - GENBANK/EU002090
SI  - GENBANK/EU002091
SI  - GENBANK/EU002092
SI  - GENBANK/EU002093
SI  - GENBANK/EU002095
SI  - GENBANK/EU002096
SI  - GENBANK/EU002097
SI  - GENBANK/EU002098
SI  - GENBANK/EU002099
SI  - GENBANK/EU002100
SI  - GENBANK/EU002101
SI  - GENBANK/EU002102
SI  - GENBANK/EU002103
SI  - GENBANK/EU002104
SI  - GENBANK/EU002105
SI  - GENBANK/EU002106
SI  - GENBANK/EU002107
SI  - GENBANK/EU002108
SI  - GENBANK/EU002109
SI  - GENBANK/EU002110
SI  - GENBANK/EU002111
SI  - GENBANK/EU002112
SI  - GENBANK/EU002113
SI  - GENBANK/EU002114
SI  - GENBANK/EU002115
SI  - GENBANK/EU002116
SI  - GENBANK/EU002117
SI  - GENBANK/EU002118
SI  - GENBANK/EU002119
SI  - GENBANK/EU002120
SI  - GENBANK/EU002121
SI  - GENBANK/EU002122
SI  - GENBANK/EU002123
SI  - GENBANK/EU002124
SI  - GENBANK/EU002125
SI  - GENBANK/EU002126
SI  - GENBANK/EU002127
SI  - GENBANK/EU002128
SI  - GENBANK/EU002129
SI  - GENBANK/EU002130
SI  - GENBANK/EU002131
SI  - GENBANK/EU002132
SI  - GENBANK/EU002133
SI  - GENBANK/EU002134
SI  - GENBANK/EU002135
SI  - GENBANK/EU002136
SI  - GENBANK/EU002137
SI  - GENBANK/EU002138
SI  - GENBANK/EU002139
SI  - GENBANK/EU002140
SI  - GENBANK/EU002141
SI  - GENBANK/EU002142
SI  - GENBANK/EU002143
SI  - GENBANK/EU002144
SI  - GENBANK/EU002145
SI  - GENBANK/EU002146
SI  - GENBANK/EU002147
SI  - GENBANK/EU002148
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Cell Nucleus/*genetics
MH  - Classification/methods
MH  - Cluster Analysis
MH  - Fishes/*classification/*genetics
MH  - Genes/*genetics
MH  - Genetic Markers
MH  - Humans
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2008/07/16 09:00
MHDA- 2010/10/01 06:00
CRDT- 2008/07/16 09:00
PHST- 2008/07/16 09:00 [pubmed]
PHST- 2010/10/01 06:00 [medline]
PHST- 2008/07/16 09:00 [entrez]
AID - 794916139 [pii]
AID - 10.1080/10635150802206883 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Aug;57(4):519-39. doi: 10.1080/10635150802206883.

PMID- 18570043
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20080623
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - Species Names in the PhyloCode: the approach adopted by the International Society
      for Phylogenetic Nomenclature.
PG  - 507-14
LID - 10.1080/10635150802172176 [doi]
FAU - Dayrat, Benoit
AU  - Dayrat B
AD  - School of Natural Sciences, University of California, Merced, CA 95344, USA.
      bdayrat@ucmerced.edu
FAU - Cantino, Philip D
AU  - Cantino PD
FAU - Clarke, Julia A
AU  - Clarke JA
FAU - de Queiroz, Kevin
AU  - de Queiroz K
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/methods
MH  - *Phylogeny
MH  - *Societies
MH  - *Terminology as Topic
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 794141112 [pii]
AID - 10.1080/10635150802172176 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):507-14. doi: 10.1080/10635150802172176.

PMID- 18570042
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20080623
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - Hemiplasy: a new term in the lexicon of phylogenetics.
PG  - 503-7
LID - 10.1080/10635150802164587 [doi]
FAU - Avise, John C
AU  - Avise JC
AD  - Department of Ecology and Evolutionary Biology, University of California, Irvine,
      CA 92697, USA. javise@uci.edu
FAU - Robinson, Terence J
AU  - Robinson TJ
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Alleles
MH  - Animals
MH  - Gorilla gorilla/classification/genetics
MH  - Humans
MH  - Pan troglodytes/classification/genetics
MH  - *Phylogeny
MH  - Polymorphism, Genetic
MH  - *Terminology as Topic
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 794140192 [pii]
AID - 10.1080/10635150802164587 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):503-7. doi: 10.1080/10635150802164587.

PMID- 18570041
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20080623
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - Morphology and placental mammal phylogeny.
PG  - 499-503
LID - 10.1080/10635150802164504 [doi]
FAU - Springer, Mark S
AU  - Springer MS
AD  - Department of Biology, University of California, Riverside, CA 92521, USA.
      mark.springer@ucr.edu
FAU - Meredith, Robert W
AU  - Meredith RW
FAU - Eizirik, Eduardo
AU  - Eizirik E
FAU - Teeling, Emma
AU  - Teeling E
FAU - Murphy, William J
AU  - Murphy WJ
LA  - eng
PT  - Comment
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2007 Aug;56(4):673-84. PMID: 17661234
MH  - Animals
MH  - Classification/methods
MH  - Extinction, Biological
MH  - Mammals/*anatomy &amp; histology/*classification/genetics
MH  - *Phylogeny
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 794140218 [pii]
AID - 10.1080/10635150802164504 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):499-503. doi: 10.1080/10635150802164504.

PMID- 18570040
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20161019
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - A novel test for host-symbiont codivergence indicates ancient origin of fungal
      endophytes in grasses.
PG  - 483-98
LID - 10.1080/10635150802172184 [doi]
AB  - Significant phylogenetic codivergence between plant or animal hosts (H) and their
      symbionts or parasites (P) indicates the importance of their interactions on
      evolutionary time scales. However, valid and realistic methods to test for
      codivergence are not fully developed. One of the systems where possible
      codivergence has been of interest involves the large subfamily of temperate
      grasses (Pooideae) and their endophytic fungi (epichloae). These widespread
      symbioses often help protect host plants from herbivory and stresses and affect
      species diversity and food web structures. Here we introduce the MRCALink
      (most-recent-common-ancestor link) method and use it to investigate the
      possibility of grass-epichloe codivergence. MRCALink applied to ultrametric H and
      P trees identifies all corresponding nodes for pairwise comparisons of MRCA ages.
      The result is compared to the space of random H and P tree pairs estimated by a
      Monte Carlo method. Compared to tree reconciliation, the method is less dependent
      on tree topologies (which often can be misleading), and it crucially improves on 
      phylogeny-independent methods such as ParaFit or the Mantel test by eliminating
      an extreme (but previously unrecognized) distortion of node-pair sampling.
      Analysis of 26 grass species-epichloe species symbioses did not reject random
      association of H and P MRCA ages. However, when five obvious host jumps were
      removed, the analysis significantly rejected random association and supported
      grass-endophyte codivergence. Interestingly, early cladogenesis events in the
      Pooideae corresponded to early cladogenesis events in epichloae, suggesting
      concomitant origins of this grass subfamily and its remarkable group of
      symbionts. We also applied our method to the well-known gopher-louse data set.
FAU - Schardl, C L
AU  - Schardl CL
AD  - Department of Plant Pathology, University of Kentucky, Lexington, KY 40546-0312, 
      USA.
FAU - Craven, K D
AU  - Craven KD
FAU - Speakman, S
AU  - Speakman S
FAU - Stromberg, A
AU  - Stromberg A
FAU - Lindstrom, A
AU  - Lindstrom A
FAU - Yoshida, R
AU  - Yoshida R
LA  - eng
GR  - R01 GM086888/GM/NIGMS NIH HHS/United States
GR  - P20 RR16481/RR/NCRR NIH HHS/United States
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
RN  - 0 (DNA, Fungal)
RN  - 0 (DNA, Intergenic)
RN  - 0 (DNA, Plant)
RN  - 0 (Peptide Elongation Factor 1)
RN  - 0 (Tubulin)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Classification/methods
MH  - DNA, Chloroplast/chemistry
MH  - DNA, Fungal/chemistry
MH  - DNA, Intergenic/chemistry
MH  - DNA, Plant/chemistry
MH  - Gophers/classification/genetics
MH  - Hypocreales/*classification/physiology
MH  - Likelihood Functions
MH  - Monte Carlo Method
MH  - Peptide Elongation Factor 1/chemistry
MH  - Phthiraptera/classification/genetics
MH  - *Phylogeny
MH  - Poaceae/*classification/microbiology
MH  - *Symbiosis
MH  - Tubulin/chemistry
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 794140276 [pii]
AID - 10.1080/10635150802172184 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):483-98. doi: 10.1080/10635150802172184.

PMID- 18570039
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - The reticulate history of Medicago (Fabaceae).
PG  - 466-82
LID - 10.1080/10635150802172168 [doi]
AB  - The phylogenetic history of Medicago was examined for 60 accessions from 56
      species using two nuclear genes (CNGC5 and beta-cop) and one mitochondrial region
      (rpS14-cob). The results of several analyses revealed that extensive robustly
      supported incongruence exists among the nuclear genes, the cause of which we seek
      to explain. After rejecting several processes, hybridization and lineage sorting 
      of ancestral polymorphisms remained as the most likely factors promoting
      incongruence. Using coalescence simulations, we rejected lineage sorting alone as
      an explanation of the differences among gene trees. The results indicate that
      hybridization has been common and ongoing among lineages since the origin of
      Medicago. Coalescence provides a good framework to test the causes of
      incongruence commonly seen among gene trees but requires knowledge of effective
      population sizes and generation times. We estimated the effective population size
      at 240,000 individuals and assumed a generation time of 1 year in Medicago (many 
      are annual plants). A sensitivity analysis showed that our conclusions remain
      unchanged using a larger effective population size and/or longer generation time.
FAU - Maureira-Butler, Ivan J
AU  - Maureira-Butler IJ
AD  - Department of Plant Biology, Cornell University, Ithaca, NY 14853, USA.
FAU - Pfeil, Bernard E
AU  - Pfeil BE
FAU - Muangprom, Amorntip
AU  - Muangprom A
FAU - Osborn, Thomas C
AU  - Osborn TC
FAU - Doyle, Jeff J
AU  - Doyle JJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Coatomer Protein)
RN  - 0 (Cyclic Nucleotide-Gated Cation Channels)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (DNA, Plant)
RN  - 0 (Plant Proteins)
SB  - IM
MH  - Bayes Theorem
MH  - Coatomer Protein/chemistry
MH  - Computer Simulation
MH  - Cyclic Nucleotide-Gated Cation Channels/chemistry
MH  - DNA, Mitochondrial/chemistry
MH  - DNA, Plant/chemistry
MH  - Diploidy
MH  - Genetic Linkage
MH  - Hybridization, Genetic
MH  - Medicago/*classification/genetics/ultrastructure
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Plant Proteins/chemistry
MH  - Polyploidy
MH  - Sequence Analysis, DNA
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 794141455 [pii]
AID - 10.1080/10635150802172168 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):466-82. doi: 10.1080/10635150802172168.

PMID- 18570038
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20080623
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - Codivergence in heteromyid rodents (Rodentia: heteromyidae) and their sucking
      lice of the genus Fahrenholzia (Phthiraptera: anoplura).
PG  - 449-65
LID - 10.1080/10635150802169610 [doi]
AB  - Although most studies of codivergence rely primarily on topological comparisons
      of host and parasite phylogenies, temporal assessments are necessary to determine
      if divergence events in host and parasite trees occurred contemporaneously. A
      combination of cophylogenetic analyses and comparisons of branch lengths are used
      in this study to understand the host-parasite association between heteromyid
      rodents (Rodentia: Heteromyidae) and their sucking lice of the genus Fahrenholzia
      (Phthiraptera: Anoplura). Cophylogenetic comparisons based on nucleotide
      substitutions in the mitochondrial COI gene reveal a significant, but not
      perfect, pattern of cophylogeny between heteromyids and their sucking lice.
      Regression analyses show a significant functional relationship between the
      lengths of analogous branches in the host and parasite trees, indicating that
      divergence events in hosts and parasites were approximately contemporaneous.
      Thus, the topological similarity observed between heteromyids and their lice is
      the result of codivergence. These analyses also show that the COI gene in lice is
      evolving two to three times faster than the same gene in their hosts (similar to 
      the results of studies of other lice and their vertebrate hosts) and that
      divergence events in lice occurred shortly after host divergence. We recommend
      that future studies of codivergence include temporal comparisons and, when
      possible, use the same molecular marker(s) in hosts and parasites to achieve the 
      greatest insight into the history of the host-parasite relationship.
FAU - Light, Jessica E
AU  - Light JE
AD  - Department of Biological Sciences and Museum of Natural Science, Louisiana State 
      University, Baton Rouge, LA 70803, USA. jlight@flmnh.ufl.edu
FAU - Hafner, Mark S
AU  - Hafner MS
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Anoplura/classification/genetics/*physiology
MH  - Electron Transport Complex IV/chemistry
MH  - Evolution, Molecular
MH  - Genetic Markers
MH  - Genetic Speciation
MH  - *Host-Parasite Interactions
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - Regression Analysis
MH  - Rodentia/classification/genetics/*parasitology
MH  - Sequence Analysis, DNA
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 794140356 [pii]
AID - 10.1080/10635150802169610 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):449-65. doi: 10.1080/10635150802169610.

PMID- 18570037
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20100609
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - Sympatry inference and network analysis in biogeography.
PG  - 432-48
LID - 10.1080/10635150802172192 [doi]
AB  - A new approach for biogeography to find patterns of sympatry, based on network
      analysis, is proposed. Biogeographic analysis focuses basically on sympatry
      patterns of species. Sympatry is a network (= relational) datum, but it has never
      been analyzed before using relational tools such as Network Analysis. Our
      approach to biogeographic analysis consists of two parts: first the sympatry
      inference and second the network analysis method (NAM). The sympatry inference
      method was designed to propose sympatry hypothesis, constructing a basal sympatry
      network based on punctual data, independent of a priori distributional area
      determination. In this way, two or more species are considered sympatric when
      there is interpenetration and relative proximity among their records of
      occurrence. In nature, groups of species presenting within-group sympatry and
      between-group allopatry constitute natural units (units of co-occurrence). These 
      allopatric units are usually connected by intermediary species. The network
      analysis method (NAM) that we propose here is based on the identification and
      removal of intermediary species to segregate units of co-occurrence, using the
      betweenness measure and the clustering coefficient. The species ranges of the
      units of co-occurrence obtained are transferred to a map, being considered as
      candidates to areas of endemism. The new approach was implemented on three
      different real complex data sets (one of them a classic example previously used
      in biogeography) resulting in (1) independence of predefined spatial units; (2)
      definition of co-occurrence patterns from the sympatry network structure, not
      from species range similarities; (3) higher stability in results despite scale
      changes; (4) identification of candidates to areas of endemism supported by
      strictly endemic species; (5) identification of intermediary species with
      particular biological attributes.
FAU - Dos Santos, Daniel A
AU  - Dos Santos DA
AD  - CONICET, Facultad de Ciencias Naturales, Universidad Nacional de Tucuman,
      Tucuman, Argentina.
FAU - Fernandez, Hugo R
AU  - Fernandez HR
FAU - Cuezzo, Maria Gabriela
AU  - Cuezzo MG
FAU - Dominguez, Eduardo
AU  - Dominguez E
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2009 Apr;58(2):271-6. PMID: 20525583
MH  - Animals
MH  - Behavior, Animal
MH  - Cluster Analysis
MH  - *Ecosystem
MH  - Fishes/physiology
MH  - Fresh Water
MH  - Gastropoda/physiology
MH  - *Geography
MH  - Models, Biological
MH  - Weevils/physiology
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 793945421 [pii]
AID - 10.1080/10635150802172192 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):432-48. doi: 10.1080/10635150802172192.

PMID- 18570036
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20080623
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - Branch lengths, support, and congruence: testing the phylogenomic approach with
      20 nuclear loci in snakes.
PG  - 420-31
LID - 10.1080/10635150802166053 [doi]
AB  - Many authors have claimed that short branches in the Tree of Life will be very
      difficult to resolve with strong support, even with the large multilocus data
      sets now made possible by genomic resources. Short branches may be especially
      problematic because the underlying gene trees are expected to have discordant
      phylogenetic histories when the time between branching events is very short.
      Although there are many examples of short branches that are difficult to resolve,
      surprisingly, no empirical studies have systematically examined the relationships
      between branch lengths, branch support, and congruence among genes. Here, we
      examine these fundamental relationships quantitatively using a data set of 20
      nuclear loci for 50 species of snakes (representing most traditionally recognized
      families). A combined maximum likelihood analysis of the 20 loci gives strong
      support for 69% of the nodes, but many remain weakly supported, with bootstrap
      values for 20% ranging from 21% to 66%. For the combined-data tree, we find
      significant correlations between the length of a branch, levels of bootstrap
      support, and the proportion of genes that are congruent with that branch in the
      separate analyses of each gene. We also find that strongly supported conflicts
      between gene trees over the resolution of individual branches are common (roughly
      35% of clades), especially for shorter branches. Overall, our results support the
      hypothesis that short branches may be very difficult to confidently resolve, even
      with large, multilocus data sets. Nevertheless, our study provides strong support
      for many clades, including several that were controversial or poorly resolved in 
      previous studies of snake phylogeny.
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Department of Ecology and Evolution, Stony Brook University, Stony Brook, NY
      11794-5245, USA. wiensj@life.bio.sunysb.edu
FAU - Kuczynski, Caitlin A
AU  - Kuczynski CA
FAU - Smith, Sarah A
AU  - Smith SA
FAU - Mulcahy, Daniel G
AU  - Mulcahy DG
FAU - Sites, Jack W Jr
AU  - Sites JW Jr
FAU - Townsend, Ted M
AU  - Townsend TM
FAU - Reeder, Tod W
AU  - Reeder TW
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Proteins)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Genomics/*methods
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - Proteins/chemistry
MH  - Sequence Analysis, DNA
MH  - Snakes/*classification/genetics
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 793974339 [pii]
AID - 10.1080/10635150802166053 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):420-31. doi: 10.1080/10635150802166053.

PMID- 18570035
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20161019
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - A Bayesian perspective on a non-parsimonious parsimony model.
PG  - 406-19
LID - 10.1080/10635150802166046 [doi]
AB  - Several stochastic models of character change, when implemented in a maximum
      likelihood framework, are known to give a correspondence between the maximum
      parsimony method and the method of maximum likelihood. One such model has an
      independently estimated branch-length parameter for each site and each branch of 
      the phylogenetic tree. This model--the no-common-mechanism model--has many
      parameters, and, in fact, the number of parameters increases as fast as the
      alignment is extended. We take a Bayesian approach to the no-common-mechanism
      model and place independent gamma prior probability distributions on the
      branch-length parameters. We are able to analytically integrate over the branch
      lengths, and this allowed us to implement an efficient Markov chain Monte Carlo
      method for exploring the space of phylogenetic trees. We were able to reliably
      estimate the posterior probabilities of clades for phylogenetic trees of up to
      500 sequences. However, the Bayesian approach to the problem, at least as
      implemented here with an independent prior on the length of each branch, does not
      tame the behavior of the branch-length parameters. The integrated likelihood
      appears to be a simple rescaling of the parsimony score for a tree, and the
      marginal posterior probability distribution of the length of a branch is
      dependent upon how the maximum parsimony method reconstructs the characters at
      the interior nodes of the tree. The method we describe, however, is of potential 
      importance in the analysis of morphological character data and also for improving
      the behavior of Markov chain Monte Carlo methods implemented for models in which 
      sites share a common branch-length parameter.
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
AD  - Department of Integrative Biology, University of California, Berkeley, CA
      94720-3140, USA. johnh@berkeley.edu
FAU - Ane, Cecile
AU  - Ane C
FAU - Larget, Bret
AU  - Larget B
FAU - Ronquist, Fredrik
AU  - Ronquist F
LA  - eng
GR  - R01 GM069801/GM/NIGMS NIH HHS/United States
GR  - GM-069801/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9004-22-2 (Globins)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Globins/chemistry
MH  - Likelihood Functions
MH  - Markov Chains
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Vertebrates/classification/genetics
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 794012413 [pii]
AID - 10.1080/10635150802166046 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):406-19. doi: 10.1080/10635150802166046.

PMID- 18570034
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - Phylogeny and evolution of glass sponges (porifera, hexactinellida).
PG  - 388-405
LID - 10.1080/10635150802161088 [doi]
AB  - Reconstructing the phylogeny of sponges (Porifera) is one of the remaining
      challenges to resolve the metazoan Tree of Life and is a prerequisite for
      understanding early animal evolution. Molecular phylogenetic analyses for two of 
      the three extant classes of the phylum, Demospongiae and Calcarea, are largely
      incongruent with traditional classifications, most likely because of a paucity of
      informative morphological characters and high levels of homoplasy. For the third 
      class, Hexactinellida (glass sponges)--predominantly deep-sea inhabitants with
      unusual morphology and biology--we present the first molecular phylogeny, along
      with a cladistic analysis of morphological characters. We collected 18S, 28S, and
      mitochondrial 16S ribosomal DNA sequences of 34 glass sponge species from 27
      genera, 9 families, and 3 orders and conducted partitioned Bayesian analyses
      using RNA secondary structure-specific substitution models (paired-sites models) 
      for stem regions. Bayes factor comparisons of different paired-sites models
      against each other and conventional (independent-sites) models revealed a
      significantly better fit of the former but, contrary to previous predictions, the
      least parameter-rich of the tested paired-sites models provided the best fit to
      our data. In contrast to Demospongiae and Calcarea, our rDNA phylogeny agrees
      well with the traditional classification and a previously proposed phylogenetic
      system, which we ascribe to a more informative morphology in Hexactinellida. We
      find high support for a close relationship of glass sponges and Demospongiae
      sensu stricto, though the latter may be paraphyletic with respect to
      Hexactinellida. Homoscleromorpha appears to be the sister group of Calcarea.
      Contrary to most previous findings from rDNA, we recover Porifera as
      monophyletic, although support for this clade is low under paired-sites models.
FAU - Dohrmann, Martin
AU  - Dohrmann M
AD  - Georg-August-Universitat Gottingen, Geowissenschaftliches Zentrum Gottingen,
      Abteilung Geobiologie, Goldschmidtstrasse 3, Germany.
FAU - Janussen, Dorte
AU  - Janussen D
FAU - Reitner, Joachim
AU  - Reitner J
FAU - Collins, Allen G
AU  - Collins AG
FAU - Worheide, Gert
AU  - Worheide G
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - DNA, Ribosomal/chemistry
MH  - Microscopy, Electron, Scanning
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Porifera/*classification/genetics/ultrastructure
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
MH  - Sequence Analysis, RNA
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 793808056 [pii]
AID - 10.1080/10635150802161088 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):388-405. doi: 10.1080/10635150802161088.

PMID- 18570033
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - Estimating evolution of temporal sequence changes: a practical approach to
      inferring ancestral developmental sequences and sequence heterochrony.
PG  - 378-87
LID - 10.1080/10635150802164421 [doi]
AB  - Developmental biology often yields data in a temporal context. Temporal data in
      phylogenetic systematics has important uses in the field of evolutionary
      developmental biology and, in general, comparative biology. The evolution of
      temporal sequences, specifically developmental sequences, has proven difficult to
      examine due to the highly variable temporal progression of development. Issues
      concerning the analysis of temporal sequences and problems with current methods
      of analysis are discussed. We present here an algorithm to infer ancestral
      temporal sequences, quantify sequence heterochronies, and estimate
      pseudoreplicate consensus support for sequence changes using Parsimov-based
      genetic inference [PGi]. Real temporal developmental sequence data sets are used 
      to compare PGi with currently used approaches, and PGi is shown to be the most
      efficient, accurate, and practical method to examine biological data and infer
      ancestral states on a phylogeny. The method is also expandable to address further
      issues in developmental evolution, namely modularity.
FAU - Harrison, Luke B
AU  - Harrison LB
AD  - Redpath Museum, McGill University, Montreal, QC, Canada. luke.harrison@mcgill.ca
FAU - Larsson, Hans C E
AU  - Larsson HC
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Animals
MH  - *Biological Evolution
MH  - Colubridae/*growth &amp; development
MH  - Computational Biology
MH  - Ducks/growth &amp; development
MH  - Osteogenesis/physiology
MH  - *Phylogeny
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 793807560 [pii]
AID - 10.1080/10635150802164421 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):378-87. doi: 10.1080/10635150802164421.

PMID- 18570032
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20080623
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - Synonymous substitutions substantially improve evolutionary inference from highly
      diverged proteins.
PG  - 367-77
LID - 10.1080/10635150802158670 [doi]
AB  - Codon-and amino acid-substitution models are widely used for the evolutionary
      analysis of protein-coding DNA sequences. Using codon models, the amounts of both
      nonsynonymous and synonymous DNA substitutions can be estimated. The ratio of
      these amounts represents the strength of selective pressure. Using amino acid
      models, the amount of nonsynonymous substitutions is estimated, but that of
      synonymous substitutions is ignored. Although amino acid models lose any
      information regarding synonymous substitutions, they explicitly incorporate the
      information for amino acid replacement, which is empirically derived from
      databases. It is often presumed that when the protein-coding sequences are highly
      divergent, synonymous substitutions might be saturated and the evolutionary
      analysis may be hampered by synonymous noise. However, there exists no
      quantitative procedure to verify whether synonymous substitutions can be ignored;
      therefore, amino acid models have been arbitrarily selected. In this study, we
      investigate the issue of a statistical comparison between codon-and amino
      acid-substitution models. For this purpose, we propose a new procedure to
      transform a 20-dimensional amino acid model to a 61-dimensional codon model. This
      transformation reveals that amino acid models belong to a subset of the codon
      models and enables us to test whether synonymous substitutions can be ignored by 
      using the likelihood ratio. Our theoretical results and analyses of real data
      indicate that synonymous substitutions are very informative and substantially
      improve evolutionary inference, even when the sequences are highly divergent.
      Therefore, we note that amino acid models should be adopted only after carefully 
      investigating and discarding the possibility that synonymous substitutions can
      reveal important evolutionary information.
FAU - Seo, Tae-Kun
AU  - Seo TK
AD  - Professional Programme for Agricultural Bioinformatics, Graduate School of
      Agricultural and Life Sciences, University of Tokyo, Tokyo, Japan.
      seo@iu.a.u-tokyo.ac.jp
FAU - Kishino, Hirohisa
AU  - Kishino H
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Fungal Proteins)
RN  - 0 (Mitochondrial Proteins)
SB  - IM
MH  - Amino Acid Sequence
MH  - *Amino Acid Substitution
MH  - Animals
MH  - Codon
MH  - Computational Biology
MH  - DNA, Mitochondrial/chemistry
MH  - *Evolution, Molecular
MH  - Fungal Proteins/*chemistry
MH  - Likelihood Functions
MH  - Mammals/genetics
MH  - Mitochondrial Proteins/*chemistry
MH  - *Models, Genetic
MH  - Phylogeny
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 793807694 [pii]
AID - 10.1080/10635150802158670 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):367-77. doi: 10.1080/10635150802158670.

PMID- 18570031
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20080623
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - Optimizing automated AFLP scoring parameters to improve phylogenetic resolution.
PG  - 347-66
LID - 10.1080/10635150802044037 [doi]
AB  - The amplified fragment length polymorphism (AFLP) technique is an increasingly
      popular component of the phylogenetic toolbox, particularly for plant species.
      Technological advances in capillary electrophoresis now allow very precise
      estimates of DNA fragment mobility and amplitude, and current AFLP software
      allows greater control of data scoring and the production of the binary character
      matrix. However, for AFLP to become a useful modern tool for large data sets,
      improvements to automated scoring are required. We design a procedure that can be
      used to optimize AFLP scoring parameters to improve phylogenetic resolution and
      demonstrate it for two AFLP scoring programs (GeneMapper and GeneMarker). In
      general, we found that there was a trade-off between getting more characters of
      lower quality and fewer characters of high quality. Conservative settings that
      gave the least error did not give the best phylogenetic resolution, as too many
      useful characters were discarded. For example, in GeneMapper, we found that bin
      width was a crucial parameter, and that although reducing bin width from 1.0 to
      0.5 base pairs increased the error rate, it nevertheless improved resolution due 
      to the increased number of informative characters. For our 30-taxon data sets,
      moving from default to optimized parameter settings gave between 3 and 11 extra
      internal edges with &gt;50% bootstrap support, in the best case increasing the
      number of resolved edges from 14 to 25 out of a possible 27. Nevertheless,
      improvements to current AFLP software packages are needed to (1) make use of
      replicate profiles to calibrate the data and perform error calculations and (2)
      perform tests to optimize scoring parameters in a rigorous and automated way.
      This is true not only when AFLP data are used for phylogenetics, but also for
      other applications, including linkage mapping and population genetics.
FAU - Holland, Barbara R
AU  - Holland BR
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, Massey University,
      Palmerston North, New Zealand. B&gt;R.Holland@massey.ac.nz
FAU - Clarke, Andrew C
AU  - Clarke AC
FAU - Meudt, Heidi M
AU  - Meudt HM
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Alleles
MH  - Amplified Fragment Length Polymorphism Analysis/*methods
MH  - Computational Biology/*methods
MH  - Ipomoea batatas/genetics
MH  - Phylogeny
MH  - Plantago/anatomy &amp; histology/genetics
MH  - Polymorphism, Genetic
MH  - *Software
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 793808319 [pii]
AID - 10.1080/10635150802044037 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):347-66. doi: 10.1080/10635150802044037.

PMID- 18570030
OWN - NLM
STAT- MEDLINE
DCOM- 20080918
LR  - 20080623
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 3
DP  - 2008 Jun
TI  - The PhyLoTA Browser: processing GenBank for molecular phylogenetics research.
PG  - 335-46
LID - 10.1080/10635150802158688 [doi]
AB  - As an archive of sequence data for over 165,000 species, GenBank is an
      indispensable resource for phylogenetic inference. Here we describe an
      informatics processing pipeline and online database, the PhyLoTA Browser
      (http://loco.biosci.arizona.edu/pb), which offers a view of GenBank tailored for 
      molecular phylogenetics. The first release of the Browser is computed from 2.6
      million sequences representing the taxonomically enriched subset of GenBank
      sequences for eukaryotes (excluding most genome survey sequences, ESTs, and other
      high-throughput data). In addition to summarizing sequence diversity and species 
      diversity across nodes in the NCBI taxonomy, it reports 87,000 potentially
      phylogenetically informative clusters of homologous sequences, which can be
      viewed or downloaded, along with provisional alignments and coarse phylogenetic
      trees. At each node in the NCBI hierarchy, the user can display a "data
      availability matrix" of all available sequences for entries in a
      subtaxa-by-clusters matrix. This matrix provides a guidepost for subsequent
      assembly of multigene data sets or supertrees. The database allows for comparison
      of results from previous GenBank releases, highlighting recent additions of
      either sequences or taxa to GenBank and letting investigators track progress on
      data availability worldwide. Although the reported alignments and trees are
      extremely approximate, the database reports several statistics correlated with
      alignment quality to help users choose from alternative data sources.
FAU - Sanderson, Michael J
AU  - Sanderson MJ
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA. sanderm@email.arizona.edu
FAU - Boss, Darren
AU  - Boss D
FAU - Chen, Duhong
AU  - Chen D
FAU - Cranston, Karen A
AU  - Cranston KA
FAU - Wehe, Andre
AU  - Wehe A
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Cluster Analysis
MH  - Computational Biology/methods
MH  - *Databases, Nucleic Acid
MH  - Internet
MH  - *Phylogeny
MH  - *Software
EDAT- 2008/06/24 09:00
MHDA- 2008/09/19 09:00
CRDT- 2008/06/24 09:00
PHST- 2008/06/24 09:00 [pubmed]
PHST- 2008/09/19 09:00 [medline]
PHST- 2008/06/24 09:00 [entrez]
AID - 793809875 [pii]
AID - 10.1080/10635150802158688 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Jun;57(3):335-46. doi: 10.1080/10635150802158688.

PMID- 18432552
OWN - NLM
STAT- MEDLINE
DCOM- 20081028
LR  - 20080424
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 2
DP  - 2008 Apr
TI  - The taming of the skew: estimating proper confidence intervals for divergence
      dates.
PG  - 317-28
LID - 10.1080/10635150802040605 [doi]
FAU - Burbrink, Frank T
AU  - Burbrink FT
AD  - The College of Staten Island/CUNY, Department of Biology, 6S-143, 2800 Victory
      Boulevard, Staten Island, NY 10314, USA. burbrink@mail.csi.cuny.edu
FAU - Pyron, R Alexander
AU  - Pyron RA
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Genetic Speciation
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Snakes/genetics
MH  - Time Factors
EDAT- 2008/04/25 09:00
MHDA- 2008/10/29 09:00
CRDT- 2008/04/25 09:00
PHST- 2008/04/25 09:00 [pubmed]
PHST- 2008/10/29 09:00 [medline]
PHST- 2008/04/25 09:00 [entrez]
AID - 792376225 [pii]
AID - 10.1080/10635150802040605 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Apr;57(2):317-28. doi: 10.1080/10635150802040605.

PMID- 18432551
OWN - NLM
STAT- MEDLINE
DCOM- 20081028
LR  - 20090331
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 2
DP  - 2008 Apr
TI  - Morphology, paleontology, and placental mammal phylogeny.
PG  - 311-7
LID - 10.1080/10635150802033022 [doi]
FAU - Asher, Robert J
AU  - Asher RJ
AD  - Department of Zoology, University of Cambridge, Downing Street, UK.
FAU - Geisler, Jonathan H
AU  - Geisler JH
FAU - Sanchez-Villagra, Marcelo R
AU  - Sanchez-Villagra MR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Extinction, Biological
MH  - Mammals/*anatomy &amp; histology/*genetics
MH  - *Paleontology
MH  - Phylogeny
MH  - Time Factors
EDAT- 2008/04/25 09:00
MHDA- 2008/10/29 09:00
CRDT- 2008/04/25 09:00
PHST- 2008/04/25 09:00 [pubmed]
PHST- 2008/10/29 09:00 [medline]
PHST- 2008/04/25 09:00 [entrez]
AID - 792383418 [pii]
AID - 10.1080/10635150802033022 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Apr;57(2):311-7. doi: 10.1080/10635150802033022.

PMID- 18432550
OWN - NLM
STAT- MEDLINE
DCOM- 20081028
LR  - 20081029
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 2
DP  - 2008 Apr
TI  - Multilocus phylogenetics of a rapid radiation in the genus Thomomys (Rodentia:
      Geomyidae).
PG  - 294-310
LID - 10.1080/10635150802044011 [doi]
AB  - Species complexes undergoing rapid radiation present a challenge in molecular
      systematics because of the possibility that ancestral polymorphism is retained in
      component gene trees. Coalescent theory has demonstrated that gene trees often
      fail to match lineage trees when taxon divergence times are less than the
      ancestral effective population sizes. Suggestions to increase the number of loci 
      and the number of individuals per taxon have been proposed; however, phylogenetic
      methods to adequately analyze these data in a coalescent framework are scarce. We
      compare two approaches to estimating lineage (species) trees using multiple
      individuals and multiple loci: the commonly used partitioned Bayesian analysis of
      concatenated sequences and a modification of a newly developed hierarchical
      Bayesian method (BEST) that simultaneously estimates gene trees and species trees
      from multilocus data. We test these approaches on a phylogeny of rapidly
      radiating species wherein divergence times are likely to be smaller than
      effective population sizes, and incomplete lineage sorting is known, in the
      rodent genus, Thomomys. We use seven independent noncoding nuclear sequence loci 
      (total approximately 4300 bp) and between 1 and 12 individuals per taxon to
      construct a phylogenetic hypothesis for eight Thomomys species. The majority-rule
      consensus tree from the partitioned concatenated analysis included 14 strongly
      supported bipartitions, corroborating monophyletic species status of five of the 
      eight named species. The BEST tree strongly supported only the split between the 
      two subgenera and showed very low support for any other clade. Comparison of both
      lineage trees to individual gene trees revealed that the concatenation method
      appears to ignore conflicting signals among gene trees, whereas the BEST tree
      considers conflicting signals and downweights support for those nodes. Bayes
      factor analysis of posterior tree distributions from both analyses strongly favor
      the model underlying the BEST analysis. This comparison underscores the risks of 
      overreliance on results from concatenation, and ignoring the properties of
      coalescence, especially in cases of recent, rapid radiations.
FAU - Belfiore, Natalia M
AU  - Belfiore NM
AD  - 13101 VLSB 3160, Museum of Vertebrate Zoology, University of California,
      Berkeley, California 94720, USA. nmb@berkeley.edu
FAU - Liu, Liang
AU  - Liu L
FAU - Moritz, Craig
AU  - Moritz C
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2008 Jun;57(3):518
MH  - Animals
MH  - Demography
MH  - *Genetic Speciation
MH  - Gophers/*genetics
MH  - Models, Genetic
MH  - North America
MH  - *Phylogeny
EDAT- 2008/04/25 09:00
MHDA- 2008/10/29 09:00
CRDT- 2008/04/25 09:00
PHST- 2008/04/25 09:00 [pubmed]
PHST- 2008/10/29 09:00 [medline]
PHST- 2008/04/25 09:00 [entrez]
AID - 792380183 [pii]
AID - 10.1080/10635150802044011 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Apr;57(2):294-310. doi: 10.1080/10635150802044011.

PMID- 18432549
OWN - NLM
STAT- MEDLINE
DCOM- 20081028
LR  - 20080424
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 2
DP  - 2008 Apr
TI  - Phylogenetic mixture models can reduce node-density artifacts.
PG  - 286-93
LID - 10.1080/10635150802044045 [doi]
AB  - We investigate the performance of phylogenetic mixture models in reducing a
      well-known and pervasive artifact of phylogenetic inference known as the
      node-density effect, comparing them to partitioned analyses of the same data. The
      node-density effect refers to the tendency for the amount of evolutionary change 
      in longer branches of phylogenies to be underestimated compared to that in
      regions of the tree where there are more nodes and thus branches are typically
      shorter. Mixture models allow more than one model of sequence evolution to
      describe the sites in an alignment without prior knowledge of the evolutionary
      processes that characterize the data or how they correspond to different sites.
      If multiple evolutionary patterns are common in sequence evolution, mixture
      models may be capable of reducing node-density effects by characterizing the
      evolutionary processes more accurately. In gene-sequence alignments simulated to 
      have heterogeneous patterns of evolution, we find that mixture models can reduce 
      node-density effects to negligible levels or remove them altogether, performing
      as well as partitioned analyses based on the known simulated patterns. The
      mixture models achieve this without knowledge of the patterns that generated the 
      data and even in some cases without specifying the full or true model of sequence
      evolution known to underlie the data. The latter result is especially important
      in real applications, as the true model of evolution is seldom known. We find the
      same patterns of results for two real data sets with evidence of complex patterns
      of sequence evolution: mixture models substantially reduced node-density effects 
      and returned better likelihoods compared to partitioning models specifically
      fitted to these data. We suggest that the presence of more than one pattern of
      evolution in the data is a common source of error in phylogenetic inference and
      that mixture models can often detect these patterns even without prior knowledge 
      of their presence in the data. Routine use of mixture models alongside other
      approaches to phylogenetic inference may often reveal hidden or unexpected
      patterns of sequence evolution and can improve phylogenetic inference.
FAU - Venditti, Chris
AU  - Venditti C
AD  - School of Biological Sciences, University of Reading, Reading, United Kingdom.
FAU - Meade, Andrew
AU  - Meade A
FAU - Pagel, Mark
AU  - Pagel M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Artifacts
MH  - Computer Simulation
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2008/04/25 09:00
MHDA- 2008/10/29 09:00
CRDT- 2008/04/25 09:00
PHST- 2008/04/25 09:00 [pubmed]
PHST- 2008/10/29 09:00 [medline]
PHST- 2008/04/25 09:00 [entrez]
AID - 792374349 [pii]
AID - 10.1080/10635150802044045 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Apr;57(2):286-93. doi: 10.1080/10635150802044045.

PMID- 18425716
OWN - NLM
STAT- MEDLINE
DCOM- 20081028
LR  - 20080421
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 2
DP  - 2008 Apr
TI  - Accounting for phylogenetic uncertainty in biogeography: a Bayesian approach to
      dispersal-vicariance analysis of the thrushes (Aves: Turdus).
PG  - 257-68
LID - 10.1080/10635150802044003 [doi]
AB  - The phylogeny of the thrushes (Aves: Turdus) has been difficult to reconstruct
      due to short internal branches and lack of node support for certain parts of the 
      tree. Reconstructing the biogeographic history of this group is further
      complicated by the fact that current implementations of biogeographic methods,
      such as dispersal-vicariance analysis (DIVA; Ronquist, 1997), require a fully
      resolved tree. Here, we apply a Bayesian approach to dispersal-vicariance
      analysis that accounts for phylogenetic uncertainty and allows a more accurate
      analysis of the biogeographic history of lineages. Specifically, ancestral area
      reconstructions can be presented as marginal distributions, thus displaying the
      underlying topological uncertainty. Moreover, if there are multiple optimal
      solutions for a single node on a certain tree, integrating over the posterior
      distribution of trees often reveals a preference for a narrower set of solutions.
      We find that despite the uncertainty in tree topology, ancestral area
      reconstructions indicate that the Turdus clade originated in the eastern
      Palearctic during the Late Miocene. This was followed by an early dispersal to
      Africa from where a worldwide radiation took place. The uncertainty in tree
      topology and short branch lengths seems to indicate that this radiation took
      place within a limited time span during the Late Pliocene. The results support
      the role of Africa as a probable source area for intercontinental dispersals as
      suggested for other passerine groups, including basal diversification within the 
      songbird tree.
FAU - Nylander, Johan A A
AU  - Nylander JA
AD  - School of Computational Science, Florida State University, Tallahassee, Florida
      32306, USA. nylander@scs.fsu.edu
FAU - Olsson, Urban
AU  - Olsson U
FAU - Alstrom, Per
AU  - Alstrom P
FAU - Sanmartin, Isabel
AU  - Sanmartin I
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Demography
MH  - *Phylogeny
MH  - Songbirds/*genetics/physiology
MH  - *Uncertainty
EDAT- 2008/04/22 09:00
MHDA- 2008/10/29 09:00
CRDT- 2008/04/22 09:00
PHST- 2008/04/22 09:00 [pubmed]
PHST- 2008/10/29 09:00 [medline]
PHST- 2008/04/22 09:00 [entrez]
AID - 792296538 [pii]
AID - 10.1080/10635150802044003 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Apr;57(2):257-68. doi: 10.1080/10635150802044003.

PMID- 18425715
OWN - NLM
STAT- MEDLINE
DCOM- 20081028
LR  - 20080421
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 2
DP  - 2008 Apr
TI  - Parsimony via consensus.
PG  - 251-6
LID - 10.1080/10635150802040597 [doi]
AB  - The parsimony score of a character on a tree equals the number of state changes
      required to fit that character onto the tree. We show that for unordered,
      reversible characters this score equals the number of tree rearrangements
      required to fit the tree onto the character. We discuss implications of this
      connection for the debate over the use of consensus trees or total evidence and
      show how it provides a link between incongruence of characters and recombination.
FAU - Bruen, Trevor C
AU  - Bruen TC
AD  - Department of Mathematics, University of California, Berkeley, USA.
      tbruen@math.berkeley.edu
FAU - Bryant, David
AU  - Bryant D
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - Genetic Speciation
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2008/04/22 09:00
MHDA- 2008/10/29 09:00
CRDT- 2008/04/22 09:00
PHST- 2008/04/22 09:00 [pubmed]
PHST- 2008/10/29 09:00 [medline]
PHST- 2008/04/22 09:00 [entrez]
AID - 792296771 [pii]
AID - 10.1080/10635150802040597 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Apr;57(2):251-6. doi: 10.1080/10635150802040597.

PMID- 18425714
OWN - NLM
STAT- MEDLINE
DCOM- 20081028
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 2
DP  - 2008 Apr
TI  - Phylogenetic analysis informed by geological history supports multiple,
      sequential invasions of the Mediterranean Basin by the angiosperm family Araceae.
PG  - 269-85
LID - 10.1080/10635150802044029 [doi]
AB  - Despite the remarkable species richness of the Mediterranean flora and its
      well-known geological history, few studies have investigated its temporal and
      spatial origins. Most importantly, the relative contribution of geological
      processes and long-distance dispersal to the composition of contemporary
      Mediterranean biotas remains largely unknown. We used phylogenetic analyses of
      sequences from six chloroplast DNA markers, Bayesian dating methods, and
      ancestral area reconstructions, in combination with paleogeographic,
      paleoclimatic, and ecological evidence, to elucidate the time frame and
      biogeographic events associated with the diversification of Araceae in the
      Mediterranean Basin. We focused on the origin of four species, Ambrosina bassii, 
      Biarum dispar, Helicodiceros muscivorus, Arum pictum, subendemic or endemic to
      Corsica, Sardinia, and the Balearic Archipelago. The results support two main
      invasions of the Mediterranean Basin by the Araceae, one from an area connecting 
      North America and Eurasia in the Late Cretaceous and one from the Anatolian
      microplate in western Asia during the Late Eocene, thus confirming the proposed
      heterogeneous origins of the Mediterranean flora. The subendemic Ambrosina bassii
      and Biarum dispar likely diverged sympatrically from their widespread
      Mediterranean sister clades in the Early-Middle Eocene and Early-Middle Miocene, 
      respectively. Combined evidence corroborates a relictual origin for the endemic
      Helicodiceros muscivorus and Arum pictum, the former apparently representing the 
      first documented case of vicariance driven by the initial splitting of the
      Hercynian belt in the Early Oligocene. A recurrent theme emerging from our
      analyses is that land connections and interruptions, caused by repeated cycles of
      marine transgressions-regressions between the Tethys and Paratethys, favored
      geodispersalist expansion of biotic ranges from western Asia into the western
      Mediterranean Basin and subsequent allopatric speciation at different points in
      time from the Late Eocene to the Late Oligocene.
FAU - Mansion, Guilhem
AU  - Mansion G
AD  - Institut fur Systematische Botanik, Universitat Zurich, Zurich, Switzerland.
      guilhem.mansion@systbot.uzh.ch
FAU - Rosenbaum, Gideon
AU  - Rosenbaum G
FAU - Schoenenberger, Nicola
AU  - Schoenenberger N
FAU - Bacchetta, Gianluigi
AU  - Bacchetta G
FAU - Rossello, Josep A
AU  - Rossello JA
FAU - Conti, Elena
AU  - Conti E
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Araceae/*genetics
MH  - Genetic Speciation
MH  - Geological Phenomena
MH  - Geology
MH  - Mediterranean Region
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2008/04/22 09:00
MHDA- 2008/10/29 09:00
CRDT- 2008/04/22 09:00
PHST- 2008/04/22 09:00 [pubmed]
PHST- 2008/10/29 09:00 [medline]
PHST- 2008/04/22 09:00 [entrez]
AID - 792293122 [pii]
AID - 10.1080/10635150802044029 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Apr;57(2):269-85. doi: 10.1080/10635150802044029.

PMID- 18404577
OWN - NLM
STAT- MEDLINE
DCOM- 20081028
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 2
DP  - 2008 Apr
TI  - Mosaicism, modules, and the evolution of birds: results from a Bayesian approach 
      to the study of morphological evolution using discrete character data.
PG  - 185-201
LID - 10.1080/10635150802022231 [doi]
AB  - The study of morphological evolution after the inferred origin of active flight
      homologous with that in Aves has historically been characterized by an emphasis
      on anatomically disjunct, mosaic patterns of change. Relatively few prior studies
      have used discrete morphological character data in a phylogenetic context to
      quantitatively investigate morphological evolution or mosaic evolution in
      particular. One such previously employed method, which used summed unambiguously 
      optimized synapomorphies, has been the basis for proposing disassociated and
      sequential "modernizing" or "fine-tuning" of pectoral and then pelvic locomotor
      systems after the origin of flight ("pectoral early-pelvic late" hypothesis). We 
      use one of the most inclusive phylogenetic data sets of basal birds to
      investigate properties of this method and to consider the application of a
      Bayesian phylogenetic approach. Bayes factor and statistical comparisons of
      branch length estimates were used to evaluate support for a mosaic pattern of
      character change and the specific pectoral early-pelvic late hypothesis.
      Partitions were defined a priori based on anatomical subregion (e.g., pelvic,
      pectoral) and were based on those hypothesized using the summed synapomorphy
      approach. We compare 80 models all implementing the M(k) model for morphological 
      data but varying in the number of anatomical subregion partitions, the models for
      among-partition rate variation and among-character rate variation, as well as the
      branch length prior. Statistical analysis reveals that partitioning data by
      anatomical subregion, independently estimating branch lengths for partitioned
      data, and use of shared or per partition gamma-shaped among-character rate
      distribution significantly increases estimated model likelihoods. Simulation
      studies reveal that partitioned models where characters are randomly assigned
      perform significantly worse than both the observed model and the single-partition
      equal-rate model, suggesting that only partitioning by anatomical subregion
      increases model performance. The preference for models with partitions defined a 
      priori by anatomical subregion is consistent with a disjunctive pattern of
      character change for the data set investigated and may have implications for
      parameterization of Bayesian analyses of morphological data more generally.
      Statistical tests of differences in estimated branch lengths from the pectoral
      and pelvic partitions do not support the specific pectoral early-pelvic late
      hypothesis proposed from the summed synapomorphy approach; however, results
      suggest limited support for some pectoral branch lengths being significantly
      longer only early at/after the origin of flight.
FAU - Clarke, Julia A
AU  - Clarke JA
AD  - Department of Marine, Earth and Atmospheric Sciences, North Carolina State
      University, Campus Box 8208, Raleigh, NC 27695-8208, USA. Julia Clarke@ncsu.edu
FAU - Middleton, Kevin M
AU  - Middleton KM
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - Birds/*anatomy &amp; histology/*genetics
MH  - Flight, Animal
MH  - *Models, Biological
EDAT- 2008/04/12 09:00
MHDA- 2008/10/29 09:00
CRDT- 2008/04/12 09:00
PHST- 2008/04/12 09:00 [pubmed]
PHST- 2008/10/29 09:00 [medline]
PHST- 2008/04/12 09:00 [entrez]
AID - 792079299 [pii]
AID - 10.1080/10635150802022231 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Apr;57(2):185-201. doi: 10.1080/10635150802022231.

PMID- 18398769
OWN - NLM
STAT- MEDLINE
DCOM- 20081028
LR  - 20080409
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 2
DP  - 2008 Apr
TI  - Maximum likelihood supertrees.
PG  - 243-50
LID - 10.1080/10635150802033014 [doi]
AB  - We analyze a maximum likelihood approach for combining phylogenetic trees into a 
      larger "supertree." This is based on a simple exponential model of phylogenetic
      error, which ensures that ML supertrees have a simple combinatorial description
      (as a median tree, minimizing a weighted sum of distances to the input trees). We
      show that this approach to ML supertree reconstruction is statistically
      consistent (it converges on the true species supertree as more input trees are
      combined), in contrast to the widely used MRP method, which we show can be
      statistically inconsistent under the exponential error model. We also show that
      this statistical consistency extends to an ML approach for constructing species
      supertrees from gene trees. In this setting, incomplete lineage sorting (due to
      coalescence rates of homologous genes being lower than speciation rates) has been
      shown to lead to gene trees that are frequently different from species trees, and
      this can confound efforts to reconstruct the species phylogeny correctly.
FAU - Steel, Mike
AU  - Steel M
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, Department of
      Mathematics and Statistics, University of Canterbury, Christchurch, New Zealand. 
      m.steel@math.canterbury.ac.nz
FAU - Rodrigo, Allen
AU  - Rodrigo A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Computer Simulation
MH  - *Likelihood Functions
MH  - Models, Genetic
MH  - Phylogeny
EDAT- 2008/04/10 09:00
MHDA- 2008/10/29 09:00
CRDT- 2008/04/10 09:00
PHST- 2008/04/10 09:00 [pubmed]
PHST- 2008/10/29 09:00 [medline]
PHST- 2008/04/10 09:00 [entrez]
AID - 792022181 [pii]
AID - 10.1080/10635150802033014 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Apr;57(2):243-50. doi: 10.1080/10635150802033014.

PMID- 18398768
OWN - NLM
STAT- MEDLINE
DCOM- 20081028
LR  - 20080409
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 2
DP  - 2008 Apr
TI  - Genomic outposts serve the phylogenomic pioneers: designing novel nuclear markers
      for genomic DNA extractions of lepidoptera.
PG  - 231-42
LID - 10.1080/10635150802033006 [doi]
AB  - Increasing the number of characters used in phylogenetic studies is the next
      crucial step towards generating robust and stable phylogenetic hypotheses - i.e.,
      strongly supported and consistent across reconstruction method. Here we describe 
      a genomic approach to finding new protein-coding genes for systematics in
      nonmodel taxa, which can be PCR amplified from standard, slightly degraded
      genomic DNA extracts. We test this approach on Lepidoptera, searching the draft
      genomic sequence of the silk moth Bombyx mori, for exons &gt; 500 bp in length,
      removing annotated gene families, and compared remaining exons with butterfly EST
      databases to identify conserved regions for primer design. These primers were
      tested on a set of 65 taxa primarily in the butterfly family Nymphalidae. We were
      able to identify and amplify six previously unused gene regions (Arginine Kinase,
      GAPDH, IDH, MDH, RpS2, and RpS5) and two rarely used gene regions (CAD and DDC)
      that when added to the three traditional gene regions (COI, EF-1alpha and
      wingless) gave a data set of 8114 bp. Phylogenetic robustness and stability
      increased with increasing numbers of genes. Smaller taxanomic subsets were also
      robust when using the full gene data set. The full 11-gene data set was robust
      and stable across reconstruction methods, recovering the major lineages and
      strongly supporting relationships within them. Our methods and insights should be
      applicable to taxonomic groups having a single genomic reference species and
      several EST databases from taxa that diverged less than 100 million years ago.
FAU - Wahlberg, Niklas
AU  - Wahlberg N
AD  - Department of Zoology, Stockholm University, Stockholm, Sweden.
      niklas.wahlberg@utu.fi
FAU - Wheat, Christopher West
AU  - Wheat CW
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - DNA/*genetics
MH  - Genetic Markers
MH  - Genetic Speciation
MH  - *Genome, Insect
MH  - Genomics/methods
MH  - Lepidoptera/*genetics
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2008/04/10 09:00
MHDA- 2008/10/29 09:00
CRDT- 2008/04/10 09:00
PHST- 2008/04/10 09:00 [pubmed]
PHST- 2008/10/29 09:00 [medline]
PHST- 2008/04/10 09:00 [entrez]
AID - 792022088 [pii]
AID - 10.1080/10635150802033006 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Apr;57(2):231-42. doi: 10.1080/10635150802033006.

PMID- 18398767
OWN - NLM
STAT- MEDLINE
DCOM- 20081028
LR  - 20080409
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 2
DP  - 2008 Apr
TI  - Testing the reliability of genetic methods of species identification via
      simulation.
PG  - 216-30
LID - 10.1080/10635150802032990 [doi]
AB  - Although genetic methods of species identification, especially DNA barcoding, are
      strongly debated, tests of these methods have been restricted to a few empirical 
      cases for pragmatic reasons. Here we use simulation to test the performance of
      methods based on sequence comparison (BLAST and genetic distance) and tree
      topology over a wide range of evolutionary scenarios. Sequences were simulated on
      a range of gene trees spanning almost three orders of magnitude in tree depth and
      in coalescent depth; that is, deep or shallow trees with deep or shallow
      coalescences. When the query's conspecific sequences were included in the
      reference alignment, the rate of positive identification was related to the
      degree to which different species were genetically differentiated. The BLAST,
      distance, and liberal tree-based methods returned higher rates of correct
      identification than did the strict tree-based requirement that the query was
      within, but not sister to, a single-species clade. Under this more conservative
      approach, ambiguous outcomes occurred in inverse proportion to the number of
      reference sequences per species. When the query's conspecific sequences were not 
      in the reference alignment, only the strict tree-based approach was relatively
      immune to making false-positive identifications. Thresholds affected the rates at
      which false-positive identifications were made when the query's species was
      unrepresented in the reference alignment but did not otherwise influence
      outcomes. A conservative approach using the strict tree-based method should be
      used initially in large-scale identification systems, with effort made to
      maximize sequence sampling within species. Once the genetic variation within a
      taxonomic group is well characterized and the taxonomy resolved, then the choice 
      of method used should be dictated by considerations of computational efficiency. 
      The requirement for extensive genetic sampling may render these techniques
      inappropriate in some circumstances.
FAU - Ross, Howard A
AU  - Ross HA
AD  - Bioinformatics Institute, University of Auckland, Private Bag 92019, Auckland
      Mail Centre, Auckland 1142, New Zealand. h.ross@auckland.ac.nz
FAU - Murugan, Sumathi
AU  - Murugan S
FAU - Li, Wai Lok Sibon
AU  - Li WL
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - *Computer Simulation
MH  - DNA, Mitochondrial/genetics
MH  - Fungi/genetics
MH  - *Genetic Speciation
MH  - Invertebrates/genetics
MH  - *Models, Genetic
MH  - Sensitivity and Specificity
MH  - Whales/genetics
EDAT- 2008/04/10 09:00
MHDA- 2008/10/29 09:00
CRDT- 2008/04/10 09:00
PHST- 2008/04/10 09:00 [pubmed]
PHST- 2008/10/29 09:00 [medline]
PHST- 2008/04/10 09:00 [entrez]
AID - 792021988 [pii]
AID - 10.1080/10635150802032990 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Apr;57(2):216-30. doi: 10.1080/10635150802032990.

PMID- 18398766
OWN - NLM
STAT- MEDLINE
DCOM- 20081028
LR  - 20181201
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 2
DP  - 2008 Apr
TI  - Inferring species membership using DNA sequences with back-propagation neural
      networks.
PG  - 202-15
LID - 10.1080/10635150802032982 [doi]
AB  - DNA barcoding as a method for species identification is rapidly increasing in
      popularity. However, there are still relatively few rigorous methodological tests
      of DNA barcoding. Current distance-based methods are frequently criticized for
      treating the nearest neighbor as the closest relative via a raw similarity score,
      lacking an objective set of criteria to delineate taxa, or for being incongruent 
      with classical character-based taxonomy. Here, we propose an artificial
      intelligence-based approach - inferring species membership via DNA barcoding with
      back-propagation neural networks (named BP-based species identification) - as a
      new advance to the spectrum of available methods. We demonstrate the value of
      this approach with simulated data sets representing different levels of sequence 
      variation under coalescent simulations with various evolutionary models, as well 
      as with two empirical data sets of COI sequences from East Asian ground beetles
      (Carabidae) and Costa Rican skipper butterflies. With a 630-to 690-bp fragment of
      the COI gene, we identified 97.50% of 80 unknown sequences of ground beetles,
      95.63%, 96.10%, and 100% of 275, 205, and 9 unknown sequences of the neotropical 
      skipper butterfly to their correct species, respectively. Our simulation studies 
      indicate that the success rates of species identification depend on the
      divergence of sequences, the length of sequences, and the number of reference
      sequences. Particularly in cases involving incomplete lineage sorting, this new
      BP-based method appears to be superior to commonly used methods for DNA-based
      species identification.
FAU - Zhang, A B
AU  - Zhang AB
AD  - Institute of Zoology, Chinese Academy of Sciences, Beijing 100080, P. R. China.
      zhangab2008@yahoo.com.cn
FAU - Sikes, D S
AU  - Sikes DS
FAU - Muster, C
AU  - Muster C
FAU - Li, S Q
AU  - Li SQ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Butterflies/genetics
MH  - Coleoptera/genetics
MH  - DNA/*genetics
MH  - Electronic Data Processing
MH  - *Genetic Speciation
MH  - Models, Genetic
MH  - *Neural Networks (Computer)
EDAT- 2008/04/10 09:00
MHDA- 2008/10/29 09:00
CRDT- 2008/04/10 09:00
PHST- 2008/04/10 09:00 [pubmed]
PHST- 2008/10/29 09:00 [medline]
PHST- 2008/04/10 09:00 [entrez]
AID - 792021901 [pii]
AID - 10.1080/10635150802032982 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Apr;57(2):202-15. doi: 10.1080/10635150802032982.

PMID- 18300130
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20100609
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - Crown clades in vertebrate nomenclature: correcting the definition of Crocodylia.
PG  - 173-81
LID - 10.1080/10635150801910469 [doi]
FAU - Martin, Jeremy E
AU  - Martin JE
AD  - Universite Lyon 1, UMR 5125 PEPS CNRS, 2, rue Dubois 69622, Villeurbanne, France.
      jeremy.martin@pepsmail.univ-lyon1.fr
FAU - Benton, Michael J
AU  - Benton MJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2009 Oct;58(5):537-43. PMID: 20525607
MH  - *Alligators and Crocodiles
MH  - Animals
MH  - *Phylogeny
MH  - *Terminology as Topic
EDAT- 2008/02/27 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/27 09:00
PHST- 2008/02/27 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/27 09:00 [entrez]
AID - 790914111 [pii]
AID - 10.1080/10635150801910469 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):173-81. doi: 10.1080/10635150801910469.

PMID- 18300030
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20080226
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - Coding meristic characters for phylogenetic analysis: a comparison of step-matrix
      gap-weighting and generalized frequency coding.
PG  - 167-73
LID - 10.1080/10635150801898938 [doi]
FAU - Lawing, A Michelle
AU  - Lawing AM
AD  - Department of Biology, The University of Texas at Arlington, Arlington, Texas
      76019, USA.
FAU - Meik, Jesse M
AU  - Meik JM
FAU - Schargel, Walter E
AU  - Schargel WE
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2008/02/27 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/27 09:00
PHST- 2008/02/27 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/27 09:00 [entrez]
AID - 790911649 [pii]
AID - 10.1080/10635150801898938 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):167-73. doi: 10.1080/10635150801898938.

PMID- 18300029
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20080226
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - Taxon sampling affects inferences of macroevolutionary processes from
      phylogenetic trees.
PG  - 160-6
LID - 10.1080/10635150701884640 [doi]
FAU - Heath, Tracy A
AU  - Heath TA
AD  - Section of Integrative Biology and Center for Computational Biology and
      Bioinformatics, University of Texas at Austin, Austin, Texas 78712, USA.
      tracyh@mail.utexas.edu
FAU - Zwickl, Derrick J
AU  - Zwickl DJ
FAU - Kim, Junhyong
AU  - Kim J
FAU - Hillis, David M
AU  - Hillis DM
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - Extinction, Biological
MH  - Genetic Speciation
MH  - Models, Biological
MH  - *Phylogeny
MH  - Sample Size
EDAT- 2008/02/27 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/27 09:00
PHST- 2008/02/27 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/27 09:00 [entrez]
AID - 790911923 [pii]
AID - 10.1080/10635150701884640 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):160-6. doi: 10.1080/10635150701884640.

PMID- 18300028
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20080226
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - Application of phylogenetically defined names does not require that every
      specifier be present on a tree.
PG  - 157-60
LID - 10.1080/10635150701883873 [doi]
FAU - Cantino, Philip D
AU  - Cantino PD
AD  - Department of Environmental and Plant Biology, Ohio University, Athens, Ohio
      45701, USA. cantino@ohio.edu
FAU - Olmstead, Richard G
AU  - Olmstead RG
LA  - eng
PT  - Comment
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2006 Oct;55(5):848-58. PMID: 17060205
MH  - *Phylogeny
MH  - *Terminology as Topic
EDAT- 2008/02/27 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/27 09:00
PHST- 2008/02/27 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/27 09:00 [entrez]
AID - 790911521 [pii]
AID - 10.1080/10635150701883873 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):157-60. doi: 10.1080/10635150701883873.

PMID- 18300027
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - The limitations of ancestral state reconstruction and the evolution of the ascus 
      in the Lecanorales (lichenized Ascomycota).
PG  - 141-56
LID - 10.1080/10635150801910451 [doi]
AB  - Ancestral state reconstructions of morphological or ecological traits on
      molecular phylogenies are becoming increasingly frequent. They rely on constancy 
      of character state change rates over trees, a correlation between neutral genetic
      change and phenotypic change, as well as on adequate likelihood models and (for
      Bayesian methods) prior distributions. This investigation explored the outcomes
      of a variety of methods for reconstructing discrete ancestral state in the ascus 
      apex of the Lecanorales, a group containing the majority of lichen-forming
      ascomycetes. Evolution of this character complex has been highly controversial in
      lichen systematics for more than two decades. The phylogeny was estimated using
      Bayesian Markov chain Monte Carlo inference on DNA sequence alignments of three
      genes (small subunit of the mitochondrial rDNA, large subunit of the nuclear
      rDNA, and largest subunit of RNA polymerase II). We designed a novel method for
      assessing the suitable number of discrete gamma categories, which relies on the
      effect on phylogeny estimates rather than on likelihoods. Ancestral state
      reconstructions were performed using maximum parsimony and maximum likelihood on 
      a posterior tree sample as well as two fully Bayesian methods. Resulting
      reconstructions were often strikingly different depending on the method used;
      different methods often assign high confidence to different states at a given
      node. The two fully Bayesian methods disagree about the most probable
      reconstruction in about half of the nodes, even when similar likelihood models
      and similar priors are used. We suggest that similar studies should use several
      methods, awaiting an improved understanding of the statistical properties of the 
      methods. A Lecanora-type ascus may have been ancestral in the Lecanorales. State 
      transformations counts, obtained using stochastic mapping, indicate that the
      number of state changes is 12 to 24, which is considerably greater than the
      minimum three changes needed to explain the four observed ascus apex types.
      Apparently, the ascus in the Lecanorales is far more apt to change than has been 
      recognized. Phylogeny corresponds well with morphology, although it partly
      contradicts currently used delimitations of the Crocyniaceae, Haematommataceae,
      Lecanoraceae, Megalariaceae, Mycoblastaceae, Pilocarpaceae, Psoraceae,
      Ramalinaceae, Scoliciosporaceae, and Squamarinaceae.
FAU - Ekman, Stefan
AU  - Ekman S
AD  - Museum of Evolution, Evolutionary Biology Centre, Uppsala University, Norbyvagen 
      16, Uppsala, Sweden. stefan.ekman@evolmuseum.uu.se
FAU - Andersen, Heidi L
AU  - Andersen HL
FAU - Wedin, Mats
AU  - Wedin M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Ascomycota/*genetics
MH  - *Biological Evolution
MH  - Lichens/*genetics
MH  - Phylogeny
EDAT- 2008/02/27 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/27 09:00
PHST- 2008/02/27 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/27 09:00 [entrez]
AID - 790911800 [pii]
AID - 10.1080/10635150801910451 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):141-56. doi: 10.1080/10635150801910451.

PMID- 18300026
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20080226
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - Discordance of species trees with their most likely gene trees: the case of five 
      taxa.
PG  - 131-40
LID - 10.1080/10635150801905535 [doi]
AB  - Under a coalescent model for within-species evolution, gene trees may differ from
      species trees to such an extent that the gene tree topology most likely to evolve
      along the branches of a species tree can disagree with the species tree topology.
      Gene tree topologies that are more likely to be produced than the topology that
      matches that of the species tree are termed anomalous, and the region of
      branch-length space that gives rise to anomalous gene trees (AGTs) is the anomaly
      zone. We examine the occurrence of anomalous gene trees for the case of five
      taxa, the smallest number of taxa for which every species tree topology has a
      nonempty anomaly zone. Considering all sets of branch lengths that give rise to
      anomalous gene trees, the largest value possible for the smallest branch length
      in the species tree is greater in the five-taxon case (0.1934 coalescent time
      units) than in the previously studied case of four taxa (0.1568). The five-taxon 
      case demonstrates the existence of three phenomena that do not occur in the
      four-taxon case. First, anomalous gene trees can have the same unlabeled topology
      as the species tree. Second, the anomaly zone does not necessarily enclose a ball
      centered at the origin in branch-length space, in which all branches are short.
      Third, as a branch length increases, it is possible for the number of AGTs to
      increase rather than decrease or remain constant. These results, which help to
      describe how the properties of anomalous gene trees increase in complexity as the
      number of taxa increases, will be useful in formulating strategies for evading
      the problem of anomalous gene trees during species tree inference from multilocus
      data.
FAU - Rosenberg, Noah A
AU  - Rosenberg NA
AD  - Department of Human Genetics, University of Michigan, Ann Arbor, Michigan
      48109-2218, USA. rnoah@umich.edu
FAU - Tao, Randa
AU  - Tao R
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Genetic Speciation
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2008/02/27 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/27 09:00
PHST- 2008/02/27 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/27 09:00 [entrez]
AID - 790911047 [pii]
AID - 10.1080/10635150801905535 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):131-40. doi: 10.1080/10635150801905535.

PMID- 18288621
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - The relative importance of body size and paleoclimatic change as explanatory
      variables influencing lineage diversification rate: an evolutionary analysis of
      bullhead catfishes (Siluriformes: Ictaluridae).
PG  - 116-30
LID - 10.1080/10635150801902193 [doi]
AB  - We applied Bayesian phylogenetics, divergence time estimation, diversification
      pattern analysis, and parsimony-based methods of ancestral state reconstruction
      to a combination of nucleotide sequences, maximum body sizes, fossils, and
      paleoclimate data to explore the influence of an extrinsic (climate change) and
      an intrinsic (maximum body size) factor on diversification rates in a North
      American clade of catfishes (Ictaluridae). We found diversification rate to have 
      been significantly variable over time, with significant (or nearly significant)
      rate increases in the early history of Noturus. Though the latter coincided
      closely with a period of dramatic climate change at the Eocene-Oligocene
      boundary, we did not detect evidence for a general association between climate
      change and diversification rate during the entire history of Ictaluridae. Within 
      Ictaluridae, small body size was found to be a near significant predictor of
      species richness. Morphological stasis of several species appears to be a
      consequence of a homoplastic increase in body size. We estimated the maximum
      standard length of the ictalurid ancestor to be approximately 50 cm, comparable
      to Eocene ictalurids (Astephus) and similar to modern sizes of Ameiurus and their
      Asian sister-taxon Cranoglanis. During the late Paleocene and early Eocene, the
      ictalurid ancestor diversified into the lineages represented by the modern
      epigean genera. The majority of modern species originated in the Oligocene and
      Miocene, most likely according to a peripheral isolates model of speciation. We
      discuss the difficulties of detecting macroevolutionary patterns within a lineage
      history and encourage the scrutiny of the terminal Eocene climatic event as a
      direct promoter of diversification.
FAU - Hardman, Michael
AU  - Hardman M
AD  - Laboratory of Molecular Systematics, Finnish Forest Research Institute,
      Jokiniemenkuja 1, 01301, Vantaa, Finland. m.hardman@mac.com
FAU - Hardman, Lotta M
AU  - Hardman LM
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Oxygen Isotopes)
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - *Body Size
MH  - *Climate
MH  - *Fossils
MH  - Ictaluridae/*genetics/growth &amp; development
MH  - Oxygen Isotopes
MH  - Probability
EDAT- 2008/02/22 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/22 09:00
PHST- 2008/02/22 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/22 09:00 [entrez]
AID - 790745723 [pii]
AID - 10.1080/10635150801902193 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):116-30. doi: 10.1080/10635150801902193.

PMID- 18288620
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20080221
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - Testing congruence in phylogenomic analysis.
PG  - 104-15
LID - 10.1080/10635150801910436 [doi]
AB  - Phylogenomic analyses of large sets of genes or proteins have the potential to
      revolutionize our understanding of the tree of life. However, problems arise
      because estimated phylogenies from individual loci often differ because of
      different histories, systematic bias, or stochastic error. We have developed
      Concaterpillar, a hierarchical clustering method based on likelihood-ratio
      testing that identifies congruent loci for phylogenomic analysis. Concaterpillar 
      also includes a test for shared relative evolutionary rates between genes
      indicating whether they should be analyzed separately or by concatenation. In
      simulation studies, the performance of this method is excellent when a multiple
      comparison correction is applied. We analyzed a phylogenomic data set of 60
      translational protein sequences from the major supergroups of eukaryotes and
      identified three congruent subsets of proteins. Analysis of the largest set
      indicates improved congruence relative to the full data set and produced a
      phylogeny with stronger support for five eukaryote supergroups including the
      Opisthokonts, the Plantae, the stramenopiles + Apicomplexa (chromalveolates), the
      Amoebozoa, and the Excavata. In contrast, the phylogeny of the second largest set
      indicates a close relationship between stramenopiles and red algae, to the
      exclusion of alveolates, suggesting gene transfer from the red algal secondary
      symbiont to the ancestral stramenopile host nucleus during the origin of their
      chloroplast. Investigating phylogenomic data sets for conflicting signals has the
      potential to both improve phylogenetic accuracy and inform our understanding of
      genome evolution.
FAU - Leigh, Jessica W
AU  - Leigh JW
AD  - Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, 
      NS, Canada.
FAU - Susko, Edward
AU  - Susko E
FAU - Baumgartner, Manuela
AU  - Baumgartner M
FAU - Roger, Andrew J
AU  - Roger AJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Ribosomal Proteins)
SB  - IM
MH  - Animals
MH  - Cluster Analysis
MH  - Genomics/*methods
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Ribosomal Proteins/genetics
MH  - Symbiosis/genetics
EDAT- 2008/02/22 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/22 09:00
PHST- 2008/02/22 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/22 09:00 [entrez]
AID - 790745874 [pii]
AID - 10.1080/10635150801910436 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):104-15. doi: 10.1080/10635150801910436.

PMID- 18278678
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20161019
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - Efficiency of Markov chain Monte Carlo tree proposals in Bayesian phylogenetics.
PG  - 86-103
LID - 10.1080/10635150801886156 [doi]
AB  - The main limiting factor in Bayesian MCMC analysis of phylogeny is typically the 
      efficiency with which topology proposals sample tree space. Here we evaluate the 
      performance of seven different proposal mechanisms, including most of those used 
      in current Bayesian phylogenetics software. We sampled 12 empirical nucleotide
      data sets--ranging in size from 27 to 71 taxa and from 378 to 2,520 sites--under 
      difficult conditions: short runs, no Metropolis-coupling, and an oversimplified
      substitution model producing difficult tree spaces (Jukes Cantor with equal site 
      rates). Convergence was assessed by comparison to reference samples obtained from
      multiple Metropolis-coupled runs. We find that proposals producing topology
      changes as a side effect of branch length changes (LOCAL and Continuous Change)
      consistently perform worse than those involving stochastic branch rearrangements 
      (nearest neighbor interchange, subtree pruning and regrafting, tree bisection and
      reconnection, or subtree swapping). Among the latter, moves that use an extension
      mechanism to mix local with more distant rearrangements show better overall
      performance than those involving only local or only random rearrangements. Moves 
      with only local rearrangements tend to mix well but have long burn-in periods,
      whereas moves with random rearrangements often show the reverse pattern.
      Combinations of moves tend to perform better than single moves. The time to
      convergence can be shortened considerably by starting with a good tree, but this 
      comes at the cost of compromising convergence diagnostics based on overdispersed 
      starting points. Our results have important implications for developers of
      Bayesian MCMC implementations and for the large group of users of Bayesian
      phylogenetics software.
FAU - Lakner, Clemens
AU  - Lakner C
AD  - Department of Biological Science, Section Ecology and Evolution, Florida State
      University, Tallahassee, Florida 32306-4120, USA. lakner@scs.fsu.edu
FAU - van der Mark, Paul
AU  - van der Mark P
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
FAU - Larget, Bret
AU  - Larget B
FAU - Ronquist, Fredrik
AU  - Ronquist F
LA  - eng
GR  - R01 GM069801/GM/NIGMS NIH HHS/United States
GR  - R01 GM-069801/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
EDAT- 2008/02/19 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/19 09:00
PHST- 2008/02/19 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/19 09:00 [entrez]
AID - 790640415 [pii]
AID - 10.1080/10635150801886156 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):86-103. doi: 10.1080/10635150801886156.

PMID- 18275003
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20080215
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - Does choice in model selection affect maximum likelihood analysis?
PG  - 76-85
LID - 10.1080/10635150801898920 [doi]
AB  - In order to have confidence in model-based phylogenetic analysis, the model of
      nucleotide substitution adopted must be selected in a statistically rigorous
      manner. Several model-selection methods are applicable to maximum likelihood (ML)
      analysis, including the hierarchical likelihood-ratio test (hLRT), Akaike
      information criterion (AIC), Bayesian information criterion (BIC), and decision
      theory (DT), but their performance relative to empirical data has not been
      investigated thoroughly. In this study, we use 250 phylogenetic data sets
      obtained from TreeBASE to examine the effects that choice in model selection has 
      on ML estimation of phylogeny, with an emphasis on optimal topology, bootstrap
      support, and hypothesis testing. We show that the use of different methods leads 
      to the selection of two or more models for approximately 80% of the data sets and
      that the AIC typically selects more complex models than alternative approaches.
      Although ML estimation with different best-fit models results in incongruent tree
      topologies approximately 50% of the time, these differences are primarily
      attributable to alternative resolutions of poorly supported nodes. Furthermore,
      topologies and bootstrap values estimated with ML using alternative statistically
      supported models are more similar to each other than to topologies and bootstrap 
      values estimated with ML under the Kimura two-parameter (K2P) model or maximum
      parsimony (MP). In addition, Swofford-Olsen-Waddell-Hillis (SOWH) tests indicate 
      that ML trees estimated with alternative best-fit models are usually not
      significantly different from each other when evaluated with the same model.
      However, ML trees estimated with statistically supported models are often
      significantly suboptimal to ML trees made with the K2P model when both are
      evaluated with K2P, indicating that not all models perform in an equivalent
      manner. Nevertheless, the use of alternative statistically supported models
      generally does not affect tests of monophyletic relationships under either the
      Shimodaira-Hasegawa (S-H) or SOWH methods. Our results suggest that although
      choice in model selection has a strong impact on optimal tree topology, it rarely
      affects evolutionary inferences drawn from the data because differences are
      mainly confined to poorly supported nodes. Moreover, since ML with alternative
      best-fit models tends to produce more similar estimates of phylogeny than ML
      under the K2P model or MP, the use of any statistically based model-selection
      method is vastly preferable to forgoing the model-selection process altogether.
FAU - Ripplinger, Jennifer
AU  - Ripplinger J
AD  - Bioinformatics and Computational Biology, University of Idaho, Moscow, Idaho
      83844-3051, USA. jripplinger@vandals.uidaho.edu
FAU - Sullivan, Jack
AU  - Sullivan J
LA  - eng
GR  - 1P20PR016448-01/PR/OCPHP CDC HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Decision Theory
MH  - *Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2008/02/16 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/16 09:00
PHST- 2008/02/16 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/16 09:00 [entrez]
AID - 790618093 [pii]
AID - 10.1080/10635150801898920 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):76-85. doi: 10.1080/10635150801898920.

PMID- 18275002
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - Historical biogeography, divergence times, and diversification patterns of bumble
      bees (Hymenoptera: Apidae: Bombus).
PG  - 58-75
LID - 10.1080/10635150801898912 [doi]
AB  - Bumble bees (Bombus) are a cold-adapted, largely alpine group that can elucidate 
      patterns of Holarctic historical biogeography, particularly in comparison to the 
      alpine plants with which they likely coevolved. A recently published molecular
      phylogeny of bumble bees provides uniquely comprehensive species sampling for
      exploring historical patterns of distribution and diversification. Using this
      phylogeny and detailed data on extant distributions, I reconstruct the historical
      distribution of bumble bees in a temporal framework, estimating divergence times 
      using fossil data and molecular rates derived from the literature. The nearly
      comprehensive phylogeny allows assessment of the tempo of diversification within 
      the bumble bees using lineage-through-time plots and diversification statistics, 
      which have been performed with special consideration to confidence intervals.
      These analyses reveal movements of Bombus concordant with geographic and climatic
      events of the late Cenozoic. The initial diversification of extant bumble bee
      lineages was estimated at around 25 to 40 Ma, near the Eocene-Oligocene boundary 
      34 Ma, a period of dramatic global cooling. Dispersal-vicariance analysis (DIVA) 
      predicted an Old World Bombus ancestor, with early diversification events largely
      restricted to the eastern Old World. The numerous intercontinental dispersal
      events occurred mostly in the direction of Old World to New World and North
      America to South America. Early movements from the Palearctic into the Nearctic
      most likely took place after 20 Ma and may have coincided with a period of
      Miocene cooling that gave rise to taiga habitat across Beringia. Subsequent
      dispersal between these regions is estimated to have occurred among boreal and
      tundra-adapted species mostly in the last 5 million years. Radiations are
      estimated in both Nearctic and Neotropical regions at approximately 6 to 8 Ma and
      after 3.5 Ma, concordant with the opening of land corridors between the
      continents.
FAU - Hines, Heather M
AU  - Hines HM
AD  - Department of Entomology, University of Illinois Urbana-Champaign, Urbana, IL
      61801, USA. hhines@life.uiuc.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Biological
MH  - Americas
MH  - Animals
MH  - Asia
MH  - *Bees
MH  - *Biological Evolution
MH  - Cold Climate
MH  - Europe
MH  - *Geography
EDAT- 2008/02/16 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/16 09:00
PHST- 2008/02/16 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/16 09:00 [entrez]
AID - 790617487 [pii]
AID - 10.1080/10635150801898912 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):58-75. doi: 10.1080/10635150801898912.

PMID- 18275001
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20080215
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - Resolving an ancient, rapid radiation in Saxifragales.
PG  - 38-57
LID - 10.1080/10635150801888871 [doi]
AB  - Despite the prior use of approximately 9000 bp, deep-level relationships within
      the angiosperm clade, Saxifragales remain enigmatic, due to an ancient, rapid
      radiation (89.5 to 110 Ma based on the fossil record). To resolve these deep
      relationships, we constructed several new data sets: (1) 16 genes representing
      the three genomic compartments within plant cells (2 nuclear, 10 plastid, 4
      mitochondrial; aligned, analyzed length = 21,460 bp) for 28 taxa; (2) the entire 
      plastid inverted repeat (IR; 26,625 bp) for 17 taxa; (3) "total evidence" (50,845
      bp) for both 17 and 28 taxa (the latter missing the IR). Bayesian and ML methods 
      yielded identical topologies across partitions with most clades receiving high
      posterior probability (pp = 1.0) and bootstrap (95% to 100%) values, suggesting
      that with sufficient data, rapid radiations can be resolved. In contrast,
      parsimony analyses of different partitions yielded conflicting topologies,
      particularly with respect to the placement of Paeoniaceae, a clade characterized 
      by a long branch. In agreement with published simulations, the addition of
      characters increased bootstrap support for the putatively erroneous placement of 
      Paeoniaceae. Although having far fewer parsimony-informative sites, slowly
      evolving plastid genes provided higher resolution and support for deep-level
      relationships than rapidly evolving plastid genes, yielding a topology close to
      the Bayesian and ML total evidence tree. The plastid IR region may be an ideal
      source of slowly evolving genes for resolution of deep-level angiosperm
      divergences that date to 90 My or more. Rapidly evolving genes provided support
      for tip relationships not recovered with slowly evolving genes, indicating some
      complementarity. Age estimates using penalized likelihood with and without age
      constraints for the 28-taxon, total evidence data set are comparable to fossil
      dates, whereas estimates based on the 17-taxon data are much older than implied
      by the fossil record. Hence, sufficient taxon density, and not simply numerous
      base pairs, is important in reliably estimating ages. Age estimates indicate that
      the early diversification of Saxifragales occurred rapidly, over a time span as
      short as 6 million years. Between 25,000 and 50,000 bp were needed to resolve
      this radiation with high support values. Extrapolating from Saxifragales, a
      similar number of base pairs may be needed to resolve the many other deep-level
      radiations of comparable age in angiosperms.
FAU - Jian, Shuguang
AU  - Jian S
AD  - South China Botanical Garden, the Chinese Academy of Sciences, Guangzhou, China.
FAU - Soltis, Pamela S
AU  - Soltis PS
FAU - Gitzendanner, Matthew A
AU  - Gitzendanner MA
FAU - Moore, Michael J
AU  - Moore MJ
FAU - Li, Ruiqi
AU  - Li R
FAU - Hendry, Tory A
AU  - Hendry TA
FAU - Qiu, Yin-Long
AU  - Qiu YL
FAU - Dhingra, Amit
AU  - Dhingra A
FAU - Bell, Charles D
AU  - Bell CD
FAU - Soltis, Douglas E
AU  - Soltis DE
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Crassulaceae/*genetics
MH  - *Evolution, Molecular
MH  - *Genome, Plastid
MH  - Hamamelidaceae/*genetics
MH  - Phylogeny
MH  - Repetitive Sequences, Nucleic Acid
MH  - Saxifragaceae/*genetics
MH  - Sequence Analysis, DNA
EDAT- 2008/02/16 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/16 09:00
PHST- 2008/02/16 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/16 09:00 [entrez]
AID - 790615049 [pii]
AID - 10.1080/10635150801888871 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):38-57. doi: 10.1080/10635150801888871.

PMID- 18266181
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - Morphological and molecular evidence for a stepwise evolutionary transition from 
      teeth to baleen in mysticete whales.
PG  - 15-37
LID - 10.1080/10635150701884632 [doi]
AB  - The origin of baleen in mysticete whales represents a major transition in the
      phylogenetic history of Cetacea. This key specialization, a keratinous sieve that
      enables filter-feeding, permitted exploitation of a new ecological niche and
      heralded the evolution of modern baleen-bearing whales, the largest animals on
      Earth. To date, all formally described mysticete fossils conform to two types:
      toothed species from Oligocene-age rocks ( approximately 24 to 34 million years
      old) and toothless species that presumably utilized baleen to feed (Recent to
      approximately 30 million years old). Here, we show that several Oligocene toothed
      mysticetes have nutrient foramina and associated sulci on the lateral portions of
      their palates, homologous structures in extant mysticetes house vessels that
      nourish baleen. The simultaneous occurrence of teeth and nutrient foramina
      implies that both teeth and baleen were present in these early mysticetes.
      Phylogenetic analyses of a supermatrix that includes extinct taxa and new data
      for 11 nuclear genes consistently resolve relationships at the base of Mysticeti.
      The combined data set of 27,340 characters supports a stepwise transition from a 
      toothed ancestor, to a mosaic intermediate with both teeth and baleen, to modern 
      baleen whales that lack an adult dentition but retain developmental and genetic
      evidence of their ancestral toothed heritage. Comparative sequence data for ENAM 
      (enamelin) and AMBN (ameloblastin) indicate that enamel-specific loci are present
      in Mysticeti but have degraded to pseudogenes in this group. The dramatic
      transformation in mysticete feeding anatomy documents an apparently rare,
      stepwise mode of evolution in which a composite phenotype bridged the gap between
      primitive and derived morphologies; a combination of fossil and molecular
      evidence provides a multifaceted record of this macroevolutionary pattern.
FAU - Demere, Thomas A
AU  - Demere TA
AD  - Department of Paleontology, San Diego Natural History Museum, San Diego,
      California 92112, USA.
FAU - McGowen, Michael R
AU  - McGowen MR
FAU - Berta, Annalisa
AU  - Berta A
FAU - Gatesy, John
AU  - Gatesy J
LA  - eng
SI  - GENBANK/EU444877
SI  - GENBANK/EU444878
SI  - GENBANK/EU444879
SI  - GENBANK/EU444880
SI  - GENBANK/EU444881
SI  - GENBANK/EU444882
SI  - GENBANK/EU444883
SI  - GENBANK/EU444884
SI  - GENBANK/EU444885
SI  - GENBANK/EU444886
SI  - GENBANK/EU444887
SI  - GENBANK/EU444888
SI  - GENBANK/EU444889
SI  - GENBANK/EU444890
SI  - GENBANK/EU444891
SI  - GENBANK/EU444892
SI  - GENBANK/EU444893
SI  - GENBANK/EU444894
SI  - GENBANK/EU444895
SI  - GENBANK/EU444896
SI  - GENBANK/EU444897
SI  - GENBANK/EU444898
SI  - GENBANK/EU444899
SI  - GENBANK/EU444900
SI  - GENBANK/EU444901
SI  - GENBANK/EU444902
SI  - GENBANK/EU444903
SI  - GENBANK/EU444904
SI  - GENBANK/EU444905
SI  - GENBANK/EU444906
SI  - GENBANK/EU444907
SI  - GENBANK/EU444908
SI  - GENBANK/EU444909
SI  - GENBANK/EU444910
SI  - GENBANK/EU444911
SI  - GENBANK/EU444912
SI  - GENBANK/EU444913
SI  - GENBANK/EU444914
SI  - GENBANK/EU444915
SI  - GENBANK/EU444916
SI  - GENBANK/EU444917
SI  - GENBANK/EU444918
SI  - GENBANK/EU444919
SI  - GENBANK/EU444920
SI  - GENBANK/EU444921
SI  - GENBANK/EU444922
SI  - GENBANK/EU444923
SI  - GENBANK/EU444924
SI  - GENBANK/EU444925
SI  - GENBANK/EU444926
SI  - GENBANK/EU444927
SI  - GENBANK/EU444928
SI  - GENBANK/EU444929
SI  - GENBANK/EU444930
SI  - GENBANK/EU444931
SI  - GENBANK/EU444932
SI  - GENBANK/EU444933
SI  - GENBANK/EU444934
SI  - GENBANK/EU444935
SI  - GENBANK/EU444936
SI  - GENBANK/EU444937
SI  - GENBANK/EU444938
SI  - GENBANK/EU444939
SI  - GENBANK/EU444940
SI  - GENBANK/EU444941
SI  - GENBANK/EU444942
SI  - GENBANK/EU444943
SI  - GENBANK/EU444944
SI  - GENBANK/EU444945
SI  - GENBANK/EU444946
SI  - GENBANK/EU444947
SI  - GENBANK/EU444948
SI  - GENBANK/EU444949
SI  - GENBANK/EU444950
SI  - GENBANK/EU444951
SI  - GENBANK/EU444952
SI  - GENBANK/EU444953
SI  - GENBANK/EU444954
SI  - GENBANK/EU444955
SI  - GENBANK/EU444956
SI  - GENBANK/EU444957
SI  - GENBANK/EU444958
SI  - GENBANK/EU444959
SI  - GENBANK/EU444960
SI  - GENBANK/EU444961
SI  - GENBANK/EU444962
SI  - GENBANK/EU444963
SI  - GENBANK/EU444964
SI  - GENBANK/EU444965
SI  - GENBANK/EU444966
SI  - GENBANK/EU444967
SI  - GENBANK/EU444968
SI  - GENBANK/EU444969
SI  - GENBANK/EU444970
SI  - GENBANK/EU444971
SI  - GENBANK/EU444972
SI  - GENBANK/EU444973
SI  - GENBANK/EU444974
SI  - GENBANK/EU444975
SI  - GENBANK/EU444976
SI  - GENBANK/EU444977
SI  - GENBANK/EU444978
SI  - GENBANK/EU444979
SI  - GENBANK/EU444980
SI  - GENBANK/EU444981
SI  - GENBANK/EU444982
SI  - GENBANK/EU444983
SI  - GENBANK/EU444984
SI  - GENBANK/EU444985
SI  - GENBANK/EU444986
SI  - GENBANK/EU444987
SI  - GENBANK/EU444988
SI  - GENBANK/EU444989
SI  - GENBANK/EU444990
SI  - GENBANK/EU444991
SI  - GENBANK/EU444992
SI  - GENBANK/EU444993
SI  - GENBANK/EU444994
SI  - GENBANK/EU444995
SI  - GENBANK/EU444996
SI  - GENBANK/EU444997
SI  - GENBANK/EU444998
SI  - GENBANK/EU444999
SI  - GENBANK/EU445000
SI  - GENBANK/EU445001
SI  - GENBANK/EU445002
SI  - GENBANK/EU445003
SI  - GENBANK/EU445004
SI  - GENBANK/EU445005
SI  - GENBANK/EU445006
SI  - GENBANK/EU445007
SI  - GENBANK/EU445008
SI  - GENBANK/EU445009
SI  - GENBANK/EU445010
SI  - GENBANK/EU445011
SI  - GENBANK/EU445012
SI  - GENBANK/EU445013
SI  - GENBANK/EU445014
SI  - GENBANK/EU445015
SI  - GENBANK/EU445016
SI  - GENBANK/EU445017
SI  - GENBANK/EU445018
SI  - GENBANK/EU445019
SI  - GENBANK/EU445020
SI  - GENBANK/EU445021
SI  - GENBANK/EU445022
SI  - GENBANK/EU445023
SI  - GENBANK/EU445024
SI  - GENBANK/EU445025
SI  - GENBANK/EU445026
SI  - GENBANK/EU445027
SI  - GENBANK/EU445028
SI  - GENBANK/EU445029
SI  - GENBANK/EU445030
SI  - GENBANK/EU445031
SI  - GENBANK/EU445032
SI  - GENBANK/EU445033
SI  - GENBANK/EU445034
SI  - GENBANK/EU445035
SI  - GENBANK/EU445036
SI  - GENBANK/EU445037
SI  - GENBANK/EU445038
SI  - GENBANK/EU445039
SI  - GENBANK/EU445040
SI  - GENBANK/EU445041
SI  - GENBANK/EU445042
SI  - GENBANK/EU445043
SI  - GENBANK/EU445044
SI  - GENBANK/EU445045
SI  - GENBANK/EU445046
SI  - GENBANK/EU445047
SI  - GENBANK/EU445048
SI  - GENBANK/EU445049
SI  - GENBANK/EU445050
SI  - GENBANK/EU445051
SI  - GENBANK/EU445052
SI  - GENBANK/EU445053
SI  - GENBANK/EU445054
SI  - GENBANK/EU445055
SI  - GENBANK/EU445056
SI  - GENBANK/EU445057
SI  - GENBANK/EU445058
SI  - GENBANK/EU445059
SI  - GENBANK/EU445060
SI  - GENBANK/EU445061
SI  - GENBANK/EU445062
SI  - GENBANK/EU445063
SI  - GENBANK/EU445064
SI  - GENBANK/EU445065
SI  - GENBANK/EU445066
SI  - GENBANK/EU445067
SI  - GENBANK/EU445068
SI  - GENBANK/EU445069
SI  - GENBANK/EU445070
SI  - GENBANK/EU445071
SI  - GENBANK/EU445072
SI  - GENBANK/EU445073
SI  - GENBANK/EU445074
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Calcium-Binding Proteins)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - *Biological Evolution
MH  - Calcium-Binding Proteins/genetics
MH  - *Fossils
MH  - Molecular Sequence Data
MH  - Palate/*anatomy &amp; histology/blood supply
MH  - Polymerase Chain Reaction
MH  - Sequence Analysis, DNA
MH  - Tooth/*anatomy &amp; histology
MH  - Whales/*anatomy &amp; histology/genetics
EDAT- 2008/02/13 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/13 09:00
PHST- 2008/02/13 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/13 09:00 [entrez]
AID - 790546792 [pii]
AID - 10.1080/10635150701884632 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):15-37. doi: 10.1080/10635150701884632.

PMID- 18253896
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - Maximum likelihood inference of geographic range evolution by dispersal, local
      extinction, and cladogenesis.
PG  - 4-14
LID - 10.1080/10635150701883881 [doi]
AB  - In historical biogeography, model-based inference methods for reconstructing the 
      evolution of geographic ranges on phylogenetic trees are poorly developed
      relative to the diversity of analogous methods available for inferring character 
      evolution. We attempt to rectify this deficiency by constructing a
      dispersal-extinction-cladogenesis (DEC) model for geographic range evolution that
      specifies instantaneous transition rates between discrete states (ranges) along
      phylogenetic branches and apply it to estimating likelihoods of ancestral states 
      (range inheritance scenarios) at cladogenesis events. Unlike an earlier version
      of this approach, the present model allows for an analytical solution to
      probabilities of range transitions as a function of time, enabling free
      parameters in the model, rates of dispersal, and local extinction to be estimated
      by maximum likelihood. Simulation results indicate that accurate parameter
      estimates may be difficult to obtain in practice but also show that ancestral
      range inheritance scenarios nevertheless can be correctly recovered with high
      success if rates of range evolution are low relative to the rate of cladogenesis.
      We apply the DEC model to a previously published, exemplary case study of island 
      biogeography involving Hawaiian endemic angiosperms in Psychotria (Rubiaceae),
      showing how the DEC model can be iteratively refined from inspecting inferences
      of range evolution and also how geological constraints involving times of island 
      origin may be imposed on the likelihood function. The DEC model is sufficiently
      similar to character models that it might serve as a gateway through which many
      existing comparative methods for characters could be imported into the realm of
      historical biogeography; moreover, it might also inspire the conceptual expansion
      of character models toward inclusion of evolutionary change as directly
      coincident, either as cause or consequence, with cladogenesis events. The DEC
      model is thus an incremental advance that highlights considerable potential in
      the nascent field of model-based historical biogeographic inference.
FAU - Ree, Richard H
AU  - Ree RH
AD  - Department of Botany, Field Museum of Natural History, Chicago, Illinois 60605,
      USA.
FAU - Smith, Stephen A
AU  - Smith SA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Extinction, Biological
MH  - *Geography
MH  - Hawaii
MH  - Likelihood Functions
MH  - *Models, Biological
MH  - *Psychotria
EDAT- 2008/02/07 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/02/07 09:00
PHST- 2008/02/07 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/02/07 09:00 [entrez]
AID - 790431854 [pii]
AID - 10.1080/10635150701883881 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):4-14. doi: 10.1080/10635150701883881.

PMID- 18188785
OWN - NLM
STAT- MEDLINE
DCOM- 20080724
LR  - 20080111
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 1
DP  - 2008 Feb
TI  - The expanding contributions of systematic biology.
PG  - 1-3
LID - 10.1080/10635150701856671 [doi]
FAU - Page, Roderic D M
AU  - Page RD
FAU - Sullivan, Jack
AU  - Sullivan J
LA  - eng
PT  - Editorial
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biology/classification/*trends
MH  - Publishing/*trends
EDAT- 2008/01/12 09:00
MHDA- 2008/07/25 09:00
CRDT- 2008/01/12 09:00
PHST- 2008/01/12 09:00 [pubmed]
PHST- 2008/07/25 09:00 [medline]
PHST- 2008/01/12 09:00 [entrez]
AID - 789468535 [pii]
AID - 10.1080/10635150701856671 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Feb;57(1):1-3. doi: 10.1080/10635150701856671.

PMID- 18075934
OWN - NLM
STAT- MEDLINE
DCOM- 20080221
LR  - 20161019
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 6
DP  - 2007 Dec
TI  - A nonparametric method for accommodating and testing across-site rate variation.
PG  - 975-87
AB  - Substitution rates are one of the most fundamental parameters in a phylogenetic
      analysis and are represented in phylogenetic models as the branch lengths on a
      tree. Variation in substitution rates across an alignment of molecular sequences 
      is well established and likely caused by variation in functional constraint
      across the genes encoded in the sequences. Rate variation across alignment sites 
      is important to accommodate in a phylogenetic analysis; failure to account for
      across-site rate variation can cause biased estimates of phylogeny or other model
      parameters. Traditionally, rate variation across sites has been modeled by
      treating the rate for a site as a random variable drawn from some probability
      distribution (such as the gamma probability distribution) or by partitioning
      sites to different rate classes and estimating the rate for each class
      independently. We consider a different approach, related to site-specific models 
      in which sites are partitioned to rate classes. However, instead of treating the 
      partitioning scheme in which sites are assigned to rate classes as a fixed
      assumption of the analysis, we treat the rate partitioning as a random variable
      under a Dirichlet process prior. We find that the Dirichlet process prior model
      for across-site rate variation fits alignments of DNA sequence data better than
      commonly used models of across-site rate variation. The method appears to
      identify the underlying codon structure of protein-coding genes; rate partitions 
      that were sampled by the Markov chain Monte Carlo procedure were closer to a
      partition in which sites are assigned to rate classes by codon position than to
      randomly permuted partitions but still allow for additional variability across
      sites.
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
AD  - Department of Integrative Biology, University of California, Berkeley, CA 94720, 
      USA. johnh@berkeley.edu
FAU - Suchard, Marc A
AU  - Suchard MA
LA  - eng
GR  - R01 GM069801/GM/NIGMS NIH HHS/United States
GR  - GM068955/GM/NIGMS NIH HHS/United States
GR  - GM069801/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Models, Genetic
MH  - Phylogeny
MH  - *Statistics, Nonparametric
EDAT- 2007/12/14 09:00
MHDA- 2008/02/22 09:00
CRDT- 2007/12/14 09:00
PHST- 2007/12/14 09:00 [pubmed]
PHST- 2008/02/22 09:00 [medline]
PHST- 2007/12/14 09:00 [entrez]
AID - 788417582 [pii]
AID - 10.1080/10635150701670569 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Dec;56(6):975-87. doi: 10.1080/10635150701670569.

PMID- 18066932
OWN - NLM
STAT- MEDLINE
DCOM- 20080221
LR  - 20071210
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 6
DP  - 2007 Dec
TI  - Naming species in phylogenetic nomenclature.
PG  - 1011-21
FAU - Wolsan, Mieczyslaw
AU  - Wolsan M
AD  - Museum and Institute of Zoology, Polish Academy of Sciences, Wilcza 64, Warszawa,
      Poland. wolsan@miiz.waw.pl
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Genetic Speciation
MH  - *Phylogeny
MH  - Terminology as Topic
EDAT- 2007/12/11 09:00
MHDA- 2008/02/22 09:00
CRDT- 2007/12/11 09:00
PHST- 2007/12/11 09:00 [pubmed]
PHST- 2008/02/22 09:00 [medline]
PHST- 2007/12/11 09:00 [entrez]
AID - 788069204 [pii]
AID - 10.1080/10635150701656352 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Dec;56(6):1011-21. doi: 10.1080/10635150701656352.

PMID- 18066931
OWN - NLM
STAT- MEDLINE
DCOM- 20080221
LR  - 20071210
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 6
DP  - 2007 Dec
TI  - Increasing the efficiency of searches for the maximum likelihood tree in a
      phylogenetic analysis of up to 150 nucleotide sequences.
PG  - 988-1010
AB  - Even when the maximum likelihood (ML) tree is a better estimate of the true
      phylogenetic tree than those produced by other methods, the result of a poor ML
      search may be no better than that of a more thorough search under some faster
      criterion. The ability to find the globally optimal ML tree is therefore
      important. Here, I compare a range of heuristic search strategies (and their
      associated computer programs) in terms of their success at locating the ML tree
      for 20 empirical data sets with 14 to 158 sequences and 411 to 120,762 aligned
      nucleotides. Three distinct topics are discussed: the success of the search
      strategies in relation to certain features of the data, the generation of
      starting trees for the search, and the exploration of multiple islands of trees. 
      As a starting tree, there was little difference among the neighbor-joining tree
      based on absolute differences (including the BioNJ tree), the stepwise-addition
      parsimony tree (with or without nearest-neighbor-interchange (NNI) branch
      swapping), and the stepwise-addition ML tree. The latter produced the best ML
      score on average but was orders of magnitude slower than the alternatives. The
      BioNJ tree was second best on average. As search strategies, star decomposition
      and quartet puzzling were the slowest and produced the worst ML scores. The
      DPRml, IQPNNI, MultiPhyl, PhyML, PhyNav, and TreeFinder programs with default
      options produced qualitatively similar results, each locating a single tree that 
      tended to be in an NNI suboptimum (rather than the global optimum) when the data 
      set had low phylogenetic information. For such data sets, there were multiple
      tree islands with very similar ML scores. The likelihood surface only became
      relatively simple for data sets that contained approximately 500 aligned
      nucleotides for 50 sequences and 3,000 nucleotides for 100 sequences. The RAxML
      and GARLI programs allowed multiple islands to be explored easily, but both
      programs also tended to find NNI suboptima. A newly developed version of the
      likelihood ratchet using PAUP* successfully found the peaks of multiple islands, 
      but its speed needs to be improved.
FAU - Morrison, David A
AU  - Morrison DA
AD  - Department of Parasitology (SWEPAR), National Veterinary Institute and Swedish
      University of Agricultural Sciences, Uppsala, Sweden. david.morrison@bvf.slu.se
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - Classification/*methods
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2007/12/11 09:00
MHDA- 2008/02/22 09:00
CRDT- 2007/12/11 09:00
PHST- 2007/12/11 09:00 [pubmed]
PHST- 2008/02/22 09:00 [medline]
PHST- 2007/12/11 09:00 [entrez]
AID - 788069580 [pii]
AID - 10.1080/10635150701779808 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Dec;56(6):988-1010. doi: 10.1080/10635150701779808.

PMID- 18066930
OWN - NLM
STAT- MEDLINE
DCOM- 20080221
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 6
DP  - 2007 Dec
TI  - Toward an integrated system of clade names.
PG  - 956-74
AB  - Although the proposition that higher taxa should correspond to clades is widely
      accepted, current nomenclature does not distinguish clearly between different
      clades in nested series. In particular, the same name is often applied to a total
      clade, its crown clade, and clades originating with various nodes, branches, and 
      apomorphies in between. An integrated system of clade names is described based on
      categories of clades defined with respect to lineages that have survived to the
      present time. In this system, the most widely known names are applied to crown
      clades, the names of total clades are formed by adding a standard prefix to the
      names of the corresponding crowns, and the names of apomorphy clades describe the
      specific apomorphies with which they originated. Relative to traditional
      approaches, this integrated approach to naming clades is both more precise
      concerning the associations of names with particular clades and more efficient
      with regard to the cognitive effort required to recognize the names of
      corresponding crown and total clades. It also seems preferable to five
      alternatives that could be used to make the same distinctions. The integrated
      system of clade names has several advantages, including the facilitation of
      communication among biologists who study distantly related clades, promoting a
      broader conceptualization of the origins of distinctive clades of extant
      organisms and emphasizing the continuous nature of evolution.
FAU - de Queiroz, Kevin
AU  - de Queiroz K
AD  - Department of Vertebrate Zoology, National Museum of Natural History, Smithsonian
      Institution, Washington, DC 20560, USA. dequeirozk@si.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - Models, Biological
MH  - Snakes/genetics
EDAT- 2007/12/11 09:00
MHDA- 2008/02/22 09:00
CRDT- 2007/12/11 09:00
PHST- 2007/12/11 09:00 [pubmed]
PHST- 2008/02/22 09:00 [medline]
PHST- 2007/12/11 09:00 [entrez]
AID - 788068784 [pii]
AID - 10.1080/10635150701656378 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Dec;56(6):956-74. doi: 10.1080/10635150701656378.

PMID- 18066929
OWN - NLM
STAT- MEDLINE
DCOM- 20080221
LR  - 20071210
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 6
DP  - 2007 Dec
TI  - The web and the structure of taxonomy.
PG  - 943-55
AB  - An easily accessible taxonomic knowledge base is critically important for all
      biodiversity-related sciences. At present, taxonomic information is organized and
      regulated by a system of rules and conventions that date back to the introduction
      of binomial nomenclature by Linnaeus. The taxonomy of any particular group of
      organisms comprises the sum information in the taxonomic literature, supported by
      designated type specimens in major collections. In this article, the way modern
      means of disseminating information will change the practice of taxonomy, in
      particular the Internet, is explored. Basic taxonomic information, such as
      specimen-level data, location of types, and name catalogues are already
      available, at least for some groups, on the Web. Specialist taxonomic databases, 
      key-construction programs, and other software useful for systematists are also
      increasingly available. There has also been a move towards Web-publishing of
      taxonomic hypotheses, though as yet this is not fully permitted by the Codes of
      Nomenclature. A further and more radical move would be to transfer taxonomy
      completely to the Web. A possible model of this is discussed, as well as a pilot 
      project, the "CATE" initiative, which seeks to explore the advantages and
      disadvantages of such a move. It is argued that taxonomy needs to forge better
      links with its user-communities to maintain its funding base, and that an
      important part of this is making the products of its research more accessible
      through the Internet.
FAU - Godfray, H C J
AU  - Godfray HC
AD  - Department of Zoology, University of Oxford, Oxford, UK.
      charles.godfray@zoo.ox.ac.uk
FAU - Clark, B R
AU  - Clark BR
FAU - Kitching, I J
AU  - Kitching IJ
FAU - Mayo, S J
AU  - Mayo SJ
FAU - Scoble, M J
AU  - Scoble MJ
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biodiversity
MH  - Classification/*methods
MH  - Genetic Speciation
MH  - *Internet
MH  - Publishing
EDAT- 2007/12/11 09:00
MHDA- 2008/02/22 09:00
CRDT- 2007/12/11 09:00
PHST- 2007/12/11 09:00 [pubmed]
PHST- 2008/02/22 09:00 [medline]
PHST- 2007/12/11 09:00 [entrez]
AID - 788069067 [pii]
AID - 10.1080/10635150701777521 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Dec;56(6):943-55. doi: 10.1080/10635150701777521.

PMID- 18066928
OWN - NLM
STAT- MEDLINE
DCOM- 20080221
LR  - 20071210
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 6
DP  - 2007 Dec
TI  - Adding more ecology into species delimitation: ecological niche models and
      phylogeography help define cryptic species in the black salamander (Aneides
      flavipunctatus).
PG  - 924-42
AB  - Being able to efficiently and accurately delimit species is one of the most basic
      and important aspects of systematics because species are the fundamental unit of 
      analysis in biogeography, ecology, and conservation. We present a rationale and
      approach for combining ecological niche modeling, spatially explicit analyses of 
      environmental data, and phylogenetics in species delimitation, and we use our
      methodology in an empirical example focusing on Aneides flavipunctatus, the black
      salamander (Caudata: Plethodontidae), in California. We assess the relationships 
      between genetic, environmental, and geographic distance among populations. We use
      11 climatic variables and point locality data from public databases to create
      ecological niche models. The suitability of potential contact zones between
      parapatric lineages is also assessed using the data from ecological niche
      modeling. Phylogenetic analyses of portions of the mitochondrial genome reveal
      morphologically cryptic mitochondrial lineages in this species. In addition, we
      find that patterns of genetic divergence are strongly associated with divergence 
      in the ecological niche. Our work demonstrates the ease and utility of using
      spatial analyses of environmental data and phylogenetics in species delimitation,
      especially for groups displaying fine-scaled endemism and cryptic species.
FAU - Rissler, Leslie J
AU  - Rissler LJ
AD  - Department of Biological Sciences, University of Alabama, Tuscaloosa, AL 35487,
      USA. rissler@bama.ua.edu
FAU - Apodaca, Joseph J
AU  - Apodaca JJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - California
MH  - Demography
MH  - *Ecosystem
MH  - Models, Biological
MH  - *Phylogeny
MH  - Urodela/*classification/*genetics
EDAT- 2007/12/11 09:00
MHDA- 2008/02/22 09:00
CRDT- 2007/12/11 09:00
PHST- 2007/12/11 09:00 [pubmed]
PHST- 2008/02/22 09:00 [medline]
PHST- 2007/12/11 09:00 [entrez]
AID - 788068253 [pii]
AID - 10.1080/10635150701703063 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Dec;56(6):924-42. doi: 10.1080/10635150701703063.

PMID- 18066927
OWN - NLM
STAT- MEDLINE
DCOM- 20080221
LR  - 20071210
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 6
DP  - 2007 Dec
TI  - Applications of ecological niche modeling for species delimitation: a review and 
      empirical evaluation using day geckos (Phelsuma) from Madagascar.
PG  - 907-23
AB  - Although the systematic utility of ecological niche modeling is generally well
      known (e.g., concerning the recognition and discovery of areas of endemism for
      biogeographic analyses), there has been little discussion of applications
      concerning species delimitation, and to date, no empirical evaluation has been
      conducted. However, ecological niche modeling can provide compelling evidence for
      allopatry between populations, and can also detect divergent ecological niches
      between candidate species. Here we present results for two taxonomically
      problematic groups of Phelsuma day geckos from Madagascar, where we integrate
      ecological niche modeling with mitochondrial DNA and morphological data to
      evaluate species limits. Despite relatively modest levels of genetic and
      morphological divergence, for both species groups we find divergent ecological
      niches between closely related species and parapatric ecological niche models.
      Niche models based on the new species limits provide a better fit to the known
      distribution than models based upon the combined (lumped) species limits. Based
      on these results, we elevate three subspecies of Phelsuma madagascariensis to
      species rank and describe a new species of Phelsuma from the P. dubia species
      group. Our phylogeny continues to support a major endemic radiation of Phelsuma
      in Madagascar, with dispersals to Pemba Island and the Mascarene Islands. We
      conclude that ecological niche modeling offers great potential for species
      delimitation, especially for taxonomic groups exhibiting low vagility and
      localized endemism and for groups with more poorly known distributions. In
      particular, niche modeling should be especially sensitive for detecting recent
      parapatric speciation driven by ecological divergence, when the environmental
      gradients driving speciation are represented within the ecological niche models.
FAU - Raxworthy, Christopher J
AU  - Raxworthy CJ
AD  - Department of Herpetology, American Museum of Natural History, New York, NY
      10024-5192, USA. rax@amnh.org
FAU - Ingram, Colleen M
AU  - Ingram CM
FAU - Rabibisoa, Nirhy
AU  - Rabibisoa N
FAU - Pearson, Richard G
AU  - Pearson RG
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Ecosystem
MH  - Lizards/*classification/*genetics
MH  - Madagascar
MH  - *Models, Biological
MH  - Phylogeny
EDAT- 2007/12/11 09:00
MHDA- 2008/02/22 09:00
CRDT- 2007/12/11 09:00
PHST- 2007/12/11 09:00 [pubmed]
PHST- 2008/02/22 09:00 [medline]
PHST- 2007/12/11 09:00 [entrez]
AID - 788067779 [pii]
AID - 10.1080/10635150701775111 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Dec;56(6):907-23. doi: 10.1080/10635150701775111.

PMID- 18066926
OWN - NLM
STAT- MEDLINE
DCOM- 20080221
LR  - 20071210
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 6
DP  - 2007 Dec
TI  - Delimiting species in recent radiations.
PG  - 896-906
AB  - Despite considerable effort from the systematics community, delimiting species
      boundaries in recent radiations remains a daunting challenge. We argue that
      genealogical approaches, although sometimes useful, may not solve this important 
      problem, because recently derived species often have not had sufficient time to
      achieve monophyly. Instead, we suggest that population genetic approaches that
      rely on large sets of informative markers like single nucleotide polymorphisms
      (SNPs) provide an alternative framework for delimiting very recently derived
      species. We address two major challenges in applying such markers to species
      delimitation: discovering markers in nonmodel systems and using them to delimit
      recently derived species. Using turtles as a test case, we explore the utility of
      a single, relatively low-coverage genomic resource as an aid in gene and marker
      discovery. We exploit an end-sequenced bacterial artificial chromosome (BAC)
      library from an individual painted turtle (Chrysemys picta) and outline a novel
      protocol that efficiently identifies primer pairs that amplify homologous
      sequences across the tree of living turtles. Preliminary data using this library 
      to discover SNPs in Emydura macquarii, a species that diverged from C. picta
      approximately 210 million years ago, indicate that sequences identified from the 
      Chrysemys BAC library provide useful SNPs even in this very distantly related
      taxon. Several recent methods in wide use in the population genetics literature
      allow one to discover potential species, or test existing species hypotheses,
      with SNP data and may be particularly informative for very recently derived
      species. As BAC and other genomic resources become increasingly available for
      scattered taxa across the tree of life, we are optimistic that these resources
      will provide abundant, inexpensive markers that will help delimit boundaries in
      problematic, recent species radiations.
FAU - Shaffer, H Bradley
AU  - Shaffer HB
AD  - Section of Evolution and Ecology, and Center for Population Biology, University
      of California, Davis, CA 95616, USA. hbshaffer@ucdavis.edu
FAU - Thomson, Robert C
AU  - Thomson RC
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
SB  - IM
MH  - Adaptation, Physiological
MH  - Animals
MH  - Cichlids
MH  - Classification
MH  - Finches
MH  - Genetic Markers
MH  - *Genetic Speciation
MH  - Time Factors
MH  - Turtles
MH  - Urodela
EDAT- 2007/12/11 09:00
MHDA- 2008/02/22 09:00
CRDT- 2007/12/11 09:00
PHST- 2007/12/11 09:00 [pubmed]
PHST- 2008/02/22 09:00 [medline]
PHST- 2007/12/11 09:00 [entrez]
AID - 788071281 [pii]
AID - 10.1080/10635150701772563 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Dec;56(6):896-906. doi: 10.1080/10635150701772563.

PMID- 18027282
OWN - NLM
STAT- MEDLINE
DCOM- 20080221
LR  - 20071120
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 6
DP  - 2007 Dec
TI  - Delimiting species without monophyletic gene trees.
PG  - 887-95
AB  - Genetic data are frequently used to delimit species, where species status is
      determined on the basis of an exclusivity criterium, such as reciprocal
      monophyly. Not only are there numerous empirical examples of incongruence between
      the boundaries inferred from such data compared to other sources like morphology 
      -- especially with recently derived species, but population genetic theory also
      clearly shows that an inevitable bias in species status results because genetic
      thresholds do not explicitly take into account how the timing of speciation
      influences patterns of genetic differentiation. This study represents a
      fundamental shift in how genetic data might be used to delimit species. Rather
      than equating gene trees with a species tree or basing species status on some
      genetic threshold, the relationship between the gene trees and the species
      history is modeled probabilistically. Here we show that the same theory that is
      used to calculate the probability of reciprocal monophyly can also be used to
      delimit species despite widespread incomplete lineage sorting. The results from a
      preliminary simulation study suggest that very recently derived species can be
      accurately identified long before the requisite time for reciprocal monophyly to 
      be achieved following speciation. The study also indicates the importance of
      sampling, both with regards to loci and individuals. Withstanding a thorough
      investigation into the conditions under which the coalescent-based approach will 
      be effective, namely how the timing of divergence relative to the effective
      population size of species affects accurate species delimitation, the results are
      nevertheless consistent with other recent studies (aimed at inferring species
      relationships), showing that despite the lack of monophyletic gene trees, a
      signal of species divergence persists and can be extracted. Using an explicit
      model-based approach also avoids two primary problems with species delimitation
      that result when genetic thresholds are applied with genetic data -- the inherent
      biases in species detection arising from when and how speciation occurred, and
      failure to take into account the high stochastic variance of genetic processes.
      Both the utility and sensitivities of the coalescent-based approach outlined here
      are discussed; most notably, a model-based approach is essential for determining 
      whether incompletely sorted gene lineages are (or are not) consistent with
      separate species lineages, and such inferences require accurate model
      parameterization (i.e., a range of realistic effective population sizes relative 
      to potential times of divergence for the purported species). It is the goal (and 
      motivation of this study) that genetic data might be used effectively as a source
      of complementation to other sources of data for diagnosing species, as opposed to
      the exclusion of other evidence for species delimitation, which will require an
      explicit consideration of the effects of the temporal dynamic of lineage
      splitting on genetic data.
FAU - Knowles, L Lacey
AU  - Knowles LL
AD  - Department of Ecology and Evolutionary Biology, Museum of Zoology, University of 
      Michigan, Ann Arbor, MI 48109-1079, USA. knowlesl@umich.edu
FAU - Carstens, Bryan C
AU  - Carstens BC
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Genetic Speciation
MH  - Genetics, Population
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2007/11/21 09:00
MHDA- 2008/02/22 09:00
CRDT- 2007/11/21 09:00
PHST- 2007/11/21 09:00 [pubmed]
PHST- 2008/02/22 09:00 [medline]
PHST- 2007/11/21 09:00 [entrez]
AID - 785835042 [pii]
AID - 10.1080/10635150701701091 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Dec;56(6):887-95. doi: 10.1080/10635150701701091.

PMID- 18027281
OWN - NLM
STAT- MEDLINE
DCOM- 20080221
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 6
DP  - 2007 Dec
TI  - Species concepts and species delimitation.
PG  - 879-86
AB  - The issue of species delimitation has long been confused with that of species
      conceptualization, leading to a half century of controversy concerning both the
      definition of the species category and methods for inferring the boundaries and
      numbers of species. Alternative species concepts agree in treating existence as a
      separately evolving metapopulation lineage as the primary defining property of
      the species category, but they disagree in adopting different properties acquired
      by lineages during the course of divergence (e.g., intrinsic reproductive
      isolation, diagnosability, monophyly) as secondary defining properties (secondary
      species criteria). A unified species concept can be achieved by treating
      existence as a separately evolving metapopulation lineage as the only necessary
      property of species and the former secondary species criteria as different lines 
      of evidence (operational criteria) relevant to assessing lineage separation. This
      unified concept of species has several consequences for species delimitation,
      including the following: First, the issues of species conceptualization and
      species delimitation are clearly separated; the former secondary species criteria
      are no longer considered relevant to species conceptualization but only to
      species delimitation. Second, all of the properties formerly treated as secondary
      species criteria are relevant to species delimitation to the extent that they
      provide evidence of lineage separation. Third, the presence of any one of the
      properties (if appropriately interpreted) is evidence for the existence of a
      species, though more properties and thus more lines of evidence are associated
      with a higher degree of corroboration. Fourth, and perhaps most significantly, a 
      unified species concept shifts emphasis away from the traditional species
      criteria, encouraging biologists to develop new methods of species delimitation
      that are not tied to those properties.
FAU - De Queiroz, Kevin
AU  - De Queiroz K
AD  - Department of Vertebrate Zoology, National Museum of Natural History, Smithsonian
      Institution, Washington, DC 20560-0162, USA. dequeirozk@si.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - J Hered. 2017 May 1;108(3):334-336. PMID: 28391307
MH  - *Biodiversity
MH  - Biological Evolution
MH  - *Classification
MH  - Demography
MH  - Models, Biological
EDAT- 2007/11/21 09:00
MHDA- 2008/02/22 09:00
CRDT- 2007/11/21 09:00
PHST- 2007/11/21 09:00 [pubmed]
PHST- 2008/02/22 09:00 [medline]
PHST- 2007/11/21 09:00 [entrez]
AID - 785834893 [pii]
AID - 10.1080/10635150701701083 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Dec;56(6):879-86. doi: 10.1080/10635150701701083.

PMID- 18027280
OWN - NLM
STAT- MEDLINE
DCOM- 20080221
LR  - 20071120
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 6
DP  - 2007 Dec
TI  - Species delimitation: new approaches for discovering diversity.
PG  - 875-8
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Department of Ecology and Evolution, Stony Brook University, Stony Brook, NY
      11794-5245, USA. wiensj@life.bio.sunysb.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biodiversity
MH  - *Classification
MH  - Demography
MH  - Species Specificity
EDAT- 2007/11/21 09:00
MHDA- 2008/02/22 09:00
CRDT- 2007/11/21 09:00
PHST- 2007/11/21 09:00 [pubmed]
PHST- 2008/02/22 09:00 [medline]
PHST- 2007/11/21 09:00 [entrez]
AID - 785834982 [pii]
AID - 10.1080/10635150701748506 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Dec;56(6):875-8. doi: 10.1080/10635150701748506.

PMID- 17957581
OWN - NLM
STAT- MEDLINE
DCOM- 20080108
LR  - 20071024
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 5
DP  - 2007 Oct
TI  - The drowning of New Zealand and the problem of Agathis.
PG  - 862-70
FAU - Knapp, Michael
AU  - Knapp M
AD  - Institute of Molecular BioSciences, Allan Wilson Centre, Massey University.
      michael_knapp@eva.mpg.de
FAU - Mudaliar, Ragini
AU  - Mudaliar R
FAU - Havell, David
AU  - Havell D
FAU - Wagstaff, Steven J
AU  - Wagstaff SJ
FAU - Lockhart, Peter J
AU  - Lockhart PJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
SB  - IM
MH  - Animals
MH  - Coniferophyta/*classification/genetics
MH  - DNA, Chloroplast/chemistry
MH  - *Environment
MH  - Evolution, Molecular
MH  - Genes, Plant
MH  - *Geography
MH  - New Zealand
MH  - *Phylogeny
EDAT- 2007/10/25 09:00
MHDA- 2008/01/09 09:00
CRDT- 2007/10/25 09:00
PHST- 2007/10/25 09:00 [pubmed]
PHST- 2008/01/09 09:00 [medline]
PHST- 2007/10/25 09:00 [entrez]
AID - 783409491 [pii]
AID - 10.1080/10635150701636412 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Oct;56(5):862-70. doi: 10.1080/10635150701636412.

PMID- 17957580
OWN - NLM
STAT- MEDLINE
DCOM- 20080108
LR  - 20071024
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 5
DP  - 2007 Oct
TI  - Haplotype networks can be misleading in the presence of missing data.
PG  - 857-62
FAU - Joly, Simon
AU  - Joly S
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, Massey University, New
      Zealand. s.joly@massey.ac.nz
FAU - Stevens, Mark I
AU  - Stevens MI
FAU - van Vuuren, Bettine Jansen
AU  - van Vuuren BJ
LA  - eng
PT  - Comparative Study
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - *Haplotypes
MH  - Phylogeny
MH  - *Software
EDAT- 2007/10/25 09:00
MHDA- 2008/01/09 09:00
CRDT- 2007/10/25 09:00
PHST- 2007/10/25 09:00 [pubmed]
PHST- 2008/01/09 09:00 [medline]
PHST- 2007/10/25 09:00 [entrez]
AID - 783409574 [pii]
AID - 10.1080/10635150701633153 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Oct;56(5):857-62. doi: 10.1080/10635150701633153.

PMID- 17934998
OWN - NLM
STAT- MEDLINE
DCOM- 20080108
LR  - 20071015
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 5
DP  - 2007 Oct
TI  - Phylogenetic systematics and biogeography of hummingbirds: Bayesian and maximum
      likelihood analyses of partitioned data and selection of an appropriate
      partitioning strategy.
PG  - 837-56
AB  - Hummingbirds are an important model system in avian biology, but to date the
      group has been the subject of remarkably few phylogenetic investigations. Here we
      present partitioned Bayesian and maximum likelihood phylogenetic analyses for 151
      of approximately 330 species of hummingbirds and 12 outgroup taxa based on two
      protein-coding mitochondrial genes (ND2 and ND4), flanking tRNAs, and two nuclear
      introns (AK1 and BFib). We analyzed these data under several partitioning
      strategies ranging between unpartitioned and a maximum of nine partitions. In
      order to select a statistically justified partitioning strategy following
      partitioned Bayesian analysis, we considered four alternative criteria including 
      Bayes factors, modified versions of the Akaike information criterion for small
      sample sizes (AIC(c)), Bayesian information criterion (BIC), and a
      decision-theoretic methodology (DT). Following partitioned maximum likelihood
      analyses, we selected a best-fitting strategy using hierarchical likelihood ratio
      tests (hLRTS), the conventional AICc, BIC, and DT, concluding that the most
      stringent criterion, the performance-based DT, was the most appropriate
      methodology for selecting amongst partitioning strategies. In the context of our 
      well-resolved and well-supported phylogenetic estimate, we consider the
      historical biogeography of hummingbirds using ancestral state reconstructions of 
      (1) primary geographic region of occurrence (i.e., South America, Central
      America, North America, Greater Antilles, Lesser Antilles), (2) Andean or
      non-Andean geographic distribution, and (3) minimum elevational occurrence. These
      analyses indicate that the basal hummingbird assemblages originated in the
      lowlands of South America, that most of the principle clades of hummingbirds (all
      but Mountain Gems and possibly Bees) originated on this continent, and that there
      have been many (at least 30) independent invasions of other primary landmasses,
      especially Central America.
FAU - McGuire, Jimmy A
AU  - McGuire JA
AD  - Museum of Vertebrate Zoology, 3101 Valley Life Sciences Building, University of
      California, Berkeley, CA 94720-3160, USA. mcguirej@berkeley.edu
FAU - Witt, Christopher C
AU  - Witt CC
FAU - Altshuler, Douglas L
AU  - Altshuler DL
FAU - Remsen, J V Jr
AU  - Remsen JV Jr
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Birds/*classification/genetics
MH  - Classification/methods
MH  - *Geography
MH  - Likelihood Functions
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2007/10/16 09:00
MHDA- 2008/01/09 09:00
CRDT- 2007/10/16 09:00
PHST- 2007/10/16 09:00 [pubmed]
PHST- 2008/01/09 09:00 [medline]
PHST- 2007/10/16 09:00 [entrez]
AID - 782955718 [pii]
AID - 10.1080/10635150701656360 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Oct;56(5):837-56. doi: 10.1080/10635150701656360.

PMID- 17934997
OWN - NLM
STAT- MEDLINE
DCOM- 20080108
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 5
DP  - 2007 Oct
TI  - The conceptual and statistical relationship between modularity and morphological 
      integration.
PG  - 818-36
AB  - The modular variation of organismal form during evolution or development can be
      viewed as the result of dissociated local developmental processes. Whereas
      studies of modularity often are experimental, morphological integration is a more
      descriptive concept applying to groups of correlated phenotypic characters. Using
      simple path models, this paper shows that the classic underlying assumption of
      modularity (high correlations within modules, lower correlations between modules)
      is met only when local developmental factors have high effects on the traits
      relative to all factors' variations of effect (i.e., allometry). Accordingly,
      many classic approaches to morphological integration are meaningful only when
      local as well as common growth factors are nearly isometric. We show that
      morphometric modules might instead be defined more generally as sets of variables
      with non-zero within-module covariances, even when the covariances due to common 
      factors have been removed, so that the residual between-module covariances are
      all near zero. Although it is still inherently unreliable to identify modules
      from phenotypic covariances in nonexperimental data, patterns of integration can 
      sometimes be estimated based on prior identification of the modules themselves.
      We outline a simple approach for this case using Wright-style factor analysis and
      demonstrate the relation of its algebra to the more familiar approach via partial
      least squares.
FAU - Mitteroecker, Philipp
AU  - Mitteroecker P
AD  - Department of Anthropology, University of Vienna, Vienna, Austria.
      philipp.mitteroecker@univie.ac.at
FAU - Bookstein, Fred
AU  - Bookstein F
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2009 Feb;58(1):146-9. PMID: 20525574
MH  - *Biological Evolution
MH  - Classification/methods
MH  - Cluster Analysis
MH  - Least-Squares Analysis
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2007/10/16 09:00
MHDA- 2008/01/09 09:00
CRDT- 2007/10/16 09:00
PHST- 2007/10/16 09:00 [pubmed]
PHST- 2008/01/09 09:00 [medline]
PHST- 2007/10/16 09:00 [entrez]
AID - 782955580 [pii]
AID - 10.1080/10635150701648029 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Oct;56(5):818-36. doi: 10.1080/10635150701648029.

PMID- 17918032
OWN - NLM
STAT- MEDLINE
DCOM- 20080108
LR  - 20071005
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 5
DP  - 2007 Oct
TI  - PhySIC: a veto supertree method with desirable properties.
PG  - 798-817
AB  - This paper focuses on veto supertree methods; i.e., methods that aim at producing
      a conservative synthesis of the relationships agreed upon by all source trees. We
      propose desirable properties that a supertree should satisfy in this framework,
      namely the non-contradiction property (PC) and the induction property (PI). The
      former requires that the supertree does not contain relationships that contradict
      one or a combination of the source topologies, whereas the latter requires that
      all topological information contained in the supertree is present in a source
      tree or collectively induced by several source trees. We provide simple examples 
      to illustrate their relevance and that allow a comparison with previously
      advocated properties. We show that these properties can be checked in polynomial 
      time for any given rooted supertree. Moreover, we introduce the PhySIC method
      (PHYlogenetic Signal with Induction and non-Contradiction). For k input trees
      spanning a set of n taxa, this method produces a supertree that satisfies the
      above-mentioned properties in O(kn(3) + n(4)) computing time. The polytomies of
      the produced supertree are also tagged by labels indicating areas of conflict as 
      well as those with insufficient overlap. As a whole, PhySIC enables the user to
      quickly summarize consensual information of a set of trees and localize groups of
      taxa for which the data require consolidation. Lastly, we illustrate the
      behaviour of PhySIC on primate data sets of various sizes, and propose a
      supertree covering 95% of all primate extant genera. The PhySIC algorithm is
      available at http://atgc.lirmm.fr/cgi-bin/PhySIC.
FAU - Ranwez, Vincent
AU  - Ranwez V
AD  - Institut des Sciences de l'Evolution (ISEM, UMR 5554 CNRS), Universite
      Montpellier II, Montpellier, Cedex 5, France. ranwez@isem.univ-montp2.fr
FAU - Berry, Vincent
AU  - Berry V
FAU - Criscuolo, Alexis
AU  - Criscuolo A
FAU - Fabre, Pierre-Henri
AU  - Fabre PH
FAU - Guillemot, Sylvain
AU  - Guillemot S
FAU - Scornavacca, Celine
AU  - Scornavacca C
FAU - Douzery, Emmanuel J P
AU  - Douzery EJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Classification/methods
MH  - Models, Biological
MH  - *Phylogeny
MH  - Primates/classification
MH  - *Software
EDAT- 2007/10/06 09:00
MHDA- 2008/01/09 09:00
CRDT- 2007/10/06 09:00
PHST- 2007/10/06 09:00 [pubmed]
PHST- 2008/01/09 09:00 [medline]
PHST- 2007/10/06 09:00 [entrez]
AID - 782748826 [pii]
AID - 10.1080/10635150701639754 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Oct;56(5):798-817. doi: 10.1080/10635150701639754.

PMID- 17907054
OWN - NLM
STAT- MEDLINE
DCOM- 20080108
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 5
DP  - 2007 Oct
TI  - Molecular phylogenetics of the lizard genus Microlophus (squamata:tropiduridae): 
      aligning and retrieving indel signal from nuclear introns.
PG  - 776-97
AB  - We use a multigene data set (the mitochondrial locus and nine nuclear gene
      regions) to test phylogenetic relationships in the South American "lava lizards" 
      (genus Microlophus) and describe a strategy for aligning noncoding sequences that
      accounts for differences in tempo and class of mutational events. We focus on
      seven nuclear introns that vary in size and frequency of multibase length
      mutations (i.e., indels) and present a manual alignment strategy that
      incorporates insertions and deletions (indels) for each intron. Our method is
      based on mechanistic explanations of intron evolution that does not require a
      guide tree. We also use a progressive alignment algorithm (Probabilistic
      Alignment Kit; PRANK) and distinguishes insertions from deletions and avoids the 
      "gapcost" conundrum. We describe an approach to selecting a guide tree purged of 
      ambiguously aligned regions and use this to refine PRANK performance. We show
      that although manual alignment is successful in finding repeat motifs and the
      most obvious indels, some regions can only be subjectively aligned, and there are
      limits to the size and complexity of a data matrix for which this approach can be
      taken. PRANK alignments identified more parsimony-informative indels while
      simultaneously increasing nucleotide identity in conserved sequence blocks
      flanking the indel regions. When comparing manual and PRANK with two widely used 
      methods (CLUSTAL, MUSCLE) for the alignment of the most length-variable intron,
      only PRANK recovered a tree congruent at deeper nodes with the combined data tree
      inferred from all nuclear gene regions. We take this concordance as an objective 
      function of alignment quality and present a strongly supported phylogenetic
      hypothesis for Microlophus relationships. From this hypothesis we show that (1) a
      coded indel data partition derived from the PRANK alignment contributed
      significantly to nodal support and (2) the indel data set permitted detection of 
      significant conflict between mitochondrial and nuclear data partitions, which we 
      hypothesize arose from secondary contact of distantly related taxa, followed by
      hybridization and mtDNA introgression.
FAU - Benavides, Edgar
AU  - Benavides E
AD  - Department of Integrative Biology, Brigham Young University, Provo, UT, USA.
      eb235@email.byu.edu
FAU - Baum, Rebecca
AU  - Baum R
FAU - McClellan, David
AU  - McClellan D
FAU - Sites, Jack W Jr
AU  - Sites JW Jr
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Ecuador
MH  - Evolution, Molecular
MH  - Genetic Variation
MH  - *Introns
MH  - Lizards/*classification/genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
MH  - Software
MH  - South America
EDAT- 2007/10/02 09:00
MHDA- 2008/01/09 09:00
CRDT- 2007/10/02 09:00
PHST- 2007/10/02 09:00 [pubmed]
PHST- 2008/01/09 09:00 [medline]
PHST- 2007/10/02 09:00 [entrez]
AID - 782595757 [pii]
AID - 10.1080/10635150701618527 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Oct;56(5):776-97. doi: 10.1080/10635150701618527.

PMID- 17886146
OWN - NLM
STAT- MEDLINE
DCOM- 20080108
LR  - 20070921
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 5
DP  - 2007 Oct
TI  - Phylogenetic mixtures on a single tree can mimic a tree of another topology.
PG  - 767-75
AB  - Phylogenetic mixtures model the inhomogeneous molecular evolution commonly
      observed in data. The performance of phylogenetic reconstruction methods where
      the underlying data are generated by a mixture model has stimulated considerable 
      recent debate. Much of the controversy stems from simulations of mixture model
      data on a given tree topology for which reconstruction algorithms output a tree
      of a different topology; these findings were held up to show the shortcomings of 
      particular tree reconstruction methods. In so doing, the underlying assumption
      was that mixture model data on one topology can be distinguished from data
      evolved on an unmixed tree of another topology given enough data and the
      "correct" method. Here we show that this assumption can be false. For biologists,
      our results imply that, for example, the combined data from two genes whose
      phylogenetic trees differ only in terms of branch lengths can perfectly fit a
      tree of a different topology.
FAU - Matsen, Frederick A
AU  - Matsen FA
AD  - Biomathematics Research Centre, University of Canterbury, Christchurch, New
      Zealand. ematsen@gmail.com
FAU - Steel, Mike
AU  - Steel M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computational Biology/methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2007/09/22 09:00
MHDA- 2008/01/09 09:00
CRDT- 2007/09/22 09:00
PHST- 2007/09/22 09:00 [pubmed]
PHST- 2008/01/09 09:00 [medline]
PHST- 2007/09/22 09:00 [entrez]
AID - 782130791 [pii]
AID - 10.1080/10635150701627304 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Oct;56(5):767-75. doi: 10.1080/10635150701627304.

PMID- 17886145
OWN - NLM
STAT- MEDLINE
DCOM- 20080108
LR  - 20170922
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 5
DP  - 2007 Oct
TI  - Fossils impact as hard as living taxa in parsimony analyses of morphology.
PG  - 753-66
AB  - Systematists disagree whether data from fossils should be included in parsimony
      analyses. In a handful of well-documented cases, the addition of fossil data
      radically overturns a hypothesis of relationships based on extant taxa alone.
      Fossils can break up long branches and preserve character combinations closer in 
      time to deep splitting events. However, fossils usually require more
      interpretation than extant taxa, introducing greater potential for spurious
      codings. Moreover, because fossils often have more "missing" codings, they are
      frequently accused of increasing numbers of MPTs, frustrating resolution and
      reducing support. Despite the controversy, remarkably little is known about the
      effects of fossils more generally. Here we provide the first systematic study,
      investigating empirically the behavior of fossil and extant taxa in 45 published 
      morphological data sets. First-order jackknifing is used to determine the effects
      that each terminal has on inferred relationships, on the number of MPTs, and on
      CI' and RI as measures of homoplasy. Bootstrap leaf stabilities provide a proxy
      for the contribution of individual taxa to the branch support in the rest of the 
      tree. There is no significant difference in the impact of fossil versus extant
      taxa on relationships, numbers of MPTs, and CI' or RI. However, adding individual
      fossil taxa is more likely to reduce the total branch support of the tree than
      adding extant taxa. This must be weighed against the superior taxon sampling
      afforded by including judiciously coded fossils, providing data from otherwise
      unsampled regions of the tree. We therefore recommend that investigators should
      include fossils, in the absence of compelling and case specific reasons for their
      exclusion.
FAU - Cobbett, Andrea
AU  - Cobbett A
AD  - Department of Biology and Biochemistry, The University of Bath, Claverton Down,
      Bath, UK.
FAU - Wilkinson, Mark
AU  - Wilkinson M
FAU - Wills, Matthew A
AU  - Wills MA
LA  - eng
GR  - BB/C006682/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Comparative Study
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/methods
MH  - Extinction, Biological
MH  - *Fossils
MH  - *Phylogeny
EDAT- 2007/09/22 09:00
MHDA- 2008/01/09 09:00
CRDT- 2007/09/22 09:00
PHST- 2007/09/22 09:00 [pubmed]
PHST- 2008/01/09 09:00 [medline]
PHST- 2007/09/22 09:00 [entrez]
AID - 782130273 [pii]
AID - 10.1080/10635150701627296 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Oct;56(5):753-66. doi: 10.1080/10635150701627296.

PMID- 17886144
OWN - NLM
STAT- MEDLINE
DCOM- 20080108
LR  - 20070921
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 5
DP  - 2007 Oct
TI  - Estimating divergence times in large phylogenetic trees.
PG  - 741-52
AB  - A new method, PATHd8, for estimating ultrametric trees from trees with edge
      (branch) lengths proportional to the number of substitutions is proposed. The
      method allows for an arbitrary number of reference nodes for time calibration,
      each defined either as absolute age, minimum age, or maximum age, and the tree
      need not be fully resolved. The method is based on estimating node ages by mean
      path lengths from the node to the leaves but correcting for deviations from a
      molecular clock suggested by reference nodes. As opposed to most existing methods
      allowing substitution rate variation, the new method smoothes substitution rates 
      locally, rather than simultaneously over the whole tree, thus allowing for
      analysis of very large trees. The performance of PATHd8 is compared with other
      frequently used methods for estimating divergence times. In analyses of three
      separate data sets, PATHd8 gives similar divergence times to other methods, the
      largest difference being between crown group ages, where unconstrained nodes get 
      younger ages when analyzed with PATHd8. Overall, chronograms obtained from other 
      methods appear smoother, whereas PATHd8 preserves more of the heterogeneity seen 
      in the original edge lengths. Divergence times are most evenly spread over the
      chronograms obtained from the Bayesian implementation and the clock-based
      Langley-Fitch method, and these two methods produce very similar ages for most
      nodes. Evaluations of PATHd8 using simulated data suggest that PATHd8 is slightly
      less precise compared with penalized likelihood, but it gives more sensible
      answers for extreme data sets. A clear advantage with PATHd8 is that it is more
      or less instantaneous even with trees having several thousand leaves, whereas
      other programs often run into problems when analyzing trees with hundreds of
      leaves. PATHd8 is implemented in freely available software.
FAU - Britton, Tom
AU  - Britton T
AD  - Department of Mathematics, Stockholm University, Stockholm, Sweden.
      tom.britton@math.su.se
FAU - Anderson, Cajsa Lisa
AU  - Anderson CL
FAU - Jacquet, David
AU  - Jacquet D
FAU - Lundqvist, Samuel
AU  - Lundqvist S
FAU - Bremer, Kare
AU  - Bremer K
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - Fossils
MH  - Likelihood Functions
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Plants/classification
MH  - Software
EDAT- 2007/09/22 09:00
MHDA- 2008/01/09 09:00
CRDT- 2007/09/22 09:00
PHST- 2007/09/22 09:00 [pubmed]
PHST- 2008/01/09 09:00 [medline]
PHST- 2007/09/22 09:00 [entrez]
AID - 782130970 [pii]
AID - 10.1080/10635150701613783 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Oct;56(5):741-52. doi: 10.1080/10635150701613783.

PMID- 17849327
OWN - NLM
STAT- MEDLINE
DCOM- 20080108
LR  - 20070912
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 5
DP  - 2007 Oct
TI  - New approaches to phylogenetic tree search and their application to large numbers
      of protein alignments.
PG  - 727-40
AB  - Phylogenetic tree estimation plays a critical role in a wide variety of molecular
      studies, including molecular systematics, phylogenetics, and comparative
      genomics. Finding the optimal tree relating a set of sequences using score-based 
      (optimality criterion) methods, such as maximum likelihood and maximum parsimony,
      may require all possible trees to be considered, which is not feasible even for
      modest numbers of sequences. In practice, trees are estimated using heuristics
      that represent a trade-off between topological accuracy and speed. I present a
      series of novel algorithms suitable for score-based phylogenetic tree
      reconstruction that demonstrably improve the accuracy of tree estimates while
      maintaining high computational speeds. The heuristics function by allowing the
      efficient exploration of large numbers of trees through novel hill-climbing and
      resampling strategies. These heuristics, and other computational approximations, 
      are implemented for maximum likelihood estimation of trees in the program Leaphy,
      and its performance is compared to other popular phylogenetic programs. Trees are
      estimated from 4059 different protein alignments using a selection of
      phylogenetic programs and the likelihoods of the tree estimates are compared.
      Trees estimated using Leaphy are found to have equal to or better likelihoods
      than trees estimated using other phylogenetic programs in 4004 (98.6%) families
      and provide a unique best tree that no other program found in 1102 (27.1%)
      families. The improvement is particularly marked for larger families (80 to 100
      sequences), where Leaphy finds a unique best tree in 81.7% of families.
FAU - Whelan, Simon
AU  - Whelan S
AD  - Faculty of Life Sciences, Michael Smith Building, University of Manchester,
      Oxford Road, Manchester, UK. simon.whelan@manchester.ac.uk
LA  - eng
PT  - Comparative Study
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Proteins)
SB  - IM
MH  - Algorithms
MH  - Amino Acid Sequence
MH  - Computational Biology
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - Proteins/chemistry/*classification
MH  - *Sequence Alignment
MH  - Sequence Analysis, Protein/*methods
MH  - *Software
EDAT- 2007/09/13 09:00
MHDA- 2008/01/09 09:00
CRDT- 2007/09/13 09:00
PHST- 2007/09/13 09:00 [pubmed]
PHST- 2008/01/09 09:00 [medline]
PHST- 2007/09/13 09:00 [entrez]
AID - 781940145 [pii]
AID - 10.1080/10635150701611134 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Oct;56(5):727-40. doi: 10.1080/10635150701611134.

PMID- 17849326
OWN - NLM
STAT- MEDLINE
DCOM- 20080108
LR  - 20070912
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 5
DP  - 2007 Oct
TI  - Exploring fast computational strategies for probabilistic phylogenetic analysis.
PG  - 711-26
AB  - In recent years, the advent of Markov chain Monte Carlo (MCMC) techniques,
      coupled with modern computational capabilities, has enabled the study of
      evolutionary models without a closed form solution of the likelihood function.
      However, current Bayesian MCMC applications can incur significant computational
      costs, as they are based on a full sampling from the posterior probability
      distribution of the parameters of interest. Here, we draw attention as to how
      MCMC techniques can be embedded within normal approximation strategies for more
      economical statistical computation. The overall procedure is based on an estimate
      of the first and second moments of the likelihood function, as well as a maximum 
      likelihood estimate. Through examples, we review several MCMC-based methods used 
      in the statistical literature for such estimation, applying the approaches to
      constructing posterior distributions under non-analytical evolutionary models
      relaxing the assumptions of rate homogeneity, and of independence between sites. 
      Finally, we use the procedures for conducting Bayesian model selection, based on 
      Laplace approximations of Bayes factors, which we find to be accurate and
      computationally advantageous. Altogether, the methods we expound here, as well as
      other related approaches from the statistical literature, should prove useful
      when investigating increasingly complex descriptions of molecular evolution,
      alleviating some of the difficulties associated with nonanalytical models.
FAU - Rodrigue, Nicolas
AU  - Rodrigue N
AD  - Canadian Institute for Advanced Research, Departement de Biochimie, Universite de
      Montreal, Quebec, Canada. nicolas.rodrigue@umontreal.ca
FAU - Philippe, Herve
AU  - Philippe H
FAU - Lartillot, Nicolas
AU  - Lartillot N
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Myoglobin)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Evolution, Molecular
MH  - Likelihood Functions
MH  - Markov Chains
MH  - *Models, Biological
MH  - Monte Carlo Method
MH  - Myoglobin/chemistry/classification
MH  - *Phylogeny
MH  - Vertebrates/classification
EDAT- 2007/09/13 09:00
MHDA- 2008/01/09 09:00
CRDT- 2007/09/13 09:00
PHST- 2007/09/13 09:00 [pubmed]
PHST- 2008/01/09 09:00 [medline]
PHST- 2007/09/13 09:00 [entrez]
AID - 781937249 [pii]
AID - 10.1080/10635150701611258 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Oct;56(5):711-26. doi: 10.1080/10635150701611258.

PMID- 17849325
OWN - NLM
STAT- MEDLINE
DCOM- 20080108
LR  - 20070912
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 5
DP  - 2007 Oct
TI  - Estimating a binary character's effect on speciation and extinction.
PG  - 701-10
AB  - Determining whether speciation and extinction rates depend on the state of a
      particular character has been of long-standing interest to evolutionary
      biologists. To assess the effect of a character on diversification rates using
      likelihood methods requires that we be able to calculate the probability that a
      group of extant species would have evolved as observed, given a particular model 
      of the character's effect. Here we describe how to calculate this probability for
      a phylogenetic tree and a two-state (binary) character under a simple model of
      evolution (the "BiSSE" model, binary-state speciation and extinction). The model 
      involves six parameters, specifying two speciation rates (rate when the lineage
      is in state 0; rate when in state 1), two extinction rates (when in state 0; when
      in state 1), and two rates of character state change (from 0 to 1, and from 1 to 
      0). Using these probability calculations, we can do maximum likelihood inference 
      to estimate the model's parameters and perform hypothesis tests (e.g., is the
      rate of speciation elevated for one character state over the other?). We
      demonstrate the application of the method using simulated data with known
      parameter values.
FAU - Maddison, Wayne P
AU  - Maddison WP
AD  - Department of Zoology, University of British Columbia, Vancouver, Canada.
      wmaddisn@interchange.ubc.ca
FAU - Midford, Peter E
AU  - Midford PE
FAU - Otto, Sarah P
AU  - Otto SP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - *Extinction, Biological
MH  - *Genetic Speciation
MH  - Likelihood Functions
MH  - *Models, Biological
MH  - Phylogeny
EDAT- 2007/09/13 09:00
MHDA- 2008/01/09 09:00
CRDT- 2007/09/13 09:00
PHST- 2007/09/13 09:00 [pubmed]
PHST- 2008/01/09 09:00 [medline]
PHST- 2007/09/13 09:00 [entrez]
AID - 781940433 [pii]
AID - 10.1080/10635150701607033 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Oct;56(5):701-10. doi: 10.1080/10635150701607033.

PMID- 17661235
OWN - NLM
STAT- MEDLINE
DCOM- 20071228
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 4
DP  - 2007 Aug
TI  - Diversity, nomenclature, and taxonomy of protists.
PG  - 684-9
FAU - Adl, Sina M
AU  - Adl SM
AD  - Department of Biology, Dalhousie University, Halifax, NS, Canada. sadl@dal.ca
FAU - Leander, Brian S
AU  - Leander BS
FAU - Simpson, Alastair G B
AU  - Simpson AG
FAU - Archibald, John M
AU  - Archibald JM
FAU - Anderson, O Roger
AU  - Anderson OR
FAU - Bass, David
AU  - Bass D
FAU - Bowser, Samuel S
AU  - Bowser SS
FAU - Brugerolle, Guy
AU  - Brugerolle G
FAU - Farmer, Mark A
AU  - Farmer MA
FAU - Karpov, Sergey
AU  - Karpov S
FAU - Kolisko, Martin
AU  - Kolisko M
FAU - Lane, Christopher E
AU  - Lane CE
FAU - Lodge, Deborah J
AU  - Lodge DJ
FAU - Mann, David G
AU  - Mann DG
FAU - Meisterfeld, Ralf
AU  - Meisterfeld R
FAU - Mendoza, Leonel
AU  - Mendoza L
FAU - Moestrup, Ojvind
AU  - Moestrup O
FAU - Mozley-Standridge, Sharon E
AU  - Mozley-Standridge SE
FAU - Smirnov, Alexey V
AU  - Smirnov AV
FAU - Spiegel, Frederick
AU  - Spiegel F
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Protozoan)
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - *Classification
MH  - DNA, Protozoan/genetics
MH  - Eukaryota/*classification/*genetics
MH  - *Genetic Variation
EDAT- 2007/07/31 09:00
MHDA- 2007/12/29 09:00
CRDT- 2007/07/31 09:00
PHST- 2007/07/31 09:00 [pubmed]
PHST- 2007/12/29 09:00 [medline]
PHST- 2007/07/31 09:00 [entrez]
AID - 780913334 [pii]
AID - 10.1080/10635150701494127 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Aug;56(4):684-9. doi: 10.1080/10635150701494127.

PMID- 17661234
OWN - NLM
STAT- MEDLINE
DCOM- 20071228
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 4
DP  - 2007 Aug
TI  - The adequacy of morphology for reconstructing the early history of placental
      mammals.
PG  - 673-84
FAU - Springer, Mark S
AU  - Springer MS
AD  - Department of Biology, University of California Riverside, Riverside, CA 92521,
      USA. mark.springer@ucr.edu
FAU - Burk-Herrick, Angela
AU  - Burk-Herrick A
FAU - Meredith, Robert
AU  - Meredith R
FAU - Eizirik, Eduardo
AU  - Eizirik E
FAU - Teeling, Emma
AU  - Teeling E
FAU - O'Brien, Stephen J
AU  - O'Brien SJ
FAU - Murphy, William J
AU  - Murphy WJ
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2008 Jun;57(3):499-503. PMID: 18570041
MH  - Animals
MH  - *Biological Evolution
MH  - Data Interpretation, Statistical
MH  - Extinction, Biological
MH  - *Fossils
MH  - Mammals/*anatomy &amp; histology/*classification/genetics
EDAT- 2007/07/31 09:00
MHDA- 2007/12/29 09:00
CRDT- 2007/07/31 09:00
PHST- 2007/07/31 09:00 [pubmed]
PHST- 2007/12/29 09:00 [medline]
PHST- 2007/07/31 09:00 [entrez]
AID - 780913390 [pii]
AID - 10.1080/10635150701491149 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Aug;56(4):673-84. doi: 10.1080/10635150701491149.

PMID- 17661233
OWN - NLM
STAT- MEDLINE
DCOM- 20071228
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 4
DP  - 2007 Aug
TI  - Mitochondrial and nuclear DNA sequences support a Cretaceous origin of
      Columbiformes and a dispersal-driven radiation in the Paleocene .
PG  - 656-72
AB  - Phylogenetic relationships among genera of pigeons and doves (Aves,
      Columbiformes) have not been fully resolved because of limited sampling of taxa
      and characters in previous studies. We therefore sequenced multiple nuclear and
      mitochondrial DNA genes totaling over 9000 bp from 33 of 41 genera plus 8
      outgroup taxa, and, together with sequences from 5 other pigeon genera retrieved 
      from GenBank, recovered a strong phylogenetic hypothesis for the Columbiformes.
      Three major clades were recovered with the combined data set, comprising the
      basally branching New World pigeons and allies (clade A) that are sister to
      Neotropical ground doves (clade B), and the Afro-Eurasian and Australasian taxa
      (clade C). None of these clades supports the monophyly of current families and
      subfamilies. The extinct, flightless dodo and solitaires (Raphidae) were embedded
      within pigeons and doves (Columbidae) in clade C, and monophyly of the subfamily 
      Columbinae was refuted because the remaining subfamilies were nested within it.
      Divergence times estimated using a Bayesian framework suggest that Columbiformes 
      diverged from outgroups such as Apodiformes and Caprimulgiformes in the
      Cretaceous before the mass extinction that marks the end of this period. Bayesian
      and maximum likelihood inferences of ancestral areas, accounting for phylogenetic
      uncertainty and divergence times, respectively, favor an ancient origin of
      Columbiformes in the Neotropical portion of what was then Gondwana. The radiation
      of modern genera of Columbiformes started in the Early Eocene to the Middle
      Miocene, as previously estimated for other avian groups such as ratites,
      tinamous, galliform birds, penguins, shorebirds, parrots, passerine birds, and
      toucans. Multiple dispersals of more derived Columbiformes between Australasian
      and Afro-Eurasian regions are required to explain current distributions.
FAU - Pereira, Sergio L
AU  - Pereira SL
AD  - Department of Natural History, Royal Ontario Museum, Toronto, Ontario, Canada.
      sergio.pereira@utoronto.ca
FAU - Johnson, Kevin P
AU  - Johnson KP
FAU - Clayton, Dale H
AU  - Clayton DH
FAU - Baker, Allan J
AU  - Baker AJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Cell Nucleus/*genetics
MH  - Columbiformes/*classification/*genetics
MH  - DNA, Mitochondrial/*genetics
MH  - *Evolution, Molecular
MH  - Geological Phenomena
MH  - Geology
MH  - Population Dynamics
MH  - Time Factors
EDAT- 2007/07/31 09:00
MHDA- 2007/12/29 09:00
CRDT- 2007/07/31 09:00
PHST- 2007/07/31 09:00 [pubmed]
PHST- 2007/12/29 09:00 [medline]
PHST- 2007/07/31 09:00 [entrez]
AID - 780913453 [pii]
AID - 10.1080/10635150701549672 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Aug;56(4):656-72. doi: 10.1080/10635150701549672.

PMID- 17661232
OWN - NLM
STAT- MEDLINE
DCOM- 20071228
LR  - 20070730
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 4
DP  - 2007 Aug
TI  - The importance of data partitioning and the utility of Bayes factors in Bayesian 
      phylogenetics.
PG  - 643-55
AB  - As larger, more complex data sets are being used to infer phylogenies, accuracy
      of these phylogenies increasingly requires models of evolution that accommodate
      heterogeneity in the processes of molecular evolution. We investigated the effect
      of improper data partitioning on phylogenetic accuracy, as well as the type I
      error rate and sensitivity of Bayes factors, a commonly used method for choosing 
      among different partitioning strategies in Bayesian analyses. We also used Bayes 
      factors to test empirical data for the need to divide data in a manner that has
      no expected biological meaning. Posterior probability estimates are misleading
      when an incorrect partitioning strategy is assumed. The error was greatest when
      the assumed model was underpartitioned. These results suggest that model
      partitioning is important for large data sets. Bayes factors performed well,
      giving a 5% type I error rate, which is remarkably consistent with standard
      frequentist hypothesis tests. The sensitivity of Bayes factors was found to be
      quite high when the across-class model heterogeneity reflected that of empirical 
      data. These results suggest that Bayes factors represent a robust method of
      choosing among partitioning strategies. Lastly, results of tests for the
      inclusion of unexpected divisions in empirical data mirrored the simulation
      results, although the outcome of such tests is highly dependent on accounting for
      rate variation among classes. We conclude by discussing other approaches for
      partitioning data, as well as other applications of Bayes factors.
FAU - Brown, Jeremy M
AU  - Brown JM
AD  - Section of Integrative Biology, The University of Texas-Austin, 1 University
      Station C0930, Austin, TX 78712, USA. jembrown@mail.utexas.edu
FAU - Lemmon, Alan R
AU  - Lemmon AR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Computer Simulation
MH  - Data Interpretation, Statistical
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Sensitivity and Specificity
EDAT- 2007/07/31 09:00
MHDA- 2007/12/29 09:00
CRDT- 2007/07/31 09:00
PHST- 2007/07/31 09:00 [pubmed]
PHST- 2007/12/29 09:00 [medline]
PHST- 2007/07/31 09:00 [entrez]
AID - 780913274 [pii]
AID - 10.1080/10635150701546249 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Aug;56(4):643-55. doi: 10.1080/10635150701546249.

PMID- 17661231
OWN - NLM
STAT- MEDLINE
DCOM- 20071228
LR  - 20070730
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 4
DP  - 2007 Aug
TI  - A model of horizontal gene transfer and the bacterial phylogeny problem.
PG  - 633-42
AB  - How much horizontal gene transfer (HGT) between species influences bacterial
      phylogenomics is a controversial issue. This debate, however, lacks any
      quantitative assessment of the impact of HGT on phylogenies and of the ability of
      tree-building methods to cope with such events. I introduce a Markov model of
      genome evolution with HGT, accounting for the constraints on time -- an HGT event
      can only occur between concomitantly living species. This model is used to
      simulate multigene sequence data sets with or without HGT. The consequences of
      HGT on phylogenomic inference are analyzed and compared to other well-known
      phylogenetic artefacts. It is found that supertree methods are quite robust to
      HGT, keeping high levels of performance even when gene trees are largely
      incongruent with each other. Gene tree incongruence per se is not indicative of
      HGT. HGT, however, removes the (otherwise observed) positive relationship between
      sequence length and gene tree congruence to the estimated species tree.
      Surprisingly, when applied to a bacterial and a eukaryotic multigene data set,
      this criterion rejects the HGT hypothesis for the former, but not the latter data
      set.
FAU - Galtier, Nicolas
AU  - Galtier N
AD  - Institut des Sciences de l'Evolution (UM2-CNRS), Universite Montpellier 2,
      Montpellier, France. galtier@univ-montp2.fr
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bacteria/*classification/*genetics
MH  - *Gene Transfer, Horizontal
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Stochastic Processes
EDAT- 2007/07/31 09:00
MHDA- 2007/12/29 09:00
CRDT- 2007/07/31 09:00
PHST- 2007/07/31 09:00 [pubmed]
PHST- 2007/12/29 09:00 [medline]
PHST- 2007/07/31 09:00 [entrez]
AID - 780913211 [pii]
AID - 10.1080/10635150701546231 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Aug;56(4):633-42. doi: 10.1080/10635150701546231.

PMID- 17654366
OWN - NLM
STAT- MEDLINE
DCOM- 20071228
LR  - 20070726
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 4
DP  - 2007 Aug
TI  - Spectral partitioning of phylogenetic data sets based on compatibility.
PG  - 623-32
AB  - We describe two new methods to partition phylogenetic data sets of discrete
      characters based on pairwise compatibility. The partitioning methods make no
      assumptions regarding the phylogeny, model of evolution, or characteristics of
      the data. The methods first build a compatibility graph, in which each node
      represents a character in the data set. Edges in the compatibility graph may
      represent strict compatibility of characters or they may be weighted based on a
      fractional compatibility scoring procedure that measures how close the characters
      are to being compatible. Given the desired number of partitions, the partitioning
      methods then seek to cluster the characters with the highest average pairwise
      compatibility, so that characters in each cluster are more compatible with each
      other than they are with characters in the other cluster(s). Partitioning
      according to these criteria is computationally intractable (NP-hard); however,
      spectral methods can quickly provide high-quality solutions. We demonstrate that 
      the spectral partitioning effectively identifies characters with different
      evolutionary histories in simulated data sets, and it is better at highlighting
      phylogenetic conflict within empirical data sets than previously used
      partitioning methods.
FAU - Chen, Duhong
AU  - Chen D
AD  - Department of Computer Science, Iowa State University, Ames, IA 50011, USA.
FAU - Burleigh, Gordon J
AU  - Burleigh GJ
FAU - Fernandez-Baca, David
AU  - Fernandez-Baca D
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Computer Simulation
MH  - *Phylogeny
MH  - *Software
EDAT- 2007/07/27 09:00
MHDA- 2007/12/29 09:00
CRDT- 2007/07/27 09:00
PHST- 2007/07/27 09:00 [pubmed]
PHST- 2007/12/29 09:00 [medline]
PHST- 2007/07/27 09:00 [entrez]
AID - 780835657 [pii]
AID - 10.1080/10635150701499571 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Aug;56(4):623-32. doi: 10.1080/10635150701499571.

PMID- 17654365
OWN - NLM
STAT- MEDLINE
DCOM- 20071228
LR  - 20070726
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 4
DP  - 2007 Aug
TI  - Experimental design criteria in phylogenetics: where to add taxa.
PG  - 609-22
AB  - Accurate phylogenetic inference is a topic of intensive research and debate and
      has been studied in response to many different factors: for example, differences 
      in the method of reconstruction, the shape of the underlying tree, the
      substitution model, and varying quantities and types of data. Investigating
      whether the conditions used might lead to inaccurate inference has been attempted
      through elaborate data exploration but less attention has been given to creating 
      a unified methodology to enable experimental designs in phylogenetic analysis to 
      be improved and so avoid suboptimal conditions. Experimental design has been part
      of the field of statistics since the seminal work of Fisher in the early 20th
      century and a large body of literature exists on how to design optimum
      experiments. Here we investigate the use of the Fisher information matrix to
      decide between candidate positions for adding a taxon to a fixed topology, and
      introduce a parameter transformation that permits comparison of these different
      designs. This extension to Goldman (1998. Proc. R. Soc. Lond. B. 265: 1779-1786) 
      thus allows investigation of "where to add taxa" in a phylogeny. We compare three
      different measures of the total information for selecting the position to add a
      taxon to a tree. Our methods are illustrated by investigating the behavior of the
      three criteria when adding a branch to model trees, and by applying the different
      criteria to two biological examples: a simplified taxon-sampling problem in the
      balsaminoid Ericales and the phylogeny of seed plants.
FAU - Geuten, Koen
AU  - Geuten K
AD  - Laboratory of Plant Systematics, KU Leuven, Belgium. koen.geuten@yale.edu
FAU - Massingham, Tim
AU  - Massingham T
FAU - Darius, Paul
AU  - Darius P
FAU - Smets, Erik
AU  - Smets E
FAU - Goldman, Nick
AU  - Goldman N
LA  - eng
GR  - Wellcome Trust/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Plants/*classification/*genetics
EDAT- 2007/07/27 09:00
MHDA- 2007/12/29 09:00
CRDT- 2007/07/27 09:00
PHST- 2007/07/27 09:00 [pubmed]
PHST- 2007/12/29 09:00 [medline]
PHST- 2007/07/27 09:00 [entrez]
AID - 780835747 [pii]
AID - 10.1080/10635150701499563 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Aug;56(4):609-22. doi: 10.1080/10635150701499563.

PMID- 17654364
OWN - NLM
STAT- MEDLINE
DCOM- 20071228
LR  - 20070726
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 4
DP  - 2007 Aug
TI  - Assessing calibration uncertainty in molecular dating: the assignment of fossils 
      to alternative calibration points.
PG  - 591-608
AB  - Although recent methodological advances have allowed the incorporation of rate
      variation in molecular dating analyses, the calibration procedure, performed
      mainly through fossils, remains resistant to improvements. One source of
      uncertainty pertains to the assignment of fossils to specific nodes in a
      phylogeny, especially when alternative possibilities exist that can be equally
      justified on morphological grounds. Here we expand on a recently developed fossil
      cross-validation method to evaluate whether alternative nodal assignments of
      multiple fossils produce calibration sets that differ in their internal
      consistency. We use an enlarged Crypteroniaceae-centered phylogeny of Myrtales,
      six fossils, and 72 combinations of calibration points, termed calibration sets, 
      to identify (i) the fossil assignments that produce the most internally
      consistent calibration sets and (ii) the mean ages, derived from these
      calibration sets, for the split of the Southeast Asian Crypteroniaceae from their
      West Gondwanan sister clade (node X). We found that a correlation exists between 
      s values, devised to measure the consistency among the calibration points of a
      calibration set (Near and Sanderson, 2004), and nodal distances among calibration
      points. By ranking all sets according to the percent deviation of s from the
      regression line with nodal distance, we identified the sets with the highest
      level of corrected calibration-set consistency. These sets generated lower
      standard deviations associated with the ages of node X than sets characterized by
      lower corrected consistency. The three calibration sets with the highest
      corrected consistencies produced mean age estimates for node X of 79.70, 79.14,
      and 78.15 My. These timeframes are most compatible with the hypothesis that the
      Crypteroniaceae stem lineage dispersed from Africa to the Deccan plate as it
      drifted northward during the Late Cretaceous.
FAU - Rutschmann, Frank
AU  - Rutschmann F
AD  - Institute of Systematic Botany, University of Zurich, Zollikerstrasse 107,
      Zurich, Switzerland. frank@plant.ch
FAU - Eriksson, Torsten
AU  - Eriksson T
FAU - Salim, Kamariah Abu
AU  - Salim KA
FAU - Conti, Elena
AU  - Conti E
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Calibration
MH  - Computer Simulation
MH  - *Fossils
MH  - *Phylogeny
MH  - Plants/*classification/*genetics
MH  - Time Factors
MH  - *Uncertainty
EDAT- 2007/07/27 09:00
MHDA- 2007/12/29 09:00
CRDT- 2007/07/27 09:00
PHST- 2007/07/27 09:00 [pubmed]
PHST- 2007/12/29 09:00 [medline]
PHST- 2007/07/27 09:00 [entrez]
AID - 780800883 [pii]
AID - 10.1080/10635150701491156 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Aug;56(4):591-608. doi: 10.1080/10635150701491156.

PMID- 17654363
OWN - NLM
STAT- MEDLINE
DCOM- 20071228
LR  - 20070726
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 4
DP  - 2007 Aug
TI  - Summarizing a posterior distribution of trees using agreement subtrees.
PG  - 578-90
AB  - Bayesian inference of phylogeny is unique among phylogenetic reconstruction
      methods in that it produces a posterior distribution of trees rather than a point
      estimate of the best tree. The most common way to summarize this distribution is 
      to report the majority-rule consensus tree annotated with the marginal posterior 
      probabilities of each partition. Reporting a single tree discards information
      contained in the full underlying distribution and reduces the Bayesian analysis
      to simply another method for finding a point estimate of the tree. Even when a
      point estimate of the phylogeny is desired, the majority-rule consensus tree is
      only one possible method, and there may be others that are more appropriate for
      the given data set and application. We present a method for summarizing the
      distribution of trees that is based on identifying agreement subtrees that are
      frequently present in the posterior distribution. This method provides fully
      resolved binary trees for subsets of taxa with high marginal posterior
      probability on the entire tree and includes additional information about the
      spread of the distribution.
FAU - Cranston, Karen A
AU  - Cranston KA
AD  - Department of Ecology, University of Arizona, Tucson, AZ 85721, USA.
      cranston@email.arizona.edu
FAU - Rannala, Bruce
AU  - Rannala B
LA  - eng
GR  - HG01988/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Bayes Theorem
MH  - Computer Simulation
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2007/07/27 09:00
MHDA- 2007/12/29 09:00
CRDT- 2007/07/27 09:00
PHST- 2007/07/27 09:00 [pubmed]
PHST- 2007/12/29 09:00 [medline]
PHST- 2007/07/27 09:00 [entrez]
AID - 780835555 [pii]
AID - 10.1080/10635150701485091 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Aug;56(4):578-90. doi: 10.1080/10635150701485091.

PMID- 17654362
OWN - NLM
STAT- MEDLINE
DCOM- 20071228
LR  - 20070726
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 4
DP  - 2007 Aug
TI  - Improvement of phylogenies after removing divergent and ambiguously aligned
      blocks from protein sequence alignments.
PG  - 564-77
AB  - Alignment quality may have as much impact on phylogenetic reconstruction as the
      phylogenetic methods used. Not only the alignment algorithm, but also the method 
      used to deal with the most problematic alignment regions, may have a critical
      effect on the final tree. Although some authors remove such problematic regions, 
      either manually or using automatic methods, in order to improve phylogenetic
      performance, others prefer to keep such regions to avoid losing any information. 
      Our aim in the present work was to examine whether phylogenetic reconstruction
      improves after alignment cleaning or not. Using simulated protein alignments with
      gaps, we tested the relative performance in diverse phylogenetic analyses of the 
      whole alignments versus the alignments with problematic regions removed with our 
      previously developed Gblocks program. We also tested the performance of more or
      less stringent conditions in the selection of blocks. Alignments constructed with
      different alignment methods (ClustalW, Mafft, and Probcons) were used to estimate
      phylogenetic trees by maximum likelihood, neighbor joining, and parsimony. We
      show that, in most alignment conditions, and for alignments that are not too
      short, removal of blocks leads to better trees. That is, despite losing some
      information, there is an increase in the actual phylogenetic signal. Overall, the
      best trees are obtained by maximum-likelihood reconstruction of alignments
      cleaned by Gblocks. In general, a relaxed selection of blocks is better for short
      alignment, whereas a stringent selection is more adequate for longer ones.
      Finally, we show that cleaned alignments produce better topologies although,
      paradoxically, with lower bootstrap. This indicates that divergent and
      problematic alignment regions may lead, when present, to apparently better
      supported although, in fact, more biased topologies.
FAU - Talavera, Gerard
AU  - Talavera G
AD  - Department of Physiology, Institute of Molecular Biology of Barcelona, Barcelona,
      Spain.
FAU - Castresana, Jose
AU  - Castresana J
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Proteins)
SB  - IM
MH  - Amino Acid Sequence
MH  - Cluster Analysis
MH  - Computer Simulation
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Proteins/*chemistry
MH  - Sequence Alignment/*methods
EDAT- 2007/07/27 09:00
MHDA- 2007/12/29 09:00
CRDT- 2007/07/27 09:00
PHST- 2007/07/27 09:00 [pubmed]
PHST- 2007/12/29 09:00 [medline]
PHST- 2007/07/27 09:00 [entrez]
AID - 780704285 [pii]
AID - 10.1080/10635150701472164 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Aug;56(4):564-77. doi: 10.1080/10635150701472164.

PMID- 17654361
OWN - NLM
STAT- MEDLINE
DCOM- 20071228
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 4
DP  - 2007 Aug
TI  - Calibration choice, rate smoothing, and the pattern of tetrapod diversification
      according to the long nuclear gene RAG-1.
PG  - 543-63
AB  - A phylogeny of tetrapods is inferred from nearly complete sequences of the
      nuclear RAG-1 gene sampled across 88 taxa encompassing all major clades, analyzed
      via parsimony and Bayesian methods. The phylogeny provides support for
      Lissamphibia, Theria, Lepidosauria, a turtle-archosaur clade, as well as most
      traditionally accepted groupings. This tree allows simultaneous molecular clock
      dating for all tetrapod groups using a set of well-corroborated calibrations.
      Relaxed clock (PLRS) methods, using the amniote = 315 Mya (million years ago)
      calibration or a set of consistent calibrations, recovers reasonable divergence
      dates for most groups. However, the analysis systematically underestimates
      divergence dates within archosaurs. The bird-crocodile split, robustly documented
      in the fossil record as being around approximately 245 Mya, is estimated at only 
      approximately 190 Mya, and dates for other divergences within archosaurs are
      similarly underestimated. Archosaurs, and particulary turtles have slow apparent 
      rates possibly confounding rate modeling, and inclusion of calibrations within
      archosaurs (despite their high deviances) not only improves divergence estimates 
      within archosaurs, but also across other groups. Notably, the monotreme-therian
      split ( approximately 210 Mya) matches the fossil record; the squamate radiation 
      ( approximately 190 Mya) is younger than suggested by some recent molecular
      studies and inconsistent with identification of approximately 220 and
      approximately 165 Myo (million-year-old) fossils as acrodont iguanians and
      approximately 95 Myo fossils colubroid snakes; the bird-lizard (reptile) split is
      considerably older than fossil estimates (&lt; or = 285 Mya); and Sphenodon is a
      remarkable phylogenetic relic, being the sole survivor of a lineage more than a
      quarter of a billion years old. Comparison with other molecular clock studies of 
      tetrapod divergences suggests that the common practice of enforcing most
      calibrations as minima, with a single liberal maximal constraint, will
      systematically overestimate divergence dates. Similarly, saturation of
      mitochondrial DNA sequences, and the resultant greater compression of basal
      branches means that using only external deep calibrations will also lead to
      inflated age estimates within the focal ingroup.
FAU - Hugall, Andrew F
AU  - Hugall AF
AD  - School of Earth and Environmental Sciences, University of Adelaide, SA,
      Australia. andrew.hugall@adelaide.edu.au
FAU - Foster, Ralph
AU  - Foster R
FAU - Lee, Michael S Y
AU  - Lee MS
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Homeodomain Proteins)
RN  - 128559-51-3 (RAG-1 protein)
SB  - IM
MH  - Amphibians/*genetics
MH  - Animals
MH  - *Biodiversity
MH  - Birds/*genetics
MH  - Calibration
MH  - Fossils
MH  - Homeodomain Proteins/*genetics
MH  - Mammals/*genetics
MH  - *Phylogeny
MH  - Reptiles/*genetics
MH  - Time Factors
EDAT- 2007/07/27 09:00
MHDA- 2007/12/29 09:00
CRDT- 2007/07/27 09:00
PHST- 2007/07/27 09:00 [pubmed]
PHST- 2007/12/29 09:00 [medline]
PHST- 2007/07/27 09:00 [entrez]
AID - 780704360 [pii]
AID - 10.1080/10635150701477825 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Aug;56(4):543-63. doi: 10.1080/10635150701477825.

PMID- 17562477
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - Taxonomy in a changing world: seeking solutions for a science in crisis.
PG  - 531-9
FAU - Agnarsson, Ingi
AU  - Agnarsson I
AD  - Department of Zoology, University of British Columbia, 2370-6270 University
      Boulevard, Vancouver, BC, Canada. iagnarsson@gmail.com
FAU - Kuntner, Matjaz
AU  - Kuntner M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biology/economics/education/*trends
MH  - *Classification
MH  - Research Support as Topic
RF  - 46
EDAT- 2007/06/15 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/06/15 09:00
PHST- 2007/06/15 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/06/15 09:00 [entrez]
AID - 779485365 [pii]
AID - 10.1080/10635150701424546 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):531-9. doi: 10.1080/10635150701424546.

PMID- 17562476
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20181201
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - WASABI: an automated sequence processing system for multigene phylogenies.
PG  - 523-31
FAU - Kauff, Frank
AU  - Kauff F
AD  - Department of Biology, Duke University, Durham, NC 27708, USA.
      fkauff@biologie.uni-kl.de
FAU - Cox, Cymon J
AU  - Cox CJ
FAU - Lutzoni, Francois
AU  - Lutzoni F
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Computational Biology/*methods
MH  - Electronic Data Processing/methods
MH  - Internet
MH  - Pattern Recognition, Automated
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA/*methods
MH  - Software
EDAT- 2007/06/15 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/06/15 09:00
PHST- 2007/06/15 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/06/15 09:00 [entrez]
AID - 779485313 [pii]
AID - 10.1080/10635150701395340 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):523-31. doi: 10.1080/10635150701395340.

PMID- 17562475
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20070612
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - Evidence for time dependency of molecular rate estimates.
PG  - 515-22
FAU - Ho, Simon Y W
AU  - Ho SY
AD  - Department of Zoology, University of Oxford, Oxford, United Kingdom.
      simon.ho@zoo.ox.ac.uk
FAU - Shapiro, Beth
AU  - Shapiro B
FAU - Phillips, Matthew J
AU  - Phillips MJ
FAU - Cooper, Alan
AU  - Cooper A
FAU - Drummond, Alexei J
AU  - Drummond AJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bison/classification
MH  - Computational Biology/*methods
MH  - *Evolution, Molecular
MH  - Humans
MH  - Time Factors
EDAT- 2007/06/15 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/06/15 09:00
PHST- 2007/06/15 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/06/15 09:00 [entrez]
AID - 779485199 [pii]
AID - 10.1080/10635150701435401 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):515-22. doi: 10.1080/10635150701435401.

PMID- 17562474
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20070612
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - Species trees from gene trees: reconstructing Bayesian posterior distributions of
      a species phylogeny using estimated gene tree distributions.
PG  - 504-14
AB  - The desire to infer the evolutionary history of a group of species should be more
      viable now that a considerable amount of multilocus molecular data is available. 
      However, the current molecular phylogenetic paradigm still reconstructs gene
      trees to represent the species tree. Further, commonly used methods of combining 
      data, such as the concatenation method, are known to be inconsistent in some
      circumstances. In this paper, we propose a Bayesian hierarchical model to
      estimate the phylogeny of a group of species using multiple estimated gene tree
      distributions, such as those that arise in a Bayesian analysis of DNA sequence
      data. Our model employs substitution models used in traditional phylogenetics but
      also uses coalescent theory to explain genealogical signals from species trees to
      gene trees and from gene trees to sequence data, thereby forming a complete
      stochastic model to estimate gene trees, species trees, ancestral population
      sizes, and species divergence times simultaneously. Our model is founded on the
      assumption that gene trees, even of unlinked loci, are correlated due to being
      derived from a single species tree and therefore should be estimated jointly. We 
      apply the method to two multilocus data sets of DNA sequences. The estimates of
      the species tree topology and divergence times appear to be robust to the prior
      of the population size, whereas the estimates of effective population sizes are
      sensitive to the prior used in the analysis. These analyses also suggest that the
      model is superior to the concatenation method in fitting these data sets and thus
      provides a more realistic assessment of the variability in the distribution of
      the species tree that may have produced the molecular information at hand. Future
      improvements of our model and algorithm should include consideration of other
      factors that can cause discordance of gene trees and species trees, such as
      horizontal transfer or gene duplication.
FAU - Liu, Liang
AU  - Liu L
AD  - Department of Statistics, The Ohio State University, Columbus, OH 43210-1293,
      USA. lliu@oeb.harvard.edu
FAU - Pearl, Dennis K
AU  - Pearl DK
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Bayes Theorem
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - Finches/classification/*genetics
MH  - Macaca/classification/*genetics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Time Factors
EDAT- 2007/06/15 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/06/15 09:00
PHST- 2007/06/15 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/06/15 09:00 [entrez]
AID - 779484657 [pii]
AID - 10.1080/10635150701429982 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):504-14. doi: 10.1080/10635150701429982.

PMID- 17562473
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - Correcting the problem of false incongruence due to noise imbalance in the
      incongruence length difference (ILD) test.
PG  - 496-503
AB  - The incongruence length difference (ILD) test is prone to suggesting significant 
      conflict between character partitions when these differ only in the amount of
      undirected homoplasy (noise). This has been shown to be due to nonlinearity in
      the relationship between tree length and noise. Here we show that by
      standardizing either tree length or 1-retention index on a 0-to-1 scale, and then
      taking the arcsine of the value, the resulting value is linearly related to noise
      except at extremely high noise levels. We then investigate the effect of
      substituting these values instead of raw tree metrics in a modified ILD test
      (here called arcsine-ILD) for two types of noise. We show that, using the
      modified metric instead of the raw length, the results of ILD tests agreed better
      with desirable properties.
FAU - Quicke, Donald L J
AU  - Quicke DL
AD  - Division of Biology, Imperial College London, Silwood Park Campus, Ascot,
      Berkshire, SL5 7PY, UK. d.quicke@imperial.ac.uk
FAU - Jones, Owen R
AU  - Jones OR
FAU - Epstein, Daniella R
AU  - Epstein DR
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Artifacts
MH  - *Biological Evolution
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - *Phylogeny
EDAT- 2007/06/15 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/06/15 09:00
PHST- 2007/06/15 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/06/15 09:00 [entrez]
AID - 779484986 [pii]
AID - 10.1080/10635150701429974 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):496-503. doi: 10.1080/10635150701429974.

PMID- 17562472
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20070612
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - On divide-and-conquer strategies for parsimony analysis of large data sets:
      Rec-I-DCM3 versus TNT.
PG  - 485-95
AB  - Roshan et al. recently described a "divide-and-conquer" technique for parsimony
      analysis of large data sets, Rec-I-DCM3, and stated that it compares very
      favorably to results using the program TNT. Their technique is based on selecting
      subsets of taxa to create reduced data sets or subproblems, finding
      most-parsimonious trees for each reduced data set, recombining all parts
      together, and then performing global TBR swapping on the combined tree. Here, we 
      contrast this approach to sectorial searches, a divide-and-conquer algorithm
      implemented in TNT. This algorithm also uses a guide tree to create subproblems, 
      with the first-pass state sets of the nodes that join the selected sectors with
      the rest of the topology; this allows exact length calculations for the entire
      topology (that is, any solution N steps shorter than the original, for the
      reduced subproblem, must also be N steps shorter for the entire topology). We
      show here that, for sectors of similar size analyzed with the same search
      algorithms, subdividing data sets with sectorial searches produces better results
      than subdividing with Rec-I-DCM3. Roshan et al.'s claim that Rec-I-DCM3
      outperforms the techniques in TNT was caused by a poor experimental design and
      algorithmic settings used for the runs in TNT. In particular, for finding trees
      at or very close to the minimum known length of the analyzed data sets, TNT
      clearly outperforms Rec-I-DCM3. Finally, we show that the performance of
      Rec-I-DCM3 is bound by the efficiency of TBR implementation for the complete data
      set, as this method behaves (after some number of iterations) as a technique for 
      cyclic perturbations and improvements more than as a divide-and-conquer strategy.
FAU - Goloboff, Pablo A
AU  - Goloboff PA
AD  - CONICET, Instituto Superior de Entomologia, Instituto Miguel Lillo, Tucuman,
      Argentina. pablogolo@csnat.unt.edu.ar
FAU - Pol, Diego
AU  - Pol D
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - Software
MH  - Time Factors
EDAT- 2007/06/15 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/06/15 09:00
PHST- 2007/06/15 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/06/15 09:00 [entrez]
AID - 779482429 [pii]
AID - 10.1080/10635150701431905 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):485-95. doi: 10.1080/10635150701431905.

PMID- 17562471
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20070612
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - The utility of amplified fragment length polymorphisms in phylogenetics: a
      comparison of homology within and between genomes.
PG  - 477-84
AB  - The amplified fragment length polymorphism (AFLP) technique is being increasingly
      used in phylogenetic studies, especially in groups of rapidly radiating taxa. One
      of the key issues in the phylogenetic suitability of this technique is whether
      the DNA fragments generated via the AFLP method are homologous within and among
      the taxa being studied. We used a bioinformatics approach to assess homology
      based on both chromosomal location and sequence similarity of AFLP fragments. The
      AFLP technique was electronically simulated on genomes from eight organisms that 
      represented a range of genome sizes. The results demonstrated that within a
      genome, the number of fragments is positively associated with genome size, and
      the degree of homology decreases with increasing numbers of fragments generated. 
      The average homology of fragments was 89% for small genomes (&lt; 400 Mb) but
      decreased to 59% for large genomes (&gt; 2 Gb). Fragment homology for large genomes 
      can be increased by excluding smaller fragments, although there is no clear upper
      limit for the size of fragments to exclude. A second approach is to increase the 
      number of selective nucleotides in the final selective amplification step. For
      strains of the same organism, homology based on chromosome location and sequence 
      similarity of fragments was 100%. Fragment homology for more distantly related
      taxa, however, decreased with greater time since divergence. We conclude that
      AFLP data are best suited for examining phylogeographic patterns within species
      and among very recently diverged species.
FAU - Althoff, David M
AU  - Althoff DM
AD  - Department of Biology, Syracuse University, Syracuse, NY 13244, USA.
      dmalthof@syr.edu
FAU - Gitzendanner, Matthew A
AU  - Gitzendanner MA
FAU - Segraves, Kari A
AU  - Segraves KA
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Drosophila/genetics
MH  - Genome/*genetics
MH  - Humans
MH  - *Phylogeny
MH  - *Polymorphism, Restriction Fragment Length
MH  - Saccharomyces cerevisiae/genetics
EDAT- 2007/06/15 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/06/15 09:00
PHST- 2007/06/15 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/06/15 09:00 [entrez]
AID - 779481819 [pii]
AID - 10.1080/10635150701427077 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):477-84. doi: 10.1080/10635150701427077.

PMID- 17562470
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20070612
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - Untangling complex histories of genome mergings in high polyploids.
PG  - 467-76
AB  - Polyploidy, the duplication of entire genomes, plays a major role in plant
      evolution. In allopolyploids, genome duplication is associated with hybridization
      between two or more divergent genomes. Successive hybridization and
      polyploidization events can build up species complexes of allopolyploids with
      complicated network-like histories, and the evolutionary history of many plant
      groups cannot be adequately represented by phylogenetic trees because of such
      reticulate events. The history of complex genome mergings within a high-polyploid
      species complex in the genus Cerastium (Caryophyllaceae) is here untangled by the
      use of a network algorithm and noncoding sequences of a low-copy number gene. The
      resulting network illustrates how hybridization and polyploidization have acted
      as key evolutionary processes in creating a plant group where high-level
      allopolyploids clearly outnumber extant parental genomes.
FAU - Brysting, Anne K
AU  - Brysting AK
AD  - National Centre for Biosystematics, Natural History Museum, University of Oslo,
      Blindern, Oslo, Norway. a.k.brysting@bio.uio.no
FAU - Oxelman, Bengt
AU  - Oxelman B
FAU - Huber, Katharina T
AU  - Huber KT
FAU - Moulton, Vincent
AU  - Moulton V
FAU - Brochmann, Christian
AU  - Brochmann C
LA  - eng
SI  - GENBANK/DQ274063
SI  - GENBANK/DQ274064
SI  - GENBANK/DQ274065
SI  - GENBANK/DQ274066
SI  - GENBANK/DQ274067
SI  - GENBANK/DQ274068
SI  - GENBANK/DQ274069
SI  - GENBANK/DQ274070
SI  - GENBANK/DQ274071
SI  - GENBANK/DQ274072
SI  - GENBANK/DQ274073
SI  - GENBANK/DQ274074
SI  - GENBANK/DQ274075
SI  - GENBANK/DQ274076
SI  - GENBANK/DQ274077
SI  - GENBANK/DQ274078
SI  - GENBANK/DQ274079
SI  - GENBANK/DQ274080
SI  - GENBANK/DQ274081
SI  - GENBANK/DQ274082
SI  - GENBANK/DQ274083
SI  - GENBANK/DQ274084
SI  - GENBANK/DQ274085
SI  - GENBANK/DQ274086
SI  - GENBANK/DQ274087
SI  - GENBANK/DQ274088
SI  - GENBANK/DQ274089
SI  - GENBANK/DQ274090
SI  - GENBANK/DQ274091
SI  - GENBANK/DQ274092
SI  - GENBANK/DQ274093
SI  - GENBANK/DQ274094
SI  - GENBANK/DQ274095
SI  - GENBANK/DQ274096
SI  - GENBANK/DQ274097
SI  - GENBANK/DQ274098
SI  - GENBANK/DQ274099
SI  - GENBANK/DQ274100
SI  - GENBANK/DQ274101
SI  - GENBANK/DQ274102
SI  - GENBANK/DQ274103
SI  - GENBANK/DQ274104
SI  - GENBANK/DQ274105
SI  - GENBANK/DQ274106
SI  - GENBANK/DQ274107
SI  - GENBANK/DQ274108
SI  - GENBANK/DQ274109
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - EC 2.7.7.- (RNA Polymerase II)
SB  - IM
MH  - Base Sequence
MH  - DNA Primers/chemistry
MH  - Evolution, Molecular
MH  - Genome, Plant/*genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - *Polyploidy
MH  - RNA Polymerase II/genetics
EDAT- 2007/06/15 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/06/15 09:00
PHST- 2007/06/15 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/06/15 09:00 [entrez]
AID - 779481346 [pii]
AID - 10.1080/10635150701424553 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):467-76. doi: 10.1080/10635150701424553.

PMID- 17558967
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20071203
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - Inferring speciation times under an episodic molecular clock.
PG  - 453-66
AB  - We extend our recently developed Markov chain Monte Carlo algorithm for Bayesian 
      estimation of species divergence times to allow variable evolutionary rates among
      lineages. The method can use heterogeneous data from multiple gene loci and
      accommodate multiple fossil calibrations. Uncertainties in fossil calibrations
      are described using flexible statistical distributions. The prior for divergence 
      times for nodes lacking fossil calibrations is specified by use of a birth-death 
      process with species sampling. The prior for lineage-specific substitution rates 
      is specified using either a model with autocorrelated rates among adjacent
      lineages (based on a geometric Brownian motion model of rate drift) or a model
      with independent rates among lineages specified by a log-normal probability
      distribution. We develop an infinite-sites theory, which predicts that when the
      amount of sequence data approaches infinity, the width of the posterior
      credibility interval and the posterior mean of divergence times form a perfect
      linear relationship, with the slope indicating uncertainties in time estimates
      that cannot be reduced by sequence data alone. Simulations are used to study the 
      influence of among-lineage rate variation and the number of loci sampled on the
      uncertainty of divergence time estimates. The analysis suggests that posterior
      time estimates typically involve considerable uncertainties even with an infinite
      amount of sequence data, and that the reliability and precision of fossil
      calibrations are critically important to divergence time estimation. We apply our
      new algorithms to two empirical data sets and compare the results with those
      obtained in previous Bayesian and likelihood analyses. The results demonstrate
      the utility of our new algorithms.
FAU - Rannala, Bruce
AU  - Rannala B
AD  - Genome Center and Section of Evolution and Ecology, University of California,
      Davis, California, USA.
FAU - Yang, Ziheng
AU  - Yang Z
LA  - eng
GR  - HG01988/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Felidae/classification
MH  - *Genetic Speciation
MH  - Humans
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2007/06/15 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/06/15 09:00
PHST- 2007/06/15 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/06/15 09:00 [entrez]
AID - 779374620 [pii]
AID - 10.1080/10635150701420643 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):453-66. doi: 10.1080/10635150701420643.

PMID- 17558966
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20070611
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - Majority-rule supertrees.
PG  - 445-52
AB  - Most supertree methods proposed to date are essentially ad hoc, rather than
      designed with particular properties in mind. Although the supertree problem
      remains difficult, one promising avenue is to develop from better understood
      consensus methods to the more general supertree setting. Here, we generalize the 
      widely used majority-rule consensus method to the supertree setting. The
      majority-rule consensus tree is the strict consensus of the median trees under
      the symmetric-difference metric, so we can generalize the consensus method by
      generalizing this metric to trees with differing leaf sets. There are two
      different natural generalizations, based on pruning or grafting leaves to produce
      comparable trees, and these two generalizations produce two different, but
      related, majority-rule supertree methods.
FAU - Cotton, James A
AU  - Cotton JA
AD  - Department of Zoology, The Natural History Museum, London SW7 5BD, UK.
      james.cotton@nuim.ie
FAU - Wilkinson, Mark
AU  - Wilkinson M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2007/06/15 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/06/15 09:00
PHST- 2007/06/15 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/06/15 09:00 [entrez]
AID - 779373116 [pii]
AID - 10.1080/10635150701416682 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):445-52. doi: 10.1080/10635150701416682.

PMID- 17558965
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20070611
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - Resource-aware taxon selection for maximizing phylogenetic diversity.
PG  - 431-44
AB  - Phylogenetic diversity (PD) is a useful metric for selecting taxa in a range of
      biological applications, for example, bioconservation and genomics, where the
      selection is usually constrained by the limited availability of resources. We
      formalize taxon selection as a conceptually simple optimization problem, aiming
      to maximize PD subject to resource constraints. This allows us to take into
      account the different amounts of resources required by the different taxa.
      Although this is a computationally difficult problem, we present a dynamic
      programming algorithm that solves it in pseudo-polynomial time. Our algorithm can
      also solve many instances of the Noah's Ark Problem, a more realistic formulation
      of taxon selection for biodiversity conservation that allows for taxon-specific
      extinction risks. These instances extend the set of problems for which solutions 
      are available beyond previously known greedy-tractable cases. Finally, we discuss
      the relevance of our results to real-life scenarios.
FAU - Pardi, Fabio
AU  - Pardi F
AD  - EMBL - European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, 
      Cambridge CB10 1SD, UK. pardi@ebi.ac.uk
FAU - Goldman, Nick
AU  - Goldman N
LA  - eng
PT  - Journal Article
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - *Biodiversity
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Conservation of Natural Resources/*methods
MH  - *Models, Biological
MH  - *Phylogeny
RF  - 65
EDAT- 2007/06/15 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/06/15 09:00
PHST- 2007/06/15 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/06/15 09:00 [entrez]
AID - 779377207 [pii]
AID - 10.1080/10635150701411279 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):431-44. doi: 10.1080/10635150701411279.

PMID- 17558964
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20070611
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - Phylogeny of extant and fossil Juglandaceae inferred from the integration of
      molecular and morphological data sets.
PG  - 412-30
AB  - It is widely acknowledged that integrating fossils into data sets of extant taxa 
      is imperative for proper placement of fossils, resolution of relationships, and a
      better understanding of character evolution. The importance of this process has
      been further magnified because of the crucial role of fossils in dating
      divergence times. Outstanding issues remain, including appropriate methods to
      place fossils in phylogenetic trees, the importance of molecules versus
      morphology in these analyses, as well as the impact of potentially large amounts 
      of missing data for fossil taxa. In this study we used the angiosperm clade
      Juglandaceae as a model for investigating methods of integrating fossils into a
      phylogenetic framework of extant taxa. The clade has a rich fossil record
      relative to low extant diversity, as well as a robust molecular phylogeny and
      morphological database for extant taxa. After combining fossil organ genera into 
      composite and terminal taxa, our objectives were to (1) compare multiple methods 
      for the integration of the fossils and extant taxa (including total evidence,
      molecular scaffolds, and molecular matrix representation with parsimony [MRP]);
      (2) explore the impact of missing data (incomplete taxa and characters) and the
      evidence for placing fossils on the topology; (3) simulate the phylogenetic
      effect of missing data by creating "artificial fossils"; and (4) place fossils
      and compare the impact of single and multiple fossil constraints in estimating
      the age of clades. Despite large and variable amounts of missing data, each of
      the methods provided reasonable placement of both fossils and simulated
      "artificial fossils" in the phylogeny previously inferred only from extant taxa. 
      Our results clearly show that the amount of missing data in any given taxon is
      not by itself an operational guideline for excluding fossils from analysis. Three
      fossil taxa (Cruciptera simsonii, Paleoplatycarya wingii, and Platycarya
      americana) were placed within crown clades containing living taxa for which
      relationships previously had been suggested based on morphology, whereas
      Polyptera manningii, a mosaic taxon with equivocal affinities, was placed firmly 
      as sister to two modern crown clades. The position of Paleooreomunnea stoneana
      was ambiguous with total evidence but conclusive with DNA scaffolds and MRP.
      There was less disturbance of relationships among extant taxa using a total
      evidence approach, and the DNA scaffold approach did not provide improved
      resolution or internal support for clades compared to total evidence, whereas
      weighted MRP retained comparable levels of support but lost crown clade
      resolution. Multiple internal minimum age constraints generally provided
      reasonable age estimates, but the use of single constraints provided by extinct
      genera tended to underestimate clade ages.
FAU - Manos, Paul S
AU  - Manos PS
AD  - Department of Biology, Duke University, Durham, NC 27708-0338, USA.
      pmanos@duke.edu
FAU - Soltis, Pamela S
AU  - Soltis PS
FAU - Soltis, Douglas E
AU  - Soltis DE
FAU - Manchester, Steven R
AU  - Manchester SR
FAU - Oh, Sang-Hun
AU  - Oh SH
FAU - Bell, Charles D
AU  - Bell CD
FAU - Dilcher, David L
AU  - Dilcher DL
FAU - Stone, Donald E
AU  - Stone DE
LA  - eng
SI  - GENBANK/EF140989
SI  - GENBANK/EF140990
SI  - GENBANK/EF140991
SI  - GENBANK/EF140992
SI  - GENBANK/EF140993
SI  - GENBANK/EF140994
SI  - GENBANK/EF140995
SI  - GENBANK/EF140996
SI  - GENBANK/EF140997
SI  - GENBANK/EF141000
SI  - GENBANK/EF141001
SI  - GENBANK/EF141002
SI  - GENBANK/EF141003
SI  - GENBANK/EF141004
SI  - GENBANK/EF141005
SI  - GENBANK/EF141006
SI  - GENBANK/EF141007
SI  - GENBANK/EF141008
SI  - GENBANK/EF141009
SI  - GENBANK/EF141010
SI  - GENBANK/EF141011
SI  - GENBANK/EF141012
SI  - GENBANK/EF141013
SI  - GENBANK/EF141014
SI  - GENBANK/EF141015
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
RN  - 0 (DNA, Ribosomal Spacer)
SB  - IM
MH  - Base Sequence
MH  - Classification/methods
MH  - DNA, Chloroplast/genetics
MH  - DNA, Ribosomal Spacer/genetics
MH  - Fossils
MH  - Fruit/anatomy &amp; histology
MH  - Juglandaceae/anatomy &amp; histology/*classification/genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Pollen/ultrastructure
MH  - Time Factors
EDAT- 2007/06/15 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/06/15 09:00
PHST- 2007/06/15 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/06/15 09:00 [entrez]
AID - 779372913 [pii]
AID - 10.1080/10635150701408523 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):412-30. doi: 10.1080/10635150701408523.

PMID- 17520504
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - Estimating species phylogeny from gene-tree probabilities despite incomplete
      lineage sorting: an example from Melanoplus grasshoppers.
PG  - 400-11
AB  - Estimating phylogenetic relationships among closely related species can be
      extremely difficult when there is incongruence among gene trees and between the
      gene trees and the species tree. Here we show that incorporating a model of the
      stochastic loss of gene lineages by genetic drift into the phylogenetic
      estimation procedure can provide a robust estimate of species relationships,
      despite widespread incomplete sorting of ancestral polymorphism. This approach is
      applied to a group of montane Melanoplus grasshoppers for which genealogical
      discordance among loci and incomplete lineage sorting obscures any obvious
      phylogenetic relationships among species. Unlike traditional treatments where
      gene trees estimated using standard phylogenetic methods are implicitly equated
      with the species tree, with the coalescent-based approach the species tree is
      modeled probabilistically from the estimated gene trees. The estimated species
      phylogeny (the ESP) is calculated for the grasshoppers from multiple gene trees
      reconstructed for nuclear loci and a mitochondrial gene. This empirical
      application is coupled with a simulation study to explore the performance of the 
      coalescent-based approach. Specifically, we test the accuracy of the ESP given
      the data based on analyses of simulated data matching the multilocus data
      collected in Melanoplus (i.e., data were simulated for each locus with the same
      number of base pairs and locus-specific mutational models). The results of the
      study show that ESPs can be computed using the coalescent-based approach long
      before reciprocal monophyly has been achieved, and that these statistical
      estimates are accurate. This contrasts with analyses of the empirical data
      collected in Melanoplus and simulated data based on concatenation of multiple
      loci, for which the incomplete lineage sorting of recently diverged species posed
      significant problems. The strengths and potential challenges associated with
      incorporating an explicit model of gene-lineage coalescence into the phylogenetic
      procedure to obtain an ESP, as illustrated by application to Melanoplus, versus
      concatenation and consensus approaches are discussed. This study represents a
      fundamental shift in how species relationships are estimated - the relationship
      between the gene trees and the species phylogeny is modeled probabilistically
      rather than equating gene trees with a species tree.
FAU - Carstens, Bryan C
AU  - Carstens BC
AD  - Department of Ecology and Evolutionary Biology, Museum of Zoology, University of 
      Michigan, Ann Arbor, MI 48109-1079, USA.
FAU - Knowles, L Lacey
AU  - Knowles LL
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Computer Simulation
MH  - *Genetic Speciation
MH  - Genetic Variation
MH  - Grasshoppers/*classification/genetics
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2007/05/24 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/05/24 09:00
PHST- 2007/05/24 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/05/24 09:00 [entrez]
AID - 778979727 [pii]
AID - 10.1080/10635150701405560 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):400-11. doi: 10.1080/10635150701405560.

PMID- 17520503
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20070523
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - Detecting and overcoming systematic errors in genome-scale phylogenies.
PG  - 389-99
AB  - Genome-scale data sets result in an enhanced resolution of the phylogenetic
      inference by reducing stochastic errors. However, there is also an increase of
      systematic errors due to model violations, which can lead to erroneous
      phylogenies. Here, we explore the impact of systematic errors on the resolution
      of the eukaryotic phylogeny using a data set of 143 nuclear-encoded proteins from
      37 species. The initial observation was that, despite the impressive amount of
      data, some branches had no significant statistical support. To demonstrate that
      this lack of resolution is due to a mutual annihilation of phylogenetic and
      nonphylogenetic signals, we created a series of data sets with slightly different
      taxon sampling. As expected, these data sets yielded strongly supported but
      mutually exclusive trees, thus confirming the presence of conflicting
      phylogenetic and nonphylogenetic signals in the original data set. To decide on
      the correct tree, we applied several methods expected to reduce the impact of
      some kinds of systematic error. Briefly, we show that (i) removing fast-evolving 
      positions, (ii) recoding amino acids into functional categories, and (iii) using 
      a site-heterogeneous mixture model (CAT) are three effective means of increasing 
      the ratio of phylogenetic to nonphylogenetic signal. Finally, our results allow
      us to formulate guidelines for detecting and overcoming phylogenetic artefacts in
      genome-scale phylogenetic analyses.
FAU - Rodriguez-Ezpeleta, Naiara
AU  - Rodriguez-Ezpeleta N
AD  - Canadian Institute for Advanced Research, Centre Robert Cedergren, Departement de
      Biochimie, Universite de Montreal, 2900 Boulevard Edouard-Montpetit, Montreal,
      Quebec, H3T 1J4, Canada.
FAU - Brinkmann, Henner
AU  - Brinkmann H
FAU - Roure, Beatrice
AU  - Roure B
FAU - Lartillot, Nicolas
AU  - Lartillot N
FAU - Lang, B Franz
AU  - Lang BF
FAU - Philippe, Herve
AU  - Philippe H
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amino Acid Sequence
MH  - Animals
MH  - Classification/*methods
MH  - Computational Biology
MH  - *Evolution, Molecular
MH  - *Genome
MH  - *Phylogeny
MH  - Principal Component Analysis
MH  - Selection Bias
EDAT- 2007/05/24 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/05/24 09:00
PHST- 2007/05/24 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/05/24 09:00 [entrez]
AID - 778977148 [pii]
AID - 10.1080/10635150701397643 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):389-99. doi: 10.1080/10635150701397643.

PMID- 17520502
OWN - NLM
STAT- MEDLINE
DCOM- 20070913
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 3
DP  - 2007 Jun
TI  - Fossils, molecules, divergence times, and the origin of lissamphibians.
PG  - 369-88
AB  - A review of the paleontological literature shows that the early dates of
      appearance of Lissamphibia recently inferred from molecular data do not favor an 
      origin of extant amphibians from temnospondyls, contrary to recent claims. A
      supertree is assembled using new Mesquite modules that allow extinct taxa to be
      incorporated into a time-calibrated phylogeny with a user-defined geological time
      scale. The supertree incorporates 223 extinct species of lissamphibians and has a
      highly significant stratigraphic fit. Some divergences can even be dated with
      sufficient precision to serve as calibration points in molecular divergence date 
      analyses. Fourteen combinations of minimal branch length settings and 10 random
      resolutions for each polytomy give much more recent minimal origination times of 
      lissamphibian taxa than recent studies based on a phylogenetic analyses of
      molecular sequences. Attempts to replicate recent molecular date estimates show
      that these estimates depend strongly on the choice of calibration points, on the 
      dating method, and on the chosen model of evolution; for instance, the estimate
      for the date of the origin of Lissamphibia can lie between 351 and 266 Mya. This 
      range of values is generally compatible with our time-calibrated supertree and
      indicates that there is no unbridgeable gap between dates obtained using the
      fossil record and those using molecular evidence, contrary to previous
      suggestions.
FAU - Marjanovic, David
AU  - Marjanovic D
AD  - Comparative Osteohistology, UMR CNRS 7179, Universite Paris 6, Paris, France.
FAU - Laurin, Michel
AU  - Laurin M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amphibians/*classification
MH  - Animals
MH  - Biodiversity
MH  - *Biological Evolution
MH  - Evolution, Molecular
MH  - *Fossils
MH  - Genetic Variation
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2007/05/24 09:00
MHDA- 2007/09/14 09:00
CRDT- 2007/05/24 09:00
PHST- 2007/05/24 09:00 [pubmed]
PHST- 2007/09/14 09:00 [medline]
PHST- 2007/05/24 09:00 [entrez]
AID - 778976765 [pii]
AID - 10.1080/10635150701397635 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Jun;56(3):369-88. doi: 10.1080/10635150701397635.

PMID- 17464890
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070427
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - How many genes should a systematist sample? Conflicting insights from a
      phylogenomic matrix characterized by replicated incongruence.
PG  - 355-63
FAU - Gatesy, John
AU  - Gatesy J
AD  - Department of Biology, University of California Riverside, Spieth Hall,
      Riverside, California 92521, USA. john.gatesy@ucr.edu
FAU - DeSalle, Rob
AU  - DeSalle R
FAU - Wahlberg, Niklas
AU  - Wahlberg N
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/methods
MH  - Genes, Fungal
MH  - Genomics
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Saccharomyces/classification/genetics
MH  - Sample Size
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777716198 [pii]
AID - 10.1080/10635150701294733 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):355-63. doi: 10.1080/10635150701294733.

PMID- 17464889
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070427
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Seeing the trees for the network: consensus, information content, and
      superphylogenies.
PG  - 345-55
FAU - Gauthier, Olivier
AU  - Gauthier O
AD  - Departement de sciences biologiques, Universite de Montreal, C.P. 6128,
      Succursale Centre-Ville, Montreal (Quebec), H3C 3J7, Canada.
      olivier.gauthier@umontreal.ca
FAU - Lapointe, Francois-Joseph
AU  - Lapointe FJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Classification/methods
MH  - Consensus
MH  - DNA, Mitochondrial/chemistry
MH  - *Evolution, Molecular
MH  - Mammals/classification/genetics
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777721075 [pii]
AID - 10.1080/10635150701286549 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):345-55. doi: 10.1080/10635150701286549.

PMID- 17464888
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070427
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Alarm bells for the molecular clock? No support for Ho et al.'s model of
      time-dependent molecular rate estimates.
PG  - 337-45
FAU - Emerson, Brent C
AU  - Emerson BC
AD  - Centre for Ecology, Evolution and Conservation, School of Biological Sciences,
      University of East Anglia, Norwich, NR4 7TJ, UK. b.emerson@uea.ac.uk
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Classification/methods
MH  - DNA, Mitochondrial/chemistry
MH  - *Evolution, Molecular
MH  - Genetic Speciation
MH  - Geography
MH  - Hominidae/genetics
MH  - Humans
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777712573 [pii]
AID - 10.1080/10635150701258795 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):337-45. doi: 10.1080/10635150701258795.

PMID- 17464887
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070427
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Properties of supertree methods in the consensus setting.
PG  - 330-7
FAU - Wilkinson, Mark
AU  - Wilkinson M
AD  - Department of Zoology, The Natural History Museum, London, SW7 5BD, UK.
      marw@nhm.ac.uk
FAU - Cotton, James A
AU  - Cotton JA
FAU - Lapointe, Francois-Joseph
AU  - Lapointe FJ
FAU - Pisani, Davide
AU  - Pisani D
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/methods
MH  - Consensus
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777720369 [pii]
AID - 10.1080/10635150701245370 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):330-7. doi: 10.1080/10635150701245370.

PMID- 17464886
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070427
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Genomic analysis and geographic visualization of the spread of avian influenza
      (H5N1).
PG  - 321-9
FAU - Janies, Daniel
AU  - Janies D
AD  - Department of Biomedical Informatics, The Ohio State University, 3190 Graves
      Hall, 333 W. 10th Ave., Columbus, Ohio 43210-1239, USA. Daniel.Janies@osumc.edu
FAU - Hill, Andrew W
AU  - Hill AW
FAU - Guralnick, Robert
AU  - Guralnick R
FAU - Habib, Farhat
AU  - Habib F
FAU - Waltari, Eric
AU  - Waltari E
FAU - Wheeler, Ward C
AU  - Wheeler WC
LA  - eng
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Birds
MH  - Drug Resistance, Viral
MH  - Evolution, Molecular
MH  - Genome, Viral
MH  - Genomics
MH  - Genotype
MH  - *Geography
MH  - Influenza A Virus, H5N1 Subtype/*genetics
MH  - Influenza in Birds/genetics/*transmission/virology
MH  - *Models, Biological
MH  - Mutation
MH  - Time Factors
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777718373 [pii]
AID - 10.1080/10635150701266848 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):321-9. doi: 10.1080/10635150701266848.

PMID- 17464885
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070427
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Distinguishing terminal monophyletic groups from reticulate taxa: performance of 
      phenetic, tree-based, and network procedures.
PG  - 302-20
AB  - Hybridization is a well-documented, natural phenomenon that is common at low
      taxonomic levels in the higher plants and other groups. In spite of the obvious
      potential for gene flow via hybridization to cause reticulation in an
      evolutionary tree, analytical methods based on a strictly bifurcating model of
      evolution have frequently been applied to data sets containing taxa known to
      hybridize in nature. Using simulated data, we evaluated the relative performance 
      of phenetic, tree-based, and network approaches for distinguishing between taxa
      with known reticulate history and taxa that were true terminal monophyletic
      groups. In all methods examined, type I error (the erroneous rejection of the
      null hypothesis that a taxon of interest is not monophyletic) was likely during
      the early stages of introgressive hybridization. We used the gradual erosion of
      type I error with continued gene flow as a metric for assessing relative
      performance. Bifurcating tree-based methods performed poorly, with highly
      supported, incorrect topologies appearing during some phases of the simulation.
      Based on our model, we estimate that many thousands of gene flow events may be
      required in natural systems before reticulate taxa will be reliably detected
      using tree-based methods of phylogeny reconstruction. We conclude that the use of
      standard bifurcating tree-based methods to identify terminal monophyletic groups 
      for the purposes of defining or delimiting phylogenetic species, or for
      prioritizing populations for conservation purposes, is difficult to justify when 
      gene flow between sampled taxa is possible. As an alternative, we explored the
      use of two network methods. Minimum spanning networks performed worse than most
      tree-based methods and did not yield topologies that were easily interpretable as
      phylogenies. The performance of NeighborNet was comparable to parsimony bootstrap
      analysis. NeighborNet and reverse successive weighting were capable of
      identifying an ephemeral signature of reticulate evolution during the early
      stages of introgression by revealing conflicting phylogenetic signal. However,
      when gene flow was topologically complex, the conflicting phylogenetic signal
      revealed by these methods resulted in a high probability of type II error
      (inferring that a monophyletic taxon has a reticulate history). Lastly, we
      present a novel application of an existing nonparametric clustering procedure
      that, when used against a density landscape derived from principal coordinate
      data, showed superior performance to the tree-based and network procedures
      tested.
FAU - Reeves, Patrick A
AU  - Reeves PA
AD  - United States Department of Agriculture, Agricultural Research Service, National 
      Center for Genetic Resources Preservation, 1111 South Mason Street, Fort Collins,
      Colorado 80521, USA.
FAU - Richards, Christopher M
AU  - Richards CM
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/methods
MH  - Computer Simulation
MH  - Evolution, Molecular
MH  - *Gene Flow
MH  - Hybridization, Genetic
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777722480 [pii]
AID - 10.1080/10635150701324225 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):302-20. doi: 10.1080/10635150701324225.

PMID- 17464884
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Foundational issues concerning taxa and taxon names.
PG  - 295-301
AB  - In a series of articles, Rieppel (2005, Biol. Philos. 20:465-487; 2006a,
      Cladistics 22:186-197; 2006b, Systematist 26:5-9), Keller et al. (2003, Bot. Rev.
      69:93-110), and Nixon and Carpenter (2000, Cladistics 16:298-318) criticize the
      philosophical foundations of the PhyloCode. They argue that species and higher
      taxa are not individuals, and they reject the view that taxon names are rigid
      designators. Furthermore, they charge supporters of the individuality thesis and 
      rigid designator theory with assuming essentialism, committing logical
      inconsistencies, and offering proposals that render taxonomy untestable. These
      charges are unsound. Such charges turn on confusions over rigid designator theory
      and the distinction between kinds and individuals. In addition, Rieppel's, Keller
      et al.'s, and Nixon and Carpenter's proposed alternatives are no better and have 
      their own problems. The individuality thesis and rigid designator theory should
      not be quickly abandoned.
FAU - Ereshefsky, Marc
AU  - Ereshefsky M
AD  - Department of Philosophy, University of Calgary, 2500 University Drive, NW
      Calgary, Alberta, T2N 1N4, Canada. ereshefs@ucalgary.ca
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/methods
MH  - *Models, Biological
MH  - *Phylogeny
MH  - *Terminology as Topic
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777716772 [pii]
AID - 10.1080/10635150701317401 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):295-301. doi: 10.1080/10635150701317401.

PMID- 17464883
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070427
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Linking of digital images to phylogenetic data matrices using a morphological
      ontology.
PG  - 283-94
AB  - Images are paramount in documentation of morphological data. Production and
      reproduction costs have traditionally limited how many illustrations taxonomy
      could afford to publish, and much comparative knowledge continues to be lost as
      generations turn over. Now digital images are cheaply produced and easily
      disseminated electronically but pose problems in maintenance, curation, sharing, 
      and use, particularly in long-term data sets involving multiple collaborators and
      institutions. We propose an efficient linkage of images to phylogenetic data sets
      via an ontology of morphological terms; an underlying, fine-grained database of
      specimens, images, and associated metadata; fixation of the meaning of
      morphological terms (homolog names) by ostensive references to particular taxa;
      and formalization of images as standard views. The ontology provides the
      intellectual structure and fundamental design of the relationships and enables
      intelligent queries to populate phylogenetic data sets with images. The database 
      itself documents primary morphological observations, their vouchers, and
      associated metadata, rather than the conventional data set cell, and thereby
      facilitates data maintenance despite character redefinition or specimen
      reidentification. It minimizes reexamination of specimens, loss of information or
      data quality, and echoes the data models of web-based repositories for images,
      specimens, and taxonomic names. Confusion and ambiguity in the meanings of
      technical morphological terms are reduced by ostensive definitions pointing to
      features in particular taxa, which may serve as reference for globally unique
      identifiers of characters. Finally, the concept of standard views (an image
      illustrating one or more homologs in a specific sex and life stage, in a specific
      orientation, using a specific device and preparation technique) enables
      efficient, dynamic linkage of images to the data set and automatic population of 
      matrix cells with images independently of scoring decisions.
FAU - Ramirez, Martin J
AU  - Ramirez MJ
AD  - Museo Argentino de Ciencias Naturales Bernardino Rivadavia - CONICET, Avenida
      Angel Gallardo 470, Buenos Aires, C1405DJR, Argentina. ramirez@macn.gov.ar
FAU - Coddington, Jonathan A
AU  - Coddington JA
FAU - Maddison, Wayne P
AU  - Maddison WP
FAU - Midford, Peter E
AU  - Midford PE
FAU - Prendini, Lorenzo
AU  - Prendini L
FAU - Miller, Jeremy
AU  - Miller J
FAU - Griswold, Charles E
AU  - Griswold CE
FAU - Hormiga, Gustavo
AU  - Hormiga G
FAU - Sierwald, Petra
AU  - Sierwald P
FAU - Scharff, Nikolaj
AU  - Scharff N
FAU - Benjamin, Suresh P
AU  - Benjamin SP
FAU - Wheeler, Ward C
AU  - Wheeler WC
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Anatomy, Comparative/methods
MH  - Animals
MH  - Classification/methods
MH  - Computational Biology
MH  - Databases, Factual
MH  - *Image Processing, Computer-Assisted
MH  - *Phylogeny
MH  - Software
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777715885 [pii]
AID - 10.1080/10635150701313848 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):283-94. doi: 10.1080/10635150701313848.

PMID- 17464882
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070427
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Geological dates and molecular rates: rapid divergence of rivers and their
      biotas.
PG  - 271-82
AB  - We highlight a novel molecular clock calibration system based on geologically
      dated river reversal and river capture events. Changes in drainage pattern may
      effect vicariant isolation of freshwater taxa, and thus provide a predictive
      framework for associated phylogeographic study. As a case in point, New Zealand's
      Pelorus and Kaituna rivers became geologically isolated from the larger Wairau
      River system 70 to 130 kyr BP. We conducted mitochondrial DNA phylogeographic
      analyses of two unrelated freshwater-limited fish taxa native to these river
      systems (Gobiomorphus breviceps, n = 63; Galaxias divergens, n = 95).
      Phylogenetic analysis of combined control region and cytochrome b sequences
      yielded reciprocally monophyletic clades of Pelorus-Kaituna and Wairau haplotypes
      for each species. Calibrated rates of molecular change based on this freshwater
      vicariant event are substantially faster than traditionally accepted rates for
      fishes but consistent with other recent inferences based on geologically young
      calibration points. A survey of freshwater phylogeographic literature reveals
      numerous examples in which the ages of recent evolutionary events may have been
      substantially overestimated through the use of "accepted" calibrations. We
      recommend that--wherever possible--biologists should start to reassess the
      conclusions of such studies by using more appropriate molecular calibrations
      derived from recent geological events.
FAU - Waters, Jonathan M
AU  - Waters JM
AD  - Department of Zoology, University of Otago, PO Box 56, Dunedin, New Zealand.
      jonathan.waters@stonebow.otago.ac.nz
FAU - Rowe, Diane L
AU  - Rowe DL
FAU - Apte, Smita
AU  - Apte S
FAU - King, Tania M
AU  - King TM
FAU - Wallis, Graham P
AU  - Wallis GP
FAU - Anderson, Leigh
AU  - Anderson L
FAU - Norris, Richard J
AU  - Norris RJ
FAU - Craw, Dave
AU  - Craw D
FAU - Burridge, Christopher P
AU  - Burridge CP
LA  - eng
SI  - GENBANK/EF140909
SI  - GENBANK/EF140910
SI  - GENBANK/EF140911
SI  - GENBANK/EF140912
SI  - GENBANK/EF140913
SI  - GENBANK/EF140914
SI  - GENBANK/EF140915
SI  - GENBANK/EF140916
SI  - GENBANK/EF140917
SI  - GENBANK/EF140918
SI  - GENBANK/EF140919
SI  - GENBANK/EF140920
SI  - GENBANK/EF140921
SI  - GENBANK/EF140922
SI  - GENBANK/EF140923
SI  - GENBANK/EF140924
SI  - GENBANK/EF140925
SI  - GENBANK/EF140926
SI  - GENBANK/EF140927
SI  - GENBANK/EF140928
SI  - GENBANK/EF140929
SI  - GENBANK/EF140930
SI  - GENBANK/EF140931
SI  - GENBANK/EF140932
SI  - GENBANK/EF140933
SI  - GENBANK/EF140934
SI  - GENBANK/EF140935
SI  - GENBANK/EF140936
SI  - GENBANK/EF140937
SI  - GENBANK/EF140938
SI  - GENBANK/EF140939
SI  - GENBANK/EF140940
SI  - GENBANK/EF140941
SI  - GENBANK/EF140942
SI  - GENBANK/EF140943
SI  - GENBANK/EF140944
SI  - GENBANK/EF140945
SI  - GENBANK/EF140946
SI  - GENBANK/EF140947
SI  - GENBANK/EF140948
SI  - GENBANK/EF140949
SI  - GENBANK/EF140950
SI  - GENBANK/EF140951
SI  - GENBANK/EF140952
SI  - GENBANK/EF140953
SI  - GENBANK/EF140954
SI  - GENBANK/EF140955
SI  - GENBANK/EF140956
SI  - GENBANK/EF140957
SI  - GENBANK/EF140958
SI  - GENBANK/EF140959
SI  - GENBANK/EF140960
SI  - GENBANK/EF140961
SI  - GENBANK/EF140962
SI  - GENBANK/EF140963
SI  - GENBANK/EF140964
SI  - GENBANK/EF140965
SI  - GENBANK/EF140966
SI  - GENBANK/EF140967
SI  - GENBANK/EF140968
SI  - GENBANK/EF140969
SI  - GENBANK/EF140970
SI  - GENBANK/EF140971
SI  - GENBANK/EF140972
SI  - GENBANK/EF140973
SI  - GENBANK/EF140974
SI  - GENBANK/EF140975
SI  - GENBANK/EF140976
SI  - GENBANK/EF140977
SI  - GENBANK/EF140978
SI  - GENBANK/EF140979
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - DNA, Mitochondrial/chemistry
MH  - Evolution, Molecular
MH  - Geography
MH  - Haplotypes
MH  - Molecular Sequence Data
MH  - Osmeriformes/classification/*genetics
MH  - Perciformes/classification/*genetics
MH  - Phylogeny
MH  - *Rivers
MH  - *Water Movements
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777717638 [pii]
AID - 10.1080/10635150701313855 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):271-82. doi: 10.1080/10635150701313855.

PMID- 17464881
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Within-species variation and measurement error in phylogenetic comparative
      methods.
PG  - 252-70
AB  - Most phylogenetically based statistical methods for the analysis of quantitative 
      or continuously varying phenotypic traits assume that variation within species is
      absent or at least negligible, which is unrealistic for many traits.
      Within-species variation has several components. Differences among populations of
      the same species may represent either phylogenetic divergence or direct effects
      of environmental factors that differ among populations (phenotypic plasticity).
      Within-population variation also contributes to within-species variation and
      includes sampling variation, instrument-related error, low repeatability caused
      by fluctuations in behavioral or physiological state, variation related to age,
      sex, season, or time of day, and individual variation within such categories.
      Here we develop techniques for analyzing phylogenetically correlated data to
      include within-species variation, or "measurement error" as it is often termed in
      the statistical literature. We derive methods for (i) univariate analyses,
      including measurement of "phylogenetic signal," (ii) correlation and principal
      components analysis for multiple traits, (iii) multiple regression, and (iv)
      inference of "functional relations," such as reduced major axis (RMA) regression.
      The methods are capable of incorporating measurement error that differs for each 
      data point (mean value for a species or population), but they can be modified for
      special cases in which less is known about measurement error (e.g., when one is
      willing to assume something about the ratio of measurement error in two traits). 
      We show that failure to incorporate measurement error can lead to both biased and
      imprecise (unduly uncertain) parameter estimates. Even previous methods that are 
      thought to account for measurement error, such as conventional RMA regression,
      can be improved by explicitly incorporating measurement error and phylogenetic
      correlation. We illustrate these methods with examples and simulations and
      provide Matlab programs.
FAU - Ives, Anthony R
AU  - Ives AR
AD  - Department of Zoology, University of Wisconsin-Madison, Madison, Wisconsin 53706,
      USA. arives@wisc.edu
FAU - Midford, Peter E
AU  - Midford PE
FAU - Garland, Theodore Jr
AU  - Garland T Jr
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Analysis of Variance
MH  - Animals
MH  - Computational Biology/methods
MH  - Computer Simulation
MH  - *Genetic Variation
MH  - Lizards/genetics
MH  - Phenotype
MH  - *Phylogeny
MH  - Regression Analysis
MH  - Software
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777722583 [pii]
AID - 10.1080/10635150701313830 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):252-70. doi: 10.1080/10635150701313830.

PMID- 17464880
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Multiple cophylogenetic analyses reveal frequent cospeciation between
      pelecaniform birds and Pectinopygus lice.
PG  - 232-51
AB  - Lice in the genus Pectinopygus parasitize a single order of birds
      (Pelecaniformes). To examine the degree of congruence between the phylogenies of 
      17 Pectinopygus species and their pelecaniform hosts, sequences from
      mitochondrial 12S rRNA, 16S rRNA, COI, and nuclear wingless and EF1-alpha genes
      (2290 nucleotides) and from mitochondrial 12S rRNA, COI, and ATPases 8 and 6
      genes (1755 nucleotides) were obtained for the lice and the birds, respectively. 
      Louse data partitions were analyzed for evidence of incongruence and evidence of 
      long-branch attraction prior to cophylogenetic analyses. Host-parasite
      coevolution was studied by different methods: TreeFitter, TreeMap, ParaFit,
      likelihood-ratio test, data-based parsimony method, and correlation of
      coalescence times. All methods agree that there has been extensive cospeciation
      in this host-parasite system, but the results are sensitive to the selection of
      different phylogenetic hypotheses and analytical methods for evaluating
      cospeciation. Perfect congruence between phylogenies is not found in this
      association, probably as a result of occasional host switching by the lice.
      Errors due to phylogenetic reconstruction methods, incorrect or incomplete taxon 
      sampling, or to different loci undergoing different evolutionary histories cannot
      be rejected, thus emphasizing the need for improved cophylogenetic methodologies.
FAU - Hughes, Joseph
AU  - Hughes J
AD  - Department of Environmental and Evolutionary Biology, University of Glasgow, UK. 
      j.hughes@bio.gla.ac.uk
FAU - Kennedy, Martyn
AU  - Kennedy M
FAU - Johnson, Kevin P
AU  - Johnson KP
FAU - Palma, Ricardo L
AU  - Palma RL
FAU - Page, Roderic D M
AU  - Page RD
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Birds/classification/*genetics/parasitology
MH  - Evolution, Molecular
MH  - *Genetic Speciation
MH  - Host-Parasite Interactions
MH  - Phthiraptera/classification/*genetics
MH  - *Phylogeny
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777715191 [pii]
AID - 10.1080/10635150701311370 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):232-51. doi: 10.1080/10635150701311370.

PMID- 17464879
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070427
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Profiling phylogenetic informativeness.
PG  - 222-31
AB  - The resolution of four controversial topics in phylogenetic experimental design
      hinges upon the informativeness of characters about the historical relationships 
      among taxa. These controversies regard the power of different classes of
      phylogenetic character, the relative utility of increased taxonomic versus
      character sampling, the differentiation between lack of phylogenetic signal and a
      historical rapid radiation, and the design of taxonomically broad phylogenetic
      studies optimized by taxonomically sparse genome-scale data. Quantification of
      the informativeness of characters for resolution of phylogenetic hypotheses
      during specified historical epochs is key to the resolution of these
      controversies. Here, such a measure of phylogenetic informativeness is
      formulated. The optimal rate of evolution of a character to resolve a dated
      four-taxon polytomy is derived. By scaling the asymptotic informativeness of a
      character evolving at a nonoptimal rate by the derived asymptotic optimum, and by
      normalizing so that net phylogenetic informativeness is equivalent for all rates 
      when integrated across all of history, an informativeness profile across history 
      is derived. Calculation of the informativeness per base pair allows estimation of
      the cost-effectiveness of character sampling. Calculation of the informativeness 
      per million years allows comparison across historical radiations of the utility
      of a gene for the inference of rapid adaptive radiation. The theory is applied to
      profile the phylogenetic informativeness of the genes BRCA1, RAG1, GHR, and c-myc
      from a muroid rodent sequence data set. Bounded integrations of the phylogenetic 
      profile of these genes over four epochs comprising the diversifications of the
      muroid rodents, the mammals, the lobe-limbed vertebrates, and the early metazoans
      demonstrate the differential power of these genes to resolve the branching order 
      among ancestral lineages. This measure of phylogenetic informativeness yields a
      new kind of information for evaluation of phylogenetic experiments. It conveys
      the utility of the addition of characters a phylogenetic study and it provides a 
      basis for deciding whether appropriate phylogenetic power has been applied to a
      polytomy that is proposed to be a rapid radiation. Moreover, it provides a
      quantitative measure of the capacity of a gene to resolve soft polytomies.
FAU - Townsend, Jeffrey P
AU  - Townsend JP
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven,
      Connecticut 06520, USA. Jeffrey.Townsend@Yale.edu
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Receptors, Somatotropin)
SB  - IM
MH  - Animals
MH  - Evolution, Molecular
MH  - Genes, BRCA1
MH  - Genes, RAG-1
MH  - Genes, myc
MH  - *Phylogeny
MH  - Receptors, Somatotropin/genetics
MH  - Rodentia/genetics
MH  - Sequence Analysis, DNA/*methods
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777708535 [pii]
AID - 10.1080/10635150701311362 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):222-31. doi: 10.1080/10635150701311362.

PMID- 17464878
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070427
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Accounting for gene rate heterogeneity in phylogenetic inference.
PG  - 194-205
AB  - Traditionally, phylogenetic analyses over many genes combine data into a
      contiguous block. Under this concatenated model, all genes are assumed to evolve 
      at the same rate. However, it is clear that genes evolve at very different rates 
      and that accounting for this rate heterogeneity is important if we are to
      accurately infer phylogenies from heterogeneous multigene data sets. There remain
      open questions regarding how best to incorporate gene rate parameters into
      phylogenetic models and which properties of real data correlate with improved fit
      over the concatenated model. In this study, two methods of accounting for gene
      rate heterogeneity are compared: the n-parameter method, which allows for each of
      the n gene partitions to have a gene rate parameter, and the alpha-parameter
      method, which fits a distribution to the gene rates. Results demonstrate that the
      n-parameter method is both computationally faster and in general provides a
      better fit over the concatenated model than the alpha-parameter method.
      Furthermore, improved model fit over the concatenated model is highly correlated 
      with the presence of a gene with a slow relative rate of evolution.
FAU - Bevan, Rachel B
AU  - Bevan RB
AD  - McGill Centre for Bioinformatics, McGill University, Duff Medical Building, 3775 
      University Street, Montreal, Quebec, H3A 2B4, Canada. rbbevan@gmail.com
FAU - Bryant, David
AU  - Bryant D
FAU - Lang, B Franz
AU  - Lang BF
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - Evolution, Molecular
MH  - Genes
MH  - *Genetic Heterogeneity
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2007/04/28 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/28 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/28 09:00 [entrez]
AID - 777715695 [pii]
AID - 10.1080/10635150701291804 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):194-205. doi: 10.1080/10635150701291804.

PMID- 17454975
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070424
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Is multiple-sequence alignment required for accurate inference of phylogeny?
PG  - 206-21
AB  - The process of inferring phylogenetic trees from molecular sequences almost
      always starts with a multiple alignment of these sequences but can also be based 
      on methods that do not involve multiple sequence alignment. Very little is known 
      about the accuracy with which such alignment-free methods recover the correct
      phylogeny or about the potential for increasing their accuracy. We conducted a
      large-scale comparison of ten alignment-free methods, among them one new approach
      that does not calculate distances and a faster variant of our pattern-based
      approach; all distance-based alignment-free methods are freely available from
      http://www.bioinformatics.org.au (as Python package decaf+py). We show that most 
      methods exhibit a higher overall reconstruction accuracy in the presence of high 
      among-site rate variation. Under all conditions that we considered, variants of
      the pattern-based approach were significantly better than the other
      alignment-free methods. The new pattern-based variant achieved a speed-up of an
      order of magnitude in the distance calculation step, accompanied by a small loss 
      of tree reconstruction accuracy. A method of Bayesian inference from k-mers did
      not improve on classical alignment-free (and distance-based) methods but may
      still offer other advantages due to its Bayesian nature. We found the optimal
      word length k of word-based methods to be stable across various data sets, and we
      provide parameter ranges for two different alphabets. The influence of these
      alphabets was analyzed to reveal a trade-off in reconstruction accuracy between
      long and short branches. We have mapped the phylogenetic accuracy for many
      alignment-free methods, among them several recently introduced ones, and
      increased our understanding of their behavior in response to biologically
      important parameters. In all experiments, the pattern-based approach emerged as
      superior, at the expense of higher resource consumption. Nonetheless, no
      alignment-free method that we examined recovers the correct phylogeny as
      accurately as does an approach based on maximum-likelihood distance estimates of 
      multiply aligned sequences.
FAU - Hohl, Michael
AU  - Hohl M
AD  - Australian Research Council Centre in Bioinformatics, and Institute for Molecular
      Bioscience, The University of Queensland, Brisbane, QLD 4072, Australia.
FAU - Ragan, Mark A
AU  - Ragan MA
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Computational Biology/methods
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
MH  - *Sequence Alignment
MH  - Sequence Analysis/*methods
EDAT- 2007/04/25 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/25 09:00
PHST- 2007/04/25 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/25 09:00 [entrez]
AID - 776485575 [pii]
AID - 10.1080/10635150701294741 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):206-21. doi: 10.1080/10635150701294741.

PMID- 17454974
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20090416
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Alignment and topological accuracy of the direct optimization approach via POY
      and traditional phylogenetics via ClustalW + PAUP*.
PG  - 182-93
AB  - Direct optimization frameworks for simultaneously estimating alignments and
      phylogenies have recently been developed. One such method, implemented in the
      program POY, is becoming more common for analyses of variable length sequences
      (e.g., analyses using ribosomal genes) and for combined evidence analyses
      (morphology + multiple genes). Simulation of sequences containing insertion and
      deletion events was performed in order to directly compare a widely used method
      of multiple sequence alignment (ClustalW) and subsequent parsimony analysis in
      PAUP* with direct optimization via POY. Data sets were simulated for pectinate,
      balanced, and random tree shapes under different conditions (clocklike,
      non-clocklike, and ultrametric). Alignment accuracy scores for the implied
      alignments from POY and the multiple sequence alignments from ClustalW were
      calculated and compared. In almost all cases (99.95%), ClustalW produced more
      accurate alignments than POY-implied alignments, judged by the proportion of
      correctly identified homologous sites. Topological accuracy (distance to the true
      tree) for POY topologies and topologies generated under parsimony in PAUP* from
      the ClustalW alignments were also compared. In 44.94% of the cases, Clustal
      alignment tree reconstructions via PAUP* were more accurate than POY, whereas in 
      16.71% of the cases POY reconstructions were more topologically accurate (38.38% 
      of the time they were equally accurate). Comparisons between POY hypothesized
      alignments and the true alignments indicated that, on average, as alignment error
      increased, topological accuracy decreased.
FAU - Ogden, T Heath
AU  - Ogden TH
AD  - Department of Biological Sciences, Idaho State University, Idaho 83209, USA.
      ogdet@isu.edu
FAU - Rosenberg, Michael S
AU  - Rosenberg MS
LA  - eng
GR  - R03-LM008637/LM/NLM NIH HHS/United States
PT  - Comparative Study
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2008 Aug;57(4):653-7. PMID: 18709601
MH  - Cluster Analysis
MH  - *Computer Simulation
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Sequence Alignment/*methods
MH  - *Sequence Analysis, DNA
MH  - *Software
EDAT- 2007/04/25 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/25 09:00
PHST- 2007/04/25 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/25 09:00 [entrez]
AID - 776485446 [pii]
AID - 10.1080/10635150701281102 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):182-93. doi: 10.1080/10635150701281102.

PMID- 17454973
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070424
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Widespread genealogical nonmonophyly in species of Pinus subgenus Strobus.
PG  - 163-81
AB  - Phylogenetic relationships among Pinus species from subgenus Strobus remain
      unresolved despite combined efforts based on nrITS and cpDNA. To provide greater 
      resolution among these taxa, a 900-bp intron from a late embryogenesis abundant
      (LEA)-like gene (IFG8612)was sequenced from 39 pine species, with two or more
      alleles representing 33 species. Nineteen of 33 species exhibited allelic
      nonmonphyly in the strict consensus tree, and 10 deviated significantly from
      allelic monophyly based on topology incongruence tests. Intraspecific nucleotide 
      diversity ranged from 0.0 to 0.0211, and analysis of variance shows that
      nucleotide diversity was strongly associated (P &lt; 0.0001)with the degree of
      species monophyly. Although species nonmonophyly complicates phylogenetic
      interpretations, this nuclear locus offers greater topological support than
      previously observed for cpDNA or nrITS. Lacking evidence for hybridization,
      recombination, or imperfect taxonomy, we feel that incomplete lineage sorting
      remains the best explanation for the polymorphisms shared among species.
      Depending on the species, coalescent expectations indicate that reciprocal
      monophyly will be more likely than paraphyly in 1.71 to 24.0 x 10(6) years, and
      that complete genome-wide coalescence in these species may require up to 76.3 x
      10(6) years. The absence of allelic coalescence is a severe constraint in the
      application of phylogenetic methods in Pinus, and taxa sharing similar life
      history traits with Pinus are likely to show species nonmonophyly using nuclear
      markers.
FAU - Syring, John
AU  - Syring J
AD  - Department of Biological and Physical Sciences, Montana State
      University-Billings, Billings, Montana 59101, USA. jsyring@msubillings.edu
FAU - Farrell, Kathleen
AU  - Farrell K
FAU - Businsky, Roman
AU  - Businsky R
FAU - Cronn, Richard
AU  - Cronn R
FAU - Liston, Aaron
AU  - Liston A
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
RN  - 0 (Plant Proteins)
SB  - IM
MH  - Alleles
MH  - Genetic Markers
MH  - Genome, Plant
MH  - Introns
MH  - *Phylogeny
MH  - Pinus/*classification/genetics
MH  - Plant Proteins/chemistry/*genetics
MH  - Recombination, Genetic
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
EDAT- 2007/04/25 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/25 09:00
PHST- 2007/04/25 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/25 09:00 [entrez]
AID - 773635072 [pii]
AID - 10.1080/10635150701258787 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):163-81. doi: 10.1080/10635150701258787.

PMID- 17454972
OWN - NLM
STAT- MEDLINE
DCOM- 20070712
LR  - 20070424
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 2
DP  - 2007 Apr
TI  - Estimation of phylogeny and invariant sites under the general Markov model of
      nucleotide sequence evolution.
PG  - 155-62
AB  - The models of nucleotide substitution used by most maximum likelihood-based
      methods assume that the evolutionary process is stationary, reversible, and
      homogeneous. We present an extension of the Barry and Hartigan model, which can
      be used to estimate parameters by maximum likelihood (ML) when the data contain
      invariant sites and there are violations of the assumptions of stationarity,
      reversibility, and homogeneity. Unlike most ML methods for estimating invariant
      sites, we estimate the nucleotide composition of invariant sites separately from 
      that of variable sites. We analyze a bacterial data set where problems due to
      lack of stationarity and homogeneity have been previously well noted and use the 
      parametric bootstrap to show that the data are consistent with our general Markov
      model. We also show that estimates of invariant sites obtained using our method
      are fairly accurate when applied to data simulated under the general Markov
      model.
FAU - Jayaswal, Vivek
AU  - Jayaswal V
AD  - Sydney Bioinformatics, University of Sydney, NSW 2006, Australia.
FAU - Robinson, John
AU  - Robinson J
FAU - Jermiin, Lars
AU  - Jermiin L
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 16S)
SB  - IM
MH  - Bacillus subtilis/classification/genetics
MH  - Bacteria/classification/*genetics
MH  - Base Sequence
MH  - Deinococcus/classification/genetics
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - Markov Chains
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - RNA, Ribosomal, 16S/chemistry
MH  - Sequence Analysis, DNA
MH  - Thermotoga maritima/classification/genetics
MH  - Thermus thermophilus/classification/genetics
EDAT- 2007/04/25 09:00
MHDA- 2007/07/13 09:00
CRDT- 2007/04/25 09:00
PHST- 2007/04/25 09:00 [pubmed]
PHST- 2007/07/13 09:00 [medline]
PHST- 2007/04/25 09:00 [entrez]
AID - 773634848 [pii]
AID - 10.1080/10635150701247921 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Apr;56(2):155-62. doi: 10.1080/10635150701247921.

PMID- 17366144
OWN - NLM
STAT- MEDLINE
DCOM- 20070427
LR  - 20070316
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 1
DP  - 2007 Feb
TI  - Opinions on multiple sequence alignment, and an empirical comparison of
      repeatability and accuracy between POY and structural alignment.
PG  - 133-46
FAU - Kjer, Karl M
AU  - Kjer KM
AD  - Department of Ecology, Evolution and Natural Resources, Rutgers University, New
      Brunswick, New Jersey 08901, USA. kjer@aesop.rutgers.edu
FAU - Gillespie, Joseph J
AU  - Gillespie JJ
FAU - Ober, Karen A
AU  - Ober KA
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Mammals/genetics
MH  - *Phylogeny
MH  - Sequence Alignment/*methods
MH  - *Sequence Homology, Nucleic Acid
MH  - *Software
EDAT- 2007/03/17 09:00
MHDA- 2007/04/28 09:00
CRDT- 2007/03/17 09:00
PHST- 2007/03/17 09:00 [pubmed]
PHST- 2007/04/28 09:00 [medline]
PHST- 2007/03/17 09:00 [entrez]
AID - 771652669 [pii]
AID - 10.1080/10635150601156305 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Feb;56(1):133-46. doi: 10.1080/10635150601156305.

PMID- 17366143
OWN - NLM
STAT- MEDLINE
DCOM- 20070427
LR  - 20070316
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 1
DP  - 2007 Feb
TI  - Time to standardize taxonomies.
PG  - 130-3
FAU - Avise, John C
AU  - Avise JC
AD  - Department of Ecology and Evolutionary Biology, University of California, Irvine,
      California 92697, USA. javise@uci.edu
FAU - Mitchell, Dale
AU  - Mitchell D
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Phylogeny
MH  - *Reference Standards
MH  - Time Factors
EDAT- 2007/03/17 09:00
MHDA- 2007/04/28 09:00
CRDT- 2007/03/17 09:00
PHST- 2007/03/17 09:00 [pubmed]
PHST- 2007/04/28 09:00 [medline]
PHST- 2007/03/17 09:00 [entrez]
AID - 771652029 [pii]
AID - 10.1080/10635150601145365 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Feb;56(1):130-3. doi: 10.1080/10635150601145365.

PMID- 17366142
OWN - NLM
STAT- MEDLINE
DCOM- 20070427
LR  - 20070316
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 1
DP  - 2007 Feb
TI  - Investigating stagnation in morphological phylogenetics using consensus data.
PG  - 125-9
FAU - Harris, Simon R
AU  - Harris SR
AD  - Department of Zoology, The Natural History Museum, London, SW7 5BD, UK.
      simon.harris@ncl.ac.uk
FAU - Pisani, Davide
AU  - Pisani D
FAU - Gower, David J
AU  - Gower DJ
FAU - Wilkinson, Mark
AU  - Wilkinson M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Anatomy, Comparative/*methods
MH  - Animals
MH  - Bone and Bones/anatomy &amp; histology
MH  - Classification/*methods
MH  - *Phylogeny
MH  - Research Design
MH  - Turtles/*anatomy &amp; histology
EDAT- 2007/03/17 09:00
MHDA- 2007/04/28 09:00
CRDT- 2007/03/17 09:00
PHST- 2007/03/17 09:00 [pubmed]
PHST- 2007/04/28 09:00 [medline]
PHST- 2007/03/17 09:00 [entrez]
AID - 771654664 [pii]
AID - 10.1080/10635150601115624 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Feb;56(1):125-9. doi: 10.1080/10635150601115624.

PMID- 17366141
OWN - NLM
STAT- MEDLINE
DCOM- 20070427
LR  - 20070316
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 1
DP  - 2007 Feb
TI  - Pitfalls of heterogeneous processes for phylogenetic reconstruction.
PG  - 113-24
AB  - Different genes often have different phylogenetic histories. Even within regions 
      having the same phylogenetic history, the mutation rates often vary. We
      investigate the prospects of phylogenetic reconstruction when all the characters 
      are generated from the same tree topology, but the branch lengths vary (with
      possibly different tree shapes). Furthering work of Kolaczkowski and Thornton
      (2004, Nature 431: 980-984) and Chang (1996, Math. Biosci. 134: 189-216), we show
      examples where maximum likelihood (under a homogeneous model) is an inconsistent 
      estimator of the tree. We then explore the prospects of phylogenetic inference
      under a heterogeneous model. In some models, there are examples where
      phylogenetic inference under any method is impossible - despite the fact that
      there is a common tree topology. In particular, there are nonidentifiable mixture
      distributions, i.e., multiple topologies generate identical mixture
      distributions. We address which evolutionary models have nonidentifiable mixture 
      distributions and prove that the following duality theorem holds for most DNA
      substitution models. The model has either: (i) nonidentifiability - two different
      tree topologies can produce identical mixture distributions, and hence
      distinguishing between the two topologies is impossible; or (ii) linear tests -
      there exist linear tests which identify the common tree topology for character
      data generated by a mixture distribution. The theorem holds for models whose
      transition matrices can be parameterized by open sets, which includes most of the
      popular models, such as Tamura-Nei and Kimura's 2-parameter model. The duality
      theorem relies on our notion of linear tests, which are related to Lake's linear 
      invariants.
FAU - Stefankovic, Daniel
AU  - Stefankovic D
AD  - Department of Computer Science, University of Rochester, Rochester, New York
      14627, USA. stefanko@cs.rochester.edu
FAU - Vigoda, Eric
AU  - Vigoda E
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Genes/*genetics
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2007/03/17 09:00
MHDA- 2007/04/28 09:00
CRDT- 2007/03/17 09:00
PHST- 2007/03/17 09:00 [pubmed]
PHST- 2007/04/28 09:00 [medline]
PHST- 2007/03/17 09:00 [entrez]
AID - 771652132 [pii]
AID - 10.1080/10635150701245388 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Feb;56(1):113-24. doi: 10.1080/10635150701245388.

PMID- 17366140
OWN - NLM
STAT- MEDLINE
DCOM- 20070427
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 1
DP  - 2007 Feb
TI  - Multistate characters and diet shifts: evolution of Erotylidae (Coleoptera).
PG  - 97-112
AB  - The dominance of angiosperms has played a direct role in the diversification of
      insects, especially Coleoptera. The shift to angiosperm feeding from other diets 
      is likely to have increased the rate of speciation in Phytophaga. However,
      Phytophaga is only one of many hyperdiverse lineages of beetles and studies of
      host-shift proliferation have been somewhat limited to groups that primitively
      feed on plants. We have studied the diet-diverse beetle family Erotylidae
      (Cucujoidea) to determine if diet is correlated with high diversification rates
      and morphological evolution by first reconstructing ancestral diets and then
      testing for associations between diet and species number and diet and ovipositor 
      type. A Bayesian phylogenetic analysis of morphological data that was previously 
      published in Leschen (2003, Pages 1-108 in Fauna of New Zealand, 47; 53 terminal 
      taxa and 1 outgroup, 120 adult characters and 1 diet character) yielded results
      that are similar to the parsimony analyses of Leschen (2003). Ancestral state
      reconstructions based on Bayesian and parsimony inference were largely congruent 
      and both reconstructed microfungal feeding (the diet of the outgroup Biphyllidae)
      at the root of the Erotylidae tree. Shifts among microfungal, saprophagous, and
      phytophagous diets were most frequent. The largest numbers of species are
      contained in lineages that are macrofungal feeders (subfamily Erotylinae) and
      phytophagous (derived Languriinae), although the Bayesian posterior predictive
      tests of character state correlation were unable to detect any significant
      associations. Ovipositor morphology correlated with diet (i.e., acute forms were 
      associated with phytophagy and unspecialized forms were associated with a mixture
      of diets). Although there is a general trend to increased species number
      associated with the shift from microfungal feeding to phytophagy (based on
      character mapping and mainly restricted to shifts in Languriinae), there is a
      large radiation of taxa feeding on macrofungi. Cycad feeding is scattered in more
      deeply diverged taxa and may have preceded the evolution of angiosperm feeding in
      some groups. Preliminary analysis of diet mapped onto higher beetle phylogenies
      suggests that about half of the major Coleoptera lineages may have had
      fungus-feeding ancestors. We discuss the roles of stochastic models and prior
      distributions of the reconstruction of ancestral character states in the context 
      of the current data.
FAU - Leschen, Richard A B
AU  - Leschen RA
AD  - New Zealand Arthropod Collection, Landcare Research, drivate Bag 92170, Auckland,
      New Zealand. leschenr@landcareresearch.co.nz
FAU - Buckley, Thomas R
AU  - Buckley TR
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Coleoptera/*anatomy &amp; histology/*genetics/*physiology
MH  - *Diet
MH  - *Genetic Speciation
MH  - Models, Statistical
MH  - Oviposition/*physiology
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2007/03/17 09:00
MHDA- 2007/04/28 09:00
CRDT- 2007/03/17 09:00
PHST- 2007/03/17 09:00 [pubmed]
PHST- 2007/04/28 09:00 [medline]
PHST- 2007/03/17 09:00 [entrez]
AID - 772601895 [pii]
AID - 10.1080/10635150701211844 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Feb;56(1):97-112. doi: 10.1080/10635150701211844.

PMID- 17366139
OWN - NLM
STAT- MEDLINE
DCOM- 20070427
LR  - 20070316
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 1
DP  - 2007 Feb
TI  - Base-compositional heterogeneity in the RAG1 locus among didelphid marsupials:
      implications for phylogenetic inference and the evolution of GC content.
PG  - 83-96
AB  - Although theoretical studies have suggested that base-compositional heterogeneity
      can adversely affect phylogenetic reconstruction, only a few empirical examples
      of this phenomenon, mostly among ancient lineages (with divergence dates &gt; 100
      Mya), have been reported. In the course of our phylogenetic research on the New
      World marsupial family Didelphidae, we sequenced 2790 bp of the RAG1 exon from
      exemplar species of most extant genera. Phylogenetic analysis of these sequences 
      recovered an anomalous node consisting of two clades previously shown to be
      distantly related based on analyses of other molecular data. These two clades
      show significantly increased GC content at RAG1 third codon positions, and the
      resulting convergence in base composition is strong enough to overwhelm
      phylogenetic signal from other genes (and morphology) in most analyses of
      concatenated datasets. This base-compositional convergence occurred relatively
      recently (over tens rather than hundreds of millions of years), and the affected 
      gene region is still in a state of evolutionary disequilibrium. Both mutation
      rate and substitution rate are higher in GC-rich didelphid taxa, observations
      consistent with RAG1 sequences having experienced a higher rate of recombination 
      in the convergent lineages.
FAU - Gruber, Karl F
AU  - Gruber KF
AD  - Bell Museum of Natural History and Department of Ecology, Evolution, and
      Behavior, University of Minnesota, St. Paul, Minnesota 55108, USA.
FAU - Voss, Robert S
AU  - Voss RS
FAU - Jansa, Sharon A
AU  - Jansa SA
LA  - eng
SI  - GENBANK/DQ865883
SI  - GENBANK/DQ865884
SI  - GENBANK/DQ865885
SI  - GENBANK/DQ865886
SI  - GENBANK/DQ865887
SI  - GENBANK/DQ865888
SI  - GENBANK/DQ865889
SI  - GENBANK/DQ865890
SI  - GENBANK/DQ865891
SI  - GENBANK/DQ865892
SI  - GENBANK/DQ865893
SI  - GENBANK/DQ865894
SI  - GENBANK/DQ865895
SI  - GENBANK/DQ865896
SI  - GENBANK/DQ865897
SI  - GENBANK/DQ865898
SI  - GENBANK/DQ865899
SI  - GENBANK/DQ865900
SI  - GENBANK/DQ865901
SI  - GENBANK/DQ865902
SI  - GENBANK/DQ865903
SI  - GENBANK/DQ865904
SI  - GENBANK/DQ865905
SI  - GENBANK/DQ865906
SI  - GENBANK/DQ865907
SI  - GENBANK/DQ865908
SI  - GENBANK/DQ865909
SI  - GENBANK/DQ865910
SI  - GENBANK/DQ865911
SI  - GENBANK/DQ865912
SI  - GENBANK/DQ865913
SI  - GENBANK/DQ865914
SI  - GENBANK/DQ865915
SI  - GENBANK/DQ865916
SI  - GENBANK/DQ865917
SI  - GENBANK/DQ865918
SI  - GENBANK/DQ865919
SI  - GENBANK/DQ865920
SI  - GENBANK/DQ865921
SI  - GENBANK/DQ865922
SI  - GENBANK/DQ865923
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (Homeodomain Proteins)
RN  - 128559-51-3 (RAG-1 protein)
SB  - IM
MH  - Animals
MH  - Base Composition/*genetics
MH  - Base Sequence
MH  - DNA Primers
MH  - *Evolution, Molecular
MH  - Gene Conversion/genetics
MH  - Homeodomain Proteins/*genetics
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Mutation/genetics
MH  - Opossums/*genetics
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2007/03/17 09:00
MHDA- 2007/04/28 09:00
CRDT- 2007/03/17 09:00
PHST- 2007/03/17 09:00 [pubmed]
PHST- 2007/04/28 09:00 [medline]
PHST- 2007/03/17 09:00 [entrez]
AID - 772607604 [pii]
AID - 10.1080/10635150601182939 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Feb;56(1):83-96. doi: 10.1080/10635150601182939.

PMID- 17366138
OWN - NLM
STAT- MEDLINE
DCOM- 20070427
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 1
DP  - 2007 Feb
TI  - Artifactual phylogenies caused by correlated distribution of substitution rates
      among sites and lineages: the good, the bad, and the ugly.
PG  - 68-82
AB  - Despite the advances in understanding molecular evolution, current phylogenetic
      methods barely take account of a fraction of the complexity of evolution. We are 
      chiefly constrained by our incomplete knowledge of molecular evolutionary
      processes and the limits of computational power. These limitations lead to the
      establishment of either biologically simplistic models that rarely account for a 
      fraction of the complexity involved or overfitting models that add little
      resolution to the problem. Such oversimplified models may lead us to assign high 
      confidence to an incorrect tree (inconsistency). Rate-across-site (RAS) models
      are commonly used evolutionary models in phylogenetic studies. These account for 
      heterogeneity in the evolutionary rates among sites but do not account for
      changing within-site rates across lineages (heterotachy). If heterotachy is
      common, using RAS models may lead to systematic errors in tree inference. In this
      work we show possible misleading effects in tree inference when the assumption of
      constant within-site rates across lineages is violated using maximum likelihood. 
      Using a simulation study, we explore the ways in which gamma stationary models
      can lead to wrong topology or to deceptive bootstrap support values when the
      within-site rates change across lineages. More precisely, we show that different 
      degrees of heterotachy mislead phylogenetic inference when the model assumed is
      stationary. Finally, we propose a geometry-based approach to visualize and to
      test for the possible existence of bias due to heterotachy.
FAU - Ruano-Rubio, Valentin
AU  - Ruano-Rubio V
AD  - Molecular Evolution and Bioinformatics Laboratory, Department of Biology,
      National University of Ireland, Maynooth, Ireland.
FAU - Fares, Mario A
AU  - Fares MA
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bias
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Research Design
EDAT- 2007/03/17 09:00
MHDA- 2007/04/28 09:00
CRDT- 2007/03/17 09:00
PHST- 2007/03/17 09:00 [pubmed]
PHST- 2007/04/28 09:00 [medline]
PHST- 2007/03/17 09:00 [entrez]
AID - 771129557 [pii]
AID - 10.1080/10635150601175578 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Feb;56(1):68-82. doi: 10.1080/10635150601175578.

PMID- 17366137
OWN - NLM
STAT- MEDLINE
DCOM- 20070427
LR  - 20070316
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 1
DP  - 2007 Feb
TI  - Imputing supertrees and supernetworks from quartets.
PG  - 57-67
AB  - Inferring species phylogenies is an important part of understanding molecular
      evolution. Even so, it is well known that an accurate phylogenetic tree
      reconstruction for a single gene does not always necessarily correspond to the
      species phylogeny. One commonly accepted strategy to cope with this problem is to
      sequence many genes; the way in which to analyze the resulting collection of
      genes is somewhat more contentious. Supermatrix and supertree methods can be
      used, although these can suppress conflicts arising from true differences in the 
      gene trees caused by processes such as lineage sorting, horizontal gene transfer,
      or gene duplication and loss. In 2004, Huson et al. (IEEE/ACM Trans. Comput.
      Biol. Bioinformatics 1:151-158) presented the Z-closure method that can
      circumvent this problem by generating a supernetwork as opposed to a supertree.
      Here we present an alternative way for generating supernetworks called
      Q-imputation. In particular, we describe a method that uses quartet information
      to add missing taxa into gene trees. The resulting trees are subsequently used to
      generate consensus networks, networks that generalize strict and majority-rule
      consensus trees. Through simulations and application to real data sets, we
      compare Q-imputation to the matrix representation with parsimony (MRP) supertree 
      method and Z-closure, and demonstrate that it provides a useful complementary
      tool.
FAU - Holland, B
AU  - Holland B
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, Massey University,
      Palmerston North, New Zealand. B.R.Holland@massey.ac.nz
FAU - Conner, Glenn
AU  - Conner G
FAU - Huber, Katharina
AU  - Huber K
FAU - Moulton, V
AU  - Moulton V
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2007/03/17 09:00
MHDA- 2007/04/28 09:00
CRDT- 2007/03/17 09:00
PHST- 2007/03/17 09:00 [pubmed]
PHST- 2007/04/28 09:00 [medline]
PHST- 2007/03/17 09:00 [entrez]
AID - 771130031 [pii]
AID - 10.1080/10635150601167013 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Feb;56(1):57-67. doi: 10.1080/10635150601167013.

PMID- 17366136
OWN - NLM
STAT- MEDLINE
DCOM- 20070427
LR  - 20071203
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 1
DP  - 2007 Feb
TI  - A step toward barcoding life: a model-based, decision-theoretic method to assign 
      genes to preexisting species groups.
PG  - 44-56
AB  - A major part of the barcoding of life problem is assigning newly sequenced or
      sampled individuals to existing groups that are preidentified externally (by a
      taxonomist, for example). This problem involves evaluating the statistical
      evidence towards associating a sequence from a new individual with one group or
      another. The main concern of our current research is to perform this task in a
      fast and accurate manner. To accomplish this we have developed a model-based,
      decision-theoretic framework based on the coalescent theory. Under this
      framework, we utilized both distance and the posterior probability of a group,
      given the sequences from members of this group and the sequence from a newly
      sampled individual to assign this new individual. We believe that this approach
      makes efficient use of the available information in the data. Our preliminary
      results indicated that this approach is more accurate than using a simple measure
      of distance for assignment.
FAU - Abdo, Zaid
AU  - Abdo Z
AD  - Department of Mathematics, University of Idaho, Moscow, Idaho 83844, USA.
FAU - Golding, G Brian
AU  - Golding GB
LA  - eng
GR  - P20 RR16448/RR/NCRR NIH HHS/United States
GR  - P20 RR16454/RR/NCRR NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biodiversity
MH  - Butterflies/genetics
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Decision Theory
MH  - Genes/*genetics
MH  - *Models, Genetic
MH  - Species Specificity
EDAT- 2007/03/17 09:00
MHDA- 2007/04/28 09:00
CRDT- 2007/03/17 09:00
PHST- 2007/03/17 09:00 [pubmed]
PHST- 2007/04/28 09:00 [medline]
PHST- 2007/03/17 09:00 [entrez]
AID - 771129755 [pii]
AID - 10.1080/10635150601167005 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Feb;56(1):44-56. doi: 10.1080/10635150601167005.

PMID- 17366135
OWN - NLM
STAT- MEDLINE
DCOM- 20070427
LR  - 20070316
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 1
DP  - 2007 Feb
TI  - Conditioned genome reconstruction: how to avoid choosing the conditioning genome.
PG  - 25-43
AB  - Genome phylogenies can be inferred from data on the presence and absence of genes
      across taxa. Logdet distances may be a good method, because they allow expected
      genome size to vary across the tree. Recently, Lake and Rivera proposed
      conditioned genome reconstruction (calculation of logdet distances using only
      those genes present in a conditioning genome) to deal with unobservable genes
      that are absent from every taxon of interest. We prove that their method can
      consistently estimate the topology for almost any choice of conditioning genome. 
      Nevertheless, the choice of conditioning genome is important for small samples.
      For real bacterial genome data, different choices of conditioning genome can
      result in strong bootstrap support for different tree topologies. To overcome
      this problem, we developed supertree methods that combine information from all
      choices of conditioning genome. One of these methods, based on the BIONJ
      algorithm, performs well on simulated data and may have applications to other
      supertree problems. However, an analysis of 40 bacterial genomes using this
      method supports an incorrect clade of parasites. This is a common feature of
      model-based gene content methods and is due to parallel gene loss.
FAU - Spencer, Matthew
AU  - Spencer M
AD  - Department of Mathematics and Statistics, Dalhousie University, Hali, Nova
      Scotia, B3H 3J5, Canada. m.spenser@liverpool.ac.uk
FAU - Bryant, David
AU  - Bryant D
FAU - Susko, Edward
AU  - Susko E
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Bacteria/genetics
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Genes/*genetics
MH  - Genome/*genetics
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2007/03/17 09:00
MHDA- 2007/04/28 09:00
CRDT- 2007/03/17 09:00
PHST- 2007/03/17 09:00 [pubmed]
PHST- 2007/04/28 09:00 [medline]
PHST- 2007/03/17 09:00 [entrez]
AID - 771129819 [pii]
AID - 10.1080/10635150601156313 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Feb;56(1):25-43. doi: 10.1080/10635150601156313.

PMID- 17366134
OWN - NLM
STAT- MEDLINE
DCOM- 20070427
LR  - 20071203
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 1
DP  - 2007 Feb
TI  - Inconsistency of phylogenetic estimates from concatenated data under coalescence.
PG  - 17-24
AB  - Although multiple gene sequences are becoming increasingly available for
      molecular phylogenetic inference, the analysis of such data has largely relied on
      inference methods designed for single genes. One of the common approaches to
      analyzing data from multiple genes is concatenation of the individual gene data
      to form a single supergene to which traditional phylogenetic inference procedures
      - e.g., maximum parsimony (MP) or maximum likelihood (ML) - are applied. Recent
      empirical studies have demonstrated that concatenation of sequences from multiple
      genes prior to phylogenetic analysis often results in inference of a single,
      well-supported phylogeny. Theoretical work, however, has shown that the
      coalescent can produce substantial variation in single-gene histories. Using
      simulation, we combine these ideas to examine the performance of the
      concatenation approach under conditions in which the coalescent produces a high
      level of discord among individual gene trees and show that it leads to
      statistically inconsistent estimation in this setting. Furthermore, use of the
      bootstrap to measure support for the inferred phylogeny can result in moderate to
      strong support for an incorrect tree under these conditions. These results
      highlight the importance of incorporating variation in gene histories into
      multilocus phylogenetics.
FAU - Kubatko, Laura Salter
AU  - Kubatko LS
AD  - Department of Statistics and Evolution, The Ohio State University, Columbus, Ohio
      43210, USA. lkubatko@stat.ohio-state.edu
FAU - Degnan, James H
AU  - Degnan JH
LA  - eng
GR  - MH59532/MH/NIMH NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Genes/*genetics
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Research Design
EDAT- 2007/03/17 09:00
MHDA- 2007/04/28 09:00
CRDT- 2007/03/17 09:00
PHST- 2007/03/17 09:00 [pubmed]
PHST- 2007/04/28 09:00 [medline]
PHST- 2007/03/17 09:00 [entrez]
AID - 771129708 [pii]
AID - 10.1080/10635150601146041 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Feb;56(1):17-24. doi: 10.1080/10635150601146041.

PMID- 17366133
OWN - NLM
STAT- MEDLINE
DCOM- 20070427
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 56
IP  - 1
DP  - 2007 Feb
TI  - From phylogenetics to phylogenomics: the evolutionary relationships of insect
      endosymbiotic gamma-Proteobacteria as a test case.
PG  - 1-16
AB  - The increasing availability of complete genome sequences and the development of
      new, faster methods for phylogenetic reconstruction allow the exploration of the 
      set of evolutionary trees for each gene in the genome of any species. This has
      led to the development of new phylogenomic methods. Here, we have compared
      different phylogenetic and phylogenomic methods in the analysis of the
      monophyletic origin of insect endosymbionts from the gamma-Proteobacteria, a
      hotly debated issue with several recent, conflicting reports. We have obtained
      the phylogenetic tree for each of the 579 identified protein-coding genes in the 
      genome of the primary endosymbiont of carpenter ants, Blochmannia floridanus,
      after determining their presumed orthologs in 20 additional Proteobacteria
      genomes. A reference phylogeny reflecting the monophyletic origin of insect
      endosymbionts was further confirmed with different approaches, which led us to
      consider it as the presumed species tree. Remarkably, only 43 individual genes
      produced exactly the same topology as this presumed species tree. Most
      discrepancies between this tree and those obtained from individual genes or by
      concatenation of different genes were due to the grouping of Xanthomonadales with
      beta-Proteobacteria and not to uncertainties over the monophyly of insect
      endosymbionts. As previously noted, operational genes were more prone to reject
      the presumed species tree than those included in information-processing
      categories, but caution should be exerted when selecting genes for phylogenetic
      inference on the basis of their functional category assignment. We have obtained 
      strong evidence in support of the monophyletic origin of gamma-Proteobacteria
      insect endosymbionts by a combination of phylogenetic and phylogenomic methods.
      In our analysis, the use of concatenated genes has shown to be a valuable tool
      for analyzing primary phylogenetic signals coded in the genomes. Nevertheless,
      other phylogenomic methods such as supertree approaches were useful in revealing 
      alternative phylogenetic signals and should be included in comprehensive
      phylogenomic studies.
FAU - Comas, Inaki
AU  - Comas I
AD  - Institut Cavanilles de Biodiversitat i Biologia Evolutiva, Universitat de
      Valencia, Valencia, Spain. inaki.comas@uv.es
FAU - Moya, Andres
AU  - Moya A
FAU - Gonzalez-Candelas, Fernando
AU  - Gonzalez-Candelas F
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Gammaproteobacteria/*genetics
MH  - Genes, Bacterial/*genetics
MH  - Genome/*genetics
MH  - Genomics/*methods
MH  - Insecta/*microbiology
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2007/03/17 09:00
MHDA- 2007/04/28 09:00
CRDT- 2007/03/17 09:00
PHST- 2007/03/17 09:00 [pubmed]
PHST- 2007/04/28 09:00 [medline]
PHST- 2007/03/17 09:00 [entrez]
AID - 771129911 [pii]
AID - 10.1080/10635150601109759 [doi]
PST - ppublish
SO  - Syst Biol. 2007 Feb;56(1):1-16. doi: 10.1080/10635150601109759.

PMID- 17345679
OWN - NLM
STAT- MEDLINE
DCOM- 20070412
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 6
DP  - 2006 Dec
TI  - Accurate branch length estimation in partitioned Bayesian analyses requires
      accommodation of among-partition rate variation and attention to branch length
      priors.
PG  - 993-1003
FAU - Marshall, David C
AU  - Marshall DC
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, U-3043, Storrs, Connecticut 06269, USA. david.marshall@uconn.edu
FAU - Simon, Chris
AU  - Simon C
FAU - Buckley, Thomas R
AU  - Buckley TR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Computer Simulation
MH  - *DNA, Mitochondrial
MH  - Hemiptera/*genetics
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2007/03/10 09:00
MHDA- 2007/04/14 09:00
CRDT- 2007/03/10 09:00
PHST- 2007/03/10 09:00 [pubmed]
PHST- 2007/04/14 09:00 [medline]
PHST- 2007/03/10 09:00 [entrez]
AID - 10.1080/10635150601087641 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Dec;55(6):993-1003. doi: 10.1080/10635150601087641.

PMID- 17345678
OWN - NLM
STAT- MEDLINE
DCOM- 20070412
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 6
DP  - 2006 Dec
TI  - Nematode small subunit phylogeny correlates with alignment parameters.
PG  - 972-92
AB  - The number of nuclear small subunit (SSU) ribosomal RNA (rRNA) sequences for
      Nematoda has increased dramatically in recent years, and although their use in
      constructing phylogenies has also increased, relatively little attention has been
      given to their alignment. Here we examined the sensitivity of the nematode SSU
      data set to different alignment parameters and to the removal of alignment
      ambiguous regions. Ten alignments were created with CLUSTAL W using different
      sets of alignment parameters (10 full alignments), and each alignment was
      examined by eye and alignment ambiguous regions were removed (creating 10 reduced
      alignments). These alignment ambiguous regions were analyzed as a third type of
      data set, culled alignments. Maximum parsimony, neighbor-joining, and parsimony
      bootstrap analyses were performed. The resulting phylogenies were compared to
      each other by the symmetric difference distance tree comparison metric (SymD).
      The correlation of the phylogenies with the alignment parameters was tested by
      comparing matrices from SymD with corresponding matrices of Manhattan distances
      representing the alignment parameters. Differences among individual parsimony
      trees from the full alignments were frequently correlated with the differences
      among alignment parameters (580/1000 tests), as were trees from the culled
      alignments (403/1000 tests). Differences among individual parsimony trees from
      the reduced alignments were less frequently correlated with the differences among
      alignment parameters (230/1000 tests). Differences among majority-rule consensus 
      trees (50%) from the parsimony analysis of the full alignments were significantly
      correlated with the differences among alignment parameters, whereas consensus
      trees from the reduced and culled analyses were not correlated with the alignment
      parameters. These patterns of correlation confirm that choice of alignment
      parameters has the potential to bias the resultant phylogenies for the nematode
      SSU data set, and suggest that the removal of alignment ambiguous regions reduces
      this effect. Finally, we discuss the implications of conservative phylogenetic
      hypotheses for Nematoda produced by exploring alignment space and removing
      alignment ambiguous regions for SSU rDNA.
FAU - Smythe, Ashleigh B
AU  - Smythe AB
AD  - Department of Nematology, University of California, One Shields Avenue, Davis,
      California 95616, USA. smythea@si.edu
FAU - Sanderson, Michael J
AU  - Sanderson MJ
FAU - Nadler, Steven A
AU  - Nadler SA
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Nematoda/*genetics
MH  - *Phylogeny
MH  - RNA, Ribosomal/*genetics
MH  - *Sequence Alignment
EDAT- 2007/03/10 09:00
MHDA- 2007/04/14 09:00
CRDT- 2007/03/10 09:00
PHST- 2007/03/10 09:00 [pubmed]
PHST- 2007/04/14 09:00 [medline]
PHST- 2007/03/10 09:00 [entrez]
AID - 10.1080/10635150601089001 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Dec;55(6):972-92. doi: 10.1080/10635150601089001.

PMID- 17345677
OWN - NLM
STAT- MEDLINE
DCOM- 20070412
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 6
DP  - 2006 Dec
TI  - Mapping uncertainty and phylogenetic uncertainty in ancestral character state
      reconstruction: an example in the moss genus Brachytheciastrum.
PG  - 957-71
AB  - The evolution of species traits along a phylogeny can be examined through an
      increasing number of possible, but not necessarily complementary, approaches. In 
      this paper, we assess whether deriving ancestral states of discrete morphological
      characters from a model whose parameters are (i) optimized by ML on a most likely
      tree; (II) optimized by ML onto each of a Bayesian sample of trees; and (III)
      sampled by a MCMC visiting the space of a Bayesian sample of trees affects the
      reconstruction of ancestral states in the moss genus Brachytheciastrum. In the
      first two methods, the choice of a single- or two-rate model and of a genetic
      distance (wherein branch lengths are used to determine the probabilities of
      change) or speciational (wherein changes are only driven by speciation events)
      model based upon a likelihood-ratio test strongly depended on the sampled trees. 
      Despite these differences in model selection, reconstructions of ancestral
      character states were strongly correlated to each others across nodes, often at r
      &gt; 0.9, for all the characters. The Bayesian approach of ancestral character state
      reconstruction offers, however, a series of advantages over the single-tree
      approach or the ML model optimization on a Bayesian sample of trees because it
      does not involve restricting model parameters prior to reconstructing ancestral
      states, but rather allows a range of model parameters and ancestral character
      states to be sampled according to their posterior probabilities. From the
      distribution of the latter, conclusions on trait evolution can be made in a more 
      satisfactorily way than when a substantial part of the uncertainty of the results
      is obscured by the focus on a single set of model parameters and associated
      ancestral states. The reconstructions of ancestral character states in
      Brachytheciastrum reveal rampant parallel morphological evolution. Most species
      previously described based on phenetic grounds are thus resolved of polyphyletic 
      origin. Species polyphylly has been increasingly reported among mosses, raising
      severe reservations regarding current species definition.
FAU - Vanderpoorten, A
AU  - Vanderpoorten A
AD  - Institute of Botany, University of Liege, B22 Sart Tilman, B-4000 Liege, Belgium.
      a.vanderpoorten@ulg.ac.be
FAU - Goffinet, B
AU  - Goffinet B
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - *Bryopsida/genetics
MH  - Phylogeny
EDAT- 2007/03/10 09:00
MHDA- 2007/04/14 09:00
CRDT- 2007/03/10 09:00
PHST- 2007/03/10 09:00 [pubmed]
PHST- 2007/04/14 09:00 [medline]
PHST- 2007/03/10 09:00 [entrez]
AID - 10.1080/10635150601088995 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Dec;55(6):957-71. doi: 10.1080/10635150601088995.

PMID- 17345676
OWN - NLM
STAT- MEDLINE
DCOM- 20070412
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 6
DP  - 2006 Dec
TI  - Short interspersed elements (SINEs) in plants: origin, classification, and use as
      phylogenetic markers.
PG  - 949-56
AB  - Short interspersed elements (SINEs) are a class of dispersed mobile sequences
      that use RNA as an intermediate in an amplification process called retroposition.
      The presence-absence of a SINE at a given locus has been used as a meaningful
      classification criterion to evaluate phylogenetic relations among species. We
      review here recent developments in the characterisation of plant SINEs and their 
      use as molecular makers to retrace phylogenetic relations among wild and
      cultivated Oryza and Brassica species. In Brassicaceae, further use of SINE
      markers is limited by our partial knowledge of endogenous SINE families (their
      origin and evolution histories) and by the absence of a clear classification. To 
      solve this problem, phylogenetic relations among all known Brassicaceae SINEs
      were analyzed and a new classification, grouping SINEs in 15 different families, 
      is proposed. The relative age and size of each Brassicaceae SINE family was
      evaluated and new phylogenetically supported subfamilies were described. We also 
      present evidence suggesting that new potentially active SINEs recently emerged in
      Brassica oleracea from the shuffling of preexisting SINE portions. Finally, the
      comparative evolution history of SINE families present in Arabidopsis thaliana
      and Brassica oleracea revealed that SINEs were in general more active in the
      Brassica lineage. The importance of these new data for the use of Brassicaceae
      SINEs as molecular markers in future applications is discussed.
FAU - Deragon, Jean-Marc
AU  - Deragon JM
AD  - CNRS UMR6547, GDR2157 Biomove, Universite Blaise Pascal, 24 Avenue des Landais,
      63177, Aubiere, France. j-marc.deragon@univ-bpclermont.fr
FAU - Zhang, Xiaoyu
AU  - Zhang X
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
SB  - IM
MH  - Arabidopsis/genetics
MH  - Brassica/genetics
MH  - Brassicaceae/*genetics
MH  - Genetic Markers
MH  - *Phylogeny
MH  - *Short Interspersed Nucleotide Elements
EDAT- 2007/03/10 09:00
MHDA- 2007/04/14 09:00
CRDT- 2007/03/10 09:00
PHST- 2007/03/10 09:00 [pubmed]
PHST- 2007/04/14 09:00 [medline]
PHST- 2007/03/10 09:00 [entrez]
AID - 10.1080/10635150601047843 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Dec;55(6):949-56. doi: 10.1080/10635150601047843.

PMID- 17345675
OWN - NLM
STAT- MEDLINE
DCOM- 20070412
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 6
DP  - 2006 Dec
TI  - Automated scanning for phylogenetically informative transposed elements in
      rodents.
PG  - 936-48
AB  - Transposed elements constitute an attractive, useful source of phylogenetic
      markers to elucidate the evolutionary history of their hosts. Frequent and
      successive amplifications over evolutionary time are important requirements for
      utilizing their presence or absence as landmarks of evolution. Although
      transposed elements are well distributed in rodent taxa, the generally high
      degree of genomic sequence divergence among species complicates our access to
      presence/absence data. With this in mind we developed a novel, high-throughput
      computational strategy, called CPAL (Conserved Presence/Absence Locus-finder), to
      identify genome-wide distributed, phylogenetically informative transposed
      elements flanked by highly conserved regions. From a total of 232 extracted
      chromosomal mouse loci we randomly selected 14 of these plus 2 others from
      previous test screens and attempted to amplify them via PCR in representative
      rodent species. All loci were amplifiable and ultimately contributed 31
      phylogenetically informative markers distributed throughout the major groups of
      Rodentia.
FAU - Farwick, Astrid
AU  - Farwick A
AD  - Institute of Experimental Pathology, ZMBE, University of Munster,
      Von-Esmarch-Str. 56, 48149 Munster, Germany.
FAU - Jordan, Ursula
AU  - Jordan U
FAU - Fuellen, Georg
AU  - Fuellen G
FAU - Huchon, Dorothee
AU  - Huchon D
FAU - Catzeflis, Francois
AU  - Catzeflis F
FAU - Brosius, Jurgen
AU  - Brosius J
FAU - Schmitz, Jurgen
AU  - Schmitz J
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Biological Evolution
MH  - Computational Biology
MH  - Genetic Markers
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Rodentia/*genetics
MH  - Sequence Alignment
MH  - *Short Interspersed Nucleotide Elements
EDAT- 2007/03/10 09:00
MHDA- 2007/04/14 09:00
CRDT- 2007/03/10 09:00
PHST- 2007/03/10 09:00 [pubmed]
PHST- 2007/04/14 09:00 [medline]
PHST- 2007/03/10 09:00 [entrez]
AID - 10.1080/10635150601064806 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Dec;55(6):936-48. doi: 10.1080/10635150601064806.

PMID- 17345674
OWN - NLM
STAT- MEDLINE
DCOM- 20070412
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 6
DP  - 2006 Dec
TI  - SINEs of a nearly perfect character.
PG  - 928-35
AB  - Mobile elements have been recognized as powerful tools for phylogenetic and
      population-level analyses. However, issues regarding potential sources of
      homoplasy and other misleading events have been raised. We have collected
      available data for all phylogenetic and population level studies of primates
      utilizing Alu insertion data and examined them for potentially homoplasious and
      other misleading events. Very low levels of each potential confounding factor in 
      a phylogenetic or population analysis (i.e., lineage sorting, parallel
      insertions, and precise excision) were found. Although taxa known to be subject
      to high levels of these types of events may indeed be subject to problems when
      using SINE analysis, we propose that most taxa will respond as the order Primates
      has--by the resolution of several long-standing problems observed using
      sequence-based methods.
FAU - Ray, David A
AU  - Ray DA
AD  - Department of Biology, West Virginia University, PO Box 6057, Morgantown, West
      Virginia 26506, USA.
FAU - Xing, Jinchuan
AU  - Xing J
FAU - Salem, Abdel-Halim
AU  - Salem AH
FAU - Batzer, Mark A
AU  - Batzer MA
LA  - eng
GR  - R01 GM59290/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Alu Elements
MH  - Animals
MH  - *Phylogeny
MH  - Primates/*genetics
EDAT- 2007/03/10 09:00
MHDA- 2007/04/14 09:00
CRDT- 2007/03/10 09:00
PHST- 2007/03/10 09:00 [pubmed]
PHST- 2007/04/14 09:00 [medline]
PHST- 2007/03/10 09:00 [entrez]
AID - 10.1080/10635150600865419 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Dec;55(6):928-35. doi: 10.1080/10635150600865419.

PMID- 17345673
OWN - NLM
STAT- MEDLINE
DCOM- 20070412
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 6
DP  - 2006 Dec
TI  - Extensive morphological convergence and rapid radiation in the evolutionary
      history of the family Geoemydidae (old world pond turtles) revealed by SINE
      insertion analysis.
PG  - 912-27
AB  - The family Geoemydidae is one of three in the superfamily Testudinoidea and is
      the most diversified family of extant turtle species. The phylogenetic
      relationships in this family and among related families have been vigorously
      investigated from both morphological and molecular viewpoints. The evolutionary
      history of Geoemydidae, however, remains controversial. Therefore, to elucidate
      the phylogenetic relationships of Geoemydidae and related species, we applied the
      SINE insertion method to investigate 49 informative SINE loci in 28 species. We
      detected four major evolutionary lineages (Testudinidae, Batagur group,
      Siebenrockiella group, and Geoemyda group) in the clade Testuguria (a clade of
      Geoemydidae + Testudinidae). All five specimens of Testudinidae form a
      monophyletic clade. The Batagur group comprises five batagurines. The
      Siebenrockiella group has one species, Siebenrockiella crassicollis. The Geoemyda
      group comprises 15 geoemydines (including three former batagurines, Mauremys
      reevesii, Mauremys sinensis, and Heosemys annandalii). Among these four groups,
      the SINE insertion patterns were inconsistent at four loci, suggesting that an
      ancestral species of Testuguria radiated and rapidly diverged into the four
      lineages during the initial stage of its evolution. Furthermore, within the
      Geoemyda group we identified three evolutionary lineages, namely Mauremys, Cuora,
      and Heosemys. The Heosemys lineage comprises Heosemys, Sacalia, Notochelys, and
      Melanochelys species, and its monophyly is a novel assemblage in Geoemydidae. Our
      SINE phylogenetic tree demonstrates extensive convergent morphological evolution 
      between the Batagur group and the three species of the Geoemyda group, M.
      reevesii, M. sinensis, and H. annandalii.
FAU - Sasaki, Takeshi
AU  - Sasaki T
AD  - Department of Evolutionary Biology and Biodiversity, National Institute for Basic
      Biology, Myodaiji, Okazaki, Japan.
FAU - Yasukawa, Yuichirou
AU  - Yasukawa Y
FAU - Takahashi, Kazuhiko
AU  - Takahashi K
FAU - Miura, Seiko
AU  - Miura S
FAU - Shedlock, Andrew M
AU  - Shedlock AM
FAU - Okada, Norihiro
AU  - Okada N
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - *Biological Evolution
MH  - Molecular Sequence Data
MH  - *Short Interspersed Nucleotide Elements
MH  - Turtles/anatomy &amp; histology/*genetics
EDAT- 2007/03/10 09:00
MHDA- 2007/04/14 09:00
CRDT- 2007/03/10 09:00
PHST- 2007/03/10 09:00 [pubmed]
PHST- 2007/04/14 09:00 [medline]
PHST- 2007/03/10 09:00 [entrez]
AID - 10.1080/10635150601058014 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Dec;55(6):912-27. doi: 10.1080/10635150601058014.

PMID- 17345672
OWN - NLM
STAT- MEDLINE
DCOM- 20070412
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 6
DP  - 2006 Dec
TI  - Phylogenomic investigation of CR1 LINE diversity in reptiles.
PG  - 902-11
AB  - It is unlikely that taxonomically diverse phylogenetic studies will be completed 
      rapidly in the near future for nonmodel organisms on a whole-genome basis.
      However, one approach to advancing the field of "phylogenomics" is to estimate
      the structure of poorly known genomes by mining libraries of clones from suites
      of taxa, rather than from single species. The present analysis adopts this
      approach by taking advantage of megabase-scale end-sequence scanning of reptilian
      genomic clones to characterize diversity of CR1-like LINEs, the dominant family
      of transposable elements (TEs) in the sister group of mammals. As such, it helps 
      close an important gap in the literature on the molecular systematics and
      evolution of retroelements in nonavian reptiles. Results from aligning more than 
      14 Mb of sequence from the American alligator (Alligator mississippiensis),
      painted turtle (Chrysemys picta), Bahamian green anole (Anolis smaragdinus),
      Tuatara (Sphenodon punctatus), Emu (Dromaius novaehollandiae), and Zebra Finch
      (Taeniopygia guttata) against a comprehensive library approximately 3000
      TE-encoding peptides reflect an increasing abundance of LINE and
      non-long-terminal-repeat (non-LTR) retrotransposon repeat types with the age of
      common ancestry among exemplar reptilian clades. The hypothesis that repeat
      diversity is correlated with basal metabolic rate was tested using comparative
      methods and a significant nonlinear relationship was indicated. This analysis
      suggests that the age of divergence between an exemplary clade and its sister
      group as well as metabolic correlates should be considered in addition to genome 
      size in explaining patterns of retroelement diversity. The first phylogenetic
      analysis of the largely unexplored chicken repeat 1 (CR1) 3' reverse
      transcriptase (RT) conserved domains 8 and 9 in nonavian reptiles reveals a
      pattern of multiple lineages with variable branch lengths, suggesting presence of
      both old and young elements and the existence of several distinct well-supported 
      clades not apparent from previous characterization of CR1 subfamily structure in 
      birds and the turtle. This mode of CR1 evolution contrasts with historical
      patterns of LINE 1 diversification in mammals and hints toward the existence of a
      rich but still largely unexplored diversity of nonavian retroelements of
      importance to advancing both comparative vertebrate genomics and amniote
      systematics.
FAU - Shedlock, Andrew M
AU  - Shedlock AM
AD  - Department of Organismic and Evolutionary Biology, Museum of Comparative Zoology,
      Harvard University, 26 Oxford Street, Cambridge, Massachusetts 02138, USA.
      shedlock@oeb.harvard.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amino Acid Sequence
MH  - Animals
MH  - Birds/*genetics
MH  - *Genomics
MH  - *Long Interspersed Nucleotide Elements
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Protein Structure, Tertiary
MH  - Reptiles/*genetics
MH  - Sequence Homology, Amino Acid
EDAT- 2007/03/10 09:00
MHDA- 2007/04/14 09:00
CRDT- 2007/03/10 09:00
PHST- 2007/03/10 09:00 [pubmed]
PHST- 2007/04/14 09:00 [medline]
PHST- 2007/03/10 09:00 [entrez]
AID - 10.1080/10635150601091924 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Dec;55(6):902-11. doi: 10.1080/10635150601091924.

PMID- 17345671
OWN - NLM
STAT- MEDLINE
DCOM- 20070412
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 6
DP  - 2006 Dec
TI  - Phylogenomic analysis of the L1 retrotransposons in Deuterostomia.
PG  - 886-901
AB  - L1 retrotransposons constitute the largest single component of mammalian genomes.
      In contrast to the single remaining lineage of L1 retrotransposons in mammalian
      genomes, some teleost fishes contain a highly diverse L1 retrotransposon
      repertoire. Major evolutionary changes in L1 retrotransposon repertoires have
      therefore taken place in the land vertebrates (Tetrapoda). The lack of sequence
      data for L1 retrotransposons in the basal living Tetrapoda lineages prompted an
      investigation of their distribution and evolution in the genomes of the key
      tetrapod lineages, amphibians and reptiles, and in lungfishes. In this study, we 
      combined genome database searches with PCR analysis to demonstrate that L1
      retrotransposons are present in the genomes of lungfishes, amphibians, and
      lepidosaurs. Phylogenomic analysis shows that the genomes of Deuterostomia
      possess three highly divergent groups of L1 retrotransposons, with distinct
      distribution patterns. The analysis of L1 diversity shows the presence of a very 
      large number of diverse L1 families, each with very low copy numbers, at the time
      of the origin of tetrapods. During the evolution of synapsids, all but one L1
      lineage have been lost. This study establishes that the loss of L1 diversity and 
      explosion in copy numbers occurred in the synapsid ancestors of mammals, and was 
      most probably caused by severe population bottlenecks.
FAU - Kordis, Dusan
AU  - Kordis D
AD  - Department of Biochemistry and Molecular Biology, Jozef Stefan Institute,
      Ljubljana, Slovenia. dusan.kordis@ijs.si
FAU - Lovsin, Nika
AU  - Lovsin N
FAU - Gubensek, Franc
AU  - Gubensek F
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Chordata/*genetics
MH  - Chordata, Nonvertebrate/*genetics
MH  - Echinodermata/*genetics
MH  - Genomics
MH  - *Long Interspersed Nucleotide Elements
MH  - *Phylogeny
MH  - Population Dynamics
EDAT- 2007/03/10 09:00
MHDA- 2007/04/14 09:00
CRDT- 2007/03/10 09:00
PHST- 2007/03/10 09:00 [pubmed]
PHST- 2007/04/14 09:00 [medline]
PHST- 2007/03/10 09:00 [entrez]
AID - 10.1080/10635150601052637 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Dec;55(6):886-901. doi: 10.1080/10635150601052637.

PMID- 17345670
OWN - NLM
STAT- MEDLINE
DCOM- 20070412
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 6
DP  - 2006 Dec
TI  - Distribution and phylogeny of Penelope-like elements in eukaryotes.
PG  - 875-85
AB  - Penelope-like elements (PLEs) are a relatively little studied class of eukaryotic
      retroelements, distinguished by the presence of the GIY-YIG endonuclease domain, 
      the ability of some representatives to retain introns, and the similarity of
      PLE-encoded reverse transcriptases to telomerases. Although these
      retrotransposons are abundant in many animal genomes, the reverse transcriptase
      moiety can also be found in several protists, fungi, and plants, indicating its
      ancient origin. A comprehensive phylogenetic analysis of PLEs was conducted,
      based on extended sequence alignments and a considerably expanded data set. PLEs 
      exhibit the pattern of evolution similar to that of non-LTR retrotransposons,
      which form deep-branching clades dating back to the Precambrian era. However,
      PLEs seem to have experienced a much higher degree of lineage losses than non-LTR
      retrotransposons. It is suggested that PLEs and non-LTR retrotransposons are
      included into a larger eTPRT (eukaryotic target-primed) group of retroelements,
      characterized by 5' truncation, variable target-site duplication, and the
      potential of the 3' end to participate in formation of non-autonomous
      derivatives.
FAU - Arkhipova, Irina R
AU  - Arkhipova IR
AD  - Department of Molecular and Cellular Biology, Harvard University, Cambridge,
      Massachusetts 02138, USA. arkhipov@fas.harvard.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
RN  - 0 (Retroelements)
SB  - IM
MH  - Amino Acid Sequence
MH  - Animals
MH  - Chordata/genetics
MH  - *Eukaryotic Cells
MH  - Fungi/genetics
MH  - Genetic Markers
MH  - Invertebrates/genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Plants/genetics
MH  - Protein Structure, Tertiary
MH  - *Retroelements
MH  - Sequence Alignment
MH  - Sequence Homology, Amino Acid
EDAT- 2007/03/10 09:00
MHDA- 2007/04/14 09:00
CRDT- 2007/03/10 09:00
PHST- 2007/03/10 09:00 [pubmed]
PHST- 2007/04/14 09:00 [medline]
PHST- 2007/03/10 09:00 [entrez]
AID - 10.1080/10635150601077683 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Dec;55(6):875-85. doi: 10.1080/10635150601077683.

PMID- 17345669
OWN - NLM
STAT- MEDLINE
DCOM- 20070412
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 6
DP  - 2006 Dec
TI  - Exploring frontiers in the DNA landscape: an introduction to the symposium
      "Genome Analysis and the Molecular Systematics of Retroelements".
PG  - 871-4
AB  - The emerging field of phylogenomics is influencing both the amount and type of
      characters being brought to bear on long-standing problems in systematic biology.
      Moreover, the proliferation of sequence information from genome projects in
      concert with the development of new informatics tools is widening access to
      comparative data on retroelements to a broad cross section of investigators.
      Motivated by this, the Society of Systematic Biologists sponsored a symposium
      entitled "Genome Analysis and the Molecular Systematics of Retroelements," and
      the resulting papers illustrate this theme of new discoveries and cover three
      basic areas of research: (i) the taxonomic distribution and phylogenetic
      structure of families of retroelements; (II) the use of SINE and LINE insertions 
      for phylogenetic inference; and (III) the informatics and classification of
      repetitive elements. Contributions of each article are briefly discussed in this 
      context and particularly fruitful directions for future research illuminated by
      results of this symposium are reviewed.
FAU - Shedlock, Andrew M
AU  - Shedlock AM
AD  - Department of Organismic and Evolutionary Biology, Museum of Comparative Zoology,
      Harvard University, 26 Oxford Street, Cambridge, Massachusetts 02138, USA.
      shedlock@oeb.harvard.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Retroelements)
SB  - IM
MH  - Animals
MH  - Classification/methods
MH  - Genomics/*trends
MH  - Phylogeny
MH  - Retroelements/*genetics
EDAT- 2007/03/10 09:00
MHDA- 2007/04/14 09:00
CRDT- 2007/03/10 09:00
PHST- 2007/03/10 09:00 [pubmed]
PHST- 2007/04/14 09:00 [medline]
PHST- 2007/03/10 09:00 [entrez]
AID - 10.1080/10635150601077634 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Dec;55(6):871-4. doi: 10.1080/10635150601077634.

PMID- 17060206
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20061024
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - Taxonomic inflation and the stability of species lists: the perils of ostrich's
      behavior.
PG  - 859-67
FAU - Padial, Jose M
AU  - Padial JM
AD  - Department of Biodiversity and Evolutionary Biology, Museo Nacional de Ciencias
      Naturales-CSIC, C/Jose Gutierrez Abascal 2, Madrid, Spain.
FAU - de la Riva, Ignacio
AU  - de la Riva I
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biodiversity
MH  - *Classification/methods
MH  - Conservation of Natural Resources
MH  - Species Specificity
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - Q588TT6U0R75X39V [pii]
AID - 10.1080/1063515060081588 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):859-67. doi: 10.1080/1063515060081588.

PMID- 17060205
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - Stability and universality in the application of taxon names in phylogenetic
      nomenclature.
PG  - 848-58
FAU - Bertrand, Yann
AU  - Bertrand Y
AD  - School of Life Sciences, Department of Biology, Sodertorn University College,
      SE-141 89, Huddinge, Sweden. yann.bertrand@sh.se
FAU - Harlin, Mikael
AU  - Harlin M
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2008 Feb;57(1):157-60. PMID: 18300028
MH  - Biodiversity
MH  - Classification/methods
MH  - Consensus
MH  - Magnoliopsida/classification
MH  - *Phylogeny
MH  - *Terminology as Topic
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - MVRN1428P4937117 [pii]
AID - 10.1080/10635150600960061 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):848-58. doi: 10.1080/10635150600960061.

PMID- 17060204
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20061024
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - Who will actually use DNA barcoding and what will it cost?
PG  - 844-7
FAU - Cameron, Stephen
AU  - Cameron S
AD  - Department of Integrative Biology, Brigham Young University, Provo, UT 84602,
      USA. stephen_cameron@byu.edu
FAU - Rubinoff, Daniel
AU  - Rubinoff D
FAU - Will, Kipling
AU  - Will K
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Classification/*methods
MH  - DNA/chemistry
MH  - Electron Transport Complex IV/chemistry/genetics
MH  - Sequence Analysis, DNA/*economics/methods
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - M6488071732QN120 [pii]
AID - 10.1080/10635150600960079 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):844-7. doi: 10.1080/10635150600960079.

PMID- 17060203
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20061024
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - Multipolar consensus for phylogenetic trees.
PG  - 837-43
AB  - Collections of phylogenetic trees are usually summarized using consensus methods.
      These methods build a single tree, supposed to be representative of the
      collection. However, in the case of heterogeneous collections of trees, the
      resulting consensus may be poorly resolved (strict consensus, majority-rule
      consensus, ...), or may perform arbitrary choices among mutually incompatible
      clades, or splits (greedy consensus). Here, we propose an alternative method,
      which we call the multipolar consensus (MPC). Its aim is to display all the
      splits having a support above a predefined threshold, in a minimum number of
      consensus trees, or poles. We show that the problem is equivalent to a
      graph-coloring problem, and propose an implementation of the method. Finally, we 
      apply the MPC to real data sets. Our results indicate that, typically, all the
      splits down to a weight of 10% can be displayed in no more than 4 trees. In
      addition, in some cases, biologically relevant secondary signals, which would not
      have been present in any of the classical consensus trees, are indeed captured by
      our method, indicating that the MPC provides a convenient exploratory method for 
      phylogenetic analysis. The method was implemented in a package freely available
      at http://www.lirmm.fr/~cbonnard/MPC.html
FAU - Bonnard, Cecile
AU  - Bonnard C
AD  - Laboratoire d'Informatique, de Robotique et de Microelectronique de Montpellier
      UMR 5506, CNRS-Universite de Montpellier 2,161, Rue Ada, Montpellier Cedex 5,
      France.
FAU - Berry, Vincent
AU  - Berry V
FAU - Lartillot, Nicolas
AU  - Lartillot N
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Consensus
MH  - *Phylogeny
MH  - Research Design
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - KP115K22126T2P53 [pii]
AID - 10.1080/10635150600969880 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):837-43. doi: 10.1080/10635150600969880.

PMID- 17060202
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20061024
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - Phylogenetic supermatrix analysis of GenBank sequences from 2228 papilionoid
      legumes.
PG  - 818-36
AB  - A comprehensive phylogeny of papilionoid legumes was inferred from sequences of
      2228 taxa in GenBank release 147. A semiautomated analysis pipeline was
      constructed to download, parse, assemble, align, combine, and build trees from a 
      pool of 11,881 sequences. Initial steps included all-against-all BLAST similarity
      searches coupled with assembly, using a novel strategy for building
      length-homogeneous primary sequence clusters. This was followed by a combination 
      of global and local alignment protocols to build larger secondary clusters of
      locally aligned sequences, thus taking into account the dramatic differences in
      length of the heterogeneous coding and noncoding sequence data present in
      GenBank. Next, clusters were checked for the presence of duplicate genes and
      other potentially misleading sequences and examined for combinability with other 
      clusters on the basis of taxon overlap. Finally, two supermatrices were
      constructed: a "sparse" matrix based on the primary clusters alone (1794 taxa x
      53,977 characters), and a somewhat more "dense" matrix based on the secondary
      clusters (2228 taxa x 33,168 characters). Both matrices were very sparse, with
      95% of their cells containing gaps or question marks. These were subjected to
      extensive heuristic parsimony analyses using deterministic and stochastic
      heuristics, including bootstrap analyses. A "reduced consensus" bootstrap
      analysis was also performed to detect cryptic signal in a subtree of the data set
      corresponding to a "backbone" phylogeny proposed in previous studies. Overall,
      the dense supermatrix appeared to provide much more satisfying results, indicated
      by better resolution of the bootstrap tree, excellent agreement with the backbone
      papilionoid tree in the reduced bootstrap consensus analysis, few problematic
      large polytomies in the strict consensus, and less fragmentation of
      conventionally recognized genera. Nevertheless, at lower taxonomic levels several
      problems were identified and diagnosed. A large number of methodological issues
      in supermatrix construction at this scale are discussed, including detection of
      annotation errors in GenBank sequences; the shortage of effective algorithms and 
      software for local multiple sequence alignment; the difficulty of overcoming
      effects of fragmentation of data into nearly disjoint blocks in sparse
      supermatrices; and the lack of informative tools to assess confidence limits in
      very large trees.
FAU - McMahon, Michelle M
AU  - McMahon MM
AD  - Section of Evolution and Ecology, University of California Davis, Davis, CA
      95616, USA. mcmahonm@email.arizona.edu
FAU - Sanderson, Michael J
AU  - Sanderson MJ
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Plant Proteins)
SB  - IM
MH  - Algorithms
MH  - Base Sequence
MH  - Cluster Analysis
MH  - Computational Biology/methods
MH  - *Databases, Nucleic Acid
MH  - Fabaceae/*classification/genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Plant Proteins/genetics
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA/*methods
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - T0622815056726L1 [pii]
AID - 10.1080/10635150600999150 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):818-36. doi: 10.1080/10635150600999150.

PMID- 17060201
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20061024
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - The ancestral distance test: what relatedness can reveal about correlated
      evolution in large lineages with missing character data and incomplete
      phylogenies.
PG  - 803-17
AB  - The ancestral distance test is introduced to detect correlated evolution between 
      two binary traits in large phylogenies that may lack resolved subclades, branch
      lengths, and/or comparative data. We define the ancestral distance as the time
      separating a randomly sampled taxon from its most recent ancestor (MRA) with
      extant descendants that have an independent trait. The sampled taxon either has
      (target sample) or lacks (nontarget sample) a dependent trait. Modeled as a
      Markov process, we show that the distribution of ancestral distances for the
      target sample is identical to that of the nontarget sample when characters are
      uncorrelated, whereas ancestral distances are smaller on average for the target
      sample when characters are correlated. Simulations suggest that the ancestral
      distance can be estimated using the time, total branch length, taxonomic rank, or
      number of speciation events between a sampled taxon and the MRA. These results
      are shown to be robust to deviations from Markov assumptions. A Monte Carlo
      technique estimates P-values when fully resolved phylogenies with branch lengths 
      are available, and we evaluate the Monte Carlo approach using a data set with
      known correlation. Measures of relatedness were found to provide a robust means
      to test hypotheses of correlated character evolution.
FAU - Hearn, David
AU  - Hearn D
AD  - Plant Sciences, University of Arizona, 303 Forbes Building, Tucson, AZ 85721,
      USA. dhearn@email.arizona.edu
FAU - Huber, Mark
AU  - Huber M
LA  - eng
PT  - Comparative Study
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Phylogeny
MH  - Plants/classification
MH  - Software
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - W11271111P02884T [pii]
AID - 10.1080/10635150600981562 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):803-17. doi: 10.1080/10635150600981562.

PMID- 17060200
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - Phyloclimatic modeling: combining phylogenetics and bioclimatic modeling.
PG  - 785-802
AB  - We investigate the impact of past climates on plant diversification by tracking
      the "footprint" of climate change on a phylogenetic tree. Diversity within the
      cosmopolitan carnivorous plant genus Drosera (Droseraceae) is focused within
      Mediterranean climate regions. We explore whether this diversity is temporally
      linked to Mediterranean-type climatic shifts of the mid-Miocene and whether
      climate preferences are conservative over phylogenetic timescales. Phyloclimatic 
      modeling combines environmental niche (bioclimatic) modeling with phylogenetics
      in order to study evolutionary patterns in relation to climate change. We present
      the largest and most complete such example to date using Drosera. The bioclimatic
      models of extant species demonstrate clear phylogenetic patterns; this is
      particularly evident for the tuberous sundews from southwestern Australia
      (subgenus Ergaleium). We employ a method for establishing confidence intervals of
      node ages on a phylogeny using replicates from a Bayesian phylogenetic analysis. 
      This chronogram shows that many clades, including subgenus Ergaleium and section 
      Bryastrum, diversified during the establishment of the Mediterranean-type
      climate. Ancestral reconstructions of bioclimatic models demonstrate a pattern of
      preference for this climate type within these groups. Ancestral bioclimatic
      models are projected into palaeo-climate reconstructions for the time periods
      indicated by the chronogram. We present two such examples that each generate
      plausible estimates of ancestral lineage distribution, which are similar to their
      current distributions. This is the first study to attempt bioclimatic projections
      on evolutionary time scales. The sundews appear to have diversified in response
      to local climate development. Some groups are specialized for Mediterranean
      climates, others show wide-ranging generalism. This demonstrates that
      Phyloclimatic modeling could be repeated for other plant groups and is
      fundamental to the understanding of evolutionary responses to climate change.
FAU - Yesson, C
AU  - Yesson C
AD  - Centre for Plant Diversity and Systematics, Plant Science Laboratories, School of
      Biological Sciences, The University of Reading, Whiteknights, Reading, Berks,
      England.
FAU - Culham, A
AU  - Culham A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Acclimatization/genetics
MH  - Australia
MH  - Bayes Theorem
MH  - *Climate
MH  - Drosera/*classification/genetics
MH  - Genetic Variation
MH  - Models, Theoretical
MH  - *Phylogeny
MH  - Rain
MH  - Temperature
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - JL850552X577RQ54 [pii]
AID - 10.1080/1063515060081570 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):785-802. doi: 10.1080/1063515060081570.

PMID- 17060199
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - Are the linnean and phylogenetic nomenclatural systems combinable?
      Recommendations for biological nomenclature.
PG  - 774-84
AB  - A combination approach between the rules and recommendations from the Linnean
      (rank-based) and phylogenetic nomenclature is proposed, with a review of the
      debate. Advantages and drawbacks of both systems are discussed. Too often the
      debates are biased and unconstructive, and there is a need for dialogue and
      compromise. Our recommendations for the future of biological classification, to
      be considered by new editions of all codes of nomenclature, would enable the
      Linnean and the phylogenetic nomenclatural systems to coexist, or be combined.
      (1) We see it as essential that species binomen, including the formal rank of
      genus, are retained, and (2) species should continue to be linked to type
      specimens. (3) The use of other formal ranks should be minimized; however, we
      suggest retaining the classical supergeneric ranks (family, class, order, phylum,
      kingdom) for purely practical reasons. (4) For these ranks and any formally
      defined clades, type taxa (species, genera) should be replaced by phylogenetic
      definitions that explicitly hypothesize monophyly. (5) In contrast, species
      monophyly should not be required, because theory predicts that many species are
      not monophyletic. (6) It should be stressed that equal ranks do not imply
      comparable evolutionary histories.
FAU - Kuntner, Matjaz
AU  - Kuntner M
AD  - Institute of Biology, Scientific Research Centre of the Slovenian Academy of
      Sciences and Arts, Ljubljana, Slovenia. kuntner@gmail.com
FAU - Agnarsson, Ingi
AU  - Agnarsson I
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/methods
MH  - *Phylogeny
MH  - *Terminology as Topic
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - W2817G845JX1K805 [pii]
AID - 10.1080/10635150600981596 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):774-84. doi: 10.1080/10635150600981596.

PMID- 17060198
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20061024
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - Phylogenetic diversity within seconds.
PG  - 769-73
AB  - We consider a (phylogenetic) tree with n labeled leaves, the taxa, and a length
      for each branch in the tree. For any subset of k taxa, the phylogenetic diversity
      is defined as the sum of the branch-lengths of the minimal subtree connecting the
      taxa in the subset. We introduce two time-efficient algorithms (greedy and
      pruning) to compute a subset of size k with maximal phylogenetic diversity in O(n
      log k) and O[n + (n-k) log (n-k)] time, respectively. The greedy algorithm is an 
      efficient implementation of the so-called greedy strategy (Steel, 2005; Pardi and
      Goldman, 2005), whereas the pruning algorithm provides an alternative description
      of the same problem. Both algorithms compute within seconds a subtree with
      maximal phylogenetic diversity for trees with 100,000 taxa or more.
FAU - Minh, Bui Quang
AU  - Minh BQ
AD  - Center for Integrative Bioinformatics, Vienna, Max F Perutz Laboratories,
      University of Vienna, Medical University of Vienna, Veterinary University of
      Vienna, Dr-Bohr-Gasse 9/6, A-1030, Vienna, Austria.
FAU - Klaere, Steffen
AU  - Klaere S
FAU - von Haeseler, Arndt
AU  - von Haeseler A
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - *Biodiversity
MH  - Classification/methods
MH  - Computer Simulation/standards
MH  - *Phylogeny
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - J6783162MW170170 [pii]
AID - 10.1080/10635150600981604 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):769-73. doi: 10.1080/10635150600981604.

PMID- 17060197
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20061024
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - Efficient likelihood computations with nonreversible models of evolution.
PG  - 756-68
AB  - Recent advances in heuristics have made maximum likelihood phylogenetic tree
      estimation tractable for hundreds of sequences. Noticeably, these algorithms are 
      currently limited to reversible models of evolution, in which Felsenstein's
      pulley principle applies. In this paper we show that by reorganizing the way
      likelihood is computed, one can efficiently compute the likelihood of a tree from
      any of its nodes with a nonreversible model of DNA sequence evolution, and hence 
      benefit from cutting-edge heuristics. This computational trick can be used with
      reversible models of evolution without any extra cost. We then introduce nhPhyML,
      the adaptation of the nonhomogeneous nonstationary model of Galtier and Gouy
      (1998; Mol. Biol. Evol. 15:871-879) to the structure of PhyML, as well as an
      approximation of the model in which the set of equilibrium frequencies is
      limited. This new version shows good results both in terms of exploration of the 
      space of tree topologies and ancestral G+C content estimation. We eventually
      apply it to rRNA sequences slowly evolving sites and conclude that the model and 
      a wider taxonomic sampling still do not plead for a hyperthermophilic last
      universal common ancestor.
FAU - Boussau, Bastien
AU  - Boussau B
AD  - Laboratoire de Biometrie et Biologie Evolutive (UMR 5558); CNRS, Universite Lyon 
      1, Villeurbanne Cedex, France. boussau@biomserv.univ-lyon1.fr
FAU - Gouy, Manolo
AU  - Gouy M
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal)
SB  - IM
MH  - Algorithms
MH  - Base Composition
MH  - Classification/methods
MH  - *Computational Biology
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - RNA, Ribosomal/chemistry
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - W41W420427628524 [pii]
AID - 10.1080/10635150600975218 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):756-68. doi: 10.1080/10635150600975218.

PMID- 17060196
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20061024
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - SDM: a fast distance-based approach for (super) tree building in phylogenomics.
PG  - 740-55
AB  - Phylogenomic studies aim to build phylogenies from large sets of homologous
      genes. Such "genome-sized" data require fast methods, because of the typically
      large numbers of taxa examined. In this framework, distance-based methods are
      useful for exploratory studies and building a starting tree to be refined by a
      more powerful maximum likelihood (ML) approach. However, estimating evolutionary 
      distances directly from concatenated genes gives poor topological signal as genes
      evolve at different rates. We propose a novel method, named super distance matrix
      (SDM), which follows the same line as average consensus supertree (ACS; Lapointe 
      and Cucumel, 1997) and combines the evolutionary distances obtained from each
      gene into a single distance supermatrix to be analyzed using a standard
      distance-based algorithm. SDM deforms the source matrices, without modifying
      their topological message, to bring them as close as possible to each other;
      these deformed matrices are then averaged to obtain the distance supermatrix. We 
      show that this problem is equivalent to the minimization of a least-squares
      criterion subject to linear constraints. This problem has a unique solution which
      is obtained by resolving a linear system. As this system is sparse, its practical
      resolution requires O(naka) time, where n is the number of taxa, k the number of 
      matrices, and a &lt; 2, which allows the distance supermatrix to be quickly
      obtained. Several uses of SDM are proposed, from fast exploratory studies to more
      accurate approaches requiring heavier computing time. Using simulations, we show 
      that SDM is a relevant alternative to the standard matrix representation with
      parsimony (MRP) method, notably when the taxa sets of the different genes have
      low overlap. We also show that SDM can be used to build an excellent starting
      tree for an ML approach, which both reduces the computing time and increases the 
      topogical accuracy. We use SDM to analyze the data set of Gatesy et al. (2002,
      Syst. Biol. 51: 652-664) that involves 48 genes of 75 placental mammals. The
      results indicate that these genes have strong rate heterogeneity and confirm the 
      simulation conclusions.
FAU - Criscuolo, Alexis
AU  - Criscuolo A
AD  - Groupe Phylogenie Moleculaire, ISEM, Universite Montpellier 2, Montpellier,
      France.
FAU - Berry, Vincent
AU  - Berry V
FAU - Douzery, Emmanuel J P
AU  - Douzery EJ
FAU - Gascuel, Olivier
AU  - Gascuel O
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Genomics
MH  - *Phylogeny
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - T055214220K23102 [pii]
AID - 10.1080/10635150600969872 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):740-55. doi: 10.1080/10635150600969872.

PMID- 17060195
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20091119
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - DNA barcoding will often fail to discover new animal species over broad parameter
      space.
PG  - 729-39
AB  - With increasing force, genetic divergence of mitochondrial DNA (mtDNA) is being
      argued as the primary tool for discovery of animal species. Two thresholds of
      single-gene divergence have been proposed: reciprocal monophyly, and 10 times
      greater genetic divergence between than within species (the "10x rule"). To
      explore quantitatively the utility of each approach, we couple neutral coalescent
      theory and the classical Bateson-Dobzhansky-Muller (BDM) model of speciation. The
      joint stochastic dynamics of these two processes demonstrate that both thresholds
      fail to "discover" many reproductively isolated lineages under a single
      incompatibility BDM model, especially when BDM loci have been subject to
      divergent selection. Only when populations have been isolated for &gt; 4 million
      generations did these thresholds achieve error rates of &lt; 10% under our model
      that incorporates variable population sizes. The high error rate evident in
      simulations is corroborated with six empirical data sets. These properties
      suggest that single-gene, high-throughput approaches to discovering new animal
      species will bias large-scale biodiversity surveys, particularly toward missing
      reproductively isolated lineages that have emerged by divergent selection or
      other mechanisms that accelerate reproductive isolation. Because single-gene
      thresholds for species discovery can result in substantial error at recent
      divergence times, they will misrepresent the correspondence between recently
      isolated populations and reproductively isolated lineages (= species).
FAU - Hickerson, Michael J
AU  - Hickerson MJ
AD  - Museum of Vertebrate Zoology, University of California, Berkeley, CA 94720-3160, 
      USA. mhick@berkeley.edu
FAU - Meyer, Christopher P
AU  - Meyer CP
FAU - Moritz, Craig
AU  - Moritz C
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Biodiversity
MH  - Classification/*methods
MH  - DNA, Mitochondrial/chemistry
MH  - Evolution, Molecular
MH  - *Genetic Speciation
MH  - *Models, Genetic
MH  - Phylogeny
MH  - Selection, Genetic
MH  - Sequence Analysis, DNA
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - K10T5260868V3860 [pii]
AID - 10.1080/10635150600969898 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):729-39. doi: 10.1080/10635150600969898.

PMID- 17060194
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - DNA barcoding and taxonomy in Diptera: a tale of high intraspecific variability
      and low identification success.
PG  - 715-28
AB  - DNA barcoding and DNA taxonomy have recently been proposed as solutions to the
      crisis of taxonomy and received significant attention from scientific journals,
      grant agencies, natural history museums, and mainstream media. Here, we test two 
      key claims of molecular taxonomy using 1333 mitochondrial COI sequences for 449
      species of Diptera. We investigate whether sequences can be used for species
      identification ("DNA barcoding") and find a relatively low success rate (&lt; 70%)
      based on tree-based and newly proposed species identification criteria.
      Misidentifications are due to wide overlap between intra- and interspecific
      genetic variability, which causes 6.5% of all query sequences to have
      allospecific or a mixture of allo- and conspecific (3.6%) best-matching barcodes.
      Even when two COI sequences are identical, there is a 6% chance that they belong 
      to different species. We also find that 21% of all species lack unique barcodes
      when consensus sequences of all conspecific sequences are used. Lastly, we test
      whether DNA sequences yield an unambiguous species-level taxonomy when sequence
      profiles are assembled based on pairwise distance thresholds. We find many
      sequence triplets for which two of the three pairwise distances remain below the 
      threshold, whereas the third exceeds it; i.e., it is impossible to consistently
      delimit species based on pairwise distances. Furthermore, for species profiles
      based on a 3% threshold, only 47% of all profiles are consistent with currently
      accepted species limits, 20% contain more than one species, and 33% only some
      sequences from one species; i.e., adopting such a DNA taxonomy would require the 
      redescription of a large proportion of the known species, thus worsening the
      taxonomic impediment. We conclude with an outlook on the prospects of obtaining
      complete barcode databases and the future use of DNA sequences in a modern
      integrative taxonomy.
FAU - Meier, Rudolf
AU  - Meier R
AD  - Department of Biological Sciences, National University of Singapore, 14 Science
      Drive 4, Singapore, Singapore. dbsmr@nus.edu.sg
FAU - Shiyang, Kwong
AU  - Shiyang K
FAU - Vaidya, Gaurav
AU  - Vaidya G
FAU - Ng, Peter K L
AU  - Ng PK
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Classification/methods
MH  - Consensus Sequence
MH  - DNA, Mitochondrial/chemistry
MH  - Diptera/*classification/genetics
MH  - Electron Transport Complex IV/chemistry/genetics
MH  - *Genetic Variation
MH  - Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2006/10/25 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/10/25 09:00
PHST- 2006/10/25 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/10/25 09:00 [entrez]
AID - P34343329RP55507 [pii]
AID - 10.1080/10635150600969864 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):715-28. doi: 10.1080/10635150600969864.

PMID- 16969944
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - Which random processes describe the tree of life? A large-scale study of
      phylogenetic tree imbalance.
PG  - 685-91
FAU - Blum, Michael G B
AU  - Blum MG
AD  - Department of Human Genetics, University of Michigan, 2017 Palmer Commons, 100
      Washtenaw Avenue, Ann Arbor, Michigan 48109-2218, USA.
FAU - Francois, Olivier
AU  - Francois O
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biodiversity
MH  - *Data Interpretation, Statistical
MH  - *Genetic Speciation
MH  - *Models, Statistical
MH  - *Phylogeny
MH  - Stochastic Processes
EDAT- 2006/09/15 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/09/15 09:00
PHST- 2006/09/15 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/09/15 09:00 [entrez]
AID - 10.1080/10635150600889625 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):685-91. doi: 10.1080/10635150600889625.

PMID- 16969943
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - The effect of combining molecular and morphological data in published
      phylogenetic analyses.
PG  - 677-85
FAU - Wortley, Alexandra H
AU  - Wortley AH
AD  - Department of Plant Sciences, University of Oxford, South Parks Road, Oxford OX1 
      3RB, UK.
FAU - Scotland, Robert W
AU  - Scotland RW
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Anatomy, Comparative/*methods
MH  - Classification/*methods
MH  - *Data Interpretation, Statistical
MH  - *Phylogeny
MH  - Research Design
MH  - Sequence Analysis, DNA/*methods
EDAT- 2006/09/15 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/09/15 09:00
PHST- 2006/09/15 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/09/15 09:00 [entrez]
AID - 10.1080/10635150600899798 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):677-85. doi: 10.1080/10635150600899798.

PMID- 16969942
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - Increasing data transparency and estimating phylogenetic uncertainty in
      supertrees: Approaches using nonparametric bootstrapping.
PG  - 662-76
AB  - The estimation of ever larger phylogenies requires consideration of alternative
      inference strategies, including divide-and-conquer approaches that decompose the 
      global inference problem to a set of smaller, more manageable component problems.
      A prominent locus of research in this area is the development of supertree
      methods, which estimate a composite tree by combining a set of partially
      overlapping component topologies. Although promising, the use of component tree
      topologies as the primary data dissociates supertrees from complexities within
      the underling character data and complicates the evaluation of phylogenetic
      uncertainty. We address these issues by exploring three approaches that variously
      incorporate nonparametric bootstrapping into a common supertree estimation
      algorithm (matrix representation with parsimony, although any algorithm might be 
      used), including bootstrap-weighting, source-tree bootstrapping, and hierarchical
      bootstrapping. We illustrate these procedures by means of hypothetical and
      empirical examples. Our preliminary experiments suggest that these methods have
      the potential to improve the correspondence of supertree estimates to those
      derived from simultaneous analysis of the combined data and to allow uncertainty 
      in supertree topologies to be quantified. The ability to increase the
      transparency of supertrees to the underlying character data has several practical
      implications and sheds new light on an old debate. These methods have been
      implemented in the freely available program, tREeBOOT.
FAU - Moore, Brian R
AU  - Moore BR
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven,
      Connecticut 06520, USA. brian.moore@yale.edu
FAU - Smith, Stephen A
AU  - Smith SA
FAU - Donoghue, Michael J
AU  - Donoghue MJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Data Interpretation, Statistical
MH  - *Models, Statistical
MH  - *Phylogeny
MH  - *Uncertainty
EDAT- 2006/09/15 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/09/15 09:00
PHST- 2006/09/15 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/09/15 09:00 [entrez]
AID - 10.1080/10635150600920693 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):662-76. doi: 10.1080/10635150600920693.

PMID- 16969941
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - A geometric approach to tree shape statistics.
PG  - 652-61
AB  - This article presents a new way to quantify the descriptive ability of tree shape
      statistics. Where before, tree shape statistics were chosen by their ability to
      distinguish between macroevolutionary models, the resolution presented in this
      paper quantifies the ability of a statistic to differentiate between similar and 
      different trees. This is termed the geometric approach to differentiate it from
      the model-based approach previously explored. A distinct advantage of this
      perspective is that it allows evaluation of multiple tree shape statistics
      describing different aspects of tree shape. After developing the methodology, it 
      is applied here to make specific recommendations for a suite of three statistics 
      that may prove useful in applications. The article ends with an application of
      the statistics to clarify the impact of taxa omission on tree shape.
FAU - Matsen, Frederick A
AU  - Matsen FA
AD  - Program for Evolutionary Dynamics and Department of Mathematics, Harvard
      University, One Brattle Square, 6th Floor, Cambridge, Massachusetts 02138, USA.
      matsen@math.harvard.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Data Interpretation, Statistical
MH  - *Models, Statistical
MH  - *Phylogeny
EDAT- 2006/09/15 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/09/15 09:00
PHST- 2006/09/15 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/09/15 09:00 [entrez]
AID - 10.1080/10635150600889617 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):652-61. doi: 10.1080/10635150600889617.

PMID- 16969940
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - Maximizing phylogenetic diversity in biodiversity conservation: Greedy solutions 
      to the Noah's Ark problem.
PG  - 644-51
AB  - The Noah's Ark Problem (NAP) is a comprehensive cost-effectiveness methodology
      for biodiversity conservation that was introduced by Weitzman (1998) and utilizes
      the phylogenetic tree containing the taxa of interest to assess biodiversity.
      Given a set of taxa, each of which has a particular survival probability that can
      be increased at some cost, the NAP seeks to allocate limited funds to conserving 
      these taxa so that the future expected biodiversity is maximized. Finding optimal
      solutions using this framework is a computationally difficult problem to which a 
      simple and efficient "greedy" algorithm has been proposed in the literature and
      applied to conservation problems. We show that, although algorithms of this type 
      cannot produce optimal solutions for the general NAP, there are two restricted
      scenarios of the NAP for which a greedy algorithm is guaranteed to produce
      optimal solutions. The first scenario requires the taxa to have equal
      conservation cost; the second scenario requires an ultrametric tree. The NAP
      assumes a linear relationship between the funding allocated to conservation of a 
      taxon and the increased survival probability of that taxon. This relationship is 
      briefly investigated and one variation is suggested that can also be solved using
      a greedy algorithm.
FAU - Hartmann, Klaas
AU  - Hartmann K
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, Biomathematics Research 
      Centre, University of Canterbury, Christchurch, New Zealand.
FAU - Steel, Mike
AU  - Steel M
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - *Biodiversity
MH  - Computer Simulation
MH  - Conservation of Natural Resources/economics/*methods
MH  - *Models, Theoretical
MH  - *Phylogeny
EDAT- 2006/09/15 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/09/15 09:00
PHST- 2006/09/15 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/09/15 09:00 [entrez]
AID - 10.1080/10635150600873876 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):644-51. doi: 10.1080/10635150600873876.

PMID- 16969939
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - Detecting the node-density artifact in phylogeny reconstruction.
PG  - 637-43
AB  - The node-density effect is an artifact of phylogeny reconstruction that can cause
      branch lengths to be underestimated in areas of the tree with fewer taxa.
      Webster, Payne, and Pagel (2003, Science 301:478) introduced a statistical
      procedure (the "delta" test) to detect this artifact, and here we report the
      results of computer simulations that examine the test's performance. In a sample 
      of 50,000 random data sets, we find that the delta test detects the artifact in
      94.4% of cases in which it is present. When the artifact is not present (n =
      10,000 simulated data sets) the test showed a type I error rate of approximately 
      1.69%, incorrectly reporting the artifact in 169 data sets. Three measures of
      tree shape or "balance" failed to predict the size of the node-density effect.
      This may reflect the relative homogeneity of our randomly generated topologies,
      but emphasizes that nearly any topology can suffer from the artifact, the effect 
      not being confined only to highly unevenly sampled or otherwise imbalanced trees.
      The ability to screen phylogenies for the node-density artifact is important for 
      phylogenetic inference and for researchers using phylogenetic trees to infer
      evolutionary processes, including their use in molecular clock dating.
FAU - Venditti, Chris
AU  - Venditti C
AD  - School of Biological Sciences, University of Reading, Whiteknights RG6 6AJ,
      Reading, England.
FAU - Meade, Andrew
AU  - Meade A
FAU - Pagel, Mark
AU  - Pagel M
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Data Interpretation, Statistical
MH  - *Evolution, Molecular
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - *Research Design
EDAT- 2006/09/15 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/09/15 09:00
PHST- 2006/09/15 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/09/15 09:00 [entrez]
AID - 10.1080/10635150600865567 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):637-43. doi: 10.1080/10635150600865567.

PMID- 16969938
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - Incorporating allelic variation for reconstructing the evolutionary history of
      organisms from multiple genes: An example from Rosa in North America.
PG  - 623-36
AB  - Allelic variation within individuals holds information regarding the
      relationships of organisms, which is expected to be particularly important for
      reconstructing the evolutionary history of closely related taxa. However, little 
      effort has been committed to incorporate such information for reconstructing the 
      phylogeny of organisms. Haplotype trees represent a solution when one
      nonrecombinant marker is considered, but there is no satisfying method when
      multiple genes are to be combined. In this paper, we propose an algorithm that
      converts a distance matrix of alleles to a distance matrix among organisms. This 
      algorithm allows the incorporation of allelic variation for reconstructing the
      phylogeny of organisms from one or more genes. The method is applied to
      reconstruct the phylogeny of the seven native diploid species of Rosa sect.
      Cinnamomeae in North America. The glyceralgehyde 3-phosphate dehydrogenase
      (GAPDH), the triose phosphate isomerase (TPI), and the malate synthase (MS) genes
      were sequenced for 40 individuals from these species. The three genes had little 
      genetic variation, and most species showed incomplete lineage sorting, suggesting
      these species have a recent origin. Despite these difficulties, the networks
      (NeighborNet) of organisms reconstructed from the matrix obtained with the
      algorithm recovered groups that more closely match taxonomic boundaries than did 
      the haplotype trees. The combined network of individuals shows that species west 
      of the Rocky Mountains, Rosa gymnocarpa and R. pisocarpa, form exclusive groups
      and that together they are distinct from eastern species. In the east, three
      groups were found to be exclusive: R. nitida-R. palustris, R. foliolosa, and R.
      blanda-R. woodsii. These groups are congruent with the morphology and the ecology
      of species. The method is also useful for representing hybrid individuals when
      the relationships are reconstructed using a phylogenetic network.
FAU - Joly, Simon
AU  - Joly S
AD  - Institut de recherche en biologie vegetale, Universite de Montreal, 4101
      Sherbrooke Est, Montreal (Quebec), H1X 2B2, Canada. simon.joly@umontreal.ca
FAU - Bruneau, Anne
AU  - Bruneau A
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 1.2.1.- (Glyceraldehyde-3-Phosphate Dehydrogenases)
RN  - EC 2.3.3.9 (Malate Synthase)
RN  - EC 5.3.1.1 (Triose-Phosphate Isomerase)
SB  - IM
MH  - *Algorithms
MH  - *Alleles
MH  - Base Sequence
MH  - Classification/*methods
MH  - *Genetic Variation
MH  - Geography
MH  - Glyceraldehyde-3-Phosphate Dehydrogenases/genetics
MH  - Malate Synthase/genetics
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - North America
MH  - *Phylogeny
MH  - Rosa/*genetics
MH  - Sequence Analysis, DNA
MH  - Species Specificity
MH  - Triose-Phosphate Isomerase/genetics
EDAT- 2006/09/15 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/09/15 09:00
PHST- 2006/09/15 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/09/15 09:00 [entrez]
AID - 10.1080/10635150600863109 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):623-36. doi: 10.1080/10635150600863109.

PMID- 16969937
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - Dating dispersal and radiation in the gymnosperm Gnetum (Gnetales)--clock
      calibration when outgroup relationships are uncertain.
PG  - 610-22
AB  - Most implementations of molecular clocks require resolved topologies. However,
      one of the Bayesian relaxed clock approaches accepts input topologies that
      include polytomies. We explored the effects of resolved and polytomous input
      topologies in a rate-heterogeneous sequence data set for Gnetum, a member of the 
      seed plant lineage Gnetales. Gnetum has 10 species in South America, 1 in
      tropical West Africa, and 20 to 25 in tropical Asia, and explanations for the
      ages of these disjunctions involve long-distance dispersal and/or the breakup of 
      Gondwana. To resolve relationships within Gnetum, we sequenced most of its
      species for six loci from the chloroplast (rbcL, matK, and the trnT-trnF region),
      the nucleus (rITS/5.8S and the LEAFY gene second intron), and the mitochondrion
      (nad1 gene second intron). Because Gnetum has no fossil record, we relied on
      fossils from other Gnetales and from the seed plant lineages conifers, Ginkgo,
      cycads, and angiosperms to constrain a molecular clock and obtain absolute times 
      for within-Gnetum divergence events. Relationships among Gnetales and the other
      seed plant lineages are still unresolved, and we therefore used differently
      resolved topologies, including one that contained a basal polytomy among
      gymnosperms. For a small set of Gnetales exemplars (n = 13) in which rbcL and
      matK satisfied the clock assumption, we also obtained time estimates from a
      strict clock, calibrated with one outgroup fossil. The changing hierarchical
      relationships among seed plants (and accordingly changing placements of distant
      fossils) resulted in small changes of within-Gnetum estimates because
      topologically closest constraints overrode more distant constraints. Regardless
      of the seed plant topology assumed, relaxed clock estimates suggest that the
      extant clades of Gnetum began diverging from each other during the Upper
      Oligocene. Strict clock estimates imply a mid-Miocene divergence. These
      estimates, together with the phylogeny for Gnetum from the six combined data
      sets, imply that the single African species of Gnetum is not a remnant of a once 
      Gondwanan distribution. Miocene and Pliocene range expansions are inferred for
      the Asian subclades of Gnetum, which stem from an ancestor that arrived from
      Africa. These findings fit with seed dispersal by water in several species of
      Gnetum, morphological similarities among apparently young species, and incomplete
      concerted evolution in the nuclear ITS region.
FAU - Won, Hyosig
AU  - Won H
AD  - Department of Biology, University of Missouri-St. Louis, 8001 Natural Bridge
      Road, St. Louis, Missouri 63121, USA. wonhs@snu.ac.kr
FAU - Renner, Susanne S
AU  - Renner SS
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA, Chloroplast/genetics
MH  - DNA, Mitochondrial/genetics
MH  - *Demography
MH  - *Evolution, Molecular
MH  - Fossils
MH  - Geography
MH  - Gnetum/*genetics
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2006/09/15 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/09/15 09:00
PHST- 2006/09/15 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/09/15 09:00 [entrez]
AID - 10.1080/10635150600812619 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):610-22. doi: 10.1080/10635150600812619.

PMID- 16967577
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - Sequence-based species delimitation for the DNA taxonomy of undescribed insects.
PG  - 595-609
AB  - Cataloging the very large number of undescribed species of insects could be
      greatly accelerated by automated DNA based approaches, but procedures for
      large-scale species discovery from sequence data are currently lacking. Here, we 
      use mitochondrial DNA variation to delimit species in a poorly known beetle
      radiation in the genus Rivacindela from arid Australia. Among 468 individuals
      sampled from 65 sites and multiple morphologically distinguishable types,
      sequence variation in three mtDNA genes (cytochrome oxidase subunit 1, cytochrome
      b, 16S ribosomal RNA) was strongly partitioned between 46 or 47 putative species 
      identified with quantitative methods of species recognition based on fixed unique
      ("diagnostic") characters. The boundaries between groups were also recognizable
      from a striking increase in branching rate in clock-constrained calibrated trees.
      Models of stochastic lineage growth (Yule models) were combined with coalescence 
      theory to develop a new likelihood method that determines the point of transition
      from species-level (speciation and extinction) to population-level (coalescence) 
      evolutionary processes. Fitting the location of the switches from speciation to
      coalescent nodes on the ultrametric tree of Rivacindela produced a transition in 
      branching rate occurring at 0.43 Mya, leading to an estimate of 48 putative
      species (confidence interval for the threshold ranging from 47 to 51 clusters
      within 2 logL units). Entities delimited in this way exhibited biological
      properties of traditionally defined species, showing coherence of geographic
      ranges, broad congruence with morphologically recognized species, and levels of
      sequence divergence typical for closely related species of insects. The finding
      of discontinuous evolutionary groupings that are readily apparent in patterns of 
      sequence variation permits largely automated species delineation from DNA surveys
      of local communities as a scaffold for taxonomy in this poorly known insect
      group.
FAU - Pons, Joan
AU  - Pons J
AD  - Department of Entomology, The Natural History Museum, London SW7 5BD, United
      Kingdom.
FAU - Barraclough, Timothy G
AU  - Barraclough TG
FAU - Gomez-Zurita, Jesus
AU  - Gomez-Zurita J
FAU - Cardoso, Anabela
AU  - Cardoso A
FAU - Duran, Daniel P
AU  - Duran DP
FAU - Hazell, Steaphan
AU  - Hazell S
FAU - Kamoun, Sophien
AU  - Kamoun S
FAU - Sumlin, William D
AU  - Sumlin WD
FAU - Vogler, Alfried P
AU  - Vogler AP
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
CIN - Syst Biol. 2009 Aug;58(4):439-42; discussion 442-4. PMID: 20525596
MH  - Animals
MH  - Australia
MH  - Base Sequence
MH  - Classification/*methods
MH  - Coleoptera/*genetics
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - *Genetic Variation
MH  - Geography
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2006/09/14 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/09/14 09:00
PHST- 2006/09/14 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/09/14 09:00 [entrez]
AID - 10.1080/10635150600852011 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):595-609. doi: 10.1080/10635150600852011.

PMID- 16952908
OWN - NLM
STAT- MEDLINE
DCOM- 20061226
LR  - 20060905
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 5
DP  - 2006 Oct
TI  - That awkward age for butterflies: insights from the age of the butterfly
      subfamily Nymphalinae (Lepidoptera: Nymphalidae).
PG  - 703-14
AB  - The study of the historical biogeography of butterflies has been hampered by a
      lack of well-resolved phylogenies and a good estimate of the temporal span over
      which butterflies have evolved. Recently there has been surge of phylogenetic
      hypotheses for various butterfly groups, but estimating ages of divergence is
      still in its infancy for this group of insects. The main problem has been the
      sparse fossil record for butterflies. In this study I have used a surprisingly
      good fossil record for the subfamily Nymphalinae (Lepidoptera: Nymphalidae) to
      estimate the ages of diversification of major lineages using Bayesian relaxed
      clock methods. I have investigated the effects of varying priors on posterior
      estimates in the analyses. For this data set, it is clear that the prior of the
      rate of molecular evolution at the ingroup node had the largest effect on the
      results. Taking this into account, I have been able to arrive at a plausible
      history of lineage splits, which appears to be correlated with known
      paleogeological events. The subfamily appears to have diversified soon after the 
      K/T event about 65 million years ago. Several splits are coincident with major
      paleogeological events, such as the connection of the African and Asian
      continents about 21 million years ago and the presence of a peninsula of land
      connecting the current Greater Antilles to the South American continent 35 to 33 
      million years ago. My results suggest that the age of Nymphalidae is older than
      the 70 million years speculated to be the age of butterflies as a whole.
FAU - Wahlberg, Niklas
AU  - Wahlberg N
AD  - Department of Zoology, Stockholm University, Stockholm, Sweden.
      niklas.wahlberg@zoologi.su.se
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Biodiversity
MH  - Butterflies/*classification/genetics
MH  - Extinction, Biological
MH  - Fossils
MH  - *Phylogeny
EDAT- 2006/09/06 09:00
MHDA- 2006/12/27 09:00
CRDT- 2006/09/06 09:00
PHST- 2006/09/06 09:00 [pubmed]
PHST- 2006/12/27 09:00 [medline]
PHST- 2006/09/06 09:00 [entrez]
AID - M48L4UL3632X3716 [pii]
AID - 10.1080/10635150600913235 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Oct;55(5):703-14. doi: 10.1080/10635150600913235.

PMID- 16861214
OWN - NLM
STAT- MEDLINE
DCOM- 20060821
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 3
DP  - 2006 Jun
TI  - Resolution of phylogenetic conflict in large data sets by increased taxon
      sampling.
PG  - 522-9
FAU - Hedtke, Shannon M
AU  - Hedtke SM
AD  - Section of Integrative Biology and Center for Computational Biology and
      Bioinformatics, University of Texas Austin, (S.M.H), Austin, Texas 78712, USA.
      s.hedtke@mail.utexas.edu
FAU - Townsend, Ted M
AU  - Townsend TM
FAU - Hillis, David M
AU  - Hillis DM
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - Genes, Fungal
MH  - Models, Biological
MH  - *Phylogeny
MH  - Saccharomyces cerevisiae/*classification/*genetics
MH  - Sample Size
EDAT- 2006/07/25 09:00
MHDA- 2006/08/22 09:00
CRDT- 2006/07/25 09:00
PHST- 2006/07/25 09:00 [pubmed]
PHST- 2006/08/22 09:00 [medline]
PHST- 2006/07/25 09:00 [entrez]
AID - T677220075318641 [pii]
AID - 10.1080/10635150600697358 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Jun;55(3):522-9. doi: 10.1080/10635150600697358.

PMID- 16861213
OWN - NLM
STAT- MEDLINE
DCOM- 20060821
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 3
DP  - 2006 Jun
TI  - Uncertainty in the age of fossils and the stratigraphic fit to phylogenies.
PG  - 512-21
AB  - The ages of first appearance of fossil taxa in the stratigraphic record are
      inherently associated to an interval of error or uncertainty, rather than being
      precise point estimates. Contrasting this temporal information with topologies of
      phylogenetic relationships is relevant to many aspects of evolutionary studies.
      Several indices have been proposed to compare the ages of first appearance of
      fossil taxa and phylogenies. For computing most of these indices, the ages of
      first appearance of fossil taxa are currently used as point estimates, ignoring
      their associated errors or uncertainties. The effect of age uncertainty on
      measures of stratigraphic fit to phylogenies is explored here for two indices
      based on the extension of ghost lineages (MSM* and GER). A solution based on
      randomization of the ages of terminal taxa is implemented, resulting in a range
      of possible values for measures of stratigraphic fit to phylogenies, rather than 
      in a precise but arbitrary stratigraphic fit value. Sample cases show that
      ignoring the age uncertainty of fossil taxa can produce misleading results when
      comparing the stratigraphic fit of competing phylogenetic hypotheses. Empirical
      test cases of alternative phylogenies of two dinosaur groups are analyzed through
      the randomization procedure proposed here.
FAU - Pol, Diego
AU  - Pol D
AD  - CONICET, Museo Paleontologico Egidio Feruglio, Av. Fontana 140, Trelew 9100,
      Chubut, Argentina. dpol@mef.org.ar
FAU - Norell, Mark A
AU  - Norell MA
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Dinosaurs
MH  - *Fossils
MH  - Geologic Sediments
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Time Factors
MH  - *Uncertainty
EDAT- 2006/07/25 09:00
MHDA- 2006/08/22 09:00
CRDT- 2006/07/25 09:00
PHST- 2006/07/25 09:00 [pubmed]
PHST- 2006/08/22 09:00 [medline]
PHST- 2006/07/25 09:00 [entrez]
AID - W35512735168540J [pii]
AID - 10.1080/10635150600755446 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Jun;55(3):512-21. doi: 10.1080/10635150600755446.

PMID- 16861212
OWN - NLM
STAT- MEDLINE
DCOM- 20060821
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 3
DP  - 2006 Jun
TI  - The phylogeny of early eureptiles: comparing parsimony and Bayesian approaches in
      the investigation of a basal fossil clade.
PG  - 503-11
AB  - For the first time the phylogenetic relationships of early eureptiles, consisting
      of captorhinids, diapsids, and protorothyridids, are investigated in a modern
      phylogenetic context using both parsimony and Bayesian approaches. Ninety
      parsimony-informative characters and 25 taxa were included in the analyses. The
      Bayesian analysis was run with and without a gamma-shape parameter allowing for
      variable rates across characters. In addition, we ran two more Bayesian analyses 
      that included 42 autapomorphies and thus parsimony-uninformative characters in
      order to test the effect of variable branch lengths. The different analyses
      largely converged to the same topology, suggesting that the "protorothyridid"
      Coelostegus is the sister taxon of all other eureptiles and that the remaining
      "protorothyridids" are paraphyletic. Also, there is a close relationship between 
      diapsids and Anthracodromeus, Cephalerpeton, and Protorothyris, a grouping of
      Thuringothyris with captorhinids, and a variable position of the
      "protorothyridids" Brouffia, Hylonomus, and Paleothyris. The lack of resolution
      in some parts of the tree might be due to "hard polytomies" and short divergence 
      times between the respective taxa. The tree topology is consistent with the
      hypothesis that the temporal fenestrations of diapsid reptiles appear to be the
      consequence of a more lightly built skeleton, indicating a significant ecological
      shift in the early stages of diapsid evolution. Bayesian analysis is a very
      useful additional approach in studies of fossil taxa in which more traditional
      statistical support like the bootstrap is often weak. However, the exclusive use 
      of the Mk model appears suitable only if autapomorphic characters are included,
      whereas the Mk+gamma model performed well with or without autapomorphies.
FAU - Muller, Johannes
AU  - Muller J
AD  - Humboldt-Universitat zu Berlin, Museum fur Naturkunde, Germany.
      johannes.mueller@museum.hu-berlin.de
FAU - Reisz, Robert R
AU  - Reisz RR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - *Fossils
MH  - *Phylogeny
MH  - Reptiles/*classification/*genetics
EDAT- 2006/07/25 09:00
MHDA- 2006/08/22 09:00
CRDT- 2006/07/25 09:00
PHST- 2006/07/25 09:00 [pubmed]
PHST- 2006/08/22 09:00 [medline]
PHST- 2006/07/25 09:00 [entrez]
AID - Q071232813801847 [pii]
AID - 10.1080/10635150600755396 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Jun;55(3):503-11. doi: 10.1080/10635150600755396.

PMID- 16861211
OWN - NLM
STAT- MEDLINE
DCOM- 20060821
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 3
DP  - 2006 Jun
TI  - Reconciling extreme branch length differences: decoupling time and rate through
      the evolutionary history of filmy ferns.
PG  - 485-502
AB  - The rate of molecular evolution is not constant across the Tree of Life.
      Characterizing rate discrepancies and evaluating the relative roles of time and
      rate along branches through the past are both critical to a full understanding of
      evolutionary history. In this study, we explore the interactions of time and rate
      in filmy ferns (Hymenophyllaceae), a lineage with extreme branch length
      differences between the two major clades. We test for the presence of significant
      rate discrepancies within and between these clades, and we separate time and rate
      across the filmy fern phylogeny to simultaneously yield an evolutionary time
      scale of filmy fern diversification and reconstructions of ancestral rates of
      molecular evolution. Our results indicate that the branch length disparity
      observed between the major lineages of filmy ferns is indeed due to a significant
      difference in molecular evolutionary rate. The estimation of divergence times
      reveals that the timing of crown group diversification was not concurrent for the
      two lineages, and the reconstruction of ancestral rates of molecular evolution
      points to a substantial rate deceleration in one of the clades. Further analysis 
      suggests that this may be due to a genome-wide deceleration in the rate of
      nucleotide substitution.
FAU - Schuettpelz, Eric
AU  - Schuettpelz E
AD  - Department of Biology, Duke University, Durham, North Carolina 27708, USA.
      ejs7@duke.edu
FAU - Pryer, Kathleen M
AU  - Pryer KM
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 4.1.1.39 (RbcL protein, plastid)
RN  - EC 4.1.1.39 (Ribulose-Bisphosphate Carboxylase)
SB  - IM
MH  - *Evolution, Molecular
MH  - Ferns/*classification/*genetics
MH  - Genetic Speciation
MH  - Genetic Variation
MH  - Plastids/genetics
MH  - Ribulose-Bisphosphate Carboxylase/genetics
MH  - Time Factors
EDAT- 2006/07/25 09:00
MHDA- 2006/08/22 09:00
CRDT- 2006/07/25 09:00
PHST- 2006/07/25 09:00 [pubmed]
PHST- 2006/08/22 09:00 [medline]
PHST- 2006/07/25 09:00 [entrez]
AID - T411K207R0183W20 [pii]
AID - 10.1080/10635150600755438 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Jun;55(3):485-502. doi: 10.1080/10635150600755438.

PMID- 16861210
OWN - NLM
STAT- MEDLINE
DCOM- 20060821
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 3
DP  - 2006 Jun
TI  - Evolutionary history of vegetative reproduction in Porpidia s.L. (Lichen-forming 
      ascomycota).
PG  - 471-84
AB  - The evolutionary history of gains and losses of vegetative reproductive
      propagules (soredia) in Porpidia s.l., a group of lichen-forming ascomycetes, was
      clarified using Bayesian Markov chain Monte Carlo (MCMC) approaches to monophyly 
      tests and a combined MCMC and maximum likelihood approach to ancestral character 
      state reconstructions. The MCMC framework provided confidence estimates for the
      reconstructions of relationships and ancestral character states, which formed the
      basis for tests of evolutionary hypotheses. Monophyly tests rejected all
      hypotheses that predicted any clustering of reproductive modes in extant taxa. In
      addition, a nearest-neighbor statistic could not reject the hypothesis that the
      vegetative reproductive mode is randomly distributed throughout the group. These 
      results show that transitions between presence and absence of the vegetative
      reproductive mode within Porpidia s.l. occurred several times and independently
      of each other. Likelihood reconstructions of ancestral character states at
      selected nodes suggest that--contrary to previous thought--the ancestor to
      Porpidia s.l. already possessed the vegetative reproductive mode. Furthermore,
      transition rates are reconstructed asymmetrically with the vegetative
      reproductive mode being gained at a much lower rate than it is lost. A cautious
      note has to be added, because a simulation study showed that the ancestral
      character state reconstructions were highly dependent on taxon sampling. However,
      our central conclusions, particularly the higher rate of change from vegetative
      reproductive mode present to absent than vice versa within Porpidia s.l., were
      found to be broadly independent of taxon sampling.
FAU - Buschbom, Jutta
AU  - Buschbom J
AD  - University of Chicago, Committee on Evolutionary Biology, IL 60637, USA.
      j.buschbom@holz.uni-hamburg.de
FAU - Barker, Daniel
AU  - Barker D
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Ascomycota/genetics/*physiology
MH  - *Biological Evolution
MH  - Lichens/genetics/*physiology
MH  - Reproduction/genetics/physiology
EDAT- 2006/07/25 09:00
MHDA- 2006/08/22 09:00
CRDT- 2006/07/25 09:00
PHST- 2006/07/25 09:00 [pubmed]
PHST- 2006/08/22 09:00 [medline]
PHST- 2006/07/25 09:00 [entrez]
AID - V61RT34220484353 [pii]
AID - 10.1080/10635150600697465 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Jun;55(3):471-84. doi: 10.1080/10635150600697465.

PMID- 16861209
OWN - NLM
STAT- MEDLINE
DCOM- 20060821
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 3
DP  - 2006 Jun
TI  - Phylogenetic relationships and historical biogeography of neotropical parrots
      (Psittaciformes: Psittacidae: Arini) inferred from mitochondrial and nuclear DNA 
      sequences.
PG  - 454-70
AB  - Previous hypotheses of phylogenetic relationships among Neotropical parrots were 
      based on limited taxon sampling and lacked support for most internal nodes. In
      this study we increased the number of taxa (29 species belonging to 25 of the 30 
      genera) and gene sequences (6388 base pairs of RAG-1, cyt b, NADH2, ATPase 6,
      ATPase 8, COIII, 12S rDNA, and 16S rDNA) to obtain a stronger molecular
      phylogenetic hypothesis for this group of birds. Analyses of the combined gene
      sequences using maximum likelihood and Bayesian methods resulted in a
      well-supported phylogeny and indicated that amazons and allies are a sister clade
      to macaws, conures, and relatives, and these two clades are in turn a sister
      group to parrotlets. Key morphological and behavioral characters used in previous
      classifications were mapped on the molecular tree and were phylogenetically
      uninformative. We estimated divergence times of taxa using the molecular tree and
      Bayesian and penalized likelihood methods that allow for rate variation in DNA
      substitutions among sites and taxa. Our estimates suggest that the Neotropical
      parrots shared a common ancestor with Australian parrots 59 Mya (million of years
      ago; 95% credibility interval (CrI) 66, 51 Mya), well before Australia separated 
      from Antarctica and South America, implying that ancestral parrots were
      widespread in Gondwanaland. Thus, the divergence of Australian and Neotropical
      parrots could be attributed to vicariance. The three major clades of Neotropical 
      parrots originated about 50 Mya (95% CrI 57, 41 Mya), coinciding with periods of 
      higher sea level when both Antarctica and South America were fragmented with
      transcontinental seaways, and likely isolated the ancestors of modern Neotropical
      parrots in different regions in these continents. The correspondence between
      major paleoenvironmental changes in South America and the diversification of
      genera in the clade of amazons and allies between 46 and 16 Mya suggests they
      diversified exclusively in South America. Conversely, ancestors of parrotlets and
      of macaws, conures, and allies may have been isolated in Antarctica and/or the
      southern cone of South America, and only dispersed out of these southern regions 
      when climate cooled and Antarctica became ice-encrusted about 35 Mya. The
      subsequent radiation of macaws and their allies in South America beginning about 
      28 Mya (95% CrI 22, 35 Mya) coincides with the uplift of the Andes and the
      subsequent formation of dry, open grassland habitats that would have facilitated 
      ecological speciation via niche expansion from forested habitats.
FAU - Tavares, Erika Sendra
AU  - Tavares ES
AD  - Departamento de Genetica e Biologia Evolutiva, Instituto de Biociencias,
      Universidade de Sao Paulo, R. do Matao 277, (C.Y.M.), 05508-090, Sao Paulo, SP,
      Brazil.
FAU - Baker, Allan J
AU  - Baker AJ
FAU - Pereira, Sergio Luiz
AU  - Pereira SL
FAU - Miyaki, Cristina Yumi
AU  - Miyaki CY
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Cell Nucleus/*genetics
MH  - DNA, Mitochondrial/analysis/*genetics
MH  - Genetic Speciation
MH  - *Geography
MH  - Parrots/*classification/*genetics
MH  - *Phylogeny
MH  - Tropical Climate
EDAT- 2006/07/25 09:00
MHDA- 2006/08/22 09:00
CRDT- 2006/07/25 09:00
PHST- 2006/07/25 09:00 [pubmed]
PHST- 2006/08/22 09:00 [medline]
PHST- 2006/07/25 09:00 [entrez]
AID - T2N060250W513471 [pii]
AID - 10.1080/10635150600697390 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Jun;55(3):454-70. doi: 10.1080/10635150600697390.

PMID- 16861208
OWN - NLM
STAT- MEDLINE
DCOM- 20060821
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 3
DP  - 2006 Jun
TI  - Reciprocal illumination in the gene content tree of life.
PG  - 441-53
AB  - Phylogenies based on gene content rely on statements of primary homology to
      characterize gene presence or absence. These statements (hypotheses) are usually 
      determined by techniques based on threshold similarity or distance measurements
      between genes. This fundamental but problematic step can be examined by
      evaluating each homology hypothesis by the extent to which it is corroborated by 
      the rest of the data. Here we test the effects of varying the stringency for
      making primary homology statements using a range of similarity (e-value) cutoffs 
      in 166 fully sequenced and annotated genomes spanning the tree of life. By
      evaluating each resulting data set with tree-based measurements of character
      consistency and information content, we find a set of homology statements that
      optimizes overall corroboration. The resulting data set produces well-resolved
      and well-supported trees of life and greatly ameliorates previously noted
      inconsistencies such as the misclassification of small genomes. The method
      presented here, which can be used to test any technique for recognizing primary
      homology, provides an objective framework for evaluating phylogenetic hypotheses 
      and data sets for the tree of life. It also can serve as a technique for
      identifying well-corroborated sets of homologous genes for functional genomic
      applications.
FAU - Lienau, E Kurt
AU  - Lienau EK
AD  - American Museum of Natural History, Molecular Laboratories, Central Park West at 
      79th Street, (P.J.P.), New York, New York 10024, USA.
FAU - DeSalle, Rob
AU  - DeSalle R
FAU - Rosenfeld, Jeffrey A
AU  - Rosenfeld JA
FAU - Planet, Paul J
AU  - Planet PJ
LA  - eng
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - Databases, Genetic
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Sequence Homology
EDAT- 2006/07/25 09:00
MHDA- 2006/08/22 09:00
CRDT- 2006/07/25 09:00
PHST- 2006/07/25 09:00 [pubmed]
PHST- 2006/08/22 09:00 [medline]
PHST- 2006/07/25 09:00 [entrez]
AID - L435106X8P683358 [pii]
AID - 10.1080/10635150600697416 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Jun;55(3):441-53. doi: 10.1080/10635150600697416.

PMID- 16861207
OWN - NLM
STAT- MEDLINE
DCOM- 20060821
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 3
DP  - 2006 Jun
TI  - Supertree bootstrapping methods for assessing phylogenetic variation among genes 
      in genome-scale data sets.
PG  - 426-40
AB  - Nonparamtric bootstrapping methods may be useful for assessing confidence in a
      supertree inference. We examined the performance of two supertree bootstrapping
      methods on four published data sets that each include sequence data from more
      than 100 genes. In "input tree bootstrapping," input gene trees are sampled with 
      replacement and then combined in replicate supertree analyses; in "stratified
      bootstrapping," trees from each gene's separate (conventional) bootstrap tree set
      are sampled randomly with replacement and then combined. Generally, support
      values from both supertree bootstrap methods were similar or slightly lower than 
      corresponding bootstrap values from a total evidence, or supermatrix, analysis.
      Yet, supertree bootstrap support also exceeded supermatrix bootstrap support for 
      a number of clades. There was little overall difference in support scores between
      the input tree and stratified bootstrapping methods. Results from supertree
      bootstrapping methods, when compared to results from corresponding supermatrix
      bootstrapping, may provide insights into patterns of variation among genes in
      genome-scale data sets.
FAU - Burleigh, J Gordon
AU  - Burleigh JG
AD  - Section of Evolution and Ecology, University of California, (J.G.B.), Davis, CA
      95616, USA. jgburleigh@ucdavis.edu
FAU - Driskell, Amy C
AU  - Driskell AC
FAU - Sanderson, Michael J
AU  - Sanderson MJ
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Genetic Variation/*genetics
MH  - Genome/*genetics
MH  - *Genomics
MH  - Models, Biological
MH  - *Phylogeny
MH  - Plants/genetics
MH  - Yeasts/genetics
EDAT- 2006/07/25 09:00
MHDA- 2006/08/22 09:00
CRDT- 2006/07/25 09:00
PHST- 2006/07/25 09:00 [pubmed]
PHST- 2006/08/22 09:00 [medline]
PHST- 2006/07/25 09:00 [entrez]
AID - GV57R640912Q71M7 [pii]
AID - 10.1080/10635150500541722 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Jun;55(3):426-40. doi: 10.1080/10635150500541722.

PMID- 16861206
OWN - NLM
STAT- MEDLINE
DCOM- 20060821
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 3
DP  - 2006 Jun
TI  - Molecular phylogenetics and evolutionary diversification of labyrinth fishes
      (Perciformes: Anabantoidei).
PG  - 374-97
AB  - Labyrinth fishes (Perciformes: Anabantoidei) are primary freshwater fishes with a
      disjunct African-Asian distribution that exhibit a wide variety of morphological 
      and behavioral traits. These intrinsic features make them particularly well
      suited for studying patterns and processes of evolutionary diversification. We
      reconstructed the first molecular-based phylogenetic hypothesis of anabantoid
      intrarelationships using both mitochondrial and nuclear nucleotide sequence data 
      to address anabantoid evolution. The mitochondrial data set included the complete
      cytochrome b, partial 12S rRNA, complete tRNA Val, and partial 16S rRNA genes
      (3332 bp) of 57 species representing all 19 anabantoid genera. The nuclear data
      set included the partial RAG1 gene (1494 bp) of 21 representative species. The
      phylogenetic analyses of a combined (mitochondrial+nuclear) data set recovered
      almost fully resolved trees at the intrafamily level with different methods of
      phylogenetic inference. Phylogenetic relationships at this taxonomic level were
      compared with previous morphology-based hypotheses. In particular, the enigmatic 
      pike-head (Luciocephalus) was confidently placed within the "spiral egg" clade,
      thus resolving the long-standing controversy on its relative phylogenetic
      position. The molecular phylogeny was used to study the evolution of the
      different forms of parental care within the suborder. Our results suggest that
      the evolution of breeding behavior in anabantoids is highly correlated with
      phylogeny, and that brood care evolved three times independently from an
      ancestral free spawning condition without parental care. Ancestral character
      state reconstructions under maximum parsimony and maximum likelihood further
      indicated that both bubble nesting and mouthbrooding have evolved recurrently
      during anabantoid evolution. The new phylogenetic framework was also used to test
      alternative biogeographic hypotheses that account for the disjunct African-Asian 
      distribution. Molecular divergence time estimates support either a drift
      vicariance linked to the breakup of Gondwana or Late Mesozoic Early Tertiary
      dispersal from Africa to Asia or vice versa.
FAU - Ruber, Lukas
AU  - Ruber L
AD  - Department of Zoology, The Natural History Museum, Cromwell Road, London, SW7
      5BD, UK. l.ruber@nhm.ac.uk
FAU - Britz, Ralf
AU  - Britz R
FAU - Zardoya, Rafael
AU  - Zardoya R
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Behavior, Animal
MH  - DNA/genetics
MH  - *Evolution, Molecular
MH  - Genetic Speciation
MH  - Genetic Variation
MH  - Perciformes/*classification/*genetics
MH  - *Phylogeny
MH  - Population Dynamics
EDAT- 2006/07/25 09:00
MHDA- 2006/08/22 09:00
CRDT- 2006/07/25 09:00
PHST- 2006/07/25 09:00 [pubmed]
PHST- 2006/08/22 09:00 [medline]
PHST- 2006/07/25 09:00 [entrez]
AID - U8108X5730H11T51 [pii]
AID - 10.1080/10635150500541664 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Jun;55(3):374-97. doi: 10.1080/10635150500541664.

PMID- 16861205
OWN - NLM
STAT- MEDLINE
DCOM- 20060821
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 3
DP  - 2006 Jun
TI  - Taxonomic indexing--extending the role of taxonomy.
PG  - 367-73
AB  - Taxonomic indexing refers to a new array of taxonomically intelligent network
      services that use nomenclatural principles and elements of expert taxonomic
      knowledge to manage information about organisms. Taxonomic indexing was
      introduced to help manage the increasing amounts of digital information about
      biology. It has been designed to form a near basal layer in a layered
      cyberinfrastructure that deals with biological information. Taxonomic Indexing
      accommodates the special problems of using names of organisms to index biological
      material. It links alternative names for the same entity (reconciliation), and
      distinguishes between uses of the same name for different entities
      (disambiguation), and names are placed within an indefinite number of
      hierarchical schemes. In order to access all information on all organisms,
      Taxonomic indexing must be able to call on a registry of all names in all forms
      for all organisms. NameBank has been developed to meet that need. Taxonomic
      indexing is an area of informatics that overlaps with taxonomy, is dependent on
      the expert input of taxonomists, and reveals the relevance of the discipline to a
      wide audience.
FAU - Patterson, David J
AU  - Patterson DJ
AD  - Marine Biological Laboratory, Woods Hole, Massachusetts 02543, USA.
      dpatterson@mbl.edu
FAU - Remsen, David
AU  - Remsen D
FAU - Marino, William A
AU  - Marino WA
FAU - Norton, Cathy
AU  - Norton C
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Abstracting and Indexing as Topic/*methods/trends
MH  - Animals
MH  - Botany
MH  - Classification/*methods
MH  - *Terminology as Topic
MH  - Zoology
EDAT- 2006/07/25 09:00
MHDA- 2006/08/22 09:00
CRDT- 2006/07/25 09:00
PHST- 2006/07/25 09:00 [pubmed]
PHST- 2006/08/22 09:00 [medline]
PHST- 2006/07/25 09:00 [entrez]
AID - G86688571837406H [pii]
AID - 10.1080/10635150500541680 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Jun;55(3):367-73. doi: 10.1080/10635150500541680.

PMID- 16857652
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20060721
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - Phylogeny and biogeography of a cosmopolitan frog radiation: Late cretaceous
      diversification resulted in continent-scale endemism in the family ranidae.
PG  - 579-94
AB  - Ranidae is a large anuran group with a nearly cosmopolitan distribution. We
      investigated the phylogenetic relationships and early biogeographic history of
      ranid frogs, using 104 representatives of all subfamilies and families, sampled
      from throughout their distribution. Analyses of approximately 1570 bp of nuclear 
      gene fragments (Rag-1, rhod, Tyr) and approximately 2100 bp of the mitochondrial 
      genome (12S rRNA, tRNAVAL, 16S rRNA) indicate that the monophyly of several taxa 
      can be rejected with high confidence. Our tree is characterized by a clear
      historical association of each major clade with one Gondwanan plate. This
      prevalence of continent-scale endemism suggests that plate tectonics has played a
      major role in the distribution of ranid frogs. We performed dispersal-vicariance 
      analyses, as well as analyses constrained by paleogeographic data, to estimate
      ancestral distributions during early ranid diversification. Additionally, we used
      molecular clock analyses to evaluate whether these scenarios fit the temporal
      framework of continental breakup. Our analyses suggest that a scenario in which
      the ancestors of several clades (Rhacophorinae, Dicroglossinae, Raninae) reached 
      Eurasia via the Indian subcontinent, and the ancestor of Ceratobatrachinae
      entered via the Australia-New Guinea plate, best fits the paleogeographic models 
      and requires the fewest number of dispersal/vicariance events. However, several
      alternatives, in which part of the ranid fauna colonized Laurasia from Africa,
      are not significantly worse. Most importantly, all hypotheses make clear
      predictions as to where to expect key fossils and where to sample other living
      ranids, and thus constitute a strong basis for further research.
FAU - Bossuyt, Franky
AU  - Bossuyt F
AD  - Biology Department, Unit of Ecology &amp; Systematics, Vrije Universiteit Brussel,
      Pleinlaan 2, B-1050 Brussels, Belgium. fbossuyt@vub.ac.be
FAU - Brown, Rafe M
AU  - Brown RM
FAU - Hillis, David M
AU  - Hillis DM
FAU - Cannatella, David C
AU  - Cannatella DC
FAU - Milinkovitch, Michel C
AU  - Milinkovitch MC
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA, Mitochondrial/genetics
MH  - *Demography
MH  - *Evolution, Molecular
MH  - *Genetic Speciation
MH  - Geography
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Models, Theoretical
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Ranidae/*genetics
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2006/07/22 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/07/22 09:00
PHST- 2006/07/22 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/07/22 09:00 [entrez]
AID - R5051H424T28H706 [pii]
AID - 10.1080/10635150600812551 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):579-94. doi: 10.1080/10635150600812551.

PMID- 16857651
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20060721
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - Multiple origins of the Juan Fernandez kelpfish fauna and evidence for frequent
      and unidirectional dispersal of cirrhitoid fishes across the South Pacific.
PG  - 566-78
AB  - Phylogenetic relationships were reconstructed among chironemid fishes based on
      morphological and molecular (lrRNA, NADH4, S7 ribosomal protein) characters. Two 
      sympatric species from Juan Fernandez in the southeast Pacific are not sister
      taxa, but rather exhibit independent relationships to Australian/New Zealand
      chironemids. The most plausible explanation for these relationships and
      contemporary distributions is an Australian/New Zealand origin of the family,
      followed by two trans-Pacific dispersal and colonization events, facilitated by
      larval entrapment within the West Wind Drift. This study demonstrates that the
      diversity of taxa on an island can reflect multiple colonizations, rather than in
      situ diversification, even in the case of very small, isolated, and geologically 
      recent islands. When taken in conjunction with studies of related taxa, our
      results indicate that transoceanic dispersal of temperate cirrhitoid fishes in
      the South Pacific has been frequent and unidirectional. Molecular estimates of
      divergence time between southeast Pacific chironemids and their western relatives
      predate the emergence of Juan Fernandez, consistent with hypotheses that much of 
      the marine nearshore faunas of young southeast Pacific islands may be the product
      of successive transfer from older, now submerged islands.
FAU - Burridge, Christopher P
AU  - Burridge CP
AD  - School of Life and Environmental Sciences, Deakin University, PO Box 423,
      Warrnambool, Victoria 3280, Australia. Chris.Burridge@stonebow.otago.ac.nz
FAU - Melendez, C Roberto
AU  - Melendez CR
FAU - Dyer, Brian S
AU  - Dyer BS
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA Primers
MH  - *Demography
MH  - Geography
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Pacific Ocean
MH  - Perciformes/*anatomy &amp; histology/*genetics/physiology
MH  - *Phylogeny
MH  - Population Dynamics
MH  - Sequence Analysis, DNA
EDAT- 2006/07/22 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/07/22 09:00
PHST- 2006/07/22 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/07/22 09:00 [entrez]
AID - UQ0J367645220030 [pii]
AID - 10.1080/10635150600812585 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):566-78. doi: 10.1080/10635150600812585.

PMID- 16857650
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20060721
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - Searching for convergence in phylogenetic Markov chain Monte Carlo.
PG  - 553-65
AB  - Markov chain Monte Carlo (MCMC) is a methodology that is gaining widespread use
      in the phylogenetics community and is central to phylogenetic software packages
      such as MrBayes. An important issue for users of MCMC methods is how to select
      appropriate values for adjustable parameters such as the length of the Markov
      chain or chains, the sampling density, the proposal mechanism, and, if
      Metropolis-coupled MCMC is being used, the number of heated chains and their
      temperatures. Although some parameter settings have been examined in detail in
      the literature, others are frequently chosen with more regard to computational
      time or personal experience with other data sets. Such choices may lead to
      inadequate sampling of tree space or an inefficient use of computational
      resources. We performed a detailed study of convergence and mixing for 70
      randomly selected, putatively orthologous protein sets with different sizes and
      taxonomic compositions. Replicated runs from multiple random starting points
      permit a more rigorous assessment of convergence, and we developed two novel
      statistics, delta and epsilon, for this purpose. Although likelihood values
      invariably stabilized quickly, adequate sampling of the posterior distribution of
      tree topologies took considerably longer. Our results suggest that multimodality 
      is common for data sets with 30 or more taxa and that this results in slow
      convergence and mixing. However, we also found that the pragmatic approach of
      combining data from several short, replicated runs into a "metachain" to estimate
      bipartition posterior probabilities provided good approximations, and that such
      estimates were no worse in approximating a reference posterior distribution than 
      those obtained using a single long run of the same length as the metachain.
      Precision appears to be best when heated Markov chains have low temperatures,
      whereas chains with high temperatures appear to sample trees with high posterior 
      probabilities only rarely.
FAU - Beiko, Robert G
AU  - Beiko RG
AD  - ARC Centre in Bioinformatics and Institute for Molecular Bioscience, The
      University of Queensland, Brisbane, Queensland 4072, Australia. r.beiko@gmail.com
FAU - Keith, Jonathan M
AU  - Keith JM
FAU - Harlow, Timothy J
AU  - Harlow TJ
FAU - Ragan, Mark A
AU  - Ragan MA
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Data Interpretation, Statistical
MH  - *Markov Chains
MH  - *Models, Genetic
MH  - *Monte Carlo Method
MH  - *Phylogeny
EDAT- 2006/07/22 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/07/22 09:00
PHST- 2006/07/22 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/07/22 09:00 [entrez]
AID - G6213Q5W084H6115 [pii]
AID - 10.1080/10635150600812544 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):553-65. doi: 10.1080/10635150600812544.

PMID- 16785212
OWN - NLM
STAT- MEDLINE
DCOM- 20061120
LR  - 20060620
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 4
DP  - 2006 Aug
TI  - Approximate likelihood-ratio test for branches: A fast, accurate, and powerful
      alternative.
PG  - 539-52
AB  - We revisit statistical tests for branches of evolutionary trees reconstructed
      upon molecular data. A new, fast, approximate likelihood-ratio test (aLRT) for
      branches is presented here as a competitive alternative to nonparametric
      bootstrap and Bayesian estimation of branch support. The aLRT is based on the
      idea of the conventional LRT, with the null hypothesis corresponding to the
      assumption that the inferred branch has length 0. We show that the LRT statistic 
      is asymptotically distributed as a maximum of three random variables drawn from
      the chi(0)2 + chi(1)2 distribution. The new aLRT of interior branch uses this
      distribution for significance testing, but the test statistic is approximated in 
      a slightly conservative but practical way as 2(l1- l2), i.e., double the
      difference between the maximum log-likelihood values corresponding to the best
      tree and the second best topological arrangement around the branch of interest.
      Such a test is fast because the log-likelihood value l2 is computed by optimizing
      only over the branch of interest and the four adjacent branches, whereas other
      parameters are fixed at their optimal values corresponding to the best ML tree.
      The performance of the new test was studied on simulated 4-, 12-, and 100-taxon
      data sets with sequences of different lengths. The aLRT is shown to be accurate, 
      powerful, and robust to certain violations of model assumptions. The aLRT is
      implemented within the algorithm used by the recent fast maximum likelihood tree 
      estimation program PHYML (Guindon and Gascuel, 2003).
FAU - Anisimova, Maria
AU  - Anisimova M
AD  - Equipe Methodes et Algorithmes pour la Bioinformatique LIRMM-CNRS, Universite
      Montpellier II, Montpellier 34392, France. manisimova@hotmail.com
FAU - Gascuel, Olivier
AU  - Gascuel O
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Likelihood Functions
MH  - *Models, Statistical
MH  - *Phylogeny
EDAT- 2006/06/21 09:00
MHDA- 2006/12/09 09:00
CRDT- 2006/06/21 09:00
PHST- 2006/06/21 09:00 [pubmed]
PHST- 2006/12/09 09:00 [medline]
PHST- 2006/06/21 09:00 [entrez]
AID - T808388N86673K61 [pii]
AID - 10.1080/10635150600755453 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Aug;55(4):539-52. doi: 10.1080/10635150600755453.

PMID- 16684720
OWN - NLM
STAT- MEDLINE
DCOM- 20060821
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 3
DP  - 2006 Jun
TI  - Differentiating between hypotheses of lineage sorting and introgression in New
      Zealand alpine cicadas (Maoricicada Dugdale).
PG  - 411-25
AB  - Lineage sorting and introgression can lead to incongruence among gene
      phylogenies, complicating the inference of species trees for large groups of taxa
      that have recently and rapidly radiated. In addition, it can be difficult to
      determine which of these processes is responsible for this incongruence. We
      explore these issues with the radiation of New Zealand alpine cicadas of the
      genus Maoricicada Dugdale. Gene trees were estimated from four putative
      independent loci: mitochondrial DNA (2274 nucleotides), elongation factor 1-alpha
      (1275 nucleotides), period (1709 nucleotides), and calmodulin (678 nucleotides). 
      We reconstructed phylogenies using maximum likelihood and Bayesian methods from
      44 individuals representing the 19 species and subspecies of Maoricicada and two 
      outgroups. Species-level relationships were reconstructed using a novel extension
      of gene tree parsimony, whereby gene trees were weighted by their Bayesian
      posterior probabilities. The inferred gene trees show marked incongruence in the 
      placement of some taxa, especially the enigmatic forest and scrub dwelling
      species, M. iolanthe. Using the species tree estimated by gene tree parsimony, we
      simulated coalescent gene trees in order to test the null hypothesis that the
      nonrandom placement of M. iolanthe among gene trees has arisen by chance. Under
      the assumptions of constant population size, known generation time, and panmixia,
      we were able to reject this null hypothesis. Furthermore, because the two
      alternative placements of M. iolanthe are in each case with species that share a 
      similar song structure, we conclude that it is more likely that an ancient
      introgression event rather than lineage sorting has caused this incongruence.
FAU - Buckley, Thomas R
AU  - Buckley TR
AD  - Landcare Research, Private Bag, 92170, Auckland, New Zealand,
      buckleyt@landcareresearch.co.nz
FAU - Cordeiro, Michael
AU  - Cordeiro M
FAU - Marshall, David C
AU  - Marshall DC
FAU - Simon, Chris
AU  - Simon C
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Calmodulin)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Insect Proteins)
SB  - IM
MH  - Animals
MH  - Calmodulin/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Hemiptera/*classification/*genetics
MH  - Insect Proteins/genetics
MH  - *Models, Biological
MH  - New Zealand
MH  - *Phylogeny
EDAT- 2006/05/11 09:00
MHDA- 2006/08/22 09:00
CRDT- 2006/05/11 09:00
PHST- 2006/05/11 09:00 [pubmed]
PHST- 2006/08/22 09:00 [medline]
PHST- 2006/05/11 09:00 [entrez]
AID - G023N5663R076T91 [pii]
AID - 10.1080/10635150600697283 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Jun;55(3):411-25. doi: 10.1080/10635150600697283.

PMID- 16684719
OWN - NLM
STAT- MEDLINE
DCOM- 20060821
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 3
DP  - 2006 Jun
TI  - Patterns of speciation in the Yucca moths: parallel species radiations within the
      Tegeticula yuccasella species complex.
PG  - 398-410
AB  - The interaction between yuccas and yucca moths has been central to understanding 
      the origin and loss of obligate mutualism and mutualism reversal. Previous
      systematic research using mtDNA sequence data and characters associated with
      genitalic morphology revealed that a widespread pollinator species in the genus
      Tegeticula was in fact a complex of pollinator species that differed in host use 
      and the placement of eggs into yucca flowers. Within this mutualistic clade two
      nonpollinating "cheater" species evolved. Cheaters feed on yucca seeds but lack
      the tentacular mouthparts necessary for yucca pollination. Previous work
      suggested that the species complex formed via a rapid radiation within the last
      several million years. In this study, we use an expanded mtDNA sequence data set 
      and AFLP markers to examine the phylogenetic relationships among this rapidly
      diverging clade of moths and compare these relationships to patterns in genitalic
      morphology. Topologies obtained from analyses of the mtDNA and AFLP data differed
      significantly. Both data sets, however, corroborated the hypothesis of a rapid
      species radiation and suggested that there were likely two independent species
      radiations. Morphological analyses based on oviposition habit produced species
      groupings more similar to the AFLP topology than the mtDNA topology and suggested
      the two radiations coincided with differences in oviposition habit. The evolution
      of cheating was reaffirmed to have evolved twice and the closest pollinating
      relative for one cheater species was identified by both mtDNA and AFLP markers.
      For the other cheater species, however, the closest pollinating relative remains 
      ambiguous, and mtDNA, AFLP, and morphological data suggest this cheater species
      may be diverged based on host use. Much of the divergence in the species complex 
      can be explained by geographic isolation associated with the evolution of two
      oviposition habits.
FAU - Althoff, David M
AU  - Althoff DM
AD  - Department of Biological Sciences, University of Idaho, Moscow, Idaho 83844-3051,
      USA. dmalthof@syr.edu
FAU - Segraves, Kari A
AU  - Segraves KA
FAU - Leebens-Mack, James
AU  - Leebens-Mack J
FAU - Pellmyr, Olle
AU  - Pellmyr O
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - DNA/genetics
MH  - *Evolution, Molecular
MH  - Female
MH  - *Genetic Speciation
MH  - Male
MH  - Moths/*classification/*genetics
MH  - *Phylogeny
EDAT- 2006/05/11 09:00
MHDA- 2006/08/22 09:00
CRDT- 2006/05/11 09:00
PHST- 2006/05/11 09:00 [pubmed]
PHST- 2006/08/22 09:00 [medline]
PHST- 2006/05/11 09:00 [entrez]
AID - JL84070346712X33 [pii]
AID - 10.1080/10635150600697325 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Jun;55(3):398-410. doi: 10.1080/10635150600697325.
</pre></body></html>