<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0035)https://www.ncbi.nlm.nih.gov/pubmed -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"></head><body><pre>PMID- 12079647
OWN - NLM
STAT- MEDLINE
DCOM- 20020930
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 3
DP  - 2002 Jun
TI  - Model misspecification and probabilistic tests of topology: evidence from
      empirical data sets.
PG  - 509-23
AB  - Probabilistic tests of topology offer a powerful means of evaluating competing
      phylogenetic hypotheses. The performance of the nonparametric Shimodaira-Hasegawa
      (SH) test, the parametric Swofford-Olsen-Waddell-Hillis (SOWH) test, and Bayesian
      posterior probabilities were explored for five data sets for which all the
      phylogenetic relationships are known with a very high degree of certainty. These 
      results are consistent with previous simulation studies that have indicated a
      tendency for the SOWH test to be prone to generating Type 1 errors because of
      model misspecification coupled with branch length heterogeneity. These results
      also suggest that the SOWH test may accord overconfidence in the true topology
      when the null hypothesis is in fact correct. In contrast, the SH test was
      observed to be much more conservative, even under high substitution rates and
      branch length heterogeneity. For some of those data sets where the SOWH test
      proved misleading, the Bayesian posterior probabilities were also misleading. The
      results of all tests were strongly influenced by the exact substitution model
      assumptions. Simple models, especially those that assume rate homogeneity among
      sites, had a higher Type 1 error rate and were more likely to generate misleading
      posterior probabilities. For some of these data sets, the commonly used
      substitution models appear to be inadequate for estimating appropriate levels of 
      uncertainty with the SOWH test and Bayesian methods. Reasons for the differences 
      in statistical power between the two maximum likelihood tests are discussed and
      are contrasted with the Bayesian approach.
FAU - Buckley, Thomas R
AU  - Buckley TR
AD  - Department of Biology, Duke University, Durham, North Carolina, USA, and Landcare
      Research, Mt. Albert, Auckland, New Zealand. buckleyt@landcareresearch.co.nz
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Nucleotides)
RN  - 9014-25-9 (RNA, Transfer)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - DNA, Mitochondrial/metabolism
MH  - Databases as Topic
MH  - Models, Statistical
MH  - Nucleotides/chemistry
MH  - *Phylogeny
MH  - RNA, Transfer/metabolism
EDAT- 2002/06/25 10:00
MHDA- 2002/10/02 04:00
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/10/02 04:00 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/10635150290069922 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Jun;51(3):509-23. doi: 10.1080/10635150290069922.

PMID- 12079646
OWN - NLM
STAT- MEDLINE
DCOM- 20020930
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 3
DP  - 2002 Jun
TI  - An approximately unbiased test of phylogenetic tree selection.
PG  - 492-508
AB  - An approximately unbiased (AU) test that uses a newly devised multiscale
      bootstrap technique was developed for general hypothesis testing of regions in an
      attempt to reduce test bias. It was applied to maximum-likelihood tree selection 
      for obtaining the confidence set of trees. The AU test is based on the theory of 
      Efron et al. (Proc. Natl. Acad. Sci. USA 93:13429-13434; 1996), but the new
      method provides higher-order accuracy yet simpler implementation. The AU test,
      like the Shimodaira-Hasegawa (SH) test, adjusts the selection bias overlooked in 
      the standard use of the bootstrap probability and Kishino-Hasegawa tests. The
      selection bias comes from comparing many trees at the same time and often leads
      to overconfidence in the wrong trees. The SH test, though safe to use, may
      exhibit another type of bias such that it appears conservative. Here I show that 
      the AU test is less biased than other methods in typical cases of tree selection.
      These points are illustrated in a simulation study as well as in the analysis of 
      mammalian mitochondrial protein sequences. The theoretical argument provides a
      simple formula that covers the bootstrap probability test, the Kishino-Hasegawa
      test, the AU test, and the Zharkikh-Li test. A practical suggestion is provided
      as to which test should be used under particular circumstances.
FAU - Shimodaira, Hidetoshi
AU  - Shimodaira H
AD  - Institute of Statistical Mathematics, 4-6-7 Minami-Azabu, Minatoku, Tokyo
      106-8569, Japan. shimo@ism.ac.jp
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Mitochondria/metabolism
MH  - Models, Statistical
MH  - *Phylogeny
EDAT- 2002/06/25 10:00
MHDA- 2002/10/02 04:00
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/10/02 04:00 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/10635150290069913 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Jun;51(3):492-508. doi: 10.1080/10635150290069913.

PMID- 12079645
OWN - NLM
STAT- MEDLINE
DCOM- 20020930
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 3
DP  - 2002 Jun
TI  - Analyzing developmental sequences within a phylogenetic framework.
PG  - 478-91
AB  - Heterochrony is important as a potential mechanism of evolutionary change.
      However, the analysis of developmental timing data within a phylogenetic
      framework to identify important shifts has proven difficult. In particular,
      analytical problems with sequence (event) heterochrony revolve around the lack of
      an absolute time frame in development to allow standardization of timing data
      across species. An important breakthrough in this regard is the method of
      "event-pairing," which compares the relative timing of developmental events in a 
      pairwise fashion. The resulting event-pair-encoded data can be mapped onto a
      phylogeny, which can provide important biological information. However,
      event-paired data are cumbersome to work with and lack a rigorous quantitative
      framework under which to analyze them. Critically, the otherwise advantageous
      relativity of event-pairing prevents an assessment of whether one or both events 
      in a single event-pair have changed position during evolutionary history.
      Building on the method of event-pairing, we describe a protocol whereby
      event-pair transformations along a given branch are analyzed en bloc. Our method 
      of "event-pair cracking" thereby allows developmental timing data to be analyzed 
      quantitatively within a phylogenetic framework to infer key heterochronic shifts.
      We demonstrate the utility of event-pair cracking through a worked example and
      show how it provides a set of desired features identified by previous authors.
FAU - Jeffery, Jonathan E
AU  - Jeffery JE
AD  - Department of Anatomy and Developmental Biology, St. George's Hospital Medical
      School, Tooting, London SW17 ORE, UK.
FAU - Richardson, Michael K
AU  - Richardson MK
FAU - Coates, Michael I
AU  - Coates MI
FAU - Bininda-Emonds, Olaf R P
AU  - Bininda-Emonds OR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Mammals/classification
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2002/06/25 10:00
MHDA- 2002/10/02 04:00
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/10/02 04:00 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/10635150290069904 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Jun;51(3):478-91. doi: 10.1080/10635150290069904.

PMID- 12079644
OWN - NLM
STAT- MEDLINE
DCOM- 20020930
LR  - 20031114
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 3
DP  - 2002 Jun
TI  - Robustness of ancestral state estimates: evolution of life history strategy in
      ichneumonoid parasitoids.
PG  - 450-77
AB  - We test hypotheses for the evolution of a life history trait among a group of
      parasitoid wasps (Hymenoptera: Ichneumonoidea), namely, the transition among
      koinobiont parasitoids (parasitoids whose hosts continue development after
      oviposition) between attacking exposed hosts and attacking hosts that are
      concealed within plant tissue. Using a range of phylogeny estimates based on 28S 
      rDNA sequences, we use maximum parsimony (MP) and maximum likelihood (ML) methods
      to estimate the ancestral life history traits for the main clades in which both
      traits occur (using the programs MacClade and Discrete, respectively). We also
      assess the robustness of these estimates; for MP, we use step matrices in PAUP*
      to find the minimum weight necessary to reverse estimates or make them ambiguous,
      and for ML, we measure the differences in likelihood after fixing the ancestral
      nodes at the alternative states. We also measure the robustness of the MP
      ancestral state estimate against uncertainties in the phylogeny estimate,
      manipulating the most-parsimonious tree in MacClade to find the shortest
      suboptimal tree in which the ancestral state estimate is reversed or made
      ambiguous. Using these methods, we find strong evidence supporting two
      transitions among koinobiont Ichneumonoidea: (1) to attacking exposed hosts in a 
      clade consisting of the Helconinae and related subfamilies, and (2) the reverse
      transition in a clade consisting of the Euphorinae and related subfamilies. In
      exploring different methods of analyzing variable-length DNA sequences, we found 
      that direct optimization with POY gave some clearly erroneous results that had a 
      profound effect on the overall phylogeny estimate. We also discuss relationships 
      within the superfamily and expand the Mesostoinae to include all the
      gall-associated braconids that form the sister group of the Aphidiinae.
FAU - Belshaw, Robert
AU  - Belshaw R
AD  - Department of Biological Sciences, Imperial College at Silwood Park, Ascot,
      Berkshire SL5 7PY, UK. r.belshaw@ic.ac.uk
FAU - Quicke, Donald L J
AU  - Quicke DL
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Sequence Homology, Nucleic Acid
MH  - Time Factors
MH  - Wasps/*classification/*genetics
EDAT- 2002/06/25 10:00
MHDA- 2002/10/02 04:00
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/10/02 04:00 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/10635150290069896 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Jun;51(3):450-77. doi: 10.1080/10635150290069896.

PMID- 12079643
OWN - NLM
STAT- MEDLINE
DCOM- 20020930
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 3
DP  - 2002 Jun
TI  - Phylogenetic placement of retropinnid fishes: data set incongruence can be
      reduced by using asymmetric character state transformation costs.
PG  - 432-49
AB  - We used mitochondrial DNA sequences to determine the phylogenetic placement of
      southern smelts (Retropinnidae), a group of diadromous fishes endemic to New
      Zealand and Australia. Our genetic data strongly support a sister group
      relationship between retropinnids and northern hemisphere smelts (Osmeridae), a
      relationship that seems consistent with the similar appearance and life history
      strategies of these two groups. Our analysis indicates that Retropinnidae and
      Osmeridae together represent the sister group to the southern hemisphere galaxiid
      fishes (Galaxiidae). However, this finding conflicts with several recent
      osteological analyses, which supported a sister relationship for Retropinnidae
      and Galaxiidae, giving a monophyletic southern hemisphere assemblage
      (Galaxioidea). We review cases of incongruence and discuss factors that might
      explain significant disagreement between molecular and morphological data
      matrices. We suggest that repeated evolutionary simplification may have
      undermined the accuracy of morphological hypotheses of osmeroid relationships.
      Although equally weighted parsimony analysis of morphological data rejects the
      molecular hypothesis (Osmeridae + Retropinnidae), implementation of a range of
      weighting schemes suggests that incongruence is nonsignificant under asymmetric
      character transformation models. We propose that a simple "equal transformation
      cost" parsimony analysis may be biologically unrealistic, especially when
      reductive homoplasy is widespread; as is increasingly being accepted, complex
      character states are more readily lost than gained. Therefore, we recommend that 
      morphological systematists routinely implement a range of character
      transformation models to assess the sensitivity of their phylogenetic
      reconstructions. We discuss the antitropical biogeography of osmeroid fishes in
      the context of vicariance and transequatorial dispersal.
FAU - Waters, Jonathan M
AU  - Waters JM
AD  - Department of Zoology, University of Otago, Dunedin, New Zealand.
      jonathan.waters@stonebow.otago.ac.nz
FAU - Saruwatari, Toshiro
AU  - Saruwatari T
FAU - Kobayashi, Takanori
AU  - Kobayashi T
FAU - Oohara, Ichiro
AU  - Oohara I
FAU - McDowall, Robert M
AU  - McDowall RM
FAU - Wallis, Graham P
AU  - Wallis GP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA, Mitochondrial/ultrastructure
MH  - Evolution, Molecular
MH  - Models, Statistical
MH  - Molecular Sequence Data
MH  - Nucleic Acid Conformation
MH  - Phylogeny
MH  - Salmoniformes/*classification/*genetics
EDAT- 2002/06/25 10:00
MHDA- 2002/10/02 04:00
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/10/02 04:00 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/10635150290069887 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Jun;51(3):432-49. doi: 10.1080/10635150290069887.

PMID- 12079642
OWN - NLM
STAT- MEDLINE
DCOM- 20020930
LR  - 20071114
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 3
DP  - 2002 Jun
TI  - Phylogenetic relationships in the genus mus, based on paternally, maternally, and
      biparentally inherited characters.
PG  - 410-31
AB  - Several species in the rodent genus Mus are used as model research organisms, but
      comparative studies of these mice have been hampered by the lack of a
      well-supported phylogeny. We used DNA sequences from six genes representing
      paternally, maternally, and biparentally inherited regions of the genome to infer
      phylogenetic relationships among 10 species of Mus commonly used in laboratory
      research. Our sample included seven species from the subgenus Mus; one species
      each from the subgenera Pyromys, Coelomys, and Nannomys; and representatives from
      three additional murine genera, which served as outgroups in the phylogenetic
      analyses. Although each of the six genes yielded a unique phylogeny, several
      clades were supported by four or more gene trees. Nodes that conflicted between
      trees were generally characterized by weak support for one or both of the
      alternative topologies, thus providing no compelling evidence that any individual
      gene, or part of the genome, was misleading with respect to the evolutionary
      history of these mice. Analysis of the combined data resulted in a fully resolved
      tree that strongly supports monophyly of the genus Mus, monophyly of the subgenus
      Mus, division of the subgenus Mus into Palearctic (M. musculus, M. macedonicus,
      M. spicilegus, and M. spretus) and Asian (M. cervicolor, M. cookii, and M.
      caroli) clades, monophyly of the house mice (M. m. musculus, "M. m. molossinus," 
      M. m. castaneus, and M. m. domesticus), and a sister-group relationship between
      M. macedonicus and M. spicilegus. Other clades that were strongly supported by
      one or more gene partitions were not strongly supported by the combined data.
      This appears to reflect a localized homoplasy in one partition obscuring the
      phylogenetic signal from another, rather than differences in gene or genome
      histories.
FAU - Lundrigan, Barbara L
AU  - Lundrigan BL
AD  - Michigan State University Museum and Department of Zoology, Michigan State
      University, East Lansing, Michigan 48824-1045, USA. lundriga@msu.edu
FAU - Jansa, Sharon A
AU  - Jansa SA
FAU - Tucker, Priscilla K
AU  - Tucker PK
LA  - eng
GR  - N01-CB-71085/CB/NCI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Fathers
MH  - Female
MH  - Likelihood Functions
MH  - Male
MH  - Mice
MH  - Models, Statistical
MH  - Mothers
MH  - Muridae/*classification/*genetics
MH  - Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2002/06/25 10:00
MHDA- 2002/10/02 04:00
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/10/02 04:00 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/10635150290069878 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Jun;51(3):410-31. doi: 10.1080/10635150290069878.

PMID- 12079641
OWN - NLM
STAT- MEDLINE
DCOM- 20020930
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 3
DP  - 2002 Jun
TI  - Classification of apocynaceae s.l. according to a new approach combining Linnaean
      and phylogenetic taxonomy.
PG  - 389-409
AB  - A new approach to a nomenclatural system, including elements from both Linnaean
      and phylogenetic nomenclature, is proposed. It is compatible with the existing
      Linnaean system, including "standard names" corresponding to principal and
      secondary ranks, and uses a variant of the definitions from the Phylocode system.
      A new infrafamilial classification, using this nomenclatural approach, of the
      Apocynaceae s.l. (i.e., including the Asclepiadaceae) based mainly on analyses of
      rbcL and ndhF data is discussed. Twenty-one tribes and four rankless taxa are
      defined.
FAU - Sennblad, Bengt
AU  - Sennblad B
AD  - Department of Systematic Botany, Evolutionary Biology Center (EBC), Uppsala
      University, Norbyv. 18D, SE-752 36 Uppsala, Sweden. Bengt.Sennblad@sbc.su.se
FAU - Bremer, Birgitta
AU  - Bremer B
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Apocynaceae/*classification
MH  - Models, Statistical
MH  - Phylogeny
MH  - *Terminology as Topic
EDAT- 2002/06/25 10:00
MHDA- 2002/10/02 04:00
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/10/02 04:00 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/10635150290069869 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Jun;51(3):389-409. doi: 10.1080/10635150290069869.

PMID- 12078648
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - The virtues of gaps: xenarthran (Edentate) monophyly supported by a unique
      deletion in alpha A-crystallin.
PG  - 94-106
AB  - Shared insertions or deletions (indels) in protein-coding DNA can be strong
      indicators of the monophyly of a taxon. A three-amino acid deletion had
      previously been noted in the eye lens protein alpha A-crystallin of two species
      of sloths and two species of anteaters, which represent the Pilosa, one of the
      two infraorders of Xenarthra (Edentata). This deletion has not been observed in
      55 species from 16 other eutherian orders, or in 2 species of marsupials, or in
      34 nonmammalian vertebrates, from birds to shark. At the genomic level, we have
      now detected this deletion in two species of armadillos of the second xenarthran 
      infraorder, Cingulata, as well as in an additional species of anteater.
      Phylogenetic trees were constructed from a 145-bp sequence of the alpha
      A-crystallin gene of 39 tetrapod species, supporting xenarthran monophyly with
      values from 76% to 90%. To quantify the additional support for xenarthran
      monophyly, as given by the three-residue deletion, we computed the probabilities 
      for the occurrence of this deletion per evolutionary time unit for alternative
      hypothetical tree topologies. In the estimates obtained, the six trees in which
      the xenarthran subgroups are unresolved or paraphyletic give an increasingly
      lower likelihood than do the two trees that assume xenarthran monophyly. For the 
      monophyletic trees, the probability that the deletion observed in the xenarthrans
      is due to a single event is &gt; 0.99. Thus, this deletion in alpha A-crystallin
      gives strong molecular support for the monophyly of this old and diverse order.
FAU - Van Dijk, M A
AU  - Van Dijk MA
AD  - Department of Biochemistry, University of Nijmegen, P.O. Box 9101, 6500 HB
      Nijmegen, The Netherlands.
FAU - Paradis, E
AU  - Paradis E
FAU - Catzeflis, F
AU  - Catzeflis F
FAU - de Jong, W W
AU  - de Jong WW
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Crystallins)
SB  - IM
MH  - Animals
MH  - Crystallins/*genetics
MH  - Gene Amplification/genetics
MH  - Phylogeny
MH  - Sequence Deletion
MH  - Xenarthra/classification/*genetics
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260463 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):94-106. doi: 10.1080/106351599260463.

PMID- 12078647
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Mammalian evolution: timing and implications from using the LogDeterminant
      transform for proteins of differing amino acid composition.
PG  - 76-93
AB  - We explore the tree of mammalian mtDNA sequences, using particularly the LogDet
      transform on amino acid sequences, the distance Hadamard transform, and the
      Closest Tree selection criterion. The amino acid composition of different species
      show significant differences, even within mammals. After compensating for these
      differences, nearest-neighbor bootstrap results suggest that the tree is locally 
      stable, though a few groups show slightly greater rearrangements when a large
      proportion of the constant sites are removed. Many parts of the trees we obtain
      agree with those on published protein ML trees. Interesting results include a
      preference for rodent monophyly. The detection of a few alternative signals to
      those on the optimal tree were obtained using the distance Hadamard transform
      (with results expressed as a Lento plot). One rearrangement suggested was the
      interchange of the position of primates and rodents on the optimal tree. The
      basic stability of the tree, combined with two calibration points (whale/cow and 
      horse/rhinoceros), together with a distant secondary calibration from the
      mammal/bird divergence, allows inferences of the times of divergence of putative 
      clades. Allowing for sampling variances due to finite sequence length, most major
      divergences amongst lineages leading to modern orders, appear to occur well
      before the Cretaceous/Tertiary (K/T) boundary. Implications arising from these
      early divergences are discussed, particularly the possibility of competition
      between the small dinosaurs and the new mammal clades.
FAU - Penny, D
AU  - Penny D
AD  - Institute for Molecular BioSciences, Massey University, Palmerston North, New
      Zealand. d.penny@massey.ac.nz
FAU - Hasegawa, M
AU  - Hasegawa M
FAU - Waddell, P J
AU  - Waddell PJ
FAU - Hendy, M D
AU  - Hendy MD
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Amino Acids)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Proteins)
SB  - IM
MH  - Amino Acids/analysis
MH  - Animals
MH  - *Biological Evolution
MH  - DNA, Mitochondrial/*chemistry/genetics
MH  - Humans
MH  - Mammals/*classification/genetics
MH  - Proteins/*chemistry
MH  - Software
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260454 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):76-93. doi: 10.1080/106351599260454.

PMID- 12078646
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Additional support for Afrotheria and Paenungulata, the performance of
      mitochondrial versus nuclear genes, and the impact of data partitions with
      heterogeneous base composition.
PG  - 65-75
AB  - We concatenated sequences for four mitochondrial genes (12S rRNA, tRNA valine,
      16S rRNA, cytochrome b) and four nuclear genes [aquaporin, alpha 2B adrenergic
      receptor (A2AB), interphotoreceptor retinoid-binding protein (IRBP), von
      Willebrand factor (vWF)] into a multigene data set representing 11 eutherian
      orders (Artiodactyla, Hyracoidea, Insectivora, Lagomorpha, Macroscelidea,
      Perissodactyla, Primates, Proboscidea, Rodentia, Sirenia, Tubulidentata). Within 
      this data set, we recognized nine mitochondrial partitions (both stems and loops,
      for each of 12S rRNA, tRNA valine, and 16S rRNA; and first, second, and third
      codon positions of cytochrome b) and 12 nuclear partitions (first, second, and
      third codon positions, respectively, of each of the four nuclear genes). Four of 
      the 21 partitions (third positions of cytochrome b, A2AB, IRBP, and vWF) showed
      significant heterogeneity in base composition across taxa. Phylogenetic analyses 
      (parsimony, minimum evolution, maximum likelihood) based on sequences for all 21 
      partitions provide 99-100% bootstrap support for Afrotheria and Paenungulata.
      With the elimination of the four partitions exhibiting heterogeneity in base
      composition, there is also high bootstrap support (89-100%) for cow + horse.
      Statistical tests reject Altungulata, Anagalida, and Ungulata. Data set
      heterogeneity between mitochondrial and nuclear genes is most evident when all
      partitions are included in the phylogenetic analyses. Mitochondrial-gene trees
      associate cow with horse, whereas nuclear-gene trees associate cow with hedgehog 
      and these two with horse. However, after eliminating third positions of A2AB,
      IRBP, and vWF, nuclear data agree with mitochondrial data in supporting cow +
      horse. Nuclear genes provide stronger support for both Afrotheria and
      Paenungulata. Removal of third positions of cytochrome b results in improved
      performance for the mitochondrial genes in recovering these clades.
FAU - Springer, M S
AU  - Springer MS
AD  - Department of Biology and Interdepartmental Graduate Program in Genetics,
      University of California, Riverside, California 92521, USA. mark.springer@ucr.edu
FAU - Amrine, H M
AU  - Amrine HM
FAU - Burk, A
AU  - Burk A
FAU - Stanhope, M J
AU  - Stanhope MJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (RNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 16S)
RN  - 0 (RNA, Transfer, Val)
RN  - 0 (RNA, ribosomal, 12S)
SB  - IM
MH  - Animals
MH  - Base Composition
MH  - Cell Nucleus/*genetics
MH  - Cytochrome b Group/genetics
MH  - DNA, Mitochondrial/*genetics
MH  - Female
MH  - *Genes
MH  - Mammals/*classification/*genetics
MH  - Phylogeny
MH  - Placenta
MH  - Pregnancy
MH  - RNA, Ribosomal/genetics
MH  - RNA, Ribosomal, 16S/genetics
MH  - RNA, Transfer, Val/genetics
MH  - Rabbits/classification
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260445 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):65-75. doi: 10.1080/106351599260445.

PMID- 12078645
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Stability of cladistic relationships between Cetacea and higher-level artiodactyl
      taxa.
PG  - 6-20
AB  - Over the past 10 years, the phylogenetic relationships among higher-level
      artiodactyl taxa have been examined with multiple data sets. Many of these data
      sets suggest that Artiodactyla (even-toed ungulates) is paraphyletic and that
      Cetacea (whales) represents a highly derived "artiodactyl" subgroup. In this
      report, phylogenetic relationships between Cetacea and artiodactyls are tested
      with a combination of 15 published data sets plus new DNA sequence data from two 
      nuclear loci, interphotoreceptor retinoid-binding protein (IRBP) and von
      Willebrand factor (vWF). The addition of the IRBP and vWF character sets disrupts
      none of the relationships supported by recent cladistic analyses of the other 15 
      data sets. Simultaneous analyses support three critical clades: (Cetacea +
      Hippopotamidae), (Cetacea + Hippopotamidae + Ruminantia), and (Cetacea +
      Hippopotamidae + Ruminantia + Suina). Perturbations of the combined matrix show
      that the above clades are stable to a variety of disturbances. A chronicle of
      phylogenetic results over the past 3 years suggests that cladistic relationships 
      between Cetacea and artiodactyls have been stable to increased taxonomic sampling
      and to the addition of more than 1,400 informative characters from 15 data sets.
FAU - Gatesy, J
AU  - Gatesy J
AD  - Laboratory of Molecular Systematics and Evolution, Department of Ecology and
      Evolutionary Biology, University of Arizona, Biosciences West, Tucson, Arizona
      85721, USA.
FAU - Milinkovitch, M
AU  - Milinkovitch M
FAU - Waddell, V
AU  - Waddell V
FAU - Stanhope, M
AU  - Stanhope M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Retinol-Binding Proteins)
SB  - IM
MH  - Animals
MH  - Artiodactyla/*classification/genetics
MH  - Cetacea/*classification/genetics
MH  - Photoreceptor Cells, Vertebrate/physiology
MH  - Phylogeny
MH  - Retinol-Binding Proteins/genetics
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260409 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):6-20. doi: 10.1080/106351599260409.

PMID- 12078644
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Phylogenetic assessment of molecular and morphological data for eutherian
      mammals.
PG  - 54-64
AB  - The interordinal relationships of eutherian (placental) mammals were evaluated by
      a phylogenetic analysis of four published data sets (three sequences and one
      morphological). The nature and degree of support and conflict for particular
      groups were assessed by separate bootstrap and homogeneity tests, which were
      followed by combined analyses of the sequence and morphological data. Between
      orders, strong support (i.e., &gt; or = 95% bootstrap scores) was found for a
      paraphyletic Artiodactyla (relative to Cetacea) and a monophyletic
      Cetartiodactyla (Artiodactyla and Cetacea) and Paenungulata (Hyracoidea,
      Proboscidea, and Sirenia). In turn, some reasonable to strong evidence (&gt; or =
      85%) was obtained for Hyracoidea with Sirenia, Dermoptera with Scandentia, Glires
      (Lagomorpha with Rodentia), and Afrotheria (Amblysomus, Macroscelidea,
      Paenungulata, and Tubulidentata). Otherwise, no other interordinal clades were
      supported at these reasonable to strong levels. This overall lack of resolution
      for eutherian interordinal clusters agrees with other studies that suggest
      further progress will continue to be slow and difficult. Further resolution will 
      require the integration of more recently published data, the continued sampling
      of taxa and characters, and the use of more powerful methods of data analysis.
FAU - Liu, F G
AU  - Liu FG
AD  - Department of Zoology, University of Florida, Gainesville, Florida 32611-8525,
      USA.
FAU - Miyamoto, M M
AU  - Miyamoto MM
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal)
RN  - 0 (RNA, ribosomal, 12S)
SB  - IM
MH  - Animals
MH  - Artiodactyla/anatomy &amp; histology/classification/genetics
MH  - Base Sequence
MH  - Cetacea/anatomy &amp; histology/classification/genetics
MH  - Mammals/anatomy &amp; histology/*classification/*genetics
MH  - RNA, Ribosomal/genetics
MH  - Reproducibility of Results
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260436 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):54-64. doi: 10.1080/106351599260436.

PMID- 12078643
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Using novel phylogenetic methods to evaluate mammalian mtDNA, including amino
      acid-invariant sites-LogDet plus site stripping, to detect internal conflicts in 
      the data, with special reference to the positions of hedgehog, armadillo, and
      elephant.
PG  - 31-53
AB  - We look at the higher-order phylogeny of mammals, analyzing in detail the
      complete mtDNA sequences of more than 40 species. We test the support for several
      proposed superordinal relationships. To this end, we apply a number of recently
      programmed methods and approaches, plus better-established methods. New pairwise 
      tests show highly significant evidence that amino acid frequencies are changing
      among nearly all the genomes studied when unvaried sites are ignored. LogDet
      amino acid distances, with modifications to take into account invariant sites,
      are combined with bootstrapping and the Neighbor Joining algorithm to account for
      these violations of standard models. To weight the more slowly evolving sites, we
      exclude the more rapidly evolving sites from the data by using "site stripping". 
      This leads to changing optimal trees with nearly all methods. The bootstrap
      support for many hypotheses varies widely between methods, and few hypotheses can
      claim unanimous support from these data. Rather, we uncover good evidence that
      many of the earlier branching patterns in the placental subtree could be
      incorrect, including the placement of the root. The tRNA genes, for example,
      favor a split between the group hedgehog, rodents, and primates versus all other 
      sequenced placentals. Such a grouping is not ruled out by the amino acid sequence
      data. A grouping of all rodents plus rabbit, the old Glires hypothesis, is also
      feasible with stripped amino acid data, and rodent monophyly is also common. The 
      elephant sequence allows confident rejection of the older taxon Ferungulata
      (Simpson, 1945). In its place, the new taxa Scrotifera and Fereuungulata are
      defined. A new likelihood ratio test is used to detect differences between the
      optimal tree for tRNA versus that for amino acids. While not clearly significant 
      as made, some results indicate the test is tending towards significance with more
      general models of evolution. Individual placement tests suggest alternative
      positions for hedgehog and elephant. Congruence arguments to support elephant and
      armadillo together are striking, suggesting a superordinal group composed of
      Xenarthra and African endemic mammals, which in turn may be near the root of the 
      placental subtree. Thus, while casting doubt on some recent conclusions, the
      analyses are also unveiling some interesting new possibilities.
FAU - Waddell, P J
AU  - Waddell PJ
AD  - Institute of Statistical Mathematics, 4-6-7 Minami-Azabu, Minato-ku, Tokyo
      106-8569, Japan. waddell@ism.ac.jp
FAU - Cao, Y
AU  - Cao Y
FAU - Hauf, J
AU  - Hauf J
FAU - Hasegawa, M
AU  - Hasegawa M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Armadillos/*genetics
MH  - DNA, Mitochondrial/*genetics
MH  - Elephants/*genetics
MH  - Hedgehogs/*genetics
MH  - Humans
MH  - Mammals/classification/*genetics
MH  - Mice
MH  - *Phylogeny
MH  - Primates/classification/genetics
MH  - Reproducibility of Results
MH  - Vertebrates/classification/genetics
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260427 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):31-53. doi: 10.1080/106351599260427.

PMID- 12078642
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Ankle morphology of the earliest Cetaceans and its implications for the
      phylogenetic relations among ungulates.
PG  - 21-30
AB  - Recent molecular studies are inconsistent with ungulate phylogenetic trees that
      are based on morphological traits. These inconsistencies especially relate to the
      position of cetaceans and perissodactyls. Evaluation of the close phylogenetic
      ties between artiodactyls and cetaceans has been hampered by the absence of
      tarsal bones of primitive cetaceans, as artiodactyls are often diagnosed on the
      basis of their tarsus. We here describe newly discovered tarsal bones that are
      the oldest cetacean tarsals known. We present a character analysis for primitive 
      ungulate tarsals and evaluate their impact on the ungulate phylogenetic tree.
      Tarsal data are consistent with some molecular studies in suggesting that the
      extant sister group of Cetacea is Artiodactyla or that Cetacea should be included
      within the latter order. Tarsal data do not support Cete (Mesonychia plus
      Cetacea) and are consistent with the exclusion of perissodactyls from
      paenungulates as suggested by some molecular studies.
FAU - Thewissen, J G
AU  - Thewissen JG
AD  - Department of Anatomy, Northeastern Ohio Universities College of Medicine,
      Rootstown, Ohio 44240, USA. thewisse@neoucom.edu
FAU - Madar, S I
AU  - Madar SI
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Cetacea/*anatomy &amp; histology/classification
MH  - Hindlimb/anatomy &amp; histology
MH  - Joints/*anatomy &amp; histology
MH  - *Phylogeny
MH  - Ruminants/*anatomy &amp; histology/classification
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260418 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):21-30. doi: 10.1080/106351599260418.

PMID- 12078641
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Delimitation of phylogenetic species with DNA sequences: a critique of Davis and 
      Nixon's population aggregation analysis.
PG  - 199-213
FAU - Brower, A V
AU  - Brower AV
AD  - Department of Entomology, Oregon State University, Corvallis, OR 97331-2907, USA.
      browera@bcc.orst.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Base Sequence
MH  - DNA/*genetics
MH  - Genetics, Population
MH  - *Models, Genetic
MH  - *Models, Statistical
MH  - Molecular Sequence Data
MH  - *Phylogeny
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260535 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):199-213. doi: 10.1080/106351599260535.

PMID- 12078640
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Multivariate regression models and geometric morphometrics: the search for causal
      factors in the analysis of shape.
PG  - 192-9
FAU - Monteiro, L R
AU  - Monteiro LR
AD  - Departamento de Zoologia, caixa postal 199, UNESP, Rio Claro, Sao Paulo, cep
      13506-900, Brazil. monteiro@life.ibrc.unesp.br
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Age Factors
MH  - Animals
MH  - Body Constitution
MH  - Bone and Bones/anatomy &amp; histology
MH  - *Models, Statistical
MH  - Multivariate Analysis
MH  - Rats/anatomy &amp; histology
MH  - Regression Analysis
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260526 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):192-9. doi: 10.1080/106351599260526.

PMID- 12078639
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Power of the concentrated changes test for correlated evolution.
PG  - 170-91
AB  - The concentrated changes test (CCT) calculates the probability that changes in a 
      binary character are distributed randomly on the branches of a cladogram. This
      test is used to examine hypotheses of correlated evolution, especially cases
      where changes in the state of one character influence changes in the state of
      another character. The test may be sensitive to the addition of branches that
      lack either trait of interest (white branches). To examine the effects of the
      proportion of white branches and of tree topology on the CCT probability, we
      conducted a simulation analysis using a series of randomly generated 100-taxon
      trees, in addition to a nearly perfectly balanced (symmetrical) and a completely 
      imbalanced (asymmetrical) 100-taxon tree. Using two models of evolution (gains
      only, or gains and losses), we evolved character pairs randomly onto these trees 
      to simulate cases where (1) characters evolve independently (i.e., no correlation
      among the traits) or (2) all changes in the dependent character occur on branches
      containing the independent trait (i.e., a strong correlation among the traits).
      This allowed us to evaluate the sensitivity of the CCT to type I and type II
      errors, respectively. In the simulations, the CCT did not appear to be overly
      sensitive to the inclusion of white branches (low likelihood of type I error with
      both CCT probabilities &lt; 0.05 and &lt; 0.01). However, the CCT was susceptible to
      type II error when the proportion of white branches was &lt; 20%. The test was also 
      sensitive to tree shape and was positively correlated to Colless's tree imbalance
      statistic I. Finally, the CCT responded differently for simulations where only
      gains were allowed and those where both gains and losses were permitted. These
      results indicate that the CCT is unlikely to detect a correlation between
      characters when no such correlation exists. However, when a trait can be gained
      but not lost, the CCT is conservative and may fail to detect true correlations
      among traits (increased type II error). Determination of the sampling universe
      (the taxa included in the comparative analysis) can strongly influence the
      probability of making such type II errors. We suggest guidelines to circumvent
      these limitations.
FAU - Lorch, P D
AU  - Lorch PD
AD  - Department of Biology, University of Toronto at Mississauga, Mississauga, Ontario
      L5L 1C6, Canada. plorch@credit.erin.utoronto.ca
FAU - Eadie, J M
AU  - Eadie JM
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Computer Simulation
MH  - Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Probability
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260517 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):170-91. doi: 10.1080/106351599260517.

PMID- 12078638
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Misinformative characters and phylogeny shape.
PG  - 153-69
AB  - The discrepancy between theoretical and observed distributions of tree shapes in 
      recent surveys of phylogeny estimates has lead to investigations of possible
      biological and methodological causes. I investigated whether the phylogenetic
      quality of characters is related to the tree shape on which they evolve.
      Simulated evolution revealed shape-related tendencies for characters to indicate 
      correct cladistic relationships; these differences were measured by examining the
      characters directly, without deriving any phylogeny estimates. Tree stemminess
      indices correlated strongly with character quality when characters evolved either
      speciationally or phyletically. Tree balance was a significant correlate of
      character quality under speciational evolution but not under phyletic evolution. 
      These results help explain the findings of other simulation studies. With
      additional study of the behavior of evolving characters and their interaction
      with phylogenetic methods, we might be able to increase the accuracy of tree
      estimation and compensate for potential biases related to shape. These results
      give further reason for caution in trusting phylogeny estimates.
FAU - Salisbury, B A
AU  - Salisbury BA
AD  - Department of Biology, University of Michigan, Ann Arbor, Michigan 48109-1048,
      USA. ben@aya.yale.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Computer Simulation
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Trees/anatomy &amp; histology/classification/genetics
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260508 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):153-69. doi: 10.1080/106351599260508.

PMID- 12078637
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Interordinal relationships of birds and other reptiles based on whole
      mitochondrial genomes.
PG  - 138-52
AB  - Several different groups of birds have been proposed as being the oldest or
      earliest diverging extant lineage within the avian phylogenetic tree,
      particularly ratites (Struthioniformes), waterfowl (Anseriformes), and shorebirds
      (Charadriiformes). Difficulty in resolving this issue stems from several factors,
      including the relatively rapid radiation of primary (ordinal) bird lineages and
      the lack of characters from an extant outgroup for birds that is closely related 
      to them by measure of time. To help resolve this question, we have sequenced
      entire mitochondrial genomes for five birds (a rhea, a duck, a falcon, and two
      perching birds), one crocodilian, and one turtle. Maximum parsimony and maximum
      likelihood analyses of these new sequences together with published sequences (18 
      taxa total) yield the same optimal tree topology, in which a perching bird
      (Passeriformes) is sister to all the other bird taxa. A basal position for
      waterfowl among the bird study taxa is rejected by maximum likelihood analyses.
      However, neither the conventional view, in which ratites (including rhea) are
      basal to other birds, nor tree topologies with falcon or chicken basal among
      birds could be rejected in the same manner. In likelihood analyses of a subset of
      seven birds, alligator, and turtle (9 taxa total), we find that increasing the
      number of parameters in the model shifts the optimal topology from one with a
      perching bird basal among birds to the conventional view with ratites diverging
      basally; moreover, likelihood scores for the two trees are not significantly
      different. Thus, although our largest set of taxa and characters supports a tree 
      with perching birds diverging basally among birds, the position of this earliest 
      divergence among birds appears unstable. Our analyses indicate a sister
      relationship between a waterfowl/chicken clade and ratites, relative to perching 
      birds and falcon. We find support for a sister relationship between turtles and a
      bird/crocodilian clade, and for rejecting both the Haemothermia hypothesis (birds
      and mammals as sister taxa) and the placement of turtles as basal within the
      phylogenetic tree for amniote animals.
FAU - Mindell, D P
AU  - Mindell DP
AD  - Department of Biology, Museum of Zoology, University of Michigan, Ann Arbor,
      Michigan 48109, USA. mindell@umich.edu
FAU - Sorenson, M D
AU  - Sorenson MD
FAU - Dimcheff, D E
AU  - Dimcheff DE
FAU - Hasegawa, M
AU  - Hasegawa M
FAU - Ast, J C
AU  - Ast JC
FAU - Yuri, T
AU  - Yuri T
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Birds/classification/*genetics
MH  - DNA, Mitochondrial/*genetics
MH  - *Genetic Variation
MH  - Genome
MH  - Phylogeny
MH  - Reptiles/classification/*genetics
MH  - Songbirds/classification/genetics
MH  - Turtles/classification/genetics
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260490 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):138-52. doi: 10.1080/106351599260490.

PMID- 12078636
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Assessing the Cretaceous superordinal divergence times within birds and placental
      mammals by using whole mitochondrial protein sequences and an extended
      statistical framework.
PG  - 119-37
AB  - Using the set of all vertebrate mtDNA protein sequences published as of May 1998,
      plus unpublished examples for elephant and birds, we examined divergence times in
      Placentalia and Aves. Using a parsimony-based test, we identified a subset of
      slower evolutionary rate placental sequences that do not appear to violate the
      clock assumption. Analyzing just these sequences decreases support for
      Marsupionta and the carnivore + perissodactyl group but increases support for
      armadillo diverging earlier than rabbit (which may represent the whole Glires
      group). A major theme of the paper is to use more comprehensive estimates of
      divergence time standard error (SE). From the well-studied horse/rhino split,
      estimated to be 55 million years before present (mybp), the splitting time within
      carnivores is confidently shown to be older than 50 million years. Some of our
      estimates of divergence times within placentals are relatively old, at up to 169 
      million years, but are within 2 SE of other published estimates. The whale/cow
      split at 65 mybp may be older than commonly assumed. All the sampled splits
      between the main groups of fereuungulates (the clade of carnivores,
      cetartiodactyls, perissodactyls, and pholidotes) seem to be distinctly before the
      Cretaceous/Tertiary boundary. Analyses suggest a close relationship between
      elephants (representing Afrotheria) and armadillos (Xenarthra), and our timing of
      this splitting is coincident with the opening of the South Atlantic, a major
      vicariant event. Recalibrating with this event (at 100 mybp), we obtain younger
      estimates for the earliest splits among placentals. Divergence times within birds
      are also assessed by using previously unpublished sequences. We fail to reject a 
      clock for all bird taxa available. Unfortunately, available deep calibration
      points for birds are questionable, so a new calibration based on the age of the
      Anseriform stem lineage is estimated. The divergence time of rhea and ostrich may
      be much more recent than commonly assumed, while that of passerines may be older.
      Our major concern is the rooting point of the bird subtree, as the nearest
      outgroup (alligator) is very distant.
FAU - Waddell, P J
AU  - Waddell PJ
AD  - Institute of Statistical Mathematics, 4-6-7 Minami-Azabu, Minato-ku, Tokyo
      106-8569, Japan. waddell@ism.ac.jp
FAU - Cao, Y
AU  - Cao Y
FAU - Hasegawa, M
AU  - Hasegawa M
FAU - Mindell, D P
AU  - Mindell DP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Proteins)
SB  - IM
EIN - Syst Biol 1999 Sep;48(3):681
MH  - Animals
MH  - Birds/*genetics
MH  - DNA, Mitochondrial/chemistry/*genetics
MH  - Female
MH  - *Genetic Variation
MH  - Mammals/*genetics
MH  - Models, Statistical
MH  - Phylogeny
MH  - Placenta
MH  - Pregnancy
MH  - Proteins/chemistry/*genetics
MH  - Reproducibility of Results
MH  - Time
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260481 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):119-37. doi: 10.1080/106351599260481.

PMID- 12078635
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - The fossil record of North American mammals: evidence for a Paleocene
      evolutionary radiation.
PG  - 107-18
AB  - Paleontologists long have argued that the most important evolutionary radiation
      of mammals occurred during the early Cenozoic, if not that all eutherians
      originated from a single common post-Cretaceous ancestor. Nonetheless, several
      recent molecular analyses claim to show that because several interordinal splits 
      occurred during the Cretaceous, a major therian radiation was then underway. This
      claim conflicts with statistical evidence from the well-sampled latest Cretaceous
      and Cenozoic North American fossil record. Paleofaunal data confirm that there
      were fewer mammalian species during the latest Cretaceous than during any
      interval of the Cenozoic, and that a massive diversification took place during
      the early Paleocene, immediately after a mass extinction. Measurement data show
      that Cretaceous mammals were on average small and occupied a narrow range of body
      sizes; after the Cretaceous-Tertiary mass extinction, there was a rapid and
      permanent shift in the mean. The fact that there was an early Cenozoic mammalian 
      radiation is entirely compatible with the existence of a few Cretaceous splits
      among modern mammal lineages.
FAU - Alroy, J
AU  - Alroy J
AD  - Department of Paleobiology, Smithsonian Institution, MRC 121, Washington, D.C.
      20560, USA. alroy@nceas.ucsb.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - *Fossils
MH  - Genetic Variation
MH  - Mammals/*classification/*genetics
MH  - North America
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/106351599260472 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):107-18. doi: 10.1080/106351599260472.

PMID- 12078634
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 1
DP  - 1999 Mar
TI  - Towards resolving the interordinal relationships of placental mammals.
PG  - 1-5
FAU - Waddell, P J
AU  - Waddell PJ
AD  - Institute of Statistical Mathematics, 4-6-7 Minami-Azabu, Minato-Ku, Tokyo
      106-8569, Japan. waddell@ism.ac.jp
FAU - Okada, N
AU  - Okada N
FAU - Hasegawa, M
AU  - Hasegawa M
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Chiroptera
MH  - Female
MH  - Mammals/*classification
MH  - Placenta/*anatomy &amp; histology
MH  - Pregnancy
EDAT- 2002/06/25 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/25 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 1999 Mar;48(1):1-5.

PMID- 12066717
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 2
DP  - 1999 Jun
TI  - Reconciling kingdoms with codes of nomenclature: is it necessary?
PG  - 406-12
FAU - Blackwell, W H
AU  - Blackwell WH
AD  - Department of Biological Sciences, University of Alabama, Tuscaloosa, Alabama
      35487, USA.
FAU - Powell, M J
AU  - Powell MJ
LA  - eng
PT  - Journal Article
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - *Classification
MH  - Phylogeny
MH  - Terminology as Topic
RF  - 67
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260382 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Jun;48(2):406-12. doi: 10.1080/106351599260382.

PMID- 12066716
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 2
DP  - 1999 Jun
TI  - Apomorphy distribution is an important aspect of cladogram symmetry.
PG  - 399-406
FAU - Pearson, P N
AU  - Pearson PN
AD  - Department of Earth Sciences, University of Bristol, Queens Road, Bristol BS8
      1RJ, United Kingdom. paul.pearson@bristol.ac.uk
LA  - eng
PT  - Journal Article
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Body Constitution
MH  - *Classification
MH  - *Models, Biological
MH  - *Phylogeny
RF  - 45
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260373 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Jun;48(2):399-406. doi: 10.1080/106351599260373.

PMID- 12066715
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 2
DP  - 1999 Jun
TI  - Is the rate of molecular evolution inversely related to body size?
PG  - 396-9
FAU - Slowinski, J B
AU  - Slowinski JB
AD  - Department of Herpetology, California Academy of Sciences, Golden Gate Park, San 
      Francisco, California 94118-4599, USA. jslowins@cas.calacademy.org
FAU - Arbogast, B S
AU  - Arbogast BS
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Body Constitution/*genetics
MH  - *Evolution, Molecular
MH  - Markov Chains
MH  - Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260364 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Jun;48(2):396-9. doi: 10.1080/106351599260364.

PMID- 12066714
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 2
DP  - 1999 Jun
TI  - Discriminating and locating character covariance: an application of permutation
      tail probability (PTP) analyses.
PG  - 380-95
AB  - The ability of permutation tail probability (PTP) analyses to discriminate
      between character covariance and noise is investigated with both hypothetical and
      published data sets. PTP is shown to be a powerful tool, not only for detecting
      character covariance, but also for locating that covariance on trees. PTP is
      especially useful for evaluating DNA sequence data that may have a high level of 
      homoplasy. A three-step PTP procedure for locating covaried characters is
      presented.
FAU - Fu, J
AU  - Fu J
AD  - Centre for Biodiversity and Conservation Biology, Royal Ontario Museum,
      Department of Zoology, University of Toronto, Toronto, Ontario, Canada M5S 2C6.
      jinzhong@rom.on.ca
FAU - Murphy, R W
AU  - Murphy RW
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Crotalus/classification
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Probability
MH  - Salamandridae/classification
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260355 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Jun;48(2):380-95. doi: 10.1080/106351599260355.

PMID- 12066713
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 2
DP  - 1999 Jun
TI  - Coding polymorphism for phylogeny reconstruction.
PG  - 365-79
AB  - The methodology of coding polymorphic taxa has received limited attention to
      date. A search of the taxonomic literature revealed seven types of coding
      methods. Apart from ignoring polymorphic characters (sometimes called the
      fixed-only method), two main categories can be distinguished: methods that
      identify the start of a new character state with the origin of an evolutionary
      novelty, and methods that identify the new state with the fixation of a novelty. 
      The methods of the first category introduce soft reversals, yielding signals that
      support cladograms incompatible with true phylogenies. We conclude that coding
      the plesiomorphy is the method to be preferred, unless the ancestral state is
      unknown, in which case coding as ambiguous is recommended. This holds for coding 
      polymorphism in species as well as in supraspecific taxa. In this light we remark
      on methods proposed by previous authors.
FAU - Kornet, D J
AU  - Kornet DJ
AD  - Institute of Evolutionary and Ecological Sciences, Leiden University, P.O. Box
      9516, 2300 RA Leiden, The Netherlands. kornet@rulsfb.leidenuniv.nl
FAU - Turner, H
AU  - Turner H
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Enzymes)
RN  - 0 (Isoenzymes)
SB  - IM
MH  - Animals
MH  - Enzymes/genetics
MH  - Genetic Code
MH  - Isoenzymes/genetics
MH  - Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - *Polymorphism, Genetic
MH  - Reproducibility of Results
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260346 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Jun;48(2):365-79. doi: 10.1080/106351599260346.

PMID- 12066712
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 2
DP  - 1999 Jun
TI  - Charles Darwin's views of classification in theory and practice.
PG  - 352-64
AB  - It has long been argued that Charles Darwin was the founder of the school of
      "evolutionary taxonomy" of the Modern Synthesis and, accordingly, that he
      recognized genealogy and similarity as dual, synergistic criteria for
      classification. This view is based on three questionable interpretations: first, 
      of isolated passages in the 13th chapter of the Origin of Species; second, of one
      phrase in a letter that Darwin wrote about the work of an author he had partly
      misunderstood; and third, of his taxonomic practice in the barnacle monographs,
      which only implicitly embody his philosophy of classification, if at all. These
      works, seen in fuller context and with the perspective of extensive
      correspondence, are consistent with the view that Darwin advocated only genealogy
      as the basis of classification, and that similarity was merely a tool for
      discovering evolutionary relationships. Darwin was neither a Mayrian taxonomist
      nor a cladist, and he did not approach systematic issues in the same terms that
      we do in the late 20th century.
FAU - Padian, K
AU  - Padian K
AD  - Department of Integrative Biology, Museum of Paleontology, 1101 VLSB, University 
      of California, Berkeley, California 94720, USA. kpadian@socrates.berkeley.edu
LA  - eng
PT  - Biography
PT  - Historical Article
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - *Genealogy and Heraldry
MH  - History, 19th Century
MH  - History, 20th Century
MH  - Models, Biological
PS  - Darwin C
FPS - Darwin, C
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260337 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Jun;48(2):352-64. doi: 10.1080/106351599260337.

PMID- 12066711
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 2
DP  - 1999 Jun
TI  - Definitions in phylogenetic taxonomy: critique and rationale.
PG  - 329-51
AB  - A general rationale for the formulation and placement of taxonomic definitions in
      phylogenetic taxonomy is proposed, and commonly used terms such as "crown taxon" 
      or "node-based definition" are more precisely defined. In the formulation of
      phylogenetic definitions, nested reference taxa stabilize taxonomic content. A
      definitional configuration termed a node-stem triplet also stabilizes the
      relationship between the trio of taxa at a branchpoint, in the face of local
      change in phylogenetic relationships or addition/deletion of taxa. Crown-total
      taxonomies use survivorship as a criterion for placement of node-stem triplets
      within a taxonomic hierarchy. Diversity, morphology, and tradition also
      constitute heuristic criteria for placement of node-stem triplets.
FAU - Sereno, P C
AU  - Sereno PC
AD  - Department of Organismal Biology and Anatomy, University of Chicago, 1027 E. 57th
      Street, Chicago, Illinois 60637, USA. dinosaur@uchicago.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Birds/classification
MH  - Classification/*methods
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Terminology as Topic
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260328 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Jun;48(2):329-51. doi: 10.1080/106351599260328.

PMID- 12066710
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 2
DP  - 1999 Jun
TI  - Effect of nonindependent substitution on phylogenetic accuracy.
PG  - 317-28
AB  - All current phylogenetic methods assume that DNA substitutions are independent
      among sites. However, ample empirical evidence suggests that the process of
      substitution is not independent but is, in fact, temporally and spatially
      correlated. The robustness of several commonly used phylogenetic methods to the
      assumption of independent substitution is examined. A compound Poisson process is
      used to model DNA substitution. This model assumes that substitution events are
      Poisson-distributed in time and that the number of substitutions associated with 
      each event is geometrically distributed. The asymptotic properties of
      phylogenetic methods do not appear to change under a compound Poisson process of 
      DNA substitution. Moreover, the rank order of the performance of different
      methods does not change. However, all phylogenetic methods become less efficient 
      when substitution follows a compound Poisson process.
FAU - Huelsenbeck, J P
AU  - Huelsenbeck JP
AD  - Department of Biology, University of Rochester, Rochester, New York 14627, USA.
      johnh@brahms.biology.rochester.edu
FAU - Nielsen, R
AU  - Nielsen R
LA  - eng
GR  - GM40282/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Base Composition
MH  - Base Sequence
MH  - DNA/*chemistry
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260319 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Jun;48(2):317-28. doi: 10.1080/106351599260319.

PMID- 12066709
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 2
DP  - 1999 Jun
TI  - Type I error and the power of the s-test: old lessons from a new, analytically
      justified statistical test for phylogenies.
PG  - 300-16
AB  - We present a new procedure for assessing the statistical significance of the most
      likely unrooted dichotomous topology inferrable from four DNA sequences. The
      procedure calculates directly a P-value for the support given to this topology by
      the informative sites congruent with it, assuming the most likely star topology
      as the null hypothesis. Informative sites are crucial in the determination of the
      maximum likelihood dichotomous topology and are therefore an obvious target for a
      statistical test of phylogenies. Our P-value is the probability of producing
      through parallel substitutions on the branches of the star topology at least as
      much support as that given to the maximum likelihood dichotomous topology by the 
      aforementioned informative sites, for any of the three possible dichotomous
      topologies. The degree of statistical significance is simply the complement of
      this P-value. Ours is therefore an a posteriori testing approach, in which no
      dichotomous topology is specified in advance. We implement the test for the case 
      in which all sites behave identically and the substitution model has a single
      parameter. Under these conditions, the P-value can be easily calculated on the
      basis of the probabilities of change on the branches of the most likely star
      topology, because under these assumptions, each site can become informative
      independently from every other site; accordingly, the total number of informative
      sites of each kind is binomially distributed. We explore the test's type I error 
      by applying it to data produced in star topologies having all branches equally
      long, or having two short and two long branches, and various degrees of
      homoplasy. The test is conservative but we demonstrate, by means of a
      discreteness correction and progressively assumption-free calculations of the
      P-values, that (1) the conservativeness is mostly due to the discrete nature of
      informative sites and (2) the P-values calculated empirically are moreover mostly
      quite accurate in absolute terms. Applying the test to data produced in
      dichotomous topologies with increasing internal branch length shows that, despite
      the test's "conservativeness," its power is much higher than that of the
      bootstrap, especially when the relevant informative sites are few.
FAU - Antezana, M A
AU  - Antezana MA
AD  - Department of Ecology and Evolutionary Biology, University of California, Irvine,
      California 92717, USA. marcos@uchicago.edu
FAU - Hudson, R R
AU  - Hudson RR
LA  - eng
GR  - GM39355/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Likelihood Functions
MH  - *Models, Statistical
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260300 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Jun;48(2):300-16. doi: 10.1080/106351599260300.

PMID- 12066708
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 2
DP  - 1999 Jun
TI  - Maximum likelihood analysis of gene-based and structure-based process partitions,
      using mammalian mitochondrial genomes.
PG  - 286-99
AB  - Aligned protein-coding genes from 19 completely sequenced mammalian mitochondrial
      genomes were examined by parsimony and maximum likelihood analyses. Particular
      attention is given to a comparison between gene-based and structure-based data
      partitions. Because actual structures are not known for most of the
      mitochondrially encoded proteins, three different surrogate partitioning schemes 
      were examined, each based on the identity of the consensus amino acid at a
      specific homologous position. One of the amino-acid-based partitioning schemes
      gave the highest likelihood, but that scheme was based on concordance with a
      well-corroborated phylogeny from an earlier parsimony analysis. The gene-based
      partitioning scheme gave a significantly higher likelihood compared to the only
      structure-based scheme examined that could be generated without prior assumptions
      about the phylogeny. Two contrasting phylogenetic inferences were supported by
      the analyses. Both unpartitioned analyses and analyses in which all partitions
      were constrained to have identical patterns of branch lengths supported
      ((Artiodactyla, Cetacea) (Perissodactyla, Carnivora)), whereas all analyses with 
      that constraint relaxed supported (((Artiodactyla, Cetacea) Carnivora)
      Perissodactyla).
FAU - DeBry, R W
AU  - DeBry RW
AD  - Department of Biological Sciences, Box 210006, University of Cincinnati,
      Cincinnati, Ohio 45221-0006, USA. ron.debry@uc.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Amino Acid Sequence
MH  - Animals
MH  - Biological Evolution
MH  - Codon/genetics
MH  - DNA, Mitochondrial/*genetics
MH  - *Genome
MH  - Humans
MH  - Likelihood Functions
MH  - Mammals/classification/*genetics
MH  - Mitochondria/*genetics
MH  - Phylogeny
MH  - Primates/classification/genetics
MH  - Vertebrates/genetics
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260292 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Jun;48(2):286-99. doi: 10.1080/106351599260292.

PMID- 12066707
OWN - NLM
STAT- MEDLINE
DCOM- 20020725
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 2
DP  - 1999 Jun
TI  - Combined molecular phylogenetic analysis of the Orthoptera (Arthropoda, Insecta) 
      and implications for their higher systematics.
PG  - 233-53
AB  - A phylogenetic analysis of mitochondrial and nuclear rDNA sequences from species 
      of all the superfamilies of the insect order Orthoptera (grasshoppers, crickets, 
      and relatives) confirmed that although mitochondrial sequences provided good
      resolution of the youngest superfamilies, nuclear rDNA sequences were necessary
      to separate the basal groups. To try to reconcile these data sets into a single, 
      fully resolved orthopteran phylogeny, we adopted consensus and combined data
      strategies. The consensus analysis produced a partially resolved tree that lacked
      several well-supported features of the individual analyses. However, this lack of
      resolution was explained by an examination of resampled data sets, which
      identified the likely source of error as the relatively short length of the
      individual mitochondrial data partitions. In a subsequent comparison in which the
      mitochondrial sequences were initially combined, we observed less conflict. We
      then used two approaches to examine the validity of combining all of the data in 
      a single analysis: comparative analysis of trees recovered from resampled data
      sets, and the application of a randomization test. Because the results did not
      point to significant levels of heterogeneity in phylogenetic signal between the
      mitochondrial and nuclear data sets, we therefore proceeded with a combined
      analysis. Reconstructing phylogenies under the minimum evolution and maximum
      likelihood optimality criteria, we examined monophyly of the major orthopteran
      groups, using nonparametric and parametric bootstrap analysis and
      Kishino-Hasegawa tests. Our analysis suggests that phylogeny reconstruction under
      the maximum likelihood criteria is the most discriminating approach for the
      combined sequences. The results indicate, moreover, that the caeliferan
      Pneumoroidea and Pamphagoidea, as previously suggested, are polyphyletic. The
      Acridoidea is redefined to include all pamphagoid families other than the
      Pyrgomorphidae, which we propose should be accorded superfamily status.
FAU - Flook, P K
AU  - Flook PK
AD  - Zoology Institute, University of Basel, 4051-Basel, Switzerland.
      flook@ubaclu.unibas.ch
FAU - Klee, S
AU  - Klee S
FAU - Rowell, C H
AU  - Rowell CH
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - 0 (RNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Arthropods/*classification/genetics
MH  - DNA, Ribosomal/genetics
MH  - Insecta/*classification/genetics
MH  - Orthoptera/*classification/genetics
MH  - *Phylogeny
MH  - RNA, Ribosomal/genetics
MH  - Sequence Alignment
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260274 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Jun;48(2):233-53. doi: 10.1080/106351599260274.

PMID- 12066692
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 3
DP  - 1998 Sep
TI  - Systematic bias in phylogenetic analysis: is the Strepsiptera problem solved?
PG  - 519-37
FAU - Huelsenbeck, J P
AU  - Huelsenbeck JP
AD  - Laboratory of Molecular Systematics, Smithsonian Museum Support Center, 4210
      Silver Hill Road, Suitland, Maryland 20746, USA. johnh@onyx.si.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - DNA
MH  - Insecta/*classification/genetics
MH  - *Observer Variation
MH  - *Phylogeny
MH  - Rodentia/classification/genetics
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 1998 Sep;47(3):519-37.

PMID- 12066691
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 3
DP  - 1998 Sep
TI  - Why morphometrics is not special: coding quantitative data for phylogenetic
      analysis.
PG  - 508-19
AB  - Thorpe (1984) and Chappill (1989) argued that selection of a coding method should
      be [figure: see text] based on the purpose of coding. In our view, the purpose of
      coding is dictated by the principles of phylogenetic systematics. The foundation 
      of phylogenetic systematics is the observation that monophyletic groups can be
      recognized if homologous character states, shared evolutionary novelties, can be 
      identified (Hennig, 1966). Unfortunately, characters do not have labels
      indicating their homology. Instead, a systematist must propose a hypothesis of
      homology and evaluate its congruence with independent hypotheses based on other
      traits. In this context, the purpose of coding is to represent those hypotheses. 
      The major obstacle to coding is that the a priori groups under analysis (i.e.,
      taxa) often have ranges of variation that overlap to some degree. This is true
      whether traits are described qualitatively or quantitatively. One advantage of
      quantitative description is that it permits a more detailed analysis of how much 
      the ranges of variation overlap. It may seem appropriate to use statistical
      methods to summarize the amount of overlap and even to decide objectively (on a
      priori grounds) whether taxa are similar or different. Above, we demonstrated
      some of the problems resulting from these uses of statistical analysis. In our
      view, the most important problem is the implication that similarity of the
      feature across taxa is the basis for inferring homology. The similarity that is
      relevant to phylogenetic analysis is not proximity in morphospace, but shared
      novelty. Statistical methods can describe proximity, but they cannot recognize
      novelty. The method of coding we recommend uses graphical displays of individual 
      values. Coding decisions are based on all of the individuals in each taxon, not
      on summaries derived from models of expected distributions. Then, the evidence
      for inferring divergence is independently evaluated for each pair of overlapping 
      taxa. Coding decisions are not based on a priori rules that have no bearing on
      recognition of evolutionary novelty. This is the same approach that is used to
      code qualitatively described traits.
FAU - Swiderski, D L
AU  - Swiderski DL
AD  - Museum of Zoology, University of Michigan, Ann Arbor, Michigan 48109, USA.
      dlswider@umich.edu
FAU - Zelditch, M L
AU  - Zelditch ML
FAU - Fink, W L
AU  - Fink WL
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Morphogenesis
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 1998 Sep;47(3):508-19.

PMID- 12066690
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 3
DP  - 1998 Sep
TI  - Properties of matrix representation with parsimony analyses.
PG  - 497-508
FAU - Bininda-Emonds, O R
AU  - Bininda-Emonds OR
AD  - Department of Zoology, University of Oxford, Oxford OX1 3PS, United Kingdom.
      olaf.bininda@zoology.oxford.ac.uk
FAU - Bryant, H N
AU  - Bryant HN
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 1998 Sep;47(3):497-508.

PMID- 12066689
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 3
DP  - 1998 Sep
TI  - Archiving molecular phylogenetic alignments as NEXUS files.
PG  - 495-6
FAU - Cohen, B L
AU  - Cohen BL
AD  - University of Glasgow, Institute of Biomedical and Life Sciences, Division of
      Molecular Genetics, Pontecorvo Building, 56 Dumbarton Road, Glasgow G11 6NU,
      Scotland. b.l.cohen@bio.gla.ac.uk
FAU - Sheps, J A
AU  - Sheps JA
FAU - Wilkinson, M
AU  - Wilkinson M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Databases, Nucleic Acid
MH  - *Databases, Protein
MH  - *Phylogeny
MH  - Sequence Alignment
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260842 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Sep;47(3):495-6. doi: 10.1080/106351598260842.

PMID- 12066688
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 3
DP  - 1998 Sep
TI  - Incongruence between morphological and mitochondrial-DNA characters suggests
      hybrid origins of parthenogenetic weevil lineages (Genus aramigus).
PG  - 475-94
AB  - An expanded matrix of morphological characters for the genus Aramigus
      (Coleoptera: Curculionidae), which includes numerous polyploid parthenogenetic
      lineages, was compared and combined with a published matrix of mitochondrial DNA 
      (mtDNA) characters. The matrix of morphological characters provides little
      resolution of the A. tessellatus and A. uruguayensis species complexes but does
      resolve previously unresolved relationships among other morphologically defined
      species (A. globoculus + A. intermedius, A. curtulus + A. planioculus). The
      morphological and mtDNA characters are significantly incongruent (0.435 &lt; or = IM
      &lt; or = 0.463; IMF = 0.0735), according to the tests of Farris et al. (P = 0.010) 
      and Templeton (P &lt; 0.005), probably because of hybrid origins of polyploid
      parthenogenetic lineages. For the few sexual lineages included in both matrices, 
      morphology and mtDNA provide congruent estimates of phylogeny. In spite of recent
      injunctions against combining data sets that are incongruent because of differing
      histories, the results of the combined analyses were used to select one of the
      most-parsimonious mtDNA trees as the best estimate of maternal-lineage genealogy 
      and to reconstruct the evolution of parthenogenesis under the assumption that
      transitions from sexuality to parthenogenesis are irreversible. Where
      cytogenetically justified, as in weevils, the irreversibility assumption is
      useful for producing conservative estimates of the age of parthenogenetic
      lineages in spite of potential sampling bias against sexuals.
FAU - Normark, B B
AU  - Normark BB
AD  - Museum of Comparative Zoology, Harvard University, Cambridge, Massachusetts
      02138, USA. bnormark@oeb.harvard.edu
FAU - Lanteri, A A
AU  - Lanteri AA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Coleoptera/classification/*genetics
MH  - DNA, Mitochondrial/*genetics
MH  - *Parthenogenesis
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260833 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Sep;47(3):475-94. doi: 10.1080/106351598260833.

PMID- 12066687
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 3
DP  - 1998 Sep
TI  - The phylogenetic position of the musky rat-kangaroo and the evolution of bipedal 
      hopping in kangaroos (Macropodidae: Diprotodontia).
PG  - 457-74
AB  - Kangaroos and their relatives (family Macropodidae) are divided into the
      subfamilies Macropodinae (kangaroos, wallabies, pademelons) and Potoroinae
      (rat-kangaroos, potoroos, bettongs). The musky rat-kangaroo, Hypsiprymnodon
      moschatus, is traditionally allied with other potoroines, based primarily on the 
      basis of osteological characters and aspects of the female reproductive system.
      Unlike other macropodids, however, which are capable of bipedal hopping,
      Hypsiprymnodon is a quadrupedal bounder and lacks several derived features of the
      pes and tarsus that are presumably adaptations for bipedal hopping. Other derived
      features, such as a complex stomach, loss of P2 with the eruption of P3, and
      reduction of litter size to one, are also lacking in Hypsiprymnodon but occur in 
      all other macropodids. Thus, available evidence suggests that Hypsiprymnodon
      either is part of a monophyletic Potoroinae or is a sister taxon to other living 
      macropodids. To test these hypotheses, we sequenced 1,170 bp base pairs of the
      mitochondrial genome for 16 macropodids. Maximum parsimony, minimum evolution,
      maximum likelihood, and quartet puzzling all support the hypothesis that
      macropodines and potoroines are united to the exclusion of Hypsiprymnodon. This
      hypothesis implies that characters such as bipedal hopping evolved only once in
      macropodid evolution. Aside from Hypsiprymnodon, the remaining macropodids
      separate into the traditional Macropodinae and Potoroinae. Macropodines further
      separate into two clades: one containing the New Guinean forest wallabies
      Dorcopsis and Dorcopsulus, and one consisting of the genera Macropus, Setonix,
      Thylogale, Onychogalea, Wallabia, Dendrolagus, Peradorcas, and Lagorchestes.
      Among potoroines, there is moderate support for the association of Bettongia and 
      Aepyprymnus to the exclusion of Potorous. Divergence times were estimated by
      using 12S ribosomal RNA transversions. At the base of the macropodid radiation,
      Hypsiprymnodon diverged from other macropodids approximately 45 million years
      ago. This estimate is comparable to divergence estimates among families of
      Australasian possums based on single-copy DNA hybridization and 12S rRNA
      transversions. Macropodines and potoroines, in turn, diverged approximately 30
      million years ago. Among macropodines, Dorcopsis and Dorcopsulus separated from
      other taxa approximately 10 million years ago.
FAU - Burk, A
AU  - Burk A
AD  - Department of Biology, University of California, Riverside, California 92521,
      USA. angburk@citrus.ucr.edu
FAU - Westerman, M
AU  - Westerman M
FAU - Springer, M
AU  - Springer M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA
MH  - Macropodidae/*classification/genetics/physiology
MH  - Molecular Sequence Data
MH  - *Movement
MH  - *Phylogeny
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260824 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Sep;47(3):457-74. doi: 10.1080/106351598260824.

PMID- 12066686
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 3
DP  - 1998 Sep
TI  - Weighted ancestral area analysis and a solution of the redundant distribution
      problem.
PG  - 445-56
AB  - A new cladistic method for the estimation of ancestral areas is based on
      reversible parsimony in combination with a weighting scheme that weights steps in
      positionally plesiomorphic branches more highly than steps in positionally
      apomorphic branches. By applying this method to cladograms of human mitochondrial
      DNA, the method is superior to previously proposed algorithms. The method is also
      an appropriate tool for the solution of the redundant distribution problem in
      area cladograms. Under the assumption of allopatric speciation, redundant
      distributions, i.e., sympatry of sister groups, show that dispersal has occurred;
      thus, the ancestral area of at least one sister group was smaller than the
      combined distribution of its descendants. With the weighted ancestral area
      analysis, the ancestral areas can be confined and at least some dispersal events 
      can be distinguished from possible vicariance events. As applied to a cladogram
      of the Polypteridae, weighted ancestral area analysis is superior to Brooks
      parsimony analysis (assumption 0) and component analysis under assumptions 1 and 
      2 (Nelson and Platnick, 1981, Systematics and biogeography: Cladistics and
      vicariance. Columbia Univ. Press, New York.) in resolving redundancies. The
      results of the weighted ancestral area analysis may differ from the results of
      dispersal-vicariance analysis, because the rules of dispersal-vicariance analysis
      indirectly favor the questionable assumption that the ancestral species occupied 
      only one unit area.
FAU - Hausdorf, B
AU  - Hausdorf B
AD  - Zoologisches Institut und Zoologisches Museum der Universitat Hamburg,
      Martin-Luther-King-Platz 3, D-20146 Hamburg, Germany.
      hausdorf@zoologie.uni-hamburg.de
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Algorithms
MH  - DNA, Mitochondrial/genetics
MH  - Geography
MH  - Humans
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260815 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Sep;47(3):445-56. doi: 10.1080/106351598260815.

PMID- 12066685
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 3
DP  - 1998 Sep
TI  - The accuracy of methods for coding and sampling higher-level taxa for
      phylogenetic analysis: a simulation study.
PG  - 397-413
AB  - Many phylogenetic analyses, particularly morphological studies, use higher taxa
      (e.g., genera, families) rather than species as terminal taxa. This general
      approach requires dealing with interspecific variation among the species that
      make up the higher taxon. In this paper, I review different parsimony methods for
      coding and sampling higher taxa and compare their relative accuracies using
      computer simulations. Despite their widespread use, methods that involve coding
      higher taxa as terminals perform poorly in simulations, relative to splitting up 
      the higher taxa and using species as terminals. Among the methods that use higher
      taxa as terminals, coding a taxon based on the most common condition among the
      included species (majority or modal coding) is generally more accurate than other
      coding methods, such as coding taxa as missing or polymorphic. The success of the
      majority method, and results of further simulations, suggest that in many cases
      "common equals primitive" within variable taxa, at least for low and intermediate
      rates of character change. The fixed-only method (excluding variable characters) 
      performs very poorly, a result that is indirectly supported by analyses of
      published data for squamate reptiles. Sampling only a single species per higher
      taxon also yields low accuracy under many conditions. Along with recent studies
      of intraspecific polymorphism, the results of this study show the general
      importance of (1) including characters despite variation within taxa and (2)
      using methods that incorporate detailed information on the distribution of states
      within variable taxa.
FAU - Wiens, J J
AU  - Wiens JJ
AD  - Section of Amphibians and Reptiles, Carnegie Museum of Natural History,
      Pittsburgh, Pennsylvania 15213-4080, USA. wiensj@clpgh.org
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Computer Simulation
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260789 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Sep;47(3):397-413. doi: 10.1080/106351598260789.

PMID- 12066684
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 3
DP  - 1998 Sep
TI  - Data sets, partitions, and characters: philosophies and procedures for analyzing 
      multiple data sets.
PG  - 367-96
AB  - We compared four approaches for analyzing three data sets derived from
      staphylinoid beetles, a superfamily whose known species diversity is roughly
      comparable to that of vertebrates. One data set is derived from adult morphology 
      and the two molecular data sets are from 12S ribosomal RNA and cytochrome b
      mitochondrial DNA. We found that taxonomic congruence following conditional data 
      combination, herein called compatible evidence (CE), resolved more nodes
      compatible with an initial conservative hypothesis than did total evidence (TE), 
      conditional data combination (CDC), or taxonomic congruence (TC). CE sets a base 
      of nodes obtained by CDC analysis and then investigates what further agreement
      may arise in a universe where these nodes are accepted as given. We suggest that 
      CE75-75 may be appropriate for future studies that aim to both generate a
      well-corroborated tree and investigate conflicts between data sets, partitions,
      and characters. CE75-75 is a 75% bootstrap consensus CDC tree followed by
      combinable-component consensus of a 75% bootstrap consensus of each homogeneous
      set of partitions having hierarchical structure.
FAU - Ballard, J W
AU  - Ballard JW
AD  - Department of Zoology, Field Museum, Roosevelt Road at Lake Shore Drive, Chicago,
      Illinois 60605, USA.
FAU - Thayer, M K
AU  - Thayer MK
FAU - Newton, A F Jr
AU  - Newton AF Jr
FAU - Grismer, E R
AU  - Grismer ER
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 0 (RNA, Ribosomal)
RN  - 0 (RNA, ribosomal, 12S)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Coleoptera/*classification/genetics
MH  - Cytochrome b Group/genetics
MH  - Molecular Sequence Data
MH  - Nucleic Acid Conformation
MH  - Phylogeny
MH  - RNA, Ribosomal/chemistry/genetics
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260770 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Sep;47(3):367-96. doi: 10.1080/106351598260770.

PMID- 12066683
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 3
DP  - 1998 Sep
TI  - Reassessment of homology of morphological characters in tetractinellid sponges
      based on molecular data.
PG  - 351-66
AB  - In sponges, as in other taxa with simple organization, the evaluation and use of 
      morphological characters is difficult. Phylogenetic analysis of the first 850
      nucleotides from the 5' end of the 28S rRNA gene is used here to assess the
      homology of spicules used in the classification of the subclass Tetractinellida. 
      A single well-supported MP tree was obtained. The monophyly of the nine
      Tetractinellida species studied confirms the tetraxon megasclere as a
      morphological synapomorphy for the Tetractinellida. Two species are reallocated, 
      Penares helleri as a Geodiidae, now thought to have lost sterraster microscleres,
      and Stryphnus mucronatus to the Streptosclerophorida. SEM micrographs of
      Stryphnus microscleres show that the morphology of the sanidasters is compatible 
      with the hypothesis that they are homologous with streptoscleres and confirm this
      reallocation. Two other synapomorphies are confirmed within the tetractinellid
      clade, the simultaneous presence of tetraxon megasclere and aster-type
      microsclere (Astrophorida) and the loss of the streptosclere and persistence of
      the euaster s.s. microscleres (Euastrophorida) evidenced by the reallocation of
      Stryphnus mucronatus. The streptosclere microscleres cannot be evaluated in terms
      of homology because Streptosclerophorida may be paraphyletic (although these
      nodes are not supported by reliable bootstrap proportions) contrary to the
      currently accepted classification.
FAU - Chombard, C
AU  - Chombard C
AD  - Service de Systematique Moleculaire, CNRS GDR 1005, 43 rue Cuvier, Museum
      National d'Histoire Naturelle, 75005 Paris, France.
FAU - Boury-Esnault, N
AU  - Boury-Esnault N
FAU - Tillier, S
AU  - Tillier S
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (RNA, Ribosomal, 28S)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA/genetics
MH  - DNA Primers
MH  - Microscopy, Electron, Scanning
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Porifera/*classification/genetics/ultrastructure
MH  - RNA, Ribosomal, 28S/genetics
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260761 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Sep;47(3):351-66. doi: 10.1080/106351598260761.

PMID- 12066317
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Phylophenetics: frequencies and polymorphic characters in genealogical
      estimation.
PG  - 737-61
FAU - Murphy, R W
AU  - Murphy RW
AD  - Centre for Biodiversity and Conservation Biology, Royal Ontario Museum, 100
      Queen's Park, Toronto, Ontario, M5S 2C6, Canada.
FAU - Doyle, K D
AU  - Doyle KD
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Genealogy and Heraldry
MH  - *Phenotype
MH  - Phylogeny
MH  - *Polymorphism, Genetic
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260734 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):737-61. doi: 10.1080/106351598260734.

PMID- 12066316
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Reverse successive weighting.
PG  - 733-7
FAU - Trueman, J W
AU  - Trueman JW
AD  - Research School of Biological Sciences, Australian National University, ACT 0200,
      Australia. trueman@rsbs.anu.edu.au
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Alligators and Crocodiles/classification
MH  - Animals
MH  - Data Interpretation, Statistical
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260725 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):733-7. doi: 10.1080/106351598260725.

PMID- 12066315
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Combinability of phylogenies and bootstrap confidence envelopes.
PG  - 727-33
FAU - Rodrigo, A G
AU  - Rodrigo AG
AD  - Department of Microbiology, University of Washington, Box 357740, Seattle,
      Washington 98195-7740, USA. rodrigo@u.washington.edu
LA  - eng
GR  - AI-27757/AI/NIAID NIH HHS/United States
GR  - AI-32885/AI/NIAID NIH HHS/United States
PT  - Journal Article
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Confidence Intervals
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260716 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):727-33. doi: 10.1080/106351598260716.

PMID- 12066314
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Phylogenetic uncertainty, molecular sequences, and the definition of taxon names.
PG  - 719-26
FAU - Lee, M S
AU  - Lee MS
AD  - School of Biological Sciences, University of Sydney, Sydney, New South Wales
      2006, Australia. mike.lee@sci.monash.edu.au
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Phylogeny
MH  - *Terminology as Topic
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260707 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):719-26. doi: 10.1080/106351598260707.

PMID- 12066313
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Interpreting sister-group tests of key innovation hypotheses.
PG  - 710-8
FAU - de Queiroz, A
AU  - de Queiroz A
AD  - Department of Environmental, Population and Organismic Biology, University
      Museum, University of Colorado, Boulder, Colorado 80309-0334, USA.
      dequeiro@stripe.colorado.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260699 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):710-8. doi: 10.1080/106351598260699.

PMID- 12066312
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Taxon sampling and the accuracy of large phylogenies.
PG  - 702-10
FAU - Rannala, B
AU  - Rannala B
AD  - Department of Ecology and Evolution, State University of New York, Stony Brook,
      NY 11794-5245, USA. rannala@life.bio.sunysb.edu
FAU - Huelsenbeck, J P
AU  - Huelsenbeck JP
FAU - Yang, Z
AU  - Yang Z
FAU - Nielsen, R
AU  - Nielsen R
LA  - eng
GR  - GM40282/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260680 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):702-10. doi: 10.1080/106351598260680.

PMID- 12066311
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Conflicting phylogenetic patterns caused by molecular mechanisms in mitochondrial
      DNA sequences.
PG  - 696-701
FAU - Broughton, R E
AU  - Broughton RE
AD  - Department of Biology, Arizona State University, Tempe, Arizona 85287-1501, USA. 
      reb17@cornell.edu
FAU - Naylor, G J
AU  - Naylor GJ
FAU - Dowling, T E
AU  - Dowling TE
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA, Mitochondrial/*genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260671 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):696-701. doi: 10.1080/106351598260671.

PMID- 12066310
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Split support and split conflict randomization tests in phylogenetic inference.
PG  - 673-95
AB  - Randomization tests allow the formulation and statistical testing of null
      hypotheses about the quality of entire data sets or the quality of fit between
      the data and particular phylogenetic hypotheses. Randomization tests of
      phylogenetic hypotheses based on the concepts of split support and split conflict
      are described here, as are tests where splits, rather than the data, are randomly
      permuted. These tree-independent randomization tests are explored through their
      application to phylogenetic data for caecilian amphibians. Of these tests, split 
      support randomization tests appear to be the most promising tools for
      phylogeneticists. These tests seem quite conservative, are applicable to nonpolar
      data and unordered multistate characters, and do not have the problems of
      nonindependence that affect split conflict and hierarchy tests. Unlike split
      conflict tests, their power does not appear to be correlated with split size.
      However, all tests are sensitive to taxonomic scope. Split support tests may help
      discern data that are likely to be affected by the problems of long-branches
      effects. Comparison of test results for mutually incompatible splits may help
      identify the presence of strong misleading signals in phylogenetic data.
      Significant split support could be a prerequisite for considering phylogenetic
      hypotheses to be well supported by the data, and split support randomization
      tests might be usefully applied prior to or as part of tree construction.
FAU - Wilkinson, M
AU  - Wilkinson M
AD  - School of Biological Sciences, University of Bristol, Bristol BS8 1UG, England.
      m.wilkinson@nhm.ac.uk
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amphibians/classification
MH  - Animals
MH  - *Phylogeny
MH  - *Random Allocation
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260662 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):673-95. doi: 10.1080/106351598260662.

PMID- 12066309
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Effects of branch length errors on the performance of phylogenetically
      independent contrasts.
PG  - 654-72
AB  - We examined Type I error rates of Felsenstein's (1985; Am. Nat. 125:1-15)
      comparative method of phylogenetically independent contrasts when branch lengths 
      are in error and the model of evolution is not Brownian motion. We used seven
      evolutionary models, six of which depart strongly from Brownian motion, to
      simulate the evolution of two continuously valued characters along two different 
      phylogenies (15 and 49 species). First, we examined the performance of
      independent contrasts when branch lengths are distorted systematically, for
      example, by taking the square root of each branch segment. These distortions
      often caused inflated Type I error rates, but performance was almost always
      restored when branch length transformations were used. Next, we investigated
      effects of random errors in branch lengths. After the data were simulated, we
      added errors to the branch lengths and then used the altered phylogenies to
      estimate character correlations. Errors in the branches could be of two types:
      fixed, where branch lengths are either shortened or lengthened by a fixed
      fraction; or variable, where the error is a normal variate with mean zero and the
      variance is scaled to the length of the branch (so that expected error relative
      to branch length is constant for the whole tree). Thus, the error added is
      unrelated to the microevolutionary model. Without branch length checks and
      transformations, independent contrasts tended to yield extremely inflated and
      highly variable Type I error rates. Type I error rates were reduced, however,
      when branch lengths were checked and transformed as proposed by Garland et al.
      (1992; Syst. Biol. 41:18-32), and almost never exceeded twice the nominal P-value
      at alpha = 0.05. Our results also indicate that, if branch length transformations
      are applied, then the appropriate degrees of freedom for testing the significance
      of a correlation coefficient should, in general, be reduced to account for
      estimation of the best branch length transformation. These results extend those
      reported in Diaz-Uriarte and Garland (1996; Syst. Biol. 45:27-47), and show that,
      even with errors in branch lengths and evolutionary models different from
      Brownian motion, independent contrasts are a robust method for testing hypotheses
      of correlated evolution.
FAU - Diaz-Uriarte, R
AU  - Diaz-Uriarte R
AD  - Department of Zoology, 430 Lincoln Drive, University of Wisconsin, Madison,
      Wisconsin 53706-1381, USA.
FAU - Garland, T Jr
AU  - Garland T Jr
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260653 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):654-72. doi: 10.1080/106351598260653.

PMID- 12066308
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Individuality and the existence of species through time.
PG  - 641-53
AB  - The individuality of species provides the basis for linking practical taxonomy
      with evolutionary and ecological theory. An individual is here defined as a
      collection of parts (lower-level entities) that are mutually connected. Different
      types of species individual exist, based on different types of connection between
      organisms. An interbreeding species is a group of organisms connected by the
      potential to share common descendants, whereas a genealogical species is
      integrated by the sharing of common ancestors. Such species definitions serve to 
      set the limits of species at a moment of time and these slices connect through
      time to form time-extended lineages. This perspective on the nature of
      individuality has implications that conflict with traditional views of species
      and lineages: (1) Several types of connections among organisms may serve to
      individuate species in parallel (species pluralism); (2) each kind of species
      corresponds to a distinct kind of lineage; (3) although lineage branching is the 
      most obvious criterion to break lineages into diachronic species, it cannot be
      justified simply by reference to species individuality; (4) species (like other
      individuals) have fuzzy boundaries; (5) if we wish to retain a species rank, we
      should focus on either the most- or least-inclusive individual in a nested
      series; (6) not all organisms will be in any species; and (7) named species taxa 
      are best interpreted as hypotheses of real species. Although species
      individuality requires significant changes to systematic practice and challenges 
      some preconceptions we may have about the ontology of species, it provides the
      only sound basis for asserting that species exist independently of human
      perception.
FAU - Baum, D A
AU  - Baum DA
AD  - Department of Organismic and Evolutionary Biology, Harvard University Herbaria,
      22 Divinity Avenue, Cambridge, Massachusetts 02138, USA. dbaum@oeb.harvard.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260644 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):641-53. doi: 10.1080/106351598260644.

PMID- 12066307
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Does adding characters with missing data increase or decrease phylogenetic
      accuracy?
PG  - 625-40
AB  - Missing data are a widely recognized nuisance factor in phylogenetic analyses,
      and the fear of missing data may deter systematists from including characters
      that are highly incomplete. In this paper, I used simulations to explore the
      consequences of including sets of characters that contain missing data. More
      specifically, I tested whether the benefits of increasing the number of
      characters outweigh the costs of adding missing data cells to a matrix. The
      results show that the addition of a set of characters with missing data is
      generally more likely to increase phylogenetic accuracy than decrease it, but the
      potential benefits of adding these characters quickly disappear as the proportion
      of missing data increases. Furthermore, despite the overall trend, adding
      characters with missing data does decrease accuracy in some cases. In these
      situations, the missing data entries are not themselves misleading, but their
      presence may mimic the effects of limited taxon sampling, which can positively
      mislead. Criteria are discussed for predicting whether adding characters with
      missing data may increase or decrease accuracy. The results of this study also
      suggest that accuracy can be increased to a surprising degree by (1) "filling the
      holes" in a data matrix as much as possible (even when relatively few taxa are
      missing data), and (2) adding fewer characters scored for all taxa rather than
      adding a larger number of characters known for fewer taxa. Missing data can also 
      be eliminated from an analysis through the exclusion of incomplete taxa rather
      than incomplete characters, but this approach may reduce the usefulness of the
      analysis and (in some cases) the accuracy of the estimated trees.
FAU - Wiens, J J
AU  - Wiens JJ
AD  - Section of Amphibians and Reptiles, Carnegie Museum of Natural History,
      Pittsburgh, Pennsylvania 15213-4080, USA. wiensj@clpgh.org
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Data Interpretation, Statistical
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260635 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):625-40. doi: 10.1080/106351598260635.

PMID- 12066306
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Using a nonrecursive formula to determine cladogram probabilities.
PG  - 617-24
AB  - Three properties of bifurcating branching diagrams that are used for representing
      a specific number of taxa are (1) the number of possible arrangements, (2) the
      number of possible topologies, and (3) the probabilities of formation according
      to particular models of cladogenesis. Of these, the probabilities have received
      the least attention in the literature. Indeed, many biologists would be
      astonished by the observation that the probability of a commonly cited cladogram 
      containing 35 phyla of the animal kingdom is &lt; 0.0072% of the value of the
      average probability taken over all possible cladograms! We reviewed works on
      cladogram arrangements and topologies and developed a computer-generated table of
      enumerations that extends and corrects such tables in the literature. We also
      developed a nonrecursive formula for the determination of cladogram
      probabilities. This formula facilitates calculation and thereby should promote
      use of cladogram probabilities, which might provide more accurate null hypotheses
      for tests of cladogenic events than do considerations of cladogram arrangements
      or topologies.
FAU - Stone, J
AU  - Stone J
AD  - Department of Zoology, University of Toronto, Toronto, Ontario M5S 3G5, Canada.
      stone@life.bio.sunysb.edu
FAU - Repka, J
AU  - Repka J
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Likelihood Functions
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260626 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):617-24. doi: 10.1080/106351598260626.

PMID- 12066305
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Measuring the phylogenetic randomness of biological data sets.
PG  - 604-16
AB  - Two qualitative taxonomic characters are potentially compatible if the states of 
      each can be ordered into a character state tree in such a way that the two
      resulting character state trees are compatible. The number of potentially
      compatible pairs (NPCP) of qualitative characters from a data set may be
      considered to be a measure of its phylogenetic randomness. The value of NPCP
      depends on the number of evolutionary units (EUs), the number of characters, the 
      number of states in the characters, the distributions of EUs among these states, 
      and the amount and distribution of missing information and so does not directly
      indicate degree of phylogenetic randomness. Thus, for an observed data set, we
      used Monte Carlo methods to estimate the probability that a data set chosen
      equiprobably from among those identical (with respect to all the other above
      determining features) to the observed data set would have as high (or low) an
      NPCP as the observed data set. This probability, the realized significance of the
      observed NPCP, is attractive as an indication of phylogenetic randomness because 
      it does not require the assumptions made by other such methods: No character
      state trees are assumed and consequently, only potential compatibility can be
      determined; no particular method of phylogenetic estimation is assumed; and no
      phylogenetic trees are constructed. We determined the values and significances of
      NPCP for analyses of 57 data sets taken from 53 published sources. All data sets 
      from 37 of those sources exhibited realized significances of &lt; 0.01, indicating
      high levels of phylogenetic nonrandomness. From each of the remaining 16 sources,
      at least one data set was more phylogenetically random. Inclusion of outgroups
      changed significance in some cases, but not always in the same direction. Data
      sets with significantly low NPCP may be consistent with an ancient hybrid origin 
      (or other ancient polyphyletic gene exchange, crossing over, viral transfer,
      etc.) of the study group.
FAU - Day, W H
AU  - Day WH
AD  - Department of Biology, University of Michigan, Ann Arbor, Michigan 48109-1048,
      USA. whday@istar.ca
FAU - Estabrook, G F
AU  - Estabrook GF
FAU - McMorris, F R
AU  - McMorris FR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - *Random Allocation
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260617 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):604-16. doi: 10.1080/106351598260617.

PMID- 12066304
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Can weighting improve bushy trees? Models of cytochrome b evolution and the
      molecular systematics of pipits and wagtails (Aves: Motacillidae).
PG  - 589-603
AB  - Among-site rate variation (alpha) and transition bias (kappa) have been shown,
      most often as independent parameters, to be important dynamics in DNA evolution. 
      Accounting for these dynamics should result in better estimates of phylogenetic
      relationships. To test this idea, we simultaneously estimated overall (averaged
      over all codon positions) and codon-specific values of alpha and kappa, using
      maximum likelihood analyses of cytochrome b data from all genera of pipits and
      wagtails (Aves: Motacillidae), and six outgroup species, using initial trees
      generated with default values. Estimates of alpha and kappa were robust to
      initial tree topology and suggested substantial among-site rate variation even
      within codon classes; alpha was lowest (large among-site rate variation) at
      second-codon and highest (low among-site rate variation) at third-codon
      positions. When overall values were applied, there were shifts in tree topology
      and dramatic and statistically significant improvements in log-likelihood scores 
      of trees compared with the scores from application of default values. Applying
      codon-specific values resulted in yet another highly significant increase in
      likelihood. However, although incorporating substitution dynamics into maximum
      likelihood, maximum parsimony, and neighbor-joining analyses resulted in
      increases in congruence among trees, there were only minor improvements in
      phylogenetic signal, and none of the successive approximations tree topologies
      were statistically distinguishable from one another by the data. We suggest that 
      the bushlike nature of many higher-level phylogenies in birds makes estimating
      the dynamics of DNA evolution less sensitive to tree topology but also less
      susceptible to improvement via weighting.
FAU - Voelker, G
AU  - Voelker G
AD  - Burke Museum, Department of Zoology, Box 353010, University of Washington,
      Seattle, Washington 98195, USA. voelker@hrc.nevada.edu
FAU - Edwards, S V
AU  - Edwards SV
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Birds/*classification/genetics
MH  - Cytochrome b Group/*genetics
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - *Models, Genetic
MH  - Species Specificity
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260608 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):589-603. doi: 10.1080/106351598260608.

PMID- 12066303
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Step matrices and the interpretation of homoplasy.
PG  - 582-8
AB  - Assumptions about the costs of character change, coded in the form of a step
      matrix, determine most-parsimonious inferences of character evolution on
      phylogenies. We present a graphical approach to exploring the relationship
      between cost assumptions and evolutionary inferences from character data. The
      number of gains and losses of a binary trait on a phylogeny can be plotted over a
      range of cost assumptions, to reveal the inflection point at which there is a
      switch from more gains to more losses and the point at which all changes are
      inferred to be in one direction or the other. Phylogenetic structure in the data,
      the tree shape, and the relative frequency of states among the taxa influence the
      shape of such graphs and complicate the interpretation of possible
      permutation-based tests for directionality of change. The costs at which the
      most-parsimonious state of each internal node switches from one state to another 
      can also be quantified by iterative ancestral-state reconstruction over a range
      of costs. This procedure helps identify the most robust inferences of change in
      each direction, which should be of use in designing comparative studies.
FAU - Ree, R H
AU  - Ree RH
AD  - Department of Organismic and Evolutionary Biology, Harvard University, 22
      Divinity Avenue, Cambridge, Massachusetts 02138, USA. rree@oeb.harvard.edu
FAU - Donoghue, M J
AU  - Donoghue MJ
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Data Interpretation, Statistical
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260590 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):582-8. doi: 10.1080/106351598260590.

PMID- 12066302
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Combining data sets with different phylogenetic histories.
PG  - 568-81
AB  - The possibility that two data sets may have different underlying phylogenetic
      histories (such as gene trees that deviate from species trees) has become an
      important argument against combining data in phylogenetic analysis. However, two 
      data sets sampled for a large number of taxa may differ in only part of their
      histories. This is a realistic scenario and one in which the relative advantages 
      of combined, separate, and consensus analysis become much less clear. I propose a
      simple methodology for dealing with this situation that involves (1) partitioning
      the available data to maximize detection of different histories, (2) performing
      separate analyses of the data sets, and (3) combining the data but considering
      questionable or unresolved those parts of the combined tree that are strongly
      contested in the separate analyses (and which therefore may have different
      histories) until a majority of unlinked data sets support one resolution over
      another. In support of this methodology, computer simulations suggest that (1)
      the accuracy of combined analysis for recovering the true species phylogeny may
      exceed that of either of two separately analyzed data sets under some conditions,
      particularly when the mismatch between phylogenetic histories is small and the
      estimates of the underlying histories are imperfect (few characters, high
      homoplasy, or both) and (2) combined analysis provides a poor estimate of the
      species tree in areas of the phylogenies with different histories but gives an
      improved estimate in regions that share the same history. Thus, when there is a
      localized mismatch between the histories of two data sets, the separate,
      consensus, and combined analyses may all give unsatisfactory results in certain
      parts of the phylogeny. Similarly, approaches that allow data combination only
      after a global test of heterogeneity will suffer from the potential failings of
      either separate or combined analysis, depending on the outcome of the test.
      Excision of conflicting taxa is also problematic, in that doing so may obfuscate 
      the position of conflicting taxa within a larger tree, even when their placement 
      is congruent between data sets. Application of the proposed methodology to
      molecular and morphological data sets for Sceloporus lizards is discussed.
FAU - Wiens, J J
AU  - Wiens JJ
AD  - Section of Amphibians and Reptiles, Carnegie Museum of Natural History,
      Pittsburgh, Pennsylvania 15213-4080, USA. wiensj@clpgh.org
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Lizards/classification/genetics
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260581 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):568-81. doi: 10.1080/106351598260581.

PMID- 12066301
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 4
DP  - 1998 Dec
TI  - Phylogenetic congruence and discordance among one morphological and three
      molecular data sets from Pontederiaceae.
PG  - 545-67
AB  - A morphological data set and three sources of data from the chloroplast genome
      (two genes and a restriction site survey) were used to reconstruct the
      phylogenetic history of the pickerelweed family Pontederiaceae. The chloroplast
      data converged towards a single tree, presumably the true chloroplast phylogeny
      of the family. Unrooted trees estimated from each of the three chloroplast data
      sets were identical or extremely similar in shape to each other and mostly
      robustly supported. There was no evidence of significant heterogeneity among the 
      data sets, and the few topological differences seen among unrooted trees from
      each chloroplast data set are probably artifacts of sampling error on short
      branches. Despite well-documented differences in rates of evolution for different
      characters in individual data sets, equally weighted parsimony permits accurate
      reconstructions of chloroplast relationships in Pontederiaceae. A separate
      morphology-based data set yielded trees that were very different from the
      chloroplast trees. Although there was substantial support from the morphological 
      evidence for several major clades supported by chloroplast trees, most of the
      conflicting phylogenetic structure on the morphology trees was not robust.
      Nonetheless, several statistical tests of incongruence indicate significant
      heterogeneity between molecules and morphology. The source of this apparent
      incongruence appears to be a low ratio of phylogenetic signal to noise in the
      morphological data.
FAU - Graham, S W
AU  - Graham SW
AD  - Department of Botany, Box 355325, University of Washington, Seattle, Washington
      98195-5325, USA.
FAU - Kohn, J R
AU  - Kohn JR
FAU - Morton, B R
AU  - Morton BR
FAU - Eckenwalder, J E
AU  - Eckenwalder JE
FAU - Barrett, S C
AU  - Barrett SC
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
SB  - IM
MH  - Base Sequence
MH  - DNA Primers
MH  - Magnoliopsida/*classification
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260572 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Dec;47(4):545-67. doi: 10.1080/106351598260572.

PMID- 12066300
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 4
DP  - 1999 Dec
TI  - How should species phylogenies be inferred from sequence data?
PG  - 814-25
FAU - Slowinski, J B
AU  - Slowinski JB
AD  - Department of Herpetology, California Academy of Sciences, Golden Gate Park, San 
      Francisco, California 94118, USA. jslowins@cas.calacademy.org
FAU - Page, R D
AU  - Page RD
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Phylogeny
MH  - Snakes/classification
MH  - *Species Specificity
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260030 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Dec;48(4):814-25. doi: 10.1080/106351599260030.

PMID- 12066299
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 4
DP  - 1999 Dec
TI  - Species names in phylogenetic nomenclature.
PG  - 790-807
AB  - Linnaean binomial nomenclature is logically incompatible with the phylogenetic
      nomenclature of de Queiroz and Gauthier (1992, Annu. Rev. Ecol. Syst.
      23:449-480): The former is based on the concept of genus, thus making this rank
      mandatory, while the latter is based on phylogenetic definitions and requires the
      abandonment of mandatory ranks. Thus, if species are to receive names under
      phylogenetic nomenclature, a different method must be devised to name them. Here,
      13 methods for naming species in the context of phylogenetic nomenclature are
      contrasted with each other and with Linnaean binomials. A fundamental dichotomy
      among the proposed methods distinguishes those that retain the entire binomial of
      a preexisting species name from those that retain only the specific epithet.
      Other relevant issues include the stability, uniqueness, and ease of
      pronunciation of species names; their capacity to convey phylogenetic
      information; and the distinguishability of species names that are governed by a
      code of phylogenetic nomenclature both from clade names and from species names
      governed by the current codes. No method is ideal. Each has advantages and
      drawbacks, and preference for one option over another will be influenced by one's
      evaluation of the relative importance of the pros and cons for each. Moreover,
      sometimes the same feature is viewed as an advantage by some and a drawback by
      others. Nevertheless, all of the proposed methods for naming species in the
      context of phylogenetic nomenclature provide names that are more stable than
      Linnaean binomials.
FAU - Cantino, P D
AU  - Cantino PD
AD  - Department of Environmental and Plant Biology, Ohio University, Athens, Ohio
      45701, USA. cantino@ohio.edu
FAU - Bryant, H N
AU  - Bryant HN
FAU - de Queiroz, K
AU  - de Queiroz K
FAU - Donoghue, M J
AU  - Donoghue MJ
FAU - Eriksson, T
AU  - Eriksson T
FAU - Hillis, D M
AU  - Hillis DM
FAU - Lee, M S
AU  - Lee MS
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Phylogeny
MH  - *Species Specificity
MH  - *Terminology as Topic
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260012 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Dec;48(4):790-807. doi: 10.1080/106351599260012.

PMID- 12066298
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 4
DP  - 1999 Dec
TI  - Molecular evidence for the compilospecies model of reticulate evolution in
      Armeria (Plumbaginaceae).
PG  - 735-54
AB  - Cladistic analyses of the nuclear ribosomal DNA (nrDNA) internal transcribed
      spacer (ITS) sequences from 55 samples corresponding to 34 taxa in the genus
      Armeria reveal that ITS sequence diversity among and within species utterly
      conflicts with patterns of morphological similarity. Three facts are apparent
      from the results here reported: (1) different samples of a single subspecies, A. 
      villosa subsp. longiaristata, appear in three of the five major clades; (2)
      samples of at least one of the six subspecies of A. villosa appear in four of the
      five major clades; and (3) the composition of major clades shows greater
      congruence with the geographic origin of plants than with the traditional
      systematic arrangement based primarily on morphology. Specifically, the clades
      here termed Ia, II, III, and IV each encompass terminals restricted to
      geographically delimited areas. There are alternative explanations for the ITS
      pattern, but the most likely one is that nucleotide positions supporting the
      major clades are due, in some of the samples, to concerted evolution following
      horizontal transfer (gene flow) rather than to recency of common ancestry. This
      interpretation is consistent with previous systematic and experimental evidence
      and implies that reticulation in Armeria may be extensive. Harlan and de Wet
      (1963, Evolution 17:497-501) proposed the compilospecies concept to account for
      situations in which a genetically "aggressive" species captures portions of the
      genome of other sympatric species by means of extensive introgression. Evidence
      of extensive reticulation, ecological diversification, and geographic pattern
      indicates that A. villosa may fit the compilospecies concept, which is here
      supported on molecular grounds for the first time.
FAU - Fuertes Aguilar, J
AU  - Fuertes Aguilar J
AD  - Real Jardin Botanico, CSIC. Plaza de Murillo, 2, 28014 Madrid, Spain.
      jfuertes@ma-rjb.csic.es
FAU - Rossello, J A
AU  - Rossello JA
FAU - Nieto Feliner, G
AU  - Nieto Feliner G
LA  - eng
SI  - GENBANK/AJ132446
SI  - GENBANK/AJ225513
SI  - GENBANK/AJ225563
SI  - GENBANK/AJ225564
SI  - GENBANK/AJ225565
SI  - GENBANK/AJ225566
SI  - GENBANK/AJ225567
SI  - GENBANK/AJ225568
SI  - GENBANK/AJ225569
SI  - GENBANK/AJ225570
SI  - GENBANK/AJ225571
SI  - GENBANK/AJ225572
SI  - GENBANK/AJ225573
SI  - GENBANK/AJ225574
SI  - GENBANK/AJ225575
SI  - GENBANK/AJ225576
SI  - GENBANK/AJ225577
SI  - GENBANK/AJ225578
SI  - GENBANK/AJ225579
SI  - GENBANK/AJ225580
SI  - GENBANK/AJ225581
SI  - GENBANK/AJ225582
SI  - GENBANK/AJ225583
SI  - GENBANK/AJ225584
SI  - GENBANK/AJ225585
SI  - GENBANK/AJ225586
SI  - GENBANK/AJ225587
SI  - GENBANK/AJ225588
SI  - GENBANK/AJ225589
SI  - GENBANK/AJ225590
SI  - GENBANK/AJ225591
SI  - GENBANK/AJ225592
SI  - GENBANK/AJ225593
SI  - GENBANK/AJ225594
SI  - GENBANK/AJ225595
SI  - GENBANK/AJ225596
SI  - GENBANK/AJ225597
SI  - GENBANK/AJ225598
SI  - GENBANK/AJ225599
SI  - GENBANK/AJ225600
SI  - GENBANK/AJ225601
SI  - GENBANK/AJ225602
SI  - GENBANK/AJ225603
SI  - GENBANK/AJ225604
SI  - GENBANK/AJ225605
SI  - GENBANK/AJ225606
SI  - GENBANK/AJ225607
SI  - GENBANK/AJ225608
SI  - GENBANK/AJ225609
SI  - GENBANK/AJ225610
SI  - GENBANK/AJ225611
SI  - GENBANK/AJ225612
SI  - GENBANK/AJ225614
SI  - GENBANK/AJ225615
SI  - GENBANK/AJ225616
SI  - GENBANK/AJ225617
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Ribosomal)
SB  - IM
EIN - Syst Biol 2000 Mar;49(1):181
MH  - Base Sequence
MH  - DNA Primers
MH  - DNA, Ribosomal/genetics
MH  - *Evolution, Molecular
MH  - Molecular Sequence Data
MH  - Plumbaginaceae/*classification/genetics
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599259997 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Dec;48(4):735-54. doi: 10.1080/106351599259997.

PMID- 12066297
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 4
DP  - 1999 Dec
TI  - Molecular phylogeny of the marmots (Rodentia: Sciuridae): tests of evolutionary
      and biogeographic hypotheses.
PG  - 715-34
AB  - There are 14 species of marmots distributed across the Holarctic, and despite
      extensive systematic study, their phylogenetic relationships remain largely
      unresolved. In particular, comprehensive studies have been lacking. A
      well-supported phylogeny is needed to place the numerous ecological and
      behavioral studies on marmots in an evolutionary context. To address this
      situation, we obtained complete cytochrome (cyt) b sequences for 13 of the
      species and a partial sequence for the 14th. We applied a statistical approach to
      both phylogeny estimation and hypothesis testing, using parsimony and maximum
      likelihood-based methods. We conducted statistical tests on a suite of previously
      proposed hypotheses of phylogenetic relationships and biogeographic histories.
      The cyt b data strongly support the monophyly of Marmota and a western montane
      clade in the Nearctic. Although some other scenarios cannot be rejected, the
      results are consistent with an initial diversification in North America, followed
      by an invasion and subsequent rapid diversification in the Palearctic. These
      analyses reject the two major competing hypotheses of M. broweri's phylogenetic
      relationships--namely, that it is the sister species to M. camtschatica of
      eastern Siberia, and that it is related closely to M. caligata of the Nearctic.
      The Alaskan distribution of M. broweri is best explained as a reinvasion from the
      Palearctic, but a Nearctic origin can not be rejected. Several other
      conventionally recognized species groups can also be rejected. Social evolution
      has been homoplastic, with large colonial systems evolving in two groups
      convergently. The cyt b data do not provide unambiguous resolution of several
      basal nodes in the Palearctic radiation, leaving some aspects of pelage and
      karyotypic evolution equivocal.
FAU - Steppan, S J
AU  - Steppan SJ
AD  - Laboratory of Molecular Systematics, Smithsonian Institution, 4210 Silver Hill
      Road, Suitland, Maryland 20746, USA. steppan@bio.fsu.edu
FAU - Akhverdyan, M R
AU  - Akhverdyan MR
FAU - Lyapunova, E A
AU  - Lyapunova EA
FAU - Fraser, D G
AU  - Fraser DG
FAU - Vorontsov, N N
AU  - Vorontsov NN
FAU - Hoffmann, R S
AU  - Hoffmann RS
FAU - Braun, M J
AU  - Braun MJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 0 (DNA Primers)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Cytochrome b Group/genetics
MH  - DNA Primers
MH  - *Evolution, Molecular
MH  - *Geography
MH  - Marmota/*classification/genetics
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599259988 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Dec;48(4):715-34. doi: 10.1080/106351599259988.

PMID- 12066296
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 4
DP  - 1999 Dec
TI  - Data set incongruence and correlated character evolution: an example of
      functional convergence in the hind-limbs of stifftail diving ducks.
PG  - 683-714
AB  - The unwitting inclusion of convergent characters in phylogenetic estimates poses 
      a serious problem for efforts to recover phylogeny. Convergence is not
      inscrutable, however, particularly when one group of characters tracks phylogeny 
      and another set tracks adaptive history. In such cases, convergent characters may
      be correlated with one or a few functional anatomical units and readily
      identifiable by using comparative methods. Stifftail ducks (Oxyurinae) offer one 
      such opportunity to study correlated character evolution and function in the
      context of phylogenetic reconstruction. Morphological analyses place stifftail
      ducks as part of a large clade of diving ducks that includes the sea ducks
      (Mergini), Hymenolaimus, Merganetta, and Tachyeres, and possibly the pochards
      (Aythyini). Molecular analyses, on the other hand, place stifftails far from
      other diving ducks and suggest, moreover, that stifftails are polyphyletic.
      Mitochondrial cytochrome b gene sequences of eight stifftail species
      traditionally supposed to form a clade were compared with each other and with
      sequences from 50 other anseriform and galliform species. Stifftail ducks are not
      the sister group of sea ducks but lie outside the typical ducks (Anatinae). Of
      the four traditional stifftail genera, monophyly of Oxyura and its sister group
      relationship with Nomonyx are strongly supported. Heteronetta probably is the
      sister group of that clade, but support is weak. Biziura is not a true stifftail.
      Within Oxyura, Old World species (O. australis, O. leucocephala, O. maccoa)
      appear to form a clade, with New World species (O. jamaicensis, O. vittata)
      branching basally. Incongruence between molecules and morphology is interpreted
      to be the result of adaptive specialization and functional convergence in the
      hind limbs of Biziura and true stifftails. When morphological characters are
      divided into classes, only hind-limb characters are significantly in conflict
      with the molecular tree. Likewise, null models of synonymous and nonsynonymous
      substitution based on patterns of codon-degeneracy and chemical dissimilarity
      indicate that the nucleotide and amino acid changes postulated by the molecular
      tree are more plausible than those postulated by the morphological tree. These
      findings teach general lessons about the utility of highly adaptive characters
      (in particular those related to foraging ecology) and underscore the problems
      that convergence can pose for attempts to recover phylogeny. They also
      demonstrate how the concept of natural data partitions and simple models of
      evolution (e.g., parsimony, likelihood, neutrality) can be used to test the
      accuracy of independent phylogenetic estimates and provide arguments in favor of 
      one tree topology over another.
FAU - McCracken, K G
AU  - McCracken KG
AD  - School of Forestry, Wildlife, and Fisheries, Louisiana State University, Baton
      Rouge, Louisiana 70803, USA. kmccrack@darwin.helios.nd.edu
FAU - Harshman, J
AU  - Harshman J
FAU - McClellan, D A
AU  - McClellan DA
FAU - Afton, A D
AU  - Afton AD
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
SB  - IM
MH  - Amino Acid Substitution
MH  - Animals
MH  - Cytochrome b Group/genetics
MH  - Ducks/*classification
MH  - *Evolution, Molecular
MH  - Hindlimb/*physiology
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599259979 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Dec;48(4):683-714. doi: 10.1080/106351599259979.

PMID- 12066295
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 3
DP  - 1999 Sep
TI  - Sampling confidence envelopes of phylogenetic trees for combinability testing: a 
      reply to Rodrigo.
PG  - 596-603
FAU - Lutzoni, F
AU  - Lutzoni F
AD  - Department of Botany, Field Museum of Natural History, Roosevelt Road at Lake
      Shore Drive, Chicago, Illinois 60605, USA. flutzoni@fmnh.org
FAU - Barker, F K
AU  - Barker FK
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Confidence Intervals
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260166 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Sep;48(3):596-603. doi: 10.1080/106351599260166.

PMID- 12066294
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 3
DP  - 1999 Sep
TI  - Assessing congruence between cladistic and stratigraphic data.
PG  - 581-96
FAU - Benton, M J
AU  - Benton MJ
AD  - Department of Earth Sciences, University of Bristol, Bristol BS8 1RJ, U.K.
      mike.benton@bristol.ac.uk
FAU - Hitchin, R
AU  - Hitchin R
FAU - Wills, M A
AU  - Wills MA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260157 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Sep;48(3):581-96. doi: 10.1080/106351599260157.

PMID- 12066293
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 3
DP  - 1999 Sep
TI  - Polytomies and phylogenetically independent contrasts: examination of the bounded
      degrees of freedom approach.
PG  - 547-58
AB  - We examined the effect of soft polytomies on the performance (Type I error rate
      and bias) of Felsenstein's (1985; Am. Nat. 125:1-15) method of phylogenetically
      independent contrasts for estimating a bivariate correlation. We specifically
      tested the adequacy of bounding degrees of freedom, as suggested by Purvis and
      Garland (1993; Syst. Biol. 42:569-575). We simulated bivariate character
      evolution under Brownian motion (assumed by independent contrasts) and eight
      other models on five phylogenetic trees. For non-Brownian motion simulations, the
      adequacy of branch-length standardization was checked with a simple diagnostic
      (Garland et al., 1992; Syst. Biol. 41:18-32), and transformations were applied as
      indicated. Surprisingly, soft polytomies tended to have negligible effects on
      Type I error rates when models other than Brownian motion were used. Overall, and
      irrespective of evolutionary model, degrees of freedom were appropriately bounded
      for hypothesis testing, and unbiased estimates of the correlation coefficient
      were obtained. Our results, along with those of previous simulation studies,
      suggest that independent contrasts can reliably be applied to real data, even
      with phylogenetic uncertainty.
FAU - Garland, T Jr
AU  - Garland T Jr
AD  - Department of Zoology, 430 Lincoln Drive, University of Wisconsin, Madison,
      Wisconsin 53706-1381, USA. tgarland@facstaff.wisc.edu
FAU - Diaz-Uriarte, R
AU  - Diaz-Uriarte R
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Models, Genetic
MH  - Phenotype
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260139 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Sep;48(3):547-58. doi: 10.1080/106351599260139.

PMID- 12066292
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 3
DP  - 1999 Sep
TI  - Primates and their pinworm parasites: the cameron hypothesis revisited.
PG  - 523-46
AB  - A morphologically based cladistic analysis of the Enterobiinae, which includes
      most of the Oxyuridae parasitic in Primates, allows a reevaluation of the
      Cameron's hypothesis of close coevolution with cospeciation between hosts and
      parasites. Each of the three genera separated in the Enterobiinae fits with one
      of the suborders defined in Primates: Lemuricola with the Strepsirhini,
      Trypanoxyuris with the Platyrrhini, and Enterobius with the Catarrhini. Inside
      each of the three main groups, the subdivisions observed in the parasite tree
      also fit with many of the subdivisions generally accepted within the Primate
      order. These results confirm the subgroups previously described in the subfamily 
      and support Cameron's hypothesis in its aspect of association by descent.
      Although the classification of the Enterobiinae generally closely underlines the 
      classification of Primates, several discordances also are observed. These are
      discussed case by case, with use of computed reconstruction scenarios. Given that
      the occurrences of the same pinworm species as a parasite for several congeneric 
      host species is not the generalized pattern, and given that several occurrences
      also are observed in which the speciations of the parasites describe a more
      complex network, Cameron's hypothesis of a slower rhythm of speciation in the
      parasites can be considered partly refuted. The presence of two genera parasitic 
      on squirrels in a family that contains primarily primate parasites also is
      discussed. The cladistic analysis does not support close relationships between
      the squirrel parasites and suggests an early separation from the Enterobiinae for
      the first (Xeroxyuris), and a tardy host-switching from the Platyrrhini to the
      squirrels for the second (Rodentoxyuris).
FAU - Hugot, J P
AU  - Hugot JP
AD  - Museum National d'Histoire Naturelle, Institut de Systematique, FR 1541 du CNRS, 
      Laboratoire de Zoologie des Mammiferes et Oiseaux (Biosystematique et Coevolution
      chez les Nematodes Parasites), 55, rue Buffon, 75231 Paris, France.
      hugot@cimrs1.mnhn.fr
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Enterobius/classification/genetics/*isolation &amp; purification
MH  - Phylogeny
MH  - Primates/*parasitology
MH  - Sciuridae/parasitology
MH  - Species Specificity
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260120 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Sep;48(3):523-46. doi: 10.1080/106351599260120.

PMID- 12066291
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 3
DP  - 1999 Sep
TI  - The position of Cetacea within mammalia: phylogenetic analysis of morphological
      data from extinct and extant taxa.
PG  - 455-90
AB  - Knowledge of the phylogenetic position of the order Cetacea (whales, dolphins,
      and porpoises) within Mammalia is of central importance to evolutionary
      biologists studying the transformations of biological form and function that
      accompanied the shift from fully terrestrial to fully aquatic life in this clade.
      Phylogenies based on molecular data and those based on morphological data both
      place cetaceans among ungulates but are incongruent in other respects.
      Morphologists argue that cetaceans are most closely related to mesonychians, an
      extinct group of terrestrial ungulates. They have disagreed, however, as to
      whether Perissodactyla (odd-toed ungulates) or Artiodactyla (even-toed ungulates)
      is the extant clade most closely related to Cetacea, and have long maintained
      that each of these orders is monophyletic. The great majority of molecule-based
      phylogenies show, by contrast, not only that artiodactyls are the closest extant 
      relatives of Cetacea, but also that Artiodactyla is paraphyletic unless cetaceans
      are nested within it, often as the sister group of hippopotamids. We tested
      morphological evidence for several hypotheses concerning the sister taxon
      relationships of Cetacea in a maximum parsimony analysis of 123 morphological
      characters from 10 extant and 30 extinct taxa. We advocate treating certain
      multistate characters as ordered because such a procedure incorporates
      information about hierarchical morphological transformation. In all
      most-parsimonious trees, whether multistate characters are ordered or unordered, 
      Artiodactyla is the extant sister taxon of Cetacea. With certain multistate
      characters ordered, the extinct clade Mesonychia (Mesonychidae + Hapalodectidae) 
      is the sister taxon of Cetacea, and Artiodactyla is monophyletic. When all
      fossils are removed from the analysis, Artiodactyla is paraphyletic with Cetacea 
      nested inside, indicating that inclusion of mesonychians and other extinct stem
      taxa in a phylogenetic analysis of the ungulate clade is integral to the recovery
      of artiodactyl monophyly. Phylogenies derived from molecular data alone may risk 
      recovering inconsistent branches because of an inability to sample extinct
      clades, which by a conservative estimate, amount to 89% of the ingroup. Addition 
      of data from recently described astragali attributed to cetaceans does not
      overturn artiodactyl monophyly.
FAU - O'Leary, M A
AU  - O'Leary MA
AD  - Department of Anatomical Sciences, State University of New York, Stony Brook, New
      York 11794-8081, USA. moleary@mail.som.sunysb.edu
FAU - Geisler, J H
AU  - Geisler JH
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Cetacea/*classification/genetics
MH  - *Phylogeny
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260102 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Sep;48(3):455-90. doi: 10.1080/106351599260102.

PMID- 12066290
OWN - NLM
STAT- MEDLINE
DCOM- 20020710
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 48
IP  - 3
DP  - 1999 Sep
TI  - More characters or more taxa for a robust phylogeny--case study from the coffee
      family (Rubiaceae).
PG  - 413-35
AB  - Using different data sets mainly from the plant family Rubiaceae, but in parts
      also from the Apocynaceae, Asteraceae, Lardizabalaceae, Saxifragaceae, and
      Solanaceae, we have investigated the effect of number of characters, number of
      taxa, and kind of data on bootstrap values within phylogenetic trees. The
      percentage of supported nodes within a tree is positively correlated with the
      number of characters, and negatively correlated with the number of taxa. The
      morphological analyses are based on few characters and weakly supported trees are
      expected. The percentage of supported nodes is also dependent on the kind of data
      analyzed. In analyses of Rubiaceae based on the same number of characters, RFLP
      data give trees with higher percentage of supported nodes than rbcL and
      morphological data. We also discuss the support values for particular nodes at
      the familial and subfamilial levels. Two new data sets of ndhF and rbcL sequences
      of Rubiaceae are analyzed and together with earlier studies of the family we can 
      conclude that the monophyly of the Rubiaceae is supported and within the family
      there are three well supported, but not easily characterized, large subfamilies, 
      Rubioideae, Cinchonoideae s.s. and Ixoroideae s.l. There are also a few genera
      (Luculia and Coptosapelta) unclassified to subfamily.
FAU - Bremer, B
AU  - Bremer B
AD  - Department of Systematic Botany, Evolutionary Biology Centre, Uppsala University,
      Norbyv, 18D, SE-75236 Uppsala, Sweden. birgitta.bremer@systbot.uu.se
FAU - Jansen, R K
AU  - Jansen RK
FAU - Oxelman, B
AU  - Oxelman B
FAU - Backlund, M
AU  - Backlund M
FAU - Lantz, H
AU  - Lantz H
FAU - Kim, K J
AU  - Kim KJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Coffee)
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Coffee/*classification/genetics
MH  - DNA, Plant
MH  - *Phylogeny
MH  - Polymorphism, Restriction Fragment Length
EDAT- 2002/06/18 10:00
MHDA- 2002/07/12 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/12 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351599260085 [doi]
PST - ppublish
SO  - Syst Biol. 1999 Sep;48(3):413-35. doi: 10.1080/106351599260085.

PMID- 12064244
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - Morphology, molecules, and the phylogenetics of cetaceans.
PG  - 90-124
AB  - Recent phylogenetic analyses of cetacean relationships based on DNA sequence data
      have challenged the traditional view that baleen whales (Mysticeti) and toothed
      whales (Odontoceti) are each monophyletic, arguing instead that baleen whales are
      the sister group of the odontocete family Physeteridae (sperm whales). We
      reexamined this issue in light of a morphological data set composed of 207
      characters and molecular data sets of published 12S, 16S, and cytochrome b
      mitochondrial DNA sequences. We reach four primary conclusions: (1) Our
      morphological data set strongly supports the traditional view of odontocete
      monophyly; (2) the unrooted molecular and morphological trees are very similar,
      and most of the conflict results from alternative rooting positions; (3) the
      rooting position of the molecular tree is sensitive to choice of artiodactyls
      outgroup taxa and the treatment of two small but ambiguously aligned regions of
      the 12S and 16S sequences, whereas the morphological root is strongly supported; 
      and (4) combined analyses of the morphological and molecular data provide a
      well-supported phylogenetic estimate consistent with that based on the
      morphological data alone (and the traditional view of toothed-whale monophyly)
      but with increased bootstrap support at nearly every node of the tree.
FAU - Messenger, S L
AU  - Messenger SL
AD  - Department of Zoology, University of Texas at Austin, Austin, Texas 78712-1064,
      USA.
FAU - McGuire, J A
AU  - McGuire JA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Body Constitution
MH  - Cetacea/*anatomy &amp; histology/classification/*genetics
MH  - Classification/methods
MH  - DNA/chemistry/genetics
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598261058 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):90-124. doi: 10.1080/106351598261058.

PMID- 12064243
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - Is it better to add taxa or characters to a difficult phylogenetic problem?
PG  - 9-17
AB  - The effects on phylogenetic accuracy of adding characters and/or taxa were
      explored using data generated by computer simulation. The conditions of this
      study were constrained but allowed for systematic investigation of certain
      parameters. The starting point for the study was a four-taxon tree in the
      "Felsenstein zone," representing a difficult phylogenetic problem with an extreme
      situation of long branch attraction. Taxa were added sequentially to this tree in
      a manner specifically designed to break up the long branches, and for each tree
      data matrices of different sizes were simulated. Phylogenetic trees were
      reconstructed from these data using the criteria of parsimony and maximum
      likelihood. Phylogenetic accuracy was measured in three ways: (1) proportion of
      trees that are completely correct, (2) proportion of correctly reconstructed
      branches in all trees, and (3) proportion of trees in which the original
      four-taxon statement is correctly reconstructed. Accuracy improved dramatically
      with the addition of taxa and much more slowly with the addition of characters.
      If taxa can be added to break up long branches, it is much more preferable to add
      taxa than characters.
FAU - Graybeal, A
AU  - Graybeal A
AD  - Department of Zoology, University of Texas, Austin, Texas 78712,
      USA.graybeal@fmpprfmnh.org
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260996 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):9-17. doi: 10.1080/106351598260996.

PMID- 12064242
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - A fast method for approximating maximum likelihoods of phylogenetic trees from
      nucleotide sequences.
PG  - 77-89
AB  - We have developed a rapid parsimony method for reconstructing ancestral
      nucleotide states that allows calculation of initial branch lengths that are good
      approximations to optimal maximum-likelihood estimates under several commonly
      used substitution models. Use of these approximate branch lengths (rather than
      fixed arbitrary values) as starting points significantly reduces the time
      required for iteration to a solution that maximizes the likelihood of a tree.
      These branch lengths are close enough to the optimal values that they can be used
      without further iteration to calculate approximate maximum-likelihood scores that
      are very close to the "exact" scores found by iteration. Several strategies are
      described for using these approximate scores to substantially reduce times needed
      for maximum-likelihood tree searches.
FAU - Rogers, J S
AU  - Rogers JS
AD  - Department of Biological Sciences, University of New Orleans, New Orleans,
      Louisiana 70148, USA. jsrbs@uno.edu
FAU - Swofford, D L
AU  - Swofford DL
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - 0 (RNA, Ribosomal)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - *Base Sequence
MH  - Chordata, Nonvertebrate
MH  - DNA/chemistry/genetics
MH  - DNA, Ribosomal/chemistry/genetics
MH  - Decision Trees
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - RNA, Ribosomal/genetics
MH  - Regression Analysis
MH  - Reproducibility of Results
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598261049 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):77-89. doi: 10.1080/106351598261049.

PMID- 12064241
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - Amphioxus mitochondrial DNA, chordate phylogeny, and the limits of inference
      based on comparisons of sequences.
PG  - 61-76
AB  - Analyses of both the nucleotide and amino acid sequences derived from all 13
      mitochondrial protein-encoding genes (12,234 bp) of 19 metazoan species,
      including that of the lancelet Branchiostoma floridae ("amphioxus"), fail to
      yield the widely accepted phylogeny for chordates and, within chordates, for
      vertebrates. Given the breadth and the compelling nature of the data supporting
      that phylogeny, relationships supported by the mitochondrial sequence comparisons
      are almost certainly incorrect, despite their being supported by equally weighted
      parsimony, distance, and maximum-likelihood analyses. The incorrect groupings
      probably result in part from convergent base-compositional similarities among
      some of the taxa, similarities that are strong enough to overwhelm the historical
      signal. Comparisons among very distantly related taxa are likely to be
      particularly susceptible to such artifacts, because the historical signal is
      already greatly attenuated. Empirical results underscore the need for approaches 
      to phylogenetic inference that go beyond simple site-by-site comparison of
      aligned sequences. This study and others indicate that, once a sequence sample of
      reasonable size has been obtained, accurate phylogenetic estimation may be better
      served by incorporating knowledge of molecular structures and processes into
      inference models and by seeking additional higher order characters embedded in
      those sequences, than by gathering ever larger sequence samples from the same
      organisms in he hope that the historical signal will eventually prevail.
FAU - Naylor, G J
AU  - Naylor GJ
AD  - Department of Zoology and Genetics, Iowa State University, Ames, Iowa 50011, USA.
      gnaylor@iastate.edu
FAU - Brown, W M
AU  - Brown WM
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Chordata, Nonvertebrate/classification/*genetics
MH  - Classification/methods
MH  - Codon/genetics
MH  - DNA, Mitochondrial/*genetics
MH  - Invertebrates/classification/genetics
MH  - Mammals/classification/genetics
MH  - Reproducibility of Results
MH  - Vertebrates/classification/genetics
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598261030 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):61-76. doi: 10.1080/106351598261030.

PMID- 12064240
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - Large-scale phylogenies and measuring the performance of phylogenetic estimators.
PG  - 43-60
AB  - Performance measures of phylogenetic estimation methods such as accuracy,
      consistency, and power are an attempt at summarizing an ensemble of a given
      estimator's behavior. These summaries characterize an ensemble behavior with a
      single number, leading to a variety of definitions. In particular, the
      relationships between different performance measures such as accuracy and
      consistency or accuracy and error depend on the exact definition of these
      measures. In addition, it is relatively common to use large-sample behavior to
      infer similar behavior for small samples. In fact, large-sample results such as
      the claimed asymptotic efficiency of the maximum-likelihood estimator are often
      uninformative for small samples. Conversely, small-sample behavior using
      simulations is sometimes used to imply large-sample behavior such as consistency.
      However, such extrapolation is often difficult. How the performance of a
      phylogenetic estimator scales with the addition of taxa must be qualified with
      respect to whether the whole tree is being estimated or a fixed subset of taxa is
      being estimated. It must also be qualified with respect to how tree models are
      sampled. Over the ensemble of all possible trees of a given size, the performance
      of the estimators for the whole tree estimate suffers when the tree size becomes 
      larger. However, under certain models of cladogenesis, the estimate can improve
      with the addition of taxa. In fact, at all numbers of taxa there are subsets of
      tree models that are easier to estimate than others. This suggests that with
      judicious addition or subtraction of taxa we can move from tree models that are
      more difficult to estimate at one number of taxa to those that are easier to
      estimate at another number of taxa.
FAU - Kim, J
AU  - Kim J
AD  - Department of Biology, Yale University, New Haven, Connecticut 06511, USA.
      junhyong_kim@quickmail.yale.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Analysis of Variance
MH  - Classification/methods
MH  - MEDLINE
MH  - Observer Variation
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598261021 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):43-60. doi: 10.1080/106351598261021.

PMID- 12064239
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - Inferring complex phylogenies using parsimony: an empirical approach using three 
      large DNA data sets for angiosperms.
PG  - 32-42
AB  - To explore the feasibility of parsimony analysis for large data sets, we
      conducted heuristic parsimony searches and bootstrap analyses on separate and
      combined DNA data sets for 190 angiosperms and three outgroups. Separate data
      sets of 18S rDNA (1,855 bp), rbcL (1,428 bp), and atpB (1,450 bp) sequences were 
      combined into a single matrix 4,733 bp in length. Analyses of the combined data
      set show great improvements in computer run times compared to those of the
      separate data sets and of the data sets combined in pairs. Six searches of the
      18S rDNA + rbcL + atpB data set were conducted; in all cases TBR branch swapping 
      was completed, generally within a few days. In contrast, TBR branch swapping was 
      not completed for any of the three separate data sets, or for the pairwise
      combined data sets. These results illustrate that it is possible to conduct a
      thorough search of tree space with large data sets, given sufficient signal. In
      this case, and probably most others, sufficient signal for a large number of taxa
      can only be obtained by combining data sets. The combined data sets also have
      higher internal support for clades than the separate data sets, and more clades
      receive bootstrap support of &gt; or = 50% in the combined analysis than in analyses
      of the separate data sets. These data suggest that one solution to the
      computational and analytical dilemmas posed by large data sets is the addition of
      nucleotides, as well as taxa.
FAU - Soltis, D E
AU  - Soltis DE
AD  - Department of Botany, Washington State University, Pullman, Washington
      99164-4238, USA. dsoltis@mail.wsu.edu
FAU - Soltis, P S
AU  - Soltis PS
FAU - Mort, M E
AU  - Mort ME
FAU - Chase, M W
AU  - Chase MW
FAU - Savolainen, V
AU  - Savolainen V
FAU - Hoot, S B
AU  - Hoot SB
FAU - Morton, C M
AU  - Morton CM
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 18S)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Classification/methods
MH  - DNA/*genetics
MH  - DNA, Ribosomal/genetics
MH  - Magnoliopsida/classification/*genetics
MH  - *Phylogeny
MH  - RNA, Ribosomal, 18S/genetics
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598261012 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):32-42. doi: 10.1080/106351598261012.

PMID- 12064238
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - Taxonomic sampling, phylogenetic accuracy, and investigator bias.
PG  - 3-8
FAU - Hillis, D M
AU  - Hillis DM
AD  - Department of Zoology, Institute of Cellular and Molecular Biology, University of
      Texas, Austin, Texas 78712, USA. hillis@phylo.zo.utexas.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Genetic Variation
MH  - Observer Variation
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Selection Bias
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260987 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):3-8. doi: 10.1080/106351598260987.

PMID- 12064237
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - Sensitivity of phylogeny estimation to taxonomic sampling.
PG  - 18-31
AB  - Recent studies have shown that addition or deletion of taxa from a data matrix
      can change the estimate of phylogeny. I used 29 data sets from the literature to 
      examine the effect of taxon sampling on phylogeny estimation within data sets. I 
      then used multiple regression to assess the effect of number of taxa, number of
      characters, homoplasy, strength of support, and tree symmetry on the sensitivity 
      of data sets to taxonomic sampling. Sensitivity to sampling was measured by
      mapping characters from a matrix of culled taxa onto optimal trees for that
      reduced matrix and onto the pruned optimal tree for the entire matrix, then
      comparing the length of the reduced tree to the length of the pruned complete
      tree. Within-data-set patterns can be described by a second-order equation
      relating fraction of taxa sampled to sensitivity to sampling. Multiple regression
      analyses found number of taxa to be a significant predictor of sensitivity to
      sampling; retention index, number of informative characters, total support index,
      and tree symmetry were nonsignificant predictors. I derived a predictive
      regression equation relating fraction of taxa sampled and number of taxa
      potentially sampled to sensitivity to taxonomic sampling and calculated values
      for this equation within the bounds of the variables examined. The length
      difference between the complete tree and a subsampled tree was generally small
      (average difference of 0-2.9 steps), indicating that subsampling taxa is probably
      not an important problem for most phylogenetic analyses using up to 20 taxa.
FAU - Poe, S
AU  - Poe S
AD  - Department of Zoology, Texas Memorial Museum, University of Texas, Austin, Texas 
      78712-1064, USA. stevepoe@mail.utexas.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Invertebrates/classification
MH  - Magnoliopsida/classification
MH  - Mammals/classification
MH  - *Phylogeny
MH  - Regression Analysis
MH  - Reproducibility of Results
MH  - Selection Bias
MH  - Vertebrates/classification
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598261003 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):18-31. doi: 10.1080/106351598261003.

PMID- 12064236
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - Partial warps, phylogeny, and ontogeny: a comment on Fink and Zelditch (1995).
PG  - 168-73
FAU - Adams, D C
AU  - Adams DC
AD  - Department of Ecology and Evolution, State University of New York, Stony Brook,
      New York 11794-5245, USA. dca2879@life.bio.sunysb.edu
FAU - Rosenberg, M S
AU  - Rosenberg MS
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Magnoliopsida/classification
MH  - Models, Biological
MH  - *Morphogenesis
MH  - *Phylogeny
MH  - Trees/classification
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598261111 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):168-73. doi: 10.1080/106351598261111.

PMID- 12064235
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - On applications of geometric morphometrics to studies of ontogeny and phylogeny.
PG  - 147-58; discussion 159-67
FAU - Rohlf, F J
AU  - Rohlf FJ
AD  - Department of Ecology and Evolution, State University of New York, Stony Brook,
      New York 11794-5245, USA. rohlf@life.bio.sunysb.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Models, Biological
MH  - Models, Theoretical
MH  - *Morphogenesis
MH  - Multivariate Analysis
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598261094 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):147-58; discussion 159-67. doi:
      10.1080/106351598261094.

PMID- 12064234
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - Lagomorphs misplaced by more characters and fewer taxa.
PG  - 138-46
FAU - Halanych, K M
AU  - Halanych KM
AD  - Department of Zoology and Entomology, University of Pretoria, Pretoria, South
      Africa. halanych@boris.rutgers.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Fibronectins)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Amino Acid Sequence
MH  - Animals
MH  - Artiodactyla/classification/genetics
MH  - Classification/methods
MH  - DNA/chemistry/genetics
MH  - Fibronectins/chemistry/*genetics
MH  - Mammals/*classification/genetics
MH  - Molecular Sequence Data
MH  - Rabbits
MH  - Sequence Alignment
MH  - Sequence Homology, Amino Acid
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598261085 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):138-46. doi: 10.1080/106351598261085.

PMID- 12064233
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - Long-branch distraction and the Strepsiptera.
PG  - 134-8
FAU - Whiting, M F
AU  - Whiting MF
AD  - Department of Zoology, M. L. Bean Life Science Museum, 574 Widtsoe Building,
      Brigham Young University, Provo, Utah 84602, USA. Michael_Whiting@byu.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/methods
MH  - Insecta/*classification
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598261076 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):134-8. doi: 10.1080/106351598261076.

PMID- 12064232
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 1
DP  - 1998 Mar
TI  - On the best evolutionary rate for phylogenetic analysis.
PG  - 125-33
AB  - The effect of the evolutionary rate of a gene on the accuracy of phylogeny
      reconstruction was examined by computer stimulation. The evolutionary rate is
      measured by the tree length, that is, the expected total number of nucleotide
      substitutions per site on the phylogeny. DNA sequence data were simulated using
      both fixed trees with specified branch lengths and random trees with branch
      lengths generated from a model of cladogenesis. The parsimony and likelihood
      methods were used for phylogeny reconstruction, and the proportion of correctly
      recovered branch partitions by each method was estimated. Phylogenetic methods
      including parsimony appear quite tolerant of multiple substitutions at the same
      site. The optimum levels of sequence divergence were even higher than upper
      limits previously suggested for saturation of substitutions, indicating that the 
      problem of saturation may have been exaggerated. Instead, the lack of information
      at low levels of divergence should be seriously considered in evaluation of a
      gene's phylogenetic utility, especially when the gene sequence is short. The
      performance of parsimony, relative to that of likelihood, does not necessarily
      decrease with the increase of the evolutionary rate.
FAU - Yang, Z
AU  - Yang Z
AD  - Department of Integrative Biology, University of California, Berkeley, California
      94720-3140, USA.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Complementary)
RN  - 0 (RNA, Ribosomal)
SB  - IM
MH  - *Biological Evolution
MH  - DNA, Complementary/genetics
MH  - Models, Biological
MH  - Phylogeny
MH  - RNA, Ribosomal/genetics
MH  - Reproducibility of Results
MH  - Time
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598261067 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Mar;47(1):125-33. doi: 10.1080/106351598261067.

PMID- 12064231
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 2
DP  - 1998 Jun
TI  - Influence of alignment on the mtDNA phylogeny of Cetacea: questionable support
      for a Mysticeti/Physeteroidea clade.
PG  - 336-44
FAU - Cerchio, S
AU  - Cerchio S
AD  - Department of Biology and Museum of Zoology, University of Michigan, Ann Arbor,
      Michigan 48109, USA. sal@umich.edu
FAU - Tucker, P
AU  - Tucker P
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Artiodactyla/classification/genetics
MH  - Cetacea/classification/*genetics
MH  - DNA, Mitochondrial/*genetics
MH  - Sequence Alignment
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260941 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Jun;47(2):336-44. doi: 10.1080/106351598260941.

PMID- 12064230
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 2
DP  - 1998 Jun
TI  - Phylogeny of frogs of the Physalaemus pustulosus species group, with an
      examination of data incongruence.
PG  - 311-35
AB  - Characters derived from advertisement calls, morphology, allozymes, and the
      sequences of the small subunit of the mitochondrial ribosomal gene (12S) and the 
      cytochrome oxidase I (COI) mitochondrial gene were used to estimate the phylogeny
      of frogs of the Physalaemus pustulosus group (Leptodactylidae). The combinability
      of these data partitions was assessed in several ways: measures of phylogenetic
      signal, character support for trees, congruence of tree topologies, compatibility
      of data partitions with suboptimal trees, and homogeneity of data partitions.
      Combined parsimony analysis of all data equally weighted yielded the same tree as
      the 12S partition analyzed under parsimony and maximum likelihood. The COI,
      allozyme, and morphology partitions were generally congruent and compatible with 
      the tree derived from combined data. The call data were significantly different
      from all other partitions, whether considered in terms of tree topology alone,
      partition homogeneity, or compatibility of data with trees derived from other
      partitions. The lack of effect of the call data on the topology of the combined
      tree is probably due to the small number of call characters. The general
      incongruence of the call data with other data partitions is consistent with the
      idea that the advertisement calls of this group of frogs are under strong sexual 
      selection.
FAU - Cannatella, D C
AU  - Cannatella DC
AD  - Department of Zoology, University of Texas, Texas Memorial Museum, Austin, Texas,
      USA.
FAU - Hillis, D M
AU  - Hillis DM
FAU - Chippindale, P T
AU  - Chippindale PT
FAU - Weigt, L
AU  - Weigt L
FAU - Rand, A S
AU  - Rand AS
FAU - Ryan, M J
AU  - Ryan MJ
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (DNA, Ribosomal)
RN  - 0 (Isoenzymes)
RN  - 0 (RNA, Ribosomal)
RN  - 0 (RNA, ribosomal, 12S)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bufonidae/classification/*genetics
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - DNA, Ribosomal/genetics
MH  - Electron Transport Complex IV/chemistry/*genetics
MH  - Isoenzymes/chemistry/genetics
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - RNA, Ribosomal/genetics
MH  - Reproducibility of Results
MH  - Sequence Alignment
MH  - Sequence Homology, Nucleic Acid
MH  - Species Specificity
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260932 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Jun;47(2):311-35. doi: 10.1080/106351598260932.

PMID- 12064229
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 2
DP  - 1998 Jun
TI  - Data decisiveness, data quality, and incongruence in phylogenetic analysis: an
      example from the monocotyledons using mitochondrial atp A sequences.
PG  - 282-310
AB  - We examined three parallel data sets with respect to qualities relevant to
      phylogenetic analysis of 20 exemplar monocotyledons and related dicotyledons. The
      three data sets represent restriction-site variation in the inverted repeat
      region of the chloroplast genome, and nucleotide sequence variation in the
      chloroplast-encoded gene rbcL and in the mitochondrion-encoded gene atpA, the
      latter of which encodes the alpha-subunit of mitochondrial ATP synthase. The
      plant mitochondrial genome has been little used in plant systematics, in part
      because nucleotide sequence evolution in enzyme-encoding genes of this genome is 
      relatively slow. The three data sets were examined in separate and combined
      analyses, with a focus on patterns of congruence, homoplasy, and data
      decisiveness. Data decisiveness (described by P. Goloboff) is a measure of
      robustness of support for most parsimonious trees by a data set in terms of the
      degree to which those trees are shorter than the average length of all possible
      trees. Because indecisive data sets require relatively fewer additional steps
      than decisive ones to be optimized on nonparsimonious trees, they will have a
      lesser tendency to be incongruent with other data sets. One consequence of this
      relationship between decisiveness and character incongruence is that if
      incongruence is used as a criterion of noncombinability, decisive data sets,
      which provide robust support for relationships, are more likely to be assessed as
      noncombinable with other data sets than are indecisive data sets, which provide
      weak support for relationships. For the sampling of taxa in this study, the atpA 
      data set has about half as many cladistically informative nucleotides as the rbcL
      data set per site examined, and is less homoplastic and more decisive. The rbcL
      data set, which is the least decisive of the three, exhibits the lowest levels of
      character incongruence. Whatever the molecular evolutionary cause of this
      phenomenon, it seems likely that the poorer performance of rbcL than atpA, in
      terms of data decisiveness, is due to both its higher overall level of homoplasy 
      and the fact that it is performing especially poorly at nonsynonymous sites.
FAU - Davis, J I
AU  - Davis JI
AD  - L. H. Bailey Hortorium, Cornell University, Ithaca, New York 14853, USA.
FAU - Simmons, M P
AU  - Simmons MP
FAU - Stevenson, D W
AU  - Stevenson DW
FAU - Wendel, J F
AU  - Wendel JF
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - EC 3.6.1.- (Adenosine Triphosphatases)
SB  - IM
MH  - Adenosine Triphosphatases/chemistry/*genetics
MH  - Base Sequence
MH  - DNA Primers
MH  - Magnoliopsida/classification/*genetics
MH  - Mitochondria/*genetics
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Restriction Mapping
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260923 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Jun;47(2):282-310. doi: 10.1080/106351598260923.

PMID- 12064228
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 2
DP  - 1998 Jun
TI  - Phylogenetic analysis and intraspecific variation: performance of parsimony,
      likelihood, and distance methods.
PG  - 228-53
AB  - Intraspecific variation is abundant in all types of systematic characters but is 
      rarely addressed in simulation studies of phylogenetic method performance. We
      compared the accuracy of 15 phylogenetic methods using simulations to (1)
      determine the most accurate method(s) for analyzing polymorphic data (under
      simplified conditions) and (2) test if generalizations about the performance of
      phylogenetic methods based on previous simulations of fixed (nonpolymorphic)
      characters are robust to a very different evolutionary model that explicitly
      includes intraspecific variation. Simulated data sets consisted of allele
      frequencies that evolved by genetic drift. The phylogenetic methods included
      eight parsimony coding methods, continuous maximum likelihood, and three distance
      methods (UPGMA, neighbor joining, and Fitch-Margoliash) applied to two genetic
      distance measures (Nei's and the modified Cavalli-Sforza and Edwards chord
      distance). Two sets of simulations were performed. The first examined the effects
      of different branch lengths, sample sizes (individuals sampled per species),
      numbers of characters, and numbers of alleles per locus in the eight-taxon case. 
      The second examined more extensively the effects of branch length in the
      four-taxon, two-allele case. Overall, the most accurate methods were likelihood, 
      the additive distance methods (neighbor joining and Fitch-Margoliash), and the
      frequency parsimony method. Despite the use of a very different evolutionary
      model in the present article, many of the results are similar to those from
      simulations of fixed characters. Similarities include the presence of the
      "Felsenstein zone," where methods often fail, which suggests that long-branch
      attraction may occur among closely related species through genetic drift.
      Differences between the results of fixed and polymorphic data simulations include
      the following: (1) UPGMA is as accurate or more accurate than nonfrequency
      parsimony methods across nearly all combinations of branch lengths, and (2)
      likelihood and the additive distance methods are not positively misled under any 
      combination of branch lengths tested (even when the assumptions of the methods
      are violated and few characters are sampled). We found that sample size is an
      important determinant of accuracy and affects the relative success of methods
      (i.e., distance and likelihood methods outperform parsimony at small sample
      sizes). Attempts to generalize about the behavior of phylogenetic methods should 
      consider the extreme examples offered by fixed-mutation models of DNA sequence
      data and genetic-drift models of allele frequencies.
FAU - Wiens, J J
AU  - Wiens JJ
AD  - Section of Amphibians and Reptiles, Carnegie Museum of Natural History,
      Pittsburgh, Pennsylvania 15213-4080, USA. wiensj@clpgh.org
FAU - Servedio, M R
AU  - Servedio MR
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - Decision Trees
MH  - *Genetic Variation
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Polymorphism, Genetic
MH  - Species Specificity
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260897 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Jun;47(2):228-53. doi: 10.1080/106351598260897.

PMID- 12064227
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 2
DP  - 1998 Jun
TI  - Genetic distances and ordination: the land snail Helix aspersa in north Africa as
      a test case.
PG  - 208-27
AB  - We examined the efficiencies of ordination methods in the treatment of gene
      frequency data at intraspecific level, using metric and nonmetric distance
      measures (Nei's and Rogers' genetic distances, chi 2 distance). We assessed
      initial processes responsible for the geographical distribution of the
      Mediterranean land snail Helix aspersa. Seventeen enzyme loci from 30 North
      African snail populations were considered in the present analysis. Five
      combinations of distance/multivariate analysis were compared: correspondence
      analysis (CA), nonmetric multidimensional scaling (NMDS) on Nei's, Rogers', and
      chi 2 distances, and principal coordinates analysis on Rogers' distances.
      Configuration of the objects resulting from ordination was projected onto
      three-dimensional graphics with the minimum spanning tree or the relative
      neighborhood graph superimposed. Pre- and postordination or clustering distance
      matrices were compared by means of correlation methods. As expected, all
      combinations led to a clear west versus east pattern of variation. However, the
      intraregional relationships and degree of connectivity between pairs of
      operational taxonomic units were not necessarily constant from one method to
      another. Ordination methods when applied with Nei's and Rogers' distances
      provided the best fit, with original distances (r = 0.98) compared with UPGMA
      clustering (r approximately 0.75). The Nei/NMDS combination seems to be a good
      compromise (distortion index dt = 10%) between Rogers/NMDS, which produces a more
      confusing pattern of differentiation (dt = 24%), and chi 2/CA, which tends to
      distort large distances (dt = 31%). NMDS obviously provides a powerful method to 
      summarize relationships between populations, when neither hierarchical structure 
      nor phylogenetic inference are required. These findings led the discussion on the
      good performance of NMDS, the appropriate distances to be used, and the potential
      application of this method to other types of allelic data (such as microsatellite
      loci) or data on nucleotide sequences of genes.
FAU - Guiller, A
AU  - Guiller A
AD  - Laboratoire de Parasitologie Pharmaceutique (CNRS UMR 6553), Faculte des Sciences
      Pharmaceutiques et Biologiques, 35043 Rennes, France.
      annie.guiller@univ-rennes1.fr
FAU - Bellido, A
AU  - Bellido A
FAU - Madec, L
AU  - Madec L
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Africa, Northern
MH  - Animals
MH  - Gene Frequency
MH  - Geography
MH  - Helix (Snails)/classification/*genetics
MH  - Mediterranean Region
MH  - Models, Biological
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260888 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Jun;47(2):208-27. doi: 10.1080/106351598260888.

PMID- 12064226
OWN - NLM
STAT- MEDLINE
DCOM- 20020724
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 47
IP  - 2
DP  - 1998 Jun
TI  - Biogeography and floral evolution of baobabs (Adansonia, Bombacaceae) as inferred
      from multiple data sets.
PG  - 181-207
AB  - The phylogeny of baobab trees was analyzed using four data sets: chloroplast DNA 
      restriction sites, sequences of the chloroplast rpl16 intron, sequences of the
      internal transcribed spacer (ITS) region of nuclear ribosomal DNA, and
      morphology. We sampled each of the eight species of Adansonia plus three outgroup
      taxa from tribe Adansonieae. These data were analyzed singly and in combination
      using parsimony. ITS and morphology provided the greatest resolution and were
      largely concordant. The two chloroplast data sets showed concordance with one
      another but showed significant conflict with ITS and morphology. A possible
      explanation for the conflict is genealogical discordance within the Malagasy
      Longitubae, perhaps due to introgression events. A maximum-likelihood analysis of
      branching times shows that the dispersal between Africa and Australia occurred
      well after the fragmentation of Gondwana and therefore involved overwater
      dispersal. The phylogeny does not permit unambiguous reconstruction of floral
      evolution but suggests the plausible hypothesis that hawkmoth pollination was
      ancestral in Adansonia and that there were two parallel switches to pollination
      by mammals in the genus.
FAU - Baum, D A
AU  - Baum DA
AD  - Department of Organismic and Evolutionary Biology, Harvard University Herbaria,
      Cambridge, Massachusetts 02138, USA. dbaum@oeb.harvard.edu
FAU - Small, R L
AU  - Small RL
FAU - Wendel, J F
AU  - Wendel JF
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Chloroplast)
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - *Biological Evolution
MH  - DNA Primers
MH  - DNA, Chloroplast/genetics
MH  - DNA, Ribosomal/genetics
MH  - Introns
MH  - Magnoliopsida/*classification/genetics
MH  - Mammals
MH  - Molecular Sequence Data
MH  - Moths
MH  - *Phylogeny
MH  - Polymerase Chain Reaction
MH  - Reproduction
MH  - Sequence Alignment
MH  - Sequence Homology, Nucleic Acid
MH  - Time
MH  - Trees/*classification/genetics
EDAT- 2002/06/18 10:00
MHDA- 2002/07/26 10:01
CRDT- 2002/06/18 10:00
PHST- 2002/06/18 10:00 [pubmed]
PHST- 2002/07/26 10:01 [medline]
PHST- 2002/06/18 10:00 [entrez]
AID - 10.1080/106351598260879 [doi]
PST - ppublish
SO  - Syst Biol. 1998 Jun;47(2):181-207. doi: 10.1080/106351598260879.

PMID- 12028738
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 2
DP  - 2002 Apr
TI  - Fragmentary taxa, missing data, and ambiguity: mistaken assumptions and
      conclusions.
PG  - 369-81
FAU - Kearney, Maureen
AU  - Kearney M
AD  - Field Museum of Natural History, Department of Zoology, Division of Amphibians
      and Reptiles, Chicago, Illinois 60605, USA. mkearney@fmnh.org
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Classification
MH  - Computational Biology/*methods
MH  - Phylogeny
MH  - Statistics as Topic
EDAT- 2002/05/25 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/05/25 10:00
PHST- 2002/05/25 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/05/25 10:00 [entrez]
AID - 10.1080/10635150252899824 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Apr;51(2):369-81. doi: 10.1080/10635150252899824.

PMID- 12028737
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 2
DP  - 2002 Apr
TI  - Tetrapod phylogeny, amphibian origins, and the definition of the name tetrapoda.
PG  - 364-9
FAU - Laurin, Michel
AU  - Laurin M
AD  - Equipe "Formations squelettiques" UMR CNRS 8570, Case 7077, Universite Paris 7,
      75005 Paris, France. laurin@ccr.jussieu.fr
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2002 Oct;51(5):822-7. PMID: 12396594
MH  - Algorithms
MH  - Amphibians/*classification/*genetics
MH  - Animals
MH  - Computational Biology/methods
MH  - Evolution, Molecular
MH  - Phylogeny
MH  - Software
MH  - Terminology as Topic
EDAT- 2002/05/25 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/05/25 10:00
PHST- 2002/05/25 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/05/25 10:00 [entrez]
AID - 10.1080/10635150252899815 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Apr;51(2):364-9. doi: 10.1080/10635150252899815.

PMID- 12028736
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 2
DP  - 2002 Apr
TI  - Mapping spatial pattern in biodiversity for regional conservation planning: where
      to from here?
PG  - 331-63
AB  - Vast gaps in available information on the spatial distribution of biodiversity
      pose a major challenge for regional conservation planning in many parts of the
      world. This problem is often addressed by basing such planning on various
      biodiversity surrogates. In some situations, distributional data for selected
      taxa may be used as surrogates for biodiversity as a whole. However, this
      approach is less effective in data-poor regions, where there may be little choice
      but to base conservation planning on some form of remote environmental mapping,
      derived, for example, from interpretation of satellite imagery or from numerical 
      classification of abiotic environmental layers. Although this alternative
      approach confers obvious benefits in terms of cost-effectiveness and rapidity of 
      application, problems may arise if congruence is poor between mapped land-classes
      and actual biological distributions. I propose three strategies for making more
      effective use of available biological data and knowledge to alleviate such
      problems by (1) more closely integrating biological and environmental data
      through predictive modeling, with increased emphasis on modeling collective
      properties of biodiversity rather than individual entities; (2) making more
      rigorous use of remotely mapped surrogates in conservation planning by
      incorporating knowledge of heterogeneity within land-classes, and of varying
      levels of distinctiveness between classes, into measures of conservation priority
      and achievement; and (3) using relatively data-rich regions as test-beds for
      evaluating the performance of surrogates that can be readily applied across
      data-poor regions.
FAU - Ferrier, Simon
AU  - Ferrier S
AD  - New South Wales National Parks and Wildlife Service, PO Box 402, Armidale, New
      South Wales 2350, Australia. simon.ferrier@npws.nsw.gov.au
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Computational Biology/methods
MH  - Ecology
MH  - *Ecosystem
MH  - Environment
MH  - Plants/genetics
EDAT- 2002/05/25 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/05/25 10:00
PHST- 2002/05/25 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/05/25 10:00 [entrez]
AID - 10.1080/10635150252899806 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Apr;51(2):331-63. doi: 10.1080/10635150252899806.

PMID- 12028735
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20020524
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 2
DP  - 2002 Apr
TI  - Integrating biosystematic data into conservation planning: perspectives from
      southern Africa's Succulent Karoo.
PG  - 317-30
AB  - In this paper we explore the role that biosystematists can play in conservation
      planning. Conservation planning concerns the location and design of reserves that
      both represent the biodiversity of a region and enable the persistence of that
      biodiversity by maintaining key ecological and evolutionary processes. For
      conservation planning to be effective, quantitative targets are needed for the
      spatial components of a region that reflect evolutionary processes. Using
      examples from southern Africa's Succulent Karoo, we demonstrate how spatially
      explicit data on morphological variation within taxa provide essential
      information for conservation planning in that such variation represents an
      important surrogate for the spatial component of lineage diversification. We also
      provide an example of how the spatial components of evolutionary processes can be
      identified and targeted for conservation action. Key to this understanding are
      the recognition and description of taxonomic units at all spatial scales. Without
      the recognition of subspecific variation, it is difficult to formulate
      evolutionary hypotheses, let alone set quantitative targets for the conservation 
      of this variation. Given the escalating threats to biodiversity, and the
      importance of planning for persistence by incorporating ecological and
      evolutionary processes into conservation plans, it is essential that systematists
      develop hypotheses on the spatial surrogates for these processes for a wide range
      of lineages. The important questions for systematists to be asking are (1) how is
      variation distributed in the landscape, and (2) how did it come about?
      Conservation planners too need to highlight these spatial components for
      conservation action.
FAU - Desmet, P G
AU  - Desmet PG
AD  - Institute for Plant Conservation, University of Cape Town, Rondebosch, 7701,
      South Africa. Desmet@borzoo.uct.ac.za
FAU - Cowling, R M
AU  - Cowling RM
FAU - Ellis, A G
AU  - Ellis AG
FAU - Pressey, R L
AU  - Pressey RL
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Africa
MH  - Classification
MH  - Computational Biology/*methods
MH  - Ecology
MH  - *Ecosystem
MH  - Evolution, Molecular
MH  - Plants/genetics
MH  - Software
EDAT- 2002/05/25 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/05/25 10:00
PHST- 2002/05/25 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/05/25 10:00 [entrez]
AID - 10.1080/10635150252899798 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Apr;51(2):317-30. doi: 10.1080/10635150252899798.

PMID- 12028734
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 2
DP  - 2002 Apr
TI  - Systematic data in biodiversity studies: use it or lose it.
PG  - 303-16
AB  - Systematic data in the form of collections data are useful in biodiversity
      studies in many ways, most importantly because they serve as the only direct
      evidence of species distributions. However, collecting bias has been demonstrated
      for most areas of the world and has led some to propose methods that circumvent
      the need for collections data. New methods that model collections data in
      combination with abiotic data and predict potential total species distribution
      are examined using 25,111 records representing 5,123 species of plants and
      animals from Guyana; some methods use the reduced number of 320 species. These
      modeled species distributions are evaluated and potential high-priority
      biodiversity sites are selected based on the concept of irreplaceability, a
      measure of uniqueness. The major impediments to using collections data are the
      lack of data that are available in a useful format and the reluctance of most
      systematists to become involved in biodiversity and conservation research.
FAU - Funk, V A
AU  - Funk VA
AD  - U.S. National Herbarium, Smithsonian Institution, Washington, D.C. 20560-0166,
      USA. funkv@nmnh.si.edu
FAU - Richardson, K S
AU  - Richardson KS
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computational Biology/*methods
MH  - Ecology
MH  - *Ecosystem
MH  - Environment
MH  - *Evolution, Molecular
MH  - Guyana
MH  - Models, Statistical
MH  - Plants/genetics
MH  - Software
MH  - Species Specificity
MH  - Statistics as Topic/methods
MH  - Temperature
EDAT- 2002/05/25 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/05/25 10:00
PHST- 2002/05/25 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/05/25 10:00 [entrez]
AID - 10.1080/10635150252899789 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Apr;51(2):303-16. doi: 10.1080/10635150252899789.

PMID- 12028733
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 2
DP  - 2002 Apr
TI  - Patterns of endangerment in the hawaiian flora.
PG  - 276-302
AB  - The Hawaiian flora, because of its great isolation, high levels of endemism,
      known lineages, and high rates of endangerment, offers unique opportunities to
      explore patterns of endangerment related to phylogeny, ecological and life
      history traits, and geographic patterns. Nine percent of the native flora of 1159
      taxa are already extinct, and 52.5% are at risk (extinct, endangered, vulnerable,
      or rare). Risk is strongly associated with limited geographic distribution at
      several scales: endemic taxa (native only to the Hawaiian Islands) are at far
      greater risk than indigenous taxa (with both Hawaiian and extra-Hawaiian ranges);
      single-island endemics are more at risk than multi-island endemics; small islands
      have the highest proportion of endemic taxa at risk; and endemics with more
      limited habitat distributions (elevation, community type) are more at risk.
      Historic population density is a strong predictor of risk, and taxa with low
      historic population densities are at greatest risk with rapid anthropogenic
      changes. Among the major islands, Maui Nui has the highest percent of taxa that
      are extinct. Kaua'i has the lowest percent of extinct taxa and the highest
      proportion of single-island endemic taxa that are rare. Endemic taxa at risk are 
      associated with distributions in shrublands, forests, bogs, and cliff habitats.
      Endemic taxa with distributions in low elevation dry habitats have the highest
      proportion of taxa at risk, but the greatest absolute numbers of taxa at risk
      have distributions in mesic lowland and montane forests, and in wet montane
      forests. The life history patterns associated with risk are complicated, and
      inclusion of the effects of evolutionary relationships (lineages) changes some of
      these patterns. Species level analyses without respect to lineage shows risk
      associated with monomorphic (hermaphroditic) breeding systems and bird
      pollination because of the large number of hermaphroditic, bird-pollinated
      species in the Campanulaceae. Analyses incorporating the effect of lineage
      greatly reduce the impact of large lineages and result in an association of risk 
      with insect pollination, and no effect of breeding system. There is no
      association of lineage size and the percent of taxa at risk within the lineage;
      endemic taxa from lineages with large radiations are at no greater risk than
      endemic single-taxon lineages. The percentages of taxa at risk at the family
      level in the Hawaiian Islands and worldwide (excluding Hawaiian taxa) are
      positively correlated, although flowering plant families in the Hawaiian Islands 
      have a much greater proportion of taxa at risk. Some of the approaches described 
      here may be useful to predict geographical and biological patterns of
      endangerment in island and island-like ecosystems under increasing pressures of
      endangerment and extinction.
FAU - Sakai, Ann K
AU  - Sakai AK
AD  - Department of Ecology and Evolutionary Biology, University of California-Irvine, 
      Irvine, California 92697, USA. aksakai@uci.edu
FAU - Wagner, Warren L
AU  - Wagner WL
FAU - Mehrhoff, Loyal A
AU  - Mehrhoff LA
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Cell Lineage
MH  - Classification
MH  - Ecology
MH  - *Ecosystem
MH  - *Evolution, Molecular
MH  - Genes, Plant/*genetics
MH  - Genetic Variation
MH  - Hawaii
MH  - Phylogeny
MH  - Risk
EDAT- 2002/05/25 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/05/25 10:00
PHST- 2002/05/25 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/05/25 10:00 [entrez]
AID - 10.1080/10635150252899770 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Apr;51(2):276-302. doi: 10.1080/10635150252899770.

PMID- 12028732
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 2
DP  - 2002 Apr
TI  - Importance of hybridization between indigenous and nonindigenous freshwater
      species: an overlooked threat to North American biodiversity.
PG  - 255-75
AB  - Biodiversity of North American freshwaters is among the greatest in the world.
      However, due to extensive habitat degradation, pollution, and introductions of
      nonindigenous species, this biodiversity is also among the most endangered.
      Unlike habitat degradation and pollution, nonindigenous species represent a
      permanent loss of biodiversity because their removal or control is often
      impossible. Most species introduced into nonnative North American ranges,
      however, are not from Eurasia but have been introduced from geographically
      isolated regions within North America. Although the ecological effects of
      introduced species have been widely documented, the effects of hybridization,
      especially between closely related species, represents an equally serious
      mechanism of extinction but is much less studied. Identification of which species
      are likely to hybridize after contact is of critical importance to prevent the
      further loss of native species. Molecular phylogenetics serves as a powerful tool
      to identify freshwater species at risk of introgression, if we can assume that
      genetic distance is a good predictor of the potential for hybridization. Although
      not a thorough review of all cases of hybridization, this article documents the
      extent and effects of hybridization in fishes, crayfishes, mussels, and other
      invertebrates in light of the currently accepted phylogenetic relationships. We
      suggest this approach may be the first step in addressing the potential threat of
      hybridization between many of the closely related species in North American fresh
      waters.
FAU - Perry, William L
AU  - Perry WL
AD  - Behavior, Ecology, Evolution, and Systematic Section, Department of Biological
      Sciences, Illinois State University, Normal, Illinois 61790-4120, USA.
      wlperry@ilstu.edu.
FAU - Lodge, David M
AU  - Lodge DM
FAU - Feder, Jeffrey L
AU  - Feder JL
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Crosses, Genetic
MH  - Crustacea/classification
MH  - *Ecosystem
MH  - Evolution, Molecular
MH  - Female
MH  - Fishes/*classification/*physiology
MH  - Male
MH  - North America
MH  - Phylogeny
EDAT- 2002/05/25 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/05/25 10:00
PHST- 2002/05/25 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/05/25 10:00 [entrez]
AID - 10.1080/10635150252899761 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Apr;51(2):255-75. doi: 10.1080/10635150252899761.

PMID- 12028731
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 2
DP  - 2002 Apr
TI  - Strategies to protect biological diversity and the evolutionary processes that
      sustain it.
PG  - 238-54
AB  - Conservation planning has tended to focus more on pattern (representation) than
      process (persistence) and, for the former, has emphasized species and ecosystem
      or community diversity over genetic diversity. Here I consider how best to
      incorporate knowledge of evolutionary processes and the distribution of genetic
      diversity into conservation planning and priority setting for populations within 
      species and for biogeographic areas within regions. Separation of genetic
      diversity into two dimensions, one concerned with adaptive variation and the
      other with neutral divergence caused by isolation, highlights different
      evolutionary processes and suggests alternative strategies for conservation.
      Planning for both species and areas should emphasize protection of historically
      isolated lineages (Evolutionarily Significant Units) because these cannot be
      recovered. By contrast, adaptive features may best be protected by maintaining
      the context for selection, heterogeneous landscapes, and viable populations,
      rather than protecting specific phenotypes. A useful strategy may be to (1)
      identify areas that are important to represent species and (vicariant) genetic
      diversity and (2) maximize within these areas the protection of contiguous
      environmental gradients across which selection and migration can interact to
      maintain population viability and (adaptive) genetic diversity. These concepts
      are illustrated with recent results from analysis of a rainforest fauna from
      northeast Australia.
FAU - Moritz, Craig
AU  - Moritz C
AD  - Museum of Vertebrate Zoology, University of California, Berkeley, California
      94720, USA. cmoritz@socrates.berkeley.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Ecology
MH  - *Ecosystem
MH  - Genetic Variation
MH  - Models, Genetic
EDAT- 2002/05/25 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/05/25 10:00
PHST- 2002/05/25 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/05/25 10:00 [entrez]
AID - 10.1080/10635150252899752 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Apr;51(2):238-54. doi: 10.1080/10635150252899752.

PMID- 12028730
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 2
DP  - 2002 Apr
TI  - Biodiversity: the interface between systematics and conservation.
PG  - 235-7
FAU - Funk, V A
AU  - Funk VA
AD  - U.S. National Herbarium, Smithsonian Institution, Washington, D.C. 20560-0166,
      USA. funkv@nmnh.si.edu
FAU - Sakai, A K
AU  - Sakai AK
FAU - Richardson, K
AU  - Richardson K
LA  - eng
PT  - Editorial
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - *Computational Biology
MH  - Ecology
MH  - *Ecosystem
EDAT- 2002/05/25 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/05/25 10:00
PHST- 2002/05/25 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/05/25 10:00 [entrez]
AID - 10.1080/10635150252899743 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Apr;51(2):235-7. doi: 10.1080/10635150252899743.

PMID- 12028729
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 2
DP  - 2002 Apr
TI  - A statistical test for host-parasite coevolution.
PG  - 217-34
AB  - A new method, ParaFit, has been developed to test the significance of a global
      hypothesis of coevolution between parasites and their hosts. Individual
      host-parasite association links can also be tested. The test statistics are
      functions of the host and parasite phylogenetic trees and of the set of
      host-parasite association links. Numerical simulations are used to show that the 
      method has correct rate of type I error and good power except under extreme error
      conditions. An application to real data (pocket gophers and chewing lice) is
      presented.
FAU - Legendre, Pierre
AU  - Legendre P
AD  - Departement de sciences biologiques, Universite de Montreal, C.P.6128, succursale
      Centre-ville, Montreal, Quebec, Canada H3C 3J7. Pierre.Legengre@umontreal.ca
FAU - Desdevises, Yves
AU  - Desdevises Y
FAU - Bazin, Eric
AU  - Bazin E
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Computational Biology
MH  - *Evolution, Molecular
MH  - *Host-Parasite Interactions
MH  - Models, Statistical
MH  - Phthiraptera
MH  - Phylogeny
MH  - Reproducibility of Results
MH  - Rodentia
EDAT- 2002/05/25 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/05/25 10:00
PHST- 2002/05/25 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/05/25 10:00 [entrez]
AID - 10.1080/10635150252899734 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Apr;51(2):217-34. doi: 10.1080/10635150252899734.

PMID- 12028728
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 2
DP  - 2002 Apr
TI  - Reconstruction of biogeographic and evolutionary networks using reticulograms.
PG  - 199-216
AB  - A reticulogram is a general network capable of representing a reticulate
      evolutionary structure. It is particularly useful for portraying relationships
      among organisms that may be related in a nonunique way to their common ancestor -
      relationships that cannot be represented by a dendrogram or a phylogenetic tree. 
      We propose a new method for constructing reticulograms that represent a given
      distance matrix. Reticulate evolution applies first to phylogenetic problems; it 
      has been found in nature, for example, in the within-species microevolution of
      eukaryotes and in lateral gene transfer in bacteria. In this paper, we propose a 
      new method for reconstructing reticulation networks and we develop applications
      of the reticulate evolution model to ecological biogeographic, population
      microevolutionary, and hybridization problems. The first example considers a
      spatially constrained reticulogram representing the postglacial dispersal of
      freshwater fishes in the Quebec peninsula; the reticulogram provides a better
      model of postglacial dispersal than does a tree model. The second example depicts
      the morphological similarities among local populations of muskrats in a river
      valley in Belgium; adding supplementary branches to a tree depicting the river
      network leads to a better representation of the morphological distances among
      local populations of muskrats than does a tree structure. A third example
      involves hybrids between plants of the genus Aphelandra.
FAU - Legendre, Pierre
AU  - Legendre P
AD  - Departement de sciences biologiques, Universite de Montreal, C.P.6128, succursale
      Centre-ville, Montreal, Quebec, Canada H3C 3J7. Pierre.Legengre@umontreal.ca
FAU - Makarenkov, Vladimir
AU  - Makarenkov V
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Arvicolinae
MH  - *Computational Biology
MH  - *Evolution, Molecular
MH  - Fishes
MH  - Models, Genetic
MH  - Models, Statistical
MH  - Phylogeny
MH  - Software
EDAT- 2002/05/25 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/05/25 10:00
PHST- 2002/05/25 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/05/25 10:00 [entrez]
AID - 10.1080/10635150252899725 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Apr;51(2):199-216. doi: 10.1080/10635150252899725.

PMID- 11975341
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 4
DP  - 1997 Dec
TI  - Prior agreement: arbitration or arbitrary?
PG  - 765-9
FAU - Siddall, M E
AU  - Siddall ME
AD  - Museum of Zoology, University of Michigan, Ann Arbor, Michigan 48109, USA.
      msiddall@umich.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Protozoan)
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - DNA, Protozoan/genetics
MH  - DNA, Ribosomal/genetics
MH  - Data Interpretation, Statistical
MH  - Eukaryota/classification/genetics
MH  - *Phylogeny
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.4.765 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Dec;46(4):765-9. doi: 10.1093/sysbio/46.4.765.

PMID- 11975340
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 4
DP  - 1997 Dec
TI  - Process partitions, congruence, and the independence of characters: inferring
      relationships among closely related Hawaiian Drosophila from multiple gene
      regions.
PG  - 751-64
FAU - DeSalle, R
AU  - DeSalle R
AD  - Department of Entomology, American Museum of Natural History, Central Park West
      at 79th Street, New York, New York 10024-5192, USA. desalle@amnh.org
FAU - Brower, A V
AU  - Brower AV
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA/genetics/isolation &amp; purification
MH  - DNA, Mitochondrial/genetics
MH  - Data Interpretation, Statistical
MH  - Drosophila/*classification/*genetics
MH  - Genes, Insect
MH  - Hawaii
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Sequence Homology, Nucleic Acid
MH  - Species Specificity
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.4.751 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Dec;46(4):751-64. doi: 10.1093/sysbio/46.4.751.

PMID- 11975339
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 4
DP  - 1997 Dec
TI  - Phylogenetic analyses of DNA and allozyme data suggest that Gonioctena leaf
      beetles (Coleoptera; Chrysomelidae) experienced convergent evolution in their
      history of host-plant family shifts.
PG  - 722-47
AB  - A phylogenetic analysis of the genus Gonioctena (Coleoptera, Chrysomelidae) based
      on allozyme data (17 loci) and mitochondrial DNA sequence data (three gene
      fragments, 1,391 sites) was performed to study the evolutionary history of
      host-plant shifts among these leaf beetles. This chrysomelid genus is
      characteristically associated with a high number of different plant families. The
      diverse molecular data gathered in this study are to a large extent congruent,
      and the analyses provide a well-supported phylogenetic hypothesis to address
      questions about the evolution of host-plant shifts in the genus Gonioctena. The
      most-parsimonious reconstruction of the ancestral host-plant associations, based 
      on the estimated phylogeny, suggests that the Fabaceae was the ancestral
      host-plant family of the genus. Although most of the host-plant shifts (between
      different host species) in Gonioctena have occurred within the same plant family 
      or within the same plant genus, at least eight shifts have occurred between hosts
      belonging to distantly related and chemically dissimilar plant families. In these
      cases, host shifts may have been simply directed toward plant species available
      in the environment. Yet, given that two Gonioctena lineages have independently
      colonized the same three new plant families (Salicaceae, Betulaceae, Rosaceae),
      including four of the same new genera (Salix, Alnus, Prunus, Sorbus), some
      constraints are likely to have limited the different possibilities of
      interfamilial host-plant shifts.
FAU - Mardulyn, P
AU  - Mardulyn P
AD  - Laboratoire de Biologie Animale et Cellulaire, CP 160/12, Free University of
      Brussels, 50, av. F. D. Roosevelt, B-1050 Brussels, Belgium.
      mardulyn@comp.uark.edu
FAU - Milinkovitch, M C
AU  - Milinkovitch MC
FAU - Pasteels, J M
AU  - Pasteels JM
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Coleoptera/classification/*enzymology/*genetics
MH  - DNA, Mitochondrial/*genetics
MH  - Evolution, Molecular
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Plants
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.4.722 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Dec;46(4):722-47. doi: 10.1093/sysbio/46.4.722.

PMID- 11975338
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 4
DP  - 1997 Dec
TI  - Evolution and phylogeny of the Diptera: a molecular phylogenetic analysis using
      28S rDNA sequences.
PG  - 674-98
AB  - Portions of the large ribosomal subunit RNA gene (28S rDNA) encompassing the D1
      and the D7 region were obtained from 16 dipteran species and families to
      reconstruct early phylogenetic events in the order Diptera. For outgroup
      comparison, the corresponding sequences were used from representative taxa of the
      Siphonaptera, Mecoptera, and Lepidoptera. A subset of 488 unambiguously alignable
      sites was analyzed with respect to important sequence evolution parameters. We
      found (1) sequence variability is significantly higher in double-stranded sites
      than in single-stranded sites, (2) transitions are close to saturation in most
      pairwise sequence comparisons, (3) significant substitution rate heterogeneity
      exists across sites, and (4) significant substitution rate heterogeneity exists
      among lineages. Tree reconstruction was carried out with the neighbor joining,
      maximum parsimony, and maximum likelihood methods. Four major subgroups are
      consistently and robustly supported: the Brachycera, the Culicomorpha, the
      Tipulomorpha sensu stricto, and the hitherto controversial Bibionomorpha sensu
      lato, which includes the families Sciaridae, Mycetophilidae, Cecidomyiidae,
      Bibionidae, Scatopsidae, and Anisopodidae. The phylogenetic relationships within 
      or among these subclades and the positions of the families Psychodidae and
      Trichoceridae were not robustly resolved. These results support the view that the
      mouthparts of extant dipteran larvae evolved from a derived ground state
      characterized by subdivided and obliquely moving mandibles. Furthermore, sequence
      divergence and the paleontological record consistently indicate that a period of 
      rapid cladogenesis gave rise to the major dipteran subgroups.
FAU - Friedrich, M
AU  - Friedrich M
AD  - California Institute of Technology, Division of Biology 156-29, Pasadena,
      California 91125, USA. mf@cco.caltech.edu
FAU - Tautz, D
AU  - Tautz D
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 28S)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA, Ribosomal/*genetics
MH  - Diptera/anatomy &amp; histology/*classification/*genetics
MH  - *Evolution, Molecular
MH  - Fossils
MH  - Genetic Variation
MH  - Mandible/anatomy &amp; histology
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - RNA, Ribosomal, 28S/genetics
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.4.674 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Dec;46(4):674-98. doi: 10.1093/sysbio/46.4.674.

PMID- 11975337
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 4
DP  - 1997 Dec
TI  - Multiple sources of character information and the phylogeny of Hawaiian
      drosophilids.
PG  - 654-73
AB  - Relationships among representatives of the five major Hawaiian Drosophila species
      groups were examined using data from eight different gene regions. A simultaneous
      analysis of these data resulted in a single most-parsimonious tree that (1)
      places the adiastola picture-winged subgroup as sister taxon to the other
      picture-winged subgroups, (2) unites the modified-tarsus species group with flies
      from the Antopocerus species group, and (3) places the white-tip scutellum
      species group as the most basal taxon. Because of the different gene sources used
      in this study, numerous process partitions can be erected within this data set.
      We examined the incongruence among these various partitions and the ramifications
      of these data for the taxonomic consensus, prior agreement, and simultaneous
      analysis approaches to phylogenetic reconstruction. Separate analyses and
      taxonomic consensus appear to be inadequate methods for dealing with the
      partitions in this study. Although detection of incongruence is possible and
      helps elucidate particular areas of disagreement among data sets, separation of
      partitions on the basis of incongruence is problematic for many reasons. First,
      analyzing all genes separately and then either presenting them all as possible
      hypotheses or taking their consensus provides virtually no information concerning
      the relationships among these flies. Second, despite some evidence of
      incongruence, there are no clear delineations among the various gene partitions
      that separate only heterogeneous data. Third, to the extent that problematic
      genes can be identified, these genes have nearly the same information content,
      within a combined analysis framework, as the remaining nonproblematic genes. Our 
      data suggest that significant incongruence among data partitions may be isolated 
      to specific relationships and the "false" signal creating this incongruence is
      most likely to be overcome by a simultaneous analysis. We present a new method,
      partitioned Bremer support, for examining the contribution of a particular data
      partition to the topological support of the simultaneous analysis tree.
FAU - Baker, R H
AU  - Baker RH
AD  - Department of Entomology, American Museum of Natural History, 79th Street at
      Central Park West, New York, New York 10024, USA. baker@amnh.org
FAU - DeSalle, R
AU  - DeSalle R
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
RN  - 0 (DNA, Mitochondrial)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Codon/genetics
MH  - DNA/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Drosophila/anatomy &amp; histology/*classification/*genetics
MH  - Hawaii
MH  - *Phylogeny
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.4.654 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Dec;46(4):654-73. doi: 10.1093/sysbio/46.4.654.

PMID- 11975336
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 4
DP  - 1997 Dec
TI  - Molecular systematics of the Canidae.
PG  - 622-53
AB  - Despite numerous systematic studies, the relationships among many species within 
      the dog family, Canidae, remain unresolved. Two problems of broad evolutionary
      significance are the origins of the taxonomically rich canidae fauna of South
      America and the development in three species of the trenchant heel, a unique
      meat-cutting blade on the lower first molar. The first problem is of interest
      because the fossil record provides little evidence for the origins of divergent
      South American species such as the maned wolf and the bush dog. The second issue 
      is problematic because the trenchant heel, although complex in form, may have
      evolved independently to assist in the processing of meat. We attempted to
      resolve these two issues and five other specific taxonomic controversies by
      phylogenetic analysis of 2,001 base pairs of mitochondrial DNA (mtDNA) sequence
      data from 23 canidae species. The mtDNA tree topology, coupled with data from the
      fossil record, and estimates of rates of DNA sequence divergence suggest at least
      three and possibly four North American invasions of South America. This result
      implies that an important chapter in the evolution of modern canids remains to be
      discovered in the fossil record and that the South American canidae endemism is
      as much the result of extinction outside of South America as it is due to
      speciation within South America. The origin of the trenchant heel is not well
      resolved by our data, although the maximum parsimony tree is weakly consistent
      with a single origin followed by multiple losses of the character in several
      extant species. A combined analysis of the mtDNA data and published morphological
      data provides unexpected support for a monophyletic South American canidae clade.
      However, the homogeneity partition tests indicate significant heterogeneity
      between the two data sets.
FAU - Wayne, R K
AU  - Wayne RK
AD  - Department of Biology, University of California, Los Angeles, California 90095,
      USA.
FAU - Geffen, E
AU  - Geffen E
FAU - Girman, D J
AU  - Girman DJ
FAU - Koepfli, K P
AU  - Koepfli KP
FAU - Lau, L M
AU  - Lau LM
FAU - Marshall, C R
AU  - Marshall CR
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Carnivora/anatomy &amp; histology/*classification/*genetics
MH  - Cytochrome b Group/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Evolution, Molecular
MH  - Molecular Sequence Data
MH  - North America
MH  - *Phylogeny
MH  - Sequence Homology, Nucleic Acid
MH  - South America
MH  - Tooth/anatomy &amp; histology
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.4.622 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Dec;46(4):622-53. doi: 10.1093/sysbio/46.4.622.

PMID- 11975335
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 4
DP  - 1997 Dec
TI  - NEXUS: an extensible file format for systematic information.
PG  - 590-621
AB  - NEXUS is a file format designed to contain systematic data for use by computer
      programs. The goals of the format are to allow future expansion, to include
      diverse kinds of information, to be independent of particular computer operating 
      systems, and to be easily processed by a program. To this end, the format is
      modular, with a file consisting of separate blocks, each containing one
      particular kind of information, and consisting of standardized commands. Public
      blocks (those containing information utilized by several programs) house
      information about taxa, morphological and molecular characters, distances,
      genetic codes, assumptions, sets, trees, etc.; private blocks contain information
      of relevance to single programs. A detailed description of commands in public
      blocks is given. Guidelines are provided for reading and writing NEXUS files and 
      for extending the format.
FAU - Maddison, D R
AU  - Maddison DR
AD  - Department of Entomology, University of Arizona, Tucson, Arizona 85721, USA.
      beetle@ag.arizona.edu
FAU - Swofford, D L
AU  - Swofford DL
FAU - Maddison, W P
AU  - Maddison WP
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 63231-63-0 (RNA)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA/genetics
MH  - *Database Management Systems
MH  - Databases, Genetic
MH  - Phylogeny
MH  - RNA/genetics
MH  - *Software
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.4.590 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Dec;46(4):590-621. doi: 10.1093/sysbio/46.4.590.

PMID- 11975334
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 4
DP  - 1997 Dec
TI  - DNA synapomorphies for a variety of taxonomic levels from a cosmid library from
      the New World bat Macrotus waterhousii.
PG  - 579-89
AB  - An effective method yielding taxon-specific markers from the genome of a single
      individual would be valuable for many types of scientific investigations,
      including systematic, forensic, conservation, and evolutionary studies. We
      explored the use of cosmid libraries, with insert sizes averaging 35 kb, to
      streamline the process of locating sequences of DNA that can serve as taxonomic
      markers from the specific to the ordinal levels. By screening approximately 2.6% 
      of the leaf-nosed bat (Macrotus waterhousii) genome, we identified several
      potential DNA fragments that appear to be synapomorphic for a variety of
      taxonomic levels. A more thorough analysis of the markers documented that 17
      Macrotus-specific clones represent three distinct DNA generic markers, whereas 30
      microchiropteran clones represent multiple copies of a single family of
      repetitive DNA. The Microchiroptera taxon markers hybridize with representatives 
      of most of the Microchiroptera families; however, no hybridization was detected
      for members of the superfamily Rhinolophoidea. These results demonstrate that
      cosmid libraries can be a valuable source for isolating taxon-specific markers
      from mammals even when the insert size is as large as 35 kb.
FAU - Baker, R J
AU  - Baker RJ
AD  - Department of Biological Sciences and The Museum, Texas Tech University, Lubbock,
      Texas 79409, USA. bjrjb@ttacs.ttu.edu
FAU - Longmire, J L
AU  - Longmire JL
FAU - Maltbie, M
AU  - Maltbie M
FAU - Hamilton, M J
AU  - Hamilton MJ
FAU - Van Den Bussche, R A
AU  - Van Den Bussche RA
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Chiroptera/*classification/*genetics
MH  - Cosmids/genetics
MH  - DNA/*genetics
MH  - Gene Library
MH  - Genetic Markers
MH  - Genetic Techniques
MH  - In Situ Hybridization, Fluorescence
MH  - Phylogeny
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.4.579 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Dec;46(4):579-89. doi: 10.1093/sysbio/46.4.579.

PMID- 11975333
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 3
DP  - 1997 Sep
TI  - Analyzing large data sets: rbcL 500 revisited.
PG  - 554-63
FAU - Rice, K A
AU  - Rice KA
AD  - Harvard University Herbaria, 22 Divinity Avenue, Cambridge, Massachusetts 02138, 
      USA. krice@saul.cis.upenn.edu
FAU - Donoghue, M J
AU  - Donoghue MJ
FAU - Olmstead, R G
AU  - Olmstead RG
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 4.1.1.39 (RbcL protein, plastid)
RN  - EC 4.1.1.39 (Ribulose-Bisphosphate Carboxylase)
SB  - IM
MH  - Biometry
MH  - Chloroplasts/genetics
MH  - Data Interpretation, Statistical
MH  - *Phylogeny
MH  - Plants/classification/enzymology/genetics
MH  - Ribulose-Bisphosphate Carboxylase/*genetics
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.3.554 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Sep;46(3):554-63. doi: 10.1093/sysbio/46.3.554.

PMID- 11975332
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 3
DP  - 1997 Sep
TI  - Trees within trees: genes and species, molecules and morphology.
PG  - 537-53
AB  - The construction and interpretation of gene trees is fundamental in molecular
      systematics. If the gene is defined in a historical (coalescent) sense, there can
      be multiple gene trees within the single contiguous set of nucleotides, and
      attempts to construct a single tree for such a sequence must deal with homoplasy 
      created by conflict among divergent histories. On a larger scale, incongruence is
      expected among gene tree topologies at different loci of individuals within
      sexually reproducing species, and it has been suggested that this discordance can
      be used to delimit species. A practical concern for such topological methods is
      that polymorphisms may be maintained through numerous cladogenic events; this
      polymorphism problem is less of a concern for nontopological approaches to
      species delimitation using molecular data. Although a central theoretical concern
      in molecular systematics is discordance between a given gene tree and the true
      "species tree," the primary empirical problem faced in reconstructing taxic
      phylogeny is incongruence among the trees inferred from different sequences.
      Linkage relationships limit character independence and thus have important
      implications for handling multiple data sets in phylogenetic analysis,
      particularly at the species level, where incongruence among different
      historically associated loci is expected. Gene trees can also be reconstructed
      for loci that influence phenotypic characters, but there is at best a tenuous
      relationship between phenotypic homoplasy and homoplasy in such gene trees.
      Nevertheless, expression patterns and orthology relationships of genes involved
      in the expression of phenotypes can in theory provide criteria for homology
      assessment of morphological characters.
FAU - Doyle, J J
AU  - Doyle JJ
AD  - L. H. Bailey Hortorium, 462 Mann Library Building, Cornell University, Ithaca,
      New York 14853, USA. jjd5@cornell.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Anatomy, Comparative
MH  - Animals
MH  - Biometry
MH  - DNA/genetics
MH  - Genes
MH  - Genetic Techniques
MH  - *Phylogeny
MH  - Recombination, Genetic
MH  - Species Specificity
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.3.537 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Sep;46(3):537-53. doi: 10.1093/sysbio/46.3.537.

PMID- 11975331
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 3
DP  - 1997 Sep
TI  - Morphology, fossils, divergence timing, and the phylogenetic relationships of
      Gavialis.
PG  - 479-522
AB  - Although morphological data have historically favored a basal position for the
      Indian gharial (Gavialis gangeticus) within Crocodylia and a Mesozoic divergence 
      between Gavialis and all other crocodylians, several recent molecular data sets
      have argued for a sister-group relationship between Gavialis and the Indonesian
      false gharial (Tomistoma schlegelii) and a divergence between them no earlier
      than the Late Tertiary. Fossils were added to a matrix of 164 discrete
      morphological characters and subjected to parsimony analysis. When morphology was
      analyzed alone, Gavialis was the sister taxon of all other extant crocodylians
      whether or not fossil ingroup taxa were included, and a sister-group relationship
      between Gavialis and Tomistoma was significantly less parsimonious. In
      combination with published sequence and restriction site fragment data, Gavialis 
      was the sister taxon of all other living crocodylians, but the position of
      Tomistoma depended on the inclusion of fossil ingroup taxa; with or without
      fossils, preferred morphological and molecular topologies were not significantly 
      different. Fossils closer to Gavialis than to Tomistoma can be recognized in the 
      Late Cretaceous, and fossil relatives of Tomistoma are known from the basal
      Eocene, strongly indicating a divergence long before the Late Tertiary.
      Comparison of minimum divergence time from the fossil record with different
      measures of molecular distance indicates evolutionary rate heterogeneity within
      Crocodylia. Fossils strongly contradict a post-Oligocene divergence between
      Gavialis and any other living crocodylian, but the phylogenetic placement of
      Gavialis is best viewed as unresolved.
FAU - Brochu, C A
AU  - Brochu CA
AD  - Department of Geological Sciences, University of Texas, Austin, Texas 78712, USA.
      gator@mail.utexas.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Alligators and Crocodiles/anatomy &amp; histology/*classification/*genetics
MH  - Animals
MH  - Base Sequence
MH  - Biometry
MH  - DNA, Ribosomal/genetics
MH  - Fossils
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Homology, Nucleic Acid
MH  - Time Factors
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.3.479 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Sep;46(3):479-522. doi: 10.1093/sysbio/46.3.479.

PMID- 11975330
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 3
DP  - 1997 Sep
TI  - Is congruence between data partitions a reliable predictor of phylogenetic
      accuracy? Empirically testing an iterative procedure for choosing among
      phylogenetic methods.
PG  - 464-78
AB  - The relationship between phylogenetic accuracy and congruence between data
      partitions collected from the same taxa was explored for mitochondrial DNA
      sequences from two well-supported vertebrate phylogenies. An iterative procedure 
      was adopted whereby accuracy, phylogenetic signal, and congruence were measured
      before and after modifying a simple reconstruction model, equally weighted
      parsimony. These modifications included transversion parsimony, successive
      weighting, and six-parameter parsimony. For the data partitions examined, there
      is a generally positive relationship between congruence and phylogenetic
      accuracy. If congruence increased without decreasing resolution or phylogenetic
      signal, this increased congruence was a good predictor of accuracy. If congruence
      increased as a result of poor resolution, the degree of congruence was not a good
      predictor of accuracy. For all sets of data partitions, six-parameter parsimony
      methods show a consistently positive relationship between congruence and
      accuracy. Unlike successive weighting, six-parameter parsimony methods were not
      strongly influenced by the starting tree.
FAU - Cunningham, C W
AU  - Cunningham CW
AD  - Zoology Department, Duke University, Durham, North Carolina 27708-0325, USA.
      cliff@duke.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Biometry/*methods
MH  - Codon/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Data Interpretation, Statistical
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Muridae/classification/genetics
MH  - Peromyscus/classification/genetics
MH  - *Phylogeny
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.3.464 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Sep;46(3):464-78. doi: 10.1093/sysbio/46.3.464.

PMID- 11975329
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 3
DP  - 1997 Sep
TI  - Phylogeography and molecular systematics of the Peromyscus aztecus species group 
      (Rodentia: Muridae) inferred using parsimony and likelihood.
PG  - 426-40
AB  - Mice of the Peromyscus aztecus species group occur at mid to high elevations in
      several mountain ranges in the highlands of Middle America (Mexico and Central
      America), a region of high endemicity. We examined the biogeography of this group
      by conducting phylogenetic analyses of 668 bp of the mitochondrial cytochrome b
      (cyt b) gene. Phylogenetic analyses under both parsimony and likelihood
      frameworks produced the same topologies, but estimates of nodal support were
      artificially high in weighted parsimony analyses. This difference is attributed
      to the inability of parsimony to optimize branch lengths when evaluating
      topologies. These data indicate that the P. aztecus-like populations from south
      and east of the Isthmus of Tehuantepec currently assigned to P. a. oaxacensis
      represent a distinct species, with genetic distances as high as 0.091. In
      addition, P. hylocetes is strongly divergent from Mexican populations of P.
      aztecus (genetic distances of 0.044-0.069), supporting the recognition of this
      taxon as a distinct species. The history of divergence in this group can be
      explained by a series of apparently early to middle Pleistocene vicariance events
      associated with glacial cycles. The Sierra Madre Occidental and Cordillera
      Transvolcanica each appear to be faunistically isolated, the Isthmus of
      Tehuantepec appears to have been a strong Pleistocene barrier, and the Sierra
      Madre Oriental has affinities with the Sierra Madre del Sur and the highlands of 
      central Oaxaca.
FAU - Sullivan, J
AU  - Sullivan J
AD  - Laboratory of Molecular Systematics, Smithsonian Museum Support Center, 4210
      Silver Hill Road, Suitland, Maryland 20746, USA. sullivan@onyx.si.edu
FAU - Markert, J A
AU  - Markert JA
FAU - Kilpatrick, C W
AU  - Kilpatrick CW
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Central America
MH  - Cytochrome b Group/genetics
MH  - DNA/genetics
MH  - Genetics, Population
MH  - Geography
MH  - Mexico
MH  - Molecular Sequence Data
MH  - Peromyscus/*classification/*genetics
MH  - *Phylogeny
MH  - Sequence Homology, Nucleic Acid
MH  - Species Specificity
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.3.426 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Sep;46(3):426-40. doi: 10.1093/sysbio/46.3.426.

PMID- 11975328
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 3
DP  - 1997 Sep
TI  - Phylogeny of lichen- and non-lichen-forming omphalinoid mushrooms and the utility
      of testing for combinability among multiple data sets.
PG  - 373-406
AB  - As an initial step toward developing a model system to study requirements for and
      consequences of transitions to mutualism, the phylogeny of a group of closely
      related lichenized and nonlichenized basidiomycetes (Omphalina) was
      reconstructed. The phylogenetic analyses are based on four data sets representing
      different regions of the nuclear ribosomal repeat unit (ITS1, 5.8S, ITS2, and
      25S) obtained from 30 species of Omphalina and related genera. The resulting
      phylogenetic trees from each of these four data sets, when analyzed separately,
      were not identical. Testing for the combinability of these four data sets
      suggested that they could not be combined in their entirety. The removal of
      ambiguous alignments and saturated sites was sufficient, after reapplying the
      combinability test on the pruned data sets, to explain the topological
      discrepancies. In this process, the first of two complementary tests developed by
      Rodrigo et al. (1993, N.Z. J. Bot. 31:257-268) to assess whether two data sets
      are the result of the same phylogenetic history was found to be biased, rejecting
      the combinability of two data sets even when they are samples of the same
      phylogenetic history. Combining the four pruned data sets yielded phylogenies
      that suggest the five lichen-forming species of Omphalina form a monophyletic
      group. The sister group to this symbiotic clade consists mostly of dark brown
      Omphalina species intermixed with species from the genera Arrhenia and
      Phaeothellus. The genera Omphalina and Gerronema are shown to be polyphyletic.
      The lichen-forming species O. ericetorum and the nonmutualistic species O.
      velutipes, O. epichysium, and O. sphagnicola are the best candidates for
      experimental work designed to gain a better understanding of mechanisms involved 
      in symbiotic interactions and the role symbiosis has played in the evolution of
      fungi.
FAU - Lutzoni, F M
AU  - Lutzoni FM
AD  - Department of Botany, Duke University, Durham, North Carolina 27708-0339, USA.
      lutzoni@fmppr.fmnh.org
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Fungal)
SB  - IM
MH  - Agaricales/classification/*genetics
MH  - Base Sequence
MH  - DNA, Fungal/genetics
MH  - Lichens/classification/*genetics
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Sequence Homology, Nucleic Acid
MH  - Symbiosis
EDAT- 2002/04/27 10:00
MHDA- 2002/06/04 10:01
CRDT- 2002/04/27 10:00
PHST- 2002/04/27 10:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 2002/04/27 10:00 [entrez]
AID - 10.1093/sysbio/46.3.373 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Sep;46(3):373-406. doi: 10.1093/sysbio/46.3.373.

PMID- 11943100
OWN - NLM
STAT- MEDLINE
DCOM- 20020722
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 1
DP  - 2002 Feb
TI  - A character-based method for measuring the fit of a cladogram to the fossil
      record.
PG  - 176-91
FAU - Angielczyk, Kenneth D
AU  - Angielczyk KD
AD  - Department of Integrative Biology and Museum of Paleontology, 1101 VLSB,
      University of California, Berkeley, California 94720, USA.
      etranger@socrates.berkeley.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Computer Simulation
MH  - Databases, Genetic
MH  - *Fossils
MH  - Genetic Techniques
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2002/04/12 10:00
MHDA- 2002/07/23 10:01
CRDT- 2002/04/12 10:00
PHST- 2002/04/12 10:00 [pubmed]
PHST- 2002/07/23 10:01 [medline]
PHST- 2002/04/12 10:00 [entrez]
AID - 10.1080/106351502753475952 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Feb;51(1):176-91. doi: 10.1080/106351502753475952.

PMID- 11943099
OWN - NLM
STAT- MEDLINE
DCOM- 20020722
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 1
DP  - 2002 Feb
TI  - Comparing the gap excess ratio and the retention index of the stratigraphic
      character.
PG  - 166-76
FAU - Finarelli, John A
AU  - Finarelli JA
AD  - Department of Earth Sciences, 56 College Road, University of New Hampshire,
      Durham, New Hampshire 03824, USA. dryopithecus@excite.com
FAU - Clyde, William C
AU  - Clyde WC
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - Data Interpretation, Statistical
MH  - Databases, Genetic
MH  - Fossils
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2002/04/12 10:00
MHDA- 2002/07/23 10:01
CRDT- 2002/04/12 10:00
PHST- 2002/04/12 10:00 [pubmed]
PHST- 2002/07/23 10:01 [medline]
PHST- 2002/04/12 10:00 [entrez]
AID - 10.1080/106351502753475943 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Feb;51(1):166-76. doi: 10.1080/106351502753475943.

PMID- 11943098
OWN - NLM
STAT- MEDLINE
DCOM- 20020722
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 1
DP  - 2002 Feb
TI  - Geographic origin of human mitochondrial DNA: accommodating phylogenetic
      uncertainty and model comparison.
PG  - 155-65
AB  - Many biogeographic problems are tested on phylogenetic trees. Typically, the
      uncertainty in the phylogeny is not accommodated when investigating the
      biogeography of the organisms. Here we present a method that accommodates
      uncertainty in the phylogenetic trees. Moreover, we describe a simple method for 
      examining the support for competing biogeographic scenarios. We illustrate the
      method using mitochondrial DNA sequences sampled from modern humans. The
      geographic origin of modern human mtDNA is inferred to be in Africa, although
      support for this hypothesis was ambiguous for data from an early paper.
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
AD  - Department of Biology, University of Rochester, Rochester, New York 14627, USA.
      johnh@brahms.biology.rochester.edu
FAU - Imennov, Nikita S
AU  - Imennov NS
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Africa
MH  - Bayes Theorem
MH  - DNA, Mitochondrial/*genetics
MH  - Data Interpretation, Statistical
MH  - Evolution, Molecular
MH  - Humans
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2002/04/12 10:00
MHDA- 2002/07/23 10:01
CRDT- 2002/04/12 10:00
PHST- 2002/04/12 10:00 [pubmed]
PHST- 2002/07/23 10:01 [medline]
PHST- 2002/04/12 10:00 [entrez]
AID - 10.1080/106351502753475934 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Feb;51(1):155-65. doi: 10.1080/106351502753475934.

PMID- 11943097
OWN - NLM
STAT- MEDLINE
DCOM- 20020722
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 1
DP  - 2002 Feb
TI  - Matrix representation with parsimony, taxonomic congruence, and total evidence.
PG  - 151-5
FAU - Pisani, Davide
AU  - Pisani D
AD  - Department of Earth Sciences, University of Bristol, Bristol BS8 1RJ, UK.
      Davide.Pisani@bristol.ac.uk
FAU - Wilkinson, Mark
AU  - Wilkinson M
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Data Interpretation, Statistical
MH  - Databases, Genetic
MH  - Genetic Techniques
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2002/04/12 10:00
MHDA- 2002/07/23 10:01
CRDT- 2002/04/12 10:00
PHST- 2002/04/12 10:00 [pubmed]
PHST- 2002/07/23 10:01 [medline]
PHST- 2002/04/12 10:00 [entrez]
AID - 10.1080/106351502753475925 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Feb;51(1):151-5. doi: 10.1080/106351502753475925.

PMID- 11943096
OWN - NLM
STAT- MEDLINE
DCOM- 20020722
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 1
DP  - 2002 Feb
TI  - Building supertrees: an empirical assessment using the grass family (Poaceae).
PG  - 136-50
AB  - Large and comprehensive phylogenetic trees are desirable for studying
      macroevolutionary processes and for classification purposes. Such trees can be
      obtained in two different ways. Either the widest possible range of taxa can be
      sampled and used in a phylogenetic analysis to produce a "big tree," or
      preexisting topologies can be used to create a supertree. Although large
      multigene analyses are often favored, combinable data are not always available,
      and supertrees offer a suitable solution. The most commonly used method of
      supertree reconstruction, matrix representation with parsimony (MRP), is
      presented here. We used a combined data set for the Poaceae to (1) assess the
      differences between an approach that uses combined data and one that uses
      different MRP modifications based on the character partitions and (2) investigate
      the advantages and disadvantages of these modifications. Baum and Ragan and
      Purvis modifications gave similar results. Incorporating bootstrap support
      associated with pre-existing topologies improved Baum and Ragan modification and 
      its similarity with a combined analysis. Finally, we used the supertree
      reconstruction approach on 55 published phylogenies to build one of most
      comprehensive phylogenetic trees published for the grass family including 403
      taxa and discuss its strengths and weaknesses in relation to other published
      hypotheses.
FAU - Salamin, Nicolas
AU  - Salamin N
AD  - Department of Botany, University of Dublin, Trinity College, Dublin 2, Ireland.
      salaminn@tcd.ie
FAU - Hodkinson, Trevor R
AU  - Hodkinson TR
FAU - Savolainen, Vincent
AU  - Savolainen V
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Data Interpretation, Statistical
MH  - Databases, Genetic
MH  - Genetic Techniques
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Poaceae/*classification/*genetics
EDAT- 2002/04/12 10:00
MHDA- 2002/07/23 10:01
CRDT- 2002/04/12 10:00
PHST- 2002/04/12 10:00 [pubmed]
PHST- 2002/07/23 10:01 [medline]
PHST- 2002/04/12 10:00 [entrez]
AID - 10.1080/106351502753475916 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Feb;51(1):136-50. doi: 10.1080/106351502753475916.

PMID- 11943095
OWN - NLM
STAT- MEDLINE
DCOM- 20020722
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 1
DP  - 2002 Feb
TI  - Phylogeny of the Lake Tanganyika cichlid species flock and its relationship to
      the Central and East African haplochromine cichlid fish faunas.
PG  - 113-35
AB  - Lake Tanganyika, the oldest of the East African Great Lakes, harbors the
      ecologically, morphologically, and behaviorally most complex of all assemblages
      of cichlid fishes, consisting of about 200 described species. The evolutionary
      old age of the cichlid assemblage, its extreme degree of morphological
      differentiation, the lack of species with intermediate morphologies, and the
      rapidity of lineage formation have made evolutionary reconstruction difficult.
      The number and origin of seeding lineages, particularly the possible contribution
      of riverine haplochromine cichlids to endemic lacustrine lineages, remains
      unclear. Our phylogenetic analyses, based on mitochondrial DNA sequences of three
      gene segments of 49 species (25% of all described species, up to 2,400 bp each), 
      yield robust phylogenies that provide new insights into the Lake Tanganyika
      adaptive radiation as well as into the origin of the Central- and East-African
      haplochromine faunas. Our data suggest that eight ancient African lineages may
      have seeded the Tanganyikan cichlid radiation. One of these seeding lineages,
      probably comprising substrate spawning Lamprologus-like species, diversified into
      six lineages that evolved mouthbrooding during the initial stage of the
      radiation. All analyzed haplochromines from surrounding rivers and lakes seem to 
      have evolved within the radiating Tanganyikan lineages. Thus, our findings
      contradict the current hypothesis that ancestral riverine haplochromines
      colonized Lake Tanganyika to give rise to at least part of its spectacular
      endemic cichlid species assemblage. Instead, the early phases of the Tanganyikan 
      radiation affected Central and East African rivers and lakes. The haplochromines 
      may have evolved in the Tanganyikan basin before the lake became a hydrologically
      and ecologically closed system and then secondarily colonized surrounding rivers.
      Apparently, therefore, the current diversity of Central and East African
      haplochromines represents a relatively young and polyphyletic fauna that evolved 
      from or in parallel to lineages now endemic to Lake Tanganyika.
FAU - Salzburger, Walter
AU  - Salzburger W
AD  - Department of Zoology and Limnology, University of Innsbruck, Technikertrasse 25,
      6020 Innsbruck, Austria. walter.salzburger@uni-konstanz.de
FAU - Meyer, Axel
AU  - Meyer A
FAU - Baric, Sanja
AU  - Baric S
FAU - Verheyen, Erik
AU  - Verheyen E
FAU - Sturmbauer, Christian
AU  - Sturmbauer C
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Africa
MH  - Animals
MH  - Cichlids/*classification/*genetics
MH  - Cytochrome b Group/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Evolution, Molecular
MH  - Fresh Water
MH  - Models, Genetic
MH  - Phylogeny
MH  - Tanzania
EDAT- 2002/04/12 10:00
MHDA- 2002/07/23 10:01
CRDT- 2002/04/12 10:00
PHST- 2002/04/12 10:00 [pubmed]
PHST- 2002/07/23 10:01 [medline]
PHST- 2002/04/12 10:00 [entrez]
AID - 10.1080/106351502753475907 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Feb;51(1):113-35. doi: 10.1080/106351502753475907.

PMID- 11943094
OWN - NLM
STAT- MEDLINE
DCOM- 20020722
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 1
DP  - 2002 Feb
TI  - A molecular phylogenetic study of the Palmae (Arecaceae) based on atpB, rbcL, and
      18S nrDNA sequences.
PG  - 92-112
AB  - Notoriously slow rates of molecular evolution and convergent evolution among some
      morphological characters have limited phylogenetic resolution for the palm family
      (Arecaceae). This study adds nuclear DNA (18S SSU rRNA) and chloroplast DNA
      (cpDNA; atpB and rbcL) sequence data for 65 genera of palms and characterizes
      molecular variation for each molecule. Phylogenetic relationships were estimated 
      with maximum likelihood and maximum parsimony techniques for the new data and for
      previously published molecular data for 45 palm genera. Maximum parsimony
      analysis was also used to compare molecular and morphological data for 33 palm
      genera. Incongruence among datasets was detected between cpDNA and 18S data and
      between molecular and morphological data. Most conflict between nuclear and cpDNA
      data was associated with the genus Nypa. Several taxa showed relatively long
      branches with 18S data, but phylogenetic resolution of these taxa was essentially
      the same for 18S and cpDNA data. Base composition bias for 18S that contributed
      to erroneous phylogenetic resolution in other taxa did not seem to be present in 
      Palmae. Morphological data were incongruent with all molecular data due to
      apparent morphological homoplasy for Caryoteae, Ceroxyloideae, Iriarteae, and
      Thrinacinae. Both cpDNA and nuclear 18S data firmly resolved Caryoteae with
      Borasseae of Coryphoideae, suggesting that at least some morphological characters
      used to place Caryoteae in Arecoideae are homoplastic. In this study, increased
      character sampling seems to be more important than increased taxon sampling; a
      comparison of the full (65-taxon) and reduced (45- and 33-taxon) datasets
      suggests little difference in core topology but considerably more nodal support
      with the increased character sample sizes. These results indicate a general trend
      toward a stable estimate of phylogenetic relationships for the Palmae. Although
      the 33-taxon topologies are even better resolved, they lack several critical taxa
      and are affected by incongruence between molecular and morphological data. As
      such, a comparison of results from the 45- and 33-taxon trees offers the best
      available reference for phylogenetic inference on palms.
FAU - Hahn, William J
AU  - Hahn WJ
AD  - Department of Ecology, Evolution, and Environmental Biology, Center for
      Environmental Research and Conservation, Columbia University, 1200 Amsterdam
      Ave., New York, New York 10027, USA. wh55@columbia.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
RN  - EC 4.1.1.39 (RbcL protein, plastid)
RN  - EC 4.1.1.39 (Ribulose-Bisphosphate Carboxylase)
SB  - IM
MH  - Arecaceae/*classification/enzymology/*genetics
MH  - DNA, Plant/*genetics
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Phylogeny
MH  - Ribulose-Bisphosphate Carboxylase/genetics
MH  - Sequence Alignment/statistics &amp; numerical data
MH  - Species Specificity
EDAT- 2002/04/12 10:00
MHDA- 2002/07/23 10:01
CRDT- 2002/04/12 10:00
PHST- 2002/04/12 10:00 [pubmed]
PHST- 2002/07/23 10:01 [medline]
PHST- 2002/04/12 10:00 [entrez]
AID - 10.1080/106351502753475899 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Feb;51(1):92-112. doi: 10.1080/106351502753475899.

PMID- 11943093
OWN - NLM
STAT- MEDLINE
DCOM- 20020722
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 1
DP  - 2002 Feb
TI  - Delimiting species using DNA and morphological variation and discordant species
      limits in spiny lizards (Sceloporus).
PG  - 69-91
AB  - Haplotype phylogenies based on DNA sequence data are increasingly being used to
      test traditional species-level taxonomies based on morphology. However, few
      studies have critically compared species limits based on morphological and DNA
      data, and the methods used to delimit species using either type of data are only 
      rarely explained. In this paper, we review three approaches for species
      delimitation (tree-based with DNA data and tree-based and character-based with
      morphological data) and propose explicit protocols for each. We then compare
      species limits inferred from these approaches, using morphological and mtDNA data
      for the Yarrow's spiny lizard (Sceloporus jarrovii), a traditionally polytypic
      species from the southwestern United States and Mexico. All three approaches
      support division of S. jarrovii into five species, but only two species are the
      same among the three approaches. We find the greatest support for the five
      species that are delimited based on mtDNA data, and we argue that mtDNA data may 
      have important (and previously unappreciated) advantages for species
      delimitation. Because different data and approaches can disagree so extensively, 
      our results demonstrate that the methodology of species delimitation is a
      critical issue in systematics.
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Section of Amphibians and Reptiles, Carnegie Museum of Natural History,
      Pittsburgh, Pennsylvania 15213-4080, USA. wiensj@carnegiemuseums.org
FAU - Penkrot, Tonya A
AU  - Penkrot TA
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - DNA, Mitochondrial/*genetics
MH  - Data Interpretation, Statistical
MH  - Genetic Variation
MH  - Haplotypes
MH  - Lizards/anatomy &amp; histology/*classification/*genetics
MH  - Mexico
MH  - Models, Genetic
MH  - Phylogeny
MH  - Species Specificity
MH  - United States
EDAT- 2002/04/12 10:00
MHDA- 2002/07/23 10:01
CRDT- 2002/04/12 10:00
PHST- 2002/04/12 10:00 [pubmed]
PHST- 2002/07/23 10:01 [medline]
PHST- 2002/04/12 10:00 [entrez]
AID - 10.1080/106351502753475880 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Feb;51(1):69-91. doi: 10.1080/106351502753475880.

PMID- 11943092
OWN - NLM
STAT- MEDLINE
DCOM- 20020722
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 1
DP  - 2002 Feb
TI  - Molecular systematics of the Eastern Fence Lizard (Sceloporus undulatus): a
      comparison of Parsimony, Likelihood, and Bayesian approaches.
PG  - 44-68
AB  - Phylogenetic analysis of large datasets using complex nucleotide substitution
      models under a maximum likelihood framework can be computationally infeasible,
      especially when attempting to infer confidence values by way of nonparametric
      bootstrapping. Recent developments in phylogenetics suggest the computational
      burden can be reduced by using Bayesian methods of phylogenetic inference.
      However, few empirical phylogenetic studies exist that explore the efficiency of 
      Bayesian analysis of large datasets. To this end, we conducted an extensive
      phylogenetic analysis of the wide-ranging and geographically variable Eastern
      Fence Lizard (Sceloporus undulatus). Maximum parsimony, maximum likelihood, and
      Bayesian phylogenetic analyses were performed on a combined mitochondrial DNA
      dataset (12S and 16S rRNA, ND1 protein-coding gene, and associated tRNA; 3,688 bp
      total) for 56 populations of S. undulatus (78 total terminals including other S. 
      undulatus group species and outgroups). Maximum parsimony analysis resulted in
      numerous equally parsimonious trees (82,646 from equally weighted parsimony and
      335 from weighted parsimony). The majority rule consensus tree derived from the
      Bayesian analysis was topologically identical to the single best phylogeny
      inferred from the maximum likelihood analysis, but required approximately 80%
      less computational time. The mtDNA data provide strong support for the monophyly 
      of the S. undulatus group and the paraphyly of "S. undulatus" with respect to S. 
      belli, S. cautus, and S. woodi. Parallel evolution of ecomorphs within "S.
      undulatus" has masked the actual number of species within this group. This
      evidence, along with convincing patterns of phylogeographic differentiation
      suggests "S. undulatus" represents at least four lineages that should be
      recognized as evolutionary species.
FAU - Leache, Adam D
AU  - Leache AD
AD  - Department of Biology, San Diego State University, San Diego, California
      92182-4614, USA. aleach1@lsu.edu
FAU - Reeder, Tod W
AU  - Reeder TW
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - DNA, Mitochondrial/genetics
MH  - Data Interpretation, Statistical
MH  - Databases, Nucleic Acid
MH  - Likelihood Functions
MH  - Lizards/*classification/*genetics
MH  - Mexico
MH  - Models, Genetic
MH  - Phylogeny
MH  - Sequence Alignment/statistics &amp; numerical data
MH  - United States
EDAT- 2002/04/12 10:00
MHDA- 2002/07/23 10:01
CRDT- 2002/04/12 10:00
PHST- 2002/04/12 10:00 [pubmed]
PHST- 2002/07/23 10:01 [medline]
PHST- 2002/04/12 10:00 [entrez]
AID - 10.1080/106351502753475871 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Feb;51(1):44-68. doi: 10.1080/106351502753475871.

PMID- 11943091
OWN - NLM
STAT- MEDLINE
DCOM- 20020722
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 1
DP  - 2002 Feb
TI  - Inferring the root of a phylogenetic tree.
PG  - 32-43
AB  - Phylogenetic trees can be rooted by a number of criteria. Here, we introduce a
      Bayesian method for inferring the root of a phylogenetic tree by using one of
      several criteria: the outgroup, molecular clock, and nonreversible model of DNA
      substitution. We perform simulation analyses to examine the relative ability of
      these three criteria to correctly identify the root of the tree. The outgroup and
      molecular clock criteria were best able to identify the root of the tree, whereas
      the nonreversible model was able to identify the root only when the substitution 
      process was highly nonreversible. We also examined the performance of the
      criteria for a tree of four species for which the topology and root position are 
      well supported. Results of the analyses of these data are consistent with the
      simulation results.
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
AD  - Department of Biology, University of Rochester, Rochester, New York 14627, USA.
      johnh@brahms.biology.rochester.edu
FAU - Bollback, Jonathan P
AU  - Bollback JP
FAU - Levine, Amy M
AU  - Levine AM
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Bayes Theorem
MH  - Computer Simulation
MH  - DNA/genetics
MH  - Databases, Nucleic Acid
MH  - Markov Chains
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2002/04/12 10:00
MHDA- 2002/07/23 10:01
CRDT- 2002/04/12 10:00
PHST- 2002/04/12 10:00 [pubmed]
PHST- 2002/07/23 10:01 [medline]
PHST- 2002/04/12 10:00 [entrez]
AID - 10.1080/106351502753475862 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Feb;51(1):32-43. doi: 10.1080/106351502753475862.

PMID- 11943090
OWN - NLM
STAT- MEDLINE
DCOM- 20020722
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 1
DP  - 2002 Feb
TI  - Increased congruence does not necessarily indicate increased phylogenetic
      accuracy--the behavior of the incongruence length difference test in mixed-model 
      analyses.
PG  - 19-31
AB  - Comprehensive phylogenetic analyses utilize data from distinct sources, including
      nuclear, mitochondrial, and plastid molecular sequences and morphology. Such
      heterogeneous datasets are likely to require distinct models of analysis, given
      the different histories of mutational biases operating on these characters. The
      incongruence length difference (ILD) test is increasingly being used to arbitrate
      between competing models of phylogenetic analysis in cases where multiple data
      partitions have been collected. Our work suggests that the ILD test is unlikely
      to be an effective measure of congruence when two datasets differ markedly in
      size. We show that models that increase the contribution of one data partition
      over another are likely to increase congruence, as measured by this test. More
      alarmingly, for many bipartition comparisons, character congruence increases
      bimodally - either increasing or decreasing the contribution of one data
      partition will increase congruence - making it impossible to arrive at a single
      optimally congruent model of analysis.
FAU - Dowton, Mark
AU  - Dowton M
AD  - Institute of Conservation Biology, Department of Biology, Wollongong University, 
      Wollongong, NSW, 2522, Australia. mdowton@uow.edu.au
FAU - Austin, Andrew D
AU  - Austin AD
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - DNA, Ribosomal/genetics
MH  - Data Interpretation, Statistical
MH  - Databases, Nucleic Acid
MH  - Electron Transport Complex IV/genetics
MH  - Genes, Insect
MH  - Hymenoptera/classification/enzymology/genetics
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2002/04/12 10:00
MHDA- 2002/07/23 10:01
CRDT- 2002/04/12 10:00
PHST- 2002/04/12 10:00 [pubmed]
PHST- 2002/07/23 10:01 [medline]
PHST- 2002/04/12 10:00 [entrez]
AID - 10.1080/106351502753475853 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Feb;51(1):19-31. doi: 10.1080/106351502753475853.

PMID- 11943089
OWN - NLM
STAT- MEDLINE
DCOM- 20020722
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 1
DP  - 2002 Feb
TI  - Combined data, Bayesian phylogenetics, and the origin of the New Zealand cicada
      genera.
PG  - 4-18
AB  - We have applied Bayesian and maximum likelihood methods of phylogenetic
      estimation to data from four mitochondrial genes (COI, COII, 12S, and 16S) and a 
      single nuclear gene (EF1alpha) from several genera of New Zealand, Australian,
      and New Caledonian cicada taxa. We specifically focused on the heterogeneity of
      phylogenetic signal among the different data partitions and the biogeographic
      origins of the New Zealand cicada fauna. The Bayesian analyses circumvent many of
      the problems associated with other statistical tests for comparing data
      partitions. We took an information-theoretic approach to model selection based on
      the Akaike Information Criterion (AIC). This approach indicated that there was
      considerable uncertainty in identifying the best-fit model for some of the
      partitions. Additionally, a large amount of uncertainty was associated with many 
      parameter estimates from the substitution model. However, a sensitivity analysis 
      on the combined dataset indicated that the model selection uncertainty had little
      effect on estimates of topology because these estimates were largely insensitive 
      to changes in the assumed model. This outcome suggests strong signal in our data.
      Our analyses support a New Caledonian affiliation of the New Zealand cicada
      genera Maoricicada, Kikihia, and Rhodopsalta and Australian affinities for the
      genera Amphipsalta and Notopsalta. This result was surprising, given that
      previous cicada biologists suspected a close relationship between Amphipsalta,
      Notopsalta, and Rhodopsalta based on genitalic characters. Relationships among
      the closely related genera Maoricicada, Kikihia, and Rhodopsalta were poorly
      resolved, the mitochondrial data and the EF1alpha data favoring different
      arrangements within this clade.
FAU - Buckley, Thomas R
AU  - Buckley TR
AD  - Institute for Molecular Systematics, School of Biological Sciences victoria
      Univesity of Wellinton P.O. Box 600, Wellington, New Zealand.
      BuckleyT@landcare.cri.nz
FAU - Arensburger, Peter
AU  - Arensburger P
FAU - Simon, Chris
AU  - Simon C
FAU - Chambers, Geoffrey K
AU  - Chambers GK
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - DNA, Mitochondrial/genetics
MH  - Electron Transport Complex IV/genetics
MH  - Evolution, Molecular
MH  - Genes, Insect
MH  - Genetic Variation
MH  - Hemiptera/*classification/enzymology/*genetics
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - New Zealand
MH  - Phylogeny
MH  - Species Specificity
EDAT- 2002/04/12 10:00
MHDA- 2002/07/23 10:01
CRDT- 2002/04/12 10:00
PHST- 2002/04/12 10:00 [pubmed]
PHST- 2002/07/23 10:01 [medline]
PHST- 2002/04/12 10:00 [entrez]
AID - 10.1080/106351502753475844 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Feb;51(1):4-18. doi: 10.1080/106351502753475844.

PMID- 11975346
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 2
DP  - 1997 Jun
TI  - On the consistency of maximum likelihood estimation of phylogenetic trees from
      nucleotide sequences.
PG  - 354-7
FAU - Rogers, J S
AU  - Rogers JS
AD  - Department of Biological Sciences, University of New Orleans, New Orleans,
      Louisiana 70148, USA. jsrbs@uno.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - *Likelihood Functions
MH  - Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Sequence Analysis/statistics &amp; numerical data
EDAT- 1997/06/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/06/01 00:00
PHST- 1997/06/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/06/01 00:00 [entrez]
AID - 10.1093/sysbio/46.2.354 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Jun;46(2):354-7. doi: 10.1093/sysbio/46.2.354.

PMID- 11975345
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 2
DP  - 1997 Jun
TI  - Site-by-site estimation of the rate of substitution and the correlation of rates 
      in mitochondrial DNA.
PG  - 346-53
AB  - A maximum likelihood method for independently estimating the relative rate of
      substitution at different nucleotide sites is presented. With this method, the
      evolution of DNA sequences can be analyzed without assuming a specific
      distribution of rates among sites. To investigate the pattern of correlation of
      rates among sites, the method was applied to a data set consisting of the
      protein-coding regions of the mitochondrial genome from 10 vertebrate species.
      Rates appear to be strongly correlated at distances up to 40 codons apart.
      Furthermore, there appears to be some higher order correlation of sites
      approximately 75 codons apart. The method of site-by-site estimation of the rate 
      of substitution may also be applied to examine other aspects of rate variation
      along a DNA sequence and to assess the difference in the support of a tree along 
      the sequence.
FAU - Nielsen, R
AU  - Nielsen R
AD  - Department of Integrative Biology, University of California, Berkeley, California
      94720, USA. rasmus@mws4.biol.berkeley.edu
LA  - eng
GR  - GM40282/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - DNA, Mitochondrial/*genetics
MH  - Evolution, Molecular
MH  - Genetic Techniques
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Vertebrates/classification/genetics
EDAT- 1997/06/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/06/01 00:00
PHST- 1997/06/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/06/01 00:00 [entrez]
AID - 10.1093/sysbio/46.2.346 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Jun;46(2):346-53. doi: 10.1093/sysbio/46.2.346.

PMID- 11975344
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 2
DP  - 1997 Jun
TI  - Molecular systematics of short-horned lizards: biogeography and taxonomy of a
      widespread species complex.
PG  - 284-305
AB  - We surveyed mitochondrial DNA (mtDNA) sequence variation in short-horned lizards 
      (Phrynosoma douglasi) from throughout western North America and used these data
      to estimate an intraspecific phylogeny and to assess biogeographic scenarios
      underlying the geographic structure of lineages in this species. We sequenced 783
      base pairs from 38 populations of P. douglasi and three putative outgroups (P.
      ditmarsi, P. orbiculare, P. platyrhinos). We detected high levels of nucleotide
      variation among populations and a spatial distribution of mtDNA lineages
      compatible with major geographic regions. The phylogenetic hypotheses best
      supported by the data suggest that P. douglasi, as currently described, is
      paraphyletic with respect to P. ditmarsi. Populations of P. douglasi from the
      Pacific Northwest (ID, CA, OR, WA) form a monophyletic group that is sister to
      the subsequent radiation of P. ditmarsi and other P. douglasi clades. These
      results suggest that divergences within this widespread species are fairly old.
      We focused on the genetic structure of populations of P. douglasi from a
      geographic perspective and interpreted the intraspecific phylogeny in light of
      geologic and climatic changes in western North America during the last 20 million
      years. The generally high levels of genetic variation found in these population
      comparisons are in accord with high levels of morphological variation in this
      species group; however, only in the Pacific Northwest region is there spatial
      congruence between these phylogenetic results and subspecific ranges based on
      previous morphological studies. We compared the evolutionary units delineated in 
      this study with previously described subspecies of P. douglasi and evaluated the 
      support (from morphology and mtDNA) for each population lineage in the phylogeny 
      and the implications for the taxonomy of this group.
FAU - Zamudio, K R
AU  - Zamudio KR
AD  - Department of Zoology, Box 351800, University of Washington, Seattle, Washington 
      98195, USA. zamudio@socrates.berkeley.edu
FAU - Jones, K B
AU  - Jones KB
FAU - Ward, R H
AU  - Ward RH
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - DNA, Mitochondrial/genetics
MH  - Environment
MH  - Genetic Variation
MH  - Genetics, Population
MH  - Geography
MH  - Likelihood Functions
MH  - Lizards/*classification/*genetics
MH  - North America
MH  - Phylogeny
EDAT- 1997/06/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/06/01 00:00
PHST- 1997/06/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/06/01 00:00 [entrez]
AID - 10.1093/sysbio/46.2.284 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Jun;46(2):284-305. doi: 10.1093/sysbio/46.2.284.

PMID- 11975343
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 2
DP  - 1997 Jun
TI  - A new nuclear gene for insect phylogenetics: dopa decarboxylase is informative of
      relationships within Heliothinae (Lepidoptera: Noctuidae).
PG  - 269-83
AB  - The lack of a readily accessible roster of nuclear genes informative at various
      taxonomic levels is a bottleneck for molecular systematics. In this report, we
      describe the first phylogenetic application of the sequence that encodes the
      enzyme dopa decarboxylase (DDC). For 14 test species within the noctuid moth
      subfamily Heliothinae that represent the previously best-supported groupings, a
      690-bp fragment of DDC resolved relationships that are largely concordant with
      prior evidence from elongation factor-1 alpha (EF-1 alpha), morphology, and
      allozymes. Although both synonymous and nonsynonymous changes occur in DDC
      substantially more rapidly than they do in EF-1 alpha, DDC divergences within
      Heliothinae are below saturation at all codon positions. Analysis of DDC and EF-1
      alpha in combination resulted in increased bootstrap support for several
      groupings. As a first estimate of previously unresolved relationships, DDC
      sequences were analyzed from 16 additional heliothines, for a total of 30
      heliothine species plus outgroups. Previous relationships based on DDC were
      generally stable with increased taxon sampling, although a two- to eightfold
      downweighting of codon position 3 was required for complete concordance with the 
      14-species result. The weighted strict consensus trees were largely resolved and 
      were congruent with most although not all previous hypotheses based on either
      morphology or EF-1 alpha. The proposed phylogeny suggests that the major
      agricultural pest heliothines belong to a single clade, characterized by
      polyphagy and associated life history traits, within this largely host-specific
      moth subfamily. DDC holds much promise for phylogenetic analysis of Tertiary-age 
      animal groups.
FAU - Fang, Q Q
AU  - Fang QQ
AD  - Center for Agricultural Biotechnology, University of Maryland Biotechnology
      Institute, College Park, Maryland 20742, USA. qfang@gasou.edu
FAU - Cho, S
AU  - Cho S
FAU - Regier, J C
AU  - Regier JC
FAU - Mitter, C
AU  - Mitter C
FAU - Matthews, M
AU  - Matthews M
FAU - Poole, R W
AU  - Poole RW
FAU - Friedlander, T P
AU  - Friedlander TP
FAU - Zhao, S
AU  - Zhao S
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Peptide Elongation Factor 1)
RN  - 9007-49-2 (DNA)
RN  - EC 4.1.1.- (Dopa Decarboxylase)
SB  - IM
MH  - Animals
MH  - DNA/genetics
MH  - Dopa Decarboxylase/genetics
MH  - *Genes, Insect
MH  - Moths/classification/*enzymology/*genetics
MH  - Peptide Elongation Factor 1/genetics
MH  - *Phylogeny
EDAT- 1997/06/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/06/01 00:00
PHST- 1997/06/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/06/01 00:00 [entrez]
AID - 10.1093/sysbio/46.2.269 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Jun;46(2):269-83. doi: 10.1093/sysbio/46.2.269.

PMID- 11975342
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 2
DP  - 1997 Jun
TI  - Tests of turtle phylogeny: molecular, morphological, and paleontological
      approaches.
PG  - 235-68
AB  - We present phylogenetic analyses of both molecular and morphological data for the
      23 major lineages of living turtles and seven key fossil taxa. Nearly 1 kilobase 
      of cytochrome b sequence, 325 base pairs of 12S ribosomal DNA, and 115
      morphological characters contained similar phylogenetic information, although
      each provided unique information on different nodes of chelonian history. A
      character-based combinability test (implemented in PAUP*) and a non-parametric
      test of taxonomic congruence indicated no strong evidence for heterogeneity among
      data sets, and we used a combined approach to estimate a final phylogeny of the
      major lineages of living turtles. This approach resulted in a very well-resolved 
      tree, with only a few of the deep branches within the Cryptodira left as an
      unresolved polytomy. The addition of six relatively complete fossils chosen to
      help resolve this basal polytomy provided little added resolution to the tree and
      resulted in a sharp decline in bootstrap proportions for nodes near the fossils. 
      Branch-length analysis and independent dates from the fossil record suggest that 
      these unresolved nodes may represent a rapid radiation of the major cryptodiran
      lineages 90-120 million years ago.
FAU - Shaffer, H B
AU  - Shaffer HB
AD  - Section of Evolution and Ecology and Center for Populatiion Biology, University
      of California, Davis, California 95616, USA. hbshaffer@ucdavis.edu
FAU - Meylan, P
AU  - Meylan P
FAU - McKnight, M L
AU  - McKnight ML
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA, Mitochondrial/genetics
MH  - DNA, Ribosomal/genetics
MH  - Fossils
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Paleontology
MH  - *Phylogeny
MH  - Sequence Homology, Nucleic Acid
MH  - Turtles/anatomy &amp; histology/*classification/*genetics
EDAT- 1997/06/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/06/01 00:00
PHST- 1997/06/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/06/01 00:00 [entrez]
AID - 10.1093/sysbio/46.2.235 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Jun;46(2):235-68. doi: 10.1093/sysbio/46.2.235.

PMID- 11975355
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 1
DP  - 1997 Mar
TI  - The radiation of characiform fishes and the limits of resolution of mitochondrial
      ribosomal DNA sequences.
PG  - 75-100
AB  - Phylogenetic relationships among fishes from ostariophysan orders, characiform
      families, and serrasalmin genera (e.g., "piranhas") were examined using partial
      mitochondrial ribosomal DNA sequences of the 12S and 16S genes. Phylogenetic
      information content of these sequences was assessed at three levels of taxonomic 
      inclusiveness by analyzing the patterns of nucleotide substitution using
      secondary structure models. Conserved and variable regions were identified,
      mapped onto the structural models, and compared at increasing levels of taxonomic
      divergence. In general, loop regions (unpaired) exhibited a higher level of
      variation than did stem regions (paired). A high proportion of compensatory
      substitutions was observed in stem regions in three data sets, suggesting strong 
      selection to maintain the secondary structure. Saturation due to multiple
      substitutions was indicated by decreasing transition/transversion ratios and
      strong structural constraints on variation in comparisons among orders of
      Ostariophysi but was not obvious among families of Characiformes and was not
      detected among serrasalmin genera. Reliable phylogenetic signal successfully
      reconstructed relationships among serrasalmin genera. However, aside from a few
      well-supported clades, relationships could not be reconstructed with confidence
      among characiform families and ostariophysan orders. The reciprocal monophyly of 
      African and Neotropical characiform lineages was rejected (based on maximum
      likelihood ratio tests), and some support for previous hypotheses based on
      morphology was provided by the molecular data. The radiation of characiform
      fishes is discussed in a historical biogeographic context.
FAU - Orti, G
AU  - Orti G
AD  - Department of Ecology and Evolution, State University of New York, Stony Brook,
      New York 11794-5245, USA. gorti@bscr.uga.edu
FAU - Meyer, A
AU  - Meyer A
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (DNA, Ribosomal)
RN  - 0 (RNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 16S)
RN  - 0 (RNA, ribosomal, 12S)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA, Mitochondrial/chemistry/*genetics
MH  - DNA, Ribosomal/chemistry/*genetics
MH  - Fishes/*classification/*genetics
MH  - Genetic Variation
MH  - Models, Molecular
MH  - Molecular Sequence Data
MH  - Nucleic Acid Conformation
MH  - RNA, Ribosomal/chemistry/genetics
MH  - RNA, Ribosomal, 16S/chemistry/genetics
MH  - Sequence Alignment
EDAT- 1997/03/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/03/01 00:00
PHST- 1997/03/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/03/01 00:00 [entrez]
AID - 10.1093/sysbio/46.1.75 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Mar;46(1):75-100. doi: 10.1093/sysbio/46.1.75.

PMID- 11975354
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 1
DP  - 1997 Mar
TI  - Is the Felsenstein zone a fly trap?
PG  - 69-74
AB  - Although long-branch attraction, the incorrect grouping of long lineages in a
      phylogeny because of systematic error, has been identified as a potential source 
      of error in phylogenetic analysis for almost two decades, no empirical examples
      of the phenomenon exist. Here, I outline several criteria for identifying
      long-branch attraction and apply these criteria to 18S ribosomal DNA (rDNA)
      sequence data for 13 insects. Parsimony and minimum evolution with p distances
      group the two longest branches together (those leading to Strepsiptera and
      Diptera). Simulation studies show that the long branches are long enough to
      attract. When a tree is assumed in which Strepsiptera and Diptera are separated
      and many data sets are simulated for that tree (using the parameter estimates for
      that tree for the original data), parsimony analysis of the simulated data
      consistently groups Strepsiptera and Diptera. Analyses of the 18S rDNA sequences 
      using methods that are less sensitive to the problem of long-branch attraction
      estimate trees in which the long branches are separate.
FAU - Huelsenbeck, J P
AU  - Huelsenbeck JP
AD  - Department of Integrative Biology, University of California, Berkeley, California
      94720, USA. johnh@mws4.biol.berkeley.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 18S)
SB  - IM
MH  - Animals
MH  - DNA, Ribosomal/genetics
MH  - Evolution, Molecular
MH  - Insecta/*classification/*genetics
MH  - *Phylogeny
MH  - RNA, Ribosomal, 18S/genetics
EDAT- 1997/03/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/03/01 00:00
PHST- 1997/03/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/03/01 00:00 [entrez]
AID - 10.1093/sysbio/46.1.69 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Mar;46(1):69-74. doi: 10.1093/sysbio/46.1.69.

PMID- 11975353
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 1
DP  - 1997 Mar
TI  - Searching for phylogenetic trees under the frequency parsimony criterion: an
      approximation using generalized parsimony.
PG  - 211-5
FAU - Berlocher, S H
AU  - Berlocher SH
AD  - Department of Entomology/320 Morrill Hall, University of Illinois, 505 S. Goodwin
      Avenue, Urbana, Illinois 61801, USA. stewartb@uiuc.edu
FAU - Swofford, D L
AU  - Swofford DL
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Data Interpretation, Statistical
MH  - Diptera/classification/genetics
MH  - Fishes/classification/genetics
MH  - Models, Statistical
MH  - *Phylogeny
EDAT- 1997/03/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/03/01 00:00
PHST- 1997/03/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/03/01 00:00 [entrez]
AID - 10.1093/sysbio/46.1.211 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Mar;46(1):211-5. doi: 10.1093/sysbio/46.1.211.

PMID- 11975352
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 1
DP  - 1997 Mar
TI  - Analyzing developmental sequences using sequence units.
PG  - 204-10
FAU - Velhagen, W A Jr
AU  - Velhagen WA Jr
AD  - Department of Zoology, Duke University, Box 90325, Durham, North Carolina
      27708-0325, USA. velhagwa@jmu.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Bone Development
MH  - Developmental Biology/*statistics &amp; numerical data
MH  - Fishes/classification/genetics/growth &amp; development
MH  - Snakes/classification/genetics/growth &amp; development
EDAT- 1997/03/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/03/01 00:00
PHST- 1997/03/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/03/01 00:00 [entrez]
AID - 10.1093/sysbio/46.1.204 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Mar;46(1):204-10. doi: 10.1093/sysbio/46.1.204.

PMID- 11975351
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 1
DP  - 1997 Mar
TI  - Evolution of male tail development in rhabditid nematodes related to
      Caenorhabditis elegans.
PG  - 145-79
AB  - The evolutionary pathway that has led to male tails of diverse morphology among
      species of the nematode family Rhabditidae was reconstructed. This family
      includes the well-studied model species Caenorhabditis elegans. By relating the
      steps of male tail morphological evolution to the phenotypic changes brought
      about by developmental mutations induced experimentally in C. elegans, the goal
      is to identify genes responsible for morphological evolution. The varying
      morphological characters of the male tails of several rhabiditid species have
      been described previously (Fitch and Emmons, 1995, Dev. Biol. 170:564-582). The
      developmental events preceding differentiation of the adult structures have also 
      been analyzed; in many cases the origins of varying adult morphological
      characters were traced to differences during ontogeny. In the present work, the
      evolutionary changes producing these differences were reconstructed in the
      context of the four possible phylogenies supported independently by sequences of 
      18S ribosomal RNA genes (rDNA). Two or more alternative states were defined for
      36 developmental and adult morphological characters. These characters alone do
      not provide sufficient data to resolve most species relationships; however, when 
      combined with the rDNA characters, they provide stronger support for one of the
      four rDNA phylogenies. Assuming a model of ordered transformations for multistate
      developmental characters generally results in greater resolution. Transformations
      between character states can be assigned unequivocally by parsimony to
      unambiguous branches for most of the characters. Correlations are thereby
      revealed for some of the developmental characters, indicating a probability of a 
      shared developmental or genetic regulatory pathway. Four of the unequivocal
      character state changes on unambiguously supported branches closely resemble the 
      phenotypic changes brought about by known mutations in C. elegans. These
      mutations define genes that are known to act in genetic regulatory hierarchies
      controlling pattern formation, differentiation, and morphogenesis. Although these
      studies are still at an early stage, these results strongly suggest that parallel
      studies of developmental mutants in C. elegans and of morphological and
      developmental evolution among related nematodes will help define genetic changes 
      underlying the evolution of form.
FAU - Fitch, D H
AU  - Fitch DH
AD  - Department of Biology, New York University, Room 1009 Main Building, 100
      Washington Square East, New York, New York 10003, USA. fitch@acf2.nyu.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Helminth)
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Caenorhabditis elegans/anatomy &amp; histology/*genetics/*growth &amp; development
MH  - DNA, Helminth/genetics
MH  - DNA, Ribosomal/genetics
MH  - Male
MH  - Mutation
MH  - Phenotype
MH  - Phylogeny
MH  - Rhabditida/anatomy &amp; histology/classification/*genetics/*growth &amp; development
MH  - Tail/anatomy &amp; histology/*growth &amp; development
EDAT- 1997/03/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/03/01 00:00
PHST- 1997/03/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/03/01 00:00 [entrez]
AID - 10.1093/sysbio/46.1.145 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Mar;46(1):145-79. doi: 10.1093/sysbio/46.1.145.

PMID- 11975350
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 1
DP  - 1997 Mar
TI  - Mitochondrial DNA rates and biogeography in European newts (genus Euproctus).
PG  - 126-44
AB  - Sequence divergence for segments of three mitochondrial DNA (mtDNA) genes
      encoding the 12S and 16S ribosomal RNA and cytochrome b was examined in newts
      belonging to the genus Euproctus (E. asper, E. montanus, E. platycephalus) and in
      three other species belonging to the same family (Salamandridae), Triturus
      carnifex, T. vulgaris, and Pleurodeles waltl. The three Euproctus species occur
      (one species each) in Corsica, Sardinia, and the Pyrenees. This vicariant
      distribution is believed to have been determined by the disjunction and rotation 
      of the Sardinia-Corsica microplate from the Pyreneean region. Because time
      estimates are available for the tectonic events that led to the separation of the
      three landmasses, we used sequence data to estimate rates of evolution for the
      three gene fragments and investigated whether they conform to the rate-constancy 
      hypothesis. By the Tajima (1993, Genetics 135:599-607) test, we could not detect 
      rate heterogeneities for the ribosomal genes and for transversions in the
      cytochrome b gene. Assuming that these sites are evolving linearly over time and 
      that cessation of gene flow occurred simultaneously with vicariant events, we
      compared the time of divergence estimated by molecular distances with the
      divergence times based on the geological estimates. Because we have two estimates
      of divergence time from the geological record, the split of Corsica/Sardinia from
      the Pyrenees and the split of Corsica from Sardinia, we could compare ratios of
      molecular divergence with the ratio of geological time divergence. The ratios are
      very similar, indicating that the molecular clock hypothesis cannot be rejected. 
      These geological events also allowed us to calculate absolute rates of evolution 
      for ribosomal and cytochrome b genes and compare them to rates for the same
      regions in other salamandrids and other vertebrates. Ribosomal mtDNA rates are
      comparable to those reported for other vertebrates, but cytochrome b rates are
      3-7 times lower in salamanders than in other ectotherms. From a phylogenetic
      perspective, our data suggest that the cladogenic events leading to species
      formation in Euproctus and Triturus occurred very closely in time, indicating
      that the two genera may not be monophyletic. A duplication of the cytochrome b
      gene in T. carnifex was found, and the implications of this finding for mtDNA
      phylogenetic studies are discussed.
FAU - Caccone, A
AU  - Caccone A
AD  - Dipartimento di Biologia, Universita di Roma Tor Vergata, 00133 Rome, Italy.
      caccone@beagle.biology.yale.edu
FAU - Milinkovitch, M C
AU  - Milinkovitch MC
FAU - Sbordoni, V
AU  - Sbordoni V
FAU - Powell, J R
AU  - Powell JR
LA  - eng
SI  - GENBANK/U04694
SI  - GENBANK/U04695
SI  - GENBANK/U04696
SI  - GENBANK/U04697
SI  - GENBANK/U04698
SI  - GENBANK/U04699
SI  - GENBANK/U04700
SI  - GENBANK/U04701
SI  - GENBANK/U04702
SI  - GENBANK/U04703
SI  - GENBANK/U04704
SI  - GENBANK/U04705
SI  - GENBANK/U55945
SI  - GENBANK/U55946
SI  - GENBANK/U55947
SI  - GENBANK/U55948
SI  - GENBANK/U55949
SI  - GENBANK/U55950
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (RNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 16S)
RN  - 0 (RNA, ribosomal, 12S)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Cytochrome b Group/genetics
MH  - DNA, Mitochondrial/*genetics
MH  - Europe
MH  - *Evolution, Molecular
MH  - Genetic Variation
MH  - Geography
MH  - Mediterranean Region
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - RNA, Ribosomal/genetics
MH  - RNA, Ribosomal, 16S/genetics
MH  - Salamandridae/*classification/*genetics
MH  - Sequence Homology, Nucleic Acid
MH  - Time Factors
EDAT- 1997/03/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/03/01 00:00
PHST- 1997/03/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/03/01 00:00 [entrez]
AID - 10.1093/sysbio/46.1.126 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Mar;46(1):126-44. doi: 10.1093/sysbio/46.1.126.

PMID- 11975349
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 1
DP  - 1997 Mar
TI  - Vagaries in the delimitation of character states in quantitative variation--an
      experimental study.
PG  - 112-25
AB  - An experimental study on the delimitation of character states in continuous
      variation indicates that (1) the way data are presented influences the assignment
      of character states and (2) states in the same data set are delimited in various 
      ways by different individuals. Forty-nine individuals were given a set of graphs 
      denoting variation of 10 characters in the genus Kalmia (Ericaceae) and
      outgroups, all identification having been removed from the graphs. The variation 
      was represented in one of three ways: as 95% confidence intervals on a linear
      scale, as 95% confidence intervals on a log10 scale, or with bars showing SD x 2 
      on a linear scale. No two individuals scored a set of graphs in the same way, and
      only one character in one representation was scored identically by all
      individuals; the scoring for this character was completely different when the
      ordinate was changed from linear to logarithmic. Together, the 49 individuals
      delimited states within each character between 9 and 16 different ways. In
      general, variation represented by 2 x SD bars elicited the largest numbers of
      different scorings, yet with a relatively low number of states; the complexity of
      the patterns in the graphs in this representation was greatest. Expert knowledge 
      appears to be of dubious value in delimiting states in such variation, and if
      such characters are to be used in phylogenetic analyses, states could be
      delimited by people who know nothing of the details of the study being scored; in
      any case, presentation of data and an explicit protocol to follow when delimiting
      states are essential. In converting data of this type into character states,
      psychological factors are particularly likely to come into play. Other
      implications of our experiments include the severe underdetermination of some
      phylogenetic hypotheses by observation and the heterogeneous nature of
      morphological data.
FAU - Gift, N
AU  - Gift N
AD  - Department of Crop Science, University of Kentucky, N212 Agricultural Science
      North, Lexington, Kentucky 40546, USA.
FAU - Stevens, P F
AU  - Stevens PF
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Data Interpretation, Statistical
MH  - Ericaceae/anatomy &amp; histology/classification/genetics
MH  - *Genetic Variation
MH  - *Phylogeny
EDAT- 1997/03/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/03/01 00:00
PHST- 1997/03/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/03/01 00:00 [entrez]
AID - 10.1093/sysbio/46.1.112 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Mar;46(1):112-25. doi: 10.1093/sysbio/46.1.112.

PMID- 11975348
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 1
DP  - 1997 Mar
TI  - An alternating least squares approach to inferring phylogenies from pairwise
      distances.
PG  - 101-11
AB  - A computational method is presented for minimizing the weighted sum of squares of
      the differences between observed and expected pairwise distances between species,
      where the expectations are generated by an additive tree model. The criteria of
      Fitch and Margoliash (1967, Science 155:279-284) and Cavalli-Sforza and Edwards
      (1967, Evolution 21:550-570) are both weighted least squares, with different
      weights. The method presented iterates lengths of adjacent branches in the tree
      three at a time. The weighted sum of squares never increases during the process
      of iteration, and the iterates approach a stationary point on the surface of the 
      sum of squares. This iterative approach makes it particularly easy to maintain
      the constraint that branch lengths never become negative, although negative
      branch lengths can also be allowed. The method is implemented in a computer
      program, FITCH, which has been distributed since 1982 as part of the PHYLIP
      package of programs for inferring phylogenies, and is also implemented in PAUP*. 
      The present method is compared, using some simulated data sets, with an
      implementation of the method of De Soete (1983, Psychometrika 48:621-626); it is 
      slower than De Soete's method but more effective at finding the least squares
      tree. The relationship of this method to the neighbor-joining method is also
      discussed.
FAU - Felsenstein, J
AU  - Felsenstein J
AD  - Department of Genetics, University of Washington, Box 357360, Seattle, Washington
      98195-7360, USA. joe@genetics.washington.edu
LA  - eng
GR  - 1 R01 GM51929-01/GM/NIGMS NIH HHS/United States
GR  - 2 R55 GM41716-04/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Chordata, Nonvertebrate/classification/genetics
MH  - *Least-Squares Analysis
MH  - *Phylogeny
MH  - Software
EDAT- 1997/03/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/03/01 00:00
PHST- 1997/03/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/03/01 00:00 [entrez]
AID - 10.1093/sysbio/46.1.101 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Mar;46(1):101-11. doi: 10.1093/sysbio/46.1.101.

PMID- 11975347
OWN - NLM
STAT- MEDLINE
DCOM- 20020603
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 46
IP  - 1
DP  - 1997 Mar
TI  - The Strepsiptera problem: phylogeny of the holometabolous insect orders inferred 
      from 18S and 28S ribosomal DNA sequences and morphology.
PG  - 1-68
AB  - Phylogenetic relationships among the holometabolous insect orders were inferred
      from cladistic analysis of nucleotide sequences of 18S ribosomal DNA (rDNA) (85
      exemplars) and 28S rDNA (52 exemplars) and morphological characters. Exemplar
      outgroup taxa were Collembola (1 sequence), Archaeognatha (1), Ephemerida (1),
      Odonata (2), Plecoptera (2), Blattodea (1), Mantodea (1), Dermaptera (1),
      Orthoptera (1), Phasmatodea (1), Embioptera (1), Psocoptera (1), Phthiraptera
      (1), Hemiptera (4), and Thysanoptera (1). Exemplar ingroup taxa were Coleoptera: 
      Archostemata (1), Adephaga (2), and Polyphaga (7); Megaloptera (1); Raphidioptera
      (1); Neuroptera (sensu stricto = Planipennia): Mantispoidea (2), Hemerobioidea
      (2), and Myrmeleontoidea (2); Hymenoptera: Symphyta (4) and Apocrita (19);
      Trichoptera: Hydropsychoidea (1) and Limnephiloidea (2); Lepidoptera: Ditrysia
      (3); Siphonaptera: Pulicoidea (1) and Ceratophylloidea (2); Mecoptera: Meropeidae
      (1), Boreidae (1), Panorpidae (1), and Bittacidae (2); Diptera: Nematocera (1),
      Brachycera (2), and Cyclorrhapha (1); and Strepsiptera: Corioxenidae (1),
      Myrmecolacidae (1), Elenchidae (1), and Stylopidae (3). We analyzed approximately
      1 kilobase of 18S rDNA, starting 398 nucleotides downstream of the 5' end, and
      approximately 400 bp of 28S rDNA in expansion segment D3. Multiple alignment of
      the 18S and 28S sequences resulted in 1,116 nucleotide positions with 24 insert
      regions and 398 positions with 14 insert regions, respectively. All Strepsiptera 
      and Neuroptera have large insert regions in 18S and 28S. The secondary structure 
      of 18S insert 23 is composed of long stems that are GC rich in the basal
      Strepsiptera and AT rich in the more derived Strepsiptera. A matrix of 176
      morphological characters was analyzed for holometabolous orders. Incongruence
      length difference tests indicate that the 28S + morphological data sets are
      incongruent but that 28S + 18S, 18S + morphology, and 28S + 18S + morphology fail
      to reject the hypothesis of congruence. Phylogenetic trees were generated by
      parsimony analysis, and clade robustness was evaluated by branch length, Bremer
      support, percentage of extra steps required to force paraphyly, and sensitivity
      analysis using the following parameters: gap weights, morphological character
      weights, methods of data set combination, removal of key taxa, and alignment
      region. The following are monophyletic under most or all combinations of
      parameter values: Holometabola, Polyphaga, Megaloptera + Raphidioptera,
      Neuroptera, Hymenoptera, Trichoptera, Lepidoptera, Amphiesmenoptera (Trichoptera 
      + Lepidoptera), Siphonaptera, Siphonaptera + Mecoptera, Strepsiptera, Diptera,
      and Strepsiptera + Diptera (Halteria). Antliophora (Mecoptera + Diptera +
      Siphonaptera + Strepsiptera), Mecopterida (Antliophora + Amphiesmenoptera), and
      Hymenoptera + Mecopterida are supported in the majority of total evidence
      analyses. Mecoptera may be paraphyletic because Boreus is often placed as sister 
      group to the fleas; hence, Siphonaptera may be subordinate within Mecoptera. The 
      18S sequences for Priacma (Coleoptera: Archostemata), Colpocaccus (Coleoptera:
      Adephaga), Agulla (Raphidioptera), and Corydalus (Megaloptera) are nearly
      identical, and Neuropterida are monophyletic only when those two beetle sequences
      are removed from the analysis. Coleoptera are therefore paraphyletic under almost
      all combinations of parameter values. Halteria and Amphiesmenoptera have high
      Bremer support values and long branch lengths. The data do not support placement 
      of Strepsiptera outside of Holometabola nor as sister group to Coleoptera. We
      reject the notion that the monophyly of Halteria is due to long branch attraction
      because Strepsiptera and Diptera do not have the longest branches and there is
      phylogenetic congruence between molecules, across the entire parameter space, and
      between morphological and molecular data.
FAU - Whiting, M F
AU  - Whiting MF
AD  - Department of Zoology and M. L. Bean Life Science Museum, Brigham Young
      University, Provo, Utah 84602, USA. michael_whiting@byu.edu
FAU - Carpenter, J C
AU  - Carpenter JC
FAU - Wheeler, Q D
AU  - Wheeler QD
FAU - Wheeler, W C
AU  - Wheeler WC
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 18S)
RN  - 0 (RNA, Ribosomal, 28S)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA, Ribosomal/chemistry/*genetics
MH  - Insecta/anatomy &amp; histology/*classification/*genetics
MH  - Molecular Sequence Data
MH  - Nucleic Acid Conformation
MH  - *Phylogeny
MH  - RNA, Ribosomal, 18S/genetics
MH  - RNA, Ribosomal, 28S/genetics
MH  - Sequence Alignment
MH  - Sequence Homology, Nucleic Acid
EDAT- 1997/03/01 00:00
MHDA- 2002/06/04 10:01
CRDT- 1997/03/01 00:00
PHST- 1997/03/01 00:00 [pubmed]
PHST- 2002/06/04 10:01 [medline]
PHST- 1997/03/01 00:00 [entrez]
AID - 10.1093/sysbio/46.1.1 [doi]
PST - ppublish
SO  - Syst Biol. 1997 Mar;46(1):1-68. doi: 10.1093/sysbio/46.1.1.

PMID- 11540538
OWN - NASA
STAT- MEDLINE
DCOM- 19980509
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 45
IP  - 4
DP  - 1996 Dec
TI  - Constraints on protein evolution and the age of the eubacteria/eukaryote split.
PG  - 568-75
FAU - Miyamoto, M M
AU  - Miyamoto MM
AD  - Department of Zoology, University of Florida, Gainesville 32611-8525, USA.
      miyamoto@zoo.ufl.edu
FAU - Fitch, W M
AU  - Fitch WM
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Archaeal Proteins)
RN  - 0 (Bacterial Proteins)
RN  - 0 (Proteins)
SB  - S
MH  - Animals
MH  - Archaea/*classification
MH  - Archaeal Proteins/chemistry
MH  - Bacteria/*classification
MH  - Bacterial Proteins/chemistry
MH  - *Biological Evolution
MH  - Echinodermata
MH  - *Eukaryotic Cells
MH  - *Evolution, Molecular
MH  - Paleontology
MH  - Prokaryotic Cells
MH  - Proteins/*chemistry
MH  - Time Factors
MH  - Vertebrates
OID - NASA: 00021692
OAB - This article discusses research done by Doolittle et al. to determine the
      evolutionary distance between eubacteria and eukaryotes. Several equations for
      estimating evolutionary distances and divergence times are considered. Molecular 
      clock calculations and their effects on these estimates are discussed. The
      authors conclude that their research supports the range of dates for the
      cenancestor of modern eubacteria, archaebacteria, and eukaryotes estimated by
      Doolittle et al.
OABL- eng
EDAT- 1996/12/01 00:00
MHDA- 2001/09/11 10:01
CRDT- 1996/12/01 00:00
PHST- 1996/12/01 00:00 [pubmed]
PHST- 2001/09/11 10:01 [medline]
PHST- 1996/12/01 00:00 [entrez]
AID - 10.1093/sysbio/45.4.568 [doi]
PST - ppublish
SO  - Syst Biol. 1996 Dec;45(4):568-75. doi: 10.1093/sysbio/45.4.568.
</pre></body></html>