<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0035)https://www.ncbi.nlm.nih.gov/pubmed -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"></head><body><pre>PMID- 25239212
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20150217
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - Phylogeny, paleontology, and primates: do incomplete fossils bias the tree of
      life?
PG  - 169-86
LID - 10.1093/sysbio/syu077 [doi]
AB  - Paleontological systematics relies heavily on morphological data that have
      undergone decay and fossilization. Here, we apply a heuristic means to assess how
      a fossil's incompleteness detracts from inferring its phylogenetic relationships.
      We compiled a phylogenetic matrix for primates and simulated the extinction of
      living species by deleting an extant taxon's molecular data and keeping only
      those morphological characters present in actual fossils. The choice of
      characters present in a given living taxon (the subject) was defined by those
      present in a given fossil (the template). By measuring congruence between a
      well-corroborated phylogeny to those incorporating artificial fossils, and by
      comparing real vs. random character distributions and states, we tested the
      information content of paleontological datasets and determined if extinction of a
      living species leads to bias in phylogeny reconstruction. We found a positive
      correlation between fossil completeness and topological congruence. Real fossil
      templates sampled for 36 or more of the 360 available morphological characters
      (including dental) performed significantly better than similarly complete
      templates with random states. Templates dominated by only one partition performed
      worse than templates with randomly sampled characters across partitions. The
      template based on the Eocene primate Darwinius masillae performs better than most
      other templates with a similar number of sampled characters, likely due to
      preservation of data across multiple partitions. Our results support the
      interpretation that Darwinius is strepsirhine, not haplorhine, and suggest that
      paleontological datasets are reliable in primate phylogeny reconstruction.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Pattinson, David J
AU  - Pattinson DJ
AD  - Department of Zoology, Downing Street, Cambridge, CB2 3EJ; Division of Ecology
      and Evolution, Imperial College London, South Kensington Campus, London, SW7 2AZ;
      Department of Life Sciences, Natural History Museum, Cromwell Road, London, SW7
      5BD; and Department of Physics, Cavendish Laboratory, JJ Thomson Avenue,
      Cambridge, CB3 0HE, UK Department of Zoology, Downing Street, Cambridge, CB2 3EJ;
      Division of Ecology and Evolution, Imperial College London, South Kensington
      Campus, London, SW7 2AZ; Department of Life Sciences, Natural History Museum,
      Cromwell Road, London, SW7 5BD; and Department of Physics, Cavendish Laboratory, 
      JJ Thomson Avenue, Cambridge, CB3 0HE, UK Department of Zoology, Downing Street, 
      Cambridge, CB2 3EJ; Division of Ecology and Evolution, Imperial College London,
      South Kensington Campus, London, SW7 2AZ; Department of Life Sciences, Natural
      History Museum, Cromwell Road, London, SW7 5BD; and Department of Physics,
      Cavendish Laboratory, JJ Thomson Avenue, Cambridge, CB3 0HE, UK.
FAU - Thompson, Richard S
AU  - Thompson RS
AD  - Department of Zoology, Downing Street, Cambridge, CB2 3EJ; Division of Ecology
      and Evolution, Imperial College London, South Kensington Campus, London, SW7 2AZ;
      Department of Life Sciences, Natural History Museum, Cromwell Road, London, SW7
      5BD; and Department of Physics, Cavendish Laboratory, JJ Thomson Avenue,
      Cambridge, CB3 0HE, UK.
FAU - Piotrowski, Aleks K
AU  - Piotrowski AK
AD  - Department of Zoology, Downing Street, Cambridge, CB2 3EJ; Division of Ecology
      and Evolution, Imperial College London, South Kensington Campus, London, SW7 2AZ;
      Department of Life Sciences, Natural History Museum, Cromwell Road, London, SW7
      5BD; and Department of Physics, Cavendish Laboratory, JJ Thomson Avenue,
      Cambridge, CB3 0HE, UK.
FAU - Asher, Robert J
AU  - Asher RJ
AD  - Department of Zoology, Downing Street, Cambridge, CB2 3EJ; Division of Ecology
      and Evolution, Imperial College London, South Kensington Campus, London, SW7 2AZ;
      Department of Life Sciences, Natural History Museum, Cromwell Road, London, SW7
      5BD; and Department of Physics, Cavendish Laboratory, JJ Thomson Avenue,
      Cambridge, CB3 0HE, UK r.asher@zoo.cam.ac.uk djp65@cam.ac.uk.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140919
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Extinction, Biological
MH  - *Fossils
MH  - Paleontology/standards
MH  - *Phylogeny
MH  - Primates/anatomy &amp; histology/*classification
OTO - NOTNLM
OT  - Artificial extinction
OT  - Darwinius
OT  - data combination
OT  - missing data
OT  - phylogeny reconstruction
OT  - primates
OT  - taphonomy
EDAT- 2014/09/23 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/09/21 06:00
PHST- 2014/09/21 06:00 [entrez]
PHST- 2014/09/23 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu077 [pii]
AID - 10.1093/sysbio/syu077 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):169-86. doi: 10.1093/sysbio/syu077. Epub 2014 Sep 19.

PMID- 25236959
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - How much information is needed to infer reticulate evolutionary histories?
PG  - 102-11
LID - 10.1093/sysbio/syu076 [doi]
AB  - Phylogenetic networks are a generalization of evolutionary trees and are an
      important tool for analyzing reticulate evolutionary histories. Recently, there
      has been great interest in developing new methods to construct rooted
      phylogenetic networks, that is, networks whose internal vertices correspond to
      hypothetical ancestors, whose leaves correspond to sampled taxa, and in which
      vertices with more than one parent correspond to taxa formed by reticulate
      evolutionary events such as recombination or hybridization. Several methods for
      constructing evolutionary trees use the strategy of building up a tree from
      simpler building blocks (such as triplets or clusters), and so it is natural to
      look for ways to construct networks from smaller networks. In this article, we
      shall demonstrate a fundamental issue with this approach. Namely, we show that
      even if we are given all of the subnetworks induced on all proper subsets of the 
      leaves of some rooted phylogenetic network, we still do not have all of the
      information required to completely determine that network. This implies that even
      if all of the building blocks for some reticulate evolutionary history were to be
      taken as the input for any given network building method, the method might still 
      output an incorrect history. We also discuss some potential consequences of this 
      result for constructing phylogenetic networks.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Huber, Katharina T
AU  - Huber KT
AD  - School of Computing Sciences, University of East Anglia, Norwich, UK, and Centrum
      Wiskunde &amp; Informatica (CWI), Amsterdam, Netherlands.
FAU - Van Iersel, Leo
AU  - Van Iersel L
AD  - School of Computing Sciences, University of East Anglia, Norwich, UK, and Centrum
      Wiskunde &amp; Informatica (CWI), Amsterdam, Netherlands.
FAU - Moulton, Vincent
AU  - Moulton V
AD  - School of Computing Sciences, University of East Anglia, Norwich, UK, and Centrum
      Wiskunde &amp; Informatica (CWI), Amsterdam, Netherlands
      vincent.moulton@cmp.uea.ac.uk.
FAU - Wu, Taoyang
AU  - Wu T
AD  - School of Computing Sciences, University of East Anglia, Norwich, UK, and Centrum
      Wiskunde &amp; Informatica (CWI), Amsterdam, Netherlands.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140918
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Cryptococcus gattii/classification/genetics
MH  - Models, Theoretical
MH  - *Phylogeny
PMC - PMC4265143
OTO - NOTNLM
OT  - Evolutionary tree
OT  - network reconstruction
OT  - phylogenetic network
OT  - reticulate evolution
EDAT- 2014/09/23 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/09/20 06:00
PHST- 2014/09/20 06:00 [entrez]
PHST- 2014/09/23 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu076 [pii]
AID - 10.1093/sysbio/syu076 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):102-11. doi: 10.1093/sysbio/syu076. Epub 2014 Sep 18.

PMID- 25209223
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20141216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - ModelOMatic: fast and automated model selection between RY, nucleotide, amino
      acid, and codon substitution models.
PG  - 42-55
LID - 10.1093/sysbio/syu062 [doi]
AB  - Molecular phylogenetics is a powerful tool for inferring both the process and
      pattern of evolution from genomic sequence data. Statistical approaches, such as 
      maximum likelihood and Bayesian inference, are now established as the preferred
      methods of inference. The choice of models that a researcher uses for inference
      is of critical importance, and there are established methods for model selection 
      conditioned on a particular type of data, such as nucleotides, amino acids, or
      codons. A major limitation of existing model selection approaches is that they
      can only compare models acting upon a single type of data. Here, we extend model 
      selection to allow comparisons between models describing different types of data 
      by introducing the idea of adapter functions, which project aggregated models
      onto the originally observed sequence data. These projections are implemented in 
      the program ModelOMatic and used to perform model selection on 3722 families from
      the PANDIT database, 68 genes from an arthropod phylogenomic data set, and 248
      genes from a vertebrate phylogenomic data set. For the PANDIT and arthropod data,
      we find that amino acid models are selected for the overwhelming majority of
      alignments; with progressively smaller numbers of alignments selecting codon and 
      nucleotide models, and no families selecting RY-based models. In contrast, nearly
      all alignments from the vertebrate data set select codon-based models. The
      sequence divergence, the number of sequences, and the degree of selection acting 
      upon the protein sequences may contribute to explaining this variation in model
      selection. Our ModelOMatic program is fast, with most families from PANDIT taking
      fewer than 150 s to complete, and should therefore be easily incorporated into
      existing phylogenetic pipelines. ModelOMatic is available at
      https://code.google.com/p/modelomatic/.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Whelan, Simon
AU  - Whelan S
AD  - Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, Uppsala
      75236, Sweden and Faculty of Life Sciences, University of Manchester, Manchester,
      UK Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, Uppsala
      75236, Sweden and Faculty of Life Sciences, University of Manchester, Manchester,
      UK simon.whelan@ebc.uu.se.
FAU - Allen, James E
AU  - Allen JE
AD  - Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, Uppsala
      75236, Sweden and Faculty of Life Sciences, University of Manchester, Manchester,
      UK.
FAU - Blackburne, Benjamin P
AU  - Blackburne BP
AD  - Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, Uppsala
      75236, Sweden and Faculty of Life Sciences, University of Manchester, Manchester,
      UK.
FAU - Talavera, David
AU  - Talavera D
AD  - Evolutionary Biology, Evolutionary Biology Centre, Uppsala University, Uppsala
      75236, Sweden and Faculty of Life Sciences, University of Manchester, Manchester,
      UK.
LA  - eng
GR  - BB/H000445/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
GR  - BB/H006818/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140909
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Amino Acids)
RN  - 0 (Codon)
RN  - 0 (Nucleotides)
SB  - IM
MH  - Amino Acids/genetics
MH  - Animals
MH  - Classification/*methods
MH  - Codon/genetics
MH  - *Models, Biological
MH  - Nucleotides/genetics
MH  - *Phylogeny
MH  - Software
OTO - NOTNLM
OT  - AIC
OT  - model selection
OT  - phylogenetics
OT  - substitution models
EDAT- 2014/09/12 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/09/12 06:00
PHST- 2014/09/12 06:00 [entrez]
PHST- 2014/09/12 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu062 [pii]
AID - 10.1093/sysbio/syu062 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):42-55. doi: 10.1093/sysbio/syu062. Epub 2014 Sep 9.

PMID- 25209222
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20141216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - The unsolved challenge to phylogenetic correlation tests for categorical
      characters.
PG  - 127-36
LID - 10.1093/sysbio/syu070 [doi]
FAU - Maddison, Wayne P
AU  - Maddison WP
AD  - Departments of Zoology and Botany and Beaty Biodiversity Museum, University of
      British Columbia, 6270 University Boulevard, Vancouver, British Columbia, Canada 
      V6T 1Z4; Departments of Zoology and Biodiversity Research Centre, University of
      British Columbia, 6270 University Boulevard, Vancouver, British Columbia, Canada 
      V6T 1Z4 and Department of Biological Sciences, Macquarie University, Sydney NSW
      2109, Australia wayne.maddison@ubc.ca.
FAU - FitzJohn, Richard G
AU  - FitzJohn RG
AD  - Departments of Zoology and Botany and Beaty Biodiversity Museum, University of
      British Columbia, 6270 University Boulevard, Vancouver, British Columbia, Canada 
      V6T 1Z4; Departments of Zoology and Biodiversity Research Centre, University of
      British Columbia, 6270 University Boulevard, Vancouver, British Columbia, Canada 
      V6T 1Z4 and Department of Biological Sciences, Macquarie University, Sydney NSW
      2109, Australia Departments of Zoology and Botany and Beaty Biodiversity Museum, 
      University of British Columbia, 6270 University Boulevard, Vancouver, British
      Columbia, Canada V6T 1Z4; Departments of Zoology and Biodiversity Research
      Centre, University of British Columbia, 6270 University Boulevard, Vancouver,
      British Columbia, Canada V6T 1Z4 and Department of Biological Sciences, Macquarie
      University, Sydney NSW 2109, Australia.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140909
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - *Classification
MH  - Genetic Speciation
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2014/09/12 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/09/12 06:00
PHST- 2014/09/12 06:00 [entrez]
PHST- 2014/09/12 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu070 [pii]
AID - 10.1093/sysbio/syu070 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):127-36. doi: 10.1093/sysbio/syu070. Epub 2014 Sep 9.

PMID- 25190593
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20141216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - Declining rates of species described per taxonomist: slowdown of progress or a
      side-effect of improved quality in taxonomy?
PG  - 144-51
LID - 10.1093/sysbio/syu069 [doi]
FAU - Sangster, George
AU  - Sangster G
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History,
      P.O. Box 50007, SE-104 05 Stockholm, Sweden; Department of Zoology, Stockholm
      University, SE-106 91 Stockholm, Sweden; and Department of Environmental Science 
      and Policy, George Mason University, 4400 University Drive, Fairfax, Virginia
      22030 4444, USA Department of Bioinformatics and Genetics, Swedish Museum of
      Natural History, P.O. Box 50007, SE-104 05 Stockholm, Sweden; Department of
      Zoology, Stockholm University, SE-106 91 Stockholm, Sweden; and Department of
      Environmental Science and Policy, George Mason University, 4400 University Drive,
      Fairfax, Virginia 22030 4444, USA g.sangster@planet.nl.
FAU - Luksenburg, Jolanda A
AU  - Luksenburg JA
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History,
      P.O. Box 50007, SE-104 05 Stockholm, Sweden; Department of Zoology, Stockholm
      University, SE-106 91 Stockholm, Sweden; and Department of Environmental Science 
      and Policy, George Mason University, 4400 University Drive, Fairfax, Virginia
      22030 4444, USA.
LA  - eng
PT  - Journal Article
DEP - 20140903
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Classification
MH  - Research/*standards/*trends
MH  - Species Specificity
EDAT- 2014/09/06 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/09/06 06:00
PHST- 2014/09/06 06:00 [entrez]
PHST- 2014/09/06 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu069 [pii]
AID - 10.1093/sysbio/syu069 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):144-51. doi: 10.1093/sysbio/syu069. Epub 2014 Sep 3.

PMID- 25183706
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - Concatabominations: identifying unstable taxa in morphological phylogenetics
      using a heuristic extension to safe taxonomic reduction.
PG  - 137-43
LID - 10.1093/sysbio/syu066 [doi]
FAU - Siu-Ting, Karen
AU  - Siu-Ting K
AD  - Department of Biology, National University of Ireland, Maynooth, Co. Kildare,
      Ireland; School of Biological Sciences, University of Bristol, Bristol BS8 1TQ,
      UK; Department of Life Sciences, The Natural History Museum, London SW7 5BD, UK; 
      and Institute of Biological, Environmental &amp; Rural Sciences, Aberystwyth
      University, Aberystwyth SY23 3FG, UK Department of Biology, National University
      of Ireland, Maynooth, Co. Kildare, Ireland; School of Biological Sciences,
      University of Bristol, Bristol BS8 1TQ, UK; Department of Life Sciences, The
      Natural History Museum, London SW7 5BD, UK; and Institute of Biological,
      Environmental &amp; Rural Sciences, Aberystwyth University, Aberystwyth SY23 3FG, UK 
      Department of Biology, National University of Ireland, Maynooth, Co. Kildare,
      Ireland; School of Biological Sciences, University of Bristol, Bristol BS8 1TQ,
      UK; Department of Life Sciences, The Natural History Museum, London SW7 5BD, UK; 
      and Institute of Biological, Environmental &amp; Rural Sciences, Aberystwyth
      University, Aberystwyth SY23 3FG, UK agalychnica@gmail.com.
FAU - Pisani, Davide
AU  - Pisani D
AD  - Department of Biology, National University of Ireland, Maynooth, Co. Kildare,
      Ireland; School of Biological Sciences, University of Bristol, Bristol BS8 1TQ,
      UK; Department of Life Sciences, The Natural History Museum, London SW7 5BD, UK; 
      and Institute of Biological, Environmental &amp; Rural Sciences, Aberystwyth
      University, Aberystwyth SY23 3FG, UK.
FAU - Creevey, Christopher J
AU  - Creevey CJ
AD  - Department of Biology, National University of Ireland, Maynooth, Co. Kildare,
      Ireland; School of Biological Sciences, University of Bristol, Bristol BS8 1TQ,
      UK; Department of Life Sciences, The Natural History Museum, London SW7 5BD, UK; 
      and Institute of Biological, Environmental &amp; Rural Sciences, Aberystwyth
      University, Aberystwyth SY23 3FG, UK.
FAU - Wilkinson, Mark
AU  - Wilkinson M
AD  - Department of Biology, National University of Ireland, Maynooth, Co. Kildare,
      Ireland; School of Biological Sciences, University of Bristol, Bristol BS8 1TQ,
      UK; Department of Life Sciences, The Natural History Museum, London SW7 5BD, UK; 
      and Institute of Biological, Environmental &amp; Rural Sciences, Aberystwyth
      University, Aberystwyth SY23 3FG, UK.
LA  - eng
GR  - BB/K007440/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140902
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Phylogeny
PMC - PMC4265141
EDAT- 2014/09/04 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/09/04 06:00
PHST- 2014/09/04 06:00 [entrez]
PHST- 2014/09/04 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu066 [pii]
AID - 10.1093/sysbio/syu066 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):137-43. doi: 10.1093/sysbio/syu066. Epub 2014 Sep 2.

PMID- 25173563
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - Unveiling the diversification dynamics of Australasian predaceous diving beetles 
      in the Cenozoic.
PG  - 3-24
LID - 10.1093/sysbio/syu067 [doi]
AB  - During the Cenozoic, Australia experienced major climatic shifts that have had
      dramatic ecological consequences for the modern biota. Mesic tropical ecosystems 
      were progressively restricted to the coasts and replaced by arid-adapted floral
      and faunal communities. Whilst the role of aridification has been investigated in
      a wide range of terrestrial lineages, the response of freshwater clades remains
      poorly investigated. To gain insights into the diversification processes
      underlying a freshwater radiation, we studied the evolutionary history of the
      Australasian predaceous diving beetles of the tribe Hydroporini (147 described
      species). We used an integrative approach including the latest methods in
      phylogenetics, divergence time estimation, ancestral character state
      reconstruction, and likelihood-based methods of diversification rate estimation. 
      Phylogenies and dating analyses were reconstructed with molecular data from seven
      genes (mitochondrial and nuclear) for 117 species (plus 12 outgroups). Robust and
      well-resolved phylogenies indicate a late Oligocene origin of Australasian
      Hydroporini. Biogeographic analyses suggest an origin in the East Coast region of
      Australia, and a dynamic biogeographic scenario implying dispersal events. The
      group successfully colonized the tropical coastal regions carved by a rampant
      desertification, and also colonized groundwater ecosystems in Central Australia. 
      Diversification rate analyses suggest that the ongoing aridification of Australia
      initiated in the Miocene contributed to a major wave of extinctions since the
      late Pliocene probably attributable to an increasing aridity, range contractions 
      and seasonally disruptions resulting from Quaternary climatic changes. When
      comparing subterranean and epigean genera, our results show that contrasting
      mechanisms drove their diversification and therefore current diversity pattern.
      The Australasian Hydroporini radiation reflects a combination of processes that
      promoted both diversification, resulting from new ecological opportunities driven
      by initial aridification, and a subsequent loss of mesic adapted diversity due to
      increasing aridity.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Toussaint, Emmanuel F A
AU  - Toussaint EF
AD  - SNSB-Bavarian State Collection of Zoology, Munchhausenstrasse 21, 81247 Munich,
      Germany; CNRS, UMR 7641 Centre de Mathematiques Appliquees (Ecole Polytechnique),
      Route de Saclay, 91128 Palaiseau cedex, France; South Australian Museum,
      Adelaide, South Australia, Australia; Centre for Integrated Ecology &amp; School of
      Life and Environmental Sciences, Deakin University, 221 Burwood Highway, Burwood,
      Victoria 3125, Australia; and GeoBioCenter, Ludwig-Maximilians University,
      Munich, Germany. Toussaint@zsm.mwn.de.
FAU - Condamine, Fabien L
AU  - Condamine FL
AD  - SNSB-Bavarian State Collection of Zoology, Munchhausenstrasse 21, 81247 Munich,
      Germany; CNRS, UMR 7641 Centre de Mathematiques Appliquees (Ecole Polytechnique),
      Route de Saclay, 91128 Palaiseau cedex, France; South Australian Museum,
      Adelaide, South Australia, Australia; Centre for Integrated Ecology &amp; School of
      Life and Environmental Sciences, Deakin University, 221 Burwood Highway, Burwood,
      Victoria 3125, Australia; and GeoBioCenter, Ludwig-Maximilians University,
      Munich, Germany.
FAU - Hawlitschek, Oliver
AU  - Hawlitschek O
AD  - SNSB-Bavarian State Collection of Zoology, Munchhausenstrasse 21, 81247 Munich,
      Germany; CNRS, UMR 7641 Centre de Mathematiques Appliquees (Ecole Polytechnique),
      Route de Saclay, 91128 Palaiseau cedex, France; South Australian Museum,
      Adelaide, South Australia, Australia; Centre for Integrated Ecology &amp; School of
      Life and Environmental Sciences, Deakin University, 221 Burwood Highway, Burwood,
      Victoria 3125, Australia; and GeoBioCenter, Ludwig-Maximilians University,
      Munich, Germany.
FAU - Watts, Chris H
AU  - Watts CH
AD  - SNSB-Bavarian State Collection of Zoology, Munchhausenstrasse 21, 81247 Munich,
      Germany; CNRS, UMR 7641 Centre de Mathematiques Appliquees (Ecole Polytechnique),
      Route de Saclay, 91128 Palaiseau cedex, France; South Australian Museum,
      Adelaide, South Australia, Australia; Centre for Integrated Ecology &amp; School of
      Life and Environmental Sciences, Deakin University, 221 Burwood Highway, Burwood,
      Victoria 3125, Australia; and GeoBioCenter, Ludwig-Maximilians University,
      Munich, Germany.
FAU - Porch, Nick
AU  - Porch N
AD  - SNSB-Bavarian State Collection of Zoology, Munchhausenstrasse 21, 81247 Munich,
      Germany; CNRS, UMR 7641 Centre de Mathematiques Appliquees (Ecole Polytechnique),
      Route de Saclay, 91128 Palaiseau cedex, France; South Australian Museum,
      Adelaide, South Australia, Australia; Centre for Integrated Ecology &amp; School of
      Life and Environmental Sciences, Deakin University, 221 Burwood Highway, Burwood,
      Victoria 3125, Australia; and GeoBioCenter, Ludwig-Maximilians University,
      Munich, Germany.
FAU - Hendrich, Lars
AU  - Hendrich L
AD  - SNSB-Bavarian State Collection of Zoology, Munchhausenstrasse 21, 81247 Munich,
      Germany; CNRS, UMR 7641 Centre de Mathematiques Appliquees (Ecole Polytechnique),
      Route de Saclay, 91128 Palaiseau cedex, France; South Australian Museum,
      Adelaide, South Australia, Australia; Centre for Integrated Ecology &amp; School of
      Life and Environmental Sciences, Deakin University, 221 Burwood Highway, Burwood,
      Victoria 3125, Australia; and GeoBioCenter, Ludwig-Maximilians University,
      Munich, Germany.
FAU - Balke, Michael
AU  - Balke M
AD  - SNSB-Bavarian State Collection of Zoology, Munchhausenstrasse 21, 81247 Munich,
      Germany; CNRS, UMR 7641 Centre de Mathematiques Appliquees (Ecole Polytechnique),
      Route de Saclay, 91128 Palaiseau cedex, France; South Australian Museum,
      Adelaide, South Australia, Australia; Centre for Integrated Ecology &amp; School of
      Life and Environmental Sciences, Deakin University, 221 Burwood Highway, Burwood,
      Victoria 3125, Australia; and GeoBioCenter, Ludwig-Maximilians University,
      Munich, Germany. SNSB-Bavarian State Collection of Zoology, Munchhausenstrasse
      21, 81247 Munich, Germany; CNRS, UMR 7641 Centre de Mathematiques Appliquees
      (Ecole Polytechnique), Route de Saclay, 91128 Palaiseau cedex, France; South
      Australian Museum, Adelaide, South Australia, Australia; Centre for Integrated
      Ecology &amp; School of Life and Environmental Sciences, Deakin University, 221
      Burwood Highway, Burwood, Victoria 3125, Australia; and GeoBioCenter,
      Ludwig-Maximilians University, Munich, Germany.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140829
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Australia
MH  - Coleoptera/*classification/genetics
MH  - Fossils
MH  - Genes, Insect/genetics
MH  - Genetic Speciation
MH  - *Genetic Variation
MH  - *Phylogeny
OTO - NOTNLM
OT  - Australian aridification
OT  - Dytiscidae
OT  - Hydroporini
OT  - Pleistocene extinction
OT  - diversification
OT  - freshwater biota
OT  - ground water organisms
EDAT- 2014/09/01 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/09/01 06:00
PHST- 2014/09/01 06:00 [entrez]
PHST- 2014/09/01 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu067 [pii]
AID - 10.1093/sysbio/syu067 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):3-24. doi: 10.1093/sysbio/syu067. Epub 2014 Aug 29.

PMID- 25164916
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - Towards improving searches for optimal phylogenies.
PG  - 56-65
LID - 10.1093/sysbio/syu065 [doi]
AB  - Finding the optimal evolutionary history for a set of taxa is a challenging
      computational problem, even when restricting possible solutions to be "tree-like"
      and focusing on the maximum-parsimony optimality criterion. This has led to much 
      work on using heuristic tree searches to find approximate solutions. We present
      an approach for finding exact optimal solutions that employs and complements the 
      current heuristic methods for finding optimal trees. Given a set of taxa and a
      set of aligned sequences of characters, there may be subsets of characters that
      are compatible, and for each such subset there is an associated (possibly
      partially resolved) phylogeny with edges corresponding to each character state
      change. These perfect phylogenies serve as anchor trees for our constrained
      search space. We show that, for sequences with compatible sites, the parsimony
      score of any tree [Formula: see text] is at least the parsimony score of the
      anchor trees plus the number of inferred changes between [Formula: see text] and 
      the anchor trees. As the maximum-parsimony optimality score is additive, the sum 
      of the lower bounds on compatible character partitions provides a lower bound on 
      the complete alignment of characters. This yields a region in the space of trees 
      within which the best tree is guaranteed to be found; limiting the search for the
      optimal tree to this region can significantly reduce the number of trees that
      must be examined in a search of the space of trees. We analyze this method
      empirically using four different biological data sets as well as surveying 400
      data sets from the TreeBASE repository, demonstrating the effectiveness of our
      technique in reducing the number of steps in exact heuristic searches for trees
      under the maximum-parsimony optimality criterion.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Ford, Eric
AU  - Ford E
AD  - Department of Computer Science, Graduate Center, CUNY, New York, NY, 10016, USA; 
      Department of Mathematics and Computer Science, Lehman College, CUNY, Bronx, NY, 
      10468, USA; and Division of Invertebrate Zoology, American Museum of Natural
      History, New York, NY, 10024, USA;
FAU - St John, Katherine
AU  - St John K
AD  - Department of Computer Science, Graduate Center, CUNY, New York, NY, 10016, USA; 
      Department of Mathematics and Computer Science, Lehman College, CUNY, Bronx, NY, 
      10468, USA; and Division of Invertebrate Zoology, American Museum of Natural
      History, New York, NY, 10024, USA; Department of Computer Science, Graduate
      Center, CUNY, New York, NY, 10016, USA; Department of Mathematics and Computer
      Science, Lehman College, CUNY, Bronx, NY, 10468, USA; and Division of
      Invertebrate Zoology, American Museum of Natural History, New York, NY, 10024,
      USA; stjohn@lehman.cuny.edu.
FAU - Wheeler, Ward C
AU  - Wheeler WC
AD  - Department of Computer Science, Graduate Center, CUNY, New York, NY, 10016, USA; 
      Department of Mathematics and Computer Science, Lehman College, CUNY, Bronx, NY, 
      10468, USA; and Division of Invertebrate Zoology, American Museum of Natural
      History, New York, NY, 10024, USA;
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140826
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Animals
MH  - Classification/*methods
MH  - Fungi/classification
MH  - Magnoliopsida/classification
MH  - *Phylogeny
OTO - NOTNLM
OT  - Character compatibility
OT  - exact search
OT  - maximum-parsimony optimality criterion
OT  - phylogenetic islands
OT  - tree search
EDAT- 2014/08/29 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/08/29 06:00
PHST- 2014/08/29 06:00 [entrez]
PHST- 2014/08/29 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu065 [pii]
AID - 10.1093/sysbio/syu065 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):56-65. doi: 10.1093/sysbio/syu065. Epub 2014 Aug 26.

PMID- 25164915
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Evaluating Summary Methods for Multilocus Species Tree Estimation in the Presence
      of Incomplete Lineage Sorting.
PG  - 366-80
LID - 10.1093/sysbio/syu063 [doi]
AB  - Species tree estimation is complicated by processes, such as gene duplication and
      loss and incomplete lineage sorting (ILS), that cause discordance between gene
      trees and the species tree. Furthermore, while concatenation, a traditional
      approach to tree estimation, has excellent performance under many conditions, the
      expectation is that the best accuracy will be obtained through the use of species
      tree estimation methods that are specifically designed to address gene tree
      discordance. In this article, we report on a study to evaluate MP-EST-one of the 
      most popular species tree estimation methods designed to address ILS-as well as
      concatenation under maximum likelihood, the greedy consensus, and two supertree
      methods (Matrix Representation with Parsimony and Matrix Representation with
      Likelihood). Our study shows that several factors impact the absolute and
      relative accuracy of methods, including the number of gene trees, the accuracy of
      the estimated gene trees, and the amount of ILS. Concatenation can be more
      accurate than the best summary methods in some cases (mostly when the gene trees 
      have poor phylogenetic signal or when the level of ILS is low), but summary
      methods are generally more accurate than concatenation when there are an adequate
      number of sufficiently accurate gene trees. Our study suggests that
      coalescent-based species tree methods may be key to estimating highly accurate
      species trees from multiple loci.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Mirarab, Siavash
AU  - Mirarab S
AD  - Department of Computer Science, University of Texas at Austin, Austin, TX, 78712,
      USA; and.
FAU - Bayzid, Md Shamsuzzoha
AU  - Bayzid MS
AD  - Department of Computer Science, University of Texas at Austin, Austin, TX, 78712,
      USA; and.
FAU - Warnow, Tandy
AU  - Warnow T
AD  - Department of Computer Science, University of Texas at Austin, Austin, TX, 78712,
      USA; and Departments of Bioengineering and Computer Science, University of
      Illinois at Urbana-Champaign, Urbana, IL, 61801, USA. warnow@illinois.edu.
LA  - eng
SI  - Dryad/10.5061/dryad.310q3
GR  - HHMI/Howard Hughes Medical Institute/United States
PT  - Journal Article
DEP - 20140826
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Gene Duplication
MH  - *Phylogeny
MH  - Probability
OTO - NOTNLM
OT  - *MP-EST
OT  - *MRL
OT  - *MRP
OT  - *concatenation
OT  - *consensus methods
OT  - *gene tree discordance
OT  - *incomplete lineage sorting
OT  - *multilocus bootstrapping
OT  - *species tree estimation
OT  - *supertree methods.
EDAT- 2014/08/29 06:00
MHDA- 2018/01/11 06:00
CRDT- 2014/08/29 06:00
PHST- 2013/12/16 00:00 [received]
PHST- 2014/08/18 00:00 [accepted]
PHST- 2014/08/29 06:00 [entrez]
PHST- 2014/08/29 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syu063 [pii]
AID - 10.1093/sysbio/syu063 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):366-80. doi: 10.1093/sysbio/syu063. Epub 2014 Aug 26.

PMID- 25151624
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20141015
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - Popperian corroboration and phylogenetics.
PG  - 1018-22
LID - 10.1093/sysbio/syu064 [doi]
FAU - de Queiroz, Kevin
AU  - de Queiroz K
AD  - Department of Vertebrate Zoology, National Museum of Natural History, Smithsonian
      Institution, Washington, DC 20560-0162, USA dequeirozk@si.edu.
LA  - eng
PT  - Journal Article
DEP - 20140822
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2014/08/26 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/08/25 06:00
PHST- 2014/08/25 06:00 [entrez]
PHST- 2014/08/26 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu064 [pii]
AID - 10.1093/sysbio/syu064 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):1018-22. doi: 10.1093/sysbio/syu064. Epub 2014 Aug 22.

PMID- 25139888
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20141015
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - Reconciling hierarchical taxonomy with molecular phylogenies.
PG  - 1010-7
LID - 10.1093/sysbio/syu061 [doi]
FAU - Holt, Ben G
AU  - Holt BG
AD  - Department of Life Sciences, Imperial College London, Silwood Park Campus, Ascot 
      SL5 7PY; and Department of Life Sciences, Natural History Museum, Cromwell Road, 
      London SW7 5BD, UK b.holt@imperial.ac.uk.
FAU - Jonsson, Knud Andreas
AU  - Jonsson KA
AD  - Department of Life Sciences, Imperial College London, Silwood Park Campus, Ascot 
      SL5 7PY; and Department of Life Sciences, Natural History Museum, Cromwell Road, 
      London SW7 5BD, UK Department of Life Sciences, Imperial College London, Silwood 
      Park Campus, Ascot SL5 7PY; and Department of Life Sciences, Natural History
      Museum, Cromwell Road, London SW7 5BD, UK.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140819
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Birds/classification/genetics
MH  - *Classification
MH  - Mammals/classification/genetics
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2014/08/21 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/08/21 06:00
PHST- 2014/08/21 06:00 [entrez]
PHST- 2014/08/21 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu061 [pii]
AID - 10.1093/sysbio/syu061 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):1010-7. doi: 10.1093/sysbio/syu061. Epub 2014 Aug 19.

PMID- 25123369
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20150528
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - Model selection in historical biogeography reveals that founder-event speciation 
      is a crucial process in Island Clades.
PG  - 951-70
LID - 10.1093/sysbio/syu056 [doi]
AB  - Founder-event speciation, where a rare jump dispersal event founds a new
      genetically isolated lineage, has long been considered crucial by many historical
      biogeographers, but its importance is disputed within the vicariance school.
      Probabilistic modeling of geographic range evolution creates the potential to
      test different biogeographical models against data using standard statistical
      model choice procedures, as long as multiple models are available. I re-implement
      the Dispersal-Extinction-Cladogenesis (DEC) model of LAGRANGE in the R package
      BioGeoBEARS, and modify it to create a new model, DEC + J, which adds
      founder-event speciation, the importance of which is governed by a new free
      parameter, [Formula: see text]. The identifiability of DEC and DEC + J is tested 
      on data sets simulated under a wide range of macroevolutionary models where
      geography evolves jointly with lineage birth/death events. The results confirm
      that DEC and DEC + J are identifiable even though these models ignore the fact
      that molecular phylogenies are missing many cladogenesis and extinction events.
      The simulations also indicate that DEC will have substantially increased errors
      in ancestral range estimation and parameter inference when the true model
      includes + J. DEC and DEC + J are compared on 13 empirical data sets drawn from
      studies of island clades. Likelihood-ratio tests indicate that all clades reject 
      DEC, and AICc model weights show large to overwhelming support for DEC + J, for
      the first time verifying the importance of founder-event speciation in island
      clades via statistical model choice. Under DEC + J, ancestral nodes are usually
      estimated to have ranges occupying only one island, rather than the widespread
      ancestors often favored by DEC. These results indicate that the assumptions of
      historical biogeography models can have large impacts on inference and require
      testing and comparison with statistical methods.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Matzke, Nicholas J
AU  - Matzke NJ
AD  - Department of Integrative Biology, University of California, Berkeley, CA
      94720-3140, USA; and National Institute for Mathematical and Biological Synthesis
      and Department of Ecology and Evolutionary Biology, University of Tennessee,
      Knoxville, TN 37996-3410, USA Department of Integrative Biology, University of
      California, Berkeley, CA 94720-3140, USA; and National Institute for Mathematical
      and Biological Synthesis and Department of Ecology and Evolutionary Biology,
      University of Tennessee, Knoxville, TN 37996-3410, USA matzke@nimbios.org.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140814
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2015 Jan;64(1):167. PMID: 25504792
MH  - *Computer Simulation
MH  - Founder Effect
MH  - Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Phylogeography/*methods
MH  - Psychotria/classification
MH  - Reproducibility of Results
OTO - NOTNLM
OT  - BioGeoBEARS
OT  - GeoSSE
OT  - LAGRANGE
OT  - cladogenesis
OT  - extinction
OT  - founder-event speciation
OT  - historical biogeography
OT  - jump dispersal
EDAT- 2014/08/16 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/08/16 06:00
PHST- 2014/08/16 06:00 [entrez]
PHST- 2014/08/16 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu056 [pii]
AID - 10.1093/sysbio/syu056 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):951-70. doi: 10.1093/sysbio/syu056. Epub 2014 Aug 14.

PMID- 25121824
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - Evolution at a different pace: distinctive phylogenetic patterns of cone snails
      from two ancient oceanic archipelagos.
PG  - 971-87
LID - 10.1093/sysbio/syu059 [doi]
AB  - Ancient oceanic archipelagos of similar geological age are expected to accrue
      comparable numbers of endemic lineages with identical life history strategies,
      especially if the islands exhibit analogous habitats. We tested this hypothesis
      using marine snails of the genus Conus from the Atlantic archipelagos of Cape
      Verde and Canary Islands. Together with Azores and Madeira, these archipelagos
      comprise the Macaronesia biogeographic region and differ remarkably in the
      diversity of this group. More than 50 endemic Conus species have been described
      from Cape Verde, whereas prior to this study, only two nonendemic species,
      including a putative species complex, were thought to occur in the Canary
      Islands. We combined molecular phylogenetic data and geometric morphometrics with
      bathymetric and paleoclimatic reconstructions to understand the contrasting
      diversification patterns found in these regions. Our results suggest that species
      diversity is even lower than previously thought in the Canary Islands, with the
      putative species complex corresponding to a single species, Conus guanche. One
      explanation for the enormous disparity in Conus diversity is that the amount of
      available habitat may differ, or may have differed in the past due to eustatic
      (global) sea level changes. Historical bathymetric data, however, indicated that 
      sea level fluctuations since the Miocene have had a similar impact on the
      available habitat area in both Cape Verde and Canary archipelagos and therefore
      do not explain this disparity. We suggest that recurrent gene flow between the
      Canary Islands and West Africa, habitat losses due to intense volcanic activity
      in combination with unsuccessful colonization of new Conus species from more
      diverse regions, were all determinant in shaping diversity patterns within the
      Canarian archipelago. Worldwide Conus species diversity follows the
      well-established pattern of latitudinal increase of species richness from the
      poles towards the tropics. However, the eastern Atlantic revealed a striking
      pattern with two main peaks of Conus species richness in the subtropical area and
      decreasing diversities toward the tropical western African coast. A Random
      Forests model using 12 oceanographic variables suggested that sea surface
      temperature is the main determinant of Conus diversity either at continental
      scales (eastern Atlantic coast) or in a broader context (worldwide). Other
      factors such as availability of suitable habitat and reduced salinity due to the 
      influx of large rivers in the tropical area also play an important role in
      shaping Conus diversity patterns in the western coast of Africa.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Cunha, Regina L
AU  - Cunha RL
AD  - Centre of Marine Sciences-CCMAR, Universidade do Algarve, Campus de Gambelas,
      8005 - 139 Faro, Portugal, CIBIO, Centro de Investigacao em Biodiversidade e
      Recursos Geneticos, Universidade do Porto, Campus Agrario de Vairao 4485-661
      Vairao, Portugal, Department CMIM y Q. Inorganica-INBIO, Facultad de Ciencias,
      Torre Norte, 1Planta, Universidad de Cadiz; 11510 Puerto Real; Cadiz, Spain; and 
      Department of Life Sciences, The Natural History Museum, London SW7 5BD, United
      Kingdom rcunha@ualg.pt.
FAU - Lima, Fernando P
AU  - Lima FP
AD  - Centre of Marine Sciences-CCMAR, Universidade do Algarve, Campus de Gambelas,
      8005 - 139 Faro, Portugal, CIBIO, Centro de Investigacao em Biodiversidade e
      Recursos Geneticos, Universidade do Porto, Campus Agrario de Vairao 4485-661
      Vairao, Portugal, Department CMIM y Q. Inorganica-INBIO, Facultad de Ciencias,
      Torre Norte, 1Planta, Universidad de Cadiz; 11510 Puerto Real; Cadiz, Spain; and 
      Department of Life Sciences, The Natural History Museum, London SW7 5BD, United
      Kingdom.
FAU - Tenorio, Manuel J
AU  - Tenorio MJ
AD  - Centre of Marine Sciences-CCMAR, Universidade do Algarve, Campus de Gambelas,
      8005 - 139 Faro, Portugal, CIBIO, Centro de Investigacao em Biodiversidade e
      Recursos Geneticos, Universidade do Porto, Campus Agrario de Vairao 4485-661
      Vairao, Portugal, Department CMIM y Q. Inorganica-INBIO, Facultad de Ciencias,
      Torre Norte, 1Planta, Universidad de Cadiz; 11510 Puerto Real; Cadiz, Spain; and 
      Department of Life Sciences, The Natural History Museum, London SW7 5BD, United
      Kingdom.
FAU - Ramos, Ana A
AU  - Ramos AA
AD  - Centre of Marine Sciences-CCMAR, Universidade do Algarve, Campus de Gambelas,
      8005 - 139 Faro, Portugal, CIBIO, Centro de Investigacao em Biodiversidade e
      Recursos Geneticos, Universidade do Porto, Campus Agrario de Vairao 4485-661
      Vairao, Portugal, Department CMIM y Q. Inorganica-INBIO, Facultad de Ciencias,
      Torre Norte, 1Planta, Universidad de Cadiz; 11510 Puerto Real; Cadiz, Spain; and 
      Department of Life Sciences, The Natural History Museum, London SW7 5BD, United
      Kingdom.
FAU - Castilho, Rita
AU  - Castilho R
AD  - Centre of Marine Sciences-CCMAR, Universidade do Algarve, Campus de Gambelas,
      8005 - 139 Faro, Portugal, CIBIO, Centro de Investigacao em Biodiversidade e
      Recursos Geneticos, Universidade do Porto, Campus Agrario de Vairao 4485-661
      Vairao, Portugal, Department CMIM y Q. Inorganica-INBIO, Facultad de Ciencias,
      Torre Norte, 1Planta, Universidad de Cadiz; 11510 Puerto Real; Cadiz, Spain; and 
      Department of Life Sciences, The Natural History Museum, London SW7 5BD, United
      Kingdom.
FAU - Williams, Suzanne T
AU  - Williams ST
AD  - Centre of Marine Sciences-CCMAR, Universidade do Algarve, Campus de Gambelas,
      8005 - 139 Faro, Portugal, CIBIO, Centro de Investigacao em Biodiversidade e
      Recursos Geneticos, Universidade do Porto, Campus Agrario de Vairao 4485-661
      Vairao, Portugal, Department CMIM y Q. Inorganica-INBIO, Facultad de Ciencias,
      Torre Norte, 1Planta, Universidad de Cadiz; 11510 Puerto Real; Cadiz, Spain; and 
      Department of Life Sciences, The Natural History Museum, London SW7 5BD, United
      Kingdom.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140812
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 16S)
SB  - IM
MH  - Animals
MH  - Azores
MH  - Biodiversity
MH  - Cabo Verde
MH  - Conus Snail/anatomy &amp; histology/*classification/*genetics
MH  - DNA, Ribosomal/genetics
MH  - *Ecosystem
MH  - Gene Flow
MH  - Genetic Speciation
MH  - Islands
MH  - *Phylogeny
MH  - Portugal
MH  - RNA, Ribosomal, 16S/genetics
MH  - Spain
OTO - NOTNLM
OT  - Conus
OT  - RF models
OT  - SST
OT  - eustatic sea level changes
OT  - latitudinal gradient of species diversity
OT  - oceanic islands
OT  - species diversity
EDAT- 2014/08/15 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/08/15 06:00
PHST- 2014/08/15 06:00 [entrez]
PHST- 2014/08/15 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu059 [pii]
AID - 10.1093/sysbio/syu059 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):971-87. doi: 10.1093/sysbio/syu059. Epub 2014 Aug 12.

PMID- 25116917
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - Known knowns, known unknowns, unknown unknowns and unknown knowns in DNA
      barcoding: a comment on Dowton et al.
PG  - 1005-9
LID - 10.1093/sysbio/syu060 [doi]
FAU - Collins, Rupert A
AU  - Collins RA
AD  - Laboratorio de Evolucao e Genetica Animal, Departamento de Biologia, Universidade
      Federal do Amazonas, Av. Rodrigo Otavio, Manaus, Amazonas, Brazil and Department 
      of Ecology, Faculty of Agriculture and Life Sciences, Lincoln University, Lincoln
      7647, Canterbury, New Zealand rupertcollins@gmail.com.
FAU - Cruickshank, Robert H
AU  - Cruickshank RH
AD  - Laboratorio de Evolucao e Genetica Animal, Departamento de Biologia, Universidade
      Federal do Amazonas, Av. Rodrigo Otavio, Manaus, Amazonas, Brazil and Department 
      of Ecology, Faculty of Agriculture and Life Sciences, Lincoln University, Lincoln
      7647, Canterbury, New Zealand.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Comment
DEP - 20140812
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2014 Jul;63(4):639-44. PMID: 24682413
MH  - Classification/*methods
MH  - DNA Barcoding, Taxonomic/*standards
MH  - *Phylogeny
EDAT- 2014/08/15 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/08/14 06:00
PHST- 2014/08/14 06:00 [entrez]
PHST- 2014/08/15 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu060 [pii]
AID - 10.1093/sysbio/syu060 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):1005-9. doi: 10.1093/sysbio/syu060. Epub 2014 Aug 12.

PMID- 25102857
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - Phylogenetics and the human microbiome.
PG  - e26-41
LID - 10.1093/sysbio/syu053 [doi]
AB  - The human microbiome is the ensemble of genes in the microbes that live inside
      and on the surface of humans. Because microbial sequencing information is now
      much easier to come by than phenotypic information, there has been an explosion
      of sequencing and genetic analysis of microbiome samples. Much of the analytical 
      work for these sequences involves phylogenetics, at least indirectly, but
      methodology has developed in a somewhat different direction than for other
      applications of phylogenetics. In this article, I review the field and its
      methods from the perspective of a phylogeneticist, as well as describing current 
      challenges for phylogenetics coming from this type of work.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Matsen, Frederick A 4th
AU  - Matsen FA 4th
AD  - Program in Computational Biology, Fred Hutchinson Cancer Research Center,
      Seattle, WA 91802, USA matsen@fhcrc.org.
LA  - eng
GR  - R01 HG005966/HG/NHGRI NIH HHS/United States
GR  - R01-HG005966-01/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Review
DEP - 20140807
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 16S)
SB  - IM
MH  - Bacteria/genetics
MH  - Humans
MH  - Microbiota/genetics
MH  - *Phylogeny
MH  - RNA, Ribosomal, 16S/genetics
PMC - PMC4265140
OTO - NOTNLM
OT  - 16S
OT  - human microbiome
OT  - human microbiota
OT  - metagenome
OT  - microbial ecology
OT  - phylogenetic methods
EDAT- 2014/08/12 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/08/09 06:00
PHST- 2014/08/09 06:00 [entrez]
PHST- 2014/08/12 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu053 [pii]
AID - 10.1093/sysbio/syu053 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):e26-41. doi: 10.1093/sysbio/syu053. Epub 2014 Aug 7.

PMID- 25096853
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - Bayesian species delimitation can be robust to guide-tree inference errors.
PG  - 993-1004
LID - 10.1093/sysbio/syu052 [doi]
FAU - Zhang, Chi
AU  - Zhang C
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History, Box
      50007, SE-104 05 Stockholm, Sweden; Genome Center and Section of Evolution and
      Ecology, University of California at Davis, One Shields Avenue, Davis, CA 95616, 
      USA; Department of Genetics, Evolution and Environment, University College
      London, Darwin Building, Gower Street, London WC1E 6BT, UK; and Center for
      Computational Genomics, Beijing Institute of Genomics, Chinese Academy of
      Sciences, Beijing 100101, China.
FAU - Rannala, Bruce
AU  - Rannala B
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History, Box
      50007, SE-104 05 Stockholm, Sweden; Genome Center and Section of Evolution and
      Ecology, University of California at Davis, One Shields Avenue, Davis, CA 95616, 
      USA; Department of Genetics, Evolution and Environment, University College
      London, Darwin Building, Gower Street, London WC1E 6BT, UK; and Center for
      Computational Genomics, Beijing Institute of Genomics, Chinese Academy of
      Sciences, Beijing 100101, China.
FAU - Yang, Ziheng
AU  - Yang Z
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History, Box
      50007, SE-104 05 Stockholm, Sweden; Genome Center and Section of Evolution and
      Ecology, University of California at Davis, One Shields Avenue, Davis, CA 95616, 
      USA; Department of Genetics, Evolution and Environment, University College
      London, Darwin Building, Gower Street, London WC1E 6BT, UK; and Center for
      Computational Genomics, Beijing Institute of Genomics, Chinese Academy of
      Sciences, Beijing 100101, China Department of Bioinformatics and Genetics,
      Swedish Museum of Natural History, Box 50007, SE-104 05 Stockholm, Sweden; Genome
      Center and Section of Evolution and Ecology, University of California at Davis,
      One Shields Avenue, Davis, CA 95616, USA; Department of Genetics, Evolution and
      Environment, University College London, Darwin Building, Gower Street, London
      WC1E 6BT, UK; and Center for Computational Genomics, Beijing Institute of
      Genomics, Chinese Academy of Sciences, Beijing 100101, China z.yang@ucl.ac.uk.
LA  - eng
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140805
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Genetic Speciation
MH  - Mitochondria/genetics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Sample Size
PMC - PMC4195854
EDAT- 2014/08/07 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/08/07 06:00
PHST- 2014/08/07 06:00 [entrez]
PHST- 2014/08/07 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu052 [pii]
AID - 10.1093/sysbio/syu052 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):993-1004. doi: 10.1093/sysbio/syu052. Epub 2014 Aug 5.

PMID- 25092479
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20141015
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - Chloroplast phylogenomic analyses resolve deep-level relationships of an
      intractable bamboo tribe Arundinarieae (poaceae).
PG  - 933-50
LID - 10.1093/sysbio/syu054 [doi]
AB  - The temperate woody bamboos constitute a distinct tribe Arundinarieae (Poaceae:
      Bambusoideae) with high species diversity. Estimating phylogenetic relationships 
      among the 11 major lineages of Arundinarieae has been particularly difficult,
      owing to a possible rapid radiation and the extremely low rate of sequence
      divergence. Here, we explore the use of chloroplast genome sequencing for
      phylogenetic inference. We sampled 25 species (22 temperate bamboos and 3
      outgroups) for the complete genome representing eight major lineages of
      Arundinarieae in an attempt to resolve backbone relationships. Phylogenetic
      analyses of coding versus noncoding sequences, and of different regions of the
      genome (large single copy and small single copy, and inverted repeat regions)
      yielded no well-supported contradicting topologies but potential incongruence was
      found between the coding and noncoding sequences. The use of various data
      partitioning schemes in analysis of the complete sequences resulted in nearly
      identical topologies and node support values, although the partitioning schemes
      were decisively different from each other as to the fit to the data. Our full
      genomic data set substantially increased resolution along the backbone and
      provided strong support for most relationships despite the very short internodes 
      and long branches in the tree. The inferred relationships were also robust to
      potential confounding factors (e.g., long-branch attraction) and received support
      from independent indels in the genome. We then added taxa from the three
      Arundinarieae lineages that were not included in the full-genome data set; each
      of these were sampled for more than 50% genome sequences. The resulting trees not
      only corroborated the reconstructed deep-level relationships but also largely
      resolved the phylogenetic placements of these three additional lineages.
      Furthermore, adding 129 additional taxa sampled for only eight chloroplast loci
      to the combined data set yielded almost identical relationships, albeit with low 
      support values. We believe that the inferred phylogeny is robust to taxon
      sampling. Having resolved the deep-level relationships of Arundinarieae, we
      illuminate how chloroplast phylogenomics can be used for elucidating difficult
      phylogeny at low taxonomic levels in intractable plant groups.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Ma, Peng-Fei
AU  - Ma PF
AD  - Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming
      Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan 650201, China; 
      Plant Germplasm and Genomics Center, Germplasm Bank of Wild Species, Kunming
      Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan 650201, China; 
      and Kunming College of Life Sciences, University of Chinese Academy of Sciences, 
      Kunming, Yunnan 650201, China Key Laboratory for Plant Diversity and Biogeography
      of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 
      Yunnan 650201, China; Plant Germplasm and Genomics Center, Germplasm Bank of Wild
      Species, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming,
      Yunnan 650201, China; and Kunming College of Life Sciences, University of Chinese
      Academy of Sciences, Kunming, Yunnan 650201, China Key Laboratory for Plant
      Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese
      Academy of Sciences, Kunming, Yunnan 650201, China; Plant Germplasm and Genomics 
      Center, Germplasm Bank of Wild Species, Kunming Institute of Botany, Chinese
      Academy of Sciences, Kunming, Yunnan 650201, China; and Kunming College of Life
      Sciences, University of Chinese Academy of Sciences, Kunming, Yunnan 650201,
      China.
FAU - Zhang, Yu-Xiao
AU  - Zhang YX
AD  - Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming
      Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan 650201, China; 
      Plant Germplasm and Genomics Center, Germplasm Bank of Wild Species, Kunming
      Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan 650201, China; 
      and Kunming College of Life Sciences, University of Chinese Academy of Sciences, 
      Kunming, Yunnan 650201, China Key Laboratory for Plant Diversity and Biogeography
      of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 
      Yunnan 650201, China; Plant Germplasm and Genomics Center, Germplasm Bank of Wild
      Species, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming,
      Yunnan 650201, China; and Kunming College of Life Sciences, University of Chinese
      Academy of Sciences, Kunming, Yunnan 650201, China.
FAU - Zeng, Chun-Xia
AU  - Zeng CX
AD  - Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming
      Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan 650201, China; 
      Plant Germplasm and Genomics Center, Germplasm Bank of Wild Species, Kunming
      Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan 650201, China; 
      and Kunming College of Life Sciences, University of Chinese Academy of Sciences, 
      Kunming, Yunnan 650201, China.
FAU - Guo, Zhen-Hua
AU  - Guo ZH
AD  - Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming
      Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan 650201, China; 
      Plant Germplasm and Genomics Center, Germplasm Bank of Wild Species, Kunming
      Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan 650201, China; 
      and Kunming College of Life Sciences, University of Chinese Academy of Sciences, 
      Kunming, Yunnan 650201, China.
FAU - Li, De-Zhu
AU  - Li DZ
AD  - Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming
      Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan 650201, China; 
      Plant Germplasm and Genomics Center, Germplasm Bank of Wild Species, Kunming
      Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan 650201, China; 
      and Kunming College of Life Sciences, University of Chinese Academy of Sciences, 
      Kunming, Yunnan 650201, China Key Laboratory for Plant Diversity and Biogeography
      of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, 
      Yunnan 650201, China; Plant Germplasm and Genomics Center, Germplasm Bank of Wild
      Species, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming,
      Yunnan 650201, China; and Kunming College of Life Sciences, University of Chinese
      Academy of Sciences, Kunming, Yunnan 650201, China dzl@mail.kib.ac.cn.
LA  - eng
SI  - GENBANK/FJ970915
SI  - GENBANK/FJ970916
SI  - GENBANK/HQ154129
SI  - GENBANK/HQ337793
SI  - GENBANK/HQ337794
SI  - GENBANK/HQ337795
SI  - GENBANK/HQ337796
SI  - GENBANK/HQ337797
SI  - GENBANK/JN415113
SI  - GENBANK/JX235347
SI  - GENBANK/JX513413
SI  - GENBANK/JX513414
SI  - GENBANK/JX513415
SI  - GENBANK/JX513416
SI  - GENBANK/JX513417
SI  - GENBANK/JX513418
SI  - GENBANK/JX513419
SI  - GENBANK/JX513420
SI  - GENBANK/JX513421
SI  - GENBANK/JX513422
SI  - GENBANK/JX513423
SI  - GENBANK/JX513424
SI  - GENBANK/JX513425
SI  - GENBANK/JX513426
SI  - GENBANK/KJ496369
SI  - GENBANK/KJ522748
SI  - GENBANK/KJ531442
SI  - GENBANK/KJ531443
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140804
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Genome, Chloroplast/*genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Poaceae/*classification/*genetics
OTO - NOTNLM
OT  - Branch length
OT  - chloroplast phylogenomics
OT  - data partitioning
OT  - low sequence divergence
OT  - sampling
OT  - temperate woody bamboos
EDAT- 2014/08/06 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/08/06 06:00
PHST- 2014/08/06 06:00 [entrez]
PHST- 2014/08/06 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu054 [pii]
AID - 10.1093/sysbio/syu054 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):933-50. doi: 10.1093/sysbio/syu054. Epub 2014 Aug 4.

PMID- 25077515
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - Coalescent versus concatenation methods and the placement of Amborella as sister 
      to water lilies.
PG  - 919-32
LID - 10.1093/sysbio/syu055 [doi]
AB  - The molecular era has fundamentally reshaped our knowledge of the evolution and
      diversification of angiosperms. One outstanding question is the phylogenetic
      placement of Amborella trichopoda Baill., commonly thought to represent the first
      lineage of extant angiosperms. Here, we leverage publicly available data and
      provide a broad coalescent-based species tree estimation of 45 seed plants. By
      incorporating 310 nuclear genes, our coalescent analyses strongly support a clade
      containing Amborella plus water lilies (i.e., Nymphaeales) that is sister to all 
      other angiosperms across different nucleotide rate partitions. Our results also
      show that commonly applied concatenation methods produce strongly supported, but 
      incongruent placements of Amborella: slow-evolving nucleotide sites corroborate
      results from coalescent analyses, whereas fast-evolving sites place Amborella
      alone as the first lineage of extant angiosperms. We further explored the
      performance of coalescent versus concatenation methods using nucleotide sequences
      simulated on (i) the two alternate placements of Amborella with branch lengths
      and substitution model parameters estimated from each of the 310 nuclear genes
      and (ii) three hypothetical species trees that are topologically identical except
      with respect to the degree of deep coalescence and branch lengths. Our results
      collectively suggest that the Amborella alone placement inferred using
      concatenation methods is likely misled by fast-evolving sites. This appears to be
      exacerbated by the combination of long branches in stem group angiosperms,
      Amborella, and Nymphaeales with the short internal branch separating Amborella
      and Nymphaeales. In contrast, coalescent methods appear to be more robust to
      elevated substitution rates.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Xi, Zhenxiang
AU  - Xi Z
AD  - Department of Organismic and Evolutionary Biology, Harvard University, Cambridge,
      MA 02138, USA; Department of Statistics and Institute of Bioinformatics,
      University of Georgia, Athens, GA 30602, USA; Department of Ecology and
      Evolution, Stony Brook University, Stony Brook, NY 11794, USA;
FAU - Liu, Liang
AU  - Liu L
AD  - Department of Organismic and Evolutionary Biology, Harvard University, Cambridge,
      MA 02138, USA; Department of Statistics and Institute of Bioinformatics,
      University of Georgia, Athens, GA 30602, USA; Department of Ecology and
      Evolution, Stony Brook University, Stony Brook, NY 11794, USA;
FAU - Rest, Joshua S
AU  - Rest JS
AD  - Department of Organismic and Evolutionary Biology, Harvard University, Cambridge,
      MA 02138, USA; Department of Statistics and Institute of Bioinformatics,
      University of Georgia, Athens, GA 30602, USA; Department of Ecology and
      Evolution, Stony Brook University, Stony Brook, NY 11794, USA;
FAU - Davis, Charles C
AU  - Davis CC
AD  - Department of Organismic and Evolutionary Biology, Harvard University, Cambridge,
      MA 02138, USA; Department of Statistics and Institute of Bioinformatics,
      University of Georgia, Athens, GA 30602, USA; Department of Ecology and
      Evolution, Stony Brook University, Stony Brook, NY 11794, USA;
      cdavis@oeb.harvard.edu.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140730
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Genes, Plant/genetics
MH  - Magnoliopsida/*classification/genetics
MH  - Nymphaea/*classification/genetics
MH  - *Phylogeny
MH  - Plastids/genetics
OTO - NOTNLM
OT  - Amborella trichopoda
OT  - Nymphaeales
OT  - coalescent methods
OT  - concatenation methods
OT  - elevated substitution rates
OT  - long-branch attraction
EDAT- 2014/08/01 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/08/01 06:00
PHST- 2014/08/01 06:00 [entrez]
PHST- 2014/08/01 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu055 [pii]
AID - 10.1093/sysbio/syu055 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):919-32. doi: 10.1093/sysbio/syu055. Epub 2014 Jul 30.

PMID- 25077514
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - Unsorted homology within locus and species trees.
PG  - 988-92
LID - 10.1093/sysbio/syu050 [doi]
FAU - Mallo, Diego
AU  - Mallo D
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Vigo
      36310, Spain dmallo@uvigo.es.
FAU - De Oliveira Martins, Leonardo
AU  - De Oliveira Martins L
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Vigo
      36310, Spain.
FAU - Posada, David
AU  - Posada D
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Vigo
      36310, Spain.
LA  - eng
GR  - 203161/European Research Council/International
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140730
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Phylogeny
MH  - Statistics as Topic/*standards
PMC - PMC5221733
MID - EMS65358
EDAT- 2014/08/01 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/08/01 06:00
PHST- 2014/08/01 06:00 [entrez]
PHST- 2014/08/01 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu050 [pii]
AID - 10.1093/sysbio/syu050 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):988-92. doi: 10.1093/sysbio/syu050. Epub 2014 Jul 30.

PMID- 25077513
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20141015
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - A novel Bayesian method for inferring and interpreting the dynamics of adaptive
      landscapes from phylogenetic comparative data.
PG  - 902-18
LID - 10.1093/sysbio/syu057 [doi]
AB  - Our understanding of macroevolutionary patterns of adaptive evolution has greatly
      increased with the advent of large-scale phylogenetic comparative methods. Widely
      used Ornstein-Uhlenbeck (OU) models can describe an adaptive process of
      divergence and selection. However, inference of the dynamics of adaptive
      landscapes from comparative data is complicated by interpretational difficulties,
      lack of identifiability among parameter values and the common requirement that
      adaptive hypotheses must be assigned a priori. Here, we develop a reversible-jump
      Bayesian method of fitting multi-optima OU models to phylogenetic comparative
      data that estimates the placement and magnitude of adaptive shifts directly from 
      the data. We show how biologically informed hypotheses can be tested against this
      inferred posterior of shift locations using Bayes Factors to establish whether
      our a priori models adequately describe the dynamics of adaptive peak shifts.
      Furthermore, we show how the inclusion of informative priors can be used to
      restrict models to biologically realistic parameter space and test particular
      biological interpretations of evolutionary models. We argue that Bayesian model
      fitting of OU models to comparative data provides a framework for integrating of 
      multiple sources of biological data-such as microevolutionary estimates of
      selection parameters and paleontological timeseries-allowing inference of
      adaptive landscape dynamics with explicit, process-based biological
      interpretations.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Uyeda, Josef C
AU  - Uyeda JC
AD  - Department of Biological Sciences, University of Idaho, Life Sciences South 252, 
      875 Perimeter Dr MS 3051, Moscow, ID., University of Idaho josef.uyeda@gmail.com.
FAU - Harmon, Luke J
AU  - Harmon LJ
AD  - Department of Biological Sciences, University of Idaho, Life Sciences South 252, 
      875 Perimeter Dr MS 3051, Moscow, ID., University of Idaho.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140730
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Data Interpretation, Statistical
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Turtles/anatomy &amp; histology/classification
OTO - NOTNLM
OT  - Ornstein-Uhlenbeck
OT  - Reversible-jump models
OT  - bayou
OT  - comparative methods
OT  - macroevolution
EDAT- 2014/08/01 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/08/01 06:00
PHST- 2014/08/01 06:00 [entrez]
PHST- 2014/08/01 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu057 [pii]
AID - 10.1093/sysbio/syu057 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):902-18. doi: 10.1093/sysbio/syu057. Epub 2014 Jul 30.

PMID- 25074580
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - Bias in tree searches and its consequences for measuring group supports.
PG  - 851-61
LID - 10.1093/sysbio/syu051 [doi]
AB  - When doing a bootstrap analysis with a single tree saved per pseudoreplicate,
      biased search algorithms may influence support values more than actual properties
      of the data set. Two methods commonly used for finding phylogenetic trees consist
      of randomizing the input order of species in multiple addition sequences followed
      by branch swapping, or using random trees as the starting point for branch
      swapping. The randomness inherent to such methods is assumed to eliminate any
      consistent preferences for some trees or unsupported groups of taxa, but both
      methods can be significantly biased. In the case of trees created by sequentially
      adding taxa, a bias may occur even if every addition sequence is equiprobable,
      and if one of the equally optimal positions for each terminal to add to the tree 
      is selected equiprobably. In the case of branch swapping, the bias can happen
      even when branch swapping equiprobably selects any of the trees of better score
      in the subtree-pruning-regrafting-neighborhood or
      tree-bisection-reconnection-neighborhood. Consequently, when the data set is
      ambiguous, both random-addition sequences and branch swapping from random trees
      may (i) find some of the optimal trees much more frequently than others and (ii) 
      find some groups with a frequency that differs from their frequency among all
      optimal trees. When the data set defines a single optimal tree, the groups
      present in that tree may have a different probability of being found by a search,
      even if supported by equal amounts of evidence. This may happen in both parsimony
      and maximum-likelihood analyses, and even in small data sets without
      incongruence.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Goloboff, Pablo A
AU  - Goloboff PA
AD  - CONICET, INSUE, Fundacion Miguel Lillo, Miguel Lillo 251, 4000 S.M. de Tucuman,
      Argentina; and Department of Biology, Colorado State University, Fort Collins, CO
      80523, USA pablogolo@yahoo.com.ar.
FAU - Simmons, Mark P
AU  - Simmons MP
AD  - CONICET, INSUE, Fundacion Miguel Lillo, Miguel Lillo 251, 4000 S.M. de Tucuman,
      Argentina; and Department of Biology, Colorado State University, Fort Collins, CO
      80523, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140728
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bias
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - *Phylogeny
MH  - Software/*standards
OTO - NOTNLM
OT  - Bootstrap
OT  - group support
OT  - jackknife
OT  - phylogenetic analysis
OT  - resampling
OT  - tree searches
EDAT- 2014/07/31 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/07/31 06:00
PHST- 2014/07/31 06:00 [entrez]
PHST- 2014/07/31 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu051 [pii]
AID - 10.1093/sysbio/syu051 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):851-61. doi: 10.1093/sysbio/syu051. Epub 2014 Jul 28.

PMID- 25070972
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - Mitochondrial phylogenomics of early land plants: mitigating the effects of
      saturation, compositional heterogeneity, and codon-usage bias.
PG  - 862-78
LID - 10.1093/sysbio/syu049 [doi]
AB  - Phylogenetic analyses using concatenation of genomic-scale data have been seen as
      the panacea for resolving the incongruences among inferences from few or single
      genes. However, phylogenomics may also suffer from systematic errors, due to the,
      perhaps cumulative, effects of saturation, among-taxa compositional (GC content) 
      heterogeneity, or codon-usage bias plaguing the individual nucleotide loci that
      are concatenated. Here, we provide an example of how these factors affect the
      inferences of the phylogeny of early land plants based on mitochondrial genomic
      data. Mitochondrial sequences evolve slowly in plants and hence are thought to be
      suitable for resolving deep relationships. We newly assembled mitochondrial
      genomes from 20 bryophytes, complemented these with 40 other streptophytes (land 
      plants plus algal outgroups), compiling a data matrix of 60 taxa and 41
      mitochondrial genes. Homogeneous analyses of the concatenated nucleotide data
      resolve mosses as sister-group to the remaining land plants. However, the
      corresponding translated amino acid data support the liverwort lineage in this
      position. Both results receive weak to moderate support in maximum-likelihood
      analyses, but strong support in Bayesian inferences. Tests of alternative
      hypotheses using either nucleotide or amino acid data provide implicit support
      for their respective optimal topologies, and clearly reject the hypotheses that
      bryophytes are monophyletic, liverworts and mosses share a unique common
      ancestor, or hornworts are sister to the remaining land plants. We determined
      that land plant lineages differ in their nucleotide composition, and in their
      usage of synonymous codon variants. Composition heterogeneous Bayesian analyses
      employing a nonstationary model that accounts for variation in among-lineage
      composition, and inferences from degenerated nucleotide data that avoid the
      effects of synonymous substitutions that underlie codon-usage bias, again
      recovered liverworts being sister to the remaining land plants but without
      support. These analyses indicate that the inference of an early-branching moss
      lineage based on the nucleotide data is caused by convergent compositional
      biases. Accommodating among-site amino acid compositional heterogeneity
      (CAT-model) yields no support for the optimal resolution of liverwort as sister
      to the rest of land plants, suggesting that the robust inference of the liverwort
      position in homogeneous analyses may be due in part to compositional biases among
      sites. All analyses support a paraphyletic bryophytes with hornworts composing
      the sister-group to tracheophytes. We conclude that while genomic data may
      generate highly supported phylogenetic trees, these inferences may be artifacts. 
      We suggest that phylogenomic analyses should assess the possible impact of
      potential biases through comparisons of protein-coding gene data and their amino 
      acid translations by evaluating the impact of substitutional saturation,
      synonymous substitutions, and compositional biases through data deletion
      strategies and by analyzing the data using heterogeneous composition models. We
      caution against relying on any one presentation of the data (nucleotide or amino 
      acid) or any one type of analysis even when analyzing large-scale data sets, no
      matter how well-supported, without fully exploring the effects of substitution
      models.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Liu, Yang
AU  - Liu Y
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut,
      Storrs, CT 06269, USA; Centro de Ciencias do Mar, Universidade do Algarve,
      Gambelas, 8005-319 Faro, Portugal; and State Key Laboratory of Systematic and
      Evolutionary Botany, Institute of Botany, Chinese Academy of Sciences, Beijing
      100093, China.
FAU - Cox, Cymon J
AU  - Cox CJ
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut,
      Storrs, CT 06269, USA; Centro de Ciencias do Mar, Universidade do Algarve,
      Gambelas, 8005-319 Faro, Portugal; and State Key Laboratory of Systematic and
      Evolutionary Botany, Institute of Botany, Chinese Academy of Sciences, Beijing
      100093, China.
FAU - Wang, Wei
AU  - Wang W
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut,
      Storrs, CT 06269, USA; Centro de Ciencias do Mar, Universidade do Algarve,
      Gambelas, 8005-319 Faro, Portugal; and State Key Laboratory of Systematic and
      Evolutionary Botany, Institute of Botany, Chinese Academy of Sciences, Beijing
      100093, China.
FAU - Goffinet, Bernard
AU  - Goffinet B
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut,
      Storrs, CT 06269, USA; Centro de Ciencias do Mar, Universidade do Algarve,
      Gambelas, 8005-319 Faro, Portugal; and State Key Laboratory of Systematic and
      Evolutionary Botany, Institute of Botany, Chinese Academy of Sciences, Beijing
      100093, China bernard.goffinet@uconn.edu.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140728
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
SB  - IM
MH  - Bias
MH  - *Classification
MH  - Codon/genetics
MH  - Embryophyta/*classification/*genetics
MH  - *Genomics
MH  - *Phylogeny
OTO - NOTNLM
OT  - Compositional heterogeneity
OT  - early land plants
OT  - evolutionary saturation
OT  - mitochondrial genome
OT  - phylogenomics
OT  - synonymous codon-usage bias
EDAT- 2014/07/30 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/07/30 06:00
PHST- 2014/07/30 06:00 [entrez]
PHST- 2014/07/30 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu049 [pii]
AID - 10.1093/sysbio/syu049 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):862-78. doi: 10.1093/sysbio/syu049. Epub 2014 Jul 28.

PMID- 25070971
OWN - NLM
STAT- MEDLINE
DCOM- 20141208
LR  - 20141015
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 6
DP  - 2014 Nov
TI  - Borneo and Indochina are major evolutionary hotspots for Southeast Asian
      biodiversity.
PG  - 879-901
LID - 10.1093/sysbio/syu047 [doi]
AB  - Tropical Southeast (SE) Asia harbors extraordinary species richness and in its
      entirety comprises four of the Earth's 34 biodiversity hotspots. Here, we examine
      the assembly of the SE Asian biota through time and space. We conduct
      meta-analyses of geological, climatic, and biological (including 61 phylogenetic)
      data sets to test which areas have been the sources of long-term biological
      diversity in SE Asia, particularly in the pre-Miocene, Miocene, and
      Plio-Pleistocene, and whether the respective biota have been dominated by in situ
      diversification, immigration and/or emigration, or equilibrium dynamics. We
      identify Borneo and Indochina, in particular, as major "evolutionary hotspots"
      for a diverse range of fauna and flora. Although most of the region's
      biodiversity is a result of both the accumulation of immigrants and in situ
      diversification, within-area diversification and subsequent emigration have been 
      the predominant signals characterizing Indochina and Borneo's biota since at
      least the early Miocene. In contrast, colonization events are comparatively rare 
      from younger volcanically active emergent islands such as Java, which show
      increased levels of immigration events. Few dispersal events were observed across
      the major biogeographic barrier of Wallace's Line. Accelerated efforts to
      conserve Borneo's flora and fauna in particular, currently housing the highest
      levels of SE Asian plant and mammal species richness, are critically required.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - de Bruyn, Mark
AU  - de Bruyn M
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA
      markus.debruyn@gmail.com.
FAU - Stelbrink, Bjorn
AU  - Stelbrink B
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA.
FAU - Morley, Robert J
AU  - Morley RJ
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA
      Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, German
FAU - Hall, Robert
AU  - Hall R
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA.
FAU - Carvalho, Gary R
AU  - Carvalho GR
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA.
FAU - Cannon, Charles H
AU  - Cannon CH
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA
      Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, German
FAU - van den Bergh, Gerrit
AU  - van den Bergh G
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA.
FAU - Meijaard, Erik
AU  - Meijaard E
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA
      Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, German
FAU - Metcalfe, Ian
AU  - Metcalfe I
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA
      Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, German
FAU - Boitani, Luigi
AU  - Boitani L
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA.
FAU - Maiorano, Luigi
AU  - Maiorano L
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA.
FAU - Shoup, Robert
AU  - Shoup R
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA.
FAU - von Rintelen, Thomas
AU  - von Rintelen T
AD  - Molecular Ecology and Fisheries Genetics Laboratory, School of Biological
      Sciences, Bangor University, Deiniol Road, Bangor LL57 2UW, UK; Museum fur
      Naturkunde, Leibniz-Institut fur Evolutions- und Biodiversitatsforschung an der
      Humboldt-Universitat zu Berlin, Invalidenstr. 43, 10115 Berlin, Germany;
      Southeast Asia Research Group, Department of Earth Sciences, Royal Holloway
      University of London, Egham Hill, Egham TW20 0EX, UK; Palynova Limited, 1 Mow Fen
      Road, Littleport, Cambs CB6 1PY, UK; Niko Asia Ltd, Plaza City View, Jl Kemang
      Timur 22, Jakarta 12510, Indonesia; Department of Biological Sciences, Texas Tech
      University, TX 79409-3131, USA; Chinese Academy of Sciences, Xishuangbanna
      Tropical Botanic Garden, Yunnan 666303, P.R. China; Centre for Archaeological
      Science, University of Wollongong, Wollongong, NSW 2522, Australia; Borneo
      Futures Project, People and Nature Consulting International, Country Woods house 
      306, JL. WR Supratman, Pondok Ranji, Ciputat, Jakarta 15412, Indonesia; School of
      Archaeology &amp; Anthropology, Building 14, Australian National University,
      Canberra, ACT 0200, Australia; School of Biological Sciences, University of
      Queensland, St Lucia, QLD 4072, Australia; Earth Sciences, School of
      Environmental and Rural Science, University of New England, Armidale, NSW 2351,
      Australia; GEMOC, ARC Centre of Excellence for Core to Crust Fluid Systems,
      Department of Earth and Planetary Sciences, Macquarie University, NSW 2109,
      Australia; Department of Biology and Biotechnologies "Charles Darwin", Sapienza
      Universita di Roma, viale dell'Universita 32, 00185 Rome, Italy; Clastic
      Reservoir Systems, 10700 Richmond Avenue, Suite 325, Houston, TX 77042, USA.
LA  - eng
PT  - Journal Article
PT  - Meta-Analysis
PT  - Research Support, Non-U.S. Gov't
DEP - 20140728
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animal Distribution
MH  - Animals
MH  - Asia, Southeastern
MH  - *Biodiversity
MH  - *Biological Evolution
MH  - Borneo
MH  - Genetic Speciation
MH  - Geological Phenomena
MH  - Phylogeny
MH  - Plant Dispersal
MH  - Plants/classification
OTO - NOTNLM
OT  - Biogeography
OT  - Ecology
OT  - Geology
OT  - Palynology
OT  - Phylogenetics
OT  - climate change
EDAT- 2014/07/30 06:00
MHDA- 2014/12/15 06:00
CRDT- 2014/07/30 06:00
PHST- 2014/07/30 06:00 [entrez]
PHST- 2014/07/30 06:00 [pubmed]
PHST- 2014/12/15 06:00 [medline]
AID - syu047 [pii]
AID - 10.1093/sysbio/syu047 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Nov;63(6):879-901. doi: 10.1093/sysbio/syu047. Epub 2014 Jul 28.

PMID- 25070970
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - The inference of gene trees with species trees.
PG  - e42-62
LID - 10.1093/sysbio/syu048 [doi]
AB  - This article reviews the various models that have been used to describe the
      relationships between gene trees and species trees. Molecular phylogeny has
      focused mainly on improving models for the reconstruction of gene trees based on 
      sequence alignments. Yet, most phylogeneticists seek to reveal the history of
      species. Although the histories of genes and species are tightly linked, they are
      seldom identical, because genes duplicate, are lost or horizontally transferred, 
      and because alleles can coexist in populations for periods that may span several 
      speciation events. Building models describing the relationship between gene and
      species trees can thus improve the reconstruction of gene trees when a species
      tree is known, and vice versa. Several approaches have been proposed to solve the
      problem in one direction or the other, but in general neither gene trees nor
      species trees are known. Only a few studies have attempted to jointly infer gene 
      trees and species trees. These models account for gene duplication and loss,
      transfer or incomplete lineage sorting. Some of them consider several types of
      events together, but none exists currently that considers the full repertoire of 
      processes that generate gene trees along the species tree. Simulations as well as
      empirical studies on genomic data show that combining gene tree-species tree
      models with models of sequence evolution improves gene tree reconstruction. In
      turn, these better gene trees provide a more reliable basis for studying genome
      evolution or reconstructing ancestral chromosomes and ancestral gene sequences.
      We predict that gene tree-species tree methods that can deal with genomic data
      sets will be instrumental to advancing our understanding of genomic evolution.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Szollosi, Gergely J
AU  - Szollosi GJ
AD  - ELTE-MTA "Lendulet" Biophysics Research Group, Pazmany P. stny. 1A., 1117
      Budapest, Hungary; Laboratoire de Biometrie et Biologie Evolutive, Centre
      National de la Recherche Scientifique, Unite Mixte de Recherche 5558, Universite 
      Lyon 1, F-69622 Villeurbanne, France; Universite de Lyon, F-69000 Lyon, France;
      and Institut National de Recherche en Informatique et en Automatique Rhone-Alpes,
      F-38334 Montbonnot, France;
FAU - Tannier, Eric
AU  - Tannier E
AD  - ELTE-MTA "Lendulet" Biophysics Research Group, Pazmany P. stny. 1A., 1117
      Budapest, Hungary; Laboratoire de Biometrie et Biologie Evolutive, Centre
      National de la Recherche Scientifique, Unite Mixte de Recherche 5558, Universite 
      Lyon 1, F-69622 Villeurbanne, France; Universite de Lyon, F-69000 Lyon, France;
      and Institut National de Recherche en Informatique et en Automatique Rhone-Alpes,
      F-38334 Montbonnot, France; ELTE-MTA "Lendulet" Biophysics Research Group,
      Pazmany P. stny. 1A., 1117 Budapest, Hungary; Laboratoire de Biometrie et
      Biologie Evolutive, Centre National de la Recherche Scientifique, Unite Mixte de 
      Recherche 5558, Universite Lyon 1, F-69622 Villeurbanne, France; Universite de
      Lyon, F-69000 Lyon, France; and Institut National de Recherche en Informatique et
      en Automatique Rhone-Alpes, F-38334 Montbonnot, France; ELTE-MTA "Lendulet"
      Biophysics Research Group, Pazmany P. stny. 1A., 1117 Budapest, Hungary;
      Laboratoire de Biometrie et Biologie Evolutive, Centre National de la Recherche
      Scientifique, Unite Mixte de Recherche 5558, Universite Lyon 1, F-69622
      Villeurbanne, France; Universite de Lyon, F-69000 Lyon, France; and Institut
      National de Recherche en Informatique et en Automatique Rhone-Alpes, F-38334
      Montbonnot, France;
FAU - Daubin, Vincent
AU  - Daubin V
AD  - ELTE-MTA "Lendulet" Biophysics Research Group, Pazmany P. stny. 1A., 1117
      Budapest, Hungary; Laboratoire de Biometrie et Biologie Evolutive, Centre
      National de la Recherche Scientifique, Unite Mixte de Recherche 5558, Universite 
      Lyon 1, F-69622 Villeurbanne, France; Universite de Lyon, F-69000 Lyon, France;
      and Institut National de Recherche en Informatique et en Automatique Rhone-Alpes,
      F-38334 Montbonnot, France; ELTE-MTA "Lendulet" Biophysics Research Group,
      Pazmany P. stny. 1A., 1117 Budapest, Hungary; Laboratoire de Biometrie et
      Biologie Evolutive, Centre National de la Recherche Scientifique, Unite Mixte de 
      Recherche 5558, Universite Lyon 1, F-69622 Villeurbanne, France; Universite de
      Lyon, F-69000 Lyon, France; and Institut National de Recherche en Informatique et
      en Automatique Rhone-Alpes, F-38334 Montbonnot, France;
FAU - Boussau, Bastien
AU  - Boussau B
AD  - ELTE-MTA "Lendulet" Biophysics Research Group, Pazmany P. stny. 1A., 1117
      Budapest, Hungary; Laboratoire de Biometrie et Biologie Evolutive, Centre
      National de la Recherche Scientifique, Unite Mixte de Recherche 5558, Universite 
      Lyon 1, F-69622 Villeurbanne, France; Universite de Lyon, F-69000 Lyon, France;
      and Institut National de Recherche en Informatique et en Automatique Rhone-Alpes,
      F-38334 Montbonnot, France; ELTE-MTA "Lendulet" Biophysics Research Group,
      Pazmany P. stny. 1A., 1117 Budapest, Hungary; Laboratoire de Biometrie et
      Biologie Evolutive, Centre National de la Recherche Scientifique, Unite Mixte de 
      Recherche 5558, Universite Lyon 1, F-69622 Villeurbanne, France; Universite de
      Lyon, F-69000 Lyon, France; and Institut National de Recherche en Informatique et
      en Automatique Rhone-Alpes, F-38334 Montbonnot, France;
      bastien.boussau@univ-lyon1.fr.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Review
DEP - 20140728
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2016 Jan;65(1):180. PMID: 26546025
MH  - Computer Simulation
MH  - Genetic Speciation
MH  - Genome/genetics
MH  - *Models, Biological
MH  - Mutation/genetics
MH  - *Phylogeny
PMC - PMC4265139
OTO - NOTNLM
OT  - Algorithm
OT  - Bayesian inference
OT  - amalgamation
OT  - birth-death model
OT  - coalescent
OT  - dynamic programming
OT  - gene duplication
OT  - gene loss
OT  - gene transfer
OT  - gene tree
OT  - hybridization
OT  - maximum likelihood
OT  - phylogenetics
OT  - species tree
EDAT- 2014/07/30 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/07/30 06:00
PHST- 2014/07/30 06:00 [entrez]
PHST- 2014/07/30 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu048 [pii]
AID - 10.1093/sysbio/syu048 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):e42-62. doi: 10.1093/sysbio/syu048. Epub 2014 Jul 28.

PMID- 24996414
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20190222
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - Maximum likelihood inference of small trees in the presence of long branches.
PG  - 798-811
LID - 10.1093/sysbio/syu044 [doi]
AB  - The statistical basis of maximum likelihood (ML), its robustness, and the fact
      that it appears to suffer less from biases lead to it being one of the most
      popular methods for tree reconstruction. Despite its popularity, very few
      analytical solutions for ML exist, so biases suffered by ML are not well
      understood. One possible bias is long branch attraction (LBA), a regularly cited 
      term generally used to describe a propensity for long branches to be joined
      together in estimated trees. Although initially mentioned in connection with
      inconsistency of parsimony, LBA has been claimed to affect all major phylogenetic
      reconstruction methods, including ML. Despite the widespread use of this term in 
      the literature, exactly what LBA is and what may be causing it is poorly
      understood, even for simple evolutionary models and small model trees. Studies
      looking at LBA have focused on the effect of two long branches on tree
      reconstruction. However, to understand the effect of two long branches it is also
      important to understand the effect of just one long branch. If ML struggles to
      reconstruct one long branch, then this may have an impact on LBA. In this study, 
      we look at the effect of one long branch on three-taxon tree reconstruction. We
      show that, counterintuitively, long branches are preferentially placed at the
      tips of the tree. This can be understood through the use of analytical solutions 
      to the ML equation and distance matrix methods. We go on to look at the placement
      of two long branches on four-taxon trees, showing that there is no attraction
      between long branches, but that for extreme branch lengths long branches are
      joined together disproportionally often. These results illustrate that even small
      model trees are still interesting to help understand how ML phylogenetic
      reconstruction works, and that LBA is a complicated phenomenon that deserves
      further study.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Parks, Sarah L
AU  - Parks SL
AD  - European Molecular Biology Laboratory, European Bioinformatics Institute
      (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, CB10 1SD, United Kingdom
      sparks@ebi.ac.uk.
FAU - Goldman, Nick
AU  - Goldman N
AD  - European Molecular Biology Laboratory, European Bioinformatics Institute
      (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, CB10 1SD, United Kingdom.
LA  - eng
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140704
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification
MH  - Computer Simulation
MH  - Evolution, Molecular
MH  - *Models, Biological
MH  - *Phylogeny
PMC - PMC6371681
EDAT- 2014/07/06 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/07/06 06:00
PHST- 2014/07/06 06:00 [entrez]
PHST- 2014/07/06 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu044 [pii]
AID - 10.1093/sysbio/syu044 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):798-811. doi: 10.1093/sysbio/syu044. Epub 2014 Jul 4.

PMID- 24996413
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Unforeseen Consequences of Excluding Missing Data from Next-Generation Sequences:
      Simulation Study of RAD Sequences.
PG  - 357-65
LID - 10.1093/sysbio/syu046 [doi]
AB  - There is a lack of consensus on how next-generation sequence (NGS) data should be
      considered for phylogenetic and phylogeographic estimates, with some studies
      excluding loci with missing data, whereas others include them, even when
      sequences are missing from a large number of individuals. Here, we use
      simulations, focusing specifically on RAD (Restriction site Associated DNA)
      sequences, to highlight some of the unforeseen consequence of excluding missing
      data from next-generation sequencing. Specifically, we show that in addition to
      the obvious effects associated with reducing the amount of data used to make
      historical inferences, the decisions we make about missing data (such as the
      minimum number of individuals with a sequence for a locus to be included in the
      study) also impact the types of loci sampled for a study. In particular, as the
      tolerance for missing data becomes more stringent, the mutational spectrum
      represented in the sampled loci becomes truncated such that loci with the highest
      mutation rates are disproportionately excluded. This effect is exacerbated
      further by factors involved in the preparation of the genomic library (i.e., the 
      use of reduced representation libraries, as well as the coverage) and the
      taxonomic diversity represented in the library (i.e., the level of divergence
      among the individuals). We demonstrate that the intuitive appeals about being
      conservative by removing loci may be misguided. [Next-generation sequencing;
      phylogenetic; phylogeography; RADseq; RADtags; species delimitation.].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Huang, Huateng
AU  - Huang H
AD  - Department of Ecology and Evolutionary Biology, Museum of Zoology, University of 
      Michigan, 1109 Geddes Avenue, Ann Arbor, MI 48109-1079, USA huatengh@umich.edu.
FAU - Knowles, L Lacey
AU  - Knowles LL
AD  - Department of Ecology and Evolutionary Biology, Museum of Zoology, University of 
      Michigan, 1109 Geddes Avenue, Ann Arbor, MI 48109-1079, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.jf361
PT  - Journal Article
DEP - 20140704
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - Genome/genetics
MH  - High-Throughput Nucleotide Sequencing
MH  - Mutation
MH  - *Phylogeny
MH  - Phylogeography
MH  - *Sequence Analysis, DNA
EDAT- 2014/07/06 06:00
MHDA- 2018/01/11 06:00
CRDT- 2014/07/06 06:00
PHST- 2013/10/03 00:00 [received]
PHST- 2014/06/22 00:00 [accepted]
PHST- 2014/07/06 06:00 [entrez]
PHST- 2014/07/06 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syu046 [pii]
AID - 10.1093/sysbio/syu046 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):357-65. doi: 10.1093/sysbio/syu046. Epub 2014 Jul 4.

PMID- 24958930
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20190222
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - Testing for universal common ancestry.
PG  - 838-42
LID - 10.1093/sysbio/syu041 [doi]
FAU - de Oliveira Martins, Leonardo
AU  - de Oliveira Martins L
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Vigo
      36310, Spain; leomrtns@uvigo.es.
FAU - Posada, David
AU  - Posada D
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Vigo
      36310, Spain;
LA  - eng
GR  - 203161/European Research Council/International
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140623
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Models, Biological
MH  - *Phylogeny
PMC - PMC5215821
MID - EMS65357
EDAT- 2014/06/25 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/06/25 06:00
PHST- 2014/06/25 06:00 [entrez]
PHST- 2014/06/25 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu041 [pii]
AID - 10.1093/sysbio/syu041 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):838-42. doi: 10.1093/sysbio/syu041. Epub 2014 Jun 23.

PMID- 24951559
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - Probabilistic graphical model representation in phylogenetics.
PG  - 753-71
LID - 10.1093/sysbio/syu039 [doi]
AB  - Recent years have seen a rapid expansion of the model space explored in
      statistical phylogenetics, emphasizing the need for new approaches to statistical
      model representation and software development. Clear communication and
      representation of the chosen model is crucial for: (i) reproducibility of an
      analysis, (ii) model development, and (iii) software design. Moreover, a unified,
      clear and understandable framework for model representation lowers the barrier
      for beginners and nonspecialists to grasp complex phylogenetic models, including 
      their assumptions and parameter/variable dependencies. Graphical modeling is a
      unifying framework that has gained in popularity in the statistical literature in
      recent years. The core idea is to break complex models into conditionally
      independent distributions. The strength lies in the comprehensibility,
      flexibility, and adaptability of this formalism, and the large body of
      computational work based on it. Graphical models are well-suited to teach
      statistical models, to facilitate communication among phylogeneticists and in the
      development of generic software for simulation and statistical inference. Here,
      we provide an introduction to graphical models for phylogeneticists and extend
      the standard graphical model representation to the realm of phylogenetics. We
      introduce a new graphical model component, tree plates, to capture the changing
      structure of the subgraph corresponding to a phylogenetic tree. We describe a
      range of phylogenetic models using the graphical model framework and introduce
      modules to simplify the representation of standard components in large and
      complex models. Phylogenetic model graphs can be readily used in simulation,
      maximum likelihood inference, and Bayesian inference using, for example,
      Metropolis-Hastings or Gibbs sampling of the posterior distribution.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Hohna, Sebastian
AU  - Hohna S
AD  - Department of Mathematics, Stockholm University, Stockholm, SE-106 91 Stockholm, 
      Sweden; Department of Evolution and Ecology, University of California, Davis,
      Storer Hall, One Shields Avenue, Davis, CA 95616, USA; Department of Integrative 
      Biology, University of California, Berkeley, CA 94720, USA; Department of Ecology
      and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, USA;
      Bioinformatics and Evolutionary Genomics, Universite de Lyon, Villeurbanne,
      France; Department of Bioinformatics and Genetics, Swedish Museum of Natural
      History, SE-10405 Stockholm, Sweden; and Department of Biological Science, King
      Abdulaziz University, Jeddah, Saudi Arabia;Department of Mathematics, Stockholm
      University, Stockholm, SE-106 91 Stockholm, Sweden; Department of Evolution and
      Ecology, University of California, Davis, Storer Hall, One Shields Avenue, Davis,
      CA 95616, USA; Department of Integrative Biology, University of California,
      Berkeley, CA 94720, USA; Department of Ecology and Evolutionary Biology,
      University of Kansas, Lawrence, KS 66045, USA; Bioinformatics and Evolutionary
      Genomics, Universite de Lyon, Villeurbanne, France; Department of Bioinformatics 
      and Genetics, Swedish Museum of Natural History, SE-10405 Stockholm, Sweden; and 
      Department of Biological Science, King Abdulaziz University, Jeddah, Saudi
      Arabia; Sebastian.Hoehna@gmail.com.
FAU - Heath, Tracy A
AU  - Heath TA
AD  - Department of Mathematics, Stockholm University, Stockholm, SE-106 91 Stockholm, 
      Sweden; Department of Evolution and Ecology, University of California, Davis,
      Storer Hall, One Shields Avenue, Davis, CA 95616, USA; Department of Integrative 
      Biology, University of California, Berkeley, CA 94720, USA; Department of Ecology
      and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, USA;
      Bioinformatics and Evolutionary Genomics, Universite de Lyon, Villeurbanne,
      France; Department of Bioinformatics and Genetics, Swedish Museum of Natural
      History, SE-10405 Stockholm, Sweden; and Department of Biological Science, King
      Abdulaziz University, Jeddah, Saudi Arabia;Department of Mathematics, Stockholm
      University, Stockholm, SE-106 91 Stockholm, Sweden; Department of Evolution and
      Ecology, University of California, Davis, Storer Hall, One Shields Avenue, Davis,
      CA 95616, USA; Department of Integrative Biology, University of California,
      Berkeley, CA 94720, USA; Department of Ecology and Evolutionary Biology,
      University of Kansas, Lawrence, KS 66045, USA; Bioinformatics and Evolutionary
      Genomics, Universite de Lyon, Villeurbanne, France; Department of Bioinformatics 
      and Genetics, Swedish Museum of Natural History, SE-10405 Stockholm, Sweden; and 
      Department of Biological Science, King Abdulaziz University, Jeddah, Saudi
      Arabia;
FAU - Boussau, Bastien
AU  - Boussau B
AD  - Department of Mathematics, Stockholm University, Stockholm, SE-106 91 Stockholm, 
      Sweden; Department of Evolution and Ecology, University of California, Davis,
      Storer Hall, One Shields Avenue, Davis, CA 95616, USA; Department of Integrative 
      Biology, University of California, Berkeley, CA 94720, USA; Department of Ecology
      and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, USA;
      Bioinformatics and Evolutionary Genomics, Universite de Lyon, Villeurbanne,
      France; Department of Bioinformatics and Genetics, Swedish Museum of Natural
      History, SE-10405 Stockholm, Sweden; and Department of Biological Science, King
      Abdulaziz University, Jeddah, Saudi Arabia;Department of Mathematics, Stockholm
      University, Stockholm, SE-106 91 Stockholm, Sweden; Department of Evolution and
      Ecology, University of California, Davis, Storer Hall, One Shields Avenue, Davis,
      CA 95616, USA; Department of Integrative Biology, University of California,
      Berkeley, CA 94720, USA; Department of Ecology and Evolutionary Biology,
      University of Kansas, Lawrence, KS 66045, USA; Bioinformatics and Evolutionary
      Genomics, Universite de Lyon, Villeurbanne, France; Department of Bioinformatics 
      and Genetics, Swedish Museum of Natural History, SE-10405 Stockholm, Sweden; and 
      Department of Biological Science, King Abdulaziz University, Jeddah, Saudi
      Arabia;
FAU - Landis, Michael J
AU  - Landis MJ
AD  - Department of Mathematics, Stockholm University, Stockholm, SE-106 91 Stockholm, 
      Sweden; Department of Evolution and Ecology, University of California, Davis,
      Storer Hall, One Shields Avenue, Davis, CA 95616, USA; Department of Integrative 
      Biology, University of California, Berkeley, CA 94720, USA; Department of Ecology
      and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, USA;
      Bioinformatics and Evolutionary Genomics, Universite de Lyon, Villeurbanne,
      France; Department of Bioinformatics and Genetics, Swedish Museum of Natural
      History, SE-10405 Stockholm, Sweden; and Department of Biological Science, King
      Abdulaziz University, Jeddah, Saudi Arabia;
FAU - Ronquist, Fredrik
AU  - Ronquist F
AD  - Department of Mathematics, Stockholm University, Stockholm, SE-106 91 Stockholm, 
      Sweden; Department of Evolution and Ecology, University of California, Davis,
      Storer Hall, One Shields Avenue, Davis, CA 95616, USA; Department of Integrative 
      Biology, University of California, Berkeley, CA 94720, USA; Department of Ecology
      and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, USA;
      Bioinformatics and Evolutionary Genomics, Universite de Lyon, Villeurbanne,
      France; Department of Bioinformatics and Genetics, Swedish Museum of Natural
      History, SE-10405 Stockholm, Sweden; and Department of Biological Science, King
      Abdulaziz University, Jeddah, Saudi Arabia;
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
AD  - Department of Mathematics, Stockholm University, Stockholm, SE-106 91 Stockholm, 
      Sweden; Department of Evolution and Ecology, University of California, Davis,
      Storer Hall, One Shields Avenue, Davis, CA 95616, USA; Department of Integrative 
      Biology, University of California, Berkeley, CA 94720, USA; Department of Ecology
      and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, USA;
      Bioinformatics and Evolutionary Genomics, Universite de Lyon, Villeurbanne,
      France; Department of Bioinformatics and Genetics, Swedish Museum of Natural
      History, SE-10405 Stockholm, Sweden; and Department of Biological Science, King
      Abdulaziz University, Jeddah, Saudi Arabia;Department of Mathematics, Stockholm
      University, Stockholm, SE-106 91 Stockholm, Sweden; Department of Evolution and
      Ecology, University of California, Davis, Storer Hall, One Shields Avenue, Davis,
      CA 95616, USA; Department of Integrative Biology, University of California,
      Berkeley, CA 94720, USA; Department of Ecology and Evolutionary Biology,
      University of Kansas, Lawrence, KS 66045, USA; Bioinformatics and Evolutionary
      Genomics, Universite de Lyon, Villeurbanne, France; Department of Bioinformatics 
      and Genetics, Swedish Museum of Natural History, SE-10405 Stockholm, Sweden; and 
      Department of Biological Science, King Abdulaziz University, Jeddah, Saudi
      Arabia;
LA  - eng
GR  - GM-069801/GM/NIGMS NIH HHS/United States
GR  - GM-086887/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140620
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Models, Statistical
MH  - *Phylogeny
PMC - PMC4184382
EDAT- 2014/06/22 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/06/22 06:00
PHST- 2014/06/22 06:00 [entrez]
PHST- 2014/06/22 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu039 [pii]
AID - 10.1093/sysbio/syu039 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):753-71. doi: 10.1093/sysbio/syu039. Epub 2014 Jun 20.

PMID- 24951558
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20140815
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - Axiomatic opportunities and obstacles for inferring a species tree from gene
      trees.
PG  - 772-8
LID - 10.1093/sysbio/syu043 [doi]
AB  - The reconstruction of a central tendency "species tree" from a large number of
      conflicting gene trees is a central problem in systematic biology. Moreover, it
      becomes particularly problematic when taxon coverage is patchy, so that not all
      taxa are present in every gene tree. Here, we list four apparently desirable
      properties that a method for estimating a species tree from gene trees could have
      (the strongest property states that building a species tree from input gene trees
      and then pruning leaves gives a tree that is the same as, or more resolved than, 
      the tree obtained by first removing the taxa from the input trees and then
      building the species tree). We show that although it is technically possible to
      simultaneously satisfy these properties when taxon coverage is complete, they
      cannot all be satisfied in the more general supertree setting. In part two, we
      discuss a concordance-based consensus method based on Baum's "plurality
      clusters", and an extension to concordance supertrees.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Steel, Mike
AU  - Steel M
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, University of
      Canterbury, Christchurch, New Zealand; and Department of Philosophy, Texas Tech
      University, Box 43092, Lubbock, TX 79409, USA; mike.steel@canterbury.ac.nz.
FAU - Velasco, Joel D
AU  - Velasco JD
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, University of
      Canterbury, Christchurch, New Zealand; and Department of Philosophy, Texas Tech
      University, Box 43092, Lubbock, TX 79409, USA;
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140619
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Sequence Analysis
MH  - Species Specificity
EDAT- 2014/06/22 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/06/22 06:00
PHST- 2014/06/22 06:00 [entrez]
PHST- 2014/06/22 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu043 [pii]
AID - 10.1093/sysbio/syu043 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):772-8. doi: 10.1093/sysbio/syu043. Epub 2014 Jun 19.

PMID- 24951557
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20140815
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - Biogeographic analysis reveals ancient continental vicariance and recent oceanic 
      dispersal in amphibians.
PG  - 779-97
LID - 10.1093/sysbio/syu042 [doi]
AB  - Amphibia comprises over 7000 extant species distributed in almost every ecosystem
      on every continent except Antarctica. Most species also show high specificity for
      particular habitats, biomes, or climatic niches, seemingly rendering
      long-distance dispersal unlikely. Indeed, many lineages still seem to show the
      signature of their Pangaean origin, approximately 300 Ma later. To date, no study
      has attempted a large-scale historical-biogeographic analysis of the group to
      understand the distribution of extant lineages. Here, I use an updated chronogram
      containing 3309 species ( approximately 45% of extant diversity) to reconstruct
      their movement between 12 global ecoregions. I find that Pangaean origin and
      subsequent Laurasian and Gondwanan fragmentation explain a large proportion of
      patterns in the distribution of extant species. However, dispersal during the
      Cenozoic, likely across land bridges or short distances across oceans, has also
      exerted a strong influence. Finally, there are at least three strongly supported 
      instances of long-distance oceanic dispersal between former Gondwanan landmasses 
      during the Cenozoic. Extinction from intervening areas seems to be a strong
      factor in shaping present-day distributions. Dispersal and extinction from and
      between ecoregions are apparently tied to the evolution of extraordinarily
      adaptive expansion-oriented phenotypes that allow lineages to easily colonize new
      areas and diversify, or conversely, to extremely specialized phenotypes or
      heavily relictual climatic niches that result in strong geographic localization
      and limited diversification.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Pyron, R Alexander
AU  - Pyron RA
AD  - Department of Biological Sciences, The George Washington University, 2023 G
      Street NW, Washington, DC 20052, USA; rpyron@colubroid.org.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140619
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amphibians/*classification/genetics/*physiology
MH  - Animal Distribution/*physiology
MH  - Animals
MH  - Biodiversity
MH  - Oceans and Seas
MH  - *Phylogeny
MH  - Phylogeography
EDAT- 2014/06/22 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/06/22 06:00
PHST- 2014/06/22 06:00 [entrez]
PHST- 2014/06/22 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu042 [pii]
AID - 10.1093/sysbio/syu042 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):779-97. doi: 10.1093/sysbio/syu042. Epub 2014 Jun 19.

PMID- 24929898
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20140815
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - Modelling competition and dispersal in a statistical phylogeographic framework.
PG  - 743-52
LID - 10.1093/sysbio/syu040 [doi]
AB  - Competition between organisms influences the processes governing the colonization
      of new habitats. As a consequence, species or populations arriving first at a
      suitable location may prevent secondary colonization. Although adaptation to
      environmental variables (e.g., temperature, altitude, etc.) is essential, the
      presence or absence of certain species at a particular location often depends on 
      whether or not competing species co-occur. For example, competition is thought to
      play an important role in structuring mammalian communities assembly. It can also
      explain spatial patterns of low genetic diversity following rapid colonization
      events or the "progression rule" displayed by phylogenies of species found on
      archipelagos. Despite the potential of competition to maintain populations in
      isolation, past quantitative analyses have largely ignored it because of the
      difficulty in designing adequate methods for assessing its impact. We present
      here a new model that integrates competition and dispersal into a Bayesian
      phylogeographic framework. Extensive simulations and analysis of real data show
      that our approach clearly outperforms the traditional Mantel test for detecting
      correlation between genetic and geographic distances. But most importantly, we
      demonstrate that competition can be detected with high sensitivity and
      specificity from the phylogenetic analysis of genetic variation in space.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Ranjard, Louis
AU  - Ranjard L
AD  - Bioinformatics Institute; Department of Computer Sciences; Department of
      Statistics, The University of Auckland, New Zealand and LIRMM, CNRS, Montpellier,
      France;
FAU - Welch, David
AU  - Welch D
AD  - Bioinformatics Institute; Department of Computer Sciences; Department of
      Statistics, The University of Auckland, New Zealand and LIRMM, CNRS, Montpellier,
      France;
FAU - Paturel, Marie
AU  - Paturel M
AD  - Bioinformatics Institute; Department of Computer Sciences; Department of
      Statistics, The University of Auckland, New Zealand and LIRMM, CNRS, Montpellier,
      France;
FAU - Guindon, Stephane
AU  - Guindon S
AD  - Bioinformatics Institute; Department of Computer Sciences; Department of
      Statistics, The University of Auckland, New Zealand and LIRMM, CNRS, Montpellier,
      France;Bioinformatics Institute; Department of Computer Sciences; Department of
      Statistics, The University of Auckland, New Zealand and LIRMM, CNRS, Montpellier,
      France; s.guindon@auckland.ac.nz.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140613
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Competitive Behavior/physiology
MH  - Ecosystem
MH  - Gryllidae/*classification/genetics
MH  - *Models, Biological
MH  - Phylogeography
MH  - Population Dynamics
EDAT- 2014/06/16 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/06/16 06:00
PHST- 2014/06/16 06:00 [entrez]
PHST- 2014/06/16 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu040 [pii]
AID - 10.1093/sysbio/syu040 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):743-52. doi: 10.1093/sysbio/syu040. Epub 2014 Jun 13.

PMID- 24929897
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - A protocol for species delineation of public DNA databases, applied to the
      Insecta.
PG  - 712-25
LID - 10.1093/sysbio/syu038 [doi]
AB  - Public DNA databases are composed of data from many different taxa, although the 
      taxonomic annotation on sequences is not always complete, which impedes the
      utilization of mined data for species-level applications. There is much ongoing
      work on species identification and delineation based on the molecular data
      itself, although applying species clustering to whole databases requires
      consolidation of results from numerous undefined gene regions, and introduces
      significant obstacles in data organization and computational load. In the current
      paper, we demonstrate an approach for species delineation of a sequence database.
      All DNA sequences for the insects were obtained and processed. After filtration
      of duplicated data, delineation of the database into species or molecular
      operational taxonomic units (MOTUs) followed a three-step process in which (i)
      the genetic loci L are partitioned, (ii) the species S are delineated within each
      locus, then (iii) species units are matched across loci to form the matrix L x S,
      a set of global (multilocus) species units. Partitioning the database into a set 
      of homologous gene fragments was achieved by Markov clustering using edge weights
      calculated from the amount of overlap between pairs of sequences, then
      delineation of species units and assignment of species names were performed for
      the set of genes necessary to capture most of the species diversity. The
      complexity of computing pairwise similarities for species clustering was
      substantial at the cytochrome oxidase subunit I locus in particular, but made
      feasible through the development of software that performs pairwise alignments
      within the taxonomic framework, while accounting for the different ranks at which
      sequences are labeled with taxonomic information. Over 24 different homologs, the
      unidentified sequences numbered approximately 194,000, containing 41,525 species 
      IDs (98.7% of all found in the insect database), and were grouped into 59,173
      single-locus MOTUs by hierarchical clustering under parameters optimized
      independently for each locus. Species units from different loci were matched
      using a multipartite matching algorithm to form multilocus species units with
      minimal incongruence between loci. After matching, the insect database as
      represented by these 24 loci was found to be composed of 78,091 species units in 
      total. 38,574 of these units contained only species labeled data, 34,891
      contained only unlabeled data, leaving 4,626 units composed both of labeled and
      unlabeled sequences. In addition to giving estimates of species diversity of
      sequence repositories, the protocol developed here will facilitate species-level 
      applications of modern-day sequence data sets. In particular, the L x S matrix
      represents a post-taxonomic framework that can be used for species-level
      organization of metagenomic data, and incorporation of these methods into
      phylogenetic pipelines will yield matrices more representative of species
      diversity.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Chesters, Douglas
AU  - Chesters D
AD  - Key Laboratory of Zoological Systematics and Evolution (CAS), Institute of
      Zoology, Chinese Academy of Sciences, Beijing 100101, PR China.
FAU - Zhu, Chao-Dong
AU  - Zhu CD
AD  - Key Laboratory of Zoological Systematics and Evolution (CAS), Institute of
      Zoology, Chinese Academy of Sciences, Beijing 100101, PR China zhucd@ioz.ac.cn.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140614
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Cluster Analysis
MH  - *Databases, Nucleic Acid
MH  - Insecta/*classification/*genetics
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2014/06/16 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/06/16 06:00
PHST- 2014/06/16 06:00 [entrez]
PHST- 2014/06/16 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu038 [pii]
AID - 10.1093/sysbio/syu038 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):712-25. doi: 10.1093/sysbio/syu038. Epub 2014 Jun 14.

PMID- 24927722
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20140815
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - Mixture models of nucleotide sequence evolution that account for heterogeneity in
      the substitution process across sites and across lineages.
PG  - 726-42
LID - 10.1093/sysbio/syu036 [doi]
AB  - Molecular phylogenetic studies of homologous sequences of nucleotides often
      assume that the underlying evolutionary process was globally stationary,
      reversible, and homogeneous (SRH), and that a model of evolution with one or more
      site-specific and time-reversible rate matrices (e.g., the GTR rate matrix) is
      enough to accurately model the evolution of data over the whole tree. However, an
      increasing body of data suggests that evolution under these conditions is an
      exception, rather than the norm. To address this issue, several non-SRH models of
      molecular evolution have been proposed, but they either ignore heterogeneity in
      the substitution process across sites (HAS) or assume it can be modeled
      accurately using the distribution. As an alternative to these models of
      evolution, we introduce a family of mixture models that approximate HAS without
      the assumption of an underlying predefined statistical distribution. This family 
      of mixture models is combined with non-SRH models of evolution that account for
      heterogeneity in the substitution process across lineages (HAL). We also present 
      two algorithms for searching model space and identifying an optimal model of
      evolution that is less likely to over- or underparameterize the data. The
      performance of the two new algorithms was evaluated using alignments of
      nucleotides with 10 000 sites simulated under complex non-SRH conditions on a
      25-tipped tree. The algorithms were found to be very successful, identifying the 
      correct HAL model with a 75% success rate (the average success rate for assigning
      rate matrices to the tree's 48 edges was 99.25%) and, for the correct HAL model, 
      identifying the correct HAS model with a 98% success rate. Finally, parameter
      estimates obtained under the correct HAL-HAS model were found to be accurate and 
      precise. The merits of our new algorithms were illustrated with an analysis of 42
      337 second codon sites extracted from a concatenation of 106 alignments of
      orthologous genes encoded by the nuclear genomes of Saccharomyces cerevisiae, S. 
      paradoxus, S. mikatae, S. kudriavzevii, S. castellii, S. kluyveri, S. bayanus,
      and Candida albicans. Our results show that second codon sites in the ancestral
      genome of these species contained 49.1% invariable sites, 39.6% variable sites
      belonging to one rate category (V1), and 11.3% variable sites belonging to a
      second rate category (V2). The ancestral nucleotide content was found to differ
      markedly across these three sets of sites, and the evolutionary processes
      operating at the variable sites were found to be non-SRH and best modeled by a
      combination of eight edge-specific rate matrices (four for V1 and four for V2).
      The number of substitutions per site at the variable sites also differed
      markedly, with sites belonging to V1 evolving slower than those belonging to V2
      along the lineages separating the seven species of Saccharomyces. Finally, sites 
      belonging to V1 appeared to have ceased evolving along the lineages separating S.
      cerevisiae, S. paradoxus, S. mikatae, S. kudriavzevii, and S. bayanus, implying
      that they might have become so selectively constrained that they could be
      considered invariable sites in these species.
CI  - (c) CSIRO 2014. Published by Oxford University Press, on behalf of the Society of
      Systematic Biologists. All rights reserved. For Permissions, please email:
      journals.permissions@oup.com.
FAU - Jayaswal, Vivek
AU  - Jayaswal V
AD  - School of Biomedical Sciences, Queensland University of Technology, Brisbane, QLD
      4000, Australia; School of Mathematics and Statistics, University of Sydney,
      Sydney, NSW 2006, Australia; CSIRO Ecosystem Sciences, Canberra, ACT 2601,
      Australia; and Centre for Mathematical Biology, University of Sydney, Sydney, NSW
      2006, AustraliaSchool of Biomedical Sciences, Queensland University of
      Technology, Brisbane, QLD 4000, Australia; School of Mathematics and Statistics, 
      University of Sydney, Sydney, NSW 2006, Australia; CSIRO Ecosystem Sciences,
      Canberra, ACT 2601, Australia; and Centre for Mathematical Biology, University of
      Sydney, Sydney, NSW 2006, Australia.
FAU - Wong, Thomas K F
AU  - Wong TK
AD  - School of Biomedical Sciences, Queensland University of Technology, Brisbane, QLD
      4000, Australia; School of Mathematics and Statistics, University of Sydney,
      Sydney, NSW 2006, Australia; CSIRO Ecosystem Sciences, Canberra, ACT 2601,
      Australia; and Centre for Mathematical Biology, University of Sydney, Sydney, NSW
      2006, Australia.
FAU - Robinson, John
AU  - Robinson J
AD  - School of Biomedical Sciences, Queensland University of Technology, Brisbane, QLD
      4000, Australia; School of Mathematics and Statistics, University of Sydney,
      Sydney, NSW 2006, Australia; CSIRO Ecosystem Sciences, Canberra, ACT 2601,
      Australia; and Centre for Mathematical Biology, University of Sydney, Sydney, NSW
      2006, AustraliaSchool of Biomedical Sciences, Queensland University of
      Technology, Brisbane, QLD 4000, Australia; School of Mathematics and Statistics, 
      University of Sydney, Sydney, NSW 2006, Australia; CSIRO Ecosystem Sciences,
      Canberra, ACT 2601, Australia; and Centre for Mathematical Biology, University of
      Sydney, Sydney, NSW 2006, Australia.
FAU - Poladian, Leon
AU  - Poladian L
AD  - School of Biomedical Sciences, Queensland University of Technology, Brisbane, QLD
      4000, Australia; School of Mathematics and Statistics, University of Sydney,
      Sydney, NSW 2006, Australia; CSIRO Ecosystem Sciences, Canberra, ACT 2601,
      Australia; and Centre for Mathematical Biology, University of Sydney, Sydney, NSW
      2006, AustraliaSchool of Biomedical Sciences, Queensland University of
      Technology, Brisbane, QLD 4000, Australia; School of Mathematics and Statistics, 
      University of Sydney, Sydney, NSW 2006, Australia; CSIRO Ecosystem Sciences,
      Canberra, ACT 2601, Australia; and Centre for Mathematical Biology, University of
      Sydney, Sydney, NSW 2006, Australia.
FAU - Jermiin, Lars S
AU  - Jermiin LS
AD  - School of Biomedical Sciences, Queensland University of Technology, Brisbane, QLD
      4000, Australia; School of Mathematics and Statistics, University of Sydney,
      Sydney, NSW 2006, Australia; CSIRO Ecosystem Sciences, Canberra, ACT 2601,
      Australia; and Centre for Mathematical Biology, University of Sydney, Sydney, NSW
      2006, Australia lars.jermiin@csiro.au.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140612
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Base Sequence/genetics
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - *Models, Statistical
MH  - Phylogeny
MH  - Yeasts/classification/genetics
EDAT- 2014/06/15 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/06/15 06:00
PHST- 2014/06/15 06:00 [entrez]
PHST- 2014/06/15 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu036 [pii]
AID - 10.1093/sysbio/syu036 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):726-42. doi: 10.1093/sysbio/syu036. Epub 2014 Jun 12.

PMID- 24852061
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20140815
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - Insights on the evolution of plant succulence from a remarkable radiation in
      Madagascar (Euphorbia).
PG  - 697-711
LID - 10.1093/sysbio/syu035 [doi]
AB  - Patterns of adaptation in response to environmental variation are central to our 
      understanding of biodiversity, but predictions of how and when broad-scale
      environmental conditions such as climate affect organismal form and function
      remain incomplete. Succulent plants have evolved in response to arid conditions
      repeatedly, with various plant organs such as leaves, stems, and roots physically
      modified to increase water storage. Here, we investigate the role played by
      climate conditions in shaping the evolution of succulent forms in a plant clade
      endemic to Madagascar and the surrounding islands, part of the hyper-diverse
      genus Euphorbia (Euphorbiaceae). We used multivariate ordination of 19 climate
      variables to identify links between particular climate variables and three major 
      forms of succulence-succulent leaves, cactiform stem succulence, and tubers. We
      then tested the relationship between climatic conditions and succulence, using
      comparative methods that account for shared evolutionary history. We confirm that
      plant water storage is associated with the two components of aridity,
      temperature, and precipitation. Cactiform stem succulence, however, is not
      prevalent in the driest environments, countering the widely held view of
      cactiforms as desert icons. Instead, leaf succulence and tubers are significantly
      associated with the lowest levels of precipitation. Our findings provide a clear 
      link between broad-scale climatic conditions and adaptation in land plants, and
      new insights into the climatic conditions favoring different forms of succulence.
      This evidence for adaptation to climate raises concern over the evolutionary
      future of succulent plants as they, along with other organisms, face
      anthropogenic climate change.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Evans, Margaret
AU  - Evans M
AD  - Laboratory of Tree-Ring Research, 1215 E Lowell Street and Department of Ecology 
      and Evolutionary Biology, 1041 E Lowell Street, University of Arizona, Tucson, AZ
      85721, USA ISYEB, Institut de Systematique, Evolution, Biodiversite (UMR 7205
      CNRS, MNHN, EPHE, UPMC), Museum national d'histoire naturelle, National
      Herbarium, CP 39, 57 rue Cuvier, 75231 Paris Cedex 05, France Department of Life 
      Sciences, Natural History Museum, Cromwell Road, London SW7 5BD, UK Department of
      Biological Sciences, Towson University, 8000 York Rd, Baltimore, MD 21252, USA
      Laboratoire d'Ecologie Alpine, UMR CNRS 5553, Universite Joseph Fourier, Grenoble
      I, BP 53, 38041 Grenoble Cedex 9, France Genoscope, Centre National de
      Sequencage, CP5706, 2 rue Gaston Cremieux, 91057 Evry Cedex, France; and Missouri
      Botanical Garden, P.O. Box 299, St. Louis, MO 63166, USALaboratory of Tree-Ring
      Research, 1215 E Lowell Street and Department of Ecology and Evolutionary
      Biology, 1041 E Lowell Street, University of Arizona, Tucson, AZ 85721, USA
      ISYEB, Institut de Systematique, Evolution, Biodiversite (UMR 7205 CNRS, MNHN,
      EPHE, UPMC), Museum national d'histoire naturelle, National Herbarium, CP 39, 57 
      rue Cuvier, 75231 Paris Cedex 05, France Department of Life Sciences, Natural
      History Museum, Cromwell Road, London SW7 5BD, UK Department of Biological
      Sciences, Towson University, 8000 York Rd, Baltimore, MD 21252, USA Laboratoire
      d'Ecologie Alpine, UMR CNRS 5553, Universite Joseph Fourier, Grenoble I, BP 53,
      38041 Grenoble Cedex 9, France Genoscope, Centre National de Sequencage, CP5706, 
      2 rue Gaston Cremieux, 91057 Evry Cedex, France; and Missouri Botanical Garden,
      P.O. Box 299, St. Louis, MO 63166, USALaboratory of Tree-Ring Research, 1215 E
      Lowell Street and Department of Ecology and Evolutionary Biology, 1041 E Lowell
      Street, University of Arizona, Tucson, AZ 85721, USA ISYEB, Institut de
      Systematique, Evolution, Biodiversite (UMR 7205 CNRS, MNHN, EPHE, UPMC), Museum
      national d'histoi
FAU - Aubriot, Xavier
AU  - Aubriot X
AD  - Laboratory of Tree-Ring Research, 1215 E Lowell Street and Department of Ecology 
      and Evolutionary Biology, 1041 E Lowell Street, University of Arizona, Tucson, AZ
      85721, USA ISYEB, Institut de Systematique, Evolution, Biodiversite (UMR 7205
      CNRS, MNHN, EPHE, UPMC), Museum national d'histoire naturelle, National
      Herbarium, CP 39, 57 rue Cuvier, 75231 Paris Cedex 05, France Department of Life 
      Sciences, Natural History Museum, Cromwell Road, London SW7 5BD, UK Department of
      Biological Sciences, Towson University, 8000 York Rd, Baltimore, MD 21252, USA
      Laboratoire d'Ecologie Alpine, UMR CNRS 5553, Universite Joseph Fourier, Grenoble
      I, BP 53, 38041 Grenoble Cedex 9, France Genoscope, Centre National de
      Sequencage, CP5706, 2 rue Gaston Cremieux, 91057 Evry Cedex, France; and Missouri
      Botanical Garden, P.O. Box 299, St. Louis, MO 63166, USALaboratory of Tree-Ring
      Research, 1215 E Lowell Street and Department of Ecology and Evolutionary
      Biology, 1041 E Lowell Street, University of Arizona, Tucson, AZ 85721, USA
      ISYEB, Institut de Systematique, Evolution, Biodiversite (UMR 7205 CNRS, MNHN,
      EPHE, UPMC), Museum national d'histoire naturelle, National Herbarium, CP 39, 57 
      rue Cuvier, 75231 Paris Cedex 05, France Department of Life Sciences, Natural
      History Museum, Cromwell Road, London SW7 5BD, UK Department of Biological
      Sciences, Towson University, 8000 York Rd, Baltimore, MD 21252, USA Laboratoire
      d'Ecologie Alpine, UMR CNRS 5553, Universite Joseph Fourier, Grenoble I, BP 53,
      38041 Grenoble Cedex 9, France Genoscope, Centre National de Sequencage, CP5706, 
      2 rue Gaston Cremieux, 91057 Evry Cedex, France; and Missouri Botanical Garden,
      P.O. Box 299, St. Louis, MO 63166, USA.
FAU - Hearn, David
AU  - Hearn D
AD  - Laboratory of Tree-Ring Research, 1215 E Lowell Street and Department of Ecology 
      and Evolutionary Biology, 1041 E Lowell Street, University of Arizona, Tucson, AZ
      85721, USA ISYEB, Institut de Systematique, Evolution, Biodiversite (UMR 7205
      CNRS, MNHN, EPHE, UPMC), Museum national d'histoire naturelle, National
      Herbarium, CP 39, 57 rue Cuvier, 75231 Paris Cedex 05, France Department of Life 
      Sciences, Natural History Museum, Cromwell Road, London SW7 5BD, UK Department of
      Biological Sciences, Towson University, 8000 York Rd, Baltimore, MD 21252, USA
      Laboratoire d'Ecologie Alpine, UMR CNRS 5553, Universite Joseph Fourier, Grenoble
      I, BP 53, 38041 Grenoble Cedex 9, France Genoscope, Centre National de
      Sequencage, CP5706, 2 rue Gaston Cremieux, 91057 Evry Cedex, France; and Missouri
      Botanical Garden, P.O. Box 299, St. Louis, MO 63166, USA.
FAU - Lanciaux, Maxime
AU  - Lanciaux M
AD  - Laboratory of Tree-Ring Research, 1215 E Lowell Street and Department of Ecology 
      and Evolutionary Biology, 1041 E Lowell Street, University of Arizona, Tucson, AZ
      85721, USA ISYEB, Institut de Systematique, Evolution, Biodiversite (UMR 7205
      CNRS, MNHN, EPHE, UPMC), Museum national d'histoire naturelle, National
      Herbarium, CP 39, 57 rue Cuvier, 75231 Paris Cedex 05, France Department of Life 
      Sciences, Natural History Museum, Cromwell Road, London SW7 5BD, UK Department of
      Biological Sciences, Towson University, 8000 York Rd, Baltimore, MD 21252, USA
      Laboratoire d'Ecologie Alpine, UMR CNRS 5553, Universite Joseph Fourier, Grenoble
      I, BP 53, 38041 Grenoble Cedex 9, France Genoscope, Centre National de
      Sequencage, CP5706, 2 rue Gaston Cremieux, 91057 Evry Cedex, France; and Missouri
      Botanical Garden, P.O. Box 299, St. Louis, MO 63166, USA.
FAU - Lavergne, Sebastien
AU  - Lavergne S
AD  - Laboratory of Tree-Ring Research, 1215 E Lowell Street and Department of Ecology 
      and Evolutionary Biology, 1041 E Lowell Street, University of Arizona, Tucson, AZ
      85721, USA ISYEB, Institut de Systematique, Evolution, Biodiversite (UMR 7205
      CNRS, MNHN, EPHE, UPMC), Museum national d'histoire naturelle, National
      Herbarium, CP 39, 57 rue Cuvier, 75231 Paris Cedex 05, France Department of Life 
      Sciences, Natural History Museum, Cromwell Road, London SW7 5BD, UK Department of
      Biological Sciences, Towson University, 8000 York Rd, Baltimore, MD 21252, USA
      Laboratoire d'Ecologie Alpine, UMR CNRS 5553, Universite Joseph Fourier, Grenoble
      I, BP 53, 38041 Grenoble Cedex 9, France Genoscope, Centre National de
      Sequencage, CP5706, 2 rue Gaston Cremieux, 91057 Evry Cedex, France; and Missouri
      Botanical Garden, P.O. Box 299, St. Louis, MO 63166, USA.
FAU - Cruaud, Corinne
AU  - Cruaud C
AD  - Laboratory of Tree-Ring Research, 1215 E Lowell Street and Department of Ecology 
      and Evolutionary Biology, 1041 E Lowell Street, University of Arizona, Tucson, AZ
      85721, USA ISYEB, Institut de Systematique, Evolution, Biodiversite (UMR 7205
      CNRS, MNHN, EPHE, UPMC), Museum national d'histoire naturelle, National
      Herbarium, CP 39, 57 rue Cuvier, 75231 Paris Cedex 05, France Department of Life 
      Sciences, Natural History Museum, Cromwell Road, London SW7 5BD, UK Department of
      Biological Sciences, Towson University, 8000 York Rd, Baltimore, MD 21252, USA
      Laboratoire d'Ecologie Alpine, UMR CNRS 5553, Universite Joseph Fourier, Grenoble
      I, BP 53, 38041 Grenoble Cedex 9, France Genoscope, Centre National de
      Sequencage, CP5706, 2 rue Gaston Cremieux, 91057 Evry Cedex, France; and Missouri
      Botanical Garden, P.O. Box 299, St. Louis, MO 63166, USA.
FAU - Lowry, Porter P 2nd
AU  - Lowry PP 2nd
AD  - Laboratory of Tree-Ring Research, 1215 E Lowell Street and Department of Ecology 
      and Evolutionary Biology, 1041 E Lowell Street, University of Arizona, Tucson, AZ
      85721, USA ISYEB, Institut de Systematique, Evolution, Biodiversite (UMR 7205
      CNRS, MNHN, EPHE, UPMC), Museum national d'histoire naturelle, National
      Herbarium, CP 39, 57 rue Cuvier, 75231 Paris Cedex 05, France Department of Life 
      Sciences, Natural History Museum, Cromwell Road, London SW7 5BD, UK Department of
      Biological Sciences, Towson University, 8000 York Rd, Baltimore, MD 21252, USA
      Laboratoire d'Ecologie Alpine, UMR CNRS 5553, Universite Joseph Fourier, Grenoble
      I, BP 53, 38041 Grenoble Cedex 9, France Genoscope, Centre National de
      Sequencage, CP5706, 2 rue Gaston Cremieux, 91057 Evry Cedex, France; and Missouri
      Botanical Garden, P.O. Box 299, St. Louis, MO 63166, USALaboratory of Tree-Ring
      Research, 1215 E Lowell Street and Department of Ecology and Evolutionary
      Biology, 1041 E Lowell Street, University of Arizona, Tucson, AZ 85721, USA
      ISYEB, Institut de Systematique, Evolution, Biodiversite (UMR 7205 CNRS, MNHN,
      EPHE, UPMC), Museum national d'histoire naturelle, National Herbarium, CP 39, 57 
      rue Cuvier, 75231 Paris Cedex 05, France Department of Life Sciences, Natural
      History Museum, Cromwell Road, London SW7 5BD, UK Department of Biological
      Sciences, Towson University, 8000 York Rd, Baltimore, MD 21252, USA Laboratoire
      d'Ecologie Alpine, UMR CNRS 5553, Universite Joseph Fourier, Grenoble I, BP 53,
      38041 Grenoble Cedex 9, France Genoscope, Centre National de Sequencage, CP5706, 
      2 rue Gaston Cremieux, 91057 Evry Cedex, France; and Missouri Botanical Garden,
      P.O. Box 299, St. Louis, MO 63166, USA.
FAU - Haevermans, Thomas
AU  - Haevermans T
AD  - Laboratory of Tree-Ring Research, 1215 E Lowell Street and Department of Ecology 
      and Evolutionary Biology, 1041 E Lowell Street, University of Arizona, Tucson, AZ
      85721, USA ISYEB, Institut de Systematique, Evolution, Biodiversite (UMR 7205
      CNRS, MNHN, EPHE, UPMC), Museum national d'histoire naturelle, National
      Herbarium, CP 39, 57 rue Cuvier, 75231 Paris Cedex 05, France Department of Life 
      Sciences, Natural History Museum, Cromwell Road, London SW7 5BD, UK Department of
      Biological Sciences, Towson University, 8000 York Rd, Baltimore, MD 21252, USA
      Laboratoire d'Ecologie Alpine, UMR CNRS 5553, Universite Joseph Fourier, Grenoble
      I, BP 53, 38041 Grenoble Cedex 9, France Genoscope, Centre National de
      Sequencage, CP5706, 2 rue Gaston Cremieux, 91057 Evry Cedex, France; and Missouri
      Botanical Garden, P.O. Box 299, St. Louis, MO 63166, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140522
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
SB  - IM
MH  - *Biodiversity
MH  - *Climate
MH  - Euphorbia/*classification/genetics/*physiology
MH  - Genetic Markers/genetics
MH  - Genetic Speciation
MH  - Madagascar
MH  - *Phylogeny
MH  - Plant Leaves/physiology
EDAT- 2014/05/24 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/05/24 06:00
PHST- 2014/05/24 06:00 [entrez]
PHST- 2014/05/24 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu035 [pii]
AID - 10.1093/sysbio/syu035 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):697-711. doi: 10.1093/sysbio/syu035. Epub 2014 May 22.

PMID- 24831669
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - Is diagnosability an indicator of speciation? Response to "Why one century of
      phenetics is enough".
PG  - 833-7
LID - 10.1093/sysbio/syu034 [doi]
FAU - Heller, Rasmus
AU  - Heller R
AD  - Department of Biology, University of Copenhagen, Ole Maaloes Vej 5, DK-2200
      Copenhagen N, Denmark; Instituto Gulbenkian de Ciencia, Rua da Quinta Grande 6,
      P-2780-156 Oeiras, Portugal; Department of Integrative Biology, University of
      California Berkeley, 1005 Valley Life Sciences Building, Berkeley, CA 94720, USA;
      and Centre for GeoGenetics, Natural History Museum, University of Copenhagen,
      Oster Voldgade 5-7, DK-1350 Copenhagen K, DenmarkDepartment of Biology,
      University of Copenhagen, Ole Maaloes Vej 5, DK-2200 Copenhagen N, Denmark;
      Instituto Gulbenkian de Ciencia, Rua da Quinta Grande 6, P-2780-156 Oeiras,
      Portugal; Department of Integrative Biology, University of California Berkeley,
      1005 Valley Life Sciences Building, Berkeley, CA 94720, USA; and Centre for
      GeoGenetics, Natural History Museum, University of Copenhagen, Oster Voldgade
      5-7, DK-1350 Copenhagen K, Denmark rheller@bio.ku.dk.
FAU - Frandsen, Peter
AU  - Frandsen P
AD  - Department of Biology, University of Copenhagen, Ole Maaloes Vej 5, DK-2200
      Copenhagen N, Denmark; Instituto Gulbenkian de Ciencia, Rua da Quinta Grande 6,
      P-2780-156 Oeiras, Portugal; Department of Integrative Biology, University of
      California Berkeley, 1005 Valley Life Sciences Building, Berkeley, CA 94720, USA;
      and Centre for GeoGenetics, Natural History Museum, University of Copenhagen,
      Oster Voldgade 5-7, DK-1350 Copenhagen K, Denmark.
FAU - Lorenzen, Eline Deirdre
AU  - Lorenzen ED
AD  - Department of Biology, University of Copenhagen, Ole Maaloes Vej 5, DK-2200
      Copenhagen N, Denmark; Instituto Gulbenkian de Ciencia, Rua da Quinta Grande 6,
      P-2780-156 Oeiras, Portugal; Department of Integrative Biology, University of
      California Berkeley, 1005 Valley Life Sciences Building, Berkeley, CA 94720, USA;
      and Centre for GeoGenetics, Natural History Museum, University of Copenhagen,
      Oster Voldgade 5-7, DK-1350 Copenhagen K, DenmarkDepartment of Biology,
      University of Copenhagen, Ole Maaloes Vej 5, DK-2200 Copenhagen N, Denmark;
      Instituto Gulbenkian de Ciencia, Rua da Quinta Grande 6, P-2780-156 Oeiras,
      Portugal; Department of Integrative Biology, University of California Berkeley,
      1005 Valley Life Sciences Building, Berkeley, CA 94720, USA; and Centre for
      GeoGenetics, Natural History Museum, University of Copenhagen, Oster Voldgade
      5-7, DK-1350 Copenhagen K, Denmark.
FAU - Siegismund, Hans R
AU  - Siegismund HR
AD  - Department of Biology, University of Copenhagen, Ole Maaloes Vej 5, DK-2200
      Copenhagen N, Denmark; Instituto Gulbenkian de Ciencia, Rua da Quinta Grande 6,
      P-2780-156 Oeiras, Portugal; Department of Integrative Biology, University of
      California Berkeley, 1005 Valley Life Sciences Building, Berkeley, CA 94720, USA;
      and Centre for GeoGenetics, Natural History Museum, University of Copenhagen,
      Oster Voldgade 5-7, DK-1350 Copenhagen K, Denmark.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Comment
DEP - 20140515
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2013 May 1;62(3):490-3. PMID: 23362112
CON - Syst Biol. 2014 Sep;63(5):819-32. PMID: 24415680
MH  - Animals
MH  - *Genetic Speciation
MH  - Ruminants/*classification/*genetics
EDAT- 2014/05/17 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/05/17 06:00
PHST- 2014/05/17 06:00 [entrez]
PHST- 2014/05/17 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - sysbio/syu034 [pii]
AID - 10.1093/sysbio/syu034 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):833-7. doi: 10.1093/sysbio/syu034. Epub 2014 May 15.

PMID- 24817532
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20180226
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - Integrating incomplete fossils by isolating conflicting signal in saturated and
      non-independent morphological characters.
PG  - 582-600
LID - 10.1093/sysbio/syu022 [doi]
AB  - Morphological characters are indispensable in phylogenetic analyses for
      understanding the pattern, process, and tempo of evolution. If characters are
      independent and free of systematic errors, then combining as many different kinds
      of characters as are available will result in the best-supported phylogenetic
      hypotheses. But since morphological characters are subject to natural selection
      for function and arise from the expression of developmental pathways, they may
      not be independent, a situation that may amplify any underlying homoplasy. Here, 
      we use new dental and multi-locus genetic data from bats (Mammalia: Chiroptera)
      to quantify saturation and similarity in morphological characters and introduce
      two likelihood-based approaches to identify strongly conflicting characters and
      integrate morphological and molecular data. We implement these methods to analyze
      the phylogeny of incomplete Miocene fossils in the radiation of Phyllostomidae
      (New World Leaf-nosed Bats), perhaps the most ecologically diverse family of
      living mammals. Morphological characters produced trees incongruent with
      molecular phylogenies, were saturated, and showed rates of change higher than
      most molecular substitution rates. Dental characters encoded variation similar to
      that in other morphological characters, while molecular characters encoded highly
      dissimilar variation in comparison. Saturation and high rates of change indicate 
      randomization of phylogenetic signal in the morphological data, and extensive
      similarity suggests characters are non-independent and errors are amplified. To
      integrate the morphological data into tree building while accounting for
      homoplasy, we used statistical molecular scaffolds and combined phylogenetic
      analyses excluding a small subset of strongly conflicting dental characters. The 
      phylogenies revealed the Miocene nectar-feeding daggerPalynephyllum nests within 
      the crown nectar-feeding South American subfamily Lonchophyllinae, while the
      Miocene genus daggerNotonycteris is sister to the extant carnivorous Vampyrum.
      These relationships imply new calibration points for timing of radiation of the
      ecologically diverse Phyllostomidae. [Chiroptera; conflict; dentition;
      morphology; Phyllostomidae; saturation; scaffold; systematic error.].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Davalos, Liliana M
AU  - Davalos LM
AD  - Department of Ecology and Evolution, Stony Brook University, 650 Life Sciences
      Building Stony Brook, NY 11794, USA; Consortium for Inter-Disciplinary
      Environmental Research, School of Marine and Atmospheric Sciences, Stony Brook
      University, 129 Dana Hall, Stony Brook, NY 11794, USA; Division of Vertebrate
      Zoology (Mammalogy), American Museum of Natural History, NY 10024, USA, and
      Committee on Evolutionary Biology, University of Chicago, 1025 E. 57th Street,
      Culver Hall 402, Chicago, IL 60637, USADepartment of Ecology and Evolution, Stony
      Brook University, 650 Life Sciences Building Stony Brook, NY 11794, USA;
      Consortium for Inter-Disciplinary Environmental Research, School of Marine and
      Atmospheric Sciences, Stony Brook University, 129 Dana Hall, Stony Brook, NY
      11794, USA; Division of Vertebrate Zoology (Mammalogy), American Museum of
      Natural History, NY 10024, USA, and Committee on Evolutionary Biology, University
      of Chicago, 1025 E. 57th Street, Culver Hall 402, Chicago, IL 60637, USA
      liliana.davalos-alvarez@stonybrook.edu.
FAU - Velazco, Paul M
AU  - Velazco PM
AD  - Department of Ecology and Evolution, Stony Brook University, 650 Life Sciences
      Building Stony Brook, NY 11794, USA; Consortium for Inter-Disciplinary
      Environmental Research, School of Marine and Atmospheric Sciences, Stony Brook
      University, 129 Dana Hall, Stony Brook, NY 11794, USA; Division of Vertebrate
      Zoology (Mammalogy), American Museum of Natural History, NY 10024, USA, and
      Committee on Evolutionary Biology, University of Chicago, 1025 E. 57th Street,
      Culver Hall 402, Chicago, IL 60637, USA.
FAU - Warsi, Omar M
AU  - Warsi OM
AD  - Department of Ecology and Evolution, Stony Brook University, 650 Life Sciences
      Building Stony Brook, NY 11794, USA; Consortium for Inter-Disciplinary
      Environmental Research, School of Marine and Atmospheric Sciences, Stony Brook
      University, 129 Dana Hall, Stony Brook, NY 11794, USA; Division of Vertebrate
      Zoology (Mammalogy), American Museum of Natural History, NY 10024, USA, and
      Committee on Evolutionary Biology, University of Chicago, 1025 E. 57th Street,
      Culver Hall 402, Chicago, IL 60637, USA.
FAU - Smits, Peter D
AU  - Smits PD
AD  - Department of Ecology and Evolution, Stony Brook University, 650 Life Sciences
      Building Stony Brook, NY 11794, USA; Consortium for Inter-Disciplinary
      Environmental Research, School of Marine and Atmospheric Sciences, Stony Brook
      University, 129 Dana Hall, Stony Brook, NY 11794, USA; Division of Vertebrate
      Zoology (Mammalogy), American Museum of Natural History, NY 10024, USA, and
      Committee on Evolutionary Biology, University of Chicago, 1025 E. 57th Street,
      Culver Hall 402, Chicago, IL 60637, USA.
FAU - Simmons, Nancy B
AU  - Simmons NB
AD  - Department of Ecology and Evolution, Stony Brook University, 650 Life Sciences
      Building Stony Brook, NY 11794, USA; Consortium for Inter-Disciplinary
      Environmental Research, School of Marine and Atmospheric Sciences, Stony Brook
      University, 129 Dana Hall, Stony Brook, NY 11794, USA; Division of Vertebrate
      Zoology (Mammalogy), American Museum of Natural History, NY 10024, USA, and
      Committee on Evolutionary Biology, University of Chicago, 1025 E. 57th Street,
      Culver Hall 402, Chicago, IL 60637, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140509
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2017 Jul 1;66(4):658-659. PMID: 28204803
MH  - Animals
MH  - Chiroptera/*anatomy &amp; histology/*classification/genetics
MH  - Classification
MH  - Exons/genetics
MH  - *Fossils
MH  - Genes, Mitochondrial/genetics
MH  - Introns/genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Tooth/anatomy &amp; histology
EDAT- 2014/05/13 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/05/13 06:00
PHST- 2014/05/13 06:00 [entrez]
PHST- 2014/05/13 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu022 [pii]
AID - 10.1093/sysbio/syu022 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):582-600. doi: 10.1093/sysbio/syu022. Epub 2014 May 9.

PMID- 24789073
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20140815
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - A generalized K statistic for estimating phylogenetic signal from shape and other
      high-dimensional multivariate data.
PG  - 685-97
LID - 10.1093/sysbio/syu030 [doi]
AB  - Phylogenetic signal is the tendency for closely related species to display
      similar trait values due to their common ancestry. Several methods have been
      developed for quantifying phylogenetic signal in univariate traits and for sets
      of traits treated simultaneously, and the statistical properties of these
      approaches have been extensively studied. However, methods for assessing
      phylogenetic signal in high-dimensional multivariate traits like shape are less
      well developed, and their statistical performance is not well characterized. In
      this article, I describe a generalization of the K statistic of Blomberg et al.
      that is useful for quantifying and evaluating phylogenetic signal in highly
      dimensional multivariate data. The method (K(mult)) is found from the equivalency
      between statistical methods based on covariance matrices and those based on
      distance matrices. Using computer simulations based on Brownian motion, I
      demonstrate that the expected value of K(mult) remains at 1.0 as trait variation 
      among species is increased or decreased, and as the number of trait dimensions is
      increased. By contrast, estimates of phylogenetic signal found with a
      squared-change parsimony procedure for multivariate data change with increasing
      trait variation among species and with increasing numbers of trait dimensions,
      confounding biological interpretations. I also evaluate the statistical
      performance of hypothesis testing procedures based on K(mult) and find that the
      method displays appropriate Type I error and high statistical power for detecting
      phylogenetic signal in high-dimensional data. Statistical properties of K(mult)
      were consistent for simulations using bifurcating and random phylogenies, for
      simulations using different numbers of species, for simulations that varied the
      number of trait dimensions, and for different underlying models of trait
      covariance structure. Overall these findings demonstrate that K(mult) provides a 
      useful means of evaluating phylogenetic signal in high-dimensional multivariate
      traits. Finally, I illustrate the utility of the new approach by evaluating the
      strength of phylogenetic signal for head shape in a lineage of Plethodon
      salamanders.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Adams, Dean C
AU  - Adams DC
AD  - Department of Ecology, Evolution, and Organismal Biology, and Department of
      Statistics, Iowa State University, Ames IA, 50011, USA dcadams@iastate.edu.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140430
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - Head/*anatomy &amp; histology
MH  - *Models, Statistical
MH  - Multivariate Analysis
MH  - Phylogeny
MH  - Urodela/*anatomy &amp; histology/*classification
EDAT- 2014/05/03 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/05/03 06:00
PHST- 2014/05/03 06:00 [entrez]
PHST- 2014/05/03 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu030 [pii]
AID - 10.1093/sysbio/syu030 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):685-97. doi: 10.1093/sysbio/syu030. Epub 2014 Apr 30.

PMID- 24789072
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - A gateway for phylogenetic analysis powered by grid computing featuring GARLI
      2.0.
PG  - 812-8
LID - 10.1093/sysbio/syu031 [doi]
AB  - We introduce molecularevolution.org, a publicly available gateway for
      high-throughput, maximum-likelihood phylogenetic analysis powered by grid
      computing. The gateway features a garli 2.0 web service that enables a user to
      quickly and easily submit thousands of maximum likelihood tree searches or
      bootstrap searches that are executed in parallel on distributed computing
      resources. The garli web service allows one to easily specify partitioned
      substitution models using a graphical interface, and it performs sophisticated
      post-processing of phylogenetic results. Although the garli web service has been 
      used by the research community for over three years, here we formally announce
      the availability of the service, describe its capabilities, highlight new
      features and recent improvements, and provide details about how the grid system
      efficiently delivers high-quality phylogenetic results.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Bazinet, Adam L
AU  - Bazinet AL
AD  - Center for Bioinformatics and Computational Biology, University of Maryland,
      College Park, MD, 20742-3360, USA, and Department of Ecology and Evolutionary
      Biology, University of Arizona, Tucson, AZ, 85721-0088, USA
      adam.bazinet@umiacs.umd.edu.
FAU - Zwickl, Derrick J
AU  - Zwickl DJ
AD  - Center for Bioinformatics and Computational Biology, University of Maryland,
      College Park, MD, 20742-3360, USA, and Department of Ecology and Evolutionary
      Biology, University of Arizona, Tucson, AZ, 85721-0088, USA.
FAU - Cummings, Michael P
AU  - Cummings MP
AD  - Center for Bioinformatics and Computational Biology, University of Maryland,
      College Park, MD, 20742-3360, USA, and Department of Ecology and Evolutionary
      Biology, University of Arizona, Tucson, AZ, 85721-0088, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140430
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Access to Information
MH  - Classification/*methods
MH  - Internet
MH  - *Phylogeny
MH  - *Software
PMC - PMC4141202
EDAT- 2014/05/03 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/05/03 06:00
PHST- 2014/05/03 06:00 [entrez]
PHST- 2014/05/03 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu031 [pii]
AID - 10.1093/sysbio/syu031 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):812-8. doi: 10.1093/sysbio/syu031. Epub 2014 Apr 30.

PMID- 24733412
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20140815
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - A rich fossil record yields calibrated phylogeny for Acanthaceae (Lamiales) and
      evidence for marked biases in timing and directionality of intercontinental
      disjunctions.
PG  - 660-84
LID - 10.1093/sysbio/syu029 [doi]
AB  - More than a decade of phylogenetic research has yielded a well-sampled, strongly 
      supported hypothesis of relationships within the large ( &gt; 4000 species) plant
      family Acanthaceae. This hypothesis points to intriguing biogeographic patterns
      and asymmetries in sister clade diversity but, absent a time-calibrated estimate 
      for this evolutionary history, these patterns have remained unexplored. Here, we 
      reconstruct divergence times within Acanthaceae using fossils as calibration
      points and experimenting with both fossil selection and effects of invoking a
      maximum age prior related to the origin of Eudicots. Contrary to earlier reports 
      of a paucity of fossils of Lamiales (an order of approximately 23,000 species
      that includes Acanthaceae) and to the expectation that a largely herbaceous to
      soft-wooded and tropical lineage would have few fossils, we recovered 51 reports 
      of fossil Acanthaceae. Rigorous evaluation of these for accurate identification, 
      quality of age assessment and utility in dating yielded eight fossils judged to
      merit inclusion in analyses. With nearly 10 kb of DNA sequence data, we used two 
      sets of fossils as constraints to reconstruct divergence times. We demonstrate
      differences in age estimates depending on fossil selection and that enforcement
      of maximum age priors substantially alters estimated clade ages, especially in
      analyses that utilize a smaller rather than larger set of fossils. Our results
      suggest that long-distance dispersal events explain present-day distributions
      better than do Gondwanan or northern land bridge hypotheses. This biogeographical
      conclusion is for the most part robust to alternative calibration schemes. Our
      data support a minimum of 13 Old World (OW) to New World (NW) dispersal events
      but, intriguingly, only one in the reverse direction. Eleven of these 13 were
      among Acanthaceae s.s., which comprises &gt; 90% of species diversity in the family.
      Remarkably, if minimum age estimates approximate true history, these 11 events
      occurred within the last approximately 20 myr even though Acanthaceae s.s is over
      3 times as old. A simulation study confirmed that these dispersal events were
      significantly skewed toward the present and not simply a chance occurrence.
      Finally, we review reports of fossils that have been assigned to Acanthaceae that
      are substantially older than the lower Cretaceous estimate for Angiosperms as a
      whole (i.e., the general consensus that has resulted from several recent dating
      and fossil-based studies in plants). This is the first study to reconstruct
      divergence times among clades of Acanthaceae and sets the stage for comparative
      evolutionary research in this and related families that have until now been
      thought to have extremely poor fossil resources.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Tripp, Erin A
AU  - Tripp EA
AD  - Department of Ecology and Evolutionary Biology and Museum of Natural History,
      University of Colorado, UCB 350, Boulder, CO 80309, USA erin.tripp@colorado.edu.
FAU - McDade, Lucinda A
AU  - McDade LA
AD  - Rancho Santa Ana Botanic Garden, 1500 N. College Avenue, Claremont, CA 91711,
      USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140414
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal Spacer)
SB  - IM
MH  - Acanthaceae/*classification/genetics
MH  - DNA, Ribosomal Spacer/genetics
MH  - *Fossils
MH  - Genes, Plant/genetics
MH  - Phylogeny
MH  - Pollen/cytology
MH  - Time
EDAT- 2014/04/16 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/04/16 06:00
PHST- 2014/04/16 06:00 [entrez]
PHST- 2014/04/16 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu029 [pii]
AID - 10.1093/sysbio/syu029 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):660-84. doi: 10.1093/sysbio/syu029. Epub 2014 Apr 14.

PMID- 24721692
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20151119
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - Disentangling methodological and biological sources of gene tree discordance on
      Oryza (Poaceae) chromosome 3.
PG  - 645-59
LID - 10.1093/sysbio/syu027 [doi]
AB  - We describe new methods for characterizing gene tree discordance in phylogenomic 
      data sets, which screen for deviations from neutral expectations, summarize
      variation in statistical support among gene trees, and allow comparison of the
      patterns of discordance induced by various analysis choices. Using an
      exceptionally complete set of genome sequences for the short arm of chromosome 3 
      in Oryza (rice) species, we applied these methods to identify the causes and
      consequences of differing patterns of discordance in the sets of gene trees
      inferred using a panel of 20 distinct analysis pipelines. We found that
      discordance patterns were strongly affected by aspects of data selection,
      alignment, and alignment masking. Unusual patterns of discordance evident when
      using certain pipelines were reduced or eliminated by using alternative
      pipelines, suggesting that they were the product of methodological biases rather 
      than evolutionary processes. In some cases, once such biases were eliminated,
      evolutionary processes such as introgression could be implicated. Additionally,
      patterns of gene tree discordance had significant downstream impacts on species
      tree inference. For example, inference from supermatrices was positively
      misleading when pipelines that led to biased gene trees were used. Several
      results may generalize to other data sets: we found that gene tree and species
      tree inference gave more reasonable results when intron sequence was included
      during sequence alignment and tree inference, the alignment software PRANK was
      used, and detectable "block-shift" alignment artifacts were removed. We discuss
      our findings in the context of well-established relationships in Oryza and
      continuing controversies regarding the domestication history of O. sativa.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Zwickl, Derrick J
AU  - Zwickl DJ
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724, USA,
      School of Plant Sciences, University of Arizona, Tucson, AZ 85721, USA, and
      Robert W. Holley Center for Agriculture and Health, United States Department of
      Agriculture-Agricultural Research Service, Ithaca, NY 14853, USA
      zwickl@email.arizona.edu.
FAU - Stein, Joshua C
AU  - Stein JC
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724, USA,
      School of Plant Sciences, University of Arizona, Tucson, AZ 85721, USA, and
      Robert W. Holley Center for Agriculture and Health, United States Department of
      Agriculture-Agricultural Research Service, Ithaca, NY 14853, USA.
FAU - Wing, Rod A
AU  - Wing RA
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724, USA,
      School of Plant Sciences, University of Arizona, Tucson, AZ 85721, USA, and
      Robert W. Holley Center for Agriculture and Health, United States Department of
      Agriculture-Agricultural Research Service, Ithaca, NY 14853, USA.
FAU - Ware, Doreen
AU  - Ware D
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724, USA,
      School of Plant Sciences, University of Arizona, Tucson, AZ 85721, USA, and
      Robert W. Holley Center for Agriculture and Health, United States Department of
      Agriculture-Agricultural Research Service, Ithaca, NY 14853, USADepartment of
      Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ 85721, USA,
      Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724, USA, School of Plant
      Sciences, University of Arizona, Tucson, AZ 85721, USA, and Robert W. Holley
      Center for Agriculture and Health, United States Department of
      Agriculture-Agricultural Research Service, Ithaca, NY 14853, USA.
FAU - Sanderson, Michael J
AU  - Sanderson MJ
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724, USA,
      School of Plant Sciences, University of Arizona, Tucson, AZ 85721, USA, and
      Robert W. Holley Center for Agriculture and Health, United States Department of
      Agriculture-Agricultural Research Service, Ithaca, NY 14853, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140409
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Chromosomes, Plant/*genetics
MH  - Classification/*methods
MH  - Genome, Plant/genetics
MH  - Oryza/*classification/*genetics
MH  - *Phylogeny
EDAT- 2014/04/12 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/04/12 06:00
PHST- 2014/04/12 06:00 [entrez]
PHST- 2014/04/12 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu027 [pii]
AID - 10.1093/sysbio/syu027 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):645-59. doi: 10.1093/sysbio/syu027. Epub 2014 Apr 9.

PMID- 24695589
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - Supertrees Based on the Subtree Prune-and-Regraft Distance.
PG  - 566-81
LID - 10.1093/sysbio/syu023 [doi]
AB  - Supertree methods reconcile a set of phylogenetic trees into a single structure
      that is often interpreted as a branching history of species. A key challenge is
      combining conflicting evolutionary histories that are due to artifacts of
      phylogenetic reconstruction and phenomena such as lateral gene transfer (LGT).
      Many supertree approaches use optimality criteria that do not reflect underlying 
      processes, have known biases, and may be unduly influenced by LGT. We present the
      first method to construct supertrees by using the subtree prune-and-regraft (SPR)
      distance as an optimality criterion. Although calculating the rooted SPR distance
      between a pair of trees is NP-hard, our new maximum agreement forest-based
      methods can reconcile trees with hundreds of taxa and&gt;50 transfers in fractions
      of a second, which enables repeated calculations during the course of an
      iterative search. Our approach can accommodate trees in which uncertain
      relationships have been collapsed to multifurcating nodes. Using a series of
      benchmark datasets simulated under plausible rates of LGT, we show that SPR
      supertrees are more similar to correct species histories than supertrees based on
      parsimony or Robinson-Foulds distance criteria. We successfully constructed an
      SPR supertree from a phylogenomic dataset of 40,631 gene trees that covered 244
      genomes representing several major bacterial phyla. Our SPR-based approach also
      allowed direct inference of highways of gene transfer between bacterial classes
      and genera. A Small number of these highways connect genera in different phyla
      and can highlight specific genes implicated in long-distance LGT. [Lateral gene
      transfer; matrix representation with parsimony; phylogenomics; prokaryotic
      phylogeny; Robinson-Foulds; subtree prune-and-regraft; supertrees.].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Whidden, Christopher
AU  - Whidden C
AD  - Faculty of Computer Science, Dalhousie University, 6050 University Avenue, PO Box
      15000, Halifax, Nova Scotia, Canada B3H 4R2.
FAU - Zeh, Norbert
AU  - Zeh N
AD  - Faculty of Computer Science, Dalhousie University, 6050 University Avenue, PO Box
      15000, Halifax, Nova Scotia, Canada B3H 4R2.
FAU - Beiko, Robert G
AU  - Beiko RG
AD  - Faculty of Computer Science, Dalhousie University, 6050 University Avenue, PO Box
      15000, Halifax, Nova Scotia, Canada B3H 4R2 beiko@cs.dal.ca.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140402
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Bacteria/*classification/genetics
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - Gene Transfer, Horizontal
MH  - Genome, Bacterial/genetics
MH  - *Phylogeny
MH  - Reproducibility of Results
PMC - PMC4055872
EDAT- 2014/04/04 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/04/04 06:00
PHST- 2014/04/04 06:00 [entrez]
PHST- 2014/04/04 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu023 [pii]
AID - 10.1093/sysbio/syu023 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):566-81. doi: 10.1093/sysbio/syu023. Epub 2014 Apr 2.

PMID- 24682414
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - Ribosomal DNA sequence heterogeneity reflects intraspecies phylogenies and
      predicts genome structure in two contrasting yeast species.
PG  - 543-54
LID - 10.1093/sysbio/syu019 [doi]
AB  - The ribosomal RNA encapsulates a wealth of evolutionary information, including
      genetic variation that can be used to discriminate between organisms at a wide
      range of taxonomic levels. For example, the prokaryotic 16S rDNA sequence is very
      widely used both in phylogenetic studies and as a marker in metagenomic surveys
      and the internal transcribed spacer region, frequently used in plant
      phylogenetics, is now recognized as a fungal DNA barcode. However, this
      widespread use does not escape criticism, principally due to issues such as
      difficulties in classification of paralogous versus orthologous rDNA units and
      intragenomic variation, both of which may be significant barriers to accurate
      phylogenetic inference. We recently analyzed data sets from the Saccharomyces
      Genome Resequencing Project, characterizing rDNA sequence variation within
      multiple strains of the baker's yeast Saccharomyces cerevisiae and its nearest
      wild relative Saccharomyces paradoxus in unprecedented detail. Notably, both
      species possess single locus rDNA systems. Here, we use these new variation
      datasets to assess whether a more detailed characterization of the rDNA locus can
      alleviate the second of these phylogenetic issues, sequence heterogeneity, while 
      controlling for the first. We demonstrate that a strong phylogenetic signal
      exists within both datasets and illustrate how they can be used, with existing
      methodology, to estimate intraspecies phylogenies of yeast strains consistent
      with those derived from whole-genome approaches. We also describe the use of
      partial Single Nucleotide Polymorphisms, a type of sequence variation found only 
      in repetitive genomic regions, in identifying key evolutionary features such as
      genome hybridization events and show their consistency with whole-genome
      Structure analyses. We conclude that our approach can transform rDNA sequence
      heterogeneity from a problem to a useful source of evolutionary information,
      enabling the estimation of highly accurate phylogenies of closely related
      organisms, and discuss how it could be extended to future studies of multilocus
      rDNA systems. [concerted evolution; genome hydridisation; phylogenetic analysis; 
      ribosomal DNA; whole genome sequencing; yeast].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - West, Claire
AU  - West C
AD  - National Collection of Yeast Cultures, Institute of Food Research, Norwich
      Research Park, Norwich NR4 7UA, UK; Bioinformatics, The Genome Analysis Centre,
      Norwich Research Park, Norwich NR4 7UH, UK; Department of Computational and
      Systems Biology, John Innes Centre, Norwich Research Park, Norwich NR4 7UH, UK.
FAU - James, Stephen A
AU  - James SA
AD  - National Collection of Yeast Cultures, Institute of Food Research, Norwich
      Research Park, Norwich NR4 7UA, UK; Bioinformatics, The Genome Analysis Centre,
      Norwich Research Park, Norwich NR4 7UH, UK; Department of Computational and
      Systems Biology, John Innes Centre, Norwich Research Park, Norwich NR4 7UH, UK.
FAU - Davey, Robert P
AU  - Davey RP
AD  - National Collection of Yeast Cultures, Institute of Food Research, Norwich
      Research Park, Norwich NR4 7UA, UK; Bioinformatics, The Genome Analysis Centre,
      Norwich Research Park, Norwich NR4 7UH, UK; Department of Computational and
      Systems Biology, John Innes Centre, Norwich Research Park, Norwich NR4 7UH,
      UKNational Collection of Yeast Cultures, Institute of Food Research, Norwich
      Research Park, Norwich NR4 7UA, UK; Bioinformatics, The Genome Analysis Centre,
      Norwich Research Park, Norwich NR4 7UH, UK; Department of Computational and
      Systems Biology, John Innes Centre, Norwich Research Park, Norwich NR4 7UH, UK.
FAU - Dicks, Jo
AU  - Dicks J
AD  - National Collection of Yeast Cultures, Institute of Food Research, Norwich
      Research Park, Norwich NR4 7UA, UK; Bioinformatics, The Genome Analysis Centre,
      Norwich Research Park, Norwich NR4 7UH, UK; Department of Computational and
      Systems Biology, John Innes Centre, Norwich Research Park, Norwich NR4 7UH,
      UKNational Collection of Yeast Cultures, Institute of Food Research, Norwich
      Research Park, Norwich NR4 7UA, UK; Bioinformatics, The Genome Analysis Centre,
      Norwich Research Park, Norwich NR4 7UH, UK; Department of Computational and
      Systems Biology, John Innes Centre, Norwich Research Park, Norwich NR4 7UH, UK.
FAU - Roberts, Ian N
AU  - Roberts IN
AD  - National Collection of Yeast Cultures, Institute of Food Research, Norwich
      Research Park, Norwich NR4 7UA, UK; Bioinformatics, The Genome Analysis Centre,
      Norwich Research Park, Norwich NR4 7UH, UK; Department of Computational and
      Systems Biology, John Innes Centre, Norwich Research Park, Norwich NR4 7UH, UK
      ian.roberts@ifr.ac.uk.
LA  - eng
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140327
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Fungal)
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - DNA, Fungal/genetics
MH  - DNA, Ribosomal/*genetics
MH  - *Genetic Heterogeneity
MH  - Genetic Variation
MH  - Genome, Fungal/*genetics
MH  - *Phylogeny
MH  - Polymorphism, Single Nucleotide/genetics
MH  - Saccharomyces/*classification/*genetics
PMC - PMC4055870
EDAT- 2014/04/01 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/04/01 06:00
PHST- 2014/04/01 06:00 [entrez]
PHST- 2014/04/01 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu019 [pii]
AID - 10.1093/sysbio/syu019 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):543-54. doi: 10.1093/sysbio/syu019. Epub 2014 Mar 27.

PMID- 24682413
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20141204
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - A preliminary framework for DNA barcoding, incorporating the multispecies
      coalescent.
PG  - 639-44
LID - 10.1093/sysbio/syu028 [doi]
FAU - Dowton, Mark
AU  - Dowton M
AD  - Centre for Medical and Molecular Bioscience; Institute for Conservation Biology
      and Environmental Management, School of Biological Sciences, University of
      Wollongong, NSW 2522; and School of Earth, Environmental and Biological Sciences,
      Queensland University of Technology, QLD 4001, Australia mdowton@uow.edu.au.
FAU - Meiklejohn, Kelly
AU  - Meiklejohn K
AD  - Centre for Medical and Molecular Bioscience; Institute for Conservation Biology
      and Environmental Management, School of Biological Sciences, University of
      Wollongong, NSW 2522; and School of Earth, Environmental and Biological Sciences,
      Queensland University of Technology, QLD 4001, Australia.
FAU - Cameron, Stephen L
AU  - Cameron SL
AD  - Centre for Medical and Molecular Bioscience; Institute for Conservation Biology
      and Environmental Management, School of Biological Sciences, University of
      Wollongong, NSW 2522; and School of Earth, Environmental and Biological Sciences,
      Queensland University of Technology, QLD 4001, Australia.
FAU - Wallman, James
AU  - Wallman J
AD  - Centre for Medical and Molecular Bioscience; Institute for Conservation Biology
      and Environmental Management, School of Biological Sciences, University of
      Wollongong, NSW 2522; and School of Earth, Environmental and Biological Sciences,
      Queensland University of Technology, QLD 4001, Australia.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140328
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2014 Nov;63(6):1005-9. PMID: 25116917
MH  - Classification/*methods
MH  - DNA Barcoding, Taxonomic/*standards/trends
MH  - Databases, Nucleic Acid
MH  - *Phylogeny
EDAT- 2014/04/01 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/04/01 06:00
PHST- 2014/04/01 06:00 [entrez]
PHST- 2014/04/01 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu028 [pii]
AID - 10.1093/sysbio/syu028 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):639-44. doi: 10.1093/sysbio/syu028. Epub 2014 Mar 28.

PMID- 24682412
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20140614
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - Analysis and visualization of complex macroevolutionary dynamics: an example from
      Australian scincid lizards.
PG  - 610-27
LID - 10.1093/sysbio/syu025 [doi]
AB  - The correlation between species diversification and morphological evolution has
      long been of interest in evolutionary biology. We investigated the relationship
      between these processes during the radiation of 250+scincid lizards that
      constitute Australia's most species-rich clade of terrestrial vertebrates. We
      generated a time-calibrated phylogenetic tree for the group that was more than
      85% complete at the species level and collected multivariate morphometric data
      for 183 species. We reconstructed the dynamics of species diversification and
      trait evolution using a Bayesian statistical framework (BAMM) that simultaneously
      accounts for variation in evolutionary rates through time and among lineages. We 
      extended the BAMM model to accommodate time-dependent phenotypic evolution, and
      we describe several new methods for summarizing and visualizing macroevolutionary
      rate heterogeneity on phylogenetic trees. Two major clades (Lerista, Ctenotus;
      &gt;90 spp. each) are associated with high rates of species diversification relative
      to the background rate across Australian sphenomorphine skinks. The Lerista clade
      is characterized by relatively high lability of body form and has undergone
      repeated instances of limb reduction, but Ctenotus is characterized by an extreme
      deceleration in the rate of body shape evolution. We estimate that rates of
      phenotypic evolution decreased by more than an order of magnitude in the common
      ancestor of the Ctenotus clade. These results provide evidence for a modal shift 
      in phenotypic evolutionary dynamics and demonstrate that major axes of
      morphological variation can be decoupled from species diversification. More
      generally, the Bayesian framework described here can be used to identify and
      characterize complex mixtures of dynamic processes on phylogenetic trees.
      [Bayesian; diversification; evolvability; lizard; macroevolution, punctuated
      equilibrium, speciation.].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Rabosky, Daniel L
AU  - Rabosky DL
AD  - Museum of Zoology, University of Michigan, Ann Arbor, MI 48109, USA; Department
      of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, MI 48109,
      USA; South Australian Museum, North Terrace, Adelaide 5000, Australia; Australian
      Centre for Evolutionary Biology and Biodiversity, University of Adelaide,
      Adelaide 5005, Australia; Cornell Lab of Ornithology, Cornell University, Ithaca,
      New York 14850, USA;Museum of Zoology, University of Michigan, Ann Arbor, MI
      48109, USA; Department of Ecology and Evolutionary Biology, University of
      Michigan, Ann Arbor, MI 48109, USA; South Australian Museum, North Terrace,
      Adelaide 5000, Australia; Australian Centre for Evolutionary Biology and
      Biodiversity, University of Adelaide, Adelaide 5005, Australia; Cornell Lab of
      Ornithology, Cornell University, Ithaca, New York 14850, USA; drabosky@umich.edu.
FAU - Donnellan, Stephen C
AU  - Donnellan SC
AD  - Museum of Zoology, University of Michigan, Ann Arbor, MI 48109, USA; Department
      of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, MI 48109,
      USA; South Australian Museum, North Terrace, Adelaide 5000, Australia; Australian
      Centre for Evolutionary Biology and Biodiversity, University of Adelaide,
      Adelaide 5005, Australia; Cornell Lab of Ornithology, Cornell University, Ithaca,
      New York 14850, USA;Museum of Zoology, University of Michigan, Ann Arbor, MI
      48109, USA; Department of Ecology and Evolutionary Biology, University of
      Michigan, Ann Arbor, MI 48109, USA; South Australian Museum, North Terrace,
      Adelaide 5000, Australia; Australian Centre for Evolutionary Biology and
      Biodiversity, University of Adelaide, Adelaide 5005, Australia; Cornell Lab of
      Ornithology, Cornell University, Ithaca, New York 14850, USA;
FAU - Grundler, Michael
AU  - Grundler M
AD  - Museum of Zoology, University of Michigan, Ann Arbor, MI 48109, USA; Department
      of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, MI 48109,
      USA; South Australian Museum, North Terrace, Adelaide 5000, Australia; Australian
      Centre for Evolutionary Biology and Biodiversity, University of Adelaide,
      Adelaide 5005, Australia; Cornell Lab of Ornithology, Cornell University, Ithaca,
      New York 14850, USA;
FAU - Lovette, Irby J
AU  - Lovette IJ
AD  - Museum of Zoology, University of Michigan, Ann Arbor, MI 48109, USA; Department
      of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, MI 48109,
      USA; South Australian Museum, North Terrace, Adelaide 5000, Australia; Australian
      Centre for Evolutionary Biology and Biodiversity, University of Adelaide,
      Adelaide 5005, Australia; Cornell Lab of Ornithology, Cornell University, Ithaca,
      New York 14850, USA;
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140328
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - *Computer Simulation
MH  - *Genetic Speciation
MH  - Lizards/anatomy &amp; histology/*classification/genetics
MH  - Models, Theoretical
MH  - *Phylogeny
EDAT- 2014/04/01 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/04/01 06:00
PHST- 2014/04/01 06:00 [entrez]
PHST- 2014/04/01 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu025 [pii]
AID - 10.1093/sysbio/syu025 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):610-27. doi: 10.1093/sysbio/syu025. Epub 2014 Mar 28.

PMID- 24671619
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - Standard sister clade comparison fails when testing derived character States.
PG  - 601-9
LID - 10.1093/sysbio/syu024 [doi]
AB  - Comparing species richness in sister clades that differ in a character state is
      one of the ways to study factors influencing diversification. While most of its
      applications have focussed on traits that increase diversification, some have
      been used to study the association of a trait with lower species richness, e.g., 
      the occurrence of dioecy in flowering plants. We show here, using simulations and
      an analytical model, that the null expectation of equal species richness that is 
      generally used in sister clade comparisons is wrong in the case of a derived
      trait occurring independently from speciation: one should expect fewer species in
      the clade with the derived character state when there is no difference in
      diversification rates. This is due to the waiting time for the derived state to
      appear, which causes it to occur more often on longer branches. This has the
      important implication that the probability for a clade to possess the derived
      state depends on the tree geometry, and thus on species richness: species-poorer 
      clades are more likely to possess the derived state. We develop a statistical
      test for sister clade comparisons to study the effect of a derived character
      state. Applying it to a data set of dioecious clades, we find that we cannot
      confirm earlier work that concluded that dioecy decreases diversification; on the
      contrary, it seems to be associated to higher species richness than expected.
      [angiosperms; dioecy; diversification; sister clades; species richness.].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Kafer, Jos
AU  - Kafer J
AD  - Universite de Lyon; Universite Lyon 1; CNRS; UMR 5558; Laboratoire de Biometrie
      et Biologie Evolutive; F-69622, Villeurbanne, France jos.kaefer@gmail.com.
FAU - Mousset, Sylvain
AU  - Mousset S
AD  - Universite de Lyon; Universite Lyon 1; CNRS; UMR 5558; Laboratoire de Biometrie
      et Biologie Evolutive; F-69622, Villeurbanne, France.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140326
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biodiversity
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Magnoliopsida/classification
MH  - *Models, Theoretical
MH  - Phenotype
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2014/03/29 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/03/28 06:00
PHST- 2014/03/28 06:00 [entrez]
PHST- 2014/03/29 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu024 [pii]
AID - 10.1093/sysbio/syu024 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):601-9. doi: 10.1093/sysbio/syu024. Epub 2014 Mar 26.

PMID- 24671618
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20140614
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - Is the tree of life the best metaphor, model, or heuristic for phylogenetics?
PG  - 628-38
LID - 10.1093/sysbio/syu026 [doi]
FAU - Morrison, David A
AU  - Morrison DA
AD  - Section for Parasitology, Swedish University of Agricultural Sciences, 751 89
      Uppsala, Sweden david.morrison@slu.se.
LA  - eng
PT  - Journal Article
DEP - 20140326
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Classification
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2014/03/29 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/03/28 06:00
PHST- 2014/03/28 06:00 [entrez]
PHST- 2014/03/29 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu026 [pii]
AID - 10.1093/sysbio/syu026 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):628-38. doi: 10.1093/sysbio/syu026. Epub 2014 Mar 26.

PMID- 24658316
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - The impact of the rate prior on Bayesian estimation of divergence times with
      multiple Loci.
PG  - 555-65
LID - 10.1093/sysbio/syu020 [doi]
AB  - Bayesian methods provide a powerful way to estimate species divergence times by
      combining information from molecular sequences with information from the fossil
      record. With the explosive increase of genomic data, divergence time estimation
      increasingly uses data of multiple loci (genes or site partitions). Widely used
      computer programs to estimate divergence times use independent and identically
      distributed (i.i.d.) priors on the substitution rates for different loci. The
      i.i.d. prior is problematic. As the number of loci (L) increases, the prior
      variance of the average rate across all loci goes to zero at the rate 1/L. As a
      consequence, the rate prior dominates posterior time estimates when many loci are
      analyzed, and if the rate prior is misspecified, the estimated divergence times
      will converge to wrong values with very narrow credibility intervals. Here we
      develop a new prior on the locus rates based on the Dirichlet distribution that
      corrects the problematic behavior of the i.i.d. prior. We use computer simulation
      and real data analysis to highlight the differences between the old and new
      priors. For a dataset for six primate species, we show that with the old i.i.d.
      prior, if the prior rate is too high (or too low), the estimated divergence times
      are too young (or too old), outside the bounds imposed by the fossil
      calibrations. In contrast, with the new Dirichlet prior, posterior time estimates
      are insensitive to the rate prior and are compatible with the fossil
      calibrations. We re-analyzed a phylogenomic data set of 36 mammal species and
      show that using many fossil calibrations can alleviate the adverse impact of a
      misspecified rate prior to some extent. We recommend the use of the new Dirichlet
      prior in Bayesian divergence time estimation. [Bayesian inference, divergence
      time, relaxed clock, rate prior, partition analysis.].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Dos Reis, Mario
AU  - Dos Reis M
AD  - Department of Genetics, Evolution and Environment, University College London,
      Darwin Building, Gower Street, London WC1E 6BT, UK;
FAU - Zhu, Tianqi
AU  - Zhu T
AD  - Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing 100101,
      China.
FAU - Yang, Ziheng
AU  - Yang Z
AD  - Department of Genetics, Evolution and Environment, University College London,
      Darwin Building, Gower Street, London WC1E 6BT, UK; z.yang@ucl.ac.uk.
LA  - eng
GR  - BB/J009709/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140321
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - *Evolution, Molecular
MH  - Fossils
MH  - Genetic Loci/*genetics
MH  - *Genetic Speciation
MH  - *Phylogeny
MH  - Primates/classification/genetics
MH  - Time
PMC - PMC4055871
EDAT- 2014/03/25 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/03/25 06:00
PHST- 2014/03/25 06:00 [entrez]
PHST- 2014/03/25 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu020 [pii]
AID - 10.1093/sysbio/syu020 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):555-65. doi: 10.1093/sysbio/syu020. Epub 2014 Mar 21.

PMID- 24627185
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20140614
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - Global biodiversity assessment and hyper-cryptic species complexes: more than one
      species of elephant in the room?
PG  - 518-33
LID - 10.1093/sysbio/syu017 [doi]
AB  - Several recent estimates of global biodiversity have concluded that the total
      number of species on Earth lies near the lower end of the wide range touted in
      previous decades. However, none of these recent estimates formally explore the
      real "elephant in the room", namely, what proportion of species are taxonomically
      invisible to conventional assessments, and thus, as undiagnosed cryptic species, 
      remain uncountable until revealed by multi-gene molecular assessments. Here we
      explore the significance and extent of so-called "hyper-cryptic" species
      complexes, using the Australian freshwater fish Galaxias olidus as a proxy for
      any organism whose taxonomy ought to be largely finalized when compared to those 
      in little-studied or morphologically undifferentiated groups. Our comprehensive
      allozyme (838 fish for 54 putative loci), mtDNA (557 fish for 605 bp of cytb),
      and morphological (1963-3389 vouchers for 17-58 characters) assessment of this
      species across its broad geographic range revealed a 1500% increase in
      species-level biodiversity, and suggested that additional taxa may remain
      undiscovered. Importantly, while all 15 candidate species were morphologically
      diagnosable a posteriori from one another, single-gene DNA barcoding proved
      largely unsuccessful as an a priori method for species identification. These
      results lead us to draw two strong inferences of relevance to estimates of global
      biodiversity. First, hyper-cryptic complexes are likely to be common in many
      organismal groups. Second, no assessment of species numbers can be considered
      "best practice" in the molecular age unless it explicitly includes estimates of
      the extent of cryptic and hyper-cryptic biodiversity. [Galaxiidae; global
      estimates; hyper-diverse; mountain galaxias; species counts; species richness.].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Adams, Mark
AU  - Adams M
AD  - Evolutionary Biology Unit, South Australian Museum, North Terrace, SA 5000,
      Australia;School of Earth and Environmental Sciences, University of Adelaide,
      Adelaide, SA 5005, Australia; mark.adams@samuseum.sa.gov.au.
FAU - Raadik, Tarmo A
AU  - Raadik TA
AD  - Aquatic Ecology Section, Arthur Rylah Institute for Environmental Research,
      Department of Environment and Primary Industries, 123 Brown Street, Heidelberg,
      VIC 3084, Australia;Institute for Applied Ecology and Collaborative Research
      Network for Murray-Darling Futures, University of Canberra, ACT 2601, Australia
      and.
FAU - Burridge, Christopher P
AU  - Burridge CP
AD  - School of Biological Sciences, University of Tasmania, Private Bag 55, Hobart,
      TAS 7001, Australia.
FAU - Georges, Arthur
AU  - Georges A
AD  - Institute for Applied Ecology and Collaborative Research Network for
      Murray-Darling Futures, University of Canberra, ACT 2601, Australia and.
LA  - eng
SI  - GENBANK/KJ511493
SI  - GENBANK/KJ511494
SI  - GENBANK/KJ511495
SI  - GENBANK/KJ511496
SI  - GENBANK/KJ511497
SI  - GENBANK/KJ511498
SI  - GENBANK/KJ511499
SI  - GENBANK/KJ511500
SI  - GENBANK/KJ511501
SI  - GENBANK/KJ511502
SI  - GENBANK/KJ511503
SI  - GENBANK/KJ511504
SI  - GENBANK/KJ511505
SI  - GENBANK/KJ511506
SI  - GENBANK/KJ511507
SI  - GENBANK/KJ511508
SI  - GENBANK/KJ511509
SI  - GENBANK/KJ511510
SI  - GENBANK/KJ511511
SI  - GENBANK/KJ511512
SI  - GENBANK/KJ511513
SI  - GENBANK/KJ511514
SI  - GENBANK/KJ511515
SI  - GENBANK/KJ511516
SI  - GENBANK/KJ511517
SI  - GENBANK/KJ511518
SI  - GENBANK/KJ511519
SI  - GENBANK/KJ511520
SI  - GENBANK/KJ511521
SI  - GENBANK/KJ511522
SI  - GENBANK/KJ511523
SI  - GENBANK/KJ511524
SI  - GENBANK/KJ511525
SI  - GENBANK/KJ511526
SI  - GENBANK/KJ511527
SI  - GENBANK/KJ511528
SI  - GENBANK/KJ511529
SI  - GENBANK/KJ511530
SI  - GENBANK/KJ511531
SI  - GENBANK/KJ511532
SI  - GENBANK/KJ511533
SI  - GENBANK/KJ511534
SI  - GENBANK/KJ511535
SI  - GENBANK/KJ511536
SI  - GENBANK/KJ511537
SI  - GENBANK/KJ511538
SI  - GENBANK/KJ511539
SI  - GENBANK/KJ511540
SI  - GENBANK/KJ511541
SI  - GENBANK/KJ511542
SI  - GENBANK/KJ511543
SI  - GENBANK/KJ511544
SI  - GENBANK/KJ511545
SI  - GENBANK/KJ511546
SI  - GENBANK/KJ511547
SI  - GENBANK/KJ511548
SI  - GENBANK/KJ511549
SI  - GENBANK/KJ511550
SI  - GENBANK/KJ511551
SI  - GENBANK/KJ511552
SI  - GENBANK/KJ511553
SI  - GENBANK/KJ511554
SI  - GENBANK/KJ511555
SI  - GENBANK/KJ511556
SI  - GENBANK/KJ511557
SI  - GENBANK/KJ511558
SI  - GENBANK/KJ511559
SI  - GENBANK/KJ511560
SI  - GENBANK/KJ511561
SI  - GENBANK/KJ511562
SI  - GENBANK/KJ511563
SI  - GENBANK/KJ511564
SI  - GENBANK/KJ511565
SI  - GENBANK/KJ511566
SI  - GENBANK/KJ511567
SI  - GENBANK/KJ511568
SI  - GENBANK/KJ511569
SI  - GENBANK/KJ511570
SI  - GENBANK/KJ511571
SI  - GENBANK/KJ511572
SI  - GENBANK/KJ511573
SI  - GENBANK/KJ511574
SI  - GENBANK/KJ511575
SI  - GENBANK/KJ511576
SI  - GENBANK/KJ511577
SI  - GENBANK/KJ511578
SI  - GENBANK/KJ511579
SI  - GENBANK/KJ511580
SI  - GENBANK/KJ511581
SI  - GENBANK/KJ511582
SI  - GENBANK/KJ511583
SI  - GENBANK/KJ511584
SI  - GENBANK/KJ511585
SI  - GENBANK/KJ511586
SI  - GENBANK/KJ511587
SI  - GENBANK/KJ511588
SI  - GENBANK/KJ511589
SI  - GENBANK/KJ511590
SI  - GENBANK/KJ511591
SI  - GENBANK/KJ511592
SI  - GENBANK/KJ511593
SI  - GENBANK/KJ511594
SI  - GENBANK/KJ511595
SI  - GENBANK/KJ511596
SI  - GENBANK/KJ511597
SI  - GENBANK/KJ511598
SI  - GENBANK/KJ511599
SI  - GENBANK/KJ511600
SI  - GENBANK/KJ511601
SI  - GENBANK/KJ511602
SI  - GENBANK/KJ511603
SI  - GENBANK/KJ511604
SI  - GENBANK/KJ511605
SI  - GENBANK/KJ511606
SI  - GENBANK/KJ511607
SI  - GENBANK/KJ511608
SI  - GENBANK/KJ511609
SI  - GENBANK/KJ511610
SI  - GENBANK/KJ511611
SI  - GENBANK/KJ511612
SI  - GENBANK/KJ511613
SI  - GENBANK/KJ511614
SI  - GENBANK/KJ511615
SI  - GENBANK/KJ511616
SI  - GENBANK/KJ511617
SI  - GENBANK/KJ511618
SI  - GENBANK/KJ511619
SI  - GENBANK/KJ511620
SI  - GENBANK/KJ511621
SI  - GENBANK/KJ511622
SI  - GENBANK/KJ511623
SI  - GENBANK/KJ511624
SI  - GENBANK/KJ511625
SI  - GENBANK/KJ511626
SI  - GENBANK/KJ511627
SI  - GENBANK/KJ511628
SI  - GENBANK/KJ511629
SI  - GENBANK/KJ511630
SI  - GENBANK/KJ511631
SI  - GENBANK/KJ511632
SI  - GENBANK/KJ511633
SI  - GENBANK/KJ511634
SI  - GENBANK/KJ511635
SI  - GENBANK/KJ511636
SI  - GENBANK/KJ511637
SI  - GENBANK/KJ511638
SI  - GENBANK/KJ511639
SI  - GENBANK/KJ511640
SI  - GENBANK/KJ511641
SI  - GENBANK/KJ511642
SI  - GENBANK/KJ511643
SI  - GENBANK/KJ511644
SI  - GENBANK/KJ511645
SI  - GENBANK/KJ511646
SI  - GENBANK/KJ511647
SI  - GENBANK/KJ511648
SI  - GENBANK/KJ511649
SI  - GENBANK/KJ511650
SI  - GENBANK/KJ511651
SI  - GENBANK/KJ511652
SI  - GENBANK/KJ511653
SI  - GENBANK/KJ511654
SI  - GENBANK/KJ511655
SI  - GENBANK/KJ511656
SI  - GENBANK/KJ511657
SI  - GENBANK/KJ511658
SI  - GENBANK/KJ511659
SI  - GENBANK/KJ511660
SI  - GENBANK/KJ511661
SI  - GENBANK/KJ511662
SI  - GENBANK/KJ511663
SI  - GENBANK/KJ511664
SI  - GENBANK/KJ511665
SI  - GENBANK/KJ511666
SI  - GENBANK/KJ511667
SI  - GENBANK/KJ511668
SI  - GENBANK/KJ511669
SI  - GENBANK/KJ511670
SI  - GENBANK/KJ511671
SI  - GENBANK/KJ511672
SI  - GENBANK/KJ511673
SI  - GENBANK/KJ511674
SI  - GENBANK/KJ511675
SI  - GENBANK/KJ511676
SI  - GENBANK/KJ511677
SI  - GENBANK/KJ511678
SI  - GENBANK/KJ511679
SI  - GENBANK/KJ511680
SI  - GENBANK/KJ511681
SI  - GENBANK/KJ511682
SI  - GENBANK/KJ511683
SI  - GENBANK/KJ511684
SI  - GENBANK/KJ511685
SI  - GENBANK/KJ511686
SI  - GENBANK/KJ511687
SI  - GENBANK/KJ511688
SI  - GENBANK/KJ511689
SI  - GENBANK/KJ511690
SI  - GENBANK/KJ511691
SI  - GENBANK/KJ511692
SI  - GENBANK/KJ511693
SI  - GENBANK/KJ511694
SI  - GENBANK/KJ511695
SI  - GENBANK/KJ511696
SI  - GENBANK/KJ511697
SI  - GENBANK/KJ511698
SI  - GENBANK/KJ511699
SI  - GENBANK/KJ511700
SI  - GENBANK/KJ511701
SI  - GENBANK/KJ511702
SI  - GENBANK/KJ511703
SI  - GENBANK/KJ511704
SI  - GENBANK/KJ511705
SI  - GENBANK/KJ511706
SI  - GENBANK/KJ511707
SI  - GENBANK/KJ511708
SI  - GENBANK/KJ511709
SI  - GENBANK/KJ511710
SI  - GENBANK/KJ511711
SI  - GENBANK/KJ511712
SI  - GENBANK/KJ511713
SI  - GENBANK/KJ511714
SI  - GENBANK/KJ511715
SI  - GENBANK/KJ511716
SI  - GENBANK/KJ511717
SI  - GENBANK/KJ511718
SI  - GENBANK/KJ511719
SI  - GENBANK/KJ511720
SI  - GENBANK/KJ511721
SI  - GENBANK/KJ511722
SI  - GENBANK/KJ511723
SI  - GENBANK/KJ511724
SI  - GENBANK/KJ511725
SI  - GENBANK/KJ511726
SI  - GENBANK/KJ511727
SI  - GENBANK/KJ511728
SI  - GENBANK/KJ511729
SI  - GENBANK/KJ511730
SI  - GENBANK/KJ511731
SI  - GENBANK/KJ511732
SI  - GENBANK/KJ511733
SI  - GENBANK/KJ511734
SI  - GENBANK/KJ511735
SI  - GENBANK/KJ511736
SI  - GENBANK/KJ511737
SI  - GENBANK/KJ511738
SI  - GENBANK/KJ511739
SI  - GENBANK/KJ511740
SI  - GENBANK/KJ511741
SI  - GENBANK/KJ511742
SI  - GENBANK/KJ511743
SI  - GENBANK/KJ511744
SI  - GENBANK/KJ511745
SI  - GENBANK/KJ511746
SI  - GENBANK/KJ511747
SI  - GENBANK/KJ511748
SI  - GENBANK/KJ511749
SI  - GENBANK/KJ511750
SI  - GENBANK/KJ511751
SI  - GENBANK/KJ511752
SI  - GENBANK/KJ511753
SI  - GENBANK/KJ511754
SI  - GENBANK/KJ511755
SI  - GENBANK/KJ511756
SI  - GENBANK/KJ511757
SI  - GENBANK/KJ511758
SI  - GENBANK/KJ511759
SI  - GENBANK/KJ511760
SI  - GENBANK/KJ511761
SI  - GENBANK/KJ511762
SI  - GENBANK/KJ511763
SI  - GENBANK/KJ511764
SI  - GENBANK/KJ511765
SI  - GENBANK/KJ511766
SI  - GENBANK/KJ511767
SI  - GENBANK/KJ511768
SI  - GENBANK/KJ511769
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140313
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Isoenzymes)
SB  - IM
MH  - Animals
MH  - *Biodiversity
MH  - DNA Barcoding, Taxonomic/standards
MH  - Genes, Mitochondrial/genetics
MH  - Isoenzymes/metabolism
MH  - Molecular Sequence Data
MH  - Osmeriformes/anatomy &amp; histology/*classification/genetics
MH  - Phylogeny
MH  - Species Specificity
EDAT- 2014/03/15 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/03/15 06:00
PHST- 2014/03/15 06:00 [entrez]
PHST- 2014/03/15 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu017 [pii]
AID - 10.1093/sysbio/syu017 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):518-33. doi: 10.1093/sysbio/syu017. Epub 2014 Mar 13.

PMID- 24627184
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - Inferring heterogeneous evolutionary processes through time: from sequence
      substitution to phylogeography.
PG  - 493-504
LID - 10.1093/sysbio/syu015 [doi]
AB  - Molecular phylogenetic and phylogeographic reconstructions generally assume
      time-homogeneous substitution processes. Motivated by computational convenience, 
      this assumption sacrifices biological realism and offers little opportunity to
      uncover the temporal dynamics in evolutionary histories. Here, we propose an
      evolutionary approach that explicitly relaxes the time-homogeneity assumption by 
      allowing the specification of different infinitesimal substitution rate matrices 
      across different time intervals, called epochs, along the evolutionary history.
      We focus on an epoch model implementation in a Bayesian inference framework that 
      offers great modeling flexibility in drawing inference about any discrete data
      type characterized as a continuous-time Markov chain, including phylogeographic
      traits. To alleviate the computational burden that the additional temporal
      heterogeneity imposes, we adopt a massively parallel approach that achieves both 
      fine- and coarse-grain parallelization of the computations across branches that
      accommodate epoch transitions, making extensive use of graphics processing units.
      Through synthetic examples, we assess model performance in recovering
      evolutionary parameters from data generated according to different evolutionary
      scenarios that comprise different numbers of epochs for both nucleotide and codon
      substitution processes. We illustrate the usefulness of our inference framework
      in two different applications to empirical data sets: the selection dynamics on
      within-host HIV populations throughout infection and the seasonality of global
      influenza circulation. In both cases, our epoch model captures key features of
      temporal heterogeneity that remained difficult to test using ad hoc procedures.
      [Bayesian inference; BEAGLE; BEAST; Epoch Model; phylogeography; Phylogenetics.].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Bielejec, Filip
AU  - Bielejec F
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven, Leuven,
      Belgium; filip.bielejec@rega.kuleuven.be.
FAU - Lemey, Philippe
AU  - Lemey P
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven, Leuven,
      Belgium;
FAU - Baele, Guy
AU  - Baele G
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven, Leuven,
      Belgium;
FAU - Rambaut, Andrew
AU  - Rambaut A
AD  - Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, United
      Kingdom;Fogarty International Center, National Institutes of Health, Bethesda,
      MD, USA;
FAU - Suchard, Marc A
AU  - Suchard MA
AD  - Departments of Biomathematics and Human Genetics, David Geffen School of Medicine
      at UCLA, University of California, Los Angeles, CA, 90095, USA;Department of
      Biostatistics, UCLA Fielding School of Public Health, University of California,
      Los Angeles, CA, 90095, USA.
LA  - eng
GR  - 095831/Wellcome Trust/United Kingdom
GR  - 260864/European Research Council/International
GR  - R01 HG006139/HG/NHGRI NIH HHS/United States
GR  - 092807/Wellcome Trust/United Kingdom
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140312
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Computer Simulation
MH  - HIV-1/classification/genetics
MH  - Humans
MH  - Influenza A virus/classification/genetics
MH  - *Phylogeny
MH  - Phylogeography
MH  - Seasons
PMC - PMC4055869
EDAT- 2014/03/15 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/03/15 06:00
PHST- 2014/03/15 06:00 [entrez]
PHST- 2014/03/15 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu015 [pii]
AID - 10.1093/sysbio/syu015 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):493-504. doi: 10.1093/sysbio/syu015. Epub 2014 Mar 12.

PMID- 24627183
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - Species delimitation using genome-wide SNP data.
PG  - 534-42
LID - 10.1093/sysbio/syu018 [doi]
AB  - The multispecies coalescent has provided important progress for evolutionary
      inferences, including increasing the statistical rigor and objectivity of
      comparisons among competing species delimitation models. However, Bayesian
      species delimitation methods typically require brute force integration over gene 
      trees via Markov chain Monte Carlo (MCMC), which introduces a large computation
      burden and precludes their application to genomic-scale data. Here we combine a
      recently introduced dynamic programming algorithm for estimating species trees
      that bypasses MCMC integration over gene trees with sophisticated methods for
      estimating marginal likelihoods, needed for Bayesian model selection, to provide 
      a rigorous and computationally tractable technique for genome-wide species
      delimitation. We provide a critical yet simple correction that brings the
      likelihoods of different species trees, and more importantly their corresponding 
      marginal likelihoods, to the same common denominator, which enables direct and
      accurate comparisons of competing species delimitation models using Bayes
      factors. We test this approach, which we call Bayes factor delimitation (*with
      genomic data; BFD*), using common species delimitation scenarios with computer
      simulations. Varying the numbers of loci and the number of samples suggest that
      the approach can distinguish the true model even with few loci and limited
      samples per species. Misspecification of the prior for population size theta has 
      little impact on support for the true model. We apply the approach to West
      African forest geckos (Hemidactylus fasciatus complex) using genome-wide SNP
      data. This new Bayesian method for species delimitation builds on a growing trend
      for objective species delimitation methods with explicit model assumptions that
      are easily tested. [Bayes factor; model testing; phylogeography; RADseq;
      simulation; speciation.].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Leache, Adam D
AU  - Leache AD
AD  - Department of Biology, University of Washington, Seattle, WA 98195, USA;Burke
      Museum of Natural History and Culture, University of Washington, Seattle, WA
      98195, USA; leache@uw.edu.
FAU - Fujita, Matthew K
AU  - Fujita MK
AD  - Department of Biology, University of Texas at Arlington, Arlington, TX 76019,
      USA;
FAU - Minin, Vladimir N
AU  - Minin VN
AD  - Department of Biology, University of Washington, Seattle, WA 98195,
      USA;Department of Statistics, University of Washington, Seattle, WA 98195, USA;
FAU - Bouckaert, Remco R
AU  - Bouckaert RR
AD  - Computational Evolution Group, University of Auckland, Auckland, New Zealand.
LA  - eng
GR  - R01 AI107034/AI/NIAID NIH HHS/United States
GR  - R01-AI107034/AI/NIAID NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140312
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Bayes Theorem
MH  - Computer Simulation
MH  - Genome/*genetics
MH  - Lizards/genetics
MH  - *Phylogeny
MH  - Phylogeography/*methods
MH  - Polymorphism, Single Nucleotide/*genetics
PMC - PMC4072903
EDAT- 2014/03/15 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/03/15 06:00
PHST- 2014/03/15 06:00 [entrez]
PHST- 2014/03/15 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu018 [pii]
AID - 10.1093/sysbio/syu018 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):534-42. doi: 10.1093/sysbio/syu018. Epub 2014 Mar 12.

PMID- 24603127
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20140614
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - Incorporating color into integrative taxonomy: analysis of the varied tit
      (Sittiparus varius) complex in East Asia.
PG  - 505-17
LID - 10.1093/sysbio/syu016 [doi]
AB  - Species designations are critically important scientific hypotheses that serve as
      the foundational units in a wide range of biological subdisciplines. A growing
      realization that some classes of data fail to delimit species under certain
      conditions has led to increasingly more integrative taxonomies, whereby species
      discovery and hypothesis testing are based on multiple kinds of data (e.g.,
      morphological, molecular, behavioral, ecological, etc.). However, although most
      taxonomic descriptions have been based on morphology, some key morphological
      features, such as color, are rarely quantified and incorporated into integrative 
      taxonomic studies. In this article, we applied a new method of ultraviolet
      digital photography to measure plumage variation in a color-variable avian
      species complex, the varied tit (Sittiparus varius). Plumage measurements
      corroborated species limits defined by morphometric, mitochondrial DNA, and
      nuclear DNA disjunctions and provided the only evidence for distinguishing two
      recently evolved species. Importantly, color quantification also provided a
      justification for lumping putative taxa with no evidence of evolutionary
      independence. Our revised taxonomy thus refines conservation units for listing
      and management and clarifies the primary units for evolutionary studies. Species 
      tree analyses, which applied the newly delimited species as operational taxonomic
      units, revealed a robust phylogenetic hypothesis for the group that establishes a
      foundation for future biogeographic analyses. This study demonstrates how digital
      photography can be used to incorporate color character variation into integrative
      taxonomies, which should lead to more informed, more rigorous, and more accurate 
      assessments of biodiversity. [Color, digital photography, integrative taxonomy,
      Sittiparus varius, species delimitation, varied tit.].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - McKay, Bailey D
AU  - McKay BD
AD  - Department of Ornithology, American Museum of Natural History, 200 Central Park
      W, New York City, NY 10024, USA; Cincinnati Museum Center, Geier Collections and 
      Research Building, 1301 Western Avenue, Cincinnati, OH 45203, USA; Medium
      Altitude Experimental Station, Endemic Species Research Institute, COA, ROC, #1
      Min-Shen East Road, Chichi, Nantou County, 552, Taiwan; Department of Life
      Science, Liaoning University, No. 66 Chongshan Middle Road, Huanggu District,
      Shenyang 110036, China; Graduate School of Media and Governance, Keio University 
      SFC, 5322 Endo, Fujisawa, Kanagawa 252-0882, Japan; and Department of Zoology,
      National Museum of Nature and Science, 4-1-1 Amakubo, Tsukuba-shi, Ibaraki
      305-0005, Japan bmckay@amnh.org.
FAU - Mays, Herman L Jr
AU  - Mays HL Jr
AD  - Department of Ornithology, American Museum of Natural History, 200 Central Park
      W, New York City, NY 10024, USA; Cincinnati Museum Center, Geier Collections and 
      Research Building, 1301 Western Avenue, Cincinnati, OH 45203, USA; Medium
      Altitude Experimental Station, Endemic Species Research Institute, COA, ROC, #1
      Min-Shen East Road, Chichi, Nantou County, 552, Taiwan; Department of Life
      Science, Liaoning University, No. 66 Chongshan Middle Road, Huanggu District,
      Shenyang 110036, China; Graduate School of Media and Governance, Keio University 
      SFC, 5322 Endo, Fujisawa, Kanagawa 252-0882, Japan; and Department of Zoology,
      National Museum of Nature and Science, 4-1-1 Amakubo, Tsukuba-shi, Ibaraki
      305-0005, Japan.
FAU - Yao, Cheng-Te
AU  - Yao CT
AD  - Department of Ornithology, American Museum of Natural History, 200 Central Park
      W, New York City, NY 10024, USA; Cincinnati Museum Center, Geier Collections and 
      Research Building, 1301 Western Avenue, Cincinnati, OH 45203, USA; Medium
      Altitude Experimental Station, Endemic Species Research Institute, COA, ROC, #1
      Min-Shen East Road, Chichi, Nantou County, 552, Taiwan; Department of Life
      Science, Liaoning University, No. 66 Chongshan Middle Road, Huanggu District,
      Shenyang 110036, China; Graduate School of Media and Governance, Keio University 
      SFC, 5322 Endo, Fujisawa, Kanagawa 252-0882, Japan; and Department of Zoology,
      National Museum of Nature and Science, 4-1-1 Amakubo, Tsukuba-shi, Ibaraki
      305-0005, Japan.
FAU - Wan, Dongmei
AU  - Wan D
AD  - Department of Ornithology, American Museum of Natural History, 200 Central Park
      W, New York City, NY 10024, USA; Cincinnati Museum Center, Geier Collections and 
      Research Building, 1301 Western Avenue, Cincinnati, OH 45203, USA; Medium
      Altitude Experimental Station, Endemic Species Research Institute, COA, ROC, #1
      Min-Shen East Road, Chichi, Nantou County, 552, Taiwan; Department of Life
      Science, Liaoning University, No. 66 Chongshan Middle Road, Huanggu District,
      Shenyang 110036, China; Graduate School of Media and Governance, Keio University 
      SFC, 5322 Endo, Fujisawa, Kanagawa 252-0882, Japan; and Department of Zoology,
      National Museum of Nature and Science, 4-1-1 Amakubo, Tsukuba-shi, Ibaraki
      305-0005, Japan.
FAU - Higuchi, Hiroyoshi
AU  - Higuchi H
AD  - Department of Ornithology, American Museum of Natural History, 200 Central Park
      W, New York City, NY 10024, USA; Cincinnati Museum Center, Geier Collections and 
      Research Building, 1301 Western Avenue, Cincinnati, OH 45203, USA; Medium
      Altitude Experimental Station, Endemic Species Research Institute, COA, ROC, #1
      Min-Shen East Road, Chichi, Nantou County, 552, Taiwan; Department of Life
      Science, Liaoning University, No. 66 Chongshan Middle Road, Huanggu District,
      Shenyang 110036, China; Graduate School of Media and Governance, Keio University 
      SFC, 5322 Endo, Fujisawa, Kanagawa 252-0882, Japan; and Department of Zoology,
      National Museum of Nature and Science, 4-1-1 Amakubo, Tsukuba-shi, Ibaraki
      305-0005, Japan.
FAU - Nishiumi, Isao
AU  - Nishiumi I
AD  - Department of Ornithology, American Museum of Natural History, 200 Central Park
      W, New York City, NY 10024, USA; Cincinnati Museum Center, Geier Collections and 
      Research Building, 1301 Western Avenue, Cincinnati, OH 45203, USA; Medium
      Altitude Experimental Station, Endemic Species Research Institute, COA, ROC, #1
      Min-Shen East Road, Chichi, Nantou County, 552, Taiwan; Department of Life
      Science, Liaoning University, No. 66 Chongshan Middle Road, Huanggu District,
      Shenyang 110036, China; Graduate School of Media and Governance, Keio University 
      SFC, 5322 Endo, Fujisawa, Kanagawa 252-0882, Japan; and Department of Zoology,
      National Museum of Nature and Science, 4-1-1 Amakubo, Tsukuba-shi, Ibaraki
      305-0005, Japan.
LA  - eng
SI  - GENBANK/KF932840
SI  - GENBANK/KF932841
SI  - GENBANK/KF932842
SI  - GENBANK/KF932843
SI  - GENBANK/KF932844
SI  - GENBANK/KF932845
SI  - GENBANK/KF932846
SI  - GENBANK/KF932847
SI  - GENBANK/KF932848
SI  - GENBANK/KF932849
SI  - GENBANK/KF932850
SI  - GENBANK/KF932851
SI  - GENBANK/KF932852
SI  - GENBANK/KF932853
SI  - GENBANK/KF932854
SI  - GENBANK/KF932855
SI  - GENBANK/KF932856
SI  - GENBANK/KF932857
SI  - GENBANK/KF932858
SI  - GENBANK/KF932859
SI  - GENBANK/KF932860
SI  - GENBANK/KF932861
SI  - GENBANK/KF932862
SI  - GENBANK/KF932863
SI  - GENBANK/KF932864
SI  - GENBANK/KF932865
SI  - GENBANK/KF932866
SI  - GENBANK/KF932867
SI  - GENBANK/KF932868
SI  - GENBANK/KF932869
SI  - GENBANK/KF932870
SI  - GENBANK/KF932871
SI  - GENBANK/KF932872
SI  - GENBANK/KF932873
SI  - GENBANK/KF932874
SI  - GENBANK/KF932875
SI  - GENBANK/KF932876
SI  - GENBANK/KF932877
SI  - GENBANK/KF932878
SI  - GENBANK/KF932879
SI  - GENBANK/KF932880
SI  - GENBANK/KF932881
SI  - GENBANK/KF932882
SI  - GENBANK/KF932883
SI  - GENBANK/KF932884
SI  - GENBANK/KF932885
SI  - GENBANK/KF932886
SI  - GENBANK/KF932887
SI  - GENBANK/KF932888
SI  - GENBANK/KF932889
SI  - GENBANK/KF932890
SI  - GENBANK/KF932891
SI  - GENBANK/KF932892
SI  - GENBANK/KF932893
SI  - GENBANK/KF932894
SI  - GENBANK/KF932895
SI  - GENBANK/KF932896
SI  - GENBANK/KF932897
SI  - GENBANK/KF932898
SI  - GENBANK/KF932899
SI  - GENBANK/KF932900
SI  - GENBANK/KF932901
SI  - GENBANK/KF932902
SI  - GENBANK/KF932903
SI  - GENBANK/KF932904
SI  - GENBANK/KF932905
SI  - GENBANK/KF932906
SI  - GENBANK/KF932907
SI  - GENBANK/KF932908
SI  - GENBANK/KF932909
SI  - GENBANK/KF932910
SI  - GENBANK/KF932911
SI  - GENBANK/KF932912
SI  - GENBANK/KF932913
SI  - GENBANK/KF932914
SI  - GENBANK/KF932915
SI  - GENBANK/KF932916
SI  - GENBANK/KF932917
SI  - GENBANK/KF932918
SI  - GENBANK/KF932919
SI  - GENBANK/KF932920
SI  - GENBANK/KF932921
SI  - GENBANK/KF932922
SI  - GENBANK/KF932923
SI  - GENBANK/KF932924
PT  - Journal Article
DEP - 20140306
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Body Size
MH  - Classification
MH  - DNA, Mitochondrial/genetics
MH  - Far East
MH  - Feathers/*chemistry
MH  - Molecular Sequence Data
MH  - Passeriformes/anatomy &amp; histology/*classification/genetics/*physiology
MH  - *Phylogeny
MH  - Pigmentation/*physiology
MH  - Species Specificity
EDAT- 2014/03/08 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/03/08 06:00
PHST- 2014/03/08 06:00 [entrez]
PHST- 2014/03/08 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu016 [pii]
AID - 10.1093/sysbio/syu016 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):505-17. doi: 10.1093/sysbio/syu016. Epub 2014 Mar 6.

PMID- 24578226
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - Competitive release leads to range expansion and rampant speciation in malagasy
      dung beetles.
PG  - 480-92
LID - 10.1093/sysbio/syu011 [doi]
AB  - Competition is often thought to promote ecological diversification and thereby to
      facilitate the coexistence of competitors during evolutionary radiations. At
      large spatial scales, species may also coexist by having allopatric
      distributions, which raises the question about the role of range expansion in the
      proliferation of species during radiations. Here, we integrate a well-sampled (50
      out of 74 species) and timed phylogeny of Nanos and Apotolamprus dung beetles
      (Canthonini) in Madagascar with data on species' geographical ranges, abundances,
      and body sizes. There is an overall decline in lineage accumulation through time 
      since the colonization of northern Madagascar in the mid Miocene (24-13 Ma). A
      clade of 24 extant Nanos species (clade L) originating 6.0 Ma exhibits an
      increase in speciation rate, which is associated with a significant increase in
      body size and strikingly allopatric distributions of the species. Large body size
      typically confers a competitive advantage in dung beetles, which is here
      reflected by strong numerical dominance of clade L species in local communities. 
      We suggest that the "key innovation" of large body size has allowed range
      expansion due to competitive release, which has created extensive opportunities
      for allopatric speciation and differentiation along environmental gradients. Most
      theories to explain diversification patterns in Madagascar rely on allopatric
      modes of speciation, but they fail to explain how ancestral species became
      widespread in the first place. The mechanism proposed here, involving range
      expansion following competitive release via a "key innovation", may have operated
      in other Malagasy taxa with large numbers of species with small geographic
      ranges. [biodiversity hotspot; competition; Madagascar; microendemism;
      radiation.].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Miraldo, Andreia
AU  - Miraldo A
AD  - Metapopulation Research Group, Department of Biosciences, P.O. Box 65
      (Viikinkaari 1), FI-00014, University of Helsinki, Finland
      andreia.miraldo@gmail.com.
FAU - Hanski, Ilkka A
AU  - Hanski IA
AD  - Metapopulation Research Group, Department of Biosciences, P.O. Box 65
      (Viikinkaari 1), FI-00014, University of Helsinki, Finland.
LA  - eng
SI  - GENBANK/KF309702
SI  - GENBANK/KF309703
SI  - GENBANK/KF309704
SI  - GENBANK/KF309705
SI  - GENBANK/KF309706
SI  - GENBANK/KF309707
SI  - GENBANK/KF309708
SI  - GENBANK/KF309709
SI  - GENBANK/KF309710
SI  - GENBANK/KF309711
SI  - GENBANK/KF309712
SI  - GENBANK/KF309713
SI  - GENBANK/KF309714
SI  - GENBANK/KF309715
SI  - GENBANK/KF309716
SI  - GENBANK/KF309717
SI  - GENBANK/KF309718
SI  - GENBANK/KF309719
SI  - GENBANK/KF309720
SI  - GENBANK/KF309721
SI  - GENBANK/KF309722
SI  - GENBANK/KF309723
SI  - GENBANK/KF309724
SI  - GENBANK/KF309725
SI  - GENBANK/KF309726
SI  - GENBANK/KF309727
SI  - GENBANK/KF309728
SI  - GENBANK/KF309729
SI  - GENBANK/KF309730
SI  - GENBANK/KF309731
SI  - GENBANK/KF309732
SI  - GENBANK/KF309733
SI  - GENBANK/KF309734
SI  - GENBANK/KF309735
SI  - GENBANK/KF309736
SI  - GENBANK/KF309737
SI  - GENBANK/KF309738
SI  - GENBANK/KF309739
SI  - GENBANK/KF309740
SI  - GENBANK/KF309741
SI  - GENBANK/KF309742
SI  - GENBANK/KF309743
SI  - GENBANK/KF309744
SI  - GENBANK/KF309745
SI  - GENBANK/KF309746
SI  - GENBANK/KF309747
SI  - GENBANK/KF309748
SI  - GENBANK/KF309749
SI  - GENBANK/KF309750
SI  - GENBANK/KF309751
SI  - GENBANK/KF309752
SI  - GENBANK/KF309753
SI  - GENBANK/KF309754
SI  - GENBANK/KF309755
SI  - GENBANK/KF309756
SI  - GENBANK/KF309757
SI  - GENBANK/KF309758
SI  - GENBANK/KF309759
SI  - GENBANK/KF309760
SI  - GENBANK/KF309761
SI  - GENBANK/KF309762
SI  - GENBANK/KF309763
SI  - GENBANK/KF309764
SI  - GENBANK/KF309765
SI  - GENBANK/KF309766
SI  - GENBANK/KF309767
SI  - GENBANK/KF309768
SI  - GENBANK/KF309769
SI  - GENBANK/KF309770
SI  - GENBANK/KF309771
SI  - GENBANK/KF309772
SI  - GENBANK/KF309773
SI  - GENBANK/KF309774
SI  - GENBANK/KF309775
SI  - GENBANK/KF309776
SI  - GENBANK/KF309777
SI  - GENBANK/KF309778
SI  - GENBANK/KF309779
SI  - GENBANK/KF309780
SI  - GENBANK/KF309781
SI  - GENBANK/KF309782
SI  - GENBANK/KF309783
SI  - GENBANK/KF309784
SI  - GENBANK/KF309785
SI  - GENBANK/KF309786
SI  - GENBANK/KF309787
SI  - GENBANK/KF309788
SI  - GENBANK/KF309789
SI  - GENBANK/KF309790
SI  - GENBANK/KF309791
SI  - GENBANK/KF309792
SI  - GENBANK/KF309793
SI  - GENBANK/KF309794
SI  - GENBANK/KF309795
SI  - GENBANK/KF309796
SI  - GENBANK/KF309797
SI  - GENBANK/KF309798
SI  - GENBANK/KF309799
SI  - GENBANK/KF309800
SI  - GENBANK/KF309801
SI  - GENBANK/KF309802
SI  - GENBANK/KF309803
SI  - GENBANK/KF309804
SI  - GENBANK/KF309805
SI  - GENBANK/KF309806
SI  - GENBANK/KF309807
SI  - GENBANK/KF309808
SI  - GENBANK/KF309809
SI  - GENBANK/KF309810
SI  - GENBANK/KF309811
SI  - GENBANK/KF309812
SI  - GENBANK/KF309813
SI  - GENBANK/KF309814
SI  - GENBANK/KF309815
SI  - GENBANK/KF309816
SI  - GENBANK/KF309817
SI  - GENBANK/KF309818
SI  - GENBANK/KF309819
SI  - GENBANK/KF309820
SI  - GENBANK/KF309821
SI  - GENBANK/KF309822
SI  - GENBANK/KF309823
SI  - GENBANK/KF309824
SI  - GENBANK/KF309825
SI  - GENBANK/KF309826
SI  - GENBANK/KF309827
SI  - GENBANK/KF309828
SI  - GENBANK/KF309829
SI  - GENBANK/KF309830
SI  - GENBANK/KF309831
SI  - GENBANK/KF309832
SI  - GENBANK/KF309833
SI  - GENBANK/KF309834
SI  - GENBANK/KF309835
SI  - GENBANK/KF309836
SI  - GENBANK/KF309837
SI  - GENBANK/KF309838
SI  - GENBANK/KF309839
SI  - GENBANK/KF309840
SI  - GENBANK/KF309841
SI  - GENBANK/KF309842
SI  - GENBANK/KF309843
SI  - GENBANK/KF309844
SI  - GENBANK/KF309845
SI  - GENBANK/KF309846
SI  - GENBANK/KF309847
SI  - GENBANK/KF309848
SI  - GENBANK/KF309849
SI  - GENBANK/KF309850
SI  - GENBANK/KF309851
SI  - GENBANK/KF309852
SI  - GENBANK/KF309853
SI  - GENBANK/KF309854
SI  - GENBANK/KF309855
SI  - GENBANK/KF309856
SI  - GENBANK/KF309857
SI  - GENBANK/KF309858
SI  - GENBANK/KF309859
SI  - GENBANK/KF309860
SI  - GENBANK/KF309861
SI  - GENBANK/KF309862
SI  - GENBANK/KF309863
SI  - GENBANK/KF309864
SI  - GENBANK/KF309865
SI  - GENBANK/KF309866
SI  - GENBANK/KF309867
SI  - GENBANK/KF309868
SI  - GENBANK/KF309869
SI  - GENBANK/KF309870
SI  - GENBANK/KF309871
SI  - GENBANK/KF309872
SI  - GENBANK/KF309873
SI  - GENBANK/KF309874
SI  - GENBANK/KF309875
SI  - GENBANK/KF309876
SI  - GENBANK/KF309877
SI  - GENBANK/KF309878
SI  - GENBANK/KF309879
SI  - GENBANK/KF309880
SI  - GENBANK/KF309881
SI  - GENBANK/KF309882
SI  - GENBANK/KF309883
SI  - GENBANK/KF309884
SI  - GENBANK/KF309885
SI  - GENBANK/KF309886
SI  - GENBANK/KF309887
SI  - GENBANK/KF309888
SI  - GENBANK/KF309889
SI  - GENBANK/KF309890
SI  - GENBANK/KF309891
SI  - GENBANK/KF309892
SI  - GENBANK/KF309893
SI  - GENBANK/KF309894
SI  - GENBANK/KF309895
SI  - GENBANK/KF309896
SI  - GENBANK/KF309897
SI  - GENBANK/KF309898
SI  - GENBANK/KF309899
SI  - GENBANK/KF309900
SI  - GENBANK/KF309901
SI  - GENBANK/KF309902
SI  - GENBANK/KF309903
SI  - GENBANK/KF309904
SI  - GENBANK/KF309905
SI  - GENBANK/KF309906
SI  - GENBANK/KF309907
SI  - GENBANK/KF309908
SI  - GENBANK/KF309909
SI  - GENBANK/KF309910
SI  - GENBANK/KF309911
SI  - GENBANK/KF309912
SI  - GENBANK/KF309913
SI  - GENBANK/KF309914
SI  - GENBANK/KF309915
SI  - GENBANK/KF309916
SI  - GENBANK/KF309917
SI  - GENBANK/KF309918
SI  - GENBANK/KF309919
SI  - GENBANK/KF309920
SI  - GENBANK/KF309921
SI  - GENBANK/KF309922
SI  - GENBANK/KF309923
SI  - GENBANK/KF309924
SI  - GENBANK/KF309925
SI  - GENBANK/KF309926
SI  - GENBANK/KF309927
SI  - GENBANK/KF309928
SI  - GENBANK/KF309929
SI  - GENBANK/KF309930
SI  - GENBANK/KF309931
SI  - GENBANK/KF309932
SI  - GENBANK/KF309933
SI  - GENBANK/KF309934
SI  - GENBANK/KF309935
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140226
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 5.99.1.- (DNA Topoisomerases)
RN  - EC 6.3.4.16 (Carbamoyl-Phosphate Synthase (Ammonia))
SB  - IM
MH  - Animals
MH  - Body Size
MH  - Carbamoyl-Phosphate Synthase (Ammonia)/genetics
MH  - Coleoptera/*classification/genetics/*physiology
MH  - DNA Topoisomerases/genetics
MH  - *Genetic Speciation
MH  - Geography
MH  - Madagascar
MH  - Molecular Sequence Data
MH  - *Phylogeny
EDAT- 2014/03/01 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/03/01 06:00
PHST- 2014/03/01 06:00 [entrez]
PHST- 2014/03/01 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu011 [pii]
AID - 10.1093/sysbio/syu011 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):480-92. doi: 10.1093/sysbio/syu011. Epub 2014 Feb 26.

PMID- 24562916
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20140415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - Predictive approaches to assessing the fit of evolutionary models.
PG  - 289-92
LID - 10.1093/sysbio/syu009 [doi]
FAU - Brown, Jeremy M
AU  - Brown JM
AD  - Department of Biological Sciences, Louisiana State University, Baton Rouge, LA
      70803, USA.
LA  - eng
PT  - Introductory Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140221
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - *Models, Biological
EDAT- 2014/02/25 06:00
MHDA- 2014/06/15 06:00
CRDT- 2014/02/25 06:00
PHST- 2014/02/25 06:00 [entrez]
PHST- 2014/02/25 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syu009 [pii]
AID - 10.1093/sysbio/syu009 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):289-92. doi: 10.1093/sysbio/syu009. Epub 2014 Feb 21.

PMID- 24562915
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20140415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - Predicting the ancestral character changes in a tree is typically easier than
      predicting the root state.
PG  - 421-35
LID - 10.1093/sysbio/syu010 [doi]
AB  - Predicting the ancestral sequences of a group of homologous sequences related by 
      a phylogenetic tree has been the subject of many studies, and numerous methods
      have been proposed for this purpose. Theoretical results are available that show 
      that when the substitution rates become too large, reconstructing the ancestral
      state at the tree root is no longer feasible. Here, we also study the
      reconstruction of the ancestral changes that occurred along the tree edges. We
      show that, that, depending on the tree and branch length distribution,
      reconstructing these changes (i.e., reconstructing the ancestral state of all
      internal nodes in the tree) may be easier or harder than reconstructing the
      ancestral root state. However, results from information theory indicate that for 
      the standard Yule tree, the task of reconstructing internal node states remains
      feasible, even for very high substitution rates. Moreover, computer simulations
      demonstrate that for more complex trees and scenarios, this result still holds.
      For a large variety of counting, parsimony- and likelihood-based methods, the
      predictive accuracy of a randomly selected internal node in the tree is indeed
      much higher than the accuracy of the same method when applied to the tree root.
      Moreover, parsimony- and likelihood-based methods appear to be remarkably robust 
      to sampling bias and model mis-specification.
FAU - Gascuel, Olivier
AU  - Gascuel O
AD  - Institut de Biologie Computationnelle, LIRMM, UMR 5506 CNRS - Univ. Montpellier
      2, Case courrier 06011, 95 rue de la Galera, 34095 Montpellier, France; Allan
      Wilson Centre, University of Canterbury, Ilam Road 8041, Christchurch, New
      Zealand.
FAU - Steel, Mike
AU  - Steel M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140221
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Classification
MH  - Computer Simulation
MH  - Likelihood Functions
MH  - *Models, Theoretical
MH  - *Phylogeny
MH  - Probability
EDAT- 2014/02/25 06:00
MHDA- 2014/06/15 06:00
CRDT- 2014/02/25 06:00
PHST- 2014/02/25 06:00 [entrez]
PHST- 2014/02/25 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syu010 [pii]
AID - 10.1093/sysbio/syu010 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):421-35. doi: 10.1093/sysbio/syu010. Epub 2014 Feb 21.

PMID- 24562813
OWN - NLM
STAT- MEDLINE
DCOM- 20140806
LR  - 20140614
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 4
DP  - 2014 Jul
TI  - The emergence of lobsters: phylogenetic relationships, morphological evolution
      and divergence time comparisons of an ancient group (decapoda: achelata,
      astacidea, glypheidea, polychelida).
PG  - 457-79
LID - 10.1093/sysbio/syu008 [doi]
AB  - Lobsters are a ubiquitous and economically important group of decapod crustaceans
      that include the infraorders Polychelida, Glypheidea, Astacidea and Achelata.
      They include familiar forms such as the spiny, slipper, clawed lobsters and
      crayfish and unfamiliar forms such as the deep-sea and "living fossil" species.
      The high degree of morphological diversity among these infraorders has led to a
      dynamic classification and conflicting hypotheses of evolutionary relationships. 
      In this study, we estimated phylogenetic relationships among the major groups of 
      all lobster families and 94% of the genera using six genes (mitochondrial and
      nuclear) and 195 morphological characters across 173 species of lobsters for the 
      most comprehensive sampling to date. Lobsters were recovered as a
      non-monophyletic assemblage in the combined (molecular + morphology) analysis.
      All families were monophyletic, with the exception of Cambaridae, and 7 of 79
      genera were recovered as poly- or paraphyletic. A rich fossil history coupled
      with dense taxon coverage allowed us to estimate and compare divergence times and
      origins of major lineages using two drastically different approaches. Age priors 
      were constructed and/or included based on fossil age information or fossil
      discovery, age, and extant species count data. Results from the two approaches
      were largely congruent across deep to shallow taxonomic divergences across major 
      lineages. The origin of the first lobster-like decapod (Polychelida) was
      estimated in the Devonian ( approximately 409-372 Ma) with all infraorders
      present in the Carboniferous ( approximately 353-318 Ma). Fossil calibration
      subsampling studies examined the influence of sampling density (number of
      fossils) and placement (deep, middle, and shallow) on divergence time estimates. 
      Results from our study suggest including at least 1 fossil per 10 operational
      taxonomic units (OTUs) in divergence dating analyses. [Dating; decapods;
      divergence; lobsters; molecular; morphology; phylogenetics.].
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved.For Permissions, please
      email: journals.permissions@oup.com.
FAU - Bracken-Grissom, Heather D
AU  - Bracken-Grissom HD
AD  - Department of Biology, Florida International University, 3000 NE 151st Street,
      North Miami, FL 33181, USA; hbracken@fiu.edu.
FAU - Ahyong, Shane T
AU  - Ahyong ST
AD  - Australian Museum, 6 College Street, Sydney, NSW 2010, Australia;University of
      New South Wales, Kensington, NSW 2052, Australia;
FAU - Wilkinson, Richard D
AU  - Wilkinson RD
AD  - School of Mathematical Sciences, University of Nottingham, Nottingham, UK;
FAU - Feldmann, Rodney M
AU  - Feldmann RM
AD  - Department of Geology, Kent State University, Kent, OH 44242, USA;
FAU - Schweitzer, Carrie E
AU  - Schweitzer CE
AD  - Kent State University at Stark, 6000 Frank Avenue NW, North Canton, OH 44720,
      USA;
FAU - Breinholt, Jesse W
AU  - Breinholt JW
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL 32611, 
      USA;
FAU - Bendall, Matthew
AU  - Bendall M
AD  - Brigham Young University, 401 WIDB, Provo, UT 84606, USA;
FAU - Palero, Ferran
AU  - Palero F
AD  - Unitat Mixta Genomica i Salut CSISP-UV, Institut Cavanilles Universitat de
      Valencia, C/Catedratico Jose Beltran 2,46980 Paterna, Spain;
FAU - Chan, Tin-Yam
AU  - Chan TY
AD  - Institute of Marine Biology and Center of Excellence for Marine Bioenvironment
      and Biotechnology, National Taiwan Ocean University, Keelung 20224, Taiwan,
      R.O.C.;
FAU - Felder, Darryl L
AU  - Felder DL
AD  - Department of Biology, University of Louisiana at Lafayette, Lafayette, LA 70504,
      USA;
FAU - Robles, Rafael
AU  - Robles R
AD  - Laboratory of Bioecology and Crustacean Systematics, Department of Biology,
      FFCLRP, University of Sao Paulo (USP), Ave. Bandeirantes 3900, CEP 14040 - 901,
      Ribeirao Preto, SP Brazil;
FAU - Chu, Ka-Hou
AU  - Chu KH
AD  - School of Life Sciences, The Chinese University of Hong Kong, Shatin, Hong Kong;
FAU - Tsang, Ling-Ming
AU  - Tsang LM
AD  - Institute of Marine Biology and Center of Excellence for Marine Bioenvironment
      and Biotechnology, National Taiwan Ocean University, Keelung 20224, Taiwan,
      R.O.C.;School of Life Sciences, The Chinese University of Hong Kong, Shatin, Hong
      Kong;
FAU - Kim, Dohyup
AU  - Kim D
AD  - Brigham Young University, 401 WIDB, Provo, UT 84606, USA;
FAU - Martin, Joel W
AU  - Martin JW
AD  - Natural History Museum of Los Angeles County, Los Angeles, CA 90007, USA;
FAU - Crandall, Keith A
AU  - Crandall KA
AD  - George Washington University, Computational Biology Institute, Ashburn, VA 20147,
      USA;
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140220
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Arthropod Proteins)
SB  - IM
MH  - Animals
MH  - Arthropod Proteins/genetics
MH  - Biological Evolution
MH  - Decapoda (Crustacea)/*anatomy &amp; histology/*classification/genetics
MH  - *Fossils
MH  - *Phylogeny
MH  - Time
EDAT- 2014/02/25 06:00
MHDA- 2014/08/07 06:00
CRDT- 2014/02/25 06:00
PHST- 2014/02/25 06:00 [entrez]
PHST- 2014/02/25 06:00 [pubmed]
PHST- 2014/08/07 06:00 [medline]
AID - syu008 [pii]
AID - 10.1093/sysbio/syu008 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jul;63(4):457-79. doi: 10.1093/sysbio/syu008. Epub 2014 Feb 20.

PMID- 24562812
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20140415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - A Bayesian method for analyzing lateral gene transfer.
PG  - 409-20
LID - 10.1093/sysbio/syu007 [doi]
AB  - Lateral gene transfer (LGT)--which transfers DNA between two non-vertically
      related individuals belonging to the same or different species--is recognized as 
      a major force in prokaryotic evolution, and evidence of its impact on eukaryotic 
      evolution is ever increasing. LGT has attracted much public attention for its
      potential to transfer pathogenic elements and antibiotic resistance in bacteria, 
      and to transfer pesticide resistance from genetically modified crops to other
      plants. In a wider perspective, there is a growing body of studies highlighting
      the role of LGT in enabling organisms to occupy new niches or adapt to
      environmental changes. The challenge LGT poses to the standard tree-based
      conception of evolution is also being debated. Studies of LGT have, however, been
      severely limited by a lack of computational tools. The best currently available
      LGT algorithms are parsimony-based phylogenetic methods, which require a
      pre-computed gene tree and cannot choose between sometimes wildly differing most 
      parsimonious solutions. Moreover, in many studies, simple heuristics are applied 
      that can only handle putative orthologs and completely disregard gene
      duplications (GDs). Consequently, proposed LGT among specific gene families, and 
      the rate of LGT in general, remain debated. We present a Bayesian Markov-chain
      Monte Carlo-based method that integrates GD, gene loss, LGT, and sequence
      evolution, and apply the method in a genome-wide analysis of two groups of
      bacteria: Mollicutes and Cyanobacteria. Our analyses show that although the LGT
      rate between distant species is high, the net combined rate of duplication and
      close-species LGT is on average higher. We also show that the common practice of 
      disregarding reconcilability in gene tree inference overestimates the number of
      LGT and duplication events.
FAU - Sjostrand, Joel
AU  - Sjostrand J
AD  - Science for Life Laboratory, Tomtebodavagen 23A, 17165 Solna, Sweden, Department 
      of Numerical Analysis and Computer Science, Stockholm University, Sweden, McGill 
      Centre for Bioinformatics, 4th floor, Bellini Building, Life Sciences Complex,
      3649 Promenade Sir William Osler, Montreal, Quebec, Canada, H3G 0B1, UMR CNRS
      5558 - LBBE, "Biometrie et Biologie evolutive", UCB Lyon 1 - Bat. Gregor Mendel, 
      43 bd du 11 novembre 1918, 69622 VILLEURBANNE cedex, Department of Mechanics,
      Osquars Backe 18, KTH, SE-100 44 Stockholm, Sweden, Karolinska University
      Hospital, CMM L8:03, Solna, SE-171 76 Stockholm, Sweden and The School of
      Computer Science and Communication, Lindstedtsvagen 3, 5, KTH CSC, SE-100 44
      Stockholm, Sweden.
FAU - Tofigh, Ali
AU  - Tofigh A
FAU - Daubin, Vincent
AU  - Daubin V
FAU - Arvestad, Lars
AU  - Arvestad L
FAU - Sennblad, Bengt
AU  - Sennblad B
FAU - Lagergren, Jens
AU  - Lagergren J
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140220
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Cyanobacteria/classification/genetics
MH  - Evolution, Molecular
MH  - *Gene Transfer, Horizontal
MH  - Models, Theoretical
MH  - Phylogeny
MH  - Tenericutes/classification/genetics
EDAT- 2014/02/25 06:00
MHDA- 2014/06/15 06:00
CRDT- 2014/02/25 06:00
PHST- 2014/02/25 06:00 [entrez]
PHST- 2014/02/25 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syu007 [pii]
AID - 10.1093/sysbio/syu007 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):409-20. doi: 10.1093/sysbio/syu007. Epub 2014 Feb 20.

PMID- 24510972
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - Bayesian estimation of speciation and extinction from incomplete fossil
      occurrence data.
PG  - 349-67
LID - 10.1093/sysbio/syu006 [doi]
AB  - The temporal dynamics of species diversity are shaped by variations in the rates 
      of speciation and extinction, and there is a long history of inferring these
      rates using first and last appearances of taxa in the fossil record.
      Understanding diversity dynamics critically depends on unbiased estimates of the 
      unobserved times of speciation and extinction for all lineages, but the inference
      of these parameters is challenging due to the complex nature of the available
      data. Here, we present a new probabilistic framework to jointly estimate
      species-specific times of speciation and extinction and the rates of the
      underlying birth-death process based on the fossil record. The rates are allowed 
      to vary through time independently of each other, and the probability of
      preservation and sampling is explicitly incorporated in the model to estimate the
      true lifespan of each lineage. We implement a Bayesian algorithm to assess the
      presence of rate shifts by exploring alternative diversification models. Tests on
      a range of simulated data sets reveal the accuracy and robustness of our approach
      against violations of the underlying assumptions and various degrees of data
      incompleteness. Finally, we demonstrate the application of our method with the
      diversification of the mammal family Rhinocerotidae and reveal a complex history 
      of repeated and independent temporal shifts of both speciation and extinction
      rates, leading to the expansion and subsequent decline of the group. The
      estimated parameters of the birth-death process implemented here are directly
      comparable with those obtained from dated molecular phylogenies. Thus, our model 
      represents a step towards integrating phylogenetic and fossil information to
      infer macroevolutionary processes.
FAU - Silvestro, Daniele
AU  - Silvestro D
AD  - Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne,
      Switzerland;
FAU - Schnitzler, Jan
AU  - Schnitzler J
FAU - Liow, Lee Hsiang
AU  - Liow LH
FAU - Antonelli, Alexandre
AU  - Antonelli A
FAU - Salamin, Nicolas
AU  - Salamin N
LA  - eng
GR  - 311024/European Research Council/International
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140208
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Biodiversity
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Extinction, Biological
MH  - *Fossils
MH  - *Genetic Speciation
MH  - Mammals/classification
MH  - *Models, Biological
PMC - PMC4361715
MID - EMS56921
OID - NLM: EMS56921
EDAT- 2014/02/11 06:00
MHDA- 2014/06/15 06:00
CRDT- 2014/02/11 06:00
PHST- 2014/02/11 06:00 [entrez]
PHST- 2014/02/11 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syu006 [pii]
AID - 10.1093/sysbio/syu006 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):349-67. doi: 10.1093/sysbio/syu006. Epub 2014 Feb 8.

PMID- 24500037
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20140415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - A linear-time algorithm for Gaussian and non-Gaussian trait evolution models.
PG  - 397-408
LID - 10.1093/sysbio/syu005 [doi]
AB  - We developed a linear-time algorithm applicable to a large class of trait
      evolution models, for efficient likelihood calculations and parameter inference
      on very large trees. Our algorithm solves the traditional computational burden
      associated with two key terms, namely the determinant of the phylogenetic
      covariance matrix V and quadratic products involving the inverse of V.
      Applications include Gaussian models such as Brownian motion-derived models like 
      Pagel's lambda, kappa, delta, and the early-burst model; Ornstein-Uhlenbeck
      models to account for natural selection with possibly varying selection
      parameters along the tree; as well as non-Gaussian models such as phylogenetic
      logistic regression, phylogenetic Poisson regression, and phylogenetic
      generalized linear mixed models. Outside of phylogenetic regression, our
      algorithm also applies to phylogenetic principal component analysis, phylogenetic
      discriminant analysis or phylogenetic prediction. The computational gain opens up
      new avenues for complex models or extensive resampling procedures on very large
      trees. We identify the class of models that our algorithm can handle as all
      models whose covariance matrix has a 3-point structure. We further show that this
      structure uniquely identifies a rooted tree whose branch lengths parametrize the 
      trait covariance matrix, which acts as a similarity matrix. The new algorithm is 
      implemented in the R package phylolm, including functions for phylogenetic linear
      regression and phylogenetic logistic regression.
FAU - Ho, Lam si Tung
AU  - Ho Ls
AD  - Department of Statistics, University of Wisconsin - Madison, Madison, WI 53706,
      USA.
FAU - Ane, Cecile
AU  - Ane C
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140204
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Software/*standards
EDAT- 2014/02/07 06:00
MHDA- 2014/06/15 06:00
CRDT- 2014/02/07 06:00
PHST- 2014/02/07 06:00 [entrez]
PHST- 2014/02/07 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syu005 [pii]
AID - 10.1093/sysbio/syu005 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):397-408. doi: 10.1093/sysbio/syu005. Epub 2014 Feb 4.

PMID- 24449041
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20140415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - Morphological clocks in paleontology, and a mid-Cretaceous origin of crown Aves.
PG  - 442-9
LID - 10.1093/sysbio/syt110 [doi]
FAU - Lee, Michael S Y
AU  - Lee MS
AD  - Earth Sciences Section, South Australian Museum, North Terrace, Adelaide 5000,
      Australia; School of Earth and Environmental Sciences, University of Adelaide
      5005, Australia; Museo Geologico e Paleontologico "Giovanni Capellini", Via
      Zamboni 63, 40126 Bologna, Italy; and Ocean and Earth Science, University of
      Southampton, Southampton SO14 3ZH, UK.
FAU - Cau, Andrea
AU  - Cau A
FAU - Naish, Darren
AU  - Naish D
FAU - Dyke, Gareth J
AU  - Dyke GJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140121
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Birds/anatomy &amp; histology/*classification/genetics
MH  - Fossils
MH  - *Paleontology
MH  - *Phylogeny
EDAT- 2014/01/23 06:00
MHDA- 2014/06/15 06:00
CRDT- 2014/01/23 06:00
PHST- 2014/01/23 06:00 [entrez]
PHST- 2014/01/23 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syt110 [pii]
AID - 10.1093/sysbio/syt110 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):442-9. doi: 10.1093/sysbio/syt110. Epub 2014 Jan 21.

PMID- 24436254
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20140415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - FlatNJ: a novel network-based approach to visualize evolutionary and
      biogeographical relationships.
PG  - 383-96
LID - 10.1093/sysbio/syu001 [doi]
AB  - Split networks are a type of phylogenetic network that allow visualization of
      conflict in evolutionary data. We present a new method for constructing such
      networks called FlatNetJoining (FlatNJ). A key feature of FlatNJ is that it
      produces networks that can be drawn in the plane in which labels may appear
      inside of the network. For complex data sets that involve, for example,
      non-neutral molecular markers, this can allow additional detail to be visualized 
      as compared to previous methods such as split decomposition and NeighborNet. We
      illustrate the application of FlatNJ by applying it to whole HIV genome
      sequences, where recombination has taken place, fluorescent proteins in corals,
      where ancestral sequences are present, and mitochondrial DNA sequences from gall 
      wasps, where biogeographical relationships are of interest. We find that the
      networks generated by FlatNJ can facilitate the study of genetic variation in the
      underlying molecular sequence data and, in particular, may help to investigate
      processes such as intra-locus recombination. FlatNJ has been implemented in Java 
      and is freely available at www.uea.ac.uk/computing/software/flatnj.
FAU - Balvocute, Monika
AU  - Balvocute M
AD  - Department of Mathematics and Computer Science, University of Greifswald,
      Walther-Rathenau-Str. 47, 17487 Greifswald, Germany and School of Computing
      Sciences, University of East Anglia, Norwich, NR4 7TJ, UK.
FAU - Spillner, Andreas
AU  - Spillner A
FAU - Moulton, Vincent
AU  - Moulton V
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140115
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - HIV/classification/genetics
MH  - Phylogeny
MH  - Phylogeography/*methods
MH  - Statistics as Topic
MH  - Wasps/classification/genetics
EDAT- 2014/01/18 06:00
MHDA- 2014/06/15 06:00
CRDT- 2014/01/18 06:00
PHST- 2014/01/18 06:00 [entrez]
PHST- 2014/01/18 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syu001 [pii]
AID - 10.1093/sysbio/syu001 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):383-96. doi: 10.1093/sysbio/syu001. Epub 2014 Jan 15.

PMID- 24415681
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20140415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - Detection of implausible phylogenetic inferences using posterior predictive
      assessment of model fit.
PG  - 334-48
LID - 10.1093/sysbio/syu002 [doi]
AB  - Systematic phylogenetic error caused by the simplifying assumptions made in
      models of molecular evolution may be impossible to avoid entirely when attempting
      to model evolution across massive, diverse data sets. However, not all
      deficiencies of inference models result in unreliable phylogenetic estimates. The
      field of phylogenetics lacks a direct method to identify cases where model
      specification adversely affects inferences. Posterior predictive simulation is a 
      flexible and intuitive approach for assessing goodness-of-fit of the assumed
      model and priors in a Bayesian phylogenetic analysis. Here, I propose new test
      statistics for use in posterior predictive assessment of model fit. These test
      statistics compare phylogenetic inferences from posterior predictive data sets to
      inferences from the original data. A simulation study demonstrates the utility of
      these new statistics. The new tests reject the plausibility of inferred tree
      lengths or topologies more often when data/model combinations produce biased
      inferences. I also apply this approach to exemplar empirical data sets,
      highlighting the value of the novel assessments.
FAU - Brown, Jeremy M
AU  - Brown JM
AD  - Department of Biological Sciences, Louisiana State University, Baton Rouge, LA
      70803, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140111
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - *Computer Simulation
MH  - Evolution, Molecular
MH  - *Models, Theoretical
MH  - *Phylogeny
EDAT- 2014/01/15 06:00
MHDA- 2014/06/15 06:00
CRDT- 2014/01/14 06:00
PHST- 2014/01/14 06:00 [entrez]
PHST- 2014/01/15 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syu002 [pii]
AID - 10.1093/sysbio/syu002 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):334-48. doi: 10.1093/sysbio/syu002. Epub 2014 Jan 11.

PMID- 24415680
OWN - NLM
STAT- MEDLINE
DCOM- 20141003
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 5
DP  - 2014 Sep
TI  - Why one century of phenetics is enough: response to "Are there really twice as
      many bovid species as we thought?".
PG  - 819-32
LID - 10.1093/sysbio/syu003 [doi]
FAU - Cotterill, Fenton P D
AU  - Cotterill FP
AD  - AEON-Africa Earth Observatory Network, Geoecodynamics Research Hub, c/o
      Department of Botany and Zoology, University of Stellenbosch, Stellenbosch 7602, 
      South Africa; SARChI Chair on Biodiversity Value &amp; Change in the Vhembe Biosphere
      Reserve &amp; Core Member of Centre for Invasion Biology, School of Mathematical and 
      Natural Sciences, University of Venda, Private Bag X5050, Thohoyandou 0950, South
      Africa; Viale Liegi 48, 00198 Roma, Italy; Department of Biological Sciences,
      University of Cape Town, Private Bag, Rondebosch 7700, South Africa; and School
      of Archaeology and Anthropology, Australian National University, Canberra 0200,
      Australia fcotterill@gmail.com.
FAU - Taylor, Peter J
AU  - Taylor PJ
AD  - AEON-Africa Earth Observatory Network, Geoecodynamics Research Hub, c/o
      Department of Botany and Zoology, University of Stellenbosch, Stellenbosch 7602, 
      South Africa; SARChI Chair on Biodiversity Value &amp; Change in the Vhembe Biosphere
      Reserve &amp; Core Member of Centre for Invasion Biology, School of Mathematical and 
      Natural Sciences, University of Venda, Private Bag X5050, Thohoyandou 0950, South
      Africa; Viale Liegi 48, 00198 Roma, Italy; Department of Biological Sciences,
      University of Cape Town, Private Bag, Rondebosch 7700, South Africa; and School
      of Archaeology and Anthropology, Australian National University, Canberra 0200,
      Australia.
FAU - Gippoliti, Spartaco
AU  - Gippoliti S
AD  - AEON-Africa Earth Observatory Network, Geoecodynamics Research Hub, c/o
      Department of Botany and Zoology, University of Stellenbosch, Stellenbosch 7602, 
      South Africa; SARChI Chair on Biodiversity Value &amp; Change in the Vhembe Biosphere
      Reserve &amp; Core Member of Centre for Invasion Biology, School of Mathematical and 
      Natural Sciences, University of Venda, Private Bag X5050, Thohoyandou 0950, South
      Africa; Viale Liegi 48, 00198 Roma, Italy; Department of Biological Sciences,
      University of Cape Town, Private Bag, Rondebosch 7700, South Africa; and School
      of Archaeology and Anthropology, Australian National University, Canberra 0200,
      Australia.
FAU - Bishop, Jacqueline M
AU  - Bishop JM
AD  - AEON-Africa Earth Observatory Network, Geoecodynamics Research Hub, c/o
      Department of Botany and Zoology, University of Stellenbosch, Stellenbosch 7602, 
      South Africa; SARChI Chair on Biodiversity Value &amp; Change in the Vhembe Biosphere
      Reserve &amp; Core Member of Centre for Invasion Biology, School of Mathematical and 
      Natural Sciences, University of Venda, Private Bag X5050, Thohoyandou 0950, South
      Africa; Viale Liegi 48, 00198 Roma, Italy; Department of Biological Sciences,
      University of Cape Town, Private Bag, Rondebosch 7700, South Africa; and School
      of Archaeology and Anthropology, Australian National University, Canberra 0200,
      Australia.
FAU - Groves, Colin P
AU  - Groves CP
AD  - AEON-Africa Earth Observatory Network, Geoecodynamics Research Hub, c/o
      Department of Botany and Zoology, University of Stellenbosch, Stellenbosch 7602, 
      South Africa; SARChI Chair on Biodiversity Value &amp; Change in the Vhembe Biosphere
      Reserve &amp; Core Member of Centre for Invasion Biology, School of Mathematical and 
      Natural Sciences, University of Venda, Private Bag X5050, Thohoyandou 0950, South
      Africa; Viale Liegi 48, 00198 Roma, Italy; Department of Biological Sciences,
      University of Cape Town, Private Bag, Rondebosch 7700, South Africa; and School
      of Archaeology and Anthropology, Australian National University, Canberra 0200,
      Australia.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Comment
DEP - 20140111
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2013 May 1;62(3):490-3. PMID: 23362112
CIN - Syst Biol. 2014 Sep;63(5):833-7. PMID: 24831669
MH  - Animals
MH  - *Genetic Speciation
MH  - Ruminants/*classification/*genetics
EDAT- 2014/01/15 06:00
MHDA- 2014/10/04 06:00
CRDT- 2014/01/14 06:00
PHST- 2014/01/14 06:00 [entrez]
PHST- 2014/01/15 06:00 [pubmed]
PHST- 2014/10/04 06:00 [medline]
AID - syu003 [pii]
AID - 10.1093/sysbio/syu003 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Sep;63(5):819-32. doi: 10.1093/sysbio/syu003. Epub 2014 Jan 11.

PMID- 24399481
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - Conflicting phylogenies for early land plants are caused by composition biases
      among synonymous substitutions.
PG  - 272-9
LID - 10.1093/sysbio/syt109 [doi]
FAU - Cox, Cymon J
AU  - Cox CJ
AD  - Centro de Ciencias do Mar, Universidade do Algarve, Gambelas, 8005-319 Faro,
      Portugal;Department of Life Sciences, Natural History Museum, London SW7 5BD, UK;
      and Institute for Cell and Molecular Biosciences, University of Newcastle,
      Newcastle upon Tyne NE2 4HH, UK.
FAU - Li, Blaise
AU  - Li B
FAU - Foster, Peter G
AU  - Foster PG
FAU - Embley, T Martin
AU  - Embley TM
FAU - Civan, Peter
AU  - Civan P
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140106
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Chloroplast Proteins)
RN  - 0 (Codon)
SB  - IM
MH  - Base Composition
MH  - Chloroplast Proteins/*genetics
MH  - Codon/genetics
MH  - Embryophyta/*classification
MH  - Genes, Plant/*genetics
MH  - Models, Genetic
MH  - Mutation
MH  - *Phylogeny
PMC - PMC3926305
EDAT- 2014/01/09 06:00
MHDA- 2014/04/20 06:00
CRDT- 2014/01/09 06:00
PHST- 2014/01/09 06:00 [entrez]
PHST- 2014/01/09 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt109 [pii]
AID - 10.1093/sysbio/syt109 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):272-9. doi: 10.1093/sysbio/syt109. Epub 2014 Jan 6.

PMID- 24391149
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - Another look at the root of the angiosperms reveals a familiar tale.
PG  - 368-82
LID - 10.1093/sysbio/syt108 [doi]
AB  - Since the advent of molecular phylogenetics more than 25 years ago, a major goal 
      of plant systematists has been to discern the root of the angiosperms. Although
      most studies indicate that Amborella trichopoda is sister to all remaining extant
      flowering plants, support for this position has varied with respect to both the
      sequence data sets and analyses employed. Recently, Goremykin et al. (2013)
      questioned the "Amborella-sister hypothesis" using a "noise-reduction" approach
      and reported a topology with Amborella + Nymphaeales (water lilies) sister to all
      remaining angiosperms. Through a series of analyses of both plastid genomes and
      mitochondrial genes, we continue to find mostly strong support for the
      Amborella-sister hypothesis and offer a rebuttal of Goremykin et al. (2013). The 
      major tenet of Goremykin et al. is that the Amborella-sister position is
      determined by noisy data--that is, characters with high rates of change and
      lacking true phylogenetic signal. To investigate the signal in these noisy data
      further, we analyzed the discarded characters from their noise-reduced
      alignments. We recovered a tree identical to that of the currently accepted
      angiosperm framework, including the position of Amborella as sister to all other 
      angiosperms, as well as all other major clades. Thus, the signal in the "noisy"
      data is consistent with that of our complete data sets--arguing against the use
      of their noise-reduction approach. We also determined that one of the alignments 
      presented by Goremykin et al. yields results at odds with their central
      claim--their data set actually supports Amborella as sister to all other
      angiosperms, as do larger plastid data sets we present here that possess more
      complete taxon sampling both within the monocots and for angiosperms in general. 
      Previous unpartitioned, multilocus analyses of mitochondrial DNA (mtDNA) data
      have provided the strongest support for Amborella + Nymphaeales as sister to
      other angiosperms. However, our analysis of third codon positions from mtDNA
      sequence data also supports the Amborella-sister hypothesis. Finally, we
      challenge the conclusion of Goremykin et al. that the first flowering plants were
      aquatic and herbaceous, reasserting that even if Amborella + water lilies, or
      water lilies alone, are sister to the rest of the angiosperms, the earliest
      angiosperms were not necessarily aquatic and/or herbaceous.
FAU - Drew, Bryan T
AU  - Drew BT
AD  - Department of Biology, University of Florida, Gainesville, FL 32611, USA; Florida
      Museum of Natural History, University of Florida, Gainesville, FL 32611, USA;
      Department of Biological Sciences, Eastern Kentucky University, Richmond, KY
      40475, USA; Department of Ecology and Evolutionary Biology, University of
      Michigan, Ann Arbor, MI 48105, USA; Department of Biology, Oberlin College,
      Oberlin, OH 44074, USA; and National Herbarium of New South Wales, Botanic
      Gardens Trust, Sydney, NSW 2000, Australia.
FAU - Ruhfel, Brad R
AU  - Ruhfel BR
FAU - Smith, Stephen A
AU  - Smith SA
FAU - Moore, Michael J
AU  - Moore MJ
FAU - Briggs, Barbara G
AU  - Briggs BG
FAU - Gitzendanner, Matthew A
AU  - Gitzendanner MA
FAU - Soltis, Pamela S
AU  - Soltis PS
FAU - Soltis, Douglas E
AU  - Soltis DE
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Comment
DEP - 20140103
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2013 Jan 1;62(1):50-61. PMID: 22851550
CIN - Syst Biol. 2015 Sep;64(5):879-91. PMID: 25995065
MH  - Magnoliopsida/*classification/*genetics
MH  - *Phylogeny
EDAT- 2014/01/07 06:00
MHDA- 2014/06/15 06:00
CRDT- 2014/01/07 06:00
PHST- 2014/01/07 06:00 [entrez]
PHST- 2014/01/07 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syt108 [pii]
AID - 10.1093/sysbio/syt108 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):368-82. doi: 10.1093/sysbio/syt108. Epub 2014 Jan 3.

PMID- 24361591
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20140218
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - Upstream analyses create problems with DNA-based species delimitation.
PG  - 263-71
LID - 10.1093/sysbio/syt106 [doi]
FAU - Olave, Melisa
AU  - Olave M
AD  - Centro Nacional Patagonico - Consejo Nacional de Investigaciones Cientificas y
      Tecnicas (CENPAT-CONICET), Puerto Madryn, Chubut U 9120 ACD, Argentina,
      Department de Genetica, Facultat de Biologia and Institut de Recerca de la
      Biodiversitat (IRBio), Universitat de Barcelona, Av. Diagonal, 643, 08028,
      Barcelona, Catalonia, Spain and Department of Ecology and Evolutionary Biology,
      The University of Michigan, Ann Arbor, MI 41809-1029, USA.
FAU - Sola, Eduard
AU  - Sola E
FAU - Knowles, L Lacey
AU  - Knowles LL
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20131220
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Genetic Speciation
MH  - Lizards/classification/genetics
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Sequence Analysis, DNA/*standards
MH  - Species Specificity
EDAT- 2013/12/24 06:00
MHDA- 2014/04/20 06:00
CRDT- 2013/12/24 06:00
PHST- 2013/12/24 06:00 [entrez]
PHST- 2013/12/24 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt106 [pii]
AID - 10.1093/sysbio/syt106 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):263-71. doi: 10.1093/sysbio/syt106. Epub 2013 Dec 20.

PMID- 24352715
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20140415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - The shape of modern tree reconstruction methods.
PG  - 436-41
LID - 10.1093/sysbio/syt103 [doi]
FAU - Holton, Therese A
AU  - Holton TA
AD  - Department of Biology, The National University of Ireland Maynooth, Maynooth, Co.
      Kildare, Ireland; UCD Complex and Adaptive Systems Laboratory/UCD Conway
      Institute of Biomolecular and Biomedical Science/School of Medicine and Medical
      Science, University College Dublin, Dublin 4, Ireland; Department of Life
      Sciences, The Natural History Museum, London SW7 5BD, UK; School of Earth
      Sciences and School of Biological Sciences, The University of Bristol, Woodland
      Road, Bristol BS8 1UG, UK.
FAU - Wilkinson, Mark
AU  - Wilkinson M
FAU - Pisani, Davide
AU  - Pisani D
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20131218
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - *Classification
MH  - *Phylogeny
EDAT- 2013/12/20 06:00
MHDA- 2014/06/15 06:00
CRDT- 2013/12/20 06:00
PHST- 2013/12/20 06:00 [entrez]
PHST- 2013/12/20 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syt103 [pii]
AID - 10.1093/sysbio/syt103 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):436-41. doi: 10.1093/sysbio/syt103. Epub 2013 Dec 18.

PMID- 24335493
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20190116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - On the biogeography of Centipeda: a species-tree diffusion approach.
PG  - 178-91
LID - 10.1093/sysbio/syt102 [doi]
AB  - Reconstructing the biogeographic history of groups present in continuous arid
      landscapes is challenging due to the difficulties in defining discrete areas for 
      analyses, and even more so when species largely overlap both in terms of
      geography and habitat preference. In this study, we use a novel approach to
      estimate ancestral areas for the small plant genus Centipeda. We apply continuous
      diffusion of geography by a relaxed random walk where each species is sampled
      from its extant distribution on an empirical distribution of time-calibrated
      species-trees. Using a distribution of previously published substitution rates of
      the internal transcribed spacer (ITS) for Asteraceae, we show how the evolution
      of Centipeda correlates with the temporal increase of aridity in the arid zone
      since the Pliocene. Geographic estimates of ancestral species show a consistent
      pattern of speciation of early lineages in the Lake Eyre region, with a division 
      in more northerly and southerly groups since approximately 840 ka. Summarizing
      the geographic slices of species-trees at the time of the latest speciation event
      ( approximately 20 ka), indicates no presence of the genus in Australia west of
      the combined desert belt of the Nullabor Plain, the Great Victoria Desert, the
      Gibson Desert, and the Great Sandy Desert, or beyond the main continental shelf
      of Australia. The result indicates all western occurrences of the genus to be a
      result of recent dispersal rather than ancient vicariance. This study contributes
      to our understanding of the spatiotemporal processes shaping the flora of the
      arid zone, and offers a significant improvement in inference of ancestral areas
      for any organismal group distributed where it remains difficult to describe
      geography in terms of discrete areas.
FAU - Nylinder, Stephan
AU  - Nylinder S
AD  - Department of Botany, Swedish Museum of Natural History, Svante Arrhenius vag 7, 
      SE-104 05 Stockholm, Sweden; Department of Microbiology and Immunology, KU
      Leuven, Minderbroederstraat 10 blok x, box 1030 BE-3000 Leuven, Belgium; School
      of Biological Sciences, Bangor University, Deiniol Road Bangor Gwynedd, Bangor,
      LL57 2UW UK; Departments of Biomathematics and Human Genetics, David Geffen
      School of Medicine at UCLA, BOX 951766, University of California, Los Angeles, CA
      90095-1766, USA; Department of Biostatistics, UCLA Fielding School of Public
      Health, University of California, Los Angeles, CA 90095-1772, USA; Department of 
      Biological and Environmental Sciences, University of Gothenburg, Carl Skottsbergs
      gata 22 B, SE-413 19 Gothenburg, Sweden; and National Herbarium of Victoria,
      Birdwood Avenue, South Yarra 3141, Australia.
FAU - Lemey, Philippe
AU  - Lemey P
FAU - De Bruyn, Mark
AU  - De Bruyn M
FAU - Suchard, Marc A
AU  - Suchard MA
FAU - Pfeil, Bernard E
AU  - Pfeil BE
FAU - Walsh, Neville
AU  - Walsh N
FAU - Anderberg, Arne A
AU  - Anderberg AA
LA  - eng
GR  - 260864/European Research Council/International
GR  - R01 HG006139/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20131212
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal Spacer)
SB  - IM
MH  - Animals
MH  - Asteraceae/*classification/genetics
MH  - Australia
MH  - DNA, Ribosomal Spacer/genetics
MH  - *Phylogeny
MH  - *Phylogeography
PMC - PMC3926304
OTO - NOTNLM
OT  - Australia
OT  - BEAST
OT  - Centipeda
OT  - Pliocene
OT  - biogeography
OT  - continuous diffusion
OT  - species-tree
EDAT- 2013/12/18 06:00
MHDA- 2014/04/20 06:00
CRDT- 2013/12/17 06:00
PHST- 2013/12/17 06:00 [entrez]
PHST- 2013/12/18 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt102 [pii]
AID - 10.1093/sysbio/syt102 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):178-91. doi: 10.1093/sysbio/syt102. Epub 2013 Dec 12.

PMID- 24335430
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20140218
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - Partial sequence homogenization in the 5S multigene families may generate
      sequence chimeras and spurious results in phylogenetic reconstructions.
PG  - 219-30
LID - 10.1093/sysbio/syt101 [doi]
AB  - Multigene families have provided opportunities for evolutionary biologists to
      assess molecular evolution processes and phylogenetic reconstructions at deep and
      shallow systematic levels. However, the use of these markers is not free of
      technical and analytical challenges. Many evolutionary studies that used the
      nuclear 5S rDNA gene family rarely used contiguous 5S coding sequences due to the
      routine use of head-to-tail polymerase chain reaction primers that are anchored
      to the coding region. Moreover, the 5S coding sequences have been concatenated
      with independent, adjacent gene units in many studies, creating simulated
      chimeric genes as the raw data for evolutionary analysis. This practice is based 
      on the tacitly assumed, but rarely tested, hypothesis that strict intra-locus
      concerted evolution processes are operating in 5S rDNA genes, without any
      empirical evidence as to whether it holds for the recovered data. The potential
      pitfalls of analysing the patterns of molecular evolution and reconstructing
      phylogenies based on these chimeric genes have not been assessed to date. Here,
      we compared the sequence integrity and phylogenetic behavior of entire versus
      concatenated 5S coding regions from a real data set obtained from closely related
      plant species (Medicago, Fabaceae). Our results suggest that within arrays
      sequence homogenization is partially operating in the 5S coding region, which is 
      traditionally assumed to be highly conserved. Consequently, concatenating 5S
      genes increases haplotype diversity, generating novel chimeric genotypes that
      most likely do not exist within the genome. In addition, the patterns of gene
      evolution are distorted, leading to incorrect haplotype relationships in some
      evolutionary reconstructions.
FAU - Galian, Jose A
AU  - Galian JA
AD  - Jardin Botanico, Universidad de Valencia, c/Quart 80, E-46008 Valencia, Spain;
      and Marimurtra Bot. Garden, Carl Faust Fdn., PO Box 112, E-17300 Blanes,
      Catalonia, Spain.
FAU - Rosato, Marcela
AU  - Rosato M
FAU - Rossello, Josep A
AU  - Rossello JA
LA  - eng
SI  - GENBANK/KF435084
SI  - GENBANK/KF435085
SI  - GENBANK/KF435086
SI  - GENBANK/KF435087
SI  - GENBANK/KF435088
SI  - GENBANK/KF435089
SI  - GENBANK/KF435090
SI  - GENBANK/KF435091
SI  - GENBANK/KF435092
SI  - GENBANK/KF435093
SI  - GENBANK/KF435094
SI  - GENBANK/KF435095
SI  - GENBANK/KF435096
SI  - GENBANK/KF435097
SI  - GENBANK/KF435098
SI  - GENBANK/KF435099
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20131210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 5S)
SB  - IM
MH  - *Evolution, Molecular
MH  - Medicago/*classification/*genetics
MH  - Molecular Sequence Data
MH  - Multigene Family/*genetics
MH  - *Phylogeny
MH  - RNA, Ribosomal, 5S/*genetics
OTO - NOTNLM
OT  - 5S rDNA
OT  - Medicago
OT  - concerted evolution
OT  - multigene families
OT  - phylogeny
EDAT- 2013/12/18 06:00
MHDA- 2014/04/20 06:00
CRDT- 2013/12/17 06:00
PHST- 2013/12/17 06:00 [entrez]
PHST- 2013/12/18 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt101 [pii]
AID - 10.1093/sysbio/syt101 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):219-30. doi: 10.1093/sysbio/syt101. Epub 2013 Dec 10.

PMID- 24335429
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20140218
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - Coalescent species delimitation in milksnakes (genus Lampropeltis) and impacts on
      phylogenetic comparative analyses.
PG  - 231-50
LID - 10.1093/sysbio/syt099 [doi]
AB  - Both gene-tree discordance and unrecognized diversity are sources of error for
      accurate estimation of species trees, and can affect downstream diversification
      analyses by obscuring the correct number of nodes, their density, and the lengths
      of the branches subtending them. Although the theoretical impact of gene-tree
      discordance on evolutionary analyses has been examined previously, the effect of 
      unsampled and cryptic diversity has not. Here, we examine how delimitation of
      previously unrecognized diversity in the milksnake (Lampropeltis triangulum) and 
      use of a species-tree approach affects both estimation of the Lampropeltis
      phylogeny and comparative analyses with respect to the timing of diversification.
      Coalescent species delimitation indicates that L. triangulum is not monophyletic 
      and that there are multiple species of milksnake, which increases the known
      species diversity in the genus Lampropeltis by 40%. Both genealogical and
      temporal discordance occurs between gene trees and the species tree, with
      evidence that mitochondrial DNA (mtDNA) introgression is a main factor. This
      discordance is further manifested in the preferred models of diversification,
      where the concatenated gene tree strongly supports an early burst of speciation
      during the Miocene, in contrast to species-tree estimates where diversification
      follows a birth-death model and speciation occurs mostly in the Pliocene and
      Pleistocene. This study highlights the crucial interaction among coalescent-based
      phylogeography and species delimitation, systematics, and species diversification
      analyses.
FAU - Ruane, Sara
AU  - Ruane S
AD  - Department of Biology, College of Staten Island, 2800 Victory Blvd., Staten
      Island, NY 10314, USA; The Graduate Center, City University of New York, 365 5th 
      Avenue, NY, NY 10016, USA; Department of Biology &amp; Burke Museum of Natural
      History and Culture, University of Washington, Box 351800, Seattle, WA
      98195-1800, USA; Department of Biological Sciences, The George Washington
      University, 2023 G St. NW, Washington, DC 20052, USA; and Department of
      Herpetology, American Museum of Natural History, Central Park West and 79th St., 
      New York, NY 10024.
FAU - Bryson, Robert W Jr
AU  - Bryson RW Jr
FAU - Pyron, R Alexander
AU  - Pyron RA
FAU - Burbrink, Frank T
AU  - Burbrink FT
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20131210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - *Classification
MH  - Colubridae/*classification/*genetics
MH  - DNA, Mitochondrial/genetics
MH  - Genetic Variation
MH  - *Phylogeny
OTO - NOTNLM
OT  - Divergence-time estimation
OT  - Lampropeltini
OT  - Pleistocene diversification
OT  - diversification rates
OT  - gene-tree/species-tree discordance
OT  - mtDNA introgression
EDAT- 2013/12/18 06:00
MHDA- 2014/04/20 06:00
CRDT- 2013/12/17 06:00
PHST- 2013/12/17 06:00 [entrez]
PHST- 2013/12/18 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt099 [pii]
AID - 10.1093/sysbio/syt099 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):231-50. doi: 10.1093/sysbio/syt099. Epub 2013 Dec 10.

PMID- 24335428
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20140218
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - Missing data estimation in morphometrics: how much is too much?
PG  - 203-18
LID - 10.1093/sysbio/syt100 [doi]
AB  - Fossil-based estimates of diversity and evolutionary dynamics mainly rely on the 
      study of morphological variation. Unfortunately, organism remains are often
      altered by post-mortem taphonomic processes such as weathering or distortion.
      Such a loss of information often prevents quantitative multivariate description
      and statistically-controlled comparisons of extinct species based on morphometric
      data. A common way to deal with missing data involves imputation methods that
      directly fill the missing cases with model estimates. Over the last years,
      several empirically-determined thresholds for the maximum acceptable proportion
      of missing values have been proposed in the literature, whereas other studies
      showed that this limit actually depends on various properties of the study data
      set and of the selected imputation method, and is by no way generalizable. We
      evaluate the relative performances of seven multiple imputation (MI) techniques
      through a simulation-based analysis under three distinct patterns of missing data
      distribution. Overall, Fully Conditional Specification and
      Expectation-Maximization algorithms provide the best compromises between
      imputation accuracy and coverage probability. MI techniques appear remarkably
      robust to the violation of basic assumptions such as the occurrence of
      taxonomically or anatomically biased patterns of missing data distribution,
      making differences in simulation results between the three patterns of missing
      data distribution much smaller than differences between the individual MI
      techniques. Based on these results, rather than proposing a new (set of)
      threshold value(s), we develop an approach combining the use of MIs with
      procrustean superimposition of principal component analysis results, in order to 
      directly visualize the effect of individual missing data imputation on an
      ordinated space. We provide an R function for users to implement the proposed
      procedure.
FAU - Clavel, Julien
AU  - Clavel J
AD  - Laboratoire de Geologie de Lyon, UMR 5276, CNRS, UCB Lyon 1, ENS Lyon, Campus de 
      la Doua, 2 rue Raphael Dubois, 69622 Villeurbanne Cedex, France; and IPHEP, UMR
      7262, CNRS &amp; Universite de Poitiers, Bat. B35, 6 rue M. Brunet, 86022 Poitiers
      Cedex, France.
FAU - Merceron, Gildas
AU  - Merceron G
FAU - Escarguel, Gilles
AU  - Escarguel G
LA  - eng
PT  - Journal Article
DEP - 20131210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Classification
MH  - Computer Simulation/standards
MH  - *Phylogeny
MH  - Principal Component Analysis
OTO - NOTNLM
OT  - Missing data
OT  - Procrustes superimposition
OT  - R function
OT  - morphometrics
OT  - multiple imputation
OT  - ordination
OT  - simulation
EDAT- 2013/12/18 06:00
MHDA- 2014/04/20 06:00
CRDT- 2013/12/17 06:00
PHST- 2013/12/17 06:00 [entrez]
PHST- 2013/12/18 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt100 [pii]
AID - 10.1093/sysbio/syt100 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):203-18. doi: 10.1093/sysbio/syt100. Epub 2013 Dec 10.

PMID- 24335427
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20140218
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - Species radiation by niche shifts in New Zealand's rockcresses (Pachycladon,
      Brassicaceae).
PG  - 192-202
LID - 10.1093/sysbio/syt104 [doi]
AB  - Adaptive radiations such as the Darwin finches in the Galapagos or the cichlid
      fishes from the Eastern African Great Lakes have been a constant source of
      inspiration for biologists and a stimulus for evolutionary thinking. A central
      concept behind adaptive radiation is that of evolution by niche shifts, or
      ecological speciation. Evidence for adaptive radiations generally requires a
      strong correlation between phenotypic traits and the environment. But adaptive
      traits are often cryptic, hence making this phenotype-environment approach
      difficult to implement. Here we propose a procedure for detecting adaptive
      radiation that focuses on species' ecological niche comparisons. It evaluates
      whether past ecological disparity in a group fits better a neutral Brownian
      motion model of ecological divergence or a niche shift model. We have evaluated
      this approach on New Zealand rockcresses (Pachycladon) that recently radiated in 
      the New Zealand Alps. We show that the pattern of ecological divergence rejects
      the neutral model and is consistent with that of a niche shift model. Our
      approach to detect adaptive radiation has the advantage over alternative
      approaches that it focuses on ecological niches, a key concept behind adaptive
      radiation. It also provides a way to evaluate the importance of ecological
      speciation in adaptive radiations and will have general application in
      evolutionary studies. In the case of Pachycladon, the high estimated
      diversification rate, the distinctive ecological niches of species, and the
      evidence for ecological speciation suggest a remarkable example of adaptive
      radiation.
FAU - Joly, Simon
AU  - Joly S
AD  - Institut de Recherche en Biologie Vegetale, Montreal Botanical Garden and
      Universite de Montreal, 4101 Sherbrooke East, Montreal H1X 2B2, Canada; Institute
      of Fundamental Sciences, Massey University, Private Bag 11222, Palmerston North, 
      New Zealand; and Landcare Research, P.O. Box 40, Lincoln, New Zealand.
FAU - Heenan, Peter B
AU  - Heenan PB
FAU - Lockhart, Peter J
AU  - Lockhart PJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20131210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Physiological
MH  - Biodiversity
MH  - Brassicaceae/*classification
MH  - *Environment
MH  - New Zealand
MH  - *Phylogeny
OTO - NOTNLM
OT  - Adaptive radiation
OT  - diversification rates
OT  - ecological niche modeling
OT  - ecological speciation
OT  - niche shifts
EDAT- 2013/12/18 06:00
MHDA- 2014/04/20 06:00
CRDT- 2013/12/17 06:00
PHST- 2013/12/17 06:00 [entrez]
PHST- 2013/12/18 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt104 [pii]
AID - 10.1093/sysbio/syt104 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):192-202. doi: 10.1093/sysbio/syt104. Epub 2013 Dec 10.

PMID- 24335426
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20140218
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - Quantifying and comparing phylogenetic evolutionary rates for shape and other
      high-dimensional phenotypic data.
PG  - 166-77
LID - 10.1093/sysbio/syt105 [doi]
AB  - Many questions in evolutionary biology require the quantification and comparison 
      of rates of phenotypic evolution. Recently, phylogenetic comparative methods have
      been developed for comparing evolutionary rates on a phylogeny for single,
      univariate traits (sigma(2)), and evolutionary rate matrices (R) for sets of
      traits treated simultaneously. However, high-dimensional traits like shape remain
      under-examined with this framework, because methods suited for such data have not
      been fully developed. In this article, I describe a method to quantify
      phylogenetic evolutionary rates for high-dimensional multivariate data (sigma2
      mult), found from the equivalency between statistical methods based on covariance
      matrices and those based on distance matrices (R-mode and Q-mode methods). I then
      use simulations to evaluate the statistical performance of hypothesis-testing
      procedures that compare sigma2 mult for two or more groups of species on a
      phylogeny. Under both isotropic and non-isotropic conditions, and for differing
      numbers of trait dimensions, the proposed method displays appropriate Type I
      error and high statistical power for detecting known differences in sigma2 mult
      among groups. In contrast, the Type I error rate of likelihood tests based on the
      evolutionary rate matrix (R) increases as the number of trait dimensions (p)
      increases, and becomes unacceptably large when only a few trait dimensions are
      considered. Further, likelihood tests based on R cannot be computed when the
      number of trait dimensions equals or exceeds the number of taxa in the phylogeny 
      (i.e., when p &gt;/= N). These results demonstrate that tests based on sigma2 mult
      provide a useful means of comparing evolutionary rates for high-dimensional data 
      that are otherwise not analytically accessible to methods based on the
      evolutionary rate matrix. This advance thus expands the phylogenetic comparative 
      toolkit for high-dimensional phenotypic traits like shape. Finally, I illustrate 
      the utility of the new approach by evaluating rates of head shape evolution in a 
      lineage of Plethodon salamanders.
FAU - Adams, Dean C
AU  - Adams DC
AD  - Department of Ecology, Evolution, and Organismal Biology; and Department of
      Statistics, Iowa State University, Ames, IA 50011, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20131210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Phenotype
MH  - *Phylogeny
MH  - Time
MH  - Urodela/anatomy &amp; histology/*classification
OTO - NOTNLM
OT  - Evolutionary rates
OT  - geometric morphometrics
OT  - macroevolution
OT  - morphological evolution
OT  - phylogenetic comparative method
EDAT- 2013/12/18 06:00
MHDA- 2014/04/20 06:00
CRDT- 2013/12/17 06:00
PHST- 2013/12/17 06:00 [entrez]
PHST- 2013/12/18 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt105 [pii]
AID - 10.1093/sysbio/syt105 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):166-77. doi: 10.1093/sysbio/syt105. Epub 2013 Dec 10.

PMID- 24304652
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20140218
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - Introgression and phenotypic assimilation in Zimmerius flycatchers (Tyrannidae): 
      population genetic and phylogenetic inferences from genome-wide SNPs.
PG  - 134-52
LID - 10.1093/sysbio/syt070 [doi]
AB  - Genetic introgression is pervasive in nature and may lead to large-scale
      phenotypic assimilation and/or admixture of populations, but there is limited
      knowledge on whether large phenotypic changes are typically accompanied by high
      levels of introgression throughout the genome. Using bioacoustic, biometric, and 
      spectrophotometric data from a flycatcher (Tyrannidae) system in the Neotropical 
      genus Zimmerius, we document a mosaic pattern of phenotypic admixture in which a 
      population of Zimmerius viridiflavus in northern Peru (henceforth "mosaic") is
      vocally and biometrically similar to conspecifics to the south but shares plumage
      characteristics with a different species (Zimmerius chrysops) to the north. To
      clarify the origins of the mosaic population, we used the RAD-seq approach to
      generate a data set of 37,361 genome-wide single nucleotide polymorphisms (SNPs).
      A range of population-genetic diagnostics shows that the genome of the mosaic
      population is largely indistinguishable from southern Z. viridiflavus and
      distinct from northern Z. chrysops, and the application of parsimony and species 
      tree methods to the genome-wide SNP data set confirms the close affinity of the
      mosaic population with southern Z. viridiflavus. Even so, using a subset of 2710 
      SNPs found across all sampled lineages in configurations appropriate for a
      recently proposed statistical ("ABBA/BABA") test that distinguishes gene flow
      from incomplete lineage sorting, we detected low levels of gene flow from
      northern Z. chrysops into the mosaic population. Mapping the candidate loci for
      introgression from Z. chrysops into the mosaic population to the zebra finch
      genome reveals close linkage with genes significantly enriched in functions
      involving cell projection and plasma membranes. Introgression of key alleles may 
      have led to phenotypic assimilation in the plumage of mosaic birds, suggesting
      that selection may have been a key factor facilitating introgression.
FAU - Rheindt, Frank E
AU  - Rheindt FE
AD  - Department of Biological Sciences, National University of Singapore, 14 Science
      Drive 4, Singapore 117543; Department of Organismic and Evolutionary Biology,
      Harvard University, 26 Oxford Street, Cambridge, MA 02138, USA; Museum of
      Comparative Zoology, 26 Oxford Street, Cambridge, MA 02138, USA; and Department
      of Biology, University of Texas at Arlington, Arlington, TX 76019, USA.
FAU - Fujita, Matthew K
AU  - Fujita MK
FAU - Wilton, Peter R
AU  - Wilton PR
FAU - Edwards, Scott V
AU  - Edwards SV
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20131204
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Gene Flow
MH  - *Genetics, Population
MH  - Genome/*genetics
MH  - Phenotype
MH  - *Phylogeny
MH  - Polymorphism, Single Nucleotide/*genetics
MH  - Songbirds/*classification/*genetics
OTO - NOTNLM
OT  - Zimmerius
OT  - gene flow
OT  - hybridization
OT  - tyrant-flycatcher
EDAT- 2013/12/07 06:00
MHDA- 2014/04/20 06:00
CRDT- 2013/12/06 06:00
PHST- 2013/12/06 06:00 [entrez]
PHST- 2013/12/07 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt070 [pii]
AID - 10.1093/sysbio/syt070 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):134-52. doi: 10.1093/sysbio/syt070. Epub 2013 Dec 4.

PMID- 24293251
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20140218
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - Conserved genes, sampling error, and phylogenomic inference.
PG  - 257-62
LID - 10.1093/sysbio/syt073 [doi]
FAU - Betancur-R, Ricardo
AU  - Betancur-R R
AD  - Department of Biological Sciences, The George Washington University, 2023 G St.
      NW, Washington, DC 20052, USA; and College of Charleston, Hollings Marine Lab,
      331 Fort Johnson Rd., Charleston, SC 29412, USA.
FAU - Naylor, Gavin J P
AU  - Naylor GJ
FAU - Orti, Guillermo
AU  - Orti G
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20131130
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Classification
MH  - Conserved Sequence
MH  - Evolution, Molecular
MH  - Genome/*genetics
MH  - *Phylogeny
MH  - Selection Bias
MH  - Yeasts/classification/genetics
EDAT- 2013/12/03 06:00
MHDA- 2014/04/20 06:00
CRDT- 2013/12/03 06:00
PHST- 2013/12/03 06:00 [entrez]
PHST- 2013/12/03 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt073 [pii]
AID - 10.1093/sysbio/syt073 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):257-62. doi: 10.1093/sysbio/syt073. Epub 2013 Nov 30.

PMID- 24287097
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - Molecular dating, evolutionary rates, and the age of the grasses.
PG  - 153-65
LID - 10.1093/sysbio/syt072 [doi]
AB  - Many questions in evolutionary biology require an estimate of divergence times
      but, for groups with a sparse fossil record, such estimates rely heavily on
      molecular dating methods. The accuracy of these methods depends on both an
      adequate underlying model and the appropriate implementation of fossil evidence
      as calibration points. We explore the effect of these in Poaceae (grasses), a
      diverse plant lineage with a very limited fossil record, focusing particularly on
      dating the early divergences in the group. We show that molecular dating based on
      a data set of plastid markers is strongly dependent on the model assumptions. In 
      particular, an acceleration of evolutionary rates at the base of Poaceae followed
      by a deceleration in the descendants strongly biases methods that assume an
      autocorrelation of rates. This problem can be circumvented by using markers that 
      have lower rate variation, and we show that phylogenetic markers extracted from
      complete nuclear genomes can be a useful complement to the more commonly used
      plastid markers. However, estimates of divergence times remain strongly affected 
      by different implementations of fossil calibration points. Analyses calibrated
      with only macrofossils lead to estimates for the age of core Poaceae
      approximately 51-55 Ma, but the inclusion of microfossil evidence pushes this age
      to 74-82 Ma and leads to lower estimated evolutionary rates in grasses. These
      results emphasize the importance of considering markers from multiple genomes and
      alternative fossil placements when addressing evolutionary issues that depend on 
      ages estimated for important groups.
FAU - Christin, Pascal-Antoine
AU  - Christin PA
AD  - Department of Animal and Plant Sciences, University of Sheffield, Western Bank,
      Sheffield S10 2TN, UK; Department of Ecology and Evolutionary Biology, Brown
      University, 80 Waterman St., Providence, RI 02912, USA; Department of Biology &amp;
      Burke Museum of Natural History and Culture, University of Washington, 24 Kincaid
      Hall, Seattle, WA 98195, USA; Department of Ecology and Evolution, Biophore,
      Quartier Sorge, University of Lausanne, 1015 Lausanne, Switzerland; Swiss
      Institute of Bioinformatics, Quartier Sorge, University of Lausanne, 1015
      Lausanne, Switzerland.
FAU - Spriggs, Elizabeth
AU  - Spriggs E
FAU - Osborne, Colin P
AU  - Osborne CP
FAU - Stromberg, Caroline A E
AU  - Stromberg CA
FAU - Salamin, Nicolas
AU  - Salamin N
FAU - Edwards, Erika J
AU  - Edwards EJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20131128
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Genome, Plant/genetics
MH  - Magnoliopsida/classification/genetics
MH  - *Phylogeny
MH  - Plastids/*genetics
MH  - Poaceae/*classification/*genetics
MH  - Time
OTO - NOTNLM
OT  - Poaceae
OT  - divergence time
OT  - molecular dating
OT  - mutation rate
OT  - phylogeny
EDAT- 2013/11/30 06:00
MHDA- 2014/04/20 06:00
CRDT- 2013/11/30 06:00
PHST- 2013/11/30 06:00 [entrez]
PHST- 2013/11/30 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt072 [pii]
AID - 10.1093/sysbio/syt072 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):153-65. doi: 10.1093/sysbio/syt072. Epub 2013 Nov 28.

PMID- 24262383
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20140218
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - Species delimitation using Bayes factors: simulations and application to the
      Sceloporus scalaris species group (Squamata: Phrynosomatidae).
PG  - 119-33
LID - 10.1093/sysbio/syt069 [doi]
AB  - Current molecular methods of species delimitation are limited by the types of
      species delimitation models and scenarios that can be tested. Bayes factors allow
      for more flexibility in testing non-nested species delimitation models and
      hypotheses of individual assignment to alternative lineages. Here, we examined
      the efficacy of Bayes factors in delimiting species through simulations and
      empirical data from the Sceloporus scalaris species group. Marginal-likelihood
      scores of competing species delimitation models, from which Bayes factor values
      were compared, were estimated with four different methods: harmonic mean
      estimation (HME), smoothed harmonic mean estimation (sHME),
      path-sampling/thermodynamic integration (PS), and stepping-stone (SS) analysis.
      We also performed model selection using a posterior simulation-based analog of
      the Akaike information criterion through Markov chain Monte Carlo analysis
      (AICM). Bayes factor species delimitation results from the empirical data were
      then compared with results from the reversible-jump MCMC (rjMCMC)
      coalescent-based species delimitation method Bayesian Phylogenetics and
      Phylogeography (BP&amp;P). Simulation results show that HME and sHME perform poorly
      compared with PS and SS marginal-likelihood estimators when identifying the true 
      species delimitation model. Furthermore, Bayes factor delimitation (BFD) of
      species showed improved performance when species limits are tested by reassigning
      individuals between species, as opposed to either lumping or splitting lineages. 
      In the empirical data, BFD through PS and SS analyses, as well as the rjMCMC
      method, each provide support for the recognition of all scalaris group taxa as
      independent evolutionary lineages. Bayes factor species delimitation and BP&amp;P
      also support the recognition of three previously undescribed lineages. In both
      simulated and empirical data sets, harmonic and smoothed harmonic mean
      marginal-likelihood estimators provided much higher marginal-likelihood estimates
      than PS and SS estimators. The AICM displayed poor repeatability in both
      simulated and empirical data sets, and produced inconsistent model rankings
      across replicate runs with the empirical data. Our results suggest that species
      delimitation through the use of Bayes factors with marginal-likelihood estimates 
      via PS or SS analyses provide a useful and complementary alternative to existing 
      species delimitation methods.
FAU - Grummer, Jared A
AU  - Grummer JA
AD  - Department of Biology, San Diego State University, San Diego, CA 92182-4614, USA 
      and Department of Biology and Burke Museum of Natural History and Culture,
      University of Washington, Box 351800, Seattle, WA 98195-1800, USA.
FAU - Bryson, Robert W Jr
AU  - Bryson RW Jr
FAU - Reeder, Tod W
AU  - Reeder TW
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20131120
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Reptilian Proteins)
SB  - IM
MH  - Animals
MH  - *Bayes Theorem
MH  - *Computer Simulation
MH  - Iguanas/anatomy &amp; histology/*classification/genetics
MH  - Markov Chains
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Reptilian Proteins/genetics
MH  - Species Specificity
OTO - NOTNLM
OT  - *BEAST
OT  - BP&amp;P
OT  - Bayesian
OT  - Mexico
OT  - incomplete lineage sorting
OT  - marginal-likelihood estimation
OT  - model choice
EDAT- 2013/11/23 06:00
MHDA- 2014/04/20 06:00
CRDT- 2013/11/23 06:00
PHST- 2013/11/23 06:00 [entrez]
PHST- 2013/11/23 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt069 [pii]
AID - 10.1093/sysbio/syt069 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):119-33. doi: 10.1093/sysbio/syt069. Epub 2013 Nov 20.

PMID- 24193892
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - Posterior predictive Bayesian phylogenetic model selection.
PG  - 309-21
LID - 10.1093/sysbio/syt068 [doi]
AB  - We present two distinctly different posterior predictive approaches to Bayesian
      phylogenetic model selection and illustrate these methods using examples from
      green algal protein-coding cpDNA sequences and flowering plant rDNA sequences.
      The Gelfand-Ghosh (GG) approach allows dissection of an overall measure of model 
      fit into components due to posterior predictive variance (GGp) and
      goodness-of-fit (GGg), which distinguishes this method from the posterior
      predictive P-value approach. The conditional predictive ordinate (CPO) method
      provides a site-specific measure of model fit useful for exploratory analyses and
      can be combined over sites yielding the log pseudomarginal likelihood (LPML)
      which is useful as an overall measure of model fit. CPO provides a useful
      cross-validation approach that is computationally efficient, requiring only a
      sample from the posterior distribution (no additional simulation is required).
      Both GG and CPO add new perspectives to Bayesian phylogenetic model selection
      based on the predictive abilities of models and complement the perspective
      provided by the marginal likelihood (including Bayes Factor comparisons) based
      solely on the fit of competing models to observed data.
FAU - Lewis, Paul O
AU  - Lewis PO
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, CT 06269, USA; AbbVie, 1 N. Waukegan Road,
      R436/AP9A-2, North Chicago, IL 60064, USA; Department of Statistics, University
      of Connecticut, 215 Glenbrook Road, Unit 4120, Storrs, CT 06269, USA; and
      Department of Bioinformatics and Computational Biology, Division of Quantitative 
      Sciences, The University of Texas M.D. Anderson Cancer Center, 1515 Holcombe
      Boulevard, Houston, TX 77030, USA.
FAU - Xie, Wangang
AU  - Xie W
FAU - Chen, Ming-Hui
AU  - Chen MH
FAU - Fan, Yu
AU  - Fan Y
FAU - Kuo, Lynn
AU  - Kuo L
LA  - eng
GR  - R01 GM070335/GM/NIGMS NIH HHS/United States
GR  - CA 74015/CA/NCI NIH HHS/United States
GR  - GM 70335/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20131104
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Chlorophyta/genetics
MH  - Classification/*methods
MH  - DNA, Plant/genetics
MH  - Magnoliopsida/genetics
MH  - *Models, Biological
MH  - *Phylogeny
PMC - PMC3985471
EDAT- 2013/11/07 06:00
MHDA- 2014/06/15 06:00
CRDT- 2013/11/07 06:00
PHST- 2013/11/07 06:00 [entrez]
PHST- 2013/11/07 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syt068 [pii]
AID - 10.1093/sysbio/syt068 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):309-21. doi: 10.1093/sysbio/syt068. Epub 2013 Nov 4.

PMID- 24178942
OWN - NLM
STAT- MEDLINE
DCOM- 20140418
LR  - 20140218
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 2
DP  - 2014 Mar
TI  - 50 years of inordinate fondness.
PG  - 251-6
LID - 10.1093/sysbio/syt067 [doi]
FAU - Bokma, Folmer
AU  - Bokma F
AD  - IceLab and Department of Ecology &amp; Environmental Science, Linnaeus vag 6, 92277, 
      Umea University, Sweden; and IceLab and Department of Physics, Linnaeus vag 24,
      Umea University, 90187 Umea, Sweden.
FAU - Baek, Seung Ki
AU  - Baek SK
FAU - Minnhagen, Petter
AU  - Minnhagen P
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20131030
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biodiversity
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2013/11/02 06:00
MHDA- 2014/04/20 06:00
CRDT- 2013/11/02 06:00
PHST- 2013/11/02 06:00 [entrez]
PHST- 2013/11/02 06:00 [pubmed]
PHST- 2014/04/20 06:00 [medline]
AID - syt067 [pii]
AID - 10.1093/sysbio/syt067 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Mar;63(2):251-6. doi: 10.1093/sysbio/syt067. Epub 2013 Oct 30.

PMID- 24149077
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20140415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - Robust regression and posterior predictive simulation increase power to detect
      early bursts of trait evolution.
PG  - 293-308
LID - 10.1093/sysbio/syt066 [doi]
AB  - A central prediction of much theory on adaptive radiations is that traits should 
      evolve rapidly during the early stages of a clade's history and subsequently
      slowdown in rate as niches become saturated--a so-called "Early Burst." Although 
      a common pattern in the fossil record, evidence for early bursts of trait
      evolution in phylogenetic comparative data has been equivocal at best. We show
      here that this may not necessarily be due to the absence of this pattern in
      nature. Rather, commonly used methods to infer its presence perform poorly when
      when the strength of the burst--the rate at which phenotypic evolution
      declines--is small, and when some morphological convergence is present within the
      clade. We present two modifications to existing comparative methods that allow
      greater power to detect early bursts in simulated datasets. First, we develop
      posterior predictive simulation approaches and show that they outperform maximum 
      likelihood approaches at identifying early bursts at moderate strength. Second,
      we use a robust regression procedure that allows for the identification and
      down-weighting of convergent taxa, leading to moderate increases in method
      performance. We demonstrate the utility and power of these approach by
      investigating the evolution of body size in cetaceans. Model fitting using
      maximum likelihood is equivocal with regards the mode of cetacean body size
      evolution. However, posterior predictive simulation combined with a robust node
      height test return low support for Brownian motion or rate shift models, but not 
      the early burst model. While the jury is still out on whether early bursts are
      actually common in nature, our approach will hopefully facilitate more robust
      testing of this hypothesis. We advocate the adoption of similar posterior
      predictive approaches to improve the fit and to assess the adequacy of
      macroevolutionary models in general.
FAU - Slater, Graham J
AU  - Slater GJ
AD  - Department of Ecology and Evolutionary Biology, University of California Los
      Angeles, 610 Charles E. Young Drive East, Los Angeles, CA, 90095-7239, USA;
      Department of Paleobiology &amp; Division of Mammals, National Museum of Natural
      History, Smithsonian Institution, MRC 121, PO Box 37012, Washington, DC.,
      20013-7012, USA; Institute for Bioinformatics and Evolutionary Studies,
      University of Idaho, 441D Life Sciences South, PO Box 443051, Moscow, ID,
      83844-3051, USA; and National Evolutionary Synthesis Center, 2024 W. Main Street,
      Suite A200, Durham, NC, 27705-4667, USA.
FAU - Pennell, Matthew W
AU  - Pennell MW
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20131022
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - Computer Simulation/*standards
MH  - *Models, Theoretical
MH  - Reproducibility of Results
EDAT- 2013/10/24 06:00
MHDA- 2014/06/15 06:00
CRDT- 2013/10/24 06:00
PHST- 2013/10/24 06:00 [entrez]
PHST- 2013/10/24 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syt066 [pii]
AID - 10.1093/sysbio/syt066 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):293-308. doi: 10.1093/sysbio/syt066. Epub 2013 Oct 22.

PMID- 24149076
OWN - NLM
STAT- MEDLINE
DCOM- 20140313
LR  - 20131219
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 1
DP  - 2014 Jan 1
TI  - Timeframes of speciation, reticulation, and hybridization in the bulldog bat
      explained through phylogenetic analyses of all genetic transmission elements.
PG  - 96-110
LID - 10.1093/sysbio/syt062 [doi]
AB  - Phylogenetic comparisons of the different mammalian genetic transmission elements
      (mtDNA, X-, Y-, and autosomal DNA) is a powerful approach for understanding the
      process of speciation in nature. Through such comparisons the unique inheritance 
      pathways of each genetic element and gender-biased processes can link genomic
      structure to the evolutionary process, especially among lineages which have
      recently diversified, in which genetic isolation may be incomplete. Bulldog bats 
      of the genus Noctilio are an exemplar lineage, being a young clade, widely
      distributed, and exhibiting unique feeding ecologies. In addition, currently
      recognized species are paraphyletic with respect to the mtDNA gene tree and
      contain morphologically identifiable clades that exhibit mtDNA divergences as
      great as among many species. To test taxonomic hypotheses and understand the
      contribution of hybridization to the extant distribution of genetic diversity in 
      Noctilio, we used phylogenetic, coalescent stochastic modeling, and divergence
      time estimates using sequence data from cytochrome-b, cytochrome c oxidase-I,
      zinc finger Y, and zinc finger X, as well as evolutionary reconstructions based
      on amplified fragment length polymorphisms (AFLPs) data. No evidence of ongoing
      hybridization between the two currently recognized species was identified.
      However, signatures of an ancient mtDNA capture were recovered in which an mtDNA 
      lineage of one species was captured early in the noctilionid radiation. Among
      subspecific mtDNA clades, which were generally coincident with morphology and
      statistically definable as species, signatures of ongoing hybridization were
      observed in sex chromosome sequences and AFLP. Divergence dating of genetic
      elements corroborates the diversification of extant Noctilio beginning about 3
      Ma, with ongoing hybridization between mitochondrial lineages separated by 2.5
      myr. The timeframe of species' divergence within Noctilio supports the hypothesis
      that shifts in the dietary strategies of gleaning insects (N. albiventris) or
      fish (N. leporinus) are among the most rapid instances of dietary evolution
      observed in mammals. This study illustrates the complex evolutionary dynamics
      shaping gene pools in nature, how comparisons of genetic elements can serve for
      understanding species boundaries, and the complex considerations for accurate
      taxonomic assignment.
FAU - Khan, Faisal Ali Anwarali
AU  - Khan FA
AD  - Department of Biological Sciences and the Museum, Texas Tech University, Lubbock,
      TX 79409, USA and Department of Zoology, Faculty of Resource Science and
      Technology, Universiti Malaysia Sarawak, Kota Samarahan, Sarawak 94300, Malaysia.
FAU - Phillips, Caleb D
AU  - Phillips CD
FAU - Baker, Robert J
AU  - Baker RJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20131022
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
SB  - IM
MH  - Animals
MH  - Chiroptera/*classification/*genetics
MH  - Female
MH  - Genes, Mitochondrial/genetics
MH  - Genetic Markers/genetics
MH  - *Genetic Speciation
MH  - Genetic Variation
MH  - *Hybridization, Genetic
MH  - Male
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Sex Chromosomes/genetics
MH  - Time
EDAT- 2013/10/24 06:00
MHDA- 2014/03/14 06:00
CRDT- 2013/10/24 06:00
PHST- 2013/10/24 06:00 [entrez]
PHST- 2013/10/24 06:00 [pubmed]
PHST- 2014/03/14 06:00 [medline]
AID - syt062 [pii]
AID - 10.1093/sysbio/syt062 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jan 1;63(1):96-110. doi: 10.1093/sysbio/syt062. Epub 2013 Oct 22.

PMID- 24021724
OWN - NLM
STAT- MEDLINE
DCOM- 20140313
LR  - 20131219
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 1
DP  - 2014 Jan 1
TI  - Target capture and massively parallel sequencing of ultraconserved elements for
      comparative studies at shallow evolutionary time scales.
PG  - 83-95
LID - 10.1093/sysbio/syt061 [doi]
AB  - Comparative genetic studies of non-model organisms are transforming rapidly due
      to major advances in sequencing technology. A limiting factor in these studies
      has been the identification and screening of orthologous loci across an
      evolutionarily distant set of taxa. Here, we evaluate the efficacy of genomic
      markers targeting ultraconserved DNA elements (UCEs) for analyses at shallow
      evolutionary timescales. Using sequence capture and massively parallel sequencing
      to generate UCE data for five co-distributed Neotropical rainforest bird species,
      we recovered 776-1516 UCE loci across the five species. Across species, 53-77% of
      the loci were polymorphic, containing between 2.0 and 3.2 variable sites per
      polymorphic locus, on average. We performed species tree construction, coalescent
      modeling, and species delimitation, and we found that the five co-distributed
      species exhibited discordant phylogeographic histories. We also found that
      species trees and divergence times estimated from UCEs were similar to the
      parameters obtained from mtDNA. The species that inhabit the understory had older
      divergence times across barriers, contained a higher number of cryptic species,
      and exhibited larger effective population sizes relative to the species
      inhabiting the canopy. Because orthologous UCEs can be obtained from a wide array
      of taxa, are polymorphic at shallow evolutionary timescales, and can be generated
      rapidly at low cost, they are an effective genetic marker for studies
      investigating evolutionary patterns and processes at shallow timescales.
FAU - Smith, Brian Tilston
AU  - Smith BT
AD  - Museum of Natural Science, Louisiana State University, Baton Rouge, LA 70803,
      USA; Department of Biological Sciences, Louisiana State University, Baton Rouge, 
      LA 70803, USA; Department of Ecology and Evolutionary Biology, University of
      California, Los Angeles, CA 90095, USA; and Department of Environmental Health
      Science, University of Georgia, Athens, GA 30602, USA.
FAU - Harvey, Michael G
AU  - Harvey MG
FAU - Faircloth, Brant C
AU  - Faircloth BC
FAU - Glenn, Travis C
AU  - Glenn TC
FAU - Brumfield, Robb T
AU  - Brumfield RT
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130910
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animal Migration
MH  - Animals
MH  - Birds/*classification/*genetics
MH  - Conserved Sequence/*genetics
MH  - *Evolution, Molecular
MH  - *Genetic Variation
MH  - *High-Throughput Nucleotide Sequencing
MH  - Phylogeny
MH  - Population Density
MH  - Sequence Analysis, DNA
EDAT- 2013/09/12 06:00
MHDA- 2014/03/14 06:00
CRDT- 2013/09/12 06:00
PHST- 2013/09/12 06:00 [entrez]
PHST- 2013/09/12 06:00 [pubmed]
PHST- 2014/03/14 06:00 [medline]
AID - syt061 [pii]
AID - 10.1093/sysbio/syt061 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jan 1;63(1):83-95. doi: 10.1093/sysbio/syt061. Epub 2013 Sep 10.

PMID- 23988674
OWN - NLM
STAT- MEDLINE
DCOM- 20140313
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 1
DP  - 2014 Jan 1
TI  - Robustness to divergence time underestimation when inferring species trees from
      estimated gene trees.
PG  - 66-82
LID - 10.1093/sysbio/syt059 [doi]
AB  - To infer species trees from gene trees estimated from phylogenomic data sets,
      tractable methods are needed that can handle dozens to hundreds of loci. We
      examine several computationally efficient approaches-MP-EST, STAR, STEAC, STELLS,
      and STEM-for inferring species trees from gene trees estimated using maximum
      likelihood (ML) and Bayesian approaches. Among the methods examined, we found
      that topology-based methods often performed better using ML gene trees and
      methods employing coalescent times typically performed better using Bayesian gene
      trees, with MP-EST, STAR, STEAC, and STELLS outperforming STEM under most
      conditions. We examine why the STEM tree (also called GLASS or Maximum Tree) is
      less accurate on estimated gene trees by comparing estimated and true coalescence
      times, performing species tree inference using simulations, and analyzing a great
      ape data set keeping track of false positive and false negative rates for
      inferred clades. We find that although true coalescence times are more ancient
      than speciation times under the multispecies coalescent model, estimated
      coalescence times are often more recent than speciation times. This
      underestimation can lead to increased bias and lack of resolution with increased 
      sampling (either alleles or loci) when gene trees are estimated with ML. The
      problem appears to be less severe using Bayesian gene-tree estimates.
FAU - DeGiorgio, Michael
AU  - DeGiorgio M
AD  - Department of Integrative Biology, University of California, Berkeley, CA 94720, 
      USA; Department of Biology, Pennsylvania State University, University Park, PA
      16802, USA; and Department of Mathematics and Statistics, University of New
      Mexico, 1 University of New Mexico, Albuquerque, NM 87131, USA.
FAU - Degnan, James H
AU  - Degnan JH
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130829
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bias
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Genetic Speciation
MH  - *Phylogeny
MH  - Primates/classification/genetics
MH  - Time
EDAT- 2013/08/31 06:00
MHDA- 2014/03/14 06:00
CRDT- 2013/08/31 06:00
PHST- 2013/08/31 06:00 [entrez]
PHST- 2013/08/31 06:00 [pubmed]
PHST- 2014/03/14 06:00 [medline]
AID - syt059 [pii]
AID - 10.1093/sysbio/syt059 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jan 1;63(1):66-82. doi: 10.1093/sysbio/syt059. Epub 2013 Aug 29.

PMID- 23985785
OWN - NLM
STAT- MEDLINE
DCOM- 20140613
LR  - 20140415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 3
DP  - 2014 May
TI  - Poor fit to the multispecies coalescent is widely detectable in empirical data.
PG  - 322-33
LID - 10.1093/sysbio/syt057 [doi]
AB  - Model checking is a critical part of Bayesian data analysis, yet it remains
      largely unused in systematic studies. Phylogeny estimation has recently moved
      into an era of increasingly complex models that simultaneously account for
      multiple evolutionary processes, the statistical fit of these models to the data 
      has rarely been tested. Here we develop a posterior predictive simulation-based
      model check for a commonly used multispecies coalescent model, implemented in
      *BEAST, and apply it to 25 published data sets. We show that poor model fit is
      detectable in the majority of data sets; that this poor fit can mislead
      phylogenetic estimation; and that in some cases it stems from processes of
      inherent interest to systematists. We suggest that as systematists scale up to
      phylogenomic data sets, which will be subject to a heterogeneous array of
      evolutionary processes, critically evaluating the fit of models to data is an
      analytical step that can no longer be ignored.
FAU - Reid, Noah M
AU  - Reid NM
AD  - Department of Biological Sciences, Louisiana State University, Baton Rouge, LA
      70803, USA; and Department of Evolution, Ecology &amp; Organismal Biology, Ohio State
      University, Columbus, OH 43210, USA.
FAU - Hird, Sarah M
AU  - Hird SM
FAU - Brown, Jeremy M
AU  - Brown JM
FAU - Pelletier, Tara A
AU  - Pelletier TA
FAU - McVay, John D
AU  - McVay JD
FAU - Satler, Jordan D
AU  - Satler JD
FAU - Carstens, Bryan C
AU  - Carstens BC
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130828
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Classification
MH  - Computer Simulation/*standards
MH  - Eukaryota/classification/genetics
MH  - *Models, Biological
MH  - Phylogeny
EDAT- 2013/08/30 06:00
MHDA- 2014/06/15 06:00
CRDT- 2013/08/30 06:00
PHST- 2013/08/30 06:00 [entrez]
PHST- 2013/08/30 06:00 [pubmed]
PHST- 2014/06/15 06:00 [medline]
AID - syt057 [pii]
AID - 10.1093/sysbio/syt057 [doi]
PST - ppublish
SO  - Syst Biol. 2014 May;63(3):322-33. doi: 10.1093/sysbio/syt057. Epub 2013 Aug 28.

PMID- 23985784
OWN - NLM
STAT- MEDLINE
DCOM- 20140313
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 1
DP  - 2014 Jan 1
TI  - Island biogeography, the effects of taxonomic effort and the importance of island
      niche diversity to single-island endemic species.
PG  - 55-65
LID - 10.1093/sysbio/syt060 [doi]
AB  - Island biogeography theory is fundamentally reliant on measuring the number of
      species per island and hence has taxonomy at its foundation. Yet as a metric used
      in tests of the theory, island species richness (S) has varied with time
      according to the level of taxonomic effort (a function of the rate of finding and
      describing species). Studies using a derivative of S, single-island endemic
      species richness (SIE S), may be prone to change in taxonomic effort. Decreases
      or increases in species numbers resulting from taxonomic revision or increased
      sampling are likely to have a large effect on values of SIE S, as they tend to be
      smaller than total S for the same island. Using simple biogeography models, we
      analysed estimates of SIE S in plants, land snails, beetles, and fungi from
      comprehensive data sets for eight island groups, produced species accumulation
      curves and applied Bayesian regression over five time periods. Explanatory power 
      differed across taxa, but area and island age were not always the best
      explanatory variables, and niche diversity appeared to be important. Changing
      levels of SIE S over time had different effects on models with different taxa and
      between island archipelagos. The results indicated that the taxonomic effort that
      determines SIE S is important. However, as this cannot often be quantified, we
      suggest Bayesian approaches should be more useful than frequentist methods in
      evaluating SIE S in island biogeography theory. Fundamentally, the article
      highlights the importance of taxonomy to theoretical biogeography.
FAU - Gray, Alan
AU  - Gray A
AD  - Centre for Ecology and Hydrology, Bush Estate, Penicuik, Midlothian, EH26 0QB,
      UK.
FAU - Cavers, Stephen
AU  - Cavers S
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130828
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - Coleoptera/classification/genetics
MH  - Ecosystem
MH  - Fungi/classification/genetics
MH  - Genetic Speciation
MH  - Islands
MH  - *Models, Biological
MH  - *Phylogeography
MH  - Plants/classification/genetics
MH  - Snails/classification/genetics
EDAT- 2013/08/30 06:00
MHDA- 2014/03/14 06:00
CRDT- 2013/08/30 06:00
PHST- 2013/08/30 06:00 [entrez]
PHST- 2013/08/30 06:00 [pubmed]
PHST- 2014/03/14 06:00 [medline]
AID - syt060 [pii]
AID - 10.1093/sysbio/syt060 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jan 1;63(1):55-65. doi: 10.1093/sysbio/syt060. Epub 2013 Aug 28.

PMID- 23963353
OWN - NLM
STAT- MEDLINE
DCOM- 20140313
LR  - 20131219
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 1
DP  - 2014 Jan 1
TI  - Accelerated rate of molecular evolution for vittarioid ferns is strong and not
      driven by selection.
PG  - 31-54
LID - 10.1093/sysbio/syt058 [doi]
AB  - Molecular evolutionary rate heterogeneity-the violation of a molecular clock-is a
      prominent feature of many phylogenetic data sets. It has particular importance to
      systematists not only because of its biological implications, but also for its
      practical effects on our ability to infer and date evolutionary events. Here we
      show, using both maximum likelihood and Bayesian approaches, that a remarkably
      strong increase in substitution rate in the vittarioid ferns is consistent across
      the nuclear and plastid genomes. Contrary to some expectations, this rate
      increase is not due to selective forces acting at the protein level on our focal 
      loci. The vittarioids bear no signature of the change in the relative strengths
      of selection and drift that one would expect if the rate increase was caused by
      altered post-mutation fixation rates. Instead, the substitution rate increase
      appears to stem from an elevated supply of mutations, perhaps limited to the
      vittarioid ancestral branch. This generalized rate increase is accompanied by
      extensive fine-scale heterogeneity in rates across loci, genomes, and taxa. Our
      analyses demonstrate the effectiveness and flexibility of trait-free
      investigations of rate heterogeneity within a model-selection framework,
      emphasize the importance of explicit tests for signatures of selection prior to
      invoking selection-related or demography-based explanations for patterns of rate 
      variation, and illustrate some unexpected nuances in the behavior of relaxed
      clock methods for modeling rate heterogeneity, with implications for our ability 
      to confidently date divergence events. In addition, our data provide strong
      support for the monophyly of Adiantum, and for the position of Calciphilopteris
      in the cheilanthoid ferns, two relationships for which convincing support was
      previously lacking.
FAU - Rothfels, Carl J
AU  - Rothfels CJ
AD  - Department of Biology, Duke University, Box 90338, Durham, NC 27708, USA;
      Department of Zoology, University of British Columbia, #4200-6270 University
      Blvd., Vancouver, BC V6T 1Z4, Canada; Department of Biology and Marine Biology,
      University of North Carolina Wilmington, 601 South College Road, Wilmington, NC
      28403, USA; and Department of Botany (MRC 166), National Museum of Natural
      History, Smithsonian Institution, PO Box 37012, Washington DC 20013-7012, USA.
FAU - Schuettpelz, Eric
AU  - Schuettpelz E
LA  - eng
SI  - GENBANK/EF452011
SI  - GENBANK/EF452017
SI  - GENBANK/EF452018
SI  - GENBANK/EF452022
SI  - GENBANK/EF452031
SI  - GENBANK/EF452035
SI  - GENBANK/EF452036
SI  - GENBANK/EF452037
SI  - GENBANK/EF452049
SI  - GENBANK/EF452050
SI  - GENBANK/EF452068
SI  - GENBANK/EF452076
SI  - GENBANK/EF452102
SI  - GENBANK/EF452126
SI  - GENBANK/EF452128
SI  - GENBANK/EF452132
SI  - GENBANK/EF452133
SI  - GENBANK/EF452134
SI  - GENBANK/EF452135
SI  - GENBANK/EF452138
SI  - GENBANK/EF452142
SI  - GENBANK/EF452146
SI  - GENBANK/EF452148
SI  - GENBANK/EF452150
SI  - GENBANK/EF452153
SI  - GENBANK/EF452156
SI  - GENBANK/EF452157
SI  - GENBANK/EF452162
SI  - GENBANK/EF452165
SI  - GENBANK/EF452166
SI  - GENBANK/EF452176
SI  - GENBANK/EU268733
SI  - GENBANK/EU268740
SI  - GENBANK/EU268741
SI  - GENBANK/EU268749
SI  - GENBANK/EU268768
SI  - GENBANK/EU268782
SI  - GENBANK/EU268794
SI  - GENBANK/JF832173
SI  - GENBANK/JQ855925
SI  - GENBANK/KC984409
SI  - GENBANK/KC984410
SI  - GENBANK/KC984411
SI  - GENBANK/KC984412
SI  - GENBANK/KC984413
SI  - GENBANK/KC984414
SI  - GENBANK/KC984415
SI  - GENBANK/KC984416
SI  - GENBANK/KC984417
SI  - GENBANK/KC984418
SI  - GENBANK/KC984419
SI  - GENBANK/KC984420
SI  - GENBANK/KC984421
SI  - GENBANK/KC984422
SI  - GENBANK/KC984423
SI  - GENBANK/KC984424
SI  - GENBANK/KC984425
SI  - GENBANK/KC984426
SI  - GENBANK/KC984427
SI  - GENBANK/KC984428
SI  - GENBANK/KC984429
SI  - GENBANK/KC984430
SI  - GENBANK/KC984431
SI  - GENBANK/KC984432
SI  - GENBANK/KC984433
SI  - GENBANK/KC984434
SI  - GENBANK/KC984435
SI  - GENBANK/KC984436
SI  - GENBANK/KC984437
SI  - GENBANK/KC984438
SI  - GENBANK/KC984439
SI  - GENBANK/KC984440
SI  - GENBANK/KC984441
SI  - GENBANK/KC984442
SI  - GENBANK/KC984443
SI  - GENBANK/KC984444
SI  - GENBANK/KC984450
SI  - GENBANK/KC984453
SI  - GENBANK/KC984455
SI  - GENBANK/KC984456
SI  - GENBANK/KC984458
SI  - GENBANK/KC984459
SI  - GENBANK/KC984460
SI  - GENBANK/KC984461
SI  - GENBANK/KC984462
SI  - GENBANK/KC984463
SI  - GENBANK/KC984464
SI  - GENBANK/KC984465
SI  - GENBANK/KC984466
SI  - GENBANK/KC984467
SI  - GENBANK/KC984468
SI  - GENBANK/KC984469
SI  - GENBANK/KC984470
SI  - GENBANK/KC984471
SI  - GENBANK/KC984472
SI  - GENBANK/KC984473
SI  - GENBANK/KC984493
SI  - GENBANK/KC984494
SI  - GENBANK/KC984495
SI  - GENBANK/KC984496
SI  - GENBANK/KC984497
SI  - GENBANK/KC984498
SI  - GENBANK/KC984499
SI  - GENBANK/KC984500
SI  - GENBANK/KC984501
SI  - GENBANK/KC984502
SI  - GENBANK/KC984503
SI  - GENBANK/KC984504
SI  - GENBANK/KC984505
SI  - GENBANK/KC984506
SI  - GENBANK/KC984507
SI  - GENBANK/KC984508
SI  - GENBANK/KC984509
SI  - GENBANK/KC984510
SI  - GENBANK/KC984511
SI  - GENBANK/KC984512
SI  - GENBANK/KC984513
SI  - GENBANK/KC984514
SI  - GENBANK/KC984515
SI  - GENBANK/KC984516
SI  - GENBANK/KC984517
SI  - GENBANK/KC984518
SI  - GENBANK/KC984519
SI  - GENBANK/KC984520
SI  - GENBANK/KC984521
SI  - GENBANK/KC984522
SI  - GENBANK/KC984523
SI  - GENBANK/KC984524
SI  - GENBANK/KC984525
PT  - Journal Article
DEP - 20130820
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - *Evolution, Molecular
MH  - Ferns/*classification/*genetics
MH  - Models, Biological
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Selection, Genetic
EDAT- 2013/08/22 06:00
MHDA- 2014/03/14 06:00
CRDT- 2013/08/22 06:00
PHST- 2013/08/22 06:00 [entrez]
PHST- 2013/08/22 06:00 [pubmed]
PHST- 2014/03/14 06:00 [medline]
AID - syt058 [pii]
AID - 10.1093/sysbio/syt058 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jan 1;63(1):31-54. doi: 10.1093/sysbio/syt058. Epub 2013 Aug 20.

PMID- 23945075
OWN - NLM
STAT- MEDLINE
DCOM- 20140313
LR  - 20131219
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 1
DP  - 2014 Jan 1
TI  - The influence of gene flow on species tree estimation: a simulation study.
PG  - 17-30
LID - 10.1093/sysbio/syt049 [doi]
AB  - Gene flow among populations or species and incomplete lineage sorting (ILS) are
      two evolutionary processes responsible for generating gene tree discordance and
      therefore hindering species tree estimation. Numerous studies have evaluated the 
      impacts of ILS on species tree inference, yet the ramifications of gene flow on
      species trees remain less studied. Here, we simulate and analyse multilocus
      sequence data generated with ILS and gene flow to quantify their impacts on
      species tree inference. We characterize species tree estimation errors under
      various models of gene flow, such as the isolation-migration model, the n-island 
      model, and gene flow between non-sister species or involving ancestral species,
      and species boundaries crossed by a single gene copy (allelic introgression) or
      by a single migrant individual. These patterns of gene flow are explored on
      species trees of different sizes (4 vs. 10 species), at different time scales
      (shallow vs. deep), and with different migration rates. Species trees are
      estimated with the multispecies coalescent model using Bayesian methods (BEST and
      *BEAST) and with a summary statistic approach (MPEST) that facilitates
      phylogenomic-scale analysis. Even in cases where the topology of the species tree
      is estimated with high accuracy, we find that gene flow can result in
      overestimates of population sizes (species tree dilation) and underestimates of
      species divergence times (species tree compression). Signatures of migration
      events remain present in the distribution of coalescent times for gene trees, and
      with sufficient data it is possible to identify those loci that have crossed
      species boundaries. These results highlight the need for careful sampling design 
      in phylogeographic and species delimitation studies as gene flow, introgression, 
      or incorrect sample assignments can bias the estimation of the species tree
      topology and of parameter estimates such as population sizes and divergence
      times.
FAU - Leache, Adam D
AU  - Leache AD
AD  - Department of Biology and Burke Museum of Natural History and Culture, University
      of Washington, Seattle, WA 98195 USA;
FAU - Harris, Rebecca B
AU  - Harris RB
FAU - Rannala, Bruce
AU  - Rannala B
FAU - Yang, Ziheng
AU  - Yang Z
LA  - eng
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130813
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Alleles
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - *Gene Flow
MH  - Genetic Speciation
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2013/08/16 06:00
MHDA- 2014/03/14 06:00
CRDT- 2013/08/16 06:00
PHST- 2013/08/16 06:00 [entrez]
PHST- 2013/08/16 06:00 [pubmed]
PHST- 2014/03/14 06:00 [medline]
AID - syt049 [pii]
AID - 10.1093/sysbio/syt049 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jan 1;63(1):17-30. doi: 10.1093/sysbio/syt049. Epub 2013 Aug 13.

PMID- 23939193
OWN - NLM
STAT- MEDLINE
DCOM- 20140313
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 63
IP  - 1
DP  - 2014 Jan 1
TI  - Constructing phylogenies in the presence of intra-individual site polymorphisms
      (2ISPs) with a focus on the nuclear ribosomal cistron.
PG  - 1-16
LID - 10.1093/sysbio/syt052 [doi]
AB  - Nuclear DNA is widely used to estimate phylogenetic and phylogeographic
      relationships. Nuclear gene variants may be present in an individual's genome,
      and these result in Intra-Individual Site Polymorphisms (2ISP; pronounced
      "twisp") in direct-PCR or individual-consensus sequences based on a sample of
      clones or fragment sequences from next generation sequencing (NGS). 2ISPs can
      occur fairly often, especially within, but not restricted to, high-copy-number
      regions such as the widely used internal transcribed spacers of the nuclear
      ribosomal cistron. Dealing with 2ISPs has been problematic as phylogeny
      reconstruction optimality criteria generally do not take account of this
      variation. Here we test whether an approach that treats 2ISPs as additional
      (termed "informative"), rather than ambiguous, characters offers improved support
      in three common criteria used for phylogenetic inference: Minimum Evolution (via 
      Neighbour Joining), Maximum Parsimony, and Maximum Likelihood. We demonstrate
      significant improvements using the 2ISP-informative treatment with simulated,
      real-world, and case-study data sets. We envisage that this 2ISP-informative
      approach will greatly aid phylogenetic inference using any nuclear DNA regions
      that contain polymorphisms within individuals (including consensus sequences
      generated from NGS), especially at the intrageneric or intraspecific level.
FAU - Potts, Alastair J
AU  - Potts AJ
AD  - Bolus Herbarium, Department of Biological Sciences, University of Cape Town, Cape
      Town, Western Cape, 7700, South Africa; and Department of Palaeobiology, Swedish 
      Museum of Natural History, P.O. Box 50007, 104 05 Stockholm, Sweden.
FAU - Hedderson, Terry A
AU  - Hedderson TA
FAU - Grimm, Guido W
AU  - Grimm GW
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130811
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - DNA, Ribosomal/genetics
MH  - Genetic Variation
MH  - Magnoliopsida/classification/genetics
MH  - *Phylogeny
MH  - Polymorphism, Genetic
EDAT- 2013/08/14 06:00
MHDA- 2014/03/14 06:00
CRDT- 2013/08/14 06:00
PHST- 2013/08/14 06:00 [entrez]
PHST- 2013/08/14 06:00 [pubmed]
PHST- 2014/03/14 06:00 [medline]
AID - syt052 [pii]
AID - 10.1093/sysbio/syt052 [doi]
PST - ppublish
SO  - Syst Biol. 2014 Jan 1;63(1):1-16. doi: 10.1093/sysbio/syt052. Epub 2013 Aug 11.

PMID- 23929779
OWN - NLM
STAT- MEDLINE
DCOM- 20140312
LR  - 20131017
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 6
DP  - 2013 Nov
TI  - Recent diversification of a marine genus (Tursiops spp.) tracks habitat
      preference and environmental change.
PG  - 865-77
LID - 10.1093/sysbio/syt051 [doi]
AB  - Understanding the evolution of diversity and the resulting systematics in marine 
      systems is confounded by the lack of clear boundaries in oceanic habitats,
      especially for highly mobile species like marine mammals. Dolphin populations and
      sibling species often show differentiation between coastal and offshore habitats,
      similar to the pelagic/littoral or benthic differentiation seen for some species 
      of fish. Here we test the hypothesis that lineages within the polytypic genus
      Tursiops track past changes in the environment reflecting ecological drivers of
      evolution facilitated by habitat release. We used a known recent time point for
      calibration (the opening of the Bosphorus) and whole mitochondrial genome
      (mitogenome) sequences for high phylogenetic resolution. The pattern of lineage
      formation suggested an origin in Australasia and several early divisions
      involving forms currently inhabiting coastal habitats. Radiation in pelagic
      environments was relatively recent, and was likely followed by a return to
      coastal habitat in some regions. The timing of some nodes defining different
      ecotypes within the genus clustered near the two most recent interglacial
      transitions. A signal for an increase in diversification was also seen for dates 
      after the last glacial maximum. Together these data suggest the tracking of
      habitat preference during geographic expansions, followed by transition points
      reflecting habitat shifts, which were likely associated with periods of
      environmental change.
FAU - Moura, Andre E
AU  - Moura AE
AD  - School of Biological and Biomedical Sciences, Durham University, South Road, DH1 
      3LE, UK; Centre for GeoGenetics, Natural History Museum of Denmark, University of
      Copenhagen, Oster Voldgade 5-7, DK-1350 Copenhagen K, Denmark; Biological
      Department, UAE University, P.O.Box 17551, Al Ain, UAE; and School of Biological 
      Sciences, Flinders University, GPO Box 2100, Adelaide, South Australia 5001,
      Australia.
FAU - Nielsen, Sandra C A
AU  - Nielsen SC
FAU - Vilstrup, Julia T
AU  - Vilstrup JT
FAU - Moreno-Mayar, J Victor
AU  - Moreno-Mayar JV
FAU - Gilbert, M Thomas P
AU  - Gilbert MT
FAU - Gray, Howard W I
AU  - Gray HW
FAU - Natoli, Ada
AU  - Natoli A
FAU - Moller, Luciana
AU  - Moller L
FAU - Hoelzel, A Rus
AU  - Hoelzel AR
LA  - eng
SI  - GENBANK/KF570315
SI  - GENBANK/KF570316
SI  - GENBANK/KF570317
SI  - GENBANK/KF570318
SI  - GENBANK/KF570319
SI  - GENBANK/KF570320
SI  - GENBANK/KF570321
SI  - GENBANK/KF570322
SI  - GENBANK/KF570323
SI  - GENBANK/KF570324
SI  - GENBANK/KF570325
SI  - GENBANK/KF570326
SI  - GENBANK/KF570327
SI  - GENBANK/KF570328
SI  - GENBANK/KF570329
SI  - GENBANK/KF570330
SI  - GENBANK/KF570331
SI  - GENBANK/KF570332
SI  - GENBANK/KF570333
SI  - GENBANK/KF570334
SI  - GENBANK/KF570335
SI  - GENBANK/KF570336
SI  - GENBANK/KF570337
SI  - GENBANK/KF570338
SI  - GENBANK/KF570339
SI  - GENBANK/KF570340
SI  - GENBANK/KF570341
SI  - GENBANK/KF570342
SI  - GENBANK/KF570343
SI  - GENBANK/KF570344
SI  - GENBANK/KF570345
SI  - GENBANK/KF570346
SI  - GENBANK/KF570347
SI  - GENBANK/KF570348
SI  - GENBANK/KF570349
SI  - GENBANK/KF570350
SI  - GENBANK/KF570351
SI  - GENBANK/KF570352
SI  - GENBANK/KF570353
SI  - GENBANK/KF570354
SI  - GENBANK/KF570355
SI  - GENBANK/KF570356
SI  - GENBANK/KF570357
SI  - GENBANK/KF570358
SI  - GENBANK/KF570359
SI  - GENBANK/KF570360
SI  - GENBANK/KF570361
SI  - GENBANK/KF570362
SI  - GENBANK/KF570363
SI  - GENBANK/KF570364
SI  - GENBANK/KF570365
SI  - GENBANK/KF570366
SI  - GENBANK/KF570367
SI  - GENBANK/KF570368
SI  - GENBANK/KF570369
SI  - GENBANK/KF570370
SI  - GENBANK/KF570371
SI  - GENBANK/KF570372
SI  - GENBANK/KF570373
SI  - GENBANK/KF570374
SI  - GENBANK/KF570375
SI  - GENBANK/KF570376
SI  - GENBANK/KF570377
SI  - GENBANK/KF570378
SI  - GENBANK/KF570379
SI  - GENBANK/KF570380
SI  - GENBANK/KF570381
SI  - GENBANK/KF570382
SI  - GENBANK/KF570383
SI  - GENBANK/KF570384
SI  - GENBANK/KF570385
SI  - GENBANK/KF570386
SI  - GENBANK/KF570387
SI  - GENBANK/KF570388
SI  - GENBANK/KF570389
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130808
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - *Aquatic Organisms/classification/genetics
MH  - *Biodiversity
MH  - DNA, Mitochondrial/genetics
MH  - Dolphins/*classification/*genetics
MH  - *Ecosystem
MH  - *Environment
MH  - Genome, Mitochondrial/genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
EDAT- 2013/08/10 06:00
MHDA- 2014/03/13 06:00
CRDT- 2013/08/10 06:00
PHST- 2013/08/10 06:00 [entrez]
PHST- 2013/08/10 06:00 [pubmed]
PHST- 2014/03/13 06:00 [medline]
AID - syt051 [pii]
AID - 10.1093/sysbio/syt051 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Nov;62(6):865-77. doi: 10.1093/sysbio/syt051. Epub 2013 Aug 8.

PMID- 23925510
OWN - NLM
STAT- MEDLINE
DCOM- 20140312
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 6
DP  - 2013 Nov
TI  - Efficient exploration of the space of reconciled gene trees.
PG  - 901-12
LID - 10.1093/sysbio/syt054 [doi]
AB  - Gene trees record the combination of gene-level events, such as duplication,
      transfer and loss (DTL), and species-level events, such as speciation and
      extinction. Gene tree-species tree reconciliation methods model these processes
      by drawing gene trees into the species tree using a series of gene and
      species-level events. The reconstruction of gene trees based on sequence alone
      almost always involves choosing between statistically equivalent or weakly
      distinguishable relationships that could be much better resolved based on a
      putative species tree. To exploit this potential for accurate reconstruction of
      gene trees, the space of reconciled gene trees must be explored according to a
      joint model of sequence evolution and gene tree-species tree reconciliation. Here
      we present amalgamated likelihood estimation (ALE), a probabilistic approach to
      exhaustively explore all reconciled gene trees that can be amalgamated as a
      combination of clades observed in a sample of gene trees. We implement the ALE
      approach in the context of a reconciliation model (Szollosi et al. 2013), which
      allows for the DTL of genes. We use ALE to efficiently approximate the sum of the
      joint likelihood over amalgamations and to find the reconciled gene tree that
      maximizes the joint likelihood among all such trees. We demonstrate using
      simulations that gene trees reconstructed using the joint likelihood are
      substantially more accurate than those reconstructed using sequence alone. Using 
      realistic gene tree topologies, branch lengths, and alignment sizes, we
      demonstrate that ALE produces more accurate gene trees even if the model of
      sequence evolution is greatly simplified. Finally, examining 1099 gene families
      from 36 cyanobacterial genomes we find that joint likelihood-based inference
      results in a striking reduction in apparent phylogenetic discord, with
      respectively. 24%, 59%, and 46% reductions in the mean numbers of duplications,
      transfers, and losses per gene family. The open source implementation of ALE is
      available from https://github.com/ssolo/ALE.git.
FAU - Szollosi, Gergely J
AU  - Szollosi GJ
AD  - Laboratoire de Biometrie et Biologie Evolutive, Centre National de la Recherche
      Scientifique, Unite Mixte de Recherche 5558, Universite Lyon 1, F-69622
      Villeurbanne;
FAU - Rosikiewicz, Wojciech
AU  - Rosikiewicz W
FAU - Boussau, Bastien
AU  - Boussau B
FAU - Tannier, Eric
AU  - Tannier E
FAU - Daubin, Vincent
AU  - Daubin V
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130806
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Cyanobacteria/classification/genetics
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Sequence Analysis, DNA
PMC - PMC3797637
EDAT- 2013/08/09 06:00
MHDA- 2014/03/13 06:00
CRDT- 2013/08/09 06:00
PHST- 2013/08/09 06:00 [entrez]
PHST- 2013/08/09 06:00 [pubmed]
PHST- 2014/03/13 06:00 [medline]
AID - syt054 [pii]
AID - 10.1093/sysbio/syt054 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Nov;62(6):901-12. doi: 10.1093/sysbio/syt054. Epub 2013 Aug 6.

PMID- 23925509
OWN - NLM
STAT- MEDLINE
DCOM- 20140312
LR  - 20131017
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 6
DP  - 2013 Nov
TI  - Bite of the cats: relationships between functional integration and mechanical
      performance as revealed by mandible geometry.
PG  - 878-900
LID - 10.1093/sysbio/syt053 [doi]
AB  - Cat-like carnivorous mammals represent a relatively homogeneous group of species 
      whose morphology appears constrained by exclusive adaptations for meat eating. We
      present the most comprehensive data set of extant and extinct cat-like species to
      test for evolutionary transformations in size, shape and mechanical performance, 
      that is, von Mises stress and surface traction, of the mandible. Size and shape
      were both quantified by means of geometric morphometrics, whereas mechanical
      performance was assessed applying finite element models to 2D geometry of the
      mandible. Additionally, we present the first almost complete composite phylogeny 
      of cat-like carnivorans for which well-preserved mandibles are known, including
      representatives of 35 extant and 59 extinct species of Felidae, Nimravidae, and
      Barbourofelidae. This phylogeny was used to test morphological differentiation,
      allometry, and covariation of mandible parts within and among clades. After
      taking phylogeny into account, we found that both allometry and mechanical
      variables exhibit a significant impact on mandible shape. We also tested whether 
      mechanical performance was linked to morphological integration. Mechanical stress
      at the coronoid process is higher in sabertoothed cats than in any other clade.
      This is strongly related to the high degree of covariation within modules of
      sabertooths mandibles. We found significant correlation between integration at
      the clade level and per-clade averaged stress values, on both original data and
      by partialling out interclade allometry from shapes when calculating integration.
      This suggests a strong interaction between natural selection and the evolution of
      developmental and functional modules at the clade level.
FAU - Piras, Paolo
AU  - Piras P
AD  - Center for Evolutionary Ecology, Largo San Leonardo Murialdo 1, 00146, Rome,
      Italy;
FAU - Maiorino, Leonardo
AU  - Maiorino L
FAU - Teresi, Luciano
AU  - Teresi L
FAU - Meloro, Carlo
AU  - Meloro C
FAU - Lucci, Federico
AU  - Lucci F
FAU - Kotsakis, Tassos
AU  - Kotsakis T
FAU - Raia, Pasquale
AU  - Raia P
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130806
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Carnivory/physiology
MH  - Cats
MH  - Computer Simulation
MH  - Felidae/*anatomy &amp; histology
MH  - Linear Models
MH  - Mandible/*anatomy &amp; histology
MH  - Models, Biological
MH  - Multivariate Analysis
MH  - *Phylogeny
MH  - Software
EDAT- 2013/08/09 06:00
MHDA- 2014/03/13 06:00
CRDT- 2013/08/09 06:00
PHST- 2013/08/09 06:00 [entrez]
PHST- 2013/08/09 06:00 [pubmed]
PHST- 2014/03/13 06:00 [medline]
AID - syt053 [pii]
AID - 10.1093/sysbio/syt053 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Nov;62(6):878-900. doi: 10.1093/sysbio/syt053. Epub 2013 Aug 6.

PMID- 23925508
OWN - NLM
STAT- MEDLINE
DCOM- 20140312
LR  - 20131017
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 6
DP  - 2013 Nov
TI  - Ecological opportunity and incumbency in the diversification of repeated
      continental colonizations by muroid rodents.
PG  - 837-64
LID - 10.1093/sysbio/syt050 [doi]
AB  - Why some clades are more species-rich than others is a central question in
      macroevolution. Most hypotheses explaining exceptionally diverse clades involve
      the emergence of an ecological opportunity caused by a major biogeographic
      transition or evolution of a key innovation. The radiation of muroid rodents is
      an ideal model for testing theories of diversification rates in relation to
      biogeography and ecological opportunity because the group is exceptionally
      species-rich (comprising nearly one-third of all mammal species), it is
      ecologically diverse, and it has colonized every major landmass except New
      Zealand and Antarctica, thus providing multiple replicate radiations. We present 
      an extension of the conventional ecological opportunity model to include a
      geographic incumbency effect, develop the largest muroid phylogeny to date, and
      use this phylogeny to test the new model. The nearly 300-species phylogeny based 
      on four nuclear genes is robustly resolved throughout. Consistent with the fossil
      record, we identified Eurasia as the most likely origin of the group and
      reconstructed five to seven colonizations of Africa, five of North America, four 
      of Southeast Asia, two of South America, two of Sahul, one of Madagascar, and
      eight to ten recolonizations of Eurasia. We accounted for incomplete taxon
      sampling by using multiple statistical methods and identified three corroborated 
      regions of the tree with significant shifts in diversification rates. In several 
      cases, higher rates were associated with the first colonization of a continental 
      area, but most colonizations were not followed by bursts of speciation. We found 
      strong evidence for diversification consistent with the ecological opportunity
      model (initial burst followed by density-dependent slowdown) in the first
      colonization of South America and partial support for this model in the first
      colonization of Sahul. Primary colonizers appear to inhibit the ultimate
      diversity of secondary colonizers, a pattern of incumbency that is consistent
      with ecological opportunity, but they did not inhibit initial diversification
      rates of secondary colonizers. These results indicate that ecological opportunity
      may be a general but weak process in muroids and one that requires specific
      circumstances to lead to an adaptive radiation. The total land area, length of
      time between colonizations, and rank of colonizations did not influence the
      diversification rates of primary colonizers. Models currently employed to test
      ecological opportunity do a poor job of explaining muroid diversity. In addition,
      the various rate-shift metrics identified different clades, suggesting that
      caution should be used when only one is applied, and we discuss which methods are
      most appropriate to address different questions of diversification.
FAU - Schenk, John J
AU  - Schenk JJ
AD  - Department of Biological Science, Florida State University, Tallahassee, FL
      32306-4295, USA; and Museum Victoria, GPO Box 666, Melbourne, Victoria 3001,
      Australia.
FAU - Rowe, Kevin C
AU  - Rowe KC
FAU - Steppan, Scott J
AU  - Steppan SJ
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130806
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biodiversity
MH  - Ecology
MH  - Genetic Speciation
MH  - *Genetic Variation
MH  - *Phylogeny
MH  - Phylogeography
MH  - Rodentia/*classification/genetics
EDAT- 2013/08/09 06:00
MHDA- 2014/03/13 06:00
CRDT- 2013/08/09 06:00
PHST- 2013/08/09 06:00 [entrez]
PHST- 2013/08/09 06:00 [pubmed]
PHST- 2014/03/13 06:00 [medline]
AID - syt050 [pii]
AID - 10.1093/sysbio/syt050 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Nov;62(6):837-64. doi: 10.1093/sysbio/syt050. Epub 2013 Aug 6.

PMID- 23843314
OWN - NLM
STAT- MEDLINE
DCOM- 20140312
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 6
DP  - 2013 Nov
TI  - Minimizing the average distance to a closest leaf in a phylogenetic tree.
PG  - 824-36
LID - 10.1093/sysbio/syt044 [doi]
AB  - When performing an analysis on a collection of molecular sequences, it can be
      convenient to reduce the number of sequences under consideration while
      maintaining some characteristic of a larger collection of sequences. For example,
      one may wish to select a subset of high-quality sequences that represent the
      diversity of a larger collection of sequences. One may also wish to specialize a 
      large database of characterized "reference sequences" to a smaller subset that is
      as close as possible on average to a collection of "query sequences" of interest.
      Such a representative subset can be useful whenever one wishes to find a set of
      reference sequences that is appropriate to use for comparative analysis of
      environmentally derived sequences, such as for selecting "reference tree"
      sequences for phylogenetic placement of metagenomic reads. In this article, we
      formalize these problems in terms of the minimization of the Average Distance to 
      the Closest Leaf (ADCL) and investigate algorithms to perform the relevant
      minimization. We show that the greedy algorithm is not effective, show that a
      variant of the Partitioning Around Medoids (PAM) heuristic gets stuck in local
      minima, and develop an exact dynamic programming approach. Using this exact
      program we note that the performance of PAM appears to be good for simulated
      trees, and is faster than the exact algorithm for small trees. On the other hand,
      the exact program gives solutions for all numbers of leaves less than or equal to
      the given desired number of leaves, whereas PAM only gives a solution for the
      prespecified number of leaves. Via application to real data, we show that the
      ADCL criterion chooses chimeric sequences less often than random subsets, whereas
      the maximization of phylogenetic diversity chooses them more often than random.
      These algorithms have been implemented in publicly available software.
FAU - Matsen, Frederick A 4th
AU  - Matsen FA 4th
AD  - Program in Computational Biology, Fred Hutchinson Cancer Research Center,
      Seattle, WA 91802, USA.
FAU - Gallagher, Aaron
AU  - Gallagher A
FAU - McCoy, Connor O
AU  - McCoy CO
LA  - eng
GR  - R37 AI038518/AI/NIAID NIH HHS/United States
PT  - Journal Article
DEP - 20130710
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - Enterobacteriaceae/classification/genetics
MH  - *Phylogeny
MH  - Reproducibility of Results
PMC - PMC3797636
EDAT- 2013/07/12 06:00
MHDA- 2014/03/13 06:00
CRDT- 2013/07/12 06:00
PHST- 2013/07/12 06:00 [entrez]
PHST- 2013/07/12 06:00 [pubmed]
PHST- 2014/03/13 06:00 [medline]
AID - syt044 [pii]
AID - 10.1093/sysbio/syt044 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Nov;62(6):824-36. doi: 10.1093/sysbio/syt044. Epub 2013 Jul 10.

PMID- 23771888
OWN - NLM
STAT- MEDLINE
DCOM- 20140312
LR  - 20131017
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 6
DP  - 2013 Nov
TI  - Multilocus species delimitation in a complex of morphologically conserved
      trapdoor spiders (mygalomorphae, antrodiaetidae, aliatypus).
PG  - 805-23
LID - 10.1093/sysbio/syt041 [doi]
AB  - Species are a fundamental unit for biological studies, yet no uniform guidelines 
      exist for determining species limits in an objective manner. Given the large
      number of species concepts available, defining species can be both highly
      subjective and biased. Although morphology has been commonly used to determine
      species boundaries, the availability and prevalence of genetic data has allowed
      researchers to use such data to make inferences regarding species limits. Genetic
      data also have been used in the detection of cryptic species, where other lines
      of evidence (morphology in particular) may underestimate species diversity. In
      this study, we investigate species limits in a complex of morphologically
      conserved trapdoor spiders (Mygalomorphae, Antrodiaetidae, Aliatypus) from
      California. Multiple approaches were used to determine species boundaries in this
      highly genetically fragmented group, including both multilocus discovery and
      validation approaches (plus a chimeric approach). Additionally, we introduce a
      novel tree-based discovery approach using species trees. Results suggest that
      this complex includes multiple cryptic species, with two groupings consistently
      recovered across analyses. Due to incongruence across analyses for the remaining 
      samples, we take a conservative approach and recognize a three species complex,
      and formally describe two new species (Aliatypus roxxiae, sp. nov. and Aliatypus 
      starretti, sp. nov.). This study helps to clarify species limits in a genetically
      fragmented group and provides a framework for identifying and defining the
      cryptic lineage diversity that prevails in many organismal groups.
FAU - Satler, Jordan D
AU  - Satler JD
AD  - Department of Evolution, Ecology and Organismal Biology, The Ohio University,
      Columbus, OH 43210, USA and Department of Biology, San Diego University, San
      Diego, CA 92182, USA.
FAU - Carstens, Bryan C
AU  - Carstens BC
FAU - Hedin, Marshal
AU  - Hedin M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130613
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - 0 (Microtubule Proteins)
RN  - 0 (Peptide Elongation Factor 1)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - California
MH  - DNA, Ribosomal/genetics
MH  - Electron Transport Complex IV/genetics
MH  - Microtubule Proteins/genetics
MH  - Peptide Elongation Factor 1/genetics
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Species Specificity
MH  - Spiders/anatomy &amp; histology/*classification/*genetics
EDAT- 2013/06/19 06:00
MHDA- 2014/03/13 06:00
CRDT- 2013/06/18 06:00
PHST- 2013/06/18 06:00 [entrez]
PHST- 2013/06/19 06:00 [pubmed]
PHST- 2014/03/13 06:00 [medline]
AID - syt041 [pii]
AID - 10.1093/sysbio/syt041 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Nov;62(6):805-23. doi: 10.1093/sysbio/syt041. Epub 2013 Jun 13.

PMID- 23749787
OWN - NLM
STAT- MEDLINE
DCOM- 20140228
LR  - 20130812
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 5
DP  - 2013 Sep
TI  - Addressing gene tree discordance and non-stationarity to resolve a multi-locus
      phylogeny of the flatfishes (Teleostei: Pleuronectiformes).
PG  - 763-85
LID - 10.1093/sysbio/syt039 [doi]
AB  - Non-homogeneous processes and, in particular, base compositional non-stationarity
      have long been recognized as a critical source of systematic error. But only a
      small fraction of current molecular systematic studies methodically examine and
      effectively account for the potentially confounding effect of non-stationarity.
      The problem is especially overlooked in multi-locus or phylogenomic scale
      analyses, in part because no efficient tools exist to accommodate base
      composition heterogeneity in large data sets. We present a detailed analysis of a
      data set with 20 genes and 214 taxa to study the phylogeny of flatfishes
      (Pleuronectiformes) and their position among percomorphs. Most genes vary
      significantly in base composition among taxa and fail to resolve flatfish
      monophyly and other emblematic groups, suggesting that non-stationarity may be
      causing systematic error. We show a strong association between base compositional
      bias and topological discordance among individual gene partitions and their
      inferred trees. Phylogenetic methods applying non-homogeneous models to
      accommodate non-stationarity have relatively minor effect to reduce gene tree
      discordance, suggesting that available computer programs applying these methods
      do not scale up efficiently to the data set of modest size analysed in this
      study. By comparing phylogenetic trees obtained with species tree (STAR) and
      concatenation approaches, we show that gene tree discordance in our data set is
      most likely due to base compositional biases than to incomplete lineage sorting. 
      Multi-locus analyses suggest that the combined phylogenetic signal from all loci 
      in a concatenated data set overcomes systematic biases induced by
      non-stationarity at each partition. Finally, relationships among flatfishes and
      their relatives are discussed in the light of these results. We find support for 
      the monophyly of flatfishes and confirm findings from previous molecular
      phylogenetic studies suggesting their close affinity with several carangimorph
      groups (i.e., jack and allies, barracuda, archerfish, billfish and swordfish,
      threadfin, moonfish, beach salmon, and snook and barramundi).
FAU - Betancur-R, Ricardo
AU  - Betancur-R R
AD  - Department of Biological Sciences, The George Washington University, 2023 G St.
      NW, Washington, D.C. 20052, USA; College of Fisheries and Life Science, Shanghai 
      Ocean University, Shanghai 201306, China; and National Systematics Laboratory
      NMFS/NOAA, Post Office Box 37012, Smithsonian Institution NHB, WC 60, MRC-153,
      Washington, D.C. 20013-7012, USA.
FAU - Li, Chenhong
AU  - Li C
FAU - Munroe, Thomas A
AU  - Munroe TA
FAU - Ballesteros, Jesus A
AU  - Ballesteros JA
FAU - Orti, Guillermo
AU  - Orti G
LA  - eng
SI  - GENBANK/JQ937381
SI  - GENBANK/JQ937382
SI  - GENBANK/JQ937383
SI  - GENBANK/JQ937384
SI  - GENBANK/JQ937385
SI  - GENBANK/JQ937386
SI  - GENBANK/JQ937387
SI  - GENBANK/JQ937388
SI  - GENBANK/JQ937389
SI  - GENBANK/JQ937390
SI  - GENBANK/JQ937391
SI  - GENBANK/JQ937392
SI  - GENBANK/JQ937393
SI  - GENBANK/JQ937394
SI  - GENBANK/JQ937395
SI  - GENBANK/JQ937396
SI  - GENBANK/JQ937397
SI  - GENBANK/JQ937398
SI  - GENBANK/JQ937399
SI  - GENBANK/JQ937400
SI  - GENBANK/JQ937401
SI  - GENBANK/JQ937402
SI  - GENBANK/JQ937403
SI  - GENBANK/JQ937404
SI  - GENBANK/JQ937405
SI  - GENBANK/JQ937406
SI  - GENBANK/JQ937407
SI  - GENBANK/JQ937408
SI  - GENBANK/JQ937409
SI  - GENBANK/JQ937410
SI  - GENBANK/JQ937411
SI  - GENBANK/JQ937412
SI  - GENBANK/JQ937413
SI  - GENBANK/JQ937414
SI  - GENBANK/JQ937415
SI  - GENBANK/JQ937416
SI  - GENBANK/JQ937417
SI  - GENBANK/JQ937418
SI  - GENBANK/JQ937419
SI  - GENBANK/JQ937420
SI  - GENBANK/JQ937421
SI  - GENBANK/JQ937422
SI  - GENBANK/JQ937423
SI  - GENBANK/JQ937424
SI  - GENBANK/JQ937425
SI  - GENBANK/JQ937426
SI  - GENBANK/JQ937427
SI  - GENBANK/JQ937428
SI  - GENBANK/JQ937429
SI  - GENBANK/JQ937430
SI  - GENBANK/JQ937431
SI  - GENBANK/JQ937432
SI  - GENBANK/JQ937433
SI  - GENBANK/JQ937434
SI  - GENBANK/JQ937435
SI  - GENBANK/JQ937436
SI  - GENBANK/JQ937437
SI  - GENBANK/JQ937438
SI  - GENBANK/JQ937439
SI  - GENBANK/JQ937440
SI  - GENBANK/JQ937441
SI  - GENBANK/JQ937442
SI  - GENBANK/JQ937443
SI  - GENBANK/JQ937444
SI  - GENBANK/JQ937445
SI  - GENBANK/JQ937446
SI  - GENBANK/JQ937447
SI  - GENBANK/JQ937448
SI  - GENBANK/JQ937449
SI  - GENBANK/JQ937450
SI  - GENBANK/JQ937451
SI  - GENBANK/JQ937452
SI  - GENBANK/JQ937453
SI  - GENBANK/JQ937454
SI  - GENBANK/JQ937455
SI  - GENBANK/JQ937456
SI  - GENBANK/JQ937457
SI  - GENBANK/JQ937458
SI  - GENBANK/JQ937459
SI  - GENBANK/JQ937460
SI  - GENBANK/JQ937461
SI  - GENBANK/JQ937462
SI  - GENBANK/JQ937463
SI  - GENBANK/JQ937464
SI  - GENBANK/JQ937465
SI  - GENBANK/JQ937466
SI  - GENBANK/JQ937467
SI  - GENBANK/JQ937468
SI  - GENBANK/JQ937469
SI  - GENBANK/JQ937470
SI  - GENBANK/JQ937471
SI  - GENBANK/JQ937472
SI  - GENBANK/JQ937473
SI  - GENBANK/JQ937474
SI  - GENBANK/JQ937475
SI  - GENBANK/JQ937476
SI  - GENBANK/JQ937477
SI  - GENBANK/JQ937478
SI  - GENBANK/JQ937479
SI  - GENBANK/JQ937480
SI  - GENBANK/JQ937481
SI  - GENBANK/JQ937482
SI  - GENBANK/JQ937483
SI  - GENBANK/JQ937484
SI  - GENBANK/JQ937485
SI  - GENBANK/JQ937486
SI  - GENBANK/JQ937487
SI  - GENBANK/JQ937488
SI  - GENBANK/JQ937489
SI  - GENBANK/JQ937490
SI  - GENBANK/JQ937491
SI  - GENBANK/JQ937492
SI  - GENBANK/JQ937493
SI  - GENBANK/JQ937494
SI  - GENBANK/JQ937495
SI  - GENBANK/JQ937496
SI  - GENBANK/JQ937497
SI  - GENBANK/JQ937498
SI  - GENBANK/JQ937499
SI  - GENBANK/JQ937500
SI  - GENBANK/JQ937501
SI  - GENBANK/JQ937502
SI  - GENBANK/JQ937503
SI  - GENBANK/JQ937504
SI  - GENBANK/JQ937505
SI  - GENBANK/JQ937506
SI  - GENBANK/JQ937507
SI  - GENBANK/JQ937508
SI  - GENBANK/JQ937509
SI  - GENBANK/JQ937510
SI  - GENBANK/JQ937511
SI  - GENBANK/JQ937512
SI  - GENBANK/JQ937513
SI  - GENBANK/JQ937514
SI  - GENBANK/JQ937515
SI  - GENBANK/JQ937516
SI  - GENBANK/JQ937517
SI  - GENBANK/JQ937518
SI  - GENBANK/JQ937519
SI  - GENBANK/JQ937520
SI  - GENBANK/JQ937521
SI  - GENBANK/JQ937522
SI  - GENBANK/JQ937523
SI  - GENBANK/JQ937524
SI  - GENBANK/JQ937525
SI  - GENBANK/JQ937526
SI  - GENBANK/JQ937527
SI  - GENBANK/JQ937528
SI  - GENBANK/JQ937529
SI  - GENBANK/JQ937530
SI  - GENBANK/JQ937531
SI  - GENBANK/JQ937532
SI  - GENBANK/JQ937533
SI  - GENBANK/JQ937534
SI  - GENBANK/JQ937535
SI  - GENBANK/JQ937536
SI  - GENBANK/JQ937537
SI  - GENBANK/JQ937538
SI  - GENBANK/JQ937539
SI  - GENBANK/JQ937540
SI  - GENBANK/JQ937541
SI  - GENBANK/JQ937542
SI  - GENBANK/JQ937543
SI  - GENBANK/JQ937544
SI  - GENBANK/JQ937545
SI  - GENBANK/JQ937546
SI  - GENBANK/JQ937547
SI  - GENBANK/JQ937548
SI  - GENBANK/JQ937549
SI  - GENBANK/JQ937550
SI  - GENBANK/JQ937551
SI  - GENBANK/JQ937552
SI  - GENBANK/JQ937553
SI  - GENBANK/JQ937554
SI  - GENBANK/JQ937555
SI  - GENBANK/JQ937556
SI  - GENBANK/JQ937557
SI  - GENBANK/JQ937558
SI  - GENBANK/JQ937559
SI  - GENBANK/JQ937560
SI  - GENBANK/JQ937561
SI  - GENBANK/JQ937562
SI  - GENBANK/JQ937563
SI  - GENBANK/JQ937564
SI  - GENBANK/JQ937565
SI  - GENBANK/JQ937566
SI  - GENBANK/JQ937567
SI  - GENBANK/JQ937568
SI  - GENBANK/JQ937569
SI  - GENBANK/JQ937570
SI  - GENBANK/JQ937571
SI  - GENBANK/JQ937572
SI  - GENBANK/JQ937573
SI  - GENBANK/JQ937574
SI  - GENBANK/JQ937575
SI  - GENBANK/JQ937576
SI  - GENBANK/JQ937577
SI  - GENBANK/JQ937578
SI  - GENBANK/JQ937579
SI  - GENBANK/JQ937580
SI  - GENBANK/JQ937581
SI  - GENBANK/JQ937582
SI  - GENBANK/JQ937583
SI  - GENBANK/JQ937584
SI  - GENBANK/JQ937585
SI  - GENBANK/JQ937586
SI  - GENBANK/JQ937587
SI  - GENBANK/JQ937588
SI  - GENBANK/JQ937589
SI  - GENBANK/JQ937590
SI  - GENBANK/JQ937591
SI  - GENBANK/JQ937592
SI  - GENBANK/JQ937593
SI  - GENBANK/JQ937594
SI  - GENBANK/JQ937595
SI  - GENBANK/JQ937596
SI  - GENBANK/JQ937597
SI  - GENBANK/JQ937598
SI  - GENBANK/JQ937599
SI  - GENBANK/JQ937600
SI  - GENBANK/JQ937601
SI  - GENBANK/JQ937602
SI  - GENBANK/JQ937603
SI  - GENBANK/JQ937604
SI  - GENBANK/JQ937605
SI  - GENBANK/JQ937606
SI  - GENBANK/JQ937607
SI  - GENBANK/JQ937608
SI  - GENBANK/JQ937609
SI  - GENBANK/JQ937610
SI  - GENBANK/JQ937611
SI  - GENBANK/JQ937612
SI  - GENBANK/JQ937613
SI  - GENBANK/JQ937614
SI  - GENBANK/JQ937615
SI  - GENBANK/JQ937616
SI  - GENBANK/JQ937617
SI  - GENBANK/JQ937618
SI  - GENBANK/JQ937619
SI  - GENBANK/JQ937620
SI  - GENBANK/JQ937621
SI  - GENBANK/JQ937622
SI  - GENBANK/JQ937623
SI  - GENBANK/JQ937624
SI  - GENBANK/JQ937625
SI  - GENBANK/JQ937626
SI  - GENBANK/JQ937627
SI  - GENBANK/JQ937628
SI  - GENBANK/JQ937629
SI  - GENBANK/JQ937630
SI  - GENBANK/JQ937631
SI  - GENBANK/JQ937632
SI  - GENBANK/JQ937633
SI  - GENBANK/JQ937634
SI  - GENBANK/JQ937635
SI  - GENBANK/JQ937636
SI  - GENBANK/JQ937637
SI  - GENBANK/JQ937638
SI  - GENBANK/JQ937639
SI  - GENBANK/JQ937640
SI  - GENBANK/JQ937641
SI  - GENBANK/JQ937642
SI  - GENBANK/JQ937643
SI  - GENBANK/JQ937644
SI  - GENBANK/JQ937645
SI  - GENBANK/JQ937646
SI  - GENBANK/JQ937647
SI  - GENBANK/JQ937648
SI  - GENBANK/JQ937649
SI  - GENBANK/JQ937650
SI  - GENBANK/JQ937651
SI  - GENBANK/JQ937652
SI  - GENBANK/JQ937653
SI  - GENBANK/JQ937654
SI  - GENBANK/JQ937655
SI  - GENBANK/JQ937656
SI  - GENBANK/JQ937657
SI  - GENBANK/JQ937658
SI  - GENBANK/JQ937659
SI  - GENBANK/JQ937660
SI  - GENBANK/JQ937661
SI  - GENBANK/JQ937662
SI  - GENBANK/JQ937663
SI  - GENBANK/JQ937664
SI  - GENBANK/JQ937665
SI  - GENBANK/JQ937666
SI  - GENBANK/JQ937667
SI  - GENBANK/JQ937668
SI  - GENBANK/JQ937669
SI  - GENBANK/JQ937670
SI  - GENBANK/JQ937671
SI  - GENBANK/JQ937672
SI  - GENBANK/JQ937673
SI  - GENBANK/JQ937674
SI  - GENBANK/JQ937675
SI  - GENBANK/JQ937676
SI  - GENBANK/JQ937677
SI  - GENBANK/JQ937678
SI  - GENBANK/JQ937679
SI  - GENBANK/JQ937680
SI  - GENBANK/JQ937681
SI  - GENBANK/JQ937682
SI  - GENBANK/JQ937683
SI  - GENBANK/JQ937684
SI  - GENBANK/JQ937685
SI  - GENBANK/JQ937686
SI  - GENBANK/JQ937687
SI  - GENBANK/JQ937688
SI  - GENBANK/JQ937689
SI  - GENBANK/JQ937690
SI  - GENBANK/JQ937691
SI  - GENBANK/JQ937692
SI  - GENBANK/JQ937693
SI  - GENBANK/JQ937694
SI  - GENBANK/JQ937695
SI  - GENBANK/JQ937696
SI  - GENBANK/JQ937697
SI  - GENBANK/JQ937698
SI  - GENBANK/JQ937699
SI  - GENBANK/JQ937700
SI  - GENBANK/JQ937701
SI  - GENBANK/JQ937702
SI  - GENBANK/JQ937703
SI  - GENBANK/JQ937704
SI  - GENBANK/JQ937705
SI  - GENBANK/JQ937706
SI  - GENBANK/JQ937707
SI  - GENBANK/JQ937708
SI  - GENBANK/JQ937709
SI  - GENBANK/JQ937710
SI  - GENBANK/JQ937711
SI  - GENBANK/JQ937712
SI  - GENBANK/JQ937713
SI  - GENBANK/JQ937714
SI  - GENBANK/JQ937715
SI  - GENBANK/JQ937716
SI  - GENBANK/JQ937717
SI  - GENBANK/JQ937718
SI  - GENBANK/JQ937719
SI  - GENBANK/JQ937720
SI  - GENBANK/JQ937721
SI  - GENBANK/JQ937722
SI  - GENBANK/JQ937723
SI  - GENBANK/JQ937724
SI  - GENBANK/JQ937725
SI  - GENBANK/JQ937726
SI  - GENBANK/JQ937727
SI  - GENBANK/JQ937728
SI  - GENBANK/JQ937729
SI  - GENBANK/JQ937730
SI  - GENBANK/JQ937731
SI  - GENBANK/JQ937732
SI  - GENBANK/JQ937733
SI  - GENBANK/JQ937734
SI  - GENBANK/JQ937735
SI  - GENBANK/JQ937736
SI  - GENBANK/JQ937737
SI  - GENBANK/JQ937738
SI  - GENBANK/JQ937739
SI  - GENBANK/JQ937740
SI  - GENBANK/JQ937741
SI  - GENBANK/JQ937742
SI  - GENBANK/JQ937743
SI  - GENBANK/JQ937744
SI  - GENBANK/JQ937745
SI  - GENBANK/JQ937746
SI  - GENBANK/JQ937747
SI  - GENBANK/JQ937748
SI  - GENBANK/JQ937749
SI  - GENBANK/JQ937750
SI  - GENBANK/JQ937751
SI  - GENBANK/JQ937752
SI  - GENBANK/JQ937753
SI  - GENBANK/JQ937754
SI  - GENBANK/JQ937755
SI  - GENBANK/JQ937756
SI  - GENBANK/JQ937757
SI  - GENBANK/JQ937758
SI  - GENBANK/JQ937759
SI  - GENBANK/JQ937760
SI  - GENBANK/JQ937761
SI  - GENBANK/JQ937762
SI  - GENBANK/JQ937763
SI  - GENBANK/JQ937764
SI  - GENBANK/JQ937765
SI  - GENBANK/JQ937766
SI  - GENBANK/JQ937767
SI  - GENBANK/JQ937768
SI  - GENBANK/JQ937769
SI  - GENBANK/JQ937770
SI  - GENBANK/JQ937771
SI  - GENBANK/JQ937772
SI  - GENBANK/JQ937773
SI  - GENBANK/JQ937774
SI  - GENBANK/JQ937775
SI  - GENBANK/JQ937776
SI  - GENBANK/JQ937777
SI  - GENBANK/JQ937778
SI  - GENBANK/JQ937779
SI  - GENBANK/JQ937780
SI  - GENBANK/JQ937781
SI  - GENBANK/JQ937782
SI  - GENBANK/JQ937783
SI  - GENBANK/JQ937784
SI  - GENBANK/JQ937785
SI  - GENBANK/JQ937786
SI  - GENBANK/JQ937787
SI  - GENBANK/JQ937788
SI  - GENBANK/JQ937789
SI  - GENBANK/JQ937790
SI  - GENBANK/JQ937791
SI  - GENBANK/JQ937792
SI  - GENBANK/JQ937793
SI  - GENBANK/JQ937794
SI  - GENBANK/JQ937795
SI  - GENBANK/JQ937796
SI  - GENBANK/JQ937797
SI  - GENBANK/JQ937798
SI  - GENBANK/JQ937799
SI  - GENBANK/JQ937800
SI  - GENBANK/JQ937801
SI  - GENBANK/JQ937802
SI  - GENBANK/JQ937803
SI  - GENBANK/JQ937804
SI  - GENBANK/JQ937805
SI  - GENBANK/JQ937806
SI  - GENBANK/JQ937807
SI  - GENBANK/JQ937808
SI  - GENBANK/JQ937809
SI  - GENBANK/JQ937810
SI  - GENBANK/JQ937811
SI  - GENBANK/JQ937812
SI  - GENBANK/JQ937813
SI  - GENBANK/JQ937814
SI  - GENBANK/JQ937815
SI  - GENBANK/JQ937816
SI  - GENBANK/JQ937817
SI  - GENBANK/JQ937818
SI  - GENBANK/JQ937819
SI  - GENBANK/JQ937820
SI  - GENBANK/JQ937821
SI  - GENBANK/JQ937822
SI  - GENBANK/JQ937823
SI  - GENBANK/JQ937824
SI  - GENBANK/JQ937825
SI  - GENBANK/JQ937826
SI  - GENBANK/JQ937827
SI  - GENBANK/JQ937828
SI  - GENBANK/JQ937829
SI  - GENBANK/JQ937830
SI  - GENBANK/JQ937831
SI  - GENBANK/JQ937832
SI  - GENBANK/JQ937833
SI  - GENBANK/JQ937834
SI  - GENBANK/JQ937835
SI  - GENBANK/JQ937836
SI  - GENBANK/JQ937837
SI  - GENBANK/JQ937838
SI  - GENBANK/JQ937839
SI  - GENBANK/JQ937840
SI  - GENBANK/JQ937841
SI  - GENBANK/JQ937842
SI  - GENBANK/JQ937843
SI  - GENBANK/JQ937844
SI  - GENBANK/JQ937845
SI  - GENBANK/JQ937846
SI  - GENBANK/JQ937847
SI  - GENBANK/JQ937848
SI  - GENBANK/JQ937849
SI  - GENBANK/JQ937850
SI  - GENBANK/JQ937851
SI  - GENBANK/JQ937852
SI  - GENBANK/JQ937853
SI  - GENBANK/JQ937854
SI  - GENBANK/JQ937855
SI  - GENBANK/JQ937856
SI  - GENBANK/JQ937857
SI  - GENBANK/JQ937858
SI  - GENBANK/JQ937859
SI  - GENBANK/JQ937860
SI  - GENBANK/JQ937861
SI  - GENBANK/JQ937862
SI  - GENBANK/JQ937863
SI  - GENBANK/JQ937864
SI  - GENBANK/JQ937865
SI  - GENBANK/JQ937866
SI  - GENBANK/JQ937867
SI  - GENBANK/JQ937868
SI  - GENBANK/JQ937869
SI  - GENBANK/JQ937870
SI  - GENBANK/JQ937871
SI  - GENBANK/JQ937872
SI  - GENBANK/JQ937873
SI  - GENBANK/JQ937874
SI  - GENBANK/JQ937875
SI  - GENBANK/JQ937876
SI  - GENBANK/JQ937877
SI  - GENBANK/JQ937878
SI  - GENBANK/JQ937879
SI  - GENBANK/JQ937880
SI  - GENBANK/JQ937881
SI  - GENBANK/JQ937882
SI  - GENBANK/JQ937883
SI  - GENBANK/JQ937884
SI  - GENBANK/JQ937885
SI  - GENBANK/JQ937886
SI  - GENBANK/JQ937887
SI  - GENBANK/JQ937888
SI  - GENBANK/JQ937889
SI  - GENBANK/JQ937890
SI  - GENBANK/JQ937891
SI  - GENBANK/JQ937892
SI  - GENBANK/JQ937893
SI  - GENBANK/JQ937894
SI  - GENBANK/JQ937895
SI  - GENBANK/JQ937896
SI  - GENBANK/JQ937897
SI  - GENBANK/JQ937898
SI  - GENBANK/JQ937899
SI  - GENBANK/JQ937900
SI  - GENBANK/JQ937901
SI  - GENBANK/JQ937902
SI  - GENBANK/JQ937903
SI  - GENBANK/JQ937904
SI  - GENBANK/JQ937905
SI  - GENBANK/JQ937906
SI  - GENBANK/JQ937907
SI  - GENBANK/JQ937908
SI  - GENBANK/JQ937909
SI  - GENBANK/JQ937910
SI  - GENBANK/JQ937911
SI  - GENBANK/JQ937912
SI  - GENBANK/JQ937913
SI  - GENBANK/JQ937914
SI  - GENBANK/JQ937915
SI  - GENBANK/JQ937916
SI  - GENBANK/JQ937917
SI  - GENBANK/JQ937918
SI  - GENBANK/JQ937919
SI  - GENBANK/JQ937920
SI  - GENBANK/JQ937921
SI  - GENBANK/JQ937922
SI  - GENBANK/JQ937923
SI  - GENBANK/JQ937924
SI  - GENBANK/JQ937925
SI  - GENBANK/JQ937926
SI  - GENBANK/JQ937927
SI  - GENBANK/JQ937928
SI  - GENBANK/JQ937929
SI  - GENBANK/JQ937930
SI  - GENBANK/JQ937931
SI  - GENBANK/JQ937932
SI  - GENBANK/JQ937933
SI  - GENBANK/JQ937934
SI  - GENBANK/JQ937935
SI  - GENBANK/JQ937936
SI  - GENBANK/JQ937937
SI  - GENBANK/JQ937938
SI  - GENBANK/JQ937939
SI  - GENBANK/JQ937940
SI  - GENBANK/JQ937941
SI  - GENBANK/JQ937942
SI  - GENBANK/JQ937943
SI  - GENBANK/JQ937944
SI  - GENBANK/JQ937945
SI  - GENBANK/JQ937946
SI  - GENBANK/JQ937947
SI  - GENBANK/JQ937948
SI  - GENBANK/JQ937949
SI  - GENBANK/JQ937950
SI  - GENBANK/JQ937951
SI  - GENBANK/JQ937952
SI  - GENBANK/JQ937953
SI  - GENBANK/JQ937954
SI  - GENBANK/JQ937955
SI  - GENBANK/JQ937956
SI  - GENBANK/JQ937957
SI  - GENBANK/JQ937958
SI  - GENBANK/JQ937959
SI  - GENBANK/JQ937960
SI  - GENBANK/JQ937961
SI  - GENBANK/JQ937962
SI  - GENBANK/JQ937963
SI  - GENBANK/JQ937964
SI  - GENBANK/JQ937965
SI  - GENBANK/JQ937966
SI  - GENBANK/JQ937967
SI  - GENBANK/JQ937968
SI  - GENBANK/JQ937969
SI  - GENBANK/JQ937970
SI  - GENBANK/JQ937971
SI  - GENBANK/JQ937972
SI  - GENBANK/JQ937973
SI  - GENBANK/JQ937974
SI  - GENBANK/JQ937975
SI  - GENBANK/JQ937976
SI  - GENBANK/JQ937977
SI  - GENBANK/JQ937978
SI  - GENBANK/JQ937979
SI  - GENBANK/JQ937980
SI  - GENBANK/JQ937981
SI  - GENBANK/JQ937982
SI  - GENBANK/JQ937983
SI  - GENBANK/JQ937984
SI  - GENBANK/JQ937985
SI  - GENBANK/JQ937986
SI  - GENBANK/JQ937987
SI  - GENBANK/JQ937988
SI  - GENBANK/JQ937989
SI  - GENBANK/JQ937990
SI  - GENBANK/JQ937991
SI  - GENBANK/JQ937992
SI  - GENBANK/JQ937993
SI  - GENBANK/JQ937994
SI  - GENBANK/JQ937995
SI  - GENBANK/JQ937996
SI  - GENBANK/JQ937997
SI  - GENBANK/JQ937998
SI  - GENBANK/JQ937999
SI  - GENBANK/JQ938000
SI  - GENBANK/JQ938001
SI  - GENBANK/JQ938002
SI  - GENBANK/JQ938003
SI  - GENBANK/JQ938004
SI  - GENBANK/JQ938005
SI  - GENBANK/JQ938006
SI  - GENBANK/JQ938007
SI  - GENBANK/JQ938008
SI  - GENBANK/JQ938009
SI  - GENBANK/JQ938010
SI  - GENBANK/JQ938011
SI  - GENBANK/JQ938012
SI  - GENBANK/JQ938013
SI  - GENBANK/JQ938014
SI  - GENBANK/JQ938015
SI  - GENBANK/JQ938016
SI  - GENBANK/JQ938017
SI  - GENBANK/JQ938018
SI  - GENBANK/JQ938019
SI  - GENBANK/JQ938020
SI  - GENBANK/JQ938021
SI  - GENBANK/JQ938022
SI  - GENBANK/JQ938023
SI  - GENBANK/JQ938024
SI  - GENBANK/JQ938025
SI  - GENBANK/JQ938026
SI  - GENBANK/JQ938027
SI  - GENBANK/JQ938028
SI  - GENBANK/JQ938029
SI  - GENBANK/JQ938030
SI  - GENBANK/JQ938031
SI  - GENBANK/JQ938032
SI  - GENBANK/JQ938033
SI  - GENBANK/JQ938034
SI  - GENBANK/JQ938035
SI  - GENBANK/JQ938036
SI  - GENBANK/JQ938037
SI  - GENBANK/JQ938038
SI  - GENBANK/JQ938039
SI  - GENBANK/JQ938040
SI  - GENBANK/JQ938041
SI  - GENBANK/JQ938042
SI  - GENBANK/JQ938043
SI  - GENBANK/JQ938044
SI  - GENBANK/JQ938045
SI  - GENBANK/JQ938046
SI  - GENBANK/JQ938047
SI  - GENBANK/JQ938048
SI  - GENBANK/JQ938049
SI  - GENBANK/JQ938050
SI  - GENBANK/JQ938051
SI  - GENBANK/JQ938052
SI  - GENBANK/JQ938053
SI  - GENBANK/JQ938054
SI  - GENBANK/JQ938055
SI  - GENBANK/JQ938056
SI  - GENBANK/JQ938057
SI  - GENBANK/JQ938058
SI  - GENBANK/JQ938059
SI  - GENBANK/JQ938060
SI  - GENBANK/JQ938061
SI  - GENBANK/JQ938062
SI  - GENBANK/JQ938063
SI  - GENBANK/JQ938064
SI  - GENBANK/JQ938065
SI  - GENBANK/JQ938066
SI  - GENBANK/JQ938067
SI  - GENBANK/JQ938068
SI  - GENBANK/JQ938069
SI  - GENBANK/JQ938070
SI  - GENBANK/JQ938071
SI  - GENBANK/JQ938072
SI  - GENBANK/JQ938073
SI  - GENBANK/JQ938074
SI  - GENBANK/JQ938075
SI  - GENBANK/JQ938076
SI  - GENBANK/JQ938077
SI  - GENBANK/JQ938078
SI  - GENBANK/JQ938079
SI  - GENBANK/JQ938080
SI  - GENBANK/JQ938081
SI  - GENBANK/JQ938082
SI  - GENBANK/JQ938083
SI  - GENBANK/JQ938084
SI  - GENBANK/JQ938085
SI  - GENBANK/JQ938086
SI  - GENBANK/JQ938087
SI  - GENBANK/JQ938088
SI  - GENBANK/JQ938089
SI  - GENBANK/JQ938090
SI  - GENBANK/JQ938091
SI  - GENBANK/JQ938092
SI  - GENBANK/JQ938093
SI  - GENBANK/JQ938094
SI  - GENBANK/JQ938095
SI  - GENBANK/JQ938096
SI  - GENBANK/JQ938097
SI  - GENBANK/JQ938098
SI  - GENBANK/JQ938099
SI  - GENBANK/JQ938100
SI  - GENBANK/JQ938101
SI  - GENBANK/JQ938102
SI  - GENBANK/JQ938103
SI  - GENBANK/JQ938104
SI  - GENBANK/JQ938105
SI  - GENBANK/JQ938106
SI  - GENBANK/JQ938107
SI  - GENBANK/JQ938108
SI  - GENBANK/JQ938109
SI  - GENBANK/JQ938110
SI  - GENBANK/JQ938111
SI  - GENBANK/JQ938112
SI  - GENBANK/JQ938113
SI  - GENBANK/JQ938114
SI  - GENBANK/JQ938115
SI  - GENBANK/JQ938116
SI  - GENBANK/JQ938117
SI  - GENBANK/JQ938118
SI  - GENBANK/JQ938119
SI  - GENBANK/JQ938120
SI  - GENBANK/JQ938121
SI  - GENBANK/JQ938122
SI  - GENBANK/JQ938123
SI  - GENBANK/JQ938124
SI  - GENBANK/JQ938125
SI  - GENBANK/JQ938126
SI  - GENBANK/JQ938127
SI  - GENBANK/JQ938128
SI  - GENBANK/JQ938129
SI  - GENBANK/JQ938130
SI  - GENBANK/JQ938131
SI  - GENBANK/JQ938132
SI  - GENBANK/JQ938133
SI  - GENBANK/JQ938134
SI  - GENBANK/JQ938135
SI  - GENBANK/JQ938136
SI  - GENBANK/JQ938137
SI  - GENBANK/JQ938138
SI  - GENBANK/JQ938139
SI  - GENBANK/JQ938140
SI  - GENBANK/JQ938141
SI  - GENBANK/JQ938142
SI  - GENBANK/JQ938143
SI  - GENBANK/JQ938144
SI  - GENBANK/JQ938145
SI  - GENBANK/JQ938146
SI  - GENBANK/JQ938147
SI  - GENBANK/JQ938148
SI  - GENBANK/JQ938149
SI  - GENBANK/JQ938150
SI  - GENBANK/JQ938151
SI  - GENBANK/JQ938152
SI  - GENBANK/JQ938153
SI  - GENBANK/JQ938154
SI  - GENBANK/JQ938155
SI  - GENBANK/JQ938156
SI  - GENBANK/JQ938157
SI  - GENBANK/JQ938158
SI  - GENBANK/JQ938159
SI  - GENBANK/JQ938160
SI  - GENBANK/JQ938161
SI  - GENBANK/JQ938162
SI  - GENBANK/JQ938163
SI  - GENBANK/JQ938164
SI  - GENBANK/JQ938165
SI  - GENBANK/JQ938166
SI  - GENBANK/JQ938167
SI  - GENBANK/JQ938168
SI  - GENBANK/JQ938169
SI  - GENBANK/JQ938170
SI  - GENBANK/JQ938171
SI  - GENBANK/JQ938172
SI  - GENBANK/JQ938173
SI  - GENBANK/JQ938174
SI  - GENBANK/JQ938175
SI  - GENBANK/JQ938176
SI  - GENBANK/JQ938177
SI  - GENBANK/JQ938178
SI  - GENBANK/JQ938179
SI  - GENBANK/JQ938180
SI  - GENBANK/JQ938181
SI  - GENBANK/JQ938182
SI  - GENBANK/JQ938183
SI  - GENBANK/JQ938184
SI  - GENBANK/JQ938185
SI  - GENBANK/JQ938186
SI  - GENBANK/JQ938187
SI  - GENBANK/JQ938188
SI  - GENBANK/JQ938189
SI  - GENBANK/JQ938190
SI  - GENBANK/JQ938191
SI  - GENBANK/JQ938192
SI  - GENBANK/JQ938193
SI  - GENBANK/JQ938194
SI  - GENBANK/JQ938195
SI  - GENBANK/JQ938196
SI  - GENBANK/JQ938197
SI  - GENBANK/JQ938198
SI  - GENBANK/JQ938199
SI  - GENBANK/JQ938200
SI  - GENBANK/JQ938201
SI  - GENBANK/JQ938202
SI  - GENBANK/JQ938203
SI  - GENBANK/JQ938204
SI  - GENBANK/JQ938205
SI  - GENBANK/JQ938206
SI  - GENBANK/JQ938207
SI  - GENBANK/JQ938208
SI  - GENBANK/JQ938209
SI  - GENBANK/JQ938210
SI  - GENBANK/JQ938211
SI  - GENBANK/JQ938212
SI  - GENBANK/JQ938213
SI  - GENBANK/JQ938214
SI  - GENBANK/JQ938215
SI  - GENBANK/JQ938216
SI  - GENBANK/JQ938217
SI  - GENBANK/JQ938218
SI  - GENBANK/JQ938219
SI  - GENBANK/JQ938220
SI  - GENBANK/JQ938221
SI  - GENBANK/JQ938222
SI  - GENBANK/JQ938223
SI  - GENBANK/JQ938224
SI  - GENBANK/JQ938225
SI  - GENBANK/JQ938226
SI  - GENBANK/JQ938227
SI  - GENBANK/JQ938228
SI  - GENBANK/JQ938229
SI  - GENBANK/JQ938230
SI  - GENBANK/JQ938231
SI  - GENBANK/JQ938232
SI  - GENBANK/JQ938233
SI  - GENBANK/JQ938234
SI  - GENBANK/JQ938235
SI  - GENBANK/JQ938236
SI  - GENBANK/JQ938237
SI  - GENBANK/JQ938238
SI  - GENBANK/JQ938239
SI  - GENBANK/JQ938240
SI  - GENBANK/JQ938241
SI  - GENBANK/JQ938242
SI  - GENBANK/JQ938243
SI  - GENBANK/JQ938244
SI  - GENBANK/JQ938245
SI  - GENBANK/JQ938246
SI  - GENBANK/JQ938247
SI  - GENBANK/JQ938248
SI  - GENBANK/JQ938249
SI  - GENBANK/JQ938250
SI  - GENBANK/JQ938251
SI  - GENBANK/JQ938252
SI  - GENBANK/JQ938253
SI  - GENBANK/JQ938254
SI  - GENBANK/JQ938255
SI  - GENBANK/JQ938256
SI  - GENBANK/JQ938257
SI  - GENBANK/JQ938258
SI  - GENBANK/JQ938259
SI  - GENBANK/JQ938260
SI  - GENBANK/JQ938261
SI  - GENBANK/JQ938262
SI  - GENBANK/JQ938263
SI  - GENBANK/JQ938264
SI  - GENBANK/JQ938265
SI  - GENBANK/JQ938266
SI  - GENBANK/JQ938267
SI  - GENBANK/JQ938268
SI  - GENBANK/JQ938269
SI  - GENBANK/JQ938270
SI  - GENBANK/JQ938271
SI  - GENBANK/JQ938272
SI  - GENBANK/JQ938273
SI  - GENBANK/JQ938274
SI  - GENBANK/JQ938275
SI  - GENBANK/JQ938276
SI  - GENBANK/JQ938277
SI  - GENBANK/JQ938278
SI  - GENBANK/JQ938279
SI  - GENBANK/JQ938280
SI  - GENBANK/JQ938281
SI  - GENBANK/JQ938282
SI  - GENBANK/JQ938283
SI  - GENBANK/JQ938284
SI  - GENBANK/JQ938285
SI  - GENBANK/JQ938286
SI  - GENBANK/JQ938287
SI  - GENBANK/JQ938288
SI  - GENBANK/JQ938289
SI  - GENBANK/JQ938290
SI  - GENBANK/JQ938291
SI  - GENBANK/JQ938292
SI  - GENBANK/JQ938293
SI  - GENBANK/JQ938294
SI  - GENBANK/JQ938295
SI  - GENBANK/JQ938296
SI  - GENBANK/JQ938297
SI  - GENBANK/JQ938298
SI  - GENBANK/JQ938299
SI  - GENBANK/JQ938300
SI  - GENBANK/JQ938301
SI  - GENBANK/JQ938302
SI  - GENBANK/JQ938303
SI  - GENBANK/JQ938304
SI  - GENBANK/JQ938305
SI  - GENBANK/JQ938306
SI  - GENBANK/JQ938307
SI  - GENBANK/JQ938308
SI  - GENBANK/JQ938309
SI  - GENBANK/JQ938310
SI  - GENBANK/JQ938311
SI  - GENBANK/JQ938312
SI  - GENBANK/JQ938313
SI  - GENBANK/JQ938314
SI  - GENBANK/JQ938315
SI  - GENBANK/JQ938316
SI  - GENBANK/JQ938317
SI  - GENBANK/JQ938318
SI  - GENBANK/JQ938319
SI  - GENBANK/JQ938320
SI  - GENBANK/JQ938321
SI  - GENBANK/JQ938322
SI  - GENBANK/JQ938323
SI  - GENBANK/JQ938324
SI  - GENBANK/JQ938325
SI  - GENBANK/JQ938326
SI  - GENBANK/JQ938327
SI  - GENBANK/JQ938328
SI  - GENBANK/JQ938329
SI  - GENBANK/JQ938330
SI  - GENBANK/JQ938331
SI  - GENBANK/JQ938332
SI  - GENBANK/JQ938333
SI  - GENBANK/JQ938334
SI  - GENBANK/JQ938335
SI  - GENBANK/JQ938336
SI  - GENBANK/JQ938337
SI  - GENBANK/JQ938338
SI  - GENBANK/JQ938339
SI  - GENBANK/JQ938340
SI  - GENBANK/JQ938341
SI  - GENBANK/JQ938342
SI  - GENBANK/JQ938343
SI  - GENBANK/JQ938344
SI  - GENBANK/JQ938345
SI  - GENBANK/JQ938346
SI  - GENBANK/JQ938347
SI  - GENBANK/JQ938348
SI  - GENBANK/JQ938349
SI  - GENBANK/JQ938350
SI  - GENBANK/JQ938351
SI  - GENBANK/JQ938352
SI  - GENBANK/JQ938353
SI  - GENBANK/JQ938354
SI  - GENBANK/JQ938355
SI  - GENBANK/JQ938356
SI  - GENBANK/JQ938357
SI  - GENBANK/JQ938358
SI  - GENBANK/JQ938359
SI  - GENBANK/JQ938360
SI  - GENBANK/JQ938361
SI  - GENBANK/JQ938362
SI  - GENBANK/JQ938363
SI  - GENBANK/JQ938364
SI  - GENBANK/JQ938365
SI  - GENBANK/JQ938366
SI  - GENBANK/JQ938367
SI  - GENBANK/JQ938368
SI  - GENBANK/JQ938369
SI  - GENBANK/JQ938370
SI  - GENBANK/JQ938371
SI  - GENBANK/JQ938372
SI  - GENBANK/JQ938373
SI  - GENBANK/JQ938374
SI  - GENBANK/JQ938375
SI  - GENBANK/JQ938376
SI  - GENBANK/JQ938377
SI  - GENBANK/JQ938378
SI  - GENBANK/JQ938379
SI  - GENBANK/JQ938380
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130608
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Classification
MH  - Flatfishes/*classification/*genetics
MH  - Genome
MH  - Molecular Sequence Data
MH  - *Phylogeny
EDAT- 2013/06/12 06:00
MHDA- 2014/03/01 06:00
CRDT- 2013/06/11 06:00
PHST- 2013/06/11 06:00 [entrez]
PHST- 2013/06/12 06:00 [pubmed]
PHST- 2014/03/01 06:00 [medline]
AID - syt039 [pii]
AID - 10.1093/sysbio/syt039 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Sep;62(5):763-85. doi: 10.1093/sysbio/syt039. Epub 2013 Jun 8.

PMID- 23736104
OWN - NLM
STAT- MEDLINE
DCOM- 20140228
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 5
DP  - 2013 Sep
TI  - Parsimonious inference of hybridization in the presence of incomplete lineage
      sorting.
PG  - 738-51
LID - 10.1093/sysbio/syt037 [doi]
AB  - Hybridization plays an important evolutionary role in several groups of
      organisms. A phylogenetic approach to detect hybridization entails sequencing
      multiple loci across the genomes of a group of species of interest,
      reconstructing their gene trees, and taking their differences as indicators of
      hybridization. However, methods that follow this approach mostly ignore
      population effects, such as incomplete lineage sorting (ILS). Given that
      hybridization occurs between closely related organisms, ILS may very well be at
      play and, hence, must be accounted for in the analysis framework. To address this
      issue, we present a parsimony criterion for reconciling gene trees within the
      branches of a phylogenetic network, and a local search heuristic for inferring
      phylogenetic networks from collections of gene-tree topologies under this
      criterion. This framework enables phylogenetic analyses while accounting for both
      hybridization and ILS. Further, we propose two techniques for incorporating
      information about uncertainty in gene-tree estimates. Our simulation studies
      demonstrate the good performance of our framework in terms of identifying the
      location of hybridization events, as well as estimating the proportions of genes 
      that underwent hybridization. Also, our framework shows good performance in terms
      of efficiency on handling large data sets in our experiments. Further, in
      analysing a yeast data set, we demonstrate issues that arise when analysing real 
      data sets. Although a probabilistic approach was recently introduced for this
      problem, and although parsimonious reconciliations have accuracy issues under
      certain settings, our parsimony framework provides a much more computationally
      efficient technique for this type of analysis. Our framework now allows for
      genome-wide scans for hybridization, while also accounting for ILS.
FAU - Yu, Yun
AU  - Yu Y
AD  - Department of Computer Science, Rice University, 6100 Main Street, Houston, TX
      77005, USA.
FAU - Barnett, R Matthew
AU  - Barnett RM
FAU - Nakhleh, Luay
AU  - Nakhleh L
LA  - eng
GR  - R01LM009494/LM/NLM NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130604
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Hybridization, Genetic
MH  - *Phylogeny
MH  - Software/standards
MH  - Yeasts/classification/genetics
PMC - PMC3739885
EDAT- 2013/06/06 06:00
MHDA- 2014/03/01 06:00
CRDT- 2013/06/06 06:00
PHST- 2013/06/06 06:00 [entrez]
PHST- 2013/06/06 06:00 [pubmed]
PHST- 2014/03/01 06:00 [medline]
AID - syt037 [pii]
AID - 10.1093/sysbio/syt037 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Sep;62(5):738-51. doi: 10.1093/sysbio/syt037. Epub 2013 Jun 4.

PMID- 23736103
OWN - NLM
STAT- MEDLINE
DCOM- 20140228
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 5
DP  - 2013 Sep
TI  - When outgroups fail; phylogenomics of rooting the emerging pathogen, Coxiella
      burnetii.
PG  - 752-62
LID - 10.1093/sysbio/syt038 [doi]
AB  - Rooting phylogenies is critical for understanding evolution, yet the importance, 
      intricacies and difficulties of rooting are often overlooked. For rooting,
      polymorphic characters among the group of interest (ingroup) must be compared to 
      those of a relative (outgroup) that diverged before the last common ancestor
      (LCA) of the ingroup. Problems arise if an outgroup does not exist, is unknown,
      or is so distant that few characters are shared, in which case duplicated genes
      originating before the LCA can be used as proxy outgroups to root diverse
      phylogenies. Here, we describe a genome-wide expansion of this technique that can
      be used to solve problems at the other end of the evolutionary scale: where
      ingroup individuals are all very closely related to each other, but the next
      closest relative is very distant. We used shared orthologous single nucleotide
      polymorphisms (SNPs) from 10 whole genome sequences of Coxiella burnetii, the
      causative agent of Q fever in humans, to create a robust, but unrooted phylogeny.
      To maximize the number of characters informative about the rooting, we searched
      entire genomes for polymorphic duplicated regions where orthologs of each paralog
      could be identified so that the paralogs could be used to root the tree. Recent
      radiations, such as those of emerging pathogens, often pose rooting challenges
      due to a lack of ingroup variation and large genomic differences with known
      outgroups. Using a phylogenomic approach, we created a robust, rooted phylogeny
      for C. burnetii. [Coxiella burnetii; paralog SNPs; pathogen evolution; phylogeny;
      recent radiation; root; rooting using duplicated genes.].
FAU - Pearson, Talima
AU  - Pearson T
AD  - Center for Microbial Genetics and Genomics, Northern Arizona University,
      Flagstaff, AZ, USA.
FAU - Hornstra, Heidie M
AU  - Hornstra HM
FAU - Sahl, Jason W
AU  - Sahl JW
FAU - Schaack, Sarah
AU  - Schaack S
FAU - Schupp, James M
AU  - Schupp JM
FAU - Beckstrom-Sternberg, Stephen M
AU  - Beckstrom-Sternberg SM
FAU - O'Neill, Matthew W
AU  - O'Neill MW
FAU - Priestley, Rachael A
AU  - Priestley RA
FAU - Champion, Mia D
AU  - Champion MD
FAU - Beckstrom-Sternberg, James S
AU  - Beckstrom-Sternberg JS
FAU - Kersh, Gilbert J
AU  - Kersh GJ
FAU - Samuel, James E
AU  - Samuel JE
FAU - Massung, Robert F
AU  - Massung RF
FAU - Keim, Paul
AU  - Keim P
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130604
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Coxiella burnetii/*classification/*genetics
MH  - Genome, Bacterial/genetics
MH  - *Genomics/standards
MH  - *Phylogeny
PMC - PMC3739886
EDAT- 2013/06/06 06:00
MHDA- 2014/03/01 06:00
CRDT- 2013/06/06 06:00
PHST- 2013/06/06 06:00 [entrez]
PHST- 2013/06/06 06:00 [pubmed]
PHST- 2014/03/01 06:00 [medline]
AID - syt038 [pii]
AID - 10.1093/sysbio/syt038 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Sep;62(5):752-62. doi: 10.1093/sysbio/syt038. Epub 2013 Jun 4.

PMID- 23736102
OWN - NLM
STAT- MEDLINE
DCOM- 20140312
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 6
DP  - 2013 Nov
TI  - Bayesian analysis of biogeography when the number of areas is large.
PG  - 789-804
LID - 10.1093/sysbio/syt040 [doi]
AB  - Historical biogeography is increasingly studied from an explicitly statistical
      perspective, using stochastic models to describe the evolution of species range
      as a continuous-time Markov process of dispersal between and extinction within a 
      set of discrete geographic areas. The main constraint of these methods is the
      computational limit on the number of areas that can be specified. We propose a
      Bayesian approach for inferring biogeographic history that extends the
      application of biogeographic models to the analysis of more realistic problems
      that involve a large number of areas. Our solution is based on a
      "data-augmentation" approach, in which we first populate the tree with a history 
      of biogeographic events that is consistent with the observed species ranges at
      the tips of the tree. We then calculate the likelihood of a given history by
      adopting a mechanistic interpretation of the instantaneous-rate matrix, which
      specifies both the exponential waiting times between biogeographic events and the
      relative probabilities of each biogeographic change. We develop this approach in 
      a Bayesian framework, marginalizing over all possible biogeographic histories
      using Markov chain Monte Carlo (MCMC). Besides dramatically increasing the number
      of areas that can be accommodated in a biogeographic analysis, our method allows 
      the parameters of a given biogeographic model to be estimated and different
      biogeographic models to be objectively compared. Our approach is implemented in
      the program, BayArea.
FAU - Landis, Michael J
AU  - Landis MJ
AD  - Department of Integrative Biology, University of California, Berkeley, CA
      94720-3140, USA; Department of Evolution and Ecology, University of California,
      Davis, Storer Hall, One Shields Avenue, Davis, CA 95616, USA; and Biology
      Department, King Abdulaziz University, Jeddah, Saudi Arabia.
FAU - Matzke, Nicholas J
AU  - Matzke NJ
FAU - Moore, Brian R
AU  - Moore BR
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
LA  - eng
GR  - GM-069801/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130604
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Bayes Theorem
MH  - Computer Simulation
MH  - Phylogeny
MH  - Phylogeography/*methods
MH  - Rhododendron/classification
PMC - PMC4064008
EDAT- 2013/06/06 06:00
MHDA- 2014/03/13 06:00
CRDT- 2013/06/06 06:00
PHST- 2013/06/06 06:00 [entrez]
PHST- 2013/06/06 06:00 [pubmed]
PHST- 2014/03/13 06:00 [medline]
AID - syt040 [pii]
AID - 10.1093/sysbio/syt040 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Nov;62(6):789-804. doi: 10.1093/sysbio/syt040. Epub 2013 Jun 4.

PMID- 23681854
OWN - NLM
STAT- MEDLINE
DCOM- 20140228
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 5
DP  - 2013 Sep
TI  - Delimiting species using single-locus data and the Generalized Mixed Yule
      Coalescent approach: a revised method and evaluation on simulated data sets.
PG  - 707-24
LID - 10.1093/sysbio/syt033 [doi]
AB  - DNA barcoding-type studies assemble single-locus data from large samples of
      individuals and species, and have provided new kinds of data for evolutionary
      surveys of diversity. An important goal of many such studies is to delimit
      evolutionarily significant species units, especially in biodiversity surveys from
      environmental DNA samples. The Generalized Mixed Yule Coalescent (GMYC) method is
      a likelihood method for delimiting species by fitting within- and between-species
      branching models to reconstructed gene trees. Although the method has been widely
      used, it has not previously been described in detail or evaluated fully against
      simulations of alternative scenarios of true patterns of population variation and
      divergence between species. Here, we present important reformulations to the GMYC
      method as originally specified, and demonstrate its robustness to a range of
      departures from its simplifying assumptions. The main factor affecting the
      accuracy of delimitation is the mean population size of species relative to
      divergence times between them. Other departures from the model assumptions, such 
      as varying population sizes among species, alternative scenarios for speciation
      and extinction, and population growth or subdivision within species, have
      relatively smaller effects. Our simulations demonstrate that support measures
      derived from the likelihood function provide a robust indication of when the
      model performs well and when it leads to inaccurate delimitations. Finally, the
      so-called single-threshold version of the method outperforms the
      multiple-threshold version of the method on simulated data: we argue that this
      might represent a fundamental limit due to the nature of evidence used to delimit
      species in this approach. Together with other studies comparing its performance
      relative to other methods, our findings support the robustness of GMYC as a tool 
      for delimiting species when only single-locus information is available.
FAU - Fujisawa, Tomochika
AU  - Fujisawa T
AD  - Department of Life Sciences, Imperial College London, Silwood Park Campus, Ascot,
      Berkshire SL5 7PY, UK; and Department of Entomology, Natural History Museum,
      London SW7 5BD, UK.
FAU - Barraclough, Timothy G
AU  - Barraclough TG
LA  - eng
GR  - BB/G004250/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130516
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Animals
MH  - Classification/*methods
MH  - Coleoptera/classification/genetics
MH  - *Computer Simulation
MH  - Models, Theoretical
MH  - *Phylogeny
MH  - Species Specificity
PMC - PMC3739884
EDAT- 2013/05/18 06:00
MHDA- 2014/03/01 06:00
CRDT- 2013/05/18 06:00
PHST- 2013/05/18 06:00 [entrez]
PHST- 2013/05/18 06:00 [pubmed]
PHST- 2014/03/01 06:00 [medline]
AID - syt033 [pii]
AID - 10.1093/sysbio/syt033 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Sep;62(5):707-24. doi: 10.1093/sysbio/syt033. Epub 2013 May 16.

PMID- 23676760
OWN - NLM
STAT- MEDLINE
DCOM- 20140228
LR  - 20130812
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 5
DP  - 2013 Sep
TI  - Identifying hidden rate changes in the evolution of a binary morphological
      character: the evolution of plant habit in campanulid angiosperms.
PG  - 725-37
LID - 10.1093/sysbio/syt034 [doi]
AB  - The growth of phylogenetic trees in scope and in size is promising from the
      standpoint of understanding a wide variety of evolutionary patterns and
      processes. With trees comprised of larger, older, and globally distributed
      clades, it is likely that the lability of a binary character will differ
      significantly among lineages, which could lead to errors in estimating transition
      rates and the associated inference of ancestral states. Here we develop and
      implement a new method for identifying different rates of evolution in a binary
      character along different branches of a phylogeny. We illustrate this approach by
      exploring the evolution of growth habit in Campanulidae, a flowering plant clade 
      containing some 35,000 species. The distribution of woody versus herbaceous
      species calls into question the use of traditional models of binary character
      evolution. The recognition and accommodation of changes in the rate of growth
      form evolution in different lineages demonstrates, for the first time, a robust
      picture of growth form evolution across a very large, very old, and very
      widespread flowering plant clade.
FAU - Beaulieu, Jeremy M
AU  - Beaulieu JM
AD  - Department of Ecology and Evolutionary Biology, Yale University, P.O. Box 208106,
      New Haven, CT 10620-8106, USA; and Department of Ecology and Evolutionary
      Biology, University of Tennessee, Knoxville, TN 37996-1610, USA.
FAU - O'Meara, Brian C
AU  - O'Meara BC
FAU - Donoghue, Michael J
AU  - Donoghue MJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130514
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biodiversity
MH  - *Biological Evolution
MH  - Campanulaceae/anatomy &amp; histology/*classification
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Models, Biological
MH  - Phylogeny
MH  - Time
EDAT- 2013/05/17 06:00
MHDA- 2014/03/01 06:00
CRDT- 2013/05/17 06:00
PHST- 2013/05/17 06:00 [entrez]
PHST- 2013/05/17 06:00 [pubmed]
PHST- 2014/03/01 06:00 [medline]
AID - syt034 [pii]
AID - 10.1093/sysbio/syt034 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Sep;62(5):725-37. doi: 10.1093/sysbio/syt034. Epub 2013 May 14.

PMID- 23652347
OWN - NLM
STAT- MEDLINE
DCOM- 20140228
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 5
DP  - 2013 Sep
TI  - A semantic model for species description applied to the ensign wasps
      (hymenoptera: evaniidae) of New Caledonia.
PG  - 639-59
LID - 10.1093/sysbio/syt028 [doi]
AB  - Taxonomic descriptions are unparalleled sources of knowledge of life's phenotypic
      diversity. As natural language prose, these data sets are largely refractory to
      computation and integration with other sources of phenotypic data. By formalizing
      taxonomic descriptions using ontology-based semantic representation, we aim to
      increase the reusability and computability of taxonomists' primary data. Here, we
      present a revision of the ensign wasp (Hymenoptera: Evaniidae) fauna of New
      Caledonia using this new model for species description. Descriptive matrices,
      specimen data, and taxonomic nomenclature are gathered in a unified Web-based
      application, mx, then exported as both traditional taxonomic treatments and
      semantic statements using the OWL Web Ontology Language.
      Character:character-state combinations are then annotated following the
      entity-quality phenotype model, originally developed to represent mutant model
      organism phenotype data; concepts of anatomy are drawn from the Hymenoptera
      Anatomy Ontology and linked to phenotype descriptors from the Phenotypic Quality 
      Ontology. The resulting set of semantic statements is provided in Resource
      Description Framework format. Applying the model to real data, that is,
      specimens, taxonomic names, diagnoses, descriptions, and redescriptions, provides
      us with a foundation to discuss limitations and potential benefits such as
      automated data integration and reasoner-driven queries. Four species of ensign
      wasp are now known to occur in New Caledonia: Szepligetella levipetiolata,
      Szepligetella deercreeki Deans and Miko sp. nov., Szepligetella irwini Deans and 
      Miko sp. nov., and the nearly cosmopolitan Evania appendigaster. A fifth species,
      Szepligetella sericea, including Szepligetella impressa, syn. nov., has not yet
      been collected in New Caledonia but can be found on islands throughout the
      Pacific and so is included in the diagnostic key.
FAU - Balhoff, James P
AU  - Balhoff JP
AD  - National Evolutionary Synthesis Center, Durham, NC 27705, USA; Department of
      Biology, University of North Carolina, Chapel Hill, NC 27599, USA; Insect Museum,
      Department of Entomology, North Carolina State University, Box 7613, Raleigh, NC 
      27695, USA; Department of Entomology, Pennsylvania State University, 501 ASI
      Building, University Park, PA 16802, USA; Illinois Natural History Survey,
      University of Illinois, 1816 South Oak Street, MC 652 Champaign, IL 61820, USA;
      and Department of Ecology, Evolution, and Organismal Biology, Iowa State
      University, Ames, IA 50011, USA.
FAU - Miko, Istvan
AU  - Miko I
FAU - Yoder, Matthew J
AU  - Yoder MJ
FAU - Mullins, Patricia L
AU  - Mullins PL
FAU - Deans, Andrew R
AU  - Deans AR
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130507
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Models, Biological
MH  - New Caledonia
MH  - *Semantics
MH  - Species Specificity
MH  - Wasps/anatomy &amp; histology/*classification
PMC - PMC3739881
EDAT- 2013/05/09 06:00
MHDA- 2014/03/01 06:00
CRDT- 2013/05/09 06:00
PHST- 2013/05/09 06:00 [entrez]
PHST- 2013/05/09 06:00 [pubmed]
PHST- 2014/03/01 06:00 [medline]
AID - syt028 [pii]
AID - 10.1093/sysbio/syt028 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Sep;62(5):639-59. doi: 10.1093/sysbio/syt028. Epub 2013 May 7.

PMID- 23652346
OWN - NLM
STAT- MEDLINE
DCOM- 20140228
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 5
DP  - 2013 Sep
TI  - Inferring phylogeny and introgression using RADseq data: an example from
      flowering plants (Pedicularis: Orobanchaceae).
PG  - 689-706
LID - 10.1093/sysbio/syt032 [doi]
AB  - Phylogenetic relationships among recently diverged species are often difficult to
      resolve due to insufficient phylogenetic signal in available markers and/or
      conflict among gene trees. Here we explore the use of reduced-representation
      genome sequencing, specifically in the form of restriction-site associated DNA
      (RAD), for phylogenetic inference and the detection of ancestral hybridization in
      non-model organisms. As a case study, we investigate Pedicularis section
      Cyathophora, a systematically recalcitrant clade of flowering plants in the
      broomrape family (Orobanchaceae). Two methods of phylogenetic inference, maximum 
      likelihood and Bayesian concordance, were applied to data sets that included as
      many as 40,000 RAD loci. Both methods yielded similar topologies that included
      two major clades: a "rex-thamnophila" clade, composed of two species and several 
      subspecies with relatively low floral diversity, and geographically widespread
      distributions at lower elevations, and a "superba" clade, composed of three
      species characterized by relatively high floral diversity and isolated geographic
      distributions at higher elevations. Levels of molecular divergence between
      subspecies in the rex-thamnophila clade are similar to those between species in
      the superba clade. Using Patterson's D-statistic test, including a novel
      extension of the method that enables finer-grained resolution of introgression
      among multiple candidate taxa by removing the effect of their shared ancestry, we
      detect significant introgression among nearly all taxa in the rex-thamnophila
      clade, but not between clades or among taxa within the superba clade. These
      results suggest an important role for geographic isolation in the emergence of
      species barriers, by facilitating local adaptation and differentiation in the
      absence of homogenizing gene flow.
FAU - Eaton, Deren A R
AU  - Eaton DA
AD  - Committee on Evolutionary Biology, University of Chicago, Chicago, IL 60637, USA;
      and Botany Department, Field Museum of Natural History, Chicago, IL 60605, USA.
FAU - Ree, Richard H
AU  - Ree RH
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130507
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - DNA, Plant/genetics
MH  - Likelihood Functions
MH  - Orobanchaceae/*classification/*genetics
MH  - *Phylogeny
MH  - *Sequence Analysis, DNA
PMC - PMC3739883
EDAT- 2013/05/09 06:00
MHDA- 2014/03/01 06:00
CRDT- 2013/05/09 06:00
PHST- 2013/05/09 06:00 [entrez]
PHST- 2013/05/09 06:00 [pubmed]
PHST- 2014/03/01 06:00 [medline]
AID - syt032 [pii]
AID - 10.1093/sysbio/syt032 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Sep;62(5):689-706. doi: 10.1093/sysbio/syt032. Epub 2013 May 7.

PMID- 23628961
OWN - NLM
STAT- MEDLINE
DCOM- 20140228
LR  - 20151119
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 5
DP  - 2013 Sep
TI  - Dating phylogenies with sequentially sampled tips.
PG  - 674-88
LID - 10.1093/sysbio/syt030 [doi]
AB  - We develop a Bayesian Markov chain Monte Carlo (MCMC) algorithm for estimating
      divergence times using sequentially sampled molecular sequences. This type of
      data is commonly collected during viral epidemics and is sometimes available from
      different species in ancient DNA studies. We derive the distribution of ages of
      nodes in the tree under a birth-death-sequential-sampling (BDSS) model and use it
      as the prior for divergence times in the dating analysis. We implement the prior 
      in the MCMCtree program in the PAML package for divergence dating. The BDSS prior
      is very flexible and, with different parameters, can generate trees of very
      different shapes, suitable for examining the sensitivity of posterior time
      estimates. We apply the method to a data set of SIV/HIV-2 genes in comparison
      with a likelihood-based dating method, and to a data set of influenza H1 genes
      from different hosts in comparison with the Bayesian program BEAST. We examined
      the impact of tree topology on time estimates and suggest that multifurcating
      consensus trees should be avoided in dating analysis. We found posterior time
      estimates for old nodes to be sensitive to the priors on times and rates and
      suggest that previous Bayesian dating studies may have produced overconfident
      estimates.
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Institut fur Integrative Biologie, Eidgenossiche Technische Hochschule Zurich,
      8092 Zurich, Switzerland; Department of Biology, University College London,
      London WC1E 6BT, UK.
FAU - Yang, Ziheng
AU  - Yang Z
LA  - eng
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130428
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Bayes Theorem
MH  - Evolution, Molecular
MH  - Genetic Speciation
MH  - HIV-2/classification/genetics
MH  - Influenza A virus/classification/genetics
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Simian Immunodeficiency Virus/classification/genetics
MH  - Time
EDAT- 2013/05/01 06:00
MHDA- 2014/03/01 06:00
CRDT- 2013/05/01 06:00
PHST- 2013/05/01 06:00 [entrez]
PHST- 2013/05/01 06:00 [pubmed]
PHST- 2014/03/01 06:00 [medline]
AID - syt030 [pii]
AID - 10.1093/sysbio/syt030 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Sep;62(5):674-88. doi: 10.1093/sysbio/syt030. Epub 2013 Apr 28.

PMID- 23628960
OWN - NLM
STAT- MEDLINE
DCOM- 20140228
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 5
DP  - 2013 Sep
TI  - Bayesian tests of topology hypotheses with an example from diving beetles.
PG  - 660-73
LID - 10.1093/sysbio/syt029 [doi]
AB  - We review Bayesian approaches to model testing in general and to the assessment
      of topological hypotheses in particular. We show that the standard way of setting
      up Bayes factor tests of the monophyly of a group, or the placement of a sample
      sequence in a known reference tree, can be misleading. The reason for this is
      related to the well-known dependency of Bayes factors on model-specific priors.
      Specifically, when testing tree hypotheses it is important that each hypothesis
      is associated with an appropriate tree space in the prior. This can be achieved
      by using appropriately constrained searches or by filtering trees in the
      posterior sample, but in a more elaborate way than typically implemented. If it
      is difficult to find the appropriate tree sets to be contrasted, then the
      posterior model odds may be more informative than the Bayes factor. We illustrate
      the recommended techniques using an empirical test case addressing the issue of
      whether two genera of diving beetles (Coleoptera: Dytiscidae), Suphrodytes and
      Hydroporus, should be synonymized. Our refined Bayes factor tests, in contrast to
      standard analyses, show that there is strong support for Suphrodytes nesting
      inside Hydroporus, and the genera are therefore synonymized.
FAU - Bergsten, Johannes
AU  - Bergsten J
AD  - Department of Entomology, Swedish Museum of Natural History, Box 50007, SE-104 05
      Stockholm, Sweden; Department of Ecology and Environmental Science, Umea
      University, SE-90187 Umea, Sweden; and Department of Biodiversity Informatics,
      Swedish Museum of Natural History, Box 50007, SE-104 05 Stockholm, Sweden.
FAU - Nilsson, Anders N
AU  - Nilsson AN
FAU - Ronquist, Fredrik
AU  - Ronquist F
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Review
DEP - 20130428
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Coleoptera/*classification
MH  - *Phylogeny
PMC - PMC3739882
EDAT- 2013/05/01 06:00
MHDA- 2014/03/01 06:00
CRDT- 2013/05/01 06:00
PHST- 2013/05/01 06:00 [entrez]
PHST- 2013/05/01 06:00 [pubmed]
PHST- 2014/03/01 06:00 [medline]
AID - syt029 [pii]
AID - 10.1093/sysbio/syt029 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Sep;62(5):660-73. doi: 10.1093/sysbio/syt029. Epub 2013 Apr 28.

PMID- 23619174
OWN - NLM
STAT- MEDLINE
DCOM- 20140108
LR  - 20130610
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 4
DP  - 2013 Jul
TI  - Nodes, branches, and phylogenetic definitions.
PG  - 625-32
LID - 10.1093/sysbio/syt027 [doi]
FAU - de Queiroz, Kevin
AU  - de Queiroz K
AD  - Department of Vertebrate Zoology, National Museum of Natural History, Smithsonian
      Institution, Washington, DC 20560, USA. dequeirozk@si.edu
LA  - eng
PT  - Journal Article
DEP - 20130424
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Phylogeny
MH  - Terminology as Topic
EDAT- 2013/04/27 06:00
MHDA- 2014/01/09 06:00
CRDT- 2013/04/27 06:00
PHST- 2013/04/27 06:00 [entrez]
PHST- 2013/04/27 06:00 [pubmed]
PHST- 2014/01/09 06:00 [medline]
AID - syt027 [pii]
AID - 10.1093/sysbio/syt027 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jul;62(4):625-32. doi: 10.1093/sysbio/syt027. Epub 2013 Apr 24.

PMID- 23589497
OWN - NLM
STAT- MEDLINE
DCOM- 20140108
LR  - 20130610
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 4
DP  - 2013 Jul
TI  - Evolutionary covariation in geometric morphometric data: analyzing integration,
      modularity, and allometry in a phylogenetic context.
PG  - 591-610
LID - 10.1093/sysbio/syt025 [doi]
AB  - Quantifying integration and modularity of evolutionary changes in morphometric
      traits is crucial for understanding how organismal shapes evolve. For this
      purpose, comparative studies are necessary, which need to take into account the
      phylogenetic structure of interspecific data. This study applies several of the
      standard tools of geometric morphometrics, which mostly have been used in
      intraspecific studies, in the new context of analyzing integration and modularity
      based on comparative data. Morphometric methods such as principal component
      analysis, multivariate regression, partial least squares, and modularity tests
      can be applied to phylogenetically independent contrasts of shape data. We
      illustrate this approach in an analysis of cranial evolution in 160 species from 
      all orders of birds. Mapping the shape information onto the phylogeny indicates
      that there is a significant phylogenetic signal in skull shape. Multivariate
      regression of independent contrasts of shape on independent contrasts of size
      reveals clear evolutionary allometry. Regardless of whether or not a correction
      for allometry is used, evolutionary integration between the face and braincase is
      strong, and tests reject the hypothesis that the face and braincase are separate 
      evolutionary modules. These analyses can easily be applied to other taxa and can 
      be combined with other morphometric tools to address a wide range of questions
      about evolutionary patterns and processes.
FAU - Klingenberg, Christian Peter
AU  - Klingenberg CP
AD  - Faculty of Life Sciences, University of Manchester, Michael Smith Building,
      Oxford Road, Manchester M13 9PT, UK. cpk@manchester.ac.uk
FAU - Marugan-Lobon, Jesus
AU  - Marugan-Lobon J
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130414
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Birds/*anatomy &amp; histology/*genetics
MH  - Least-Squares Analysis
MH  - Multivariate Analysis
MH  - *Phylogeny
MH  - Principal Component Analysis
MH  - Skull/anatomy &amp; histology
EDAT- 2013/04/17 06:00
MHDA- 2014/01/09 06:00
CRDT- 2013/04/17 06:00
PHST- 2013/04/17 06:00 [entrez]
PHST- 2013/04/17 06:00 [pubmed]
PHST- 2014/01/09 06:00 [medline]
AID - syt025 [pii]
AID - 10.1093/sysbio/syt025 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jul;62(4):591-610. doi: 10.1093/sysbio/syt025. Epub 2013 Apr 14.

PMID- 23576318
OWN - NLM
STAT- MEDLINE
DCOM- 20140108
LR  - 20130610
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 4
DP  - 2013 Jul
TI  - Anomalous unrooted gene trees.
PG  - 574-90
LID - 10.1093/sysbio/syt023 [doi]
AB  - The coalescent and multispecies coalescent model rooted genealogies backward
      through time. Often, the direction of time is unknown in trees estimated from
      molecular sequences due to reversible mutation models, absence of an appropriate 
      outgroup, and the absence of the molecular clock. In this article, probabilities 
      of unrooted gene-tree topologies under the multispecies coalescent are
      considered. The main result is that for any species-tree topology with seven or
      more taxa, there exist branch lengths such that there are unrooted gene-tree
      topologies that are more likely than the one that has the same unrooted topology 
      as the species tree. Species trees with such anomalous unrooted gene trees
      (AUGTs) are characterized for trees with five and six taxa, and patterns of
      branch lengths leading to AUGTs and rooted anomalous gene trees are explored. The
      results could be useful for understanding gene tree discordance and designing
      simulations studies for inferring challenging species trees.
FAU - Degnan, James H
AU  - Degnan JH
AD  - Department of Mathematics and Statistics, University of Canterbury, Private Bag
      4800, Christchurch, New Zealand. j.degnan@math.canterbury.ac.nz
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130410
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Computational Biology
MH  - *Evolution, Molecular
MH  - Genetic Speciation
MH  - Models, Genetic
MH  - Pedigree
MH  - *Phylogeny
MH  - Probability
EDAT- 2013/04/12 06:00
MHDA- 2014/01/09 06:00
CRDT- 2013/04/12 06:00
PHST- 2013/04/12 06:00 [entrez]
PHST- 2013/04/12 06:00 [pubmed]
PHST- 2014/01/09 06:00 [medline]
AID - syt023 [pii]
AID - 10.1093/sysbio/syt023 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jul;62(4):574-90. doi: 10.1093/sysbio/syt023. Epub 2013 Apr 10.

PMID- 23576317
OWN - NLM
STAT- MEDLINE
DCOM- 20140108
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 4
DP  - 2013 Jul
TI  - More taxonomists describing significantly fewer species per unit effort may
      indicate that most species have been discovered.
PG  - 616-24
LID - 10.1093/sysbio/syt024 [doi]
FAU - Costello, Mark J
AU  - Costello MJ
AD  - Institute of Marine Science, University of Auckland, PO Box 349, Warkworth 0941, 
      New Zealand. mcostello@auckland.ac.nz
FAU - Wilson, Simon
AU  - Wilson S
FAU - Houlding, Brett
AU  - Houlding B
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130410
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Archaea/classification
MH  - Bacteria/classification
MH  - *Biodiversity
MH  - Classification/*methods
MH  - Conservation of Natural Resources
MH  - Eukaryota/classification
MH  - Insecta/classification
EDAT- 2013/04/12 06:00
MHDA- 2014/01/09 06:00
CRDT- 2013/04/12 06:00
PHST- 2013/04/12 06:00 [entrez]
PHST- 2013/04/12 06:00 [pubmed]
PHST- 2014/01/09 06:00 [medline]
AID - syt024 [pii]
AID - 10.1093/sysbio/syt024 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jul;62(4):616-24. doi: 10.1093/sysbio/syt024. Epub 2013 Apr 10.

PMID- 23572489
OWN - NLM
STAT- MEDLINE
DCOM- 20140108
LR  - 20130610
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 4
DP  - 2013 Jul
TI  - New approaches to the biogeography and areas of endemism of red oaks (Quercus L.,
      section Lobatae).
PG  - 555-73
LID - 10.1093/sysbio/syt021 [doi]
AB  - An area of endemism is defined by the spatial congruence among two or more
      species with distributions that are limited by barriers. In this study, we
      explored and discussed the use of the network analysis method (NAM) and
      neighbor-joining (NJ) to analyze the areas of endemism of Quercus sect. Lobatae
      (red oak species) in Mexico and Central America. We compared the NAM and NJ with 
      other methods commonly used in biogeographic studies to show the advantages of
      these new approaches and to identify the shortcomings of other approaches. The
      NAM used in this study is based on notions of centrality measures, such as
      betweenness. We incorporated the strength of the ties within the internal
      networks through p-cores and aggregate constraints in iterative analyses. The NAM
      based on betweenness is ideal for recognizing completely allopatric areas of
      endemism. The iterative NAMs increase the number of possible areas of endemism
      because they minimize the effect of minimal overlap, and the p-core is efficient 
      at identifying the closest relationships among species in the cases in which
      betweenness is not informative. The number of areas of endemism increases when
      the sympatry matrix minimizes the dispersal effect and the sample effort is
      maximized, allowing the identification of the greatest number of these areas. The
      NJ method supports the idea that areas diverge among themselves in a differential
      way; the long branches correspond to zones with high speciation rates and complex
      histories (biotic and tectonic), and the short branches correspond to zones with 
      low speciation rates and simple histories. In a classification scheme, NJ was
      capable of identifying the areas that are considered biotically complex because
      of their high speciation rates. The results obtained with the NAM and NJ showed
      that the physiographic regions of Mexico are not natural units and that many of
      them are composed of at least two different biotic components.
FAU - Torres-Miranda, Andres
AU  - Torres-Miranda A
AD  - Centro de Investigaciones en Ecosistemas, Universidad Nacional Autonoma de
      Mexico-UNAM, Ant. Carr. a Patzcuaro 8701, Col. Ex-Hda. de San Jose de la Huerta, 
      58190 Morelia, Michoacan, Mexico.
FAU - Luna-Vega, Isolda
AU  - Luna-Vega I
FAU - Oyama, Ken
AU  - Oyama K
LA  - eng
PT  - Comparative Study
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130409
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biodiversity
MH  - Central America
MH  - Geography/*methods
MH  - Mexico
MH  - *Models, Biological
MH  - *Plant Dispersal
MH  - Quercus/*physiology
EDAT- 2013/04/11 06:00
MHDA- 2014/01/09 06:00
CRDT- 2013/04/11 06:00
PHST- 2013/04/11 06:00 [entrez]
PHST- 2013/04/11 06:00 [pubmed]
PHST- 2014/01/09 06:00 [medline]
AID - syt021 [pii]
AID - 10.1093/sysbio/syt021 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jul;62(4):555-73. doi: 10.1093/sysbio/syt021. Epub 2013 Apr 9.

PMID- 23564032
OWN - NLM
STAT- MEDLINE
DCOM- 20140108
LR  - 20130610
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 4
DP  - 2013 Jul
TI  - PhyloBayes MPI: phylogenetic reconstruction with infinite mixtures of profiles in
      a parallel environment.
PG  - 611-5
LID - 10.1093/sysbio/syt022 [doi]
AB  - Modeling across site variation of the substitution process is increasingly
      recognized as important for obtaining more accurate phylogenetic reconstructions.
      Both finite and infinite mixture models have been proposed and have been shown to
      significantly improve on classical single-matrix models. Compared with their
      finite counterparts, infinite mixtures have a greater expressivity. However, they
      are computationally more challenging. This has resulted in practical compromises 
      in the design of infinite mixture models. In particular, a fast but simplified
      version of a Dirichlet process model over equilibrium frequency profiles
      implemented in PhyloBayes has often been used in recent phylogenomics studies,
      while more refined model structures, more realistic and empirically more fit,
      have been practically out of reach. We introduce a message passing interface
      version of PhyloBayes, implementing the Dirichlet process mixture models as well 
      as more classical empirical matrices and finite mixtures. The parallelization is 
      made efficient thanks to the combination of two algorithmic strategies: a partial
      Gibbs sampling update of the tree topology and the use of a truncated
      stick-breaking representation for the Dirichlet process prior. The implementation
      shows close to linear gains in computational speed for up to 64 cores, thus
      allowing faster phylogenetic reconstruction under complex mixture models.
      PhyloBayes MPI is freely available from our website www.phylobayes.org.
FAU - Lartillot, Nicolas
AU  - Lartillot N
AD  - Centre Robert Cedergren pour la Bioinformatique, Departement de Biochimie,
      Universite de Montreal, C.P. 6128, Succursale Centre-ville. Montreal, Quebec H3C 
      3J7, Canada. nicolas.lartillot@umontreal.ca
FAU - Rodrigue, Nicolas
AU  - Rodrigue N
FAU - Stubbs, Daniel
AU  - Stubbs D
FAU - Richer, Jacques
AU  - Richer J
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130405
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Bayes Theorem
MH  - Computational Biology/*methods
MH  - Models, Genetic
MH  - *Phylogeny
MH  - *Software
EDAT- 2013/04/09 06:00
MHDA- 2014/01/09 06:00
CRDT- 2013/04/09 06:00
PHST- 2013/04/09 06:00 [entrez]
PHST- 2013/04/09 06:00 [pubmed]
PHST- 2014/01/09 06:00 [medline]
AID - syt022 [pii]
AID - 10.1093/sysbio/syt022 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jul;62(4):611-5. doi: 10.1093/sysbio/syt022. Epub 2013 Apr 5.

PMID- 23503595
OWN - NLM
STAT- MEDLINE
DCOM- 20140108
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 4
DP  - 2013 Jul
TI  - Next-generation museomics disentangles one of the largest primate radiations.
PG  - 539-54
LID - 10.1093/sysbio/syt018 [doi]
AB  - Guenons (tribe Cercopithecini) are one of the most diverse groups of primates.
      They occupy all of sub-Saharan Africa and show great variation in ecology,
      behavior, and morphology. This variation led to the description of over 60
      species and subspecies. Here, using next-generation DNA sequencing (NGS) in
      combination with targeted DNA capture, we sequenced 92 mitochondrial genomes from
      museum-preserved specimens as old as 117 years. We infer evolutionary
      relationships and estimate divergence times of almost all guenon taxa based on
      mitochondrial genome sequences. Using this phylogenetic framework, we infer
      divergence dates and reconstruct ancestral geographic ranges. We conclude that
      the extraordinary radiation of guenons has been a complex process driven by,
      among other factors, localized fluctuations of African forest cover. We find
      incongruences between phylogenetic trees reconstructed from mitochondrial and
      nuclear DNA sequences, which can be explained by either incomplete lineage
      sorting or hybridization. Furthermore, having produced the largest mitochondrial 
      DNA data set from museum specimens, we document how NGS technologies can "unlock"
      museum collections, thereby helping to unravel the tree-of-life.
FAU - Guschanski, Katerina
AU  - Guschanski K
AD  - Imperial College London, Silwood Park Campus, Buckhurst Road, Ascot SL5 7PY, UK.
FAU - Krause, Johannes
AU  - Krause J
FAU - Sawyer, Susanna
AU  - Sawyer S
FAU - Valente, Luis M
AU  - Valente LM
FAU - Bailey, Sebastian
AU  - Bailey S
FAU - Finstermeier, Knut
AU  - Finstermeier K
FAU - Sabin, Richard
AU  - Sabin R
FAU - Gilissen, Emmanuel
AU  - Gilissen E
FAU - Sonet, Gontran
AU  - Sonet G
FAU - Nagy, Zoltan T
AU  - Nagy ZT
FAU - Lenglet, Georges
AU  - Lenglet G
FAU - Mayer, Frieder
AU  - Mayer F
FAU - Savolainen, Vincent
AU  - Savolainen V
LA  - eng
SI  - GENBANK/JQ256909
SI  - GENBANK/JQ256910
SI  - GENBANK/JQ256911
SI  - GENBANK/JQ256912
SI  - GENBANK/JQ256913
SI  - GENBANK/JQ256914
SI  - GENBANK/JQ256915
SI  - GENBANK/JQ256916
SI  - GENBANK/JQ256917
SI  - GENBANK/JQ256918
SI  - GENBANK/JQ256919
SI  - GENBANK/JQ256920
SI  - GENBANK/JQ256921
SI  - GENBANK/JQ256922
SI  - GENBANK/JQ256923
SI  - GENBANK/JQ256924
SI  - GENBANK/JQ256925
SI  - GENBANK/JQ256926
SI  - GENBANK/JQ256927
SI  - GENBANK/JQ256928
SI  - GENBANK/JQ256929
SI  - GENBANK/JQ256930
SI  - GENBANK/JQ256931
SI  - GENBANK/JQ256932
SI  - GENBANK/JQ256933
SI  - GENBANK/JQ256934
SI  - GENBANK/JQ256935
SI  - GENBANK/JQ256936
SI  - GENBANK/JQ256937
SI  - GENBANK/JQ256938
SI  - GENBANK/JQ256939
SI  - GENBANK/JQ256940
SI  - GENBANK/JQ256941
SI  - GENBANK/JQ256942
SI  - GENBANK/JQ256943
SI  - GENBANK/JQ256944
SI  - GENBANK/JQ256945
SI  - GENBANK/JQ256946
SI  - GENBANK/JQ256947
SI  - GENBANK/JQ256948
SI  - GENBANK/JQ256949
SI  - GENBANK/JQ256950
SI  - GENBANK/JQ256951
SI  - GENBANK/JQ256952
SI  - GENBANK/JQ256953
SI  - GENBANK/JQ256954
SI  - GENBANK/JQ256955
SI  - GENBANK/JQ256956
SI  - GENBANK/JQ256957
SI  - GENBANK/JQ256958
SI  - GENBANK/JQ256959
SI  - GENBANK/JQ256960
SI  - GENBANK/JQ256961
SI  - GENBANK/JQ256962
SI  - GENBANK/JQ256963
SI  - GENBANK/JQ256964
SI  - GENBANK/JQ256965
SI  - GENBANK/JQ256966
SI  - GENBANK/JQ256967
SI  - GENBANK/JQ256968
SI  - GENBANK/JQ256969
SI  - GENBANK/JQ256970
SI  - GENBANK/JQ256971
SI  - GENBANK/JQ256972
SI  - GENBANK/JQ256973
SI  - GENBANK/JQ256974
SI  - GENBANK/JQ256975
SI  - GENBANK/JQ256976
SI  - GENBANK/JQ256977
SI  - GENBANK/JQ256978
SI  - GENBANK/JQ256979
SI  - GENBANK/JQ256980
SI  - GENBANK/JQ256981
SI  - GENBANK/JQ256982
SI  - GENBANK/JQ256983
SI  - GENBANK/JQ256984
SI  - GENBANK/JQ256985
SI  - GENBANK/JQ256986
SI  - GENBANK/JQ256987
SI  - GENBANK/JQ256988
SI  - GENBANK/JQ256989
SI  - GENBANK/JQ256990
SI  - GENBANK/JQ256991
SI  - GENBANK/JQ256992
SI  - GENBANK/JQ256993
SI  - GENBANK/JQ256994
SI  - GENBANK/JQ256995
SI  - GENBANK/JQ256996
SI  - GENBANK/JQ256997
SI  - GENBANK/JQ256998
SI  - GENBANK/JQ256999
SI  - GENBANK/JQ257000
SI  - GENBANK/JQ257001
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130316
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Cercopithecinae/*classification/*genetics/metabolism
MH  - Conservation of Natural Resources
MH  - DNA, Mitochondrial/genetics/metabolism
MH  - *Evolution, Molecular
MH  - Genome, Mitochondrial
MH  - High-Throughput Nucleotide Sequencing
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Polymerase Chain Reaction
MH  - Sequence Analysis, DNA
PMC - PMC3676678
EDAT- 2013/03/19 06:00
MHDA- 2014/01/09 06:00
CRDT- 2013/03/19 06:00
PHST- 2013/03/19 06:00 [entrez]
PHST- 2013/03/19 06:00 [pubmed]
PHST- 2014/01/09 06:00 [medline]
AID - syt018 [pii]
AID - 10.1093/sysbio/syt018 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jul;62(4):539-54. doi: 10.1093/sysbio/syt018. Epub 2013 Mar 16.

PMID- 23479066
OWN - NLM
STAT- MEDLINE
DCOM- 20140108
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 4
DP  - 2013 Jul
TI  - The estimation of tree posterior probabilities using conditional clade
      probability distributions.
PG  - 501-11
LID - 10.1093/sysbio/syt014 [doi]
AB  - In this article I introduce the idea of conditional independence of separated
      subtrees as a principle by which to estimate the posterior probability of trees
      using conditional clade probability distributions rather than simple sample
      relative frequencies. I describe an algorithm for these calculations and software
      which implements these ideas. I show that these alternative calculations are very
      similar to simple sample relative frequencies for high probability trees but are 
      substantially more accurate for relatively low probability trees. The method
      allows the posterior probability of unsampled trees to be calculated when these
      trees contain only clades that are in other sampled trees. Furthermore, the
      method can be used to estimate the total probability of the set of sampled trees 
      which provides a measure of the thoroughness of a posterior sample.
FAU - Larget, Bret
AU  - Larget B
AD  - Departments of Botany and Statistics, University of Wisconsin, Madison, WI 53706,
      USA. brlarget@wisc.edu
LA  - eng
GR  - 1R01 GM086887/GM/NIGMS NIH HHS/United States
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130311
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Carnivora/genetics/metabolism
MH  - Computational Biology/*methods
MH  - Electron Transport Complex IV/genetics/metabolism
MH  - Models, Genetic
MH  - *Phylogeny
MH  - *Probability
MH  - Software
PMC - PMC3676676
EDAT- 2013/03/13 06:00
MHDA- 2014/01/09 06:00
CRDT- 2013/03/13 06:00
PHST- 2013/03/13 06:00 [entrez]
PHST- 2013/03/13 06:00 [pubmed]
PHST- 2014/01/09 06:00 [medline]
AID - syt014 [pii]
AID - 10.1093/sysbio/syt014 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jul;62(4):501-11. doi: 10.1093/sysbio/syt014. Epub 2013 Mar 11.

PMID- 23475623
OWN - NLM
STAT- MEDLINE
DCOM- 20140108
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 4
DP  - 2013 Jul
TI  - A branch-heterogeneous model of protein evolution for efficient inference of
      ancestral sequences.
PG  - 523-38
LID - 10.1093/sysbio/syt016 [doi]
AB  - Most models of nucleotide or amino acid substitution used in phylogenetic studies
      assume that the evolutionary process has been homogeneous across lineages and
      that composition of nucleotides or amino acids has remained the same throughout
      the tree. These oversimplified assumptions are refuted by the observation that
      compositional variability characterizes extant biological sequences.
      Branch-heterogeneous models of protein evolution that account for compositional
      variability have been developed, but are not yet in common use because of the
      large number of parameters required, leading to high computational costs and
      potential overparameterization. Here, we present a new branch-nonhomogeneous and 
      nonstationary model of protein evolution that captures more accurately the high
      complexity of sequence evolution. This model, henceforth called Correspondence
      and likelihood analysis (COaLA), makes use of a correspondence analysis to reduce
      the number of parameters to be optimized through maximum likelihood, focusing on 
      most of the compositional variation observed in the data. The model was
      thoroughly tested on both simulated and biological data sets to show its high
      performance in terms of data fitting and CPU time. COaLA efficiently estimates
      ancestral amino acid frequencies and sequences, making it relevant for studies
      aiming at reconstructing and resurrecting ancestral amino acid sequences.
      Finally, we applied COaLA on a concatenate of universal amino acid sequences to
      confirm previous results obtained with a nonhomogeneous Bayesian model regarding 
      the early pattern of adaptation to optimal growth temperature, supporting the
      mesophilic nature of the Last Universal Common Ancestor.
FAU - Groussin, M
AU  - Groussin M
AD  - Laboratoire de Biometrie et Biologie Evolutive, Universite de Lyon, Universite
      Lyon 1, CNRS, UMR5558, Villeurbanne, France. mathieu.groussin@univ-lyon1.fr
FAU - Boussau, B
AU  - Boussau B
FAU - Gouy, M
AU  - Gouy M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130307
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Archaeal Proteins)
RN  - 0 (Bacterial Proteins)
RN  - 0 (Proteins)
SB  - IM
MH  - Archaea/genetics/metabolism
MH  - Archaeal Proteins/genetics/metabolism
MH  - Bacteria/genetics/metabolism
MH  - Bacterial Proteins/genetics/metabolism
MH  - Computer Simulation
MH  - Eukaryota/genetics/metabolism
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Phylogeny
MH  - Proteins/*genetics/metabolism
MH  - Sequence Alignment
MH  - Sequence Analysis, Protein
MH  - Sequence Homology
PMC - PMC3676677
EDAT- 2013/03/12 06:00
MHDA- 2014/01/09 06:00
CRDT- 2013/03/12 06:00
PHST- 2013/03/12 06:00 [entrez]
PHST- 2013/03/12 06:00 [pubmed]
PHST- 2014/01/09 06:00 [medline]
AID - syt016 [pii]
AID - 10.1093/sysbio/syt016 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jul;62(4):523-38. doi: 10.1093/sysbio/syt016. Epub 2013 Mar 7.

PMID- 23461879
OWN - NLM
STAT- MEDLINE
DCOM- 20140108
LR  - 20130610
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 4
DP  - 2013 Jul
TI  - Timetree of Aselloidea reveals species diversification dynamics in groundwater.
PG  - 512-22
LID - 10.1093/sysbio/syt015 [doi]
AB  - A key challenge for biologists is to document and explain global patterns of
      diversification in a wide range of environments. Here, we explore patterns of
      continental-scale diversification in a groundwater species-rich clade, the
      superfamily Aselloidea (Pancrustacea: Isopoda). Our analyses supported a constant
      diversification rate during most of the course of Aselloidea evolution, until
      4-15 Ma when diversification rates started to decrease. This constant
      accumulation of lineages challenges the view that groundwater species
      diversification in temperate regions might have been primarily driven by major
      changes in physical environment leading to the extinction of surface populations 
      and subsequent synchronous isolation of multiple groundwater populations. Rather 
      than acting synchronously over broad geographic regions, factors causing
      extinction of surface populations and subsequent reproductive isolation of
      groundwater populations may act in a local and asynchronous manner, thereby
      resulting in a constant speciation rate over time. Our phylogeny also revealed
      several cases of parapatric distributions among closely related surface-water and
      groundwater species suggesting that species diversification could also arise from
      a process of disruptive selection along the surface-subterranean environmental
      gradient. Our results call for re-evaluating the spatial scale and timing of
      factors causing diversification events in groundwater.
FAU - Morvan, Claire
AU  - Morvan C
AD  - UMR5023 Ecologie des Hydrosystemes Naturels et Anthropises, Universite Lyon 1,
      Universite de Lyon, ENTPE, CNRS, 6 Rue Raphael Dubois, 69622 Villeurbanne,
      France.
FAU - Malard, Florian
AU  - Malard F
FAU - Paradis, Emmanuel
AU  - Paradis E
FAU - Lefebure, Tristan
AU  - Lefebure T
FAU - Konecny-Dupre, Lara
AU  - Konecny-Dupre L
FAU - Douady, Christophe J
AU  - Douady CJ
LA  - eng
SI  - GENBANK/JQ921017
SI  - GENBANK/JQ921018
SI  - GENBANK/JQ921019
SI  - GENBANK/JQ921020
SI  - GENBANK/JQ921021
SI  - GENBANK/JQ921022
SI  - GENBANK/JQ921023
SI  - GENBANK/JQ921024
SI  - GENBANK/JQ921025
SI  - GENBANK/JQ921026
SI  - GENBANK/JQ921027
SI  - GENBANK/JQ921028
SI  - GENBANK/JQ921029
SI  - GENBANK/JQ921030
SI  - GENBANK/JQ921031
SI  - GENBANK/JQ921032
SI  - GENBANK/JQ921033
SI  - GENBANK/JQ921034
SI  - GENBANK/JQ921035
SI  - GENBANK/JQ921036
SI  - GENBANK/JQ921037
SI  - GENBANK/JQ921038
SI  - GENBANK/JQ921039
SI  - GENBANK/JQ921040
SI  - GENBANK/JQ921041
SI  - GENBANK/JQ921042
SI  - GENBANK/JQ921043
SI  - GENBANK/JQ921044
SI  - GENBANK/JQ921045
SI  - GENBANK/JQ921046
SI  - GENBANK/JQ921047
SI  - GENBANK/JQ921048
SI  - GENBANK/JQ921049
SI  - GENBANK/JQ921050
SI  - GENBANK/JQ921051
SI  - GENBANK/JQ921052
SI  - GENBANK/JQ921053
SI  - GENBANK/JQ921054
SI  - GENBANK/JQ921055
SI  - GENBANK/JQ921056
SI  - GENBANK/JQ921057
SI  - GENBANK/JQ921058
SI  - GENBANK/JQ921059
SI  - GENBANK/JQ921060
SI  - GENBANK/JQ921061
SI  - GENBANK/JQ921062
SI  - GENBANK/JQ921063
SI  - GENBANK/JQ921064
SI  - GENBANK/JQ921065
SI  - GENBANK/JQ921066
SI  - GENBANK/JQ921067
SI  - GENBANK/JQ921068
SI  - GENBANK/JQ921069
SI  - GENBANK/JQ921070
SI  - GENBANK/JQ921071
SI  - GENBANK/JQ921072
SI  - GENBANK/JQ921073
SI  - GENBANK/JQ921074
SI  - GENBANK/JQ921075
SI  - GENBANK/JQ921076
SI  - GENBANK/JQ921077
SI  - GENBANK/JQ921078
SI  - GENBANK/JQ921079
SI  - GENBANK/JQ921080
SI  - GENBANK/JQ921081
SI  - GENBANK/JQ921082
SI  - GENBANK/JQ921083
SI  - GENBANK/JQ921084
SI  - GENBANK/JQ921085
SI  - GENBANK/JQ921086
SI  - GENBANK/JQ921087
SI  - GENBANK/JQ921088
SI  - GENBANK/JQ921089
SI  - GENBANK/JQ921090
SI  - GENBANK/JQ921091
SI  - GENBANK/JQ921092
SI  - GENBANK/JQ921093
SI  - GENBANK/JQ921094
SI  - GENBANK/JQ921095
SI  - GENBANK/JQ921096
SI  - GENBANK/JQ921097
SI  - GENBANK/JQ921098
SI  - GENBANK/JQ921099
SI  - GENBANK/JQ921100
SI  - GENBANK/JQ921101
SI  - GENBANK/JQ921102
SI  - GENBANK/JQ921103
SI  - GENBANK/JQ921104
SI  - GENBANK/JQ921105
SI  - GENBANK/JQ921106
SI  - GENBANK/JQ921107
SI  - GENBANK/JQ921108
SI  - GENBANK/JQ921109
SI  - GENBANK/JQ921110
SI  - GENBANK/JQ921111
SI  - GENBANK/JQ921112
SI  - GENBANK/JQ921113
SI  - GENBANK/JQ921114
SI  - GENBANK/JQ921115
SI  - GENBANK/JQ921116
SI  - GENBANK/JQ921117
SI  - GENBANK/JQ921118
SI  - GENBANK/JQ921119
SI  - GENBANK/JQ921120
SI  - GENBANK/JQ921121
SI  - GENBANK/JQ921122
SI  - GENBANK/JQ921123
SI  - GENBANK/JQ921124
SI  - GENBANK/JQ921125
SI  - GENBANK/JQ921126
SI  - GENBANK/JQ921127
SI  - GENBANK/JQ921128
SI  - GENBANK/JQ921129
SI  - GENBANK/JQ921130
SI  - GENBANK/JQ921131
SI  - GENBANK/JQ921132
SI  - GENBANK/JQ921133
SI  - GENBANK/JQ921134
SI  - GENBANK/JQ921135
SI  - GENBANK/JQ921136
SI  - GENBANK/JQ921137
SI  - GENBANK/JQ921138
SI  - GENBANK/JQ921139
SI  - GENBANK/JQ921140
SI  - GENBANK/JQ921141
SI  - GENBANK/JQ921142
SI  - GENBANK/JQ921143
SI  - GENBANK/JQ921144
SI  - GENBANK/JQ921145
SI  - GENBANK/JQ921146
SI  - GENBANK/JQ921147
SI  - GENBANK/JQ921148
SI  - GENBANK/JQ921149
SI  - GENBANK/JQ921150
SI  - GENBANK/JQ921151
SI  - GENBANK/JQ921152
SI  - GENBANK/JQ921153
SI  - GENBANK/JQ921154
SI  - GENBANK/JQ921155
SI  - GENBANK/JQ921156
SI  - GENBANK/JQ921157
SI  - GENBANK/JQ921158
SI  - GENBANK/JQ921159
SI  - GENBANK/JQ921160
SI  - GENBANK/JQ921161
SI  - GENBANK/JQ921162
SI  - GENBANK/JQ921163
SI  - GENBANK/JQ921164
SI  - GENBANK/JQ921165
SI  - GENBANK/JQ921166
SI  - GENBANK/JQ921167
SI  - GENBANK/JQ921168
SI  - GENBANK/JQ921169
SI  - GENBANK/JQ921170
SI  - GENBANK/JQ921171
SI  - GENBANK/JQ921172
SI  - GENBANK/JQ921173
SI  - GENBANK/JQ921174
SI  - GENBANK/JQ921175
SI  - GENBANK/JQ921176
SI  - GENBANK/JQ921177
SI  - GENBANK/JQ921178
SI  - GENBANK/JQ921179
SI  - GENBANK/JQ921180
SI  - GENBANK/JQ921181
SI  - GENBANK/JQ921182
SI  - GENBANK/JQ921183
SI  - GENBANK/JQ921184
SI  - GENBANK/JQ921185
SI  - GENBANK/JQ921186
SI  - GENBANK/JQ921187
SI  - GENBANK/JQ921188
SI  - GENBANK/JQ921189
SI  - GENBANK/JQ921190
SI  - GENBANK/JQ921191
SI  - GENBANK/JQ921192
SI  - GENBANK/JQ921193
SI  - GENBANK/JQ921194
SI  - GENBANK/JQ921195
SI  - GENBANK/JQ921196
SI  - GENBANK/JQ921197
SI  - GENBANK/JQ921198
SI  - GENBANK/JQ921199
SI  - GENBANK/JQ921200
SI  - GENBANK/JQ921201
SI  - GENBANK/JQ921202
SI  - GENBANK/JQ921203
SI  - GENBANK/JQ921204
SI  - GENBANK/JQ921205
SI  - GENBANK/JQ921206
SI  - GENBANK/JQ921207
SI  - GENBANK/JQ921208
SI  - GENBANK/JQ921209
SI  - GENBANK/JQ921210
SI  - GENBANK/JQ921211
SI  - GENBANK/JQ921212
SI  - GENBANK/JQ921213
SI  - GENBANK/JQ921214
SI  - GENBANK/JQ921215
SI  - GENBANK/JQ921216
SI  - GENBANK/JQ921217
SI  - GENBANK/JQ921218
SI  - GENBANK/JQ921219
SI  - GENBANK/JQ921220
SI  - GENBANK/JQ921221
SI  - GENBANK/JQ921222
SI  - GENBANK/JQ921223
SI  - GENBANK/JQ921224
SI  - GENBANK/JQ921225
SI  - GENBANK/JQ921226
SI  - GENBANK/JQ921227
SI  - GENBANK/JQ921228
SI  - GENBANK/JQ921229
SI  - GENBANK/JQ921230
SI  - GENBANK/JQ921231
SI  - GENBANK/JQ921232
SI  - GENBANK/JQ921233
SI  - GENBANK/JQ921234
SI  - GENBANK/JQ921235
SI  - GENBANK/JQ921236
SI  - GENBANK/JQ921237
SI  - GENBANK/JQ921238
SI  - GENBANK/JQ921239
SI  - GENBANK/JQ921240
SI  - GENBANK/JQ921241
SI  - GENBANK/JQ921242
SI  - GENBANK/JQ921243
SI  - GENBANK/JQ921244
SI  - GENBANK/JQ921245
SI  - GENBANK/JQ921246
SI  - GENBANK/JQ921247
SI  - GENBANK/JQ921248
SI  - GENBANK/JQ921249
SI  - GENBANK/JQ921250
SI  - GENBANK/JQ921251
SI  - GENBANK/JQ921252
SI  - GENBANK/JQ921253
SI  - GENBANK/JQ921254
SI  - GENBANK/JQ921255
SI  - GENBANK/JQ921256
SI  - GENBANK/JQ921257
SI  - GENBANK/JQ921258
SI  - GENBANK/JQ921259
SI  - GENBANK/JQ921260
SI  - GENBANK/JQ921261
SI  - GENBANK/JQ921262
SI  - GENBANK/JQ921263
SI  - GENBANK/JQ921264
SI  - GENBANK/JQ921265
SI  - GENBANK/JQ921266
SI  - GENBANK/JQ921267
SI  - GENBANK/JQ921268
SI  - GENBANK/JQ921269
SI  - GENBANK/JQ921270
SI  - GENBANK/JQ921271
SI  - GENBANK/JQ921272
SI  - GENBANK/JQ921273
SI  - GENBANK/JQ921274
SI  - GENBANK/JQ921275
SI  - GENBANK/JQ921276
SI  - GENBANK/JQ921277
SI  - GENBANK/JQ921278
SI  - GENBANK/JQ921279
SI  - GENBANK/JQ921280
SI  - GENBANK/JQ921281
SI  - GENBANK/JQ921282
SI  - GENBANK/JQ921283
SI  - GENBANK/JQ921284
SI  - GENBANK/JQ921285
SI  - GENBANK/JQ921286
SI  - GENBANK/JQ921287
SI  - GENBANK/JQ921288
SI  - GENBANK/JQ921289
SI  - GENBANK/JQ921290
SI  - GENBANK/JQ921291
SI  - GENBANK/JQ921292
SI  - GENBANK/JQ921293
SI  - GENBANK/JQ921294
SI  - GENBANK/JQ921295
SI  - GENBANK/JQ921296
SI  - GENBANK/JQ921297
SI  - GENBANK/JQ921298
SI  - GENBANK/JQ921299
SI  - GENBANK/JQ921300
SI  - GENBANK/JQ921301
SI  - GENBANK/JQ921302
SI  - GENBANK/JQ921303
SI  - GENBANK/JQ921304
SI  - GENBANK/JQ921305
SI  - GENBANK/JQ921306
SI  - GENBANK/JQ921307
SI  - GENBANK/JQ921308
SI  - GENBANK/JQ921309
SI  - GENBANK/JQ921310
SI  - GENBANK/JQ921311
SI  - GENBANK/JQ921312
SI  - GENBANK/JQ921313
SI  - GENBANK/JQ921314
SI  - GENBANK/JQ921315
SI  - GENBANK/JQ921316
SI  - GENBANK/JQ921317
SI  - GENBANK/JQ921318
SI  - GENBANK/JQ921319
SI  - GENBANK/JQ921320
SI  - GENBANK/JQ921321
SI  - GENBANK/JQ921322
SI  - GENBANK/JQ921323
SI  - GENBANK/JQ921324
SI  - GENBANK/JQ921325
SI  - GENBANK/JQ921326
SI  - GENBANK/JQ921327
SI  - GENBANK/JQ921328
SI  - GENBANK/JQ921329
SI  - GENBANK/JQ921330
SI  - GENBANK/JQ921331
SI  - GENBANK/JQ921332
SI  - GENBANK/JQ921333
SI  - GENBANK/JQ921334
SI  - GENBANK/JQ921335
SI  - GENBANK/JQ921336
SI  - GENBANK/JQ921337
SI  - GENBANK/JQ921338
SI  - GENBANK/JQ921339
SI  - GENBANK/JQ921340
SI  - GENBANK/JQ921341
SI  - GENBANK/JQ921342
SI  - GENBANK/JQ921343
SI  - GENBANK/JQ921344
SI  - GENBANK/JQ921345
SI  - GENBANK/JQ921346
SI  - GENBANK/JQ921347
SI  - GENBANK/JQ921348
SI  - GENBANK/JQ921349
SI  - GENBANK/JQ921350
SI  - GENBANK/JQ921351
SI  - GENBANK/JQ921352
SI  - GENBANK/JQ921353
SI  - GENBANK/JQ921354
SI  - GENBANK/JQ921355
SI  - GENBANK/JQ921356
SI  - GENBANK/JQ921357
SI  - GENBANK/JQ921358
SI  - GENBANK/JQ921359
SI  - GENBANK/JQ921360
SI  - GENBANK/JQ921361
SI  - GENBANK/JQ921362
SI  - GENBANK/JQ921363
SI  - GENBANK/JQ921364
SI  - GENBANK/JQ921365
SI  - GENBANK/JQ921366
SI  - GENBANK/JQ921367
SI  - GENBANK/JQ921368
SI  - GENBANK/JQ921369
SI  - GENBANK/JQ921370
SI  - GENBANK/JQ921371
SI  - GENBANK/JQ921372
SI  - GENBANK/JQ921373
SI  - GENBANK/JQ921374
SI  - GENBANK/JQ921375
SI  - GENBANK/JQ921376
SI  - GENBANK/JQ921377
SI  - GENBANK/JQ921378
SI  - GENBANK/JQ921379
SI  - GENBANK/JQ921380
SI  - GENBANK/JQ921381
SI  - GENBANK/JQ921382
SI  - GENBANK/JQ921383
SI  - GENBANK/JQ921384
SI  - GENBANK/JQ921385
SI  - GENBANK/JQ921386
SI  - GENBANK/JQ921387
SI  - GENBANK/JQ921388
SI  - GENBANK/JQ921389
SI  - GENBANK/JQ921390
SI  - GENBANK/JQ921391
SI  - GENBANK/JQ921392
SI  - GENBANK/JQ921393
SI  - GENBANK/JQ921394
SI  - GENBANK/JQ921395
SI  - GENBANK/JQ921396
SI  - GENBANK/JQ921397
SI  - GENBANK/JQ921398
SI  - GENBANK/JQ921399
SI  - GENBANK/JQ921400
SI  - GENBANK/JQ921401
SI  - GENBANK/JQ921402
SI  - GENBANK/JQ921403
SI  - GENBANK/JQ921404
SI  - GENBANK/JQ921405
SI  - GENBANK/JQ921406
SI  - GENBANK/JQ921407
SI  - GENBANK/JQ921408
SI  - GENBANK/JQ921409
SI  - GENBANK/JQ921410
SI  - GENBANK/JQ921411
SI  - GENBANK/JQ921412
SI  - GENBANK/JQ921413
SI  - GENBANK/JQ921414
SI  - GENBANK/JQ921415
SI  - GENBANK/JQ921416
SI  - GENBANK/JQ921417
SI  - GENBANK/JQ921418
SI  - GENBANK/JQ921419
SI  - GENBANK/JQ921420
SI  - GENBANK/JQ921421
SI  - GENBANK/JQ921422
SI  - GENBANK/JQ921423
SI  - GENBANK/JQ921424
SI  - GENBANK/JQ921425
SI  - GENBANK/JQ921426
SI  - GENBANK/JQ921427
SI  - GENBANK/JQ921428
SI  - GENBANK/JQ921429
SI  - GENBANK/JQ921430
SI  - GENBANK/JQ921431
SI  - GENBANK/JQ921432
SI  - GENBANK/JQ921433
SI  - GENBANK/JQ921434
SI  - GENBANK/JQ921435
SI  - GENBANK/JQ921436
SI  - GENBANK/JQ921437
SI  - GENBANK/JQ921438
SI  - GENBANK/JQ921439
SI  - GENBANK/JQ921440
SI  - GENBANK/JQ921441
SI  - GENBANK/JQ921442
SI  - GENBANK/JQ921443
SI  - GENBANK/JQ921444
SI  - GENBANK/JQ921445
SI  - GENBANK/JQ921446
SI  - GENBANK/JQ921447
SI  - GENBANK/JQ921448
SI  - GENBANK/JQ921449
SI  - GENBANK/JQ921450
SI  - GENBANK/JQ921451
SI  - GENBANK/JQ921452
SI  - GENBANK/JQ921453
SI  - GENBANK/JQ921454
SI  - GENBANK/JQ921455
SI  - GENBANK/JQ921456
SI  - GENBANK/JQ921457
SI  - GENBANK/JQ921458
SI  - GENBANK/JQ921459
SI  - GENBANK/JQ921460
SI  - GENBANK/JQ921461
SI  - GENBANK/JQ921462
SI  - GENBANK/JQ921463
SI  - GENBANK/JQ921464
SI  - GENBANK/JQ921465
SI  - GENBANK/JQ921466
SI  - GENBANK/JQ921467
SI  - GENBANK/JQ921468
SI  - GENBANK/JQ921469
SI  - GENBANK/JQ921470
SI  - GENBANK/JQ921471
SI  - GENBANK/JQ921472
SI  - GENBANK/JQ921473
SI  - GENBANK/JQ921474
SI  - GENBANK/JQ921475
SI  - GENBANK/JQ921476
SI  - GENBANK/JQ921477
SI  - GENBANK/JQ921478
SI  - GENBANK/JQ921479
SI  - GENBANK/JQ921480
SI  - GENBANK/JQ921481
SI  - GENBANK/JQ921482
SI  - GENBANK/JQ921483
SI  - GENBANK/JQ921484
SI  - GENBANK/JQ921485
SI  - GENBANK/JQ921486
SI  - GENBANK/JQ921487
SI  - GENBANK/JQ921488
SI  - GENBANK/JQ921489
SI  - GENBANK/JQ921490
SI  - GENBANK/JQ921491
SI  - GENBANK/JQ921492
SI  - GENBANK/JQ921493
SI  - GENBANK/JQ921494
SI  - GENBANK/JQ921495
SI  - GENBANK/JQ921496
SI  - GENBANK/JQ921497
SI  - GENBANK/JQ921498
SI  - GENBANK/JQ921499
SI  - GENBANK/JQ921500
SI  - GENBANK/JQ921501
SI  - GENBANK/JQ921502
SI  - GENBANK/JQ921503
SI  - GENBANK/JQ921504
SI  - GENBANK/JQ921505
SI  - GENBANK/JQ921506
SI  - GENBANK/JQ921507
SI  - GENBANK/JQ921508
SI  - GENBANK/JQ921509
SI  - GENBANK/JQ921510
SI  - GENBANK/JQ921511
SI  - GENBANK/JQ921512
SI  - GENBANK/JQ921513
SI  - GENBANK/JQ921514
SI  - GENBANK/JQ921515
SI  - GENBANK/JQ921516
SI  - GENBANK/JQ921517
SI  - GENBANK/JQ921518
SI  - GENBANK/JQ921519
SI  - GENBANK/JQ921520
SI  - GENBANK/JQ921521
SI  - GENBANK/JQ921522
SI  - GENBANK/JQ921523
SI  - GENBANK/JQ921524
SI  - GENBANK/JQ921525
SI  - GENBANK/JQ921526
SI  - GENBANK/JQ921527
SI  - GENBANK/JQ921528
SI  - GENBANK/JQ921529
SI  - GENBANK/JQ921530
SI  - GENBANK/JQ921531
SI  - GENBANK/JQ921532
SI  - GENBANK/JQ921533
SI  - GENBANK/JQ921534
SI  - GENBANK/JQ921535
SI  - GENBANK/JQ921536
SI  - GENBANK/JQ921537
SI  - GENBANK/JQ921538
SI  - GENBANK/JQ921539
SI  - GENBANK/JQ921540
SI  - GENBANK/JQ921541
SI  - GENBANK/JQ921542
SI  - GENBANK/JQ921543
SI  - GENBANK/JQ921544
SI  - GENBANK/JQ921545
SI  - GENBANK/JQ921546
SI  - GENBANK/JQ921547
SI  - GENBANK/JQ921548
SI  - GENBANK/JQ921549
SI  - GENBANK/JQ921550
SI  - GENBANK/JQ921551
SI  - GENBANK/JQ921552
SI  - GENBANK/JQ921553
SI  - GENBANK/JQ921554
SI  - GENBANK/JQ921555
SI  - GENBANK/JQ921556
SI  - GENBANK/JQ921557
SI  - GENBANK/JQ921558
SI  - GENBANK/JQ921559
SI  - GENBANK/JQ921560
SI  - GENBANK/JQ921561
SI  - GENBANK/JQ921562
SI  - GENBANK/JQ921563
SI  - GENBANK/JQ921564
SI  - GENBANK/JQ921565
SI  - GENBANK/JQ921566
SI  - GENBANK/JQ921567
SI  - GENBANK/JQ921568
SI  - GENBANK/JQ921569
SI  - GENBANK/JQ921570
SI  - GENBANK/JQ921571
SI  - GENBANK/JQ921572
SI  - GENBANK/JQ921573
SI  - GENBANK/JQ921574
SI  - GENBANK/JQ921575
SI  - GENBANK/JQ921576
SI  - GENBANK/JQ921577
SI  - GENBANK/JQ921578
SI  - GENBANK/JQ921579
SI  - GENBANK/JQ921580
SI  - GENBANK/JQ921581
SI  - GENBANK/JQ921582
SI  - GENBANK/JQ921583
SI  - GENBANK/JQ921584
SI  - GENBANK/JQ921585
SI  - GENBANK/JQ921586
SI  - GENBANK/JQ921587
SI  - GENBANK/JQ921588
SI  - GENBANK/JQ921589
SI  - GENBANK/JQ921590
SI  - GENBANK/JQ921591
SI  - GENBANK/JQ921592
SI  - GENBANK/JQ921593
SI  - GENBANK/JQ921594
SI  - GENBANK/JQ921595
SI  - GENBANK/JQ921596
SI  - GENBANK/JQ921597
SI  - GENBANK/JQ921598
SI  - GENBANK/JQ921599
SI  - GENBANK/JQ921600
SI  - GENBANK/JQ921601
SI  - GENBANK/JQ921602
SI  - GENBANK/JQ921603
SI  - GENBANK/JQ921604
SI  - GENBANK/JQ921605
SI  - GENBANK/JQ921606
SI  - GENBANK/JQ921607
SI  - GENBANK/JQ921608
SI  - GENBANK/JQ921609
SI  - GENBANK/JQ921610
SI  - GENBANK/JQ921611
SI  - GENBANK/JQ921612
SI  - GENBANK/JQ921613
SI  - GENBANK/JQ921614
SI  - GENBANK/JQ921615
SI  - GENBANK/JQ921616
SI  - GENBANK/JQ921617
SI  - GENBANK/JQ921618
SI  - GENBANK/JQ921619
SI  - GENBANK/JQ921620
SI  - GENBANK/JQ921621
SI  - GENBANK/JQ921622
SI  - GENBANK/JQ921623
SI  - GENBANK/JQ921624
SI  - GENBANK/JQ921625
SI  - GENBANK/JQ921626
SI  - GENBANK/JQ921627
SI  - GENBANK/JQ921628
SI  - GENBANK/JQ921629
SI  - GENBANK/JQ921630
SI  - GENBANK/JQ921631
SI  - GENBANK/JQ921632
SI  - GENBANK/JQ921633
SI  - GENBANK/JQ921634
SI  - GENBANK/JQ921635
SI  - GENBANK/JQ921636
SI  - GENBANK/JQ921637
SI  - GENBANK/JQ921638
SI  - GENBANK/JQ921639
SI  - GENBANK/JQ921640
SI  - GENBANK/JQ921641
SI  - GENBANK/JQ921642
SI  - GENBANK/JQ921643
SI  - GENBANK/JQ921644
SI  - GENBANK/JQ921645
SI  - GENBANK/JQ921646
SI  - GENBANK/JQ921647
SI  - GENBANK/JQ921648
SI  - GENBANK/JQ921649
SI  - GENBANK/JQ921650
SI  - GENBANK/JQ921651
SI  - GENBANK/JQ921652
SI  - GENBANK/JQ921653
SI  - GENBANK/JQ921654
SI  - GENBANK/JQ921655
SI  - GENBANK/JQ921656
SI  - GENBANK/JQ921657
SI  - GENBANK/JQ921658
SI  - GENBANK/JQ921659
SI  - GENBANK/JQ921660
SI  - GENBANK/JQ921661
SI  - GENBANK/JQ921662
SI  - GENBANK/JQ921663
SI  - GENBANK/JQ921664
SI  - GENBANK/JQ921665
SI  - GENBANK/JQ921666
SI  - GENBANK/JQ921667
SI  - GENBANK/JQ921668
SI  - GENBANK/JQ921669
SI  - GENBANK/JQ921670
SI  - GENBANK/JQ921671
SI  - GENBANK/JQ921672
SI  - GENBANK/JQ921673
SI  - GENBANK/JQ921674
SI  - GENBANK/JQ921675
SI  - GENBANK/JQ921676
SI  - GENBANK/JQ921677
SI  - GENBANK/JQ921678
SI  - GENBANK/JQ921679
SI  - GENBANK/JQ921680
SI  - GENBANK/JQ921681
SI  - GENBANK/JQ921682
SI  - GENBANK/JQ921683
SI  - GENBANK/JQ921684
SI  - GENBANK/JQ921685
SI  - GENBANK/JQ921686
SI  - GENBANK/JQ921687
SI  - GENBANK/JQ921688
SI  - GENBANK/JQ921689
SI  - GENBANK/JQ921690
SI  - GENBANK/JQ921691
SI  - GENBANK/JQ921692
SI  - GENBANK/JQ921693
SI  - GENBANK/JQ921694
SI  - GENBANK/JQ921695
SI  - GENBANK/JQ921696
SI  - GENBANK/JQ921697
SI  - GENBANK/JQ921698
SI  - GENBANK/JQ921699
SI  - GENBANK/JQ921700
SI  - GENBANK/JQ921701
SI  - GENBANK/JQ921702
SI  - GENBANK/JQ921703
SI  - GENBANK/JQ921704
SI  - GENBANK/JQ921705
SI  - GENBANK/JQ921706
SI  - GENBANK/JQ921707
SI  - GENBANK/JQ921708
SI  - GENBANK/JQ921709
SI  - GENBANK/JQ921710
SI  - GENBANK/JQ921711
SI  - GENBANK/JQ921712
SI  - GENBANK/JQ921713
SI  - GENBANK/JQ921714
SI  - GENBANK/JQ921715
SI  - GENBANK/JQ921716
SI  - GENBANK/JQ921717
SI  - GENBANK/JQ921718
SI  - GENBANK/JQ921719
SI  - GENBANK/JQ921720
SI  - GENBANK/JQ921721
SI  - GENBANK/JQ921722
SI  - GENBANK/JQ921723
SI  - GENBANK/JQ921724
SI  - GENBANK/JQ921725
SI  - GENBANK/JQ921726
SI  - GENBANK/JQ921727
SI  - GENBANK/JQ921728
SI  - GENBANK/JQ921729
SI  - GENBANK/JQ921730
SI  - GENBANK/JQ921731
SI  - GENBANK/JQ921732
SI  - GENBANK/JQ921733
SI  - GENBANK/JQ921734
SI  - GENBANK/JQ921735
SI  - GENBANK/JQ921736
SI  - GENBANK/JQ921737
SI  - GENBANK/JQ921738
SI  - GENBANK/JQ921739
SI  - GENBANK/JQ921740
SI  - GENBANK/JQ921741
SI  - GENBANK/JQ921742
SI  - GENBANK/JQ921743
SI  - GENBANK/JQ921744
SI  - GENBANK/JQ921745
SI  - GENBANK/JQ921746
SI  - GENBANK/JQ921747
SI  - GENBANK/JQ921748
SI  - GENBANK/JQ921749
SI  - GENBANK/JQ921750
SI  - GENBANK/JQ921751
SI  - GENBANK/JQ921752
SI  - GENBANK/JQ921753
SI  - GENBANK/JQ921754
SI  - GENBANK/JQ921755
SI  - GENBANK/JQ921756
SI  - GENBANK/JQ921757
SI  - GENBANK/JQ921758
SI  - GENBANK/JQ921759
SI  - GENBANK/JQ921760
SI  - GENBANK/JQ921761
SI  - GENBANK/JQ921762
SI  - GENBANK/JQ921763
SI  - GENBANK/JQ921764
SI  - GENBANK/JQ921765
SI  - GENBANK/JQ921766
SI  - GENBANK/JQ921767
SI  - GENBANK/JQ921768
SI  - GENBANK/JQ921769
SI  - GENBANK/JQ921770
SI  - GENBANK/JQ921771
SI  - GENBANK/JQ921772
SI  - GENBANK/JQ921773
SI  - GENBANK/JQ921774
SI  - GENBANK/JQ921775
SI  - GENBANK/JQ921776
SI  - GENBANK/JQ921777
SI  - GENBANK/JQ921778
SI  - GENBANK/JQ921779
SI  - GENBANK/JQ921780
SI  - GENBANK/JQ921781
SI  - GENBANK/JQ921782
SI  - GENBANK/JQ921783
SI  - GENBANK/JQ921784
SI  - GENBANK/JQ921785
SI  - GENBANK/JQ921786
SI  - GENBANK/JQ921787
SI  - GENBANK/JQ921788
SI  - GENBANK/JQ921789
SI  - GENBANK/JQ921790
SI  - GENBANK/JQ921791
SI  - GENBANK/JQ921792
SI  - GENBANK/JQ921793
SI  - GENBANK/JQ921794
SI  - GENBANK/JQ921795
SI  - GENBANK/JQ921796
SI  - GENBANK/JQ921797
SI  - GENBANK/JQ921798
SI  - GENBANK/JQ921799
SI  - GENBANK/JQ921800
SI  - GENBANK/JQ921801
SI  - GENBANK/JQ921802
SI  - GENBANK/JQ921803
SI  - GENBANK/JQ921804
SI  - GENBANK/JQ921805
SI  - GENBANK/JQ921806
SI  - GENBANK/JQ921807
SI  - GENBANK/JQ921808
SI  - GENBANK/JQ921809
SI  - GENBANK/JQ921810
SI  - GENBANK/JQ921811
SI  - GENBANK/JQ921812
SI  - GENBANK/JQ921813
SI  - GENBANK/JQ921814
SI  - GENBANK/JQ921815
SI  - GENBANK/JQ921816
SI  - GENBANK/JQ921817
SI  - GENBANK/JQ921818
SI  - GENBANK/JQ921819
SI  - GENBANK/JQ921820
SI  - GENBANK/JQ921821
SI  - GENBANK/JQ921822
SI  - GENBANK/JQ921823
SI  - GENBANK/JQ921824
SI  - GENBANK/JQ921825
SI  - GENBANK/JQ921826
SI  - GENBANK/JQ921827
SI  - GENBANK/JQ921828
SI  - GENBANK/JQ921829
SI  - GENBANK/JQ921830
SI  - GENBANK/JQ921831
SI  - GENBANK/JQ921832
SI  - GENBANK/JQ921833
SI  - GENBANK/JQ921834
SI  - GENBANK/JQ921835
SI  - GENBANK/JQ921836
SI  - GENBANK/JQ921837
SI  - GENBANK/JQ921838
SI  - GENBANK/JQ921839
SI  - GENBANK/JQ921840
SI  - GENBANK/JQ921841
SI  - GENBANK/JQ921842
SI  - GENBANK/JQ921843
SI  - GENBANK/JQ921844
SI  - GENBANK/JQ921845
SI  - GENBANK/JQ921846
SI  - GENBANK/JQ921847
SI  - GENBANK/JQ921848
SI  - GENBANK/JQ921849
SI  - GENBANK/JQ921850
SI  - GENBANK/JQ921851
SI  - GENBANK/JQ921852
SI  - GENBANK/JQ921853
SI  - GENBANK/JQ921854
SI  - GENBANK/JQ921855
SI  - GENBANK/JQ921856
SI  - GENBANK/JQ921857
SI  - GENBANK/JQ921858
SI  - GENBANK/JQ921859
SI  - GENBANK/JQ921860
SI  - GENBANK/JQ921861
SI  - GENBANK/JQ921862
SI  - GENBANK/JQ921863
SI  - GENBANK/JQ921864
SI  - GENBANK/JQ921865
SI  - GENBANK/JQ921866
SI  - GENBANK/JQ921867
SI  - GENBANK/JQ921868
SI  - GENBANK/JQ921869
SI  - GENBANK/JQ921870
SI  - GENBANK/JQ921871
SI  - GENBANK/JQ921872
SI  - GENBANK/JQ921873
SI  - GENBANK/JQ921874
SI  - GENBANK/JQ921875
SI  - GENBANK/JQ921876
SI  - GENBANK/JQ921877
SI  - GENBANK/JQ921878
SI  - GENBANK/JQ921879
SI  - GENBANK/JQ921880
SI  - GENBANK/JQ921881
SI  - GENBANK/JQ921882
SI  - GENBANK/JQ921883
SI  - GENBANK/JQ921884
SI  - GENBANK/JQ921885
SI  - GENBANK/JQ921886
SI  - GENBANK/JQ921887
SI  - GENBANK/JQ921888
SI  - GENBANK/JQ921889
SI  - GENBANK/JQ921890
SI  - GENBANK/JQ921891
SI  - GENBANK/JQ921892
SI  - GENBANK/JQ921893
SI  - GENBANK/JQ921894
SI  - GENBANK/JQ921895
SI  - GENBANK/JQ921896
SI  - GENBANK/JQ921897
SI  - GENBANK/JQ921898
SI  - GENBANK/JQ921899
SI  - GENBANK/JQ921900
SI  - GENBANK/JQ921901
SI  - GENBANK/JQ921902
SI  - GENBANK/JQ921903
SI  - GENBANK/JQ921904
SI  - GENBANK/JQ921905
SI  - GENBANK/JQ921906
SI  - GENBANK/JQ921907
SI  - GENBANK/JQ921908
SI  - GENBANK/JQ921909
SI  - GENBANK/JQ921910
SI  - GENBANK/JQ921911
SI  - GENBANK/JQ921912
SI  - GENBANK/JQ921913
SI  - GENBANK/JQ921914
SI  - GENBANK/JQ921915
SI  - GENBANK/JQ921916
SI  - GENBANK/JQ921917
SI  - GENBANK/JQ921918
SI  - GENBANK/JQ921919
SI  - GENBANK/JQ921920
SI  - GENBANK/JQ921921
SI  - GENBANK/JQ921922
SI  - GENBANK/JQ921923
SI  - GENBANK/JQ921924
SI  - GENBANK/JQ921925
SI  - GENBANK/JQ921926
SI  - GENBANK/JQ921927
SI  - GENBANK/JQ921928
SI  - GENBANK/JQ921929
SI  - GENBANK/JQ921930
SI  - GENBANK/JQ921931
SI  - GENBANK/JQ921932
SI  - GENBANK/JQ921933
SI  - GENBANK/JQ921934
SI  - GENBANK/JQ921935
SI  - GENBANK/JQ921936
SI  - GENBANK/JQ921937
SI  - GENBANK/JQ921938
SI  - GENBANK/JQ921939
SI  - GENBANK/JQ921940
SI  - GENBANK/JQ921941
SI  - GENBANK/JQ921942
SI  - GENBANK/JQ921943
SI  - GENBANK/JQ921944
SI  - GENBANK/JQ921945
SI  - GENBANK/JQ921946
SI  - GENBANK/JQ921947
SI  - GENBANK/JQ921948
SI  - GENBANK/JQ921949
SI  - GENBANK/JQ921950
SI  - GENBANK/JQ921951
SI  - GENBANK/JQ921952
SI  - GENBANK/JQ921953
SI  - GENBANK/JQ921954
SI  - GENBANK/JQ921955
SI  - GENBANK/JQ921956
SI  - GENBANK/JQ921957
SI  - GENBANK/JQ921958
SI  - GENBANK/JQ921959
SI  - GENBANK/JQ921960
SI  - GENBANK/JQ921961
SI  - GENBANK/JQ921962
SI  - GENBANK/JQ921963
SI  - GENBANK/JQ921964
SI  - GENBANK/JQ921965
SI  - GENBANK/JQ921966
SI  - GENBANK/JQ921967
SI  - GENBANK/JQ921968
SI  - GENBANK/JQ921969
SI  - GENBANK/JQ921970
SI  - GENBANK/JQ921971
SI  - GENBANK/JQ921972
SI  - GENBANK/JQ921973
SI  - GENBANK/JQ921974
SI  - GENBANK/JQ921975
SI  - GENBANK/JQ921976
SI  - GENBANK/JQ921977
SI  - GENBANK/JQ921978
SI  - GENBANK/JQ921979
SI  - GENBANK/JQ921980
SI  - GENBANK/JQ921981
SI  - GENBANK/JQ921982
SI  - GENBANK/JQ921983
SI  - GENBANK/JQ921984
SI  - GENBANK/JQ921985
SI  - GENBANK/JQ921986
SI  - GENBANK/JQ921987
SI  - GENBANK/JQ921988
SI  - GENBANK/JQ921989
SI  - GENBANK/JQ921990
SI  - GENBANK/JQ921991
SI  - GENBANK/JQ921992
SI  - GENBANK/JQ921993
SI  - GENBANK/JQ921994
SI  - GENBANK/JQ921995
SI  - GENBANK/JQ921996
SI  - GENBANK/JQ921997
SI  - GENBANK/JQ921998
SI  - GENBANK/JQ921999
SI  - GENBANK/JQ922000
SI  - GENBANK/JQ922001
SI  - GENBANK/JQ922002
SI  - GENBANK/JQ922003
SI  - GENBANK/JQ922004
SI  - GENBANK/JQ922005
SI  - GENBANK/JQ922006
SI  - GENBANK/JQ922007
SI  - GENBANK/JQ922008
SI  - GENBANK/JQ922009
SI  - GENBANK/JQ922010
SI  - GENBANK/JQ922011
SI  - GENBANK/JQ922012
SI  - GENBANK/JQ922013
SI  - GENBANK/JQ922014
SI  - GENBANK/JQ922015
SI  - GENBANK/JQ922016
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130305
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - Electron Transport Complex IV/genetics/metabolism
MH  - Europe
MH  - *Evolution, Molecular
MH  - Evolution, Planetary
MH  - Genetic Speciation
MH  - Groundwater
MH  - Isopoda/*classification/*genetics/metabolism
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Polymerase Chain Reaction
MH  - United States
EDAT- 2013/03/07 06:00
MHDA- 2014/01/09 06:00
CRDT- 2013/03/07 06:00
PHST- 2013/03/07 06:00 [entrez]
PHST- 2013/03/07 06:00 [pubmed]
PHST- 2014/01/09 06:00 [medline]
AID - syt015 [pii]
AID - 10.1093/sysbio/syt015 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jul;62(4):512-22. doi: 10.1093/sysbio/syt015. Epub 2013 Mar 5.

PMID- 23427289
OWN - NLM
STAT- MEDLINE
DCOM- 20130930
LR  - 20130412
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 3
DP  - 2013 May 1
TI  - Statistical inference of allopolyploid species networks in the presence of
      incomplete lineage sorting.
PG  - 467-78
LID - 10.1093/sysbio/syt012 [doi]
AB  - Polyploidy is an important speciation mechanism, particularly in land plants.
      Allopolyploid species are formed after hybridization between otherwise
      intersterile parental species. Recent theoretical progress has led to successful 
      implementation of species tree models that take population genetic parameters
      into account. However, these models have not included allopolyploid hybridization
      and the special problems imposed when species trees of allopolyploids are
      inferred. Here, 2 new models for the statistical inference of the evolutionary
      history of allopolyploids are evaluated using simulations and demonstrated on 2
      empirical data sets. It is assumed that there has been a single hybridization
      event between 2 diploid species resulting in a genomic allotetraploid. The
      evolutionary history can be represented as a species network or as a multilabeled
      species tree, in which some pairs of tips are labeled with the same species. In
      one of the models (AlloppMUL), the multilabeled species tree is inferred
      directly. This is the simplest model and the most widely applicable, since fewer 
      assumptions are made. The second model (AlloppNET) incorporates the hybridization
      event explicitly which means that fewer parameters need to be estimated. Both
      models are implemented in the BEAST framework. Simulations show that both models 
      are useful and that AlloppNET is more accurate if the assumptions it is based on 
      are valid. The models are demonstrated on previously analyzed data from the
      genera Pachycladon (Brassicaceae) and Silene (Caryophyllaceae).
FAU - Jones, Graham
AU  - Jones G
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Gothenburg, Sweden. art@gjones.name
FAU - Sagitov, Serik
AU  - Sagitov S
FAU - Oxelman, Bengt
AU  - Oxelman B
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130219
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Brassicaceae/*genetics
MH  - *Evolution, Molecular
MH  - Hybridization, Genetic
MH  - Models, Genetic
MH  - *Polyploidy
MH  - Silene/*genetics
EDAT- 2013/02/22 06:00
MHDA- 2013/10/01 06:00
CRDT- 2013/02/22 06:00
PHST- 2013/02/22 06:00 [entrez]
PHST- 2013/02/22 06:00 [pubmed]
PHST- 2013/10/01 06:00 [medline]
AID - syt012 [pii]
AID - 10.1093/sysbio/syt012 [doi]
PST - ppublish
SO  - Syst Biol. 2013 May 1;62(3):467-78. doi: 10.1093/sysbio/syt012. Epub 2013 Feb 19.

PMID- 23427288
OWN - NLM
STAT- MEDLINE
DCOM- 20130930
LR  - 20130412
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 3
DP  - 2013 May 1
TI  - Biogeography off the tracks.
PG  - 494-8
LID - 10.1093/sysbio/syt013 [doi]
FAU - Waters, Jonathan M
AU  - Waters JM
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, Department of Zoology,
      University of Otago, PO Box 56, Dunedin, New Zealand. jon.waters@otago.ac.nz
FAU - Trewick, Steven A
AU  - Trewick SA
FAU - Paterson, Adrian M
AU  - Paterson AM
FAU - Spencer, Hamish G
AU  - Spencer HG
FAU - Kennedy, Martyn
AU  - Kennedy M
FAU - Craw, Dave
AU  - Craw D
FAU - Burridge, Christopher P
AU  - Burridge CP
FAU - Wallis, Graham P
AU  - Wallis GP
LA  - eng
PT  - Journal Article
DEP - 20130219
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Animal Distribution
MH  - Animals
MH  - *Biological Evolution
MH  - Geography
MH  - Palaeognathae/physiology
MH  - *Plant Dispersal
MH  - *Plants
EDAT- 2013/02/22 06:00
MHDA- 2013/10/01 06:00
CRDT- 2013/02/22 06:00
PHST- 2013/02/22 06:00 [entrez]
PHST- 2013/02/22 06:00 [pubmed]
PHST- 2013/10/01 06:00 [medline]
AID - syt013 [pii]
AID - 10.1093/sysbio/syt013 [doi]
PST - ppublish
SO  - Syst Biol. 2013 May 1;62(3):494-8. doi: 10.1093/sysbio/syt013. Epub 2013 Feb 19.

PMID- 23417682
OWN - NLM
STAT- MEDLINE
DCOM- 20130930
LR  - 20130412
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 3
DP  - 2013 May 1
TI  - Is permanent parasitism reversible?--critical evidence from early evolution of
      house dust mites.
PG  - 411-23
LID - 10.1093/sysbio/syt008 [doi]
AB  - Long-term specialization may limit the ability of a species to respond to new
      environmental conditions and lead to a higher likelihood of extinction. For
      permanent parasites and other symbionts, the most intriguing question is whether 
      these organisms can return to a free-living lifestyle and, thus, escape an
      evolutionary "dead end." This question is directly related to Dollo's law, which 
      stipulates that a complex trait (such as being free living vs. parasitic) cannot 
      re-evolve again in the same form. Here, we present conclusive evidence that house
      dust mites, a group of medically important free-living organisms, evolved from
      permanent parasites of warm-blooded vertebrates. A robust, multigene topology
      (315 taxa, 8942 nt), ancestral character state reconstruction, and a test for
      irreversible evolution (Dollo's law) demonstrate that house dust mites have
      abandoned a parasitic lifestyle, secondarily becoming free living, and then
      speciated in several habitats. Hence, as exemplified by this model system, highly
      specialized permanent parasites may drastically de-specialize to the extent of
      becoming free living and, thus escape from dead-end evolution. Our phylogenetic
      and historical ecological framework explains the limited cross-reactivity between
      allergens from the house dust mites and "storage" mites and the ability of the
      dust mites to inhibit host immune responses. It also provides insights into how
      ancestral features related to parasitism (frequent ancestral shifts to unrelated 
      hosts, tolerance to lower humidity, and pre-existing enzymes targeting skin and
      keratinous materials) played a major role in reversal to the free-living state.
      We propose that parasitic ancestors of pyroglyphids shifted to nests of
      vertebrates. Later the nest-inhabiting pyroglyphids expanded into human dwellings
      to become a major source of allergens.
FAU - Klimov, Pavel B
AU  - Klimov PB
AD  - Department of Ecology and Evolutionary Biology, University of Michigan, Ann
      Arbor, MI 48109-1079, USA. pklimov@umich.edu
FAU - OConnor, Barry
AU  - OConnor B
LA  - eng
SI  - GENBANK/JQ000032
SI  - GENBANK/JQ000033
SI  - GENBANK/JQ000034
SI  - GENBANK/JQ000035
SI  - GENBANK/JQ000036
SI  - GENBANK/JQ000037
SI  - GENBANK/JQ000038
SI  - GENBANK/JQ000039
SI  - GENBANK/JQ000040
SI  - GENBANK/JQ000041
SI  - GENBANK/JQ000042
SI  - GENBANK/JQ000043
SI  - GENBANK/JQ000044
SI  - GENBANK/JQ000045
SI  - GENBANK/JQ000046
SI  - GENBANK/JQ000047
SI  - GENBANK/JQ000048
SI  - GENBANK/JQ000049
SI  - GENBANK/JQ000050
SI  - GENBANK/JQ000051
SI  - GENBANK/JQ000052
SI  - GENBANK/JQ000053
SI  - GENBANK/JQ000054
SI  - GENBANK/JQ000055
SI  - GENBANK/JQ000056
SI  - GENBANK/JQ000057
SI  - GENBANK/JQ000058
SI  - GENBANK/JQ000059
SI  - GENBANK/JQ000060
SI  - GENBANK/JQ000061
SI  - GENBANK/JQ000062
SI  - GENBANK/JQ000063
SI  - GENBANK/JQ000064
SI  - GENBANK/JQ000065
SI  - GENBANK/JQ000066
SI  - GENBANK/JQ000067
SI  - GENBANK/JQ000068
SI  - GENBANK/JQ000069
SI  - GENBANK/JQ000070
SI  - GENBANK/JQ000071
SI  - GENBANK/JQ000072
SI  - GENBANK/JQ000073
SI  - GENBANK/JQ000074
SI  - GENBANK/JQ000075
SI  - GENBANK/JQ000076
SI  - GENBANK/JQ000077
SI  - GENBANK/JQ000078
SI  - GENBANK/JQ000079
SI  - GENBANK/JQ000080
SI  - GENBANK/JQ000081
SI  - GENBANK/JQ000082
SI  - GENBANK/JQ000083
SI  - GENBANK/JQ000084
SI  - GENBANK/JQ000085
SI  - GENBANK/JQ000086
SI  - GENBANK/JQ000087
SI  - GENBANK/JQ000088
SI  - GENBANK/JQ000089
SI  - GENBANK/JQ000090
SI  - GENBANK/JQ000091
SI  - GENBANK/JQ000092
SI  - GENBANK/JQ000093
SI  - GENBANK/JQ000094
SI  - GENBANK/JQ000095
SI  - GENBANK/JQ000096
SI  - GENBANK/JQ000097
SI  - GENBANK/JQ000098
SI  - GENBANK/JQ000099
SI  - GENBANK/JQ000100
SI  - GENBANK/JQ000101
SI  - GENBANK/JQ000102
SI  - GENBANK/JQ000103
SI  - GENBANK/JQ000104
SI  - GENBANK/JQ000105
SI  - GENBANK/JQ000106
SI  - GENBANK/JQ000107
SI  - GENBANK/JQ000108
SI  - GENBANK/JQ000109
SI  - GENBANK/JQ000110
SI  - GENBANK/JQ000111
SI  - GENBANK/JQ000112
SI  - GENBANK/JQ000113
SI  - GENBANK/JQ000114
SI  - GENBANK/JQ000115
SI  - GENBANK/JQ000116
SI  - GENBANK/JQ000117
SI  - GENBANK/JQ000118
SI  - GENBANK/JQ000119
SI  - GENBANK/JQ000120
SI  - GENBANK/JQ000121
SI  - GENBANK/JQ000122
SI  - GENBANK/JQ000123
SI  - GENBANK/JQ000124
SI  - GENBANK/JQ000125
SI  - GENBANK/JQ000126
SI  - GENBANK/JQ000127
SI  - GENBANK/JQ000128
SI  - GENBANK/JQ000129
SI  - GENBANK/JQ000130
SI  - GENBANK/JQ000131
SI  - GENBANK/JQ000132
SI  - GENBANK/JQ000133
SI  - GENBANK/JQ000134
SI  - GENBANK/JQ000135
SI  - GENBANK/JQ000136
SI  - GENBANK/JQ000137
SI  - GENBANK/JQ000138
SI  - GENBANK/JQ000139
SI  - GENBANK/JQ000140
SI  - GENBANK/JQ000141
SI  - GENBANK/JQ000142
SI  - GENBANK/JQ000143
SI  - GENBANK/JQ000144
SI  - GENBANK/JQ000145
SI  - GENBANK/JQ000146
SI  - GENBANK/JQ000147
SI  - GENBANK/JQ000148
SI  - GENBANK/JQ000149
SI  - GENBANK/JQ000150
SI  - GENBANK/JQ000151
SI  - GENBANK/JQ000152
SI  - GENBANK/JQ000153
SI  - GENBANK/JQ000154
SI  - GENBANK/JQ000155
SI  - GENBANK/JQ000156
SI  - GENBANK/JQ000157
SI  - GENBANK/JQ000158
SI  - GENBANK/JQ000159
SI  - GENBANK/JQ000160
SI  - GENBANK/JQ000161
SI  - GENBANK/JQ000162
SI  - GENBANK/JQ000163
SI  - GENBANK/JQ000164
SI  - GENBANK/JQ000165
SI  - GENBANK/JQ000166
SI  - GENBANK/JQ000167
SI  - GENBANK/JQ000168
SI  - GENBANK/JQ000169
SI  - GENBANK/JQ000170
SI  - GENBANK/JQ000171
SI  - GENBANK/JQ000172
SI  - GENBANK/JQ000173
SI  - GENBANK/JQ000174
SI  - GENBANK/JQ000175
SI  - GENBANK/JQ000176
SI  - GENBANK/JQ000177
SI  - GENBANK/JQ000178
SI  - GENBANK/JQ000179
SI  - GENBANK/JQ000180
SI  - GENBANK/JQ000181
SI  - GENBANK/JQ000182
SI  - GENBANK/JQ000183
SI  - GENBANK/JQ000184
SI  - GENBANK/JQ000185
SI  - GENBANK/JQ000186
SI  - GENBANK/JQ000187
SI  - GENBANK/JQ000188
SI  - GENBANK/JQ000189
SI  - GENBANK/JQ000190
SI  - GENBANK/JQ000191
SI  - GENBANK/JQ000192
SI  - GENBANK/JQ000193
SI  - GENBANK/JQ000194
SI  - GENBANK/JQ000195
SI  - GENBANK/JQ000196
SI  - GENBANK/JQ000197
SI  - GENBANK/JQ000198
SI  - GENBANK/JQ000199
SI  - GENBANK/JQ000200
SI  - GENBANK/JQ000201
SI  - GENBANK/JQ000202
SI  - GENBANK/JQ000203
SI  - GENBANK/JQ000204
SI  - GENBANK/JQ000205
SI  - GENBANK/JQ000206
SI  - GENBANK/JQ000207
SI  - GENBANK/JQ000208
SI  - GENBANK/JQ000209
SI  - GENBANK/JQ000210
SI  - GENBANK/JQ000211
SI  - GENBANK/JQ000212
SI  - GENBANK/JQ000213
SI  - GENBANK/JQ000214
SI  - GENBANK/JQ000215
SI  - GENBANK/JQ000216
SI  - GENBANK/JQ000217
SI  - GENBANK/JQ000218
SI  - GENBANK/JQ000219
SI  - GENBANK/JQ000220
SI  - GENBANK/JQ000221
SI  - GENBANK/JQ000222
SI  - GENBANK/JQ000223
SI  - GENBANK/JQ000224
SI  - GENBANK/JQ000225
SI  - GENBANK/JQ000226
SI  - GENBANK/JQ000227
SI  - GENBANK/JQ000228
SI  - GENBANK/JQ000229
SI  - GENBANK/JQ000230
SI  - GENBANK/JQ000231
SI  - GENBANK/JQ000232
SI  - GENBANK/JQ000233
SI  - GENBANK/JQ000234
SI  - GENBANK/JQ000235
SI  - GENBANK/JQ000236
SI  - GENBANK/JQ000237
SI  - GENBANK/JQ000238
SI  - GENBANK/JQ000239
SI  - GENBANK/JQ000240
SI  - GENBANK/JQ000241
SI  - GENBANK/JQ000242
SI  - GENBANK/JQ000243
SI  - GENBANK/JQ000244
SI  - GENBANK/JQ000245
SI  - GENBANK/JQ000246
SI  - GENBANK/JQ000247
SI  - GENBANK/JQ000248
SI  - GENBANK/JQ000249
SI  - GENBANK/JQ000250
SI  - GENBANK/JQ000251
SI  - GENBANK/JQ000252
SI  - GENBANK/JQ000253
SI  - GENBANK/JQ000254
SI  - GENBANK/JQ000255
SI  - GENBANK/JQ000256
SI  - GENBANK/JQ000257
SI  - GENBANK/JQ000258
SI  - GENBANK/JQ000259
SI  - GENBANK/JQ000260
SI  - GENBANK/JQ000261
SI  - GENBANK/JQ000262
SI  - GENBANK/JQ000263
SI  - GENBANK/JQ000264
SI  - GENBANK/JQ000265
SI  - GENBANK/JQ000266
SI  - GENBANK/JQ000267
SI  - GENBANK/JQ000268
SI  - GENBANK/JQ000269
SI  - GENBANK/JQ000270
SI  - GENBANK/JQ000271
SI  - GENBANK/JQ000272
SI  - GENBANK/JQ000273
SI  - GENBANK/JQ000274
SI  - GENBANK/JQ000275
SI  - GENBANK/JQ000276
SI  - GENBANK/JQ000277
SI  - GENBANK/JQ000278
SI  - GENBANK/JQ000279
SI  - GENBANK/JQ000280
SI  - GENBANK/JQ000281
SI  - GENBANK/JQ000282
SI  - GENBANK/JQ000283
SI  - GENBANK/JQ000284
SI  - GENBANK/JQ000285
SI  - GENBANK/JQ000286
SI  - GENBANK/JQ000287
SI  - GENBANK/JQ000288
SI  - GENBANK/JQ000289
SI  - GENBANK/JQ000290
SI  - GENBANK/JQ000291
SI  - GENBANK/JQ000292
SI  - GENBANK/JQ000293
SI  - GENBANK/JQ000294
SI  - GENBANK/JQ000295
SI  - GENBANK/JQ000296
SI  - GENBANK/JQ000297
SI  - GENBANK/JQ000298
SI  - GENBANK/JQ000299
SI  - GENBANK/JQ000300
SI  - GENBANK/JQ000301
SI  - GENBANK/JQ000302
SI  - GENBANK/JQ000303
SI  - GENBANK/JQ000304
SI  - GENBANK/JQ000305
SI  - GENBANK/JQ000306
SI  - GENBANK/JQ000307
SI  - GENBANK/JQ000308
SI  - GENBANK/JQ000309
SI  - GENBANK/JQ000310
SI  - GENBANK/JQ000311
SI  - GENBANK/JQ000312
SI  - GENBANK/JQ000313
SI  - GENBANK/JQ000314
SI  - GENBANK/JQ000315
SI  - GENBANK/JQ000316
SI  - GENBANK/JQ000317
SI  - GENBANK/JQ000318
SI  - GENBANK/JQ000319
SI  - GENBANK/JQ000320
SI  - GENBANK/JQ000321
SI  - GENBANK/JQ000322
SI  - GENBANK/JQ000323
SI  - GENBANK/JQ000324
SI  - GENBANK/JQ000325
SI  - GENBANK/JQ000326
SI  - GENBANK/JQ000327
SI  - GENBANK/JQ000328
SI  - GENBANK/JQ000329
SI  - GENBANK/JQ000330
SI  - GENBANK/JQ000331
SI  - GENBANK/JQ000332
SI  - GENBANK/JQ000333
SI  - GENBANK/JQ000334
SI  - GENBANK/JQ000335
SI  - GENBANK/JQ000336
SI  - GENBANK/JQ000337
SI  - GENBANK/JQ000338
SI  - GENBANK/JQ000339
SI  - GENBANK/JQ000340
SI  - GENBANK/JQ000341
SI  - GENBANK/JQ000342
SI  - GENBANK/JQ000343
SI  - GENBANK/JQ000344
SI  - GENBANK/JQ000345
SI  - GENBANK/JQ000346
SI  - GENBANK/JQ000347
SI  - GENBANK/JQ000348
SI  - GENBANK/JQ000349
SI  - GENBANK/JQ000350
SI  - GENBANK/JQ000351
SI  - GENBANK/JQ000352
SI  - GENBANK/JQ000353
SI  - GENBANK/JQ000354
SI  - GENBANK/JQ000355
SI  - GENBANK/JQ000356
SI  - GENBANK/JQ000357
SI  - GENBANK/JQ000358
SI  - GENBANK/JQ000359
SI  - GENBANK/JQ000360
SI  - GENBANK/JQ000361
SI  - GENBANK/JQ000362
SI  - GENBANK/JQ000363
SI  - GENBANK/JQ000364
SI  - GENBANK/JQ000365
SI  - GENBANK/JQ000366
SI  - GENBANK/JQ000367
SI  - GENBANK/JQ000368
SI  - GENBANK/JQ000369
SI  - GENBANK/JQ000370
SI  - GENBANK/JQ000371
SI  - GENBANK/JQ000372
SI  - GENBANK/JQ000373
SI  - GENBANK/JQ000374
SI  - GENBANK/JQ000375
SI  - GENBANK/JQ000376
SI  - GENBANK/JQ000377
SI  - GENBANK/JQ000378
SI  - GENBANK/JQ000379
SI  - GENBANK/JQ000380
SI  - GENBANK/JQ000381
SI  - GENBANK/JQ000382
SI  - GENBANK/JQ000383
SI  - GENBANK/JQ000384
SI  - GENBANK/JQ000385
SI  - GENBANK/JQ000386
SI  - GENBANK/JQ000387
SI  - GENBANK/JQ000388
SI  - GENBANK/JQ000389
SI  - GENBANK/JQ000390
SI  - GENBANK/JQ000391
SI  - GENBANK/JQ000392
SI  - GENBANK/JQ000393
SI  - GENBANK/JQ000394
SI  - GENBANK/JQ000395
SI  - GENBANK/JQ000396
SI  - GENBANK/JQ000397
SI  - GENBANK/JQ000398
SI  - GENBANK/JQ000399
SI  - GENBANK/JQ000400
SI  - GENBANK/JQ000401
SI  - GENBANK/JQ000402
SI  - GENBANK/JQ000403
SI  - GENBANK/JQ000404
SI  - GENBANK/JQ000405
SI  - GENBANK/JQ000406
SI  - GENBANK/JQ000407
SI  - GENBANK/JQ000408
SI  - GENBANK/JQ000409
SI  - GENBANK/JQ000410
SI  - GENBANK/JQ000411
SI  - GENBANK/JQ000412
SI  - GENBANK/JQ000413
SI  - GENBANK/JQ000414
SI  - GENBANK/JQ000415
SI  - GENBANK/JQ000416
SI  - GENBANK/JQ000417
SI  - GENBANK/JQ000418
SI  - GENBANK/JQ000419
SI  - GENBANK/JQ000420
SI  - GENBANK/JQ000421
SI  - GENBANK/JQ000422
SI  - GENBANK/JQ000423
SI  - GENBANK/JQ000424
SI  - GENBANK/JQ000425
SI  - GENBANK/JQ000426
SI  - GENBANK/JQ000427
SI  - GENBANK/JQ000428
SI  - GENBANK/JQ000429
SI  - GENBANK/JQ000430
SI  - GENBANK/JQ000431
SI  - GENBANK/JQ000432
SI  - GENBANK/JQ000433
SI  - GENBANK/JQ000434
SI  - GENBANK/JQ000435
SI  - GENBANK/JQ000436
SI  - GENBANK/JQ000437
SI  - GENBANK/JQ000438
SI  - GENBANK/JQ000439
SI  - GENBANK/JQ000440
SI  - GENBANK/JQ000441
SI  - GENBANK/JQ000442
SI  - GENBANK/JQ000443
SI  - GENBANK/JQ000444
SI  - GENBANK/JQ000445
SI  - GENBANK/JQ000446
SI  - GENBANK/JQ000447
SI  - GENBANK/JQ000448
SI  - GENBANK/JQ000449
SI  - GENBANK/JQ000450
SI  - GENBANK/JQ000451
SI  - GENBANK/JQ000452
SI  - GENBANK/JQ000453
SI  - GENBANK/JQ000454
SI  - GENBANK/JQ000455
SI  - GENBANK/JQ000456
SI  - GENBANK/JQ000457
SI  - GENBANK/JQ000458
SI  - GENBANK/JQ000459
SI  - GENBANK/JQ000460
SI  - GENBANK/JQ000461
SI  - GENBANK/JQ000462
SI  - GENBANK/JQ000463
SI  - GENBANK/JQ000464
SI  - GENBANK/JQ000465
SI  - GENBANK/JQ000466
SI  - GENBANK/JQ000467
SI  - GENBANK/JQ000468
SI  - GENBANK/JQ000469
SI  - GENBANK/JQ000470
SI  - GENBANK/JQ000471
SI  - GENBANK/JQ000472
SI  - GENBANK/JQ000473
SI  - GENBANK/JQ000474
SI  - GENBANK/JQ000475
SI  - GENBANK/JQ000476
SI  - GENBANK/JQ000477
SI  - GENBANK/JQ000478
SI  - GENBANK/JQ000479
SI  - GENBANK/JQ000480
SI  - GENBANK/JQ000481
SI  - GENBANK/JQ000482
SI  - GENBANK/JQ000483
SI  - GENBANK/JQ000484
SI  - GENBANK/JQ000485
SI  - GENBANK/JQ000486
SI  - GENBANK/JQ000487
SI  - GENBANK/JQ000488
SI  - GENBANK/JQ000489
SI  - GENBANK/JQ000490
SI  - GENBANK/JQ000491
SI  - GENBANK/JQ000492
SI  - GENBANK/JQ000493
SI  - GENBANK/JQ000494
SI  - GENBANK/JQ000495
SI  - GENBANK/JQ000496
SI  - GENBANK/JQ000497
SI  - GENBANK/JQ000498
SI  - GENBANK/JQ000499
SI  - GENBANK/JQ000500
SI  - GENBANK/JQ000501
SI  - GENBANK/JQ000502
SI  - GENBANK/JQ000503
SI  - GENBANK/JQ000504
SI  - GENBANK/JQ000505
SI  - GENBANK/JQ000506
SI  - GENBANK/JQ000507
SI  - GENBANK/JQ000508
SI  - GENBANK/JQ000509
SI  - GENBANK/JQ000510
SI  - GENBANK/JQ000511
SI  - GENBANK/JQ000512
SI  - GENBANK/JQ000513
SI  - GENBANK/JQ000514
SI  - GENBANK/JQ000515
SI  - GENBANK/JQ000516
SI  - GENBANK/JQ000517
SI  - GENBANK/JQ000518
SI  - GENBANK/JQ000519
SI  - GENBANK/JQ000520
SI  - GENBANK/JQ000521
SI  - GENBANK/JQ000522
SI  - GENBANK/JQ000523
SI  - GENBANK/JQ000524
SI  - GENBANK/JQ000525
SI  - GENBANK/JQ000526
SI  - GENBANK/JQ000527
SI  - GENBANK/JQ000528
SI  - GENBANK/JQ000529
SI  - GENBANK/JQ000530
SI  - GENBANK/JQ000531
SI  - GENBANK/JQ000532
SI  - GENBANK/JQ000533
SI  - GENBANK/JQ000534
SI  - GENBANK/JQ000535
SI  - GENBANK/JQ000536
SI  - GENBANK/JQ000537
SI  - GENBANK/JQ000538
SI  - GENBANK/JQ000539
SI  - GENBANK/JQ000540
SI  - GENBANK/JQ000541
SI  - GENBANK/JQ000542
SI  - GENBANK/JQ000543
SI  - GENBANK/JQ000544
SI  - GENBANK/JQ000545
SI  - GENBANK/JQ000546
SI  - GENBANK/JQ000547
SI  - GENBANK/JQ000548
SI  - GENBANK/JQ000549
SI  - GENBANK/JQ000550
SI  - GENBANK/JQ000551
SI  - GENBANK/JQ000552
SI  - GENBANK/JQ000553
SI  - GENBANK/JQ000554
SI  - GENBANK/JQ000555
SI  - GENBANK/JQ000556
SI  - GENBANK/JQ000557
SI  - GENBANK/JQ000558
SI  - GENBANK/JQ000559
SI  - GENBANK/JQ000560
SI  - GENBANK/JQ000561
SI  - GENBANK/JQ000562
SI  - GENBANK/JQ000563
SI  - GENBANK/JQ000564
SI  - GENBANK/JQ000565
SI  - GENBANK/JQ000566
SI  - GENBANK/JQ000567
SI  - GENBANK/JQ000568
SI  - GENBANK/JQ000569
SI  - GENBANK/JQ000570
SI  - GENBANK/JQ000571
SI  - GENBANK/JQ000572
SI  - GENBANK/JQ000573
SI  - GENBANK/JQ000574
SI  - GENBANK/JQ000575
SI  - GENBANK/JQ000576
SI  - GENBANK/JQ000577
SI  - GENBANK/JQ000578
SI  - GENBANK/JQ000579
SI  - GENBANK/JQ000580
SI  - GENBANK/JQ000581
SI  - GENBANK/JQ000582
SI  - GENBANK/JQ000583
SI  - GENBANK/JQ000584
SI  - GENBANK/JQ000585
SI  - GENBANK/JQ000586
SI  - GENBANK/JQ000587
SI  - GENBANK/JQ000588
SI  - GENBANK/JQ000589
SI  - GENBANK/JQ000590
SI  - GENBANK/JQ000591
SI  - GENBANK/JQ000592
SI  - GENBANK/JQ000593
SI  - GENBANK/JQ000594
SI  - GENBANK/JQ000595
SI  - GENBANK/JQ000596
SI  - GENBANK/JQ000597
SI  - GENBANK/JQ000598
SI  - GENBANK/JQ000599
SI  - GENBANK/JQ000600
SI  - GENBANK/JQ000601
SI  - GENBANK/JQ000602
SI  - GENBANK/JQ000603
SI  - GENBANK/JQ000604
SI  - GENBANK/JQ000605
SI  - GENBANK/JQ000606
SI  - GENBANK/JQ000607
SI  - GENBANK/JQ000608
SI  - GENBANK/JQ000609
SI  - GENBANK/JQ000610
SI  - GENBANK/JQ000611
SI  - GENBANK/JQ000612
SI  - GENBANK/JQ000613
SI  - GENBANK/JQ000614
SI  - GENBANK/JQ000615
SI  - GENBANK/JQ000616
SI  - GENBANK/JQ000617
SI  - GENBANK/JQ000618
SI  - GENBANK/JQ000619
SI  - GENBANK/JQ000620
SI  - GENBANK/JQ000621
SI  - GENBANK/JQ000622
SI  - GENBANK/JQ000623
SI  - GENBANK/JQ000624
SI  - GENBANK/JQ000625
SI  - GENBANK/JQ000626
SI  - GENBANK/JQ000627
SI  - GENBANK/JQ000628
SI  - GENBANK/JQ000629
SI  - GENBANK/JQ000630
SI  - GENBANK/JQ000631
SI  - GENBANK/JQ000632
SI  - GENBANK/JQ000633
SI  - GENBANK/JQ000634
SI  - GENBANK/JQ000635
SI  - GENBANK/JQ000636
SI  - GENBANK/JQ000637
SI  - GENBANK/JQ000638
SI  - GENBANK/JQ000639
SI  - GENBANK/JQ000640
SI  - GENBANK/JQ000641
SI  - GENBANK/JQ000642
SI  - GENBANK/JQ000643
SI  - GENBANK/JQ000644
SI  - GENBANK/JQ000645
SI  - GENBANK/JQ000646
SI  - GENBANK/JQ000647
SI  - GENBANK/JQ000648
SI  - GENBANK/JQ000649
SI  - GENBANK/JQ000650
SI  - GENBANK/JQ000651
SI  - GENBANK/JQ000652
SI  - GENBANK/JQ000653
SI  - GENBANK/JQ000654
SI  - GENBANK/JQ000655
SI  - GENBANK/JQ000656
SI  - GENBANK/JQ000657
SI  - GENBANK/JQ000658
SI  - GENBANK/JQ000659
SI  - GENBANK/JQ000660
SI  - GENBANK/JQ000661
SI  - GENBANK/JQ000662
SI  - GENBANK/JQ000663
SI  - GENBANK/JQ000664
SI  - GENBANK/JQ000665
SI  - GENBANK/JQ000666
SI  - GENBANK/JQ000667
SI  - GENBANK/JQ000668
SI  - GENBANK/JQ000669
SI  - GENBANK/JQ000670
SI  - GENBANK/JQ000671
SI  - GENBANK/JQ000672
SI  - GENBANK/JQ000673
SI  - GENBANK/JQ000674
SI  - GENBANK/JQ000675
SI  - GENBANK/JQ000676
SI  - GENBANK/JQ000677
SI  - GENBANK/JQ000678
SI  - GENBANK/JQ000679
SI  - GENBANK/JQ000680
SI  - GENBANK/JQ000681
SI  - GENBANK/JQ000682
SI  - GENBANK/JQ000683
SI  - GENBANK/JQ000684
SI  - GENBANK/JQ000685
SI  - GENBANK/JQ000686
SI  - GENBANK/JQ000687
SI  - GENBANK/JQ000688
SI  - GENBANK/JQ000689
SI  - GENBANK/JQ000690
SI  - GENBANK/JQ000691
SI  - GENBANK/JQ000692
SI  - GENBANK/JQ000693
SI  - GENBANK/JQ000694
SI  - GENBANK/JQ000695
SI  - GENBANK/JQ000696
SI  - GENBANK/JQ000697
SI  - GENBANK/JQ000698
SI  - GENBANK/JQ000699
SI  - GENBANK/JQ000700
SI  - GENBANK/JQ000701
SI  - GENBANK/JQ000702
SI  - GENBANK/JQ000703
SI  - GENBANK/JQ000704
SI  - GENBANK/JQ000705
SI  - GENBANK/JQ000706
SI  - GENBANK/JQ000707
SI  - GENBANK/JQ000708
SI  - GENBANK/JQ000709
SI  - GENBANK/JQ000710
SI  - GENBANK/JQ000711
SI  - GENBANK/JQ000712
SI  - GENBANK/JQ000713
SI  - GENBANK/JQ000714
SI  - GENBANK/JQ000715
SI  - GENBANK/JQ000716
SI  - GENBANK/JQ000717
SI  - GENBANK/JQ000718
SI  - GENBANK/JQ000719
SI  - GENBANK/JQ000720
SI  - GENBANK/JQ000721
SI  - GENBANK/JQ000722
SI  - GENBANK/JQ000723
SI  - GENBANK/JQ000724
SI  - GENBANK/JQ000725
SI  - GENBANK/JQ000726
SI  - GENBANK/JQ000727
SI  - GENBANK/JQ000728
SI  - GENBANK/JQ000729
SI  - GENBANK/JQ000730
SI  - GENBANK/JQ000731
SI  - GENBANK/JQ000732
SI  - GENBANK/JQ000733
SI  - GENBANK/JQ000734
SI  - GENBANK/JQ000735
SI  - GENBANK/JQ000736
SI  - GENBANK/JQ000737
SI  - GENBANK/JQ000738
SI  - GENBANK/JQ000739
SI  - GENBANK/JQ000740
SI  - GENBANK/JQ000741
SI  - GENBANK/JQ000742
SI  - GENBANK/JQ000743
SI  - GENBANK/JQ000744
SI  - GENBANK/JQ000745
SI  - GENBANK/JQ000746
SI  - GENBANK/JQ000747
SI  - GENBANK/JQ000748
SI  - GENBANK/JQ000749
SI  - GENBANK/JQ000750
SI  - GENBANK/JQ000751
SI  - GENBANK/JQ000752
SI  - GENBANK/JQ000753
SI  - GENBANK/JQ000754
SI  - GENBANK/JQ000755
SI  - GENBANK/JQ000756
SI  - GENBANK/JQ000757
SI  - GENBANK/JQ000758
SI  - GENBANK/JQ000759
SI  - GENBANK/JQ000760
SI  - GENBANK/JQ000761
SI  - GENBANK/JQ000762
SI  - GENBANK/JQ000763
SI  - GENBANK/JQ000764
SI  - GENBANK/JQ000765
SI  - GENBANK/JQ000766
SI  - GENBANK/JQ000767
SI  - GENBANK/JQ000768
SI  - GENBANK/JQ000769
SI  - GENBANK/JQ000770
SI  - GENBANK/JQ000771
SI  - GENBANK/JQ000772
SI  - GENBANK/JQ000773
SI  - GENBANK/JQ000774
SI  - GENBANK/JQ000775
SI  - GENBANK/JQ000776
SI  - GENBANK/JQ000777
SI  - GENBANK/JQ000778
SI  - GENBANK/JQ000779
SI  - GENBANK/JQ000780
SI  - GENBANK/JQ000781
SI  - GENBANK/JQ000782
SI  - GENBANK/JQ000783
SI  - GENBANK/JQ000784
SI  - GENBANK/JQ000785
SI  - GENBANK/JQ000786
SI  - GENBANK/JQ000787
SI  - GENBANK/JQ000788
SI  - GENBANK/JQ000789
SI  - GENBANK/JQ000790
SI  - GENBANK/JQ000791
SI  - GENBANK/JQ000792
SI  - GENBANK/JQ000793
SI  - GENBANK/JQ000794
SI  - GENBANK/JQ000795
SI  - GENBANK/JQ000796
SI  - GENBANK/JQ000797
SI  - GENBANK/JQ000798
SI  - GENBANK/JQ000799
SI  - GENBANK/JQ000800
SI  - GENBANK/JQ000801
SI  - GENBANK/JQ000802
SI  - GENBANK/JQ000803
SI  - GENBANK/JQ000804
SI  - GENBANK/JQ000805
SI  - GENBANK/JQ000806
SI  - GENBANK/JQ000807
SI  - GENBANK/JQ000808
SI  - GENBANK/JQ000809
SI  - GENBANK/JQ000810
SI  - GENBANK/JQ000811
SI  - GENBANK/JQ000812
SI  - GENBANK/JQ000813
SI  - GENBANK/JQ000814
SI  - GENBANK/JQ000815
SI  - GENBANK/JQ000816
SI  - GENBANK/JQ000817
SI  - GENBANK/JQ000818
SI  - GENBANK/JQ000819
SI  - GENBANK/JQ000820
SI  - GENBANK/JQ000821
SI  - GENBANK/JQ000822
SI  - GENBANK/JQ000823
SI  - GENBANK/JQ000824
SI  - GENBANK/JQ000825
SI  - GENBANK/JQ000826
SI  - GENBANK/JQ000827
SI  - GENBANK/JQ000828
SI  - GENBANK/JQ000829
SI  - GENBANK/JQ000830
SI  - GENBANK/JQ000831
SI  - GENBANK/JQ000832
SI  - GENBANK/JQ000833
SI  - GENBANK/JQ000834
SI  - GENBANK/JQ000835
SI  - GENBANK/JQ000836
SI  - GENBANK/JQ000837
SI  - GENBANK/JQ000838
SI  - GENBANK/JQ000839
SI  - GENBANK/JQ000840
SI  - GENBANK/JQ000841
SI  - GENBANK/JQ000842
SI  - GENBANK/JQ000843
SI  - GENBANK/JQ000844
SI  - GENBANK/JQ000845
SI  - GENBANK/JQ000846
SI  - GENBANK/JQ000847
SI  - GENBANK/JQ000848
SI  - GENBANK/JQ000849
SI  - GENBANK/JQ000850
SI  - GENBANK/JQ000851
SI  - GENBANK/JQ000852
SI  - GENBANK/JQ000853
SI  - GENBANK/JQ000854
SI  - GENBANK/JQ000855
SI  - GENBANK/JQ000856
SI  - GENBANK/JQ000857
SI  - GENBANK/JQ000858
SI  - GENBANK/JQ000859
SI  - GENBANK/JQ000860
SI  - GENBANK/JQ000861
SI  - GENBANK/JQ000862
SI  - GENBANK/JQ000863
SI  - GENBANK/JQ000864
SI  - GENBANK/JQ000865
SI  - GENBANK/JQ000866
SI  - GENBANK/JQ000867
SI  - GENBANK/JQ000868
SI  - GENBANK/JQ000869
SI  - GENBANK/JQ000870
SI  - GENBANK/JQ000871
SI  - GENBANK/JQ000872
SI  - GENBANK/JQ000873
SI  - GENBANK/JQ000874
SI  - GENBANK/JQ000875
SI  - GENBANK/JQ000876
SI  - GENBANK/JQ000877
SI  - GENBANK/JQ000878
SI  - GENBANK/JQ000879
SI  - GENBANK/JQ000880
SI  - GENBANK/JQ000881
SI  - GENBANK/JQ000882
SI  - GENBANK/JQ000883
SI  - GENBANK/JQ000884
SI  - GENBANK/JQ000885
SI  - GENBANK/JQ000886
SI  - GENBANK/JQ000887
SI  - GENBANK/JQ000888
SI  - GENBANK/JQ000889
SI  - GENBANK/JQ000890
SI  - GENBANK/JQ000891
SI  - GENBANK/JQ000892
SI  - GENBANK/JQ000893
SI  - GENBANK/JQ000894
SI  - GENBANK/JQ000895
SI  - GENBANK/JQ000896
SI  - GENBANK/JQ000897
SI  - GENBANK/JQ000898
SI  - GENBANK/JQ000899
SI  - GENBANK/JQ000900
SI  - GENBANK/JQ000901
SI  - GENBANK/JQ000902
SI  - GENBANK/JQ000903
SI  - GENBANK/JQ000904
SI  - GENBANK/JQ000905
SI  - GENBANK/JQ000906
SI  - GENBANK/JQ000907
SI  - GENBANK/JQ000908
SI  - GENBANK/JQ000909
SI  - GENBANK/JQ000910
SI  - GENBANK/JQ000911
SI  - GENBANK/JQ000912
SI  - GENBANK/JQ000913
SI  - GENBANK/JQ000914
SI  - GENBANK/JQ000915
SI  - GENBANK/JQ000916
SI  - GENBANK/JQ000917
SI  - GENBANK/JQ000918
SI  - GENBANK/JQ000919
SI  - GENBANK/JQ000920
SI  - GENBANK/JQ000921
SI  - GENBANK/JQ000922
SI  - GENBANK/JQ000923
SI  - GENBANK/JQ000924
SI  - GENBANK/JQ000925
SI  - GENBANK/JQ000926
SI  - GENBANK/JQ000927
SI  - GENBANK/JQ000928
SI  - GENBANK/JQ000929
SI  - GENBANK/JQ000930
SI  - GENBANK/JQ000931
SI  - GENBANK/JQ000932
SI  - GENBANK/JQ000933
SI  - GENBANK/JQ000934
SI  - GENBANK/JQ000935
SI  - GENBANK/JQ000936
SI  - GENBANK/JQ000937
SI  - GENBANK/JQ000938
SI  - GENBANK/JQ000939
SI  - GENBANK/JQ000940
SI  - GENBANK/JQ000941
SI  - GENBANK/JQ000942
SI  - GENBANK/JQ000943
SI  - GENBANK/JQ000944
SI  - GENBANK/JQ000945
SI  - GENBANK/JQ000946
SI  - GENBANK/JQ000947
SI  - GENBANK/JQ000948
SI  - GENBANK/JQ000949
SI  - GENBANK/JQ000950
SI  - GENBANK/JQ000951
SI  - GENBANK/JQ000952
SI  - GENBANK/JQ000953
SI  - GENBANK/JQ000954
SI  - GENBANK/JQ000955
SI  - GENBANK/JQ000956
SI  - GENBANK/JQ000957
SI  - GENBANK/JQ000958
SI  - GENBANK/JQ000959
SI  - GENBANK/JQ000960
SI  - GENBANK/JQ000961
SI  - GENBANK/JQ000962
SI  - GENBANK/JQ000963
SI  - GENBANK/JQ000964
SI  - GENBANK/JQ000965
SI  - GENBANK/JQ000966
SI  - GENBANK/JQ000967
SI  - GENBANK/JQ000968
SI  - GENBANK/JQ000969
SI  - GENBANK/JQ000970
SI  - GENBANK/JQ000971
SI  - GENBANK/JQ000972
SI  - GENBANK/JQ000973
SI  - GENBANK/JQ000974
SI  - GENBANK/JQ000975
SI  - GENBANK/JQ000976
SI  - GENBANK/JQ000977
SI  - GENBANK/JQ000978
SI  - GENBANK/JQ000979
SI  - GENBANK/JQ000980
SI  - GENBANK/JQ000981
SI  - GENBANK/JQ000982
SI  - GENBANK/JQ000983
SI  - GENBANK/JQ000984
SI  - GENBANK/JQ000985
SI  - GENBANK/JQ000986
SI  - GENBANK/JQ000987
SI  - GENBANK/JQ000988
SI  - GENBANK/JQ000989
SI  - GENBANK/JQ000990
SI  - GENBANK/JQ000991
SI  - GENBANK/JQ000992
SI  - GENBANK/JQ000993
SI  - GENBANK/JQ000994
SI  - GENBANK/JQ000995
SI  - GENBANK/JQ000996
SI  - GENBANK/JQ000997
SI  - GENBANK/JQ000998
SI  - GENBANK/JQ000999
SI  - GENBANK/JQ001000
SI  - GENBANK/JQ001001
SI  - GENBANK/JQ001002
SI  - GENBANK/JQ001003
SI  - GENBANK/JQ001004
SI  - GENBANK/JQ001005
SI  - GENBANK/JQ001006
SI  - GENBANK/JQ001007
SI  - GENBANK/JQ001008
SI  - GENBANK/JQ001009
SI  - GENBANK/JQ001010
SI  - GENBANK/JQ001011
SI  - GENBANK/JQ001012
SI  - GENBANK/JQ001013
SI  - GENBANK/JQ001014
SI  - GENBANK/JQ001015
SI  - GENBANK/JQ001016
SI  - GENBANK/JQ001017
SI  - GENBANK/JQ001018
SI  - GENBANK/JQ001019
SI  - GENBANK/JQ001020
SI  - GENBANK/JQ001021
SI  - GENBANK/JQ001022
SI  - GENBANK/JQ001023
SI  - GENBANK/JQ001024
SI  - GENBANK/JQ001025
SI  - GENBANK/JQ001026
SI  - GENBANK/JQ001027
SI  - GENBANK/JQ001028
SI  - GENBANK/JQ001029
SI  - GENBANK/JQ001030
SI  - GENBANK/JQ001031
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130215
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Cell Nucleus/genetics
MH  - DNA/analysis
MH  - Genetic Speciation
MH  - *Host-Parasite Interactions
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Pyroglyphidae/cytology/genetics/*physiology
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
MH  - Vertebrates/*parasitology
EDAT- 2013/02/19 06:00
MHDA- 2013/10/01 06:00
CRDT- 2013/02/19 06:00
PHST- 2013/02/19 06:00 [entrez]
PHST- 2013/02/19 06:00 [pubmed]
PHST- 2013/10/01 06:00 [medline]
AID - syt008 [pii]
AID - 10.1093/sysbio/syt008 [doi]
PST - ppublish
SO  - Syst Biol. 2013 May 1;62(3):411-23. doi: 10.1093/sysbio/syt008. Epub 2013 Feb 15.

PMID- 23417681
OWN - NLM
STAT- MEDLINE
DCOM- 20130930
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 3
DP  - 2013 May 1
TI  - Resolving ambiguity of species limits and concatenation in multilocus sequence
      data for the construction of phylogenetic supermatrices.
PG  - 456-66
LID - 10.1093/sysbio/syt011 [doi]
AB  - Public DNA databases are becoming too large and too complex for manual methods to
      generate phylogenetic supermatrices from multiple gene sequences. Delineating the
      terminals based on taxonomic labels is no longer practical because species
      identifications are frequently incomplete and gene trees are incongruent with
      Linnaean binomials, which results in uncertainty about how to combine species
      units among unlinked loci. We developed a procedure that minimizes the problem of
      forming multilocus species units in a large phylogenetic data set using
      algorithms from graph theory. An initial step established sequence clusters for
      each locus that broadly correspond to the species level. These clusters
      frequently include sequences labeled with various binomials and specimen
      identifiers that create multiple alternatives for concatenation. To choose among 
      these possibilities, we minimize taxonomic conflict among the species units
      globally in the data set using a multipartite heuristic algorithm. The procedure 
      was applied to all available GenBank data for Coleoptera (beetles) including &gt; 10
      500 taxon labels and &gt; 23 500 sequences of 4 loci, which were grouped into 11 241
      clusters or divergent singletons by the BlastClust software. Within each cluster,
      unidentified sequences could be assigned to a species name through the
      association with fully identified sequences, resulting in 510 new identifications
      (13.9% of total unidentified sequences) of which nearly half were "trans-locus"
      identifications by clustering of sequences at a secondary locus. The limits of
      DNA-based clusters were inconsistent with the Linnaean binomials for 1518
      clusters (13.5%) that contained more than one binomial or split a single binomial
      among multiple clusters. By applying a scoring scheme for full and partial name
      matches in pairs of clusters, a maximum weight set of 7366 global species units
      was produced. Varying the match weights for partial matches had little effect on 
      the number of units, although if partial matches were disallowed, the number
      increased greatly. Trees from the resulting supermatrices generally produced tree
      topologies in good agreement with the higher taxonomy of Coleoptera, with fewer
      terminals compared with trees generated according to standard filtering of
      sequences using species labels. The study illustrates a strategy for assembling
      the tree-of-life from an ever more complex primary database.
FAU - Chesters, Douglas
AU  - Chesters D
AD  - Department of Entomology, Natural History Museum, London SW7 5BD, UK.
FAU - Vogler, Alfried P
AU  - Vogler AP
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130215
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Cluster Analysis
MH  - Coleoptera/*classification/*genetics
MH  - DNA/analysis
MH  - *Genetic Speciation
MH  - Models, Genetic
MH  - Multilocus Sequence Typing/*methods
MH  - *Phylogeny
MH  - Sequence Alignment
EDAT- 2013/02/19 06:00
MHDA- 2013/10/01 06:00
CRDT- 2013/02/19 06:00
PHST- 2013/02/19 06:00 [entrez]
PHST- 2013/02/19 06:00 [pubmed]
PHST- 2013/10/01 06:00 [medline]
AID - syt011 [pii]
AID - 10.1093/sysbio/syt011 [doi]
PST - ppublish
SO  - Syst Biol. 2013 May 1;62(3):456-66. doi: 10.1093/sysbio/syt011. Epub 2013 Feb 15.

PMID- 23417680
OWN - NLM
STAT- MEDLINE
DCOM- 20130930
LR  - 20130412
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 3
DP  - 2013 May 1
TI  - Phylogenetic signal variation in the genomes of Medicago (Fabaceae).
PG  - 424-38
LID - 10.1093/sysbio/syt009 [doi]
AB  - Genome-scale data offer the opportunity to clarify phylogenetic relationships
      that are difficult to resolve with few loci, but they can also identify genomic
      regions with evolutionary history distinct from that of the species history. We
      collected whole-genome sequence data from 29 taxa in the legume genus Medicago,
      then aligned these sequences to the Medicago truncatula reference genome to
      confidently identify 87 596 variable homologous sites. We used this data set to
      estimate phylogenetic relationships among Medicago species, to investigate the
      number of sites needed to provide robust phylogenetic estimates and to identify
      specific genomic regions supporting topologies in conflict with the genome-wide
      phylogeny. Our full genomic data set resolves relationships within the genus that
      were previously intractable. Subsampling the data reveals considerable variation 
      in phylogenetic signal and power in smaller subsets of the data. Even when
      sampling 5000 sites, no random sample of the data supports a topology identical
      to that of the genome-wide phylogeny. Phylogenetic relationships estimated from
      500-site sliding windows revealed genome regions supporting several alternative
      species relationships among recently diverged taxa, consistent with the expected 
      effects of deep coalescence or introgression in the recent history of Medicago.
FAU - Yoder, Jeremy B
AU  - Yoder JB
AD  - Department of Plant Biology, University of Minnesota, Saint Paul MN 55108, USA.
FAU - Briskine, Roman
AU  - Briskine R
FAU - Mudge, Joann
AU  - Mudge J
FAU - Farmer, Andrew
AU  - Farmer A
FAU - Paape, Timothy
AU  - Paape T
FAU - Steele, Kelly
AU  - Steele K
FAU - Weiblen, George D
AU  - Weiblen GD
FAU - Bharti, Arvind K
AU  - Bharti AK
FAU - Zhou, Peng
AU  - Zhou P
FAU - May, Gregory D
AU  - May GD
FAU - Young, Nevin D
AU  - Young ND
FAU - Tiffin, Peter
AU  - Tiffin P
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130215
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Cell Nucleus/genetics
MH  - Chloroplasts/genetics
MH  - Evolution, Molecular
MH  - Gene Library
MH  - *Genome, Plant
MH  - Medicago/cytology/*genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
EDAT- 2013/02/19 06:00
MHDA- 2013/10/01 06:00
CRDT- 2013/02/19 06:00
PHST- 2013/02/19 06:00 [entrez]
PHST- 2013/02/19 06:00 [pubmed]
PHST- 2013/10/01 06:00 [medline]
AID - syt009 [pii]
AID - 10.1093/sysbio/syt009 [doi]
PST - ppublish
SO  - Syst Biol. 2013 May 1;62(3):424-38. doi: 10.1093/sysbio/syt009. Epub 2013 Feb 15.

PMID- 23417679
OWN - NLM
STAT- MEDLINE
DCOM- 20130930
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 3
DP  - 2013 May 1
TI  - Diversity, disparity, and evolutionary rate estimation for unresolved Yule trees.
PG  - 439-55
LID - 10.1093/sysbio/syt010 [doi]
AB  - The branching structure of biological evolution confers statistical dependencies 
      on phenotypic trait values in related organisms. For this reason, comparative
      macroevolutionary studies usually begin with an inferred phylogeny that describes
      the evolutionary relationships of the organisms of interest. The probability of
      the observed trait data can be computed by assuming a model for trait evolution, 
      such as Brownian motion, over the branches of this fixed tree. However, the
      phylogenetic tree itself contributes statistical uncertainty to estimates of
      rates of phenotypic evolution, and many comparative evolutionary biologists
      regard the tree as a nuisance parameter. In this article, we present a framework 
      for analytically integrating over unknown phylogenetic trees in comparative
      evolutionary studies by assuming that the tree arises from a continuous-time
      Markov branching model called the Yule process. To do this, we derive a
      closed-form expression for the distribution of phylogenetic diversity (PD), which
      is the sum of branch lengths connecting the species in a clade. We then present a
      generalization of PD which is equivalent to the expected trait disparity in a set
      of taxa whose evolutionary relationships are generated by a Yule process and
      whose traits evolve by Brownian motion. We find expressions for the distribution 
      of expected trait disparity under a Yule tree. Given one or more observations of 
      trait disparity in a clade, we perform fast likelihood-based estimation of the
      Brownian variance for unresolved clades. Our method does not require simulation
      or a fixed phylogenetic tree. We conclude with a brief example illustrating
      Brownian rate estimation for 12 families in the mammalian order Carnivora, in
      which the phylogenetic tree for each family is unresolved.
FAU - Crawford, Forrest W
AU  - Crawford FW
AD  - Department of Biostatistics, Yale University, New Haven, CT, USA.
      forrest.crawford@yale.edu
FAU - Suchard, Marc A
AU  - Suchard MA
LA  - eng
GR  - T32GM008185/GM/NIGMS NIH HHS/United States
GR  - HG006139/HG/NHGRI NIH HHS/United States
GR  - R01 GM086887/GM/NIGMS NIH HHS/United States
GR  - R01 HG006139/HG/NHGRI NIH HHS/United States
GR  - T32 GM008185/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130215
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Body Size
MH  - Carnivora/anatomy &amp; histology/*classification/*genetics
MH  - Genetic Speciation
MH  - Likelihood Functions
MH  - *Markov Chains
MH  - *Models, Genetic
MH  - Phylogeny
PMC - PMC3622899
EDAT- 2013/02/19 06:00
MHDA- 2013/10/01 06:00
CRDT- 2013/02/19 06:00
PHST- 2013/02/19 06:00 [entrez]
PHST- 2013/02/19 06:00 [pubmed]
PHST- 2013/10/01 06:00 [medline]
AID - syt010 [pii]
AID - 10.1093/sysbio/syt010 [doi]
PST - ppublish
SO  - Syst Biol. 2013 May 1;62(3):439-55. doi: 10.1093/sysbio/syt010. Epub 2013 Feb 15.

PMID- 23391942
OWN - NLM
STAT- MEDLINE
DCOM- 20130930
LR  - 20130412
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 3
DP  - 2013 May 1
TI  - Paleo-drainage basin connectivity predicts evolutionary relationships across
      three Southeast Asian biodiversity hotspots.
PG  - 398-410
LID - 10.1093/sysbio/syt007 [doi]
AB  - Understanding factors driving diversity across biodiversity hotspots is critical 
      for formulating conservation priorities in the face of ongoing and escalating
      environmental deterioration. While biodiversity hotspots encompass a small
      fraction of Earth's land surface, more than half the world's plants and
      two-thirds of terrestrial vertebrate species are endemic to these hotspots.
      Tropical Southeast (SE) Asia displays extraordinary species richness,
      encompassing four biodiversity hotspots, though disentangling multiple potential 
      drivers of species richness is confounded by the region's dynamic geological and 
      climatic history. Here, we use multilocus molecular genetic data from dense
      multispecies sampling of freshwater fishes across three biodiversity hotspots, to
      test the effect of Quaternary climate change and resulting drainage
      rearrangements on aquatic faunal diversification. While Cenozoic geological
      processes have clearly shaped evolutionary history in SE Asian halfbeak fishes,
      we show that paleo-drainage re-arrangements resulting from Quaternary climate
      change played a significant role in the spatiotemporal evolution of lowland
      aquatic taxa, and provide priorities for conservation efforts.
FAU - de Bruyn, Mark
AU  - de Bruyn M
AD  - School of Biological Sciences, Environment Centre Wales, Bangor University,
      Deiniol Rd, Bangor LL57 2UW, UK. kus.debruyn@gmail.com
FAU - Ruber, Lukas
AU  - Ruber L
FAU - Nylinder, Stephan
AU  - Nylinder S
FAU - Stelbrink, Bjorn
AU  - Stelbrink B
FAU - Lovejoy, Nathan R
AU  - Lovejoy NR
FAU - Lavoue, Sebastien
AU  - Lavoue S
FAU - Tan, Heok Hui
AU  - Tan HH
FAU - Nugroho, Estu
AU  - Nugroho E
FAU - Wowor, Daisy
AU  - Wowor D
FAU - Ng, Peter K L
AU  - Ng PK
FAU - Siti Azizah, M N
AU  - Siti Azizah MN
FAU - Von Rintelen, Thomas
AU  - Von Rintelen T
FAU - Hall, Robert
AU  - Hall R
FAU - Carvalho, Gary R
AU  - Carvalho GR
LA  - eng
SI  - GENBANK/JQ429800
SI  - GENBANK/JQ429801
SI  - GENBANK/JQ429802
SI  - GENBANK/JQ429803
SI  - GENBANK/JQ429804
SI  - GENBANK/JQ429805
SI  - GENBANK/JQ429806
SI  - GENBANK/JQ429807
SI  - GENBANK/JQ429808
SI  - GENBANK/JQ429809
SI  - GENBANK/JQ429810
SI  - GENBANK/JQ429811
SI  - GENBANK/JQ429812
SI  - GENBANK/JQ429813
SI  - GENBANK/JQ429814
SI  - GENBANK/JQ429815
SI  - GENBANK/JQ429816
SI  - GENBANK/JQ429817
SI  - GENBANK/JQ429818
SI  - GENBANK/JQ429819
SI  - GENBANK/JQ429820
SI  - GENBANK/JQ429821
SI  - GENBANK/JQ429822
SI  - GENBANK/JQ429823
SI  - GENBANK/JQ429824
SI  - GENBANK/JQ429825
SI  - GENBANK/JQ429826
SI  - GENBANK/JQ429827
SI  - GENBANK/JQ429828
SI  - GENBANK/JQ429829
SI  - GENBANK/JQ429830
SI  - GENBANK/JQ429831
SI  - GENBANK/JQ429832
SI  - GENBANK/JQ429833
SI  - GENBANK/JQ429834
SI  - GENBANK/JQ429835
SI  - GENBANK/JQ429836
SI  - GENBANK/JQ429837
SI  - GENBANK/JQ429838
SI  - GENBANK/JQ429839
SI  - GENBANK/JQ429840
SI  - GENBANK/JQ429841
SI  - GENBANK/JQ429842
SI  - GENBANK/JQ429843
SI  - GENBANK/JQ429844
SI  - GENBANK/JQ429845
SI  - GENBANK/JQ429846
SI  - GENBANK/JQ429847
SI  - GENBANK/JQ429848
SI  - GENBANK/JQ429849
SI  - GENBANK/JQ429850
SI  - GENBANK/JQ429851
SI  - GENBANK/JQ429852
SI  - GENBANK/JQ429853
SI  - GENBANK/JQ429854
SI  - GENBANK/JQ429855
SI  - GENBANK/JQ429856
SI  - GENBANK/JQ429857
SI  - GENBANK/JQ429858
SI  - GENBANK/JQ429859
SI  - GENBANK/JQ429860
SI  - GENBANK/JQ429861
SI  - GENBANK/JQ429862
SI  - GENBANK/JQ429863
SI  - GENBANK/JQ429864
SI  - GENBANK/JQ429865
SI  - GENBANK/JQ429866
SI  - GENBANK/JQ429867
SI  - GENBANK/JQ429868
SI  - GENBANK/JQ429869
SI  - GENBANK/JQ429870
SI  - GENBANK/JQ429871
SI  - GENBANK/JQ429872
SI  - GENBANK/JQ429873
SI  - GENBANK/JQ429874
SI  - GENBANK/JQ429875
SI  - GENBANK/JQ429876
SI  - GENBANK/JQ429877
SI  - GENBANK/JQ429878
SI  - GENBANK/JQ429879
SI  - GENBANK/JQ429880
SI  - GENBANK/JQ429881
SI  - GENBANK/JQ429882
SI  - GENBANK/JQ429883
SI  - GENBANK/JQ429884
SI  - GENBANK/JQ429885
SI  - GENBANK/JQ429886
SI  - GENBANK/JQ429887
SI  - GENBANK/JQ429888
SI  - GENBANK/JQ429889
SI  - GENBANK/JQ429890
SI  - GENBANK/JQ429891
SI  - GENBANK/JQ429892
SI  - GENBANK/JQ429893
SI  - GENBANK/JQ429894
SI  - GENBANK/JQ429895
SI  - GENBANK/JQ429896
SI  - GENBANK/JQ429897
SI  - GENBANK/JQ429898
SI  - GENBANK/JQ429899
SI  - GENBANK/JQ429900
SI  - GENBANK/JQ429901
SI  - GENBANK/JQ429902
SI  - GENBANK/JQ429903
SI  - GENBANK/JQ429904
SI  - GENBANK/JQ429905
SI  - GENBANK/JQ429906
SI  - GENBANK/JQ429907
SI  - GENBANK/JQ429908
SI  - GENBANK/JQ429909
SI  - GENBANK/JQ429910
SI  - GENBANK/JQ429911
SI  - GENBANK/JQ429912
SI  - GENBANK/JQ429913
SI  - GENBANK/JQ429914
SI  - GENBANK/JQ429915
SI  - GENBANK/JQ429916
SI  - GENBANK/JQ429917
SI  - GENBANK/JQ429918
SI  - GENBANK/JQ429919
SI  - GENBANK/JQ429920
SI  - GENBANK/JQ429921
SI  - GENBANK/JQ429922
SI  - GENBANK/JQ429923
SI  - GENBANK/JQ429924
SI  - GENBANK/JQ429925
SI  - GENBANK/JQ429926
SI  - GENBANK/JQ429927
SI  - GENBANK/JQ429928
SI  - GENBANK/JQ429929
SI  - GENBANK/JQ429930
SI  - GENBANK/JQ429931
SI  - GENBANK/JQ429932
SI  - GENBANK/JQ429933
SI  - GENBANK/JQ429934
SI  - GENBANK/JQ429935
SI  - GENBANK/JQ429936
SI  - GENBANK/JQ429937
SI  - GENBANK/JQ429938
SI  - GENBANK/JQ429939
SI  - GENBANK/JQ429940
SI  - GENBANK/JQ429941
SI  - GENBANK/JQ429942
SI  - GENBANK/JQ429943
SI  - GENBANK/JQ429944
SI  - GENBANK/JQ429945
SI  - GENBANK/JQ429946
SI  - GENBANK/JQ429947
SI  - GENBANK/JQ429948
SI  - GENBANK/JQ429949
SI  - GENBANK/JQ429950
SI  - GENBANK/JQ429951
SI  - GENBANK/JQ429952
SI  - GENBANK/JQ429953
SI  - GENBANK/JQ429954
SI  - GENBANK/JQ429955
SI  - GENBANK/JQ429956
SI  - GENBANK/JQ429957
SI  - GENBANK/JQ429958
SI  - GENBANK/JQ429959
SI  - GENBANK/JQ429960
SI  - GENBANK/JQ429961
SI  - GENBANK/JQ429962
SI  - GENBANK/JQ429963
SI  - GENBANK/JQ429964
SI  - GENBANK/JQ429965
SI  - GENBANK/JQ429966
SI  - GENBANK/JQ429967
SI  - GENBANK/JQ429968
SI  - GENBANK/JQ429969
SI  - GENBANK/JQ429970
SI  - GENBANK/JQ429971
SI  - GENBANK/JQ429972
SI  - GENBANK/JQ429973
SI  - GENBANK/JQ429974
SI  - GENBANK/JQ429975
SI  - GENBANK/JQ429976
SI  - GENBANK/JQ429977
SI  - GENBANK/JQ429978
SI  - GENBANK/JQ429979
SI  - GENBANK/JQ429980
SI  - GENBANK/JQ429981
SI  - GENBANK/JQ429982
SI  - GENBANK/JQ429983
SI  - GENBANK/JQ429984
SI  - GENBANK/JQ429985
SI  - GENBANK/JQ429986
SI  - GENBANK/JQ429987
SI  - GENBANK/JQ429988
SI  - GENBANK/JQ429989
SI  - GENBANK/JQ429990
SI  - GENBANK/JQ429991
SI  - GENBANK/JQ429992
SI  - GENBANK/JQ429993
SI  - GENBANK/JQ429994
SI  - GENBANK/JQ429995
SI  - GENBANK/JQ429996
SI  - GENBANK/JQ429997
SI  - GENBANK/JQ429998
SI  - GENBANK/JQ429999
SI  - GENBANK/JQ430000
SI  - GENBANK/JQ430001
SI  - GENBANK/JQ430002
SI  - GENBANK/JQ430003
SI  - GENBANK/JQ430004
SI  - GENBANK/JQ430005
SI  - GENBANK/JQ430006
SI  - GENBANK/JQ430007
SI  - GENBANK/JQ430008
SI  - GENBANK/JQ430009
SI  - GENBANK/JQ430010
SI  - GENBANK/JQ430011
SI  - GENBANK/JQ430012
SI  - GENBANK/JQ430013
SI  - GENBANK/JQ430014
SI  - GENBANK/JQ430015
SI  - GENBANK/JQ430016
SI  - GENBANK/JQ430017
SI  - GENBANK/JQ430018
SI  - GENBANK/JQ430019
SI  - GENBANK/JQ430020
SI  - GENBANK/JQ430021
SI  - GENBANK/JQ430022
SI  - GENBANK/JQ430023
SI  - GENBANK/JQ430024
SI  - GENBANK/JQ430025
SI  - GENBANK/JQ430026
SI  - GENBANK/JQ430027
SI  - GENBANK/JQ430028
SI  - GENBANK/JQ430029
SI  - GENBANK/JQ430030
SI  - GENBANK/JQ430031
SI  - GENBANK/JQ430032
SI  - GENBANK/JQ430033
SI  - GENBANK/JQ430034
SI  - GENBANK/JQ430035
SI  - GENBANK/JQ430036
SI  - GENBANK/JQ430037
SI  - GENBANK/JQ430038
SI  - GENBANK/JQ430039
SI  - GENBANK/JQ430040
SI  - GENBANK/JQ430041
SI  - GENBANK/JQ430042
SI  - GENBANK/JQ430043
SI  - GENBANK/JQ430044
SI  - GENBANK/JQ430045
SI  - GENBANK/JQ430046
SI  - GENBANK/JQ430047
SI  - GENBANK/JQ430048
SI  - GENBANK/JQ430049
SI  - GENBANK/JQ430050
SI  - GENBANK/JQ430051
SI  - GENBANK/JQ430052
SI  - GENBANK/JQ430053
SI  - GENBANK/JQ430054
SI  - GENBANK/JQ430055
SI  - GENBANK/JQ430056
SI  - GENBANK/JQ430057
SI  - GENBANK/JQ430058
SI  - GENBANK/JQ430059
SI  - GENBANK/JQ430060
SI  - GENBANK/JQ430061
SI  - GENBANK/JQ430062
SI  - GENBANK/JQ430063
SI  - GENBANK/JQ430064
SI  - GENBANK/JQ430065
SI  - GENBANK/JQ430066
SI  - GENBANK/JQ430067
SI  - GENBANK/JQ430068
SI  - GENBANK/JQ430069
SI  - GENBANK/JQ430070
SI  - GENBANK/JQ430071
SI  - GENBANK/JQ430072
SI  - GENBANK/JQ430073
SI  - GENBANK/JQ430074
SI  - GENBANK/JQ430075
SI  - GENBANK/JQ430076
SI  - GENBANK/JQ430077
SI  - GENBANK/JQ430078
SI  - GENBANK/JQ430079
SI  - GENBANK/JQ430080
SI  - GENBANK/JQ430081
SI  - GENBANK/JQ430082
SI  - GENBANK/JQ430083
SI  - GENBANK/JQ430084
SI  - GENBANK/JQ430085
SI  - GENBANK/JQ430086
SI  - GENBANK/JQ430087
SI  - GENBANK/JQ430088
SI  - GENBANK/JQ430089
SI  - GENBANK/JQ430090
SI  - GENBANK/JQ430091
SI  - GENBANK/JQ430092
SI  - GENBANK/JQ430093
SI  - GENBANK/JQ430094
SI  - GENBANK/JQ430095
SI  - GENBANK/JQ430096
SI  - GENBANK/JQ430097
SI  - GENBANK/JQ430098
SI  - GENBANK/JQ430099
SI  - GENBANK/JQ430100
SI  - GENBANK/JQ430101
SI  - GENBANK/JQ430102
SI  - GENBANK/JQ430103
SI  - GENBANK/JQ430104
SI  - GENBANK/JQ430105
SI  - GENBANK/JQ430106
SI  - GENBANK/JQ430107
SI  - GENBANK/JQ430108
SI  - GENBANK/JQ430109
SI  - GENBANK/JQ430110
SI  - GENBANK/JQ430111
SI  - GENBANK/JQ430112
SI  - GENBANK/JQ430113
SI  - GENBANK/JQ430114
SI  - GENBANK/JQ430115
SI  - GENBANK/JQ430116
SI  - GENBANK/JQ430117
SI  - GENBANK/JQ430118
SI  - GENBANK/JQ430119
SI  - GENBANK/JQ430120
SI  - GENBANK/JQ430121
SI  - GENBANK/JQ430122
SI  - GENBANK/JQ430123
SI  - GENBANK/JQ430124
SI  - GENBANK/JQ430125
SI  - GENBANK/JQ430126
SI  - GENBANK/JQ430127
SI  - GENBANK/JQ430128
SI  - GENBANK/JQ430129
SI  - GENBANK/JQ430130
SI  - GENBANK/JQ430131
SI  - GENBANK/JQ430132
SI  - GENBANK/JQ430133
SI  - GENBANK/JQ430134
SI  - GENBANK/JQ430135
SI  - GENBANK/JQ430136
SI  - GENBANK/JQ430137
SI  - GENBANK/JQ430138
SI  - GENBANK/JQ430139
SI  - GENBANK/JQ430140
SI  - GENBANK/JQ430141
SI  - GENBANK/JQ430142
SI  - GENBANK/JQ430143
SI  - GENBANK/JQ430144
SI  - GENBANK/JQ430145
SI  - GENBANK/JQ430146
SI  - GENBANK/JQ430147
SI  - GENBANK/JQ430148
SI  - GENBANK/JQ430149
SI  - GENBANK/JQ430150
SI  - GENBANK/JQ430151
SI  - GENBANK/JQ430152
SI  - GENBANK/JQ430153
SI  - GENBANK/JQ430154
SI  - GENBANK/JQ430155
SI  - GENBANK/JQ430156
SI  - GENBANK/JQ430157
SI  - GENBANK/JQ430158
SI  - GENBANK/JQ430159
SI  - GENBANK/JQ430160
SI  - GENBANK/JQ430161
SI  - GENBANK/JQ430162
SI  - GENBANK/JQ430163
SI  - GENBANK/JQ430164
SI  - GENBANK/JQ430165
SI  - GENBANK/JQ430166
SI  - GENBANK/JQ430167
SI  - GENBANK/JQ430168
SI  - GENBANK/JQ430169
SI  - GENBANK/JQ430170
SI  - GENBANK/JQ430171
SI  - GENBANK/JQ430172
SI  - GENBANK/JQ430173
SI  - GENBANK/JQ430174
SI  - GENBANK/JQ430175
SI  - GENBANK/JQ430176
SI  - GENBANK/JQ430177
SI  - GENBANK/JQ430178
SI  - GENBANK/JQ430179
SI  - GENBANK/JQ430180
SI  - GENBANK/JQ430181
SI  - GENBANK/JQ430182
SI  - GENBANK/JQ430183
SI  - GENBANK/JQ430184
SI  - GENBANK/JQ430185
SI  - GENBANK/JQ430186
SI  - GENBANK/JQ430187
SI  - GENBANK/JQ430188
SI  - GENBANK/JQ430189
SI  - GENBANK/JQ430190
SI  - GENBANK/JQ430191
SI  - GENBANK/JQ430192
SI  - GENBANK/JQ430193
SI  - GENBANK/JQ430194
SI  - GENBANK/JQ430195
SI  - GENBANK/JQ430196
SI  - GENBANK/JQ430197
SI  - GENBANK/JQ430198
SI  - GENBANK/JQ430199
SI  - GENBANK/JQ430200
SI  - GENBANK/JQ430201
SI  - GENBANK/JQ430202
SI  - GENBANK/JQ430203
SI  - GENBANK/JQ430204
SI  - GENBANK/JQ430205
SI  - GENBANK/JQ430206
SI  - GENBANK/JQ430207
SI  - GENBANK/JQ430208
SI  - GENBANK/JQ430209
SI  - GENBANK/JQ430210
SI  - GENBANK/JQ430211
SI  - GENBANK/JQ430212
SI  - GENBANK/JQ430213
SI  - GENBANK/JQ430214
SI  - GENBANK/JQ430215
SI  - GENBANK/JQ430216
SI  - GENBANK/JQ430217
SI  - GENBANK/JQ430218
SI  - GENBANK/JQ430219
SI  - GENBANK/JQ430220
SI  - GENBANK/JQ430221
SI  - GENBANK/JQ430222
SI  - GENBANK/JQ430223
SI  - GENBANK/JQ430224
SI  - GENBANK/JQ430225
SI  - GENBANK/JQ430226
SI  - GENBANK/JQ430227
SI  - GENBANK/JQ430228
SI  - GENBANK/JQ430229
SI  - GENBANK/JQ430230
SI  - GENBANK/JQ430231
SI  - GENBANK/JQ430232
SI  - GENBANK/JQ430233
SI  - GENBANK/JQ430234
SI  - GENBANK/JQ430235
SI  - GENBANK/JQ430236
SI  - GENBANK/JQ430237
SI  - GENBANK/JQ430238
SI  - GENBANK/JQ430239
SI  - GENBANK/JQ430240
SI  - GENBANK/JQ430241
SI  - GENBANK/JQ430242
SI  - GENBANK/JQ430243
SI  - GENBANK/JQ430244
SI  - GENBANK/JQ430245
SI  - GENBANK/JQ430246
SI  - GENBANK/JQ430247
SI  - GENBANK/JQ430248
SI  - GENBANK/JQ430249
SI  - GENBANK/JQ430250
SI  - GENBANK/JQ430251
SI  - GENBANK/JQ430252
SI  - GENBANK/JQ430253
SI  - GENBANK/JQ430254
SI  - GENBANK/JQ430255
SI  - GENBANK/JQ430256
SI  - GENBANK/JQ430257
SI  - GENBANK/JQ430258
SI  - GENBANK/JQ430259
SI  - GENBANK/JQ430260
SI  - GENBANK/JQ430261
SI  - GENBANK/JQ430262
SI  - GENBANK/JQ430263
SI  - GENBANK/JQ430264
SI  - GENBANK/JQ430265
SI  - GENBANK/JQ430266
SI  - GENBANK/JQ430267
SI  - GENBANK/JQ430268
SI  - GENBANK/JQ430269
SI  - GENBANK/JQ430270
SI  - GENBANK/JQ430271
SI  - GENBANK/JQ430272
SI  - GENBANK/JQ430273
SI  - GENBANK/JQ430274
SI  - GENBANK/JQ430275
SI  - GENBANK/JQ430276
SI  - GENBANK/JQ430277
SI  - GENBANK/JQ430278
SI  - GENBANK/JQ430279
SI  - GENBANK/JQ430280
SI  - GENBANK/JQ430281
SI  - GENBANK/JQ430282
SI  - GENBANK/JQ430283
SI  - GENBANK/JQ430284
SI  - GENBANK/JQ430285
SI  - GENBANK/JQ430286
SI  - GENBANK/JQ430287
SI  - GENBANK/JQ430288
SI  - GENBANK/JQ430289
SI  - GENBANK/JQ430290
SI  - GENBANK/JQ430291
SI  - GENBANK/JQ430292
SI  - GENBANK/JQ430293
SI  - GENBANK/JQ430294
SI  - GENBANK/JQ430295
SI  - GENBANK/JQ430296
SI  - GENBANK/JQ430297
SI  - GENBANK/JQ430298
SI  - GENBANK/JQ430299
SI  - GENBANK/JQ430300
SI  - GENBANK/JQ430301
SI  - GENBANK/JQ430302
SI  - GENBANK/JQ430303
SI  - GENBANK/JQ430304
SI  - GENBANK/JQ430305
SI  - GENBANK/JQ430306
SI  - GENBANK/JQ430307
SI  - GENBANK/JQ430308
SI  - GENBANK/JQ430309
SI  - GENBANK/JQ430310
SI  - GENBANK/JQ430311
SI  - GENBANK/JQ430312
SI  - GENBANK/JQ430313
SI  - GENBANK/JQ430314
SI  - GENBANK/JQ430315
SI  - GENBANK/JQ430316
SI  - GENBANK/JQ430317
SI  - GENBANK/JQ430318
SI  - GENBANK/JQ430319
SI  - GENBANK/JQ430320
SI  - GENBANK/JQ430321
SI  - GENBANK/JQ430322
SI  - GENBANK/JQ430323
SI  - GENBANK/JQ430324
SI  - GENBANK/JQ430325
SI  - GENBANK/JQ430326
SI  - GENBANK/JQ430327
SI  - GENBANK/JQ430328
SI  - GENBANK/JQ430329
SI  - GENBANK/JQ430330
SI  - GENBANK/JQ430331
SI  - GENBANK/JQ430332
SI  - GENBANK/JQ430333
SI  - GENBANK/JQ430334
SI  - GENBANK/JQ430335
SI  - GENBANK/JQ430336
SI  - GENBANK/JQ430337
SI  - GENBANK/JQ430338
SI  - GENBANK/JQ430339
SI  - GENBANK/JQ430340
SI  - GENBANK/JQ430341
SI  - GENBANK/JQ430342
SI  - GENBANK/JQ430343
SI  - GENBANK/JQ430344
SI  - GENBANK/JQ430345
SI  - GENBANK/JQ430346
SI  - GENBANK/JQ430347
SI  - GENBANK/JQ430348
SI  - GENBANK/JQ430349
SI  - GENBANK/JQ430350
SI  - GENBANK/JQ430351
SI  - GENBANK/JQ430352
SI  - GENBANK/JQ430353
SI  - GENBANK/JQ430354
SI  - GENBANK/JQ430355
SI  - GENBANK/JQ430356
SI  - GENBANK/JQ430357
SI  - GENBANK/JQ430358
SI  - GENBANK/JQ430359
SI  - GENBANK/JQ430360
SI  - GENBANK/JQ430361
SI  - GENBANK/JQ430362
SI  - GENBANK/JQ430363
SI  - GENBANK/JQ430364
SI  - GENBANK/JQ430365
SI  - GENBANK/JQ430366
SI  - GENBANK/JQ430367
SI  - GENBANK/JQ430368
SI  - GENBANK/JQ430369
SI  - GENBANK/JQ430370
SI  - GENBANK/JQ430371
SI  - GENBANK/JQ430372
SI  - GENBANK/JQ430373
SI  - GENBANK/JQ430374
SI  - GENBANK/JQ430375
SI  - GENBANK/JQ430376
SI  - GENBANK/JQ430377
SI  - GENBANK/JQ430378
SI  - GENBANK/JQ430379
SI  - GENBANK/JQ430380
SI  - GENBANK/JQ430381
SI  - GENBANK/JQ430382
SI  - GENBANK/JQ430383
SI  - GENBANK/JQ430384
SI  - GENBANK/JQ430385
SI  - GENBANK/JQ430386
SI  - GENBANK/JQ430387
SI  - GENBANK/JQ430388
SI  - GENBANK/JQ430389
SI  - GENBANK/JQ430390
SI  - GENBANK/JQ430391
SI  - GENBANK/JQ430392
SI  - GENBANK/JQ430393
SI  - GENBANK/JQ430394
SI  - GENBANK/JQ430395
SI  - GENBANK/JQ430396
SI  - GENBANK/JQ430397
SI  - GENBANK/JQ430398
SI  - GENBANK/JQ430399
SI  - GENBANK/JQ430400
SI  - GENBANK/JQ430401
SI  - GENBANK/JQ430402
SI  - GENBANK/JQ430403
SI  - GENBANK/JQ430404
SI  - GENBANK/JQ430405
SI  - GENBANK/JQ430406
SI  - GENBANK/JQ430407
SI  - GENBANK/JQ430408
SI  - GENBANK/JQ430409
SI  - GENBANK/JQ430410
SI  - GENBANK/JQ430411
SI  - GENBANK/JQ430412
SI  - GENBANK/JQ430413
SI  - GENBANK/JQ430414
SI  - GENBANK/JQ430415
SI  - GENBANK/JQ430416
SI  - GENBANK/JQ430417
SI  - GENBANK/JQ430418
SI  - GENBANK/JQ430419
SI  - GENBANK/JQ430420
SI  - GENBANK/JQ430421
SI  - GENBANK/JQ430422
SI  - GENBANK/JQ430423
SI  - GENBANK/JQ430424
SI  - GENBANK/JQ430425
SI  - GENBANK/JQ430426
SI  - GENBANK/JQ430427
SI  - GENBANK/JQ430428
SI  - GENBANK/JQ430429
SI  - GENBANK/JQ430430
SI  - GENBANK/JQ430431
SI  - GENBANK/JQ430432
SI  - GENBANK/JQ430433
SI  - GENBANK/JQ430434
SI  - GENBANK/JQ430435
SI  - GENBANK/JQ430436
SI  - GENBANK/JQ430437
SI  - GENBANK/JQ430438
SI  - GENBANK/JQ430439
SI  - GENBANK/JQ430440
SI  - GENBANK/JQ430441
SI  - GENBANK/JQ430442
SI  - GENBANK/JQ430443
SI  - GENBANK/JQ430444
SI  - GENBANK/JQ430445
SI  - GENBANK/JQ430446
SI  - GENBANK/JQ430447
SI  - GENBANK/JQ430448
SI  - GENBANK/JQ430449
SI  - GENBANK/JQ430450
SI  - GENBANK/JQ430451
SI  - GENBANK/JQ430452
SI  - GENBANK/JQ430453
SI  - GENBANK/JQ430454
SI  - GENBANK/JQ430455
SI  - GENBANK/JQ430456
SI  - GENBANK/JQ430457
SI  - GENBANK/JQ430458
SI  - GENBANK/JQ430459
SI  - GENBANK/JQ430460
SI  - GENBANK/JQ430461
SI  - GENBANK/JQ430462
SI  - GENBANK/JQ430463
SI  - GENBANK/JQ430464
SI  - GENBANK/JQ430465
SI  - GENBANK/JQ430466
SI  - GENBANK/JQ430467
SI  - GENBANK/JQ430468
SI  - GENBANK/JQ430469
SI  - GENBANK/JQ430470
SI  - GENBANK/JQ430471
SI  - GENBANK/JQ430472
SI  - GENBANK/JQ430473
SI  - GENBANK/JQ430474
SI  - GENBANK/JQ430475
SI  - GENBANK/JQ430476
SI  - GENBANK/JQ430477
SI  - GENBANK/JQ430478
SI  - GENBANK/JQ430479
SI  - GENBANK/JQ430480
SI  - GENBANK/JQ430481
SI  - GENBANK/JQ430482
SI  - GENBANK/JQ430483
SI  - GENBANK/JQ430484
SI  - GENBANK/JQ430485
SI  - GENBANK/JQ430486
SI  - GENBANK/JQ430487
SI  - GENBANK/JQ430488
SI  - GENBANK/JQ430489
SI  - GENBANK/JQ430490
SI  - GENBANK/JQ430491
SI  - GENBANK/JQ430492
SI  - GENBANK/JQ430493
SI  - GENBANK/JQ430494
SI  - GENBANK/JQ430495
SI  - GENBANK/JQ430496
SI  - GENBANK/JQ430497
SI  - GENBANK/JQ430498
SI  - GENBANK/JQ430499
SI  - GENBANK/JQ430500
SI  - GENBANK/JQ430501
SI  - GENBANK/JQ430502
SI  - GENBANK/JQ430503
SI  - GENBANK/JQ430504
SI  - GENBANK/JQ430505
SI  - GENBANK/JQ430506
SI  - GENBANK/JQ430507
SI  - GENBANK/JQ430508
SI  - GENBANK/JQ430509
SI  - GENBANK/JQ430510
SI  - GENBANK/JQ430511
SI  - GENBANK/JQ430512
SI  - GENBANK/JQ430513
SI  - GENBANK/JQ430514
SI  - GENBANK/JQ430515
SI  - GENBANK/JQ430516
SI  - GENBANK/JQ430517
SI  - GENBANK/JQ430518
SI  - GENBANK/JQ430519
SI  - GENBANK/JQ430520
SI  - GENBANK/JQ430521
SI  - GENBANK/JQ430522
SI  - GENBANK/JQ430523
SI  - GENBANK/JQ430524
SI  - GENBANK/JQ430525
SI  - GENBANK/JQ430526
SI  - GENBANK/JQ430527
SI  - GENBANK/JQ430528
SI  - GENBANK/JQ430529
SI  - GENBANK/JQ430530
SI  - GENBANK/JQ430531
SI  - GENBANK/JQ430532
SI  - GENBANK/JQ430533
SI  - GENBANK/JQ430534
SI  - GENBANK/JQ430535
SI  - GENBANK/JQ430536
SI  - GENBANK/JQ430537
SI  - GENBANK/JQ430538
SI  - GENBANK/JQ430539
SI  - GENBANK/JQ430540
SI  - GENBANK/JQ430541
SI  - GENBANK/JQ430542
SI  - GENBANK/JQ430543
SI  - GENBANK/JQ430544
SI  - GENBANK/JQ430545
SI  - GENBANK/JQ430546
SI  - GENBANK/JQ430547
SI  - GENBANK/JQ430548
SI  - GENBANK/JQ430549
SI  - GENBANK/JQ430550
SI  - GENBANK/JQ430551
SI  - GENBANK/JQ430552
SI  - GENBANK/JQ430553
SI  - GENBANK/JQ430554
SI  - GENBANK/JQ430555
SI  - GENBANK/JQ430556
SI  - GENBANK/JQ430557
SI  - GENBANK/JQ430558
SI  - GENBANK/JQ430559
SI  - GENBANK/JQ430560
SI  - GENBANK/JQ430561
SI  - GENBANK/JQ430562
SI  - GENBANK/JQ430563
SI  - GENBANK/JQ430564
SI  - GENBANK/JQ430565
SI  - GENBANK/JQ430566
SI  - GENBANK/JQ430567
SI  - GENBANK/JQ430568
SI  - GENBANK/JQ430569
SI  - GENBANK/JQ430570
SI  - GENBANK/JQ430571
SI  - GENBANK/JQ430572
SI  - GENBANK/JQ430573
SI  - GENBANK/JQ430574
SI  - GENBANK/JQ430575
SI  - GENBANK/JQ430576
SI  - GENBANK/JQ430577
SI  - GENBANK/JQ430578
SI  - GENBANK/JQ430579
SI  - GENBANK/JQ430580
SI  - GENBANK/JQ430581
SI  - GENBANK/JQ430582
SI  - GENBANK/JQ430583
SI  - GENBANK/JQ430584
SI  - GENBANK/JQ430585
SI  - GENBANK/JQ430586
SI  - GENBANK/JQ430587
SI  - GENBANK/JQ430588
SI  - GENBANK/JQ430589
SI  - GENBANK/JQ430590
SI  - GENBANK/JQ430591
SI  - GENBANK/JQ430592
SI  - GENBANK/JQ430593
SI  - GENBANK/JQ430594
SI  - GENBANK/JQ430595
SI  - GENBANK/JQ430596
SI  - GENBANK/JQ430597
SI  - GENBANK/JQ430598
SI  - GENBANK/JQ430599
SI  - GENBANK/JQ430600
SI  - GENBANK/JQ430601
SI  - GENBANK/JQ430602
SI  - GENBANK/JQ430603
SI  - GENBANK/JQ430604
SI  - GENBANK/JQ430605
SI  - GENBANK/JQ430606
SI  - GENBANK/JQ430607
SI  - GENBANK/JQ430608
SI  - GENBANK/JQ430609
SI  - GENBANK/JQ430610
SI  - GENBANK/JQ430611
SI  - GENBANK/JQ430612
SI  - GENBANK/JQ430613
SI  - GENBANK/JQ430614
SI  - GENBANK/JQ430615
SI  - GENBANK/JQ430616
SI  - GENBANK/JQ430617
SI  - GENBANK/JQ430618
SI  - GENBANK/JQ430619
SI  - GENBANK/JQ430620
SI  - GENBANK/JQ430621
SI  - GENBANK/JQ430622
SI  - GENBANK/JQ430623
SI  - GENBANK/JQ430624
SI  - GENBANK/JQ430625
SI  - GENBANK/JQ430626
SI  - GENBANK/JQ430627
SI  - GENBANK/JQ430628
SI  - GENBANK/JQ430629
SI  - GENBANK/JQ430630
SI  - GENBANK/JQ430631
SI  - GENBANK/JQ430632
SI  - GENBANK/JQ430633
SI  - GENBANK/JQ430634
SI  - GENBANK/JQ430635
SI  - GENBANK/JQ430636
SI  - GENBANK/JQ430637
SI  - GENBANK/JQ430638
SI  - GENBANK/JQ430639
SI  - GENBANK/JQ430640
SI  - GENBANK/JQ430641
SI  - GENBANK/JQ430642
SI  - GENBANK/JQ430643
SI  - GENBANK/JQ430644
SI  - GENBANK/JQ430645
SI  - GENBANK/JQ430646
SI  - GENBANK/JQ430647
SI  - GENBANK/JQ430648
SI  - GENBANK/JQ430649
SI  - GENBANK/JQ430650
SI  - GENBANK/JQ430651
SI  - GENBANK/JQ430652
SI  - GENBANK/JQ430653
SI  - GENBANK/JQ430654
SI  - GENBANK/JQ430655
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130207
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Asia, Southeastern
MH  - Beloniformes/classification/*genetics
MH  - Biodiversity
MH  - Biological Evolution
MH  - *Climate Change
MH  - Conservation of Natural Resources
MH  - DNA/analysis
MH  - *Ecosystem
MH  - *Evolution, Molecular
MH  - Fresh Water
MH  - *Genetic Speciation
MH  - Molecular Sequence Data
MH  - Multilocus Sequence Typing
MH  - Phylogeny
MH  - Polymerase Chain Reaction
EDAT- 2013/02/09 06:00
MHDA- 2013/10/01 06:00
CRDT- 2013/02/09 06:00
PHST- 2013/02/09 06:00 [entrez]
PHST- 2013/02/09 06:00 [pubmed]
PHST- 2013/10/01 06:00 [medline]
AID - syt007 [pii]
AID - 10.1093/sysbio/syt007 [doi]
PST - ppublish
SO  - Syst Biol. 2013 May 1;62(3):398-410. doi: 10.1093/sysbio/syt007. Epub 2013 Feb 7.

PMID- 23362112
OWN - NLM
STAT- MEDLINE
DCOM- 20130930
LR  - 20140926
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 3
DP  - 2013 May 1
TI  - Are there really twice as many bovid species as we thought?
PG  - 490-3
LID - 10.1093/sysbio/syt004 [doi]
FAU - Heller, Rasmus
AU  - Heller R
AD  - Department of Biology, University of Copenhagen, Ole Maaloes Vej 5, DK-2200
      Copenhagen N, Denmark. rheller@bio.ku.dk
FAU - Frandsen, Peter
AU  - Frandsen P
FAU - Lorenzen, Eline D
AU  - Lorenzen ED
FAU - Siegismund, Hans R
AU  - Siegismund HR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130129
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2014 Sep;63(5):819-32. PMID: 24415680
CIN - Syst Biol. 2014 Sep;63(5):833-7. PMID: 24831669
MH  - Animals
MH  - Biological Evolution
MH  - *Genetic Speciation
MH  - Phylogeny
MH  - Ruminants/*classification/*genetics
EDAT- 2013/01/31 06:00
MHDA- 2013/10/01 06:00
CRDT- 2013/01/31 06:00
PHST- 2013/01/31 06:00 [entrez]
PHST- 2013/01/31 06:00 [pubmed]
PHST- 2013/10/01 06:00 [medline]
AID - syt004 [pii]
AID - 10.1093/sysbio/syt004 [doi]
PST - ppublish
SO  - Syst Biol. 2013 May 1;62(3):490-3. doi: 10.1093/sysbio/syt004. Epub 2013 Jan 29.

PMID- 23355531
OWN - NLM
STAT- MEDLINE
DCOM- 20130930
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 3
DP  - 2013 May 1
TI  - Lateral gene transfer from the dead.
PG  - 386-97
LID - 10.1093/sysbio/syt003 [doi]
AB  - In phylogenetic studies, the evolution of molecular sequences is assumed to have 
      taken place along the phylogeny traced by the ancestors of extant species. In the
      presence of lateral gene transfer, however, this may not be the case, because the
      species lineage from which a gene was transferred may have gone extinct or not
      have been sampled. Because it is not feasible to specify or reconstruct the
      complete phylogeny of all species, we must describe the evolution of genes
      outside the represented phylogeny by modeling the speciation dynamics that gave
      rise to the complete phylogeny. We demonstrate that if the number of sampled
      species is small compared with the total number of existing species, the
      overwhelming majority of gene transfers involve speciation to and evolution along
      extinct or unsampled lineages. We show that the evolution of genes along extinct 
      or unsampled lineages can to good approximation be treated as those of
      independently evolving lineages described by a few global parameters. Using this 
      result, we derive an algorithm to calculate the probability of a gene tree and
      recover the maximum-likelihood reconciliation given the phylogeny of the sampled 
      species. Examining 473 near-universal gene families from 36 cyanobacteria, we
      find that nearly a third of transfer events (28%) appear to have topological
      signatures of evolution along extinct species, but only approximately 6% of
      transfers trace their ancestry to before the common ancestor of the sampled
      cyanobacteria.
FAU - Szollosi, Gergely J
AU  - Szollosi GJ
AD  - Laboratoire de Biometrie et Biologie Evolutive, Centre National de la Recherche
      Scientifique, Unite Mixte de Recherche 5558, Universite Lyon 1, F-69622
      Villeurbanne, France. ssolo@elte.hu
FAU - Tannier, Eric
AU  - Tannier E
FAU - Lartillot, Nicolas
AU  - Lartillot N
FAU - Daubin, Vincent
AU  - Daubin V
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130125
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Bacterial)
SB  - IM
MH  - Algorithms
MH  - Biodiversity
MH  - Biological Evolution
MH  - Cyanobacteria/*genetics
MH  - DNA, Bacterial/*analysis
MH  - *Evolution, Molecular
MH  - *Gene Transfer, Horizontal
MH  - *Genes, Bacterial
MH  - Genetic Speciation
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
PMC - PMC3622898
EDAT- 2013/01/29 06:00
MHDA- 2013/10/01 06:00
CRDT- 2013/01/29 06:00
PHST- 2013/01/29 06:00 [entrez]
PHST- 2013/01/29 06:00 [pubmed]
PHST- 2013/10/01 06:00 [medline]
AID - syt003 [pii]
AID - 10.1093/sysbio/syt003 [doi]
PST - ppublish
SO  - Syst Biol. 2013 May 1;62(3):386-97. doi: 10.1093/sysbio/syt003. Epub 2013 Jan 25.

PMID- 23325808
OWN - NLM
STAT- MEDLINE
DCOM- 20130930
LR  - 20130412
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 3
DP  - 2013 May 1
TI  - Geometric morphometric character suites as phylogenetic data: extracting
      phylogenetic signal from gastropod shells.
PG  - 366-85
LID - 10.1093/sysbio/syt002 [doi]
AB  - Despite being the objects of numerous macroevolutionary studies, many of the best
      represented constituents of the fossil record-including diverse examples such as 
      foraminifera, brachiopods, and mollusks-have mineralized skeletons with limited
      discrete characteristics, making morphological phylogenies difficult to
      construct. In contrast to their paucity of phylogenetic characters, the
      mineralized structures (tests and shells) of these fossil groups frequently have 
      distinctive shapes that have long proved useful for their classification. The
      recent introduction of methodologies for including continuous data directly in a 
      phylogenetic analysis has increased the number of available characters, making it
      possible to produce phylogenies based, in whole or part, on continuous character 
      data collected from such taxa. Geometric morphometric methods provide tools for
      accurately characterizing shape variation and can produce quantitative data that 
      can therefore now be included in a phylogenetic matrix in a nonarbitrary manner. 
      Here, the marine gastropod genus Conus is used to evaluate the ability of
      continuous characters-generated from a geometric morphometric analysis of shell
      shape-to contribute to a total evidence phylogenetic hypothesis constructed using
      molecular and morphological data. Furthermore, the ability of continuous
      characters derived from geometric morphometric analyses to place fossil taxa with
      limited discrete characters into a phylogeny with their extant relatives was
      tested by simulating the inclusion of fossil taxa. This was done by removing the 
      molecular partition of individual extant species to produce a "cladistic
      pseudofossil" with only the geometric morphometric derived characters coded. The 
      phylogenetic position of each cladistic pseudofossil taxon was then compared with
      its placement in the total evidence tree and a symmetric resampling tree to
      evaluate the degree to which morphometric characters alone can correctly place
      simulated fossil species. In 33-45% of the test cases (depending upon the
      approach used for measuring success), it was possible to place the pseudofossil
      taxon into the correct regions of the phylogeny using only the morphometric
      characters. This suggests that the incorporation of extinct Conus taxa into
      phylogenetic hypotheses will be possible, permitting a wide range of
      macroevolutionary questions to be addressed within this genus. This methodology
      also has potential to contribute to phylogenetic reconstructions for other major 
      components of the fossil record that lack numerous discrete characters.
FAU - Smith, Ursula E
AU  - Smith UE
AD  - Department of Earth and Atmospheric Sciences, Cornell University, Ithaca, NY
      14853, USA. ues4@cornell.edu
FAU - Hendricks, Jonathan R
AU  - Hendricks JR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130116
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Calmodulin)
RN  - 0 (RNA, Ribosomal, 16S)
SB  - IM
MH  - Animal Shells/*anatomy &amp; histology
MH  - Animals
MH  - Biological Evolution
MH  - Calmodulin/genetics
MH  - Conus Snail/*anatomy &amp; histology/*genetics
MH  - Evolution, Molecular
MH  - Fossils
MH  - *Phylogeny
MH  - RNA, Ribosomal, 16S/genetics
MH  - Sequence Analysis, DNA
EDAT- 2013/01/18 06:00
MHDA- 2013/10/01 06:00
CRDT- 2013/01/18 06:00
PHST- 2013/01/18 06:00 [entrez]
PHST- 2013/01/18 06:00 [pubmed]
PHST- 2013/10/01 06:00 [medline]
AID - syt002 [pii]
AID - 10.1093/sysbio/syt002 [doi]
PST - ppublish
SO  - Syst Biol. 2013 May 1;62(3):366-85. doi: 10.1093/sysbio/syt002. Epub 2013 Jan 16.

PMID- 23302956
OWN - NLM
STAT- MEDLINE
DCOM- 20130930
LR  - 20130412
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 3
DP  - 2013 May 1
TI  - Continental diversification of an African catfish radiation (Mochokidae:
      Synodontis).
PG  - 351-65
LID - 10.1093/sysbio/syt001 [doi]
AB  - Despite African rivers containing high species diversity, continental-scale
      studies investigating the mechanisms generating biological diversity of African
      riverine faunas are limited compared with lacustrine systems. To investigate the 
      build-up of diversity in a tropical aquatic continental radiation, we test
      different models of lineage diversification and reconstruct the biogeographic
      history in a species-rich siluriform genus, Synodontis (~130 species), with a
      broad distribution across all major tropical African drainage basins. The
      resulting robust species-level phylogeny (~60% complete, based on a multigene
      data set) exhibits a near constant rate of lineage accumulation throughout the
      mid-Cenozoic to recent, irrespective of missing species and despite the changing 
      environmental conditions that were prevalent during this time period. This
      pattern contrasts with the findings for species-level diversification of large
      clades that commonly show an early burst of cladogenesis followed by declining
      rates through time. The identification of distinct biogeographic clades
      demonstrates a correlation between river hydrology and cladogenesis, although
      there is evidence of recent repeat dispersal into the southern range of the focal
      group. We conclude that diverse freshwater fish radiations with tropical
      continental distributions represent important organisms to test hypotheses of
      diversification and investigate the effects of palaeo-landscapes and climates on 
      present day biodiversity.
FAU - Day, Julia J
AU  - Day JJ
AD  - Department of Genetics, Evolution &amp; Environment, University College London, Gower
      Street, London WC1E 6BT, UK. j.day@ucl.ac.uk
FAU - Peart, Claire R
AU  - Peart CR
FAU - Brown, Katherine J
AU  - Brown KJ
FAU - Friel, John P
AU  - Friel JP
FAU - Bills, Roger
AU  - Bills R
FAU - Moritz, Timo
AU  - Moritz T
LA  - eng
SI  - GENBANK/HF565585
SI  - GENBANK/HF565586
SI  - GENBANK/HF565587
SI  - GENBANK/HF565588
SI  - GENBANK/HF565589
SI  - GENBANK/HF565590
SI  - GENBANK/HF565591
SI  - GENBANK/HF565592
SI  - GENBANK/HF565593
SI  - GENBANK/HF565594
SI  - GENBANK/HF565595
SI  - GENBANK/HF565596
SI  - GENBANK/HF565597
SI  - GENBANK/HF565598
SI  - GENBANK/HF565599
SI  - GENBANK/HF565600
SI  - GENBANK/HF565601
SI  - GENBANK/HF565602
SI  - GENBANK/HF565603
SI  - GENBANK/HF565604
SI  - GENBANK/HF565605
SI  - GENBANK/HF565606
SI  - GENBANK/HF565607
SI  - GENBANK/HF565608
SI  - GENBANK/HF565609
SI  - GENBANK/HF565610
SI  - GENBANK/HF565611
SI  - GENBANK/HF565612
SI  - GENBANK/HF565613
SI  - GENBANK/HF565614
SI  - GENBANK/HF565615
SI  - GENBANK/HF565616
SI  - GENBANK/HF565617
SI  - GENBANK/HF565618
SI  - GENBANK/HF565619
SI  - GENBANK/HF565620
SI  - GENBANK/HF565621
SI  - GENBANK/HF565622
SI  - GENBANK/HF565623
SI  - GENBANK/HF565624
SI  - GENBANK/HF565625
SI  - GENBANK/HF565626
SI  - GENBANK/HF565627
SI  - GENBANK/HF565628
SI  - GENBANK/HF565629
SI  - GENBANK/HF565630
SI  - GENBANK/HF565631
SI  - GENBANK/HF565632
SI  - GENBANK/HF565633
SI  - GENBANK/HF565634
SI  - GENBANK/HF565635
SI  - GENBANK/HF565636
SI  - GENBANK/HF565637
SI  - GENBANK/HF565638
SI  - GENBANK/HF565639
SI  - GENBANK/HF565640
SI  - GENBANK/HF565641
SI  - GENBANK/HF565642
SI  - GENBANK/HF565643
SI  - GENBANK/HF565644
SI  - GENBANK/HF565645
SI  - GENBANK/HF565646
SI  - GENBANK/HF565647
SI  - GENBANK/HF565648
SI  - GENBANK/HF565649
SI  - GENBANK/HF565650
SI  - GENBANK/HF565651
SI  - GENBANK/HF565652
SI  - GENBANK/HF565653
SI  - GENBANK/HF565654
SI  - GENBANK/HF565655
SI  - GENBANK/HF565656
SI  - GENBANK/HF565657
SI  - GENBANK/HF565658
SI  - GENBANK/HF565659
SI  - GENBANK/HF565660
SI  - GENBANK/HF565661
SI  - GENBANK/HF565662
SI  - GENBANK/HF565663
SI  - GENBANK/HF565664
SI  - GENBANK/HF565665
SI  - GENBANK/HF565666
SI  - GENBANK/HF565667
SI  - GENBANK/HF565668
SI  - GENBANK/HF565669
SI  - GENBANK/HF565670
SI  - GENBANK/HF565671
SI  - GENBANK/HF565672
SI  - GENBANK/HF565673
SI  - GENBANK/HF565674
SI  - GENBANK/HF565675
SI  - GENBANK/HF565676
SI  - GENBANK/HF565677
SI  - GENBANK/HF565678
SI  - GENBANK/HF565679
SI  - GENBANK/HF565680
SI  - GENBANK/HF565681
SI  - GENBANK/HF565682
SI  - GENBANK/HF565683
SI  - GENBANK/HF565684
SI  - GENBANK/HF565685
SI  - GENBANK/HF565686
SI  - GENBANK/HF565687
SI  - GENBANK/HF565688
SI  - GENBANK/HF565689
SI  - GENBANK/HF565690
SI  - GENBANK/HF565691
SI  - GENBANK/HF565692
SI  - GENBANK/HF565693
SI  - GENBANK/HF565694
SI  - GENBANK/HF565695
SI  - GENBANK/HF565696
SI  - GENBANK/HF565697
SI  - GENBANK/HF565698
SI  - GENBANK/HF565699
SI  - GENBANK/HF565700
SI  - GENBANK/HF565701
SI  - GENBANK/HF565702
SI  - GENBANK/HF565703
SI  - GENBANK/HF565704
SI  - GENBANK/HF565705
SI  - GENBANK/HF565706
SI  - GENBANK/HF565707
SI  - GENBANK/HF565708
SI  - GENBANK/HF565709
SI  - GENBANK/HF565710
SI  - GENBANK/HF565711
SI  - GENBANK/HF565712
SI  - GENBANK/HF565713
SI  - GENBANK/HF565714
SI  - GENBANK/HF565715
SI  - GENBANK/HF565716
SI  - GENBANK/HF565717
SI  - GENBANK/HF565718
SI  - GENBANK/HF565719
SI  - GENBANK/HF565720
SI  - GENBANK/HF565721
SI  - GENBANK/HF565722
SI  - GENBANK/HF565723
SI  - GENBANK/HF565724
SI  - GENBANK/HF565725
SI  - GENBANK/HF565726
SI  - GENBANK/HF565727
SI  - GENBANK/HF565728
SI  - GENBANK/HF565729
SI  - GENBANK/HF565730
SI  - GENBANK/HF565731
SI  - GENBANK/HF565732
SI  - GENBANK/HF565733
SI  - GENBANK/HF565734
SI  - GENBANK/HF565735
SI  - GENBANK/HF565736
SI  - GENBANK/HF565737
SI  - GENBANK/HF565738
SI  - GENBANK/HF565739
SI  - GENBANK/HF565740
SI  - GENBANK/HF565741
SI  - GENBANK/HF565742
SI  - GENBANK/HF565743
SI  - GENBANK/HF565744
SI  - GENBANK/HF565745
SI  - GENBANK/HF565746
SI  - GENBANK/HF565747
SI  - GENBANK/HF565748
SI  - GENBANK/HF565749
SI  - GENBANK/HF565750
SI  - GENBANK/HF565751
SI  - GENBANK/HF565752
SI  - GENBANK/HF565753
SI  - GENBANK/HF565754
SI  - GENBANK/HF565755
SI  - GENBANK/HF565756
SI  - GENBANK/HF565757
SI  - GENBANK/HF565758
SI  - GENBANK/HF565759
SI  - GENBANK/HF565760
SI  - GENBANK/HF565761
SI  - GENBANK/HF565762
SI  - GENBANK/HF565763
SI  - GENBANK/HF565764
SI  - GENBANK/HF565765
SI  - GENBANK/HF565766
SI  - GENBANK/HF565767
SI  - GENBANK/HF565768
SI  - GENBANK/HF565769
SI  - GENBANK/HF565770
SI  - GENBANK/HF565771
SI  - GENBANK/HF565772
SI  - GENBANK/HF565773
SI  - GENBANK/HF565774
SI  - GENBANK/HF565775
SI  - GENBANK/HF565776
SI  - GENBANK/HF565777
SI  - GENBANK/HF565778
SI  - GENBANK/HF565779
SI  - GENBANK/HF565780
SI  - GENBANK/HF565781
SI  - GENBANK/HF565782
SI  - GENBANK/HF565783
SI  - GENBANK/HF565784
SI  - GENBANK/HF565785
SI  - GENBANK/HF565786
SI  - GENBANK/HF565787
SI  - GENBANK/HF565788
SI  - GENBANK/HF565789
SI  - GENBANK/HF565790
SI  - GENBANK/HF565791
SI  - GENBANK/HF565792
SI  - GENBANK/HF565793
SI  - GENBANK/HF565794
SI  - GENBANK/HF565795
SI  - GENBANK/HF565796
SI  - GENBANK/HF565797
SI  - GENBANK/HF565798
SI  - GENBANK/HF565799
SI  - GENBANK/HF565800
SI  - GENBANK/HF565801
SI  - GENBANK/HF565802
SI  - GENBANK/HF565803
SI  - GENBANK/HF565804
SI  - GENBANK/HF565805
SI  - GENBANK/HF565806
SI  - GENBANK/HF565807
SI  - GENBANK/HF565808
SI  - GENBANK/HF565809
SI  - GENBANK/HF565810
SI  - GENBANK/HF565811
SI  - GENBANK/HF565812
SI  - GENBANK/HF565813
SI  - GENBANK/HF565814
SI  - GENBANK/HF565815
SI  - GENBANK/HF565816
SI  - GENBANK/HF565817
SI  - GENBANK/HF565818
SI  - GENBANK/HF565819
SI  - GENBANK/HF565820
SI  - GENBANK/HF565821
SI  - GENBANK/HF565822
SI  - GENBANK/HF565823
SI  - GENBANK/HF565824
SI  - GENBANK/HF565825
SI  - GENBANK/HF565826
SI  - GENBANK/HF565827
SI  - GENBANK/HF565828
SI  - GENBANK/HF565829
SI  - GENBANK/HF565830
SI  - GENBANK/HF565831
SI  - GENBANK/HF565832
SI  - GENBANK/HF565833
SI  - GENBANK/HF565834
SI  - GENBANK/HF565835
SI  - GENBANK/HF565836
SI  - GENBANK/HF565837
SI  - GENBANK/HF565838
SI  - GENBANK/HF565839
SI  - GENBANK/HF565840
SI  - GENBANK/HF565841
SI  - GENBANK/HF565842
SI  - GENBANK/HF565843
SI  - GENBANK/HF565844
SI  - GENBANK/HF565845
SI  - GENBANK/HF565846
SI  - GENBANK/HF565847
SI  - GENBANK/HF565848
SI  - GENBANK/HF565849
SI  - GENBANK/HF565850
SI  - GENBANK/HF565851
SI  - GENBANK/HF565852
SI  - GENBANK/HF565853
SI  - GENBANK/HF565854
SI  - GENBANK/HF565855
SI  - GENBANK/HF565856
SI  - GENBANK/HF565857
SI  - GENBANK/HF565858
SI  - GENBANK/HF565859
SI  - GENBANK/HF565860
SI  - GENBANK/HF565861
SI  - GENBANK/HF565862
SI  - GENBANK/HF565863
SI  - GENBANK/HF565864
SI  - GENBANK/HF565865
SI  - GENBANK/HF565866
SI  - GENBANK/HF565867
SI  - GENBANK/HF565868
SI  - GENBANK/HF565869
SI  - GENBANK/HF565870
SI  - GENBANK/HF565871
SI  - GENBANK/HF565872
SI  - GENBANK/HF565873
SI  - GENBANK/HF565874
SI  - GENBANK/HF565875
SI  - GENBANK/HF565876
SI  - GENBANK/HF565877
SI  - GENBANK/HF565878
SI  - GENBANK/HF565879
SI  - GENBANK/HF565880
SI  - GENBANK/HF565881
SI  - GENBANK/HF565882
SI  - GENBANK/HF565883
SI  - GENBANK/HF565884
SI  - GENBANK/HF565885
SI  - GENBANK/HF565886
SI  - GENBANK/HF565887
SI  - GENBANK/HF565888
SI  - GENBANK/HF565889
SI  - GENBANK/HF565890
SI  - GENBANK/HF565891
SI  - GENBANK/HF565892
SI  - GENBANK/HF565893
SI  - GENBANK/HF565894
SI  - GENBANK/HF565895
SI  - GENBANK/HF565896
SI  - GENBANK/HF565897
SI  - GENBANK/HF565898
SI  - GENBANK/HF565899
SI  - GENBANK/HF565900
SI  - GENBANK/HF565901
SI  - GENBANK/HF565902
SI  - GENBANK/HF565903
SI  - GENBANK/HF565904
SI  - GENBANK/HF565905
SI  - GENBANK/HF565906
SI  - GENBANK/HF565907
SI  - GENBANK/HF565908
SI  - GENBANK/HF565909
SI  - GENBANK/HF565910
SI  - GENBANK/HF565911
SI  - GENBANK/HF565912
SI  - GENBANK/HF565913
SI  - GENBANK/HF565914
SI  - GENBANK/HF565915
SI  - GENBANK/HF565916
SI  - GENBANK/HF565917
SI  - GENBANK/HF565918
SI  - GENBANK/HF565919
SI  - GENBANK/HF565920
SI  - GENBANK/HF565921
SI  - GENBANK/HF565922
SI  - GENBANK/HF565923
SI  - GENBANK/HF565924
SI  - GENBANK/HF565925
SI  - GENBANK/HF565926
SI  - GENBANK/HF565927
SI  - GENBANK/HF565928
SI  - GENBANK/HF565929
SI  - GENBANK/HF565930
SI  - GENBANK/HF565931
SI  - GENBANK/HF565932
SI  - GENBANK/HF565933
SI  - GENBANK/HF565934
SI  - GENBANK/HF565935
SI  - GENBANK/HF565936
SI  - GENBANK/HF565937
SI  - GENBANK/HF565938
SI  - GENBANK/HF565939
SI  - GENBANK/HF565940
SI  - GENBANK/HF565941
SI  - GENBANK/HF565942
SI  - GENBANK/HF565943
SI  - GENBANK/HF565944
SI  - GENBANK/HF565945
SI  - GENBANK/HF565946
SI  - GENBANK/HF565947
SI  - GENBANK/HF565948
SI  - GENBANK/HF565949
SI  - GENBANK/HF565950
SI  - GENBANK/HF565951
SI  - GENBANK/HF565952
SI  - GENBANK/HF565953
SI  - GENBANK/HF565954
SI  - GENBANK/HF565955
SI  - GENBANK/HF565956
SI  - GENBANK/HF565957
SI  - GENBANK/HF565958
SI  - GENBANK/HF565959
SI  - GENBANK/HF565960
SI  - GENBANK/HF565961
SI  - GENBANK/HF565962
SI  - GENBANK/HF565963
SI  - GENBANK/HF565964
SI  - GENBANK/HF565965
SI  - GENBANK/HF565966
SI  - GENBANK/HF565967
SI  - GENBANK/HF565968
SI  - GENBANK/HF565969
SI  - GENBANK/HF565970
SI  - GENBANK/HF565971
SI  - GENBANK/HF565972
SI  - GENBANK/HF565973
SI  - GENBANK/HF565974
SI  - GENBANK/HF565975
SI  - GENBANK/HF565976
SI  - GENBANK/HF565977
SI  - GENBANK/HF565978
SI  - GENBANK/HF565979
SI  - GENBANK/HF565980
SI  - GENBANK/HF565981
SI  - GENBANK/HF565982
SI  - GENBANK/HF565983
SI  - GENBANK/HF565984
SI  - GENBANK/HF565985
SI  - GENBANK/HF565986
SI  - GENBANK/HF565987
SI  - GENBANK/HF565988
SI  - GENBANK/HF565989
SI  - GENBANK/HF565990
SI  - GENBANK/HF565991
SI  - GENBANK/HF565992
SI  - GENBANK/HF565993
SI  - GENBANK/HF565994
SI  - GENBANK/HF565995
SI  - GENBANK/HF565996
SI  - GENBANK/HF565997
SI  - GENBANK/HF565998
SI  - GENBANK/HF565999
SI  - GENBANK/HF566000
SI  - GENBANK/HF566001
SI  - GENBANK/HF566002
SI  - GENBANK/HF566003
SI  - GENBANK/HF566004
SI  - GENBANK/HF566005
SI  - GENBANK/HF566006
SI  - GENBANK/HF566007
SI  - GENBANK/HF566008
SI  - GENBANK/HF566009
SI  - GENBANK/HF566010
SI  - GENBANK/HF566011
SI  - GENBANK/HF566012
SI  - GENBANK/HF566013
SI  - GENBANK/HF566014
SI  - GENBANK/HF566015
SI  - GENBANK/HF566016
SI  - GENBANK/HF566017
SI  - GENBANK/HF566018
SI  - GENBANK/HF566019
SI  - GENBANK/HF566020
SI  - GENBANK/HF566021
SI  - GENBANK/HF566022
SI  - GENBANK/HF566023
SI  - GENBANK/HF566024
SI  - GENBANK/HF566025
SI  - GENBANK/HF566026
SI  - GENBANK/HF566027
SI  - GENBANK/HF566028
SI  - GENBANK/HF566029
SI  - GENBANK/HF566030
SI  - GENBANK/HF566031
SI  - GENBANK/HF566032
SI  - GENBANK/HF566033
SI  - GENBANK/HF566034
SI  - GENBANK/HF566035
SI  - GENBANK/HF566036
SI  - GENBANK/HF566037
SI  - GENBANK/HF566038
SI  - GENBANK/HF566039
SI  - GENBANK/HF566040
SI  - GENBANK/HF566041
SI  - GENBANK/HF566042
SI  - GENBANK/HF566043
SI  - GENBANK/HF566044
SI  - GENBANK/HF566045
SI  - GENBANK/HF566046
SI  - GENBANK/HF566047
SI  - GENBANK/HF566048
SI  - GENBANK/HF566049
SI  - GENBANK/HF566050
SI  - GENBANK/HF566051
SI  - GENBANK/HF566052
SI  - GENBANK/HF566053
SI  - GENBANK/HF566054
SI  - GENBANK/HF566055
SI  - GENBANK/HF566056
SI  - GENBANK/HF566057
SI  - GENBANK/HF566058
SI  - GENBANK/HF566059
SI  - GENBANK/HF566060
SI  - GENBANK/HF566061
SI  - GENBANK/HF566062
SI  - GENBANK/HF566063
SI  - GENBANK/HF566064
SI  - GENBANK/HF566065
SI  - GENBANK/HF566066
SI  - GENBANK/HF566067
SI  - GENBANK/HF566068
SI  - GENBANK/HF566069
SI  - GENBANK/HF566070
SI  - GENBANK/HF566071
SI  - GENBANK/HF566072
SI  - GENBANK/HF566073
SI  - GENBANK/HF566074
SI  - GENBANK/HF566075
SI  - GENBANK/HF566076
SI  - GENBANK/HF566077
SI  - GENBANK/HF566078
SI  - GENBANK/HF566079
SI  - GENBANK/HF566080
SI  - GENBANK/HF566081
SI  - GENBANK/HF566082
SI  - GENBANK/HF566083
SI  - GENBANK/HF566084
SI  - GENBANK/HF566085
SI  - GENBANK/HF566086
SI  - GENBANK/HF566087
SI  - GENBANK/HF566088
SI  - GENBANK/HF566089
SI  - GENBANK/HF566090
SI  - GENBANK/HF566091
SI  - GENBANK/HF566092
SI  - GENBANK/HF566093
SI  - GENBANK/HF566094
SI  - GENBANK/HF566095
SI  - GENBANK/HF566096
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20130109
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Africa
MH  - *Animal Distribution
MH  - Animals
MH  - Bayes Theorem
MH  - Biodiversity
MH  - Biological Evolution
MH  - Catfishes/*classification/*genetics
MH  - Cell Nucleus/genetics
MH  - DNA/analysis
MH  - DNA, Mitochondrial/analysis
MH  - *Evolution, Molecular
MH  - *Genetic Speciation
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Polymerase Chain Reaction
MH  - Rivers
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
EDAT- 2013/01/11 06:00
MHDA- 2013/10/01 06:00
CRDT- 2013/01/11 06:00
PHST- 2013/01/11 06:00 [entrez]
PHST- 2013/01/11 06:00 [pubmed]
PHST- 2013/10/01 06:00 [medline]
AID - syt001 [pii]
AID - 10.1093/sysbio/syt001 [doi]
PST - ppublish
SO  - Syst Biol. 2013 May 1;62(3):351-65. doi: 10.1093/sysbio/syt001. Epub 2013 Jan 9.

PMID- 23291311
OWN - NLM
STAT- MEDLINE
DCOM- 20130930
LR  - 20130412
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 3
DP  - 2013 May 1
TI  - The tree of life: metaphor, model, and heuristic device.
PG  - 479-89
LID - 10.1093/sysbio/sys115 [doi]
FAU - Mindell, David P
AU  - Mindell DP
AD  - Department of Biochemistry &amp; Biophysics, University of California, San Francisco,
      CA 94158, USA. dpmindell@gmail.com
LA  - eng
PT  - Journal Article
DEP - 20130103
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Learning
MH  - *Metaphor
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2013/01/08 06:00
MHDA- 2013/10/01 06:00
CRDT- 2013/01/08 06:00
PHST- 2013/01/08 06:00 [entrez]
PHST- 2013/01/08 06:00 [pubmed]
PHST- 2013/10/01 06:00 [medline]
AID - sys115 [pii]
AID - 10.1093/sysbio/sys115 [doi]
PST - ppublish
SO  - Syst Biol. 2013 May 1;62(3):479-89. doi: 10.1093/sysbio/sys115. Epub 2013 Jan 3.

PMID- 23229025
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20130208
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 2
DP  - 2013 Mar
TI  - Going to extremes: contrasting rates of diversification in a recent radiation of 
      new world passerine birds.
PG  - 298-320
LID - 10.1093/sysbio/sys094 [doi]
AB  - Recent analyses suggest that a few major shifts in diversification rate may be
      enough to explain most of the disparity in diversity among vertebrate lineages.
      At least one significant increase in diversification rate appears to have
      occurred within the birds; however, several nested lineages within birds have
      been identified as hyperdiverse by different studies. A clade containing the
      finches and relatives (within the avian order Passeriformes), including a large
      radiation endemic to the New World that comprises ~8% of all bird species, may be
      the true driver of this rate increase. Understanding the patterns and processes
      of diversification of this diverse lineage may go a long way toward explaining
      the apparently rapid diversification rates of both passerines and of birds as a
      whole. We present the first multilocus phylogenetic analyses of this endemic New 
      World radiation of finch relatives that include sampling of all recognized
      genera, a relaxed molecular clock analysis of its divergence history, and an
      analysis of its broad-scale diversification patterns. These analyses recovered 5 
      major lineages traditionally recognized as avian families, but identified an
      additional 10 relatively ancient lineages worthy of recognition at the family
      level. Time-calibrated diversification analyses suggested that at least 3 of the 
      15 family-level lineages were significantly species poor given the entire group's
      background diversification rate, whereas at least one-the tanagers of family
      Thraupidae-appeared significantly more diverse. Lack of an age-diversity
      relationship within this clade suggests that, due to rapid initial speciation, it
      may have experienced density-dependent ecological limits on its overall
      diversity.
FAU - Barker, F Keith
AU  - Barker FK
AD  - Department of Ecology, Evolution and Behavior, University of Minnesota, 100
      Ecology, 1987 Upper Buford Circle, St Paul, MN 55108, USA. barke042@umn.edu
FAU - Burns, Kevin J
AU  - Burns KJ
FAU - Klicka, John
AU  - Klicka J
FAU - Lanyon, Scott M
AU  - Lanyon SM
FAU - Lovette, Irby J
AU  - Lovette IJ
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20121209
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9035-37-4 (Cytochromes b)
RN  - EC 1.6.99.3 (NADH Dehydrogenase)
SB  - IM
MH  - Americas
MH  - Animals
MH  - *Biodiversity
MH  - Cytochromes b/genetics
MH  - Genetic Speciation
MH  - NADH Dehydrogenase/genetics
MH  - Passeriformes/*classification/genetics
MH  - *Phylogeny
MH  - Time
EDAT- 2012/12/12 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/12/12 06:00
PHST- 2012/12/12 06:00 [entrez]
PHST- 2012/12/12 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys094 [pii]
AID - 10.1093/sysbio/sys094 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Mar;62(2):298-320. doi: 10.1093/sysbio/sys094. Epub 2012 Dec 9.

PMID- 23220768
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 2
DP  - 2013 Mar
TI  - Relaxed phylogenetics and the palaeoptera problem: resolving deep ancestral
      splits in the insect phylogeny.
PG  - 285-97
LID - 10.1093/sysbio/sys093 [doi]
AB  - The order in which the 3 groups of winged insects (the Pterygota) diverged from
      their common ancestor has important implications for understanding the origin of 
      insect flight. But despite this importance, the split between the Odonata
      (dragonflies and damselflies), Ephemeroptera (mayflies), and Neoptera (the other 
      winged orders) remains very much unresolved. Indeed, previous studies have
      obtained strong apparent support for each of the 3 possible branching patterns.
      Here, we present a systematic reinvestigation of the basal pterygote split. Our
      results suggest that outgroup choice and limited taxon sampling have been major
      sources of systematic error, even for data sets with a large number of characters
      (e.g., in phylogenomic data sets). In particular, a data set of 113 taxa provides
      consistent support for the Palaeoptera hypothesis (the grouping of Odonata with
      Ephemeroptera), whereas results from data sets with fewer taxa give inconsistent 
      results and are highly sensitive to minor changes in data and methods. We also
      focus on recent methods that exploit temporal information using fossil
      calibrations, combined with additional assumptions about the evolutionary
      process, and so reduce the influence of outgroup choice. These methods are shown 
      to provide more consistent results, for example, supporting Palaeoptera, even for
      data sets that previously supported other hypotheses. Together, these results
      have implications for understanding insect origins and for resolving other
      problematic splits in the tree of life.
FAU - Thomas, Jessica A
AU  - Thomas JA
AD  - Department of Biology, University of York, York YO10 5DD, UK.
      jessica.thomas@york.ac.uk
FAU - Trueman, John W H
AU  - Trueman JW
FAU - Rambaut, Andrew
AU  - Rambaut A
FAU - Welch, John J
AU  - Welch JJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20121205
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Genes, Insect/genetics
MH  - Insecta/*classification/genetics
MH  - *Phylogeny
EDAT- 2012/12/12 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/12/11 06:00
PHST- 2012/12/11 06:00 [entrez]
PHST- 2012/12/12 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys093 [pii]
AID - 10.1093/sysbio/sys093 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Mar;62(2):285-97. doi: 10.1093/sysbio/sys093. Epub 2012 Dec 5.

PMID- 23192838
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20130208
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 2
DP  - 2013 Mar
TI  - Treating fossils as terminal taxa in divergence time estimation reveals ancient
      vicariance patterns in the palpimanoid spiders.
PG  - 264-84
LID - 10.1093/sysbio/sys092 [doi]
AB  - Incorporation of fossils into biogeographic studies can have a profound effect on
      the conclusions that result, particularly when fossil ranges are nonoverlapping
      with extant ranges. This is the case in archaeid spiders, where there are known
      fossils from the Northern Hemisphere, yet all living members are restricted to
      the Southern Hemisphere. To better understand the biogeographic patterns of
      archaeid spiders and their palpimanoid relatives, we estimate a dated phylogeny
      using a relaxed clock on a combined molecular and morphological data set. Dating 
      information is compared with treating the archaeid fossil taxa as both node
      calibrations and as noncontemporaneous terminal tips, both with and without
      additional calibration points. Estimation of ancestral biogeographic ranges is
      then performed, using likelihood and Bayesian methods to take into account
      uncertainty in phylogeny and in dating. We find that treating the fossils as
      terminal tips within a Bayesian framework, as opposed to dating the phylogeny
      based only on molecular data with the dates coming from node calibrations,
      removes the subjectivity involved in assigning priors, which has not been
      possible with previous methods. Our analyses suggest that the diversification of 
      the northern and southern archaeid lineages was congruent with the breakup of
      Pangaea into Laurasia and Gondwanaland. This analysis provides a rare example,
      and perhaps the most strongly supported, where a dated phylogeny confirms a
      biogeographical hypothesis based on vicariance due to the breakup of the ancient 
      continental plates.
FAU - Wood, Hannah Marie
AU  - Wood HM
AD  - Department of Environmental Science, Policy and Management, University of
      California, 137 Mulford Hall, Berkeley, CA 94720, USA. woodhannahmarie@gmail.com
FAU - Matzke, Nicholas J
AU  - Matzke NJ
FAU - Gillespie, Rosemary G
AU  - Gillespie RG
FAU - Griswold, Charles E
AU  - Griswold CE
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20121128
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Fossils
MH  - *Phylogeny
MH  - Phylogeography
MH  - Spiders/*classification/genetics
MH  - Time
EDAT- 2012/11/30 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/11/30 06:00
PHST- 2012/11/30 06:00 [entrez]
PHST- 2012/11/30 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys092 [pii]
AID - 10.1093/sysbio/sys092 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Mar;62(2):264-84. doi: 10.1093/sysbio/sys092. Epub 2012 Nov 28.

PMID- 23179602
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20130208
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 2
DP  - 2013 Mar
TI  - Amalgamating source trees with different taxonomic levels.
PG  - 231-49
LID - 10.1093/sysbio/sys090 [doi]
AB  - Supertree methods combine a collection of source trees into a single parent tree 
      or supertree. For almost all such methods, the terminal taxa across the source
      trees have to be non-nested for the output supertree to make sense. Motivated by 
      Page, the first supertree method for combining rooted source trees where the taxa
      can be hierarchically nested is called AncestralBuild. In addition to taxa
      labeling the leaves, this method allows the rooted source trees to have taxa
      labeling some of the interior nodes at a higher taxonomic level than their
      descendants (e.g., genera vs. species). However, the utility of AncestralBuild is
      somewhat restricted as it is mostly intended to decide if a collection of rooted 
      source trees is compatible. If the initial collection is not compatible, then no 
      tree is returned. To overcome this restriction, we introduce here the
      MultiLevelSupertree (MLS) supertree method whose input is the same as that for
      AncestralBuild, but which accommodates incompatibilities among rooted source
      trees using a MinCut-like procedure. We show that MLS has several desirable
      properties including the preservation of common subtrees among the source trees, 
      the preservation of ancestral relationships whenever they are compatible, as well
      as running in polynomial time. Furthermore, application to a small test data set 
      (the mammalian carnivore family Phocidae) indicates that the method correctly
      places nested taxa at different taxonomic levels (reflecting vertical signal),
      even in cases where the input trees harbor a significant level of conflict
      between their clades (i.e., in their horizontal signal).
FAU - Berry, Vincent
AU  - Berry V
AD  - Methodes et Algorithmes pour la Bioinformatique MAB team, Universite Montpellier 
      2, L.I.R.M.M. - C.N.R.S., 161 rue Ada, 34095 Montpellier Cedex 5, France.
FAU - Bininda-Emonds, Olaf R P
AU  - Bininda-Emonds OR
FAU - Semple, Charles
AU  - Semple C
LA  - eng
PT  - Journal Article
DEP - 20121123
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Animals
MH  - Classification/*methods
MH  - Models, Biological
MH  - *Phylogeny
MH  - Seals, Earless/classification
EDAT- 2012/11/28 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/11/27 06:00
PHST- 2012/11/27 06:00 [entrez]
PHST- 2012/11/28 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys090 [pii]
AID - 10.1093/sysbio/sys090 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Mar;62(2):231-49. doi: 10.1093/sysbio/sys090. Epub 2012 Nov 23.

PMID- 23179601
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 2
DP  - 2013 Mar
TI  - The identification of concerted convergence in insect heads corroborates
      palaeoptera.
PG  - 250-63
LID - 10.1093/sysbio/sys091 [doi]
AB  - The relationships of the 3 major clades of winged insects-Ephemeroptera, Odonata,
      and Neoptera-are still unclear. Many morphologists favor a clade Metapterygota
      (Odonata +Neoptera), but Chiastomyaria (Ephemeroptera + Neoptera) or Palaeoptera 
      (Ephemeroptera +Odonata) has also been supported in some older and more recent
      studies. A possible explanation for the difficulties in resolving these
      relationships is concerted convergence-the convergent evolution of entire
      character complexes under the same or similar selective pressures. In this study,
      we analyze possible instances of this phenomenon in the context of head
      structures of Ephemeroptera, Odonata, and Neoptera. We apply a recently
      introduced formal approach to detect the occurrence of concerted convergence. We 
      found that characters of the tentorium and mandibles in particular, but also some
      other head structures, have apparently not evolved independently, and thus can
      cause artifacts in tree reconstruction. Our subsequent analyses, which exclude
      character sets that may be affected by concerted convergence, corroborate the
      Palaeoptera concept. We show that the analysis of homoplasy and its influence on 
      tree inference can be formally improved with important consequences for the
      identification of incompatibilities between data sets. Our results suggest that
      modified weighting (or exclusion of characters) in cases of formally identified
      correlated cliques of characters may improve morphology-based tree
      reconstruction.
FAU - Blanke, Alexander
AU  - Blanke A
AD  - Zoologisches Forschungsmuseum Alexander Koenig, Zentrum fur molekulare
      Biodiversitat, Adenauerallee 160, 53113 Bonn, Germany. blanke@uni-bonn.de
FAU - Greve, Carola
AU  - Greve C
FAU - Wipfler, Benjamin
AU  - Wipfler B
FAU - Beutel, Rolf G
AU  - Beutel RG
FAU - Holland, Barbara R
AU  - Holland BR
FAU - Misof, Bernhard
AU  - Misof B
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20121123
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 18S)
RN  - 0 (RNA, Ribosomal, 28S)
SB  - IM
EIN - Syst Biol. 2013 Sep;62(5):787
MH  - Animals
MH  - Head/anatomy &amp; histology
MH  - Insecta/*anatomy &amp; histology/*classification/genetics
MH  - *Phylogeny
MH  - RNA, Ribosomal, 18S/genetics
MH  - RNA, Ribosomal, 28S/genetics
EDAT- 2012/11/28 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/11/27 06:00
PHST- 2012/11/27 06:00 [entrez]
PHST- 2012/11/28 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys091 [pii]
AID - 10.1093/sysbio/sys091 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Mar;62(2):250-63. doi: 10.1093/sysbio/sys091. Epub 2012 Nov 23.

PMID- 23103590
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20130208
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 2
DP  - 2013 Mar
TI  - Using supermatrices for phylogenetic inquiry: an example using the sedges.
PG  - 205-19
LID - 10.1093/sysbio/sys088 [doi]
AB  - In this article, we use supermatrix data-mining methods to reconstruct a large,
      highly inclusive phylogeny of Cyperaceae from nucleotide data available on
      GenBank. We explore the properties of these trees and their utility for
      phylogenetic inference, and show that even the highly incomplete alignments
      characteristic of supermatrix approaches may yield very good estimates of
      phylogeny. We present a novel pipeline for filtering sparse alignments to improve
      their phylogenetic utility by maximizing the partial decisiveness of the matrices
      themselves through a technique we call "phylogenetic scaffolding," and we present
      a new method of scoring tip instability (i.e. "rogue taxa") based on the I
      statistic implemented in the software Mesquite. The modified statistic, which we 
      call I(S), is somewhat more straightforward to interpret than similar statistics,
      and our implementation of it may be applied to large sets of large trees. The
      largest sedge trees presented here contain more than 1500 tips (about one quarter
      of all sedge species) and are based on multigene alignments with more than 20 000
      sites and more than 90% missing data. These trees match well with previously
      supported phylogenetic hypotheses, but have lower overall support values and less
      resolution than more heavily filtered trees. Our best-resolved trees are
      characterized by stronger support values than any previously published sedge
      phylogenies, and show some relationships that are incongruous with previous
      studies. Overall, we show that supermatrix methods offer powerful means of
      pursuing phylogenetic study and these tools have high potential value for many
      systematic biologists.
FAU - Hinchliff, Cody E
AU  - Hinchliff CE
AD  - School of Biological Sciences, Washington State University, Pullman, WA
      99164-4236, USA. cody.hinchliff@gmail.com
FAU - Roalson, Eric H
AU  - Roalson EH
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20121026
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - Cyperaceae/*classification/*genetics
MH  - Data Mining
MH  - *Phylogeny
EDAT- 2012/10/30 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/10/30 06:00
PHST- 2012/10/30 06:00 [entrez]
PHST- 2012/10/30 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys088 [pii]
AID - 10.1093/sysbio/sys088 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Mar;62(2):205-19. doi: 10.1093/sysbio/sys088. Epub 2012 Oct 26.

PMID- 23103589
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20130208
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 2
DP  - 2013 Mar
TI  - Estimating speciation and extinction rates for phylogenies of higher taxa.
PG  - 220-30
LID - 10.1093/sysbio/sys087 [doi]
AB  - Speciation and extinction rates can be estimated from molecular phylogenies.
      Recently, a number of methods have been published showing that these rates can be
      estimated even if the phylogeny is incomplete, that is, if not all extant species
      are included. We show that the accuracy of such methods strongly depends on
      making the correct assumptions about how the sampling process was performed. We
      focus on phylogenies that are incomplete because some subclades (e.g., genera and
      families) are each represented as a single lineage. We show that previous methods
      implicitly assumed that such subclades are defined by randomly (or in an extreme 
      deterministic way) choosing the edges that define the subclades from the complete
      species phylogeny. We show that these methods produce biased results if higher
      taxa are defined in a different manner. We introduce strict higher level
      phylogenies where subclades are defined so that the phylogeny is fully resolved
      from its origin to time x(cut), and fully unresolved thereafter, so that for all 
      subclades, stem age &gt; x(cut) &gt; crown age. We present estimates of speciation and 
      extinction rates from a phylogeny of birds in which this subclade definition was 
      applied. However, for most higher level phylogenies in the literature, it is
      unclear how higher taxa were defined, but often such phylogenies can be easily
      transformed into strict higher level phylogenies, as we illustrate by estimating 
      speciation and extinction rates from a near-complete but only partly resolved
      species-level phylogeny of mammals. The accuracy of our methods is verified using
      simulations.
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Department of Environmental Systems Science, ETH Zurich, 8092 Zurich,
      Switzerland.
FAU - Bokma, Folmer
AU  - Bokma F
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20121025
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Birds
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Extinction, Biological
MH  - *Genetic Speciation
MH  - Mammals
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2012/10/30 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/10/30 06:00
PHST- 2012/10/30 06:00 [entrez]
PHST- 2012/10/30 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys087 [pii]
AID - 10.1093/sysbio/sys087 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Mar;62(2):220-30. doi: 10.1093/sysbio/sys087. Epub 2012 Oct 25.

PMID- 23034385
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 2
DP  - 2013 Mar
TI  - Phylogenetic analysis using Levy processes: finding jumps in the evolution of
      continuous traits.
PG  - 193-204
LID - 10.1093/sysbio/sys086 [doi]
AB  - Gaussian processes, a class of stochastic processes including Brownian motion and
      the Ornstein-Uhlenbeck process, are widely used to model continuous trait
      evolution in statistical phylogenetics. Under such processes, observations at the
      tips of a phylogenetic tree have a multivariate Gaussian distribution, which may 
      lead to suboptimal model specification under certain evolutionary conditions, as 
      supposed in models of punctuated equilibrium or adaptive radiation. To consider
      non-normally distributed continuous trait evolution, we introduce a method to
      compute posterior probabilities when modeling continuous trait evolution as a
      Levy process. Through data simulation and model testing, we establish that
      single-rate Brownian motion (BM) and Levy processes with jumps generate distinct 
      patterns in comparative data. We then analyzed body mass and endocranial volume
      measurements for 126 primates. We rejected single-rate BM in favor of a Levy
      process with jumps for each trait, with the lineage leading to most recent common
      ancestor of great apes showing particularly strong evidence against single-rate
      BM.
FAU - Landis, Michael J
AU  - Landis MJ
AD  - Department of Integrative Biology, University of California, Berkeley, CA
      94720-3140, USA.
FAU - Schraiber, Joshua G
AU  - Schraiber JG
FAU - Liang, Mason
AU  - Liang M
LA  - eng
GR  - R01-GM40282/GM/NIGMS NIH HHS/United States
GR  - R01 GM086887/GM/NIGMS NIH HHS/United States
GR  - R01 GM069801/GM/NIGMS NIH HHS/United States
GR  - R01-GM069801/GM/NIGMS NIH HHS/United States
GR  - T32-HG00047/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
DEP - 20121003
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Body Mass Index
MH  - Computer Simulation
MH  - *Models, Biological
MH  - *Phenotype
MH  - *Phylogeny
MH  - Primates/classification/physiology
PMC - PMC3566600
EDAT- 2012/10/05 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/10/05 06:00
PHST- 2012/10/05 06:00 [entrez]
PHST- 2012/10/05 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys086 [pii]
AID - 10.1093/sysbio/sys086 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Mar;62(2):193-204. doi: 10.1093/sysbio/sys086. Epub 2012 Oct 3.

PMID- 23027088
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20130208
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 2
DP  - 2013 Mar
TI  - A comment on the use of stochastic character maps to estimate evolutionary rate
      variation in a continuously valued trait.
PG  - 339-45
LID - 10.1093/sysbio/sys084 [doi]
FAU - Revell, Liam J
AU  - Revell LJ
AD  - Department of Biology, University of Massachusetts Boston, Boston, MA 02125, USA.
      liam.revell@umb.edu
LA  - eng
PT  - Journal Article
DEP - 20121001
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2012/10/03 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/10/03 06:00
PHST- 2012/10/03 06:00 [entrez]
PHST- 2012/10/03 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys084 [pii]
AID - 10.1093/sysbio/sys084 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Mar;62(2):339-45. doi: 10.1093/sysbio/sys084. Epub 2012 Oct 1.

PMID- 23024153
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20130208
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 2
DP  - 2013 Mar
TI  - Comparing evolutionary rates for different phenotypic traits on a phylogeny using
      likelihood.
PG  - 181-92
LID - 10.1093/sysbio/sys083 [doi]
AB  - In recent years, likelihood-based approaches have been used with increasing
      frequency to evaluate macroevolutionary hypotheses of phenotypic evolution under 
      distinct evolutionary processes in a phylogenetic context (e.g., Brownian motion,
      Ornstein-Uhlenbeck, etc.), and to compare one or more evolutionary rates for the 
      same phenotypic trait along a phylogeny. It is also of interest to determine
      whether one trait evolves at a faster rate than another trait. However, to date
      no study has compared phylogenetic evolutionary rates between traits using
      likelihood, because a formal approach has not yet been proposed. In this article,
      I describe a new likelihood procedure for comparing evolutionary rates for two or
      more phenotypic traits on a phylogeny. This approach compares the likelihood of a
      model where each trait evolves at a distinct evolutionary rate to the likelihood 
      of a model where all traits are constrained to evolve at a common evolutionary
      rate. The method can also account for within-species measurement error and
      within-species trait covariation if available. Simulations revealed that the
      method has appropriate Type I error rates and statistical power. Importantly,
      when compared with existing approaches based on phylogenetically independent
      contrasts and methods that compare confidence intervals for model parameters, the
      likelihood method displays preferable statistical properties for a wide range of 
      simulated conditions. Thus, this likelihood-based method extends the phylogenetic
      comparative biology toolkit and provides evolutionary biologists with a more
      powerful means of determining when evolutionary rates differ between phenotypic
      traits. Finally, I provide an empirical example illustrating the approach by
      comparing rates of evolution for several phenotypic traits in Plethodon
      salamanders.
FAU - Adams, Dean C
AU  - Adams DC
AD  - Department of Ecology, Evolution, and Organismal Biology and Department of
      Statistics, Iowa State University, Ames, IA 50011, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120927
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - *Models, Statistical
MH  - *Phenotype
MH  - Phylogeny
MH  - Urodela/classification/physiology
EDAT- 2012/10/02 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/10/02 06:00
PHST- 2012/10/02 06:00 [entrez]
PHST- 2012/10/02 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys083 [pii]
AID - 10.1093/sysbio/sys083 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Mar;62(2):181-92. doi: 10.1093/sysbio/sys083. Epub 2012 Sep 27.

PMID- 22997022
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20121221
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - Solution of the Generalized Noah's Ark Problem.
PG  - 147-56
LID - 10.1093/sysbio/sys081 [doi]
AB  - The phylogenetic diversity (PD) of a set of species is a measure of the
      evolutionary distance among the species in the collection, based on a
      phylogenetic tree. Such a tree is composed of a root, internal nodes, and leaves 
      that correspond to the set of taxa under study. With each edge of the tree is
      associated a non-negative branch length (evolutionary distance). If a particular 
      survival probability is associated with each taxon, the PD measure becomes the
      expected PD measure. In the Noah's Ark Problem (NAP) introduced by Weitzman
      (1998), these survival probabilities can be increased at some cost. The problem
      is to determine how best to allocate a limited amount of resources to maximize
      the expected PD of the considered species. It is easy to formulate the NAP as a
      (difficult) nonlinear 0-1 programming problem. The aim of this article is to show
      that a general version of the NAP (GNAP) can be solved simply and efficiently
      with any set of edge weights and any set of survival probabilities by using
      standard mixed-integer linear programming software. The crucial point to move
      from a nonlinear program in binary variables to a mixed-integer linear program,
      is to approximate the logarithmic function by the lower envelope of a set of
      tangents to the curve. Solving the obtained mixed-integer linear program provides
      not only a near-optimal solution but also an upper bound on the value of the
      optimal solution. We also applied this approach to a generalization of the nature
      reserve problem (GNRP) that consists of selecting a set of regions to be
      conserved so that the expected PD of the set of species present in these regions 
      is maximized. In this case, the survival probabilities of different taxa are not 
      independent of each other. Computational results are presented to illustrate
      potentialities of the approach. Near-optimal solutions with hypothetical
      phylogenetic trees comprising about 4000 taxa are obtained in a few seconds or
      minutes of computing time for the GNAP, and in about 30 min for the GNRP. In all 
      the cases the average guarantee varies from 0% to 1.20%.
FAU - Billionnet, Alain
AU  - Billionnet A
AD  - Laboratoire CEDRIC, Ecole Nationale Superieure d'Informatique pour l'Industrie et
      l'Entreprise 1, square de la Resistance, 91025 Evry cedex, France.
      Alain.Billionnet@ensiie.fr
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120920
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Survival
EDAT- 2012/09/22 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/09/22 06:00
PHST- 2012/09/22 06:00 [entrez]
PHST- 2012/09/22 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys081 [pii]
AID - 10.1093/sysbio/sys081 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):147-56. doi: 10.1093/sysbio/sys081. Epub 2012 Sep 20.

PMID- 22993142
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20121221
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - Shape variation in outline shapes.
PG  - 134-46
LID - 10.1093/sysbio/sys080 [doi]
AB  - A general morphometric method for describing shape variation in a sample
      consisting of landmarks and multiple outline shapes is developed in this article.
      A distance metric is developed for such data and is used to embed the data in a
      low-dimensional Euclidean space. The Euclidean space is used to generate summary 
      statistics such as mean and principal shape variation which are implicitly
      represented in the original space using elements of the sample. A new distance
      metric for outline shapes is proposed based on Procrustes distance that does not 
      require the extraction of discrete points along the curve. The outline distance
      metric can be naturally combined with distances between landmarks. A method for
      aligning outlines and multiple outlines is developed that minimizes the distance 
      metric. The method is compared with semilandmarks on synthetic data and 2 real
      data sets. Outline methods produce useful and valid results when suitably
      constrained by landmarks and are useful visualization aids, but questions remain 
      about their suitability for answering biological questions until appropriate
      distance metrics can be biologically validated.
FAU - McCane, Brendan
AU  - McCane B
AD  - Department of Computer Science, University of Otago, Dunedin, New Zealand.
      mccane@cs.otago.ac.nz
LA  - eng
PT  - Journal Article
DEP - 20120919
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Animals
MH  - Biometry/*methods
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Drosophila/anatomy &amp; histology/classification
MH  - Principal Component Analysis
EDAT- 2012/09/21 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/09/21 06:00
PHST- 2012/09/21 06:00 [entrez]
PHST- 2012/09/21 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys080 [pii]
AID - 10.1093/sysbio/sys080 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):134-46. doi: 10.1093/sysbio/sys080. Epub 2012 Sep 19.

PMID- 22962038
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20130208
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 2
DP  - 2013 Mar
TI  - How can we improve accuracy of macroevolutionary rate estimates?
PG  - 321-9
LID - 10.1093/sysbio/sys073 [doi]
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Institute of Integrative Biology, ETH Zurich, 8092 Zurich, Switzerland.
      tanja.stadler@env.ethz.ch
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120908
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - Genetic Speciation
MH  - Models, Biological
MH  - Models, Statistical
MH  - Phylogeny
EDAT- 2012/09/11 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/09/11 06:00
PHST- 2012/09/11 06:00 [entrez]
PHST- 2012/09/11 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys073 [pii]
AID - 10.1093/sysbio/sys073 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Mar;62(2):321-9. doi: 10.1093/sysbio/sys073. Epub 2012 Sep 8.

PMID- 22962005
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - Serine codon-usage bias in deep phylogenomics: pancrustacean relationships as a
      case study.
PG  - 121-33
LID - 10.1093/sysbio/sys077 [doi]
AB  - Phylogenomic analyses of ancient relationships are usually performed using amino 
      acid data, but it is unclear whether amino acids or nucleotides should be
      preferred. With the 2-fold aim of addressing this problem and clarifying
      pancrustacean relationships, we explored the signals in the 62 protein-coding
      genes carefully assembled by Regier et al. in 2010. With reference to the
      pancrustaceans, this data set infers a highly supported nucleotide tree that is
      substantially different to the corresponding, but poorly supported, amino acid
      one. We show that the discrepancy between the nucleotide-based and the amino
      acids-based trees is caused by substitutions within synonymous codon families
      (especially those of serine-TCN and AGY). We show that different arthropod
      lineages are differentially biased in their usage of serine, arginine, and
      leucine synonymous codons, and that the serine bias is correlated with the
      topology derived from the nucleotides, but not the amino acids. We suggest that a
      parallel, partially compositionally driven, synonymous codon-usage bias affects
      the nucleotide topology. As substitutions between serine codon families can
      proceed through threonine or cysteine intermediates, amino acid data sets might
      also be affected by the serine codon-usage bias. We suggest that a Dayhoff
      recoding strategy would partially ameliorate the effects of such bias. Although
      amino acids provide an alternative hypothesis of pancrustacean relationships,
      neither the nucleotides nor the amino acids version of this data set seems to
      bring enough genuine phylogenetic information to robustly resolve the
      relationships within group, which should still be considered unresolved.
FAU - Rota-Stabelli, Omar
AU  - Rota-Stabelli O
AD  - Department of Biology, The National University of Ireland, Maynooth, Co. Kildare,
      Ireland. omar.rota@iasma.it
FAU - Lartillot, Nicolas
AU  - Lartillot N
FAU - Philippe, Herve
AU  - Philippe H
FAU - Pisani, Davide
AU  - Pisani D
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120906
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Amino Acids)
RN  - 0 (Codon)
RN  - 452VLY9402 (Serine)
RN  - EC 2.7.6.- (Diphosphotransferases)
RN  - EC 2.7.6.- (nucleotide 3'-pyrophosphokinase-2',3'-cyclic monophosphokinase)
SB  - IM
MH  - Amino Acids/genetics
MH  - Animals
MH  - Arthropods/classification/genetics
MH  - Bias
MH  - Codon/*genetics
MH  - Crustacea/*classification/*genetics
MH  - Diphosphotransferases/genetics
MH  - *Genomics
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Serine/*genetics
EDAT- 2012/09/11 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/09/11 06:00
PHST- 2012/09/11 06:00 [entrez]
PHST- 2012/09/11 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys077 [pii]
AID - 10.1093/sysbio/sys077 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):121-33. doi: 10.1093/sysbio/sys077. Epub 2012 Sep 6.

PMID- 22962004
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - Pruning rogue taxa improves phylogenetic accuracy: an efficient algorithm and
      webservice.
PG  - 162-6
LID - 10.1093/sysbio/sys078 [doi]
AB  - The presence of rogue taxa (rogues) in a set of trees can frequently have a
      negative impact on the results of a bootstrap analysis (e.g., the overall support
      in consensus trees). We introduce an efficient graph-based algorithm for rogue
      taxon identification as well as an interactive webservice implementing this
      algorithm. Compared with our previous method, the new algorithm is up to 4 orders
      of magnitude faster, while returning qualitatively identical results. Because of 
      this significant improvement in scalability, the new algorithm can now identify
      substantially more complex and compute-intensive rogue taxon constellations. On a
      large and diverse collection of real-world data sets, we show that our method
      yields better supported reduced/pruned consensus trees than any competing rogue
      taxon identification method. Using the parallel version of our open-source code, 
      we successfully identified rogue taxa in a set of 100 trees with 116 334 taxa
      each. For simulated data sets, we show that when removing/pruning rogue taxa with
      our method from a tree set, we consistently obtain bootstrap consensus trees as
      well as maximum-likelihood trees that are topologically closer to the respective 
      true trees.
FAU - Aberer, Andre J
AU  - Aberer AJ
AD  - Exelixis Laboratory, Scientific Computing Group, Heidelberg Institute for
      Theoretical Studies (HITS gGmbH), Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg,
      Germany. andre.aberer@h-its.org
FAU - Krompass, Denis
AU  - Krompass D
FAU - Stamatakis, Alexandros
AU  - Stamatakis A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120906
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Internet
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - *Software
PMC - PMC3526802
EDAT- 2012/09/11 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/09/11 06:00
PHST- 2012/09/11 06:00 [entrez]
PHST- 2012/09/11 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys078 [pii]
AID - 10.1093/sysbio/sys078 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):162-6. doi: 10.1093/sysbio/sys078. Epub 2012 Sep 6.

PMID- 22949484
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - TreeFix: statistically informed gene tree error correction using species trees.
PG  - 110-20
LID - 10.1093/sysbio/sys076 [doi]
AB  - Accurate gene tree reconstruction is a fundamental problem in phylogenetics, with
      many important applications. However, sequence data alone often lack enough
      information to confidently support one gene tree topology over many competing
      alternatives. Here, we present a novel framework for combining sequence data and 
      species tree information, and we describe an implementation of this framework in 
      TreeFix, a new phylogenetic program for improving gene tree reconstructions.
      Given a gene tree (preferably computed using a maximum-likelihood phylogenetic
      program), TreeFix finds a "statistically equivalent" gene tree that minimizes a
      species tree-based cost function. We have applied TreeFix to 2 clades of 12
      Drosophila and 16 fungal genomes, as well as to simulated phylogenies and show
      that it dramatically improves reconstructions compared with current
      state-of-the-art programs. Given its accuracy, speed, and simplicity, TreeFix
      should be applicable to a wide range of analyses and have many important
      implications for future investigations of gene evolution. The source code and a
      sample data set are available at http://compbio.mit.edu/treefix.
FAU - Wu, Yi-Chieh
AU  - Wu YC
AD  - Department of Electrical Engineering and Computer Science, Computer Science and
      Artificial Intelligence Laboratory, Massachusetts Institute of Technology,
      Cambridge, MA 02139, USA.
FAU - Rasmussen, Matthew D
AU  - Rasmussen MD
FAU - Bansal, Mukul S
AU  - Bansal MS
FAU - Kellis, Manolis
AU  - Kellis M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120904
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Drosophila/classification/genetics
MH  - Fungi/classification/genetics
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - *Software
PMC - PMC3526801
EDAT- 2012/09/06 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/09/06 06:00
PHST- 2012/09/06 06:00 [entrez]
PHST- 2012/09/06 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys076 [pii]
AID - 10.1093/sysbio/sys076 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):110-20. doi: 10.1093/sysbio/sys076. Epub 2012 Sep 4.

PMID- 22949483
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20121221
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - Phylogenomic insights into the cambrian explosion, the colonization of land and
      the evolution of flight in arthropoda.
PG  - 93-109
LID - 10.1093/sysbio/sys074 [doi]
AB  - The timing of the origin of arthropods in relation to the Cambrian explosion is
      still controversial, as are the timing of other arthropod macroevolutionary
      events such as the colonization of land and the evolution of flight. Here we
      assess the power of a phylogenomic approach to shed light on these major events
      in the evolutionary history of life on earth. Analyzing a large phylogenomic
      dataset (122 taxa, 62 genes) with a Bayesian-relaxed molecular clock, we
      simultaneously reconstructed the phylogenetic relationships and the absolute
      times of divergences among the arthropods. Simulations were used to test whether 
      our analysis could distinguish between alternative Cambrian explosion scenarios
      with increasing levels of autocorrelated rate variation. Our analyses support
      previous phylogenomic hypotheses and simulations indicate a Precambrian origin of
      the arthropods. Our results provide insights into the 3 independent colonizations
      of land by arthropods and suggest that evolution of insect wings happened much
      earlier than the fossil record indicates, with flight evolving during a period of
      increasing oxygen levels and impressively large forests. These and other findings
      provide a foundation for macroevolutionary and comparative genomic study of
      Arthropoda.
FAU - Wheat, Christopher W
AU  - Wheat CW
AD  - Department of Biosciences, PL 65, Viikinkaari 1, 00014 University of Helsinki,
      Finland.
FAU - Wahlberg, Niklas
AU  - Wahlberg N
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120904
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Arthropods/*classification/*genetics
MH  - Computer Simulation
MH  - Fossils
MH  - *Genomics
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2012/09/06 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/09/06 06:00
PHST- 2012/09/06 06:00 [entrez]
PHST- 2012/09/06 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys074 [pii]
AID - 10.1093/sysbio/sys074 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):93-109. doi: 10.1093/sysbio/sys074. Epub 2012 Sep 4.

PMID- 22949482
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20130208
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 2
DP  - 2013 Mar
TI  - Problems with estimation of ancestral frequencies under stationary models.
PG  - 330-8
LID - 10.1093/sysbio/sys075 [doi]
FAU - Susko, Edward
AU  - Susko E
AD  - Centre for Comparative Genomics and Evolutionary Bioinformatics Dalhousie
      University, Halifax, Nova Scotia, Canada B3H 4H7. susko@mathstat.dal.ca
FAU - Roger, Andrew J
AU  - Roger AJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120904
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
EDAT- 2012/09/06 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/09/06 06:00
PHST- 2012/09/06 06:00 [entrez]
PHST- 2012/09/06 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys075 [pii]
AID - 10.1093/sysbio/sys075 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Mar;62(2):330-8. doi: 10.1093/sysbio/sys075. Epub 2012 Sep 4.

PMID- 22914977
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20121221
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - Novel distances for dollo data.
PG  - 62-77
LID - 10.1093/sysbio/sys071 [doi]
AB  - We investigate distances on binary (presence/absence) data in the context of a
      Dollo process, where a trait can only arise once on a phylogenetic tree but may
      be lost many times. We introduce a novel distance, the Additive Dollo Distance
      (ADD), that applies to data generated under a Dollo model and show that it has
      some useful theoretical properties including an intriguing link to the
      LogDet/paralinear distance. Simulations of Dollo data are used to compare a
      number of binary distances including ADD, LogDet, a restriction-site-based
      distance, and some simple, but to our knowledge previously unstudied, variations 
      on common binary distances. The simulations suggest that ADD outperforms other
      distances on Dollo data. Interestingly, we found that the LogDet distance
      performs poorly in the context of a Dollo process; this may have implications for
      its use in connection with conditioned genome reconstruction. We apply the ADD to
      two Diversity Arrays Technology data sets, one that broadly covers Eucalyptus
      species and one that focuses on the Eucalyptus series Adnataria. We also
      reanalyze gene family presence/absence data from bacterial genomes obtained from 
      the COG database and compare the results with previous phylogenies estimated
      using the conditioned genome reconstruction approach. The results for these case 
      studies are largely congruent with previous studies, in some cases giving more
      phylogenetic resolution.
FAU - Woodhams, Michael
AU  - Woodhams M
AD  - School of Mathematics and Physics, CRC for Forestry, School of Plant Science,
      University of Tasmania, Private Bag 55, Hobart 7001, Australia.
FAU - Steane, Dorothy A
AU  - Steane DA
FAU - Jones, Rebecca C
AU  - Jones RC
FAU - Nicolle, Dean
AU  - Nicolle D
FAU - Moulton, Vincent
AU  - Moulton V
FAU - Holland, Barbara R
AU  - Holland BR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120822
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bacteria/classification/genetics
MH  - Computer Simulation
MH  - Eucalyptus/classification/genetics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Statistics as Topic
EDAT- 2012/08/24 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/08/24 06:00
PHST- 2012/08/24 06:00 [entrez]
PHST- 2012/08/24 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys071 [pii]
AID - 10.1093/sysbio/sys071 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):62-77. doi: 10.1093/sysbio/sys071. Epub 2012 Aug 22.

PMID- 22914976
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20121221
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - Low-parameter phylogenetic inference under the general markov model.
PG  - 78-92
LID - 10.1093/sysbio/sys072 [doi]
AB  - In their 2008 and 2009 articles, Sumner and colleagues introduced the
      "squangles"-a small set of Markov invariants for phylogenetic quartets. The
      squangles are consistent with the general Markov (GM) model and can be used to
      infer quartets without the need to explicitly estimate all parameters. As the GM 
      model is inhomogeneous and hence nonstationary, the squangles are expected to
      perform well compared with standard approaches when there are changes in base
      composition among species. However, the GM model assumes constant rates across
      sites, so the squangles should be confounded by data generated with invariant
      sites or other forms of rate-variation across sites. Here we implement the
      squangles in a least-squares setting that returns quartets weighted by either
      confidence or internal edge lengths, and we show how these weighted quartets can 
      be used as input into a variety of supertree and supernetwork methods. For the
      first time, we quantitatively investigate the robustness of the squangles to
      breaking of the constant rates-across-sites assumption on both simulated and real
      data sets; and we suggest a modification that improves the performance of the
      squangles in the presence of invariant sites. Our conclusion is that the
      squangles provide a novel tool for phylogenetic estimation that is complementary 
      to methods that explicitly account for rate-variation across sites, but rely on
      homogeneous-and hence stationary-models.
FAU - Holland, Barbara R
AU  - Holland BR
AD  - School of Mathematics and Physics, University of Tasmania, Hobart 7001,
      Australia. barbara.holland@utas.edu.au
FAU - Jarvis, Peter D
AU  - Jarvis PD
FAU - Sumner, Jeremy G
AU  - Sumner JG
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120822
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Least-Squares Analysis
MH  - Mammals/classification/genetics
MH  - Markov Chains
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2012/08/24 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/08/24 06:00
PHST- 2012/08/24 06:00 [entrez]
PHST- 2012/08/24 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys072 [pii]
AID - 10.1093/sysbio/sys072 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):78-92. doi: 10.1093/sysbio/sys072. Epub 2012 Aug 22.

PMID- 22887013
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20120813
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - The promises and pitfalls of next-generation sequencing data in phylogeography.
PG  - 713-5
LID - 10.1093/sysbio/sys050 [doi]
FAU - Carstens, Bryan
AU  - Carstens B
AD  - Department of Biological Science, Louisiana State University, Baton Rouge, LA
      70808, USA. bryan.c.carstens@gmail.com
FAU - Lemmon, Alan R
AU  - Lemmon AR
FAU - Lemmon, Emily Moriarty
AU  - Lemmon EM
LA  - eng
PT  - Introductory Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Phylogeography/instrumentation/*methods
MH  - Plants/genetics
MH  - Sequence Analysis, DNA/instrumentation/*methods
MH  - Vertebrates/genetics
EDAT- 2012/08/14 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/08/14 06:00
PHST- 2012/08/14 06:00 [entrez]
PHST- 2012/08/14 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys050 [pii]
AID - 10.1093/sysbio/sys050 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):713-5. doi: 10.1093/sysbio/sys050.

PMID- 22863681
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20121221
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - IKey+: a new single-access key generation web service.
PG  - 157-61
LID - 10.1093/sysbio/sys069 [doi]
AB  - Single-access keys are a major tool for biologists who need to identify
      specimens. The construction process of these keys is particularly complex
      (especially if the input data set is large) so having an automatic single-access 
      key generation tool is essential. As part of the European project ViBRANT, our
      aim was to develop such a tool as a web service, thus allowing end-users to
      integrate it directly into their workflow. IKey+generates single-access keys on
      demand, for single users or research institutions. It receives user input data
      (using the standard SDD format), accepts several key-generation parameters
      (affecting the key topology and representation), and supports several output
      formats. IKey+is freely available (sources and binary packages) at
      www.identificationkey.fr. Furthermore, it is deployed on our server and can be
      queried (for testing purposes) through a simple web client also available at
      www.identificationkey.fr (last accessed 13 August 2012). Finally, a client plugin
      will be integrated to the Scratchpads biodiversity networking tool
      (scratchpads.eu).
FAU - Burguiere, Thomas
AU  - Burguiere T
AD  - Laboratoire Informatique et Systematique, Museum National d'Histoire Naturelle,
      Departement Histoire de la Terre, UMR 7207 - C.N.R.S. - M.N.H.N. - U.P.M.C.,
      Paris, France. thomas.burguiere@upmc.fr
FAU - Causse, Florian
AU  - Causse F
FAU - Ung, Visotheary
AU  - Ung V
FAU - Vignes-Lebbe, Regine
AU  - Vignes-Lebbe R
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120803
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Asteraceae/classification
MH  - Classification/*methods
MH  - *Internet
MH  - Reproducibility of Results
MH  - *Software
EDAT- 2012/08/07 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/08/07 06:00
PHST- 2012/08/07 06:00 [entrez]
PHST- 2012/08/07 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys069 [pii]
AID - 10.1093/sysbio/sys069 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):157-61. doi: 10.1093/sysbio/sys069. Epub 2012 Aug 3.

PMID- 22851550
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - The evolutionary root of flowering plants.
PG  - 50-61
LID - 10.1093/sysbio/sys070 [doi]
AB  - Correct rooting of the angiosperm radiation is both challenging and necessary for
      understanding the origins and evolution of physiological and phenotypic traits in
      flowering plants. The problem is known to be difficult due to the large genetic
      distance separating flowering plants from other seed plants and the sparse taxon 
      sampling among basal angiosperms. Here, we provide further evidence for concern
      over substitution model misspecification in analyses of chloroplast DNA
      sequences. We show that support for Amborella as the sole representative of the
      most basal angiosperm lineage is founded on sequence site patterns poorly
      described by time-reversible substitution models. Improving the fit between
      sequence data and substitution model identifies Trithuria, Nymphaeaceae, and
      Amborella as surviving relatives of the most basal lineage of flowering plants.
      This finding indicates that aquatic and herbaceous species dominate the earliest 
      extant lineage of flowering plants. [; ; ; ; ; .].
FAU - Goremykin, Vadim V
AU  - Goremykin VV
AD  - IASMA Research Center, Via E. Mach 1, 38010 San Michele all'Adige (TN), Italy.
      Vadim.Goremykin@iasma.it
FAU - Nikiforova, Svetlana V
AU  - Nikiforova SV
FAU - Biggs, Patrick J
AU  - Biggs PJ
FAU - Zhong, Bojian
AU  - Zhong B
FAU - Delange, Peter
AU  - Delange P
FAU - Martin, William
AU  - Martin W
FAU - Woetzel, Stefan
AU  - Woetzel S
FAU - Atherton, Robin A
AU  - Atherton RA
FAU - McLenachan, Patricia A
AU  - McLenachan PA
FAU - Lockhart, Peter J
AU  - Lockhart PJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120731
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
SB  - IM
CIN - Syst Biol. 2014 May;63(3):368-82. PMID: 24391149
CIN - Syst Biol. 2015 Sep;64(5):879-91. PMID: 25995065
MH  - Coniferophyta/classification/genetics
MH  - DNA, Chloroplast/genetics
MH  - Genetic Heterogeneity
MH  - Magnoliopsida/*classification/*genetics
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Sequence Alignment
EDAT- 2012/08/02 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/08/02 06:00
PHST- 2012/08/02 06:00 [entrez]
PHST- 2012/08/02 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys070 [pii]
AID - 10.1093/sysbio/sys070 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):50-61. doi: 10.1093/sysbio/sys070. Epub 2012 Jul 31.

PMID- 22848088
OWN - NLM
STAT- MEDLINE
DCOM- 20130305
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 6
DP  - 2012 Dec 1
TI  - An extreme case of plant-insect codiversification: figs and fig-pollinating
      wasps.
PG  - 1029-47
LID - 10.1093/sysbio/sys068 [doi]
AB  - It is thought that speciation in phytophagous insects is often due to
      colonization of novel host plants, because radiations of plant and insect
      lineages are typically asynchronous. Recent phylogenetic comparisons have
      supported this model of diversification for both insect herbivores and
      specialized pollinators. An exceptional case where contemporaneous plant-insect
      diversification might be expected is the obligate mutualism between fig trees
      (Ficus species, Moraceae) and their pollinating wasps (Agaonidae, Hymenoptera).
      The ubiquity and ecological significance of this mutualism in tropical and
      subtropical ecosystems has long intrigued biologists, but the systematic
      challenge posed by &gt;750 interacting species pairs has hindered progress toward
      understanding its evolutionary history. In particular, taxon sampling and
      analytical tools have been insufficient for large-scale cophylogenetic analyses. 
      Here, we sampled nearly 200 interacting pairs of fig and wasp species from across
      the globe. Two supermatrices were assembled: on an average, wasps had sequences
      from 77% of 6 genes (5.6 kb), figs had sequences from 60% of 5 genes (5.5 kb),
      and overall 850 new DNA sequences were generated for this study. We also
      developed a new analytical tool, Jane 2, for event-based phylogenetic
      reconciliation analysis of very large data sets. Separate Bayesian phylogenetic
      analyses for figs and fig wasps under relaxed molecular clock assumptions
      indicate Cretaceous diversification of crown groups and contemporaneous
      divergence for nearly half of all fig and pollinator lineages. Event-based
      cophylogenetic analyses further support the codiversification hypothesis.
      Biogeographic analyses indicate that the present-day distribution of fig and
      pollinator lineages is consistent with a Eurasian origin and subsequent
      dispersal, rather than with Gondwanan vicariance. Overall, our findings indicate 
      that the fig-pollinator mutualism represents an extreme case among plant-insect
      interactions of coordinated dispersal and long-term codiversification.
      [Biogeography; coevolution; cospeciation; host switching; long-branch attraction;
      phylogeny.].
FAU - Cruaud, Astrid
AU  - Cruaud A
AD  - INRA, UMR1062 CBGP, F-34988 Montferrier-sur-Lez, France; Jodrell Laboratory,
      Royal Botanic Gardens, Kew, TW9 3DS, UK.
FAU - Ronsted, Nina
AU  - Ronsted N
FAU - Chantarasuwan, Bhanumas
AU  - Chantarasuwan B
FAU - Chou, Lien Siang
AU  - Chou LS
FAU - Clement, Wendy L
AU  - Clement WL
FAU - Couloux, Arnaud
AU  - Couloux A
FAU - Cousins, Benjamin
AU  - Cousins B
FAU - Genson, Gwenaelle
AU  - Genson G
FAU - Harrison, Rhett D
AU  - Harrison RD
FAU - Hanson, Paul E
AU  - Hanson PE
FAU - Hossaert-McKey, Martine
AU  - Hossaert-McKey M
FAU - Jabbour-Zahab, Roula
AU  - Jabbour-Zahab R
FAU - Jousselin, Emmanuelle
AU  - Jousselin E
FAU - Kerdelhue, Carole
AU  - Kerdelhue C
FAU - Kjellberg, Finn
AU  - Kjellberg F
FAU - Lopez-Vaamonde, Carlos
AU  - Lopez-Vaamonde C
FAU - Peebles, John
AU  - Peebles J
FAU - Peng, Yan-Qiong
AU  - Peng YQ
FAU - Pereira, Rodrigo Augusto Santinelo
AU  - Pereira RA
FAU - Schramm, Tselil
AU  - Schramm T
FAU - Ubaidillah, Rosichon
AU  - Ubaidillah R
FAU - van Noort, Simon
AU  - van Noort S
FAU - Weiblen, George D
AU  - Weiblen GD
FAU - Yang, Da-Rong
AU  - Yang DR
FAU - Yodpinyanee, Anak
AU  - Yodpinyanee A
FAU - Libeskind-Hadas, Ran
AU  - Libeskind-Hadas R
FAU - Cook, James M
AU  - Cook JM
FAU - Rasplus, Jean-Yves
AU  - Rasplus JY
FAU - Savolainen, Vincent
AU  - Savolainen V
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120730
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Ficus/*classification/genetics
MH  - Genetic Speciation
MH  - *Phylogeny
MH  - Phylogeography
MH  - Pollination
MH  - Symbiosis
MH  - Wasps/*classification/genetics
PMC - PMC3478567
EDAT- 2012/08/01 06:00
MHDA- 2013/03/06 06:00
CRDT- 2012/08/01 06:00
PHST- 2012/08/01 06:00 [entrez]
PHST- 2012/08/01 06:00 [pubmed]
PHST- 2013/03/06 06:00 [medline]
AID - sys068 [pii]
AID - 10.1093/sysbio/sys068 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Dec 1;61(6):1029-47. doi: 10.1093/sysbio/sys068. Epub 2012 Jul
      30.

PMID- 22831877
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20121221
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - Ratite nonmonophyly: independent evidence from 40 novel Loci.
PG  - 35-49
LID - 10.1093/sysbio/sys067 [doi]
AB  - Large-scale multilocus studies have become common in molecular phylogenetics, but
      the best way to interpret these studies when their results strongly conflict with
      prior information about phylogeny remains unclear. An example of such a conflict 
      is provided by the ratites (the large flightless birds of southern land masses,
      including ostriches, emus, and rheas). Ratite monophyly is strongly supported by 
      both morphological data and many earlier molecular studies and is used as a
      textbook example of vicariance biogeography. However, recent studies have
      indicated that ratites are not monophyletic; instead, the volant tinamous nest
      inside the ratites rather than forming their sister group within the avian
      superorder Palaeognathae. Large-scale studies can exhibit biases that reflect a
      number of factors, including limitations in the fit of the evolutionary models
      used for analyses and problems with sequence alignment, so the unexpected
      conclusion that ratites are not monophyletic needs to be rigorously evaluated. A 
      rigorous approach to testing novel hypotheses generated by large-scale studies is
      to collect independent evidence (i.e., excluding the loci and/or traits used to
      generate the hypotheses). We used 40 nuclear loci not used in previous studies
      that investigated the relationship among ratites and tinamous. Our results
      strongly support the recent molecular studies, revealing that the deepest branch 
      within Palaeognathae separates the ostrich from other members of the clade,
      rather than the traditional hypothesis that separates the tinamous from the
      ratites. To ensure these results reflected evolutionary history, we examined
      potential biases in types of loci used, heterotachy, alignment biases, and
      discordance between gene trees and the species tree. All analyses consistently
      supported nonmonophyly of the ratites and no confounding biases were identified. 
      This confirmation that ratites are not monophyletic using independent evidence
      will hopefully stimulate further comparative research on paleognath development
      and genetics that might reveal the basis of the morphological convergence in
      these large, flightless birds.
FAU - Smith, Jordan V
AU  - Smith JV
AD  - Department of Biology, University of Florida, P.O. Box 118525, Gainesville, FL
      32611, USA.
FAU - Braun, Edward L
AU  - Braun EL
FAU - Kimball, Rebecca T
AU  - Kimball RT
LA  - eng
SI  - GENBANK/JX120800
SI  - GENBANK/JX120801
SI  - GENBANK/JX120802
SI  - GENBANK/JX120803
SI  - GENBANK/JX120804
SI  - GENBANK/JX120805
SI  - GENBANK/JX120806
SI  - GENBANK/JX120807
SI  - GENBANK/JX120808
SI  - GENBANK/JX120809
SI  - GENBANK/JX120810
SI  - GENBANK/JX120811
SI  - GENBANK/JX120812
SI  - GENBANK/JX120813
SI  - GENBANK/JX120814
SI  - GENBANK/JX120815
SI  - GENBANK/JX120816
SI  - GENBANK/JX120817
SI  - GENBANK/JX120818
SI  - GENBANK/JX120819
SI  - GENBANK/JX120820
SI  - GENBANK/JX120821
SI  - GENBANK/JX120822
SI  - GENBANK/JX120823
SI  - GENBANK/JX120824
SI  - GENBANK/JX120825
SI  - GENBANK/JX120826
SI  - GENBANK/JX120827
SI  - GENBANK/JX120828
SI  - GENBANK/JX120829
SI  - GENBANK/JX120830
SI  - GENBANK/JX120831
SI  - GENBANK/JX120832
SI  - GENBANK/JX120833
SI  - GENBANK/JX120834
SI  - GENBANK/JX120835
SI  - GENBANK/JX120836
SI  - GENBANK/JX120837
SI  - GENBANK/JX120838
SI  - GENBANK/JX120839
SI  - GENBANK/JX120840
SI  - GENBANK/JX120841
SI  - GENBANK/JX120842
SI  - GENBANK/JX120843
SI  - GENBANK/JX120844
SI  - GENBANK/JX120845
SI  - GENBANK/JX120846
SI  - GENBANK/JX120847
SI  - GENBANK/JX120848
SI  - GENBANK/JX120849
SI  - GENBANK/JX120850
SI  - GENBANK/JX120851
SI  - GENBANK/JX120852
SI  - GENBANK/JX120853
SI  - GENBANK/JX120854
SI  - GENBANK/JX120855
SI  - GENBANK/JX120856
SI  - GENBANK/JX120857
SI  - GENBANK/JX120858
SI  - GENBANK/JX120859
SI  - GENBANK/JX120860
SI  - GENBANK/JX120861
SI  - GENBANK/JX120862
SI  - GENBANK/JX120863
SI  - GENBANK/JX120864
SI  - GENBANK/JX120865
SI  - GENBANK/JX120866
SI  - GENBANK/JX120867
SI  - GENBANK/JX120868
SI  - GENBANK/JX120869
SI  - GENBANK/JX120870
SI  - GENBANK/JX120871
SI  - GENBANK/JX120872
SI  - GENBANK/JX120873
SI  - GENBANK/JX120874
SI  - GENBANK/JX120875
SI  - GENBANK/JX120876
SI  - GENBANK/JX120877
SI  - GENBANK/JX120878
SI  - GENBANK/JX120879
SI  - GENBANK/JX120880
SI  - GENBANK/JX120881
SI  - GENBANK/JX120882
SI  - GENBANK/JX120883
SI  - GENBANK/JX120884
SI  - GENBANK/JX120885
SI  - GENBANK/JX120886
SI  - GENBANK/JX120887
SI  - GENBANK/JX120888
SI  - GENBANK/JX120889
SI  - GENBANK/JX120890
SI  - GENBANK/JX120891
SI  - GENBANK/JX120892
SI  - GENBANK/JX120893
SI  - GENBANK/JX120894
SI  - GENBANK/JX120895
SI  - GENBANK/JX120896
SI  - GENBANK/JX120897
SI  - GENBANK/JX120898
SI  - GENBANK/JX120899
SI  - GENBANK/JX120900
SI  - GENBANK/JX120901
SI  - GENBANK/JX120902
SI  - GENBANK/JX120903
SI  - GENBANK/JX120904
SI  - GENBANK/JX120905
SI  - GENBANK/JX120906
SI  - GENBANK/JX120907
SI  - GENBANK/JX120908
SI  - GENBANK/JX120909
SI  - GENBANK/JX120910
SI  - GENBANK/JX120911
SI  - GENBANK/JX120912
SI  - GENBANK/JX120913
SI  - GENBANK/JX120914
SI  - GENBANK/JX120915
SI  - GENBANK/JX120916
SI  - GENBANK/JX120917
SI  - GENBANK/JX120918
SI  - GENBANK/JX120919
SI  - GENBANK/JX120920
SI  - GENBANK/JX120921
SI  - GENBANK/JX120922
SI  - GENBANK/JX120923
SI  - GENBANK/JX120924
SI  - GENBANK/JX120925
SI  - GENBANK/JX120926
SI  - GENBANK/JX120927
SI  - GENBANK/JX120928
SI  - GENBANK/JX120929
SI  - GENBANK/JX120930
SI  - GENBANK/JX120931
SI  - GENBANK/JX120932
SI  - GENBANK/JX120933
SI  - GENBANK/JX120934
SI  - GENBANK/JX120935
SI  - GENBANK/JX120936
SI  - GENBANK/JX120937
SI  - GENBANK/JX120938
SI  - GENBANK/JX120939
SI  - GENBANK/JX120940
SI  - GENBANK/JX120941
SI  - GENBANK/JX120942
SI  - GENBANK/JX120943
SI  - GENBANK/JX120944
SI  - GENBANK/JX120945
SI  - GENBANK/JX120946
SI  - GENBANK/JX120947
SI  - GENBANK/JX120948
SI  - GENBANK/JX120949
SI  - GENBANK/JX120950
SI  - GENBANK/JX120951
SI  - GENBANK/JX120952
SI  - GENBANK/JX120953
SI  - GENBANK/JX120954
SI  - GENBANK/JX120955
SI  - GENBANK/JX120956
SI  - GENBANK/JX120957
SI  - GENBANK/JX120958
SI  - GENBANK/JX120959
SI  - GENBANK/JX120960
SI  - GENBANK/JX120961
SI  - GENBANK/JX120962
SI  - GENBANK/JX120963
SI  - GENBANK/JX120964
SI  - GENBANK/JX120965
SI  - GENBANK/JX120966
SI  - GENBANK/JX120967
SI  - GENBANK/JX120968
SI  - GENBANK/JX120969
SI  - GENBANK/JX120970
SI  - GENBANK/JX120971
SI  - GENBANK/JX120972
SI  - GENBANK/JX120973
SI  - GENBANK/JX120974
SI  - GENBANK/JX120975
SI  - GENBANK/JX120976
SI  - GENBANK/JX120977
SI  - GENBANK/JX120978
SI  - GENBANK/JX120979
SI  - GENBANK/JX120980
SI  - GENBANK/JX120981
SI  - GENBANK/JX120982
SI  - GENBANK/JX120983
SI  - GENBANK/JX120984
SI  - GENBANK/JX120985
SI  - GENBANK/JX120986
SI  - GENBANK/JX120987
SI  - GENBANK/JX120988
SI  - GENBANK/JX120989
SI  - GENBANK/JX120990
SI  - GENBANK/JX120991
SI  - GENBANK/JX120992
SI  - GENBANK/JX120993
SI  - GENBANK/JX120994
SI  - GENBANK/JX120995
SI  - GENBANK/JX120996
SI  - GENBANK/JX120997
SI  - GENBANK/JX120998
SI  - GENBANK/JX120999
SI  - GENBANK/JX121000
SI  - GENBANK/JX121001
SI  - GENBANK/JX121002
SI  - GENBANK/JX121003
SI  - GENBANK/JX121004
SI  - GENBANK/JX121005
SI  - GENBANK/JX121006
SI  - GENBANK/JX121007
SI  - GENBANK/JX121008
SI  - GENBANK/JX121009
SI  - GENBANK/JX121010
SI  - GENBANK/JX121011
SI  - GENBANK/JX121012
SI  - GENBANK/JX121013
SI  - GENBANK/JX121014
SI  - GENBANK/JX121015
SI  - GENBANK/JX121016
SI  - GENBANK/JX121017
SI  - GENBANK/JX121018
SI  - GENBANK/JX121019
SI  - GENBANK/JX121020
SI  - GENBANK/JX121021
SI  - GENBANK/JX121022
SI  - GENBANK/JX121023
SI  - GENBANK/JX121024
SI  - GENBANK/JX121025
SI  - GENBANK/JX121026
SI  - GENBANK/JX121027
SI  - GENBANK/JX121028
SI  - GENBANK/JX121029
SI  - GENBANK/JX121030
SI  - GENBANK/JX121031
SI  - GENBANK/JX121032
SI  - GENBANK/JX121033
SI  - GENBANK/JX121034
SI  - GENBANK/JX121035
SI  - GENBANK/JX121036
SI  - GENBANK/JX121037
SI  - GENBANK/JX121038
SI  - GENBANK/JX121039
SI  - GENBANK/JX121040
SI  - GENBANK/JX121041
SI  - GENBANK/JX121042
SI  - GENBANK/JX121043
SI  - GENBANK/JX121044
SI  - GENBANK/JX121045
SI  - GENBANK/JX121046
SI  - GENBANK/JX121047
SI  - GENBANK/JX121048
SI  - GENBANK/JX121049
SI  - GENBANK/JX121050
SI  - GENBANK/JX121051
SI  - GENBANK/JX121052
SI  - GENBANK/JX121053
SI  - GENBANK/JX121054
SI  - GENBANK/JX121055
SI  - GENBANK/JX121056
SI  - GENBANK/JX121057
SI  - GENBANK/JX121058
SI  - GENBANK/JX121059
SI  - GENBANK/JX121060
SI  - GENBANK/JX121061
SI  - GENBANK/JX121062
SI  - GENBANK/JX121063
SI  - GENBANK/JX121064
SI  - GENBANK/JX121065
SI  - GENBANK/JX121066
SI  - GENBANK/JX121067
SI  - GENBANK/JX121068
SI  - GENBANK/JX121069
SI  - GENBANK/JX121070
SI  - GENBANK/JX121071
SI  - GENBANK/JX121072
SI  - GENBANK/JX121073
SI  - GENBANK/JX121074
SI  - GENBANK/JX121075
SI  - GENBANK/JX121076
SI  - GENBANK/JX121077
SI  - GENBANK/JX121078
SI  - GENBANK/JX121079
SI  - GENBANK/JX121080
SI  - GENBANK/JX121081
SI  - GENBANK/JX121082
SI  - GENBANK/JX121083
SI  - GENBANK/JX121084
SI  - GENBANK/JX121085
SI  - GENBANK/JX121086
SI  - GENBANK/JX121087
SI  - GENBANK/JX121088
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120724
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Genetic Loci/*genetics
MH  - Molecular Sequence Data
MH  - Palaeognathae/*classification/*genetics
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
EDAT- 2012/07/27 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/07/27 06:00
PHST- 2012/07/27 06:00 [entrez]
PHST- 2012/07/27 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys067 [pii]
AID - 10.1093/sysbio/sys067 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):35-49. doi: 10.1093/sysbio/sys067. Epub 2012 Jul 24.

PMID- 22798332
OWN - NLM
STAT- MEDLINE
DCOM- 20130305
LR  - 20121024
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 6
DP  - 2012 Dec 1
TI  - When do phylogenetic mixture models mimic other phylogenetic models?
PG  - 1049-59
LID - 10.1093/sysbio/sys064 [doi]
AB  - Phylogenetic mixture models, in which the sites in sequences undergo different
      substitution processes along the same or different trees, allow the description
      of heterogeneous evolutionary processes. As data sets consisting of longer
      sequences become available, it is important to understand such models, for both
      theoretical insights and use in statistical analyses. Some recent articles have
      highlighted disturbing "mimicking" behavior in which a distribution from a
      mixture model is identical to one arising on a different tree or trees. Other
      works have indicated such problems are unlikely to occur in practice, as they
      require very special parameter choices. After surveying some of these works on
      mixture models, we give several new results. In general, if the number of
      components in a generating mixture is not too large and we disallow zero or
      infinite branch lengths, then it cannot mimic the behavior of a nonmixture on a
      different tree. On the other hand, if the mixture model is locally
      overparameterized, it is possible for a phylogenetic mixture model to mimic
      distributions of another tree model. Although theoretical questions remain, these
      sorts of results can serve as a guide to when the use of mixture models in either
      maximum likelihood or Bayesian frameworks is likely to lead to statistically
      consistent inference, and when mimicking due to heterogeneity should be
      considered a realistic possibility. [Phylogenetic mixture models; parameter
      identifiability; heterogeneous sequence evolution.].
FAU - Allman, Elizabeth S
AU  - Allman ES
AD  - Department of Mathematics and Statistics, University of Alaska Fairbanks, PO Box 
      756660, Fairbanks, AK 99775, USA.
FAU - Rhodes, John A
AU  - Rhodes JA
FAU - Sullivant, Seth
AU  - Sullivant S
LA  - eng
PT  - Journal Article
DEP - 20120712
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Computer Simulation
MH  - Data Interpretation, Statistical
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2012/07/17 06:00
MHDA- 2013/03/06 06:00
CRDT- 2012/07/17 06:00
PHST- 2012/07/17 06:00 [entrez]
PHST- 2012/07/17 06:00 [pubmed]
PHST- 2013/03/06 06:00 [medline]
AID - sys064 [pii]
AID - 10.1093/sysbio/sys064 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Dec 1;61(6):1049-59. doi: 10.1093/sysbio/sys064. Epub 2012 Jul
      12.

PMID- 22798331
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20121221
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - From trajectories to averages: an improved description of the heterogeneity of
      substitution rates along lineages.
PG  - 22-34
LID - 10.1093/sysbio/sys063 [doi]
AB  - The accuracy and precision of species divergence date estimation from molecular
      data strongly depend on the models describing the variation of substitution rates
      along a phylogeny. These models generally assume that rates randomly fluctuate
      along branches from one node to the next. However, for mathematical convenience, 
      the stochasticity of such a process is ignored when translating these rate
      trajectories into branch lengths. This study addresses this shortcoming. A new
      approach is described that explicitly considers the average substitution rates
      along branches as random quantities, resulting in a more realistic description of
      the variations of evolutionary rates along lineages. The proposed method provides
      more precise estimates of the rate autocorrelation parameter as well as
      divergence times. Also, simulation results indicate that ignoring the stochastic 
      variation of rates along edges can lead to significant overestimation of specific
      node ages. Altogether, the new approach introduced in this study is a step
      forward to designing biologically relevant models of rate evolution that are well
      suited to data sets with dense taxon sampling which are likely to present rate
      autocorrelation. The computer programme PhyTime, part of the PhyML package and
      implementing the new approach, is available from http://code.google.com/p/phyml
      (last accessed 1 August 2012).
FAU - Guindon, Stephane
AU  - Guindon S
AD  - Department of Statistics, University of Auckland, Auckland, 1010, New Zealand.
      s.guindon@auckland.ac.nz
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120712
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Butterflies/classification/genetics
MH  - Cebidae/classification/genetics
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Genetic Heterogeneity
MH  - Genetic Speciation
MH  - HIV-1/classification/genetics
MH  - *Models, Genetic
MH  - Reproducibility of Results
MH  - Rodentia/classification/genetics
MH  - Software
EDAT- 2012/07/17 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/07/17 06:00
PHST- 2012/07/17 06:00 [entrez]
PHST- 2012/07/17 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys063 [pii]
AID - 10.1093/sysbio/sys063 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):22-34. doi: 10.1093/sysbio/sys063. Epub 2012 Jul 12.

PMID- 22780991
OWN - NLM
STAT- MEDLINE
DCOM- 20130305
LR  - 20121024
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 6
DP  - 2012 Dec 1
TI  - Dendroscope 3: an interactive tool for rooted phylogenetic trees and networks.
PG  - 1061-7
LID - 10.1093/sysbio/sys062 [doi]
AB  - Dendroscope 3 is a new program for working with rooted phylogenetic trees and
      networks. It provides a number of methods for drawing and comparing rooted
      phylogenetic networks, and for computing them from rooted trees. The program can 
      be used interactively or in command-line mode. The program is written in Java,
      use of the software is free, and installers for all 3 major operating systems can
      be downloaded from www.dendroscope.org. [Phylogenetic trees; phylogenetic
      networks; software.].
FAU - Huson, Daniel H
AU  - Huson DH
AD  - Department of Computer Science, Center for Bioinformatics (ZBIT), University of
      Tubingen, 72076 Tubingen, Germany. daniel.huson@uni-tuebingen.de
FAU - Scornavacca, Celine
AU  - Scornavacca C
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120710
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Phylogeny
MH  - *Software
EDAT- 2012/07/12 06:00
MHDA- 2013/03/06 06:00
CRDT- 2012/07/12 06:00
PHST- 2012/07/12 06:00 [entrez]
PHST- 2012/07/12 06:00 [pubmed]
PHST- 2013/03/06 06:00 [medline]
AID - sys062 [pii]
AID - 10.1093/sysbio/sys062 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Dec 1;61(6):1061-7. doi: 10.1093/sysbio/sys062. Epub 2012 Jul 10.

PMID- 22744774
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20180222
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - Integrating fuzzy logic and statistics to improve the reliable delimitation of
      biogeographic regions and transition zones.
PG  - 1-21
LID - 10.1093/sysbio/sys061 [doi]
AB  - This study uses the amphibian species of the Mediterranean basin to develop a
      consistent procedure based on fuzzy sets with which biogeographic regions and
      biotic transition zones can be objectively detected and reliably mapped.
      Biogeographical regionalizations are abstractions of the geographical
      organization of life on Earth that provide frameworks for cataloguing species and
      ecosystems, for answering basic questions in biogeography, evolutionary biology, 
      and systematics, and for assessing priorities for conservation. On the other
      hand, limits between regions may form sharply defined boundaries along some parts
      of their borders, whereas elsewhere they may consist of broad transition zones.
      The fuzzy set approach provides a heuristic way to analyse the complexity of the 
      biota within an area; significantly different regions are detected whose mutual
      limits are sometimes fuzzy, sometimes clearly crisp. Most of the regionalizations
      described in the literature for the Mediterranean biogeographical area present a 
      certain degree of convergence when they are compared within the context of fuzzy 
      interpretation, as many of the differences found between regionalizations are
      located in transition zones, according to our case study. Compared with other
      classification procedures based on fuzzy sets, the novelty of our method is that 
      both fuzzy logic and statistics are used together in a synergy in order to avoid 
      arbitrary decisions in the definition of biogeographic regions and transition
      zones.
FAU - Olivero, Jesus
AU  - Olivero J
AD  - Grupo de Biogeografia, Diversidad y Conservacion, Departamento de Biologia
      Animal, Facultad de Ciencias, Universidad de Malaga, 29071 Malaga, Spain.
      jesusolivero@uma.es
FAU - Marquez, Ana L
AU  - Marquez AL
FAU - Real, Raimundo
AU  - Real R
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120628
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2016 Nov;65(6):1123. This article, which was published in its accepted
      format on 8 July 2016, was published in error as a duplicate of
      10.1093/sysbio/sys061 "Integrating Fuzzy Logic and Statistics to Improve the
      Reliable Delimitation of Biogeographic Regions and Transition Z. PMID: 27401436
MH  - Amphibians/classification/*physiology
MH  - Animals
MH  - Ecosystem
MH  - *Fuzzy Logic
MH  - Mediterranean Region
MH  - Models, Statistical
MH  - Phylogeny
MH  - Phylogeography/*methods
EDAT- 2012/06/30 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/06/30 06:00
PHST- 2012/06/30 06:00 [entrez]
PHST- 2012/06/30 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys061 [pii]
AID - 10.1093/sysbio/sys061 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):1-21. doi: 10.1093/sysbio/sys061. Epub 2012 Jun 28.

PMID- 22744773
OWN - NLM
STAT- MEDLINE
DCOM- 20130305
LR  - 20121024
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 6
DP  - 2012 Dec 1
TI  - The evolution of pharyngognathy: a phylogenetic and functional appraisal of the
      pharyngeal jaw key innovation in labroid fishes and beyond.
PG  - 1001-27
LID - 10.1093/sysbio/sys060 [doi]
AB  - The perciform group Labroidei includes approximately 2600 species and comprises
      some of the most diverse and successful lineages of teleost fishes. Composed of
      four major clades, Cichlidae, Labridae (wrasses, parrotfishes, and weed
      whitings), Pomacentridae (damselfishes), and Embiotocidae (surfperches); labroids
      have been an icon for studies of biodiversity, adaptive radiation, and sexual
      selection. The success and diversification of labroids have been largely
      attributed to the presence of a major innovation in the pharyngeal jaw apparatus,
      pharyngognathy, which is hypothesized to increase feeding capacity and
      versatility. We present results of large-scale phylogenetic analyses and a survey
      of pharyngeal jaw functional morphology that allow us to examine the evolution of
      pharyngognathy in a historical context. Phylogenetic analyses were based on a
      sample of 188 acanthomorph (spiny-rayed fish) species, primarily percomorphs
      (perch-like fishes), and DNA sequence data collected from 10 nuclear loci that
      have been previously used to resolve higher level ray-finned fish relationships. 
      Phylogenies inferred from this dataset using maximum likelihood, Bayesian, and
      species tree analyses indicate polyphyly of the traditional Labroidei and clearly
      separate Labridae from the remainder of the traditional labroid lineages
      (Cichlidae, Embiotocidae, and Pomacentridae). These three "chromide" families
      grouped within a newly discovered clade of 40 families and more than 4800 species
      (&gt;27% of percomorphs and &gt;16% of all ray-finned fishes), which we name
      Ovalentaria for its characteristic demersal, adhesive eggs with chorionic
      filaments. This fantastically diverse clade includes some of the most
      species-rich lineages of marine and freshwater fishes, including all
      representatives of the Cichlidae, Embiotocidae, Pomacentridae, Ambassidae,
      Gobiesocidae, Grammatidae, Mugilidae, Opistognathidae, Pholidichthyidae,
      Plesiopidae (including Notograptus), Polycentridae, Pseudochromidae,
      Atherinomorpha, and Blennioidei. Beyond the discovery of Ovalentaria, this study 
      provides a surprising, but well-supported, hypothesis for a convict-blenny
      (Pholidichthys) sister group to the charismatic cichlids and new insights into
      the evolution of pharyngognathy. Bayesian stochastic mapping ancestral state
      reconstructions indicate that pharyngognathy has evolved at least six times in
      percomorphs, including four separate origins in members of the former Labroidei, 
      one origin in the Centrogenyidae, and one origin within Beloniformes. Our
      analyses indicate that all pharyngognathous fishes have a mechanically efficient 
      biting mechanism enabled by the muscular sling and a single lower jaw element.
      However, a major distinction exists between Labridae, which lacks the widespread,
      generalized percomorph pharyngeal biting mechanism, and all other
      pharyngognathous clades, which possess this generalized biting mechanism in
      addition to pharyngognathy. Our results reveal a remarkable history of
      pharyngognathy: far from a single origin, it appears to have evolved at least six
      times, and its status as a major evolutionary innovation is reinforced by it
      being a synapomorphy for several independent major radiations, including some of 
      the most species rich and ecologically diverse percomorph clades of coral reef
      and tropical freshwater fishes, Labridae and Cichlidae. [Acanthomorpha;
      Beloniformes; Centrogenyidae; key innovation; Labroidei; Ovalentaria; pharyngeal 
      jaws; Perciformes.].
FAU - Wainwright, Peter C
AU  - Wainwright PC
AD  - Department of Evolution and Ecology, University of California, One Shields
      Avenue, Davis, CA 95616, USA. pcwainwright@ucdavis.edu
FAU - Smith, W Leo
AU  - Smith WL
FAU - Price, Samantha A
AU  - Price SA
FAU - Tang, Kevin L
AU  - Tang KL
FAU - Sparks, John S
AU  - Sparks JS
FAU - Ferry, Lara A
AU  - Ferry LA
FAU - Kuhn, Kristen L
AU  - Kuhn KL
FAU - Eytan, Ron I
AU  - Eytan RI
FAU - Near, Thomas J
AU  - Near TJ
LA  - eng
SI  - GENBANK/JX188676
SI  - GENBANK/JX188677
SI  - GENBANK/JX188678
SI  - GENBANK/JX188679
SI  - GENBANK/JX188680
SI  - GENBANK/JX188681
SI  - GENBANK/JX188682
SI  - GENBANK/JX188683
SI  - GENBANK/JX188684
SI  - GENBANK/JX188685
SI  - GENBANK/JX188686
SI  - GENBANK/JX188687
SI  - GENBANK/JX188688
SI  - GENBANK/JX188689
SI  - GENBANK/JX188690
SI  - GENBANK/JX188691
SI  - GENBANK/JX188692
SI  - GENBANK/JX188693
SI  - GENBANK/JX188694
SI  - GENBANK/JX188695
SI  - GENBANK/JX188696
SI  - GENBANK/JX188697
SI  - GENBANK/JX188698
SI  - GENBANK/JX188699
SI  - GENBANK/JX188700
SI  - GENBANK/JX188701
SI  - GENBANK/JX188702
SI  - GENBANK/JX188703
SI  - GENBANK/JX188704
SI  - GENBANK/JX188705
SI  - GENBANK/JX188706
SI  - GENBANK/JX188707
SI  - GENBANK/JX188708
SI  - GENBANK/JX188709
SI  - GENBANK/JX188710
SI  - GENBANK/JX188711
SI  - GENBANK/JX188712
SI  - GENBANK/JX188713
SI  - GENBANK/JX188714
SI  - GENBANK/JX188715
SI  - GENBANK/JX188716
SI  - GENBANK/JX188717
SI  - GENBANK/JX188718
SI  - GENBANK/JX188719
SI  - GENBANK/JX188720
SI  - GENBANK/JX188721
SI  - GENBANK/JX188722
SI  - GENBANK/JX188723
SI  - GENBANK/JX188724
SI  - GENBANK/JX188725
SI  - GENBANK/JX188726
SI  - GENBANK/JX188727
SI  - GENBANK/JX188728
SI  - GENBANK/JX188729
SI  - GENBANK/JX188730
SI  - GENBANK/JX188731
SI  - GENBANK/JX188732
SI  - GENBANK/JX188733
SI  - GENBANK/JX188734
SI  - GENBANK/JX188735
SI  - GENBANK/JX188736
SI  - GENBANK/JX188737
SI  - GENBANK/JX188738
SI  - GENBANK/JX188739
SI  - GENBANK/JX188740
SI  - GENBANK/JX188741
SI  - GENBANK/JX188742
SI  - GENBANK/JX188743
SI  - GENBANK/JX188744
SI  - GENBANK/JX188745
SI  - GENBANK/JX188746
SI  - GENBANK/JX188747
SI  - GENBANK/JX188748
SI  - GENBANK/JX188749
SI  - GENBANK/JX188750
SI  - GENBANK/JX188751
SI  - GENBANK/JX188752
SI  - GENBANK/JX188753
SI  - GENBANK/JX188754
SI  - GENBANK/JX188755
SI  - GENBANK/JX188756
SI  - GENBANK/JX188757
SI  - GENBANK/JX188758
SI  - GENBANK/JX188759
SI  - GENBANK/JX188760
SI  - GENBANK/JX188761
SI  - GENBANK/JX188762
SI  - GENBANK/JX188763
SI  - GENBANK/JX188764
SI  - GENBANK/JX188765
SI  - GENBANK/JX188766
SI  - GENBANK/JX188767
SI  - GENBANK/JX188768
SI  - GENBANK/JX188769
SI  - GENBANK/JX188770
SI  - GENBANK/JX188771
SI  - GENBANK/JX188772
SI  - GENBANK/JX188773
SI  - GENBANK/JX188774
SI  - GENBANK/JX188775
SI  - GENBANK/JX188776
SI  - GENBANK/JX188777
SI  - GENBANK/JX188778
SI  - GENBANK/JX188779
SI  - GENBANK/JX188780
SI  - GENBANK/JX188781
SI  - GENBANK/JX188782
SI  - GENBANK/JX188783
SI  - GENBANK/JX188784
SI  - GENBANK/JX188785
SI  - GENBANK/JX188786
SI  - GENBANK/JX188787
SI  - GENBANK/JX188788
SI  - GENBANK/JX188789
SI  - GENBANK/JX188790
SI  - GENBANK/JX188791
SI  - GENBANK/JX188792
SI  - GENBANK/JX188793
SI  - GENBANK/JX188794
SI  - GENBANK/JX188795
SI  - GENBANK/JX188796
SI  - GENBANK/JX188797
SI  - GENBANK/JX188798
SI  - GENBANK/JX188799
SI  - GENBANK/JX188800
SI  - GENBANK/JX188801
SI  - GENBANK/JX188802
SI  - GENBANK/JX188803
SI  - GENBANK/JX188804
SI  - GENBANK/JX188805
SI  - GENBANK/JX188806
SI  - GENBANK/JX188807
SI  - GENBANK/JX188808
SI  - GENBANK/JX188809
SI  - GENBANK/JX188810
SI  - GENBANK/JX188811
SI  - GENBANK/JX188812
SI  - GENBANK/JX188813
SI  - GENBANK/JX188814
SI  - GENBANK/JX188815
SI  - GENBANK/JX188816
SI  - GENBANK/JX188817
SI  - GENBANK/JX188818
SI  - GENBANK/JX188819
SI  - GENBANK/JX188820
SI  - GENBANK/JX188821
SI  - GENBANK/JX188822
SI  - GENBANK/JX188823
SI  - GENBANK/JX188824
SI  - GENBANK/JX188825
SI  - GENBANK/JX188826
SI  - GENBANK/JX188827
SI  - GENBANK/JX188828
SI  - GENBANK/JX188829
SI  - GENBANK/JX188830
SI  - GENBANK/JX188831
SI  - GENBANK/JX188832
SI  - GENBANK/JX188833
SI  - GENBANK/JX188834
SI  - GENBANK/JX188835
SI  - GENBANK/JX188836
SI  - GENBANK/JX188837
SI  - GENBANK/JX188838
SI  - GENBANK/JX188839
SI  - GENBANK/JX188840
SI  - GENBANK/JX188841
SI  - GENBANK/JX188842
SI  - GENBANK/JX188843
SI  - GENBANK/JX188844
SI  - GENBANK/JX188845
SI  - GENBANK/JX188846
SI  - GENBANK/JX188847
SI  - GENBANK/JX188848
SI  - GENBANK/JX188849
SI  - GENBANK/JX188850
SI  - GENBANK/JX188851
SI  - GENBANK/JX188852
SI  - GENBANK/JX188853
SI  - GENBANK/JX188854
SI  - GENBANK/JX188855
SI  - GENBANK/JX188856
SI  - GENBANK/JX188857
SI  - GENBANK/JX188858
SI  - GENBANK/JX188859
SI  - GENBANK/JX188860
SI  - GENBANK/JX188861
SI  - GENBANK/JX188862
SI  - GENBANK/JX188863
SI  - GENBANK/JX188864
SI  - GENBANK/JX188865
SI  - GENBANK/JX188866
SI  - GENBANK/JX188867
SI  - GENBANK/JX188868
SI  - GENBANK/JX188869
SI  - GENBANK/JX188870
SI  - GENBANK/JX188871
SI  - GENBANK/JX188872
SI  - GENBANK/JX188873
SI  - GENBANK/JX188874
SI  - GENBANK/JX188875
SI  - GENBANK/JX188876
SI  - GENBANK/JX188877
SI  - GENBANK/JX188878
SI  - GENBANK/JX188879
SI  - GENBANK/JX188880
SI  - GENBANK/JX188881
SI  - GENBANK/JX188882
SI  - GENBANK/JX188883
SI  - GENBANK/JX188884
SI  - GENBANK/JX188885
SI  - GENBANK/JX188886
SI  - GENBANK/JX188887
SI  - GENBANK/JX188888
SI  - GENBANK/JX188889
SI  - GENBANK/JX188890
SI  - GENBANK/JX188891
SI  - GENBANK/JX188892
SI  - GENBANK/JX188893
SI  - GENBANK/JX188894
SI  - GENBANK/JX188895
SI  - GENBANK/JX188896
SI  - GENBANK/JX188897
SI  - GENBANK/JX188898
SI  - GENBANK/JX188899
SI  - GENBANK/JX188900
SI  - GENBANK/JX188901
SI  - GENBANK/JX188902
SI  - GENBANK/JX188903
SI  - GENBANK/JX188904
SI  - GENBANK/JX188905
SI  - GENBANK/JX188906
SI  - GENBANK/JX188907
SI  - GENBANK/JX188908
SI  - GENBANK/JX188909
SI  - GENBANK/JX188910
SI  - GENBANK/JX188911
SI  - GENBANK/JX188912
SI  - GENBANK/JX188913
SI  - GENBANK/JX188914
SI  - GENBANK/JX188915
SI  - GENBANK/JX188916
SI  - GENBANK/JX188917
SI  - GENBANK/JX188918
SI  - GENBANK/JX188919
SI  - GENBANK/JX188920
SI  - GENBANK/JX188921
SI  - GENBANK/JX188922
SI  - GENBANK/JX188923
SI  - GENBANK/JX188924
SI  - GENBANK/JX188925
SI  - GENBANK/JX188926
SI  - GENBANK/JX188927
SI  - GENBANK/JX188928
SI  - GENBANK/JX188929
SI  - GENBANK/JX188930
SI  - GENBANK/JX188931
SI  - GENBANK/JX188932
SI  - GENBANK/JX188933
SI  - GENBANK/JX188934
SI  - GENBANK/JX188935
SI  - GENBANK/JX188936
SI  - GENBANK/JX188937
SI  - GENBANK/JX188938
SI  - GENBANK/JX188939
SI  - GENBANK/JX188940
SI  - GENBANK/JX188941
SI  - GENBANK/JX188942
SI  - GENBANK/JX188943
SI  - GENBANK/JX188944
SI  - GENBANK/JX188945
SI  - GENBANK/JX188946
SI  - GENBANK/JX188947
SI  - GENBANK/JX188948
SI  - GENBANK/JX188949
SI  - GENBANK/JX188950
SI  - GENBANK/JX188951
SI  - GENBANK/JX188952
SI  - GENBANK/JX188953
SI  - GENBANK/JX188954
SI  - GENBANK/JX188955
SI  - GENBANK/JX188956
SI  - GENBANK/JX188957
SI  - GENBANK/JX188958
SI  - GENBANK/JX188959
SI  - GENBANK/JX188960
SI  - GENBANK/JX188961
SI  - GENBANK/JX188962
SI  - GENBANK/JX188963
SI  - GENBANK/JX188964
SI  - GENBANK/JX188965
SI  - GENBANK/JX188966
SI  - GENBANK/JX188967
SI  - GENBANK/JX188968
SI  - GENBANK/JX188969
SI  - GENBANK/JX188970
SI  - GENBANK/JX188971
SI  - GENBANK/JX188972
SI  - GENBANK/JX188973
SI  - GENBANK/JX188974
SI  - GENBANK/JX188975
SI  - GENBANK/JX188976
SI  - GENBANK/JX188977
SI  - GENBANK/JX188978
SI  - GENBANK/JX188979
SI  - GENBANK/JX188980
SI  - GENBANK/JX188981
SI  - GENBANK/JX188982
SI  - GENBANK/JX188983
SI  - GENBANK/JX188984
SI  - GENBANK/JX188985
SI  - GENBANK/JX188986
SI  - GENBANK/JX188987
SI  - GENBANK/JX188988
SI  - GENBANK/JX188989
SI  - GENBANK/JX188990
SI  - GENBANK/JX188991
SI  - GENBANK/JX188992
SI  - GENBANK/JX188993
SI  - GENBANK/JX188994
SI  - GENBANK/JX188995
SI  - GENBANK/JX188996
SI  - GENBANK/JX188997
SI  - GENBANK/JX188998
SI  - GENBANK/JX188999
SI  - GENBANK/JX189000
SI  - GENBANK/JX189001
SI  - GENBANK/JX189002
SI  - GENBANK/JX189003
SI  - GENBANK/JX189004
SI  - GENBANK/JX189005
SI  - GENBANK/JX189006
SI  - GENBANK/JX189007
SI  - GENBANK/JX189008
SI  - GENBANK/JX189009
SI  - GENBANK/JX189010
SI  - GENBANK/JX189011
SI  - GENBANK/JX189012
SI  - GENBANK/JX189013
SI  - GENBANK/JX189014
SI  - GENBANK/JX189015
SI  - GENBANK/JX189016
SI  - GENBANK/JX189017
SI  - GENBANK/JX189018
SI  - GENBANK/JX189019
SI  - GENBANK/JX189020
SI  - GENBANK/JX189021
SI  - GENBANK/JX189022
SI  - GENBANK/JX189023
SI  - GENBANK/JX189024
SI  - GENBANK/JX189025
SI  - GENBANK/JX189026
SI  - GENBANK/JX189027
SI  - GENBANK/JX189028
SI  - GENBANK/JX189029
SI  - GENBANK/JX189030
SI  - GENBANK/JX189031
SI  - GENBANK/JX189032
SI  - GENBANK/JX189033
SI  - GENBANK/JX189034
SI  - GENBANK/JX189035
SI  - GENBANK/JX189036
SI  - GENBANK/JX189037
SI  - GENBANK/JX189038
SI  - GENBANK/JX189039
SI  - GENBANK/JX189040
SI  - GENBANK/JX189041
SI  - GENBANK/JX189042
SI  - GENBANK/JX189043
SI  - GENBANK/JX189044
SI  - GENBANK/JX189045
SI  - GENBANK/JX189046
SI  - GENBANK/JX189047
SI  - GENBANK/JX189048
SI  - GENBANK/JX189049
SI  - GENBANK/JX189050
SI  - GENBANK/JX189051
SI  - GENBANK/JX189052
SI  - GENBANK/JX189053
SI  - GENBANK/JX189054
SI  - GENBANK/JX189055
SI  - GENBANK/JX189056
SI  - GENBANK/JX189057
SI  - GENBANK/JX189058
SI  - GENBANK/JX189059
SI  - GENBANK/JX189060
SI  - GENBANK/JX189061
SI  - GENBANK/JX189062
SI  - GENBANK/JX189063
SI  - GENBANK/JX189064
SI  - GENBANK/JX189065
SI  - GENBANK/JX189066
SI  - GENBANK/JX189067
SI  - GENBANK/JX189068
SI  - GENBANK/JX189069
SI  - GENBANK/JX189070
SI  - GENBANK/JX189071
SI  - GENBANK/JX189072
SI  - GENBANK/JX189073
SI  - GENBANK/JX189074
SI  - GENBANK/JX189075
SI  - GENBANK/JX189076
SI  - GENBANK/JX189077
SI  - GENBANK/JX189078
SI  - GENBANK/JX189079
SI  - GENBANK/JX189080
SI  - GENBANK/JX189081
SI  - GENBANK/JX189082
SI  - GENBANK/JX189083
SI  - GENBANK/JX189084
SI  - GENBANK/JX189085
SI  - GENBANK/JX189086
SI  - GENBANK/JX189087
SI  - GENBANK/JX189088
SI  - GENBANK/JX189089
SI  - GENBANK/JX189090
SI  - GENBANK/JX189091
SI  - GENBANK/JX189092
SI  - GENBANK/JX189093
SI  - GENBANK/JX189094
SI  - GENBANK/JX189095
SI  - GENBANK/JX189096
SI  - GENBANK/JX189097
SI  - GENBANK/JX189098
SI  - GENBANK/JX189099
SI  - GENBANK/JX189100
SI  - GENBANK/JX189101
SI  - GENBANK/JX189102
SI  - GENBANK/JX189103
SI  - GENBANK/JX189104
SI  - GENBANK/JX189105
SI  - GENBANK/JX189106
SI  - GENBANK/JX189107
SI  - GENBANK/JX189108
SI  - GENBANK/JX189109
SI  - GENBANK/JX189110
SI  - GENBANK/JX189111
SI  - GENBANK/JX189112
SI  - GENBANK/JX189113
SI  - GENBANK/JX189114
SI  - GENBANK/JX189115
SI  - GENBANK/JX189116
SI  - GENBANK/JX189117
SI  - GENBANK/JX189118
SI  - GENBANK/JX189119
SI  - GENBANK/JX189120
SI  - GENBANK/JX189121
SI  - GENBANK/JX189122
SI  - GENBANK/JX189123
SI  - GENBANK/JX189124
SI  - GENBANK/JX189125
SI  - GENBANK/JX189126
SI  - GENBANK/JX189127
SI  - GENBANK/JX189128
SI  - GENBANK/JX189129
SI  - GENBANK/JX189130
SI  - GENBANK/JX189131
SI  - GENBANK/JX189132
SI  - GENBANK/JX189133
SI  - GENBANK/JX189134
SI  - GENBANK/JX189135
SI  - GENBANK/JX189136
SI  - GENBANK/JX189137
SI  - GENBANK/JX189138
SI  - GENBANK/JX189139
SI  - GENBANK/JX189140
SI  - GENBANK/JX189141
SI  - GENBANK/JX189142
SI  - GENBANK/JX189143
SI  - GENBANK/JX189144
SI  - GENBANK/JX189145
SI  - GENBANK/JX189146
SI  - GENBANK/JX189147
SI  - GENBANK/JX189148
SI  - GENBANK/JX189149
SI  - GENBANK/JX189150
SI  - GENBANK/JX189151
SI  - GENBANK/JX189152
SI  - GENBANK/JX189153
SI  - GENBANK/JX189154
SI  - GENBANK/JX189155
SI  - GENBANK/JX189156
SI  - GENBANK/JX189157
SI  - GENBANK/JX189158
SI  - GENBANK/JX189159
SI  - GENBANK/JX189160
SI  - GENBANK/JX189161
SI  - GENBANK/JX189162
SI  - GENBANK/JX189163
SI  - GENBANK/JX189164
SI  - GENBANK/JX189165
SI  - GENBANK/JX189166
SI  - GENBANK/JX189167
SI  - GENBANK/JX189168
SI  - GENBANK/JX189169
SI  - GENBANK/JX189170
SI  - GENBANK/JX189171
SI  - GENBANK/JX189172
SI  - GENBANK/JX189173
SI  - GENBANK/JX189174
SI  - GENBANK/JX189175
SI  - GENBANK/JX189176
SI  - GENBANK/JX189177
SI  - GENBANK/JX189178
SI  - GENBANK/JX189179
SI  - GENBANK/JX189180
SI  - GENBANK/JX189181
SI  - GENBANK/JX189182
SI  - GENBANK/JX189183
SI  - GENBANK/JX189184
SI  - GENBANK/JX189185
SI  - GENBANK/JX189186
SI  - GENBANK/JX189187
SI  - GENBANK/JX189188
SI  - GENBANK/JX189189
SI  - GENBANK/JX189190
SI  - GENBANK/JX189191
SI  - GENBANK/JX189192
SI  - GENBANK/JX189193
SI  - GENBANK/JX189194
SI  - GENBANK/JX189195
SI  - GENBANK/JX189196
SI  - GENBANK/JX189197
SI  - GENBANK/JX189198
SI  - GENBANK/JX189199
SI  - GENBANK/JX189200
SI  - GENBANK/JX189201
SI  - GENBANK/JX189202
SI  - GENBANK/JX189203
SI  - GENBANK/JX189204
SI  - GENBANK/JX189205
SI  - GENBANK/JX189206
SI  - GENBANK/JX189207
SI  - GENBANK/JX189208
SI  - GENBANK/JX189209
SI  - GENBANK/JX189210
SI  - GENBANK/JX189211
SI  - GENBANK/JX189212
SI  - GENBANK/JX189213
SI  - GENBANK/JX189214
SI  - GENBANK/JX189215
SI  - GENBANK/JX189216
SI  - GENBANK/JX189217
SI  - GENBANK/JX189218
SI  - GENBANK/JX189219
SI  - GENBANK/JX189220
SI  - GENBANK/JX189221
SI  - GENBANK/JX189222
SI  - GENBANK/JX189223
SI  - GENBANK/JX189224
SI  - GENBANK/JX189225
SI  - GENBANK/JX189226
SI  - GENBANK/JX189227
SI  - GENBANK/JX189228
SI  - GENBANK/JX189229
SI  - GENBANK/JX189230
SI  - GENBANK/JX189231
SI  - GENBANK/JX189232
SI  - GENBANK/JX189233
SI  - GENBANK/JX189234
SI  - GENBANK/JX189235
SI  - GENBANK/JX189236
SI  - GENBANK/JX189237
SI  - GENBANK/JX189238
SI  - GENBANK/JX189239
SI  - GENBANK/JX189240
SI  - GENBANK/JX189241
SI  - GENBANK/JX189242
SI  - GENBANK/JX189243
SI  - GENBANK/JX189244
SI  - GENBANK/JX189245
SI  - GENBANK/JX189246
SI  - GENBANK/JX189247
SI  - GENBANK/JX189248
SI  - GENBANK/JX189249
SI  - GENBANK/JX189250
SI  - GENBANK/JX189251
SI  - GENBANK/JX189252
SI  - GENBANK/JX189253
SI  - GENBANK/JX189254
SI  - GENBANK/JX189255
SI  - GENBANK/JX189256
SI  - GENBANK/JX189257
SI  - GENBANK/JX189258
SI  - GENBANK/JX189259
SI  - GENBANK/JX189260
SI  - GENBANK/JX189261
SI  - GENBANK/JX189262
SI  - GENBANK/JX189263
SI  - GENBANK/JX189264
SI  - GENBANK/JX189265
SI  - GENBANK/JX189266
SI  - GENBANK/JX189267
SI  - GENBANK/JX189268
SI  - GENBANK/JX189269
SI  - GENBANK/JX189270
SI  - GENBANK/JX189271
SI  - GENBANK/JX189272
SI  - GENBANK/JX189273
SI  - GENBANK/JX189274
SI  - GENBANK/JX189275
SI  - GENBANK/JX189276
SI  - GENBANK/JX189277
SI  - GENBANK/JX189278
SI  - GENBANK/JX189279
SI  - GENBANK/JX189280
SI  - GENBANK/JX189281
SI  - GENBANK/JX189282
SI  - GENBANK/JX189283
SI  - GENBANK/JX189284
SI  - GENBANK/JX189285
SI  - GENBANK/JX189286
SI  - GENBANK/JX189287
SI  - GENBANK/JX189288
SI  - GENBANK/JX189289
SI  - GENBANK/JX189290
SI  - GENBANK/JX189291
SI  - GENBANK/JX189292
SI  - GENBANK/JX189293
SI  - GENBANK/JX189294
SI  - GENBANK/JX189295
SI  - GENBANK/JX189296
SI  - GENBANK/JX189297
SI  - GENBANK/JX189298
SI  - GENBANK/JX189299
SI  - GENBANK/JX189300
SI  - GENBANK/JX189301
SI  - GENBANK/JX189302
SI  - GENBANK/JX189303
SI  - GENBANK/JX189304
SI  - GENBANK/JX189305
SI  - GENBANK/JX189306
SI  - GENBANK/JX189307
SI  - GENBANK/JX189308
SI  - GENBANK/JX189309
SI  - GENBANK/JX189310
SI  - GENBANK/JX189311
SI  - GENBANK/JX189312
SI  - GENBANK/JX189313
SI  - GENBANK/JX189314
SI  - GENBANK/JX189315
SI  - GENBANK/JX189316
SI  - GENBANK/JX189317
SI  - GENBANK/JX189318
SI  - GENBANK/JX189319
SI  - GENBANK/JX189320
SI  - GENBANK/JX189321
SI  - GENBANK/JX189322
SI  - GENBANK/JX189323
SI  - GENBANK/JX189324
SI  - GENBANK/JX189325
SI  - GENBANK/JX189326
SI  - GENBANK/JX189327
SI  - GENBANK/JX189328
SI  - GENBANK/JX189329
SI  - GENBANK/JX189330
SI  - GENBANK/JX189331
SI  - GENBANK/JX189332
SI  - GENBANK/JX189333
SI  - GENBANK/JX189334
SI  - GENBANK/JX189335
SI  - GENBANK/JX189336
SI  - GENBANK/JX189337
SI  - GENBANK/JX189338
SI  - GENBANK/JX189339
SI  - GENBANK/JX189340
SI  - GENBANK/JX189341
SI  - GENBANK/JX189342
SI  - GENBANK/JX189343
SI  - GENBANK/JX189344
SI  - GENBANK/JX189345
SI  - GENBANK/JX189346
SI  - GENBANK/JX189347
SI  - GENBANK/JX189348
SI  - GENBANK/JX189349
SI  - GENBANK/JX189350
SI  - GENBANK/JX189351
SI  - GENBANK/JX189352
SI  - GENBANK/JX189353
SI  - GENBANK/JX189354
SI  - GENBANK/JX189355
SI  - GENBANK/JX189356
SI  - GENBANK/JX189357
SI  - GENBANK/JX189358
SI  - GENBANK/JX189359
SI  - GENBANK/JX189360
SI  - GENBANK/JX189361
SI  - GENBANK/JX189362
SI  - GENBANK/JX189363
SI  - GENBANK/JX189364
SI  - GENBANK/JX189365
SI  - GENBANK/JX189366
SI  - GENBANK/JX189367
SI  - GENBANK/JX189368
SI  - GENBANK/JX189369
SI  - GENBANK/JX189370
SI  - GENBANK/JX189371
SI  - GENBANK/JX189372
SI  - GENBANK/JX189373
SI  - GENBANK/JX189374
SI  - GENBANK/JX189375
SI  - GENBANK/JX189376
SI  - GENBANK/JX189377
SI  - GENBANK/JX189378
SI  - GENBANK/JX189379
SI  - GENBANK/JX189380
SI  - GENBANK/JX189381
SI  - GENBANK/JX189382
SI  - GENBANK/JX189383
SI  - GENBANK/JX189384
SI  - GENBANK/JX189385
SI  - GENBANK/JX189386
SI  - GENBANK/JX189387
SI  - GENBANK/JX189388
SI  - GENBANK/JX189389
SI  - GENBANK/JX189390
SI  - GENBANK/JX189391
SI  - GENBANK/JX189392
SI  - GENBANK/JX189393
SI  - GENBANK/JX189394
SI  - GENBANK/JX189395
SI  - GENBANK/JX189396
SI  - GENBANK/JX189397
SI  - GENBANK/JX189398
SI  - GENBANK/JX189399
SI  - GENBANK/JX189400
SI  - GENBANK/JX189401
SI  - GENBANK/JX189402
SI  - GENBANK/JX189403
SI  - GENBANK/JX189404
SI  - GENBANK/JX189405
SI  - GENBANK/JX189406
SI  - GENBANK/JX189407
SI  - GENBANK/JX189408
SI  - GENBANK/JX189409
SI  - GENBANK/JX189410
SI  - GENBANK/JX189411
SI  - GENBANK/JX189412
SI  - GENBANK/JX189413
SI  - GENBANK/JX189414
SI  - GENBANK/JX189415
SI  - GENBANK/JX189416
SI  - GENBANK/JX189417
SI  - GENBANK/JX189418
SI  - GENBANK/JX189419
SI  - GENBANK/JX189420
SI  - GENBANK/JX189421
SI  - GENBANK/JX189422
SI  - GENBANK/JX189423
SI  - GENBANK/JX189424
SI  - GENBANK/JX189425
SI  - GENBANK/JX189426
SI  - GENBANK/JX189427
SI  - GENBANK/JX189428
SI  - GENBANK/JX189429
SI  - GENBANK/JX189430
SI  - GENBANK/JX189431
SI  - GENBANK/JX189432
SI  - GENBANK/JX189433
SI  - GENBANK/JX189434
SI  - GENBANK/JX189435
SI  - GENBANK/JX189436
SI  - GENBANK/JX189437
SI  - GENBANK/JX189438
SI  - GENBANK/JX189439
SI  - GENBANK/JX189440
SI  - GENBANK/JX189441
SI  - GENBANK/JX189442
SI  - GENBANK/JX189443
SI  - GENBANK/JX189444
SI  - GENBANK/JX189445
SI  - GENBANK/JX189446
SI  - GENBANK/JX189447
SI  - GENBANK/JX189448
SI  - GENBANK/JX189449
SI  - GENBANK/JX189450
SI  - GENBANK/JX189451
SI  - GENBANK/JX189452
SI  - GENBANK/JX189453
SI  - GENBANK/JX189454
SI  - GENBANK/JX189455
SI  - GENBANK/JX189456
SI  - GENBANK/JX189457
SI  - GENBANK/JX189458
SI  - GENBANK/JX189459
SI  - GENBANK/JX189460
SI  - GENBANK/JX189461
SI  - GENBANK/JX189462
SI  - GENBANK/JX189463
SI  - GENBANK/JX189464
SI  - GENBANK/JX189465
SI  - GENBANK/JX189466
SI  - GENBANK/JX189467
SI  - GENBANK/JX189468
SI  - GENBANK/JX189469
SI  - GENBANK/JX189470
SI  - GENBANK/JX189471
SI  - GENBANK/JX189472
SI  - GENBANK/JX189473
SI  - GENBANK/JX189474
SI  - GENBANK/JX189475
SI  - GENBANK/JX189476
SI  - GENBANK/JX189477
SI  - GENBANK/JX189478
SI  - GENBANK/JX189479
SI  - GENBANK/JX189480
SI  - GENBANK/JX189481
SI  - GENBANK/JX189482
SI  - GENBANK/JX189483
SI  - GENBANK/JX189484
SI  - GENBANK/JX189485
SI  - GENBANK/JX189486
SI  - GENBANK/JX189487
SI  - GENBANK/JX189488
SI  - GENBANK/JX189489
SI  - GENBANK/JX189490
SI  - GENBANK/JX189491
SI  - GENBANK/JX189492
SI  - GENBANK/JX189493
SI  - GENBANK/JX189494
SI  - GENBANK/JX189495
SI  - GENBANK/JX189496
SI  - GENBANK/JX189497
SI  - GENBANK/JX189498
SI  - GENBANK/JX189499
SI  - GENBANK/JX189500
SI  - GENBANK/JX189501
SI  - GENBANK/JX189502
SI  - GENBANK/JX189503
SI  - GENBANK/JX189504
SI  - GENBANK/JX189505
SI  - GENBANK/JX189506
SI  - GENBANK/JX189507
SI  - GENBANK/JX189508
SI  - GENBANK/JX189509
SI  - GENBANK/JX189510
SI  - GENBANK/JX189511
SI  - GENBANK/JX189512
SI  - GENBANK/JX189513
SI  - GENBANK/JX189514
SI  - GENBANK/JX189515
SI  - GENBANK/JX189516
SI  - GENBANK/JX189517
SI  - GENBANK/JX189518
SI  - GENBANK/JX189519
SI  - GENBANK/JX189520
SI  - GENBANK/JX189521
SI  - GENBANK/JX189522
SI  - GENBANK/JX189523
SI  - GENBANK/JX189524
SI  - GENBANK/JX189525
SI  - GENBANK/JX189526
SI  - GENBANK/JX189527
SI  - GENBANK/JX189528
SI  - GENBANK/JX189529
SI  - GENBANK/JX189530
SI  - GENBANK/JX189531
SI  - GENBANK/JX189532
SI  - GENBANK/JX189533
SI  - GENBANK/JX189534
SI  - GENBANK/JX189535
SI  - GENBANK/JX189536
SI  - GENBANK/JX189537
SI  - GENBANK/JX189538
SI  - GENBANK/JX189539
SI  - GENBANK/JX189540
SI  - GENBANK/JX189541
SI  - GENBANK/JX189542
SI  - GENBANK/JX189543
SI  - GENBANK/JX189544
SI  - GENBANK/JX189545
SI  - GENBANK/JX189546
SI  - GENBANK/JX189547
SI  - GENBANK/JX189548
SI  - GENBANK/JX189549
SI  - GENBANK/JX189550
SI  - GENBANK/JX189551
SI  - GENBANK/JX189552
SI  - GENBANK/JX189553
SI  - GENBANK/JX189554
SI  - GENBANK/JX189555
SI  - GENBANK/JX189556
SI  - GENBANK/JX189557
SI  - GENBANK/JX189558
SI  - GENBANK/JX189559
SI  - GENBANK/JX189560
SI  - GENBANK/JX189561
SI  - GENBANK/JX189562
SI  - GENBANK/JX189563
SI  - GENBANK/JX189564
SI  - GENBANK/JX189565
SI  - GENBANK/JX189566
SI  - GENBANK/JX189567
SI  - GENBANK/JX189568
SI  - GENBANK/JX189569
SI  - GENBANK/JX189570
SI  - GENBANK/JX189571
SI  - GENBANK/JX189572
SI  - GENBANK/JX189573
SI  - GENBANK/JX189574
SI  - GENBANK/JX189575
SI  - GENBANK/JX189576
SI  - GENBANK/JX189577
SI  - GENBANK/JX189578
SI  - GENBANK/JX189579
SI  - GENBANK/JX189580
SI  - GENBANK/JX189581
SI  - GENBANK/JX189582
SI  - GENBANK/JX189583
SI  - GENBANK/JX189584
SI  - GENBANK/JX189585
SI  - GENBANK/JX189586
SI  - GENBANK/JX189587
SI  - GENBANK/JX189588
SI  - GENBANK/JX189589
SI  - GENBANK/JX189590
SI  - GENBANK/JX189591
SI  - GENBANK/JX189592
SI  - GENBANK/JX189593
SI  - GENBANK/JX189594
SI  - GENBANK/JX189595
SI  - GENBANK/JX189596
SI  - GENBANK/JX189597
SI  - GENBANK/JX189598
SI  - GENBANK/JX189599
SI  - GENBANK/JX189600
SI  - GENBANK/JX189601
SI  - GENBANK/JX189602
SI  - GENBANK/JX189603
SI  - GENBANK/JX189604
SI  - GENBANK/JX189605
SI  - GENBANK/JX189606
SI  - GENBANK/JX189607
SI  - GENBANK/JX189608
SI  - GENBANK/JX189609
SI  - GENBANK/JX189610
SI  - GENBANK/JX189611
SI  - GENBANK/JX189612
SI  - GENBANK/JX189613
SI  - GENBANK/JX189614
SI  - GENBANK/JX189615
SI  - GENBANK/JX189616
SI  - GENBANK/JX189617
SI  - GENBANK/JX189618
SI  - GENBANK/JX189619
SI  - GENBANK/JX189620
SI  - GENBANK/JX189621
SI  - GENBANK/JX189622
SI  - GENBANK/JX189623
SI  - GENBANK/JX189624
SI  - GENBANK/JX189625
SI  - GENBANK/JX189626
SI  - GENBANK/JX189627
SI  - GENBANK/JX189628
SI  - GENBANK/JX189629
SI  - GENBANK/JX189630
SI  - GENBANK/JX189631
SI  - GENBANK/JX189632
SI  - GENBANK/JX189633
SI  - GENBANK/JX189634
SI  - GENBANK/JX189635
SI  - GENBANK/JX189636
SI  - GENBANK/JX189637
SI  - GENBANK/JX189638
SI  - GENBANK/JX189639
SI  - GENBANK/JX189640
SI  - GENBANK/JX189641
SI  - GENBANK/JX189642
SI  - GENBANK/JX189643
SI  - GENBANK/JX189644
SI  - GENBANK/JX189645
SI  - GENBANK/JX189646
SI  - GENBANK/JX189647
SI  - GENBANK/JX189648
SI  - GENBANK/JX189649
SI  - GENBANK/JX189650
SI  - GENBANK/JX189651
SI  - GENBANK/JX189652
SI  - GENBANK/JX189653
SI  - GENBANK/JX189654
SI  - GENBANK/JX189655
SI  - GENBANK/JX189656
SI  - GENBANK/JX189657
SI  - GENBANK/JX189658
SI  - GENBANK/JX189659
SI  - GENBANK/JX189660
SI  - GENBANK/JX189661
SI  - GENBANK/JX189662
SI  - GENBANK/JX189663
SI  - GENBANK/JX189664
SI  - GENBANK/JX189665
SI  - GENBANK/JX189666
SI  - GENBANK/JX189667
SI  - GENBANK/JX189668
SI  - GENBANK/JX189669
SI  - GENBANK/JX189670
SI  - GENBANK/JX189671
SI  - GENBANK/JX189672
SI  - GENBANK/JX189673
SI  - GENBANK/JX189674
SI  - GENBANK/JX189675
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120627
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Jaw/*anatomy &amp; histology
MH  - Molecular Sequence Data
MH  - *Perciformes/anatomy &amp; histology/classification/genetics
MH  - *Phylogeny
EDAT- 2012/06/30 06:00
MHDA- 2013/03/06 06:00
CRDT- 2012/06/30 06:00
PHST- 2012/06/30 06:00 [entrez]
PHST- 2012/06/30 06:00 [pubmed]
PHST- 2013/03/06 06:00 [medline]
AID - sys060 [pii]
AID - 10.1093/sysbio/sys060 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Dec 1;61(6):1001-27. doi: 10.1093/sysbio/sys060. Epub 2012 Jun
      27.

PMID- 22730418
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - The information content of taxon names: a reply to de Queiroz and Donoghue.
PG  - 175-6
LID - 10.1093/sysbio/sys059 [doi]
FAU - Platnick, Norman I
AU  - Platnick NI
LA  - eng
PT  - Letter
PT  - Comment
DEP - 20120622
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2012 Mar;61(2):360-1. PMID: 22201160
CON - Syst Biol. 2013 Jan 1;62(1):167-74. PMID: 22649180
MH  - Animals
MH  - Classification/*methods
MH  - *Phylogeny
MH  - *Terminology as Topic
EDAT- 2012/06/26 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/06/26 06:00
PHST- 2012/06/26 06:00 [entrez]
PHST- 2012/06/26 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys059 [pii]
AID - 10.1093/sysbio/sys059 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):175-6. doi: 10.1093/sysbio/sys059. Epub 2012 Jun 22.

PMID- 22723471
OWN - NLM
STAT- MEDLINE
DCOM- 20130305
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 6
DP  - 2012 Dec 1
TI  - A total-evidence approach to dating with fossils, applied to the early radiation 
      of the hymenoptera.
PG  - 973-99
LID - 10.1093/sysbio/sys058 [doi]
AB  - Phylogenies are usually dated by calibrating interior nodes against the fossil
      record. This relies on indirect methods that, in the worst case, misrepresent the
      fossil information. Here, we contrast such node dating with an approach that
      includes fossils along with the extant taxa in a Bayesian total-evidence
      analysis. As a test case, we focus on the early radiation of the Hymenoptera,
      mostly documented by poorly preserved impression fossils that are difficult to
      place phylogenetically. Specifically, we compare node dating using nine
      calibration points derived from the fossil record with total-evidence dating
      based on 343 morphological characters scored for 45 fossil (4--20 complete) and
      68 extant taxa. In both cases we use molecular data from seven markers (
      approximately 5 kb) for the extant taxa. Because it is difficult to model
      speciation, extinction, sampling, and fossil preservation realistically, we
      develop a simple uniform prior for clock trees with fossils, and we use relaxed
      clock models to accommodate rate variation across the tree. Despite considerable 
      uncertainty in the placement of most fossils, we find that they contribute
      significantly to the estimation of divergence times in the total-evidence
      analysis. In particular, the posterior distributions on divergence times are less
      sensitive to prior assumptions and tend to be more precise than in node dating.
      The total-evidence analysis also shows that four of the seven Hymenoptera
      calibration points used in node dating are likely to be based on erroneous or
      doubtful assumptions about the fossil placement. With respect to the early
      radiation of Hymenoptera, our results suggest that the crown group dates back to 
      the Carboniferous, approximately 309 Ma (95% interval: 291--347 Ma), and
      diversified into major extant lineages much earlier than previously thought, well
      before the Triassic. [Bayesian inference; fossil dating; morphological evolution;
      relaxed clock; statistical phylogenetics.].
FAU - Ronquist, Fredrik
AU  - Ronquist F
AD  - Department of Biodiversity Informatics, Swedish Museum of Natural History, Box
      50007, SE-104 05 Stockholm, Sweden. fredrik.ronquist@nrm.se
FAU - Klopfstein, Seraina
AU  - Klopfstein S
FAU - Vilhelmsen, Lars
AU  - Vilhelmsen L
FAU - Schulmeister, Susanne
AU  - Schulmeister S
FAU - Murray, Debra L
AU  - Murray DL
FAU - Rasnitsyn, Alexandr P
AU  - Rasnitsyn AP
LA  - eng
PT  - Journal Article
DEP - 20120620
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Fossils
MH  - Genetic Speciation
MH  - Hymenoptera/anatomy &amp; histology/*classification/genetics
MH  - Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Statistics as Topic
MH  - Time Factors
PMC - PMC3478566
EDAT- 2012/06/23 06:00
MHDA- 2013/03/06 06:00
CRDT- 2012/06/23 06:00
PHST- 2012/06/23 06:00 [entrez]
PHST- 2012/06/23 06:00 [pubmed]
PHST- 2013/03/06 06:00 [medline]
AID - sys058 [pii]
AID - 10.1093/sysbio/sys058 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Dec 1;61(6):973-99. doi: 10.1093/sysbio/sys058. Epub 2012 Jun 20.

PMID- 22649181
OWN - NLM
STAT- MEDLINE
DCOM- 20130305
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 6
DP  - 2012 Dec 1
TI  - Assessing parameter identifiability in phylogenetic models using data cloning.
PG  - 955-72
LID - 10.1093/sysbio/sys055 [doi]
AB  - The success of model-based methods in phylogenetics has motivated much research
      aimed at generating new, biologically informative models. This new
      computer-intensive approach to phylogenetics demands validation studies and sound
      measures of performance. To date there has been little practical guidance
      available as to when and why the parameters in a particular model can be
      identified reliably. Here, we illustrate how Data Cloning (DC), a recently
      developed methodology to compute the maximum likelihood estimates along with
      their asymptotic variance, can be used to diagnose structural parameter
      nonidentifiability (NI) and distinguish it from other parameter estimability
      problems, including when parameters are structurally identifiable, but are not
      estimable in a given data set (INE), and when parameters are identifiable, and
      estimable, but only weakly so (WE). The application of the DC theorem uses
      well-known and widely used Bayesian computational techniques. With the DC
      approach, practitioners can use Bayesian phylogenetics software to diagnose
      nonidentifiability. Theoreticians and practitioners alike now have a powerful,
      yet simple tool to detect nonidentifiability while investigating complex modeling
      scenarios, where getting closed-form expressions in a probabilistic study is
      complicated. Furthermore, here we also show how DC can be used as a tool to
      examine and eliminate the influence of the priors, in particular if the process
      of prior elicitation is not straightforward. Finally, when applied to
      phylogenetic inference, DC can be used to study at least two important
      statistical questions: assessing identifiability of discrete parameters, like the
      tree topology, and developing efficient sampling methods for computationally
      expensive posterior densities.
FAU - Ponciano, Jose Miguel
AU  - Ponciano JM
AD  - Department of Biology, University of Florida, Gainesville, FL 32611, USA.
      josemi@ufl.edu
FAU - Burleigh, J Gordon
AU  - Burleigh JG
FAU - Braun, Edward L
AU  - Braun EL
FAU - Taper, Mark L
AU  - Taper ML
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120530
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Chloroplasts/genetics
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - *Models, Statistical
MH  - *Phylogeny
MH  - Plants/classification/genetics
PMC - PMC3478565
EDAT- 2012/06/01 06:00
MHDA- 2013/03/06 06:00
CRDT- 2012/06/01 06:00
PHST- 2012/06/01 06:00 [entrez]
PHST- 2012/06/01 06:00 [pubmed]
PHST- 2013/03/06 06:00 [medline]
AID - sys055 [pii]
AID - 10.1093/sysbio/sys055 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Dec 1;61(6):955-72. doi: 10.1093/sysbio/sys055. Epub 2012 May 30.

PMID- 22649180
OWN - NLM
STAT- MEDLINE
DCOM- 20130307
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 62
IP  - 1
DP  - 2013 Jan 1
TI  - Phylogenetic nomenclature, hierarchical information, and testability.
PG  - 167-74
LID - 10.1093/sysbio/sys054 [doi]
FAU - de Queiroz, Kevin
AU  - de Queiroz K
AD  - Department of Vertebrate Zoology, National Museum of Natural History, Smithsonian
      Institution, Washington, DC 20560-0162, USA. dequeirozk@si.edu
FAU - Donoghue, Michael J
AU  - Donoghue MJ
LA  - eng
PT  - Journal Article
PT  - Comment
DEP - 20120530
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2012 Mar;61(2):360-1. PMID: 22201160
CIN - Syst Biol. 2013 Jan 1;62(1):175-6. PMID: 22730418
MH  - Animals
MH  - Classification/*methods
MH  - *Phylogeny
MH  - *Terminology as Topic
EDAT- 2012/06/01 06:00
MHDA- 2013/03/08 06:00
CRDT- 2012/06/01 06:00
PHST- 2012/06/01 06:00 [entrez]
PHST- 2012/06/01 06:00 [pubmed]
PHST- 2013/03/08 06:00 [medline]
AID - sys054 [pii]
AID - 10.1093/sysbio/sys054 [doi]
PST - ppublish
SO  - Syst Biol. 2013 Jan 1;62(1):167-74. doi: 10.1093/sysbio/sys054. Epub 2012 May 30.

PMID- 22610088
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20120813
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - High-throughput identification of informative nuclear loci for shallow-scale
      phylogenetics and phylogeography.
PG  - 745-61
LID - 10.1093/sysbio/sys051 [doi]
AB  - One of the major challenges for researchers studying phylogeography and
      shallow-scale phylogenetics is the identification of highly variable and
      informative nuclear loci for the question of interest. Previous approaches to
      locus identification have generally required extensive testing of anonymous
      nuclear loci developed from genomic libraries of the target taxon, testing of
      loci of unknown utility from other systems, or identification of loci from the
      nearest model organism with genomic resources. Here, we present a fast and
      economical approach to generating thousands of variable, single-copy nuclear loci
      for any system using next-generation sequencing. We performed Illumina paired-end
      sequencing of three reduced-representation libraries (RRLs) in chorus frogs
      (Pseudacris) to identify orthologous, single-copy loci across libraries and to
      estimate sequence divergence at multiple taxonomic levels. We also conducted PCR 
      testing of these loci across the genus Pseudacris and outgroups to determine
      whether loci developed for phylogeography can be extended to deeper phylogenetic 
      levels. Prior to sequencing, we conducted in silico digestion of the most closely
      related reference genome (Xenopus tropicalis) to generate expectations for the
      number of loci and degree of coverage for a particular experimental design. Using
      the RRL approach, we: (i) identified more than 100,000 single-copy nuclear loci, 
      6339 of which were obtained for divergent conspecifics and 904 of which were
      obtained for heterospecifics; (ii) estimated average nuclear sequence divergence 
      at 0.1% between alleles within an individual, 1.1% between conspecific
      individuals that represent two different clades, and 1.8% between species; and
      (iii) determined from PCR testing that 53% of the loci successfully amplify
      within-species and also many amplify to the genus-level and deeper in the
      phylogeny (16%). Our study effectively identified nuclear loci present in the
      genome that have levels of sequence divergence on par with mitochondrial loci
      commonly used in phylogeography. Specifically, we estimated that ~7% of loci in
      the chorus frog genome are &gt;3% divergent within species; this translates to a
      prediction of approximately 50,000 single-copy loci in the genome with &gt;3%
      divergence. Moreover, successful amplification of many loci at deeper
      phylogenetic levels indicates that the RRL approach represents an efficient
      method for rapid identification of informative loci for both phylogenetics and
      phylogeography. We conclude by making recommendations for minimizing the cost and
      maximizing the efficiency of locus identification for future studies in this
      field.
FAU - Lemmon, Alan R
AU  - Lemmon AR
AD  - Department of Scientific Computing, Florida State University, 400 Dirac Science
      Library, Tallahassee, FL, 32306-4102, USA. alemmon@fsu.edu
FAU - Lemmon, Emily Moriarty
AU  - Lemmon EM
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120518
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Anura/classification/*genetics
MH  - Cell Nucleus/genetics
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - High-Throughput Nucleotide Sequencing/economics/*methods
MH  - *Phylogeny
MH  - Phylogeography/*methods
MH  - Polymerase Chain Reaction
MH  - Sequence Analysis, DNA/economics/*methods
EDAT- 2012/05/23 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/05/22 06:00
PHST- 2012/05/22 06:00 [entrez]
PHST- 2012/05/23 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys051 [pii]
AID - 10.1093/sysbio/sys051 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):745-61. doi: 10.1093/sysbio/sys051. Epub 2012 May 18.

PMID- 22605266
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20120813
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - Anchored hybrid enrichment for massively high-throughput phylogenomics.
PG  - 727-44
LID - 10.1093/sysbio/sys049 [doi]
AB  - The field of phylogenetics is on the cusp of a major revolution, enabled by new
      methods of data collection that leverage both genomic resources and recent
      advances in DNA sequencing. Previous phylogenetic work has required
      labor-intensive marker development coupled with single-locus polymerase chain
      reaction and DNA sequencing on clade-by-clade and locus-by-locus basis. Here, we 
      present a new, cost-efficient, and rapid approach to obtaining data from hundreds
      of loci for potentially hundreds of individuals for deep and shallow phylogenetic
      studies. Specifically, we designed probes for target enrichment of &gt;500 loci in
      highly conserved anchor regions of vertebrate genomes (flanked by less conserved 
      regions) from five model species and tested enrichment efficiency in nonmodel
      species up to 508 million years divergent from the nearest model. We found that
      hybrid enrichment using conserved probes (anchored enrichment) can recover a
      large number of unlinked loci that are useful at a diversity of phylogenetic
      timescales. This new approach has the potential not only to expedite resolution
      of deep-scale portions of the Tree of Life but also to greatly accelerate
      resolution of the large number of shallow clades that remain unresolved. The
      combination of low cost (~1% of the cost of traditional Sanger sequencing and
      ~3.5% of the cost of high-throughput amplicon sequencing for projects on the
      scale of 500 loci x 100 individuals) and rapid data collection (~2 weeks of
      laboratory time) are expected to make this approach tractable even for
      researchers working on systems with limited or nonexistent genomic resources.
FAU - Lemmon, Alan R
AU  - Lemmon AR
AD  - Department of Scientific Computing, Florida State University, Dirac Science
      Library, Tallahassee, FL 32306-4102, USA. alemmon@fsu.edu
FAU - Emme, Sandra A
AU  - Emme SA
FAU - Lemmon, Emily Moriarty
AU  - Lemmon EM
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120517
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Probes)
SB  - IM
MH  - Animals
MH  - DNA Probes/*genetics
MH  - Genome
MH  - High-Throughput Nucleotide Sequencing/economics/*methods
MH  - Humans
MH  - *Phylogeny
MH  - Sequence Analysis, DNA/economics/*methods
MH  - Vertebrates/classification/*genetics
EDAT- 2012/05/19 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/05/19 06:00
PHST- 2012/05/19 06:00 [entrez]
PHST- 2012/05/19 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys049 [pii]
AID - 10.1093/sysbio/sys049 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):727-44. doi: 10.1093/sysbio/sys049. Epub 2012 May 17.

PMID- 22556200
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20120813
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - Deep phylogeographic structure and environmental differentiation in the
      carnivorous plant Sarracenia alata.
PG  - 763-77
LID - 10.1093/sysbio/sys048 [doi]
AB  - We collected ~29 kb of sequence data using Roche 454 pyrosequencing in order to
      estimate the timing and pattern of diversification in the carnivorous pitcher
      plant Sarracenia alata. Utilizing modified protocols for reduced representation
      library construction, we generated sequence data from 86 individuals across 10
      populations from throughout the range of the species. We identified 76
      high-quality and high-coverage loci (containing over 500 SNPs) using the
      bioinformatics pipeline PRGmatic. Results from a Bayesian clustering analysis
      indicate that populations are highly structured, and are similar in pattern to
      the topology of a population tree estimated using *BEAST. The pattern of
      diversification within Sarracenia alata implies that riverine barriers are the
      primary factor promoting population diversification, with divergence across the
      Mississippi River occurring more than 60,000 generations before present. Further,
      significant patterns of niche divergence and the identification of several
      outlier loci suggest that selection may contribute to population divergence. Our 
      results demonstrate the feasibility of using next-generation sequencing to
      investigate intraspecific genetic variation in nonmodel species.
FAU - Zellmer, Amanda J
AU  - Zellmer AJ
AD  - Department of Biological Sciences, Louisiana State University, Baton Rouge, LA
      70803, USA.
FAU - Hanes, Margaret M
AU  - Hanes MM
FAU - Hird, Sarah M
AU  - Hird SM
FAU - Carstens, Bryan C
AU  - Carstens BC
LA  - eng
SI  - GENBANK/JN665096
SI  - GENBANK/JN665097
SI  - GENBANK/JN665098
SI  - GENBANK/JN665099
SI  - GENBANK/JN665100
SI  - GENBANK/JN665101
SI  - GENBANK/JN665102
SI  - GENBANK/JN665103
SI  - GENBANK/JN665104
SI  - GENBANK/JN665105
SI  - GENBANK/JN665106
SI  - GENBANK/JN665107
SI  - GENBANK/JN665108
SI  - GENBANK/JN665109
SI  - GENBANK/JN665110
SI  - GENBANK/JN665111
SI  - GENBANK/JN665112
SI  - GENBANK/JN665113
SI  - GENBANK/JN665114
SI  - GENBANK/JN665115
SI  - GENBANK/JN665116
SI  - GENBANK/JN665117
SI  - GENBANK/JN665118
SI  - GENBANK/JN665119
SI  - GENBANK/JN665120
SI  - GENBANK/JN665121
SI  - GENBANK/JN665122
SI  - GENBANK/JN665123
SI  - GENBANK/JN665124
SI  - GENBANK/JN665125
SI  - GENBANK/JN665126
SI  - GENBANK/JN665127
SI  - GENBANK/JN665128
SI  - GENBANK/JN665129
SI  - GENBANK/JN665130
SI  - GENBANK/JN665131
SI  - GENBANK/JN665132
SI  - GENBANK/JN665133
SI  - GENBANK/JN665134
SI  - GENBANK/JN665135
SI  - GENBANK/JN665136
SI  - GENBANK/JN665137
SI  - GENBANK/JN665138
SI  - GENBANK/JN665139
SI  - GENBANK/JN665140
SI  - GENBANK/JN665141
SI  - GENBANK/JN665142
SI  - GENBANK/JN665143
SI  - GENBANK/JN665144
SI  - GENBANK/JN665145
SI  - GENBANK/JN665146
SI  - GENBANK/JN665147
SI  - GENBANK/JN665148
SI  - GENBANK/JN665149
SI  - GENBANK/JN665150
SI  - GENBANK/JN665151
SI  - GENBANK/JN665152
SI  - GENBANK/JN665153
SI  - GENBANK/JN665154
SI  - GENBANK/JN665155
SI  - GENBANK/JN665156
SI  - GENBANK/JN665157
SI  - GENBANK/JN665158
SI  - GENBANK/JN665159
SI  - GENBANK/JN665160
SI  - GENBANK/JN665161
SI  - GENBANK/JN665162
SI  - GENBANK/JN665163
SI  - GENBANK/JN665164
SI  - GENBANK/JN665165
SI  - GENBANK/JN665166
SI  - GENBANK/JN665167
SI  - GENBANK/JN665168
SI  - GENBANK/JN665169
SI  - GENBANK/JN665170
SI  - GENBANK/JN665171
SI  - GENBANK/JN665172
SI  - GENBANK/JN665173
SI  - GENBANK/JN665174
SI  - GENBANK/JN665175
SI  - GENBANK/JN665176
SI  - GENBANK/JN665177
SI  - GENBANK/JN665178
SI  - GENBANK/JN665179
SI  - GENBANK/JN665180
SI  - GENBANK/JN665181
SI  - GENBANK/JN665182
SI  - GENBANK/JN665183
SI  - GENBANK/JN665184
SI  - GENBANK/JN665185
SI  - GENBANK/JN665186
SI  - GENBANK/JN665187
SI  - GENBANK/JN665188
SI  - GENBANK/JN665189
SI  - GENBANK/JN665190
SI  - GENBANK/JN665191
SI  - GENBANK/JN665192
SI  - GENBANK/JN665193
SI  - GENBANK/JN665194
SI  - GENBANK/JN665195
SI  - GENBANK/JN665196
SI  - GENBANK/JN665197
SI  - GENBANK/JN665198
SI  - GENBANK/JN665199
SI  - GENBANK/JN665200
SI  - GENBANK/JN665201
SI  - GENBANK/JN665202
SI  - GENBANK/JN665203
SI  - GENBANK/JN665204
SI  - GENBANK/JN665205
SI  - GENBANK/JN665206
SI  - GENBANK/JN665207
SI  - GENBANK/JN665208
SI  - GENBANK/JN665209
SI  - GENBANK/JN665210
SI  - GENBANK/JN665211
SI  - GENBANK/JN665212
SI  - GENBANK/JN665213
SI  - GENBANK/JN665214
SI  - GENBANK/JN665215
SI  - GENBANK/JN665216
SI  - GENBANK/JN665217
SI  - GENBANK/JN665218
SI  - GENBANK/JN665219
SI  - GENBANK/JN665220
SI  - GENBANK/JN665221
SI  - GENBANK/JN665222
SI  - GENBANK/JN665223
SI  - GENBANK/JN665224
SI  - GENBANK/JN665225
SI  - GENBANK/JN665226
SI  - GENBANK/JN665227
SI  - GENBANK/JN665228
SI  - GENBANK/JN665229
SI  - GENBANK/JN665230
SI  - GENBANK/JN665231
SI  - GENBANK/JN665232
SI  - GENBANK/JN665233
SI  - GENBANK/JN665234
SI  - GENBANK/JN665235
SI  - GENBANK/JN665236
SI  - GENBANK/JN665237
SI  - GENBANK/JN665238
SI  - GENBANK/JN665239
SI  - GENBANK/JN665240
SI  - GENBANK/JN665241
SI  - GENBANK/JN665242
SI  - GENBANK/JN665243
SI  - GENBANK/JN665244
SI  - GENBANK/JN665245
SI  - GENBANK/JN665246
SI  - GENBANK/JN665247
SI  - GENBANK/JN665248
SI  - GENBANK/JN665249
SI  - GENBANK/JN665250
SI  - GENBANK/JN665251
SI  - GENBANK/JN665252
SI  - GENBANK/JN665253
SI  - GENBANK/JN665254
SI  - GENBANK/JN665255
SI  - GENBANK/JN665256
SI  - GENBANK/JN665257
SI  - GENBANK/JN665258
SI  - GENBANK/JN665259
SI  - GENBANK/JN665260
SI  - GENBANK/JN665261
SI  - GENBANK/JN665262
SI  - GENBANK/JN665263
SI  - GENBANK/JN665264
SI  - GENBANK/JN665265
SI  - GENBANK/JN665266
SI  - GENBANK/JN665267
SI  - GENBANK/JN665268
SI  - GENBANK/JN665269
SI  - GENBANK/JN665270
SI  - GENBANK/JN665271
SI  - GENBANK/JN665272
SI  - GENBANK/JN665273
SI  - GENBANK/JN665274
SI  - GENBANK/JN665275
SI  - GENBANK/JN665276
SI  - GENBANK/JN665277
SI  - GENBANK/JN665278
SI  - GENBANK/JN665279
SI  - GENBANK/JN665280
SI  - GENBANK/JN665281
SI  - GENBANK/JN665282
SI  - GENBANK/JN665283
SI  - GENBANK/JN665284
SI  - GENBANK/JN665285
SI  - GENBANK/JN665286
SI  - GENBANK/JN665287
SI  - GENBANK/JN665288
SI  - GENBANK/JN665289
SI  - GENBANK/JN665290
SI  - GENBANK/JN665291
SI  - GENBANK/JN665292
SI  - GENBANK/JN665293
SI  - GENBANK/JN665294
SI  - GENBANK/JN665295
SI  - GENBANK/JN665296
SI  - GENBANK/JN665297
SI  - GENBANK/JN665298
SI  - GENBANK/JN665299
SI  - GENBANK/JN665300
SI  - GENBANK/JN665301
SI  - GENBANK/JN665302
SI  - GENBANK/JN665303
SI  - GENBANK/JN665304
SI  - GENBANK/JN665305
SI  - GENBANK/JN665306
SI  - GENBANK/JN665307
SI  - GENBANK/JN665308
SI  - GENBANK/JN665309
SI  - GENBANK/JN665310
SI  - GENBANK/JN665311
SI  - GENBANK/JN665312
SI  - GENBANK/JN665313
SI  - GENBANK/JN665314
SI  - GENBANK/JN665315
SI  - GENBANK/JN665316
SI  - GENBANK/JN665317
SI  - GENBANK/JN665318
SI  - GENBANK/JN665319
SI  - GENBANK/JN665320
SI  - GENBANK/JN665321
SI  - GENBANK/JN665322
SI  - GENBANK/JN665323
SI  - GENBANK/JN665324
SI  - GENBANK/JN665325
SI  - GENBANK/JN665326
SI  - GENBANK/JN665327
SI  - GENBANK/JN665328
SI  - GENBANK/JN665329
SI  - GENBANK/JN665330
SI  - GENBANK/JN665331
SI  - GENBANK/JN665332
SI  - GENBANK/JN665333
SI  - GENBANK/JN665334
SI  - GENBANK/JN665335
SI  - GENBANK/JN665336
SI  - GENBANK/JN665337
SI  - GENBANK/JN665338
SI  - GENBANK/JN665339
SI  - GENBANK/JN665340
SI  - GENBANK/JN665341
SI  - GENBANK/JN665342
SI  - GENBANK/JN665343
SI  - GENBANK/JN665344
SI  - GENBANK/JN665345
SI  - GENBANK/JN665346
SI  - GENBANK/JN665347
SI  - GENBANK/JN665348
SI  - GENBANK/JN665349
SI  - GENBANK/JN665350
SI  - GENBANK/JN665351
SI  - GENBANK/JN665352
SI  - GENBANK/JN665353
SI  - GENBANK/JN665354
SI  - GENBANK/JN665355
SI  - GENBANK/JN665356
SI  - GENBANK/JN665357
SI  - GENBANK/JN665358
SI  - GENBANK/JN665359
SI  - GENBANK/JN665360
SI  - GENBANK/JN665361
SI  - GENBANK/JN665362
SI  - GENBANK/JN665363
SI  - GENBANK/JN665364
SI  - GENBANK/JN665365
SI  - GENBANK/JN665366
SI  - GENBANK/JN665367
SI  - GENBANK/JN665368
SI  - GENBANK/JN665369
SI  - GENBANK/JN665370
SI  - GENBANK/JN665371
SI  - GENBANK/JN665372
SI  - GENBANK/JN665373
SI  - GENBANK/JN665374
SI  - GENBANK/JN665375
SI  - GENBANK/JN665376
SI  - GENBANK/JN665377
SI  - GENBANK/JN665378
SI  - GENBANK/JN665379
SI  - GENBANK/JN665380
SI  - GENBANK/JN665381
SI  - GENBANK/JN665382
SI  - GENBANK/JN665383
SI  - GENBANK/JN665384
SI  - GENBANK/JN665385
SI  - GENBANK/JN665386
SI  - GENBANK/JN665387
SI  - GENBANK/JN665388
SI  - GENBANK/JN665389
SI  - GENBANK/JN665390
SI  - GENBANK/JN665391
SI  - GENBANK/JN665392
SI  - GENBANK/JN665393
SI  - GENBANK/JN665394
SI  - GENBANK/JN665395
SI  - GENBANK/JN665396
SI  - GENBANK/JN665397
SI  - GENBANK/JN665398
SI  - GENBANK/JN665399
SI  - GENBANK/JN665400
SI  - GENBANK/JN665401
SI  - GENBANK/JN665402
SI  - GENBANK/JN665403
SI  - GENBANK/JN665404
SI  - GENBANK/JN665405
SI  - GENBANK/JN665406
SI  - GENBANK/JN665407
SI  - GENBANK/JN665408
SI  - GENBANK/JN665409
SI  - GENBANK/JN665410
SI  - GENBANK/JN665411
SI  - GENBANK/JN665412
SI  - GENBANK/JN665413
SI  - GENBANK/JN665414
SI  - GENBANK/JN665415
SI  - GENBANK/JN665416
SI  - GENBANK/JN665417
SI  - GENBANK/JN665418
SI  - GENBANK/JN665419
SI  - GENBANK/JN665420
SI  - GENBANK/JN665421
SI  - GENBANK/JN665422
SI  - GENBANK/JN665423
SI  - GENBANK/JN665424
SI  - GENBANK/JN665425
SI  - GENBANK/JN665426
SI  - GENBANK/JN665427
SI  - GENBANK/JN665428
SI  - GENBANK/JN665429
SI  - GENBANK/JN665430
SI  - GENBANK/JN665431
SI  - GENBANK/JN665432
SI  - GENBANK/JN665433
SI  - GENBANK/JN665434
SI  - GENBANK/JN665435
SI  - GENBANK/JN665436
SI  - GENBANK/JN665437
SI  - GENBANK/JN665438
SI  - GENBANK/JN665439
SI  - GENBANK/JN665440
SI  - GENBANK/JN665441
SI  - GENBANK/JN665442
SI  - GENBANK/JN665443
SI  - GENBANK/JN665444
SI  - GENBANK/JN665445
SI  - GENBANK/JN665446
SI  - GENBANK/JN665447
SI  - GENBANK/JN665448
SI  - GENBANK/JN665449
SI  - GENBANK/JN665450
SI  - GENBANK/JN665451
SI  - GENBANK/JN665452
SI  - GENBANK/JN665453
SI  - GENBANK/JN665454
SI  - GENBANK/JN665455
SI  - GENBANK/JN665456
SI  - GENBANK/JN665457
SI  - GENBANK/JN665458
SI  - GENBANK/JN665459
SI  - GENBANK/JN665460
SI  - GENBANK/JN665461
SI  - GENBANK/JN665462
SI  - GENBANK/JN665463
SI  - GENBANK/JN665464
SI  - GENBANK/JN665465
SI  - GENBANK/JN665466
SI  - GENBANK/JN665467
SI  - GENBANK/JN665468
SI  - GENBANK/JN665469
SI  - GENBANK/JN665470
SI  - GENBANK/JN665471
SI  - GENBANK/JN665472
SI  - GENBANK/JN665473
SI  - GENBANK/JN665474
SI  - GENBANK/JN665475
SI  - GENBANK/JN665476
SI  - GENBANK/JN665477
SI  - GENBANK/JN665478
SI  - GENBANK/JN665479
SI  - GENBANK/JN665480
SI  - GENBANK/JN665481
SI  - GENBANK/JN665482
SI  - GENBANK/JN665483
SI  - GENBANK/JN665484
SI  - GENBANK/JN665485
SI  - GENBANK/JN665486
SI  - GENBANK/JN665487
SI  - GENBANK/JN665488
SI  - GENBANK/JN665489
SI  - GENBANK/JN665490
SI  - GENBANK/JN665491
SI  - GENBANK/JN665492
SI  - GENBANK/JN665493
SI  - GENBANK/JN665494
SI  - GENBANK/JN665495
SI  - GENBANK/JN665496
SI  - GENBANK/JN665497
SI  - GENBANK/JN665498
SI  - GENBANK/JN665499
SI  - GENBANK/JN665500
SI  - GENBANK/JN665501
SI  - GENBANK/JN665502
SI  - GENBANK/JN665503
SI  - GENBANK/JN665504
SI  - GENBANK/JN665505
SI  - GENBANK/JN665506
SI  - GENBANK/JN665507
SI  - GENBANK/JN665508
SI  - GENBANK/JN665509
SI  - GENBANK/JN665510
SI  - GENBANK/JN665511
SI  - GENBANK/JN665512
SI  - GENBANK/JN665513
SI  - GENBANK/JN665514
SI  - GENBANK/JN665515
SI  - GENBANK/JN665516
SI  - GENBANK/JN665517
SI  - GENBANK/JN665518
SI  - GENBANK/JN665519
SI  - GENBANK/JN665520
SI  - GENBANK/JN665521
SI  - GENBANK/JN665522
SI  - GENBANK/JN665523
SI  - GENBANK/JN665524
SI  - GENBANK/JN665525
SI  - GENBANK/JN665526
SI  - GENBANK/JN665527
SI  - GENBANK/JN665528
SI  - GENBANK/JN665529
SI  - GENBANK/JN665530
SI  - GENBANK/JN665531
SI  - GENBANK/JN665532
SI  - GENBANK/JN665533
SI  - GENBANK/JN665534
SI  - GENBANK/JN665535
SI  - GENBANK/JN665536
SI  - GENBANK/JN665537
SI  - GENBANK/JN665538
SI  - GENBANK/JN665539
SI  - GENBANK/JN665540
SI  - GENBANK/JN665541
SI  - GENBANK/JN665542
SI  - GENBANK/JN665543
SI  - GENBANK/JN665544
SI  - GENBANK/JN665545
SI  - GENBANK/JN665546
SI  - GENBANK/JN665547
SI  - GENBANK/JN665548
SI  - GENBANK/JN665549
SI  - GENBANK/JN665550
SI  - GENBANK/JN665551
SI  - GENBANK/JN665552
SI  - GENBANK/JN665553
SI  - GENBANK/JN665554
SI  - GENBANK/JN665555
SI  - GENBANK/JN665556
SI  - GENBANK/JN665557
SI  - GENBANK/JN665558
SI  - GENBANK/JN665559
SI  - GENBANK/JN665560
SI  - GENBANK/JN665561
SI  - GENBANK/JN665562
SI  - GENBANK/JN665563
SI  - GENBANK/JN665564
SI  - GENBANK/JN665565
SI  - GENBANK/JN665566
SI  - GENBANK/JN665567
SI  - GENBANK/JN665568
SI  - GENBANK/JN665569
SI  - GENBANK/JN665570
SI  - GENBANK/JN665571
SI  - GENBANK/JN665572
SI  - GENBANK/JN665573
SI  - GENBANK/JN665574
SI  - GENBANK/JN665575
SI  - GENBANK/JN665576
SI  - GENBANK/JN665577
SI  - GENBANK/JN665578
SI  - GENBANK/JN665579
SI  - GENBANK/JN665580
SI  - GENBANK/JN665581
SI  - GENBANK/JN665582
SI  - GENBANK/JN665583
SI  - GENBANK/JN665584
SI  - GENBANK/JN665585
SI  - GENBANK/JN665586
SI  - GENBANK/JN665587
SI  - GENBANK/JN665588
SI  - GENBANK/JN665589
SI  - GENBANK/JN665590
SI  - GENBANK/JN665591
SI  - GENBANK/JN665592
SI  - GENBANK/JN665593
SI  - GENBANK/JN665594
SI  - GENBANK/JN665595
SI  - GENBANK/JN665596
SI  - GENBANK/JN665597
SI  - GENBANK/JN665598
SI  - GENBANK/JN665599
SI  - GENBANK/JN665600
SI  - GENBANK/JN665601
SI  - GENBANK/JN665602
SI  - GENBANK/JN665603
SI  - GENBANK/JN665604
SI  - GENBANK/JN665605
SI  - GENBANK/JN665606
SI  - GENBANK/JN665607
SI  - GENBANK/JN665608
SI  - GENBANK/JN665609
SI  - GENBANK/JN665610
SI  - GENBANK/JN665611
SI  - GENBANK/JN665612
SI  - GENBANK/JN665613
SI  - GENBANK/JN665614
SI  - GENBANK/JN665615
SI  - GENBANK/JN665616
SI  - GENBANK/JN665617
SI  - GENBANK/JN665618
SI  - GENBANK/JN665619
SI  - GENBANK/JN665620
SI  - GENBANK/JN665621
SI  - GENBANK/JN665622
SI  - GENBANK/JN665623
SI  - GENBANK/JN665624
SI  - GENBANK/JN665625
SI  - GENBANK/JN665626
SI  - GENBANK/JN665627
SI  - GENBANK/JN665628
SI  - GENBANK/JN665629
SI  - GENBANK/JN665630
SI  - GENBANK/JN665631
SI  - GENBANK/JN665632
SI  - GENBANK/JN665633
SI  - GENBANK/JN665634
SI  - GENBANK/JN665635
SI  - GENBANK/JN665636
SI  - GENBANK/JN665637
SI  - GENBANK/JN665638
SI  - GENBANK/JN665639
SI  - GENBANK/JN665640
SI  - GENBANK/JN665641
SI  - GENBANK/JN665642
SI  - GENBANK/JN665643
SI  - GENBANK/JN665644
SI  - GENBANK/JN665645
SI  - GENBANK/JN665646
SI  - GENBANK/JN665647
SI  - GENBANK/JN665648
SI  - GENBANK/JN665649
SI  - GENBANK/JN665650
SI  - GENBANK/JN665651
SI  - GENBANK/JN665652
SI  - GENBANK/JN665653
SI  - GENBANK/JN665654
SI  - GENBANK/JN665655
SI  - GENBANK/JN665656
SI  - GENBANK/JN665657
SI  - GENBANK/JN665658
SI  - GENBANK/JN665659
SI  - GENBANK/JN665660
SI  - GENBANK/JN665661
SI  - GENBANK/JN665662
SI  - GENBANK/JN665663
SI  - GENBANK/JN665664
SI  - GENBANK/JN665665
SI  - GENBANK/JN665666
SI  - GENBANK/JN665667
SI  - GENBANK/JN665668
SI  - GENBANK/JN665669
SI  - GENBANK/JN665670
SI  - GENBANK/JN665671
SI  - GENBANK/JN665672
SI  - GENBANK/JN665673
SI  - GENBANK/JN665674
SI  - GENBANK/JN665675
SI  - GENBANK/JN665676
SI  - GENBANK/JN665677
SI  - GENBANK/JN665678
SI  - GENBANK/JN665679
SI  - GENBANK/JN665680
SI  - GENBANK/JN665681
SI  - GENBANK/JN665682
SI  - GENBANK/JN665683
SI  - GENBANK/JN665684
SI  - GENBANK/JN665685
SI  - GENBANK/JN665686
SI  - GENBANK/JN665687
SI  - GENBANK/JN665688
SI  - GENBANK/JN665689
SI  - GENBANK/JN665690
SI  - GENBANK/JN665691
SI  - GENBANK/JN665692
SI  - GENBANK/JN665693
SI  - GENBANK/JN665694
SI  - GENBANK/JN665695
SI  - GENBANK/JN665696
SI  - GENBANK/JN665697
SI  - GENBANK/JN665698
SI  - GENBANK/JN665699
SI  - GENBANK/JN665700
SI  - GENBANK/JN665701
SI  - GENBANK/JN665702
SI  - GENBANK/JN665703
SI  - GENBANK/JN665704
SI  - GENBANK/JN665705
SI  - GENBANK/JN665706
SI  - GENBANK/JN665707
SI  - GENBANK/JN665708
SI  - GENBANK/JN665709
SI  - GENBANK/JN665710
SI  - GENBANK/JN665711
SI  - GENBANK/JN665712
SI  - GENBANK/JN665713
SI  - GENBANK/JN665714
SI  - GENBANK/JN665715
SI  - GENBANK/JN665716
SI  - GENBANK/JN665717
SI  - GENBANK/JN665718
SI  - GENBANK/JN665719
SI  - GENBANK/JN665720
SI  - GENBANK/JN665721
SI  - GENBANK/JN665722
SI  - GENBANK/JN665723
SI  - GENBANK/JN665724
SI  - GENBANK/JN665725
SI  - GENBANK/JN665726
SI  - GENBANK/JN665727
SI  - GENBANK/JN665728
SI  - GENBANK/JN665729
SI  - GENBANK/JN665730
SI  - GENBANK/JN665731
SI  - GENBANK/JN665732
SI  - GENBANK/JN665733
SI  - GENBANK/JN665734
SI  - GENBANK/JN665735
SI  - GENBANK/JN665736
SI  - GENBANK/JN665737
SI  - GENBANK/JN665738
SI  - GENBANK/JN665739
SI  - GENBANK/JN665740
SI  - GENBANK/JN665741
SI  - GENBANK/JN665742
SI  - GENBANK/JN665743
SI  - GENBANK/JN665744
SI  - GENBANK/JN665745
SI  - GENBANK/JN665746
SI  - GENBANK/JN665747
SI  - GENBANK/JN665748
SI  - GENBANK/JN665749
SI  - GENBANK/JN665750
SI  - GENBANK/JN665751
SI  - GENBANK/JN665752
SI  - GENBANK/JN665753
SI  - GENBANK/JN665754
SI  - GENBANK/JN665755
SI  - GENBANK/JN665756
SI  - GENBANK/JN665757
SI  - GENBANK/JN665758
SI  - GENBANK/JN665759
SI  - GENBANK/JN665760
SI  - GENBANK/JN665761
SI  - GENBANK/JN665762
SI  - GENBANK/JN665763
SI  - GENBANK/JN665764
SI  - GENBANK/JN665765
SI  - GENBANK/JN665766
SI  - GENBANK/JN665767
SI  - GENBANK/JN665768
SI  - GENBANK/JN665769
SI  - GENBANK/JN665770
SI  - GENBANK/JN665771
SI  - GENBANK/JN665772
SI  - GENBANK/JN665773
SI  - GENBANK/JN665774
SI  - GENBANK/JN665775
SI  - GENBANK/JN665776
SI  - GENBANK/JN665777
SI  - GENBANK/JN665778
SI  - GENBANK/JN665779
SI  - GENBANK/JN665780
SI  - GENBANK/JN665781
SI  - GENBANK/JN665782
SI  - GENBANK/JN665783
SI  - GENBANK/JN665784
SI  - GENBANK/JN665785
SI  - GENBANK/JN665786
SI  - GENBANK/JN665787
SI  - GENBANK/JN665788
SI  - GENBANK/JN665789
SI  - GENBANK/JN665790
SI  - GENBANK/JN665791
SI  - GENBANK/JN665792
SI  - GENBANK/JN665793
SI  - GENBANK/JN665794
SI  - GENBANK/JN665795
SI  - GENBANK/JN665796
SI  - GENBANK/JN665797
SI  - GENBANK/JN665798
SI  - GENBANK/JN665799
SI  - GENBANK/JN665800
SI  - GENBANK/JN665801
SI  - GENBANK/JN665802
SI  - GENBANK/JN665803
SI  - GENBANK/JN665804
SI  - GENBANK/JN665805
SI  - GENBANK/JN665806
SI  - GENBANK/JN665807
SI  - GENBANK/JN665808
SI  - GENBANK/JN665809
SI  - GENBANK/JN665810
SI  - GENBANK/JN665811
SI  - GENBANK/JN665812
SI  - GENBANK/JN665813
SI  - GENBANK/JN665814
SI  - GENBANK/JN665815
SI  - GENBANK/JN665816
SI  - GENBANK/JN665817
SI  - GENBANK/JN665818
SI  - GENBANK/JN665819
SI  - GENBANK/JN665820
SI  - GENBANK/JN665821
SI  - GENBANK/JN665822
SI  - GENBANK/JN665823
SI  - GENBANK/JN665824
SI  - GENBANK/JN665825
SI  - GENBANK/JN665826
SI  - GENBANK/JN665827
SI  - GENBANK/JN665828
SI  - GENBANK/JN665829
SI  - GENBANK/JN665830
SI  - GENBANK/JN665831
SI  - GENBANK/JN665832
SI  - GENBANK/JN665833
SI  - GENBANK/JN665834
SI  - GENBANK/JN665835
SI  - GENBANK/JN665836
SI  - GENBANK/JN665837
SI  - GENBANK/JN665838
SI  - GENBANK/JN665839
SI  - GENBANK/JN665840
SI  - GENBANK/JN665841
SI  - GENBANK/JN665842
SI  - GENBANK/JN665843
SI  - GENBANK/JN665844
SI  - GENBANK/JN665845
SI  - GENBANK/JN665846
SI  - GENBANK/JN665847
SI  - GENBANK/JN665848
SI  - GENBANK/JN665849
SI  - GENBANK/JN665850
SI  - GENBANK/JN665851
SI  - GENBANK/JN665852
SI  - GENBANK/JN665853
SI  - GENBANK/JN665854
SI  - GENBANK/JN665855
SI  - GENBANK/JN665856
SI  - GENBANK/JN665857
SI  - GENBANK/JN665858
SI  - GENBANK/JN665859
SI  - GENBANK/JN665860
SI  - GENBANK/JN665861
SI  - GENBANK/JN665862
SI  - GENBANK/JN665863
SI  - GENBANK/JN665864
SI  - GENBANK/JN665865
SI  - GENBANK/JN665866
SI  - GENBANK/JN665867
SI  - GENBANK/JN665868
SI  - GENBANK/JN665869
SI  - GENBANK/JN665870
SI  - GENBANK/JN665871
SI  - GENBANK/JN665872
SI  - GENBANK/JN665873
SI  - GENBANK/JN665874
SI  - GENBANK/JN665875
SI  - GENBANK/JN665876
SI  - GENBANK/JN665877
SI  - GENBANK/JN665878
SI  - GENBANK/JN665879
SI  - GENBANK/JN665880
SI  - GENBANK/JN665881
SI  - GENBANK/JN665882
SI  - GENBANK/JN665883
SI  - GENBANK/JN665884
SI  - GENBANK/JN665885
SI  - GENBANK/JN665886
SI  - GENBANK/JN665887
SI  - GENBANK/JN665888
SI  - GENBANK/JN665889
SI  - GENBANK/JN665890
SI  - GENBANK/JN665891
SI  - GENBANK/JN665892
SI  - GENBANK/JN665893
SI  - GENBANK/JN665894
SI  - GENBANK/JN665895
SI  - GENBANK/JN665896
SI  - GENBANK/JN665897
SI  - GENBANK/JN665898
SI  - GENBANK/JN665899
SI  - GENBANK/JN665900
SI  - GENBANK/JN665901
SI  - GENBANK/JN665902
SI  - GENBANK/JN665903
SI  - GENBANK/JN665904
SI  - GENBANK/JN665905
SI  - GENBANK/JN665906
SI  - GENBANK/JN665907
SI  - GENBANK/JN665908
SI  - GENBANK/JN665909
SI  - GENBANK/JN665910
SI  - GENBANK/JN665911
SI  - GENBANK/JN665912
SI  - GENBANK/JN665913
SI  - GENBANK/JN665914
SI  - GENBANK/JN665915
SI  - GENBANK/JN665916
SI  - GENBANK/JN665917
SI  - GENBANK/JN665918
SI  - GENBANK/JN665919
SI  - GENBANK/JN665920
SI  - GENBANK/JN665921
SI  - GENBANK/JN665922
SI  - GENBANK/JN665923
SI  - GENBANK/JN665924
SI  - GENBANK/JN665925
SI  - GENBANK/JN665926
SI  - GENBANK/JN665927
SI  - GENBANK/JN665928
SI  - GENBANK/JN665929
SI  - GENBANK/JN665930
SI  - GENBANK/JN665931
SI  - GENBANK/JN665932
SI  - GENBANK/JN665933
SI  - GENBANK/JN665934
SI  - GENBANK/JN665935
SI  - GENBANK/JN665936
SI  - GENBANK/JN665937
SI  - GENBANK/JN665938
SI  - GENBANK/JN665939
SI  - GENBANK/JN665940
SI  - GENBANK/JN665941
SI  - GENBANK/JN665942
SI  - GENBANK/JN665943
SI  - GENBANK/JN665944
SI  - GENBANK/JN665945
SI  - GENBANK/JN665946
SI  - GENBANK/JN665947
SI  - GENBANK/JN665948
SI  - GENBANK/JN665949
SI  - GENBANK/JN665950
SI  - GENBANK/JN665951
SI  - GENBANK/JN665952
SI  - GENBANK/JN665953
SI  - GENBANK/JN665954
SI  - GENBANK/JN665955
SI  - GENBANK/JN665956
SI  - GENBANK/JN665957
SI  - GENBANK/JN665958
SI  - GENBANK/JN665959
SI  - GENBANK/JN665960
SI  - GENBANK/JN665961
SI  - GENBANK/JN665962
SI  - GENBANK/JN665963
SI  - GENBANK/JN665964
SI  - GENBANK/JN665965
SI  - GENBANK/JN665966
SI  - GENBANK/JN665967
SI  - GENBANK/JN665968
SI  - GENBANK/JN665969
SI  - GENBANK/JN665970
SI  - GENBANK/JN665971
SI  - GENBANK/JN665972
SI  - GENBANK/JN665973
SI  - GENBANK/JN665974
SI  - GENBANK/JN665975
SI  - GENBANK/JN665976
SI  - GENBANK/JN665977
SI  - GENBANK/JN665978
SI  - GENBANK/JN665979
SI  - GENBANK/JN665980
SI  - GENBANK/JN665981
SI  - GENBANK/JN665982
SI  - GENBANK/JN665983
SI  - GENBANK/JN665984
SI  - GENBANK/JN665985
SI  - GENBANK/JN665986
SI  - GENBANK/JN665987
SI  - GENBANK/JN665988
SI  - GENBANK/JN665989
SI  - GENBANK/JN665990
SI  - GENBANK/JN665991
SI  - GENBANK/JN665992
SI  - GENBANK/JN665993
SI  - GENBANK/JN665994
SI  - GENBANK/JN665995
SI  - GENBANK/JN665996
SI  - GENBANK/JN665997
SI  - GENBANK/JN665998
SI  - GENBANK/JN665999
SI  - GENBANK/JN666000
SI  - GENBANK/JN666001
SI  - GENBANK/JN666002
SI  - GENBANK/JN666003
SI  - GENBANK/JN666004
SI  - GENBANK/JN666005
SI  - GENBANK/JN666006
SI  - GENBANK/JN666007
SI  - GENBANK/JN666008
SI  - GENBANK/JN666009
SI  - GENBANK/JN666010
SI  - GENBANK/JN666011
SI  - GENBANK/JN666012
SI  - GENBANK/JN666013
SI  - GENBANK/JN666014
SI  - GENBANK/JN666015
SI  - GENBANK/JN666016
SI  - GENBANK/JN666017
SI  - GENBANK/JN666018
SI  - GENBANK/JN666019
SI  - GENBANK/JN666020
SI  - GENBANK/JN666021
SI  - GENBANK/JN666022
SI  - GENBANK/JN666023
SI  - GENBANK/JN666024
SI  - GENBANK/JN666025
SI  - GENBANK/JN666026
SI  - GENBANK/JN666027
SI  - GENBANK/JN666028
SI  - GENBANK/JN666029
SI  - GENBANK/JN666030
SI  - GENBANK/JN666031
SI  - GENBANK/JN666032
SI  - GENBANK/JN666033
SI  - GENBANK/JN666034
SI  - GENBANK/JN666035
SI  - GENBANK/JN666036
SI  - GENBANK/JN666037
SI  - GENBANK/JN666038
SI  - GENBANK/JN666039
SI  - GENBANK/JN666040
SI  - GENBANK/JN666041
SI  - GENBANK/JN666042
SI  - GENBANK/JN666043
SI  - GENBANK/JN666044
SI  - GENBANK/JN666045
SI  - GENBANK/JN666046
SI  - GENBANK/JN666047
SI  - GENBANK/JN666048
SI  - GENBANK/JN666049
SI  - GENBANK/JN666050
SI  - GENBANK/JN666051
SI  - GENBANK/JN666052
SI  - GENBANK/JN666053
SI  - GENBANK/JN666054
SI  - GENBANK/JN666055
SI  - GENBANK/JN666056
SI  - GENBANK/JN666057
SI  - GENBANK/JN666058
SI  - GENBANK/JN666059
SI  - GENBANK/JN666060
SI  - GENBANK/JN666061
SI  - GENBANK/JN666062
SI  - GENBANK/JN666063
SI  - GENBANK/JN666064
SI  - GENBANK/JN666065
SI  - GENBANK/JN666066
SI  - GENBANK/JN666067
SI  - GENBANK/JN666068
SI  - GENBANK/JN666069
SI  - GENBANK/JN666070
SI  - GENBANK/JN666071
SI  - GENBANK/JN666072
SI  - GENBANK/JN666073
SI  - GENBANK/JN666074
SI  - GENBANK/JN666075
SI  - GENBANK/JN666076
SI  - GENBANK/JN666077
SI  - GENBANK/JN666078
SI  - GENBANK/JN666079
SI  - GENBANK/JN666080
SI  - GENBANK/JN666081
SI  - GENBANK/JN666082
SI  - GENBANK/JN666083
SI  - GENBANK/JN666084
SI  - GENBANK/JN666085
SI  - GENBANK/JN666086
SI  - GENBANK/JN666087
SI  - GENBANK/JN666088
SI  - GENBANK/JN666089
SI  - GENBANK/JN666090
SI  - GENBANK/JN666091
SI  - GENBANK/JN666092
SI  - GENBANK/JN666093
SI  - GENBANK/JN666094
SI  - GENBANK/JN666095
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120503
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Plant Proteins)
SB  - IM
MH  - Bayes Theorem
MH  - Cell Nucleus/genetics
MH  - *Environment
MH  - *Genetic Variation
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Phylogeography
MH  - Plant Proteins/*genetics
MH  - Sarraceniaceae/classification/*genetics
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
MH  - Southeastern United States
MH  - Texas
EDAT- 2012/05/05 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/05/05 06:00
PHST- 2012/05/05 06:00 [entrez]
PHST- 2012/05/05 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys048 [pii]
AID - 10.1093/sysbio/sys048 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):763-77. doi: 10.1093/sysbio/sys048. Epub 2012 May 3.

PMID- 22511121
OWN - NLM
STAT- MEDLINE
DCOM- 20130305
LR  - 20121024
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 6
DP  - 2012 Dec 1
TI  - Testing of the effect of missing data estimation and distribution in morphometric
      multivariate data analyses.
PG  - 941-54
LID - 10.1093/sysbio/sys047 [doi]
AB  - Missing data are an unavoidable problem in biological data sets and the
      performance of missing data deletion and estimation techniques in morphometric
      data sets is poorly understood. Here, a novel method is used to measure the
      introduced error of multiple techniques on a representative sample. A large
      sample of extant crocodilian skulls was measured and analyzed with principal
      component analysis (PCA). Twenty-three different proportions of missing data were
      introduced into the data set, estimated, analyzed, and compared with the original
      result using Procrustes superimposition. Previous work investigating the effects 
      of missing data input missing values randomly, a non-biological phenomenon. Here,
      missing data were introduced into the data set using three methodologies: purely 
      at random, as a function of the Euclidean distance between respective
      measurements (simulating anatomical regions), and as a function of the portion of
      the sample occupied by each taxon (simulating unequal missing data in rare taxa).
      Gower's distance was found to be the best performing non-estimation method, and
      Bayesian PCA the best performing estimation method. Specimens of the taxa with
      small sample sizes and those most morphologically disparate had the highest
      estimation error. Distribution of missing data had a significant effect on the
      estimation error for almost all methods and proportions. Taxonomically biased
      missing data tended to show similar trends to random, but with higher error
      rates. Anatomically biased missing data showed a much greater deviation from
      random than the taxonomic bias, and with magnitudes dependent on the estimation
      method.
FAU - Brown, Caleb Marshall
AU  - Brown CM
AD  - Department of Ecology and Evolutionary Biology, University of Toronto, 25
      Willcocks Street, Toronto, Ontario M5S 3B2, Canada. caleb.brown@utoronto.ca
FAU - Arbour, Jessica H
AU  - Arbour JH
FAU - Jackson, Donald A
AU  - Jackson DA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120417
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Alligators and Crocodiles/anatomy &amp; histology/classification
MH  - Animals
MH  - *Data Interpretation, Statistical
MH  - Principal Component Analysis
EDAT- 2012/04/19 06:00
MHDA- 2013/03/06 06:00
CRDT- 2012/04/19 06:00
PHST- 2012/04/19 06:00 [entrez]
PHST- 2012/04/19 06:00 [pubmed]
PHST- 2013/03/06 06:00 [medline]
AID - sys047 [pii]
AID - 10.1093/sysbio/sys047 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Dec 1;61(6):941-54. doi: 10.1093/sysbio/sys047. Epub 2012 Apr 17.

PMID- 22508720
OWN - NLM
STAT- MEDLINE
DCOM- 20130305
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 6
DP  - 2012 Dec 1
TI  - Fitting nonstationary general-time-reversible models to obtain edge-lengths and
      frequencies for the barry-hartigan model.
PG  - 927-40
LID - 10.1093/sysbio/sys046 [doi]
AB  - Among models of nucleotide evolution, the Barry and Hartigan (BH) model (also
      known as the General Markov Model) is very flexible as it allows separate
      arbitrary substitution matrices along edges. For a given tree, the estimates of
      the BH model are a set of joint probability matrices, each giving the pairwise
      frequencies of nucleotides at the ends of the edge. We have previously shown
      that, due to an identifiability problem, these cannot be expected to consistently
      estimate the actual pairwise frequencies. A further consequence is that internal 
      node frequency estimates are likely to be incorrect. Here we define a
      nonstationary GTR model for each edge that we refer to as the NSGTR model. We fit
      the NSGTR model by minimizing the sums of squares between the estimates of
      transition probabilities under the NSGTR model and the estimates provided by a
      fitted BH model. This NSGTR model provides estimates that avoid the
      identifiability difficulties of the BH model while closely fitting it. With the
      best-fitting NSGTR estimates, we are able to get interpretable frequency vectors 
      at internal nodes as well as edge length estimates that are otherwise not yielded
      by the BH model. These edge lengths are interpretable as the expected number of
      substitutions along an edge for the model. We also show that for a nonstationary 
      continuous-time model these are not the same as the edge length parameters for
      conventional substitution matrices that are output by nonstationary model
      phylogenetic estimation programs such as nhPhyML.
FAU - Zou, Liwen
AU  - Zou L
AD  - Bioinformatics Research Center, Department of Genetics, North Carolina State
      University, NC, USA.
FAU - Susko, Edward
AU  - Susko E
FAU - Field, Chris
AU  - Field C
FAU - Roger, Andrew J
AU  - Roger AJ
LA  - eng
GR  - R01 GM070806/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120416
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - Evolution, Molecular
MH  - Genetic Heterogeneity
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - Plasmodium/classification/genetics
PMC - PMC3888140
EDAT- 2012/04/18 06:00
MHDA- 2013/03/06 06:00
CRDT- 2012/04/18 06:00
PHST- 2012/04/18 06:00 [entrez]
PHST- 2012/04/18 06:00 [pubmed]
PHST- 2013/03/06 06:00 [medline]
AID - sys046 [pii]
AID - 10.1093/sysbio/sys046 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Dec 1;61(6):927-40. doi: 10.1093/sysbio/sys046. Epub 2012 Apr 16.

PMID- 22508719
OWN - NLM
STAT- MEDLINE
DCOM- 20121018
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 4
DP  - 2012 Jul
TI  - Peptidomics-based phylogeny and biogeography of Mantophasmatodea (Hexapoda).
PG  - 609-29
LID - 10.1093/sysbio/sys003 [doi]
AB  - The insect order Mantophasmatodea was described in 2002. Prior to that time,
      several generations of entomologists had assumed that all major insect taxa were 
      known; thus, its description was a sensation for zoologists. Since then, a
      surprising abundance and species diversity of this taxon have been found,
      particularly in the winter rainfall region of South Africa. To learn more about
      the evolutionary lineages, speciation, and biogeography of Mantophasmatodea, we
      applied an unusual peptidomics approach. We collected specimens of almost all
      known and novel taxa of these insects, developed methods for immediate sample
      preparation in the field, introduced peptide mass fingerprints for the
      unambiguous identification of taxa, and subsequently analyzed the most extensive 
      data set on peptide hormones ever compiled for insect taxa. To account for
      intraspecific variation, we analyzed several individuals per putative species.
      Increased taxon sampling was preferred over a further increase in the number of
      characters to optimize the accuracy of phylogenetic analyses. The large data set 
      made it possible to test the validity of using neuropeptide sequences, which
      coevolve with their respective receptors, to analyze phylogenetic relationships
      among closely related taxa. Altogether, the data from 71 populations of
      Mantophasmatodea were sufficient to clearly separate the major clades of
      Mantophasmatodea, including previously undescribed taxa such as Pachyphasma,
      Striatophasma, and Austrophasmatidae gen. et sp. nov. "RV." The data confirm the 
      monophyly of Austrophasmatidae and show a relatively recent and extensive
      radiation in the winter rainfall region of South Africa but also suggest that the
      species-level diversification of Namibian Mantophasma is less marked than
      previously thought. We discuss the biogeographical and ecological factors that
      may have resulted in different regional patterns of endemism and species
      diversity in Mantophasmatodea. The unique development of the neuroendocrine
      capa-neurons in the ventral nervous system is described as synapomorphy of
      Mantophasmatodea + Grylloblattodea and is a further argument for a close
      relationship between these insect taxa.
FAU - Predel, Reinhard
AU  - Predel R
AD  - Biocenter, Institute for Zoology, University of Cologne, Zulpicher Strasse 47b,
      50674 Cologne, Germany. reinhard.predel@uni-koeln.de
FAU - Neupert, Susanne
AU  - Neupert S
FAU - Huetteroth, Wolf
AU  - Huetteroth W
FAU - Kahnt, Jorg
AU  - Kahnt J
FAU - Waidelich, Dietmar
AU  - Waidelich D
FAU - Roth, Steffen
AU  - Roth S
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120416
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Insect Hormones)
RN  - 0 (Insect Proteins)
RN  - 0 (Neuropeptides)
SB  - IM
MH  - Amino Acid Sequence
MH  - Animals
MH  - *Evolution, Molecular
MH  - Insect Hormones/chemistry/genetics
MH  - Insect Proteins/chemistry/*genetics
MH  - Insecta/chemistry/*classification/*genetics
MH  - Molecular Sequence Data
MH  - Namibia
MH  - Neuropeptides/chemistry/*genetics
MH  - Peptide Mapping
MH  - *Phylogeography
MH  - Proteomics/methods
MH  - Sequence Alignment
MH  - South Africa
EDAT- 2012/04/18 06:00
MHDA- 2012/10/19 06:00
CRDT- 2012/04/18 06:00
PHST- 2012/04/18 06:00 [entrez]
PHST- 2012/04/18 06:00 [pubmed]
PHST- 2012/10/19 06:00 [medline]
AID - sys003 [pii]
AID - 10.1093/sysbio/sys003 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jul;61(4):609-29. doi: 10.1093/sysbio/sys003. Epub 2012 Apr 16.

PMID- 22492540
OWN - NLM
STAT- MEDLINE
DCOM- 20121018
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 4
DP  - 2012 Jul
TI  - Systematics Agenda 2020: the mission evolves.
PG  - 549-52
LID - 10.1093/sysbio/sys044 [doi]
FAU - Daly, Marymegan
AU  - Daly M
AD  - Department of Evolution, Ecology and Organismal Biology, The Ohio State
      University, 1315 Kinnear Road, Columbus, OH 43212, USA. daly.66@osu.edu
FAU - Herendeen, Patrick S
AU  - Herendeen PS
FAU - Guralnick, Robert P
AU  - Guralnick RP
FAU - Westneat, Mark W
AU  - Westneat MW
FAU - McDade, Lucinda
AU  - McDade L
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120403
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biodiversity
MH  - Biological Evolution
MH  - Biology/trends
MH  - *Classification
MH  - *Information Dissemination
MH  - Phylogeny
PMC - PMC3376376
EDAT- 2012/04/12 06:00
MHDA- 2012/10/19 06:00
CRDT- 2012/04/12 06:00
PHST- 2012/04/12 06:00 [entrez]
PHST- 2012/04/12 06:00 [pubmed]
PHST- 2012/10/19 06:00 [medline]
AID - sys044 [pii]
AID - 10.1093/sysbio/sys044 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jul;61(4):549-52. doi: 10.1093/sysbio/sys044. Epub 2012 Apr 3.

PMID- 22442193
OWN - NLM
STAT- MEDLINE
DCOM- 20130305
LR  - 20121024
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 6
DP  - 2012 Dec 1
TI  - Is the general time-reversible model bad for molecular phylogenetics?
PG  - 1069-74
LID - 10.1093/sysbio/sys042 [doi]
FAU - Sumner, Jeremy G
AU  - Sumner JG
AD  - School of Mathematics and Physics, University of Tasmania, Hobart 7001,
      Australia.jsumner@utas.edu.au
FAU - Jarvis, Peter D
AU  - Jarvis PD
FAU - Fernandez-Sanchez, Jesus
AU  - Fernandez-Sanchez J
FAU - Kaine, Bodie T
AU  - Kaine BT
FAU - Woodhams, Michael D
AU  - Woodhams MD
FAU - Holland, Barbara R
AU  - Holland BR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120322
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Models, Theoretical
MH  - *Phylogeny
EDAT- 2012/03/24 06:00
MHDA- 2013/03/06 06:00
CRDT- 2012/03/24 06:00
PHST- 2012/03/24 06:00 [entrez]
PHST- 2012/03/24 06:00 [pubmed]
PHST- 2013/03/06 06:00 [medline]
AID - sys042 [pii]
AID - 10.1093/sysbio/sys042 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Dec 1;61(6):1069-74. doi: 10.1093/sysbio/sys042. Epub 2012 Mar
      22.

PMID- 22438331
OWN - NLM
STAT- MEDLINE
DCOM- 20130305
LR  - 20121024
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 6
DP  - 2012 Dec 1
TI  - The pipid root.
PG  - 913-26
LID - 10.1093/sysbio/sys039 [doi]
AB  - The estimation of phylogenetic relationships is an essential component of
      understanding evolution. Accurate phylogenetic estimation is difficult, however, 
      when internodes are short and old, when genealogical discordance is common due to
      large ancestral effective population sizes or ancestral population structure, and
      when homoplasy is prevalent. Inference of divergence times is also hampered by
      unknown and uneven rates of evolution, the incomplete fossil record, uncertainty 
      in relationships between fossil and extant lineages, and uncertainty in the age
      of fossils. Ideally, these challenges can be overcome by developing large
      "phylogenomic" data sets and by analyzing them with methods that accommodate
      features of the evolutionary process, such as genealogical discordance, recurrent
      substitution, recombination, ancestral population structure, gene flow after
      speciation among sampled and unsampled taxa, and variation in evolutionary rates.
      In some phylogenetic problems, it is possible to use information that is
      independent of fossils, such as the geological record, to identify putative
      triggers for diversification whose associated estimated divergence times can then
      be compared a posteriori with estimated relationships and ages of fossils. The
      history of diversification of pipid frog genera Pipa, Hymenochirus, Silurana, and
      Xenopus, for instance, is characterized by many of these evolutionary and
      analytical challenges. These frogs diversified dozens of millions of years ago,
      they have a relatively rich fossil record, their distributions span continental
      plates with a well characterized geological record of ancient connectivity, and
      there is considerable disagreement across studies in estimated evolutionary
      relationships. We used high throughput sequencing and public databases to
      generate a large phylogenomic data set with which we estimated evolutionary
      relationships using multilocus coalescence methods. We collected sequence data
      from Pipa, Hymenochirus, Silurana, and Xenopus and the outgroup taxon
      Rhinophrynus dorsalis from coding sequence of 113 autosomal regions, averaging
      approximately 300 bp in length (range: 102-1695 bp) and also a portion of the
      mitochondrial genome. Analysis of these data using multiple approaches recovers
      strong support for the ((Xenopus, Silurana)(Pipa, Hymenochirus)) topology, and
      geologically calibrated divergence time estimates that are consistent with
      estimated ages and phylogenetic affinities of many fossils. These results provide
      new insights into the biogeography and chronology of pipid diversification during
      the breakup of Gondwanaland and illustrate how phylogenomic data may be necessary
      to tackle tough problems in molecular systematics. [Coalescence; gene tree;
      high-throughout sequencing; lineage sorting; pipid; species tree; Xenopus.].
FAU - Bewick, Adam J
AU  - Bewick AJ
AD  - Department of Biology, Life Sciences Building Room 328, McMaster University, 1280
      Main Street West Hamilton, ON L8S 4K1, Canada.
FAU - Chain, Frederic J J
AU  - Chain FJ
FAU - Heled, Joseph
AU  - Heled J
FAU - Evans, Ben J
AU  - Evans BJ
LA  - eng
PT  - Journal Article
DEP - 20120320
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Anura/*classification/*genetics
MH  - *Biological Evolution
MH  - *Models, Genetic
MH  - Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2012/03/23 06:00
MHDA- 2013/03/06 06:00
CRDT- 2012/03/23 06:00
PHST- 2012/03/23 06:00 [entrez]
PHST- 2012/03/23 06:00 [pubmed]
PHST- 2013/03/06 06:00 [medline]
AID - sys039 [pii]
AID - 10.1093/sysbio/sys039 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Dec 1;61(6):913-26. doi: 10.1093/sysbio/sys039. Epub 2012 Mar 20.

PMID- 22438330
OWN - NLM
STAT- MEDLINE
DCOM- 20130305
LR  - 20121024
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 6
DP  - 2012 Dec 1
TI  - Gaps: an elusive source of phylogenetic information.
PG  - 1075-82
LID - 10.1093/sysbio/sys043 [doi]
FAU - Saurabh, Kumar
AU  - Saurabh K
AD  - IMBS/IFS/INR, Massey University, Palmerston North 4442, New Zealand.
FAU - Holland, Barbara R
AU  - Holland BR
FAU - Gibb, Gillian C
AU  - Gibb GC
FAU - Penny, David
AU  - Penny D
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120321
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Birds/classification/genetics
MH  - Evolution, Molecular
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
EDAT- 2012/03/23 06:00
MHDA- 2013/03/06 06:00
CRDT- 2012/03/23 06:00
PHST- 2012/03/23 06:00 [entrez]
PHST- 2012/03/23 06:00 [pubmed]
PHST- 2013/03/06 06:00 [medline]
AID - sys043 [pii]
AID - 10.1093/sysbio/sys043 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Dec 1;61(6):1075-82. doi: 10.1093/sysbio/sys043. Epub 2012 Mar
      21.

PMID- 22398122
OWN - NLM
STAT- MEDLINE
DCOM- 20130305
LR  - 20121024
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 6
DP  - 2012 Dec 1
TI  - A unifying model for the analysis of phenotypic, genetic, and geographic data.
PG  - 897-911
LID - 10.1093/sysbio/sys038 [doi]
AB  - Recognition of evolutionary units (species, populations) requires integrating
      several kinds of data, such as genetic or phenotypic markers or spatial
      information in order to get a comprehensive view concerning the differentiation
      of the units. We propose a statistical model with a double original advantage:
      (i) it incorporates information about the spatial distribution of the samples,
      with the aim to increase inference power and to relate more explicitly observed
      patterns to geography and (ii) it allows one to analyze genetic and phenotypic
      data within a unified model and inference framework, thus opening the way to
      robust comparisons between markers and possibly combined analyses. We show from
      simulated data as well as real data that our method estimates parameters
      accurately and is an improvement over alternative approaches in many situations. 
      The power of this method is exemplified using an intricate case of inter- and
      intraspecies differentiation based on an original data set of georeferenced
      genetic and morphometric markers obtained on Myodes voles from Sweden. A computer
      program is made available as an extension of the R package Geneland.
FAU - Guillot, Gilles
AU  - Guillot G
AD  - Informatics Department, Technical University of Denmark, Richard Petersens Plads,
      2800 Lyngby, Denmark. gigu@im.dtu.dk
FAU - Renaud, Sabrina
AU  - Renaud S
FAU - Ledevin, Ronan
AU  - Ledevin R
FAU - Michaux, Johan
AU  - Michaux J
FAU - Claude, Julien
AU  - Claude J
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120306
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Arvicolinae/classification/genetics
MH  - *Genetic Variation
MH  - Geography
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - *Phenotype
MH  - Software
MH  - Species Specificity
EDAT- 2012/03/09 06:00
MHDA- 2013/03/06 06:00
CRDT- 2012/03/09 06:00
PHST- 2012/03/09 06:00 [entrez]
PHST- 2012/03/09 06:00 [pubmed]
PHST- 2013/03/06 06:00 [medline]
AID - sys038 [pii]
AID - 10.1093/sysbio/sys038 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Dec 1;61(6):897-911. doi: 10.1093/sysbio/sys038. Epub 2012 Mar 6.

PMID- 22398121
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - The effect of geographical scale of sampling on DNA barcoding.
PG  - 851-69
AB  - Eight years after DNA barcoding was formally proposed on a large scale, CO1
      sequences are rapidly accumulating from around the world. While studies to date
      have mostly targeted local or regional species assemblages, the recent launch of 
      the global iBOL project (International Barcode of Life), highlights the need to
      understand the effects of geographical scale on Barcoding's goals. Sampling has
      been central in the debate on DNA Barcoding, but the effect of the geographical
      scale of sampling has not yet been thoroughly and explicitly tested with
      empirical data. Here, we present a CO1 data set of aquatic predaceous diving
      beetles of the tribe Agabini, sampled throughout Europe, and use it to
      investigate how the geographic scale of sampling affects 1) the estimated
      intraspecific variation of species, 2) the genetic distance to the most closely
      related heterospecific, 3) the ratio of intraspecific and interspecific
      variation, 4) the frequency of taxonomically recognized species found to be
      monophyletic, and 5) query identification performance based on 6 different
      species assignment methods. Intraspecific variation was significantly correlated 
      with the geographical scale of sampling (R-square = 0.7), and more than half of
      the species with 10 or more sampled individuals (N = 29) showed higher
      intraspecific variation than 1% sequence divergence. In contrast, the distance to
      the closest heterospecific showed a significant decrease with increasing
      geographical scale of sampling. The average genetic distance dropped from &gt; 7%
      for samples within 1 km, to &lt; 3.5% for samples up to &gt; 6000 km apart. Over a
      third of the species were not monophyletic, and the proportion increased through 
      locally, nationally, regionally, and continentally restricted subsets of the
      data. The success of identifying queries decreased with increasing spatial scale 
      of sampling; liberal methods declined from 100% to around 90%, whereas strict
      methods dropped to below 50% at continental scales. The proportion of query
      identifications considered uncertain (more than one species &lt; 1% distance from
      query) escalated from zero at local, to 50% at continental scale. Finally, by
      resampling the most widely sampled species we show that even if samples are
      collected to maximize the geographical coverage, up to 70 individuals are
      required to sample 95% of intraspecific variation. The results show that the
      geographical scale of sampling has a critical impact on the global application of
      DNA barcoding. Scale-effects result from the relative importance of different
      processes determining the composition of regional species assemblages (dispersal 
      and ecological assembly) and global clades (demography, speciation, and
      extinction). The incorporation of geographical information, where available, will
      be required to obtain identification rates at global scales equivalent to those
      in regional barcoding studies. Our result hence provides an impetus for both
      smarter barcoding tools and sprouting national barcoding initiatives-smaller
      geographical scales deliver higher accuracy.
FAU - Bergsten, Johannes
AU  - Bergsten J
AD  - Department of Entomology, Swedish Museum of Natural History, Stockholm, Sweden.
      johannes.bergsten@nrm.se
FAU - Bilton, David T
AU  - Bilton DT
FAU - Fujisawa, Tomochika
AU  - Fujisawa T
FAU - Elliott, Miranda
AU  - Elliott M
FAU - Monaghan, Michael T
AU  - Monaghan MT
FAU - Balke, Michael
AU  - Balke M
FAU - Hendrich, Lars
AU  - Hendrich L
FAU - Geijer, Joja
AU  - Geijer J
FAU - Herrmann, Jan
AU  - Herrmann J
FAU - Foster, Garth N
AU  - Foster GN
FAU - Ribera, Ignacio
AU  - Ribera I
FAU - Nilsson, Anders N
AU  - Nilsson AN
FAU - Barraclough, Timothy G
AU  - Barraclough TG
FAU - Vogler, Alfried P
AU  - Vogler AP
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120307
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Insect Proteins)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Coleoptera/*genetics
MH  - DNA Barcoding, Taxonomic/*methods
MH  - DNA, Mitochondrial/genetics
MH  - Electron Transport Complex IV/genetics
MH  - Europe
MH  - Evolution, Molecular
MH  - *Genetic Variation
MH  - Geography
MH  - Insect Proteins/genetics
MH  - Iran
MH  - Molecular Sequence Data
MH  - Morocco
MH  - Phylogeny
MH  - Phylogeography/*methods
MH  - Polymerase Chain Reaction
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
PMC - PMC3417044
EDAT- 2012/03/09 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/03/09 06:00
PHST- 2012/03/09 06:00 [entrez]
PHST- 2012/03/09 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys037 [pii]
AID - 10.1093/sysbio/sys037 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):851-69. doi: 10.1093/sysbio/sys037. Epub 2012 Mar 7.

PMID- 22389443
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20120813
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - Phylogenetic signal and noise: predicting the power of a data set to resolve
      phylogeny.
PG  - 835-49
LID - 10.1093/sysbio/sys036 [doi]
AB  - A principal objective for phylogenetic experimental design is to predict the
      power of a data set to resolve nodes in a phylogenetic tree. However, proactively
      assessing the potential for phylogenetic noise compared with signal in a
      candidate data set has been a formidable challenge. Understanding the impact of
      collection of additional sequence data to resolve recalcitrant internodes at
      diverse historical times will facilitate increasingly accurate and cost-effective
      phylogenetic research. Here, we derive theory based on the fundamental unit of
      the phylogenetic tree, the quartet, that applies estimates of the state space and
      the rates of evolution of characters in a data set to predict phylogenetic signal
      and phylogenetic noise and therefore to predict the power to resolve internodes. 
      We develop and implement a Monte Carlo approach to estimating power to resolve as
      well as deriving a nearly equivalent faster deterministic calculation. These
      approaches are applied to describe the distribution of potential signal,
      polytomy, or noise for two example data sets, one recent (cytochrome c oxidase I 
      and 28S ribosomal rRNA sequences from Diplazontinae parasitoid wasps) and one
      deep (eight nuclear genes and a phylogenomic sequence for diverse microbial
      eukaryotes including Stramenopiles, Alveolata, and Rhizaria). The predicted power
      of resolution for the loci analyzed is consistent with the historic use of the
      genes in phylogenetics.
FAU - Townsend, Jeffrey P
AU  - Townsend JP
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT
      06520, USA. Jeffrey.Townsend@Yale.edu
FAU - Su, Zhuo
AU  - Su Z
FAU - Tekle, Yonas I
AU  - Tekle YI
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120303
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Insect Proteins)
RN  - 0 (Proteins)
RN  - 0 (RNA, Ribosomal, 28S)
RN  - 0 (Ribosomal Proteins)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Alveolata/classification/*genetics
MH  - Animals
MH  - Cell Nucleus/genetics
MH  - Classification/*methods
MH  - Electron Transport Complex IV/genetics
MH  - Genes, Insect
MH  - Insect Proteins/genetics
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Proteins/genetics
MH  - RNA, Ribosomal, 28S/genetics
MH  - Ribosomal Proteins/genetics
MH  - Sequence Analysis, DNA
MH  - Wasps/classification/*genetics
EDAT- 2012/03/06 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/03/06 06:00
PHST- 2012/03/06 06:00 [entrez]
PHST- 2012/03/06 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys036 [pii]
AID - 10.1093/sysbio/sys036 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):835-49. doi: 10.1093/sysbio/sys036. Epub 2012 Mar 3.

PMID- 22382300
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20120813
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - Species and phylogenetic nomenclature.
PG  - 885-91
LID - 10.1093/sysbio/sys035 [doi]
FAU - Cellinese, Nico
AU  - Cellinese N
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL
      32611-7800, USA. ncellinese@flmnh.ufl.edu
FAU - Baum, David A
AU  - Baum DA
FAU - Mishler, Brent D
AU  - Mishler BD
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120229
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biodiversity
MH  - Classification/*methods
MH  - Phylogeny
MH  - *Terminology as Topic
EDAT- 2012/03/03 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/03/03 06:00
PHST- 2012/03/03 06:00 [entrez]
PHST- 2012/03/03 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys035 [pii]
AID - 10.1093/sysbio/sys035 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):885-91. doi: 10.1093/sysbio/sys035. Epub 2012 Feb 29.

PMID- 22357728
OWN - NLM
STAT- MEDLINE
DCOM- 20121018
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 4
DP  - 2012 Jul
TI  - NeXML: rich, extensible, and verifiable representation of comparative data and
      metadata.
PG  - 675-89
LID - 10.1093/sysbio/sys025 [doi]
AB  - In scientific research, integration and synthesis require a common understanding 
      of where data come from, how much they can be trusted, and what they may be used 
      for. To make such an understanding computer-accessible requires standards for
      exchanging richly annotated data. The challenges of conveying reusable data are
      particularly acute in regard to evolutionary comparative analysis, which
      comprises an ever-expanding list of data types, methods, research aims, and
      subdisciplines. To facilitate interoperability in evolutionary comparative
      analysis, we present NeXML, an XML standard (inspired by the current standard,
      NEXUS) that supports exchange of richly annotated comparative data. NeXML defines
      syntax for operational taxonomic units, character-state matrices, and
      phylogenetic trees and networks. Documents can be validated unambiguously.
      Importantly, any data element can be annotated, to an arbitrary degree of
      richness, using a system that is both flexible and rigorous. We describe how the 
      use of NeXML by the TreeBASE and Phenoscape projects satisfies user needs that
      cannot be satisfied with other available file formats. By relying on XML Schema
      Definition, the design of NeXML facilitates the development and deployment of
      software for processing, transforming, and querying documents. The adoption of
      NeXML for practical use is facilitated by the availability of (1) an online
      manual with code samples and a reference to all defined elements and attributes, 
      (2) programming toolkits in most of the languages used commonly in evolutionary
      informatics, and (3) input-output support in several widely used software
      applications. An active, open, community-based development process enables future
      revision and expansion of NeXML.
FAU - Vos, Rutger A
AU  - Vos RA
AD  - NCB Naturalis, Leiden, The Netherlands. rutger.vos@ncbnaturalis.nl
FAU - Balhoff, James P
AU  - Balhoff JP
FAU - Caravas, Jason A
AU  - Caravas JA
FAU - Holder, Mark T
AU  - Holder MT
FAU - Lapp, Hilmar
AU  - Lapp H
FAU - Maddison, Wayne P
AU  - Maddison WP
FAU - Midford, Peter E
AU  - Midford PE
FAU - Priyam, Anurag
AU  - Priyam A
FAU - Sukumaran, Jeet
AU  - Sukumaran J
FAU - Xia, Xuhua
AU  - Xia X
FAU - Stoltzfus, Arlin
AU  - Stoltzfus A
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
DEP - 20120222
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biodiversity
MH  - *Biological Evolution
MH  - Classification
MH  - Computational Biology/*standards
MH  - Informatics
MH  - Models, Biological
MH  - Phylogeny
MH  - *Programming Languages
MH  - Software
PMC - PMC3376374
EDAT- 2012/02/24 06:00
MHDA- 2012/10/19 06:00
CRDT- 2012/02/24 06:00
PHST- 2012/02/24 06:00 [entrez]
PHST- 2012/02/24 06:00 [pubmed]
PHST- 2012/10/19 06:00 [medline]
AID - sys025 [pii]
AID - 10.1093/sysbio/sys025 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jul;61(4):675-89. doi: 10.1093/sysbio/sys025. Epub 2012 Feb 22.

PMID- 22357727
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a 
      large model space.
PG  - 539-42
LID - 10.1093/sysbio/sys029 [doi]
AB  - Since its introduction in 2001, MrBayes has grown in popularity as a software
      package for Bayesian phylogenetic inference using Markov chain Monte Carlo (MCMC)
      methods. With this note, we announce the release of version 3.2, a major upgrade 
      to the latest official release presented in 2003. The new version provides
      convergence diagnostics and allows multiple analyses to be run in parallel with
      convergence progress monitored on the fly. The introduction of new proposals and 
      automatic optimization of tuning parameters has improved convergence for many
      problems. The new version also sports significantly faster likelihood
      calculations through streaming single-instruction-multiple-data extensions (SSE) 
      and support of the BEAGLE library, allowing likelihood calculations to be
      delegated to graphics processing units (GPUs) on compatible hardware. Speedup
      factors range from around 2 with SSE code to more than 50 with BEAGLE for codon
      problems. Checkpointing across all models allows long runs to be completed even
      when an analysis is prematurely terminated. New models include relaxed clocks,
      dating, model averaging across time-reversible substitution models, and support
      for hard, negative, and partial (backbone) tree constraints. Inference of species
      trees from gene trees is supported by full incorporation of the Bayesian
      estimation of species trees (BEST) algorithms. Marginal model likelihoods for
      Bayes factor tests can be estimated accurately across the entire model space
      using the stepping stone method. The new version provides more output options
      than previously, including samples of ancestral states, site rates, site
      d(N)/d(S) rations, branch rates, and node dates. A wide range of statistics on
      tree parameters can also be output for visualization in FigTree and compatible
      software.
FAU - Ronquist, Fredrik
AU  - Ronquist F
AD  - Department of Biodiversity Informatics, Swedish Museum of Natural History,
      SE-10405 Stockholm, Sweden. fredrik.ronquist@nrm.se
FAU - Teslenko, Maxim
AU  - Teslenko M
FAU - van der Mark, Paul
AU  - van der Mark P
FAU - Ayres, Daniel L
AU  - Ayres DL
FAU - Darling, Aaron
AU  - Darling A
FAU - Hohna, Sebastian
AU  - Hohna S
FAU - Larget, Bret
AU  - Larget B
FAU - Liu, Liang
AU  - Liu L
FAU - Suchard, Marc A
AU  - Suchard MA
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
LA  - eng
GR  - R01 GM086887/GM/NIGMS NIH HHS/United States
GR  - R01 HG006139/HG/NHGRI NIH HHS/United States
GR  - HG-006139/HG/NHGRI NIH HHS/United States
GR  - GM-086887/GM/NIGMS NIH HHS/United States
GR  - GM-069801/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120222
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Markov Chains
MH  - Models, Biological
MH  - Monte Carlo Method
MH  - Phylogeny
MH  - *Software
PMC - PMC3329765
EDAT- 2012/02/24 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/02/24 06:00
PHST- 2012/02/24 06:00 [entrez]
PHST- 2012/02/24 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys029 [pii]
AID - 10.1093/sysbio/sys029 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):539-42. doi: 10.1093/sysbio/sys029. Epub 2012 Feb 22.

PMID- 22357726
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - Distribution models and a dated phylogeny for Chilean Oxalis species reveal
      occupation of new habitats by different lineages, not rapid adaptive radiation.
PG  - 823-34
LID - 10.1093/sysbio/sys034 [doi]
AB  - Among the World's most challenging environments for plant life is the Atacama
      Desert, an arid zone extending over 1300 km and from sea level to 2000/3000 m
      altitude along the southwestern Andean foothills. Plants there and in the
      adjacent Mediterranean zone exhibit striking adaptations, and we here address the
      question whether in a species-rich clade such adaptations arose in parallel, at
      different times, or simultaneously. Answering this type of question has been a
      major concern of evolutionary biology over the past few years, with a growing
      consensus that lineages tend to be conservative in their vegetative traits and
      niche requirements. Combined nuclear and chloroplast DNA sequences for 112
      species of Oxalidales (4900 aligned nucleotides) were used for a
      fossil-calibrated phylogeny that includes 43 of the 54 species of Chilean Oxalis,
      and species distribution models (SDMs) incorporating precipitation, temperature, 
      and fog, and the phylogeny were used to reconstruct ancestral habitat
      preferences, relying on likelihood and Bayesian techniques. Since uneven
      collecting can reduce the power of SDMs, we compared 3 strategies to correct for 
      collecting effort. Unexpectedly, the Oxalis flora of Chile consists of 7 distant 
      lineages that originated at different times prior to the last Andean uplift
      pulse; some had features preadapting them to seasonally arid or xeric conditions.
      Models that incorporated fog and a "collecting activity surface" performed best
      and identified the Mediterranean zone as a hotspot of Oxalis species as well as
      lineage diversity because it harbors a mix of ancient and young groups, including
      insufficiently arid-adapted species. There is no evidence of rapid adaptive
      radiation.
FAU - Heibl, Christoph
AU  - Heibl C
AD  - Systematic Botany and Mycology, University of Munich, Menzinger Strasse 67, 80638
      Munich, Germany.
FAU - Renner, Susanne S
AU  - Renner SS
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120222
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Chloroplast Proteins)
RN  - 0 (Plant Proteins)
SB  - IM
MH  - Cell Nucleus/genetics
MH  - Chile
MH  - Chloroplast Proteins/genetics
MH  - *Ecosystem
MH  - *Evolution, Molecular
MH  - Evolution, Planetary
MH  - Magnoliopsida/classification/*genetics
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Plant Proteins/*genetics
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
EDAT- 2012/02/24 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/02/24 06:00
PHST- 2012/02/24 06:00 [entrez]
PHST- 2012/02/24 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys034 [pii]
AID - 10.1093/sysbio/sys034 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):823-34. doi: 10.1093/sysbio/sys034. Epub 2012 Feb 22.

PMID- 22337142
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20120813
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - The probability of correctly resolving a split as an experimental design
      criterion in phylogenetics.
PG  - 811-21
LID - 10.1093/sysbio/sys033 [doi]
AB  - We illustrate how recently developed large sequence-length approximations to
      probabilities of correct phylogenetic reconstruction for maximum likelihood
      estimation can be used to evaluate experimental design strategies. The specific
      criterion of interest is the probability of correctly resolving an a priori
      defined split of interest in a phylogenetic tree. Design strategies considered
      include increased taxon sampling and increasing sequence length. Our analyses of 
      specific examples strongly suggest that it is better to sample taxa that connect 
      as close as possible to the split of interest. Assuming this can be done, these
      examples suggest it is better to sample additional taxa than to add a comparable 
      number of sites for the existing taxa. If the rates of evolution in the added
      taxa are slow, it is better to choose taxa connecting to a long edge, but if
      rates are comparable to a sister lineage, it is not necessarily the best strategy
      to sample taxa connected to a long edge. We also examined deleting taxa while
      increasing the number of sites. Although deleting a small number of taxa distant 
      from the split of interest can be beneficial, deleting too many or making poor
      choices as to what should be deleted can lead to smaller probabilities of correct
      reconstruction than for the original sequence data.
FAU - Susko, Edward
AU  - Susko E
AD  - Department of Mathematics and Statistics, Dalhousie University, Halifax, Nova
      Scotia, Canada B3H 4H7. susko@mathstat.dal.ca
FAU - Roger, Andrew J
AU  - Roger AJ
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120215
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Embryophyta/classification/genetics
MH  - *Evolution, Molecular
MH  - Genes, Plant
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - Probability
MH  - Research Design
EDAT- 2012/02/18 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/02/17 06:00
PHST- 2012/02/17 06:00 [entrez]
PHST- 2012/02/18 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys033 [pii]
AID - 10.1093/sysbio/sys033 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):811-21. doi: 10.1093/sysbio/sys033. Epub 2012 Feb 15.

PMID- 22334344
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20120813
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - A new method for handling missing species in diversification analysis applicable 
      to randomly or nonrandomly sampled phylogenies.
PG  - 785-92
LID - 10.1093/sysbio/sys031 [doi]
AB  - Chronograms from molecular dating are increasingly being used to infer rates of
      diversification and their change over time. A major limitation in such analyses
      is incomplete species sampling that moreover is usually nonrandom. While the
      widely used gamma statistic with the Monte Carlo constant-rates test or the
      birth-death likelihood analysis with the delta AICrc test statistic are
      appropriate for comparing the fit of different diversification models in
      phylogenies with random species sampling, no objective automated method has been 
      developed for fitting diversification models to nonrandomly sampled phylogenies. 
      Here, we introduce a novel approach, CorSiM, which involves simulating missing
      splits under a constant rate birth-death model and allows the user to specify
      whether species sampling in the phylogeny being analyzed is random or nonrandom. 
      The completed trees can be used in subsequent model-fitting analyses. This is
      fundamentally different from previous diversification rate estimation methods,
      which were based on null distributions derived from the incomplete trees. CorSiM 
      is automated in an R package and can easily be applied to large data sets. We
      illustrate the approach in two Araceae clades, one with a random species sampling
      of 52% and one with a nonrandom sampling of 55%. In the latter clade, the CorSiM 
      approach detects and quantifies an increase in diversification rate, whereas
      classic approaches prefer a constant rate model; in the former clade, results do 
      not differ among methods (as indeed expected since the classic approaches are
      valid only for randomly sampled phylogenies). The CorSiM method greatly reduces
      the type I error in diversification analysis, but type II error remains a
      methodological problem.
FAU - Cusimano, Natalie
AU  - Cusimano N
AD  - Systematic Botany and Mycology, University of Munich, Menzinger Str. 67, 80638
      Munich, German.
FAU - Stadler, Tanja
AU  - Stadler T
FAU - Renner, Susanne S
AU  - Renner SS
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120214
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Plant Proteins)
SB  - IM
MH  - Araceae/classification/*genetics
MH  - Asia, Southeastern
MH  - Cell Nucleus/genetics
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Mediterranean Region
MH  - *Phylogeny
MH  - Plant Proteins/*genetics
EDAT- 2012/02/16 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/02/16 06:00
PHST- 2012/02/16 06:00 [entrez]
PHST- 2012/02/16 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys031 [pii]
AID - 10.1093/sysbio/sys031 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):785-92. doi: 10.1093/sysbio/sys031. Epub 2012 Feb 14.

PMID- 22334343
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - A hierarchical Bayesian model for calibrating estimates of species divergence
      times.
PG  - 793-809
LID - 10.1093/sysbio/sys032 [doi]
AB  - In Bayesian divergence time estimation methods, incorporating calibrating
      information from the fossil record is commonly done by assigning prior densities 
      to ancestral nodes in the tree. Calibration prior densities are typically
      parametric distributions offset by minimum age estimates provided by the fossil
      record. Specification of the parameters of calibration densities requires the
      user to quantify his or her prior knowledge of the age of the ancestral node
      relative to the age of its calibrating fossil. The values of these parameters
      can, potentially, result in biased estimates of node ages if they lead to overly 
      informative prior distributions. Accordingly, determining parameter values that
      lead to adequate prior densities is not straightforward. In this study, I present
      a hierarchical Bayesian model for calibrating divergence time analyses with
      multiple fossil age constraints. This approach applies a Dirichlet process prior 
      as a hyperprior on the parameters of calibration prior densities. Specifically,
      this model assumes that the rate parameters of exponential prior distributions on
      calibrated nodes are distributed according to a Dirichlet process, whereby the
      rate parameters are clustered into distinct parameter categories. Both simulated 
      and biological data are analyzed to evaluate the performance of the Dirichlet
      process hyperprior. Compared with fixed exponential prior densities, the
      hierarchical Bayesian approach results in more accurate and precise estimates of 
      internal node ages. When this hyperprior is applied using Markov chain Monte
      Carlo methods, the ages of calibrated nodes are sampled from mixtures of
      exponential distributions and uncertainty in the values of calibration density
      parameters is taken into account.
FAU - Heath, Tracy A
AU  - Heath TA
AD  - Department of Integrative Biology, University of California, Berkeley, CA 94720, 
      USA. tracyh@berkeley.edu
LA  - eng
GR  - R01 GM069801/GM/NIGMS NIH HHS/United States
GR  - R01 GM086887/GM/NIGMS NIH HHS/United States
GR  - GM-069801/GM/NIGMS NIH HHS/United States
GR  - GM-086887/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120214
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Mitochondrial Proteins)
SB  - IM
MH  - Animals
MH  - *Bayes Theorem
MH  - Cell Nucleus/genetics
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - *Fossils
MH  - Markov Chains
MH  - Mitochondrial Proteins/genetics
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Turtles/classification/genetics
PMC - PMC3417043
EDAT- 2012/02/16 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/02/16 06:00
PHST- 2012/02/16 06:00 [entrez]
PHST- 2012/02/16 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys032 [pii]
AID - 10.1093/sysbio/sys032 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):793-809. doi: 10.1093/sysbio/sys032. Epub 2012 Feb 14.

PMID- 22334342
OWN - NLM
STAT- MEDLINE
DCOM- 20121018
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 4
DP  - 2012 Jul
TI  - Turning the crown upside down: gene tree parsimony roots the eukaryotic tree of
      life.
PG  - 653-60
LID - 10.1093/sysbio/sys026 [doi]
AB  - The first analyses of gene sequence data indicated that the eukaryotic tree of
      life consisted of a long stem of microbial groups "topped" by a crown-containing 
      plants, animals, and fungi and their microbial relatives. Although more recent
      multigene concatenated analyses have refined the relationships among the many
      branches of eukaryotes, the root of the eukaryotic tree of life has remained
      elusive. Inferring the root of extant eukaryotes is challenging because of the
      age of the group ( approximately 1.7-2.1 billion years old), tremendous
      heterogeneity in rates of evolution among lineages, and lack of obvious outgroups
      for many genes. Here, we reconstruct a rooted phylogeny of extant eukaryotes
      based on minimizing the number of duplications and losses among a collection of
      gene trees. This approach does not require outgroup sequences or assumptions of
      orthology among sequences. We also explore the impact of taxon and gene sampling 
      and assess support for alternative hypotheses for the root. Using 20 gene trees
      from 84 diverse eukaryotic lineages, this approach recovers robust eukaryotic
      clades and reveals evidence for a eukaryotic root that lies between the
      Opisthokonta (animals, fungi and their microbial relatives) and all remaining
      eukaryotes.
FAU - Katz, Laura A
AU  - Katz LA
AD  - Department of Biological Sciences, Smith College, 44 College Lane, Northampton,
      MA 01063, USA. lkatz@smith.edu
FAU - Grant, Jessica R
AU  - Grant JR
FAU - Parfrey, Laura Wegener
AU  - Parfrey LW
FAU - Burleigh, J Gordon
AU  - Burleigh JG
LA  - eng
GR  - 1R15GM081865-01/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120214
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - DNA, Ribosomal/genetics
MH  - Eukaryota/*genetics
MH  - *Evolution, Molecular
MH  - Gene Duplication
MH  - Genome
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
PMC - PMC3376375
EDAT- 2012/02/16 06:00
MHDA- 2012/10/19 06:00
CRDT- 2012/02/16 06:00
PHST- 2012/02/16 06:00 [entrez]
PHST- 2012/02/16 06:00 [pubmed]
PHST- 2012/10/19 06:00 [medline]
AID - sys026 [pii]
AID - 10.1093/sysbio/sys026 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jul;61(4):653-60. doi: 10.1093/sysbio/sys026. Epub 2012 Feb 14.

PMID- 22331438
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20120420
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - Accurate estimation of substitution rates with neighbor-dependent models in a
      phylogenetic context.
PG  - 510-21
LID - 10.1093/sysbio/sys024 [doi]
AB  - Most models and algorithms developed to perform statistical inference from DNA
      data make the assumption that substitution processes affecting distinct
      nucleotide sites are stochastically independent. This assumption ensures both
      mathematical and computational tractability but is in disagreement with observed 
      data in many situations--one well-known example being CpG dinucleotide
      hypermutability in mammalian genomes. In this paper, we consider the class of
      RN95 + YpR substitution models, which allows neighbor-dependent
      effects--including CpG hypermutability--to be taken into account, through
      transitions between pyrimidine-purine dinucleotides. We show that it is possible 
      to adapt inference methods originally developed under the assumption of
      independence between sites to RN95 + YpR models, using a mathematically rigorous 
      framework provided by specific structural properties of this class of models. We 
      assess how efficient this approach is at inferring the CpG hypermutability rate
      from aligned DNA sequences. The method is tested on simulated data and compared
      against several alternatives; the results suggest that it delivers a high degree 
      of accuracy at a low computational cost. We then apply our method to an alignment
      of 10 DNA sequences from primate species. Model comparisons within the RN95 + YpR
      class show the importance of taking into account neighbor-dependent effects. An
      application of the method to the detection of hypomethylated islands is
      discussed.
FAU - Berard, Jean
AU  - Berard J
AD  - Institut Camille Jordan, UMR CNRS 5208, Universite Lyon 1, Villeurbanne F-69622
      Cedex, Universite de Lyon, Lyon 69003, France.
FAU - Gueguen, Laurent
AU  - Gueguen L
LA  - eng
PT  - Journal Article
DEP - 20120213
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Humans
MH  - *Models, Biological
MH  - Phylogeny
MH  - Primates/classification/genetics
EDAT- 2012/02/15 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/02/15 06:00
PHST- 2012/02/15 06:00 [entrez]
PHST- 2012/02/15 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys024 [pii]
AID - 10.1093/sysbio/sys024 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):510-21. doi: 10.1093/sysbio/sys024. Epub 2012 Feb 13.

PMID- 22328570
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20120813
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - Robustness of compound Dirichlet priors for Bayesian inference of branch lengths.
PG  - 779-84
AB  - We modified the phylogenetic program MrBayes 3.1.2 to incorporate the compound
      Dirichlet priors for branch lengths proposed recently by Rannala, Zhu, and Yang
      (2012. Tail paradox, partial identifiability and influential priors in Bayesian
      branch length inference. Mol. Biol. Evol. 29:325-335.) as a solution to the
      problem of branch-length overestimation in Bayesian phylogenetic inference. The
      compound Dirichlet prior specifies a fairly diffuse prior on the tree length (the
      sum of branch lengths) and uses a Dirichlet distribution to partition the tree
      length into branch lengths. Six problematic data sets originally analyzed by
      Brown, Hedtke, Lemmon, and Lemmon (2010. When trees grow too long: investigating 
      the causes of highly inaccurate Bayesian branch-length estimates. Syst. Biol.
      59:145-161) are reanalyzed using the modified version of MrBayes to investigate
      properties of Bayesian branch-length estimation using the new priors. While the
      default exponential priors for branch lengths produced extremely long trees, the 
      compound Dirichlet priors produced posterior estimates that are much closer to
      the maximum likelihood estimates. Furthermore, the posterior tree lengths were
      quite robust to changes in the parameter values in the compound Dirichlet priors,
      for example, when the prior mean of tree length changed over several orders of
      magnitude. Our results suggest that the compound Dirichlet priors may be useful
      for correcting branch-length overestimation in phylogenetic analyses of empirical
      data sets.
FAU - Zhang, Chi
AU  - Zhang C
AD  - Center for Computational and Evolutionary Biology, Institute of Zoology, Chinese 
      Academy of Sciences, Beijing 100101, China.
FAU - Rannala, Bruce
AU  - Rannala B
FAU - Yang, Ziheng
AU  - Yang Z
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Anura/classification
MH  - *Bayes Theorem
MH  - Bivalvia/classification
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Lizards/classification
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2012/02/14 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/02/14 06:00
PHST- 2012/02/14 06:00 [entrez]
PHST- 2012/02/14 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys030 [pii]
AID - 10.1093/sysbio/sys030 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):779-84. doi: 10.1093/sysbio/sys030. Epub 2012 Feb 10.

PMID- 22328569
OWN - NLM
STAT- MEDLINE
DCOM- 20121018
LR  - 20120618
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 4
DP  - 2012 Jul
TI  - Phylogeny and biogeography of the core babblers (Aves: Timaliidae).
PG  - 631-51
LID - 10.1093/sysbio/sys027 [doi]
AB  - The avian family Timaliidae is a species rich and morphologically diverse
      component of African and Asian tropical forests. The morphological diversity
      within the family has attracted interest from ecologists and evolutionary
      biologists, but systematists have long suspected that this diversity might also
      mislead taxonomy, and recent molecular phylogenetic work has supported this
      hypothesis. We produced and analyzed a data set of 6 genes and almost 300
      individuals to assess the evolutionary history of the family. Although
      phylogenetic analysis required extensive adjustment of program settings, we
      ultimately produced a well-resolved phylogeny for the family. The resulting
      phylogeny provided strong support for major subclades within the family but
      extensive paraphyly of genera. Only 3 genera represented by more than 3 species
      were monophyletic. Biogeographic reconstruction indicated a mainland Asian origin
      for the family and most major clades. Colonization of Africa, Sundaland, and the 
      Philippines occurred relatively late in the family's history and was mostly
      unidirectional. Several putative babbler genera, such as Robsonius, Malia,
      Leonardina, and Micromacronus are only distantly related to the Timaliidae.
FAU - Moyle, Robert G
AU  - Moyle RG
AD  - Biodiversity Institute and Department of Ecology and Evolutionary Biology,
      University of Kansas, Lawrence, KS 66045-7561, USA. moyle@ku.edu
FAU - Andersen, Michael J
AU  - Andersen MJ
FAU - Oliveros, Carl H
AU  - Oliveros CH
FAU - Steinheimer, Frank D
AU  - Steinheimer FD
FAU - Reddy, Sushma
AU  - Reddy S
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Africa
MH  - Animals
MH  - Asia
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - Cell Nucleus/*genetics
MH  - DNA, Mitochondrial/*genetics
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeography
MH  - Sequence Analysis, DNA
MH  - Songbirds/classification/*genetics
EDAT- 2012/02/14 06:00
MHDA- 2012/10/19 06:00
CRDT- 2012/02/14 06:00
PHST- 2012/02/14 06:00 [entrez]
PHST- 2012/02/14 06:00 [pubmed]
PHST- 2012/10/19 06:00 [medline]
AID - sys027 [pii]
AID - 10.1093/sysbio/sys027 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jul;61(4):631-51. doi: 10.1093/sysbio/sys027. Epub 2012 Feb 10.

PMID- 22328568
OWN - NLM
STAT- MEDLINE
DCOM- 20121018
LR  - 20120618
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 4
DP  - 2012 Jul
TI  - Experimental design in phylogenetics: testing predictions from expected
      information.
PG  - 661-74
LID - 10.1093/sysbio/sys028 [doi]
AB  - Taxon and character sampling are central to phylogenetic experimental design;
      yet, we lack general rules. Goldman introduced a method to construct efficient
      sampling designs in phylogenetics, based on the calculation of expected Fisher
      information given a probabilistic model of sequence evolution. The considerable
      potential of this approach remains largely unexplored. In an earlier study, we
      applied Goldman's method to a problem in the phylogenetics of caecilian
      amphibians and made an a priori evaluation and testable predictions of which
      taxon additions would increase information about a particular weakly supported
      branch of the caecilian phylogeny by the greatest amount. We have now gathered
      mitogenomic and rag1 sequences (some newly determined for this study) from
      additional caecilian species and studied how information (both expected and
      observed) and bootstrap support vary as each new taxon is individually added to
      our previous data set. This provides the first empirical test of specific
      predictions made using Goldman's method for phylogenetic experimental design. Our
      results empirically validate the top 3 (more intuitive) taxon addition
      predictions made in our previous study, but only information results validate
      unambiguously the 4th (less intuitive) prediction. This highlights a complex
      relationship between information and support, reflecting that each measures
      different things: Information is related to the ability to estimate branch length
      accurately and support to the ability to estimate the tree topology accurately.
      Thus, an increase in information may be correlated with but does not necessitate 
      an increase in support. Our results also provide the first empirical validation
      of the widely held intuition that additional taxa that join the tree proximal to 
      poorly supported internal branches are more informative and enhance support more 
      than additional taxa that join the tree more distally. Our work supports the view
      that adding more data for a single (well chosen) taxon may increase phylogenetic 
      resolution and support in weakly supported parts of the tree without adding more 
      characters/genes. Altogether our results corroborate that, although still
      underexplored, Goldman's method offers a powerful tool for experimental design in
      molecular phylogenetic studies. However, there are still several drawbacks to
      overcome, and further assessment of the method is needed in order to make it
      better understood, more accessible, and able to assess the addition of multiple
      taxa.
FAU - San Mauro, Diego
AU  - San Mauro D
AD  - Department of Zoology, The Natural History Museum, Cromwell Road, London SW7 5BD,
      UK. dsanmauro@ub.edu
FAU - Gower, David J
AU  - Gower DJ
FAU - Cotton, James A
AU  - Cotton JA
FAU - Zardoya, Rafael
AU  - Zardoya R
FAU - Wilkinson, Mark
AU  - Wilkinson M
FAU - Massingham, Tim
AU  - Massingham T
LA  - eng
SI  - GENBANK/HQ444127
SI  - GENBANK/HQ444128
SI  - GENBANK/HQ444129
SI  - GENBANK/HQ444130
SI  - GENBANK/HQ444131
SI  - GENBANK/HQ444132
SI  - GENBANK/HQ444133
SI  - GENBANK/HQ444134
SI  - GENBANK/HQ444135
SI  - GENBANK/HQ444136
SI  - GENBANK/HQ444137
SI  - GENBANK/HQ444138
SI  - GENBANK/HQ444139
SI  - GENBANK/HQ456774
SI  - GENBANK/JN089397
SI  - GENBANK/JN089398
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amphibians/classification/*genetics
MH  - Animals
MH  - *Evolution, Molecular
MH  - Genes, RAG-1/*genetics
MH  - *Genome, Mitochondrial
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2012/02/14 06:00
MHDA- 2012/10/19 06:00
CRDT- 2012/02/14 06:00
PHST- 2012/02/14 06:00 [entrez]
PHST- 2012/02/14 06:00 [pubmed]
PHST- 2012/10/19 06:00 [medline]
AID - sys028 [pii]
AID - 10.1093/sysbio/sys028 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jul;61(4):661-74. doi: 10.1093/sysbio/sys028. Epub 2012 Feb 10.

PMID- 22234419
OWN - NLM
STAT- MEDLINE
DCOM- 20121018
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 4
DP  - 2012 Jul
TI  - Searching for ancestral areas and artifactual centers of origin in biogeography: 
      with comment on east-west patterns across Southern Australia.
PG  - 703-8
LID - 10.1093/sysbio/sys005 [doi]
FAU - Ladiges, Pauline Y
AU  - Ladiges PY
AD  - School of Botany, The University of Melbourne, Victoria 3010, Australia.
      p.ladiges@unimelb.edu.au
FAU - Bayly, Michael J
AU  - Bayly MJ
FAU - Nelson, Gareth
AU  - Nelson G
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120110
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Australia
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - Evolution, Molecular
MH  - Geography
MH  - Magnoliopsida/genetics/physiology
MH  - *Phylogeny
EDAT- 2012/01/12 06:00
MHDA- 2012/10/19 06:00
CRDT- 2012/01/12 06:00
PHST- 2012/01/12 06:00 [entrez]
PHST- 2012/01/12 06:00 [pubmed]
PHST- 2012/10/19 06:00 [medline]
AID - sys005 [pii]
AID - 10.1093/sysbio/sys005 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jul;61(4):703-8. doi: 10.1093/sysbio/sys005. Epub 2012 Jan 10.

PMID- 22232343
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20120813
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - Ultraconserved elements anchor thousands of genetic markers spanning multiple
      evolutionary timescales.
PG  - 717-26
LID - 10.1093/sysbio/sys004 [doi]
AB  - Although massively parallel sequencing has facilitated large-scale DNA
      sequencing, comparisons among distantly related species rely upon small portions 
      of the genome that are easily aligned. Methods are needed to efficiently obtain
      comparable DNA fragments prior to massively parallel sequencing, particularly for
      biologists working with non-model organisms. We introduce a new class of
      molecular marker, anchored by ultraconserved genomic elements (UCEs), that
      universally enable target enrichment and sequencing of thousands of orthologous
      loci across species separated by hundreds of millions of years of evolution. Our 
      analyses here focus on use of UCE markers in Amniota because UCEs and
      phylogenetic relationships are well-known in some amniotes. We perform an in
      silico experiment to demonstrate that sequence flanking 2030 UCEs contains
      information sufficient to enable unambiguous recovery of the established primate 
      phylogeny. We extend this experiment by performing an in vitro enrichment of 2386
      UCE-anchored loci from nine, non-model avian species. We then use alignments of
      854 of these loci to unambiguously recover the established evolutionary
      relationships within and among three ancient bird lineages. Because many
      organismal lineages have UCEs, this type of genetic marker and the analytical
      framework we outline can be applied across the tree of life, potentially
      reshaping our understanding of phylogeny at many taxonomic levels.
FAU - Faircloth, Brant C
AU  - Faircloth BC
AD  - Department of Ecology and Evolutionary Biology, University of California, Los
      Angeles, CA 90095, USA. brant@faircloth-lab.org
FAU - McCormack, John E
AU  - McCormack JE
FAU - Crawford, Nicholas G
AU  - Crawford NG
FAU - Harvey, Michael G
AU  - Harvey MG
FAU - Brumfield, Robb T
AU  - Brumfield RT
FAU - Glenn, Travis C
AU  - Glenn TC
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120109
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
SB  - IM
MH  - Animals
MH  - Computer Simulation
MH  - *Conserved Sequence
MH  - *Evolution, Molecular
MH  - Genetic Markers
MH  - Genome
MH  - High-Throughput Nucleotide Sequencing/methods
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA/*methods
MH  - Vertebrates/classification/*genetics
EDAT- 2012/01/11 06:00
MHDA- 2013/01/23 06:00
CRDT- 2012/01/11 06:00
PHST- 2012/01/11 06:00 [entrez]
PHST- 2012/01/11 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - sys004 [pii]
AID - 10.1093/sysbio/sys004 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):717-26. doi: 10.1093/sysbio/sys004. Epub 2012 Jan 9.

PMID- 22228800
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20120216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - Taxon influence index: assessing taxon-induced incongruities in phylogenetic
      inference.
PG  - 337-45
LID - 10.1093/sysbio/syr129 [doi]
AB  - Understanding the evolutionary history of species is at the core of molecular
      evolution and is done using several inference methods. The critical issue is to
      quantify the uncertainty of the inference. The posterior probabilities in
      Bayesian phylogenetic inference and the bootstrap values in frequentist
      approaches measure the variability of the estimates due to the sampling of sites 
      from genes and the sampling of genes from genomes. However, they do not measure
      the uncertainty due to taxon sampling. Taxa that experienced molecular homoplasy,
      recent selection, a spur of evolution, and so forth may disrupt the inference and
      cause incongruences in the estimated phylogeny. We define a taxon influence index
      to assess the influence of each taxon on the phylogeny. We found that although
      most taxa have a weak influence on the phylogeny, a small fraction of influential
      taxa strongly alter it even in clades only loosely related to them. We conclude
      that highly influential taxa should be given special attention and sampling them 
      more thoroughly can lead to more dependable phylogenies.
FAU - Mariadassou, Mahendra
AU  - Mariadassou M
AD  - Department of Mathematics and Informatics, MAP5, Universite Paris Descartes, 45
      rue des Saints Peres, 75270 Paris Cedex 06, France.
      mahendra.mariadassou@jouy.inra.fr
FAU - Bar-Hen, Avner
AU  - Bar-Hen A
FAU - Kishino, Hirohisa
AU  - Kishino H
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120105
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Classification/methods
MH  - Likelihood Functions
MH  - Mammals/classification/*genetics
MH  - *Phylogeny
MH  - Reptiles/classification/*genetics
EDAT- 2012/01/10 06:00
MHDA- 2012/06/26 06:00
CRDT- 2012/01/10 06:00
PHST- 2012/01/10 06:00 [entrez]
PHST- 2012/01/10 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr129 [pii]
AID - 10.1093/sysbio/syr129 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):337-45. doi: 10.1093/sysbio/syr129. Epub 2012 Jan 5.

PMID- 22228799
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - Multiple continental radiations and correlates of diversification in Lupinus
      (Leguminosae): testing for key innovation with incomplete taxon sampling.
PG  - 443-60
LID - 10.1093/sysbio/syr126 [doi]
AB  - Replicate radiations provide powerful comparative systems to address questions
      about the interplay between opportunity and innovation in driving episodes of
      diversification and the factors limiting their subsequent progression. However,
      such systems have been rarely documented at intercontinental scales. Here, we
      evaluate the hypothesis of multiple radiations in the genus Lupinus
      (Leguminosae), which exhibits some of the highest known rates of net
      diversification in plants. Given that incomplete taxon sampling, background
      extinction, and lineage-specific variation in diversification rates can confound 
      macroevolutionary inferences regarding the timing and mechanisms of cladogenesis,
      we used Bayesian relaxed clock phylogenetic analyses as well as MEDUSA and BiSSE 
      birth-death likelihood models of diversification, to evaluate the evolutionary
      patterns of lineage accumulation in Lupinus. We identified 3 significant shifts
      to increased rates of net diversification (r) relative to background levels in
      the genus (r = 0.18-0.48 lineages/myr). The primary shift occurred approximately 
      4.6 Ma (r = 0.48-1.76) in the montane regions of western North America, followed 
      by a secondary shift approximately 2.7 Ma (r = 0.89-3.33) associated with range
      expansion and diversification of allopatrically distributed sister clades in the 
      Mexican highlands and Andes. We also recovered evidence for a third independent
      shift approximately 6.5 Ma at the base of a lower elevation eastern South
      American grassland and campo rupestre clade (r = 0.36-1.33). Bayesian ancestral
      state reconstructions and BiSSE likelihood analyses of correlated diversification
      indicated that increased rates of speciation are strongly associated with the
      derived evolution of perennial life history and invasion of montane ecosystems.
      Although we currently lack hard evidence for "replicate adaptive radiations" in
      the sense of convergent morphological and ecological trajectories among species
      in different clades, these results are consistent with the hypothesis that
      iteroparity functioned as an adaptive key innovation, providing a mechanism for
      range expansion and rapid divergence in upper elevation regions across much of
      the New World.
FAU - Drummond, Christopher S
AU  - Drummond CS
AD  - Department of Fish and Wildlife Resources, College of Natural Resources,
      University of Idaho, Moscow, ID 83844, USA.
FAU - Eastwood, Ruth J
AU  - Eastwood RJ
FAU - Miotto, Silvia T S
AU  - Miotto ST
FAU - Hughes, Colin E
AU  - Hughes CE
LA  - eng
GR  - P20RR016454/RR/NCRR NIH HHS/United States
GR  - P20RR16448/RR/NCRR NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
DEP - 20120105
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Biodiversity
MH  - DNA, Plant/genetics
MH  - Likelihood Functions
MH  - Lupinus/*classification/genetics
MH  - *Phylogeny
MH  - Phylogeography/*methods
PMC - PMC3329764
EDAT- 2012/01/10 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/01/10 06:00
PHST- 2012/01/10 06:00 [entrez]
PHST- 2012/01/10 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - syr126 [pii]
AID - 10.1093/sysbio/syr126 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):443-60. doi: 10.1093/sysbio/syr126. Epub 2012 Jan 5.

PMID- 22228798
OWN - NLM
STAT- MEDLINE
DCOM- 20121018
LR  - 20120618
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 4
DP  - 2012 Jul
TI  - Mutualism with plants drives primate diversification.
PG  - 567-77
LID - 10.1093/sysbio/syr127 [doi]
AB  - Understanding the origin of diversity is a fundamental problem in biology.
      Evolutionary diversification has been intensely explored during the last years
      due to the development of molecular tools and the comparative method. However,
      most studies are conducted using only information from extant species. This
      approach probably leads to misleading conclusions, especially because of
      inaccuracy in the estimation of extinction rates. It is critical to integrate the
      information generated by extant organisms with the information obtained from the 
      fossil record. Unfortunately, this integrative approach has been seldom
      performed, and thus, our understanding of the factors fueling diversification is 
      still deficient. Ecological interactions are a main factor shaping evolutionary
      diversification by influencing speciation and extinction rates. Most attention
      has focused on the effect of antagonistic interactions on evolutionary
      diversification. In contrast, the role of mutualistic interactions in shaping
      diversification has been much less explored. In this study, by combining
      phylogenetic, neontological, and paleontological information, we show that a
      facultative mutualistic plant-animal interaction emerging from frugivory and seed
      dispersal has most likely contributed to the diversification of our own lineage, 
      the primates. We compiled diet and seed dispersal ability in 381 extant and 556
      extinct primates. Using well-established molecular phylogenies, we demonstrated
      that mutualistic extant primates had higher speciation rates, lower extinction
      rates, and thereby higher diversification rates than nonmutualistic ones.
      Similarly, mutualistic fossil primates had higher geological durations and
      smaller per capita rates of extinction than nonmutualistic ones. As a mechanism
      underlying this pattern, we found that mutualistic extinct and extant primates
      have significantly larger geographic ranges, which promotes diversification by
      hampering extinction and increasing geographic speciation. All these outcomes
      together strongly suggest that the establishment of a facultative mutualism with 
      plants has greatly benefited primate evolution and fueled its taxonomic
      diversification.
FAU - Gomez, Jose M
AU  - Gomez JM
AD  - Departamento de Ecologia, University of Granada, E-18071 Granada, Spain.
      jmgreyes@ugr.es
FAU - Verdu, Miguel
AU  - Verdu M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120105
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - *Biological Evolution
MH  - Body Size
MH  - *Diet
MH  - Extinction, Biological
MH  - *Food Chain
MH  - Fossils
MH  - Genetic Speciation
MH  - Phylogeny
MH  - *Plant Physiological Phenomena
MH  - Primates/genetics/*physiology
MH  - *Symbiosis
MH  - Vision, Ocular
EDAT- 2012/01/10 06:00
MHDA- 2012/10/19 06:00
CRDT- 2012/01/10 06:00
PHST- 2012/01/10 06:00 [entrez]
PHST- 2012/01/10 06:00 [pubmed]
PHST- 2012/10/19 06:00 [medline]
AID - syr127 [pii]
AID - 10.1093/sysbio/syr127 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jul;61(4):567-77. doi: 10.1093/sysbio/syr127. Epub 2012 Jan 5.

PMID- 22223449
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20120420
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - Overcoming deep roots, fast rates, and short internodes to resolve the ancient
      rapid radiation of eupolypod II ferns.
PG  - 490-509
LID - 10.1093/sysbio/sys001 [doi]
AB  - Backbone relationships within the large eupolypod II clade, which includes nearly
      a third of extant fern species, have resisted elucidation by both molecular and
      morphological data. Earlier studies suggest that much of the phylogenetic
      intractability of this group is due to three factors: (i) a long root that
      reduces apparent levels of support in the ingroup; (ii) long ingroup branches
      subtended by a series of very short backbone internodes (the "ancient rapid
      radiation" model); and (iii) significantly heterogeneous lineage-specific rates
      of substitution. To resolve the eupolypod II phylogeny, with a particular
      emphasis on the backbone internodes, we assembled a data set of five plastid loci
      (atpA, atpB, matK, rbcL, and trnG-R) from a sample of 81 accessions selected to
      capture the deepest divergences in the clade. We then evaluated our phylogenetic 
      hypothesis against potential confounding factors, including those induced by
      rooting, ancient rapid radiation, rate heterogeneity, and the Bayesian star-tree 
      paradox artifact. While the strong support we inferred for the backbone
      relationships proved robust to these potential problems, their investigation
      revealed unexpected model-mediated impacts of outgroup composition, divergent
      effects of methods for countering the star-tree paradox artifact, and gave no
      support to concerns about the applicability of the unrooted model to data sets
      with heterogeneous lineage-specific rates of substitution. This study is among
      few to investigate these factors with empirical data, and the first to compare
      the performance of the two primary methods for overcoming the Bayesian star-tree 
      paradox artifact. Among the significant phylogenetic results is the near-complete
      support along the eupolypod II backbone, the demonstrated paraphyly of
      Woodsiaceae as currently circumscribed, and the well-supported placement of the
      enigmatic genera Homalosorus, Diplaziopsis, and Woodsia.
FAU - Rothfels, Carl J
AU  - Rothfels CJ
AD  - Department of Biology, Duke University, Box 90338, Durham, NC 27708, USA.
      carl.rothfels@duke.edu
FAU - Larsson, Anders
AU  - Larsson A
FAU - Kuo, Li-Yaung
AU  - Kuo LY
FAU - Korall, Petra
AU  - Korall P
FAU - Chiou, Wen-Liang
AU  - Chiou WL
FAU - Pryer, Kathleen M
AU  - Pryer KM
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120104
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Ferns/*classification/*genetics
MH  - Genes, Plant/genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
EDAT- 2012/01/10 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/01/07 06:00
PHST- 2012/01/07 06:00 [entrez]
PHST- 2012/01/10 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - sys001 [pii]
AID - 10.1093/sysbio/sys001 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):490-509. doi: 10.1093/sysbio/sys001. Epub 2012 Jan 4.

PMID- 22223448
OWN - NLM
STAT- MEDLINE
DCOM- 20121018
LR  - 20120618
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 4
DP  - 2012 Jul
TI  - The evolution of defense mechanisms correlate with the explosive diversification 
      of autodigesting Coprinellus mushrooms (Agaricales, Fungi).
PG  - 595-607
LID - 10.1093/sysbio/sys002 [doi]
AB  - Bursts of diversification are known to have contributed significantly to the
      extant morphological and species diversity, but evidence for many of the
      theoretical predictions about adaptive radiations have remained contentious.
      Despite their tremendous diversity, patterns of evolutionary diversification and 
      the contribution of explosive episodes in fungi are largely unknown. Here, using 
      the genus Coprinellus (Psathyrellaceae, Agaricales) as a model, we report the
      first explosive fungal radiation and infer that the onset of the radiation
      correlates with a change from a multilayered to a much simpler defense structure 
      on the fruiting bodies. We hypothesize that this change constitutes a key
      innovation, probably relaxing constraints on diversification imposed by
      nutritional investment into the development of protective tissues of fruiting
      bodies. Fossil calibration suggests that Coprinellus mushrooms radiated during
      the Miocene coinciding with global radiation of large grazing mammals following
      expansion of dry open grasslands. In addition to diversification rate-based
      methods, we test the hard polytomy hypothesis, by analyzing the resolvability of 
      internal nodes of the backbone of the putative radiation using Reversible-Jump
      MCMC. We discuss potential applications and pitfalls of this approach as well as 
      how biologically meaningful polytomies can be distinguished from alignment
      shortcomings. Our data provide insights into the nature of adaptive radiations in
      general by revealing a deceleration of morphological diversification through
      time. The dynamics of morphological diversification was approximated by obtaining
      the temporal distribution of state changes in discrete traits along the trees and
      comparing it with the tempo of lineage accumulation. We found that the number of 
      state changes correlate with the number of lineages, even in parts of the tree
      with short internal branches, and peaks around the onset of the explosive
      radiation followed by a slowdown, most likely because of the decrease in
      available niches.
FAU - Nagy, Laszlo G
AU  - Nagy LG
AD  - Department of Microbiology, Faculty of Science and Informatics, University of
      Szeged, Kozep fasor 52., H-6726 Szeged, Hungary. cortinarius2000@yahoo.co.uk
FAU - Hazi, Judit
AU  - Hazi J
FAU - Szappanos, Balazs
AU  - Szappanos B
FAU - Kocsube, Sandor
AU  - Kocsube S
FAU - Balint, Balazs
AU  - Balint B
FAU - Rakhely, Gabor
AU  - Rakhely G
FAU - Vagvolgyi, Csaba
AU  - Vagvolgyi C
FAU - Papp, Tamas
AU  - Papp T
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120104
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Fungal)
SB  - IM
MH  - Agaricales/classification/*cytology/*genetics/physiology
MH  - DNA, Fungal/genetics
MH  - Evolution, Molecular
MH  - Fossils
MH  - Fruiting Bodies, Fungal/classification/cytology/genetics
MH  - Genetic Speciation
MH  - Likelihood Functions
MH  - Markov Chains
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Monte Carlo Method
MH  - Multigene Family
MH  - *Phylogeny
MH  - Polymerase Chain Reaction
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2012/01/10 06:00
MHDA- 2012/10/19 06:00
CRDT- 2012/01/07 06:00
PHST- 2012/01/07 06:00 [entrez]
PHST- 2012/01/10 06:00 [pubmed]
PHST- 2012/10/19 06:00 [medline]
AID - sys002 [pii]
AID - 10.1093/sysbio/sys002 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jul;61(4):595-607. doi: 10.1093/sysbio/sys002. Epub 2012 Jan 4.

PMID- 22223447
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20120420
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - Effects of phylogenetic signal on ancestral state reconstruction.
PG  - 533-8
LID - 10.1093/sysbio/syr124 [doi]
FAU - Litsios, Glenn
AU  - Litsios G
AD  - Department of Ecology and Evolution, Biophore, University of Lausanne, 1015
      Lausanne, Switzerland.
FAU - Salamin, Nicolas
AU  - Salamin N
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120104
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Phylogeny
MH  - Statistics as Topic/standards
EDAT- 2012/01/10 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/01/07 06:00
PHST- 2012/01/07 06:00 [entrez]
PHST- 2012/01/10 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - syr124 [pii]
AID - 10.1093/sysbio/syr124 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):533-8. doi: 10.1093/sysbio/syr124. Epub 2012 Jan 4.

PMID- 22223446
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20120420
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - Multiple quaternary refugia in the eastern Guiana shield revealed by comparative 
      phylogeography of 12 frog species.
PG  - 461-89
LID - 10.1093/sysbio/syr130 [doi]
AB  - The Guiana Shield (GS) is one of the most pristine regions of Amazonia and
      biologically one of the richest areas on Earth. How and when this massive
      diversity arose remains the subject of considerable debate. The prevailing
      hypothesis of Quaternary glacial refugia suggests that a part of the eastern GS, 
      among other areas in Amazonia, served as stable forested refugia during periods
      of aridity. However, the recently proposed disturbance-vicariance hypothesis
      proposes that fluctuations in temperature on orbital timescales, with some
      associated aridity, have driven Neotropical diversification. The expectations of 
      the temporal and spatial organization of biodiversity differ between these two
      hypotheses. Here, we compare the genetic structure of 12 leaf-litter inhabiting
      frog species from the GS lowlands using a combination of mitochondrial and
      nuclear sequences in an integrative analytical approach that includes
      phylogenetic reconstructions, molecular dating, and Geographic Information System
      methods. This comparative and integrated approach overcomes the well-known
      limitations of phylogeographic inference based on single species and single loci.
      All of the focal species exhibit distinct phylogeographic patterns highlighting
      taxon-specific historical distributions, ecological tolerances to climatic
      disturbance, and dispersal abilities. Nevertheless, all but one species exhibit a
      history of fragmentation/isolation within the eastern GS during the Quaternary
      with spatial and temporal concordance among species. The signature of isolation
      in northern French Guiana (FG) during the early Pleistocene is particularly
      clear. Approximate Bayesian Computation supports the synchrony of the divergence 
      between northern FG and other GS lineages. Substructure observed throughout the
      GS suggests further Quaternary fragmentation and a role for rivers. Our findings 
      support fragmentation of moist tropical forest in the eastern GS during this
      period when the refuge hypothesis would have the region serving as a contiguous
      wet-forest refuge.
FAU - Fouquet, Antoine
AU  - Fouquet A
AD  - Molecular Ecology Laboratory, School of Biological Sciences, University of
      Canterbury, Private Bag 4800, Christchurch, 8140 New Zealand.
      fouquet.antoine@gmail.com
FAU - Noonan, Brice P
AU  - Noonan BP
FAU - Rodrigues, Miguel T
AU  - Rodrigues MT
FAU - Pech, Nicolas
AU  - Pech N
FAU - Gilles, Andre
AU  - Gilles A
FAU - Gemmell, Neil J
AU  - Gemmell NJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120104
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Anura/*classification/genetics
MH  - *Biodiversity
MH  - Guyana
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - *Phylogeography
MH  - Tropical Climate
EDAT- 2012/01/10 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/01/07 06:00
PHST- 2012/01/07 06:00 [entrez]
PHST- 2012/01/10 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - syr130 [pii]
AID - 10.1093/sysbio/syr130 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):461-89. doi: 10.1093/sysbio/syr130. Epub 2012 Jan 4.

PMID- 22223445
OWN - NLM
STAT- MEDLINE
DCOM- 20121018
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 4
DP  - 2012 Jul
TI  - Phylogenetic inference via sequential Monte Carlo.
PG  - 579-93
LID - 10.1093/sysbio/syr131 [doi]
AB  - Bayesian inference provides an appealing general framework for phylogenetic
      analysis, able to incorporate a wide variety of modeling assumptions and to
      provide a coherent treatment of uncertainty. Existing computational approaches to
      bayesian inference based on Markov chain Monte Carlo (MCMC) have not, however,
      kept pace with the scale of the data analysis problems in phylogenetics, and this
      has hindered the adoption of bayesian methods. In this paper, we present an
      alternative to MCMC based on Sequential Monte Carlo (SMC). We develop an
      extension of classical SMC based on partially ordered sets and show how to apply 
      this framework--which we refer to as PosetSMC--to phylogenetic analysis. We
      provide a theoretical treatment of PosetSMC and also present experimental
      evaluation of PosetSMC on both synthetic and real data. The empirical results
      demonstrate that PosetSMC is a very promising alternative to MCMC, providing up
      to two orders of magnitude faster convergence. We discuss other factors favorable
      to the adoption of PosetSMC in phylogenetics, including its ability to estimate
      marginal likelihoods, its ready implementability on parallel and distributed
      computing platforms, and the possibility of combining with MCMC in hybrid
      MCMC-SMC schemes. Software for PosetSMC is available at http://www.stat.ubc.ca/
      bouchard/PosetSMC.
FAU - Bouchard-Cote, Alexandre
AU  - Bouchard-Cote A
AD  - Department of Statistics, University of British Columbia, Vancouver, BC V6T 1Z2, 
      Canada.
FAU - Sankararaman, Sriram
AU  - Sankararaman S
FAU - Jordan, Michael I
AU  - Jordan MI
LA  - eng
GR  - R01 GM071749/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120104
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 16S)
SB  - IM
MH  - Algorithms
MH  - Bayes Theorem
MH  - Gene Frequency
MH  - Humans
MH  - Markov Chains
MH  - *Models, Genetic
MH  - *Monte Carlo Method
MH  - *Phylogeny
MH  - RNA, Ribosomal, 16S/genetics
MH  - Software
PMC - PMC3376373
EDAT- 2012/01/10 06:00
MHDA- 2012/10/19 06:00
CRDT- 2012/01/07 06:00
PHST- 2012/01/07 06:00 [entrez]
PHST- 2012/01/10 06:00 [pubmed]
PHST- 2012/10/19 06:00 [medline]
AID - syr131 [pii]
AID - 10.1093/sysbio/syr131 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jul;61(4):579-93. doi: 10.1093/sysbio/syr131. Epub 2012 Jan 4.

PMID- 22223444
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20120420
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - Miocene dispersal drives island radiations in the palm tribe Trachycarpeae
      (Arecaceae).
PG  - 426-42
LID - 10.1093/sysbio/syr123 [doi]
AB  - The study of three island groups of the palm tribe Trachycarpeae
      (Arecaceae/Palmae) permits both the analysis of each independent radiation and
      comparisons across the tribe to address general processes that drive island
      diversification. Phylogenetic relationships of Trachycarpeae were inferred from
      three plastid and three low-copy nuclear genes. The incongruent topological
      position of Brahea in CISP5 was hypothesized to be caused by a gene duplication
      event and was addressed using uninode coding. The resulting phylogenetic trees
      were well-resolved and the genera were all highly supported except for
      Johannesteijsmannia and Serenoa. Divergence time analysis estimated the stem of
      the tribe to be approximately 86 Ma and the crown to be 38 Ma, indicating that
      significant extinction may have occurred along this branch. Historical
      biogeographic analysis suggested that Trachycarpeae are of southern North
      American, Central American, or Caribbean origin and supports previous hypotheses 
      of a Laurasian origin. The biogeography and disjunctions within the tribe were
      interpreted with respect to divergence times, the fossil record, and geological
      factors such as the formation of the Greater Antilles--Aves Ridge, the Bering and
      the North Atlantic land bridges, tectonic movement in Southeast Asia, climatic
      shifts between the Eocene and Pliocene, and volcanism in the Pacific basin. In
      considering the three major island radiations within Trachycarpeae, Miocene
      dispersal appears to have been the driving force in allopatric speciation and is 
      highlighted here as an emerging pattern across the tree of life.
FAU - Bacon, Christine D
AU  - Bacon CD
AD  - Department of Biology, Colorado State University, Fort Collins, CO 80523-1878,
      USA. cbacon@rams.colostate.edu
FAU - Baker, William J
AU  - Baker WJ
FAU - Simmons, Mark P
AU  - Simmons MP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120105
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Arecaceae/*classification/*genetics
MH  - *Biodiversity
MH  - Fossils
MH  - Genome, Plant/genetics
MH  - Genome, Plastid/genetics
MH  - *Islands
MH  - Molecular Sequence Data
MH  - *Phylogeny
EDAT- 2012/01/10 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/01/07 06:00
PHST- 2012/01/07 06:00 [entrez]
PHST- 2012/01/10 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - syr123 [pii]
AID - 10.1093/sysbio/syr123 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):426-42. doi: 10.1093/sysbio/syr123. Epub 2012 Jan 5.

PMID- 22215721
OWN - NLM
STAT- MEDLINE
DCOM- 20121018
LR  - 20120618
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 4
DP  - 2012 Jul
TI  - Is recombination a problem for species-tree analyses?
PG  - 691-701
LID - 10.1093/sysbio/syr128 [doi]
FAU - Lanier, Hayley C
AU  - Lanier HC
AD  - Department of Ecology and Evolutionary Biology, Museum of Zoology, University of 
      Michigan, Ann Arbor, MI 48109-1079, USA. hclanier@umich.edu
FAU - Knowles, L Lacey
AU  - Knowles LL
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120102
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Evolution, Molecular
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - *Recombination, Genetic
EDAT- 2012/01/05 06:00
MHDA- 2012/10/19 06:00
CRDT- 2012/01/05 06:00
PHST- 2012/01/05 06:00 [entrez]
PHST- 2012/01/05 06:00 [pubmed]
PHST- 2012/10/19 06:00 [medline]
AID - syr128 [pii]
AID - 10.1093/sysbio/syr128 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jul;61(4):691-701. doi: 10.1093/sysbio/syr128. Epub 2012 Jan 2.

PMID- 22215720
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20120420
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - Independent contrasts and PGLS regression estimators are equivalent.
PG  - 382-91
LID - 10.1093/sysbio/syr118 [doi]
AB  - We prove that the slope parameter of the ordinary least squares regression of
      phylogenetically independent contrasts (PICs) conducted through the origin is
      identical to the slope parameter of the method of generalized least squares
      (GLSs) regression under a Brownian motion model of evolution. This equivalence
      has several implications: 1. Understanding the structure of the linear model for 
      GLS regression provides insight into when and why phylogeny is important in
      comparative studies. 2. The limitations of the PIC regression analysis are the
      same as the limitations of the GLS model. In particular, phylogenetic covariance 
      applies only to the response variable in the regression and the explanatory
      variable should be regarded as fixed. Calculation of PICs for explanatory
      variables should be treated as a mathematical idiosyncrasy of the PIC regression 
      algorithm. 3. Since the GLS estimator is the best linear unbiased estimator
      (BLUE), the slope parameter estimated using PICs is also BLUE. 4. If the slope is
      estimated using different branch lengths for the explanatory and response
      variables in the PIC algorithm, the estimator is no longer the BLUE, so this is
      not recommended. Finally, we discuss whether or not and how to accommodate
      phylogenetic covariance in regression analyses, particularly in relation to the
      problem of phylogenetic uncertainty. This discussion is from both frequentist and
      Bayesian perspectives.
FAU - Blomberg, Simon P
AU  - Blomberg SP
AD  - School of Biological Sciences, University of Queensland, St. Lucia, Queensland
      4072, Australia. s.blomberg1@uq.edu.au
FAU - Lefevre, James G
AU  - Lefevre JG
FAU - Wells, Jessie A
AU  - Wells JA
FAU - Waterhouse, Mary
AU  - Waterhouse M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120103
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Data Interpretation, Statistical
MH  - *Least-Squares Analysis
MH  - Phylogeny
EDAT- 2012/01/05 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/01/05 06:00
PHST- 2012/01/05 06:00 [entrez]
PHST- 2012/01/05 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - syr118 [pii]
AID - 10.1093/sysbio/syr118 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):382-91. doi: 10.1093/sysbio/syr118. Epub 2012 Jan 3.

PMID- 22213711
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20120420
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - Are Asteraceae 1.5 billion years old? A reply to heads.
PG  - 522-32
LID - 10.1093/sysbio/syr121 [doi]
FAU - Swenson, Ulf
AU  - Swenson U
AD  - Department of Phanerogamic Botany, Swedish Museum of Natural History, PO Box
      50007, 10405 Stockholm, Sweden. ulf.swenson@nrm.se
FAU - Nylinder, Stephan
AU  - Nylinder S
FAU - Wagstaff, Steven J
AU  - Wagstaff SJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120102
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Asteraceae/*classification
MH  - *Phylogeny
EDAT- 2012/01/04 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/01/04 06:00
PHST- 2012/01/04 06:00 [entrez]
PHST- 2012/01/04 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - syr121 [pii]
AID - 10.1093/sysbio/syr121 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):522-32. doi: 10.1093/sysbio/syr121. Epub 2012 Jan 2.

PMID- 22213710
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20120420
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - Simultaneously mapping and superimposing landmark configurations with parsimony
      as optimality criterion.
PG  - 392-400
LID - 10.1093/sysbio/syr119 [doi]
AB  - All methods proposed to date for mapping landmark configurations on a
      phylogenetic tree start from an alignment generated by methods that make no use
      of phylogenetic information, usually by superimposing all configurations against 
      a consensus configuration. In order to properly interpret differences between
      landmark configurations along the tree as changes in shape, the metric chosen to 
      define the ancestral assignments should also form the basis to superimpose the
      configurations. Thus, we present here a method that merges both steps, map and
      align, into a single procedure that (for the given tree) produces a multiple
      alignment and ancestral assignments such that the sum of the Euclidean distances 
      between the corresponding landmarks along tree nodes is minimized. This approach 
      is an extension of the method proposed by Catalano et al. (2010. Phylogenetic
      morphometrics (I): the use of landmark data in a phylogenetic framework.
      Cladistics. 26:539-549) for mapping landmark data with parsimony as optimality
      criterion. In the context of phylogenetics, this method allows maximizing the
      degree to which similarity in landmark positions can be accounted for by common
      ancestry. In the context of morphometrics, this approach guarantees (heuristics
      aside) that all the transformations inferred on the tree represent changes in
      shape. The performance of the method was evaluated on different data sets,
      indicating that the method produces marked improvements in tree score (up to 5%
      compared with generalized superimpositions, up to 11% compared with ordinary
      superimpositions). These empirical results stress the importance of incorporating
      the phylogenetic information into the alignment step.
FAU - Catalano, S A
AU  - Catalano SA
AD  - Consejo Nacional de Investigaciones Cientificas y Tecnicas, Argentina.
      sacatalano@gmail.com
FAU - Goloboff, P A
AU  - Goloboff PA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120102
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - Phylogeny
EDAT- 2012/01/04 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/01/04 06:00
PHST- 2012/01/04 06:00 [entrez]
PHST- 2012/01/04 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - syr119 [pii]
AID - 10.1093/sysbio/syr119 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):392-400. doi: 10.1093/sysbio/syr119. Epub 2012 Jan 2.

PMID- 22213709
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - Pleistocene speciation in the genus Populus (salicaceae).
PG  - 401-12
LID - 10.1093/sysbio/syr120 [doi]
AB  - The macroevolutionary consequences of recent climate change remain controversial,
      and there is little paleobotanical or morphological evidence that Pleistocene
      (1.8-0.12 Ma) glacial cycles acted as drivers of speciation, especially among
      lineages with long generation times, such as trees. We combined genetic and
      ecogeographic data from 2 closely related North American tree species, Populus
      balsamifera and P. trichocarpa (Salicacaeae), to determine if their divergence
      coincided with and was possibly caused by Pleistocene climatic events. We
      analyzed 32 nuclear loci from individuals of P. balsamifera and P. trichocarpa to
      produce coalescent-based estimates of the divergence time between the 2 species. 
      We coupled the coalescent analyses with paleodistribution models to assess the
      influence of climate change on species' range. Furthermore, measures of niche
      overlap were used to investigate patterns of ecological differentiation between
      species. We estimated the divergence date of P. balsamifera and P. trichocarpa at
      approximately 75 Ka, which corresponds closely with the onset of Marine Isotope
      Stage 4 ( approximately 76 Ka) and a rapid increase in global ice volume.
      Significance tests of niche overlap, in conjunction with genetic estimates of
      migration, suggested that speciation occurred in allopatry, possibly resulting
      from the environmental effects of Pleistocene glacial cycles. Our results
      indicate that the divergence of keystone tree species, which have shaped
      community diversity in northern North American ecosystems, was recent and may
      have been a consequence of Pleistocene-era glaciation and climate change.
FAU - Levsen, Nicholas D
AU  - Levsen ND
AD  - Department of Biological Sciences, Texas Tech University, Lubbock, TX 79409, USA.
      nicholas.levsen@ttu.edu
FAU - Tiffin, Peter
AU  - Tiffin P
FAU - Olson, Matthew S
AU  - Olson MS
LA  - eng
SI  - GENBANK/JN706765
SI  - GENBANK/JN706766
SI  - GENBANK/JN706767
SI  - GENBANK/JN706768
SI  - GENBANK/JN706769
SI  - GENBANK/JN706770
SI  - GENBANK/JN706771
SI  - GENBANK/JN706772
SI  - GENBANK/JN706773
SI  - GENBANK/JN706774
SI  - GENBANK/JN706775
SI  - GENBANK/JN706776
SI  - GENBANK/JN706777
SI  - GENBANK/JN706778
SI  - GENBANK/JN706779
SI  - GENBANK/JN706780
SI  - GENBANK/JN706781
SI  - GENBANK/JN706782
SI  - GENBANK/JN706783
SI  - GENBANK/JN706784
SI  - GENBANK/JN706785
SI  - GENBANK/JN706786
SI  - GENBANK/JN706787
SI  - GENBANK/JN706788
SI  - GENBANK/JN706789
SI  - GENBANK/JN706790
SI  - GENBANK/JN706791
SI  - GENBANK/JN706792
SI  - GENBANK/JN706793
SI  - GENBANK/JN706794
SI  - GENBANK/JN706795
SI  - GENBANK/JN706796
SI  - GENBANK/JN706797
SI  - GENBANK/JN706798
SI  - GENBANK/JN706799
SI  - GENBANK/JN706800
SI  - GENBANK/JN706801
SI  - GENBANK/JN706802
SI  - GENBANK/JN706803
SI  - GENBANK/JN706804
SI  - GENBANK/JN706805
SI  - GENBANK/JN706806
SI  - GENBANK/JN706807
SI  - GENBANK/JN706808
SI  - GENBANK/JN706809
SI  - GENBANK/JN706810
SI  - GENBANK/JN706811
SI  - GENBANK/JN706812
SI  - GENBANK/JN706813
SI  - GENBANK/JN706814
SI  - GENBANK/JN706815
SI  - GENBANK/JN706816
SI  - GENBANK/JN706817
SI  - GENBANK/JN706818
SI  - GENBANK/JN706819
SI  - GENBANK/JN706820
SI  - GENBANK/JN706821
SI  - GENBANK/JN706822
SI  - GENBANK/JN706823
SI  - GENBANK/JN706824
SI  - GENBANK/JN706825
SI  - GENBANK/JN706826
SI  - GENBANK/JN706827
SI  - GENBANK/JN706828
SI  - GENBANK/JN706829
SI  - GENBANK/JN706830
SI  - GENBANK/JN706831
SI  - GENBANK/JN706832
SI  - GENBANK/JN706833
SI  - GENBANK/JN706834
SI  - GENBANK/JN706835
SI  - GENBANK/JN706836
SI  - GENBANK/JN706837
SI  - GENBANK/JN706838
SI  - GENBANK/JN706839
SI  - GENBANK/JN706840
SI  - GENBANK/JN706841
SI  - GENBANK/JN706842
SI  - GENBANK/JN706843
SI  - GENBANK/JN706844
SI  - GENBANK/JN706845
SI  - GENBANK/JN706846
SI  - GENBANK/JN706847
SI  - GENBANK/JN706848
SI  - GENBANK/JN706849
SI  - GENBANK/JN706850
SI  - GENBANK/JN706851
SI  - GENBANK/JN706852
SI  - GENBANK/JN706853
SI  - GENBANK/JN706854
SI  - GENBANK/JN706855
SI  - GENBANK/JN706856
SI  - GENBANK/JN706857
SI  - GENBANK/JN706858
SI  - GENBANK/JN706859
SI  - GENBANK/JN706860
SI  - GENBANK/JN706861
SI  - GENBANK/JN706862
SI  - GENBANK/JN706863
SI  - GENBANK/JN706864
SI  - GENBANK/JN706865
SI  - GENBANK/JN706866
SI  - GENBANK/JN706867
SI  - GENBANK/JN706868
SI  - GENBANK/JN706869
SI  - GENBANK/JN706870
SI  - GENBANK/JN706871
SI  - GENBANK/JN706872
SI  - GENBANK/JN706873
SI  - GENBANK/JN706874
SI  - GENBANK/JN706875
SI  - GENBANK/JN706876
SI  - GENBANK/JN706877
SI  - GENBANK/JN706878
SI  - GENBANK/JN706879
SI  - GENBANK/JN706880
SI  - GENBANK/JN706881
SI  - GENBANK/JN706882
SI  - GENBANK/JN706883
SI  - GENBANK/JN706884
SI  - GENBANK/JN706885
SI  - GENBANK/JN706886
SI  - GENBANK/JN706887
SI  - GENBANK/JN706888
SI  - GENBANK/JN706889
SI  - GENBANK/JN706890
SI  - GENBANK/JN706891
SI  - GENBANK/JN706892
SI  - GENBANK/JN706893
SI  - GENBANK/JN706894
SI  - GENBANK/JN706895
SI  - GENBANK/JN706896
SI  - GENBANK/JN706897
SI  - GENBANK/JN706898
SI  - GENBANK/JN706899
SI  - GENBANK/JN706900
SI  - GENBANK/JN706901
SI  - GENBANK/JN706902
SI  - GENBANK/JN706903
SI  - GENBANK/JN706904
SI  - GENBANK/JN706905
SI  - GENBANK/JN706906
SI  - GENBANK/JN706907
SI  - GENBANK/JN706908
SI  - GENBANK/JN706909
SI  - GENBANK/JN706910
SI  - GENBANK/JN706911
SI  - GENBANK/JN706912
SI  - GENBANK/JN706913
SI  - GENBANK/JN706914
SI  - GENBANK/JN706915
SI  - GENBANK/JN706916
SI  - GENBANK/JN706917
SI  - GENBANK/JN706918
SI  - GENBANK/JN706919
SI  - GENBANK/JN706920
SI  - GENBANK/JN706921
SI  - GENBANK/JN706922
SI  - GENBANK/JN706923
SI  - GENBANK/JN706924
SI  - GENBANK/JN706925
SI  - GENBANK/JN706926
SI  - GENBANK/JN706927
SI  - GENBANK/JN706928
SI  - GENBANK/JN706929
SI  - GENBANK/JN706930
SI  - GENBANK/JN706931
SI  - GENBANK/JN706932
SI  - GENBANK/JN706933
SI  - GENBANK/JN706934
SI  - GENBANK/JN706935
SI  - GENBANK/JN706936
SI  - GENBANK/JN706937
SI  - GENBANK/JN706938
SI  - GENBANK/JN706939
SI  - GENBANK/JN706940
SI  - GENBANK/JN706941
SI  - GENBANK/JN706942
SI  - GENBANK/JN706943
SI  - GENBANK/JN706944
SI  - GENBANK/JN706945
SI  - GENBANK/JN706946
SI  - GENBANK/JN706947
SI  - GENBANK/JN706948
SI  - GENBANK/JN706949
SI  - GENBANK/JN706950
SI  - GENBANK/JN706951
SI  - GENBANK/JN706952
SI  - GENBANK/JN706953
SI  - GENBANK/JN706954
SI  - GENBANK/JN706955
SI  - GENBANK/JN706956
SI  - GENBANK/JN706957
SI  - GENBANK/JN706958
SI  - GENBANK/JN706959
SI  - GENBANK/JN706960
SI  - GENBANK/JN706961
SI  - GENBANK/JN706962
SI  - GENBANK/JN706963
SI  - GENBANK/JN706964
SI  - GENBANK/JN706965
SI  - GENBANK/JN706966
SI  - GENBANK/JN706967
SI  - GENBANK/JN706968
SI  - GENBANK/JN706969
SI  - GENBANK/JN706970
SI  - GENBANK/JN706971
SI  - GENBANK/JN706972
SI  - GENBANK/JN706973
SI  - GENBANK/JN706974
SI  - GENBANK/JN706975
SI  - GENBANK/JN706976
SI  - GENBANK/JN706977
SI  - GENBANK/JN706978
SI  - GENBANK/JN706979
SI  - GENBANK/JN706980
SI  - GENBANK/JN706981
SI  - GENBANK/JN706982
SI  - GENBANK/JN706983
SI  - GENBANK/JN706984
SI  - GENBANK/JN706985
SI  - GENBANK/JN706986
SI  - GENBANK/JN706987
SI  - GENBANK/JN706988
SI  - GENBANK/JN706989
SI  - GENBANK/JN706990
SI  - GENBANK/JN706991
SI  - GENBANK/JN706992
SI  - GENBANK/JN706993
SI  - GENBANK/JN706994
SI  - GENBANK/JN706995
SI  - GENBANK/JN706996
SI  - GENBANK/JN706997
SI  - GENBANK/JN706998
SI  - GENBANK/JN706999
SI  - GENBANK/JN707000
SI  - GENBANK/JN707001
SI  - GENBANK/JN707002
SI  - GENBANK/JN707003
SI  - GENBANK/JN707004
SI  - GENBANK/JN707005
SI  - GENBANK/JN707006
SI  - GENBANK/JN707007
SI  - GENBANK/JN707008
SI  - GENBANK/JN707009
SI  - GENBANK/JN707010
SI  - GENBANK/JN707011
SI  - GENBANK/JN707012
SI  - GENBANK/JN707013
SI  - GENBANK/JN707014
SI  - GENBANK/JN707015
SI  - GENBANK/JN707016
SI  - GENBANK/JN707017
SI  - GENBANK/JN707018
SI  - GENBANK/JN707019
SI  - GENBANK/JN707020
SI  - GENBANK/JN707021
SI  - GENBANK/JN707022
SI  - GENBANK/JN707023
SI  - GENBANK/JN707024
SI  - GENBANK/JN707025
SI  - GENBANK/JN707026
SI  - GENBANK/JN707027
SI  - GENBANK/JN707028
SI  - GENBANK/JN707029
SI  - GENBANK/JN707030
SI  - GENBANK/JN707031
SI  - GENBANK/JN707032
SI  - GENBANK/JN707033
SI  - GENBANK/JN707034
SI  - GENBANK/JN707035
SI  - GENBANK/JN707036
SI  - GENBANK/JN707037
SI  - GENBANK/JN707038
SI  - GENBANK/JN707039
SI  - GENBANK/JN707040
SI  - GENBANK/JN707041
SI  - GENBANK/JN707042
SI  - GENBANK/JN707043
SI  - GENBANK/JN707044
SI  - GENBANK/JN707045
SI  - GENBANK/JN707046
SI  - GENBANK/JN707047
SI  - GENBANK/JN707048
SI  - GENBANK/JN707049
SI  - GENBANK/JN707050
SI  - GENBANK/JN707051
SI  - GENBANK/JN707052
SI  - GENBANK/JN707053
SI  - GENBANK/JN707054
SI  - GENBANK/JN707055
SI  - GENBANK/JN707056
SI  - GENBANK/JN707057
SI  - GENBANK/JN707058
SI  - GENBANK/JN707059
SI  - GENBANK/JN707060
SI  - GENBANK/JN707061
SI  - GENBANK/JN707062
SI  - GENBANK/JN707063
SI  - GENBANK/JN707064
SI  - GENBANK/JN707065
SI  - GENBANK/JN707066
SI  - GENBANK/JN707067
SI  - GENBANK/JN707068
SI  - GENBANK/JN707069
SI  - GENBANK/JN707070
SI  - GENBANK/JN707071
SI  - GENBANK/JN707072
SI  - GENBANK/JN707073
SI  - GENBANK/JN707074
SI  - GENBANK/JN707075
SI  - GENBANK/JN707076
SI  - GENBANK/JN707077
SI  - GENBANK/JN707078
SI  - GENBANK/JN707079
SI  - GENBANK/JN707080
SI  - GENBANK/JN707081
SI  - GENBANK/JN707082
SI  - GENBANK/JN707083
SI  - GENBANK/JN707084
SI  - GENBANK/JN707085
SI  - GENBANK/JN707086
SI  - GENBANK/JN707087
SI  - GENBANK/JN707088
SI  - GENBANK/JN707089
SI  - GENBANK/JN707090
SI  - GENBANK/JN707091
SI  - GENBANK/JN707092
SI  - GENBANK/JN707093
SI  - GENBANK/JN707094
SI  - GENBANK/JN707095
SI  - GENBANK/JN707096
SI  - GENBANK/JN707097
SI  - GENBANK/JN707098
SI  - GENBANK/JN707099
SI  - GENBANK/JN707100
SI  - GENBANK/JN707101
SI  - GENBANK/JN707102
SI  - GENBANK/JN707103
SI  - GENBANK/JN707104
SI  - GENBANK/JN707105
SI  - GENBANK/JN707106
SI  - GENBANK/JN707107
SI  - GENBANK/JN707108
SI  - GENBANK/JN707109
SI  - GENBANK/JN707110
SI  - GENBANK/JN707111
SI  - GENBANK/JN707112
SI  - GENBANK/JN707113
SI  - GENBANK/JN707114
SI  - GENBANK/JN707115
SI  - GENBANK/JN707116
SI  - GENBANK/JN707117
SI  - GENBANK/JN707118
SI  - GENBANK/JN707119
SI  - GENBANK/JN707120
SI  - GENBANK/JN707121
SI  - GENBANK/JN707122
SI  - GENBANK/JN707123
SI  - GENBANK/JN707124
SI  - GENBANK/JN707125
SI  - GENBANK/JN707126
SI  - GENBANK/JN707127
SI  - GENBANK/JN707128
SI  - GENBANK/JN707129
SI  - GENBANK/JN707130
SI  - GENBANK/JN707131
SI  - GENBANK/JN707132
SI  - GENBANK/JN707133
SI  - GENBANK/JN707134
SI  - GENBANK/JN707135
SI  - GENBANK/JN707136
SI  - GENBANK/JN707137
SI  - GENBANK/JN707138
SI  - GENBANK/JN707139
SI  - GENBANK/JN707140
SI  - GENBANK/JN707141
SI  - GENBANK/JN707142
SI  - GENBANK/JN707143
SI  - GENBANK/JN707144
SI  - GENBANK/JN707145
SI  - GENBANK/JN707146
SI  - GENBANK/JN707147
SI  - GENBANK/JN707148
SI  - GENBANK/JN707149
SI  - GENBANK/JN707150
SI  - GENBANK/JN707151
SI  - GENBANK/JN707152
SI  - GENBANK/JN707153
SI  - GENBANK/JN707154
SI  - GENBANK/JN707155
SI  - GENBANK/JN707156
SI  - GENBANK/JN707157
SI  - GENBANK/JN707158
SI  - GENBANK/JN707159
SI  - GENBANK/JN707160
SI  - GENBANK/JN707161
SI  - GENBANK/JN707162
SI  - GENBANK/JN707163
SI  - GENBANK/JN707164
SI  - GENBANK/JN707165
SI  - GENBANK/JN707166
SI  - GENBANK/JN707167
SI  - GENBANK/JN707168
SI  - GENBANK/JN707169
SI  - GENBANK/JN707170
SI  - GENBANK/JN707171
SI  - GENBANK/JN707172
SI  - GENBANK/JN707173
SI  - GENBANK/JN707174
SI  - GENBANK/JN707175
SI  - GENBANK/JN707176
SI  - GENBANK/JN707177
SI  - GENBANK/JN707178
SI  - GENBANK/JN707179
SI  - GENBANK/JN707180
SI  - GENBANK/JN707181
SI  - GENBANK/JN707182
SI  - GENBANK/JN707183
SI  - GENBANK/JN707184
SI  - GENBANK/JN707185
SI  - GENBANK/JN707186
SI  - GENBANK/JN707187
SI  - GENBANK/JN707188
SI  - GENBANK/JN707189
SI  - GENBANK/JN707190
SI  - GENBANK/JN707191
SI  - GENBANK/JN707192
SI  - GENBANK/JN707193
SI  - GENBANK/JN707194
SI  - GENBANK/JN707195
SI  - GENBANK/JN707196
SI  - GENBANK/JN707197
SI  - GENBANK/JN707198
SI  - GENBANK/JN707199
SI  - GENBANK/JN707200
SI  - GENBANK/JN707201
SI  - GENBANK/JN707202
SI  - GENBANK/JN707203
SI  - GENBANK/JN707204
SI  - GENBANK/JN707205
SI  - GENBANK/JN707206
SI  - GENBANK/JN707207
SI  - GENBANK/JN707208
SI  - GENBANK/JN707209
SI  - GENBANK/JN707210
SI  - GENBANK/JN707211
SI  - GENBANK/JN707212
SI  - GENBANK/JN707213
SI  - GENBANK/JN707214
SI  - GENBANK/JN707215
SI  - GENBANK/JN707216
SI  - GENBANK/JN707217
SI  - GENBANK/JN707218
SI  - GENBANK/JN707219
SI  - GENBANK/JN707220
SI  - GENBANK/JN707221
SI  - GENBANK/JN707222
SI  - GENBANK/JN707223
SI  - GENBANK/JN707224
SI  - GENBANK/JN707225
SI  - GENBANK/JN707226
SI  - GENBANK/JN707227
SI  - GENBANK/JN707228
SI  - GENBANK/JN707229
SI  - GENBANK/JN707230
SI  - GENBANK/JN707231
SI  - GENBANK/JN707232
SI  - GENBANK/JN707233
SI  - GENBANK/JN707234
SI  - GENBANK/JN707235
SI  - GENBANK/JN707236
SI  - GENBANK/JN707237
SI  - GENBANK/JN707238
SI  - GENBANK/JN707239
SI  - GENBANK/JN707240
SI  - GENBANK/JN707241
SI  - GENBANK/JN707242
SI  - GENBANK/JN707243
SI  - GENBANK/JN707244
SI  - GENBANK/JN707245
SI  - GENBANK/JN707246
SI  - GENBANK/JN707247
SI  - GENBANK/JN707248
SI  - GENBANK/JN707249
SI  - GENBANK/JN707250
SI  - GENBANK/JN707251
SI  - GENBANK/JN707252
SI  - GENBANK/JN707253
SI  - GENBANK/JN707254
SI  - GENBANK/JN707255
SI  - GENBANK/JN707256
SI  - GENBANK/JN707257
SI  - GENBANK/JN707258
SI  - GENBANK/JN707259
SI  - GENBANK/JN707260
SI  - GENBANK/JN707261
SI  - GENBANK/JN707262
SI  - GENBANK/JN707263
SI  - GENBANK/JN707264
SI  - GENBANK/JN707265
SI  - GENBANK/JN707266
SI  - GENBANK/JN707267
SI  - GENBANK/JN707268
SI  - GENBANK/JN707269
SI  - GENBANK/JN707270
SI  - GENBANK/JN707271
SI  - GENBANK/JN707272
SI  - GENBANK/JN707273
SI  - GENBANK/JN707274
SI  - GENBANK/JN707275
SI  - GENBANK/JN707276
SI  - GENBANK/JN707277
SI  - GENBANK/JN707278
SI  - GENBANK/JN707279
SI  - GENBANK/JN707280
SI  - GENBANK/JN707281
SI  - GENBANK/JN707282
SI  - GENBANK/JN707283
SI  - GENBANK/JN707284
SI  - GENBANK/JN707285
SI  - GENBANK/JN707286
SI  - GENBANK/JN707287
SI  - GENBANK/JN707288
SI  - GENBANK/JN707289
SI  - GENBANK/JN707290
SI  - GENBANK/JN707291
SI  - GENBANK/JN707292
SI  - GENBANK/JN707293
SI  - GENBANK/JN707294
SI  - GENBANK/JN707295
SI  - GENBANK/JN707296
SI  - GENBANK/JN707297
SI  - GENBANK/JN707298
SI  - GENBANK/JN707299
SI  - GENBANK/JN707300
SI  - GENBANK/JN707301
SI  - GENBANK/JN707302
SI  - GENBANK/JN707303
SI  - GENBANK/JN707304
SI  - GENBANK/JN707305
SI  - GENBANK/JN707306
SI  - GENBANK/JN707307
SI  - GENBANK/JN707308
SI  - GENBANK/JN707309
SI  - GENBANK/JN707310
SI  - GENBANK/JN707311
SI  - GENBANK/JN707312
SI  - GENBANK/JN707313
SI  - GENBANK/JN707314
SI  - GENBANK/JN707315
SI  - GENBANK/JN707316
SI  - GENBANK/JN707317
SI  - GENBANK/JN707318
SI  - GENBANK/JN707319
SI  - GENBANK/JN707320
SI  - GENBANK/JN707321
SI  - GENBANK/JN707322
SI  - GENBANK/JN707323
SI  - GENBANK/JN707324
SI  - GENBANK/JN707325
SI  - GENBANK/JN707326
SI  - GENBANK/JN707327
SI  - GENBANK/JN707328
SI  - GENBANK/JN707329
SI  - GENBANK/JN707330
SI  - GENBANK/JN707331
SI  - GENBANK/JN707332
SI  - GENBANK/JN707333
SI  - GENBANK/JN707334
SI  - GENBANK/JN707335
SI  - GENBANK/JN707336
SI  - GENBANK/JN707337
SI  - GENBANK/JN707338
SI  - GENBANK/JN707339
SI  - GENBANK/JN707340
SI  - GENBANK/JN707341
SI  - GENBANK/JN707342
SI  - GENBANK/JN707343
SI  - GENBANK/JN707344
SI  - GENBANK/JN707345
GR  - RR016466/RR/NCRR NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20120102
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Climate Change
MH  - Ecosystem
MH  - Genes, Plant/genetics
MH  - *Genetic Speciation
MH  - Genetic Variation
MH  - Models, Biological
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Populus/*classification/*genetics
PMC - PMC3529545
EDAT- 2012/01/04 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/01/04 06:00
PHST- 2012/01/04 06:00 [entrez]
PHST- 2012/01/04 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - syr120 [pii]
AID - 10.1093/sysbio/syr120 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):401-12. doi: 10.1093/sysbio/syr120. Epub 2012 Jan 2.
</pre></body></html>