<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0035)https://www.ncbi.nlm.nih.gov/pubmed -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"></head><body><pre>PMID- 31372653
OWN - NLM
STAT- Publisher
LR  - 20190802
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Aug 2
TI  - The Invariant Nature of a Morphological Character and Character State: Insights
      from Gene Regulatory Networks.
LID - syz050 [pii]
LID - 10.1093/sysbio/syz050 [doi]
AB  - What constitutes a discrete morphological character versus character state has
      been long discussed in the systematics literature but the consensus on this issue
      is still missing. Different methods of classifying organismal features into
      characters and character states can dramatically affect the results of
      phylogenetic analyses. Here, I show that, in the framework of Markov models, the 
      modular structure of the gene regulatory network (GRN) underlying trait
      development, and the hierarchical nature of GRN evolution, essentially remove the
      distinction between morphological character and character state, thus endowing
      the character and character state with an invariant property with respect to each
      other. This property allows the states of one character to be represented as
      several individual characters and vice versa. In practice, this means that a
      phenotype can be encoded using a set of characters or just one complex character 
      with numerous states. The representation of a phenotype using one complex
      character can be implemented in Markov models of trait evolution by properly
      structuring transition rate matrix.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Tarasov, Sergei
AU  - Tarasov S
AD  - Finnish Museum of Natural History, Pohjoinen Rautatiekatu 13, FI-00014 Helsinki, 
      Finland.
AD  - Department of Biological Sciences, Virginia Tech, 4076 Derring Hall, 926 West
      Campus Drive, Blacksburg, VA 24061, USA.
AD  - National Institute for Mathematical and Biological Synthesis, University of
      Tennessee, Knoxville, TN 37996.
LA  - eng
PT  - Journal Article
DEP - 20190802
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
OTO - NOTNLM
OT  - character
OT  - character state
OT  - combinations of states
OT  - gene regulatory networks
OT  - invariance
OT  - morphology
OT  - structured Markov models
EDAT- 2019/08/03 06:00
MHDA- 2019/08/03 06:00
CRDT- 2019/08/03 06:00
PHST- 2018/09/27 00:00 [received]
PHST- 2019/06/13 00:00 [revised]
PHST- 2019/07/17 00:00 [revised]
PHST- 2019/07/22 00:00 [accepted]
PHST- 2019/08/03 06:00 [entrez]
PHST- 2019/08/03 06:00 [pubmed]
PHST- 2019/08/03 06:00 [medline]
AID - 5541792 [pii]
AID - 10.1093/sysbio/syz050 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Aug 2. pii: 5541792. doi: 10.1093/sysbio/syz050.

PMID- 31372642
OWN - NLM
STAT- Publisher
LR  - 20190802
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Aug 2
TI  - An Integrative Approach using Phylogenomics and High-resolution X-ray Computed
      Tomography (CT) for Species Delimitation in Cryptic Taxa.
LID - syz048 [pii]
LID - 10.1093/sysbio/syz048 [doi]
AB  - - Morphologically cryptic taxa have proved to be a long-standing challenge for
      taxonomists. Lineages that show strong genomic structuring across the landscape
      but are phenotypically similar pose a conundrum, with traditional morphological
      analyses of these cryptic lineages struggling to keep up with species
      delimitation advances. Micro X-ray computed tomography (CT) combined with
      geometric morphometric analyses provides a promising avenue for identification of
      morphologically cryptic taxa, given its ability to detect subtle differences in
      anatomical structures. This approach, however, has yet to be used in combination 
      with genomic data in a comparative analytical framework to distinguish cryptic
      taxa. We present an integrative approach incorporating genomic and geometric
      morphometric evidence to assess the species delimitation of grassland earless
      dragons (Tympanocryptis spp.) in north-eastern Australia. Using mitochondrial and
      nuclear genes (ND2 and RAG1, respectively), along with &gt;8500 SNPs (nuclear single
      nucleotide polymorphisms), we assess the evolutionary independence of target
      lineages and several closely related species. We then integrate phylogenomic data
      with osteological cranial variation between lineages using landmark-based
      analyses of three-dimensional CT models. High levels of genomic differentiation
      between the three target lineages were uncovered, also supported by significant
      osteological differences. By incorporating multiple lines of evidence we provide 
      strong support that there are three undescribed cryptic lineages of
      Tympanocryptis in north-eastern Australia that warrant taxonomic review. Our
      approach demonstrates the successful application of CT with integrative taxonomic
      approaches for cryptic species delimitation, which is broadly applicable across
      vertebrates containing morphologically similar yet genetically distinct lineages.
      Additionally, we provide a review of recent integrative taxonomic approaches for 
      cryptic species delimitation, and an assessment of how our approach can value-add
      to taxonomic research.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Chaplin, Kirilee
AU  - Chaplin K
AD  - Sciences Department Museums Victoria, GPO Box 666, Melbourne, Victoria,
      Australia.
AD  - School of BioSciences, University of Melbourne, Parkville, Victoria, Australia.
FAU - Sumner, Joanna
AU  - Sumner J
AD  - Sciences Department Museums Victoria, GPO Box 666, Melbourne, Victoria,
      Australia.
FAU - Hipsley, Christy
AU  - Hipsley C
AD  - Sciences Department Museums Victoria, GPO Box 666, Melbourne, Victoria,
      Australia.
AD  - School of BioSciences, University of Melbourne, Parkville, Victoria, Australia.
FAU - Melville, Jane
AU  - Melville J
AD  - Sciences Department Museums Victoria, GPO Box 666, Melbourne, Victoria,
      Australia.
LA  - eng
PT  - Journal Article
DEP - 20190802
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
OTO - NOTNLM
OT  - SNPs
OT  - Tympanocryptis
OT  - geometric morphometrics
OT  - grassland earless dragons
OT  - integrative taxonomy
OT  - micro X-ray computed tomography
OT  - phylogenomics
OT  - species delimitation
EDAT- 2019/08/03 06:00
MHDA- 2019/08/03 06:00
CRDT- 2019/08/03 06:00
PHST- 2019/02/22 00:00 [received]
PHST- 2019/06/18 00:00 [revised]
PHST- 2019/07/12 00:00 [accepted]
PHST- 2019/08/03 06:00 [entrez]
PHST- 2019/08/03 06:00 [pubmed]
PHST- 2019/08/03 06:00 [medline]
AID - 5542840 [pii]
AID - 10.1093/sysbio/syz048 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Aug 2. pii: 5542840. doi: 10.1093/sysbio/syz048.

PMID- 31364711
OWN - NLM
STAT- Publisher
LR  - 20190731
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Jul 31
TI  - GHOST: Recovering Historical Signal from Heterotachously-evolved Sequence
      Alignments.
LID - syz051 [pii]
LID - 10.1093/sysbio/syz051 [doi]
AB  - Molecular sequence data that have evolved under the influence of heterotachous
      evolutionary processes are known to mislead phylogenetic inference. We introduce 
      the General Heterogeneous evolution On a Single Topology (GHOST) model of
      sequence evolution, implemented under a maximum-likelihood framework in the
      phylogenetic program IQ-TREE (http://www.iqtree.org). Simulations show that using
      the GHOST model, IQ-TREE can accurately recover the tree topology, branch lengths
      and substitution model parameters from heterotachously-evolved sequences. We
      investigate the performance of the GHOST model on empirical data by sampling
      phylogenomic alignments of varying lengths from a plastome alignment. We then
      carry out inference under the GHOST model on a phylogenomic dataset composed of
      248 genes from 16 taxa, where we find the GHOST model concurs with the currently 
      accepted view, placing turtles as a sister lineage of archosaurs, in contrast to 
      results obtained using traditional variable rates-across-sites models. Finally,
      we apply the model to a dataset composed of a sodium channel gene of 11 fish
      taxa, finding that the GHOST model is able to elucidate a subtle component of the
      historical signal, linked to the previously established convergent evolution of
      the electric organ in two geographically distinct lineages of electric fish. We
      compare inference under the GHOST model to partitioning by codon position and
      show that, owing to the minimization of model constraints, the GHOST model offers
      unique biological insights when applied to empirical data.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Crotty, Stephen M
AU  - Crotty SM
AD  - Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories,
      University of Vienna and Medical University of Vienna, Vienna, Austria.
AD  - School of Mathematical Sciences, University of Adelaide, Adelaide, SA, Australia.
AD  - ARC Centre of Excellence for Mathematical and Statistical Frontiers, The
      University of Adelaide, Adelaide, SA, Australia.
FAU - Minh, Bui Quang
AU  - Minh BQ
AD  - Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories,
      University of Vienna and Medical University of Vienna, Vienna, Austria.
FAU - Bean, Nigel G
AU  - Bean NG
AD  - School of Mathematical Sciences, University of Adelaide, Adelaide, SA, Australia.
AD  - ARC Centre of Excellence for Mathematical and Statistical Frontiers, The
      University of Adelaide, Adelaide, SA, Australia.
FAU - Holland, Barbara R
AU  - Holland BR
AD  - School of Physical Sciences, University of Tasmania, Hobart, TAS, Australia.
FAU - Tuke, Jonathan
AU  - Tuke J
AD  - School of Mathematical Sciences, University of Adelaide, Adelaide, SA, Australia.
AD  - ARC Centre of Excellence for Mathematical and Statistical Frontiers, The
      University of Adelaide, Adelaide, SA, Australia.
FAU - Jermiin, Lars S
AU  - Jermiin LS
AD  - CSIRO Land &amp; Water, Black Mountain Laboratories, Canberra, ACT, Australia.
AD  - Research School of Biology, Australian National University, Canberra, ACT,
      Australia.
FAU - von Haeseler, Arndt
AU  - von Haeseler A
AD  - Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories,
      University of Vienna and Medical University of Vienna, Vienna, Austria.
AD  - Bioinformatics &amp; Computational Biology, Faculty of Computer Science, University
      of Vienna, Vienna, Austria.
LA  - eng
PT  - Journal Article
DEP - 20190731
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
OTO - NOTNLM
OT  - Phylogenetics
OT  - convergent evolution
OT  - heterotachy
OT  - maximum likelihood
OT  - mixture model
EDAT- 2019/08/01 06:00
MHDA- 2019/08/01 06:00
CRDT- 2019/08/01 06:00
PHST- 2017/08/04 00:00 [received]
PHST- 2018/12/02 00:00 [revised]
PHST- 2019/07/18 00:00 [revised]
PHST- 2019/07/22 00:00 [accepted]
PHST- 2019/08/01 06:00 [entrez]
PHST- 2019/08/01 06:00 [pubmed]
PHST- 2019/08/01 06:00 [medline]
AID - 5541793 [pii]
AID - 10.1093/sysbio/syz051 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Jul 31. pii: 5541793. doi: 10.1093/sysbio/syz051.

PMID- 31364707
OWN - NLM
STAT- Publisher
LR  - 20190731
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Jul 31
TI  - Graph Splitting: A Graph-Based Approach for Superfamily-scale Phylogenetic Tree
      Reconstruction.
LID - syz049 [pii]
LID - 10.1093/sysbio/syz049 [doi]
AB  - A protein superfamily contains distantly related proteins that have acquired
      diverse biological functions through a long evolutionary history. Phylogenetic
      analysis of the early evolution of protein superfamilies is a key challenge
      because existing phylogenetic methods show poor performance when protein
      sequences are too diverged to construct an informative multiple sequence
      alignment. Here, we propose the Graph Splitting (GS) method, which rapidly
      reconstructs a protein superfamily-scale phylogenetic tree using a graph-based
      approach. Evolutionary simulation showed that the GS method can accurately
      reconstruct phylogenetic trees and be robust to major problems in phylogenetic
      estimation, such as biased taxon sampling, heterogeneous evolutionary rates, and 
      long-branch attraction when sequences are substantially diverged. Its application
      to an empirical dataset of the triosephosphate isomerase (TIM)-barrel superfamily
      suggests rapid evolution of protein-mediated pyrimidine biosynthesis, likely
      taking place after the RNA world. Furthermore, the GS method can also
      substantially improve performance of widely used multiple sequence alignment
      methods by providing accurate guide trees.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Matsui Su, Motomu
AU  - Matsui Su M
AD  - Department of Biological Sciences, Graduate School of Science, The University of 
      Tokyo, Bunkyo-ku, Tokyo 113-0032, Japan.
FAU - Iwasaki, Wataru
AU  - Iwasaki W
AD  - Department of Biological Sciences, Graduate School of Science, The University of 
      Tokyo, Bunkyo-ku, Tokyo 113-0032, Japan.
AD  - Department of Computational Biology and Medical Sciences, Graduate School of
      Frontier Sciences, The University of Tokyo, Kashiwa, Chiba 277-8568, Japan.
AD  - Atmosphere and Ocean Research Institute, The University of Tokyo, Kashiwa, Chiba 
      277-8564, Japan.
LA  - eng
PT  - Journal Article
DEP - 20190731
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
OTO - NOTNLM
OT  - Bioinformatics
OT  - Early evolution
OT  - Network analysis
OT  - Phylogenetic method
OT  - TIM-barrel superfamily
EDAT- 2019/08/01 06:00
MHDA- 2019/08/01 06:00
CRDT- 2019/08/01 06:00
PHST- 2018/11/07 00:00 [received]
PHST- 2019/03/18 00:00 [revised]
PHST- 2019/07/09 00:00 [revised]
PHST- 2019/07/20 00:00 [accepted]
PHST- 2019/08/01 06:00 [entrez]
PHST- 2019/08/01 06:00 [pubmed]
PHST- 2019/08/01 06:00 [medline]
AID - 5541794 [pii]
AID - 10.1093/sysbio/syz049 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Jul 31. pii: 5541794. doi: 10.1093/sysbio/syz049.

PMID- 31290974
OWN - NLM
STAT- Publisher
LR  - 20190710
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Jul 10
TI  - INSTRAL: Discordance-aware Phylogenetic Placement using Quartet Scores.
LID - syz045 [pii]
LID - 10.1093/sysbio/syz045 [doi]
AB  - Phylogenomic analyses have increasingly adopted species tree reconstruction using
      methods that account for gene tree discordance using pipelines that require both 
      human effort and computational resources. As the number of available genomes
      continues to increase, a new problem is facing researchers. Once more species
      become available, they have to repeat the whole process from the beginning
      because updating species trees is currently not possible. However, the de novo
      inference can be prohibitively costly in human effort or machine time. In this
      paper, we introduce INSTRAL, a method that extends ASTRAL to enable phylogenetic 
      placement. INSTRAL is designed to place a new species on an existing species tree
      after sequences from the new species have already been added to gene trees; thus,
      INSTRAL is complementary to existing placement methods that update gene trees.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Rabiee, Maryam
AU  - Rabiee M
AD  - Department of Computer Science and Engineering, UC San Diego, La Jolla, CA, USA.
FAU - Mirarab, Siavash
AU  - Mirarab S
AD  - Department of Electrical and Computer Engineering, UC San Diego, La Jolla, CA,
      USA.
LA  - eng
PT  - Journal Article
DEP - 20190710
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - ASTRAL
OT  - ILS
OT  - phylogenetic placement
OT  - species tree reconstruction
EDAT- 2019/07/11 06:00
MHDA- 2019/07/11 06:00
CRDT- 2019/07/11 06:00
PHST- 2018/11/04 00:00 [received]
PHST- 2019/03/20 00:00 [revised]
PHST- 2019/06/25 00:00 [revised]
PHST- 2019/07/02 00:00 [accepted]
PHST- 2019/07/11 06:00 [entrez]
PHST- 2019/07/11 06:00 [pubmed]
PHST- 2019/07/11 06:00 [medline]
AID - 5530610 [pii]
AID - 10.1093/sysbio/syz045 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Jul 10. pii: 5530610. doi: 10.1093/sysbio/syz045.

PMID- 31225616
OWN - NLM
STAT- Publisher
LR  - 20190621
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Jun 21
TI  - Phylogenomics Uncovers Confidence and Conflict in the Rapid Radiation of
      Australo-Papuan Rodents.
LID - syz044 [pii]
LID - 10.1093/sysbio/syz044 [doi]
AB  - The estimation of robust and accurate measures of branch support has proven
      challenging in the era of phylogenomics. In datasets of potentially millions of
      sites, bootstrap support for bifurcating relationships around very short internal
      branches can be inappropriately inflated. Such over-estimation of branch support 
      may be particularly problematic in rapid radiations, where phylogenetic signal is
      low and incomplete lineage sorting severe. Here, we explore this issue by
      comparing various branch support estimates under both concatenated and coalescent
      frameworks, in the recent radiation Australo-Papuan murine rodents (Muridae:
      Hydromyini). Using nucleotide sequence data from 1245 independent loci and
      several phylogenomic inference methods, we unequivocally resolve the majority of 
      genus-level relationships within Hydromyini. However, at four nodes we recover
      inconsistency in branch support estimates both within and among concatenated and 
      coalescent approaches. In most cases, concatenated likelihood approaches using
      standard fast bootstrap algorithms did not detect any uncertainty at these four
      nodes, regardless of partitioning strategy. However, we found this could be
      overcome with two-stage resampling, i.e. across genes and sites within genes
      (using -bsam GENESITE in IQtree). In addition, low confidence at recalcitrant
      nodes was recovered using UFBoot2, a recent revision to the bootstrap protocol in
      IQtree, but this depended on partitioning strategy. Summary coalescent approaches
      also failed to detect uncertainty under some circumstances. For each of four
      recalcitrant nodes, an equivalent (or close to equivalent) number of genes were
      in strong support (&gt; 75% bootstrap) of both the primary and at least one
      alternative topological hypothesis, suggesting notable phylogenetic conflict
      among loci not detected using some standard branch support metrics. Recent debate
      has focused on the appropriateness of concatenated versus multi-genealogical
      approaches to resolving species relationships, but less so on accurately
      estimating uncertainty in large datasets. Our results demonstrate the importance 
      of employing multiple approaches when assessing confidence, and highlight the
      need for greater attention to the development of robust measures of uncertainty
      in the era of phylogenomics.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Roycroft, Emily J
AU  - Roycroft EJ
AD  - School of BioSciences, The University of Melbourne, Parkville, Victoria,
      Australia.
AD  - Sciences Department, Museums Victoria, GPO Box 666, Melbourne, Victoria,
      Australia.
FAU - Moussalli, Adnan
AU  - Moussalli A
AD  - School of BioSciences, The University of Melbourne, Parkville, Victoria,
      Australia.
AD  - Sciences Department, Museums Victoria, GPO Box 666, Melbourne, Victoria,
      Australia.
FAU - Rowe, Kevin C
AU  - Rowe KC
AD  - School of BioSciences, The University of Melbourne, Parkville, Victoria,
      Australia.
AD  - Sciences Department, Museums Victoria, GPO Box 666, Melbourne, Victoria,
      Australia.
LA  - eng
PT  - Journal Article
DEP - 20190621
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Murinae
OT  - bootstrap
OT  - branch support
OT  - coalescence
OT  - concatenation
OT  - exon capture
OT  - incomplete lineage sorting
OT  - species tree
EDAT- 2019/06/22 06:00
MHDA- 2019/06/22 06:00
CRDT- 2019/06/22 06:00
PHST- 2018/08/18 00:00 [received]
PHST- 2019/03/15 00:00 [revised]
PHST- 2019/06/02 00:00 [revised]
PHST- 2019/06/12 00:00 [accepted]
PHST- 2019/06/22 06:00 [entrez]
PHST- 2019/06/22 06:00 [pubmed]
PHST- 2019/06/22 06:00 [medline]
AID - 5521411 [pii]
AID - 10.1093/sysbio/syz044 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Jun 21. pii: 5521411. doi: 10.1093/sysbio/syz044.

PMID- 31188455
OWN - NLM
STAT- Publisher
LR  - 20190612
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Apr 23
TI  - The Origins and Diversification of the Exceptionally Rich Gemsnakes (Colubroidea:
      Lamprophiidae: Pseudoxyrhophiinae) in Madagascar.
LID - syz026 [pii]
LID - 10.1093/sysbio/syz026 [doi]
AB  - Processes leading to spectacular diversity of both form and species on islands
      have been well-documented under island biogeography theory, where distance from
      source and island size are key factors determining immigration and extinction
      resistance. But far less understood are the processes governing in situ
      diversification on the world's mega islands, where large and isolated land masses
      produced morphologically distinct radiations from related taxa on continental
      regions. Madagascar has long been recognized as a natural laboratory due to its
      isolation, lack of influence from adjacent continents, and diversification of
      spectacular vertebrate radiations. However, only a handful of studies have
      examined rate shifts of in situ diversification for this island. Here, we examine
      rates of diversification in the Malagasy snakes of the family Pseudoxyrhophiinae 
      (gemsnakes) to understand if rates of speciation were initially high, enhanced by
      diversification into distinct biomes, and associated with key dentition traits.
      Using a genomic sequence-capture data set for 366 samples, we determine that all 
      previously described and newly discovered species are delimitable and therefore
      useful candidates for understanding diversification trajectories through time.
      Our analysis detected no shifts in diversification rate between clades or changes
      in biome or dentition type. Remarkably, we demonstrate that rates of
      diversification of the gemsnake radiation, which originated in Madagascar during 
      the early Miocene, remained steady throughout the Neogene. However, we do detect 
      a significant slowdown in diversification during the Pleistocene. We also comment
      on the apparent paradox where most living species originated in the Pleistocene, 
      despite diversification rates being substantially higher during the earlier 15
      myr. [Gemsnakes; in situ diversification; island biogeography; Neogene;
      Pseudoxyrhophiinae; speciation.].
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Burbrink, Frank T
AU  - Burbrink FT
AD  - Department of Herpetology, The American Museum of Natural History, 79th Street at
      Central Park West, New York, NY 10024, USA.
FAU - Ruane, Sara
AU  - Ruane S
AD  - Department of Biological Sciences, 206 Boyden Hall, Rutgers University-Newark,
      195 University Ave, Newark, NJ 07102, USA.
FAU - Kuhn, Arianna
AU  - Kuhn A
AD  - Department of Herpetology, The American Museum of Natural History, 79th Street at
      Central Park West, New York, NY 10024, USA.
AD  - Department of Biology, The Graduate School and University Center, The City
      University of New York, 365 Fifth Ave., New York, NY 10016, USA.
FAU - Rabibisoa, Nirhy
AU  - Rabibisoa N
AD  - Mention Sciences de la Vie et de l'Environnement, Faculte des Sciences, de
      Technologies et de l'Environnement, Universite de Mahajanga, Campus Universitaire
      d'Ambondrona, BP 652, Mahajanga 401, Madagascar.
FAU - Randriamahatantsoa, Bernard
AU  - Randriamahatantsoa B
AD  - Mention Sciences de la Vie et de l'Environnement, Faculte des Sciences, de
      Technologies et de l'Environnement, Universite de Mahajanga, Campus Universitaire
      d'Ambondrona, BP 652, Mahajanga 401, Madagascar.
FAU - Raselimanana, Achille P
AU  - Raselimanana AP
AD  - Mention: Zoologie et Biodiversite Animale, Faculte des Sciences, Universite
      d'Antananarivo, BP 906, Antananarivo 101, Madagascar.
FAU - Andrianarimalala, Mamy S M
AU  - Andrianarimalala MSM
AD  - Mention: Zoologie et Biodiversite Animale, Faculte des Sciences, Universite
      d'Antananarivo, BP 906, Antananarivo 101, Madagascar.
FAU - Cadle, John E
AU  - Cadle JE
AD  - Department of Biology, East Georgia State College, Swainsboro, GA 30401, USA.
FAU - Lemmon, Alan R
AU  - Lemmon AR
AD  - Department of Scientific Computing, Florida State University, Dirac Science
      Library, Tallahassee, FL 32306-4102, USA.
FAU - Lemmon, Emily Moriarty
AU  - Lemmon EM
AD  - Department of Biological Science, Florida State University, 319 Stadium Drive,
      Tallahassee, FL 32306-4295, USA.
FAU - Nussbaum, Ronald A
AU  - Nussbaum RA
AD  - Division of Reptiles and Amphibians, Museum of Zoology, Research Museums Center, 
      3600 Varsity Drive, University of Michigan, Ann Arbor, MI 48108, USA.
FAU - Jones, Leonard N
AU  - Jones LN
AD  - Department of Biology, University of Washington, Seattle, WA 98195-1800, USA.
FAU - Pearson, Richard
AU  - Pearson R
AD  - Centre for Biodiversity &amp; Environment Research, Department of Genetics, Evolution
      and Environment, University College London, Gower Street, London WC1E 6BT, UK.
FAU - Raxworthy, Christopher J
AU  - Raxworthy CJ
AD  - Department of Herpetology, The American Museum of Natural History, 79th Street at
      Central Park West, New York, NY 10024, USA.
LA  - eng
PT  - Journal Article
DEP - 20190423
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/06/13 06:00
MHDA- 2019/06/13 06:00
CRDT- 2019/06/13 06:00
PHST- 2018/06/26 00:00 [received]
PHST- 2019/04/05 00:00 [revised]
PHST- 2019/04/09 00:00 [accepted]
PHST- 2019/06/13 06:00 [entrez]
PHST- 2019/06/13 06:00 [pubmed]
PHST- 2019/06/13 06:00 [medline]
AID - 5477410 [pii]
AID - 10.1093/sysbio/syz026 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Apr 23. pii: 5477410. doi: 10.1093/sysbio/syz026.

PMID- 31180508
OWN - NLM
STAT- Publisher
LR  - 20190610
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Jun 10
TI  - The Multispecies Coalescent Over-splits Species in the Case of Geographically
      Widespread Taxa.
LID - syz042 [pii]
LID - 10.1093/sysbio/syz042 [doi]
AB  - Many recent species delimitation studies rely exclusively on limited analyses of 
      genetic data analyzed under the multispecies coalescent (MSC) model, and results 
      from these studies often are regarded as conclusive support for taxonomic
      changes. However, most MSC-based species delimitation methods have well-known and
      often unmet assumptions. Uncritical application of these genetic-based approaches
      (without due consideration of sampling design, the effects of a priori group
      designations, isolation by distance, cytoplasmic-nuclear mismatch, and population
      structure) can lead to over-splitting of species. Here, we argue that in many
      common biological scenarios, researchers must be particularly cautious regarding 
      these limitations, especially in cases of well-studied, geographically variable, 
      and parapatrically-distributed species complexes. We consider these points with
      respect to a historically controversial species group, the American milksnakes
      (Lampropeltis triangulum complex), using genetic data from a recent analysis
      (Ruane et al. 2014; Syst. Biol. 63:231-250). We show that over-reliance on the
      program BPP, without adequate consideration of its assumptions and of sampling
      limitations, resulted in over-splitting of species in this study. Several of the 
      hypothesized species of milksnakes instead appear to represent arbitrary slices
      of continuous geographic clines. We conclude that the best available evidence
      supports three, rather than seven, species within this complex. More generally,
      we recommend that coalescent-based species delimitation studies incorporate
      thorough analyses of geographic variation and carefully examine putative contact 
      zones among delimited species before making taxonomic changes.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Chambers, E Anne
AU  - Chambers EA
AD  - Department of Integrative Biology and Biodiversity Center, The University of
      Texas at Austin, Austin TX USA.
FAU - Hillis, David M
AU  - Hillis DM
AD  - Department of Integrative Biology and Biodiversity Center, The University of
      Texas at Austin, Austin TX USA.
LA  - eng
PT  - Journal Article
DEP - 20190610
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Speciation
OT  - classification
OT  - species concepts
OT  - species delimitation
OT  - taxonomy
EDAT- 2019/06/11 06:00
MHDA- 2019/06/11 06:00
CRDT- 2019/06/11 06:00
PHST- 2018/11/19 00:00 [received]
PHST- 2019/05/22 00:00 [revised]
PHST- 2019/05/24 00:00 [accepted]
PHST- 2019/06/11 06:00 [entrez]
PHST- 2019/06/11 06:00 [pubmed]
PHST- 2019/06/11 06:00 [medline]
AID - 5513370 [pii]
AID - 10.1093/sysbio/syz042 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Jun 10. pii: 5513370. doi: 10.1093/sysbio/syz042.

PMID- 31173141
OWN - NLM
STAT- Publisher
LR  - 20190607
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Jun 6
TI  - An Annealed Sequential Monte Carlo Method for Bayesian Phylogenetics.
LID - syz028 [pii]
LID - 10.1093/sysbio/syz028 [doi]
AB  - We describe an "embarrassingly parallel" method for Bayesian phylogenetic
      inference, annealed Sequential Monte Carlo (SMC), based on recent advances in the
      SMC literature such as adaptive determination of annealing parameters. The
      algorithm provides an approximate posterior distribution over trees and
      evolutionary parameters as well as an unbiased estimator for the marginal
      likelihood. This unbiasedness property can be used for the purpose of testing the
      correctness of posterior simulation software. We evaluate the performance of
      phylogenetic annealed SMC by reviewing and comparing with other computational
      Bayesian phylogenetic methods, in particular, different marginal likelihood
      estimation methods. Unlike previous SMC methods in phylogenetics, our annealed
      method can utilize standard Markov chain Monte Carlo (MCMC) tree moves and hence 
      benefit from the large inventory of such moves available in the literature.
      Consequently, the annealed SMC method should be relatively easy to incorporate
      into existing phylogenetic software packages based on MCMC algorithms. We
      illustrate our method using simulation studies and real data analysis. [Marginal 
      likelihood; phylogenetics; Sequential Monte Carlo.].
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Wang, Liangliang
AU  - Wang L
AD  - Department of Statistics and Actuarial Science, Simon Fraser University, Burnaby,
      British Columbia V5A 1S6, Canada.
FAU - Wang, Shijia
AU  - Wang S
AD  - Department of Statistics and Actuarial Science, Simon Fraser University, Burnaby,
      British Columbia V5A 1S6, Canada.
FAU - Bouchard-Cote, Alexandre
AU  - Bouchard-Cote A
AD  - Department of Statistics, University of British Columbia, Vancouver, British
      Columbia V6T 1Z4, Canada.
LA  - eng
PT  - Journal Article
DEP - 20190606
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/06/08 06:00
MHDA- 2019/06/08 06:00
CRDT- 2019/06/08 06:00
PHST- 2018/06/01 00:00 [received]
PHST- 2019/04/12 00:00 [revised]
PHST- 2019/04/20 00:00 [accepted]
PHST- 2019/06/08 06:00 [entrez]
PHST- 2019/06/08 06:00 [pubmed]
PHST- 2019/06/08 06:00 [medline]
AID - 5484767 [pii]
AID - 10.1093/sysbio/syz028 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Jun 6. pii: 5484767. doi: 10.1093/sysbio/syz028.

PMID- 31165170
OWN - NLM
STAT- Publisher
LR  - 20190605
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Jun 4
TI  - Monophyly, Taxon Sampling, and the Nature of Ranks in the Classification of
      Orb-weaving Spiders (Araneae: Araneoidea).
LID - syz043 [pii]
LID - 10.1093/sysbio/syz043 [doi]
AB  - We address some of the taxonomic and classification changes proposed by Kuntner
      et al. (in press) in a comparative study on the evolution of sexual size
      dimorphism in nephiline spiders. Their proposal to re-circumscribe araneids and
      to rank the subfamily Nephilinae is fundamentally flawed as it renders the family
      Araneidae paraphyletic. We discuss the importance of monophyly, outgroup
      selection and taxon sampling, the subjectivity of ranks, and the implications of 
      the age of origin criterion to assign categorical ranks in biological
      classifications. We explore the outcome of applying the approach of Kuntner et
      al. (in press) to the classification of spiders with emphasis on the ecribellate 
      orb-weavers (Araneoidea) using a recently published dated phylogeny. We discuss
      the implications of including the putative sister group of Nephilinae (the
      sexually dimorphic genus Paraplectanoides) and the putative sister group of
      Araneidae (the miniature, monomorphic family Theridiosomatidae). We propose
      continuation of the phylogenetic classification put forth by Dimitrov et al.
      (2017) and we formally rank Nephilinae and Phonognathinae as subfamilies of
      Araneidae. Our classification better reflects the understanding of the
      phylogenetic placement and evolutionary history of nephilines and phonognathines 
      while maintaining the diagnosability of Nephilinae. It also fulfills the
      fundamental requirement that taxa must be monophyletic, and thus avoids the
      paraphyly of Araneidae implied by Kuntner et al. (in press).
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Kallal, Robert J
AU  - Kallal RJ
AD  - Department of Biological Sciences, The George Washington University, 2029 G St.
      NW, Washington, D.C., USA.
FAU - Dimitrov, Dimitar
AU  - Dimitrov D
AD  - Department of Natural History, University Museum of Bergen, University of Bergen,
      P.O. Box 7800, Bergen, Norway.
FAU - Arnedo, Miquel
AU  - Arnedo M
AD  - Department of Evolutionary Biology, Ecology and Environmental Sciences, &amp;
      Biodiversity Research Institute (IRBio) Universitat de Barcelona, Avinguda
      Diagonal 643, Barcelona, Spain.
AD  - Museum of Comparative Zoology, Department of Organismic and Evolutionary Biology,
      Harvard University, 26 Oxford Street, Cambridge, MA, USA.
FAU - Giribet, Gonzalo
AU  - Giribet G
AD  - Museum of Comparative Zoology, Department of Organismic and Evolutionary Biology,
      Harvard University, 26 Oxford Street, Cambridge, MA, USA.
FAU - Hormiga, Gustavo
AU  - Hormiga G
AD  - Department of Biological Sciences, The George Washington University, 2029 G St.
      NW, Washington, D.C., USA.
LA  - eng
PT  - Journal Article
DEP - 20190604
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Araneidae
OT  - Nephilinae
OT  - nomenclature
OT  - taxonomy
OT  - time banding
EDAT- 2019/06/06 06:00
MHDA- 2019/06/06 06:00
CRDT- 2019/06/06 06:00
PHST- 2019/01/25 00:00 [received]
PHST- 2019/04/09 00:00 [revised]
PHST- 2019/05/21 00:00 [revised]
PHST- 2019/05/24 00:00 [accepted]
PHST- 2019/06/06 06:00 [entrez]
PHST- 2019/06/06 06:00 [pubmed]
PHST- 2019/06/06 06:00 [medline]
AID - 5511413 [pii]
AID - 10.1093/sysbio/syz043 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Jun 4. pii: 5511413. doi: 10.1093/sysbio/syz043.

PMID- 31165169
OWN - NLM
STAT- Publisher
LR  - 20190605
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Jun 4
TI  - Mean and Variance of Phylogenetic Trees.
LID - syz041 [pii]
LID - 10.1093/sysbio/syz041 [doi]
AB  - We describe the use of the Frechet mean and variance in the
      Billera-Holmes-Vogtmann (BHV) treespace to summarize and explore the diversity of
      a set of phylogenetic trees. We show that the Frechet mean is comparable to other
      summary methods, and, despite its stickiness property, is more likely to be
      binary than the majority-rule consensus tree. We show that the Frechet variance
      is faster and more precise than commonly used variance measures. The Frechet mean
      and variance are more theoretically justified, and more robust, than previous
      estimates of this type, and can be estimated reasonably efficiently, providing a 
      foundation for building more advanced statistical methods and leading to
      applications such as mean hypothesis testing and outlier detection.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Brown, Daniel G
AU  - Brown DG
AD  - David R. Cheriton School of Computer Science, University of Waterloo, Waterloo,
      N2L 3G1, Canada.
FAU - Owen, Megan
AU  - Owen M
AD  - Department of Mathematics, Lehman College - City University of New York, New
      York, 10468, USA.
LA  - eng
PT  - Journal Article
DEP - 20190604
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Frechet mean
OT  - Frechet variance
OT  - phylogenetics
OT  - treespace
OT  - two-sample test
EDAT- 2019/06/06 06:00
MHDA- 2019/06/06 06:00
CRDT- 2019/06/06 06:00
PHST- 2017/08/02 00:00 [received]
PHST- 2018/05/10 00:00 [revised]
PHST- 2018/12/17 00:00 [revised]
PHST- 2019/05/13 00:00 [revised]
PHST- 2019/05/24 00:00 [accepted]
PHST- 2019/06/06 06:00 [entrez]
PHST- 2019/06/06 06:00 [pubmed]
PHST- 2019/06/06 06:00 [medline]
AID - 5511414 [pii]
AID - 10.1093/sysbio/syz041 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Jun 4. pii: 5511414. doi: 10.1093/sysbio/syz041.

PMID- 31157892
OWN - NLM
STAT- Publisher
LR  - 20190603
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Apr 23
TI  - Metapopulation Vicariance, Age of Island Taxa and Dispersal: A Case Study Using
      the Pacific Plant Genus Planchonella (Sapotaceae).
LID - syz025 [pii]
LID - 10.1093/sysbio/syz025 [doi]
AB  - Oceanic islands originate from volcanism or tectonic activity without connections
      to continental landmasses, are colonized by organisms, and eventually vanish due 
      to erosion and subsidence. Colonization of oceanic islands occurs through
      long-distance dispersals (LDDs) or metapopulation vicariance, the latter
      resulting in lineages being older than the islands they inhabit. If
      metapopulation vicariance is valid, island ages cannot be reliably used to
      provide maximum age constraints for molecular dating. We explore the
      relationships between the ages of members of a widespread plant genus
      (Planchonella, Sapotaceae) and their host islands across the Pacific to test
      various assumptions of dispersal and metapopulation vicariance. We sampled three 
      nuclear DNA markers from 156 accessions representing some 100 Sapotaceae taxa,
      and analyzed these in BEAST with a relaxed clock to estimate divergence times and
      with a phylogeographic diffusion model to estimate range expansions over time.
      The phylogeny was calibrated with a secondary point (the root) and fossils from
      New Zealand. The dated phylogeny reveals that the ages of Planchonella species
      are, in most cases, consistent with the ages of the islands they inhabit.
      Planchonella is inferred to have originated in the Sahul Shelf region, to which
      it back-dispersed multiple times. Fiji has been an important source for range
      expansion in the Pacific for the past 23 myr. Our analyses reject metapopulation 
      vicariance in all cases tested, including between oceanic islands, evolution of
      an endemic Fiji-Vanuatu flora, and westward rollback vicariance between Vanuatu
      and the Loyalty Islands. Repeated dispersal is the only mechanism able to explain
      the empirical data. The longest (8900 km) identified dispersal is between Palau
      in the Pacific and the Seychelles in the Indian Ocean, estimated at 2.2 Ma
      (0.4-4.8 Ma). The first split in a Hawaiian lineage (P. sandwicensis) matches the
      age of Necker Island (11.0 Ma), when its ancestor diverged into two species that 
      are distinguished by purple and yellow fruits. Subsequent establishment across
      the Hawaiian archipelago supports, in part, progression rule colonization. In
      summary, we found no explanatory power in metapopulation vicariance and conclude 
      that Planchonella has expanded its range across the Pacific by LDD. We contend
      that this will be seen in many other groups when analyzed in detail. [Dispersal; 
      divergence times; Fiji; Hawaii; metapopulation vicariance; molecular dating;
      Society Islands; Vanuatu.].
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Swenson, Ulf
AU  - Swenson U
AD  - Department of Botany, Swedish Museum of Natural History, Box 50007, SE-104 05
      Stockholm, Sweden.
FAU - Havran, J Christopher
AU  - Havran JC
AD  - Department of Biological Sciences, Campbell University, 205 Day Dorm Road, Buies 
      Creek, NC 27506, USA.
FAU - Munzinger, Jerome
AU  - Munzinger J
AD  - AMAP, IRD, CIRAD, CNRS, INRA, Universite Montpellier, 34000 Montpellier, France.
FAU - Mcloughlin, Stephen
AU  - Mcloughlin S
AD  - Department of Palaeobiology, Swedish Museum of Natural History, Box 50007, SE-104
      05 Stockholm, Sweden.
FAU - Nylinder, Stephan
AU  - Nylinder S
AD  - Department of Psychology, Gothenburg University, Box 500, SE-405 30 Goteborg,
      Sweden.
LA  - eng
PT  - Journal Article
DEP - 20190423
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/06/04 06:00
MHDA- 2019/06/04 06:00
CRDT- 2019/06/04 06:00
PHST- 2018/04/10 00:00 [received]
PHST- 2019/03/27 00:00 [revised]
PHST- 2019/04/09 00:00 [accepted]
PHST- 2019/06/04 06:00 [entrez]
PHST- 2019/06/04 06:00 [pubmed]
PHST- 2019/06/04 06:00 [medline]
AID - 5477413 [pii]
AID - 10.1093/sysbio/syz025 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Apr 23. pii: 5477413. doi: 10.1093/sysbio/syz025.

PMID- 31140573
OWN - NLM
STAT- Publisher
LR  - 20190529
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Apr 23
TI  - Sexual Dichromatism Drives Diversification within a Major Radiation of African
      Amphibians.
LID - syz023 [pii]
LID - 10.1093/sysbio/syz023 [doi]
AB  - Theory predicts that sexually dimorphic traits under strong sexual selection,
      particularly those involved with intersexual signaling, can accelerate speciation
      and produce bursts of diversification. Sexual dichromatism (sexual dimorphism in 
      color) is widely used as a proxy for sexual selection and is associated with
      rapid diversification in several animal groups, yet studies using phylogenetic
      comparative methods to explicitly test for an association between sexual
      dichromatism and diversification have produced conflicting results. Sexual
      dichromatism is rare in frogs, but it is both striking and prevalent in African
      reed frogs, a major component of the diverse frog radiation termed Afrobatrachia.
      In contrast to most other vertebrates, reed frogs display female-biased
      dichromatism in which females undergo color transformation, often resulting in
      more ornate coloration in females than in males. We produce a robust phylogeny of
      Afrobatrachia to investigate the evolutionary origins of sexual dichromatism in
      this radiation and examine whether the presence of dichromatism is associated
      with increased rates of net diversification. We find that sexual dichromatism
      evolved once within hyperoliids and was followed by numerous independent
      reversals to monochromatism. We detect significant diversification rate
      heterogeneity in Afrobatrachia and find that sexually dichromatic lineages have
      double the average net diversification rate of monochromatic lineages. By
      conducting trait simulations on our empirical phylogeny, we demonstrate that our 
      inference of trait-dependent diversification is robust. Although sexual
      dichromatism in hyperoliid frogs is linked to their rapid diversification and
      supports macroevolutionary predictions of speciation by sexual selection, the
      function of dichromatism in reed frogs remains unclear. We propose that reed
      frogs are a compelling system for studying the roles of natural and sexual
      selection on the evolution of sexual dichromatism across micro- and
      macroevolutionary timescales. [Afrobatrachia; Anura; color evolution;
      diversification; macroevolution; sexual selection.].
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Portik, Daniel M
AU  - Portik DM
AD  - Museum of Vertebrate Zoology, University of California, Berkeley, CA 94720, USA.
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA.
FAU - Bell, Rayna C
AU  - Bell RC
AD  - Museum of Vertebrate Zoology, University of California, Berkeley, CA 94720, USA.
AD  - Department of Vertebrate Zoology, National Museum of Natural History, Smithsonian
      Institution, Washington, DC 20560-0162, USA.
FAU - Blackburn, David C
AU  - Blackburn DC
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL 32611, 
      USA.
FAU - Bauer, Aaron M
AU  - Bauer AM
AD  - Department of Biology, Villanova University, 800 Lancaster Avenue, Villanova, PA 
      19085, USA.
FAU - Barratt, Christopher D
AU  - Barratt CD
AD  - Department of Environmental Sciences, University of Basel, Basel 4056,
      Switzerland.
AD  - German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,
      Leipzig 0413, Germany.
AD  - Max Planck Institute for Evolutionary Anthropology, Leipzig 0413, Germany.
FAU - Branch, William R
AU  - Branch WR
AD  - Port Elizabeth Museum, P.O. Box 11347, Humewood 6013, South Africa.
AD  - Department of Zoology, Nelson Mandela Metropolitan University, P.O. Box 77000,
      Port Elizabeth 6031, South Africa.
FAU - Burger, Marius
AU  - Burger M
AD  - African Amphibian Conservation Research Group, Unit for Environmental Sciences
      and Management, North-West University, Potchefstroom 2520, South Africa.
AD  - Flora Fauna &amp; Man, Ecological Services Ltd. Tortola, British Virgin, Island.
FAU - Channing, Alan
AU  - Channing A
AD  - Unit for Environmental Sciences and Management, North-West University,
      Potchefstroom 2520, South Africa.
FAU - Colston, Timothy J
AU  - Colston TJ
AD  - Department of Biological Sciences, Florida State University, Tallahassee, FL
      32306, USA.
AD  - Zoological Natural History Museum, Addis Ababa University, Arat Kilo, Addis
      Ababa, Ethiopia.
FAU - Conradie, Werner
AU  - Conradie W
AD  - Port Elizabeth Museum, P.O. Box 11347, Humewood 6013, South Africa.
AD  - School of Natural Resource Management, Nelson Mandela University, George Campus, 
      George 6530, South Africa.
FAU - Dehling, J Maximilian
AU  - Dehling JM
AD  - Department of Biology, Institute of Sciences, University of Koblenz-Landau,
      Universitatsstr. 1, D-56070 Koblenz, Germany.
FAU - Drewes, Robert C
AU  - Drewes RC
AD  - California Academy of Sciences, San Francisco, CA 94118, USA.
FAU - Ernst, Raffael
AU  - Ernst R
AD  - Museum of Zoology, Senckenberg Natural History Collections Dresden, Konigsbrucker
      Landstr. 159, Dresden 01109, Germany.
AD  - Department of Ecology, Technische Universitat Berlin, Rothenburgstr. 12, Berlin
      12165, Germany.
FAU - Greenbaum, Eli
AU  - Greenbaum E
AD  - Department of Biological Sciences, University of Texas at El Paso, El Paso, TX
      79968, USA.
FAU - Gvozdik, Vaclav
AU  - Gvozdik V
AD  - The Czech Academy of Sciences, Institute of Vertebrate Biology, Brno, Czech
      Republic.
AD  - Department of Zoology, National Museum, Prague, Czech Republic.
FAU - Harvey, James
AU  - Harvey J
AD  - Pietermaritzburg, KwaZulu-Natal, South Africa.
FAU - Hillers, Annika
AU  - Hillers A
AD  - Museum fur Naturkunde, Leibniz Institute for Evolution and Biodiversity Science, 
      Biodiversity Dynamics, Invalidenstr. 43, Berlin 10115, Germany.
AD  - Across the River - A Transboundary Peace Park for Sierra Leone and Liberia, The
      Royal Society for the Protection of Birds, 164 Dama Road, Kenema, Sierra Leone.
FAU - Hirschfeld, Mareike
AU  - Hirschfeld M
AD  - Museum fur Naturkunde, Leibniz Institute for Evolution and Biodiversity Science, 
      Biodiversity Dynamics, Invalidenstr. 43, Berlin 10115, Germany.
FAU - Jongsma, Gregory F M
AU  - Jongsma GFM
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL 32611, 
      USA.
FAU - Kielgast, Jos
AU  - Kielgast J
AD  - Natural History Museum of Denmark, University of Copenhagen, Universitetsparken
      15, Copenhagen 2100, Denmark.
FAU - Kouete, Marcel T
AU  - Kouete MT
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL 32611, 
      USA.
FAU - Lawson, Lucinda P
AU  - Lawson LP
AD  - Department of Biological Sciences, University of Cincinnati, 614 Rieveschl Hall, 
      Cincinnati, OH 45220, USA.
AD  - Life Sciences, Field Museum of Natural History, 1400 S. Lake Shore Dr., Chicago, 
      IL 60605, USA.
FAU - Leache, Adam D
AU  - Leache AD
AD  - Department of Biology, Burke Museum of Natural History and Culture, University of
      Washington, Seattle, WA, USA.
FAU - Loader, Simon P
AU  - Loader SP
AD  - Life Sciences Department, Natural History Museum, London SW7 5BD, UK.
FAU - Lotters, Stefan
AU  - Lotters S
AD  - Biogeography Department, Trier University, Universitatsring 15, Trier 54296,
      Germany.
FAU - Meijden, Arie Van Der
AU  - Meijden AV
AD  - CIBIO Research Centre in Biodiversity and Genetic Resources, InBIO, Universidade 
      do Porto, Campus Agrario de Vairao, Rua Padre Armando Quintas, No. 7, 4485-661
      Vairao, Vila do Conde, Portugal.
FAU - Menegon, Michele
AU  - Menegon M
AD  - Tropical Biodiversity Section, Science Museum of Trento, Corso del lavoro e della
      Scienza 3, Trento 38122, Italy.
FAU - Muller, Susanne
AU  - Muller S
AD  - Biogeography Department, Trier University, Universitatsring 15, Trier 54296,
      Germany.
FAU - Nagy, Zoltan T
AU  - Nagy ZT
AD  - Royal Belgian Institute of Natural Sciences, OD Taxonomy and Phylogeny, Rue
      Vautier 29, B-1000 Brussels, Belgium.
FAU - Ofori-Boateng, Caleb
AU  - Ofori-Boateng C
AD  - Forestry Research Institute of Ghana, P.O. Box 63, Fumesua, Kumasi, Ghana.
FAU - Ohler, Annemarie
AU  - Ohler A
AD  - Departement Origines et Evolution, Museum National d'Histoire Naturelle, UMR 7205
      ISYEB, 25 rue Cuvier, Paris 75005, France.
FAU - Papenfuss, Theodore J
AU  - Papenfuss TJ
AD  - Museum of Vertebrate Zoology, University of California, Berkeley, CA 94720, USA.
FAU - Rossler, Daniela
AU  - Rossler D
AD  - Biogeography Department, Trier University, Universitatsring 15, Trier 54296,
      Germany.
FAU - Sinsch, Ulrich
AU  - Sinsch U
AD  - Department of Biology, Institute of Sciences, University of Koblenz-Landau,
      Universitatsstr. 1, D-56070 Koblenz, Germany.
FAU - Rodel, Mark-Oliver
AU  - Rodel MO
AD  - Museum fur Naturkunde, Leibniz Institute for Evolution and Biodiversity Science, 
      Biodiversity Dynamics, Invalidenstr. 43, Berlin 10115, Germany.
FAU - Veith, Michael
AU  - Veith M
AD  - Biogeography Department, Trier University, Universitatsring 15, Trier 54296,
      Germany.
FAU - Vindum, Jens
AU  - Vindum J
AD  - California Academy of Sciences, San Francisco, CA 94118, USA.
FAU - Zassi-Boulou, Ange-Ghislain
AU  - Zassi-Boulou AG
AD  - Institut National de Recherche en Sciences Exactes et Naturelles, Brazzaville BP 
      2400, Republique du Congo.
FAU - McGuire, Jimmy A
AU  - McGuire JA
AD  - Museum of Vertebrate Zoology, University of California, Berkeley, CA 94720, USA.
LA  - eng
PT  - Journal Article
DEP - 20190423
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/05/30 06:00
MHDA- 2019/05/30 06:00
CRDT- 2019/05/30 06:00
PHST- 2018/07/27 00:00 [received]
PHST- 2019/02/15 00:00 [revised]
PHST- 2019/03/18 00:00 [revised]
PHST- 2019/04/09 00:00 [accepted]
PHST- 2019/05/30 06:00 [entrez]
PHST- 2019/05/30 06:00 [pubmed]
PHST- 2019/05/30 06:00 [medline]
AID - 5477408 [pii]
AID - 10.1093/sysbio/syz023 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Apr 23. pii: 5477408. doi: 10.1093/sysbio/syz023.

PMID- 31140564
OWN - NLM
STAT- Publisher
LR  - 20190529
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Apr 23
TI  - The Relative Importance of Modeling Site Pattern Heterogeneity Versus
      Partition-Wise Heterotachy in Phylogenomic Inference.
LID - syz021 [pii]
LID - 10.1093/sysbio/syz021 [doi]
AB  - Large taxa-rich genome-scale data sets are often necessary for resolving ancient 
      phylogenetic relationships. But accurate phylogenetic inference requires that
      they are analyzed with realistic models that account for the heterogeneity in
      substitution patterns amongst the sites, genes and lineages. Two kinds of
      adjustments are frequently used: models that account for heterogeneity in amino
      acid frequencies at sites in proteins, and partitioned models that accommodate
      the heterogeneity in rates (branch lengths) among different proteins in different
      lineages (protein-wise heterotachy). Although partitioned and site-heterogeneous 
      models are both widely used in isolation, their relative importance to the
      inference of correct phylogenies has not been carefully evaluated. We conducted
      several empirical analyses and a large set of simulations to compare the relative
      performances of partitioned models, site-heterogeneous models, and combined
      partitioned site heterogeneous models. In general, site-homogeneous models
      (partitioned or not) performed worse than site heterogeneous, except in
      simulations with extreme protein-wise heterotachy. Furthermore, simulations using
      empirically-derived realistic parameter settings showed a marked long-branch
      attraction (LBA) problem for analyses employing protein-wise partitioning even
      when the generating model included partitioning. This LBA problem results from a 
      small sample bias compounded over many single protein alignments. In some cases, 
      this problem was ameliorated by clustering similarly-evolving proteins together
      into larger partitions using the PartitionFinder method. Similar results were
      obtained under simulations with larger numbers of taxa or heterogeneity in
      simulating topologies over genes. For an empirical Microsporidia test data set,
      all but one tested site-heterogeneous models (with or without partitioning)
      obtain the correct Microsporidia+Fungi grouping, whereas site-homogenous models
      (with or without partitioning) did not. The single exception was the fully
      partitioned site-heterogeneous analysis that succumbed to the compounded small
      sample LBA bias. In general unless protein-wise heterotachy effects are extreme, 
      it is more important to model site-heterogeneity than protein-wise heterotachy in
      phylogenomic analyses. Complete protein-wise partitioning should be avoided as it
      can lead to a serious LBA bias. In cases of extreme protein-wise heterotachy,
      approaches that cluster similarly-evolving proteins together and coupled with
      site-heterogeneous models work well for phylogenetic estimation. [Gene tree
      heterogeneity; heterotachy; long-branch attraction; mixture model; partition
      model; site heterogeneity.].
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Wang, Huai-Chun
AU  - Wang HC
AD  - Department of Mathematics and Statistics, Dalhousie University, 6316 Coburg Road,
      Halifax, Nova Scotia B3H 4R2, Canada.
AD  - Centre for Comparative Genomics and Evolutionary Bioinformatics, Dalhousie
      University, 5850 College Street, Halifax, Nova Scotia B3H 4R2, Canada.
FAU - Susko, Edward
AU  - Susko E
AD  - Department of Mathematics and Statistics, Dalhousie University, 6316 Coburg Road,
      Halifax, Nova Scotia B3H 4R2, Canada.
AD  - Centre for Comparative Genomics and Evolutionary Bioinformatics, Dalhousie
      University, 5850 College Street, Halifax, Nova Scotia B3H 4R2, Canada.
FAU - Roger, Andrew J
AU  - Roger AJ
AD  - Centre for Comparative Genomics and Evolutionary Bioinformatics, Dalhousie
      University, 5850 College Street, Halifax, Nova Scotia B3H 4R2, Canada.
AD  - Department of Biochemistry and Molecular Biology, Dalhousie University, 5850
      College Street, Halifax, Nova Scotia B3H 4R2, Canada.
LA  - eng
PT  - Journal Article
DEP - 20190423
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/05/30 06:00
MHDA- 2019/05/30 06:00
CRDT- 2019/05/30 06:00
PHST- 2018/07/16 00:00 [received]
PHST- 2019/02/04 00:00 [revised]
PHST- 2019/03/10 00:00 [revised]
PHST- 2019/04/09 00:00 [accepted]
PHST- 2019/05/30 06:00 [entrez]
PHST- 2019/05/30 06:00 [pubmed]
PHST- 2019/05/30 06:00 [medline]
AID - 5477411 [pii]
AID - 10.1093/sysbio/syz021 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Apr 23. pii: 5477411. doi: 10.1093/sysbio/syz021.

PMID- 31135923
OWN - NLM
STAT- Publisher
LR  - 20190528
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 May 27
TI  - Biogeographical Network Analysis of Cretaceous Terrestrial Tetrapods: A
      Phylogeny-Based Approach.
LID - syz024 [pii]
LID - 10.1093/sysbio/syz024 [doi]
AB  - Network methods are widely used to represent and analyze biogeography. It is
      difficult, however, to convert occurrence data of fossil vertebrates to a
      biogeographical network, as most species were known from a single locality. A new
      method for creating a biogeographical network that can incorporate phylogenetic
      information is proposed in this study, which increases the number of edges in the
      network of fossil vertebrates and enables the application of various network
      methods. Using ancestral state reconstruction via maximum parsimony, the method
      first estimates the biogeographical regions of all internal nodes of a given
      phylogeny using biogeographical information on the terminal taxa. Then, each
      internal node in the phylogenetic tree is converted to an edge in the
      biogeographical network that connects the region(s), if unambiguously estimated, 
      of its two descendants. The new method was applied to phylogenetic trees
      generated by a birth-death model. Under all conditions tested, an average of
      $&gt;$70% of the internal nodes in phylogenetic trees were converted into edges.
      Three network indices-link density, average link weight, and endemism index
      (EI)-were evaluated for their usefulness in comparing different biogeographical
      networks. The EI reflects the rate of dispersal; the other indices reflect
      nonbiogeographical parameters, the number of taxa and regions, which highlights
      the importance of evaluating network indices before applying them to
      biogeographical studies. Multiple Cretaceous biogeographical networks were
      constructed from the phylogenies of five tetrapod taxa: terrestrial
      crocodyliforms, terrestrial turtles, nonavian dinosaurs, avians, and pterosaurs. 
      The networks of avians and pterosaurs showed similar topologies and a strong
      correlation, and unexpectedly high endemism indices. These similarities were
      probably a result of shared taphonomic biases (i.e., the Lagerstatten effect) for
      volant taxa with fragile skeletons. The crocodyliform network was partitioned
      into the Gondwanan and Laurasian continents. The dinosaur network was partitioned
      into three groups of continents: 1) North America, Asia, and Australia; 2) Europe
      and Africa; and 3) India, Madagascar, and South America. When Early and Late
      Cretaceous dinosaurs were analyzed separately, the dinosaur networks were divided
      into 1) North America, Asia, and Australia; and 2) Europe, Africa, India, and
      South America for the Early Cretaceous and 1) North America, Asia, and Europe;
      and 2) India, Madagascar, and South America for the Late Cretaceous. This
      partitioning of dinosaur and crocodyliform networks corroborates the results of
      previous biogeographical studies and indicates that the method introduced here
      can retrieve biogeographical signals from a source phylogeny when sufficient data
      are available for most targeted biogeographical regions. [Biogeography;
      Cretaceous period; network analysis; network index; phylogeny; simulation;
      terrestrial tetrapods.].
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Kubo, Tai
AU  - Kubo T
AD  - The University Museum, The University of Tokyo, 7-3-1 Hongo, Bunkyou-ku, Tokyo
      113-0033, Japan.
LA  - eng
PT  - Journal Article
DEP - 20190527
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/05/29 06:00
MHDA- 2019/05/29 06:00
CRDT- 2019/05/29 06:00
PHST- 2018/04/16 00:00 [received]
PHST- 2019/03/13 00:00 [revised]
PHST- 2019/04/09 00:00 [accepted]
PHST- 2019/05/29 06:00 [entrez]
PHST- 2019/05/29 06:00 [pubmed]
PHST- 2019/05/29 06:00 [medline]
AID - 5477412 [pii]
AID - 10.1093/sysbio/syz024 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 May 27. pii: 5477412. doi: 10.1093/sysbio/syz024.

PMID- 31135914
OWN - NLM
STAT- Publisher
LR  - 20190528
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Apr 23
TI  - Whole-Genome Analyses Resolve the Phylogeny of Flightless Birds (Palaeognathae)
      in the Presence of an Empirical Anomaly Zone.
LID - syz019 [pii]
LID - 10.1093/sysbio/syz019 [doi]
AB  - Palaeognathae represent one of the two basal lineages in modern birds, and
      comprise the volant (flighted) tinamous and the flightless ratites. Resolving
      palaeognath phylogenetic relationships has historically proved difficult, and
      short internal branches separating major palaeognath lineages in previous
      molecular phylogenies suggest that extensive incomplete lineage sorting (ILS)
      might have accompanied a rapid ancient divergence. Here, we investigate
      palaeognath relationships using genome-wide data sets of three types of noncoding
      nuclear markers, together totaling 20,850 loci and over 41 million base pairs of 
      aligned sequence data. We recover a fully resolved topology placing rheas as the 
      sister to kiwi and emu + cassowary that is congruent across marker types for two 
      species tree methods (MP-EST and ASTRAL-II). This topology is corroborated by
      patterns of insertions for 4274 CR1 retroelements identified from multispecies
      whole-genome screening, and is robustly supported by phylogenomic subsampling
      analyses, with MP-EST demonstrating particularly consistent performance across
      subsampling replicates as compared to ASTRAL. In contrast, analyses of
      concatenated data supermatrices recover rheas as the sister to all other
      nonostrich palaeognaths, an alternative that lacks retroelement support and shows
      inconsistent behavior under subsampling approaches. While statistically
      supporting the species tree topology, conflicting patterns of retroelement
      insertions also occur and imply high amounts of ILS across short successive
      internal branches, consistent with observed patterns of gene tree heterogeneity. 
      Coalescent simulations and topology tests indicate that the majority of observed 
      topological incongruence among gene trees is consistent with coalescent variation
      rather than arising from gene tree estimation error alone, and estimated branch
      lengths for short successive internodes in the inferred species tree fall within 
      the theoretical range encompassing the anomaly zone. Distributions of empirical
      gene trees confirm that the most common gene tree topology for each marker type
      differs from the species tree, signifying the existence of an empirical anomaly
      zone in palaeognaths. [Anomaly zone; incomplete lineage sorting; multispecies
      coalescent; palaeognath; phylogenomics; ratite; retroelement; species tree.].
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Cloutier, Alison
AU  - Cloutier A
AD  - Department of Organismic and Evolutionary Biology, Harvard University, 26 Oxford 
      Street, Cambridge, MA 02138, USA.
AD  - Department of Ornithology, Museum of Comparative Zoology, Harvard University, 26 
      Oxford Street, Cambridge, MA 02138, USA.
FAU - Sackton, Timothy B
AU  - Sackton TB
AD  - Informatics Group, Harvard University, 28 Oxford Street, Cambridge, MA 02138,
      USA.
FAU - Grayson, Phil
AU  - Grayson P
AD  - Department of Organismic and Evolutionary Biology, Harvard University, 26 Oxford 
      Street, Cambridge, MA 02138, USA.
AD  - Department of Ornithology, Museum of Comparative Zoology, Harvard University, 26 
      Oxford Street, Cambridge, MA 02138, USA.
FAU - Clamp, Michele
AU  - Clamp M
AD  - Informatics Group, Harvard University, 28 Oxford Street, Cambridge, MA 02138,
      USA.
FAU - Baker, Allan J
AU  - Baker AJ
AD  - Department of Ecology and Evolutionary Biology, University of Toronto, 25 Willcox
      Street, Toronto, Ontario M5S 3B2, Canada.
AD  - Department of Natural History, Royal Ontario Museum, 100 Queen's Park, Toronto,
      Ontario M5S 2C6, Canada.
FAU - Edwards, Scott V
AU  - Edwards SV
AD  - Department of Organismic and Evolutionary Biology, Harvard University, 26 Oxford 
      Street, Cambridge, MA 02138, USA.
AD  - Department of Ornithology, Museum of Comparative Zoology, Harvard University, 26 
      Oxford Street, Cambridge, MA 02138, USA.
LA  - eng
PT  - Journal Article
DEP - 20190423
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/05/29 06:00
MHDA- 2019/05/29 06:00
CRDT- 2019/05/29 06:00
PHST- 2018/02/09 00:00 [received]
PHST- 2019/03/06 00:00 [revised]
PHST- 2019/04/09 00:00 [accepted]
PHST- 2019/05/29 06:00 [entrez]
PHST- 2019/05/29 06:00 [pubmed]
PHST- 2019/05/29 06:00 [medline]
AID - 5477409 [pii]
AID - 10.1093/sysbio/syz019 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Apr 23. pii: 5477409. doi: 10.1093/sysbio/syz019.

PMID- 31135030
OWN - NLM
STAT- Publisher
LR  - 20190528
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 May 28
TI  - Swapping Birth and Death: Symmetries and Transformations in Phylodynamic Models.
LID - syz039 [pii]
LID - 10.1093/sysbio/syz039 [doi]
AB  - Stochastic birth-death models provide the foundation for studying and simulating 
      evolutionary trees in phylodynamics. A curious feature of such models is that
      they exhibit fundamental symmetries when the birth and death rates are
      interchanged. In this paper, we first provide intuitive reasons for these known
      transformational symmetries. We then show that these transformational symmetries 
      (encoded in algebraic identities) are preserved even when individuals at the
      present are sampled with some probability. However, these extended symmetries
      require the death rate parameter to sometimes take a negative value. In the last 
      part of this paper, we describe the relevance of these transformations and their 
      application to computational phylodynamics, particularly to maximum likelihood
      and Bayesian inference methods, as well as to model selection.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Department for Biosystems Science and Engineering, ETH Zurich, Basel,
      Switzerland.
FAU - Steel, Mike
AU  - Steel M
AD  - Biomathematics Research Centre, University of Canterbury, Christchurch, New
      Zealand.
LA  - eng
PT  - Journal Article
DEP - 20190528
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Bayesian inference
OT  - Phylodynamics
OT  - algebraic symmetries
OT  - birth-death models
OT  - maximum likelihood
OT  - phylogenetics
OT  - speciation-extinction models
EDAT- 2019/05/29 06:00
MHDA- 2019/05/29 06:00
CRDT- 2019/05/29 06:00
PHST- 2018/12/12 00:00 [received]
PHST- 2019/05/17 00:00 [accepted]
PHST- 2019/05/29 06:00 [entrez]
PHST- 2019/05/29 06:00 [pubmed]
PHST- 2019/05/29 06:00 [medline]
AID - 5499129 [pii]
AID - 10.1093/sysbio/syz039 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 May 28. pii: 5499129. doi: 10.1093/sysbio/syz039.

PMID- 31135028
OWN - NLM
STAT- Publisher
LR  - 20190528
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Apr 23
TI  - Dense Geographic and Genomic Sampling Reveals Paraphyly and a Cryptic Lineage in 
      a Classic Sibling Species Complex.
LID - syz027 [pii]
LID - 10.1093/sysbio/syz027 [doi]
AB  - Incomplete or geographically biased sampling poses significant problems for
      research in phylogeography, population genetics, phylogenetics, and species
      delimitation. Despite the power of using genome-wide genetic markers in
      systematics and related fields, approaches such as the multispecies coalescent
      remain unable to easily account for unsampled lineages. The Empidonax
      difficilis/Empidonax occidentalis complex of small tyrannid flycatchers (Aves:
      Tyrannidae) is a classic example of widely distributed species with limited
      phenotypic geographic variation that was broken into two largely cryptic (or
      "sibling") lineages following extensive study. Though the group is
      well-characterized north of the US Mexico border, the evolutionary
      distinctiveness and phylogenetic relationships of southern populations remain
      obscure. In this article, we use dense genomic and geographic sampling across the
      majority of the range of the E. difficilis/E. occidentalis complex to assess
      whether current taxonomy and species limits reflect underlying evolutionary
      patterns, or whether they are an artifact of historically biased or incomplete
      sampling. We find that additional samples from Mexico render the widely
      recognized species-level lineage E. occidentalis paraphyletic, though it retains 
      support in the best-fit species delimitation model from clustering analyses. We
      further identify a highly divergent unrecognized lineage in a previously
      unsampled portion of the group's range, which a cline analysis suggests is more
      reproductively isolated than the currently recognized species E.difficilis and E.
      occidentalis. Our phylogeny supports a southern origin of these taxa. Our results
      highlight the pervasive impacts of biased geographic sampling, even in
      well-studied vertebrate groups like birds, and illustrate what is a common
      problem when attempting to define species in the face of recent divergence and
      reticulate evolution.[Phylogenomics; RADseq; Tyrannidae; ultraconserved
      elements.].
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Linck, Ethan
AU  - Linck E
AD  - Burke Museum of Natural History and Department of Biology, University of
      Washington, Seattle, WA 98195, USA.
FAU - Epperly, Kevin
AU  - Epperly K
AD  - Burke Museum of Natural History and Department of Biology, University of
      Washington, Seattle, WA 98195, USA.
FAU - Van Els, Paul
AU  - Van Els P
AD  - Groningen Institute for Evolutionary Life Sciences, University of Groningen,
      Groningen 9700 CC, The Netherlands.
AD  - Museum of Natural Science, 119 Foster Hall, Louisiana State University, Baton
      Rouge, LA 70803, USA.
FAU - Spellman, Garth M
AU  - Spellman GM
AD  - Department of Zoology, Denver Museum of Nature &amp; Science, Denver, CO 80205, USA.
FAU - Bryson, Robert W
AU  - Bryson RW
AD  - Burke Museum of Natural History and Department of Biology, University of
      Washington, Seattle, WA 98195, USA.
FAU - McCormack, John E
AU  - McCormack JE
AD  - Moore Laboratory of Zoology, Occidental College, Los Angeles, CA 90041, USA.
FAU - Canales-Del-Castillo, Ricardo
AU  - Canales-Del-Castillo R
AD  - Facultad de Ciencias Biologicas, Universidad Autonoma de Nuevo Leon, San Nicolas 
      de los Garza, Nuevo Leon 66455, Mexico.
FAU - Klicka, John
AU  - Klicka J
AD  - Burke Museum of Natural History and Department of Biology, University of
      Washington, Seattle, WA 98195, USA.
LA  - eng
PT  - Journal Article
DEP - 20190423
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/05/29 06:00
MHDA- 2019/05/29 06:00
CRDT- 2019/05/29 06:00
PHST- 2018/12/10 00:00 [received]
PHST- 2019/04/10 00:00 [revised]
PHST- 2019/04/12 00:00 [accepted]
PHST- 2019/05/29 06:00 [entrez]
PHST- 2019/05/29 06:00 [pubmed]
PHST- 2019/05/29 06:00 [medline]
AID - 5477406 [pii]
AID - 10.1093/sysbio/syz027 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Apr 23. pii: 5477406. doi: 10.1093/sysbio/syz027.

PMID- 31132125
OWN - NLM
STAT- Publisher
LR  - 20190527
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 May 27
TI  - A Simulation-Based Evaluation of Tip-Dating Under the Fossilized Birth-Death
      Process.
LID - syz038 [pii]
LID - 10.1093/sysbio/syz038 [doi]
AB  - Bayesian molecular dating is widely used to study evolutionary timescales. This
      procedure usually involves phylogenetic analysis of nucleotide sequence data,
      with fossil-based calibrations applied as age constraints on internal nodes of
      the tree. An alternative approach is tip-dating, which explicitly includes fossil
      data in the analysis. This can be done, for example, through the joint analysis
      of molecular data from present-day taxa and morphological data from both extant
      and fossil taxa. In the context of tip-dating, an important development has been 
      the fossilized birth-death process, which allows non-contemporaneous tips and
      sampled ancestors while providing a model of lineage diversification for the
      prior on the tree topology and internal node times. However, tip-dating with
      fossils faces a number of considerable challenges, especially those associated
      with fossil sampling and evolutionary models for morphological characters. We
      conducted a simulation study to evaluate the performance of tip-dating using the 
      fossilized birth-death model. We simulated fossil occurrences and the evolution
      of nucleotide sequences and morphological characters under a wide range of
      conditions. Our analyses of these data show that the number and the maximum age
      of fossil occurrences have a greater influence than the degree of among-lineage
      rate variation or the number of morphological characters on estimates of node
      times and the tree topology. Tip-dating with the fossilized birth-death model
      generally performs well in recovering the relationships among extant taxa, but
      has difficulties in correctly placing fossil taxa in the tree and identifying the
      number of sampled ancestors. The method yields accurate estimates of the ages of 
      the root and crown group, although the precision of these estimates varies with
      the probability of fossil occurrence. The exclusion of morphological characters
      results in a slight overestimation of node times, whereas the exclusion of
      nucleotide sequences has a negative impact on inference of the tree topology. Our
      results provide an overview of the performance of tip-dating using the fossilized
      birth-death model, which will inform further development of the method and its
      application to key questions in evolutionary biology.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Luo, Arong
AU  - Luo A
AD  - Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology,
      Chinese Academy of Sciences, Beijing, China.
FAU - Duchene, David A
AU  - Duchene DA
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney, New
      South Wales, Australia.
FAU - Zhang, Chi
AU  - Zhang C
AD  - Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate
      Paleontology and Paleoanthropology, Chinese Academy of Sciences, Beijing, China.
AD  - Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, 
      Beijing, China.
FAU - Zhu, Chao-Dong
AU  - Zhu CD
AD  - Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology,
      Chinese Academy of Sciences, Beijing, China.
AD  - State Key Laboratory of Integrated Pest Management, Institute of Zoology, Chinese
      Academy of Sciences, Beijing, China.
AD  - College of Life Sciences, University of Chinese Academy of Sciences, Beijing,
      China.
FAU - Ho, Simon
AU  - Ho S
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney, New
      South Wales, Australia.
LA  - eng
PT  - Journal Article
DEP - 20190527
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Bayesian phylogenetics
OT  - evolutionary simulation
OT  - fossilized birth-death process
OT  - molecular clock
OT  - tip-dating
OT  - total-evidence dating
EDAT- 2019/05/28 06:00
MHDA- 2019/05/28 06:00
CRDT- 2019/05/28 06:00
PHST- 2018/10/03 00:00 [received]
PHST- 2019/03/15 00:00 [revised]
PHST- 2019/05/13 00:00 [revised]
PHST- 2019/05/17 00:00 [accepted]
PHST- 2019/05/28 06:00 [entrez]
PHST- 2019/05/28 06:00 [pubmed]
PHST- 2019/05/28 06:00 [medline]
AID - 5498873 [pii]
AID - 10.1093/sysbio/syz038 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 May 27. pii: 5498873. doi: 10.1093/sysbio/syz038.

PMID- 31127947
OWN - NLM
STAT- Publisher
LR  - 20190525
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 May 25
TI  - Analysis of Spounaviruses as a Case Study for the Overdue Reclassification of
      Tailed Phages.
LID - syz036 [pii]
LID - 10.1093/sysbio/syz036 [doi]
AB  - Tailed bacteriophages are the most abundant and diverse viruses in the world,
      with genome sizes ranging from 10 kbp to over 500 kbp. Yet, due to historical
      reasons, all this diversity is confined to a single virus order - Caudovirales,
      composed of just four families: Myoviridae, Siphoviridae, Podoviridae, and the
      newly created Ackermannviridae family. In recent years this morphology-based
      classification scheme has started to crumble under the constant flood of phage
      sequences, revealing that tailed phages are even more genetically diverse than
      once thought. This prompted us, the Bacterial and Archaeal Viruses Subcommittee
      of the International Committee on Taxonomy of Viruses (ICTV) to consider overall 
      reorganization of phage taxonomy. In this study, we used a wide range of
      complementary methods - including comparative genomics, core genome analysis, and
      marker gene phylogenetics - to show that the group of Bacillus phage SPO1-related
      viruses previously classified into the Spounavirinae subfamily, is clearly
      distinct from other members of the family Myoviridae and its diversity deserves
      the rank of an autonomous family. Thus, we removed this group from the Myoviridae
      family and created the family Herelleviridae - a new taxon of the same rank. In
      the process of the taxon evaluation we explored the feasibility of different
      demarcation criteria and critically evaluated the usefulness of our methods for
      phage classification. The convergence of results, drawing a consistent and
      comprehensive picture of a new family with associated subfamilies, regardless of 
      method demonstrates that the tools applied here are particularly useful in phage 
      taxonomy. We are convinced that creation of this novel family is a crucial
      milestone towards much-needed re-classification in the Caudovirales order.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Barylski, Jakub
AU  - Barylski J
AD  - Adam Mickiewicz University, Institute of Experimental Biology, Department of
      Molecular Virology, Poznan, Poland.
FAU - Enault, Francois
AU  - Enault F
AD  - Universite Clermont Auvergne, CNRS, LMGE, F-63000, Clermont-Ferrand, France.
FAU - Dutilh, Bas E
AU  - Dutilh BE
AD  - Theoretical Biology and Bioinformatics, Department of Biology, Science for Life, 
      Utrecht University, Utrecht, The Netherlands.
AD  - Centre for Molecular and Biomolecular Informatics, Radboud Institute for
      Molecular Life Sciences, Radboud University Medical Centre, Nijmegen, The
      Netherlands.
FAU - Schuller, Margo B P
AU  - Schuller MBP
AD  - Theoretical Biology and Bioinformatics, Department of Biology, Science for Life, 
      Utrecht University, Utrecht, The Netherlands.
FAU - Edwards, Robert A
AU  - Edwards RA
AD  - Departments of Biology and Computer Science, San Diego State University, San
      Diego, CA, USA.
FAU - Gillis, Annika
AU  - Gillis A
AD  - Laboratory of Food and Environmental Microbiology, Universite Catholique de
      Louvain, Louvain-la-Neuve, Belgium.
FAU - Klumpp, Jochen
AU  - Klumpp J
AD  - Institute of Food, Nutrition and Health, ETH Zurich, Switzerland.
FAU - Knezevic, Petar
AU  - Knezevic P
AD  - Department of Biology and Ecology, Faculty of Sciences, University of Novi Sad,
      Novi Sad, Serbia.
FAU - Krupovic, Mart
AU  - Krupovic M
AD  - Unite Biologie Moleculaire du Gene chez les Extremophiles, Institut Pasteur,
      Paris, France.
FAU - Kuhn, Jens H
AU  - Kuhn JH
AD  - Integrated Research Facility at Fort Detrick,National Institute of Allergy and
      Infectious Diseases, National Institutes of Health, Fort Detrick, Frederick, USA.
FAU - Lavigne, Rob
AU  - Lavigne R
AD  - Laboratory of Gene Technology, Department of Biosystems, KU Leuven, Belgium.
FAU - Oksanen, Hanna M
AU  - Oksanen HM
AD  - Molecular and Integrative Biosciences Research Program, Faculty of Biological and
      Environmental Sciences,University of Helsinki, Helsinki, Finland.
FAU - Sullivan, Matthew B
AU  - Sullivan MB
AD  - Departments of Microbiology and Civil, Environmental, and Geodetic Engineering,
      The Ohio State University, Columbus, OH, USA.
FAU - Jang, Ho Bin
AU  - Jang HB
AD  - Departments of Microbiology and Civil, Environmental, and Geodetic Engineering,
      The Ohio State University, Columbus, OH, USA.
FAU - Simmonds, Peter
AU  - Simmonds P
AD  - Nuffield Department of Medicine, University of Oxford, Peter Medawar Building,
      South Parks Road, Oxford, OX1 3SY, UK.
FAU - Aiewsakun, Pakorn
AU  - Aiewsakun P
AD  - Nuffield Department of Medicine, University of Oxford, Peter Medawar Building,
      South Parks Road, Oxford, OX1 3SY, UK.
AD  - Department of Microbiology, Faculty of Science, Mahidol University, Bangkok,
      10400, Thailand.
FAU - Wittmann, Johannes
AU  - Wittmann J
AD  - Leibniz-Institut DSMZ-German Collection of Microorganisms and Cell Cultures,
      Braunschweig, Germany.
FAU - Tolstoy, Igor
AU  - Tolstoy I
AD  - National Center for Biotechnology Information, National Library of Medicine,
      National Institutes of Health, Bethesda, MD, USA.
FAU - Brister, J Rodney
AU  - Brister JR
AD  - National Center for Biotechnology Information, National Library of Medicine,
      National Institutes of Health, Bethesda, MD, USA.
FAU - Kropinski, Andrew M
AU  - Kropinski AM
AD  - Departments of Food Science and Pathobiology, University of Guelph, Guelph,
      Ontario, Canada.
FAU - Adriaenssens, Evelien M
AU  - Adriaenssens EM
AD  - Institute of Integrative Biology, University of Liverpool, Biosciences Building, 
      Crown Street, Liverpool L69 7ZB, United Kingdom.
AD  - Quadram Institute Bioscience, Norwich Research Park, Colney Lane, NR4 7UQ
      Norwich, UK.
LA  - eng
PT  - Journal Article
DEP - 20190525
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Caudovirales
OT  - Herelleviridae
OT  - phylogenetics
OT  - spounaviruses
OT  - virus classification
OT  - virus taxonomy
EDAT- 2019/05/28 06:00
MHDA- 2019/05/28 06:00
CRDT- 2019/05/26 06:00
PHST- 2018/02/07 00:00 [received]
PHST- 2019/05/09 00:00 [revised]
PHST- 2019/05/17 00:00 [accepted]
PHST- 2019/05/26 06:00 [entrez]
PHST- 2019/05/28 06:00 [pubmed]
PHST- 2019/05/28 06:00 [medline]
AID - 5498714 [pii]
AID - 10.1093/sysbio/syz036 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 May 25. pii: 5498714. doi: 10.1093/sysbio/syz036.

PMID- 31127939
OWN - NLM
STAT- Publisher
LR  - 20190525
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 May 25
TI  - Phylogenomic Relationships of Diploids and the Origins of Allotetraploids in
      Dactylorhiza (Orchidaceae).
LID - syz035 [pii]
LID - 10.1093/sysbio/syz035 [doi]
AB  - Disentangling phylogenetic relationships proves challenging for groups that have 
      evolved recently, especially if there is ongoing reticulation. Although they are 
      in most cases immediately isolated from diploid relatives, sets of sibling
      allopolyploids often hybridize with each other, thereby increasing the complexity
      of an already challenging situation. Dactylorhiza (Orchidaceae: Orchidinae) is a 
      genus much affected by allopolyploid speciation and reticulate phylogenetic
      relationships. Here we use genetic variation at tens of thousands of genomic
      positions to unravel the convoluted evolutionary history of Dactylorhiza. We
      first investigate circumscription and relationships of diploid species in the
      genus using coalescent and maximum likelihood methods, and then group 16
      allotetraploids by maximum affiliation to their putative parental diploids,
      implementing a method based on genotype likelihoods. The direction of hybrid
      crosses is inferred for each allotetraploid using information from maternally
      inherited plastid RADseq loci. Starting from age estimates of parental taxa, the 
      relative ages of these allotetraploid entities are inferred by quantifying their 
      genetic similarity to the diploids and numbers of private alleles compared with
      sibling allotetraploids. Whereas northwestern Europe is dominated by young
      allotetraploids of postglacial origins, comparatively older allotetraploids are
      distributed further south, where climatic conditions remained relatively stable
      during the Pleistocene glaciations. Our bioinformatics approach should prove
      effective for the study of other naturally occurring, non-model, polyploid plant 
      complexes.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Brandrud, Marie K
AU  - Brandrud MK
AD  - Department of Botany and Biodiversity Research, University of Vienna, Rennweg 14,
      A-1030 Vienna, Austria.
FAU - Baar, Juliane
AU  - Baar J
AD  - Department of Botany and Biodiversity Research, University of Vienna, Rennweg 14,
      A-1030 Vienna, Austria.
FAU - Lorenzo, Maria T
AU  - Lorenzo MT
AD  - Department of Botany and Biodiversity Research, University of Vienna, Rennweg 14,
      A-1030 Vienna, Austria.
FAU - Athanasiadis, Alexander
AU  - Athanasiadis A
AD  - Department of Botany and Biodiversity Research, University of Vienna, Rennweg 14,
      A-1030 Vienna, Austria.
FAU - Bateman, Richard M
AU  - Bateman RM
AD  - Royal Botanic Gardens Kew, Richmond, Surrey, UK.
FAU - Chase, Mark W
AU  - Chase MW
AD  - Royal Botanic Gardens Kew, Richmond, Surrey, UK.
AD  - Department of Environment and Agriculture, Curtin University, Bentley, Western
      Australia 6102, Australia.
FAU - Hedren, Mikael
AU  - Hedren M
AD  - Department of Biology, University of Lund, Solvegatan 37, SE-223 62 Lund, Sweden.
FAU - Paun, Ovidiu
AU  - Paun O
AD  - Department of Botany and Biodiversity Research, University of Vienna, Rennweg 14,
      A-1030 Vienna, Austria.
LA  - eng
PT  - Journal Article
DEP - 20190525
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Dactylorhiza
OT  - Allopolyploidy
OT  - RADseq
OT  - coalescent
OT  - phylogenomics
OT  - reticulate evolution
OT  - speciation
EDAT- 2019/05/28 06:00
MHDA- 2019/05/28 06:00
CRDT- 2019/05/26 06:00
PHST- 2018/10/19 00:00 [received]
PHST- 2019/03/13 00:00 [revised]
PHST- 2019/05/12 00:00 [revised]
PHST- 2019/05/17 00:00 [accepted]
PHST- 2019/05/26 06:00 [entrez]
PHST- 2019/05/28 06:00 [pubmed]
PHST- 2019/05/28 06:00 [medline]
AID - 5498715 [pii]
AID - 10.1093/sysbio/syz035 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 May 25. pii: 5498715. doi: 10.1093/sysbio/syz035.

PMID- 31127936
OWN - NLM
STAT- Publisher
LR  - 20190525
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 May 25
TI  - The Effect of Fossil Sampling on the Estimation of Divergence Times With The
      Fossilised Birth Death Process.
LID - syz037 [pii]
LID - 10.1093/sysbio/syz037 [doi]
AB  - Timescales are of fundamental importance to evolutionary biology as they
      facilitate hypothesis tests of historical evolutionary processes. Through the
      incorporation of fossil occurrence data, the fossilised birth-death (FBD) process
      provides a framework for estimating divergence times using more palaeontological 
      data than traditional node calibration approaches have allowed. The inclusion of 
      more data can refine evolutionary timescale estimates, but for many taxonomic
      groups it is computationally infeasible to include all fossil occurrence data.
      Here, we utilise both empirical data and a simulation framework to identify
      approaches to subsampling fossil occurrence data that result in the most accurate
      estimates of divergence times. To achieve this we assess the performance of the
      FBD-Skyline model when implementing multiple approaches to incorporating
      subsampled fossil occurrences. Our results demonstrate that it is necessary to
      account for all available fossil occurrence data to achieve the most accurate
      estimates of clade age. We show that this can be achieved if an empirical Bayes
      approach to account for fossil sampling through time is applied to the FBD
      process. Random subsampling of occurrence data can lead to estimates of clade age
      that are incompatible with fossil evidence if no control over the affinities of
      fossil occurrences is enforced. Our results call into question the accuracy of
      previous divergence time studies incorporating the FBD process that have used
      only a subsample of all available fossil occurrence data.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - O'Reilly, Joseph E
AU  - O'Reilly JE
AD  - School of Earth Sciences, University of Bristol, Life Sciences Building, Tyndall 
      Avenue, Bristol BS8 1TQ, UK.
FAU - Donoghue, Philip C J
AU  - Donoghue PCJ
AD  - School of Earth Sciences, University of Bristol, Life Sciences Building, Tyndall 
      Avenue, Bristol BS8 1TQ, UK.
LA  - eng
PT  - Journal Article
DEP - 20190525
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Divergence Time Estimation
OT  - Fossil Sampling
OT  - Fossilised Birth-Death Process
OT  - Molecular Clock
EDAT- 2019/05/28 06:00
MHDA- 2019/05/28 06:00
CRDT- 2019/05/26 06:00
PHST- 2018/08/22 00:00 [received]
PHST- 2019/05/07 00:00 [revised]
PHST- 2019/05/17 00:00 [accepted]
PHST- 2019/05/26 06:00 [entrez]
PHST- 2019/05/28 06:00 [pubmed]
PHST- 2019/05/28 06:00 [medline]
AID - 5498713 [pii]
AID - 10.1093/sysbio/syz037 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 May 25. pii: 5498713. doi: 10.1093/sysbio/syz037.

PMID- 31125064
OWN - NLM
STAT- Publisher
LR  - 20190524
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 May 24
TI  - Statistical Modeling of Distribution Patterns: a Markov Random Field
      Implementation and its Application on Areas of Endemism.
LID - syz033 [pii]
LID - 10.1093/sysbio/syz033 [doi]
AB  - A statistical framework to infer areas of endemism from geographic distributions 
      is proposed. This novel method is based on Hidden Markov Random Fields, a type of
      undirected graph model commonly used in computer vision. This framework assumes
      areas of endemism are the states of the hidden layer of the model, whereas taxon 
      distributions are emitted values in the observed layer. Taxon distributions are
      associated to the observed layer through a clustering procedure based on the
      extent of overlap. Observations are emitted by the hidden layer according to a
      Gaussian distribution, whereas the joint distribution of the hidden layer follows
      a Potts model. State and parameter inference of the maximum a posteriori
      configuration is performed through a modified version of the
      Expectation-Maximization algorithm. The optimal number of areas of endemism in
      the dataset is estimated through the Pseudolikelihood Information Criterion, a
      model selection procedure that uses an approximation to likelihood. The
      performance of the new algorithm was assessed on simulated data, and compared to 
      the most popular methods for delimitation of areas of endemism: Biotic Element
      Analysis, Parsimony Analysis of Endemism, and Endemicity Analysis. Hidden Markov 
      Random Fields efficiently recovered the true pattern across a wide range of
      uncertainty values. The performance was also examined on empirical data: South
      African weevils (Sciobius), and Central American ground beetles and funnel-web
      tarantulas (Carabidae and Dipluridae, respectively). Hidden Markov Random Fields 
      uncovered six areas of endemism from the weevil dataset, whereas eight were
      estimated for the Central American arthropods (compared to 3-5 and 3-14 from the 
      other methods, respectively.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Salinas, Nelson R
AU  - Salinas NR
AD  - Division of Invertebrate Zoology, American Museum of Natural History, New York
      City, NY 10024, USA.
FAU - Wheeler, Ward C
AU  - Wheeler WC
AD  - Division of Invertebrate Zoology, American Museum of Natural History, New York
      City, NY 10024, USA.
LA  - eng
PT  - Journal Article
DEP - 20190524
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Hidden Markov Random Fields
OT  - areas of endemism
OT  - geographic distributions
EDAT- 2019/05/28 06:00
MHDA- 2019/05/28 06:00
CRDT- 2019/05/25 06:00
PHST- 2017/05/17 00:00 [received]
PHST- 2018/07/28 00:00 [revised]
PHST- 2019/05/01 00:00 [revised]
PHST- 2019/05/04 00:00 [accepted]
PHST- 2019/05/25 06:00 [entrez]
PHST- 2019/05/28 06:00 [pubmed]
PHST- 2019/05/28 06:00 [medline]
AID - 5492080 [pii]
AID - 10.1093/sysbio/syz033 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 May 24. pii: 5492080. doi: 10.1093/sysbio/syz033.

PMID- 31111152
OWN - NLM
STAT- Publisher
LR  - 20190521
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Apr 23
TI  - The Molecular Clock in the Evolution of Protein Structures.
LID - syz022 [pii]
LID - 10.1093/sysbio/syz022 [doi]
AB  - The molecular clock hypothesis, which states that substitutions accumulate in
      protein sequences at a constant rate, plays a fundamental role in molecular
      evolution but it is violated when selective or mutational processes vary with
      time. Such violations of the molecular clock have been widely investigated for
      protein sequences, but not yet for protein structures. Here, we introduce a novel
      statistical test (Significant Clock Violations) and perform a large scale
      assessment of the molecular clock in the evolution of both protein sequences and 
      structures in three large superfamilies. After validating our method with
      computer simulations, we find that clock violations are generally consistent in
      sequence and structure evolution, but they tend to be larger and more significant
      in structure evolution. Moreover, changes of function assessed through Gene
      Ontology and InterPro terms are associated with large and significant clock
      violations in structure evolution. We found that almost one third of significant 
      clock violations are significant in structure evolution but not in sequence
      evolution, highlighting the advantage to use structure information for assessing 
      accelerated evolution and gathering hints of positive selection. Clock violations
      between closely related pairs are frequently significant in sequence evolution,
      consistent with the observed time dependence of the substitution rate attributed 
      to segregation of neutral and slightly deleterious polymorphisms, but not in
      structure evolution, suggesting that these substitutions do not affect protein
      structure although they may affect stability. These results are consistent with
      the view that natural selection, both negative and positive, constrains more
      strongly protein structures than protein sequences. Our code for computing clock 
      violations is freely available at
      https://github.com/ugobas/Molecular_clock.[Co-evolution; molecular clock; protein
      structure evolution; selection.].
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Pascual-Garcia, Alberto
AU  - Pascual-Garcia A
AD  - Centro de Biologia Molecular "Severo Ochoa" CSIC-UAM Cantoblanco, 28049 Madrid,
      Spain.
AD  - Department of Life Sciences, Imperial College London, Silwood Park Campus, Ascot,
      UK.
AD  - Institute of Integrative Biology, ETH Zurich, Zurich, Switzerland.
FAU - Arenas, Miguel
AU  - Arenas M
AD  - Centro de Biologia Molecular "Severo Ochoa" CSIC-UAM Cantoblanco, 28049 Madrid,
      Spain.
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Spain.
FAU - Bastolla, Ugo
AU  - Bastolla U
AD  - Centro de Biologia Molecular "Severo Ochoa" CSIC-UAM Cantoblanco, 28049 Madrid,
      Spain.
LA  - eng
PT  - Journal Article
DEP - 20190423
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/05/22 06:00
MHDA- 2019/05/22 06:00
CRDT- 2019/05/22 06:00
PHST- 2018/03/08 00:00 [received]
PHST- 2019/03/20 00:00 [revised]
PHST- 2019/04/09 00:00 [accepted]
PHST- 2019/05/22 06:00 [entrez]
PHST- 2019/05/22 06:00 [pubmed]
PHST- 2019/05/22 06:00 [medline]
AID - 5477407 [pii]
AID - 10.1093/sysbio/syz022 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Apr 23. pii: 5477407. doi: 10.1093/sysbio/syz022.

PMID- 31099388
OWN - NLM
STAT- Publisher
LR  - 20190517
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 May 17
TI  - Statistical Comparison of Trait-dependent Biogeographical Models indicates that
      Podocarpaceae Dispersal is influenced by both Seed Cone Traits and Geographical
      Distance.
LID - syz034 [pii]
LID - 10.1093/sysbio/syz034 [doi]
AB  - The ability of lineages to disperse long distances over evolutionary timescales
      may be influenced by the gain or loss of traits adapted to enhance local,
      ecological dispersal. For example, some species in the southern conifer sister
      families Podocarpaceae and Araucariaceae have fleshy cones that encourage bird
      dispersal, but it is unknown how this trait has influenced the clade's historical
      biogeography, or its importance compared to other predictors of dispersal such as
      the geographic distance between regions. We answer these questions quantitatively
      by using a dated phylogeny of 197 species of southern conifers to statistically
      compare standard, trait-independent biogeography models with new BioGeoBEARS
      models where an evolving trait can influence dispersal probability, and trait
      history, biogeographical history, and model parameters are jointly inferred. We
      validate the method with simulation-inference experiments. Comparing all models, 
      those that include trait-dependent dispersal accrue 87.5% of the AICc model
      weight. Averaged across all models, lineages with non-fleshy cones had a
      dispersal probability multiplier of 0.49 compared to lineages with fleshy cones. 
      Distance is included as a predictor of dispersal in all credible models (100%
      model weight). However, models with changing geography earned only 22.0% of the
      model weight, and models submerging New Caledonia/New Zealand earned only 0.01%. 
      The importance of traits and distance suggests that long-distance dispersal over 
      macroevolutionary timespans should not be thought of as a highly unpredictable
      chance event. Instead, long-distance dispersal can be modelled, allowing
      statistical model comparison to quantify support for different hypotheses.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Klaus, Kristina V
AU  - Klaus KV
AD  - Department of Evolution and Biodiversity of Plants, Ruhr-Universitat Bochum,
      Germany.
FAU - Matzke, Nicholas J
AU  - Matzke NJ
AD  - Moritz Lab, Centre for Biodiversity Analysis, Division of Ecology and Evolution, 
      Research School of Biology, The Australian National University, Canberra,
      Australia.
AD  - School of Biological Sciences, University of Auckland, Auckland, New Zealand.
LA  - eng
PT  - Journal Article
DEP - 20190517
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - BioGeoBEARS
OT  - New Caledonia
OT  - New Zealand
OT  - avian endozoochory
OT  - historical biogeography
OT  - long-distance dispersal
OT  - seed dispersal
OT  - trait-dependent dispersal models
EDAT- 2019/05/18 06:00
MHDA- 2019/05/18 06:00
CRDT- 2019/05/18 06:00
PHST- 2017/06/12 00:00 [received]
PHST- 2018/10/10 00:00 [revised]
PHST- 2019/03/13 00:00 [revised]
PHST- 2019/05/02 00:00 [revised]
PHST- 2019/05/04 00:00 [accepted]
PHST- 2019/05/18 06:00 [entrez]
PHST- 2019/05/18 06:00 [pubmed]
PHST- 2019/05/18 06:00 [medline]
AID - 5490843 [pii]
AID - 10.1093/sysbio/syz034 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 May 17. pii: 5490843. doi: 10.1093/sysbio/syz034.

PMID- 31086978
OWN - NLM
STAT- Publisher
LR  - 20190515
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 May 14
TI  - Probabilistic species tree distances: implementing the multispecies coalescent to
      compare species trees within the same model-based framework used to estimate
      them.
LID - syz031 [pii]
LID - 10.1093/sysbio/syz031 [doi]
AB  - Despite the ubiquitous use of statistical models for phylogenomic and population 
      genomic inferences, this model-based rigor is rarely applied to post-hoc
      comparison of trees. In a recent study, Garba and colleagues derived new methods 
      for measuring the distance between two gene trees computed as the difference in
      their site pattern probability distributions. Unlike traditional metrics that
      compare trees solely in terms of geometry, these measures consider gene trees and
      associated parameters as probabilistic models that can be compared using standard
      information theoretic approaches. Consequently, probabilistic measures of
      phylogenetic tree distance can be far more informative than simply comparisons of
      topology and/or branch lengths alone. However, in their current form, these
      distance measures are not suitable for the comparison of species tree models in
      the presence of gene tree heterogeneity. Here we demonstrate an approach for how 
      the theory of Garba et al. (2018), which is based on gene tree distances, can be 
      extended naturally to the comparison of species tree models. Multispecies
      coalescent models (MSC) parameterize the discrete probability distribution of
      gene trees conditioned upon a species tree with a particular topology and set of 
      divergence times (in coalescent units), and thus provide a framework for
      measuring distances between species tree models in terms of their corresponding
      gene tree topology probabilities. We describe the computation of probabilistic
      species tree distances in the context of standard MSC models, which assume
      complete genetic isolation post-speciation, as well as recent theoretical
      extensions to the MSC in the form of network-based MSC models that relax this
      assumption and permit hybridization among taxa. We demonstrate these metrics
      using simulations and empirical species tree estimates and discuss both the
      benefits and limitations of these approaches. We make our species-tree distance
      approach available as an R package called pSTDistanceR, for open use by the
      community.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Adams, Richard H
AU  - Adams RH
AD  - Department of Biology, 501 S. Nedderman Dr., University of Texas at Arlington,
      Arlington, TX USA.
FAU - Castoe, Todd A
AU  - Castoe TA
AD  - Department of Biology, 501 S. Nedderman Dr., University of Texas at Arlington,
      Arlington, TX USA.
LA  - eng
PT  - Journal Article
DEP - 20190514
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - hypothesis testing
OT  - information theory
OT  - model comparison
OT  - species tree estimation
EDAT- 2019/05/16 06:00
MHDA- 2019/05/16 06:00
CRDT- 2019/05/16 06:00
PHST- 2018/09/04 00:00 [received]
PHST- 2019/01/15 00:00 [revised]
PHST- 2019/04/30 00:00 [revised]
PHST- 2019/05/02 00:00 [accepted]
PHST- 2019/05/16 06:00 [entrez]
PHST- 2019/05/16 06:00 [pubmed]
PHST- 2019/05/16 06:00 [medline]
AID - 5488962 [pii]
AID - 10.1093/sysbio/syz031 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 May 14. pii: 5488962. doi: 10.1093/sysbio/syz031.

PMID- 31062852
OWN - NLM
STAT- Publisher
LR  - 20190507
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 May 7
TI  - Model Choice, Missing Data and Taxon Sampling Impact Phylogenomic Inference of
      Deep Basidiomycota Relationships.
LID - syz029 [pii]
LID - 10.1093/sysbio/syz029 [doi]
AB  - Resolving deep divergences in the tree of life is challenging even for analyses
      of genome-scale phylogenetic datasets. Relationships between Basidiomycota
      subphyla, the rusts and allies (Pucciniomycotina), smuts and allies
      (Ustilaginomycotina) and mushroom-forming fungi and allies (Agaricomycotina) were
      found particularly recalcitrant both to traditional multigene and genome-scale
      phylogenetics. Here, we address basal Basidiomycota relationships using
      concatenated and gene-tree based analyses of various phylogenomic datasets to
      examine the contribution of several potential sources of bias. We evaluate the
      contribution of biological causes (hard polytomy, incomplete lineage sorting)
      versus unmodeled evolutionary processes and factors that exacerbate their effects
      (e.g. fast-evolving sites and long-branch taxa) to inferences of basal
      Basidiomycota relationships. Bayesian MCMC and likelihood mapping analyses reject
      the hard-polytomy with confidence. In concatenated analyses, fast-evolving sites 
      and oversimplified models of amino acid substitution favored the grouping of
      smuts with mushroom-forming fungi, often leading to maximal bootstrap support in 
      both concatenation and coalescent analyses. On the contrary, the most conserved
      data subsets grouped rusts and allies with mushroom-forming fungi, although this 
      relationship proved labile, sensitive to model choice, to different data subsets 
      and to missing data. Excluding putative long branch taxa, genes with high
      proportions of missing data and/or with strong signal failed to reveal a
      consistent trend toward one or the other topology, suggesting that additional
      sources of conflict are at play. While concatenated analyses yielded strong but
      conflicting support, individual gene trees mostly provided poor support for any
      resolution of rusts, smuts and mushroom-forming fungi, suggesting that the true
      Basidiomycota tree might be in a part of tree space that is difficult to access
      using both concatenation and gene-tree based approaches. Inference-based
      assessments of absolute model fit strongly reject best fit models for the vast
      majority of genes, indicating a poor fit of even the most commonly used models.
      While this is consistent with previous assessments of site-homogenous models of
      amino acid evolution, this does not appear to be the sole source of confounding
      signal. Our analyses suggest that topologies uniting smuts with mushroom-forming 
      fungi can arise as a result of inappropriate modeling of amino acid sites that
      might be prone to systematic bias. We speculate that improved models of sequence 
      evolution could shed more light on basal splits in the Basidiomycota, which, for 
      now, remain unresolved despite the use of whole genome data.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Prasanna, Arun N
AU  - Prasanna AN
AD  - Synthetic and Systems Biology Unit, Institute of Biochemistry, BRC-HAS, Szeged,
      Hungary.
FAU - Gerber, Daniel
AU  - Gerber D
AD  - Synthetic and Systems Biology Unit, Institute of Biochemistry, BRC-HAS, Szeged,
      Hungary.
FAU - Kijpornyongpan, Teeratas
AU  - Kijpornyongpan T
AD  - Department of Botany and Plant Pathology, Purdue University, West Lafayette, IN
      USA.
FAU - Aime, M Catherine
AU  - Aime MC
AD  - Department of Botany and Plant Pathology, Purdue University, West Lafayette, IN
      USA.
FAU - Doyle, Vinson P
AU  - Doyle VP
AD  - Department of Plant Pathology and Crop Physiology, Louisiana State University
      AgCenter, Baton Rouge, LA USA.
FAU - Nagy, Laszlo G
AU  - Nagy LG
AD  - Synthetic and Systems Biology Unit, Institute of Biochemistry, BRC-HAS, Szeged,
      Hungary.
LA  - eng
PT  - Journal Article
DEP - 20190507
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - absolute model fit
OT  - concatenation
OT  - fungi
OT  - long-branch attraction
OT  - model violation
OT  - phylogenomics
EDAT- 2019/05/08 06:00
MHDA- 2019/05/08 06:00
CRDT- 2019/05/08 06:00
PHST- 2017/10/19 00:00 [received]
PHST- 2018/11/12 00:00 [revised]
PHST- 2019/04/21 00:00 [revised]
PHST- 2019/04/26 00:00 [accepted]
PHST- 2019/05/08 06:00 [entrez]
PHST- 2019/05/08 06:00 [pubmed]
PHST- 2019/05/08 06:00 [medline]
AID - 5486399 [pii]
AID - 10.1093/sysbio/syz029 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 May 7. pii: 5486399. doi: 10.1093/sysbio/syz029.

PMID- 31062850
OWN - NLM
STAT- Publisher
LR  - 20190507
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 May 7
TI  - Whole genome shotgun phylogenomics resolves the pattern and timing of swallowtail
      butterfly evolution.
LID - syz030 [pii]
LID - 10.1093/sysbio/syz030 [doi]
AB  - Evolutionary relationships have remained unresolved in many well-studied groups, 
      even though advances in next-generation sequencing and analysis, using approaches
      such as transcriptomics, anchored hybrid enrichment, or ultraconserved elements, 
      have brought systematics to the brink of whole genome phylogenomics. Recently, it
      has become possible to sequence the entire genomes of numerous non-biological
      models in parallel at reasonable cost, particularly with shotgun sequencing. Here
      we identify orthologous coding sequences from whole-genome shotgun sequences,
      which we then use to investigate the relevance and power of phylogenomic
      relationship inference and time-calibrated tree estimation. We study an iconic
      group of butterflies - swallowtails of the family Papilionidae - that has
      remained phylogenetically unresolved, with continued debate about the timing of
      their diversification. Low-coverage whole genomes were obtained using Illumina
      shotgun sequencing for all genera. Genome assembly coupled to BLAST-based
      orthology searches allowed extraction of 6,621 orthologous protein-coding genes
      for 45 Papilionidae species and 16 outgroup species (with 32% missing data after 
      cleaning phases). Supermatrix phylogenomic analyses were performed with both
      maximum-likelihood (IQ-TREE) and Bayesian mixture models (PhyloBayes) for amino
      acid sequences, which produced a fully resolved phylogeny providing new insights 
      into controversial relationships. Species tree reconstruction from gene trees was
      performed with ASTRAL and SuperTriplets and recovered the same phylogeny. We
      estimated gene site concordant factors to complement traditional node-support
      measures, which strengthens the robustness of inferred phylogenies. Bayesian
      estimates of divergence times based on a reduced dataset (760 orthologs and 12%
      missing data) indicate a mid-Cretaceous origin of Papilionoidea around 99.2
      million years ago (Ma) (95% credibility interval: 68.6-142.7 Ma) and Papilionidae
      around 71.4 Ma (49.8-103.6 Ma), with subsequent diversification of modern
      lineages well after the Cretaceous-Paleogene event. These results show that
      shotgun sequencing of whole genomes, even when highly fragmented, represents a
      powerful approach to phylogenomics and molecular dating in a group that has
      previously been refractory to resolution.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Allio, Remi
AU  - Allio R
AD  - Institut des Sciences de l'Evolution de Montpellier (Universite de Montpellier | 
      CNRS | IRD | EPHE), Place Eugene Bataillon, 34095 Montpellier, France.
FAU - Scornavacca, Celine
AU  - Scornavacca C
AD  - Institut des Sciences de l'Evolution de Montpellier (Universite de Montpellier | 
      CNRS | IRD | EPHE), Place Eugene Bataillon, 34095 Montpellier, France.
AD  - Institut de Biologie Computationnelle (IBC), Montpellier, France.
FAU - Benoit, Nabholz
AU  - Benoit N
AD  - Institut des Sciences de l'Evolution de Montpellier (Universite de Montpellier | 
      CNRS | IRD | EPHE), Place Eugene Bataillon, 34095 Montpellier, France.
FAU - Clamens, Anne-Laure
AU  - Clamens AL
AD  - INRA, UMR 1062 Centre de Biologie pour la Gestion des Populations (INRA, IRD,
      CIRAD, Montpellier SupAgro), 755 avenue du Campus Agropolis, 34988
      Montferrier-sur-Lez, France.
AD  - University of Alberta, Department of Biological Sciences, Edmonton T6G 2E9, AB,
      Canada.
FAU - Sperling, Felix A H
AU  - Sperling FAH
AD  - University of Alberta, Department of Biological Sciences, Edmonton T6G 2E9, AB,
      Canada.
FAU - Condamine, Fabien L
AU  - Condamine FL
AD  - Institut des Sciences de l'Evolution de Montpellier (Universite de Montpellier | 
      CNRS | IRD | EPHE), Place Eugene Bataillon, 34095 Montpellier, France.
AD  - University of Alberta, Department of Biological Sciences, Edmonton T6G 2E9, AB,
      Canada.
LA  - eng
PT  - Journal Article
DEP - 20190507
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Computational limitations
OT  - Papilionidae
OT  - cross contamination
OT  - divergence times
OT  - exon capture
OT  - fragmented genomes
OT  - low-coverage whole genomes
OT  - orthology
OT  - shotgun sequencing
OT  - supermatrix
OT  - supertree
EDAT- 2019/05/08 06:00
MHDA- 2019/05/08 06:00
CRDT- 2019/05/08 06:00
PHST- 2018/12/07 00:00 [received]
PHST- 2019/03/25 00:00 [revised]
PHST- 2019/04/26 00:00 [revised]
PHST- 2019/04/28 00:00 [accepted]
PHST- 2019/05/08 06:00 [entrez]
PHST- 2019/05/08 06:00 [pubmed]
PHST- 2019/05/08 06:00 [medline]
AID - 5486398 [pii]
AID - 10.1093/sysbio/syz030 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 May 7. pii: 5486398. doi: 10.1093/sysbio/syz030.

PMID- 31058981
OWN - NLM
STAT- Publisher
LR  - 20190506
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 May 6
TI  - Accounting for uncertainty in the evolutionary timescale of green plants through 
      clock-partitioning and fossil calibration strategies.
LID - syz032 [pii]
LID - 10.1093/sysbio/syz032 [doi]
AB  - Establishing an accurate evolutionary timescale for green plants (Viridiplantae) 
      is essential to understanding their interaction and coevolution with the Earth's 
      climate and the many organisms that rely on green plants. Despite being the focus
      of numerous studies, the timing of the origin of green plants and the divergence 
      of major clades within this group remain highly controversial. Here, we infer the
      evolutionary timescale of green plants by analysing 81 protein-coding genes from 
      99 chloroplast genomes, using a core set of 21 fossil calibrations. We test the
      sensitivity of our divergence-time estimates to various components of Bayesian
      molecular dating, including the tree topology, clock models, clock-partitioning
      schemes, rate priors, and fossil calibrations. We find that the choice of clock
      model affects date estimation and that the independent-rates model provides a
      better fit to the data than the autocorrelated-rates model. Varying the rate
      prior and tree topology had little impact on age estimates, with far greater
      differences observed among calibration choices and clock-partitioning schemes.
      Our analyses yield date estimates ranging from the Paleoproterozoic to
      Mesoproterozoic for crown-group green plants, and from the Ediacaran to Middle
      Ordovician for crown-group land plants. We present divergence-time estimates of
      the major groups of green plants that take into account various sources of
      uncertainty. Our proposed timeline lays the foundation for further investigations
      into how green plants shaped the global climate and ecosystems, and how
      embryophytes became dominant in terrestrial environments.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Nie, Yuan
AU  - Nie Y
AD  - College of Life Sciences, Nanjing Normal University, Nanjing, China.
FAU - Foster, Charles S P
AU  - Foster CSP
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney,
      Australia.
FAU - Zhu, Tianqi
AU  - Zhu T
AD  - National Center for Mathematics and Interdisciplinary Sciences, Key Laboratory of
      Random Complex Structures and Data Science, Academy of Mathematics and Systems
      Science, Chinese Academy of Sciences, Beijing, China.
FAU - Yao, Ru
AU  - Yao R
AD  - College of Life Sciences, Nanjing Normal University, Nanjing, China.
FAU - Duchene, David A
AU  - Duchene DA
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney,
      Australia.
FAU - Ho, Simon Y W
AU  - Ho SYW
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney,
      Australia.
FAU - Zhong, Bojian
AU  - Zhong B
AD  - College of Life Sciences, Nanjing Normal University, Nanjing, China.
LA  - eng
PT  - Journal Article
DEP - 20190506
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Green plants
OT  - data partitioning
OT  - divergence times
OT  - fossil calibration
OT  - model selection
OT  - molecular dating
EDAT- 2019/05/07 06:00
MHDA- 2019/05/07 06:00
CRDT- 2019/05/07 06:00
PHST- 2018/07/08 00:00 [received]
PHST- 2019/04/10 00:00 [revised]
PHST- 2019/04/21 00:00 [revised]
PHST- 2019/04/30 00:00 [revised]
PHST- 2019/05/02 00:00 [accepted]
PHST- 2019/05/07 06:00 [entrez]
PHST- 2019/05/07 06:00 [pubmed]
PHST- 2019/05/07 06:00 [medline]
AID - 5486070 [pii]
AID - 10.1093/sysbio/syz032 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 May 6. pii: 5486070. doi: 10.1093/sysbio/syz032.

PMID- 31034053
OWN - NLM
STAT- Publisher
LR  - 20190429
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Apr 23
TI  - BEAGLE 3: Improved Performance, Scaling, and Usability for a High-Performance
      Computing Library for Statistical Phylogenetics.
LID - syz020 [pii]
LID - 10.1093/sysbio/syz020 [doi]
AB  - BEAGLE is a high-performance likelihood-calculation library for phylogenetic
      inference. The BEAGLE library defines a simple, but flexible, application
      programming interface (API), and includes a collection of efficient
      implementations for calculation under a variety of evolutionary models on
      different hardware devices. The library has been integrated into recent versions 
      of popular phylogenetics software packages including BEAST and MrBayes, and has
      been widely used across a diverse range of evolutionary studies. Here we present 
      BEAGLE 3, with new parallel implementations, increased performance for
      challenging data sets, improved scalability, and better usability. We have added 
      new openCL and CPU (central processing unit)-threaded implementations to the
      library, allowing the effective utilization of a wider range of modern hardware. 
      Further, we have extended the API and library to support concurrent computation
      of independent partial likelihood arrays, for increased performance of
      nucleotide-model analyses with greater flexibility of data partitioning. For
      better scalability and usability, we have improved how phylogenetic software
      packages use BEAGLE in multi-GPU (graphics processing unit) and cluster
      environments, and introduced an automated method to select the fastest device
      given the data set, evolutionary model, and hardware. For application developers 
      who wish to integrate the library, we also have developed an online tutorial. To 
      evaluate the effect of the improvements, we ran a variety of benchmarks on
      state-of-the-art hardware. For a partitioned exemplar analysis, we observe
      run-time performance improvements as high as 5.9-fold over our previous GPU
      implementation. BEAGLE 3 is free, open-source software licensed under the Lesser 
      GPL and available at https://beagle-dev.github.io.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Ayres, Daniel L
AU  - Ayres DL
AD  - Center for Bioinformatics and Computational Biology, University of Maryland,
      College Park, MD, USA.
FAU - Cummings, Michael P
AU  - Cummings MP
AD  - Center for Bioinformatics and Computational Biology, University of Maryland,
      College Park, MD, USA.
FAU - Baele, Guy
AU  - Baele G
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven - University
      of Leuven, Leuven, Belgium.
FAU - Darling, Aaron E
AU  - Darling AE
AD  - The ithree institute, University of Technology Sydney, Ultimo, NSW, Australia.
FAU - Lewis, Paul O
AU  - Lewis PO
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, Unit
      3043, Storrs, CT, USA.
FAU - Swofford, David L
AU  - Swofford DL
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL, USA.
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
AD  - Department of Integrative Biology, University of California, Berkeley, CA USA.
FAU - Lemey, Philippe
AU  - Lemey P
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven - University
      of Leuven, Leuven, Belgium.
FAU - Rambaut, Andrew
AU  - Rambaut A
AD  - Institute of Evolutionary Biology, University of Edinburgh, King's Buildings, EH9
      3FL, UK.
AD  - Fogarty International Center, National Institutes of Health, Bethesda, MD, USA.
FAU - Suchard, Marc A
AU  - Suchard MA
AD  - Department of Biomathematics, University of California, Los Angeles, CA, USA.
AD  - Department of Biostatistics, University of California, Los Angeles, CA, USA.
AD  - Department of Human Genetics, University of California, Los Angeles, CA, USA.
LA  - eng
PT  - Journal Article
DEP - 20190423
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Bayesian phylogenetics
OT  - GPU
OT  - maximum likelihood
OT  - multicore processing
OT  - parallel computing
EDAT- 2019/04/30 06:00
MHDA- 2019/04/30 06:00
CRDT- 2019/04/30 06:00
PHST- 2018/04/27 00:00 [received]
PHST- 2018/12/23 00:00 [revised]
PHST- 2019/03/08 00:00 [revised]
PHST- 2019/04/10 00:00 [revised]
PHST- 2019/04/10 00:00 [accepted]
PHST- 2019/04/30 06:00 [entrez]
PHST- 2019/04/30 06:00 [pubmed]
PHST- 2019/04/30 06:00 [medline]
AID - 5477405 [pii]
AID - 10.1093/sysbio/syz020 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Apr 23. pii: 5477405. doi: 10.1093/sysbio/syz020.

PMID- 31028695
OWN - NLM
STAT- Publisher
LR  - 20190427
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Apr 27
TI  - Erratum to: Robust Design for Coalescent Model Inference.
LID - syz018 [pii]
LID - 10.1093/sysbio/syz018 [doi]
LA  - eng
PT  - Journal Article
DEP - 20190427
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/04/28 06:00
MHDA- 2019/04/28 06:00
CRDT- 2019/04/28 06:00
PHST- 2019/04/28 06:00 [entrez]
PHST- 2019/04/28 06:00 [pubmed]
PHST- 2019/04/28 06:00 [medline]
AID - 5480922 [pii]
AID - 10.1093/sysbio/syz018 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Apr 27. pii: 5480922. doi: 10.1093/sysbio/syz018.

PMID- 30917194
OWN - NLM
STAT- Publisher
LR  - 20190327
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Mar 27
TI  - A Critical Appraisal of the Placement of Xiphosura (Chelicerata) with Account of 
      Known Sources of Phylogenetic Error.
LID - syz011 [pii]
LID - 10.1093/sysbio/syz011 [doi]
AB  - Horseshoe crabs (Xiphosura) are traditionally regarded as sister group to the
      clade of terrestrial chelicerates (Arachnida). This hypothesis has been
      challenged by recent phylogenomic analyses, but the non-monophyly of Arachnida
      has consistently been disregarded as artifactual. We re-evaluated the placement
      of Xiphosura among chelicerates using the most complete phylogenetic data set to 
      date, expanding outgroup sampling, and including data from whole genome
      sequencing projects. In spite of uncertainty in the placement of some arachnid
      clades, all analyses show Xiphosura consistently nested within Arachnida as the
      sister group to Ricinulei (hooded tick spiders). It is apparent that the
      radiation of arachnids is an old one and occurred over a brief period of time,
      resulting in several consecutive short internodes, and thus is a potential case
      for the confounding effects of incomplete lineage sorting (ILS). We simulated
      coalescent gene trees to explore the effects of increasing levels of ILS on the
      placement of horseshoe crabs. In addition, common sources of systematic error
      were evaluated, as well as the effects of fast-evolving partitions and the
      dynamics of problematic long branch orders. Our results indicated that the
      placement of horseshoe crabs cannot be explained by missing data, compositional
      biases, saturation, or ILS. Interrogation of the phylogenetic signal showed that 
      the majority of loci favor the derived placement of Xiphosura over a monophyletic
      Arachnida. Our analyses support the inference that horseshoe crabs represent a
      group of aquatic arachnids, comparable to aquatic mites, breaking a long-standing
      paradigm in chelicerate evolution and altering previous interpretations of the
      ancestral transition to the terrestrial habitat. Future studies testing
      chelicerate relationships should approach the task with a sampling strategy where
      the monophyly of Arachnida is not held as the premise.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Ballesteros, Jesus A
AU  - Ballesteros JA
AD  - Department of Integrative Biology, University of Wisconsin-Madison, Madison, WI
      53706, USA.
FAU - Sharma, Prashant P
AU  - Sharma PP
AD  - Department of Integrative Biology, University of Wisconsin-Madison, Madison, WI
      53706, USA.
LA  - eng
PT  - Journal Article
DEP - 20190327
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Arachnids
OT  - Arthropods
OT  - ILS
OT  - composition bias
OT  - long branch attraction
OT  - phylogenetics
OT  - signal
EDAT- 2019/03/28 06:00
MHDA- 2019/03/28 06:00
CRDT- 2019/03/28 06:00
PHST- 2018/06/08 00:00 [received]
PHST- 2018/12/20 00:00 [revised]
PHST- 2019/02/10 00:00 [accepted]
PHST- 2019/03/28 06:00 [entrez]
PHST- 2019/03/28 06:00 [pubmed]
PHST- 2019/03/28 06:00 [medline]
AID - 5319972 [pii]
AID - 10.1093/sysbio/syz011 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Mar 27. pii: 5319972. doi: 10.1093/sysbio/syz011.

PMID- 30865279
OWN - NLM
STAT- Publisher
LR  - 20190313
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Feb 14
TI  - A Third Strike Against Perfect Phylogeny.
LID - syz009 [pii]
LID - 10.1093/sysbio/syz009 [doi]
AB  - Perfect phylogenies are fundamental in the study of evolutionary trees because
      they capture the situation when each evolutionary trait emerges only once in
      history; if such events are believed to be rare, then by Occam's Razor such
      parsimonious trees are preferable as a hypothesis of evolution. A classical
      result states that 2-state characters permit a perfect phylogeny precisely if
      each subset of 2 characters permits one. More recently, it was shown that for
      3-state characters the same property holds but for size-3 subsets. A
      long-standing open problem asked whether such a constant exists for each number
      of states. More precisely, it has been conjectured that for any fixed number of
      states $r$ there exists a constant $f(r)$ such that a set of $r$-state characters
      $C$ has a perfect phylogeny if and only if every subset of at most $f(r)$
      characters has a perfect phylogeny. Informally, the conjecture states that
      checking fixed-size subsets of characters is enough to correctly determine
      whether input data permits a perfect phylogeny, irrespective of the number of
      characters in the input. In this article, we show that this conjecture is false. 
      In particular, we show that for any constant $t$, there exists a set $C$ of
      $8$-state characters such that $C$ has no perfect phylogeny, but there exists a
      perfect phylogeny for every subset of at most $t$ characters. Moreover, there
      already exists a perfect phylogeny when ignoring just one of the characters,
      independent of which character you ignore. This negative result complements the
      two negative results ("strikes") of Bodlaender et al. (1992,2000). We reflect on 
      the consequences of this third strike, pointing out that while it does close off 
      some routes for efficient algorithm development, many others remain open.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Iersel, Leo Van
AU  - Iersel LV
AD  - Delft Institute of Applied Mathematics, Delft University of Technology, Van
      Mourik Broekmanweg 6, 2628 XE, Delft, The Netherlands.
FAU - Jones, Mark
AU  - Jones M
AD  - Delft Institute of Applied Mathematics, Delft University of Technology, Van
      Mourik Broekmanweg 6, 2628 XE, Delft, The Netherlands.
FAU - Kelk, Steven
AU  - Kelk S
AD  - Department of Data Science and Knowledge Engineering (DKE), Maastricht
      University, Bouillonstraat 8-10 6211 LH, Maastricht, The Netherlands.
LA  - eng
PT  - Journal Article
DEP - 20190214
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Four gamete condition
OT  - local obstructions conjecture
OT  - maximum parsimony
OT  - perfect phylogeny
OT  - phylogenetic tree
EDAT- 2019/03/14 06:00
MHDA- 2019/03/14 06:00
CRDT- 2019/03/14 06:00
PHST- 2018/11/22 00:00 [received]
PHST- 2019/01/28 00:00 [revised]
PHST- 2019/02/04 00:00 [accepted]
PHST- 2019/03/14 06:00 [entrez]
PHST- 2019/03/14 06:00 [pubmed]
PHST- 2019/03/14 06:00 [medline]
AID - 5319938 [pii]
AID - 10.1093/sysbio/syz009 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Feb 14. pii: 5319938. doi: 10.1093/sysbio/syz009.

PMID- 30825372
OWN - NLM
STAT- Publisher
LR  - 20190302
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Mar 2
TI  - Automated Taxonomic Identification of Insects with Expert-Level Accuracy Using
      Effective Feature Transfer from Convolutional Networks.
LID - syz014 [pii]
LID - 10.1093/sysbio/syz014 [doi]
AB  - Rapid and reliable identification of insects is important in many contexts, from 
      the detection of disease vectors and invasive species to the sorting of material 
      from biodiversity inventories. Because of the shortage of adequate expertise,
      there has long been an interest in developing automated systems for this task.
      Previous attempts have been based on laborious and complex handcrafted extraction
      of image features, but in recent years it has been shown that sophisticated
      convolutional neural networks (CNNs) can learn to extract relevant features
      automatically, without human intervention. Unfortunately, reaching expert-level
      accuracy in CNN identifications requires substantial computational power and huge
      training datasets, which are often not available for taxonomic tasks. This can be
      addressed using feature transfer: a CNN that has been pretrained on a generic
      image classification task is exposed to the taxonomic images of interest, and
      information about its perception of those images is used in training a simpler,
      dedicated identification system. Here, we develop an effective method of CNN
      feature transfer, which achieves expert-level accuracy in taxonomic
      identification of insects with training sets of 100 images or less per category, 
      depending on the nature of dataset. Specifically, we extract rich representations
      of intermediate to high-level image features from the CNN architecture VGG16
      pretrained on the ImageNet dataset. This information is submitted to a linear
      support vector machine classifier, which is trained on the target problem. We
      tested the performance of our approach on two types of challenging taxonomic
      tasks: (1) identifying insects to higher groups when they are likely to belong to
      subgroups that have not been seen previously; and (2) identifying visually
      similar species that are difficult to separate even for experts. For the first
      task, our approach reached &gt; 92 % accuracy on one dataset (884 face images of 11 
      families of Diptera, all specimens representing unique species), and &gt; 96 %
      accuracy on another (2936 dorsal habitus images of 14 families of Coleoptera,
      over 90 % of specimens belonging to unique species). For the second task, our
      approach outperformed a leading taxonomic expert on one dataset (339 images of
      three species of the Coleoptera genus Oxythyrea; 97 % accuracy), and both humans 
      and traditional automated identification systems on another dataset (3845 images 
      of nine species of Plecoptera larvae; 98.6 % accuracy). Reanalyzing several
      biological image identification tasks studied in the recent literature, we show
      that our approach is broadly applicable and provides significant improvements
      over previous methods, whether based on dedicated CNNs, CNN feature transfer, or 
      more traditional techniques. Thus, our method, which is easy to apply, can be
      highly successful in developing automated taxonomic identification systems even
      when training datasets are small and computational budgets limited. We conclude
      by briefly discussing some promising CNN-based research directions in
      morphological systematics opened up by the success of these techniques in
      providing accurate diagnostic tools.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Valan, Miroslav
AU  - Valan M
AD  - Savantic AB, Rosenlundsgatan 52, Stockholm, Sweden.
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History,
      Frescativagen 40, Stockholm, Sweden.
AD  - Department of Zoology, Stockholm University, Universitetsvagen 10, Stockholm,
      Sweden.
FAU - Makonyi, Karoly
AU  - Makonyi K
AD  - Savantic AB, Rosenlundsgatan 52, Stockholm, Sweden.
AD  - Uppsala University, Disciplinary Domain of Science and Technology, Physics,
      Department of Physics and Astronomy, Nuclear Physics, 751 20 Uppsala, Sweden;
      5School of Electrical Engineering and Computer Science, KTH Royal.
FAU - Maki, Atsuto
AU  - Maki A
AD  - Institute of Technology, Stockholm, SE-10044 Sweden.
FAU - Vondracek, Dominik
AU  - Vondracek D
AD  - Department of Zoology, Faculty of Science, Charles University in Prague, Vinicna 
      7, CZ-128 43 Praha 2, Czech Republic.
AD  - Department of Entomology, National Museum, Cirkusova 1740, CZ-193 00 Praha 9 -
      Horni Pocernice, Czech Republic.
FAU - Ronquist, Fredrik
AU  - Ronquist F
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History,
      Frescativagen 40, Stockholm, Sweden.
LA  - eng
PT  - Journal Article
DEP - 20190302
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/03/03 06:00
MHDA- 2019/03/03 06:00
CRDT- 2019/03/03 06:00
PHST- 2018/02/21 00:00 [received]
PHST- 2019/10/08 00:00 [revised]
PHST- 2019/02/13 00:00 [revised]
PHST- 2019/02/20 00:00 [accepted]
PHST- 2019/03/03 06:00 [entrez]
PHST- 2019/03/03 06:00 [pubmed]
PHST- 2019/03/03 06:00 [medline]
AID - 5368535 [pii]
AID - 10.1093/sysbio/syz014 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Mar 2. pii: 5368535. doi: 10.1093/sysbio/syz014.

PMID- 30816949
OWN - NLM
STAT- Publisher
LR  - 20190228
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Feb 28
TI  - Lack of signal for the impact of conotoxin gene diversity on speciation rates in 
      cone snails.
LID - syz016 [pii]
LID - 10.1093/sysbio/syz016 [doi]
AB  - Understanding why some groups of organisms are more diverse than others is a
      central goal in macroevolution. Evolvability, or the intrinsic capacity of
      lineages for evolutionary change, is thought to influence disparities in species 
      diversity across taxa. Over macroevolutionary time scales, clades that exhibit
      high evolvability are expected to have higher speciation rates. Cone snails
      (family: Conidae, &gt;900 spp.) provide a unique opportunity to test this prediction
      because their toxin genes can be used to characterize differences in evolvability
      between clades. Cone snails are carnivorous, use prey-specific venom (conotoxins)
      to capture prey, and the genes that encode venom are known and diversify through 
      gene duplication. Theory predicts that higher gene diversity confers a greater
      potential to generate novel phenotypes for specialization and adaptation.
      Therefore, if conotoxin gene diversity gives rise to varying levels of
      evolvability, conotoxin gene diversity should be coupled with macroevolutionary
      speciation rates. We applied exon capture techniques to recover phylogenetic
      markers and conotoxin loci across 314 species, the largest venom discovery effort
      in a single study. We paired a reconstructed timetree using 12 fossil
      calibrations with species-specific estimates of conotoxin gene diversity and used
      trait-dependent diversification methods to test the impact of evolvability on
      diversification patterns. Surprisingly, we did not detect any signal for the
      relationship between conotoxin gene diversity and speciation rates, suggesting
      that venom evolution may not be the rate-limiting factor controlling
      diversification dynamics in Conidae. Comparative analyses showed some signal for 
      the impact of diet and larval dispersal strategy on diversification patterns,
      though detection of a signal depended on the dataset and the method. If our
      results remain true with increased taxonomic sampling in future studies, they
      suggest that the rapid evolution of conid venom may cause other factors to become
      more critical to diversification, such as ecological opportunity or traits that
      promote isolation among lineages.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Phuong, Mark A
AU  - Phuong MA
AD  - Department of Ecology and Evolutionary Biology, University of California, Los
      Angeles, CA USA.
FAU - Alfaro, Michael E
AU  - Alfaro ME
AD  - Department of Ecology and Evolutionary Biology, University of California, Los
      Angeles, CA USA.
FAU - Mahardika, Gusti N
AU  - Mahardika GN
AD  - Animal Biomedical and Molecular Biology Laboratory, Faculty of Veterinary
      Medicine, Udayana University Bali, Jl Sesetan-Markisa 6, Denpasar, Bali 80225,
      Indonesia.
FAU - Marwoto, Ristiyanti M
AU  - Marwoto RM
AD  - Zoology Division (Museum Zoologicum Bogoriense), Research Center for Biology,
      LIPI, Cibinong, Bogor, Indonesia.
FAU - Prabowo, Romanus Edy
AU  - Prabowo RE
AD  - Faculty of Biology, Jenderal Soedirman University, Purwokerto, Indonesia.
FAU - von Rintelen, Thomas
AU  - von Rintelen T
AD  - Museum fur Naturkunde - Leibniz Institute for Evolution and Biodiversity Science,
      Berlin, Germany.
FAU - Wh Vogt, Philipp
AU  - Wh Vogt P
AD  - Museum fur Naturkunde - Leibniz Institute for Evolution and Biodiversity Science,
      Berlin, Germany.
FAU - Hendricks, Jonathan R
AU  - Hendricks JR
AD  - Paleontological Research Institution, Ithaca, NY, USA.
FAU - Puillandre, Nicolas
AU  - Puillandre N
AD  - Institut Systematique Evolution Biodiversite (ISYEB), Museum national d'Histoire 
      naturelle, CNRS, Sorbonne Universite, EPHE, 57 rue Cuvier, CP 26, 75005 Paris,
      France.
LA  - eng
PT  - Journal Article
DEP - 20190228
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/03/01 06:00
MHDA- 2019/03/01 06:00
CRDT- 2019/03/01 06:00
PHST- 2019/03/01 06:00 [entrez]
PHST- 2019/03/01 06:00 [pubmed]
PHST- 2019/03/01 06:00 [medline]
AID - 5366707 [pii]
AID - 10.1093/sysbio/syz016 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Feb 28. pii: 5366707. doi: 10.1093/sysbio/syz016.

PMID- 30816937
OWN - NLM
STAT- Publisher
LR  - 20190228
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Feb 28
TI  - Bayesian Estimation of Species Divergence Times Using Correlated Quantitative
      Characters.
LID - syz015 [pii]
LID - 10.1093/sysbio/syz015 [doi]
AB  - Discrete morphological data have been widely used to study species evolution, but
      the use of quantitative (or continuous) morphological characters is less common. 
      Here, we implement a Bayesian method to estimate species divergence times using
      quantitative characters. Quantitative character evolution is modelled using
      Brownian diffusion with character correlation and character variation within
      populations. Through simulations, we demonstrate that ignoring the population
      variation (or population "noise") and the correlation among characters leads to
      biased estimates of divergence times and rate, especially if the correlation and 
      population noise are high. We apply our new method to the analysis of
      quantitative characters (cranium landmarks) and molecular data from carnivoran
      mammals. Our results show that time estimates are affected by whether the
      correlations and population noise are accounted for or ignored in the analysis.
      The estimates are also affected by the type of data analysed, with analyses of
      morphological characters only, molecular data only, or a combination of both;
      showing noticeable differences among the time estimates. Rate variation of
      morphological characters among the carnivoran species appears to be very high,
      with Bayesian model selection indicating that the independent-rates model fits
      the morphological data better than the autocorrelated-rates model. We suggest
      that using morphological continuous characters, together with molecular data, can
      bring a new perspective to the study of species evolution. Our new model is
      implemented in the MCMCtree computer program for Bayesian inference of divergence
      times.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Alvarez-Carretero, Sandra
AU  - Alvarez-Carretero S
AD  - School of Biological and Chemical Sciences, Queen Mary University of London,
      London E1 4NS, UK.
FAU - Goswami, Anjali
AU  - Goswami A
AD  - Department of Genetics, Evolution and Environment, University College London,
      London WC1E 6BT, UK.
AD  - Department of Life Sciences, The Natural History Museum, London SW7 5DB, UK.
FAU - Yang, Ziheng
AU  - Yang Z
AD  - Department of Genetics, Evolution and Environment, University College London,
      London WC1E 6BT, UK.
FAU - Dos Reis, Mario
AU  - Dos Reis M
AD  - School of Biological and Chemical Sciences, Queen Mary University of London,
      London E1 4NS, UK.
LA  - eng
PT  - Journal Article
DEP - 20190228
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Bayesian inference
OT  - Procrustes alignment
OT  - continuous morphological characters
OT  - divergence times
OT  - geometric morphometrics
OT  - molecular clock
OT  - phylogeny
EDAT- 2019/03/01 06:00
MHDA- 2019/03/01 06:00
CRDT- 2019/03/01 06:00
PHST- 2018/10/09 00:00 [received]
PHST- 2019/01/29 00:00 [revised]
PHST- 2019/02/16 00:00 [revised]
PHST- 2019/02/29 00:00 [accepted]
PHST- 2019/03/01 06:00 [entrez]
PHST- 2019/03/01 06:00 [pubmed]
PHST- 2019/03/01 06:00 [medline]
AID - 5366706 [pii]
AID - 10.1093/sysbio/syz015 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Feb 28. pii: 5366706. doi: 10.1093/sysbio/syz015.

PMID- 30796841
OWN - NLM
STAT- Publisher
LR  - 20190223
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Feb 23
TI  - The Search for Common Origin: Homology revisited.
LID - syz013 [pii]
LID - 10.1093/sysbio/syz013 [doi]
AB  - Understanding the evolution of biodiversity on Earth is a central aim in biology.
      Currently, various disciplines of science contribute to unravel evolution at all 
      levels of life, from individual organisms to species and higher ranks, using
      different approaches and specific terminologies. The search for common origin,
      traditionally called homology, is a connecting paradigm of all studies related to
      evolution. However, it is not always sufficiently taken into account that
      defining homology depends on the hierarchical level studied (organism,
      population, species), which can cause confusion. Therefore, we propose a
      framework to define homologies making use of existing terms, which refer to
      homology in different fields, but restricting them to an unambiguous meaning and 
      a particular hierarchical level. We propose to use the overarching term
      'homology' only when 'morphological homology', 'vertical gene transfer' and
      'phylogenetic homology' are confirmed. Consequently, neither phylogenetic nor
      morphological homology are equal to homology. This paper is intended for readers 
      with different research backgrounds. We challenge their traditional approaches,
      inviting them to consider the proposed framework and offering them a new
      perspective for their own research.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Ochoterena, Helga
AU  - Ochoterena H
AD  - Instituto de Biologia, Universidad Nacional Autonoma de Mexico, Apdo. Postal
      70-233, CdMx, Mexico.
FAU - Vrijdaghs, Alexander
AU  - Vrijdaghs A
AD  - Ecology, Evolution and Biodiversity Conservation, KU Leuven, Kasteelpark Arenberg
      31, Box 2437, BE-3001 Leuven, Belgium.
AD  - Botanic Garden Meise, Nieuwelaan 38 BE-1860 Meise, Belgium.
FAU - Smets, Erik
AU  - Smets E
AD  - Ecology, Evolution and Biodiversity Conservation, KU Leuven, Kasteelpark Arenberg
      31, Box 2437, BE-3001 Leuven, Belgium.
AD  - Naturalis Biodiversity Center, PO Box 9517, RA, Leiden, The Netherlands.
FAU - Classen-Bockhoff, Regine
AU  - Classen-Bockhoff R
AD  - Institute of Organismic and Molecular Evolution (iomE), Johannes
      Gutenberg-Universitat Mainz, Saarstrasse, Mainz, Germany.
LA  - eng
PT  - Journal Article
DEP - 20190223
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - analogy
OT  - character
OT  - common ancestry
OT  - genealogy
OT  - homology
OT  - homoplasy
OT  - orthology
OT  - paralogy
EDAT- 2019/02/24 06:00
MHDA- 2019/02/24 06:00
CRDT- 2019/02/24 06:00
PHST- 2018/05/21 00:00 [received]
PHST- 2018/10/26 00:00 [revised]
PHST- 2019/02/11 00:00 [revised]
PHST- 2019/02/14 00:00 [accepted]
PHST- 2019/02/24 06:00 [entrez]
PHST- 2019/02/24 06:00 [pubmed]
PHST- 2019/02/24 06:00 [medline]
AID - 5364027 [pii]
AID - 10.1093/sysbio/syz013 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Feb 23. pii: 5364027. doi: 10.1093/sysbio/syz013.

PMID- 30785196
OWN - NLM
STAT- Publisher
LR  - 20190220
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Feb 20
TI  - Using historical biogeography models to study color pattern evolution.
LID - syz012 [pii]
LID - 10.1093/sysbio/syz012 [doi]
AB  - Color is among the most striking features of organisms, varying not only in
      spectral properties like hue and brightness, but also in where and how it is
      produced on the body. Different combinations of colors on a bird's body are
      important in both environmental and social contexts. Previous comparative studies
      have treated plumage patches individually or derived plumage complexity scores
      from color measurements across a bird's body. However, these approaches do not
      consider the multivariate nature of plumages (allowing for plumage to evolve as a
      whole) or account for interpatch distances. Here, we leverage a rich toolkit used
      in historical biogeography to assess color pattern evolution in a cosmopolitan
      radiation of birds, kingfishers (Aves: Alcedinidae). We demonstrate the utility
      of this approach and test hypotheses about the tempo and mode of color evolution 
      in kingfishers. Our results highlight the importance of considering interpatch
      distances in understanding macroevolutionary trends in color diversity and
      demonstrate how historical biogeography models are a useful way to model plumage 
      color pattern evolution. Furthermore, they show that distinct color mechanisms
      (pigments or structural colors) spread across the body in different ways and at
      different rates. Specifically, net rates are higher for structural colors than
      pigment-based colors. Together, our study suggests a role for both development
      and selection in driving extraordinary color pattern diversity in kingfishers. We
      anticipate this approach will be useful for modeling other complex phenotypes
      besides color, such as parasite evolution across the body.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Eliason, Chad M
AU  - Eliason CM
AD  - Integrative Research Center, Field Museum of Natural History, Chicago, IL, USA.
FAU - Andersen, Michael J
AU  - Andersen MJ
AD  - Department of Biology and Museum of Southwestern Biology, University of New
      Mexico, Albuquerque, NM, USA.
FAU - Hackett, Shannon J
AU  - Hackett SJ
AD  - Integrative Research Center, Field Museum of Natural History, Chicago, IL, USA.
LA  - eng
PT  - Journal Article
DEP - 20190220
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - RevBayes
OT  - evolutionary rates
OT  - feathers
OT  - plumage
OT  - spectrophotometry
EDAT- 2019/02/21 06:00
MHDA- 2019/02/21 06:00
CRDT- 2019/02/21 06:00
PHST- 2018/08/29 00:00 [received]
PHST- 2018/12/17 00:00 [revised]
PHST- 2019/02/11 00:00 [revised]
PHST- 2019/02/14 00:00 [accepted]
PHST- 2019/02/21 06:00 [entrez]
PHST- 2019/02/21 06:00 [pubmed]
PHST- 2019/02/21 06:00 [medline]
AID - 5345573 [pii]
AID - 10.1093/sysbio/syz012 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Feb 20. pii: 5345573. doi: 10.1093/sysbio/syz012.

PMID- 30753719
OWN - NLM
STAT- Publisher
LR  - 20190212
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Feb 11
TI  - Fossils with Feathers and Philosophy of Science.
LID - 10.1093/sysbio/syz010 [doi]
AB  - The last half century of paleornithological research has transformed the way that
      biologists perceive the evolutionary history of birds. This transformation has
      been driven, since 1969, by a series of exciting fossil discoveries combined with
      intense scientific debate over how best to interpret these discoveries. Ideally, 
      as evidence accrues and results accumulate, interpretive scientific agreement
      forms. But this has not entirely happened in the debate over avian origins: the
      accumulation of scientific evidence and analyses has had some effect, but not a
      conclusive one, in terms of resolving the question of avian origins. Although the
      majority of biologists have come to accept that birds are dinosaurs, there is
      lingering and, in some quarters, strident opposition to this view. In order to
      both understand the ongoing disagreement about avian origins and generate a
      prediction about the future of the debate, here we use a revised model of
      scientific practice to assess the current and historical state of play
      surrounding the topic of bird evolutionary origins. Many scientists are familiar 
      with the meta-scientific scholars Sir Karl Popper and Thomas Kuhn, and these are 
      the primary figures that have been appealed to so far, in prior attempts to
      assess the dispute. But we demonstrate that a variation of Imre Lakatos's model
      of progressive versus degenerative research programmes provides a novel and
      productive assessment of the debate. We establish that a refurbished Lakatosian
      account both explains the intractability of the dispute and predicts a likely
      outcome for the debate about avian origins. In short, here we offer a
      meta-scientific tool for rationally assessing competing theories-one that allows 
      researchers involved in seemingly intractable scientific disputes to advance
      their debates.
FAU - Havstad, Joyce C
AU  - Havstad JC
AD  - Department of Philosophy, Oakland University, Rochester, MI.
FAU - Smith, N Adam
AU  - Smith NA
AD  - Campbell Geology Museum, Clemson University, Clemson, SC.
LA  - eng
PT  - Journal Article
DEP - 20190211
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/02/13 06:00
MHDA- 2019/02/13 06:00
CRDT- 2019/02/13 06:00
PHST- 2018/02/08 00:00 [received]
PHST- 2019/02/06 00:00 [accepted]
PHST- 2019/02/13 06:00 [entrez]
PHST- 2019/02/13 06:00 [pubmed]
PHST- 2019/02/13 06:00 [medline]
AID - 5315532 [pii]
AID - 10.1093/sysbio/syz010 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Feb 11. pii: 5315532. doi: 10.1093/sysbio/syz010.

PMID- 30726979
OWN - NLM
STAT- Publisher
LR  - 20190206
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Feb 6
TI  - Robust Design for Coalescent Model Inference.
LID - 10.1093/sysbio/syz008 [doi]
AB  - The coalescent process describes how changes in the size or structure of a
      population influence the genealogical patterns of sequences sampled from that
      population. The estimation of (effective) population size changes from
      genealogies that are reconstructed from these sampled sequences is an important
      problem in many biological fields. Often, population size is characterised by a
      piecewise-constant function, with each piece serving as a population size
      parameter to be estimated. Estimation quality depends on both the statistical
      coalescent inference method employed, and on the experimental protocol, which
      controls variables such as the sampling of sequences through time and space, or
      the transformation of model parameters. While there is an extensive literature on
      coalescent inference methodology, there is comparatively little work on
      experimental design. The research that does exist is largely simulation-based,
      precluding the development of provable or general design theorems. We examine
      three key design problems: temporal sampling of sequences under the skyline
      demographic coalescent model, spatio-temporal sampling under the structured
      coalescent model, and time discretisation for sequentially Markovian coalescent
      models. In all cases we prove that (i) working in the logarithm of the parameters
      to be inferred (e.g. population size), and (ii) distributing informative
      coalescent events uniformly among these log-parameters, is uniquely robust.
      `Robust' means that the total and maximum uncertainty of our parameter estimates 
      are minimised, and made insensitive to their unknown (true) values. This robust
      design theorem provides rigorous justification for several existing coalescent
      experimental design decisions, and leads to usable guidelines for future
      empirical or simulation-based investigations. Given its persistence among models,
      this theorem may form the basis of an experimental design paradigm for coalescent
      inference.
FAU - Parag, Kris V
AU  - Parag KV
AD  - Department of Zoology, University of Oxford, Oxford, OX1 3SY, UK.
FAU - Pybus, Oliver G
AU  - Pybus OG
AD  - Department of Zoology, University of Oxford, Oxford, OX1 3SY, UK.
LA  - eng
PT  - Journal Article
DEP - 20190206
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/02/07 06:00
MHDA- 2019/02/07 06:00
CRDT- 2019/02/07 06:00
PHST- 2018/08/22 00:00 [received]
PHST- 2019/02/04 00:00 [accepted]
PHST- 2019/02/07 06:00 [entrez]
PHST- 2019/02/07 06:00 [pubmed]
PHST- 2019/02/07 06:00 [medline]
AID - 5307781 [pii]
AID - 10.1093/sysbio/syz008 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Feb 6. pii: 5307781. doi: 10.1093/sysbio/syz008.

PMID- 30726954
OWN - NLM
STAT- Publisher
LR  - 20190214
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Feb 6
TI  - Assessing Combinability of Phylogenomic Data using Bayes Factors.
LID - 10.1093/sysbio/syz007 [doi]
AB  - With the rapid reduction in sequencing costs of high-throughput genomic data, it 
      has become commonplace to use hundreds of genes to infer phylogeny of any study
      system. While sampling a large number of genes has given us a tremendous
      opportunity to uncover previously unknown relationships and improve phylogenetic 
      resolution, it also presents us with new challenges when the phylogenetic signal 
      is confused by differences in the evolutionary histories of sampled genes. Given 
      the incorporation of accurate marginal likelihood estimation methods into popular
      Bayesian software programs, it is natural to consider using the Bayes Factor (BF)
      to compare different partition models in which genes within any given partition
      subset share both tree topology and edge lengths. We explore using marginal
      likelihood to assess data subset combinability when data subsets have varying
      levels of phylogenetic discordance due to deep coalescence events among genes
      (simulated within a species tree), and compare the results with our
      recently-described phylogenetic informational dissonance index (D) estimated for 
      each data set. BF effectively detects phylogenetic incongruence, and provides a
      way to assess the statistical significance of D values. We use BFs to assess data
      combinability using an empirical data set comprising 56 plastid genes from green 
      algae order Volvocales. We also discuss the potential need for calibrating BFs
      and demonstrate that BFs used in this study are correctly calibrated.
FAU - Neupane, Suman
AU  - Neupane S
AD  - Virginia Tech University, 4076 Derring Hall, 926 West Campus Drive, Blacksburg,
      VA., USA.
FAU - Fucikova, Karolina
AU  - Fucikova K
AD  - Department of Natural Sciences, Assumption College, 500 Salisbury St., Worcester,
      MA, USA.
FAU - Lewis, Louise A
AU  - Lewis LA
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, Connecticut, USA.
FAU - Kuo, Lynn
AU  - Kuo L
AD  - Department of Statistics, University of Connecticut, 215 Glenbrook Road, Unit
      4120, Storrs, Connecticut, USA.
FAU - Chen, Ming-Hui
AU  - Chen MH
AD  - Department of Statistics, University of Connecticut, 215 Glenbrook Road, Unit
      4120, Storrs, Connecticut, USA.
FAU - Lewis, Paul O
AU  - Lewis PO
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, Connecticut, USA.
LA  - eng
GR  - P01 CA142538/CA/NCI NIH HHS/United States
PT  - Journal Article
DEP - 20190206
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/02/07 06:00
MHDA- 2019/02/07 06:00
CRDT- 2019/02/07 06:00
PHST- 2018/01/18 00:00 [received]
PHST- 2019/02/04 00:00 [accepted]
PHST- 2019/02/07 06:00 [entrez]
PHST- 2019/02/07 06:00 [pubmed]
PHST- 2019/02/07 06:00 [medline]
AID - 5307780 [pii]
AID - 10.1093/sysbio/syz007 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Feb 6. pii: 5307780. doi: 10.1093/sysbio/syz007.

PMID- 30690622
OWN - NLM
STAT- Publisher
LR  - 20190128
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Jan 25
TI  - Priors and Posteriors in Bayesian Timing of Divergence Analyses: the Age of
      Butterflies Revisited.
LID - 10.1093/sysbio/syz002 [doi]
AB  - The need for robust estimates of times of divergence is essential for downstream 
      analyses, yet assessing this robustness is still rare. We generated a
      time-calibrated genus-level phylogeny of butterflies (Papilionoidea), including
      994 taxa, up to 10 gene fragments and an unprecedented set of 12 fossils and 10
      host-plant node calibration points. We compared marginal priors and posterior
      distributions to assess the relative importance of the former on the latter. This
      approach revealed a strong influence of the set of priors on the root age but for
      most calibrated nodes posterior distributions shifted from the marginal prior,
      indicating significant information in the molecular dataset. Using a very
      conservative approach we estimated an origin of butterflies at 107.6 Ma,
      approximately equivalent to the latest Early Cretaceous, with a credibility
      interval ranging from 89.5 Ma (mid Late Cretaceous) to 129.5 Ma (mid Early
      Cretaceous). In addition, we tested the effects of changing fossil calibration
      priors, tree prior, different sets of calibrations and different sampling
      fractions but our estimate remained robust to these alternative assumptions. With
      994 genera, this tree provides a comprehensive source of secondary calibrations
      for studies on butterflies.
FAU - Chazot, Nicolas
AU  - Chazot N
AD  - Department of Biology, Lunds Universitet, Solvegatan 37, Lund, Sweden.
FAU - Wahlberg, Niklas
AU  - Wahlberg N
AD  - Department of Biology, Lunds Universitet, Solvegatan 37, Lund, Sweden.
FAU - Freitas, Andr Eacute Victor Lucci
AU  - Freitas AEVL
AD  - Departamento de Biologia Animal, Instituto de Biologia, Universidade Estadual de 
      Campinas (UNICAMP), Cidade Universitaria Zeferino Vaz, Caixa postal 6109, Barao
      Geraldo, Campinas, SP, Brazil.
FAU - Mitter, Charles
AU  - Mitter C
AD  - Department of Entomology, University of Maryland, College Park, MD, U.S.A.
FAU - Labandeira, Conrad
AU  - Labandeira C
AD  - Department of Entomology, University of Maryland, College Park, MD, U.S.A.
AD  - Department of Paleobiology, National Museum of Natural History, Smithsonian
      Institution, Washington, DC 20013, USA; Department of Entomology and BEES
      Program, University of Maryland, College Park, MD 20741; and Key Lab of Insect
      Evolution and Environmental Change, School of Life Sciences, Capital Normal
      University, Beijing, China.
FAU - Sohn, Jae-Cheon
AU  - Sohn JC
AD  - Department of Science Education, Gongju National University of Education, Gongju,
      Chungnam, Republic of Korea.
FAU - Sahoo, Ranjit Kumar
AU  - Sahoo RK
AD  - IISER-TVM Centre for Research and Education in Ecology and Evolution (ICREEE),
      School of Biology, Indian Institute of Science Education and Research,
      Thiruvananthapuram, Kerala, India.
FAU - Seraphim, Noemy
AU  - Seraphim N
AD  - Instituto Federal de Educacao, Ciencia e Tecnologia de Sao Paulo, Campus
      Campinas, CTI Renato Archer - Av. Comendador Aladino Selmi, s/n - Amarais,
      Campinas - SP, Brazil.
FAU - de Jong, Rienk
AU  - de Jong R
AD  - Naturalis Biodiversity Center, Department of Entomology, PO Box 9517, RA Leiden, 
      The Netherlands.
FAU - Heikkila, Maria
AU  - Heikkila M
AD  - Finnish Museum of Natural History LUOMUS, Zoology Unit, P.O. Box 17, FI
      University of Helsinki, Finland.
LA  - eng
PT  - Journal Article
DEP - 20190125
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/01/29 06:00
MHDA- 2019/01/29 06:00
CRDT- 2019/01/29 06:00
PHST- 2018/02/05 00:00 [received]
PHST- 2019/01/15 00:00 [accepted]
PHST- 2019/01/29 06:00 [entrez]
PHST- 2019/01/29 06:00 [pubmed]
PHST- 2019/01/29 06:00 [medline]
AID - 5300234 [pii]
AID - 10.1093/sysbio/syz002 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Jan 25. pii: 5300234. doi: 10.1093/sysbio/syz002.

PMID- 30668834
OWN - NLM
STAT- Publisher
LR  - 20190122
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Jan 22
TI  - Marginal likelihoods in phylogenetics: a review of methods and applications.
LID - 10.1093/sysbio/syz003 [doi]
AB  - By providing a framework of accounting for the shared ancestry inherent to all
      life, phylogenetics is becoming the statistical foundation of biology. The
      importance of model choice continues to grow as phylogenetic models continue to
      increase in complexity to better capture micro and macroevolutionary processes.
      In a Bayesian framework, the marginal likelihood is how data update our prior
      beliefs about models, which gives us an intuitive measure of comparing model fit 
      that is grounded in probability theory. Given the rapid increase in the number
      and complexity of phylogenetic models, methods for approximating marginal
      likelihoods are increasingly important. Here we try to provide an intuitive
      description of marginal likelihoods and why they are important in Bayesian model 
      testing. We also categorize and review methods for estimating marginal
      likelihoods of phylogenetic models, highlighting several recent methods that
      provide well-behaved estimates. Furthermore, we review some empirical studies
      that demonstrate how marginal likelihoods can be used to learn about models of
      evolution from biological data. We discuss promising alternatives that can
      complement marginal likelihoods for Bayesian model choice, including
      posterior-predictive methods. Using simulations, we find one alternative method
      based on approximate-Bayesian computation (ABC) to be biased. We conclude by
      discussing the challenges of Bayesian model choice and future directions that
      promise to improve the approximation of marginal likelihoods and Bayesian
      phylogenetics as a whole.
FAU - Oaks, Jamie R
AU  - Oaks JR
AD  - Department of Biological Sciences &amp; Museum of Natural History, Auburn University,
      Auburn, Alabama.
FAU - Cobb, Kerry A
AU  - Cobb KA
AD  - Department of Biological Sciences &amp; Museum of Natural History, Auburn University,
      Auburn, Alabama.
FAU - Minin, Vladimir N
AU  - Minin VN
AD  - Department of Statistics, University of California, Irvine, California.
FAU - Leache, Adam D
AU  - Leache AD
AD  - Department of Biology &amp; Burke Museum of Natural History and Culture, University
      of Washington, Seattle, Washington.
LA  - eng
PT  - Journal Article
DEP - 20190122
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/01/23 06:00
MHDA- 2019/01/23 06:00
CRDT- 2019/01/23 06:00
PHST- 2018/03/27 00:00 [received]
PHST- 2019/01/15 00:00 [accepted]
PHST- 2019/01/23 06:00 [entrez]
PHST- 2019/01/23 06:00 [pubmed]
PHST- 2019/01/23 06:00 [medline]
AID - 5298739 [pii]
AID - 10.1093/sysbio/syz003 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Jan 22. pii: 5298739. doi: 10.1093/sysbio/syz003.

PMID- 30668824
OWN - NLM
STAT- Publisher
LR  - 20190122
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Jan 22
TI  - OSF-Builder: A new tool for constructing and representing evolutionary histories 
      involving introgression.
LID - 10.1093/sysbio/syz004 [doi]
AB  - Introgression is an evolutionary process which provides an important source of
      innovation for evolution. Although various methods have been used to detect
      introgression, very few methods are currently available for constructing
      evolutionary histories involving introgression. In this paper we propose a new
      method for constructing such evolutionary histories whose starting point is a
      species forest (consisting of a collection of lineage trees, usually arising as a
      collection of clades or monophyletic groups in a species tree), and a gene tree
      for a specific allele of interest, or allele tree for short. Our method is based 
      on representing introgression in terms of a certain 'overlay' of the allele tree 
      over the lineage trees, called an overlaid species forest (OSF). OSFs are similar
      to phylogenetic networks although a key difference is that they typically have
      multiple roots because each monophyletic group in the species tree has a
      different point of origin. Employing a new model for introgression, we derive an 
      efficient algorithm for building OSFs called OSF-Builder that is guaranteed to
      return an optimal OSF in the sense that the number of potential introgression
      events is minimized. As well as using simulations to assess the performance of
      OSF-Builder, we illustrate its use on a butterfly dataset in which introgression 
      has been previously inferred. The OSF-Builder software is available for download 
      from https://www.uea.ac.uk/computing/software/OSF-Builder.
FAU - Scholz, Guillaume E
AU  - Scholz GE
AD  - School of Computing Sciences, University of East Anglia, Norwich, United Kingdom.
FAU - Popescu, Andrei-Alin
AU  - Popescu AA
AD  - School of Computing Sciences, University of East Anglia, Norwich, United Kingdom.
FAU - Taylor, Martin I
AU  - Taylor MI
AD  - School of Biological Sciences, University of East Anglia, Norwich, United
      Kingdom.
FAU - Moulton, Vincent
AU  - Moulton V
AD  - School of Computing Sciences, University of East Anglia, Norwich, United Kingdom.
FAU - Huber, Katharina T
AU  - Huber KT
AD  - School of Computing Sciences, University of East Anglia, Norwich, United Kingdom.
LA  - eng
PT  - Journal Article
DEP - 20190122
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/01/23 06:00
MHDA- 2019/01/23 06:00
CRDT- 2019/01/23 06:00
PHST- 2017/10/10 00:00 [received]
PHST- 2019/01/15 00:00 [accepted]
PHST- 2019/01/23 06:00 [entrez]
PHST- 2019/01/23 06:00 [pubmed]
PHST- 2019/01/23 06:00 [medline]
AID - 5298741 [pii]
AID - 10.1093/sysbio/syz004 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Jan 22. pii: 5298741. doi: 10.1093/sysbio/syz004.

PMID- 30668800
OWN - NLM
STAT- Publisher
LR  - 20190122
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2019 Jan 22
TI  - Integration of anatomy ontologies and evo-devo using structured Markov models
      suggests a new framework for modeling discrete phenotypic traits.
LID - 10.1093/sysbio/syz005 [doi]
AB  - Modeling discrete phenotypic traits for either ancestral character state
      reconstruction or morphology-based phylogenetic inference suffers from
      ambiguities of character coding, homology assessment, dependencies, and selection
      of adequate models. These drawbacks occur because trait evolution is driven by
      two key processes - hierarchical and hidden - which are not accommodated
      simultaneously by the available phylogenetic methods. The hierarchical process
      refers to the dependencies between anatomical body parts, while the hidden
      process refers to the evolution of gene regulatory networks underlying trait
      development. Herein, I demonstrate that these processes can be efficiently
      modeled using structured Markov models equipped with hidden states, which
      resolves the majority of the problems associated with discrete traits.
      Integration of structured Markov models with anatomy ontologies can adequately
      incorporate the hierarchical dependencies, while the use of the hidden states
      accommodates hidden evolution of gene regulatory networks and substitution rate
      heterogeneity. I assess the new models using simulations and theoretical
      synthesis. The new approach solves the long-standing "tail color problem," in
      which the trait is scored for species with tails of different colors or no tails.
      It also presents a previously unknown issue called the "two-scientist paradox,"
      in which the nature of coding the trait and the hidden processes driving the
      trait's evolution are confounded; failing to account for the hidden process may
      result in a bias, which can be avoided by using hidden state models. All this
      provides a clear guideline for coding traits into characters. This paper gives
      practical examples of using the new framework for phylogenetic inference and
      comparative analysis.
FAU - Tarasov, Sergei
AU  - Tarasov S
AD  - National Institute for Mathematical and Biological Synthesis, University of
      Tennessee, Knoxville, TN, USA.
AD  - Department of Biological Sciences, Virginia Tech, 4076 Derring Hall, 926 West
      Campus Drive, Blacksburg, VA.
LA  - eng
PT  - Journal Article
DEP - 20190122
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/01/23 06:00
MHDA- 2019/01/23 06:00
CRDT- 2019/01/23 06:00
PHST- 2017/09/19 00:00 [received]
PHST- 2019/01/15 00:00 [accepted]
PHST- 2019/01/23 06:00 [entrez]
PHST- 2019/01/23 06:00 [pubmed]
PHST- 2019/01/23 06:00 [medline]
AID - 5298740 [pii]
AID - 10.1093/sysbio/syz005 [doi]
PST - aheadofprint
SO  - Syst Biol. 2019 Jan 22. pii: 5298740. doi: 10.1093/sysbio/syz005.

PMID- 30649562
OWN - NLM
STAT- In-Data-Review
LR  - 20190614
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 4
DP  - 2019 Jul 1
TI  - Homoplasy-Based Partitioning Outperforms Alternatives in Bayesian Analysis of
      Discrete Morphological Data.
PG  - 657-671
LID - 10.1093/sysbio/syz001 [doi]
AB  - Bayesian analysis of morphological data is becoming increasingly popular mainly
      (but not only) because it allows for time-calibrated phylogenetic inference using
      relaxed morphological clocks and tip dating whenever fossils are available. As
      with molecular data, recent studies have shown that modeling among-character rate
      variation (ACRV) in morphological matrices greatly improves phylogenetic
      inference. In a likelihood framework this may be accomplished, for instance, by
      employing a hidden Markov model to assign characters to rate categories drawn
      from a (discretized) $\Gamma$ distribution and/or by partitioning data sets
      according to rate heterogeneity and estimating per-partition branch lengths,
      conditioned on a single topology. While the first approach is available in many
      phylogenetic analysis software, there is still no clear consensus on how to
      partition data, except perhaps in the simplest cases (e.g., "by codon"
      partitioning of coding sequences). Additionally, there is a trade-off between
      improvement in likelihood scores and the number of free parameters in the
      analysis, which rises quickly with the number of partitions. This trade-off may
      be dealt with by employing statistics that penalize overfitting of complex
      models, such as Akaike or Bayesian information criteria, or the more recently
      introduced stepping-stone method for marginal likelihood approximation. We
      applied the latter to three distinct matrices of discrete morphological data and 
      demonstrated that sorting characters by homoplasy scores (obtained from implied
      weighting parsimony analysis) outperformed other partitioning strategies
      (anatomically-based and PartitionFinder2). The method was in fact so efficient in
      segregating characters by rates of evolution that no within-partition ACRV
      modeling was necessary, while among-partition rate variation was adequately
      accommodated by rate multipliers. We conclude that partitioning by homoplasy is a
      powerful and easy-to-implement strategy to address ACRV in complex data sets. We 
      provide some guidelines focusing on morphological matrices, although this
      approach may be also applicable to molecular data sets.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Rosa, Brunno B
AU  - Rosa BB
AD  - Laboratorio de Biologia Comparada de Hymenoptera, Departamento de Zoologia,
      Universidade Federal do Parana, Caixa Postal 19020, Curitiba 81530-980, Brazil.
FAU - Melo, Gabriel A R
AU  - Melo GAR
AD  - Laboratorio de Biologia Comparada de Hymenoptera, Departamento de Zoologia,
      Universidade Federal do Parana, Caixa Postal 19020, Curitiba 81530-980, Brazil.
FAU - Barbeitos, Marcos S
AU  - Barbeitos MS
AD  - Laboratorio de Evolucao dos Organismos Marinhos, Departamento de Zoologia,
      Universidade Federal do Parana, Curitiba 81530-980, Brazil.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Aves
OT  - Hymenoptera
OT  - MrBayes
OT  - TNT
OT  - fossil
OT  - homoplasy
OT  - implied weighting parsimony
OT  - stepping-stone
EDAT- 2019/01/17 06:00
MHDA- 2019/01/17 06:00
CRDT- 2019/01/17 06:00
PHST- 2017/07/24 00:00 [received]
PHST- 2018/12/20 00:00 [revised]
PHST- 2019/01/04 00:00 [accepted]
PHST- 2019/01/17 06:00 [pubmed]
PHST- 2019/01/17 06:00 [medline]
PHST- 2019/01/17 06:00 [entrez]
AID - 5288246 [pii]
AID - 10.1093/sysbio/syz001 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jul 1;68(4):657-671. doi: 10.1093/sysbio/syz001.

PMID- 30605547
OWN - NLM
STAT- In-Data-Review
LR  - 20190614
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 4
DP  - 2019 Jul 1
TI  - Convergent Evolution of the Army Ant Syndrome and Congruence in Big-Data
      Phylogenetics.
PG  - 642-656
LID - 10.1093/sysbio/syy088 [doi]
AB  - Army ants are a charismatic group of organisms characterized by a suite of
      morphological and behavioral adaptations that includes obligate collective
      foraging, frequent colony relocation, and highly specialized wingless queens.
      This army ant syndrome underlies the ecological success of army ants and its
      evolution has been the subject of considerable debate. It has been argued to have
      arisen once or multiple times within the ant subfamily Dorylinae. To address this
      question in a phylogenetic framework I generated data from 2166 loci and a
      comprehensive taxon sampling representing all 27 genera and 155 or approximately 
      22% of doryline species. Most analyses show strong support for convergent
      evolution of the army ant syndrome in the Old and New World but certain
      relationships are sensitive to analytics. I examine the signal present in this
      data set and find that conflict is diminished when only loci less likely to
      violate common phylogenetic model assumptions are considered. I also provide a
      temporal and spatial context for doryline evolution with time-calibrated,
      biogeographic, and diversification rate shift analyses. This study shows that the
      army ant syndrome is both an example of remarkable convergence of a complex set
      of traits and a case of long-term evolutionary stasis. The sensitivity of some of
      the phylogenetic results underscores the need for cautious analysis of
      phylogenomic data and calls for more efficient algorithms employing
      better-fitting models of molecular evolution. Congruence among results obtained
      using different analytics may be used to assess robustness in phylogenomics.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Borowiec, Marek L
AU  - Borowiec ML
AD  - Department of Entomology, Plant Pathology and Nematology, 875 Perimeter Drive,
      University of Idaho, Moscow, ID 83844, USA.
AD  - School of Life Sciences, Social Insect Research Group, Arizona State University, 
      Tempe, AZ 85287, USA.
AD  - Department of Entomology and Nematology, One Shields Avenue, University of
      California at Davis, Davis, CA 95616, USA.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Dorylinae
OT  - Formicidae
OT  - k-means
OT  - partitioning
OT  - phylogenomics
OT  - ultraconserved elements
EDAT- 2019/01/04 06:00
MHDA- 2019/01/04 06:00
CRDT- 2019/01/04 06:00
PHST- 2018/05/24 00:00 [received]
PHST- 2018/11/09 00:00 [revised]
PHST- 2018/12/15 00:00 [accepted]
PHST- 2019/01/04 06:00 [pubmed]
PHST- 2019/01/04 06:00 [medline]
PHST- 2019/01/04 06:00 [entrez]
AID - 5272507 [pii]
AID - 10.1093/sysbio/syy088 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jul 1;68(4):642-656. doi: 10.1093/sysbio/syy088.

PMID- 30597118
OWN - NLM
STAT- Publisher
LR  - 20181231
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2018 Dec 31
TI  - Strengthening the Interaction of the Virology Community with the International
      Committee on Taxonomy of Viruses (ICTV) by Linking Virus Names and Their
      Abbreviations to Virus Species.
LID - 10.1093/sysbio/syy087 [doi]
AB  - The International Committee on Taxonomy of Viruses (ICTV) is tasked with
      classifying viruses into taxa (orders to species) and devising taxon names. Virus
      names and virus name abbreviations are currently not within the ICTV's official
      remit and are not regulated by an official entity. Many scientists,
      medical/veterinary professionals, and regulatory agencies do not address
      evolutionary questions nor are they concerned with the hierarchical organization 
      of the viral world and therefore have limited use for ICTV-devised taxa. Instead,
      these professionals look to the ICTV as an expert point source that provides the 
      most current taxonomic affiliations of viruses of interests to facilitate
      document writing. These needs are currently unmet as an ICTV-supported,
      easily-searchable database that includes all published virus names and
      abbreviations linked to their taxa is not available. In addition, in stark
      contrast to other biological taxonomic frameworks, virus taxonomy currently
      permits individual species to have several members. Consequently, confusion
      emerges among those who are not aware of the difference between taxa and viruses,
      and because certain well-known viruses cannot be located in ICTV publications or 
      be linked to their species. In addition, the number of duplicate names and
      abbreviations has increased dramatically in the literature. To solve this
      conundrum, the ICTV could mandate listing all viruses of established species and 
      all reported unclassified viruses in forthcoming online ICTV Reports and create a
      searchable webpage using this information. The International Union of
      Microbiology Societies could also consider changing the mandate of the ICTV to
      include the nomenclature of all viruses in addition to taxon considerations. With
      such a mandate expansion, official virus names and name abbreviations could be
      catalogued and virus nomenclature could be standardized. As a result, the ICTV
      would become an even more useful resource for all stakeholders in virology.
FAU - Calisher, Charles H
AU  - Calisher CH
AD  - Arthropod-borne and Infectious Diseases Laboratory, Department of Microbiology,
      Immunology and Pathology, College of Veterinary Medicine and Biomedical Sciences,
      Colorado State University, 1692 Campus Delivery, Fort Collins, CO, USA.
FAU - Briese, Thomas
AU  - Briese T
AD  - Center for Infection and Immunity, Department of Epidemiology, Mailman School of 
      Public Health, Columbia University, 722 West 168th St., New York, NY, USA.
FAU - Rodney Brister, J
AU  - Rodney Brister J
AD  - National Center for Biotechnology, National Library of Medicine,National
      Institutes of Health, 8600 Rockville Pike, Bethesda, MD, USA.
FAU - Charrel, Remi N
AU  - Charrel RN
AD  - Unite des Virus Emergents (Faculte de Medecine, Universite Aix-Marseille-IRD
      190-Inserm 1207-IHU Mediterranee Infection), Marseille, France.
FAU - Durrwald, Ralf
AU  - Durrwald R
AD  - Robert Koch-Institut, Postfach 65 02 61, Berlin, Germany.
FAU - Ebihara, Hideki
AU  - Ebihara H
AD  - Department of Molecular Medicine, Mayo Clinic, Guggenheim Building, 18th Floor,
      200 First St. SW, Rochester, MN, USA.
FAU - Fulhorst, Charles F
AU  - Fulhorst CF
AD  - Department of Pathology, University of Texas Medical Branch, 301 University Blvd,
      Galveston, TX, USA.
FAU - Fu Gao, George
AU  - Fu Gao G
AD  - National Institute for Viral Disease Control and Prevention, Chinese Center for
      Disease Control and Prevention, 155 Changbai Rd., Changping District, Beijing,
      China.
FAU - Groschup, Martin H
AU  - Groschup MH
AD  - Institut fur Virusdiagnostik, Friedrich-Loeffler-Institut, Sudufer 10,
      Greifswald-Insel Riems, Germany.
FAU - Haddow, Andrew D
AU  - Haddow AD
AD  - United States Army Medical Research Institute of Infectious Diseases, 1425 Porter
      Street, Fort Detrick, Frederick, MD, USA.
FAU - Hyndman, Timothy H
AU  - Hyndman TH
AD  - College of Veterinary Medicine, School of Veterinary and Life Sciences, Murdoch
      University, 90 South St., Murdoch, Western Australia, Australia.
FAU - Junglen, Sandra
AU  - Junglen S
AD  - Institut fur Virologie, Charite-Universitatsmedizin Charitepl. 1, 10117 Berlin,
      corporate member of Freie Universitat, Kaiserswerther Str. 16-18, Berlin.
AD  - Humboldt-Universitat, Unter den Linden 6, Berlin.
AD  - Berlin Institute of Health, Anna-Louisa-Karsch-Str. 2, Berlin, Germany.
AD  - Deutsches Zentrum fur Infektionsforschung, Inhoffenstrasse 7, Braunschweig,
      Germany.
FAU - Klempa, Boris
AU  - Klempa B
AD  - Institut fur Virologie, Charite-Universitatsmedizin Charitepl. 1, 10117 Berlin,
      corporate member of Freie Universitat, Kaiserswerther Str. 16-18, Berlin.
AD  - Humboldt-Universitat, Unter den Linden 6, Berlin.
AD  - Berlin Institute of Health, Anna-Louisa-Karsch-Str. 2, Berlin, Germany.
AD  - Biomedicinske centrum, Virologicky ustav, Slovenska Akademia Vied, Dubravska
      cesta 9, Bratislava, Slovakia.
FAU - Klingstrom, Jonas
AU  - Klingstrom J
AD  - Centre for Infectious Medicine, Institutionen for Medicin, Karolinska Institutet,
      Solnavagen 1, Stockholm, Sweden.
FAU - Kropinski, Andrew M
AU  - Kropinski AM
AD  - Departments of Food Science and Pathobiology, University of Guelph, Guelph,
      Canada.
FAU - Krupovic, Mart
AU  - Krupovic M
AD  - Departement of Microbiology, Institut Pasteur, rue du Docteur Roux, Paris,
      France.
FAU - LaBeaud, A Desiree
AU  - LaBeaud AD
AD  - Department of Pediatrics, Division of Infectious Diseases, Stanford University
      School of Medicine, 300 Pasteur Dr., G312 Stanford, CA, USA.
FAU - Maes, Piet
AU  - Maes P
AD  - KU Leuven, Zoonotic Infectious Diseases Unit, Leuven, Belgium.
FAU - Nowotny, Norbert
AU  - Nowotny N
AD  - Institut fur Virologie, Vetmeduni, Veterinarplatz 1, Vienna, Austria.
AD  - Department of Basic Medical Sciences, College of Medicine, Mohammed Bin Rashid
      University of Medicine and Health Sciences, Dubai, United Arab Emirates.
FAU - Roberto Teixeira Nunes, Marcio
AU  - Roberto Teixeira Nunes M
AD  - Evandro Chagas Institute, Ministry of Health, Ananindeua, Para, Brazil.
FAU - Payne, Susan L
AU  - Payne SL
AD  - Department of Veterinary Pathobiology, College of Veterinary Medicine and
      Biomedical Sciences, Veterinary Medical Sciences Building, 400 Raymond Stotzer
      Parkway, Texas A&amp;M University 4467, College Station, TX, USA.
FAU - Radoshitzky, Sheli R
AU  - Radoshitzky SR
AD  - United States Army Medical Research Institute of Infectious Diseases, 1425 Porter
      Street, Fort Detrick, Frederick, MD, USA.
FAU - Rubbenstroth, Dennis
AU  - Rubbenstroth D
AD  - Institut fur Virusdiagnostik, Friedrich-Loeffler-Institut, Sudufer 10,
      Greifswald-Insel Riems, Germany.
AD  - Institut fur Virologie, Universitatsklinikum Freiburg, Hermann-Herder-Strasse 11,
      Freiburg, Germany.
FAU - Sabanadzovic, Sead
AU  - Sabanadzovic S
AD  - Department of Biochemistry, Molecular Biology, Entomology and Plant
      Pathology,Mississippi State University, 402 Dorman Hall, Box 9655, Mississippi
      State, MS, USA.
FAU - Sasaya, Takahide
AU  - Sasaya T
AD  - Department of Planning and Coordination, National Agriculture and Food Research
      Organization, Tsukuba, Japan.
FAU - Stenglein, Mark D
AU  - Stenglein MD
AD  - Department of Microbiology, Immunology and Pathology, Colorado State University, 
      Fort Collins, CO, USA.
FAU - Varsani, Arvind
AU  - Varsani A
AD  - The Biodesign Center for Fundamental and Applied Microbiomics, School of Life
      Sciences, Center for Evolution and Medicine, Arizona State University, 727 E.
      Tyler St., P.O. Box 876101, Tempe, AZ, USA.
FAU - Wahl, Victoria
AU  - Wahl V
AD  - National Biodefense Analysis and Countermeasures Center, 8300 Research Plaza,
      Fort Detrick, Frederick, MD, USA.
FAU - Weaver, Scott C
AU  - Weaver SC
AD  - Institute for Human Infections and Department of Microbiology and Immunology,
      University of Texas Medical Branch, 301 University Boulevard, Galveston, TX, USA.
FAU - Zerbini, Francisco Murilo
AU  - Zerbini FM
AD  - National Institute of Science and Technology in Plant-Pest Interactions/BIOAGRO, 
      Universidade Federal de Vicosa, Vicosa, Minas Gerais, Brazil.
FAU - Vasilakis, Nikos
AU  - Vasilakis N
AD  - Department of Pathology, Center for Biodefense and Emerging Infectious Diseases, 
      Center for Tropical Diseases, Institute for Human Infection and Immunity,
      University of Texas Medical Branch, Galveston, TX, USA.
FAU - Kuhn, Jens H
AU  - Kuhn JH
AD  - Integrated Research Facility at Fort Detrick, B-8200 Research Plaza, Fort
      Detrick, Frederick, MD, USA.
LA  - eng
PT  - Journal Article
DEP - 20181231
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2019/01/01 06:00
MHDA- 2019/01/01 06:00
CRDT- 2019/01/01 06:00
PHST- 2018/06/10 00:00 [received]
PHST- 2018/12/19 00:00 [accepted]
PHST- 2019/01/01 06:00 [entrez]
PHST- 2019/01/01 06:00 [pubmed]
PHST- 2019/01/01 06:00 [medline]
AID - 5267841 [pii]
AID - 10.1093/sysbio/syy087 [doi]
PST - aheadofprint
SO  - Syst Biol. 2018 Dec 31. pii: 5267841. doi: 10.1093/sysbio/syy087.

PMID- 30597116
OWN - NLM
STAT- In-Data-Review
LR  - 20190614
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 4
DP  - 2019 Jul 1
TI  - A General Method for Simultaneously Accounting for Phylogenetic and Species
      Sampling Uncertainty via Rubin's Rules in Comparative Analysis.
PG  - 632-641
LID - 10.1093/sysbio/syy089 [doi]
AB  - Phylogenetic comparative methods (PCMs), especially ones based on linear models, 
      have played a central role in understanding species' trait evolution. These
      methods, however, usually assume that phylogenetic trees are known without error 
      or uncertainty, but this assumption is most likely incorrect. So far, Markov
      chain Monte Carlo (MCMC)-based Bayesian methods have mainly been deployed to
      account for such "phylogenetic uncertainty" in PCMs. Herein, we propose an
      approach with which phylogenetic uncertainty is incorporated in a simple, readily
      implementable and reliable manner. Our approach uses Rubin's rules, which are an 
      integral part of a standard multiple imputation procedure, often employed to
      recover missing data. We see true phylogenetic trees as missing data under this
      approach. Further, unmeasured species in comparative data (i.e., missing trait
      data) can be seen as another source of uncertainty in PCMs because arbitrary
      sampling of species in a given taxon or "species sampling uncertainty" can affect
      estimation in PCMs. Using two simulation studies, we show our method can account 
      for phylogenetic uncertainty under many different scenarios (e.g., uncertainty in
      topology and branch lengths) and, at the same time, it can handle missing trait
      data (i.e., species sampling uncertainty). A unique property of the multiple
      imputation procedure is that an index, named "relative efficiency," could be used
      to quantify the number of trees required for incorporating phylogenetic
      uncertainty. Thus, by using the relative efficiency, we show the required tree
      number is surprisingly small ($\sim$50 trees). However, the most notable
      advantage of our method is that it could be combined seamlessly with PCMs that
      utilize multiple imputation to handle simultaneously phylogenetic uncertainty
      (i.e., missing true trees) and species sampling uncertainty (i.e., missing trait 
      data) in PCMs.
CI  - (c) The Author(s) 2019. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Nakagawa, Shinichi
AU  - Nakagawa S
AD  - Evolution &amp; Ecology Research Centre and School of Biological, Earth and
      Environmental Sciences, University of New South Wales, Sydney, NSW 2052,
      Australia.
AD  - Diabetes and Metabolism Division, Garvan Institute of Medical Research, Sydney,
      NSW 2010, Australia.
FAU - De Villemereuil, Pierre
AU  - De Villemereuil P
AD  - CEFE, CNRS, Universite de Montpellier, Universite Paul Valery Montpellier 3,
      EPHE, IRD, Montpellier, France Shinichi Nakagawa and Pierre de Villemereuil
      contributed equally to this article.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Bayesian statistics
OT  - comparative analysis
OT  - data augmentation
OT  - information theory
OT  - model averaging
OT  - phylogenetics
EDAT- 2019/01/01 06:00
MHDA- 2019/01/01 06:00
CRDT- 2019/01/01 06:00
PHST- 2018/06/18 00:00 [received]
PHST- 2018/12/06 00:00 [revised]
PHST- 2018/12/15 00:00 [accepted]
PHST- 2019/01/01 06:00 [pubmed]
PHST- 2019/01/01 06:00 [medline]
PHST- 2019/01/01 06:00 [entrez]
AID - 5267840 [pii]
AID - 10.1093/sysbio/syy089 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jul 1;68(4):632-641. doi: 10.1093/sysbio/syy089.

PMID- 30597106
OWN - NLM
STAT- In-Data-Review
LR  - 20190619
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 4
DP  - 2019 Jul 1
TI  - Phylotocol: Promoting Transparency and Overcoming Bias in Phylogenetics.
PG  - 672-678
LID - 10.1093/sysbio/syy090 [doi]
AB  - The integrity of science requires that the process be based on sound experimental
      design and objective methodology. Strategies that increase reproducibility and
      transparency in science protect this integrity by reducing conscious and
      unconscious biases. Given the large number of analysis options and the constant
      development of new methodologies in phylogenetics, this field is one that would
      particularly benefit from more transparent research design. Herein, we introduce 
      phylotocol (fi lo 'ta kol), an a priori protocol-driven approach in which all
      analyses are planned and documented at the start of a project. The phylotocol
      template is simple and the implementation options are flexible to reduce
      administrative burdens and allow researchers to adapt it to their needs without
      restricting scientific creativity. While the primary goal of phylotocol is to
      increase transparency and accountability, it has a number of auxiliary benefits
      including improving study design and reproducibility, enhancing collaboration and
      education, and increasing the likelihood of project completion. Our goal with
      this Point of View article is to encourage a dialog about transparency in
      phylogenetics and the best strategies to bring transparent research practices to 
      our field.
CI  - (c) The Author(s) 2018. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - DeBiasse, Melissa B
AU  - DeBiasse MB
AD  - Whitney Laboratory for Marine Bioscience, 9505 Ocean Shore Boulevard, St.
      Augustine, FL 32080, USA.
AD  - Department of Biology, University of Florida, 220 Bartram Hall, Gainesville, FL, 
      32611, USA.
FAU - Ryan, Joseph F
AU  - Ryan JF
AD  - Whitney Laboratory for Marine Bioscience, 9505 Ocean Shore Boulevard, St.
      Augustine, FL 32080, USA.
AD  - Department of Biology, University of Florida, 220 Bartram Hall, Gainesville, FL, 
      32611, USA.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
PMC - PMC6568013
OTO - NOTNLM
OT  - Accountability
OT  - confirmation bias
OT  - open science
OT  - phylogenetics
OT  - phylotocol
OT  - protocol
OT  - transparency
EDAT- 2019/01/01 06:00
MHDA- 2019/01/01 06:00
CRDT- 2019/01/01 06:00
PHST- 2018/11/20 00:00 [received]
PHST- 2018/12/20 00:00 [revised]
PHST- 2018/12/20 00:00 [accepted]
PHST- 2019/01/01 06:00 [pubmed]
PHST- 2019/01/01 06:00 [medline]
PHST- 2019/01/01 06:00 [entrez]
AID - 5267839 [pii]
AID - 10.1093/sysbio/syy090 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jul 1;68(4):672-678. doi: 10.1093/sysbio/syy090.

PMID- 30544232
OWN - NLM
STAT- MEDLINE
DCOM- 20181220
LR  - 20181220
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 1
DP  - 2019 Jan 1
TI  - How Well Can We Estimate Diversity Dynamics for Clades in Diversity Decline?
PG  - 47-62
LID - 10.1093/sysbio/syy037 [doi]
AB  - The fossil record shows that the vast majority of all species that ever existed
      are extinct and that most lineages go through an expansion and decline in
      diversity. However, macroevolutionary analyses based upon molecular phylogenies
      have difficulty inferring extinction dynamics, raising questions about whether
      the neontological record can contribute to an understanding of the decline
      phenomenon. Two recently developed diversification methods for molecular
      phylogenies (RPANDA and BAMM) incorporate models that theoretically have the
      capacity to capture decline dynamics by allowing extinction to be higher than
      speciation. However, the performance of these frameworks over a wide range of
      decline scenarios has not been studied. Here, we investigate the behavior of
      these methods under decline scenarios caused by decreasing speciation and
      increasing extinction through time on simulated trees at fixed intervals over
      diversity trajectories with expansion and decline phases. We also compared method
      performance over a comprehensive data set of 214 empirical trees. Our results
      show that both methods perform equally well when varying speciation rates control
      decline. When decline was only caused by an increase in extinction rates both
      methods wrongly assign the variation in net diversification to a drop in
      speciation, even though the positive gamma values of those trees would suggest
      otherwise. We also found a tendency for RPANDA to favor increasing extinction and
      BAMM to favor decreasing speciation as the most common cause of decline in
      empirical trees. Overall our results shed light on the limitations of both
      methods, encouraging researchers to carefully interpret the results from
      diversification studies.
FAU - Burin, Gustavo
AU  - Burin G
AD  - Departamento de Ecologia, Instituto de Biociencias, Universidade de Sao Paulo,
      Rua do Matao, Travessa 14, Sao Paulo - SP 05508-900, Brazil.
FAU - Alencar, Laura R V
AU  - Alencar LRV
AD  - Departamento de Ecologia, Instituto de Biociencias, Universidade de Sao Paulo,
      Rua do Matao, Travessa 14, Sao Paulo - SP 05508-900, Brazil.
FAU - Chang, Jonathan
AU  - Chang J
AD  - Department of Ecology and Evolutionary Biology, University of California, 610
      Young Drive South, Los Angeles, CA 90095, USA.
FAU - Alfaro, Michael E
AU  - Alfaro ME
AD  - Department of Ecology and Evolutionary Biology, University of California, 610
      Young Drive South, Los Angeles, CA 90095, USA.
FAU - Quental, Tiago B
AU  - Quental TB
AD  - Departamento de Ecologia, Instituto de Biociencias, Universidade de Sao Paulo,
      Rua do Matao, Travessa 14, Sao Paulo - SP 05508-900, Brazil.
LA  - eng
SI  - Dryad/10.5061/dryad.tt10r84
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biodiversity
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Models, Biological
MH  - Phylogeny
EDAT- 2018/12/14 06:00
MHDA- 2018/12/21 06:00
CRDT- 2018/12/14 06:00
PHST- 2016/11/25 00:00 [received]
PHST- 2018/05/09 00:00 [accepted]
PHST- 2018/12/14 06:00 [entrez]
PHST- 2018/12/14 06:00 [pubmed]
PHST- 2018/12/21 06:00 [medline]
AID - 4996309 [pii]
AID - 10.1093/sysbio/syy037 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jan 1;68(1):47-62. doi: 10.1093/sysbio/syy037.

PMID- 30535394
OWN - NLM
STAT- In-Data-Review
LR  - 20190619
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 4
DP  - 2019 Jul 1
TI  - A Universal Probe Set for Targeted Sequencing of 353 Nuclear Genes from Any
      Flowering Plant Designed Using k-Medoids Clustering.
PG  - 594-606
LID - 10.1093/sysbio/syy086 [doi]
AB  - Sequencing of target-enriched libraries is an efficient and cost-effective method
      for obtaining DNA sequence data from hundreds of nuclear loci for phylogeny
      reconstruction. Much of the cost of developing targeted sequencing approaches is 
      associated with the generation of preliminary data needed for the identification 
      of orthologous loci for probe design. In plants, identifying orthologous loci has
      proven difficult due to a large number of whole-genome duplication events,
      especially in the angiosperms (flowering plants). We used multiple sequence
      alignments from over 600 angiosperms for 353 putatively single-copy
      protein-coding genes identified by the One Thousand Plant Transcriptomes
      Initiative to design a set of targeted sequencing probes for phylogenetic studies
      of any angiosperm group. To maximize the phylogenetic potential of the probes,
      while minimizing the cost of production, we introduce a k-medoids clustering
      approach to identify the minimum number of sequences necessary to represent each 
      coding sequence in the final probe set. Using this method, 5-15 representative
      sequences were selected per orthologous locus, representing the sequence
      diversity of angiosperms more efficiently than if probes were designed using
      available sequenced genomes alone. To test our approximately 80,000 probes, we
      hybridized libraries from 42 species spanning all higher-order groups of
      angiosperms, with a focus on taxa not present in the sequence alignments used to 
      design the probes. Out of a possible 353 coding sequences, we recovered an
      average of 283 per species and at least 100 in all species. Differences among
      taxa in sequence recovery could not be explained by relatedness to the
      representative taxa selected for probe design, suggesting that there is no
      phylogenetic bias in the probe set. Our probe set, which targeted 260 kbp of
      coding sequence, achieved a median recovery of 137 kbp per taxon in coding
      regions, a maximum recovery of 250 kbp, and an additional median of 212 kbp per
      taxon in flanking non-coding regions across all species. These results suggest
      that the Angiosperms353 probe set described here is effective for any group of
      flowering plants and would be useful for phylogenetic studies from the species
      level to higher-order groups, including the entire angiosperm clade itself.
CI  - (c) The Author(s) 2018. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - Johnson, Matthew G
AU  - Johnson MG
AD  - Department of Biological Sciences, Texas Tech University, Lubbock, TX 79409, USA.
AD  - Plant Science and Conservation, Chicago Botanic Garden, 1000 Lake Cook Road,
      Glencoe, IL 60022, USA.
FAU - Pokorny, Lisa
AU  - Pokorny L
AD  - Department of Comparative Plant and Fungal Biology, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3AE, UK.
FAU - Dodsworth, Steven
AU  - Dodsworth S
AD  - Department of Comparative Plant and Fungal Biology, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3AE, UK.
AD  - School of Life Sciences, University of Bedfordshire, University Square, Luton LU1
      3JU, UK.
FAU - Botigue, Laura R
AU  - Botigue LR
AD  - Department of Comparative Plant and Fungal Biology, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3AE, UK.
AD  - Centre for Research in Agricultural Genomics, Campus UAB, Edifici CRAG,
      Bellaterra Cerdanyola del Valles, 08193 Barcelona, Spain.
FAU - Cowan, Robyn S
AU  - Cowan RS
AD  - Department of Comparative Plant and Fungal Biology, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3AE, UK.
FAU - Devault, Alison
AU  - Devault A
AD  - Arbor Biosciences, 5840 Interface Dr, Suite 101, Ann Arbor, MI 48103, USA.
FAU - Eiserhardt, Wolf L
AU  - Eiserhardt WL
AD  - Department of Comparative Plant and Fungal Biology, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3AE, UK.
AD  - Department of Bioscience, Aarhus University, 8000 Aarhus C, Denmark.
FAU - Epitawalage, Niroshini
AU  - Epitawalage N
AD  - Department of Comparative Plant and Fungal Biology, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3AE, UK.
FAU - Forest, Felix
AU  - Forest F
AD  - Department of Comparative Plant and Fungal Biology, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3AE, UK.
FAU - Kim, Jan T
AU  - Kim JT
AD  - Department of Comparative Plant and Fungal Biology, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3AE, UK.
FAU - Leebens-Mack, James H
AU  - Leebens-Mack JH
AD  - Department of Plant Biology, University of Georgia, 2502 Miller Plant Sciences,
      Athens, GA 30602, USA.
FAU - Leitch, Ilia J
AU  - Leitch IJ
AD  - Department of Comparative Plant and Fungal Biology, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3AE, UK.
FAU - Maurin, Olivier
AU  - Maurin O
AD  - Department of Comparative Plant and Fungal Biology, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3AE, UK.
FAU - Soltis, Douglas E
AU  - Soltis DE
AD  - Department of Biology, University of Florida, 220 Bartram Hall, Gainesville, FL
      32611-8525, USA.
AD  - Florida Museum of Natural History, University of Florida, 3215 Hull Road,
      Gainesville, FL 32611-2710, USA.
FAU - Soltis, Pamela S
AU  - Soltis PS
AD  - Department of Biology, University of Florida, 220 Bartram Hall, Gainesville, FL
      32611-8525, USA.
AD  - Florida Museum of Natural History, University of Florida, 3215 Hull Road,
      Gainesville, FL 32611-2710, USA.
FAU - Wong, Gane Ka-Shu
AU  - Wong GK
AD  - BGI-Shenzhen, Beishan Industrial Zone, Yantian District, Shenzhen 518083, China.
AD  - Department of Biological Sciences, University of Alberta, Edmonton, AB T6G 2E9,
      Canada.
AD  - Department of Medicine, University of Alberta, Edmonton, AB T6G 2E1, Canada.
FAU - Baker, William J
AU  - Baker WJ
AD  - Department of Comparative Plant and Fungal Biology, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3AE, UK.
FAU - Wickett, Norman J
AU  - Wickett NJ
AD  - Plant Science and Conservation, Chicago Botanic Garden, 1000 Lake Cook Road,
      Glencoe, IL 60022, USA.
AD  - Program in Plant Biology and Conservation, Northwestern University, 2205 Tech
      Drive, Evanston, IL 60208, USA.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
PMC - PMC6568016
OTO - NOTNLM
OT  - Angiosperms
OT  - Hyb-Seq
OT  - k-means clustering
OT  - k-medoids clustering
OT  - machine learning
OT  - nuclear genes
OT  - phylogenomics
OT  - sequence capture
OT  - target enrichment
EDAT- 2018/12/12 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/12/12 06:00
PHST- 2018/07/02 00:00 [received]
PHST- 2018/11/29 00:00 [revised]
PHST- 2018/12/03 00:00 [accepted]
PHST- 2018/12/12 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
PHST- 2018/12/12 06:00 [entrez]
AID - 5237557 [pii]
AID - 10.1093/sysbio/syy086 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jul 1;68(4):594-606. doi: 10.1093/sysbio/syy086.

PMID- 30535172
OWN - NLM
STAT- In-Data-Review
LR  - 20190619
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 4
DP  - 2019 Jul 1
TI  - An algorithm for Morphological Phylogenetic Analysis with Inapplicable Data.
PG  - 619-631
LID - 10.1093/sysbio/syy083 [doi]
AB  - Morphological data play a key role in the inference of biological relationships
      and evolutionary history and are essential for the interpretation of the fossil
      record. The hierarchical interdependence of many morphological characters,
      however, complicates phylogenetic analysis. In particular, many characters only
      apply to a subset of terminal taxa. The widely used "reductive coding" approach
      treats taxa in which a character is inapplicable as though the character's state 
      is simply missing (unknown). This approach has long been known to create spurious
      tree length estimates on certain topologies, potentially leading to erroneous
      results in phylogenetic searches-but pratical solutions have yet to be proposed
      and implemented. Here, we present a single-character algorithm for reconstructing
      ancestral states in reductively coded data sets, following the theoretical
      guideline of minimizing homoplasy over all characters. Our algorithm uses up to
      three traversals to score a tree, and a fourth to fully resolve final states at
      each node within the tree. We use explicit criteria to resolve ambiguity in
      applicable/inapplicable dichotomies, and to optimize missing data. So that it can
      be applied to single characters, the algorithm employs local optimization; as
      such, the method provides a fast but approximate inference of ancestral states
      and tree score. The application of our method to published morphological data
      sets indicates that, compared to traditional methods, it identifies different
      trees as "optimal." As such, the use of our algorithm to handle inapplicable data
      may significantly alter the outcome of tree searches, modifying the inferred
      placement of living and fossil taxa and potentially leading to major differences 
      in reconstructions of evolutionary history.
CI  - (c) The Author(s) 2018. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Brazeau, Martin D
AU  - Brazeau MD
AD  - Department of Life Sciences, Imperial College London, Silwood Park Campus,
      Buckhurst Road, Ascot SL5 7PY, UK.
AD  - Department of Earth Sciences, Natural History Museum, Cromwell Road, London SW7
      5BD, UK.
FAU - Guillerme, Thomas
AU  - Guillerme T
AD  - Department of Life Sciences, Imperial College London, Silwood Park Campus,
      Buckhurst Road, Ascot SL5 7PY, UK.
AD  - School of Biological Sciences, The University of Queensland, St. Lucia 4067,
      Queensland, Australia.
FAU - Smith, Martin R
AU  - Smith MR
AD  - Department of Earth Sciences, University of Cambridge, Downing Street, Cambridge 
      CB2 3EQ, UK.
AD  - Department of Earth Sciences, Mountjoy Site, Durham University, South Road,
      Durham DH1 3LE, UK.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
PMC - PMC6568014
OTO - NOTNLM
OT  - Character independence
OT  - character optimization
OT  - cladistic analysis
OT  - inapplicable data
OT  - phylogenetic tree search
EDAT- 2018/12/12 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/12/12 06:00
PHST- 2017/10/25 00:00 [received]
PHST- 2018/11/27 00:00 [revised]
PHST- 2018/11/03 00:00 [accepted]
PHST- 2018/12/12 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
PHST- 2018/12/12 06:00 [entrez]
AID - 5238046 [pii]
AID - 10.1093/sysbio/syy083 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jul 1;68(4):619-631. doi: 10.1093/sysbio/syy083.

PMID- 30521050
OWN - NLM
STAT- MEDLINE
DCOM- 20190408
LR  - 20190408
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - Restriction-Site-Associated DNA Sequencing Reveals a Cryptic Viburnum Species on 
      the North American Coastal Plain.
PG  - 187-203
LID - 10.1093/sysbio/syy084 [doi]
AB  - Species are the starting point for most studies of ecology and evolution, but the
      proper circumscription of species can be extremely difficult in morphologically
      variable lineages, and there are still few convincing examples of molecularly
      informed species delimitation in plants. Here, we focus on the Viburnum nudum
      complex, a highly variable clade that is widely distributed in eastern North
      America. Taxonomic treatments have mostly divided this complex into northern (V. 
      nudum var. cassinoides) and southern (V. nudum var. nudum) entities, but
      additional names have been proposed. We used multiple lines of evidence,
      including RADseq, morphological, and geographic data, to test how many
      independently evolving lineages exist within the V. nudum complex. Genetic
      clustering and phylogenetic methods revealed three distinct groups-one lineage
      that is highly divergent, and two others that are recently diverged and
      morphologically similar. A combination of evidence that includes reciprocal
      monophyly, lack of introgression, and discrete rather than continuous patterns of
      variation supports the recognition of all three lineages as separate species.
      These results identify a surprising case of cryptic diversity in which two
      broadly sympatric species have consistently been lumped in taxonomic treatments. 
      The clarity of our findings is directly related to the dense sampling and
      high-quality genetic data in this study. We argue that there is a critical need
      for carefully sampled and integrative species delimitation studies to clarify
      species boundaries even in well-known plant lineages. Studies following the model
      that we have developed here are likely to identify many more cryptic lineages and
      will fundamentally improve our understanding of plant speciation and patterns of 
      species richness.
FAU - Spriggs, Elizabeth L
AU  - Spriggs EL
AD  - Department of Ecology and Evolutionary Biology, Yale University, PO Box 208106,
      New Haven, CT 06520-8106, USA.
FAU - Eaton, Deren A R
AU  - Eaton DAR
AD  - Department of Ecology and Evolutionary Biology, Yale University, PO Box 208106,
      New Haven, CT 06520-8106, USA.
AD  - Department of Ecology, Evolution, and Environmental Biology, Columbia University,
      New York, NY 10027, USA.
FAU - Sweeney, Patrick W
AU  - Sweeney PW
AD  - Division of Botany, Peabody Museum of Natural History, Yale University, PO Box
      208118, New Haven CT 06529, USA.
FAU - Schlutius, Caroline
AU  - Schlutius C
AD  - Department of Ecology and Evolutionary Biology, Yale University, PO Box 208106,
      New Haven, CT 06520-8106, USA.
FAU - Edwards, Erika J
AU  - Edwards EJ
AD  - Department of Ecology and Evolutionary Biology, Yale University, PO Box 208106,
      New Haven, CT 06520-8106, USA.
AD  - Department of Ecology, Evolution, and Environmental Biology, Columbia University,
      New York, NY 10027, USA.
FAU - Donoghue, Michael J
AU  - Donoghue MJ
AD  - Department of Ecology and Evolutionary Biology, Yale University, PO Box 208106,
      New Haven, CT 06520-8106, USA.
AD  - Division of Botany, Peabody Museum of Natural History, Yale University, PO Box
      208118, New Haven CT 06529, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.m1vc0
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - DNA, Plant/chemistry/*genetics
MH  - Genetic Speciation
MH  - Phylogeny
MH  - Restriction Mapping
MH  - Sequence Analysis, DNA
MH  - Species Specificity
MH  - United States
MH  - Viburnum/anatomy &amp; histology/*classification/*genetics
EDAT- 2018/12/07 06:00
MHDA- 2019/04/09 06:00
CRDT- 2018/12/07 06:00
PHST- 2017/10/10 00:00 [received]
PHST- 2018/12/03 00:00 [accepted]
PHST- 2018/12/07 06:00 [pubmed]
PHST- 2019/04/09 06:00 [medline]
PHST- 2018/12/07 06:00 [entrez]
AID - 5232147 [pii]
AID - 10.1093/sysbio/syy084 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):187-203. doi: 10.1093/sysbio/syy084.

PMID- 30521024
OWN - NLM
STAT- In-Data-Review
LR  - 20190614
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 4
DP  - 2019 Jul 1
TI  - Resolving Deep Nodes in an Ancient Radiation of Neotropical Fishes in the
      Presence of Conflicting Signals from Incomplete Lineage Sorting.
PG  - 573-593
LID - 10.1093/sysbio/syy085 [doi]
AB  - Resolving patterns of ancient and rapid diversifications is one of the most
      challenging tasks in evolutionary biology. These difficulties arise from
      confusing phylogenetic signals that are associated with the interplay of
      incomplete lineage sorting (ILS) and homoplasy. Phylogenomic analyses of
      hundreds, or even thousands, of loci offer the potential to resolve such
      contentious relationships. Yet, how much useful phylogenetic information these
      large data sets contain remains uncertain and often goes untested. Here, we
      assess the utility of different data filtering approaches to maximize
      phylogenetic information and minimize noise when reconstructing an ancient
      radiation of Neotropical electric knifefishes (Order Gymnotiformes) using
      ultraconserved elements. We found two contrasting hypotheses of gymnotiform
      evolutionary relationships depending on whether phylogenetic inferences were
      based on concatenation or coalescent methods. In the first case, all analyses
      inferred a previously-and commonly-proposed hypothesis, where the family
      Apteronotidae was found as the sister group to all other gymnotiform families. In
      contrast, coalescent-based analyses suggested a novel hypothesis where families
      producing pulse-type (viz., Gymnotidae, Hypopomidae, and Rhamphichthyidae) and
      wave-type electric signals (viz., Apteronotidae, Sternopygidae) were reciprocally
      monophyletic. Nodal support for this second hypothesis increased when analyzing
      loci with the highest phylogenetic information content and further increased when
      data were pruned using targeted filtering methods that maximized phylogenetic
      informativeness at the deepest nodes of the Gymnotiformes. Bayesian concordance
      analyses and topology tests of individual gene genealogies demonstrated that the 
      difficulty of resolving this radiation was likely due to high gene-tree
      incongruences that resulted from ILS. We show that data filtering reduces
      gene-tree heterogeneity and increases nodal support and consistency of species
      trees using coalescent methods; however, we failed to observe the same effect
      when using concatenation methods. Furthermore, the targeted filtering strategies 
      applied here support the use of "gene data interrogation" rather than "gene
      genealogy interrogation" approaches in phylogenomic analyses, to extract
      phylogenetic signal from intractable portions of the Tree of Life.
CI  - (c) The Author(s) 2018. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Alda, Fernando
AU  - Alda F
AD  - Museum of Natural Science, Department of Biological Sciences, Louisiana State
      University, Baton Rouge, LA 70803, USA.
AD  - Department of Biology, Geology and Environmental Science, University of Tennessee
      at Chattanooga, Chattanooga, TN 37403, USA.
FAU - Tagliacollo, Victor A
AU  - Tagliacollo VA
AD  - Museu de Zoologia da Universidade de Sao Paulo (MZUSP), Ipirianga, 04263-000, Sao
      Paulo, Sao Paulo, Brazil.
FAU - Bernt, Maxwell J
AU  - Bernt MJ
AD  - Department of Biology, University of Louisiana at Lafayette, Lafayette, LA 70503,
      USA.
FAU - Waltz, Brandon T
AU  - Waltz BT
AD  - Department of Biology, University of Louisiana at Lafayette, Lafayette, LA 70503,
      USA.
FAU - Ludt, William B
AU  - Ludt WB
AD  - Museum of Natural Science, Department of Biological Sciences, Louisiana State
      University, Baton Rouge, LA 70803, USA.
FAU - Faircloth, Brant C
AU  - Faircloth BC
AD  - Museum of Natural Science, Department of Biological Sciences, Louisiana State
      University, Baton Rouge, LA 70803, USA.
FAU - Alfaro, Michael E
AU  - Alfaro ME
AD  - Department of Ecology and Evolutionary Biology, University of California, Los
      Angeles, CA 90095, USA.
FAU - Albert, James S
AU  - Albert JS
AD  - Department of Biology, University of Louisiana at Lafayette, Lafayette, LA 70503,
      USA.
FAU - Chakrabarty, Prosanta
AU  - Chakrabarty P
AD  - Museum of Natural Science, Department of Biological Sciences, Louisiana State
      University, Baton Rouge, LA 70803, USA.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Coalescence
OT  - Gymnotiformes
OT  - ILS
OT  - UCEs
OT  - concatenation
OT  - gene tree
OT  - phylogenomics
OT  - species tree
EDAT- 2018/12/07 06:00
MHDA- 2018/12/07 06:00
CRDT- 2018/12/07 06:00
PHST- 2018/06/12 00:00 [received]
PHST- 2018/11/30 00:00 [revised]
PHST- 2018/12/03 00:00 [accepted]
PHST- 2018/12/07 06:00 [pubmed]
PHST- 2018/12/07 06:00 [medline]
PHST- 2018/12/07 06:00 [entrez]
AID - 5232148 [pii]
AID - 10.1093/sysbio/syy085 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jul 1;68(4):573-593. doi: 10.1093/sysbio/syy085.

PMID- 30517732
OWN - NLM
STAT- In-Data-Review
LR  - 20190619
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 4
DP  - 2019 Jul 1
TI  - Golden Orbweavers Ignore Biological Rules: Phylogenomic and Comparative Analyses 
      Unravel a Complex Evolution of Sexual Size Dimorphism.
PG  - 555-572
LID - 10.1093/sysbio/syy082 [doi]
AB  - Instances of sexual size dimorphism (SSD) provide the context for rigorous tests 
      of biological rules of size evolution, such as Cope's rule (phyletic size
      increase), Rensch's rule (allometric patterns of male and female size), as well
      as male and female body size optima. In certain spider groups, such as the golden
      orbweavers (Nephilidae), extreme female-biased SSD (eSSD, female:male body length
      $\ge$2) is the norm. Nephilid genera construct webs of exaggerated proportions,
      which can be aerial, arboricolous, or intermediate (hybrid). First, we
      established the backbone phylogeny of Nephilidae using 367 anchored hybrid
      enrichment markers, then combined these data with classical markers for a
      reference species-level phylogeny. Second, we used the phylogeny to test Cope and
      Rensch's rules, sex specific size optima, and the coevolution of web size, type, 
      and features with female and male body size and their ratio, SSD. Male, but not
      female, size increases significantly over time, and refutes Cope's rule.
      Allometric analyses reject the converse, Rensch's rule. Male and female body
      sizes are uncorrelated. Female size evolution is random, but males evolve toward 
      an optimum size (3.2-4.9 mm). Overall, female body size correlates positively
      with absolute web size. However, intermediate sized females build the largest
      webs (of the hybrid type), giant female Nephila and Trichonephila build smaller
      webs (of the aerial type), and the smallest females build the smallest webs (of
      the arboricolous type). We propose taxonomic changes based on the criteria of
      clade age, monophyly and exclusivity, classification information content, and
      diagnosability. Spider families, as currently defined, tend to be between 37
      million years old and 98 million years old, and Nephilidae is estimated at 133 Ma
      (97-146), thus deserving family status. We, therefore, resurrect the family
      Nephilidae Simon 1894 that contains Clitaetra Simon 1889, the Cretaceous
      GeratonephilaPoinar and Buckley (2012), Herennia Thorell 1877, IndoetraKuntner
      2006, new rank, Nephila Leach 1815, Nephilengys L. Koch 1872, Nephilingis Kuntner
      2013, Palaeonephila Wunderlich 2004 from Tertiary Baltic amber, and
      TrichonephilaDahl 1911, new rank. We propose the new clade Orbipurae to contain
      Araneidae Clerck 1757, Phonognathidae Simon 1894, new rank, and Nephilidae.
      Nephilid female gigantism is a phylogenetically ancient phenotype (over 100 Ma), 
      as is eSSD, though their magnitudes vary by lineage.
CI  - (c) The Author(s) 2018. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - Kuntner, Matjaz
AU  - Kuntner M
AD  - Evolutionary Zoology Laboratory, Department of Organisms and Ecosystems Research,
      National Institute of Biology, Vecna pot 111, SI-1000 Ljubljana, Slovenia.
AD  - Evolutionary Zoology Laboratory, Biological Institute ZRC SAZU, Novi trg 2,
      SI-1001 Ljubljana, Slovenia.
AD  - Department of Entomology, National Museum of Natural History, Smithsonian
      Institution, 10th and Constitution, NW, Washington, DC 20560-0105, USA.
AD  - Centre for Behavioural Ecology and Evolution, College of Life Sciences, Hubei
      University, 368 Youyi Road, Wuhan, Hubei 430062, China.
FAU - Hamilton, Chris A
AU  - Hamilton CA
AD  - Department of Entomology, Plant Pathology, &amp; Nematology, University of Idaho, 875
      Perimeter Dr. MS 2329, Moscow, ID 83844-2329, USA.
FAU - Cheng, Ren-Chung
AU  - Cheng RC
AD  - Evolutionary Zoology Laboratory, Biological Institute ZRC SAZU, Novi trg 2,
      SI-1001 Ljubljana, Slovenia.
AD  - Department of Life Sciences, National Chung Hsing University, No.145 Xingda Rd., 
      South Dist., Taichung City 402, Taiwan.
FAU - Gregoric, Matjaz
AU  - Gregoric M
AD  - Evolutionary Zoology Laboratory, Biological Institute ZRC SAZU, Novi trg 2,
      SI-1001 Ljubljana, Slovenia.
FAU - Lupse, Nik
AU  - Lupse N
AD  - Evolutionary Zoology Laboratory, Biological Institute ZRC SAZU, Novi trg 2,
      SI-1001 Ljubljana, Slovenia.
AD  - Division of Animal Evolutionary Biology, Department of Zoology, Faculty of
      Science, Charles University, Vinicna 7, 128 44 Prague, Czech Republic.
FAU - Lokovsek, Tjasa
AU  - Lokovsek T
AD  - Evolutionary Zoology Laboratory, Biological Institute ZRC SAZU, Novi trg 2,
      SI-1001 Ljubljana, Slovenia.
FAU - Lemmon, Emily Moriarty
AU  - Lemmon EM
AD  - Department of Biological Science, Florida State University, 319 Stadium Dr.,
      Tallahassee, FL 32306-4295, USA.
FAU - Lemmon, Alan R
AU  - Lemmon AR
AD  - Department of Scientific Computing, Florida State University, 400 Dirac Science
      Library, Tallahassee, FL 32306-4120, USA.
FAU - Agnarsson, Ingi
AU  - Agnarsson I
AD  - Department of Entomology, National Museum of Natural History, Smithsonian
      Institution, 10th and Constitution, NW, Washington, DC 20560-0105, USA.
AD  - Department of Biology, University of Vermont, 316 Marsh Life Science Building,
      109 Carrigan Drive, Burlington, VT 05405-0086, USA.
FAU - Coddington, Jonathan A
AU  - Coddington JA
AD  - Department of Entomology, National Museum of Natural History, Smithsonian
      Institution, 10th and Constitution, NW, Washington, DC 20560-0105, USA.
FAU - Bond, Jason E
AU  - Bond JE
AD  - Department of Entomology and Nematology, University of California Davis, 1
      Shields Drive, Davis, CA 95616, USA.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
PMC - PMC6568015
OTO - NOTNLM
OT  - Body size evolution
OT  - Cope's rule
OT  - Nephilidae
OT  - Orbipurae
OT  - Phonognathidae
OT  - Rensch's rule
OT  - female gigantism
OT  - web asymmetry
EDAT- 2018/12/06 06:00
MHDA- 2018/12/06 06:00
CRDT- 2018/12/06 06:00
PHST- 2018/07/05 00:00 [received]
PHST- 2018/11/20 00:00 [revised]
PHST- 2018/11/26 00:00 [accepted]
PHST- 2018/12/06 06:00 [pubmed]
PHST- 2018/12/06 06:00 [medline]
PHST- 2018/12/06 06:00 [entrez]
AID - 5229942 [pii]
AID - 10.1093/sysbio/syy082 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jul 1;68(4):555-572. doi: 10.1093/sysbio/syy082.

PMID- 30481358
OWN - NLM
STAT- MEDLINE
DCOM- 20190430
LR  - 20190430
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 3
DP  - 2019 May 1
TI  - Reconstructing the Evolution of Giant Extinct Kangaroos: Comparing the Utility of
      DNA, Morphology, and Total Evidence.
PG  - 520-537
LID - 10.1093/sysbio/syy080 [doi]
AB  - Combined "total evidence" analysis of molecular and morphological data offers the
      opportunity to objectively merge fossils into the tree of life, and challenges
      the primacy of solely DNA based phylogenetic and dating inference, even among
      modern taxa. To investigate the relative utility of DNA, morphology, and total
      evidence for evolutionary inference, we sequenced the first near-complete
      mitochondrial genomes from extinct Australian megafauna: a 40-50 thousand year
      old giant short-faced kangaroo (Simosthenurus occidentalis) and giant wallaby
      (Protemnodon anak). We analyzed the ancient DNA and fossil data alongside
      comparable data from extant species to infer phylogeny, divergence times, and
      ancestral body mass among macropods (kangaroos and wallabies). Our results
      confirm a close relationship between Protemnodon and the iconic kangaroo genus
      complex "Macropus", and unite the giant Simothenurus with the hare-sized
      Lagostrophus fasciatus (banded hare-wallaby), suggesting that the latter is the
      closest living link to the once diverse sthenurine kangaroo radiation. We find
      that large body size evolved multiple times among kangaroos, coincident with
      expansion of open woodland habitats beginning in the Late Miocene. In addition,
      our results suggest that morphological data mislead macropod phylogeny
      reconstruction and in turn can distort total evidence estimation of divergence
      dates. However, a novel result with potentially broad application is that the
      accuracy and precision of reconstructing ancestral body mass was improved by
      tracing body mass on morphological branch lengths. This is likely due to positive
      allometric correlation between morphological and body size variation-a
      relationship that may be masked or even misleadingly inverted with the temporal
      or molecular branch lengths that typically underpin ancestral body size
      reconstruction. Our study supports complementary roles for DNA and morphology in 
      evolutionary inference, and opens a new window into the evolution of Australia's 
      unique marsupial fauna.
CI  - (c) The Author(s) 2018. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Cascini, Manuela
AU  - Cascini M
AD  - School of Earth, Environmental and Biological Sciences, Queensland University of 
      Technology, 2, George Street, Brisbane, QLD 4000, Australia.
FAU - Mitchell, Kieren J
AU  - Mitchell KJ
AD  - Australian Centre for Ancient DNA, School of Biological Sciences, University of
      Adelaide, North Terrace Campus, South Australia 5005, Australia.
FAU - Cooper, Alan
AU  - Cooper A
AD  - Australian Centre for Ancient DNA, School of Biological Sciences, University of
      Adelaide, North Terrace Campus, South Australia 5005, Australia.
FAU - Phillips, Matthew J
AU  - Phillips MJ
AD  - School of Earth, Environmental and Biological Sciences, Queensland University of 
      Technology, 2, George Street, Brisbane, QLD 4000, Australia.
LA  - eng
SI  - Dryad/10.5061/dryad.7809km8
SI  - figshare/10.25909/5bf245f03a511
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ancient)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Classification
MH  - DNA/*genetics
MH  - DNA, Ancient
MH  - Fossils/*anatomy &amp; histology
MH  - Macropodidae/anatomy &amp; histology/*classification/genetics
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Ancestral state reconstruction
OT  - *Macropodidae
OT  - *ancient DNA
OT  - *morphology
OT  - *phylogenetic inference
OT  - *total evidence
EDAT- 2018/11/28 06:00
MHDA- 2019/05/01 06:00
CRDT- 2018/11/28 06:00
PHST- 2018/07/05 00:00 [received]
PHST- 2018/11/20 00:00 [revised]
PHST- 2018/11/20 00:00 [accepted]
PHST- 2018/11/28 06:00 [pubmed]
PHST- 2019/05/01 06:00 [medline]
PHST- 2018/11/28 06:00 [entrez]
AID - 5210880 [pii]
AID - 10.1093/sysbio/syy080 [doi]
PST - ppublish
SO  - Syst Biol. 2019 May 1;68(3):520-537. doi: 10.1093/sysbio/syy080.

PMID- 30481343
OWN - NLM
STAT- MEDLINE
DCOM- 20190430
LR  - 20190430
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 3
DP  - 2019 May 1
TI  - Phylogenies and Diversification Rates: Variance Cannot Be Ignored.
PG  - 538-550
LID - 10.1093/sysbio/syy079 [doi]
AB  - A recent pair of articles published in the journal Evolution presented a test for
      assessing the validity of hierarchical macroevolutionary models. The premise of
      the test is to compare numerical point estimates of parameters from two levels of
      analysis; if the estimates differ, the hierarchical model is purportedly flawed. 
      The articles in question apply their proposed test to BAMM, a scientific software
      program that uses a Bayesian mixture model to estimate rates of evolution from
      phylogenetic trees. The authors use BAMM to estimate rates from large phylogenies
      ($n &gt; 60$ tips) and they apply the method separately to subclades within those
      phylogenies (median size: $n = 3$ tips); they find that point estimates of rates 
      differ between these levels and conclude that the method is flawed, but they do
      not test whether the observed differences are statistically meaningful. There is 
      no consideration of sampling variation and its impact at any level of their
      analysis. Here, I show that numerical differences across groups that they report 
      are fully explained by a failure to account for sampling variation in their point
      estimates. Variance in evolutionary rate estimates-from BAMM and all other
      methods-is an inverse function of clade size; this variance is extreme for clades
      with five or fewer tips (e.g., 70% of clades in the focal study). The articles in
      question rely on negative results that are easily explained by low statistical
      power to reject their preferred null hypothesis, and this low power is a trivial 
      consequence of high variance in their point estimates. I describe additional
      mathematical and statistical mistakes that render the proposed testing framework 
      invalid on first principles. Evolutionary rates are no different than any other
      population parameters we might wish to estimate, and biologists should use the
      training and tools already at their disposal to avoid erroneous results that
      follow from the neglect of variance. [Confidence interval; diversification;
      estimation; null hypothesis; phylogeny; statistics; type I error].
CI  - (c) The Author(s) 2018. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Rabosky, Daniel L
AU  - Rabosky DL
AD  - Museum of Zoology and Department of Ecology and Evolutionary Biology, Biological 
      Sciences Building, 1105 North University Avenue, University of Michigan, Ann
      Arbor, MI 48109, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.61jn743
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - *Biodiversity
MH  - *Classification
MH  - Computer Simulation
MH  - Genetic Speciation
MH  - Models, Biological
MH  - *Models, Statistical
MH  - *Phylogeny
EDAT- 2018/11/28 06:00
MHDA- 2019/05/01 06:00
CRDT- 2018/11/28 06:00
PHST- 2018/09/23 00:00 [received]
PHST- 2018/11/16 00:00 [revised]
PHST- 2018/11/17 00:00 [accepted]
PHST- 2018/11/28 06:00 [pubmed]
PHST- 2019/05/01 06:00 [medline]
PHST- 2018/11/28 06:00 [entrez]
AID - 5210881 [pii]
AID - 10.1093/sysbio/syy079 [doi]
PST - ppublish
SO  - Syst Biol. 2019 May 1;68(3):538-550. doi: 10.1093/sysbio/syy079.

PMID- 30476308
OWN - NLM
STAT- MEDLINE
DCOM- 20190430
LR  - 20190430
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 3
DP  - 2019 May 1
TI  - Stochastic Character Mapping of State-Dependent Diversification Reveals the Tempo
      of Evolutionary Decline in Self-Compatible Onagraceae Lineages.
PG  - 505-519
LID - 10.1093/sysbio/syy078 [doi]
AB  - A major goal of evolutionary biology is to identify key evolutionary transitions 
      that correspond with shifts in speciation and extinction rates. Stochastic
      character mapping has become the primary method used to infer the timing, nature,
      and number of character state transitions along the branches of a phylogeny. The 
      method is widely employed for standard substitution models of character
      evolution. However, current approaches cannot be used for models that
      specifically test the association of character state transitions with shifts in
      diversification rates such as state-dependent speciation and extinction (SSE)
      models. Herein, we introduce a new stochastic character mapping algorithm that
      overcomes these limitations, and apply it to study mating system evolution over a
      time-calibrated phylogeny of the plant family Onagraceae. Utilizing a hidden
      state SSE model we tested the association of the loss of self-incompatibility
      (SI) with shifts in diversification rates. We found that self-compatible lineages
      have higher extinction rates and lower net-diversification rates compared with
      self-incompatible lineages. Furthermore, these results provide empirical evidence
      for the "senescing" diversification rates predicted in highly selfing lineages:
      our mapped character histories show that the loss of SI is followed by a
      short-term spike in speciation rates, which declines after a time lag of several 
      million years resulting in negative net-diversification. Lineages that have long 
      been self-compatible, such as Fuchsia and Clarkia, are in a previously
      unrecognized and ongoing evolutionary decline. Our results demonstrate that
      stochastic character mapping of SSE models is a powerful tool for examining the
      timing and nature of both character state transitions and shifts in
      diversification rates over the phylogeny.
CI  - (c) The Author(s) 2018. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Freyman, William A
AU  - Freyman WA
AD  - Department of Integrative Biology, University of California, Berkeley, 3040
      Valley Life Sciences Building #3140, CA 94720, USA.
FAU - Hohna, Sebastian
AU  - Hohna S
AD  - Division of Evolutionary Biology, Ludwig-Maximilians-Universitat Munchen,
      Geschwister-Scholl-Platz 1, 80539 Munich, Germany.
LA  - eng
SI  - Dryad/10.5061/dryad.9jj428f
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Extinction, Biological
MH  - Genetic Speciation
MH  - *Models, Biological
MH  - Onagraceae/*classification
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Bayesian inference
OT  - *Onagraceae
OT  - *self-incompatibility
OT  - *state-dependent speciation and extinction
OT  - *stochastic character mapping
EDAT- 2018/11/27 06:00
MHDA- 2019/05/01 06:00
CRDT- 2018/11/27 06:00
PHST- 2018/01/16 00:00 [received]
PHST- 2018/11/05 00:00 [revised]
PHST- 2018/11/13 00:00 [accepted]
PHST- 2018/11/27 06:00 [pubmed]
PHST- 2019/05/01 06:00 [medline]
PHST- 2018/11/27 06:00 [entrez]
AID - 5200730 [pii]
AID - 10.1093/sysbio/syy078 [doi]
PST - ppublish
SO  - Syst Biol. 2019 May 1;68(3):505-519. doi: 10.1093/sysbio/syy078.

PMID- 30445649
OWN - NLM
STAT- MEDLINE
DCOM- 20190430
LR  - 20190430
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 3
DP  - 2019 May 1
TI  - True Homoplasy of Retrotransposon Insertions in Primates.
PG  - 482-493
LID - 10.1093/sysbio/syy076 [doi]
AB  - How reliable are the presence/absence insertion patterns of the supposedly
      homoplasy-free retrotransposons, which were randomly inserted in the quasi
      infinite genomic space? To systematically examine this question in an up-to-date,
      multigenome comparison, we screened millions of primate transposed Alu SINE
      elements for incidences of homoplasious precise insertions and deletions. In
      genome-wide analyses, we identified and manually verified nine cases of precise
      parallel Alu insertions of apparently identical elements at orthologous positions
      in two ape lineages and twelve incidences of precise deletions of previously
      established SINEs. Correspondingly, eight precise parallel insertions and no
      exact deletions were detected in a comparison of lemuriform primate and human
      insertions spanning the range of primate diversity. With an overall frequency of 
      homoplasious Alu insertions of only 0.01% (for human-chimpanzee-rhesus macaque)
      and 0.02-0.04% (for human-bushbaby-lemurs) and precise Alu deletions of
      0.001-0.002% (for human-chimpanzee-rhesus macaque), real homoplasy is not
      considered to be a quantitatively relevant source of evolutionary noise. Thus,
      presence/absence patterns of Alu retrotransposons and, presumably, all
      LINE1-mobilized elements represent indeed the virtually homoplasy-free markers
      they are considered to be. Therefore, ancestral incomplete lineage sorting and
      hybridization remain the only serious sources of conflicting presence/absence
      patterns of retrotransposon insertions, and as such are detectable and
      quantifiable. [Homoplasy; precise deletions; precise parallel insertions;
      primates; retrotransposons.].
CI  - (c) The Author(s) 2018. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Doronina, Liliya
AU  - Doronina L
AD  - Institute of Experimental Pathology (ZMBE), University of Munster,
      Von-Esmarch-Str. 56, D-48149 Munster, Germany.
FAU - Reising, Olga
AU  - Reising O
AD  - Institute of Experimental Pathology (ZMBE), University of Munster,
      Von-Esmarch-Str. 56, D-48149 Munster, Germany.
FAU - Clawson, Hiram
AU  - Clawson H
AD  - Department of Biomolecular Engineering, University of California, 1156 High
      Street, Santa Cruz, CA, USA.
FAU - Ray, David A
AU  - Ray DA
AD  - Department of Biological Sciences, Texas Tech University, 2901 Main Street,
      Lubbock, TX, USA.
FAU - Schmitz, Jurgen
AU  - Schmitz J
AD  - Institute of Experimental Pathology (ZMBE), University of Munster,
      Von-Esmarch-Str. 56, D-48149 Munster, Germany.
LA  - eng
SI  - Dryad/10.5061/dryad.532c28n
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Retroelements)
SB  - IM
MH  - Alu Elements/*genetics
MH  - Animals
MH  - Evolution, Molecular
MH  - Genetic Variation
MH  - Humans
MH  - Mutagenesis, Insertional/*genetics
MH  - Phylogeny
MH  - Primates/classification/*genetics
MH  - Retroelements/*genetics
EDAT- 2018/11/18 06:00
MHDA- 2019/05/01 06:00
CRDT- 2018/11/17 06:00
PHST- 2018/08/17 00:00 [received]
PHST- 2018/11/05 00:00 [revised]
PHST- 2018/11/13 00:00 [accepted]
PHST- 2018/11/18 06:00 [pubmed]
PHST- 2019/05/01 06:00 [medline]
PHST- 2018/11/17 06:00 [entrez]
AID - 5184276 [pii]
AID - 10.1093/sysbio/syy076 [doi]
PST - ppublish
SO  - Syst Biol. 2019 May 1;68(3):482-493. doi: 10.1093/sysbio/syy076.

PMID- 30445627
OWN - NLM
STAT- MEDLINE
DCOM- 20190430
LR  - 20190430
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 3
DP  - 2019 May 1
TI  - Morphological Data Sets Fit a Common Mechanism Much More Poorly than DNA
      Sequences and Call Into Question the Mkv Model.
PG  - 494-504
LID - 10.1093/sysbio/syy077 [doi]
AB  - The Mkv evolutionary model, based on minor modifications to models of molecular
      evolution, is being increasingly used to infer phylogenies from discrete
      morphological data, often producing different results from parsimony. The
      critical difference between Mkv and parsimony is the assumption of a "common
      mechanism" in the Mkv model, with branch lengths determining that probability of 
      change for all characters increases or decreases at the same tree branches by the
      same exponential factor. We evaluate whether the assumption of a common mechanism
      applies to morphology, by testing the implicit prediction that branch lengths
      calculated from different subsets of characters will be significantly correlated.
      Our analysis shows that DNA (38 data sets tested) is often compatible with a
      common mechanism, but morphology (86 data sets tested) generally is not, showing 
      very disparate branch lengths for different character partitions. The low levels 
      of branch length correlation demonstrated for morphology (fitting models without 
      a common mechanism) suggest that the Mkv model is too unrealistic and inadequate 
      for the analysis of most morphological data sets. [Bayesian analysis; Mkv model; 
      morphological data; phylogenetics.].
CI  - (c) The Author(s) 2018. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Goloboff, Pablo A
AU  - Goloboff PA
AD  - Unidad Ejecutora Lillo (UEL), Consejo Nacional de Investigaciones Cientificas y
      Tecnicas (CONICET), S.M. Tucuman, Argentina.
FAU - Pittman, Michael
AU  - Pittman M
AD  - Vertebrate Palaeontology Laboratory, Department of Earth Sciences, University of 
      Hong Kong, Pokfulam, Hong Kong.
FAU - Pol, Diego
AU  - Pol D
AD  - Museo Egidio Feruglio, Consejo Nacional de Investigaciones Cientificas y Tecnicas
      (CONICET), Trelew, Argentina.
FAU - Xu, Xing
AU  - Xu X
AD  - Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate
      Paleontology and Paleoanthropology, Chinese Academy of Sciences, Beijing, China.
LA  - eng
SI  - Dryad/10.5061/dryad.74ms329
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Base Sequence
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Evolution, Molecular
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2018/11/18 06:00
MHDA- 2019/05/01 06:00
CRDT- 2018/11/17 06:00
PHST- 2018/06/29 00:00 [received]
PHST- 2018/11/11 00:00 [revised]
PHST- 2018/11/13 00:00 [accepted]
PHST- 2018/11/18 06:00 [pubmed]
PHST- 2019/05/01 06:00 [medline]
PHST- 2018/11/17 06:00 [entrez]
AID - 5184275 [pii]
AID - 10.1093/sysbio/syy077 [doi]
PST - ppublish
SO  - Syst Biol. 2019 May 1;68(3):494-504. doi: 10.1093/sysbio/syy077.

PMID- 30418649
OWN - NLM
STAT- In-Data-Review
LR  - 20190614
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 4
DP  - 2019 Jul 1
TI  - Exploring and Visualizing Spaces of Tree Reconciliations.
PG  - 607-618
LID - 10.1093/sysbio/syy075 [doi]
AB  - Tree reconciliation is the mathematical tool that is used to investigate the
      coevolution of organisms, such as hosts and parasites. A common approach to tree 
      reconciliation involves specifying a model that assigns costs to certain events, 
      such as cospeciation, and then tries to find a mapping between two specified
      phylogenetic trees which minimizes the total cost of the implied events. For such
      models, it has been shown that there may be a huge number of optimal solutions,
      or at least solutions that are close to optimal. It is therefore of interest to
      be able to systematically compare and visualize whole collections of
      reconciliations between a specified pair of trees. In this article, we consider
      various metrics on the set of all possible reconciliations between a pair of
      trees, some that have been defined before but also new metrics that we shall
      propose. We show that the diameter for the resulting spaces of reconciliations
      can in some cases be determined theoretically, information that we use to
      normalize and compare properties of the metrics. We also implement the metrics
      and compare their behavior on several host parasite data sets, including the
      shapes of their distributions. In addition, we show that in combination with
      multidimensional scaling, the metrics can be useful for visualizing large
      collections of reconciliations, much in the same way as phylogenetic tree metrics
      can be used to explore collections of phylogenetic trees. Implementations of the 
      metrics can be downloaded from:
      https://team.inria.fr/erable/en/team-members/blerina-sinaimeri/reconciliation-dis
      tances/.
CI  - (c) The Author(s) 2018. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Huber, Katharina T
AU  - Huber KT
AD  - School of Computing Sciences, University of East Anglia, Norwich, UK.
FAU - Moulton, Vincent
AU  - Moulton V
AD  - School of Computing Sciences, University of East Anglia, Norwich, UK.
FAU - Sagot, Marie-France
AU  - Sagot MF
AD  - Inria Grenoble - Rhone-Alpes; Inovallee 655, Avenue de l'Europe, Montbonnot,
      38334 Saint Ismier Cedex, France.
AD  - Universite de Lyon, F-69000 Lyon, France.
AD  - Universite Lyon 1, CNRS, UMR5558, 43 Boulevard du 11 Novembre 1918, 69622
      Villeurbanne Cedex, France.
FAU - Sinaimeri, Blerina
AU  - Sinaimeri B
AD  - Inria Grenoble - Rhone-Alpes; Inovallee 655, Avenue de l'Europe, Montbonnot,
      38334 Saint Ismier Cedex, France.
AD  - Universite de Lyon, F-69000 Lyon, France.
AD  - Universite Lyon 1, CNRS, UMR5558, 43 Boulevard du 11 Novembre 1918, 69622
      Villeurbanne Cedex, France.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
OTO - NOTNLM
OT  - Coevolution
OT  - phylogenetic tree
OT  - reconciliation
OT  - reconciliation space
EDAT- 2018/11/13 06:00
MHDA- 2018/11/13 06:00
CRDT- 2018/11/13 06:00
PHST- 2018/04/20 00:00 [received]
PHST- 2018/10/31 00:00 [revised]
PHST- 2018/11/02 00:00 [accepted]
PHST- 2018/11/13 06:00 [pubmed]
PHST- 2018/11/13 06:00 [medline]
PHST- 2018/11/13 06:00 [entrez]
AID - 5172365 [pii]
AID - 10.1093/sysbio/syy075 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jul 1;68(4):607-618. doi: 10.1093/sysbio/syy075.

PMID- 30395332
OWN - NLM
STAT- MEDLINE
DCOM- 20190408
LR  - 20190804
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - Phylogenomic Signatures of Ancient Introgression in a Rogue Lineage of Darters
      (Teleostei: Percidae).
PG  - 329-346
LID - 10.1093/sysbio/syy074 [doi]
AB  - Evolutionary history is typically portrayed as a branching phylogenetic tree, yet
      not all evolution proceeds in a purely bifurcating manner. Introgressive
      hybridization is one process that results in reticulate evolution. Most known
      examples of genome-wide introgression occur among closely related species with
      relatively recent common ancestry; however, we present evidence for ancient
      hybridization and genome-wide introgression between major stem lineages of
      darters, a species-rich clade of North American freshwater fishes. Previous
      attempts to resolve the relationships of darters have been confounded by the
      uncertain phylogenetic resolution of the lineage Allohistium. In this study, we
      investigate the phylogenomics of darters, specifically the relationships of
      Allohistium, through analyses of approximately 30,000 RADseq loci sampled from
      112 species. Our phylogenetic inferences are based on traditional approaches in
      combination with strategies that accommodate reticulate evolution. These analyses
      result in a novel phylogenetic hypothesis for darters that includes ancient
      introgression between Allohistium and other two major darter lineages, minimally 
      occurring 20 million years ago. Darters offer a compelling case for the necessity
      of incorporating phylogenetic networks in reconstructing the evolutionary history
      of diversification in species-rich lineages. We anticipate that the growing
      wealth of genomic data for clades of non-model organisms will reveal more
      examples of ancient hybridization, eventually requiring a re-evaluation of how
      evolutionary history is visualized and utilized in macroevolutonary
      investigations.
FAU - MacGuigan, Daniel J
AU  - MacGuigan DJ
AD  - Department of Ecology and Evolutionary Biology, Yale University, P.O. Box 208106,
      New Haven, CT 06520, USA.
FAU - Near, Thomas J
AU  - Near TJ
AD  - Department of Ecology and Evolutionary Biology, Yale University, P.O. Box 208106,
      New Haven, CT 06520, USA.
AD  - Peabody Museum of Natural History, Yale University, New Haven, CT 06520, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.1c9g5mm
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Genetic Variation
MH  - Genome/genetics
MH  - North America
MH  - Perches/*classification/genetics
MH  - *Phylogeny
PMC - PMC6657582
EDAT- 2018/11/06 06:00
MHDA- 2019/04/09 06:00
CRDT- 2018/11/06 06:00
PMCR- 2020/03/01 00:00
PHST- 2018/04/24 00:00 [received]
PHST- 2018/10/29 00:00 [accepted]
PHST- 2020/03/01 00:00 [pmc-release]
PHST- 2018/11/06 06:00 [pubmed]
PHST- 2019/04/09 06:00 [medline]
PHST- 2018/11/06 06:00 [entrez]
AID - 5159012 [pii]
AID - 10.1093/sysbio/syy074 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):329-346. doi: 10.1093/sysbio/syy074.

PMID- 30371872
OWN - NLM
STAT- MEDLINE
DCOM- 20190408
LR  - 20190408
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - Species Limits and Phylogenomic Relationships of Darwin's Finches Remain
      Unresolved: Potential Consequences of a Volatile Ecological Setting.
PG  - 347-357
LID - 10.1093/sysbio/syy073 [doi]
AB  - Island biotas have become paradigms for illustrating many evolutionary processes.
      The fauna of the Galapagos Islands includes several taxa that have been focal
      points for evolutionary studies. Perhaps their most famous inhabitants, Darwin's 
      finches, represent a go-to icon when thinking about how species originate and
      adapt to the environment. However, unlike other adaptive radiations, past
      morphological and molecular studies of Darwin's finches have yielded inconsistent
      hypotheses of species limits and phylogenetic relationships. Expecting that
      idiosyncrasies of prior data and analytic methods explained different proposed
      classifications, we were surprised to observe that three new phylogenetic
      hypotheses derived mostly from the same genomics data were topologically
      inconsistent. We found that the differences between some of these genomics trees 
      were as great as one would expect between two random trees with the same number
      of taxa. Thus, the phylogeny of Darwin's finches remains unresolved, as it has
      for more than a century. A component of phylogenetic uncertainty comes from
      unclear species limits, under any species concept, in the ground finches
      (Geospiza) and tree finches (Camarhynchus). We suggest that past authors should
      have tested the species limits of Lack, rather than uncritically accepting them. 
      In fact, the impressive amount of genomics data do not provide unambiguous
      hypotheses of the number of species of Geospiza or Camarhynchus, although they
      imply greater species diversity than Lack's taxonomy. We suggest that
      insufficient sampling of species populations across islands (35.6% for
      morphometrics and 20.4% for genomics) prevents accurate diagnoses of species
      limits. However, it is unknown whether samples from a greater number of islands
      might result in bridging differences between species, or reveal many new ones. We
      conclude that attempts to interpret patterns of variation among the finches under
      standard evolutionary paradigms have obscured some major messages, most
      specifically the ongoing reciprocal interactions between geographic isolation and
      lineage divergence, and dispersal and gene flow caused by the volatile ecological
      conditions in the islands. Although the finches provide textbook examples of
      natural selection, better understanding of species limits and a robust
      phylogenetic hypothesis are required to corroborate past hypotheses of speciation
      and adaptive radiation in the finches of the Galapagos.
FAU - Zink, Robert M
AU  - Zink RM
AD  - School of Natural Resources.
AD  - School of Biological Sciences.
AD  - Nebraska State Museum, University of Nebraska-Lincoln, Lincoln, NE 68583, USA.
FAU - Vazquez-Miranda, Hernan
AU  - Vazquez-Miranda H
AD  - School of Natural Resources.
AD  - Nebraska State Museum, University of Nebraska-Lincoln, Lincoln, NE 68583, USA.
AD  - Coleccion Nacional de Aves (CNAV), Departamento de Zoologia, Instituto de
      Biologia, Universidad Nacional Autonoma de Mexico, Ciudad de Mexico, C.P. 04500, 
      Mexico.
LA  - eng
SI  - Dryad/10.5061/dryad.dc82gp3
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Ecosystem
MH  - Ecuador
MH  - Finches/*classification/genetics
MH  - Genetic Variation
MH  - Genome/genetics
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2018/10/30 06:00
MHDA- 2019/04/09 06:00
CRDT- 2018/10/30 06:00
PHST- 2018/05/26 00:00 [received]
PHST- 2018/10/25 00:00 [accepted]
PHST- 2018/10/30 06:00 [pubmed]
PHST- 2019/04/09 06:00 [medline]
PHST- 2018/10/30 06:00 [entrez]
AID - 5146315 [pii]
AID - 10.1093/sysbio/syy073 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):347-357. doi: 10.1093/sysbio/syy073.

PMID- 30365031
OWN - NLM
STAT- MEDLINE
DCOM- 20190430
LR  - 20190430
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 3
DP  - 2019 May 1
TI  - Changing Ecological Opportunities Facilitated the Explosive Diversification of
      New Caledonian Oxera (Lamiaceae).
PG  - 460-481
LID - 10.1093/sysbio/syy070 [doi]
AB  - Phylogenies recurrently demonstrate that oceanic island systems have been home to
      rapid clade diversification and adaptive radiations. The existence of adaptive
      radiations posits a central role of natural selection causing ecological
      divergence and speciation, and some plant radiations have been highlighted as
      paradigmatic examples of such radiations. However, neutral processes may also
      drive speciation during clade radiations, with ecological divergence occurring
      following speciation. Here, we document an exceptionally rapid and unique
      radiation of Lamiaceae within the New Caledonian biodiversity hotspot.
      Specifically, we investigated various biological, ecological, and geographical
      drivers of species diversification within the genus Oxera. We found that Oxera
      underwent an initial process of rapid cladogenesis likely triggered by a dramatic
      period of aridity during the early Pliocene. This early diversification of Oxera 
      was associated with an important phase of ecological diversification triggered by
      significant shifts of pollination syndromes, dispersal modes, and life forms.
      Finally, recent diversification of Oxera appears to have been further driven by
      the interplay of allopatry and habitat shifts likely related to climatic
      oscillations. This suggests that Oxera could be regarded as an adaptive radiation
      at an early evolutionary stage that has been obscured by more recent joint
      habitat diversification and neutral geographical processes. Diversification
      within Oxera has perhaps been triggered by varied ecological and biological
      drivers acting in a leapfrog pattern, but geographic processes may have been an
      equally important driver. We suspect that strictly adaptive radiations may be
      rare in plants and that most events of rapid clade diversification may have
      involved a mixture of geographical and ecological divergence.
CI  - (c) The Author(s) 2018. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - Barrabe, Laure
AU  - Barrabe L
AD  - Institut Agronomique neo-Caledonien (IAC), Equipes ARBOREAL and SOLVEG, BP 711,
      Mont-Dore 98810, New Caledonia.
AD  - Endemia, Plant Red List Authority, 7 rue Pierre Artigue, Noumea 98800, New
      Caledonia.
FAU - Lavergne, Sebastien
AU  - Lavergne S
AD  - Laboratoire d'Ecologie Alpine, CNRS - Universite Grenoble Alpes, UMR 5553,
      Grenoble F-38000, France.
FAU - Karnadi-Abdelkader, Giliane
AU  - Karnadi-Abdelkader G
AD  - Institut Agronomique neo-Caledonien (IAC), Equipes ARBOREAL and SOLVEG, BP 711,
      Mont-Dore 98810, New Caledonia.
FAU - Drew, Bryan T
AU  - Drew BT
AD  - Department of Biology, University of Nebraska-Kearney, Kearney, NE 68849, USA.
FAU - Birnbaum, Philippe
AU  - Birnbaum P
AD  - Institut Agronomique neo-Caledonien (IAC), Equipes ARBOREAL and SOLVEG, BP 711,
      Mont-Dore 98810, New Caledonia.
AD  - UMR AMAP, Universite de Montpellier, CIRAD, CNRS, INRA, IRD, Montpellier 34398,
      France.
FAU - Gateble, Gildas
AU  - Gateble G
AD  - Institut Agronomique neo-Caledonien (IAC), Equipes ARBOREAL and SOLVEG, BP 711,
      Mont-Dore 98810, New Caledonia.
LA  - eng
SI  - Dryad/10.5061/dryad.mm1ft40
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Physiological
MH  - Biodiversity
MH  - *Ecosystem
MH  - *Genetic Speciation
MH  - Lamiaceae/*classification/*physiology
MH  - Phylogeny
PMC - PMC6472440
OTO - NOTNLM
OT  - *Adaptive radiation
OT  - *Lamiaceae
OT  - *New Caledonia
OT  - *Oxera
OT  - *allopatry
OT  - *leapfrog radiation
OT  - *niche shifts
EDAT- 2018/10/27 06:00
MHDA- 2019/05/01 06:00
CRDT- 2018/10/27 06:00
PHST- 2018/07/09 00:00 [received]
PHST- 2018/10/04 00:00 [revised]
PHST- 2018/10/23 00:00 [accepted]
PHST- 2018/10/27 06:00 [pubmed]
PHST- 2019/05/01 06:00 [medline]
PHST- 2018/10/27 06:00 [entrez]
AID - 5145071 [pii]
AID - 10.1093/sysbio/syy070 [doi]
PST - ppublish
SO  - Syst Biol. 2019 May 1;68(3):460-481. doi: 10.1093/sysbio/syy070.

PMID- 30365021
OWN - NLM
STAT- MEDLINE
DCOM- 20190408
LR  - 20190408
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - Serial Homology and Correlated Characters in Morphological Phylogenetics:
      Modeling the Evolution of Dental Crests in Placentals.
PG  - 267-280
LID - 10.1093/sysbio/syy071 [doi]
AB  - Accurate modeling of the complexity of morphological evolution is crucial for
      morphological phylogenetics and for performing tests on a wide variety of
      evolutionary scenarios. In this context, morphological integration and the
      problem of correlated categorical characters represent a major challenge. In
      particular, the magnitude and implications of correlations among serially
      homologous structures such as teeth have been much debated but were never tested 
      statistically within a broad phylogenetic context. Here, we present a large-scale
      empirical study analyzing the serial variation of cingular crests on successive
      molars (M1, M2, and M3) of 274 placental species in a phylogenetic context. Both 
      likelihood analyses and analysis of phylogenetic co-distributions demonstrated
      highly correlated evolution in the entire sample and thus the non-independence of
      these serial features at a macroevolutionary scale. Likelihood analyses show that
      their serial variation should be better scored within a single composite
      character model with constrained paths for transitions enabling simultaneous
      changes on all three molars, which suggests a strong developmental or genetic
      integration. These results are congruent with current genetic and developmental
      knowledge related to dental morphological variation and call into question the
      frequent use of separate characters scored on serially homologous structures of
      the dentition in phylogenetic analyses. Overall, they provide long-overdue and
      clear empirical evidence that in-depth studies of patterns of integration
      constitute an essential step toward more realistic character construction and
      modeling. This approach is critical for more accurate morphological phylogenetics
      and, more generally, for testing macroevolutionary scenarios on groups of
      correlated characters.
FAU - Billet, Guillaume
AU  - Billet G
AD  - CR2P, UMR 7207, Museum national d'Histoire naturelle, CNRS, Sorbonne Universite, 
      8 rue Buffon 75005 Paris, France.
FAU - Bardin, Jeremie
AU  - Bardin J
AD  - CR2P, UMR 7207, Sorbonne Universite, MNHN, CNRS, T.46-56, E.5, case 104, 4 place 
      Jussieu, 75252 Paris cedex 05, France.
LA  - eng
SI  - Dryad/10.5061/dryad.mj2j4v2
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biometry
MH  - Classification/*methods
MH  - Eutheria/*anatomy &amp; histology/*classification
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Tooth/*anatomy &amp; histology
EDAT- 2018/10/27 06:00
MHDA- 2019/04/09 06:00
CRDT- 2018/10/27 06:00
PHST- 2018/03/06 00:00 [received]
PHST- 2018/10/22 00:00 [accepted]
PHST- 2018/10/27 06:00 [pubmed]
PHST- 2019/04/09 06:00 [medline]
PHST- 2018/10/27 06:00 [entrez]
AID - 5145070 [pii]
AID - 10.1093/sysbio/syy071 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):267-280. doi: 10.1093/sysbio/syy071.

PMID- 30364986
OWN - NLM
STAT- MEDLINE
DCOM- 20190430
LR  - 20190430
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 3
DP  - 2019 May 1
TI  - Sex-Biased Dispersal Obscures Species Boundaries in Integrative Species
      Delimitation Approaches.
PG  - 441-459
LID - 10.1093/sysbio/syy072 [doi]
AB  - Accurate delimitation of species is crucial for a stable taxonomy, which provides
      the foundation for the study of evolutionary biology, ecology, and essentially
      all biological disciplines. Several approaches toward impartial and repeatable
      taxonomic practices are available but all existing methods have potentially
      unacceptable shortcomings. In particular, problems can arise when the underlying 
      model assumptions are violated, for instance, in the presence of reduced gene
      flow. This is observed in the context of sex-biased dispersal, which is a common 
      but underappreciated feature in many groups of organisms. Previously, simulations
      have indicated that sex-biased dispersal may lead to erroneous estimations of the
      true species numbers. However, this phenomenon has never been examined using
      empirical data. We evaluate the bias introduced by extreme female philopatry on a
      range of de novo [GMYC, PTP, ABGD, statistical parsimony, trinomial distribution 
      of triplets model (tr2)] and validation (STACEY, iBPP) approaches to species
      delimitation in the scarab beetle genus Pachypus. Since female philopatry
      exhibited in this genus in particular can affect mitochondrial gene flow, we
      compared the results from analyses of single loci, mitochondrial loci, nuclear
      loci and combined data, as well as the performance of morphometric data as a
      secondary data source in a fully integrative Bayesian framework. Large
      overestimation of species numbers was observed across all analyses of combined
      and mitochondrial DNA data sets, suggesting specimens from nearly every sampling 
      location as separate species. The use of nuclear data resulted in more reasonable
      estimations of species boundaries, which were largely supported by morphometrics 
      of linear measurements, while geometric morphometrics of body outlines resulted
      in stronger splitting. Simulations of population divergence with migration,
      corresponding to the biology of Pachypus, showed that female philopatry strongly 
      increases reciprocal monophyly of mitochondrial markers and may substantially
      contribute to over-splitting in species delimitation. Robust results recovered
      using nuclear DNA and morphological data nevertheless enabled us to reach novel
      conclusions about species boundaries in Pachypus. Our findings suggest that
      mitochondrial DNA will be less suited to species delimitation in many cases, in
      particular in the presence of sex-biased dispersal.
CI  - (c) The Author(s) 2018. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Eberle, Jonas
AU  - Eberle J
AD  - Zoological Research Museum Alexander Koenig, Centre of Taxonomy and Evolutionary 
      Research, Adenauerallee 160, 53113 Bonn, Germany.
FAU - Bazzato, Erika
AU  - Bazzato E
AD  - Zoological Research Museum Alexander Koenig, Centre of Taxonomy and Evolutionary 
      Research, Adenauerallee 160, 53113 Bonn, Germany.
AD  - Department of Life and Environmental Sciences, Botany Division, University of
      Cagliari, viale Sant'Ignazio da Laconi 13, 09123, Cagliari (CA), Italy.
FAU - Fabrizi, Silvia
AU  - Fabrizi S
AD  - Zoological Research Museum Alexander Koenig, Centre of Taxonomy and Evolutionary 
      Research, Adenauerallee 160, 53113 Bonn, Germany.
FAU - Rossini, Michele
AU  - Rossini M
AD  - Zoological Research Museum Alexander Koenig, Centre of Taxonomy and Evolutionary 
      Research, Adenauerallee 160, 53113 Bonn, Germany.
AD  - Strada dei Guazzi, 1/3, 61122 Pesaro (PU), Italy.
FAU - Colomba, Mariastella
AU  - Colomba M
AD  - Universita di Urbino, Dipartimento di Scienze Biomolecolari (DiSB), Via maggetti 
      22, 61029 Urbino (PU), Italy.
FAU - Cillo, Davide
AU  - Cillo D
AD  - Via Zeffiro 8, 09130 Cagliari, Italy.
FAU - Uliana, Marco
AU  - Uliana M
AD  - Museo di Storia Naturale, Santa Croce 1730, 30135 Venezia, Italy.
FAU - Sparacio, Ignazio
AU  - Sparacio I
AD  - via Principe di Paterno 3, 90143 Palermo, Italy.
FAU - Sabatinelli, Guido
AU  - Sabatinelli G
AD  - Museum d'Histoire Naturelle, Route de Malagnou 1, 1208 Geneva, Switzerland.
FAU - Warnock, Rachel C M
AU  - Warnock RCM
AD  - ETH Zurich, D-BSSE, Mattenstrasse 26, 4058 Basel, Switzerland.
FAU - Carpaneto, Giuseppe
AU  - Carpaneto G
AD  - Dipartimento di Scienze, Universita Roma Tre, Viale Marconi 446, 00146 Roma,
      Italy.
FAU - Ahrens, Dirk
AU  - Ahrens D
AD  - Zoological Research Museum Alexander Koenig, Centre of Taxonomy and Evolutionary 
      Research, Adenauerallee 160, 53113 Bonn, Germany.
LA  - eng
SI  - Dryad/10.5061/dryad.g97d432
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Animal Distribution
MH  - Animals
MH  - Coleoptera/classification/*physiology
MH  - Computer Simulation
MH  - Female
MH  - Gene Flow
MH  - Genes, Mitochondrial/genetics
MH  - Male
MH  - Sex Factors
OTO - NOTNLM
OT  - *Deep mitochondrial coalescence
OT  - *integrative species delimitation
OT  - *mitochondrial DNA
OT  - *morphometrics
OT  - *multispecies coalescent
OT  - *nuclear DNA
OT  - *sex-biased dispersal
EDAT- 2018/10/27 06:00
MHDA- 2019/05/01 06:00
CRDT- 2018/10/27 06:00
PHST- 2017/10/25 00:00 [received]
PHST- 2018/10/22 00:00 [revised]
PHST- 2018/10/23 00:00 [accepted]
PHST- 2018/10/27 06:00 [pubmed]
PHST- 2019/05/01 06:00 [medline]
PHST- 2018/10/27 06:00 [entrez]
AID - 5145072 [pii]
AID - 10.1093/sysbio/syy072 [doi]
PST - ppublish
SO  - Syst Biol. 2019 May 1;68(3):441-459. doi: 10.1093/sysbio/syy072.

PMID- 30339252
OWN - NLM
STAT- MEDLINE
DCOM- 20181211
LR  - 20181211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 6
DP  - 2018 Nov 1
TI  - Phylogenomic Analysis of the Explosive Adaptive Radiation of the Espeletia
      Complex (Asteraceae) in the Tropical Andes.
PG  - 1041-1060
LID - 10.1093/sysbio/syy022 [doi]
AB  - The subtribe Espeletiinae (Asteraceae), endemic to the high-elevations in the
      Northern Andes, exhibits an exceptional diversity of species, growth-forms, and
      reproductive strategies. This complex of 140 species includes large trees,
      dichotomous trees, shrubs and the extraordinary giant caulescent rosettes,
      considered as a classic example of adaptation in tropical high-elevation
      ecosystems. The subtribe has also long been recognized as a prominent case of
      adaptive radiation, but the understanding of its evolution has been hampered by a
      lack of phylogenetic resolution. Herein, we produce the first fully resolved
      phylogeny of all morphological groups of Espeletiinae, using whole plastomes and 
      about a million nuclear nucleotides obtained with an original de novo assembly
      procedure without reference genome, and analyzed with traditional and
      coalescent-based approaches that consider the possible impact of incomplete
      lineage sorting and hybridization on phylogenetic inference. We show that the
      diversification of Espeletiinae started from a rosette ancestor about 2.3 Ma,
      after the final uplift of the Northern Andes. This was followed by two
      independent radiations in the Colombian and Venezuelan Andes, with a few
      trans-cordilleran dispersal events among low-elevation tree lineages but none
      among high-elevation rosettes. We demonstrate complex scenarios of morphological 
      change in Espeletiinae, usually implying the convergent evolution of growth-forms
      with frequent loss/gains of various traits. For instance, caulescent rosettes
      evolved independently in both countries, likely as convergent adaptations to life
      in tropical high-elevation habitats. Tree growth-forms evolved independently
      three times from the repeated colonization of lower elevations by high-elevation 
      rosette ancestors. The rate of morphological diversification increased during the
      early phase of the radiation, after which it decreased steadily towards the
      present. On the other hand, the rate of species diversification in the
      best-sampled Venezuelan radiation was on average very high (3.1 spp/My), with
      significant rate variation among growth-forms (much higher in polycarpic
      caulescent rosettes). Our results point out a scenario where both adaptive
      morphological evolution and geographical isolation due to Pleistocene climatic
      oscillations triggered an exceptionally rapid radiation for a continental plant
      group.
FAU - Pouchon, Charles
AU  - Pouchon C
AD  - Laboratoire d'Ecologie Alpine, UMR 5553, Universite Grenoble Alpes-CNRS,
      Grenoble, France.
FAU - Fernandez, Angel
AU  - Fernandez A
AD  - Herbario IVIC, Centro de Biofisica y Bioquimica, Instituto Venezolano de
      Investigaciones Cientificas, Apartado 20632, Caracas 1020-A, Venezuela.
FAU - Nassar, Jafet M
AU  - Nassar JM
AD  - Laboratorio de Biologia de Organismos, Centro de Ecologia, Instituto Venezolano
      de Investigaciones Cientificas, Apartado 20632, Caracas 1020-A, Venezuela.
FAU - Boyer, Frederic
AU  - Boyer F
AD  - Laboratoire d'Ecologie Alpine, UMR 5553, Universite Grenoble Alpes-CNRS,
      Grenoble, France.
FAU - Aubert, Serge
AU  - Aubert S
AD  - Laboratoire d'Ecologie Alpine, UMR 5553, Universite Grenoble Alpes-CNRS,
      Grenoble, France.
AD  - Station alpine Joseph-Fourier, UMS 3370, Universite Grenoble Alpes-CNRS,
      Grenoble, France.
FAU - Lavergne, Sebastien
AU  - Lavergne S
AD  - Laboratoire d'Ecologie Alpine, UMR 5553, Universite Grenoble Alpes-CNRS,
      Grenoble, France.
FAU - Mavarez, Jesus
AU  - Mavarez J
AD  - Laboratoire d'Ecologie Alpine, UMR 5553, Universite Grenoble Alpes-CNRS,
      Grenoble, France.
LA  - eng
SI  - Dryad/10.5061/dryad.m1t32
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Physiological/genetics
MH  - Asteraceae/*classification/*genetics
MH  - Colombia
MH  - Genome, Plant/*genetics
MH  - *Phylogeny
MH  - Tropical Climate
MH  - Venezuela
EDAT- 2018/10/20 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/10/20 06:00
PHST- 2017/03/14 00:00 [received]
PHST- 2018/03/15 00:00 [accepted]
PHST- 2018/10/20 06:00 [entrez]
PHST- 2018/10/20 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
AID - 4948752 [pii]
AID - 10.1093/sysbio/syy022 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Nov 1;67(6):1041-1060. doi: 10.1093/sysbio/syy022.

PMID- 30339251
OWN - NLM
STAT- MEDLINE
DCOM- 20181211
LR  - 20181211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 6
DP  - 2018 Nov 1
TI  - Targeted Sampling and Target Capture: Assessing Phylogeographic Concordance with 
      Genome-wide Data.
PG  - 979-996
LID - 10.1093/sysbio/syy021 [doi]
AB  - Comparative phylogeography provides the necessary framework to examine the
      factors influencing population divergence, persistence, and change over time.
      Avise (2000) outlined four aspects of concordance that result when data exhibit
      significant phylogeographic signal: concordance among sites within a locus, among
      multiple loci within a species, among multiple species within a region, and
      between genetic patterns and established biogeographic provinces. To fully
      address each aspect of concordance, we combined target capture of a set of
      orthologous loci with targeted geographic sampling of multiple species, thus
      removing any variability introduced by using different genetic markers and
      heterogeneous sampling distributions. We used hybrid enrichment and
      high-throughput sequencing of four anuran species sampled from 36 congruent
      localities in the Southeastern United States Coastal Plain, a region that
      represents one of the classic systems in phylogeography. In total, we recovered &gt;
      375 of the same nuclear loci across species and assembled mitochondrial genomes, 
      resulting in one of the most comprehensive comparative phylogeographic datasets
      in any region or taxon to date. We used these data to evaluate concordance,
      compare genetic structure across species, and test previously described
      biogeographic features in the region including major river drainages and suture
      zones. We then applied a recently-developed framework to quantify concordance
      across species using phylogeographic concordance factors. For the four species
      examined, which have higher dispersal and potentially limited structure compared 
      to many amphibians, we found poor resolution in individual nuclear gene trees
      even with long (~ 1400 bp) nuclear sequences. The mitochondrial and multi-locus
      nuclear datasets, however, produced similar patterns within species, indicated
      high discordance among species, and suggested little correspondence of genetic
      patterns with putative biogeographic barriers. Variation in the phylogeographic
      structure detected may be related to differences in natural history, in that the 
      two habitat generalists exhibited less structure. Our study demonstrates the
      utility of combining target capture, which is highly repeatable and produces
      comparable datasets, with a targeted sampling strategy to quantify
      phylogeographic concordance across diverse taxa in a region with a complex
      history.
FAU - Barrow, Lisa N
AU  - Barrow LN
AD  - Department of Biology, Museum of Southwestern Biology, MSC03 2020, 1 University
      of New Mexico, Albuquerque, NM 87131-0001, USA.
AD  - Department of Biological Science, Florida State University, 319 Stadium Drive, PO
      Box 3064295, Tallahassee, FL 32306-4295, USA.
FAU - Lemmon, Alan R
AU  - Lemmon AR
AD  - Department of Scientific Computing, Florida State University, Dirac Science
      Library, Tallahassee, FL 32306-4120, USA.
FAU - Lemmon, Emily Moriarty
AU  - Lemmon EM
AD  - Department of Biological Science, Florida State University, 319 Stadium Drive, PO
      Box 3064295, Tallahassee, FL 32306-4295, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.1qt0k
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amphibians/*classification/*genetics
MH  - Animals
MH  - Genome-Wide Association Study/*standards
MH  - Phylogeography/*standards
MH  - Southwestern United States
MH  - Species Specificity
EDAT- 2018/10/20 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/10/20 06:00
PHST- 2017/04/20 00:00 [received]
PHST- 2018/03/15 00:00 [accepted]
PHST- 2018/10/20 06:00 [entrez]
PHST- 2018/10/20 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
AID - 4948751 [pii]
AID - 10.1093/sysbio/syy021 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Nov 1;67(6):979-996. doi: 10.1093/sysbio/syy021.

PMID- 30329135
OWN - NLM
STAT- MEDLINE
DCOM- 20190430
LR  - 20190430
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 3
DP  - 2019 May 1
TI  - Evaluating Statistical Multiple Sequence Alignment in Comparison to Other
      Alignment Methods on Protein Data Sets.
PG  - 396-411
LID - 10.1093/sysbio/syy068 [doi]
AB  - The estimation of multiple sequence alignments of protein sequences is a basic
      step in many bioinformatics pipelines, including protein structure prediction,
      protein family identification, and phylogeny estimation. Statistical coestimation
      of alignments and trees under stochastic models of sequence evolution has long
      been considered the most rigorous technique for estimating alignments and trees, 
      but little is known about the accuracy of such methods on biological benchmarks. 
      We report the results of an extensive study evaluating the most popular protein
      alignment methods as well as the statistical coestimation method BAli-Phy on 1192
      protein data sets from established benchmarks as well as on 120 simulated data
      sets. Our study (which used more than 230 CPU years for the BAli-Phy analyses
      alone) shows that BAli-Phy has better precision and recall (with respect to the
      true alignments) than the other alignment methods on the simulated data sets but 
      has consistently lower recall on the biological benchmarks (with respect to the
      reference alignments) than many of the other methods. In other words, we find
      that BAli-Phy systematically underaligns when operating on biological sequence
      data but shows no sign of this on simulated data. There are several potential
      causes for this change in performance, including model misspecification, errors
      in the reference alignments, and conflicts between structural alignment and
      evolutionary alignments, and future research is needed to determine the most
      likely explanation. We conclude with a discussion of the potential ramifications 
      for each of these possibilities. [BAli-Phy; homology; multiple sequence
      alignment; protein sequences; structural alignment.].
CI  - (c) The Author(s) 2018. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Nute, Michael
AU  - Nute M
AD  - Department of Statistics, University of Illinois at Urbana-Champaign, 725 S
      Wright St #101, Champaign, IL 61820, USA.
FAU - Saleh, Ehsan
AU  - Saleh E
AD  - Department of Computer Science, University of Illinois at Urbana-Champaign, 201
      N. Goodwin Ave, Urbana, IL 61801, USA.
FAU - Warnow, Tandy
AU  - Warnow T
AD  - Department of Computer Science, University of Illinois at Urbana-Champaign, 201
      N. Goodwin Ave, Urbana, IL 61801, USA.
AD  - Carl R. Woese Institute for Genomic Biology, University of Illinois at
      Urbana-Champaign, 1205 W. Clark St., Urbana, IL 61801, USA.
AD  - National Center for Supercomputing Applications, University of Illinois at
      Urbana-Champaign, Urbana, IL 61801, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.8k821ds
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Databases, Protein
MH  - Datasets as Topic
MH  - *Models, Statistical
MH  - Sequence Alignment/*standards
PMC - PMC6472439
EDAT- 2018/10/18 06:00
MHDA- 2019/05/01 06:00
CRDT- 2018/10/18 06:00
PHST- 2018/04/18 00:00 [received]
PHST- 2018/09/27 00:00 [revised]
PHST- 2018/10/11 00:00 [accepted]
PHST- 2018/10/18 06:00 [pubmed]
PHST- 2019/05/01 06:00 [medline]
PHST- 2018/10/18 06:00 [entrez]
AID - 5133545 [pii]
AID - 10.1093/sysbio/syy068 [doi]
PST - ppublish
SO  - Syst Biol. 2019 May 1;68(3):396-411. doi: 10.1093/sysbio/syy068.

PMID- 30329125
OWN - NLM
STAT- MEDLINE
DCOM- 20181220
LR  - 20181220
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 1
DP  - 2019 Jan 1
TI  - How Well Does Your Phylogenetic Model Fit Your Data?
PG  - 157-167
LID - 10.1093/sysbio/syy066 [doi]
AB  - The test for model-to-data fitness is a fundamental principle within the
      statistical sciences. The purpose of such a test is to assess whether the
      selected best-fitting model adequately describes the behavior in the data.
      Despite their broad application across many areas of statistics, goodness of fit 
      tests for phylogenetic models have received much less attention than model
      selection methods in the last decade. At present a number of approaches have been
      suggested. However, these are often flawed, with problems ranging from the
      presence of systematic error in the models themselves to the difficulties
      presented by the nature of phylogenetic data. Ultimately these problems lead to
      an inadequate choice of statistic. This is one of the main reasons why goodness
      of fit assessment is often a neglected step within phylogenetic analysis. We
      argue not only for the necessity of these goodness of fit measures to test how
      well the model reflects the data, but additionally for the need for "useful"
      tests that explain why the model-to-data fit may be inadequate. Such tests are a 
      critical part of the model building process, allowing the model to be adapted to 
      provide a better model-to-data fit or to reject a model class outright due to
      such an inadequate fit that the intended use of the class may be compromised.
      Proposed and existing methods in both the maximum likelihood and Bayesian
      framework will be discussed here, whilst highlighting their strengths and
      limitations for assessing goodness of fit. The final section discusses some
      critical open statistical problems in goodness of fit assessment for this field, 
      with the hope of encouraging more research into such a fundamental yet
      underdeveloped area of phylogenetic inference. [Bayesian phylogenetics; Goodness 
      of fit; maximum likelihood; molecular phylogenetics; outlier detection; residual 
      diagnostics.].
FAU - A Shepherd, Daisy
AU  - A Shepherd D
AD  - Department of Statistics, The University of Auckland, Auckland, New Zealand.
FAU - Klaere, Steffen
AU  - Klaere S
AD  - Department of Statistics, The University of Auckland, Auckland, New Zealand.
AD  - School of Biological Sciences, The University of Auckland, Auckland, New Zealand.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2018/10/18 06:00
MHDA- 2018/12/21 06:00
CRDT- 2018/10/18 06:00
PHST- 2016/12/11 00:00 [received]
PHST- 2018/10/11 00:00 [accepted]
PHST- 2018/10/18 06:00 [pubmed]
PHST- 2018/12/21 06:00 [medline]
PHST- 2018/10/18 06:00 [entrez]
AID - 5133546 [pii]
AID - 10.1093/sysbio/syy066 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jan 1;68(1):157-167. doi: 10.1093/sysbio/syy066.

PMID- 30329124
OWN - NLM
STAT- MEDLINE
DCOM- 20190430
LR  - 20190430
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 3
DP  - 2019 May 1
TI  - Estimating Correlated Rates of Trait Evolution with Uncertainty.
PG  - 412-429
LID - 10.1093/sysbio/syy067 [doi]
AB  - Correlated evolution among traits, which can happen due to genetic constraints,
      ontogeny, and selection, can have an important impact on the trajectory of
      phenotypic evolution. For example, shifts in the pattern of evolutionary
      integration may allow the exploration of novel regions of the morphospace by
      lineages. Here, we use phylogenetic trees to study the pace of evolution of
      several traits and their pattern of evolutionary correlation across clades and
      over time. We use regimes mapped to the branches of the phylogeny to test for
      shifts in evolutionary integration while incorporating the uncertainty related to
      trait evolution and ancestral regimes with joint estimation of all parameters of 
      the model using Bayesian Markov chain Monte Carlo. We implemented the use of
      summary statistics to test for regime shifts based on a series of attributes of
      the model that can be directly relevant to biological hypotheses. In addition, we
      extend Felsenstein's pruning algorithm to the case of multivariate Brownian
      motion models with multiple rate regimes. We performed extensive simulations to
      explore the performance of the method under a series of scenarios. Finally, we
      provide two test cases; the evolution of a novel buccal morphology in fishes of
      the family Centrarchidae and a shift in the trajectory of evolution of traits
      during the radiation of anole lizards to and from the Caribbean islands. [Anolis;
      Centrarchidae; comparative methods; evolutionary integration; evolutionary rates;
      modularity; pruning algorithm.].
CI  - (c) The Author(s) 2018. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      email: journals.permissions@oup.com.
FAU - Caetano, D S
AU  - Caetano DS
AD  - Department of Biological Sciences, Institute for Bioinformatics and Evolutionary 
      Studies (IBEST), University of Idaho, Moscow, ID 83843, USA.
AD  - Department of Biological Sciences, University of Arkansas, Fayetteville, AR
      72701, USA.
FAU - Harmon, L J
AU  - Harmon LJ
AD  - Department of Biological Sciences, Institute for Bioinformatics and Evolutionary 
      Studies (IBEST), University of Idaho, Moscow, ID 83843, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.dt732vc
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Uncertainty
EDAT- 2018/10/18 06:00
MHDA- 2019/05/01 06:00
CRDT- 2018/10/18 06:00
PHST- 2017/01/26 00:00 [received]
PHST- 2018/10/01 00:00 [revised]
PHST- 2018/10/11 00:00 [accepted]
PHST- 2018/10/18 06:00 [pubmed]
PHST- 2019/05/01 06:00 [medline]
PHST- 2018/10/18 06:00 [entrez]
AID - 5133548 [pii]
AID - 10.1093/sysbio/syy067 [doi]
PST - ppublish
SO  - Syst Biol. 2019 May 1;68(3):412-429. doi: 10.1093/sysbio/syy067.

PMID- 30295908
OWN - NLM
STAT- MEDLINE
DCOM- 20181211
LR  - 20181211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 6
DP  - 2018 Nov 1
TI  - Generalized Bootstrap Supports for Phylogenetic Analyses of Protein Sequences
      Incorporating Alignment Uncertainty.
PG  - 997-1009
LID - 10.1093/sysbio/syx096 [doi]
AB  - Phylogenetic reconstructions are essential in genomics data analyses and depend
      on accurate multiple sequence alignment (MSA) models. We show that all currently 
      available large-scale progressive multiple alignment methods are numerically
      unstable when dealing with amino-acid sequences. They produce significantly
      different output when changing sequence input order. We used the HOMFAM protein
      sequences dataset to show that on datasets larger than 100 sequences, this
      instability affects on average 21.5% of the aligned residues. The resulting
      Maximum Likelihood (ML) trees estimated from these MSAs are equally unstable with
      over 38% of the branches being sensitive to the sequence input order. We
      established that about two-thirds of this uncertainty stems from the unordered
      nature of children nodes within the guide trees used to estimate MSAs. To
      quantify this uncertainty we developed unistrap, a novel approach that estimates 
      the combined effect of alignment uncertainty and site sampling on phylogenetic
      tree branch supports. Compared with the regular bootstrap procedure, unistrap
      provides branch support estimates that take into account a larger fraction of the
      parameters impacting tree instability when processing datasets containing a large
      number of sequences.
FAU - Chatzou, Maria
AU  - Chatzou M
AD  - Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and
      Technology, Dr. Aiguader 88, Barcelona 08003, Spain.
AD  - Universitat Pompeu Fabra (UPF), Dr. Aiguader 88, Barcelona 08003, Spain.
FAU - Floden, Evan W
AU  - Floden EW
AD  - Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and
      Technology, Dr. Aiguader 88, Barcelona 08003, Spain.
AD  - Universitat Pompeu Fabra (UPF), Dr. Aiguader 88, Barcelona 08003, Spain.
FAU - Di Tommaso, Paolo
AU  - Di Tommaso P
AD  - Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and
      Technology, Dr. Aiguader 88, Barcelona 08003, Spain.
AD  - Universitat Pompeu Fabra (UPF), Dr. Aiguader 88, Barcelona 08003, Spain.
FAU - Gascuel, Olivier
AU  - Gascuel O
AD  - Unite Bioinformatique Evolutive, C3BI USR 3756 CNRS &amp; Institut Pasteur, 25-28,
      rue du Docteur-Roux, 75724 Paris Cedex 15, France.
AD  - Methodes et Algorithmes pour la Bioinformatique, IBC - LIRMM UMR5506, CNRS &amp;
      Universite de Montpellier, CC477, 161 rue Ada, 34095 Montpellier Cedex 5, France.
FAU - Notredame, Cedric
AU  - Notredame C
AD  - Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and
      Technology, Dr. Aiguader 88, Barcelona 08003, Spain.
AD  - Universitat Pompeu Fabra (UPF), Dr. Aiguader 88, Barcelona 08003, Spain.
LA  - eng
SI  - Dryad/10.5061/dryad.bk203
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Proteins)
SB  - IM
MH  - Classification/*methods
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Proteins/chemistry/*genetics
MH  - Sequence Alignment
MH  - Software
MH  - Uncertainty
EDAT- 2018/10/09 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/10/09 06:00
PHST- 2016/08/16 00:00 [received]
PHST- 2017/12/17 00:00 [accepted]
PHST- 2018/10/09 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
PHST- 2018/10/09 06:00 [entrez]
AID - 4948750 [pii]
AID - 10.1093/sysbio/syx096 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Nov 1;67(6):997-1009. doi: 10.1093/sysbio/syx096.

PMID- 30257005
OWN - NLM
STAT- MEDLINE
DCOM- 20190408
LR  - 20190408
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - Detecting the Dependence of Diversification on Multiple Traits from Phylogenetic 
      Trees and Trait Data.
PG  - 317-328
LID - 10.1093/sysbio/syy057 [doi]
AB  - Species diversification may be determined by many different variables, including 
      the traits of the diversifying lineages. The state-dependent speciation and
      extinction (SSE) framework contains methods to detect the dependence of
      diversification on these traits. For the analysis of traits with multiple states,
      MuSSE (multiple-states dependent speciation and extinction) was developed.
      However, MuSSE and other SSE models have been shown to yield false positives,
      because they cannot separate differential diversification rates from dependence
      of diversification on the observed traits. The recently introduced method HiSSE
      (hidden-state-dependent speciation and extinction) resolves this problem by
      allowing a hidden state to affect diversification rates. Unfortunately, HiSSE
      does not allow traits with more than two states, and, perhaps more interestingly,
      the simultaneous action of multiple traits on diversification. Herein, we
      introduce an R package (SecSSE: several examined and concealed states-dependent
      speciation and extinction) that combines the features of HiSSE and MuSSE to
      simultaneously infer state-dependent diversification across two or more examined 
      (observed) traits or states while accounting for the role of a possible concealed
      (hidden) trait. Moreover, SecSSE also has improved functionality when compared
      with its two "parents." First, it allows for an observed trait being in two or
      more states simultaneously, which is useful for example when a taxon is a
      generalist or when the exact state is not precisely known. Second, it provides
      the correct likelihood when conditioned on nonextinction, which has been
      incorrectly implemented in HiSSE and other SSE models. To illustrate our method, 
      we apply SecSSE to seven previous studies that used MuSSE, and find that in five 
      out of seven cases, the conclusions drawn based on MuSSE were premature. We test 
      with simulations whether SecSSE sacrifices statistical power to avoid the high
      Type I error problem of MuSSE, but we find that this is not the case: for the
      majority of simulations where the observed traits affect diversification, SecSSE 
      detects this.
FAU - Herrera-Alsina, Leonel
AU  - Herrera-Alsina L
AD  - Groningen Institute for Evolutionary Life Sciences, University of Groningen, PO
      Box 11103, Groningen 9700 CC, The Netherlands.
FAU - van Els, Paul
AU  - van Els P
AD  - Groningen Institute for Evolutionary Life Sciences, University of Groningen, PO
      Box 11103, Groningen 9700 CC, The Netherlands.
FAU - Etienne, Rampal S
AU  - Etienne RS
AD  - Groningen Institute for Evolutionary Life Sciences, University of Groningen, PO
      Box 11103, Groningen 9700 CC, The Netherlands.
LA  - eng
SI  - Dryad/10.5061/dryad.qf3g0
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Genetic Speciation
MH  - Models, Biological
MH  - Phenotype
MH  - *Phylogeny
EDAT- 2018/09/27 06:00
MHDA- 2019/04/09 06:00
CRDT- 2018/09/27 06:00
PHST- 2017/12/01 00:00 [received]
PHST- 2018/09/08 00:00 [accepted]
PHST- 2018/09/27 06:00 [pubmed]
PHST- 2019/04/09 06:00 [medline]
PHST- 2018/09/27 06:00 [entrez]
AID - 5107025 [pii]
AID - 10.1093/sysbio/syy057 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):317-328. doi: 10.1093/sysbio/syy057.

PMID- 30247732
OWN - NLM
STAT- MEDLINE
DCOM- 20190408
LR  - 20190408
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - Long-Branch Attraction in Species Tree Estimation: Inconsistency of Partitioned
      Likelihood and Topology-Based Summary Methods.
PG  - 281-297
LID - 10.1093/sysbio/syy061 [doi]
AB  - With advances in sequencing technologies, there are now massive amounts of
      genomic data from across all life, leading to the possibility that a robust Tree 
      of Life can be constructed. However, "gene tree heterogeneity", which is when
      different genomic regions can evolve differently, is a common phenomenon in
      multi-locus data sets, and reduces the accuracy of standard methods for species
      tree estimation that do not take this heterogeneity into account. New methods
      have been developed for species tree estimation that specifically address gene
      tree heterogeneity, and that have been proven to converge to the true species
      tree when the number of loci and number of sites per locus both increase (i.e.,
      the methods are said to be "statistically consistent"). Yet, little is known
      about the biologically realistic condition where the number of sites per locus is
      bounded. We show that when the sequence length of each locus is bounded (by any
      arbitrarily chosen value), the most common approaches to species tree estimation 
      that take heterogeneity into account (i.e., traditional fully partitioned
      concatenated maximum likelihood and newer approaches, called summary methods,
      that estimate the species tree by combining estimated gene trees) are not
      statistically consistent, even when the heterogeneity is extremely constrained.
      The main challenge is the presence of conditions such as long branch attraction
      that create biased tree estimation when the number of sites is restricted. Hence,
      our study uncovers a fundamental challenge to species tree estimation using both 
      traditional and new methods.
FAU - Roch, Sebastien
AU  - Roch S
AD  - Department of Mathematics, University of Wisconsin-Madison, 480 Lincoln Dr,
      Madison, WI 53706, USA.
FAU - Nute, Michael
AU  - Nute M
AD  - Department of Statistics, The University of Illinois at Urbana-Champaign, 725 S
      Wright St #101, Champaign, IL 61820, USA.
FAU - Warnow, Tandy
AU  - Warnow T
AD  - Department of Computer Science, The University of Illinois at Urbana-Champaign,
      201 North Goodwin Avenue, Urbana, IL 61801-2302, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2018/09/25 06:00
MHDA- 2019/04/09 06:00
CRDT- 2018/09/25 06:00
PHST- 2018/03/16 00:00 [received]
PHST- 2018/09/12 00:00 [accepted]
PHST- 2018/09/25 06:00 [pubmed]
PHST- 2019/04/09 06:00 [medline]
PHST- 2018/09/25 06:00 [entrez]
AID - 5104882 [pii]
AID - 10.1093/sysbio/syy061 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):281-297. doi: 10.1093/sysbio/syy061.

PMID- 30239978
OWN - NLM
STAT- MEDLINE
DCOM- 20190430
LR  - 20190430
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 3
DP  - 2019 May 1
TI  - Mitochondrial Genome Fragmentation Unites the Parasitic Lice of Eutherian
      Mammals.
PG  - 430-440
LID - 10.1093/sysbio/syy062 [doi]
AB  - Organelle genome fragmentation has been found in a wide range of eukaryotic
      lineages; however, its use in phylogenetic reconstruction has not been
      demonstrated. We explored the use of mitochondrial (mt) genome fragmentation in
      resolving the controversial suborder-level phylogeny of parasitic lice (order
      Phthiraptera). There are approximately 5000 species of parasitic lice in four
      suborders (Amblycera, Ischnocera, Rhynchophthirina, and Anoplura), which infest
      mammals and birds. The phylogenetic relationships among these suborders are
      unresolved despite decades of studies. We sequenced the mt genomes of eight
      species of parasitic lice and compared them with 17 other species of parasitic
      lice sequenced previously. We found that the typical single-chromosome mt genome 
      is retained in the lice of birds but fragmented into many minichromosomes in the 
      lice of eutherian mammals. The shared derived feature of mt genome fragmentation 
      unites the eutherian mammal lice of Ischnocera (family Trichodectidae) with
      Anoplura and Rhynchophthirina to the exclusion of the bird lice of Ischnocera
      (family Philopteridae). The novel clade, namely Mitodivisia, is also supported by
      phylogenetic analysis of mt genome and cox1 gene sequences. Our results
      demonstrate, for the first time, that organelle genome fragmentation is
      informative for resolving controversial high-level phylogenies.
CI  - (c) The Author(s) 2018. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Song, Fan
AU  - Song F
AD  - Key Laboratory of Pest Monitoring and Green Management, Ministry of Agriculture, 
      Department of Entomology, College of Plant Protection, China Agricultural
      University, Beijing 100193, China.
FAU - Li, Hu
AU  - Li H
AD  - Key Laboratory of Pest Monitoring and Green Management, Ministry of Agriculture, 
      Department of Entomology, College of Plant Protection, China Agricultural
      University, Beijing 100193, China.
FAU - Liu, Guo-Hua
AU  - Liu GH
AD  - Hunan Provincial Key Laboratory of Protein Engineering in Animal Vaccines,
      College of Veterinary Medicine, Hunan Agricultural University, Changsha, Hunan
      Province 410128, China.
FAU - Wang, Wei
AU  - Wang W
AD  - School of Science and Engineering, GeneCology Research Centre, Animal Research
      Centre, University of the Sunshine Coast, Sippy Downs, Queensland 4556,
      Australia.
FAU - James, Peter
AU  - James P
AD  - Queensland Alliance for Agriculture and Food Innovation, University of
      Queensland, Dutton Park, Queensland, Australia.
FAU - Colwell, Douglas D
AU  - Colwell DD
AD  - Lethbridge Research and Development Centre, Agriculture and Agri-Food Canada,
      Lethbridge, Alberta, Canada.
FAU - Tran, Anette
AU  - Tran A
AD  - School of Science and Engineering, GeneCology Research Centre, Animal Research
      Centre, University of the Sunshine Coast, Sippy Downs, Queensland 4556,
      Australia.
FAU - Gong, Siyu
AU  - Gong S
AD  - Key Laboratory of Pest Monitoring and Green Management, Ministry of Agriculture, 
      Department of Entomology, College of Plant Protection, China Agricultural
      University, Beijing 100193, China.
FAU - Cai, Wanzhi
AU  - Cai W
AD  - Key Laboratory of Pest Monitoring and Green Management, Ministry of Agriculture, 
      Department of Entomology, College of Plant Protection, China Agricultural
      University, Beijing 100193, China.
FAU - Shao, Renfu
AU  - Shao R
AD  - School of Science and Engineering, GeneCology Research Centre, Animal Research
      Centre, University of the Sunshine Coast, Sippy Downs, Queensland 4556,
      Australia.
LA  - eng
SI  - Dryad/10.5061/dryad.qg8vf45
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Anoplura/*classification/*genetics
MH  - DNA Fragmentation
MH  - Eutheria/*parasitology
MH  - Genome, Mitochondrial/*genetics
MH  - *Phylogeny
PMC - PMC6472445
OTO - NOTNLM
OT  - *Genome fragmentation
OT  - *mitochondrial genome
OT  - *parasitic lice
OT  - *phylogeny
EDAT- 2018/09/22 06:00
MHDA- 2019/05/01 06:00
CRDT- 2018/09/22 06:00
PHST- 2018/03/28 00:00 [received]
PHST- 2018/09/11 00:00 [revised]
PHST- 2018/09/12 00:00 [accepted]
PHST- 2018/09/22 06:00 [pubmed]
PHST- 2019/05/01 06:00 [medline]
PHST- 2018/09/22 06:00 [entrez]
AID - 5098615 [pii]
AID - 10.1093/sysbio/syy062 [doi]
PST - ppublish
SO  - Syst Biol. 2019 May 1;68(3):430-440. doi: 10.1093/sysbio/syy062.

PMID- 30239975
OWN - NLM
STAT- MEDLINE
DCOM- 20190408
LR  - 20190408
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - R$^{2}$s for Correlated Data: Phylogenetic Models, LMMs, and GLMMs.
PG  - 234-251
LID - 10.1093/sysbio/syy060 [doi]
AB  - Many researchers want to report an $R^{2}$ to measure the variance explained by a
      model. When the model includes correlation among data, such as phylogenetic
      models and mixed models, defining an $R^{2}$ faces two conceptual problems. (i)
      It is unclear how to measure the variance explained by predictor (independent)
      variables when the model contains covariances. (ii) Researchers may want the
      $R^{2}$ to include the variance explained by the covariances by asking questions 
      such as "How much of the data is explained by phylogeny?" Here, I investigated
      three $R^{2}$s for phylogenetic and mixed models. $R^{2}_{resid}$ is an extension
      of the ordinary least-squares $R^{2}$ that weights residuals by variances and
      covariances estimated by the model; it is closely related to $R^{2}_{glmm}$
      presented by Nakagawa and Schielzeth (2013. A general and simple method for
      obtaining R2 from generalized linear mixed-effects models. Methods Ecol. Evol.
      4:133-142). $R^{2}_{pred}$ is based on predicting each residual from the fitted
      model and computing the variance between observed and predicted values.
      $R^{2}_{lik}$ is based on the likelihood of fitted models, and therefore,
      reflects the amount of information that the models contain. These three $R^{2}$s 
      are formulated as partial $R^{2}$s, making it possible to compare the
      contributions of predictor variables and variance components (phylogenetic signal
      and random effects) to the fit of models. Because partial $R^{2}$s compare a full
      model with a reduced model without components of the full model, they are
      distinct from marginal $R^{2}$s that partition additive components of the
      variance. I assessed the properties of the $R^{2}$s for phylogenetic models using
      simulations for continuous and binary response data (phylogenetic generalized
      least squares and phylogenetic logistic regression). Because the $R^{2}$s are
      designed broadly for any model for correlated data, I also compared $R^{2}$s for 
      linear mixed models and generalized linear mixed models. $R^{2}_{resid}$,
      $R^{2}_{pred}$, and $R^{2}_{lik}$ all have similar performance in describing the 
      variance explained by different components of models. However, $R^{2}_{pred}$
      gives the most direct answer to the question of how much variance in the data is 
      explained by a model. $R^{2}_{resid}$ is most appropriate for comparing models
      fit to different data sets, because it does not depend on sample sizes. And
      $R^{2}_{lik}$ is most appropriate to assess the importance of different
      components within the same model applied to the same data, because it is most
      closely associated with statistical significance tests.
FAU - Ives, Anthony R
AU  - Ives AR
AD  - Department of Integrative Biology, UW-Madison, Madison, WI 53706, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.345v6
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Models, Statistical
MH  - *Phylogeny
EDAT- 2018/09/22 06:00
MHDA- 2019/04/09 06:00
CRDT- 2018/09/22 06:00
PHST- 2017/08/14 00:00 [received]
PHST- 2018/09/12 00:00 [accepted]
PHST- 2018/09/22 06:00 [pubmed]
PHST- 2019/04/09 06:00 [medline]
PHST- 2018/09/22 06:00 [entrez]
AID - 5098616 [pii]
AID - 10.1093/sysbio/syy060 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):234-251. doi: 10.1093/sysbio/syy060.

PMID- 30239971
OWN - NLM
STAT- MEDLINE
DCOM- 20190329
LR  - 20190329
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - Phylogeny, Macroevolutionary Trends and Historical Biogeography of Sloths:
      Insights From a Bayesian Morphological Clock Analysis.
PG  - 204-218
LID - 10.1093/sysbio/syy058 [doi]
AB  - Sloths, like other xenarthrans, are an extremely interesting group of mammals
      that, after a long history of evolution and diversification in South America,
      became established on islands in the Caribbean and later reached North America
      during the Great American Biotic Interchange. In all three regions, they were
      part of the impressive Pleistocene megafauna. Most taxa became extinct and only
      two small, distantly related tree-dwelling genera survived. Here, we incorporate 
      several recently described genera of sloths into an assembled morphological data 
      supermatrix and apply Bayesian inference, using phylogenetic and morphological
      clock methods, to 64 sloth genera. Thus, we investigate the evolution of the
      group in terms of the timing of divergence of different lineages and their
      diversity, morphological disparity and biogeographical history. The phylogeny
      obtained supports the existence of the commonly recognized clades for the group. 
      Our results provide divergence time estimates for the major clades within
      Folivora that could not be dated with molecular methods. Lineage diversity shows 
      an early increase, reaching a peak in the Early Miocene followed by a major drop 
      at the end of the Santacrucian (Early Miocene). A second peak in the Late Miocene
      was also followed by a major drop at the end of the Huayquerian (Late Miocene).
      Both events show differential impact at the family level. After that, a slight
      Plio-Pleistocene decline was observed before the marked drop with the extinction 
      at the end of the Pleistocene. Phenotypic evolutionary rates were high during the
      early history of the clade, mainly associated with Mylodontidae, but rapidly
      decreased to lower values around 25 Ma, whereas Megalonychidae had lower rates at
      the beginning followed by a steady increase, peaking during the Late Miocene and 
      the Pliocene. Morphological disparity showed a similar trend, with an early
      increase, followed by a slowly increasing phase through the Late Oligocene and
      Early Miocene, and ending with another increase beginning at the middle of the
      Miocene. Biogeographic analysis showed southern South America as the most
      probable area of origin of the clade and the main region in which the early
      diversification events took place. Both Megatheriinae and Nothrotheriinae basal
      nodes were strongly correlated with Andean uplift events, whereas the early
      history of Mylodontidae is closely associated with southern South America and
      also shows an early occupation of the northern regions. Within Megalonychidae,
      our results show Choloepus as a descendant of an island dispersing ancestor and a
      probable re-ingression to South America by a clade that originated in Central or 
      North America.
FAU - Varela, Luciano
AU  - Varela L
AD  - Departamento de Paleontologia, Facultad de Ciencias, Universidad de la Republica,
      Igua 4225, 11400 Montevideo, Uruguay.
FAU - Tambusso, P Sebastian
AU  - Tambusso PS
AD  - Departamento de Paleontologia, Facultad de Ciencias, Universidad de la Republica,
      Igua 4225, 11400 Montevideo, Uruguay.
FAU - McDonald, H Gregory
AU  - McDonald HG
AD  - Bureau of Land Management, Utah State Office, 440 West 200 South, Salt Lake City,
      UT 84101 USA.
FAU - Farina, Richard A
AU  - Farina RA
AD  - Departamento de Paleontologia, Facultad de Ciencias, Universidad de la Republica,
      Igua 4225, 11400 Montevideo, Uruguay.
LA  - eng
SI  - Dryad/10.5061/dryad.v27k6d0
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Central America
MH  - *Phylogeny
MH  - Phylogeography
MH  - Sloths/*anatomy &amp; histology/*classification
MH  - South America
EDAT- 2018/09/22 06:00
MHDA- 2019/03/30 06:00
CRDT- 2018/09/22 06:00
PHST- 2017/11/03 00:00 [received]
PHST- 2018/09/10 00:00 [accepted]
PHST- 2018/09/22 06:00 [pubmed]
PHST- 2019/03/30 06:00 [medline]
PHST- 2018/09/22 06:00 [entrez]
AID - 5098296 [pii]
AID - 10.1093/sysbio/syy058 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):204-218. doi: 10.1093/sysbio/syy058.

PMID- 30239963
OWN - NLM
STAT- MEDLINE
DCOM- 20190408
LR  - 20190408
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - Impacts of Inference Method and Data set Filtering on Phylogenomic Resolution in 
      a Rapid Radiation of Ground Squirrels (Xerinae: Marmotini).
PG  - 298-316
LID - 10.1093/sysbio/syy064 [doi]
AB  - Phylogenomic data sets are illuminating many areas of the Tree of Life. However, 
      the large size of these data sets alone may be insufficient to resolve
      problematic nodes in the most rapid evolutionary radiations, because inferences
      in zones of extraordinarily low phylogenetic signal can be sensitive to the model
      and method of inference, as well as the information content of loci employed. We 
      used a data set of $&gt;$3950 ultraconserved element (UCE) loci from a classic
      mammalian radiation, ground-dwelling squirrels of the tribe Marmotini (Sciuridae:
      Xerinae), to assess sensitivity of phylogenetic estimates to varying per-locus
      information content across four different inference methods (RAxML, ASTRAL, NJst,
      and SVDquartets). Persistent discordance was found in topology and bootstrap
      support between concatenation- and coalescent-based inferences; among methods
      within the coalescent framework; and within all methods in response to different 
      filtering scenarios. Contrary to some recent empirical UCE-based studies,
      filtering by information content did not promote complete among-method
      concordance. Nevertheless, filtering did improve concordance relative to randomly
      selected locus sets, largely via improved consistency of two-step summary methods
      (particularly NJst) under conditions of higher average per-locus variation (and
      thus increasing gene tree precision). The benefits of phylogenomic data set
      filtering are variable among classes of inference methods and across different
      evolutionary scenarios, reiterating the complexities of resolving rapid
      radiations, even with robust taxon and character sampling.
FAU - Mclean, Bryan S
AU  - Mclean BS
AD  - Department of Biology and Museum of Southwestern Biology, 1 University of New
      Mexico, MSC03-2020, Albuquerque, NM 87131, USA.
AD  - Florida Museum of Natural History, University of Florida, 1659 Museum Road,
      Gainesville, FL 32611, USA.
FAU - Bell, Kayce C
AU  - Bell KC
AD  - Department of Biology and Museum of Southwestern Biology, 1 University of New
      Mexico, MSC03-2020, Albuquerque, NM 87131, USA.
AD  - Department of Invertebrate Zoology, Smithsonian Institution National Museum of
      Natural History, P.O. Box 37012, MRC 163, Washington, DC 20013-7012, USA.
FAU - Allen, Julie M
AU  - Allen JM
AD  - Department of Biology, University of Nevada, 1664 N. Virginia Street, Reno, NV
      89557, USA.
FAU - Helgen, Kristofer M
AU  - Helgen KM
AD  - Department of Ecology and Evolutionary Biology, School of Biological Sciences,
      University of Adelaide, North Terrace, Adelaide SA 5005, Australia.
FAU - Cook, Joseph A
AU  - Cook JA
AD  - Department of Biology and Museum of Southwestern Biology, 1 University of New
      Mexico, MSC03-2020, Albuquerque, NM 87131, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.t358vv1
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Genetic Speciation
MH  - Genome
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Sciuridae/*classification/*genetics
EDAT- 2018/09/22 06:00
MHDA- 2019/04/09 06:00
CRDT- 2018/09/22 06:00
PHST- 2017/02/17 00:00 [received]
PHST- 2018/09/12 00:00 [accepted]
PHST- 2018/09/22 06:00 [pubmed]
PHST- 2019/04/09 06:00 [medline]
PHST- 2018/09/22 06:00 [entrez]
AID - 5100879 [pii]
AID - 10.1093/sysbio/syy064 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):298-316. doi: 10.1093/sysbio/syy064.

PMID- 30239957
OWN - NLM
STAT- MEDLINE
DCOM- 20190408
LR  - 20190408
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - A Simulation-Based Approach to Statistical Alignment.
PG  - 252-266
LID - 10.1093/sysbio/syy059 [doi]
AB  - Classic alignment algorithms utilize scoring functions which maximize similarity 
      or minimize edit distances. These scoring functions account for both
      insertion-deletion (indel) and substitution events. In contrast, alignments based
      on stochastic models aim to explicitly describe the evolutionary dynamics of
      sequences by inferring relevant probabilistic parameters from input sequences.
      Despite advances in stochastic modeling during the last two decades,
      scoring-based methods are still dominant, partially due to slow running times of 
      probabilistic approaches. Alignment inference using stochastic models involves
      estimating the probability of events, such as the insertion or deletion of a
      specific number of characters. In this work, we present SimBa-SAl, a
      simulation-based approach to statistical alignment inference, which relies on an 
      explicit continuous time Markov model for both indels and substitutions.
      SimBa-SAl has several advantages. First, using simulations, it decouples the
      estimation of event probabilities from the inference stage, which allows the
      introduction of accelerations to the alignment inference procedure. Second, it is
      general and can accommodate various stochastic models of indel formation.
      Finally, it allows computing the maximum-likelihood alignment, the probability of
      a given pair of sequences integrated over all possible alignments, and sampling
      alternative alignments according to their probability. We first show that
      SimBa-SAl allows accurate estimation of parameters of the long-indel model
      previously developed by Miklos et al. (2004). We next show that SimBa-SAl is more
      accurate than previously developed pairwise alignment algorithms, when analyzing 
      simulated as well as empirical data sets. Finally, we study the goodness-of-fit
      of the long-indel and TKF91 models. We show that although the long-indel model
      fits the data sets better than TKF91, there is still room for improvement
      concerning the realistic modeling of evolutionary sequence dynamics.
FAU - Levy Karin, Eli
AU  - Levy Karin E
AD  - School of Molecular Cell Biology &amp; Biotechnology, George S. Wise Faculty of Life 
      Sciences, Tel-Aviv University, Tel-Aviv 69978, Israel.
FAU - Ashkenazy, Haim
AU  - Ashkenazy H
AD  - School of Molecular Cell Biology &amp; Biotechnology, George S. Wise Faculty of Life 
      Sciences, Tel-Aviv University, Tel-Aviv 69978, Israel.
FAU - Hein, Jotun
AU  - Hein J
AD  - School of Molecular Cell Biology &amp; Biotechnology, George S. Wise Faculty of Life 
      Sciences, Tel-Aviv University, Tel-Aviv 69978, Israel.
AD  - Department of Statistics, University of Oxford, Oxford, UK.
FAU - Pupko, Tal
AU  - Pupko T
AD  - School of Molecular Cell Biology &amp; Biotechnology, George S. Wise Faculty of Life 
      Sciences, Tel-Aviv University, Tel-Aviv 69978, Israel.
LA  - eng
SI  - Dryad/10.5061/dryad.p069231
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Evolution, Molecular
MH  - INDEL Mutation/genetics
MH  - *Models, Statistical
MH  - *Phylogeny
EDAT- 2018/09/22 06:00
MHDA- 2019/04/09 06:00
CRDT- 2018/09/22 06:00
PHST- 2018/02/04 00:00 [received]
PHST- 2018/09/10 00:00 [accepted]
PHST- 2018/09/22 06:00 [pubmed]
PHST- 2019/04/09 06:00 [medline]
PHST- 2018/09/22 06:00 [entrez]
AID - 5098294 [pii]
AID - 10.1093/sysbio/syy059 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):252-266. doi: 10.1093/sysbio/syy059.

PMID- 30239868
OWN - NLM
STAT- MEDLINE
DCOM- 20190430
LR  - 20190430
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 3
DP  - 2019 May 1
TI  - Full Bayesian Comparative Phylogeography from Genomic Data.
PG  - 371-395
LID - 10.1093/sysbio/syy063 [doi]
AB  - A challenge to understanding biological diversification is accounting for
      community-scale processes that cause multiple, co-distributed lineages to
      co-speciate. Such processes predict non-independent, temporally clustered
      divergences across taxa. Approximate-likelihood Bayesian computation (ABC)
      approaches to inferring such patterns from comparative genetic data are very
      sensitive to prior assumptions and often biased toward estimating shared
      divergences. We introduce a full-likelihood Bayesian approach, ecoevolity, which 
      takes full advantage of information in genomic data. By analytically integrating 
      over gene trees, we are able to directly calculate the likelihood of the
      population history from genomic data, and efficiently sample the model-averaged
      posterior via Markov chain Monte Carlo algorithms. Using simulations, we find
      that the new method is much more accurate and precise at estimating the number
      and timing of divergence events across pairs of populations than existing
      approximate-likelihood methods. Our full Bayesian approach also requires several 
      orders of magnitude less computational time than existing ABC approaches. We find
      that despite assuming unlinked characters (e.g., unlinked single-nucleotide
      polymorphisms), the new method performs better if this assumption is violated in 
      order to retain the constant characters of whole linked loci. In fact, retaining 
      constant characters allows the new method to robustly estimate the correct number
      of divergence events with high posterior probability in the face of
      character-acquisition biases, which commonly plague loci assembled from
      reduced-representation genomic libraries. We apply our method to genomic data
      from four pairs of insular populations of Gekko lizards from the Philippines that
      are not expected to have co-diverged. Despite all four pairs diverging very
      recently, our method strongly supports that they diverged independently, and
      these results are robust to very disparate prior assumptions.
CI  - (c) The Author(s) 2018. Published by Oxford University Press.
FAU - Oaks, Jamie R
AU  - Oaks JR
AD  - Department of Biological Sciences &amp; Museum of Natural History, Auburn University,
      101 Rouse Life Sciences Building, Auburn, AL 36849, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.4b3j2bj
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Genome/*genetics
MH  - Genomics
MH  - Lizards/classification/genetics
MH  - Philippines
MH  - Phylogeny
MH  - *Phylogeography
PMC - PMC6472446
OTO - NOTNLM
OT  - *Bayesian model choice
OT  - *Dirichlet process prior
OT  - *biogeography
OT  - *phylogeography
EDAT- 2018/09/22 06:00
MHDA- 2019/05/01 06:00
CRDT- 2018/09/22 06:00
PHST- 2018/06/08 00:00 [received]
PHST- 2018/08/30 00:00 [revised]
PHST- 2018/09/10 00:00 [accepted]
PHST- 2018/09/22 06:00 [pubmed]
PHST- 2019/05/01 06:00 [medline]
PHST- 2018/09/22 06:00 [entrez]
AID - 5098295 [pii]
AID - 10.1093/sysbio/syy063 [doi]
PST - ppublish
SO  - Syst Biol. 2019 May 1;68(3):371-395. doi: 10.1093/sysbio/syy063.

PMID- 30165689
OWN - NLM
STAT- MEDLINE
DCOM- 20190408
LR  - 20190408
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - EPA-ng: Massively Parallel Evolutionary Placement of Genetic Sequences.
PG  - 365-369
LID - 10.1093/sysbio/syy054 [doi]
AB  - Next generation sequencing (NGS) technologies have led to a ubiquity of molecular
      sequence data. This data avalanche is particularly challenging in metagenetics,
      which focuses on taxonomic identification of sequences obtained from diverse
      microbial environments. Phylogenetic placement methods determine how these
      sequences fit into an evolutionary context. Previous implementations of
      phylogenetic placement algorithms, such as the evolutionary placement algorithm
      (EPA) included in RAxML, or PPLACER, are being increasingly used for this
      purpose. However, due to the steady progress in NGS technologies, the current
      implementations face substantial scalability limitations. Herein, we present
      EPA-NG, a complete reimplementation of the EPA that is substantially faster,
      offers a distributed memory parallelization, and integrates concepts from both,
      RAxML-EPA and PPLACER. EPA-NG can be executed on standard shared memory, as well 
      as on distributed memory systems (e.g., computing clusters). To demonstrate the
      scalability of EPA-NG, we placed $1$ billion metagenetic reads from the Tara
      Oceans Project onto a reference tree with 3748 taxa in just under $7$ h, using
      2048 cores. Our performance assessment shows that EPA-NG outperforms RAxML-EPA
      and PPLACER by up to a factor of $30$ in sequential execution mode, while
      attaining comparable parallel efficiency on shared memory systems. We further
      show that the distributed memory parallelization of EPA-NG scales well up to 2048
      cores. EPA-NG is available under the AGPLv3 license:
      https://github.com/Pbdas/epa-ng.
FAU - Barbera, Pierre
AU  - Barbera P
AD  - Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, 69118
      Heidelberg, Germany.
FAU - Kozlov, Alexey M
AU  - Kozlov AM
AD  - Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, 69118
      Heidelberg, Germany.
FAU - Czech, Lucas
AU  - Czech L
AD  - Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, 69118
      Heidelberg, Germany.
FAU - Morel, Benoit
AU  - Morel B
AD  - Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, 69118
      Heidelberg, Germany.
FAU - Darriba, Diego
AU  - Darriba D
AD  - Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, 69118
      Heidelberg, Germany.
AD  - Department of Computer Engineering, University of A Coruna, 15071 A Coruna,
      Spain.
FAU - Flouri, Tomas
AU  - Flouri T
AD  - Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, 69118
      Heidelberg, Germany.
AD  - Department of Genetics, Evolution and Environment,University College London,
      Gower St., Bloomsbury, London WC1E 6BT, UK.
FAU - Stamatakis, Alexandros
AU  - Stamatakis A
AD  - Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, 69118
      Heidelberg, Germany.
AD  - Karlsruhe Institute of Technology, Department of Informatics, Institute of
      Theoretical Informatics, Postfach 6980, 76128 Karlsruhe, Germany.
LA  - eng
SI  - Dryad/10.5061/dryad.kb505nc
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - *Phylogeny
MH  - *Sequence Analysis, DNA
MH  - Software
PMC - PMC6368480
EDAT- 2018/08/31 06:00
MHDA- 2019/04/09 06:00
CRDT- 2018/08/31 06:00
PHST- 2018/03/30 00:00 [received]
PHST- 2018/08/21 00:00 [accepted]
PHST- 2018/08/31 06:00 [pubmed]
PHST- 2019/04/09 06:00 [medline]
PHST- 2018/08/31 06:00 [entrez]
AID - 5079844 [pii]
AID - 10.1093/sysbio/syy054 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):365-369. doi: 10.1093/sysbio/syy054.

PMID- 29982825
OWN - NLM
STAT- MEDLINE
DCOM- 20181220
LR  - 20181220
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 1
DP  - 2019 Jan 1
TI  - The Spectre of Too Many Species.
PG  - 168-181
LID - 10.1093/sysbio/syy051 [doi]
AB  - Recent simulation studies examining the performance of Bayesian species
      delimitation as implemented in the bpp program have suggested that bpp may detect
      population splits but not species divergences and that it tends to over-split
      when data of many loci are analyzed. Here, we confirm these results and provide
      the mathematical justifications. We point out that the distinction between
      population and species splits made in the protracted speciation model (PSM) has
      no influence on the generation of gene trees and sequence data, which explains
      why no method can use such data to distinguish between population splits and
      speciation. We suggest that the PSM is unrealistic as its mechanism for assigning
      species status assumes instantaneous speciation, contradicting prevailing
      taxonomic practice. We confirm the suggestion, based on simulation, that in the
      case of speciation with gene flow, Bayesian model selection as implemented in bpp
      tends to detect population splits when the amount of data (the number of loci)
      increases. We discuss the use of a recently proposed empirical genealogical
      divergence index (gdi) for species delimitation and illustrate that parameter
      estimates produced by a full likelihood analysis as implemented in bpp provide
      much more reliable inference under the gdi than the approximate method phrapl. We
      distinguish between Bayesian model selection and parameter estimation and suggest
      that the model selection approach is useful for identifying sympatric cryptic
      species, while the parameter estimation approach may be used to implement
      empirical criteria for determining species status among allopatric populations.
FAU - Leache, Adam D
AU  - Leache AD
AD  - Department of Biology &amp; Burke Museum of Natural History and Culture, University
      of Washington, Seattle, USA.
FAU - Zhu, Tianqi
AU  - Zhu T
AD  - National Center for Mathematics and Interdisciplinary Sciences, Academy of
      Mathematics and Systems Science, Chinese Academy of Sciences, China.
AD  - Key Laboratory of Random Complex Structures and Data Science, Academy of
      Mathematics and Systems Science, Chinese Academy of Sciences, China.
FAU - Rannala, Bruce
AU  - Rannala B
AD  - Department of Evolution and Ecology, University of California Davis, One Shields 
      Avenue, Davis, USA.
FAU - Yang, Ziheng
AU  - Yang Z
AD  - National Center for Mathematics and Interdisciplinary Sciences, Academy of
      Mathematics and Systems Science, Chinese Academy of Sciences, China.
AD  - Department of Genetics, University College London, London, UK.
AD  - Radcliffe Institute for Advanced Studies, Harvard University, Cambridge, USA.
LA  - eng
GR  - BB/P006493/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Genetic Speciation
MH  - *Models, Biological
PMC - PMC6292489
EDAT- 2018/07/10 06:00
MHDA- 2018/12/21 06:00
CRDT- 2018/07/09 06:00
PHST- 2017/11/07 00:00 [received]
PHST- 2018/06/29 00:00 [accepted]
PHST- 2018/07/10 06:00 [pubmed]
PHST- 2018/12/21 06:00 [medline]
PHST- 2018/07/09 06:00 [entrez]
AID - 5049398 [pii]
AID - 10.1093/sysbio/syy051 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jan 1;68(1):168-181. doi: 10.1093/sysbio/syy051.

PMID- 29961836
OWN - NLM
STAT- MEDLINE
DCOM- 20190408
LR  - 20190408
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - Model Selection and Parameter Inference in Phylogenetics Using Nested Sampling.
PG  - 219-233
LID - 10.1093/sysbio/syy050 [doi]
AB  - Bayesian inference methods rely on numerical algorithms for both model selection 
      and parameter inference. In general, these algorithms require a high
      computational effort to yield reliable estimates. One of the major challenges in 
      phylogenetics is the estimation of the marginal likelihood. This quantity is
      commonly used for comparing different evolutionary models, but its calculation,
      even for simple models, incurs high computational cost. Another interesting
      challenge relates to the estimation of the posterior distribution. Often, long
      Markov chains are required to get sufficient samples to carry out parameter
      inference, especially for tree distributions. In general, these problems are
      addressed separately by using different procedures. Nested sampling (NS) is a
      Bayesian computation algorithm, which provides the means to estimate marginal
      likelihoods together with their uncertainties, and to sample from the posterior
      distribution at no extra cost. The methods currently used in phylogenetics for
      marginal likelihood estimation lack in practicality due to their dependence on
      many tuning parameters and their inability of most implementations to provide a
      direct way to calculate the uncertainties associated with the estimates, unlike
      NS. In this article, we introduce NS to phylogenetics. Its performance is
      analysed under different scenarios and compared to established methods. We
      conclude that NS is a competitive and attractive algorithm for phylogenetic
      inference. An implementation is available as a package for BEAST 2 under the LGPL
      licence, accessible at https://github.com/BEAST2-Dev/nested-sampling.
FAU - Russel, Patricio Maturana
AU  - Russel PM
AD  - Department of Statistics, The University of Auckland, Auckland, New Zealand.
FAU - Brewer, Brendon J
AU  - Brewer BJ
AD  - Department of Statistics, The University of Auckland, Auckland, New Zealand.
FAU - Klaere, Steffen
AU  - Klaere S
AD  - Department of Statistics, The University of Auckland, Auckland, New Zealand.
AD  - School of Biological Sciences, University of Auckland, Auckland, New Zealand.
FAU - Bouckaert, Remco R
AU  - Bouckaert RR
AD  - Center of Computational Evolution, University of Auckland, Auckland, New Zealand.
AD  - Max Planck Institute for the Science of Human History, Jena, Germany.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2018/07/03 06:00
MHDA- 2019/04/09 06:00
CRDT- 2018/07/03 06:00
PHST- 2016/04/05 00:00 [received]
PHST- 2018/06/22 00:00 [accepted]
PHST- 2018/07/03 06:00 [pubmed]
PHST- 2019/04/09 06:00 [medline]
PHST- 2018/07/03 06:00 [entrez]
AID - 5046926 [pii]
AID - 10.1093/sysbio/syy050 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):219-233. doi: 10.1093/sysbio/syy050.

PMID- 29945220
OWN - NLM
STAT- MEDLINE
DCOM- 20190408
LR  - 20190408
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 2
DP  - 2019 Mar 1
TI  - Phylodynamic Model Adequacy Using Posterior Predictive Simulations.
PG  - 358-364
LID - 10.1093/sysbio/syy048 [doi]
AB  - Rapidly evolving pathogens, such as viruses and bacteria, accumulate genetic
      change at a similar timescale over which their epidemiological processes occur,
      such that, it is possible to make inferences about their infectious spread using 
      phylogenetic time-trees. For this purpose it is necessary to choose a
      phylodynamic model. However, the resulting inferences are contingent on whether
      the model adequately describes key features of the data. Model adequacy methods
      allow formal rejection of a model if it cannot generate the main features of the 
      data. We present TreeModelAdequacy, a package for the popular BEAST2 software
      that allows assessing the adequacy of phylodynamic models. We illustrate its
      utility by analyzing phylogenetic trees from two viral outbreaks of Ebola and
      H1N1 influenza. The main features of the Ebola data were adequately described by 
      the coalescent exponential-growth model, whereas the H1N1 influenza data were
      best described by the birth-death susceptible-infected-recovered model.
FAU - Duchene, Sebastian
AU  - Duchene S
AD  - Department of Biochemistry and Molecular Biology, Bio21 Molecular Science and
      Biotechnology Institute, University of Melbourne, Melbourne, Australia.
FAU - Bouckaert, Remco
AU  - Bouckaert R
AD  - Centre for Computational Evolution, University of Auckland, Auckland, New
      Zealand.
AD  - Max Planck Institute for the Science of Human History, Jena, Germany.
FAU - Duchene, David A
AU  - Duchene DA
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney,
      Australia.
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Basel, Switzerland.
AD  - Swiss Institute of Bioinformatics, Lausanne, Switzerland.
FAU - Drummond, Alexei J
AU  - Drummond AJ
AD  - Centre for Computational Evolution, University of Auckland, Auckland, New
      Zealand.
LA  - eng
SI  - Dryad/10.5061/dryad.65p331m
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Computer Simulation
MH  - Ebolavirus/*classification/*genetics
MH  - Genome, Viral/*genetics
MH  - Hemorrhagic Fever, Ebola/epidemiology/virology
MH  - Humans
MH  - Influenza A Virus, H1N1 Subtype/*classification/*genetics
MH  - Influenza, Human/epidemiology/virology
MH  - *Phylogeny
MH  - Software
PMC - PMC6368481
EDAT- 2018/06/27 06:00
MHDA- 2019/04/09 06:00
CRDT- 2018/06/27 06:00
PHST- 2018/02/25 00:00 [received]
PHST- 2018/06/15 00:00 [accepted]
PHST- 2018/06/27 06:00 [pubmed]
PHST- 2019/04/09 06:00 [medline]
PHST- 2018/06/27 06:00 [entrez]
AID - 5044211 [pii]
AID - 10.1093/sysbio/syy048 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Mar 1;68(2):358-364. doi: 10.1093/sysbio/syy048.

PMID- 29939352
OWN - NLM
STAT- MEDLINE
DCOM- 20181220
LR  - 20181220
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 1
DP  - 2019 Jan 1
TI  - Signatures of Microevolutionary Processes in Phylogenetic Patterns.
PG  - 131-144
LID - 10.1093/sysbio/syy049 [doi]
AB  - Phylogenetic trees are representations of evolutionary relationships among
      species and contain signatures of the processes responsible for the speciation
      events they display. Inferring processes from tree properties, however, is
      challenging. To address this problem, we analyzed a spatially-explicit model of
      speciation where genome size and mating range can be controlled. We simulated
      parapatric and sympatric (narrow and wide mating range, respectively) radiations 
      and constructed their phylogenetic trees, computing structural properties such as
      tree balance and speed of diversification. We showed that parapatric and
      sympatric speciation are well separated by these structural tree properties.
      Balanced trees with constant rates of diversification only originate in sympatry 
      and genome size affected both the balance and the speed of diversification of the
      simulated trees. Comparison with empirical data showed that most of the
      evolutionary radiations considered to have developed in parapatry or sympatry are
      in good agreement with model predictions. Even though additional forces other
      than spatial restriction of gene flow, genome size, and genetic
      incompatibilities, do play a role in the evolution of species formation, the
      microevolutionary processes modeled here capture signatures of the
      diversification pattern of evolutionary radiations, regarding the symmetry and
      speed of diversification of lineages.
FAU - Costa, Carolina L N
AU  - Costa CLN
AD  - Instituto de Fisica 'Gleb Wataghin', Universidade Estadual de Campinas, Unicamp, 
      13083-970 Campinas, SP, Brazil.
AD  - Instituto de Biologia, Universidade Estadual de Campinas, Unicamp, 13083-970
      Campinas, SP, Brazil.
FAU - Lemos-Costa, Paula
AU  - Lemos-Costa P
AD  - Instituto de Fisica 'Gleb Wataghin', Universidade Estadual de Campinas, Unicamp, 
      13083-970 Campinas, SP, Brazil.
AD  - Instituto de Biologia, Universidade Estadual de Campinas, Unicamp, 13083-970
      Campinas, SP, Brazil.
FAU - Marquitti, Flavia M D
AU  - Marquitti FMD
AD  - Instituto de Fisica 'Gleb Wataghin', Universidade Estadual de Campinas, Unicamp, 
      13083-970 Campinas, SP, Brazil.
FAU - Fernandes, Lucas D
AU  - Fernandes LD
AD  - Departamento de Entomologia e Acarologia, Escola Superior de Agricultura Luiz de 
      Queiroz, Universidade de Sao Paulo, 13418-900 Piracicaba, SP, Brazil.
FAU - Ramos, Marlon F
AU  - Ramos MF
AD  - Instituto de Fisica 'Gleb Wataghin', Universidade Estadual de Campinas, Unicamp, 
      13083-970 Campinas, SP, Brazil.
FAU - Schneider, David M
AU  - Schneider DM
AD  - Instituto de Fisica 'Gleb Wataghin', Universidade Estadual de Campinas, Unicamp, 
      13083-970 Campinas, SP, Brazil.
FAU - Martins, Ayana B
AU  - Martins AB
AD  - Instituto de Fisica 'Gleb Wataghin', Universidade Estadual de Campinas, Unicamp, 
      13083-970 Campinas, SP, Brazil.
AD  - Department of Fish Ecology &amp; Evolution, Centre of Ecology, Evolution and
      Biogeochemistry, Eawag Swiss Federal Institute of Aquatic Science and Technology,
      Kastanienbaum, Switzerland.
FAU - de Aguiar, Marcus A M
AU  - de Aguiar MAM
AD  - Instituto de Fisica 'Gleb Wataghin', Universidade Estadual de Campinas, Unicamp, 
      13083-970 Campinas, SP, Brazil.
LA  - eng
SI  - Dryad/10.5061/dryad.4kn5j5d
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Gene Flow
MH  - Genetic Speciation
MH  - Genome Size
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2018/06/26 06:00
MHDA- 2018/12/21 06:00
CRDT- 2018/06/26 06:00
PHST- 2017/12/19 00:00 [received]
PHST- 2018/06/13 00:00 [accepted]
PHST- 2018/06/26 06:00 [pubmed]
PHST- 2018/12/21 06:00 [medline]
PHST- 2018/06/26 06:00 [entrez]
AID - 5043295 [pii]
AID - 10.1093/sysbio/syy049 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jan 1;68(1):131-144. doi: 10.1093/sysbio/syy049.

PMID- 29939341
OWN - NLM
STAT- MEDLINE
DCOM- 20181220
LR  - 20181220
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 1
DP  - 2019 Jan 1
TI  - Optimal Rates for Phylogenetic Inference and Experimental Design in the Era of
      Genome-Scale Data Sets.
PG  - 145-156
LID - 10.1093/sysbio/syy047 [doi]
AB  - With the rise of genome-scale data sets, there has been a call for increased data
      scrutiny and careful selection of loci that are appropriate to use in an attempt 
      to resolve a phylogenetic problem. Such loci should maximize phylogenetic
      information content while minimizing the risk of homoplasy. Theory posits the
      existence of characters that evolve at an optimum rate, and efforts to determine 
      optimal rates of inference have been a cornerstone of phylogenetic experimental
      design for over two decades. However, both theoretical and empirical
      investigations of optimal rates have varied dramatically in their conclusions:
      spanning no relationship to a tight relationship between the rate of change and
      phylogenetic utility. Herein, we synthesize these apparently contradictory views,
      demonstrating both empirical and theoretical conditions under which each is
      correct. We find that optimal rates of characters-not genes-are generally robust 
      to most experimental design decisions. Moreover, consideration of site rate
      heterogeneity within a given locus is critical to accurate predictions of
      utility. Factors such as taxon sampling or the targeted number of characters
      providing support for a topology are additionally critical to the predictions of 
      phylogenetic utility based on the rate of character change. Further, optimality
      of rates and predictions of phylogenetic utility are not equivalent,
      demonstrating the need for further development of comprehensive theory of
      phylogenetic experimental design. [Divergence time; GC bias; homoplasy;
      incongruence; information content; internode length; optimal rates; phylogenetic 
      informativeness; phylogenetic theory; phylogenetic utility; phylogenomics; signal
      and noise; subtending branch length; state space; taxon and character sampling.].
FAU - Dornburg, Alex
AU  - Dornburg A
AD  - North Carolina Museum of Natural Sciences, Raleigh, 1671 Goldstar Drive, NC
      27601, USA.
FAU - Su, Zhuo
AU  - Su Z
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven, 165
      Prospect Street, CT 06525, USA.
FAU - Townsend, Jeffrey P
AU  - Townsend JP
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven, 165
      Prospect Street, CT 06525, USA.
AD  - Department of Biostatistics, Yale University, New Haven, 60 College Street, CT
      06510, USA.
AD  - Program in Computational Biology and Bioinformatics, Yale University, New Haven, 
      300 George Street, CT 06511, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Genome
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Research Design/*trends
EDAT- 2018/06/26 06:00
MHDA- 2018/12/21 06:00
CRDT- 2018/06/26 06:00
PHST- 2018/02/13 00:00 [received]
PHST- 2018/06/13 00:00 [accepted]
PHST- 2018/06/26 06:00 [pubmed]
PHST- 2018/12/21 06:00 [medline]
PHST- 2018/06/26 06:00 [entrez]
AID - 5043533 [pii]
AID - 10.1093/sysbio/syy047 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jan 1;68(1):145-156. doi: 10.1093/sysbio/syy047.

PMID- 29931325
OWN - NLM
STAT- MEDLINE
DCOM- 20181220
LR  - 20181220
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 1
DP  - 2019 Jan 1
TI  - Early Arrival and Climatically-Linked Geographic Expansion of New World Monkeys
      from Tiny African Ancestors.
PG  - 78-92
LID - 10.1093/sysbio/syy046 [doi]
AB  - New World Monkeys (NWM) (platyrrhines) are one of the most diverse groups of
      primates, occupying today a wide range of ecosystems in the American tropics and 
      exhibiting large variations in ecology, morphology, and behavior. Although the
      relationships among the almost 200 living species are relatively well understood,
      we lack robust estimates of the timing of origin, ancestral morphology, and
      geographic range evolution of the clade. Herein, we integrate paleontological and
      molecular evidence to assess the evolutionary dynamics of extinct and extant
      platyrrhines. We develop novel analytical frameworks to infer the evolution of
      body mass, changes in latitudinal ranges through time, and species
      diversification rates using a phylogenetic tree of living and fossil taxa. Our
      results show that platyrrhines originated 5-10 million years earlier than
      previously assumed, dating back to the Middle Eocene. The estimated ancestral
      platyrrhine was small-weighing 0.4 kg-and matched the size of their presumed
      African ancestors. As the three platyrrhine families diverged, we recover a rapid
      change in body mass range. During the Miocene Climatic Optimum, fossil diversity 
      peaked and platyrrhines reached their widest latitudinal range, expanding as far 
      South as Patagonia, favored by warm and humid climate and the lower elevation of 
      the Andes. Finally, global cooling and aridification after the middle Miocene
      triggered a geographic contraction of NWM and increased their extinction rates.
      These results unveil the full evolutionary trajectory of an iconic and
      ecologically important radiation of monkeys and showcase the necessity of
      integrating fossil and molecular data for reliably estimating evolutionary rates 
      and trends.
FAU - Silvestro, Daniele
AU  - Silvestro D
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Carl Skottsbergs gata 22B, Gothenburg 41319, Sweden.
AD  - Department of Computational Biology, University of Lausanne, 1015 Lausanne,
      Switzerland.
AD  - Gothenburg Global Biodiversity Center, Carl Skottsbergs gata 22B, Gothenburg
      41319, Sweden.
AD  - Swiss Institute of Bioinformatics, Quartier Sorge, 1015 Lausanne, Switzerland.
AD  - These authors contributed equally to this work.
FAU - Tejedor, Marcelo F
AU  - Tejedor MF
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Carl Skottsbergs gata 22B, Gothenburg 41319, Sweden.
AD  - Gothenburg Global Biodiversity Center, Carl Skottsbergs gata 22B, Gothenburg
      41319, Sweden.
AD  - Swiss Institute of Bioinformatics, Quartier Sorge, 1015 Lausanne, Switzerland.
AD  - Instituto Patagonico de Geologia y Paleontologia (CCT CONICET-CENPAT), Boulevard 
      Almirante Brown 2915, 9120 Puerto Madryn, Chubut, Argentina.
AD  - Facultad de Ciencias Naturales, Sede Trelew, Universidad Nacional de la Patagonia
      'San Juan Bosco', 9100 Trelew, Chubut, Argentina.
AD  - These authors contributed equally to this work.
FAU - Serrano-Serrano, Martha L
AU  - Serrano-Serrano ML
AD  - Department of Computational Biology, University of Lausanne, 1015 Lausanne,
      Switzerland.
FAU - Loiseau, Oriane
AU  - Loiseau O
AD  - Department of Computational Biology, University of Lausanne, 1015 Lausanne,
      Switzerland.
AD  - Swiss Institute of Bioinformatics, Quartier Sorge, 1015 Lausanne, Switzerland.
FAU - Rossier, Victor
AU  - Rossier V
AD  - Department of Computational Biology, University of Lausanne, 1015 Lausanne,
      Switzerland.
AD  - Swiss Institute of Bioinformatics, Quartier Sorge, 1015 Lausanne, Switzerland.
FAU - Rolland, Jonathan
AU  - Rolland J
AD  - Department of Computational Biology, University of Lausanne, 1015 Lausanne,
      Switzerland.
AD  - Department of Zoology, University of British Columbia, 2212 Main Mall, Vancouver,
      BC Canada.
FAU - Zizka, Alexander
AU  - Zizka A
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Carl Skottsbergs gata 22B, Gothenburg 41319, Sweden.
AD  - Gothenburg Global Biodiversity Center, Carl Skottsbergs gata 22B, Gothenburg
      41319, Sweden.
FAU - Hohna, Sebastian
AU  - Hohna S
AD  - Division of Evolutionary Biology, Ludwig-Maximilians-Universitat Munchen,
      Grosshaderner Strasse 2, 82152 Munich, Germany.
FAU - Antonelli, Alexandre
AU  - Antonelli A
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Carl Skottsbergs gata 22B, Gothenburg 41319, Sweden.
AD  - Gothenburg Global Biodiversity Center, Carl Skottsbergs gata 22B, Gothenburg
      41319, Sweden.
AD  - Gothenburg Botanical Garden, Carl Skottsbergs gata 22A, 413 19 Gothenburg,
      Sweden.
AD  - Department of Organismic and Evolutionary Biology, Harvard University, 26 Oxford 
      St., Cambridge, MA 02138, USA.
AD  - These authors are severs as a co-last authorship.
FAU - Salamin, Nicolas
AU  - Salamin N
AD  - Department of Computational Biology, University of Lausanne, 1015 Lausanne,
      Switzerland.
AD  - Swiss Institute of Bioinformatics, Quartier Sorge, 1015 Lausanne, Switzerland.
AD  - These authors are severs as a co-last authorship.
LA  - eng
SI  - Dryad/10.5061/dryad.sv43650
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Africa
MH  - Animals
MH  - *Climate
MH  - *Fossils
MH  - *Phylogeny
MH  - Platyrrhini/anatomy &amp; histology/*classification
PMC - PMC6292484
EDAT- 2018/06/23 06:00
MHDA- 2018/12/21 06:00
CRDT- 2018/06/23 06:00
PHST- 2018/01/22 00:00 [received]
PHST- 2018/06/06 00:00 [accepted]
PHST- 2018/06/23 06:00 [pubmed]
PHST- 2018/12/21 06:00 [medline]
PHST- 2018/06/23 06:00 [entrez]
AID - 5040681 [pii]
AID - 10.1093/sysbio/syy046 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jan 1;68(1):78-92. doi: 10.1093/sysbio/syy046.

PMID- 29931145
OWN - NLM
STAT- MEDLINE
DCOM- 20181220
LR  - 20181220
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 1
DP  - 2019 Jan 1
TI  - A Penalized Likelihood Framework for High-Dimensional Phylogenetic Comparative
      Methods and an Application to New-World Monkeys Brain Evolution.
PG  - 93-116
LID - 10.1093/sysbio/syy045 [doi]
AB  - Working with high-dimensional phylogenetic comparative data sets is challenging
      because likelihood-based multivariate methods suffer from low statistical
      performances as the number of traits $p $ approaches the number of species $n $
      and because some computational complications occur when $p $ exceeds $n$.
      Alternative phylogenetic comparative methods have recently been proposed to deal 
      with the large $p $ small $n $ scenario but their use and performances are
      limited. Herein, we develop a penalized likelihood (PL) framework to deal with
      high-dimensional comparative data sets. We propose various penalizations and
      methods for selecting the intensity of the penalties. We apply this general
      framework to the estimation of parameters (the evolutionary trait covariance
      matrix and parameters of the evolutionary model) and model comparison for the
      high-dimensional multivariate Brownian motion (BM), Early-burst (EB),
      Ornstein-Uhlenbeck (OU), and Pagel's lambda models. We show using simulations
      that our PL approach dramatically improves the estimation of evolutionary trait
      covariance matrices and model parameters when $p$ approaches $n$, and allows for 
      their accurate estimation when $p$ equals or exceeds $n$. In addition, we show
      that PL models can be efficiently compared using generalized information
      criterion (GIC). We implement these methods, as well as the related estimation of
      ancestral states and the computation of phylogenetic principal component analysis
      in the R package RPANDA and mvMORPH. Finally, we illustrate the utility of the
      new proposed framework by evaluating evolutionary models fit, analyzing
      integration patterns, and reconstructing evolutionary trajectories for a
      high-dimensional 3D data set of brain shape in the New World monkeys. We find a
      clear support for an EB model suggesting an early diversification of brain
      morphology during the ecological radiation of the clade. PL offers an efficient
      way to deal with high-dimensional multivariate comparative data.
FAU - Clavel, Julien
AU  - Clavel J
AD  - Ecole Normale Superieure, Paris Sciences et Lettres (PSL) Research University,
      Institut de Biologie de l'Ecole Normale Superieure (IBENS), CNRS UMR 8197, INSERM
      U1024, 46 rue d'Ulm, F-75005 Paris, France.
FAU - Aristide, Leandro
AU  - Aristide L
AD  - Ecole Normale Superieure, Paris Sciences et Lettres (PSL) Research University,
      Institut de Biologie de l'Ecole Normale Superieure (IBENS), CNRS UMR 8197, INSERM
      U1024, 46 rue d'Ulm, F-75005 Paris, France.
FAU - Morlon, Helene
AU  - Morlon H
AD  - Ecole Normale Superieure, Paris Sciences et Lettres (PSL) Research University,
      Institut de Biologie de l'Ecole Normale Superieure (IBENS), CNRS UMR 8197, INSERM
      U1024, 46 rue d'Ulm, F-75005 Paris, France.
LA  - eng
SI  - Dryad/10.5061/dryad.rf7317t
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Brain/*anatomy &amp; histology
MH  - Classification/*methods
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Platyrrhini/*anatomy &amp; histology/*classification
EDAT- 2018/06/23 06:00
MHDA- 2018/12/21 06:00
CRDT- 2018/06/23 06:00
PHST- 2017/10/23 00:00 [received]
PHST- 2018/06/13 00:00 [accepted]
PHST- 2018/06/23 06:00 [pubmed]
PHST- 2018/12/21 06:00 [medline]
PHST- 2018/06/23 06:00 [entrez]
AID - 5040209 [pii]
AID - 10.1093/sysbio/syy045 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jan 1;68(1):93-116. doi: 10.1093/sysbio/syy045.

PMID- 29893968
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - Analyzing Contentious Relationships and Outlier Genes in Phylogenomics.
PG  - 916-924
LID - 10.1093/sysbio/syy043 [doi]
AB  - Recent studies have demonstrated that conflict is common among gene trees in
      phylogenomic studies, and that less than one percent of genes may ultimately
      drive species tree inference in supermatrix analyses. Herein, we examined two
      data sets where supermatrix and coalescent-based species trees conflict. We
      identified two highly influential "outlier" genes in each data set. When removed 
      from each data set, the inferred supermatrix trees matched the topologies
      obtained from coalescent analyses. We also demonstrate that, while the outlier
      genes in the vertebrate data set have been shown in a previous study to be the
      result of errors in orthology detection, the outlier genes from a plant data set 
      did not exhibit any obvious systematic error, and therefore, may be the result of
      some biological process yet to be determined. While topological comparisons among
      a small set of alternate topologies can be helpful in discovering outlier genes, 
      they can be limited in several ways, such as assuming all genes share the same
      topology. Coalescent species tree methods relax this assumption but do not
      explicitly facilitate the examination of specific edges. Coalescent methods often
      also assume that conflict is the result of incomplete lineage sorting. Herein, we
      explored a framework that allows for quickly examining alternative edges and
      support for large phylogenomic data sets that does not assume a single topology
      for all genes. For both data sets, these analyses provided detailed results
      confirming the support for coalescent-based topologies. This framework suggests
      that we can improve our understanding of the underlying signal in phylogenomic
      data sets by asking more targeted edge-based questions.
FAU - Walker, Joseph F
AU  - Walker JF
AD  - Department of Ecology and Evolutionary Biology, University of Michigan, Ann
      Arbor, MI 48109, USA.
FAU - Brown, Joseph W
AU  - Brown JW
AD  - Department of Animal and Plant Sciences, University of Sheffield, Sheffield S10
      2TN, UK.
FAU - Smith, Stephen A
AU  - Smith SA
AD  - Department of Ecology and Evolutionary Biology, University of Michigan, Ann
      Arbor, MI 48109, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.br381mg
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Caryophyllales/*classification/genetics
MH  - *Genomics
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Vertebrates/*classification/genetics
EDAT- 2018/06/13 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/06/13 06:00
PHST- 2017/02/03 00:00 [received]
PHST- 2018/06/02 00:00 [accepted]
PHST- 2018/06/13 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/06/13 06:00 [entrez]
AID - 5034973 [pii]
AID - 10.1093/sysbio/syy043 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):916-924. doi: 10.1093/sysbio/syy043.

PMID- 29893962
OWN - NLM
STAT- MEDLINE
DCOM- 20181211
LR  - 20181211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 6
DP  - 2018 Nov 1
TI  - The Increasing Disconnection of Primary Biodiversity Data from Specimens: How
      Does It Happen and How to Handle It?
PG  - 1110-1119
LID - 10.1093/sysbio/syy044 [doi]
AB  - Primary biodiversity data represent the fundamental elements of any study in
      systematics and evolution. They are, however, no longer gathered as they used to 
      be and the mass-production of observation-based (OB) occurrences is overthrowing 
      the collection of specimen-based (SB) occurrences. Although this change in
      practice is a major upheaval with significant consequences in the study of
      biodiversity, it remains understudied and has not attracted yet the attention it 
      deserves. Analyzing 536 million occurrences from the Global Biodiversity
      Information Facility (GBIF) mediated data, we show that this spectacular change
      affects the 24 eukaryote taxonomic classes we targeted: from 1970 to 2016 the
      proportion of occurrences marked as traceable to tangible material (i.e., SB
      occurrences) fell from 68% to 18%; moreover, most of those specimen
      based-occurrences cannot be readily traced back to a specimen because the
      necessary information is missing. Ethical, practical or legal reasons responsible
      for this shift are known, and this situation appears unlikely to be reversed.
      Still, we urge scholars to acknowledge this dramatic change, embrace it and
      actively deal with it. Specifically, we emphasize why SB occurrences must be
      gathered, as a warrant to allow both repeating evolutionary studies and
      conducting rich and diverse investigations. When impossible to secure, voucher
      specimens must be replaced with OB occurrences combined with ancillary data
      (e.g., pictures, recordings, samples, DNA sequences). Ancillary data are
      instrumental for the usefulness of biodiversity occurrences and we show that,
      despite improving technologies to collate them, they remain rarely shared. The
      consequences of such a change are not yet clear but we advocate collecting
      material evidence or ancillary data to ensure that primary biodiversity data
      collected lately do not partly become obsolete when doubtful.
FAU - Troudet, Julien
AU  - Troudet J
AD  - Institut Systematique, Evolution, Biodiversite (ISYEB), Museum national
      d'Histoire naturelle, CNRS, Sorbonne Universite, EPHE, 57 rue Cuvier, CP50, 75005
      Paris, France.
FAU - Vignes-Lebbe, Regine
AU  - Vignes-Lebbe R
AD  - Institut Systematique, Evolution, Biodiversite (ISYEB), Museum national
      d'Histoire naturelle, CNRS, Sorbonne Universite, EPHE, 57 rue Cuvier, CP50, 75005
      Paris, France.
FAU - Grandcolas, Philippe
AU  - Grandcolas P
AD  - Institut Systematique, Evolution, Biodiversite (ISYEB), Museum national
      d'Histoire naturelle, CNRS, Sorbonne Universite, EPHE, 57 rue Cuvier, CP50, 75005
      Paris, France.
FAU - Legendre, Frederic
AU  - Legendre F
AD  - Institut Systematique, Evolution, Biodiversite (ISYEB), Museum national
      d'Histoire naturelle, CNRS, Sorbonne Universite, EPHE, 57 rue Cuvier, CP50, 75005
      Paris, France.
LA  - eng
SI  - Dryad/10.5061/dryad.gr883k8
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biodiversity
MH  - Classification/*methods
MH  - Data Analysis/standards
EDAT- 2018/06/13 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/06/13 06:00
PHST- 2017/10/18 00:00 [received]
PHST- 2018/06/02 00:00 [accepted]
PHST- 2018/06/13 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
PHST- 2018/06/13 06:00 [entrez]
AID - 5034972 [pii]
AID - 10.1093/sysbio/syy044 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Nov 1;67(6):1110-1119. doi: 10.1093/sysbio/syy044.

PMID- 29846734
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181114
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - Modeling Hybridization Under the Network Multispecies Coalescent.
PG  - 786-799
LID - 10.1093/sysbio/syy040 [doi]
AB  - Simultaneously modeling hybridization and the multispecies coalescent is becoming
      increasingly common, and inference of species networks in this context is now
      implemented in several software packages. This article addresses some of the
      conceptual issues and decisions to be made in this modeling, including whether or
      not to use branch lengths and issues with model identifiability. This article is 
      based on a talk given at a Spotlight Session at Evolution 2017 meeting in
      Portland, Oregon. This session included several talks about modeling
      hybridization and gene flow in the presence of incomplete lineage sorting. Other 
      talks given at this meeting are also included in this special issue of Systematic
      Biology.
FAU - Degnan, James H
AU  - Degnan JH
AD  - Department of Mathematics and Statistics, University of New Mexico, Albuquerque, 
      NM 87131, USA.
LA  - eng
GR  - R01 GM117590/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Gene Flow
MH  - *Hybridization, Genetic
MH  - *Models, Genetic
MH  - Phylogeny
PMC - PMC6101600
EDAT- 2018/05/31 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/05/31 06:00
PMCR- 2019/09/01 00:00
PHST- 2017/09/16 00:00 [received]
PHST- 2018/05/16 00:00 [accepted]
PHST- 2019/09/01 00:00 [pmc-release]
PHST- 2018/05/31 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/05/31 06:00 [entrez]
AID - 5017269 [pii]
AID - 10.1093/sysbio/syy040 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):786-799. doi: 10.1093/sysbio/syy040.

PMID- 29800341
OWN - NLM
STAT- MEDLINE
DCOM- 20181220
LR  - 20181220
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 1
DP  - 2019 Jan 1
TI  - Divergence Estimation in the Presence of Incomplete Lineage Sorting and
      Migration.
PG  - 19-31
LID - 10.1093/sysbio/syy041 [doi]
AB  - This article focuses on the problem of estimating a species tree from multilocus 
      data in the presence of incomplete lineage sorting and migration. I develop a
      mathematical model similar to IMa2 (Hey 2010) for the relevant evolutionary
      processes which allows both the population size parameters and the migration
      rates between pairs of species tree branches to be integrated out. I then
      describe a BEAST2 package DENIM (Divergence estimation notwithstanding ILS and
      migration) which is based on this model and which uses an approximation to sample
      from the posterior. The approximation is based on the assumption that migrations 
      are rare, and it only samples from certain regions of the posterior which seem
      likely given this assumption. The method breaks down if there is a lot of
      migration. Using simulations, Leache et al. (2014) showed that using the standard
      multispecies coalescent model to infer a species tree can result in poor accuracy
      if migration is present. I reanalyze this simulated data to explore DENIM's
      performance and demonstrate substantial improvements in accuracy over *BEAST. I
      also reanalyze an empirical data set.
FAU - Jones, Graham R
AU  - Jones GR
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE 405 30 Goteborg, Sweden.
LA  - eng
SI  - Dryad/10.5061/dryad.0jt38
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Models, Biological
MH  - Population Density
EDAT- 2018/05/26 06:00
MHDA- 2018/12/21 06:00
CRDT- 2018/05/26 06:00
PHST- 2017/10/17 00:00 [received]
PHST- 2018/05/16 00:00 [accepted]
PHST- 2018/05/26 06:00 [pubmed]
PHST- 2018/12/21 06:00 [medline]
PHST- 2018/05/26 06:00 [entrez]
AID - 5003138 [pii]
AID - 10.1093/sysbio/syy041 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jan 1;68(1):19-31. doi: 10.1093/sysbio/syy041.

PMID- 29788496
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20190412
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - Distinguishing Between Convergent Evolution and Violation of the Molecular Clock 
      for Three Taxa.
PG  - 905-915
LID - 10.1093/sysbio/syy038 [doi]
AB  - We give a non-technical introduction to convergence-divergence models, a new
      modeling approach for phylogenetic data that allows for the usual divergence of
      lineages after lineage-splitting but also allows for taxa to converge, i.e.
      become more similar over time. By examining the $3$-taxon case in some detail, we
      illustrate that phylogeneticists have been "spoiled" in the sense of not having
      to think about the structural parameters in their models by virtue of the strong 
      assumption that evolution is tree-like. We show that there are not always good
      statistical reasons to prefer the usual class of tree-like models over more
      general convergence-divergence models. Specifically, we show many $3$-taxon data 
      sets can be equally well explained by supposing violation of the molecular clock 
      due to change in the rate of evolution along different edges, or by keeping the
      assumption of a constant rate of evolution but instead assuming that evolution is
      not a purely divergent process. Given the abundance of evidence that evolution is
      not strictly tree-like, our discussion is an illustration that as
      phylogeneticists we need to think clearly about the structural form of the models
      we use. For cases with four taxa, we show that there will be far greater ability 
      to distinguish models with convergence from non-clock-like tree models. [Akaike
      information criterion; convergence-divergence models; distinguishability;
      identifiability; likelihood; molecular clock; phylogeny.].
FAU - Mitchell, Jonathan D
AU  - Mitchell JD
AD  - School of Natural Sciences (Maths/Physics), University of Tasmania, Private Bag
      37, Hobart, Tasmania 7001, Australia.
AD  - Department of Mathematics and Statistics, University of Alaska Fairbanks, P.O.
      Box 756660, Fairbanks, Alaska 99775-6660, USA.
FAU - Sumner, Jeremy G
AU  - Sumner JG
AD  - School of Natural Sciences (Maths/Physics), University of Tasmania, Private Bag
      37, Hobart, Tasmania 7001, Australia.
FAU - Holland, Barbara R
AU  - Holland BR
AD  - School of Natural Sciences (Maths/Physics), University of Tasmania, Private Bag
      37, Hobart, Tasmania 7001, Australia.
LA  - eng
SI  - Dryad/10.5061/dryad.n8m9c
GR  - R01 GM117590/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - *Evolution, Molecular
MH  - *Models, Genetic
MH  - *Phylogeny
PMC - PMC6454552
EDAT- 2018/05/23 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/05/23 06:00
PMCR- 2019/09/01 00:00
PHST- 2017/09/14 00:00 [received]
PHST- 2018/05/09 00:00 [accepted]
PHST- 2019/09/01 00:00 [pmc-release]
PHST- 2018/05/23 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/05/23 06:00 [entrez]
AID - 4999318 [pii]
AID - 10.1093/sysbio/syy038 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):905-915. doi: 10.1093/sysbio/syy038.

PMID- 29788398
OWN - NLM
STAT- MEDLINE
DCOM- 20181220
LR  - 20181220
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 1
DP  - 2019 Jan 1
TI  - Inferring Diversification Rate Variation From Phylogenies With Fossils.
PG  - 1-18
LID - 10.1093/sysbio/syy035 [doi]
AB  - Time-calibrated phylogenies of living species have been widely used to study the 
      tempo and mode of species diversification. However, it is increasingly clear that
      inferences about species diversification-extinction rates in particular-can be
      unreliable in the absence of paleontological data. We introduce a general
      framework based on the fossilized birth-death process for studying
      speciation-extinction dynamics on phylogenies of extant and extinct species. The 
      model assumes that phylogenies can be modeled as a mixture of distinct
      evolutionary rate regimes and that a hierarchical Poisson process governs the
      number of such rate regimes across a tree. We implemented the model in BAMM, a
      computational framework that uses reversible jump Markov chain Monte Carlo to
      simulate a posterior distribution of macroevolutionary rate regimes conditional
      on the branching times and topology of a phylogeny. The implementation, we
      describe can be applied to paleontological phylogenies, neontological
      phylogenies, and to phylogenies that include both extant and extinct taxa. We
      evaluate performance of the model on data sets simulated under a range of
      diversification scenarios. We find that speciation rates are reliably inferred in
      the absence of paleontological data. However, the inclusion of fossil
      observations substantially increases the accuracy of extinction rate estimates.
      We demonstrate that inferences are relatively robust to at least some violations 
      of model assumptions, including heterogeneity in preservation rates and
      misspecification of the number of occurrences in paleontological data sets.
FAU - Mitchell, Jonathan S
AU  - Mitchell JS
AD  - Museum of Zoology and Department of Ecology and Evolutionary Biology, University 
      of Michigan, 1105 North University Avenue, Ann Arbor, Michigan 48109 USA.
AD  - Department of Biology, West Virginia University Institute of Technology, 410
      Neville Street, Beckley, WV 25801, USA.
FAU - Etienne, Rampal S
AU  - Etienne RS
AD  - Groningen Institute for Evolutionary Life Sciences, University of Groningen, Box 
      11103, 9700 CC Groningen, The Netherlands.
FAU - Rabosky, Daniel L
AU  - Rabosky DL
AD  - Museum of Zoology and Department of Ecology and Evolutionary Biology, University 
      of Michigan, 1105 North University Avenue, Ann Arbor, Michigan 48109 USA.
LA  - eng
SI  - Dryad/10.5061/dryad.50m70
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - Computer Simulation
MH  - *Fossils
MH  - Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Time
EDAT- 2018/05/23 06:00
MHDA- 2018/12/21 06:00
CRDT- 2018/05/23 06:00
PHST- 2017/03/07 00:00 [received]
PHST- 2018/05/09 00:00 [accepted]
PHST- 2018/05/23 06:00 [pubmed]
PHST- 2018/12/21 06:00 [medline]
PHST- 2018/05/23 06:00 [entrez]
AID - 4999317 [pii]
AID - 10.1093/sysbio/syy035 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jan 1;68(1):1-18. doi: 10.1093/sysbio/syy035.

PMID- 29771371
OWN - NLM
STAT- MEDLINE
DCOM- 20181220
LR  - 20181220
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 1
DP  - 2019 Jan 1
TI  - Allele Phasing Greatly Improves the Phylogenetic Utility of Ultraconserved
      Elements.
PG  - 32-46
LID - 10.1093/sysbio/syy039 [doi]
AB  - Advances in high-throughput sequencing techniques now allow relatively easy and
      affordable sequencing of large portions of the genome, even for nonmodel
      organisms. Many phylogenetic studies reduce costs by focusing their sequencing
      efforts on a selected set of targeted loci, commonly enriched using sequence
      capture. The advantage of this approach is that it recovers a consistent set of
      loci, each with high sequencing depth, which leads to more confidence in the
      assembly of target sequences. High sequencing depth can also be used to identify 
      phylogenetically informative allelic variation within sequenced individuals, but 
      allele sequences are infrequently assembled in phylogenetic studies. Instead,
      many scientists perform their phylogenetic analyses using contig sequences which 
      result from the de novo assembly of sequencing reads into contigs containing only
      canonical nucleobases, and this may reduce both statistical power and
      phylogenetic accuracy. Here, we develop an easy-to-use pipeline to recover allele
      sequences from sequence capture data, and we use simulated and empirical data to 
      demonstrate the utility of integrating these allele sequences to analyses
      performed under the multispecies coalescent model. Our empirical analyses of
      ultraconserved element locus data collected from the South American hummingbird
      genus Topaza demonstrate that phased allele sequences carry sufficient
      phylogenetic information to infer the genetic structure, lineage divergence, and 
      biogeographic history of a genus that diversified during the last 3 myr. The
      phylogenetic results support the recognition of two species and suggest a high
      rate of gene flow across large distances of rainforest habitats but rare
      admixture across the Amazon River. Our simulations provide evidence that
      analyzing allele sequences leads to more accurate estimates of tree topology and 
      divergence times than the more common approach of using contig sequences.
FAU - Andermann, Tobias
AU  - Andermann T
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      SE-413 19 Goteborg, Sweden.
AD  - Gothenburg Global Biodiversity Centre, Box 461, SE-405 30 Goteborg, Sweden.
FAU - Fernandes, Alexandre M
AU  - Fernandes AM
AD  - Universidade Federal Rural de Pernambuco, Serra Talhada, Brazil.
FAU - Olsson, Urban
AU  - Olsson U
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      SE-413 19 Goteborg, Sweden.
AD  - Gothenburg Global Biodiversity Centre, Box 461, SE-405 30 Goteborg, Sweden.
FAU - Topel, Mats
AU  - Topel M
AD  - Gothenburg Global Biodiversity Centre, Box 461, SE-405 30 Goteborg, Sweden.
AD  - Department of Marine Sciences, University of Gothenburg, SE-413 19 Goteborg,
      Sweden.
FAU - Pfeil, Bernard
AU  - Pfeil B
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      SE-413 19 Goteborg, Sweden.
AD  - Gothenburg Global Biodiversity Centre, Box 461, SE-405 30 Goteborg, Sweden.
FAU - Oxelman, Bengt
AU  - Oxelman B
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      SE-413 19 Goteborg, Sweden.
AD  - Gothenburg Global Biodiversity Centre, Box 461, SE-405 30 Goteborg, Sweden.
FAU - Aleixo, Alexandre
AU  - Aleixo A
AD  - Museu Paraense Emilio Goeldi, Collection of Birds, Belem, Brazil.
FAU - Faircloth, Brant C
AU  - Faircloth BC
AD  - Department of Biological Sciences and Museum of Natural Science, Louisiana State 
      University, Baton Rouge, LA, USA.
FAU - Antonelli, Alexandre
AU  - Antonelli A
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      SE-413 19 Goteborg, Sweden.
AD  - Gothenburg Global Biodiversity Centre, Box 461, SE-405 30 Goteborg, Sweden.
AD  - Gothenburg Botanical Garden, SE-413 19 Goteborg, Sweden.
AD  - Department of Organismic and Evolutionary Biology, Harvard University, Cambridge,
      MA, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.hq3vq
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Alleles
MH  - Animals
MH  - Birds/classification/genetics
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Conserved Sequence/*genetics
MH  - Ecosystem
MH  - *Phylogeny
PMC - PMC6292485
EDAT- 2018/05/18 06:00
MHDA- 2018/12/21 06:00
CRDT- 2018/05/18 06:00
PHST- 2017/12/21 00:00 [received]
PHST- 2018/05/10 00:00 [accepted]
PHST- 2018/05/18 06:00 [pubmed]
PHST- 2018/12/21 06:00 [medline]
PHST- 2018/05/18 06:00 [entrez]
AID - 4996310 [pii]
AID - 10.1093/sysbio/syy039 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jan 1;68(1):32-46. doi: 10.1093/sysbio/syy039.

PMID- 29771363
OWN - NLM
STAT- MEDLINE
DCOM- 20181220
LR  - 20190804
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 1
DP  - 2019 Jan 1
TI  - Multiple Sequence Alignment Averaging Improves Phylogeny Reconstruction.
PG  - 117-130
LID - 10.1093/sysbio/syy036 [doi]
AB  - The classic methodology of inferring a phylogenetic tree from sequence data is
      composed of two steps. First, a multiple sequence alignment (MSA) is computed.
      Then, a tree is reconstructed assuming the MSA is correct. Yet, inferred MSAs
      were shown to be inaccurate and alignment errors reduce tree inference accuracy. 
      It was previously proposed that filtering unreliable alignment regions can
      increase the accuracy of tree inference. However, it was also demonstrated that
      the benefit of this filtering is often obscured by the resulting loss of
      phylogenetic signal. In this work we explore an approach, in which instead of
      relying on a single MSA, we generate a large set of alternative MSAs and
      concatenate them into a single SuperMSA. By doing so, we account for phylogenetic
      signals contained in columns that are not present in the single MSA computed by
      alignment algorithms. Using simulations, we demonstrate that this approach
      results, on average, in more accurate trees compared to 1) using an unfiltered
      MSA and 2) using a single MSA with weights assigned to columns according to their
      reliability. Next, we explore in which regions of the MSA space our approach is
      expected to be beneficial. Finally, we provide a simple criterion for deciding
      whether or not the extra effort of computing a SuperMSA and inferring a tree from
      it is beneficial. Based on these assessments, we expect our methodology to be
      useful for many cases in which diverged sequences are analyzed. The option to
      generate such a SuperMSA is available at http://guidance.tau.ac.il.
FAU - Ashkenazy, Haim
AU  - Ashkenazy H
AD  - Department of Cell Research and Immunology, George S. Wise Faculty of Life
      Sciences, Tel Aviv University, Ramat Aviv 69978, Tel Aviv, Israel.
FAU - Sela, Itamar
AU  - Sela I
AD  - National Center for Biotechnology Information, National Library of Medicine,
      National Institutes of Health, Bethesda, MD 20894, USA.
FAU - Levy Karin, Eli
AU  - Levy Karin E
AD  - Department of Cell Research and Immunology, George S. Wise Faculty of Life
      Sciences, Tel Aviv University, Ramat Aviv 69978, Tel Aviv, Israel.
AD  - Department of Molecular Biology &amp; Ecology of Plants, George S. Wise Faculty of
      Life Sciences, Tel Aviv University, Tel Aviv 69978, Israel.
FAU - Landan, Giddy
AU  - Landan G
AD  - Institute of Microbiology, Christian-Albrechts-University of Kiel, 24118 Kiel,
      Germany.
FAU - Pupko, Tal
AU  - Pupko T
AD  - Department of Cell Research and Immunology, George S. Wise Faculty of Life
      Sciences, Tel Aviv University, Ramat Aviv 69978, Tel Aviv, Israel.
LA  - eng
SI  - Dryad/10.5061/dryad.4j1qt
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - *Sequence Alignment
MH  - *Software
PMC - PMC6657586
EDAT- 2018/05/18 06:00
MHDA- 2018/12/21 06:00
CRDT- 2018/05/18 06:00
PMCR- 2020/01/01 00:00
PHST- 2017/05/12 00:00 [received]
PHST- 2018/05/09 00:00 [accepted]
PHST- 2020/01/01 00:00 [pmc-release]
PHST- 2018/05/18 06:00 [pubmed]
PHST- 2018/12/21 06:00 [medline]
PHST- 2018/05/18 06:00 [entrez]
AID - 4996308 [pii]
AID - 10.1093/sysbio/syy036 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jan 1;68(1):117-130. doi: 10.1093/sysbio/syy036.

PMID- 29757422
OWN - NLM
STAT- MEDLINE
DCOM- 20181211
LR  - 20181211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 6
DP  - 2018 Nov 1
TI  - Assessing the Impacts of Positive Selection on Coalescent-Based Species Tree
      Estimation and Species Delimitation.
PG  - 1076-1090
LID - 10.1093/sysbio/syy034 [doi]
AB  - The assumption of strictly neutral evolution is fundamental to the multispecies
      coalescent model and permits the derivation of gene tree distributions and
      coalescent times conditioned on a given species tree. In this study, we conduct
      computer simulations to explore the effects of violating this assumption in the
      form of species-specific positive selection when estimating species trees,
      species delimitations, and coalescent parameters under the model. We simulated
      data sets under an array of evolutionary scenarios that differ in both speciation
      parameters (i.e., divergence times, strength of selection) and experimental
      design (i.e., number of loci sampled) and incorporated species-specific positive 
      selection occurring within branches of a species tree to identify the effects of 
      selection on multispecies coalescent inferences. Our results highlight particular
      evolutionary scenarios and parameter combinations in which inferences may be
      more, or less, susceptible to the effects of positive selection. In some extreme 
      cases, selection can decrease error in species delimitation and increase error in
      species tree estimation, yet these inferences appear to be largely robust to the 
      effects of positive selection under many conditions likely to be encountered in
      empirical data sets.
FAU - Adams, Richard H
AU  - Adams RH
AD  - Department of Biology, University of Texas at Arlington, 501 S. Nedderman Dr.,
      Arlington, TX 76019, USA.
FAU - Schield, Drew R
AU  - Schield DR
AD  - Department of Biology, University of Texas at Arlington, 501 S. Nedderman Dr.,
      Arlington, TX 76019, USA.
FAU - Card, Daren C
AU  - Card DC
AD  - Department of Biology, University of Texas at Arlington, 501 S. Nedderman Dr.,
      Arlington, TX 76019, USA.
FAU - Castoe, Todd A
AU  - Castoe TA
AD  - Department of Biology, University of Texas at Arlington, 501 S. Nedderman Dr.,
      Arlington, TX 76019, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.5v3b5
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - *Genetic Speciation
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - *Selection, Genetic
MH  - Species Specificity
EDAT- 2018/05/15 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/05/15 06:00
PHST- 2017/06/20 00:00 [received]
PHST- 2018/05/05 00:00 [accepted]
PHST- 2018/05/15 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
PHST- 2018/05/15 06:00 [entrez]
AID - 4995179 [pii]
AID - 10.1093/sysbio/syy034 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Nov 1;67(6):1076-1090. doi: 10.1093/sysbio/syy034.

PMID- 29718447
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - Posterior Summarization in Bayesian Phylogenetics Using Tracer 1.7.
PG  - 901-904
LID - 10.1093/sysbio/syy032 [doi]
AB  - Bayesian inference of phylogeny using Markov chain Monte Carlo (MCMC) plays a
      central role in understanding evolutionary history from molecular sequence data. 
      Visualizing and analyzing the MCMC-generated samples from the posterior
      distribution is a key step in any non-trivial Bayesian inference. We present the 
      software package Tracer (version 1.7) for visualizing and analyzing the MCMC
      trace files generated through Bayesian phylogenetic inference. Tracer provides
      kernel density estimation, multivariate visualization, demographic trajectory
      reconstruction, conditional posterior distribution summary, and more. Tracer is
      open-source and available at http://beast.community/tracer.
FAU - Rambaut, Andrew
AU  - Rambaut A
AD  - Institute of Evolutionary Biology, University of Edinburgh, Ashworth
      Laboratories, King's Buildings, Edinburgh, EH9 3FL, UK.
FAU - Drummond, Alexei J
AU  - Drummond AJ
AD  - Department of Computer Science, University of Auckland, 303/38 Princes St,
      Auckland, 1010, NZ.
AD  - Centre for Computational Evolution, University of Auckland, 303/38 Princes St,
      Auckland, 1010, NZ.
FAU - Xie, Dong
AU  - Xie D
AD  - Department of Computer Science, University of Auckland, 303/38 Princes St,
      Auckland, 1010, NZ.
AD  - Centre for Computational Evolution, University of Auckland, 303/38 Princes St,
      Auckland, 1010, NZ.
FAU - Baele, Guy
AU  - Baele G
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven - University
      of Leuven, Herestraat 49, 3000 Leuven, Belgium.
FAU - Suchard, Marc A
AU  - Suchard MA
AD  - Department of Human Genetics, University of California, Los Angeles, 695 Charles 
      E. Young Dr., Los Angeles, CA 90095, USA.
AD  - Department of Biostatistics, University of California, Los Angeles, 650 Charles
      E. Young Dr., Los Angeles, CA 90095, USA.
LA  - eng
GR  - U19 AI135995/AI/NIAID NIH HHS/United States
GR  - 206298/Z/17/Z/Wellcome Trust/United Kingdom
GR  - R01 AI107034/AI/NIAID NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Evolution, Molecular
MH  - Markov Chains
MH  - Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - *Software
PMC - PMC6101584
EDAT- 2018/05/03 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/05/03 06:00
PHST- 2018/02/13 00:00 [received]
PHST- 2018/04/20 00:00 [accepted]
PHST- 2018/05/03 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/05/03 06:00 [entrez]
AID - 4989127 [pii]
AID - 10.1093/sysbio/syy032 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):901-904. doi: 10.1093/sysbio/syy032.

PMID- 29701838
OWN - NLM
STAT- MEDLINE
DCOM- 20181211
LR  - 20181211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 6
DP  - 2018 Nov 1
TI  - Rethinking phylogenetic comparative methods.
PG  - 1091-1109
LID - 10.1093/sysbio/syy031 [doi]
AB  - As a result of the process of descent with modification, closely related species 
      tend to be similar to one another in a myriad different ways. In statistical
      terms, this means that traits measured on one species will not be independent of 
      traits measured on others. Since their introduction in the 1980s, phylogenetic
      comparative methods (PCMs) have been framed as a solution to this problem. In
      this article, we argue that this way of thinking about PCMs is deeply misleading.
      Not only has this sowed widespread confusion in the literature about what PCMs
      are doing but has led us to develop methods that are susceptible to the very
      thing we sought to build defenses against-unreplicated evolutionary events.
      Through three Case Studies, we demonstrate that the susceptibility to singular
      events is indeed a recurring problem in comparative biology that links several
      seemingly unrelated controversies. In each Case Study, we propose a potential
      solution to the problem. While the details of our proposed solutions differ, they
      share a common theme: unifying hypothesis testing with data-driven approaches
      (which we term "phylogenetic natural history") to disentangle the impact of
      singular evolutionary events from that of the factors we are investigating. More 
      broadly, we argue that our field has, at times, been sloppy when weighing
      evidence in support of causal hypotheses. We suggest that one way to refine our
      inferences is to re-imagine phylogenies as probabilistic graphical models;
      adopting this way of thinking will help clarify precisely what we are testing and
      what evidence supports our claims.
FAU - Uyeda, Josef C
AU  - Uyeda JC
AD  - Department of Biological Sciences, Virginia Polytechnic Institute and State
      University, 926 West Campus Drive, Blacksburg, VA 24061 USA.
FAU - Zenil-Ferguson, Rosana
AU  - Zenil-Ferguson R
AD  - Department of Biological Sciences, University of Idaho, 875 Perimeter Drive,
      Moscow, ID 83844 USA.
AD  - Department of Ecology, Evolution and Behavior, University of Minnesota, 1479
      Gortner Avenue, St. Paul, MN 55108 USA.
FAU - Pennell, Matthew W
AU  - Pennell MW
AD  - Department of Zoology and Biodiversity Research Centre, University of British
      Columbia, #4200-6700 University Blvd., Vancouver, BC V6T 1Z4, Canada.
LA  - eng
SI  - Dryad/10.5061/dryad.p8066hd
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Phylogeny
EDAT- 2018/04/28 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/04/28 06:00
PHST- 2017/11/21 00:00 [received]
PHST- 2018/04/17 00:00 [accepted]
PHST- 2018/04/28 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
PHST- 2018/04/28 06:00 [entrez]
AID - 4985805 [pii]
AID - 10.1093/sysbio/syy031 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Nov 1;67(6):1091-1109. doi: 10.1093/sysbio/syy031.

PMID- 29701821
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - Phylogenetic Comparative Methods on Phylogenetic Networks with Reticulations.
PG  - 800-820
LID - 10.1093/sysbio/syy033 [doi]
AB  - The goal of phylogenetic comparative methods (PCMs) is to study the distribution 
      of quantitative traits among related species. The observed traits are often seen 
      as the result of a Brownian Motion (BM) along the branches of a phylogenetic
      tree. Reticulation events such as hybridization, gene flow or horizontal gene
      transfer, can substantially affect a species' traits, but are not modeled by a
      tree. Phylogenetic networks have been designed to represent reticulate evolution.
      As they become available for downstream analyses, new models of trait evolution
      are needed, applicable to networks. We develop here an efficient recursive
      algorithm to compute the phylogenetic variance matrix of a trait on a network, in
      only one preorder traversal of the network. We then extend the standard PCM tools
      to this new framework, including phylogenetic regression with covariates (or
      phylogenetic ANOVA), ancestral trait reconstruction, and Pagel's $\lambda$ test
      of phylogenetic signal. The trait of a hybrid is sometimes outside of the range
      of its two parents, for instance because of hybrid vigor or hybrid depression.
      These two phenomena are rather commonly observed in present-day hybrids.
      Transgressive evolution can be modeled as a shift in the trait value following a 
      reticulation point. We develop a general framework to handle such shifts and take
      advantage of the phylogenetic regression view of the problem to design
      statistical tests for ancestral transgressive evolution in the evolutionary
      history of a group of species. We study the power of these tests in several
      scenarios and show that recent events have indeed the strongest impact on the
      trait distribution of present-day taxa. We apply those methods to a data set of
      Xiphophorus fishes, to confirm and complete previous analysis in this group. All 
      the methods developed here are available in the Julia package PhyloNetworks.
FAU - Bastide, Paul
AU  - Bastide P
AD  - Unite Mixte de Recherche Mathematiques et Informatique Appliquees (MIA - Paris), 
      AgroParisTech, Institut National de la Recherche Agronomique (INRA), Universite
      Paris-Saclay, 16 rue Claude Bernard, 75005 Paris, France.
AD  - Unite de Recherche Mathematiques et Informatique Appliquees du Genome
      al'Environnement (MaIAGE), Institut National de la Recherche Agronomique (INRA), 
      Universite Paris-Saclay, Domaine de Vilvert, 78352 Jouy-en-Josas, France.
FAU - Solis-Lemus, Claudia
AU  - Solis-Lemus C
AD  - Department of Statistics, University of Wisconsin-Madison, 1300 University
      Avenue, Madison, WI 53706, USA.
FAU - Kriebel, Ricardo
AU  - Kriebel R
AD  - Department of Botany, University of Wisconsin-Madison, 430 Lincoln Drive,
      Madison, WI 53706, USA.
FAU - William Sparks, K
AU  - William Sparks K
AD  - Department of Botany, University of Wisconsin-Madison, 430 Lincoln Drive,
      Madison, WI 53706, USA.
FAU - Ane, Cecile
AU  - Ane C
AD  - Department of Statistics, University of Wisconsin-Madison, 1300 University
      Avenue, Madison, WI 53706, USA.
AD  - Department of Botany, University of Wisconsin-Madison, 430 Lincoln Drive,
      Madison, WI 53706, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.nt2g6
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Cyprinodontiformes/classification/*genetics
MH  - *Evolution, Molecular
MH  - *Gene Flow
MH  - *Gene Transfer, Horizontal
MH  - *Hybridization, Genetic
MH  - Models, Genetic
MH  - Phenotype
MH  - *Phylogeny
EDAT- 2018/04/28 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/04/28 06:00
PHST- 2017/09/19 00:00 [received]
PHST- 2018/04/20 00:00 [accepted]
PHST- 2018/04/28 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/04/28 06:00 [entrez]
AID - 4985806 [pii]
AID - 10.1093/sysbio/syy033 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):800-820. doi: 10.1093/sysbio/syy033.

PMID- 29669054
OWN - NLM
STAT- MEDLINE
DCOM- 20181211
LR  - 20181211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 6
DP  - 2018 Nov 1
TI  - Ranked Tree Shapes, Nonrandom Extinctions, and the Loss of Phylogenetic
      Diversity.
PG  - 1025-1040
LID - 10.1093/sysbio/syy030 [doi]
AB  - Phylogenetic diversity (PD) is a measure of the evolutionary legacy of a group of
      species, which can be used to define conservation priorities. It has been shown
      that an important loss of species diversity can sometimes lead to a much less
      important loss of PD, depending on the topology of the species tree and on the
      distribution of its branch lengths. However, the rate of decrease of PD strongly 
      depends on the relative depths of the nodes in the tree and on the order in which
      species become extinct. We introduce a new, sampling-consistent, three-parameter 
      model generating random trees with covarying topology, clades relative depths,
      and clades relative extinction risks. This model can be seen as an extension to
      Aldous' one parameter splitting model ($\beta$, which controls for tree balance) 
      with two additional parameters: a new parameter $\alpha$ quantifying the relation
      between age and richness of subclades, and a parameter $\eta$ quantifying the
      relation between relative abundance and richness of subclades, taken herein as a 
      proxy for overall extinction risk. We show on simulated phylogenies that loss of 
      PD depends on the combined effect of all three parameters, $\beta$, $\alpha,$ and
      $\eta$. In particular, PD may decrease as fast as species diversity when high
      extinction risks are clustered within small, old clades, corresponding to a
      parameter range that we term the "danger zone" ($\beta&lt;-1$ or $\alpha&lt;0$;
      $\eta&gt;1$). Besides, when high extinction risks are clustered within large clades,
      the loss of PD can be higher in trees that are more balanced ($\beta&gt;0$), in
      contrast to the predictions of earlier studies based on simpler models. We
      propose a Monte-Carlo algorithm, tested on simulated data, to infer all three
      parameters. Applying it to a real data set comprising 120 bird clades (class
      Aves) with known range sizes, we show that parameter estimates precisely fall
      close to the danger zone: the combination of their ranking tree shape and
      nonrandom extinctions risks makes them prone to a sudden collapse of PD.
FAU - Maliet, Odile
AU  - Maliet O
AD  - Institut de Biologie de l'Ecole Normale Superieure (IBENS), Ecole Normale
      Superieure, CNRS, INSERM, PSL Research University, Paris, France.
AD  - ED 227, Sorbonne Universites, Paris, France.
FAU - Gascuel, Fanny
AU  - Gascuel F
AD  - Institut de Biologie de l'Ecole Normale Superieure (IBENS), Ecole Normale
      Superieure, CNRS, INSERM, PSL Research University, Paris, France.
AD  - ED 227, Sorbonne Universites, Paris, France.
AD  - Center for Interdisciplinary Research in Biology (CIRB), College de France, CNRS,
      INSERM, PSL Research University, Paris, France.
FAU - Lambert, Amaury
AU  - Lambert A
AD  - Center for Interdisciplinary Research in Biology (CIRB), College de France, CNRS,
      INSERM, PSL Research University, Paris, France.
AD  - Laboratoire Probabilites, Statistique et Modelisation (LPSM), Sorbonne
      Universite, CNRS, Paris, France.
LA  - eng
SI  - Dryad/10.5061/dryad.mv980
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - *Biodiversity
MH  - Classification
MH  - *Extinction, Biological
MH  - *Models, Biological
MH  - Monte Carlo Method
MH  - *Phylogeny
EDAT- 2018/04/19 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/04/19 06:00
PHST- 2016/06/03 00:00 [received]
PHST- 2018/04/08 00:00 [accepted]
PHST- 2018/04/19 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
PHST- 2018/04/19 06:00 [entrez]
AID - 4972608 [pii]
AID - 10.1093/sysbio/syy030 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Nov 1;67(6):1025-1040. doi: 10.1093/sysbio/syy030.

PMID- 29669028
OWN - NLM
STAT- MEDLINE
DCOM- 20181220
LR  - 20181220
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 68
IP  - 1
DP  - 2019 Jan 1
TI  - The Grass was Greener: Repeated Evolution of Specialized Morphologies and Habitat
      Shifts in Ghost Spiders Following Grassland Expansion in South America.
PG  - 63-77
LID - 10.1093/sysbio/syy028 [doi]
AB  - While grasslands, one of Earth's major biomes, are known for their close
      evolutionary ties with ungulate grazers, these habitats are also paramount to the
      origins and diversification of other animals. Within the primarily South American
      spider subfamily Amaurobioidinae (Anyphaenidae), several species are found living
      in the continent's grasslands, with some displaying putative morphological
      adaptations to dwelling unnoticed in the grass blades. Herein, a dated molecular 
      phylogeny provides the backbone for analyses revealing the ecological and
      morphological processes behind these spiders' grassland adaptations. The multiple
      switches from Patagonian forests to open habitats coincide with the expansion of 
      South America's grasslands during the Miocene, while the specialized morphology
      of several grass-dwelling spiders originated at least three independent times and
      is best described as the result of different selective regimes operating on
      macroevolutionary timescales. Although grass-adapted lineages evolved towards
      different peaks in adaptive landscape, they all share one characteristic: an
      anterior narrowing of the prosoma allowing spiders to extend the first two pairs 
      of legs, thus maintaining a slender resting posture in the grass blade. By
      combining phylogenetic, morphological, and biogeographic perspectives we
      disentangle multiple factors determining the evolution of a clade of terrestrial 
      invertebrate predators alongside their biomes.
FAU - Ceccarelli, F Sara
AU  - Ceccarelli FS
AD  - Division de Aracnologia, Museo Argentino de Ciencias Naturales "Bernardino
      Rivadavia", Av. Angel Gallardo 470, C1405DJR, Buenos Aires, Argentina.
AD  - Departamento de Biologia de la Conservacion, CONACYT-Centro de Investigacion
      Cientifica y de Educacion Superior de Ensenada, Carr Tijuana-Ensenada 3918, 22860
      Ensenada, B.C., Mexico.
FAU - Koch, Nicolas Mongiardino
AU  - Koch NM
AD  - Department of Geology &amp; Geophysics, Yale University, 210 Whitney Avenue, New
      Haven, CT 06511, USA.
FAU - Soto, Eduardo M
AU  - Soto EM
AD  - Departamento de Ecologia, Genetica y Evolucion, IEGEBA (CONICET-UBA), Facultad de
      Ciencias Exactas y Naturales, Universidad de Buenos Aires. Ciudad Universitaria, 
      Pabellon II (C1428 EHA), Buenos Aires, Argentina.
FAU - Barone, Mariana L
AU  - Barone ML
AD  - Division de Aracnologia, Museo Argentino de Ciencias Naturales "Bernardino
      Rivadavia", Av. Angel Gallardo 470, C1405DJR, Buenos Aires, Argentina.
FAU - Arnedo, Miquel A
AU  - Arnedo MA
AD  - Departament de Biologia Evolutiva, Ecologia i Ciencies Ambientals &amp; Institut de
      Recerca de la Biodiversitat (IRBio), Universitat de Barcelona, Av. Diagonal 645, 
      E-8028 Barcelona, Spain.
FAU - Ramirez, Martin J
AU  - Ramirez MJ
AD  - Division de Aracnologia, Museo Argentino de Ciencias Naturales "Bernardino
      Rivadavia", Av. Angel Gallardo 470, C1405DJR, Buenos Aires, Argentina.
LA  - eng
SI  - Dryad/10.5061/dryad.20257
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Physiological/genetics/*physiology
MH  - Animals
MH  - Biological Evolution
MH  - Ecosystem
MH  - *Grassland
MH  - *Phylogeny
MH  - South America
MH  - Spiders/*anatomy &amp; histology/classification/*genetics
EDAT- 2018/04/19 06:00
MHDA- 2018/12/21 06:00
CRDT- 2018/04/19 06:00
PHST- 2017/08/09 00:00 [received]
PHST- 2018/04/10 00:00 [accepted]
PHST- 2018/04/19 06:00 [pubmed]
PHST- 2018/12/21 06:00 [medline]
PHST- 2018/04/19 06:00 [entrez]
AID - 4971554 [pii]
AID - 10.1093/sysbio/syy028 [doi]
PST - ppublish
SO  - Syst Biol. 2019 Jan 1;68(1):63-77. doi: 10.1093/sysbio/syy028.

PMID- 29669013
OWN - NLM
STAT- MEDLINE
DCOM- 20181211
LR  - 20181211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 6
DP  - 2018 Nov 1
TI  - Information Dropout Patterns in Restriction Site Associated DNA Phylogenomics and
      a Comparison with Multilocus Sanger Data in a Species-Rich Moth Genus.
PG  - 925-939
LID - 10.1093/sysbio/syy029 [doi]
AB  - A rapid shift from traditional Sanger sequencing-based molecular methods to the
      phylogenomic approach with large numbers of loci is underway. Among phylogenomic 
      methods, restriction site associated DNA (RAD) sequencing approaches have gained 
      much attention as they enable rapid generation of up to thousands of loci
      randomly scattered across the genome and are suitable for nonmodel species. RAD
      data sets however suffer from large amounts of missing data and rapid locus
      dropout along with decreasing relatedness among taxa. The relationship between
      locus dropout and the amount of phylogenetic information retained in the data has
      remained largely uninvestigated. Similarly, phylogenetic hypotheses based on RAD 
      have rarely been compared with phylogenetic hypotheses based on multilocus Sanger
      sequencing, even less so using exactly the same species and specimens. We
      compared the Sanger-based phylogenetic hypothesis (8 loci; 6172 bp) of 32 species
      of the diverse moth genus Eupithecia (Lepidoptera, Geometridae) to that based on 
      double-digest RAD sequencing (3256 loci; 726,658 bp). We observed that topologies
      were largely congruent, with some notable exceptions that we discuss. The locus
      dropout effect was strong. We demonstrate that number of loci is not a precise
      measure of phylogenetic information since the number of single-nucleotide
      polymorphisms (SNPs) may remain low at very shallow phylogenetic levels despite
      large numbers of loci. As we hypothesize, the number of SNPs and parsimony
      informative SNPs (PIS) is low at shallow phylogenetic levels, peaks at
      intermediate levels and, thereafter, declines again at the deepest levels as a
      result of decay of available loci. Similarly, we demonstrate with empirical data 
      that the locus dropout affects the type of loci retained, the loci found in many 
      species tending to show lower interspecific distances than those shared among
      fewer species. We also examine the effects of the numbers of loci, SNPs, and PIS 
      on nodal bootstrap support, but could not demonstrate with our data our
      expectation of a positive correlation between them. We conclude that RAD methods 
      provide a powerful tool for phylogenomics at an intermediate phylogenetic level
      as indicated by its broad congruence with an eight-gene Sanger data set in a
      genus of moths. When assessing the quality of the data for phylogenetic
      inference, the focus should be on the distribution and number of SNPs and PIS
      rather than on loci.
FAU - Lee, Kyung Min
AU  - Lee KM
AD  - Department of Ecology and Genetics, University of Oulu, Pentti Kaiteran katu 1,
      FI-90014, Oulu, Finland.
FAU - Kivela, Sami M
AU  - Kivela SM
AD  - Department of Ecology and Genetics, University of Oulu, Pentti Kaiteran katu 1,
      FI-90014, Oulu, Finland.
AD  - Department of Zoology, Institute of Ecology and Earth Sciences, University of
      Tartu, Vanemuise 46, EE-51014 Tartu, Estonia.
FAU - Ivanov, Vladislav
AU  - Ivanov V
AD  - Department of Ecology and Genetics, University of Oulu, Pentti Kaiteran katu 1,
      FI-90014, Oulu, Finland.
FAU - Hausmann, Axel
AU  - Hausmann A
AD  - SNSB - Bavarian State Collection of Zoology, Munchhausenstr. 21, D-81247 Munich, 
      Germany.
FAU - Kaila, Lauri
AU  - Kaila L
AD  - Finnish Museum of Natural History, Zoology Unit, FI-00014 University of Helsinki,
      P. Rautatiekatu 13, P.O. Box 17, Helsinki, Finland.
FAU - Wahlberg, Niklas
AU  - Wahlberg N
AD  - Department of Biology, Solvegatan 37, Lund University, SE-223 62 Lund, Sweden.
FAU - Mutanen, Marko
AU  - Mutanen M
AD  - Department of Ecology and Genetics, University of Oulu, Pentti Kaiteran katu 1,
      FI-90014, Oulu, Finland.
LA  - eng
SI  - Dryad/10.5061/dryad.474nd
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Data Analysis/standards
MH  - Genome/*genetics
MH  - *Genomics
MH  - High-Throughput Nucleotide Sequencing/standards
MH  - Moths/*genetics
MH  - *Phylogeny
EDAT- 2018/04/19 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/04/19 06:00
PHST- 2017/04/07 00:00 [received]
PHST- 2018/04/10 00:00 [accepted]
PHST- 2018/04/19 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
PHST- 2018/04/19 06:00 [entrez]
AID - 4971555 [pii]
AID - 10.1093/sysbio/syy029 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Nov 1;67(6):925-939. doi: 10.1093/sysbio/syy029.

PMID- 29635540
OWN - NLM
STAT- MEDLINE
DCOM- 20181211
LR  - 20181211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 6
DP  - 2018 Nov 1
TI  - A Framework for Resolving Cryptic Species: A Case Study from the Lizards of the
      Australian Wet Tropics.
PG  - 1061-1075
LID - 10.1093/sysbio/syy026 [doi]
AB  - As we collect range-wide genetic data for morphologically-defined species, we
      increasingly unearth evidence for cryptic diversity. Delimiting this cryptic
      diversity is challenging, both because the divergences span a continuum and
      because the lack of overt morphological differentiation suggests divergence has
      proceeded heterogeneously. Herein, we address these challenges as we diagnose and
      describe species in three co-occurring species groups of Australian lizards. By
      integrating genomic and morphological data with data on hybridization and
      introgression from contact zones, we explore several approaches-and their
      relative benefits and weaknesses-for testing the validity of cryptic lineages.
      More generally, we advocate that genetic delimitations of cryptic diversity must 
      consider whether these lineages are likely to be durable and persistent through
      evolutionary time.
FAU - Singhal, Sonal
AU  - Singhal S
AD  - Museum of Zoology and Department of Ecology and Evolutionary Biology, University 
      of Michigan, Ann Arbor, MI 48109, USA.
AD  - Department of Biology, California State University-Dominguez Hills, Carson, CA
      90747, USA.
FAU - Hoskin, Conrad J
AU  - Hoskin CJ
AD  - College of Science &amp; Engineering, James Cook University, Townsville, Queensland
      4811, Australia.
FAU - Couper, Patrick
AU  - Couper P
AD  - Biodiversity Program, Queensland Museum, South Brisbane, Queensland 4101,
      Australia.
FAU - Potter, Sally
AU  - Potter S
AD  - Division of Ecology and Evolution, Research School of Biology and Centre for
      Biodiversity Analysis, Australian National University, Acton, ACT 2601,
      Australia.
FAU - Moritz, Craig
AU  - Moritz C
AD  - Division of Ecology and Evolution, Research School of Biology and Centre for
      Biodiversity Analysis, Australian National University, Acton, ACT 2601,
      Australia.
LA  - eng
SI  - Dryad/10.5061/dryad.g7v1b
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Australia
MH  - Classification/*methods
MH  - DNA, Mitochondrial/genetics
MH  - Genetic Speciation
MH  - Genetic Variation
MH  - Lizards/*classification/*genetics
MH  - *Phylogeny
EDAT- 2018/04/11 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/04/11 06:00
PHST- 2017/11/20 00:00 [received]
PHST- 2018/03/27 00:00 [accepted]
PHST- 2018/04/11 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
PHST- 2018/04/11 06:00 [entrez]
AID - 4960881 [pii]
AID - 10.1093/sysbio/syy026 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Nov 1;67(6):1061-1075. doi: 10.1093/sysbio/syy026.

PMID- 29566212
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - The Effect of Gene Flow on Coalescent-based Species-Tree Inference.
PG  - 770-785
LID - 10.1093/sysbio/syy020 [doi]
AB  - Most current methods for inferring species-level phylogenies under the coalescent
      model assume that no gene flow occurs following speciation. Several studies have 
      examined the impact of gene flow (e.g., Eckert and Carstens 2008; Chung and Ane
      2011; Leache et al. 2014; Solis-Lemus et al. 2016) and of ancestral population
      structure (DeGeorgio and Rosenberg 2016) on the performance of species-level
      phylogenetic inference, and analytic results have been proven for network models 
      of gene flow (e.g., Solis-Lemus et al. 2016; Zhu et al. 2016). However, there are
      few analytic results for a continuous model of gene flow following speciation,
      despite the development of mathematical tools that could facilitate such study
      (e.g., Hobolth et al. 2011; Andersen et al. 2014; Tian and Kubatko 2016). In this
      article, we consider a three-taxon isolation-with-migration model that allows
      gene flow between sister taxa for a brief period following speciation, as well as
      variation in the effective population sizes across the species tree. We derive
      the probabilities of each of the three gene tree topologies under this model, and
      show that for certain choices of the gene flow and effective population size
      parameters, anomalous gene trees (i.e., gene trees that are discordant with the
      species tree but that have higher probability than the gene tree concordant with 
      the species tree) exist. We characterize the region of parameter space producing 
      anomalous trees and show that the probability of the gene tree that is concordant
      with the species tree can be arbitrarily small. We then show that there is
      theoretical support for using SVDQuartets with an outgroup to infer the rooted
      three-taxon species tree in a model of gene flow between sister taxa. We study
      the performance of SVDQuartets on simulated data and compare it to three other
      commonly-used methods for species tree inference, ASTRAL, MP-EST, and
      concatenation. The simulations show that ASTRAL, MP-EST, and concatenation can be
      statistically inconsistent when gene flow is present, while SVDQuartets performs 
      well, though large sample sizes may be required for certain parameter choices.
FAU - Long, Colby
AU  - Long C
AD  - Mathematical Biosciences Institute, The Ohio State University, 1735 Neil Ave.,
      Columbus, OH, USA.
FAU - Kubatko, Laura
AU  - Kubatko L
AD  - Department of Statistics, Department of Evolution, Ecology, and Organismal
      Biology, The Ohio State University, 1958 Neil Ave., Columbus, OH, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.j11fh
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Gene Flow
MH  - *Genetic Speciation
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Population Density
MH  - Probability
EDAT- 2018/03/23 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/03/23 06:00
PHST- 2017/09/19 00:00 [received]
PHST- 2018/03/06 00:00 [accepted]
PHST- 2018/03/23 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/03/23 06:00 [entrez]
AID - 4939203 [pii]
AID - 10.1093/sysbio/syy020 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):770-785. doi: 10.1093/sysbio/syy020.

PMID- 29562307
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20190412
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - HyDe: A Python Package for Genome-Scale Hybridization Detection.
PG  - 821-829
LID - 10.1093/sysbio/syy023 [doi]
AB  - The analysis of hybridization and gene flow among closely related taxa is a
      common goal for researchers studying speciation and phylogeography. Many methods 
      for hybridization detection use simple site pattern frequencies from observed
      genomic data and compare them to null models that predict an absence of gene
      flow. The theory underlying the detection of hybridization using these site
      pattern probabilities exploits the relationship between the coalescent process
      for gene trees within population trees and the process of mutation along the
      branches of the gene trees. For certain models, site patterns are predicted to
      occur in equal frequency (i.e., their difference is 0), producing a set of
      functions called phylogenetic invariants. In this article, we introduce HyDe, a
      software package for detecting hybridization using phylogenetic invariants
      arising under the coalescent model with hybridization. HyDe is written in Python 
      and can be used interactively or through the command line using pre-packaged
      scripts. We demonstrate the use of HyDe on simulated data, as well as on two
      empirical data sets from the literature. We focus in particular on identifying
      individual hybrids within population samples and on distinguishing between hybrid
      speciation and gene flow. HyDe is freely available as an open source Python
      package under the GNU GPL v3 on both GitHub (https://github.com/pblischak/HyDe)
      and the Python Package Index (PyPI: https://pypi.python.org/pypi/phyde).
FAU - Blischak, Paul D
AU  - Blischak PD
AD  - Department of Evolution, Ecology, and Organismal Biology, The Ohio State
      University, Columbus, OH 43210, USA.
FAU - Chifman, Julia
AU  - Chifman J
AD  - Department of Mathematics and Statistics, American University, Washington, DC
      20016, USA.
FAU - Wolfe, Andrea D
AU  - Wolfe AD
AD  - Department of Evolution, Ecology, and Organismal Biology, The Ohio State
      University, Columbus, OH 43210, USA.
FAU - Kubatko, Laura S
AU  - Kubatko LS
AD  - Department of Evolution, Ecology, and Organismal Biology, The Ohio State
      University, Columbus, OH 43210, USA.
AD  - Department of Statistics, The Ohio State University, Columbus, OH 43210, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.372sq
GR  - T32 CA079448/CA/NCI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computational Biology/*methods
MH  - *Gene Flow
MH  - *Genetic Speciation
MH  - *Hybridization, Genetic
MH  - Software
PMC - PMC6454532
EDAT- 2018/03/22 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/03/22 06:00
PMCR- 2019/09/01 00:00
PHST- 2017/09/14 00:00 [received]
PHST- 2018/03/15 00:00 [accepted]
PHST- 2019/09/01 00:00 [pmc-release]
PHST- 2018/03/22 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/03/22 06:00 [entrez]
AID - 4944070 [pii]
AID - 10.1093/sysbio/syy023 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):821-829. doi: 10.1093/sysbio/syy023.

PMID- 29562303
OWN - NLM
STAT- MEDLINE
DCOM- 20181211
LR  - 20181211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 6
DP  - 2018 Nov 1
TI  - Dating the Species Network: Allopolyploidy and Repetitive DNA Evolution in
      American Daisies (Melampodium sect. Melampodium, Asteraceae).
PG  - 1010-1024
LID - 10.1093/sysbio/syy024 [doi]
AB  - Allopolyploidy has played an important role in the evolution of the flowering
      plants. Genome mergers are often accompanied by significant and rapid alterations
      of genome size and structure via chromosomal rearrangements and altered dynamics 
      of tandem and dispersed repetitive DNA families. Recent developments in
      sequencing technologies and bioinformatic methods allow for a comprehensive
      investigation of the repetitive component of plant genomes. Interpretation of
      evolutionary dynamics following allopolyploidization requires both the knowledge 
      of parentage and the age of origin of an allopolyploid. Whereas parentage is
      typically inferred from cytogenetic and phylogenetic data, age inference is
      hampered by the reticulate nature of the phylogenetic relationships. Treating
      subgenomes of allopolyploids as if they belonged to different species (i.e., no
      recombination among subgenomes) and applying cross-bracing (i.e., putting a
      constraint on the age difference of nodes pertaining to the same event), we can
      infer the age of allopolyploids within the framework of the multispecies
      coalescent within BEAST2. Together with a comprehensive characterization of the
      repetitive DNA fraction using the RepeatExplorer pipeline, we apply the dating
      approach in a group of closely related allopolyploids and their progenitor
      species in the plant genus Melampodium (Asteraceae). We dated the origin of both 
      the allotetraploid, Melampodium strigosum, and its two allohexaploid derivatives,
      Melampodium pringlei and Melampodium sericeum, which share both parentage and the
      direction of the cross, to the Pleistocene ($&lt;$1.4 Ma). Thus, Pleistocene
      climatic fluctuations may have triggered formation of allopolyploids possibly in 
      short intervals, contributing to difficulties in inferring the precise temporal
      order of allopolyploid species divergence of M. sericeum and M. pringlei. The
      relatively recent origin of the allopolyploids likely played a role in the
      near-absence of major changes in the repetitive fraction of the polyploids'
      genomes. The repetitive elements most affected by the postpolyploidization
      changes represented retrotransposons of the Ty1-copia lineage Maximus and, to a
      lesser extent, also Athila elements of Ty3-gypsy family.
FAU - Mccann, Jamie
AU  - Mccann J
AD  - Department of Botany and Biodiversity Research, University of Vienna, Rennweg 14,
      A-1030 Vienna, Austria.
FAU - Jang, Tae-Soo
AU  - Jang TS
AD  - Department of Botany and Biodiversity Research, University of Vienna, Rennweg 14,
      A-1030 Vienna, Austria.
AD  - Department of Biology, College of Bioscience and Biotechnology, Chungnam National
      University, Daejeon 34134, South Korea.
FAU - Macas, Jiri
AU  - Macas J
AD  - Biology Centre, Czech Academy of Sciences, Institute of Plant Molecular Biology, 
      Branisovska 31, CZ-37005, Ceske Budejovice, Czech Republic.
FAU - Schneeweiss, Gerald M
AU  - Schneeweiss GM
AD  - Department of Botany and Biodiversity Research, University of Vienna, Rennweg 14,
      A-1030 Vienna, Austria.
FAU - Matzke, Nicholas J
AU  - Matzke NJ
AD  - Division of Ecology and Evolution, Research School of Biology, Australian
      National University, Canberra, ACT 2601, Australia.
FAU - Novak, Petr
AU  - Novak P
AD  - Biology Centre, Czech Academy of Sciences, Institute of Plant Molecular Biology, 
      Branisovska 31, CZ-37005, Ceske Budejovice, Czech Republic.
FAU - Stuessy, Tod F
AU  - Stuessy TF
AD  - Department of Botany and Biodiversity Research, University of Vienna, Rennweg 14,
      A-1030 Vienna, Austria.
AD  - Herbarium and Department of Evolution, Ecology and Organismal Biology, 1315
      Kinnear Road, The Ohio State University, Columbus, Ohio 43212, USA.
FAU - Villasenor, Jose L
AU  - Villasenor JL
AD  - Department of Botany, UNAM, Tercer Circuito s/n, Ciudad Universitaria, Delegacion
      Coyoacan, MX-04510 Mexico, D.F., Mexico.
FAU - Weiss-Schneeweiss, Hanna
AU  - Weiss-Schneeweiss H
AD  - Department of Botany and Biodiversity Research, University of Vienna, Rennweg 14,
      A-1030 Vienna, Austria.
LA  - eng
SI  - Dryad/10.5061/dryad.dg8q0
GR  - P 25131/Austrian Science Fund FWF/Austria
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Asteraceae/*classification/*genetics
MH  - DNA, Plant/genetics
MH  - *Evolution, Molecular
MH  - Genome, Plant/*genetics
MH  - Phylogeny
MH  - Polyploidy
MH  - Repetitive Sequences, Nucleic Acid/genetics
PMC - PMC6193527
EDAT- 2018/03/22 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/03/22 06:00
PHST- 2017/07/04 00:00 [received]
PHST- 2018/03/15 00:00 [accepted]
PHST- 2018/03/22 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
PHST- 2018/03/22 06:00 [entrez]
AID - 4944071 [pii]
AID - 10.1093/sysbio/syy024 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Nov 1;67(6):1010-1024. doi: 10.1093/sysbio/syy024.

PMID- 29548012
OWN - NLM
STAT- MEDLINE
DCOM- 20181211
LR  - 20181211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 6
DP  - 2018 Nov 1
TI  - Intraspecific Niche Models for Ponderosa Pine (Pinus ponderosa) Suggest Potential
      Variability in Population-Level Response to Climate Change.
PG  - 965-978
LID - 10.1093/sysbio/syy017 [doi]
AB  - Unique responses to climate change can occur across intraspecific levels,
      resulting in individualistic adaptation or movement patterns among populations
      within a given species. Thus, the need to model potential responses among
      genetically distinct populations within a species is increasingly recognized.
      However, predictive models of future distributions are regularly fit at the
      species level, often because intraspecific variation is unknown or is identified 
      only within limited sample locations. In this study, we considered the role of
      intraspecific variation to shape the geographic distribution of ponderosa pine
      (Pinus ponderosa), an ecologically and economically important tree species in
      North America. Morphological and genetic variation across the distribution of
      ponderosa pine suggest the need to model intraspecific populations: the two
      varieties (var. ponderosa and var. scopulorum) and several haplotype groups
      within each variety have been shown to occupy unique climatic niches, suggesting 
      populations have distinct evolutionary lineages adapted to different
      environmental conditions. We utilized a recently available, geographically
      widespread dataset of intraspecific variation (haplotypes) for ponderosa pine and
      a recently devised lineage distance modeling approach to derive additional,
      likely intraspecific occurrence locations. We confirmed the relative uniqueness
      of each haplotype-climate relationship using a niche-overlap analysis, and
      developed ecological niche models (ENMs) to project the distribution for two
      varieties and eight haplotypes under future climate forecasts. Future projections
      of haplotype niche distributions generally revealed greater potential range loss 
      than predicted for the varieties. This difference may reflect intraspecific
      responses of distinct evolutionary lineages. However, directional trends are
      generally consistent across intraspecific levels, and include a loss of
      distributional area and an upward shift in elevation. Our results demonstrate the
      utility in modeling intraspecific response to changing climate and they inform
      management and conservation strategies, by identifying haplotypes and geographic 
      areas that may be most at risk, or most secure, under projected climate change.
FAU - Maguire, Kaitlin C
AU  - Maguire KC
AD  - U.S. Geological Survey, Forest and Rangeland Ecosystem Science Center, 970 S.
      Lusk Street Boise, ID 83706, USA.
FAU - Shinneman, Douglas J
AU  - Shinneman DJ
AD  - U.S. Geological Survey, Forest and Rangeland Ecosystem Science Center, 970 S.
      Lusk Street Boise, ID 83706, USA.
FAU - Potter, Kevin M
AU  - Potter KM
AD  - Department of Forestry and Environmental Resources, North Carolina State
      University, Research Triangle Park, NC 27695, USA.
FAU - Hipkins, Valerie D
AU  - Hipkins VD
AD  - National Forest Genetics Laboratory, USDA Forest Service, 2480 Carson Road,
      Placerville, CA 95667, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.jj264
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Climate Change
MH  - *Ecosystem
MH  - *Models, Biological
MH  - Phylogeny
MH  - Pinus ponderosa/genetics/*physiology
EDAT- 2018/03/17 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/03/17 06:00
PHST- 2017/06/16 00:00 [received]
PHST- 2018/03/05 00:00 [accepted]
PHST- 2018/03/17 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
PHST- 2018/03/17 06:00 [entrez]
AID - 4934308 [pii]
AID - 10.1093/sysbio/syy017 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Nov 1;67(6):965-978. doi: 10.1093/sysbio/syy017.

PMID- 29534232
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - The Biogeography of Deep Time Phylogenetic Reticulation.
PG  - 743-744
LID - 10.1093/sysbio/syy019 [doi]
AB  - Most phylogenies are typically represented as purely bifurcating. However, as
      genomic data have become more common in phylogenetic studies, it is not unusual
      to find reticulation among terminal lineages or among internal nodes (deep time
      reticulation; DTR). In these situations, gene flow must have happened in the same
      or adjacent geographic areas for these DTRs to have occurred and therefore
      biogeographic reconstruction should provide similar area estimates for parental
      nodes, provided extinction or dispersal has not eroded these patterns. We examine
      the phylogeny of the widely distributed New World kingsnakes (Lampropeltis),
      determine if DTR is present in this group, and estimate the ancestral area for
      reticulation. Importantly, we develop a new method that uses coalescent
      simulations in a machine learning framework to show conclusively that this
      phylogeny is best represented as reticulating at deeper time. Using joint
      probabilities of ancestral area reconstructions on the bifurcating parental
      lineages from the reticulating node, we show that this reticulation likely
      occurred in northwestern Mexico/southwestern US, and subsequently, led to the
      diversification of the Mexican kingsnakes. This region has been previously
      identified as an area important for understanding speciation and secondary
      contact with gene flow in snakes and other squamates. This research shows that
      phylogenetic reticulation is common, even in well-studied groups, and that the
      geographic scope of ancient hybridization is recoverable.
FAU - Burbrink, Frank T
AU  - Burbrink FT
AD  - Department of Herpetology, The American Museum of Natural History, 79th Street at
      Central Park West, New York, NY 10024, USA.
FAU - Gehara, Marcelo
AU  - Gehara M
AD  - Department of Herpetology, The American Museum of Natural History, 79th Street at
      Central Park West, New York, NY 10024, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.4qs50
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Colubridae/classification/*genetics
MH  - *Evolution, Molecular
MH  - *Hybridization, Genetic
MH  - Mexico
MH  - Models, Genetic
MH  - Neural Networks (Computer)
MH  - *Phylogeny
MH  - Southwestern United States
EDAT- 2018/03/14 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/03/14 06:00
PHST- 2017/09/18 00:00 [received]
PHST- 2018/03/05 00:00 [accepted]
PHST- 2018/03/14 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/03/14 06:00 [entrez]
AID - 4925328 [pii]
AID - 10.1093/sysbio/syy019 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):743-744. doi: 10.1093/sysbio/syy019.

PMID- 29528459
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - Molecular Phylogeny and Dating Reveal a Terrestrial Origin in the Early
      Carboniferous for Ascaridoid Nematodes.
PG  - 888-900
LID - 10.1093/sysbio/syy018 [doi]
AB  - Ascaridoids are among the commonest groups of zooparasitic nematodes (roundworms)
      and occur in the alimentary canal of all major vertebrate groups, including
      humans. They have an extremely high diversity and are of major socio-economic
      importance. However, their evolutionary history remains poorly known. Herein, we 
      performed a comprehensive phylogenetic analysis of the Ascaridoidea. Our results 
      divided the Ascaridoidea into six monophyletic major clades, i.e., the
      Heterocheilidae, Acanthocheilidae, Anisakidae, Ascarididae, Toxocaridae, and
      Raphidascarididae, among which the Heterocheilidae, rather than the
      Acanthocheilidae, represents the sister clade to the remaining ascaridoids. The
      phylogeny was calibrated using an approach that involves time priors from fossils
      of the co-evolving hosts, and dates the common ancestor of the Ascaridoidea back 
      to the Early Carboniferous (approximately 360.47-325.27 Ma). The divergence dates
      and ancestral host types indicated by our study suggest that members of the
      Ascaridoidea first parasitized terrestrial tetrapods, and subsequently, extended 
      their host range to elasmobranchs and teleosts. We also propose that the
      fundamental terrestrial-aquatic switches of these nematodes were affected by
      changes in sea-level during the Triassic to the Early Cretaceous.
FAU - Li, Liang
AU  - Li L
AD  - Key Laboratory of Animal Physiology, Biochemistry and Molecular Biology of Hebei 
      Province, College of Life Sciences, Hebei Normal University, 050024 Shijiazhuang,
      Hebei Province, P. R. China.
FAU - Lu, Liang
AU  - Lu L
AD  - Key Laboratory of Animal Physiology, Biochemistry and Molecular Biology of Hebei 
      Province, College of Life Sciences, Hebei Normal University, 050024 Shijiazhuang,
      Hebei Province, P. R. China.
AD  - Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology,
      Chinese Academy of Sciences, 1 Beichen West Rd., Chaoyang District, Beijing
      100101, China.
FAU - Nadler, Steven A
AU  - Nadler SA
AD  - Department of Entomology and Nematology, University of California, Davis, CA
      95616, USA.
FAU - Gibson, David I
AU  - Gibson DI
AD  - Department of Life Sciences, Natural History Museum, Cromwell Road, London SW7
      5BD, UK.
FAU - Zhang, Lu-Ping
AU  - Zhang LP
AD  - Key Laboratory of Animal Physiology, Biochemistry and Molecular Biology of Hebei 
      Province, College of Life Sciences, Hebei Normal University, 050024 Shijiazhuang,
      Hebei Province, P. R. China.
FAU - Chen, Hui-Xia
AU  - Chen HX
AD  - Key Laboratory of Animal Physiology, Biochemistry and Molecular Biology of Hebei 
      Province, College of Life Sciences, Hebei Normal University, 050024 Shijiazhuang,
      Hebei Province, P. R. China.
FAU - Zhao, Wen-Ting
AU  - Zhao WT
AD  - Key Laboratory of Animal Physiology, Biochemistry and Molecular Biology of Hebei 
      Province, College of Life Sciences, Hebei Normal University, 050024 Shijiazhuang,
      Hebei Province, P. R. China.
FAU - Guo, Yan-Ning
AU  - Guo YN
AD  - Key Laboratory of Animal Physiology, Biochemistry and Molecular Biology of Hebei 
      Province, College of Life Sciences, Hebei Normal University, 050024 Shijiazhuang,
      Hebei Province, P. R. China.
LA  - eng
SI  - Dryad/10.5061/dryad.28g16
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Ascaridoidea/classification/*genetics
MH  - *Biological Evolution
MH  - Evolution, Molecular
MH  - Genes, Mitochondrial
MH  - *Phylogeny
EDAT- 2018/03/13 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/03/13 06:00
PHST- 2017/08/08 00:00 [received]
PHST- 2018/03/05 00:00 [accepted]
PHST- 2018/03/13 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/03/13 06:00 [entrez]
AID - 4924882 [pii]
AID - 10.1093/sysbio/syy018 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):888-900. doi: 10.1093/sysbio/syy018.

PMID- 29514307
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20190701
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 4
DP  - 2018 Jul 1
TI  - Inferring Phylogenetic Networks Using PhyloNet.
PG  - 735-740
LID - 10.1093/sysbio/syy015 [doi]
AB  - PhyloNet was released in 2008 as a software package for representing and
      analyzing phylogenetic networks. At the time of its release, the main
      functionalities in PhyloNet consisted of measures for comparing network
      topologies and a single heuristic for reconciling gene trees with a species tree.
      Since then, PhyloNet has grown significantly. The software package now includes a
      wide array of methods for inferring phylogenetic networks from data sets of
      unlinked loci while accounting for both reticulation (e.g., hybridization) and
      incomplete lineage sorting. In particular, PhyloNet now allows for maximum
      parsimony, maximum likelihood, and Bayesian inference of phylogenetic networks
      from gene tree estimates. Furthermore, Bayesian inference directly from sequence 
      data (sequence alignments or biallelic markers) is implemented. Maximum parsimony
      is based on an extension of the "minimizing deep coalescences" criterion to
      phylogenetic networks, whereas maximum likelihood and Bayesian inference are
      based on the multispecies network coalescent. All methods allow for multiple
      individuals per species. As computing the likelihood of a phylogenetic network is
      computationally hard, PhyloNet allows for evaluation and inference of networks
      using a pseudolikelihood measure. PhyloNet summarizes the results of the various 
      analyzes and generates phylogenetic networks in the extended Newick format that
      is readily viewable by existing visualization software.
FAU - Wen, Dingqiao
AU  - Wen D
AD  - Computer Science.
FAU - Yu, Yun
AU  - Yu Y
AD  - Computer Science.
FAU - Zhu, Jiafan
AU  - Zhu J
AD  - Computer Science.
FAU - Nakhleh, Luay
AU  - Nakhleh L
AD  - Computer Science.
AD  - BioSciences, Rice University, 6100 Main Street, Houston, TX 77005, USA.
LA  - eng
GR  - R01 LM009494/LM/NLM NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - *Evolution, Molecular
MH  - Hybridization, Genetic
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - *Software
PMC - PMC6005058
EDAT- 2018/03/08 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/03/08 06:00
PHST- 2017/12/21 00:00 [received]
PHST- 2018/02/23 00:00 [accepted]
PHST- 2018/03/08 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/03/08 06:00 [entrez]
AID - 4921127 [pii]
AID - 10.1093/sysbio/syy015 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jul 1;67(4):735-740. doi: 10.1093/sysbio/syy015.

PMID- 29481653
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - When Darwin's Special Difficulty Promotes Diversification in Insects.
PG  - 873-887
LID - 10.1093/sysbio/syy014 [doi]
AB  - Eusociality, Darwin's special difficulty, has been widely investigated but
      remains a topic of great debate in organismal biology. Eusocial species challenge
      existing theories, and the impact of highly integrated societies on
      diversification dynamics is controversial with opposing assertions and hypotheses
      in the literature. Here, using phylogenetic approaches in termites-the first
      group that has evolved eusociality-we assessed the fundamental prediction that
      eusocial lineages have higher diversification rates than non-eusocial clades. We 
      found multiple lines of evidence that eusociality provided higher diversification
      as compared to non-eusociality. This is particularly exacerbated for eusocial
      species with "true" workers as compared to species with "false" workers. Because 
      most species with "true" workers have an entirely prokaryotic microbiota, the
      latter feature is also related to higher diversification rates, but it should be 
      investigated further, notably in relation to angiosperm diversification. Overall,
      this study suggests that societies with "true" workers are not only more
      successful at ecological timescales but also over millions of years, which
      further implies that both organism- and species-level traits act on species
      selection.
FAU - Legendre, Frederic
AU  - Legendre F
AD  - Museum national d'Histoire naturelle, UMR 7205 Institut de Systematique,
      Evolution, Biodiversite ISYEB - UMR 7205 - MNHN CNRS UPMC EPHE, Sorbonne
      Universites, 57 rue Cuvier, 75231 Paris Cedex 05, France.
FAU - Condamine, Fabien L
AU  - Condamine FL
AD  - CNRS, UMR 5554 Institut des Sciences de l'Evolution (Universite de Montpellier), 
      Place Eugene Bataillon, 34095 Montpellier, France.
LA  - eng
SI  - Dryad/10.5061/dryad.2tg34
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - *Genetic Speciation
MH  - Isoptera/*physiology
MH  - Phenotype
MH  - Phylogeny
MH  - Social Behavior
EDAT- 2018/02/27 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/02/27 06:00
PHST- 2017/04/10 00:00 [received]
PHST- 2018/02/14 00:00 [accepted]
PHST- 2018/02/27 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/02/27 06:00 [entrez]
AID - 4898374 [pii]
AID - 10.1093/sysbio/syy014 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):873-887. doi: 10.1093/sysbio/syy014.

PMID- 29471536
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - Variation Across Mitochondrial Gene Trees Provides Evidence for Systematic Error:
      How Much Gene Tree Variation Is Biological?
PG  - 847-860
LID - 10.1093/sysbio/syy013 [doi]
AB  - The use of large genomic data sets in phylogenetics has highlighted extensive
      topological variation across genes. Much of this discordance is assumed to result
      from biological processes. However, variation among gene trees can also be a
      consequence of systematic error driven by poor model fit, and the relative
      importance of biological vs. methodological factors in explaining gene tree
      variation is a major unresolved question. Using mitochondrial genomes to control 
      for biological causes of gene tree variation, we estimate the extent of gene tree
      discordance driven by systematic error and employ posterior prediction to
      highlight the role of model fit in producing this discordance. We find that the
      amount of discordance among mitochondrial gene trees is similar to the amount of 
      discordance found in other studies that assume only biological causes of
      variation. This similarity suggests that the role of systematic error in
      generating gene tree variation is underappreciated and critical evaluation of fit
      between assumed models and the data used for inference is important for the
      resolution of unresolved phylogenetic questions.
FAU - Richards, Emilie J
AU  - Richards EJ
AD  - Department of Biology, University of Hawai'i, 2538 McCarthy Mall, Edmondson Hall 
      2016, Honolulu, HI 96822, USA.
AD  - Department of Biology, University of North Carolina, 120 South Road, Coker Hall
      CB 3280 Chapel Hill, NC 27599, USA.
FAU - Brown, Jeremy M
AU  - Brown JM
AD  - Department of Biological Sciences and Museum of Natural Science, Louisiana State 
      University, 202 Life Sciences Building, Baton Rouge, LA 70803, USA.
FAU - Barley, Anthony J
AU  - Barley AJ
AD  - Department of Biology, University of Hawai'i, 2538 McCarthy Mall, Edmondson Hall 
      2016, Honolulu, HI 96822, USA.
FAU - Chong, Rebecca A
AU  - Chong RA
AD  - Department of Biology, University of Hawai'i, 2538 McCarthy Mall, Edmondson Hall 
      2016, Honolulu, HI 96822, USA.
FAU - Thomson, Robert C
AU  - Thomson RC
AD  - Department of Biology, University of Hawai'i, 2538 McCarthy Mall, Edmondson Hall 
      2016, Honolulu, HI 96822, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.hj07m
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Genome, Mitochondrial
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Vertebrates/*classification/genetics
EDAT- 2018/02/23 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/02/23 06:00
PHST- 2017/08/01 00:00 [received]
PHST- 2018/02/15 00:00 [accepted]
PHST- 2018/02/23 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/02/23 06:00 [entrez]
AID - 4877124 [pii]
AID - 10.1093/sysbio/syy013 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):847-860. doi: 10.1093/sysbio/syy013.

PMID- 29471501
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - Contemporary Ecological Interactions Improve Models of Past Trait Evolution.
PG  - 861-872
LID - 10.1093/sysbio/syy012 [doi]
AB  - Despite the fact that natural selection underlies both traits and interactions,
      evolutionary models often neglect that ecological interactions may, and in many
      cases do, influence the evolution of traits. Herein, we explore the
      interdependence of ecological interactions and functional traits in the
      pollination associations of hawkmoths and flowering plants. Specifically, we
      develop an adaptation of the Ornstein-Uhlenbeck model of trait evolution that
      allows us to study the influence of plant corolla depth and observed
      hawkmoth-plant interactions on the evolution of hawkmoth proboscis length. Across
      diverse modelling scenarios, we find that the inclusion of contemporary
      interactions can provide a better description of trait evolution than the null
      expectation. Moreover, we show that the pollination interactions provide
      more-likely models of hawkmoth trait evolution when interactions are considered
      at increasingly fine-scale groups of hawkmoths. Finally, we demonstrate how the
      results of best-fit modeling approaches can implicitly support the association
      between interactions and trait evolution that our method explicitly examines. In 
      showing that contemporary interactions can provide insight into the historical
      evolution of hawkmoth proboscis length, we demonstrate the clear utility of
      incorporating additional ecological information to models designed to study past 
      trait evolution.
FAU - Hutchinson, Matthew C
AU  - Hutchinson MC
AD  - Centre for Integrative Ecology, School of Biological Sciences, University of
      Canterbury, Private Bag 4800, Christchurch 8140, New Zealand.
AD  - Department of Ecology and Evolutionary Biology, Princeton University, Princeton, 
      NJ 08544, USA.
FAU - Gaiarsa, Marilia P
AU  - Gaiarsa MP
AD  - Centre for Integrative Ecology, School of Biological Sciences, University of
      Canterbury, Private Bag 4800, Christchurch 8140, New Zealand.
AD  - Departamento de Ecologia, Instituto de Biociencias, Universidade de Sao Paulo,
      11294, 05422-970 Sao Paulo, Brazil.
FAU - Stouffer, Daniel B
AU  - Stouffer DB
AD  - Centre for Integrative Ecology, School of Biological Sciences, University of
      Canterbury, Private Bag 4800, Christchurch 8140, New Zealand.
LA  - eng
SI  - Dryad/10.5061/dryad.3vb73
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Flowers/anatomy &amp; histology
MH  - Magnoliopsida/*anatomy &amp; histology
MH  - Models, Biological
MH  - Moths/*anatomy &amp; histology
MH  - Phenotype
MH  - Pollination
EDAT- 2018/02/23 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/02/23 06:00
PHST- 2017/04/20 00:00 [received]
PHST- 2018/02/10 00:00 [accepted]
PHST- 2018/02/23 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/02/23 06:00 [entrez]
AID - 4877123 [pii]
AID - 10.1093/sysbio/syy012 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):861-872. doi: 10.1093/sysbio/syy012.

PMID- 29462495
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - Comparison of Methods for Molecular Species Delimitation Across a Range of
      Speciation Scenarios.
PG  - 830-846
LID - 10.1093/sysbio/syy011 [doi]
AB  - Species are fundamental units in biological research and can be defined on the
      basis of various operational criteria. There has been growing use of molecular
      approaches for species delimitation. Among the most widely used methods, the
      generalized mixed Yule-coalescent (GMYC) and Poisson tree processes (PTP) were
      designed for the analysis of single-locus data but are often applied to
      concatenations of multilocus data. In contrast, the Bayesian multispecies
      coalescent approach in the software Bayesian Phylogenetics and Phylogeography
      (BPP) explicitly models the evolution of multilocus data. In this study, we
      compare the performance of GMYC, PTP, and BPP using synthetic data generated by
      simulation under various speciation scenarios. We show that in the absence of
      gene flow, the main factor influencing the performance of these methods is the
      ratio of population size to divergence time, while number of loci and sample size
      per species have smaller effects. Given appropriate priors and correct guide
      trees, BPP shows lower rates of species overestimation and underestimation, and
      is generally robust to various potential confounding factors except high levels
      of gene flow. The single-threshold GMYC and the best strategy that we identified 
      in PTP generally perform well for scenarios involving more than a single putative
      species when gene flow is absent, but PTP outperforms GMYC when fewer species are
      involved. Both methods are more sensitive than BPP to the effects of gene flow
      and potential confounding factors. Case studies of bears and bees further
      validate some of the findings from our simulation study, and reveal the
      importance of using an informed starting point for molecular species
      delimitation. Our results highlight the key factors affecting the performance of 
      molecular species delimitation, with potential benefits for using these methods
      within an integrative taxonomic framework.
FAU - Luo, Arong
AU  - Luo A
AD  - Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology,
      Chinese Academy of Sciences, Beijing 100101, China.
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney, New
      South Wales 2006, Australia.
FAU - Ling, Cheng
AU  - Ling C
AD  - Department of Computer Science and Technology, College of Information Science and
      Technology, Beijing University of Chemical Technology, Beijing 100029, China.
FAU - Ho, Simon Y W
AU  - Ho SYW
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney, New
      South Wales 2006, Australia.
FAU - Zhu, Chao-Dong
AU  - Zhu CD
AD  - Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology,
      Chinese Academy of Sciences, Beijing 100101, China.
AD  - College of Life Sciences, University of Chinese Academy of Sciences, Beijing
      100049, China.
LA  - eng
SI  - Dryad/10.5061/dryad.739bs
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bees/*classification
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Gene Flow
MH  - Population Density
MH  - Software
MH  - Ursidae/*classification
PMC - PMC6101526
EDAT- 2018/02/21 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/02/21 06:00
PHST- 2017/06/25 00:00 [received]
PHST- 2018/02/10 00:00 [accepted]
PHST- 2018/02/21 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/02/21 06:00 [entrez]
AID - 4866060 [pii]
AID - 10.1093/sysbio/syy011 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):830-846. doi: 10.1093/sysbio/syy011.

PMID- 29462459
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 5
DP  - 2018 Sep 1
TI  - Evidence that Myotis lucifugus "Subspecies" are Five Nonsister Species, Despite
      Gene Flow.
PG  - 756-769
LID - 10.1093/sysbio/syy010 [doi]
AB  - While genetic exchange between nonsister species was traditionally considered to 
      be rare in mammals, analyses of molecular data in multiple systems suggest that
      it may be common. Interspecific gene flow, if present, is problematic for
      phylogenetic inference, particularly for analyses near the species level. Here,
      we explore how to detect and account for gene flow during phylogeny estimation
      using data from a clade of North American Myotis bats where previous results have
      led researchers to suspect that gene flow among lineages is present. Initial
      estimates of phylogenetic networks and species trees indicate that subspecies
      described within Myotis lucifugus are paraphyletic. In order to explore the
      extent to which gene flow is likely to interfere with phylogeny estimation, we
      use posterior predictive simulation and a novel Approximate Bayesian Computation 
      approach based on gene tree distances. The former indicates that the species tree
      model is a poor fit to the data, and the latter provides evidence that a species 
      tree with gene flow is a better fit. Taken together, we present evidence that the
      currently recognized M. lucifugus subspecies are paraphyletic, exchange alleles
      with other Myotis species in regions of secondary contact, and should be
      considered independent evolutionary lineages despite their morphological
      similarity.
FAU - Morales, Ariadna E
AU  - Morales AE
AD  - Department of Evolution, Ecology and Organismal Biology, Ohio State University,
      318 W. 12th Avenue, Columbus, OH 43210, USA.
FAU - Carstens, Bryan C
AU  - Carstens BC
AD  - Department of Evolution, Ecology and Organismal Biology, Ohio State University,
      318 W. 12th Avenue, Columbus, OH 43210, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.9d6b2
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Canada
MH  - Chiroptera/classification/*genetics
MH  - Evolution, Molecular
MH  - *Gene Flow
MH  - *Genetic Speciation
MH  - Models, Genetic
MH  - *Phylogeny
MH  - United States
EDAT- 2018/02/21 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/02/21 06:00
PHST- 2017/09/13 00:00 [received]
PHST- 2018/02/10 00:00 [accepted]
PHST- 2018/02/21 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/02/21 06:00 [entrez]
AID - 4866059 [pii]
AID - 10.1093/sysbio/syy010 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Sep 1;67(5):756-769. doi: 10.1093/sysbio/syy010.

PMID- 29462409
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 4
DP  - 2018 Jul 1
TI  - The Behavior of Metropolis-Coupled Markov Chains When Sampling Rugged
      Phylogenetic Distributions.
PG  - 729-734
LID - 10.1093/sysbio/syy008 [doi]
AB  - Bayesian phylogenetic inference relies on the use of Markov chain Monte Carlo
      (MCMC) to provide numerical approximations of high-dimensional integrals and
      estimate posterior probabilities. However, MCMC performs poorly when posteriors
      are very rugged (i.e., regions of high posterior density are separated by regions
      of low posterior density). One technique that has become popular for improving
      numerical estimates from MCMC when distributions are rugged is Metropolis
      coupling (MC$^3$). In MC$^3$, additional chains are employed to sample flattened 
      transformations of the posterior and improve mixing. Here, we highlight several
      underappreciated behaviors of MC3. Notably, estimated posterior probabilities may
      be incorrect but appear to converge, when individual chains do not mix well,
      despite different chains sampling trees from all relevant areas in tree space.
      Counterintuitively, such behavior can be more difficult to diagnose with
      increased numbers of chains. We illustrate these surprising behaviors of MC$^3$
      using a simple, non-phylogenetic example and phylogenetic examples involving both
      constrained and unconstrained analyses. To detect and mitigate the effects of
      these behaviors, we recommend increasing the number of independent analyses and
      varying the temperature of the hottest chain in current versions of Bayesian
      phylogenetic software. Convergence diagnostics based on the behavior of the
      hottest chain may also help detect these behaviors and could form a useful
      addition to future software releases.
FAU - Brown, Jeremy M
AU  - Brown JM
AD  - Department of Biological Sciences and Museum of Natural Science, Louisiana State 
      University, 202 Life Sciences Building, Baton Rouge, LA 70803, USA.
FAU - Thomson, Robert C
AU  - Thomson RC
AD  - Department of Biology, University of Hawai'i 2538 McCarthy Mall, Edmondson Hall
      Room 216, Honolulu, HI 96822, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.584m3
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - *Markov Chains
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Probability
EDAT- 2018/02/21 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/02/21 06:00
PHST- 2017/11/03 00:00 [received]
PHST- 2018/02/06 00:00 [accepted]
PHST- 2018/02/21 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/02/21 06:00 [entrez]
AID - 4866058 [pii]
AID - 10.1093/sysbio/syy008 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jul 1;67(4):729-734. doi: 10.1093/sysbio/syy008.

PMID- 29438538
OWN - NLM
STAT- MEDLINE
DCOM- 20181211
LR  - 20181211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 6
DP  - 2018 Nov 1
TI  - Testing the Role of the Red Queen and Court Jester as Drivers of the
      Macroevolution of Apollo Butterflies.
PG  - 940-964
LID - 10.1093/sysbio/syy009 [doi]
AB  - In macroevolution, the Red Queen (RQ) model posits that biodiversity dynamics
      depend mainly on species-intrinsic biotic factors such as interactions among
      species or life-history traits, while the Court Jester (CJ) model states that
      extrinsic environmental abiotic factors have a stronger role. Until recently, a
      lack of relevant methodological approaches has prevented the unraveling of
      contributions from these 2 types of factors to the evolutionary history of a
      lineage. Herein, we take advantage of the rapid development of new macroevolution
      models that tie diversification rates to changes in paleoenvironmental
      (extrinsic) and/or biotic (intrinsic) factors. We inferred a robust and
      fully-sampled species-level phylogeny, as well as divergence times and ancestral 
      geographic ranges, and related these to the radiation of Apollo butterflies
      (Parnassiinae) using both extant (molecular) and extinct (fossil/morphological)
      evidence. We tested whether their diversification dynamics are better explained
      by an RQ or CJ hypothesis, by assessing whether speciation and extinction were
      mediated by diversity-dependence (niche filling) and clade-dependent host-plant
      association (RQ) or by large-scale continuous changes in extrinsic factors such
      as climate or geology (CJ). For the RQ hypothesis, we found significant
      differences in speciation rates associated with different host-plants but
      detected no sign of diversity-dependence. For CJ, the role of Himalayan-Tibetan
      building was substantial for biogeography but not a driver of high speciation,
      while positive dependence between warm climate and speciation/extinction was
      supported by continuously varying maximum-likelihood models. We find that rather 
      than a single factor, the joint effect of multiple factors (biogeography, species
      traits, environmental drivers, and mass extinction) is responsible for current
      diversity patterns and that the same factor might act differently across clades, 
      emphasizing the notion of opportunity. This study confirms the importance of the 
      confluence of several factors rather than single explanations in modeling
      diversification within lineages.
FAU - Condamine, Fabien L
AU  - Condamine FL
AD  - CNRS, UMR 5554 Institut des Sciences de l'Evolution (Universite de Montpellier|
      CNRS IRD| EPHE), Place Eugene Bataillon, 34095 Montpellier, France.
AD  - Department of Biodiversity and Conservation, Real Jardin Botanico, CSIC, Plaza de
      Murillo, 2, 28014 Madrid, Spain.
AD  - Department of Biological Sciences, University of Alberta, Edmonton T6G 2E9,
      Alberta, Canada.
FAU - Rolland, Jonathan
AU  - Rolland J
AD  - Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne,
      Switzerland.
FAU - Hohna, Sebastian
AU  - Hohna S
AD  - Division of Evolutionary Biology, Ludwig-Maximilian-Universitat Munchen,
      Grosshaderner Strasse 2, Planegg-Martinsried 82152, Germany.
FAU - Sperling, Felix A H
AU  - Sperling FAH
AD  - Department of Biological Sciences, University of Alberta, Edmonton T6G 2E9,
      Alberta, Canada.
FAU - Sanmartin, Isabel
AU  - Sanmartin I
AD  - Department of Biodiversity and Conservation, Real Jardin Botanico, CSIC, Plaza de
      Murillo, 2, 28014 Madrid, Spain.
LA  - eng
SI  - Dryad/10.5061/dryad.32bp4
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - *Biological Evolution
MH  - Butterflies/*classification/genetics
MH  - Genetic Speciation
MH  - *Models, Biological
MH  - Phylogeny
EDAT- 2018/02/14 06:00
MHDA- 2018/12/12 06:00
CRDT- 2018/02/14 06:00
PHST- 2016/06/25 00:00 [received]
PHST- 2018/02/06 00:00 [accepted]
PHST- 2018/02/14 06:00 [pubmed]
PHST- 2018/12/12 06:00 [medline]
PHST- 2018/02/14 06:00 [entrez]
AID - 4847851 [pii]
AID - 10.1093/sysbio/syy009 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Nov 1;67(6):940-964. doi: 10.1093/sysbio/syy009.

PMID- 29432602
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 4
DP  - 2018 Jul 1
TI  - Modeling the Growth and Decline of Pathogen Effective Population Size Provides
      Insight into Epidemic Dynamics and Drivers of Antimicrobial Resistance.
PG  - 719-728
LID - 10.1093/sysbio/syy007 [doi]
AB  - Nonparametric population genetic modeling provides a simple and flexible approach
      for studying demographic history and epidemic dynamics using pathogen sequence
      data. Existing Bayesian approaches are premised on stochastic processes with
      stationary increments which may provide an unrealistic prior for epidemic
      histories which feature extended period of exponential growth or decline. We show
      that nonparametric models defined in terms of the growth rate of the effective
      population size can provide a more realistic prior for epidemic history. We
      propose a nonparametric autoregressive model on the growth rate as a prior for
      effective population size, which corresponds to the dynamics expected under many 
      epidemic situations. We demonstrate the use of this model within a Bayesian
      phylodynamic inference framework. Our method correctly reconstructs trends of
      epidemic growth and decline from pathogen genealogies even when genealogical data
      are sparse and conventional skyline estimators erroneously predict stable
      population size. We also propose a regression approach for relating growth rates 
      of pathogen effective population size and time-varying variables that may impact 
      the replicative fitness of a pathogen. The model is applied to real data from
      rabies virus and Staphylococcus aureus epidemics. We find a close correspondence 
      between the estimated growth rates of a lineage of methicillin-resistant S.
      aureus and population-level prescription rates of $\beta$-lactam antibiotics. The
      new models are implemented in an open source R package called skygrowth which is 
      available at https://github.com/mrc-ide/skygrowth.
FAU - Volz, Erik M
AU  - Volz EM
AD  - Department of Infectious Disease Epidemiology, Imperial College London, Norfolk
      Place, W2 1PG, UK.
FAU - Didelot, Xavier
AU  - Didelot X
AD  - Department of Infectious Disease Epidemiology, Imperial College London, Norfolk
      Place, W2 1PG, UK.
LA  - eng
SI  - Dryad/10.5061/dryad.9qh7t9t
GR  - U01 GM110749/GM/NIGMS NIH HHS/United States
GR  - MR/K010174/Medical Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Anti-Bacterial Agents)
RN  - 0 (beta-Lactams)
SB  - IM
MH  - Anti-Bacterial Agents/administration &amp; dosage/pharmacology
MH  - Bayes Theorem
MH  - Methicillin-Resistant Staphylococcus aureus/drug effects/physiology
MH  - *Models, Genetic
MH  - Population Density
MH  - Population Growth
MH  - Rabies/*virology
MH  - Rabies virus/*physiology
MH  - Staphylococcal Infections/*microbiology
MH  - Staphylococcus aureus/*physiology
MH  - Statistics, Nonparametric
MH  - beta-Lactams/administration &amp; dosage/pharmacology
PMC - PMC6005154
EDAT- 2018/02/13 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/02/13 06:00
PHST- 2017/10/19 00:00 [received]
PHST- 2018/02/04 00:00 [accepted]
PHST- 2018/02/13 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/02/13 06:00 [entrez]
AID - 4844079 [pii]
AID - 10.1093/sysbio/syy007 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jul 1;67(4):719-728. doi: 10.1093/sysbio/syy007.

PMID- 29385563
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 4
DP  - 2018 Jul 1
TI  - Phenotypic and Genetic Structure Support Gene Flow Generating Gene Tree
      Discordances in an Amazonian Floodplain Endemic Species.
PG  - 700-718
LID - 10.1093/sysbio/syy004 [doi]
AB  - Before populations become independent evolutionary lineages, the effects of micro
      evolutionary processes tend to generate complex scenarios of diversification that
      may affect phylogenetic reconstruction. Not accounting for gene flow in species
      tree estimates can directly impact topology, effective population sizes and
      branch lengths, and the resulting estimation errors are still poorly understood
      in wild populations. In this study, we used an integrative approach, including
      sequence capture of ultra-conserved elements (UCEs), mtDNA Sanger sequencing and 
      morphological data to investigate species limits and phylogenetic relationships
      in face of gene flow in an Amazonian endemic species (Myrmoborus lugubris: Aves).
      We used commonly implemented species tree and model-based approaches to
      understand the potential effects of gene flow in phylogenetic reconstructions.
      The genetic structure observed was congruent with the four recognized subspecies 
      of M. lugubris. Morphological and UCEs data supported the presence of a wide
      hybrid zone between M. l. femininus from the Madeira river and M. l. lugubris
      from the Middle and lower Amazon river, which were recovered as sister taxa by
      species tree methods. When fitting gene flow into simulated demographic models
      with different topologies, the best-fit model indicated these two taxa as
      non-sister lineages, a finding that is in agreement with the results of
      mitochondrial and morphological analyses. Our results demonstrated that failing
      to account for gene flow when estimating phylogenies at shallow divergence levels
      can generate topological uncertainty, which can nevertheless be statistically
      well supported, and that model testing approaches using simulated data can be
      useful tools to test alternative phylogenetic hypotheses.
FAU - Thom, Gregory
AU  - Thom G
AD  - Departamento de Genetica e Biologia Evolutiva, Universidade de Sao Paulo, Rua do 
      Matao, 277, Cidade Universitaria, Sao Paulo, SP 05508-090, Brazil.
FAU - Amaral, Fabio Raposo Do
AU  - Amaral FRD
AD  - Instituto de Ciencias Ambientais, Quimicas e Farmaceuticas, Laboratorio de
      Genetica Evolutiva, Universidade Federal de Sao Paulo, Rua Professor Artur
      Riedel, 275, Diadema, SP 09972-270, Brazil.
FAU - Hickerson, Michael J
AU  - Hickerson MJ
AD  - Department of Biology, Marshak Science Building, City College of New York, 160,
      Convent Avenue, 10031 New York, NY, USA.
FAU - Aleixo, Alexandre
AU  - Aleixo A
AD  - Departamento de Ornitologia, Museu Paraense Emilio Goeldi (MPEG), Caixa Postal
      399, Belem, PA 66040-170, Brazil.
FAU - Araujo-Silva, Lucas E
AU  - Araujo-Silva LE
AD  - Departamento de Ornitologia, Museu Paraense Emilio Goeldi (MPEG), Caixa Postal
      399, Belem, PA 66040-170, Brazil.
FAU - Ribas, Camila C
AU  - Ribas CC
AD  - Coordenacao de biodiversidade, Instituto Nacional de Pesquisas da Amazonia
      (INPA), Av. Andre Araujo 2936, Manaus, AM 69060-001, Brazil.
FAU - Choueri, Erik
AU  - Choueri E
AD  - Coordenacao de biodiversidade, Instituto Nacional de Pesquisas da Amazonia
      (INPA), Av. Andre Araujo 2936, Manaus, AM 69060-001, Brazil.
FAU - Miyaki, Cristina Y
AU  - Miyaki CY
AD  - Departamento de Genetica e Biologia Evolutiva, Universidade de Sao Paulo, Rua do 
      Matao, 277, Cidade Universitaria, Sao Paulo, SP 05508-090, Brazil.
LA  - eng
SI  - Dryad/10.5061/dryad.13b88
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Brazil
MH  - DNA, Mitochondrial/analysis
MH  - *Evolution, Molecular
MH  - *Gene Flow
MH  - *Genetic Speciation
MH  - Genotype
MH  - Models, Genetic
MH  - Passeriformes/*classification/genetics
MH  - Phenotype
MH  - *Phylogeny
EDAT- 2018/02/01 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/02/01 06:00
PHST- 2017/04/28 00:00 [received]
PHST- 2018/01/23 00:00 [accepted]
PHST- 2018/02/01 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/02/01 06:00 [entrez]
AID - 4828098 [pii]
AID - 10.1093/sysbio/syy004 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jul 1;67(4):700-718. doi: 10.1093/sysbio/syy004.

PMID- 29385558
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 4
DP  - 2018 Jul 1
TI  - Accounting for Calibration Uncertainty: Bayesian Molecular Dating as a "Doubly
      Intractable" Problem.
PG  - 651-661
LID - 10.1093/sysbio/syy003 [doi]
AB  - This study introduces a new Bayesian technique for molecular dating that
      explicitly accommodates for uncertainty in the phylogenetic position of
      calibrated nodes derived from the analysis of fossil data. The proposed approach 
      thus defines an adequate framework for incorporating expert knowledge and/or
      prior information about the way fossils were collected in the inference of node
      ages. Although it belongs to the class of "node-dating" approaches, this method
      shares interesting properties with "tip-dating" techniques. Yet, it alleviates
      some of the computational and modeling difficulties that hamper tip-dating
      approaches. The influence of fossil data on the probabilistic distribution of
      trees is the crux of the matter considered here. More specifically, among all the
      phylogenies that a tree model (e.g., the birth-death process) generates, only a
      fraction of them "agree" with the fossil data. Bayesian inference under the new
      model requires taking this fraction into account. However, evaluating this
      quantity is difficult in practice. A generic solution to this issue is presented 
      here. The proposed approach relies on a recent statistical technique, the
      so-called exchange algorithm, dedicated to drawing samples from "doubly
      intractable" distributions. A small example illustrates the problem of interest
      and the impact of uncertainty in the placement of calibration constraints in the 
      phylogeny given fossil data. An analysis of land plant sequences and multiple
      fossils further highlights the pertinence of the proposed approach.
FAU - Guindon, Stephane
AU  - Guindon S
AD  - Laboratoire d'Informatique, de Robotique et de Microelectronique de Montpellier, 
      UMR 5506, CNRS, Universite de Montpellier, Montpellier, France.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Calibration
MH  - Embryophyta/*classification
MH  - *Evolution, Molecular
MH  - Fossils
MH  - *Genetic Speciation
MH  - Models, Biological
MH  - *Models, Genetic
EDAT- 2018/02/01 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/02/01 06:00
PHST- 2017/02/06 00:00 [received]
PHST- 2018/01/23 00:00 [accepted]
PHST- 2018/02/01 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/02/01 06:00 [entrez]
AID - 4827614 [pii]
AID - 10.1093/sysbio/syy003 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jul 1;67(4):651-661. doi: 10.1093/sysbio/syy003.

PMID- 29385556
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 4
DP  - 2018 Jul 1
TI  - Inference of Adaptive Shifts for Multivariate Correlated Traits.
PG  - 662-680
LID - 10.1093/sysbio/syy005 [doi]
AB  - To study the evolution of several quantitative traits, the classical phylogenetic
      comparative framework consists of a multivariate random process running along the
      branches of a phylogenetic tree. The Ornstein-Uhlenbeck (OU) process is sometimes
      preferred to the simple Brownian motion (BM) as it models stabilizing selection
      toward an optimum. The optimum for each trait is likely to be changing over the
      long periods of time spanned by large modern phylogenies. Our goal is to
      automatically detect the position of these shifts on a phylogenetic tree, while
      accounting for correlations between traits, which might exist because of
      structural or evolutionary constraints. We show that, in the presence of shifts, 
      phylogenetic Principal Component Analysis fails to decorrelate traits
      efficiently, so that any method aiming at finding shifts needs to deal with
      correlation simultaneously. We introduce here a simplification of the full
      multivariate OU model, named scalar OU, which allows for noncausal correlations
      and is still computationally tractable. We extend the equivalence between the OU 
      and a BM on a rescaled tree to our multivariate framework. We describe an
      Expectation-Maximization (EM) algorithm that allows for a maximum likelihood
      estimation of the shift positions, associated with a new model selection
      criterion, accounting for the identifiability issues for the shift localization
      on the tree. The method, freely available as an R-package (PhylogeneticEM) is
      fast, and can deal with missing values. We demonstrate its efficiency and
      accuracy compared to another state-of-the-art method ($\ell$1ou) on a wide range 
      of simulated scenarios and use this new framework to reanalyze recently gathered 
      data sets on New World Monkeys and Anolis lizards.
FAU - Bastide, Paul
AU  - Bastide P
AD  - Unite Mixte de Recherche Mathematiques et Informatique Appliquees (MIA - Paris), 
      AgroParisTech, Institut National de la Recherche Agronomique (INRA), Universite
      Paris-Saclay, 16 rue Claude Bernard, 75005 Paris, France.
AD  - Unite de Recherche Mathematiques et Informatique Appliquees du Genome a
      l'Environnement (MaIAGE), Institut National de la Recherche Agronomique (INRA),
      Universite Paris-Saclay, Domaine de Vilvert, 78352 Jouy-en-Josas, France.
FAU - Ane, Cecile
AU  - Ane C
AD  - Department of Statistics, University of Wisconsin-Madison, 1300 University
      Avenue, Madison, WI 53706, USA.
AD  - Department of Botany, University of Wisconsin-Madison, 430 Lincoln Drive,
      Madison, WI 53706, USA.
FAU - Robin, Stephane
AU  - Robin S
AD  - Unite Mixte de Recherche Mathematiques et Informatique Appliquees (MIA - Paris), 
      AgroParisTech, Institut National de la Recherche Agronomique (INRA), Universite
      Paris-Saclay, 16 rue Claude Bernard, 75005 Paris, France.
FAU - Mariadassou, Mahendra
AU  - Mariadassou M
AD  - Unite de Recherche Mathematiques et Informatique Appliquees du Genome a
      l'Environnement (MaIAGE), Institut National de la Recherche Agronomique (INRA),
      Universite Paris-Saclay, Domaine de Vilvert, 78352 Jouy-en-Josas, France.
LA  - eng
SI  - Dryad/10.5061/dryad.60t0f
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Adaptation, Biological
MH  - Algorithms
MH  - Animals
MH  - *Biological Evolution
MH  - *Lizards
MH  - *Phenotype
MH  - Phylogeny
MH  - *Platyrrhini
EDAT- 2018/02/01 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/02/01 06:00
PHST- 2017/11/15 00:00 [received]
PHST- 2018/01/23 00:00 [accepted]
PHST- 2018/02/01 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/02/01 06:00 [entrez]
AID - 4827615 [pii]
AID - 10.1093/sysbio/syy005 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jul 1;67(4):662-680. doi: 10.1093/sysbio/syy005.

PMID- 29385552
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 4
DP  - 2018 Jul 1
TI  - Bayesian Divergence-Time Estimation with Genome-Wide Single-Nucleotide
      Polymorphism Data of Sea Catfishes (Ariidae) Supports Miocene Closure of the
      Panamanian Isthmus.
PG  - 681-699
LID - 10.1093/sysbio/syy006 [doi]
AB  - The closure of the Isthmus of Panama has long been considered to be one of the
      best defined biogeographic calibration points for molecular divergence-time
      estimation. However, geological and biological evidence has recently cast doubt
      on the presumed timing of the initial isthmus closure around 3 Ma but has instead
      suggested the existence of temporary land bridges as early as the Middle or Late 
      Miocene. The biological evidence supporting these earlier land bridges was based 
      either on only few molecular markers or on concatenation of genome-wide sequence 
      data, an approach that is known to result in potentially misleading branch
      lengths and divergence times, which could compromise the reliability of this
      evidence. To allow divergence-time estimation with genomic data using the more
      appropriate multispecies coalescent (MSC) model, we here develop a new method
      combining the single-nucleotide polymorphism-based Bayesian species-tree
      inference of the software SNAPP with a molecular clock model that can be
      calibrated with fossil or biogeographic constraints. We validate our approach
      with simulations and use our method to reanalyze genomic data of Neotropical army
      ants (Dorylinae) that previously supported divergence times of Central and South 
      American populations before the isthmus closure around 3 Ma. Our reanalysis with 
      the MSC model shifts all of these divergence times to ages younger than 3 Ma,
      suggesting that the older estimates supporting the earlier existence of temporary
      land bridges were artifacts resulting at least partially from the use of
      concatenation. We then apply our method to a new restriction-site associated
      DNA-sequencing data set of Neotropical sea catfishes (Ariidae) and calibrate
      their species tree with extensive information from the fossil record. We identify
      a series of divergences between groups of Caribbean and Pacific sea catfishes
      around 10 Ma, indicating that processes related to the emergence of the isthmus
      led to vicariant speciation already in the Late Miocene, millions of years before
      the final isthmus closure.
FAU - Stange, Madlen
AU  - Stange M
AD  - Department of Palaeontology and Museum, University of Zurich, Karl-Schmid-Strasse
      4, 8006 Zurich, Switzerland.
AD  - Zoological Institute, University of Basel, Vesalgasse 1, 4051 Basel, Switzerland.
FAU - Sanchez-Villagra, Marcelo R
AU  - Sanchez-Villagra MR
AD  - Department of Palaeontology and Museum, University of Zurich, Karl-Schmid-Strasse
      4, 8006 Zurich, Switzerland.
FAU - Salzburger, Walter
AU  - Salzburger W
AD  - Zoological Institute, University of Basel, Vesalgasse 1, 4051 Basel, Switzerland.
AD  - Centre for Ecological and Evolutionary Synthesis (CEES), Department of
      Biosciences, University of Oslo, 0316 Oslo, Norway.
FAU - Matschiner, Michael
AU  - Matschiner M
AD  - Zoological Institute, University of Basel, Vesalgasse 1, 4051 Basel, Switzerland.
AD  - Centre for Ecological and Evolutionary Synthesis (CEES), Department of
      Biosciences, University of Oslo, 0316 Oslo, Norway.
LA  - eng
SI  - Dryad/10.5061/dryad.f8k84
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Ants/*classification/genetics
MH  - Bayes Theorem
MH  - Caribbean Region
MH  - Catfishes/*classification/genetics
MH  - *Evolution, Molecular
MH  - Pacific Ocean
MH  - Panama
MH  - *Polymorphism, Single Nucleotide
PMC - PMC6005153
EDAT- 2018/02/01 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/02/01 06:00
PHST- 2017/01/23 00:00 [received]
PHST- 2018/01/23 00:00 [accepted]
PHST- 2018/02/01 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/02/01 06:00 [entrez]
AID - 4827616 [pii]
AID - 10.1093/sysbio/syy006 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jul 1;67(4):681-699. doi: 10.1093/sysbio/syy006.

PMID- 29342307
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 4
DP  - 2018 Jul 1
TI  - Using Phylogenomic Data to Explore the Effects of Relaxed Clocks and Calibration 
      Strategies on Divergence Time Estimation: Primates as a Test Case.
PG  - 594-615
LID - 10.1093/sysbio/syy001 [doi]
AB  - Primates have long been a test case for the development of phylogenetic methods
      for divergence time estimation. Despite a large number of studies, however, the
      timing of origination of crown Primates relative to the Cretaceous-Paleogene
      (K-Pg) boundary and the timing of diversification of the main crown groups remain
      controversial. Here, we analysed a data set of 372 taxa (367 Primates and 5
      outgroups, 3.4 million aligned base pairs) that includes nine primate genomes. We
      systematically explore the effect of different interpretations of fossil
      calibrations and molecular clock models on primate divergence time estimates. We 
      find that even small differences in the construction of fossil calibrations can
      have a noticeable impact on estimated divergence times, especially for the oldest
      nodes in the tree. Notably, choice of molecular rate model (autocorrelated or
      independently distributed rates) has an especially strong effect on estimated
      times, with the independent rates model producing considerably more ancient age
      estimates for the deeper nodes in the phylogeny. We implement thermodynamic
      integration, combined with Gaussian quadrature, in the program MCMCTree, and use 
      it to calculate Bayes factors for clock models. Bayesian model selection
      indicates that the autocorrelated rates model fits the primate data substantially
      better, and we conclude that time estimates under this model should be preferred.
      We show that for eight core nodes in the phylogeny, uncertainty in time estimates
      is close to the theoretical limit imposed by fossil uncertainties. Thus, these
      estimates are unlikely to be improved by collecting additional molecular sequence
      data. All analyses place the origin of Primates close to the K-Pg boundary,
      either in the Cretaceous or straddling the boundary into the Palaeogene.
FAU - Reis, Mario Dos
AU  - Reis MD
AD  - School of Biological and Chemical Sciences, Queen Mary University of London, Mile
      End Road, London E1 4NS, UK.
AD  - Department of Genetics, Evolution and Environment, University College London,
      Gower Street, London WC1E 6BT, UK.
FAU - Gunnell, Gregg F
AU  - Gunnell GF
AD  - Division of Fossil Primates, Duke University Lemur Center, Durham, 1013 Broad
      Street, NC 27705, USA.
FAU - Barba-Montoya, Jose
AU  - Barba-Montoya J
AD  - Department of Genetics, Evolution and Environment, University College London,
      Gower Street, London WC1E 6BT, UK.
FAU - Wilkins, Alex
AU  - Wilkins A
AD  - Division of Fossil Primates, Duke University Lemur Center, Durham, 1013 Broad
      Street, NC 27705, USA.
AD  - Department of Anthropology, The Ohio State University, Columbus, OH 43210, USA.
FAU - Yang, Ziheng
AU  - Yang Z
AD  - Department of Genetics, Evolution and Environment, University College London,
      Gower Street, London WC1E 6BT, UK.
FAU - Yoder, Anne D
AU  - Yoder AD
AD  - Department of Biology, Duke University, Durham, NC 27708, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.c020q
GR  - S10 OD018164/OD/NIH HHS/United States
GR  - BB/J009709/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Calibration
MH  - *Evolution, Molecular
MH  - Fossils/anatomy &amp; histology
MH  - *Genome
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Primates/anatomy &amp; histology/*classification/genetics
PMC - PMC6005039
EDAT- 2018/01/18 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/01/18 06:00
PHST- 2017/03/25 00:00 [received]
PHST- 2018/01/05 00:00 [accepted]
PHST- 2018/01/18 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/01/18 06:00 [entrez]
AID - 4802240 [pii]
AID - 10.1093/sysbio/syy001 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jul 1;67(4):594-615. doi: 10.1093/sysbio/syy001.

PMID- 29325126
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 4
DP  - 2018 Jul 1
TI  - Automated Integration of Trees and Traits: A Case Study Using Paired Fin Loss
      Across Teleost Fishes.
PG  - 559-575
LID - 10.1093/sysbio/syx098 [doi]
AB  - Data synthesis required for large-scale macroevolutionary studies is challenging 
      with the current tools available for integration. Using a classic question
      regarding the frequency of paired fin loss in teleost fishes as a case study, we 
      sought to create automated methods to facilitate the integration of broad-scale
      trait data with a sizable species-level phylogeny. Similar to the evolutionary
      pattern previously described for limbs, pelvic and pectoral fin reduction and
      loss are thought to have occurred independently multiple times in the evolution
      of fishes. We developed a bioinformatics pipeline to identify the presence and
      absence of pectoral and pelvic fins of 12,582 species. To do this, we integrated 
      a synthetic morphological supermatrix of phenotypic data for the pectoral and
      pelvic fins for teleost fishes from the Phenoscape Knowledgebase (two
      presence/absence characters for 3047 taxa) with a species-level tree for teleost 
      fishes from the Open Tree of Life project (38,419 species). The integration
      method detailed herein harnessed a new combined approach by utilizing data based 
      on ontological inference, as well as phylogenetic propagation, to reduce overall 
      data loss. Using inference enabled by ontology-based annotations, missing data
      were reduced from 98.0% to 85.9%, and further reduced to 34.8% by phylogenetic
      data propagation. These methods allowed us to extend the data to an additional
      11,293 species for a total of 12,582 species with trait data. The pectoral fin
      appears to have been independently lost in a minimum of 19 lineages and the
      pelvic fin in 48. Though interpretation is limited by lack of phylogenetic
      resolution at the species level, it appears that following loss, both pectoral
      and pelvic fins were regained several (3) to many (14) times respectively.
      Focused investigation into putative regains of the pectoral fin, all within one
      clade (Anguilliformes), showed that the pectoral fin was regained at least twice 
      following loss. Overall, this study points to specific teleost clades where
      strategic phylogenetic resolution and genetic investigation will be necessary to 
      understand the pattern and frequency of pectoral fin reversals.
FAU - Jackson, Laura M
AU  - Jackson LM
AD  - Department of Biology, University of South Dakota, 414 East Clark St.,
      Vermillion, SD 57069, USA.
FAU - Fernando, Pasan C
AU  - Fernando PC
AD  - Department of Biology, University of South Dakota, 414 East Clark St.,
      Vermillion, SD 57069, USA.
FAU - Hanscom, Josh S
AU  - Hanscom JS
AD  - Department of Biology, University of South Dakota, 414 East Clark St.,
      Vermillion, SD 57069, USA.
FAU - Balhoff, James P
AU  - Balhoff JP
AD  - Renaissance Computing Institute, University of North Carolina, 100 Europa Drive
      Suite 540, Chapel Hill, NC 27517, USA.
FAU - Mabee, Paula M
AU  - Mabee PM
AD  - Department of Biology, University of South Dakota, 414 East Clark St.,
      Vermillion, SD 57069, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.v0s27
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animal Fins/*anatomy &amp; histology/growth &amp; development
MH  - Animals
MH  - *Biological Evolution
MH  - Body Patterning
MH  - Computational Biology/*methods
MH  - Fishes/*anatomy &amp; histology/growth &amp; development
MH  - Phylogeny
PMC - PMC6005059
EDAT- 2018/01/13 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/01/12 06:00
PHST- 2017/01/19 00:00 [received]
PHST- 2017/12/21 00:00 [accepted]
PHST- 2018/01/13 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/01/12 06:00 [entrez]
AID - 4794777 [pii]
AID - 10.1093/sysbio/syx098 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jul 1;67(4):559-575. doi: 10.1093/sysbio/syx098.

PMID- 29319797
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 4
DP  - 2018 Jul 1
TI  - An In Silico Comparison of Protocols for Dated Phylogenomics.
PG  - 633-650
LID - 10.1093/sysbio/syx089 [doi]
AB  - In the age of genome-scale DNA sequencing, choice of molecular marker arguably
      remains an important decision in planning a phylogenetic study. Using published
      genomes from 23 primate species, we make a standardized comparison of four of the
      most frequently used protocols in phylogenomics, viz., targeted
      sequence-enrichment using ultraconserved element and exon-capture probes, and
      restriction-site-associated DNA sequencing (RADseq and ddRADseq). Here, we
      present a procedure to perform in silico extractions from genomes and create
      directly comparable data sets for each class of marker. We then compare these
      data sets in terms of both phylogenetic resolution and ability to consistently
      and precisely estimate clade ages using fossil-calibrated molecular-clock models.
      Furthermore, we were also able to directly compare these results to previously
      published data sets from Sanger-sequenced nuclear exons and mitochondrial genomes
      under the same analytical conditions. Our results show-although with the
      exception of the mitochondrial genome data set and the smallest ddRADseq data
      set-that for uncontroversial nodes all data classes performed equally well, that 
      is they recovered the same well supported topology. However, for one
      difficult-to-resolve node comprising a rapid diversification, we report well
      supported but conflicting topologies among the marker classes consistent with the
      mismodeling of gene tree heterogeneity as demonstrated by species tree analyses
      of single nucleotide polymorphisms. Likewise, clade age estimates showed
      consistent discrepancies between data sets under strict and relaxed clock models;
      for recent nodes, clade ages estimated by nuclear exon data sets were younger
      than those of the UCE, RADseq and mitochondrial data, but vice versa for the
      deepest nodes in the primate phylogeny. This observation is explained by temporal
      differences in phylogenetic informativeness (PI), with the data sets with strong 
      PI peaks toward the present underestimating the deepest node ages. Finally, we
      conclude by emphasizing that while huge numbers of loci are probably not required
      for uncontroversial phylogenetic questions-for which practical considerations
      such as ease of data generation, sharing, and aggregating, therefore become
      increasingly important-accurately modeling heterogeneous data remains as relevant
      as ever for the more recalcitrant problems.
FAU - Collins, Rupert A
AU  - Collins RA
AD  - Laboratorio de Evolucao e Genetica Animal, Department of Genetics, Federal
      University of Amazonas, Av. Rodrigo Otavio Ramos, 3000, Manaus, AM, 69077-000,
      Brazil.
AD  - School of Biological Sciences, Life Sciences Building, University of Bristol, 24 
      Tyndall Ave, Bristol BS8 1TH, UK.
FAU - Hrbek, Tomas
AU  - Hrbek T
AD  - Laboratorio de Evolucao e Genetica Animal, Department of Genetics, Federal
      University of Amazonas, Av. Rodrigo Otavio Ramos, 3000, Manaus, AM, 69077-000,
      Brazil.
AD  - Department of Biology, 4102 LSB Brigham Young University, Provo, UT, 84602, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.8hs71
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Computer Simulation
MH  - Exons
MH  - Genomics/*methods
MH  - *Phylogeny
MH  - Primates/*classification/genetics
MH  - Sequence Analysis, DNA/*methods
EDAT- 2018/01/11 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/01/11 06:00
PHST- 2015/11/23 00:00 [received]
PHST- 2017/10/24 00:00 [accepted]
PHST- 2018/01/11 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/01/11 06:00 [entrez]
AID - 4793286 [pii]
AID - 10.1093/sysbio/syx089 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jul 1;67(4):633-650. doi: 10.1093/sysbio/syx089.

PMID- 29309694
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20190701
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 4
DP  - 2018 Jul 1
TI  - Information Criteria for Comparing Partition Schemes.
PG  - 616-632
LID - 10.1093/sysbio/syx097 [doi]
AB  - When inferring phylogenies, one important decision is whether and how nucleotide 
      substitution parameters should be shared across different subsets or partitions
      of the data. One sort of partitioning error occurs when heterogeneous subsets are
      mistakenly lumped together and treated as if they share parameter values. The
      opposite kind of error is mistakenly treating homogeneous subsets as if they
      result from distinct sets of parameters. Lumping and splitting errors are not
      equally bad. Lumping errors can yield parameter estimates that do not accurately 
      reflect any of the subsets that were combined whereas splitting errors yield
      estimates that did not benefit from sharing information across partitions.
      Phylogenetic partitioning decisions are often made by applying information
      criteria such as the Akaike information criterion (AIC). As with other
      information criteria, the AIC evaluates a model or partition scheme by combining 
      the maximum log-likelihood value with a penalty that depends on the number of
      parameters being estimated. For the purpose of selecting an optimal partitioning 
      scheme, we derive an adjustment to the AIC that we refer to as the AIC$^{(p)}$
      and that is motivated by the idea that splitting errors are less serious than
      lumping errors. We also introduce a similar adjustment to the Bayesian
      information criterion (BIC) that we refer to as the BIC$^{(p)}$. Via simulation
      and empirical data analysis, we contrast AIC and BIC behavior to our suggested
      adjustments. We discuss these results and also emphasize why we expect the
      probability of lumping errors with the AIC$^{(p)}$ and the BIC$^{(p)}$ to be
      relatively robust to model parameterization.
FAU - Seo, Tae-Kun
AU  - Seo TK
AD  - Department of Biological Sciences, Korea Polar Research Institute, 26
      Songdomirae-ro, Yeonsu-gu, Incheon 406-840, Republic of Korea.
FAU - Thorne, Jeffrey L
AU  - Thorne JL
AD  - Bioinformatics Research Center, Box 7566, North Carolina State University,
      Raleigh NC 27695-7566, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.qq586
GR  - R01 GM118508/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Computational Biology/*methods
MH  - Models, Biological
MH  - Models, Genetic
MH  - *Phylogeny
PMC - PMC6005138
EDAT- 2018/01/09 06:00
MHDA- 2018/10/31 06:00
CRDT- 2018/01/09 06:00
PHST- 2017/06/16 00:00 [received]
PHST- 2017/12/17 00:00 [accepted]
PHST- 2018/01/09 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2018/01/09 06:00 [entrez]
AID - 4772580 [pii]
AID - 10.1093/sysbio/syx097 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jul 1;67(4):616-632. doi: 10.1093/sysbio/syx097.

PMID- 29272547
OWN - NLM
STAT- MEDLINE
DCOM- 20181030
LR  - 20181030
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 4
DP  - 2018 Jul 1
TI  - Detecting Environment-Dependent Diversification From Phylogenies: A Simulation
      Study and Some Empirical Illustrations.
PG  - 576-593
LID - 10.1093/sysbio/syx095 [doi]
AB  - Understanding the relative influence of various abiotic and biotic variables on
      diversification dynamics is a major goal of macroevolutionary studies. Recently, 
      phylogenetic approaches have been developed that make it possible to estimate the
      role of various environmental variables on diversification using time-calibrated 
      species trees, paleoenvironmental data, and maximum-likelihood techniques. These 
      approaches have been effectively employed to estimate how speciation and
      extinction rates vary with key abiotic variables, such as temperature and sea
      level, and we can anticipate that they will be increasingly used in the future.
      Here we compile a series of biotic and abiotic paleodatasets that can be used as 
      explanatory variables in these models and use simulations to assess the
      statistical properties of the approach when applied to these paleodatasets. We
      demonstrate that environment-dependent models perform well in recovering
      environment-dependent speciation and extinction parameters, as well as in
      correctly identifying the simulated environmental model when speciation is
      environment-dependent. We explore how the strength of the environment-dependency,
      tree size, missing taxa, and characteristics of the paleoenvironmental curves
      influence the performance of the models. Finally, using these models, we infer
      environment-dependent diversification in two empirical phylogenies:
      temperature-dependence in Cetacea and $\delta^{13}C$-dependence in Ruminantia. We
      illustrate how to evaluate the relative importance of abiotic and biotic
      variables in these two clades and interpret these results in light of
      macroevolutionary hypotheses. Given the important role paleoenvironments are
      presumed to have played in species evolution, our statistical assessment of how
      environment-dependent models behave is crucial for their utility in
      macroevolutionary analysis.
FAU - Lewitus, Eric
AU  - Lewitus E
AD  - IBENS, Departement de Biologie, Ecole Normale Superieure, CNRS, Inserm, PSL
      Research University, F-75005 Paris, France.
FAU - Morlon, Helene
AU  - Morlon H
AD  - IBENS, Departement de Biologie, Ecole Normale Superieure, CNRS, Inserm, PSL
      Research University, F-75005 Paris, France.
LA  - eng
SI  - Dryad/10.5061/dryad.m96r3
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - *Cetacea
MH  - Computer Simulation
MH  - *Environment
MH  - *Models, Biological
MH  - Phylogeny
MH  - *Ruminants
EDAT- 2017/12/23 06:00
MHDA- 2018/10/31 06:00
CRDT- 2017/12/23 06:00
PHST- 2017/01/26 00:00 [received]
PHST- 2017/12/11 00:00 [accepted]
PHST- 2017/12/23 06:00 [pubmed]
PHST- 2018/10/31 06:00 [medline]
PHST- 2017/12/23 06:00 [entrez]
AID - 4767821 [pii]
AID - 10.1093/sysbio/syx095 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jul 1;67(4):576-593. doi: 10.1093/sysbio/syx095.

PMID- 29272537
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20181023
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - Improved Maximum Parsimony Models for Phylogenetic Networks.
PG  - 518-542
LID - 10.1093/sysbio/syx094 [doi]
AB  - Phylogenetic networks are well suited to represent evolutionary histories
      comprising reticulate evolution. Several methods aiming at reconstructing
      explicit phylogenetic networks have been developed in the last two decades. In
      this article, we propose a new definition of maximum parsimony for phylogenetic
      networks that permits to model biological scenarios that cannot be modeled by the
      definitions currently present in the literature (namely, the "hardwired" and
      "softwired" parsimony). Building on this new definition, we provide several
      algorithmic results that lay the foundations for new parsimony-based methods for 
      phylogenetic network reconstruction.
FAU - Van Iersel, Leo
AU  - Van Iersel L
AD  - Delft Institute of Applied Mathematics, Delft University of Technology, P.O. Box 
      5, 2600 AA Delft, the Netherlands.
FAU - Jones, Mark
AU  - Jones M
AD  - Delft Institute of Applied Mathematics, Delft University of Technology, P.O. Box 
      5, 2600 AA Delft, the Netherlands.
FAU - Scornavacca, Celine
AU  - Scornavacca C
AD  - Institut des Sciences de l'Evolution Universite de Montpellier, CNRS, IRD, EPHE
      CC 064, Place Eugene Bataillon 34095 Montpellier Cedex 05, France.
AD  - Institut de Biologie Computationnelle (IBC), Montpellier, France.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2017/12/23 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/12/23 06:00
PHST- 2017/05/03 00:00 [received]
PHST- 2017/12/11 00:00 [accepted]
PHST- 2017/12/23 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/12/23 06:00 [entrez]
AID - 4767820 [pii]
AID - 10.1093/sysbio/syx094 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):518-542. doi: 10.1093/sysbio/syx094.

PMID- 29253296
OWN - NLM
STAT- MEDLINE
DCOM- 20180220
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 1
DP  - 2018 Jan 1
TI  - Crowds Replicate Performance of Scientific Experts Scoring Phylogenetic Matrices 
      of Phenotypes.
PG  - 49-60
LID - 10.1093/sysbio/syx052 [doi]
AB  - Scientists building the Tree of Life face an overwhelming challenge to categorize
      phenotypes (e.g., anatomy, physiology) from millions of living and fossil
      species. This biodiversity challenge far outstrips the capacities of trained
      scientific experts. Here we explore whether crowdsourcing can be used to collect 
      matrix data on a large scale with the participation of nonexpert students, or
      "citizen scientists." Crowdsourcing, or data collection by nonexperts, frequently
      via the internet, has enabled scientists to tackle some large-scale data
      collection challenges too massive for individuals or scientific teams alone. The 
      quality of work by nonexpert crowds is, however, often questioned and little data
      have been collected on how such crowds perform on complex tasks such as
      phylogenetic character coding. We studied a crowd of over 600 nonexperts and
      found that they could use images to identify anatomical similarity (hypotheses of
      homology) with an average accuracy of 82% compared with scores provided by
      experts in the field. This performance pattern held across the Tree of Life, from
      protists to vertebrates. We introduce a procedure that predicts the difficulty of
      each character and that can be used to assign harder characters to experts and
      easier characters to a nonexpert crowd for scoring. We test this procedure in a
      controlled experiment comparing crowd scores to those of experts and show that
      crowds can produce matrices with over 90% of cells scored correctly while
      reducing the number of cells to be scored by experts by 50%. Preparation time,
      including image collection and processing, for a crowdsourcing experiment is
      significant, and does not currently save time of scientific experts overall.
      However, if innovations in automation or robotics can reduce such effort, then
      large-scale implementation of our method could greatly increase the collective
      scientific knowledge of species phenotypes for phylogenetic tree building. For
      the field of crowdsourcing, we provide a rare study with ground truth, or an
      experimental control that many studies lack, and contribute new methods on how to
      coordinate the work of experts and nonexperts. We show that there are important
      instances in which crowd consensus is not a good proxy for correctness.
CI  - (c) The Author(s) 2017. Published by Oxford University Press on behalf of the
      Systematic Biologists. All rights reserved. For Permissions, please email:
      journals.permissions@oup.com.
FAU - O'Leary, Maureen A
AU  - O'Leary MA
AD  - Department of Anatomical Sciences, HSC T-8 (040), Stony Brook University, Stony
      Brook, NY 11794-8081, USA.
FAU - Alphonse, Kenzley
AU  - Alphonse K
AD  - Kenx Technology, Inc., 1170 N. Milwaukee Ave. Chicago, IL 60642, USA.
FAU - Mariangeles, Arce H
AU  - Mariangeles AH
AD  - Department of Anatomical Sciences, HSC T-8 (040), Stony Brook University, Stony
      Brook, NY 11794-8081, USA.
AD  - Department of Ichthyology, The Academy of Natural Sciences of Drexel University, 
      1900 Benjamin Franklin Parkway, Philadelphia PA 19103, USA.
FAU - Cavaliere, Dario
AU  - Cavaliere D
AD  - New York Botanical Garden, 2900 Southern Boulevard, Bronx, NY 10458-5126, USA.
FAU - Cirranello, Andrea
AU  - Cirranello A
AD  - Department of Mammalogy, American Museum of Natural History, 79th Street at
      Central Park West, New York, NY 10024-5192, USA.
FAU - Dietterich, Thomas G
AU  - Dietterich TG
AD  - School of Electrical Engineering and Computer Science, 1148 Kelley Engineering
      Center, Oregon State University, Corvallis, OR 97331-5501, USA.
FAU - Julius, Matthew
AU  - Julius M
AD  - Department of Biological Sciences, St. Cloud State University, WSB-225, 720
      Fourth Avenue South, St. Cloud, MN 56301-4498, USA.
FAU - Kaufman, Seth
AU  - Kaufman S
AD  - Whirl-i-gig, 109 South 5th Street, Suite 608, Brooklyn, NY 10012, USA.
FAU - Law, Edith
AU  - Law E
AD  - David R. Cheriton School of Computer Sciences, University of Waterloo, Waterloo, 
      Ontario, Canada N2L 3G1.
FAU - Passarotti, Maria
AU  - Passarotti M
AD  - Whirl-i-gig, 109 South 5th Street, Suite 608, Brooklyn, NY 10012, USA.
FAU - Reft, Abigail
AU  - Reft A
AD  - Department of Evolution, Ecology and Organismal Biology, The Ohio State
      University, 43210, USA.
AD  - Biology Department, College of William and Mary, Williamsburg, VA 23187, USA.
FAU - Robalino, Javier
AU  - Robalino J
AD  - Department of Anatomical Sciences, HSC T-8 (040), Stony Brook University, Stony
      Brook, NY 11794-8081, USA.
AD  - Science Education Department, Howard Hughes Medical Institute, 4000 Jones Bridge 
      Road, Chevy Chase, MD 20815, USA.
FAU - Simmons, Nancy B
AU  - Simmons NB
AD  - Department of Mammalogy, American Museum of Natural History, 79th Street at
      Central Park West, New York, NY 10024-5192, USA.
FAU - Smith, Selena Y
AU  - Smith SY
AD  - Department of Earth &amp; Environmental Sciences and Museum of Paleontology,
      University of Michigan, 2534 C.C. Little Building, 1100 North University Avenue, 
      Ann Arbor, MI 48109, USA.
FAU - Stevenson, Dennis W
AU  - Stevenson DW
AD  - New York Botanical Garden, 2900 Southern Boulevard, Bronx, NY 10458-5126, USA.
FAU - Theriot, Ed
AU  - Theriot E
AD  - Department of Integrative Biology, University of Texas at Austin, 1 University
      Station #C0930, Austin, Texas, 78712, USA.
FAU - Velazco, Paul M
AU  - Velazco PM
AD  - Department of Vertebrate Paleontology, American Museum of Natural History, 79th
      Street at Central Park West, New York, NY 10024-5192, USA.
FAU - Walls, Ramona L
AU  - Walls RL
AD  - New York Botanical Garden, 2900 Southern Boulevard, Bronx, NY 10458-5126, USA.
AD  - CyVerse, University of Arizona, 1657 E. Helen St., Tucson, AZ 85721, USA.
FAU - Yu, Mengjie
AU  - Yu M
AD  - Department of Integrative Biology, University of Texas at Austin, 1 University
      Station #C0930, Austin, Texas, 78712, USA.
FAU - Daly, Marymegan
AU  - Daly M
AD  - Department of Evolution, Ecology and Organismal Biology, The Ohio State
      University, 43210, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.766cp.2
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Crowdsourcing/*standards
MH  - Phenotype
MH  - *Phylogeny
MH  - Professional Competence
MH  - Reproducibility of Results
OTO - NOTNLM
OT  - *Citizen science
OT  - *MorphoBank
OT  - *crowdsourcing
OT  - *morphology
OT  - *phenomics
OT  - *phylogenetics
EDAT- 2017/12/19 06:00
MHDA- 2018/02/21 06:00
CRDT- 2017/12/19 06:00
PHST- 2016/06/22 00:00 [received]
PHST- 2017/05/24 00:00 [accepted]
PHST- 2017/12/19 06:00 [entrez]
PHST- 2017/12/19 06:00 [pubmed]
PHST- 2018/02/21 06:00 [medline]
AID - 3858082 [pii]
AID - 10.1093/sysbio/syx052 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jan 1;67(1):49-60. doi: 10.1093/sysbio/syx052.

PMID- 29244177
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - Online Bayesian Phylogenetic Inference: Theoretical Foundations via Sequential
      Monte Carlo.
PG  - 503-517
LID - 10.1093/sysbio/syx087 [doi]
AB  - Phylogenetics, the inference of evolutionary trees from molecular sequence data
      such as DNA, is an enterprise that yields valuable evolutionary understanding of 
      many biological systems. Bayesian phylogenetic algorithms, which approximate a
      posterior distribution on trees, have become a popular if computationally
      expensive means of doing phylogenetics. Modern data collection technologies are
      quickly adding new sequences to already substantial databases. With all current
      techniques for Bayesian phylogenetics, computation must start anew each time a
      sequence becomes available, making it costly to maintain an up-to-date estimate
      of a phylogenetic posterior. These considerations highlight the need for an
      online Bayesian phylogenetic method which can update an existing posterior with
      new sequences. Here, we provide theoretical results on the consistency and
      stability of methods for online Bayesian phylogenetic inference based on
      Sequential Monte Carlo (SMC) and Markov chain Monte Carlo. We first show a
      consistency result, demonstrating that the method samples from the correct
      distribution in the limit of a large number of particles. Next, we derive the
      first reported set of bounds on how phylogenetic likelihood surfaces change when 
      new sequences are added. These bounds enable us to characterize the theoretical
      performance of sampling algorithms by bounding the effective sample size (ESS)
      with a given number of particles from below. We show that the ESS is guaranteed
      to grow linearly as the number of particles in an SMC sampler grows.
      Surprisingly, this result holds even though the dimensions of the phylogenetic
      model grow with each new added sequence.
FAU - Dinh, Vu
AU  - Dinh V
AD  - Department of Mathematical Sciences, University of Delaware, 312 Ewing Hall,
      Newark, DE 19716, USA.
FAU - Darling, Aaron E
AU  - Darling AE
AD  - The ithree institute, University of Technology Sydney, 15 Broadway, Ultimo NSW
      2007, Australia.
FAU - Matsen Iv, Frederick A
AU  - Matsen Iv FA
AD  - Program in Computational Biology, Fred Hutchinson Cancer Research Center, 1100
      Fairview Avenue North, Seattle, WA 98109, USA.
LA  - eng
GR  - U54 GM111274/GM/NIGMS NIH HHS/United States
GR  - HHMI/Howard Hughes Medical Institute/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - *Models, Biological
MH  - Monte Carlo Method
MH  - *Phylogeny
PMC - PMC5920340
EDAT- 2017/12/16 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/12/16 06:00
PHST- 2017/02/02 00:00 [received]
PHST- 2017/11/09 00:00 [accepted]
PHST- 2017/12/16 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/12/16 06:00 [entrez]
AID - 4735261 [pii]
AID - 10.1093/sysbio/syx087 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):503-517. doi: 10.1093/sysbio/syx087.

PMID- 29186593
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - Complex Models of Sequence Evolution Require Accurate Estimators as Exemplified
      with the Invariable Site Plus Gamma Model.
PG  - 552-558
LID - 10.1093/sysbio/syx092 [doi]
AB  - The invariable site plus $\Gamma$ model (I$+\Gamma)$ is widely used to model rate
      heterogeneity among alignment sites in maximum likelihood and Bayesian
      phylogenetic analyses. The proof that the I$+$ continuous $\Gamma$ model is
      identifiable (model parameters can be inferred correctly given enough data) has
      increased the creditability of its application to phylogeny reconstruction.
      However, most phylogenetic software implement the I$+$ discrete $\Gamma$ model,
      whose identifiability is likely but unproven. How well the parameters of the I$+$
      discrete $\Gamma$ model are estimated is still disputed. Especially the
      correlation between the fraction of invariable sites and the fractions of sites
      with a slow evolutionary rate is discussed as being problematic. We show that
      optimization heuristics as implemented in frequently used phylogenetic software
      (PhyML, RAxML, IQ-TREE, and MrBayes) cannot always reliably estimate the shape
      parameter, the proportion of invariable sites, and the tree length. Here, we
      propose an improved optimization heuristic that accurately estimates the three
      parameters. While research efforts mainly focus on tree search methods, our
      results signify the equal importance of verifying and developing effective
      estimation methods for complex models of sequence evolution.
FAU - Nguyen, Lam-Tung
AU  - Nguyen LT
AD  - Center for Integrative Bioinformatics Vienna, Department of Structural and
      Computational Biology, Max F. Perutz Laboratories, University of Vienna, Medical 
      University Vienna, Campus Vienna Biocenter 5, A-1030, Vienna, Austria.
FAU - von Haeseler, Arndt
AU  - von Haeseler A
AD  - Center for Integrative Bioinformatics Vienna, Department of Structural and
      Computational Biology, Max F. Perutz Laboratories, University of Vienna, Medical 
      University Vienna, Campus Vienna Biocenter 5, A-1030, Vienna, Austria.
AD  - Bioinformatics and Computational Biology, Faculty of Computer Science, University
      of Vienna, Waehringer Strasse 29, A-1090 Vienna, Austria.
FAU - Minh, Bui Quang
AU  - Minh BQ
AD  - Center for Integrative Bioinformatics Vienna, Department of Structural and
      Computational Biology, Max F. Perutz Laboratories, University of Vienna, Medical 
      University Vienna, Campus Vienna Biocenter 5, A-1030, Vienna, Austria.
LA  - eng
SI  - Dryad/10.5061/dryad.4j5c7
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - *Models, Genetic
MH  - Phylogeny
MH  - *Software
PMC - PMC6204645
EDAT- 2017/12/01 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/11/30 06:00
PHST- 2017/03/08 00:00 [received]
PHST- 2017/11/23 00:00 [accepted]
PHST- 2017/12/01 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/11/30 06:00 [entrez]
AID - 4665709 [pii]
AID - 10.1093/sysbio/syx092 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):552-558. doi: 10.1093/sysbio/syx092.

PMID- 29186587
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - Effective Online Bayesian Phylogenetics via Sequential Monte Carlo with Guided
      Proposals.
PG  - 490-502
LID - 10.1093/sysbio/syx090 [doi]
AB  - Modern infectious disease outbreak surveillance produces continuous streams of
      sequence data which require phylogenetic analysis as data arrives. Current
      software packages for Bayesian phylogenetic inference are unable to quickly
      incorporate new sequences as they become available, making them less useful for
      dynamically unfolding evolutionary stories. This limitation can be addressed by
      applying a class of Bayesian statistical inference algorithms called sequential
      Monte Carlo (SMC) to conduct online inference, wherein new data can be
      continuously incorporated to update the estimate of the posterior probability
      distribution. In this article, we describe and evaluate several different online 
      phylogenetic sequential Monte Carlo (OPSMC) algorithms. We show that proposing
      new phylogenies with a density similar to the Bayesian prior suffers from poor
      performance, and we develop "guided" proposals that better match the proposal
      density to the posterior. Furthermore, we show that the simplest guided proposals
      can exhibit pathological behavior in some situations, leading to poor results,
      and that the situation can be resolved by heating the proposal density. The
      results demonstrate that relative to the widely used MCMC-based algorithm
      implemented in MrBayes, the total time required to compute a series of
      phylogenetic posteriors as sequences arrive can be significantly reduced by the
      use of OPSMC, without incurring a significant loss in accuracy.
FAU - Fourment, Mathieu
AU  - Fourment M
AD  - ithree institute, University of Technology Sydney, Ultimo, NSW 2007, Australia.
FAU - Claywell, Brian C
AU  - Claywell BC
AD  - Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA.
FAU - Dinh, Vu
AU  - Dinh V
AD  - Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA.
FAU - McCoy, Connor
AU  - McCoy C
AD  - Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA.
FAU - Matsen Iv, Frederick A
AU  - Matsen Iv FA
AD  - Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA.
FAU - Darling, Aaron E
AU  - Darling AE
AD  - ithree institute, University of Technology Sydney, Ultimo, NSW 2007, Australia.
LA  - eng
SI  - Dryad/10.5061/dryad.n7n85
GR  - U54 GM111274/GM/NIGMS NIH HHS/United States
GR  - HHMI/Howard Hughes Medical Institute/United States
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, N.I.H., Extramural
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Internet
MH  - *Models, Biological
MH  - Monte Carlo Method
MH  - *Phylogeny
PMC - PMC5920299
EDAT- 2017/12/01 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/11/30 06:00
PHST- 2017/06/02 00:00 [received]
PHST- 2017/11/20 00:00 [accepted]
PHST- 2017/12/01 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/11/30 06:00 [entrez]
AID - 4665707 [pii]
AID - 10.1093/sysbio/syx090 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):490-502. doi: 10.1093/sysbio/syx090.

PMID- 29186553
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20180310
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - Corrigendum to: "Probabilistic Distances Between Trees".
PG  - 366
LID - 10.1093/sysbio/syx083 [doi]
LA  - eng
PT  - Journal Article
PT  - Published Erratum
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EFR - Syst Biol. 2018 Mar 1;67(2):320-327. PMID: 29029295
PMC - PMC5837674
EDAT- 2017/12/01 06:00
MHDA- 2017/12/01 06:01
CRDT- 2017/11/30 06:00
PHST- 2017/12/01 06:00 [pubmed]
PHST- 2017/12/01 06:01 [medline]
PHST- 2017/11/30 06:00 [entrez]
AID - 4653339 [pii]
AID - 10.1093/sysbio/syx083 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):366. doi: 10.1093/sysbio/syx083.

PMID- 29165679
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20190501
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - A Two-State Model of Tree Evolution and Its Applications to Alu
      Retrotransposition.
PG  - 475-489
LID - 10.1093/sysbio/syx088 [doi]
AB  - Models of tree evolution have mostly focused on capturing the cladogenesis
      processes behind speciation. Processes that derive the evolution of genomic
      elements, such as repeats, are not necessarily captured by these existing models.
      In this article, we design a model of tree evolution that we call the dual-birth 
      model, and we show how it can be useful in studying the evolution of short Alu
      repeats found in the human genome in abundance. The dual-birth model extends the 
      traditional birth-only model to have two rates of propagation, one for active
      nodes that propagate often, and another for inactive nodes, that with a lower
      rate, activate and start propagating. Adjusting the ratio of the rates controls
      the expected tree balance. We present several theoretical results under the
      dual-birth model, introduce parameter estimation techniques, and study the
      properties of the model in simulations. We then use the dual-birth model to
      estimate the number of active Alu elements and their rates of propagation and
      activation in the human genome based on a large phylogenetic tree that we build
      from close to one million Alu sequences.
FAU - Moshiri, Niema
AU  - Moshiri N
AD  - Bioinformatics and Systems Biology Graduate Program, UC San Diego, 9500 Gilman
      Dr., La Jolla, CA 92093, USA.
FAU - Mirarab, Siavash
AU  - Mirarab S
AD  - Department of Electrical and Computer Engineering, UC San Diego, 9500 Gilman Dr.,
      La Jolla, CA 92093, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.13n52
GR  - P30 AI027767/AI/NIAID NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Alu Elements/*genetics
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - Genome, Human/*genetics
MH  - Humans
MH  - *Models, Biological
MH  - *Phylogeny
PMC - PMC5920319
EDAT- 2017/11/23 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/11/23 06:00
PHST- 2017/03/27 00:00 [received]
PHST- 2017/11/15 00:00 [accepted]
PHST- 2017/11/23 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/11/23 06:00 [entrez]
AID - 4642947 [pii]
AID - 10.1093/sysbio/syx088 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):475-489. doi: 10.1093/sysbio/syx088.

PMID- 29106675
OWN - NLM
STAT- MEDLINE
DCOM- 20180316
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - The Efficacy of Consensus Tree Methods for Summarizing Phylogenetic Relationships
      from a Posterior Sample of Trees Estimated from Morphological Data.
PG  - 354-362
LID - 10.1093/sysbio/syx086 [doi]
AB  - Consensus trees are required to summarize trees obtained through MCMC sampling of
      a posterior distribution, providing an overview of the distribution of estimated 
      parameters such as topology, branch lengths, and divergence times. Numerous
      consensus tree construction methods are available, each presenting a different
      interpretation of the tree sample. The rise of morphological clock and
      sampled-ancestor methods of divergence time estimation, in which times and
      topology are coestimated, has increased the popularity of the maximum clade
      credibility (MCC) consensus tree method. The MCC method assumes that the sampled,
      fully resolved topology with the highest clade credibility is an adequate summary
      of the most probable clades, with parameter estimates from compatible sampled
      trees used to obtain the marginal distributions of parameters such as clade ages 
      and branch lengths. Using both simulated and empirical data, we demonstrate that 
      MCC trees, and trees constructed using the similar maximum a posteriori (MAP)
      method, often include poorly supported and incorrect clades when summarizing
      diffuse posterior samples of trees. We demonstrate that the paucity of
      information in morphological data sets contributes to the inability of MCC and
      MAP trees to accurately summarise of the posterior distribution. Conversely,
      majority-rule consensus (MRC) trees represent a lower proportion of incorrect
      nodes when summarizing the same posterior samples of trees. Thus, we advocate the
      use of MRC trees, in place of MCC or MAP trees, in attempts to summarize the
      results of Bayesian phylogenetic analyses of morphological data.
FAU - O'Reilly, Joseph E
AU  - O'Reilly JE
AD  - School of Earth Sciences, University of Bristol, Life Sciences Building, Tyndall 
      Avenue, Bristol BS8 1TQ, UK.
FAU - Donoghue, Philip C J
AU  - Donoghue PCJ
AD  - School of Earth Sciences, University of Bristol, Life Sciences Building, Tyndall 
      Avenue, Bristol BS8 1TQ, UK.
LA  - eng
SI  - Dryad/10.5061/dryad.66s9h
GR  - BB/N000919/1 /Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Phylogeny
PMC - PMC5837340
EDAT- 2017/11/07 06:00
MHDA- 2018/03/17 06:00
CRDT- 2017/11/07 06:00
PHST- 2017/02/06 00:00 [received]
PHST- 2017/10/24 00:00 [accepted]
PHST- 2017/11/07 06:00 [pubmed]
PHST- 2018/03/17 06:00 [medline]
PHST- 2017/11/07 06:00 [entrez]
AID - 4587515 [pii]
AID - 10.1093/sysbio/syx086 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):354-362. doi: 10.1093/sysbio/syx086.

PMID- 29088474
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20181023
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - Reconstructing Ecological Niche Evolution When Niches Are Incompletely
      Characterized.
PG  - 428-438
LID - 10.1093/sysbio/syx084 [doi]
AB  - Evolutionary dynamics of abiotic ecological niches across phylogenetic history
      can shed light on large-scale biogeographic patterns, macroevolutionary rate
      shifts, and the relative ability of lineages to respond to global change. An
      unresolved question is how best to represent and reconstruct evolution of these
      complex traits at coarse spatial scales through time. Studies have approached
      this question by integrating phylogenetic comparative methods with niche
      estimates inferred from correlative and other models. However, methods for
      estimating niches often produce incomplete characterizations, as they are
      inferred from present-day distributions that may be limited in full expression of
      the fundamental ecological niche by biotic interactions, dispersal limitations,
      and the existing set of environmental conditions. Here, we test whether
      incomplete niche characterizations inherent in most estimates of species' niches 
      bias phylogenetic reconstructions of niche evolution, using simulations of
      virtual species with known niches. Results establish that incompletely
      characterized niches inflate estimates of evolutionary change and lead to error
      in ancestral state reconstructions. Our analyses also provide a potential
      mechanism to explain the frequent observation that maximum thermal tolerances are
      more conserved than minimum thermal tolerances: populations and species
      experience more spatial variation in minimum temperature than in maximum
      temperature across their distributions and, consequently, may experience stronger
      diversifying selection for cold tolerance.
FAU - Saupe, Erin E
AU  - Saupe EE
AD  - Department of Earth Sciences, University of Oxford, South Parks Road, Oxford OX1 
      3AN, UK.
FAU - Barve, Narayani
AU  - Barve N
AD  - Florida Museum of Natural History, University of Florida, Dickinson Hall, 1659
      Museum Road Gainesville, FL 32611, USA.
FAU - Owens, Hannah L
AU  - Owens HL
AD  - Florida Museum of Natural History, University of Florida, Dickinson Hall, 1659
      Museum Road Gainesville, FL 32611, USA.
FAU - Cooper, Jacob C
AU  - Cooper JC
AD  - Committee on Evolutionary Biology, University of Chicago, 1025 East 57th Street, 
      IL 60637, USA.
FAU - Hosner, Peter A
AU  - Hosner PA
AD  - Department of Biology, University of Florida, 220 Bartram Hall, Gainesville, FL
      32611, USA.
FAU - Peterson, A Townsend
AU  - Peterson AT
AD  - Biodiversity Institute, University of Kansas, Dyche Hall, 1345 Jayhawk Blvd.,
      Lawrence, KS 66045, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.j3f5j
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - *Computer Simulation
MH  - *Ecosystem
MH  - Environment
MH  - *Models, Biological
EDAT- 2017/11/01 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/11/01 06:00
PHST- 2017/04/21 00:00 [received]
PHST- 2017/10/24 00:00 [accepted]
PHST- 2017/11/01 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/11/01 06:00 [entrez]
AID - 4568575 [pii]
AID - 10.1093/sysbio/syx084 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):428-438. doi: 10.1093/sysbio/syx084.

PMID- 29088409
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20181023
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - Coestimating Reticulate Phylogenies and Gene Trees from Multilocus Sequence Data.
PG  - 439-457
LID - 10.1093/sysbio/syx085 [doi]
AB  - The multispecies network coalescent (MSNC) is a stochastic process that captures 
      how gene trees grow within the branches of a phylogenetic network. Coupling the
      MSNC with a stochastic mutational process that operates along the branches of the
      gene trees gives rise to a generative model of how multiple loci from within and 
      across species evolve in the presence of both incomplete lineage sorting (ILS)
      and reticulation (e.g., hybridization). We report on a Bayesian method for
      sampling the parameters of this generative model, including the species
      phylogeny, gene trees, divergence times, and population sizes, from DNA sequences
      of multiple independent loci. We demonstrate the utility of our method by
      analyzing simulated data and reanalyzing an empirical data set. Our results
      demonstrate the significance of not only coestimating species phylogenies and
      gene trees, but also accounting for reticulation and ILS simultaneously. In
      particular, we show that when gene flow occurs, our method accurately estimates
      the evolutionary histories, coalescence times, and divergence times. Tree
      inference methods, on the other hand, underestimate divergence times and
      overestimate coalescence times when the evolutionary history is reticulate. While
      the MSNC corresponds to an abstract model of "intermixture," we study the
      performance of the model and method on simulated data generated under a gene flow
      model. We show that the method accurately infers the most recent time at which
      gene flow occurs. Finally, we demonstrate the application of the new method to a 
      106-locus yeast data set.
FAU - Wen, Dingqiao
AU  - Wen D
AD  - Department of Computer Science.
FAU - Nakhleh, Luay
AU  - Nakhleh L
AD  - Department of Computer Science.
AD  - Department of BioSciences, Rice University, 6100 Main Street, Houston, TX 77005, 
      USA.
LA  - eng
SI  - Dryad/10.5061/dryad.3h185
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Computer Simulation
MH  - Gene Flow
MH  - Genetic Speciation
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Saccharomyces cerevisiae/classification/genetics
EDAT- 2017/11/01 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/11/01 06:00
PHST- 2017/04/24 00:00 [received]
PHST- 2017/10/24 00:00 [accepted]
PHST- 2017/11/01 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/11/01 06:00 [entrez]
AID - 4568576 [pii]
AID - 10.1093/sysbio/syx085 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):439-457. doi: 10.1093/sysbio/syx085.

PMID- 29069434
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - Estimating Age-Dependent Extinction: Contrasting Evidence from Fossils and
      Phylogenies.
PG  - 458-474
LID - 10.1093/sysbio/syx082 [doi]
AB  - The estimation of diversification rates is one of the most vividly debated topics
      in modern systematics, with considerable controversy surrounding the power of
      phylogenetic and fossil-based approaches in estimating extinction. Van Valen's
      seminal work from 1973 proposed the "Law of constant extinction," which states
      that the probability of extinction of taxa is not dependent on their age. This
      assumption of age-independent extinction has prevailed for decades with its
      assessment based on survivorship curves, which, however, do not directly account 
      for the incompleteness of the fossil record, and have rarely been applied at the 
      species level. Here, we present a Bayesian framework to estimate extinction rates
      from the fossil record accounting for age-dependent extinction (ADE). Our
      approach, unlike previous implementations, explicitly models unobserved species
      and accounts for the effects of fossil preservation on the observed longevity of 
      sampled lineages. We assess the performance and robustness of our method through 
      extensive simulations and apply it to a fossil data set of terrestrial Carnivora 
      spanning the past 40 myr. We find strong evidence of ADE, as we detect the
      extinction rate to be highest in young species and declining with increasing
      species age. For comparison, we apply a recently developed analogous ADE model to
      a dated phylogeny of extant Carnivora. Although the phylogeny-based analysis also
      infers ADE, it indicates that the extinction rate, instead, increases with
      increasing taxon age. The estimated mean species longevity also differs
      substantially, with the fossil-based analyses estimating 2.0 myr, in contrast to 
      9.8 myr derived from the phylogeny-based inference. Scrutinizing these
      discrepancies, we find that both fossil and phylogeny-based ADE models are prone 
      to high error rates when speciation and extinction rates increase or decrease
      through time. However, analyses of simulated and empirical data show that
      fossil-based inferences are more robust. This study shows that an accurate
      estimation of ADE from incomplete fossil data is possible when the effects of
      preservation are jointly modeled, thus allowing for a reassessment of Van Valen's
      model as a general rule in macroevolution.
FAU - Hagen, Oskar
AU  - Hagen O
AD  - Swiss Federal Research Institute WSL, 8903 Birmensdorf, Switzerland.
AD  - Landscape Ecology, Institute of Terrestrial Ecosystems, ETH Zurich, 8092 Zurich, 
      Switzerland.
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      SE-405 30 Goteborg, Sweden.
FAU - Andermann, Tobias
AU  - Andermann T
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      SE-405 30 Goteborg, Sweden.
AD  - Gothenburg Global Biodiversity Centre, Box 461, SE-405 30 Goteborg, Sweden.
FAU - Quental, Tiago B
AU  - Quental TB
AD  - Departamento de Ecologia, Universidade de Sao Paulo, 05508-900 Sao Paulo, Brazil.
FAU - Antonelli, Alexandre
AU  - Antonelli A
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      SE-405 30 Goteborg, Sweden.
AD  - Gothenburg Global Biodiversity Centre, Box 461, SE-405 30 Goteborg, Sweden.
AD  - Gothenburg Botanical Garden, Carl Skottsbergs gata 22A, SE-413 19 Goteborg,
      Sweden.
FAU - Silvestro, Daniele
AU  - Silvestro D
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      SE-405 30 Goteborg, Sweden.
AD  - Gothenburg Global Biodiversity Centre, Box 461, SE-405 30 Goteborg, Sweden.
AD  - Department of Computational Biology, University of Lausanne, 1015 Lausanne,
      Switzerland.
LA  - eng
SI  - Dryad/10.5061/dryad.r5f70
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Extinction, Biological
MH  - *Fossils
MH  - Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
PMC - PMC5920349
EDAT- 2017/10/27 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/10/26 06:00
PHST- 2016/12/19 00:00 [received]
PHST- 2017/10/15 00:00 [accepted]
PHST- 2017/10/27 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/10/26 06:00 [entrez]
AID - 4563320 [pii]
AID - 10.1093/sysbio/syx082 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):458-474. doi: 10.1093/sysbio/syx082.

PMID- 29029343
OWN - NLM
STAT- MEDLINE
DCOM- 20180220
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 1
DP  - 2018 Jan 1
TI  - An Evaluation of Different Partitioning Strategies for Bayesian Estimation of
      Species Divergence Times.
PG  - 61-77
LID - 10.1093/sysbio/syx061 [doi]
AB  - The explosive growth of molecular sequence data has made it possible to estimate 
      species divergence times under relaxed-clock models using genome-scale data sets 
      with many gene loci. In order to improve both model realism and to best extract
      information about relative divergence times in the sequence data, it is important
      to account for the heterogeneity in the evolutionary process across genes or
      genomic regions. Partitioning is a commonly used approach to achieve those goals.
      We group sites that have similar evolutionary characteristics into the same
      partition and those with different characteristics into different partitions, and
      then use different models or different values of model parameters for different
      partitions to account for the among-partition heterogeneity. However, how to
      partition data in practical phylogenetic analysis, and in particular in
      relaxed-clock dating analysis, is more art than science. Here, we use computer
      simulation and real data analysis to study the impact of the partition scheme on 
      divergence time estimation. The partition schemes had relatively minor effects on
      the accuracy of posterior time estimates when the prior assumptions were correct 
      and the clock was not seriously violated, but showed large differences when the
      clock was seriously violated, when the fossil calibrations were in conflict or
      incorrect, or when the rate prior was mis-specified. Concatenation produced the
      widest posterior intervals with the least precision. Use of many partitions
      increased the precision, as predicted by the infinite-sites theory, but the
      posterior intervals might fail to include the true ages because of the
      conflicting fossil calibrations or mis-specified rate priors. We analyzed a data 
      set of 78 plastid genes from 15 plant species with serious clock violation and
      showed that time estimates differed significantly among partition schemes,
      irrespective of the rate drift model used. Multiple and precise fossil
      calibrations reduced the differences among partition schemes and were important
      to improving the precision of divergence time estimates. While the use of many
      partitions is an important approach to reducing the uncertainty in posterior time
      estimates, we do not recommend its general use for the present, given the
      limitations of current models of rate drift for partitioned data and the
      challenges of interpreting the fossil evidence to construct accurate and
      informative calibrations.
CI  - (c) The Author(s) 2017. Published by Oxford University Press on behalf of the
      Systematic Biologists.
FAU - Angelis, Konstantinos
AU  - Angelis K
AD  - Department of Genetics, Evolution and Environment, University College London, UK.
FAU - Alvarez-Carretero, Sandra
AU  - Alvarez-Carretero S
AD  - School of Biological and Chemical Sciences, Queen Mary University of London,
      London E1 4NS, UK.
FAU - Dos Reis, Mario
AU  - Dos Reis M
AD  - Department of Genetics, Evolution and Environment, University College London, UK.
AD  - School of Biological and Chemical Sciences, Queen Mary University of London,
      London E1 4NS, UK.
FAU - Yang, Ziheng
AU  - Yang Z
AD  - Department of Genetics, Evolution and Environment, University College London, UK.
LA  - eng
SI  - Dryad/10.5061/dryad.d7839
GR  - BB/N000609/1 /Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Fossils
MH  - *Genetic Speciation
MH  - Plants/classification/genetics
MH  - Plastids/genetics
MH  - Reproducibility of Results
MH  - Time
PMC - PMC5790132
OTO - NOTNLM
OT  - *Bayesian inference
OT  - *genomic data
OT  - *infinite-sites theory
OT  - *molecular clock dating
OT  - *partition analysis
EDAT- 2017/10/14 06:00
MHDA- 2018/02/21 06:00
CRDT- 2017/10/14 06:00
PHST- 2016/11/21 00:00 [received]
PHST- 2017/06/28 00:00 [accepted]
PHST- 2017/10/14 06:00 [pubmed]
PHST- 2018/02/21 06:00 [medline]
PHST- 2017/10/14 06:00 [entrez]
AID - 3921199 [pii]
AID - 10.1093/sysbio/syx061 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jan 1;67(1):61-77. doi: 10.1093/sysbio/syx061.

PMID- 29029339
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20181023
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - Targeted Enrichment of Large Gene Families for Phylogenetic Inference: Phylogeny 
      and Molecular Evolution of Photosynthesis Genes in the Portullugo Clade
      (Caryophyllales).
PG  - 367-383
LID - 10.1093/sysbio/syx078 [doi]
AB  - Hybrid enrichment is an increasingly popular approach for obtaining hundreds of
      loci for phylogenetic analysis across many taxa quickly and cheaply. The genes
      targeted for sequencing are typically single-copy loci, which facilitate a more
      straightforward sequence assembly and homology assignment process. However, this 
      approach limits the inclusion of most genes of functional interest, which often
      belong to multi-gene families. Here, we demonstrate the feasibility of including 
      large gene families in hybrid enrichment protocols for phylogeny reconstruction
      and subsequent analyses of molecular evolution, using a new set of bait sequences
      designed for the "portullugo" (Caryophyllales), a moderately sized lineage of
      flowering plants (~ 2200 species) that includes the cacti and harbors many
      evolutionary transitions to C$_{\mathrm{4}}$ and CAM photosynthesis. Including
      multi-gene families allowed us to simultaneously infer a robust phylogeny and
      construct a dense sampling of sequences for a major enzyme of C$_{\mathrm{4}}$
      and CAM photosynthesis, which revealed the accumulation of adaptive amino acid
      substitutions associated with C$_{\mathrm{4}}$ and CAM origins in particular
      paralogs. Our final set of matrices for phylogenetic analyses included 75-218
      loci across 74 taxa, with ~ 50% matrix completeness across data sets.
      Phylogenetic resolution was greatly improved across the tree, at both shallow and
      deep levels. Concatenation and coalescent-based approaches both resolve the
      sister lineage of the cacti with strong support: Anacampserotaceae $+$
      Portulacaceae, two lineages of mostly diminutive succulent herbs of warm, arid
      regions. In spite of this congruence, BUCKy concordance analyses demonstrated
      strong and conflicting signals across gene trees. Our results add to the growing 
      number of examples illustrating the complexity of phylogenetic signals in
      genomic-scale data.
FAU - Moore, Abigail J
AU  - Moore AJ
AD  - Department of Ecology and Evolutionary Biology, Brown University, Box G-W,
      Providence, RI 02912, USA.
AD  - Department of Microbiology and Plant Biology and Oklahoma Biological Survey,
      University of Oklahoma, 770 Van Vleet Oval, Norman, OK 73019, USA.
FAU - Vos, Jurriaan M De
AU  - Vos JM
AD  - Department of Ecology and Evolutionary Biology, Brown University, Box G-W,
      Providence, RI 02912, USA.
AD  - Department of Comparative Plant and Fungal Biology, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3AE, UK.
AD  - Department of Environmental Sciences-Botany, University of Basel, Totengasslein
      3, 4051 Basel, Switzerland.
FAU - Hancock, Lillian P
AU  - Hancock LP
AD  - Department of Ecology and Evolutionary Biology, Brown University, Box G-W,
      Providence, RI 02912, USA.
FAU - Goolsby, Eric
AU  - Goolsby E
AD  - Department of Ecology and Evolutionary Biology, Brown University, Box G-W,
      Providence, RI 02912, USA.
AD  - Department of Ecology and Evolutionary Biology, Yale University, PO Box 208105,
      New Haven, CT 06520, USA.
FAU - Edwards, Erika J
AU  - Edwards EJ
AD  - Department of Ecology and Evolutionary Biology, Brown University, Box G-W,
      Providence, RI 02912, USA.
AD  - Department of Ecology and Evolutionary Biology, Yale University, PO Box 208105,
      New Haven, CT 06520, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.7h3f6
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Caryophyllales/*classification/*genetics
MH  - *Evolution, Molecular
MH  - Genome, Plant/genetics
MH  - Photosynthesis/*genetics
MH  - *Phylogeny
EDAT- 2017/10/14 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/10/14 06:00
PHST- 2017/06/02 00:00 [received]
PHST- 2017/09/18 00:00 [accepted]
PHST- 2017/10/14 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/10/14 06:00 [entrez]
AID - 4222382 [pii]
AID - 10.1093/sysbio/syx078 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):367-383. doi: 10.1093/sysbio/syx078.

PMID- 29029338
OWN - NLM
STAT- MEDLINE
DCOM- 20180316
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - To Include or Not to Include: The Impact of Gene Filtering on Species Tree
      Estimation Methods.
PG  - 285-303
LID - 10.1093/sysbio/syx077 [doi]
AB  - With the increasing availability of whole genome data, many species trees are
      being constructed from hundreds to thousands of loci. Although concatenation
      analysis using maximum likelihood is a standard approach for estimating species
      trees, it does not account for gene tree heterogeneity, which can occur due to
      many biological processes, such as incomplete lineage sorting. Coalescent species
      tree estimation methods, many of which are statistically consistent in the
      presence of incomplete lineage sorting, include Bayesian methods that coestimate 
      the gene trees and the species tree, summary methods that compute the species
      tree by combining estimated gene trees, and site-based methods that infer the
      species tree from site patterns in the alignments of different loci. Due to
      concerns that poor quality loci will reduce the accuracy of estimated species
      trees, many recent phylogenomic studies have removed or filtered genes on the
      basis of phylogenetic signal and/or missing data prior to inferring species
      trees; little is known about the performance of species tree estimation methods
      when gene filtering is performed. We examine how incomplete lineage sorting,
      phylogenetic signal of individual loci, and missing data affect the absolute and 
      the relative accuracy of species tree estimation methods and show how these
      properties affect methods' responses to gene filtering strategies. In particular,
      summary methods (ASTRAL-II, ASTRID, and MP-EST), a site-based coalescent method
      (SVDquartets within PAUP*), and an unpartitioned concatenation analysis using
      maximum likelihood (RAxML) were evaluated on a heterogeneous collection of
      simulated multilocus data sets, and the following trends were observed. Filtering
      genes based on gene tree estimation error improved the accuracy of the summary
      methods when levels of incomplete lineage sorting were low to moderate but did
      not benefit the summary methods under higher levels of incomplete lineage
      sorting, unless gene tree estimation error was also extremely high (a model
      condition with few replicates). Neither SVDquartets nor concatenation analysis
      using RAxML benefited from filtering genes on the basis of gene tree estimation
      error. Finally, filtering genes based on missing data was either neutral (i.e.,
      did not impact accuracy) or else reduced the accuracy of all five methods. By
      providing insight into the consequences of gene filtering, we offer
      recommendations for estimating species tree in the presence of incomplete lineage
      sorting and reconcile seemingly conflicting observations made in prior studies
      regarding the impact of gene filtering.
FAU - Molloy, Erin K
AU  - Molloy EK
AD  - Department of Computer Science, University of Illinois at Urbana-Champaign,
      Urbana, IL, USA.
FAU - Warnow, Tandy
AU  - Warnow T
AD  - Department of Computer Science, University of Illinois at Urbana-Champaign,
      Urbana, IL, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.km24v
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Genetic Speciation
MH  - Genomics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Sequence Analysis
EDAT- 2017/10/14 06:00
MHDA- 2018/03/17 06:00
CRDT- 2017/10/14 06:00
PHST- 2016/12/08 00:00 [received]
PHST- 2017/09/13 00:00 [accepted]
PHST- 2017/10/14 06:00 [pubmed]
PHST- 2018/03/17 06:00 [medline]
PHST- 2017/10/14 06:00 [entrez]
AID - 4159193 [pii]
AID - 10.1093/sysbio/syx077 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):285-303. doi: 10.1093/sysbio/syx077.

PMID- 29029306
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20181023
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - An Assessment of Phylogenetic Tools for Analyzing the Interplay Between
      Interspecific Interactions and Phenotypic Evolution.
PG  - 413-427
LID - 10.1093/sysbio/syx079 [doi]
AB  - Much ecological and evolutionary theory predicts that interspecific interactions 
      often drive phenotypic diversification and that species phenotypes in turn
      influence species interactions. Several phylogenetic comparative methods have
      been developed to assess the importance of such processes in nature; however, the
      statistical properties of these methods have gone largely untested. Focusing
      mainly on scenarios of competition between closely-related species, we assess the
      performance of available comparative approaches for analyzing the interplay
      between interspecific interactions and species phenotypes. We find that many
      currently used statistical methods often fail to detect the impact of
      interspecific interactions on trait evolution, that sister-taxa analyses are
      particularly unreliable in general, and that recently developed process-based
      models have more satisfactory statistical properties. Methods for detecting
      predictors of species interactions are generally more reliable than methods for
      detecting character displacement. In weighing the strengths and weaknesses of
      different approaches, we hope to provide a clear guide for empiricists testing
      hypotheses about the reciprocal effect of interspecific interactions and species 
      phenotypes and to inspire further development of process-based models.
FAU - Drury, J P
AU  - Drury JP
AD  - Departement de Biologie, Institut de Biologie, Ecole Normale Superieure, CNRS UMR
      8197, 46 rue d'Ulm 75005 Paris, France.
AD  - Department of Ecology &amp; Evolutionary Biology, University of California, 621
      Charles E. Young Dr. S, Los Angeles, CA 90095-1606, USA.
FAU - Grether, G F
AU  - Grether GF
AD  - Department of Ecology &amp; Evolutionary Biology, University of California, 621
      Charles E. Young Dr. S, Los Angeles, CA 90095-1606, USA.
FAU - Garland, T Jr
AU  - Garland T Jr
AD  - Department of Evolution, Ecology, and Organismal Biology, University of
      California, 900 University Ave., Riverside, CA 92521, USA.
FAU - Morlon, H
AU  - Morlon H
AD  - Departement de Biologie, Institut de Biologie, Ecole Normale Superieure, CNRS UMR
      8197, 46 rue d'Ulm 75005 Paris, France.
LA  - eng
SI  - Dryad/10.5061/dryad.ch0vn
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Ecosystem
MH  - *Models, Biological
MH  - Phenotype
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2017/10/14 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/10/14 06:00
PHST- 2016/10/25 00:00 [received]
PHST- 2017/09/21 00:00 [accepted]
PHST- 2017/10/14 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/10/14 06:00 [entrez]
AID - 4344840 [pii]
AID - 10.1093/sysbio/syx079 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):413-427. doi: 10.1093/sysbio/syx079.

PMID- 29029295
OWN - NLM
STAT- MEDLINE
DCOM- 20180316
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - Probabilistic Distances Between Trees.
PG  - 320-327
LID - 10.1093/sysbio/syx080 [doi]
AB  - Most existing measures of distance between phylogenetic trees are based on the
      geometry or topology of the trees. Instead, we consider distance measures which
      are based on the underlying probability distributions on genetic sequence data
      induced by trees. Monte Carlo schemes are necessary to calculate these distances 
      approximately, and we describe efficient sampling procedures. Key features of the
      distances are the ability to include substitution model parameters and to handle 
      trees with different taxon sets in a principled way. We demonstrate some of the
      properties of these new distance measures and compare them to existing distances,
      in particular by applying multidimensional scaling to data sets previously
      reported as containing phylogenetic islands. [Metric; probability distribution;
      multidimensional scaling; information geometry.
FAU - Garba, Maryam K
AU  - Garba MK
AD  - School of Mathematics &amp; Statistics, Newcastle University, Newcastle upon Tyne NE1
      7RU, UK.
AD  - Department of Mathematical Sciences, Bayero University, Kano, Nigeria.
FAU - Nye, Tom M W
AU  - Nye TMW
AD  - School of Mathematics &amp; Statistics, Newcastle University, Newcastle upon Tyne NE1
      7RU, UK.
FAU - Boys, Richard J
AU  - Boys RJ
AD  - School of Mathematics &amp; Statistics, Newcastle University, Newcastle upon Tyne NE1
      7RU, UK.
LA  - eng
SI  - Dryad/10.5061/dryad.69bb2
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2018 Mar 1;67(2):366. PMID: 29186553
MH  - Classification/*methods
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Probability
PMC - PMC5837584
EDAT- 2017/10/14 06:00
MHDA- 2018/03/17 06:00
CRDT- 2017/10/14 06:00
PHST- 2017/02/24 00:00 [received]
PHST- 2017/09/18 00:00 [accepted]
PHST- 2017/10/14 06:00 [pubmed]
PHST- 2018/03/17 06:00 [medline]
PHST- 2017/10/14 06:00 [entrez]
AID - 4344841 [pii]
AID - 10.1093/sysbio/syx080 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):320-327. doi: 10.1093/sysbio/syx080.

PMID- 29029264
OWN - NLM
STAT- MEDLINE
DCOM- 20180316
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - A General Model for Estimating Macroevolutionary Landscapes.
PG  - 304-319
LID - 10.1093/sysbio/syx075 [doi]
AB  - The evolution of quantitative characters over long timescales is often studied
      using stochastic diffusion models. The current toolbox available to students of
      macroevolution is however limited to two main models: Brownian motion and the
      Ornstein-Uhlenbeck process, plus some of their extensions. Here, we present a
      very general model for inferring the dynamics of quantitative characters evolving
      under both random diffusion and deterministic forces of any possible shape and
      strength, which can accommodate interesting evolutionary scenarios like
      directional trends, disruptive selection, or macroevolutionary landscapes with
      multiple peaks. This model is based on a general partial differential equation
      widely used in statistical mechanics: the Fokker-Planck equation, also known in
      population genetics as the Kolmogorov forward equation. We thus call the model
      FPK, for Fokker-Planck-Kolmogorov. We first explain how this model can be used to
      describe macroevolutionary landscapes over which quantitative traits evolve and, 
      more importantly, we detail how it can be fitted to empirical data. Using
      simulations, we show that the model has good behavior both in terms of
      discrimination from alternative models and in terms of parameter inference. We
      provide R code to fit the model to empirical data using either maximum-likelihood
      or Bayesian estimation, and illustrate the use of this code with two empirical
      examples of body mass evolution in mammals. FPK should greatly expand the set of 
      macroevolutionary scenarios that can be studied since it opens the way to
      estimating macroevolutionary landscapes of any conceivable shape. [Adaptation;
      bounds; diffusion; FPK model; macroevolution; maximum-likelihood estimation; MCMC
      methods; phylogenetic comparative data; selection.].
FAU - Boucher, Florian C
AU  - Boucher FC
AD  - Department of Systematic and Evolutionary Botany (ISEB), University of Zurich,
      Zurich, Switzerland.
AD  - Department of Botany and Zoology, University of Stellenbosch, Stellenbosch, South
      Africa.
FAU - Demery, Vincent
AU  - Demery V
AD  - Gulliver, CNRS, ESPCI Paris, PSL Research University, 10 rue Vauquelin, Paris,
      France.
FAU - Conti, Elena
AU  - Conti E
AD  - Department of Systematic and Evolutionary Botany (ISEB), University of Zurich,
      Zurich, Switzerland.
FAU - Harmon, Luke J
AU  - Harmon LJ
AD  - Department of Biological Sciences and Institute for Bioinformatics and
      Evolutionary Studies (IBEST), University of Idaho, Moscow, ID, USA.
AD  - Department of Fish Ecology and Evolution, Swiss Federal Institute of Aquatic
      Science and Technology (Eawag), Center for Ecology, Evolution, and
      Biogeochemistry, 6047 Kastanienbaum, Switzerland.
FAU - Uyeda, Josef
AU  - Uyeda J
AD  - Department of Biological Sciences and Institute for Bioinformatics and
      Evolutionary Studies (IBEST), University of Idaho, Moscow, ID, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.28g37
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - *Models, Biological
MH  - Phylogeny
MH  - Software
EDAT- 2017/10/14 06:00
MHDA- 2018/03/17 06:00
CRDT- 2017/10/14 06:00
PHST- 2017/01/10 00:00 [received]
PHST- 2017/09/08 00:00 [accepted]
PHST- 2017/10/14 06:00 [pubmed]
PHST- 2018/03/17 06:00 [medline]
PHST- 2017/10/14 06:00 [entrez]
AID - 4210009 [pii]
AID - 10.1093/sysbio/syx075 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):304-319. doi: 10.1093/sysbio/syx075.

PMID- 29029231
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20181023
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - Analysis of Phylogenomic Tree Space Resolves Relationships Among Marsupial
      Families.
PG  - 400-412
LID - 10.1093/sysbio/syx076 [doi]
AB  - A fundamental challenge in resolving evolutionary relationships across the tree
      of life is to account for heterogeneity in the evolutionary signal across loci.
      Studies of marsupial mammals have demonstrated that this heterogeneity can be
      substantial, leaving considerable uncertainty in the evolutionary timescale and
      relationships within the group. Using simulations and a new phylogenomic data set
      comprising nucleotide sequences of 1550 loci from 18 of the 22 extant marsupial
      families, we demonstrate the power of a method for identifying clusters of loci
      that support different phylogenetic trees. We find two distinct clusters of loci,
      each providing an estimate of the species tree that matches previously proposed
      resolutions of the marsupial phylogeny. We also identify a well-supported
      placement for the enigmatic marsupial moles (Notoryctes) that contradicts
      previous molecular estimates but is consistent with morphological evidence. The
      pattern of gene-tree variation across tree-space is characterized by changes in
      information content, GC content, substitution-model adequacy, and signatures of
      purifying selection in the data. In a simulation study, we show that incomplete
      lineage sorting can explain the division of loci into the two tree-topology
      clusters, as found in our phylogenomic analysis of marsupials. We also
      demonstrate the potential benefits of minimizing uncertainty from phylogenetic
      conflict for molecular dating. Our analyses reveal that Australasian marsupials
      appeared in the early Paleocene, whereas the diversification of present-day
      families occurred primarily during the late Eocene and early Oligocene. Our
      methods provide an intuitive framework for improving the accuracy and precision
      of phylogenetic inference and molecular dating using genome-scale data.
FAU - Duchene, David A
AU  - Duchene DA
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney, NSW
      2006, Australia.
FAU - Bragg, Jason G
AU  - Bragg JG
AD  - Research School of Biology, Australian National University, Canberra, ACT 2601,
      Australia.
AD  - National Herbarium of NSW, The Royal Botanic Gardens and Domain Trust, Sydney,
      NSW 2000, Australia.
FAU - Duchene, Sebastian
AU  - Duchene S
AD  - Centre for Systems Genomics, The University of Melbourne, Melbourne, VIC 3010,
      Australia.
FAU - Neaves, Linda E
AU  - Neaves LE
AD  - Australian Museum Research Institute, Australian Museum, 1 William Street,
      Sydney, NSW 2000, Australia.
FAU - Potter, Sally
AU  - Potter S
AD  - Research School of Biology, Australian National University, Canberra, ACT 2601,
      Australia.
AD  - Australian Museum Research Institute, Australian Museum, 1 William Street,
      Sydney, NSW 2000, Australia.
FAU - Moritz, Craig
AU  - Moritz C
AD  - Research School of Biology, Australian National University, Canberra, ACT 2601,
      Australia.
FAU - Johnson, Rebecca N
AU  - Johnson RN
AD  - Australian Museum Research Institute, Australian Museum, 1 William Street,
      Sydney, NSW 2000, Australia.
FAU - Ho, Simon Y W
AU  - Ho SYW
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney, NSW
      2006, Australia.
FAU - Eldridge, Mark D B
AU  - Eldridge MDB
AD  - Australian Museum Research Institute, Australian Museum, 1 William Street,
      Sydney, NSW 2000, Australia.
LA  - eng
SI  - Dryad/10.5061/dryad.353q5
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Computer Simulation
MH  - *Genomics
MH  - Marsupialia/*classification/genetics
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2017/10/14 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/10/14 06:00
PHST- 2017/07/29 00:00 [received]
PHST- 2017/09/08 00:00 [accepted]
PHST- 2017/10/14 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/10/14 06:00 [entrez]
AID - 4175806 [pii]
AID - 10.1093/sysbio/syx076 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):400-412. doi: 10.1093/sysbio/syx076.

PMID- 28973686
OWN - NLM
STAT- MEDLINE
DCOM- 20180316
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - Resolving Recent Plant Radiations: Power and Robustness of
      Genotyping-by-Sequencing.
PG  - 250-268
LID - 10.1093/sysbio/syx062 [doi]
AB  - Disentangling species boundaries and phylogenetic relationships within recent
      evolutionary radiations is a challenge due to the poor morphological
      differentiation and low genetic divergence between species, frequently
      accompanied by phenotypic convergence, interspecific gene flow and incomplete
      lineage sorting. Here we employed a genotyping-by-sequencing (GBS) approach, in
      combination with morphometric analyses, to investigate a small western
      Mediterranean clade in the flowering plant genus Linaria that radiated in the
      Quaternary. After confirming the morphological and genetic distinctness of eight 
      species, we evaluated the relative performances of concatenation and coalescent
      methods to resolve phylogenetic relationships. Specifically, we focused on
      assessing the robustness of both approaches to variations in the parameter used
      to estimate sequence homology (clustering threshold). Concatenation analyses
      suffered from strong systematic bias, as revealed by the high statistical support
      for multiple alternative topologies depending on clustering threshold values. By 
      contrast, topologies produced by two coalescent-based methods
      (NJ$_{\mathrm{st}}$, SVDquartets) were robust to variations in the clustering
      threshold. Reticulate evolution may partly explain incongruences between
      NJ$_{\mathrm{st}}$, SVDquartets and concatenated trees. Integration of
      morphometric and coalescent-based phylogenetic results revealed (i) extensive
      morphological divergence associated with recent splits between geographically
      close or sympatric sister species and (ii) morphological convergence in
      geographically disjunct species. These patterns are particularly true for floral 
      traits related to pollinator specialization, including nectar spur length, tube
      width and corolla color, suggesting pollinator-driven diversification. Given its 
      relatively simple and inexpensive implementation, GBS is a promising technique
      for the phylogenetic and systematic study of recent radiations, but care must be 
      taken to evaluate the robustness of results to variation of data assembly
      parameters.
FAU - Fernandez-Mazuecos, Mario
AU  - Fernandez-Mazuecos M
AD  - Department of Plant Sciences, University of Cambridge, Cambridge CB2 3EA, UK.
FAU - Mellers, Greg
AU  - Mellers G
AD  - Department of Plant Sciences, University of Cambridge, Cambridge CB2 3EA, UK.
FAU - Vigalondo, Beatriz
AU  - Vigalondo B
AD  - Departamento de Biologia (Botanica), Facultad de Ciencias, Universidad Autonoma
      de Madrid, 28049 Madrid, Spain.
FAU - Saez, Llorenc
AU  - Saez L
AD  - Departament de Biologia Animal, Biologia Vegetal i Ecologia, Unitat de Botanica, 
      Facultat de Ciencies, Universitat Autonoma de Barcelona, 08193 Bellaterra,
      Barcelona, Spain.
FAU - Vargas, Pablo
AU  - Vargas P
AD  - Departamento de Biodiversidad y Conservacion, Real Jardin Botanico (RJB-CSIC),
      Plaza de Murillo 2, 28014 Madrid, Spain.
FAU - Glover, Beverley J
AU  - Glover BJ
AD  - Department of Plant Sciences, University of Cambridge, Cambridge CB2 3EA, UK.
LA  - eng
SI  - Dryad/10.5061/dryad.mp818
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Genotype
MH  - Linaria/*classification/*genetics
MH  - *Plant Dispersal
MH  - Sequence Analysis, DNA
EDAT- 2017/10/04 06:00
MHDA- 2018/03/17 06:00
CRDT- 2017/10/04 06:00
PHST- 2016/10/28 00:00 [received]
PHST- 2017/06/30 00:00 [accepted]
PHST- 2017/10/04 06:00 [pubmed]
PHST- 2018/03/17 06:00 [medline]
PHST- 2017/10/04 06:00 [entrez]
AID - 3953673 [pii]
AID - 10.1093/sysbio/syx062 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):250-268. doi: 10.1093/sysbio/syx062.

PMID- 28973673
OWN - NLM
STAT- MEDLINE
DCOM- 20180220
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 1
DP  - 2018 Jan 1
TI  - Why Concatenation Fails Near the Anomaly Zone.
PG  - 158-169
LID - 10.1093/sysbio/syx063 [doi]
AB  - Genome-scale sequencing has been of great benefit in recovering species trees but
      has not provided final answers. Despite the rapid accumulation of molecular
      sequences, resolving short and deep branches of the tree of life has remained a
      challenge and has prompted the development of new strategies that can make the
      best use of available data. One such strategy-the concatenation of gene
      alignments-can be successful when coupled with many tree estimation methods, but 
      has also been shown to fail when there are high levels of incomplete lineage
      sorting. Here, we focus on the failure of likelihood-based methods in retrieving 
      a rooted, asymmetric four-taxon species tree from concatenated data when the
      species tree is in or near the anomaly zone-a region of parameter space where the
      most common gene tree does not match the species tree because of incomplete
      lineage sorting. First, we use coalescent theory to prove that most informative
      sites will support the species tree in the anomaly zone, and that as a
      consequence maximum-parsimony succeeds in recovering the species tree from
      concatenated data. We further show that maximum-likelihood tree estimation from
      concatenated data fails both inside and outside the anomaly zone, and that this
      failure cannot be easily predicted from the topology of the most common gene
      tree. We demonstrate that likelihood-based methods often fail in a region
      partially overlapping the anomaly zone, likely because of the lower relative cost
      of substitutions on discordant gene tree branches that are absent from the
      species tree. Our results confirm and extend previous reports on the performance 
      of these methods applied to concatenated data from a rooted, asymmetric
      four-taxon species tree, and highlight avenues for future work improving the
      performance of methods aimed at recovering species tree.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Mendes, Fabio K
AU  - Mendes FK
AD  - Department of Biology, Indiana University, Bloomington, IN 47405, USA.
FAU - Hahn, Matthew W
AU  - Hahn MW
AD  - Department of Biology, Indiana University, Bloomington, IN 47405, USA.
AD  - Department of Computer Science, Indiana University, Bloomington, IN 47405, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.j6000
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Genetic Speciation
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Sequence Analysis, DNA
OTO - NOTNLM
OT  - *Anomalous gene tree
OT  - *coalescent
OT  - *incomplete lineage sorting
OT  - *parsimony
OT  - *species tree
EDAT- 2017/10/04 06:00
MHDA- 2018/02/21 06:00
CRDT- 2017/10/04 06:00
PHST- 2017/03/13 00:00 [received]
PHST- 2017/06/30 00:00 [accepted]
PHST- 2017/10/04 06:00 [pubmed]
PHST- 2018/02/21 06:00 [medline]
PHST- 2017/10/04 06:00 [entrez]
AID - 3953674 [pii]
AID - 10.1093/sysbio/syx063 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jan 1;67(1):158-169. doi: 10.1093/sysbio/syx063.

PMID- 28973546
OWN - NLM
STAT- MEDLINE
DCOM- 20180220
LR  - 20190107
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 1
DP  - 2018 Jan 1
TI  - Genomic Signature of an Avian Lilliput Effect across the K-Pg Extinction.
PG  - 1-13
LID - 10.1093/sysbio/syx064 [doi]
AB  - Survivorship following major mass extinctions may be associated with a decrease
      in body size-a phenomenon called the Lilliput Effect. Body size is a strong
      predictor of many life history traits (LHTs), and is known to influence
      demography and intrinsic biological processes. Pronounced changes in organismal
      size throughout Earth history are therefore likely to be associated with
      concomitant genome-wide changes in evolutionary rates. Here, we report pronounced
      heterogeneity in rates of molecular evolution (varying up to $\sim$20-fold)
      across a large-scale avian phylogenomic data set and show that nucleotide
      substitution rates are strongly correlated with body size and metabolic rate. We 
      also identify potential body size reductions associated with the
      Cretaceous-Paleogene (K-Pg) transition, consistent with a Lilliput Effect in the 
      wake of that mass extinction event. We posit that selection for reduced body size
      across the K-Pg extinction horizon may have resulted in transient increases in
      substitution rate along the deepest branches of the extant avian tree of life.
      This "hidden" rate acceleration may result in both strict and relaxed molecular
      clocks over-estimating the age of the avian crown group through the relationship 
      between life history and demographic parameters that scale with molecular
      substitution rate. If reductions in body size (and/or selection for related
      demographic parameters like short generation times) are a common property of
      lineages surviving mass extinctions, this phenomenon may help resolve persistent 
      divergence time debates across the tree of life. Furthermore, our results suggest
      that selection for certain LHTs may be associated with deterministic molecular
      evolutionary outcomes.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Berv, Jacob S
AU  - Berv JS
AD  - Department of Ecology &amp; Evolutionary Biology, Cornell University, 215 Tower Road,
      Ithaca NY, 14853, USA.
FAU - Field, Daniel J
AU  - Field DJ
AD  - Department of Geology &amp; Geophysics, Yale University, 210 Whitney Avenue New
      Haven, CT, 06511, USA.
AD  - Department of Biology and Biochemistry, Milner Centre for Evolution, University
      of Bath, Building 4 South, Claverton Down, Bath, BA2 7AY, UK.
LA  - eng
SI  - Dryad/10.5061/dryad.nr654
GR  - S10 RR019895/RR/NCRR NIH HHS/United States
GR  - S10 RR029676/RR/NCRR NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amino Acid Substitution/genetics
MH  - Animals
MH  - Body Size/*genetics
MH  - *Evolution, Molecular
MH  - *Extinction, Biological
MH  - Fossils
MH  - Life History Traits
PMC - PMC5837713
OTO - NOTNLM
OT  - *Birds
OT  - *K-Pg
OT  - *body size
OT  - *divergence times
OT  - *life history evolution
OT  - *mass extinction
OT  - *metabolic rate
OT  - *molecular clocks
EDAT- 2017/10/04 06:00
MHDA- 2018/02/21 06:00
CRDT- 2017/10/04 06:00
PHST- 2017/02/12 00:00 [received]
PHST- 2017/07/05 00:00 [accepted]
PHST- 2017/10/04 06:00 [pubmed]
PHST- 2018/02/21 06:00 [medline]
PHST- 2017/10/04 06:00 [entrez]
AID - 3960267 [pii]
AID - 10.1093/sysbio/syx064 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jan 1;67(1):1-13. doi: 10.1093/sysbio/syx064.

PMID- 28950376
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20190501
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - Phylogenetic Factor Analysis.
PG  - 384-399
LID - 10.1093/sysbio/syx066 [doi]
AB  - Phylogenetic comparative methods explore the relationships between quantitative
      traits adjusting for shared evolutionary history. This adjustment often occurs
      through a Brownian diffusion process along the branches of the phylogeny that
      generates model residuals or the traits themselves. For high-dimensional traits, 
      inferring all pair-wise correlations within the multivariate diffusion is
      limiting. To circumvent this problem, we propose phylogenetic factor analysis
      (PFA) that assumes a small unknown number of independent evolutionary factors
      arise along the phylogeny and these factors generate clusters of dependent
      traits. Set in a Bayesian framework, PFA provides measures of uncertainty on the 
      factor number and groupings, combines both continuous and discrete traits,
      integrates over missing measurements and incorporates phylogenetic uncertainty
      with the help of molecular sequences. We develop Gibbs samplers based on dynamic 
      programming to estimate the PFA posterior distribution, over 3-fold faster than
      for multivariate diffusion and a further order-of-magnitude more efficiently in
      the presence of latent traits. We further propose a novel marginal likelihood
      estimator for previously impractical models with discrete data and find that PFA 
      also provides a better fit than multivariate diffusion in evolutionary questions 
      in columbine flower development, placental reproduction transitions and
      triggerfish fin morphometry.
FAU - Tolkoff, Max R
AU  - Tolkoff MR
AD  - Department of Biostatistics, Jonathan and Karin Fielding School of Public Health,
      University of California, 650 Charles E. Young Dr. South Los Angeles, CA
      90095-1772, USA.
FAU - Alfaro, Michael E
AU  - Alfaro ME
AD  - Department of Ecology and Evolutionary Biology, University of California, 610
      Charles E. Young Drive South Los Angeles, CA 90095-1606, USA.
FAU - Baele, Guy
AU  - Baele G
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven,
      Minderbroederstraat 10 BE-3000 Leuven, Belgium.
FAU - Lemey, Philippe
AU  - Lemey P
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven,
      Minderbroederstraat 10 BE-3000 Leuven, Belgium.
FAU - Suchard, Marc A
AU  - Suchard MA
AD  - Department of Biostatistics, Jonathan and Karin Fielding School of Public Health,
      University of California, 650 Charles E. Young Dr. South Los Angeles, CA
      90095-1772, USA.
AD  - Department of Biomathematics, David Geffen School of Medicine at UCLA, University
      of California, 650 Charles E. Young Dr., South Los Angeles, CA 90095-1766, USA.
AD  - Department of Human Genetics, David Geffen School of Medicine at UCLA, University
      of California, 695 Charles E. Young Drive South Los Angeles, CA 90095-7088, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.6320t
GR  - R01 AI107034/AI/NIAID NIH HHS/United States
GR  - R01 AI117011/AI/NIAID NIH HHS/United States
GR  - R01 HG006139/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, N.I.H., Extramural
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Factor Analysis, Statistical
MH  - *Models, Genetic
MH  - *Phylogeny
PMC - PMC5920329
EDAT- 2017/09/28 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/09/27 06:00
PHST- 2017/01/18 00:00 [received]
PHST- 2017/07/21 00:00 [accepted]
PHST- 2017/09/28 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/09/27 06:00 [entrez]
AID - 4076231 [pii]
AID - 10.1093/sysbio/syx066 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):384-399. doi: 10.1093/sysbio/syx066.

PMID- 28950365
OWN - NLM
STAT- MEDLINE
DCOM- 20180316
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - Modeling Site Heterogeneity with Posterior Mean Site Frequency Profiles
      Accelerates Accurate Phylogenomic Estimation.
PG  - 216-235
LID - 10.1093/sysbio/syx068 [doi]
AB  - Proteins have distinct structural and functional constraints at different sites
      that lead to site-specific preferences for particular amino acid residues as the 
      sequences evolve. Heterogeneity in the amino acid substitution process between
      sites is not modeled by commonly used empirical amino acid exchange matrices.
      Such model misspecification can lead to artefacts in phylogenetic estimation such
      as long-branch attraction. Although sophisticated site-heterogeneous mixture
      models have been developed to address this problem in both Bayesian and maximum
      likelihood (ML) frameworks, their formidable computational time and memory usage 
      severely limits their use in large phylogenomic analyses. Here we propose a
      posterior mean site frequency (PMSF) method as a rapid and efficient
      approximation to full empirical profile mixture models for ML analysis. The PMSF 
      approach assigns a conditional mean amino acid frequency profile to each site
      calculated based on a mixture model fitted to the data using a preliminary guide 
      tree. These PMSF profiles can then be used for in-depth tree-searching in place
      of the full mixture model. Compared with widely used empirical mixture models
      with $k$ classes, our implementation of PMSF in IQ-TREE (http://www.iqtree.org)
      speeds up the computation by approximately $k$/1.5-fold and requires a small
      fraction of the RAM. Furthermore, this speedup allows, for the first time, full
      nonparametric bootstrap analyses to be conducted under complex site-heterogeneous
      models on large concatenated data matrices. Our simulations and empirical data
      analyses demonstrate that PMSF can effectively ameliorate long-branch attraction 
      artefacts. In some empirical and simulation settings PMSF provided more accurate 
      estimates of phylogenies than the mixture models from which they derive.
FAU - Wang, Huai-Chun
AU  - Wang HC
AD  - Department of Mathematics and Statistics, 6316 Coburg Road.
AD  - Department of Biochemistry and Molecular Biology, 5850 College Street, Dalhousie 
      University, Halifax, Nova Scotia B3H 4R2, Canada.
AD  - Centre for Comparative Genomics and Evolutionary Bioinformatics, Dalhousie
      University, Halifax, Nova Scotia B3H 4R2, Canada.
FAU - Minh, Bui Quang
AU  - Minh BQ
AD  - Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories,
      University of Vienna and Medical University of Vienna, Austria.
FAU - Susko, Edward
AU  - Susko E
AD  - Department of Mathematics and Statistics, 6316 Coburg Road.
AD  - Centre for Comparative Genomics and Evolutionary Bioinformatics, Dalhousie
      University, Halifax, Nova Scotia B3H 4R2, Canada.
FAU - Roger, Andrew J
AU  - Roger AJ
AD  - Department of Biochemistry and Molecular Biology, 5850 College Street, Dalhousie 
      University, Halifax, Nova Scotia B3H 4R2, Canada.
AD  - Centre for Comparative Genomics and Evolutionary Bioinformatics, Dalhousie
      University, Halifax, Nova Scotia B3H 4R2, Canada.
LA  - eng
SI  - Dryad/10.5061/dryad.gv1q5
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amino Acid Substitution
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Evolution, Molecular
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Statistics, Nonparametric
EDAT- 2017/09/28 06:00
MHDA- 2018/03/17 06:00
CRDT- 2017/09/27 06:00
PHST- 2016/10/21 00:00 [received]
PHST- 2017/08/02 00:00 [accepted]
PHST- 2017/09/28 06:00 [pubmed]
PHST- 2018/03/17 06:00 [medline]
PHST- 2017/09/27 06:00 [entrez]
AID - 4076233 [pii]
AID - 10.1093/sysbio/syx068 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):216-235. doi: 10.1093/sysbio/syx068.

PMID- 28945917
OWN - NLM
STAT- MEDLINE
DCOM- 20180316
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - Cladogenetic and Anagenetic Models of Chromosome Number Evolution: A Bayesian
      Model Averaging Approach.
PG  - 195-215
LID - 10.1093/sysbio/syx065 [doi]
AB  - ABSSTRACT: Chromosome number is a key feature of the higher-order organization of
      the genome, and changes in chromosome number play a fundamental role in
      evolution. Dysploid gains and losses in chromosome number, as well as
      polyploidization events, may drive reproductive isolation and lineage
      diversification. The recent development of probabilistic models of chromosome
      number evolution in the groundbreaking work by Mayrose et al. (2010, ChromEvol)
      have enabled the inference of ancestral chromosome numbers over molecular
      phylogenies and generated new interest in studying the role of chromosome changes
      in evolution. However, the ChromEvol approach assumes all changes occur
      anagenetically (along branches), and does not model events that are specifically 
      cladogenetic. Cladogenetic changes may be expected if chromosome changes result
      in reproductive isolation. Here we present a new class of models of chromosome
      number evolution (called ChromoSSE) that incorporate both anagenetic and
      cladogenetic change. The ChromoSSE models allow us to determine the mode of
      chromosome number evolution; is chromosome evolution occurring primarily within
      lineages, primarily at lineage splitting, or in clade-specific combinations of
      both? Furthermore, we can estimate the location and timing of possible chromosome
      speciation events over the phylogeny. We implemented ChromoSSE in a Bayesian
      statistical framework, specifically in the software RevBayes, to accommodate
      uncertainty in parameter estimates while leveraging the full power of likelihood 
      based methods. We tested ChromoSSE's accuracy with simulations and re-examined
      chromosomal evolution in Aristolochia, Carex section Spirostachyae, Helianthus,
      Mimulus sensu lato (s.l.), and Primula section Aleuritia, finding evidence for
      clade-specific combinations of anagenetic and cladogenetic dysploid and polyploid
      modes of chromosome evolution. [Anagenetic; Bayes factors; chromosome evolution; 
      chromosome speciation; chromoSSE; cladogenetic; dysploidy; phylogenetic models;
      polyploidy; reversible-jump Markov chain Monte Carlo; whole genome duplication.].
FAU - Freyman, William A
AU  - Freyman WA
AD  - Department of Integrative Biology, University of California, 3040 Valley Life
      Sciences Building #3140, Berkeley, CA 94720, USA.
FAU - Hohna, Sebastian
AU  - Hohna S
AD  - Department of Integrative Biology, University of California, 3040 Valley Life
      Sciences Building #3140, Berkeley, CA 94720, USA.
AD  - Department of Statistics, University of California, 367 Evans Hall, Berkeley, CA 
      94720, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.46m4b
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - *Classification
MH  - *Evolution, Molecular
MH  - *Models, Genetic
MH  - Phylogeny
MH  - Plants/classification/genetics
EDAT- 2017/09/26 06:00
MHDA- 2018/03/17 06:00
CRDT- 2017/09/26 06:00
PHST- 2016/11/08 00:00 [received]
PHST- 2017/07/01 00:00 [accepted]
PHST- 2017/09/26 06:00 [pubmed]
PHST- 2018/03/17 06:00 [medline]
PHST- 2017/09/26 06:00 [entrez]
AID - 4037179 [pii]
AID - 10.1093/sysbio/syx065 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):195-215. doi: 10.1093/sysbio/syx065.

PMID- 28945912
OWN - NLM
STAT- MEDLINE
DCOM- 20180316
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - The Past Sure is Tense: On Interpreting Phylogenetic Divergence Time Estimates.
PG  - 340-353
LID - 10.1093/sysbio/syx074 [doi]
AB  - Divergence time estimation-the calibration of a phylogeny to geological time-is
      an integral first step in modeling the tempo of biological evolution (traits and 
      lineages). However, despite increasingly sophisticated methods to infer
      divergence times from molecular genetic sequences, the estimated age of many
      nodes across the tree of life contrast significantly and consistently with
      timeframes conveyed by the fossil record. This is perhaps best exemplified by
      crown angiosperms, where molecular clock (Triassic) estimates predate the oldest 
      (Early Cretaceous) undisputed angiosperm fossils by tens of millions of years or 
      more. While the incompleteness of the fossil record is a common concern, issues
      of data limitation and model inadequacy are viable (if underexplored) alternative
      explanations. In this vein, Beaulieu et al. (2015) convincingly demonstrated how 
      methods of divergence time inference can be misled by both (i) extreme
      state-dependent molecular substitution rate heterogeneity and (ii) biased
      sampling of representative major lineages. These results demonstrate the impact
      of (potentially common) model violations. Here, we suggest another potential
      challenge: that the configuration of the statistical inference problem (i.e., the
      parameters, their relationships, and associated priors) alone may preclude the
      reconstruction of the paleontological timeframe for the crown age of angiosperms.
      We demonstrate, through sampling from the joint prior (formed by combining the
      tree (diversification) prior with the calibration densities specified for
      fossil-calibrated nodes) that with no data present at all, that an Early
      Cretaceous crown angiosperms is rejected (i.e., has essentially zero
      probability). More worrisome, however, is that for the 24 nodes calibrated by
      fossils, almost all have indistinguishable marginal prior and posterior age
      distributions when employing routine lognormal fossil calibration priors. These
      results indicate that there is inadequate information in the data to over-rule
      the joint prior. Given that these calibrated nodes are strategically placed in
      disparate regions of the tree, they act to anchor the tree scaffold, and so the
      posterior inference for the tree as a whole is largely determined by the
      pseudodata present in the (often arbitrary) calibration densities. We recommend, 
      as for any Bayesian analysis, that marginal prior and posterior distributions be 
      carefully compared to determine whether signal is coming from the data or prior
      belief, especially for parameters of direct interest. This recommendation is not 
      novel. However, given how rarely such checks are carried out in evolutionary
      biology, it bears repeating. Our results demonstrate the fundamental importance
      of prior/posterior comparisons in any Bayesian analysis, and we hope that they
      further encourage both researchers and journals to consistently adopt this
      crucial step as standard practice. Finally, we note that the results presented
      here do not refute the biological modeling concerns identified by Beaulieu et al.
      (2015). Both sets of issues remain apposite to the goals of accurate divergence
      time estimation, and only by considering them in tandem can we move forward more 
      confidently.
FAU - Brown, Joseph W
AU  - Brown JW
AD  - Department of Ecology &amp; Evolutionary Biology, University of Michigan, 830 North
      University Avenue, Ann Arbor, MI 48109, USA.
FAU - Smith, Stephen A
AU  - Smith SA
AD  - Department of Ecology &amp; Evolutionary Biology, University of Michigan, 830 North
      University Avenue, Ann Arbor, MI 48109, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.rs449
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - Fossils
MH  - Magnoliopsida/classification/genetics
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Time
EDAT- 2017/09/26 06:00
MHDA- 2018/03/17 06:00
CRDT- 2017/09/26 06:00
PHST- 2017/03/07 00:00 [received]
PHST- 2017/09/04 00:00 [accepted]
PHST- 2017/09/26 06:00 [pubmed]
PHST- 2018/03/17 06:00 [medline]
PHST- 2017/09/26 06:00 [entrez]
AID - 4107448 [pii]
AID - 10.1093/sysbio/syx074 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):340-353. doi: 10.1093/sysbio/syx074.

PMID- 28945906
OWN - NLM
STAT- MEDLINE
DCOM- 20180316
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - Use of Continuous Traits Can Improve Morphological Phylogenetics.
PG  - 328-339
LID - 10.1093/sysbio/syx072 [doi]
AB  - The recent surge in enthusiasm for simultaneously inferring relationships from
      extinct and extant species has reinvigorated interest in statistical approaches
      for modeling morphological evolution. Current statistical methods use the Mk
      model to describe substitutions between discrete character states. Although
      representing a significant step forward, the Mk model presents challenges in
      biological interpretation, and its adequacy in modeling morphological evolution
      has not been well explored. Another major hurdle in morphological phylogenetics
      concerns the process of character coding of discrete characters. The often
      subjective nature of discrete character coding can generate discordant results
      that are rooted in individual researchers' subjective interpretations. Employing 
      continuous measurements to infer phylogenies may alleviate some of these issues. 
      Although not widely used in the inference of topology, models describing the
      evolution of continuous characters have been well examined, and their statistical
      behavior is well understood. Also, continuous measurements avoid the substantial 
      ambiguity often associated with the assignment of discrete characters to states. 
      I present a set of simulations to determine whether use of continuous characters 
      is a feasible alternative or supplement to discrete characters for inferring
      phylogeny. I compare relative reconstruction accuracy by inferring phylogenies
      from simulated continuous and discrete characters. These tests demonstrate
      significant promise for continuous traits by demonstrating their higher overall
      accuracy as compared to reconstruction from discrete characters under Mk when
      simulated under unbounded Brownian motion, and equal performance when simulated
      under an Ornstein-Uhlenbeck model. Continuous characters also perform reasonably 
      well in the presence of covariance between sites. I argue that inferring
      phylogenies directly from continuous traits may be benefit efforts to maximize
      phylogenetic information in morphological data sets by preserving larger
      variation in state space compared to many discretization schemes. I also suggest 
      that the use of continuous trait models in phylogenetic reconstruction may
      alleviate potential concerns of discrete character model adequacy, while
      identifying areas that require further study in this area. This study provides an
      initial controlled demonstration of the efficacy of continuous characters in
      phylogenetic inference.
FAU - Parins-Fukuchi, Caroline
AU  - Parins-Fukuchi C
AD  - Department of Ecology and Evolutionary Biology, University of Michigan, 830 N.
      University, Ann Arbor, MI 48109, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.40b70
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Models, Biological
MH  - Phenotype
MH  - *Phylogeny
EDAT- 2017/09/26 06:00
MHDA- 2018/03/17 06:00
CRDT- 2017/09/26 06:00
PHST- 2017/02/05 00:00 [received]
PHST- 2017/08/30 00:00 [accepted]
PHST- 2017/09/26 06:00 [pubmed]
PHST- 2018/03/17 06:00 [medline]
PHST- 2017/09/26 06:00 [entrez]
AID - 4102005 [pii]
AID - 10.1093/sysbio/syx072 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):328-339. doi: 10.1093/sysbio/syx072.

PMID- 28945903
OWN - NLM
STAT- MEDLINE
DCOM- 20180316
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - Impact of Model Violations on the Inference of Species Boundaries Under the
      Multispecies Coalescent.
PG  - 269-284
LID - 10.1093/sysbio/syx073 [doi]
AB  - The use of genetic data for identifying species-level lineages across the tree of
      life has received increasing attention in the field of systematics over the past 
      decade. The multispecies coalescent model provides a framework for understanding 
      the process of lineage divergence and has become widely adopted for delimiting
      species. However, because these studies lack an explicit assessment of model fit,
      in many cases, the accuracy of the inferred species boundaries are unknown. This 
      is concerning given the large amount of empirical data and theory that highlight 
      the complexity of the speciation process. Here, we seek to fill this gap by using
      simulation to characterize the sensitivity of inference under the multispecies
      coalescent (MSC) to several violations of model assumptions thought to be common 
      in empirical data. We also assess the fit of the MSC model to empirical data in
      the context of species delimitation. Our results show substantial variation in
      model fit across data sets. Posterior predictive tests find the poorest model
      performance in data sets that were hypothesized to be impacted by model
      violations. We also show that while the inferences assuming the MSC are robust to
      minor model violations, such inferences can be biased under some biologically
      plausible scenarios. Taken together, these results suggest that researchers can
      identify individual data sets in which species delimitation under the MSC is
      likely to be problematic, thereby highlighting the cases where additional lines
      of evidence to identify species boundaries are particularly important to collect.
      Our study supports a growing body of work highlighting the importance of model
      checking in phylogenetics, and the usefulness of tailoring tests of model fit to 
      assess the reliability of particular inferences. [Populations structure, gene
      flow, demographic changes, posterior prediction, simulation, genetics.].
FAU - Barley, Anthony J
AU  - Barley AJ
AD  - Department of Biology, University of Hawai'i, 2538 McCarthy Mall, Edmondson Hall 
      216, Honolulu, HI 96822, USA.
FAU - Brown, Jeremy M
AU  - Brown JM
AD  - Department of Biological Sciences and Museum of Natural Science, Louisiana State 
      University, 202 Life Sciences Building, Baton Rouge, LA 70803, USA.
FAU - Thomson, Robert C
AU  - Thomson RC
AD  - Department of Biology, University of Hawai'i, 2538 McCarthy Mall, Edmondson Hall 
      216, Honolulu, HI 96822, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.h6s2k
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Genetic Speciation
MH  - *Models, Genetic
EDAT- 2017/09/26 06:00
MHDA- 2018/03/17 06:00
CRDT- 2017/09/26 06:00
PHST- 2017/03/15 00:00 [received]
PHST- 2017/08/31 00:00 [accepted]
PHST- 2017/09/26 06:00 [pubmed]
PHST- 2018/03/17 06:00 [medline]
PHST- 2017/09/26 06:00 [entrez]
AID - 4103348 [pii]
AID - 10.1093/sysbio/syx073 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):269-284. doi: 10.1093/sysbio/syx073.

PMID- 28945876
OWN - NLM
STAT- MEDLINE
DCOM- 20180316
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - Issues and Perspectives in Species Delimitation using Phenotypic Data: Atlantean 
      Evolution in Darwin's Finches.
PG  - 181-194
LID - 10.1093/sysbio/syx071 [doi]
AB  - Progress in the development and use of methods for species delimitation employing
      phenotypic data lags behind conceptual and practical advances in molecular
      genetic approaches. The basic evolutionary model underlying the use of phenotypic
      data to delimit species assumes random mating and quantitative polygenic traits, 
      so that phenotypic distributions within a species should be approximately normal 
      for individuals of the same sex and age. Accordingly, two or more distinct normal
      distributions of phenotypic traits suggest the existence of multiple species. In 
      light of this model, we show that analytical approaches employed in taxonomic
      studies using phenotypic data are often compromised by three issues: 1) reliance 
      on graphical analyses that convey little information on phenotype frequencies; 2)
      exclusion of characters potentially important for species delimitation following 
      reduction of data dimensionality; and 3) use of measures of central tendency to
      evaluate phenotypic distinctiveness. We outline approaches to overcome these
      issues based on statistical developments related to normal mixture models (NMMs) 
      and illustrate them empirically with a reanalysis of morphological data recently 
      used to claim that there are no morphologically distinct species of Darwin's
      ground-finches (Geospiza). We found negligible support for this claim relative to
      taxonomic hypotheses recognizing multiple species. Although species limits among 
      ground-finches merit further assessments using additional sources of information,
      our results bear implications for other areas of inquiry including speciation
      research: because ground-finches have likely speciated and are not trapped in a
      process of "Sisyphean" evolution as recently argued, they remain useful models to
      understand the evolutionary forces involved in speciation. Our work underscores
      the importance of statistical approaches grounded on appropriate evolutionary
      models for species delimitation. We discuss how NMMs offer new perspectives in
      the kind of inferences available to systematists, with significant repercussions 
      on ideas about the phenotypic structure of biodiversity.
FAU - Cadena, Carlos Daniel
AU  - Cadena CD
AD  - Departamento de Ciencias Biologicas, Universidad de los Andes, Bogota, Colombia.
FAU - Zapata, Felipe
AU  - Zapata F
AD  - Department of Ecology and Evolutionary Biology, University of California, Los
      Angeles, CA, USA.
FAU - Jimenez, Ivan
AU  - Jimenez I
AD  - Center for Conservation and Sustainable Development, Missouri Botanical Garden,
      St. Louis, MO, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.9gh90
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - Data Interpretation, Statistical
MH  - Finches/genetics/*physiology
MH  - *Models, Biological
MH  - Phenotype
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2017/09/26 06:00
MHDA- 2018/03/17 06:00
CRDT- 2017/09/26 06:00
PHST- 2017/04/05 00:00 [received]
PHST- 2017/08/30 00:00 [accepted]
PHST- 2017/09/26 06:00 [pubmed]
PHST- 2018/03/17 06:00 [medline]
PHST- 2017/09/26 06:00 [entrez]
AID - 4102004 [pii]
AID - 10.1093/sysbio/syx071 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):181-194. doi: 10.1093/sysbio/syx071.

PMID- 28945874
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20180227
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - Erratum.
PG  - 1065
LID - 10.1093/sysbio/syx069 [doi]
LA  - eng
PT  - Journal Article
PT  - Published Erratum
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EFR - Syst Biol. 2018 Jan 1;67(1):145-157. PMID: 28637223
EDAT- 2017/09/26 06:00
MHDA- 2017/09/26 06:01
CRDT- 2017/09/26 06:00
PHST- 2017/09/26 06:00 [pubmed]
PHST- 2017/09/26 06:01 [medline]
PHST- 2017/09/26 06:00 [entrez]
AID - 4108094 [pii]
AID - 10.1093/sysbio/syx069 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):1065. doi: 10.1093/sysbio/syx069.

PMID- 28945862
OWN - NLM
STAT- MEDLINE
DCOM- 20180316
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 2
DP  - 2018 Mar 1
TI  - Conflicting Evolutionary Histories of the Mitochondrial and Nuclear Genomes in
      New World Myotis Bats.
PG  - 236-249
LID - 10.1093/sysbio/syx070 [doi]
AB  - The rapid diversification of Myotis bats into more than 100 species is one of the
      most extensive mammalian radiations available for study. Efforts to understand
      relationships within Myotis have primarily utilized mitochondrial markers and
      trees inferred from nuclear markers lacked resolution. Our current understanding 
      of relationships within Myotis is therefore biased towards a set of phylogenetic 
      markers that may not reflect the history of the nuclear genome. To resolve this, 
      we sequenced the full mitochondrial genomes of 37 representative Myotis,
      primarily from the New World, in conjunction with targeted sequencing of 3648
      ultraconserved elements (UCEs). We inferred the phylogeny and explored the
      effects of concatenation and summary phylogenetic methods, as well as
      combinations of markers based on informativeness or levels of missing data, on
      our results. Of the 294 phylogenies generated from the nuclear UCE data, all are 
      significantly different from phylogenies inferred using mitochondrial genomes.
      Even within the nuclear data, quartet frequencies indicate that around half of
      all UCE loci conflict with the estimated species tree. Several factors can drive 
      such conflict, including incomplete lineage sorting, introgressive hybridization,
      or even phylogenetic error. Despite the degree of discordance between nuclear UCE
      loci and the mitochondrial genome and among UCE loci themselves, the most common 
      nuclear topology is recovered in one quarter of all analyses with strong nodal
      support. Based on these results, we re-examine the evolutionary history of Myotis
      to better understand the phenomena driving their unique nuclear, mitochondrial,
      and biogeographic histories.
FAU - Platt, Roy N 2nd
AU  - Platt RN 2nd
AD  - Department of Biological Sciences, Texas Tech University, 2901 Main St, Lubbock, 
      TX, USA.
FAU - Faircloth, Brant C
AU  - Faircloth BC
AD  - Department of Biological Sciences and Museum of Natural Science, Louisiana State 
      University, 202 Life Science Building, Baton Rouge, LA, USA.
FAU - Sullivan, Kevin A M
AU  - Sullivan KAM
AD  - Department of Biological Sciences, Texas Tech University, 2901 Main St, Lubbock, 
      TX, USA.
FAU - Kieran, Troy J
AU  - Kieran TJ
AD  - Department of Environmental Health Science, University of Georgia, 206
      Environmental Health Sciences Building, Athens, GA, USA.
FAU - Glenn, Travis C
AU  - Glenn TC
AD  - Department of Environmental Health Science, University of Georgia, 206
      Environmental Health Sciences Building, Athens, GA, USA.
FAU - Vandewege, Michael W
AU  - Vandewege MW
AD  - Department of Biological Sciences, Texas Tech University, 2901 Main St, Lubbock, 
      TX, USA.
FAU - Lee, Thomas E Jr
AU  - Lee TE Jr
AD  - Department of Biology, Abilene Christian University, 1600 Campus Ct. Abilene, TX,
      USA.
FAU - Baker, Robert J
AU  - Baker RJ
AD  - Department of Biological Sciences, Texas Tech University, 2901 Main St, Lubbock, 
      TX, USA.
FAU - Stevens, Richard D
AU  - Stevens RD
AD  - Natural Resource Management, Texas Tech University, 2901 Main St, Lubbock, TX,
      USA.
FAU - Ray, David A
AU  - Ray DA
AD  - Department of Biological Sciences, Texas Tech University, 2901 Main St, Lubbock, 
      TX, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.5g205
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Chiroptera/*classification/*genetics
MH  - Genome/*genetics
MH  - Genome, Mitochondrial/*genetics
MH  - *Phylogeny
PMC - PMC5837689
EDAT- 2017/09/26 06:00
MHDA- 2018/03/17 06:00
CRDT- 2017/09/26 06:00
PHST- 2017/03/17 00:00 [received]
PHST- 2017/08/15 00:00 [accepted]
PHST- 2017/09/26 06:00 [pubmed]
PHST- 2018/03/17 06:00 [medline]
PHST- 2017/09/26 06:00 [entrez]
AID - 4095671 [pii]
AID - 10.1093/sysbio/syx070 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Mar 1;67(2):236-249. doi: 10.1093/sysbio/syx070.

PMID- 28673048
OWN - NLM
STAT- MEDLINE
DCOM- 20180220
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 1
DP  - 2018 Jan 1
TI  - Taming the BEAST-A Community Teaching Material Resource for BEAST 2.
PG  - 170-174
LID - 10.1093/sysbio/syx060 [doi]
AB  - Phylogenetics and phylodynamics are central topics in modern evolutionary
      biology. Phylogenetic methods reconstruct the evolutionary relationships among
      organisms, whereas phylodynamic approaches reveal the underlying diversification 
      processes that lead to the observed relationships. These two fields have many
      practical applications in disciplines as diverse as epidemiology, developmental
      biology, palaeontology, ecology, and linguistics. The combination of increasingly
      large genetic data sets and increases in computing power is facilitating the
      development of more sophisticated phylogenetic and phylodynamic methods. Big data
      sets allow us to answer complex questions. However, since the required analyses
      are highly specific to the particular data set and question, a black-box method
      is not sufficient anymore. Instead, biologists are required to be actively
      involved with modeling decisions during data analysis. The modular design of the 
      Bayesian phylogenetic software package BEAST 2 enables, and in fact enforces,
      this involvement. At the same time, the modular design enables computational
      biology groups to develop new methods at a rapid rate. A thorough understanding
      of the models and algorithms used by inference software is a critical
      prerequisite for successful hypothesis formulation and assessment. In particular,
      there is a need for more readily available resources aimed at helping interested 
      scientists equip themselves with the skills to confidently use cutting-edge
      phylogenetic analysis software. These resources will also benefit researchers who
      do not have access to similar courses or training at their home institutions.
      Here, we introduce the "Taming the Beast" (https://taming-the-beast.github.io/)
      resource, which was developed as part of a workshop series bearing the same name,
      to facilitate the usage of the Bayesian phylogenetic software package BEAST 2.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Barido-Sottani, Joelle
AU  - Barido-Sottani J
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Mattenstrasse 26,
      4058 Basel, Switzerland.
AD  - Swiss Institute of Bioinformatics (SIB), Quartier Sorge - Batiment Genopode, 1015
      Lausanne, Switzerland.
FAU - Boskova, Veronika
AU  - Boskova V
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Mattenstrasse 26,
      4058 Basel, Switzerland.
AD  - Swiss Institute of Bioinformatics (SIB), Quartier Sorge - Batiment Genopode, 1015
      Lausanne, Switzerland.
FAU - Plessis, Louis Du
AU  - Plessis LD
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Mattenstrasse 26,
      4058 Basel, Switzerland.
AD  - Department of Zoology, University of Oxford, Peter Medawar Building South Parks
      Road Oxford, OX1 3SY, UK.
FAU - Kuhnert, Denise
AU  - Kuhnert D
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Mattenstrasse 26,
      4058 Basel, Switzerland.
AD  - Swiss Institute of Bioinformatics (SIB), Quartier Sorge - Batiment Genopode, 1015
      Lausanne, Switzerland.
AD  - Department of Environmental Sciences, ETH Zurich, Universitatsstrasse 16, 8092
      Zurich, Switzerland.
FAU - Magnus, Carsten
AU  - Magnus C
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Mattenstrasse 26,
      4058 Basel, Switzerland.
AD  - Swiss Institute of Bioinformatics (SIB), Quartier Sorge - Batiment Genopode, 1015
      Lausanne, Switzerland.
FAU - Mitov, Venelin
AU  - Mitov V
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Mattenstrasse 26,
      4058 Basel, Switzerland.
AD  - Swiss Institute of Bioinformatics (SIB), Quartier Sorge - Batiment Genopode, 1015
      Lausanne, Switzerland.
FAU - Muller, Nicola F
AU  - Muller NF
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Mattenstrasse 26,
      4058 Basel, Switzerland.
AD  - Swiss Institute of Bioinformatics (SIB), Quartier Sorge - Batiment Genopode, 1015
      Lausanne, Switzerland.
FAU - PecErska, Julija
AU  - PecErska J
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Mattenstrasse 26,
      4058 Basel, Switzerland.
AD  - Swiss Institute of Bioinformatics (SIB), Quartier Sorge - Batiment Genopode, 1015
      Lausanne, Switzerland.
FAU - Rasmussen, David A
AU  - Rasmussen DA
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Mattenstrasse 26,
      4058 Basel, Switzerland.
AD  - Swiss Institute of Bioinformatics (SIB), Quartier Sorge - Batiment Genopode, 1015
      Lausanne, Switzerland.
FAU - Zhang, Chi
AU  - Zhang C
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Mattenstrasse 26,
      4058 Basel, Switzerland.
AD  - Swiss Institute of Bioinformatics (SIB), Quartier Sorge - Batiment Genopode, 1015
      Lausanne, Switzerland.
FAU - Drummond, Alexei J
AU  - Drummond AJ
AD  - Centre for Computational Evolution, University of Auckland, New Zealand.
FAU - Heath, Tracy A
AU  - Heath TA
AD  - Department of Ecology, Evolution, and Organismal Biology, Iowa State University, 
      2200 Osborn Dr., Ames, IA 50011 USA.
FAU - Pybus, Oliver G
AU  - Pybus OG
AD  - Department of Zoology, University of Oxford, Peter Medawar Building South Parks
      Road Oxford, OX1 3SY, UK.
FAU - Vaughan, Timothy G
AU  - Vaughan TG
AD  - Centre for Computational Evolution, University of Auckland, New Zealand.
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Mattenstrasse 26,
      4058 Basel, Switzerland.
AD  - Swiss Institute of Bioinformatics (SIB), Quartier Sorge - Batiment Genopode, 1015
      Lausanne, Switzerland.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Computational Biology/*education/*methods
MH  - *Phylogeny
MH  - *Software
MH  - *Teaching Materials
PMC - PMC5925777
OTO - NOTNLM
OT  - *Bayesian inference
OT  - *MCMC
OT  - *phylodynamics
OT  - *phylogenetics
EDAT- 2017/07/05 06:00
MHDA- 2018/02/21 06:00
CRDT- 2017/07/05 06:00
PHST- 2016/12/21 00:00 [received]
PHST- 2017/06/25 00:00 [accepted]
PHST- 2017/07/05 06:00 [pubmed]
PHST- 2018/02/21 06:00 [medline]
PHST- 2017/07/05 06:00 [entrez]
AID - 3897660 [pii]
AID - 10.1093/sysbio/syx060 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jan 1;67(1):170-174. doi: 10.1093/sysbio/syx060.

PMID- 28645204
OWN - NLM
STAT- MEDLINE
DCOM- 20181023
LR  - 20181023
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 3
DP  - 2018 May 1
TI  - When Homoplasy Is Not Homoplasy: Dissecting Trait Evolution by Contrasting
      Composite and Reductive Coding.
PG  - 543-551
LID - 10.1093/sysbio/syx053 [doi]
AB  - The conceptualization and coding of characters is a difficult issue in
      phylogenetic systematics, no matter which inference method is used when
      reconstructing phylogenetic trees or if the characters are just mapped onto a
      specific tree. Complex characters are groups of features that can be divided into
      simpler hierarchical characters (reductive coding), although the implied
      hierarchical relational information may change depending on the type of coding
      (composite vs. reductive). Up to now, there is no common agreement to either code
      characters as complex or simple. Phylogeneticists have discussed which coding
      method is best but have not incorporated the heuristic process of reciprocal
      illumination to evaluate the coding. Composite coding allows to test whether 1)
      several characters were linked resulting in a structure described as a complex
      character or trait or 2) independently evolving characters resulted in the
      configuration incorrectly interpreted as a complex character. We propose that
      complex characters or character states should be decomposed iteratively into
      simpler characters when the original homology hypothesis is not corroborated by a
      phylogenetic analysis, and the character or character state is retrieved as
      homoplastic. We tested this approach using the case of fruit types within
      subfamily Cinchonoideae (Rubiaceae). The iterative reductive coding of characters
      associated with drupes allowed us to unthread fruit evolution within
      Cinchonoideae. Our results show that drupes and berries are not homologous. As a 
      consequence, a more precise ontology for the Cinchonoideae drupes is required.
FAU - Torres-Montufar, Alejandro
AU  - Torres-Montufar A
AD  - Posgrado en Ciencias Biologicas, Universidad Nacional Autonoma de Mexico, Avenida
      Universidad 3000, Ciudad Universitaria, Coyoacan, Mexico, D.F. C.P. 04510,
      Mexico.
AD  - Departamento de Botanica, Instituto de Biologia, Universidad Nacional Autonoma de
      Mexico, Tercer Circuito s/n, Ciudad Universitaria, Coyoacan, Mexico, D.F. C.P.
      04510, Mexico.
FAU - Borsch, Thomas
AU  - Borsch T
AD  - Botanischer Garten und Botanisches Museum Berlin, Freie Universitat Berlin,
      Konigin-Luise-Strasse 6-8, 14195 Berlin, Germany.
FAU - Ochoterena, Helga
AU  - Ochoterena H
AD  - Departamento de Botanica, Instituto de Biologia, Universidad Nacional Autonoma de
      Mexico, Tercer Circuito s/n, Ciudad Universitaria, Coyoacan, Mexico, D.F. C.P.
      04510, Mexico.
LA  - eng
SI  - Dryad/10.5061/dryad.472s0
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - Classification/*methods
MH  - *Models, Biological
MH  - Phenotype
MH  - *Phylogeny
MH  - Rubiaceae/classification
EDAT- 2017/06/25 06:00
MHDA- 2018/10/24 06:00
CRDT- 2017/06/25 06:00
PHST- 2017/02/03 00:00 [received]
PHST- 2017/05/24 00:00 [accepted]
PHST- 2017/06/25 06:00 [pubmed]
PHST- 2018/10/24 06:00 [medline]
PHST- 2017/06/25 06:00 [entrez]
AID - 3883871 [pii]
AID - 10.1093/sysbio/syx053 [doi]
PST - ppublish
SO  - Syst Biol. 2018 May 1;67(3):543-551. doi: 10.1093/sysbio/syx053.

PMID- 28637293
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - Conserved Nonexonic Elements: A Novel Class of Marker for Phylogenomics.
PG  - 1028-1044
LID - 10.1093/sysbio/syx058 [doi]
AB  - Noncoding markers have a particular appeal as tools for phylogenomic analysis
      because, at least in vertebrates, they appear less subject to strong variation in
      GC content among lineages. Thus far, ultraconserved elements (UCEs) and introns
      have been the most widely used noncoding markers. Here we analyze and study the
      evolutionary properties of a new type of noncoding marker, conserved nonexonic
      elements (CNEEs), which consists of noncoding elements that are estimated to
      evolve slower than the neutral rate across a set of species. Although they often 
      include UCEs, CNEEs are distinct from UCEs because they are not ultraconserved,
      and, most importantly, the core region alone is analyzed, rather than both the
      core and its flanking regions. Using a data set of 16 birds plus an alligator
      outgroup, and approximately 3600- approximately 3800 loci per marker type, we
      found that although CNEEs were less variable than bioinformatically derived UCEs 
      or introns and in some cases exhibited a slower approach to branch resolution as 
      determined by phylogenomic subsampling, the quality of CNEE alignments was
      superior to those of the other markers, with fewer gaps and missing species.
      Phylogenetic resolution using coalescent approaches was comparable among the
      three marker types, with most nodes being fully and congruently resolved.
      Comparison of phylogenetic results across the three marker types indicated that
      one branch, the sister group to the passerine + falcon clade, was resolved
      differently and with moderate (&gt;70%) bootstrap support between CNEEs and UCEs or 
      introns. Overall, CNEEs appear to be promising as phylogenomic markers, yielding 
      phylogenetic resolution as high as for UCEs and introns but with fewer gaps, less
      ambiguity in alignments and with patterns of nucleotide substitution more
      consistent with the assumptions of commonly used methods of phylogenetic
      analysis.
CI  - (c) The Author(s) 2017. Published by Oxford University Press on behalf of the
      Systematic Biologists.
FAU - Edwards, Scott V
AU  - Edwards SV
AD  - Department of Organismic and Evolutionary Biology and Museum of Comparative
      Zoology, 26 Oxford Street, Harvard University, Cambridge, MA 02138 USA.
FAU - Cloutier, Alison
AU  - Cloutier A
AD  - Department of Organismic and Evolutionary Biology and Museum of Comparative
      Zoology, 26 Oxford Street, Harvard University, Cambridge, MA 02138 USA.
AD  - Department of Natural History, Royal Ontario Museum, 100 Queen's Park, Toronto,
      Ontario, M5S 2C6 Canada.
AD  - Department of Ecology and Evolutionary Biology, University of Toronto, 25 Willcox
      Street, Toronto, Ontario, M5S 3B2 Canada.
FAU - Baker, Allan J
AU  - Baker AJ
AD  - Department of Natural History, Royal Ontario Museum, 100 Queen's Park, Toronto,
      Ontario, M5S 2C6 Canada.
AD  - Department of Ecology and Evolutionary Biology, University of Toronto, 25 Willcox
      Street, Toronto, Ontario, M5S 3B2 Canada.
LA  - eng
SI  - Dryad/10.5061/dryad.25f7g
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
SB  - IM
MH  - Alligators and Crocodiles/classification/genetics
MH  - Animals
MH  - Birds/classification/genetics
MH  - Classification/*methods
MH  - Conserved Sequence/*genetics
MH  - *Genetic Markers
MH  - Genome/genetics
MH  - Introns/genetics
MH  - *Phylogeny
PMC - PMC5790140
OTO - NOTNLM
OT  - *Biased-gene conversion
OT  - *conserved element
OT  - *incomplete lineage sorting
OT  - *intron
OT  - *multispecies coalescent
EDAT- 2017/06/24 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/06/23 06:00
PHST- 2016/12/02 00:00 [received]
PHST- 2017/06/06 00:00 [accepted]
PHST- 2017/06/24 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/06/23 06:00 [entrez]
AID - 3868964 [pii]
AID - 10.1093/sysbio/syx058 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):1028-1044. doi: 10.1093/sysbio/syx058.

PMID- 28637223
OWN - NLM
STAT- MEDLINE
DCOM- 20180220
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 1
DP  - 2018 Jan 1
TI  - Millions of Years Behind: Slow Adaptation of Ruminants to Grasslands.
PG  - 145-157
LID - 10.1093/sysbio/syx059 [doi]
AB  - The Late Cretaceous appearance of grasses, followed by the Cenozoic advancement
      of grasslands as dominant biomes, has contributed to the evolution of a range of 
      specialized herbivores adapted to new diets, as well as to increasingly open and 
      arid habitats. Many mammals including ruminants, the most diversified ungulate
      suborder, evolved high-crowned (hypsodont) teeth as an adaptation to
      tooth-wearing diets and habitats. The impact of different causes of tooth wear is
      still a matter of debate, and the temporal pattern of hypsodonty evolution in
      relation to the evolution of grasslands remains unclear. We present an improved
      time-calibrated molecular phylogeny of Cetartiodactyla, with phylogenetic
      reconstruction of ancestral ruminant diets and habitats, based on characteristics
      of extant taxa. Using this timeline, as well as the fossil record of grasslands, 
      we conduct phylogenetic comparative analyses showing that hypsodonty in ruminants
      evolved as an adaptation to both diet and habitat. Our results demonstrate a
      slow, perhaps constrained, evolution of hypsodonty toward estimated optimal
      states, excluding the possibility of immediate adaptation. This augments recent
      findings that slow adaptation is not uncommon on million-year time scales.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Toljagic, Olja
AU  - Toljagic O
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis,
      University of Oslo, Blindernveien 31, NO-0371 Oslo, Norway.
FAU - Voje, Kjetil L
AU  - Voje KL
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis,
      University of Oslo, Blindernveien 31, NO-0371 Oslo, Norway.
FAU - Matschiner, Michael
AU  - Matschiner M
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis,
      University of Oslo, Blindernveien 31, NO-0371 Oslo, Norway.
AD  - Zoological Institute, University of Basel, Vesalgasse 1, 4051 Basel, Switzerland.
FAU - Liow, Lee Hsiang
AU  - Liow LH
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis,
      University of Oslo, Blindernveien 31, NO-0371 Oslo, Norway.
AD  - Natural History Museum, University of Oslo, Sars gate 1, NO-0562 Oslo, Norway.
FAU - Hansen, Thomas F
AU  - Hansen TF
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis,
      University of Oslo, Blindernveien 31, NO-0371 Oslo, Norway.
LA  - eng
SI  - Dryad/10.5061/dryad.b2v7v
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2017 Nov 1;66(6):1065. PMID: 28945874
MH  - Adaptation, Physiological/*physiology
MH  - Animals
MH  - *Biological Evolution
MH  - Diet
MH  - Fossils
MH  - *Grassland
MH  - Phylogeny
MH  - Poaceae
MH  - Ruminants/anatomy &amp; histology/*classification/genetics
OTO - NOTNLM
OT  - *Cetartiodactyla
OT  - *Macroevolution
OT  - *Ornstein-Uhlenbeck process
OT  - *adaptation
OT  - *grassland
OT  - *hypsodonty
OT  - *phylogenetic comparative method
OT  - *ruminant
EDAT- 2017/06/24 06:00
MHDA- 2018/02/21 06:00
CRDT- 2017/06/23 06:00
PHST- 2016/08/18 00:00 [received]
PHST- 2017/06/15 00:00 [accepted]
PHST- 2017/06/24 06:00 [pubmed]
PHST- 2018/02/21 06:00 [medline]
PHST- 2017/06/23 06:00 [entrez]
AID - 3871340 [pii]
AID - 10.1093/sysbio/syx059 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jan 1;67(1):145-157. doi: 10.1093/sysbio/syx059.

PMID- 28633306
OWN - NLM
STAT- MEDLINE
DCOM- 20180220
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 1
DP  - 2018 Jan 1
TI  - Multivariate Phylogenetic Comparative Methods: Evaluations, Comparisons, and
      Recommendations.
PG  - 14-31
LID - 10.1093/sysbio/syx055 [doi]
AB  - Recent years have seen increased interest in phylogenetic comparative analyses of
      multivariate data sets, but to date the varied proposed approaches have not been 
      extensively examined. Here we review the mathematical properties required of any 
      multivariate method, and specifically evaluate existing multivariate phylogenetic
      comparative methods in this context. Phylogenetic comparative methods based on
      the full multivariate likelihood are robust to levels of covariation among trait 
      dimensions and are insensitive to the orientation of the data set, but display
      increasing model misspecification as the number of trait dimensions increases.
      This is because the expected evolutionary covariance matrix (V) used in the
      likelihood calculations becomes more ill-conditioned as trait dimensionality
      increases, and as evolutionary models become more complex. Thus, these approaches
      are only appropriate for data sets with few traits and many species. Methods that
      summarize patterns across trait dimensions treated separately (e.g., SURFACE)
      incorrectly assume independence among trait dimensions, resulting in nearly a
      100% model misspecification rate. Methods using pairwise composite likelihood are
      highly sensitive to levels of trait covariation, the orientation of the data set,
      and the number of trait dimensions. The consequences of these debilitating
      deficiencies are that a user can arrive at differing statistical conclusions, and
      therefore biological inferences, simply from a dataspace rotation, like principal
      component analysis. By contrast, algebraic generalizations of the standard
      phylogenetic comparative toolkit that use the trace of covariance matrices are
      insensitive to levels of trait covariation, the number of trait dimensions, and
      the orientation of the data set. Further, when appropriate permutation tests are 
      used, these approaches display acceptable Type I error and statistical power. We 
      conclude that methods summarizing information across trait dimensions, as well as
      pairwise composite likelihood methods should be avoided, whereas algebraic
      generalizations of the phylogenetic comparative toolkit provide a useful means of
      assessing macroevolutionary patterns in multivariate data. Finally, we discuss
      areas in which multivariate phylogenetic comparative methods are still in need of
      future development; namely highly multivariate Ornstein-Uhlenbeck models and
      approaches for multivariate evolutionary model comparisons.
CI  - (c) The Author(s) 2017. Published by Oxford University Press on behalf of the
      Systematic Biology. All rights reserved. For Permissions, please email:
      journals.permissions@oup.com.
FAU - Adams, Dean C
AU  - Adams DC
AD  - Department of Ecology, Evolution, and Organismal Biology, Iowa State University, 
      Ames, IA, USA.
AD  - Department of Statistics, Iowa State University, Ames, IA, USA.
FAU - Collyer, Michael L
AU  - Collyer ML
AD  - Department of Science, Chatham University, Pittsburgh, PA, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.29722
PT  - Journal Article
PT  - Review
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Classification
MH  - Computer Simulation
MH  - Models, Biological
MH  - *Multivariate Analysis
MH  - Phylogeny
OTO - NOTNLM
OT  - *Multivariate
OT  - *high-dimensional data
OT  - *phylogenetic comparative methods
EDAT- 2017/06/22 06:00
MHDA- 2018/02/21 06:00
CRDT- 2017/06/22 06:00
PHST- 2016/11/13 00:00 [received]
PHST- 2017/05/30 00:00 [accepted]
PHST- 2017/06/22 06:00 [pubmed]
PHST- 2018/02/21 06:00 [medline]
PHST- 2017/06/22 06:00 [entrez]
AID - 3867043 [pii]
AID - 10.1093/sysbio/syx055 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jan 1;67(1):14-31. doi: 10.1093/sysbio/syx055.

PMID- 28595366
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - Conserving Phylogenetic Diversity Can Be a Poor Strategy for Conserving
      Functional Diversity.
PG  - 1019-1027
LID - 10.1093/sysbio/syx054 [doi]
AB  - For decades, academic biologists have advocated for making conservation decisions
      in light of evolutionary history. Specifically, they suggest that policy makers
      should prioritize conserving phylogenetically diverse assemblages. The most
      prominent argument is that conserving phylogenetic diversity (PD) will also
      conserve diversity in traits and features (functional diversity [FD]), which may 
      be valuable for a number of reasons. The claim that PD-maximized ("maxPD") sets
      of taxa will also have high FD is often taken at face value and in cases where
      researchers have actually tested it, they have done so by measuring the
      phylogenetic signal in ecologically important functional traits. The rationale is
      that if traits closely mirror phylogeny, then saving the maxPD set of taxa will
      tend to maximize FD and if traits do not have phylogenetic structure, then saving
      the maxPD set of taxa will be no better at capturing FD than criteria that ignore
      PD. Here, we suggest that measuring the phylogenetic signal in traits is
      uninformative for evaluating the effectiveness of using PD in conservation. We
      evolve traits under several different models and, for the first time, directly
      compare the FD of a set of taxa that maximize PD to the FD of a random set of the
      same size. Under many common models of trait evolution and tree shapes,
      conserving the maxPD set of taxa will conserve more FD than conserving a random
      set of the same size. However, this result cannot be generalized to other classes
      of models. We find that under biologically plausible scenarios, using PD to
      select species can actually lead to less FD compared with a random set.
      Critically, this can occur even when there is phylogenetic signal in the traits. 
      Predicting exactly when we expect using PD to be a good strategy for conserving
      FD is challenging, as it depends on complex interactions between tree shape and
      the assumptions of the evolutionary model. Nonetheless, if our goal is to
      maintain trait diversity, the fact that conserving taxa based on PD will not
      reliably conserve at least as much FD as choosing randomly raises serious
      concerns about the general utility of PD in conservation.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Mazel, Florent
AU  - Mazel F
AD  - Department of Biological Sciences, Simon Fraser University, 8888 University
      Drive, Burnaby BC V5A 1S6, Canada.
FAU - Mooers, Arne O
AU  - Mooers AO
AD  - Department of Biological Sciences, Simon Fraser University, 8888 University
      Drive, Burnaby BC V5A 1S6, Canada.
FAU - Riva, Giulio Valentino Dalla
AU  - Riva GVD
AD  - Department of Zoology and Biodiversity Research Centre, University of British
      Columbia, Vancouver BC V6T 1Z4, Canada.
FAU - Pennell, Matthew W
AU  - Pennell MW
AD  - Department of Zoology and Biodiversity Research Centre, University of British
      Columbia, Vancouver BC V6T 1Z4, Canada.
LA  - eng
SI  - Dryad/10.5061/dryad.57rk7
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biodiversity
MH  - Biological Evolution
MH  - *Conservation of Natural Resources
MH  - Environmental Policy
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Conservation
OT  - *evolutionary diversity
OT  - *functional diversity
OT  - *species prioritization
OT  - *trait evolution
EDAT- 2017/06/09 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/06/09 06:00
PHST- 2017/02/20 00:00 [received]
PHST- 2017/05/24 00:00 [accepted]
PHST- 2017/06/09 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/06/09 06:00 [entrez]
AID - 3863141 [pii]
AID - 10.1093/sysbio/syx054 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):1019-1027. doi: 10.1093/sysbio/syx054.

PMID- 28595363
OWN - NLM
STAT- MEDLINE
DCOM- 20180130
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 5
DP  - 2017 Sep 1
TI  - More on the Best Evolutionary Rate for Phylogenetic Analysis.
PG  - 769-785
LID - 10.1093/sysbio/syx051 [doi]
AB  - The accumulation of genome-scale molecular data sets for nonmodel taxa brings us 
      ever closer to resolving the tree of life of all living organisms. However,
      despite the depth of data available, a number of studies that each used thousands
      of genes have reported conflicting results. The focus of phylogenomic projects
      must thus shift to more careful experimental design. Even though we still have a 
      limited understanding of what are the best predictors of the phylogenetic
      informativeness of a gene, there is wide agreement that one key factor is its
      evolutionary rate; but there is no consensus as to whether the rates derived as
      optimal in various analytical, empirical, and simulation approaches have any
      general applicability. We here use simulations to infer optimal rates in a set of
      realistic phylogenetic scenarios with varying tree sizes, numbers of terminals,
      and tree shapes. Furthermore, we study the relationship between the optimal rate 
      and rate variation among sites and among lineages. Finally, we examine how well
      the predictions made by a range of experimental design methods correlate with the
      observed performance in our simulations.We find that the optimal level of
      divergence is surprisingly robust to differences in taxon sampling and even to
      among-site and among-lineage rate variation as often encountered in empirical
      data sets. This finding encourages the use of methods that rely on a single
      optimal rate to predict a gene's utility. Focusing on correct recovery either of 
      the most basal node in the phylogeny or of the entire topology, the optimal rate 
      is about 0.45 substitutions from root to tip in average Yule trees and about 0.2 
      in difficult trees with short basal and long-apical branches, but all rates
      leading to divergence levels between about 0.1 and 0.5 perform reasonably
      well.Testing the performance of six methods that can be used to predict a gene's 
      utility against our simulation results, we find that the probability of
      resolution, signal-noise analysis, and Fisher information are good predictors of 
      phylogenetic informativeness, but they require specification of at least part of 
      a model tree. Likelihood quartet mapping also shows very good performance but
      only requires sequence alignments and is thus applicable without making
      assumptions about the phylogeny. Despite them being the most commonly used
      methods for experimental design, geometric quartet mapping and the integration of
      phylogenetic informativeness curves perform rather poorly in our comparison.
      Instead of derived predictors of phylogenetic informativeness, we suggest that
      the number of sites in a gene that evolve at near-optimal rates (as inferred
      here) could be used directly to prioritize genes for phylogenetic inference. In
      combination with measures of model fit, especially with respect to compositional 
      biases and among-site and among-lineage rate variation, such an approach has the 
      potential to greatly improve marker choice and should be tested on empirical
      data.
CI  - (c) The Author(s) 2017. Published by Oxford University Press on behalf of the
      Systematic Biology.
FAU - Klopfstein, Seraina
AU  - Klopfstein S
AD  - Naturhistorisches Museum der Burgergemeinde Bern, Bernastr. 15, CH-3005 Bern,
      Switzerland.
AD  - University of Bern, Institute of Ecology and Evolution, Baltzerstr. 6, CH-3012
      Bern, Switzerland.
FAU - Massingham, Tim
AU  - Massingham T
AD  - European Molecular Biology Laboratory, European Bioinformatics Institute
      (EMBL-EBI), Wellcome Genome Campus, Hinxton CB10 1SD, UK.
FAU - Goldman, Nick
AU  - Goldman N
AD  - European Molecular Biology Laboratory, European Bioinformatics Institute
      (EMBL-EBI), Wellcome Genome Campus, Hinxton CB10 1SD, UK.
LA  - eng
SI  - Dryad/10.5061/dryad.s342d
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Genome/genetics
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Probability
MH  - Time
PMC - PMC5790136
OTO - NOTNLM
OT  - *Experimental design
OT  - *phylogenomics
EDAT- 2017/06/09 06:00
MHDA- 2018/01/31 06:00
CRDT- 2017/06/09 06:00
PHST- 2016/08/19 00:00 [received]
PHST- 2017/05/24 00:00 [accepted]
PHST- 2017/06/09 06:00 [pubmed]
PHST- 2018/01/31 06:00 [medline]
PHST- 2017/06/09 06:00 [entrez]
AID - 3863140 [pii]
AID - 10.1093/sysbio/syx051 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Sep 1;66(5):769-785. doi: 10.1093/sysbio/syx051.

PMID- 28482055
OWN - NLM
STAT- MEDLINE
DCOM- 20180220
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 1
DP  - 2018 Jan 1
TI  - Combined Analysis of Extant Rhynchonellida (Brachiopoda) using Morphological and 
      Molecular Data.
PG  - 32-48
LID - 10.1093/sysbio/syx049 [doi]
AB  - Independent molecular and morphological phylogenetic analyses have often produced
      discordant results for certain groups which, for fossil-rich groups, raises the
      possibility that morphological data might mislead in those groups for which we
      depend upon morphology the most. Rhynchonellide brachiopods, with more than 500
      extinct genera but only 19 extant genera represented today, provide an
      opportunity to explore the factors that produce contentious phylogenetic signal
      across datasets, as previous phylogenetic hypotheses generated from molecular
      sequence data bear little agreement with those constructed using morphological
      characters. Using a revised matrix of 66 morphological characters, and published 
      ribosomal DNA sequences, we performed a series of combined phylogenetic analyses 
      to identify conflicting phylogenetic signals. We completed a series of
      parsimony-based and Bayesian analyses, varying the data used, the taxa included, 
      and the models used in the Bayesian analyses. We also performed simulation-based 
      sensitivity analyses to assess whether the small size of the morphological data
      partition relative to the molecular data influenced the results of the combined
      analyses. In order to compare and contrast a large number of phylogenetic
      analyses and their resulting summary trees, we developed a measure for the
      incongruence between two topologies and simultaneously ignore any differences in 
      phylogenetic resolution. Phylogenetic hypotheses generated using only
      morphological characters differed among each other, and with previous analyses,
      whereas molecular-only and combined Bayesian analyses produced extremely similar 
      topologies. Characters historically associated with traditional classification in
      the Rhynchonellida have very low consistency indices on the topology preferred by
      the combined Bayesian analyses. Overall, this casts doubt on the use of
      morphological systematics to resolve relationships among the crown rhynchonellide
      brachiopods. However, expanding our dataset to a larger number of extinct taxa
      with intermediate morphologies is necessary to exclude the possibility that the
      morphology of extant taxa is not dominated by convergence along long branches.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Bapst, David W
AU  - Bapst DW
AD  - Department of Earth and Planetary Sciences, University of California, Davis, One 
      Shields Avenue, Davis, CA 95616, USA.
FAU - Schreiber, Holly A
AU  - Schreiber HA
AD  - Department of Earth and Planetary Sciences, University of California, Davis, One 
      Shields Avenue, Davis, CA 95616, USA.
AD  - Penn Dixie Fossil Park and Nature Reserve, 3556 Lakeshore Rd, Ste. 210 Blasdell, 
      NY 14219, USA.
FAU - Carlson, Sandra J
AU  - Carlson SJ
AD  - Department of Earth and Planetary Sciences, University of California, Davis, One 
      Shields Avenue, Davis, CA 95616, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.31048.2
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Computer Simulation
MH  - DNA, Ribosomal/genetics
MH  - *Fossils
MH  - Invertebrates/anatomy &amp; histology/*classification/genetics
MH  - Models, Biological
MH  - *Phylogeny
PMC - PMC5790130
OTO - NOTNLM
OT  - *Brachiopoda
OT  - *combined analyses
OT  - *morphology
OT  - *paleontology
OT  - *phylogenetics
EDAT- 2017/05/10 06:00
MHDA- 2018/02/21 06:00
CRDT- 2017/05/09 06:00
PHST- 2016/08/05 00:00 [received]
PHST- 2017/04/28 00:00 [accepted]
PHST- 2017/05/10 06:00 [pubmed]
PHST- 2018/02/21 06:00 [medline]
PHST- 2017/05/09 06:00 [entrez]
AID - 3805381 [pii]
AID - 10.1093/sysbio/syx049 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jan 1;67(1):32-48. doi: 10.1093/sysbio/syx049.

PMID- 28472519
OWN - NLM
STAT- MEDLINE
DCOM- 20180220
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 1
DP  - 2018 Jan 1
TI  - Resolving Relationships among the Megadiverse Butterflies and Moths with a Novel 
      Pipeline for Anchored Phylogenomics.
PG  - 78-93
LID - 10.1093/sysbio/syx048 [doi]
AB  - The advent of next-generation sequencing technology has allowed for thecollection
      of large portions of the genome for phylogenetic analysis. Hybrid enrichment and 
      transcriptomics are two techniques that leverage next-generation sequencing and
      have shown much promise. However, methods for processing hybrid enrichment data
      are still limited. We developed a pipeline for anchored hybrid enrichment (AHE)
      read assembly, orthology determination, contamination screening, and data
      processing for sequences flanking the target "probe" region. We apply this
      approach to study the phylogeny of butterflies and moths (Lepidoptera), a
      megadiverse group of more than 157,000 described species with poorly understood
      deep-level phylogenetic relationships. We introduce a new, 855 locus AHE kit for 
      Lepidoptera phylogenetics and compare resulting trees to those from
      transcriptomes. The enrichment kit was designed from existing genomes,
      transcriptomes, and expressed sequence tags and was used to capture sequence data
      from 54 species from 23 lepidopteran families. Phylogenies estimated from AHE
      data were largely congruent with trees generated from transcriptomes, with strong
      support for relationships at all but the deepest taxonomic levels. We combine AHE
      and transcriptomic data to generate a new Lepidoptera phylogeny, representing 76 
      exemplar species in 42 families. The tree provides robust support for many
      relationships, including those among the seven butterfly families. The addition
      of AHE data to an existing transcriptomic dataset lowers node support along the
      Lepidoptera backbone, but firmly places taxa with AHE data on the phylogeny.
      Combining taxa sequenced for AHE with existing transcriptomes and genomes
      resulted in a tree with strong support for (Calliduloidea $+$ Gelechioidea $+$
      Thyridoidea) $+$ (Papilionoidea $+$ Pyraloidea $+$ Macroheterocera). To examine
      the efficacy of AHE at a shallow taxonomic level, phylogenetic analyses were also
      conducted on a sister group representing a more recent divergence, the
      Saturniidae and Sphingidae. These analyses utilized sequences from the probe
      region and data flanking it, nearly doubled the size of the dataset; resulting
      trees supported new phylogenetics relationships, especially within the
      Saturniidae and Sphingidae (e.g., Hemarina derived in the latter). We hope that
      our data processing pipeline, hybrid enrichment gene set, and approach of
      combining AHE data with transcriptomes will be useful for the broader systematics
      community.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Breinholt, Jesse W
AU  - Breinholt JW
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL 32611, 
      USA.
AD  - RAPiD Genomics, Gainesville, FL 32601, USA.
FAU - Earl, Chandra
AU  - Earl C
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL 32611, 
      USA.
FAU - Lemmon, Alan R
AU  - Lemmon AR
AD  - Department of Scientific Computing.
FAU - Lemmon, Emily Moriarty
AU  - Lemmon EM
AD  - Department of Biological Science, Florida State University, Tallahassee, FL
      32306, USA.
FAU - Xiao, Lei
AU  - Xiao L
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL 32611, 
      USA.
FAU - Kawahara, Akito Y
AU  - Kawahara AY
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL 32611, 
      USA.
LA  - eng
SI  - Dryad/10.5061/dryad.rf7g5
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Butterflies/*classification/genetics
MH  - Classification/*methods
MH  - Moths/*classification/genetics
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
OTO - NOTNLM
OT  - *Exon
OT  - *hybrid enrichment
OT  - *phylogenetics
OT  - *sequence capture
OT  - *target capture
OT  - *transcriptomics
OT  - *ultraconserved elements
EDAT- 2017/05/05 06:00
MHDA- 2018/02/21 06:00
CRDT- 2017/05/05 06:00
PHST- 2016/07/01 00:00 [received]
PHST- 2017/04/28 00:00 [accepted]
PHST- 2017/05/05 06:00 [pubmed]
PHST- 2018/02/21 06:00 [medline]
PHST- 2017/05/05 06:00 [entrez]
AID - 3796843 [pii]
AID - 10.1093/sysbio/syx048 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jan 1;67(1):78-93. doi: 10.1093/sysbio/syx048.

PMID- 28472459
OWN - NLM
STAT- MEDLINE
DCOM- 20180220
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 1
DP  - 2018 Jan 1
TI  - Resolving Rapid Radiations within Angiosperm Families Using Anchored
      Phylogenomics.
PG  - 94-112
LID - 10.1093/sysbio/syx050 [doi]
AB  - Despite the promise that molecular data would provide a seemingly unlimited
      source of independent characters, many plant phylogenetic studies are still based
      on only two regions, the plastid genome and nuclear ribosomal DNA (nrDNA). Their 
      popularity can be explained by high-copy numbers and universal polymerase chain
      reaction (PCR) primers that make their sequences easily amplified and converted
      into parallel datasets. Unfortunately, their utility is limited by linked loci
      and limited characters resulting in low confidence in the accuracy of
      phylogenetic estimates, especially when rapid radiations occur. In another
      contribution on anchored phylogenomics in angiosperms, we presented flowering
      plant-specific anchored enrichment probes for hundreds of conserved nuclear genes
      and demonstrated their use at the level of all angiosperms. In this contribution,
      we focus on a common problem in phylogenetic reconstructions below the family
      level: Weak or unresolved backbone due to rapid radiations ($\leqslant $10
      million years) followed by long divergence, using the
      Cariceae-Dulichieae-Scirpeae (CDS, Cyperaceae) clade as a test case. By comparing
      our nuclear matrix of 461 genes to a typical Sanger-sequence dataset consisting
      of a few plastid genes (matK, ndhF) and an nrDNA marker (ETS), we demonstrate
      that our nuclear data is fully compatible with the Sanger dataset and resolves
      short backbone internodes with high support in both concatenated and
      coalescence-based analyses. In addition, we show that nuclear gene tree
      incongruence is inversely proportional to phylogenetic information content,
      indicating that incongruence is mostly due to gene tree estimation error. This
      suggests that large numbers of conserved nuclear loci could produce more accurate
      trees than sampling rapidly evolving regions prone to saturation and long-branch 
      attraction. The robust phylogenetic estimates obtained here, and high congruence 
      with previous morphological and molecular analyses, are strong evidence for a
      complete tribal revision of CDS clade. The anchored hybrid enrichment probes used
      in this study should be similarly effective in other flowering plant groups.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Leveille-Bourret, Etienne
AU  - Leveille-Bourret E
AD  - Department of Biology, University of Ottawa, 30 Marie Curie, K1N 6N5 Ottawa,
      Ontario, Canada.
FAU - Starr, Julian R
AU  - Starr JR
AD  - Department of Biology, University of Ottawa, 30 Marie Curie, K1N 6N5 Ottawa,
      Ontario, Canada.
FAU - Ford, Bruce A
AU  - Ford BA
AD  - Department of Biological Sciences, University of Manitoba, 50 Sifton Road, R3T
      2N2 Winnipeg, Manitoba, Canada.
FAU - Moriarty Lemmon, Emily
AU  - Moriarty Lemmon E
AD  - Department of Biological Science, 89 Chieftain Way.
FAU - Lemmon, Alan R
AU  - Lemmon AR
AD  - Department of Scientific Computing, Florida State University, Tallahassee, FL
      32306, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.55h30
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Genetic Speciation
MH  - Genome, Plant
MH  - Magnoliopsida/*classification/*genetics
MH  - *Phylogeny
MH  - Reproducibility of Results
OTO - NOTNLM
OT  - *Carex
OT  - *coalescent based species tree
OT  - *flowering plants
OT  - *low-copy nuclear genes
OT  - *low-level phylogenetics
OT  - *universal hybrid enrichment probes
EDAT- 2017/05/05 06:00
MHDA- 2018/02/21 06:00
CRDT- 2017/05/05 06:00
PHST- 2016/11/09 00:00 [received]
PHST- 2017/04/28 00:00 [accepted]
PHST- 2017/05/05 06:00 [pubmed]
PHST- 2018/02/21 06:00 [medline]
PHST- 2017/05/05 06:00 [entrez]
AID - 3796844 [pii]
AID - 10.1093/sysbio/syx050 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jan 1;67(1):94-112. doi: 10.1093/sysbio/syx050.

PMID- 28472435
OWN - NLM
STAT- MEDLINE
DCOM- 20180220
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 1
DP  - 2018 Jan 1
TI  - A Metric on Phylogenetic Tree Shapes.
PG  - 113-126
LID - 10.1093/sysbio/syx046 [doi]
AB  - The shapes of evolutionary trees are influenced by the nature of the evolutionary
      process but comparisons of trees from different processes are hindered by the
      challenge of completely describing tree shape. We present a full characterization
      of the shapes of rooted branching trees in a form that lends itself to natural
      tree comparisons. We use this characterization to define a metric, in the sense
      of a true distance function, on tree shapes. The metric distinguishes trees from 
      random models known to produce different tree shapes. It separates trees derived 
      from tropical versus USA influenza A sequences, which reflect the differing
      epidemiology of tropical and seasonal flu. We describe several metrics based on
      the same core characterization, and illustrate how to extend the metric to
      incorporate trees' branch lengths or other features such as overall imbalance.
      Our approach allows us to construct addition and multiplication on trees, and to 
      create a convex metric on tree shapes which formally allows computation of
      average tree shapes.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Colijn, C
AU  - Colijn C
AD  - Department of Mathematics, Imperial College, 180 Queen's Gate, London SW7 2AZ,
      UK.
FAU - Plazzotta, G
AU  - Plazzotta G
AD  - Department of Mathematics, Imperial College, 180 Queen's Gate, London SW7 2AZ,
      UK.
LA  - eng
SI  - Dryad/10.5061/dryad.3r8v1
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - *Classification
MH  - *Models, Genetic
MH  - *Phylogeny
PMC - PMC5790134
OTO - NOTNLM
OT  - *phylodynamics
OT  - *tree metric
OT  - *tree shapes
EDAT- 2017/05/05 06:00
MHDA- 2018/02/21 06:00
CRDT- 2017/05/05 06:00
PHST- 2016/07/29 00:00 [received]
PHST- 2017/04/11 00:00 [accepted]
PHST- 2017/05/05 06:00 [pubmed]
PHST- 2018/02/21 06:00 [medline]
PHST- 2017/05/05 06:00 [entrez]
AID - 3788885 [pii]
AID - 10.1093/sysbio/syx046 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jan 1;67(1):113-126. doi: 10.1093/sysbio/syx046.

PMID- 28472434
OWN - NLM
STAT- MEDLINE
DCOM- 20180220
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 67
IP  - 1
DP  - 2018 Jan 1
TI  - Lineage Diversity and Size Disparity in Musteloidea: Testing Patterns of Adaptive
      Radiation Using Molecular and Fossil-Based Methods.
PG  - 127-144
LID - 10.1093/sysbio/syx047 [doi]
AB  - Adaptive radiation is hypothesized to be a primary mechanism that drives the
      remarkable species diversity and morphological disparity across the Tree of Life.
      Tests for adaptive radiation in extant taxa are traditionally estimated from
      calibrated molecular phylogenies with little input from extinct taxa. With 85
      putative species in 33 genera and over 400 described extinct species, the
      carnivoran superfamily Musteloidea is a prime candidate to investigate patterns
      of adaptive radiation using both extant- and fossil-based macroevolutionary
      methods. The species diversity and equally impressive ecological and phenotypic
      diversity found across Musteloidea is often attributed to two adaptive radiations
      coinciding with two major climate events, the Eocene-Oligocene transition and the
      Mid-Miocene Climate Transition. Here, we compiled a novel time-scaled phylogeny
      for 88% of extant musteloids and used it as a framework for testing the
      predictions of adaptive radiation hypotheses with respect to rates of lineage
      diversification and phenotypic evolution. Contrary to expectations, we found no
      evidence for rapid bursts of lineage diversification at the origin of
      Musteloidea, and further analyses of lineage diversification rates using
      molecular and fossil-based methods did not find associations between rates of
      lineage diversification and the Eocene-Oligocene transition or Mid-Miocene
      Climate Transition as previously hypothesized. Rather, we found support for
      decoupled diversification dynamics driven by increased clade carrying capacity in
      the branches leading to a subclade of elongate mustelids. Supporting decoupled
      diversification dynamics between the subclade of elongate mustelids and the
      ancestral musteloid regime is our finding of increased rates of body length
      evolution, but not body mass evolution, within the decoupled mustelid subclade.
      The lack of correspondence in rates of body mass and length evolution suggest
      that phenotypic evolutionary rates under a single morphological metric, even one 
      as influential as mass, may not capture the evolution of diversity in clades that
      exhibit elongate body shapes. The discordance in evolutionary rates between body 
      length and body mass along with evidence of decoupled diversification dynamics
      suggests that body elongation might be an innovation for the exploitation of
      novel Mid-Miocene resources, resulting in the radiation of some musteloids.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Law, Chris J
AU  - Law CJ
AD  - Department of Ecology and Evolutionary Biology, Long Marine Lab, University of
      California, Santa Cruz, 115 McAllister Way, Santa Cruz, CA 95060, USA.
FAU - Slater, Graham J
AU  - Slater GJ
AD  - Department of the Geophysical Sciences, University of Chicago, 5734 S. Ellis
      Avenue, Chicago, IL 60637 USA.
FAU - Mehta, Rita S
AU  - Mehta RS
AD  - Department of Ecology and Evolutionary Biology, Long Marine Lab, University of
      California, Santa Cruz, 115 McAllister Way, Santa Cruz, CA 95060, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.nj1bp
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - *Body Size
MH  - Carnivora/anatomy &amp; histology/*classification/genetics
MH  - *Fossils
MH  - Genetic Speciation
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Carnivora
OT  - *adaptive radiation
OT  - *body size evolution
OT  - *evolutionary rates
OT  - *key innovation
OT  - *lineage diversification
OT  - *phylogeny
EDAT- 2017/05/05 06:00
MHDA- 2018/02/21 06:00
CRDT- 2017/05/05 06:00
PHST- 2015/06/04 00:00 [received]
PHST- 2017/04/28 00:00 [accepted]
PHST- 2017/05/05 06:00 [pubmed]
PHST- 2018/02/21 06:00 [medline]
PHST- 2017/05/05 06:00 [entrez]
AID - 3796842 [pii]
AID - 10.1093/sysbio/syx047 [doi]
PST - ppublish
SO  - Syst Biol. 2018 Jan 1;67(1):127-144. doi: 10.1093/sysbio/syx047.

PMID- 28460129
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20180226
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Corrigendum.
PG  - 662
LID - 10.1093/sysbio/syx034 [doi]
LA  - eng
PT  - Journal Article
PT  - Published Erratum
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EFR - Syst Biol. 2017 Jul 1;66(4):637-643. PMID: 27770056
EDAT- 2017/05/02 06:00
MHDA- 2017/05/02 06:01
CRDT- 2017/05/02 06:00
PHST- 2017/05/02 06:00 [pubmed]
PHST- 2017/05/02 06:01 [medline]
PHST- 2017/05/02 06:00 [entrez]
AID - 3778223 [pii]
AID - 10.1093/sysbio/syx034 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):662. doi: 10.1093/sysbio/syx034.

PMID- 28431159
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - Likelihood of Tree Topologies with Fossils and Diversification Rate Estimation.
PG  - 964-987
LID - 10.1093/sysbio/syx045 [doi]
AB  - Since the diversification process cannot be directly observed at the human scale,
      it has to be studied from the information available, namely the extant taxa and
      the fossil record. In this sense, phylogenetic trees including both extant taxa
      and fossils are the most complete representations of the diversification process 
      that one can get. Such phylogenetic trees can be reconstructed from molecular and
      morphological data, to some extent. Among the temporal information of such
      phylogenetic trees, fossil ages are by far the most precisely known (divergence
      times are inferences calibrated mostly with fossils). We propose here a method to
      compute the likelihood of a phylogenetic tree with fossils in which the only
      considered time information is the fossil ages, and apply it to the estimation of
      the diversification rates from such data. Since it is required in our
      computation, we provide a method for determining the probability of a tree
      topology under the standard diversification model. Testing our approach on
      simulated data shows that the maximum likelihood rate estimates from the
      phylogenetic tree topology and the fossil dates are almost as accurate as those
      obtained by taking into account all the data, including the divergence times.
      Moreover, they are substantially more accurate than the estimates obtained only
      from the exact divergence times (without taking into account the fossil record). 
      We also provide an empirical example composed of 50 Permo-Carboniferous
      eupelycosaur (early synapsid) taxa ranging in age from about 315 Ma (Late
      Carboniferous) to 270 Ma (shortly after the end of the Early Permian). Our
      analyses suggest a speciation (cladogenesis, or birth) rate of about 0.1 per
      lineage and per myr, a marginally lower extinction rate, and a considerable
      hidden paleobiodiversity of early synapsids. [Extinction rate; fossil ages;
      maximum likelihood estimation; speciation rate.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Didier, Gilles
AU  - Didier G
AD  - Aix Marseille Univ, CNRS, Centrale Marseille, I2M, Marseille, France.
FAU - Fau, Marine
AU  - Fau M
AD  - Departement Histoire de la Terre CR2P, Sorbonne Universites, CNRS/MNHN/UPMC,
      Museum National d'Histoire Naturelle, 75005 Paris, France.
FAU - Laurin, Michel
AU  - Laurin M
AD  - Departement Histoire de la Terre CR2P, Sorbonne Universites, CNRS/MNHN/UPMC,
      Museum National d'Histoire Naturelle, 75005 Paris, France.
LA  - eng
SI  - Dryad/10.5061/dryad.96d55
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Computer Simulation
MH  - *Fossils
MH  - *Genetic Speciation
MH  - *Phylogeny
MH  - Reptiles/classification
MH  - Time
EDAT- 2017/04/22 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/04/22 06:00
PHST- 2016/05/11 00:00 [received]
PHST- 2017/04/05 00:00 [accepted]
PHST- 2017/04/22 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/04/22 06:00 [entrez]
AID - 3738525 [pii]
AID - 10.1093/sysbio/syx045 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):964-987. doi: 10.1093/sysbio/syx045.

PMID- 28431105
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - Extinction vs. Rapid Radiation: The Juxtaposed Evolutionary Histories of
      Coelotine Spiders Support the Eocene-Oligocene Orogenesis of the Tibetan Plateau.
PG  - 988-1006
LID - 10.1093/sysbio/syx042 [doi]
AB  - Evolutionary biology has long been concerned with how changing environments
      affect and drive the spatiotemporal development of organisms. Coelotine spiders
      (Agelenidae: Coelotinae) are common species in the temperate and subtropical
      areas of the Northern Hemisphere. Their long evolutionary history and the
      extremely imbalanced distribution of species richness suggest that Eurasian
      environments, especially since the Cenozoic, are the drivers of their
      diversification. We use phylogenetics, molecular dating, ancestral area
      reconstructions, diversity, and ecological niche analyses to investigate the
      spatiotemporal evolution of 286 coelotine species from throughout the region.
      Based on eight genes (6.5 kb) and 2323 de novo DNA sequences, analyses suggest an
      Eocene South China origin for them. Most extant, widespread species belong to the
      southern (SCG) or northern (NCG) clades. The origin of coelotine spiders appears 
      to associate with either the Paleocene-Eocene Thermal Maximum or the hot period
      in early Eocene. Tibetan uplifting events influenced the current diversity
      patterns of coelotines. The origin of SCG lies outside of the Tibetan Plateau.
      Uplifting in the southeastern area of the plateau blocked dispersal since the
      Late Eocene. Continuous orogenesis appears to have created localized vicariant
      events, which drove rapid radiation in SCG. North-central Tibet is the likely
      location of origin for NCG and many lineages likely experienced extinction owing 
      to uplifting since early Oligocene. Their evolutionary histories correspond with 
      recent geological evidence that high-elevation orographical features existed in
      the Tibetan region as early as 40-35 Ma. Our discoveries may be the first
      empirical evidence that links the evolution of organisms to the Eocene-Oligocene 
      uplifting of the Tibetan Plateau. [Tibet; biogeography; ecology; molecular clock;
      diversification.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Zhao, Zhe
AU  - Zhao Z
AD  - Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology,
      Chinese Academy of Sciences, Beijing 100101, China.
AD  - Southeast Asia Biodiversity Research Institute, Chinese Academy of Sciences,
      Yezin, Nay Pyi Taw 05282, Myanmar.
FAU - Li, Shuqiang
AU  - Li S
AD  - Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology,
      Chinese Academy of Sciences, Beijing 100101, China.
AD  - University of Chinese Academy of Sciences, Beijing 100049, China.
LA  - eng
SI  - Dryad/10.5061/dryad.4pm73
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Extinction, Biological
MH  - *Genetic Speciation
MH  - *Phylogeny
MH  - Spiders/*classification/genetics
MH  - Tibet
EDAT- 2017/04/22 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/04/22 06:00
PHST- 2016/01/20 00:00 [received]
PHST- 2017/03/24 00:00 [accepted]
PHST- 2017/04/22 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/04/22 06:00 [entrez]
AID - 3098198 [pii]
AID - 10.1093/sysbio/syx042 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):988-1006. doi: 10.1093/sysbio/syx042.

PMID- 28419377
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - Gene-Tree Reconciliation with MUL-Trees to Resolve Polyploidy Events.
PG  - 1007-1018
LID - 10.1093/sysbio/syx044 [doi]
AB  - Polyploidy can have a huge impact on the evolution of species, and it is a common
      occurrence, especially in plants. The two types of polyploids-autopolyploids and 
      allopolyploids-differ in the level of divergence between the genes that are
      brought together in the new polyploid lineage. Because allopolyploids are formed 
      via hybridization, the homoeologous copies of genes within them are at least as
      divergent as orthologs in the parental species that came together to form them.
      This means that common methods for estimating the parental lineages of
      allopolyploidy events are not accurate, and can lead to incorrect inferences
      about the number of gene duplications and losses. Here, we have adapted an
      algorithm for topology-based gene-tree reconciliation to work with multi-labeled 
      trees (MUL-trees). By definition, MUL-trees have some tips with identical labels,
      which makes them a natural representation of the genomes of polyploids. Using
      this new reconciliation algorithm we can: accurately place allopolyploidy events 
      on a phylogeny, identify the parental lineages that hybridized to form
      allopolyploids, distinguish between allo-, auto-, and (in most cases) no
      polyploidy, and correctly count the number of duplications and losses in a set of
      gene trees. We validate our method using gene trees simulated with and without
      polyploidy, and revisit the history of polyploidy in data from the clades
      including both baker's yeast and bread wheat. Our re-analysis of the yeast data
      confirms the allopolyploid origin and parental lineages previously identified for
      this group. The method presented here should find wide use in the growing number 
      of genomes from species with a history of polyploidy. [Polyploidy;
      reconciliation; whole-genome duplication.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Gregg, W C Thomas
AU  - Gregg WCT
AD  - Department of Biology and School of Informatics and Computing, Indiana
      University, Bloomington, IN 47405, USA.
FAU - Ather, S Hussain
AU  - Ather SH
AD  - Department of Biology and School of Informatics and Computing, Indiana
      University, Bloomington, IN 47405, USA.
FAU - Hahn, Matthew W
AU  - Hahn MW
AD  - Department of Biology and School of Informatics and Computing, Indiana
      University, Bloomington, IN 47405, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.pp837
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Phylogeny
MH  - *Polyploidy
MH  - Saccharomyces cerevisiae/classification/genetics
MH  - Triticum/classification/genetics
EDAT- 2017/04/19 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/04/19 06:00
PHST- 2016/08/18 00:00 [received]
PHST- 2017/03/30 00:00 [accepted]
PHST- 2017/04/19 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/04/19 06:00 [entrez]
AID - 3610602 [pii]
AID - 10.1093/sysbio/syx044 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):1007-1018. doi: 10.1093/sysbio/syx044.

PMID- 28369655
OWN - NLM
STAT- MEDLINE
DCOM- 20180130
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 5
DP  - 2017 Sep 1
TI  - Why Do Phylogenomic Data Sets Yield Conflicting Trees? Data Type Influences the
      Avian Tree of Life more than Taxon Sampling.
PG  - 857-879
LID - 10.1093/sysbio/syx041 [doi]
AB  - Phylogenomics, the use of large-scale data matrices in phylogenetic analyses, has
      been viewed as the ultimate solution to the problem of resolving difficult nodes 
      in the tree of life. However, it has become clear that analyses of these large
      genomic data sets can also result in conflicting estimates of phylogeny. Here, we
      use the early divergences in Neoaves, the largest clade of extant birds, as a
      "model system" to understand the basis for incongruence among phylogenomic trees.
      We were motivated by the observation that trees from two recent avian
      phylogenomic studies exhibit conflicts. Those studies used different strategies: 
      1) collecting many characters [$\sim$ 42 mega base pairs (Mbp) of sequence data] 
      from 48 birds, sometimes including only one taxon for each major clade; and 2)
      collecting fewer characters ($\sim$ 0.4 Mbp) from 198 birds, selected to
      subdivide long branches. However, the studies also used different data types: the
      taxon-poor data matrix comprised 68% non-coding sequences whereas coding exons
      dominated the taxon-rich data matrix. This difference raises the question of
      whether the primary reason for incongruence is the number of sites, the number of
      taxa, or the data type. To test among these alternative hypotheses we assembled a
      novel, large-scale data matrix comprising 90% non-coding sequences from 235 bird 
      species. Although increased taxon sampling appeared to have a positive impact on 
      phylogenetic analyses the most important variable was data type. Indeed, by
      analyzing different subsets of the taxa in our data matrix we found that
      increased taxon sampling actually resulted in increased congruence with the tree 
      from the previous taxon-poor study (which had a majority of non-coding data)
      instead of the taxon-rich study (which largely used coding data). We suggest that
      the observed differences in the estimates of topology for these studies reflect
      data-type effects due to violations of the models used in phylogenetic analyses, 
      some of which may be difficult to detect. If incongruence among trees estimated
      using phylogenomic methods largely reflects problems with model fit developing
      more "biologically-realistic" models is likely to be critical for efforts to
      reconstruct the tree of life. [Birds; coding exons; GTR model; model fit;
      Neoaves; non-coding DNA; phylogenomics; taxon sampling.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Reddy, Sushma
AU  - Reddy S
AD  - Biology Department, Loyola University Chicago, 1032 West Sheridan Road, Chicago, 
      IL 60660, USA.
FAU - Kimball, Rebecca T
AU  - Kimball RT
AD  - Department of Biology, University of Florida, Gainesville, FL 32607, USA.
FAU - Pandey, Akanksha
AU  - Pandey A
AD  - Department of Biology, University of Florida, Gainesville, FL 32607, USA.
FAU - Hosner, Peter A
AU  - Hosner PA
AD  - Department of Biology, University of Florida, Gainesville, FL 32607, USA.
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL 32607, 
      USA.
FAU - Braun, Michael J
AU  - Braun MJ
AD  - Behavior, Ecology, Evolution, and Systematics Program, University of Maryland,
      College Park, MD 20742, USA.
AD  - Department of Vertebrate Zoology, National Museum of Natural History, Smithsonian
      Institution-MRC 163, PO Box 37012, Washington, DC 20013-7012, USA.
FAU - Hackett, Shannon J
AU  - Hackett SJ
AD  - Zoology Department, Field Museum of Natural History, 1400 South Lake Shore Drive,
      Chicago, IL 60605, USA.
FAU - Han, Kin-Lan
AU  - Han KL
AD  - Department of Biology, University of Florida, Gainesville, FL 32607, USA.
FAU - Harshman, John
AU  - Harshman J
AD  - 4869 Pepperwood Way, San Jose, CA 95124, USA.
FAU - Huddleston, Christopher J
AU  - Huddleston CJ
AD  - Collections Program, National Museum of Natural History, Smithsonian Institution,
      4210 Silver Hill Road, Suitland, MD 20746, USA.
FAU - Kingston, Sarah
AU  - Kingston S
AD  - Behavior, Ecology, Evolution, and Systematics Program, University of Maryland,
      College Park, MD 20742, USA.
AD  - Department of Vertebrate Zoology, National Museum of Natural History, Smithsonian
      Institution-MRC 163, PO Box 37012, Washington, DC 20013-7012, USA.
AD  - Bowdoin College, Department of Biology and Coastal Studies Center, 6500 College
      Station, Brunwick, ME 04011, USA.
FAU - Marks, Ben D
AU  - Marks BD
AD  - Zoology Department, Field Museum of Natural History, 1400 South Lake Shore Drive,
      Chicago, IL 60605, USA.
FAU - Miglia, Kathleen J
AU  - Miglia KJ
AD  - Department of Biological Sciences, Wayne State University, 5047 Gullen Mall,
      Detroit, MI 48202, USA.
FAU - Moore, William S
AU  - Moore WS
AD  - Department of Biological Sciences, Wayne State University, 5047 Gullen Mall,
      Detroit, MI 48202, USA.
FAU - Sheldon, Frederick H
AU  - Sheldon FH
AD  - Museum of Natural Science and Department of Biological Sciences, Louisiana State 
      University, 119 Foster Hall, Baton Rouge, LA 70803, USA.
FAU - Witt, Christopher C
AU  - Witt CC
AD  - Department of Biology and Museum of Southwestern Biology, University 15 of New
      Mexico, Albuquerque, New Mexico 87131, USA.
FAU - Yuri, Tamaki
AU  - Yuri T
AD  - Department of Biology, University of Florida, Gainesville, FL 32607, USA.
AD  - Sam Noble Museum, University of Oklahoma, 2401 Chautauqua Avenue, Norman, OK
      73072, USA.
FAU - Braun, Edward L
AU  - Braun EL
AD  - Department of Biology, University of Florida, Gainesville, FL 32607, USA.
AD  - Genetics Institute, University of Florida, Gainesville, FL 32607, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.6536v
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Birds/*classification
MH  - Classification/*methods
MH  - *Datasets as Topic
MH  - Genome/genetics
MH  - Genomics
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2017/04/04 06:00
MHDA- 2018/01/31 06:00
CRDT- 2017/04/04 06:00
PHST- 2016/10/15 00:00 [received]
PHST- 2017/03/22 00:00 [accepted]
PHST- 2017/04/04 06:00 [pubmed]
PHST- 2018/01/31 06:00 [medline]
PHST- 2017/04/04 06:00 [entrez]
AID - 3091102 [pii]
AID - 10.1093/sysbio/syx041 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Sep 1;66(5):857-879. doi: 10.1093/sysbio/syx041.

PMID- 28365762
OWN - NLM
STAT- MEDLINE
DCOM- 20180301
LR  - 20180301
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 2
DP  - 2017 Mar 1
TI  - Society for the Study of Systematic Biology symposium: Frontiers in Parametric
      Biogeography.
PG  - 125-127
LID - 10.1093/sysbio/syx036 [doi]
FAU - Albert, James S
AU  - Albert JS
AD  - University of Louisiana at Lafayette, Lafayette, LA 70504, USA.
FAU - Antonelli, Alexandre
AU  - Antonelli A
AD  - Gothenburg Global Biodiversity Centre, Box 461, SE-405 30 Goteborg, Sweden.
LA  - eng
PT  - Introductory Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Brazil
MH  - *Classification
MH  - Congresses as Topic
MH  - *Phylogeography
EDAT- 2017/04/04 06:00
MHDA- 2018/03/02 06:00
CRDT- 2017/04/03 06:00
PHST- 2017/04/03 06:00 [entrez]
PHST- 2017/04/04 06:00 [pubmed]
PHST- 2018/03/02 06:00 [medline]
AID - 3074781 [pii]
AID - 10.1093/sysbio/syx036 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Mar 1;66(2):125-127. doi: 10.1093/sysbio/syx036.

PMID- 28334310
OWN - NLM
STAT- MEDLINE
DCOM- 20180130
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 5
DP  - 2017 Sep 1
TI  - Different from Trees, more than Metaphors: Branching Silhouettes-Corals, Cacti,
      and the Oaks.
PG  - 737-753
LID - 10.1093/sysbio/syx039 [doi]
AB  - There has long been ambiguity in the use of the term tree in phylogenetic
      systematics, which is a continuous source of misinterpretation of evolutionary
      relationships. The basic problem is that while many trees with phylogenetic or
      evolutionary relevance, such as cladograms, are consistent with graph theory,
      tree-like visualization of phylogeny may also be done via other types of
      graphics, especially botanical (or literal) tree drawings. As a consequence, the 
      meaning of such diagrams is not always clear: a given picture may have multiple
      interpretations in its different parts and two figures that look similar may
      actually carry quite different information. I show that these are not merely
      metaphors of phylogeny and may be defined mathematically as geometric shapes
      called branching silhouette diagrams, the name referring to their most apparent
      features. They have four basic forms depending on whether the time factor is
      considered in their construction and the relationships portrayed are of the
      ancestor-descendant or sister group type. By revitalizing Darwin's early
      suggestions on the illustration of change in the living world, I propose the term
      coral for the most common forms of branching silhouette which consider time and
      show ancestry. The others appear rarely in present-day practice and are mostly of
      historical value-these are cacti (named after one of Bessey's diagrams) and two
      types of oaks (with reference to several of Haeckel's motifs used in depicting
      phylogeny in the nineteenth century). The use of these terms in phylogenetic
      systematics facilitates better understanding of how organization of biodiversity 
      has been visualized in science [Bessey; cladograms; coral-thinking; Darwin; graph
      theory; Haeckel; phylogeny; tree thinking.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Podani, Janos
AU  - Podani J
AD  - Department of Plant Systematics, Ecology and Theoretical Biology, Institute of
      Biology, L. Eotvos University, and MTA-ELTE-MTM Ecology Research Group, Pazmany
      P. s. 1/c, H-1117 Budapest, Hungary.
LA  - eng
SI  - Dryad/10.5061/dryad.st550
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Classification
MH  - Metaphor
MH  - *Phylogeny
MH  - *Terminology as Topic
EDAT- 2017/03/24 06:00
MHDA- 2018/01/31 06:00
CRDT- 2017/03/24 06:00
PHST- 2016/05/26 00:00 [received]
PHST- 2017/02/26 00:00 [accepted]
PHST- 2017/03/24 06:00 [pubmed]
PHST- 2018/01/31 06:00 [medline]
PHST- 2017/03/24 06:00 [entrez]
AID - 3066298 [pii]
AID - 10.1093/sysbio/syx039 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Sep 1;66(5):737-753. doi: 10.1093/sysbio/syx039.

PMID- 28334227
OWN - NLM
STAT- MEDLINE
DCOM- 20180130
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 5
DP  - 2017 Sep 1
TI  - A Phylogenetic, Biogeographic, and Taxonomic study of all Extant Species of
      Anolis (Squamata; Iguanidae).
PG  - 663-697
LID - 10.1093/sysbio/syx029 [doi]
AB  - Anolis lizards (anoles) are textbook study organisms in evolution and ecology.
      Although several topics in evolutionary biology have been elucidated by the study
      of anoles, progress in some areas has been hampered by limited phylogenetic
      information on this group. Here, we present a phylogenetic analysis of all 379
      extant species of Anolis, with new phylogenetic data for 139 species including
      new DNA data for 101 species. We use the resulting estimates as a basis for
      defining anole clade names under the principles of phylogenetic nomenclature and 
      to examine the biogeographic history of anoles. Our new taxonomic treatment
      achieves the supposed advantages of recent subdivisions of anoles that employed
      ranked Linnaean-based nomenclature while avoiding the pitfalls of those
      approaches regarding artificial constraints imposed by ranks. Our biogeographic
      analyses demonstrate complexity in the dispersal history of anoles, including
      multiple crossings of the Isthmus of Panama, two invasions of the Caribbean,
      single invasions to Jamaica and Cuba, and a single evolutionary dispersal from
      the Caribbean to the mainland that resulted in substantial anole diversity. Our
      comprehensive phylogenetic estimate of anoles should prove useful for rigorous
      testing of many comparative evolutionary hypotheses. [Anoles; biogeography;
      lizards; Neotropics; phylogeny; taxonomy].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Poe, Steven
AU  - Poe S
AD  - Department of Biology and Museum of Southwestern Biology, University of New
      Mexico, Albuquerque, NM 87131, USA.
FAU - Nieto-Montes de Oca, Adrian
AU  - Nieto-Montes de Oca A
AD  - Departamento de Biologia Evolutiva, Facultad de Ciencias, Universidad Nacional
      Autonoma de Mexico, Mexico, D.F. 04510, Mexico.
FAU - Torres-Carvajal, Omar
AU  - Torres-Carvajal O
AD  - Museo de Zoologia, Escuela de Biologia, Pontificia Universidad Catolica del
      Ecuador, Avenida 12 de Octubre y Roca, Apartado 17-01-2184, Quito, Ecuador.
FAU - De Queiroz, Kevin
AU  - De Queiroz K
AD  - Smithsonian Institution, National Museum of Natural History, Washington DC
      20560-0162, USA.
FAU - Velasco, Julian A
AU  - Velasco JA
AD  - Laboratorio de Analisis Espaciales, Instituto de Biologia, Universidad Nacional
      Autonoma de Mexico, Mexico, D.F., Mexico &amp; Grupo de Investigacion en Ecologia
      Animal, Departamento de Biologia, Facultad de Ciencias Naturales y Exactas,
      Universidad del Valle, Cali-Colombia.
FAU - Truett, Brad
AU  - Truett B
AD  - Department of Biology and Museum of Southwestern Biology, University of New
      Mexico, Albuquerque, NM 87131, USA.
FAU - Gray, Levi N
AU  - Gray LN
AD  - Department of Biology and Museum of Southwestern Biology, University of New
      Mexico, Albuquerque, NM 87131, USA.
FAU - Ryan, Mason J
AU  - Ryan MJ
AD  - Department of Biology and Museum of Southwestern Biology, University of New
      Mexico, Albuquerque, NM 87131, USA.
FAU - Kohler, Gunther
AU  - Kohler G
AD  - Forschungsinstitut und Naturmuseum Senckenberg, Senckenberganlage 25, 60325
      Frankfurt a.M., Germany.
FAU - Ayala-Varela, Fernando
AU  - Ayala-Varela F
AD  - Museo de Zoologia, Escuela de Biologia, Pontificia Universidad Catolica del
      Ecuador, Avenida 12 de Octubre y Roca, Apartado 17-01-2184, Quito, Ecuador.
FAU - Latella, Ian
AU  - Latella I
AD  - Department of Biology and Museum of Southwestern Biology, University of New
      Mexico, Albuquerque, NM 87131, USA.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Americas
MH  - Animal Distribution
MH  - Animals
MH  - Biodiversity
MH  - Caribbean Region
MH  - *Classification
MH  - Lizards/*classification/genetics
MH  - *Phylogeny
MH  - Phylogeography
EDAT- 2017/03/24 06:00
MHDA- 2018/01/31 06:00
CRDT- 2017/03/24 06:00
PHST- 2015/12/21 00:00 [received]
PHST- 2017/01/21 00:00 [accepted]
PHST- 2017/03/24 06:00 [pubmed]
PHST- 2018/01/31 06:00 [medline]
PHST- 2017/03/24 06:00 [entrez]
AID - 3056323 [pii]
AID - 10.1093/sysbio/syx029 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Sep 1;66(5):663-697. doi: 10.1093/sysbio/syx029.

PMID- 28334223
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Is BAMM Flawed? Theoretical and Practical Concerns in the Analysis of Multi-Rate 
      Diversification Models.
PG  - 477-498
LID - 10.1093/sysbio/syx037 [doi]
AB  - Bayesian analysis of macroevolutionary mixtures (BAMM) is a statistical framework
      that uses reversible jump Markov chain Monte Carlo to infer complex
      macroevolutionary dynamics of diversification and phenotypic evolution on
      phylogenetic trees. A recent article by Moore et al. (MEA) reported a number of
      theoretical and practical concerns with BAMM. Major claims from MEA are that (i) 
      BAMM's likelihood function is incorrect, because it does not account for
      unobserved rate shifts; (ii) the posterior distribution on the number of rate
      shifts is overly sensitive to the prior; and (iii) diversification rate estimates
      from BAMM are unreliable. Here, we show that these and other conclusions from MEA
      are generally incorrect or unjustified. We first demonstrate that MEA's numerical
      assessment of the BAMM likelihood is compromised by their use of an invalid
      likelihood function. We then show that "unobserved rate shifts" appear to be
      irrelevant for biologically plausible parameterizations of the diversification
      process. We find that the purportedly extreme prior sensitivity reported by MEA
      cannot be replicated with standard usage of BAMM v2.5, or with any other version 
      when conventional Bayesian model selection is performed. Finally, we demonstrate 
      that BAMM performs very well at estimating diversification rate variation across 
      the ${\sim}$20% of simulated trees in MEA's data set for which it is
      theoretically possible to infer rate shifts with confidence. Due to ascertainment
      bias, the remaining 80% of their purportedly variable-rate phylogenies are
      statistically indistinguishable from those produced by a constant-rate
      birth-death process and were thus poorly suited for the summary statistics used
      in their performance assessment. We demonstrate that inferences about
      diversification rates have been accurate and consistent across all major previous
      releases of the BAMM software. We recognize an acute need to address the
      theoretical foundations of rate-shift models for phylogenetic trees, and we
      expect BAMM and other modeling frameworks to improve in response to mathematical 
      and computational innovations. However, we remain optimistic that that the
      imperfect tools currently available to comparative biologists have provided and
      will continue to provide important insights into the diversification of life on
      Earth.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Rabosky, Daniel L
AU  - Rabosky DL
AD  - Department of Ecology and Evolutionary Biology and Museum of Zoology, University 
      of Michigan, Ann Arbor, MI 48109, USA.
FAU - Mitchell, Jonathan S
AU  - Mitchell JS
AD  - Department of Ecology and Evolutionary Biology and Museum of Zoology, University 
      of Michigan, Ann Arbor, MI 48109, USA.
FAU - Chang, Jonathan
AU  - Chang J
AD  - Department of Ecology and Evolutionary Biology, University of California, Los
      Angeles, CA 90095, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.36g21
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Biodiversity
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - Likelihood Functions
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Software
PMC - PMC5790138
OTO - NOTNLM
OT  - *BAMM
OT  - *diversification
OT  - *macroevolution
OT  - *phylogeny
OT  - *speciation
EDAT- 2017/03/24 06:00
MHDA- 2018/01/30 06:00
CRDT- 2017/03/24 06:00
PHST- 2016/09/23 00:00 [received]
PHST- 2017/02/14 00:00 [accepted]
PHST- 2017/03/24 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2017/03/24 06:00 [entrez]
AID - 3039233 [pii]
AID - 10.1093/sysbio/syx037 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):477-498. doi: 10.1093/sysbio/syx037.

PMID- 28334176
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - Phylogenomic Systematics of Ostariophysan Fishes: Ultraconserved Elements Support
      the Surprising Non-Monophyly of Characiformes.
PG  - 881-895
LID - 10.1093/sysbio/syx038 [doi]
AB  - Ostariophysi is a superorder of bony fishes including more than 10,300 species in
      1100 genera and 70 families. This superorder is traditionally divided into five
      major groups (orders): Gonorynchiformes (milkfishes and sandfishes),
      Cypriniformes (carps and minnows), Characiformes (tetras and their allies),
      Siluriformes (catfishes), and Gymnotiformes (electric knifefishes). Unambiguous
      resolution of the relationships among these lineages remains elusive, with
      previous molecular and morphological analyses failing to produce a consensus
      phylogeny. In this study, we use over 350 ultraconserved element (UCEs) loci
      comprising 5 million base pairs collected across 35 representative ostariophysan 
      species to compile one of the most data-rich phylogenies of fishes to date. We
      use these data to infer higher level (interordinal) relationships among
      ostariophysan fishes, focusing on the monophyly of the Characiformes-one of the
      most contentiously debated groups in fish systematics. As with most previous
      molecular studies, we recover a non-monophyletic Characiformes with the two
      monophyletic suborders, Citharinoidei and Characoidei, more closely related to
      other ostariophysan clades than to each other. We also explore incongruence
      between results from different UCE data sets, issues of orthology, and the use of
      morphological characters in combination with our molecular data. [Conserved
      sequence; ichthyology; massively parallel sequencing; morphology; next-generation
      sequencing; UCEs.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Chakrabarty, Prosanta
AU  - Chakrabarty P
AD  - Museum of Natural Science and Department of Biological Sciences, Louisiana State 
      University, 119 Foster Hall, Baton Rouge, LA 70803, USA.
FAU - Faircloth, Brant C
AU  - Faircloth BC
AD  - Museum of Natural Science and Department of Biological Sciences, Louisiana State 
      University, 119 Foster Hall, Baton Rouge, LA 70803, USA.
FAU - Alda, Fernando
AU  - Alda F
AD  - Museum of Natural Science and Department of Biological Sciences, Louisiana State 
      University, 119 Foster Hall, Baton Rouge, LA 70803, USA.
FAU - Ludt, William B
AU  - Ludt WB
AD  - Museum of Natural Science and Department of Biological Sciences, Louisiana State 
      University, 119 Foster Hall, Baton Rouge, LA 70803, USA.
FAU - Mcmahan, Caleb D
AU  - Mcmahan CD
AD  - Museum of Natural Science and Department of Biological Sciences, Louisiana State 
      University, 119 Foster Hall, Baton Rouge, LA 70803, USA.
AD  - The Field Museum of Natural History, 1400 S Lake Shore Dr, Chicago, IL 60605,
      USA.
FAU - Near, Thomas J
AU  - Near TJ
AD  - Department of Ecology and Evolutionary Biology, and Peabody Museum of Natural
      History, Yale University, New Haven, CT 06520, USA.
FAU - Dornburg, Alex
AU  - Dornburg A
AD  - North Carolina Museum of Natural Sciences, Raleigh, NC 27601, USA.
FAU - Albert, James S
AU  - Albert JS
AD  - Department of Biology, University of Louisiana, Lafayette, LA 70504, USA.
FAU - Arroyave, Jairo
AU  - Arroyave J
AD  - Instituto de Biologia, Universidad Nacional Autonoma de Mexico, Ciudad de Mexico,
      Mexico.
FAU - Stiassny, Melanie L J
AU  - Stiassny MLJ
AD  - Department of Ichthyology, American Museum of Natural History, Central Park West 
      at 79th Street, New York, NY 10024, USA.
FAU - Sorenson, Laurie
AU  - Sorenson L
AD  - Museum of Natural Science and Department of Biological Sciences, Louisiana State 
      University, 119 Foster Hall, Baton Rouge, LA 70803, USA.
AD  - Department of Ecology and Evolutionary Biology, University of California Los
      Angeles, 610 Yound Drive South, Los Angeles, CA 90095, USA.
FAU - Alfaro, Michael E
AU  - Alfaro ME
AD  - Department of Ecology and Evolutionary Biology, University of California Los
      Angeles, 610 Yound Drive South, Los Angeles, CA 90095, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.n9g60
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Characiformes/*classification/*genetics
MH  - Conserved Sequence/*genetics
MH  - *Phylogeny
EDAT- 2017/03/24 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/03/24 06:00
PHST- 2015/08/17 00:00 [received]
PHST- 2016/02/24 00:00 [accepted]
PHST- 2017/03/24 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/03/24 06:00 [entrez]
AID - 3056324 [pii]
AID - 10.1093/sysbio/syx038 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):881-895. doi: 10.1093/sysbio/syx038.

PMID- 28204803
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20180226
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Erratum.
PG  - 658-659
LID - 10.1093/sysbio/syw110 [doi]
LA  - eng
PT  - Journal Article
PT  - Published Erratum
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EFR - Syst Biol. 2014 Jul;63(4):582-600. PMID: 24817532
EDAT- 2017/02/17 06:00
MHDA- 2017/02/17 06:01
CRDT- 2017/02/17 06:00
PHST- 2017/02/17 06:00 [pubmed]
PHST- 2017/02/17 06:01 [medline]
PHST- 2017/02/17 06:00 [entrez]
AID - 2993001 [pii]
AID - 10.1093/sysbio/syw110 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):658-659. doi: 10.1093/sysbio/syw110.

PMID- 28204787
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20190118
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - Inference of Evolutionary Jumps in Large Phylogenies using Levy Processes.
PG  - 950-963
LID - 10.1093/sysbio/syx028 [doi]
AB  - Although it is now widely accepted that the rate of phenotypic evolution may not 
      necessarily be constant across large phylogenies, the frequency and phylogenetic 
      position of periods of rapid evolution remain unclear. In his highly influential 
      view of evolution, G. G. Simpson supposed that such evolutionary jumps occur when
      organisms transition into so-called new adaptive zones, for instance after
      dispersal into a new geographic area, after rapid climatic changes, or following 
      the appearance of an evolutionary novelty. Only recently, large, accurate and
      well calibrated phylogenies have become available that allow testing this
      hypothesis directly, yet inferring evolutionary jumps remains computationally
      very challenging. Here, we develop a computationally highly efficient algorithm
      to accurately infer the rate and strength of evolutionary jumps as well as their 
      phylogenetic location. Following previous work we model evolutionary jumps as a
      compound process, but introduce a novel approach to sample jump configurations
      that does not require matrix inversions and thus naturally scales to large trees.
      We then make use of this development to infer evolutionary jumps in Anolis
      lizards and Loriinii parrots where we find strong signal for such jumps at the
      basis of clades that transitioned into new adaptive zones, just as postulated by 
      Simpson's hypothesis. [evolutionary jump; Levy process; phenotypic evolution;
      punctuated equilibrium; quantitative traits.
CI  - The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Duchen, Pablo
AU  - Duchen P
AD  - Faculty of Mathematics and Natural Sciences, Department of Biology, University of
      Fribourg, Chemin du Musee 10, 1700 Fribourg, Switzerland.
FAU - Leuenberger, Christoph
AU  - Leuenberger C
AD  - Faculty of Mathematics and Natural Sciences, Department of Biology, University of
      Fribourg, Chemin du Musee 10, 1700 Fribourg, Switzerland.
AD  - Faculty of Mathematics and Natural Sciences, Department of Mathematics,
      University of Fribourg, Chemin du Musee 23, 1700 Fribourg, Switzerland.
FAU - Szilagyi, Sandor M
AU  - Szilagyi SM
AD  - Faculty of Mathematics and Natural Sciences, Department of Informatics,
      University of Fribourg, Boulevard de Peroles 90, 1700 Fribourg, Switzerland.
AD  - Faculty of Sciences and Letters, Department of Informatics, Petru Maior
      University, Str. N. Iorga Nr. 1, 540088 Tirgu Mures, Romania.
AD  - Faculty of Electrical Engineering and Informatics, Department of Control
      Engineering and Information Technology, Budapest University of Technology and
      Economics, Magyar tudosok krt. 2, H-1117 Budapest, Hungary.
FAU - Harmon, Luke
AU  - Harmon L
AD  - Department of Biological Sciences, University of Idaho, 875 Perimeter Drive MS
      3051, 1700 Moscow-Idaho, United States.
FAU - Eastman, Jonathan
AU  - Eastman J
AD  - Department of Biological Sciences, University of Idaho, 875 Perimeter Drive MS
      3051, 1700 Moscow-Idaho, United States.
FAU - Schweizer, Manuel
AU  - Schweizer M
AD  - Naturhistorisches Museum der Burgergemeinde Bern, Bernastrasse 15, 3005 Bern,
      Switzerland.
FAU - Wegmann, Daniel
AU  - Wegmann D
AD  - Faculty of Mathematics and Natural Sciences, Department of Biology, University of
      Fribourg, Chemin du Musee 10, 1700 Fribourg, Switzerland.
LA  - eng
SI  - Dryad/10.5061/dryad.170rb
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Lizards/classification
MH  - *Models, Genetic
MH  - Parrots/classification
MH  - *Phylogeny
PMC - PMC5790141
EDAT- 2017/02/17 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/02/17 06:00
PHST- 2016/07/07 00:00 [received]
PHST- 2016/12/28 00:00 [accepted]
PHST- 2017/02/17 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/02/17 06:00 [entrez]
AID - 2999289 [pii]
AID - 10.1093/sysbio/syx028 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):950-963. doi: 10.1093/sysbio/syx028.

PMID- 28204784
OWN - NLM
STAT- MEDLINE
DCOM- 20180130
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 5
DP  - 2017 Sep 1
TI  - A Poissonian Model of Indel Rate Variation for Phylogenetic Tree Inference.
PG  - 698-714
LID - 10.1093/sysbio/syx033 [doi]
AB  - While indel rate variation has been observed and analyzed in detail, it is not
      taken into account by current indel-aware phylogenetic reconstruction methods. In
      this work, we introduce a continuous time stochastic process, the geometric
      Poisson indel process, that generalizes the Poisson indel process by allowing
      insertion and deletion rates to vary across sites. We design an efficient
      algorithm for computing the probability of a given multiple sequence alignment
      based on our new indel model. We describe a method to construct phylogeny
      estimates from a fixed alignment using neighbor joining. Using simulation
      studies, we show that ignoring indel rate variation may have a detrimental effect
      on the accuracy of the inferred phylogenies, and that our proposed method can
      sidestep this issue by inferring latent indel rate categories. We also show that 
      our phylogenetic inference method may be more stable to taxa subsampling than
      methods that either ignore indels or indel rate variation. [evolutionary
      stochastic process; indel rate variation; Poisson indel process; TKF91.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Zhai, Yongliang
AU  - Zhai Y
AD  - Department of Statistics, University of British Columbia, Vancouver, British
      Columbia, V6T 1Z4, Canada.
FAU - Alexandre, Bouchard-Cote
AU  - Alexandre BC
AD  - Department of Statistics, University of British Columbia, Vancouver, British
      Columbia, V6T 1Z4, Canada.
LA  - eng
SI  - Dryad/10.5061/dryad.95h17
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Evolution, Molecular
MH  - *INDEL Mutation
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Sequence Alignment
EDAT- 2017/02/17 06:00
MHDA- 2018/01/31 06:00
CRDT- 2017/02/17 06:00
PHST- 2015/03/04 00:00 [received]
PHST- 2017/01/27 00:00 [accepted]
PHST- 2017/02/17 06:00 [pubmed]
PHST- 2018/01/31 06:00 [medline]
PHST- 2017/02/17 06:00 [entrez]
AID - 2999315 [pii]
AID - 10.1093/sysbio/syx033 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Sep 1;66(5):698-714. doi: 10.1093/sysbio/syx033.

PMID- 28204782
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - PHRAPL: Phylogeographic Inference Using Approximate Likelihoods.
PG  - 1045-1053
LID - 10.1093/sysbio/syx001 [doi]
AB  - The demographic history of most species is complex, with multiple evolutionary
      processes combining to shape the observed patterns of genetic diversity. To infer
      this history, the discipline of phylogeography has (to date) used models that
      simplify the historical demography of the focal organism, for example by assuming
      or ignoring ongoing gene flow between populations or by requiring a priori
      specification of divergence history. Since no single model incorporates every
      possible evolutionary process, researchers rely on intuition to choose the models
      that they use to analyze their data. Here, we describe an approximate likelihood 
      approach that reduces this reliance on intuition. PHRAPL allows users to
      calculate the probability of a large number of complex demographic histories
      given a set of gene trees, enabling them to identify the most likely underlying
      model and estimate parameters for a given system. Available model parameters
      include coalescence time among populations or species, gene flow, and population 
      size. We describe the method and test its performance in model selection and
      parameter estimation using simulated data. We also compare model probabilities
      estimated using our approximate likelihood method to those obtained using
      standard analytical likelihood. The method performs well under a wide range of
      scenarios, although this is sometimes contingent on sampling many loci. In most
      scenarios, as long as there are enough loci and if divergence among populations
      is sufficiently deep, PHRAPL can return the true model in nearly all simulated
      replicates. Parameter estimates from the method are also generally accurate in
      most cases. PHRAPL is a valuable new method for phylogeographic model selection
      and will be particularly useful as a tool to more extensively explore demographic
      model space than is typically done or to estimate parameters for complex models
      that are not readily implemented using current methods. Estimating relevant
      parameters using the most appropriate demographic model can help to sharpen our
      understanding of the evolutionary processes giving rise to phylogeographic
      patterns. [AIC; grid search; isolation-with-migration; migration rate;
      multispecies coalescent; parameter optimization; population genetics; tree
      topologies.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Jackson, Nathon D
AU  - Jackson ND
AD  - Department of Ecology and Evolutionary Biology, University of Tennessee, 442
      Hesler Biology Building, Knoxville, TN 37996, USA.
FAU - Morales, Ariadna E
AU  - Morales AE
AD  - Department of Ecology and Evolutionary Biology, University of Tennessee, 442
      Hesler Biology Building, Knoxville, TN 37996, USA.
FAU - Carstens, Bryan C
AU  - Carstens BC
AD  - Department of Evolution, Ecology and Organismal Biology, Ohio State University,
      318 W. 12th Avenue, Columbus, OH 43210, USA.
FAU - O'Meara, Brian C
AU  - O'Meara BC
AD  - Department of Ecology and Evolutionary Biology, University of Tennessee, 442
      Hesler Biology Building, Knoxville, TN 37996, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.1414v
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - Computer Simulation
MH  - Genetic Variation
MH  - Likelihood Functions
MH  - *Models, Biological
MH  - Phylogeny
MH  - Phylogeography/*methods
EDAT- 2017/02/17 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/02/17 06:00
PHST- 2016/05/18 00:00 [received]
PHST- 2017/01/04 00:00 [accepted]
PHST- 2017/02/17 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/02/17 06:00 [entrez]
AID - 2999288 [pii]
AID - 10.1093/sysbio/syx001 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):1045-1053. doi: 10.1093/sysbio/syx001.

PMID- 28204758
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20180226
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Erratum.
PG  - 661
LID - 10.1093/sysbio/syw118 [doi]
LA  - eng
PT  - Journal Article
PT  - Published Erratum
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EFR - Syst Biol. 2016 May;65(3):432-48. PMID: 26865275
EDAT- 2017/02/17 06:00
MHDA- 2017/02/17 06:01
CRDT- 2017/02/17 06:00
PHST- 2017/02/17 06:00 [pubmed]
PHST- 2017/02/17 06:01 [medline]
PHST- 2017/02/17 06:00 [entrez]
AID - 2993003 [pii]
AID - 10.1093/sysbio/syw118 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):661. doi: 10.1093/sysbio/syw118.

PMID- 28180902
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20180310
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Erratum.
PG  - 657
LID - 10.1093/sysbio/syw091 [doi]
LA  - eng
PT  - Journal Article
PT  - Published Erratum
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EFR - Syst Biol. 2015 May;64(3):432-40. PMID: 25575504
PMC - PMC5837464
EDAT- 2017/02/10 06:00
MHDA- 2017/02/10 06:01
CRDT- 2017/02/10 06:00
PHST- 2017/02/10 06:00 [pubmed]
PHST- 2017/02/10 06:01 [medline]
PHST- 2017/02/10 06:00 [entrez]
AID - 2979216 [pii]
AID - 10.1093/sysbio/syw091 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):657. doi: 10.1093/sysbio/syw091.

PMID- 28177510
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - An Integrated Model of Phenotypic Trait Changes and Site-Specific Sequence
      Evolution.
PG  - 917-933
LID - 10.1093/sysbio/syx032 [doi]
AB  - Recent years have seen a constant rise in the availability of trait data,
      including morphological features, ecological preferences, and life history
      characteristics. These phenotypic data provide means to associate genomic regions
      with phenotypic attributes, thus allowing the identification of phenotypic traits
      associated with the rate of genome and sequence evolution. However, inference
      methodologies that analyze sequence and phenotypic data in a unified statistical 
      framework are still scarce. Here, we present TraitRateProp, a probabilistic
      method that allows testing whether the rate of sequence evolution is associated
      with a binary phenotypic character trait. The method further allows the detection
      of specific sequence sites whose evolutionary rate is most noticeably affected
      following the character transition, suggesting a shift in functional/structural
      constraints. TraitRateProp is first evaluated in simulations and then applied to 
      study the evolutionary process of plastid plant genomes upon a transition to a
      heterotrophic lifestyle. To this end, we analyze 20 plastid genes across 85
      orchid species, spanning different lifestyles and representing different genera
      in this large family of flowering plants. Our results indicate higher
      evolutionary rates following repeated transitions to a heterotrophic lifestyle in
      all but four of the loci analyzed. [Evolutionary models; evolutionary rate;
      genotype-phenotype; orchids; plastome; rate shift.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Levy Karin, Eli
AU  - Levy Karin E
AD  - Department of Cell Research and Immunology, George S. Wise Faculty of Life
      Sciences, Tel Aviv University, Tel Aviv 69978, Israel.
AD  - Department of Molecular Biology &amp; Ecology of Plants, George S. Wise Faculty of
      Life Sciences, Tel Aviv University, Tel Aviv 69978, Israel.
FAU - Wicke, Susann
AU  - Wicke S
AD  - Institute for Evolution and Biodiversity, University of Muenster, Muenster,
      Germany.
FAU - Pupko, Tal
AU  - Pupko T
AD  - Department of Cell Research and Immunology, George S. Wise Faculty of Life
      Sciences, Tel Aviv University, Tel Aviv 69978, Israel.
FAU - Mayrose, Itay
AU  - Mayrose I
AD  - Department of Molecular Biology &amp; Ecology of Plants, George S. Wise Faculty of
      Life Sciences, Tel Aviv University, Tel Aviv 69978, Israel.
LA  - eng
SI  - Dryad/10.5061/dryad.d4j55.2
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Genome, Plant/*genetics
MH  - Genome, Plastid/genetics
MH  - *Models, Genetic
MH  - Phenotype
EDAT- 2017/02/09 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/02/09 06:00
PHST- 2016/06/10 00:00 [received]
PHST- 2017/01/24 00:00 [accepted]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/02/09 06:00 [entrez]
AID - 2978030 [pii]
AID - 10.1093/sysbio/syx032 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):917-933. doi: 10.1093/sysbio/syx032.

PMID- 28177088
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - Using Genomic Location and Coalescent Simulation to Investigate Gene Tree
      Discordance in Medicago L.
PG  - 934-949
LID - 10.1093/sysbio/syx035 [doi]
AB  - Several well-documented evolutionary processes are known to cause conflict
      between species-level phylogenies and gene-level phylogenies. Three of the most
      challenging processes for species tree inference are incomplete lineage sorting, 
      hybridization and gene duplication, which may result in unwarranted comparisons
      of paralogous genes. Several existing methods have dealt with these processes but
      none has yet been able to untangle all three at once. Here, we propose a stepwise
      method by which these processes can be discerned using information on genomic
      location coupled with coalescent simulations. In the first step, highly
      discordant genes within genomic blocks (putative paralogs) are identified and
      excluded from the data set and, in the second step, blocks of linked genes are
      grouped according to their hybrid history. Existing multispecies coalescent
      software can then be applied to recover the principal tree(s) that make up the
      species tree/network without violating the underlying model. The potential of the
      approach is evaluated on simulated data derived from a species network composed
      of nine species, of which one is of hybrid origin, and displaying a single-gene
      duplication that leads to paralogous comparisons. We apply our method to an
      empirical set of 12 genes from 7 species sampled in the plant genus Medicago that
      display phylogenetic discordance. We identify the causes of the discordance and
      demonstrate that the Medicago orbicularis lineage experienced an episode of
      ancient hybridization. Our results show promise as a new way to explore
      phylogenetic sequence data that can significantly improve species tree inference 
      in presence of hybridization and undetected paralogy or other causes leading to
      extremely discordant gene trees. [Coalescent simulation; gene tree; genomic
      location; hybridization; incomplete lineage sorting; paralogy; phylogenetic
      incongruence; principal tree; species tree.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Sousa, F
AU  - Sousa F
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, 40530 Gothenburg, Sweden.
FAU - Bertrand, Y J K
AU  - Bertrand YJK
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, 40530 Gothenburg, Sweden.
FAU - Doyle, J J
AU  - Doyle JJ
AD  - Department of Plant Biology, Cornell University, 404 Mann Library Building,
      Ithaca, NY 14853, USA.
FAU - Oxelman, B
AU  - Oxelman B
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, 40530 Gothenburg, Sweden.
FAU - Pfeil, B E
AU  - Pfeil BE
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, 40530 Gothenburg, Sweden.
LA  - eng
SI  - Dryad/10.5061/dryad.85c0s
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Computer Simulation
MH  - Genome, Plant/*genetics
MH  - Hybridization, Genetic
MH  - Medicago/*classification/*genetics
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Software
EDAT- 2017/02/09 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/02/09 06:00
PHST- 2015/06/05 00:00 [received]
PHST- 2017/02/01 00:00 [accepted]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/02/09 06:00 [entrez]
AID - 2977664 [pii]
AID - 10.1093/sysbio/syx035 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):934-949. doi: 10.1093/sysbio/syx035.

PMID- 28175922
OWN - NLM
STAT- MEDLINE
DCOM- 20180213
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Fundamentals and Recent Developments in Approximate Bayesian Computation.
PG  - e66-e82
LID - 10.1093/sysbio/syw077 [doi]
AB  - Bayesian inference plays an important role in phylogenetics, evolutionary
      biology, and in many other branches of science. It provides a principled
      framework for dealing with uncertainty and quantifying how it changes in the
      light of new evidence. For many complex models and inference problems, however,
      only approximate quantitative answers are obtainable. Approximate Bayesian
      computation (ABC) refers to a family of algorithms for approximate inference that
      makes a minimal set of assumptions by only requiring that sampling from a model
      is possible. We explain here the fundamentals of ABC, review the classical
      algorithms, and highlight recent developments. [ABC; approximate Bayesian
      computation; Bayesian inference; likelihood-free inference; phylogenetics;
      simulator-based models; stochastic simulation models; tree-based models.]
FAU - Lintusaari, Jarno
AU  - Lintusaari J
AD  - Department of Computer Science, Aalto University, Espoo, Finland.
AD  - Helsinki Institute for Information Technology HIIT, Espoo, Finland.
FAU - Gutmann, Michael U
AU  - Gutmann MU
AD  - Department of Computer Science, Aalto University, Espoo, Finland.
AD  - Helsinki Institute for Information Technology HIIT, Espoo, Finland.
AD  - Department of Mathematics and Statistics, University of Helsinki, Helsinki,
      Finland.
FAU - Dutta, Ritabrata
AU  - Dutta R
AD  - Department of Computer Science, Aalto University, Espoo, Finland.
AD  - Helsinki Institute for Information Technology HIIT, Espoo, Finland.
FAU - Kaski, Samuel
AU  - Kaski S
AD  - Department of Computer Science, Aalto University, Espoo, Finland.
AD  - Helsinki Institute for Information Technology HIIT, Espoo, Finland.
FAU - Corander, Jukka
AU  - Corander J
AD  - Helsinki Institute for Information Technology HIIT, Espoo, Finland.
AD  - Department of Mathematics and Statistics, University of Helsinki, Helsinki,
      Finland.
AD  - Department of Biostatistics, University of Oslo, Oslo, Norway
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Bayes Theorem
MH  - *Classification
MH  - *Models, Biological
MH  - *Phylogeny
PMC - PMC5837704
EDAT- 2017/02/09 06:00
MHDA- 2018/02/14 06:00
CRDT- 2017/02/09 06:00
PHST- 2015/10/30 00:00 [received]
PHST- 2016/08/09 00:00 [revised]
PHST- 2016/08/09 00:00 [accepted]
PHST- 2017/02/09 06:00 [entrez]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/14 06:00 [medline]
AID - 2420817 [pii]
AID - 10.1093/sysbio/syw077 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):e66-e82. doi: 10.1093/sysbio/syw077.

PMID- 28173602
OWN - NLM
STAT- MEDLINE
DCOM- 20180213
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Novel Approaches for Phylogenetic Inference from Morphological Data and
      Total-Evidence Dating in Squamate Reptiles (Lizards, Snakes, and Amphisbaenians).
PG  - 38-56
LID - 10.1093/sysbio/syw068 [doi]
AB  - Here, I combine previously underutilized models and priors to perform more
      biologically realistic phylogenetic inference from morphological data, with an
      example from squamate reptiles. When coding morphological characters, it is often
      possible to denote ordered states with explicit reference to observed or
      hypothetical ancestral conditions. Using this logic, we can integrate across
      character-state labels and estimate meaningful rates of forward and backward
      transitions from plesiomorphy to apomorphy. I refer to this approach as MkA, for 
      "asymmetric." The MkA model incorporates the biological reality of limited
      reversal for many phylogenetically informative characters, and significantly
      increases likelihoods in the empirical data sets. Despite this, the phylogeny of 
      Squamata remains contentious. Total-evidence analyses using combined
      morphological and molecular data and the MkA approach tend toward recent
      consensus estimates supporting a nested Iguania. However, support for this
      topology is not unambiguous across data sets or analyses, and no mechanism has
      been proposed to explain the widespread incongruence between partitions, or the
      hidden support for various topologies in those partitions. Furthermore, different
      morphological data sets produced by different authors contain both different
      characters and different states for the same or similar characters, resulting in 
      drastically different placements for many important fossil lineages. Effort is
      needed to standardize ontology for morphology, resolve incongruence, and estimate
      a robust phylogeny. The MkA approach provides a preliminary avenue for
      investigating morphological evolution while accounting for temporal evidence and 
      asymmetry in character-state changes.
FAU - Pyron, R Alexander
AU  - Pyron RA
AD  - Department of Biological Sciences, The George Washington University, 2023 G St.
      NW, Washington, DC, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.dp3js
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Fossils
MH  - *Phylogeny
MH  - Reptiles/*anatomy &amp; histology/*classification/genetics
MH  - Time
OTO - NOTNLM
OT  - *Congruence
OT  - *convergence
OT  - *dating
OT  - *molecular discordance
OT  - *morphological phylogenetics
OT  - *reversals
OT  - *Squamata
OT  - *total evidence
EDAT- 2017/02/09 06:00
MHDA- 2018/02/14 06:00
CRDT- 2017/02/08 06:00
PHST- 2015/10/06 00:00 [received]
PHST- 2016/07/12 00:00 [revised]
PHST- 2016/07/20 00:00 [accepted]
PHST- 2017/02/08 06:00 [entrez]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/14 06:00 [medline]
AID - 2670090 [pii]
AID - 10.1093/sysbio/syw068 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):38-56. doi: 10.1093/sysbio/syw068.

PMID- 28173598
OWN - NLM
STAT- MEDLINE
DCOM- 20180213
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Macroevolution of Specificity in Cyanolichens of the Genus Peltigera Section
      Polydactylon (Lecanoromycetes, Ascomycota).
PG  - 74-99
LID - 10.1093/sysbio/syw065 [doi]
AB  - Patterns of specificity among symbiotic partners are key to a comprehensive
      understanding of the evolution of symbiotic systems. Specificity of mutualistic
      partners, within a widespread monophyletic group for which all species are
      sampled has rarely been explored. Here, we assess the level of specificity
      between the cosmopolitan lichen-forming fungus (mycobiont) from the genus
      Peltigera, section Polydactylon, and its cyanobacterial partner Nostoc
      (cyanobiont). The mycobiont and cyanobiont phylogenies are inferred from five
      nuclear loci and the rbcLX region, respectively. These sequences were obtained
      from 206 lichen thalli, representing ca. 40 closely related Peltigera species
      sampled worldwide, doubling the number of known species in this group. We found a
      broad spectrum of specificity for both partners ranging from strict specialists
      to generalists. Overall, mycobionts are more specialized than cyanobionts by
      associating mostly with one or a few Nostoc phylogroups, whereas most cyanobionts
      associate frequently with several Peltigera species. Specialist mycobionts are
      older than generalists, supporting the hypothesis that specialization of
      mycobionts to one or few cyanobionts, is favored through time in geographic areas
      where species have been established for long periods of time. The relatively
      recent colonization of a new geographic area (Central and South America) by
      members of section Polydactylon is associated with a switch to a generalist
      pattern of association and an increased diversification rate by the fungal
      partner, suggesting that switches to generalism are rare events that are
      advantageous in new environments. We detected higher genetic diversity in
      generalist mycobionts. We also found that Peltigera species specialized on a
      single Nostoc phylogroup have narrower geographical distributions compared with
      generalist species.
FAU - Magain, N
AU  - Magain N
AD  - Evolution and Conservation Biology, University of Liege, Liege, Belgium.
AD  - Department of Biology, Duke University, Durham, NC, USA.
FAU - Miadlikowska, J
AU  - Miadlikowska J
AD  - Evolution and Conservation Biology, University of Liege, Liege, Belgium.
FAU - Goffinet, B
AU  - Goffinet B
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut,
      Storrs, CT, USA.
FAU - Serusiaux, E
AU  - Serusiaux E
AD  - Evolution and Conservation Biology, University of Liege, Liege, Belgium.
FAU - Lutzoni, F
AU  - Lutzoni F
AD  - Department of Biology, Duke University, Durham, NC, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.h6v7g
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Ascomycota/*classification/*physiology
MH  - *Biological Evolution
MH  - Host Specificity
MH  - Lichens/*microbiology/*physiology
MH  - Phylogeny
MH  - South America
MH  - *Symbiosis
OTO - NOTNLM
OT  - *Cyanobiont
OT  - *lichen symbiosis
OT  - *multilocus phylogeny
OT  - *mutualistic interactions
OT  - *mycobiont
OT  - *photobiont
OT  - *species delimitation
EDAT- 2017/02/09 06:00
MHDA- 2018/02/14 06:00
CRDT- 2017/02/08 06:00
PHST- 2015/04/16 00:00 [received]
PHST- 2015/08/06 00:00 [revised]
PHST- 2016/07/17 00:00 [accepted]
PHST- 2017/02/08 06:00 [entrez]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/14 06:00 [medline]
AID - 2670079 [pii]
AID - 10.1093/sysbio/syw065 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):74-99. doi: 10.1093/sysbio/syw065.

PMID- 28173595
OWN - NLM
STAT- MEDLINE
DCOM- 20180213
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Ontogenetic Data Analyzed As Such in Phylogenies.
PG  - 23-37
LID - 10.1093/sysbio/syw052 [doi]
AB  - Ontogeny is rarely included in phylogenetic analyses of morphological data. When 
      used, the ontogenetic information is reduced to one character for two or three
      different ontogenetic stages. Several examples show that current methods miss a
      major part of the information. We here propose a new method for including the
      ontogenetic dimension in coding schemes of phylogenetic analyses. Our goal was to
      maximize the phylogenetic information extracted from ontogenetic trajectories.
      For discrete features, we recommend including precise timings of
      transformation(s) from one state to another in the ontogenetic trajectories. For 
      continuously varying features, growth laws are modeled on raw data using
      least-square regressions. Then, parameters of models are included in the coding
      scheme as continuous characters. This method is employed to reconstruct
      phylogenetic relationships using the ammonite family Amaltheidae as a test
      subject. Based on the same data set, a second analysis has been performed only
      for characters of the adult stage. Comparisons of retention index, bootstrap
      support, and stratigraphic congruence between the two analyses show that the
      inclusion of ontogeny yields better phylogenetic reconstruction. Morphological
      traits in ammonites which are usually the most homoplastic show a better fit to
      most parsimonious trees by including the ontogenetic dimension. In several cases,
      growth rates and patterns of growth have better fit to phylogeny than adult
      shapes, implying that paths of ontogeny can be more relevant than its products.
FAU - Bardin, Jeremie
AU  - Bardin J
AD  - Sorbonne Universites, UPMC Univ Paris, Paris, France.
AD  - Museum National d'Histoire Naturelle (MNHN), Paris, France.
AD  - Centre National de la Recherche Scientifique (CNRS), Paris, France.
FAU - Rouget, Isabelle
AU  - Rouget I
AD  - Sorbonne Universites, UPMC Univ Paris, Paris, France.
AD  - Museum National d'Histoire Naturelle (MNHN), Paris, France.
AD  - Centre National de la Recherche Scientifique (CNRS), Paris, France.
FAU - Cecca, Fabrizio
AU  - Cecca F
AD  - Sorbonne Universites, UPMC Univ Paris, Paris, France.
AD  - Museum National d'Histoire Naturelle (MNHN), Paris, France.
AD  - Centre National de la Recherche Scientifique (CNRS), Paris, France.
LA  - eng
SI  - Dryad/10.5061/dryad.0c1p0
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Arthropods/anatomy &amp; histology/classification/growth &amp; development
MH  - Classification/*methods
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Amaltheidae
OT  - *ammonite
OT  - *cladistics
OT  - *development
OT  - *Jurassic
OT  - *ontogeny
OT  - *phylogeny
OT  - *Pliensbachian
EDAT- 2017/02/09 06:00
MHDA- 2018/02/14 06:00
CRDT- 2017/02/08 06:00
PHST- 2014/11/24 00:00 [received]
PHST- 2016/05/02 00:00 [revised]
PHST- 2016/06/01 00:00 [accepted]
PHST- 2017/02/08 06:00 [entrez]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/14 06:00 [medline]
AID - 2669991 [pii]
AID - 10.1093/sysbio/syw052 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):23-37. doi: 10.1093/sysbio/syw052.

PMID- 28173592
OWN - NLM
STAT- MEDLINE
DCOM- 20180213
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Rarefaction and Extrapolation: Making Fair Comparison of Abundance-Sensitive
      Phylogenetic Diversity among Multiple Assemblages.
PG  - 100-111
LID - 10.1093/sysbio/syw073 [doi]
AB  - Measures of phylogenetic diversity are basic tools in many studies of systematic 
      biology. Faith's PD (sum of branch lengths of a phylogenetic tree connecting all 
      focal species) is the most widely used phylogenetic measure. Like species
      richness, Faith's PD based on sampling data is highly dependent on sample size
      and sample completeness. The sample-size- and sample-coverage-based integration
      of rarefaction and extrapolation of Faith's PD was recently developed to make
      fair comparison across multiple assemblages. However, species abundances are not 
      considered in Faith's PD. Based on the framework of Hill numbers, Faith's PD was 
      generalized to a class of phylogenetic diversity measures that incorporates
      species abundances. In this article, we develop both theoretical formulae and
      analytic estimators for seamless rarefaction and extrapolation for this class of 
      abundance-sensitive phylogenetic measures, which includes simple transformations 
      of phylogenetic entropy and of quadratic entropy. This work generalizes the
      previous rarefaction/extrapolation model of Faith's PD to incorporate species
      abundance, and also extends the previous rarefaction/extrapolation model of Hill 
      numbers to include phylogenetic differences among species. Thus a unified
      approach to assessing and comparing species/taxonomic diversity and phylogenetic 
      diversity can be established. A bootstrap method is suggested for constructing
      confidence intervals around the phylogenetic diversity, facilitating the
      comparison of multiple assemblages. Our formulation and estimators can be
      extended to incidence data collected from multiple sampling units. We also
      illustrate the formulae and estimators using bacterial sequence data from the
      human distal esophagus and phyllostomid bat data from three habitats.
FAU - Hsieh, T C
AU  - Hsieh TC
AD  - Institute of Statistics, National Tsing Hua University, Hsin-Chu, Taiwan.
FAU - Chao, Anne
AU  - Chao A
AD  - Institute of Statistics, National Tsing Hua University, Hsin-Chu, Taiwan.
LA  - eng
SI  - Dryad/10.5061/dryad.qk58h
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bacteria/classification/genetics
MH  - Biodiversity
MH  - Chiroptera/classification
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - Ecosystem
MH  - Esophagus/microbiology
MH  - Genes, Bacterial/genetics
MH  - Humans
MH  - Microbiota
MH  - *Models, Biological
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Extrapolation
OT  - *diversity
OT  - *Hill numbers
OT  - *interpolation
OT  - *prediction
OT  - *rarefaction
OT  - *sample completeness
OT  - *sample coverage
OT  - *phylogenetic diversity
EDAT- 2017/02/09 06:00
MHDA- 2018/02/14 06:00
CRDT- 2017/02/08 06:00
PHST- 2015/08/18 00:00 [received]
PHST- 2016/08/22 00:00 [revised]
PHST- 2016/08/22 00:00 [accepted]
PHST- 2017/02/08 06:00 [entrez]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/14 06:00 [medline]
AID - 2418029 [pii]
AID - 10.1093/sysbio/syw073 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):100-111. doi: 10.1093/sysbio/syw073.

PMID- 28173588
OWN - NLM
STAT- MEDLINE
DCOM- 20180213
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Bayesian Phylogenetic Estimation of Clade Ages Supports Trans-Atlantic Dispersal 
      of Cichlid Fishes.
PG  - 3-22
LID - 10.1093/sysbio/syw076 [doi]
AB  - Divergence-time estimation based on molecular phylogenies and the fossil record
      has provided insights into fundamental questions of evolutionary biology. In
      Bayesian node dating, phylogenies are commonly time calibrated through the
      specification of calibration densities on nodes representing clades with known
      fossil occurrences. Unfortunately, the optimal shape of these calibration
      densities is usually unknown and they are therefore often chosen arbitrarily,
      which directly impacts the reliability of the resulting age estimates. As
      possible solutions to this problem, two nonexclusive alternative approaches have 
      recently been developed, the "fossilized birth-death" (FBD) model and
      "total-evidence dating." While these approaches have been shown to perform well
      under certain conditions, they require including all (or a random subset) of the 
      fossils of each clade in the analysis, rather than just relying on the oldest
      fossils of clades. In addition, both approaches assume that fossil records of
      different clades in the phylogeny are all the product of the same underlying
      fossil sampling rate, even though this rate has been shown to differ strongly
      between higher level taxa. We here develop a flexible new approach to Bayesian
      age estimation that combines advantages of node dating and the FBD model. In our 
      new approach, calibration densities are defined on the basis of first fossil
      occurrences and sampling rate estimates that can be specified separately for all 
      clades. We verify our approach with a large number of simulated data sets, and
      compare its performance to that of the FBD model. We find that our approach
      produces reliable age estimates that are robust to model violation, on par with
      the FBD model. By applying our approach to a large data set including sequence
      data from over 1000 species of teleost fishes as well as 147 carefully selected
      fossil constraints, we recover a timeline of teleost diversification that is
      incompatible with previously assumed vicariant divergences of freshwater fishes. 
      Our results instead provide strong evidence for transoceanic dispersal of
      cichlids and other groups of teleost fishes.
FAU - Matschiner, Michael
AU  - Matschiner M
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis
      (CEES), University of Oslo, Oslo, Norway.
AD  - Zoological Institute, University of Basel, Basel, Switzerland.
FAU - Musilova, Zuzana
AU  - Musilova Z
AD  - Department of Zoology, Faculty of Science, Charles University in Prague, Prague, 
      Czech Republic.
AD  - Zoological Institute, University of Basel, Basel, Switzerland.
FAU - Barth, Julia M I
AU  - Barth JM
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis
      (CEES), University of Oslo, Oslo, Norway.
FAU - Starostova, Zuzana
AU  - Starostova Z
AD  - Department of Zoology, Faculty of Science, Charles University in Prague, Prague, 
      Czech Republic.
FAU - Salzburger, Walter
AU  - Salzburger W
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis
      (CEES), University of Oslo, Oslo, Norway.
AD  - Zoological Institute, University of Basel, Basel, Switzerland.
FAU - Steel, Mike
AU  - Steel M
AD  - Department of Mathematics and Statistics, University of Canterbury, Christchurch,
      New Zealand.
FAU - Bouckaert, Remco
AU  - Bouckaert R
AD  - Department of Computer Science, University of Auckland, Auckland, New Zealand.
AD  - Computational Evolution Group, University of Auckland, Auckland, New Zealand.
LA  - eng
SI  - Dryad/10.5061/dryad.1q994
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Atlantic Ocean
MH  - Bayes Theorem
MH  - Biodiversity
MH  - Cichlids/*classification
MH  - Fossils
MH  - Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Time
OTO - NOTNLM
OT  - *Bayesian inference
OT  - *calibration density
OT  - *Cichlidae
OT  - *fossil record
OT  - *marine dispersal
OT  - *phylogeny
OT  - *relaxed molecular clock
EDAT- 2017/02/09 06:00
MHDA- 2018/02/14 06:00
CRDT- 2017/02/08 06:00
PHST- 2016/03/21 00:00 [received]
PHST- 2016/06/21 00:00 [revised]
PHST- 2016/08/23 00:00 [accepted]
PHST- 2017/02/08 06:00 [entrez]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/14 06:00 [medline]
AID - 2418030 [pii]
AID - 10.1093/sysbio/syw076 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):3-22. doi: 10.1093/sysbio/syw076.

PMID- 28173586
OWN - NLM
STAT- MEDLINE
DCOM- 20180213
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Evolutionary Patterns and Processes: Lessons from Ancient DNA.
PG  - e1-e29
LID - 10.1093/sysbio/syw059 [doi]
AB  - Ever since its emergence in 1984, the field of ancient DNA has struggled to
      overcome the challenges related to the decay of DNA molecules in the fossil
      record. With the recent development of high-throughput DNA sequencing
      technologies and molecular techniques tailored to ultra-damaged templates, it has
      now come of age, merging together approaches in phylogenomics, population
      genomics, epigenomics, and metagenomics. Leveraging on complete temporal sample
      series, ancient DNA provides direct access to the most important dimension in
      evolution-time, allowing a wealth of fundamental evolutionary processes to be
      addressed at unprecedented resolution. This review taps into the most recent
      findings in ancient DNA research to present analyses of ancient genomic and
      metagenomic data.
FAU - Leonardi, Michela
AU  - Leonardi M
AD  - Centre for GeoGenetics, Natural History Museum of Denmark, University of
      Copenhagen, Oster Voldgade, Copenhagen, Denmark.
FAU - Librado, Pablo
AU  - Librado P
AD  - Centre for GeoGenetics, Natural History Museum of Denmark, University of
      Copenhagen, Oster Voldgade, Copenhagen, Denmark.
FAU - Der Sarkissian, Clio
AU  - Der Sarkissian C
AD  - Centre for GeoGenetics, Natural History Museum of Denmark, University of
      Copenhagen, Oster Voldgade, Copenhagen, Denmark.
FAU - Schubert, Mikkel
AU  - Schubert M
AD  - Centre for GeoGenetics, Natural History Museum of Denmark, University of
      Copenhagen, Oster Voldgade, Copenhagen, Denmark.
FAU - Alfarhan, Ahmed H
AU  - Alfarhan AH
AD  - Zoology Department, College of Science, King Saud University, Riyadh, Saudi
      Arabia.
FAU - Alquraishi, Saleh A
AU  - Alquraishi SA
AD  - Zoology Department, College of Science, King Saud University, Riyadh, Saudi
      Arabia.
FAU - Al-Rasheid, Khaled A S
AU  - Al-Rasheid KA
AD  - Zoology Department, College of Science, King Saud University, Riyadh, Saudi
      Arabia.
FAU - Gamba, Cristina
AU  - Gamba C
AD  - Centre for GeoGenetics, Natural History Museum of Denmark, University of
      Copenhagen, Oster Voldgade, Copenhagen, Denmark.
FAU - Willerslev, Eske
AU  - Willerslev E
AD  - Centre for GeoGenetics, Natural History Museum of Denmark, University of
      Copenhagen, Oster Voldgade, Copenhagen, Denmark.
AD  - Zoology Department, College of Science, King Saud University, Riyadh, Saudi
      Arabia.
FAU - Orlando, Ludovic
AU  - Orlando L
AD  - Centre for GeoGenetics, Natural History Museum of Denmark, University of
      Copenhagen, Oster Voldgade, Copenhagen, Denmark.
AD  - Universite de Toulouse, University Paul Sabatier (UPS), Laboratoire AMIS,
      Toulouse, France.
LA  - eng
SI  - Dryad/10.5061/dryad.tt78r
PT  - Journal Article
PT  - Review
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ancient)
SB  - IM
EIN - Syst Biol. 2017 Jul 1;66(4):660. PMID: 28123113
MH  - *Biological Evolution
MH  - *DNA, Ancient
MH  - Evolution, Molecular
MH  - Fossils
MH  - Genomics/trends
MH  - Research/*trends
PMC - PMC5410953
OTO - NOTNLM
OT  - *Ancient DNA
OT  - *metagenomics
OT  - *palaeogenomics
OT  - *population genomics
OT  - *temporal sampling
EDAT- 2017/02/09 06:00
MHDA- 2018/02/14 06:00
CRDT- 2017/02/08 06:00
PHST- 2016/03/26 00:00 [received]
PHST- 2016/06/04 00:00 [revised]
PHST- 2016/06/06 00:00 [accepted]
PHST- 2017/02/08 06:00 [entrez]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/14 06:00 [medline]
AID - 2670024 [pii]
AID - 10.1093/sysbio/syw059 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):e1-e29. doi: 10.1093/sysbio/syw059.

PMID- 28173553
OWN - NLM
STAT- MEDLINE
DCOM- 20180213
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Statistical Inference in the Wright-Fisher Model Using Allele Frequency Data.
PG  - e30-e46
LID - 10.1093/sysbio/syw056 [doi]
AB  - The Wright-Fisher model provides an elegant mathematical framework for
      understanding allele frequency data. In particular, the model can be used to
      infer the demographic history of species and identify loci under selection. A
      crucial quantity for inference under the Wright-Fisher model is the distribution 
      of allele frequencies (DAF). Despite the apparent simplicity of the model, the
      calculation of the DAF is challenging. We review and discuss strategies for
      approximating the DAF, and how these are used in methods that perform inference
      from allele frequency data. Various evolutionary forces can be incorporated in
      the Wright-Fisher model, and we consider these in turn. We begin our review with 
      the basic bi-allelic Wright-Fisher model where random genetic drift is the only
      evolutionary force. We then consider mutation, migration, and selection. In
      particular, we compare diffusion-based and moment-based methods in terms of
      accuracy, computational efficiency, and analytical tractability. We conclude with
      a brief overview of the multi-allelic process with a general mutation model.
FAU - Tataru, Paula
AU  - Tataru P
AD  - Bioinformatics Research Centre, Aarhus University, Aarhus, Denmark.
FAU - Simonsen, Maria
AU  - Simonsen M
AD  - Bioinformatics Research Centre, Aarhus University, Aarhus, Denmark.
FAU - Bataillon, Thomas
AU  - Bataillon T
AD  - Bioinformatics Research Centre, Aarhus University, Aarhus, Denmark.
FAU - Hobolth, Asger
AU  - Hobolth A
AD  - Bioinformatics Research Centre, Aarhus University, Aarhus, Denmark.
LA  - eng
PT  - Journal Article
PT  - Review
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Evolution, Molecular
MH  - Gene Frequency/*genetics
MH  - Genetic Drift
MH  - *Models, Genetic
MH  - Mutation
PMC - PMC5837693
OTO - NOTNLM
OT  - *Allele frequency
OT  - *diffusion
OT  - *inference
OT  - *moments
OT  - *selection
OT  - *Wright-Fisher
EDAT- 2017/02/09 06:00
MHDA- 2018/02/14 06:00
CRDT- 2017/02/08 06:00
PHST- 2015/12/04 00:00 [received]
PHST- 2016/05/31 00:00 [revised]
PHST- 2016/06/06 00:00 [accepted]
PHST- 2017/02/08 06:00 [entrez]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/14 06:00 [medline]
AID - 2670014 [pii]
AID - 10.1093/sysbio/syw056 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):e30-e46. doi: 10.1093/sysbio/syw056.

PMID- 28173538
OWN - NLM
STAT- MEDLINE
DCOM- 20180213
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Review Paper: The Shape of Phylogenetic Treespace.
PG  - e83-e94
LID - 10.1093/sysbio/syw025 [doi]
AB  - Trees are a canonical structure for representing evolutionary histories. Many
      popular criteria used to infer optimal trees are computationally hard, and the
      number of possible tree shapes grows super-exponentially in the number of taxa.
      The underlying structure of the spaces of trees yields rich insights that can
      improve the search for optimal trees, both in accuracy and in running time, and
      the analysis and visualization of results. We review the past work on analyzing
      and comparing trees by their shape as well as recent work that incorporates trees
      with weighted branch lengths.
FAU - St. John, Katherine
AU  - St. John K
AD  - Department of Mathematics and Computer Science, Lehman College, NY, USA.
LA  - eng
PT  - Journal Article
PT  - Review
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Classification
MH  - *Phylogeny
MH  - Reproducibility of Results
PMC - PMC5837343
OTO - NOTNLM
OT  - *Maximum likelihood
OT  - *maximum parsimony
OT  - *tree metrics
OT  - *treespace
EDAT- 2017/02/09 06:00
MHDA- 2018/02/14 06:00
CRDT- 2017/02/08 06:00
PHST- 2015/11/20 00:00 [received]
PHST- 2015/12/16 00:00 [revised]
PHST- 2016/03/22 00:00 [accepted]
PHST- 2017/02/08 06:00 [entrez]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/14 06:00 [medline]
AID - 2669975 [pii]
AID - 10.1093/sysbio/syw025 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):e83-e94. doi: 10.1093/sysbio/syw025.

PMID- 28173531
OWN - NLM
STAT- MEDLINE
DCOM- 20180213
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Bayesian Total-Evidence Dating Reveals the Recent Crown Radiation of Penguins.
PG  - 57-73
LID - 10.1093/sysbio/syw060 [doi]
AB  - The total-evidence approach to divergence time dating uses molecular and
      morphological data from extant and fossil species to infer phylogenetic
      relationships, species divergence times, and macroevolutionary parameters in a
      single coherent framework. Current model-based implementations of this approach
      lack an appropriate model for the tree describing the diversification and
      fossilization process and can produce estimates that lead to erroneous
      conclusions. We address this shortcoming by providing a total-evidence method
      implemented in a Bayesian framework. This approach uses a mechanistic tree prior 
      to describe the underlying diversification process that generated the tree of
      extant and fossil taxa. Previous attempts to apply the total-evidence approach
      have used tree priors that do not account for the possibility that fossil samples
      may be direct ancestors of other samples, that is, ancestors of fossil or extant 
      species or of clades. The fossilized birth-death (FBD) process explicitly models 
      the diversification, fossilization, and sampling processes and naturally allows
      for sampled ancestors. This model was recently applied to estimate divergence
      times based on molecular data and fossil occurrence dates. We incorporate the FBD
      model and a model of morphological trait evolution into a Bayesian total-evidence
      approach to dating species phylogenies. We apply this method to extant and fossil
      penguins and show that the modern penguins radiated much more recently than has
      been previously estimated, with the basal divergence in the crown clade occurring
      at approximately 12.7 approximately 12.7 Ma and most splits leading to extant
      species occurring in the last 2 myr. Our results demonstrate that including
      stem-fossil diversity can greatly improve the estimates of the divergence times
      of crown taxa. The method is available in BEAST2 (version 2.4) software
      www.beast2.org with packages SA (version at least 1.1.4) and morph-models
      (version at least 1.0.4) installed.
FAU - Gavryushkina, Alexandra
AU  - Gavryushkina A
AD  - Centre for Computational Evolution, University of Auckland, Auckland, New
      Zealand.
AD  - Department of Computer Science, University of Auckland, Auckland, New Zealand.
FAU - Heath, Tracy A
AU  - Heath TA
AD  - Department of Ecology, Evolution, &amp; Organismal Biology, Iowa State University,
      Ames, IA, USA.
FAU - Ksepka, Daniel T
AU  - Ksepka DT
AD  - Bruce Museum, Greenwich, CT, USA.
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Department of Biosystems Science &amp; Engineering, Eidgenossische Technische
      Hochschule Zurich, Basel, Switzerland.
FAU - Welch, David
AU  - Welch D
AD  - Centre for Computational Evolution, University of Auckland, Auckland, New
      Zealand.
AD  - Department of Computer Science, University of Auckland, Auckland, New Zealand.
FAU - Drummond, Alexei J
AU  - Drummond AJ
AD  - Centre for Computational Evolution, University of Auckland, Auckland, New
      Zealand.
AD  - Department of Computer Science, University of Auckland, Auckland, New Zealand.
LA  - eng
SI  - Dryad/10.5061/dryad.44pf8
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Fossils
MH  - Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Spheniscidae/anatomy &amp; histology/*classification/genetics
MH  - Time Factors
PMC - PMC5410945
OTO - NOTNLM
OT  - *Birth-death process
OT  - *calibration
OT  - *divergence times
OT  - *MCMC
OT  - *phylogenetics
EDAT- 2017/02/09 06:00
MHDA- 2018/02/14 06:00
CRDT- 2017/02/08 06:00
PHST- 2015/06/14 00:00 [received]
PHST- 2016/06/08 00:00 [revised]
PHST- 2016/06/09 00:00 [accepted]
PHST- 2017/02/08 06:00 [entrez]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/14 06:00 [medline]
AID - 2670056 [pii]
AID - 10.1093/sysbio/syw060 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):57-73. doi: 10.1093/sysbio/syw060.

PMID- 28173504
OWN - NLM
STAT- MEDLINE
DCOM- 20180213
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Emerging Concepts of Data Integration in Pathogen Phylodynamics.
PG  - e47-e65
LID - 10.1093/sysbio/syw054 [doi]
AB  - Phylodynamics has become an increasingly popular statistical framework to extract
      evolutionary and epidemiological information from pathogen genomes. By harnessing
      such information, epidemiologists aim to shed light on the spatio-temporal
      patterns of spread and to test hypotheses about the underlying interaction of
      evolutionary and ecological dynamics in pathogen populations. Although the field 
      has witnessed a rich development of statistical inference tools with increasing
      levels of sophistication, these tools initially focused on sequences as their
      sole primary data source. Integrating various sources of information, however,
      promises to deliver more precise insights in infectious diseases and to increase 
      opportunities for statistical hypothesis testing. Here, we review how the
      emerging concept of data integration is stimulating new advances in Bayesian
      evolutionary inference methodology which formalize a marriage of statistical
      thinking and evolutionary biology. These approaches include connecting sequence
      to trait evolution, such as for host, phenotypic and geographic sampling
      information, but also the incorporation of covariates of evolutionary and
      epidemic processes in the reconstruction procedures. We highlight how a full
      Bayesian approach to covariate modeling and testing can generate further insights
      into sequence evolution, trait evolution, and population dynamics in pathogen
      populations. Specific examples demonstrate how such approaches can be used to
      test the impact of host on rabies and HIV evolutionary rates, to identify the
      drivers of influenza dispersal as well as the determinants of rabies
      cross-species transmissions, and to quantify the evolutionary dynamics of
      influenza antigenicity. Finally, we briefly discuss how data integration is now
      also permeating through the inference of transmission dynamics, leading to novel 
      insights into tree-generative processes and detailed reconstructions of
      transmission trees. [Bayesian inference; birth-death models; coalescent models;
      continuous trait evolution; covariates; data integration; discrete trait
      evolution; pathogen phylodynamics.
FAU - Baele, Guy
AU  - Baele G
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven - University
      of Leuven, Leuven, Belgium.
FAU - Suchard, Marc A
AU  - Suchard MA
AD  - Department of Biomathematics, David Geffen School of Medicine, University of
      California, Los Angeles, CA, USA.
AD  - Department of Human Genetics, David Geffen School of Medicine, University of
      California, Los Angeles, CA, USA.
AD  - Department of Biostatistics, School of Public Health, University of California,
      Los Angeles, CA, USA.
FAU - Rambaut, Andrew
AU  - Rambaut A
AD  - Institute of Evolutionary Biology, University of Edinburgh, Kings Buildings,
      Edinburgh, UK.
AD  - Centre for Immunity, Infection and Evolution, University of Edinburgh, Kings
      Buildings, Edinburgh, UK.
FAU - Lemey, Philippe
AU  - Lemey P
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven - University
      of Leuven, Leuven, Belgium.
LA  - eng
GR  - R01 AI107034/AI/NIAID NIH HHS/United States
GR  - R01 HG006139/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Review
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - *Epidemiology
MH  - Host-Pathogen Interactions
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Virus Diseases/epidemiology/transmission/virology
MH  - Viruses/classification/genetics/pathogenicity
PMC - PMC5837209
EDAT- 2017/02/09 06:00
MHDA- 2018/02/14 06:00
CRDT- 2017/02/08 06:00
PHST- 2015/11/14 00:00 [received]
PHST- 2016/06/02 00:00 [accepted]
PHST- 2017/02/08 06:00 [entrez]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/14 06:00 [medline]
AID - 2670001 [pii]
AID - 10.1093/sysbio/syw054 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):e47-e65. doi: 10.1093/sysbio/syw054.
</pre></body></html>