<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0035)https://www.ncbi.nlm.nih.gov/pubmed -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"></head><body><pre>PMID- 12857645
OWN - NLM
STAT- MEDLINE
DCOM- 20030916
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 4
DP  - 2003 Aug
TI  - Fishes and birds: Gondwana life rafts reconsidered.
PG  - 548-53
FAU - Briggs, John C
AU  - Briggs JC
AD  - Zoological Collections, Georgia Museum of Natural History, University of Georgia,
      Athens, Georgia 30602, USA. johnc.briggs@verizon.net
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
CIN - Syst Biol. 2005 Feb;54(1):158-65. PMID: 15823966
MH  - Africa
MH  - Animals
MH  - Australia
MH  - Birds/*classification/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Fishes/*classification/genetics
MH  - Fossils
MH  - Geography
MH  - India
MH  - *Phylogeny
MH  - South America
EDAT- 2003/07/15 05:00
MHDA- 2003/09/17 05:00
CRDT- 2003/07/15 05:00
PHST- 2003/07/15 05:00 [pubmed]
PHST- 2003/09/17 05:00 [medline]
PHST- 2003/07/15 05:00 [entrez]
AID - V4XU3BUXPH0TCT7B [pii]
AID - 10.1080/10635150390218385 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Aug;52(4):548-53. doi: 10.1080/10635150390218385.

PMID- 12857644
OWN - NLM
STAT- MEDLINE
DCOM- 20030916
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 4
DP  - 2003 Aug
TI  - Phylogeny reconstruction: the role of morphology.
PG  - 539-48
FAU - Scotland, Robert W
AU  - Scotland RW
AD  - Department of Plant Sciences, University of Oxford, South Parks Road, Oxford OX1 
      3RB, U.K. robert.scotland@plant-sciences.ox.ac.uk
FAU - Olmstead, Richard G
AU  - Olmstead RG
FAU - Bennett, Jonathan R
AU  - Bennett JR
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2004 Apr;53(2):333-42. PMID: 15205057
CIN - Syst Biol. 2005 Feb;54(1):166-73. PMID: 15805017
MH  - *Anatomy, Comparative
MH  - *Phylogeny
MH  - Sample Size
EDAT- 2003/07/15 05:00
MHDA- 2003/09/17 05:00
CRDT- 2003/07/15 05:00
PHST- 2003/07/15 05:00 [pubmed]
PHST- 2003/09/17 05:00 [medline]
PHST- 2003/07/15 05:00 [entrez]
AID - 6D6FEC35GJG1WJYF [pii]
AID - 10.1080/10635150390223613 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Aug;52(4):539-48. doi: 10.1080/10635150390223613.

PMID- 12857643
OWN - NLM
STAT- MEDLINE
DCOM- 20030916
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 4
DP  - 2003 Aug
TI  - Missing data, incomplete taxa, and phylogenetic accuracy.
PG  - 528-38
AB  - The problem of missing data is often considered to be the most important obstacle
      in reconstructing the phylogeny of fossil taxa and in combining data from diverse
      characters and taxa for phylogenetic analysis. Empirical and theoretical studies 
      show that including highly incomplete taxa can lead to multiple equally
      parsimonious trees, poorly resolved consensus trees, and decreased phylogenetic
      accuracy. However, the mechanisms that cause incomplete taxa to be problematic
      have remained unclear. It has been widely assumed that incomplete taxa are
      problematic because of the proportion or amount of missing data that they bear.
      In this study, I use simulations to show that the reduced accuracy associated
      with including incomplete taxa is caused by these taxa bearing too few complete
      characters rather than too many missing data cells. This seemingly subtle
      distinction has a number of important implications. First, the so-called missing 
      data problem for incomplete taxa is, paradoxically, not directly related to their
      amount or proportion of missing data. Thus, the level of completeness alone
      should not guide the exclusion of taxa (contrary to common practice), and these
      results may explain why empirical studies have sometimes found little
      relationship between the completeness of a taxon and its impact on an analysis.
      These results also (1) suggest a more effective strategy for dealing with
      incomplete taxa, (2) call into question a justification of the controversial
      phylogenetic supertree approach, and (3) show the potential for the accurate
      phylogenetic placement of highly incomplete taxa, both when combining diverse
      data sets and when analyzing relationships of fossil taxa.
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Department of Ecology and Evolution, State University of New York, Stony Brook,
      New York 11794-5245, USA. wiensj@life.bio.sunysb.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - Data Interpretation, Statistical
MH  - Fossils
MH  - *Phylogeny
MH  - *Research Design
EDAT- 2003/07/15 05:00
MHDA- 2003/09/17 05:00
CRDT- 2003/07/15 05:00
PHST- 2003/07/15 05:00 [pubmed]
PHST- 2003/09/17 05:00 [medline]
PHST- 2003/07/15 05:00 [entrez]
AID - 01G6R26VC07T0PJ1 [pii]
AID - 10.1080/10635150390218330 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Aug;52(4):528-38. doi: 10.1080/10635150390218330.

PMID- 12857642
OWN - NLM
STAT- MEDLINE
DCOM- 20030916
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 4
DP  - 2003 Aug
TI  - The roots of phylogeny: how did Haeckel build his trees?
PG  - 515-27
AB  - Haeckel created much of our current vocabulary in evolutionary biology, such as
      the term phylogeny, which is currently used to designate trees. Assuming that
      Haeckel gave the same meaning to this term, one often reproduces Haeckel's trees 
      as the first illustrations of phylogenetic trees. A detailed analysis of
      Haeckel's own evolutionary vocabulary and theory revealed that Haeckel's trees
      were genealogical trees and that Haeckel's phylogeny was a morphological concept.
      However, phylogeny was actually the core of Haeckel's tree reconstruction, and
      understanding the exact meaning Haeckel gave to phylogeny is crucial to
      understanding the information Haeckel wanted to convey in his famous trees.
      Haeckel's phylogeny was a linear series of main morphological stages along the
      line of descent of a given species. The phylogeny of a single species would
      provide a trunk around which lateral branches were added as mere ornament; the
      phylogeny selected for drawing a tree of a given group was considered the most
      complete line of progress from lower to higher forms of this group, such as the
      phylogeny of Man for the genealogical tree of Vertebrates. Haeckel's phylogeny
      was mainly inspired by the idea of the scala naturae, or scale of being.
      Therefore, Haeckel's genealogical trees, which were only branched on the surface,
      mainly represented the old idea of scale of being. Even though Haeckel decided to
      draw genealogical trees after reading On the Origin of Species and was called the
      German Darwin, he did not draw Darwinian branching diagrams. Although Haeckel
      always saw Lamarck, Goethe, and Darwin as the three fathers of the theory of
      evolution, he was mainly influenced by Lamarck and Goethe in his approach to tree
      reconstruction.
FAU - Dayrat, Benoit
AU  - Dayrat B
AD  - Department of Invertebrate Zoology and Geology, California Academy of Sciences,
      Golden Gate Park, San Francisco, California 94118, USA. bdayrat@calacademy.org
LA  - eng
PT  - Historical Article
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Anatomy, Comparative/history
MH  - Developmental Biology/history
MH  - History, 19th Century
MH  - *Phylogeny
EDAT- 2003/07/15 05:00
MHDA- 2003/09/17 05:00
CRDT- 2003/07/15 05:00
PHST- 2003/07/15 05:00 [pubmed]
PHST- 2003/09/17 05:00 [medline]
PHST- 2003/07/15 05:00 [entrez]
AID - 6LE0F7DGNDWMYA1L [pii]
AID - 10.1080/10635150390218277 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Aug;52(4):515-27. doi: 10.1080/10635150390218277.

PMID- 12857641
OWN - NLM
STAT- MEDLINE
DCOM- 20030916
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 4
DP  - 2003 Aug
TI  - When are phylogenetic analyses misled by convergence? A case study in Texas cave 
      salamanders.
PG  - 501-14
AB  - Convergence, i.e., similarity between organisms that is not the direct result of 
      shared phylogenetic history (and that may instead result from independent
      adaptations to similar environments), is a fundamental issue that lies at the
      interface of systematics and evolutionary biology. Although convergence is often 
      cited as an important problem in morphological phylogenetics, there have been few
      well-documented examples of strongly supported and misleading phylogenetic
      estimates that result from adaptive convergence in morphology. In this article,
      we propose criteria that can be used to infer whether or not a phylogenetic
      analysis has been misled by convergence. We then apply these criteria in a study 
      of central Texas cave salamanders (genus Eurycea). Morphological characters
      (apparently related to cave-dwelling habitat use) support a clade uniting the
      species E. rathbuni and E. tridentifera, whereas mitochondrial DNA sequences and 
      allozyme data show that these two species are not closely related. We suggest
      that a likely explanation for the paucity of examples of strongly misleading
      morphological convergence is that the conditions under which adaptive convergence
      is most likely to produce strongly misleading results are limited. Specifically, 
      convergence is most likely to be problematic in groups (such as the central Texas
      Eurycea) in which most species are morphologically very similar and some of the
      species have invaded and adapted to a novel selective environment.
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Department of Ecology and Evolution, State University of New York at Stony Brook,
      Stony Brook, New York 11794-5245, USA. wiensj@life.bio.sunysb.edu
FAU - Chippindale, Paul T
AU  - Chippindale PT
FAU - Hillis, David M
AU  - Hillis DM
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - *Adaptation, Biological
MH  - Animals
MH  - DNA, Mitochondrial/genetics
MH  - *Phylogeny
MH  - Texas
MH  - Urodela/anatomy &amp; histology/*classification
EDAT- 2003/07/15 05:00
MHDA- 2003/09/17 05:00
CRDT- 2003/07/15 05:00
PHST- 2003/07/15 05:00 [pubmed]
PHST- 2003/09/17 05:00 [medline]
PHST- 2003/07/15 05:00 [entrez]
AID - YA6M1XGAC3468KC0 [pii]
AID - 10.1080/10635150390218222 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Aug;52(4):501-14. doi: 10.1080/10635150390218222.

PMID- 12857640
OWN - NLM
STAT- MEDLINE
DCOM- 20030916
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 4
DP  - 2003 Aug
TI  - The effect of phylogeny on interspecific body shape variation in darters (Pisces:
      Percidae).
PG  - 488-500
AB  - We conducted a geometric morphometric analysis of interspecific body shape
      variation among representatives of 31 species of darters (Pisces: Percidae) to
      determine whether there is evidence of a phylogenetic effect in body shape
      variation. Cartesian transformation grids representing relative shape differences
      of individual species and subspecies revealed qualitative similarities within
      most traditionally recognized taxonomic groups (genera and subgenera). Canonical 
      variates analysis and a UPGMA cluster analysis were conducted to explore further 
      the relationships among body shapes of species; both analyses revealed patterns
      of variation consistent with the interpretation that shape is associated with
      taxonomic affinities. Normalized Mantel statistics revealed a significant
      positive association between body shape differences and phylogenetic
      interrelatedness for each of four recent phylogenetic hypotheses, providing
      evidence of a phylogenetic effect. This result is somewhat surprising, however,
      given the largely incompatible nature of these four phylogenies. We provide
      evidence that this result may be due to (1) the inclusion of multiple sets of
      closely related species to represent the traditionally recognized genera and
      subgenera within each phylogeny and/or (2) the inclusion of several species with 
      relatively divergent shapes and their particular positions within the phylogenies
      relative to one another or to the other species of darters.
FAU - Guill, J Michael
AU  - Guill JM
AD  - Department of Ecology and Evolutionary Biology, Tulane University, New Orleans,
      Louisiana 70118, USA. guill@tulane.edu
FAU - Heins, David C
AU  - Heins DC
FAU - Hood, Craig S
AU  - Hood CS
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biometry
MH  - Cluster Analysis
MH  - Perciformes/anatomy &amp; histology/*classification
MH  - *Phylogeny
EDAT- 2003/07/15 05:00
MHDA- 2003/09/17 05:00
CRDT- 2003/07/15 05:00
PHST- 2003/07/15 05:00 [pubmed]
PHST- 2003/09/17 05:00 [medline]
PHST- 2003/07/15 05:00 [entrez]
AID - KXQV7UV7CW7R6CEP [pii]
AID - 10.1080/10635150390197019 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Aug;52(4):488-500. doi: 10.1080/10635150390197019.

PMID- 12857639
OWN - NLM
STAT- MEDLINE
DCOM- 20030916
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 4
DP  - 2003 Aug
TI  - Comparing bootstrap and posterior probability values in the four-taxon case.
PG  - 477-87
AB  - Assessment of the reliability of a given phylogenetic hypothesis is an important 
      step in phylogenetic analysis. Historically, the nonparametric bootstrap
      procedure has been the most frequently used method for assessing the support for 
      specific phylogenetic relationships. The recent employment of Bayesian methods
      for phylogenetic inference problems has resulted in clade support being expressed
      in terms of posterior probabilities. We used simulated data and the four-taxon
      case to explore the relationship between nonparametric bootstrap values (as
      inferred by maximum likelihood) and posterior probabilities (as inferred by
      Bayesian analysis). The results suggest a complex association between the two
      measures. Three general regions of tree space can be identified: (1) the neutral 
      zone, where differences between mean bootstrap and mean posterior probability
      values are not significant, (2) near the two-branch corner, and (3) deep in the
      two-branch corner. In the last two regions, significant differences occur between
      mean bootstrap and mean posterior probability values. Whether bootstrap or
      posterior probability values are higher depends on the data in support of
      alternative topologies. Examination of star topologies revealed that both
      bootstrap and posterior probability values differ significantly from theoretical 
      expectations; in particular, there are more posterior probability values in the
      range 0.85-1 than expected by theory. Therefore, our results corroborate the
      findings of others that posterior probability values are excessively high. Our
      results also suggest that extrapolations from single topology branch-length
      studies are unlikely to provide any general conclusions regarding the
      relationship between bootstrap and posterior probability values.
FAU - Cummings, Michael P
AU  - Cummings MP
AD  - Center for Bioinformatics and Computational Biology, 2119 A.V. Williams,
      University of Maryland, College Park, Maryland 20742, USA. mike@mbl.edu
FAU - Handley, Scott A
AU  - Handley SA
FAU - Myers, Daniel S
AU  - Myers DS
FAU - Reed, David L
AU  - Reed DL
FAU - Rokas, Antonis
AU  - Rokas A
FAU - Winka, Katarina
AU  - Winka K
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - Models, Statistical
MH  - *Phylogeny
MH  - *Probability
MH  - Sequence Analysis/methods
EDAT- 2003/07/15 05:00
MHDA- 2003/09/17 05:00
CRDT- 2003/07/15 05:00
PHST- 2003/07/15 05:00 [pubmed]
PHST- 2003/09/17 05:00 [medline]
PHST- 2003/07/15 05:00 [entrez]
AID - B3JKUM7VGBLJEREF [pii]
AID - 10.1080/10635150390218213 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Aug;52(4):477-87. doi: 10.1080/10635150390218213.

PMID- 12857638
OWN - NLM
STAT- MEDLINE
DCOM- 20030916
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 4
DP  - 2003 Aug
TI  - Intraspecific phylogeny of the New Zealand short-tailed bat Mystacina tuberculata
      inferred from multiple mitochondrial gene sequences.
PG  - 460-76
AB  - An intraspecific phylogeny was established for the New Zealand short-tailed bat
      Mystacina tuberculata using a 2,878-bp sequence alignment from multiple
      mitochondrial genes (control region, ND2, 12S ribosomal RNA [rRNA], 16S rRNA, and
      tRNA). The inferred phylogeny comprises six lineages, with estimated divergences 
      extending back between 0.93 and 0.68 million years to the middle Pleistocene. The
      lineages do not correspond to the existing subspecific taxonomy. Although
      multiple lineages occur sympatrically in many populations, the lineages are
      geographically structured. This structure has persisted despite repeated cycles
      of range expansion and contraction in response to climatic oscillations and
      catastrophic volcanic eruptions. The distribution of lineages among populations
      in central North Island indicates that a hybrid zone was formed by simultaneous
      colonization from single-lineage source populations inhabiting remote forest
      refugia. The observed pattern is not typical of microbats, which because of their
      high mobility generally exhibit low levels of genetic differentiation and
      geographic structure over continental ranges. Although lineages of M. tuberculata
      occur sympatrically in many populations, genetic distances between them are
      sufficiently large to suggest that they may be considered evolutionary
      significant units or taxonomic subspecies.
FAU - Lloyd, Brian D
AU  - Lloyd BD
AD  - Department of Conservation, P.O. Box 10420, Wellington, New Zealand.
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence/genetics
MH  - Chiroptera/*classification/*genetics
MH  - DNA, Mitochondrial/*genetics
MH  - Evolution, Molecular
MH  - Geography
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - New Zealand
MH  - *Phylogeny
EDAT- 2003/07/15 05:00
MHDA- 2003/09/17 05:00
CRDT- 2003/07/15 05:00
PHST- 2003/07/15 05:00 [pubmed]
PHST- 2003/09/17 05:00 [medline]
PHST- 2003/07/15 05:00 [entrez]
AID - N5DLVXVQBUBRCMUA [pii]
AID - 10.1080/10635150390218187 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Aug;52(4):460-76. doi: 10.1080/10635150390218187.

PMID- 12857637
OWN - NLM
STAT- MEDLINE
DCOM- 20030916
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 4
DP  - 2003 Aug
TI  - Phylogenetics of advanced snakes (Caenophidia) based on four mitochondrial genes.
PG  - 439-59
AB  - Phylogenetic relationships among advanced snakes (Acrochordus + Colubroidea =
      Caenophidia) and the position of the genus Acrochordus relative to colubroid taxa
      are contentious. These concerns were investigated by phylogenetic analysis of
      fragments from four mitochondrial genes representing 62 caenophidian genera and 5
      noncaenophidian taxa. Four methods of phylogeny reconstruction were applied:
      matrix representation with parsimony (MRP) supertree consensus, maximum
      parsimony, maximum likelihood, and Bayesian analysis. Because of incomplete
      sampling, extensive missing data were inherent in this study. Analyses of
      individual genes retrieved roughly the same clades, but branching order varied
      greatly between gene trees, and nodal support was poor. Trees generated from
      combined data sets using maximum parsimony, maximum likelihood, and Bayesian
      analysis had medium to low nodal support but were largely congruent with each
      other and with MRP supertrees. Conclusions about caenophidian relationships were 
      based on these combined analyses. The Xenoderminae, Viperidae, Pareatinae,
      Psammophiinae, Pseudoxyrophiinae, Homalopsinae, Natricinae, Xenodontinae, and
      Colubrinae (redefined) emerged as monophyletic, whereas Lamprophiinae,
      Atractaspididae, and Elapidae were not in one or more topologies. A clade
      comprising Acrochordus and Xenoderminae branched closest to the root, and when
      Acrochordus was assessed in relation to a colubroid subsample and all five
      noncaenophidians, it remained associated with the Colubroidea. Thus, Acrochordus 
      + Xenoderminae appears to be the sister group to the Colubroidea, and
      Xenoderminae should be excluded from Colubroidea. Within Colubroidea, Viperidae
      was the most basal clade. Other relationships appearing in all final topologies
      were (1) a clade comprising Psammophiinae, Lamprophiinae, Atractaspididae,
      Pseudoxyrophiinae, and Elapidae, within which the latter four taxa formed a
      subclade, and (2) a clade comprising Colubrinae, Natricinae, and Xenodontinae,
      within which the latter two taxa formed a subclade. Pareatinae and Homalopsinae
      were the most unstable clades.
FAU - Kelly, Christopher M R
AU  - Kelly CM
AD  - Molecular Ecology and Systematics Facility, Department of Botany, Rhodes
      University, Grahamstown, South Africa.
FAU - Barker, Nigel P
AU  - Barker NP
FAU - Villet, Martin H
AU  - Villet MH
LA  - eng
SI  - GENBANK/AY235721
SI  - GENBANK/AY235722
SI  - GENBANK/AY235723
SI  - GENBANK/AY235724
SI  - GENBANK/AY235725
SI  - GENBANK/AY235726
SI  - GENBANK/AY235727
SI  - GENBANK/AY235728
SI  - GENBANK/AY235729
SI  - GENBANK/AY235730
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Cytochrome b Group/genetics
MH  - DNA, Mitochondrial/*genetics
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Snakes/*classification/*genetics
EDAT- 2003/07/15 05:00
MHDA- 2003/09/17 05:00
CRDT- 2003/07/15 05:00
PHST- 2003/07/15 05:00 [pubmed]
PHST- 2003/09/17 05:00 [medline]
PHST- 2003/07/15 05:00 [entrez]
AID - H8UQB5ULRH329TPQ [pii]
AID - 10.1080/10635150390218132 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Aug;52(4):439-59. doi: 10.1080/10635150390218132.

PMID- 12775530
OWN - NLM
STAT- MEDLINE
DCOM- 20030909
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 3
DP  - 2003 Jun
TI  - The taxonomic impediment overcome: NSF's Partnerships for Enhancing Expertise in 
      Taxonomy (PEET) as a model.
PG  - 428-35
FAU - Rodman, James E
AU  - Rodman JE
AD  - Division of Environmental Biology, National Science Foundation, 4201 Wilson
      Boulevard, Suite 635, Arlington, Virginia 22230, USA. jrodman@nsf.gov
FAU - Cody, Jeannine H
AU  - Cody JH
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biology/*education
MH  - *Classification
MH  - *Training Support
MH  - United States
EDAT- 2003/05/31 05:00
MHDA- 2003/09/10 05:00
CRDT- 2003/05/31 05:00
PHST- 2003/05/31 05:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/05/31 05:00 [entrez]
AID - CQJP8BMPK7T5LEPD [pii]
AID - 10.1080/10635150390197055 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Jun;52(3):428-35. doi: 10.1080/10635150390197055.

PMID- 12775529
OWN - NLM
STAT- MEDLINE
DCOM- 20030909
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 3
DP  - 2003 Jun
TI  - Evaluation of the strategy of long-branch subdivision to improve the accuracy of 
      phylogenetic methods.
PG  - 423-8
FAU - Poe, Steven
AU  - Poe S
AD  - Department of Biology, University of New Mexico, Albuquerque, New Mexico 87131,
      USA. anolis@unm.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - *Data Interpretation, Statistical
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
MH  - *Sample Size
EDAT- 2003/05/31 05:00
MHDA- 2003/09/10 05:00
CRDT- 2003/05/31 05:00
PHST- 2003/05/31 05:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/05/31 05:00 [entrez]
AID - FQ2MXNCRUX5FNG3G [pii]
AID - 10.1080/10635150390197046 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Jun;52(3):423-8. doi: 10.1080/10635150390197046.

PMID- 12775528
OWN - NLM
STAT- MEDLINE
DCOM- 20030909
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 3
DP  - 2003 Jun
TI  - Combined support for wholesale taxic atavism in gavialine crocodylians.
PG  - 403-22
AB  - Morphological and molecular data sets favor robustly supported, contradictory
      interpretations of crocodylian phylogeny. A longstanding perception in the field 
      of systematics is that such significantly conflicting data sets should be
      analyzed separately. Here we utilize a combined approach, simultaneous analyses
      of all relevant character data, to summarize common support and to reconcile
      discrepancies among data sets. By conjoining rather than separating incongruent
      classes of data, secondary phylogenetic signals emerge from both molecular and
      morphological character sets and provide solid evidence for a unified hypothesis 
      of crocodylian phylogeny. Simultaneous analyses of four gene sequences and
      paleontological data suggest that putative adaptive convergences in the jaws of
      gavialines (gavials) and tomistomines (false gavials) offer character support for
      a grouping of these taxa, making Gavialinae an atavistic taxon. Simple new
      methods for measuring the influence of extinct taxa on topological support
      indicate that in this vertebrate order fossils generally stabilize relationships 
      and accentuate hidden phylogenetic signals. Remaining inconsistencies in minimum 
      length trees, including concentrated hierarchical patterns of homoplasy and
      extensive gaps in the fossil record, indicate where future work in crocodylian
      systematics should be directed.
FAU - Gatesy, John
AU  - Gatesy J
AD  - Department of Biology, University of California, Riverside, California 92521,
      USA. johnga@citrus.ucr.edu
FAU - Amato, George
AU  - Amato G
FAU - Norell, Mark
AU  - Norell M
FAU - DeSalle, Rob
AU  - DeSalle R
FAU - Hayashi, Cheryl
AU  - Hayashi C
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
EIN - Syst Biol. 2003 Oct;52(5):734
MH  - *Adaptation, Biological
MH  - Alligators and Crocodiles/anatomy &amp; histology/*classification/genetics
MH  - Animals
MH  - Base Sequence
MH  - Biological Evolution
MH  - DNA, Mitochondrial/genetics
MH  - Fossils
MH  - Genes, RAG-1/genetics
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Alignment/methods
EDAT- 2003/05/31 05:00
MHDA- 2003/09/10 05:00
CRDT- 2003/05/31 05:00
PHST- 2003/05/31 05:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/05/31 05:00 [entrez]
AID - P5Q9UBE68E7134Q6 [pii]
AID - 10.1080/10635150390197037 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Jun;52(3):403-22. doi: 10.1080/10635150390197037.

PMID- 12775527
OWN - NLM
STAT- MEDLINE
DCOM- 20030909
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 3
DP  - 2003 Jun
TI  - True and false gharials: a nuclear gene phylogeny of crocodylia.
PG  - 386-402
AB  - The phylogeny of Crocodylia offers an unusual twist on the usual molecules versus
      morphology story. The true gharial (Gavialis gangeticus) and the false gharial
      (Tomistoma schlegelii), as their common names imply, have appeared in all
      cladistic morphological analyses as distantly related species, convergent upon a 
      similar morphology. In contrast, all previous molecular studies have shown them
      to be sister taxa. We present the first phylogenetic study of Crocodylia using a 
      nuclear gene. We cloned and sequenced the c-myc proto-oncogene from Alligator
      mississippiensis to facilitate primer design and then sequenced an 1,100-base
      pair fragment that includes both coding and noncoding regions and informative
      indels for one species in each extant crocodylian genus and six avian outgroups. 
      Phylogenetic analyses using parsimony, maximum likelihood, and Bayesian inference
      all strongly agreed on the same tree, which is identical to the tree found in
      previous molecular analyses: Gavialis and Tomistoma are sister taxa and together 
      are the sister group of Crocodylidae. Kishino-Hasegawa tests rejected the
      morphological tree in favor of the molecular tree. We excluded long-branch
      attraction and variation in base composition among taxa as explanations for this 
      topology. To explore the causes of discrepancy between molecular and
      morphological estimates of crocodylian phylogeny, we examined puzzling features
      of the morphological data using a priori partitions of the data based on
      anatomical regions and investigated the effects of different coding schemes for
      two obvious morphological similarities of the two gharials.
FAU - Harshman, John
AU  - Harshman J
AD  - Department of Systematic Biology, National Museum of Natural History, Smithsonian
      Institution, Suitland, Maryland 20746, USA. jharshman@pacbell.net
FAU - Huddleston, Christopher J
AU  - Huddleston CJ
FAU - Bollback, Jonathan P
AU  - Bollback JP
FAU - Parsons, Thomas J
AU  - Parsons TJ
FAU - Braun, Michael J
AU  - Braun MJ
LA  - eng
SI  - GENBANK/AY277487
SI  - GENBANK/AY277488
SI  - GENBANK/AY277489
SI  - GENBANK/AY277490
SI  - GENBANK/AY277491
SI  - GENBANK/AY277492
SI  - GENBANK/AY277493
SI  - GENBANK/AY277494
SI  - GENBANK/AY277495
SI  - GENBANK/AY277496
SI  - GENBANK/AY277497
SI  - GENBANK/AY277498
SI  - GENBANK/AY277499
SI  - GENBANK/AY277500
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Alligators and Crocodiles/anatomy &amp; histology/*classification/genetics
MH  - Animals
MH  - Base Composition
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Data Interpretation, Statistical
MH  - Genes, myc/genetics
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - *Phylogeny
EDAT- 2003/05/31 05:00
MHDA- 2003/09/10 05:00
CRDT- 2003/05/31 05:00
PHST- 2003/05/31 05:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/05/31 05:00 [entrez]
AID - 7TVT4AGAPBK2ND57 [pii]
AID - 10.1080/10635150390197028 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Jun;52(3):386-402. doi: 10.1080/10635150390197028.

PMID- 12775526
OWN - NLM
STAT- MEDLINE
DCOM- 20030909
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 3
DP  - 2003 Jun
TI  - Ancient allopolyploid speciation in Geinae (Rosaceae): evidence from nuclear
      granule-bound starch synthase (GBSSI) gene sequences.
PG  - 374-85
AB  - A nuclear low-copy gene phylogeny provides strong evidence for the hybrid origin 
      of seven polyploid species in Geinae (Rosaceae). In a gene tree, alleles at
      homologous loci in an allopolyploid species are expected to be sisters to
      orthologues in the ancestral taxa rather than to each other. Alleles at a
      duplicated locus in an autopolyploid, however, are expected to be more closely
      related to each other than they are to any orthologous copies in closely related 
      species. We cloned and sequenced about 1.9 kilobases from the 5' end of the
      GBSSI-1 gene from two diploid, one tetraploid, and six hexaploid species. Each of
      the three loci in the hexaploid species forms a separate group, two of which are 
      more closely related to copies in other species than they are to each other. This
      finding indicates that the hexaploid lineage evolved through two consecutive
      allopolyploidization events. Based on the GBSSI-1 gene tree, we hypothesized that
      there was an initial hybridization between a diploid species from the ancestral
      lineage of Coluria and Waldsteinia and an unknown diploid species to form the
      tetraploid Geum heterocarpum lineage. Backcrossing of G. heterocarpum with a
      representative of the unknown diploid lineage then resulted in a hexaploid
      lineage that has radiated considerably since its origin, comprising at least 40
      extant species with various morphologies. A penalized likelihood analysis
      indicated that Geinae may be about 17 million years old, implying that the
      hypothesized allopolyploid speciation events are relatively ancient. Six of the
      22 cloned Geinae GBSSI-1 copies in this study, which all are duplicate copies in 
      polyploid taxa, may have become pseudogenes. We compared the GBSSI-1 phylogeny
      with one from chloroplast data and explored implications for the evolution of
      some fruit characters.
FAU - Smedmark, Jenny E E
AU  - Smedmark JE
AD  - Bergius Foundation, Royal Swedish Academy of Sciences, and Department of Botany, 
      Stockholm University, Sweden. jenny@bergianska.se
FAU - Eriksson, Torsten
AU  - Eriksson T
FAU - Evans, Rodger C
AU  - Evans RC
FAU - Campbell, Christopher S
AU  - Campbell CS
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 2.4.1.- (granule-bound starch synthase I)
RN  - EC 2.4.1.21 (Starch Synthase)
SB  - IM
MH  - Base Sequence
MH  - Bayes Theorem
MH  - *Evolution, Molecular
MH  - Geum/anatomy &amp; histology/classification/*genetics
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - *Polyploidy
MH  - Starch Synthase/*genetics
EDAT- 2003/05/31 05:00
MHDA- 2003/09/10 05:00
CRDT- 2003/05/31 05:00
PHST- 2003/05/31 05:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/05/31 05:00 [entrez]
AID - XC4GQ50J51FQ61J2 [pii]
AID - 10.1080/10635150390197000 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Jun;52(3):374-85. doi: 10.1080/10635150390197000.

PMID- 12775525
OWN - NLM
STAT- MEDLINE
DCOM- 20030909
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 3
DP  - 2003 Jun
TI  - Accelerated likelihood surface exploration: the likelihood ratchet.
PG  - 368-73
AB  - The existence of multiple likelihood maxima necessitates algorithms that explore 
      a large part of the tree space. However, because of computational constraints,
      stepwise addition-based tree-searching methods do not allow for this exploration 
      in reasonable time. Here, I present an algorithm that increases the speed at
      which the likelihood landscape can be explored. The iterative algorithm combines 
      the computational speed of distance-based tree construction methods to arrive at 
      approximations of the global optimum with the accuracy of optimality criterion
      based branch-swapping methods to improve on the result of the starting tree. The 
      algorithm moves between local optima by iteratively perturbing the tree landscape
      through a process of reweighting randomly drawn samples of the underlying
      sequence data set. Tests on simulated and real data sets demonstrated that the
      optimal solution obtained using stepwise addition-based heuristic searches was
      found faster using the algorithm presented here. Tests on a previously published 
      data set that established the presence of tree islands under maximum likelihood
      demonstrated that the algorithm identifies the same tree islands in a shorter
      amount of time than that needed using stepwise addition. The algorithm can be
      readily applied using standard software for phylogenetic inference.
FAU - Vos, R A
AU  - Vos RA
AD  - Department of Biological Sciences, Simon Fraser University, 8888 University
      Drive, Burnaby, British Columbia V5A 1S6, Canada. rvosa@sfu.ca
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Computer Simulation
MH  - *Likelihood Functions
MH  - *Phylogeny
EDAT- 2003/05/31 05:00
MHDA- 2003/09/10 05:00
CRDT- 2003/05/31 05:00
PHST- 2003/05/31 05:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/05/31 05:00 [entrez]
AID - QUVWKYQJRPPUX67H [pii]
AID - 10.1080/10635150390196993 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Jun;52(3):368-73. doi: 10.1080/10635150390196993.

PMID- 12775524
OWN - NLM
STAT- MEDLINE
DCOM- 20030909
LR  - 20030530
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 3
DP  - 2003 Jun
TI  - Failed refutations: further comments on parsimony and likelihood methods and
      their relationship to Popper's degree of corroboration.
PG  - 352-67
AB  - Kluge's (2001, Syst. Biol. 50:322-330) continued arguments that phylogenetic
      methods based on the statistical principle of likelihood are incompatible with
      the philosophy of science described by Karl Popper are based on false premises
      related to Kluge's misrepresentations of Popper's philosophy. Contrary to Kluge's
      conjectures, likelihood methods are not inherently verificationist; they do not
      treat every instance of a hypothesis as confirmation of that hypothesis. The
      historical nature of phylogeny does not preclude phylogenetic hypotheses from
      being evaluated using the probability of evidence. The low absolute probabilities
      of hypotheses are irrelevant to the correct interpretation of Popper's concept
      termed degree of corroboration, which is defined entirely in terms of relative
      probabilities. Popper did not advocate minimizing background knowledge; in any
      case, the background knowledge of both parsimony and likelihood methods consists 
      of the general assumption of descent with modification and additional assumptions
      that are deterministic, concerning which tree is considered most highly
      corroborated. Although parsimony methods do not assume (in the sense of
      entailing) that homoplasy is rare, they do assume (in the sense of requiring to
      obtain a correct phylogenetic inference) certain things about patterns of
      homoplasy. Both parsimony and likelihood methods assume (in the sense of implying
      by the manner in which they operate) various things about evolutionary processes,
      although violation of those assumptions does not always cause the methods to
      yield incorrect phylogenetic inferences. Test severity is increased by sampling
      additional relevant characters rather than by character reanalysis, although
      either interpretation is compatible with the use of phylogenetic likelihood
      methods. Neither parsimony nor likelihood methods assess test severity (critical 
      evidence) when used to identify a most highly corroborated tree(s) based on a
      single method or model and a single body of data; however, both classes of
      methods can be used to perform severe tests. The assumption of descent with
      modification is insufficient background knowledge to justify cladistic parsimony 
      as a method for assessing degree of corroboration. Invoking equivalency between
      parsimony methods and likelihood models that assume no common mechanism
      emphasizes the necessity of additional assumptions, at least some of which are
      probabilistic in nature. Incongruent characters do not qualify as falsifiers of
      phylogenetic hypotheses except under extremely unrealistic evolutionary models;
      therefore, justifications of parsimony methods as falsificationist based on the
      idea that they minimize the ad hoc dismissal of falsifiers are questionable.
      Probabilistic concepts such as degree of corroboration and likelihood provide a
      more appropriate framework for understanding how phylogenetics conforms with
      Popper's philosophy of science. Likelihood ratio tests do not assume what is at
      issue but instead are methods for testing hypotheses according to an accepted
      standard of statistical significance and for incorporating considerations about
      test severity. These tests are fundamentally similar to Popper's degree of
      corroboration in being based on the relationship between the probability of the
      evidence e in the presence versus absence of the hypothesis h, i.e., between
      p(e|hb) and p(e|b), where b is the background knowledge. Both parsimony and
      likelihood methods are inductive in that their inferences (particular trees)
      contain more information than (and therefore do not follow necessarily from) the 
      observations upon which they are based; however, both are deductive in that their
      conclusions (tree lengths and likelihoods) follow necessarily from their premises
      (particular trees, observed character state distributions, and evolutionary
      models). For these and other reasons, phylogenetic likelihood methods are highly 
      compatible with Karl Popper's philosophy of science and offer several advantages 
      over parsimony methods in this context.
FAU - de Queiroz, Kevin
AU  - de Queiroz K
AD  - Department of Systematic Biology, National Museum of Natural History, Smithsonian
      Institution, Washington, DC 20560-0162, USA. dequeiroz.kevin@nmnh.si.edu
FAU - Poe, Steven
AU  - Poe S
LA  - eng
PT  - Comment
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2001 Jun;50(3):322-30. PMID: 12116578
MH  - *Knowledge
MH  - *Likelihood Functions
MH  - *Philosophy
MH  - *Phylogeny
MH  - Probability Theory
EDAT- 2003/05/31 05:00
MHDA- 2003/09/10 05:00
CRDT- 2003/05/31 05:00
PHST- 2003/05/31 05:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/05/31 05:00 [entrez]
AID - 98X1CYFJK802TQVY [pii]
PST - ppublish
SO  - Syst Biol. 2003 Jun;52(3):352-67.

PMID- 12775523
OWN - NLM
STAT- MEDLINE
DCOM- 20030909
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 3
DP  - 2003 Jun
TI  - The systematic utility of floral and vegetative fragrance in two genera of
      nyctaginaceae.
PG  - 334-51
AB  - We examined relationships between fragrance and phylogeny using a number of
      approaches to coding fragrance data and comparing the hierarchical information in
      fragrance data with the phylogenetic signal in a DNA sequence data set. We first 
      used distance analyses to determine which coding method(s) best distinguishes
      species while grouping conspecifics. Results suggest that interspecific
      differences in fragrance composition were maximized by coding as presence/absence
      of fragrance compounds and biosynthetic pathways rather than when quantitative
      information was also included. Useful systematic information came from both
      compounds and pathways and from fragrance emitted by both floral and vegetative
      tissues. The coding methods that emerged from the distance analyses as best
      distinguishing species were then adapted for use in phylogenetic analysis.
      Although hierarchical signal among fragrance data sets was congruent, this signal
      was highly incongruent with the phylogenetic signal in the DNA sequence data.
      Notably, topologies inferred from fragrance data sets were congruent with the DNA
      topology only in the most distal portions (e.g., sister group pairs or closely
      related species that had similar fragrance profiles were often recovered by
      analyses of fragrance). Examination of consistency and retention indices for
      individual fragrance compounds and pathways as optimized onto one of the
      most-parsimonious trees inferred from DNA data revealed that although most
      compounds were homoplastic, some compounds were perfectly congruent with the DNA 
      phylogeny. In particular, compounds and pathways found in a few taxa were less
      homoplastic than those found in many taxa. Pathways that synthesize few volatiles
      also seem to have lower homoplasy than those that produce many. Although
      fragrance data as a whole may not be useful in phylogeny reconstruction, these
      data can provide additional support for clades reconstructed with other types of 
      characters. Factors other than phylogeny, including pollinator interactions, also
      likely influence fragrance composition.
FAU - Levin, Rachel A
AU  - Levin RA
AD  - Department of Ecology &amp; Evolutionary Biology, University of Arizona, Tucson,
      Arizona 85721, USA. rlevin@lab.si.edu
FAU - McDade, Lucinda A
AU  - McDade LA
FAU - Raguso, Robert A
AU  - Raguso RA
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
RN  - 0 (DNA, Ribosomal)
RN  - 0 (Oils, Volatile)
SB  - IM
MH  - Base Sequence
MH  - DNA, Chloroplast/genetics
MH  - DNA, Ribosomal/genetics
MH  - Molecular Sequence Data
MH  - Nyctaginaceae/*classification/genetics/metabolism
MH  - Odorants/analysis
MH  - Oils, Volatile/metabolism
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2003/05/31 05:00
MHDA- 2003/09/10 05:00
CRDT- 2003/05/31 05:00
PHST- 2003/05/31 05:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/05/31 05:00 [entrez]
AID - 0048514LXLA2UBEG [pii]
AID - 10.1080/10635150390196975 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Jun;52(3):334-51. doi: 10.1080/10635150390196975.

PMID- 12775522
OWN - NLM
STAT- MEDLINE
DCOM- 20030909
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 3
DP  - 2003 Jun
TI  - Phylogeny and evolution of calcareous sponges: monophyly of calcinea and
      calcaronea, high level of morphological homoplasy, and the primitive nature of
      axial symmetry.
PG  - 311-33
AB  - Because calcareous sponges are triggering renewed interest with respect to basal 
      metazoan evolution, a phylogenetic framework of their internal relationships is
      needed to clarify the evolutionary history of key morphological characters.
      Morphological variation was scored at the suprageneric level within Calcispongia,
      but little phylogenetic information could be retrieved from morphological
      characters. For the main subdivision of Calcispongia, the analysis of
      morphological data weakly supports a classification based upon cytological and
      embryological characters (Calcinea/Calcaronea) rather than the older
      classification scheme based upon the aquiferous system (Homocoela/Heterocoela).
      The 18S ribosomal RNA data were then analyzed, both alone and in combination with
      morphological characters. The monophyly of Calcispongia is highly supported, but 
      the position of this group with respect to other sponge lineages and to
      eumetazoan taxa is not resolved. The monophyly of both Calcinea and Calcaronea is
      retrieved, and the data strongly rejected the competing Homocoela/Heterocoela
      hypothesis. The phylogeny implies that characters of the skeleton architecture
      are highly homoplastic, as are characters of the aquiferous system. However,
      axial symmetry seems to be primitive for all Calcispongia, a conclusion that has 
      potentially far-reaching implications for hypotheses of early body plan evolution
      in Metazoa.
FAU - Manuel, Michael
AU  - Manuel M
AD  - Centre d'Oceanologie de Marseille, Station Marine d'Endoume, Universite de la
      Mediterranee, UMR-CNRS 6540 DIMAR, rue de la Batterie des Lions, 13007 Marseille,
      France. michael.manuel@snv.jussieu.fr
FAU - Borchiellini, Carole
AU  - Borchiellini C
FAU - Alivon, Eliane
AU  - Alivon E
FAU - Le Parco, Yannick
AU  - Le Parco Y
FAU - Vacelet, Jean
AU  - Vacelet J
FAU - Boury-Esnault, Nicole
AU  - Boury-Esnault N
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 18S)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Porifera/anatomy &amp; histology/*classification
MH  - RNA, Ribosomal, 18S/*genetics
MH  - Sequence Alignment
EDAT- 2003/05/31 05:00
MHDA- 2003/09/10 05:00
CRDT- 2003/05/31 05:00
PHST- 2003/05/31 05:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/05/31 05:00 [entrez]
AID - 51N8LGYEHDLBXQRV [pii]
AID - 10.1080/10635150390196966 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Jun;52(3):311-33. doi: 10.1080/10635150390196966.

PMID- 12775521
OWN - NLM
STAT- MEDLINE
DCOM- 20030909
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 3
DP  - 2003 Jun
TI  - The value of idiosyncratic markers and changes to conserved tRNA sequences from
      the mitochondrial genome of hard ticks (Acari: Ixodida: Ixodidae) for
      phylogenetic inference.
PG  - 296-310
AB  - Idiosyncratic markers are features of genes and genomes that are so unusual that 
      it is unlikely that they evolved more than once in a lineage of organisms. Here
      we explore further the potential of idiosyncratic markers and changes to
      typically conserved tRNA sequences for phylogenetic inference. Hard ticks were
      chosen as the model group because their phylogeny has been studied extensively.
      Fifty-eight candidate markers from hard ticks (family Ixodidae) and 22 markers
      from the subfamily Rhipicephalinae sensu lato were mapped onto phylogenies of
      these groups. Two of the most interesting markers, features of the secondary
      structure of two different tRNAs, gave strong support to the hypothesis that
      species of the Prostriata (Ixodes spp.) are monophyletic. Previous analyses of
      genes and morphology did not strongly support this relationship, instead
      suggesting that the Prostriata is paraphyletic with respect to the Metastriata
      (the rest of the hard ticks). Parallel or convergent evolution was not found in
      the arrangements of mitochondrial genes in ticks nor were there any reversals to 
      the ancestral arthropod character state. Many of the markers identified were
      phylogenetically informative, whereas others should be informative with study of 
      additional taxa. Idiosyncratic markers and changes to typically conserved
      nucleotides in tRNAs that are phylogenetically informative were common in this
      data set, and thus these types of markers might be found in other organisms.
FAU - Murrell, Anna
AU  - Murrell A
AD  - Department of Microbiology and Parasitology, Institute for Molecular Biosciences,
      University of Queensland, Brisbane, Queensland 4072, Australia.
      a.murrell@uq.edu.au
FAU - Campbell, Nick J H
AU  - Campbell NJ
FAU - Barker, Stephen C
AU  - Barker SC
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Genetic Markers)
RN  - 9014-25-9 (RNA, Transfer)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Conserved Sequence/*genetics
MH  - DNA, Mitochondrial/genetics
MH  - Gene Duplication
MH  - Genetic Markers/*genetics
MH  - Ixodidae/*genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Point Mutation
MH  - RNA, Transfer/*genetics
EDAT- 2003/05/31 05:00
MHDA- 2003/09/10 05:00
CRDT- 2003/05/31 05:00
PHST- 2003/05/31 05:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/05/31 05:00 [entrez]
AID - AR0Q6BYPQYEER4TH [pii]
AID - 10.1080/10635150390196957 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Jun;52(3):296-310. doi: 10.1080/10635150390196957.

PMID- 12775520
OWN - NLM
STAT- MEDLINE
DCOM- 20030909
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 3
DP  - 2003 Jun
TI  - 18S ribosomal RNA and tetrapod phylogeny.
PG  - 283-95
AB  - Previous phylogenetic analyses of tetrapod 18S ribosomal RNA (rRNA) sequences
      support the grouping of birds with mammals, whereas other molecular data, and
      morphological and paleontological data favor the grouping of birds with
      crocodiles. The 18S rRNA gene has consequently been considered odd, serving as
      "definitive evidence of different genes providing significantly different
      estimates of phylogeny in higher organisms" (p. 156; Huelsenbeck et al., 1996,
      Trends Ecol. Evol. 11:152-158). Our research indicates that the previous
      discrepancy of phylogenetic results between the 18S rRNA gene and other genes is 
      caused mainly by (1) the misalignment of the sequences, (2) the inappropriate use
      of the frequency parameters, and (3) poor sequence quality. When the sequences
      are aligned with the aide of the secondary structure of the 18S rRNA molecule and
      when the frequency parameters are estimated either from all sites or from the
      variable domains where substitutions have occurred, the 18S rRNA sequences no
      longer support the grouping of the avian species with the mammalian species.
FAU - Xia, Xuhua
AU  - Xia X
AD  - Department of Biology, University of Ottawa, 150 Louis, P.O. Box 450, Station A, 
      Ottawa, Ontario K1N 6N5, Canada. xxia@uottawa.ca
FAU - Xie, Zheng
AU  - Xie Z
FAU - Kjer, Karl M
AU  - Kjer KM
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 18S)
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Birds/*classification
MH  - Genetic Variation/genetics
MH  - Mammals/*classification
MH  - *Phylogeny
MH  - RNA, Ribosomal, 18S/*genetics
MH  - Reptiles/*classification
MH  - Sequence Alignment
EDAT- 2003/05/31 05:00
MHDA- 2003/09/10 05:00
CRDT- 2003/05/31 05:00
PHST- 2003/05/31 05:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/05/31 05:00 [entrez]
AID - UGUV9WAQP64AXWHB [pii]
AID - 10.1080/10635150390196948 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Jun;52(3):283-95. doi: 10.1080/10635150390196948.

PMID- 12746153
OWN - NLM
STAT- MEDLINE
DCOM- 20030915
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 2
DP  - 2003 Apr
TI  - Using a null model to recognize significant co-occurrence prior to identifying
      candidate areas of endemism.
PG  - 271-80
FAU - Mast, Austin R
AU  - Mast AR
AD  - Institute of Systematic Botany, University of Zurich, Zollikerstrasse 107, 8008
      Zurich, Switzerland. amast@access.unizh.ch
FAU - Nyffeler, Reto
AU  - Nyffeler R
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Africa, Southern
MH  - Algorithms
MH  - Animals
MH  - Computer Simulation
MH  - *Demography
MH  - Geography
MH  - Insecta
MH  - *Models, Statistical
EDAT- 2003/05/15 05:00
MHDA- 2003/09/16 05:00
CRDT- 2003/05/15 05:00
PHST- 2003/05/15 05:00 [pubmed]
PHST- 2003/09/16 05:00 [medline]
PHST- 2003/05/15 05:00 [entrez]
AID - 7UXGH5XCW6JTKDKQ [pii]
AID - 10.1080/10635150390192799 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Apr;52(2):271-80. doi: 10.1080/10635150390192799.

PMID- 12746152
OWN - NLM
STAT- MEDLINE
DCOM- 20030915
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 2
DP  - 2003 Apr
TI  - Popper and systematics.
PG  - 259-71
FAU - Rieppel, Olivier
AU  - Rieppel O
AD  - Department of Geology, Field Museum, 1400 S. Lake Shore Drive, Chicago, Illinois 
      60605-2496, USA. rieppel@fieldmuseum.org
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2003 Aug;52(4):569
MH  - Knowledge
MH  - *Logic
MH  - Models, Theoretical
MH  - *Phylogeny
MH  - Probability Theory
MH  - Semantics
EDAT- 2003/05/15 05:00
MHDA- 2003/09/16 05:00
CRDT- 2003/05/15 05:00
PHST- 2003/05/15 05:00 [pubmed]
PHST- 2003/09/16 05:00 [medline]
PHST- 2003/05/15 05:00 [entrez]
AID - JC3K93HPQPVFTADV [pii]
AID - 10.1080/10635150390192762 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Apr;52(2):259-71. doi: 10.1080/10635150390192762.

PMID- 12746151
OWN - NLM
STAT- MEDLINE
DCOM- 20030915
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 2
DP  - 2003 Apr
TI  - Component coding, three-item coding, and consensus methods.
PG  - 255-9
FAU - Williams, David M
AU  - Williams DM
AD  - Department of Botany, Natural History Museum, Cromwell Road, London SW7 5BD, UK. 
      dmw@nhm.ac.uk
FAU - Humphries, Christopher J
AU  - Humphries CJ
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - *Data Interpretation, Statistical
EDAT- 2003/05/15 05:00
MHDA- 2003/09/16 05:00
CRDT- 2003/05/15 05:00
PHST- 2003/05/15 05:00 [pubmed]
PHST- 2003/09/16 05:00 [medline]
PHST- 2003/05/15 05:00 [entrez]
AID - 9YVRER18RBG3TMFH [pii]
AID - 10.1080/10635150390192753 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Apr;52(2):255-9. doi: 10.1080/10635150390192753.

PMID- 12746150
OWN - NLM
STAT- MEDLINE
DCOM- 20030915
LR  - 20040220
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 2
DP  - 2003 Apr
TI  - Clarifying aetosaur phylogeny requires more fossils, not more trees--reply to
      intraorganismal homology, character construction, and the phylogeny of
      aetosaurian archosaurs (reptilia, diapsida).
PG  - 253-5
FAU - Heckert, Andrew B
AU  - Heckert AB
AD  - New Mexico Museum of Natural History and Science, 1801 Mountain Road NW,
      Albuquerque, New Mexico 87104-1375, USA. aheckert@nmmnh.state.nm.us
FAU - Lucas, Spencer G
AU  - Lucas SG
LA  - eng
PT  - Comment
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2003 Apr;52(2):239-52. PMID: 12746149
CIN - Syst Biol. 2003 Dec;52(6):851-2. PMID: 14668122
MH  - Alligators and Crocodiles/*anatomy &amp; histology/*classification
MH  - Animals
MH  - Fossils
MH  - Research Design
EDAT- 2003/05/15 05:00
MHDA- 2003/09/16 05:00
CRDT- 2003/05/15 05:00
PHST- 2003/05/15 05:00 [pubmed]
PHST- 2003/09/16 05:00 [medline]
PHST- 2003/05/15 05:00 [entrez]
AID - RATBJGV3MUJT503D [pii]
PST - ppublish
SO  - Syst Biol. 2003 Apr;52(2):253-5.

PMID- 12746149
OWN - NLM
STAT- MEDLINE
DCOM- 20030915
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 2
DP  - 2003 Apr
TI  - Intraorganismal homology, character construction, and the phylogeny of
      aetosaurian archosaurs (reptilia, diapsida).
PG  - 239-52
AB  - Character construction, the methods by which characters and character states are 
      produced from observations of variation, is a crucial but poorly understood step 
      in phylogenetic analysis. Alternative approaches are used in practice, but there 
      has been relatively little investigation of their theoretical bases and
      analytical consequences. We reviewed three published numerical analyses of the
      phylogenetic relationships within the Triassic Aetosauria. Combined data from
      these studies were used to explore the impact of alternative approaches to
      character construction. Some previous aetosaurian characters represent parallel
      variations in the morphology of osteoderms from different body regions, and their
      independence is questionable, leading us to propose more composite alternative
      constructions. Phylogenetic analyses revealed that inferred relationships within 
      the Aetosauria are in general poorly resolved and weakly supported by the
      available data and are sensitive to alternative approaches to character
      construction. Thus, the results from this and previous studies should not, for
      the most part, be accepted as robust hypotheses of aetosaurian
      interrelationships. The treatment of systems of intraorganismal (e.g. serial,
      antimeric) homologues, such as osteoderms, in character construction is
      discussed. Applied to parallel variations in systems of intraorganismal
      homologues, previous advice on choosing among alternative character constructions
      and Hennig's auxiliary principle agree in favoring a more composite approach, in 
      accordance with common practice.
FAU - Harris, Simon R
AU  - Harris SR
AD  - Department of Zoology, Natural History Museum, Cromwell Road, London SW7 5BD, UK.
FAU - Gower, David J
AU  - Gower DJ
FAU - Wilkinson, Mark
AU  - Wilkinson M
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2003 Apr;52(2):253-5. PMID: 12746150
MH  - Alligators and Crocodiles/*anatomy &amp; histology/*classification
MH  - Animals
MH  - Body Weights and Measures
MH  - Fossils
MH  - *Phylogeny
MH  - Research Design
EDAT- 2003/05/15 05:00
MHDA- 2003/09/16 05:00
CRDT- 2003/05/15 05:00
PHST- 2003/05/15 05:00 [pubmed]
PHST- 2003/09/16 05:00 [medline]
PHST- 2003/05/15 05:00 [entrez]
AID - UHT68RP0VBXC3JE3 [pii]
AID - 10.1080/10635150309341 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Apr;52(2):239-52. doi: 10.1080/10635150309341.

PMID- 12746148
OWN - NLM
STAT- MEDLINE
DCOM- 20030915
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 2
DP  - 2003 Apr
TI  - Outgroup misplacement and phylogenetic inaccuracy under a molecular clock--a
      simulation study.
PG  - 229-38
AB  - We conducted a simulation study of the phylogenetic methods UPGMA, neighbor
      joining, maximum parsimony, and maximum likelihood for a five-taxon tree under a 
      molecular clock. The parameter space included a small region where maximum
      parsimony is inconsistent, so we tested inconsistency correction for parsimony
      and distance correction for neighbor joining. As expected, corrected parsimony
      was consistent. For these data, maximum likelihood with the clock assumption
      outperformed each of the other methods tested. The distance-based methods
      performed marginally better than did maximum parsimony and maximum likelihood
      without the clock assumption. Data correction was generally detrimental to
      accuracy, especially for short sequence lengths. We identified another region of 
      the parameter space where, although consistent for a given method, some incorrect
      trees were each selected with up to twice the frequency of the correct
      (generating) tree for sequences of bounded length. These incorrect trees are
      those where the outgroup has been incorrectly placed. In addition to this
      problem, the placement of the outgroup sequence can have a confounding effect on 
      the ingroup tree, whereby the ingroup is correct when using the ingroup sequences
      alone, but with the inclusion of the outgroup the ingroup tree becomes incorrect.
FAU - Holland, B R
AU  - Holland BR
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, Massey University,
      Palmerston North, New Zealand. b.r.holland@massey.ac.nz
FAU - Penny, D
AU  - Penny D
FAU - Hendy, M D
AU  - Hendy MD
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Cluster Analysis
MH  - *Computer Simulation
MH  - Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Research Design
EDAT- 2003/05/15 05:00
MHDA- 2003/09/16 05:00
CRDT- 2003/05/15 05:00
PHST- 2003/05/15 05:00 [pubmed]
PHST- 2003/09/16 05:00 [medline]
PHST- 2003/05/15 05:00 [entrez]
AID - 5NGVC7XN4CAQNVAR [pii]
AID - 10.1080/10635150390192771 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Apr;52(2):229-38. doi: 10.1080/10635150390192771.

PMID- 12746147
OWN - NLM
STAT- MEDLINE
DCOM- 20030915
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 2
DP  - 2003 Apr
TI  - Molecular and morphological phylogenies of ruminantia and the alternative
      position of the moschidae.
PG  - 206-28
AB  - The ruminants constitute the largest group of ungulates, with &gt;190 species, and
      its distribution is widespread throughout all continents except Australia and
      Antarctica. Six families are traditionally recognized within the suborder
      Ruminantia: Antilocapridae (pronghorns), Bovidae (cattle, sheep, and antelopes), 
      Cervidae (deer), Giraffidae (giraffes and okapis), Moschidae (musk deer), and
      Tragulidae (chevrotains). The interrelationships of the families have been an
      area of controversy among morphology, palaeontology, and molecular studies, and
      almost all possible evolutionary scenarios have been proposed in the literature. 
      We analyzed a large DNA data set (5,322 nucleotides) for 23 species including
      both mitochondrial (cytochrome b, 12S ribosomal RNA (rRNA), and 16S rRNA) and
      nuclear (kappa-casein, cytochrome P-450, lactoferrin, and alpha-lactalbumin)
      markers. Our results show that the family Tragulidae occupies a basal position
      with respect to all other ruminant families, confirming the traditional view that
      separates Tragulina and Pecora. Within the pecorans, Antilocapridae and
      Giraffidae emerge first, and the families Bovidae, Moschidae, and Cervidae are
      allied, with the unexpected placement of Moschus close to bovids rather than to
      cervids. We used these molecular results to assess the homoplastic evolution of
      morphological characters within the Ruminantia. A Bayesian relaxed molecular
      clock approach based on the continuous autocorrelation of evolutionary rates
      along branches was applied to estimate the divergence ages between the major
      clades of ruminants. The evolutionary radiation of Pecora occurred at the
      Early/Late Oligocene transition, and Pecoran families diversified and dispersed
      rapidly during the Early and Middle Miocene. We propose a biogeographic scenario 
      to explain the extraordinary expansion of this group during the Cenozoic era.
FAU - Hassanin, Alexandre
AU  - Hassanin A
AD  - Laboratoire de Paleontologie, Paleobiologie et Phylogenie, Institut des Sciences 
      de l'Evolution UMR 5554 CNRS, Universite Montpellier II, CC064, Place E.
      Bataillon, 34 095 Montpellier Cedex 05, France.
FAU - Douzery, Emmanuel J P
AU  - Douzery EJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Fossils
MH  - Geography
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Ruminants/anatomy &amp; histology/*classification/genetics
EDAT- 2003/05/15 05:00
MHDA- 2003/09/16 05:00
CRDT- 2003/05/15 05:00
PHST- 2003/05/15 05:00 [pubmed]
PHST- 2003/09/16 05:00 [medline]
PHST- 2003/05/15 05:00 [entrez]
AID - Y0C5477XY6063CAF [pii]
AID - 10.1080/10635150390192726 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Apr;52(2):206-28. doi: 10.1080/10635150390192726.

PMID- 12746146
OWN - NLM
STAT- MEDLINE
DCOM- 20030915
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 2
DP  - 2003 Apr
TI  - The geometry of the marmot (rodentia: sciuridae) mandible: phylogeny and patterns
      of morphological evolution.
PG  - 186-205
AB  - Marmots have a prominent role in the study of mammalian social evolution, but
      only recently has their systematics received the attention it deserves if
      sociobiological studies are to be placed in a phylogenetic context. Sciurid
      morphology can be used as model to test the congruence between morphological
      change and phylogeny because sciurid skeletal characters are considered to be
      inclined to convergence. However, no morphological study involving all marmot
      species has ever been undertaken. Geometric morphometric techniques were applied 
      in a comparative study of the marmot mandible. The adults of all 14 living marmot
      species were compared, and mean mandible shape were used to investigate
      morphological evolution in the genus Marmota. Three major trends were observed.
      First, the phylogenetic signal in the variation of landmark geometry, which
      describes mandible morphology, seems to account for the shape differences at
      intermediate taxonomic levels. The subgenera Marmota and Petromarmota, recently
      proposed on the basis of mitochondrial cytochrome b sequence, receive support
      from mandible morphology. When other sciurid genera were included in the
      analysis, the monophyly of the genus Marmota and that of the tribe Marmotini
      (i.e., marmots, prairie dogs, and ground squirrels) was strengthened by the
      morphological data. Second, the marmotine mandible may have evolved as a mosaic
      of characters and does not show convergence determined by size similarities.
      Third, allopatric speciation in peripheral isolates may have acted as a powerful 
      force for modeling shape. This hypothesis is strongly supported by the peculiar
      mandible of M. vancouverensis and, to a lesser degree, by that of M. olympus,
      both thought to have originated as isolated populations in Pleistocene ice-free
      refugia.
FAU - Cardini, Andrea
AU  - Cardini A
AD  - Division of Mammals, Department of Vertebrate Zoology, NHB 390, MRC 108, National
      Museum of Natural History, Smithsonian Institution, P.O. Box 37012, Washington,
      DC 20013-7012, USA. cardini@nmnh.si.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Biological
MH  - Analysis of Variance
MH  - Animals
MH  - Body Weights and Measures
MH  - Mandible/*anatomy &amp; histology
MH  - Marmota/*anatomy &amp; histology/classification
MH  - Models, Biological
MH  - *Phylogeny
MH  - Regression Analysis
MH  - Sex Characteristics
EDAT- 2003/05/15 05:00
MHDA- 2003/09/16 05:00
CRDT- 2003/05/15 05:00
PHST- 2003/05/15 05:00 [pubmed]
PHST- 2003/09/16 05:00 [medline]
PHST- 2003/05/15 05:00 [entrez]
AID - RNHLWK7060H0C1TB [pii]
AID - 10.1080/10635150390192807 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Apr;52(2):186-205. doi: 10.1080/10635150390192807.

PMID- 12746145
OWN - NLM
STAT- MEDLINE
DCOM- 20030915
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 2
DP  - 2003 Apr
TI  - Sampling strategies for delimiting species: genes, individuals, and populations
      in the Liolaemus elongatus-kriegi complex (Squamata: Liolaemidae) in
      Andean-Patagonian South America.
PG  - 159-85
AB  - Recovery of evolutionary history and delimiting species boundaries in widely
      distributed, poorly known groups requires extensive geographic sampling, but
      sampling regimes are difficult to design a priori because evolutionary diversity 
      is often "hidden" by inadequate taxonomy. Large data sets are needed, and these
      provide unique challenges for analysis when they span intra- and interspecific
      levels of divergence. However, protocols have been designed to combine methods of
      analysis for DNA sequences that exhibit both very shallow and relatively deeper
      divergences. In this study, we combined several tree-based phylogeny
      reconstruction methods with nested-clade analysis to extract maximum historical
      signal at various levels in the poorly known Liolaemus elongatus-kriegi lizard
      complex in temperate South America. We implemented a recently descrirbed
      tree-based protocol for DNA sequences to test for species boundaries, and we
      propose modifications to accommodate large data sets and gene regions with
      heterogeneous substitution rates. Combining haplotype trees with nested-clade
      analyses allowed testing of species boundaries on the basis of a priori defined
      criteria. The results obtained suggest that the number of putative species in the
      L. elongatus-kriegi complex could be doubled. We discuss these findings in the
      context of the advantages and limitations of a combined approach for retrieval of
      maximum historical information in large data sets and with reference to the yet
      formidable unresolved issues of sampling strategies.
FAU - Morando, Mariana
AU  - Morando M
AD  - Department of Integrative Biology and M. L. Bean Life Science Museum, Brigham
      Young University, Provo, Utah 84602, USA. mariana@email.byu.edu
FAU - Avila, Luciano J
AU  - Avila LJ
FAU - Sites, Jack W Jr
AU  - Sites JW Jr
LA  - eng
SI  - GENBANK/AY173521
SI  - GENBANK/AY173522
SI  - GENBANK/AY173523
SI  - GENBANK/AY173524
SI  - GENBANK/AY173525
SI  - GENBANK/AY173526
SI  - GENBANK/AY173527
SI  - GENBANK/AY173528
SI  - GENBANK/AY173529
SI  - GENBANK/AY173530
SI  - GENBANK/AY173531
SI  - GENBANK/AY173532
SI  - GENBANK/AY173533
SI  - GENBANK/AY173534
SI  - GENBANK/AY173535
SI  - GENBANK/AY173536
SI  - GENBANK/AY173537
SI  - GENBANK/AY173538
SI  - GENBANK/AY173539
SI  - GENBANK/AY173540
SI  - GENBANK/AY173541
SI  - GENBANK/AY173542
SI  - GENBANK/AY173543
SI  - GENBANK/AY173544
SI  - GENBANK/AY173545
SI  - GENBANK/AY173546
SI  - GENBANK/AY173547
SI  - GENBANK/AY173548
SI  - GENBANK/AY173549
SI  - GENBANK/AY173550
SI  - GENBANK/AY173551
SI  - GENBANK/AY173552
SI  - GENBANK/AY173553
SI  - GENBANK/AY173554
SI  - GENBANK/AY173555
SI  - GENBANK/AY173556
SI  - GENBANK/AY173557
SI  - GENBANK/AY173558
SI  - GENBANK/AY173559
SI  - GENBANK/AY173560
SI  - GENBANK/AY173561
SI  - GENBANK/AY173562
SI  - GENBANK/AY173563
SI  - GENBANK/AY173564
SI  - GENBANK/AY173565
SI  - GENBANK/AY173566
SI  - GENBANK/AY173567
SI  - GENBANK/AY173568
SI  - GENBANK/AY173569
SI  - GENBANK/AY173570
SI  - GENBANK/AY173571
SI  - GENBANK/AY173572
SI  - GENBANK/AY173573
SI  - GENBANK/AY173574
SI  - GENBANK/AY173575
SI  - GENBANK/AY173576
SI  - GENBANK/AY173577
SI  - GENBANK/AY173578
SI  - GENBANK/AY173579
SI  - GENBANK/AY173580
SI  - GENBANK/AY173581
SI  - GENBANK/AY173582
SI  - GENBANK/AY173583
SI  - GENBANK/AY173584
SI  - GENBANK/AY173585
SI  - GENBANK/AY173586
SI  - GENBANK/AY173587
SI  - GENBANK/AY173588
SI  - GENBANK/AY173589
SI  - GENBANK/AY173590
SI  - GENBANK/AY173591
SI  - GENBANK/AY173592
SI  - GENBANK/AY173593
SI  - GENBANK/AY173594
SI  - GENBANK/AY173595
SI  - GENBANK/AY173596
SI  - GENBANK/AY173597
SI  - GENBANK/AY173598
SI  - GENBANK/AY173599
SI  - GENBANK/AY173600
SI  - GENBANK/AY173601
SI  - GENBANK/AY173602
SI  - GENBANK/AY173603
SI  - GENBANK/AY173604
SI  - GENBANK/AY173605
SI  - GENBANK/AY173606
SI  - GENBANK/AY173607
SI  - GENBANK/AY173608
SI  - GENBANK/AY173609
SI  - GENBANK/AY173610
SI  - GENBANK/AY173611
SI  - GENBANK/AY173612
SI  - GENBANK/AY173613
SI  - GENBANK/AY173614
SI  - GENBANK/AY173615
SI  - GENBANK/AY173616
SI  - GENBANK/AY173617
SI  - GENBANK/AY173618
SI  - GENBANK/AY173619
SI  - GENBANK/AY173620
SI  - GENBANK/AY173621
SI  - GENBANK/AY173622
SI  - GENBANK/AY173623
SI  - GENBANK/AY173624
SI  - GENBANK/AY173625
SI  - GENBANK/AY173626
SI  - GENBANK/AY173627
SI  - GENBANK/AY173628
SI  - GENBANK/AY173629
SI  - GENBANK/AY173630
SI  - GENBANK/AY173631
SI  - GENBANK/AY173632
SI  - GENBANK/AY173633
SI  - GENBANK/AY173634
SI  - GENBANK/AY173635
SI  - GENBANK/AY173636
SI  - GENBANK/AY173637
SI  - GENBANK/AY173638
SI  - GENBANK/AY173639
SI  - GENBANK/AY173640
SI  - GENBANK/AY173641
SI  - GENBANK/AY173642
SI  - GENBANK/AY173643
SI  - GENBANK/AY173644
SI  - GENBANK/AY173645
SI  - GENBANK/AY173646
SI  - GENBANK/AY173647
SI  - GENBANK/AY173648
SI  - GENBANK/AY173649
SI  - GENBANK/AY173650
SI  - GENBANK/AY173651
SI  - GENBANK/AY173652
SI  - GENBANK/AY173653
SI  - GENBANK/AY173654
SI  - GENBANK/AY173655
SI  - GENBANK/AY173656
SI  - GENBANK/AY173657
SI  - GENBANK/AY173658
SI  - GENBANK/AY173659
SI  - GENBANK/AY173660
SI  - GENBANK/AY173661
SI  - GENBANK/AY173662
SI  - GENBANK/AY173663
SI  - GENBANK/AY173664
SI  - GENBANK/AY173665
SI  - GENBANK/AY173666
SI  - GENBANK/AY173667
SI  - GENBANK/AY173668
SI  - GENBANK/AY173669
SI  - GENBANK/AY173670
SI  - GENBANK/AY173671
SI  - GENBANK/AY173672
SI  - GENBANK/AY173673
SI  - GENBANK/AY173674
SI  - GENBANK/AY173675
SI  - GENBANK/AY173676
SI  - GENBANK/AY173677
SI  - GENBANK/AY173678
SI  - GENBANK/AY173679
SI  - GENBANK/AY173680
SI  - GENBANK/AY173681
SI  - GENBANK/AY173682
SI  - GENBANK/AY173683
SI  - GENBANK/AY173684
SI  - GENBANK/AY173685
SI  - GENBANK/AY173686
SI  - GENBANK/AY173687
SI  - GENBANK/AY173688
SI  - GENBANK/AY173689
SI  - GENBANK/AY173690
SI  - GENBANK/AY173691
SI  - GENBANK/AY173692
SI  - GENBANK/AY173693
SI  - GENBANK/AY173694
SI  - GENBANK/AY173695
SI  - GENBANK/AY173696
SI  - GENBANK/AY173697
SI  - GENBANK/AY173698
SI  - GENBANK/AY173699
SI  - GENBANK/AY173700
SI  - GENBANK/AY173701
SI  - GENBANK/AY173702
SI  - GENBANK/AY173703
SI  - GENBANK/AY173704
SI  - GENBANK/AY173705
SI  - GENBANK/AY173706
SI  - GENBANK/AY173707
SI  - GENBANK/AY173708
SI  - GENBANK/AY173709
SI  - GENBANK/AY173710
SI  - GENBANK/AY173711
SI  - GENBANK/AY173712
SI  - GENBANK/AY173713
SI  - GENBANK/AY173714
SI  - GENBANK/AY173715
SI  - GENBANK/AY173716
SI  - GENBANK/AY173717
SI  - GENBANK/AY173718
SI  - GENBANK/AY173719
SI  - GENBANK/AY173720
SI  - GENBANK/AY173721
SI  - GENBANK/AY173722
SI  - GENBANK/AY173723
SI  - GENBANK/AY173724
SI  - GENBANK/AY173725
SI  - GENBANK/AY173726
SI  - GENBANK/AY173727
SI  - GENBANK/AY173728
SI  - GENBANK/AY173729
SI  - GENBANK/AY173730
SI  - GENBANK/AY173731
SI  - GENBANK/AY173732
SI  - GENBANK/AY173733
SI  - GENBANK/AY173734
SI  - GENBANK/AY173735
SI  - GENBANK/AY173736
SI  - GENBANK/AY173737
SI  - GENBANK/AY173738
SI  - GENBANK/AY173739
SI  - GENBANK/AY173740
SI  - GENBANK/AY173741
SI  - GENBANK/AY173742
SI  - GENBANK/AY173743
SI  - GENBANK/AY173744
SI  - GENBANK/AY173745
SI  - GENBANK/AY173746
SI  - GENBANK/AY173747
SI  - GENBANK/AY173748
SI  - GENBANK/AY173749
SI  - GENBANK/AY173750
SI  - GENBANK/AY173751
SI  - GENBANK/AY173752
SI  - GENBANK/AY173753
SI  - GENBANK/AY173754
SI  - GENBANK/AY173755
SI  - GENBANK/AY173756
SI  - GENBANK/AY173757
SI  - GENBANK/AY173758
SI  - GENBANK/AY173759
SI  - GENBANK/AY173760
SI  - GENBANK/AY173761
SI  - GENBANK/AY173762
SI  - GENBANK/AY173763
SI  - GENBANK/AY173764
SI  - GENBANK/AY173765
SI  - GENBANK/AY173766
SI  - GENBANK/AY173767
SI  - GENBANK/AY173768
SI  - GENBANK/AY173769
SI  - GENBANK/AY173770
SI  - GENBANK/AY173771
SI  - GENBANK/AY173772
SI  - GENBANK/AY173773
SI  - GENBANK/AY173774
SI  - GENBANK/AY173775
SI  - GENBANK/AY173776
SI  - GENBANK/AY173777
SI  - GENBANK/AY173778
SI  - GENBANK/AY173779
SI  - GENBANK/AY173780
SI  - GENBANK/AY173781
SI  - GENBANK/AY173782
SI  - GENBANK/AY173783
SI  - GENBANK/AY173784
SI  - GENBANK/AY173785
SI  - GENBANK/AY173786
SI  - GENBANK/AY173787
SI  - GENBANK/AY173788
SI  - GENBANK/AY173789
SI  - GENBANK/AY173790
SI  - GENBANK/AY173791
SI  - GENBANK/AY173792
SI  - GENBANK/AY173793
SI  - GENBANK/AY173794
SI  - GENBANK/AY173795
SI  - GENBANK/AY173796
SI  - GENBANK/AY173797
SI  - GENBANK/AY173798
SI  - GENBANK/AY173799
SI  - GENBANK/AY173800
SI  - GENBANK/AY173801
SI  - GENBANK/AY173802
SI  - GENBANK/AY173803
SI  - GENBANK/AY173804
SI  - GENBANK/AY173805
SI  - GENBANK/AY173806
SI  - GENBANK/AY173807
SI  - GENBANK/AY173808
SI  - GENBANK/AY173809
SI  - GENBANK/AY173810
SI  - GENBANK/AY173811
SI  - GENBANK/AY173812
SI  - GENBANK/AY173813
SI  - GENBANK/AY173814
SI  - GENBANK/AY173815
SI  - GENBANK/AY173816
SI  - GENBANK/AY173817
SI  - GENBANK/AY173818
SI  - GENBANK/AY173819
SI  - GENBANK/AY173820
SI  - GENBANK/AY173821
SI  - GENBANK/AY173822
SI  - GENBANK/AY173823
SI  - GENBANK/AY173824
SI  - GENBANK/AY173825
SI  - GENBANK/AY173826
SI  - GENBANK/AY173827
SI  - GENBANK/AY173828
SI  - GENBANK/AY173829
SI  - GENBANK/AY173830
SI  - GENBANK/AY173831
SI  - GENBANK/AY173832
SI  - GENBANK/AY173833
SI  - GENBANK/AY173834
SI  - GENBANK/AY173835
SI  - GENBANK/AY173836
SI  - GENBANK/AY173837
SI  - GENBANK/AY173838
SI  - GENBANK/AY173839
SI  - GENBANK/AY173840
SI  - GENBANK/AY173841
SI  - GENBANK/AY173842
SI  - GENBANK/AY173843
SI  - GENBANK/AY173844
SI  - GENBANK/AY173845
SI  - GENBANK/AY173846
SI  - GENBANK/AY173847
SI  - GENBANK/AY173848
SI  - GENBANK/AY173849
SI  - GENBANK/AY173850
SI  - GENBANK/AY173851
SI  - GENBANK/AY173852
SI  - GENBANK/AY173853
SI  - GENBANK/AY173854
SI  - GENBANK/AY173855
SI  - GENBANK/AY173856
SI  - GENBANK/AY173857
SI  - GENBANK/AY173858
SI  - GENBANK/AY173859
SI  - GENBANK/AY173860
SI  - GENBANK/AY173861
SI  - GENBANK/AY173862
SI  - GENBANK/AY173863
SI  - GENBANK/AY173864
SI  - GENBANK/AY173865
SI  - GENBANK/AY173866
SI  - GENBANK/AY173867
SI  - GENBANK/AY173868
SI  - GENBANK/AY173869
SI  - GENBANK/AY173870
SI  - GENBANK/AY173871
SI  - GENBANK/AY173872
SI  - GENBANK/AY173873
SI  - GENBANK/AY173874
SI  - GENBANK/AY173875
SI  - GENBANK/AY173876
SI  - GENBANK/AY173877
SI  - GENBANK/AY173878
SI  - GENBANK/AY173879
SI  - GENBANK/AY173880
SI  - GENBANK/AY173881
SI  - GENBANK/AY173882
SI  - GENBANK/AY173883
SI  - GENBANK/AY173884
SI  - GENBANK/AY173885
SI  - GENBANK/AY173886
SI  - GENBANK/AY173887
SI  - GENBANK/AY173888
SI  - GENBANK/AY173889
SI  - GENBANK/AY173890
SI  - GENBANK/AY173891
SI  - GENBANK/AY173892
SI  - GENBANK/AY173893
SI  - GENBANK/AY173894
SI  - GENBANK/AY173895
SI  - GENBANK/AY173896
SI  - GENBANK/AY173897
SI  - GENBANK/AY173898
SI  - GENBANK/AY173899
SI  - GENBANK/AY173900
SI  - GENBANK/AY173901
SI  - GENBANK/AY173902
SI  - GENBANK/AY173903
SI  - GENBANK/AY173904
SI  - GENBANK/AY173905
SI  - GENBANK/AY173906
SI  - GENBANK/AY173907
SI  - GENBANK/AY173908
SI  - GENBANK/AY173909
SI  - GENBANK/AY173910
SI  - GENBANK/AY173911
SI  - GENBANK/AY173912
SI  - GENBANK/AY173913
SI  - GENBANK/AY173914
SI  - GENBANK/AY173915
SI  - GENBANK/AY173916
SI  - GENBANK/AY173917
SI  - GENBANK/AY173918
SI  - GENBANK/AY173919
SI  - GENBANK/AY173920
SI  - GENBANK/AY173921
SI  - GENBANK/AY173922
SI  - GENBANK/AY173923
SI  - GENBANK/AY173924
SI  - GENBANK/AY173925
SI  - GENBANK/AY173926
SI  - GENBANK/AY173927
SI  - GENBANK/AY173928
SI  - GENBANK/AY173929
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Argentina
MH  - Base Sequence
MH  - DNA, Mitochondrial/genetics
MH  - Geography
MH  - Iguanas/*classification/genetics
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Population Dynamics
MH  - Sample Size
EDAT- 2003/05/15 05:00
MHDA- 2003/09/16 05:00
CRDT- 2003/05/15 05:00
PHST- 2003/05/15 05:00 [pubmed]
PHST- 2003/09/16 05:00 [medline]
PHST- 2003/05/15 05:00 [entrez]
AID - 5D53ECFCTUCT3XA5 [pii]
AID - 10.1080/10635150390192717 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Apr;52(2):159-85. doi: 10.1080/10635150390192717.

PMID- 12746144
OWN - NLM
STAT- MEDLINE
DCOM- 20030915
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 2
DP  - 2003 Apr
TI  - Stochastic mapping of morphological characters.
PG  - 131-58
AB  - Many questions in evolutionary biology are best addressed by comparing traits in 
      different species. Often such studies involve mapping characters on phylogenetic 
      trees. Mapping characters on trees allows the nature, number, and timing of the
      transformations to be identified. The parsimony method is the only method
      available for mapping morphological characters on phylogenies. Although the
      parsimony method often makes reasonable reconstructions of the history of a
      character, it has a number of limitations. These limitations include the
      inability to consider more than a single change along a branch on a tree and the 
      uncoupling of evolutionary time from amount of character change. We extended a
      method described by Nielsen (2002, Syst. Biol. 51:729-739) to the mapping of
      morphological characters under continuous-time Markov models and demonstrate here
      the utility of the method for mapping characters on trees and for identifying
      character correlation.
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
AD  - Section of Ecology, Behavior and Evolution, Division of Biology, University of
      California-San Diego, La Jolla, California 92093-0116, USA.
FAU - Nielsen, Rasmus
AU  - Nielsen R
FAU - Bollback, Jonathan P
AU  - Bollback JP
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Aphids/classification/genetics
MH  - Bayes Theorem
MH  - Models, Genetic
MH  - Monte Carlo Method
MH  - *Phenotype
MH  - *Phylogeny
MH  - Pontederiaceae/classification/genetics
MH  - Starfish/classification/genetics
MH  - Stochastic Processes
EDAT- 2003/05/15 05:00
MHDA- 2003/09/16 05:00
CRDT- 2003/05/15 05:00
PHST- 2003/05/15 05:00 [pubmed]
PHST- 2003/09/16 05:00 [medline]
PHST- 2003/05/15 05:00 [entrez]
AID - WFBKUNE0G83NWNAF [pii]
AID - 10.1080/10635150390192780 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Apr;52(2):131-58. doi: 10.1080/10635150390192780.

PMID- 12554461
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - A phylogeny of the tinamous (aves: palaeognathiformes) based on integumentary
      characters.
PG  - 959-79
AB  - A cladistic analysis of the tinamous, including the 47 currently recognized
      species and some distinct subspecies, was conducted based on 80 integumentary
      characters from adult and natal plumage, ramphoteca (corneum sheath of bill), and
      podoteca (horny scales of legs). For the adult plumage (50 characters), we
      studied feather pigmentation patterns from different pterylae (feather tracts). A
      criterion of overlap of basic pigmentation elements was used to assign costs to
      the transformation between the states in most of these characters in such a way
      that transformations between more similar conditions were less costly. The
      consensus tree was almost fully resolved, and about 50% of its groups were
      relatively well supported. Because the only outgroup that could be used provided 
      a poor root, two possible rootings of the ingroup subtree were considered; in
      both cases, only one of the two traditional subfamilies (the steppe tinamous) was
      recovered, and the other (the forest tinamous) appeared as paraphyletic. The
      results of the present analysis are compared with those from an osteological data
      set, using a strict supertree technique. The combined tree has a large number of 
      nodes, indicating a high degree of congruence between the two data sets.
FAU - Bertelli, Sara
AU  - Bertelli S
AD  - Programa de Investigaciones de Biodiversidad Argentina, Facultad de Ciencias
      Naturales e Instituto Miguel Lillo, Codigo Postal 4000, Argentina.
      pidba@infovia.com.ar
FAU - Giannini, Norberto P
AU  - Giannini NP
FAU - Goloboff, Pablo A
AU  - Goloboff PA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Birds/*anatomy &amp; histology/*genetics
MH  - Evolution, Molecular
MH  - Feathers
MH  - Phylogeny
MH  - Species Specificity
MH  - Statistics as Topic
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - L09C2PH7D9TJ00GR [pii]
AID - 10.1080/10635150290102492 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):959-79. doi: 10.1080/10635150290102492.

PMID- 12554460
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - Combined nuclear and mitochondrial DNA sequences resolve generic relationships
      within the Cracidae (Galliformes, Aves).
PG  - 946-58
AB  - The Cracidae is one of the most endangered and distinctive bird families in the
      Neotropics, yet the higher relationships among taxa remain uncertain. The
      molecular phylogeny of its 11 genera was inferred using 10,678 analyzable sites
      (5,412 from seven different mitochondrial segments and 5,266 sites from four
      nuclear genes). We performed combinability tests to check conflicts in
      phylogenetic signals of separate genes and genomes. Phylogenetic analysis showed 
      that the unrooted tree of ((curassows, horned guan) (guans, chachalacas)) was
      favored by most data partitions and that different data partitions provided
      support for different parts of the tree. In particular, the concatenated
      mitochondrial DNA (mtDNA) genes resolved shallower nodes, whereas the combined
      nuclear sequences resolved the basal connections among the major clades of
      curassows, horned guan, chachalacas, and guans. Therefore, we decided that for
      the Cracidae all data should be combined for phylogenetic analysis. Maximum
      parsimony (MP), maximum likelihood (ML), and Bayesian analyses of this large data
      set produced similar trees. The MP tree indicated that guans are the sister group
      to (horned guan, (curassows, chachalacas)), whereas the ML and Bayesian analysis 
      recovered a tree where the horned guan is a sister clade to curassows, and these 
      two clades had the chachalacas as a sister group. Parametric bootstrapping showed
      that alternative trees previously proposed for the cracid genera are
      significantly less likely than our estimate of their relationships. A likelihood 
      ratio test of the hypothesis of a molecular clock for cracid mtDNA sequences
      using the optimal ML topology did not reject rate constancy of substitutions
      through time. We estimated cracids to have originated between 64 and 90 million
      years ago (MYA), with a mean estimate of 76 MYA. Diversification of the genera
      occurred approximately 41-3 MYA, corresponding with periods of global climate
      change and other Earth history events that likely promoted divergences of higher 
      level taxa.
FAU - Pereira, Sergio L
AU  - Pereira SL
AD  - Departamento de Biologia, Universidade de Sao Paulo, Rua do Matao 277, Brazil
      05508-900.
FAU - Baker, Allan J
AU  - Baker AJ
FAU - Wajntal, Anita
AU  - Wajntal A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - Birds/*genetics
MH  - DNA/*ultrastructure
MH  - DNA, Mitochondrial/*genetics
MH  - Evolution, Molecular
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - CJX7GQ9HJAGQ3DTN [pii]
AID - 10.1080/10635150290102519 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):946-58. doi: 10.1080/10635150290102519.

PMID- 12554459
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - Molecular phylogeny of grouse: individual and combined performance of W-linked,
      autosomal, and mitochondrial loci.
PG  - 930-45
AB  - The phylogeny of grouse (Aves: Tetraoninae) was reconstructed using four
      noncoding loci: two were W-linked, one was autosomal, and one was the
      mitochondrial control region (CR). The rapidly evolving CR provided resolution
      throughout the tree, whereas the slowly evolving nuclear loci failed to resolve
      deeper nodes. The tree based on all four loci combined was almost identical to
      the CR tree and did not improve resolution or bootstrap support. The stemminess
      and imbalance of the trees were good determinants of the quality of the
      phylogenetic signal. The skewness of the tree score distribution (g(1)) behaved
      contrary to prediction; loci that had a more symmetric tree score distribution
      produced trees that had greater stemminess and balance. The quality of the
      phylogenetic signal was related to the evolutionary rate. Four clades of grouse
      were discovered. Two of these clades corresponded to currently recognized genera 
      Bonasa and Lagopus. Bonasa was the sister to other grouse and Lagopus was the
      sister to the other two non-Bonasa clades. The third clade included Falcipennis, 
      Tetrao, and Lyrurus. The fourth clade included the genera Centrocercus,
      Dendragapus, and Tympanuchus. The data support recognition of Falcipennis
      canadensis franklinii and Dendragapus obscurus fuliginosus as species.
FAU - Drovetski, Sergei V
AU  - Drovetski SV
AD  - Burke Museum and Department of Zoology, University of Washington, Box 353010,
      Seattle 98195-3010, USA. drove001@tc.umn.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Aggrecans)
RN  - 0 (Extracellular Matrix Proteins)
RN  - 0 (Lectins, C-Type)
RN  - 0 (Proteoglycans)
SB  - IM
MH  - Aggrecans
MH  - Animals
MH  - Birds/*genetics
MH  - Cell Nucleus/metabolism
MH  - Chromosome Mapping
MH  - *Evolution, Molecular
MH  - *Extracellular Matrix Proteins
MH  - Lectins, C-Type
MH  - Mitochondria/genetics/metabolism
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Polymerase Chain Reaction
MH  - Proteoglycans/genetics
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - YTATREKG8J0K9DMW [pii]
AID - 10.1080/10635150290102500 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):930-45. doi: 10.1080/10635150290102500.

PMID- 12554458
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - Contingent predictability in evolution: key traits and diversification.
PG  - 917-29
AB  - Key innovations have often been invoked to explain the exceptional
      diversification of particular groups. However, there are few convincing examples 
      of traits that are repeatedly and consistently associated with increased
      diversification. The paucity of such cases may reflect the contingent nature of
      the diversifying effect of key traits. These contingencies can be viewed as
      statistical interactions between the trait and at least three kinds of factors:
      (1) other taxa, (2) other traits of the group itself, and (3) the physical
      environment. I describe tentative examples in each of these categories: (1) a
      dampening of the diversification of clades with image-forming eyes by groups that
      earlier evolved such eyes, (2) an effect of growth form (woody or herbaceous) on 
      the diversifying effect of biotic seed dispersal in angiosperms, and (3) an
      effect of atmospheric CO(2) level on the diversifying effect of C(4)
      photosynthesis in monocots. These examples suggest the need for more complex
      analyses of the relationship between possible key traits and diversification.
      They also suggest that radiations may be predictable given certain circumstances,
      thus supporting a view of evolution as both predictable and contingent.
      Ironically, a certain degree of predictability may be critical to arguments for
      evolutionary contingency.
FAU - de Queiroz, Alan
AU  - de Queiroz A
AD  - Department of Environmental, Population, and Organismic Biology and Natural
      History Museum, University of Colorado, Boulder 80309-0334, USA.
      dequeiro@stripe.colorado.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 142M471B3J (Carbon Dioxide)
SB  - IM
MH  - *Biological Evolution
MH  - Carbon Dioxide/metabolism
MH  - Evolution, Molecular
MH  - Genes, Plant
MH  - Genetic Variation
MH  - Magnoliopsida/genetics
MH  - Models, Statistical
MH  - Photosynthesis
MH  - Seeds
MH  - Time Factors
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - GQR23BHECMK9B1HJ [pii]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):917-29.

PMID- 12554457
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - Tree balance, time slices, and evolutionary turnover in cretaceous planktonic
      foraminifera.
PG  - 908-16
AB  - Studies of phylogenetic tree shape often concentrate on the balance of
      phylogenies of extant taxa. Paleontological phylogenies (which include extinct
      taxa) can contain additional useful information and can directly document changes
      in tree shape through evolutionary time. Unfortunately, the inclusion of extinct 
      taxa lowers the power of direct examinations of tree balance because it increases
      the range of tree shapes expected under null models of evolution (with equal
      rates of speciation and extinction across lineages). A promising approach for the
      analysis of tree shape in paleontological phylogenies is to break the phylogeny
      down into time slices, examining the shape of the phylogeny of taxa alive at each
      time slice and changes in that shape between successive time slices. This method 
      was illustrated with 57 time slices through a stratophenetic phylogeny of the
      Cretaceous planktonic foraminiferal superfamily Globotruncanacea. At 3 of 56
      intervals between time slices, 93-92.5 million years ago (MYA), 89-88.5 MYA, and 
      85.5-84 MYA, the group showed steep increases in imbalance. Although none of
      these increases were significant after Bonferroni correction, these points in the
      history of the Globotruncanacea were nevertheless identified as deserving of
      further macroevolutionary investigation. The 84 MYA time slice coincides with a
      peak in species turnover for the superfamily. Time slices through phylogenies may
      prove useful for identifying periods of time when evolution was proceeding in a
      nonstochastic manner.
FAU - Harcourt-Brown, Katherine G
AU  - Harcourt-Brown KG
AD  - Department of Earth Sciences, University of Bristol, Wills Memorial Building,
      Queens Road, UK.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - Evolution, Molecular
MH  - Paleontology
MH  - *Phylogeny
MH  - Plankton/*genetics
MH  - Time Factors
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - 9PBGY306ET10N61T [pii]
AID - 10.1080/10635150290102618 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):908-16. doi: 10.1080/10635150290102618.

PMID- 12554456
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - Phylogenetic balance and ecological evenness.
PG  - 898-907
AB  - The frequency distribution of numbers of species in taxonomic groups, where many 
      species belong to a few very diverse higher taxa, is mirrored by that of species 
      in most communities, where many individuals belong to a few very abundant
      species. Various hypotheses mechanistically link a species' community abundance
      with the diversity of the higher level taxon (genus, family, order) to which it
      belongs, but empirical data are equivocal about general trends in the relation
      between rank-taxon diversity and mean abundance. One reason for this inconclusive
      result may be the effect of the semisubjective nature of rank-based
      classification. We assessed the relationship between clade diversity and mean
      species abundance for two diverse tropical tree communities, using both
      traditional rank-based analysis and two new phylogenetic analyses (based on the
      ratio of individuals to taxa at each node in the phylogeny). Both rank-based and 
      phylogenetic analyses using taxonomic ranks above the species level as terminal
      taxa detected a trend associating common species with species-rich families. In
      contrast, phylogenetic analyses using species as terminal taxa could not
      distinguish the observed distribution of species abundances from a random
      distribution with respect to clade diversity. The difference between these
      results might be due to (1) the absence of a real phylogeny-wide relationship
      between clade abundance and diversity, (2) the influence of poor phylogenetic
      resolution within families in our phylogenies, or (3) insufficient sensitivity of
      our metrics to subtle tree-wide effects. Further development and application of
      phylogeny-based methods for testing abundance-diversity relationships is needed.
FAU - Webb, Campbell O
AU  - Webb CO
AD  - Department of Ecology and Evolutionary Biology, Yale University, Box 208106, New 
      Haven, Connecticut 06511, USA. campbell.webb@yale.edu
FAU - Pitman, Nigel C A
AU  - Pitman NC
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Ecology
MH  - Genetic Variation
MH  - *Phylogeny
MH  - Time Factors
MH  - Trees/genetics
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - VEPEWWDFALAPA8X6 [pii]
AID - 10.1080/10635150290102609 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):898-907. doi: 10.1080/10635150290102609.

PMID- 12554455
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - Signatures of random and selective mass extinctions in phylogenetic tree balance.
PG  - 889-97
AB  - Current models of diversification with evolving speciation rates have trouble
      mimicking the extreme imbalance seen in estimated phylogenies. However, these
      models have not incorporated extinction. Here, we report on a simple simulation
      model that includes heritable and evolving speciation rates coupled with mass
      extinctions, Random (but not selective) mass extinctions, coupled with evolving
      among-lineage variation in speciation rates, increase imbalance of postrecovery
      clades. Thus, random mass extinctions are plausible contributors to the imbalance
      of modern clades. Paleontological evidence suggests that mass extinctions are
      often random with respect to ecological and morphological traits, consistent with
      our simulations. In contrast, evidence that the current anthropogenic mass
      extinction is phylogenetically selective suggests that the current extinction
      episode may be qualitatively different from past ones in the way it reshapes
      future biotas.
FAU - Heard, Stephen B
AU  - Heard SB
AD  - Department of Biology, University of New Brunswick, Fredericton, Canada.
      sheard@unb.ca
FAU - Mooers, Arne O
AU  - Mooers AO
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Ecology
MH  - *Extinction, Psychological
MH  - Genetic Variation
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - J0H4HY9DAAEN4KCV [pii]
AID - 10.1080/10635150290102591 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):889-97. doi: 10.1080/10635150290102591.

PMID- 12554454
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - New inferences from tree shape: numbers of missing taxa and population growth
      rates.
PG  - 881-8
AB  - The relative positions of branching events in a phylogeny contain information
      about evolutionary and population dynamic processes. We provide new summary
      statistics of branching event times and describe how these statistics can be used
      to infer rates of species diversification from interspecies trees or rates of
      population growth from intraspecies trees. We also introduce a phylogenetic
      method for estimating the level of taxon sampling in a clade. Different
      evolutionary models and different sampling regimes can produce similar patterns
      of branching events, so it is important to consider explicitly the model
      assumptions involved when making evolutionary inferences. Results of an analysis 
      of the phylogeny of the mosquito-borne flaviviruses suggest that there could be
      several thousand currently unidentified viruses in this clade.
FAU - Pybus, O G
AU  - Pybus OG
AD  - Department of Zoology, University of Oxford, South Parks Road, UK.
FAU - Rambaut, A
AU  - Rambaut A
FAU - Holmes, E C
AU  - Holmes EC
FAU - Harvey, P H
AU  - Harvey PH
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Culicidae
MH  - Evolution, Molecular
MH  - Extinction, Psychological
MH  - Flavivirus/genetics
MH  - Genetics, Population
MH  - Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - HGD90T5UFULHWBVG [pii]
AID - 10.1080/10635150290102582 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):881-8. doi: 10.1080/10635150290102582.

PMID- 12554453
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - Phylogeny shape and the phylogenetic comparative method.
PG  - 873-80
AB  - We explored the impact of phylogeny shape on the results of interspecific
      statistical analyses incorporating phylogenetic information. In most phylogenetic
      comparative methods (PCMs), the phylogeny can be represented as a relationship
      matrix, and the hierarchical nature of interspecific phylogenies translates into 
      a distinctive blocklike matrix that can be described by its eigenvectors
      (topology) and eigenvalues (branch lengths). Thus, differences in the
      eigenvectors and eigenvalues of different relationship matrices can be used to
      gauge the impact of possible phylogeny errors by comparing the actual phylogeny
      used in a PCM analysis with a second phylogenetic hypothesis that may be more
      accurate. For example, we can use the sum of inverse eigenvalues as a rough index
      to compare the impact of phylogenies with different branch lengths. Topological
      differences are better described by the eigenvectors. In general, phylogeny
      errors that involve deep splits in the phylogeny (e.g., moving a taxon across the
      base of the phylogeny) are likely to have much greater impact than will those
      involving small perturbations in the fine structure near the tips. Small
      perturbations, however, may have more of an impact if the phylogeny structure is 
      highly dependent (with many recent splits near the tips of the tree).
      Unfortunately, the impact of any phylogeny difference on the results of a PCM
      depends on the details of the data being considered. Recommendations regarding
      the choice, design, and statistical power of interspecific analyses are also
      made.
FAU - Martins, Emilia P
AU  - Martins EP
AD  - Department of Biology, Indiana University, Bloomington 47405, USA.
      emartins@indiana.edu
FAU - Housworth, Elizabeth A
AU  - Housworth EA
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Evolution, Molecular
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Principal Component Analysis
MH  - Statistics as Topic
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - 5AQMJECHAUPRW03D [pii]
AID - 10.1080/10635150290102573 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):873-80. doi: 10.1080/10635150290102573.

PMID- 12554452
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - Power of eight tree shape statistics to detect nonrandom diversification: a
      comparison by simulation of two models of cladogenesis.
PG  - 866-72
AB  - We used simulations to compare the relative power of eight statistical tests to
      detect imbalance in phylogenies that is too great to be ascribed to an
      equal-rates Markov null model. Three of these tests have never had their power
      assessed before. Our simulations are the first to assess performance under
      scenarios in which the speciation rates of various lineages can evolve
      independently. In one of the scenarios explored, rates depend upon the value of
      an evolving trait, whereas in the other the probability that a species will
      speciate declines with the time since it last did so. The results indicate that
      the relative performance of the methods depends upon how the imbalance is
      generated. Different types of processes lead to different imbalance signatures,
      i.e., different patterns of imbalance at different depths in the phylogeny, and
      the measures of tree shape differ in the depth of phylogeny at which they are
      most sensitive. Relative performance is also affected by tree size but does not
      appear to depend greatly upon the degree of speciation rate variation among
      lineages. Two of the indices (Colless's index I(c) and Shao and Sokal's Nmacr;)
      show reasonable performance throughout, but another (Shao and Sokal's B(2)) is
      never indicated to be a preferred method. Two tests that do not require
      completely resolved phylogenies, mean I' and mean I'(10), have reasonable power.
FAU - Agapow, Paul-Michael
AU  - Agapow PM
AD  - School of Applied Statistics, University of Reading, UK.
FAU - Purvis, Andy
AU  - Purvis A
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Evolution, Molecular
MH  - *Models, Statistical
MH  - *Phylogeny
MH  - Statistics as Topic
MH  - Time Factors
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - D0PWBLLD9D4UYB2J [pii]
AID - 10.1080/10635150290102564 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):866-72. doi: 10.1080/10635150290102564.

PMID- 12554451
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - Whole-tree methods for detecting differential diversification rates.
PG  - 855-65
AB  - Prolific cladogenesis, adaptive radiation, species selection, key innovations,
      and mass extinctions are a few examples of biological phenomena that lead to
      differential diversification among lineages. Central to the study of differential
      diversification rates is the ability to distinguish chance variation from that
      which requires deterministic explanation. To detect diversification rate
      variation among lineages, we propose a number of methods that incorporate
      information on the topological distribution of species diversity from all
      internal nodes of a phylogenetic tree. These whole-tree methods (M(Pi), M(Sigma),
      and M(R)) are explicitly connected to a null model of random diversification--the
      equal-rates Markov (ERM) random branching model--and an alternative model of
      differential diversification: M(Pi) is based on the product of individual nodal
      ERM probabilities; M(Sigma) is based on the sum of individual nodal ERM
      probabilities, and M(R) is based on a transformation of ERM probabilities that
      corresponds to a formalized system that orders trees by their relative symmetry. 
      These methods have been implemented in a freely available computer program,
      SYMMETREE, to detect clades with variable diversification rates, thereby allowing
      the study of biological processes correlated with and possibly causal to shifts
      in diversification rate. Application of these methods to several published
      phylogenies demonstrates their ability to contend with relatively large,
      incompletely resolved trees. These topology-based methods do not require
      estimates of relative branch lengths, which should facilitate the analysis of
      phylogenies, such as supertrees, for which such data are unreliable or
      unavailable.
FAU - Chan, Kai M A
AU  - Chan KM
AD  - Department of Ecology and Evolutionary Biology, Princeton University, New Jersey 
      08544-1003, USA.
FAU - Moore, Brian R
AU  - Moore BR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Evolution, Molecular
MH  - Markov Chains
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Statistics as Topic
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - L8QV56JBB61V1ULV [pii]
AID - 10.1080/10635150290102555 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):855-65. doi: 10.1080/10635150290102555.

PMID- 12554450
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - Phylogeny imbalance: taxonomic level matters.
PG  - 844-54
AB  - Two lines of evidence indicate that the degree of symmetry in phylogenetic
      topologies differs at different hierarchical levels. First, in a set of 61
      phylogenies with superspecific taxa as their terminals, trees were on average
      more unbalanced (asymmetric) when the species richness of terminals was
      considered than when it was not. Second, nodes with a given number of higher taxa
      descended from them were on average more unbalanced than were nodes with the same
      number of species as descendants. There are several possible reasons--some
      biological, some artifactual--for the differences. Whatever the reason, these
      results caution against treating species-level and higher level phylogenies as
      equivalent when considering tree shape. The imbalance measure adopted here
      permits the use of trees that contain polytomies, facilitating a larger sample
      than has been achieved previously.
FAU - Purvis, Andy
AU  - Purvis A
AD  - Department of Biological Sciences, Imperial College, Silwood Park, Ascot SL5 7PY,
      UK.
FAU - Agapow, Paul-Michael
AU  - Agapow PM
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Arthropods/genetics
MH  - *Biological Evolution
MH  - Genes, Plant
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Statistics as Topic
MH  - Vertebrates/genetics
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - 959AH8Y85LN1C78M [pii]
AID - 10.1080/10635150290102546 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):844-54. doi: 10.1080/10635150290102546.

PMID- 12554449
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - Is cladogenesis heritable?
PG  - 835-43
AB  - The heritability of speciation rates and extinction risks is a crucial parameter 
      in models of macroevolution, but little direct evidence has been available to
      assess the occurrence, strength, or generality of this heritability. We tested
      for heritability using correlations between ancestral and descendent branch
      lengths in phylogenetic trees, an approach first applied to a bird phylogeny by
      Harvey et al. (1991, pages 123-137 in Genes in ecology [R. J. Berry et al.,
      eds.], Blackwell Scientific, Oxford). We applied Harvey et al.'s test to some of 
      the largest DNA sequence-based phylogenetic analyses published to date for
      plants, insects, fungi, and bacteria. If one of two parent lineages splits first 
      and if this is the case for any heritable reason, then on average we expect its
      daughter lineages to also split first. We also used a randomization procedure to 
      assess significance of branch length heritability. Using maximum parsimony and
      maximum likelihood branch lengths and trees made ultrametric after nonparametric 
      rate smoothing or by enforcing a molecular clock, we found a pattern for most
      clades consistent with heritable net cladogenesis. Heritability of cladogenesis
      may be a general phenomenon, detectable across a large number of lineages and a
      broad range of taxa.
FAU - Savolainen, Vincent
AU  - Savolainen V
AD  - Molecular Systematics Section, Jodrell Laboratory, Royal Botanic Gardens, Kew TW9
      3DS, UK. v.savolainen@rbgkew.org.uk
FAU - Heard, Stephen B
AU  - Heard SB
FAU - Powell, Martyn P
AU  - Powell MP
FAU - Davies, T Jonathan
AU  - Davies TJ
FAU - Mooers, Arne O
AU  - Mooers AO
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Birds
MH  - Genes, Bacterial
MH  - Genes, Fungal
MH  - Genes, Plant
MH  - Insecta/genetics
MH  - Likelihood Functions
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Random Allocation
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - G10JDQU83K2NRJ0L [pii]
AID - 10.1080/10635150290102537 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):835-43. doi: 10.1080/10635150290102537.

PMID- 12554448
OWN - NLM
STAT- MEDLINE
DCOM- 20030926
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 6
DP  - 2002 Dec
TI  - Using tree shape.
PG  - 833-4
FAU - Mooers, Arne O
AU  - Mooers AO
AD  - Department of Biological Sciences, Simon Fraser University, Burnaby, British
      Columbia V5A 1S6, Canada.
FAU - Heard, Stephen B
AU  - Heard SB
LA  - eng
PT  - Journal Article
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - Models, Statistical
MH  - *Phylogeny
RF  - 17
EDAT- 2003/01/30 04:00
MHDA- 2003/09/27 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/27 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - RJ65W7K7YMKE5E3Y [pii]
AID - 10.1080/10635150290102528 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Dec;51(6):833-4. doi: 10.1080/10635150290102528.

PMID- 12554447
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - Branch lengths and support.
PG  - 127-30
FAU - Wilkinson, Mark
AU  - Wilkinson M
AD  - Department of Zoology, Natural History Museum, London SW7 5BD, UK.
FAU - Lapointe, Francois-Joseph
AU  - Lapointe FJ
FAU - Gower, David J
AU  - Gower DJ
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2003 Dec;52(6):849-51. PMID: 14668121
MH  - *Data Interpretation, Statistical
MH  - Likelihood Functions
MH  - Models, Statistical
MH  - *Phylogeny
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - 2C63E4V3FHLLU2R2 [pii]
AID - 10.1080/10635150390132939 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):127-30. doi: 10.1080/10635150390132939.

PMID- 12554446
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - Is sparse taxon sampling a problem for phylogenetic inference?
PG  - 124-6
FAU - Hillis, David M
AU  - Hillis DM
AD  - Section of Integrative Biology and Center for Computational Biology and
      Bioinformatics, University of Texas, 1 University Station C0930, Austin, Texas
      78712-0253, USA. dhillis@mail.utexas.edu
FAU - Pollock, David D
AU  - Pollock DD
FAU - McGuire, Jimmy A
AU  - McGuire JA
FAU - Zwickl, Derrick J
AU  - Zwickl DJ
LA  - eng
GR  - R21 GM065612/GM/NIGMS NIH HHS/United States
GR  - R21 GM065612-01/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Data Interpretation, Statistical
MH  - *Phylogeny
MH  - *Sample Size
PMC - PMC2943953
MID - NIHMS194765
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - UXLWQE536LWAJTF1 [pii]
AID - 10.1080/10635150390132911 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):124-6. doi: 10.1080/10635150390132911.

PMID- 12554445
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - Taxon sampling, bioinformatics, and phylogenomics.
PG  - 119-24
FAU - Rosenberg, Michael S
AU  - Rosenberg MS
AD  - Department of Biology, Arizona State University, Tempe, Arizona 85287-1501, USA. 
      msr@asu.edu
FAU - Kumar, Sudhir
AU  - Kumar S
LA  - eng
GR  - R01 HG002096/HG/NHGRI NIH HHS/United States
GR  - R01 HG002096-03/HG/NHGRI NIH HHS/United States
GR  - HG-02096/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Computational Biology/*methods
MH  - Genomics/*methods
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Sample Size
PMC - PMC2796430
MID - NIHMS145019
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - 08NRKY7RC6HHB66Y [pii]
AID - 10.1080/10635150390132894 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):119-24. doi: 10.1080/10635150390132894.

PMID- 12554444
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - The combinatorics of tandem duplication trees.
PG  - 110-8
AB  - We developed a recurrence relation that counts the number of tandem duplication
      trees (either rooted or unrooted) that are consistent with a set of n tandemly
      repeated sequences generated under the standard unequal recombination (or
      crossover) model of tandem duplications. The number of rooted duplication trees
      is exactly twice the number of unrooted trees, which means that on average only
      two positions for a root on a duplication tree are possible. Using the
      recurrence, we tabulated these numbers for small values of n. We also developed
      an asymptotic formula that for large n provides estimates for these numbers.
      These numbers give a priori probabilities for phylogenies of the repeated
      sequences to be duplication trees. This work extends earlier studies where
      exhaustive counts of the numbers for small n were obtained. One application
      showed the significance of finding that most maximum-parsimony trees constructed 
      from repeat sequences from human immunoglobins and T-cell receptors were tandem
      duplication trees. Those findings provided strong support to the proposed
      mechanisms of tandem gene duplication. The recurrence relation also suggests
      efficient algorithms to recognize duplication trees and to generate random
      duplication trees for simulation. We present a linear-time recognition algorithm.
FAU - Gascuel, Olivier
AU  - Gascuel O
AD  - Departement d'Informatique Fondamentale et Applications, LIRMM, 161 Rue Ada,
      34392 Montpellier, France.
FAU - Hendy, Michael D
AU  - Hendy MD
FAU - Jean-Marie, Alain
AU  - Jean-Marie A
FAU - McLachlan, Robert
AU  - McLachlan R
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Gene Duplication
MH  - Models, Statistical
MH  - *Phylogeny
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - PE1NA1GVPFTLV1KV [pii]
AID - 10.1080/10635150390132821 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):110-8. doi: 10.1080/10635150390132821.

PMID- 12554443
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - Molecular systematics and adaptive radiation of Hawaii's endemic Damselfly genus 
      Megalagrion (Odonata: Coenagrionidae).
PG  - 89-109
AB  - Damselflies of the endemic Hawaiian genus Megalagrion have radiated into a wide
      variety of habitats and are an excellent model group for the study of adaptive
      radiation. Past phylogenetic analysis based on morphological characters has been 
      problematic. Here, we examine relationships among 56 individuals from 20 of the
      23 described species using maximum likelihood (ML) and Bayesian phylogenetic
      analysis of mitochondrial (1287 bp) and nuclear (1039 bp) DNA sequence data.
      Models of evolution were chosen using the Akaike information criterion. Problems 
      with distant outgroups were accommodated by constraining the best ML ingroup
      topology but allowing the outgroups to attach to any ingroup branch in a
      bootstrap analysis. No strong contradictions were obtained between either data
      partition and the combined data set. Areas of disagreement are mainly confined to
      clades that are strongly supported by the mitochondrial DNA and weakly supported 
      by the elongation factor 1alpha data because of lack of changes. However, the
      combined analysis resulted in a unique tree. Correlation between Bayesian
      posterior probabilities and bootstrap percentages decreased in concert with
      decreasing information in the data partitions. In cases where nodes were
      supported by single characters bootstrap proportions were dramatically reduced
      compared with posterior probabilities. Two speciation patterns were evident from 
      the phylogenetic analysis. First, most speciation is interisland and occurred as 
      members of established ecological guilds colonized new volcanoes after they
      emerged from the sea. Second, there are several instances of rapid radiation into
      a variety of specialized habitats, in one case entirely within the island of
      Kauai. Application of a local clock procedure to the mitochondrial DNA topology
      suggests that two of these radiations correspond to the development of habitat on
      the islands of Kauai and Oahu. About 4.0 million years ago, species
      simultaneously moved into fast streams and plant leaf axils on Kauai, and about
      1.5 million years later another group moved simultaneously to seeps and
      terrestrial habitats on Oahu. Results from the local clock analysis also strongly
      suggest that Megalagrion arrived in Hawaii about 10 million years ago, well
      before the emergence of Kauai. Date estimates were more sensitive to the
      particular node that was fixed in time than to the model of local branch
      evolution used. We propose a general model for the development of endemic
      damselfly species on Hawaiian Islands and document five potential cases of
      hybridization (M. xanthomelas x M. pacificum, M. eudytum x M. vagabundum, M.
      orobates x M. oresitrophum, M. nesiotes x M. oahuense, and M. mauka x M.
      paludicola).
FAU - Jordan, Steve
AU  - Jordan S
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, Box
      U-3043, Storrs, Connecticut 06269, USA. steve.jordon@ujf-grenoble.fr
FAU - Simon, Chris
AU  - Simon C
FAU - Polhemus, Dan
AU  - Polhemus D
LA  - eng
SI  - GENBANK/AY179038
SI  - GENBANK/AY179039
SI  - GENBANK/AY179040
SI  - GENBANK/AY179041
SI  - GENBANK/AY179042
SI  - GENBANK/AY179043
SI  - GENBANK/AY179044
SI  - GENBANK/AY179045
SI  - GENBANK/AY179046
SI  - GENBANK/AY179047
SI  - GENBANK/AY179048
SI  - GENBANK/AY179049
SI  - GENBANK/AY179050
SI  - GENBANK/AY179051
SI  - GENBANK/AY179052
SI  - GENBANK/AY179053
SI  - GENBANK/AY179054
SI  - GENBANK/AY179055
SI  - GENBANK/AY179056
SI  - GENBANK/AY179057
SI  - GENBANK/AY179058
SI  - GENBANK/AY179059
SI  - GENBANK/AY179060
SI  - GENBANK/AY179061
SI  - GENBANK/AY179062
SI  - GENBANK/AY179063
SI  - GENBANK/AY179064
SI  - GENBANK/AY179065
SI  - GENBANK/AY179066
SI  - GENBANK/AY179067
SI  - GENBANK/AY179068
SI  - GENBANK/AY179069
SI  - GENBANK/AY179070
SI  - GENBANK/AY179071
SI  - GENBANK/AY179072
SI  - GENBANK/AY179073
SI  - GENBANK/AY179074
SI  - GENBANK/AY179075
SI  - GENBANK/AY179076
SI  - GENBANK/AY179077
SI  - GENBANK/AY179078
SI  - GENBANK/AY179079
SI  - GENBANK/AY179080
SI  - GENBANK/AY179081
SI  - GENBANK/AY179082
SI  - GENBANK/AY179083
SI  - GENBANK/AY179084
SI  - GENBANK/AY179085
SI  - GENBANK/AY179086
SI  - GENBANK/AY179087
SI  - GENBANK/AY179088
SI  - GENBANK/AY179089
SI  - GENBANK/AY179090
SI  - GENBANK/AY179091
SI  - GENBANK/AY179092
SI  - GENBANK/AY179093
SI  - GENBANK/AY179094
SI  - GENBANK/AY179095
SI  - GENBANK/AY179096
SI  - GENBANK/AY179097
SI  - GENBANK/AY179098
SI  - GENBANK/AY179099
SI  - GENBANK/AY179100
SI  - GENBANK/AY179101
SI  - GENBANK/AY179102
SI  - GENBANK/AY179103
SI  - GENBANK/AY179104
SI  - GENBANK/AY179105
SI  - GENBANK/AY179106
SI  - GENBANK/AY179107
SI  - GENBANK/AY179108
SI  - GENBANK/AY179109
SI  - GENBANK/AY179110
SI  - GENBANK/AY179111
SI  - GENBANK/AY179112
SI  - GENBANK/AY179113
SI  - GENBANK/AY179114
SI  - GENBANK/AY179115
SI  - GENBANK/AY179116
SI  - GENBANK/AY179117
SI  - GENBANK/AY179118
SI  - GENBANK/AY179119
SI  - GENBANK/AY179120
SI  - GENBANK/AY179121
SI  - GENBANK/AY179122
SI  - GENBANK/AY179123
SI  - GENBANK/AY179124
SI  - GENBANK/AY179125
SI  - GENBANK/AY179126
SI  - GENBANK/AY179127
SI  - GENBANK/AY179128
SI  - GENBANK/AY179129
SI  - GENBANK/AY179130
SI  - GENBANK/AY179131
SI  - GENBANK/AY179132
SI  - GENBANK/AY179133
SI  - GENBANK/AY179134
SI  - GENBANK/AY179135
SI  - GENBANK/AY179136
SI  - GENBANK/AY179137
SI  - GENBANK/AY179138
SI  - GENBANK/AY179139
SI  - GENBANK/AY179140
SI  - GENBANK/AY179141
SI  - GENBANK/AY179142
SI  - GENBANK/AY179143
SI  - GENBANK/AY179144
SI  - GENBANK/AY179145
SI  - GENBANK/AY179146
SI  - GENBANK/AY179147
SI  - GENBANK/AY179148
SI  - GENBANK/AY179149
SI  - GENBANK/AY179150
SI  - GENBANK/AY179151
SI  - GENBANK/AY179152
SI  - GENBANK/AY179153
SI  - GENBANK/AY179154
SI  - GENBANK/AY179155
SI  - GENBANK/AY179156
SI  - GENBANK/AY179157
SI  - GENBANK/AY179158
SI  - GENBANK/AY179159
SI  - GENBANK/AY179160
SI  - GENBANK/AY179161
SI  - GENBANK/AY179162
SI  - GENBANK/AY179163
SI  - GENBANK/AY179164
SI  - GENBANK/AY179165
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Peptide Elongation Factor 1)
SB  - IM
MH  - Adaptation, Biological/*physiology
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA, Mitochondrial/genetics
MH  - Evolution, Molecular
MH  - Geography
MH  - Hawaii
MH  - Insecta/*classification/*genetics
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - Peptide Elongation Factor 1/genetics
MH  - *Phylogeny
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - 8E3E9ARVRM4L35LD [pii]
AID - 10.1080/10635150390132803 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):89-109. doi: 10.1080/10635150390132803.

PMID- 12554442
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - Speciation on a conveyor belt: sequential colonization of the hawaiian islands by
      Orsonwelles spiders (Araneae, Linyphiidae).
PG  - 70-88
AB  - Spiders of the recently described linyphiid genus Orsonwelles (Araneae,
      Linyphiidae) are one of the most conspicuous groups of terrestrial arthropods of 
      Hawaiian native forests. There are 13 known Orsonwelles species, and all are
      single- island endemics. This radiation provides an excellent example of insular 
      gigantism. We reconstructed the cladistic relationships of Orsonwelles species
      using a combination of morphological and molecular characters (both mitochondrial
      and nuclear sequences) within a parsimony framework. We explored and quantified
      the contribution of different character partitions and their sensitivity to
      changes in the traditional parameters (gap, transition, and transversion costs). 
      The character data show a strong phylogenetic signal, robust to parameter
      changes. The monophyly of the genus Orsonwelles is strongly supported. The
      parsimony analysis of all character evidence combined recovered a clade with of
      all the non-Kauai Orsonwelles species; the species from Kauai form a paraphyletic
      assemblage with respect to the latter former clade. The biogeographic pattern of 
      the Hawaiian Orsonwelles species is consistent with colonization by island
      progression, but alternative explanations for our data exist. Although the
      geographic origin of the radiation remains unknown, it appears that the ancestral
      colonizing species arrived first on Kauai (or an older island). The ambiguity in 
      the area cladogram (i.e., post-Oahu colonization) is not derived from conflicting
      or unresolved phylogenetic signal among Orsonwelles species but rather from the
      number of taxa on the youngest islands. Speciation in Orsonwelles occurred more
      often within islands (8 of the 12 cladogenic events) than between islands. A
      molecular clock was rejected for the sequence data. Divergence times were
      estimated by using the nonparametric rate smoothing method of Sanderson (1997,
      Mol. Biol. Evol. 14:1218-1231) and the available geological data for calibration.
      The results suggest that the oldest divergences of Orsonwelles spiders (on Kauai)
      go back about 4 million years.
FAU - Hormiga, Gustavo
AU  - Hormiga G
AD  - Department of Biological Sciences, George Washington University, Washington, DC
      20052, USA. hormiga@gwu.edu
FAU - Arnedo, Miquel
AU  - Arnedo M
FAU - Gillespie, Rosemary G
AU  - Gillespie RG
LA  - eng
SI  - GENBANK/AY078660
SI  - GENBANK/AY078661
SI  - GENBANK/AY078662
SI  - GENBANK/AY078663
SI  - GENBANK/AY078664
SI  - GENBANK/AY078665
SI  - GENBANK/AY078666
SI  - GENBANK/AY078667
SI  - GENBANK/AY078668
SI  - GENBANK/AY078669
SI  - GENBANK/AY078670
SI  - GENBANK/AY078671
SI  - GENBANK/AY078672
SI  - GENBANK/AY078673
SI  - GENBANK/AY078674
SI  - GENBANK/AY078675
SI  - GENBANK/AY078676
SI  - GENBANK/AY078677
SI  - GENBANK/AY078686
SI  - GENBANK/AY078687
SI  - GENBANK/AY078688
SI  - GENBANK/AY078689
SI  - GENBANK/AY078690
SI  - GENBANK/AY078691
SI  - GENBANK/AY078692
SI  - GENBANK/AY078693
SI  - GENBANK/AY078694
SI  - GENBANK/AY078695
SI  - GENBANK/AY078696
SI  - GENBANK/AY078697
SI  - GENBANK/AY078698
SI  - GENBANK/AY078699
SI  - GENBANK/AY078700
SI  - GENBANK/AY078701
SI  - GENBANK/AY078702
SI  - GENBANK/AY078703
SI  - GENBANK/AY078704
SI  - GENBANK/AY078705
SI  - GENBANK/AY078706
SI  - GENBANK/AY078707
SI  - GENBANK/AY078708
SI  - GENBANK/AY078709
SI  - GENBANK/AY078711
SI  - GENBANK/AY078712
SI  - GENBANK/AY078713
SI  - GENBANK/AY078714
SI  - GENBANK/AY078715
SI  - GENBANK/AY078716
SI  - GENBANK/AY078717
SI  - GENBANK/AY078718
SI  - GENBANK/AY078719
SI  - GENBANK/AY078720
SI  - GENBANK/AY078721
SI  - GENBANK/AY078722
SI  - GENBANK/AY078723
SI  - GENBANK/AY078724
SI  - GENBANK/AY078725
SI  - GENBANK/AY078744
SI  - GENBANK/AY078745
SI  - GENBANK/AY078746
SI  - GENBANK/AY078747
SI  - GENBANK/AY078748
SI  - GENBANK/AY078749
SI  - GENBANK/AY078750
SI  - GENBANK/AY078751
SI  - GENBANK/AY078752
SI  - GENBANK/AY078753
SI  - GENBANK/AY078754
SI  - GENBANK/AY078755
SI  - GENBANK/AY078756
SI  - GENBANK/AY078757
SI  - GENBANK/AY078758
SI  - GENBANK/AY078759
SI  - GENBANK/AY078774
SI  - GENBANK/AY078775
SI  - GENBANK/AY078776
SI  - GENBANK/AY078777
SI  - GENBANK/AY078778
SI  - GENBANK/AY078779
SI  - GENBANK/AY078780
SI  - GENBANK/AY078781
SI  - GENBANK/AY078782
SI  - GENBANK/AY078783
SI  - GENBANK/AY078784
SI  - GENBANK/AY078785
SI  - GENBANK/AY078786
SI  - GENBANK/AY078787
SI  - GENBANK/AY078788
SI  - GENBANK/AY078789
SI  - GENBANK/AY078790
SI  - GENBANK/AY078791
SI  - GENBANK/AY078792
SI  - GENBANK/AY078793
SI  - GENBANK/AY078794
SI  - GENBANK/AY078795
SI  - GENBANK/AY078796
SI  - GENBANK/AY078797
SI  - GENBANK/AY078798
SI  - GENBANK/AY078799
SI  - GENBANK/AY078800
SI  - GENBANK/AY078801
SI  - GENBANK/AY078802
SI  - GENBANK/AY078803
SI  - GENBANK/AY078804
SI  - GENBANK/AY078805
SI  - GENBANK/AY078806
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA, Mitochondrial/genetics
MH  - DNA, Ribosomal/genetics
MH  - Evolution, Molecular
MH  - Geography
MH  - Hawaii
MH  - *Models, Biological
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Population Dynamics
MH  - Spiders/anatomy &amp; histology/classification/*genetics
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - 3M1X7UDCYNUAVD3A [pii]
AID - 10.1080/10635150390132786 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):70-88. doi: 10.1080/10635150390132786.

PMID- 12554441
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - Computing the uniform component of shape variation.
PG  - 66-9
AB  - Any change in shape of a configuration of landmark points in two or three
      dimensions includes a uniform component, a component that is a wholly linear
      (affine) transformation. The formulas for estimating this component have been
      standardized for two-dimensional data but not for three-dimensional data. We
      suggest estimating the component by way of the complementarity between the
      uniform component and the space of partial warps. The component can be estimated 
      by regression in either one space or the other: regression on the partial warps, 
      followed by their removal, or regression on a basis for the uniform component
      itself. Either of the new methods can be used for both two- and three-dimensional
      landmark data and thus generalize Bookstein's (1996, pages 153-168 in Advances in
      morphometrics [L. F. Marcus et al., eds.], Plenum, New York) linearized
      Procrustes formula for estimating the uniform component in two dimensions.
FAU - Rohlf, F James
AU  - Rohlf FJ
AD  - Department of Ecology and Evolution, State University of New York, Stony Brook,
      New York 11794-5245, USA. rohlf@life.bio.sunysb.edu
FAU - Bookstein, Fred L
AU  - Bookstein FL
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Biometry/*methods
MH  - Body Weights and Measures/methods
MH  - *Data Interpretation, Statistical
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - M4B3R1L41JFLQ49G [pii]
AID - 10.1080/10635150390132759 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):66-9. doi: 10.1080/10635150390132759.

PMID- 12554440
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - Assessing progress in systematics with continuous jackknife function analysis.
PG  - 55-65
AB  - Systematists expect their hypotheses to be asymptotically precise. As the number 
      of phylogenetically informative characters for a set of taxa increases, the
      relationships implied should stabilize on some topology. If true, this increasing
      stability should clearly manifest itself if an index of congruence is plotted
      against the accumulating number of characters. Continuous jackknife function
      (CJF) analysis is a new graphical method that portrays the extent to which
      available data converge on a specified phylogenetic hypothesis, the reference
      tree. The method removes characters with increasing probability, analyzes the
      rarefied data matrices phylogenetically, and scores the clades shared between
      each of the resulting trees and the reference tree. As more characters are
      removed, the number of shared clades must decrease, but the rate of decrease will
      depend on how decisively the data support the reference tree. Curves for stable
      phylogenies are clearly asymptotic with nearly 100% congruence for a substantial 
      part of the curve. Less stable phylogenies lose congruent nodes quickly as
      characters are excluded, resulting in a more linear or even a sigmoidal
      relationship. Curves can be interpreted as predictors of whether the addition of 
      new data of the same type is likely to alter the hypothesis under test.
      Continuous jackknife function analysis makes statistical assumptions about the
      collection of character data. To the extent that CJF curves are sensitive to
      violations of unbiased character collection, they will be misleading as
      predictors. Convergence of data on a reference tree does not guarantee historical
      accuracy, but it does predict that the accumulation of further data under the
      sampling model will not lead to rapid changes in the hypothesis.
FAU - Miller, Jeremy A
AU  - Miller JA
AD  - Department of Systematic Biology--Entomology, National Museum of Natural History,
      NHB-105, Smithsonian Institution, PO Box 37012, Washington, DC 20013-7012, USA.
      zmjeremy@gwu.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - *Data Interpretation, Statistical
MH  - *Models, Statistical
MH  - *Phylogeny
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - Q3N014APA6585TT2 [pii]
AID - 10.1080/10635150390132731 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):55-65. doi: 10.1080/10635150390132731.

PMID- 12554439
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - Testing a molecular clock without an outgroup: derivations of induced priors on
      branch-length restrictions in a Bayesian framework.
PG  - 48-54
AB  - We propose a Bayesian method for testing molecular clock hypotheses for use with 
      aligned sequence data from multiple taxa. Our method utilizes a nonreversible
      nucleotide substitution model to avoid the necessity of specifying either a known
      tree relating the taxa or an outgroup for rooting the tree. We employ reversible 
      jump Markov chain Monte Carlo to sample from the posterior distribution of the
      phylogenetic model parameters and conduct hypothesis testing using Bayes factors,
      the ratio of the posterior to prior odds of competing models. Here, the Bayes
      factors reflect the relative support of the sequence data for equal rates of
      evolutionary change between taxa versus unequal rates, averaged over all possible
      phylogenetic parameters, including the tree and root position. As the molecular
      clock model is a restriction of the more general unequal rates model, we use the 
      Savage-Dickey ratio to estimate the Bayes factors. The Savage-Dickey ratio
      provides a convenient approach to calculating Bayes factors in favor of sharp
      hypotheses. Critical to calculating the Savage-Dickey ratio is a determination of
      the prior induced on the modeling restrictions. We demonstrate our method on a
      well-studied mtDNA sequence data set consisting of nine primates. We find strong 
      support against a global molecular clock, but do find support for a local clock
      among the anthropoids. We provide mathematical derivations of the induced priors 
      on branch length restrictions assuming equally likely trees. These derivations
      also have more general applicability to the examination of prior assumptions in
      Bayesian phylogenetics.
FAU - Suchard, Marc A
AU  - Suchard MA
AD  - Department of Biomathematics, David Geffen School of Medicine, University of
      California, Los Angeles, California 90095-1766, USA.
FAU - Weiss, Robert E
AU  - Weiss RE
FAU - Sinsheimer, Janet S
AU  - Sinsheimer JS
LA  - eng
GR  - CA16042/CA/NCI NIH HHS/United States
GR  - GM08042/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - *Bayes Theorem
MH  - DNA, Mitochondrial/genetics
MH  - *Data Interpretation, Statistical
MH  - Evolution, Molecular
MH  - Models, Statistical
MH  - Monte Carlo Method
MH  - Mutation/*genetics
MH  - *Phylogeny
MH  - Primates/genetics
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - AKU5QK317QAP8X1G [pii]
AID - 10.1080/10635150390132713 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):48-54. doi: 10.1080/10635150390132713.

PMID- 12554438
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - When do parasites fail to speciate in response to host speciation?
PG  - 37-47
AB  - Cospeciation generally increases the similarity between host and parasite
      phylogenies. Incongruence between host and parasite phylogenies has previously
      been explained in terms of host switching, sorting, and duplication events. Here,
      we describe an additional process, failure of the parasite to speciate in
      response to host speciation, that may be important in some host-parasite systems.
      Failure to speciate is likely to occur when gene flow among parasite populations 
      is much higher than that of their hosts. We reconstructed trees from
      mitochondrial and nuclear DNA sequences for pigeons and doves (Aves:
      Columbiformes) and their feather lice in the genus Columbicola (Insecta:
      Phthiraptera). Although comparisons of the trees from each group revealed a
      significant amount of cospeciation, there was also a significant degree of
      incongruence. Cophylogenetic analyses generally indicated that host switching may
      be an important process in the history of this host-parasite association. Using
      terminal sister taxon comparisons, we also identified three apparent cases where 
      the host has speciated but the associated parasite has not. In two of these cases
      of failure to speciate, these comparisons involve allopatric sister taxa of hosts
      whose lice also occur on hosts sympatric with both of the allopatric sisters.
      These additional hosts for generalist lice may promote gene flow with lice on the
      allopatric sister species. Relative rate comparisons for the mitochondrial
      cytochrome oxidase I gene indicate that molecular substitution occurs about 11
      times faster in lice than in their avian hosts.
FAU - Johnson, Kevin P
AU  - Johnson KP
AD  - Illinois Natural History Survey, 607 East Peabody Drive, Champaign, Illinois
      61820, USA.
FAU - Adams, Richard J
AU  - Adams RJ
FAU - Page, Roderic D M
AU  - Page RD
FAU - Clayton, Dale H
AU  - Clayton DH
LA  - eng
SI  - GENBANK/AF278608
SI  - GENBANK/AF278609
SI  - GENBANK/AF278610
SI  - GENBANK/AF278611
SI  - GENBANK/AF278612
SI  - GENBANK/AF278613
SI  - GENBANK/AF278614
SI  - GENBANK/AF278615
SI  - GENBANK/AF278616
SI  - GENBANK/AF278617
SI  - GENBANK/AF278618
SI  - GENBANK/AF278619
SI  - GENBANK/AF278620
SI  - GENBANK/AF278621
SI  - GENBANK/AF278622
SI  - GENBANK/AF278623
SI  - GENBANK/AF278624
SI  - GENBANK/AF278625
SI  - GENBANK/AF278626
SI  - GENBANK/AF278627
SI  - GENBANK/AF278628
SI  - GENBANK/AF278629
SI  - GENBANK/AF278630
SI  - GENBANK/AF278631
SI  - GENBANK/AF278632
SI  - GENBANK/AF278633
SI  - GENBANK/AF278634
SI  - GENBANK/AF278635
SI  - GENBANK/AF278636
SI  - GENBANK/AF278637
SI  - GENBANK/AF278638
SI  - GENBANK/AF278639
SI  - GENBANK/AF278640
SI  - GENBANK/AF278641
SI  - GENBANK/AF278642
SI  - GENBANK/AF278643
SI  - GENBANK/AF279704
SI  - GENBANK/AF279705
SI  - GENBANK/AF279706
SI  - GENBANK/AF279707
SI  - GENBANK/AF279708
SI  - GENBANK/AF279709
SI  - GENBANK/AF279710
SI  - GENBANK/AF279711
SI  - GENBANK/AF279712
SI  - GENBANK/AF279713
SI  - GENBANK/AF279714
SI  - GENBANK/AF279715
SI  - GENBANK/AF279716
SI  - GENBANK/AF279717
SI  - GENBANK/AF279718
SI  - GENBANK/AF279719
SI  - GENBANK/AF279720
SI  - GENBANK/AF279721
SI  - GENBANK/AF279722
SI  - GENBANK/AF279723
SI  - GENBANK/AF279724
SI  - GENBANK/AF279725
SI  - GENBANK/AF279726
SI  - GENBANK/AF279727
SI  - GENBANK/AF279728
SI  - GENBANK/AF279729
SI  - GENBANK/AF279730
SI  - GENBANK/AF279731
SI  - GENBANK/AF279732
SI  - GENBANK/AF279733
SI  - GENBANK/AF279734
SI  - GENBANK/AF279735
SI  - GENBANK/AF279736
SI  - GENBANK/AF279737
SI  - GENBANK/AF279738
SI  - GENBANK/AF279739
SI  - GENBANK/AF279740
SI  - GENBANK/AF279741
SI  - GENBANK/AF279742
SI  - GENBANK/AF279743
SI  - GENBANK/AY151003
SI  - GENBANK/AY151004
SI  - GENBANK/AY151005
SI  - GENBANK/AY151006
SI  - GENBANK/AY151007
SI  - GENBANK/AY151008
SI  - GENBANK/AY151009
SI  - GENBANK/AY151010
SI  - GENBANK/AY151011
SI  - GENBANK/AY151012
SI  - GENBANK/AY151013
SI  - GENBANK/AY151014
SI  - GENBANK/AY151015
SI  - GENBANK/AY151016
SI  - GENBANK/AY151017
SI  - GENBANK/AY151018
SI  - GENBANK/AY151019
SI  - GENBANK/AY151020
SI  - GENBANK/AY151021
SI  - GENBANK/AY151022
SI  - GENBANK/AY151023
SI  - GENBANK/AY151024
SI  - GENBANK/AY151025
SI  - GENBANK/AY151026
GR  - 5P30CA42014/CA/NCI NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Columbidae/*parasitology
MH  - Electron Transport Complex IV/genetics
MH  - Host-Parasite Interactions/*genetics
MH  - Molecular Sequence Data
MH  - Phthiraptera/genetics/*physiology
MH  - *Phylogeny
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - JF8959128JU409RB [pii]
AID - 10.1080/10635150390132704 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):37-47. doi: 10.1080/10635150390132704.

PMID- 12554437
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - Phylogeny of eusocial Lasioglossum reveals multiple losses of eusociality within 
      a primitively eusocial clade of bees (Hymenoptera: Halictidae).
PG  - 23-36
AB  - We performed a phylogenetic analysis of the species, species groups, and
      subgenera within the predominantly eusocial lineage of Lasioglossum (the
      Hemihalictus series) based on three protein coding genes: mitochondrial
      cytochrome oxidase I, nuclear elongation factor 1alpha and long-wavelength
      rhodopsin. The entire data set consisted of 3421 aligned nucleotide sites, 854 of
      which were parsimony informative. Analyses by equal weights parsimony, maximum
      likelihood, and Bayesian methods yielded good resolution among the 53
      taxa/populations, with strong bootstrap support and high posterior probabilities 
      for most nodes. There was no significant incongruence among genes, and parsimony,
      maximum likelihood, and Bayesian methods yielded congruent results. We mapped
      social behavior onto the resulting tree for 42 of the taxa/populations to infer
      the likely history of social evolution within Lasioglossum. Our results indicate 
      that eusociality had a single origin within Lasioglossum. Within the
      predominantly eusocial clade, however, there have been multiple (six) reversals
      from eusociality to solitary nesting, social polymorphism, or social parasitism, 
      suggesting that these reversals may be more common in primitively eusocial
      Hymenoptera than previously anticipated. Our results support the view that
      eusociality is hard to evolve but easily lost. This conclusion is potentially
      important for understanding the early evolution of the advanced eusocial insects,
      such as ants, termites, and corbiculate bees.
FAU - Danforth, Bryan N
AU  - Danforth BN
AD  - Department of Entomology, Comstock Hall, Cornell University, Ithaca, New York
      14853-0901, USA. bnd1@cornell.edu
FAU - Conway, Lindsay
AU  - Conway L
FAU - Ji, Shuqing
AU  - Ji S
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bees/classification/*genetics
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - *Sociobiology
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - QKDHRNCHYDLR6VXB [pii]
AID - 10.1080/10635150390132687 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):23-36. doi: 10.1080/10635150390132687.

PMID- 12554436
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - Partitioned likelihood support and the evaluation of data set conflict.
PG  - 15-22
AB  - In simultaneous analyses of multiple data partitions, the trees relevant when
      measuring support for a clade are the optimal tree, and the best tree lacking the
      clade (i.e., the most reasonable alternative). The parsimony-based method of
      partitioned branch support (PBS) "forces" each data set to arbitrate between the 
      two relevant trees. This value is the amount each data set contributes to clade
      support in the combined analysis, and can be very different to support apparent
      in separate analyses. The approach used in PBS can also be employed in
      likelihood: a simultaneous analysis of all data retrieves the maximum likelihood 
      tree, and the best tree without the clade of interest is also found. Each data
      set is fitted to the two trees and the log-likelihood difference calculated,
      giving "partitioned likelihood support" (PLS) for each data set. These
      calculations can be performed regardless of the complexity of the ML model
      adopted. The significance of PLS can be evaluated using a variety of resampling
      methods, such as the Kishino-Hasegawa test, the Shimodiara-Hasegawa test, or
      likelihood weights, although the appropriateness and assumptions of these tests
      remains debated.
FAU - Lee, M S Y
AU  - Lee MS
AD  - Department of Environmental Biology, University of Adelaide, South Australia,
      Australia. lee.mike@saugov.sa.gov.au
FAU - Hugall, A F
AU  - Hugall AF
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Artiodactyla/genetics
MH  - *Data Interpretation, Statistical
MH  - Evolution, Molecular
MH  - *Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Statistics, Nonparametric
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - 9BGVMQKPTB8WW076 [pii]
AID - 10.1080/10635150390132650 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):15-22. doi: 10.1080/10635150390132650.

PMID- 12554435
OWN - NLM
STAT- MEDLINE
DCOM- 20030908
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 1
DP  - 2003 Feb
TI  - Molecular phylogenetics of allodapine bees, with implications for the evolution
      of sociality and progressive rearing.
PG  - 1-14
AB  - Allodapine bees have long been regarded as providing useful material for
      examining the origins of social behavior. Previous researchers have assumed that 
      sociality arose within the Allodapini and have linked the evolution of sociality 
      to a transition from mass provisioning to progressive provisioning of brood.
      Early phylogenetic studies of allodapines were based on morphological and
      life-history data, but critical aspects of these studies relied on small
      character sets, where the polarity and coding of characters is problematic. We
      used nucleotide sequence data from one nuclear and two mitochondrial gene
      fragments to examine phylogenetic structure among nine allodapine genera. Our
      data set comprised 1506 nucleotide positions, of which 402 were parsimony
      informative. Maximum parsimony, log determinant, and maximum likelihood analyses 
      produced highly similar phylogenetic topologies, and all analyses indicated that 
      the tropical African genus Macrogalea was the sister group to all other
      allodapines. This finding conflicts with that of previous studies, in which
      Compsomelissa + Halterapis formed the most basal group. Changing the basal node
      of the Allodapini has major consequences for understanding evolution in this
      tribe. Our results cast doubt on the previous hypotheses that progressive
      provisioning and castelike social behavior evolved among lineages leading to the 
      extant allodapine taxa. Instead, our results suggest that mass provisioning in
      Halterapis is a derived feature and that social behavior is an ancestral trait
      for all allodapine lineages. The forms of social behavior present in extant
      allodapines are likely to have resulted from a long evolutionary history, which
      may help explain the complexity of social traits found in many allodapine bees.
FAU - Schwarz, Michael P
AU  - Schwarz MP
AD  - School of Biological Sciences, Flinders University, GPO Box 2100, Adelaide, South
      Australia 5001, Australia.
FAU - Bull, Nicholas J
AU  - Bull NJ
FAU - Cooper, Steven J B
AU  - Cooper SJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Peptide Elongation Factor 1)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Africa
MH  - Animals
MH  - Australia
MH  - Base Composition/genetics
MH  - Base Sequence
MH  - Bees/classification/*genetics
MH  - Electron Transport Complex IV/genetics
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - Peptide Elongation Factor 1/genetics
MH  - *Phylogeny
MH  - *Sociobiology
EDAT- 2003/01/30 04:00
MHDA- 2003/09/10 05:00
CRDT- 2003/01/30 04:00
PHST- 2003/01/30 04:00 [pubmed]
PHST- 2003/09/10 05:00 [medline]
PHST- 2003/01/30 04:00 [entrez]
AID - 3BATNU3FRC2HMG55 [pii]
AID - 10.1080/10635150390132632 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Feb;52(1):1-14. doi: 10.1080/10635150390132632.

PMID- 12396595
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - New Zealand Kauri (Agathis australis (D.Don) Lindl., Araucariaceae) survives
      Oligocene drowning.
PG  - 827-32
FAU - Stockler, Karen
AU  - Stockler K
AD  - The Allan Wilson Centre for Molecular Biology and Evolution, Institute of
      Molecular BioSciences, Massey University, Palmerston North, New Zealand.
FAU - Daniel, Ian L
AU  - Daniel IL
FAU - Lockhart, Peter J
AU  - Lockhart PJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Disasters
MH  - Environment
MH  - Extinction, Psychological
MH  - Fossils
MH  - Geography
MH  - New Zealand
MH  - *Paleontology
MH  - Phylogeny
MH  - *Plants
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102474 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):827-32. doi: 10.1080/10635150290102474.

PMID- 12396594
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - Use of well-known names in phylogenetic nomenclature: a reply to Laurin.
PG  - 822-7
FAU - Anderson, Jason S
AU  - Anderson JS
AD  - Department of Biology, University of Toronto, 3359 Mississauga Road, Mississauga,
      Ontario L5L 1C6, Canada. jsanders@utm.utoronto.ca
LA  - eng
PT  - Comment
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2002 Apr;51(2):364-9. PMID: 12028737
MH  - Algorithms
MH  - Amphibians/*classification/*genetics
MH  - Animals
MH  - Computational Biology/methods
MH  - Evolution, Molecular
MH  - Phylogeny
MH  - Terminology as Topic
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102447 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):822-7. doi: 10.1080/10635150290102447.

PMID- 12396593
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20031114
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - Growth and shape: measurements and metrics.
PG  - 817-22
FAU - Sheets, H David
AU  - Sheets HD
AD  - Department of Physics, Canisius College, Buffalo, New York 14208, USA.
      sheets@canisius.edu
FAU - Zelditch, Miriam L
AU  - Zelditch ML
FAU - Swiderski, Donald L
AU  - Swiderski DL
LA  - eng
PT  - Comment
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2001 Sep-Oct;50(5):640-56. PMID: 12116936
MH  - Analysis of Variance
MH  - Animals
MH  - Biometry
MH  - Body Constitution
MH  - *Growth
MH  - *Models, Biological
MH  - Models, Theoretical
MH  - Phylogeny
MH  - Turtles/anatomy &amp; histology/*growth &amp; development
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102438 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):817-22. doi: 10.1080/10635150290102438.

PMID- 12396592
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - An optimality criterion to determine areas of endemism.
PG  - 806-16
AB  - A formal method was developed to determine areas of endemism. The study region is
      divided into cells, and the number of species that can be considered as endemic
      is counted for a given set of cells (= area). Thus, the areas with the maximum
      number of species considered endemic are preferred. This is the first method for 
      the identification of areas of endemism that implements an optimality criterion
      directly based on considering the aspects of species distribution that are
      relevant to endemism. The method is implemented in two computer programs, NDM and
      VNDM, available from the authors.
FAU - Szumik, Claudia A
AU  - Szumik CA
AD  - Consejo Nacional de Investigaciones Cientificas y Tecnicas, Miguel Lillo 205,
      4000 San Miguel de Tucuman, Tucuman, Argentina.
FAU - Cuezzo, Fabiana
AU  - Cuezzo F
FAU - Goloboff, Pablo A
AU  - Goloboff PA
FAU - Chalup, Adriana E
AU  - Chalup AE
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Demography
MH  - *Ecosystem
MH  - Geography
MH  - *Models, Theoretical
MH  - Species Specificity
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102483 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):806-16. doi: 10.1080/10635150290102483.

PMID- 12396591
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - Phylogeny of the Oriental Drosophila melanogaster species group: a multilocus
      reconstruction.
PG  - 786-805
AB  - The melanogaster species group of Drosophila (subgenus Sophophora) has long been 
      a favored model for evolutionary studies because of its morphological and
      ecological diversity and wide geographic distribution. However, phylogenetic
      relationships among species and subgroups within this lineage are not well
      understood. We reconstructed the phylogeny of 17 species representing 7
      "oriental" species subgroups, which are especially closely related to D.
      melanogaster. We used DNA sequences of four nuclear and two mitochondrial loci in
      an attempt to obtain the best possible estimate of species phylogeny and to
      assess the extent and sources of remaining uncertainties. Comparison of trees
      derived from single-gene data sets allowed us to identify several strongly
      supported clades, which were also consistently seen in combined analyses. The
      relationships among these clades are less certain. The combined data set contains
      data partitions that are incongruent with each other. Trees reconstructed from
      the combined data set and from internally homogenous data sets consisting of
      three or four genes each differ at several deep nodes. The total data set tree is
      fully resolved and strongly supported at most nodes. Statistical tests indicated 
      that this tree is compatible with all individual and combined data sets.
      Therefore, we accepted this tree as the most likely model of historical
      relationships. We compared the new molecular phylogeny to earlier estimates based
      on morphology and chromosome structure and discuss its taxonomic and evolutionary
      implications.
FAU - Kopp, Artyom
AU  - Kopp A
AD  - Howard Hughes Medical Institute and Laboratory of Molecular Biology, University
      of Wisconsin, Madison, Wisconsin 53706, USA. akopp@facstaff.wisc.edu
FAU - True, John R
AU  - True JR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Cell Nucleus/metabolism
MH  - DNA, Mitochondrial/metabolism
MH  - Drosophila melanogaster/classification/*genetics
MH  - Evolution, Molecular
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102410 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):786-805. doi: 10.1080/10635150290102410.

PMID- 12396590
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - Molecular and morphological phylogenetics of weevils (coleoptera,
      curculionoidea): do niche shifts accompany diversification?
PG  - 761-85
AB  - The main goals of this study were to provide a robust phylogeny for the families 
      of the superfamily Curculionoidea, to discover relationships and major natural
      groups within the family Curculionidae, and to clarify the evolution of larval
      habits and host-plant associations in weevils to analyze their role in weevil
      diversification. Phylogenetic relationships among the weevils (Curculionoidea)
      were inferred from analysis of nucleotide sequences of 18S ribosomal DNA (rDNA;
      approximately 2,000 bases) and 115 morphological characters of larval and adult
      stages. A worldwide sample of 100 species was compiled to maximize representation
      of weevil morphological and ecological diversity. All families and the main
      subfamilies of Curculionoidea were represented. The family Curculionidae sensu
      lato was represented by about 80 species in 30 "subfamilies" of traditional
      classifications. Phylogenetic reconstruction was accomplished by parsimony
      analysis of separate and combined molecular and morphological data matrices and
      Bayesian analysis of the molecular data; tree topology support was evaluated.
      Results of the combined analysis of 18S rDNA and morphological data indicate that
      monophyly of and relationships among each of the weevil families are well
      supported with the topology ((Nemonychidae, Anthribidae) (Belidae (Attelabidae
      (Caridae (Brentidae, Curculionidae))))). Within the clade Curculionidae sensu
      lato, the basal positions are occupied by mostly monocot-associated taxa with the
      primitive type of male genitalia followed by the Curculionidae sensu stricto,
      which is made up of groups with the derived type of male genitalia. High support 
      values were found for the monophyly of some distinct curculionid groups such as
      Dryophthorinae (several tribes represented) and Platypodinae (Tesserocerini plus 
      Platypodini), among others. However, the subfamilial relationships in
      Curculionidae are unresolved or weakly supported. The phylogeny estimate based on
      combined 18S rDNA and morphological data suggests that diversification in weevils
      was accompanied by niche shifts in host-plant associations and larval habits.
      Pronounced conservatism is evident in larval feeding habits, particularly in the 
      host tissue consumed. Multiple shifts to use of angiosperms in Curculionoidea
      were identified, each time associated with increases in weevil diversity and
      subsequent shifts back to gymnosperms, particularly in the Curculionidae.
FAU - Marvaldi, Adriana E
AU  - Marvaldi AE
AD  - Museum of Comparative Zoology, Harvard University, 26 Oxford Street, Cambridge,
      Massachusetts 02138, USA. marvaldi@lab.cricyt.edu.ar
FAU - Sequeira, Andrea S
AU  - Sequeira AS
FAU - O'Brien, Charles W
AU  - O'Brien CW
FAU - Farrell, Brian D
AU  - Farrell BD
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 18S)
SB  - IM
MH  - Animals
MH  - Coleoptera
MH  - DNA, Ribosomal/*genetics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - RNA, Ribosomal, 18S/genetics
MH  - Sequence Analysis, DNA
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102465 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):761-85. doi: 10.1080/10635150290102465.

PMID- 12396589
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - Identifiability of parameters in MCMC Bayesian inference of phylogeny.
PG  - 754-60
AB  - Methods for Bayesian inference of phylogeny using DNA sequences based on Markov
      chain Monte Carlo (MCMC) techniques allow the incorporation of arbitrarily
      complex models of the DNA substitution process, and other aspects of evolution.
      This has increased the realism of models, potentially improving the accuracy of
      the methods, and is largely responsible for their recent popularity. Another
      consequence of the increased complexity of models in Bayesian phylogenetics is
      that these models have, in several cases, become overparameterized. In such
      cases, some parameters of the model are not identifiable; different combinations 
      of nonidentifiable parameters lead to the same likelihood, making it impossible
      to decide among the potential parameter values based on the data.
      Overparameterized models can also slow the rate of convergence of MCMC algorithms
      due to large negative correlations among parameters in the posterior probability 
      distribution. Functions of parameters can sometimes be found, in
      overparameterized models, that are identifiable, and inferences based on these
      functions are legitimate. Examples are presented of overparameterized models that
      have been proposed in the context of several Bayesian methods for inferring the
      relative ages of nodes in a phylogeny when the substitution rate evolves over
      time.
FAU - Rannala, Bruce
AU  - Rannala B
AD  - Department of Medical Genetics, University of Alberta, Edmonton, Alberta, T6G
      2H7, Canada.
LA  - eng
GR  - HG01988/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - Evolution, Molecular
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102429 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):754-60. doi: 10.1080/10635150290102429.

PMID- 12396588
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - An examination of the monophyly of morning glory taxa using Bayesian phylogenetic
      inference.
PG  - 740-53
AB  - The objective of this study was to obtain a quantitative assessment of the
      monophyly of morning glory taxa, specifically the genus Ipomoea and the tribe
      Argyreieae. Previous systematic studies of morning glories intimated the
      paraphyly of Ipomoea by suggesting that the genera within the tribe Argyreieae
      are derived from within Ipomoea; however, no quantitative estimates of
      statistical support were developed to address these questions. We applied a
      Bayesian analysis to provide quantitative estimates of monophyly in an
      investigation of morning glory relationships using DNA sequence data. We also
      explored various approaches for examining convergence of the Markov chain Monte
      Carlo (MCMC) simulation of the Bayesian analysis by running 18 separate analyses 
      varying in length. We found convergence of the important components of the
      phylogenetic model (the tree with the maximum posterior probability, branch
      lengths, the parameter values from the DNA substitution model, and the posterior 
      probabilities for clade support) for these data after one million generations of 
      the MCMC simulations. In the process, we identified a run where the parameter
      values obtained were often outside the range of values obtained from the other
      runs, suggesting an aberrant result. In addition, we compared the Bayesian method
      of phylogenetic analysis to maximum likelihood and maximum parsimony. The results
      from the Bayesian analysis and the maximum likelihood analysis were similar for
      topology, branch lengths, and parameters of the DNA substitution model.
      Topologies also were similar in the comparison between the Bayesian analysis and 
      maximum parsimony, although the posterior probabilities and the bootstrap
      proportions exhibited some striking differences. In a Bayesian analysis of three 
      data sets (ITS sequences, waxy sequences, and ITS + waxy sequences) no supoort
      for the monophyly of the genus Ipomoea, or for the tribe Argyreieae, was
      observed, with the estimate of the probability of the monophyly of these taxa
      being less than 3.4 x 10(-7).
FAU - Miller, Richard E
AU  - Miller RE
AD  - Department of Biology, Duke University, Durham, North Carolina 27708, USA.
FAU - Buckley, Thomas R
AU  - Buckley TR
FAU - Manos, Paul S
AU  - Manos PS
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Bayes Theorem
MH  - DNA/genetics
MH  - Genes, Plant
MH  - Ipomoea/*genetics
MH  - Likelihood Functions
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102401 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):740-53. doi: 10.1080/10635150290102401.

PMID- 12396587
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - Mapping mutations on phylogenies.
PG  - 729-39
AB  - Mapping of mutations on a phylogeny has been a commonly used analytical tool in
      phylogenetics and molecular evolution. However, the common approaches for mapping
      mutations based on parsimony have lacked a solid statistical foundation. Here, I 
      present a Bayesian method for mapping mutations on a phylogeny. I illustrate some
      of the common problems associated with using parsimony and suggest instead that
      inferences in molecular evolution can be made on the basis of the posterior
      distribution of the mappings of mutations. A method for simulating a mapping from
      the posterior distribution of mappings is also presented, and the utility of the 
      method is illustrated on two previously published data sets. Applications include
      a method for testing for variation in the substitution rate along the sequence
      and a method for testing whether the d(N)/d(S) ratio varies among lineages in the
      phylogeny.
FAU - Nielsen, Rasmus
AU  - Nielsen R
AD  - Department of Biological Statistics, Cornell University, 439 Warren Hall, Ithaca,
      New York 14853-7801, USA. rn28@cornell.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Chromosome Mapping
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - Monte Carlo Method
MH  - *Mutation
MH  - *Phylogeny
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102393 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):729-39. doi: 10.1080/10635150290102393.

PMID- 12396586
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20071114
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - Oh brother, where art thou? A Bayes factor test for recombination with uncertain 
      heritage.
PG  - 715-28
AB  - Current methods to identify recombination between subtypes of human
      immunodeficiency virus 1 (HIV-1) fall into a sequential testing trap, in which
      significance is assessed conditional on parental representative sequences and
      crossover points (COPs) that maximize the same test statistic. We overcame this
      shortfall by testing for recombination while inferring parental heritage and COPs
      using an extended Bayesian multiple change-point model. The model assumes that
      aligned molecular sequence data consist of an unknown number of contiguous
      segments that may support alternative topologies or varying evolutionary
      pressures. We allowed for heterogeneity in the substitution process and
      specifically tested for intersubtype recombination using Bayes factors. We also
      developed a new class of priors to assess significance across a wide range of
      support for recombination in the data. We applied our method to three putative
      gag gene recombinants. HIV-1 isolate RW024 decisively supported recombination
      with an inferred parental heritage of AD and a COP 95% Bayesian credible interval
      of (1,152, 1,178) using the HXB2 numbering scheme. HIV-1 isolate VI557 barely
      supported recombination. HIV-1 isolate RF decisively rejected recombination as
      expected, given that the sequence is commonly used as a reference sequence for
      subtype B. We employed scaled regeneration quantile plots to assess convergence
      and found this approach convenient to use even for our variable dimensional model
      parameter space.
FAU - Suchard, Marc A
AU  - Suchard MA
AD  - Department of Biomathematics, School of Medicine, University of California, Los
      Angeles, California 90095-1766, USA.
FAU - Weiss, Robert E
AU  - Weiss RE
FAU - Dorman, Karin S
AU  - Dorman KS
FAU - Sinsheimer, Janet S
AU  - Sinsheimer JS
LA  - eng
GR  - CA16042/CA/NCI NIH HHS/United States
GR  - GM08042/GM/NIGMS NIH HHS/United States
GR  - GM08185/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Gene Products, gag)
SB  - IM
MH  - Algorithms
MH  - Bayes Theorem
MH  - Gene Products, gag/genetics
MH  - HIV-1/genetics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - *Recombination, Genetic
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102384 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):715-28. doi: 10.1080/10635150290102384.

PMID- 12396585
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - Effects of models of rate evolution on estimation of divergence dates with
      special reference to the metazoan 18S ribosomal RNA phylogeny.
PG  - 703-14
AB  - The molecular clock, i.e., constancy of the rate of evolution over time, is
      commonly assumed in estimating divergence dates. However, this assumption is
      often violated and has drastic effects on date estimation. Recently, a number of 
      attempts have been made to relax the clock assumption. One approach is to use
      maximum likelihood, which assigns rates to branches and allows the estimation of 
      both rates and times. An alternative is the Bayes approach, which models the
      change of the rate over time. A number of models of rate change have been
      proposed. We have extended and evaluated models of rate evolution, i.e., the
      lognormal and its recent variant, along with the gamma, the exponential, and the 
      Ornstein-Uhlenbeck processes. These models were first applied to a small hominoid
      data set, where an empirical Bayes approach was used to estimate the
      hyperparameters that measure the amount of rate variation. Estimation of
      divergence times was sensitive to these hyperparameters, especially when the
      assumed model is close to the clock assumption. The rate and date estimates
      varied little from model to model, although the posterior Bayes factor indicated 
      the Ornstein-Uhlenbeck process outperformed the other models. To demonstrate the 
      importance of allowing for rate change across lineages, this general approach was
      used to analyze a larger data set consisting of the 18S ribosomal RNA gene of 39 
      metazoan species. We obtained date estimates consistent with paleontological
      records, the deepest split within the group being about 560 million years ago.
      Estimates of the rates were in accordance with the Cambrian explosion hypothesis 
      and suggested some more recent lineage-specific bursts of evolution.
FAU - Aris-Brosou, Stephane
AU  - Aris-Brosou S
AD  - Department of Biology, University College London, Darwin Building, Gower Street, 
      London WC1E 6BT, England.
FAU - Yang, Ziheng
AU  - Yang Z
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 18S)
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - Invertebrates/*genetics
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - Phylogeny
MH  - RNA, Ribosomal, 18S/*genetics
MH  - Time Factors
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102375 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):703-14. doi: 10.1080/10635150290102375.

PMID- 12396584
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - Divergence time and evolutionary rate estimation with multilocus data.
PG  - 689-702
AB  - Bayesian methods for estimating evolutionary divergence times are extended to
      multigene data sets, and a technique is described for detecting correlated
      changes in evolutionary rates among genes. Simulations are employed to explore
      the effect of multigene data on divergence time estimation, and the methodology
      is illustrated with a previously published data set representing diverse plant
      taxa. The fact that evolutionary rates and times are confounded when sequence
      data are compared is emphasized and the importance of fossil information for
      disentangling rates and times is stressed.
FAU - Thorne, Jeffrey L
AU  - Thorne JL
AD  - Bioinformatics Research Center, Box 7566, North Carolina State University,
      Raleigh, North Carolina 27695-7566, USA. thorne@statgen.nscu.edu
FAU - Kishino, Hirohisa
AU  - Kishino H
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - Evolution, Molecular
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Plants/genetics
MH  - Time Factors
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102456 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):689-702. doi: 10.1080/10635150290102456.

PMID- 12396583
OWN - NLM
STAT- MEDLINE
DCOM- 20030513
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 5
DP  - 2002 Oct
TI  - Potential applications and pitfalls of Bayesian inference of phylogeny.
PG  - 673-88
AB  - Only recently has Bayesian inference of phylogeny been proposed. The method is
      now a practical alternative to the other methods; indeed, the method appears to
      possess advantages over the other methods in terms of ability to use complex
      models of evolution, ease of interpretation of the results, and computational
      efficiency. However, the method should be used cautiously. The results of a
      Bayesian analysis should be examined with respect to the sensitivity of the
      results to the priors used and the reliability of the Markov chain Monte Carlo
      approximation of the probabilities of trees.
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
AD  - Section of Ecology, Behavior and Evolution, Division of Biology, University of
      California-San Diego, La Jolla, CA 92093-0116, USA.
FAU - Larget, Bret
AU  - Larget B
FAU - Miller, Richard E
AU  - Miller RE
FAU - Ronquist, Fredrik
AU  - Ronquist F
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Genetics
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
RF  - 65
EDAT- 2002/10/25 04:00
MHDA- 2003/05/14 05:00
CRDT- 2002/10/25 04:00
PHST- 2002/10/25 04:00 [pubmed]
PHST- 2003/05/14 05:00 [medline]
PHST- 2002/10/25 04:00 [entrez]
AID - 10.1080/10635150290102366 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Oct;51(5):673-88. doi: 10.1080/10635150290102366.

PMID- 12228008
OWN - NLM
STAT- MEDLINE
DCOM- 20021024
LR  - 20190108
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 4
DP  - 2002 Aug
TI  - Increased taxon sampling is advantageous for phylogenetic inference.
PG  - 664-71
FAU - Pollock, David D
AU  - Pollock DD
AD  - Department of Biological Sciences and Biological Computation and Visualization
      Center, Louisiana State University, Baton Rouge, Louisiana 70803, USA.
      dpollock@lsu.edu
FAU - Zwickl, Derrick J
AU  - Zwickl DJ
FAU - McGuire, Jimmy A
AU  - McGuire JA
FAU - Hillis, David M
AU  - Hillis DM
LA  - eng
GR  - R21 GM065612/GM/NIGMS NIH HHS/United States
GR  - R21 GM065612-01/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Likelihood Functions
MH  - Mammals
MH  - *Phylogeny
PMC - PMC2943957
MID - NIHMS194775
EDAT- 2002/09/14 10:00
MHDA- 2002/10/31 04:00
CRDT- 2002/09/14 10:00
PHST- 2002/09/14 10:00 [pubmed]
PHST- 2002/10/31 04:00 [medline]
PHST- 2002/09/14 10:00 [entrez]
AID - 10.1080/10635150290102357 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Aug;51(4):664-71. doi: 10.1080/10635150290102357.

PMID- 12228007
OWN - NLM
STAT- MEDLINE
DCOM- 20021024
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 4
DP  - 2002 Aug
TI  - Resolution of a supertree/supermatrix paradox.
PG  - 652-64
FAU - Gatesy, John
AU  - Gatesy J
AD  - Department of Biology, University of California, Riverside, California 92521,
      USA.
FAU - Matthee, Conrad
AU  - Matthee C
FAU - DeSalle, Rob
AU  - DeSalle R
FAU - Hayashi, Cheryl
AU  - Hayashi C
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2003 Oct;52(5):724-9. PMID: 14530139
MH  - Animals
MH  - Artiodactyla/classification
MH  - *Phylogeny
EDAT- 2002/09/14 10:00
MHDA- 2002/10/31 04:00
CRDT- 2002/09/14 10:00
PHST- 2002/09/14 10:00 [pubmed]
PHST- 2002/10/31 04:00 [medline]
PHST- 2002/09/14 10:00 [entrez]
AID - 10.1080/10635150290102311 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Aug;51(4):652-64. doi: 10.1080/10635150290102311.

PMID- 12228006
OWN - NLM
STAT- MEDLINE
DCOM- 20021024
LR  - 20020913
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 4
DP  - 2002 Aug
TI  - Units in biogeography.
PG  - 648-52
FAU - Hausdorf, Bernhard
AU  - Hausdorf B
AD  - Zoologisches Institut und Zoologisches Museum der Universitat Hamburg,
      Martin-Luther-King-Platz 3, D-20146 Hamburg, Germany.
      hausdorf@zoologie.uni-hamburg.de
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Geography
MH  - *Phylogeny
EDAT- 2002/09/14 10:00
MHDA- 2002/10/31 04:00
CRDT- 2002/09/14 10:00
PHST- 2002/09/14 10:00 [pubmed]
PHST- 2002/10/31 04:00 [medline]
PHST- 2002/09/14 10:00 [entrez]
AID - 10.1080/10635150290102320 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Aug;51(4):648-52. doi: 10.1080/10635150290102320.

PMID- 12228005
OWN - NLM
STAT- MEDLINE
DCOM- 20021024
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 4
DP  - 2002 Aug
TI  - Phylogeny reconstruction and functional constraints in organellar genomes:
      plastid atpB and rbcL sequences versus animal mitochondrion.
PG  - 638-47
FAU - Savolainen, Vincent
AU  - Savolainen V
AD  - Molecular Systematics Section, Jodrell Laboratory, Royal Botanic Gardens, Kew,
      Richmond, TW9 3DS, UK. vsavolainen@kew.org
FAU - Chase, Mark W
AU  - Chase MW
FAU - Salamin, Nicolas
AU  - Salamin N
FAU - Soltis, Douglas E
AU  - Soltis DE
FAU - Soltis, Pamela S
AU  - Soltis PS
FAU - Lopez, Andres J
AU  - Lopez AJ
FAU - Fedrigo, Olivier
AU  - Fedrigo O
FAU - Naylor, Gavin J P
AU  - Naylor GJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
RN  - EC 4.1.1.39 (RbcL protein, plastid)
RN  - EC 4.1.1.39 (Ribulose-Bisphosphate Carboxylase)
SB  - IM
MH  - Animals
MH  - Codon
MH  - *Genome
MH  - Mitochondria/*genetics
MH  - *Phylogeny
MH  - Plants/genetics
MH  - Plastids/*genetics
MH  - Ribulose-Bisphosphate Carboxylase/*genetics
EDAT- 2002/09/14 10:00
MHDA- 2002/10/31 04:00
CRDT- 2002/09/14 10:00
PHST- 2002/09/14 10:00 [pubmed]
PHST- 2002/10/31 04:00 [medline]
PHST- 2002/09/14 10:00 [entrez]
AID - 10.1080/10635150290102348 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Aug;51(4):638-47. doi: 10.1080/10635150290102348.

PMID- 12228004
OWN - NLM
STAT- MEDLINE
DCOM- 20021024
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 4
DP  - 2002 Aug
TI  - The utility of the incongruence length difference test.
PG  - 625-37
FAU - Barker, F Keith
AU  - Barker FK
AD  - Department of Ornithology, American Museum of Natural History, Central Park West 
      at 79th Street, New York, New York 10024, USA. fbarker@amnh.org
FAU - Lutzoni, Francois M
AU  - Lutzoni FM
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2002/09/14 10:00
MHDA- 2002/10/31 04:00
CRDT- 2002/09/14 10:00
PHST- 2002/09/14 10:00 [pubmed]
PHST- 2002/10/31 04:00 [medline]
PHST- 2002/09/14 10:00 [entrez]
AID - 10.1080/10635150290102302 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Aug;51(4):625-37. doi: 10.1080/10635150290102302.

PMID- 12228003
OWN - NLM
STAT- MEDLINE
DCOM- 20021024
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 4
DP  - 2002 Aug
TI  - Examining Basal avian divergences with mitochondrial sequences: model complexity,
      taxon sampling, and sequence length.
PG  - 614-25
FAU - Braun, Edward L
AU  - Braun EL
AD  - Department of Zoology, 223 Bartram Hall, P.O. Box 118525, University of Florida, 
      Gainesville, Florida 32611-8525, USA. ebraun@zoo.ufl.edu
FAU - Kimball, Rebecca T
AU  - Kimball RT
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Birds/*classification/genetics
MH  - DNA, Mitochondrial/*genetics
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2002/09/14 10:00
MHDA- 2002/10/31 04:00
CRDT- 2002/09/14 10:00
PHST- 2002/09/14 10:00 [pubmed]
PHST- 2002/10/31 04:00 [medline]
PHST- 2002/09/14 10:00 [entrez]
AID - 10.1080/10635150290102294 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Aug;51(4):614-25. doi: 10.1080/10635150290102294.

PMID- 12228002
OWN - NLM
STAT- MEDLINE
DCOM- 20021024
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 4
DP  - 2002 Aug
TI  - A genomic schism in birds revealed by phylogenetic analysis of DNA strings.
PG  - 599-613
AB  - The molecular systematics of vertebrates has been based entirely on alignments of
      primary structures of macromolecules; however, higher order features of DNA
      sequences not used in traditional studies also contain valuable phylogenetic
      information. Recent molecular data sets conflict over the phylogenetic placement 
      of flightless birds (ratites - paleognaths), but placement of this clade
      critically influences interpretation of character change in birds. To help
      resolve this issue, we applied a new bioinformatics approach to the largest
      molecular data set currently available. We distilled nearly one megabase (1
      million base pairs) of heterogeneous avian genomic DNA from 20 birds and an
      alligator into genomic signatures, defined as the complete set of frequencies of 
      short sequence motifs (strings), thereby providing a way to directly compare
      higher order features of nonhomologous DNA sequences. Phylogenetic analysis and
      principal component analysis of the signatures strongly support the traditional
      hypothesis of basal ratites and monophyly of the nonratite birds (neognaths) and 
      imply that ratite genomes are linguistically primitive within birds, despite
      their base compositional similarity to neognath genomes. Our analyses show
      further that the phylogenetic signal of genomic signatures are strongest among
      deep splits within vertebrates. Despite clear problems with phylogenetic analysis
      of genomic signatures, our study raises intriguing issues about the biological
      and genomic differences that fundamentally differentiate paleognaths and
      neognaths.
FAU - Edwards, Scott V
AU  - Edwards SV
AD  - Department of Zoology and Burke Museum, University of Washington, Box 351800,
      Seattle, Washington 98195, USA. sedwards@u.washington.edu
FAU - Fertil, Bernard
AU  - Fertil B
FAU - Giron, Alain
AU  - Giron A
FAU - Deschavanne, Patrick J
AU  - Deschavanne PJ
LA  - eng
SI  - GENBANK/AB000967
SI  - GENBANK/AB001981
SI  - GENBANK/AB005529
SI  - GENBANK/AB005531
SI  - GENBANK/AB005532
SI  - GENBANK/AB005533
SI  - GENBANK/AB005912
SI  - GENBANK/AB006694
SI  - GENBANK/AB006695
SI  - GENBANK/AB006754
SI  - GENBANK/AB007195
SI  - GENBANK/AB017036
SI  - GENBANK/AB017906
SI  - GENBANK/AB019555
SI  - GENBANK/AB022344
SI  - GENBANK/AB024279
SI  - GENBANK/AB029075
SI  - GENBANK/AB030749
SI  - GENBANK/AF006002
SI  - GENBANK/AF018266
SI  - GENBANK/AF018267
SI  - GENBANK/AF030997
SI  - GENBANK/AF033106
SI  - GENBANK/AF033107
SI  - GENBANK/AF039749
SI  - GENBANK/AF077830
SI  - GENBANK/AF082667
SI  - GENBANK/AF105022
SI  - GENBANK/AF113513
SI  - GENBANK/AF137264
SI  - GENBANK/AF139128
SI  - GENBANK/AF143724
SI  - GENBANK/AF143727
SI  - GENBANK/AF143730
SI  - GENBANK/AF143732
SI  - GENBANK/AF143738
SI  - GENBANK/AF170972
SI  - GENBANK/AF173605
SI  - GENBANK/AF173606
SI  - GENBANK/AF173607
SI  - GENBANK/AF173608
SI  - GENBANK/AF173610
SI  - GENBANK/AF173612
SI  - GENBANK/AF173630
SI  - GENBANK/AF173632
SI  - GENBANK/AF189778
SI  - GENBANK/AF205032
SI  - GENBANK/AF231111
SI  - GENBANK/AF246975
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Birds/classification/*genetics
MH  - DNA/*genetics
MH  - *Genome
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2002/09/14 10:00
MHDA- 2002/10/31 04:00
CRDT- 2002/09/14 10:00
PHST- 2002/09/14 10:00 [pubmed]
PHST- 2002/10/31 04:00 [medline]
PHST- 2002/09/14 10:00 [entrez]
AID - 10.1080/10635150290102285 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Aug;51(4):599-613. doi: 10.1080/10635150290102285.

PMID- 12228001
OWN - NLM
STAT- MEDLINE
DCOM- 20021024
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 4
DP  - 2002 Aug
TI  - Increased taxon sampling greatly reduces phylogenetic error.
PG  - 588-98
AB  - Several authors have argued recently that extensive taxon sampling has a positive
      and important effect on the accuracy of phylogenetic estimates. However, other
      authors have argued that there is little benefit of extensive taxon sampling, and
      so phylogenetic problems can or should be reduced to a few exemplar taxa as a
      means of reducing the computational complexity of the phylogenetic analysis. In
      this paper we examined five aspects of study design that may have led to these
      different perspectives. First, we considered the measurement of phylogenetic
      error across a wide range of taxon sample sizes, and conclude that the expected
      error based on randomly selecting trees (which varies by taxon sample size) must 
      be considered in evaluating error in studies of the effects of taxon sampling.
      Second, we addressed the scope of the phylogenetic problems defined by different 
      samples of taxa, and argue that phylogenetic scope needs to be considered in
      evaluating the importance of taxon-sampling strategies. Third, we examined the
      claim that fast and simple tree searches are as effective as more thorough
      searches at finding near-optimal trees that minimize error. We show that a more
      complete search of tree space reduces phylogenetic error, especially as the taxon
      sample size increases. Fourth, we examined the effects of simple versus complex
      simulation models on taxonomic sampling studies. Although benefits of taxon
      sampling are apparent for all models, data generated under more complex models of
      evolution produce higher overall levels of error and show greater positive
      effects of increased taxon sampling. Fifth, we asked if different phylogenetic
      optimality criteria show different effects of taxon sampling. Although we found
      strong differences in effectiveness of different optimality criteria as a
      function of taxon sample size, increased taxon sampling improved the results from
      all the common optimality criteria. Nonetheless, the method that showed the
      lowest overall performance (minimum evolution) also showed the least improvement 
      from increased taxon sampling. Taking each of these results into account
      re-enforces the conclusion that increased sampling of taxa is one of the most
      important ways to increase overall phylogenetic accuracy.
FAU - Zwickl, Derrick J
AU  - Zwickl DJ
AD  - Section of Integrative Biology and Center for Computational Biology and
      Bioinformatics, University of Texas, Austin, Texas 78712, USA.
      zwickl@mail.utexas.edu
FAU - Hillis, David M
AU  - Hillis DM
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - *Research Design
EDAT- 2002/09/14 10:00
MHDA- 2002/10/31 04:00
CRDT- 2002/09/14 10:00
PHST- 2002/09/14 10:00 [pubmed]
PHST- 2002/10/31 04:00 [medline]
PHST- 2002/09/14 10:00 [entrez]
AID - 10.1080/10635150290102339 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Aug;51(4):588-98. doi: 10.1080/10635150290102339.

PMID- 12228000
OWN - NLM
STAT- MEDLINE
DCOM- 20021024
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 4
DP  - 2002 Aug
TI  - Perils of paralogy: using HSP70 genes for inferring organismal phylogenies.
PG  - 570-87
AB  - Conserved genes have found their way into the mainstream of molecular
      systematics. Many of these genes are members of multigene families. A difficulty 
      with using single genes of multigene families for phylogenetic inference is that 
      genes from one species may be paralogous to those from another taxon. We focus
      attention on this problem using heat shock 70 (HSP70) genes. Using polymerase
      chain reaction techniques with genomic DNA, we isolated and sequenced 123
      distinct sequences from 12 species of sharks. Phylogenetic analysis indicated
      that the sequences cluster with constituitively expressed cytoplasmic heat
      shock-like genes. Three highly divergent gene clades were sampled. A number of
      similar sequences were sampled from each species within each distinct gene clade.
      Comparison of published species trees with an HSP70 gene tree inferred using
      Bayesian phylogenetic analysis revealed several cases of gene duplication and
      differential sorting of gene lineages within this group of sharks. Gene tree
      parsimony based on the objective criteria of duplication and losses showed that
      previously published hypotheses of species relationships and two novel hypothesis
      based on Bayesian phylogenetics were concordant with the history of HSP70 gene
      duplication and loss. By contrast, two published hypotheses based on
      morphological data were not significantly different from the null hypothesis of a
      random association between species relatedness and the HSP70 gene tree. These
      results suggest that gene tree parsimony using data from multigene families can
      be used for inferring species relationships or testing published alternative
      hypotheses. More importantly, the results suggest that systematic studies relying
      on phylogenetic inferences from HSP70 genes may by plagued by unrecognized
      paralogy of sampled genes. Our results underscore the distinction between gene
      and species trees and highlight an underappreciated source of discordance between
      gene trees and organismal phylogeny, i.e., unrecognized paralogy of sampled
      genes.
FAU - Martin, Andrew P
AU  - Martin AP
AD  - Department of EPO Biology, University of Colorado, Boulder, Colorado 80309, USA. 
      am@stripe.colorado.edu
FAU - Burg, Theresa M
AU  - Burg TM
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (HSP70 Heat-Shock Proteins)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - HSP70 Heat-Shock Proteins/*genetics
MH  - *Phylogeny
MH  - Sharks/classification
MH  - Species Specificity
EDAT- 2002/09/14 10:00
MHDA- 2002/10/31 04:00
CRDT- 2002/09/14 10:00
PHST- 2002/09/14 10:00 [pubmed]
PHST- 2002/10/31 04:00 [medline]
PHST- 2002/09/14 10:00 [entrez]
AID - 10.1080/10635150290069995 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Aug;51(4):570-87. doi: 10.1080/10635150290069995.

PMID- 12227999
OWN - NLM
STAT- MEDLINE
DCOM- 20021024
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 4
DP  - 2002 Aug
TI  - Analysis of color spectra in comparative evolutionary studies: molecular
      phylogeny and habitat adaptation in the St. Vincent Anole (Anolis trinitatis).
PG  - 554-69
AB  - The use of color (as distinct from color pattern) in comparative evolutionary
      studies is important, and objective, independent characters are needed. A new
      method was employed to investigate geographic color variation in the small
      arboreal lizard Anolis trinitatis on the island of St. Vincent. The simple delta 
      analysis (based on the difference between eigenvector coefficients for adjacent
      regions of the spectrum) is aimed at increasing the objectivity with which a
      spectrum is cut into independent segments and does not predetermine segment width
      or number. There are distinct habitat types within this small island and distinct
      phylogenetic lineages (based on a kilobase of cytochrome b sequence) within this 
      species. A series of matrix correspondence (Mantel) tests indicate that aspects
      of color are associated with habitat type (e.g., green dorsum in rain forest
      lizards), molecular phylogeny, or both. Hence, both adaptation by selection and
      historical processes are implicated as causes of geographic variation in color.
      The dewlap variation (e.g., strong ultraviolet reflectance in some Atlantic
      coastal sites) is very pronounced and, contrary to some expectations, may result 
      in reproductive isolation even within small Lesser Antillean islands.
FAU - Thorpe, Roger S
AU  - Thorpe RS
AD  - School of Biological Sciences, University of Wales Bangor, Bangor, Gwynedd LL57
      2UW, UK. r.s.thorpe@bangor.ac.uk
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
SB  - IM
MH  - *Adaptation, Physiological
MH  - Animals
MH  - Base Sequence
MH  - *Color
MH  - DNA Primers
MH  - *Environment
MH  - *Evolution, Molecular
MH  - Lizards/*classification/genetics/physiology
MH  - Male
MH  - *Phylogeny
MH  - Spectrophotometry
EDAT- 2002/09/14 10:00
MHDA- 2002/10/31 04:00
CRDT- 2002/09/14 10:00
PHST- 2002/09/14 10:00 [pubmed]
PHST- 2002/10/31 04:00 [medline]
PHST- 2002/09/14 10:00 [entrez]
AID - 10.1080/10635150290069986 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Aug;51(4):554-69. doi: 10.1080/10635150290069986.

PMID- 12227998
OWN - NLM
STAT- MEDLINE
DCOM- 20021024
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 4
DP  - 2002 Aug
TI  - The effects of topological inaccuracy in evolutionary trees on the phylogenetic
      comparative method of independent contrasts.
PG  - 541-53
AB  - Computer simulations were used to test the effect of increasing phylogenetic
      topological inaccuracy on the results obtained from correlation tests of
      independent contrasts. Predictably, increasing the number of disruptions in the
      tree increases the likelihood of significant error in the r values produced and
      in the statistical conclusions drawn from the analysis. However, the position of 
      the disruption in the tree is important: Disruptions closer to the tips of the
      tree have a greater effect than do disruptions that are close to the root of the 
      tree. Independent contrasts derived from inaccurate topologies are more likely to
      lead to erroneous conclusions when there is a true significant relationship
      between the variables being tested (i.e., they tend to be conservative). The
      results also suggest that random phylogenies perform no better than
      nonphylogenetic analyses and, under certain conditions, may perform even worse
      than analyses using raw species data. Therefore, the use of random phylogenies is
      not beneficial in the absence of knowledge of the true phylogeny.
FAU - Symonds, Matthew R E
AU  - Symonds MR
AD  - University Museum of Zoology, Downing Street, Cambridge CB2 3EJ, UK.
      m.symonds@unimelb.edu.au
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - *Phylogeny
EDAT- 2002/09/14 10:00
MHDA- 2002/10/31 04:00
CRDT- 2002/09/14 10:00
PHST- 2002/09/14 10:00 [pubmed]
PHST- 2002/10/31 04:00 [medline]
PHST- 2002/09/14 10:00 [entrez]
AID - 10.1080/10635150290069977 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Aug;51(4):541-53. doi: 10.1080/10635150290069977.

PMID- 12118412
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20020716
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 2
DP  - 2000 Jun
TI  - Gaps as characters in sequence-based phylogenetic analyses.
PG  - 369-81
FAU - Simmons, M P
AU  - Simmons MP
AD  - L.H. Bailey Hortorium, 462 Mann Library, Ithaca, New York 14853, USA.
      mps14@cornell.edu
FAU - Ochoterena, H
AU  - Ochoterena H
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Proteins)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - *Amino Acid Sequence
MH  - *Base Sequence
MH  - DNA/chemistry/*genetics
MH  - Decision Trees
MH  - *Phylogeny
MH  - Proteins/chemistry/*genetics
MH  - Reproducibility of Results
EDAT- 2002/07/18 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/18 10:00
PHST- 2002/07/18 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/18 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2000 Jun;49(2):369-81.

PMID- 12118411
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20020716
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 2
DP  - 2000 Jun
TI  - Simple but fundamental limitations on supertree and consensus tree methods.
PG  - 363-8
FAU - Steel, M
AU  - Steel M
AD  - Biomathematics Research Centre, University of Canterbury, Private Bag 4800,
      Christchurch, New Zealand. m.steel@math.canterbury.ac.nz
FAU - Dress, A W
AU  - Dress AW
FAU - Bocker, S
AU  - Bocker S
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/methods
MH  - *Consensus Sequence
MH  - Decision Trees
MH  - *Phylogeny
EDAT- 2002/07/18 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/18 10:00
PHST- 2002/07/18 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/18 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2000 Jun;49(2):363-8.

PMID- 12118410
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 2
DP  - 2000 Jun
TI  - Phylogenetics of flowering plants based on combined analysis of plastid atpB and 
      rbcL gene sequences.
PG  - 306-62
AB  - Following (1) the large-scale molecular phylogeny of seed plants based on plastid
      rbcL gene sequences (published in 1993 by Chase et al., Ann. Missouri Bot. Gard. 
      80:528-580) and (2) the 18S nuclear phylogeny of flowering plants (published in
      1997 by Soltis et al., Ann. Missouri Bot. Gard. 84:1-49), we present a
      phylogenetic analysis of flowering plants based on a second plastid gene, atpB,
      analyzed separately and in combination with rbcL sequences for 357 taxa. Despite 
      some discrepancies, the atpB-based phylogenetic trees were highly congruent with 
      those derived from the analysis of rbcL and 18S rDNA, and the combination of atpB
      and rbcL DNA sequences (comprising approximately 3000 base pairs) produced
      increased bootstrap support for many major sets of taxa. The angiosperms are
      divided into two major groups: noneudicots with inaperturate or uniaperturate
      pollen (monocots plus Laurales, Magnoliales, Piperales, Ceratophyllales, and
      Amborellaceae-Nymphaeaceae-Illiciaceae) and the eudicots with triaperturate
      pollen (particularly asterids and rosids). Based on rbcL alone and atpB/rbcL
      combined, the noneudicots (excluding Ceratophyllum) are monophyletic, whereas in 
      the atpB trees they form a grade. Ceratophyllum is sister to the rest of
      angiosperms with rbcL alone and in the combined atpB/rbcL analysis, whereas with 
      atpB alone, Amborellaceae, Nymphaeaceae, and Illiciaceae/Schisandraceae form a
      grade at the base of the angiosperms. The phylogenetic information at each codon 
      position and the different types of substitutions (observed transitions and
      transversions in the trees vs. pairwise comparisons) were examined; taking into
      account their respective consistency and retention indices, we demonstrate that
      third-codon positions and transitions are the most useful characters in these
      phylogenetic reconstructions. This study further demonstrates that phylogenetic
      analysis of large matrices is feasible.
FAU - Savolainen, V
AU  - Savolainen V
AD  - Molecular Systematics Section, Jodrell Laboratory, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3DS, UK. v.savolainen@rbgkew.org.uk
FAU - Chase, M W
AU  - Chase MW
FAU - Hoot, S B
AU  - Hoot SB
FAU - Morton, C M
AU  - Morton CM
FAU - Soltis, D E
AU  - Soltis DE
FAU - Bayer, C
AU  - Bayer C
FAU - Fay, M F
AU  - Fay MF
FAU - de Bruijn, A Y
AU  - de Bruijn AY
FAU - Sullivan, S
AU  - Sullivan S
FAU - Qiu, Y L
AU  - Qiu YL
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
RN  - 0 (DNA, Plant)
RN  - 0 (Plant Proteins)
RN  - EC 4.1.1.39 (RbcL protein, plastid)
RN  - EC 4.1.1.39 (Ribulose-Bisphosphate Carboxylase)
SB  - IM
MH  - Base Sequence
MH  - Codon/genetics
MH  - DNA, Plant/*genetics
MH  - Magnoliopsida/*classification/*genetics
MH  - *Phylogeny
MH  - Plant Proteins/*genetics
MH  - Plastids/*genetics
MH  - Ribulose-Bisphosphate Carboxylase/*genetics
EDAT- 2002/07/18 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/18 10:00
PHST- 2002/07/18 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/18 10:00 [entrez]
AID - 10.1093/sysbio/49.2.306 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Jun;49(2):306-62. doi: 10.1093/sysbio/49.2.306.

PMID- 12118409
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 2
DP  - 2000 Jun
TI  - Phylogenetic relationships of agaric fungi based on nuclear large subunit
      ribosomal DNA sequences.
PG  - 278-305
AB  - Phylogenetic relationships of mushrooms and their relatives within the order
      Agaricales were addressed by using nuclear large subunit ribosomal DNA sequences.
      Approximately 900 bases of the 5' end of the nucleus-encoded large subunit RNA
      gene were sequenced for 154 selected taxa representing most families within the
      Agaricales. Several phylogenetic methods were used, including weighted and
      equally weighted parsimony (MP), maximum likelihood (ML), and distance methods
      (NJ). The starting tree for branch swapping in the ML analyses was the tree with 
      the highest ML score among previously produced MP and NJ trees. A high degree of 
      consensus was observed between phylogenetic estimates obtained through MP and ML.
      NJ trees differed according to the distance model that was used; however, all NJ 
      trees still supported most of the same terminal groupings as the MP and ML trees 
      did. NJ trees were always significantly suboptimal when evaluated against the
      best MP and ML trees, by both parsimony and likelihood tests. Our analyses
      suggest that weighted MP and ML provide the best estimates of Agaricales
      phylogeny. Similar support was observed between bootstrapping and jackknifing
      methods for evaluation of tree robustness. Phylogenetic analyses revealed many
      groups of agaricoid fungi that are supported by moderate to high bootstrap or
      jackknife values or are consistent with morphology-based classification schemes. 
      Analyses also support separate placement of the boletes and russules, which are
      basal to the main core group of gilled mushrooms (the Agaricineae of Singer).
      Examples of monophyletic groups include the families Amanitaceae, Coprinaceae
      (excluding Coprinus comatus and subfamily Panaeolideae), Agaricaceae (excluding
      the Cystodermateae), and Strophariaceae pro parte (Stropharia, Pholiota, and
      Hypholoma); the mycorrhizal species of Tricholoma (including Leucopaxillus, also 
      mycorrhizal); Mycena and Resinomycena; Termitomyces, Podabrella, and Lyophyllum; 
      and Pleurotus with Hohenbuehelia. Several groups revealed by these data to be
      nonmonophyletic include the families Tricholomataceae, Cortinariaceae, and
      Hygrophoraceae and the genera Clitocybe, Omphalina, and Marasmius. This study
      provides a framework for future systematics studies in the Agaricales and
      suggestions for analyzing large molecular data sets.
FAU - Moncalvo, J M
AU  - Moncalvo JM
AD  - Department of Botany, Duke University, Durham, North Carolina 27708, USA.
      jeanmarc@duke.edu
FAU - Lutzoni, F M
AU  - Lutzoni FM
FAU - Rehner, S A
AU  - Rehner SA
FAU - Johnson, J
AU  - Johnson J
FAU - Vilgalys, R
AU  - Vilgalys R
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Fungal)
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Agaricales/*classification/*genetics
MH  - Base Sequence
MH  - Biological Evolution
MH  - Classification/methods
MH  - DNA, Fungal/*genetics
MH  - DNA, Ribosomal/*genetics
MH  - Genetic Variation
MH  - *Phylogeny
EDAT- 2002/07/18 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/18 10:00
PHST- 2002/07/18 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/18 10:00 [entrez]
AID - 10.1093/sysbio/49.2.278 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Jun;49(2):278-305. doi: 10.1093/sysbio/49.2.278.

PMID- 12118408
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 2
DP  - 2000 Jun
TI  - Evolution and phylogenetic information content of mitochondrial genomic
      structural features illustrated with acrodont lizards.
PG  - 257-77
AB  - DNA sequences from 195 squamate reptiles indicate that mitochondrial gene order
      is the most reliable phylogenetic character establishing monophyly of acrodont
      lizards and of the snake families Boidae, Colubridae, and Viperidae. Gene order
      shows no evidence of evolutionary parallelisms or reversals in these taxa.
      Derived secondary structures of mitochondrial tRNAs also prove to be useful
      phylogenetic characters showing no reversals. Parallelisms for secondary
      structures of tRNAs are restricted to deep lineages that are separated by at
      least 200 million years of independent evolution. Presence of a stem-and-loop
      structure between the genes encoding tRNA(Asn) and tRNA(Cys), where the
      replication origin for light-strand synthesis is typically located in vertebrate 
      mitochondrial genomes, is found to undergo at least three and possibly as many as
      seven evolutionary shifts, most likely parallel losses. This character is
      therefore a less desirable phylogenetic marker than the other structural changes 
      examined. Sequencing regions that contain multiple genes, including tRNA genes,
      may be preferable to the common practice of obtaining single-gene fragments for
      phylogenetic inference because it permits observation of major structural changes
      in the mitochondrial genome. Such characters may occasionally provide
      phylogenetic information on relatively short internal branches for which base
      substitutional changes are expected to be relatively uninformative.
FAU - Macey, J R
AU  - Macey JR
AD  - Department of Biology, Box 1137, Washington University, St. Louis, MO 63130-4899,
      USA. macey@biology.wustl.edu
FAU - Schulte, J A 2nd
AU  - Schulte JA 2nd
FAU - Larson, A
AU  - Larson A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (RNA, Transfer, Asn)
RN  - 0 (RNA, Transfer, Phe)
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - DNA Replication
MH  - DNA, Mitochondrial/*genetics
MH  - Lizards/*classification/*genetics
MH  - Molecular Sequence Data
MH  - Nucleic Acid Conformation
MH  - *Phylogeny
MH  - RNA, Transfer, Asn/genetics
MH  - RNA, Transfer, Phe/genetics
MH  - Reptiles/classification/genetics
EDAT- 2002/07/18 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/18 10:00
PHST- 2002/07/18 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/18 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2000 Jun;49(2):257-77.

PMID- 12118407
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 2
DP  - 2000 Jun
TI  - Evaluating trans-tethys migration: an example using acrodont lizard
      phylogenetics.
PG  - 233-56
AB  - A phylogenetic tree for acrodont lizards (Chamaeleonidae and Agamidae) is
      established based on 1434 bases (1041 informative) of aligned DNA positions from 
      a 1685-1778 base pair region of the mitochondrial genome. Sequences from three
      protein-coding genes (ND1, ND2, and COI) are combined with sequences from eight
      intervening tRNA genes for samples of 70 acrodont taxa and two outgroups.
      Parsimony analysis of nucleotide sequences identifies eight major clades in the
      Acrodonta. Most agamid lizards are placed into three distinct clades. One clade
      is composed of all taxa occurring in Australia and New Guinea; Physignathus
      cocincinus from Southeast Asia is the sister taxon to the Australia-New Guinea
      clade. A second clade is composed of taxa occurring from Tibet and the Indian
      Subcontinent east through South and East Asia. A third clade is composed of taxa 
      occurring from Africa east through Arabia and West Asia to Tibet and the Indian
      Subcontinent. These three clades contain all agamid lizards except Uromastyx,
      Leiolepis, and Hydrosaurus, which represent three additional clades of the
      Agamidae. The Chamaeleonidae forms another clade weakly supported as the sister
      taxon to the Agamidae. All eight clades of the Acrodonta contain members
      occurring on land masses derived from Gondwanaland. A hypothesis of agamid
      lizards rafting with Gondwanan plates is examined statistically. This hypothesis 
      suggests that the African/West Asian clade is of African or Indian origin, and
      the South Asian clade is either of Indian or Southeast Asian origin. The shortest
      tree suggests a possible African origin for the former and an Indian origin for
      the latter, but this result is not statistically robust. The Australia-New Guinea
      clade rafted with the Australia-New Guinea plate and forms the sister group to a 
      Southeast Asian taxon that occurs on plates that broke from northern
      Australia-New Guinea. Other acrodont taxa are inferred to be associated with the 
      plates of Afro-Arabia and Madagascar (Chameleonidae), India (Uromastyx), or
      southeast Asia (Hydrosaurus and Leiolepis). Introduction of different biotic
      elements to Asia by way of separate Gondwanan plates may be a major theme of
      Asian biogeography. Three historical events may be responsible for the sharp
      faunal barrier between Southeast Asia and Australia-New Guinea, known as
      Wallace's line: (1) primary vicariance caused by plate separations; (2) secondary
      contact of Southeast Asian plates with Eurasia, leading to dispersal from Eurasia
      into Southeast Asia, and (3) dispersal of the Indian fauna (after collision of
      that subcontinent) to Southeast Asia. Acrodont lizards show the first and third
      of these biogeographic patterns and anguid lizards exhibit the second pattern.
      Modern faunal diversity may be influenced primarily by historical events such as 
      tectonic collisions and land bridge connections, which are expected to promote
      episodic turnover of continental faunas by introducing new faunal elements into
      an area. Repeated tectonic collisions may be one of the most important phenomena 
      promoting continental biodiversity. Phylogenetics is a powerful method for
      investigating these processes.
FAU - Macey, J R
AU  - Macey JR
AD  - Department of Biology, Box 1137, Washington University, St. Louis, Missouri
      63130-4899, USA. macey@biology.wustl.edu
FAU - Schulte, J A 2nd
AU  - Schulte JA 2nd
FAU - Larson, A
AU  - Larson A
FAU - Ananjeva, N B
AU  - Ananjeva NB
FAU - Wang, Y
AU  - Wang Y
FAU - Pethiyagoda, R
AU  - Pethiyagoda R
FAU - Rastegar-Pouyani, N
AU  - Rastegar-Pouyani N
FAU - Papenfuss, T J
AU  - Papenfuss TJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA/genetics/isolation &amp; purification
MH  - DNA Primers
MH  - Geography
MH  - Lizards/*classification/*genetics
MH  - *Phylogeny
EDAT- 2002/07/18 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/18 10:00
PHST- 2002/07/18 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/18 10:00 [entrez]
AID - 10.1093/sysbio/49.2.233 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Jun;49(2):233-56. doi: 10.1093/sysbio/49.2.233.

PMID- 12118406
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 2
DP  - 2000 Jun
TI  - Invariable sites models and their use in phylogeny reconstruction.
PG  - 225-32
AB  - Phylogenetic inference is well known to be problematic if both long and short
      branches occur together in the underlying tree. With biological data, correcting 
      for this problem may require simultaneous consideration for both substitution
      biases and rate heterogeneity between lineages and across sequence positions. A
      particular form of the latter is the presence of invariable sites, which are well
      known to mislead estimation of genetic divergences. Here we describe a
      capture-recapture method to estimate the proportion of invariable sites in an
      alignment of amino acids or nucleotides. We use it to investigate phylogenetic
      signals in 18S ribosomal DNA sequences from Holometabolus insects. Our results
      suggest that, as taxa diverged, their 18S rDNA sequences have altered in both
      their distribution of sites that can vary as well as in their base compositions.
FAU - Steel, M
AU  - Steel M
AD  - Biomathematics Research Centre, University of Canterbury, Christchurch, New
      Zealand. m.steel@math.canterbury.ac.nz
FAU - Huson, D
AU  - Huson D
FAU - Lockhart, P J
AU  - Lockhart PJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 18S)
SB  - IM
MH  - Animals
MH  - DNA, Ribosomal/genetics
MH  - Diptera/*classification/*genetics
MH  - *Models, Biological
MH  - *Phylogeny
MH  - RNA, Ribosomal, 18S/genetics
MH  - Sequence Alignment
EDAT- 2002/07/18 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/18 10:00
PHST- 2002/07/18 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/18 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2000 Jun;49(2):225-32.

PMID- 12118405
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 2
DP  - 2000 Jun
TI  - More taxa or more characters revisited: combining data from nuclear
      protein-encoding genes for phylogenetic analyses of Noctuoidea (Insecta:
      Lepidoptera).
PG  - 202-24
AB  - A central question concerning data collection strategy for molecular phylogenies 
      has been, is it better to increase the number of characters or the number of taxa
      sampled to improve the robustness of a phylogeny estimate? A recent simulation
      study concluded that increasing the number of taxa sampled is preferable to
      increasing the number of nucleotide characters, if taxa are chosen specifically
      to break up long branches. We explore this hypothesis by using empirical data
      from noctuoid moths, one of the largest superfamilies of insects. Separate
      studies of two nuclear genes, elongation factor-1 alpha (EF-1 alpha) and dopa
      decarboxylase (DDC), have yielded similar gene trees and high concordance with
      morphological groupings for 49 exemplar species. However, support levels were
      quite low for nodes deeper than the subfamily level. We tested the effects on
      phylogenetic signal of (1) increasing the taxon sampling by nearly 60%, to 77
      species, and (2) combining data from the two genes in a single analysis.
      Surprisingly, the increased taxon sampling, although designed to break up long
      branches, generated greater disagreement between the two gene data sets and
      decreased support levels for deeper nodes. We appear to have inadvertently
      introduced new long branches, and breaking these up may require a yet larger
      taxon sample. Sampling additional characters (combining data) greatly increased
      the phylogenetic signal. To contrast the potential effect of combining data from 
      independent genes with collection of the same total number of characters from a
      single gene, we simulated the latter by bootstrap augmentation of the single-gene
      data sets. Support levels for combined data were at least as high as those for
      the bootstrap-augmented data set for DDC and were much higher than those for the 
      augmented EF-1 alpha data set. This supports the view that in obtaining
      additional sequence data to solve a refractory systematic problem, it is prudent 
      to take them from an independent gene.
FAU - Mitchell, A
AU  - Mitchell A
AD  - Department of Entomology, University of Maryland, College Park, Maryland 20742,
      USA. am16@ualberta.ca
FAU - Mitter, C
AU  - Mitter C
FAU - Regier, J C
AU  - Regier JC
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (Insect Proteins)
RN  - 0 (Nuclear Proteins)
RN  - 0 (Peptide Elongation Factor 1)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA Primers
MH  - Insect Proteins/genetics
MH  - Lepidoptera/*classification/*genetics
MH  - Nuclear Proteins/*genetics
MH  - Peptide Elongation Factor 1/genetics
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Software
EDAT- 2002/07/18 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/18 10:00
PHST- 2002/07/18 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/18 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2000 Jun;49(2):202-24.

PMID- 12118404
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 2
DP  - 2000 Jun
TI  - A new method to localize and test the significance of incongruence: detecting
      domain shuffling in the nuclear receptor superfamily.
PG  - 183-201
AB  - When a data set is partitioned, the resulting subsets may contain
      phylogenetically conflicting signals if they have different evolutionary
      histories. In a data set with many taxa, a single taxon that contains multiple
      phylogenetic histories may result in global incongruence, but no methods are
      available in a parsimony framework to localize incongruence to specific clades in
      a phylogeny or to test the significance of incongruence on a local scale. Here we
      present a new method to quantify the conflict between data partitions for any
      clade in a phylogeny and to test the statistical significance of that conflict by
      using a metric called the local incongruence length difference. We apply this
      method to the evolutionary history of the nuclear receptor superfamily, a large
      group of transcriptional regulators that play essential roles in metazoan
      development and physiology. All nuclear receptors are composed of several
      discrete domains, including one that binds to DNA response elements on specific
      target genes and another that binds to the appropriate ligand. We have performed 
      combined and separate phylogenetic analyses of these two domains and have tested 
      the hypothesis that nuclear receptors evolved by a simple process of lineage
      splitting and divergence, without domain shuffling or other forms of sequence
      transfer between proteins. Our analysis indicates that significant conflict
      exists between the partitined domains at a few nodes on the tree, suggesting that
      several groups of receptors are "hybrid proteins" formed by domain shuffling or
      other forms of sequence transfer between more ancient nuclear receptors.
FAU - Thornton, J W
AU  - Thornton JW
AD  - Department of Biological Sciences, Center for Environmental Research and
      Conservation, Columbia University, MC-5557, New York, New York 10027 USA.
      jt121@columbia.edu
FAU - DeSalle, R
AU  - DeSalle R
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Receptors, Cytoplasmic and Nuclear)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - DNA/genetics
MH  - Humans
MH  - *Phylogeny
MH  - Receptors, Cytoplasmic and Nuclear/*classification/*genetics
MH  - Reproducibility of Results
EDAT- 2002/07/18 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/18 10:00
PHST- 2002/07/18 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/18 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2000 Jun;49(2):183-201.

PMID- 12116944
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 5
DP  - 2001 Sep-Oct
TI  - Improving interpretation of the decay index for DNA sequence data.
PG  - 742-52
FAU - DeBry, R W
AU  - DeBry RW
AD  - Department of Biological Sciences, Box 210006, University of Cincinnati,
      Cincinnati, Ohio 45221-0006, USA. ron.debry@uc.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biometry
MH  - Computer Simulation
MH  - Databases, Nucleic Acid
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Sequence Analysis, DNA/*statistics &amp; numerical data
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753328866 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Sep-Oct;50(5):742-52. doi: 10.1080/106351501753328866.

PMID- 12116943
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 5
DP  - 2001 Sep-Oct
TI  - Bilaterian phylogeny and uncritical recycling of morphological data sets.
PG  - 730-42
FAU - Jenner, R A
AU  - Jenner RA
AD  - Institute for Biodiversity and Ecosystem Dynamics, University of Amsterdam, P.O. 
      Box 94766, 1090GT, Amsterdam, The Netherlands. jenner@science.uva.nl
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biometry
MH  - Databases, Factual
MH  - Genetic Variation
MH  - *Models, Genetic
MH  - Morphogenesis
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753328857 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Sep-Oct;50(5):730-42. doi: 10.1080/106351501753328857.

PMID- 12116942
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 5
DP  - 2001 Sep-Oct
TI  - Should we use model-based methods for phylogenetic inference when we know that
      assumptions about among-site rate variation and nucleotide substitution pattern
      are violated?
PG  - 723-9
FAU - Sullivan, J
AU  - Sullivan J
AD  - Department of Biological Sciences, Box 443051, University of Idaho, Moscow, Idaho
      83844-3051, USA. jacks@uidaho.edu
FAU - Swofford, D L
AU  - Swofford DL
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Base Sequence
MH  - Computer Simulation
MH  - DNA/genetics
MH  - Genetic Variation
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753328848 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Sep-Oct;50(5):723-9. doi: 10.1080/106351501753328848.

PMID- 12116941
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 5
DP  - 2001 Sep-Oct
TI  - Maximum likelihood estimation of phylogenetic trees is consistent when
      substitution rates vary according to the invariable sites plus gamma
      distribution.
PG  - 713-22
AB  - Maximum likelihood estimation of phylogenetic trees from nucleotide sequences is 
      completely consistent when nucleotide substitution is governed by the general
      time reversible (GTR) model with rates that vary over sites according to the
      invariable sites plus gamma (I + gamma) distribution.
FAU - Rogers, J S
AU  - Rogers JS
AD  - Department of Biological Sciences, University of New Orleans, New Orleans,
      Louisiana 70148, USA. jsrogers@uno.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753328839 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Sep-Oct;50(5):713-22. doi: 10.1080/106351501753328839.

PMID- 12116940
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 5
DP  - 2001 Sep-Oct
TI  - Repeated evolution of dioecy from monoecy in Siparunaceae (Laurales).
PG  - 700-12
AB  - Siparunaceae comprise Glossocalyx with one species in West Africa and Siparuna
      with 65 species in the neotropics; all have unisexual flowers, and 15 species are
      monoecious, 50 dioecious. Parsimony and maximum likelihood analyses of combined
      nuclear ribosomal ITS and chloroplast trnL-trnF intergenic spacer sequences
      yielded almost identical topologies, which were used to trace the evolution of
      the two sexual systems. The African species, which is dioecious, was sister to
      all neotropical species, and the monoecious species formed a grade basal to a
      large dioecious Andean clade. Dioecy evolved a second time within the monoecious 
      grade. Geographical mapping of 6,496 herbarium collections from all species
      sorted by sexual system showed that monoecy is confined to low-lying areas
      (altitude &lt; 700 m) in the Amazon basin and southern Central America. The only
      morphological trait with a strong phylogenetic signal is leaf margin shape
      (entire or toothed), although this character also correlates with altitude,
      probably reflecting selection on leaf shapes by temperature and rainfall regimes.
      The data do not reject the molecular clock, and branch lengths suggest that the
      shift to dioecy in the lowlands occurred many million years after the shift to
      dioecy in the ancestor of the Andean clade.
FAU - Renner, S S
AU  - Renner SS
AD  - Department of Biology, University of Missouri-St. Louis, 8001 Natural Bridge Rd.,
      St. Louis, MO 63121-4499, USA. renner@umsl.edu
FAU - Won, H
AU  - Won H
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Intergenic)
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Base Composition
MH  - DNA, Intergenic/chemistry/genetics
MH  - DNA, Plant/chemistry/genetics
MH  - *Evolution, Molecular
MH  - Magnoliopsida/anatomy &amp; histology/*classification/*genetics
MH  - Models, Genetic
MH  - Phylogeny
MH  - Plant Leaves/anatomy &amp; histology
MH  - South America
MH  - Time Factors
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753328820 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Sep-Oct;50(5):700-12. doi: 10.1080/106351501753328820.

PMID- 12116939
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 5
DP  - 2001 Sep-Oct
TI  - Character analysis in morphological phylogenetics: problems and solutions.
PG  - 689-99
AB  - Many aspects of morphological phylogenetics are controversial in the theoretical 
      systematics literature and yet are often poorly explained and justified in
      empirical studies. In this paper, I argue that most morphological characters
      describe variation that is fundamentally quantitative, regardless of whether they
      are coded qualitatively or quantitatively by systematists. Given this view, three
      fundamental problems in morphological character analysis (definition,
      delimitation, and ordering of character states) may have a common solution:
      coding morphological characters as continuous quantitative traits. A new
      parsimony method (step-matrix gap-weighting, a modification of Thiele's approach)
      is proposed that allows quantitative traits to be analyzed as continuous
      variables. The problem of scaling or weighting quantitative characters relative
      to qualitative characters (and to each other) is reviewed, and three possible
      solutions are described. The new coding method is applied to data from
      hoplocercid lizards, and the results show the sensitivity of phylogenetic
      conclusions to different scaling methods. Although some authors reject the use of
      continuous, overlapping, quantitative characters in phylogenetic analysis,
      quantitative data from hoplocercid lizards that are coded using the new approach 
      contain significant phylogenetic structure and exhibit levels of homoplasy
      similar to those seen in data that are coded qualitatively.
FAU - Wiens, J J
AU  - Wiens JJ
AD  - Section of Amphibians and Reptiles, Carnegie Museum of Natural History,
      Pittsburgh, Pennsylvania 15213-4080, USA. wiensj@carnegiemuseums.org
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biometry
MH  - Lizards/anatomy &amp; histology/genetics
MH  - Models, Anatomic
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Quantitative Trait, Heritable
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753328811 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Sep-Oct;50(5):689-99. doi: 10.1080/106351501753328811.

PMID- 12116938
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 5
DP  - 2001 Sep-Oct
TI  - mtDNA ribosomal gene phylogeny of sea hares in the genus Aplysia (Gastropoda,
      Opisthobranchia, Anaspidea): implications for comparative neurobiology.
PG  - 676-88
AB  - Sea hares within the genus Aplysia are important neurobiological model organisms;
      as more studies based on different Aplysia species are appearing in the
      literature, a phylogenetic framework has become essential. We present a
      phylogenetic hypothesis for this genus, based on portions of two mitochondrial
      genes (12S and 16S). In addition, we reconstruct the evolution of several
      behavioral characters of interest to neurobiologists to illustrate the potential 
      benefits of a phylogeny for the genus Aplysia. These benefits include
      determination of ancestral traits, direction and timing of evolution of
      characters, prediction of the distribution of traits, and identification of cases
      of independent acquisition of traits within lineages. This last benefit may prove
      especially useful in understanding the linkage between behaviors and their
      underlying neurological bases.
FAU - Medina, M
AU  - Medina M
AD  - Rosenstiel School of Marine and Atmospheric Science, Division of Marine Biology
      and Fisheries, University of Miami, 4600 Rickenbacker Causeway, Miami, FL 33149, 
      USA. M_Medina@lbl.gov
FAU - Collins, T M
AU  - Collins TM
FAU - Walsh, P J
AU  - Walsh PJ
LA  - eng
GR  - RR 10294/RR/NCRR NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (DNA, Ribosomal)
RN  - 0 (RNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 16S)
RN  - 0 (RNA, ribosomal, 12S)
SB  - IM
MH  - Animals
MH  - Aplysia/classification/*genetics/physiology
MH  - Base Sequence
MH  - Behavior, Animal
MH  - Brain/growth &amp; development
MH  - DNA, Mitochondrial/*genetics
MH  - DNA, Ribosomal/*genetics
MH  - Evolution, Molecular
MH  - Models, Genetic
MH  - Models, Molecular
MH  - Molecular Sequence Data
MH  - Neurobiology
MH  - Nucleic Acid Conformation
MH  - *Phylogeny
MH  - RNA, Ribosomal/chemistry/genetics
MH  - RNA, Ribosomal, 16S/chemistry/genetics
MH  - Reflex
MH  - Sequence Homology, Nucleic Acid
MH  - Swimming
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753328802 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Sep-Oct;50(5):676-88. doi: 10.1080/106351501753328802.

PMID- 12116937
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 5
DP  - 2001 Sep-Oct
TI  - Inferring and testing hypotheses of cladistic character dependence by using
      character compatibility.
PG  - 657-75
AB  - The notion that two characters evolve independently is of interest for two
      reasons. First, theories of biological integration often predict that change in
      one character requires complementary change in another. Second, character
      independence is a basic assumption of most phylogenetic inference methods, and
      dependent characters might confound attempts at phylogenetic inference.
      Previously proposed tests of correlated character evolution require a model
      phylogeny and therefore assume that nonphylogenetic correlation has a negligible 
      effect on initial tree construction. This paper develops "tree-free" methods for 
      testing the independence of cladistic characters. These methods can test the
      character independence model as a hypothesis before phylogeny reconstruction, or 
      can be used simply to test for correlated evolution. We first develop an approach
      for visualizing suites of correlated characters by using character compatibility.
      Two characters are compatible if they can be used to construct a tree without
      homoplasy. The approach is based on the examination of mutual compatibilities
      between characters. The number of times two characters i and j share
      compatibility with a third character is calculated, and a pairwise shared
      compatibility matrix is constructed. From this matrix, an association matrix
      analogous to a dissimilarity matrix is derived. Eigenvector analyses of this
      association matrix reveal suites of characters with similar compatibility
      patterns. A priori character subsets can be tested for significant correlation on
      these axes. Monte Carlo tests are performed to determine the expected
      distribution of mutual compatibilities, given various criteria from the original 
      data set. These simulated distributions are then used to test whether the
      observed amounts of nonphylogenetic correlation in character suites can be
      attributed to chance alone. We have applied these methods to published
      morphological data for caecilian amphibians. The analyses corroborate instances
      of dependent evolution hypothesized by previous workers and also identify novel
      partitions. Phylogenetic analysis is performed after reducing correlated suites
      to single characters. The resulting cladogram has greater topological resolution 
      and implies appreciably less change among the remaining characters than does a
      tree derived from the raw data matrix.
FAU - O'Keefe, F R
AU  - O'Keefe FR
AD  - Department of Anatomy, New York College of Osteopathic Medicine, New York
      Institute of Technology, Old Westbury, New York 11568-8000, USA.
      rokeefe@iris.nyit.edu
FAU - Wagner, P J
AU  - Wagner PJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amphibians/anatomy &amp; histology/classification/genetics
MH  - Animals
MH  - *Biological Evolution
MH  - Biometry
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - Multivariate Analysis
MH  - Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753328794 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Sep-Oct;50(5):657-75. doi: 10.1080/106351501753328794.

PMID- 12116936
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 5
DP  - 2001 Sep-Oct
TI  - Comparing ontogenetic trajectories using growth process data.
PG  - 640-56
AB  - Ontogenetic trajectories are commonly quantified by characterizing changes in the
      sizes and shapes of organisms over the course of development. This formulation of
      ontogenetic transformations can be misleading in that it ignores critical aspects
      of the biological processes responsible for constructing morphology. Hypothetical
      examples are used to illustrate some of the shortcomings of methods that rely
      exclusively on size and shape data for ontogenetic analyses. By characterizing
      growth as a vector field, and representing growth vectors as complex numbers, one
      can simultaneously analyze size, shape, and growth processes. The utility of such
      an approach is demonstrated in a study of shape and growth process variation in
      turtle shells.
FAU - Magwene, P M
AU  - Magwene PM
AD  - Department of Ecology and Evolutionary Biology, Yale University, P.O. Box 208106,
      New Haven, Connecticut 06520, USA. paul.magwene@yale.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2002 Oct;51(5):817-22. PMID: 12396593
MH  - Analysis of Variance
MH  - Animals
MH  - Biometry
MH  - Body Constitution
MH  - *Growth
MH  - *Models, Biological
MH  - Phylogeny
MH  - Turtles/anatomy &amp; histology/growth &amp; development
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753328785 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Sep-Oct;50(5):640-56. doi: 10.1080/106351501753328785.

PMID- 12116935
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 5
DP  - 2001 Sep-Oct
TI  - Random sampling of constrained phylogenies: conducting phylogenetic analyses when
      the phylogeny is partially known.
PG  - 628-39
AB  - Statistical randomization tests in evolutionary biology often require a set of
      random, computer-generated trees. For example, earlier studies have shown how
      large numbers of computer-generated trees can be used to conduct phylogenetic
      comparative analyses even when the phylogeny is uncertain or unknown. These
      methods were limited, however, in that (in the absence of molecular sequence or
      other data) they allowed users to assume that no phylogenetic information was
      available or that all possible trees were known. Intermediate situations where
      only a taxonomy or other limited phylogenetic information (e.g., polytomies) are 
      available are technically more difficult. The current study describes a procedure
      for generating random samples of phylogenies while incorporating limited
      phylogenetic information (e.g., four taxa belong together in a subclade). The
      procedure can be used to conduct comparative analyses when the phylogeny is only 
      partially resolved or can be used in other randomization tests in which large
      numbers of possible phylogenies are needed.
FAU - Housworth, E A
AU  - Housworth EA
AD  - Departments of Biology and Mathematics, University of Oregon, Eugene, Oregon
      97403, USA. houswrth@darkwing.uoregon.edu
FAU - Martins, E P
AU  - Martins EP
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biometry
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Random Allocation
MH  - Species Specificity
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753328776 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Sep-Oct;50(5):628-39. doi: 10.1080/106351501753328776.

PMID- 12116934
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 5
DP  - 2001 Sep-Oct
TI  - Strengths and limitations of the minimum evolution principle.
PG  - 621-7
FAU - Gascuel, O
AU  - Gascuel O
AD  - Departement Informatique Fondamentale et Applications, L.I.R.M.M., 161 rue Ada,
      34392, Montpellier, France. gascuel@lirmm.fr
FAU - Bryant, D
AU  - Bryant D
FAU - Denis, F
AU  - Denis F
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Analysis of Variance
MH  - *Biological Evolution
MH  - Least-Squares Analysis
MH  - *Models, Genetic
MH  - Phylogeny
MH  - Species Specificity
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753328767 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Sep-Oct;50(5):621-7. doi: 10.1080/106351501753328767.

PMID- 12116933
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 2
DP  - 2001 Apr
TI  - Species or supraspecific taxa as terminals in cladistic analysis? Groundplans
      versus exemplars revisited.
PG  - 290-300
FAU - Prendini, L
AU  - Prendini L
AD  - Percy FitzPatrick Institute, University of Cape Town, Rondebosch, 7701, South
      Africa. lprendin@botzoo.uct.ac.za
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biometry
MH  - Classification
MH  - Genetic Variation
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Species Specificity
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150118650 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Apr;50(2):290-300. doi: 10.1080/10635150118650.

PMID- 12116932
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 2
DP  - 2001 Apr
TI  - Does the T-PTP test tell us anything we want to know?
PG  - 284-9
FAU - Harshman, J
AU  - Harshman J
AD  - National Museum of Natural History, Laboratory of Molecular Systematics, 4210
      Silver Hill Rd., Suitland, MD 20746, USA. harshman@lms.si.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biometry/*methods
MH  - Classification
MH  - Mathematics
MH  - *Phylogeny
MH  - Probability
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150121530 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Apr;50(2):284-9. doi: 10.1080/10635150121530.

PMID- 12116931
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 2
DP  - 2001 Apr
TI  - Australian Lasioglossum + Homalictus form a monophyletic group: resolving the
      "Australian enigma".
PG  - 268-83
AB  - The bee genus Lasioglossum includes &gt; 1,000 species of bees distributed on all
      continents except Antarctica. Lasioglossum is a major component of the bee fauna 
      in the Holarctic, Ethiopian, and Asian regions and is an important group for
      investigating the evolution of social behavior in bees. Given its cosmopolitan
      distribution, the historical biogeography of the genus is of considerable
      interest. We reconstructed phylogenetic relationships among the subgenera and
      species within Lasioglossum s.s., using DNA sequence data from a slowly evolving 
      nuclear gene, elongation factor-1 alpha. The entire data set includes &gt; 1,604
      aligned nucleotide sites (including three exons plus two introns) for 89 species 
      (17 outgroups plus 72 ingroups). Parsimony and maximum likelihood analyses
      provide strong evidence that the primarily Indoaustralian subgenera (Homalictus, 
      Chilalictus, Parasphecodes) form a monophyletic group. Bootstrap support for the 
      Australian clade ranged from 73% to 77%, depending on the method of analysis.
      Monophyly of the Australian Lasioglossum suggests that a single colonization
      event (by way of Southeast Asia and New Guinea) gave rise to a lineage of &gt; 350
      native Indoaustralian bees. We discuss the implications of Australian monophyly
      for resolving the "Australian enigma"--the similarity in social behavior among
      the Australian halictine bees relative to that of Holarctic groups.
FAU - Danforth, B N
AU  - Danforth BN
AD  - Department of Entomology, Comstock Hall, Cornell University, Ithaca, New York
      14853-0901, USA. bnd1@cornell.edu
FAU - Ji, S
AU  - Ji S
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Peptide Elongation Factor 1)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Australia
MH  - Base Composition
MH  - Bees/*classification/*genetics/physiology
MH  - DNA/chemistry/genetics
MH  - Evolution, Molecular
MH  - Female
MH  - Genes, Insect
MH  - Likelihood Functions
MH  - Male
MH  - Peptide Elongation Factor 1/genetics
MH  - *Phylogeny
MH  - Social Behavior
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Apr;50(2):268-83.

PMID- 12116930
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 2
DP  - 2001 Apr
TI  - Phylogenetic relationships of fig wasps pollinating functionally dioecious Ficus 
      based on mitochondrial DNA sequences and morphology.
PG  - 243-67
AB  - The obligate mutualism between pollinating fig wasps in the family Agaonidae
      (Hymenoptera: Chalcidoidea) and Ficus species (Moraceae) is often regarded as an 
      example of co-evolution but little is known about the history of the interaction,
      and understanding the origin of functionally dioecious fig pollination has been
      especially difficult. The phylogenetic relationships of fig wasps pollinating
      functionally dioecious Ficus were inferred from mitochondrial cytochrome oxidase 
      gene sequences (mtDNA) and morphology. Separate and combined analyses indicated
      that the pollinators of functionally dioecious figs are not monophyletic.
      However, pollinator relationships were generally congruent with host phylogeny
      and support a revised classification of Ficus. Ancestral changes in pollinator
      ovipositor length also correlated with changes in fig breeding systems. In
      particular, the relative elongation of the ovipositor was associated with the
      repeated loss of functionally dioecious pollination. The concerted evolution of
      interacting morphologies may bias estimates of phylogeny based on female head
      characters, but homoplasy is not so strong in other morphological traits. The
      lesser phylogenetic utility of morphology than of mtDNA is not due to rampant
      convergence in morphology but rather to the greater number of potentially
      informative characters in DNA sequence data; patterns of nucleotide substitution 
      also limit the utility of mtDNA findings. Nonetheless, inferring the ancestral
      associations of fig pollinators from the best-supported phylogeny provided strong
      evidence of host conservatism in this highly specialized mutualism.
FAU - Weiblen, G D
AU  - Weiblen GD
AD  - Harvard University Herbaria, 22 Divinity Avenue, Cambridge, Massachusetts 02138, 
      USA. gweiblen@oeb.harvard.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (RNA, Transfer, Leu)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Biological Evolution
MH  - Biometry
MH  - DNA, Mitochondrial/*genetics
MH  - Electron Transport Complex IV/genetics
MH  - Female
MH  - *Ficus/anatomy &amp; histology/classification
MH  - Likelihood Functions
MH  - Male
MH  - *Phylogeny
MH  - RNA, Transfer, Leu/genetics
MH  - Wasps/anatomy &amp; histology/*classification/*genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Apr;50(2):243-67.

PMID- 12116929
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 2
DP  - 2001 Apr
TI  - Analysis of character correlations among wood decay mechanisms, mating systems,
      and substrate ranges in homobasidiomycetes.
PG  - 215-42
AB  - Homobasidiomycetes include the majority of wood-decaying fungi. Two basic forms
      of wood decay are known in homobasidiomycetes: white rot, in which lignin and
      cellulose are degraded, and brown rot, in which lignin is not appreciably
      degraded. An apparent correlation has been noted between production of a brown
      rot, decay of conifer substrates, and possession of a bipolar mating system
      (which has a single mating-type locus, in contrast to tetrapolar systems, which
      have two mating-type loci). The goals of this study were to infer the historical 
      pattern of transformations in decay mode, mating type, and substrate range
      characters, and to determine if a causal relationship exists among them. Using
      nuclear and mitochondrial rDNA sequences, we performed a phylogenetic analysis of
      130 species of homobasidiomycetes and performed ancestral state reconstructions
      by using parsimony on a range of trees, with various loss:gain cost ratios. We
      evaluated pairwise character correlations by using the concentrated changes test 
      (CCT) of Maddison and the maximum likelihood (ML) method of Pagel. White rot,
      tetrapolar mating systems, and the ability to decay conifers and hardwoods appear
      to be plesiomorphic in homobasidiomycetes, whereas brown rot, bipolar mating
      systems, and exclusive decay of conifers appear to have evolved repeatedly. The
      only significant correlation among characters was that between brown rot (as the 
      independent character) and exclusive decay of conifer substrates (P &lt; 0.03). This
      correlation was supported by the CCT on a range of plausible trees, although not 
      with every reconstruction of ancestral states, and by the ML test. Our findings
      suggest that the evolution of brown rot has promoted repeated shifts to
      specialization for confier substrates.
FAU - Hibbett, D S
AU  - Hibbett DS
AD  - Harvard University Herbaria, Department of Organismic and Evolutionary Biology,
      Harvard University, 22 Divinity Avenue, Cambridge, Massachusetts 02138, USA.
      dhibbett@black.clarku.edu
FAU - Donoghue, M J
AU  - Donoghue MJ
LA  - eng
SI  - GENBANK/AF334868
SI  - GENBANK/AF334869
SI  - GENBANK/AF334870
SI  - GENBANK/AF334871
SI  - GENBANK/AF334872
SI  - GENBANK/AF334873
SI  - GENBANK/AF334874
SI  - GENBANK/AF334875
SI  - GENBANK/AF334876
SI  - GENBANK/AF334877
SI  - GENBANK/AF334878
SI  - GENBANK/AF334879
SI  - GENBANK/AF334880
SI  - GENBANK/AF334881
SI  - GENBANK/AF334882
SI  - GENBANK/AF334883
SI  - GENBANK/AF334884
SI  - GENBANK/AF334885
SI  - GENBANK/AF334886
SI  - GENBANK/AF334887
SI  - GENBANK/AF334888
SI  - GENBANK/AF334889
SI  - GENBANK/AF334890
SI  - GENBANK/AF334891
SI  - GENBANK/AF334892
SI  - GENBANK/AF334893
SI  - GENBANK/AF334894
SI  - GENBANK/AF334895
SI  - GENBANK/AF334896
SI  - GENBANK/AF334897
SI  - GENBANK/AF334898
SI  - GENBANK/AF334899
SI  - GENBANK/AF334900
SI  - GENBANK/AF334901
SI  - GENBANK/AF334902
SI  - GENBANK/AF334903
SI  - GENBANK/AF334904
SI  - GENBANK/AF334905
SI  - GENBANK/AF334906
SI  - GENBANK/AF334907
SI  - GENBANK/AF334908
SI  - GENBANK/AF334909
SI  - GENBANK/AF334910
SI  - GENBANK/AF334911
SI  - GENBANK/AF334912
SI  - GENBANK/AF334913
SI  - GENBANK/AF334914
SI  - GENBANK/AF334915
SI  - GENBANK/AF334916
SI  - GENBANK/AF334917
SI  - GENBANK/AF334918
SI  - GENBANK/AF334919
SI  - GENBANK/AF334920
SI  - GENBANK/AF334921
SI  - GENBANK/AF334922
SI  - GENBANK/AF334923
SI  - GENBANK/AF334924
SI  - GENBANK/AF334925
SI  - GENBANK/AF334926
SI  - GENBANK/AF334927
SI  - GENBANK/AF334928
SI  - GENBANK/AF334929
SI  - GENBANK/AF334930
SI  - GENBANK/AF334931
SI  - GENBANK/AF334932
SI  - GENBANK/AF334933
SI  - GENBANK/AF334934
SI  - GENBANK/AF334935
SI  - GENBANK/AF334936
SI  - GENBANK/AF334937
SI  - GENBANK/AF334938
SI  - GENBANK/AF334939
SI  - GENBANK/AF334940
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Fungal)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Basidiomycota/classification/*genetics/*physiology
MH  - Biodegradation, Environmental
MH  - Biometry
MH  - DNA, Fungal/genetics
MH  - DNA, Mitochondrial/genetics
MH  - DNA, Ribosomal/genetics
MH  - Ecosystem
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Wood
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Apr;50(2):215-42.

PMID- 12116928
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 2
DP  - 2001 Apr
TI  - Multiple molecular data sets suggest independent origins of highly eusocial
      behavior in bees (Hymenoptera:Apinae).
PG  - 194-214
AB  - Different views of the pattern of social evolution among the highly eusocial bees
      have arisen as a result of discordance between past molecular and
      morphology-based phylogenies. Here we present new data and taxa for four
      molecular data sets and reassess the morphological characters available to date. 
      We show there is no significant character incongruence between four molecular
      data sets (two nuclear and two mitochondrial), but highly significant character
      incongruence leads to topological incongruence between the molecular and
      morphological data. We investigate the effects of using different outgroup
      combinations to root the estimated tree. We also consider various ways in which
      biases in the sequence data could be misleading, using several maximum likelihood
      models, LogDet corrections, and spectral analyses. Ultimately, we concede there
      is strong discordance between the molecular and morphological data partitions and
      appropriately apply the conditional combination approach in this case. We also
      find two equally well supported placements of the root for the molecular trees,
      one supported by 16S and 28S sequences, the other supported by cytochrome b and
      opsin. The strength of the evidence leads us to accept two equally well supported
      hypotheses based on analyses of the molecular data sets. These are the most
      rigorously supported hypotheses of corbiculate bee relationships at this time,
      and frame our argument that highly eusocial behavior within the corbiculate bees 
      evolved twice independently.
FAU - Cameron, S A
AU  - Cameron SA
AD  - Department of Entomology, University of Arkansas, Fayetteville, Arkansas 72701,
      USA. scameron@comp.uark.edu
FAU - Mardulyn, P
AU  - Mardulyn P
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bees/anatomy &amp; histology/classification/*genetics/*physiology
MH  - Biometry
MH  - Databases, Nucleic Acid
MH  - *Evolution, Molecular
MH  - Phylogeny
MH  - *Social Behavior
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Apr;50(2):194-214.

PMID- 12116927
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 2
DP  - 2001 Apr
TI  - The phylogenetic trunk: maximal inclusion of taxa with missing data in an
      analysis of the lepospondyli (Vertebrata, Tetrapoda).
PG  - 170-93
AB  - The importance of fossils to phylogenetic reconstruction is well established.
      However, analyses of fossil data sets are confounded by problems related to the
      less complete nature of the specimens. Taxa that are incompletely known are
      problematic because of the uncertainty of their placement within a tree, leading 
      to a proliferation of most-parsimonious solutions and "wild card" behavior.
      Problematic taxa are commonly deleted based on a priori criteria of completeness.
      Paradoxically, a taxon's problematic behavior is tree dependent, and levels of
      completeness are not directly associated with problematic behavior. Exclusion of 
      taxa on the basis of completeness eliminates real character conflict and, by not 
      allowing incomplete taxa to determine tree topology, diminishes the phylogenetic 
      hypothesis. Here, the phylogenetic trunk approach is proposed to allow
      optimization of taxonomic inclusion and tree stability. The use of this method in
      an analysis of the Paleozoic Lepospondyli finds a single most-parsimonious tree, 
      or trunk, after the removal of one taxon identified as being problematic.
      Moreover, the 38 trees found at one additional step from this primary trunk were 
      reduced to 2 by removal of one additional taxon. These trunks are compared with
      the trees that were found by excluding taxa with various degrees of completeness,
      and the effects of incomplete taxa are explored with regard to use of the trunk. 
      Correlated characters associated with limblessness are discussed regarding the
      assumption of character independence; however, inclusion of intermediate taxa is 
      found to be the single best method for breaking down long branches.
FAU - Anderson, J S
AU  - Anderson JS
AD  - Redpath Museum, Department of Biology, McGill University, Montreal, QC, Canada
      H3A 2K6. jander3@po-box.mcgill.ca jsanders@credit.erin.utoronto.ca
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biometry
MH  - Bone and Bones/anatomy &amp; histology
MH  - Fossils
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Vertebrates/anatomy &amp; histology/*classification/*genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150119889 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Apr;50(2):170-93. doi: 10.1080/10635150119889.

PMID- 12116926
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 2
DP  - 2001 Apr
TI  - Generalized frequency coding: a method of preparing polymorphic multistate
      characters for phylogenetic analysis.
PG  - 156-69
AB  - A new method of coding polymorphic multiistate characters for phylogenetic
      analysis is presented. By dividing such characters into subcharacters, their
      frequency distributions can be represented with discrete states. Differential
      weighting is used to counter the effect of representing one character with
      multiple characters. The new method, generalized frequency coding (GFC), is
      potentially superior to previously used methods in that it incorporates more
      information and is applicable to both qualitative and quantitative characters.
      When applied to a previously published data set that includes both types of
      polymorphic multistate characters, the method performed well, as assessed with g1
      and nonparametric bootstrap statistics and giving results congruent with those of
      other studies. The data set was also used to compare GFC with both gap-weighting 
      and Manhattan distance step matrix coding. On these grounds and for philosophical
      reasons, we consider GFC to be a better estimator of phylogeny.
FAU - Smith, E N
AU  - Smith EN
AD  - Department of Biology, University of Texas at Arlington, Arlington, Texas
      76019-0498, USA. e.smith@uta.edu
FAU - Gutberlet, R L Jr
AU  - Gutberlet RL Jr
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biometry/*methods
MH  - Databases, Genetic
MH  - *Phylogeny
MH  - *Polymorphism, Genetic
MH  - Viperidae/classification/genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Apr;50(2):156-69.

PMID- 12116925
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20041117
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 2
DP  - 2001 Apr
TI  - SSZ 1970-1989: a view of the years of conflict.
PG  - 153-5
FAU - Funk, V A
AU  - Funk VA
AD  - U.S. National Herbarium, NHB 166, Smithsonian Institution, Washington, D.C.
      20560, USA. Funk.Vicki@NMNH.SI.EDU
LA  - eng
PT  - Historical Article
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - History, 20th Century
MH  - Phylogeny
MH  - Societies, Scientific/*history
MH  - United States
MH  - Zoology/*history
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Apr;50(2):153-5.

PMID- 12116658
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - Misleading results from the use of ambiguity coding to score polymorphisms in
      higher-level taxa.
PG  - 613-20
FAU - Simmons, N B
AU  - Simmons NB
AD  - Department of Mammalogy, Division of Vertebrate Zoology, American Museum of
      Natural History, New York, New York 10024, USA. simmons@amnh.org
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biometry
MH  - Classification
MH  - Models, Genetic
MH  - *Phylogeny
MH  - *Polymorphism, Genetic
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150120578 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):613-20. doi: 10.1080/10635150120578.

PMID- 12116657
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - Efficiency of strict consensus trees.
PG  - 610-3
FAU - Wilkinson, M
AU  - Wilkinson M
AD  - Department of Zoology, Natural History Museum, London SW7 5BD, UK. marw@nhm.ac.uk
FAU - Thorley, J L
AU  - Thorley JL
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biometry
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):610-3.

PMID- 12116656
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - Finding fault with vicariance: a critique of Heads (1998).
PG  - 602-9
FAU - Wallis, G P
AU  - Wallis GP
AD  - Department of Zoology, University of Otago, PO Box 56, Dunedin, New Zealand.
      graham.wallis@stonebow.otago.ac.nz
FAU - Trewick, S A
AU  - Trewick SA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Classification
MH  - Ecosystem
MH  - Geological Phenomena
MH  - *Geology
MH  - New Zealand
MH  - Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):602-9.

PMID- 12116655
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20071114
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - Selecting the best-fit model of nucleotide substitution.
PG  - 580-601
AB  - Despite the relevant role of models of nucleotide substitution in phylogenetics, 
      choosing among different models remains a problem. Several statistical methods
      for selecting the model that best fits the data at hand have been proposed, but
      their absolute and relative performance has not yet been characterized. In this
      study, we compare under various conditions the performance of different
      hierarchical and dynamic likelihood ratio tests, and of Akaike and Bayesian
      information methods, for selecting best-fit models of nucleotide substitution. We
      specifically examine the role of the topology used to estimate the likelihood of 
      the different models and the importance of the order in which hypotheses are
      tested. We do this by simulating DNA sequences under a known model of nucleotide 
      substitution and recording how often this true model is recovered by the
      different methods. Our results suggest that model selection is reasonably
      accurate and indicate that some likelihood ratio test methods perform overall
      better than the Akaike or Bayesian information criteria. The tree used to
      estimate the likelihood scores does not influence model selection unless it is a 
      randomly chosen tree. The order in which hypotheses are tested, and the
      complexity of the initial model in the sequence of tests, influence model
      selection in some cases. Model fitting in phylogenetics has been suggested for
      many years, yet many authors still arbitrarily choose their models, often using
      the default models implemented in standard computer programs for phylogenetic
      estimation. We show here that a best-fit model can be readily identified.
      Consequently, given the relevance of models, model fitting should be routine in
      any phylogenetic analysis that uses models of evolution.
FAU - Posada, D
AU  - Posada D
AD  - Department of Zoology, Brigham Young University, Provo, Utah 84602-5255, USA.
      dp47@email.byu.edu
FAU - Crandall, K A
AU  - Crandall KA
LA  - eng
GR  - R01-HD 34350-01A1/HD/NICHD NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Algorithms
MH  - Base Sequence
MH  - Biometry
MH  - DNA/*genetics
MH  - Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):580-601.

PMID- 12116654
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - Assessment of the accuracy of matrix representation with parsimony analysis
      supertree construction.
PG  - 565-79
AB  - Despite the growing popularity of supertree construction for combining
      phylogenetic information to produce more inclusive phylogenies, large-scale
      performance testing of this method has not been done. Through simulation, we
      tested the accuracy of the most widely used supertree method, matrix
      representation with parsimony analysis (MRP), with respect to a (maximum
      parsimony) total evidence solution and a known model tree. When source trees
      overlap completely, MRP provided a reasonable approximation of the total evidence
      tree; agreement was usually &gt; 85%. Performance improved slightly when using
      smaller, more numerous, or more congruent source trees, and especially when
      elements were weighted in proportion to the bootstrap frequencies of the nodes
      they represented on each source tree ("weighted MRP"). Although total evidence
      always estimated the model tree slightly better than nonweighted MRP methods,
      weighted MRP in turn usually out-performed total evidence slightly. When source
      studies were even moderately nonoverlapping (i.e., sharing only three-quarters of
      the taxa), the high proportion of missing data caused a loss in resolution that
      severely degraded the performance for all methods, including total evidence. In
      such cases, even combining more trees, which had positive effects elsewhere, did 
      not improve accuracy. Instead, "seeding" the supertree or total evidence analyses
      with a single largely complete study improved performance substantially. This
      finding could be an important strategy for any studies that seek to combine
      phylogenetic information. Overall, our results suggest that MRP supertree
      construction provides a reasonable approximation of a total evidence solution and
      that weighted MRP should be used whenever possible.
FAU - Bininda-Emonds, O R
AU  - Bininda-Emonds OR
AD  - Section of Evolution and Ecology, University of California at Davis, Davis,
      California 95616, USA. bininda@rulsfb.leidenuniv.nl
FAU - Sanderson, M J
AU  - Sanderson MJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biometry
MH  - Classification
MH  - Models, Genetic
MH  - Models, Statistical
MH  - Multivariate Analysis
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):565-79.

PMID- 12116653
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - Ancestral state estimation and taxon sampling density.
PG  - 557-64
AB  - A set of experiments based on simulation and analysis found that using the
      parsimony algorithm for ancestral state estimation can benefit from increased
      sampling of terminal taxa. Estimation at the base of small clades showed strong
      sensitivity to tree topology and number of descendent tips. These effects were
      largely driven by the creation and negation of ambiguity across a topology. Root 
      state and internal state estimation showed similar behavior. We conclude that
      increased taxon sampling density is generally advisable, and attention to
      topological effects may be advisable in evaluating the confidence placed in state
      estimation. We also explore the factors affecting ancestral state estimation and 
      conjecture that as taxa are added to a tree, the total amount of information for 
      root state estimation depends on the tree topology and distance to root state of 
      added taxa. For a pure-birth model tree, we conjecture that the addition of N
      taxa increases root state information in proportion to log(N).
FAU - Salisbury, B A
AU  - Salisbury BA
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven,
      Connecticut 06520-8106, USA. ben@aya.yale.edu
FAU - Kim, J
AU  - Kim J
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Biological Evolution
MH  - Biometry
MH  - Classification
MH  - Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):557-64.

PMID- 12116652
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190523
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - Toward understanding Anophelinae (Diptera, Culicidae) phylogeny: insights from
      nuclear single-copy genes and the weight of evidence.
PG  - 540-56
AB  - A phylogeny of the mosquito subfamily Anophelinae was inferred from fragments of 
      two protein-coding nuclear genes, G6pd (462 bp) and white (801 bp), and from a
      combined data set (2,136 bp) that included a portion of the mitochondrial gene
      ND5 and the D2 region of the ribosomal 28S gene. Sixteen species from all three
      anopheline genera and six Anopheles subgenera were sampled, along with six
      species of other mosquitoes used as an outgroup. Each of four genes analyzed
      individually recovered the same well-supported clades; topological incongruence
      was limited to unsupported or poorly supported nodes. As assessed by the
      incongruence length difference test, most of the conflicting signal was
      contributed by third codon positions. Strong structural constraints, as observed 
      in white and G6pd, apparently had little impact on phylogenetic inference.
      Compared with the other genes, white provided a superior source of phylogenetic
      information. However, white appears to have experienced accelerated rates of
      evolution in few lineages, the affinities of which are therefore suspect. In
      combined analyses, most of the inferred relationship were well-supported and in
      agreement with previous studies: monophyly of Anophelinae, basal position of
      Chagasia, monophyly of Anopheles subgenera, and subgenera Nyssorhynchus +
      Kerteszia as sister taxa. The results suggested also monophyletic origin of
      subgenera Cellia + Anopheles, and the white gene analysis supported genus
      Bironella as a sister taxon to Anopheles. The present data and other available
      evidence suggest a South American origin of Anophelinae, probably in the
      Mesozoic; a rapid diversification of Bironella and basal subgeneric lineages of
      Anopheles, potentially associated with the breakup of Gondwanaland; and a
      relatively recent and rapid dispersion of subgenus Anopheles.
FAU - Krzywinski, J
AU  - Krzywinski J
AD  - Department of Biological Sciences, University of Notre Dame, P.O. Box 369, Notre 
      Dame, Indiana 46556-0369, USA. jaroslaw.krzywinski.1@nd.edu
FAU - Wilkerson, R C
AU  - Wilkerson RC
FAU - Besansky, N J
AU  - Besansky NJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (ATP-Binding Cassette Transporters)
RN  - 0 (Codon)
RN  - 0 (DNA, Ribosomal)
RN  - 0 (Drosophila Proteins)
RN  - 0 (Eye Proteins)
RN  - 0 (Insect Proteins)
RN  - 0 (w protein, Drosophila)
RN  - EC 1.1.1.49 (Glucosephosphate Dehydrogenase)
SB  - IM
MH  - *ATP-Binding Cassette Transporters
MH  - Animals
MH  - Base Composition
MH  - Base Sequence
MH  - Codon/genetics
MH  - Culicidae/*classification/*genetics
MH  - DNA, Ribosomal/genetics
MH  - Drosophila Proteins/genetics
MH  - Evolution, Molecular
MH  - Eye Proteins/genetics
MH  - *Genes, Insect
MH  - Glucosephosphate Dehydrogenase/genetics
MH  - Insect Proteins/genetics
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Species Specificity
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):540-56.

PMID- 12116651
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - Bias in phylogenetic estimation and its relevance to the choice between parsimony
      and likelihood methods.
PG  - 525-39
FAU - Swofford, D L
AU  - Swofford DL
AD  - Laboratory of Molecular Systematics, National Museum of Natural History,
      Smithsonian Institution Museum Support Center, 4210 Silver Hill Road, Suitland,
      Maryland 20746, USA. swofford@lms.si.edu
FAU - Waddell, P J
AU  - Waddell PJ
FAU - Huelsenbeck, J P
AU  - Huelsenbeck JP
FAU - Foster, P G
AU  - Foster PG
FAU - Lewis, P O
AU  - Lewis PO
FAU - Rogers, J S
AU  - Rogers JS
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bias
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Stochastic Processes
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):525-39.

PMID- 12116650
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - Bacterial species and speciation.
PG  - 513-24
AB  - Bacteria are profoundly different from eukaryotes in their patterns of genetic
      exchange. Nevertheless, ecological diversity is organized in the same way across 
      all of life: individual organisms fall into more less discrete clusters on the
      basis of their phenotypic, ecological, and DNA sequence characteristics. Each
      sequence cluster in the bacterial world appears to correspond to an "ecotype,"
      defined as a population of cells in the same ecological niche, which would all be
      out-competed by any adaptive mutant coming from the population. Ecotypes, so
      defined, share many of the dynamic properties attributed to eukaryotic species:
      genetic diversity within an ecotype is limited by a force of cohesion (in this
      case, periodic selection); different ecotypes are free to diverge without
      constraint from one another; and ecotypes are ecologically distinct. Also,
      ecotypes can be discovered and classified as DNA sequence clusters, even when we 
      are ignorant of their ecology. Owing to the rarity and promiscuity of bacterial
      genetic exchange, speciation in the bacterial world is expected to be much less
      constrained than in the world of animals and plants.
FAU - Cohan, F M
AU  - Cohan FM
AD  - Department of Biology, Wesleyan University, Middletown, Connecticut 06459-0170,
      USA. fcohan@wesleyan.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bacteria/*classification/*genetics
MH  - Ecosystem
MH  - Genetic Variation
MH  - Phylogeny
MH  - Recombination, Genetic
MH  - Species Specificity
RF  - 87
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150118398 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):513-24. doi: 10.1080/10635150118398.

PMID- 12116649
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - Genomic and phylogenetic perspectives on the evolution of prokaryotes.
PG  - 497-512
AB  - Prokaryotes have been at the forefront of the genome sequencing revolution. Many 
      genomes have been completely sequenced, revealing much about bacterial and
      archaeal genome content and organization. Yet, a meaningful evolutionary picture 
      of prokaryotes still eludes us. Much of the problem lies in understanding the
      mode and tempo of genome evolution. Here phenylalanyl-tRNA synthetase is used as 
      an example of the complex interplay among lateral gene transfer, operon
      recombination, and gene recruitment in the evolution of some prokaryotic genes.
      Promising new approaches to genomic analyses, which could add to our
      understanding prokaryotic evolution and help in their classification, are
      discussed.
FAU - Brown, J R
AU  - Brown JR
AD  - Microbial Bioinformatics Department, Division of Bioinformatics, Glaxo SmithKline
      Pharmaceuticals, 1250 South Collegeville Road, P.O. Box 5089, UP1345,
      Collegeville, Pennsylvania 19426-0989, USA. James_R_Brown@gsk.com
LA  - eng
PT  - Journal Article
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Protein Subunits)
RN  - EC 6.1.1.20 (Phenylalanine-tRNA Ligase)
SB  - IM
MH  - Amino Acid Sequence
MH  - Archaea/classification/*genetics
MH  - Bacteria/classification/*genetics
MH  - Conserved Sequence
MH  - Evolution, Molecular
MH  - Genome
MH  - Molecular Sequence Data
MH  - Operon
MH  - Phenylalanine-tRNA Ligase/chemistry/genetics
MH  - *Phylogeny
MH  - Prokaryotic Cells
MH  - Protein Subunits
MH  - Sequence Homology, Amino Acid
RF  - 101
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150117729 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):497-512. doi: 10.1080/10635150117729.

PMID- 12116648
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - Catalyzing bacterial speciation: correlating lateral transfer with genetic
      headroom.
PG  - 479-96
AB  - Unlike crown eukaryotic species, microbial species are created by continual
      processes of gene loss and acquisition promoted by horizontal genetic transfer.
      The amounts of foreign DNA in bacterial genomes, and the rate at which this is
      acquired, are consistent with gene transfer as the primary catalyst for microbial
      differentiation. However, the rate of successful gene transfer varies among
      bacterial lineages. The heterogeneity in foreign DNA content is directly
      correlated with amount of genetic headroom intrinsic to a bacterial species.
      Genetic headroom reflects the amount of potentially dispensable
      information--reflected in codon usage bias and codon context bias--that can be
      transiently sacrificed to allow experimentation with functions introduced by gene
      transfer. In this way, genetic headroom offers a potential metric for assessing
      the propensity of a lineage to speciate.
FAU - Lawrence, J G
AU  - Lawrence JG
AD  - Department of Biological Sciences, University of Pittsburgh, Pittsburgh,
      Pennsylvania 15260, USA. jlawrenc@pitt.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
RN  - 0 (DNA, Bacterial)
SB  - IM
MH  - Bacteria/*classification/*genetics/metabolism
MH  - Codon/genetics
MH  - DNA, Bacterial/genetics
MH  - Ecosystem
MH  - Evolution, Molecular
MH  - Gene Transfer, Horizontal
MH  - Genome, Bacterial
MH  - Models, Genetic
MH  - Point Mutation
MH  - Species Specificity
RF  - 97
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150120286 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):479-96. doi: 10.1080/10635150120286.

PMID- 12116647
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - Environmental diversity of bacteria and archaea.
PG  - 470-8
AB  - The microbial way of life spans at least 3.8 billion years of evolution.
      Microbial organisms are pervasive, ubiquitous, and essential components of all
      ecosystems. The geochemical composition of Earth's biosphere has been molded
      largely by microbial activities. Yet, despite the predominance of microbes during
      the course of life's history, general principles and theory of microbial
      evolution and ecology are not well developed. Until recently, investigators had
      no idea how accurately cultivated microorganisms represented overall microbial
      diversity. The development of molecular phylogenetics has recently enabled
      characterization of naturally occurring microbial biota without cultivation. Free
      from the biases of culture-based studies, molecular phylogenetic surveys have
      revealed a vast array of new microbial groups. Many of these new microbes are
      widespread and abundant among contemporary microbiota and fall within novel
      divisions that branch deep within the tree of life. The breadth and extent of
      extant microbial diversity has become much clearer. A remaining challenge for
      microbial biologists is to better characterize the biological properties of these
      newly described microbial taxa. This more comprehensive picture will provide much
      better perspective on the natural history, ecology, and evolution of extant
      microbial life.
FAU - DeLong, E F
AU  - DeLong EF
AD  - Monterey Bay Aquarium Research Institute, 7700 Sandholdt Rd., Moss Landing,
      California 95039, USA. delong@mbari.org
FAU - Pace, N R
AU  - Pace NR
LA  - eng
PT  - Journal Article
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Archaea/classification/*genetics
MH  - Bacteria/classification/*genetics
MH  - Ecosystem
MH  - Environmental Microbiology
MH  - Genetic Variation
MH  - Phylogeny
RF  - 38
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):470-8.

PMID- 12116646
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - The evolutionary biology of ... everything.
PG  - 468-9
FAU - Kane, M D
AU  - Kane MD
AD  - Department of Entomology, National Museum of Natural History, Smithsonian
      Institution, Washington, DC 20560, USA. mkane@nsf.gov
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Archaea/genetics
MH  - Bacteria/genetics
MH  - *Biological Evolution
MH  - Microbiology
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150120966 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):468-9. doi: 10.1080/10635150120966.

PMID- 12116645
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 4
DP  - 2001 Aug
TI  - The troubled growth of statistical phylogenetics.
PG  - 465-7
FAU - Felsenstein, J
AU  - Felsenstein J
AD  - Department of Genetics, University of Washington, Seattle, Washington 98919, USA.
      joe@genetics.washington.edu
LA  - eng
PT  - Historical Article
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biometry/history
MH  - Classification
MH  - History, 20th Century
MH  - Humans
MH  - Periodicals as Topic/history
MH  - *Phylogeny
MH  - Societies, Scientific/history
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150119297 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Aug;50(4):465-7. doi: 10.1080/10635150119297.

PMID- 12116644
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - Difficulties in detecting hybridization.
PG  - 978-82
FAU - Holder, M T
AU  - Holder MT
AD  - Section of Integrative Biology, School of Biological Sciences, Institute for
      Cellular and Molecular Biology, University of Texas, Austin, Texas 78712, USA.
      mtholder@mail.utexas.edu
FAU - Anderson, J A
AU  - Anderson JA
FAU - Holloway, A K
AU  - Holloway AK
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Classification
MH  - Computer Simulation
MH  - *Hybridization, Genetic
MH  - Models, Genetic
MH  - Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462911 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):978-82. doi: 10.1080/106351501753462911.

PMID- 12116643
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - Complexity of the likelihood surface for a large DNA dataset.
PG  - 970-8
FAU - Salter, L A
AU  - Salter LA
AD  - Department of Mathematics and Statistics, University of New Mexico, Albuquerque, 
      New Mexico 87131, USA. salter@stat.unm.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Biometry
MH  - DNA/*genetics
MH  - *Databases, Nucleic Acid
MH  - *Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462902 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):970-8. doi: 10.1080/106351501753462902.

PMID- 12116642
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - Sequence alignment of 18S ribosomal RNA and the basal relationships of Adephagan 
      beetles: evidence for monophyly of aquatic families and the placement of
      Trachypachidae.
PG  - 945-69
AB  - Current hypotheses regarding family relationships in the suborder Adephaga
      (Coleoptera) are conflicting. Here we report full-length 18S ribosomal RNA
      sequences of 39 adephagans and 13 outgroup taxa. Data analysis focused on the
      impact of sequence alignment on tree topology, using two principally different
      approaches. Tree alignments, which seek to minimize indels and substitutions on
      the tree in a single step, as implemented in an approximate procedure by the
      computer program POY, were contrasted with a more traditional procedure based on 
      alignments followed by phylogenetic inference based on parsimony, likelihood, and
      distance analyses. Despite substantial differences between the procedures,
      phylogenetic conclusions regarding basal relationships within Adephaga and
      relationships between the four suborders of Coleoptera were broadly similar. The 
      analysis weakly supports monophyly of Adephaga, with Polyphaga usually as its
      sister, and the two small suborders Myxophaga and Archostemata basal to them. In 
      some analyses, however, Polyphaga was reconstructed as having arisen from within 
      Hydradephaga. Adephaga generally split into two monophyletic groups,
      corresponding to the terrestrial Geadephaga and the aquatic Hydradephaga, as
      initially proposed by Crowson in 1955, consistent with a single colonization of
      the aquatic environment by adephagan ancestors and contradicting the recent
      proposition of three independent invasions. A monophyletic Hydradephaga is
      consistently, though not strongly, supported under most analyses, and a
      parametric bootstrapping test significantly rejects an hypothesis of
      nonmonophyly. The enigmatic Trachypachidae, which exhibit many similarities to
      aquatic forms but whose species are entirely terrestrial, were usually recovered 
      as a basal lineage within Geadephaga. Strong evidence opposes the view that
      terrestrial trachypachids are related to the dytiscoid water beetles.
FAU - Shull, V L
AU  - Shull VL
AD  - Department of Entomology, Natural History Museum, London SW7 5BD, UK.
FAU - Vogler, A P
AU  - Vogler AP
FAU - Baker, M D
AU  - Baker MD
FAU - Maddison, D R
AU  - Maddison DR
FAU - Hammond, P M
AU  - Hammond PM
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 18S)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Coleoptera/*classification/*genetics
MH  - DNA/genetics
MH  - Databases, Nucleic Acid
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - RNA, Ribosomal, 18S/*genetics
MH  - Sequence Alignment
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462894 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):945-69. doi: 10.1080/106351501753462894.

PMID- 12116641
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - Unraveling the evolutionary radiation of the families of the Zingiberales using
      morphological and molecular evidence.
PG  - 926-44
AB  - The Zingiberales are a tropical group of monocotyledons that includes bananas,
      gingers, and their relatives. The phylogenetic relationships among the eight
      families currently recognized are investigated here by using parsimony and
      maximum likelihood analyses of four character sets: morphological features (1),
      and sequence data of the (2) chloroplast rbcL gene, (3) chloroplast atpB gene,
      and (4) nuclear 18S rDNA gene. Outgroups for the analyses include the closely
      related Commelinaceae + Philydraceae + Haemodoraceae + Pontederiaceae +
      Hanguanaceae as well as seven more distantly related monocots and paleoherbs.
      Only slightly different estimates of evolutionary relationships result from the
      analysis of each character set. The morphological data yield a single fully
      resolved most-parsimonious tree. None of the molecular datasets alone completely 
      resolves interfamilial relationships. The analyses of the combined molecular
      dataset provide more resolution than do those of individual genes, and the
      addition of the morphological data provides a well-supported estimate of
      phylogenetic relationships: (Musaceae ((Strelitziaceae, Lowiaceae) (Heliconiaceae
      ((Zingiberaceae, Costaceae) (Cannaceae, Marantaceae))))). Evidence from branch
      lengths in the parsimony analyses and from the fossil record suggests that the
      Zingiberales originated in the Early Cretaceous and underwent a rapid radiation
      in the mid-Cretaceous, by which time most extant family lineages had diverged.
FAU - Kress, W J
AU  - Kress WJ
AD  - Department of Systematic Biology-Botany, MRC-166, National Museum of Natural
      History, Smithsonian Institution, Washington, DC 20560, USA.
      kress.john@nmnh.si.edu
FAU - Prince, L M
AU  - Prince LM
FAU - Hahn, W J
AU  - Hahn WJ
FAU - Zimmer, E A
AU  - Zimmer EA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
RN  - 0 (RNA, Ribosomal, 18S)
RN  - EC 4.1.1.39 (RbcL protein, plastid)
RN  - EC 4.1.1.39 (Ribulose-Bisphosphate Carboxylase)
SB  - IM
MH  - DNA, Plant/genetics
MH  - Databases, Factual
MH  - Databases, Genetic
MH  - *Evolution, Molecular
MH  - Fossils
MH  - Genes, Plant
MH  - Models, Genetic
MH  - Phylogeny
MH  - RNA, Ribosomal, 18S/genetics
MH  - Ribulose-Bisphosphate Carboxylase/genetics
MH  - Zingiberales/anatomy &amp; histology/*classification/*genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462885 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):926-44. doi: 10.1080/106351501753462885.

PMID- 12116640
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - A likelihood approach to estimating phylogeny from discrete morphological
      character data.
PG  - 913-25
AB  - Evolutionary biologists have adopted simple likelihood models for purposes of
      estimating ancestral states and evaluating character independence on specified
      phylogenies; however, for purposes of estimating phylogenies by using discrete
      morphological data, maximum parsimony remains the only option. This paper
      explores the possibility of using standard, well-behaved Markov models for
      estimating morphological phylogenies (including branch lengths) under the
      likelihood criterion. An important modification of standard Markov models
      involves making the likelihood conditional on characters being variable, because 
      constant characters are absent in morphological data sets. Without this
      modification, branch lengths are often overestimated, resulting in potentially
      serious biases in tree topology selection. Several new avenues of research are
      opened by an explicitly model-based approach to phylogenetic analysis of discrete
      morphological data, including combined-data likelihood analyses (morphology +
      sequence data), likelihood ratio tests, and Bayesian analyses.
FAU - Lewis, P O
AU  - Lewis PO
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut,
      Storrs, Connecticut 06269-3043, USA. paul.lewis@uconn.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biometry
MH  - Likelihood Functions
MH  - Markov Chains
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462876 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):913-25. doi: 10.1080/106351501753462876.

PMID- 12116639
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - On areas of endemism, with an example from the African Restionaceae.
PG  - 892-912
AB  - Areas of endemism are central to cladistic biogeography. The concept has been
      much debated in the past, and from this has emerged the generally accepted
      definition as an area to which at least two species are endemic. Protocols for
      locating areas of endemism have been neglected, and to date no attempt has been
      made to develop optimality criteria against which to evaluate competing
      hypotheses of areas of endemism. Here various protocols for finding areas of
      endemism are evaluated--protocols based on both phonetic and parsimony analyses, 
      on both unweighted data and data weighted by various criteria. The optimality
      criteria used to compare the performance of the methods include the number of
      species included in the areas of endemism, the number of areas delimited, and the
      degree of distributional congruency of the species restricted to each area of
      endemism. These methods are applied to the African Restionaceae in the Cape
      Floristic Region. Parsimony methods using weighted data are shown to perform best
      on the combination of all three optimality criteria. By varying the weighting
      parameters, the size of the areas of endemism can be varied. This provides a very
      useful tool for locating areas of endemism that satisfy prespecified scale
      criteria.
FAU - Linder, H P
AU  - Linder HP
AD  - Institute for Systematic Botany, CH-8008 Zurich, Switzerland.
      plinder@systbot.unizh.ch
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Demography
MH  - Ecosystem
MH  - Geography
MH  - *Magnoliopsida
MH  - South Africa
MH  - Species Specificity
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462867 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):892-912. doi: 10.1080/106351501753462867.

PMID- 12116638
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - War and peace in phylogenetics: a rejoinder on total evidence and consensus.
PG  - 881-91
AB  - For more than 10 years, systematists have been debating the superiority of
      character or taxonomic congruence in phylogenetic analysis. In this paper, we
      demonstrate that the competing approaches can converge to the same solution when 
      a consensus method that accounts for branch lengths is selected. Thus, we propose
      to use both methods in combination, as a way to corroborate the results of
      combined and separate analyses. This so-called "global congruence" approach is
      tested with a wide variety of examples sampled from the literature, and the
      results are compared with those obtained by standard consensus methods. Our
      analyses show that when the total evidence and consensus trees differ
      topologically, collapsing weakly supported nodes with low bootstrap support
      usually improves "global congruence".
FAU - Levasseur, C
AU  - Levasseur C
AD  - Departement de sciences biologiques, Universite de Montreal, C.P. 6128,
      Succursale centre-ville, Montreal, Quebec, H3C 3J7, Canada.
      claudine.lavasseur@umontreal.ca
FAU - Lapointe, F J
AU  - Lapointe FJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Classification
MH  - Databases, Genetic
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462858 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):881-91. doi: 10.1080/106351501753462858.

PMID- 12116637
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - Combining and comparing morphometric shape descriptors with a molecular
      phylogeny: the case of fruit type evolution in Bornean Lithocarpus (Fagaceae).
PG  - 860-80
AB  - Fruit type in the genus Lithocarpus (Fagaceae) includes both classic oak acorns
      and novel modifications. Bornean taxa with modified fruits can be separated into 
      two sections (Synaedrys and Lithocarpus) based on subtle shape differences. By
      following strict criteria for homology and representation, this variation in
      shape can be captured and the sections distinguished by using elliptic Fourier or
      eigenshape analysis. Phenograms of fruit shape, constructed by using restricted
      maximum likelihood techniques and these morphometric descriptors, were
      incorporated into combined and comparative analyses with molecular sequence data 
      from the internal transcribed spacer (ITS) region of the nuclear rDNA, using
      branch-weighted matrix representation. The combined analysis strongly suggested
      independent derivation of the novel fruit type in the two sections from different
      acornlike ancestors, while the comparative analysis indicated frequent decoupling
      between the molecular and morphological changes as inferred at well-supported
      nodes. The acorn fruit type has undergone little modification between ingroup and
      outgroup, despite large molecular distance. Greater morphological than molecular 
      change was inferred at critical transitions between acorn and novel fruit types, 
      particularly for section Lithocarpus. The combination of these two different
      types of data improved our understanding of the macroevolution of fruit type in
      this difficult group, and the comparative analysis highlighted the significant
      incongruities in evolutionary pattern between the two datasets.
FAU - Cannon, C H
AU  - Cannon CH
AD  - Institute for Biodiversity and Environmental Conservation, University of
      Malaysia, Sarawak, Kota Samarahan, Sarawak, Malaysia 94300. chc2@duke.edu
FAU - Manos, P S
AU  - Manos PS
LA  - eng
SI  - GENBANK/AF389086
SI  - GENBANK/AF389087
SI  - GENBANK/AF389088
SI  - GENBANK/AF389089
SI  - GENBANK/AF389090
SI  - GENBANK/AF389091
SI  - GENBANK/AF389092
SI  - GENBANK/AF389093
SI  - GENBANK/AF389094
SI  - GENBANK/AF389095
SI  - GENBANK/AF389096
SI  - GENBANK/AF389097
SI  - GENBANK/AF389098
SI  - GENBANK/AF389099
SI  - GENBANK/AF389100
SI  - GENBANK/AF389101
SI  - GENBANK/AF389102
SI  - GENBANK/AF389103
SI  - GENBANK/AF389104
SI  - GENBANK/AF389105
SI  - GENBANK/AF389106
SI  - GENBANK/AF389107
SI  - GENBANK/AF389410
SI  - GENBANK/AF389428
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Borneo
MH  - DNA, Plant/genetics
MH  - Evolution, Molecular
MH  - Fagaceae/*anatomy &amp; histology/classification/*genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462849 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):860-80. doi: 10.1080/106351501753462849.

PMID- 12116636
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - Molecular data indicate the protostome affinity of brachiopods.
PG  - 848-59
AB  - Although the phylogenetic position of brachiopods has always been subject to
      debate, many authors place them as a sister group to deuterostomes on the basis
      of morphological and developmental characters. However, molecular phylogeny
      consistently places them among protostomes. More precisely, brachiopods are
      predicted to branch inside the lophotrochozoan assemblage, together with
      annelids, molluscs, nemerteans, flatworms, and others. That result has been
      criticized on the basis of (1) prior knowledge of brachiopod morphology and (2)
      the known limitations of molecular phylogenies. Here I review recent data of
      molecular origin, particularly those displaying qualitative properties close to
      those of morphological characters. The complement of Hox genes present in all
      metazoa tested to date has proved to be a powerful tool for broad phylogenetic
      reconstruction. The mitochondrial genome also provides qualitative characters,
      showing discrete events of gene rearrangements. After discussing the data and the
      way they should be interpreted in the perspective of several hypotheses for
      metazoan phylogeny, I conclude that they argue strongly in favor of the
      protostome (and lophotrochozoan) affinity of the brachiopods. There is therefore 
      a need for a reinterpretation of brachiopod morphological and developmental
      characters. I also identify some research axes on brachiopod morphology.
FAU - de Rosa, R
AU  - de Rosa R
AD  - Centre de Genetique Moleculaire, UPR 9061 CNRS, Batiment 26, avenue de la
      Terrasse, 91198 Gif-sur-Yvette, France. renaud.derosa@cgm.cnrs-gif.fr
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Homeodomain Proteins)
SB  - IM
MH  - Amino Acid Sequence
MH  - Animals
MH  - DNA, Mitochondrial/genetics
MH  - Genes, Homeobox
MH  - Homeodomain Proteins/genetics
MH  - Invertebrates/*classification/*genetics
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Sequence Homology, Amino Acid
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462830 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):848-59. doi: 10.1080/106351501753462830.

PMID- 12116635
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - Phylogeny of Saxifragales (angiosperms, eudicots): analysis of a rapid, ancient
      radiation.
PG  - 817-47
AB  - Rapid, ancient radiations pose one of the most difficult challenges for
      phylogenetic estimation. We used DNA sequence data of 9,006 aligned base pairs
      from five genes (chloroplast atpB, matK, rbcL, and 18S and 26S nrDNA) to
      elucidate relationships among major lineages of Saxifragales (angiosperms,
      eudicots). These relationships were poorly supported in previous studies,
      apparently because the lineages originated in rapid succession. Using an array of
      methods that explicitly incorporate assumptions about evolutionary process
      (weighted maximum parsimony, maximum likelihood, LogDet/paralinear transformed
      distances), we show that the initial diversification of Saxifragales was indeed
      rapid. We suggest that the poor resolution of our best phylogenetic estimate is
      not due to violations of assumptions or to combining data partitions having
      conflicting histories or processes. We show that estimated branch lengths during 
      the initial diversification are exceedingly short, and we estimate that acquiring
      sufficient sequence data to resolve these relationships would require an
      extraordinary effort (approximately 10(7) bp), assuming a linear increase in
      branch support with branch length. However, our simulation of much larger data
      sets containing a distribution of phylogenetic signal similar to that of the five
      sampled gene sequences suggests a limit to achievable branch support. Using
      statistical tests of differences in the likelihoods of topologies, we evaluated
      whether the initial radiation of Saxifragales involved the simultaneous origin of
      major lineages. Our results are consistent with predictions that resolving the
      branching order of rapid, ancient radiations requires sampling characters that
      evolved rapidly at the time of the radiation but have since experienced a slower 
      evolutionary rate.
FAU - Fishbein, M
AU  - Fishbein M
AD  - School of Biological Sciences, Washington State University, Pullman, Washington
      99164-4236, USA. fishbein@uidaho.edu
FAU - Hibsch-Jetter, C
AU  - Hibsch-Jetter C
FAU - Soltis, D E
AU  - Soltis DE
FAU - Hufford, L
AU  - Hufford L
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
RN  - 0 (DNA, Plant)
RN  - 0 (DNA, Ribosomal)
RN  - EC 2.7.7.- (Nucleotidyltransferases)
RN  - EC 2.7.7.- (mRNA maturase)
RN  - EC 3.1.- (Endoribonucleases)
RN  - EC 4.1.1.39 (RbcL protein, plastid)
RN  - EC 4.1.1.39 (Ribulose-Bisphosphate Carboxylase)
SB  - IM
MH  - Base Composition
MH  - DNA, Chloroplast/genetics
MH  - DNA, Plant/chemistry/genetics
MH  - DNA, Ribosomal/genetics
MH  - Endoribonucleases/genetics
MH  - Evolution, Molecular
MH  - Genes, Plant
MH  - Genome, Plant
MH  - Magnoliopsida/*classification/*genetics
MH  - Models, Genetic
MH  - Nucleotidyltransferases/genetics
MH  - *Phylogeny
MH  - Ribulose-Bisphosphate Carboxylase/genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462821 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):817-47. doi: 10.1080/106351501753462821.

PMID- 12116634
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - Phylogeny of Trichoptera (caddisflies): characterization of signal and noise
      within multiple datasets.
PG  - 781-816
AB  - Trichoptera are holometabolous insects with aquatic larvae that, together with
      the Lepidoptera, make up the Amphiesmenoptera. Despite extensive previous
      morphological work, little phylogenetic agreement has been reached about the
      relationship among the three suborders--Annulipalpia, Spicipalpia, and
      Integripalpia--or about the monophyly of Spicipalpia. In an effort to resolve
      this conflict, we sequenced fragments of the large and small subunit nuclear
      ribosomal RNAs (1078 nt; D1, D3, V4-5), the nuclear elongation factor 1 alpha
      gene (EF-1 alpha; 1098 nt), and a fragment of mitochondrial cytochrome oxidase I 
      (COI; 411 nt). Seventy adult and larval morphological characters were reanalyzed 
      and added to molecular data in a combined analysis. We evaluated signal and
      homoplasy in each of the molecular datasets and attempted to rank the particular 
      datasets according to how appropriate they were for inferring relationships among
      suborders. This evaluation included testing for conflict among datasets,
      comparing tree lengths among alternative hypotheses, measuring the left-skew of
      tree-length distributions from maximally divergent sets of taxa, evaluating the
      recovery of expected clades, visualizing whether or not substitutions were
      accumulating with time, and estimating nucleotide compositional bias. Although
      all these measures cast doubt on the reliability of the deep-level signal coming 
      from the nucleotides of the COI and EF-1 alpha genes, these data could still be
      included in combined analyses without overturning the results from the most
      conservative marker, the rRNA. The different datasets were found to be evolving
      under extremely different rates. A site-specific likelihood method for dealing
      with combined data with nonoverlapping parameters was proposed, and a similar
      weighting scheme under parsimony was evaluated. Among our phylogenetic
      conclusions, we found Annulipalpia to be the most basal of the three suborders,
      with Spicipalpia and Integripalpia forming a clade. Monophyly of Annulipalpia and
      Integripalpia was confirmed, but the relationships among spicipalpians remain
      equivocal.
FAU - Kjer, K M
AU  - Kjer KM
AD  - Department of Entomology, Rutgers University, Cook College, New Brunswick, New
      Jersey 08901, USA. kjer@aesop.rutgers.edu
FAU - Blahnik, R J
AU  - Blahnik RJ
FAU - Holzenthal, R W
AU  - Holzenthal RW
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Peptide Elongation Factor 1)
RN  - 0 (RNA, Ribosomal)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
EIN - Syst Biol 2002 Apr;51(2):385-7
MH  - Animals
MH  - Base Sequence
MH  - Biometry
MH  - Databases, Genetic
MH  - Electron Transport Complex IV/genetics
MH  - Genes, Insect
MH  - Insecta/*classification/enzymology/*genetics
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Peptide Elongation Factor 1/genetics
MH  - *Phylogeny
MH  - RNA, Ribosomal/genetics
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462812 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):781-816. doi: 10.1080/106351501753462812.

PMID- 12116633
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - Exploring data interaction and nucleotide alignment in a multiple gene analysis
      of Ips (Coleoptera: Scolytinae).
PG  - 758-80
AB  - The possibility of gene tree incongruence in a species-level phylogenetic
      analysis of the genus Ips (Coleoptera: Scolytidae) was investigated based on
      mitochondrial 16S rRNA (16S) and nuclear elongation factor-1 alpha (EF-1 alpha)
      sequences, and existing cytochrome oxidase I (COI) and nonmolecular data sets.
      Separate cladistic analyses of the data partitions resulted in partially
      discordant most-parsimonious trees but revealed only low conflict of the
      phylogenetic signal. Interactions among data partitions, which differed in the
      extent of sequence divergence (COI &gt; 16S &gt; EF-1 alpha), base composition, and
      homoplasy, revealed that much of the branch support emerges only in the
      simultaneous analysis, particularly for deeper nodes in the tree, which are
      almost entirely supported through "hidden support" (sensu Gatesy et al.,
      Cladistics 15:271-313, 1999). Apparent incongruence between data partitions is in
      part due to suboptimal alignments and bias of character transformations, but
      little evidence supports invoking incongruent phylogenetic histories of genetic
      loci. There is also no justification for eliminating or downweighting gene
      partitions on the basis of their apparent homoplasy or incongruence with other
      partitions, because the signal emerges only in the interaction of all data. In
      comparison with traditional taxonomy, the pini, plastographus, and perturbatus
      groups are polyphyletic, whereas the grandicollis group is monophyletic except
      for inclusion of the (monophyletic) calligraphus group. The latidens group and
      some European species are distantly related and closer to other genera within
      Ipini. Our robust cladogram was used to revise the classification of Ips. We
      provide new diagnoses for Ips and four subgeneric taxa.
FAU - Cognato, A I
AU  - Cognato AI
AD  - Department of Entomology, Natural History Museum, London SW7 5BD, UK.
      a-cognato@tamu.edu
FAU - Vogler, A P
AU  - Vogler AP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Peptide Elongation Factor 1)
RN  - 0 (RNA, Ribosomal, 16S)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Coleoptera/classification/enzymology/*genetics
MH  - Databases, Nucleic Acid
MH  - Electron Transport Complex IV/genetics
MH  - *Genes, Insect
MH  - Peptide Elongation Factor 1/genetics
MH  - Phylogeny
MH  - RNA, Ribosomal, 16S/genetics
MH  - Sequence Alignment
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462803 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):758-80. doi: 10.1080/106351501753462803.

PMID- 12116632
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 6
DP  - 2001 Nov-Dec
TI  - A wish list for systematic biology.
PG  - 755-7
FAU - Donoghue, M J
AU  - Donoghue MJ
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven,
      Connecticut 06520, USA.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - *Biology/trends
MH  - Phylogeny
MH  - Publishing
MH  - *Societies, Scientific/trends
MH  - United States
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501753462795 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Nov-Dec;50(6):755-7. doi: 10.1080/106351501753462795.

PMID- 12116597
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190523
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 1
DP  - 2001 Feb
TI  - Phylogenetic utility of different types of molecular data used to infer
      evolutionary relationships among stalk-eyed flies (Diopsidae).
PG  - 87-105
AB  - A phylogenetic hypothesis of relationships among 33 species of stalk-eyed flies
      was generated from a molecular data set comprising three mitochondrial and three 
      nuclear gene regions. A combined analysis of all the data equally weighted
      produced a single most-parsimonious cladogram with relatively strong support at
      the majority of nodes. The phylogenetic utility of different classes of molecular
      data was also examined. In particular, using a number of different measures of
      utility in both a combined and separate analysis framework, we focused on the
      distinction between mitochondrial and nuclear genes and between faster-evolving
      characters and slower-evolving characters. For the first comparison, by nearly
      any measure of utility, the nuclear genes are substantially more informative for 
      resolving diopsid relationships than are the mitochondrial genes. The nuclear
      genes exhibit less homoplasy, are less incongruent with one another and with the 
      combined data, and contribute more support to the combined analysis topology than
      do the mitochondrial genes. Results from the second comparison, however, provide 
      little evidence of a clear difference in utility. Despite indications of rapid
      divergence and saturation, faster-evolving characters in both the nuclear and
      mitochondrial data sets still provide substantial phylogenetic signal. In
      general, inclusion of the more rapidly evolving data consistently improves the
      congruence among partitions.
FAU - Baker, R H
AU  - Baker RH
AD  - Department of Entomology, American Museum of Natural History, 79th Street at
      Central Park West, New York, NY 10024, USA.
FAU - Wilkinson, G S
AU  - Wilkinson GS
FAU - DeSalle, R
AU  - DeSalle R
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (ATP-Binding Cassette Transporters)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Drosophila Proteins)
RN  - 0 (Eye Proteins)
RN  - 0 (Peptide Elongation Factor 1)
RN  - 0 (Proto-Oncogene Proteins)
RN  - 0 (RNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 16S)
RN  - 0 (RNA, ribosomal, 12S)
RN  - 0 (Wnt1 Protein)
RN  - 0 (w protein, Drosophila)
RN  - 0 (wg protein, Drosophila)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - *ATP-Binding Cassette Transporters
MH  - Animals
MH  - Base Sequence
MH  - Biometry
MH  - DNA, Mitochondrial/genetics
MH  - Diptera/*classification/*genetics
MH  - Drosophila Proteins/genetics
MH  - Electron Transport Complex IV/genetics
MH  - Evolution, Molecular
MH  - Eye Proteins/genetics
MH  - Female
MH  - Genes, Insect
MH  - Male
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Peptide Elongation Factor 1/genetics
MH  - *Phylogeny
MH  - Proto-Oncogene Proteins/genetics
MH  - RNA, Ribosomal/genetics
MH  - RNA, Ribosomal, 16S/genetics
MH  - Species Specificity
MH  - Wnt1 Protein
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Feb;50(1):87-105.

PMID- 12116596
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 1
DP  - 2001 Feb
TI  - Stochastic search strategy for estimation of maximum likelihood phylogenetic
      trees.
PG  - 7-17
AB  - The maximum likelihood (ML) method of phylogenetic tree construction is not as
      widely used as other tree construction methods (e.g., parsimony,
      neighbor-joining) because of the prohibitive amount of time required to find the 
      ML tree when the number of sequences under consideration is large. To overcome
      this difficulty, we propose a stochastic search strategy for estimation of the ML
      tree that is based on a simulated annealing algorithm. The algorithm works by
      moving through tree space by way of a "local rearrangement" strategy so that
      topologies that improve the likelihood are always accepted, whereas those that
      decrease the likelihood are accepted with a probability that is related to the
      proportionate decrease in likelihood. Besides greatly reducing the time required 
      to estimate the ML tree, the stochastic search strategy is less likely to become 
      trapped in local optima than are existing algorithms for ML tree estimation. We
      demonstrate the success of the modified simulated annealing algorithm by
      comparing it with two existing algorithms (Swofford's PAUP* and Felsenstein's
      DNAMLK) for several theoretical and real data examples.
FAU - Salter, L A
AU  - Salter LA
AD  - Department of Mathematics and Statistics, University of New Mexico, Albuquerque, 
      NM 87131, USA. salter@stat.unm.edu
FAU - Pearl, D K
AU  - Pearl DK
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (DNA, Viral)
RN  - 63231-63-0 (RNA)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Algorithms
MH  - Biometry
MH  - DNA/genetics
MH  - DNA, Mitochondrial/genetics
MH  - DNA, Viral/genetics
MH  - Humans
MH  - Likelihood Functions
MH  - Papillomaviridae/classification/genetics
MH  - *Phylogeny
MH  - RNA/genetics
MH  - Stochastic Processes
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Feb;50(1):7-17.

PMID- 12116595
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 1
DP  - 2001 Feb
TI  - Exploring among-site rate variation models in a maximum likelihood framework
      using empirical data: effects of model assumptions on estimates of topology,
      branch lengths, and bootstrap support.
PG  - 67-86
AB  - We have investigated the effects of different among-site rate variation models on
      the estimation of substitution model parameters, branch lengths, topology, and
      bootstrap proportions under minimum evolution (ME) and maximum likelihood (ML).
      Specifically, we examined equal rates, invariable sites, gamma-distributed rates,
      and site-specific rates (SSR) models, using mitochondrial DNA sequence data from 
      three protein-coding genes and one tRNA gene from species of the New Zealand
      cicada genus Maoricicada. Estimates of topology were relatively insensitive to
      the substitution model used; however, estimates of bootstrap support, branch
      lengths, and R-matrices (underlying relative substitution rate matrix) were
      strongly influenced by the assumptions of the substitution model. We identified
      one situation where ME and ML tree building became inaccurate when implemented
      with an inappropriate among-site rate variation model. Despite the fact the SSR
      models often have a better fit to the data than do invariable sites and gamma
      rates models, SSR models have some serious weaknesses. First, SSR rate parameters
      are not comparable across data sets, unlike the proportion of invariable sites or
      the alpha shape parameter of the gamma distribution. Second, the extreme
      among-site rate variation within codon positions is problematic for SSR models,
      which explicitly assume rate homogeneity within each rate class. Third, the SSR
      models appear to give severe underestimates of R-matrices and branch lengths
      relative to invariable sites and gamma rates models in this example. We recommend
      performing phylogenetic analyses under a range of substitution models to test the
      effects of model assumptions not only on estimates of topology but also on
      estimates of branch length and nodal support.
FAU - Buckley, T R
AU  - Buckley TR
AD  - Institute for Molecular Systematics, School of Biological Sciences, Victoria
      University of Wellington, P.O. Box 600, Wellington, New Zealand.
      tbuckley@duke.edu
FAU - Simon, C
AU  - Simon C
FAU - Chambers, G K
AU  - Chambers GK
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (RNA, Transfer, Asp)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Biometry
MH  - DNA, Mitochondrial/genetics
MH  - Databases, Genetic
MH  - Evolution, Molecular
MH  - Genes, Insect
MH  - Genetic Variation
MH  - Hemiptera/classification/genetics
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - Nucleic Acid Conformation
MH  - *Phylogeny
MH  - RNA, Transfer, Asp/chemistry/genetics
MH  - Species Specificity
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Feb;50(1):67-86.

PMID- 12116594
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 1
DP  - 2001 Feb
TI  - Changing the landscape: a new strategy for estimating large phylogenies.
PG  - 60-6
AB  - In this paper we describe a new heuristic strategy designed to find optimal
      (parsimonious) trees for data sets with large numbers of taxa and characters.
      This new strategy uses an iterative searching process of branch swapping with
      equally weighted characters, followed by swapping with reweighted characters.
      This process increases the efficiency of the search because, after each round of 
      swapping with reweighted characters, the subsequent swapping with equal weights
      will start from a different group (island) of trees that are only slightly, if at
      all, less optimal. In contrast, conventional heuristic searching with constant
      equal weighting can become trapped on islands of suboptimal trees. We test the
      new strategy against a conventional strategy and a modified conventional strategy
      and show that, within a given time, the new strategy finds trees that are
      markedly more parsimonious. We also compare our new strategy with a recent,
      independently developed strategy known as the Parsimony Ratchet.
FAU - Quicke, D L
AU  - Quicke DL
AD  - Unit of Parasite Systematics, CABI, Bioscience UK Centre (Ascot), Department of
      Biology, Imperial College, Silwood Park, Ascot, Berkshire, SL5 7PY.
      d.quicke@ic.ac.uk
FAU - Taylor, J
AU  - Taylor J
FAU - Purvis, A
AU  - Purvis A
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biometry
MH  - Databases, Genetic
MH  - Phthiraptera/classification/genetics
MH  - *Phylogeny
MH  - Software Design
MH  - Wasps/classification/genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Feb;50(1):60-6.

PMID- 12116593
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 1
DP  - 2001 Feb
TI  - Incongruence of mitochondrial and nuclear gene trees in the Carabid beetles
      Ohomopterus.
PG  - 39-59
AB  - We studied the molecular phylogeny of the carabid subgenus Ohomopterus (genus
      Carabus), using two mitochondrial (mt) DNA regions (16SrRNA and NADH
      dehydrogenase subunit 5) and three nuclear DNA regions (wingless,
      phosphoenolpyruvate carboxykinase, and an anonymous locus). We revisited the
      previously reported incongruence between the distribution of mtDNA markers and
      morphologically defined species (Su et al., 1996; J. Mol. Evol. 43:662-671),
      which those authors attributed to "type switching", a concerted change in many
      morphological characters that results in the repeated evolution of a particular
      morphological type. Our mtDNA gene tree obtained from 44 individuals representing
      all 15 currently recognized species of Ohomopterus revealed that haplotypes
      isolated from individuals of a single "species" were frequently separated into
      distant clades, confirming the previous report. The three nuclear markers
      generally conformed better-with the morphologically defined species than did the 
      mitochondrial markers. The phylogenetic signal in mtDNA and nuclear DNA data
      differed strongly, and these two partitions were significantly incongruent with
      each other according to the incongruence length difference test of Farris et al. 
      (1994; Cladistics 10:315-320), although the three nuclear partitions were not
      homogeneous either. Our results did not support the type-switching hypothesis
      that had been proposed to fit the morphological data to the mitochondrial gene
      tree: The incongruence of the mtDNA tree with other nuclear markers indicates
      that the mtDNA-based tree does not reflect species history any better than the
      morphological data do. Incongruence of gene trees in Ohomopterus may have been
      promoted by the complex processes of geographic isolation and hybridization in
      the Japanese Archipelago that have led to occasional gene flow and recombination 
      between separated entities. The occurrence of reticulate patterns in this group
      is intriguing, because species of Ohomopterus exhibit extremely divergent
      genitalic structures that represent a highly efficient reproductive isolation
      mechanism.
FAU - Sota, T
AU  - Sota T
AD  - Department of Zoology, Faculty of Science, Kyoto University, Sakyo, Kyoto
      606-8502, Japan. sota@terra.zool.kyoto-u.ac.jp
FAU - Vogler, A P
AU  - Vogler AP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Drosophila Proteins)
RN  - 0 (Proto-Oncogene Proteins)
RN  - 0 (RNA, Ribosomal, 16S)
RN  - 0 (Wnt1 Protein)
RN  - 0 (wg protein, Drosophila)
RN  - EC 1.6.99.3 (NADH Dehydrogenase)
RN  - EC 4.1.1.32 (Phosphoenolpyruvate Carboxykinase (GTP))
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Biometry
MH  - Coleoptera/classification/*genetics
MH  - DNA, Mitochondrial/genetics
MH  - Databases, Nucleic Acid
MH  - *Drosophila Proteins
MH  - Female
MH  - *Genes, Insect
MH  - Japan
MH  - Male
MH  - NADH Dehydrogenase/genetics
MH  - Phosphoenolpyruvate Carboxykinase (GTP)/genetics
MH  - *Phylogeny
MH  - Proto-Oncogene Proteins/genetics
MH  - RNA, Ribosomal, 16S/genetics
MH  - Wnt1 Protein
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Feb;50(1):39-59.

PMID- 12116592
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 1
DP  - 2001 Feb
TI  - Molecular and morphological phylogenetic analysis of an insular radiation in
      Pacific black flies (Simulium).
PG  - 18-38
AB  - Ecological adaptation within islands may have figured prominently in the insular 
      radiation of black flies (subgenus Inseliellum) in the Society Islands, French
      Polynesia. To aid in understanding the sequence of ecological shifts in this
      group, we have constructed a phylogeny by using morphology, the cytochrome
      oxidase I (COI) gene, and the small ribosomal subunit (12S) gene. The strong
      influence of COI on the combined analysis tree was evident from its contribution 
      to the partitioned Bremer support (62%). The net effect of including 12S was to
      reduce overall tree support. Different character sets resolved different portions
      of the combined analysis tree, with COI resolving recent lineages, 12S resolving 
      basal relationships, and morphology supporting the monophyly of taxa having
      smaller larval feeding fans (oviceps group). The Partition Homogeneity and
      Kashino-Hasegawa tests indicated significant incongruence between morphological
      and mitochondrial data. The Templeton test revealed that morphology and the
      combined (COI + 12S) mitochondrial data were incongruent. This conflict stems
      primarily from disagreement over the monophyly of taxa having much smaller larval
      feeding fans. Either convergence in a subset of morphological characters, low
      phylogenetic signal among mitochondrial sequences, or lineage-sorting causing the
      mitochondrial data to track an incorrect evolutionary history may be responsible 
      for these results.
FAU - Joy, D A
AU  - Joy DA
AD  - Department of Biology, University of Vermont, Burlington, VT 05405-0086, USA.
      dajoy@zoo.uvm.edu
FAU - Conn, J E
AU  - Conn JE
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal)
RN  - 0 (RNA, ribosomal, 12S)
RN  - 9007-49-2 (DNA)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Biometry
MH  - DNA/genetics/isolation &amp; purification
MH  - Ecosystem
MH  - Electron Transport Complex IV/genetics
MH  - Evolution, Molecular
MH  - Genes, Insect
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Polynesia
MH  - RNA, Ribosomal/genetics
MH  - Sequence Alignment
MH  - Sequence Homology, Nucleic Acid
MH  - Simuliidae/*classification/enzymology/*genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Feb;50(1):18-38.

PMID- 12116591
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 1
DP  - 2001 Feb
TI  - Landmark coordinates aligned by procrustes analysis do not lie in Kendall's shape
      space.
PG  - 141-9
FAU - Slice, D E
AU  - Slice DE
AD  - Department of Ecology and Evolution, State University of New York, Stony Brook,
      NY 11794-5245, USA. Dennis.Slice@sunysb.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - *Biometry
MH  - Ecology
MH  - Mathematics
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150119110 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Feb;50(1):141-9. doi: 10.1080/10635150119110.

PMID- 12116590
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 1
DP  - 2001 Feb
TI  - Useful characters in gastropod phylogeny: soft information or hard facts?
PG  - 136-41
FAU - Schander, C
AU  - Schander C
AD  - Goteborg University, Department of Zoology, P.O. Box 463, SE-405 30 Goteborg,
      Sweden. c.schander@zool.gu.se
FAU - Sundberg, P
AU  - Sundberg P
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biometry
MH  - Mollusca/anatomy &amp; histology/*classification/*genetics
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Feb;50(1):136-41.

PMID- 12116589
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 1
DP  - 2001 Feb
TI  - Taxon sampling and the phylogenetic position of Passeriformes: evidence from 916 
      avian cytochrome b sequences.
PG  - 128-36
FAU - Johnson, K P
AU  - Johnson KP
AD  - Department of Biology, University of Utah, Salt Lake City, UT 84112-0840, USA.
      Kjohnson@inhs.uiuc.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Biometry
MH  - Birds/classification/genetics
MH  - Cytochrome b Group/*genetics
MH  - DNA/genetics
MH  - Databases, Nucleic Acid
MH  - *Phylogeny
MH  - Songbirds/*classification/*genetics
MH  - Species Specificity
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Feb;50(1):128-36.

PMID- 12116588
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 1
DP  - 2001 Feb
TI  - A partitioned likelihood analysis of swallowtail butterfly phylogeny
      (Lepidoptera:Papilionidae).
PG  - 106-27
AB  - Although it is widely agreed that data from multiple sources are necessary to
      confidently resolve phylogenetic relationships, procedures for accommodating and 
      incorporating heterogeneity in such data remain underdeveloped. We explored the
      use of partitioned, model-based analyses of heterogeneous molecular data in the
      context of a phylogenetic study of swallowtail butterflies (Lepidoptera:
      Papilionidae). Despite substantial basic and applied study, phylogenetic
      relationships among the major lineages of this prominent group remain
      contentious. We sequenced 3.3 kb of mitochondrial and nuclear DNA (2.3 kb of
      cytochrome oxidase I and II and 1.0 kb of elongation factor-1 alpha,
      respectively) from 22 swallowtails, including representatives of Baroniinae,
      Parnassiinae, and Papilioninae, and from several moth and butterfly outgroups.
      Using parsimony, we encountered considerable difficulty in resolving the deepest 
      splits among these taxa. We therefore chose two outgroups with undisputed
      relationships to each other and to Papilionidae and undertook detailed likelihood
      analyses of alternative topologies. Following from previous studies that have
      demonstrated substantial heterogeneity in the evolutionary dynamics among process
      partitions of these genes, we estimated evolutionary parameters separately for
      gene-based and codon-based partitions. These values were then used as the basis
      for examining the likelihoods of possible resolutions and rootings under several 
      partitioned and unpartitioned likelihood models. Partitioned models gave markedly
      better fits to the data than did unpartitioned models and supported different
      topologies. However, the most likely topology varied from model to model. The
      most likely ingroup topology under the best-fitting, six-partition GTR + gamma
      model favors a paraphyletic Parnassiinae. However, when examining the likelihoods
      of alternative rootings of this tree relative to rootings of the classical
      hypothesis, two rootings of the latter emerge as most likely. Of these two, the
      most likely rooting is within the Papilioninae, although a rooting between
      Baronia and the remaining Papilionidae is only nonsignificantly less likely.
FAU - Caterino, M S
AU  - Caterino MS
AD  - Division of Insect Biology, 201 Wellman Hall, University of California, Berkeley,
      CA 94720-3112, USA. histerid@nhm.ac.uk
FAU - Reed, R D
AU  - Reed RD
FAU - Kuo, M M
AU  - Kuo MM
FAU - Sperling, F A
AU  - Sperling FA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Peptide Elongation Factor 1)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Biometry
MH  - Butterflies/*classification/*genetics
MH  - DNA, Mitochondrial/genetics
MH  - Electron Transport Complex IV/genetics
MH  - Genes, Insect
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Peptide Elongation Factor 1/genetics
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Feb;50(1):106-27.

PMID- 12116587
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20020715
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 3
DP  - 2001 Jun
TI  - Incorporation, relative homoplasy, and effect of gap characters in sequence-based
      phylogenetic analyses.
PG  - 454-62
FAU - Simmons, M P
AU  - Simmons MP
AD  - L.H. Bailey Hortorium, 462 Mann Library, Cornell University, Ithaca, New York
      14853, USA. mps14@cornell.edu
FAU - Ochoterena, H
AU  - Ochoterena H
FAU - Carr, T G
AU  - Carr TG
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biometry
MH  - Databases, Genetic
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Sequence Alignment
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Jun;50(3):454-62.

PMID- 12116586
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20040220
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 3
DP  - 2001 Jun
TI  - Are the fossil data really at odds with the molecular data? Morphological
      evidence for cetartiodactyla phylogeny reexamined.
PG  - 444-53
FAU - Naylor, G J
AU  - Naylor GJ
AD  - Department of Zoology and Genetics, Iowa State University, Ames, Iowa 50010, USA.
      gnaylor@iastate.edu
FAU - Adams, D C
AU  - Adams DC
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2003 Dec;52(6):853-64; discussion 864-5. PMID: 14668123
EIN - Syst Biol 2001 Nov-Dec;50(6):1026
MH  - Animals
MH  - Artiodactyla/anatomy &amp; histology/*classification/*genetics
MH  - Cetacea/anatomy &amp; histology/*classification/*genetics
MH  - *Fossils
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Tooth/anatomy &amp; histology
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Jun;50(3):444-53.

PMID- 12116585
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20041117
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 3
DP  - 2001 Jun
TI  - Popper and likelihood versus "Popper".
PG  - 438-44
FAU - Farris, J S
AU  - Farris JS
AD  - Molekylarsytematiska laboratoriet, Naturhistoriska riksmuseet, Box 50007, SE-104 
      05 Stockholm, Sweden.
FAU - Kluge, A G
AU  - Kluge AG
FAU - Carpenter, J M
AU  - Carpenter JM
LA  - eng
PT  - Biography
PT  - Historical Article
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - History, 20th Century
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Probability
PS  - Popper K
FPS - Popper, K
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Jun;50(3):438-44.

PMID- 12116584
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 3
DP  - 2001 Jun
TI  - A conditional probability of reconstruction measure for internal cladogram
      branches.
PG  - 425-37
AB  - The conditional probability of reconstruction is a measure of the robustness of
      cladogram internodes and, unlike Bremer support and bootstrapping values,
      directly gauges probability. The new method compares the three putative branch
      lengths (the optimal and two alternatives) obtained through branch recalculation 
      after nearest neighbor interchange and recalculation under constraint. With
      rooted trees, one switches the two free lineages attached at the distal end of an
      internal branch with the basal lineage. Probabilistic reconstruction of a branch 
      for small data sets (e.g., morphological) is defined as having no contrary
      support for the two alternative branches and, when sufficient data are available 
      (e.g., molecular studies), as meeting a selected confidence limit in chi-squared 
      analysis. The exact probability that the internal branch is reconstructed is the 
      same as the preselected confidence level met with chi-squared analysis;
      alternatively, it is a simple calculation of the length of the optimal branch
      divided by the sum of the lengths of all three putative branches. This new
      measure of robustness allows calculation of summary probabilities of subclade and
      tree reconstruction. The measure is conditional on a particular data set and
      optimization method but may also compare support from conflicting gene trees.
      Examples are provided by a morphological data set (the bryophyte Didymodon) and a
      molecular data set (primates).
FAU - Zander, R H
AU  - Zander RH
AD  - Buffalo Museum of Science, 1020 Humboldt Parkway, Buffalo, New York 14211-1293,
      USA. rzander@sciencebuff.org
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biometry
MH  - Classification
MH  - Humans
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Primates/classification/genetics
MH  - Probability
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150119747 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Jun;50(3):425-37. doi: 10.1080/10635150119747.

PMID- 12116583
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20070727
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 3
DP  - 2001 Jun
TI  - Failure of the ILD to determine data combinability for slow loris phylogeny.
PG  - 408-24
AB  - Tests for incongruence as an indicator of among-data partition conflict have
      played an important role in conditional data combination. When such tests reveal 
      significant incongruence, this has been interpreted as a rationale for not
      combining data into a single phylogenetic analysis. In this study of lorisiform
      phylogeny, we use the incongruence length difference (ILD) test to assess
      conflict among three independent data sets. A large morphological data set and
      two unlinked molecular data sets--the mitochondrial cytochrome b gene and the
      nuclear interphotoreceptor retinoid binding protein (exon 1)--are analyzed with
      various optimality criteria and weighting mechanisms to determine the
      phylogenetic relationships among slow lorises (Primates, Loridae). When analyzed 
      separately, the morphological data show impressive statistical support for a
      monophyletic Loridae. Both molecular data sets resolve the Loridae as
      paraphyletic, though with different branching orders depending on the optimality 
      criterion or character weighting used. When the three data partitions are
      analyzed in various combinations, an inverse relationship between congruence and 
      phylogenetic accuracy is observed. Nearly all combined analyses that recover
      monophyly indicate strong data partition incongruence (P = 0.00005 in the most
      extreme case), whereas all analyses that recover paraphyly indicate lack of
      significant incongruence. Numerous lines of evidence verify that monophyly is the
      accurate phylogenetic result. Therefore, this study contributes to a growing body
      of information affirming that measures of incongruence should not be used as
      indicators of data set combinability.
FAU - Yoder, A D
AU  - Yoder AD
AD  - Department of Cell and Molecular Biology, Northwestern University Medical School,
      303 E. Chicago Avenue, Chicago, Illinois 60611, USA. ayoder@nwu.edu
FAU - Irwin, J A
AU  - Irwin JA
FAU - Payseur, B A
AU  - Payseur BA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Eye Proteins)
RN  - 0 (Retinol-Binding Proteins)
RN  - 0 (interstitial retinol-binding protein)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Biometry
MH  - Cytochrome b Group/genetics
MH  - DNA/genetics
MH  - DNA, Mitochondrial/genetics
MH  - *Eye Proteins
MH  - Lorisidae/*classification/*genetics
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Retinol-Binding Proteins/genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Jun;50(3):408-24.

PMID- 12116582
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 3
DP  - 2001 Jun
TI  - The importance of time/space in diagnosing the causality of phylogenetic events: 
      towards a "chronobiogeographical" paradigm?
PG  - 391-407
AB  - A shift from a traditional biogeographical paradigm in cladistic biogeography to 
      a chronobiogeographical paradigm is proposed. The chronobiogeographical paradigm 
      aims to utilize temporal data in conjunction with spatial data in the detection
      of discrete historical events, such as vicariance and vicariant speciation, in
      cladograms. The concepts of primary and secondary congruency are introduced in
      relation to the distinction between repeated area relationships (primary
      congruency) and common extrinsic causality (secondary congruency). Simple
      hypothetical examples demonstrate that area cladograms cannot be safely
      interpreted purely as representing the sequence of area fragmentation; rather,
      they reflect recency of biotic interaction. Temporal data are shown to have a
      direct and potentially profound influence on the results of traditional cladistic
      biogeographical analyses, indicating the necessity of developing a
      chronobiogeographical approach. The implementation of the paradigm is considered 
      first from a theoretical viewpoint and then in the context of the type of
      empirical data usually available. An as yet undevised "time/space algorithm" is
      deemed necessary for the latter, and guidelines are presented for the development
      of such an algorithm. Finally, we argue that the most rigorous and
      philosophically justified approach to the detection of phylogenetic causal events
      can be found only when temporal and spatial data are considered simultaneously.
      Consequently, the chronobiogeographical paradigm is seen as a logical elaboration
      of, not a replacement for, the biogeographical paradigm.
FAU - Hunn, C A
AU  - Hunn CA
AD  - Department of Earth Sciences, University of Cambridge, Cambridge, CB2 3EQ, UK.
      cah34@esc.cam.ac.uk
FAU - Upchurch, P
AU  - Upchurch P
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Jun;50(3):391-407.

PMID- 12116581
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 3
DP  - 2001 Jun
TI  - Mining the mammalian genome for artiodactyl systematics.
PG  - 367-90
AB  - A total of 7,806 nucleotide positions derived from one mitochondrial and eight
      nuclear DNA segments were used to provide a robust phylogeny for members of the
      order Artiodactyla. Twenty-four artiodactyl and two cetacean species were
      included, and the horse (order Perissodactyla) was used as the outgroup. Limited 
      rate heterogeneity was observed among the nuclear genes. The partition
      homogeneity tests indicated no conflicting signal among the nuclear genes
      fragments, so the sequence data were analyzed together and as separate loci.
      Analyses based on the individual nuclear DNA fragments and on 34 unique indels
      all produced phylogenies largely congruent with the topology from the combined
      data set. In sharp contrast to the nuclear DNA data, the mtDNA cytochrome b
      sequence data showed high levels of homoplasy, failed to produce a robust
      phylogeny, and were remarkably sensitive to taxon sampling. The nuclear DNA data 
      clearly support the paraphyletic nature of the Artiodactyla. Additionally, the
      family Suidae is diphyletic, and the nonruminating pigs and peccaries (Suiformes)
      were the most basal cetartiodactyl group. The morphologically derived Ruminantia 
      was always monophyletic; within this group, all taxa with paired bony structures 
      on their skulls clustered together. The nuclear DNA data suggest that the
      Antilocaprinae account for a unique evolutionary lineage, the Cervidae and
      Bovidae are sister taxa, and the Giraffidae are more primitive.
FAU - Matthee, C A
AU  - Matthee CA
AD  - Department of Animal Science, Texas A&amp;M University, College Station, Texas, USA. 
      cam@maties.sun.ac.za
FAU - Burzlaff, J D
AU  - Burzlaff JD
FAU - Taylor, J F
AU  - Taylor JF
FAU - Davis, S K
AU  - Davis SK
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 0 (DNA, Mitochondrial)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Artiodactyla/*classification/*genetics
MH  - Base Sequence
MH  - Cytochrome b Group/genetics
MH  - DNA/genetics
MH  - DNA, Mitochondrial/genetics
MH  - *Genome
MH  - Phylogeny
MH  - Sequence Alignment
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Jun;50(3):367-90.

PMID- 12116580
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 3
DP  - 2001 Jun
TI  - Empirical and hierarchical Bayesian estimation of ancestral states.
PG  - 351-66
AB  - Several methods have been proposed to infer the states at the ancestral nodes on 
      a phylogeny. These methods assume a specific tree and set of branch lengths when 
      estimating the ancestral character state. Inferences of the ancestral states,
      then, are conditioned on the tree and branch lengths being true. We develop a
      hierarchical Bayes method for inferring the ancestral states on a tree. The
      method integrates over uncertainty in the tree, branch lengths, and substitution 
      model parameters by using Markov chain Monte Carlo. We compare the hierarchical
      Bayes inferences of ancestral states with inferences of ancestral states made
      under the assumption that a specific tree is correct. We find that the methods
      are correlated, but that accommodating uncertainty in parameters of the
      phylogenetic model can make inferences of ancestral states even more uncertain
      than they would be in an empirical Bayes analysis.
FAU - Huelsenbeck, J P
AU  - Huelsenbeck JP
AD  - Department of Biology, University of Rochester, Rochester, New York 14627, USA.
      johnh@brahms.biology.rochester.edu
FAU - Bollback, J P
AU  - Bollback JP
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - DNA/genetics
MH  - Humans
MH  - Likelihood Functions
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Jun;50(3):351-66.

PMID- 12116579
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20020715
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 3
DP  - 2001 Jun
TI  - Towards an inclusive philosophy for phylogenetic inference.
PG  - 331-50
AB  - We defend and expand on our earlier proposal for an inclusive philosophical
      framework for phylogenetics, based on an interpretation of Popperian
      corroboration that is decoupled from the popular falsificationist interpretation 
      of Popperian philosophy. Any phylogenetic inference method can provide Popperian 
      "evidence" or "test statements" based on the method's goodness-of-fit values for 
      different tree hypotheses. Corroboration, or the severity of that test, requires 
      that the evidence is improbable without the hypothesis, given only background
      knowledge that includes elements of chance. This framework contrasts with
      attempted Popperian justifications for cladistic parsimony--in which evidence is 
      the data, background knowledge is restricted to descent with modification, and
      "corroboration," as a by-product of nonfalsification, is to be measured by
      cladistic parsimony. Recognition that cladistic "corroboration" reflects only
      goodness-of-fit, not corroboration/severity, makes it clear that standard
      cladistic prohibitions, such as restrictions on the evolutionary models to be
      included in "background knowledge," have no philosophical status. The capacity to
      assess Popperian corroboration neither justifies nor excludes any phylogenetic
      method, but it does provide a framework in phylogenetics for learning from
      errors--cases where apparent good evidence is probable even without the
      hypothesis. We explore these issues in the context of corroboration assessments
      applied to likelihood methods and to a new form of parsimony. These different
      forms of evidence and corroboration assessment point also to a new way to combine
      evidence--not at the level of overall fit, but at the level of overall
      corroboration/severity. We conclude that progress in an inclusive phylogenetics
      will be well served by the rejection of cladistic philosophy.
FAU - Faith, D P
AU  - Faith DP
AD  - Australian Museum, 6 College Street, Sydney 2000, Australia. danf@austmus.gov.au
FAU - Trueman, J W
AU  - Trueman JW
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Philosophy
MH  - *Phylogeny
MH  - Probability Theory
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
PST - ppublish
SO  - Syst Biol. 2001 Jun;50(3):331-50.

PMID- 12116578
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 3
DP  - 2001 Jun
TI  - Philosophical conjectures and their refutation.
PG  - 322-30
AB  - Sir Karl Popper is well known for explicating science in falsificationist terms, 
      for which his degree of corroboration formalism, C(h,e,b), has become little more
      than a symbol. For example, de Queiroz and Poe in this issue argue that C(h,e,b) 
      reduces to a single relative (conditional) probability, p(e,hb), the likelihood
      of evidence e, given both hypothesis h and background knowledge b, and in
      reaching that conclusion, without stating or expressing it, they render Popper a 
      verificationist. The contradiction they impose is easily explained--de Queiroz
      and Poe fail to take account of the fact that Popper derived C(h,e,b) from
      absolute (logical) probability and severity of test, S(e,h,b), where critical
      evidence, p(e,b), is fundamental. Thus, de Queiroz and Poe's conjecture that
      p(e,hb) = C(h,e,b) is refuted. Falsificationism, not verificationism, remains a
      fair description of the parsimony method of inference used in phylogenetic
      systematics, not withstanding de Queiroz and Poe's mistaken understanding that
      "statistical" probability justifies that method. Although de Queiroz and Poe
      assert that maximum likelihood has the power "to explain data", they do not
      successfully demonstrate how causal explanation is achieved or what it is that is
      being explained. This is not surprising, bearing in mind that what is assumed
      about character evolution in the accompanying likelihood model M cannot then be
      explained by the results of a maximum likelihood analysis.
FAU - Kluge, A G
AU  - Kluge AG
AD  - Museum of Zoology, University of Michigan, Ann Arbor, Michigan 48109-1079, USA.
      akluge@umich.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2003 Jun;52(3):352-67. PMID: 12775524
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Philosophy
MH  - *Phylogeny
MH  - Probability Theory
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150119615 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Jun;50(3):322-30. doi: 10.1080/10635150119615.

PMID- 12116577
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 3
DP  - 2001 Jun
TI  - Philosophy and phylogenetic inference: a comparison of likelihood and parsimony
      methods in the context of Karl Popper's writings on corroboration.
PG  - 305-21
AB  - Advocates of cladistic parsimony methods have invoked the philosophy of Karl
      Popper in an attempt to argue for the superiority of those methods over
      phylogenetic methods based on Ronald Fisher's statistical principle of
      likelihood. We argue that the concept of likelihood in general, and its
      application to problems of phylogenetic inference in particular, are highly
      compatible with Popper's philosophy. Examination of Popper's writings reveals
      that his concept of corroboration is, in fact, based on likelihood. Moreover,
      because probabilistic assumptions are necessary for calculating the probabilities
      that define Popper's corroboration, likelihood methods of phylogenetic
      inference--with their explicit probabilistic basis--are easily reconciled with
      his concept. In contrast, cladistic parsimony methods, at least as described by
      certain advocates of those methods, are less easily reconciled with Popper's
      concept of corroboration. If those methods are interpreted as lacking
      probabilistic assumptions, then they are incompatible with corroboration.
      Conversely, if parsimony methods are to be considered compatible with
      corroboration, then they must be interpreted as carrying implicit probabilistic
      assumptions. Thus, the non-probabilistic interpretation of cladistic parsimony
      favored by some advocates of those methods is contradicted by an attempt by the
      same authors to justify parsimony methods in terms of Popper's concept of
      corroboration. In addition to being compatible with Popperian corroboration, the 
      likelihood approach to phylogenetic inference permits researchers to test the
      assumptions of their analytical methods (models) in a way that is consistent with
      Popper's ideas about the provisional nature of background knowledge.
FAU - de Queiroz, K
AU  - de Queiroz K
AD  - Department of Systematic Biology, National Museum of Natural History, Smithsonian
      Institution, Washington, DC 20560-0162, USA. dequeiroz.kevin@nmnh.si.edu
FAU - Poe, S
AU  - Poe S
LA  - eng
PT  - Biography
PT  - Comparative Study
PT  - Historical Article
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - History, 20th Century
MH  - *Likelihood Functions
MH  - Models, Genetic
MH  - Models, Statistical
MH  - *Philosophy/history
MH  - *Phylogeny
PS  - Popper KR
FPS - Popper, K R
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501300317941 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Jun;50(3):305-21. doi: 10.1080/106351501300317941.

PMID- 12116576
OWN - NLM
STAT- MEDLINE
DCOM- 20020812
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 50
IP  - 3
DP  - 2001 Jun
TI  - The emergence of Systematic Biology.
PG  - 301-3
FAU - Hillis, D M
AU  - Hillis DM
AD  - Section of Integrative Biology, University of Texas, Austin, Texas 78712, USA.
LA  - eng
PT  - Historical Article
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biology/*history
MH  - History, 20th Century
MH  - Periodicals as Topic/history
MH  - Publishing
MH  - Societies, Scientific/history
MH  - United States
EDAT- 2002/07/16 10:00
MHDA- 2002/08/13 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/13 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351501300317923 [doi]
PST - ppublish
SO  - Syst Biol. 2001 Jun;50(3):301-3. doi: 10.1080/106351501300317923.

PMID- 12116485
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 1
DP  - 2000 Mar
TI  - Phylogenetic relationship among horseshoe crab species: effect of substitution
      models on phylogenetic analyses.
PG  - 87-100
AB  - The horseshoe crabs, known as living fossils, have maintained their morphology
      almost unchanged for the past 150 million years. The little morphological
      differentiation among horseshoe crab lineages has resulted in substantial
      controversy concerning the phylogenetic relationship among the extant species of 
      horseshoe crabs, especially among the three species in the Indo-Pacific region.
      Previous studies suggest that the three species constitute a phylogenetically
      unresolvable trichotomy, the result of a cladogenetic process leading to the
      formation of all three Indo-Pacific species in a short geological time. Data from
      two mitochondrial genes (for 16S ribosomal rRNA and cytochrome oxidase subunit I)
      and one nuclear gene (for coagulogen) in the four species of horseshoe crabs and 
      outgroup species were used in a phylogenetic analysis with various substitution
      models. All three genes yield the same tree topology, with Tachypleus-gigas and
      Carcinoscorpius-rotundicauda grouped together as a monophyletic taxon. This
      topology is significantly better than all the alternatives when evaluated with
      the RELL (resampling estimated log-likelihood) method.
FAU - Xia, X
AU  - Xia X
AD  - Molecular Laboratory, Department of Ecology &amp; Biodiversity, University of Hong
      Kong, Pokfulam Road, Hong Kong. xxia@hkusua.hku.hk
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (RNA, Ribosomal, 16S)
SB  - IM
MH  - Animals
MH  - Artemia/classification/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Horseshoe Crabs/*classification/genetics
MH  - Models, Biological
MH  - *Phylogeny
MH  - RNA, Ribosomal, 16S/genetics
RF  - 68
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150050207401 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Mar;49(1):87-100. doi: 10.1080/10635150050207401.

PMID- 12116484
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 1
DP  - 2000 Mar
TI  - The quality of the fossil record and the accuracy of phylogenetic inferences
      about sampling and diversity.
PG  - 65-86
AB  - Because phylogenies can be estimated without stratigraphic data and because
      estimated phylogenies also infer gaps in sampling, some workers have used
      phylogeny estimates as templates for evaluating sampling from the fossil record
      and for "correcting" historical diversity patterns. However, it is not known how 
      sampling intensity (the probability of sampling taxa per unit time) and
      completeness (the proportion of taxa sampled) affect the accuracy of phylogenetic
      inferences, nor how phylogenetically inferred estimates of sampling and diversity
      respond to inaccurate estimates of phylogeny. Both issues are addressed with a
      series of simulations using simple models of character evolution, varying
      speciation patterns, and various rates of speciation, extinction, character
      change, and preservation. Parsimony estimates of simulated phylogenies become
      less accurate as sampling decreases, and inaccurate trees chronically
      underestimate sampling. Biotic factors such as rates of morphologic change and
      extinction both affect the accuracy of phylogenetic estimates and thus affect
      estimated gaps in sampling, indicating that differences in implied sampling need 
      not reflect actual differences in sampling. Errors in inferred diversity are
      concentrated early in the history of a clade. This, coupled with failure to
      account for true extinction times (i.e., the Signor-Lipps effect), inflates
      relative diversity levels early in clade histories. Because factors other than
      differences in sampling predict differences in the numbers of gaps implied by
      phylogeny estimates, inferred phylogenies can be misleading templates for
      evaluating sampling or historical diversity patterns.
FAU - Wagner, P J
AU  - Wagner PJ
AD  - Department of Geology, Field Museum of Natural History, Roosevelt Road at Lake
      Shore Drive, Chicago, Illinois 60605-2496, USA. pwagner@fmppr.fmnh.org
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Fossils
MH  - Genetic Variation
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Selection Bias
RF  - 103
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150050207393 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Mar;49(1):65-86. doi: 10.1080/10635150050207393.

PMID- 12116483
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 1
DP  - 2000 Mar
TI  - Urochordates are monophyletic within the deuterostomes.
PG  - 52-64
AB  - Understanding the phylogenetic relationships of the three major urochordate
      groups within the deuterostomes is central to understanding the evolution of the 
      chordates. We have prepared a detailed phylogenetic analysis of urochordates
      based on comparisons of 10 new urochordate 18S ribosomal DNA sequences with other
      urochordate sequences in GenBank. Maximum parsimony, neighbor-joining, minimum
      evolution, and maximum likelihood analyses of this large urochordate data set are
      consistent with a topology in which the urochordates are monophyletic within the 
      deuterostomes and there are four separate clades of urochordates. These four
      distinct clades--styelid + pyurid ascidians, molgulid ascidians, phlebobranch
      ascidians + thaliaceans, and larvaceans--are mostly consistent with traditional
      morphological hypotheses and classifications. However, we find that the ascidians
      may not be a monophyletic group (as they have been considered traditionally) but 
      instead appear paraphyletic. Another disparity with traditional classification is
      that the thaliaceans do not form a separate urochordate clade but rather cluster 
      with the phlebobranch ascidians. Larvaceans have long branch lengths, which can
      be problematic for molecular phylogenetic methods, and their position within the 
      urochordates cannot be unequivocally determined with 18S rDNA. This is important 
      because the tadpole morphology of larvacean and ascidian larvae is the key trait 
      of interest that distinguishes urochordates as chordates. Nevertheless, the
      present data set resolves at least three clades of urochordates and suggests
      strongly that urochordates form a monophyletic clade within the deuterostomes.
FAU - Swalla, B J
AU  - Swalla BJ
AD  - Department of Biology, Vanderbilt University, Nashville, Tennessee 37235, USA.
      bjswalla@u.washington.edu
FAU - Cameron, C B
AU  - Cameron CB
FAU - Corley, L S
AU  - Corley LS
FAU - Garey, J R
AU  - Garey JR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - DNA/*genetics
MH  - Phylogeny
MH  - Urochordata/*classification/genetics
RF  - 53
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150050207384 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Mar;49(1):52-64. doi: 10.1080/10635150050207384.

PMID- 12116482
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 1
DP  - 2000 Mar
TI  - Developmental variation, homology, and the pharyngula stage.
PG  - 3-18
AB  - Understanding how development varies both inter- and intraspecifically can be
      important for systematic and evolutionary studies. This review will explore three
      different ways such understanding can be applied to evolutionary analyses. First,
      developmental data can be useful for homology determination. Interspecific
      variation in development has been thought to make developmental data poor
      candidates for determining homology. However, an updated developmental criterion 
      that is more broadly comparative and mechanistic augments the available criteria 
      used in homology determination. Second, modern cell and molecular biology are
      providing a better understanding of the many developmental processes involved in 
      a structure's formation and will augment the number of characters available for
      phylogenetic analyses. Recent work has revealed that what had been thought to be 
      a highly conserved developmental stage, the pharyngula (the phylotypic and
      zootypic stage of craniates) is highly variable. This variation can be seen in
      the development of such tissues as neural crest and placodes. These tissues are
      particularly interesting from a phylogenetic standpoint because they and the
      structures they form contribute to key synapomorphies of craniates. Finally,
      understanding developmental processes and how they form the variety of
      morphologies seen in nature will help in constructing the transformations that
      occurred during evolution. One such example involves descriptions of how lateral 
      line development is affected in different mutant lines of zebrafish. The many
      species of teleost fishes express great variation in the patterns of their
      lateral lines, and this is often an important systematic character. Understanding
      the genetic basis of lateral line development would help not only in
      hypothesizing possible transformational series but also in determining how many
      genes may have been required for these transformations.
FAU - Collazo, A
AU  - Collazo A
AD  - House Ear Institute, 2100 West Third St., Los Angeles, California 90057, USA.
      acollazo@hei.org
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, P.H.S.
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Embryo, Nonmammalian/*physiology
MH  - Embryonic and Fetal Development/*physiology
MH  - Mammals
MH  - *Morphogenesis
MH  - Nematoda/embryology
MH  - Phylogeny
MH  - Species Specificity
MH  - Zebrafish/embryology
RF  - 105
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150050207357 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Mar;49(1):3-18. doi: 10.1080/10635150050207357.

PMID- 12116481
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 1
DP  - 2000 Mar
TI  - Developmental regulatory genes and echinoderm evolution.
PG  - 28-51
AB  - Modified interactions among developmental regulatory genes and changes in their
      expression domains are likely to be an important part of the developmental basis 
      for evolutionary changes in morphology. Although developmental regulatory genes
      are now being studied in an increasing number of taxa, there has been little
      attempt to analyze the resulting data within an explicit phylogenetic context.
      Here we present comparative analyses of expression data from regulatory genes in 
      the phylum Echinodermata, considering the implications for understanding both
      echinoderm evolution as well as the evolution of regulatory genes in general.
      Reconstructing the independent evolutionary histories of regulatory genes, their 
      expression domains, their developmental roles, and the structures in which they
      are expressed reveals a number of distinct evolutionary patterns. A few of these 
      patterns correspond to interpretations common in the literature, whereas others
      have received little prior mention. Together, the analyses indicate that the
      evolution of echinoderms involved: (1) the appearance of many apomorphic
      developmental roles and expression domains, some of which have plesiomorphic
      bilateral symmetry and others of which have apomorphic radial symmetry or
      left-right asymmetry; (2) the loss of some developmental roles and expression
      domains thought to be plesiomorphic for Bilateria; and (3) the retention of some 
      developmental roles thought to be plesiomorphic for Bilateria, although with
      modification in expression domains. Some of the modifications within the
      Echinodermata concern adult structures; others, transient larval structures. Some
      changes apparently appeared early in echinoderm evolution (&gt; 450 Ma), whereas
      others probably happened more recently (&lt; 50 Ma). Cases of likely convergence in 
      expression domains suggest caution when using developmental regulatory genes to
      make inferences about homology among morphological structures of distantly
      related taxa.
FAU - Wray, G A
AU  - Wray GA
AD  - Department of Ecology and Evolution, State University of New York, Stony Brook,
      New York 11794-5245, USA. gwray@duke.edu
FAU - Lowe, C J
AU  - Lowe CJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
SB  - S
MH  - Animals
MH  - *Biological Evolution
MH  - Echinodermata/classification/*genetics
MH  - *Genes, Regulator
MH  - *Phylogeny
RF  - 122
OTO - NASA
OT  - Non-programmatic
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150050207375 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Mar;49(1):28-51. doi: 10.1080/10635150050207375.

PMID- 12116480
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 1
DP  - 2000 Mar
TI  - Transformationalism, taxism, and developmental biology in systematics.
PG  - 19-27
AB  - Issues concerning transformational and taxic comparisons are central to
      understanding the impact of the recent proliferation of molecular developmental
      data on evolutionary biology. More importantly, an understanding of taxism and
      transformationalism in comparative biology is critical to assessing the impact of
      the recent developmental data on systematic theory and practice. We examine the
      philosophical and practical aspects of the transformational approach and the
      relevance of this approach to recent molecular-based developmental data. We also 
      examine the theoretical basis of the taxic approach to molecular developmental
      data and suggest that developmental data are perfectly amenable to the taxic
      approach. Two recent examples from the molecular developmental biology
      literature--the evolution of insect wings and the evolution of dorsal ventral
      inversion in vertebrates and invertebrates--are used to compare the taxic and
      transformational approaches. We conclude that the transformational approach is
      entirely appropriate for ontogenetic studies and furthermore can serve as an
      excellent source of hypotheses about the evolution of characters. However, the
      taxic approach is the ultimate arbiter of these hypotheses.
FAU - Bang, R
AU  - Bang R
AD  - Department of Entomology, American Museum of Natural History, 79th Street at
      Central Park West, New York, New York 10024, USA.
FAU - DeSalle, R
AU  - DeSalle R
FAU - Wheeler, W
AU  - Wheeler W
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Review
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/methods
MH  - Developmental Biology/*methods
MH  - Models, Biological
MH  - Morphogenesis
RF  - 77
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150050207366 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Mar;49(1):19-27. doi: 10.1080/10635150050207366.

PMID- 12116479
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 1
DP  - 2000 Mar
TI  - A simulation study of reduced tree-search effort in bootstrap resampling
      analysis.
PG  - 171-9
FAU - DeBry, R W
AU  - DeBry RW
AD  - Department of Biological Sciences, Box 210006, University of Cincinnati,
      Cincinnati, Ohio 45221-0006, USA. ron.debry@uc.edu
FAU - Olmstead, R G
AU  - Olmstead RG
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Classification/methods
MH  - *Computer Simulation
MH  - *Decision Trees
MH  - *Phylogeny
MH  - Plants/classification/genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150050207465 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Mar;49(1):171-9. doi: 10.1080/10635150050207465.

PMID- 12116478
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 1
DP  - 2000 Mar
TI  - Comparison of three methods for estimating internal support on phylogenetic
      trees.
PG  - 160-71
FAU - Mort, M E
AU  - Mort ME
AD  - School of Biological Sciences, Washington State University, Pullman, Washington
      99164-4236, USA.
FAU - Soltis, P S
AU  - Soltis PS
FAU - Soltis, D E
AU  - Soltis DE
FAU - Mabry, M L
AU  - Mabry ML
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
RN  - EC 4.1.1.39 (RbcL protein, plastid)
RN  - EC 4.1.1.39 (Ribulose-Bisphosphate Carboxylase)
SB  - IM
MH  - DNA, Plant/*genetics
MH  - Magnoliopsida/*classification/*genetics
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Plastids/genetics
MH  - Ribulose-Bisphosphate Carboxylase/genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150050207456 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Mar;49(1):160-71. doi: 10.1080/10635150050207456.

PMID- 12116477
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 1
DP  - 2000 Mar
TI  - War of the Iguanas: conflicting molecular and morphological phylogenies and
      long-branch attraction in iguanid lizards.
PG  - 143-59
AB  - Recent studies based on different types of data (i.e., morphology, molecules)
      have found strongly conflicting phylogenies for the genera of iguanid lizards but
      have been unable to explain the basis for this incongruence. We reanalyze
      published data from morphology and from the mitochondrial ND4, cytochrome b, 12S,
      and 16S genes to explore the sources of incongruence and resolve these conflicts.
      Much of the incongruence centers on the genus Cyclura, which is the sister taxon 
      of Iguana, according to parsimony analyses of the morphology and the ribosomal
      genes, but is the sister taxon of all other Iguanini, according to the
      protein-coding genes. Maximum likelihood analyses show that there has been an
      increase in the rate of nucleotide substitution in Cyclura in the two
      protein-coding genes (ND4 and cytochrome b), although this increase is not as
      clear when parsimony is used to estimate branch lengths. Parametric simulations
      suggest that Cyclura may be misplaced by the protein-coding genes as a result of 
      long-branch attraction; even when Cyclura and Iguana are sister taxa in a
      simulated phylogeny, Cyclura is still placed as the basal member of the Iguanini 
      by parsimony analysis in 55% of the replicates. A similar long-branch attraction 
      problem may also exist in the morphological data with regard to the placement of 
      Sauromalus with the Galapagos iguanas (Amblyrhynchus and Conolophus). The results
      have many implications for the analysis of diverse data sets, the impact of long 
      branches on parsimony and likelihood methods, and the use of certain
      protein-coding genes in phylogeny reconstruction.
FAU - Wiens, J J
AU  - Wiens JJ
AD  - Section of Amphibians and Reptiles, Carnegie Museum of Natural History,
      Pittsburgh, Pennsylvania 15213-4080, USA. wiensj@clpgh.org
FAU - Hollingsworth, B D
AU  - Hollingsworth BD
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA/*genetics
MH  - Iguanas/*classification/*genetics
MH  - Lizards/*classification/*genetics
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150050207447 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Mar;49(1):143-59. doi: 10.1080/10635150050207447.

PMID- 12116476
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 1
DP  - 2000 Mar
TI  - Early evolution of the bilateria.
PG  - 130-42
AB  - The phylogeny of the Bilateria and especially the early steps in the evolution of
      the bilaterian bauplan are still a controversial topic. In this context the
      relationships of the platyhelminths and the nematodes play a crucial role.
      Previous molecular studies of the relationships of these groups, which were based
      on 18S ribosomal DNA sequences, yielded conflicting results. In the present study
      a new framework is developed for the phylogenetic analysis of bilaterian
      relationships, using concatenated amino acid sequences of several nuclear genes. 
      In this analysis, the rhabditophoran platyhelminths are probably the sister group
      of all other analyzed Bilateria, the Eubilateria, which are characterized by a
      one-way intestine with an anus. The Eubilateria are split into the nematode
      lineage and the coelomates. The phylogenetic results of the present study
      indicate that genetic features found in the model organisms Caenorhabditis and
      Drosophila might be found in all Eubilateria. Estimations of the divergence times
      show that the major bilaterian phyla did not originate in an explosive radiation 
      during the Cambrian but rather that the Bilateria have a several hundred million 
      years long Precambrian history.
FAU - Hausdorf, B
AU  - Hausdorf B
AD  - Zoologisches Institut und Zoologisches Museum, Universitat Hamburg,
      Martin-Luther-King-Platz 3, D-20146 Hamburg, Germany.
      hausdorf@zoologie.uni-hamburg.de
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Arabidopsis/classification/genetics
MH  - *Biological Evolution
MH  - Caenorhabditis/classification/genetics
MH  - Functional Laterality
MH  - Hominidae/classification/genetics
MH  - Humans
MH  - Muridae/classification/genetics
MH  - *Phylogeny
MH  - Schistosoma/classification/genetics
MH  - Time
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150050207438 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Mar;49(1):130-42. doi: 10.1080/10635150050207438.

PMID- 12116475
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 1
DP  - 2000 Mar
TI  - Mitochondrial phylogeny of notothenioids: a molecular approach to Antarctic fish 
      evolution and biogeography.
PG  - 114-29
AB  - Antarctic waters represent a unique marine environment delimited by an
      oceanographic barrier, the Polar Front Zone, and characterized by constant
      subzero temperatures and presence of sea ice. A group of teleost fish, the
      Notothenioidei, have adapted to these challenging environmental conditions,
      undergoing a remarkable diversification. In the present study a total of 798 base
      pairs, generated from partial sequencing of 16S and 12S mitochondrial ribosomal
      RNA genes, were examined in 33 notothenioid species representative of all
      families included in the suborder Notothenioidei. Phylogenetic trees,
      reconstructed on the basis of sequence data by different methods, indicate that
      traditional hypotheses on notothenioid systematics and biogeography might be in
      need of reexamination. Molecular evidence suggests that vicariant speciation
      could be invoked to explain the early divergence of Eleginops maclovinus, a
      species previously included in the family Nototheniidae, which is now proposed as
      the closest sister group to all the rest of notothenioids apart from bovichtids. 
      On the other hand, repeated, independent dispersal through the Polar Front is
      proposed for the divergence of other subantarctic notothenioid species. Likewise,
      multiple, independent transitions from benthic to pelagic habit are inferred from
      molecular data, at variance with the more conservative hypothesis based on
      cladograms reconstructed from morphological data.
FAU - Bargelloni, L
AU  - Bargelloni L
AD  - Dipartimento di Biologia, Universita di Padova, Via Ugo Bassi 58/B, I-35131
      Padova, Italy. bargel@bio.unipd.it
FAU - Marcato, S
AU  - Marcato S
FAU - Zane, L
AU  - Zane L
FAU - Patarnello, T
AU  - Patarnello T
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (RNA, Ribosomal)
RN  - 0 (RNA, ribosomal, 12S)
SB  - IM
MH  - Animals
MH  - Antarctic Regions
MH  - DNA, Mitochondrial/*genetics
MH  - Fishes/classification/*genetics
MH  - Phylogeny
MH  - RNA, Ribosomal/genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150050207429 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Mar;49(1):114-29. doi: 10.1080/10635150050207429.

PMID- 12116474
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 1
DP  - 2000 Mar
TI  - Is there a justification for differential a priori weighting in coding sequences?
      A case study from rbcL and Apocynaceae s.l.
PG  - 101-13
AB  - Functional constraints are often assumed to influence the performance of
      nucleotide characters in phylogenetic analysis: First and second codon positions 
      and sites of structural importance are considered to show less homoplasy. We
      investigate the performance of rbcL characters with differential functional
      constraints in a cladistic analysis of the plant family Apocynaceae s.l.
      (Sennblad and Bremer, in prep.). Performance is measured as rescaled consistency 
      indices (rc). We show there is no significant difference in performance between
      parsimony-informative sites constrained by function in the enzyme, and sites that
      are not. Furthermore, the substitutions in third-codon position performed
      significantly better than those in first and second. The variation of rc within
      the different classes was high, however. Consequently, there is no support for
      routinely applied a priori differential weighting, neither of codon positions,
      nor of different functional classes from the present analysis of rbcL data in the
      Apocynaceae s.l.
FAU - Sennblad, B
AU  - Sennblad B
AD  - Department of Systematic Botany, Evolutionary Biology Centre (EBC), Uppsala
      University, Norbyv. 18D, S-752 36 Uppsala, Sweden. Bengt.Sennblad@systbot.uu.se
FAU - Bremer, B
AU  - Bremer B
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 4.1.1.39 (RbcL protein, plastid)
RN  - EC 4.1.1.39 (Ribulose-Bisphosphate Carboxylase)
SB  - IM
MH  - Asteraceae/*classification/*genetics
MH  - *Phylogeny
MH  - Plastids/genetics
MH  - Ribulose-Bisphosphate Carboxylase/*genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635150050207410 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Mar;49(1):101-13. doi: 10.1080/10635150050207410.

PMID- 12116444
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - Tree robustness and clade significance.
PG  - 829-36
FAU - Lee, M S
AU  - Lee MS
AD  - Department of Zoology, University of Queensland, Brisbane, QLD 4072, Australia.
      mlee@zoology.uq.edu.au
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Decision Trees
MH  - Evolution, Molecular
MH  - Lizards/classification/genetics
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049879 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):829-36. doi: 10.1080/106351500750049879.

PMID- 12116443
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - Parametric phylogenetics?
PG  - 817-29
FAU - Sanderson, M J
AU  - Sanderson MJ
AD  - Section of Evolution and Ecology, University of California, Davis, California
      95616, USA. mjsanderson@ucdavis.edu
FAU - Kim, J
AU  - Kim J
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/methods
MH  - Computer Simulation
MH  - *Models, Statistical
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049860 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):817-29. doi: 10.1080/106351500750049860.

PMID- 12116442
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - SINE evolution, missing data, and the origin of whales.
PG  - 808-17
FAU - Shedlock, A M
AU  - Shedlock AM
AD  - Tokyo Institute of Technology, Molecular Evolution Laboratory, Graduate School of
      Bioscience and Biotechnology, 4259 Nagatsuta-cho, Midori-ku, Yokohama 226-8501,
      Japan.
FAU - Milinkovitch, M C
AU  - Milinkovitch MC
FAU - Okada, N
AU  - Okada N
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Models, Genetic
MH  - Origin of Life
MH  - Reproducibility of Results
MH  - Whales/*classification/*genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049851 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):808-17. doi: 10.1080/106351500750049851.

PMID- 12116441
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - Linked branch support and tree stability.
PG  - 800-7
FAU - Gatesy, J
AU  - Gatesy J
AD  - Laboratory of Molecular Systematics and Evolution, Department of Ecology and
      Evolutionary Biology, University of Arizona, Biosciences West, Tucson, Arizona
      85721, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Decision Trees
MH  - Molecular Biology/methods
MH  - *Phylogeny
MH  - Ruminants/classification/*genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049842 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):800-7. doi: 10.1080/106351500750049842.

PMID- 12116440
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - Why morphometrics is special: the problem with using partial warps as characters 
      for phylogenetic inference.
PG  - 796-800
FAU - Monteiro, L R
AU  - Monteiro LR
AD  - Laboratorio de Ciencias Ambientais-CBB-UENF, Av. Alberto Lamego 2000, Horto, cep:
      28015-620, Campos dos Goytacazes, RJ, Brasil. lrmont@cbb.uenf.br
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Anatomy/methods
MH  - Animals
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049833 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):796-800. doi: 10.1080/106351500750049833.

PMID- 12116439
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - Molecular phylogenetics and biogeography of galaxiid fishes (Osteichthyes:
      Galaxiidae): dispersal, vicariance, and the position of Lepidogalaxias
      salamandroides.
PG  - 777-95
AB  - The galaxiid fishes exhibit a gondwanan distribution. We use mitochondrial DNA
      sequences to test conflicting vicariant and dispersal biogeographic hypotheses
      regarding the Southern Hemisphere range of this freshwater group. Although
      phylogenetic resolution of cytochrome b and 16S rRNA sequences is largely limited
      to more recent divergences, our data indicate that the radiation can be
      interpreted as several relatively recent dispersal events superimposed on an
      ancient gondwanan radiation. Genetic relationships contradict the findings of
      recent morphological analyses of galaxioid fishes. In particular, we examine
      several hypotheses regarding phylogenetic placement of the enigmatic
      Lepidogalaxias. Although most workers consider Lepidogalaxias to be an unusual
      scaled member of the Southern Hemisphere galaxioids, it has also been suggested
      that this species is related to the Northern Hemisphere esocoids. Our data
      strongly suggest that this species is not a galaxiid, and the alternative
      hypothesized esocoid relationship cannot be rejected. The species-rich genus
      Galaxias is shown to be polyphyletic and the generic taxonomy of the Galaxiinae
      is reassessed in the light of phylogenetic relationships. Juvenile
      saltwater-tolerance is phylogenetically distributed throughout the Galaxiinae,
      and the loss of this migratory phase may be a major cause of speciation.
FAU - Waters, J M
AU  - Waters JM
AD  - Department of Zoology, University of Otago, Dunedin, New Zealand.
      jonathan.waters@toroa.otago.ac.nz
FAU - Lopez, J A
AU  - Lopez JA
FAU - Wallis, G P
AU  - Wallis GP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 63231-63-0 (RNA)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Classification/methods
MH  - DNA/genetics
MH  - DNA, Mitochondrial/chemistry/genetics
MH  - Fishes/*classification/*genetics
MH  - Geography
MH  - Molecular Sequence Data
MH  - Nucleic Acid Conformation
MH  - *Phylogeny
MH  - RNA/chemistry/genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049824 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):777-95. doi: 10.1080/106351500750049824.

PMID- 12116438
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - A chain is no stronger than its weakest link: double decay analysis of
      phylogenetic hypotheses.
PG  - 754-76
AB  - In decay analyses the support for a particular split in most-parsimonious trees
      is its decay index, that is, the extra steps required of the shortest trees that 
      do not include the split. By focusing solely on the support for splits,
      traditional decay analysis may provide an incomplete and potentially misleading
      summary of the support for phylogenetic relationships common to the
      most-parsimonious tree or trees. Here, we introduce double decay analysis, a new 
      approach to assessing support for phylogenetic relationships. Double decay
      analysis is the determination of the decay indices of all n-taxon
      statements/partitions common to the most-parsimonious tree. The results of double
      decay analyses are presented in a partition table, but various approaches to
      graphical representation of the results, including the use of reduced consensus
      support trees, are also discussed. Double decay analysis provides a more
      comprehensive summary and facilitates a better understanding of the strengths and
      weaknesses of complex phylogenetic hypotheses than does traditional decay
      analysis. The limitations of traditional decay analyses and the utility of double
      decay analyses are illustrated with both contrived data and real data for
      sauropod dinosaurs.
FAU - Wilkinson, M
AU  - Wilkinson M
AD  - Department of Zoology, Natural History Museum, London SW7 5BD, UK. marw@nhm.ac.uk
FAU - Thorley, J L
AU  - Thorley JL
FAU - Upchurch, P
AU  - Upchurch P
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Fossils
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Vertebrates/classification
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049815 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):754-76. doi: 10.1080/106351500750049815.

PMID- 12116437
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - Use of two-block partial least-squares to study covariation in shape.
PG  - 740-53
AB  - The relatively new two-block partial least-squares method for analyzing the
      covariance between two sets of variables is described and contrasted with the
      well-known method of canonical correlation analysis. Their statistical
      properties, type of answers, and visualization techniques are discussed. Examples
      are given to show its usefulness in comparing two sets of variables--especially
      when one or both of the sets of variables are shape variables from a geometric
      morphometric study.
FAU - Rohlf, F J
AU  - Rohlf FJ
AD  - Department of Ecology and Evolution, State University of New York, Stony Brook,
      New York 11794-5245, USA. rohlf@life.bio.sunysb.edu
FAU - Corti, M
AU  - Corti M
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Analysis of Variance
MH  - Animals
MH  - Bone and Bones/*anatomy &amp; histology
MH  - Fibula/anatomy &amp; histology
MH  - Humerus/anatomy &amp; histology
MH  - Least-Squares Analysis
MH  - Mice/classification/genetics
MH  - Skull/*anatomy &amp; histology
MH  - Tibia/anatomy &amp; histology
MH  - Ulna/anatomy &amp; histology
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049806 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):740-53. doi: 10.1080/106351500750049806.

PMID- 12116436
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - Multiple data sets, congruence, and hypothesis testing for the phylogeny of basal
      groups of the lizard genus Sceloporus (Squamata, Phrynosomatidae).
PG  - 713-39
AB  - Several data partitions, including nuclear and mitochondrial gene sequences,
      chromosomes, isoenzymes, and morphological characters, were used to propose a new
      phylogeny and to test previously published hypotheses about the phylogenetic
      positions of basal clades of the lizard genus Sceloporus and the relationship of 
      Sceloporus to the former genus "Sator". In accord with earlier studies, our
      results grouped "Sator" as internal to Sceloporus, and both support a hypothesis 
      of transgulfian vicariance for the origin of the former genus "Sator" on islands 
      in the Sea of Cortez. Robustness of support for internal nodes in our best tree
      was established though widely used indices (bootstrap proportions, decay values) 
      but also through congruence among independent data partitions. Several deep nodes
      in the tree recovered by several methods, including equally weighted and
      differentially weighted parsimony and maximum likelihood models, are only weakly 
      supported by the traditional indices. This methodological concordance is taken as
      evidence for insensitivity of the deep structure of the topology to alternative
      assumptions.
FAU - Flores-Villa, O
AU  - Flores-Villa O
AD  - Department of Zoology and M. L. Bean Museum, Brigham Young University, Provo,
      Utah 84602, USA. ofv@hp.fciencias.unam.mx
FAU - Kjer, K M
AU  - Kjer KM
FAU - Benabib, M
AU  - Benabib M
FAU - Sites, J W Jr
AU  - Sites JW Jr
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Enzymes)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Enzymes/genetics
MH  - Lizards/*classification/*genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049798 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):713-39. doi: 10.1080/106351500750049798.

PMID- 12116435
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - A phylogenetic perspective on habitat shifts and diversity in the North American 
      Enallagma damselflies.
PG  - 697-712
AB  - Community ecologists are increasingly aware that the regional history of taxon
      diversification can have an important influence on community structure. Likewise,
      systematists recognize that ecological context can have an important influence on
      the processes of speciation and extinction that create patterns of descent. We
      present a phylogenetic analysis of 33 species of a North American radiation of
      damselflies (Zygoptera: Coenagrionidae: Enallagma Selys), which have been well
      studied ecologically, to elucidate the evolutionary mechanisms that have
      contributed to differences in diversity between larval habitats (lakes with and
      without fish predators). Analysis of molecular variation in 842 bp of the
      mitochondrial cytochrome oxidase I and II subunit and of the intervening Leu-tRNA
      and 37 morphological characters resulted in three well-defined clades that are
      only partially congruent with previous phylogenetic hypotheses. Molecular and
      morphological data partitions were significantly incongruent (p &lt; .01). Lack of
      haplotype monophyly within species and small amounts of sequence divergence (&lt;
      1%) between related species in three of the four clades suggest that recent, and 
      parallel, speciation has been an important source of community diversity.
      Reconstruction of habitat preference over the phylogeny suggests that the greater
      species diversity in fish-containing lake habitats reflects the recency of shifts
      into the fishless lake habit, although a difference in speciation or extinction
      rates between the two habitats is difficult to exclude as an additional
      mechanism.
FAU - Brown, J M
AU  - Brown JM
AD  - Biology Department, Grinnell College, P.O. Box 805, Grinnell, Iowa 50112, USA.
      brownj@grinnell.edu
FAU - McPeek, M A
AU  - McPeek MA
FAU - May, M L
AU  - May ML
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA Primers
MH  - DNA, Mitochondrial/*genetics
MH  - Environment
MH  - *Genetic Variation
MH  - Haplotypes
MH  - Insecta/classification/*genetics
MH  - Male
MH  - North America
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049789 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):697-712. doi: 10.1080/106351500750049789.

PMID- 12116434
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - Ability of geometric morphometric methods to estimate a known covariance matrix.
PG  - 686-96
AB  - Landmark-based morphometric methods must estimate the amounts of translation,
      rotation, and scaling (or, nuisance) parameters to remove nonshape variation from
      a set of digitized figures. Errors in estimates of these nuisance variables will 
      be reflected in the covariance structure of the coordinates, such as the
      residuals from a superimposition, or any linear combination of the coordinates,
      such as the partial warp and standard uniform scores. A simulation experiment was
      used to compare the ability of the generalized resistant fit (GRF) and a relative
      warp analysis (RWA) to estimate known covariance matrices with various
      correlations and variance structures. Random covariance matrices were perturbed
      so as to vary the magnitude of the average correlation among coordinates, the
      number of landmarks with excessive variance, and the magnitude of the excessive
      variance. The covariance structure was applied to random figures with between 6
      and 20 landmarks. The results show the expected performance of GRF and RWA across
      a broad spectrum of conditions. The performance of both GRF and RWA depended most
      strongly on the number of landmarks. RWA performance decreased slightly when one 
      or a few landmarks had excessive variance. GRF performance peaked when
      approximately 25% of the landmarks had excessive variance. In general, both RWA
      and GRF performed better at estimating the direction of the first principal axis 
      of the covariance matrix than the structure of the entire covariance matrix. RWA 
      tended to outperform GRF when &gt; approximately 75% of the coordinates had
      excessive variance. When &lt; 75% of the coordinates had excessive variance, the
      relative performance of RWA and GRF depended on the magnitude of the excessive
      variance; when the landmarks with excessive variance had standard deviations
      (sigma) &gt; or = 4 sigma minimum, GRF regularly outperformed RWA.
FAU - Walker, J A
AU  - Walker JA
AD  - Department of Zoology, Field Museum of Natural History, Roosevelt Road at Lake
      Shore Drive, Chicago, Illinois 60605, USA. walker@usm.maine.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Analysis of Variance
MH  - Computer Simulation
MH  - *Models, Biological
MH  - *Models, Statistical
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049770 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):686-96. doi: 10.1080/106351500750049770.

PMID- 12116433
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - Improved bootstrap confidence limits in large-scale phylogenies, with an example 
      from Neo-Astragalus (Leguminosae).
PG  - 671-85
AB  - Phylogenetic analyses of large data sets pose special challenges, including the
      apparent tendency for the bootstrap support for a clade to decline with increased
      taxon sampling of that clade. We document this decline in data sets with
      increasing numbers of taxa in Astragalus, the most species-rich angiosperm genus.
      Support for one subclade, Neo-Astragalus, declined monotonically with increased
      sampling of taxa inside Neo-Astragalus, irrespective of whether parsimony or
      neighbor-joining methods were used or of which particular heuristic search
      algorithm was used (although more stringent algorithms tended to yield higher
      support). Three possible explanations for this decline were examined, including
      (1) mistaken assignment of the most recent common ancestor of the taxon sample
      (and its bootstrap support) with the most recent common ancestor of the clade
      from which it was sampled; (2) computational limitations of heuristic search
      strategies; and (3) statistical bias in bootstrap proportions, especially that
      from random homoplasy distributed among taxa. The best explanation appears to be 
      (3), although computational shortcomings (2) may explain some of the problem. The
      bootstrap proportion, as currently used in phylogenetic analysis, does not
      accurately capture the classical notion of confidence assessments on the null
      hypothesis of nonmonophyly, especially in large data sets. More accurate
      assessments of confidence as type I error levels (relying on iterated bootstrap
      methods) remove most of the monotonic decline in confidence with increasing
      numbers of taxa.
FAU - Sanderson, M J
AU  - Sanderson MJ
AD  - Section of Evolution and Ecology, University of California, Davis, California
      95616, USA. mjsanderson@ucdavis.edu
FAU - Wojciechowski, M F
AU  - Wojciechowski MF
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/methods
MH  - Confidence Intervals
MH  - Fabaceae/classification/*genetics
MH  - Models, Statistical
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049761 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):671-85. doi: 10.1080/106351500750049761.

PMID- 12116432
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - Likelihood-based tests of topologies in phylogenetics.
PG  - 652-70
AB  - Likelihood-based statistical tests of competing evolutionary hypotheses (tree
      topologies) have been available for approximately a decade. By far the most
      commonly used is the Kishino-Hasegawa test. However, the assumptions that have to
      be made to ensure the validity of the Kishino-Hasegawa test place important
      restrictions on its applicability. In particular, it is only valid when the
      topologies being compared are specified a priori. Unfortunately, this means that 
      the Kishino-Hasegawa test may be severely biased in many cases in which it is now
      commonly used: for example, in any case in which one of the competing topologies 
      has been selected for testing because it is the maximum likelihood topology for
      the data set at hand. We review the theory of the Kishino-Hasegawa test and
      contend that for the majority of popular applications this test should not be
      used. Previously published results from invalid applications of the
      Kishino-Hasegawa test should be treated extremely cautiously, and future
      applications should use appropriate alternative tests instead. We review such
      alternative tests, both nonparametric and parametric, and give two examples which
      illustrate the importance of our contentions.
FAU - Goldman, N
AU  - Goldman N
AD  - University Museum of Zoology, Department of Zoology, University of Cambridge,
      Cambridge CB2 3EJ, UK. N.Goldman@zoo.cam.ac.uk
FAU - Anderson, J P
AU  - Anderson JP
FAU - Rodrigo, A G
AU  - Rodrigo AG
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Likelihood Functions
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049752 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):652-70. doi: 10.1080/106351500750049752.

PMID- 12116431
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - Integrating ambiguously aligned regions of DNA sequences in phylogenetic analyses
      without violating positional homology.
PG  - 628-51
AB  - Phylogenetic analyses of non-protein-coding nucleotide sequences such as
      ribosomal RNA genes, internal transcribed spacers, and introns are often impeded 
      by regions of the alignments that are ambiguously aligned. These regions are
      characterized by the presence of gaps and their uncertain positions, no matter
      which optimization criteria are used. This problem is particularly acute in
      large-scale phylogenetic studies and when aligning highly diverged sequences.
      Accommodating these regions, where positional homology is likely to be violated, 
      in phylogenetic analyses has been dealt with very differently by molecular
      systematists and evolutionists, ranging from the total exclusion of these regions
      to the inclusion of every position regardless of ambiguity in the alignment. We
      present a new method that allows the inclusion of ambiguously aligned regions
      without violating homology. In this three-step procedure, first homologous
      regions of the alignment containing ambiguously aligned sequences are delimited. 
      Second, each ambiguously aligned region is unequivocally coded as a new
      character, replacing its respective ambiguous region. Third, each of the coded
      characters is subjected to a specific step matrix to account for the differential
      number of changes (summing substitutions and indels) needed to transform one
      sequence to another. The optimal number of steps included in the step matrix is
      the one derived from the pairwise alignment with the greatest similarity and the 
      least number of steps. In addition to potentially enhancing phylogenetic
      resolution and support, by integrating previously nonaccessible characters
      without violating positional homology, this new approach can improve branch
      length estimations when using parsimony.
FAU - Lutzoni, F
AU  - Lutzoni F
AD  - Department of Botany, Field Museum of Natural History, 1400 S. Lake Shore Drive, 
      Chicago, Illinois 60605-2496, USA. flutzoni@fmnh.org
FAU - Wagner, P
AU  - Wagner P
FAU - Reeb, V
AU  - Reeb V
FAU - Zoller, S
AU  - Zoller S
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Alligators and Crocodiles/classification/*genetics
MH  - Animals
MH  - Base Sequence
MH  - DNA/chemistry/*genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049743 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):628-51. doi: 10.1080/106351500750049743.

PMID- 12116430
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 4
DP  - 2000 Dec
TI  - Quantification of homoplasy for nucleotide transitions and transversions and a
      reexamination of assumptions in weighted phylogenetic analysis.
PG  - 617-27
AB  - Nucleotide transitions are frequently down-weighted relative to transversions in 
      phylogenetic analysis. This is based on the assumption that transitions, by
      virtue of their greater evolutionary rate, exhibit relatively more homoplasy and 
      are therefore less reliable phylogenetic characters. Relative amounts of
      homoplastic and consistent transition and transversion changes in mitochondrial
      protein coding genes were determined from character-state reconstructions on a
      highly corroborated phylogeny of mammals. We found that although homoplasy was
      related to evolutionary rates and was greater for transitions, the absolute
      number of consistent transitions greatly exceeded the number of consistent
      transversions. Consequently, transitions provided substantially more useful
      phylogenetic information than transversions. These results suggest that
      down-weighting transitions may be unwarranted in many cases. This conclusion was 
      supported by the fact that a range of transition: transversion weighting schemes 
      applied to various mitochondrial genes and genomic partitions rarely provided
      improvement in phylogenetic estimates relative to equal weighting, and in some
      cases weighting transitions more heavily than transversions was most effective.
FAU - Broughton, R E
AU  - Broughton RE
AD  - Department of Ecology and Evolutionary Biology, Cornell University, Ithaca, New
      York 14853-2701, USA.
FAU - Stanley, S E
AU  - Stanley SE
FAU - Durrett, R T
AU  - Durrett RT
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Base Sequence
MH  - Mutation
MH  - *Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/106351500750049734 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Dec;49(4):617-27. doi: 10.1080/106351500750049734.

PMID- 12116429
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 3
DP  - 2000 Sep
TI  - Phylogenetic relationships among the phrynosomatid sand lizards inferred from
      mitochondrial DNA sequences generated by heterogeneous evolutionary processes.
PG  - 592-612
AB  - Nucleotide sequences of the mitochondrial protein coding cytochrome b (cyt b; 650
      bp) and small-subunit 12S ribosomal RNA (approximately 350 bp) genes were used in
      analyses of phylogenetic relationships among extant phrynosomatid sand lizards,
      including an examination of competing hypotheses regarding the evolution of
      "earlessness." Sequences were obtained from all currently recognized species of
      sand lizards as well as representatives of the first and second outgroups and
      analyzed using both parsimony and likelihood methods. The cyt b data offer strong
      support for relationships that correspond with relatively recent divergences and 
      moderate to low support for relationships reflecting more ancient divergences
      within the clade. These data support monophyly of the "earless" taxa, the
      placement of Uma as the sister taxon to the other sand lizards, and monophyly of 
      all four taxa traditionally ranked as genera. All well-supported relationships in
      the 12S phylogeny are completely congruent with well-supported relationships in
      the cyt b phylogeny; however, the 12S data alone provide very little support for 
      deeper divergences. Phylogenetic relationships within species are concordant with
      geography and suggest patterns of phylogeographic differentiation, including the 
      conclusion that at least one currently recognized species (Holbrookia maculata)
      actually consists of more than one species. By independently optimizing
      likelihood model parameters for various subsets of the data, we found that
      nucleotide substitution processes vary widely between genes and among the
      structural and functional regions or classes of sites within each gene.
      Therefore, we compared competing phylogenetic hypotheses, using parameter
      estimates specific to those subsets, analyzing the subsets separately and in
      various combinations. The hypothesis supported by the cyt b data was favored over
      rival hypotheses in all but one of the five comparisons made with the entire data
      set, including the set of partitions that best explained the data, although we
      were unable to confidently reject (P &lt; 0.05) alternative hypotheses. Our results 
      highlight the importance of optimizing models and parameter estimates for
      different genes or parts thereof--a strategy that takes advantages of the
      strengths of both combining and partitioning data.
FAU - Wilgenbusch, J
AU  - Wilgenbusch J
AD  - Laboratory of Molecular Systematics, MRC 534, MSC, Smithsonian Institution,
      Washington, D.C. 20560, USA. jim@lms.si.edu
FAU - de Queiroz, K
AU  - de Queiroz K
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - *Biological Evolution
MH  - Classification/methods
MH  - Colorado
MH  - Cytochrome b Group/genetics
MH  - DNA Primers
MH  - DNA, Mitochondrial/*genetics
MH  - Genetic Variation
MH  - Liver/chemistry
MH  - Lizards/classification/*genetics
MH  - Muscle, Skeletal/chemistry
MH  - Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635159950127411 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Sep;49(3):592-612. doi: 10.1080/10635159950127411.

PMID- 12116428
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 3
DP  - 2000 Sep
TI  - Timing transantarctic disjunctions in the Atherospermataceae (Laurales): evidence
      from coding and noncoding chloroplast sequences.
PG  - 579-91
AB  - Previous studies of the small Southern Hemisphere family Atherospermataceae have 
      drawn contradictory conclusions regarding the number of transantarctic
      disjunctions and role of transoceanic dispersal in its evolution. Clarification
      of intergeneric relationships is critical to resolving (1) whether the two
      Chilean species, Laurelia sempervirens and Laureliopsis philippiana, are related 
      to different Austral-Pacific species, implying two transantarctic disjunctions as
      suggested by morphology; (2) where the group is likely to have originated; and
      (3) whether observed disjunctions reflect the breakup of Gondwana. We analyzed
      chloroplast DNA sequences from six regions (the rbcL gene, the rpl16 intron, and 
      the trnL-trnF, trnT-trnL, psbA-trnH, and atpB-rbcL spacer regions; for all six
      regions, 4,372 bp) for all genera and most species of Atherospermataceae, using
      parsimony and maximum likelihood (ML). The family's sister group, the Chilean
      endemic Gomortega nitida (Gomortegaceae), was used to root the tree. Parsimony
      and ML yielded identical single best trees that contain three well-supported
      clades (&gt; or = 75% bootstrap): Daphnandra and Doryphora from south-eastern
      Australia; Atherosperma and Nemuaron from Australia-Tasmania and New Caledonia,
      respectively; and Laurelia novac-zelandiac and Laureliopsis philippiana from New 
      Zealand and Chile, respectively. The second Chilean species, Laurelia
      sempervirens, is sister to this last clade. Likelihood ratio testing did not
      reject the molecular clock assumption for the rbcL data, which can therefore be
      used for divergence time estimates. The atherosperm fossil record, which goes
      back to the Upper Cretaceous, includes pollen, wood, and leaf fossils from
      Europe, Africa, South America, Antarctica, New Zealand, and Tasmania. Calibration
      of rbcL substitution rates with the fossils suggests an initial diversification
      of the family at 100-140 million years ago (MYA), probably in West Gondwana,
      early entry into Antarctica, and long-distance dispersal to New Zealand and New
      Caledonia at 50-30 MYA by the ancestors of L. novae-zelandiae and Nemuaron.
FAU - Renner, S S
AU  - Renner SS
AD  - Department of Biology, University of Missouri, St. Louis, Missouri 63121-4499,
      USA. Renner@admiral.umsl.edu
FAU - Foreman, D B
AU  - Foreman DB
FAU - Murray, D
AU  - Murray D
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Antarctic Regions
MH  - Chloroplasts/*genetics
MH  - Magnoliopsida/*classification/*genetics
MH  - Phylogeny
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635159950127402 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Sep;49(3):579-91. doi: 10.1080/10635159950127402.

PMID- 12116427
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 3
DP  - 2000 Sep
TI  - Variation in mandible shape in Thrichomys apereoides (Mammalia: Rodentia):
      geometric analysis of a complex morphological structure.
PG  - 563-78
AB  - The model of development and evolution of complex morphological structures
      conceived by Atchley and Hall in 1991 (Biol. Rev. 66:101-157), which establishes 
      that changes at the macroscopic, morphogenetic level can be statistically
      detected as variation in skeletal units at distinct scales, was applied in
      combination with the formalism of geometric morphometrics to study variation in
      mandible shape among populations of the rodent species Thrichomys apereoides. The
      thin-plate spline technique produced geometric descriptors of shape derived from 
      anatomical landmarks in the mandible, which we used with graphical and
      inferential approaches to partition the contribution of global and localized
      components to the observed differentiation in mandible shape. A major pattern of 
      morphological differentiation in T. apereoides is attributable to localized
      components of shape at smaller geometric scales associated with specific
      morphogenetic units of the mandible. On the other hand, a clinical trend of
      variation is associated primarily with localized components of shape at larger
      geometric scales. Morphogenetic mechanisms assumed to be operating to produce the
      observed differentiation in the specific units of the mandible include
      mesenchymal condensation differentiation, muscle hypertrophy, and tooth growth.
      Perspectives for the application of models of morphological evolution and
      geometric morphometrics to morphologically based systematic biology are
      considered.
FAU - Duarte, L C
AU  - Duarte LC
AD  - Programa de Pos-Graduacao em Zoologia, Universidade Estadual Paulista, Caixa
      Postal 179, 13506-900 Rio Claro, Sao Paulo, Brazil.
FAU - Monteiro, L R
AU  - Monteiro LR
FAU - Von Zuben, F J
AU  - Von Zuben FJ
FAU - Dos Reis, S F
AU  - Dos Reis SF
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Genetic Variation
MH  - Mandible/*anatomy &amp; histology
MH  - Rodentia/*anatomy &amp; histology/classification/*genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635159950127394 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Sep;49(3):563-78. doi: 10.1080/10635159950127394.

PMID- 12116426
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 3
DP  - 2000 Sep
TI  - Triploblastic relationships with emphasis on the acoelomates and the position of 
      Gnathostomulida, Cycliophora, Plathelminthes, and Chaetognatha: a combined
      approach of 18S rDNA sequences and morphology.
PG  - 539-62
AB  - Triploblastic relationships were examined in the light of molecular and
      morphological evidence. Representatives for all triploblastic "phyla" (except
      Loricifera) were represented by both sources of phylogenetic data. The 18S
      ribosomal (rDNA) sequence data for 145 terminal taxa and 276 morphological
      characters coded for 36 supraspecific taxa were combined in a total evidence
      regime to determine the most consistent picture of triploblastic relationships
      for these data. Only triploblastic taxa are used to avoid rooting with distant
      outgroups, which seems to happen because of the extreme distance that separates
      diploblastic from triploblastic taxa according to the 18S rDNA data. Multiple
      phylogenetic analyses performed with variable analysis parameters yield largely
      inconsistent results for certain groups such as Chaetognatha, Acoela, and
      Nemertodermatida. A normalized incongruence length metric is used to assay the
      relative merit of the multiple analyses. The combined analysis having the least
      character incongruence yields the following scheme of relationships of four main 
      clades: (1) Deuterostomia [((Echinodermata + Enteropneusta) (Cephalochordata
      (Urochordata + Vertebrata)))]; (2) Ecdysozoa [(((Priapulida + Kinorhyncha)
      (Nematoda + Nematomorpha)) ((Onychophora + Tardigrada) Arthropoda))]; (3)
      Trochozoa [((Phoronida + Brachiopoda) (Entoprocta (Nemertea (Sipuncula (Mollusca 
      (Pogonophora (Echiura + Annelida)))))))]; and (4) Platyzoa [((Gnathostomulida
      (Cycliophora + Syndermata)) (Gastrotricha + Plathelminthes))]. Chaetognatha,
      Nemertodermatida, and Bryozoa cannot be assigned to any one of these four groups.
      For the first time, a data analysis recognizes a clade of acoelomates, the
      Platyzoa (sensu Cavalier-Smith, Biol. Rev. 73:203-266, 1998). Other relationships
      that corroborate some morphological analyses are the existence of a clade that
      groups Gnathostomulida + Syndermata (= Gnathifera), which is expanded to include 
      the enigmatic phylum Cycliophora, as sister group to Syndermata.
FAU - Giribet, G
AU  - Giribet G
AD  - Division of Invertebrate Zoology, American Museum of Natural History, New York,
      New York 10024-5192, USA. ggiribet@oeb.harvard.edu
FAU - Distel, D L
AU  - Distel DL
FAU - Polz, M
AU  - Polz M
FAU - Sterrer, W
AU  - Sterrer W
FAU - Wheeler, W C
AU  - Wheeler WC
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - 0 (RNA, Ribosomal, 18S)
SB  - IM
MH  - Animals
MH  - DNA, Ribosomal/*genetics
MH  - Gnathostoma/classification/*genetics
MH  - Invertebrates/classification/*genetics
MH  - *Phylogeny
MH  - Platyhelminths/classification/*genetics
MH  - RNA, Ribosomal, 18S/*genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635159950127385 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Sep;49(3):539-62. doi: 10.1080/10635159950127385.

PMID- 12116425
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 3
DP  - 2000 Sep
TI  - Hidden morphological support for the phylogenetic placement of Pseudoryx
      nghetinhensis with bovine bovids: a combined analysis of gross anatomical
      evidence and DNA sequences from five genes.
PG  - 515-38
AB  - The saola, Pseudoryx nghetinhensis, was unknown to science until its formal
      description in 1993. This endangered species is a member of the ruminant
      artiodactyl family Bovidae (cattle, sheep, goats, and antelopes). However, given 
      its puzzling combination of morphological traits, the specific affinities of
      Pseudoryx within Bovidae are controversial. A preliminary genetic investigation
      suggested that Pseudoryx should be placed in the subfamily Bovinae (cattle,
      buffaloes, spiral-horned antelopes, and nilgai), but a recent cladistic analysis 
      of skeletal and dental characters allied Pseudoryx with caprine bovids (sheep,
      goats, musk oxen, goat antelopes, and Pantholops). The morphological and
      molecular hypotheses differ in assigning the saola to either of the two most
      divergent clades of Bovidae. In this report, phylogenetic analyses of DNA
      sequences from five genes are used to test these alternatives. Protein coding
      regions, introns, and ribosomal DNAs from the nuclear and mitochondrial genomes
      discount the hypothesis that Pseudoryx is a close relative of Caprinae. Instead, 
      combined analyses of the DNA data and published morphological evidence place
      Pseudoryx with Bovini (cattle and buffaloes), a subclade of Bovinae. In a
      separate analysis, the matrix of morphological characters links Pseudoryx with
      caprine bovids, but in the context of the molecular data, the gross anatomical
      evidence strongly supports a grouping of Pseudoryx with Bovinae. Surprisingly,
      the morphological partition provides the most character support in the combined
      analysis. This striking result is obscured by separate analyses of the individual
      data sets and the taxonomic congruence approach.
FAU - Gatesy, J
AU  - Gatesy J
AD  - Laboratory of Molecular Systematics and Evolution, Department of Ecology and
      Evolutionary Biology, University of Arizona, Biosciences West, Tucson, Arizona
      85721, USA. hayashi@uwyo.edu
FAU - Arctander, P
AU  - Arctander P
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Cattle/classification/*genetics
MH  - DNA/*genetics
MH  - DNA Primers
MH  - *Genes
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Ruminants/classification/*genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635159950127376 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Sep;49(3):515-38. doi: 10.1080/10635159950127376.

PMID- 12116424
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 3
DP  - 2000 Sep
TI  - c-myc gene sequences and the phylogeny of bats and other eutherian mammals.
PG  - 501-14
AB  - The complete protein-coding sequences of the c-myc proto-oncogene were determined
      for five species of four new orders of eutherian (placental) mammals. These newly
      obtained sequences were aligned to each other and to other available orthologs
      for the phylogenetic estimation of eutherian interordinal relationships. Several 
      measures of sequence difference and base composition were first calculated to
      assess the major evolutionary properties of the three codon positions and two
      protein-coding exons of the gene. On the basis of these calculations, different
      parsimony, distance, and maximum likelihood approaches were adopted, with the
      most sophisticated involving the separate, then combined, likelihood analyses of 
      the third codon positions of exon 2 versus all other sites. These phylogenetic
      approaches provided clear support for the grouping of Chiroptera (bats) with
      Artiodactyla (ruminants, camels, and pigs) and Carnivora (cats, dogs, and their
      allies), an interordinal arrangement that receives strong corroboration from
      other lines of evidence including complete mitochondrial DNA sequences. In
      contrast, these analyses failed to provide strong to reasonable support for any
      other interordinal group. This study concludes with specific recommendations
      about sampling and other strategies for maximizing the phylogenetic contributions
      of the c-myc gene to the continued resolution of the eutherian ordinal tree.
FAU - Miyamoto, M M
AU  - Miyamoto MM
AD  - Department of Zoology, P.O. Box 118525, University of Florida, Gainesville,
      Florida 32611-8525, USA. miyamoto@zoo.ufl.edu
FAU - Porter, C A
AU  - Porter CA
FAU - Goodman, M
AU  - Goodman M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Cats/genetics
MH  - Chiroptera/classification/*genetics
MH  - DNA Primers
MH  - Dogs/genetics
MH  - *Genes, myc
MH  - Humans
MH  - Mammals/classification/*genetics
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Homology, Nucleic Acid
MH  - Species Specificity
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635159950127367 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Sep;49(3):501-14. doi: 10.1080/10635159950127367.

PMID- 12116423
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 3
DP  - 2000 Sep
TI  - Taxic homology and three-taxon statement analysis.
PG  - 480-500
AB  - Three-taxon statement analysis (3TA) and standard cladistic analysis (SCA) were
      evaluated relative to propositions of taxic homology. There are definite
      distinctions between complement relation homologs and paired homologs. The
      complement relation is discussed, relative to rooting, parsimony, and taxic
      propositions of homology. The complement relation, as implemented in SCA, makes
      sense only because SCA is a simple evolutionary model of character-state
      transformation. 3TA is a method for implementing complement relation data from a 
      taxic perspective. The standard approach to cladistic analysis distinguishes taxa
      by rooting a tree, which means that that approach is incompatible with taxic
      propositions of homology, because a taxic homology is a hypothesis of
      relationship between taxa that possess a homolog relative to taxa that lack a
      homolog. It is not necessary to treat paired homologs from a transformational
      perspective to distinguish informative from uninformative data. 3TA yields
      results markedly different from those of SCA. SCA, which seeks to minimize tree
      length, may not maximize the relation of homology (congruence) relative to a
      tree.
FAU - Scotland, R W
AU  - Scotland RW
AD  - Department of Plant Sciences, University of Oxford, South Parks Road, Oxford OX1 
      3RB, United Kingdom. Robert.Scotland@Plant-Sciences.ox.ac.uk
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Models, Biological
MH  - Reproducibility of Results
MH  - Sequence Homology, Nucleic Acid
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635159950127358 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Sep;49(3):480-500. doi: 10.1080/10635159950127358.

PMID- 12116422
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 3
DP  - 2000 Sep
TI  - Age rank/clade rank metrics--sampling, taxonomy, and the meaning of
      "stratigraphic consistency".
PG  - 463-79
AB  - Paleontologists frequently contrast clade rank (i.e., nodal or patristic distance
      from the base of a cladogram) with age rank (i.e., relative first known
      appearances of the analyzed taxa) to measure the degree of congruence between the
      estimated phylogeny and the fossil record. Although some potential biases of
      these methods have been examined (e.g., the effect of tree imbalance), other
      properties of age rank/clade rank (ARCR) comparisons have not been studied in
      detail. A basic premise of ARCR metrics is that outgroup taxa diverged earlier
      than ingroups and thus should first appear in older strata. For example, given
      phylogeny (A,(B,C)), then taxon A should be sampled before either taxon B or
      taxon C. We examine this premise in the context of (1) phylogenetic theory, (2)
      taxonomic practice, (3) sampling intensity (R), and (4) factors other than
      sampling intensity (including cladogram accuracy). Simulations combining clade
      evolution and sampling over time indicate a poor relationship between ARCR
      metrics and R when all taxa are apomorphy-based monophyletic groups. However, a
      good relationship exists when taxa are either stem-based monophyletic groups or
      if workers include taxa without a priori decisions about monophyly or paraphyly. 
      These results are not surprising because cladograms predict the order in which
      lineages diverged (which applies to stem-based monophyletic taxa) and the order
      in which morphologic grades appeared (which applies to paraphyletic taxa relative
      to derived monophyletic groups). Other factors that increase ARCR metrics when
      the average R stays the same include high temporal variation in R, budding
      instead of bifurcating speciation patterns, low extinction rates, cladogram
      inaccuracy, and (to a much lesser extent) large clade size. These results suggest
      several plausible explanations for patterned differences in ARCR metrics among
      clades, thereby compromising their validity as measures of the quality of the
      fossil record.
FAU - Wagner, P J
AU  - Wagner PJ
AD  - Department of Geology, Field Museum of Natural History, Roosevelt Road at Lake
      Shore Drive, Chicago, Illinois 60605, USA. pwagner@fmnh.org
FAU - Sidor, C A
AU  - Sidor CA
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Models, Biological
MH  - Paleontology/methods
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635159950127349 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Sep;49(3):463-79. doi: 10.1080/10635159950127349.

PMID- 12116421
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 3
DP  - 2000 Sep
TI  - The phylogenetic basis of sexual size dimorphism in orb-weaving spiders (Araneae,
      Orbiculariae).
PG  - 435-62
AB  - Extreme sexual body size dimorphism (SSD), in which males are only a small
      fraction of the size of the females, occurs only in a few, mostly marine,
      taxonomic groups. Spiders are the only terrestrial group in which small males are
      relatively common, particularly among orb-weavers (especially in the families
      Tetragnathidae and Araneidae) and crab spiders (Thomisidae). We used a taxonomic 
      sample of 80 genera to study the phylogenetic patterns (origins and reversals) of
      SSD in orb-weaving spiders (Orbiculariae). We collected and compiled male and
      female size data (adult body length) for 536 species. Size data were treated as a
      continuous character, and ancestral sizes, for males and females separately, were
      reconstructed by using Wagner parsimony on a cladogram for the 80 genera used in 
      this study. Of these 80 genera, 24 were female-biased dimorphic (twice or more
      the body length of the male); the remaining 56 genera were monomorphic. Under
      parsimony only four independent origins of dimorphism are required: in the
      theridiid genus Tidarren, in the distal nephilines, in the "argiopoid clade," and
      in the araneid genus Kaira. Dimorphism has reversed to monomorphism at least
      seven times, all of them within the large "argiopoid clade." The four independent
      origins of dimorphism represent two separate instances of an increase in female
      size coupled with a decrease of male size (involving only two genera), and two
      separate instances of an increase in female size with male size either remaining 
      the same or increasing, but not as much as females (involving 30 genera). In
      orb-weaving spiders, far more taxa are sexually dimorphic as a result of female
      size increase (22 genera) than as a result of male size decrease (two genera).
      SSD in orb-weaving spiders encompasses several independent evolutionary histories
      that together suggest a variety of evolutionary pathways. This multiplicity
      strongly refutes all efforts thus far to find a general explanation for either
      the origin or maintenance (or both) of SSD, because the different pathways very
      likely will require distinctly different, possibly unique, explanations. Each
      pattern must be understood historically before its origin and maintenance can be 
      explained in ecological and evolutionary terms. The most frequently cited example
      of male dwarfism in spiders, the golden orb-weaving spider genus Nephila
      (Tetragnathidae), is in fact a case of female giantism, not male dwarfism.
FAU - Hormiga, G
AU  - Hormiga G
AD  - Department of Biological Sciences, George Washington University, Department of
      Entomology, National Museum of Natural History, Smithsonian Institution,
      Washington, D.C., USA. hormiga@gwu.edu
FAU - Scharff, N
AU  - Scharff N
FAU - Coddington, J A
AU  - Coddington JA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Body Constitution/genetics
MH  - Classification/methods
MH  - Female
MH  - Male
MH  - *Phylogeny
MH  - Sex Characteristics
MH  - Spiders/anatomy &amp; histology/*classification
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635159950127330 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Sep;49(3):435-62. doi: 10.1080/10635159950127330.

PMID- 12116420
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 3
DP  - 2000 Sep
TI  - Testing hybridization hypotheses based on incongruent gene trees.
PG  - 422-34
AB  - Hybridization is an important evolutionary mechanism in plants and has been
      increasingly documented in animals. Difficulty in reconstruction of reticulate
      evolution, however, has been a long-standing problem in phylogenetics.
      Consequently, hybrid speciation may play a major role in causing topological
      incongruence between gene trees. The incongruence, in turn, offers an opportunity
      to detect hybrid speciation. Here we characterized certain distinctions between
      hybridization and other biological processes, including lineage sorting,
      paralogy, and lateral gene transfer, that are responsible for topological
      incongruence between gene trees. Consider two incongruent gene trees with three
      taxa, A, B, and C, where B is a sister group of A on gene tree 1 but a sister
      group of C on gene tree 2. With a theoretical model based on the molecular clock,
      we demonstrate that time of divergence of each gene between taxa A and C is
      nearly equal in the case of hybridization (B is a hybrid) or lateral gene
      transfer, but differs significantly in the case of lineage sorting or paralogy.
      After developing a bootstrap test to test these alternative hypotheses, we
      extended the model and test to account for incongruent gene trees with numerous
      taxa. Computer simulation studies supported the validity of the theoretical model
      and bootstrap test when each gene evolved at a constant rate. The computer
      simulation also suggested that the model remained valid as long as the rate
      heterogeneity was occurring proportionally in the same taxa for both genes.
      Although the model could not test hypotheses of hybridization versus lateral gene
      transfer as the cause of incongruence, these two processes may be distinguished
      by comparing phylogenies of multiple unlinked genes.
FAU - Sang, T
AU  - Sang T
AD  - Department of Botany and Plant Pathology, Michigan State University, East
      Lansing, Michigan 48824, USA. sang@pilot.msu.edu
FAU - Zhong, Y
AU  - Zhong Y
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - *Nucleic Acid Hybridization
MH  - *Phylogeny
MH  - Plants/classification/*genetics
MH  - Reproducibility of Results
MH  - Trees/classification/genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635159950127321 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Sep;49(3):422-34. doi: 10.1080/10635159950127321.

PMID- 12116419
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 3
DP  - 2000 Sep
TI  - Divergence and reticulation among montane populations of a jumping spider
      (Habronattus pugillis Griswold).
PG  - 400-21
AB  - Populations of the jumping spider Habronattus pugillis Griswold isolated on
      nearby mountain ranges in southern Arizona are differentiated in many features of
      the males (color, shape, and orientation of setae on face; shape of carapace;
      markings of palpi and legs; motions during courtship behavior). These features
      are (mostly) consistent within a range and different between ranges. The
      concentration of differences in male courtship behavior and body parts exposed to
      the female during courtship and correlations between form and courtship behavior 
      suggest sexual selection was involved in the differentiation. A phylogenetic
      analysis of the populations yields a tree that for the most part groups
      geographic neighbors, but the history of H. pugillis populations may not be
      adequately described by a tree. Geographic proximity of apparent convergences
      suggests that populations from at least some of the mountain ranges acquired
      characteristics through introgression. Lowering of the woodland habitat during
      the last glacial period probably brought some populations into contact, but it is
      not clear whether the interrange woodlands would have provided corridors for
      extensive mixing.
FAU - Maddison, W
AU  - Maddison W
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson,
      Arizona 85721, USA.
FAU - McMahon, M
AU  - McMahon M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Altitude
MH  - Animals
MH  - Face/anatomy &amp; histology
MH  - *Genetic Variation
MH  - Geography
MH  - Male
MH  - Spiders/anatomy &amp; histology/*classification/*genetics
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635159950127312 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Sep;49(3):400-21. doi: 10.1080/10635159950127312.

PMID- 12116418
OWN - NLM
STAT- MEDLINE
DCOM- 20020809
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 49
IP  - 3
DP  - 2000 Sep
TI  - Seabird and louse coevolution: complex histories revealed by 12S rRNA sequences
      and reconciliation analyses.
PG  - 383-99
AB  - We investigated the coevolutionary history of seabirds (orders Procellariiformes 
      and Sphenisciformes) and their lice (order Phthiraptera). Independent trees were 
      produced for the seabirds (tree derived from 12S ribosomal RNA, isoenzyme, and
      behavioral data) and their lice (trees derived from 12S rRNA data). Brook's
      parsimony analysis (BPA) supported a general history of cospeciation (consistency
      index = 0.84, retention index = 0.81). We inferred that the homoplasy in the BPA 
      was caused by one intrahost speciation, one potential host-switching, and eight
      or nine sorting events. Using reconciliation analysis, we quantified the cost of 
      fitting the louse tree onto the seabird tree. The reconciled trees postulated one
      host-switching, nine cospeciation, three or four intrahost speciation, and 11 to 
      14 sorting events. The number of cospeciation events was significantly more than 
      would be expected from chance alone (P &lt; 0.01). The sequence data were used to
      test for rate heterogeneity for both seabirds and lice. Neither data set
      displayed significant rate heterogeneity. An examination of the codivergent nodes
      revealed that seabirds and lice have cospeciated synchronously and that lice have
      evolved at approximately 5.5 times the rate of seabirds. The degree of sequence
      divergence supported some of the postulated intrahost speciation events (e.g.,
      Halipeurus predated the evolution of their present hosts). The sequence data also
      supported some of the postulated host-switching events. These results demonstrate
      the value of sequence data and reconciliation analyses in unraveling complex
      histories between hosts and their parasites.
FAU - Paterson, A M
AU  - Paterson AM
AD  - Ecology and Entomology Group, Lincoln University, P.O. Box 84, Lincoln, New
      Zealand. PatersoA@lincoln.ac.nz
FAU - Wallis, G P
AU  - Wallis GP
FAU - Wallis, L J
AU  - Wallis LJ
FAU - Gray, R D
AU  - Gray RD
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal)
RN  - 0 (RNA, ribosomal, 12S)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - *Biological Evolution
MH  - Birds/classification/*genetics/parasitology
MH  - Genetic Variation
MH  - Host-Parasite Interactions
MH  - Phthiraptera/classification/*genetics
MH  - *Phylogeny
MH  - RNA, Ribosomal/*genetics
MH  - Reproducibility of Results
MH  - Seawater
EDAT- 2002/07/16 10:00
MHDA- 2002/08/10 10:01
CRDT- 2002/07/16 10:00
PHST- 2002/07/16 10:00 [pubmed]
PHST- 2002/08/10 10:01 [medline]
PHST- 2002/07/16 10:00 [entrez]
AID - 10.1080/10635159950127303 [doi]
PST - ppublish
SO  - Syst Biol. 2000 Sep;49(3):383-99. doi: 10.1080/10635159950127303.

PMID- 12079649
OWN - NLM
STAT- MEDLINE
DCOM- 20020930
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 3
DP  - 2002 Jun
TI  - Preferential host switching by primate lentiviruses can account for phylogenetic 
      similarity with the primate phylogeny.
PG  - 528-35
AB  - Primate lentiviruses (PLV) from closely related primate species have been
      observed to be more closely related to each other than to PLV from more distantly
      related primate species. The current explanation for this observation is the
      codivergence hypothesis; that is, the divergence of a virus lineage results from 
      the divergence of the host lineage. We show that, alternatively, frequent
      cross-species transmission of PLV, coupled with a tendency for more closely
      related primate species to exchange viruses "successfully," can result in
      apparent codivergence. This host-switching hypothesis reconciles several puzzling
      observations related to the evolution of PLV.
FAU - Charleston, M A
AU  - Charleston MA
AD  - Department of Zoology, University of Oxford, South Parks Road, Oxford, OX1 3PS,
      UK. michael.charleston@zoo.ox.ac.uk
FAU - Robertson, D L
AU  - Robertson DL
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Computer Simulation
MH  - Genetic Vectors
MH  - Humans
MH  - Lentivirus/*genetics
MH  - Pan troglodytes
MH  - *Phylogeny
EDAT- 2002/06/25 10:00
MHDA- 2002/10/02 04:00
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/10/02 04:00 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/10635150290069940 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Jun;51(3):528-35. doi: 10.1080/10635150290069940.

PMID- 12079648
OWN - NLM
STAT- MEDLINE
DCOM- 20020930
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 51
IP  - 3
DP  - 2002 Jun
TI  - Type 1 error rates of the parsimony permutation tail probability test.
PG  - 524-7
FAU - Wilkinson, Mark
AU  - Wilkinson M
AD  - Department of Zoology, The Natural History Museum, London SW7 5BD, UK.
      marw@nhm.ac.uk
FAU - Peres-Neto, Pedro R
AU  - Peres-Neto PR
FAU - Foster, Peter G
AU  - Foster PG
FAU - Moncrieff, Clive B
AU  - Moncrieff CB
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Computer Simulation
MH  - Databases as Topic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2002/06/25 10:00
MHDA- 2002/10/02 04:00
CRDT- 2002/06/25 10:00
PHST- 2002/06/25 10:00 [pubmed]
PHST- 2002/10/02 04:00 [medline]
PHST- 2002/06/25 10:00 [entrez]
AID - 10.1080/10635150290069931 [doi]
PST - ppublish
SO  - Syst Biol. 2002 Jun;51(3):524-7. doi: 10.1080/10635150290069931.
</pre></body></html>