<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0035)https://www.ncbi.nlm.nih.gov/pubmed -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"></head><body><pre>PMID- 22213708
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - Interpreting the evolutionary regression: the interplay between observational and
      biological errors in phylogenetic comparative studies.
PG  - 413-25
LID - 10.1093/sysbio/syr122 [doi]
AB  - Regressions of biological variables across species are rarely perfect. Usually,
      there are residual deviations from the estimated model relationship, and such
      deviations commonly show a pattern of phylogenetic correlations indicating that
      they have biological causes. We discuss the origins and effects of
      phylogenetically correlated biological variation in regression studies. In
      particular, we discuss the interplay of biological deviations with deviations due
      to observational or measurement errors, which are also important in comparative
      studies based on estimated species means. We show how bias in estimated
      evolutionary regressions can arise from several sources, including phylogenetic
      inertia and either observational or biological error in the predictor variables. 
      We show how all these biases can be estimated and corrected for in the presence
      of phylogenetic correlations. We present general formulas for incorporating
      measurement error in linear models with correlated data. We also show how
      alternative regression models, such as major axis and reduced major axis
      regression, which are often recommended when there is error in predictor
      variables, are strongly biased when there is biological variation in any part of 
      the model. We argue that such methods should never be used to estimate
      evolutionary or allometric regression slopes.
FAU - Hansen, Thomas F
AU  - Hansen TF
AD  - Department of Biology, Centre for Ecological and Evolutionary Synthesis,
      University of Oslo, PB 1066, Blindern, N-0316 Oslo, Norway.
      Thomas.Hansen@bio.uio.no
FAU - Bartoszek, Krzysztof
AU  - Bartoszek K
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20120102
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bias
MH  - Linear Models
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Regression Analysis
EDAT- 2012/01/04 06:00
MHDA- 2013/04/02 06:00
CRDT- 2012/01/04 06:00
PHST- 2012/01/04 06:00 [entrez]
PHST- 2012/01/04 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - syr122 [pii]
AID - 10.1093/sysbio/syr122 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):413-25. doi: 10.1093/sysbio/syr122. Epub 2012 Jan 2.

PMID- 22201160
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20181201
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - The poverty of the phylocode: a reply to de Queiroz and Donoghue.
PG  - 360-1
LID - 10.1093/sysbio/syr117 [doi]
FAU - Platnick, Norman I
AU  - Platnick NI
AD  - Division of Invertebrate Zoology, American Museum of Natural History, New York,
      NY 10024, USA. platnick@amnh.org
LA  - eng
PT  - Journal Article
PT  - Comment
DEP - 20111226
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2011 Dec;60(6):887-92. PMID: 21865335
CIN - Syst Biol. 2013 Jan 1;62(1):167-74. PMID: 22649180
CIN - Syst Biol. 2013 Jan 1;62(1):175-6. PMID: 22730418
MH  - Animals
MH  - Classification/*methods
MH  - *Phylogeny
MH  - *Terminology as Topic
EDAT- 2011/12/28 06:00
MHDA- 2012/06/26 06:00
CRDT- 2011/12/28 06:00
PHST- 2011/12/28 06:00 [entrez]
PHST- 2011/12/28 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr117 [pii]
AID - 10.1093/sysbio/syr117 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):360-1. doi: 10.1093/sysbio/syr117. Epub 2011 Dec 26.

PMID- 22201159
OWN - NLM
STAT- MEDLINE
DCOM- 20130401
LR  - 20120420
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 3
DP  - 2012 May
TI  - Recurrent introgression of mitochondrial DNA among hares (Lepus spp.) revealed by
      species-tree inference and coalescent simulations.
PG  - 367-81
LID - 10.1093/sysbio/syr114 [doi]
AB  - Understanding recent speciation history requires merging phylogenetic and
      population genetics approaches, taking into account the persistence of ancestral 
      polymorphism and possible introgression. The emergence of a clear phylogeny of
      hares (genus Lepus) has been hampered by poor genomic sampling and possible
      occurrence of mitochondrial DNA (mtDNA) introgression from the arctic/boreal
      Lepus timidus into several European temperate and possibly American boreal
      species. However, no formal test of introgression, taking also incomplete lineage
      sorting into account, has been done. Here, to clarify the yet poorly resolved
      species phylogeny of hares and test hypotheses of mtDNA introgression, we
      sequenced 14 nuclear DNA and 2 mtDNA fragments (8205 and 1113 bp, respectively)
      in 50 specimens from 11 hare species from Eurasia, North America, and Africa. By 
      applying an isolation-with-migration model to the nuclear data on subsets of
      species, we find evidence for very limited gene flow from L. timidus into most
      temperate European species, and not into the American boreal ones. Using a
      multilocus coalescent-based method, we infer the species phylogeny, which we find
      highly incongruent with mtDNA phylogeny using parametric bootstrap. Simulations
      of mtDNA evolution under the speciation history inferred from nuclear genes did
      not support the hypothesis of mtDNA introgression from L. timidus into the
      American L. townsendii but did suggest introgression from L. timidus into 4
      temperate European species. One such event likely resulted in the complete
      replacement of the aboriginal mtDNA of L. castroviejoi and of its sister species 
      L. corsicanus. It is remarkable that mtDNA introgression in hares is frequent,
      extensive, and always from the same donor arctic species. We discuss possible
      explanations for the phenomenon in relation to the dynamics of range expansions
      and species replacements during the climatic oscillations of the Pleistocene.
FAU - Melo-Ferreira, J
AU  - Melo-Ferreira J
AD  - CIBIO, Centro de Investigacao em Biodiversidade e Recursos Geneticos,
      Universidade do Porto, Campus Agrario de Vairao, 4485-661 Vairao, Portugal.
      jmeloferreira@cibio.up.pt
FAU - Boursot, P
AU  - Boursot P
FAU - Carneiro, M
AU  - Carneiro M
FAU - Esteves, P J
AU  - Esteves PJ
FAU - Farelo, L
AU  - Farelo L
FAU - Alves, P C
AU  - Alves PC
LA  - eng
SI  - GENBANK/JN036862
SI  - GENBANK/JN036863
SI  - GENBANK/JN036864
SI  - GENBANK/JN036865
SI  - GENBANK/JN036866
SI  - GENBANK/JN036867
SI  - GENBANK/JN036868
SI  - GENBANK/JN036869
SI  - GENBANK/JN036870
SI  - GENBANK/JN036871
SI  - GENBANK/JN036872
SI  - GENBANK/JN036873
SI  - GENBANK/JN036874
SI  - GENBANK/JN036875
SI  - GENBANK/JN036876
SI  - GENBANK/JN036877
SI  - GENBANK/JN036878
SI  - GENBANK/JN036879
SI  - GENBANK/JN036880
SI  - GENBANK/JN036881
SI  - GENBANK/JN036882
SI  - GENBANK/JN036883
SI  - GENBANK/JN036884
SI  - GENBANK/JN036885
SI  - GENBANK/JN036886
SI  - GENBANK/JN036887
SI  - GENBANK/JN036888
SI  - GENBANK/JN036889
SI  - GENBANK/JN036890
SI  - GENBANK/JN036891
SI  - GENBANK/JN036892
SI  - GENBANK/JN036893
SI  - GENBANK/JN036894
SI  - GENBANK/JN036895
SI  - GENBANK/JN036896
SI  - GENBANK/JN036897
SI  - GENBANK/JN036898
SI  - GENBANK/JN036899
SI  - GENBANK/JN036900
SI  - GENBANK/JN036901
SI  - GENBANK/JN036902
SI  - GENBANK/JN036903
SI  - GENBANK/JN036904
SI  - GENBANK/JN036905
SI  - GENBANK/JN036906
SI  - GENBANK/JN036907
SI  - GENBANK/JN036908
SI  - GENBANK/JN036909
SI  - GENBANK/JN036910
SI  - GENBANK/JN036911
SI  - GENBANK/JN036912
SI  - GENBANK/JN036913
SI  - GENBANK/JN036914
SI  - GENBANK/JN036915
SI  - GENBANK/JN036916
SI  - GENBANK/JN036917
SI  - GENBANK/JN036918
SI  - GENBANK/JN036919
SI  - GENBANK/JN036920
SI  - GENBANK/JN036921
SI  - GENBANK/JN036922
SI  - GENBANK/JN036923
SI  - GENBANK/JN036924
SI  - GENBANK/JN036925
SI  - GENBANK/JN036926
SI  - GENBANK/JN036927
SI  - GENBANK/JN036928
SI  - GENBANK/JN036929
SI  - GENBANK/JN036930
SI  - GENBANK/JN036931
SI  - GENBANK/JN036932
SI  - GENBANK/JN036933
SI  - GENBANK/JN036934
SI  - GENBANK/JN036935
SI  - GENBANK/JN036936
SI  - GENBANK/JN036937
SI  - GENBANK/JN036938
SI  - GENBANK/JN036939
SI  - GENBANK/JN036940
SI  - GENBANK/JN036941
SI  - GENBANK/JN036942
SI  - GENBANK/JN036943
SI  - GENBANK/JN036944
SI  - GENBANK/JN036945
SI  - GENBANK/JN036946
SI  - GENBANK/JN036947
SI  - GENBANK/JN036948
SI  - GENBANK/JN036949
SI  - GENBANK/JN036950
SI  - GENBANK/JN036951
SI  - GENBANK/JN036952
SI  - GENBANK/JN036953
SI  - GENBANK/JN036954
SI  - GENBANK/JN036955
SI  - GENBANK/JN036956
SI  - GENBANK/JN036957
SI  - GENBANK/JN036958
SI  - GENBANK/JN036959
SI  - GENBANK/JN036960
SI  - GENBANK/JN036961
SI  - GENBANK/JN036962
SI  - GENBANK/JN036963
SI  - GENBANK/JN036964
SI  - GENBANK/JN036965
SI  - GENBANK/JN036966
SI  - GENBANK/JN036967
SI  - GENBANK/JN036968
SI  - GENBANK/JN036969
SI  - GENBANK/JN036970
SI  - GENBANK/JN036971
SI  - GENBANK/JN036972
SI  - GENBANK/JN036973
SI  - GENBANK/JN036974
SI  - GENBANK/JN036975
SI  - GENBANK/JN036976
SI  - GENBANK/JN036977
SI  - GENBANK/JN036978
SI  - GENBANK/JN036979
SI  - GENBANK/JN036980
SI  - GENBANK/JN036981
SI  - GENBANK/JN036982
SI  - GENBANK/JN036983
SI  - GENBANK/JN036984
SI  - GENBANK/JN036985
SI  - GENBANK/JN036986
SI  - GENBANK/JN036987
SI  - GENBANK/JN036988
SI  - GENBANK/JN036989
SI  - GENBANK/JN036990
SI  - GENBANK/JN036991
SI  - GENBANK/JN036992
SI  - GENBANK/JN036993
SI  - GENBANK/JN036994
SI  - GENBANK/JN036995
SI  - GENBANK/JN036996
SI  - GENBANK/JN036997
SI  - GENBANK/JN036998
SI  - GENBANK/JN036999
SI  - GENBANK/JN037000
SI  - GENBANK/JN037001
SI  - GENBANK/JN037002
SI  - GENBANK/JN037003
SI  - GENBANK/JN037004
SI  - GENBANK/JN037005
SI  - GENBANK/JN037006
SI  - GENBANK/JN037007
SI  - GENBANK/JN037008
SI  - GENBANK/JN037009
SI  - GENBANK/JN037010
SI  - GENBANK/JN037011
SI  - GENBANK/JN037012
SI  - GENBANK/JN037013
SI  - GENBANK/JN037014
SI  - GENBANK/JN037015
SI  - GENBANK/JN037016
SI  - GENBANK/JN037017
SI  - GENBANK/JN037018
SI  - GENBANK/JN037019
SI  - GENBANK/JN037020
SI  - GENBANK/JN037021
SI  - GENBANK/JN037022
SI  - GENBANK/JN037023
SI  - GENBANK/JN037024
SI  - GENBANK/JN037025
SI  - GENBANK/JN037026
SI  - GENBANK/JN037027
SI  - GENBANK/JN037028
SI  - GENBANK/JN037029
SI  - GENBANK/JN037030
SI  - GENBANK/JN037031
SI  - GENBANK/JN037032
SI  - GENBANK/JN037033
SI  - GENBANK/JN037034
SI  - GENBANK/JN037035
SI  - GENBANK/JN037036
SI  - GENBANK/JN037037
SI  - GENBANK/JN037038
SI  - GENBANK/JN037039
SI  - GENBANK/JN037040
SI  - GENBANK/JN037041
SI  - GENBANK/JN037042
SI  - GENBANK/JN037043
SI  - GENBANK/JN037044
SI  - GENBANK/JN037045
SI  - GENBANK/JN037046
SI  - GENBANK/JN037047
SI  - GENBANK/JN037048
SI  - GENBANK/JN037049
SI  - GENBANK/JN037050
SI  - GENBANK/JN037051
SI  - GENBANK/JN037052
SI  - GENBANK/JN037053
SI  - GENBANK/JN037054
SI  - GENBANK/JN037055
SI  - GENBANK/JN037056
SI  - GENBANK/JN037057
SI  - GENBANK/JN037058
SI  - GENBANK/JN037059
SI  - GENBANK/JN037060
SI  - GENBANK/JN037061
SI  - GENBANK/JN037062
SI  - GENBANK/JN037063
SI  - GENBANK/JN037064
SI  - GENBANK/JN037065
SI  - GENBANK/JN037066
SI  - GENBANK/JN037067
SI  - GENBANK/JN037068
SI  - GENBANK/JN037069
SI  - GENBANK/JN037070
SI  - GENBANK/JN037071
SI  - GENBANK/JN037072
SI  - GENBANK/JN037073
SI  - GENBANK/JN037074
SI  - GENBANK/JN037075
SI  - GENBANK/JN037076
SI  - GENBANK/JN037077
SI  - GENBANK/JN037078
SI  - GENBANK/JN037079
SI  - GENBANK/JN037080
SI  - GENBANK/JN037081
SI  - GENBANK/JN037082
SI  - GENBANK/JN037083
SI  - GENBANK/JN037084
SI  - GENBANK/JN037085
SI  - GENBANK/JN037086
SI  - GENBANK/JN037087
SI  - GENBANK/JN037088
SI  - GENBANK/JN037089
SI  - GENBANK/JN037090
SI  - GENBANK/JN037091
SI  - GENBANK/JN037092
SI  - GENBANK/JN037093
SI  - GENBANK/JN037094
SI  - GENBANK/JN037095
SI  - GENBANK/JN037096
SI  - GENBANK/JN037097
SI  - GENBANK/JN037098
SI  - GENBANK/JN037099
SI  - GENBANK/JN037100
SI  - GENBANK/JN037101
SI  - GENBANK/JN037102
SI  - GENBANK/JN037103
SI  - GENBANK/JN037104
SI  - GENBANK/JN037105
SI  - GENBANK/JN037106
SI  - GENBANK/JN037107
SI  - GENBANK/JN037108
SI  - GENBANK/JN037109
SI  - GENBANK/JN037110
SI  - GENBANK/JN037111
SI  - GENBANK/JN037112
SI  - GENBANK/JN037113
SI  - GENBANK/JN037114
SI  - GENBANK/JN037115
SI  - GENBANK/JN037116
SI  - GENBANK/JN037117
SI  - GENBANK/JN037118
SI  - GENBANK/JN037119
SI  - GENBANK/JN037120
SI  - GENBANK/JN037121
SI  - GENBANK/JN037122
SI  - GENBANK/JN037123
SI  - GENBANK/JN037124
SI  - GENBANK/JN037125
SI  - GENBANK/JN037126
SI  - GENBANK/JN037127
SI  - GENBANK/JN037128
SI  - GENBANK/JN037129
SI  - GENBANK/JN037130
SI  - GENBANK/JN037131
SI  - GENBANK/JN037132
SI  - GENBANK/JN037133
SI  - GENBANK/JN037134
SI  - GENBANK/JN037135
SI  - GENBANK/JN037136
SI  - GENBANK/JN037137
SI  - GENBANK/JN037138
SI  - GENBANK/JN037139
SI  - GENBANK/JN037140
SI  - GENBANK/JN037141
SI  - GENBANK/JN037142
SI  - GENBANK/JN037143
SI  - GENBANK/JN037144
SI  - GENBANK/JN037145
SI  - GENBANK/JN037146
SI  - GENBANK/JN037147
SI  - GENBANK/JN037148
SI  - GENBANK/JN037149
SI  - GENBANK/JN037150
SI  - GENBANK/JN037151
SI  - GENBANK/JN037152
SI  - GENBANK/JN037153
SI  - GENBANK/JN037154
SI  - GENBANK/JN037155
SI  - GENBANK/JN037156
SI  - GENBANK/JN037157
SI  - GENBANK/JN037158
SI  - GENBANK/JN037159
SI  - GENBANK/JN037160
SI  - GENBANK/JN037161
SI  - GENBANK/JN037162
SI  - GENBANK/JN037163
SI  - GENBANK/JN037164
SI  - GENBANK/JN037165
SI  - GENBANK/JN037166
SI  - GENBANK/JN037167
SI  - GENBANK/JN037168
SI  - GENBANK/JN037169
SI  - GENBANK/JN037170
SI  - GENBANK/JN037171
SI  - GENBANK/JN037172
SI  - GENBANK/JN037173
SI  - GENBANK/JN037174
SI  - GENBANK/JN037175
SI  - GENBANK/JN037176
SI  - GENBANK/JN037177
SI  - GENBANK/JN037178
SI  - GENBANK/JN037179
SI  - GENBANK/JN037180
SI  - GENBANK/JN037181
SI  - GENBANK/JN037182
SI  - GENBANK/JN037183
SI  - GENBANK/JN037184
SI  - GENBANK/JN037185
SI  - GENBANK/JN037186
SI  - GENBANK/JN037187
SI  - GENBANK/JN037188
SI  - GENBANK/JN037189
SI  - GENBANK/JN037190
SI  - GENBANK/JN037191
SI  - GENBANK/JN037192
SI  - GENBANK/JN037193
SI  - GENBANK/JN037194
SI  - GENBANK/JN037195
SI  - GENBANK/JN037196
SI  - GENBANK/JN037197
SI  - GENBANK/JN037198
SI  - GENBANK/JN037199
SI  - GENBANK/JN037200
SI  - GENBANK/JN037201
SI  - GENBANK/JN037202
SI  - GENBANK/JN037203
SI  - GENBANK/JN037204
SI  - GENBANK/JN037205
SI  - GENBANK/JN037206
SI  - GENBANK/JN037207
SI  - GENBANK/JN037208
SI  - GENBANK/JN037209
SI  - GENBANK/JN037210
SI  - GENBANK/JN037211
SI  - GENBANK/JN037212
SI  - GENBANK/JN037213
SI  - GENBANK/JN037214
SI  - GENBANK/JN037215
SI  - GENBANK/JN037216
SI  - GENBANK/JN037217
SI  - GENBANK/JN037218
SI  - GENBANK/JN037219
SI  - GENBANK/JN037220
SI  - GENBANK/JN037221
SI  - GENBANK/JN037222
SI  - GENBANK/JN037223
SI  - GENBANK/JN037224
SI  - GENBANK/JN037225
SI  - GENBANK/JN037226
SI  - GENBANK/JN037227
SI  - GENBANK/JN037228
SI  - GENBANK/JN037229
SI  - GENBANK/JN037230
SI  - GENBANK/JN037231
SI  - GENBANK/JN037232
SI  - GENBANK/JN037233
SI  - GENBANK/JN037234
SI  - GENBANK/JN037235
SI  - GENBANK/JN037236
SI  - GENBANK/JN037237
SI  - GENBANK/JN037238
SI  - GENBANK/JN037239
SI  - GENBANK/JN037240
SI  - GENBANK/JN037241
SI  - GENBANK/JN037242
SI  - GENBANK/JN037243
SI  - GENBANK/JN037244
SI  - GENBANK/JN037245
SI  - GENBANK/JN037246
SI  - GENBANK/JN037247
SI  - GENBANK/JN037248
SI  - GENBANK/JN037249
SI  - GENBANK/JN037250
SI  - GENBANK/JN037251
SI  - GENBANK/JN037252
SI  - GENBANK/JN037253
SI  - GENBANK/JN037254
SI  - GENBANK/JN037255
SI  - GENBANK/JN037256
SI  - GENBANK/JN037257
SI  - GENBANK/JN037258
SI  - GENBANK/JN037259
SI  - GENBANK/JN037260
SI  - GENBANK/JN037261
SI  - GENBANK/JN037262
SI  - GENBANK/JN037263
SI  - GENBANK/JN037264
SI  - GENBANK/JN037265
SI  - GENBANK/JN037266
SI  - GENBANK/JN037267
SI  - GENBANK/JN037268
SI  - GENBANK/JN037269
SI  - GENBANK/JN037270
SI  - GENBANK/JN037271
SI  - GENBANK/JN037272
SI  - GENBANK/JN037273
SI  - GENBANK/JN037274
SI  - GENBANK/JN037275
SI  - GENBANK/JN037276
SI  - GENBANK/JN037277
SI  - GENBANK/JN037278
SI  - GENBANK/JN037279
SI  - GENBANK/JN037280
SI  - GENBANK/JN037281
SI  - GENBANK/JN037282
SI  - GENBANK/JN037283
SI  - GENBANK/JN037284
SI  - GENBANK/JN037285
SI  - GENBANK/JN037286
SI  - GENBANK/JN037287
SI  - GENBANK/JN037288
SI  - GENBANK/JN037289
SI  - GENBANK/JN037290
SI  - GENBANK/JN037291
SI  - GENBANK/JN037292
SI  - GENBANK/JN037293
SI  - GENBANK/JN037294
SI  - GENBANK/JN037295
SI  - GENBANK/JN037296
SI  - GENBANK/JN037297
SI  - GENBANK/JN037298
SI  - GENBANK/JN037299
SI  - GENBANK/JN037300
SI  - GENBANK/JN037301
SI  - GENBANK/JN037302
SI  - GENBANK/JN037303
SI  - GENBANK/JN037304
SI  - GENBANK/JN037305
SI  - GENBANK/JN037306
SI  - GENBANK/JN037307
SI  - GENBANK/JN037308
SI  - GENBANK/JN037309
SI  - GENBANK/JN037310
SI  - GENBANK/JN037311
SI  - GENBANK/JN037312
SI  - GENBANK/JN037313
SI  - GENBANK/JN037314
SI  - GENBANK/JN037315
SI  - GENBANK/JN037316
SI  - GENBANK/JN037317
SI  - GENBANK/JN037318
SI  - GENBANK/JN037319
SI  - GENBANK/JN037320
SI  - GENBANK/JN037321
SI  - GENBANK/JN037322
SI  - GENBANK/JN037323
SI  - GENBANK/JN037324
SI  - GENBANK/JN037325
SI  - GENBANK/JN037326
SI  - GENBANK/JN037327
SI  - GENBANK/JN037328
SI  - GENBANK/JN037329
SI  - GENBANK/JN037330
SI  - GENBANK/JN037331
SI  - GENBANK/JN037332
SI  - GENBANK/JN037333
SI  - GENBANK/JN037334
SI  - GENBANK/JN037335
SI  - GENBANK/JN037336
SI  - GENBANK/JN037337
SI  - GENBANK/JN037338
SI  - GENBANK/JN037339
SI  - GENBANK/JN037340
SI  - GENBANK/JN037341
SI  - GENBANK/JN037342
SI  - GENBANK/JN037343
SI  - GENBANK/JN037344
SI  - GENBANK/JN037345
SI  - GENBANK/JN037346
SI  - GENBANK/JN037347
SI  - GENBANK/JN037348
SI  - GENBANK/JN037349
SI  - GENBANK/JN037350
SI  - GENBANK/JN037351
SI  - GENBANK/JN037352
SI  - GENBANK/JN037353
SI  - GENBANK/JN037354
SI  - GENBANK/JN037355
SI  - GENBANK/JN037356
SI  - GENBANK/JN037357
SI  - GENBANK/JN037358
SI  - GENBANK/JN037359
SI  - GENBANK/JN037360
SI  - GENBANK/JN037361
SI  - GENBANK/JN037362
SI  - GENBANK/JN037363
SI  - GENBANK/JN037364
SI  - GENBANK/JN037365
SI  - GENBANK/JN037366
SI  - GENBANK/JN037367
SI  - GENBANK/JN037368
SI  - GENBANK/JN037369
SI  - GENBANK/JN037370
SI  - GENBANK/JN037371
SI  - GENBANK/JN037372
SI  - GENBANK/JN037373
SI  - GENBANK/JN037374
SI  - GENBANK/JN037375
SI  - GENBANK/JN037376
SI  - GENBANK/JN037377
SI  - GENBANK/JN037378
SI  - GENBANK/JN037379
SI  - GENBANK/JN037380
SI  - GENBANK/JN037381
SI  - GENBANK/JN037382
SI  - GENBANK/JN037383
SI  - GENBANK/JN037384
SI  - GENBANK/JN037385
SI  - GENBANK/JN037386
SI  - GENBANK/JN037387
SI  - GENBANK/JN037388
SI  - GENBANK/JN037389
SI  - GENBANK/JN037390
SI  - GENBANK/JN037391
SI  - GENBANK/JN037392
SI  - GENBANK/JN037393
SI  - GENBANK/JN037394
SI  - GENBANK/JN037395
SI  - GENBANK/JN037396
SI  - GENBANK/JN037397
SI  - GENBANK/JN037398
SI  - GENBANK/JN037399
SI  - GENBANK/JN037400
SI  - GENBANK/JN037401
SI  - GENBANK/JN037402
SI  - GENBANK/JN037403
SI  - GENBANK/JN037404
SI  - GENBANK/JN037405
SI  - GENBANK/JN037406
SI  - GENBANK/JN037407
SI  - GENBANK/JN037408
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20111226
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Computer Simulation
MH  - DNA, Mitochondrial/*genetics
MH  - *Gene Flow
MH  - Genetic Speciation
MH  - Genome, Mitochondrial
MH  - Hares/*classification/*genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
EDAT- 2011/12/28 06:00
MHDA- 2013/04/02 06:00
CRDT- 2011/12/28 06:00
PHST- 2011/12/28 06:00 [entrez]
PHST- 2011/12/28 06:00 [pubmed]
PHST- 2013/04/02 06:00 [medline]
AID - syr114 [pii]
AID - 10.1093/sysbio/syr114 [doi]
PST - ppublish
SO  - Syst Biol. 2012 May;61(3):367-81. doi: 10.1093/sysbio/syr114. Epub 2011 Dec 26.

PMID- 22201158
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20120216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - Testing the impact of calibration on molecular divergence times using a
      fossil-rich group: the case of Nothofagus (Fagales).
PG  - 289-313
LID - 10.1093/sysbio/syr116 [doi]
AB  - Although temporal calibration is widely recognized as critical for obtaining
      accurate divergence-time estimates using molecular dating methods, few studies
      have evaluated the variation resulting from different calibration strategies.
      Depending on the information available, researchers have often used primary
      calibrations from the fossil record or secondary calibrations from previous
      molecular dating studies. In analyses of flowering plants, primary calibration
      data can be obtained from macro- and mesofossils (e.g., leaves, flowers, and
      fruits) or microfossils (e.g., pollen). Fossil data can vary substantially in
      accuracy and precision, presenting a difficult choice when selecting appropriate 
      calibrations. Here, we test the impact of eight plausible calibration scenarios
      for Nothofagus (Nothofagaceae, Fagales), a plant genus with a particularly rich
      and well-studied fossil record. To do so, we reviewed the phylogenetic placement 
      and geochronology of 38 fossil taxa of Nothofagus and other Fagales, and we
      identified minimum age constraints for up to 18 nodes of the phylogeny of
      Fagales. Molecular dating analyses were conducted for each scenario using maximum
      likelihood (RAxML + r8s) and Bayesian (BEAST) approaches on sequence data from
      six regions of the chloroplast and nuclear genomes. Using either ingroup or
      outgroup constraints, or both, led to similar age estimates, except near strongly
      influential calibration nodes. Using "early but risky" fossil constraints in
      addition to "safe but late" constraints, or using assumptions of vicariance
      instead of fossil constraints, led to older age estimates. In contrast, using
      secondary calibration points yielded drastically younger age estimates. This
      empirical study highlights the critical influence of calibration on molecular
      dating analyses. Even in a best-case situation, with many thoroughly vetted
      fossils available, substantial uncertainties can remain in the estimates of
      divergence times. For example, our estimates for the crown group age of
      Nothofagus varied from 13 to 113 Ma across our full range of calibration
      scenarios. We suggest that increased background research should be made at all
      stages of the calibration process to reduce errors wherever possible, from
      verifying the geochronological data on the fossils to critical reassessment of
      their phylogenetic position.
FAU - Sauquet, Herve
AU  - Sauquet H
AD  - Laboratoire Ecologie, Systematique, Evolution, Universite Paris-Sud, CNRS UMR
      8079, 91405 Orsay, France. herve.sauquet@u-psud.fr
FAU - Ho, Simon Y W
AU  - Ho SY
FAU - Gandolfo, Maria A
AU  - Gandolfo MA
FAU - Jordan, Gregory J
AU  - Jordan GJ
FAU - Wilf, Peter
AU  - Wilf P
FAU - Cantrill, David J
AU  - Cantrill DJ
FAU - Bayly, Michael J
AU  - Bayly MJ
FAU - Bromham, Lindell
AU  - Bromham L
FAU - Brown, Gillian K
AU  - Brown GK
FAU - Carpenter, Raymond J
AU  - Carpenter RJ
FAU - Lee, Daphne M
AU  - Lee DM
FAU - Murphy, Daniel J
AU  - Murphy DJ
FAU - Sniderman, J M Kale
AU  - Sniderman JM
FAU - Udovicic, Frank
AU  - Udovicic F
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20111226
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Biodiversity
MH  - Calibration
MH  - Classification/methods
MH  - DNA, Plant/chemistry
MH  - Fagus/classification/*genetics
MH  - *Fossils
MH  - Genetic Variation
MH  - Phylogeny
MH  - Sequence Alignment
MH  - Time Factors
EDAT- 2011/12/28 06:00
MHDA- 2012/06/26 06:00
CRDT- 2011/12/28 06:00
PHST- 2011/12/28 06:00 [entrez]
PHST- 2011/12/28 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr116 [pii]
AID - 10.1093/sysbio/syr116 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):289-313. doi: 10.1093/sysbio/syr116. Epub 2011 Dec 26.

PMID- 22199008
OWN - NLM
STAT- MEDLINE
DCOM- 20121018
LR  - 20120618
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 4
DP  - 2012 Jul
TI  - A new global palaeobiogeographical model for the late Mesozoic and early
      Tertiary.
PG  - 553-66
LID - 10.1093/sysbio/syr115 [doi]
AB  - Late Mesozoic palaeobiogeography has been characterized by a distinction between 
      the northern territories of Laurasia and the southern landmasses of Gondwana. The
      repeated discovery of Gondwanan lineages in Laurasia has led to the proposal of
      alternative scenarios to explain these anomalous occurrences. A new
      biogeographical model for late Mesozoic terrestrial ecosystems is here proposed
      in which Europe and "Gondwanan" territories possessed a common Eurogondwanan
      fauna during the earliest Cretaceous. Subsequently, following the Hauterivian,
      the European territories severed from Africa and then connected to Asiamerica
      resulting in a faunal interchange. This model explains the presence of Gondwanan 
      taxa in Laurasia and the absence of Laurasian forms in the southern territories
      during the Cretaceous. In order to test this new palaeobiogeographical model,
      tree reconciliation analyses (TRAs) were performed based on biogeographical
      signals provided by a supertree of late Mesozoic archosaurs. The TRAs found
      significant evidence for the presence of an earliest Cretaceous Eurogondwanan
      fauna followed by a relatively short-term Gondwana-Laurasia dichotomy. The
      analysis recovered evidence for a biogeographical reconnection of the European
      territories with Africa and South America-Antarctica during the Campanian to
      Maastrichtian time-slice. This biogeographical scenario appears to continue
      through the early Tertiary and sheds light on the trans-Atlantic disjunct
      distributions of several extant plant and animal groups.
FAU - Ezcurra, Martin D
AU  - Ezcurra MD
AD  - Laboratorio de Anatomia Comparada y Evolucion de los Vertebrados, Museo Argentino
      de Ciencias Naturales B. Rivadavia, Angel Gallardo 470 C1405DJR, Buenos Aires,
      Argentina. martindezcurra@yahoo.com.ar
FAU - Agnolin, Federico L
AU  - Agnolin FL
LA  - eng
PT  - Journal Article
DEP - 20111223
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Dinosaurs/*classification/genetics
MH  - *Fossils
MH  - Geography
MH  - Models, Biological
MH  - Paleontology
MH  - Reptiles/*classification/genetics
EDAT- 2011/12/27 06:00
MHDA- 2012/10/19 06:00
CRDT- 2011/12/27 06:00
PHST- 2011/12/27 06:00 [entrez]
PHST- 2011/12/27 06:00 [pubmed]
PHST- 2012/10/19 06:00 [medline]
AID - syr115 [pii]
AID - 10.1093/sysbio/syr115 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jul;61(4):553-66. doi: 10.1093/sysbio/syr115. Epub 2011 Dec 23.

PMID- 22139466
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20111222
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - SATe-II: very fast and accurate simultaneous estimation of multiple sequence
      alignments and phylogenetic trees.
PG  - 90-106
LID - 10.1093/sysbio/syr095 [doi]
AB  - Highly accurate estimation of phylogenetic trees for large data sets is
      difficult, in part because multiple sequence alignments must be accurate for
      phylogeny estimation methods to be accurate. Coestimation of alignments and trees
      has been attempted but currently only SATe estimates reasonably accurate trees
      and alignments for large data sets in practical time frames (Liu K., Raghavan S.,
      Nelesen S., Linder C.R., Warnow T. 2009b. Rapid and accurate large-scale
      coestimation of sequence alignments and phylogenetic trees. Science.
      324:1561-1564). Here, we present a modification to the original SATe algorithm
      that improves upon SATe (which we now call SATe-I) in terms of speed and of
      phylogenetic and alignment accuracy. SATe-II uses a different divide-and-conquer 
      strategy than SATe-I and so produces smaller more closely related subsets than
      SATe-I; as a result, SATe-II produces more accurate alignments and trees, can
      analyze larger data sets, and runs more efficiently than SATe-I. Generally, SATe 
      is a metamethod that takes an existing multiple sequence alignment method as an
      input parameter and boosts the quality of that alignment method. SATe-II-boosted 
      alignment methods are significantly more accurate than their unboosted versions, 
      and trees based upon these improved alignments are more accurate than trees based
      upon the original alignments. Because SATe-I used maximum likelihood (ML) methods
      that treat gaps as missing data to estimate trees and because we found a
      correlation between the quality of tree/alignment pairs and ML scores, we
      explored the degree to which SATe's performance depends on using ML with gaps
      treated as missing data to determine the best tree/alignment pair. We present two
      lines of evidence that using ML with gaps treated as missing data to optimize the
      alignment and tree produces very poor results. First, we show that the
      optimization problem where a set of unaligned DNA sequences is given and the
      output is the tree and alignment of those sequences that maximize likelihood
      under the Jukes-Cantor model is uninformative in the worst possible sense. For
      all inputs, all trees optimize the likelihood score. Second, we show that a
      greedy heuristic that uses GTR+Gamma ML to optimize the alignment and the tree
      can produce very poor alignments and trees. Therefore, the excellent performance 
      of SATe-II and SATe-I is not because ML is used as an optimization criterion for 
      choosing the best tree/alignment pair but rather due to the particular
      divide-and-conquer realignment techniques employed.
FAU - Liu, Kevin
AU  - Liu K
AD  - Department of Computer Science, University of Texas at Austin, Austin, TX 78712, 
      USA.
FAU - Warnow, Tandy J
AU  - Warnow TJ
FAU - Holder, Mark T
AU  - Holder MT
FAU - Nelesen, Serita M
AU  - Nelesen SM
FAU - Yu, Jiaye
AU  - Yu J
FAU - Stamatakis, Alexandros P
AU  - Stamatakis AP
FAU - Linder, C Randal
AU  - Linder CR
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20111201
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Algorithms
MH  - Automation
MH  - Computer Simulation
MH  - DNA
MH  - Evolution, Molecular
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - Sequence Alignment/*methods
MH  - *Software
EDAT- 2011/12/06 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/12/06 06:00
PHST- 2011/12/06 06:00 [entrez]
PHST- 2011/12/06 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr095 [pii]
AID - 10.1093/sysbio/syr095 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):90-106. doi: 10.1093/sysbio/syr095. Epub 2011 Dec 1.

PMID- 22105867
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - Best practices for justifying fossil calibrations.
PG  - 346-59
LID - 10.1093/sysbio/syr107 [doi]
FAU - Parham, James F
AU  - Parham JF
AD  - Alabama Museum of Natural History, University of Alabama, 427 6th Avenue, Smith
      Hall, Box 870340, Tuscaloosa, AL 35487, USA. jparham@csub.edu
FAU - Donoghue, Philip C J
AU  - Donoghue PC
FAU - Bell, Christopher J
AU  - Bell CJ
FAU - Calway, Tyler D
AU  - Calway TD
FAU - Head, Jason J
AU  - Head JJ
FAU - Holroyd, Patricia A
AU  - Holroyd PA
FAU - Inoue, Jun G
AU  - Inoue JG
FAU - Irmis, Randall B
AU  - Irmis RB
FAU - Joyce, Walter G
AU  - Joyce WG
FAU - Ksepka, Daniel T
AU  - Ksepka DT
FAU - Patane, Jose S L
AU  - Patane JS
FAU - Smith, Nathan D
AU  - Smith ND
FAU - Tarver, James E
AU  - Tarver JE
FAU - van Tuinen, Marcel
AU  - van Tuinen M
FAU - Yang, Ziheng
AU  - Yang Z
FAU - Angielczyk, Kenneth D
AU  - Angielczyk KD
FAU - Greenwood, Jenny M
AU  - Greenwood JM
FAU - Hipsley, Christy A
AU  - Hipsley CA
FAU - Jacobs, Louis
AU  - Jacobs L
FAU - Makovicky, Peter J
AU  - Makovicky PJ
FAU - Muller, Johannes
AU  - Muller J
FAU - Smith, Krister T
AU  - Smith KT
FAU - Theodor, Jessica M
AU  - Theodor JM
FAU - Warnock, Rachel C M
AU  - Warnock RC
FAU - Benton, Michael J
AU  - Benton MJ
LA  - eng
GR  - BB/G006660/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20111121
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Calibration
MH  - Classification/methods
MH  - *Fossils
MH  - *Phylogeny
PMC - PMC3280042
EDAT- 2011/11/23 06:00
MHDA- 2012/06/26 06:00
CRDT- 2011/11/23 06:00
PHST- 2011/11/23 06:00 [entrez]
PHST- 2011/11/23 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr107 [pii]
AID - 10.1093/sysbio/syr107 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):346-59. doi: 10.1093/sysbio/syr107. Epub 2011 Nov 21.

PMID- 22076302
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20120216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - Characterizing the phylogenetic tree-search problem.
PG  - 228-39
LID - 10.1093/sysbio/syr097 [doi]
AB  - Phylogenetic trees are important in many areas of biological research, ranging
      from systematic studies to the methods used for genome annotation. Finding the
      best scoring tree under any optimality criterion is an NP-hard problem, which
      necessitates the use of heuristics for tree-search. Although tree-search plays a 
      major role in obtaining a tree estimate, there remains a limited understanding of
      its characteristics and how the elements of the statistical inferential procedure
      interact with the algorithms used. This study begins to answer some of these
      questions through a detailed examination of maximum likelihood tree-search on a
      wide range of real genome-scale data sets. We examine all 10,395 trees for each
      of the 106 genes of an eight-taxa yeast phylogenomic data set, then apply
      different tree-search algorithms to investigate their performance. We extend our 
      findings by examining two larger genome-scale data sets and a large disparate
      data set that has been previously used to benchmark the performance of
      tree-search programs. We identify several broad trends occurring during
      tree-search that provide an insight into the performance of heuristics and may,
      in the future, aid their development. These trends include a tendency for the
      true maximum likelihood (best) tree to also be the shortest tree in terms of
      branch lengths, a weak tendency for tree-search to recover the best tree, and a
      tendency for tree-search to encounter fewer local optima in genes that have a
      high information content. When examining current heuristics for tree-search, we
      find that nearest-neighbor-interchange performs poorly, and frequently finds
      trees that are significantly different from the best tree. In contrast,
      subtree-pruning-and-regrafting tends to perform well, nearly always finding trees
      that are not significantly different to the best tree. Finally, we demonstrate
      that the precise implementation of a tree-search strategy, including when and
      where parameters are optimized, can change the character of tree-search, and that
      good strategies for tree-search may combine existing tree-search programs.
FAU - Money, Daniel
AU  - Money D
AD  - Faculty of Life Sciences, University of Manchester, Michael Smith Building,
      Oxford Road, Manchester M13 9PT, UK.
FAU - Whelan, Simon
AU  - Whelan S
LA  - eng
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20111110
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/methods
MH  - Computational Biology
MH  - Genome
MH  - Likelihood Functions
MH  - Models, Biological
MH  - *Phylogeny
MH  - Yeasts/classification/*genetics
EDAT- 2011/11/15 06:00
MHDA- 2012/06/26 06:00
CRDT- 2011/11/15 06:00
PHST- 2011/11/15 06:00 [entrez]
PHST- 2011/11/15 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr097 [pii]
AID - 10.1093/sysbio/syr097 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):228-39. doi: 10.1093/sysbio/syr097. Epub 2011 Nov 10.

PMID- 22076301
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20120216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - Accuracy and precision of species trees: effects of locus, individual, and base
      pair sampling on inference of species trees in lizards of the Liolaemus darwinii 
      group (Squamata, Liolaemidae).
PG  - 272-88
LID - 10.1093/sysbio/syr105 [doi]
AB  - Molecular phylogenetics has entered a new era in which species trees are
      estimated from a collection of gene trees using methods that accommodate their
      heterogeneity and discordance with the species tree. Empirical evaluation of
      species trees is necessary to assess the performance (i.e., accuracy and
      precision) of these methods with real data, which consists of gene genealogies
      likely shaped by different historical and demographic processes. We analyzed 20
      loci for 16 species of the South American lizards of the Liolaemus darwinii
      species group and reconstructed a species tree with *BEAST, then compared the
      performance of this method under different sampling strategies of loci,
      individuals, and sequence lengths. We found an increase in the accuracy and
      precision of species trees with the number of loci, but for any number of loci,
      accuracy substantially decreased only when using only one individual per species 
      or 25% of the full sequence length ( approximately 147 bp). In addition, locus
      "informativeness" was an important factor in the accuracy/precision of species
      trees when using a few loci, but it became increasingly irrelevant with
      additional loci. Our empirical results combined with the previous simulation
      studies suggest that there is an optimal range of sampling effort of loci,
      individuals, and sequence lengths for a given speciation history and information 
      content of the data. Future studies should be directed toward further assessment 
      of other factors that can impact performance of species trees, including gene
      flow, locus "informativeness," tree shape, missing data, and errors in species
      delimitation.
FAU - Camargo, Arley
AU  - Camargo A
AD  - Department of Biology and Bean Life Science Museum, Brigham Young University,
      Provo, UT 84602, USA. arley.camargo@gmail.com
FAU - Avila, Luciano J
AU  - Avila LJ
FAU - Morando, Mariana
AU  - Morando M
FAU - Sites, Jack W Jr
AU  - Sites JW Jr
LA  - eng
SI  - GENBANK/JN682514
SI  - GENBANK/JN682515
SI  - GENBANK/JN682516
SI  - GENBANK/JN682517
SI  - GENBANK/JN682518
SI  - GENBANK/JN682519
SI  - GENBANK/JN682520
SI  - GENBANK/JN682521
SI  - GENBANK/JN682522
SI  - GENBANK/JN682523
SI  - GENBANK/JN682524
SI  - GENBANK/JN682525
SI  - GENBANK/JN682526
SI  - GENBANK/JN682527
SI  - GENBANK/JN682528
SI  - GENBANK/JN682529
SI  - GENBANK/JN682530
SI  - GENBANK/JN682531
SI  - GENBANK/JN682532
SI  - GENBANK/JN682533
SI  - GENBANK/JN682534
SI  - GENBANK/JN682535
SI  - GENBANK/JN682536
SI  - GENBANK/JN682537
SI  - GENBANK/JN682538
SI  - GENBANK/JN682539
SI  - GENBANK/JN682540
SI  - GENBANK/JN682541
SI  - GENBANK/JN682542
SI  - GENBANK/JN682543
SI  - GENBANK/JN682544
SI  - GENBANK/JN682545
SI  - GENBANK/JN682546
SI  - GENBANK/JN682547
SI  - GENBANK/JN682548
SI  - GENBANK/JN682549
SI  - GENBANK/JN682550
SI  - GENBANK/JN682551
SI  - GENBANK/JN682552
SI  - GENBANK/JN682553
SI  - GENBANK/JN682554
SI  - GENBANK/JN682555
SI  - GENBANK/JN682556
SI  - GENBANK/JN682557
SI  - GENBANK/JN682558
SI  - GENBANK/JN682559
SI  - GENBANK/JN682560
SI  - GENBANK/JN682561
SI  - GENBANK/JN682562
SI  - GENBANK/JN682563
SI  - GENBANK/JN682564
SI  - GENBANK/JN682565
SI  - GENBANK/JN682566
SI  - GENBANK/JN682567
SI  - GENBANK/JN682568
SI  - GENBANK/JN682569
SI  - GENBANK/JN682570
SI  - GENBANK/JN682571
SI  - GENBANK/JN682572
SI  - GENBANK/JN682573
SI  - GENBANK/JN682574
SI  - GENBANK/JN682575
SI  - GENBANK/JN682576
SI  - GENBANK/JN682577
SI  - GENBANK/JN682578
SI  - GENBANK/JN682579
SI  - GENBANK/JN682580
SI  - GENBANK/JN682581
SI  - GENBANK/JN682582
SI  - GENBANK/JN682583
SI  - GENBANK/JN682584
SI  - GENBANK/JN682585
SI  - GENBANK/JN682586
SI  - GENBANK/JN682587
SI  - GENBANK/JN682588
SI  - GENBANK/JN682589
SI  - GENBANK/JN682590
SI  - GENBANK/JN682591
SI  - GENBANK/JN682592
SI  - GENBANK/JN682593
SI  - GENBANK/JN682594
SI  - GENBANK/JN682595
SI  - GENBANK/JN682596
SI  - GENBANK/JN682597
SI  - GENBANK/JN682598
SI  - GENBANK/JN682599
SI  - GENBANK/JN682600
SI  - GENBANK/JN682601
SI  - GENBANK/JN682602
SI  - GENBANK/JN682603
SI  - GENBANK/JN682604
SI  - GENBANK/JN682605
SI  - GENBANK/JN682606
SI  - GENBANK/JN682607
SI  - GENBANK/JN682608
SI  - GENBANK/JN682609
SI  - GENBANK/JN682610
SI  - GENBANK/JN682611
SI  - GENBANK/JN682612
SI  - GENBANK/JN682613
SI  - GENBANK/JN682614
SI  - GENBANK/JN682615
SI  - GENBANK/JN682616
SI  - GENBANK/JN682617
SI  - GENBANK/JN682618
SI  - GENBANK/JN682619
SI  - GENBANK/JN682620
SI  - GENBANK/JN682621
SI  - GENBANK/JN682622
SI  - GENBANK/JN682623
SI  - GENBANK/JN682624
SI  - GENBANK/JN682625
SI  - GENBANK/JN682626
SI  - GENBANK/JN682627
SI  - GENBANK/JN682628
SI  - GENBANK/JN682629
SI  - GENBANK/JN682630
SI  - GENBANK/JN682631
SI  - GENBANK/JN682632
SI  - GENBANK/JN682633
SI  - GENBANK/JN682634
SI  - GENBANK/JN682635
SI  - GENBANK/JN682636
SI  - GENBANK/JN682637
SI  - GENBANK/JN682638
SI  - GENBANK/JN682639
SI  - GENBANK/JN682640
SI  - GENBANK/JN682641
SI  - GENBANK/JN682642
SI  - GENBANK/JN682643
SI  - GENBANK/JN682644
SI  - GENBANK/JN682645
SI  - GENBANK/JN682646
SI  - GENBANK/JN682647
SI  - GENBANK/JN682648
SI  - GENBANK/JN682649
SI  - GENBANK/JN682650
SI  - GENBANK/JN682651
SI  - GENBANK/JN682652
SI  - GENBANK/JN682653
SI  - GENBANK/JN682654
SI  - GENBANK/JN682655
SI  - GENBANK/JN682656
SI  - GENBANK/JN682657
SI  - GENBANK/JN682658
SI  - GENBANK/JN682659
SI  - GENBANK/JN682660
SI  - GENBANK/JN682661
SI  - GENBANK/JN682662
SI  - GENBANK/JN682663
SI  - GENBANK/JN682664
SI  - GENBANK/JN682665
SI  - GENBANK/JN682666
SI  - GENBANK/JN682667
SI  - GENBANK/JN682668
SI  - GENBANK/JN682669
SI  - GENBANK/JN682670
SI  - GENBANK/JN682671
SI  - GENBANK/JN682672
SI  - GENBANK/JN682673
SI  - GENBANK/JN682674
SI  - GENBANK/JN682675
SI  - GENBANK/JN682676
SI  - GENBANK/JN682677
SI  - GENBANK/JN682678
SI  - GENBANK/JN682679
SI  - GENBANK/JN682680
SI  - GENBANK/JN682681
SI  - GENBANK/JN682682
SI  - GENBANK/JN682683
SI  - GENBANK/JN682684
SI  - GENBANK/JN682685
SI  - GENBANK/JN682686
SI  - GENBANK/JN682687
SI  - GENBANK/JN682688
SI  - GENBANK/JN682689
SI  - GENBANK/JN682690
SI  - GENBANK/JN682691
SI  - GENBANK/JN682692
SI  - GENBANK/JN682693
SI  - GENBANK/JN682694
SI  - GENBANK/JN682695
SI  - GENBANK/JN682696
SI  - GENBANK/JN682697
SI  - GENBANK/JN682698
SI  - GENBANK/JN682699
SI  - GENBANK/JN682700
SI  - GENBANK/JN682701
SI  - GENBANK/JN682702
SI  - GENBANK/JN682703
SI  - GENBANK/JN682704
SI  - GENBANK/JN682705
SI  - GENBANK/JN682706
SI  - GENBANK/JN682707
SI  - GENBANK/JN682708
SI  - GENBANK/JN682709
SI  - GENBANK/JN682710
SI  - GENBANK/JN682711
SI  - GENBANK/JN682712
SI  - GENBANK/JN682713
SI  - GENBANK/JN682714
SI  - GENBANK/JN682715
SI  - GENBANK/JN682716
SI  - GENBANK/JN682717
SI  - GENBANK/JN682718
SI  - GENBANK/JN682719
SI  - GENBANK/JN682720
SI  - GENBANK/JN682721
SI  - GENBANK/JN682722
SI  - GENBANK/JN682723
SI  - GENBANK/JN682724
SI  - GENBANK/JN682725
SI  - GENBANK/JN682726
SI  - GENBANK/JN682727
SI  - GENBANK/JN682728
SI  - GENBANK/JN682729
SI  - GENBANK/JN682730
SI  - GENBANK/JN682731
SI  - GENBANK/JN682732
SI  - GENBANK/JN682733
SI  - GENBANK/JN682734
SI  - GENBANK/JN682735
SI  - GENBANK/JN682736
SI  - GENBANK/JN682737
SI  - GENBANK/JN682738
SI  - GENBANK/JN682739
SI  - GENBANK/JN682740
SI  - GENBANK/JN682741
SI  - GENBANK/JN682742
SI  - GENBANK/JN682743
SI  - GENBANK/JN682744
SI  - GENBANK/JN682745
SI  - GENBANK/JN682746
SI  - GENBANK/JN682747
SI  - GENBANK/JN682748
SI  - GENBANK/JN682749
SI  - GENBANK/JN682750
SI  - GENBANK/JN682751
SI  - GENBANK/JN682752
SI  - GENBANK/JN682753
SI  - GENBANK/JN682754
SI  - GENBANK/JN682755
SI  - GENBANK/JN682756
SI  - GENBANK/JN682757
SI  - GENBANK/JN682758
SI  - GENBANK/JN682759
SI  - GENBANK/JN682760
SI  - GENBANK/JN682761
SI  - GENBANK/JN682762
SI  - GENBANK/JN682763
SI  - GENBANK/JN682764
SI  - GENBANK/JN682765
SI  - GENBANK/JN682766
SI  - GENBANK/JN682767
SI  - GENBANK/JN682768
SI  - GENBANK/JN682769
SI  - GENBANK/JN682770
SI  - GENBANK/JN682771
SI  - GENBANK/JN682772
SI  - GENBANK/JN682773
SI  - GENBANK/JN682774
SI  - GENBANK/JN682775
SI  - GENBANK/JN682776
SI  - GENBANK/JN682777
SI  - GENBANK/JN682778
SI  - GENBANK/JN682779
SI  - GENBANK/JN682780
SI  - GENBANK/JN682781
SI  - GENBANK/JN682782
SI  - GENBANK/JN682783
SI  - GENBANK/JN682784
SI  - GENBANK/JN682785
SI  - GENBANK/JN682786
SI  - GENBANK/JN682787
SI  - GENBANK/JN682788
SI  - GENBANK/JN682789
SI  - GENBANK/JN682790
SI  - GENBANK/JN682791
SI  - GENBANK/JN682792
SI  - GENBANK/JN682793
SI  - GENBANK/JN682794
SI  - GENBANK/JN682795
SI  - GENBANK/JN682796
SI  - GENBANK/JN682797
SI  - GENBANK/JN682798
SI  - GENBANK/JN682799
SI  - GENBANK/JN682800
SI  - GENBANK/JN682801
SI  - GENBANK/JN682802
SI  - GENBANK/JN682803
SI  - GENBANK/JN682804
SI  - GENBANK/JN682805
SI  - GENBANK/JN682806
SI  - GENBANK/JN682807
SI  - GENBANK/JN682808
SI  - GENBANK/JN682809
SI  - GENBANK/JN682810
SI  - GENBANK/JN682811
SI  - GENBANK/JN682812
SI  - GENBANK/JN682813
SI  - GENBANK/JN682814
SI  - GENBANK/JN682815
SI  - GENBANK/JN682816
SI  - GENBANK/JN682817
SI  - GENBANK/JN682818
SI  - GENBANK/JN682819
SI  - GENBANK/JN682820
SI  - GENBANK/JN682821
SI  - GENBANK/JN682822
SI  - GENBANK/JN682823
SI  - GENBANK/JN682824
SI  - GENBANK/JN682825
SI  - GENBANK/JN682826
SI  - GENBANK/JN682827
SI  - GENBANK/JN682828
SI  - GENBANK/JN682829
SI  - GENBANK/JN682830
SI  - GENBANK/JN682831
SI  - GENBANK/JN682832
SI  - GENBANK/JN682833
SI  - GENBANK/JN682834
SI  - GENBANK/JN682835
SI  - GENBANK/JN682836
SI  - GENBANK/JN682837
SI  - GENBANK/JN682838
SI  - GENBANK/JN682839
SI  - GENBANK/JN682840
SI  - GENBANK/JN682841
SI  - GENBANK/JN682842
SI  - GENBANK/JN682843
SI  - GENBANK/JN682844
SI  - GENBANK/JN682845
SI  - GENBANK/JN682846
SI  - GENBANK/JN682847
SI  - GENBANK/JN682848
SI  - GENBANK/JN682849
SI  - GENBANK/JN682850
SI  - GENBANK/JN682851
SI  - GENBANK/JN682852
SI  - GENBANK/JN682853
SI  - GENBANK/JN682854
SI  - GENBANK/JN682855
SI  - GENBANK/JN682856
SI  - GENBANK/JN682857
SI  - GENBANK/JN682858
SI  - GENBANK/JN682859
SI  - GENBANK/JN682860
SI  - GENBANK/JN682861
SI  - GENBANK/JN682862
SI  - GENBANK/JN682863
SI  - GENBANK/JN682864
SI  - GENBANK/JN682865
SI  - GENBANK/JN682866
SI  - GENBANK/JN682867
SI  - GENBANK/JN682868
SI  - GENBANK/JN682869
SI  - GENBANK/JN682870
SI  - GENBANK/JN682871
SI  - GENBANK/JN682872
SI  - GENBANK/JN682873
SI  - GENBANK/JN682874
SI  - GENBANK/JN682875
SI  - GENBANK/JN682876
SI  - GENBANK/JN682877
SI  - GENBANK/JN682878
SI  - GENBANK/JN682879
SI  - GENBANK/JN682880
SI  - GENBANK/JN682881
SI  - GENBANK/JN682882
SI  - GENBANK/JN682883
SI  - GENBANK/JN682884
SI  - GENBANK/JN682885
SI  - GENBANK/JN682886
SI  - GENBANK/JN682887
SI  - GENBANK/JN682888
SI  - GENBANK/JN682889
SI  - GENBANK/JN682890
SI  - GENBANK/JN682891
SI  - GENBANK/JN682892
SI  - GENBANK/JN682893
SI  - GENBANK/JN682894
SI  - GENBANK/JN682895
SI  - GENBANK/JN682896
SI  - GENBANK/JN682897
SI  - GENBANK/JN682898
SI  - GENBANK/JN682899
SI  - GENBANK/JN682900
SI  - GENBANK/JN682901
SI  - GENBANK/JN682902
SI  - GENBANK/JN682903
SI  - GENBANK/JN682904
SI  - GENBANK/JN682905
SI  - GENBANK/JN682906
SI  - GENBANK/JN682907
SI  - GENBANK/JN682908
SI  - GENBANK/JN682909
SI  - GENBANK/JN682910
SI  - GENBANK/JN682911
SI  - GENBANK/JN682912
SI  - GENBANK/JN682913
SI  - GENBANK/JN682914
SI  - GENBANK/JN682915
SI  - GENBANK/JN682916
SI  - GENBANK/JN682917
SI  - GENBANK/JN682918
SI  - GENBANK/JN682919
SI  - GENBANK/JN682920
SI  - GENBANK/JN682921
SI  - GENBANK/JN682922
SI  - GENBANK/JN682923
SI  - GENBANK/JN682924
SI  - GENBANK/JN682925
SI  - GENBANK/JN682926
SI  - GENBANK/JN682927
SI  - GENBANK/JN682928
SI  - GENBANK/JN682929
SI  - GENBANK/JN682930
SI  - GENBANK/JN682931
SI  - GENBANK/JN682932
SI  - GENBANK/JN682933
SI  - GENBANK/JN682934
SI  - GENBANK/JN682935
SI  - GENBANK/JN682936
SI  - GENBANK/JN682937
SI  - GENBANK/JN682938
SI  - GENBANK/JN682939
SI  - GENBANK/JN682940
SI  - GENBANK/JN682941
SI  - GENBANK/JN682942
SI  - GENBANK/JN682943
SI  - GENBANK/JN682944
SI  - GENBANK/JN682945
SI  - GENBANK/JN682946
SI  - GENBANK/JN682947
SI  - GENBANK/JN682948
SI  - GENBANK/JN682949
SI  - GENBANK/JN682950
SI  - GENBANK/JN682951
SI  - GENBANK/JN682952
SI  - GENBANK/JN682953
SI  - GENBANK/JN682954
SI  - GENBANK/JN682955
SI  - GENBANK/JN682956
SI  - GENBANK/JN682957
SI  - GENBANK/JN682958
SI  - GENBANK/JN682959
SI  - GENBANK/JN682960
SI  - GENBANK/JN682961
SI  - GENBANK/JN682962
SI  - GENBANK/JN682963
SI  - GENBANK/JN682964
SI  - GENBANK/JN682965
SI  - GENBANK/JN682966
SI  - GENBANK/JN682967
SI  - GENBANK/JN682968
SI  - GENBANK/JN682969
SI  - GENBANK/JN682970
SI  - GENBANK/JN682971
SI  - GENBANK/JN682972
SI  - GENBANK/JN682973
SI  - GENBANK/JN682974
SI  - GENBANK/JN682975
SI  - GENBANK/JN682976
SI  - GENBANK/JN682977
SI  - GENBANK/JN682978
SI  - GENBANK/JN682979
SI  - GENBANK/JN682980
SI  - GENBANK/JN682981
SI  - GENBANK/JN682982
SI  - GENBANK/JN682983
SI  - GENBANK/JN682984
SI  - GENBANK/JN682985
SI  - GENBANK/JN682986
SI  - GENBANK/JN682987
SI  - GENBANK/JN682988
SI  - GENBANK/JN682989
SI  - GENBANK/JN682990
SI  - GENBANK/JN682991
SI  - GENBANK/JN682992
SI  - GENBANK/JN682993
SI  - GENBANK/JN682994
SI  - GENBANK/JN682995
SI  - GENBANK/JN682996
SI  - GENBANK/JN682997
SI  - GENBANK/JN682998
SI  - GENBANK/JN682999
SI  - GENBANK/JN683000
SI  - GENBANK/JN683001
SI  - GENBANK/JN683002
SI  - GENBANK/JN683003
SI  - GENBANK/JN683004
SI  - GENBANK/JN683005
SI  - GENBANK/JN683006
SI  - GENBANK/JN683007
SI  - GENBANK/JN683008
SI  - GENBANK/JN683009
SI  - GENBANK/JN683010
SI  - GENBANK/JN683011
SI  - GENBANK/JN683012
SI  - GENBANK/JN683013
SI  - GENBANK/JN683014
SI  - GENBANK/JN683015
SI  - GENBANK/JN683016
SI  - GENBANK/JN683017
SI  - GENBANK/JN683018
SI  - GENBANK/JN683019
SI  - GENBANK/JN683020
SI  - GENBANK/JN683021
SI  - GENBANK/JN683022
SI  - GENBANK/JN683023
SI  - GENBANK/JN683024
SI  - GENBANK/JN683025
SI  - GENBANK/JN683026
SI  - GENBANK/JN683027
SI  - GENBANK/JN683028
SI  - GENBANK/JN683029
SI  - GENBANK/JN683030
SI  - GENBANK/JN683031
SI  - GENBANK/JN683032
SI  - GENBANK/JN683033
SI  - GENBANK/JN683034
SI  - GENBANK/JN683035
SI  - GENBANK/JN683036
SI  - GENBANK/JN683037
SI  - GENBANK/JN683038
SI  - GENBANK/JN683039
SI  - GENBANK/JN683040
SI  - GENBANK/JN683041
SI  - GENBANK/JN683042
SI  - GENBANK/JN683043
SI  - GENBANK/JN683044
SI  - GENBANK/JN683045
SI  - GENBANK/JN683046
SI  - GENBANK/JN683047
SI  - GENBANK/JN683048
SI  - GENBANK/JN683049
SI  - GENBANK/JN683050
SI  - GENBANK/JN683051
SI  - GENBANK/JN683052
SI  - GENBANK/JN683053
SI  - GENBANK/JN683054
SI  - GENBANK/JN683055
SI  - GENBANK/JN683056
SI  - GENBANK/JN683057
SI  - GENBANK/JN683058
SI  - GENBANK/JN683059
SI  - GENBANK/JN683060
SI  - GENBANK/JN683061
SI  - GENBANK/JN683062
SI  - GENBANK/JN683063
SI  - GENBANK/JN683064
SI  - GENBANK/JN683065
SI  - GENBANK/JN683066
SI  - GENBANK/JN683067
SI  - GENBANK/JN683068
SI  - GENBANK/JN683069
SI  - GENBANK/JN683070
SI  - GENBANK/JN683071
SI  - GENBANK/JN683072
SI  - GENBANK/JN683073
SI  - GENBANK/JN683074
SI  - GENBANK/JN683075
SI  - GENBANK/JN683076
SI  - GENBANK/JN683077
SI  - GENBANK/JN683078
SI  - GENBANK/JN683079
SI  - GENBANK/JN683080
SI  - GENBANK/JN683081
SI  - GENBANK/JN683082
SI  - GENBANK/JN683083
SI  - GENBANK/JN683084
SI  - GENBANK/JN683085
SI  - GENBANK/JN683086
SI  - GENBANK/JN683087
SI  - GENBANK/JN683088
SI  - GENBANK/JN683089
SI  - GENBANK/JN683090
SI  - GENBANK/JN683091
SI  - GENBANK/JN683092
SI  - GENBANK/JN683093
SI  - GENBANK/JN683094
SI  - GENBANK/JN683095
SI  - GENBANK/JN683096
SI  - GENBANK/JN683097
SI  - GENBANK/JN683098
SI  - GENBANK/JN683099
SI  - GENBANK/JN683100
SI  - GENBANK/JN683101
SI  - GENBANK/JN683102
SI  - GENBANK/JN683103
SI  - GENBANK/JN683104
SI  - GENBANK/JN683105
SI  - GENBANK/JN683106
SI  - GENBANK/JN683107
SI  - GENBANK/JN683108
SI  - GENBANK/JN683109
SI  - GENBANK/JN683110
SI  - GENBANK/JN683111
SI  - GENBANK/JN683112
SI  - GENBANK/JN683113
SI  - GENBANK/JN683114
SI  - GENBANK/JN683115
SI  - GENBANK/JN683116
SI  - GENBANK/JN683117
SI  - GENBANK/JN683118
SI  - GENBANK/JN683119
SI  - GENBANK/JN683120
SI  - GENBANK/JN683121
SI  - GENBANK/JN683122
SI  - GENBANK/JN683123
SI  - GENBANK/JN683124
SI  - GENBANK/JN683125
SI  - GENBANK/JN683126
SI  - GENBANK/JN683127
SI  - GENBANK/JN683128
SI  - GENBANK/JN683129
SI  - GENBANK/JN683130
SI  - GENBANK/JN683131
SI  - GENBANK/JN683132
SI  - GENBANK/JN683133
SI  - GENBANK/JN683134
SI  - GENBANK/JN683135
SI  - GENBANK/JN683136
SI  - GENBANK/JN683137
SI  - GENBANK/JN683138
SI  - GENBANK/JN683139
SI  - GENBANK/JN683140
SI  - GENBANK/JN683141
SI  - GENBANK/JN683142
SI  - GENBANK/JN683143
SI  - GENBANK/JN683144
SI  - GENBANK/JN683145
SI  - GENBANK/JN683146
SI  - GENBANK/JN683147
SI  - GENBANK/JN683148
SI  - GENBANK/JN683149
SI  - GENBANK/JN683150
SI  - GENBANK/JN683151
SI  - GENBANK/JN683152
SI  - GENBANK/JN683153
SI  - GENBANK/JN683154
SI  - GENBANK/JN683155
SI  - GENBANK/JN683156
SI  - GENBANK/JN683157
SI  - GENBANK/JN683158
SI  - GENBANK/JN683159
SI  - GENBANK/JN683160
SI  - GENBANK/JN683161
SI  - GENBANK/JN683162
SI  - GENBANK/JN683163
SI  - GENBANK/JN683164
SI  - GENBANK/JN683165
SI  - GENBANK/JN683166
SI  - GENBANK/JN683167
SI  - GENBANK/JN683168
SI  - GENBANK/JN683169
SI  - GENBANK/JN683170
SI  - GENBANK/JN683171
SI  - GENBANK/JN683172
SI  - GENBANK/JN683173
SI  - GENBANK/JN683174
SI  - GENBANK/JN683175
SI  - GENBANK/JN683176
SI  - GENBANK/JN683177
SI  - GENBANK/JN683178
SI  - GENBANK/JN683179
SI  - GENBANK/JN683180
SI  - GENBANK/JN683181
SI  - GENBANK/JN683182
SI  - GENBANK/JN683183
SI  - GENBANK/JN683184
SI  - GENBANK/JN683185
SI  - GENBANK/JN683186
SI  - GENBANK/JN683187
SI  - GENBANK/JN683188
SI  - GENBANK/JN683189
SI  - GENBANK/JN683190
SI  - GENBANK/JN683191
SI  - GENBANK/JN683192
SI  - GENBANK/JN683193
SI  - GENBANK/JN683194
SI  - GENBANK/JN683195
SI  - GENBANK/JN683196
SI  - GENBANK/JN683197
SI  - GENBANK/JN683198
SI  - GENBANK/JN683199
SI  - GENBANK/JN683200
SI  - GENBANK/JN683201
SI  - GENBANK/JN683202
SI  - GENBANK/JN683203
SI  - GENBANK/JN683204
SI  - GENBANK/JN683205
SI  - GENBANK/JN683206
SI  - GENBANK/JN683207
SI  - GENBANK/JN683208
SI  - GENBANK/JN683209
SI  - GENBANK/JN683210
SI  - GENBANK/JN683211
SI  - GENBANK/JN683212
SI  - GENBANK/JN683213
SI  - GENBANK/JN683214
SI  - GENBANK/JN683215
SI  - GENBANK/JN683216
SI  - GENBANK/JN683217
SI  - GENBANK/JN683218
SI  - GENBANK/JN683219
SI  - GENBANK/JN683220
SI  - GENBANK/JN683221
SI  - GENBANK/JN683222
SI  - GENBANK/JN683223
SI  - GENBANK/JN683224
SI  - GENBANK/JN683225
SI  - GENBANK/JN683226
SI  - GENBANK/JN683227
SI  - GENBANK/JN683228
SI  - GENBANK/JN683229
SI  - GENBANK/JN683230
SI  - GENBANK/JN683231
SI  - GENBANK/JN683232
SI  - GENBANK/JN683233
SI  - GENBANK/JN683234
SI  - GENBANK/JN683235
SI  - GENBANK/JN683236
SI  - GENBANK/JN683237
SI  - GENBANK/JN683238
SI  - GENBANK/JN683239
SI  - GENBANK/JN683240
SI  - GENBANK/JN683241
SI  - GENBANK/JN683242
SI  - GENBANK/JN683243
SI  - GENBANK/JN683244
SI  - GENBANK/JN683245
SI  - GENBANK/JN683246
SI  - GENBANK/JN683247
SI  - GENBANK/JN683248
SI  - GENBANK/JN683249
SI  - GENBANK/JN683250
SI  - GENBANK/JN683251
SI  - GENBANK/JN683252
SI  - GENBANK/JN683253
SI  - GENBANK/JN683254
SI  - GENBANK/JN683255
SI  - GENBANK/JN683256
SI  - GENBANK/JN683257
SI  - GENBANK/JN683258
SI  - GENBANK/JN683259
SI  - GENBANK/JN683260
SI  - GENBANK/JN683261
SI  - GENBANK/JN683262
SI  - GENBANK/JN683263
SI  - GENBANK/JN683264
SI  - GENBANK/JN683265
SI  - GENBANK/JN683266
SI  - GENBANK/JN683267
SI  - GENBANK/JN683268
SI  - GENBANK/JN683269
SI  - GENBANK/JN683270
SI  - GENBANK/JN683271
SI  - GENBANK/JN683272
SI  - GENBANK/JN683273
SI  - GENBANK/JN683274
SI  - GENBANK/JN683275
SI  - GENBANK/JN683276
SI  - GENBANK/JN683277
SI  - GENBANK/JN683278
SI  - GENBANK/JN683279
SI  - GENBANK/JN683280
SI  - GENBANK/JN683281
SI  - GENBANK/JN683282
SI  - GENBANK/JN683283
SI  - GENBANK/JN683284
SI  - GENBANK/JN683285
SI  - GENBANK/JN683286
SI  - GENBANK/JN683287
SI  - GENBANK/JN683288
SI  - GENBANK/JN683289
SI  - GENBANK/JN683290
SI  - GENBANK/JN683291
SI  - GENBANK/JN683292
SI  - GENBANK/JN683293
SI  - GENBANK/JN683294
SI  - GENBANK/JN683295
SI  - GENBANK/JN683296
SI  - GENBANK/JN683297
SI  - GENBANK/JN683298
SI  - GENBANK/JN683299
SI  - GENBANK/JN683300
SI  - GENBANK/JN683301
SI  - GENBANK/JN683302
SI  - GENBANK/JN683303
SI  - GENBANK/JN683304
SI  - GENBANK/JN683305
SI  - GENBANK/JN683306
SI  - GENBANK/JN683307
SI  - GENBANK/JN683308
SI  - GENBANK/JN683309
SI  - GENBANK/JN683310
SI  - GENBANK/JN683311
SI  - GENBANK/JN683312
SI  - GENBANK/JN683313
SI  - GENBANK/JN683314
SI  - GENBANK/JN683315
SI  - GENBANK/JN683316
SI  - GENBANK/JN683317
SI  - GENBANK/JN683318
SI  - GENBANK/JN683319
SI  - GENBANK/JN683320
SI  - GENBANK/JN683321
SI  - GENBANK/JN683322
SI  - GENBANK/JN683323
SI  - GENBANK/JN683324
SI  - GENBANK/JN683325
SI  - GENBANK/JN683326
SI  - GENBANK/JN683327
SI  - GENBANK/JN683328
SI  - GENBANK/JN683329
SI  - GENBANK/JN683330
SI  - GENBANK/JN683331
SI  - GENBANK/JN683332
SI  - GENBANK/JN683333
SI  - GENBANK/JN683334
SI  - GENBANK/JN683335
SI  - GENBANK/JN683336
SI  - GENBANK/JN683337
SI  - GENBANK/JN683338
SI  - GENBANK/JN683339
SI  - GENBANK/JN683340
SI  - GENBANK/JN683341
SI  - GENBANK/JN683342
SI  - GENBANK/JN683343
SI  - GENBANK/JN683344
SI  - GENBANK/JN683345
SI  - GENBANK/JN683346
SI  - GENBANK/JN683347
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20111110
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Classification/methods
MH  - DNA/chemistry
MH  - Lizards/classification/*genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2011/11/15 06:00
MHDA- 2012/06/26 06:00
CRDT- 2011/11/15 06:00
PHST- 2011/11/15 06:00 [entrez]
PHST- 2011/11/15 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr105 [pii]
AID - 10.1093/sysbio/syr105 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):272-88. doi: 10.1093/sysbio/syr105. Epub 2011 Nov 10.

PMID- 21978990
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20120216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - A species tree for the Australo-Papuan Fairy-wrens and allies (Aves: Maluridae).
PG  - 253-71
LID - 10.1093/sysbio/syr101 [doi]
AB  - We explored the efficacy of species tree methods at the family level in birds,
      using the Australo-Papuan Fairy-wrens (Passeriformes: Maluridae) as a model
      system. Fairy-wrens of the genus Malurus are known for high intensities of sexual
      selection, resulting in some cases in rapid speciation. This history suggests
      that incomplete lineage sorting (ILS) of neutrally evolving loci could be
      substantial, a situation that could compromise traditional methods of combining
      loci in phylogenetic analysis. Using 18 molecular markers (5 anonymous loci, 7
      exons, 5 introns, and 1 mitochondrial DNA locus), we show that gene tree
      monophyly across species could be rejected for 16 of 18 loci, suggesting
      substantial ILS at the family level in these birds. Using the software
      Concaterpillar, we also detect three statistically distinct clusters of gene
      trees among the 18 loci. Despite substantial variation in gene trees, species
      trees constructed using four different species tree estimation methods (BEST,
      BUCKy, and STAR) were generally well supported and similar to each other and to
      the concatenation tree, with a few mild discordances at nodes that could be
      explained by rapid and recent speciation events. By contrast, minimizing deep
      coalescences produced a species tree that was topologically more divergent from
      those of the other methods as measured by multidimensional scaling of trees.
      Additionally, gene and species trees were topologically more similar in the BEST 
      analysis, presumably because of the species tree prior employed in BEST which
      appropriately assumes that gene trees are correlated with each other and with the
      species tree. Among the 18 loci, we also discovered 102 independent indel
      markers, which also proved phylogenetically informative, primarily among genera, 
      and displayed a approximately 4-fold bias towards deletions. As suggested in
      earlier work, the grasswrens (Amytornis) are sister to the rest of the family and
      the emu-wrens (Stipiturus) are sister to fairy-wrens (Malurus, Clytomyias). Our
      study shows that ILS is common at the family level in birds yet, despite this,
      species tree methods converge on broadly similar results for this family.
FAU - Lee, June Y
AU  - Lee JY
AD  - Museum of Comparative Zoology and Department of Organismic &amp; Evolutionary
      Biology, Harvard University, 26 Oxford Street, Cambridge, MA 02138, USA.
FAU - Joseph, Leo
AU  - Joseph L
FAU - Edwards, Scott V
AU  - Edwards SV
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20111006
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
SB  - IM
MH  - Animals
MH  - Classification/methods
MH  - Computational Biology
MH  - Genetic Markers
MH  - Genetic Speciation
MH  - Genetic Variation
MH  - Haplotypes
MH  - *Phylogeny
MH  - Software
MH  - Songbirds/classification/*genetics
EDAT- 2011/10/08 06:00
MHDA- 2012/06/26 06:00
CRDT- 2011/10/08 06:00
PHST- 2011/10/08 06:00 [entrez]
PHST- 2011/10/08 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr101 [pii]
AID - 10.1093/sysbio/syr101 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):253-71. doi: 10.1093/sysbio/syr101. Epub 2011 Oct 6.

PMID- 21963610
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - BEAGLE: an application programming interface and high-performance computing
      library for statistical phylogenetics.
PG  - 170-3
LID - 10.1093/sysbio/syr100 [doi]
AB  - Phylogenetic inference is fundamental to our understanding of most aspects of the
      origin and evolution of life, and in recent years, there has been a concentration
      of interest in statistical approaches such as Bayesian inference and maximum
      likelihood estimation. Yet, for large data sets and realistic or interesting
      models of evolution, these approaches remain computationally demanding.
      High-throughput sequencing can yield data for thousands of taxa, but scaling to
      such problems using serial computing often necessitates the use of nonstatistical
      or approximate approaches. The recent emergence of graphics processing units
      (GPUs) provides an opportunity to leverage their excellent floating-point
      computational performance to accelerate statistical phylogenetic inference. A
      specialized library for phylogenetic calculation would allow existing software
      packages to make more effective use of available computer hardware, including
      GPUs. Adoption of a common library would also make it easier for other emerging
      computing architectures, such as field programmable gate arrays, to be used in
      the future. We present BEAGLE, an application programming interface (API) and
      library for high-performance statistical phylogenetic inference. The API provides
      a uniform interface for performing phylogenetic likelihood calculations on a
      variety of compute hardware platforms. The library includes a set of efficient
      implementations and can currently exploit hardware including GPUs using NVIDIA
      CUDA, central processing units (CPUs) with Streaming SIMD Extensions and related 
      processor supplementary instruction sets, and multicore CPUs via OpenMP. To
      demonstrate the advantages of a common API, we have incorporated the library into
      several popular phylogenetic software packages. The BEAGLE library is free open
      source software licensed under the Lesser GPL and available from
      http://beagle-lib.googlecode.com. An example client program is available as
      public domain software.
FAU - Ayres, Daniel L
AU  - Ayres DL
AD  - Center for Bioinformatics and Computational Biology, University of Maryland,
      College Park, MD 20742, USA. ayres@umiacs.umd.edu
FAU - Darling, Aaron
AU  - Darling A
FAU - Zwickl, Derrick J
AU  - Zwickl DJ
FAU - Beerli, Peter
AU  - Beerli P
FAU - Holder, Mark T
AU  - Holder MT
FAU - Lewis, Paul O
AU  - Lewis PO
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
FAU - Ronquist, Fredrik
AU  - Ronquist F
FAU - Swofford, David L
AU  - Swofford DL
FAU - Cummings, Michael P
AU  - Cummings MP
FAU - Rambaut, Andrew
AU  - Rambaut A
FAU - Suchard, Marc A
AU  - Suchard MA
LA  - eng
GR  - R01-GM078985/GM/NIGMS NIH HHS/United States
GR  - R01 GM086887/GM/NIGMS NIH HHS/United States
GR  - R01 GM037841/GM/NIGMS NIH HHS/United States
GR  - R01 HG006139/HG/NHGRI NIH HHS/United States
GR  - R01 GM069801/GM/NIGMS NIH HHS/United States
GR  - 095831/Wellcome Trust/United Kingdom
GR  - WT092807MA/Wellcome Trust/United Kingdom
GR  - R01-HG006139/HG/NHGRI NIH HHS/United States
GR  - R01-GM037841/GM/NIGMS NIH HHS/United States
GR  - BB/H011285/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
GR  - R01-GM086887/GM/NIGMS NIH HHS/United States
GR  - R01-NS063897/NS/NINDS NIH HHS/United States
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20111001
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Computational Biology/*methods
MH  - Computing Methodologies
MH  - Evolution, Molecular
MH  - Genome
MH  - *Phylogeny
MH  - *Software
PMC - PMC3243739
EDAT- 2011/10/04 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/10/04 06:00
PHST- 2011/10/04 06:00 [entrez]
PHST- 2011/10/04 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr100 [pii]
AID - 10.1093/sysbio/syr100 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):170-3. doi: 10.1093/sysbio/syr100. Epub 2011 Oct 1.

PMID- 21934137
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20120216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - SuperFine: fast and accurate supertree estimation.
PG  - 214-27
LID - 10.1093/sysbio/syr092 [doi]
AB  - Many research groups are estimating trees containing anywhere from a few
      thousands to hundreds of thousands of species, toward the eventual goal of the
      estimation of a Tree of Life, containing perhaps as many as several million
      leaves. These phylogenetic estimations present enormous computational challenges,
      and current computational methods are likely to fail to run even on data sets in 
      the low end of this range. One approach to estimate a large species tree is to
      use phylogenetic estimation methods (such as maximum likelihood) on a supermatrix
      produced by concatenating multiple sequence alignments for a collection of
      markers; however, the most accurate of these phylogenetic estimation methods are 
      extremely computationally intensive for data sets with more than a few thousand
      sequences. Supertree methods, which assemble phylogenetic trees from a collection
      of trees on subsets of the taxa, are important tools for phylogeny estimation
      where phylogenetic analyses based upon maximum likelihood (ML) are infeasible. In
      this paper, we introduce SuperFine, a meta-method that utilizes a novel two-step 
      procedure in order to improve the accuracy and scalability of supertree methods. 
      Our study, using both simulated and empirical data, shows that SuperFine-boosted 
      supertree methods produce more accurate trees than standard supertree methods,
      and run quickly on very large data sets with thousands of sequences. Furthermore,
      SuperFine-boosted matrix representation with parsimony (MRP, the most well-known 
      supertree method) approaches the accuracy of ML methods on supermatrix data sets 
      under realistic conditions.
FAU - Swenson, M Shel
AU  - Swenson MS
AD  - Department of Computer Science, The University of Texas at Austin, Austin, TX,
      USA. shelswenson@gmail.com
FAU - Suri, Rahul
AU  - Suri R
FAU - Linder, C Randal
AU  - Linder CR
FAU - Warnow, Tandy
AU  - Warnow T
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110920
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/methods
MH  - Computational Biology
MH  - Computer Simulation
MH  - Likelihood Functions
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2011/09/22 06:00
MHDA- 2012/06/26 06:00
CRDT- 2011/09/22 06:00
PHST- 2011/09/22 06:00 [entrez]
PHST- 2011/09/22 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr092 [pii]
AID - 10.1093/sysbio/syr092 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):214-27. doi: 10.1093/sysbio/syr092. Epub 2011 Sep 20.

PMID- 21918178
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - Inferring species networks from gene trees in high-polyploid North American and
      Hawaiian violets (Viola, Violaceae).
PG  - 107-26
LID - 10.1093/sysbio/syr096 [doi]
AB  - The phylogenies of allopolyploids take the shape of networks and cannot be
      adequately represented as bifurcating trees. Especially for high polyploids
      (i.e., organisms with more than six sets of nuclear chromosomes), the signatures 
      of gene homoeolog loss, deep coalescence, and polyploidy may become confounded,
      with the result that gene trees may be congruent with more than one species
      network. Herein, we obtained the most parsimonious species network by objective
      comparison of competing scenarios involving polyploidization and homoeolog loss
      in a high-polyploid lineage of violets (Viola, Violaceae) mostly or entirely
      restricted to North America, Central America, or Hawaii. We amplified homoeologs 
      of the low-copy nuclear gene, glucose-6-phosphate isomerase (GPI), by
      single-molecule polymerase chain reaction (PCR) and the chloroplast trnL-F region
      by conventional PCR for 51 species and subspecies. Topological incongruence among
      GPI homoeolog subclades, owing to deep coalescence and two instances of putative 
      loss (or lack of detection) of homoeologs, were reconciled by applying the
      maximum tree topology for each subclade. The most parsimonious species network
      and the fossil-based calibration of the homoeolog tree favored monophyly of the
      high polyploids, which has resulted from allodecaploidization 9-14 Ma, involving 
      sympatric ancestors from the extant Viola sections Chamaemelanium (diploid),
      Plagiostigma (paleotetraploid), and Viola (paleotetraploid). Although two of the 
      high-polyploid lineages (Boreali-Americanae, Pedatae) remained decaploid,
      recurrent polyploidization with tetraploids of section Plagiostigma within the
      last 5 Ma has resulted in two 14-ploid lineages (Mexicanae, Nosphinium) and one
      18-ploid lineage (Langsdorffianae). This implies a more complex phylogenetic and 
      biogeographic origin of the Hawaiian violets (Nosphinium) than that previously
      inferred from rDNA data and illustrates the necessity of considering polyploidy
      in phylogenetic and biogeographic reconstruction.
FAU - Marcussen, Thomas
AU  - Marcussen T
AD  - 1Department of Biology, Centre for Ecological and Evolutionary Synthesis (CEES), 
      University of Oslo, PO Box 1066 Blindern, NO-0316 Oslo, Norway.
FAU - Jakobsen, Kjetill S
AU  - Jakobsen KS
FAU - Danihelka, Jiri
AU  - Danihelka J
FAU - Ballard, Harvey E
AU  - Ballard HE
FAU - Blaxland, Kim
AU  - Blaxland K
FAU - Brysting, Anne K
AU  - Brysting AK
FAU - Oxelman, Bengt
AU  - Oxelman B
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110914
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
RN  - 0 (DNA, Plant)
RN  - 0 (RNA, Plant)
RN  - EC 5.3.1.9 (Glucose-6-Phosphate Isomerase)
SB  - IM
MH  - Cell Nucleus/genetics
MH  - DNA, Chloroplast/genetics
MH  - DNA, Plant/genetics
MH  - *Evolution, Molecular
MH  - Glucose-6-Phosphate Isomerase/genetics
MH  - Hawaii
MH  - North America
MH  - *Phylogeny
MH  - Polymerase Chain Reaction
MH  - Polyploidy
MH  - RNA, Plant/genetics
MH  - Sequence Analysis, DNA
MH  - Viola/chemistry/*classification/*genetics
PMC - PMC3243738
EDAT- 2011/09/16 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/09/16 06:00
PHST- 2011/09/16 06:00 [entrez]
PHST- 2011/09/16 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr096 [pii]
AID - 10.1093/sysbio/syr096 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):107-26. doi: 10.1093/sysbio/syr096. Epub 2011 Sep 14.

PMID- 21918177
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - Towards a dynamic analysis of weighted networks in biogeography.
PG  - 240-52
LID - 10.1093/sysbio/syr098 [doi]
AB  - An improvement to the Network Analysis Method (NAM) in Biogeography based on
      weighted inference and dynamic exploration of sympatry networks is proposed.
      Intricate distributions of species result in a reticulated structure of spatial
      associations. Species are geographically connected through sympatry links forming
      an overall natural network in biogeography. Spatial records are the signals that 
      provide evidence to infer these sympatry links in the network. Punctual data are 
      independent of a priori area determination. NAM is oriented to detect groups of
      species embedded into the global network that are internally sustained by
      sympatric cohesiveness but weakly connected (or disconnected) to outgroup
      entities. These groups, called units of co-occurrence (UCs), are segregated
      through the iterative removal of intermediary species according to their
      betweenness scores. Instances of analysis of the original NAM are improved
      through the following changes and extensions: (i) inference of weighted sympatry 
      networks using new measures sensitive to the strength of overlap and topological 
      resemblance between set of points; (ii) construction of a basal network
      discriminating major from minor sympatry associations; (iii) evaluation of the
      entire process of iterative removal of intermediary species for the selection of 
      UCs found on different subnetworks; (iv) network partitioning based on the
      intrinsic cohesiveness of the UCs; (v) production of a graphical tool
      (cleavogram) depicting the structural changes of the network along the removal
      process. Improvements are tested using real and hypothetical data sets.
      Resolution of patterns is notably increased due to a more accurate recognition of
      allopatric patterns and the possibility of segregating spatially overlapped UCs. 
      As in original NAM, spatial expressions of UCs are building blocks for
      biogeography supported by strictly endemic and connected species through sympatry
      paths.
FAU - Dos Santos, Daniel A
AU  - Dos Santos DA
AD  - Consejo Nacional de Investigaciones Cientificas y Tecnicas, Tucuman, Argentina.
FAU - Gabriela Cuezzo, Maria
AU  - Gabriela Cuezzo M
FAU - Celina Reynaga, Maria
AU  - Celina Reynaga M
FAU - Dominguez, Eduardo
AU  - Dominguez E
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110914
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - Insecta/classification
MH  - Models, Biological
MH  - New Zealand
MH  - Phylogeography/*methods
MH  - Species Specificity
EDAT- 2011/09/16 06:00
MHDA- 2012/06/26 06:00
CRDT- 2011/09/16 06:00
PHST- 2011/09/16 06:00 [entrez]
PHST- 2011/09/16 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr098 [pii]
AID - 10.1093/sysbio/syr098 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):240-52. doi: 10.1093/sysbio/syr098. Epub 2011 Sep 14.

PMID- 21900649
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - Phylogenomic analysis resolves the interordinal relationships and rapid
      diversification of the laurasiatherian mammals.
PG  - 150-64
LID - 10.1093/sysbio/syr089 [doi]
AB  - Although great progress has been made in resolving the relationships of placental
      mammals, the position of several clades in Laurasiatheria remain controversial.
      In this study, we performed a phylogenetic analysis of 97 orthologs (46,152 bp)
      for 15 taxa, representing all laurasiatherian orders. Additionally, phylogenetic 
      trees of laurasiatherian mammals with draft genome sequences were reconstructed
      based on 1608 exons (2,175,102 bp). Our reconstructions resolve the interordinal 
      relationships within Laurasiatheria and corroborate the clades Scrotifera,
      Fereuungulata, and Cetartiodactyla. Furthermore, we tested alternative topologies
      within Laurasiatheria, and among alternatives for the phylogenetic position of
      Perissodactyla, a sister-group relationship with Cetartiodactyla receives the
      highest support. Thus, Pegasoferae (Perissodactyla + Carnivora + Pholidota +
      Chiroptera) does not appear to be a natural group. Divergence time estimates from
      these genes were compared with published estimates for splits within
      Laurasiatheria. Our estimates were similar to those of several studies and
      suggest that the divergences among these orders occurred within just a few
      million years.
FAU - Zhou, Xuming
AU  - Zhou X
AD  - Jiangsu Key Laboratory for Biodiversity and Biotechnology, College of Life
      Sciences, Nanjing Normal University, Nanjing 210046, China.
FAU - Xu, Shixia
AU  - Xu S
FAU - Xu, Junxiao
AU  - Xu J
FAU - Chen, Bingyao
AU  - Chen B
FAU - Zhou, Kaiya
AU  - Zhou K
FAU - Yang, Guang
AU  - Yang G
LA  - eng
SI  - GENBANK/GU991249
SI  - GENBANK/GU991250
SI  - GENBANK/GU991251
SI  - GENBANK/GU991252
SI  - GENBANK/GU991253
SI  - GENBANK/GU991254
SI  - GENBANK/GU991255
SI  - GENBANK/GU991256
SI  - GENBANK/GU991257
SI  - GENBANK/GU991258
SI  - GENBANK/GU991259
SI  - GENBANK/GU991260
SI  - GENBANK/GU991261
SI  - GENBANK/GU991262
SI  - GENBANK/GU991263
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110907
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amino Acid Sequence
MH  - Animals
MH  - Cell Nucleus/genetics
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - *Genome
MH  - Mammals/*classification/*genetics
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Sequence Alignment/veterinary
MH  - Sequence Analysis, DNA/veterinary
PMC - PMC3243735
EDAT- 2011/09/09 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/09/09 06:00
PHST- 2011/09/09 06:00 [entrez]
PHST- 2011/09/09 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr089 [pii]
AID - 10.1093/sysbio/syr089 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):150-64. doi: 10.1093/sysbio/syr089. Epub 2011 Sep 7.

PMID- 21878471
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - Phylogeny estimation of the radiation of western North American chipmunks
      (Tamias) in the face of introgression using reproductive protein genes.
PG  - 44-62
LID - 10.1093/sysbio/syr094 [doi]
AB  - The causes and consequences of rapid radiations are major unresolved issues in
      evolutionary biology. This is in part because phylogeny estimation is confounded 
      by processes such as stochastic lineage sorting and hybridization. Because these 
      processes are expected to be heterogeneous across the genome, comparison among
      marker classes may provide a means of disentangling these elements. Here we use
      introns from nuclear-encoded reproductive protein genes expected to be resistant 
      to introgression to estimate the phylogeny of the western chipmunks (Tamias:
      subgenus: Neotamias), a rapid radiation that has experienced introgressive
      hybridization of mitochondrial DNA (mtDNA). We analyze the nuclear loci using
      coalescent-based species-tree estimation methods and concatenation to estimate a 
      species tree and we use parametric bootstraps and coalescent simulations to
      differentiate between phylogenetic error, coalescent stochasticity and
      introgressive hybridization. Results indicate that the mtDNA gene tree reflects
      several introgression events that have occurred between taxa of varying levels of
      divergence and at different time points in the tree. T. panamintinus and T.
      speciosus appear to be fixed for ancient mitochondrial introgressions from T.
      minimus. A southern Rocky Mountains clade appears well sorted (i.e., species are 
      largely monophyletic) at multiple nuclear loci, while five of six taxa are
      nonmonophyletic based on cytochrome b. Our simulations reject phylogenetic error 
      and coalescent stochasticity as causes. The results represent an advance in our
      understanding of the processes at work during the radiation of Tamias and suggest
      that sampling reproductive-protein genes may be a viable strategy for phylogeny
      estimation of rapid radiations in which reproductive isolation is incomplete.
      However, a genome-scale survey that can statistically compare heterogeneity of
      genealogical process at many more loci will be necessary to test this conclusion.
FAU - Reid, Noah
AU  - Reid N
AD  - Department of Biological Sciences, Room 105, Louisiana State University, Baton
      Rouge, LA 70803, USA. nreid1@tigers.lsu.edu
FAU - Demboski, John R
AU  - Demboski JR
FAU - Sullivan, Jack
AU  - Sullivan J
LA  - eng
GR  - P20 RR16448/RR/NCRR NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110830
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Membrane Proteins)
RN  - 0 (Proteins)
RN  - 9035-37-4 (Cytochromes b)
RN  - EC 3.4.21.10 (Acrosin)
SB  - IM
MH  - Acrosin/genetics
MH  - Animals
MH  - Cell Nucleus/genetics
MH  - Computer Simulation
MH  - Cytochromes b/genetics
MH  - DNA, Mitochondrial/genetics
MH  - *Genetic Speciation
MH  - Haplotypes
MH  - Hybridization, Genetic
MH  - Introns
MH  - Membrane Proteins/genetics
MH  - Molecular Sequence Data
MH  - North America
MH  - Phylogeny
MH  - Polymerase Chain Reaction/veterinary
MH  - Proteins/*genetics
MH  - Reproductive Isolation
MH  - Sciuridae/classification/*genetics
MH  - Sequence Analysis, DNA
PMC - PMC3243737
EDAT- 2011/09/01 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/09/01 06:00
PHST- 2011/09/01 06:00 [entrez]
PHST- 2011/09/01 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr094 [pii]
AID - 10.1093/sysbio/syr094 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):44-62. doi: 10.1093/sysbio/syr094. Epub 2011 Aug 30.

PMID- 21876212
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - Evaluation of a bayesian coalescent method of species delimitation.
PG  - 747-61
LID - 10.1093/sysbio/syr071 [doi]
AB  - A Bayesian coalescent-based method has recently been proposed to delimit species 
      using multilocus genetic sequence data. Posterior probabilities of different
      species delimitation models are calculated using reversible-jump Markov chain
      Monte Carlo algorithms. The method accounts for species phylogenies and
      coalescent events in both extant and extinct species and accommodates lineage
      sorting and uncertainties in the gene trees. Although the method is theoretically
      appealing, its utility in practical data analysis is yet to be rigorously
      examined. In particular, the analysis may be sensitive to priors on ancestral
      population sizes and on species divergence times and to gene flow between
      species. Here we conduct a computer simulation to evaluate the statistical
      performance of the method, such as the false negatives (the error of lumping
      multiple species into one) and false positives (the error of splitting one
      species into several). We found that the correct species model was inferred with 
      high posterior probability with only one or two loci when 5 or 10 sequences were 
      sampled from each population, or with 50 loci when only one sequence was sampled.
      We also simulated data allowing migration under a two-species model, a
      mainland-island model and a stepping-stone model to assess the impact of gene
      flow (hybridization or introgression). The behavior of the method was
      diametrically different depending on the migration rate. Low rates at &lt; 0.1
      migrants per generation had virtually no effect, so that the method, while
      assuming no hybridization between species, identified distinct species despite
      small amounts of gene flow. This behavior appears to be consistent with
      biologists' practice. In contrast, higher migration rates at &gt;/= 10 migrants per 
      generation caused the method to infer one species. At intermediate levels of
      migration, the method is indecisive. Our results suggest that Bayesian analysis
      under the multispecies coalescent model may provide important insights into
      population divergences, and may be useful for generating hypotheses of species
      delimitation, to be assessed with independent information from anatomical,
      behavioral, and ecological data.
FAU - Zhang, Chi
AU  - Zhang C
AD  - Center for Computational and Evolutionary Biology, Institute of Zoology, Chinese 
      Academy of Sciences, Beijing, China.
FAU - Zhang, De-Xing
AU  - Zhang DX
FAU - Zhu, Tianqi
AU  - Zhu T
FAU - Yang, Ziheng
AU  - Yang Z
LA  - eng
GR  - BB/G006431/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110829
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animal Migration
MH  - Animals
MH  - Butterflies/genetics
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - Gene Flow
MH  - Genetic Variation
MH  - Genetics, Population/*methods
MH  - Mutation Rate
EDAT- 2011/08/31 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/08/31 06:00
PHST- 2011/08/31 06:00 [entrez]
PHST- 2011/08/31 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr071 [pii]
AID - 10.1093/sysbio/syr071 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):747-61. doi: 10.1093/sysbio/syr071. Epub 2011 Aug 29.

PMID- 21873377
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - Generalized mixture models for molecular phylogenetic estimation.
PG  - 12-21
LID - 10.1093/sysbio/syr093 [doi]
AB  - The rapidly growing availability of multigene sequence data during the past
      decade has enabled phylogeny estimation at phylogenomic scales. However, dealing 
      with evolutionary process heterogeneity across the genome becomes increasingly
      challenging. Here we develop a mixture model approach that uses reversible jump
      Markov chain Monte Carlo (MCMC) estimation to permit as many distinct models as
      the data require. Each additional model considered may be a fully parametrized
      general time-reversible model or any of its special cases. Furthermore, we expand
      the usual proposal mechanisms for topology changes to permit hard polytomies
      (i.e., zero-length internal branches). This new approach is implemented in the
      Crux software toolkit. We demonstrate the feasibility of using reversible jump
      MCMC on mixture models by reexamining a well-known 44-taxon mammalian data set
      comprising 22 concatenated genes. We are able to reproduce the results of the
      original analysis (with respect to bipartition support) when we make identical
      assumptions, but when we allow for polytomies and/or use data-driven mixture
      model estimation, we infer much lower bipartition support values for several key 
      bipartitions.
FAU - Evans, Jason
AU  - Evans J
AD  - Program in Bioinformatics and Computational Biology, Department of Biological
      Sciences, University of Idaho, PO Box 443051, Moscow, ID 83844-3051, USA.
      jasone@canonware.com
FAU - Sullivan, Jack
AU  - Sullivan J
LA  - eng
GR  - P20 RR016454/RR/NCRR NIH HHS/United States
GR  - P20 RR16448/RR/NCRR NIH HHS/United States
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
DEP - 20110826
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - *Bayes Theorem
MH  - Biological Evolution
MH  - Cell Nucleus/genetics
MH  - Classification/*methods
MH  - DNA, Mitochondrial/genetics
MH  - *Data Interpretation, Statistical
MH  - Likelihood Functions
MH  - Mammals/classification/genetics
MH  - Markov Chains
MH  - Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
PMC - PMC3243736
EDAT- 2011/08/30 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/08/30 06:00
PHST- 2011/08/30 06:00 [entrez]
PHST- 2011/08/30 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr093 [pii]
AID - 10.1093/sysbio/syr093 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):12-21. doi: 10.1093/sysbio/syr093. Epub 2011 Aug 26.

PMID- 21873376
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - Prolonging the past counteracts the pull of the present: protracted speciation
      can explain observed slowdowns in diversification.
PG  - 204-13
LID - 10.1093/sysbio/syr091 [doi]
AB  - Phylogenetic trees show a remarkable slowdown in the increase of number of
      lineages towards the present, a phenomenon which cannot be explained by the
      standard birth-death model of diversification with constant speciation and
      extinction rates. The birth-death model instead predicts a constant or
      accelerating increase in the number of lineages, which has been called the pull
      of the present. The observed slowdown has been attributed to nonconstancy of the 
      speciation and extinction rates due to some form of diversity dependence (i.e.,
      species-level density dependence), but the mechanisms underlying this are still
      unclear. Here, we propose an alternative explanation based on the simple concept 
      that speciation takes time to complete. We show that this idea of "protracted"
      speciation can be incorporated in the standard birth-death model of
      diversification. The protracted birth-death model predicts a realistic slowdown
      in the rate of increase of number of lineages in the phylogeny and provides a
      compelling fit to four bird phylogenies with realistic parameter values. Thus,
      the effect of recognizing the generally accepted fact that speciation is not an
      instantaneous event is significant; even if it cannot account for all the
      observed patterns, it certainly contributes substantially and should therefore be
      incorporated into future studies.
FAU - Etienne, Rampal S
AU  - Etienne RS
AD  - Community and Conservation Ecology Group, Centre for Ecological and Evolutionary 
      Studies, University of Groningen, Box 11103, 9700 CC Groningen, The Netherlands. 
      r.s.etienne@rug.nl
FAU - Rosindell, James
AU  - Rosindell J
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110826
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biodiversity
MH  - *Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
PMC - PMC3280041
EDAT- 2011/08/30 06:00
MHDA- 2012/06/26 06:00
CRDT- 2011/08/30 06:00
PHST- 2011/08/30 06:00 [entrez]
PHST- 2011/08/30 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr091 [pii]
AID - 10.1093/sysbio/syr091 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):204-13. doi: 10.1093/sysbio/syr091. Epub 2011 Aug 26.

PMID- 21865338
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - Tree models for macroevolution and phylogenetic analysis.
PG  - 735-46
LID - 10.1093/sysbio/syr086 [doi]
AB  - It has long been recognized that phylogenetic trees are more unbalanced than
      those generated by a Yule process. Recently, the degree of this imbalance has
      been quantified using the large set of phylogenetic trees available in the
      TreeBASE data set. In this article, a more precise analysis of imbalance is
      undertaken. Trees simulated under a range of models are compared with trees from 
      TreeBASE and two smaller data sets. Several simple models can match the amount of
      imbalance measured in real data. Most of them also match the variance of
      imbalance among empirical trees to a remarkable degree. Statistics are developed 
      to measure balance and to distinguish between trees with the same overall
      imbalance. The match between models and data for these statistics is
      investigated. In particular, age-dependent (Bellman-Harris) branching process are
      studied in detail. It remains difficult to separate the process of macroevolution
      from biases introduced by sampling. The lessons for phylogenetic analysis are
      clearer. In particular, the use of the usual proportional to distinguishable
      arrangements (uniform) prior on tree topologies in Bayesian phylogenetic analysis
      is not recommended.
FAU - Jones, Graham R
AU  - Jones GR
AD  - Balnakeil, Durness, Lairg, UK. art@gjones.name
LA  - eng
PT  - Journal Article
DEP - 20110824
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2011/08/26 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/08/26 06:00
PHST- 2011/08/26 06:00 [entrez]
PHST- 2011/08/26 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr086 [pii]
AID - 10.1093/sysbio/syr086 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):735-46. doi: 10.1093/sysbio/syr086. Epub 2011 Aug 24.

PMID- 21865337
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20121115
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - Dispersal between shallow and abyssal seas and evolutionary loss and regain of
      compound eyes in cylindroleberidid ostracods: conflicting conclusions from
      different comparative methods.
PG  - 314-36
LID - 10.1093/sysbio/syr085 [doi]
AB  - Complex organs such as eyes are commonly lost during evolution, but the timescale
      on which lost phenotypes could be reactivated is a matter of long-standing
      debate, with important implications for the molecular mechanisms of trait loss.
      Two phylogenetic approaches have been used to test whether regain of traits has
      occurred. One way is by comparison of nested, continuous-time Markov models of
      trait evolution, approaches that we term tree-based tests. A second way to
      demonstrate statistical support for trait regain is through use of node-based
      tests that employ explicit estimation of ancestral node states. Here, we estimate
      new molecular and morphological phylogenies and use them to examine the
      possibility of eye regain and dispersal between abyssal and shallow seas during
      the history of cylindroleberidid ostracods, a family of about 200 species,
      comprising both eyeless and sighted species. First, we confirmed that eye
      presence/absence is correlated with habitat depth. Parameter estimates from a
      phylogenetic model indicate that speciation is more rapid in deep-sea eyeless
      clades compared with shallow-water sighted clades. In addition, we found that
      tree-based statistical tests usually indicated reversals, including both
      transitions from deep to shallow seas and regain of eyes. In contrast, node-based
      statistical tests usually failed to show significant support for reversals. These
      results also hold for simulated phylogenies, indicating that they are not unique 
      to the current data set. We recommend that both tree-based and node-based tests
      should be examined before making conclusions about character reversal and that
      ideally, alternative character histories should be tested using additional data, 
      besides just the phylogenetic distribution of presence/absence of the characters.
FAU - Syme, Anna E
AU  - Syme AE
AD  - Ecology, Evolution, Marine Biology and Marine Science Institute, University of
      California-Santa Barbara, Santa Barbara, CA 93106, USA.
FAU - Oakley, Todd H
AU  - Oakley TH
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110824
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - *Compound Eye, Arthropod
MH  - *Crustacea
MH  - Genetic Speciation
MH  - Markov Chains
MH  - Models, Biological
MH  - Oceans and Seas
MH  - *Phylogeny
EDAT- 2011/08/26 06:00
MHDA- 2012/06/26 06:00
CRDT- 2011/08/26 06:00
PHST- 2011/08/26 06:00 [entrez]
PHST- 2011/08/26 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr085 [pii]
AID - 10.1093/sysbio/syr085 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):314-36. doi: 10.1093/sysbio/syr085. Epub 2011 Aug 24.

PMID- 21865336
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20120216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - Branch lengths on birth-death trees and the expected loss of phylogenetic
      diversity.
PG  - 195-203
LID - 10.1093/sysbio/syr090 [doi]
AB  - Diversification is nested, and early models suggested this could lead to a great 
      deal of evolutionary redundancy in the Tree of Life. This result is based on a
      particular set of branch lengths produced by the common coalescent, where pendant
      branches leading to tips can be very short compared with branches deeper in the
      tree. Here, we analyze alternative and more realistic Yule and birth-death
      models. We show how censoring at the present both makes average branches one half
      what we might expect and makes pendant and interior branches roughly equal in
      length. Although dependent on whether we condition on the size of the tree, its
      age, or both, these results hold both for the Yule model and for birth-death
      models with moderate extinction. Importantly, the rough equivalency in interior
      and exterior branch lengths means that the loss of evolutionary history with loss
      of species can be roughly linear. Under these models, the Tree of Life may offer 
      limited redundancy in the face of ongoing species loss.
FAU - Mooers, Arne
AU  - Mooers A
AD  - The Interdisciplinary Research in Computing and Mathematical Sciences Center, and
      BioSciences, Simon Fraser University, 8888 University Drive, Burnaby, BC, Canada 
      V5A 1S6. amooers@sfu.ca
FAU - Gascuel, Olivier
AU  - Gascuel O
FAU - Stadler, Tanja
AU  - Stadler T
FAU - Li, Heyang
AU  - Li H
FAU - Steel, Mike
AU  - Steel M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110824
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/methods
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2011/08/26 06:00
MHDA- 2012/06/26 06:00
CRDT- 2011/08/26 06:00
PHST- 2011/08/26 06:00 [entrez]
PHST- 2011/08/26 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr090 [pii]
AID - 10.1093/sysbio/syr090 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):195-203. doi: 10.1093/sysbio/syr090. Epub 2011 Aug 24.

PMID- 21865335
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - Phylogenetic nomenclature, three-taxon statements, and unnecessary name changes.
PG  - 887-92
LID - 10.1093/sysbio/syr070 [doi]
FAU - de Queiroz, Kevin
AU  - de Queiroz K
AD  - Department of Vertebrate Zoology, National Museum of Natural History, Smithsonian
      Institution, Washington, DC 20560-0162, USA.
FAU - Donoghue, Michael J
AU  - Donoghue MJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110824
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CIN - Syst Biol. 2012 Mar;61(2):360-1. PMID: 22201160
MH  - Animals
MH  - Classification/*methods
MH  - *Phylogeny
MH  - *Terminology as Topic
EDAT- 2011/08/26 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/08/26 06:00
PHST- 2011/08/26 06:00 [entrez]
PHST- 2011/08/26 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr070 [pii]
AID - 10.1093/sysbio/syr070 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):887-92. doi: 10.1093/sysbio/syr070. Epub 2011 Aug 24.

PMID- 21856636
OWN - NLM
STAT- MEDLINE
DCOM- 20111220
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 5
DP  - 2011 Oct
TI  - Mass extinction, gradual cooling, or rapid radiation? Reconstructing the
      spatiotemporal evolution of the ancient angiosperm genus Hedyosmum
      (Chloranthaceae) using empirical and simulated approaches.
PG  - 596-615
LID - 10.1093/sysbio/syr062 [doi]
AB  - Chloranthaceae is a small family of flowering plants (65 species) with an
      extensive fossil record extending back to the Early Cretaceous. Within
      Chloranthaceae, Hedyosmum is remarkable because of its disjunct distribution--1
      species in the Paleotropics and 44 confined to the Neotropics--and a long
      "temporal gap" between its stem age (Early Cretaceous) and the beginning of the
      extant radiation (late Cenozoic). Is this gap real, reflecting low
      diversification and a recent radiation, or the signature of extinction? Here we
      use paleontological data, relaxed-clock molecular dating, diversification
      analyses, and parametric ancestral area reconstruction to investigate the timing,
      tempo, and mode of diversification in Hedyosmum. Our results, based on analyses
      of plastid and nuclear sequences for 40 species, suggest that the ancestor of
      Chloranthaceae and the Hedyosmum stem lineages were widespread in the Holarctic
      in the Late Cretaceous. High extinction rates, possibly associated with Cenozoic 
      climatic fluctuations, may have been responsible for the low extant diversity of 
      the family. Crown group Hedyosmum originated c. 36-43 Ma and colonized South
      America from the north during the Early-Middle Miocene (c. 20 Ma). This coincided
      with an increase in diversification rates, probably triggered by the uplift of
      the Northern Andes from the Mid-Miocene onward. This study illustrates the
      advantages of combining paleontological, phylogenetic, and biogeographic data to 
      reconstruct the spatiotemporal evolution of an ancient lineage, for which the
      extant diversity is only a remnant of past radiations. It also shows the
      difficulties of inferring patterns of lineage diversification when incomplete
      taxon sampling is combined with high extinction rates.
FAU - Antonelli, Alexandre
AU  - Antonelli A
AD  - Gothenburg Botanical Garden, Carl Skottsbergs gata 22A, 41319 Goteborg, Sweden.
      alexandre.antonelli@dpes.gu.se
FAU - Sanmartin, Isabel
AU  - Sanmartin I
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - Cell Nucleus/genetics
MH  - Climate Change
MH  - *Evolution, Molecular
MH  - Extinction, Biological
MH  - Fossils
MH  - Magnoliopsida/*classification/*genetics
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Plastids/genetics
MH  - Sequence Analysis, DNA
MH  - South America
MH  - Statistics, Nonparametric
EDAT- 2011/08/23 06:00
MHDA- 2011/12/21 06:00
CRDT- 2011/08/23 06:00
PHST- 2011/08/23 06:00 [entrez]
PHST- 2011/08/23 06:00 [pubmed]
PHST- 2011/12/21 06:00 [medline]
AID - syr062 [pii]
AID - 10.1093/sysbio/syr062 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Oct;60(5):596-615. doi: 10.1093/sysbio/syr062.

PMID- 21856631
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - Calibrated tree priors for relaxed phylogenetics and divergence time estimation.
PG  - 138-49
LID - 10.1093/sysbio/syr087 [doi]
AB  - The use of fossil evidence to calibrate divergence time estimation has a long
      history. More recently, Bayesian Markov chain Monte Carlo has become the dominant
      method of divergence time estimation, and fossil evidence has been reinterpreted 
      as the specification of prior distributions on the divergence times of
      calibration nodes. These so-called "soft calibrations" have become widely used
      but the statistical properties of calibrated tree priors in a Bayesian setting
      hashave not been carefully investigated. Here, we clarify that calibration
      densities, such as those defined in BEAST 1.5, do not represent the marginal
      prior distribution of the calibration node. We illustrate this with a number of
      analytical results on small trees. We also describe an alternative construction
      for a calibrated Yule prior on trees that allows direct specification of the
      marginal prior distribution of the calibrated divergence time, with or without
      the restriction of monophyly. This method requires the computation of the Yule
      prior conditional on the height of the divergence being calibrated.
      Unfortunately, a practical solution for multiple calibrations remains elusive.
      Our results suggest that direct estimation of the prior induced by specifying
      multiple calibration densities should be a prerequisite of any divergence time
      dating analysis.
FAU - Heled, Joseph
AU  - Heled J
AD  - Department of Computer Science, University of Auckland, Auckland, New Zealand.
      jheled@gmail.com
FAU - Drummond, Alexei J
AU  - Drummond AJ
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110818
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Bayes Theorem
MH  - Calibration
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Fossils
MH  - Markov Chains
MH  - Marsupialia/classification/genetics
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - Papio/classification/genetics
MH  - *Phylogeny
PMC - PMC3243734
EDAT- 2011/08/23 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/08/23 06:00
PHST- 2011/08/23 06:00 [entrez]
PHST- 2011/08/23 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr087 [pii]
AID - 10.1093/sysbio/syr087 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):138-49. doi: 10.1093/sysbio/syr087. Epub 2011 Aug 18.

PMID- 21856630
OWN - NLM
STAT- MEDLINE
DCOM- 20130122
LR  - 20120813
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 5
DP  - 2012 Oct
TI  - Predicting total global species richness using rates of species description and
      estimates of taxonomic effort.
PG  - 871-83
LID - 10.1093/sysbio/syr080 [doi]
AB  - We found that trends in the rate of description of 580,000 marine and terrestrial
      species, in the taxonomically authoritative World Register of Marine Species and 
      Catalogue of Life databases, were similar until the 1950s. Since then, the
      relative number of marine to terrestrial species described per year has
      increased, reflecting the less explored nature of the oceans. From the mid-19th
      century, the cumulative number of species described has been linear, with the
      highest number of species described in the decade of 1900, and fewer species
      described and fewer authors active during the World Wars. There were more authors
      describing species since the 1960s, indicating greater taxonomic effort. There
      were fewer species described per author since the 1920s, suggesting it has become
      more difficult to discover new species. There was no evidence of any change in
      individual effort by taxonomists. Using a nonhomogeneous renewal process model we
      predicted that 24-31% to 21-29% more marine and terrestrial species remain to be 
      discovered, respectively. We discuss why we consider that marine species comprise
      only 16% of all species on Earth although the oceans contain a greater
      phylogenetic diversity than occurs on land. We predict that there may be 1.8-2.0 
      million species on Earth, of which about 0.3 million are marine, significantly
      less than some previous estimates.
FAU - Costello, Mark J
AU  - Costello MJ
AD  - Leigh Marine Laboratory, University of Auckland, Warkworth, New Zealand.
      m.costello@auckland.ac.nz
FAU - Wilson, Simon
AU  - Wilson S
FAU - Houlding, Brett
AU  - Houlding B
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110818
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biodiversity
MH  - *Classification
MH  - *Ecosystem
MH  - Logistic Models
MH  - Models, Biological
MH  - Phylogeny
MH  - Species Specificity
MH  - Stochastic Processes
EDAT- 2011/08/23 06:00
MHDA- 2013/01/23 06:00
CRDT- 2011/08/23 06:00
PHST- 2011/08/23 06:00 [entrez]
PHST- 2011/08/23 06:00 [pubmed]
PHST- 2013/01/23 06:00 [medline]
AID - syr080 [pii]
AID - 10.1093/sysbio/syr080 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Oct;61(5):871-83. doi: 10.1093/sysbio/syr080. Epub 2011 Aug 18.

PMID- 21856629
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - Correlating early evolution of parasitic platyhelminths to Gondwana breakup.
PG  - 762-81
LID - 10.1093/sysbio/syr078 [doi]
AB  - Investigating patterns and processes of parasite diversification over ancient
      geological periods should involve comparisons of host and parasite phylogenies in
      a biogeographic context. It has been shown previously that the geographical
      distribution of host-specific parasites of sarcopterygians was guided, from
      Palaeozoic to Cainozoic times, mostly by evolution and diversification of their
      freshwater hosts. Here, we propose phylogenies of neobatrachian frogs and their
      specific parasites (Platyhelminthes, Monogenea) to investigate coevolutionary
      processes and historical biogeography of polystomes and further discuss all the
      possible assumptions that may account for the early evolution of these parasites.
      Phylogenetic analyses of concatenated rRNA nuclear genes (18S and partial 28S)
      supplemented by cophylogenetic and biogeographic vicariance analyses reveal four 
      main parasite lineages that can be ascribed to centers of diversity, namely
      Australia, India, Africa, and South America. In addition, the relationships among
      these biogeographical monophyletic groups, substantiated by molecular dating,
      reflect sequential origins during the breakup of Gondwana. The Australian
      polystome lineage may have been isolated during the first stages of the breakup, 
      whereas the Indian lineage would have arisen after the complete separation of
      western and eastern Gondwanan components. Next, polystomes would have codiverged 
      with hyloid sensu stricto and ranoid frog lineages before the completion of South
      American and African plate separation. Ultimately, they would have undergone an
      extensive diversification in South America when their ancestral host families
      diversified. Therefore, the presence of polystome parasites in specific anuran
      host clades and in discrete geographic areas reveals the importance of
      biogeographic vicariance in diversification processes and supports the occurrence
      and radiation of amphibians over ancient and recent geological periods.
FAU - Badets, Mathieu
AU  - Badets M
AD  - UMR 5244 CNRS-UPVD, Biologie et Ecologie Tropicale et Mediterraneenne,
      Parasitologie Fonctionnelle et Evolutive, Universite de Perpignan Via Domitia, 52
      Avenue Paul Alduy, Perpignan Cedex, France.
FAU - Whittington, Ian
AU  - Whittington I
FAU - Lalubin, Fabrice
AU  - Lalubin F
FAU - Allienne, Jean-Francois
AU  - Allienne JF
FAU - Maspimby, Jean-Luc
AU  - Maspimby JL
FAU - Bentz, Sophie
AU  - Bentz S
FAU - Du Preez, Louis H
AU  - Du Preez LH
FAU - Barton, Diane
AU  - Barton D
FAU - Hasegawa, Hideo
AU  - Hasegawa H
FAU - Tandon, Veena
AU  - Tandon V
FAU - Imkongwapang, Rangpenyuba
AU  - Imkongwapang R
FAU - Ohler, Annemarie
AU  - Ohler A
FAU - Combes, Claude
AU  - Combes C
FAU - Verneau, Olivier
AU  - Verneau O
LA  - eng
SI  - GENBANK/AM157183
SI  - GENBANK/AM157184
SI  - GENBANK/AM157185
SI  - GENBANK/AM157186
SI  - GENBANK/AM157187
SI  - GENBANK/AM157188
SI  - GENBANK/AM157189
SI  - GENBANK/AM157190
SI  - GENBANK/AM157191
SI  - GENBANK/AM157192
SI  - GENBANK/AM157193
SI  - GENBANK/AM157194
SI  - GENBANK/AM157195
SI  - GENBANK/AM157196
SI  - GENBANK/AM157197
SI  - GENBANK/AM157198
SI  - GENBANK/AM157199
SI  - GENBANK/AM157200
SI  - GENBANK/AM157201
SI  - GENBANK/AM157202
SI  - GENBANK/AM157203
SI  - GENBANK/AM157204
SI  - GENBANK/AM157205
SI  - GENBANK/AM157206
SI  - GENBANK/AM157207
SI  - GENBANK/AM157208
SI  - GENBANK/AM157209
SI  - GENBANK/AM157210
SI  - GENBANK/AM157211
SI  - GENBANK/AM157212
SI  - GENBANK/AM157213
SI  - GENBANK/AM157214
SI  - GENBANK/AM157215
SI  - GENBANK/AM157216
SI  - GENBANK/AM157217
SI  - GENBANK/AM157218
SI  - GENBANK/AM157219
SI  - GENBANK/AM157220
SI  - GENBANK/AM157221
SI  - GENBANK/AM157222
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110818
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 18S)
RN  - 0 (RNA, Ribosomal, 28S)
SB  - IM
EIN - Syst Biol. 2012 Jan 1;61(1):178. Imkongwapang, Rangpenyubai [corrected to
      Imkongwapang, Rangpenyuba]
MH  - Animals
MH  - Anura/*classification/genetics/*parasitology
MH  - *Biological Evolution
MH  - Genetic Variation
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Phylogeography
MH  - Platyhelminths/*classification/genetics
MH  - RNA, Ribosomal, 18S/genetics
MH  - RNA, Ribosomal, 28S/genetics
MH  - Sequence Alignment
MH  - Trematode Infections/*parasitology
EDAT- 2011/08/23 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/08/23 06:00
PHST- 2011/08/23 06:00 [entrez]
PHST- 2011/08/23 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr078 [pii]
AID - 10.1093/sysbio/syr078 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):762-81. doi: 10.1093/sysbio/syr078. Epub 2011 Aug 18.

PMID- 21856628
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20111222
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - Biogeography revisited with network theory: retracing the history of hydrothermal
      vent communities.
PG  - 127-37
LID - 10.1093/sysbio/syr088 [doi]
AB  - Defining biogeographic provinces to understand the history and evolution of
      communities associated with a given kind of ecosystem is challenging and usually 
      requires a priori assumptions to be made. We applied network theory, a holistic
      and exploratory method, to the most complete database of faunal distribution
      available on oceanic hydrothermal vents, environments which support fragmented
      and unstable ecosystems, to infer the processes driving their worldwide
      biogeography. Besides the identification of robust provinces, the network
      topology allowed us to identify preferential pathways that had hitherto been
      overlooked. These pathways are consistent with the previously proposed hypothesis
      of a role of plate tectonics in the biogeographical history of hydrothermal vent 
      communities. A possible ancestral position of the Western Pacific is also
      suggested for the first time. Finally, this work provides an innovative example
      of the potential of network tools to unravel the biogeographic history of faunal 
      assemblages and to supply comprehensive information for the conservation and
      management of biodiversity.
FAU - Moalic, Yann
AU  - Moalic Y
AD  - Departement Etude des Ecosystemes Profonds (DEEP), IFREMER, Institut Francais de 
      Recherche pour l'Exploitation de la Mer, Centre de Brest, BP70, 29280 Plouzane,
      France.
FAU - Desbruyeres, Daniel
AU  - Desbruyeres D
FAU - Duarte, Carlos M
AU  - Duarte CM
FAU - Rozenfeld, Alejandro F
AU  - Rozenfeld AF
FAU - Bachraty, Charleyne
AU  - Bachraty C
FAU - Arnaud-Haond, Sophie
AU  - Arnaud-Haond S
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110818
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - *Biological Evolution
MH  - Computer Simulation
MH  - Geography
MH  - *Hydrothermal Vents
MH  - Invertebrates/*classification/*physiology
MH  - *Models, Biological
MH  - Pacific Ocean
MH  - Phylogeny
EDAT- 2011/08/23 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/08/23 06:00
PHST- 2011/08/23 06:00 [entrez]
PHST- 2011/08/23 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr088 [pii]
AID - 10.1093/sysbio/syr088 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):127-37. doi: 10.1093/sysbio/syr088. Epub 2011 Aug 18.

PMID- 21840843
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20111222
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - Evaluating fossil calibrations for dating phylogenies in light of rates of
      molecular evolution: a comparison of three approaches.
PG  - 22-43
LID - 10.1093/sysbio/syr075 [doi]
AB  - Evolutionary and biogeographic studies increasingly rely on calibrated molecular 
      clocks to date key events. Although there has been significant recent progress in
      development of the techniques used for molecular dating, many issues remain. In
      particular, controversies abound over the appropriate use and placement of
      fossils for calibrating molecular clocks. Several methods have been proposed for 
      evaluating candidate fossils; however, few studies have compared the results
      obtained by different approaches. Moreover, no previous study has incorporated
      the effects of nucleotide saturation from different data types in the evaluation 
      of candidate fossils. In order to address these issues, we compared three
      approaches for evaluating fossil calibrations: the single-fossil cross-validation
      method of Near, Meylan, and Shaffer (2005. Assessing concordance of fossil
      calibration points in molecular clock studies: an example using turtles. Am. Nat.
      165:137-146), the empirical fossil coverage method of Marshall (2008. A simple
      method for bracketing absolute divergence times on molecular phylogenies using
      multiple fossil calibration points. Am. Nat. 171:726-742), and the Bayesian
      multicalibration method of Sanders and Lee (2007. Evaluating molecular clock
      calibrations using Bayesian analyses with soft and hard bounds. Biol. Lett.
      3:275-279) and explicitly incorporate the effects of data type (nuclear vs.
      mitochondrial DNA) for identifying the most reliable or congruent fossil
      calibrations. We used advanced (Caenophidian) snakes as a case study; however,
      our results are applicable to any taxonomic group with multiple candidate
      fossils, provided appropriate taxon sampling and sufficient molecular sequence
      data are available. We found that data type strongly influenced which fossil
      calibrations were identified as outliers, regardless of which method was used.
      Despite the use of complex partitioned models of sequence evolution and multiple 
      calibrations throughout the tree, saturation severely compressed basal branch
      lengths obtained from mitochondrial DNA compared with nuclear DNA. The effects of
      mitochondrial saturation were not ameliorated by analyzing a combined nuclear and
      mitochondrial data set. Although removing the third codon positions from the
      mitochondrial coding regions did not ameliorate saturation effects in the
      single-fossil cross-validations, it did in the Bayesian multicalibration
      analyses. Saturation significantly influenced the fossils that were selected as
      most reliable for all three methods evaluated. Our findings highlight the need to
      critically evaluate the fossils selected by data with different rates of
      nucleotide substitution and how data with different evolutionary rates affect the
      results of each method for evaluating fossils. Our empirical evaluation
      demonstrates that the advantages of using multiple independent fossil
      calibrations significantly outweigh any disadvantages.
FAU - Lukoschek, Vimoksalehi
AU  - Lukoschek V
AD  - Department of Ecology and Evolutionary Biology, University of California at
      Irvine, Irvine, CA 92697, USA. vimoksalehi.lukoschek@jcu.edu.au
FAU - Scott Keogh, J
AU  - Scott Keogh J
FAU - Avise, John C
AU  - Avise JC
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110813
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Calibration
MH  - Cell Nucleus/genetics
MH  - Classification/*methods
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - *Fossils
MH  - Genetic Speciation
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Snakes/anatomy &amp; histology/*classification/genetics
EDAT- 2011/08/16 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/08/16 06:00
PHST- 2011/08/16 06:00 [entrez]
PHST- 2011/08/16 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr075 [pii]
AID - 10.1093/sysbio/syr075 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):22-43. doi: 10.1093/sysbio/syr075. Epub 2011 Aug 13.

PMID- 21840842
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - Can deliberately incomplete gene sample augmentation improve a phylogeny estimate
      for the advanced moths and butterflies (Hexapoda: Lepidoptera)?
PG  - 782-96
LID - 10.1093/sysbio/syr079 [doi]
AB  - This paper addresses the question of whether one can economically improve the
      robustness of a molecular phylogeny estimate by increasing gene sampling in only 
      a subset of taxa, without having the analysis invalidated by artifacts arising
      from large blocks of missing data. Our case study stems from an ongoing effort to
      resolve poorly understood deeper relationships in the large clade Ditrysia ( &gt;
      150,000 species) of the insect order Lepidoptera (butterflies and moths). Seeking
      to remedy the overall weak support for deeper divergences in an initial study
      based on five nuclear genes (6.6 kb) in 123 exemplars, we nearly tripled the
      total gene sample (to 26 genes, 18.4 kb) but only in a third (41) of the taxa.
      The resulting partially augmented data matrix (45% intentionally missing data)
      consistently increased bootstrap support for groupings previously identified in
      the five-gene (nearly) complete matrix, while introducing no contradictory
      groupings of the kind that missing data have been predicted to produce. Our
      results add to growing evidence that data sets differing substantially in gene
      and taxon sampling can often be safely and profitably combined. The strongest
      overall support for nodes above the family level came from including all
      nucleotide changes, while partitioning sites into sets undergoing mostly
      nonsynonymous versus mostly synonymous change. In contrast, support for the
      deepest node for which any persuasive molecular evidence has yet emerged (78-85% 
      bootstrap) was weak or nonexistent unless synonymous change was entirely
      excluded, a result plausibly attributed to compositional heterogeneity. This node
      (Gelechioidea + Apoditrysia), tentatively proposed by previous authors on the
      basis of four morphological synapomorphies, is the first major subset of
      ditrysian superfamilies to receive strong statistical support in any phylogenetic
      study. A "more-genes-only" data set (41 taxax26 genes) also gave strong signal
      for a second deep grouping (Macrolepidoptera) that was obscured, but not strongly
      contradicted, in more taxon-rich analyses.
FAU - Cho, Soowon
AU  - Cho S
AD  - Department of Entomology, University of Maryland, College Park, MD 20742, USA.
FAU - Zwick, Andreas
AU  - Zwick A
FAU - Regier, Jerome C
AU  - Regier JC
FAU - Mitter, Charles
AU  - Mitter C
FAU - Cummings, Michael P
AU  - Cummings MP
FAU - Yao, Jianxiu
AU  - Yao J
FAU - Du, Zaile
AU  - Du Z
FAU - Zhao, Hong
AU  - Zhao H
FAU - Kawahara, Akito Y
AU  - Kawahara AY
FAU - Weller, Susan
AU  - Weller S
FAU - Davis, Donald R
AU  - Davis DR
FAU - Baixeras, Joaquin
AU  - Baixeras J
FAU - Brown, John W
AU  - Brown JW
FAU - Parr, Cynthia
AU  - Parr C
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110816
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Nucleotides)
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Genes, Insect/genetics
MH  - Genetic Heterogeneity
MH  - Lepidoptera/*classification/*genetics
MH  - Nucleotides/genetics
MH  - *Phylogeny
MH  - Statistics as Topic
PMC - PMC3193767
EDAT- 2011/08/16 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/08/16 06:00
PHST- 2011/08/16 06:00 [entrez]
PHST- 2011/08/16 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr079 [pii]
AID - 10.1093/sysbio/syr079 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):782-96. doi: 10.1093/sysbio/syr079. Epub 2011 Aug 16.

PMID- 21840841
OWN - NLM
STAT- MEDLINE
DCOM- 20120625
LR  - 20120216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 2
DP  - 2012 Mar
TI  - Species delimitation: inferring gaps in morphology across geography.
PG  - 179-94
LID - 10.1093/sysbio/syr084 [doi]
AB  - Species are commonly delimited on the basis of gaps in patterns of morphological 
      variation, but there seems to be little recent work on methods to objectively
      assess such gaps. Here, we introduce a statistical approach that uses
      measurements of continuous morphological characters and geographic variation in
      those characters to (i) measure the strength of the evidence for the existence of
      a gap in morphological variation between two hypothesized species and (ii)
      examine if a gap in morphological variation between two hypothesized species can 
      be explained by an alternative hypothesis of geographic variation within a
      species. This approach is based on recent developments in analyses of
      multivariate normal mixtures, estimates of multivariate tolerance regions, and
      principal coordinates of neighboring matrices. We demonstrate the application of 
      the approach by examining previously proposed hypotheses of species limits in the
      plant genus Escallonia. We discuss the main features of the method, including
      potential limitations, in relation to other approaches that use gaps in
      morphological variation as a criterion for species delimitation. The method we
      propose can help strengthen the link between the theory and practice of species
      delimitation by increasing the transparency and consistency of taxonomic
      decisions based on morphology, thus contributing to integrative approaches for
      species delimitation that consider morphological and geographic data on an equal 
      footing with other kinds of information.
FAU - Zapata, Felipe
AU  - Zapata F
AD  - Department of Biology, University of Missouri-St. Louis, One University
      Boulevard, St. Louis, MO 63121, USA. felipe.zapata@berkeley.edu
FAU - Jimenez, Ivan
AU  - Jimenez I
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110816
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Multivariate Analysis
MH  - Phylogeny
MH  - Phylogeography/*methods
MH  - Plants/anatomy &amp; histology/*classification
MH  - Species Specificity
EDAT- 2011/08/16 06:00
MHDA- 2012/06/26 06:00
CRDT- 2011/08/16 06:00
PHST- 2011/08/16 06:00 [entrez]
PHST- 2011/08/16 06:00 [pubmed]
PHST- 2012/06/26 06:00 [medline]
AID - syr084 [pii]
AID - 10.1093/sysbio/syr084 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Mar;61(2):179-94. doi: 10.1093/sysbio/syr084. Epub 2011 Aug 16.

PMID- 21835822
OWN - NLM
STAT- MEDLINE
DCOM- 20111220
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 5
DP  - 2011 Oct
TI  - Hermit to king, or hermit to all: multiple transitions to crab-like forms from
      hermit crab ancestors.
PG  - 616-29
LID - 10.1093/sysbio/syr063 [doi]
AB  - The Anomura presents the greatest degree of morphological disparity in the
      decapod Crustacea, with body forms ranging from the symmetrical and asymmetrical 
      hermit crabs to squat lobsters and king crabs. The phylogeny of the anomurans has
      been fraught with controversy. Recent debate has focused primarily on the
      phenomenon of carcinization, the evolution of crab-like form from a non-crab-like
      ancestor, focused chiefly on derivation of king crabs from asymmetrical hermit
      crabs--the "hermit to king" hypothesis. We show by phylogenetic analysis of five 
      nuclear protein-coding gene sequences that hermit crabs have a single origin, but
      surprisingly, that almost all other major clades and body forms within the
      Anomura, are derived from within the hermit crabs. The crab-like form and squat
      lobster form have each evolved at least twice from separate symmetrical hermit
      crab ancestors. In each case, a carcinization trend can be posited via a
      transition series from the initial symmetrical long-tailed hermit crab form,
      through the intermediate squat lobster or asymmetrical hermit crab form, to the
      final crab-like form. Adaptation to dextral shell habitation evolved at least
      twice, once in an exclusively deep-water clade and once in the common ancestor of
      all other asymmetrical hermit crabs (from which king crabs are derived). These
      remarkable cases of parallelism suggest considerable phenotypic flexibility
      within the hermit crab ground plan, with a general tendency toward carcinization.
      Rather than having a separate origin from other major clades, hermit crabs have
      given rise to most other major anomuran body types.
FAU - Tsang, Ling Ming
AU  - Tsang LM
AD  - Simon F.S. Li Marine Science Laboratory, School of Life Sciences, The Chinese
      University of Hong Kong, Shatin, Hong Kong.
FAU - Chan, Tin-Yam
AU  - Chan TY
FAU - Ahyong, Shane T
AU  - Ahyong ST
FAU - Chu, Ka Hou
AU  - Chu KH
LA  - eng
SI  - GENBANK/GU382856
SI  - GENBANK/GU382857
SI  - GENBANK/GU382858
SI  - GENBANK/GU382859
SI  - GENBANK/GU382860
SI  - GENBANK/GU382861
SI  - GENBANK/GU382862
SI  - GENBANK/GU382863
SI  - GENBANK/GU382864
SI  - GENBANK/GU382865
SI  - GENBANK/GU382866
SI  - GENBANK/GU382867
SI  - GENBANK/GU382868
SI  - GENBANK/GU382869
SI  - GENBANK/GU382870
SI  - GENBANK/GU382871
SI  - GENBANK/GU382872
SI  - GENBANK/GU382873
SI  - GENBANK/GU382874
SI  - GENBANK/GU382875
SI  - GENBANK/GU382876
SI  - GENBANK/GU382877
SI  - GENBANK/GU382878
SI  - GENBANK/GU382879
SI  - GENBANK/GU382880
SI  - GENBANK/GU382881
SI  - GENBANK/GU382882
SI  - GENBANK/GU382883
SI  - GENBANK/GU382884
SI  - GENBANK/GU382885
SI  - GENBANK/GU382886
SI  - GENBANK/GU382887
SI  - GENBANK/GU382888
SI  - GENBANK/GU382889
SI  - GENBANK/GU382890
SI  - GENBANK/GU382891
SI  - GENBANK/GU382892
SI  - GENBANK/GU382893
SI  - GENBANK/GU382894
SI  - GENBANK/GU382895
SI  - GENBANK/GU382896
SI  - GENBANK/GU382897
SI  - GENBANK/GU382898
SI  - GENBANK/GU382899
SI  - GENBANK/GU382900
SI  - GENBANK/GU382901
SI  - GENBANK/GU382902
SI  - GENBANK/GU382903
SI  - GENBANK/GU382904
SI  - GENBANK/GU382905
SI  - GENBANK/GU382906
SI  - GENBANK/GU382907
SI  - GENBANK/GU382908
SI  - GENBANK/GU382909
SI  - GENBANK/GU382910
SI  - GENBANK/GU382911
SI  - GENBANK/GU382912
SI  - GENBANK/GU382913
SI  - GENBANK/GU382914
SI  - GENBANK/GU382915
SI  - GENBANK/GU382916
SI  - GENBANK/GU382917
SI  - GENBANK/GU382918
SI  - GENBANK/GU382919
SI  - GENBANK/GU382920
SI  - GENBANK/GU382921
SI  - GENBANK/GU382922
SI  - GENBANK/GU382923
SI  - GENBANK/GU382924
SI  - GENBANK/GU382925
SI  - GENBANK/GU382926
SI  - GENBANK/GU382927
SI  - GENBANK/GU382928
SI  - GENBANK/GU382929
SI  - GENBANK/GU382930
SI  - GENBANK/GU382931
SI  - GENBANK/GU382932
SI  - GENBANK/GU382933
SI  - GENBANK/GU382934
SI  - GENBANK/GU382935
SI  - GENBANK/GU382936
SI  - GENBANK/GU382937
SI  - GENBANK/GU382938
SI  - GENBANK/GU382939
SI  - GENBANK/GU382940
SI  - GENBANK/GU382941
SI  - GENBANK/GU382942
SI  - GENBANK/GU382943
SI  - GENBANK/GU382944
SI  - GENBANK/GU382945
SI  - GENBANK/GU382946
SI  - GENBANK/GU382947
SI  - GENBANK/GU382948
SI  - GENBANK/GU382949
SI  - GENBANK/GU382950
SI  - GENBANK/GU382951
SI  - GENBANK/GU382952
SI  - GENBANK/GU382953
SI  - GENBANK/GU382954
SI  - GENBANK/GU382955
SI  - GENBANK/GU382956
SI  - GENBANK/GU382957
SI  - GENBANK/GU382958
SI  - GENBANK/GU382959
SI  - GENBANK/GU382960
SI  - GENBANK/GU382961
SI  - GENBANK/GU382962
SI  - GENBANK/GU382963
SI  - GENBANK/GU382964
SI  - GENBANK/GU382965
SI  - GENBANK/GU382966
SI  - GENBANK/GU382967
SI  - GENBANK/GU382968
SI  - GENBANK/GU382969
SI  - GENBANK/GU382970
SI  - GENBANK/GU382971
SI  - GENBANK/GU382972
SI  - GENBANK/GU382973
SI  - GENBANK/GU382974
SI  - GENBANK/GU382975
SI  - GENBANK/GU382976
SI  - GENBANK/GU382977
SI  - GENBANK/GU382978
SI  - GENBANK/GU382979
SI  - GENBANK/GU382980
SI  - GENBANK/GU382981
SI  - GENBANK/GU382982
SI  - GENBANK/GU382983
SI  - GENBANK/GU382984
SI  - GENBANK/GU382985
SI  - GENBANK/GU382986
SI  - GENBANK/GU382987
SI  - GENBANK/GU382988
SI  - GENBANK/GU382989
SI  - GENBANK/GU382990
SI  - GENBANK/GU382991
SI  - GENBANK/GU382992
SI  - GENBANK/GU382993
SI  - GENBANK/GU382994
SI  - GENBANK/GU382995
SI  - GENBANK/GU382996
SI  - GENBANK/GU382997
SI  - GENBANK/GU382998
SI  - GENBANK/GU382999
SI  - GENBANK/GU383000
SI  - GENBANK/GU383001
SI  - GENBANK/GU383002
SI  - GENBANK/GU383003
SI  - GENBANK/GU383004
SI  - GENBANK/GU383005
SI  - GENBANK/GU383006
SI  - GENBANK/GU383007
SI  - GENBANK/GU383008
SI  - GENBANK/GU383009
SI  - GENBANK/GU383010
SI  - GENBANK/GU383011
SI  - GENBANK/GU383012
SI  - GENBANK/GU383013
SI  - GENBANK/GU383014
SI  - GENBANK/GU383015
SI  - GENBANK/GU383016
SI  - GENBANK/GU383017
SI  - GENBANK/GU383018
SI  - GENBANK/GU383019
SI  - GENBANK/GU383020
SI  - GENBANK/GU383021
SI  - GENBANK/GU383022
SI  - GENBANK/GU383023
SI  - GENBANK/GU383024
SI  - GENBANK/GU383025
SI  - GENBANK/GU383026
SI  - GENBANK/GU383027
SI  - GENBANK/GU383028
SI  - GENBANK/GU383029
SI  - GENBANK/GU383030
SI  - GENBANK/GU383031
SI  - GENBANK/GU383032
SI  - GENBANK/GU383033
SI  - GENBANK/GU383035
SI  - GENBANK/GU383036
SI  - GENBANK/GU383037
SI  - GENBANK/GU383038
SI  - GENBANK/GU383039
SI  - GENBANK/GU383040
SI  - GENBANK/GU383041
SI  - GENBANK/GU383042
SI  - GENBANK/GU383043
SI  - GENBANK/GU383044
SI  - GENBANK/GU383045
SI  - GENBANK/GU383046
SI  - GENBANK/GU383047
SI  - GENBANK/GU383048
SI  - GENBANK/GU383049
SI  - GENBANK/GU383050
SI  - GENBANK/GU383051
SI  - GENBANK/GU383052
SI  - GENBANK/GU383053
SI  - GENBANK/GU383054
SI  - GENBANK/GU383055
SI  - GENBANK/GU383056
SI  - GENBANK/GU383057
SI  - GENBANK/GU383058
SI  - GENBANK/GU383059
SI  - GENBANK/GU383060
SI  - GENBANK/GU383061
SI  - GENBANK/GU383062
SI  - GENBANK/GU383063
SI  - GENBANK/GU383064
SI  - GENBANK/GU383065
SI  - GENBANK/GU383066
SI  - GENBANK/GU383067
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110810
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Anomura/*anatomy &amp; histology/classification/*genetics
MH  - *Biological Evolution
MH  - Cell Nucleus/genetics
MH  - Molecular Sequence Data
MH  - Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2011/08/13 06:00
MHDA- 2011/12/21 06:00
CRDT- 2011/08/13 06:00
PHST- 2011/08/13 06:00 [entrez]
PHST- 2011/08/13 06:00 [pubmed]
PHST- 2011/12/21 06:00 [medline]
AID - syr063 [pii]
AID - 10.1093/sysbio/syr063 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Oct;60(5):616-29. doi: 10.1093/sysbio/syr063. Epub 2011 Aug 10.

PMID- 21828084
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - Analysis of ratios in multivariate morphometry.
PG  - 813-25
LID - 10.1093/sysbio/syr061 [doi]
AB  - The analysis of ratios of body measurements is deeply ingrained in the taxonomic 
      literature. Whether for plants or animals, certain ratios are commonly indicated 
      in identification keys, diagnoses, and descriptions. They often provide the only 
      means for separation of cryptic species that mostly lack distinguishing
      qualitative characters. Additionally, they provide an obvious way to study
      differences in body proportions, as ratios reflect geometric shape differences.
      However, when it comes to multivariate analysis of body measurements, for
      instance, with linear discriminant analysis (LDA) or principal component analysis
      (PCA), interpretation using body ratios is difficult. Both techniques are
      commonly applied for separating similar taxa or for exploring the structure of
      variation, respectively, and require standardized raw or log-transformed
      variables as input. Here, we develop statistical procedures for the analysis of
      body ratios in a consistent multivariate statistical framework. In particular, we
      present algorithms adapted to LDA and PCA that allow the interpretation of
      numerical results in terms of body proportions. We first introduce a method
      called the "LDA ratio extractor," which reveals the best ratios for separation of
      two or more groups with the help of discriminant analysis. We also provide
      measures for deciding how much of the total differences between individuals or
      groups of individuals is due to size and how much is due to shape. The second
      method, a graphical tool called the "PCA ratio spectrum," aims at the
      interpretation of principal components in terms of body ratios. Based on a
      similar idea, the "allometry ratio spectrum" is developed which can be used for
      studying the allometric behavior of ratios. Because size can be defined in
      different ways, we discuss several concepts of size. Central to this discussion
      is Jolicoeur's multivariate generalization of the allometry equation, a concept
      that was derived only with a heuristic argument. Here we present a statistical
      derivation of the allometric size vector using the method of least squares. The
      application of the above methods is extensively demonstrated using published data
      sets from parasitic wasps and rock crabs.
FAU - Baur, Hannes
AU  - Baur H
AD  - Department of Invertebrates, Natural History Museum, Bernastrasse 15, Bern,
      Switzerland. hannes.baur@nmbe.ch
FAU - Leuenberger, Christoph
AU  - Leuenberger C
LA  - eng
PT  - Journal Article
DEP - 20110809
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Brachyura/anatomy &amp; histology/classification
MH  - Classification/*methods
MH  - Male
MH  - Multivariate Analysis
MH  - Species Specificity
MH  - Wasps/anatomy &amp; histology/classification
PMC - PMC3193766
EDAT- 2011/08/11 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/08/11 06:00
PHST- 2011/08/11 06:00 [entrez]
PHST- 2011/08/11 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr061 [pii]
AID - 10.1093/sysbio/syr061 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):813-25. doi: 10.1093/sysbio/syr061. Epub 2011 Aug 9.

PMID- 21828083
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20111222
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - Taxonomic structure of the fossil record is shaped by sampling bias.
PG  - 80-9
LID - 10.1093/sysbio/syr076 [doi]
AB  - Understanding biases that affect how species are partitioned into higher taxa is 
      critical for much of paleobiology, as higher taxa are commonly used to estimate
      species diversity through time. We test the validity of using higher taxa as a
      proxy for species diversity for the first time by examining one of the best
      fossil records we have, that of deep-sea microfossils. Using a new, taxonomically
      standardized, data set of coccolithophorid species and genera recorded from 143
      deep-sea drilling sites in the North Atlantic, Caribbean, and Mediterranean, we
      show that there is a two-stepped change in the ratio of species to genera over
      the last 150 myr. This change is highly unexpected and correlates strongly with
      changes in both the number of deep-sea sites yielding coccolithophorids that have
      been studied and with the number of taxonomists who have published on those
      sections. The same pattern is present in both structurally complex
      heterococcoliths and the simpler nannoliths, suggesting that increasing
      complexity is not the driving factor. As a stepped species-to-genus ratio exists 
      even after subsampling to standardize either the numbers of sites or numbers of
      papers, both factors must be contributing substantially to the observed pattern. 
      Although some limited biological signature from major extinction events can be
      recognized from changes in the species-to-genus ratio, the numbers of sites and
      the numbers of taxonomists combined explain some 82% of the observed variation
      over long periods of geological time. Such a strong correlation argues against
      using raw species-to-genus ratios to infer biological processes without taking
      sampling into account and suggests that higher taxa cannot be taken as unbiased
      proxies for species diversity.
FAU - Lloyd, Graeme T
AU  - Lloyd GT
AD  - Department of Palaeontology, The Natural History Museum, Cromwell Road, London
      SW75BD, UK.
FAU - Young, Jeremy R
AU  - Young JR
FAU - Smith, Andrew B
AU  - Smith AB
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110809
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Atlantic Ocean
MH  - *Biodiversity
MH  - Biological Evolution
MH  - Caribbean Region
MH  - Data Interpretation, Statistical
MH  - *Fossils
MH  - Haptophyta/*classification/growth &amp; development
MH  - Mediterranean Sea
MH  - Paleontology
MH  - Reproducibility of Results
EDAT- 2011/08/11 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/08/11 06:00
PHST- 2011/08/11 06:00 [entrez]
PHST- 2011/08/11 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr076 [pii]
AID - 10.1093/sysbio/syr076 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):80-9. doi: 10.1093/sysbio/syr076. Epub 2011 Aug 9.

PMID- 21828082
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20111222
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - Phylogenetic and coalescent strategies of species delimitation in snubnose
      darters (Percidae: Etheostoma).
PG  - 63-79
LID - 10.1093/sysbio/syr077 [doi]
AB  - The rapid accumulation of multilocus data sets has led to dramatic advances in
      methodologies for estimating evolutionary relationships among closely related
      species, but relatively less advancement has been made in methods for
      discriminating between competing species delimitation hypotheses. Multilocus data
      sets provide an advantage in testing species delimitation scenarios because they 
      offer a direct test of species monophyly and aid in the biological interpretation
      of such phenomena as allele-sharing and deep coalescent events. Most species tree
      estimation methods that are designed to analyze multilocus data sets require the 
      a priori assignment of individuals to species categories and therefore do not
      provide a strategy to directly test competing species delimitation scenarios. An 
      approach was recently proposed that utilizes a coalescent-based species tree
      estimation method to inform species delimitation decisions by comparing
      likelihood scores that measure the fit of gene trees within a given species tree.
      We use a multilocus nuclear and mitochondrial DNA sequence data set to both
      reexamine a recently proposed species delimitation scenario in the Etheostoma
      simoterum species complex and test the utility of species tree estimation methods
      in testing species delimitation hypotheses. Descriptions of species in the E.
      simoterum species complex of snubnose darters, a group of six teleost freshwater 
      fish species, are based largely on male nuptial coloration. Most of the putative 
      species are nonmonophyletic at every examined locus. Using a novel combination of
      Bayesian-estimated gene tree topologies, Bayesian phylogenetic species tree
      inferences, coalescent simulations, and examination of phenotypic variation, we
      assess the occurrence of shared alleles among species, and we propose that
      results from our analyses support a three-species rather than a six-species
      delimitation scenario in the E. simoterum complex. We found that comparing
      likelihood scores from the species tree estimation approach used across many
      potential delimitation scenarios resulted in a systematic bias toward
      over-splitting in the E. simoterum complex and failed to support a species
      delimitation scenario that was consistent with geography, phenotype, or any
      previous species delimitation hypothesis. Despite common expectations, we
      demonstrate that application of molecular approaches to species delimitation can 
      result in the recognition of fewer, instead of a larger number of species. In
      addition, our analyses highlight the importance of phenotypic character
      information in providing an independent assessment of alternative species
      delimitation hypotheses in the E. simoterum species complex.
FAU - Harrington, Richard C
AU  - Harrington RC
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT
      06520, USA. richard.harrington@yale.edu
FAU - Near, Thomas J
AU  - Near TJ
LA  - eng
SI  - GENBANK/JK497168
SI  - GENBANK/JK497169
SI  - GENBANK/JK497170
SI  - GENBANK/JK497171
SI  - GENBANK/JK497172
SI  - GENBANK/JK497173
SI  - GENBANK/JK497174
SI  - GENBANK/JK497175
SI  - GENBANK/JK497176
SI  - GENBANK/JK497177
SI  - GENBANK/JK497178
SI  - GENBANK/JK497179
SI  - GENBANK/JK497180
SI  - GENBANK/JK497181
SI  - GENBANK/JK497182
SI  - GENBANK/JK497183
SI  - GENBANK/JK497184
SI  - GENBANK/JK497185
SI  - GENBANK/JK497186
SI  - GENBANK/JK497187
SI  - GENBANK/JK497188
SI  - GENBANK/JK497189
SI  - GENBANK/JK497190
SI  - GENBANK/JK497191
SI  - GENBANK/JK497192
SI  - GENBANK/JK497193
SI  - GENBANK/JK497194
SI  - GENBANK/JK497195
SI  - GENBANK/JK497196
SI  - GENBANK/JK497197
SI  - GENBANK/JK497198
SI  - GENBANK/JK497199
SI  - GENBANK/JK497200
SI  - GENBANK/JK497201
SI  - GENBANK/JK497202
SI  - GENBANK/JK497203
SI  - GENBANK/JK497204
SI  - GENBANK/JK497205
SI  - GENBANK/JK497206
SI  - GENBANK/JK497207
SI  - GENBANK/JK497208
SI  - GENBANK/JK497209
SI  - GENBANK/JK497210
SI  - GENBANK/JK497211
SI  - GENBANK/JK497212
SI  - GENBANK/JK497213
SI  - GENBANK/JK497214
SI  - GENBANK/JK497215
SI  - GENBANK/JK497216
SI  - GENBANK/JK497217
SI  - GENBANK/JK497218
SI  - GENBANK/JK497219
SI  - GENBANK/JK497220
SI  - GENBANK/JK497221
SI  - GENBANK/JK497222
SI  - GENBANK/JK497223
SI  - GENBANK/JK497224
SI  - GENBANK/JK497225
SI  - GENBANK/JK497226
SI  - GENBANK/JK497227
SI  - GENBANK/JK497228
SI  - GENBANK/JK497229
SI  - GENBANK/JK497230
SI  - GENBANK/JK497231
SI  - GENBANK/JK497232
SI  - GENBANK/JK497233
SI  - GENBANK/JK497234
SI  - GENBANK/JK497235
SI  - GENBANK/JK497236
SI  - GENBANK/JK497237
SI  - GENBANK/JK497238
SI  - GENBANK/JK497239
SI  - GENBANK/JK497240
SI  - GENBANK/JK497241
SI  - GENBANK/JK497242
SI  - GENBANK/JK497243
SI  - GENBANK/JK497244
SI  - GENBANK/JK497245
SI  - GENBANK/JK497246
SI  - GENBANK/JK497247
SI  - GENBANK/JK497248
SI  - GENBANK/JK497249
SI  - GENBANK/JK497250
SI  - GENBANK/JK497251
SI  - GENBANK/JK497252
SI  - GENBANK/JK497253
SI  - GENBANK/JK497254
SI  - GENBANK/JK497255
SI  - GENBANK/JK497256
SI  - GENBANK/JK497257
SI  - GENBANK/JK497258
SI  - GENBANK/JK497259
SI  - GENBANK/JK497260
SI  - GENBANK/JK497261
SI  - GENBANK/JK497262
SI  - GENBANK/JK497263
SI  - GENBANK/JK497264
SI  - GENBANK/JK497265
SI  - GENBANK/JK497266
SI  - GENBANK/JK497267
SI  - GENBANK/JK497268
SI  - GENBANK/JK497269
SI  - GENBANK/JK497270
SI  - GENBANK/JK497271
SI  - GENBANK/JK497272
SI  - GENBANK/JK497273
SI  - GENBANK/JK497274
SI  - GENBANK/JK497275
SI  - GENBANK/JK497276
SI  - GENBANK/JK497277
SI  - GENBANK/JK497278
SI  - GENBANK/JK497279
SI  - GENBANK/JK497280
SI  - GENBANK/JK497281
SI  - GENBANK/JK497282
SI  - GENBANK/JK497283
SI  - GENBANK/JK497284
SI  - GENBANK/JK497285
SI  - GENBANK/JK497286
SI  - GENBANK/JK497287
SI  - GENBANK/JK497288
SI  - GENBANK/JK497289
SI  - GENBANK/JK497290
SI  - GENBANK/JK497291
SI  - GENBANK/JK497292
SI  - GENBANK/JK497293
SI  - GENBANK/JK497294
SI  - GENBANK/JK497295
SI  - GENBANK/JK497296
SI  - GENBANK/JK497297
SI  - GENBANK/JK497298
SI  - GENBANK/JK497299
SI  - GENBANK/JK497300
SI  - GENBANK/JK497301
SI  - GENBANK/JK497302
SI  - GENBANK/JK497303
SI  - GENBANK/JK497304
SI  - GENBANK/JK497305
SI  - GENBANK/JK497306
SI  - GENBANK/JK497307
SI  - GENBANK/JK497308
SI  - GENBANK/JK497309
SI  - GENBANK/JK497310
SI  - GENBANK/JK497311
SI  - GENBANK/JK497312
SI  - GENBANK/JK497313
SI  - GENBANK/JK497314
SI  - GENBANK/JK497315
SI  - GENBANK/JK497316
SI  - GENBANK/JK497317
SI  - GENBANK/JK497318
SI  - GENBANK/JK497319
SI  - GENBANK/JK497320
SI  - GENBANK/JK497321
SI  - GENBANK/JK497322
SI  - GENBANK/JK497323
SI  - GENBANK/JK497324
SI  - GENBANK/JK497325
SI  - GENBANK/JK497326
SI  - GENBANK/JK497327
SI  - GENBANK/JK497328
SI  - GENBANK/JK497329
SI  - GENBANK/JK497330
SI  - GENBANK/JK497331
SI  - GENBANK/JK497332
SI  - GENBANK/JK497333
SI  - GENBANK/JK497334
SI  - GENBANK/JK497335
SI  - GENBANK/JK497336
SI  - GENBANK/JK497337
SI  - GENBANK/JK497338
SI  - GENBANK/JK497339
SI  - GENBANK/JK497340
SI  - GENBANK/JK497341
SI  - GENBANK/JK497342
SI  - GENBANK/JK497343
SI  - GENBANK/JK497344
SI  - GENBANK/JK497345
SI  - GENBANK/JK497346
SI  - GENBANK/JK497347
SI  - GENBANK/JK497348
SI  - GENBANK/JK497349
SI  - GENBANK/JK497350
SI  - GENBANK/JK497351
SI  - GENBANK/JK497352
SI  - GENBANK/JK497353
SI  - GENBANK/JK497354
SI  - GENBANK/JK497355
SI  - GENBANK/JK497356
SI  - GENBANK/JK497357
SI  - GENBANK/JK497358
SI  - GENBANK/JK497359
SI  - GENBANK/JK497360
SI  - GENBANK/JK497361
SI  - GENBANK/JK497362
SI  - GENBANK/JK497363
SI  - GENBANK/JK497364
SI  - GENBANK/JK497365
SI  - GENBANK/JK497366
SI  - GENBANK/JK497367
SI  - GENBANK/JK497368
SI  - GENBANK/JK497369
SI  - GENBANK/JK497370
SI  - GENBANK/JK497371
SI  - GENBANK/JK497372
SI  - GENBANK/JK497373
SI  - GENBANK/JK497374
SI  - GENBANK/JK497375
SI  - GENBANK/JK497376
SI  - GENBANK/JK497377
SI  - GENBANK/JK497378
SI  - GENBANK/JK497379
SI  - GENBANK/JK497380
SI  - GENBANK/JK497381
SI  - GENBANK/JK497382
SI  - GENBANK/JK497383
SI  - GENBANK/JK497384
SI  - GENBANK/JK497385
SI  - GENBANK/JK497386
SI  - GENBANK/JK497387
SI  - GENBANK/JK497388
SI  - GENBANK/JK497389
SI  - GENBANK/JK497390
SI  - GENBANK/JK497391
SI  - GENBANK/JK497392
SI  - GENBANK/JK497393
SI  - GENBANK/JK497394
SI  - GENBANK/JK497395
SI  - GENBANK/JK497396
SI  - GENBANK/JK497397
SI  - GENBANK/JK497398
SI  - GENBANK/JK497399
SI  - GENBANK/JK497400
SI  - GENBANK/JK497401
SI  - GENBANK/JK497402
SI  - GENBANK/JK497403
SI  - GENBANK/JK497404
SI  - GENBANK/JK497405
SI  - GENBANK/JK497406
SI  - GENBANK/JK497407
SI  - GENBANK/JK497408
SI  - GENBANK/JK497409
SI  - GENBANK/JK497410
SI  - GENBANK/JK497411
SI  - GENBANK/JK497412
SI  - GENBANK/JK497413
SI  - GENBANK/JK497414
SI  - GENBANK/JK497415
SI  - GENBANK/JK497416
SI  - GENBANK/JK497417
SI  - GENBANK/JK497418
SI  - GENBANK/JK497419
SI  - GENBANK/JK497420
SI  - GENBANK/JK497421
SI  - GENBANK/JK497422
SI  - GENBANK/JK497423
SI  - GENBANK/JK497424
SI  - GENBANK/JK497425
SI  - GENBANK/JK497426
SI  - GENBANK/JK497427
SI  - GENBANK/JK497428
SI  - GENBANK/JK497429
SI  - GENBANK/JK497430
SI  - GENBANK/JK497431
SI  - GENBANK/JK497432
SI  - GENBANK/JK497433
SI  - GENBANK/JK497434
SI  - GENBANK/JK497435
SI  - GENBANK/JK497436
SI  - GENBANK/JK497437
SI  - GENBANK/JK497438
SI  - GENBANK/JK497439
SI  - GENBANK/JK497440
SI  - GENBANK/JK497441
SI  - GENBANK/JK497442
SI  - GENBANK/JK497443
SI  - GENBANK/JK497444
SI  - GENBANK/JK497445
SI  - GENBANK/JK497446
SI  - GENBANK/JK497447
SI  - GENBANK/JK497448
SI  - GENBANK/JK497449
SI  - GENBANK/JK497450
SI  - GENBANK/JK497451
SI  - GENBANK/JK497452
SI  - GENBANK/JK497453
SI  - GENBANK/JK497454
SI  - GENBANK/JK497455
SI  - GENBANK/JK497456
SI  - GENBANK/JK497457
SI  - GENBANK/JK497458
SI  - GENBANK/JK497459
SI  - GENBANK/JK497460
SI  - GENBANK/JK497461
SI  - GENBANK/JK497462
SI  - GENBANK/JK497463
SI  - GENBANK/JK497464
SI  - GENBANK/JK497465
SI  - GENBANK/JK497466
SI  - GENBANK/JK497467
SI  - GENBANK/JK497468
SI  - GENBANK/JK497469
SI  - GENBANK/JK497470
SI  - GENBANK/JK497471
SI  - GENBANK/JK497472
SI  - GENBANK/JK497473
SI  - GENBANK/JK497474
SI  - GENBANK/JK497475
SI  - GENBANK/JK497476
SI  - GENBANK/JK497477
SI  - GENBANK/JK497478
SI  - GENBANK/JK497479
SI  - GENBANK/JK497480
SI  - GENBANK/JK497481
SI  - GENBANK/JK497482
SI  - GENBANK/JK497483
SI  - GENBANK/JK497484
SI  - GENBANK/JK497485
SI  - GENBANK/JK497486
SI  - GENBANK/JK497487
SI  - GENBANK/JK497488
SI  - GENBANK/JK497489
SI  - GENBANK/JK497490
SI  - GENBANK/JK497491
SI  - GENBANK/JK497492
SI  - GENBANK/JK497493
SI  - GENBANK/JK497494
SI  - GENBANK/JK497495
SI  - GENBANK/JK497496
SI  - GENBANK/JK497497
SI  - GENBANK/JK497498
SI  - GENBANK/JK497499
SI  - GENBANK/JK497500
SI  - GENBANK/JK497501
SI  - GENBANK/JK497502
SI  - GENBANK/JK497503
SI  - GENBANK/JK497504
SI  - GENBANK/JK497505
SI  - GENBANK/JK497506
SI  - GENBANK/JK497507
SI  - GENBANK/JK497508
SI  - GENBANK/JK497509
SI  - GENBANK/JK497510
SI  - GENBANK/JK497511
SI  - GENBANK/JK497512
SI  - GENBANK/JK497513
SI  - GENBANK/JK497514
SI  - GENBANK/JK497515
SI  - GENBANK/JK497516
SI  - GENBANK/JK497517
SI  - GENBANK/JK497518
SI  - GENBANK/JK497519
SI  - GENBANK/JK497520
SI  - GENBANK/JK497521
SI  - GENBANK/JK497522
SI  - GENBANK/JK497523
SI  - GENBANK/JK497524
SI  - GENBANK/JK497525
SI  - GENBANK/JK497526
SI  - GENBANK/JK497527
SI  - GENBANK/JK497528
SI  - GENBANK/JK497529
SI  - GENBANK/JK497530
SI  - GENBANK/JK497531
SI  - GENBANK/JK497532
SI  - GENBANK/JK497533
SI  - GENBANK/JK497534
SI  - GENBANK/JK497535
SI  - GENBANK/JK497536
SI  - GENBANK/JK497537
SI  - GENBANK/JK497538
SI  - GENBANK/JK497539
SI  - GENBANK/JK497540
SI  - GENBANK/JK497541
SI  - GENBANK/JK497542
SI  - GENBANK/JK497543
SI  - GENBANK/JK497544
SI  - GENBANK/JK497545
SI  - GENBANK/JK497546
SI  - GENBANK/JK497547
SI  - GENBANK/JK497548
SI  - GENBANK/JK497549
SI  - GENBANK/JK497550
SI  - GENBANK/JK497551
SI  - GENBANK/JK497552
SI  - GENBANK/JK497553
SI  - GENBANK/JK497554
SI  - GENBANK/JK497555
SI  - GENBANK/JK497556
SI  - GENBANK/JK497557
SI  - GENBANK/JK497558
SI  - GENBANK/JK497559
SI  - GENBANK/JK497560
SI  - GENBANK/JK497561
SI  - GENBANK/JK497562
SI  - GENBANK/JK497563
SI  - GENBANK/JK497564
SI  - GENBANK/JK497565
SI  - GENBANK/JK497566
SI  - GENBANK/JK497567
SI  - GENBANK/JK497568
SI  - GENBANK/JK497569
SI  - GENBANK/JK497570
SI  - GENBANK/JK497571
SI  - GENBANK/JK497572
SI  - GENBANK/JK497573
SI  - GENBANK/JK497574
SI  - GENBANK/JK497575
SI  - GENBANK/JK497576
SI  - GENBANK/JK497577
SI  - GENBANK/JK497578
SI  - GENBANK/JK497579
SI  - GENBANK/JK497580
SI  - GENBANK/JK497581
SI  - GENBANK/JK497582
SI  - GENBANK/JK497583
SI  - GENBANK/JK497584
SI  - GENBANK/JK497585
SI  - GENBANK/JK497586
SI  - GENBANK/JK497587
SI  - GENBANK/JK497588
SI  - GENBANK/JK497589
SI  - GENBANK/JK497590
SI  - GENBANK/JK497591
SI  - GENBANK/JK497592
SI  - GENBANK/JK497593
SI  - GENBANK/JK497594
SI  - GENBANK/JK497595
SI  - GENBANK/JK497596
SI  - GENBANK/JK497597
SI  - GENBANK/JK497598
SI  - GENBANK/JK497599
SI  - GENBANK/JK497600
SI  - GENBANK/JK497601
SI  - GENBANK/JK497602
SI  - GENBANK/JK497603
SI  - GENBANK/JK497604
SI  - GENBANK/JK497605
SI  - GENBANK/JK497606
SI  - GENBANK/JK497607
SI  - GENBANK/JK497608
SI  - GENBANK/JK497609
SI  - GENBANK/JK497610
SI  - GENBANK/JK497611
SI  - GENBANK/JK497612
SI  - GENBANK/JK497613
SI  - GENBANK/JK497614
SI  - GENBANK/JK497615
SI  - GENBANK/JK497616
SI  - GENBANK/JK497617
SI  - GENBANK/JK497618
SI  - GENBANK/JK497619
SI  - GENBANK/JK497620
SI  - GENBANK/JK497621
SI  - GENBANK/JK497622
SI  - GENBANK/JK497623
SI  - GENBANK/JK497624
SI  - GENBANK/JK497625
SI  - GENBANK/JK497626
SI  - GENBANK/JK497627
SI  - GENBANK/JK497628
SI  - GENBANK/JK497629
SI  - GENBANK/JK497630
SI  - GENBANK/JK497631
SI  - GENBANK/JK497632
SI  - GENBANK/JK497633
SI  - GENBANK/JK497634
SI  - GENBANK/JK497635
SI  - GENBANK/JK497636
SI  - GENBANK/JK497637
SI  - GENBANK/JK497638
SI  - GENBANK/JK497639
SI  - GENBANK/JK497640
SI  - GENBANK/JK497641
SI  - GENBANK/JK497642
SI  - GENBANK/JK497643
SI  - GENBANK/JK497644
SI  - GENBANK/JK497645
SI  - GENBANK/JK497646
SI  - GENBANK/JK497647
SI  - GENBANK/JK497648
SI  - GENBANK/JK497649
SI  - GENBANK/JK497650
SI  - GENBANK/JK497651
SI  - GENBANK/JK497652
SI  - GENBANK/JK497653
SI  - GENBANK/JK497654
SI  - GENBANK/JK497655
SI  - GENBANK/JK497656
SI  - GENBANK/JK497657
SI  - GENBANK/JK497658
SI  - GENBANK/JK497659
SI  - GENBANK/JK497660
SI  - GENBANK/JK497661
SI  - GENBANK/JK497662
SI  - GENBANK/JK497663
SI  - GENBANK/JK497664
SI  - GENBANK/JK497665
SI  - GENBANK/JK497666
SI  - GENBANK/JK497667
SI  - GENBANK/JK497668
SI  - GENBANK/JK497669
SI  - GENBANK/JK497670
SI  - GENBANK/JK497671
SI  - GENBANK/JK497672
SI  - GENBANK/JK497673
SI  - GENBANK/JK497674
SI  - GENBANK/JK497675
SI  - GENBANK/JK497676
SI  - GENBANK/JK497677
SI  - GENBANK/JK497678
SI  - GENBANK/JK497679
SI  - GENBANK/JK497680
SI  - GENBANK/JK497681
SI  - GENBANK/JK497682
SI  - GENBANK/JK497683
SI  - GENBANK/JK497684
SI  - GENBANK/JK497685
SI  - GENBANK/JK497686
SI  - GENBANK/JK497687
SI  - GENBANK/JK497688
SI  - GENBANK/JK497689
SI  - GENBANK/JK497690
SI  - GENBANK/JK497691
SI  - GENBANK/JK497692
SI  - GENBANK/JK497693
SI  - GENBANK/JK497694
SI  - GENBANK/JK497695
SI  - GENBANK/JK497696
SI  - GENBANK/JK497697
SI  - GENBANK/JK497698
SI  - GENBANK/JK497699
SI  - GENBANK/JK497700
SI  - GENBANK/JK497701
SI  - GENBANK/JK497702
SI  - GENBANK/JK497703
SI  - GENBANK/JK497704
SI  - GENBANK/JK497705
SI  - GENBANK/JK497706
SI  - GENBANK/JK497707
SI  - GENBANK/JK497708
SI  - GENBANK/JK497709
SI  - GENBANK/JK497710
SI  - GENBANK/JK497711
SI  - GENBANK/JK497712
SI  - GENBANK/JK497713
SI  - GENBANK/JK497714
SI  - GENBANK/JK497715
SI  - GENBANK/JK497716
SI  - GENBANK/JK497717
SI  - GENBANK/JK497718
SI  - GENBANK/JK497719
SI  - GENBANK/JK497720
SI  - GENBANK/JK497721
SI  - GENBANK/JK497722
SI  - GENBANK/JK497723
SI  - GENBANK/JK497724
SI  - GENBANK/JK497725
SI  - GENBANK/JK497726
SI  - GENBANK/JK497727
SI  - GENBANK/JK497728
SI  - GENBANK/JK497729
SI  - GENBANK/JK497730
SI  - GENBANK/JK497731
SI  - GENBANK/JK497732
SI  - GENBANK/JK497733
SI  - GENBANK/JK497734
SI  - GENBANK/JK497735
SI  - GENBANK/JK497736
SI  - GENBANK/JK497737
SI  - GENBANK/JK497738
SI  - GENBANK/JK497739
SI  - GENBANK/JK497740
SI  - GENBANK/JK497741
SI  - GENBANK/JK497742
SI  - GENBANK/JK497743
SI  - GENBANK/JK497744
SI  - GENBANK/JK497745
SI  - GENBANK/JK497746
SI  - GENBANK/JK497747
SI  - GENBANK/JK497748
SI  - GENBANK/JK497749
SI  - GENBANK/JK497750
SI  - GENBANK/JK497751
SI  - GENBANK/JK497752
SI  - GENBANK/JK497753
SI  - GENBANK/JK497754
SI  - GENBANK/JK497755
SI  - GENBANK/JK497756
SI  - GENBANK/JK497757
SI  - GENBANK/JK497758
SI  - GENBANK/JK497759
SI  - GENBANK/JK497760
SI  - GENBANK/JK497761
SI  - GENBANK/JK497762
SI  - GENBANK/JK497763
SI  - GENBANK/JK497764
SI  - GENBANK/JK497765
SI  - GENBANK/JK497766
SI  - GENBANK/JK497767
SI  - GENBANK/JK497768
SI  - GENBANK/JK497769
SI  - GENBANK/JK497770
SI  - GENBANK/JK497771
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110809
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Cell Nucleus/genetics
MH  - Classification/*methods
MH  - Computer Simulation
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - Female
MH  - Likelihood Functions
MH  - Male
MH  - Molecular Sequence Data
MH  - Perches/anatomy &amp; histology/*classification/*genetics
MH  - Phylogeny
MH  - Phylogeography
MH  - Pigmentation
MH  - Polymerase Chain Reaction/veterinary
MH  - Population Density
MH  - Sequence Analysis, DNA
MH  - United States
EDAT- 2011/08/11 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/08/11 06:00
PHST- 2011/08/11 06:00 [entrez]
PHST- 2011/08/11 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr077 [pii]
AID - 10.1093/sysbio/syr077 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):63-79. doi: 10.1093/sysbio/syr077. Epub 2011 Aug 9.

PMID- 21828081
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20111222
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - Guided tree topology proposals for Bayesian phylogenetic inference.
PG  - 1-11
LID - 10.1093/sysbio/syr074 [doi]
AB  - Increasingly, large data sets pose a challenge for computationally intensive
      phylogenetic methods such as Bayesian Markov chain Monte Carlo (MCMC). Here, we
      investigate the performance of common MCMC proposal distributions in terms of
      median and variance of run time to convergence on 11 data sets. We introduce two 
      new Metropolized Gibbs Samplers for moving through "tree space." MCMC simulation 
      using these new proposals shows faster average run time and dramatically improved
      predictability in performance, with a 20-fold reduction in the variance of the
      time to estimate the posterior distribution to a given accuracy. We also
      introduce conditional clade probabilities and demonstrate that they provide a
      superior means of approximating tree topology posterior probabilities from
      samples recorded during MCMC.
FAU - Hohna, Sebastian
AU  - Hohna S
AD  - Department of Mathematics, Stockholm University, SE-10691 Stockholm, Sweden.
      hoehna@math.su.se
FAU - Drummond, Alexei J
AU  - Drummond AJ
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
DEP - 20110809
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - Markov Chains
MH  - Models, Genetic
MH  - Monte Carlo Method
MH  - Phylogeny
MH  - Probability
EDAT- 2011/08/11 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/08/11 06:00
PHST- 2011/08/11 06:00 [entrez]
PHST- 2011/08/11 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr074 [pii]
AID - 10.1093/sysbio/syr074 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):1-11. doi: 10.1093/sysbio/syr074. Epub 2011 Aug 9.

PMID- 21828080
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - How the worm got its pharynx: phylogeny, classification and Bayesian assessment
      of character evolution in Acoela.
PG  - 845-71
LID - 10.1093/sysbio/syr073 [doi]
AB  - Acoela are marine microscopic worms currently thought to be the sister taxon of
      all other bilaterians. Acoels have long been used as models in evolutionary
      scenarios, and generalized conclusions about acoel and bilaterian ancestral
      features are frequently drawn from studies of single acoel species. There is no
      extensive phylogenetic study of Acoela and the taxonomy of the 380 species is
      chaotic. Here we use two nuclear ribosomal genes and one mitochondrial gene in
      combination with 37 morphological characters in an analysis of 126 acoel
      terminals (about one-third of the described species) to estimate the phylogeny
      and character evolution of Acoela. We present an estimate of posterior
      probabilities for ancestral character states at 31 control nodes in the
      phylogeny. The overall reconstruction signal based on the shape of the posterior 
      distribution of character states was computed for all morphological characters
      and control nodes to assess how well these were reconstructed. The body-wall
      musculature appears more clearly reconstructed than the reproductive organs.
      Posterior similarity to the root was calculated by averaging the divergence
      between the posterior distributions at the nodes and the root over all
      morphological characters. Diopisthoporidae is the sister group to all other
      acoels and has the highest posterior similarity to the root. Convolutidae,
      including several "model" acoels, is most divergent. Finally, we present a
      phylogenetic classification of Acoela down to the family level where six previous
      family level taxa are synonymized.
FAU - Jondelius, Ulf
AU  - Jondelius U
AD  - Department of Invertebrate Zoology, Swedish Museum of Natural History, Stockholm,
      Sweden. ulf.jondelius@nrm.se
FAU - Wallberg, Andreas
AU  - Wallberg A
FAU - Hooge, Matthew
AU  - Hooge M
FAU - Raikova, Olga I
AU  - Raikova OI
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110809
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 18S)
RN  - 0 (RNA, Ribosomal, 28S)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - Classification
MH  - Electron Transport Complex IV/genetics
MH  - Pharynx/anatomy &amp; histology
MH  - *Phylogeny
MH  - RNA, Ribosomal, 18S/genetics
MH  - RNA, Ribosomal, 28S/genetics
MH  - Turbellaria/*anatomy &amp; histology/*classification/genetics
EDAT- 2011/08/11 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/08/11 06:00
PHST- 2011/08/11 06:00 [entrez]
PHST- 2011/08/11 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr073 [pii]
AID - 10.1093/sysbio/syr073 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):845-71. doi: 10.1093/sysbio/syr073. Epub 2011 Aug 9.

PMID- 21804094
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - Euclidean nature of phylogenetic distance matrices.
PG  - 826-32
LID - 10.1093/sysbio/syr066 [doi]
AB  - Phylogenies are fundamental to comparative biology as they help to identify
      independent events on which statistical tests rely. Two groups of phylogenetic
      comparative methods (PCMs) can be distinguished: those that take phylogenies into
      account by introducing explicit models of evolution and those that only consider 
      phylogenies as a statistical constraint and aim at partitioning trait values into
      a phylogenetic component (phylogenetic inertia) and one or multiple specific
      components related to adaptive evolution. The way phylogenetic information is
      incorporated into the PCMs depends on the method used. For the first group of
      methods, phylogenies are converted into variance-covariance matrices of traits
      following a given model of evolution such as Brownian motion (BM). For the second
      group of methods, phylogenies are converted into distance matrices that are
      subsequently transformed into Euclidean distances to perform principal coordinate
      analyses. Here, we show that simply taking the elementwise square root of a
      distance matrix extracted from a phylogenetic tree ensures having a Euclidean
      distance matrix. This is true for any type of distances between species
      (patristic or nodal) and also for trees harboring multifurcating nodes. Moreover,
      we illustrate that this simple transformation using the square root imposes less 
      geometric distortion than more complex transformations classically used in the
      literature such as the Cailliez method. Given the Euclidean nature of the
      elementwise square root of phylogenetic distance matrices, the positive
      semidefinitiveness of the phylogenetic variance-covariance matrix of a trait
      following a BM model, or related models of trait evolution, can be established.
      In that way, we build a bridge between the two groups of statistical methods
      widely used in comparative analysis. These results should be of great interest
      for ecologists and evolutionary biologists performing statistical analyses
      incorporating phylogenies.
FAU - de Vienne, Damien M
AU  - de Vienne DM
AD  - Departement Genetique et Ecologie Evolutives, unite Ecologie, Systematique et
      Evolution, Universite Paris-Sud 11, Orsay cedex, France. damien.de-vienne@crg.es
FAU - Aguileta, Gabriela
AU  - Aguileta G
FAU - Ollier, Sebastien
AU  - Ollier S
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110729
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Phylogeny
EDAT- 2011/08/02 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/08/02 06:00
PHST- 2011/08/02 06:00 [entrez]
PHST- 2011/08/02 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr066 [pii]
AID - 10.1093/sysbio/syr066 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):826-32. doi: 10.1093/sysbio/syr066. Epub 2011 Jul 29.

PMID- 21804093
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - A method for inferring the rate of evolution of homologous characters that can
      potentially improve phylogenetic inference, resolve deep divergence and correct
      systematic biases.
PG  - 833-44
LID - 10.1093/sysbio/syr064 [doi]
AB  - Current phylogenetic methods attempt to account for evolutionary rate variation
      across characters in a matrix. This is generally achieved by the use of
      sophisticated evolutionary models, combined with dense sampling of large numbers 
      of characters. However, systematic biases and superimposed substitutions make
      this task very difficult. Model adequacy can sometimes be achieved at the cost of
      adding large numbers of free parameters, with each parameter being optimized
      according to some criterion, resulting in increased computation times and large
      variances in the model estimates. In this study, we develop a simple approach
      that estimates the relative evolutionary rate of each homologous character. The
      method that we describe uses the similarity between characters as a proxy for
      evolutionary rate. In this article, we work on the premise that if the
      character-state distribution of a homologous character is similar to many other
      characters, then this character is likely to be relatively slowly evolving. If
      the character-state distribution of a homologous character is not similar to many
      or any of the rest of the characters in a data set, then it is likely to be the
      result of rapid evolution. We show that in some test cases, at least, the premise
      can hold and the inferences are robust. Importantly, the method does not use a
      "starting tree" to make the inference and therefore is tree independent. We
      demonstrate that this approach can work as well as a maximum likelihood (ML)
      approach, though the ML method needs to have a known phylogeny, or at least a
      very good estimate of that phylogeny. We then demonstrate some uses for this
      method of analysis, including the improvement in phylogeny reconstruction for
      both deep-level and recent relationships and overcoming systematic biases such as
      base composition bias. Furthermore, we compare this approach to two
      well-established methods for reweighting or removing characters. These other
      methods are tree-based and we show that they can be systematically biased. We
      feel this method can be useful for phylogeny reconstruction, understanding
      evolutionary rate variation, and for understanding selection variation on
      different characters.
FAU - Cummins, Carla A
AU  - Cummins CA
AD  - Molecular Evolution and Bioinformatics Unit, Department of Biology, National
      University of Ireland, Maynooth, County Kildare, Ireland.
FAU - McInerney, James O
AU  - McInerney JO
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110729
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bias
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Humans
MH  - *Phylogeny
MH  - Primates/classification/genetics
EDAT- 2011/08/02 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/08/02 06:00
PHST- 2011/08/02 06:00 [entrez]
PHST- 2011/08/02 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr064 [pii]
AID - 10.1093/sysbio/syr064 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):833-44. doi: 10.1093/sysbio/syr064. Epub 2011 Jul 29.

PMID- 21804092
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - Fast Bayesian choice of phylogenetic models: prospecting data augmentation-based 
      thermodynamic integration.
PG  - 881-7
LID - 10.1093/sysbio/syr065 [doi]
FAU - Rodrigue, Nicolas
AU  - Rodrigue N
AD  - Department of Biology and Center for Advanced Research in Environmental Genomics,
      University of Ottawa, 30 Marie Curie Pvt., Ottawa, ON, Canada.
FAU - Aris-Brosou, Stephane
AU  - Aris-Brosou S
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110729
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - *Phylogeny
EDAT- 2011/08/02 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/08/02 06:00
PHST- 2011/08/02 06:00 [entrez]
PHST- 2011/08/02 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr065 [pii]
AID - 10.1093/sysbio/syr065 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):881-7. doi: 10.1093/sysbio/syr065. Epub 2011 Jul 29.

PMID- 21775340
OWN - NLM
STAT- MEDLINE
DCOM- 20111220
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 5
DP  - 2011 Oct
TI  - Phylogeny and temporal diversification of darters (Percidae: Etheostomatinae).
PG  - 565-95
LID - 10.1093/sysbio/syr052 [doi]
AB  - Discussions aimed at resolution of the Tree of Life are most often focused on the
      interrelationships of major organismal lineages. In this study, we focus on the
      resolution of some of the most apical branches in the Tree of Life through
      exploration of the phylogenetic relationships of darters, a species-rich clade of
      North American freshwater fishes. With a near-complete taxon sampling of close to
      250 species, we aim to investigate strategies for efficient multilocus data
      sampling and the estimation of divergence times using relaxed-clock methods when 
      a clade lacks a fossil record. Our phylogenetic data set comprises a single
      mitochondrial DNA (mtDNA) gene and two nuclear genes sampled from 245 of the 248 
      darter species. This dense sampling allows us to determine if a modest amount of 
      nuclear DNA sequence data can resolve relationships among closely related animal 
      species. Darters lack a fossil record to provide age calibration priors in
      relaxed-clock analyses. Therefore, we use a near-complete species-sampled
      phylogeny of the perciform clade Centrarchidae, which has a rich fossil record,
      to assess two distinct strategies of external calibration in relaxed-clock
      divergence time estimates of darters: using ages inferred from the fossil record 
      and molecular evolutionary rate estimates. Comparison of Bayesian phylogenies
      inferred from mtDNA and nuclear genes reveals that heterospecific mtDNA is
      present in approximately 12.5% of all darter species. We identify three patterns 
      of mtDNA introgression in darters: proximal mtDNA transfer, which involves the
      transfer of mtDNA among extant and sympatric darter species, indeterminate
      introgression, which involves the transfer of mtDNA from a lineage that cannot be
      confidently identified because the introgressed haplotypes are not clearly
      referable to mtDNA haplotypes in any recognized species, and deep introgression, 
      which is characterized by species diversification within a recipient clade
      subsequent to the transfer of heterospecific mtDNA. The results of our analyses
      indicate that DNA sequences sampled from single-copy nuclear genes can provide
      appreciable phylogenetic resolution for closely related animal species. A
      well-resolved near-complete species-sampled phylogeny of darters was estimated
      with Bayesian methods using a concatenated mtDNA and nuclear gene data set with
      all identified heterospecific mtDNA haplotypes treated as missing data. The
      relaxed-clock analyses resulted in very similar posterior age estimates across
      the three sampled genes and methods of calibration and therefore offer a viable
      strategy for estimating divergence times for clades that lack a fossil record. In
      addition, an informative rank-free clade-based classification of darters that
      preserves the rich history of nomenclature in the group and provides formal
      taxonomic communication of darter clades was constructed using the mtDNA and
      nuclear gene phylogeny. On the whole, the appeal of mtDNA for phylogeny inference
      among closely related animal species is diminished by the observations of
      extensive mtDNA introgression and by finding appreciable phylogenetic signal in a
      modest sampling of nuclear genes in our phylogenetic analyses of darters.
FAU - Near, Thomas J
AU  - Near TJ
AD  - Department of Ecology and Evolutionary Biology and Peabody Museum of Natural
      History, Yale University, New Haven, CT 06520, USA. thomas.near@yale.edu
FAU - Bossu, Christen M
AU  - Bossu CM
FAU - Bradburd, Gideon S
AU  - Bradburd GS
FAU - Carlson, Rose L
AU  - Carlson RL
FAU - Harrington, Richard C
AU  - Harrington RC
FAU - Hollingsworth, Phillip R Jr
AU  - Hollingsworth PR Jr
FAU - Keck, Benjamin P
AU  - Keck BP
FAU - Etnier, David A
AU  - Etnier DA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110720
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Cytochrome b Group)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Ribosomal Proteins)
RN  - 0 (ribosomal protein S7)
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Cell Nucleus/genetics
MH  - Cytochrome b Group/genetics
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - Exons/genetics
MH  - Genes, RAG-1/genetics
MH  - Haplotypes
MH  - Hybridization, Genetic
MH  - Introns/genetics
MH  - Molecular Sequence Data
MH  - Perches/*classification/*genetics
MH  - Perciformes/classification/genetics
MH  - Phylogeny
MH  - Ribosomal Proteins/genetics
MH  - Sequence Analysis, DNA
EDAT- 2011/07/22 06:00
MHDA- 2011/12/21 06:00
CRDT- 2011/07/22 06:00
PHST- 2011/07/22 06:00 [entrez]
PHST- 2011/07/22 06:00 [pubmed]
PHST- 2011/12/21 06:00 [medline]
AID - syr052 [pii]
AID - 10.1093/sysbio/syr052 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Oct;60(5):565-95. doi: 10.1093/sysbio/syr052. Epub 2011 Jul 20.

PMID- 21712480
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - Nonrandom variation in within-species sample size and missing data in
      phylogenetic comparative studies.
PG  - 876-80
LID - 10.1093/sysbio/syr060 [doi]
FAU - Garamszegi, Laszlo Zsolt
AU  - Garamszegi LZ
AD  - Department of Evolutionary Ecology, Estacion Biologica de Donana-CSIC, c/ Americo
      Vespucio s/n, Seville, Spain.
FAU - Moller, Anders Pape
AU  - Moller AP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110628
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Data Interpretation, Statistical
MH  - Genetic Variation
MH  - Phylogeny
EDAT- 2011/06/30 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/06/30 06:00
PHST- 2011/06/30 06:00 [entrez]
PHST- 2011/06/30 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr060 [pii]
AID - 10.1093/sysbio/syr060 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):876-80. doi: 10.1093/sysbio/syr060. Epub 2011 Jun 28.

PMID- 21551126
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - A phylogenetic analysis of pygmy perches (Teleostei: Percichthyidae) with an
      assessment of the major historical influences on aquatic biogeography in southern
      Australia.
PG  - 797-812
LID - 10.1093/sysbio/syr042 [doi]
AB  - The biogeography of southern Australia is characterized by a repeated pattern of 
      relatedness between the biota of southwestern and southeastern Australia. Both
      areas possess a temperate climate but are separated by a vast arid region,
      currently lacking permanent freshwater habitats, which has become increasingly
      drier since about 15 Ma. Aquatic organisms have thus potentially remained
      isolated for a considerable time. Pygmy perches (Nannatherina and Nannoperca,
      Percichthyidae) provide an excellent scenario for investigating biogeographic
      relationships between southwestern and southeastern regions as multiple species
      occur on either side of Australia. This allows us to potentially differentiate
      between "Multiple Invasion" and "Endemic Speciation," the two major hypotheses
      proposed to account for current distributions. The first suggests that multiple
      east-west movements have occurred, whereas the second suggests a single east-west
      split, with current biodiversity in each region being reciprocally monophyletic. 
      Systematic relationships within this group were investigated with the
      mitochondrial cytochrome b gene; nuclear intron and exon sequences from S7, RAG1,
      and RAG2; and 53 allozyme loci. Our data supported the hypothesis of multiple
      movements across southern Australia based on a consistent lack of support for
      reciprocal monophyly of eastern and western species. This study appears to be the
      first example of an animal group displaying clear multiple east-west movement in 
      southern Australia, as all other aquatic and terrestrial fauna previously
      examined displayed a single east-west split. Despite a high degree of sympatry
      within each region, the only evidence for hybridization was found between
      Nannoperca australis and N. obscura, with the latter having its mitochondrial
      genome completely replaced by that of N. australis, with no evidence for nuclear 
      introgression. This is one of only a few confirmed examples of complete
      replacement of the mitochondrial genome in one species with that of another.
      Cryptic differentiation was also evident within the two most widespread species, 
      N. australis and N. vittata, indicating that these likely consist of multiple
      species. We also highlight the need for multiple molecular markers with different
      strengths in order to obtain a more robust phylogeny, despite problems resulting 
      from potential incongruences between data sets.
FAU - Unmack, Peter J
AU  - Unmack PJ
AD  - School of Life Sciences, Arizona State University, Tempe, AZ 85287-4501, USA.
FAU - Hammer, Michael P
AU  - Hammer MP
FAU - Adams, Mark
AU  - Adams M
FAU - Dowling, Thomas E
AU  - Dowling TE
LA  - eng
SI  - GENBANK/HQ713602
SI  - GENBANK/HQ713603
SI  - GENBANK/HQ713604
SI  - GENBANK/HQ713605
SI  - GENBANK/HQ713606
SI  - GENBANK/HQ713607
SI  - GENBANK/HQ713608
SI  - GENBANK/HQ713609
SI  - GENBANK/HQ713610
SI  - GENBANK/HQ713611
SI  - GENBANK/HQ713612
SI  - GENBANK/HQ713613
SI  - GENBANK/HQ713614
SI  - GENBANK/HQ713615
SI  - GENBANK/HQ713616
SI  - GENBANK/HQ713617
SI  - GENBANK/HQ713618
SI  - GENBANK/HQ713619
SI  - GENBANK/HQ713620
SI  - GENBANK/HQ713621
SI  - GENBANK/HQ713622
SI  - GENBANK/HQ713623
SI  - GENBANK/HQ713624
SI  - GENBANK/HQ713625
SI  - GENBANK/HQ713626
SI  - GENBANK/HQ713627
SI  - GENBANK/HQ713628
SI  - GENBANK/HQ713629
SI  - GENBANK/HQ713630
SI  - GENBANK/HQ713631
SI  - GENBANK/HQ713632
SI  - GENBANK/HQ713633
SI  - GENBANK/HQ713634
SI  - GENBANK/HQ713635
SI  - GENBANK/HQ713636
SI  - GENBANK/HQ713637
SI  - GENBANK/HQ713638
SI  - GENBANK/HQ713639
SI  - GENBANK/HQ713640
SI  - GENBANK/HQ713641
SI  - GENBANK/HQ713642
SI  - GENBANK/HQ713643
SI  - GENBANK/HQ713644
SI  - GENBANK/HQ713645
SI  - GENBANK/HQ713646
SI  - GENBANK/HQ713647
SI  - GENBANK/HQ713648
SI  - GENBANK/HQ713649
SI  - GENBANK/HQ713650
SI  - GENBANK/HQ713651
SI  - GENBANK/HQ713652
SI  - GENBANK/HQ713653
SI  - GENBANK/HQ713654
SI  - GENBANK/HQ713655
SI  - GENBANK/HQ713656
SI  - GENBANK/HQ713657
SI  - GENBANK/HQ713658
SI  - GENBANK/HQ713659
SI  - GENBANK/HQ713660
SI  - GENBANK/HQ713661
SI  - GENBANK/HQ713662
SI  - GENBANK/HQ713663
SI  - GENBANK/HQ713664
SI  - GENBANK/HQ713665
SI  - GENBANK/HQ713666
SI  - GENBANK/HQ713667
SI  - GENBANK/HQ713668
SI  - GENBANK/HQ713669
SI  - GENBANK/HQ713670
SI  - GENBANK/HQ713671
SI  - GENBANK/HQ713672
SI  - GENBANK/HQ713673
SI  - GENBANK/HQ713674
SI  - GENBANK/HQ713675
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110505
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Isoenzymes)
SB  - IM
MH  - Animals
MH  - Australia
MH  - Fresh Water
MH  - Genetic Variation
MH  - Isoenzymes/genetics
MH  - Molecular Sequence Data
MH  - Perches/*classification/*genetics
MH  - *Phylogeny
MH  - Phylogeography
EDAT- 2011/05/10 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/05/10 06:00
PHST- 2011/05/10 06:00 [entrez]
PHST- 2011/05/10 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr042 [pii]
AID - 10.1093/sysbio/syr042 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):797-812. doi: 10.1093/sysbio/syr042. Epub 2011 May 5.

PMID- 21551125
OWN - NLM
STAT- MEDLINE
DCOM- 20111020
LR  - 20110615
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 4
DP  - 2011 Jul
TI  - Phylogenetic inference of reciprocal effects between geographic range evolution
      and diversification.
PG  - 451-65
LID - 10.1093/sysbio/syr046 [doi]
AB  - Geographic characters--traits describing the spatial distribution of a
      species-may both affect and be affected by processes associated with lineage
      birth and death. This is potentially confounding to comparative analyses of
      species distributions because current models do not allow reciprocal interactions
      between the evolution of ranges and the growth of phylogenetic trees. Here, we
      introduce a likelihood-based approach to estimating region-dependent rates of
      speciation, extinction, and range evolution from a phylogeny, using a new model
      in which these processes are interdependent. We demonstrate the method with
      simulation tests that accurately recover parameters relating to the mode of
      speciation and source-sink dynamics. We then apply it to the evolution of habitat
      occupancy in Californian plant communities, where we find higher rates of
      speciation in chaparral than in forests and evidence for expanding habitat
      tolerances.
CI  - (c) The Author(s) 2011. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved.
FAU - Goldberg, Emma E
AU  - Goldberg EE
AD  - Department of Biological Sciences, University of Illinois at Chicago, Chicago, IL
      60607, USA. eeg@uic.edu
FAU - Lancaster, Lesley T
AU  - Lancaster LT
FAU - Ree, Richard H
AU  - Ree RH
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110505
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - California
MH  - Computer Simulation
MH  - Ecosystem
MH  - Extinction, Biological
MH  - Genetic Speciation
MH  - *Geography
MH  - *Phylogeny
MH  - Plants/classification/*genetics
EDAT- 2011/05/10 06:00
MHDA- 2011/10/21 06:00
CRDT- 2011/05/10 06:00
PHST- 2011/05/10 06:00 [entrez]
PHST- 2011/05/10 06:00 [pubmed]
PHST- 2011/10/21 06:00 [medline]
AID - syr046 [pii]
AID - 10.1093/sysbio/syr046 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jul;60(4):451-65. doi: 10.1093/sysbio/syr046. Epub 2011 May 5.

PMID- 21540409
OWN - NLM
STAT- MEDLINE
DCOM- 20111220
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 5
DP  - 2011 Oct
TI  - Survey of branch support methods demonstrates accuracy, power, and robustness of 
      fast likelihood-based approximation schemes.
PG  - 685-99
LID - 10.1093/sysbio/syr041 [doi]
AB  - Phylogenetic inference and evaluating support for inferred relationships is at
      the core of many studies testing evolutionary hypotheses. Despite the popularity 
      of nonparametric bootstrap frequencies and Bayesian posterior probabilities, the 
      interpretation of these measures of tree branch support remains a source of
      discussion. Furthermore, both methods are computationally expensive and become
      prohibitive for large data sets. Recent fast approximate likelihood-based
      measures of branch supports (approximate likelihood ratio test [aLRT] and
      Shimodaira-Hasegawa [SH]-aLRT) provide a compelling alternative to these slower
      conventional methods, offering not only speed advantages but also excellent
      levels of accuracy and power. Here we propose an additional method: a
      Bayesian-like transformation of aLRT (aBayes). Considering both probabilistic and
      frequentist frameworks, we compare the performance of the three fast
      likelihood-based methods with the standard bootstrap (SBS), the Bayesian
      approach, and the recently introduced rapid bootstrap. Our simulations and real
      data analyses show that with moderate model violations, all tests are
      sufficiently accurate, but aLRT and aBayes offer the highest statistical power
      and are very fast. With severe model violations aLRT, aBayes and Bayesian
      posteriors can produce elevated false-positive rates. With data sets for which
      such violation can be detected, we recommend using SH-aLRT, the nonparametric
      version of aLRT based on a procedure similar to the Shimodaira-Hasegawa tree
      selection. In general, the SBS seems to be excessively conservative and is much
      slower than our approximate likelihood-based methods.
FAU - Anisimova, Maria
AU  - Anisimova M
AD  - Department of Computer Science, Swiss Federal Institute of Technology (ETH),
      Zurich, Switzerland. maria.anisimova@inf.ethz.ch
FAU - Gil, Manuel
AU  - Gil M
FAU - Dufayard, Jean-Francois
AU  - Dufayard JF
FAU - Dessimoz, Christophe
AU  - Dessimoz C
FAU - Gascuel, Olivier
AU  - Gascuel O
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110503
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Amino Acids)
SB  - IM
MH  - Amino Acids/genetics
MH  - Animals
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Models, Statistical
MH  - Orchidaceae/genetics
MH  - Phylogeny
MH  - Statistics, Nonparametric
PMC - PMC3158332
EDAT- 2011/05/05 06:00
MHDA- 2011/12/21 06:00
CRDT- 2011/05/05 06:00
PHST- 2011/05/05 06:00 [entrez]
PHST- 2011/05/05 06:00 [pubmed]
PHST- 2011/12/21 06:00 [medline]
AID - syr041 [pii]
AID - 10.1093/sysbio/syr041 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Oct;60(5):685-99. doi: 10.1093/sysbio/syr041. Epub 2011 May 3.

PMID- 21540408
OWN - NLM
STAT- MEDLINE
DCOM- 20111020
LR  - 20110615
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 4
DP  - 2011 Jul
TI  - Divergence time estimation using fossils as terminal taxa and the origins of
      Lissamphibia.
PG  - 466-81
LID - 10.1093/sysbio/syr047 [doi]
AB  - Were molecular data available for extinct taxa, questions regarding the origins
      of many groups could be settled in short order. As this is not the case, various 
      strategies have been proposed to combine paleontological and neontological data
      sets. The use of fossil dates as node age calibrations for divergence time
      estimation from molecular phylogenies is commonplace. In addition, simulations
      suggest that the addition of morphological data from extinct taxa may improve
      phylogenetic estimation when combined with molecular data for extant species, and
      some studies have merged morphological and molecular data to estimate combined
      evidence phylogenies containing both extinct and extant taxa. However, few, if
      any, studies have attempted to estimate divergence times using phylogenies
      containing both fossil and living taxa sampled for both molecular and
      morphological data. Here, I infer both the phylogeny and the time of origin for
      Lissamphibia and a number of stem tetrapods using Bayesian methods based on a
      data set containing morphological data for extinct taxa, molecular data for
      extant taxa, and molecular and morphological data for a subset of extant taxa.
      The results suggest that Lissamphibia is monophyletic, nested within
      Lepospondyli, and originated in the late Carboniferous at the earliest. This
      research illustrates potential pitfalls for the use of fossils as post hoc age
      constraints on internal nodes and highlights the importance of explicit
      phylogenetic analysis of extinct taxa. These results suggest that the application
      of fossils as minima or maxima on molecular phylogenies should be supplemented or
      supplanted by combined evidence analyses whenever possible.
CI  - (c) The Author(s) 2011. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved.
FAU - Pyron, R Alexander
AU  - Pyron RA
AD  - Department of Ecology and Evolution, Stony Brook University, Stony Brook, NY
      11794-5245, USA. rpyron@colubroid.org
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110503
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amphibians/anatomy &amp; histology/*classification/genetics
MH  - Animals
MH  - Bayes Theorem
MH  - *Fossils
MH  - Genetic Speciation
MH  - *Phylogeny
MH  - Radiometric Dating
EDAT- 2011/05/05 06:00
MHDA- 2011/10/21 06:00
CRDT- 2011/05/05 06:00
PHST- 2011/05/05 06:00 [entrez]
PHST- 2011/05/05 06:00 [pubmed]
PHST- 2011/10/21 06:00 [medline]
AID - syr047 [pii]
AID - 10.1093/sysbio/syr047 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jul;60(4):466-81. doi: 10.1093/sysbio/syr047. Epub 2011 May 3.

PMID- 21527771
OWN - NLM
STAT- MEDLINE
DCOM- 20111020
LR  - 20110615
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 4
DP  - 2011 Jul
TI  - Examination of hemiplasy, homoplasy and phylogenetic discordance in chromosomal
      evolution of the Bovidae.
PG  - 439-50
LID - 10.1093/sysbio/syr045 [doi]
AB  - Robertsonian chromosomal fusions predominate in shaping the genomes of many
      species of Bovidae. These and other cytogenetic data (from 52 taxa representing
      51 species and 9 tribes of Bovidae) were (i) examined for usefulness in defining 
      phylogenetic relationships and (ii) subsequently mapped to a consensus tree based
      on mitochondrial and nuclear DNA gene sequences with divergence dates of the
      corresponding species calculated from cytochrome b sequences. This permitted
      persistence time estimates for the various rearrangements. The chromosomal data
      resulted in an unsupported higher-level topology, but with recognition of the
      monophyly of some genera and tribes within Bovidae. The distribution and temporal
      spread of character states on the species tree is suggestive of a restricted role
      for hemiplasy (the retention of an ancestral chromosomal polymorphism through
      multiple speciation events) and for introgression (resulting from secondary
      contact among taxa), processes that can potentially lead to phylogenetic
      discordance. We conclude that the most probable interpretation for these data is 
      that genuine karyotypic homoplasy predominates, but that hemiplasy (and/or
      introgression) is a realistic hypothesis for the observed patterns of several
      shared characters in Bovidae.
CI  - (c) The Author(s) 2011. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved.
FAU - Robinson, Terence J
AU  - Robinson TJ
AD  - Evolutionary Genomics Group, Department of Botany and Zoology, University of
      Stellenbosch, Private Bag X1, Matieland 7602, Stellenbosch, South Africa.
      tjr@sun.ac.za
FAU - Ropiquet, Anne
AU  - Ropiquet A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110428
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Chromosomes, Mammalian/*chemistry
MH  - DNA/chemistry
MH  - DNA, Mitochondrial/chemistry
MH  - *Evolution, Molecular
MH  - Genetic Speciation
MH  - *Phylogeny
MH  - Polymorphism, Genetic
MH  - Ruminants/classification/*genetics
MH  - Species Specificity
EDAT- 2011/04/30 06:00
MHDA- 2011/10/21 06:00
CRDT- 2011/04/30 06:00
PHST- 2011/04/30 06:00 [entrez]
PHST- 2011/04/30 06:00 [pubmed]
PHST- 2011/10/21 06:00 [medline]
AID - syr045 [pii]
AID - 10.1093/sysbio/syr045 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jul;60(4):439-50. doi: 10.1093/sysbio/syr045. Epub 2011 Apr 28.

PMID- 21525529
OWN - NLM
STAT- MEDLINE
DCOM- 20111020
LR  - 20110615
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 4
DP  - 2011 Jul
TI  - Echinoderm phylogeny including Xyloplax, a progenetic asteroid.
PG  - 420-38
LID - 10.1093/sysbio/syr044 [doi]
AB  - Reconstruction of the phylogeny of the five extant classes of the phylum
      Echinodermata has proven difficult. Results concerning higher-level taxonomic
      relationships among echinoderms are sensitive to the choice of analytical
      parameters and methods. Moreover, the proposal of a putative sixth class based on
      a small enigmatic disc-shaped echinoderm, Xyloplax, from the deep seas of the
      Bahamas and New Zealand in the 1980s further complicated the problem. Although
      clearly an echinoderm, Xyloplax did not have clear affinity among known groups.
      Using molecular sequence and developmental data from recently collected Xyloplax 
      adults and embryos, we show that rather than representing an ancient distinct
      lineage as implied by its status as a class, Xyloplax is simply a starfish that
      is closely related to the asteroid family Pterasteridae. Many members of the
      Pterasteridae and all Xyloplax inhabit deep or polar seas and brood young.
      Brooding pterasterids and Xyloplax hold their young in specialized adult chambers
      until the young reach an advanced juvenile stage after which they are released as
      free-living individuals. We hypothesize that the unique morphology of Xyloplax
      evolved via progenesis--the truncation of somatic growth at a juvenile body plan 
      but with gonadal growth to maturity. Although the overall phylogeny of extant
      echinoderms remains sensitive to the choice of analytical methods, the placement 
      of Xyloplax as sister to pterasterid asteroids is unequivocal. Based on this, we 
      argue that the proposed class and infraclass status of Xyloplax should be
      suppressed.
CI  - (c) The Author(s) 2011. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved.
FAU - Janies, Daniel A
AU  - Janies DA
AD  - Department of Biomedical Informatics, The Ohio State University, 3190 Graves
      Hall, 333 West 10th Avenue, Columbus, OH 43210, USA. Daniel.Janies@osumc.edu
FAU - Voight, Janet R
AU  - Voight JR
FAU - Daly, Marymegan
AU  - Daly M
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110427
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amino Acid Substitution
MH  - Animals
MH  - Echinodermata/anatomy &amp; histology/*classification/genetics
MH  - Embryo, Nonmammalian/anatomy &amp; histology
MH  - INDEL Mutation
MH  - Metamorphosis, Biological
MH  - *Phylogeny
MH  - Starfish/anatomy &amp; histology/classification/genetics
EDAT- 2011/04/29 06:00
MHDA- 2011/10/21 06:00
CRDT- 2011/04/29 06:00
PHST- 2011/04/29 06:00 [entrez]
PHST- 2011/04/29 06:00 [pubmed]
PHST- 2011/10/21 06:00 [medline]
AID - syr044 [pii]
AID - 10.1093/sysbio/syr044 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jul;60(4):420-38. doi: 10.1093/sysbio/syr044. Epub 2011 Apr 27.

PMID- 21486744
OWN - NLM
STAT- MEDLINE
DCOM- 20111220
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 5
DP  - 2011 Oct
TI  - Improved least squares topology testing and estimation.
PG  - 668-75
LID - 10.1093/sysbio/syr028 [doi]
AB  - Generalized least squares (GLS) methods provide a relatively fast means of
      constructing a confidence set of topologies. Because they utilize information
      about the covariances between distances, it is reasonable to expect additional
      efficiency in estimation and confidence set construction relative to other least 
      squares (LS) methods. Difficulties have been found to arise in a number of
      practical settings due to estimates of covariance matrices being ill conditioned 
      or even noninvertible. We present here new ways of estimating the covariance
      matrices for distances that are much more likely to be positive definite, as the 
      actual covariance matrices are. A thorough investigation of performance is also
      conducted. An alternative to GLS that has been proposed for constructing
      confidence sets of topologies is weighted least squares (WLS). As currently
      implemented, this approach is equivalent to the use of GLS but with covariances
      set to zero rather than being estimated. In effect, this approach assumes
      normality of the estimated distances and zero covariances. As the results here
      illustrate, this assumption leads to poor performance. A 95% confidence set is
      almost certain to contain the true topology but will contain many more topologies
      than are needed. On the other hand, the results here also indicate that, among LS
      methods, WLS performs quite well at estimating the correct topology. It turns out
      to be possible to improve the performance of WLS for confidence set construction 
      through a relatively inexpensive normal parametric bootstrap that utilizes the
      same variances and covariances of GLS. The resulting procedure is shown to
      perform at least as well as GLS and thus provides a reasonable alternative in
      cases where covariance matrices are ill conditioned.
FAU - Susko, Edward
AU  - Susko E
AD  - Department of Mathematics and Statistics, Dalhousie University, Halifax, Nova
      Scotia, Canada B3H3J5. susko@mathstat.dal.ca
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110411
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - *Least-Squares Analysis
MH  - *Models, Theoretical
EDAT- 2011/04/14 06:00
MHDA- 2011/12/21 06:00
CRDT- 2011/04/14 06:00
PHST- 2011/04/14 06:00 [entrez]
PHST- 2011/04/14 06:00 [pubmed]
PHST- 2011/12/21 06:00 [medline]
AID - syr028 [pii]
AID - 10.1093/sysbio/syr028 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Oct;60(5):668-75. doi: 10.1093/sysbio/syr028. Epub 2011 Apr 11.

PMID- 21482553
OWN - NLM
STAT- MEDLINE
DCOM- 20120406
LR  - 20111222
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 61
IP  - 1
DP  - 2012 Jan
TI  - Determining species boundaries in a world full of rarity: singletons, species
      delimitation methods.
PG  - 165-9
LID - 10.1093/sysbio/syr030 [doi]
FAU - Lim, Gwynne S
AU  - Lim GS
AD  - Department of Biological Sciences, National University of Singapore, Science
      Drive 4, Singapore 117543, Singapore.
FAU - Balke, Michael
AU  - Balke M
FAU - Meier, Rudolf
AU  - Meier R
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110411
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biodiversity
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Genetic Techniques
MH  - Invertebrates/*classification
MH  - Models, Biological
MH  - Plants/*classification
MH  - Population Density
MH  - Vertebrates/*classification
EDAT- 2011/04/13 06:00
MHDA- 2012/04/07 06:00
CRDT- 2011/04/13 06:00
PHST- 2011/04/13 06:00 [entrez]
PHST- 2011/04/13 06:00 [pubmed]
PHST- 2012/04/07 06:00 [medline]
AID - syr030 [pii]
AID - 10.1093/sysbio/syr030 [doi]
PST - ppublish
SO  - Syst Biol. 2012 Jan;61(1):165-9. doi: 10.1093/sysbio/syr030. Epub 2011 Apr 11.

PMID- 21482552
OWN - NLM
STAT- MEDLINE
DCOM- 20111220
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 5
DP  - 2011 Oct
TI  - Simulating trees with a fixed number of extant species.
PG  - 676-84
LID - 10.1093/sysbio/syr029 [doi]
AB  - In this paper, I develop efficient tools to simulate trees with a fixed number of
      extant species. The tools are provided in my open source R-package TreeSim
      available on CRAN. The new model presented here is a constant rate birth-death
      process with mass extinction and/or rate shift events at arbitrarily fixed times 
      1) before the present or 2) after the origin. The simulation approach for case
      (2) can also be used to simulate under more general models with fixed events
      after the origin. I use the developed simulation tools for showing that a mass
      extinction event cannot be distinguished from a model with constant speciation
      and extinction rates interrupted by a phase of stasis based on trees consisting
      of only extant species. However, once we distinguish between mass extinction and 
      period of stasis based on paleontological data, fast simulations of trees with a 
      fixed number of species allow inference of speciation and extinction rates using 
      approximate Bayesian computation and allow for robustness analysis once maximum
      likelihood parameter estimations are available.
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Institut fur Integrative Biologie, ETH Zurich, Universitatsstr. 16, 8092 Zurich, 
      Switzerland. tanja.stadler@env.ethz.ch
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110411
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Biodiversity
MH  - *Biological Evolution
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - *Extinction, Biological
MH  - Genetic Speciation
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Models, Statistical
MH  - Paleontology
EDAT- 2011/04/13 06:00
MHDA- 2011/12/21 06:00
CRDT- 2011/04/13 06:00
PHST- 2011/04/13 06:00 [entrez]
PHST- 2011/04/13 06:00 [pubmed]
PHST- 2011/12/21 06:00 [medline]
AID - syr029 [pii]
AID - 10.1093/sysbio/syr029 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Oct;60(5):676-84. doi: 10.1093/sysbio/syr029. Epub 2011 Apr 11.

PMID- 21471560
OWN - NLM
STAT- MEDLINE
DCOM- 20111020
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 4
DP  - 2011 Jul
TI  - Bayes estimators for phylogenetic reconstruction.
PG  - 528-40
LID - 10.1093/sysbio/syr021 [doi]
AB  - Tree reconstruction methods are often judged by their accuracy, measured by how
      close they get to the true tree. Yet, most reconstruction methods like maximum
      likelihood (ML) do not explicitly maximize this accuracy. To address this
      problem, we propose a Bayesian solution. Given tree samples, we propose finding
      the tree estimate that is closest on average to the samples. This "median" tree
      is known as the Bayes estimator (BE). The BE literally maximizes posterior
      expected accuracy, measured in terms of closeness (distance) to the true tree. We
      discuss a unified framework of BE trees, focusing especially on tree distances
      that are expressible as squared euclidean distances. Notable examples include
      Robinson-Foulds (RF) distance, quartet distance, and squared path difference.
      Using both simulated and real data, we show that BEs can be estimated in practice
      by hill-climbing. In our simulation, we find that BEs tend to be closer to the
      true tree, compared with ML and neighbor joining. In particular, the BE under
      squared path difference tends to perform well in terms of both path difference
      and RF distances.
CI  - (c) The Author(s) 2011. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved.
FAU - Huggins, P M
AU  - Huggins PM
AD  - Lane Center for Computational Biology, Carnegie Mellon University, Mellon
      Institute Building, 4400 Fifth Avenue, Pittsburgh, PA 15213, USA.
FAU - Li, W
AU  - Li W
FAU - Haws, D
AU  - Haws D
FAU - Friedrich, T
AU  - Friedrich T
FAU - Liu, J
AU  - Liu J
FAU - Yoshida, R
AU  - Yoshida R
LA  - eng
GR  - R01 GM086888/GM/NIGMS NIH HHS/United States
GR  - 1R01GM086888-01/GM/NIGMS NIH HHS/United States
GR  - 5R01GM086888-02/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
DEP - 20110406
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Computer Simulation
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Software
MH  - Urodela/classification/genetics
PMC - PMC3114872
EDAT- 2011/04/08 06:00
MHDA- 2011/10/21 06:00
CRDT- 2011/04/08 06:00
PHST- 2011/04/08 06:00 [entrez]
PHST- 2011/04/08 06:00 [pubmed]
PHST- 2011/10/21 06:00 [medline]
AID - syr021 [pii]
AID - 10.1093/sysbio/syr021 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jul;60(4):528-40. doi: 10.1093/sysbio/syr021. Epub 2011 Apr 6.

PMID- 21471308
OWN - NLM
STAT- MEDLINE
DCOM- 20111220
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 5
DP  - 2011 Oct
TI  - Fuzzy chorotypes as a conceptual tool to improve insight into biogeographic
      patterns.
PG  - 645-60
LID - 10.1093/sysbio/syr026 [doi]
AB  - Chorotypes--statistically significant groups of coincident distribution
      areas--constitute biogeographic units that are fuzzy by nature. This quality has 
      been referred to in the literature but has not been analyzed in depth or
      methodologically developed. The present work redefines chorotypes as fuzzy sets
      from a pragmatic perspective and basically focuses on the methodological and
      interpretative implications of this approach. The amphibian fauna in the Iberian 
      Peninsula was used as an example to explore the fuzzy nature of chorotypes. The
      method on which this article is based is a widely used technique to define
      chorotypes. This method involves the fuzziness that is inherent to the
      identification between degree of similarity and degree of membership and includes
      a probabilistic analysis of the classification for the objective delimitation of 
      chorotypes. The main innovation of this paper is a procedure to analyze
      chorotypes as fuzzy biogeographic units. A set of fuzzy parameters to deal with
      the biogeographic interpretation of fuzzy chorotypes is also described. A
      computer program has been developed and is freely available. History may be
      related to the degree of fuzziness of chorotypes. In our example, with amphibian 
      distributions in Iberia, less fuzzy chorotypes could have a historical
      explanation, and the internal fuzziness of chorotypes increases with their
      distance to hypothetical Pleistocene refugia.
FAU - Olivero, Jesus
AU  - Olivero J
AD  - Grupo de Biogeografia, Diversidad y Conservacion, Departamento de Biologia
      Animal, Facultad de Ciencias, Universidad de Malaga, 29071 Malaga, Spain.
      jesusolivero@uma.es
FAU - Real, Raimundo
AU  - Real R
FAU - Marquez, Ana L
AU  - Marquez AL
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Validation Studies
DEP - 20110406
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Amphibians/classification/*physiology
MH  - Animals
MH  - Biodiversity
MH  - *Ecosystem
MH  - *Fuzzy Logic
MH  - Geography/*methods
MH  - Portugal
MH  - Spain
EDAT- 2011/04/08 06:00
MHDA- 2011/12/21 06:00
CRDT- 2011/04/08 06:00
PHST- 2011/04/08 06:00 [entrez]
PHST- 2011/04/08 06:00 [pubmed]
PHST- 2011/12/21 06:00 [medline]
AID - syr026 [pii]
AID - 10.1093/sysbio/syr026 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Oct;60(5):645-60. doi: 10.1093/sysbio/syr026. Epub 2011 Apr 6.

PMID- 21471307
OWN - NLM
STAT- MEDLINE
DCOM- 20120103
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 6
DP  - 2011 Dec
TI  - The parameters of the Barry and Hartigan general Markov model are statistically
      nonidentifiable.
PG  - 872-5
LID - 10.1093/sysbio/syr034 [doi]
FAU - Zou, Liwen
AU  - Zou L
AD  - Department of Mathematics and Statistics, Centre for Comparative Genomics and
      Evolutionary Bioinformatics, Dalhousie University, Halifax, Nova Scotia, Canada.
FAU - Susko, Edward
AU  - Susko E
FAU - Field, Chris
AU  - Field C
FAU - Roger, Andrew J
AU  - Roger AJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110406
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Models, Genetic
MH  - Phylogeny
EDAT- 2011/04/08 06:00
MHDA- 2012/01/04 06:00
CRDT- 2011/04/08 06:00
PHST- 2011/04/08 06:00 [entrez]
PHST- 2011/04/08 06:00 [pubmed]
PHST- 2012/01/04 06:00 [medline]
AID - syr034 [pii]
AID - 10.1093/sysbio/syr034 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Dec;60(6):872-5. doi: 10.1093/sysbio/syr034. Epub 2011 Apr 6.

PMID- 21471306
OWN - NLM
STAT- MEDLINE
DCOM- 20111020
LR  - 20110615
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 4
DP  - 2011 Jul
TI  - Hybridization, mitochondrial DNA phylogeography, and prediction of the early
      stages of reproductive isolation: lessons from New Zealand cicadas (genus
      Kikihia).
PG  - 482-502
LID - 10.1093/sysbio/syr017 [doi]
AB  - One of the major tenets of the modern synthesis is that genetic differentiation
      among subpopulations is translated over time into genetic differentiation among
      species. Phylogeographic exploration is therefore essential to the study of
      speciation because it can reveal the presence of subpopulations that may go on to
      become species or that may already represent cryptic species. Acoustic
      species-specific mating signals provide a significant advantage for the
      recognition of cryptic or incipient species. Because the majority of species do
      not have such easily recognized premating signals, data from acoustically
      signaling species can serve as a valuable heuristic tool. Acoustic signals are
      also convenient tools for recognizing hybridization events. Here, we demonstrate 
      that evidence of hybridization in the form of intermediate song phenotypes is
      present in many contact zones between species of the New Zealand grass cicadas of
      the Kikihia muta species complex and that recurring mitochondrial DNA (mtDNA)
      introgression has created misleading patterns that make it difficult to identify 
      certain taxa using song or mtDNA alone. In one case, introgression appears to
      have occurred between allopatric taxa by dispersal of introgressed populations of
      an intermediary species ("hybridization by proxy"). We also present a comparison 
      of mtDNA-tree- and song-based taxonomies obtained for the K. muta complex. We
      find that 12 mtDNA candidate species are identified using shifts in phylogenetic 
      branching rate found by a single-threshold mixed Yule-coalescent lineage model,
      while only 7 candidate species are identified using songs. Results from the
      Yule-coalescent model are dependent on factors such as the number of modeled
      thresholds and the inclusion of duplicate haplotypes. Genetic distances within
      song species reach a maximum at about 0.028 substitutions/site when likely cases 
      of hybridization and introgression are excluded. Large genetic breaks or "gaps"
      are not observed between some northern (warmer climate) song clades, possibly
      because climate-induced bottlenecks have been less severe. These results support 
      ongoing calls for multimarker genetic studies as well as "integrative taxonomy"
      that combines information from multiple character sources, including behavior,
      ecology, geography, and morphology.
CI  - (c) The Author(s) 2011. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved.
FAU - Marshall, David C
AU  - Marshall DC
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75
      North Eagleville Road, Storrs, CT 06269, USA. david.marshall@uconn.edu
FAU - Hill, Kathy B R
AU  - Hill KB
FAU - Cooley, John R
AU  - Cooley JR
FAU - Simon, Chris
AU  - Simon C
LA  - eng
SI  - GENBANK/EF051374
SI  - GENBANK/EF051382
SI  - GENBANK/EF051383
SI  - GENBANK/EF051384
SI  - GENBANK/EF051388
SI  - GENBANK/EF051390
SI  - GENBANK/EF051402
SI  - GENBANK/EF051403
SI  - GENBANK/EF051405
SI  - GENBANK/EF051413
SI  - GENBANK/EF051414
SI  - GENBANK/EF051415
SI  - GENBANK/EU717570
SI  - GENBANK/EU717578
SI  - GENBANK/EU717579
SI  - GENBANK/EU717584
SI  - GENBANK/EU717585
SI  - GENBANK/EU717586
SI  - GENBANK/EU717587
SI  - GENBANK/EU717589
SI  - GENBANK/EU717590
SI  - GENBANK/EU717597
SI  - GENBANK/EU717598
SI  - GENBANK/EU717603
SI  - GENBANK/EU717604
SI  - GENBANK/EU717605
SI  - GENBANK/EU717606
SI  - GENBANK/HQ830560
SI  - GENBANK/HQ830561
SI  - GENBANK/HQ830562
SI  - GENBANK/HQ830563
SI  - GENBANK/HQ830564
SI  - GENBANK/HQ830565
SI  - GENBANK/HQ830566
SI  - GENBANK/HQ830567
SI  - GENBANK/HQ830568
SI  - GENBANK/HQ830569
SI  - GENBANK/HQ830570
SI  - GENBANK/HQ830571
SI  - GENBANK/HQ830572
SI  - GENBANK/HQ830573
SI  - GENBANK/HQ830574
SI  - GENBANK/HQ830575
SI  - GENBANK/HQ830576
SI  - GENBANK/HQ830577
SI  - GENBANK/HQ830578
SI  - GENBANK/HQ830579
SI  - GENBANK/HQ830580
SI  - GENBANK/HQ830581
SI  - GENBANK/HQ830582
SI  - GENBANK/HQ830583
SI  - GENBANK/HQ830584
SI  - GENBANK/HQ830585
SI  - GENBANK/HQ830586
SI  - GENBANK/HQ830587
SI  - GENBANK/HQ830588
SI  - GENBANK/HQ830589
SI  - GENBANK/HQ830590
SI  - GENBANK/HQ830591
SI  - GENBANK/HQ830592
SI  - GENBANK/HQ830593
SI  - GENBANK/HQ830594
SI  - GENBANK/HQ830595
SI  - GENBANK/HQ830596
SI  - GENBANK/HQ830597
SI  - GENBANK/HQ830598
SI  - GENBANK/HQ830599
SI  - GENBANK/HQ830600
SI  - GENBANK/HQ830601
SI  - GENBANK/HQ830602
SI  - GENBANK/HQ830603
SI  - GENBANK/HQ830604
SI  - GENBANK/HQ830605
SI  - GENBANK/HQ830606
SI  - GENBANK/HQ830608
SI  - GENBANK/HQ830609
SI  - GENBANK/HQ830610
SI  - GENBANK/HQ830611
SI  - GENBANK/HQ830612
SI  - GENBANK/HQ830613
SI  - GENBANK/HQ830614
SI  - GENBANK/HQ830615
SI  - GENBANK/HQ830616
SI  - GENBANK/HQ830617
SI  - GENBANK/HQ830618
SI  - GENBANK/HQ830619
SI  - GENBANK/HQ830620
SI  - GENBANK/HQ830621
SI  - GENBANK/HQ830622
SI  - GENBANK/HQ830623
SI  - GENBANK/HQ830624
SI  - GENBANK/HQ830625
SI  - GENBANK/HQ830626
SI  - GENBANK/HQ830627
SI  - GENBANK/HQ830628
SI  - GENBANK/HQ830629
SI  - GENBANK/HQ830630
SI  - GENBANK/HQ830631
SI  - GENBANK/HQ830632
SI  - GENBANK/HQ830633
SI  - GENBANK/HQ830634
SI  - GENBANK/HQ830635
SI  - GENBANK/HQ830636
SI  - GENBANK/HQ830637
SI  - GENBANK/HQ830638
SI  - GENBANK/HQ830639
SI  - GENBANK/HQ830640
SI  - GENBANK/HQ830641
SI  - GENBANK/HQ830642
SI  - GENBANK/HQ830643
SI  - GENBANK/HQ830644
SI  - GENBANK/HQ830645
SI  - GENBANK/HQ830646
SI  - GENBANK/HQ830647
SI  - GENBANK/HQ830648
SI  - GENBANK/HQ830649
SI  - GENBANK/HQ830650
SI  - GENBANK/HQ830651
SI  - GENBANK/HQ830652
SI  - GENBANK/HQ830653
SI  - GENBANK/HQ830654
SI  - GENBANK/HQ830655
SI  - GENBANK/HQ830656
SI  - GENBANK/HQ830657
SI  - GENBANK/HQ830658
SI  - GENBANK/HQ830659
SI  - GENBANK/HQ830660
SI  - GENBANK/HQ830661
SI  - GENBANK/HQ830662
SI  - GENBANK/HQ830663
SI  - GENBANK/HQ830664
SI  - GENBANK/HQ830665
SI  - GENBANK/HQ830666
SI  - GENBANK/HQ830667
SI  - GENBANK/HQ830668
SI  - GENBANK/HQ830669
SI  - GENBANK/HQ830670
SI  - GENBANK/HQ830671
SI  - GENBANK/HQ830672
SI  - GENBANK/HQ830673
SI  - GENBANK/HQ830674
SI  - GENBANK/HQ830675
SI  - GENBANK/HQ830676
SI  - GENBANK/HQ830677
SI  - GENBANK/HQ830678
SI  - GENBANK/HQ830679
SI  - GENBANK/HQ830680
SI  - GENBANK/HQ830681
SI  - GENBANK/HQ830682
SI  - GENBANK/HQ830683
SI  - GENBANK/HQ830684
SI  - GENBANK/HQ830685
SI  - GENBANK/HQ830686
SI  - GENBANK/HQ830687
SI  - GENBANK/HQ830688
SI  - GENBANK/HQ830689
SI  - GENBANK/HQ830690
SI  - GENBANK/HQ830691
SI  - GENBANK/HQ830692
SI  - GENBANK/HQ830693
SI  - GENBANK/HQ830694
SI  - GENBANK/HQ830695
SI  - GENBANK/HQ830697
SI  - GENBANK/HQ830698
SI  - GENBANK/HQ830699
SI  - GENBANK/HQ830700
SI  - GENBANK/HQ830701
SI  - GENBANK/HQ830702
SI  - GENBANK/HQ830703
SI  - GENBANK/HQ830704
SI  - GENBANK/HQ830705
SI  - GENBANK/HQ830706
SI  - GENBANK/HQ830707
SI  - GENBANK/HQ830708
SI  - GENBANK/HQ830709
SI  - GENBANK/HQ830710
SI  - GENBANK/HQ830711
SI  - GENBANK/HQ830712
SI  - GENBANK/HQ830713
SI  - GENBANK/HQ830714
SI  - GENBANK/HQ830715
SI  - GENBANK/HQ830716
SI  - GENBANK/HQ830717
SI  - GENBANK/HQ830718
SI  - GENBANK/HQ830719
SI  - GENBANK/HQ830720
SI  - GENBANK/HQ830721
SI  - GENBANK/HQ830722
SI  - GENBANK/HQ830723
SI  - GENBANK/HQ830724
SI  - GENBANK/HQ830725
SI  - GENBANK/HQ830726
SI  - GENBANK/HQ830727
SI  - GENBANK/HQ830728
SI  - GENBANK/HQ830729
SI  - GENBANK/HQ830730
SI  - GENBANK/HQ830731
SI  - GENBANK/HQ830732
SI  - GENBANK/HQ830733
SI  - GENBANK/HQ830734
SI  - GENBANK/HQ830735
SI  - GENBANK/HQ830736
SI  - GENBANK/HQ830737
SI  - GENBANK/HQ830738
SI  - GENBANK/HQ830739
SI  - GENBANK/HQ830740
SI  - GENBANK/HQ830741
SI  - GENBANK/HQ830742
SI  - GENBANK/HQ830743
SI  - GENBANK/HQ830744
SI  - GENBANK/HQ830745
SI  - GENBANK/HQ830746
SI  - GENBANK/HQ830747
SI  - GENBANK/HQ830748
SI  - GENBANK/HQ830749
SI  - GENBANK/HQ830750
SI  - GENBANK/HQ830752
SI  - GENBANK/HQ830753
SI  - GENBANK/HQ830754
SI  - GENBANK/HQ830755
SI  - GENBANK/HQ830756
SI  - GENBANK/HQ830757
SI  - GENBANK/HQ830758
SI  - GENBANK/HQ830759
SI  - GENBANK/HQ830760
SI  - GENBANK/HQ830761
SI  - GENBANK/HQ830762
SI  - GENBANK/HQ830763
SI  - GENBANK/HQ830764
SI  - GENBANK/HQ830765
SI  - GENBANK/HQ830766
SI  - GENBANK/HQ830767
SI  - GENBANK/HQ830768
SI  - GENBANK/HQ830770
SI  - GENBANK/HQ830771
SI  - GENBANK/HQ830772
SI  - GENBANK/HQ830773
SI  - GENBANK/HQ830774
SI  - GENBANK/HQ830775
SI  - GENBANK/HQ830776
SI  - GENBANK/HQ830777
SI  - GENBANK/HQ830778
SI  - GENBANK/HQ830781
SI  - GENBANK/HQ830782
SI  - GENBANK/HQ830783
SI  - GENBANK/HQ830784
SI  - GENBANK/HQ830785
SI  - GENBANK/HQ830786
SI  - GENBANK/HQ830787
SI  - GENBANK/HQ830788
SI  - GENBANK/HQ830789
SI  - GENBANK/HQ830790
SI  - GENBANK/HQ830791
SI  - GENBANK/HQ830792
SI  - GENBANK/HQ830793
SI  - GENBANK/HQ830794
SI  - GENBANK/HQ830795
SI  - GENBANK/HQ830796
SI  - GENBANK/HQ830797
SI  - GENBANK/HQ830798
SI  - GENBANK/HQ830799
SI  - GENBANK/HQ830800
SI  - GENBANK/HQ830801
SI  - GENBANK/HQ830802
SI  - GENBANK/HQ830803
SI  - GENBANK/HQ830804
SI  - GENBANK/HQ830805
SI  - GENBANK/HQ830806
SI  - GENBANK/HQ830807
SI  - GENBANK/HQ830808
SI  - GENBANK/HQ830809
SI  - GENBANK/HQ830810
SI  - GENBANK/HQ830811
SI  - GENBANK/HQ830812
SI  - GENBANK/HQ830813
SI  - GENBANK/HQ830814
SI  - GENBANK/HQ830815
SI  - GENBANK/HQ830816
SI  - GENBANK/HQ830817
SI  - GENBANK/HQ830818
SI  - GENBANK/HQ830819
SI  - GENBANK/HQ830820
SI  - GENBANK/HQ830821
SI  - GENBANK/HQ830822
SI  - GENBANK/HQ830823
SI  - GENBANK/HQ830824
SI  - GENBANK/HQ830825
SI  - GENBANK/HQ830826
SI  - GENBANK/HQ830827
SI  - GENBANK/HQ830828
SI  - GENBANK/HQ830829
SI  - GENBANK/HQ830830
SI  - GENBANK/HQ830831
SI  - GENBANK/HQ830832
SI  - GENBANK/HQ830833
SI  - GENBANK/HQ830834
SI  - GENBANK/HQ830835
SI  - GENBANK/HQ830836
SI  - GENBANK/HQ830838
SI  - GENBANK/HQ830839
SI  - GENBANK/HQ830840
SI  - GENBANK/HQ830842
SI  - GENBANK/HQ830843
SI  - GENBANK/HQ830844
SI  - GENBANK/HQ830845
SI  - GENBANK/HQ830846
SI  - GENBANK/HQ830847
SI  - GENBANK/HQ830848
SI  - GENBANK/HQ840749
SI  - GENBANK/HQ840750
SI  - GENBANK/HQ840751
SI  - GENBANK/HQ840752
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110406
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - DNA, Mitochondrial/*chemistry
MH  - Female
MH  - Genetic Speciation
MH  - Haplotypes
MH  - Hemiptera/classification/*genetics/physiology
MH  - *Hybridization, Genetic
MH  - Male
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - New Zealand
MH  - Phenotype
MH  - Phylogeography
MH  - Reproduction
MH  - Sequence Alignment
MH  - Sexual Behavior, Animal
MH  - Social Isolation
MH  - Vocalization, Animal
EDAT- 2011/04/08 06:00
MHDA- 2011/10/21 06:00
CRDT- 2011/04/08 06:00
PHST- 2011/04/08 06:00 [entrez]
PHST- 2011/04/08 06:00 [pubmed]
PHST- 2011/10/21 06:00 [medline]
AID - syr017 [pii]
AID - 10.1093/sysbio/syr017 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jul;60(4):482-502. doi: 10.1093/sysbio/syr017. Epub 2011 Apr 6.

PMID- 21460386
OWN - NLM
STAT- MEDLINE
DCOM- 20111020
LR  - 20110615
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 4
DP  - 2011 Jul
TI  - Correlations of life-history and distributional-range variation with salamander
      diversification rates: evidence for species selection.
PG  - 503-18
LID - 10.1093/sysbio/syr020 [doi]
AB  - Evolutionary biologists have long debated the relative influence of species
      selection on evolutionary patterns. As a test, we apply a statistical
      phylogenetic approach to evaluate the influence of traits related to species
      distribution and life-history characteristics on patterns of diversification in
      salamanders. We use independent contrasts to test trait-mediated diversification 
      while accommodating phylogenetic uncertainty in relationships among all
      salamander families. Using a neontological data set, we find several
      species-level traits to be variable, heritable, and associated with differential 
      success (i.e., higher diversification rates) at higher taxonomic categories.
      Specifically, the macroecological trait of small geographic-range size is
      strongly correlated with a higher rate of net diversification. We further
      consider the role that plasticity in life-history traits appears to fulfill in
      macroevolutionary processes of lineage divergence and durability. We find that
      pedotypy--wherein some, but not all, organisms of a species mature in the gilled 
      form without metamorphosing-is also associated with higher net diversification
      rate than is the absence of developmental plasticity. Often dismissed as an
      insignificant process in evolution, we provide direct evidence for the role of
      species selection in lineage diversification of salamanders.
CI  - (c) The Author(s) 2011. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved.
FAU - Eastman, Jonathan M
AU  - Eastman JM
AD  - School of Biological Sciences, Washington State University, Pullman, WA
      99164-4236, USA. jonathan.eastman@gmail.com
FAU - Storfer, Andrew
AU  - Storfer A
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110402
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Genetic Speciation
MH  - Geography
MH  - Homing Behavior
MH  - *Phylogeny
MH  - Selection, Genetic
MH  - Urodela/*classification/genetics
EDAT- 2011/04/05 06:00
MHDA- 2011/10/21 06:00
CRDT- 2011/04/05 06:00
PHST- 2011/04/05 06:00 [entrez]
PHST- 2011/04/05 06:00 [pubmed]
PHST- 2011/10/21 06:00 [medline]
AID - syr020 [pii]
AID - 10.1093/sysbio/syr020 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jul;60(4):503-18. doi: 10.1093/sysbio/syr020. Epub 2011 Apr 2.

PMID- 21447483
OWN - NLM
STAT- MEDLINE
DCOM- 20111220
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 5
DP  - 2011 Oct
TI  - Missing data in phylogenetic analysis: reconciling results from simulations and
      empirical data.
PG  - 719-31
LID - 10.1093/sysbio/syr025 [doi]
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Department of Ecology and Evolution, Stony Brook University, Stony Brook, NY
      11794-5245, USA. wiensj@life.bio.sunysb.edu
FAU - Morrill, Matthew C
AU  - Morrill MC
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110328
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - Computer Simulation
MH  - Data Interpretation, Statistical
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Urodela/*classification/genetics
EDAT- 2011/03/31 06:00
MHDA- 2011/12/21 06:00
CRDT- 2011/03/31 06:00
PHST- 2011/03/31 06:00 [entrez]
PHST- 2011/03/31 06:00 [pubmed]
PHST- 2011/12/21 06:00 [medline]
AID - syr025 [pii]
AID - 10.1093/sysbio/syr025 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Oct;60(5):719-31. doi: 10.1093/sysbio/syr025. Epub 2011 Mar 28.

PMID- 21447482
OWN - NLM
STAT- MEDLINE
DCOM- 20111220
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 5
DP  - 2011 Oct
TI  - Developmental characters in phylogenetic inference and their absolute timing
      information.
PG  - 630-44
LID - 10.1093/sysbio/syr024 [doi]
AB  - Despite the recent surge of interest in studying the evolution of development,
      surprisingly little work has been done to investigate the phylogenetic signal in 
      developmental characters. Yet, both the potential usefulness of developmental
      characters in phylogenetic reconstruction and the validity of inferences on the
      evolution of developmental characters depend on the presence of such a
      phylogenetic signal and on the ability of our coding scheme to capture it. In a
      recent study, we showed, using simulations, that a new method (called the
      continuous analysis) using standardized time or ontogenetic sequence data and
      squared-change parsimony outperformed event pairing and event cracking in
      analyzing developmental data on a reference phylogeny. Using the same simulated
      data, we demonstrate that all these coding methods (event pairing and
      standardized time or ontogenetic sequence data) can be used to produce
      phylogenetically informative data. Despite some dependence between characters
      (the position of an event in an ontogenetic sequence is not independent of the
      position of other events in the same sequence), parsimony analysis of such
      characters converges on the correct phylogeny as the amount of data increases. In
      this context, the new coding method (developed for the continuous analysis)
      outperforms event pairing; it recovers a lower proportion of incorrect clades.
      This study thus validates the use of ontogenetic data in phylogenetic inference
      and presents a simple coding scheme that can extract a reliable phylogenetic
      signal from these data.
FAU - Laurin, Michel
AU  - Laurin M
AD  - UMR CNRS 7207 "Centre de Recherches sur la Paleobiodiversite et les
      Paleoenvironnements," Museum National d'Histoire Naturelle, Departement Histoire 
      de la Terre, Batiment de Geologie, Case Postale 48, 43 Rue Buffon, F-75231 Paris 
      Cedex 05, France. michel.laurin@upmc.fr
FAU - Germain, Damien
AU  - Germain D
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110328
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Computer Simulation
MH  - Models, Biological
MH  - Models, Statistical
MH  - Osteogenesis
MH  - Phylogeny
MH  - Reproducibility of Results
MH  - Skull/physiology
MH  - Time Factors
MH  - Urodela/*classification/physiology
EDAT- 2011/03/31 06:00
MHDA- 2011/12/21 06:00
CRDT- 2011/03/31 06:00
PHST- 2011/03/31 06:00 [entrez]
PHST- 2011/03/31 06:00 [pubmed]
PHST- 2011/12/21 06:00 [medline]
AID - syr024 [pii]
AID - 10.1093/sysbio/syr024 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Oct;60(5):630-44. doi: 10.1093/sysbio/syr024. Epub 2011 Mar 28.

PMID- 21447481
OWN - NLM
STAT- MEDLINE
DCOM- 20111220
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 5
DP  - 2011 Oct
TI  - Estimating species trees from unrooted gene trees.
PG  - 661-7
LID - 10.1093/sysbio/syr027 [doi]
AB  - In this study, we develop a distance method for inferring unrooted species trees 
      from a collection of unrooted gene trees. The species tree is estimated by the
      neighbor joining (NJ) tree built from a distance matrix in which the distance
      between two species is defined as the average number of internodes between two
      species across gene trees, that is, average gene-tree internode distance. The
      distance method is named NJ(st) to distinguish it from the original NJ method.
      Under the coalescent model, we show that if gene trees are known or estimated
      correctly, the NJ(st) method is statistically consistent in estimating unrooted
      species trees. The simulation results suggest that NJ(st) and STAR (another
      coalescence-based method for inferring species trees) perform almost equally well
      in estimating topologies of species trees, whereas the Bayesian coalescence-based
      method, BEST, outperforms both NJ(st) and STAR. Unlike BEST and STAR, the NJ(st) 
      method can take unrooted gene trees to infer species trees without using an
      outgroup. In addition, the NJ(st) method can handle missing data and is thus
      useful in phylogenomic studies in which data sets often contain missing loci for 
      some individuals.
FAU - Liu, Liang
AU  - Liu L
AD  - Department of Agriculture and Natural Resources, Delaware State University,
      Dover, DE 19901, USA. lliu@desu.edu
FAU - Yu, Lili
AU  - Yu L
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
DEP - 20110328
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - Models, Genetic
MH  - Phylogeny
MH  - Saccharomyces/*classification/*genetics
EDAT- 2011/03/31 06:00
MHDA- 2011/12/21 06:00
CRDT- 2011/03/31 06:00
PHST- 2011/03/31 06:00 [entrez]
PHST- 2011/03/31 06:00 [pubmed]
PHST- 2011/12/21 06:00 [medline]
AID - syr027 [pii]
AID - 10.1093/sysbio/syr027 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Oct;60(5):661-7. doi: 10.1093/sysbio/syr027. Epub 2011 Mar 28.

PMID- 21447480
OWN - NLM
STAT- MEDLINE
DCOM- 20111220
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 5
DP  - 2011 Oct
TI  - On Rogers' proof of identifiability for the GTR + Gamma + I model.
PG  - 713-8
LID - 10.1093/sysbio/syr023 [doi]
FAU - Chai, Juanjuan
AU  - Chai J
AD  - Department of Mathematics, Indiana University, Bloomington IN 47405, USA.
FAU - Housworth, Elizabeth A
AU  - Housworth EA
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
DEP - 20110328
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - Computational Biology/*methods
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Models, Statistical
MH  - *Phylogeny
EDAT- 2011/03/31 06:00
MHDA- 2011/12/21 06:00
CRDT- 2011/03/31 06:00
PHST- 2011/03/31 06:00 [entrez]
PHST- 2011/03/31 06:00 [pubmed]
PHST- 2011/12/21 06:00 [medline]
AID - syr023 [pii]
AID - 10.1093/sysbio/syr023 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Oct;60(5):713-8. doi: 10.1093/sysbio/syr023. Epub 2011 Mar 28.

PMID- 21436107
OWN - NLM
STAT- MEDLINE
DCOM- 20111020
LR  - 20181201
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 4
DP  - 2011 Jul
TI  - The devil in the details: interactions between the branch-length prior and
      likelihood model affect node support and branch lengths in the phylogeny of the
      Psoraceae.
PG  - 541-61
LID - 10.1093/sysbio/syr022 [doi]
AB  - In popular use of Bayesian phylogenetics, a default branch-length prior is almost
      universally applied without knowing how a different prior would have affected the
      outcome. We performed Bayesian and maximum likelihood (ML) inference of phylogeny
      based on empirical nucleotide sequence data from a family of lichenized
      ascomycetes, the Psoraceae, the morphological delimitation of which has been
      controversial. We specifically assessed the influence of the combination of
      Bayesian branch-length prior and likelihood model on the properties of the Markov
      chain Monte Carlo tree sample, including node support, branch lengths, and taxon 
      stability. Data included two regions of the mitochondrial ribosomal RNA gene, the
      internal transcribed spacer region of the nuclear ribosomal RNA gene, and the
      protein-coding largest subunit of RNA polymerase II. Data partitioning was
      performed using Bayes' factors, whereas the best-fitting model of each partition 
      was selected using the Bayesian information criterion (BIC). Given the data and
      model, short Bayesian branch-length priors generate higher numbers of strongly
      supported nodes as well as short and topologically similar trees sampled from
      parts of tree space that are largely unexplored by the ML bootstrap. Long
      branch-length priors generate fewer strongly supported nodes and longer and more 
      dissimilar trees that are sampled mostly from inside the range of tree space
      sampled by the ML bootstrap. Priors near the ML distribution of branch lengths
      generate the best marginal likelihood and the highest frequency of "rogue"
      (unstable) taxa. The branch-length prior was shown to interact with the
      likelihood model. Trees inferred under complex partitioned models are more
      affected by the stretching effect of the branch-length prior. Fewer nodes are
      strongly supported under a complex model given the same branch-length prior.
      Irrespective of model, internal branches make up a larger proportion of total
      tree length under the shortest branch-length priors compared with longer priors. 
      Relative effects on branch lengths caused by the branch-length prior can be
      problematic to downstream phylogenetic comparative methods making use of the
      branch lengths. Furthermore, given the same branch-length prior, trees are on
      average more dissimilar under a simple unpartitioned model compared with a more
      complex partitioned models. The distribution of ML branch lengths was shown to
      better fit a gamma or Pareto distribution than an exponential one. Model adequacy
      tests indicate that the best-fitting model selected by the BIC is insufficient
      for describing data patterns in 5 of 8 partitions. More general substitution
      models are required to explain the data in three of these partitions, one of
      which also requires nonstationarity. The two mitochondrial ribosomal RNA gene
      partitions need heterotachous models. We found no significant correlations
      between, on the one hand, the amount of ambiguous data or the smallest
      branch-length distance to another taxon and, on the other hand, the topological
      stability of individual taxa. Integrating over several exponentially distributed 
      means under the best-fitting model, node support for the family Psoraceae,
      including Psora, Protoblastenia, and the Micarea sylvicola group, is
      approximately 0.96. Support for the genus Psora is distinctly lower, but we found
      no evidence to contradict the current classification.
CI  - (c) The Author(s) 2011. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved.
FAU - Ekman, Stefan
AU  - Ekman S
AD  - Museum of Evolution, Uppsala University, Norbyvagen 16, SE-752 36 Uppsala,
      Sweden. stefan.ekman@em.uu.se
FAU - Blaalid, Rakel
AU  - Blaalid R
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110324
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal Spacer)
RN  - 0 (RNA, Mitochondrial)
RN  - 0 (RNA, Ribosomal)
RN  - 63231-63-0 (RNA)
RN  - EC 2.7.7.- (RNA Polymerase II)
SB  - IM
MH  - Ascomycota/classification/*genetics
MH  - Bayes Theorem
MH  - DNA, Ribosomal Spacer/chemistry
MH  - Likelihood Functions
MH  - Markov Chains
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - RNA/chemistry
MH  - RNA Polymerase II/chemistry/genetics
MH  - RNA, Mitochondrial
MH  - RNA, Ribosomal/chemistry
EDAT- 2011/03/26 06:00
MHDA- 2011/10/21 06:00
CRDT- 2011/03/26 06:00
PHST- 2011/03/26 06:00 [entrez]
PHST- 2011/03/26 06:00 [pubmed]
PHST- 2011/10/21 06:00 [medline]
AID - syr022 [pii]
AID - 10.1093/sysbio/syr022 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jul;60(4):541-61. doi: 10.1093/sysbio/syr022. Epub 2011 Mar 24.

PMID- 21436106
OWN - NLM
STAT- MEDLINE
DCOM- 20111220
LR  - 20120322
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 5
DP  - 2011 Oct
TI  - The trouble with topology: phylogenies without fossils provide a revisionist
      perspective of evolutionary history in topological analyses of diversity.
PG  - 700-12
LID - 10.1093/sysbio/syr018 [doi]
FAU - Tarver, James E
AU  - Tarver JE
AD  - School of Earth Sciences, University of Bristol, Wills Memorial Building, Queen's
      Road, Bristol BS81RJ, UK. james.tarver@bristol.ac.uk
FAU - Donoghue, Philip C J
AU  - Donoghue PC
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110323
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Alligators and Crocodiles/classification
MH  - Animals
MH  - Biodiversity
MH  - *Biological Evolution
MH  - Carnivora/classification
MH  - Extinction, Biological
MH  - *Fossils
MH  - Genetic Speciation
MH  - Models, Biological
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Time
EDAT- 2011/03/26 06:00
MHDA- 2011/12/21 06:00
CRDT- 2011/03/26 06:00
PHST- 2011/03/26 06:00 [entrez]
PHST- 2011/03/26 06:00 [pubmed]
PHST- 2011/12/21 06:00 [medline]
AID - syr018 [pii]
AID - 10.1093/sysbio/syr018 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Oct;60(5):700-12. doi: 10.1093/sysbio/syr018. Epub 2011 Mar 23.

PMID- 21436105
OWN - NLM
STAT- MEDLINE
DCOM- 20110812
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 3
DP  - 2011 May
TI  - Performance, accuracy, and Web server for evolutionary placement of short
      sequence reads under maximum likelihood.
PG  - 291-302
LID - 10.1093/sysbio/syr010 [doi]
AB  - We present an evolutionary placement algorithm (EPA) and a Web server for the
      rapid assignment of sequence fragments (short reads) to edges of a given
      phylogenetic tree under the maximum-likelihood model. The accuracy of the
      algorithm is evaluated on several real-world data sets and compared with
      placement by pair-wise sequence comparison, using edit distances and BLAST. We
      introduce a slow and accurate as well as a fast and less accurate placement
      algorithm. For the slow algorithm, we develop additional heuristic techniques
      that yield almost the same run times as the fast version with only a small loss
      of accuracy. When those additional heuristics are employed, the run time of the
      more accurate algorithm is comparable with that of a simple BLAST search for data
      sets with a high number of short query sequences. Moreover, the accuracy of the
      EPA is significantly higher, in particular when the sample of taxa in the
      reference topology is sparse or inadequate. Our algorithm, which has been
      integrated into RAxML, therefore provides an equally fast but more accurate
      alternative to BLAST for tree-based inference of the evolutionary origin and
      composition of short sequence reads. We are also actively developing a Web server
      that offers a freely available service for computing read placements on trees
      using the EPA.
FAU - Berger, Simon A
AU  - Berger SA
AD  - The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for
      Theoretical Studies, Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany.
FAU - Krompass, Denis
AU  - Krompass D
FAU - Stamatakis, Alexandros
AU  - Stamatakis A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110323
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Amino Acid Sequence
MH  - Base Sequence
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Internet
MH  - *Likelihood Functions
MH  - *Phylogeny
MH  - Sequence Alignment/*methods
MH  - Sequence Analysis, DNA/methods
MH  - Sequence Analysis, Protein/methods
MH  - Sequence Analysis, RNA/methods
MH  - Software
PMC - PMC3078422
EDAT- 2011/03/26 06:00
MHDA- 2011/08/13 06:00
CRDT- 2011/03/26 06:00
PHST- 2011/03/26 06:00 [entrez]
PHST- 2011/03/26 06:00 [pubmed]
PHST- 2011/08/13 06:00 [medline]
AID - syr010 [pii]
AID - 10.1093/sysbio/syr010 [doi]
PST - ppublish
SO  - Syst Biol. 2011 May;60(3):291-302. doi: 10.1093/sysbio/syr010. Epub 2011 Mar 23.

PMID- 21436104
OWN - NLM
STAT- MEDLINE
DCOM- 20111020
LR  - 20110615
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 4
DP  - 2011 Jul
TI  - Integrating fossil preservation biases in the selection of calibrations for
      molecular divergence time estimation.
PG  - 519-27
LID - 10.1093/sysbio/syr019 [doi]
AB  - The selection of fossil data to use as calibration age priors in molecular
      divergence time estimates inherently links neontological methods with
      paleontological theory. However, few neontological studies have taken into
      account the possibility of a taphonomic bias in the fossil record when developing
      approaches to fossil calibration selection. The Sppil-Rongis effect may bias the 
      first appearance of a lineage toward the recent causing most objective
      calibration selection approaches to erroneously exclude appropriate calibrations 
      or to incorporate multiple calibrations that are too young to accurately
      represent the divergence times of target lineages. Using turtles as a case study,
      we develop a Bayesian extension to the fossil selection approach developed by
      Marshall (2008. A simple method for bracketing absolute divergence times on
      molecular phylogenies using multiple fossil calibrations points. Am. Nat.
      171:726-742) that takes into account this taphonomic bias. Our method has the
      advantage of identifying calibrations that may bias age estimates to be too
      recent while incorporating uncertainty in phylogenetic parameter estimates such
      as tree topology and branch lengths. Additionally, this method is easily adapted 
      to assess the consistency of potential calibrations to any one calibration in the
      candidate pool.
CI  - (c) The Author(s) 2011. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved.
FAU - Dornburg, Alex
AU  - Dornburg A
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT
      06520, USA. alex.dornburg@yale.edu
FAU - Beaulieu, Jeremy M
AU  - Beaulieu JM
FAU - Oliver, Jeffrey C
AU  - Oliver JC
FAU - Near, Thomas J
AU  - Near TJ
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110323
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Fossils
MH  - Genetic Speciation
MH  - *Phylogeny
MH  - Preservation, Biological
MH  - Turtles/anatomy &amp; histology/classification/*genetics
EDAT- 2011/03/26 06:00
MHDA- 2011/10/21 06:00
CRDT- 2011/03/26 06:00
PHST- 2011/03/26 06:00 [entrez]
PHST- 2011/03/26 06:00 [pubmed]
PHST- 2011/10/21 06:00 [medline]
AID - syr019 [pii]
AID - 10.1093/sysbio/syr019 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jul;60(4):519-27. doi: 10.1093/sysbio/syr019. Epub 2011 Mar 23.

PMID- 21398626
OWN - NLM
STAT- MEDLINE
DCOM- 20110812
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 3
DP  - 2011 May
TI  - History can matter: non-Markovian behavior of ancestral lineages.
PG  - 276-90
LID - 10.1093/sysbio/syr012 [doi]
AB  - Although most of the important evolutionary events in the history of biology can 
      only be studied via interspecific comparisons, it is challenging to apply the
      rich body of population genetic theory to the study of interspecific genetic
      variation. Probabilistic modeling of the substitution process would ideally be
      derived from first principles of population genetics, allowing a quantitative
      connection to be made between the parameters describing mutation, selection,
      drift, and the patterns of interspecific variation. There has been progress in
      reconciling population genetics and interspecific evolution for the case where
      mutation rates are sufficiently low, but when mutation rates are higher,
      reconciliation has been hampered due to complications from how the loss or
      fixation of new mutations can be influenced by linked nonneutral polymorphisms
      (i.e., the Hill-Robertson effect). To investigate the generation of interspecific
      genetic variation when concurrent fitness-affecting polymorphisms are common and 
      the Hill-Robertson effect is thereby potentially strong, we used the
      Wright-Fisher model of population genetics to simulate very many generations of
      mutation, natural selection, and genetic drift. This was done so that the
      chronological history of advantageous, deleterious, and neutral substitutions
      could be traced over time along the ancestral lineage. Our simulations show that 
      the process by which a nonrecombining sequence changes over time can markedly
      deviate from the Markov assumption that is ubiquitous in molecular phylogenetics.
      In particular, we find tendencies for advantageous substitutions to be followed
      by deleterious ones and for deleterious substitutions to be followed by
      advantageous ones. Such non-Markovian patterns reflect the fact that the fate of 
      the ancestral lineage depends not only on its current allelic state but also on
      gene copies not belonging to the ancestral lineage. Although our simulations
      describe nonrecombining sequences, we conclude by discussing how non-Markovian
      behavior of the ancestral lineage is plausible even when recombination rates are 
      not low. As a result, we believe that increased attention needs to be devoted to 
      the robustness of evolutionary inference procedures that rely upon the Markov
      assumption.
FAU - Cartwright, Reed A
AU  - Cartwright RA
AD  - Department of Genetics, Bioinformatics Research Center, North Carolina State
      University, Raleigh, NC 27695-7566, USA.
FAU - Lartillot, Nicolas
AU  - Lartillot N
FAU - Thorne, Jeffrey L
AU  - Thorne JL
LA  - eng
GR  - R01 GM070806-06/GM/NIGMS NIH HHS/United States
GR  - GM070806/GM/NIGMS NIH HHS/United States
GR  - R01 GM070806/GM/NIGMS NIH HHS/United States
GR  - GM090201/GM/NIGMS NIH HHS/United States
GR  - LM010009/LM/NLM NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
DEP - 20110311
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Genetic Fitness
MH  - Genetic Variation
MH  - *Markov Chains
MH  - *Models, Genetic
MH  - Polymorphism, Genetic
MH  - Selection, Genetic
PMC - PMC3078423
EDAT- 2011/03/15 06:00
MHDA- 2011/08/13 06:00
CRDT- 2011/03/15 06:00
PHST- 2011/03/15 06:00 [entrez]
PHST- 2011/03/15 06:00 [pubmed]
PHST- 2011/08/13 06:00 [medline]
AID - syr012 [pii]
AID - 10.1093/sysbio/syr012 [doi]
PST - ppublish
SO  - Syst Biol. 2011 May;60(3):276-90. doi: 10.1093/sysbio/syr012. Epub 2011 Mar 11.

PMID- 21389297
OWN - NLM
STAT- MEDLINE
DCOM- 20111020
LR  - 20110615
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 4
DP  - 2011 Jul
TI  - Inferring species-level phylogenies and taxonomic distinctiveness using
      multilocus data in Sistrurus rattlesnakes.
PG  - 393-409
LID - 10.1093/sysbio/syr011 [doi]
AB  - Phylogenetic relationships and taxonomic distinctiveness of closely related
      species and subspecies are most accurately inferred from data derived from
      multiple independent loci. Here, we apply several approaches for understanding
      species-level relationships using data from 18 nuclear DNA loci and 1
      mitochondrial DNA locus within currently described species and subspecies of
      Sistrurus rattlesnakes. Collectively, these methods provide evidence that a
      currently described species, the massasauga rattlesnake (Sistrurus catenatus),
      consists of two well-supported clades, one composed of the two western subspecies
      (S. c. tergeminus and S. c. edwardsii) and the other the eastern subspecies (S.
      c. catenatus). Within pigmy rattlesnakes (S. miliarius), however, there is not
      strong support across methods for any particular grouping at the subspecific
      level. Monophyly based tests for taxonomic distinctiveness show evidence for
      distinctiveness of all subspecies but this support is strongest by far for the S.
      c. catenatus clade. Because support for the distinctiveness of S. c. catenatus is
      both strong and consistent across methods, and due to its morphological
      distinctiveness and allopatric distribution, we suggest that this subspecies be
      elevated to full species status, which has significant conservation implications.
      Finally, most divergence time estimates based upon a fossil-calibrated species
      tree are &gt; 50% younger than those from a concatenated gene tree analysis and
      suggest that an active period of speciation within Sistrurus occurred within the 
      late Pliocene/Pleistocene eras.
CI  - (c) The Author(s) 2011. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved.
FAU - Kubatko, Laura S
AU  - Kubatko LS
AD  - Department of Statistics, The Ohio State University, Columbus, OH 43210, USA.
      kubatko.2@osu.edu
FAU - Gibbs, H Lisle
AU  - Gibbs HL
FAU - Bloomquist, Erik W
AU  - Bloomquist EW
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110309
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Crotalus/*classification/genetics
MH  - DNA/chemistry
MH  - DNA, Mitochondrial/chemistry
MH  - Genetic Speciation
MH  - *Phylogeny
MH  - Recombination, Genetic
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2011/03/11 06:00
MHDA- 2011/10/21 06:00
CRDT- 2011/03/11 06:00
PHST- 2011/03/11 06:00 [entrez]
PHST- 2011/03/11 06:00 [pubmed]
PHST- 2011/10/21 06:00 [medline]
AID - syr011 [pii]
AID - 10.1093/sysbio/syr011 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jul;60(4):393-409. doi: 10.1093/sysbio/syr011. Epub 2011 Mar 9.

PMID- 21386113
OWN - NLM
STAT- MEDLINE
DCOM- 20110812
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 3
DP  - 2011 May
TI  - Evaluation of Bayesian models of substitution rate evolution--parental guidance
      versus mutual independence.
PG  - 329-42
LID - 10.1093/sysbio/syr009 [doi]
AB  - We have evaluated the performance of two classes of probabilistic models for
      substitution rate variation over phylogenetic trees. In the first class, branch
      rates are considered to be independent and identically distributed (i.i.d.)
      stochastic variables. Three versions with respect to the underlying distribution 
      (Gamma, Inverse Gaussian, and LogNormal) are considered. The i.i.d. models are
      compared with the autocorrelated (AC) model, where rates of adjacent nodes in the
      tree are AC, so that a node rate is LogNormal distributed around the rate of the 
      parent node. The performance of different models is evaluated using three
      empirical data sets. For all data sets, it was clear that all tested models
      extracted substantial knowledge from data when posterior divergence time
      distributions were compared with the prior distributions and, furthermore, that
      they clearly outperformed a molecular clock. Moreover, the descriptive power of
      the i.i.d. models, as evaluated by Bayes factors, was either equal to or clearly 
      better than that of the AC model. The latter effect increased with extended taxon
      sampling. Likewise, under none of the models could we find compelling evidence,
      in any of the data sets, for rate correlation between adjacent branches/nodes.
      These findings challenge previous suggestions of universality of autocorrelation 
      in sequence evolution. We also performed an additional comparison with a
      divergence time prior including calibration information from fossil evidence.
      Adding fossil information to the prior had negligible effect on Bayes factors and
      mainly affected the width of the posterior distribution of the divergence times, 
      whereas the relative position of the mean divergence times were largely
      unaffected.
FAU - Linder, Martin
AU  - Linder M
AD  - Department of Mathematics, Uppsala University, Box 480, SE-751 06 Uppsala,
      Sweden.
FAU - Britton, Tom
AU  - Britton T
FAU - Sennblad, Bengt
AU  - Sennblad B
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110308
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Plant Proteins)
RN  - 9014-25-9 (RNA, Transfer)
RN  - EC 4.1.1.39 (RbcL protein, plastid)
RN  - EC 4.1.1.39 (Ribulose-Bisphosphate Carboxylase)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - Fossils
MH  - Genes, rRNA/genetics
MH  - Haplorhini/classification/genetics
MH  - Magnoliopsida/classification/genetics
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Plant Proteins/genetics
MH  - RNA, Transfer/genetics
MH  - Ribulose-Bisphosphate Carboxylase/genetics
MH  - Sequence Alignment
EDAT- 2011/03/10 06:00
MHDA- 2011/08/13 06:00
CRDT- 2011/03/10 06:00
PHST- 2011/03/10 06:00 [entrez]
PHST- 2011/03/10 06:00 [pubmed]
PHST- 2011/08/13 06:00 [medline]
AID - syr009 [pii]
AID - 10.1093/sysbio/syr009 [doi]
PST - ppublish
SO  - Syst Biol. 2011 May;60(3):329-42. doi: 10.1093/sysbio/syr009. Epub 2011 Mar 8.

PMID- 21386112
OWN - NLM
STAT- MEDLINE
DCOM- 20110812
LR  - 20110418
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 3
DP  - 2011 May
TI  - Five statistical questions about the tree of life.
PG  - 318-28
LID - 10.1093/sysbio/syr008 [doi]
AB  - Stochastic modeling of phylogenies raises five questions that have received
      varying levels of attention from quantitatively inclined biologists. 1) How large
      do we expect (from the model) the ratio of maximum historical diversity to
      current diversity to be? 2) From a correct phylogeny of the extant species of a
      clade, what can we deduce about past speciation and extinction rates? 3) What
      proportion of extant species are in fact descendants of still-extant ancestral
      species, and how does this compare with predictions of models? 4) When one moves 
      from trees on species to trees on sets of species (whether traditional higher
      order taxa or clades within PhyloCode), does one expect trees to become more
      unbalanced as a purely logical consequence of tree structure, without signifying 
      any real biological phenomenon? 5) How do we expect that fluctuation rates for
      counts of higher order taxa should compare with fluctuation rates for number of
      species? We present a mathematician's view based on an oversimplified modeling
      framework in which all these questions can be studied coherently.
FAU - Aldous, David J
AU  - Aldous DJ
AD  - Department of Statistics, University of California, Berkeley, 367 Evans Hall #
      3860, Berkeley, CA 94720-3860, USA. aldous@stat.berkeley.edu
FAU - Krikun, Maxim A
AU  - Krikun MA
FAU - Popovic, Lea
AU  - Popovic L
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110308
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biodiversity
MH  - Biological Evolution
MH  - Extinction, Biological
MH  - *Models, Biological
MH  - *Phylogeny
MH  - *Stochastic Processes
EDAT- 2011/03/10 06:00
MHDA- 2011/08/13 06:00
CRDT- 2011/03/10 06:00
PHST- 2011/03/10 06:00 [entrez]
PHST- 2011/03/10 06:00 [pubmed]
PHST- 2011/08/13 06:00 [medline]
AID - syr008 [pii]
AID - 10.1093/sysbio/syr008 [doi]
PST - ppublish
SO  - Syst Biol. 2011 May;60(3):318-28. doi: 10.1093/sysbio/syr008. Epub 2011 Mar 8.

PMID- 21378083
OWN - NLM
STAT- MEDLINE
DCOM- 20111020
LR  - 20110615
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 4
DP  - 2011 Jul
TI  - Testing for temporal variation in diversification rates when sampling is
      incomplete and nonrandom.
PG  - 410-9
LID - 10.1093/sysbio/syr007 [doi]
AB  - A common pattern found in phylogeny-based empirical studies of diversification is
      a decrease in the rate of lineage accumulation toward the present. This
      early-burst pattern of cladogenesis is often interpreted as a signal of adaptive 
      radiation or density-dependent processes of diversification. However, incomplete 
      taxonomic sampling is also known to artifactually produce patterns of rapid
      initial diversification. The Monte Carlo constant rates (MCCR) test, based upon
      Pybus and Harvey's gamma (gamma)-statistic, is commonly used to accommodate
      incomplete sampling, but this test assumes that missing taxa have been randomly
      pruned from the phylogeny. Here we use simulations to show that preferentially
      sampling disparate lineages within a clade can produce severely inflated type-I
      error rates of the MCCR test, especially when taxon sampling drops below 75%. We 
      first propose two corrections for the standard MCCR test, the proportionally
      deeper splits that assumes missing taxa are more likely to be recently diverged, 
      and the deepest splits only MCCR that assumes that all missing taxa are the
      youngest lineages in the clade, and assess their statistical properties. We then 
      extend these two tests into a generalized form that allows the degree of
      nonrandom sampling (NRS)to be controlled by a scaling parameter, alpha. This
      generalized test is then applied to two recent studies. This new test allows
      systematists to account for nonrandom taxonomic sampling when assessing temporal 
      patterns of lineage diversification in empirical trees. Given the dramatic affect
      NRS can have on the behavior of the MCCR test, we argue that evaluating the
      sensitivity of this test to NRS should become the norm when investigating
      patterns of cladogenesis in incompletely sampled phylogenies.
CI  - (c) The Author(s) 2011. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved.
FAU - Brock, Chad D
AU  - Brock CD
AD  - Department of Zoology, School of Biological Sciences, Washington State
      University, Pullman, WA 99164-4236, USA. maximaul@msn.com
FAU - Harmon, Luke J
AU  - Harmon LJ
FAU - Alfaro, Michael E
AU  - Alfaro ME
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110304
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biodiversity
MH  - *Computer Simulation
MH  - Genetic Speciation
MH  - *Models, Biological
MH  - Monte Carlo Method
MH  - *Phylogeny
EDAT- 2011/03/08 06:00
MHDA- 2011/10/21 06:00
CRDT- 2011/03/08 06:00
PHST- 2011/03/08 06:00 [entrez]
PHST- 2011/03/08 06:00 [pubmed]
PHST- 2011/10/21 06:00 [medline]
AID - syr007 [pii]
AID - 10.1093/sysbio/syr007 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jul;60(4):410-9. doi: 10.1093/sysbio/syr007. Epub 2011 Mar 4.

PMID- 21368324
OWN - NLM
STAT- MEDLINE
DCOM- 20110812
LR  - 20110418
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 3
DP  - 2011 May
TI  - Comparing two Bayesian methods for gene tree/species tree reconstruction:
      simulations with incomplete lineage sorting and horizontal gene transfer.
PG  - 261-75
LID - 10.1093/sysbio/syr003 [doi]
AB  - With the increasing interest in recognizing the discordance between gene
      genealogies, various gene tree/species tree reconciliation methods have been
      developed. We present here the first attempt to assess and compare two such
      Bayesian methods, Bayesian estimation of species trees (BEST) and BUCKy (Bayesian
      untangling of concordance knots), in the presence of several known processes of
      gene tree discordance. DNA alignments were simulated under the influence of
      incomplete lineage sorting (ILS) and of horizontal gene transfer (HGT). BEST and 
      BUCKy both account for uncertainty in gene tree estimation but differ
      substantially in their assumptions of what caused gene tree discordance. BEST
      estimates a species tree using the coalescent model, assuming that all gene tree 
      discordance is due to ILS. BUCKy does not assume any specific biological process 
      of gene tree discordance through the use of a nonparametric clustering of
      concordant genes. BUCKy estimates the concordance factor (CF) of a clade, which
      is defined as the proportion of genes that truly have the clade in their trees.
      The estimated concordance tree is then built from clades with the highest
      estimated CFs. Because of their different assumptions, it was expected that BEST 
      would perform better in the presence of ILS and that BUCKy would perform better
      in the presence of HGT. As expected, the species tree was more accurately
      reconstructed by BUCKy in the presence of HGT, when the HGT events were unevenly 
      placed across the species tree. BUCKy and BEST performed similarly in most other 
      cases, including in the presence of strong ILS and of HGT events that were evenly
      placed across the tree. However, BUCKy was shown to underestimate the uncertainty
      in CF estimation, with short credibility intervals. Despite this, the discordance
      pattern estimated by BUCKy could be compared with the signature of ILS. The
      resulting test for the adequacy of the coalescent model proved to have low Type I
      error. It was powerful when HGT was the major source of discordance and when HGT 
      events were unevenly placed across the species tree.
FAU - Chung, Yujin
AU  - Chung Y
AD  - Department of Statistics, University of Wisconsin, 1300 University Avenue,
      Madison, WI 53706, USA. ychung@stat.wisc.edu
FAU - Ane, Cecile
AU  - Ane C
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110302
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - *Gene Transfer, Horizontal
MH  - Genetic Speciation
MH  - *Phylogeny
MH  - *Software
EDAT- 2011/03/04 06:00
MHDA- 2011/08/13 06:00
CRDT- 2011/03/04 06:00
PHST- 2011/03/04 06:00 [entrez]
PHST- 2011/03/04 06:00 [pubmed]
PHST- 2011/08/13 06:00 [medline]
AID - syr003 [pii]
AID - 10.1093/sysbio/syr003 [doi]
PST - ppublish
SO  - Syst Biol. 2011 May;60(3):261-75. doi: 10.1093/sysbio/syr003. Epub 2011 Mar 2.

PMID- 21368323
OWN - NLM
STAT- MEDLINE
DCOM- 20110812
LR  - 20110418
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 3
DP  - 2011 May
TI  - Understanding the evolutionary processes of fungal fruiting bodies: correlated
      evolution and divergence times in the Psathyrellaceae.
PG  - 303-17
LID - 10.1093/sysbio/syr005 [doi]
AB  - Fruiting body evolution is one of the central topics in fungal evolutionary
      biology. A number of hypotheses have been developed to explain the contemporary
      diversity of fruiting body forms, but their evaluation has been hampered by the
      lack of well-sampled data sets and suitable statistical methods. Phylogenetic
      evidence of the physiological changes that accompany switches in fruiting body
      type is lacking, and very little is known about the age of major events of
      fruiting body evolution. Based on a new multigene phylogeny, by using Bayesian
      methods, we demonstrate the existence of correlation between a number of
      morphological features and switches from nondeliquescent to deliquescent
      (autodigesting) fruiting bodies in the mushroom family Psathyrellaceae. Our
      results show that switches in the anatomy of two types of spacer cells (cystidia 
      and pseudoparaphyses) and basidia (bimorphic or monomorphic) as well as the
      structure of the mushroom cap follow the evolution of deliquescent fruiting
      bodies, which suggests strong functional linkage between these traits. We
      performed Bayes factor-based tests, referred hereafter to as evolutionary pathway
      test (EPT), to decide which of the correlated characters were gained first during
      evolution. The EPTs strongly suggest that deliquescence was gained first,
      followed after short waiting times by the other morphological features. Bayesian 
      relaxed molecular clock analyses suggest that the various events of switching
      between fruiting body types occurred independently at various ages during the
      history of the family. The utility of two mushroom fossils (Archaemarasmius and
      Protomycena), the only ones with unambiguous taxonomic positions, for the
      calibration of agaric trees were also examined. Based on our results, we suggest 
      that the evolutionary benefit of deliquescence may be prevention against
      desiccation via accelerated ontogeny of the fruiting body. Hypotheses regarding
      the functional significance of the correlated evolution are presented and
      discussed. Further, we argue that the changes in fruiting body types in mushrooms
      in general can be attributed to independent events (e.g., dispersal and
      adaptation) and not to particular geologic ages.
FAU - Nagy, Laszlo G
AU  - Nagy LG
AD  - Department of Microbiology, Faculty of Science and Informatics, University of
      Szeged, Kozep fasor 52, H-6726 Szeged, Hungary. cortinarius2000@yahoo.co.uk
FAU - Walther, Grit
AU  - Walther G
FAU - Hazi, Judit
AU  - Hazi J
FAU - Vagvolgyi, Csaba
AU  - Vagvolgyi C
FAU - Papp, Tamas
AU  - Papp T
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110302
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Fungal)
SB  - IM
MH  - Agaricales/*classification/*cytology/genetics
MH  - Bayes Theorem
MH  - DNA, Fungal/genetics
MH  - Evolution, Molecular
MH  - Fossils
MH  - Fruiting Bodies, Fungal/classification/cytology/genetics
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
EDAT- 2011/03/04 06:00
MHDA- 2011/08/13 06:00
CRDT- 2011/03/04 06:00
PHST- 2011/03/04 06:00 [entrez]
PHST- 2011/03/04 06:00 [pubmed]
PHST- 2011/08/13 06:00 [medline]
AID - syr005 [pii]
AID - 10.1093/sysbio/syr005 [doi]
PST - ppublish
SO  - Syst Biol. 2011 May;60(3):303-17. doi: 10.1093/sysbio/syr005. Epub 2011 Mar 2.

PMID- 21362644
OWN - NLM
STAT- MEDLINE
DCOM- 20110812
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 3
DP  - 2011 May
TI  - Causes of plant diversification in the Cape biodiversity hotspot of South Africa.
PG  - 343-57
LID - 10.1093/sysbio/syr006 [doi]
AB  - The Cape region of South Africa is one of the most remarkable hotspots of
      biodiversity with a flora comprising more than 9000 plant species, almost 70% of 
      which are endemic, within an area of only +/- 90,000 km2. Much of the diversity
      is due to an exceptionally large contribution of just a few clades that radiated 
      substantially within this region, but little is known about the causes of these
      radiations. Here, we present a comprehensive analysis of plant diversification,
      using near complete species-level phylogenies of four major Cape clades (more
      than 470 species): the genus Protea, a tribe of legumes (Podalyrieae) and two
      speciose genera within the iris family (Babiana and Moraea), representing three
      of the seven largest plant families in this biodiversity hotspot. Combining these
      molecular phylogenetic data with ecological and biogeographical information, we
      tested key hypotheses that have been proposed to explain the radiation of the
      Cape flora. Our results show that the radiations started throughout the Oligocene
      and Miocene and that net diversification rates have remained constant through
      time at globally moderate rates. Furthermore, using sister-species comparisons to
      assess the impact of different factors on speciation, we identified soil type
      shifts as the most important cause of speciation in Babiana, Moraea, and Protea, 
      whereas shifts in fire-survival strategy is the most important factor for
      Podalyrieae. Contrary to previous findings in other groups, such as orchids,
      pollination syndromes show a high degree of phylogenetic conservatism, including 
      groups with a large number of specialized pollination syndromes like Moraea. We
      conclude that the combination of complex environmental conditions together with
      relative climatic stability promoted high speciation and/or low extinction rates 
      as the most likely scenario leading to present-day patterns of hyperdiversity in 
      the Cape.
FAU - Schnitzler, Jan
AU  - Schnitzler J
AD  - Division of Biology, Imperial College London, Silwood Park Campus, Ascot,
      Berkshire SL5 7PY, UK. jan.schnitzler@senckenberg.de
FAU - Barraclough, Timothy G
AU  - Barraclough TG
FAU - Boatwright, James S
AU  - Boatwright JS
FAU - Goldblatt, Peter
AU  - Goldblatt P
FAU - Manning, John C
AU  - Manning JC
FAU - Powell, Martyn P
AU  - Powell MP
FAU - Rebelo, Tony
AU  - Rebelo T
FAU - Savolainen, Vincent
AU  - Savolainen V
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110228
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
RN  - 0 (Soil)
SB  - IM
MH  - *Biodiversity
MH  - DNA, Plant/classification/genetics/isolation &amp; purification
MH  - Evolution, Molecular
MH  - Fabaceae/classification/genetics
MH  - *Fires
MH  - *Genetic Speciation
MH  - Iridaceae/classification/genetics
MH  - Magnoliopsida/*classification/genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Pollination
MH  - Proteaceae/classification/genetics
MH  - Sequence Analysis, DNA
MH  - Soil/*chemistry
MH  - South Africa
EDAT- 2011/03/03 06:00
MHDA- 2011/08/13 06:00
CRDT- 2011/03/03 06:00
PHST- 2011/03/03 06:00 [entrez]
PHST- 2011/03/03 06:00 [pubmed]
PHST- 2011/08/13 06:00 [medline]
AID - syr006 [pii]
AID - 10.1093/sysbio/syr006 [doi]
PST - ppublish
SO  - Syst Biol. 2011 May;60(3):343-57. doi: 10.1093/sysbio/syr006. Epub 2011 Feb 28.

PMID- 21325220
OWN - NLM
STAT- MEDLINE
DCOM- 20110812
LR  - 20190110
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 3
DP  - 2011 May
TI  - Why the phylogenetic regression appears robust to tree misspecification.
PG  - 245-60
LID - 10.1093/sysbio/syq098 [doi]
AB  - The phylogenetic comparative method uses estimates of evolutionary relationships 
      to explicitly model the covariance structure of interspecific data. By accounting
      for common ancestry, the coevolution between 2 or more traits, as a response to
      one another or to environmental variables, can be studied without confounding
      similarities due to identity by descent. Because the true phylogeny is
      unknowable, an estimate must be used, introducing a source of error into
      phylogenetic comparative analysis that can be difficult to quantify. This
      manuscript aims to elucidate how tree misspecification is propagated through a
      comparative analysis. I focus on the phylogenetic regression under a Brownian
      motion model of evolution and consider the effect of local phylogenetic
      perturbations on the regression fit. Motivated by Felsenstein's method of
      independent contrasts, I derive a matrix square root of the phylogenetic
      covariance matrix that has an obvious phylogenetic interpretation. I use this
      result to transform the perturbed phylogenetic regression model into an ordinary 
      linear regression in which one interpretable point has been affected. The
      simplicity of this formulation allows the contributions of data and phylogeny to 
      be disentangled when studying the effect of tree misspecification.
      Consequentially, I find that branch length misspecification can be easily
      explained in terms of the reweighting of contrast scores between subtrees. An
      analytical consideration of this and other perturbations helps to explain why the
      phylogenetic regression appears generally to be robust to tree misspecification, 
      and I am able to identify conditions under which the regression may not yield
      robust results. I discuss why soft polytomies do not meet these problematic
      conditions, leading to the conclusion that unresolved bifurcations should have
      only modest effects on the regression fit.
FAU - Stone, Eric A
AU  - Stone EA
AD  - Department of Genetics, North Carolina State University, Raleigh, NC 27695-8203, 
      USA. eric stone@ncsu.edu
LA  - eng
GR  - R01 GM070806/GM/NIGMS NIH HHS/United States
GR  - R01 GM070806-06/GM/NIGMS NIH HHS/United States
GR  - R01GM070806/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
DEP - 20110215
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Computer Simulation
MH  - *Linear Models
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Primates/classification
PMC - PMC3078421
EDAT- 2011/02/18 06:00
MHDA- 2011/08/13 06:00
CRDT- 2011/02/18 06:00
PHST- 2011/02/18 06:00 [entrez]
PHST- 2011/02/18 06:00 [pubmed]
PHST- 2011/08/13 06:00 [medline]
AID - syq098 [pii]
AID - 10.1093/sysbio/syq098 [doi]
PST - ppublish
SO  - Syst Biol. 2011 May;60(3):245-60. doi: 10.1093/sysbio/syq098. Epub 2011 Feb 15.

PMID- 21303824
OWN - NLM
STAT- MEDLINE
DCOM- 20110812
LR  - 20110418
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 3
DP  - 2011 May
TI  - Taxon sampling and the optimal rates of evolution for phylogenetic inference.
PG  - 358-65
LID - 10.1093/sysbio/syq097 [doi]
FAU - Townsend, Jeffrey P
AU  - Townsend JP
AD  - Department of Ecology and Evolutionary Biology, Yale University, 165 Prospect
      Street, New Haven, CT 06520-8106, USA. jeffrey.townsend@yale.edu
FAU - Leuenberger, Christoph
AU  - Leuenberger C
LA  - eng
PT  - Comment
PT  - Journal Article
DEP - 20110208
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (RNA, Ribosomal, 28S)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
CON - Syst Biol. 2010 Mar;59(2):226-41. PMID: 20525632
MH  - Animals
MH  - Base Sequence
MH  - Classification/*methods
MH  - DNA Primers/genetics
MH  - Electron Transport Complex IV/genetics
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - RNA, Ribosomal, 28S/genetics
MH  - Research Design
MH  - Sequence Analysis, DNA
MH  - Wasps/*genetics
EDAT- 2011/02/10 06:00
MHDA- 2011/08/13 06:00
CRDT- 2011/02/10 06:00
PHST- 2011/02/10 06:00 [entrez]
PHST- 2011/02/10 06:00 [pubmed]
PHST- 2011/08/13 06:00 [medline]
AID - syq097 [pii]
AID - 10.1093/sysbio/syq097 [doi]
PST - ppublish
SO  - Syst Biol. 2011 May;60(3):358-65. doi: 10.1093/sysbio/syq097. Epub 2011 Feb 8.

PMID- 21303823
OWN - NLM
STAT- MEDLINE
DCOM- 20110812
LR  - 20110418
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 3
DP  - 2011 May
TI  - Are transposable element insertions homoplasy free?: an examination using the
      avian tree of life.
PG  - 375-86
LID - 10.1093/sysbio/syq100 [doi]
FAU - Han, Kin-Lan
AU  - Han KL
AD  - Department of Biology, University of Florida, Gainesville, FL 32611, USA.
      hankin@ufl.edu
FAU - Braun, Edward L
AU  - Braun EL
FAU - Kimball, Rebecca T
AU  - Kimball RT
FAU - Reddy, Sushma
AU  - Reddy S
FAU - Bowie, Rauri C K
AU  - Bowie RC
FAU - Braun, Michael J
AU  - Braun MJ
FAU - Chojnowski, Jena L
AU  - Chojnowski JL
FAU - Hackett, Shannon J
AU  - Hackett SJ
FAU - Harshman, John
AU  - Harshman J
FAU - Huddleston, Christopher J
AU  - Huddleston CJ
FAU - Marks, Ben D
AU  - Marks BD
FAU - Miglia, Kathleen J
AU  - Miglia KJ
FAU - Moore, William S
AU  - Moore WS
FAU - Sheldon, Frederick H
AU  - Sheldon FH
FAU - Steadman, David W
AU  - Steadman DW
FAU - Witt, Christopher C
AU  - Witt CC
FAU - Yuri, Tamaki
AU  - Yuri T
LA  - eng
SI  - GENBANK/HM439436
SI  - GENBANK/HM439437
SI  - GENBANK/HM439438
SI  - GENBANK/HM439439
SI  - GENBANK/HM439440
SI  - GENBANK/HM439441
SI  - GENBANK/HM439442
SI  - GENBANK/HM439443
SI  - GENBANK/HM439444
SI  - GENBANK/HM439445
SI  - GENBANK/HM439446
SI  - GENBANK/HM439447
SI  - GENBANK/HM439448
SI  - GENBANK/HM439449
SI  - GENBANK/HM439450
SI  - GENBANK/HM439451
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110208
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Transposable Elements)
RN  - 0 (HMGN2 Protein)
RN  - 0 (Untranslated Regions)
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Birds/*classification/*genetics
MH  - *DNA Transposable Elements
MH  - Evolution, Molecular
MH  - Exons
MH  - *Genomics
MH  - HMGN2 Protein
MH  - Introns
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
MH  - Untranslated Regions
EDAT- 2011/02/10 06:00
MHDA- 2011/08/13 06:00
CRDT- 2011/02/10 06:00
PHST- 2011/02/10 06:00 [entrez]
PHST- 2011/02/10 06:00 [pubmed]
PHST- 2011/08/13 06:00 [medline]
AID - syq100 [pii]
AID - 10.1093/sysbio/syq100 [doi]
PST - ppublish
SO  - Syst Biol. 2011 May;60(3):375-86. doi: 10.1093/sysbio/syq100. Epub 2011 Feb 8.

PMID- 21296909
OWN - NLM
STAT- MEDLINE
DCOM- 20110812
LR  - 20110418
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 3
DP  - 2011 May
TI  - Bayesian estimation of substitution rates from ancient DNA sequences with low
      information content.
PG  - 366-75
LID - 10.1093/sysbio/syq099 [doi]
FAU - Ho, Simon Y W
AU  - Ho SY
AD  - Centre for Macroevolution and Macroecology, Research School of Biology,
      Australian National University, Canberra, ACT 0200, Australia.
      simon.ho@sydney.edu.au
FAU - Lanfear, Robert
AU  - Lanfear R
FAU - Phillips, Matthew J
AU  - Phillips MJ
FAU - Barnes, Ian
AU  - Barnes I
FAU - Thomas, Jessica A
AU  - Thomas JA
FAU - Kolokotronis, Sergios-Orestis
AU  - Kolokotronis SO
FAU - Shapiro, Beth
AU  - Shapiro B
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110204
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - *Artifacts
MH  - Bayes Theorem
MH  - Calibration
MH  - DNA/*genetics
MH  - *Evolution, Molecular
MH  - Mammals/genetics
MH  - Models, Genetic
MH  - Palaeognathae/genetics
MH  - Zea mays/genetics
EDAT- 2011/02/08 06:00
MHDA- 2011/08/13 06:00
CRDT- 2011/02/08 06:00
PHST- 2011/02/08 06:00 [entrez]
PHST- 2011/02/08 06:00 [pubmed]
PHST- 2011/08/13 06:00 [medline]
AID - syq099 [pii]
AID - 10.1093/sysbio/syq099 [doi]
PST - ppublish
SO  - Syst Biol. 2011 May;60(3):366-75. doi: 10.1093/sysbio/syq099. Epub 2011 Feb 4.

PMID- 21252387
OWN - NLM
STAT- MEDLINE
DCOM- 20110414
LR  - 20110214
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 2
DP  - 2011 Mar
TI  - Consequences of different null models on the tree shape bias of supertree
      methods.
PG  - 218-25
LID - 10.1093/sysbio/syq086 [doi]
FAU - Kupczok, Anne
AU  - Kupczok A
AD  - Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories,
      University of Vienna, Medical University of Vienna, University of Veterinary
      Medicine Vienna, Dr. Bohr-Gasse 9, A-1030 Vienna, Austria. anne.kupczok@ist.ac.at
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110120
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/methods
MH  - *Models, Genetic
MH  - Phylogeny
EDAT- 2011/01/22 06:00
MHDA- 2011/04/16 06:00
CRDT- 2011/01/22 06:00
PHST- 2011/01/22 06:00 [entrez]
PHST- 2011/01/22 06:00 [pubmed]
PHST- 2011/04/16 06:00 [medline]
AID - syq086 [pii]
AID - 10.1093/sysbio/syq086 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Mar;60(2):218-25. doi: 10.1093/sysbio/syq086. Epub 2011 Jan 20.

PMID- 21252386
OWN - NLM
STAT- MEDLINE
DCOM- 20110414
LR  - 20110214
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 2
DP  - 2011 Mar
TI  - Phylogenetic utility of nuclear introns in interfamilial relationships of
      Caniformia (order Carnivora).
PG  - 175-87
LID - 10.1093/sysbio/syq090 [doi]
AB  - The monophyletic group Caniformia (dog-like carnivores) in the order Carnivora
      comprises 9 families. Except for the general consensus for the earliest
      divergence of Canidae and the grouping of Procyonidae and Mustelidae, conflicting
      phylogenetic hypotheses exist for the other caniformian families. In the present 
      study, a data set comprising &gt; 22 kb of 22 nuclear intron loci from 16
      caniformian species is used to investigate the phylogenetic utility of nuclear
      introns in resolving the interfamilial relationships of Caniformia. Our
      phylogenetic analyses support Ailuridae as the sister taxon to a clade containing
      Procyonidae and Mustelidae, with Mephitinae being the sister taxon to all of
      them. The unresolved placements of Ursidae and Pinnipeds here emphasize a need to
      add more data and include more taxa to resolve this problem. The present study
      not only resolves some of the ambiguous relationships in Caniformia phylogeny but
      also shows that the noncoding nuclear markers can offer powerful complementary
      data for estimating the species tree. None of the newly developed introns here
      have previously been used for phylogeny reconstruction, thus increasing the
      spectrum of molecular markers available to mammalian systematics. Interestingly, 
      all the newly developed intron data partitions exhibit intraindividual allele
      heterozygotes (IIAHs). There are 115 cases of IIAHs in total. The incorporation
      of IIAHs into phylogenetic analysis not only provides insights into the
      interfamilial relationships of Caniformia but also identifies two potential
      hybridization events occurred within Ursidae and Otariidae, respectively.
      Finally, the powers and pitfalls of phylogenetics using nuclear introns as
      markers are discussed in the context of Caniformia phylogeny.
FAU - Yu, Li
AU  - Yu L
AD  - Laboratory for Conservation and Utilization of Bio-Resources and Key Laboratory
      for Microbial Resources of the Ministry of Education, Yunnan University, Kunming 
      650091, China. yuli1220@yahoo.com.cn
FAU - Luan, Peng-Tao
AU  - Luan PT
FAU - Jin, Wei
AU  - Jin W
FAU - Ryder, Oliver A
AU  - Ryder OA
FAU - Chemnick, Leona G
AU  - Chemnick LG
FAU - Davis, Heidi A
AU  - Davis HA
FAU - Zhang, Ya-Ping
AU  - Zhang YP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110120
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Carnivora/*classification/*genetics
MH  - Cell Nucleus/*genetics
MH  - Heterozygote
MH  - Humans
MH  - *Introns
MH  - *Phylogeny
EDAT- 2011/01/22 06:00
MHDA- 2011/04/16 06:00
CRDT- 2011/01/22 06:00
PHST- 2011/01/22 06:00 [entrez]
PHST- 2011/01/22 06:00 [pubmed]
PHST- 2011/04/16 06:00 [medline]
AID - syq090 [pii]
AID - 10.1093/sysbio/syq090 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Mar;60(2):175-87. doi: 10.1093/sysbio/syq090. Epub 2011 Jan 20.

PMID- 21252385
OWN - NLM
STAT- MEDLINE
DCOM- 20110414
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 2
DP  - 2011 Mar
TI  - Biologically inspired phylogenetic models strongly outperform the no common
      mechanism model.
PG  - 225-32
LID - 10.1093/sysbio/syq089 [doi]
AB  - But Tuffley and Steel (1997) introduced a model called No Common Mechanism (NCM),
      in which characters may-but are not required to-vary their relative rates
      independently, both within and between branches. Because the independent
      variation is taken only as a possibility, not as a requirement, NCM would apply
      to almost any situation, and so may be accepted as realistic. This is useful
      because Tuffley and Steel also showed that maximum likelihood under NCM selects
      the same trees as does parsimony. With the realistic NCM in the background, then,
      most parsimonious trees have greatest power to explain available observations.
      -Farris (2008).
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
AD  - Department of Integrative Biology, University of California, Berkeley, CA
      94720-3140, USA. johnh@berkeley.edu
FAU - Alfaro, Michael E
AU  - Alfaro ME
FAU - Suchard, Marc A
AU  - Suchard MA
LA  - eng
GR  - R01 GM069801/GM/NIGMS NIH HHS/United States
GR  - R01 GM086887/GM/NIGMS NIH HHS/United States
GR  - GM-069801/GM/NIGMS NIH HHS/United States
GR  - GM-086887/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110120
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
PMC - PMC3038349
EDAT- 2011/01/22 06:00
MHDA- 2011/04/16 06:00
CRDT- 2011/01/22 06:00
PHST- 2011/01/22 06:00 [entrez]
PHST- 2011/01/22 06:00 [pubmed]
PHST- 2011/04/16 06:00 [medline]
AID - syq089 [pii]
AID - 10.1093/sysbio/syq089 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Mar;60(2):225-32. doi: 10.1093/sysbio/syq089. Epub 2011 Jan 20.

PMID- 21248369
OWN - NLM
STAT- MEDLINE
DCOM- 20110414
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 2
DP  - 2011 Mar
TI  - Coalescent histories on phylogenetic networks and detection of hybridization
      despite incomplete lineage sorting.
PG  - 138-49
LID - 10.1093/sysbio/syq084 [doi]
AB  - Analyses of the increasingly available genomic data continue to reveal the extent
      of hybridization and its role in the evolutionary diversification of various
      groups of species. We show, through extensive coalescent-based simulations of
      multilocus data sets on phylogenetic networks, how divergence times before and
      after hybridization events can result in incomplete lineage sorting with gene
      tree incongruence signatures identical to those exhibited by hybridization.
      Evolutionary analysis of such data under the assumption of a species tree model
      can miss all hybridization events, whereas analysis under the assumption of a
      species network model would grossly overestimate hybridization events. These
      issues necessitate a paradigm shift in evolutionary analysis under these
      scenarios, from a model that assumes a priori a single source of gene tree
      incongruence to one that integrates multiple sources in a unifying framework. We 
      propose a framework of coalescence within the branches of a phylogenetic network 
      and show how this framework can be used to detect hybridization despite
      incomplete lineage sorting. We apply the model to simulated data and show that
      the signature of hybridization can be revealed as long as the interval between
      the divergence times of the species involved in hybridization is not too small.
      We reanalyze a data set of 106 loci from 7 in-group Saccharomyces species for
      which a species tree with no hybridization has been reported in the literature.
      Our analysis supports the hypothesis that hybridization occurred during the
      evolution of this group, explaining a large amount of the incongruence in the
      data. Our findings show that an integrative approach to gene tree incongruence
      and its reconciliation is needed. Our framework will help in systematically
      analyzing genomic data for the occurrence of hybridization and elucidating its
      evolutionary role.
FAU - Yu, Yun
AU  - Yu Y
AD  - Department of Computer Science, Rice University, 6100 Main Street, Houston, TX
      77005, USA.
FAU - Than, Cuong
AU  - Than C
FAU - Degnan, James H
AU  - Degnan JH
FAU - Nakhleh, Luay
AU  - Nakhleh L
LA  - eng
GR  - R01LM009494/LM/NLM NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110119
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Genes, Fungal
MH  - *Hybridization, Genetic
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Saccharomyces/*classification/*genetics
PMC - PMC3167682
EDAT- 2011/01/21 06:00
MHDA- 2011/04/16 06:00
CRDT- 2011/01/21 06:00
PHST- 2011/01/21 06:00 [entrez]
PHST- 2011/01/21 06:00 [pubmed]
PHST- 2011/04/16 06:00 [medline]
AID - syq084 [pii]
AID - 10.1093/sysbio/syq084 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Mar;60(2):138-49. doi: 10.1093/sysbio/syq084. Epub 2011 Jan 19.

PMID- 21239388
OWN - NLM
STAT- MEDLINE
DCOM- 20110414
LR  - 20110214
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 2
DP  - 2011 Mar
TI  - Recent and rapid speciation with limited morphological disparity in the genus
      Rattus.
PG  - 188-203
LID - 10.1093/sysbio/syq092 [doi]
AB  - Recent and rapid radiations provide rich material to examine the factors that
      drive speciation. Most recent and rapid radiations that have been
      well-characterized involve species that exhibit overt ecomorphological
      differences associated with clear partitioning of ecological niches in sympatry. 
      The most diverse genus of rodents, Rattus (66 species), evolved fairly recently, 
      but without overt ecomorphological divergence among species. We used multilocus
      molecular phylogenetic data and five fossil calibrations to estimate the tempo of
      diversification in Rattus, and their radiation on Australia and New Guinea
      (Sahul, 24 species). Based on our analyses, the genus Rattus originated at a date
      centered on the Pliocene-Pleistocene boundary (1.84-3.17 Ma) with a subsequent
      colonization of Sahul in the middle Pleistocene (0.85-1.28 Ma). Given these
      dates, the per lineage diversification rates in Rattus and Sahulian Rattus are
      among the highest reported for vertebrates (1.1-1.9 and 1.6-3.0 species per
      lineage per million years, respectively). Despite their rapid diversification,
      Rattus display little ecomorphological divergence among species and do not fit
      clearly into current models of adaptive radiations. Lineage through time plots
      and ancestral state reconstruction of ecological characters suggest that
      diversification of Sahulian Rattus was most rapid early on as they expanded into 
      novel ecological conditions. However, rapid lineage accumulation occurred even
      when morphological disparity within lineages was low suggesting that future
      studies consider other phenotypes in the diversification of Rattus.
FAU - Rowe, Kevin C
AU  - Rowe KC
AD  - Museum of Vertebrate Zoology, University of California, Berkeley, Berkeley, CA
      94720-3160, USA. rattus@berkeley.edu
FAU - Aplin, Ken P
AU  - Aplin KP
FAU - Baverstock, Peter R
AU  - Baverstock PR
FAU - Moritz, Craig
AU  - Moritz C
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110113
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Fossils
MH  - *Genetic Speciation
MH  - Murinae/classification/genetics
MH  - *Phylogeny
MH  - Rats/anatomy &amp; histology/*classification/*genetics
EDAT- 2011/01/18 06:00
MHDA- 2011/04/16 06:00
CRDT- 2011/01/18 06:00
PHST- 2011/01/18 06:00 [entrez]
PHST- 2011/01/18 06:00 [pubmed]
PHST- 2011/04/16 06:00 [medline]
AID - syq092 [pii]
AID - 10.1093/sysbio/syq092 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Mar;60(2):188-203. doi: 10.1093/sysbio/syq092. Epub 2011 Jan 13.

PMID- 21233085
OWN - NLM
STAT- MEDLINE
DCOM- 20110414
LR  - 20110214
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 2
DP  - 2011 Mar
TI  - What's in a likelihood? Simple models of protein evolution and the contribution
      of structurally viable reconstructions to the likelihood.
PG  - 161-74
LID - 10.1093/sysbio/syq088 [doi]
AB  - Most phylogenetic models of protein evolution assume that sites are independent
      and identically distributed. Interactions between sites are ignored, and the
      likelihood can be conveniently calculated as the product of the individual site
      likelihoods. The calculation considers all possible transition paths (also called
      substitution histories or mappings) that are consistent with the observed states 
      at the terminals, and the probability density of any particular reconstruction
      depends on the substitution model. The likelihood is the integral of the
      probability density of each substitution history taken over all possible
      histories that are consistent with the observed data. We investigated the extent 
      to which transition paths that are incompatible with a protein's
      three-dimensional structure contribute to the likelihood. Several empirical amino
      acid models were tested for sequence pairs of different degrees of divergence.
      When simulating substitutional histories starting from a real sequence, the
      structural integrity of the simulated sequences quickly disintegrated. This
      result indicates that simple models are clearly unable to capture the constraints
      on sequence evolution. However, when we sampled transition paths between real
      sequences from the posterior probability distribution according to these same
      models, we found that the sampled histories were largely consistent with the
      tertiary structure. This suggests that simple empirical substitution models may
      be adequate for interpolating changes between observed sequences during
      phylogenetic inference despite the fact that the models cannot predict the
      effects of structural constraints from first principles. This study is
      significant because it provides a quantitative assessment of the biological
      realism of substitution models from the perspective of protein structure, and it 
      provides insight on the prospects for improving models of protein sequence
      evolution.
FAU - Lakner, Clemens
AU  - Lakner C
AD  - Department of Biological Science, Section of Ecology and Evolution, Florida State
      University, Tallahassee, FL 32306-4120, USA. lakner@scs.fsu.edu
FAU - Holder, Mark T
AU  - Holder MT
FAU - Goldman, Nick
AU  - Goldman N
FAU - Naylor, Gavin J P
AU  - Naylor GJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20110112
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Proteins)
SB  - IM
MH  - Animals
MH  - *Evolution, Molecular
MH  - Humans
MH  - Likelihood Functions
MH  - Phylogeny
MH  - Probability
MH  - Proteins/*chemistry/*genetics
EDAT- 2011/01/15 06:00
MHDA- 2011/04/16 06:00
CRDT- 2011/01/15 06:00
PHST- 2011/01/15 06:00 [entrez]
PHST- 2011/01/15 06:00 [pubmed]
PHST- 2011/04/16 06:00 [medline]
AID - syq088 [pii]
AID - 10.1093/sysbio/syq088 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Mar;60(2):161-74. doi: 10.1093/sysbio/syq088. Epub 2011 Jan 12.

PMID- 21212163
OWN - NLM
STAT- MEDLINE
DCOM- 20110414
LR  - 20110214
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 2
DP  - 2011 Mar
TI  - Conservative supertrees.
PG  - 232-8
LID - 10.1093/sysbio/syq091 [doi]
FAU - McMorris, F R
AU  - McMorris FR
AD  - Department of Applied Mathematics, Illinois Institute of Technology, Chicago, IL 
      60616-3793, USA. mcmorris@iit.edu
FAU - Wilkinson, Mark
AU  - Wilkinson M
LA  - eng
GR  - 40/18385/Biotechnology and Biological Sciences Research Council/United Kingdom
GR  - BBG 0247071/Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20110106
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Cluster Analysis
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2011/01/08 06:00
MHDA- 2011/04/16 06:00
CRDT- 2011/01/08 06:00
PHST- 2011/01/08 06:00 [entrez]
PHST- 2011/01/08 06:00 [pubmed]
PHST- 2011/04/16 06:00 [medline]
AID - syq091 [pii]
AID - 10.1093/sysbio/syq091 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Mar;60(2):232-8. doi: 10.1093/sysbio/syq091. Epub 2011 Jan 6.

PMID- 21187451
OWN - NLM
STAT- MEDLINE
DCOM- 20110414
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 2
DP  - 2011 Mar
TI  - Improving marginal likelihood estimation for Bayesian phylogenetic model
      selection.
PG  - 150-60
LID - 10.1093/sysbio/syq085 [doi]
AB  - The marginal likelihood is commonly used for comparing different evolutionary
      models in Bayesian phylogenetics and is the central quantity used in computing
      Bayes Factors for comparing model fit. A popular method for estimating marginal
      likelihoods, the harmonic mean (HM) method, can be easily computed from the
      output of a Markov chain Monte Carlo analysis but often greatly overestimates the
      marginal likelihood. The thermodynamic integration (TI) method is much more
      accurate than the HM method but requires more computation. In this paper, we
      introduce a new method, steppingstone sampling (SS), which uses importance
      sampling to estimate each ratio in a series (the "stepping stones") bridging the 
      posterior and prior distributions. We compare the performance of the SS approach 
      to the TI and HM methods in simulation and using real data. We conclude that the 
      greatly increased accuracy of the SS and TI methods argues for their use instead 
      of the HM method, despite the extra computation needed.
FAU - Xie, Wangang
AU  - Xie W
AD  - Abbott, 100 Abbott Park, R436/AP9A-2, Abbott Park, IL 60064, USA.
FAU - Lewis, Paul O
AU  - Lewis PO
FAU - Fan, Yu
AU  - Fan Y
FAU - Kuo, Lynn
AU  - Kuo L
FAU - Chen, Ming-Hui
AU  - Chen MH
LA  - eng
GR  - R01 GM070335/GM/NIGMS NIH HHS/United States
GR  - CA 74015/CA/NCI NIH HHS/United States
GR  - GM 70335/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20101227
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Likelihood Functions
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
PMC - PMC3038348
EDAT- 2010/12/29 06:00
MHDA- 2011/04/16 06:00
CRDT- 2010/12/29 06:00
PHST- 2010/12/29 06:00 [entrez]
PHST- 2010/12/29 06:00 [pubmed]
PHST- 2011/04/16 06:00 [medline]
AID - syq085 [pii]
AID - 10.1093/sysbio/syq085 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Mar;60(2):150-60. doi: 10.1093/sysbio/syq085. Epub 2010 Dec 27.

PMID- 21186249
OWN - NLM
STAT- MEDLINE
DCOM- 20110414
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 2
DP  - 2011 Mar
TI  - Genome-scale phylogenetics: inferring the plant tree of life from 18,896 gene
      trees.
PG  - 117-25
LID - 10.1093/sysbio/syq072 [doi]
AB  - Phylogenetic analyses using genome-scale data sets must confront incongruence
      among gene trees, which in plants is exacerbated by frequent gene duplications
      and losses. Gene tree parsimony (GTP) is a phylogenetic optimization criterion in
      which a species tree that minimizes the number of gene duplications induced among
      a set of gene trees is selected. The run time performance of previous
      implementations has limited its use on large-scale data sets. We used new
      software that incorporates recent algorithmic advances to examine the performance
      of GTP on a plant data set consisting of 18,896 gene trees containing 510,922
      protein sequences from 136 plant taxa (giving a combined alignment length of &gt;2.9
      million characters). The relationships inferred from the GTP analysis were
      largely consistent with previous large-scale studies of backbone plant phylogeny 
      and resolved some controversial nodes. The placement of taxa that were present in
      few gene trees generally varied the most among GTP bootstrap replicates.
      Excluding these taxa either before or after the GTP analysis revealed high levels
      of phylogenetic support across plants. The analyses supported magnoliids sister
      to a eudicot + monocot clade and did not support the eurosid I and II clades.
      This study presents a nuclear genomic perspective on the broad-scale phylogenic
      relationships among plants, and it demonstrates that nuclear genes with a history
      of duplication and loss can be phylogenetically informative for resolving the
      plant tree of life.
FAU - Burleigh, J Gordon
AU  - Burleigh JG
AD  - Department of Biology, University of Florida, Gainesville, FL 32609, USA.
      gburleigh@ufl.edu
FAU - Bansal, Mukul S
AU  - Bansal MS
FAU - Eulenstein, Oliver
AU  - Eulenstein O
FAU - Hartmann, Stefanie
AU  - Hartmann S
FAU - Wehe, Andre
AU  - Wehe A
FAU - Vision, Todd J
AU  - Vision TJ
LA  - eng
GR  - R01-GM078991/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, P.H.S.
DEP - 20101224
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Expressed Sequence Tags
MH  - Genomics
MH  - *Phylogeny
MH  - Plants/*classification/*genetics
PMC - PMC3038350
EDAT- 2010/12/28 06:00
MHDA- 2011/04/16 06:00
CRDT- 2010/12/28 06:00
PHST- 2010/12/28 06:00 [entrez]
PHST- 2010/12/28 06:00 [pubmed]
PHST- 2011/04/16 06:00 [medline]
AID - syq072 [pii]
AID - 10.1093/sysbio/syq072 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Mar;60(2):117-25. doi: 10.1093/sysbio/syq072. Epub 2010 Dec 24.

PMID- 21169241
OWN - NLM
STAT- MEDLINE
DCOM- 20110414
LR  - 20110214
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 2
DP  - 2011 Mar
TI  - Old taxa on young islands: a critique of the use of island age to date
      island-endemic clades and calibrate phylogenies.
PG  - 204-18
LID - 10.1093/sysbio/syq075 [doi]
FAU - Heads, Michael
AU  - Heads M
AD  - Buffalo Museum of Science, 1020 Humboldt Parkway, Buffalo, NY 14211-1293, USA.
      michael.heads@yahoo.com
LA  - eng
PT  - Journal Article
DEP - 20101216
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Molecular Biology
MH  - Pacific Islands
MH  - *Phylogeny
MH  - Phylogeography
EDAT- 2010/12/21 06:00
MHDA- 2011/04/16 06:00
CRDT- 2010/12/21 06:00
PHST- 2010/12/21 06:00 [entrez]
PHST- 2010/12/21 06:00 [pubmed]
PHST- 2011/04/16 06:00 [medline]
AID - syq075 [pii]
AID - 10.1093/sysbio/syq075 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Mar;60(2):204-18. doi: 10.1093/sysbio/syq075. Epub 2010 Dec 16.

PMID- 21088008
OWN - NLM
STAT- MEDLINE
DCOM- 20110217
LR  - 20101207
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 1
DP  - 2011 Jan
TI  - Species delimitation under the general lineage concept: an empirical example
      using wild North American hops (Cannabaceae: Humulus lupulus).
PG  - 45-59
LID - 10.1093/sysbio/syq056 [doi]
AB  - There is an emerging consensus that the intent of most species concepts is to
      identify evolutionarily distinct lineages. However, the criteria used to identify
      lineages differ among concepts depending on the perceived importance of various
      attributes of evolving populations. We have examined five different species
      criteria to ask whether the three taxonomic varieties of Humulus lupulus (hops)
      native to North America are distinct lineages. Three criteria (monophyly, absence
      of genetic intermediates, and diagnosability) focus on evolutionary patterns and 
      two (intrinsic reproductive isolation and niche specialization) consider
      evolutionary processes. Phylogenetic analysis of amplified fragment length
      polymorphism (AFLP) data under a relaxed molecular clock, a stochastic Dollo
      substitution model, and parsimony identified all varieties as monophyletic, thus 
      they satisfy the monophyly criterion for species delimitation. Principal
      coordinate analysis and a Bayesian assignment procedure revealed deep genetic
      subdivisions and little admixture between varieties, indicating an absence of
      genetic intermediates and compliance with the genotypic cluster species
      criterion. Diagnostic morphological and AFLP characters were found for all
      varieties, thus they meet the diagnosability criterion. Natural history
      information suggests that reproductive isolating barriers may have evolved in
      var. pubescens, potentially qualifying it as a species under a criterion of
      intrinsic reproductive isolation. Environmental niche modeling showed that the
      preferred habitat of var. neomexicanus is climatically unique, suggesting niche
      specialization and thus compliance with an ecological species criterion.
      Isolation by distance coupled with imperfect sampling can lead to erroneous
      lineage identification using some species criteria. Compliance with complementary
      pattern- and process-oriented criteria provides powerful corroboration for a
      species hypothesis and mitigates the necessity for comprehensive sampling of the 
      entire species range, a practical impossibility in many systems. We hypothesize
      that var. pubescens maintains its genetic identity, despite substantial niche
      overlap with var. lupuloides, via the evolution of partial reproductive isolating
      mechanisms. Variety neomexicanus, conversely, will likely persist as a distinct
      lineage, regardless of limited gene flow with vars. lupuloides and pubescens
      because of ecological isolation--adaptation to the unique conditions of the Rocky
      Mountain cordillera. Thus, we support recognition of vars. neomexicanus and
      pubescens as species, but delay making a recommendation for var. lupuloides until
      sampling of genetic variation is complete or a stable biological process can be
      identified to explain its observed genetic divergence.
FAU - Reeves, Patrick A
AU  - Reeves PA
AD  - United States Department of Agriculture, Agricultural Research Service, National 
      Center for Genetic Resources Preservation, 1111 South Mason Street, Fort Collins,
      CO 80521, USA.
FAU - Richards, Christopher M
AU  - Richards CM
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20101117
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Amplified Fragment Length Polymorphism Analysis/*methods
MH  - DNA, Plant/genetics
MH  - Ecosystem
MH  - Evolution, Molecular
MH  - Gene Flow
MH  - Genetic Speciation
MH  - Genetic Variation
MH  - Genetics, Population
MH  - Humulus/*classification/genetics/physiology
MH  - Models, Genetic
MH  - North America
MH  - Phylogeny
EDAT- 2010/11/23 06:00
MHDA- 2011/02/18 06:00
CRDT- 2010/11/20 06:00
PHST- 2010/11/20 06:00 [entrez]
PHST- 2010/11/23 06:00 [pubmed]
PHST- 2011/02/18 06:00 [medline]
AID - syq056 [pii]
AID - 10.1093/sysbio/syq056 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jan;60(1):45-59. doi: 10.1093/sysbio/syq056. Epub 2010 Nov 17.

PMID- 21088009
OWN - NLM
STAT- MEDLINE
DCOM- 20110414
LR  - 20110214
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 2
DP  - 2011 Mar
TI  - The accuracy of species tree estimation under simulation: a comparison of
      methods.
PG  - 126-37
LID - 10.1093/sysbio/syq073 [doi]
AB  - Numerous simulation studies have investigated the accuracy of phylogenetic
      inference of gene trees under maximum parsimony, maximum likelihood, and Bayesian
      techniques. The relative accuracy of species tree inference methods under
      simulation has received less study. The number of analytical techniques available
      for inferring species trees is increasing rapidly, and in this paper, we compare 
      the performance of several species tree inference techniques at estimating recent
      species divergences using computer simulation. Simulating gene trees within
      species trees of different shapes and with varying tree lengths (T) and
      population sizes (), and evolving sequences on those gene trees, allows us to
      determine how phylogenetic accuracy changes in relation to different levels of
      deep coalescence and phylogenetic signal. When the probability of discordance
      between the gene trees and the species tree is high (i.e., T is small and/or is
      large), Bayesian species tree inference using the multispecies coalescent (BEST) 
      outperforms other methods. The performance of all methods improves as the total
      length of the species tree is increased, which reflects the combined benefits of 
      decreasing the probability of discordance between species trees and gene trees
      and gaining more accurate estimates for gene trees. Decreasing the probability of
      deep coalescences by reducing also leads to accuracy gains for most methods.
      Increasing the number of loci from 10 to 100 improves accuracy under difficult
      demographic scenarios (i.e., coalescent units &lt;/= 4N(e)), but 10 loci are
      adequate for estimating the correct species tree in cases where deep coalescence 
      is limited or absent. In general, the correlation between the phylogenetic
      accuracy and the posterior probability values obtained from BEST is high,
      although posterior probabilities are overestimated when the prior distribution
      for is misspecified.
FAU - Leache, Adam D
AU  - Leache AD
AD  - Genome Center and Department of Evolution and Ecology, University of California, 
      Davis, CA 95616, USA. leache@uw.edu
FAU - Rannala, Bruce
AU  - Rannala B
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20101118
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Genetic Speciation
MH  - *Phylogeny
MH  - Plants/classification/genetics
EDAT- 2010/11/20 06:00
MHDA- 2011/04/16 06:00
CRDT- 2010/11/20 06:00
PHST- 2010/11/20 06:00 [entrez]
PHST- 2010/11/20 06:00 [pubmed]
PHST- 2011/04/16 06:00 [medline]
AID - syq073 [pii]
AID - 10.1093/sysbio/syq073 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Mar;60(2):126-37. doi: 10.1093/sysbio/syq073. Epub 2010 Nov 18.

PMID- 21084501
OWN - NLM
STAT- MEDLINE
DCOM- 20110217
LR  - 20110606
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 1
DP  - 2011 Jan
TI  - Can we avoid "SIN" in the house of "no common mechanism"?
PG  - 96-109
LID - 10.1093/sysbio/syq069 [doi]
FAU - Steel, Mike
AU  - Steel M
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, Biomathematics Research 
      Centre, University of Canterbury, Private Bag 4800, Christchurch, New Zealand.
      m.steel@math.canterbury.ac.nz
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20101117
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2011 May;60(3):392
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
EDAT- 2010/11/19 06:00
MHDA- 2011/02/18 06:00
CRDT- 2010/11/19 06:00
PHST- 2010/11/19 06:00 [entrez]
PHST- 2010/11/19 06:00 [pubmed]
PHST- 2011/02/18 06:00 [medline]
AID - syq069 [pii]
AID - 10.1093/sysbio/syq069 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jan;60(1):96-109. doi: 10.1093/sysbio/syq069. Epub 2010 Nov 17.

PMID- 21081482
OWN - NLM
STAT- MEDLINE
DCOM- 20110217
LR  - 20101207
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 1
DP  - 2011 Jan
TI  - Two stationary nonhomogeneous Markov models of nucleotide sequence evolution.
PG  - 74-86
LID - 10.1093/sysbio/syq076 [doi]
AB  - The general Markov model (GMM) of nucleotide substitution does not assume the
      evolutionary process to be stationary, reversible, or homogeneous. The GMM can be
      simplified by assuming the evolutionary process to be stationary. A stationary
      GMM is appropriate for analyses of phylogenetic data sets that are
      compositionally homogeneous; a data set is considered to be compositionally
      homogeneous if a statistical test does not detect significant differences in the 
      marginal distributions of the sequences. Though the general time-reversible (GTR)
      model assumes stationarity, it also assumes reversibility and homogeneity. We
      propose two new stationary and nonhomogeneous models--one constrains the GMM to
      be reversible, whereas the other does not. The two models, coupled with the GTR
      model, comprise a set of nested models that can be used to test the assumptions
      of reversibility and homogeneity for stationary processes. The two models are
      extended to incorporate invariable sites and used to analyze a seven-taxon
      hominoid data set that displays compositional homogeneity. We show that within
      the class of stationary models, a nonhomogeneous model fits the hominoid data
      better than the GTR model. We note that if one considers a wider set of models
      that are not constrained to be stationary, then an even better fit can be
      obtained for the hominoid data. However, the methods for reducing model
      complexity from an extremely large set of nonstationary models are yet to be
      developed.
FAU - Jayaswal, Vivek
AU  - Jayaswal V
AD  - School of Mathematics and Statistics, University of Sydney, Sydney, NSW 2006,
      Australia.
FAU - Jermiin, Lars S
AU  - Jermiin LS
FAU - Poladian, Leon
AU  - Poladian L
FAU - Robinson, John
AU  - Robinson J
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20101116
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Hominidae/*classification/genetics
MH  - Hylobates/*classification/genetics
MH  - Macaca/*classification/genetics
MH  - Markov Chains
MH  - Models, Genetic
MH  - Phylogeny
EDAT- 2010/11/18 06:00
MHDA- 2011/02/18 06:00
CRDT- 2010/11/18 06:00
PHST- 2010/11/18 06:00 [entrez]
PHST- 2010/11/18 06:00 [pubmed]
PHST- 2011/02/18 06:00 [medline]
AID - syq076 [pii]
AID - 10.1093/sysbio/syq076 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jan;60(1):74-86. doi: 10.1093/sysbio/syq076. Epub 2010 Nov 16.

PMID- 21081481
OWN - NLM
STAT- MEDLINE
DCOM- 20110217
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 1
DP  - 2011 Jan
TI  - Quantifying the impact of dependent evolution among sites in phylogenetic
      inference.
PG  - 60-73
LID - 10.1093/sysbio/syq074 [doi]
AB  - Nearly all commonly used methods of phylogenetic inference assume that characters
      in an alignment evolve independently of one another. This assumption is
      attractive for simplicity and computational tractability but is not biologically 
      reasonable for RNAs and proteins that have secondary and tertiary structures.
      Here, we simulate RNA and protein-coding DNA sequence data under a general model 
      of dependence in order to assess the robustness of traditional methods of
      phylogenetic inference to violation of the assumption of independence among
      sites. We find that the accuracy of independence-assuming methods is reduced by
      the dependence among sites; for proteins this reduction is relatively mild, but
      for RNA this reduction may be substantial. We introduce the concept of effective 
      sequence length and its utility for considering information content in
      phylogenetics.
FAU - Nasrallah, Chris A
AU  - Nasrallah CA
AD  - Department of Integrative Biology, University of California, Berkeley, 3060
      Valley Life Sciences Building #3140, Berkeley, CA 94720-3140, USA.
      nasrallah@berkeley.edu
FAU - Mathews, David H
AU  - Mathews DH
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
LA  - eng
GR  - R01 GM069801/GM/NIGMS NIH HHS/United States
GR  - MCB-0075404/PHS HHS/United States
GR  - R01GM076485/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
DEP - 20101115
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Myoglobin)
RN  - 0 (Proteins)
RN  - 63231-63-0 (RNA)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bombyx/genetics
MH  - Computer Simulation
MH  - DNA/chemistry/genetics
MH  - Escherichia coli/genetics
MH  - *Evolution, Molecular
MH  - *Models, Genetic
MH  - Myoglobin/genetics
MH  - Nucleic Acid Conformation
MH  - *Phylogeny
MH  - Protein Conformation
MH  - Proteins/*chemistry/*genetics
MH  - RNA/*chemistry/*genetics
MH  - Sequence Alignment/methods
MH  - Sperm Whale/genetics
PMC - PMC2997629
EDAT- 2010/11/18 06:00
MHDA- 2011/02/18 06:00
CRDT- 2010/11/18 06:00
PHST- 2010/11/18 06:00 [entrez]
PHST- 2010/11/18 06:00 [pubmed]
PHST- 2011/02/18 06:00 [medline]
AID - syq074 [pii]
AID - 10.1093/sysbio/syq074 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jan;60(1):60-73. doi: 10.1093/sysbio/syq074. Epub 2010 Nov 15.

PMID- 21068445
OWN - NLM
STAT- MEDLINE
DCOM- 20110217
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 1
DP  - 2011 Jan
TI  - Comparative performance of supertree algorithms in large data sets using the
      soapberry family (Sapindaceae) as a case study.
PG  - 32-44
LID - 10.1093/sysbio/syq057 [doi]
AB  - For the last 2 decades, supertree reconstruction has been an active field of
      research and has seen the development of a large number of major algorithms.
      Because of the growing popularity of the supertree methods, it has become
      necessary to evaluate the performance of these algorithms to determine which are 
      the best options (especially with regard to the supermatrix approach that is
      widely used). In this study, seven of the most commonly used supertree methods
      are investigated by using a large empirical data set (in terms of number of taxa 
      and molecular markers) from the worldwide flowering plant family Sapindaceae.
      Supertree methods were evaluated using several criteria: similarity of the
      supertrees with the input trees, similarity between the supertrees and the total 
      evidence tree, level of resolution of the supertree and computational time
      required by the algorithm. Additional analyses were also conducted on a reduced
      data set to test if the performance levels were affected by the heuristic
      searches rather than the algorithms themselves. Based on our results, two main
      groups of supertree methods were identified: on one hand, the matrix
      representation with parsimony (MRP), MinFlip, and MinCut methods performed well
      according to our criteria, whereas the average consensus, split fit, and most
      similar supertree methods showed a poorer performance or at least did not behave 
      the same way as the total evidence tree. Results for the super distance matrix,
      that is, the most recent approach tested here, were promising with at least one
      derived method performing as well as MRP, MinFlip, and MinCut. The output of each
      method was only slightly improved when applied to the reduced data set,
      suggesting a correct behavior of the heuristic searches and a relatively low
      sensitivity of the algorithms to data set sizes and missing data. Results also
      showed that the MRP analyses could reach a high level of quality even when using 
      a simple heuristic search strategy, with the exception of MRP with Purvis coding 
      scheme and reversible parsimony. The future of supertrees lies in the
      implementation of a standardized heuristic search for all methods and the
      increase in computing power to handle large data sets. The latter would prove to 
      be particularly useful for promising approaches such as the maximum quartet fit
      method that yet requires substantial computing power.
FAU - Buerki, Sven
AU  - Buerki S
AD  - Real Jardin Botanico, Department of Biodiversity and Conservation, CSIC, Plaza de
      Murillo 2, 28014 Madrid, Spain. s.buerki@kew.org
FAU - Forest, Felix
AU  - Forest F
FAU - Salamin, Nicolas
AU  - Salamin N
FAU - Alvarez, Nadir
AU  - Alvarez N
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20101110
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - DNA, Plant/genetics
MH  - Magnoliopsida/classification/genetics
MH  - Phylogeny
MH  - Sapindaceae/*classification/genetics
EDAT- 2010/11/12 06:00
MHDA- 2011/02/18 06:00
CRDT- 2010/11/12 06:00
PHST- 2010/11/12 06:00 [entrez]
PHST- 2010/11/12 06:00 [pubmed]
PHST- 2011/02/18 06:00 [medline]
AID - syq057 [pii]
AID - 10.1093/sysbio/syq057 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jan;60(1):32-44. doi: 10.1093/sysbio/syq057. Epub 2010 Nov 10.

PMID- 21060067
OWN - NLM
STAT- MEDLINE
DCOM- 20110217
LR  - 20101207
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 1
DP  - 2011 Jan
TI  - The potential role of androgenesis in cytoplasmic-nuclear phylogenetic
      discordance.
PG  - 87-96
LID - 10.1093/sysbio/syq070 [doi]
FAU - Hedtke, Shannon M
AU  - Hedtke SM
AD  - Section of Integrative Biology, University of Texas at Austin, Austin, TX 78712, 
      USA. s.hedtke@mail.utexas.edu
FAU - Hillis, David M
AU  - Hillis DM
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20101108
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Bivalvia/classification/genetics
MH  - Cell Nucleus/*genetics
MH  - *Extrachromosomal Inheritance
MH  - Female
MH  - Male
MH  - *Phylogeny
MH  - Plants/classification/genetics
MH  - *Reproduction, Asexual
MH  - Sex Determination Processes
EDAT- 2010/11/10 06:00
MHDA- 2011/02/18 06:00
CRDT- 2010/11/10 06:00
PHST- 2010/11/10 06:00 [entrez]
PHST- 2010/11/10 06:00 [pubmed]
PHST- 2011/02/18 06:00 [medline]
AID - syq070 [pii]
AID - 10.1093/sysbio/syq070 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jan;60(1):87-96. doi: 10.1093/sysbio/syq070. Epub 2010 Nov 8.

PMID- 21051775
OWN - NLM
STAT- MEDLINE
DCOM- 20110217
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 1
DP  - 2011 Jan
TI  - Dating primate divergences through an integrated analysis of palaeontological and
      molecular data.
PG  - 16-31
LID - 10.1093/sysbio/syq054 [doi]
AB  - Estimation of divergence times is usually done using either the fossil record or 
      sequence data from modern species. We provide an integrated analysis of
      palaeontological and molecular data to give estimates of primate divergence times
      that utilize both sources of information. The number of preserved primate species
      discovered in the fossil record, along with their geological age distribution, is
      combined with the number of extant primate species to provide initial estimates
      of the primate and anthropoid divergence times. This is done by using a
      stochastic forwards-modeling approach where speciation and fossil preservation
      and discovery are simulated forward in time. We use the posterior distribution
      from the fossil analysis as a prior distribution on node ages in a molecular
      analysis. Sequence data from two genomic regions (CFTR on human chromosome 7 and 
      the CYP7A1 region on chromosome 8) from 15 primate species are used with the
      birth-death model implemented in mcmctree in PAML to infer the posterior
      distribution of the ages of 14 nodes in the primate tree. We find that these age 
      estimates are older than previously reported dates for all but one of these
      nodes. To perform the inference, a new approximate Bayesian computation (ABC)
      algorithm is introduced, where the structure of the model can be exploited in an 
      ABC-within-Gibbs algorithm to provide a more efficient analysis.
FAU - Wilkinson, Richard D
AU  - Wilkinson RD
AD  - School of Mathmatical Sciences, University of Nottingham, University Park,
      Nottingham NG7 2RD, UK. r.d.wilkinson@nottingham.ac.uk
FAU - Steiper, Michael E
AU  - Steiper ME
FAU - Soligo, Christophe
AU  - Soligo C
FAU - Martin, Robert D
AU  - Martin RD
FAU - Yang, Ziheng
AU  - Yang Z
FAU - Tavare, Simon
AU  - Tavare S
LA  - eng
GR  - RR03037/RR/NCRR NIH HHS/United States
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
DEP - 20101104
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - Evolution, Molecular
MH  - *Fossils
MH  - Humans
MH  - Markov Chains
MH  - Monte Carlo Method
MH  - Paleontology/methods
MH  - Phylogeny
MH  - Primates/*classification/*genetics
PMC - PMC2997628
EDAT- 2010/11/06 06:00
MHDA- 2011/02/18 06:00
CRDT- 2010/11/06 06:00
PHST- 2010/11/06 06:00 [entrez]
PHST- 2010/11/06 06:00 [pubmed]
PHST- 2011/02/18 06:00 [medline]
AID - syq054 [pii]
AID - 10.1093/sysbio/syq054 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jan;60(1):16-31. doi: 10.1093/sysbio/syq054. Epub 2010 Nov 4.

PMID- 20952757
OWN - NLM
STAT- MEDLINE
DCOM- 20101214
LR  - 20101028
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 6
DP  - 2010 Dec
TI  - The shape and temporal dynamics of phylogenetic trees arising from geographic
      speciation.
PG  - 660-73
LID - 10.1093/sysbio/syq058 [doi]
AB  - Phylogenetic trees often depart from the expectations of stochastic models,
      exhibiting imbalance in diversification among lineages and slowdowns in the rate 
      of lineage accumulation through time. Such departures have led to a widespread
      perception that ecological differences among species or adaptation and subsequent
      niche filling are required to explain patterns of diversification. However, a key
      element missing from models of diversification is the geographical context of
      speciation and extinction. In this study, we develop a spatially explicit model
      of geographic range evolution and cladogenesis, where speciation arises via
      vicariance or peripatry, and explore the effects of these processes on patterns
      of diversification. We compare the results with those observed in 41
      reconstructed avian trees. Our model shows that nonconstant rates of speciation
      and extinction are emergent properties of the apportioning of geographic ranges
      that accompanies speciation. The dynamics of diversification exhibit wide
      variation, depending on the mode of speciation, tendency for range expansion, and
      rate of range evolution. By varying these parameters, the model is able to
      capture many, but not all, of the features exhibited by birth-death trees and
      extant bird clades. Under scenarios with relatively stable geographic ranges,
      strong slowdowns in diversification rates are produced, with faster rates of
      range dynamics leading to constant or accelerating rates of apparent
      diversification. A peripatric model of speciation with stable ranges also
      generates highly unbalanced trees typical of bird phylogenies but fails to
      produce realistic range size distributions among the extant species. Results most
      similar to those of a birth-death process are reached under a peripatric
      speciation scenario with highly volatile range dynamics. Taken together, our
      results demonstrate that considering the geographical context of speciation and
      extinction provides a more conservative null model of diversification and offers 
      a very different perspective on the phylogenetic patterns expected in the absence
      of ecology.
FAU - Pigot, Alex L
AU  - Pigot AL
AD  - Division of Biology, Department of Life Sciences, Imperial College London, Ascot,
      Berkshire, UK. alexander.pigot@imperial.ac.uk
FAU - Phillimore, Albert B
AU  - Phillimore AB
FAU - Owens, Ian P F
AU  - Owens IP
FAU - Orme, C David L
AU  - Orme CD
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20101015
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Extinction, Biological
MH  - *Genetic Speciation
MH  - *Geography
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2010/10/19 06:00
MHDA- 2010/12/16 06:00
CRDT- 2010/10/19 06:00
PHST- 2010/10/19 06:00 [entrez]
PHST- 2010/10/19 06:00 [pubmed]
PHST- 2010/12/16 06:00 [medline]
AID - syq058 [pii]
AID - 10.1093/sysbio/syq058 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Dec;59(6):660-73. doi: 10.1093/sysbio/syq058. Epub 2010 Oct 15.

PMID- 20952756
OWN - NLM
STAT- MEDLINE
DCOM- 20110217
LR  - 20101207
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 60
IP  - 1
DP  - 2011 Jan
TI  - Accommodating heterogenous rates of evolution in molecular divergence dating
      methods: an example using intercontinental dispersal of Plestiodon (Eumeces)
      lizards.
PG  - 3-15
LID - 10.1093/sysbio/syq045 [doi]
AB  - Identifying and dating historical biological events is a fundamental goal of
      evolutionary biology, and recent analytical advances permit the modeling of
      factors known to affect both the accuracy and the precision of molecular date
      estimates. As the use of multilocus data sets becomes increasingly routine, it
      becomes more important to evaluate the potentially confounding effects of rate
      heterogeneity both within (e.g., codon positions) and among loci when estimating 
      divergence times. Here, using Plestiodon lizards as a test case, we examine the
      effects of accommodating rate heterogeneity among data partitions on divergence
      time estimation. Plestiodon inhabits both East Asia and North America, yet both
      the geographic origin of the genus and timing of dispersal between the continents
      have been debated. For each of the eight independently evolving loci and a
      combined data set, we conduct single model and partitioned analyses. We found
      that extreme saturation has obscured the underlying rate of evolution in the
      mitochondrial DNA (mtDNA), resulting in severe underestimation of the rate in
      this locus. As a result, the age of the crown Plestiodon clade was overestimated 
      by 15-17 Myr by the unpartitioned analysis of the combined loci data. However,
      the application of partition-specific models to the combined data resulted in
      ages that were fully congruent with those inferred by the individual nuclear
      loci. Although partitioning improved divergence date estimates of the mtDNA-only 
      analysis, the ages were nonetheless overestimated, thus indicating an inadequacy 
      of our current models to capture the complex nature of mtDNA evolution in over
      large time scales. Finally, the statistically incongruent age distributions
      inferred by the partitioned and unpartitioned analyses of the combined data
      support mutually exclusive hypotheses of the timing of intercontinental dispersal
      of Plestiodon from Asia to North America. Analyses that best capture the rate of 
      evolution in the combined data set infer that this exchange occurred via Beringia
      approximately 18.0-30 Ma.
FAU - Brandley, Matthew C
AU  - Brandley MC
AD  - Museum of Vertebrate Zoology, Department of Integrative Biology, University of
      California, Berkeley, CA 94720-3160, USA. mbrandley@gmail.com
FAU - Wang, Yuezhao
AU  - Wang Y
FAU - Guo, Xianguang
AU  - Guo X
FAU - de Oca, Adrian Nieto Montes
AU  - de Oca AN
FAU - Feria-Ortiz, Manuel
AU  - Feria-Ortiz M
FAU - Hikida, Tsutomu
AU  - Hikida T
FAU - Ota, Hidetoshi
AU  - Ota H
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20101015
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Reptilian Proteins)
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Cell Nucleus/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Evolution, Molecular
MH  - Far East
MH  - Lizards/*classification/*genetics
MH  - Models, Genetic
MH  - North America
MH  - Phylogeny
MH  - Reptilian Proteins/genetics
EDAT- 2010/10/19 06:00
MHDA- 2011/02/18 06:00
CRDT- 2010/10/19 06:00
PHST- 2010/10/19 06:00 [entrez]
PHST- 2010/10/19 06:00 [pubmed]
PHST- 2011/02/18 06:00 [medline]
AID - syq045 [pii]
AID - 10.1093/sysbio/syq045 [doi]
PST - ppublish
SO  - Syst Biol. 2011 Jan;60(1):3-15. doi: 10.1093/sysbio/syq045. Epub 2010 Oct 15.

PMID- 20947700
OWN - NLM
STAT- MEDLINE
DCOM- 20101214
LR  - 20101028
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 6
DP  - 2010 Dec
TI  - Evolutionary bangs and whimpers: methodological advances and conceptual
      frameworks for studying exceptional diversification.
PG  - 615-8
LID - 10.1093/sysbio/syq061 [doi]
FAU - Rabosky, Daniel L
AU  - Rabosky DL
AD  - Department of Integrative Biology, University of California, Berkeley, CA 94720, 
      USA. drabosky@berkeley.edu
FAU - Alfaro, Michael E
AU  - Alfaro ME
LA  - eng
PT  - Introductory Journal Article
DEP - 20101014
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Biology/*methods
MH  - *Genetic Speciation
MH  - Phylogeny
EDAT- 2010/10/16 06:00
MHDA- 2010/12/16 06:00
CRDT- 2010/10/16 06:00
PHST- 2010/10/16 06:00 [entrez]
PHST- 2010/10/16 06:00 [pubmed]
PHST- 2010/12/16 06:00 [medline]
AID - syq061 [pii]
AID - 10.1093/sysbio/syq061 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Dec;59(6):615-8. doi: 10.1093/sysbio/syq061. Epub 2010 Oct 14.

PMID- 20937759
OWN - NLM
STAT- MEDLINE
DCOM- 20101214
LR  - 20101028
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 6
DP  - 2010 Dec
TI  - Assessment of the relative merits of a few methods to detect evolutionary trends.
PG  - 689-704
LID - 10.1093/sysbio/syq059 [doi]
AB  - Some of the most basic questions about the history of life concern evolutionary
      trends. These include determining whether or not metazoans have become more
      complex over time, whether or not body size tends to increase over time (the
      Cope-Deperet rule), or whether or not brain size has increased over time in
      various taxa, such as mammals and birds. Despite the proliferation of studies on 
      such topics, assessment of the reliability of results in this field is hampered
      by the variability of techniques used and the lack of statistical validation of
      these methods. To solve this problem, simulations are performed using a variety
      of evolutionary models (gradual Brownian motion, speciational Brownian motion,
      and Ornstein-Uhlenbeck), with or without a drift of variable amplitude, with
      variable variance of tips, and with bounds placed close or far from the starting 
      values and final means of simulated characters. These are used to assess the
      relative merits (power, Type I error rate, bias, and mean absolute value of error
      on slope estimate) of several statistical methods that have recently been used to
      assess the presence of evolutionary trends in comparative data. Results show
      widely divergent performance of the methods. The simple, nonphylogenetic
      regression (SR) and variance partitioning using phylogenetic eigenvector
      regression (PVR) with a broken stick selection procedure have greatly inflated
      Type I error rate (0.123-0.180 at a 0.05 threshold), which invalidates their use 
      in this context. However, they have the greatest power. Most variants of
      Felsenstein's independent contrasts (FIC; five of which are presented) have
      adequate Type I error rate, although two have a slightly inflated Type I error
      rate with at least one of the two reference trees (0.064-0.090 error rate at a
      0.05 threshold). The power of all contrast-based methods is always much lower
      than that of SR and PVR, except under Brownian motion with a strong trend and
      distant bounds. Mean absolute value of error on slope of all FIC methods is
      slightly higher than that of phylogenetic generalized least squares (PGLS), SR,
      and PVR. PGLS performs well, with low Type I error rate, low error on regression 
      coefficient, and power comparable with some FIC methods. Four variants of
      skewness analysis are examined, and a new method to assess significance of
      results is presented. However, all have consistently low power, except in rare
      combinations of trees, trend strength, and distance between final means and
      bounds. Globally, the results clearly show that FIC-based methods and PGLS are
      globally better than nonphylogenetic methods and variance partitioning with PVR. 
      FIC methods and PGLS are sensitive to the model of evolution (and, hence, to
      branch length errors). Our results suggest that regressing raw character
      contrasts against raw geological age contrasts yields a good combination of power
      and Type I error rate. New software to facilitate batch analysis is presented.
FAU - Laurin, Michel
AU  - Laurin M
AD  - UMR 7207, CNRS/MNHN/UPMC, Centre de Recherches sur la Paleobiodiversite et les
      Paleoenvironnements, Departement Histoire de la Terre, Museum National d'Histoire
      Naturelle, Batiment de Geologie, Paris, France. michel.laurin@upmc.fr
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20101011
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Computer Simulation
MH  - Humans
MH  - *Models, Genetic
MH  - Phylogeny
EDAT- 2010/10/13 06:00
MHDA- 2010/12/16 06:00
CRDT- 2010/10/13 06:00
PHST- 2010/10/13 06:00 [entrez]
PHST- 2010/10/13 06:00 [pubmed]
PHST- 2010/12/16 06:00 [medline]
AID - syq059 [pii]
AID - 10.1093/sysbio/syq059 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Dec;59(6):689-704. doi: 10.1093/sysbio/syq059. Epub 2010 Oct 11.

PMID- 20930035
OWN - NLM
STAT- MEDLINE
DCOM- 20101214
LR  - 20101028
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 6
DP  - 2010 Dec
TI  - Combining phylogenomics and fossils in higher-level squamate reptile phylogeny:
      molecular data change the placement of fossil taxa.
PG  - 674-88
LID - 10.1093/sysbio/syq048 [doi]
AB  - Molecular data offer great potential to resolve the phylogeny of living taxa but 
      can molecular data improve our understanding of relationships of fossil taxa?
      Simulations suggest that this is possible, but few empirical examples have
      demonstrated the ability of molecular data to change the placement of fossil
      taxa. We offer such an example here. We analyze the placement of snakes among
      squamate reptiles, combining published morphological data (363 characters) and
      new DNA sequence data (15,794 characters, 22 nuclear loci) for 45 living and 19
      fossil taxa. We find several intriguing results. First, some fossil taxa undergo 
      major changes in their phylogenetic position when molecular data are added.
      Second, most fossil taxa are placed with strong support in the expected clades by
      the combined data Bayesian analyses, despite each having &gt;98% missing cells and
      despite recent suggestions that extensive missing data are problematic for
      Bayesian phylogenetics. Third, morphological data can change the placement of
      living taxa in combined analyses, even when there is an overwhelming majority of 
      molecular characters. Finally, we find strong but apparently misleading signal in
      the morphological data, seemingly associated with a burrowing lifestyle in
      snakes, amphisbaenians, and dibamids. Overall, our results suggest promise for an
      integrated and comprehensive Tree of Life by combining molecular and
      morphological data for living and fossil taxa.
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Department of Ecology and Evolution, Stony Brook University, Stony Brook, NY
      11794-5245, USA. wiensj@life.bio.sunysb.edu
FAU - Kuczynski, Caitlin A
AU  - Kuczynski CA
FAU - Townsend, Ted
AU  - Townsend T
FAU - Reeder, Tod W
AU  - Reeder TW
FAU - Mulcahy, Daniel G
AU  - Mulcahy DG
FAU - Sites, Jack W Jr
AU  - Sites JW Jr
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20101007
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Fossils
MH  - Genomics
MH  - Phylogeny
MH  - Reptiles/*classification/*genetics
MH  - Snakes/*classification/*genetics
EDAT- 2010/10/12 06:00
MHDA- 2010/12/16 06:00
CRDT- 2010/10/09 06:00
PHST- 2010/10/09 06:00 [entrez]
PHST- 2010/10/12 06:00 [pubmed]
PHST- 2010/12/16 06:00 [medline]
AID - syq048 [pii]
AID - 10.1093/sysbio/syq048 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Dec;59(6):674-88. doi: 10.1093/sysbio/syq048. Epub 2010 Oct 7.

PMID- 20926600
OWN - NLM
STAT- MEDLINE
DCOM- 20101104
LR  - 20101118
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 5
DP  - 2010 Oct
TI  - Time, species, and separating their effects on trait variance in clades.
PG  - 602-7
LID - 10.1093/sysbio/syq029 [doi]
FAU - Bokma, Folmer
AU  - Bokma F
AD  - Department of Ecology and Environmental Science, Umea University, Umea S-90187,
      Sweden. folmer.bokma@emg.umu.se
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100531
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - *Genetic Variation
MH  - Likelihood Functions
MH  - Neural Networks (Computer)
MH  - Phenotype
MH  - Phylogeny
EDAT- 2010/10/12 06:00
MHDA- 2010/11/05 06:00
CRDT- 2010/10/08 06:00
PHST- 2010/10/08 06:00 [entrez]
PHST- 2010/10/12 06:00 [pubmed]
PHST- 2010/11/05 06:00 [medline]
AID - syq029 [pii]
AID - 10.1093/sysbio/syq029 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Oct;59(5):602-7. doi: 10.1093/sysbio/syq029. Epub 2010 May 31.

PMID- 20926597
OWN - NLM
STAT- MEDLINE
DCOM- 20101214
LR  - 20110606
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 6
DP  - 2010 Dec
TI  - Primary controls on species richness in higher taxa.
PG  - 634-45
LID - 10.1093/sysbio/syq060 [doi]
AB  - The disparity in species richness across the tree of life is one of the most
      striking and pervasive features of biological diversity. Some groups are
      exceptionally diverse, whereas many other groups are species poor. Differences in
      diversity among groups are frequently assumed to result from primary control by
      differential rates of net diversification. However, a major alternative
      explanation is that ecological and other factors exert primary control on clade
      diversity, such that apparent variation in net diversification rates is a
      secondary consequence of ecological limits on clade growth. Here, I consider a
      likelihood framework for distinguishing between these competing hypotheses. I
      incorporate hierarchical modeling to explicitly relax assumptions about the
      constancy of diversification rates across clades, and I propose several
      statistics for a posteriori evaluation of model adequacy. I apply the framework
      to a recent dated phylogeny of ants. My results reject the hypothesis that net
      diversification rates exert primary control on species richness in this group and
      demonstrate that clade diversity is better explained by total time-integrated
      speciation. These results further suggest that it may not possible to estimate
      meaningful speciation and extinction rates from higher-level phylogenies of
      extant taxa only.
FAU - Rabosky, Daniel L
AU  - Rabosky DL
AD  - Department of Integrative Biology, University of California, Berkeley, CA 94720, 
      USA. drabosky@berkeley.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20101006
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2011 May;60(3):391
MH  - Animals
MH  - *Biodiversity
MH  - *Genetic Speciation
MH  - Likelihood Functions
MH  - Models, Genetic
EDAT- 2010/10/12 06:00
MHDA- 2010/12/16 06:00
CRDT- 2010/10/08 06:00
PHST- 2010/10/08 06:00 [entrez]
PHST- 2010/10/12 06:00 [pubmed]
PHST- 2010/12/16 06:00 [medline]
AID - syq060 [pii]
AID - 10.1093/sysbio/syq060 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Dec;59(6):634-45. doi: 10.1093/sysbio/syq060. Epub 2010 Oct 6.

PMID- 20889732
OWN - NLM
STAT- MEDLINE
DCOM- 20101214
LR  - 20101028
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 6
DP  - 2010 Dec
TI  - Rate heterogeneity, ancestral character state reconstruction, and the evolution
      of limb morphology in Lerista (Scincidae, Squamata).
PG  - 723-40
LID - 10.1093/sysbio/syq055 [doi]
AB  - Rates of phenotypic evolution derive from numerous interrelated processes acting 
      at varying spatial and temporal scales and frequently differ substantially among 
      lineages. Although current models employed in reconstructing ancestral character 
      states permit independent rates for distinct types of transition (forward and
      reverse transitions and transitions between different states), these rates are
      typically assumed to be identical for all branches in a phylogeny. In this paper,
      I present a general model of character evolution enabling rate heterogeneity
      among branches. This model is employed in assessing the extent to which the
      assumption of uniform transition rates affects reconstructions of ancestral limb 
      morphology in the scincid lizard clade Lerista and, accordingly, the potential
      for rate variability to mislead inferences of evolutionary patterns. Permitting
      rate variation among branches significantly improves model fit for both the manus
      and the pes. A constrained model in which the rate of digit acquisition is
      assumed to be effectively zero is strongly supported in each case; when compared 
      with a model assuming unconstrained transition rates, this model provides a
      substantially better fit for the manus and a nearly identical fit for the pes.
      Ancestral states reconstructed assuming the constrained model imply patterns of
      limb evolution differing significantly from those implied by reconstructions for 
      uniform-rate models, particularly for the pes; whereas ancestral states for the
      uniform-rate models consistently entail the reacquisition of pedal digits, those 
      for the model incorporating among-lineage rate heterogeneity imply repeated,
      unreversed digit loss. These results indicate that the assumption of identical
      transition rates for all branches in a phylogeny may be inappropriate in modeling
      the evolution of phenotypic traits and emphasize the need for careful evaluation 
      of phylogenetic tests of Dollo's law.
FAU - Skinner, Adam
AU  - Skinner A
AD  - School of Earth and Environmental Sciences, The University of Adelaide, Adelaide,
      South Australia, Australia. Adam.Skinner@adelaide.edu.au
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20101001
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Extremities/anatomy &amp; histology
MH  - Lizards/*anatomy &amp; histology/classification/*genetics
MH  - Phylogeny
MH  - Reptiles
EDAT- 2010/10/05 06:00
MHDA- 2010/12/16 06:00
CRDT- 2010/10/05 06:00
PHST- 2010/10/05 06:00 [entrez]
PHST- 2010/10/05 06:00 [pubmed]
PHST- 2010/12/16 06:00 [medline]
AID - syq055 [pii]
AID - 10.1093/sysbio/syq055 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Dec;59(6):723-40. doi: 10.1093/sysbio/syq055. Epub 2010 Oct 1.

PMID- 20884813
OWN - NLM
STAT- MEDLINE
DCOM- 20101214
LR  - 20101028
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 6
DP  - 2010 Dec
TI  - Quantitative traits and diversification.
PG  - 619-33
LID - 10.1093/sysbio/syq053 [doi]
AB  - Quantitative traits have long been hypothesized to affect speciation and
      extinction rates. For example, smaller body size or increased specialization may 
      be associated with increased rates of diversification. Here, I present a
      phylogenetic likelihood-based method (quantitative state speciation and
      extinction [QuaSSE]) that can be used to test such hypotheses using extant
      character distributions. This approach assumes that diversification follows a
      birth-death process where speciation and extinction rates may vary with one or
      more traits that evolve under a diffusion model. Speciation and extinction rates 
      may be arbitrary functions of the character state, allowing much flexibility in
      testing models of trait-dependent diversification. I test the approach using
      simulated phylogenies and show that a known relationship between speciation and a
      quantitative character could be recovered in up to 80% of the cases on large
      trees (500 species). Consistent with other approaches, detecting shifts in
      diversification due to differences in extinction rates was harder than when due
      to differences in speciation rates. Finally, I demonstrate the application of
      QuaSSE to investigate the correlation between body size and diversification in
      primates, concluding that clade-specific differences in diversification may be
      more important than size-dependent diversification in shaping the patterns of
      diversity within this group.
FAU - FitzJohn, Richard G
AU  - FitzJohn RG
AD  - Department of Zoology, University of British Columbia, Vancouver, BC, Canada.
      mail: fitzjohn@zoology.ubc.ca
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100930
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Body Size
MH  - Extinction, Biological
MH  - Female
MH  - *Genetic Speciation
MH  - *Models, Genetic
MH  - Phylogeny
MH  - Primates/genetics/physiology
MH  - *Quantitative Trait, Heritable
EDAT- 2010/10/05 06:00
MHDA- 2010/12/16 06:00
CRDT- 2010/10/02 06:00
PHST- 2010/10/02 06:00 [entrez]
PHST- 2010/10/05 06:00 [pubmed]
PHST- 2010/12/16 06:00 [medline]
AID - syq053 [pii]
AID - 10.1093/sysbio/syq053 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Dec;59(6):619-33. doi: 10.1093/sysbio/syq053. Epub 2010 Sep 30.

PMID- 20861283
OWN - NLM
STAT- MEDLINE
DCOM- 20101214
LR  - 20101028
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 6
DP  - 2010 Dec
TI  - When can decreasing diversification rates be detected with molecular phylogenies 
      and the fossil record?
PG  - 646-59
LID - 10.1093/sysbio/syq052 [doi]
AB  - Traditionally, patterns and processes of diversification could only be inferred
      from the fossil record. However, there are an increasing number of tools that
      enable diversification dynamics to be inferred from molecular phylogenies. The
      application of these tools to new data sets has renewed interest in the question 
      of the prevalence of diversity-dependent diversification. However, there is
      growing recognition that the absence of extinct species in molecular phylogenies 
      may prevent accurate inferences about the underlying diversification dynamics. On
      the other hand, even though the fossil record provides direct data on extinct
      species, its incompleteness can also mask true diversification processes. Here,
      using computer-generated diversity-dependent phylogenies, we mimicked molecular
      phylogenies by eliminating extinct lineages. We also simulated the fossil record 
      by converting the temporal axis into discrete intervals and imposing a variety of
      preservation processes on the lineages. Given the lack of reliable phylogenies
      for many fossil marine taxa, we also stripped away phylogenetic information from 
      the computer-generated phylogenies. For the simulated molecular phylogenies, we
      examined the efficacy of the standard metric (the gamma statistic) for
      identifying decreasing rates of diversification. We find that the underlying
      decreasing rate of diversification is detected only when the rate of change in
      the diversification rate is high, and if the molecular phylogeny happens to
      capture the diversification process as the equilibrium diversity is first reached
      or shortly thereafter. In contrast, estimating rates of diversification from the 
      simulated fossil record captures the expected zero rate of diversification after 
      equilibrium is reached under a wide range of preservation scenarios. The ability 
      to detect the initial decreasing rate of diversification is lost as the temporal 
      resolution of the fossil record drops and with a decreased quality of
      preservation. When the rate of change of the diversification rate is low, the
      gamma statistic will typically fail to detect the decreasing rate of
      diversification, as will the fossil record, although the fossil record still
      retains the signature of the diversity dependence in yielding approximately zero 
      diversification rates. Thus, although a significantly negative gamma value for a 
      molecular phylogeny indicates a decreasing rate of diversification, a
      nonsignificantly negative or positive gamma value might mean exponential
      diversification, or a slowly decreasing rate of diversification, or simply
      species turnover at a constant diversity. The fossil record can be of assistance 
      in helping choose among these possibilities.
FAU - Liow, Lee Hsiang
AU  - Liow LH
AD  - Centre for Ecological and Evolutionary Synthesis, Department of Biology,
      University of Oslo, Oslo, Norway. l.h.liow@bio.uio.no
FAU - Quental, Tiago B
AU  - Quental TB
FAU - Marshall, Charles R
AU  - Marshall CR
LA  - eng
PT  - Journal Article
DEP - 20100922
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biodiversity
MH  - Computer Simulation
MH  - *Extinction, Biological
MH  - *Fossils
MH  - *Phylogeny
EDAT- 2010/09/24 06:00
MHDA- 2010/12/16 06:00
CRDT- 2010/09/24 06:00
PHST- 2010/09/24 06:00 [entrez]
PHST- 2010/09/24 06:00 [pubmed]
PHST- 2010/12/16 06:00 [medline]
AID - syq052 [pii]
AID - 10.1093/sysbio/syq052 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Dec;59(6):646-59. doi: 10.1093/sysbio/syq052. Epub 2010 Sep 22.

PMID- 20841320
OWN - NLM
STAT- MEDLINE
DCOM- 20101214
LR  - 20101028
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 6
DP  - 2010 Dec
TI  - Tracing the temporal and spatial origins of island endemics in the Mediterranean 
      region: a case study from the citrus family (Ruta L., Rutaceae).
PG  - 705-22
LID - 10.1093/sysbio/syq046 [doi]
AB  - Understanding the origin of island endemics is a central task of historical
      biogeography. Recent methodological advances provide a rigorous framework to
      determine the relative contribution of different biogeographic processes (e.g.,
      vicariance, land migration, long-distance dispersal) to the origin of island
      endemics. With its complex but well-known history of microplate movements and
      climatic oscillations, the Mediterranean region (including the Mediterranean
      basin and Macaronesia) provides the geographic backdrop for the diversification
      of Ruta L., the type genus of Rutaceae (citrus family). Phylogenetic, molecular
      dating, and ancestral range reconstruction analyses were carried out to
      investigate the extent to which past geological connections and climatic history 
      of the Mediterranean region explain the current distribution of species in Ruta, 
      with emphasis on its island endemics. The analyses showed that Ruta invaded the
      region from the north well before the onset of the Mediterranean climate and
      diversified in situ as the climate became Mediterranean. The continental fragment
      island endemics of the genus originated via processes of land
      migration/vicariance driven by connections/disconnections between microplates,
      whereas the oceanic island endemics were the product of a single colonization
      event from the mainland followed by in situ diversification. This study
      emphasizes the need for an integrative, hypothesis-based approach to historical
      biogeography and stresses the importance of temporary land connections and
      colonization opportunity in the biotic assembly of continental fragment and
      oceanic islands, respectively.
FAU - Salvo, Gabriele
AU  - Salvo G
AD  - Institute of Systematic Botany, University of Zurich, Zurich, Switzerland.
      salvo@systbot.uzh.ch
FAU - Ho, Simon Y W
AU  - Ho SY
FAU - Rosenbaum, Gideon
AU  - Rosenbaum G
FAU - Ree, Richard
AU  - Ree R
FAU - Conti, Elena
AU  - Conti E
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100913
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Climate
MH  - Geography
MH  - Mediterranean Region
MH  - Phylogeny
MH  - Rutaceae/classification/*genetics
EDAT- 2010/09/16 06:00
MHDA- 2010/12/16 06:00
CRDT- 2010/09/16 06:00
PHST- 2010/09/16 06:00 [entrez]
PHST- 2010/09/16 06:00 [pubmed]
PHST- 2010/12/16 06:00 [medline]
AID - syq046 [pii]
AID - 10.1093/sysbio/syq046 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Dec;59(6):705-22. doi: 10.1093/sysbio/syq046. Epub 2010 Sep 13.

PMID- 20834011
OWN - NLM
STAT- MEDLINE
DCOM- 20101104
LR  - 20101007
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 5
DP  - 2010 Oct
TI  - Genomic scans support repetitive continental colonization events during the rapid
      radiation of voles (Rodentia: Microtus): the utility of AFLPs versus
      mitochondrial and nuclear sequence markers.
PG  - 548-72
LID - 10.1093/sysbio/syq042 [doi]
AB  - Single locus studies might not resolve phylogenetic relationships and the
      evolutionary history of taxa. The analysis of multiple markers promises higher
      resolution, and congruence among loci may indicate that the phylogenies represent
      the underlying species history. Here, we examine the utility of a genome-wide
      approach based on amplified fragment length polymorphisms (AFLP) and several DNA 
      sequence markers in resolving phylogenetic signals in the rapidly radiating
      rodent genus Microtus which produced about 70 vole species within the last 1.2-2 
      myr. The current Holarctic distribution of Microtus is assumed to have resulted
      from three independent colonization events out of Asia to North America, Europe, 
      and northern Asia without subsequent colonization, which would have led to deep
      splits between species from different continents. We investigated this hypothesis
      of three single colonization events by reconstructing the phylogenetic
      relationships among species from all three continents based on data from the
      first exon of the nuclear arginine vasopressin receptor 1a gene (EXON1), an
      adjacent noncoding region and the mitochondrial cytochrome b gene. The
      phylogenetic patterns obtained from these sequence markers are contrasted to
      genome-wide data on more than 1800 amplified fragment length polymorphisms (AFLP)
      analyzed for the same samples. Our results show that the single sequence markers 
      partially resolve the phylogenetic relationships within Microtus, but with some
      incongruence mostly between EXON1 and the other loci. However, deeper nodes of
      the radiation are only weakly supported and neither the combination of the
      markers nor additional nuclear sequences improved the resolution significantly.
      AFLPs provided much stronger support for major continent-specific clades, and
      show also that reciprocal monophyly of American and European voles is incomplete.
      Our results demonstrate that Microtus voles colonized the American and European
      continents each repeatedly in several independent events on similar colonization 
      routes during their radiation. More generally, this study supports the
      suitability of AFLPs as an alternative to sequence markers to resolve the
      evolutionary history of rapidly radiating taxa.
FAU - Fink, Sabine
AU  - Fink S
AD  - Department of Biology, Computational and Molecular Population Genetics (CMPG),
      Institute of Ecology and Evolution, University of Bern, Baltzerstrasse 6, CH-3012
      Bern, Switzerland.
FAU - Fischer, Martin C
AU  - Fischer MC
FAU - Excoffier, Laurent
AU  - Excoffier L
FAU - Heckel, Gerald
AU  - Heckel G
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100910
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Amplified Fragment Length Polymorphism Analysis
MH  - *Animal Migration
MH  - Animals
MH  - Arvicolinae/*classification/*genetics
MH  - Cell Nucleus/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Genome
MH  - Phylogeny
MH  - Rodentia/genetics
EDAT- 2010/09/14 06:00
MHDA- 2010/11/05 06:00
CRDT- 2010/09/14 06:00
PHST- 2010/09/14 06:00 [entrez]
PHST- 2010/09/14 06:00 [pubmed]
PHST- 2010/11/05 06:00 [medline]
AID - syq042 [pii]
AID - 10.1093/sysbio/syq042 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Oct;59(5):548-72. doi: 10.1093/sysbio/syq042. Epub 2010 Sep 10.

PMID- 20833951
OWN - NLM
STAT- MEDLINE
DCOM- 20101104
LR  - 20101007
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 5
DP  - 2010 Oct
TI  - Sources of error inherent in species-tree estimation: impact of mutational and
      coalescent effects on accuracy and implications for choosing among different
      methods.
PG  - 573-83
LID - 10.1093/sysbio/syq047 [doi]
AB  - Discord in the estimated gene trees among loci can be attributed to both the
      process of mutation and incomplete lineage sorting. Effectively modeling these
      two sources of variation--mutational and coalescent variance--provides two
      distinct challenges for phylogenetic studies. Despite extensive investigation on 
      mutational models for gene-tree estimation over the past two decades and recent
      attention to modeling of the coalescent process for phylogenetic estimation, the 
      effects of these two variances have yet to be evaluated simultaneously. Here, we 
      partition the effects of mutational and coalescent processes on phylogenetic
      accuracy by comparing the accuracy of species trees estimated from gene trees
      (i.e., the actual coalescent genealogies) with that of species trees estimated
      from estimated gene trees (i.e., trees estimated from nucleotide sequences, which
      contain both coalescent and mutational variance). Not only is there a significant
      contribution of both mutational and coalescent variance to errors in species-tree
      estimates, but the relative magnitude of the effects on the accuracy of
      species-tree estimation also differs systematically depending on 1) the timing of
      divergence, 2) the sampling design, and 3) the method used for species-tree
      estimation. These findings explain why using more information contained in gene
      trees (e.g., topology and branch lengths as opposed to just topology) does not
      necessarily translate into pronounced gains in accuracy, highlighting the
      strengths and limits of different methods for species-tree estimation.
      Differences in accuracy scores between methods for different sampling regimes
      also emphasize that it would be a mistake to assume more computationally
      intensive species-tree estimation procedures that will always provide better
      estimates of species trees. To the contrary, the performance of a method depends 
      not only on the method per se but also on the compatibilities between the input
      genetic data and the method as determined by the relative impact of mutational
      and coalescent variance.
FAU - Huang, Huateng
AU  - Huang H
AD  - Department of Ecology and Evolutionary Biology, Museum of Zoology, University of 
      Michigan, Ann Arbor, MI 48109-1079, USA.
FAU - He, Qixin
AU  - He Q
FAU - Kubatko, Laura S
AU  - Kubatko LS
FAU - Knowles, L Lacey
AU  - Knowles LL
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20100910
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Analysis of Variance
MH  - Genetic Speciation
MH  - *Models, Genetic
MH  - *Mutation
MH  - *Phylogeny
MH  - *Research Design
EDAT- 2010/09/14 06:00
MHDA- 2010/11/05 06:00
CRDT- 2010/09/14 06:00
PHST- 2010/09/14 06:00 [entrez]
PHST- 2010/09/14 06:00 [pubmed]
PHST- 2010/11/05 06:00 [medline]
AID - syq047 [pii]
AID - 10.1093/sysbio/syq047 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Oct;59(5):573-83. doi: 10.1093/sysbio/syq047. Epub 2010 Sep 10.

PMID- 20833950
OWN - NLM
STAT- MEDLINE
DCOM- 20101214
LR  - 20101028
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 6
DP  - 2010 Dec
TI  - Geophylogenies and the Map of Life.
PG  - 741-52
LID - 10.1093/sysbio/syq043 [doi]
FAU - Kidd, David M
AU  - Kidd DM
AD  - 1NESCent (National Evolutionary Synthesis Center), Durham, NC 27005, USA.
      d.kidd@imperial.ac.uk
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20100910
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Geography
MH  - *Models, Genetic
MH  - Phylogeny
MH  - Ursidae/genetics
EDAT- 2010/09/14 06:00
MHDA- 2010/12/16 06:00
CRDT- 2010/09/14 06:00
PHST- 2010/09/14 06:00 [entrez]
PHST- 2010/09/14 06:00 [pubmed]
PHST- 2010/12/16 06:00 [medline]
AID - syq043 [pii]
AID - 10.1093/sysbio/syq043 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Dec;59(6):741-52. doi: 10.1093/sysbio/syq043. Epub 2010 Sep 10.

PMID- 20817714
OWN - NLM
STAT- MEDLINE
DCOM- 20101104
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 5
DP  - 2010 Oct
TI  - An intuitive, informative, and most balanced representation of phylogenetic
      topologies.
PG  - 584-93
LID - 10.1093/sysbio/syq044 [doi]
AB  - The recent explosion in the availability of genetic sequence data has made
      large-scale phylogenetic inference routine in many life sciences laboratories.
      The outcomes of such analyses are, typically, a variety of candidate phylogenetic
      relationships or tree topologies, even when the power of genome-scale data is
      exploited. Because much phylogenetic information must be buried in such topology 
      distributions, it is important to reveal that information as effectively as
      possible; however, existing methods need to adopt complex structures to represent
      such information. Hence, researchers, in particular those not experts in
      evolutionary studies, sometimes hesitate to adopt these methods and much
      phylogenetic information could be overlooked and wasted. In this paper, we
      propose the centroid wheel tree representation, which is an informative
      representation of phylogenetic topology distributions, and which can be readily
      interpreted even by nonexperts. Furthermore, we mathematically prove this to be
      the most balanced representation of phylogenetic topologies and efficiently
      solvable in the framework of the traveling salesman problem, for which very
      sophisticated program packages are available. This theoretically and practically 
      superior representation should aid biologists faced with abundant data. The
      centroid representation introduced here is fairly general, so it can be applied
      to other fields that are characterized by high-dimensional solution spaces and
      large quantities of noisy data. The software is implemented in Java and available
      via http://cwt.cb.k.u-tokyo.ac.jp/.
FAU - Iwasaki, Wataru
AU  - Iwasaki W
AD  - Department of Computational Biology, Graduate School of Frontier Sciences, The
      University of Tokyo, Kashiwa, Chiba 277-8568, Japan. iwasaki@k.u-tokyo.ac.jp
FAU - Takagi, Toshihisa
AU  - Takagi T
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100903
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Humans
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - *Software
PMC - PMC2950835
EDAT- 2010/09/08 06:00
MHDA- 2010/11/05 06:00
CRDT- 2010/09/07 06:00
PHST- 2010/09/07 06:00 [entrez]
PHST- 2010/09/08 06:00 [pubmed]
PHST- 2010/11/05 06:00 [medline]
AID - syq044 [pii]
AID - 10.1093/sysbio/syq044 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Oct;59(5):584-93. doi: 10.1093/sysbio/syq044. Epub 2010 Sep 3.

PMID- 20705909
OWN - NLM
STAT- MEDLINE
DCOM- 20101104
LR  - 20101007
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 5
DP  - 2010 Oct
TI  - A comparison of global, gene-specific, and relaxed clock methods in a comparative
      genomics framework: dating the polyploid history of soybean (Glycine max).
PG  - 534-47
LID - 10.1093/sysbio/syq041 [doi]
AB  - It is widely recognized that many genes and lineages do not adhere to a molecular
      clock, yet molecular clocks are commonly used to date divergences in comparative 
      genomic studies. We test the application of a molecular clock across genes and
      lineages in a phylogenetic framework utilizing 12 genes linked in a 1-Mb region
      on chromosome 13 of soybean (Glycine max); homoeologous copies of these genes
      formed by polyploidy in Glycine; and orthologous copies in G. tomentella,
      Phaseolus vulgaris, and Medicago truncatula. We compare divergence dates
      estimated by two methods each in three frameworks: a global molecular clock with 
      a single rate across genes and lineages using full and approximate likelihood
      methods based on synonymous substitutions, a gene-specific clock assuming rate
      constancy over lineages but allowing a different rate for each gene, and a
      relaxed molecular clock where rates may vary across genes and lineages estimated 
      under penalized likelihood and Bayesian inference. We use the cumulative variance
      across genes as a means of quantifying precision. Our results suggest that
      divergence dating methods produce results that are correlated, but that older
      nodes are more variable and more difficult to estimate with precision and
      accuracy. We also find that models incorporating less rate heterogeneity estimate
      older dates of divergence than more complex models, as node age increases. A
      mixed model nested analysis of variance testing the effects of framework, method,
      and gene found that framework had a significant effect on the divergence date
      estimates but that most variation among dates is due to variation among genes,
      suggesting a need to further characterize and understand the evolutionary
      phenomena underlying rate variation within genomes, among genes, and across
      lineages.
FAU - Egan, Ashley N
AU  - Egan AN
AD  - Department of Plant Biology, L.H. Bailey Hortorium, Cornell University, 412 Mann 
      Library Building, Ithaca, NY 14853, USA. egana@ecu.edu
FAU - Doyle, Jeff
AU  - Doyle J
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20100812
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Analysis of Variance
MH  - *Evolution, Molecular
MH  - Fabaceae/genetics
MH  - Genome, Plant
MH  - Models, Genetic
MH  - Phylogeny
MH  - *Polyploidy
MH  - Soybeans/*genetics
EDAT- 2010/08/14 06:00
MHDA- 2010/11/05 06:00
CRDT- 2010/08/14 06:00
PHST- 2010/08/14 06:00 [entrez]
PHST- 2010/08/14 06:00 [pubmed]
PHST- 2010/11/05 06:00 [medline]
AID - syq041 [pii]
AID - 10.1093/sysbio/syq041 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Oct;59(5):534-47. doi: 10.1093/sysbio/syq041. Epub 2010 Aug 12.

PMID- 20693311
OWN - NLM
STAT- MEDLINE
DCOM- 20101104
LR  - 20101007
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 5
DP  - 2010 Oct
TI  - Species delimitation using dominant and codominant multilocus markers.
PG  - 491-503
LID - 10.1093/sysbio/syq039 [doi]
AB  - We propose a method for delimiting species based on dominant or codominant
      multilocus data using Gaussian clustering with a noise component for outliers.
      Case studies show that provisional species delimited using Gaussian clustering
      based on dominant multilocus data correspond well with provisional species
      delimited based on other data. However, the performance of Gaussian clustering in
      delimiting species based on few codominant markers was only moderate. Species
      represented by few individuals are usually included in the noise component
      because clusters are difficult to recognize with limited data. As alternative
      methods, we evaluated two model-based clustering methods originally proposed to
      infer population structure and assign individuals to populations based on the
      assumption of Hardy-Weinberg equilibrium within populations, namely STRUCTURE and
      STRUCTURAMA, as well as the "fields for recombination" approach. The latter
      resulted in lumping all individuals of each data set with codominant markers
      together, and whereas STRUCTURE often provides no decision about the number of
      clusters, STRUCTURAMA usually yields correct or almost correct numbers of
      clusters. The classification success of STRUCTURAMA analyses based on codominant 
      markers was very good, but its performance with dominant markers was less
      consistent. Based on the classification success of the different methods for
      delimiting species with dominant and codominant multilocus markers in the case
      studies, we recommend using Gaussian clustering for data sets with dominant
      markers and STRUCTURAMA for data sets with codominant markers.
FAU - Hausdorf, Bernhard
AU  - Hausdorf B
AD  - Zoological Museum, University of Hamburg, Martin-Luther-King-Platz 3, 20146
      Hamburg, Germany. hausdorf@zoologie.uni-hamburg.de
FAU - Hennig, Christian
AU  - Hennig C
LA  - eng
PT  - Journal Article
DEP - 20100806
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amplified Fragment Length Polymorphism Analysis
MH  - Animals
MH  - Cluster Analysis
MH  - *Genetic Speciation
MH  - Hymenoptera/classification/genetics
MH  - Microsatellite Repeats
MH  - *Models, Genetic
MH  - Orchidaceae/classification/genetics
MH  - *Software
MH  - Veronica/classification/genetics
EDAT- 2010/08/10 06:00
MHDA- 2010/11/05 06:00
CRDT- 2010/08/10 06:00
PHST- 2010/08/10 06:00 [entrez]
PHST- 2010/08/10 06:00 [pubmed]
PHST- 2010/11/05 06:00 [medline]
AID - syq039 [pii]
AID - 10.1093/sysbio/syq039 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Oct;59(5):491-503. doi: 10.1093/sysbio/syq039. Epub 2010 Aug 6.

PMID- 20656852
OWN - NLM
STAT- MEDLINE
DCOM- 20101104
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 5
DP  - 2010 Oct
TI  - Broadly sampled multigene analyses yield a well-resolved eukaryotic tree of life.
PG  - 518-33
LID - 10.1093/sysbio/syq037 [doi]
AB  - An accurate reconstruction of the eukaryotic tree of life is essential to
      identify the innovations underlying the diversity of microbial and macroscopic
      (e.g., plants and animals) eukaryotes. Previous work has divided eukaryotic
      diversity into a small number of high-level "supergroups," many of which receive 
      strong support in phylogenomic analyses. However, the abundance of data in
      phylogenomic analyses can lead to highly supported but incorrect relationships
      due to systematic phylogenetic error. Furthermore, the paucity of major
      eukaryotic lineages (19 or fewer) included in these genomic studies may
      exaggerate systematic error and reduce power to evaluate hypotheses. Here, we use
      a taxon-rich strategy to assess eukaryotic relationships. We show that analyses
      emphasizing broad taxonomic sampling (up to 451 taxa representing 72 major
      lineages) combined with a moderate number of genes yield a well-resolved
      eukaryotic tree of life. The consistency across analyses with varying numbers of 
      taxa (88-451) and levels of missing data (17-69%) supports the accuracy of the
      resulting topologies. The resulting stable topology emerges without the removal
      of rapidly evolving genes or taxa, a practice common to phylogenomic analyses.
      Several major groups are stable and strongly supported in these analyses (e.g.,
      SAR, Rhizaria, Excavata), whereas the proposed supergroup "Chromalveolata" is
      rejected. Furthermore, extensive instability among photosynthetic lineages
      suggests the presence of systematic biases including endosymbiotic gene transfer 
      from symbiont (nucleus or plastid) to host. Our analyses demonstrate that stable 
      topologies of ancient evolutionary relationships can be achieved with broad
      taxonomic sampling and a moderate number of genes. Finally, taxon-rich analyses
      such as presented here provide a method for testing the accuracy of relationships
      that receive high bootstrap support (BS) in phylogenomic analyses and enable
      placement of the multitude of lineages that lack genome scale data.
FAU - Parfrey, Laura Wegener
AU  - Parfrey LW
AD  - Program in Organismic and Evolutionary Biology, University of Massachusetts, 611 
      North Pleasant Street, Amherst, MA 01003, USA.
FAU - Grant, Jessica
AU  - Grant J
FAU - Tekle, Yonas I
AU  - Tekle YI
FAU - Lasek-Nesselquist, Erica
AU  - Lasek-Nesselquist E
FAU - Morrison, Hilary G
AU  - Morrison HG
FAU - Sogin, Mitchell L
AU  - Sogin ML
FAU - Patterson, David J
AU  - Patterson DJ
FAU - Katz, Laura A
AU  - Katz LA
LA  - eng
GR  - 5R01AI058054-05/AI/NIAID NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20100723
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Cell Nucleus/genetics
MH  - Eukaryota/*classification/*genetics
MH  - Phylogeny
MH  - Plastids/genetics
MH  - Rhizaria/classification/genetics
PMC - PMC2950834
EDAT- 2010/07/27 06:00
MHDA- 2010/11/05 06:00
CRDT- 2010/07/27 06:00
PHST- 2010/07/27 06:00 [entrez]
PHST- 2010/07/27 06:00 [pubmed]
PHST- 2010/11/05 06:00 [medline]
AID - syq037 [pii]
AID - 10.1093/sysbio/syq037 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Oct;59(5):518-33. doi: 10.1093/sysbio/syq037. Epub 2010 Jul 23.

PMID- 20603441
OWN - NLM
STAT- MEDLINE
DCOM- 20101104
LR  - 20101118
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 5
DP  - 2010 Oct
TI  - Gene trees versus species trees: reassessing life-history evolution in a
      freshwater fish radiation.
PG  - 504-17
LID - 10.1093/sysbio/syq031 [doi]
AB  - Mechanisms of speciation are best understood in the context of phylogenetic
      relationships and as such have often been inferred from single gene trees,
      typically those derived from mitochondrial DNA (mtDNA) markers. Recent studies,
      however, have noted the potential for phylogenetic discordance between gene trees
      and underlying species trees (e.g., due to stochastic lineage sorting,
      introgression, or selection). Here, we employ a variety of nuclear DNA loci to
      reassess evolutionary relationships within a recent freshwater fish radiation to 
      reappraise modes of speciation. New Zealand's freshwater-limited Galaxias
      vulgaris complex is thought to have evolved from G. brevipinnis, a widespread
      migratory species that retains a plesiomorphic marine juvenile phase. A
      well-resolved tree, based on four mtDNA regions, previously suggested that marine
      migratory ability has been lost on 3 independent occasions in the evolution of
      this species flock (assuming that loss of diadromy is irreversible). Here, we use
      pseudogene (galaxiid Numt: 1801 bp), intron (S: 903 bp), and exon (RAG-1: 1427
      bp) markers, together with mtDNA, to reevaluate this hypothesis of parallel
      evolution. Interestingly, partitioned Bayesian analysis of concatenated nuclear
      sequences (3141 bp) and concatenated nuclear and mtDNA (4770 bp) both recover
      phylogenies implying a single loss of diadromy, not three parallel losses as
      previously inferred from mtDNA alone. This phylogenetic result is reinforced by a
      multilocus analysis performed using Bayesian estimation of species trees (BEST)
      software that estimates the posterior distribution of species trees under a
      coalescent model. We discuss factors that might explain the apparently misleading
      phylogenetic inferences generated by mtDNA.
FAU - Waters, Jonathan M
AU  - Waters JM
AD  - Department of Zoology, University of Otago, PO Box 56, Dunedin 9054, New Zealand.
      jonathan.waters@otago.ac.nz
FAU - Rowe, Diane L
AU  - Rowe DL
FAU - Burridge, Christopher P
AU  - Burridge CP
FAU - Wallis, Graham P
AU  - Wallis GP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100705
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Cell Nucleus/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Osmeriformes/*classification/*genetics
MH  - Phylogeny
EDAT- 2010/07/07 06:00
MHDA- 2010/11/05 06:00
CRDT- 2010/07/07 06:00
PHST- 2010/07/07 06:00 [entrez]
PHST- 2010/07/07 06:00 [pubmed]
PHST- 2010/11/05 06:00 [medline]
AID - syq031 [pii]
AID - 10.1093/sysbio/syq031 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Oct;59(5):504-17. doi: 10.1093/sysbio/syq031. Epub 2010 Jul 5.

PMID- 20547783
OWN - NLM
STAT- MEDLINE
DCOM- 20101020
LR  - 20101118
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 4
DP  - 2010 Jul
TI  - The akaike information criterion will not choose the no common mechanism model.
PG  - 477-85
LID - 10.1093/sysbio/syq028 [doi]
FAU - Holder, Mark T
AU  - Holder MT
AD  - Department of Ecology and Evolutionary Biology, University of Kansas, 1200
      Sunnyside Avenue, Lawrence, KS 66045, USA. mtholder@ku.edu
FAU - Lewis, Paul O
AU  - Lewis PO
FAU - Swofford, David L
AU  - Swofford DL
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20100531
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - *Models, Genetic
MH  - *Models, Statistical
EDAT- 2010/06/16 06:00
MHDA- 2010/10/21 06:00
CRDT- 2010/06/16 06:00
PHST- 2010/06/16 06:00 [entrez]
PHST- 2010/06/16 06:00 [pubmed]
PHST- 2010/10/21 06:00 [medline]
AID - syq028 [pii]
AID - 10.1093/sysbio/syq028 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jul;59(4):477-85. doi: 10.1093/sysbio/syq028. Epub 2010 May 31.

PMID- 20547782
OWN - NLM
STAT- MEDLINE
DCOM- 20101020
LR  - 20101118
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 4
DP  - 2010 Jul
TI  - Sampling trees from evolutionary models.
PG  - 465-76
LID - 10.1093/sysbio/syq026 [doi]
AB  - A wide range of evolutionary models for species-level (and higher)
      diversification have been developed. These models can be used to test
      evolutionary hypotheses and provide comparisons with phylogenetic trees
      constructed from real data. To carry out these tests and comparisons, it is often
      necessary to sample, or simulate, trees from the evolutionary models. Sampling
      trees from these models is more complicated than it may appear at first glance,
      necessitating careful consideration and mathematical rigor. Seemingly
      straightforward sampling methods may produce trees that have systematically
      biased shapes or branch lengths. This is particularly problematic as there is no 
      simple method for determining whether the sampled trees are appropriate. In this 
      paper, we show why a commonly used simple sampling approach (SSA)-simulating
      trees forward in time until n species are first reached-should only be applied to
      the simplest pure birth model, the Yule model. We provide an alternative general 
      sampling approach (GSA) that can be applied to most other models. Furthermore, we
      introduce the constant-rate birth-death model sampling approach, which samples
      trees very efficiently from a widely used class of models. We explore the bias
      produced by SSA and identify situations in which this bias is particularly
      pronounced. We show that using SSA can lead to erroneous conclusions: When using 
      the inappropriate SSA, the variance of a gradually evolving trait does not
      correlate with the age of the tree; when the correct GSA is used, the trait
      variance correlates with tree age. The algorithms presented here are available in
      the Perl Bio::Phylo package, as a stand-alone program TreeSample, and in the R
      TreeSim package.
FAU - Hartmann, Klaas
AU  - Hartmann K
AD  - Tasmanian Aquaculture and Fisheries Institute, University of Tasmania, Hobart,
      Australia.
FAU - Wong, Dennis
AU  - Wong D
FAU - Stadler, Tanja
AU  - Stadler T
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100528
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Computer Simulation
MH  - Genetic Speciation
MH  - *Models, Genetic
MH  - Selection Bias
EDAT- 2010/06/16 06:00
MHDA- 2010/10/21 06:00
CRDT- 2010/06/16 06:00
PHST- 2010/06/16 06:00 [entrez]
PHST- 2010/06/16 06:00 [pubmed]
PHST- 2010/10/21 06:00 [medline]
AID - syq026 [pii]
AID - 10.1093/sysbio/syq026 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jul;59(4):465-76. doi: 10.1093/sysbio/syq026. Epub 2010 May 28.

PMID- 20547781
OWN - NLM
STAT- MEDLINE
DCOM- 20101020
LR  - 20100615
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 4
DP  - 2010 Jul
TI  - Slowdowns in diversification rates from real phylogenies may not be real.
PG  - 458-64
LID - 10.1093/sysbio/syq032 [doi]
AB  - Studies of diversification patterns often find a slowing in lineage accumulation 
      toward the present. This seemingly pervasive pattern of rate downturns has been
      taken as evidence for adaptive radiations, density-dependent regulation, and
      metacommunity species interactions. The significance of rate downturns is
      evaluated with statistical tests (the gamma statistic and Monte Carlo constant
      rates (MCCR) test; birth-death likelihood models and Akaike Information Criterion
      [AIC] scores) that rely on null distributions, which assume that the included
      species are a random sample of the entire clade. Sampling in real phylogenies,
      however, often is nonrandom because systematists try to include early-diverging
      species or representatives of previous intrataxon classifications. We studied the
      effects of biased sampling, structured sampling, and random sampling by
      experimentally pruning simulated trees (60 and 150 species) as well as a
      completely sampled empirical tree (58 species) and then applying the gamma
      statistic/MCCR test and birth-death likelihood models/AIC scores to assess rate
      changes. For trees with random species sampling, the true model (i.e., the one
      fitting the complete phylogenies) could be inferred in most cases. Oversampling
      deep nodes, however, strongly biases inferences toward downturns, with
      simulations of structured and biased sampling suggesting that this occurs when
      sampling percentages drop below 80%. The magnitude of the effect and the
      sensitivity of diversification rate models is such that a useful rule of thumb
      may be not to infer rate downturns from real trees unless they have &gt;80% species 
      sampling.
FAU - Cusimano, Natalie
AU  - Cusimano N
AD  - Systematic Botany and Mycology, Department of Biology, University of Munich,
      D-80638 Munich, Germany. cusimano@lrz.uni-muenchen.de
FAU - Renner, Susanne S
AU  - Renner SS
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100603
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Genetic Speciation
MH  - Models, Genetic
MH  - Momordica/genetics
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2010/06/16 06:00
MHDA- 2010/10/21 06:00
CRDT- 2010/06/16 06:00
PHST- 2010/06/16 06:00 [entrez]
PHST- 2010/06/16 06:00 [pubmed]
PHST- 2010/10/21 06:00 [medline]
AID - syq032 [pii]
AID - 10.1093/sysbio/syq032 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jul;59(4):458-64. doi: 10.1093/sysbio/syq032. Epub 2010 Jun 3.

PMID- 20547780
OWN - NLM
STAT- MEDLINE
DCOM- 20101020
LR  - 20100615
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 4
DP  - 2010 Jul
TI  - Optimal selection of gene and ingroup taxon sampling for resolving phylogenetic
      relationships.
PG  - 446-57
LID - 10.1093/sysbio/syq025 [doi]
AB  - A controversial topic that underlies much of phylogenetic experimental design is 
      the relative utility of increased taxonomic versus character sampling.
      Conclusions about the relative utility of adding characters or taxa to a current 
      phylogenetic study have subtly hinged upon the appropriateness of the rate of
      evolution of the characters added for resolution of the phylogeny in question.
      Clearly, the addition of characters evolving at optimal rates will have much
      greater impact upon accurate phylogenetic analysis than will the addition of
      characters with an inappropriate rate of evolution. Development of practical
      analytical predictions of the asymptotic impact of adding additional taxa would
      complement computational investigations of the relative utility of these two
      methods of expanding acquired data. Accordingly, we here formulate a measure of
      the phylogenetic informativeness of the additional sampling of character states
      from a new taxon added to the canonical phylogenetic quartet. We derive the
      optimal rate of evolution for characters assessed in taxa to be sampled and a
      metric of informativeness based on the rate of evolution of the characters
      assessed in the new taxon and the distance of the new taxon from the internode of
      interest. Calculation of the informativeness per base pair of additional
      character sampling for included taxa versus additional character sampling for
      novel taxa can be used to estimate cost-effectiveness and optimal efficiency of
      phylogenetic experimental design. The approach requires estimation of rates of
      evolution of individual sites based on an alignment of genes orthologous to those
      to be sequenced, which may be identified in a well-established clade of sister
      taxa or of related taxa diverging at a deeper phylogenetic scale. Some
      approximate idea of the potential phylogenetic relationships of taxa to be
      sequenced is also desirable, such as may be obtained from ribosomal RNA sequence 
      alone. Application to the solution of recalcitrant unresolved nodes in an
      otherwise well-known phylogeny is the most obvious application. We validate the
      theory by analysis of its predictions regarding the phylogenetic informativeness 
      for taxon addition of 46 amino acid alignments of 21 fungal taxa. Gene and taxon 
      sampling according to the theory herein and following a "deepest ingroup"
      heuristic are shown to provide significantly improved resolution of specified
      deep internodes.
FAU - Townsend, Jeffrey P
AU  - Townsend JP
AD  - Department of Ecology and Evolutionary Biology, Yale University,New Haven, CT
      06520, USA. Jeffrey.Townsend@Yale.edu
FAU - Lopez-Giraldez, Francesc
AU  - Lopez-Giraldez F
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100519
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Ascomycota/genetics
MH  - Genetic Speciation
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2010/06/16 06:00
MHDA- 2010/10/21 06:00
CRDT- 2010/06/16 06:00
PHST- 2010/06/16 06:00 [entrez]
PHST- 2010/06/16 06:00 [pubmed]
PHST- 2010/10/21 06:00 [medline]
AID - syq025 [pii]
AID - 10.1093/sysbio/syq025 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jul;59(4):446-57. doi: 10.1093/sysbio/syq025. Epub 2010 May 19.

PMID- 20547779
OWN - NLM
STAT- MEDLINE
DCOM- 20101020
LR  - 20101118
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 4
DP  - 2010 Jul
TI  - Identifying cliques of convergent characters: concerted evolution in the
      cormorants and shags.
PG  - 433-45
LID - 10.1093/sysbio/syq023 [doi]
AB  - A phylogenetic tree comprising clades with high bootstrap values or other strong 
      measures of statistical support is usually interpreted as providing a good
      estimate of the true phylogeny. Convergent evolution acting on groups of
      characters in concert, however, can lead to highly supported but erroneous
      phylogenies. Identifying such groups of phylogenetically misleading characters is
      obviously desirable. Here we present a procedure that uses an independent data
      source to identify sets of characters that have undergone concerted convergent
      evolution. We examine the problematic case of the cormorants and shags, for which
      trees constructed using osteological and molecular characters both have strong
      statistical support and yet are fundamentally incongruent. We find that the
      osteological characters can be separated into those that fit the phylogenetic
      history implied by the molecular data set and those that do not. Moreover, these 
      latter nonfitting osteological characters are internally consistent and form
      groups of mutually compatible characters or "cliques," which are significantly
      larger than cliques of shuffled characters. We suggest, therefore, that these
      cliques of characters are the result of similar selective pressures and are a
      signature of concerted convergence.
FAU - Holland, Barbara R
AU  - Holland BR
AD  - Institute of Fundamental Sciences, Massey University, Private Bag 11 222,
      Palmerston North 4442, New Zealand. b.r.holland@massey.ac.nz
FAU - Spencer, Hamish G
AU  - Spencer HG
FAU - Worthy, Trevor H
AU  - Worthy TH
FAU - Kennedy, Martyn
AU  - Kennedy M
LA  - eng
PT  - Journal Article
DEP - 20100514
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Birds/anatomy &amp; histology/classification/*genetics
MH  - Selection, Genetic
EDAT- 2010/06/16 06:00
MHDA- 2010/10/21 06:00
CRDT- 2010/06/16 06:00
PHST- 2010/06/16 06:00 [entrez]
PHST- 2010/06/16 06:00 [pubmed]
PHST- 2010/10/21 06:00 [medline]
AID - syq023 [pii]
AID - 10.1093/sysbio/syq023 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jul;59(4):433-45. doi: 10.1093/sysbio/syq023. Epub 2010 May 14.

PMID- 20547778
OWN - NLM
STAT- MEDLINE
DCOM- 20101020
LR  - 20100615
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 4
DP  - 2010 Jul
TI  - Evaluating nested clade phylogeographic analysis under models of restricted gene 
      flow.
PG  - 415-32
LID - 10.1093/sysbio/syq022 [doi]
AB  - Nested clade phylogeographic analysis (NCPA) is a widely used method that aims to
      identify past demographic events that have shaped the history of a population. In
      an earlier study, NCPA has been fully automated, allowing it to be tested with
      simulated data sets generated under a null model in which samples simulated from 
      a panmictic population are geographically distributed. It was noted that NCPA was
      prone to inferring false positives, corroborating earlier findings. The present
      study aims to evaluate both single-locus and multilocus NCPA under the scenario
      of restricted gene flow among spatially distributed populations. We have
      developed a new program, ANeCA-ML, which implements multilocus NCPA. Data were
      simulated under 3 models of gene flow: a stepping stone model, an island model,
      and a stepping stone model with some long-distance dispersal. Results indicate
      that single-locus NCPA tends to give a high frequency of false positives, but,
      unlike the random-mating scenario presented previously, inferences are not
      limited to restricted gene flow with isolation by distance or contiguous range
      expansion. The proportion of single-locus data sets that contained false
      inferences was 76% for the panmictic case, 87% for the stepping stone model, 79% 
      for the stepping stone model with long-distance dispersal, and more than 99% for 
      the island model. The frequency of inferences is inversely related to the amount 
      of gene flow between demes. We performed multilocus NCPA by grouping the
      simulated loci into data sets of 5 loci. The false-positive rate was reduced in
      multilocus NCPA for some inferences but remained high for others. The proportion 
      of multilocus data sets that contained false inferences was 17% for the panmictic
      case, 30% for the stepping stone model, 4% for the stepping stone model with
      long-distance dispersal, and 54% for the island model. Multilocus NCPA reduces
      the false-positive rate by restricting the sensitivity of the method but does not
      appear to increase the accuracy of the approach. Three classical tests-the
      analysis of molecular variance method, Fu's Fs, and the Mantel test-show that
      there is information in the data that gives rise to explicable results using
      these standard approaches. In conclusion, for the scenarios that we have
      examined, our simulation study suggests that the NCPA method is unreliable and
      its inferences may be misleading. We suggest that the NCPA method should not be
      used without objective simulation-based testing by independent researchers.
FAU - Panchal, Mahesh
AU  - Panchal M
AD  - School of Biological Sciences, University of Reading, Whiteknights, PO Box 228,
      Reading RG6 6AJ, UK. panchal@evolbio.mpg.de
FAU - Beaumont, Mark A
AU  - Beaumont MA
LA  - eng
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100524
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Computer Simulation
MH  - Demography
MH  - *Gene Flow
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2010/06/16 06:00
MHDA- 2010/10/21 06:00
CRDT- 2010/06/16 06:00
PHST- 2010/06/16 06:00 [entrez]
PHST- 2010/06/16 06:00 [pubmed]
PHST- 2010/10/21 06:00 [medline]
AID - syq022 [pii]
AID - 10.1093/sysbio/syq022 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jul;59(4):415-32. doi: 10.1093/sysbio/syq022. Epub 2010 May 24.

PMID- 20547777
OWN - NLM
STAT- MEDLINE
DCOM- 20101020
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 4
DP  - 2010 Jul
TI  - Species delimitation using a combined coalescent and information-theoretic
      approach: an example from North American Myotis bats.
PG  - 400-14
LID - 10.1093/sysbio/syq024 [doi]
AB  - Coalescent model-based methods for phylogeny estimation force systematists to
      confront issues related to the identification of species boundaries. Unlike
      conventional phylogenetic analysis, where species membership can be assessed
      qualitatively after the phylogeny is estimated, the phylogenies that are
      estimated under a coalescent model treat aggregates of individuals as the
      operational taxonomic units and thus require a priori definition of these sets
      because the models assume that the alleles in a given lineage are sampled from a 
      single panmictic population. Fortunately, the use of coalescent model-based
      approaches allows systematists to conduct probabilistic tests of species limits
      by calculating the probability of competing models of lineage composition. Here, 
      we conduct the first exploration of the issues related to applying such tests to 
      a complex empirical system. Sequence data from multiple loci were used to assess 
      species limits and phylogeny in a clade of North American Myotis bats. After
      estimating gene trees at each locus, the likelihood of models representing all
      hierarchical permutations of lineage composition was calculated and Akaike
      information criterion scores were computed. Metrics borrowed from information
      theory suggest that there is strong support for several models that include
      multiple evolutionary lineages within the currently described species Myotis
      lucifugus and M. evotis. Although these results are preliminary, they illustrate 
      the practical importance of coupled species delimitation and phylogeny
      estimation.
FAU - Carstens, Bryan C
AU  - Carstens BC
AD  - Department of Biological Sciences, Louisiana State University, 202 Life Sciences 
      Building, Baton Rouge, LA 70808, USA. carstens@lsu.edu
FAU - Dewey, Tanya A
AU  - Dewey TA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100524
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Chiroptera/*genetics
MH  - Demography
MH  - Genes, Mitochondrial
MH  - *Genetic Speciation
MH  - Genetic Variation
MH  - Information Theory
PMC - PMC2885268
EDAT- 2010/06/16 06:00
MHDA- 2010/10/21 06:00
CRDT- 2010/06/16 06:00
PHST- 2010/06/16 06:00 [entrez]
PHST- 2010/06/16 06:00 [pubmed]
PHST- 2010/10/21 06:00 [medline]
AID - syq024 [pii]
AID - 10.1093/sysbio/syq024 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jul;59(4):400-14. doi: 10.1093/sysbio/syq024. Epub 2010 May 24.

PMID- 20547776
OWN - NLM
STAT- MEDLINE
DCOM- 20101020
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 4
DP  - 2010 Jul
TI  - The teleost anatomy ontology: anatomical representation for the genomics age.
PG  - 369-83
LID - 10.1093/sysbio/syq013 [doi]
AB  - The rich knowledge of morphological variation among organisms reported in the
      systematic literature has remained in free-text format, impractical for use in
      large-scale synthetic phylogenetic work. This noncomputable format has also
      precluded linkage to the large knowledgebase of genomic, genetic, developmental, 
      and phenotype data in model organism databases. We have undertaken an effort to
      prototype a curated, ontology-based evolutionary morphology database that maps to
      these genetic databases (http://kb.phenoscape.org) to facilitate investigation
      into the mechanistic basis and evolution of phenotypic diversity. Among the first
      requirements in establishing this database was the development of a multispecies 
      anatomy ontology with the goal of capturing anatomical data in a systematic and
      computable manner. An ontology is a formal representation of a set of concepts
      with defined relationships between those concepts. Multispecies anatomy
      ontologies in particular are an efficient way to represent the diversity of
      morphological structures in a clade of organisms, but they present challenges in 
      their development relative to single-species anatomy ontologies. Here, we
      describe the Teleost Anatomy Ontology (TAO), a multispecies anatomy ontology for 
      teleost fishes derived from the Zebrafish Anatomical Ontology (ZFA) for the
      purpose of annotating varying morphological features across species. To
      facilitate interoperability with other anatomy ontologies, TAO uses the Common
      Anatomy Reference Ontology as a template for its upper level nodes, and TAO and
      ZFA are synchronized, with zebrafish terms specified as subtypes of teleost
      terms. We found that the details of ontology architecture have ramifications for 
      querying, and we present general challenges in developing a multispecies anatomy 
      ontology, including refinement of definitions, taxon-specific relationships among
      terms, and representation of taxonomically variable developmental pathways.
FAU - Dahdul, Wasila M
AU  - Dahdul WM
AD  - Department of Biology, University of South Dakota, Vermillion, SD 57069, USA.
      wasila.dahdul@usd.edu
FAU - Lundberg, John G
AU  - Lundberg JG
FAU - Midford, Peter E
AU  - Midford PE
FAU - Balhoff, James P
AU  - Balhoff JP
FAU - Lapp, Hilmar
AU  - Lapp H
FAU - Vision, Todd J
AU  - Vision TJ
FAU - Haendel, Melissa A
AU  - Haendel MA
FAU - Westerfield, Monte
AU  - Westerfield M
FAU - Mabee, Paula M
AU  - Mabee PM
LA  - eng
GR  - HG002659/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20100329
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Classification
MH  - Computational Biology
MH  - Databases, Factual
MH  - Fishes/*anatomy &amp; histology/*genetics
MH  - Genomics
PMC - PMC2885267
EDAT- 2010/06/16 06:00
MHDA- 2010/10/21 06:00
CRDT- 2010/06/16 06:00
PHST- 2010/06/16 06:00 [entrez]
PHST- 2010/06/16 06:00 [pubmed]
PHST- 2010/10/21 06:00 [medline]
AID - syq013 [pii]
AID - 10.1093/sysbio/syq013 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jul;59(4):369-83. doi: 10.1093/sysbio/syq013. Epub 2010 Mar 29.

PMID- 20538759
OWN - NLM
STAT- MEDLINE
DCOM- 20101020
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 4
DP  - 2010 Jul
TI  - Using fossils to break long branches in molecular dating: a comparison of relaxed
      clocks applied to the origin of angiosperms.
PG  - 384-99
LID - 10.1093/sysbio/syq027 [doi]
AB  - Long branches are potentially problematic in molecular dating because they can
      encompass a vast number of combinations of substitution rate and time. A long
      branch is suspected to have biased molecular clock estimates of the age of
      flowering plants (angiosperms) to be much older than their earliest fossils. This
      study explores the effect of the long branch subtending angiosperms in molecular 
      dating and how different relaxed clocks react to it. Fossil angiosperm relatives,
      identified through a combined morphological and molecular phylogenetic analysis
      for living and fossil seed plants, were used to break the long angiosperm stem
      branch. Nucleotide sequences of angiosperm fossil relatives were simulated using 
      a phylogeny and model parameters from living taxa and incorporated in molecular
      dating. Three relaxed clocks, which implement among-lineage rate heterogeneity
      differently, were used: penalized likelihood (using 2 different rate smoothing
      optimization criteria), a Bayesian rate-autocorrelated method, and a Bayesian
      uncorrelated method. Different clocks provided highly correlated ages across the 
      tree. Breaking the angiosperm stem branch did not result in major age
      differences, except for a few sensitive nodes. Breaking the angiosperm stem
      branch resulted in a substantially younger age for crown angiosperms only with 1 
      of the 4 methods, but, nevertheless, the obtained age is considerably older than 
      the oldest angiosperm fossils. The origin of crown angiosperms is estimated
      between the Upper Triassic and the early Permian. The difficulty in estimating
      crown angiosperm age probably lies in a combination of intrinsic and extrinsic
      complicating factors, including substantial molecular rate heterogeneity among
      lineages and through time. A more adequate molecular dating approach might
      combine moderate background rate heterogeneity with large changes in rate at
      particular points in the tree.
FAU - Magallon, Susana
AU  - Magallon S
AD  - Departamento de Botanica, Instituto de Biologia, Universidad Nacional Autonoma de
      Mexico, 3er Circuito de Ciudad Universitaria, Del. Coyoacan, Mexico D.F. 04510,
      Mexico. s.magallon@ibiologia.unam.mx
LA  - eng
PT  - Journal Article
DEP - 20100610
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Clocks
MH  - *Evolution, Molecular
MH  - *Fossils
MH  - Magnoliopsida/*genetics
MH  - Phylogeny
MH  - Time Factors
EDAT- 2010/06/12 06:00
MHDA- 2010/10/21 06:00
CRDT- 2010/06/12 06:00
PHST- 2010/06/12 06:00 [entrez]
PHST- 2010/06/12 06:00 [pubmed]
PHST- 2010/10/21 06:00 [medline]
AID - syq027 [pii]
AID - 10.1093/sysbio/syq027 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jul;59(4):384-99. doi: 10.1093/sysbio/syq027. Epub 2010 Jun 10.

PMID- 20530131
OWN - NLM
STAT- MEDLINE
DCOM- 20101104
LR  - 20101007
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 5
DP  - 2010 Oct
TI  - Did Kauri (Agathis: Araucariaceae) really survive the Oligocene drowning of New
      Zealand?
PG  - 594-602
LID - 10.1093/sysbio/syq030 [doi]
FAU - Biffin, Ed
AU  - Biffin E
AD  - Australian Centre for Evolutionary Biology and Biodiversity, School of Earth and 
      Environmental Science, University of Adelaide, Adelaide, South Australia,
      Australia.
FAU - Hill, Robert S
AU  - Hill RS
FAU - Lowe, Andrew J
AU  - Lowe AJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100607
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Coniferophyta/classification/*genetics
MH  - DNA, Plant/analysis
MH  - Fossils
MH  - Geological Phenomena
MH  - New Zealand
MH  - Phylogeny
MH  - Plastids/genetics
EDAT- 2010/06/10 06:00
MHDA- 2010/11/05 06:00
CRDT- 2010/06/10 06:00
PHST- 2010/06/10 06:00 [entrez]
PHST- 2010/06/10 06:00 [pubmed]
PHST- 2010/11/05 06:00 [medline]
AID - syq030 [pii]
AID - 10.1093/sysbio/syq030 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Oct;59(5):594-602. doi: 10.1093/sysbio/syq030. Epub 2010 Jun 7.

PMID- 20525640
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 3
DP  - 2010 May
TI  - Phylogeny and biogeography of dolichoderine ants: effects of data partitioning
      and relict taxa on historical inference.
PG  - 342-62
LID - 10.1093/sysbio/syq012 [doi]
AB  - Ants (Hymenoptera: Formicidae) are conspicuous organisms in most terrestrial
      ecosystems, often attaining high levels of abundance and diversity. In this
      study, we investigate the evolutionary history of a major clade of ants, the
      subfamily Dolichoderinae, whose species frequently achieve ecological dominance
      in ant communities. This group has also produced some of the world's most
      successful invasive ants. We use an extensive molecular data set ( approximately 
      9 kb of sequence data from 10 nuclear genes, covering 48 dolichoderine species
      and 6 outgroup taxa) to infer the phylogenetic relationships, divergence dates,
      and biogeographic history of these ants. We evaluate the effects of data
      partitioning and outgroup composition on phylogenetic inference by estimating
      relationships under a series of increasingly partitioned data sets and by running
      analyses both with and without Aneuretus simoni, a rare and localized species
      that is the nearest living relative of Dolichoderinae. We also examine the
      effects of excluding 2 data partitions with significant base composition
      heterogeneity. Our results reveal 4 well-supported and mutually exclusive clades 
      of dolichoderines, corresponding to 4 newly defined tribes: Bothriomyrmecini (B),
      Dolichoderini (D), Leptomyrmecini (L), and Tapinomini (T). All Bayesian and
      likelihood analyses yield the same unrooted (ingroup-only) topology,
      ((D,L),(B,T)), with the outgroups attaching either on the Dolichoderini branch or
      on the Tapinomini branch. Placement of the root is highly sensitive to choice of 
      model partition and to inclusion/exclusion of Aneuretus. Bayes' factors strongly 
      favor the more partitioned models, and in these Tapinomini is recovered as sister
      to the remaining dolichoderines, but only if Aneuretus is included. Exclusion of 
      Aneuretus precludes recovery of this topology in all but the most highly
      partitioned Bayesian analyses and then only with nonsignificant support,
      underscoring the importance of relict, taxonomically isolated taxa for
      phylogenetic inference. Removal of 2 partitions with heterogeneous base
      composition also markedly increases support for placement of the root on the
      Tapinomini branch. Our divergence date estimates and biogeographic analyses
      indicate that crown-group dolichoderines arose about 65 million years ago (Ma),
      although this was preceded by a substantial period (30 million years) of stem
      group evolution. The 4 extant tribes are estimated to have crown-group origins in
      the late Paleocene or Eocene (40-60 Ma). Tapinomini and Bothriomyrmecini
      originated in the Paleotropics and subsequently dispersed to other biogeographic 
      regions. Crown-group Leptomyrmecini arose and diversified in the Neotropics, but 
      they also gave rise to one clade that colonized Australia about 30 Ma and
      subsequently experienced a massive radiation on that continent. This event
      occurred later than the diversification of dolichoderines in the northern
      hemisphere, so that by the time dolichoderines came to dominate the Australian
      fauna they had already declined in abundance in the Holarctic region.
FAU - Ward, Philip S
AU  - Ward PS
AD  - Department of Entomology, University of California, Davis, CA 95616, USA.
      psward@ucdavis.edu
FAU - Brady, Sean G
AU  - Brady SG
FAU - Fisher, Brian L
AU  - Fisher BL
FAU - Schultz, Ted R
AU  - Schultz TR
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20100331
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Ants/classification/*genetics
MH  - Base Composition
MH  - Base Sequence
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - *Demography
MH  - Geography
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syq012 [pii]
AID - 10.1093/sysbio/syq012 [doi]
PST - ppublish
SO  - Syst Biol. 2010 May;59(3):342-62. doi: 10.1093/sysbio/syq012. Epub 2010 Mar 31.

PMID- 20525639
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 3
DP  - 2010 May
TI  - Combining historical biogeography with niche modeling in the Caprifolium clade of
      Lonicera (Caprifoliaceae, Dipsacales).
PG  - 322-41
LID - 10.1093/sysbio/syq011 [doi]
AB  - The Lonicera clade Caprifolium contains approximately 25 species distributed
      around the Northern Hemisphere, including in the Mediterranean climates of
      California and Europe. We sequenced the second intron of LFY to help resolve
      relationships within the clade where the internal transcribed spacer and
      chloroplast markers had previously failed to do so. Divergence time estimation
      and biogeographic analyses over the posterior distribution of dated trees suggest
      that a widespread ancestor was distributed across the Northern Hemisphere some
      7-17 million years ago. Asian species form a sister group to a clade in which the
      European species are sister to the North American species. We use climatic niche 
      modeling and divergence time estimates to explore the evolution of climate
      variables in the group. Principal component analyses help to identify instances
      of convergence, especially between distantly related species in the Mediterranean
      basin and in the chaparral of California. We document several cases of
      significant divergence between sister species in eastern North America and
      western North America. Climatic models were projected from one continent into the
      others (e.g., North American species projected into Asia and Europe) to examine
      whether species living in these areas occupy similar climates. This study
      demonstrates the utility of combining niche modeling with historical
      biogeographic analyses and documents significant climatic niche evolution within 
      a group of species distributed throughout the Northern Hemisphere. These results 
      suggest a possible model for the origin of the Madrean-Tethyan disjunction
      pattern.
FAU - Smith, Stephen A
AU  - Smith SA
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT
      06520, USA. sasmith@nescent.org
FAU - Donoghue, Michael J
AU  - Donoghue MJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100322
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (Plant Proteins)
RN  - 0 (Transcription Factors)
SB  - IM
MH  - Base Sequence
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - *Climate
MH  - DNA Primers/genetics
MH  - *Demography
MH  - *Ecosystem
MH  - Lonicera/classification/*genetics/physiology
MH  - Models, Genetic
MH  - *Models, Theoretical
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Plant Proteins/genetics
MH  - Principal Component Analysis
MH  - Sequence Analysis, DNA
MH  - Species Specificity
MH  - Transcription Factors/genetics
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syq011 [pii]
AID - 10.1093/sysbio/syq011 [doi]
PST - ppublish
SO  - Syst Biol. 2010 May;59(3):322-41. doi: 10.1093/sysbio/syq011. Epub 2010 Mar 22.

PMID- 20525638
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 3
DP  - 2010 May
TI  - New algorithms and methods to estimate maximum-likelihood phylogenies: assessing 
      the performance of PhyML 3.0.
PG  - 307-21
LID - 10.1093/sysbio/syq010 [doi]
AB  - PhyML is a phylogeny software based on the maximum-likelihood principle. Early
      PhyML versions used a fast algorithm performing nearest neighbor interchanges to 
      improve a reasonable starting tree topology. Since the original publication
      (Guindon S., Gascuel O. 2003. A simple, fast and accurate algorithm to estimate
      large phylogenies by maximum likelihood. Syst. Biol. 52:696-704), PhyML has been 
      widely used (&gt;2500 citations in ISI Web of Science) because of its simplicity and
      a fair compromise between accuracy and speed. In the meantime, research around
      PhyML has continued, and this article describes the new algorithms and methods
      implemented in the program. First, we introduce a new algorithm to search the
      tree space with user-defined intensity using subtree pruning and regrafting
      topological moves. The parsimony criterion is used here to filter out the least
      promising topology modifications with respect to the likelihood function. The
      analysis of a large collection of real nucleotide and amino acid data sets of
      various sizes demonstrates the good performance of this method. Second, we
      describe a new test to assess the support of the data for internal branches of a 
      phylogeny. This approach extends the recently proposed approximate
      likelihood-ratio test and relies on a nonparametric, Shimodaira-Hasegawa-like
      procedure. A detailed analysis of real alignments sheds light on the links
      between this new approach and the more classical nonparametric bootstrap method. 
      Overall, our tests show that the last version (3.0) of PhyML is fast, accurate,
      stable, and ready to use. A Web server and binary files are available from
      http://www.atgc-montpellier.fr/phyml/.
FAU - Guindon, Stephane
AU  - Guindon S
AD  - Methodes et Algorithmes pour la Bioinformatique, LIRMM, Centre National de la
      Recherche Scientifique, Universite de Montpellier, Montpellier Cedex 5, France.
FAU - Dufayard, Jean-Francois
AU  - Dufayard JF
FAU - Lefort, Vincent
AU  - Lefort V
FAU - Anisimova, Maria
AU  - Anisimova M
FAU - Hordijk, Wim
AU  - Hordijk W
FAU - Gascuel, Olivier
AU  - Gascuel O
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100329
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - *Software
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syq010 [pii]
AID - 10.1093/sysbio/syq010 [doi]
PST - ppublish
SO  - Syst Biol. 2010 May;59(3):307-21. doi: 10.1093/sysbio/syq010. Epub 2010 Mar 29.

PMID- 20525637
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 3
DP  - 2010 May
TI  - Regional variation exaggerates ecological divergence in niche models.
PG  - 298-306
LID - 10.1093/sysbio/syq005 [doi]
AB  - Traditionally, the goal of systematics has been to produce classifications that
      are both strongly supported and biologically meaningful. In recent years several 
      authors have advocated complementing phylogenetic analyses with measures of
      another form of evolutionary change, ecological divergence. These analyses
      frequently rely on ecological niche models to determine if species have
      comparable environmental requirements, but it has heretofore been difficult to
      test the accuracy of these inferences. To address this problem, I simulate the
      geographic distributions of allopatric species with identical environmental
      requirements. I then test whether existing analyses based on geographic
      distributions will correctly infer that the 2 species' requirements are
      identical. This work demonstrates that when taxa disperse to different
      environments, many analyses can erroneously infer changes in environmental
      requirements, but the severity of the problem depends on the method used. As this
      could exaggerate the number of ecologically distinct taxa in a clade, I suggest
      diagnostics to mitigate this problem.
FAU - Godsoe, William
AU  - Godsoe W
AD  - National Institute for Mathematical and Biological Synthesis, University of
      Tennessee, Knoxville, TN 37996-1527, USA.
LA  - eng
GR  - P20 RR 016454/RR/NCRR NIH HHS/United States
GR  - P20 RR 16448/RR/NCRR NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20100226
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Demography
MH  - *Ecosystem
MH  - *Genetic Speciation
MH  - Geography
MH  - *Models, Theoretical
MH  - Multivariate Analysis
MH  - Species Specificity
PMC - PMC2910897
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syq005 [pii]
AID - 10.1093/sysbio/syq005 [doi]
PST - ppublish
SO  - Syst Biol. 2010 May;59(3):298-306. doi: 10.1093/sysbio/syq005. Epub 2010 Feb 26.

PMID- 20525636
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 3
DP  - 2010 May
TI  - Phylogenetic tree reconstruction accuracy and model fit when proportions of
      variable sites change across the tree.
PG  - 288-97
LID - 10.1093/sysbio/syq003 [doi]
AB  - Commonly used phylogenetic models assume a homogeneous process through time in
      all parts of the tree. However, it is known that these models can be too
      simplistic as they do not account for nonhomogeneous lineage-specific properties.
      In particular, it is now widely recognized that as constraints on sequences
      evolve, the proportion and positions of variable sites can vary between lineages 
      causing heterotachy. The extent to which this model misspecification affects tree
      reconstruction is still unknown. Here, we evaluate the effect of changes in the
      proportions and positions of variable sites on model fit and tree estimation. We 
      consider 5 current models of nucleotide sequence evolution in a Bayesian Markov
      chain Monte Carlo framework as well as maximum parsimony (MP). We show that for a
      tree with 4 lineages where 2 nonsister taxa undergo a change in the proportion of
      variable sites tree reconstruction under the best-fitting model, which is chosen 
      using a relative test, often results in the wrong tree. In this case, we found
      that an absolute test of model fit is a better predictor of tree estimation
      accuracy. We also found further evidence that MP is not immune to heterotachy. In
      addition, we show that increased sampling of taxa that have undergone a change in
      proportion and positions of variable sites is critical for accurate tree
      reconstruction.
FAU - Shavit Grievink, Liat
AU  - Shavit Grievink L
AD  - Institut fur Botanik III, Heinrich-Heine Universitat, Universitatstrasse 1,
      Dusseldorf, Germany. liat.shavitgrievink@uni-duesseldorf.de
FAU - Penny, David
AU  - Penny D
FAU - Hendy, Michael D
AU  - Hendy MD
FAU - Holland, Barbara R
AU  - Holland BR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100301
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - *Evolution, Molecular
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Sample Size
PMC - PMC2850392
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syq003 [pii]
AID - 10.1093/sysbio/syq003 [doi]
PST - ppublish
SO  - Syst Biol. 2010 May;59(3):288-97. doi: 10.1093/sysbio/syq003. Epub 2010 Mar 1.

PMID- 20525635
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 3
DP  - 2010 May
TI  - Accounting for solvent accessibility and secondary structure in protein
      phylogenetics is clearly beneficial.
PG  - 277-87
LID - 10.1093/sysbio/syq002 [doi]
AB  - Amino acid substitution models are essential to most methods to infer phylogenies
      from protein data. These models represent the ways in which proteins evolve and
      substitutions accumulate along the course of time. It is widely accepted that the
      substitution processes vary depending on the structural configuration of the
      protein residues. However, this information is very rarely used in phylogenetic
      studies, though the 3-dimensional structure of dozens of thousands of proteins
      has been elucidated. Here, we reinvestigate the question in order to fill this
      gap. We use an improved estimation methodology and a very large database
      comprising 1471 nonredundant globular protein alignments with structural
      annotations to estimate new amino acid substitution models accounting for the
      secondary structure and solvent accessibility of the residues. These models
      incorporate a confidence coefficient that is estimated from the data and reflects
      the reliability and usefulness of structural annotations in the analyzed
      sequences. Our results with 300 independent test alignments show an impressive
      likelihood gain compared with standard models such as JTT or WAG. Moreover, the
      use of these models induces significant topological changes in the inferred
      trees, which should be of primary interest to phylogeneticists. Our data, models,
      and software are available for download from
      http://atgc.lirmm.fr/phyml-structure/.
FAU - Le, Si Quang
AU  - Le SQ
AD  - Methodes et Algorithmes pour la Bioinformatique, LIRMM, CNRS-Universite
      Montpellier II, 161 rue Ada, Montpellier Cedex 5, France.
FAU - Gascuel, Olivier
AU  - Gascuel O
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100310
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Proteins)
SB  - IM
MH  - Amino Acid Sequence
MH  - Classification/*methods
MH  - Databases, Genetic
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - *Protein Conformation
MH  - Proteins/*genetics
MH  - Sequence Alignment
MH  - Software
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syq002 [pii]
AID - 10.1093/sysbio/syq002 [doi]
PST - ppublish
SO  - Syst Biol. 2010 May;59(3):277-87. doi: 10.1093/sysbio/syq002. Epub 2010 Mar 10.

PMID- 20525634
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 3
DP  - 2010 May
TI  - A network approach to study karyotypic evolution: the chromosomal races of the
      common shrew (Sorex araneus) and house mouse (Mus musculus) as model systems.
PG  - 262-76
LID - 10.1093/sysbio/syq004 [doi]
AB  - The development of methods to reconstruct phylogenies from karyotypic data has
      lagged behind what has been achieved with molecular and morphological characters.
      This hampers our understanding of the role of chromosomal rearrangements in
      speciation, which depends on knowledge of the karyotypic relationships both among
      forms that have recently speciated and among forms within species that may
      speciate in the future. Here, we present a new approach to reconstruct
      chromosomal phylogenies. Our approach involves the use of networks, which we
      believe offer a flexible alternative to bifurcating phylogenetic trees for
      chromosomal phylogenetic analyses, and can incorporate a wide range of
      chromosomal mutations as well as allowing reticulate evolution through
      hybridization. In this paper, we apply our method at the within-species level to 
      establish the phylogenetic history, in terms of minimum number of evolutionary
      steps, of chromosomal races within both the common shrew (Sorex araneus) and the 
      house mouse (Mus musculus). There have been several previous attempts to
      reconstruct the phylogenies of chromosomal races within shrews and mice, but we
      describe the first computer-based analysis that considers the whole range of
      possible mutations generating new races (Robertsonian fusions and fissions and
      whole-arm reciprocal translocations [WARTs]) and other race-generating processes 
      (zonal raciation events involving both acrocentric and recombinant peaks)
      postulated for these species. The analysis for common shrew chromosomal races
      reveals a greater importance of zonal raciation and WARTs than has been suggested
      hitherto.
FAU - White, Thomas A
AU  - White TA
AD  - School of Biological and Biomedical Sciences, Durham University, South Road,
      Durham, UK. tawhite201@hotmail.com
FAU - Bordewich, Magnus
AU  - Bordewich M
FAU - Searle, Jeremy B
AU  - Searle JB
LA  - eng
PT  - Comparative Study
PT  - Journal Article
DEP - 20100301
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Chromosomes, Mammalian/*genetics
MH  - Classification/*methods
MH  - Europe
MH  - *Evolution, Molecular
MH  - *Hybridization, Genetic
MH  - Karyotyping/methods
MH  - Mice
MH  - Mutation/genetics
MH  - *Phylogeny
MH  - Shrews/*genetics
MH  - Species Specificity
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syq004 [pii]
AID - 10.1093/sysbio/syq004 [doi]
PST - ppublish
SO  - Syst Biol. 2010 May;59(3):262-76. doi: 10.1093/sysbio/syq004. Epub 2010 Mar 1.

PMID- 20525633
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20170922
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 3
DP  - 2010 May
TI  - Testing and quantifying phylogenetic signals and homoplasy in morphometric data.
PG  - 245-61
LID - 10.1093/sysbio/syp106 [doi]
AB  - The relationship between morphometrics and phylogenetic analysis has long been
      controversial. Here we propose an approach that is based on mapping morphometric 
      traits onto phylogenies derived from other data and thus avoids the pitfalls
      encountered by previous studies. This method treats shape as a single,
      multidimensional character. We propose a test for the presence of a phylogenetic 
      signal in morphometric data, which simulates the null hypothesis of the complete 
      absence of phylogenetic structure by permutation of the shape data among the
      terminal taxa. We also propose 2 measures of the fit of morphometric data to the 
      phylogeny that are direct extensions of the consistency index and retention index
      used in traditional cladistics. We apply these methods to a small study of the
      evolution of wing shape in the Drosophila melanogaster subgroup, for which a very
      strongly supported phylogeny is available. This case study reveals a significant 
      phylogenetic signal and a relatively low degree of homoplasy. Despite the low
      homoplasy, the shortest tree computed from landmark data on wing shape is
      inconsistent with the well-supported phylogenetic tree from molecular data,
      underscoring that morphometric data may not provide reliable information for
      inferring phylogeny.
FAU - Klingenberg, Christian Peter
AU  - Klingenberg CP
AD  - Faculty of Life Sciences, University of Manchester, Michael Smith Building,
      Oxford Road, Manchester, UK. cpk@manchester.ac.uk
FAU - Gidaszewski, Nelly A
AU  - Gidaszewski NA
LA  - eng
GR  - BBS/B/12253/Biotechnology and Biological Sciences Research Council/United Kingdom
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100115
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biometry/*methods
MH  - Body Weights and Measures/*methods
MH  - Classification/*methods
MH  - Drosophila/*anatomy &amp; histology/genetics
MH  - *Phylogeny
MH  - Wings, Animal/*anatomy &amp; histology
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp106 [pii]
AID - 10.1093/sysbio/syp106 [doi]
PST - ppublish
SO  - Syst Biol. 2010 May;59(3):245-61. doi: 10.1093/sysbio/syp106. Epub 2010 Jan 15.

PMID- 20525632
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110603
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 2
DP  - 2010 Mar
TI  - An evaluation of phylogenetic informativeness profiles and the molecular
      phylogeny of diplazontinae (Hymenoptera, Ichneumonidae).
PG  - 226-41
LID - 10.1093/sysbio/syp105 [doi]
AB  - How to quantify the phylogenetic information content of a data set is a
      longstanding question in phylogenetics, influencing both the assessment of data
      quality in completed studies and the planning of future phylogenetic projects.
      Recently, a method has been developed that profiles the phylogenetic
      informativeness (PI) of a data set through time by linking its site-specific
      rates of change to its power to resolve relationships at different timescales.
      Here, we evaluate the performance of this method in the case of 2 standard
      genetic markers for phylogenetic reconstruction, 28S ribosomal RNA and cytochrome
      oxidase subunit 1 (CO1) mitochondrial DNA, with maximum parsimony, maximum
      likelihood, and Bayesian analyses of relationships within a group of parasitoid
      wasps (Hymenoptera: Ichneumonidae, Diplazontinae). Retrieving PI profiles of the 
      2 genes from our own and from 3 additional data sets, we find that the method
      repeatedly overestimates the performance of the more quickly evolving CO1
      compared with 28S. We explore possible reasons for this bias, including
      phylogenetic uncertainty, violation of the molecular clock assumption, model
      misspecification, and nonstationary nucleotide composition. As none of these
      provides a sufficient explanation of the observed discrepancy, we use simulated
      data sets, based on an idealized setting, to show that the optimum evolutionary
      rate decreases with increasing number of taxa. We suggest that this relationship 
      could explain why the formula derived from the 4-taxon case overrates the
      performance of higher versus lower rates of evolution in our case and that
      caution should be taken when the method is applied to data sets including more
      than 4 taxa.
FAU - Klopfstein, Seraina
AU  - Klopfstein S
AD  - Department of Invertebrates, Natural History Museum, Bernastrasse 15, CH-3005
      Bern, Switzerland. klopfstein@nmbe.ch
FAU - Kropf, Christian
AU  - Kropf C
FAU - Quicke, Donald L J
AU  - Quicke DL
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100113
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (RNA, Ribosomal, 28S)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
CIN - Syst Biol. 2011 May;60(3):358-65. PMID: 21303824
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - DNA Primers/genetics
MH  - Electron Transport Complex IV/genetics
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - RNA, Ribosomal, 28S/genetics
MH  - *Research Design
MH  - Sequence Analysis, DNA
MH  - Wasps/*genetics
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp105 [pii]
AID - 10.1093/sysbio/syp105 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Mar;59(2):226-41. doi: 10.1093/sysbio/syp105. Epub 2010 Jan 13.

PMID- 20525631
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 2
DP  - 2010 Mar
TI  - An evolutionary analysis of lateral gene transfer in thymidylate synthase
      enzymes.
PG  - 212-25
LID - 10.1093/sysbio/syp104 [doi]
AB  - Thymidylate synthases (Thy) are key enzymes in the synthesis of deoxythymidylate,
      1 of the 4 building blocks of DNA. As such, they are essential for all DNA-based 
      forms of life and therefore implicated in the hypothesized transition from RNA
      genomes to DNA genomes. Two evolutionally unrelated Thy enzymes, ThyA and ThyX,
      are known to catalyze the same biochemical reaction. Both enzymes are
      sporadically distributed within each of the 3 domains of life in a pattern that
      suggests multiple nonhomologous lateral gene transfer (LGT) events. We present a 
      phylogenetic analysis of the evolution of the 2 enzymes, aimed at unraveling
      their entangled evolutionary history and tracing their origin back to early life.
      A novel probabilistic evolutionary model was developed, which allowed us to
      compute the posterior probabilities and the posterior expectation of the number
      of LGT events. Simulation studies were performed to validate the model's ability 
      to accurately detect LGT events, which have occurred throughout a large
      phylogeny. Applying the model to the Thy data revealed widespread nonhomologous
      LGT between and within all 3 domains of life. By reconstructing the ThyA and ThyX
      gene trees, the most likely donor of each LGT event was inferred. The role of
      viruses in LGT of Thy is finally discussed.
FAU - Stern, Adi
AU  - Stern A
AD  - Department of Cell Research and Immunology, George S. Wise Faculty of Life
      Sciences, Tel Aviv University, Tel Aviv 69978, Israel.
FAU - Mayrose, Itay
AU  - Mayrose I
FAU - Penn, Osnat
AU  - Penn O
FAU - Shaul, Shaul
AU  - Shaul S
FAU - Gophna, Uri
AU  - Gophna U
FAU - Pupko, Tal
AU  - Pupko T
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100115
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 2.1.1.45 (Thymidylate Synthase)
SB  - IM
MH  - Base Composition
MH  - Base Sequence
MH  - Classification/methods
MH  - Computational Biology
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - *Gene Transfer, Horizontal
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Thymidylate Synthase/*genetics
PMC - PMC2826268
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp104 [pii]
AID - 10.1093/sysbio/syp104 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Mar;59(2):212-25. doi: 10.1093/sysbio/syp104. Epub 2010 Jan 15.

PMID- 20525630
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20100607
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 2
DP  - 2010 Mar
TI  - Inferring and validating horizontal gene transfer events using bipartition
      dissimilarity.
PG  - 195-211
LID - 10.1093/sysbio/syp103 [doi]
AB  - Horizontal gene transfer (HGT) is one of the main mechanisms driving the
      evolution of microorganisms. Its accurate identification is one of the major
      challenges posed by reticulate evolution. In this article, we describe a new
      polynomial-time algorithm for inferring HGT events and compare 3 existing and 1
      new tree comparison indices in the context of HGT identification. The proposed
      algorithm can rely on different optimization criteria, including least squares
      (LS), Robinson and Foulds (RF) distance, quartet distance (QD), and bipartition
      dissimilarity (BD), when searching for an optimal scenario of subtree prune and
      regraft (SPR) moves needed to transform the given species tree into the given
      gene tree. As the simulation results suggest, the algorithmic strategy based on
      BD, introduced in this article, generally provides better results than those
      based on LS, RF, and QD. The BD-based algorithm also proved to be more accurate
      and faster than a well-known polynomial time heuristic RIATA-HGT. Moreover, the
      HGT recovery results yielded by BD were generally equivalent to those provided by
      the exponential-time algorithm LatTrans, but a clear gain in running time was
      obtained using the new algorithm. Finally, a statistical framework for assessing 
      the reliability of obtained HGTs by bootstrap analysis is also presented.
FAU - Boc, Alix
AU  - Boc A
AD  - Departement d'informatique, Universite du Quebec a Montreal, C.P. 8888, Succ.
      Centre-ville, Montreal, Quebec, Canada. boc.alix@courrier.uqam.ca
FAU - Philippe, Herve
AU  - Philippe H
FAU - Makarenkov, Vladimir
AU  - Makarenkov V
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20100121
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - *Data Interpretation, Statistical
MH  - *Evolution, Molecular
MH  - Gene Transfer, Horizontal/*genetics
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp103 [pii]
AID - 10.1093/sysbio/syp103 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Mar;59(2):195-211. doi: 10.1093/sysbio/syp103. Epub 2010 Jan 21.

PMID- 20525629
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20100607
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 2
DP  - 2010 Mar
TI  - A likelihood method for assessing molecular divergence time estimates and the
      placement of fossil calibrations.
PG  - 185-94
LID - 10.1093/sysbio/syp090 [doi]
AB  - Estimating divergence times using molecular sequence data has become a common
      application of statistical phylogenetics. However, disparities between estimated 
      ages for clades produced in different studies have become equally commonplace.
      Here, I propose a method for the objective assessment of the likelihood of
      inferred divergence times to evaluate the placement of fossil constraints using
      information from the broader fossil record. The inclusion of nodes from the Tree 
      of Life for which credible age ranges are known, in addition to the fossil
      constraints used in the ingroup, will allow for the comparison of alternate
      fossil placements when the phylogenetic affinity of a fossil is ambiguous as well
      as provide a heuristic assessment of the global likelihood of estimated
      divergence times. The use of these "likelihood checkpoints" will allow for the
      comparison of inferred dates across data sets and across taxonomic groups to
      place divergence time estimates into a broader evolutionary timescale. The method
      is illustrated with an example using an expanded phylogenetic estimate of the
      Gnathostomata, inferred with relaxed-clock molecular dating methods.
FAU - Pyron, R Alexander
AU  - Pyron RA
AD  - The Graduate School and University Center, The City University of New York, 365
      Fifth Avenue, New York, NY 10016, USA. rpyron@life.bio.sunysb.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20091217
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - *Fossils
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Vertebrates/genetics
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp090 [pii]
AID - 10.1093/sysbio/syp090 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Mar;59(2):185-94. doi: 10.1093/sysbio/syp090. Epub 2009 Dec 17.

PMID- 20525628
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20100607
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 2
DP  - 2010 Mar
TI  - Species delimitation: a case study in a problematic ant taxon.
PG  - 162-84
LID - 10.1093/sysbio/syp089 [doi]
AB  - Species delimitation has been invigorated as a discipline in systematics by an
      influx of new character sets, analytical methods, and conceptual advances. We use
      genetic data from 68 markers, combined with distributional, bioclimatic, and
      coloration information, to hypothesize boundaries of evolutionarily independent
      lineages (species) within the widespread and highly variable nominal fire ant
      species Solenopsis saevissima, a member of a species group containing invasive
      pests as well as species that are models for ecological and evolutionary
      research. Our integrated approach uses diverse methods of analysis to
      sequentially test whether populations meet specific operational criteria
      (contingent properties) for candidacy as morphologically cryptic species,
      including genetic clustering, monophyly, reproductive isolation, and occupation
      of distinctive niche space. We hypothesize that nominal S. saevissima comprises
      at least 4-6 previously unrecognized species, including several pairs whose
      parapatric distributions implicate the development of intrinsic premating or
      postmating barriers to gene flow. Our genetic data further suggest that regional 
      genetic differentiation in S. saevissima has been influenced by hybridization
      with other nominal species occurring in sympatry or parapatry, including the
      quite distantly related Solenopsis geminata. The results of this study illustrate
      the importance of employing different classes of genetic data (coding and
      noncoding regions and nuclear and mitochondrial DNA [mtDNA] markers), different
      methods of genetic data analysis (tree-based and non-tree based methods), and
      different sources of data (genetic, morphological, and ecological data) to
      explicitly test various operational criteria for species boundaries in clades of 
      recently diverged lineages, while warning against over reliance on any single
      data type (e.g., mtDNA sequence variation) when drawing inferences.
FAU - Ross, Kenneth G
AU  - Ross KG
AD  - Department of Entomology, University of Georgia, Athens, GA 30602, USA.
      kenross@uga.edu
FAU - Gotzek, Dietrich
AU  - Gotzek D
FAU - Ascunce, Marina S
AU  - Ascunce MS
FAU - Shoemaker, D DeWayne
AU  - Shoemaker DD
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20091214
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Isoenzymes)
SB  - IM
MH  - Animals
MH  - Ants/*classification/*genetics
MH  - Bayes Theorem
MH  - Brazil
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Demography
MH  - Gene Flow/*genetics
MH  - Gene Frequency
MH  - *Genetic Variation
MH  - Genotype
MH  - Isoenzymes/genetics
MH  - Likelihood Functions
MH  - Microsatellite Repeats/genetics
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp089 [pii]
AID - 10.1093/sysbio/syp089 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Mar;59(2):162-84. doi: 10.1093/sysbio/syp089. Epub 2009 Dec 14.

PMID- 20525627
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20100607
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 2
DP  - 2010 Mar
TI  - When trees grow too long: investigating the causes of highly inaccurate bayesian 
      branch-length estimates.
PG  - 145-61
LID - 10.1093/sysbio/syp081 [doi]
AB  - A surprising number of recent Bayesian phylogenetic analyses contain
      branch-length estimates that are several orders of magnitude longer than
      corresponding maximum-likelihood estimates. The levels of divergence implied by
      such branch lengths are unreasonable for studies using biological data and are
      known to be false for studies using simulated data. We conducted additional
      Bayesian analyses and studied approximate-posterior surfaces to investigate the
      causes underlying these large errors. We manipulated the starting parameter
      values of the Markov chain Monte Carlo (MCMC) analyses, the moves used by the
      MCMC analyses, and the prior-probability distribution on branch lengths. We
      demonstrate that inaccurate branch-length estimates result from either 1) poor
      mixing of MCMC chains or 2) posterior distributions with excessive weight at long
      tree lengths. Both effects are caused by a rapid increase in the volume of
      branch-length space as branches become longer. In the former case, both an MCMC
      move that scales all branch lengths in the tree simultaneously and the use of
      overdispersed starting branch lengths allow the chain to accurately sample the
      posterior distribution and should be used in Bayesian analyses of phylogeny. In
      the latter case, branch-length priors can have strong effects on resulting
      inferences and should be carefully chosen to reflect biological expectations. We 
      provide a formula to calculate an exponential rate parameter for the
      branch-length prior that should eliminate inference of biased branch lengths in
      many cases. In any phylogenetic analysis, the biological plausibility of
      branch-length output must be carefully considered.
FAU - Brown, Jeremy M
AU  - Brown JM
AD  - Section of Integrative Biology and Center for Computational Biology and
      Bioinformatics, University of Texas at Austin, 1 University Station C0930,
      Austin, TX 78712, USA. jembrown@berkeley.edu
FAU - Hedtke, Shannon M
AU  - Hedtke SM
FAU - Lemmon, Alan R
AU  - Lemmon AR
FAU - Lemmon, Emily Moriarty
AU  - Lemmon EM
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20091210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - Markov Chains
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - *Research Design
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp081 [pii]
AID - 10.1093/sysbio/syp081 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Mar;59(2):145-61. doi: 10.1093/sysbio/syp081. Epub 2009 Dec 10.

PMID- 20525626
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20100607
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 2
DP  - 2010 Mar
TI  - Probabilistic models of chromosome number evolution and the inference of
      polyploidy.
PG  - 132-44
LID - 10.1093/sysbio/syp083 [doi]
AB  - Polyploidy, the genome wide duplication of chromosome number, is a key feature in
      eukaryote evolution. Polyploidy exists in diverse groups including animals,
      fungi, and invertebrates but is especially prevalent in plants with most, if not 
      all, plant species having descended from a polyploidization event. Polyploids
      often differ markedly from their diploid progenitors in morphological,
      physiological, and life history characteristics as well as rates of adaptation.
      The altered characteristics displayed by polyploids may contribute to their
      success in novel ecological habitats. Clearly, a better understanding of the
      processes underlying changes in the number of chromosomes within genomes is a key
      goal in our understanding of speciation and adaptation for a wide range of
      families and genera. Despite the fundamental role of chromosome number change in 
      eukaryotic evolution, probabilistic models describing the evolution of chromosome
      number along a phylogeny have not yet been formulated. We present a series of
      likelihood models, each representing a different hypothesis regarding the
      evolution of chromosome number along a given phylogeny. These models allow us to 
      reconstruct ancestral chromosome numbers and to estimate the expected number of
      polyploidization events and single chromosome changes (dysploidy) that occurred
      along a phylogeny. We test, using simulations, the accuracy of this approach and 
      its dependence on the number of taxa and tree length. We then demonstrate the
      application of the method for the study of chromosome number evolution in 4 plant
      genera: Aristolochia, Carex, Passiflora, and Helianthus. Considering the depth of
      the available cytological and phylogenetic data, formal models of chromosome
      number evolution are expected to advance significantly our understanding of the
      importance of polyploidy and dysploidy across different taxonomic groups.
FAU - Mayrose, Itay
AU  - Mayrose I
AD  - Department of Zoology, University of British Columbia, Vancouver, BC V6T 1Z4,
      Canada. mayrose@zoology.ubc.ca
FAU - Barker, Michael S
AU  - Barker MS
FAU - Otto, Sarah P
AU  - Otto SP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20091210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Biological/*genetics
MH  - Chromosomes/*genetics
MH  - Classification/methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Plants/genetics
MH  - *Polyploidy
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp083 [pii]
AID - 10.1093/sysbio/syp083 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Mar;59(2):132-44. doi: 10.1093/sysbio/syp083. Epub 2009 Dec 10.

PMID- 20525625
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20100607
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 2
DP  - 2010 Mar
TI  - Bayesian dating of shallow phylogenies with a relaxed clock.
PG  - 119-31
LID - 10.1093/sysbio/syp082 [doi]
AB  - Bayesian methods are increasingly being used to estimate divergence times without
      the restrictive assumption of a global clock. Little is known about their
      reliability for shallow phylogenies where DNA sequence divergence is low. We
      analyzed both simulated and real sequences to evaluate dating methods in
      phylogenies with mid-late Miocene roots. A large number of data sets (5000) with 
      10 taxa each were simulated under a rate-drift model for trees with 2 topologies 
      (balanced or unbalanced) and with different sets of divergence times
      (characterized by long or short external branches). Data were analyzed using
      Bayesian Markov chain Monte Carlo methods in which the prior on divergence times 
      was specified from a birth-death process with species sampling (BDS) or a
      Dirichlet distribution using the programs MCMCTREE and MULTIDIVTIME. The programs
      generally performed well on shallow phylogenies, but posterior mean node ages
      were biased and 95% posterior intervals included true ages in fewer than 95% of
      trees in some analyses. This typically occurred when the 95% prior interval did
      not include the true age and/or sequence lengths were &lt;/= 1 kbp. Widths of
      posterior intervals were also very dependent on the position of the calibrated
      node within the tree, particularly when sequences were short. Different
      divergence times priors within MCMCTREE, MULTIDIVTIME, and BEAST were used to
      analyze mitochondrial DNA data sets from a Bovid subfamily (the Caprinae) from
      Asian Laudakia and North African Chalcides lizards. Posterior divergence times
      were quite sensitive to different BDS priors but less sensitive to different
      Dirichlet priors. Our study demonstrates the impact of the prior on divergence
      times in shallow phylogenies and shows that 1) prior intervals on nodes should be
      assessed as a prerequisite to a dating analysis, 2) &gt;or= 1 kbp of quite rapidly
      evolving sequence may be required to obtain accurate posterior means and usefully
      narrow posterior intervals.
FAU - Brown, Richard P
AU  - Brown RP
AD  - School of Natural Sciences and Psychology, Liverpool John Moores University,
      Liverpool L3 3AF, UK. r.p.brown@ljmu.ac.uk .
FAU - Yang, Ziheng
AU  - Yang Z
LA  - eng
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20091210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Lizards/genetics
MH  - Markov Chains
MH  - Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Ruminants/genetics
MH  - Species Specificity
MH  - Time Factors
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp082 [pii]
AID - 10.1093/sysbio/syp082 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Mar;59(2):119-31. doi: 10.1093/sysbio/syp082. Epub 2009 Dec 10.

PMID- 20525623
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20100607
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 1
DP  - 2010 Jan
TI  - Cryptic failure of partitioned Bayesian phylogenetic analyses: lost in the land
      of long trees.
PG  - 108-17
LID - 10.1093/sysbio/syp080 [doi]
AB  - Partitioned Bayesian phylogenetic analyses of routine genetic data sets,
      constructed using MrBayes (Ronquist and Huelsenbeck 2003), can become trapped in 
      regions of parameter space characterized by unrealistically long trees and
      distorted partition rate multipliers. Such analyses commonly fail to reach
      stationarity during hundreds of millions of generations of sampling-many times
      longer than most published analyses. Some data sets are so prone to this problem 
      that paired MrBayes runs begun from different starting trees repeatedly find the 
      same incorrect long-tree solutions and consequently pass the most commonly
      employed tests of stationarity, including the average standard deviation of split
      frequencies (ASDSF) and the potential scale reduction factor (PSRF) statistics
      offered by MrBayes (Gelman and Rubin 1992). In these situations, failure to reach
      stationarity is recognizable only in light of prior knowledge of model
      parameters, such as the expectation that third-codon-position sites usually
      evolve fastest in protein-coding genes. The conditions that lead to the long-tree
      problem are frequently encountered in phylogenetic studies today, and I present 6
      demonstration examples from the literature. Although the effects on tree length
      (TL) are often dramatic, effects on topology appear to be subtle. Susceptibility 
      to the problem is sometimes predicted by the difference between the true TL and
      the starting TL. In some cases, the problems described here can be avoided or
      reduced by manipulation of the starting TL and/or by adjustments to the prior on 
      branch lengths. In more difficult situations, accurate branch length estimation
      may not be possible with Bayesian methods because of dependence of the solution
      on the branch length prior.
FAU - Marshall, David C
AU  - Marshall DC
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, U-3043, Storrs, CT 06269, USA. david.marshall@uconn.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20091117
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - *Data Collection
MH  - *Evolution, Molecular
MH  - Markov Chains
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp080 [pii]
AID - 10.1093/sysbio/syp080 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jan;59(1):108-17. doi: 10.1093/sysbio/syp080. Epub 2009 Nov 17.

PMID- 20525622
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20101118
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 1
DP  - 2010 Jan
TI  - Tinamous and moa flock together: mitochondrial genome sequence analysis reveals
      independent losses of flight among ratites.
PG  - 90-107
LID - 10.1093/sysbio/syp079 [doi]
AB  - Ratites are large, flightless birds and include the ostrich, rheas, kiwi, emu,
      and cassowaries, along with extinct members, such as moa and elephant birds.
      Previous phylogenetic analyses of complete mitochondrial genome sequences have
      reinforced the traditional belief that ratites are monophyletic and tinamous are 
      their sister group. However, in these studies ratite monophyly was enforced in
      the analyses that modeled rate heterogeneity among variable sites. Relaxing this 
      topological constraint results in strong support for the tinamous (which fly)
      nesting within ratites. Furthermore, upon reducing base compositional bias and
      partitioning models of sequence evolution among protein codon positions and RNA
      structures, the tinamou-moa clade grouped with kiwi, emu, and cassowaries to the 
      exclusion of the successively more divergent rheas and ostrich. These
      relationships are consistent with recent results from a large nuclear data set,
      whereas our strongly supported finding of a tinamou-moa grouping further resolves
      palaeognath phylogeny. We infer flight to have been lost among ratites multiple
      times in temporally close association with the Cretaceous-Tertiary extinction
      event. This circumvents requirements for transient microcontinents and island
      chains to explain discordance between ratite phylogeny and patterns of
      continental breakup. Ostriches may have dispersed to Africa from Eurasia, putting
      in question the status of ratites as an iconic Gondwanan relict taxon.
FAU - Phillips, Matthew J
AU  - Phillips MJ
AD  - Centre for Macroevolution and Macroecology, School of Botany and Zoology,
      Australian National University, Canberra, ACT 0200, Australia.
      matt.phillips@anu.edu.au
FAU - Gibb, Gillian C
AU  - Gibb GC
FAU - Crimp, Elizabeth A
AU  - Crimp EA
FAU - Penny, David
AU  - Penny D
LA  - eng
PT  - Journal Article
DEP - 20091113
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Composition
MH  - Base Sequence
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - DNA, Mitochondrial/*genetics
MH  - Evolution, Molecular
MH  - *Flight, Animal
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - Palaeognathae/*genetics
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp079 [pii]
AID - 10.1093/sysbio/syp079 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jan;59(1):90-107. doi: 10.1093/sysbio/syp079. Epub 2009 Nov 13.

PMID- 20525621
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20171110
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 1
DP  - 2010 Jan
TI  - The impact of the representation of fossil calibrations on Bayesian estimation of
      species divergence times.
PG  - 74-89
LID - 10.1093/sysbio/syp078 [doi]
AB  - Bayesian inference provides a powerful framework for integrating different
      sources of information (in particular, molecules and fossils) to derive estimates
      of species divergence times. Indeed, it is currently the only framework that can 
      adequately account for uncertainties in fossil calibrations. We use 2 Bayesian
      Markov chain Monte Carlo programs, MULTIDIVTIME and MCMCTREE, to analyze 3
      empirical datasets to estimate divergence times in amphibians, actinopterygians, 
      and felids. We evaluate the impact of various factors, including the priors on
      rates and times, fossil calibrations, substitution model, the violation of the
      molecular clock and the rate-drift model, and the exact and approximate
      likelihood calculation. Assuming the molecular clock caused seriously biased time
      estimates when the clock is violated, but 2 different rate-drift models produced 
      similar estimates. The prior on times, which incorporates fossil-calibration
      information, had the greatest impact on posterior time estimation. In particular,
      the strategies used by the 2 programs to incorporate minimum- and maximum-age
      bounds led to very different time priors and were responsible for large
      differences in posterior time estimates in a previous study. The results
      highlight the critical importance of fossil calibrations to molecular dating and 
      the need for probabilistic modeling of fossil depositions, preservations, and
      sampling to provide statistical summaries of information in the fossil record
      concerning species divergence times.
FAU - Inoue, Jun
AU  - Inoue J
AD  - Department of Biology, Galton Laboratory, University College London, Darwin
      Building, Gower Street, London WC1E 6BT, UK.
FAU - Donoghue, Philip C J
AU  - Donoghue PC
FAU - Yang, Ziheng
AU  - Yang Z
LA  - eng
GR  - BB/G006660/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20091125
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - *Fossils
MH  - *Genetic Speciation
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Software
MH  - Time Factors
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp078 [pii]
AID - 10.1093/sysbio/syp078 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jan;59(1):74-89. doi: 10.1093/sysbio/syp078. Epub 2009 Nov 25.

PMID- 20525620
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 1
DP  - 2010 Jan
TI  - New heuristic methods for joint species delimitation and species tree inference.
PG  - 59-73
LID - 10.1093/sysbio/syp077 [doi]
AB  - Species delimitation and species tree inference are difficult problems in cases
      of recent divergence, especially when different loci have different histories.
      This paper quantifies the difficulty of jointly finding the division of samples
      to species and estimating a species tree without constraining the possible
      assignments a priori. It introduces a parametric and a nonparametric method,
      including new heuristic search strategies, to do this delimitation and tree
      inference using individual gene trees as input. The new methods were evaluated
      using thousands of simulations and 4 empirical data sets. These analyses suggest 
      that the new methods, especially the nonparametric one, may provide useful
      insights for systematists working at the species level with molecular data.
      However, they still often return incorrect results.
FAU - O'Meara, Brian C
AU  - O'Meara BC
AD  - Department of Ecology &amp; EvolutionaryBiology, University of Tennessee, 569 Dabney 
      Hall, Knoxville, TN 37996-1610, USA. bomeara@utk.edu
LA  - eng
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20091110
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Animals
MH  - Basidiomycota/classification/genetics
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Drosophila/classification/genetics
MH  - *Evolution, Molecular
MH  - *Genetic Speciation
MH  - Grasshoppers/classification/genetics
MH  - Passeriformes/classification/genetics
MH  - *Phylogeny
MH  - Species Specificity
PMC - PMC5841455
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp077 [pii]
AID - 10.1093/sysbio/syp077 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jan;59(1):59-73. doi: 10.1093/sysbio/syp077. Epub 2009 Nov 10.

PMID- 20525619
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20100607
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 1
DP  - 2010 Jan
TI  - Sparse supermatrices for phylogenetic inference: taxonomy, alignment, rogue taxa,
      and the phylogeny of living turtles.
PG  - 42-58
LID - 10.1093/sysbio/syp075 [doi]
AB  - As phylogenetic data sets grow in size and number, objective methods to summarize
      this information are becoming increasingly important. Supermatrices can combine
      existing data directly and in principle provide effective syntheses of
      phylogenetic information that may reveal new relationships. However, several
      serious difficulties exist in the construction of large supermatrices that must
      be overcome before these approaches will enjoy broad utility. We present analyses
      that examine the performance of sparse supermatrices constructed from large
      sequence databases for the reconstruction of species-level phylogenies. We
      develop a largely automated informatics pipeline that allows for the construction
      of sparse supermatrices from GenBank data. In doing so, we develop strategies for
      alleviating some of the outstanding impediments to accurate phylogenetic
      inference using these approaches. These include taxonomic standardization,
      automated alignment, and the identification of rogue taxa. We use turtles as an
      exemplar clade and present a well-supported species-level phylogeny for
      two-thirds of all turtle species based on a approximately 50 kb supermatrix
      consisting of 93% missing data. Finally, we discuss some of the remaining
      pitfalls and concerns associated with supermatrix analyses, provide comparisons
      to supertree approaches, and suggest areas for future research.
FAU - Thomson, Robert C
AU  - Thomson RC
AD  - Department of Evolution and Ecology and Center for Population Biology, University
      of California, Davis, CA 95616, USA. rcthomson@ucdavis.edu
FAU - Shaffer, H Bradley
AU  - Shaffer HB
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20091111
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - *Data Collection
MH  - *Phylogeny
MH  - Sequence Alignment/*methods
MH  - Turtles/*classification/*genetics
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp075 [pii]
AID - 10.1093/sysbio/syp075 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jan;59(1):42-58. doi: 10.1093/sysbio/syp075. Epub 2009 Nov 11.

PMID- 20525618
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 1
DP  - 2010 Jan
TI  - Unifying vertical and nonvertical evolution: a stochastic ARG-based framework.
PG  - 27-41
LID - 10.1093/sysbio/syp076 [doi]
AB  - Evolutionary biologists have introduced numerous statistical approaches to
      explore nonvertical evolution, such as horizontal gene transfer, recombination,
      and genomic reassortment, through collections of Markov-dependent gene trees.
      These tree collections allow for inference of nonvertical evolution, but only
      indirectly, making findings difficult to interpret and models difficult to
      generalize. An alternative approach to explore nonvertical evolution relies on
      phylogenetic networks. These networks provide a framework to model nonvertical
      evolution but leave unanswered questions such as the statistical significance of 
      specific nonvertical events. In this paper, we begin to correct the shortcomings 
      of both approaches by introducing the "stochastic model for reassortment and
      transfer events" (SMARTIE) drawing upon ancestral recombination graphs (ARGs).
      ARGs are directed graphs that allow for formal probabilistic inference on
      vertical speciation events and nonvertical evolutionary events. We apply SMARTIE 
      to phylogenetic data. Because of this, we can typically infer a single most
      probable ARG, avoiding coarse population dynamic summary statistics. In addition,
      a focus on phylogenetic data suggests novel probability distributions on ARGs. To
      make inference with our model, we develop a reversible jump Markov chain Monte
      Carlo sampler to approximate the posterior distribution of SMARTIE. Using the
      BEAST phylogenetic software as a foundation, the sampler employs a parallel
      computing approach that allows for inference on large-scale data sets. To
      demonstrate SMARTIE, we explore 2 separate phylogenetic applications, one
      involving pathogenic Leptospirochete and the other Saccharomyces.
FAU - Bloomquist, Erik W
AU  - Bloomquist EW
AD  - Department of Biostatistics, UCLA School of Public Health, Los Angeles, CA 90095,
      USA.
FAU - Suchard, Marc A
AU  - Suchard MA
LA  - eng
GR  - AI07370/AI/NIAID NIH HHS/United States
GR  - GM086887/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
DEP - 20091109
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - Gene Transfer, Horizontal/*genetics
MH  - Leptospiraceae/genetics
MH  - Likelihood Functions
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Saccharomyces/genetics
PMC - PMC2909786
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp076 [pii]
AID - 10.1093/sysbio/syp076 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jan;59(1):27-41. doi: 10.1093/sysbio/syp076. Epub 2009 Nov 9.

PMID- 20525617
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20101118
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 1
DP  - 2010 Jan
TI  - Phylogenetic logistic regression for binary dependent variables.
PG  - 9-26
LID - 10.1093/sysbio/syp074 [doi]
AB  - We develop statistical methods for phylogenetic logistic regression in which the 
      dependent variable is binary (0 or 1) and values are nonindependent among
      species, with phylogenetically related species tending to have the same value of 
      the dependent variable. The methods are based on an evolutionary model of binary 
      traits in which trait values switch between 0 and 1 as species evolve up a
      phylogenetic tree. The more frequently the trait values switch (i.e., the higher 
      the rate of evolution), the more rapidly correlations between trait values for
      phylogenetically related species break down. Therefore, the statistical methods
      also give a way to estimate the phylogenetic signal of binary traits. More
      generally, the methods can be applied with continuous- and/or discrete-valued
      independent variables. Using simulations, we assess the statistical properties of
      the methods, including bias in the estimates of the logistic regression
      coefficients and the parameter that estimates the strength of phylogenetic signal
      in the dependent variable. These analyses show that, as with the case for
      continuous-valued dependent variables, phylogenetic logistic regression should be
      used rather than standard logistic regression when there is the possibility of
      phylogenetic correlations among species. Standard logistic regression does not
      properly account for the loss of information caused by resemblance of relatives
      and as a result is likely to give inflated type I error rates, incorrectly
      identifying regression parameters as statistically significantly different from
      zero when they are not.
FAU - Ives, Anthony R
AU  - Ives AR
AD  - Department of Zoology, University of Wisconsin-Madison, Madison, WI 53706, USA.
      arives@wisc.edu
FAU - Garland, Theodore Jr
AU  - Garland T Jr
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20091104
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Analysis of Variance
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Logistic Models
MH  - *Models, Theoretical
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp074 [pii]
AID - 10.1093/sysbio/syp074 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jan;59(1):9-26. doi: 10.1093/sysbio/syp074. Epub 2009 Nov 4.

PMID- 20525616
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 59
IP  - 1
DP  - 2010 Jan
TI  - Relaxed molecular clocks, the bias-variance trade-off, and the quality of
      phylogenetic inference.
PG  - 1-8
LID - 10.1093/sysbio/syp072 [doi]
AB  - Because a constant rate of DNA sequence evolution cannot be assumed to be
      ubiquitous, relaxed molecular clock inference models have proven useful when
      estimating rates and divergence dates. Furthermore, it has been recently
      suggested that using relaxed molecular clocks may provide superior accuracy and
      precision in phylogenetic inference compared with traditional time-free methods
      that do not incorporate a molecular clock. We perform a simulation study to
      determine if assuming a relaxed molecular clock does indeed improve the quality
      of phylogenetic inference. We analyze sequence data simulated under various rate 
      distributions using relaxed-clocks, strict-clocks, and time-free Bayesian
      phylogenetic inference models. Our results indicate that no difference exists in 
      the quality of phylogenetic inference between assuming a relaxed molecular clock 
      and making no assumption about the clock-likeness of sequence evolution. This
      pattern is likely due to the bias-variance trade-off inherent in this type of
      phylogenetic inference. We also compared the quality of inference between
      Bayesian and maximum likelihood time-free inference models and found them to be
      qualitatively similar.
FAU - Wertheim, Joel O
AU  - Wertheim JO
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA. wertheim@email.arizona.edu
FAU - Sanderson, Michael J
AU  - Sanderson MJ
FAU - Worobey, Michael
AU  - Worobey M
FAU - Bjork, Adam
AU  - Bjork A
LA  - eng
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
DEP - 20091013
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence/*genetics
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
PMC - PMC2909785
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp072 [pii]
AID - 10.1093/sysbio/syp072 [doi]
PST - ppublish
SO  - Syst Biol. 2010 Jan;59(1):1-8. doi: 10.1093/sysbio/syp072. Epub 2009 Oct 13.

PMID- 20525615
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 6
DP  - 2009 Dec
TI  - Testing species-level diversification hypotheses in Madagascar: the case of
      microendemic Brookesia leaf chameleons.
PG  - 641-56
LID - 10.1093/sysbio/syp073 [doi]
AB  - Madagascar's flora and fauna are remarkable both for their diversity and
      supraspecific endemism. Moreover, many taxa contain large numbers of species with
      limited distributions. Several hypotheses have been proposed to explain this high
      level of microendemism, including 1) riverine barrier, 2) mountain refuge, and 3)
      watershed contraction hypotheses, the latter 2 of which center on fragmentation
      due to climatic shifts associated with Pliocene/Pleistocene glaciations. The
      Malagasy leaf chameleon genus Brookesia is a speciose group with a high
      proportion of microendemic taxa, thus making it an excellent candidate to test
      these vicariance scenarios. We used mitochondrial and nuclear sequence data to
      construct a Brookesia phylogeny, and temporal concordance with
      Pliocene/Pleistocene speciation scenarios was tested by estimating divergence
      dates using a relaxed-clock Bayesian method. We strongly reject a role for
      Pliocene/Pleistocene climatic fluctuations in species-level diversification of
      Brookesia. We also used simulations to test the spatial predictions of the
      watershed contraction model in a phylogenetic context, independent of its
      temporal component, and found no statistical support for this model. The riverine
      barrier model is likewise a qualitatively poor fit to our data, but some
      relationships support a more ancient mountain refuge effect. We assessed support 
      for the 3 hypotheses in a nonphylogenetic context by examining altitude and
      species richness and found a significant positive correlation between these
      variables. This is consistent with a mountain refuge effect but does not support 
      the watershed contraction or riverine barrier models. Finally, we find repeated
      higher level east-west divergence patterns 1) between the 2 sister clades
      comprising the Brookesia minima group and 2) within the clade of larger leaf
      chameleons, which shows a basal divergence between western and eastern/northern
      sister clades. Our results highlight the central role of phylogeny in any
      meaningful tests of species-level diversification theories.
FAU - Townsend, Ted M
AU  - Townsend TM
AD  - Department of Biology, San Diego State University, 5500 Campanile Drive, San
      Diego, CA 92182-4164, USA. townsend@sciences.sdsu.edu
FAU - Vieites, David R
AU  - Vieites DR
FAU - Glaw, Frank
AU  - Glaw F
FAU - Vences, Miguel
AU  - Vences M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20091110
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Altitude
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - *Biodiversity
MH  - *Climate
MH  - Computer Simulation
MH  - DNA, Mitochondrial/genetics
MH  - Demography
MH  - Evolution, Molecular
MH  - *Genetic Speciation
MH  - *Geography
MH  - Likelihood Functions
MH  - Lizards/*genetics
MH  - Madagascar
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp073 [pii]
AID - 10.1093/sysbio/syp073 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Dec;58(6):641-56. doi: 10.1093/sysbio/syp073. Epub 2009 Nov 10.

PMID- 20525614
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 6
DP  - 2009 Dec
TI  - Heritability of extinction rates links diversification patterns in molecular
      phylogenies and fossils.
PG  - 629-40
LID - 10.1093/sysbio/syp069 [doi]
AB  - Time-calibrated molecular phylogenies provide a valuable window into the tempo
      and mode of species diversification, especially for the large number of groups
      that lack adequate fossil records. Molecular phylogenetic data frequently suggest
      an initial "explosive speciation" phase, leading to widespread speculation that
      ecological niche-filling processes might govern the dynamics of species
      diversification during evolutionary radiations. However, these patterns are
      difficult to reconcile with the fossil record. The fossil record strongly
      suggests that extinction rates have been high relative to speciation rates, but
      such elevated background extinction should erase the signal of early, rapid
      speciation from molecular phylogenies. For this reason, extinction rates in
      molecular phylogenies are frequently estimated as zero under the widely used
      birth-death model. Here, I construct a simple model that combines
      phylogenetically patterned extinction with pulsed turnover dynamics and constant 
      diversity through time. Using approximate Bayesian methods, I show that heritable
      extinction can easily explain the phenomenon of explosive early diversification, 
      even when net diversification rates are equal to zero. Several assumptions of the
      model are more consistent with both the fossil record and neontological data than
      the standard birth-death model and it may thus represent a viable alternative
      interpretation of phylogenetic diversification patterns. These results suggest
      that variation in the absolute rate of lineage turnover through time, in
      conjunction with phylogenetically nonrandom extinction, may underlie the apparent
      diversity-dependent speciation observed in molecular phylogenies.
FAU - Rabosky, Daniel L
AU  - Rabosky DL
AD  - Department of Ecology and Evolutionary Biology, Cornell University, Ithaca, NY
      14853, USA. drabosky@berkeley.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20091005
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Biological/genetics
MH  - Bayes Theorem
MH  - Biodiversity
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - *Extinction, Biological
MH  - *Fossils
MH  - *Genetic Speciation
MH  - *Models, Theoretical
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp069 [pii]
AID - 10.1093/sysbio/syp069 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Dec;58(6):629-40. doi: 10.1093/sysbio/syp069. Epub 2009 Oct 5.

PMID- 20525613
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 6
DP  - 2009 Dec
TI  - Reticulation, data combination, and inferring evolutionary history: an example
      from Danthonioideae (Poaceae).
PG  - 612-28
LID - 10.1093/sysbio/syp068 [doi]
AB  - We explore the potential impact of conflicting gene trees on inferences of
      evolutionary history above the species level. When conflict between gene trees is
      discovered, it is common practice either to analyze the data separately or to
      combine the data having excluded the conflicting taxa or data partitions for
      those taxa (which are then recoded as missing). We demonstrate an alternative
      approach, which involves duplicating conflicting taxa in the matrix, such that
      each duplicate is represented by one partition only. This allows the combination 
      of all available data in standard phylogenetic analyses, despite reticulations.
      We show how interpretation of contradictory gene trees can lead to conflicting
      inferences of both morphological evolution and biogeographic history, using the
      example of the pampas grasses, Cortaderia. The characteristic morphological
      syndrome of Cortaderia can be inferred as having arisen multiple times
      (chloroplast DNA [cpDNA]) or just once (nuclear ribosomal DNA [nrDNA]). The
      distributions of species of Cortaderia and related genera in Australia/New
      Guinea, New Zealand, and South America can be explained by few (nrDNA) or several
      (cpDNA) dispersals between the southern continents. These contradictions can be
      explained by past hybridization events, which have linked gains of complex
      morphologies with unrelated chloroplast lineages and have erased evidence of
      dispersals from the nuclear genome. Given the discrepancies between inferences
      based on the gene trees individually, we urge the use of approaches such as ours 
      that take multiple gene trees into account.
FAU - Pirie, Michael D
AU  - Pirie MD
AD  - Institute for Systematic Botany, University of Zurich, CH-8008 Zurich,
      Switzerland. mpirie@sun.ac.za
FAU - Humphreys, Aelys M
AU  - Humphreys AM
FAU - Barker, Nigel P
AU  - Barker NP
FAU - Linder, H Peter
AU  - Linder HP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20091021
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
SB  - IM
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA, Chloroplast/genetics
MH  - Demography
MH  - *Evolution, Molecular
MH  - Genes, Plant/*genetics
MH  - Geography
MH  - *Hybridization, Genetic
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Poaceae/*genetics
MH  - *Research Design
MH  - Sequence Analysis, DNA
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp068 [pii]
AID - 10.1093/sysbio/syp068 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Dec;58(6):612-28. doi: 10.1093/sysbio/syp068. Epub 2009 Oct 21.

PMID- 20525612
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 6
DP  - 2009 Dec
TI  - Estimating trait-dependent speciation and extinction rates from incompletely
      resolved phylogenies.
PG  - 595-611
LID - 10.1093/sysbio/syp067 [doi]
AB  - Species traits may influence rates of speciation and extinction, affecting both
      the patterns of diversification among lineages and the distribution of traits
      among species. Existing likelihood approaches for detecting differential
      diversification require complete phylogenies; that is, every extant species must 
      be present in a well-resolved phylogeny. We developed 2 likelihood methods that
      can be used to infer the effect of a trait on speciation and extinction without
      complete phylogenetic information, generalizing the recent binary-state
      speciation and extinction method. Our approaches can be used where a phylogeny
      can be reasonably assumed to be a random sample of extant species or where all
      extant species are included but some are assigned only to terminal unresolved
      clades. We explored the effects of decreasing phylogenetic resolution on the
      ability of our approach to detect differential diversification within a Bayesian 
      framework using simulated phylogenies. Differential diversification caused by an 
      asymmetry in speciation rates was nearly as well detected with only 50% of extant
      species phylogenetically resolved as with complete phylogenetic knowledge. We
      demonstrate our unresolved clade method with an analysis of sexual dimorphism and
      diversification in shorebirds (Charadriiformes). Our methods allow for the direct
      estimation of the effect of a trait on speciation and extinction rates using
      incompletely resolved phylogenies.
FAU - FitzJohn, Richard G
AU  - FitzJohn RG
AD  - Department of Zoology, University of British Columbia, Vancouver, BC, V6T 1Z4
      Canada. fitzjohn@zoology.ubc.ca
FAU - Maddison, Wayne P
AU  - Maddison WP
FAU - Otto, Sarah P
AU  - Otto SP
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20091015
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Charadriiformes/*anatomy &amp; histology/genetics
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - *Extinction, Biological
MH  - Female
MH  - *Genetic Speciation
MH  - Likelihood Functions
MH  - Male
MH  - Models, Theoretical
MH  - *Phylogeny
MH  - *Sex Characteristics
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp067 [pii]
AID - 10.1093/sysbio/syp067 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Dec;58(6):595-611. doi: 10.1093/sysbio/syp067. Epub 2009 Oct 15.

PMID- 20525611
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 6
DP  - 2009 Dec
TI  - Taxon Selection under Split Diversity.
PG  - 586-94
LID - 10.1093/sysbio/syp058 [doi]
AB  - The "phylogenetic diversity" (PD) measure of biodiversity is evaluated using a
      phylogenetic tree, usually inferred from morphological or molecular data.
      Consequently, it is vulnerable to errors in that tree, including those resulting 
      from sampling error, model misspecification, or conflicting signals. To improve
      the robustness of PD, we can evaluate the measure using either a collection (or
      distribution) of trees or a phylogenetic network. Recently, it has been shown
      that these 2 approaches are equivalent but that the problem of maximizing PD in
      the general concept is NP-hard. In this study, we provide an efficient dynamic
      programming algorithm for maximizing PD when splits in the trees or network form 
      a circular split system. We illustrate our method using a case study of game
      birds ("Galliformes") and discuss the different choices of taxa based on our
      approach and PD.
FAU - Minh, Bui Quang
AU  - Minh BQ
AD  - Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories,
      Dr.-Bohr-Gasse 9/6, 1030 Vienna, Austria.
FAU - Klaere, Steffen
AU  - Klaere S
FAU - von Haeseler, Arndt
AU  - von Haeseler A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090921
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Animals
MH  - *Biodiversity
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - Conservation of Natural Resources/methods
MH  - Galliformes/genetics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Research Design
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp058 [pii]
AID - 10.1093/sysbio/syp058 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Dec;58(6):586-94. doi: 10.1093/sysbio/syp058. Epub 2009 Sep 21.

PMID- 20525610
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20190124
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 6
DP  - 2009 Dec
TI  - Radiation of extant cetaceans driven by restructuring of the oceans.
PG  - 573-85
LID - 10.1093/sysbio/syp060 [doi]
AB  - The remarkable fossil record of whales and dolphins (Cetacea) has made them an
      exemplar of macroevolution. Although their overall adaptive transition from
      terrestrial to fully aquatic organisms is well known, this is not true for the
      radiation of modern whales. Here, we explore the diversification of extant
      cetaceans by constructing a robust molecular phylogeny that includes 87 of 89
      extant species. The phylogeny and divergence times are derived from nuclear and
      mitochondrial markers, calibrated with fossils. We find that the toothed whales
      are monophyletic, suggesting that echolocation evolved only once early in that
      lineage some 36-34 Ma. The rorqual family (Balaenopteridae) is restored with the 
      exclusion of the gray whale, suggesting that gulp feeding evolved 18-16 Ma.
      Delphinida, comprising all living dolphins and porpoises other than the
      Ganges/Indus dolphins, originated about 26 Ma; it contains the taxonomically rich
      delphinids, which began diversifying less than 11 Ma. We tested 2 hypothesized
      drivers of the extant cetacean radiation by assessing the tempo of lineage
      accumulation through time. We find no support for a rapid burst of speciation
      early in the history of extant whales, contrasting with expectations of an
      adaptive radiation model. However, we do find support for increased
      diversification rates during periods of pronounced physical restructuring of the 
      oceans. The results imply that paleogeographic and paleoceanographic changes,
      such as closure of major seaways, have influenced the dynamics of radiation in
      extant cetaceans.
FAU - Steeman, Mette E
AU  - Steeman ME
AD  - Centre for GeoGenetics, Natural History Museum of Denmark, 2100 Copenhagen,
      Denmark.
FAU - Hebsgaard, Martin B
AU  - Hebsgaard MB
FAU - Fordyce, R Ewan
AU  - Fordyce RE
FAU - Ho, Simon Y W
AU  - Ho SY
FAU - Rabosky, Daniel L
AU  - Rabosky DL
FAU - Nielsen, Rasmus
AU  - Nielsen R
FAU - Rahbek, Carsten
AU  - Rahbek C
FAU - Glenner, Henrik
AU  - Glenner H
FAU - Sorensen, Martin V
AU  - Sorensen MV
FAU - Willerslev, Eske
AU  - Willerslev E
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20091005
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Cetacea/*genetics/physiology
MH  - Computational Biology
MH  - Echolocation/physiology
MH  - *Environment
MH  - *Evolution, Molecular
MH  - Feeding Behavior/physiology
MH  - *Fossils
MH  - *Genetic Speciation
MH  - Likelihood Functions
MH  - Oceans and Seas
MH  - *Phylogeny
MH  - Sequence Alignment
PMC - PMC2777972
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp060 [pii]
AID - 10.1093/sysbio/syp060 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Dec;58(6):573-85. doi: 10.1093/sysbio/syp060. Epub 2009 Oct 5.

PMID- 20525609
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 6
DP  - 2009 Dec
TI  - The use and validity of composite taxa in phylogenetic analysis.
PG  - 560-72
LID - 10.1093/sysbio/syp056 [doi]
AB  - In phylogenetic analysis, one possible approach to minimize missing data in DNA
      supermatrices consists in sampling sequences from different species to obtain a
      complete sequence for all genes included in the study. We refer to those complete
      sequences as composite taxa because DNA sequences that are combined belong to
      different species. An alternative approach is to analyze incomplete supermatrices
      by coding unavailable DNA sequences as missing. The accuracy of phylogenetic
      trees estimated using matrices that include composite taxa has recently been
      questioned, and the best approach for analyzing incomplete supermatrices is
      highly debated. Through computer simulations, we compared the phylogenetic
      accuracy of the 2 competing approaches. We explored the effect of composite taxa 
      when inferring higher level relationships, that is, relationships between
      monophyletic groups. DNA sequences were simulated on a 42-taxon model tree and
      incomplete supermatrices containing different percentages of missing data were
      generated. These incomplete supermatrices were analyzed either by coding the
      missing data with "?" or by reducing the amount of missing data through the
      combination of 2 or more taxa to generate composite taxa. Of 180 comparisons (18 
      simulation cases with 2 different inference methods and 5 levels of
      incompleteness), we observed significantly higher phylogenetic accuracies for
      composite matrices in 46 comparisons, whereas missing data matrices outperformed 
      composites in 8 comparisons. In all other cases, the phylogenetic accuracy
      obtained with composite matrices was not significantly different from that of
      missing data matrices. This study demonstrates that composite taxa represent an
      interesting approach to minimize the amount of missing data in supermatrices and 
      we suggest that it is the optimal approach to use in phylogenomic studies to
      reduce computing time.
FAU - Campbell, Veronique
AU  - Campbell V
AD  - Departement de sciences biologiques, Universite de Montreal, C.P. 6128, Succ.
      Centre-ville, Montreal, Quebec H3C 3J7, Canada. veronique.campbell@umontreal.ca
FAU - Lapointe, Francois-Joseph
AU  - Lapointe FJ
LA  - eng
PT  - Comparative Study
PT  - Evaluation Studies
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090921
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - Classification/*methods
MH  - Cluster Analysis
MH  - Computer Simulation
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
MH  - *Research Design
MH  - Sequence Analysis, DNA/methods
MH  - Species Specificity
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp056 [pii]
AID - 10.1093/sysbio/syp056 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Dec;58(6):560-72. doi: 10.1093/sysbio/syp056. Epub 2009 Sep 21.

PMID- 20525608
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 6
DP  - 2009 Dec
TI  - Species tree discordance traces to phylogeographic clade boundaries in North
      American fence lizards (Sceloporus).
PG  - 547-59
LID - 10.1093/sysbio/syp057 [doi]
AB  - I investigated the impacts of phylogeographic sampling decisions on species tree 
      estimation in the Sceloporus undulatus species group, a recent radiation of
      small, insectivorous lizards connected by parapatric and peripatric distribution 
      across North America, using a variety of species tree inference methods (Bayesian
      estimation of species trees, Bayesian untangling of concordance knots, and
      minimize deep coalescences). Phylogenetic analyses of 16 specimens representing 4
      putative species within S. "undulatus" using complete (8 loci, &gt;5.5 kb) and
      incomplete (29 loci, &gt;23.6 kb) nuclear data sets result in species trees that
      share features with the mitochondrial DNA (mtDNA) genealogy at the
      phylogeographic level but provide new insights into the evolutionary history of
      the species group. The concatenated nuclear data and mtDNA data both recover 4
      major clades connecting populations across North America; however, instances of
      discordance are localized at the contact zones between adjacent phylogeographic
      groups. A random sub-sampling experiment designed to vary the phylogeographic
      samples included across hundreds of replicate species tree inferences suggests
      that inaccurate species assignments can result in inferred phylogenetic
      relationships that are dependent upon which particular populations are used as
      exemplars to represent species and can lead to increased estimates of effective
      population size. For the phylogeographic data presented here, reassigning
      specimens with introgressed mtDNA genomes to their prospective species, or
      excluding them from the analysis altogether, produces species tree topologies
      that are distinctly different from analyses that utilize mtDNA-based species
      assignments. Evolutionary biologists working at the interface of phylogeography
      and phylogenetics are likely to encounter multiple processes influencing gene
      trees congruence, which increases the relevance of estimating species trees with 
      multilocus nuclear data and models that accommodate deep coalescence.
FAU - Leache, Adam D
AU  - Leache AD
AD  - Museum of Vertebrate Zoology, 3101 Valley Life Sciences Building, University of
      California, Berkeley, CA 94720, USA. aleache@ucdavis.edu
LA  - eng
SI  - GENBANK/GQ494358
SI  - GENBANK/GQ494359
SI  - GENBANK/GQ494360
SI  - GENBANK/GQ494361
SI  - GENBANK/GQ494362
SI  - GENBANK/GQ494363
SI  - GENBANK/GQ494364
SI  - GENBANK/GQ494365
SI  - GENBANK/GQ494366
SI  - GENBANK/GQ494367
SI  - GENBANK/GQ494368
SI  - GENBANK/GQ494369
SI  - GENBANK/GQ494370
SI  - GENBANK/GQ494371
SI  - GENBANK/GQ494372
SI  - GENBANK/GQ494373
SI  - GENBANK/GQ494374
SI  - GENBANK/GQ494375
SI  - GENBANK/GQ494376
SI  - GENBANK/GQ494377
SI  - GENBANK/GQ494378
SI  - GENBANK/GQ494379
SI  - GENBANK/GQ494380
SI  - GENBANK/GQ494381
SI  - GENBANK/GQ494382
SI  - GENBANK/GQ494383
SI  - GENBANK/GQ494384
SI  - GENBANK/GQ494385
SI  - GENBANK/GQ494386
SI  - GENBANK/GQ494387
SI  - GENBANK/GQ494388
SI  - GENBANK/GQ494389
SI  - GENBANK/GQ494390
SI  - GENBANK/GQ494391
SI  - GENBANK/GQ494392
SI  - GENBANK/GQ494393
SI  - GENBANK/GQ494394
SI  - GENBANK/GQ494395
SI  - GENBANK/GQ494396
SI  - GENBANK/GQ494397
SI  - GENBANK/GQ494398
SI  - GENBANK/GQ494399
SI  - GENBANK/GQ494400
SI  - GENBANK/GQ494401
SI  - GENBANK/GQ494402
SI  - GENBANK/GQ494403
SI  - GENBANK/GQ494404
SI  - GENBANK/GQ494405
SI  - GENBANK/GQ494406
SI  - GENBANK/GQ494407
SI  - GENBANK/GQ494408
SI  - GENBANK/GQ494409
SI  - GENBANK/GQ494410
SI  - GENBANK/GQ494411
SI  - GENBANK/GQ494412
SI  - GENBANK/GQ494413
SI  - GENBANK/GQ494414
SI  - GENBANK/GQ494415
SI  - GENBANK/GQ494416
SI  - GENBANK/GQ494417
SI  - GENBANK/GQ494418
SI  - GENBANK/GQ494419
SI  - GENBANK/GQ494420
SI  - GENBANK/GQ494421
SI  - GENBANK/GQ494422
SI  - GENBANK/GQ494423
SI  - GENBANK/GQ494424
SI  - GENBANK/GQ494425
SI  - GENBANK/GQ494426
SI  - GENBANK/GQ494427
SI  - GENBANK/GQ494428
SI  - GENBANK/GQ494429
SI  - GENBANK/GQ494430
SI  - GENBANK/GQ494431
SI  - GENBANK/GQ494432
SI  - GENBANK/GQ494433
SI  - GENBANK/GQ494434
SI  - GENBANK/GQ494435
SI  - GENBANK/GQ494436
SI  - GENBANK/GQ494437
SI  - GENBANK/GQ494438
SI  - GENBANK/GQ494439
SI  - GENBANK/GQ494440
SI  - GENBANK/GQ494441
SI  - GENBANK/GQ494442
SI  - GENBANK/GQ494443
SI  - GENBANK/GQ494444
SI  - GENBANK/GQ494445
SI  - GENBANK/GQ494446
SI  - GENBANK/GQ494447
SI  - GENBANK/GQ494448
SI  - GENBANK/GQ494449
SI  - GENBANK/GQ494450
SI  - GENBANK/GQ494451
SI  - GENBANK/GQ494452
SI  - GENBANK/GQ494453
SI  - GENBANK/GQ494454
SI  - GENBANK/GQ494455
SI  - GENBANK/GQ494456
SI  - GENBANK/GQ494457
SI  - GENBANK/GQ494458
SI  - GENBANK/GQ494459
SI  - GENBANK/GQ494460
SI  - GENBANK/GQ494461
SI  - GENBANK/GQ494462
SI  - GENBANK/GQ494463
SI  - GENBANK/GQ494464
SI  - GENBANK/GQ494465
SI  - GENBANK/GQ494466
SI  - GENBANK/GQ494467
SI  - GENBANK/GQ494468
SI  - GENBANK/GQ494469
SI  - GENBANK/GQ494470
SI  - GENBANK/GQ494471
SI  - GENBANK/GQ494472
SI  - GENBANK/GQ494473
SI  - GENBANK/GQ494474
SI  - GENBANK/GQ494475
SI  - GENBANK/GQ494476
SI  - GENBANK/GQ494477
SI  - GENBANK/GQ494478
SI  - GENBANK/GQ494479
SI  - GENBANK/GQ494480
SI  - GENBANK/GQ494481
SI  - GENBANK/GQ494482
SI  - GENBANK/GQ494483
SI  - GENBANK/GQ494484
SI  - GENBANK/GQ494485
SI  - GENBANK/GQ494486
SI  - GENBANK/GQ494487
SI  - GENBANK/GQ494488
SI  - GENBANK/GQ494489
SI  - GENBANK/GQ494490
SI  - GENBANK/GQ494491
SI  - GENBANK/GQ494492
SI  - GENBANK/GQ494493
SI  - GENBANK/GQ494494
SI  - GENBANK/GQ494495
SI  - GENBANK/GQ494496
SI  - GENBANK/GQ494497
SI  - GENBANK/GQ494498
SI  - GENBANK/GQ494499
SI  - GENBANK/GQ494500
SI  - GENBANK/GQ494501
SI  - GENBANK/GQ494502
SI  - GENBANK/GQ494503
SI  - GENBANK/GQ494504
SI  - GENBANK/GQ494505
SI  - GENBANK/GQ494506
SI  - GENBANK/GQ494507
SI  - GENBANK/GQ494508
SI  - GENBANK/GQ494509
SI  - GENBANK/GQ494510
SI  - GENBANK/GQ494511
SI  - GENBANK/GQ494512
SI  - GENBANK/GQ494513
SI  - GENBANK/GQ494514
SI  - GENBANK/GQ494515
SI  - GENBANK/GQ494516
SI  - GENBANK/GQ494517
SI  - GENBANK/GQ494518
SI  - GENBANK/GQ494519
SI  - GENBANK/GQ494520
SI  - GENBANK/GQ494521
SI  - GENBANK/GQ494522
SI  - GENBANK/GQ494523
SI  - GENBANK/GQ494524
SI  - GENBANK/GQ494525
SI  - GENBANK/GQ494526
SI  - GENBANK/GQ494527
SI  - GENBANK/GQ494528
SI  - GENBANK/GQ494529
SI  - GENBANK/GQ494530
SI  - GENBANK/GQ494531
SI  - GENBANK/GQ494532
SI  - GENBANK/GQ494533
SI  - GENBANK/GQ494534
SI  - GENBANK/GQ494535
SI  - GENBANK/GQ494536
SI  - GENBANK/GQ494537
SI  - GENBANK/GQ494538
SI  - GENBANK/GQ494539
SI  - GENBANK/GQ494540
SI  - GENBANK/GQ494541
SI  - GENBANK/GQ494542
SI  - GENBANK/GQ494543
SI  - GENBANK/GQ494544
SI  - GENBANK/GQ494545
SI  - GENBANK/GQ494546
SI  - GENBANK/GQ494547
SI  - GENBANK/GQ494548
SI  - GENBANK/GQ494549
SI  - GENBANK/GQ494550
SI  - GENBANK/GQ494551
SI  - GENBANK/GQ494552
SI  - GENBANK/GQ494553
SI  - GENBANK/GQ494554
SI  - GENBANK/GQ494555
SI  - GENBANK/GQ494556
SI  - GENBANK/GQ494557
SI  - GENBANK/GQ494558
SI  - GENBANK/GQ494559
SI  - GENBANK/GQ494560
SI  - GENBANK/GQ494561
SI  - GENBANK/GQ494562
SI  - GENBANK/GQ494563
SI  - GENBANK/GQ494564
SI  - GENBANK/GQ494565
SI  - GENBANK/GQ494566
SI  - GENBANK/GQ494567
SI  - GENBANK/GQ494568
SI  - GENBANK/GQ494569
SI  - GENBANK/GQ494570
SI  - GENBANK/GQ494571
SI  - GENBANK/GQ494572
SI  - GENBANK/GQ494573
SI  - GENBANK/GQ494574
SI  - GENBANK/GQ494575
SI  - GENBANK/GQ494576
SI  - GENBANK/GQ494577
SI  - GENBANK/GQ494578
SI  - GENBANK/GQ494579
SI  - GENBANK/GQ494580
SI  - GENBANK/GQ494581
SI  - GENBANK/GQ494582
SI  - GENBANK/GQ494583
SI  - GENBANK/GQ494584
SI  - GENBANK/GQ494585
SI  - GENBANK/GQ494586
SI  - GENBANK/GQ494587
SI  - GENBANK/GQ494588
SI  - GENBANK/GQ494589
SI  - GENBANK/GQ494590
SI  - GENBANK/GQ494591
SI  - GENBANK/GQ494592
SI  - GENBANK/GQ494593
SI  - GENBANK/GQ494594
SI  - GENBANK/GQ494595
SI  - GENBANK/GQ494596
SI  - GENBANK/GQ494597
SI  - GENBANK/GQ494598
SI  - GENBANK/GQ494599
SI  - GENBANK/GQ494600
SI  - GENBANK/GQ494601
SI  - GENBANK/GQ494602
SI  - GENBANK/GQ494603
SI  - GENBANK/GQ494604
SI  - GENBANK/GQ494605
SI  - GENBANK/GQ494606
SI  - GENBANK/GQ494607
SI  - GENBANK/GQ494608
SI  - GENBANK/GQ494609
SI  - GENBANK/GQ494610
SI  - GENBANK/GQ494611
SI  - GENBANK/GQ494612
SI  - GENBANK/GQ494613
SI  - GENBANK/GQ494614
SI  - GENBANK/GQ494615
SI  - GENBANK/GQ494616
SI  - GENBANK/GQ494617
SI  - GENBANK/GQ494618
SI  - GENBANK/GQ494619
SI  - GENBANK/GQ494620
SI  - GENBANK/GQ494621
SI  - GENBANK/GQ494622
SI  - GENBANK/GQ494623
SI  - GENBANK/GQ494624
SI  - GENBANK/GQ494625
SI  - GENBANK/GQ494626
SI  - GENBANK/GQ494627
SI  - GENBANK/GQ494628
SI  - GENBANK/GQ494629
SI  - GENBANK/GQ494630
SI  - GENBANK/GQ494631
SI  - GENBANK/GQ494632
SI  - GENBANK/GQ494633
SI  - GENBANK/GQ494634
SI  - GENBANK/GQ494635
SI  - GENBANK/GQ494636
SI  - GENBANK/GQ494637
SI  - GENBANK/GQ494638
SI  - GENBANK/GQ494639
SI  - GENBANK/GQ494640
SI  - GENBANK/GQ494641
SI  - GENBANK/GQ494642
SI  - GENBANK/GQ494643
SI  - GENBANK/GQ494644
SI  - GENBANK/GQ494645
SI  - GENBANK/GQ494646
SI  - GENBANK/GQ494647
SI  - GENBANK/GQ494648
SI  - GENBANK/GQ494649
SI  - GENBANK/GQ494650
SI  - GENBANK/GQ494651
SI  - GENBANK/GQ494652
SI  - GENBANK/GQ494653
SI  - GENBANK/GQ494654
SI  - GENBANK/GQ494655
SI  - GENBANK/GQ494656
SI  - GENBANK/GQ494657
SI  - GENBANK/GQ494658
SI  - GENBANK/GQ494659
SI  - GENBANK/GQ494660
SI  - GENBANK/GQ494661
SI  - GENBANK/GQ494662
SI  - GENBANK/GQ494663
SI  - GENBANK/GQ494664
SI  - GENBANK/GQ494665
SI  - GENBANK/GQ494666
SI  - GENBANK/GQ494667
SI  - GENBANK/GQ494668
SI  - GENBANK/GQ494669
SI  - GENBANK/GQ494670
SI  - GENBANK/GQ494671
SI  - GENBANK/GQ494672
SI  - GENBANK/GQ494673
SI  - GENBANK/GQ494674
SI  - GENBANK/GQ494675
SI  - GENBANK/GQ494676
SI  - GENBANK/GQ494677
SI  - GENBANK/GQ494678
SI  - GENBANK/GQ494679
SI  - GENBANK/GQ494680
SI  - GENBANK/GQ494681
SI  - GENBANK/GQ494682
SI  - GENBANK/GQ494683
SI  - GENBANK/GQ494684
SI  - GENBANK/GQ494685
SI  - GENBANK/GQ494686
SI  - GENBANK/GQ494687
SI  - GENBANK/GQ494688
SI  - GENBANK/GQ494689
SI  - GENBANK/GQ494690
SI  - GENBANK/GQ494691
SI  - GENBANK/GQ494692
SI  - GENBANK/GQ494693
SI  - GENBANK/GQ494694
SI  - GENBANK/GQ494695
SI  - GENBANK/GQ494696
SI  - GENBANK/GQ494697
SI  - GENBANK/GQ494698
SI  - GENBANK/GQ494699
SI  - GENBANK/GQ494700
SI  - GENBANK/GQ494701
SI  - GENBANK/GQ494702
SI  - GENBANK/GQ494703
SI  - GENBANK/GQ494704
SI  - GENBANK/GQ494705
SI  - GENBANK/GQ494706
SI  - GENBANK/GQ494707
SI  - GENBANK/GQ494708
SI  - GENBANK/GQ494709
SI  - GENBANK/GQ494710
SI  - GENBANK/GQ494711
SI  - GENBANK/GQ494712
SI  - GENBANK/GQ494713
SI  - GENBANK/GQ494714
SI  - GENBANK/GQ494715
SI  - GENBANK/GQ494716
SI  - GENBANK/GQ494717
SI  - GENBANK/GQ494718
SI  - GENBANK/GQ494719
SI  - GENBANK/GQ494720
SI  - GENBANK/GQ494721
SI  - GENBANK/GQ494722
SI  - GENBANK/GQ494723
SI  - GENBANK/GQ494724
SI  - GENBANK/GQ494725
SI  - GENBANK/GQ494726
SI  - GENBANK/GQ494727
SI  - GENBANK/GQ494728
SI  - GENBANK/GQ494729
SI  - GENBANK/GQ494730
SI  - GENBANK/GQ494731
SI  - GENBANK/GQ494732
SI  - GENBANK/GQ494733
SI  - GENBANK/GQ494734
SI  - GENBANK/GQ494735
SI  - GENBANK/GQ494736
SI  - GENBANK/GQ494737
SI  - GENBANK/GQ494738
SI  - GENBANK/GQ494739
SI  - GENBANK/GQ494740
SI  - GENBANK/GQ494741
SI  - GENBANK/GQ494742
SI  - GENBANK/GQ494743
SI  - GENBANK/GQ494744
SI  - GENBANK/GQ494745
SI  - GENBANK/GQ494746
SI  - GENBANK/GQ494747
SI  - GENBANK/GQ494748
SI  - GENBANK/GQ494749
SI  - GENBANK/GQ494750
SI  - GENBANK/GQ494751
SI  - GENBANK/GQ494752
SI  - GENBANK/GQ494753
SI  - GENBANK/GQ494754
SI  - GENBANK/GQ494755
SI  - GENBANK/GQ494756
SI  - GENBANK/GQ494757
SI  - GENBANK/GQ494758
SI  - GENBANK/GQ494759
SI  - GENBANK/GQ494760
SI  - GENBANK/GQ494761
SI  - GENBANK/GQ494762
SI  - GENBANK/GQ494763
SI  - GENBANK/GQ494764
SI  - GENBANK/GQ494765
SI  - GENBANK/GQ494766
SI  - GENBANK/GQ494767
SI  - GENBANK/GQ494768
SI  - GENBANK/GQ494769
SI  - GENBANK/GQ494770
SI  - GENBANK/GQ494771
SI  - GENBANK/GQ494772
SI  - GENBANK/GQ494773
SI  - GENBANK/GQ494774
SI  - GENBANK/GQ494775
SI  - GENBANK/GQ494776
SI  - GENBANK/GQ494777
SI  - GENBANK/GQ494778
SI  - GENBANK/GQ494779
SI  - GENBANK/GQ494780
SI  - GENBANK/GQ494781
SI  - GENBANK/GQ494782
SI  - GENBANK/GQ494783
SI  - GENBANK/GQ494784
SI  - GENBANK/GQ494785
SI  - GENBANK/GQ494786
SI  - GENBANK/GQ494787
SI  - GENBANK/GQ494788
SI  - GENBANK/GQ494789
SI  - GENBANK/GQ494790
SI  - GENBANK/GQ494791
SI  - GENBANK/GQ494792
SI  - GENBANK/GQ494793
SI  - GENBANK/GQ494794
SI  - GENBANK/GQ494795
SI  - GENBANK/GQ494796
SI  - GENBANK/GQ494797
SI  - GENBANK/GQ494798
SI  - GENBANK/GQ494799
SI  - GENBANK/GQ494800
SI  - GENBANK/GQ494801
SI  - GENBANK/GQ494802
SI  - GENBANK/GQ494803
SI  - GENBANK/GQ494804
SI  - GENBANK/GQ494805
SI  - GENBANK/GQ494806
SI  - GENBANK/GQ494807
SI  - GENBANK/GQ494808
SI  - GENBANK/GQ494809
SI  - GENBANK/GQ494810
SI  - GENBANK/GQ494811
SI  - GENBANK/GQ494812
SI  - GENBANK/GQ494813
SI  - GENBANK/GQ494814
SI  - GENBANK/GQ494815
SI  - GENBANK/GQ494816
SI  - GENBANK/GQ494817
SI  - GENBANK/GQ494818
SI  - GENBANK/GQ494819
SI  - GENBANK/GQ494820
SI  - GENBANK/GQ494821
SI  - GENBANK/GQ494822
SI  - GENBANK/GQ494823
SI  - GENBANK/GQ494824
SI  - GENBANK/GQ494825
SI  - GENBANK/GQ494826
SI  - GENBANK/GQ494827
SI  - GENBANK/GQ494828
SI  - GENBANK/GQ494829
SI  - GENBANK/GQ494830
SI  - GENBANK/GQ494831
SI  - GENBANK/GQ494832
SI  - GENBANK/GQ494833
SI  - GENBANK/GQ494834
SI  - GENBANK/GQ494835
SI  - GENBANK/GQ494836
SI  - GENBANK/GQ494837
SI  - GENBANK/GQ494838
SI  - GENBANK/GQ494839
SI  - GENBANK/GQ494840
SI  - GENBANK/GQ494841
SI  - GENBANK/GQ494842
SI  - GENBANK/GQ494843
SI  - GENBANK/GQ494844
SI  - GENBANK/GQ494845
SI  - GENBANK/GQ494846
SI  - GENBANK/GQ494847
SI  - GENBANK/GQ494848
SI  - GENBANK/GQ494849
SI  - GENBANK/GQ494850
SI  - GENBANK/GQ494851
SI  - GENBANK/GQ494852
SI  - GENBANK/GQ494853
SI  - GENBANK/GQ494854
SI  - GENBANK/GQ494855
SI  - GENBANK/GQ494856
SI  - GENBANK/GQ494857
SI  - GENBANK/GQ494858
SI  - GENBANK/GQ494859
SI  - GENBANK/GQ494860
SI  - GENBANK/GQ494861
SI  - GENBANK/GQ494862
SI  - GENBANK/GQ494863
SI  - GENBANK/GQ494864
SI  - GENBANK/GQ494865
SI  - GENBANK/GQ494866
SI  - GENBANK/GQ494867
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090921
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - *Demography
MH  - Gene Flow/genetics
MH  - Genes/*genetics
MH  - *Genetics, Population
MH  - Lizards/*genetics
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - North America
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp057 [pii]
AID - 10.1093/sysbio/syp057 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Dec;58(6):547-59. doi: 10.1093/sysbio/syp057. Epub 2009 Sep 21.

PMID- 20525607
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 5
DP  - 2009 Oct
TI  - A correction corrected: consensus over the meaning of Crocodylia and why it
      matters.
PG  - 537-43
LID - 10.1093/sysbio/syp053 [doi]
FAU - Brochu, Christopher A
AU  - Brochu CA
AD  - Department of Geoscience, University of Iowa, Iowa City, IA 52242, USA.
FAU - Wagner, Jonathan R
AU  - Wagner JR
FAU - Jouve, Stephane
AU  - Jouve S
FAU - Sumrall, Colin D
AU  - Sumrall CD
FAU - Densmore, Llewellyn D
AU  - Densmore LD
LA  - eng
PT  - Comment
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090828
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2008 Feb;57(1):173-81. PMID: 18300130
MH  - Alligators and Crocodiles/*classification
MH  - Animals
MH  - *Phylogeny
MH  - *Terminology as Topic
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp053 [pii]
AID - 10.1093/sysbio/syp053 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Oct;58(5):537-43. doi: 10.1093/sysbio/syp053. Epub 2009 Aug 28.

PMID- 20525606
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 5
DP  - 2009 Oct
TI  - What is the danger of the anomaly zone for empirical phylogenetics?
PG  - 527-36
LID - 10.1093/sysbio/syp047 [doi]
AB  - The increasing number of observations of gene trees with discordant topologies in
      phylogenetic studies has raised awareness about the problems of incongruence
      between species trees and gene trees. Moreover, theoretical treatments focusing
      on the impact of coalescent variance on phylogenetic study have also identified
      situations where the most probable gene trees are ones that do not match the
      underlying species tree (i.e., anomalous gene trees [AGTs]). However, although
      the theoretical proof of the existence of AGTs is alarming, the actual risk that 
      AGTs pose to empirical phylogenetic study is far from clear. Establishing the
      conditions (i.e., the branch lengths in a species tree) for which AGTs are
      possible does not address the critical issue of how prevalent they might be.
      Furthermore, theoretical characterization of the species trees for which AGTs may
      pose a problem (i.e., the anomaly zone or the species histories for which AGTs
      are theoretically possible) is based on consideration of just one source of
      variance that contributes to species tree and gene tree discord-gene lineage
      coalescence. Yet, empirical data contain another important stochastic
      component-mutational variance. Estimated gene trees will differ from the
      underlying gene trees (i.e., the actual genealogy) because of the random process 
      of mutation. Here, we take a simulation approach to investigate the prevalence of
      AGTs, among estimated gene trees, thereby characterizing the boundaries of the
      anomaly zone taking into account both coalescent and mutational variances. We
      also determine the frequency of realized AGTs, which is critical to putting the
      theoretical work on AGTs into a realistic biological context. Two salient results
      emerge from this investigation. First, our results show that mutational variance 
      can indeed expand the parameter space (i.e., the relative branch lengths in a
      species tree) where AGTs might be observed in empirical data. By exploring the
      underlying cause for the expanded anomaly zone, we identify aspects of empirical 
      data relevant to avoiding the problems that AGTs pose for species tree inference 
      from multilocus data. Second, for the empirical species histories where AGTs are 
      possible, unresolved trees-not AGTs-predominate the pool of estimated gene trees.
      This result suggests that the risk of AGTs, while they exist in theory, may
      rarely be realized in practice. By considering the biological realities of both
      mutational and coalescent variances, the study has refined, and redefined, what
      the actual challenges are for empirical phylogenetic study of recently diverged
      taxa that have speciated rapidly-AGTs themselves are unlikely to pose a
      significant danger to empirical phylogenetic study.
FAU - Huang, Huateng
AU  - Huang H
AD  - Department of Ecology and Evolutionary Biology, Museum of Zoology, University of 
      Michigan, 1109 Geddes Avenue, Ann Arbor, MI 48109-1079, USA. huatengh@umich.edu
FAU - Knowles, L Lacey
AU  - Knowles LL
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090826
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Genes/*genetics
MH  - *Models, Genetic
MH  - Mutation/genetics
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp047 [pii]
AID - 10.1093/sysbio/syp047 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Oct;58(5):527-36. doi: 10.1093/sysbio/syp047. Epub 2009 Aug 26.

PMID- 20525605
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 5
DP  - 2009 Oct
TI  - Molecular phylogenetics of Thecata (Hydrozoa, Cnidaria) reveals long-term
      maintenance of life history traits despite high frequency of recent character
      changes.
PG  - 509-26
LID - 10.1093/sysbio/syp044 [doi]
AB  - Two fundamental life cycle types are recognized among hydrozoan cnidarians, the
      benthic (generally colonial) polyp stage either producing pelagic sexual medusae 
      or directly releasing gametes elaborated from an attached gonophore. The
      existence of intermediate forms, with polyps producing simple medusoids, has been
      classically considered compelling evidence in favor of phyletic gradualism. In
      order to gain insights about the evolution of hydrozoan life history traits, we
      inferred phylogenetic relationships of 142 species of Thecata (= Leptothecata,
      Leptomedusae), the most species-rich hydrozoan group, using 3 different ribosomal
      RNA markers (16S, 18S, and 28S). In conflict with morphology-derived
      classifications, most thecate species fell in 2 well-supported clades named here 
      Statocysta and Macrocolonia. We inferred many independent medusa losses among
      Statocysta. Several instances of secondary regain of medusoids (but not of full
      medusa) from medusa-less ancestors were supported among Macrocolonia.
      Furthermore, life cycle character changes were significantly correlated with
      changes affecting colony shape. For both traits, changes did not reflect graded
      and progressive loss or gain of complexity. They were concentrated in recent
      branches, with intermediate character states being relatively short lived at a
      large evolutionary scale. This punctuational pattern supports the existence of 2 
      alternative stable evolutionary strategies: simple stolonal colonies with medusae
      (the ancestral strategy, seen in most Statocysta species) versus large complex
      colonies with fixed gonophores (the derived strategy, seen in most Macrocolonia
      species). Hypotheses of species selection are proposed to explain the apparent
      long-term stability of these life history traits despite a high frequency of
      character change. Notably, maintenance of the medusa across geological time in
      Statocysta might be due to higher extinction rates for species that have lost
      this dispersive stage.
FAU - Leclere, Lucas
AU  - Leclere L
AD  - Universite Paris 06, UMR 7138/Centre National de la Recherche Scientifique UPMC
      MNHN IRD, 7 quai St Bernard, Paris, France. lucas.leclere@snv.jussieu.fr
FAU - Schuchert, Peter
AU  - Schuchert P
FAU - Cruaud, Corinne
AU  - Cruaud C
FAU - Couloux, Arnaud
AU  - Couloux A
FAU - Manuel, Michael
AU  - Manuel M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090821
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (RNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - *Biological Evolution
MH  - DNA Primers/genetics
MH  - Hydrozoa/anatomy &amp; histology/*genetics
MH  - Life Cycle Stages/*genetics
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - RNA, Ribosomal/genetics
MH  - Sequence Analysis, DNA
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp044 [pii]
AID - 10.1093/sysbio/syp044 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Oct;58(5):509-26. doi: 10.1093/sysbio/syp044. Epub 2009 Aug 21.

PMID- 20525604
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 5
DP  - 2009 Oct
TI  - Maximum likelihood estimates of species trees: how accuracy of phylogenetic
      inference depends upon the divergence history and sampling design.
PG  - 501-8
LID - 10.1093/sysbio/syp045 [doi]
AB  - The understanding that gene trees are often in discord with each other and with
      the species trees that contain them has led researchers to methods that
      incorporate the inherent stochasticity of genetic processes in the phylogenetic
      estimation procedure. Recently developed methods for species-tree estimation that
      not only consider the retention and sorting of ancestral polymorphism but also
      quantify the actual probabilities of incomplete lineage sorting are expected to
      provide an improvement over earlier summary-statistic based approaches that
      discard much of the information content of gene trees. However, these new methods
      have yet to be tested on truly challenging evolutionary histories such as those
      marked by recent rapid speciation where high levels of incomplete lineage sorting
      and discord among gene trees predominate. Here, we test a new maximum-likelihood 
      method that incorporates stochastic models of both nucleotide substitution and
      lineage sorting for species-tree estimation. Using a simulation approach, we
      consider a broad range of species-tree topologies under 2 scenarios representing 
      moderate and severe incomplete lineage sorting. We show that the
      maximum-likelihood method results in more accurate species trees than a
      summary-statistic based approach, demonstrating that information contained in
      discordant gene trees can be effectively extracted using a full probabilistic
      model. Moreover, we demonstrate that the shape of the original species tree
      (i.e., the relative lengths of internal branches) has a significant impact on
      whether the species tree is estimated accurately. In the speciation histories
      explored here, it is not just the recent origin of species that affects the
      accuracy of the estimates but the variance in relative species divergence times
      as well. Additionally, we show that sampling effort (number of individuals and/or
      loci) and sampling design (ratio of individuals to loci) are both important
      factors affecting the accuracy of species-tree estimates, which is again affected
      by the relative timing of divergence among species. The inherent difficulties of 
      estimating relationships when species have undergone a recent radiation are
      discussed, and in particular, the limitations with maximum-likelihood estimates
      of species trees that do not consider uncertainty in the estimated gene trees of 
      individual loci. Thus, despite substantial improvements over current
      summary-statistic based approaches, and the increased sophistication of
      procedures that incorporate the process of gene lineage coalescence, recent
      radiations still appear to pose daunting challenges for phylogenetics.
FAU - McCormack, John E
AU  - McCormack JE
AD  - Department of Ecology and Evolutionary Biology, and the Museum of Zoology,
      University of Michigan, Ann Arbor, MI 48109-1079, USA.
FAU - Huang, Huateng
AU  - Huang H
FAU - Knowles, L Lacey
AU  - Knowles LL
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090820
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Genes/*genetics
MH  - Genetic Speciation
MH  - *Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - *Research Design
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp045 [pii]
AID - 10.1093/sysbio/syp045 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Oct;58(5):501-8. doi: 10.1093/sysbio/syp045. Epub 2009 Aug 20.

PMID- 20525603
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20151119
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 5
DP  - 2009 Oct
TI  - Species trees from highly incongruent gene trees in rice.
PG  - 489-500
LID - 10.1093/sysbio/syp054 [doi]
AB  - Several methods have recently been developed to infer multilocus phylogenies by
      incorporating information from topological incongruence of the individual genes. 
      In this study, we investigate 2 such methods, Bayesian concordance analysis and
      Bayesian estimation of species trees. Our test data are a collection of genes
      from cultivated rice (genus Oryza) and the most closely related wild species,
      generated using a high-throughput sequencing protocol and bioinformatics
      pipeline. Trees inferred from independent genes display levels of topological
      incongruence that far exceed that seen in previous data sets analyzed with these 
      species tree methods. We identify differences in phylogenetic results between
      inference methods that incorporate gene tree incongruence. Finally, we discuss
      the challenges of scaling these analyses for data sets with thousands of gene
      trees and extensive levels of missing data.
FAU - Cranston, Karen A
AU  - Cranston KA
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA. kcranston@fieldmuseum.org
FAU - Hurwitz, Bonnie
AU  - Hurwitz B
FAU - Ware, Doreen
AU  - Ware D
FAU - Stein, Lincoln
AU  - Stein L
FAU - Wing, Rod A
AU  - Wing RA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090921
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - Genes/*genetics
MH  - *Models, Genetic
MH  - Oryza/*genetics
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
MH  - Software
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp054 [pii]
AID - 10.1093/sysbio/syp054 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Oct;58(5):489-500. doi: 10.1093/sysbio/syp054. Epub 2009 Sep 21.

PMID- 20525602
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 5
DP  - 2009 Oct
TI  - Identifying hybridization events in the presence of coalescence via model
      selection.
PG  - 478-88
LID - 10.1093/sysbio/syp055 [doi]
AB  - As DNA sequences have become more readily available, it has become increasingly
      desirable to infer species phylogenies from multigene data sets. Much recent work
      has centered around the recognition that substantial incongruence in single-gene 
      phylogenies necessitates the development of statistical procedures to estimate
      species phylogenies that appropriately model the process of evolution at the
      level of the individual genes. One process that gives rise to variation in the
      histories of individual genes is incomplete lineage sorting, which is commonly
      modeled by the coalescent, and thus much current work is focused on proper
      estimation of species phylogenies under the coalescent model. A second common
      source of discord in single-gene phylogenies is hybridization, a process that is 
      ubiquitous in many groups of plants and animals. Although methods to incorporate 
      hybridization into phylogenetic estimation have also been developed, only a
      handful of methods that address both coalescence and hybridization have been
      proposed. Here, I propose an extension of an existing model that incorporates
      both of these processes simultaneously by utilizing gene trees for inference in a
      likelihood framework. The model allows examination of the evidence for
      hybridization in the presence of incomplete lineage sorting due to deep
      coalescence via model selection using standard information criteria (e.g., Akaike
      information criterion and Bayesian information criterion). The potential of the
      method is evaluated using simulated data.
FAU - Kubatko, Laura Salter
AU  - Kubatko LS
AD  - Department of Statistics, The Ohio State University, Columbus, OH 43210, USA.
      lkubatko@stat.ohio-state.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090916
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - *Hybridization, Genetic
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp055 [pii]
AID - 10.1093/sysbio/syp055 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Oct;58(5):478-88. doi: 10.1093/sysbio/syp055. Epub 2009 Sep 16.

PMID- 20525601
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 5
DP  - 2009 Oct
TI  - Estimating species phylogenies using coalescence times among sequences.
PG  - 468-77
LID - 10.1093/sysbio/syp031 [doi]
AB  - The estimation of species trees (phylogenies) is one of the most important
      problems in evolutionary biology, and recently, there has been greater
      appreciation of the need to estimate species trees directly rather than using
      gene trees as a surrogate. A Bayesian method constructed under the multispecies
      coalescent model can consistently estimate species trees but involves intensive
      computation, which can hinder its application to the phylogenetic analysis of
      large-scale genomic data. Many summary statistics-based approaches, such as
      shallowest coalescences (SC) and Global LAteSt Split (GLASS), have been developed
      to infer species phylogenies for multilocus data sets. In this paper, we propose 
      2 methods, species tree estimation using average ranks of coalescences (STAR) and
      species tree estimation using average coalescence times (STEAC), based on the
      summary statistics of coalescence times. It can be shown that the 2 methods are
      statistically consistent under the multispecies coalescent model. STAR uses the
      ranks of coalescences and is thus resistant to variable substitution rates along 
      the branches in gene trees. A simulation study suggests that STAR consistently
      outperforms STEAC, SC, and GLASS when the substitution rates among lineages are
      highly variable. Two real genomic data sets were analyzed by the 2 methods and
      produced species trees that are consistent with previous results.
FAU - Liu, Liang
AU  - Liu L
AD  - Department of Organismic and Evolutionary Biology, Harvard University, Cambridge,
      MA 02138, USA. lliu@oeb.harvard.edu
FAU - Yu, Lili
AU  - Yu L
FAU - Pearl, Dennis K
AU  - Pearl DK
FAU - Edwards, Scott V
AU  - Edwards SV
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090716
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp031 [pii]
AID - 10.1093/sysbio/syp031 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Oct;58(5):468-77. doi: 10.1093/sysbio/syp031. Epub 2009 Jul 16.

PMID- 20525600
OWN - NLM
STAT- MEDLINE
DCOM- 20101012
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 5
DP  - 2009 Oct
TI  - Estimating species trees: methods of phylogenetic analysis when there is
      incongruence across genes.
PG  - 463-7
LID - 10.1093/sysbio/syp061 [doi]
FAU - Knowles, L Lacey
AU  - Knowles LL
AD  - Department of Ecology and Evolutionary Biology, Museum of Zoology, University of 
      Michigan, 1109 Geddes Avenue, Ann Arbor, MI 48109-1079, USA. knowlesl@umich.edu
LA  - eng
PT  - Introductory Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090917
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - Genes/*genetics
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2010/06/09 06:00
MHDA- 2010/10/13 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/10/13 06:00 [medline]
AID - syp061 [pii]
AID - 10.1093/sysbio/syp061 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Oct;58(5):463-7. doi: 10.1093/sysbio/syp061. Epub 2009 Sep 17.

PMID- 20525599
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 4
DP  - 2009 Aug
TI  - Phylogenetic analysis in the anomaly zone.
PG  - 452-60
LID - 10.1093/sysbio/syp034 [doi]
FAU - Liu, Liang
AU  - Liu L
AD  - Department of Organismic and Evolutionary Biology, Harvard University, 26 Oxford 
      Street, Cambridge, MA 02138, USA.
FAU - Edwards, Scott V
AU  - Edwards SV
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090709
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Alleles
MH  - Computer Simulation
MH  - Genetic Techniques
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp034 [pii]
AID - 10.1093/sysbio/syp034 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Aug;58(4):452-60. doi: 10.1093/sysbio/syp034. Epub 2009 Jul 9.

PMID- 20525598
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 4
DP  - 2009 Aug
TI  - Barcoding bamboozled by bacteria: convergence to metazoan mitochondrial primer
      targets by marine microbes.
PG  - 445-51
LID - 10.1093/sysbio/syp033 [doi]
FAU - Siddall, Mark E
AU  - Siddall ME
AD  - Sackler Institute for Comparative Genomics, American Museum of Natural History,
      New York, New York, USA.
FAU - Fontanella, Frank M
AU  - Fontanella FM
FAU - Watson, Sara C
AU  - Watson SC
FAU - Kvist, Sebastian
AU  - Kvist S
FAU - Erseus, Christer
AU  - Erseus C
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090706
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Electron Transport Complex IV/*genetics
MH  - Gammaproteobacteria/*genetics
MH  - *Genetic Techniques
MH  - Seawater/microbiology
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp033 [pii]
AID - 10.1093/sysbio/syp033 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Aug;58(4):445-51. doi: 10.1093/sysbio/syp033. Epub 2009 Jul 6.

PMID- 20525596
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 4
DP  - 2009 Aug
TI  - Can mtDNA barcodes be used to delimit species? A response to Pons et al. (2006).
PG  - 439-42; discussion 442-4
LID - 10.1093/sysbio/syp039 [doi]
FAU - Lohse, Konrad
AU  - Lohse K
AD  - Institute of Evolutionary Biology, University of Edinburgh, King's Buildings,
      Edinburgh EH9 3JT, UK. K.R.Lohse@sms.ed.ac.uk
LA  - eng
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Comment
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090715
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
CON - Syst Biol. 2006 Aug;55(4):595-609. PMID: 16967577
MH  - Classification/*methods
MH  - Computer Simulation
MH  - DNA, Mitochondrial/*genetics
MH  - Models, Genetic
MH  - Selection Bias
MH  - Species Specificity
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp039 [pii]
AID - 10.1093/sysbio/syp039 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Aug;58(4):439-42; discussion 442-4. doi: 10.1093/sysbio/syp039.
      Epub 2009 Jul 15.

PMID- 20525595
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 4
DP  - 2009 Aug
TI  - Experimental design in caecilian systematics: phylogenetic information of
      mitochondrial genomes and nuclear rag1.
PG  - 425-38
LID - 10.1093/sysbio/syp043 [doi]
AB  - In molecular phylogenetic studies, a major aspect of experimental design concerns
      the choice of markers and taxa. Although previous studies have investigated the
      phylogenetic performance of different genes and the effectiveness of increasing
      taxon sampling, their conclusions are partly contradictory, probably because they
      are highly context specific and dependent on the group of organisms used in each 
      study. Goldman introduced a method for experimental design in phylogenetics based
      on the expected information to be gained that has barely been used in practice.
      Here we use this method to explore the phylogenetic utility of mitochondrial (mt)
      genes, mt genomes, and nuclear rag1 for studies of the systematics of caecilian
      amphibians, as well as the effect of taxon addition on the stabilization of a
      controversial branch of the tree. Overall phylogenetic information estimates per 
      gene, specific estimates per branch of the tree, estimates for combined
      (mitogenomic) data sets, and estimates as a hypothetical new taxon is added to
      different parts of the caecilian tree are calculated and compared. In general,
      the most informative data sets are those for mt transfer and ribosomal RNA genes.
      Our results also show at which positions in the caecilian tree the addition of
      taxa have the greatest potential to increase phylogenetic information with
      respect to the controversial relationships of Scolecomorphus, Boulengerula, and
      all other teresomatan caecilians. These positions are, as intuitively expected,
      mostly (but not all) adjacent to the controversial branch. Generating whole
      mitogenomic and rag1 data for additional taxa joining the Scolecomorphus branch
      may be a more efficient strategy than sequencing a similar amount of additional
      nucleotides spread across the current caecilian taxon sampling. The methodology
      employed in this study allows an a priori evaluation and testable predictions of 
      the appropriateness of particular experimental designs to solve specific
      questions at different levels of the caecilian phylogeny.
FAU - San Mauro, Diego
AU  - San Mauro D
AD  - Department of Zoology, The Natural History Museum, Cromwell Road, London SW7 5BD,
      UK. d.san-mauro@nhm.ac.uk
FAU - Gower, David J
AU  - Gower DJ
FAU - Massingham, Tim
AU  - Massingham T
FAU - Wilkinson, Mark
AU  - Wilkinson M
FAU - Zardoya, Rafael
AU  - Zardoya R
FAU - Cotton, James A
AU  - Cotton JA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090818
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amphibians/classification/*genetics
MH  - Animals
MH  - Evolution, Molecular
MH  - *Genes, RAG-1
MH  - *Genome, Mitochondrial
MH  - *Phylogeny
MH  - Research Design
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp043 [pii]
AID - 10.1093/sysbio/syp043 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Aug;58(4):425-38. doi: 10.1093/sysbio/syp043. Epub 2009 Aug 18.

PMID- 20525594
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 4
DP  - 2009 Aug
TI  - Probabilistic orthology analysis.
PG  - 411-24
LID - 10.1093/sysbio/syp046 [doi]
AB  - Orthology analysis aims at identifying orthologous genes and gene products from
      different organisms and, therefore, is a powerful tool in modern computational
      and experimental biology. Although reconciliation-based orthology methods are
      generally considered more accurate than distance-based ones, the traditional
      parsimony-based implementation of reconciliation-based orthology analysis (most
      parsimonious reconciliation [MPR]) suffers from a number of shortcomings. For
      example, 1) it is limited to orthology predictions from the reconciliation that
      minimizes the number of gene duplication and loss events, 2) it cannot evaluate
      the support of this reconciliation in relation to the other reconciliations, and 
      3) it cannot make use of prior knowledge (e.g., about species divergence times)
      that provides auxiliary information for orthology predictions. We present a
      probabilistic approach to reconciliation-based orthology analysis that addresses 
      all these issues by estimating orthology probabilities. The method is based on
      the gene evolution model, an explicit evolutionary model for gene duplication and
      gene loss inside a species tree, that generalizes the standard birth-death
      process. We describe the probabilistic approach to orthology analysis using 2
      experimental data sets and show that the use of orthology probabilities allows a 
      more informative analysis than MPR and, in particular, that it is less sensitive 
      to taxon sampling problems. We generalize these anecdotal observations and show, 
      using data generated under biologically realistic conditions, that MPR give false
      orthology predictions at a substantial frequency. Last, we provide a new
      orthology prediction method that allows an orthology and paralogy classification 
      with any chosen sensitivity/specificity combination from the spectra of
      achievable combinations. We conclude that probabilistic orthology analysis is a
      strong and more advanced alternative to traditional orthology analysis and that
      it provides a framework for sophisticated comparative studies of processes in
      genome evolution.
FAU - Sennblad, Bengt
AU  - Sennblad B
AD  - Stockholm Bioinformatics Center, Department of Biochemistry, Stockholm
      University, AlbaNova, 106 91 Stockholm, Sweden. bengt.sennblad@sbc.su.se
FAU - Lagergren, Jens
AU  - Lagergren J
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090818
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Computational Biology/*methods
MH  - *Genes
MH  - Genes, MHC Class I
MH  - *Genetic Techniques
MH  - Humans
MH  - *Probability
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp046 [pii]
AID - 10.1093/sysbio/syp046 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Aug;58(4):411-24. doi: 10.1093/sysbio/syp046. Epub 2009 Aug 18.

PMID- 20525593
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 4
DP  - 2009 Aug
TI  - Recent long-distance dispersal overshadows ancient biogeographical patterns in a 
      pantropical angiosperm family (Simaroubaceae, Sapindales).
PG  - 395-410
LID - 10.1093/sysbio/syp041 [doi]
AB  - Detailed biogeographic studies of pantropical clades are still relatively few,
      and those conducted to date typically use parsimony or event-based methods to
      reconstruct ancestral areas. In this study, a recently developed likelihood
      method for reconstructing ancestral areas (the dispersal-extinction cladogenesis 
      [DEC] model) is applied to the angiosperm family Simaroubaceae, a geographically 
      widespread and ecologically diverse clade of pantropical and temperate trees and 
      shrubs. To estimate divergence dates in the family, Bayesian uncorrelated rates
      analyses and robust fossil calibrations are applied to the well-sampled and
      strongly supported phylogeny. For biogeographic analyses, the effects of
      parameter configurations in the DEC model are assessed for different possible
      ancestral ranges, and the likelihood method is compared with dispersal-vicariance
      analysis (DIVA). Regardless of the parameters used, likelihood analyses show a
      common pattern of multiple recent range shifts that overshadow reconstruction of 
      events deeper in the family's history. DIVA produced results similar to the DEC
      model when ancestral ranges were restricted to two areas, but some improbable
      ancestral ranges were also observed. Simaroubaceae exhibit an early history of
      range expansion between major continental areas in the Northern Hemisphere, but
      reconstruction of ancestral areas for lineages diverging in the early Tertiary
      are sensitive to the parameters of the model used. A North American origin is
      suggested for the family, with migration via Beringia by ancestral taxa. In
      contrast to traditional views, long-distance dispersal events are common,
      particularly in the Late Oligocene and later. Notable dispersals are inferred to 
      have occurred across the Atlantic Ocean in both directions, as well as between
      Africa and Asia, and around the Indian Ocean basin and Pacific islands.
FAU - Clayton, Joshua W
AU  - Clayton JW
AD  - Department of Botany, University of Florida, Gainesville, FL 32611-7800, USA.
FAU - Soltis, Pamela S
AU  - Soltis PS
FAU - Soltis, Douglas E
AU  - Soltis DE
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090814
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Evolution, Molecular
MH  - *Fossils
MH  - *Gene Flow
MH  - *Geography
MH  - Phylogeny
MH  - Simaroubaceae/*genetics
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp041 [pii]
AID - 10.1093/sysbio/syp041 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Aug;58(4):395-410. doi: 10.1093/sysbio/syp041. Epub 2009 Aug 14.

PMID- 20525592
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 4
DP  - 2009 Aug
TI  - Nonstationary evolution and compositional heterogeneity in beetle mitochondrial
      phylogenomics.
PG  - 381-94
LID - 10.1093/sysbio/syp037 [doi]
AB  - Many published phylogenies are based on methods that assume equal nucleotide
      composition among taxa. Studies have shown, however, that this assumption is
      often not accurate, particularly in divergent lineages. Nonstationary sequence
      evolution, when taxa in different lineages evolve in different ways, can lead to 
      unequal nucleotide composition. This can cause inference methods to fail and
      phylogenies to be inaccurate. Recent advancements in phylogenetic theory have
      proposed new models of nonstationary sequence evolution; these models often
      outperform equivalent stationary models. A variety of new phylogenetic software
      implementing such models has been developed, but the studies employing the new
      methodology are still few. We discovered convergence of nucleotide composition
      within mitochondrial genomes of the insect order Coleoptera (beetles). We found
      variation in base content both among species and among genes in the genome. To
      this data set, we have applied a broad range of phylogenetic methods, including
      some traditional stationary models of evolution and all the more recent
      nonstationary models. We compare 8 inference methods applied to the same data
      set. Although the more commonly used methods universally fail to recover
      established clades, we find that some of the newer software packages are more
      appropriate for data of this nature. The software packages p4, PHASE, and nhPhyML
      were able to overcome the systematic bias in our data set, but parsimony,
      MrBayes, NJ, LogDet, and PhyloBayes were not.
FAU - Sheffield, Nathan C
AU  - Sheffield NC
AD  - Department of Biology, Brigham Young University, Provo, UT 84602, USA.
      nathan.sheffield@duke.edu
FAU - Song, Hojun
AU  - Song H
FAU - Cameron, Stephen L
AU  - Cameron SL
FAU - Whiting, Michael F
AU  - Whiting MF
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090715
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Base Composition
MH  - Coleoptera/*genetics
MH  - Genome, Mitochondrial
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp037 [pii]
AID - 10.1093/sysbio/syp037 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Aug;58(4):381-94. doi: 10.1093/sysbio/syp037. Epub 2009 Jul 15.

PMID- 20525591
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 3
DP  - 2009 Jun
TI  - Accounting for calibration uncertainty in phylogenetic estimation of evolutionary
      divergence times.
PG  - 367-80
LID - 10.1093/sysbio/syp035 [doi]
FAU - Ho, Simon Y W
AU  - Ho SY
AD  - Centre for Macroevolution and Macroecology, Research School of Biology,
      Australian National University, Canberra, ACT 0200, Australia.
      simon.ho@anu.edu.au
FAU - Phillips, Matthew J
AU  - Phillips MJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090703
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Birds/*genetics
MH  - Calibration
MH  - Fossils
MH  - *Phylogeny
MH  - Statistical Distributions
MH  - *Time
MH  - Uncertainty
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp035 [pii]
AID - 10.1093/sysbio/syp035 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Jun;58(3):367-80. doi: 10.1093/sysbio/syp035. Epub 2009 Jul 3.

PMID- 20525590
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 3
DP  - 2009 Jun
TI  - Properties of majority-rule supertrees.
PG  - 360-7
LID - 10.1093/sysbio/syp032 [doi]
FAU - Dong, Jianrong
AU  - Dong J
AD  - Department of Computer Science, Iowa State University, Ames, IA 50011, USA.
FAU - Fernandez-Baca, David
AU  - Fernandez-Baca D
LA  - eng
PT  - Journal Article
DEP - 20090703
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Genetic Techniques
MH  - *Phylogeny
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp032 [pii]
AID - 10.1093/sysbio/syp032 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Jun;58(3):360-7. doi: 10.1093/sysbio/syp032. Epub 2009 Jul 3.

PMID- 20525589
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 3
DP  - 2009 Jun
TI  - Time dependency of molecular rates in ancient DNA data sets, a sampling artifact?
PG  - 348-60
LID - 10.1093/sysbio/syp028 [doi]
FAU - Debruyne, Regis
AU  - Debruyne R
AD  - Ancient DNA Centre, Department of Anthropology, McMaster University, 1280 Main
      Street West, Hamilton, Ontario, Canada. debruyne@mcmaster.ca
FAU - Poinar, Hendrik N
AU  - Poinar HN
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090701
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Artifacts
MH  - Bayes Theorem
MH  - Bison/genetics
MH  - Calibration
MH  - *Evolution, Molecular
MH  - Hominidae/genetics
MH  - Mammoths/genetics
MH  - Models, Genetic
MH  - *Time
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp028 [pii]
AID - 10.1093/sysbio/syp028 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Jun;58(3):348-60. doi: 10.1093/sysbio/syp028. Epub 2009 Jul 1.

PMID- 20525588
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 3
DP  - 2009 Jun
TI  - Examining modularity via partial correlations: a rejoinder to a comment by Paul
      Magwene.
PG  - 346-8
LID - 10.1093/sysbio/syp040 [doi]
FAU - Mitteroecker, Philipp
AU  - Mitteroecker P
AD  - Department of Theoretical Biology, University of Vienna, Althanstrasse 14,
      Vienna, Austria. philipp.mitteroecker@univie.ac.at
FAU - Bookstein, Fred L
AU  - Bookstein FL
LA  - eng
PT  - Comment
PT  - Journal Article
DEP - 20090717
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2009 Feb;58(1):146-9. PMID: 20525574
MH  - Biological Evolution
MH  - Classification/*methods
MH  - *Models, Statistical
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp040 [pii]
AID - 10.1093/sysbio/syp040 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Jun;58(3):346-8. doi: 10.1093/sysbio/syp040. Epub 2009 Jul 17.

PMID- 20525587
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 3
DP  - 2009 Jun
TI  - Hybrid origins and homoploid reticulate evolution within Heliosperma (Sileneae,
      Caryophyllaceae)--a multigene phylogenetic approach with relative dating.
PG  - 328-45
LID - 10.1093/sysbio/syp030 [doi]
AB  - We used four potentially unlinked nuclear DNA regions from the gene family
      encoding the second largest subunit of the RNA polymerases, as well as the
      psbE-petG spacer and the rps16 intron from the chloroplast genome, to evaluate
      the origin of and relationships within Heliosperma (Sileneae, Caryophyllaceae).
      Relative dates of divergence times are used to discriminate between hybridization
      and gene duplication/loss as alternative explanations for topological conflicts
      between gene trees. The observed incongruent relationships among the three major 
      lineages of Heliosperma are better explained by homoploid hybridization than by
      gene duplication/losses because species branching events exceed gene coalescence 
      times under biologically reasonable population sizes and generation times, making
      lineage sorting an unlikely explanation. The origin of Heliosperma is complex and
      the gene trees likely reflect both reticulate evolution and sorting events. At
      least two lineages have been involved in the origin of Heliosperma, one most
      closely related to the ancestor of Viscaria and Atocion and the other to
      Eudianthe and/or Petrocoptis.
FAU - Frajman, Bozo
AU  - Frajman B
AD  - Department of Systematic Botany, Evolutionary Biology Centre, Uppsala University,
      Norbyvagen 18D, Uppsala, Sweden.
FAU - Eggens, Frida
AU  - Eggens F
FAU - Oxelman, Bengt
AU  - Oxelman B
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090703
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
RN  - EC 2.7.7.6 (DNA-Directed RNA Polymerases)
SB  - IM
MH  - Caryophyllaceae/*genetics
MH  - DNA, Chloroplast/*genetics
MH  - DNA-Directed RNA Polymerases/*genetics
MH  - *Hybridization, Genetic
MH  - *Phylogeny
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp030 [pii]
AID - 10.1093/sysbio/syp030 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Jun;58(3):328-45. doi: 10.1093/sysbio/syp030. Epub 2009 Jul 3.

PMID- 20525586
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 3
DP  - 2009 Jun
TI  - Testing the impact of miniaturization on phylogeny: Paleozoic dissorophoid
      amphibians.
PG  - 312-27
LID - 10.1093/sysbio/syp029 [doi]
AB  - Among the diverse clade of Paleozoic dissorophoid amphibians, the small,
      terrestrial amphibamids and the neotenic branchiosaurids have frequently been
      suggested as possible antecedents of either all or some of the modern amphibian
      clades. Classically, amphibamids and branchiosaurids have been considered to
      represent distinct, but closely related clades within dissorophoids, but despite 
      their importance for the controversial lissamphibian origins, a comprehensive
      phylogenetic analysis of small dissorophoids has thus far not been attempted. On 
      the basis of an integrated data set, the relationships of amphibamids and
      branchiosaurids were analyzed using parsimony and Bayesian approaches. Both
      groups represent miniaturized forms and it was tested whether similar
      developmental pathways, associated with miniaturization, lead to an artificial
      close relationship of branchiosaurids and amphibamids. Moreover, the fit of the
      resulting tree topologies to the distribution of fossil taxa in the stratigraphic
      rock record was assessed as an additional source of information. The results show
      that characters associated with a miniaturized morphology are not responsible for
      the close clustering of branchiosaurids and amphibamids. Instead, all analyses
      invariably demonstrate a monophyletic clade of branchiosaurids highly nested
      within derived amphibamids, indicating that branchiosaurids represent a group of 
      secondarily neotenic amphibamid dissorophoids. This understanding of the
      phylogenetic relationships of small dissorophoid amphibians provides a new
      framework for the discussion of their evolutionary history and the evolution of
      characters shared by branchiosaurids and/or amphibamids with modern amphibian
      taxa.
FAU - Frobisch, Nadia B
AU  - Frobisch NB
AD  - Redpath Museum, McGill University, 859 Sherbrooke Street West, Montreal, QC,
      Canada. nfrobisch@uchicago.edu
FAU - Schoch, Rainer R
AU  - Schoch RR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090703
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amphibians/*genetics/growth &amp; development
MH  - Animals
MH  - Bayes Theorem
MH  - *Body Size
MH  - *Fossils
MH  - *Phylogeny
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp029 [pii]
AID - 10.1093/sysbio/syp029 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Jun;58(3):312-27. doi: 10.1093/sysbio/syp029. Epub 2009 Jul 3.

PMID- 20525585
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 3
DP  - 2009 Jun
TI  - Accelerated species inventory on Madagascar using coalescent-based models of
      species delineation.
PG  - 298-311
LID - 10.1093/sysbio/syp027 [doi]
AB  - High-throughput DNA sequencing has the potential to accelerate species discovery 
      if it is able to recognize evolutionary entities from sequence data that are
      comparable to species. The general mixed Yule-coalescent (GMYC) model estimates
      the species boundary from DNA surveys by identifying independently evolving
      lineages as a transition from coalescent to speciation branching patterns on a
      phylogenetic tree. Applied here to 12 families from 4 orders of insects in
      Madagascar, we used the model to delineate 370 putative species from
      mitochondrial DNA sequence variation among 1614 individuals. These were compared 
      with data from the nuclear genome and morphological identification and found to
      be highly congruent (98% and 94%). We developed a modified GMYC that allows for a
      variable transition from coalescent to speciation among lineages. This revised
      model increased the congruence with morphology (97%), suggesting that a variable 
      threshold better reflects the clustering of sequence data into biological
      species. Local endemism was pronounced in all 5 insect groups. Most species
      (60-91%) and haplotypes (88-99%) were found at only 1 of the 5 study sites
      (40-1000 km apart). This pronounced endemism resulted in a 37% increase in
      species numbers using diagnostic nucleotides in a population aggregation
      analysis. Sample sizes between 7 and 10 individuals represented a threshold above
      which there was minimal increase in genetic diversity, broadly agreeing with
      coalescent theory and other empirical studies. Our results from &gt; 1.4 Mb of
      empirical data suggest that the GMYC model captures species boundaries comparable
      to those from traditional methods without the need for prior hypotheses of
      population coherence. This provides a method of species discovery and
      biodiversity assessment using single-locus data from mixed or environmental
      samples while building a globally available taxonomic database for future
      identifications.
FAU - Monaghan, Michael T
AU  - Monaghan MT
AD  - Entomology Department, Natural History Museum, London, UK. monaghan@igb-berlin.de
FAU - Wild, Ruth
AU  - Wild R
FAU - Elliot, Miranda
AU  - Elliot M
FAU - Fujisawa, Tomochika
AU  - Fujisawa T
FAU - Balke, Michael
AU  - Balke M
FAU - Inward, Daegan J G
AU  - Inward DJ
FAU - Lees, David C
AU  - Lees DC
FAU - Ranaivosolo, Ravo
AU  - Ranaivosolo R
FAU - Eggleton, Paul
AU  - Eggleton P
FAU - Barraclough, Timothy G
AU  - Barraclough TG
FAU - Vogler, Alfried P
AU  - Vogler AP
LA  - eng
GR  - BBS/B/04358/Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090701
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biodiversity
MH  - Genes, rRNA
MH  - Genetic Variation
MH  - Haplotypes
MH  - Insecta/*genetics
MH  - Madagascar
MH  - *Models, Genetic
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp027 [pii]
AID - 10.1093/sysbio/syp027 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Jun;58(3):298-311. doi: 10.1093/sysbio/syp027. Epub 2009 Jul 1.

PMID- 20525584
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 3
DP  - 2009 Jun
TI  - A phylogenetic estimation of trophic transition networks for ascomycetous fungi: 
      are lichens cradles of symbiotrophic fungal diversification?
PG  - 283-97
LID - 10.1093/sysbio/syp001 [doi]
AB  - Fungi associated with photosynthetic organisms are major determinants of
      terrestrial biomass, nutrient cycling, and ecosystem productivity from the poles 
      to the equator. Whereas most fungi are known because of their fruit bodies (e.g.,
      saprotrophs), symptoms (e.g., pathogens), or emergent properties as symbionts
      (e.g., lichens), the majority of fungal diversity is thought to occur among
      species that rarely manifest their presence with visual cues on their substrate
      (e.g., the apparently hyperdiverse fungal endophytes associated with foliage of
      plants). Fungal endophytes are ubiquitous among all lineages of land plants and
      live within overtly healthy tissues without causing disease, but the evolutionary
      origins of these highly diverse symbionts have not been explored. Here, we show
      that a key to understanding both the evolution of endophytism and the
      diversification of the most species-rich phylum of Fungi (Ascomycota) lies in
      endophyte-like fungi that can be isolated from the interior of apparently healthy
      lichens. These "endolichenic" fungi are distinct from lichen mycobionts or any
      other previously recognized fungal associates of lichens, represent the same
      major lineages of Ascomycota as do endophytes, largely parallel the high
      diversity of endophytes from the arctic to the tropics, and preferentially
      associate with green algal photobionts in lichen thalli. Using phylogenetic
      analyses that incorporate these newly recovered fungi and ancestral state
      reconstructions that take into account phylogenetic uncertainty, we show that
      endolichenism is an incubator for the evolution of endophytism. In turn,
      endophytism is evolutionarily transient, with endophytic lineages frequently
      transitioning to and from pathogenicity. Although symbiotrophic lineages
      frequently give rise to free-living saprotrophs, reversions to symbiosis are
      rare. Together, these results provide the basis for estimating trophic transition
      networks in the Ascomycota and provide a first set of hypotheses regarding the
      evolution of symbiotrophy and saprotrophy in the most species-rich fungal phylum.
      [Ancestral state reconstruction; Ascomycota; Bayesian analysis; endolichenic
      fungi; fungal endophytes; lichens; pathogens; phylogeny; saprotrophy;
      symbiotrophy; trophic transition network.].
FAU - Arnold, A Elizabeth
AU  - Arnold AE
AD  - Department of Biology, Duke University, Durham, NC 27708, USA.
      Arnold@ag.arizona.edu
FAU - Miadlikowska, Jolanta
AU  - Miadlikowska J
FAU - Higgins, K Lindsay
AU  - Higgins KL
FAU - Sarvate, Snehal D
AU  - Sarvate SD
FAU - Gugger, Paul
AU  - Gugger P
FAU - Way, Amanda
AU  - Way A
FAU - Hofstetter, Valerie
AU  - Hofstetter V
FAU - Kauff, Frank
AU  - Kauff F
FAU - Lutzoni, Francois
AU  - Lutzoni F
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090711
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Ascomycota
MH  - *Biological Evolution
MH  - Lichens/*microbiology
MH  - Plants/*microbiology
MH  - *Symbiosis
EDAT- 2010/06/09 06:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2010/06/09 06:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp001 [pii]
AID - 10.1093/sysbio/syp001 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Jun;58(3):283-97. doi: 10.1093/sysbio/syp001. Epub 2009 Jul 11.

PMID- 20525583
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 2
DP  - 2009 Apr
TI  - Proximity, interpenetration, and sympatry networks: a reply to Dos Santos et al.
PG  - 271-6
LID - 10.1093/sysbio/syp022 [doi]
FAU - Casagranda, M Dolores
AU  - Casagranda MD
AD  - Consejo Nacional de Investigaciones Cientificas y Tecnicas, Instituto Superior de
      Entomologia, Facultad de Ciencias Naturales, Miguel Lillo 205, 4000 S. M. de
      Tucumin, Argentina.
FAU - Arias, J Salvador
AU  - Arias JS
FAU - Goloboff, Pablo A
AU  - Goloboff PA
FAU - Szumik, Claudia A
AU  - Szumik CA
FAU - Taher, Leila M
AU  - Taher LM
FAU - Escalante, Tania
AU  - Escalante T
FAU - Morrone, Juan J
AU  - Morrone JJ
LA  - eng
PT  - Comment
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090609
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2008 Jun;57(3):432-48. PMID: 18570037
MH  - *Ecosystem
MH  - *Geography
EDAT- 2009/04/01 00:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/04/01 00:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp022 [pii]
AID - 10.1093/sysbio/syp022 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Apr;58(2):271-6. doi: 10.1093/sysbio/syp022. Epub 2009 Jun 9.

PMID- 20525582
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 2
DP  - 2009 Apr
TI  - Networks, trees, and treeshrews: assessing support and identifying conflict with 
      multiple loci and a problematic root.
PG  - 257-70
LID - 10.1093/sysbio/syp025 [doi]
AB  - Multiple unlinked genetic loci often provide a more comprehensive picture of
      evolutionary history than any single gene can, but analyzing multigene data
      presents particular challenges. Differing rates and patterns of nucleotide
      substitution, combined with the limited information available in any data set,
      can make it difficult to specify a model of evolution. In addition, conflict
      among loci can be the result of real differences in evolutionary process or of
      stochastic variance and errors in reconstruction. We used 6 presumably unlinked
      nuclear loci to investigate relationships within the mammalian family Tupaiidae
      (Scandentia), containing all but one of the extant tupaiid genera. We used a
      phylogenetic mixture model to analyze the concatenated data and compared this
      with results using partitioned models. We found that more complex models were not
      necessarily preferred under tests using Bayes factors and that model complexity
      affected both tree length and parameter variance. We also compared the results of
      single-gene and multigene analyses and used splits networks to analyze the source
      and degree of conflict among genes. Networks can show specific relationships that
      are inconsistent with each other; these conflicting and minority relationships,
      which are implicitly ignored or collapsed by traditional consensus methods, can
      be useful in identifying the underlying causes of topological uncertainty. In our
      data, conflict is concentrated around particular relationships, not widespread
      throughout the tree. This pattern is further clarified by considering conflict
      surrounding the root separately from conflict within the ingroup. Uncertainty in 
      rooting may be because of the apparent evolutionary distance separating these
      genera and our outgroup, the tupaiid genus Dendrogale. Unlike a previous
      mitochondrial study, these nuclear data strongly suggest that the genus Tupaia is
      not monophyletic with respect to the monotypic Urogale, even when uncertainty
      about rooting is taken into account. These data concur with mitochondrial DNA on 
      other relationships, including the close affinity of Tupaia tana with the
      enigmatic Tupaia splendidula and of Tupaia belangeri with Tupaia glis. We also
      discuss the taxonomic and biogeographic implications of these results.
FAU - Roberts, Trina E
AU  - Roberts TE
AD  - University of Alaska Museum and Institute of Arctic Biology, University of
      Alaska-Fairbanks, Fairbanks, AK 99775, USA. trina.roberts@duke.edu
FAU - Sargis, Eric J
AU  - Sargis EJ
FAU - Olson, Link E
AU  - Olson LE
LA  - eng
GR  - RR016466/RR/NCRR NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090609
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Tupaiidae/classification/*genetics
PMC - PMC2715937
EDAT- 2009/04/01 00:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/04/01 00:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp025 [pii]
AID - 10.1093/sysbio/syp025 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Apr;58(2):257-70. doi: 10.1093/sysbio/syp025. Epub 2009 Jun 9.

PMID- 20525581
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 2
DP  - 2009 Apr
TI  - Complete generic-level phylogenetic analyses of palms (Arecaceae) with
      comparisons of supertree and supermatrix approaches.
PG  - 240-56
LID - 10.1093/sysbio/syp021 [doi]
AB  - Supertree and supermatrix methods have great potential in the quest to build the 
      tree of life and yet they remain controversial, with most workers opting for one 
      approach or the other, but rarely both. Here, we employed both methods to
      construct phylogenetic trees of all genera of palms (Arecaceae/Palmae), an iconic
      angiosperm family of great economic importance. We assembled a supermatrix
      consisting of 16 partitions, comprising DNA sequence data, plastid restriction
      fragment length polymorphism data, and morphological data for all genera, from
      which a highly resolved and well-supported phylogenetic tree was built despite
      abundant missing data. To construct supertrees, we used variants of matrix
      representation with parsimony (MRP) analysis based on input trees generated
      directly from subsamples of the supermatrix. All supertrees were highly resolved.
      Standard MRP with bootstrap-weighted matrix elements performed most effectively
      in this case, generating trees with the greatest congruence with the supermatrix 
      tree and fewest clades unsupported by any input tree. Nonindependence due to
      input trees based on combinations of data partitions was an acceptable trade-off 
      for improvements in supertree performance. Irreversible MRP and the use of
      strictly independent input trees only provided no obvious benefits. Contrary to
      previous claims, we found that unsupported clades are not infrequent under some
      MRP implementations, with up to 13% of clades lacking support from any input tree
      in some irreversible MRP supertrees. To build a formal synthesis, we assessed the
      cross-corroboration between supermatrix trees and the variant supertrees using
      semistrict consensus, enumerating shared clades and compatible clades. The
      semistrict consensus of the supermatrix tree and the most congruent supertree
      contained 160 clades (of a maximum of 204), 137 of which were present in both
      trees. The relationships recovered by these trees strongly support the current
      phylogenetic classification of palms. We evaluate 2 composite supertree support
      measures (rQS and V) and conclude that it is more informative to report numbers
      of input trees that support or conflict with a given supertree clade. This study 
      demonstrates that supertree and supermatrix methods can provide effective,
      explicit, and complimentary mechanisms for synthesizing disjointed phylogenetic
      evidence while emphasizing the need for further refinement of supertree methods.
FAU - Baker, William J
AU  - Baker WJ
AD  - Royal Botanic Gardens, Kew, Richmond, Surrey, UK.
FAU - Savolainen, Vincent
AU  - Savolainen V
FAU - Asmussen-Lange, Conny B
AU  - Asmussen-Lange CB
FAU - Chase, Mark W
AU  - Chase MW
FAU - Dransfield, John
AU  - Dransfield J
FAU - Forest, Felix
AU  - Forest F
FAU - Harley, Madeline M
AU  - Harley MM
FAU - Uhl, Natalie W
AU  - Uhl NW
FAU - Wilkinson, Mark
AU  - Wilkinson M
LA  - eng
PT  - Comparative Study
PT  - Journal Article
DEP - 20090530
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Arecaceae/classification/*genetics
MH  - *Phylogeny
EDAT- 2009/04/01 00:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/04/01 00:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp021 [pii]
AID - 10.1093/sysbio/syp021 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Apr;58(2):240-56. doi: 10.1093/sysbio/syp021. Epub 2009 May 30.

PMID- 20525580
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 2
DP  - 2009 Apr
TI  - The Ascomycota tree of life: a phylum-wide phylogeny clarifies the origin and
      evolution of fundamental reproductive and ecological traits.
PG  - 224-39
LID - 10.1093/sysbio/syp020 [doi]
AB  - We present a 6-gene, 420-species maximum-likelihood phylogeny of Ascomycota, the 
      largest phylum of Fungi. This analysis is the most taxonomically complete to date
      with species sampled from all 15 currently circumscribed classes. A number of
      superclass-level nodes that have previously evaded resolution and were unnamed in
      classifications of the Fungi are resolved for the first time. Based on the 6-gene
      phylogeny we conducted a phylogenetic informativeness analysis of all 6 genes and
      a series of ancestral character state reconstructions that focused on morphology 
      of sporocarps, ascus dehiscence, and evolution of nutritional modes and
      ecologies. A gene-by-gene assessment of phylogenetic informativeness yielded
      higher levels of informativeness for protein genes (RPB1, RPB2, and TEF1) as
      compared with the ribosomal genes, which have been the standard bearer in fungal 
      systematics. Our reconstruction of sporocarp characters is consistent with 2
      origins for multicellular sexual reproductive structures in Ascomycota, once in
      the common ancestor of Pezizomycotina and once in the common ancestor of
      Neolectomycetes. This first report of dual origins of ascomycete sporocarps
      highlights the complicated nature of assessing homology of morphological traits
      across Fungi. Furthermore, ancestral reconstruction supports an open sporocarp
      with an exposed hymenium (apothecium) as the primitive morphology for
      Pezizomycotina with multiple derivations of the partially (perithecia) or
      completely enclosed (cleistothecia) sporocarps. Ascus dehiscence is most
      informative at the class level within Pezizomycotina with most superclass nodes
      reconstructed equivocally. Character-state reconstructions support a terrestrial,
      saprobic ecology as ancestral. In contrast to previous studies, these analyses
      support multiple origins of lichenization events with the loss of lichenization
      as less frequent and limited to terminal, closely related species.
FAU - Schoch, Conrad L
AU  - Schoch CL
AD  - Department of Botany and Plant Pathology, Oregon State University, Corvallis, OR 
      97331, USA.
FAU - Sung, Gi-Ho
AU  - Sung GH
FAU - Lopez-Giraldez, Francesc
AU  - Lopez-Giraldez F
FAU - Townsend, Jeffrey P
AU  - Townsend JP
FAU - Miadlikowska, Jolanta
AU  - Miadlikowska J
FAU - Hofstetter, Valerie
AU  - Hofstetter V
FAU - Robbertse, Barbara
AU  - Robbertse B
FAU - Matheny, P Brandon
AU  - Matheny PB
FAU - Kauff, Frank
AU  - Kauff F
FAU - Wang, Zheng
AU  - Wang Z
FAU - Gueidan, Cecile
AU  - Gueidan C
FAU - Andrie, Rachael M
AU  - Andrie RM
FAU - Trippe, Kristin
AU  - Trippe K
FAU - Ciufetti, Linda M
AU  - Ciufetti LM
FAU - Wynns, Anja
AU  - Wynns A
FAU - Fraker, Emily
AU  - Fraker E
FAU - Hodkinson, Brendan P
AU  - Hodkinson BP
FAU - Bonito, Gregory
AU  - Bonito G
FAU - Groenewald, Johannes Z
AU  - Groenewald JZ
FAU - Arzanlou, Mahdi
AU  - Arzanlou M
FAU - de Hoog, G Sybren
AU  - de Hoog GS
FAU - Crous, Pedro W
AU  - Crous PW
FAU - Hewitt, David
AU  - Hewitt D
FAU - Pfister, Donald H
AU  - Pfister DH
FAU - Peterson, Kristin
AU  - Peterson K
FAU - Gryzenhout, Marieka
AU  - Gryzenhout M
FAU - Wingfield, Michael J
AU  - Wingfield MJ
FAU - Aptroot, Andre
AU  - Aptroot A
FAU - Suh, Sung-Oui
AU  - Suh SO
FAU - Blackwell, Meredith
AU  - Blackwell M
FAU - Hillis, David M
AU  - Hillis DM
FAU - Griffith, Gareth W
AU  - Griffith GW
FAU - Castlebury, Lisa A
AU  - Castlebury LA
FAU - Rossman, Amy Y
AU  - Rossman AY
FAU - Lumbsch, H Thorsten
AU  - Lumbsch HT
FAU - Lucking, Robert
AU  - Lucking R
FAU - Budel, Burkhard
AU  - Budel B
FAU - Rauhut, Alexandra
AU  - Rauhut A
FAU - Diederich, Paul
AU  - Diederich P
FAU - Ertz, Damien
AU  - Ertz D
FAU - Geiser, David M
AU  - Geiser DM
FAU - Hosaka, Kentaro
AU  - Hosaka K
FAU - Inderbitzin, Patrik
AU  - Inderbitzin P
FAU - Kohlmeyer, Jan
AU  - Kohlmeyer J
FAU - Volkmann-Kohlmeyer, Brigitte
AU  - Volkmann-Kohlmeyer B
FAU - Mostert, Lizel
AU  - Mostert L
FAU - O'Donnell, Kerry
AU  - O'Donnell K
FAU - Sipman, Harrie
AU  - Sipman H
FAU - Rogers, Jack D
AU  - Rogers JD
FAU - Shoemaker, Robert A
AU  - Shoemaker RA
FAU - Sugiyama, Junta
AU  - Sugiyama J
FAU - Summerbell, Richard C
AU  - Summerbell RC
FAU - Untereiner, Wendy
AU  - Untereiner W
FAU - Johnston, Peter R
AU  - Johnston PR
FAU - Stenroos, Soili
AU  - Stenroos S
FAU - Zuccaro, Alga
AU  - Zuccaro A
FAU - Dyer, Paul S
AU  - Dyer PS
FAU - Crittenden, Peter D
AU  - Crittenden PD
FAU - Cole, Mariette S
AU  - Cole MS
FAU - Hansen, Karen
AU  - Hansen K
FAU - Trappe, James M
AU  - Trappe JM
FAU - Yahr, Rebecca
AU  - Yahr R
FAU - Lutzoni, Francois
AU  - Lutzoni F
FAU - Spatafora, Joseph W
AU  - Spatafora JW
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090604
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Ascomycota/classification/cytology/*genetics
MH  - Ecosystem
MH  - Genes, Fungal
MH  - *Phylogeny
MH  - Reproduction
EDAT- 2009/04/01 00:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/04/01 00:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp020 [pii]
AID - 10.1093/sysbio/syp020 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Apr;58(2):224-39. doi: 10.1093/sysbio/syp020. Epub 2009 Jun 4.

PMID- 20525579
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 2
DP  - 2009 Apr
TI  - Bootstrap support is not first-order correct.
PG  - 211-23
LID - 10.1093/sysbio/syp016 [doi]
AB  - The appropriate interpretation of bootstrap support for splits and the question
      of what constitutes large bootstrap support have received considerable attention.
      One desirable interpretation, indeed the interpretation that was put forward when
      bootstrap support for splits was first introduced, is that 1-minus bootstrap
      support is a P value for the hypothesis that the split is not well resolved. As a
      P value, bootstrap support has been argued to be first-order correct. By
      obtaining the limiting distribution of bootstrap support for a split when maximum
      likelihood estimation is conducted, it is shown that bootstrap support is not
      first-order correct and insight is provided into the nature of the problem.
      Borrowing from earlier results, it is also shown that similar results hold when
      the neighbor-joining algorithm is used. Examples suggest that bootstrap support
      is generally conservative as a P value and give insight as to why this is usually
      the case. The analysis indicates that the problem is largely due to the unusual
      nature of tree space where boundary trees always have at least 2 neighbors.
FAU - Susko, Edward
AU  - Susko E
AD  - Department of Mathematics and Statistics, Dalhousie University, Halifax, Nova
      Scotia, Canada. susko@mathstat.dal.ca
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090701
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - *Phylogeny
MH  - *Probability
EDAT- 2009/04/01 00:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/04/01 00:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp016 [pii]
AID - 10.1093/sysbio/syp016 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Apr;58(2):211-23. doi: 10.1093/sysbio/syp016. Epub 2009 Jul 1.

PMID- 20525578
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 2
DP  - 2009 Apr
TI  - Statistical comparison of nucleotide, amino acid, and codon substitution models
      for evolutionary analysis of protein-coding sequences.
PG  - 199-210
LID - 10.1093/sysbio/syp015 [doi]
AB  - Statistical models for the evolution of molecular sequences play an important
      role in the study of evolutionary processes. For the evolutionary analysis of
      protein-coding sequences, 3 types of evolutionary models are available: 1)
      nucleotide, 2) amino acid, and 3) codon substitution models. Selecting
      appropriate models can greatly improve the estimation of phylogenies and
      divergence times and the detection of positive selection. Although much attention
      has been paid to the comparisons among the same types of models, relatively
      little attention has been paid to the comparisons among the different types of
      models. Additionally, because such models have different data structures,
      comparison of those models using conventional model selection criteria such as
      Akaike information criterion (AIC) or Bayesian information criterion (BIC) is not
      straightforward. Here, we suggest new procedures to convert models of the
      above-mentioned 3 types to 64-dimensional models with nucleotide triplet
      substitution. These conversion procedures render it possible to statistically
      compare the models of these 3 types by using AIC or BIC. By analyzing divergent
      and conserved interspecific mammalian sequences and intraspecific human
      population data, we show the superiority of the codon substitution models and
      discuss the advantages and disadvantages of the models of the 3 types.
FAU - Seo, Tae-Kun
AU  - Seo TK
AD  - Professional Programme for Agricultural Bioinformatics, University of Tokyo,
      1-1-1 Yayoi Bunkyo-Ku, Tokyo, Japan. seo@iu.a.u-tokyo.ac.jp
FAU - Kishino, Hirohisa
AU  - Kishino H
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090629
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Evolution, Molecular
MH  - *Models, Genetic
MH  - *Models, Statistical
EDAT- 2009/04/01 00:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/04/01 00:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp015 [pii]
AID - 10.1093/sysbio/syp015 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Apr;58(2):199-210. doi: 10.1093/sysbio/syp015. Epub 2009 Jun 29.

PMID- 20525577
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 2
DP  - 2009 Apr
TI  - Homoplasy and clade support.
PG  - 184-98
LID - 10.1093/sysbio/syp019 [doi]
AB  - Distinguishing phylogenetic signal from homoplasy (shared similarities among taxa
      that do not arise by common ancestry) is an implicit goal of any phylogenetic
      study. Large amounts of homoplasy can interfere with accurate tree inference, and
      it is expected that common measures of clade support, including bootstrap
      proportions and Bayesian posterior probabilities, should also be impacted to some
      degree by homoplasy. Through data simulation and analysis of 38 empirical data
      sets, we show that high amounts of homoplasy will affect all measures of clade
      support in a manner that is dependent on clade size. More specifically, the
      smallest taxon bipartitions in an unrooted tree topology will receive higher
      support relative to clades of intermediate sizes, even when all clades are
      supported by the same amount of data. We determine that the ultimate causes of
      this effect are the inclusion of random trees (due to homoplasy) during bootstrap
      resampling and Markov chain Monte Carlo (MCMC) topology searching and the higher 
      relative proportion of small taxon bipartitions (i.e., 2 or 3 taxa) to larger
      sized bipartitions. However, the use of explicit model-based methods, especially 
      Bayesian MCMC methods, effectively overcomes this clade size effect even when
      very small amounts of phylogenetic signal are present. We develop a post hoc
      statistic, the clade disparity index (CDI), to measure both the relative
      magnitude of the clade size effect and its statistical significance. In analyses 
      of both simulated and empirical data, CDI values indicate that Bayesian MCMC
      analyses are substantially more likely to estimate clade support values that are 
      uncorrelated with clade size than are maximum parsimony and maximum likelihood
      bootstrap analyses and thus less affected by homoplasy. These results may be
      especially relevant to "deep" phylogenetic problems, such as reconstructing the
      tree of life, as they represent the largest possible extremes of time and
      evolutionary rates, 2 factors that cause homoplasy.
FAU - Brandley, Matthew C
AU  - Brandley MC
AD  - Museum of Vertebrate Zoology and Department of Integrative Biology, University of
      California, Berkeley, CA 94720-3160, USA.
FAU - Warren, Dan L
AU  - Warren DL
FAU - Leache, Adam D
AU  - Leache AD
FAU - McGuire, Jimmy A
AU  - McGuire JA
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090629
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - Models, Genetic
MH  - *Phylogeny
EDAT- 2009/04/01 00:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/04/01 00:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp019 [pii]
AID - 10.1093/sysbio/syp019 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Apr;58(2):184-98. doi: 10.1093/sysbio/syp019. Epub 2009 Jun 29.

PMID- 20525576
OWN - NLM
STAT- MEDLINE
DCOM- 20100914
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 2
DP  - 2009 Apr
TI  - A method for investigating relative timing information on phylogenetic trees.
PG  - 167-83
LID - 10.1093/sysbio/syp018 [doi]
AB  - In this paper, we present a new way to describe the timing of branching events in
      phylogenetic trees. Our description is in terms of the relative timing of
      diversification events between sister clades; as such it is complementary to
      existing methods using lineages-through-time plots which consider diversification
      in aggregate. The method can be applied to look for evidence of diversification
      happening in lineage-specific "bursts", or the opposite, where diversification
      between 2 clades happens in an unusually regular fashion. In order to be able to 
      distinguish interesting events from stochasticity, we discuss 2 classes of
      neutral models on trees with relative timing information and develop a
      statistical framework for testing these models. These model classes include both 
      the coalescent with ancestral population size variation and global rate
      speciation-extinction models. We end the paper with 2 example applications:
      first, we show that the evolution of the hepatitis C virus deviates from the
      coalescent with arbitrary population size. Second, we analyze a large tree of
      ants, demonstrating that a period of elevated diversification rates does not
      appear to have occurred in a bursting manner.
FAU - Ford, Daniel
AU  - Ford D
AD  - Google Inc., 1600 Amphitheatre Parkway, Mountain View, CA 94043, USA.
FAU - Matsen, Frederick A
AU  - Matsen FA
FAU - Stadler, Tanja
AU  - Stadler T
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090612
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - *Time
EDAT- 2009/04/01 00:00
MHDA- 2010/09/15 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/04/01 00:00 [pubmed]
PHST- 2010/09/15 06:00 [medline]
AID - syp018 [pii]
AID - 10.1093/sysbio/syp018 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Apr;58(2):167-83. doi: 10.1093/sysbio/syp018. Epub 2009 Jun 12.

PMID- 20525575
OWN - NLM
STAT- MEDLINE
DCOM- 20100819
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 1
DP  - 2009 Feb
TI  - Why would phylogeneticists ignore computerized sequence alignment?
PG  - 150-8
LID - 10.1093/sysbio/syp009 [doi]
FAU - Morrison, David A
AU  - Morrison DA
LA  - eng
PT  - Journal Article
DEP - 20090325
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - Bryophyta/classification/genetics
MH  - Classification/*methods
MH  - *Phylogeny
MH  - Plants/classification/genetics
MH  - *Sequence Alignment
MH  - *Sequence Analysis, DNA
EDAT- 2009/02/01 00:00
MHDA- 2010/08/20 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/02/01 00:00 [pubmed]
PHST- 2010/08/20 06:00 [medline]
AID - syp009 [pii]
AID - 10.1093/sysbio/syp009 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Feb;58(1):150-8. doi: 10.1093/sysbio/syp009. Epub 2009 Mar 25.

PMID- 20525574
OWN - NLM
STAT- MEDLINE
DCOM- 20100819
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 1
DP  - 2009 Feb
TI  - Statistical methods for studying modularity: a reply to Mitteroecker and
      Bookstein.
PG  - 146-9
LID - 10.1093/sysbio/syp007 [doi]
FAU - Magwene, Paul M
AU  - Magwene PM
LA  - eng
PT  - Comment
PT  - Letter
DEP - 20090522
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2007 Oct;56(5):818-36. PMID: 17934997
CIN - Syst Biol. 2009 Jun;58(3):346-8. PMID: 20525588
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Models, Genetic
MH  - *Models, Statistical
MH  - Phylogeny
EDAT- 2009/02/01 00:00
MHDA- 2010/08/20 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/02/01 00:00 [pubmed]
PHST- 2010/08/20 06:00 [medline]
AID - syp007 [pii]
AID - 10.1093/sysbio/syp007 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Feb;58(1):146-9. doi: 10.1093/sysbio/syp007. Epub 2009 May 22.

PMID- 20525573
OWN - NLM
STAT- MEDLINE
DCOM- 20100819
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 1
DP  - 2009 Feb
TI  - The effect of ambiguous data on phylogenetic estimates obtained by maximum
      likelihood and Bayesian inference.
PG  - 130-45
LID - 10.1093/sysbio/syp017 [doi]
AB  - Although an increasing number of phylogenetic data sets are incomplete, the
      effect of ambiguous data on phylogenetic accuracy is not well understood. We use 
      4-taxon simulations to study the effects of ambiguous data (i.e., missing
      characters or gaps) in maximum likelihood (ML) and Bayesian frameworks. By
      introducing ambiguous data in a way that removes confounding factors, we provide 
      the first clear understanding of 1 mechanism by which ambiguous data can mislead 
      phylogenetic analyses. We find that in both ML and Bayesian frameworks,
      among-site rate variation can interact with ambiguous data to produce misleading 
      estimates of topology and branch lengths. Furthermore, within a Bayesian
      framework, priors on branch lengths and rate heterogeneity parameters can
      exacerbate the effects of ambiguous data, resulting in strongly misleading
      bipartition posterior probabilities. The magnitude and direction of the ambiguous
      data bias are a function of the number and taxonomic distribution of ambiguous
      characters, the strength of topological support, and whether or not the model is 
      correctly specified. The results of this study have major implications for all
      analyses that rely on accurate estimates of topology or branch lengths, including
      divergence time estimation, ancestral state reconstruction, tree-dependent
      comparative methods, rate variation analysis, phylogenetic hypothesis testing,
      and phylogeographic analysis.
FAU - Lemmon, Alan R
AU  - Lemmon AR
AD  - Section of Integrative Biology, University of Texas at Austin, 1 University
      Station C0930, Austin, TX 78712, USA. alemmon@evotutor.org
FAU - Brown, Jeremy M
AU  - Brown JM
FAU - Stanger-Hall, Kathrin
AU  - Stanger-Hall K
FAU - Lemmon, Emily Moriarty
AU  - Lemmon EM
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090522
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Urodela/*classification/genetics
EDAT- 2009/02/01 00:00
MHDA- 2010/08/20 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/02/01 00:00 [pubmed]
PHST- 2010/08/20 06:00 [medline]
AID - syp017 [pii]
AID - 10.1093/sysbio/syp017 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Feb;58(1):130-45. doi: 10.1093/sysbio/syp017. Epub 2009 May 22.

PMID- 20525572
OWN - NLM
STAT- MEDLINE
DCOM- 20100819
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 1
DP  - 2009 Feb
TI  - Gene trees reveal repeated instances of mitochondrial DNA introgression in
      orangethroat darters (percidae: etheostoma).
PG  - 114-29
LID - 10.1093/sysbio/syp014 [doi]
AB  - Phylogenies of closely related animal species are often inferred using
      mitochondrial DNA (mtDNA) gene sequences. The accuracy of mtDNA gene trees is
      compromised through hybridization that leads to introgression of mitochondrial
      genomes. Using DNA sequences from 6 single-copy nuclear genes and 2 regions of
      the mitochondrial genome, we investigated the temporal and geographic signature
      of mitochondrial and nuclear introgression in the Etheostoma spectabile darter
      clade. Phylogenetic analyses of the nuclear genes result in the monophyly of the 
      E. spectabile clade; however, with respect to sampled specimens of 5 species
      (Etheostoma fragi, Etheostoma uniporum, Etheostoma pulchellum, Etheostoma burri, 
      and E. spectabile), the mitochondrial phylogeny is inconsistent with E.
      spectabile clade monophyly. Etheostoma uniporum and E. fragi are both fixed for
      heterospecific mitochondrial genomes. Limited nuclear introgression is restricted
      to E. uniporum. Our analyses show that the pattern of introgression is
      consistently asymmetric, with movement of heterospecific mitochondrial haplotypes
      and nuclear alleles into E. spectabile clade species; introgressive hybridization
      spans broad temporal scales; and introgression is restricted to species and
      populations in the Ozarks. The introgressed mitochondrial genome observed in E.
      fragi has an obscure phylogenetic placement among darters, an ancient age, and is
      possibly a mitochondrial fossil from an Etheostoma species that has subsequently 
      gone extinct. These results indicate that introgression, both ancient and more
      contemporaneous, characterizes the history of diversification in the E.
      spectabile species clade and may be relatively common among clades comprising the
      species-rich North American freshwater fauna.
FAU - Bossu, Christen M
AU  - Bossu CM
AD  - Department of Ecology and Evolutionary Biology, University of Tennessee,
      Knoxville, TN 37917, USA.
FAU - Near, Thomas J
AU  - Near TJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090522
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - DNA, Mitochondrial/*genetics
MH  - North America
MH  - Perches/*classification/*genetics
MH  - Phylogeny
EDAT- 2009/02/01 00:00
MHDA- 2010/08/20 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/02/01 00:00 [pubmed]
PHST- 2010/08/20 06:00 [medline]
AID - syp014 [pii]
AID - 10.1093/sysbio/syp014 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Feb;58(1):114-29. doi: 10.1093/sysbio/syp014. Epub 2009 May 22.

PMID- 20525571
OWN - NLM
STAT- MEDLINE
DCOM- 20100819
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 1
DP  - 2009 Feb
TI  - Measuring branch support in species trees obtained by gene tree parsimony.
PG  - 100-13
LID - 10.1093/sysbio/syp013 [doi]
AB  - Several methods have recently been developed that allow the reconstruction of
      species trees from gene trees, an important achievement in our ongoing quest to
      obtain reliable species phylogenies. However, considerably less attention has
      been given to evaluating the accuracy of species trees' estimates. Four methods
      for measuring branch support of species trees are tested in this study in a gene 
      tree parsimony framework: 1) bootstrap lineages (BL) (sequences) within species, 
      2) bootstrap characters (BC) within genes (i.e., the standard nonparametric
      bootstrap), 3) bootstrap lineages and characters (BLC), and 4) posterior
      probability gene tree sampling (PPGTS) (where, for each resampled data set, gene 
      trees are sampled according to their posterior probability). For each method, n
      species trees are reconstructed from n resampled data sets and the branch support
      consists in the percentage of the n species trees in which a branch is recovered.
      The 4 methods were tested for several species trees and for different sampling
      efforts (i.e., number of genes and individuals sampled) using coalescent
      simulations. PPGTS performed best overall with lowest Type I and II error rates, 
      followed by BLC. The BL and BC methods had higher error rates. This suggests that
      in order to properly measure branch support in a species tree context, it is
      important to account for the uncertainty involved in reconstructing gene trees
      from DNA sequences as well as that involved in reconstructing the species tree
      from individual gene trees. With the parameters used in the simulations, sampling
      more individuals per species resulted in similar improvements in support values
      as when sampling more genes. Moreover, sampling more individuals per species
      appeared to be important for escaping the anomaly zone present when only 1
      sequence was sampled. We also apply the 4 methods to obtain branch supports for
      the species phylogeny of diploid wild roses (Rosa) in North America.
FAU - Joly, Simon
AU  - Joly S
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, Massey University,
      Palmerston North, New Zealand. simon.joly@mail.mcgill.ca
FAU - Bruneau, Anne
AU  - Bruneau A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090525
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - DNA, Plant/analysis
MH  - Diploidy
MH  - Humans
MH  - *Models, Genetic
MH  - North America
MH  - *Phylogeny
MH  - Rosa/*classification/*genetics
EDAT- 2009/02/01 00:00
MHDA- 2010/08/20 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/02/01 00:00 [pubmed]
PHST- 2010/08/20 06:00 [medline]
AID - syp013 [pii]
AID - 10.1093/sysbio/syp013 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Feb;58(1):100-13. doi: 10.1093/sysbio/syp013. Epub 2009 May 25.

PMID- 20525570
OWN - NLM
STAT- MEDLINE
DCOM- 20100819
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 1
DP  - 2009 Feb
TI  - Paleontology, genomics, and combined-data phylogenetics: can molecular data
      improve phylogeny estimation for fossil taxa?
PG  - 87-99
LID - 10.1093/sysbio/syp012 [doi]
AB  - The genomics revolution offers great promise for resolving the phylogeny of
      living taxa, but does it offer any benefits for reconstructing relationships
      among extinct (fossil) taxa? Superficially, the answer would seem to be "no,"
      given that molecular data cannot be obtained for most fossil taxa. However,
      because fossil taxa often interdigitate among living taxa on the Tree of Life,
      molecular data may indirectly enhance phylogenetic accuracy for fossil taxa in
      the context of a combined analysis of morphological and molecular data for living
      and fossil taxa. Here, I use simulations to assess accuracy for fossil taxa in a 
      mixed analysis of living and fossil taxa, before and after addition of molecular 
      data to the living taxa. The results show conditions where the accuracy for
      fossil taxa is greatly increased by adding molecular data, sometimes by as much
      as 100%. In other cases, the increase is negligible, such as when fossil taxa
      greatly outnumber living taxa in the analysis. However, there were few cases
      where accuracy was significantly decreased by the addition of the molecular data,
      suggesting that this practice may range from highly beneficial to mostly
      harmless. Overall, the results suggest that improvements in molecular
      phylogenetics can potentially benefit phylogeny reconstruction for fossil taxa.
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Department of Ecology and Evolution, Stony Brook University, Stony Brook, NY
      11794-5245, USA. wiensj@life.bio.sunysb.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090519
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification
MH  - *Fossils
MH  - *Genomics
MH  - *Phylogeny
EDAT- 2009/02/01 00:00
MHDA- 2010/08/20 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/02/01 00:00 [pubmed]
PHST- 2010/08/20 06:00 [medline]
AID - syp012 [pii]
AID - 10.1093/sysbio/syp012 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Feb;58(1):87-99. doi: 10.1093/sysbio/syp012. Epub 2009 May 19.

PMID- 20525569
OWN - NLM
STAT- MEDLINE
DCOM- 20100819
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 1
DP  - 2009 Feb
TI  - Species as ranked taxa.
PG  - 74-86
LID - 10.1093/sysbio/syp011 [doi]
AB  - Because species names play an important role in scientific communication, it is
      more important that species be understood to be taxa than that they be equated
      with functional ecological or evolutionary entities. Although most biologists
      would agree that taxa are composed of organisms that share a unique common
      history, 2 major challenges remain in developing a species-as-taxa concept.
      First, grouping: in the face of genealogical discordance at all levels in the
      taxonomic hierarchy, how can we understand the nature of taxa? Second, ranking:
      what criteria should be used to designate certain taxa in a nested series as
      being species? The grouping problem can be solved by viewing taxa as exclusive
      groups of organisms- sets of organisms that form a clade for a plurality of the
      genome (more than any conflicting set). However, no single objective criterion of
      species rank can be proposed. Instead, the species rank should be assigned by
      practitioners based on the semisubjective application of a set of species-ranking
      criteria. Although these criteria can be designed to yield species taxa that
      approximately match the ecological, evolutionary, and morphological entities that
      taxonomists have traditionally associated with the species rank, such a
      correspondence cannot be enforced without undermining the assumption that species
      are taxa. The challenge and art of monography is to use genealogical and other
      kinds of data to assign all organisms to one and only one species-ranked taxon.
      Various implications of the species-as-ranked-taxa view are discussed, including 
      the synchronic nature of taxa, fossil species, the treatment of hybrids, and
      species nomenclature. I conclude that, although challenges remain, adopting the
      view that species are ranked taxa will facilitate a much-needed revolution in
      taxonomy that will allow it to better serve the biodiversity informatic needs of 
      the 21st century.
FAU - Baum, David A
AU  - Baum DA
AD  - Department of Botany, University of Wisconsin-Madison, 430 Lincoln Drive,
      Madison, WI 53706, USA. dbaum@wisc.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090519
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Classification
MH  - Fossils
MH  - Genetic Speciation
MH  - Humans
MH  - Hybridization, Genetic
MH  - Phylogeny
EDAT- 2009/02/01 00:00
MHDA- 2010/08/20 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/02/01 00:00 [pubmed]
PHST- 2010/08/20 06:00 [medline]
AID - syp011 [pii]
AID - 10.1093/sysbio/syp011 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Feb;58(1):74-86. doi: 10.1093/sysbio/syp011. Epub 2009 May 19.

PMID- 20525568
OWN - NLM
STAT- MEDLINE
DCOM- 20100819
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 1
DP  - 2009 Feb
TI  - Disentangling reticulate evolution in an arctic-alpine polyploid complex.
PG  - 55-73
LID - 10.1093/sysbio/syp010 [doi]
AB  - Although polyploidy plays a fundamental role in plant evolution, the elucidation 
      of polyploid origins is fraught with methodological challenges. For example,
      allopolyploid species may confound phylogenetic reconstruction because commonly
      used methods are designed to trace divergent, rather than reticulate patterns.
      Recently developed techniques of phylogenetic network estimation allow for a more
      effective identification of incongruence among trees. However, incongruence can
      also be caused by incomplete lineage sorting, paralogy, concerted evolution, and 
      recombination. Thus, initial hypotheses of hybridization need to be examined via 
      additional sources of evidence, including the partitioning of infraspecific
      genetic polymorphisms, morphological characteristics, chromosome numbers,
      crossing experiments, and distributional patterns. Primula sect. Aleuritia
      subsect. Aleuritia (Aleuritia) represents an ideal case study to examine
      reticulation because specific hypotheses have been derived from morphology,
      karyology, interfertility, and distribution to explain the observed variation of 
      ploidy levels, ranging from diploidy to 14-ploidy. Sequences from 5 chloroplast
      and 1 nuclear ribosomal DNA (nrDNA) markers were analyzed to generate the
      respective phylogenies and consensus networks. Furthermore, extensive cloning of 
      the nrDNA marker allowed for the identification of shared nucleotides at
      polymorphic sites, investigation of infraspecific genetic polymorphisms via
      principal coordinate analyses PCoAs, and detection of recombination between
      putative progenitor sequences. The results suggest that most surveyed polyploids 
      originated via hybridization and that 2 taxonomic species formed recurrently from
      different progenitors, findings that are congruent with the expectations of
      speciation via secondary contact. Overall, the study highlights the importance of
      using multiple experimental and analytical approaches to disentangle complex
      patterns of reticulation.
FAU - Guggisberg, Alessia
AU  - Guggisberg A
AD  - Institut fur Systematische Botanik, Universitat Zurich, Zollikerstrasse 107,
      CH-8008 Zurich, Switzerland. alessiag@interchange.ubc.ca
FAU - Mansion, Guilhem
AU  - Mansion G
FAU - Conti, Elena
AU  - Conti E
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090604
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - *Biological Evolution
MH  - DNA, Plant/genetics
MH  - Phylogeny
MH  - Polymerase Chain Reaction
MH  - Polyploidy
MH  - Primula/classification/*genetics
EDAT- 2009/02/01 00:00
MHDA- 2010/08/20 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/02/01 00:00 [pubmed]
PHST- 2010/08/20 06:00 [medline]
AID - syp010 [pii]
AID - 10.1093/sysbio/syp010 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Feb;58(1):55-73. doi: 10.1093/sysbio/syp010. Epub 2009 Jun 4.

PMID- 20525567
OWN - NLM
STAT- MEDLINE
DCOM- 20100819
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 1
DP  - 2009 Feb
TI  - Properties of consensus methods for inferring species trees from gene trees.
PG  - 35-54
LID - 10.1093/sysbio/syp008 [doi]
AB  - Consensus methods provide a useful strategy for summarizing information from a
      collection of gene trees. An important application of consensus methods is to
      combine gene trees to estimate a species tree. To investigate the theoretical
      properties of consensus trees that would be obtained from large numbers of loci
      evolving according to a basic evolutionary model, we construct consensus trees
      from rooted gene trees that occur in proportion to gene-tree probabilities
      derived from coalescent theory. We consider majority-rule, rooted triple (R(*)), 
      and greedy consensus trees obtained from known, rooted gene trees, both in the
      asymptotic case as numbers of gene trees approach infinity and for finite numbers
      of genes. Our results show that for some combinations of species-tree branch
      lengths, increasing the number of independent loci can make the rooted
      majority-rule consensus tree more likely to be at least partially unresolved.
      However, the probability that the R(*) consensus tree has the species-tree
      topology approaches 1 as the number of gene trees approaches infinity. Although
      the greedy consensus algorithm can be the quickest to converge on the correct
      species-tree topology when increasing the number of gene trees, it can also be
      positively misleading. The majority-rule consensus tree is not a misleading
      estimator of the species-tree topology, and the R(*) consensus tree is a
      statistically consistent estimator of the species-tree topology. Our results
      therefore suggest a method for using multiple loci to infer the species-tree
      topology, even when it is discordant with the most likely gene tree.
FAU - Degnan, James H
AU  - Degnan JH
AD  - Department of Human Genetics, 1241 East Catherine Street, University of Michigan,
      Ann Arbor, MI 48109-0618, USA. J.Degnan@math.canterbury.ac.nz
FAU - DeGiorgio, Michael
AU  - DeGiorgio M
FAU - Bryant, David
AU  - Bryant D
FAU - Rosenberg, Noah A
AU  - Rosenberg NA
LA  - eng
GR  - T32 GM070449/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090604
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Biological Evolution
MH  - Computational Biology/*methods
MH  - Genetic Speciation
MH  - *Models, Genetic
MH  - *Phylogeny
PMC - PMC2909780
EDAT- 2009/02/01 00:00
MHDA- 2010/08/20 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/02/01 00:00 [pubmed]
PHST- 2010/08/20 06:00 [medline]
AID - syp008 [pii]
AID - 10.1093/sysbio/syp008 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Feb;58(1):35-54. doi: 10.1093/sysbio/syp008. Epub 2009 Jun 4.

PMID- 20525566
OWN - NLM
STAT- MEDLINE
DCOM- 20100819
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 1
DP  - 2009 Feb
TI  - A Monte Carlo approach successfully identifies randomness in multiple sequence
      alignments: a more objective means of data exclusion.
PG  - 21-34
LID - 10.1093/sysbio/syp006 [doi]
AB  - Random similarity of sequences or sequence sections can impede phylogenetic
      analyses or the identification of gene homologies. Additionally, randomly similar
      sequences or ambiguously aligned sequence sections can negatively interfere with 
      the estimation of substitution model parameters. Phylogenomic studies have shown 
      that biases in model estimation and tree reconstructions do not disappear even
      with large data sets. In fact, these biases can become pronounced with more data.
      It is therefore important to identify possible random similarity within sequence 
      alignments in advance of model estimation and tree reconstructions. Different
      approaches have been already suggested to identify and treat problematic
      alignment sections. We propose an alternative method that can identify random
      similarity within multiple sequence alignments (MSAs) based on Monte Carlo
      resampling within a sliding window. The method infers similarity profiles from
      pairwise sequence comparisons and subsequently calculates a consensus profile.
      This consensus profile represents a summary of all calculated single similarity
      profiles. In consequence, consensus profiles identify dominating patterns of
      nonrandom similarity or randomness within sections of MSAs. We show that the
      approach clearly identifies randomness in simulated and real data. After the
      exclusion of putative random sections, node support drastically improves in tree 
      reconstructions of both data. It thus appears to be a powerful tool to identify
      possible biases of tree reconstructions or gene identification. The method is
      currently restricted to nucleotide data but will be extended to protein data in
      the near future.
FAU - Misof, Bernhard
AU  - Misof B
AD  - Zoologisches Forschungsmuseum Alexander Koenig, Molecular Biology Unit,
      Adenauerallee 160, 53113 Bonn, Germany. bernhard.misof@uni-hamburg.de
FAU - Misof, Katharina
AU  - Misof K
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20090520
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Base Sequence
MH  - *Models, Genetic
MH  - *Monte Carlo Method
MH  - *Phylogeny
MH  - Sequence Alignment/*methods
MH  - Software
EDAT- 2009/02/01 00:00
MHDA- 2010/08/20 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/02/01 00:00 [pubmed]
PHST- 2010/08/20 06:00 [medline]
AID - syp006 [pii]
AID - 10.1093/sysbio/syp006 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Feb;58(1):21-34. doi: 10.1093/sysbio/syp006. Epub 2009 May 20.

PMID- 20525565
OWN - NLM
STAT- MEDLINE
DCOM- 20100819
LR  - 20110805
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 58
IP  - 1
DP  - 2009 Feb
TI  - Conflicting mitochondrial and nuclear phylogenies for the widely disjunct Emys
      (Testudines: Emydidae) species complex, and what they tell us about biogeography 
      and hybridization.
PG  - 1-20
LID - 10.1093/sysbio/syp005 [doi]
AB  - Understanding the mechanisms by which widely disjunct members of a clade came to 
      occupy their current distribution is one of the fundamental challenges of
      biogeography. Here, we used data from 7 nuclear and 1 mitochondrial gene to
      examine the phylogenetic and biogeographic history of Emys, a clade of turtles
      that is broadly disjunct in western and eastern North America and Europe. We
      found strong disagreement between mitochondrial and nuclear gene trees, with
      mitochondrial DNA supporting the monophyly of the North American taxa (marmorata 
      + blandingii) to the exclusion of the European orbicularis, and nuclear genes
      supporting the monophyly of (blandingii + orbicularis) to the exclusion of
      marmorata. We used fossil-calibrated molecular chronograms, in combination with
      supporting evidence from the fossil record and paleoclimatology, to identify a
      potential example of ancient hybridization and mitochondrial gene capture 12
      million years ago, which explains this discrepancy. Based on the weight of
      evidence, we argue that the invasion of Eurasia by Emys orbicularis occurred
      about 16 Ma via a trans-Beringian land bridge. The case of Emys emphasizes how
      single-gene trees can be strongly affected by population processes, including
      hybridization, and that the effects of these processes can persist through long
      periods of evolutionary history. Given the chaotic state of the current taxonomy 
      of these turtles, our work also emphasizes the care that should be used in
      implementing taxonomic changes based on 1 or a few gene trees and the importance 
      of taking a conservative approach in renaming or splitting higher taxa based on
      apparent nonmonophyly.
FAU - Spinks, Phillip Q
AU  - Spinks PQ
AD  - Department of Evolution and Ecology and Center for Population Biology, University
      of California, Davis, CA 95616, USA. pqspinks@ucdavis.edu
FAU - Shaffer, H Bradley
AU  - Shaffer HB
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20090528
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Cell Nucleus/genetics
MH  - DNA, Mitochondrial/*genetics
MH  - Phylogeny
MH  - Turtles/*classification/*genetics
EDAT- 2009/02/01 00:00
MHDA- 2010/08/20 06:00
CRDT- 2010/06/08 06:00
PHST- 2010/06/08 06:00 [entrez]
PHST- 2009/02/01 00:00 [pubmed]
PHST- 2010/08/20 06:00 [medline]
AID - syp005 [pii]
AID - 10.1093/sysbio/syp005 [doi]
PST - ppublish
SO  - Syst Biol. 2009 Feb;58(1):1-20. doi: 10.1093/sysbio/syp005. Epub 2009 May 28.

PMID- 19085335
OWN - NLM
STAT- MEDLINE
DCOM- 20090202
LR  - 20081216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 6
DP  - 2008 Dec
TI  - Consistent estimation of divergence times in phylogenetic trees with local
      molecular clocks.
PG  - 947-54
LID - 10.1080/10635150802562400 [doi]
FAU - Svennblad, Bodil
AU  - Svennblad B
AD  - Department of Mathematics, Uppsala University, Uppsala, Sweden.
      bodil.svennblad@ucr.uu.se
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Genetic Variation
MH  - *Phylogeny
MH  - Statistics as Topic
MH  - Time Factors
EDAT- 2008/12/17 09:00
MHDA- 2009/02/03 09:00
CRDT- 2008/12/17 09:00
PHST- 2008/12/17 09:00 [entrez]
PHST- 2008/12/17 09:00 [pubmed]
PHST- 2009/02/03 09:00 [medline]
AID - 906576189 [pii]
AID - 10.1080/10635150802562400 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Dec;57(6):947-54. doi: 10.1080/10635150802562400.

PMID- 19085334
OWN - NLM
STAT- MEDLINE
DCOM- 20090202
LR  - 20081216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 6
DP  - 2008 Dec
TI  - Filtered Z-closure supernetworks for extracting and visualizing recurrent signal 
      from incongruent gene trees.
PG  - 939-47
LID - 10.1080/10635150802552849 [doi]
FAU - Whitfield, James B
AU  - Whitfield JB
AD  - Department of Entomology, University of Illinois, Urbana, Illinois 61801, USA.
      jwhitfie@life.uiuc.edu
FAU - Cameron, Sydney A
AU  - Cameron SA
FAU - Huson, Daniel H
AU  - Huson DH
FAU - Steel, Mike A
AU  - Steel MA
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bees/classification/genetics
MH  - Classification/*methods
MH  - *Phylogeny
MH  - Viruses/classification/genetics
MH  - Wasps/classification/genetics/virology
EDAT- 2008/12/17 09:00
MHDA- 2009/02/03 09:00
CRDT- 2008/12/17 09:00
PHST- 2008/12/17 09:00 [entrez]
PHST- 2008/12/17 09:00 [pubmed]
PHST- 2009/02/03 09:00 [medline]
AID - 906546725 [pii]
AID - 10.1080/10635150802552849 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Dec;57(6):939-47. doi: 10.1080/10635150802552849.

PMID- 19085333
OWN - NLM
STAT- MEDLINE
DCOM- 20090202
LR  - 20081216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 57
IP  - 6
DP  - 2008 Dec
TI  - Resolving arthropod phylogeny: exploring phylogenetic signal within 41 kb of
      protein-coding nuclear gene sequence.
PG  - 920-38
LID - 10.1080/10635150802570791 [doi]
AB  - This study attempts to resolve relationships among and within the four basal
      arthropod lineages (Pancrustacea, Myriapoda, Euchelicerata, Pycnogonida) and to
      assess the widespread expectation that remaining phylogenetic problems will yield
      to increasing amounts of sequence data. Sixty-eight regions of 62 protein-coding 
      nuclear genes (approximately 41 kilobases (kb)/taxon) were sequenced for 12
      taxonomically diverse arthropod taxa and a tardigrade outgroup. Parsimony,
      likelihood, and Bayesian analyses of total nucleotide data generally strongly
      supported the monophyly of each of the basal lineages represented by more than
      one species. Other relationships within the Arthropoda were also supported, with 
      support levels depending on method of analysis and inclusion/exclusion of
      synonymous changes. Removing third codon positions, where the assumption of base 
      compositional homogeneity was rejected, altered the results. Removing the final
      class of synonymous mutations--first codon positions encoding leucine and
      arginine, which were also compositionally heterogeneous--yielded a data set that 
      was consistent with a hypothesis of base compositional homogeneity. Furthermore, 
      under such a data-exclusion regime, all 68 gene regions individually were
      consistent with base compositional homogeneity. Restricting likelihood analyses
      to nonsynonymous change recovered trees with strong support for the basal
      lineages but not for other groups that were variably supported with more
      inclusive data sets. In a further effort to increase phylogenetic signal, three
      types of data exploration were undertaken. (1) Individual genes were ranked by
      their average rate of nonsynonymous change, and three rate categories were
      assigned--fast, intermediate, and slow. Then, bootstrap analysis of each gene was
      performed separately to see which taxonomic groups received strong support. Five 
      taxonomic groups were strongly supported independently by two or more genes, and 
      these genes mostly belonged to the slow or intermediate categories, whereas
      groups supported only by a single gene region tended to be from genes of the fast
      category, arguing that fast genes provide a less consistent signal. (2) A
      sensitivity analysis was performed in which increasing numbers of genes were
      excluded, beginning with the fastest. The number of strongly supported nodes
      increased up to a point and then decreased slightly. Recovery of Hexapoda
      required removal of fast genes. Support for Mandibulata (Pancrustacea +
      Myriapoda) also increased, at times to "strong" levels, with removal of the
      fastest genes. (3) Concordance selection was evaluated by clustering genes
      according to their ability to recover Pancrustacea, Euchelicerata, or Myriapoda
      and analyzing the three clusters separately. All clusters of genes recovered the 
      three concordance clades but were at times inconsistent in the relationships
      recovered among and within these clades, a result that indicates that the a
      priori concordance criteria may bias phylogenetic signal in unexpected ways. In a
      further attempt to increase support of taxonomic relationships, sequence data
      from 49 additional taxa for three slow genes (i.e., EF-1 alpha, EF-2, and Pol II)
      were combined with the various 13-taxon data sets. The 62-taxon analyses
      supported the results of the 13-taxon analyses and provided increased support for
      additional pancrustacean clades found in an earlier analysis including only EF-1 
      alpha, EF-2, and Pol II.
FAU - Regier, Jerome C
AU  - Regier JC
AD  - Center for Biosystems Research, University of Maryland Biotechnology Institute,
      College Park, Maryland 20742, USA. regier@umbi.umd.edu
FAU - Shultz, Jeffrey W
AU  - Shultz JW
FAU - Ganley, Austen R D
AU  - Ganley AR
FAU - Hussey, April
AU  - Hussey A
FAU - Shi, Diane
AU  - Shi D
FAU - Ball, Bernard
AU  - Ball B
FAU - Zwick, Andreas
AU  - Zwick A
FAU - Stajich, Jason E
AU  - Stajich JE
FAU - Cummings, Michael P
AU  - Cummings MP
FAU - Martin, Joel W
AU  - Martin JW
FAU - Cunningham, Clifford W
AU  - Cunningham CW
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Arthropods/*classification/*genetics
MH  - Base Composition/genetics
MH  - Cell Nucleus/genetics
MH  - Open Reading Frames/*genetics
MH  - *Phylogeny
EDAT- 2008/12/17 09:00
MHDA- 2009/02/03 09:00
CRDT- 2008/12/17 09:00
PHST- 2008/12/17 09:00 [entrez]
PHST- 2008/12/17 09:00 [pubmed]
PHST- 2009/02/03 09:00 [medline]
AID - 906542386 [pii]
AID - 10.1080/10635150802570791 [doi]
PST - ppublish
SO  - Syst Biol. 2008 Dec;57(6):920-38. doi: 10.1080/10635150802570791.
</pre></body></html>