<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0035)https://www.ncbi.nlm.nih.gov/pubmed -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"></head><body><pre>PMID- 16611605
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - Goodbye Gondwana? New Zealand biogeography, geology, and the problem of
      circularity.
PG  - 351-6
FAU - Waters, Jonathan M
AU  - Waters JM
AD  - Department of Zoology, University of Otago, PO Box 56, Dunedin, New Zealand.
      jonathan.waters@stonebow.otago.ac.nz
FAU - Craw, Dave
AU  - Craw D
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Anura/genetics
MH  - Fishes/genetics
MH  - *Genetic Speciation
MH  - *Geography
MH  - Geological Phenomena
MH  - *Geology
MH  - New Zealand
MH  - Palaeognathae/genetics
MH  - Plants/genetics
MH  - Reptiles/genetics
EDAT- 2006/04/14 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/04/14 09:00
PHST- 2006/04/14 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/04/14 09:00 [entrez]
AID - T0414P33V4026014 [pii]
AID - 10.1080/10635150600681659 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):351-6. doi: 10.1080/10635150600681659.

PMID- 16611604
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - Discriminating supported and unsupported relationships in supertrees using
      triplets.
PG  - 345-50
FAU - Cotton, James A
AU  - Cotton JA
AD  - Department of Zoology, The Natural History Museum, Cromwell Road, SW7 5BD,
      London, UK. james.cotton@nhm.ac.uk
FAU - Slater, Claire S C
AU  - Slater CS
FAU - Wilkinson, Mark
AU  - Wilkinson M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2006/04/14 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/04/14 09:00
PHST- 2006/04/14 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/04/14 09:00 [entrez]
AID - W47270541844Q198 [pii]
AID - 10.1080/10635150500481556 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):345-50. doi: 10.1080/10635150500481556.

PMID- 16611603
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - Utility of nuclear allele networks for the analysis of closely related species in
      the genus Carabus, subgenus Ohomopterus.
PG  - 329-44
AB  - Nuclear DNA sequence data for diploid organisms are potentially a rich source of 
      phylogenetic information for disentangling the evolutionary relationships of
      closely related organisms, but present special phylogenetic problems owing to
      difficulties arising from heterozygosity and recombination. We analyzed allelic
      relationships for two nuclear gene regions (phosphoenolpyruvate carboxykinase and
      elongation factor-1a), along with a mitochondrial gene region (NADH dehydrogenase
      subunit 5), for an assemblage of closely related species of carabid beetles
      (Carabus subgenus Ohomopterus). We used a network approach to examine whether the
      nuclear gene sequences provide substantial phylogenetic information on species
      relationships and evolutionary history. The mitochondrial gene genealogy strongly
      contradicted the morphological species boundary as a result of introgression of
      heterospecific mitochondria. Two nuclear gene regions showed high allelic
      diversity within species, and this diversity was partially attributable to
      recombination between various alleles and high variability in the intron region. 
      Shared nuclear alleles among species were rare and were considered to represent
      shared ancestral polymorphism. Despite the presence of recombination, nuclear
      allelic networks recovered species monophyly more often and presented genetic
      differentiation patterns (low to high) among species more clearly. Overall,
      nuclear gene networks provide clear evidence for separate biological species and 
      information on the phylogenetic relationships among closely related carabid
      beetles.
FAU - Sota, Teiji
AU  - Sota T
AD  - Department of Zoology, Graduate School of Science, Kyoto University, Sakyo,
      Kyoto, 606-8502, Japan. sota@terra.zool.kyoto-u.ac.jp
FAU - Sasabe, Masataka
AU  - Sasabe M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - *Alleles
MH  - Animals
MH  - Coleoptera/anatomy &amp; histology/*classification/*genetics
MH  - DNA/*genetics
MH  - Genetic Speciation
MH  - Genetic Variation
MH  - Male
MH  - Phylogeny
MH  - Recombination, Genetic
MH  - Species Specificity
EDAT- 2006/04/14 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/04/14 09:00
PHST- 2006/04/14 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/04/14 09:00 [entrez]
AID - V0LM73750847607N [pii]
AID - 10.1080/10635150500541607 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):329-44. doi: 10.1080/10635150500541607.

PMID- 16611602
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20131210
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - Multiple sequence alignment accuracy and phylogenetic inference.
PG  - 314-28
AB  - Phylogenies are often thought to be more dependent upon the specifics of the
      sequence alignment rather than on the method of reconstruction. Simulation of
      sequences containing insertion and deletion events was performed in order to
      determine the role that alignment accuracy plays during phylogenetic inference.
      Data sets were simulated for pectinate, balanced, and random tree shapes under
      different conditions (ultrametric equal branch length, ultrametric random branch 
      length, nonultrametric random branch length). Comparisons between hypothesized
      alignments and true alignments enabled determination of two measures of alignment
      accuracy, that of the total data set and that of individual branches. In general,
      our results indicate that as alignment error increases, topological accuracy
      decreases. This trend was much more pronounced for data sets derived from more
      pectinate topologies. In contrast, for balanced, ultrametric, equal branch length
      tree shapes, alignment inaccuracy had little average effect on tree
      reconstruction. These conclusions are based on average trends of many analyses
      under different conditions, and any one specific analysis, independent of the
      alignment accuracy, may recover very accurate or inaccurate topologies. Maximum
      likelihood and Bayesian, in general, outperformed neighbor joining and maximum
      parsimony in terms of tree reconstruction accuracy. Results also indicated that
      as the length of the branch and of the neighboring branches increase, alignment
      accuracy decreases, and the length of the neighboring branches is the major
      factor in topological accuracy. Thus, multiple-sequence alignment can be an
      important factor in downstream effects on topological reconstruction.
FAU - Ogden, T Heath
AU  - Ogden TH
AD  - Center for Evolutionary Functional Genomics, The Biodesign Institute, and the
      School of Life Sciences, Arizona State University, Tempe, Arizona 85287-4501,
      USA. heath_ogden@asu.edu
FAU - Rosenberg, Michael S
AU  - Rosenberg MS
LA  - eng
GR  - R03-LM008637/LM/NLM NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Base Sequence
MH  - DNA/*genetics
MH  - Evolution, Molecular
MH  - Likelihood Functions
MH  - Models, Biological
MH  - *Phylogeny
MH  - Sequence Alignment/*methods
EDAT- 2006/04/14 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/04/14 09:00
PHST- 2006/04/14 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/04/14 09:00 [entrez]
AID - P4K066N6N345N408 [pii]
AID - 10.1080/10635150500541730 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):314-28. doi: 10.1080/10635150500541730.

PMID- 16611601
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - Ancestral state reconstruction of body size in the Caniformia (Carnivora,
      Mammalia): the effects of incorporating data from the fossil record.
PG  - 301-13
AB  - A recent molecular phylogeny of the mammalian order Carnivora implied large body 
      size as the ancestral condition for the caniform subclade Arctoidea using the
      distribution of species mean body sizes among living taxa. "Extant taxa-only"
      approaches such as these discount character state observations for fossil members
      of living clades and completely ignore data from extinct lineages. To more
      rigorously reconstruct body sizes of ancestral forms within the Caniformia, body 
      size and first appearance data were collected for 149 extant and 367 extinct
      taxa. Body sizes were reconstructed for four ancestral nodes using weighted
      squared-change parsimony on log-transformed body mass data. Reconstructions based
      on extant taxa alone favored large body sizes (on the order of 10 to 50 kg) for
      the last common ancestors of both the Caniformia and Arctoidea. In contrast,
      reconstructions incorporating fossil data support small body sizes (&lt; 5 kg) for
      the ancestors of those clades. When the temporal information associated with
      fossil data was discarded, body size reconstructions became ambiguous,
      demonstrating that incorporating both character state and temporal information
      from fossil taxa unambiguously supports a small ancestral body size, thereby
      falsifying hypotheses derived from extant taxa alone. Body size reconstructions
      for Caniformia, Arctoidea, and Musteloidea were not sensitive to potential errors
      introduced by uncertainty in the position of extinct lineages relative to the
      molecular topology, or to missing body size data for extinct members of an entire
      major clade (the aquatic Pinnipedia). Incorporating character state observations 
      and temporal information from the fossil record into hypothesis testing has a
      significant impact on the ability to reconstruct ancestral characters and
      constrains the range of potential hypotheses of character evolution. Fossil data 
      here provide the evidence to reliably document trends of both increasing and
      decreasing body size in several caniform clades. More generally, including
      fossils in such analyses incorporates evidence of directional trends, thereby
      yielding more reliable ancestral character state reconstructions.
FAU - Finarelli, John A
AU  - Finarelli JA
AD  - Committee on Evolutionary Biology, University of Chicago, Illinois 60637, USA.
      johnf@uchicago.edu
FAU - Flynn, John J
AU  - Flynn JJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - *Body Size
MH  - Carnivora/*anatomy &amp; histology/classification
MH  - *Fossils
EDAT- 2006/04/14 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/04/14 09:00
PHST- 2006/04/14 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/04/14 09:00 [entrez]
AID - R022600P902HU876 [pii]
AID - 10.1080/10635150500541698 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):301-13. doi: 10.1080/10635150500541698.

PMID- 16611600
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20070719
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - Evolutionary rates, divergence dates, and the performance of mitochondrial genes 
      in Bayesian phylogenetic analysis.
PG  - 289-300
AB  - The mitochondrial genome is one of the most frequently used loci in phylogenetic 
      and phylogeographic analyses, and it is becoming increasingly possible to
      sequence and analyze this genome in its entirety from diverse taxa. However,
      sequencing the entire genome is not always desirable or feasible. Which genes
      should be selected to best infer the evolutionary history of the mitochondria
      within a group of organisms, and what properties of a gene determine its
      phylogenetic performance? The current study addresses these questions in a
      Bayesian phylogenetic framework with reference to a phylogeny of plethodontid and
      related salamanders derived from 27 complete mitochondrial genomes; this topology
      is corroborated by nuclear DNA and morphological data. Evolutionary rates for
      each mitochondrial gene and divergence dates for all nodes in the plethodontid
      mitochondrial genome phylogeny were estimated in both Bayesian and maximum
      likelihood frameworks using multiple fossil calibrations, multiple data
      partitions, and a clock-independent approach. Bayesian analyses of individual
      genes were performed, and the resulting trees compared against the reference
      topology. Ordinal logistic regression analysis of molecular evolution rate, gene 
      length, and the G-shape parameter a demonstrated that slower rate of evolution
      and longer gene length both increased the probability that a gene would perform
      well phylogenetically. Estimated rates of molecular evolution vary 84-fold among 
      different mitochondrial genes and different salamander lineages, and mean rates
      among genes vary 15-fold. Despite having conserved amino acid sequences, cox1,
      cox2, cox3, and cob have the fastest mean rates of nucleotide substitution, and
      the greatest variation in rates, whereas rrnS and rrnL have the slowest rates.
      Reasons underlying this rate variation are discussed, as is the extensive rate
      variation in cox1 in light of its proposed role in DNA barcoding.
FAU - Mueller, Rachel Lockridge
AU  - Mueller RL
AD  - Museum of Vertebrate Zoology, University of California, 3101 Valley Life Sciences
      Building Berkeley, California 94720-3160, USA. rmueller@uchicago.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2007 Jun;56(3):542
MH  - Animals
MH  - Bayes Theorem
MH  - *Evolution, Molecular
MH  - Genes, Mitochondrial/*genetics
MH  - Mitochondria/*genetics
MH  - *Phylogeny
MH  - Urodela/genetics
EDAT- 2006/04/14 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/04/14 09:00
PHST- 2006/04/14 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/04/14 09:00 [entrez]
AID - T8733337X7687241 [pii]
AID - 10.1080/10635150500541672 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):289-300. doi: 10.1080/10635150500541672.

PMID- 16611599
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - Fast computation of supertrees for compatible phylogenies with nested taxa.
PG  - 270-88
AB  - Typically, supertree methods combine a collection of source trees in which just
      the leaves are labeled by taxa. In such methods the resulting supertree is also
      leaf labeled. An underlying assumption in these methods is that across all trees 
      in the collection, no two of the taxa are nested; for example, "buttercups" and
      "plants" are nested taxa. Motivated by Page, the first supertree algorithm for
      allowing the source trees to collectively have nested taxa is called
      AncestralBuild. Here, in addition to taxa labeling the leaves, the source trees
      may have taxa labeling some of their interior nodes. Taxa-labeling interior nodes
      are at a higher taxonomic level than that of their descendants (for example,
      genera versus species). Analogous to the supertree method Build for deciding the 
      compatibility of a collection of source trees in which just the leaves are
      labeled, AncestralBuild is a polynomial-time algorithm for deciding the
      compatibility of a collection of source trees in which some of the interior nodes
      are also labeled by taxa. Although a more general method, in this paper we show
      that the original description of AncestralBuild can be modified so that the
      running time is as fast as the current fastest running time for Build. Fast
      computation for deciding compatibility is essential if one is to make use of
      phylogenetic databases that contain thousands of trees on tens of thousands of
      taxa. This is particularly so as AncestralBuild is incorporated as a basic tool
      inside more general supertree methods (that is, methods that always output a tree
      regardless of the compatibility of the source trees). We apply the method to
      propose a comprehensive phylogeny of the strepsirrhines, a major group of the
      primates.
FAU - Berry, Vincent
AU  - Berry V
AD  - Departement Informatique, L.I.R.M.M.-C.N.R.S., 161 rue Ada, 34392 Montpellier
      Cedex 5, France. vberry@lirmm.fr
FAU - Semple, Charles
AU  - Semple C
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Computer Simulation
MH  - Models, Biological
MH  - *Phylogeny
MH  - Software
MH  - Strepsirhini/classification/genetics
MH  - Time Factors
EDAT- 2006/04/14 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/04/14 09:00
PHST- 2006/04/14 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/04/14 09:00 [entrez]
AID - M33725745505H733 [pii]
AID - 10.1080/10635150500541649 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):270-88. doi: 10.1080/10635150500541649.

PMID- 16551582
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - Inferring complex DNA substitution processes on phylogenies using uniformization 
      and data augmentation.
PG  - 259-69
AB  - A new method is developed for calculating sequence substitution probabilities
      using Markov chain Monte Carlo (MCMC) methods. The basic strategy is to use
      uniformization to transform the original continuous time Markov process into a
      Poisson substitution process and a discrete Markov chain of state transitions. An
      efficient MCMC algorithm for evaluating substitution probabilities by this
      approach using a continuous gamma distribution to model site-specific rates is
      outlined. The method is applied to the problem of inferring branch lengths and
      site-specific rates from nucleotide sequences under a general time-reversible
      (GTR) model and a computer program BYPASSR is developed. Simulations are used to 
      examine the performance of the new program relative to an existing program BASEML
      that uses a discrete approximation for the gamma distributed prior on
      site-specific rates. It is found that BASEML and BYPASSR are in close agreement
      when inferring branch lengths, regardless of the number of rate categories used, 
      but that BASEML tends to underestimate high site-specific substitution rates, and
      to overestimate intermediate rates, when fewer than 50 rate categories are used. 
      Rate estimates obtained using BASEML agree more closely with those of BYPASSR as 
      the number of rate categories increases. Analyses of the posterior distributions 
      of site-specific rates from BYPASSR suggest that a large number of taxa are
      needed to obtain precise estimates of site-specific rates, especially when rates 
      are very high or very low. The method is applied to analyze 45 sequences of the
      alpha 2B adrenergic receptor gene (A2AB) from a sample of eutherian taxa. In
      general, the pattern expected for regions under negative selection is observed
      with third codon positions having the highest inferred rates, followed by first
      codon positions and with second codon positions having the lowest inferred rates.
      Several sites show exceptionally high substitution rates at second codon
      positions that may represent the effects of positive selection.
FAU - Mateiu, Ligia
AU  - Mateiu L
AD  - Department of Medical Genetics, University of Alberta, Edmonton, Alberta, Canada.
FAU - Rannala, Bruce
AU  - Rannala B
LA  - eng
GR  - HG01988/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Receptors, Adrenergic, alpha-2)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Base Sequence
MH  - DNA/*genetics
MH  - Mammals/genetics
MH  - Markov Chains
MH  - *Models, Biological
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Receptors, Adrenergic, alpha-2/genetics
MH  - Software
EDAT- 2006/03/23 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/03/23 09:00
PHST- 2006/03/23 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/03/23 09:00 [entrez]
AID - R05Q3757V645376J [pii]
AID - 10.1080/10635150500541599 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):259-69. doi: 10.1080/10635150500541599.

PMID- 16551581
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - How can third codon positions outperform first and second codon positions in
      phylogenetic inference? An empirical example from the seed plants.
PG  - 245-58
AB  - Greater phylogenetic signal is often found in parsimony-based analyses of third
      codon positions of protein-coding genes relative to their corresponding first and
      second codon positions, even for early-derived ("basal") clades. We used the
      Soltis et al. (2000; Bot. J. Linn. Soc. 133:381-461) data matrix of atpB and rbcL
      from 567 seed plants to quantify how each of six factors (observed
      character-state space, frequencies of observed character states, substitution
      probabilities among nucleotides, rate heterogeneity among sites, overall rate of 
      evolution, and number of parsimony-informative characters) contributed to this
      phenomenon. Each of these six factors was estimated from the original data matrix
      for parsimony-informative third codon positions considered separately from first 
      and second codon positions combined. One of the most parsimonious trees found was
      used as the constraint topology; branch lengths were estimated using
      likelihood-based distances, and characters were simulated on this tree.
      Differential frequencies of observed character states were found to be the most
      limiting of the factors simulated for all three codon positions. Differential
      frequencies of observed character states and differential substitution
      probabilities among states were relatively advantageous for first and second
      codon positions. In contrast, differential numbers of observed character states, 
      differential rate heterogeneity among sites, the greater number of
      parsimony-informative characters, and the higher overall rate of evolution were
      relatively advantageous for third codon positions. The amount of possible
      synapomorphy was predictive of the overall success of resolution.
FAU - Simmons, Mark P
AU  - Simmons MP
AD  - Department of Biology, Colorado State University, Fort Collins, Colorado
      80523-1878, USA. psimmons@lamar.colostate.edu
FAU - Zhang, Li-Bing
AU  - Zhang LB
FAU - Webb, Colleen T
AU  - Webb CT
FAU - Reeves, Aaron
AU  - Reeves A
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
RN  - EC 4.1.1.39 (RbcL protein, plastid)
RN  - EC 4.1.1.39 (Ribulose-Bisphosphate Carboxylase)
SB  - IM
MH  - Base Sequence
MH  - Codon/*genetics
MH  - Evolution, Molecular
MH  - Gene Expression Regulation, Plant
MH  - Magnoliopsida/*genetics
MH  - Models, Biological
MH  - *Phylogeny
MH  - Ribulose-Bisphosphate Carboxylase/genetics
EDAT- 2006/03/23 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/03/23 09:00
PHST- 2006/03/23 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/03/23 09:00 [entrez]
AID - T382031288437360 [pii]
AID - 10.1080/10635150500481473 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):245-58. doi: 10.1080/10635150500481473.

PMID- 16551580
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - Arrival and diversification of caviomorph rodents and platyrrhine primates in
      South America.
PG  - 228-44
AB  - Platyrrhine primates and caviomorph rodents are clades of mammals that colonized 
      South America during its period of isolation from the other continents, between
      100 and 3 million years ago (Mya). Until now, no molecular study investigated the
      timing of the South American colonization by these two lineages with the same
      molecular data set. Using sequences from three nuclear genes (ADRA2B, vWF, and
      IRBP, both separate and combined) from 60 species, and eight fossil calibration
      constraints, we estimated the times of origin and diversification of platyrrhines
      and caviomorphs via a Bayesian relaxed molecular clock approach. To account for
      the possible effect of an accelerated rate of evolution of the IRBP gene along
      the branch leading to the anthropoids, we performed the datings with and without 
      IRBP (3768 sites and 2469 sites, respectively). The time window for the
      colonization of South America by primates and by rodents is demarcated by the
      dates of origin (upper bound) and radiation (lower bound) of platyrrhines and
      caviomorphs. According to this approach, platyrrhine primates colonized South
      America between 37.0 +/- 3.0 Mya (or 38.9 +/- 4.0 Mya without IRBP) and 16.8 +/- 
      2.3 (or 20.1 +/- 3.3) Mya, and caviomorph rodents between 45.4 +/- 4.1 (or 43.7
      +/- 4.8) Mya and 36.7 +/- 3.7 (or 35.8 +/- 4.3) Mya. Considering both the fossil 
      record and these molecular datings, the favored scenarios are a trans-Atlantic
      migration of primates from Africa at the end of the Eocene or beginning of the
      Oligocene, and a colonization of South America by rodents during the Middle or
      Late Eocene. Based on our nuclear DNA data, we cannot rule out the possibility of
      a concomitant arrival of primates and rodents in South America. The caviomorphs
      radiated soon after their arrival, before the Oligocene glaciations, and these
      early caviomorph lineages persisted until the present. By contrast, few
      platyrrhine fossils are known in the Oligocene, and the present-day taxa are the 
      result of a quite recent, Early Miocene diversification.
FAU - Poux, Celine
AU  - Poux C
AD  - Laboratoire de Paleontologie, Phylogenie et Paleobiologie, CC064, Institut des
      Sciences de l'Evolution (UMR 5554/CNRS), Universite Montpellier II, Place E.,
      Bataillon, 34 095, Montpellier Cedex 05, France.
FAU - Chevret, Pascale
AU  - Chevret P
FAU - Huchon, Dorothee
AU  - Huchon D
FAU - de Jong, Wilfried W
AU  - de Jong WW
FAU - Douzery, Emmanuel J P
AU  - Douzery EJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
RN  - 0 (Receptors, Adrenergic, alpha-2)
RN  - 0 (Retinol-Binding Proteins)
RN  - 0 (von Willebrand Factor)
SB  - IM
MH  - Animal Migration
MH  - Animals
MH  - Bayes Theorem
MH  - *Evolution, Molecular
MH  - Fossils
MH  - Genetic Markers/genetics
MH  - Genetic Variation
MH  - *Phylogeny
MH  - Platyrrhini/*classification/*genetics
MH  - Receptors, Adrenergic, alpha-2/genetics
MH  - Retinol-Binding Proteins/genetics
MH  - Rodentia/*classification/*genetics
MH  - South America
MH  - von Willebrand Factor/genetics
EDAT- 2006/03/23 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/03/23 09:00
PHST- 2006/03/23 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/03/23 09:00 [entrez]
AID - G175KX2542585505 [pii]
AID - 10.1080/10635150500481390 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):228-44. doi: 10.1080/10635150500481390.

PMID- 16537291
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - Molecular phylogenetic analyses indicate multiple independent emergences of
      parasitism in Myzostomida (Protostomia).
PG  - 208-27
AB  - The fossil record indicates that Myzostomida, an enigmatic group of marine worms,
      traditionally considered as annelids, have exhibited a symbiotic relationship
      with echinoderms, especially crinoids, for nearly 350 million years. All known
      extant myzostomids are associated with echinoderms and infest their integument,
      gonads, celom, or digestive system. Using nuclear (18S rDNA) and mitochondrial
      (16S and COI) DNA sequence data from 37 myzostomid species representing nine
      genera, we report here the first molecular phylogeny of the Myzostomida and
      investigate the evolution of their various symbiotic associations. Our analyses
      indicate that the two orders Proboscidea and Pharyngidea do not constitute
      natural groupings. Character reconstruction analyses strongly suggest that (1)
      the ancestor of all extant myzostomids was an ectocommensal that first infested
      crinoids, and then asteroids and ophiuroids, and (2) parasitism in myzostomids
      emerged multiple times independently.
FAU - Lanterbecq, Deborah
AU  - Lanterbecq D
AD  - Marine Biology Laboratory, University of Mons-Hainaut, 6 Avenue du champ de Mars,
      Life Sciences building, 7000 Mons, Belgium. deborah.lanterbecq@umh.ac.be
FAU - Rouse, Greg W
AU  - Rouse GW
FAU - Milinkovitch, Michel C
AU  - Milinkovitch MC
FAU - Eeckhaut, Igor
AU  - Eeckhaut I
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Annelida/*genetics/*physiology/ultrastructure
MH  - Echinodermata/*parasitology
MH  - *Evolution, Molecular
MH  - Host-Parasite Interactions
MH  - *Phylogeny
MH  - *Symbiosis
EDAT- 2006/03/16 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/03/16 09:00
PHST- 2006/03/16 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/03/16 09:00 [entrez]
AID - T127351LPM06186Q [pii]
AID - 10.1080/10635150500481317 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):208-27. doi: 10.1080/10635150500481317.

PMID- 16522570
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - Computing Bayes factors using thermodynamic integration.
PG  - 195-207
AB  - In the Bayesian paradigm, a common method for comparing two models is to compute 
      the Bayes factor, defined as the ratio of their respective marginal likelihoods. 
      In recent phylogenetic works, the numerical evaluation of marginal likelihoods
      has often been performed using the harmonic mean estimation procedure. In the
      present article, we propose to employ another method, based on an analogy with
      statistical physics, called thermodynamic integration. We describe the method,
      propose an implementation, and show on two analytical examples that this
      numerical method yields reliable estimates. In contrast, the harmonic mean
      estimator leads to a strong overestimation of the marginal likelihood, which is
      all the more pronounced as the model is higher dimensional. As a result, the
      harmonic mean estimator systematically favors more parameter-rich models, an
      artefact that might explain some recent puzzling observations, based on harmonic 
      mean estimates, suggesting that Bayes factors tend to overscore complex models.
      Finally, we apply our method to the comparison of several alternative models of
      amino-acid replacement. We confirm our previous observations, indicating that
      modeling pattern heterogeneity across sites tends to yield better models than
      standard empirical matrices.
FAU - Lartillot, Nicolas
AU  - Lartillot N
AD  - Laboratoire d'Informatique, de Robotique et de Microelectronique de Montpellier
      UMR 5506, CNRS-Universite, de Montpellier 2, 161, rue Ada, 34392 Montpellier
      Cedex 5, France. nicolas.lartillot@lirmm.fr
FAU - Philippe, Herve
AU  - Philippe H
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Proteins)
SB  - IM
MH  - Amino Acid Sequence
MH  - Animals
MH  - *Bayes Theorem
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - Models, Biological
MH  - *Phylogeny
MH  - Proteins/chemistry/genetics
MH  - Thermodynamics
EDAT- 2006/03/09 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/03/09 09:00
PHST- 2006/03/09 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/03/09 09:00 [entrez]
AID - G44K65W276345678 [pii]
AID - 10.1080/10635150500433722 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):195-207. doi: 10.1080/10635150500433722.

PMID- 16522569
OWN - NLM
STAT- MEDLINE
DCOM- 20060613
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 2
DP  - 2006 Apr
TI  - Tenrec phylogeny and the noninvasive extraction of nuclear DNA.
PG  - 181-94
AB  - Due in part to scarcity of material, no published study has yet cladistically
      addressed the systematics of living and fossil Tenrecidae (Mammalia, Afrotheria).
      Using a noninvasive technique for sampling nuclear DNA from museum specimens, we 
      investigate the evolution of the Tenrecidae and assess the extent to which
      tenrecids fit patterns of relationships proposed for other terrestrial mammals on
      Madagascar. Application of several tree-reconstruction techniques on sequences of
      the nuclear growth hormone receptor gene and morphological data for all
      recognized tenrecid genera supports monophyly of Malagasy tenrecids to the
      exclusion of the two living African genera. However, both parsimony and Bayesian 
      methods favor a close relationship between fossil African tenrecs and the
      Malagasy Geogale, supporting the hypothesis of island paraphyly, but not
      polyphyly. More generally, the noninvasive extraction technique can be applied
      with minimal risk to rare/unique specimens and, by better utilizing museum
      collections for genetic work, can greatly mitigate field expenses and disturbance
      of natural populations.
FAU - Asher, Robert J
AU  - Asher RJ
AD  - Museum fur Naturkunde, D-10115, Berlin, Germany. robert.asher@museum.hu-berlin.de
FAU - Hofreiter, Michael
AU  - Hofreiter M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Cell Nucleus/*genetics
MH  - DNA/genetics/*isolation &amp; purification
MH  - Fossils
MH  - Insectivora/*classification/*genetics
MH  - *Phylogeny
MH  - Skull
EDAT- 2006/03/09 09:00
MHDA- 2006/06/14 09:00
CRDT- 2006/03/09 09:00
PHST- 2006/03/09 09:00 [pubmed]
PHST- 2006/06/14 09:00 [medline]
PHST- 2006/03/09 09:00 [entrez]
AID - QW77988407Q75822 [pii]
AID - 10.1080/10635150500433649 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Apr;55(2):181-94. doi: 10.1080/10635150500433649.

PMID- 16507535
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20060301
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - A robust distance coefficient between distribution areas incorporating geographic
      distances.
PG  - 170-5
FAU - Hennig, Christian
AU  - Hennig C
AD  - University College London, Department of Statistical Science, 1-19 Torrington
      Place, London, WC1E 6BT, United Kingdom.
FAU - Hausdorf, Bernhard
AU  - Hausdorf B
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Demography
MH  - *Geography
MH  - *Models, Biological
MH  - Selection Bias
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - U0W687N3H61387HH [pii]
AID - 10.1080/10635150500481523 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):170-5. doi: 10.1080/10635150500481523.

PMID- 16507534
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20071114
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - Statistical approaches for DNA barcoding.
PG  - 162-9
FAU - Nielsen, Rasmus
AU  - Nielsen R
AD  - Department of Biological Statistics and Computational Biology, Center for
      Bioinformatics, University of Copenhagen Universitetsparken 15, 2100 Copenhagen, 
      Denmark. rn28@cornell.edu
FAU - Matz, Mikhail
AU  - Matz M
LA  - eng
GR  - GM066243/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Anura/classification/genetics
MH  - Bayes Theorem
MH  - Butterflies/classification/genetics
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - WVHHX8X716312020 [pii]
AID - 10.1080/10635150500431239 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):162-9. doi: 10.1080/10635150500431239.

PMID- 16507533
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - The PhyloCode and the distinction between taxonomy and nomenclature.
PG  - 160-2
FAU - Queiroz, Kevin de
AU  - Queiroz Kd
AD  - Department of Vertebrate Zoology, National Museum of Natural History, Smithsonian
      Institution, Washington DC 20560, USA. dequeirk@si.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Phylogeny
MH  - *Terminology as Topic
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - M5380H652732R858 [pii]
AID - 10.1080/10635150500431221 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):160-2. doi: 10.1080/10635150500431221.

PMID- 16507532
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - Time, species, and the generation of trait variance in clades.
PG  - 151-9
FAU - Ricklefs, Robert E
AU  - Ricklefs RE
AD  - Department of Biology, University of Missouri-St. Louis 8001 Natural Bridge Road,
      St. Louis, Missouri 63121-4499, USA. ricklefs@umsl.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Computer Simulation
MH  - Genetic Speciation
MH  - Genetic Variation
MH  - Models, Biological
MH  - *Phenotype
MH  - Phylogeny
MH  - Time Factors
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - P14587X6U92346T0 [pii]
AID - 10.1080/10635150500431205 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):151-9. doi: 10.1080/10635150500431205.

PMID- 16507530
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20060301
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - Are unequal clade priors problematic for Bayesian phylogenetics?
PG  - 138-46; author reply 147-51
FAU - Brandley, Matthew C
AU  - Brandley MC
AD  - Museum of Vertebrate Zoology and Department of Integrative Biology, University of
      California, Berkeley, California 94720-3160, USA. brandley@berkeley.edu
FAU - Leache, Adam D
AU  - Leache AD
FAU - Warren, Dan L
AU  - Warren DL
FAU - McGuire, Jimmy A
AU  - McGuire JA
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - N96160U0M30W2634 [pii]
AID - 10.1080/10635150500431155 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):138-46; author reply 147-51. doi:
      10.1080/10635150500431155.

PMID- 16507529
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - Combined mitochondrial and nuclear DNA sequences resolve the interrelations of
      the major Australasian marsupial radiations.
PG  - 122-37
AB  - Australasian marsupials include three major radiations, the
      insectivorous/carnivorous Dasyuromorphia, the omnivorous bandicoots
      (Peramelemorphia), and the largely herbivorous diprotodontians. Morphologists
      have generally considered the bandicoots and diprotodontians to be closely
      related, most prominently because they are both syndactylous (with the 2nd and
      3rd pedal digits being fused). Molecular studies have been unable to confirm or
      reject this Syndactyla hypothesis. Here we present new mitochondrial (mt) genomes
      from a spiny bandicoot (Echymipera rufescens) and two dasyurids, a fat-tailed
      dunnart (Sminthopsis crassicaudata) and a northern quoll (Dasyurus hallucatus).
      By comparing trees derived from pairwise base-frequency differences between taxa 
      with standard (absolute, uncorrected) distance trees, we infer that composition
      bias among mt protein-coding and RNA sequences is sufficient to mislead tree
      reconstruction. This can explain incongruence between trees obtained from mt and 
      nuclear data sets. However, after excluding major sources of compositional
      heterogeneity, both the "reduced-bias" mt and nuclear data sets clearly favor a
      bandicoot plus dasyuromorphian association, as well as a grouping of kangaroos
      and possums (Phalangeriformes) among diprotodontians. Notably, alternatives to
      these groupings could only be confidently rejected by combining the mt and
      nuclear data. Elsewhere on the tree, Dromiciops appears to be sister to the
      monophyletic Australasian marsupials, whereas the placement of the marsupial mole
      (Notoryctes) remains problematic. More generally, we contend that it is desirable
      to combine mt genome and nuclear sequences for inferring vertebrate phylogeny,
      but as separately modeled process partitions. This strategy depends on detecting 
      and excluding (or accounting for) major sources of non-historical signal, such as
      from compositional non-stationarity. [Base composition; combined data; marsupial;
      mitochondrial genome; phylogeny.].
FAU - Phillips, Matthew J
AU  - Phillips MJ
AD  - Henry Wellcome Ancient Biomolecules Center, Department of Zoology, Oxford
      University, South Parks Road, Oxford OX1 3PS, United Kingdom.
      mattphillips73@hotmail.com
FAU - McLenachan, Patricia A
AU  - McLenachan PA
FAU - Down, Christin
AU  - Down C
FAU - Gibb, Gillian C
AU  - Gibb GC
FAU - Penny, David
AU  - Penny D
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Australasia
MH  - Biological Evolution
MH  - Classification/methods
MH  - DNA, Mitochondrial/chemistry
MH  - DNA, Ribosomal/chemistry
MH  - Marsupialia/*classification/genetics
MH  - Phylogeny
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - T2878L8006306856 [pii]
AID - 10.1080/10635150500481614 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):122-37. doi: 10.1080/10635150500481614.

PMID- 16507528
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - Fundamental differences between the methods of maximum likelihood and maximum
      posterior probability in phylogenetics.
PG  - 116-21
AB  - Using a four-taxon example under a simple model of evolution, we show that the
      methods of maximum likelihood and maximum posterior probability (which is a
      Bayesian method of inference) may not arrive at the same optimal tree topology.
      Some patterns that are separately uninformative under the maximum likelihood
      method are separately informative under the Bayesian method. We also show that
      this difference has impact on the bootstrap frequencies and the posterior
      probabilities of topologies, which therefore are not necessarily approximately
      equal. Efron et al. (Proc. Natl. Acad. Sci. USA 93:13429-13434, 1996) stated that
      bootstrap frequencies can, under certain circumstances, be interpreted as
      posterior probabilities. This is true only if one includes a non-informative
      prior distribution of the possible data patterns, and most often the prior
      distributions are instead specified in terms of topology and branch lengths.
      [Bayesian inference; maximum likelihood method; Phylogeny; support.].
FAU - Svennblad, Bodil
AU  - Svennblad B
AD  - Department of Mathematics, Uppsala University, Box 480, SE-751 06 Uppsala,
      Sweden. bodil.svennblad@math.uu.se
FAU - Erixon, Per
AU  - Erixon P
FAU - Oxelman, Bengt
AU  - Oxelman B
FAU - Britton, Tom
AU  - Britton T
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Computational Biology
MH  - Likelihood Functions
MH  - Models, Biological
MH  - *Phylogeny
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - T7773756054765T8 [pii]
AID - 10.1080/10635150500481648 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):116-21. doi: 10.1080/10635150500481648.

PMID- 16507527
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - Medusozoan phylogeny and character evolution clarified by new large and small
      subunit rDNA data and an assessment of the utility of phylogenetic mixture
      models.
PG  - 97-115
AB  - A newly compiled data set of nearly complete sequences of the large subunit of
      the nuclear ribosome (LSU or 28S) sampled from 31 diverse medusozoans greatly
      clarifies the phylogenetic history of Cnidaria. These data have substantial power
      to discern among many of the competing hypotheses of relationship derived from
      prior work. Moreover, LSU data provide strong support at key nodes that were
      equivocal based on other molecular markers. Combining LSU sequences with those of
      the small subunit of the nuclear ribosome (SSU or 18S), we present a detailed
      working hypothesis of medusozoan relationships and discuss character evolution
      within this diverse clade. Stauromedusae, comprising the benthic, so-called
      stalked jellyfish, appears to be the sister group of all other medusozoans,
      implying that the free-swimming medusa stage, the motor nerve net, and statocysts
      of ecto-endodermal origin are features derived within Medusozoa. Cubozoans, which
      have had uncertain phylogenetic affinities since the elucidation of their life
      cycles, form a clade-named Acraspeda-with the scyphozoan groups Coronatae,
      Rhizostomeae, and Semaeostomeae. The polyps of both cubozoans and hydrozoans
      appear to be secondarily simplified. Hydrozoa is comprised by two well-supported 
      clades, Trachylina and Hydroidolina. The position of Limnomedusae within
      Trachylina indicates that the ancestral hydrozoan had a biphasic life cycle and
      that the medusa was formed via an entocodon. Recently hypothesized homologies
      between the entocodon and bilaterian mesoderm are therefore suspect.
      Laingiomedusae, which has often been viewed as a close ally of the trachyline
      group Narcomedusae, is instead shown to be unambiguously a member of
      Hydroidolina. The important model organisms of the Hydra species complex are part
      of a clade, Aplanulata, with other hydrozoans possessing direct development not
      involving a ciliated planula stage. Finally, applying phylogenetic mixture models
      to our data proved to be of little additional value over a more traditional
      phylogenetic approach involving explicit hypothesis testing and bootstrap
      analyses under multiple optimality criteria. [18S; 28S; Cubozoa; Hydrozoa;
      medusa; molecular systematics; polyp; Scyphozoa; Staurozoa.].
FAU - Collins, Allen G
AU  - Collins AG
AD  - NMFS, National Systematics Laboratory, National Museum of Natural History,
      MRC-153, Smithsonian Institution, Washington DC 20013-7012, USA. CollinsA@SI.edu
FAU - Schuchert, Peter
AU  - Schuchert P
FAU - Marques, Antonio C
AU  - Marques AC
FAU - Jankowski, Thomas
AU  - Jankowski T
FAU - Medina, Monica
AU  - Medina M
FAU - Schierwater, Bernd
AU  - Schierwater B
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Cnidaria/anatomy &amp; histology/*classification/genetics
MH  - Cubozoa/anatomy &amp; histology/classification/genetics
MH  - DNA, Ribosomal/chemistry
MH  - Hydrozoa/anatomy &amp; histology/classification/genetics
MH  - Models, Biological
MH  - *Phylogeny
MH  - Scyphozoa/anatomy &amp; histology/classification/genetics
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - V1Q6P715Q2147UV4 [pii]
AID - 10.1080/10635150500433615 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):97-115. doi: 10.1080/10635150500433615.

PMID- 16507526
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20161019
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - Comparative performance of Bayesian and AIC-based measures of phylogenetic model 
      uncertainty.
PG  - 89-96
AB  - Reversible-jump Markov chain Monte Carlo (RJ-MCMC) is a technique for
      simultaneously evaluating multiple related (but not necessarily nested)
      statistical models that has recently been applied to the problem of phylogenetic 
      model selection. Here we use a simulation approach to assess the performance of
      this method and compare it to Akaike weights, a measure of model uncertainty that
      is based on the Akaike information criterion. Under conditions where the
      assumptions of the candidate models matched the generating conditions, both
      Bayesian and AIC-based methods perform well. The 95% credible interval contained 
      the generating model close to 95% of the time. However, the size of the credible 
      interval differed with the Bayesian credible set containing approximately 25% to 
      50% fewer models than an AIC-based credible interval. The posterior probability
      was a better indicator of the correct model than the Akaike weight when all
      assumptions were met but both measures performed similarly when some model
      assumptions were violated. Models in the Bayesian posterior distribution were
      also more similar to the generating model in their number of parameters and were 
      less biased in their complexity. In contrast, Akaike-weighted models were more
      distant from the generating model and biased towards slightly greater complexity.
      The AIC-based credible interval appeared to be more robust to the violation of
      the rate homogeneity assumption. Both AIC and Bayesian approaches suggest that
      substantial uncertainty can accompany the choice of model for phylogenetic
      analyses, suggesting that alternative candidate models should be examined in
      analysis of phylogenetic data. [AIC; Akaike weights; Bayesian phylogenetics;
      model averaging; model selection; model uncertainty; posterior probability;
      reversible jump.].
FAU - Alfaro, Michael E
AU  - Alfaro ME
AD  - School of Biological Sciences, P.O. Box 644236, Pullman, Washington 99164-4236,
      USA. alfaro@wsu.edu
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
LA  - eng
GR  - R01 GM069801/GM/NIGMS NIH HHS/United States
GR  - GM-069801/GM/NIGMS NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Markov Chains
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Uncertainty
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - H35R26MU72100328 [pii]
AID - 10.1080/10635150500433565 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):89-96. doi: 10.1080/10635150500433565.

PMID- 16507525
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - The pattern and timing of diversification of Philippine endemic rodents: evidence
      from mitochondrial and nuclear gene sequences.
PG  - 73-88
AB  - The 22 genera and 64 species of rodents (Muridae: Murinae) distributed in the
      Philippine Islands provide a unique opportunity to study patterns and processes
      of diversification in island systems. Over 90% of these rodent species are
      endemic to the archipelago, but the relative importance of dispersal from the
      mainland, dispersal within the archipelago, and in situ differentiation as
      explanations of this diversity remains unclear, as no phylogenetic hypothesis for
      these species and relevant mainland forms is currently available. Here we report 
      the results of phylogenetic analyses of the endemic Philippine murines and a wide
      sampling of murine diversity from outside the archipelago, based on the
      mitochondrial cytochrome b gene and the nuclear-encoded IRBP exon 1. Analysis of 
      our combined gene data set consistently identified five clades comprising endemic
      Philippine genera, suggesting multiple invasions of the archipelago. Molecular
      dating analyses using parametric and semiparametric methods suggest that
      colonization occurred in at least two stages, one ca. 15 Mya, and another 8 to 12
      million years later, consistent with the previous recognition of "Old" and "New" 
      endemic rodent faunas. Ancestral area analysis suggests that the Old Endemics
      invaded landmasses that are now part of the island of Luzon, whereas the three
      New Endemic clades may have colonized through either Mindanao, Luzon, or both.
      Further, our results suggest that most of the diversification of Philippine
      murines took place within the archipelago. Despite heterogeneity between nuclear 
      and mitochondrial genes in most model parameters, combined analysis of the two
      data sets using both parsimony and likelihood increased phylogenetic resolution; 
      however, the effect of data combination on support for resolved nodes was method 
      dependent. In contrast, our results suggest that combination of mitochondrial and
      nuclear data to estimate relatively ancient divergence times can severely
      compromise those estimates, even when specific methods that account for rate
      heterogeneity among genes are employed. [Biogeography; divergence date
      estimation; mitochondrial DNA; molecular systematics; Murinae; nuclear exon;
      Philippines; phylogeny.].
FAU - Jansa, Sharon A
AU  - Jansa SA
AD  - Bell Museum of Natural History, University of Minnesota, 1987 Upper Buford
      Circle, (S.A.J.), St. Paul, Minnesota 55108, USA. jansa003@umn.edu
FAU - Barker, F Keith
AU  - Barker FK
FAU - Heaney, Lawrence R
AU  - Heaney LR
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Eye Proteins)
RN  - 0 (Retinol-Binding Proteins)
RN  - 0 (interstitial retinol-binding protein)
RN  - 9035-37-4 (Cytochromes b)
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Cytochromes b/genetics
MH  - DNA, Mitochondrial/chemistry
MH  - Eye Proteins/genetics
MH  - Geography
MH  - *Murinae/classification/genetics
MH  - Philippines
MH  - Phylogeny
MH  - Retinol-Binding Proteins/genetics
MH  - Sequence Analysis, DNA
MH  - Time Factors
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - H02Q0282680U3436 [pii]
AID - 10.1080/10635150500431254 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):73-88. doi: 10.1080/10635150500431254.

PMID- 16507524
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20070813
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - Optimal intron analyses in the trimeresurus radiation of Asian pitvipers.
PG  - 57-72
AB  - Nuclear introns are commonly used as phylogenetic markers, but a number of issues
      related to alignment strategies, indel treatments, and the incorporation of
      length-variant heterozygotes (LVHs) are not routinely addressed when generating
      phylogenetic hypotheses. Topological congruence in relation to an extensive
      mitochondrial DNA multigene phylogeny (derived from 2,423 bp of 12S, 16S, ND4,
      and CYTB genes) of the Asian pitviper Trimeresurus radiation was used to compare 
      combinations of "by eye" and edited and unedited ClustalX 1.8 alignments of two
      nuclear introns. Indels were treated as missing data, fifth character states, and
      assigned simple and multistate codes. Upon recovery of the optimal alignment and 
      indel treatment strategy, a total evidence approach was used to investigate the
      phylogenetic utility of the indels and test new generic arrangements within
      Trimeresurus. Approximately one third of the intron data partitions exhibited
      LVHs, suggesting that they are common in introns. Furthermore, a simple
      concatenation approach can facilitate the incorporation of LVHs into phylogenetic
      analyses to make use of all available data and investigate mechanisms of
      molecular evolution. Analyses of ClustalX 1.8-assisted alignments were generally 
      more congruent than the "by eye" alignment and the analysis of a simple coded,
      edited ClustalX 1.8 (gap opening cost 5, gap extension cost 1) alignment revealed
      the most congruent tree. The total evidence approach supported the new
      arrangements within Trimeresurus, suggesting that the phylogeny should be
      considered as a working benchmark in Asian pitviper systematics. Finally, a
      critical appraisal of the diverse array of indels (56 to 57 per intron, ranging
      from 1 to 151 bp in length) suggested that they are a combination of Hennigian
      and homoplasious events unrelated to indel size or location within the intron.
      [Alignment; indels; intron analysis; length-variant heterozygotes;
      Trimeresurus.].
FAU - Creer, Simon
AU  - Creer S
AD  - School of Biological Sciences, University of Wales, Bangor, Gwynedd LL57 2UW, UK.
      s.creer@bangor.ac.uk
FAU - Pook, Catharine E
AU  - Pook CE
FAU - Malhotra, Anita
AU  - Malhotra A
FAU - Thorpe, Roger S
AU  - Thorpe RS
LA  - eng
GR  - 057257/Z/99/Z/Wellcome Trust/United Kingdom
GR  - 060384/Z/00/Z/Wellcome Trust/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - DNA, Mitochondrial/chemistry
MH  - Genetic Speciation
MH  - *Introns
MH  - Phylogeny
MH  - Polymorphism, Genetic
MH  - Sequence Alignment/*methods
MH  - Sequence Analysis, DNA
MH  - Trimeresurus/*classification/genetics
MH  - Viperidae/*classification/genetics
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - U587V414805G5N46 [pii]
AID - 10.1080/10635150500431213 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):57-72. doi: 10.1080/10635150500431213.

PMID- 16507523
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - Hybrids in real time.
PG  - 46-56
AB  - We describe some new and recent results that allow for the analysis and
      representation of reticulate evolution by non-tree networks. In particular, we
      (1) present a simple result to show that, despite the presence of reticulation,
      there is always a well-defined underlying tree that corresponds to those parts of
      life that do not have a history of reticulation; (2) describe and apply new
      theory for determining the smallest number of hybridization events required to
      explain conflicting gene trees; and (3) present a new algorithm to determine
      whether an arbitrary rooted network can be realized by contemporaneous
      reticulation events. We illustrate these results with examples. [Directed acyclic
      graph; reticulate evolution; hybrid species; sub-tree prune and re-graft.].
FAU - Baroni, Mihaela
AU  - Baroni M
AD  - Biomathematics Research Centre, Department of Mathematics and Statistics,
      University of Canterbury, Christchurch, New Zealand.
FAU - Semple, Charles
AU  - Semple C
FAU - Steel, Mike
AU  - Steel M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - *Hybridization, Genetic
MH  - Models, Biological
MH  - Phylogeny
MH  - Time Factors
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - G8567675Q3457054 [pii]
AID - 10.1080/10635150500431197 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):46-56. doi: 10.1080/10635150500431197.

PMID- 16507522
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20121115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - Molecular phylogenetics of the exoneurine allodapine bees reveal an ancient and
      puzzling dispersal from Africa to Australia.
PG  - 31-45
AB  - Previous phylogenetic studies of the bee tribe Allodapini suggested a puzzling
      biogeographic problem: one of the key basal divergences involved separation of
      the southern African and southern Australian clades at a very early stage in
      allodapine evolution, but no taxa occur in the Palaearctic or Asian regions that 
      might suggest a Laurasian dispersal route. However, these studies lacked
      sufficient sequence data and appropriate maximum likelihood partition models to
      provide reliable phylogenetic estimates and enable alternative biogeographic
      hypotheses to be distinguished. Using Bayesian and penalized likelihood
      approaches and an expanded sequence and taxon set we examine phylogenetic
      relationships between the Australian, African, and Malagasy groups and estimate
      divergence times for key nodes. We show that divergence of the three basal
      Australian clades (known as the exoneurines) occurred at least 25 Mya following a
      single colonization event, and that this group diverged from the African +
      Madagascan clade at least 30 Mya, but actual divergence dates are likely to be
      much older than these very conservative limits. The bifurcation order of the
      exoneurine clades was not resolved and analyses could not rule out the existence 
      of a hard polytomy, suggesting rapid radiation after colonization of Australia.
      Their divergence involved major transitions in life history traits and these
      placed constraints on the kinds of social organization that subsequently evolved 
      in each lineage. Early divergence between the African, Malagasy, and Australian
      clades presents a major puzzle for historical biogeography: node ages are too
      recent for Gondwanan vicariance hypotheses, but too early for Laurasian dispersal
      scenarios. We suggest a scenario involving island hopping across the Indian Ocean
      via a series of now largely submerged elements of the Kergulen Plateau and Broken
      Ridge provinces, both of which are known to have had subaerial formations during 
      the Cenozoic. [Bayesian; biogeography; dispersal; Gondwana; Kerguelen Plateau;
      penalized likelihood.].
FAU - Schwarz, Michael P
AU  - Schwarz MP
AD  - Biological Sciences, Flinders University, GPO Box 2100, Adelaide, SA 5001,
      Australia. Michael.Schwarz@flinders.edu.au
FAU - Fuller, Susan
AU  - Fuller S
FAU - Tierney, Simon M
AU  - Tierney SM
FAU - Cooper, Steven J B
AU  - Cooper SJ
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Africa
MH  - Animals
MH  - Australia
MH  - Bayes Theorem
MH  - *Bees/classification/genetics
MH  - Evolution, Molecular
MH  - Genetic Speciation
MH  - Geography
MH  - Oceans and Seas
MH  - Phylogeny
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - QUT3R81V397UV3X0 [pii]
AID - 10.1080/10635150500431148 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):31-45. doi: 10.1080/10635150500431148.

PMID- 16507521
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - Inferring phylogeny despite incomplete lineage sorting.
PG  - 21-30
AB  - It is now well known that incomplete lineage sorting can cause serious
      difficulties for phylogenetic inference, but little attention has been paid to
      methods that attempt to overcome these difficulties by explicitly considering the
      processes that produce them. Here we explore approaches to phylogenetic inference
      designed to consider retention and sorting of ancestral polymorphism. We examine 
      how the reconstructability of a species (or population) phylogeny is affected by 
      (a) the number of loci used to estimate the phylogeny and (b) the number of
      individuals sampled per species. Even in difficult cases with considerable
      incomplete lineage sorting (times between divergences less than 1 N(e)
      generations), we found the reconstructed species trees matched the "true" species
      trees in at least three out of five partitions, as long as a reasonable number of
      individuals per species were sampled. We also studied the tradeoff between
      sampling more loci versus more individuals. Although increasing the number of
      loci gives more accurate trees for a given sampling effort with deeper species
      trees (e.g., total depth of 10 N(e) generations), sampling more individuals often
      gives better results than sampling more loci with shallower species trees (e.g., 
      depth = 1 N(e)). Taken together, these results demonstrate that gene sequences
      retain enough signal to achieve an accurate estimate of phylogeny despite
      widespread incomplete lineage sorting. Continued improvement in our methods to
      reconstruct phylogeny near the species level will require a shift to a compound
      model that considers not only nucleotide or character state substitutions, but
      also the population genetics processes of lineage sorting. [Coalescence;
      divergence; population; speciation.].
FAU - Maddison, Wayne P
AU  - Maddison WP
AD  - Department of Zoology Research Centre, University of British Columbia, V6T 1Z4
      Vancouver, B.C., Canada. wmaddisn@interchange.ubc.ca
FAU - Knowles, L Lacey
AU  - Knowles LL
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Evolution, Molecular
MH  - *Genetic Speciation
MH  - Genetics, Population
MH  - *Phylogeny
MH  - *Polymorphism, Genetic
MH  - Selection Bias
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - Q1126T60VN7L1015 [pii]
AID - 10.1080/10635150500354928 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):21-30. doi: 10.1080/10635150500354928.

PMID- 16507520
OWN - NLM
STAT- MEDLINE
DCOM- 20060418
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 55
IP  - 1
DP  - 2006 Feb
TI  - Phylogeny of Eunicida (Annelida) and exploring data congruence using a partition 
      addition bootstrap alteration (PABA) approach.
PG  - 1-20
AB  - Even though relationships within Annelida are poorly understood, Eunicida is one 
      of only a few major annelid lineages well supported by morphology. The seven
      recognized eunicid families possess sclerotized jaws that include mandibles and a
      maxillary apparatus. The maxillary apparatuses vary in shape and number of
      elements, and three main types are recognized in extant taxa: ctenognath,
      labidognath, and prionognath. Ctenognath jaws are usually considered to represent
      the plesiomorphic state of Eunicida, whereas taxa with labidognath and
      prionognath are thought to form a derived monophyletic assemblage. However, this 
      hypothesis has never been tested in a statistical framework even though it holds 
      considerable importance for understanding annelid phylogeny and possibly
      lophotrochozoan evolution because Eunicida has the best annelid fossil record.
      Therefore, we used maximum likelihood and Bayesian inference approaches to
      reconstruct Eunicida phylogeny using sequence data from nuclear 18S and 28S rDNA 
      genes and mitochondrial 16S rDNA and cytochrome c oxidase subunit I genes.
      Additionally, we conducted three different tests to investigate suitability of
      combining data sets. Incongruence length difference (ILD) and Shimodaira-Hasegawa
      (SH) test comparisons of resultant trees under different data partitions have
      been widely used previously but do not give a good indication as to which nodes
      may be causing the conflict. Thus, we developed a partition addition bootstrap
      alteration (PABA) approach that evaluates congruence or conflict for any given
      node by determining how bootstrap scores are altered when different data
      partitions are added. PABA shows the contribution of each partition to the
      phylogeny obtained in the combined analysis. Generally, the ILD test performed
      worse than the other approaches in detecting incongruence. Both PABA and the SH
      approach indicated the 28S and COI data sets add conflicting signal, but PABA is 
      more informative for elucidating which data partition may be misleading at a
      given node. All our analyses indicate that the monophyly of the
      labidognath/prionognath taxa and even a labidognath clade (i.e., a
      "Eunicidae"/Onuphidae/Lumbrineridae clade) is significantly rejected. We show
      that the definition of both the labidognath and ctenognath jaw type does not
      address adequately the variation within Eunicida and thus misleads our current
      evolutionary understanding. Based on the presented results a symmetric maxillary 
      apparatus with a carrier and four to six maxillae is most likely the
      plesiomorphic condition for Eunicida. [COI; conflicting data; fossil record; ILD;
      Jaw Evolution; molecular phylogeny; rDNA; SH test.].
FAU - Struck, Torsten H
AU  - Struck TH
AD  - Auburn University, 101 Rouse Building, Auburn, Alabama 36849, USA.
      structh@auburn.edu
FAU - Purschke, Gunter
AU  - Purschke G
FAU - Halanych, Kenneth M
AU  - Halanych KM
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (DNA, Ribosomal)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - DNA, Mitochondrial/chemistry
MH  - DNA, Ribosomal/chemistry
MH  - Electron Transport Complex IV/genetics
MH  - Jaw/anatomy &amp; histology
MH  - Phylogeny
MH  - Polychaeta/*classification/genetics
MH  - Sequence Analysis, DNA
EDAT- 2006/03/02 09:00
MHDA- 2006/04/19 09:00
CRDT- 2006/03/02 09:00
PHST- 2006/03/02 09:00 [pubmed]
PHST- 2006/04/19 09:00 [medline]
PHST- 2006/03/02 09:00 [entrez]
AID - L68557L77G4G4315 [pii]
AID - 10.1080/10635150500354910 [doi]
PST - ppublish
SO  - Syst Biol. 2006 Feb;55(1):1-20. doi: 10.1080/10635150500354910.

PMID- 16385778
OWN - NLM
STAT- MEDLINE
DCOM- 20060126
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 6
DP  - 2005 Dec
TI  - Underparameterized model of sequence evolution leads to bias in the estimation of
      diversification rates from molecular phylogenies.
PG  - 973-83
FAU - Revell, Liam J
AU  - Revell LJ
AD  - Department of Biology, Campus Box 1229, Washington University, St. Louis,
      Missouri 63130, USA. ljrevell@wustl.edu
FAU - Harmon, Luke J
AU  - Harmon LJ
FAU - Glor, Richard E
AU  - Glor RE
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - Bias
MH  - *Evolution, Molecular
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Species Specificity
MH  - Statistics as Topic
EDAT- 2006/01/03 09:00
MHDA- 2006/01/27 09:00
CRDT- 2006/01/03 09:00
PHST- 2006/01/03 09:00 [pubmed]
PHST- 2006/01/27 09:00 [medline]
PHST- 2006/01/03 09:00 [entrez]
AID - 10.1080/10635150500354647 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Dec;54(6):973-83. doi: 10.1080/10635150500354647.

PMID- 16385777
OWN - NLM
STAT- MEDLINE
DCOM- 20060126
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 6
DP  - 2005 Dec
TI  - Characters, states, and homology.
PG  - 965-73
FAU - Freudenstein, John V
AU  - Freudenstein JV
AD  - Department of Evolution, Ecology and Organismal Biology, Ohio State University
      Herbarium, 1315 Kinnear Road, Columbus, 43212, USA. freudenstein.1@osu.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Knowledge
MH  - *Phylogeny
MH  - Sequence Homology, Nucleic Acid
EDAT- 2006/01/03 09:00
MHDA- 2006/01/27 09:00
CRDT- 2006/01/03 09:00
PHST- 2006/01/03 09:00 [pubmed]
PHST- 2006/01/27 09:00 [medline]
PHST- 2006/01/03 09:00 [entrez]
AID - 10.1080/10635150500354654 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Dec;54(6):965-73. doi: 10.1080/10635150500354654.

PMID- 16385776
OWN - NLM
STAT- MEDLINE
DCOM- 20060126
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 6
DP  - 2005 Dec
TI  - Hastings ratio of the LOCAL proposal used in Bayesian phylogenetics.
PG  - 961-5
FAU - Holder, Mark T
AU  - Holder MT
AD  - School of Computational Science, Florida State University, Tallahassee,
      32306-4120, USA. mholder@csit.fsu.edu
FAU - Lewis, Paul O
AU  - Lewis PO
FAU - Swofford, David L
AU  - Swofford DL
FAU - Larget, Bret
AU  - Larget B
LA  - eng
GR  - R01 GM068950-01/GM/NIGMS NIH HHS/United States
GR  - R01 GM069801-01/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - *Bayes Theorem
MH  - Computational Biology/*methods/*statistics &amp; numerical data
MH  - *Phylogeny
EDAT- 2006/01/03 09:00
MHDA- 2006/01/27 09:00
CRDT- 2006/01/03 09:00
PHST- 2006/01/03 09:00 [pubmed]
PHST- 2006/01/27 09:00 [medline]
PHST- 2006/01/03 09:00 [entrez]
AID - 10.1080/10635150500354670 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Dec;54(6):961-5. doi: 10.1080/10635150500354670.

PMID- 16385775
OWN - NLM
STAT- MEDLINE
DCOM- 20060126
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 6
DP  - 2005 Dec
TI  - Between two extremes: mitochondrial DNA is neither the panacea nor the nemesis of
      phylogenetic and taxonomic inference.
PG  - 952-61
FAU - Rubinoff, Daniel
AU  - Rubinoff D
AD  - Department of Plant and Environmental Protection Sciences, University of Hawaii, 
      3050 Maile Way, 310 Gilmore Hall, Honolulu, 96822, USA. rubinoff@hawaii.edu
FAU - Holland, Brenden S
AU  - Holland BS
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Biological Evolution
MH  - Classification/*methods
MH  - DNA, Mitochondrial/*classification/genetics
MH  - Data Interpretation, Statistical
MH  - *Phylogeny
MH  - Plants/classification
EDAT- 2006/01/03 09:00
MHDA- 2006/01/27 09:00
CRDT- 2006/01/03 09:00
PHST- 2006/01/03 09:00 [pubmed]
PHST- 2006/01/27 09:00 [medline]
PHST- 2006/01/03 09:00 [entrez]
AID - 10.1080/10635150500234674 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Dec;54(6):952-61. doi: 10.1080/10635150500234674.

PMID- 16385774
OWN - NLM
STAT- MEDLINE
DCOM- 20060126
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 6
DP  - 2005 Dec
TI  - A tale of two processes.
PG  - 948-51
FAU - Lockhart, Peter
AU  - Lockhart P
AD  - Institute for Molecular BioSciences, Allan Wilson Centre for Molecular Ecology
      and Evolution, Massey University, Palmerston North, New Zealand.
      P.J.Lockhart@massey.ac.nz
FAU - Steel, Mike
AU  - Steel M
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Base Sequence
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - Likelihood Functions
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2006/01/03 09:00
MHDA- 2006/01/27 09:00
CRDT- 2006/01/03 09:00
PHST- 2006/01/03 09:00 [pubmed]
PHST- 2006/01/27 09:00 [medline]
PHST- 2006/01/03 09:00 [entrez]
AID - 10.1080/10635150500234682 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Dec;54(6):948-51. doi: 10.1080/10635150500234682.

PMID- 16338765
OWN - NLM
STAT- MEDLINE
DCOM- 20060126
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 6
DP  - 2005 Dec
TI  - Unearthing the molecular phylodiversity of desert soil green algae (Chlorophyta).
PG  - 936-47
AB  - Deserts are not usually considered biodiversity hotspots, but desert microbiotic 
      crust communities exhibit a rich diversity of both eukaryotic and prokaryotic
      life forms. Like many communities dominated by microscopic organisms, they defy
      characterization by traditional species-counting approaches to assessing
      biodiversity. Here we use exclusive molecular phylodiversity (E) to quantify the 
      amount of evolutionary divergence unique to desert-dwelling green algae
      (Chlorophyta) in microbiotic crust communities. Given a phylogenetic tree with
      branch lengths expressed in units of expected substitutions per site, E is the
      total length of all tree segments representing exclusively desert lineages. Using
      MCMC to integrate over tree topologies and branch lengths provides 95% Bayesian
      credible intervals for phylodiversity measures. We found substantial exclusive
      molecular phylodiversity based on 18S rDNA data, showing that desert lineages are
      distantly related to their nearest aquatic relatives. Our results challenge
      conventional wisdom, which holds that there was a single origin of terrestrial
      green plants and that green algae are merely incidental visitors rather than
      indigenous components of desert communities. We identify examples of lineage
      diversification within deserts and at least 12 separate transitions from aquatic 
      to terrestrial life apart from the most celebrated transition leading to the
      embryophyte land plants. [Bayesian phylogenetics; biodiversity; exclusive
      molecular phylodiversity; microbiotic crusts.].
FAU - Lewis, Louise A
AU  - Lewis LA
AD  - Department of Ecology and Evolutionary Biology, The University of Connecticut, 75
      North Eagleville Road, Storrs, 06269-3043, USA. louise.lewis@uconn.edu
FAU - Lewis, Paul O
AU  - Lewis PO
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 18S)
RN  - 0 (Soil)
SB  - IM
MH  - *Biodiversity
MH  - *Biological Evolution
MH  - Chlorophyta/*classification/*genetics
MH  - Desert Climate
MH  - Environment
MH  - *Genetic Speciation
MH  - Likelihood Functions
MH  - Models, Biological
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - RNA, Ribosomal, 18S/genetics
MH  - Sequence Analysis, DNA
MH  - Soil/analysis
EDAT- 2005/12/13 09:00
MHDA- 2006/01/27 09:00
CRDT- 2005/12/13 09:00
PHST- 2005/12/13 09:00 [pubmed]
PHST- 2006/01/27 09:00 [medline]
PHST- 2005/12/13 09:00 [entrez]
AID - G643701275733674 [pii]
AID - 10.1080/10635150500354852 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Dec;54(6):936-47. doi: 10.1080/10635150500354852.

PMID- 16338764
OWN - NLM
STAT- MEDLINE
DCOM- 20060126
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 6
DP  - 2005 Dec
TI  - Molecular phylogenetics of the siphonophora (Cnidaria), with implications for the
      evolution of functional specialization.
PG  - 916-35
AB  - Siphonophores are a group of pelagic colonial hydrozoans (Cnidaria) that have
      long been of general interest because of the division of labor between the polyps
      and medusae that make up these "superorganisms." These polyps and medusae are
      each homologous to free living animals but are generated by an incomplete asexual
      budding process that leaves them physiologically integrated. They are
      functionally specialized for different tasks and are precisely organized within
      each colony. The number of functional types of polyps and medusae varies across
      taxa, and different authors have used this character to construct phylogenies
      polarized in opposite directions, depending on whether they thought siphonophore 
      evolution proceeded by a reduction or an increase in functional specialization.
      We have collected taxa across all major groups of siphonophores, many of which
      are found exclusively in the deep sea, using remotely operated underwater
      vehicles (ROVs) and by SCUBA diving from ships in the open ocean. We have used 52
      siphonophores and four outgroup taxa to estimate the siphonophore phylogeny with 
      molecular data from the nuclear small subunit ribosomal RNA gene (18S) and the
      mitochondrial large subunit ribosomal RNA gene (16S). Parsimony reconstructions
      indicate that functionally specialized polyps and medusae have been gained and
      lost across the phylogeny. Maximum likelihood and Bayesian analyses of
      morphological data suggest that the transition rate for decreased functional
      specialization is greater than the transition rate for increased functional
      specialization for three out of the four investigated categories of polyps and
      medusae. The present analysis also bears on several long-standing questions about
      siphonophore systematics. It indicates that the cystonects are sister to all
      other siphonophores, a group that we call the Codonophora. We also find that the 
      Calycophorae are nested within the Physonectae, and that the Brachystelia, a
      historically recognized grouping of short-stemmed taxa, are polyphyletic.
      [Cnidaria; colonial animals; deep sea; division of labor; functional
      specialization; Hydrozoa; phylogenetics; Siphonophores.].
FAU - Dunn, Casey W
AU  - Dunn CW
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven,
      Connecticut, USA. cdunn@hawaii.edu
FAU - Pugh, Philip R
AU  - Pugh PR
FAU - Haddock, Steven H D
AU  - Haddock SH
LA  - eng
SI  - GENBANK/AY935270
SI  - GENBANK/AY935271
SI  - GENBANK/AY935272
SI  - GENBANK/AY935273
SI  - GENBANK/AY935274
SI  - GENBANK/AY935275
SI  - GENBANK/AY935276
SI  - GENBANK/AY935277
SI  - GENBANK/AY935278
SI  - GENBANK/AY935279
SI  - GENBANK/AY935280
SI  - GENBANK/AY935281
SI  - GENBANK/AY935282
SI  - GENBANK/AY935283
SI  - GENBANK/AY935284
SI  - GENBANK/AY935285
SI  - GENBANK/AY935286
SI  - GENBANK/AY935287
SI  - GENBANK/AY935288
SI  - GENBANK/AY935289
SI  - GENBANK/AY935290
SI  - GENBANK/AY935291
SI  - GENBANK/AY935292
SI  - GENBANK/AY935293
SI  - GENBANK/AY935294
SI  - GENBANK/AY935295
SI  - GENBANK/AY935296
SI  - GENBANK/AY935297
SI  - GENBANK/AY935298
SI  - GENBANK/AY935299
SI  - GENBANK/AY935300
SI  - GENBANK/AY935301
SI  - GENBANK/AY935302
SI  - GENBANK/AY935303
SI  - GENBANK/AY935304
SI  - GENBANK/AY935305
SI  - GENBANK/AY935306
SI  - GENBANK/AY935307
SI  - GENBANK/AY935308
SI  - GENBANK/AY935309
SI  - GENBANK/AY935310
SI  - GENBANK/AY935311
SI  - GENBANK/AY935312
SI  - GENBANK/AY935313
SI  - GENBANK/AY935314
SI  - GENBANK/AY935315
SI  - GENBANK/AY935316
SI  - GENBANK/AY935317
SI  - GENBANK/AY935318
SI  - GENBANK/AY935319
SI  - GENBANK/AY935320
SI  - GENBANK/AY935321
SI  - GENBANK/AY935322
SI  - GENBANK/AY935323
SI  - GENBANK/AY937312
SI  - GENBANK/AY937313
SI  - GENBANK/AY937314
SI  - GENBANK/AY937315
SI  - GENBANK/AY937316
SI  - GENBANK/AY937317
SI  - GENBANK/AY937318
SI  - GENBANK/AY937319
SI  - GENBANK/AY937320
SI  - GENBANK/AY937321
SI  - GENBANK/AY937322
SI  - GENBANK/AY937323
SI  - GENBANK/AY937324
SI  - GENBANK/AY937325
SI  - GENBANK/AY937326
SI  - GENBANK/AY937327
SI  - GENBANK/AY937328
SI  - GENBANK/AY937329
SI  - GENBANK/AY937330
SI  - GENBANK/AY937331
SI  - GENBANK/AY937332
SI  - GENBANK/AY937333
SI  - GENBANK/AY937334
SI  - GENBANK/AY937335
SI  - GENBANK/AY937336
SI  - GENBANK/AY937337
SI  - GENBANK/AY937338
SI  - GENBANK/AY937339
SI  - GENBANK/AY937340
SI  - GENBANK/AY937341
SI  - GENBANK/AY937342
SI  - GENBANK/AY937343
SI  - GENBANK/AY937344
SI  - GENBANK/AY937345
SI  - GENBANK/AY937346
SI  - GENBANK/AY937347
SI  - GENBANK/AY937348
SI  - GENBANK/AY937349
SI  - GENBANK/AY937350
SI  - GENBANK/AY937351
SI  - GENBANK/AY937352
SI  - GENBANK/AY937353
SI  - GENBANK/AY937354
SI  - GENBANK/AY937355
SI  - GENBANK/AY937356
SI  - GENBANK/AY937357
SI  - GENBANK/AY937358
SI  - GENBANK/AY937359
SI  - GENBANK/AY937360
SI  - GENBANK/AY937361
SI  - GENBANK/AY937362
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 16S)
RN  - 0 (RNA, Ribosomal, 18S)
SB  - IM
MH  - Animals
MH  - *Evolution, Molecular
MH  - *Genetic Speciation
MH  - Hydrozoa/*classification/genetics
MH  - Models, Biological
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Protein Structure, Secondary
MH  - RNA, Ribosomal, 16S/genetics
MH  - RNA, Ribosomal, 18S/genetics
MH  - Sequence Analysis, DNA
EDAT- 2005/12/13 09:00
MHDA- 2006/01/27 09:00
CRDT- 2005/12/13 09:00
PHST- 2005/12/13 09:00 [pubmed]
PHST- 2006/01/27 09:00 [medline]
PHST- 2005/12/13 09:00 [entrez]
AID - X0233Q72W4370746 [pii]
AID - 10.1080/10635150500354837 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Dec;54(6):916-35. doi: 10.1080/10635150500354837.

PMID- 16282169
OWN - NLM
STAT- MEDLINE
DCOM- 20060126
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 6
DP  - 2005 Dec
TI  - Calculating the evolutionary rates of different genes: a fast, accurate estimator
      with applications to maximum likelihood phylogenetic analysis.
PG  - 900-15
AB  - In phylogenetic analyses with combined multigene or multiprotein data sets,
      accounting for differing evolutionary dynamics at different loci is essential for
      accurate tree prediction. Existing maximum likelihood (ML) and Bayesian
      approaches are computationally intensive. We present an alternative approach that
      is orders of magnitude faster. The method, Distance Rates (DistR), estimates
      rates based upon distances derived from gene/protein sequence data. Simulation
      studies indicate that this technique is accurate compared with other methods and 
      robust to missing sequence data. The DistR method was applied to a fungal
      mitochondrial data set, and the rate estimates compared well to those obtained
      using existing ML and Bayesian approaches. Inclusion of the protein rates
      estimated from the DistR method into the ML calculation of trees as a branch
      length multiplier resulted in a significantly improved fit as measured by the
      Akaike Information Criterion (AIC). Furthermore, bootstrap support for the ML
      topology was significantly greater when protein rates were used, and some evident
      errors in the concatenated ML tree topology (i.e., without protein rates) were
      corrected. [Bayesian credible intervals; DistR method; multigene phylogeny;
      PHYML; rate heterogeneity.].
FAU - Bevan, Rachel B
AU  - Bevan RB
AD  - McGill Centre for Bioinformatics, Duff Medical Building, 3775 University Street, 
      Montreal, Quebec, H3A 2B4, Canada. rachel@mcb.mcgill.ca
FAU - Lang, B Franz
AU  - Lang BF
FAU - Bryant, David
AU  - Bryant D
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Fungal Proteins)
SB  - IM
MH  - Algorithms
MH  - Classification/methods
MH  - Computational Biology/methods
MH  - *Computer Simulation
MH  - *Evolution, Molecular
MH  - Fungal Proteins/classification
MH  - Likelihood Functions
MH  - *Models, Statistical
MH  - *Phylogeny
MH  - Sequence Analysis, Protein/*methods
MH  - Statistics as Topic/methods
EDAT- 2005/11/12 09:00
MHDA- 2006/01/27 09:00
CRDT- 2005/11/12 09:00
PHST- 2005/11/12 09:00 [pubmed]
PHST- 2006/01/27 09:00 [medline]
PHST- 2005/11/12 09:00 [entrez]
AID - M525300127M3P541 [pii]
AID - 10.1080/10635150500354829 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Dec;54(6):900-15. doi: 10.1080/10635150500354829.

PMID- 16282168
OWN - NLM
STAT- MEDLINE
DCOM- 20060126
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 6
DP  - 2005 Dec
TI  - Nodes in phylogenetic trees: the relation between imbalance and number of
      descendent species.
PG  - 895-9
AB  - The imbalance of a node in a phylogenetic tree can be defined in terms of the
      relative numbers of species (or higher taxa) on the branches that originate at
      the node. Empirically, imbalance also turns out to depend on the absolute total
      number of species on the branches: in a sample of large trees, nodes with more
      descendent species tend to be more unbalanced. Subsidiary analyses suggest that
      this pattern is not a result of errors in tree estimation. Instead, the increase 
      in imbalance with species is consistent with a cumulative effect of differences
      in diversification rates between branches. [Equal-rates Markov model; imbalance; 
      phylogeny shape; proportional-to-distinguishable-arrangements model.].
FAU - Holman, Eric W
AU  - Holman EW
AD  - Department of Psychology, University of California, Los Angeles, 90095, USA.
      holman@psych.ucla.edu
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Classification/methods
MH  - Computational Biology/*methods
MH  - *Genetic Speciation
MH  - Markov Chains
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2005/11/12 09:00
MHDA- 2006/01/27 09:00
CRDT- 2005/11/12 09:00
PHST- 2005/11/12 09:00 [pubmed]
PHST- 2006/01/27 09:00 [medline]
PHST- 2005/11/12 09:00 [entrez]
AID - G26M61275847571V [pii]
AID - 10.1080/10635150500354696 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Dec;54(6):895-9. doi: 10.1080/10635150500354696.

PMID- 16282167
OWN - NLM
STAT- MEDLINE
DCOM- 20060126
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 6
DP  - 2005 Dec
TI  - Mosaics of convergences and noise in morphological phylogenies: what's in a
      viverrid-like carnivoran?
PG  - 865-94
AB  - Adaptive convergence in morphological characters has not been thoroughly
      investigated, and the processes by which phylogenetic relationships may be misled
      by morphological convergence remains unclear. We undertook a case study on the
      morphological evolution of viverrid-like feliformians (Nandinia, Cryptoprocta,
      Fossa, Eupleres, Prionodon) and built the largest morphological matrix concerning
      the suborder Feliformia to date. A total of 349 characters grouped into four
      anatomical partitions were used for all species of Viverridae and viverrid-like
      taxa plus representatives of the Felidae, Hyaenidae, Herpestidae, and one
      Malagasy mongoose. Recent molecular phylogenetic analyses suggest that
      viverrid-like morphotypes appeared independently at least three times during
      feliformian evolution. We thus used a synthetic molecular tree to assess
      morphological evolutionary patterns characterizing the viverrid-like taxa. We
      examined phylogenetic signal, convergence and noise in morphological characters
      using (a) tree-length distribution (g1), (b) partitioned Bremer support, (c) RI
      values and their distribution, (d) respective contributions of diagnostic
      synapomorphies at the nodes for each partition, (e) patterns of shared
      convergences among viverrid-like taxa and other feliformian lineages, (f)
      tree-length differences among alternative hypotheses, and (g) the successive
      removal of convergent character states from the original matrix. In addition, the
      lability of complex morphological structures was assessed by mapping them onto
      the synthetic molecular tree. The unconstrained morphological analysis yielded
      phylogenetic groupings that closely reflected traditional classification. The use
      of a synthetic molecular tree (constraint) combined with our thorough
      morphological investigations revealed the mosaics of convergences likely to have 
      contributed to part of the historical uncertainty over viverrid classification.
      It also showed that complex morphological structures could be subjected to
      reversible evolutionary trends. The morphological matrix proved useful in
      characterizing several feliformian clades with diagnostic synapomorphies. These
      results support the removal from the traditionally held Viverridae of several
      viverrid-like taxa into three distinct families: Nandiniidae (Nandinia),
      Prionodontidae (Prionodon), and the newly defined Eupleridae (including
      Cryptoprocta, Fossa, Eupleres plus all "mongoose-like" Malagasy taxa). No clearly
      "phylogenetically misleading" data subsets could be identified, and the great
      majority of morphological convergences appeared to be nonadaptive. The multiple
      approaches used in this study revealed that the most disruptive element with
      regards to morphological phylogenetic reconstruction was noise, which blured the 
      expression of phylogenetic signal. This study demonstrates the crucial need to
      consider independent (molecular) phylogenies in order to produce reliable
      evolutionary hypotheses and should promote a new approach to the definition of
      morphological characters in mammals. [Constrained analysis; convergence;
      evolutionary scenario; Feliformia; morphology; noise; phylogenetic signal;
      phylogeny; Viverridae.].
FAU - Gaubert, Philippe
AU  - Gaubert P
AD  - Unite Origine, Structure et Evolution de la Biodiversite, CNRS UMR 5202,
      Departement Systematique et Evolution, Museum National d'Histoire Naturelle,
      Zoologie: Mammiferes et Oiseaux, CP 51, 57 rue Cuvier, 75231 Paris Cedex 05,
      France. gaubert@ebd.csic.es
FAU - Wozencraft, W Chris
AU  - Wozencraft WC
FAU - Cordeiro-Estrela, Pedro
AU  - Cordeiro-Estrela P
FAU - Veron, Geraldine
AU  - Veron G
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Anatomy, Comparative/*methods
MH  - Animals
MH  - *Biological Evolution
MH  - Biometry
MH  - Body Weights and Measures/*classification
MH  - Carnivora/anatomy &amp; histology/classification
MH  - Genetic Speciation
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Viverridae/anatomy &amp; histology/*classification
EDAT- 2005/11/12 09:00
MHDA- 2006/01/27 09:00
CRDT- 2005/11/12 09:00
PHST- 2005/11/12 09:00 [pubmed]
PHST- 2006/01/27 09:00 [medline]
PHST- 2005/11/12 09:00 [entrez]
AID - U7XL2444M330322G [pii]
AID - 10.1080/10635150500232769 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Dec;54(6):865-94. doi: 10.1080/10635150500232769.

PMID- 16243770
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - The promise of DNA barcoding for taxonomy.
PG  - 852-9
FAU - Hebert, Paul D N
AU  - Hebert PD
AD  - Department of Integrative Biology, Biodiversity Institute of Ontario, University 
      of Guelph, Guelph, Ontario N1G 2W1, Canada. phebert@uoguelph.ca
FAU - Gregory, T Ryan
AU  - Gregory TR
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Databases, Genetic
MH  - Sequence Analysis, DNA/*methods
MH  - Species Specificity
EDAT- 2005/10/26 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/26 09:00
PHST- 2005/10/26 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/26 09:00 [entrez]
AID - P642568W06G713U8 [pii]
AID - 10.1080/10635150500354886 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):852-9. doi: 10.1080/10635150500354886.

PMID- 16243769
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20051024
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - The perils of DNA barcoding and the need for integrative taxonomy.
PG  - 844-51
FAU - Will, Kipling W
AU  - Will KW
AD  - ESPM Department-Insect Biology, University of California, Berkeley, California
      94720, USA.
FAU - Mishler, Brent D
AU  - Mishler BD
FAU - Wheeler, Quentin D
AU  - Wheeler QD
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Databases, Genetic
MH  - Sequence Analysis, DNA/*methods
MH  - Species Specificity
EDAT- 2005/10/26 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/26 09:00
PHST- 2005/10/26 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/26 09:00 [entrez]
AID - G4131P7446704167 [pii]
AID - 10.1080/10635150500354878 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):844-51. doi: 10.1080/10635150500354878.

PMID- 16243768
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - DNA barcoding: perspectives from a "Partnerships for Enhancing Expertise in
      Taxonomy" (PEET) debate.
PG  - 841-4
FAU - Smith, Vincent S
AU  - Smith VS
AD  - Illinois Natural History Survey, 607 East Peabody Drive, Champaign, Illinois
      61820, USA. vsmith@inhs.uiuc.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Databases, Genetic
MH  - Sequence Analysis, DNA/*methods
MH  - Species Specificity
EDAT- 2005/10/26 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/26 09:00
PHST- 2005/10/26 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/26 09:00 [entrez]
AID - R604728X601WG0T8 [pii]
AID - 10.1080/10635150500354894 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):841-4. doi: 10.1080/10635150500354894.

PMID- 16243767
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - On probability and systematics: possibility, probability, and phylogenetic
      inference.
PG  - 831-41
FAU - Haber, Matthew H
AU  - Haber MH
AD  - Department of Philosophy, Center for Population Biology, University of
      California, Davis, Davis, California 95818, USA. mhhaber@ucdavis.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Philosophy
MH  - *Phylogeny
MH  - *Probability
EDAT- 2005/10/26 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/26 09:00
PHST- 2005/10/26 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/26 09:00 [entrez]
AID - UX4750U715223148 [pii]
AID - 10.1080/106351591007444 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):831-41. doi: 10.1080/106351591007444.

PMID- 16243766
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - Measuring support and finding unsupported relationships in supertrees.
PG  - 823-31
FAU - Wilkinson, Mark
AU  - Wilkinson M
AD  - Department of Zoology, The Natural History Museum, London SW7 5BD, UK.
      marw@nhm.ac.uk
FAU - Pisani, Davide
AU  - Pisani D
FAU - Cotton, James A
AU  - Cotton JA
FAU - Corfe, Ian
AU  - Corfe I
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Phylogeny
MH  - *Reproducibility of Results
EDAT- 2005/10/26 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/26 09:00
PHST- 2005/10/26 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/26 09:00 [entrez]
AID - TM6462X7051PT347 [pii]
AID - 10.1080/10635150590950362 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):823-31. doi: 10.1080/10635150590950362.

PMID- 16243765
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20051024
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - An application of tissue and DNA banking for genomics and conservation: the
      Ambrose Monell Cryo-Collection (AMCC).
PG  - 819-23
FAU - Corthals, Angelique
AU  - Corthals A
AD  - American Museum of Natural History, New York, NY, 10024, USA. corthals@amnh.org
FAU - Desalle, Rob
AU  - Desalle R
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Conservation of Natural Resources/*methods
MH  - Cryopreservation/*instrumentation/*methods
MH  - *Databases, Genetic
MH  - Genomics/*methods
MH  - *Tissue Banks
EDAT- 2005/10/26 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/26 09:00
PHST- 2005/10/26 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/26 09:00 [entrez]
AID - G253QQ5465548010 [pii]
AID - 10.1080/10635150590950353 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):819-23. doi: 10.1080/10635150590950353.

PMID- 16243764
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - An empirical examination of the utility of codon-substitution models in phylogeny
      reconstruction.
PG  - 808-18
AB  - Models of codon substitution have been commonly used to compare protein-coding
      DNA sequences and are particularly effective in detecting signals of natural
      selection acting on the protein. Their utility in reconstructing molecular
      phylogenies and in dating species divergences has not been explored. Codon models
      naturally accommodate synonymous and nonsynonymous substitutions, which occur at 
      very different rates and may be informative for recent and ancient divergences,
      respectively. Thus codon models may be expected to make an efficient use of
      phylogenetic information in protein-coding DNA sequences. Here we applied codon
      models to 106 protein-coding genes from eight yeast species to reconstruct
      phylogenies using the maximum likelihood method, in comparison with nucleotide-
      and amino acid-based analyses. The results appeared to confirm that expectation. 
      Nucleotide-based analysis, under simplistic substitution models, were efficient
      in recovering recent divergences whereas amino acid-based analysis performed
      better at recovering deep divergences. Codon models appeared to combine the
      advantages of amino acid and nucleotide data and had good performance at
      recovering both recent and deep divergences. Estimation of relative species
      divergence times using amino acid and codon models suggested that translation of 
      gene sequences into proteins led to information loss of from 30% for deep nodes
      to 66% for recent nodes. Although computational burden makes codon models
      unfeasible for tree search in large data sets, we suggest that they may be useful
      for comparing candidate trees. Nucleotide models that accommodate the differences
      in evolutionary dynamics at the three codon positions also performed well, at
      much less computational cost. We discuss the relationship between a model's fit
      to data and its utility in phylogeny reconstruction and caution against use of
      overly complex substitution models.
FAU - Ren, Fengrong
AU  - Ren F
AD  - Advanced Biomedical Information, Center for Information Medicine, Tokyo Medical
      and Dental University, Japan.
FAU - Tanaka, Hiroshi
AU  - Tanaka H
FAU - Yang, Ziheng
AU  - Yang Z
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
SB  - IM
MH  - Classification/*methods
MH  - Codon/*genetics
MH  - Computational Biology
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Saccharomyces/*genetics
EDAT- 2005/10/26 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/26 09:00
PHST- 2005/10/26 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/26 09:00 [entrez]
AID - T42QL22X6K041U0K [pii]
AID - 10.1080/10635150500354688 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):808-18. doi: 10.1080/10635150500354688.

PMID- 16243763
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - Resolving deep phylogenetic relationships in salamanders: analyses of
      mitochondrial and nuclear genomic data.
PG  - 758-77
AB  - Phylogenetic relationships among salamander families illustrate analytical
      challenges inherent to inferring phylogenies in which terminal branches are
      temporally very long relative to internal branches. We present new mitochondrial 
      DNA sequences, approximately 2,100 base pairs from the genes encoding ND1, ND2,
      COI, and the intervening tRNA genes for 34 species representing all 10 salamander
      families, to examine these relationships. Parsimony analysis of these mtDNA
      sequences supports monophyly of all families except Proteidae, but yields a tree 
      largely unresolved with respect to interfamilial relationships and the
      phylogenetic positions of the proteid genera Necturus and Proteus. In contrast,
      Bayesian and maximum-likelihood analyses of the mtDNA data produce a topology
      concordant with phylogenetic results from nuclear-encoded rRNA sequences, and
      they statistically reject monophyly of the internally fertilizing salamanders,
      suborder Salamandroidea. Phylogenetic simulations based on our mitochondrial DNA 
      sequences reveal that Bayesian analyses outperform parsimony in reconstructing
      short branches located deep in the phylogenetic history of a taxon. However,
      phylogenetic conflicts between our results and a recent analysis of nuclear RAG-1
      gene sequences suggest that statistical rejection of a monophyletic
      Salamandroidea by Bayesian analyses of our mitochondrial genomic data is probably
      erroneous. Bayesian and likelihood-based analyses may overestimate phylogenetic
      precision when estimating short branches located deep in a phylogeny from data
      showing substitutional saturation; an analysis of nucleotide substitutions
      indicates that these methods may be overly sensitive to a relatively small number
      of sites that show substitutions judged uncommon by the favored evolutionary
      model.
FAU - Weisrock, David W
AU  - Weisrock DW
AD  - Department of Biology, Campus Box 1137, Washington University, St. Louis,
      Missouri, 63130, USA. weisrock@uky.edu
FAU - Harmon, Luke J
AU  - Harmon LJ
FAU - Larson, Allan
AU  - Larson A
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Computer Simulation
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - *Research Design
MH  - Sequence Analysis, DNA
MH  - Urodela/*genetics
EDAT- 2005/10/26 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/26 09:00
PHST- 2005/10/26 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/26 09:00 [entrez]
AID - N014835604T18416 [pii]
AID - 10.1080/10635150500234641 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):758-77. doi: 10.1080/10635150500234641.

PMID- 16243762
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20091119
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - An empirical assessment of long-branch attraction artefacts in deep eukaryotic
      phylogenomics.
PG  - 743-57
AB  - In the context of exponential growing molecular databases, it becomes
      increasingly easy to assemble large multigene data sets for phylogenomic studies.
      The expected increase of resolution due to the reduction of the sampling
      (stochastic) error is becoming a reality. However, the impact of systematic
      biases will also become more apparent or even dominant. We have chosen to study
      the case of the long-branch attraction artefact (LBA) using real instead of
      simulated sequences. Two fast-evolving eukaryotic lineages, whose evolutionary
      positions are well established, microsporidia and the nucleomorph of
      cryptophytes, were chosen as model species. A large data set was assembled (44
      species, 133 genes, and 24,294 amino acid positions) and the resulting rooted
      eukaryotic phylogeny (using a distant archaeal outgroup) is positively misled by 
      an LBA artefact despite the use of a maximum likelihood-based tree reconstruction
      method with a complex model of sequence evolution. When the fastest evolving
      proteins from the fast lineages are progressively removed (up to 90%), the
      bootstrap support for the apparently artefactual basal placement decreases to
      virtually 0%, and conversely only the expected placement, among all the possible 
      locations of the fast-evolving species, receives increasing support that
      eventually converges to 100%. The percentage of removal of the fastest evolving
      proteins constitutes a reliable estimate of the sensitivity of phylogenetic
      inference to LBA. This protocol confirms that both a rich species sampling
      (especially the presence of a species that is closely related to the
      fast-evolving lineage) and a probabilistic method with a complex model are
      important to overcome the LBA artefact. Finally, we observed that phylogenetic
      inference methods perform strikingly better with simulated as opposed to real
      data, and suggest that testing the reliability of phylogenetic inference methods 
      with simulated data leads to overconfidence in their performance. Although
      phylogenomic studies can be affected by systematic biases, the possibility of
      discarding a large amount of data containing most of the nonphylogenetic signal
      allows recovering a phylogeny that is less affected by systematic biases, while
      maintaining a high statistical support.
FAU - Brinkmann, Henner
AU  - Brinkmann H
AD  - Canadian Institute for Advanced Research, Centre Robert Cedergren, Departement de
      Biochimie, Universite de Montreal, Succursale Centre-Ville, Montreal, Quebec
      H3C3J7, Canada.
FAU - van der Giezen, Mark
AU  - van der Giezen M
FAU - Zhou, Yan
AU  - Zhou Y
FAU - Poncelin de Raucourt, Gaetan
AU  - Poncelin de Raucourt G
FAU - Philippe, Herve
AU  - Philippe H
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computational Biology
MH  - Computer Simulation
MH  - Eukaryota/*genetics
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Sample Size
MH  - Selection Bias
EDAT- 2005/10/26 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/26 09:00
PHST- 2005/10/26 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/26 09:00 [entrez]
AID - R21667433G8176J4 [pii]
AID - 10.1080/10635150500234609 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):743-57. doi: 10.1080/10635150500234609.

PMID- 16243761
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - Can incomplete taxa rescue phylogenetic analyses from long-branch attraction?
PG  - 731-42
AB  - Taxon sampling may be critically important for phylogenetic accuracy because
      adding taxa can help to subdivide misleading long branches. Although the idea
      that added taxa can break up long branches was exemplified by a study of
      "incomplete" fossil taxa, the issue of taxon completeness (i.e., proportion of
      missing data) has been largely ignored in most subsequent discussions of taxon
      sampling and long-branch attraction. In this article, I use simulations to test
      the ability of incomplete taxa to subdivide long branches and improve
      phylogenetic accuracy in situations of potential long-branch attraction. The
      results show that for most methods and conditions examined, adding taxa that are 
      only 50% complete may provide similar benefits to adding the same number of
      complete taxa (suggesting that the advantages of increased taxon sampling may be 
      obtained with less data than previously considered). For parsimony, taxa that are
      less complete (5% to 25% complete) may often have limited ability to rescue
      analyses from long-branch attraction. In contrast, highly incomplete taxa can be 
      surprisingly beneficial when using model-based methods. The results also suggest 
      the importance of model-based methods in phylogenetic analyses that combine
      molecular and fossil data.
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Department of Ecology and Evolution, Stony Brook University, Stony Brook, New
      York 11794-5245, USA. wiensj@life.bio.sunysb.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Models, Genetic
MH  - *Phylogeny
MH  - *Sample Size
EDAT- 2005/10/26 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/26 09:00
PHST- 2005/10/26 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/26 09:00 [entrez]
AID - L38N12T134075387 [pii]
AID - 10.1080/10635150500234583 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):731-42. doi: 10.1080/10635150500234583.

PMID- 16243760
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - Hylid frog phylogeny and sampling strategies for speciose clades.
PG  - 778-807
AB  - How should characters and taxa be sampled to resolve efficiently the phylogeny of
      ancient and highly speciose groups? We addressed this question empirically in the
      treefrog family Hylidae, which contains &gt; 800 species and may be nonmonophyletic 
      with respect to other anuran families. We sampled 81 species (54 hylids and 27
      outgroups) for two mitochondrial genes (12S, ND1), two nuclear genes (POMC,
      c-myc), and morphology (144 characters) in an attempt to resolve higher-level
      relationships. We then added 117 taxa to the combined data set, many of which
      were sampled for only one gene (12S). Despite the relative incompleteness of the 
      majority of taxa, the resulting trees placed all taxa in the expected
      higher-level clades with strong support, despite some taxa being &gt; 90%
      incomplete. Furthermore, we found no relationship between the completeness of a
      taxon and the support (parsimony bootstrap or Bayesian posterior probabilities)
      for its localized placement on the tree. Separate analysis of the data set with
      the most taxa (12S) gives a somewhat problematic estimate of higher-level
      relationships, suggesting that data sets scored only for some taxa (ND1, nuclear 
      genes, morphology) are important in determining the outcome of the combined
      analysis. The results show that hemiphractine hylids are not closely related to
      other hylids and should be recognized as a distinct family. They also show that
      the speciose genus Hyla is polyphyletic, but that its species can be arranged
      into three monophyletic genera. A new classification of hylid frogs is proposed. 
      Several potentially misleading signals in the morphological data are discussed.
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Department of Ecology and Evolution, Stony Brook University, Stony Brook, New
      York 11794-5245, USA. wiensj@life.bio.sunysb.edu
FAU - Fetzner, James W
AU  - Fetzner JW
FAU - Parkinson, Christopher L
AU  - Parkinson CL
FAU - Reeder, Tod W
AU  - Reeder TW
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
RN  - 66796-54-1 (Pro-Opiomelanocortin)
SB  - IM
MH  - Animals
MH  - Anura/*anatomy &amp; histology/classification/*genetics
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - Genes, myc/genetics
MH  - Larva/anatomy &amp; histology
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Pro-Opiomelanocortin/genetics
MH  - *Sample Size
MH  - Sequence Analysis, DNA
EDAT- 2005/10/26 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/26 09:00
PHST- 2005/10/26 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/26 09:00 [entrez]
AID - J8P663588647K913 [pii]
AID - 10.1080/10635150500234625 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):778-807. doi: 10.1080/10635150500234625.

PMID- 16243759
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - Asynchronous colonization of Madagascar by the four endemic clades of primates,
      tenrecs, carnivores, and rodents as inferred from nuclear genes.
PG  - 719-30
AB  - Madagascar harbors four large adaptive radiations of endemic terrestrial mammals:
      lemurs, tenrecs, carnivorans, and rodents. These rank among the most spectacular 
      examples of evolutionary diversification, but their monophyly and origins are
      debated. The lack of Tertiary fossils from Madagascar leaves molecular studies as
      most promising to solve these controversies. We provide a simultaneous
      reconstruction of phylogeny and age of the four radiations based on a 3.5-kb data
      set from three nuclear genes (ADRA2B, vWF, and AR). The analysis supports each as
      a monophyletic clade, sister to African taxa, and thereby identifies four events 
      of colonization out of Africa. To infer the time windows for colonization, we
      take into account both the divergence from the closest non-insular sister group
      and the initial intra-insular radiation, which is a novel but conservative
      approach in studies of the colonization history of Madagascar. We estimate that
      lemurs colonized Madagascar between 60 million years ago (Mya) (split from
      lorises) and 50 Mya (lemur radiation) (70-41 Mya taking 95% credibility intervals
      into account), tenrecs between 42 and 25 Mya (50-20 Mya), carnivorans between 26 
      and 19 Mya (33-14 Mya), and rodents between 24 and 20 Mya (30-15 Mya). These
      datings suggest at least two asynchronous colonization events: by lemurs in the
      Late Cretaceous-Middle Eocene, and by carnivorans and rodents in the Early
      Oligocene-Early Miocene. The colonization by tenrecs may have taken place
      simultaneously with either of these two events, or in a third event in the Late
      Eocene-Oligocene. Colonization by at least lemurs, rodents, and carnivorans
      appears to have occurred by overseas rafting rather than via a land bridge
      hypothesized to have existed between 45 and 26 Mya, but the second scenario
      cannot be ruled out if credibility intervals are taken into account.
FAU - Poux, Celine
AU  - Poux C
AD  - Department of Biochemistry 161, Radboud University Nijmegen, 6500 HB, Nijmegen,
      PO Box 9101, The Netherlands.
FAU - Madsen, Ole
AU  - Madsen O
FAU - Marquard, Elisabeth
AU  - Marquard E
FAU - Vieites, David R
AU  - Vieites DR
FAU - de Jong, Wilfried W
AU  - de Jong WW
FAU - Vences, Miguel
AU  - Vences M
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Receptors, Adrenergic, alpha-2)
RN  - 0 (Receptors, Androgen)
RN  - 0 (von Willebrand Factor)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - *Demography
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - Madagascar
MH  - Mammals/*genetics/*physiology
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Population Dynamics
MH  - Receptors, Adrenergic, alpha-2/genetics
MH  - Receptors, Androgen/genetics
MH  - Sequence Analysis, DNA
MH  - von Willebrand Factor/genetics
EDAT- 2005/10/26 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/26 09:00
PHST- 2005/10/26 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/26 09:00 [entrez]
AID - UK68391910P8204P [pii]
AID - 10.1080/10635150500234534 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):719-30. doi: 10.1080/10635150500234534.

PMID- 16195215
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - Comparing host and parasite phylogenies: gyrodactylus flatworms jumping from goby
      to goby.
PG  - 710-8
AB  - The combination of exceptionally high species diversity, high host specificity,
      and a complex reproduction system raises many questions about the underlying
      mechanisms triggering speciation in the flatworm genus Gyrodactylus. The
      coevolutionary history with their goby hosts was investigated using both
      topology- and distance-based approaches; phylogenies were constructed of the V4
      region of the 18S rRNA and the complete ITS rDNA region for the parasites, and
      12S and 16S mtDNA fragments for the hosts. The overall fit between both trees was
      significant according to the topology-based programs (TreeMap 1.0, 2.0 beta and
      TreeFitter), but not according to the timed analysis in TreeMap 2.0 beta and the 
      distance-based method (ParaFit). An absolute timing of speciation events in host 
      and parasite ruled out the possibility of synchronous speciation for the gill
      parasites, favouring the distance-based result. Based on this information
      together with the biological background of host and parasite, the following
      TreeMap solution was selected. The group of gill parasites evolved from a host
      switch from G. arcuatus, parasitizing the three-spined stickleback onto the
      gobies, followed by several host-switching events among the respective goby
      hosts. The timing of these events is estimated to date back to the Late
      Pleistocene, suggesting a role for refugia-mediated mixing of parasite species.
      In contrast, it is suggested that co-speciation in the fin-parasites resulted in 
      several host-associated species complexes. This illustrates that phylogenetically
      conserved host-switching mimics the phylogenetic signature of co-speciation,
      confounding topology-based programs.
FAU - Huyse, Tine
AU  - Huyse T
AD  - Katholieke Universiteit Leuven, Laboratory of Aquatic Ecology, Ch. De
      Beriotstraat 32, Leuven B-3000, Belgium. Tine.Huyse@bio.kuleuven.ac.be
FAU - Volckaert, Filip A M
AU  - Volckaert FA
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Atlantic Ocean
MH  - Classification/*methods
MH  - DNA, Ribosomal/genetics
MH  - *Evolution, Molecular
MH  - *Genetic Variation
MH  - Host-Parasite Interactions
MH  - Perciformes/*genetics/*parasitology
MH  - *Phylogeny
MH  - Platyhelminths/*genetics
MH  - Species Specificity
EDAT- 2005/10/01 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/01 09:00
PHST- 2005/10/01 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/01 09:00 [entrez]
AID - M5642V5323201W74 [pii]
AID - 10.1080/10635150500221036 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):710-8. doi: 10.1080/10635150500221036.

PMID- 16195214
OWN - NLM
STAT- MEDLINE
DCOM- 20060505
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 5
DP  - 2005 Oct
TI  - How much data are needed to resolve a difficult phylogeny?: case study in
      Lamiales.
PG  - 697-709
AB  - Reconstructing phylogeny is a crucial target of contemporary biology, now
      commonly approached through computerized analysis of genetic sequence data. In
      angiosperms, despite recent progress at the ordinal level, many relationships
      between families remain unclear. Here we take a case study from Lamiales, an
      angiosperm order in which interfamilial relationships have so far proved
      particularly problematic. We examine the effect of changing one factor-the
      quantity of sequence data analyzed-on phylogeny reconstruction in this group. We 
      use simulation to estimate a priori the sequence data that would be needed to
      resolve an accurate, supported phylogeny of Lamiales. We investigate the effect
      of increasing the length of sequence data analyzed, the rate of substitution in
      the sequences used, and of combining gene partitions. This method could be a
      valuable technique for planning systematic investigations in other problematic
      groups. Our results suggest that increasing sequence length is a better way to
      improve support, resolution, and accuracy than employing sequences with a faster 
      substitution rate. Indeed, the latter may in some cases have detrimental effects 
      on phylogeny reconstruction. Further molecular sequencing-of at least 10,000
      bp-should result in a fully resolved and supported phylogeny of Lamiales, but at 
      present the problematic aspects of this tree model remain.
FAU - Wortley, Alexandra H
AU  - Wortley AH
AD  - Department of Plant Sciences, University of Oxford, South Parks Road, Oxford, OX1
      3RB, UK.
FAU - Rudall, Paula J
AU  - Rudall PJ
FAU - Harris, David J
AU  - Harris DJ
FAU - Scotland, Robert W
AU  - Scotland RW
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - Classification/*methods
MH  - Cluster Analysis
MH  - Computer Simulation
MH  - Magnoliopsida/*genetics
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - *Sample Size
MH  - Sequence Analysis, DNA
EDAT- 2005/10/01 09:00
MHDA- 2006/05/06 09:00
CRDT- 2005/10/01 09:00
PHST- 2005/10/01 09:00 [pubmed]
PHST- 2006/05/06 09:00 [medline]
PHST- 2005/10/01 09:00 [entrez]
AID - U34315546X797153 [pii]
AID - 10.1080/10635150500221028 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Oct;54(5):697-709. doi: 10.1080/10635150500221028.

PMID- 16126665
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Counterexample to a claim about the reconstruction of ancestral character states.
PG  - 693-5
FAU - Lucena, Brian
AU  - Lucena B
AD  - Division of Computer Science, University of California, Berkeley, California,
      USA. lucena@cs.berkeley.edu
FAU - Haussler, David
AU  - Haussler D
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence/genetics
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - *Evolution, Molecular
MH  - *Phylogeny
EDAT- 2005/08/30 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/30 09:00
PHST- 2005/08/30 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/30 09:00 [entrez]
AID - TGV18J7424428716 [pii]
AID - 10.1080/10635150590950344 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):693-5. doi: 10.1080/10635150590950344.

PMID- 16126664
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Species concepts, species boundaries and species identification: a view from the 
      tropics.
PG  - 689-93
FAU - Balakrishnan, Rohini
AU  - Balakrishnan R
AD  - Centre for Ecological Sciences, Indian Institute of Science, Bangalore, India.
      rohini@ces.iisc.ernet.in
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Conservation of Natural Resources/methods
MH  - Phylogeny
MH  - Species Specificity
MH  - *Terminology as Topic
EDAT- 2005/08/30 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/30 09:00
PHST- 2005/08/30 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/30 09:00 [entrez]
AID - L25KJ276101664V3 [pii]
AID - 10.1080/10635150590950308 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):689-93. doi: 10.1080/10635150590950308.

PMID- 16126663
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20170922
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Distances and directions in multidimensional shape spaces: implications for
      morphometric applications.
PG  - 678-88
FAU - Klingenberg, Christian Peter
AU  - Klingenberg CP
AD  - Faculty of Life Sciences, The University of Manchester, Michael Smith Building,
      Oxford Road, Manchester M13 9PT, United Kingdom. cpk@manchester.ac.uk
FAU - Monteiro, Leandro R
AU  - Monteiro LR
LA  - eng
GR  - BBS/B/12253/Biotechnology and Biological Sciences Research Council/United Kingdom
GR  - Wellcome Trust/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Anatomy/*methods
MH  - Biometry/*methods
MH  - Body Weights and Measures/*methods
MH  - Computational Biology/*methods
MH  - Models, Genetic
MH  - *Multivariate Analysis
MH  - Selection, Genetic
EDAT- 2005/08/30 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/30 09:00
PHST- 2005/08/30 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/30 09:00 [entrez]
AID - RUH24Q32508782N4 [pii]
AID - 10.1080/10635150590947258 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):678-88. doi: 10.1080/10635150590947258.

PMID- 16126662
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Molecular dates for the "cambrian explosion": the influence of prior assumptions.
PG  - 672-8
FAU - Welch, John J
AU  - Welch JJ
AD  - Centre for the Study of Evolution, School of Life Sciences, University of Sussex,
      Falmer, Brighton BN1 9QG, United Kingdom.
FAU - Fontanillas, Eric
AU  - Fontanillas E
FAU - Bromham, Lindell
AU  - Bromham L
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - Invertebrates/*genetics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Research Design
EDAT- 2005/08/30 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/30 09:00
PHST- 2005/08/30 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/30 09:00 [entrez]
AID - X02318VJ8183333M [pii]
AID - 10.1080/10635150590947212 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):672-8. doi: 10.1080/10635150590947212.

PMID- 16126661
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Getting to the roots of matrix representation.
PG  - 668-72
FAU - Bininda-Emonds, Olaf R P
AU  - Bininda-Emonds OR
AD  - Lehrstuhl fur Tierzucht, Technical University of Munich, Hochfeldweg 1, 85354
      Freising-Weihenstephan, Germany. Olaf.Bininda@tierzucht.tum.de
FAU - Beck, Robin M D
AU  - Beck RM
FAU - Purvis, Andy
AU  - Purvis A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computational Biology/*methods
MH  - *Phylogeny
MH  - Research Design
EDAT- 2005/08/30 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/30 09:00
PHST- 2005/08/30 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/30 09:00 [entrez]
AID - U4WW04663466R742 [pii]
AID - 10.1080/10635150590947113 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):668-72. doi: 10.1080/10635150590947113.

PMID- 16126660
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20181201
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Automated phylogenetic taxonomy: an example in the homobasidiomycetes
      (mushroom-forming fungi).
PG  - 660-8
FAU - Hibbett, David S
AU  - Hibbett DS
AD  - Biology Department, Clark University, Worcester, Massachusetts 01610, USA.
      dhibbett@black.clarku.edu
FAU - Nilsson, R Henrik
AU  - Nilsson RH
FAU - Snyder, Marc
AU  - Snyder M
FAU - Fonseca, Mario
AU  - Fonseca M
FAU - Costanzo, Janine
AU  - Costanzo J
FAU - Shonfeld, Moran
AU  - Shonfeld M
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Basidiomycota/*genetics
MH  - Classification/*methods
MH  - Electronic Data Processing/*methods
MH  - *Phylogeny
EDAT- 2005/08/30 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/30 09:00
PHST- 2005/08/30 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/30 09:00 [entrez]
AID - J02X4010540162P7 [pii]
AID - 10.1080/10635150590947104 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):660-8. doi: 10.1080/10635150590947104.

PMID- 16126659
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Poriferan mtDNA and animal phylogeny based on mitochondrial gene arrangements.
PG  - 651-9
AB  - Phylogenetic relationships among the metazoan phyla are the subject of an ongoing
      controversy. Analysis of mitochondrial gene arrangements is a powerful tool to
      investigate these relationships; however, its previous application outside of
      individual animal phyla has been hampered by the lack of informative out-group
      data. To address this shortcoming, we determined complete mitochondrial DNA
      sequences for the demosponges Geodia neptuni and Tethya actinia, two
      representatives of the most basal animal phylum, the Porifera. With sponges as an
      outgroup, we investigated phylogenetic relationships of nine bilaterian phyla
      using both breakpoint analysis of global mitochondrial gene arrangements and
      maximum parsimony analysis of mitochondrial gene adjacencies. Our results provide
      strong support for a group that includes protostome (but not deuterostome)
      coelomate, pseudocoelomate, and acoelomate animals, thus clearly rejecting the
      Coelomata hypothesis. Two other groups of bilaterian animals, Lophotrochozoa and 
      Ambulacraria, are also supported by our analyses. However, due to the remarkable 
      stability of mitochondrial gene arrangements in Deuterostomia and the Ecdysozoa, 
      conclusions on their evolutionary history cannot be drawn.
FAU - Lavrov, Dennis V
AU  - Lavrov DV
AD  - Departement de Biochimie, Universite de Montreal, Succursale Centre-Ville,
      Montreal, Que, H3C 3J7, Canada. dlavrov@iastate.edu
FAU - Lang, B Franz
AU  - Lang BF
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA, Mitochondrial/*genetics
MH  - Gene Order/*genetics
MH  - Invertebrates/*genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
MH  - Vertebrates/*genetics
EDAT- 2005/08/30 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/30 09:00
PHST- 2005/08/30 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/30 09:00 [entrez]
AID - XGM5J662H8257616 [pii]
AID - 10.1080/10635150500221044 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):651-9. doi: 10.1080/10635150500221044.

PMID- 16109706
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Patterns of cladogenesis in the venomous marine gastropod genus Conus from the
      Cape Verde islands.
PG  - 634-50
AB  - Isolated oceanic archipelagos are excellent model systems to study speciation,
      biogeography, and evolutionary factors underlying the generation of biological
      diversity. Despite the wealth of studies documenting insular speciation, few of
      them focused on marine organisms. Here, we reconstruct phylogenetic relationships
      among species of the marine venomous gastropod genus Conus from the Cape Verde
      archipelago. This small island chain located in the Central Atlantic hosts 10% of
      the worldwide species diversity of Conus. Analyses were based on mtDNA sequences,
      and a novel nuclear marker, a megalin-like protein, member of the low-density
      lipoprotein receptor gene family. The inferred phylogeny recovered two
      well-defined clades within Conus. One includes Cape Verde endemic species with
      larger shells, known as the "venulatus" complex together with C. pulcher from the
      Canary Islands. The other is composed of Cape Verde endemic and West Africa and
      Canary Island "small" shelled species. In both clades, nonendemic Conus were
      resolved as sister groups of the Cape Verde endemics, respectively. Our results
      indicate that the ancestors of "small" and "large" shelled lineages independently
      colonized Cape Verde. The resulting biogeographical pattern shows the grouping of
      most Cape Verde endemics in monophyletic island assemblages. Statistical tests
      supported a recent radiation event within the "small shell" clade. Using a
      molecular clock, we estimated that the colonization of the islands by the "small"
      shelled species occurred relatively close to the origin of the islands whereas
      the arrival of "large" shelled Conus is more recent. Our results suggest that the
      main factor responsible for species diversity in the archipelago may be
      allopatric speciation promoted by the reduced dispersal capacity of nonplanktonic
      lecithotrophic larvae.
FAU - Cunha, Regina L
AU  - Cunha RL
AD  - Departamento de Biodiversidad y Biologia Evolutiva, Museo Nacional de Ciencias
      Naturales-CSIC, Jose Gutierrez Abascal, 2, Madrid 28006, Spain.
      reginac@mncn.csic.es
FAU - Castilho, Rita
AU  - Castilho R
FAU - Ruber, Lukas
AU  - Ruber L
FAU - Zardoya, Rafael
AU  - Zardoya R
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Africa, Western
MH  - Animal Structures/*anatomy &amp; histology
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Conus Snail/anatomy &amp; histology/*genetics
MH  - DNA, Mitochondrial/genetics
MH  - *Demography
MH  - *Evolution, Molecular
MH  - Geography
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Population Dynamics
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2005/08/20 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/20 09:00
PHST- 2005/08/20 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/20 09:00 [entrez]
AID - M322K43067627811 [pii]
AID - 10.1080/106351591007471 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):634-50. doi: 10.1080/106351591007471.

PMID- 16109705
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Untangling long branches: identifying conflicting phylogenetic signals using
      spectral analysis, neighbor-net, and consensus networks.
PG  - 620-33
AB  - Long-branch attraction is a well-known source of systematic error that can
      mislead phylogenetic methods; it is frequently invoked post hoc, upon recovering 
      a different tree from the one expected based on prior evidence. We demonstrate
      that methods that do not force the data onto a single tree, such as spectral
      analysis, Neighbor-Net, and consensus networks, can be used to detect conflicting
      signals within the data, including those caused by long-branch attraction. We
      illustrate this approach using a set of taxa from three unambiguously
      monophyletic families within the Pelecaniformes: the darters, the cormorants and 
      shags, and the gannets and boobies. These three families are universally
      acknowledged as forming a monophyletic group, but the relationship between the
      families remains contentious. Using sequence data from three mitochondrial genes 
      (12S, ATPase 6, and ATPase 8) we demonstrate that the relationship between these 
      three families is difficult to resolve because they are separated by a short
      internal branch and there are conflicting signals due to long-branch attraction, 
      which are confounded with nonhomogeneous sequence evolution across the different 
      genes. Spectral analysis, Neighbor-Net, and consensus networks reveal conflicting
      signals regarding the placement of one of the darters, with support found for
      darter monophyly, but also support for a conflicting grouping with the outgroup, 
      pelicans. Furthermore, parsimony and maximum-likelihood analyses produced
      different trees, with one of the two most parsimonious trees not supporting the
      monophyly of the darters. Monte Carlo simulations, however, were not sensitive
      enough to reveal long-branch attraction unless the branches are longer than those
      actually observed. These results indicate that spectral analysis, Neighbor-Net,
      and consensus networks offer a powerful approach to detecting and understanding
      the source of conflicting signals within phylogenetic data.
FAU - Kennedy, Martyn
AU  - Kennedy M
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, Department of Zoology,
      University of Otago, Dunedin, New Zealand. martyn.kennedy@stonebow.otago.ac.nz
FAU - Holland, Barbara R
AU  - Holland BR
FAU - Gray, Russell D
AU  - Gray RD
FAU - Spencer, Hamish G
AU  - Spencer HG
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Birds/*genetics
MH  - Classification/*methods
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Research Design
MH  - Sequence Analysis, DNA
EDAT- 2005/08/20 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/20 09:00
PHST- 2005/08/20 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/20 09:00 [entrez]
AID - PN021G5778R77243 [pii]
AID - 10.1080/106351591007462 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):620-33. doi: 10.1080/106351591007462.

PMID- 16109704
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - The logical basis of phylogenetic taxonomy.
PG  - 595-619
AB  - Phylogenetic taxonomy, like modern Linnean taxonomy, was modeled on a
      phylogenetic tree rather than a cladogram and, like its predecessor, perpetuates 
      the use of morphology as a means of recognizing clades. Both practices have
      generated confusion in graphical representation, operational terminology, and
      definitional rationale in phylogenetic taxonomy, the history of which is traced. 
      The following points are made: (1) cladograms, rather than trees or hybrid
      cladogram-trees, provide the framework for the simplest graphical depiction of
      phylogenetic definitions; (2) a complete notational scheme for phylogenetic
      definitions is presented that distinguishes symbolic notation from shorthand and 
      longhand versions; (3) phylogenetic definitions are composed of three components 
      (paradigm, specifier, qualifier) arranged in two fundamental patterns-node and
      stem; (4) apomorphies do not constitute a fundamental definitional pattern but
      rather serve to qualify a stem-based definition (as do time and geographic
      range); (5) formulation of phylogenetic definitions involves three heuristic
      criteria (stability, simplicity, prior use); (6) reasoned definitional revision
      is encouraged and better defined (textual substitution, first-and second-order
      revision); and (7) a database, TaxonSearch, allows rapid recall of taxonomic and 
      definitional information.
FAU - Sereno, Paul C
AU  - Sereno PC
AD  - Department of Organismal Biology and Anatomy, University of Chicago, Chicago,
      Illinois 60637, USA. dinosaur@uchicago.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Databases, Factual
MH  - *Logic
MH  - *Phylogeny
MH  - Species Specificity
MH  - *Terminology as Topic
EDAT- 2005/08/20 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/20 09:00
PHST- 2005/08/20 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/20 09:00 [entrez]
AID - U122854XG4044105 [pii]
AID - 10.1080/106351591007453 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):595-619. doi: 10.1080/106351591007453.

PMID- 16085576
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Evolutionary rates analysis of Leguminosae implicates a rapid diversification of 
      lineages during the tertiary.
PG  - 575-94
AB  - Tertiary macrofossils of the flowering plant family Leguminosae (legumes) were
      used as time constraints to estimate ages of the earliest branching clades
      identified in separate plastid matK and rbcL gene phylogenies. Penalized
      likelihood rate smoothing was performed on sets of Bayesian likelihood trees
      generated with the AIC-selected GTR+ Gamma +I substitution model. Unequivocal
      legume fossils dating from the Recent continuously back to about 56 million years
      ago were used to fix the family stem clade at 60 million years (Ma), and at 1-Ma 
      intervals back to 70 Ma. Specific fossils that showed distinctive combinations of
      apomorphic traits were used to constrain the minimum age of 12 specific internal 
      nodes. These constraints were placed on stem rather than respective crown clades 
      in order to bias for younger age estimates. Regardless, the mean age of the
      legume crown clade differs by only 1.0 to 2.5 Ma from the fixed age of the legume
      stem clade. Additionally, the oldest caesalpinioid, mimosoid, and papilionoid
      crown clades show approximately the same age range of 39 to 59 Ma. These findings
      all point to a rapid family-wide diversification, and predict few if any legume
      fossils prior to the Cenozoic. The range of the matK substitution rate, 2.1-24.6 
      x 10(-10) substitutions per site per year, is higher than that of rbcL, 1.6- 8.6 
      x 10(-10), and is accompanied by more uniform rate variation among codon
      positions. The matK and rbcL substitution rates are highly correlated across the 
      legume family. For example, both loci have the slowest substitution rates among
      the mimosoids and the fastest rates among the millettioid legumes. This explains 
      why groups such as the millettioids are amenable to species-level phylogenetic
      analysis with these loci, whereas other legume groups are not.
FAU - Lavin, Matt
AU  - Lavin M
AD  - Department of Plant Sciences, Montana State University, Bozeman, Montana 59717,
      USA. mlavin@montana.edu
FAU - Herendeen, Patrick S
AU  - Herendeen PS
FAU - Wojciechowski, Martin F
AU  - Wojciechowski MF
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 4.1.1.39 (RbcL protein, plastid)
RN  - EC 4.1.1.39 (Ribulose-Bisphosphate Carboxylase)
SB  - IM
MH  - Bayes Theorem
MH  - *Evolution, Molecular
MH  - Fabaceae/*anatomy &amp; histology/*genetics
MH  - *Fossils
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Plastids/genetics
MH  - Ribulose-Bisphosphate Carboxylase/genetics
EDAT- 2005/08/09 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/09 09:00
PHST- 2005/08/09 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/09 09:00 [entrez]
AID - L0023715T1544124 [pii]
AID - 10.1080/10635150590947131 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):575-94. doi: 10.1080/10635150590947131.

PMID- 16085575
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Phylogenetic signal in bone microstructure of sauropsids.
PG  - 562-74
AB  - In spite of the fact that the potential usefulness of bone histology in
      systematics has been discussed for over one and a half centuries, the presence of
      a phylogenetic signal in the variation of histological characters has rarely been
      assessed. A quantitative assessment of phylogenetic signal in bone histological
      characters could provide a justification for performing optimizations of these
      traits onto independently generated phylogenetic trees (as has been done in
      recent years). Here we present an investigation on the quantification of the
      phylogenetic signal in the following bone histological, microanatomical, and
      morphological traits in a sample of femora of 35 species of sauropsids: vascular 
      density, vascular orientation, index of Haversian remodeling, cortical thickness,
      and cross-sectional area (bone size). For this purpose, we use two methods,
      regressions on distance matrices tested for significance using permutations (a
      Mantel test) and random tree length distribution. Within sauropsids, these bone
      microstructural traits have an optimal systematic value in archosaurs. In this
      taxon, a Mantel test shows that the phylogeny explains 81.8% of the variation of 
      bone size and 86.2% of the variation of cortical thickness. In contrast, a Mantel
      test suggests that the phylogenetic signal in histological traits is weak:
      although the phylogeny explains 18.7% of the variation of vascular density in
      archosaurs, the phylogenetic signal is not significant either for vascular
      orientation or for the index of Haversian remodeling. However, Mantel tests seem 
      to underestimate the proportion of variance of the dependent character explained 
      by the phylogeny, as suggested by a PVR (phylogenetic eigenvector) analysis. We
      also deal with some complementary questions. First, we evaluate the functional
      dependence of bone vascular density on bone size by using phylogenetically
      independent contrasts. Second, we perform a variation partitioning analysis and
      show that the phylogenetic signal in bone vascular density is not a by-product of
      phylogentic signal in bone size. Finally, we analyze the evolution of cortical
      thickness in diapsids by using an optimization by squared change parsimony and
      discuss the functional significance of this character in terms of decreased
      buoyancy in crocodiles and mass saving in birds. These results are placed in the 
      framework of the constructional morphology model, according to which the
      variation of a character in a clade has a historical (phylogenetic) component, a 
      functional (adaptive) component, and a structural (architectural) component.
FAU - Cubo, J
AU  - Cubo J
AD  - Comparative Osteohistology, FRE CNRS 2696, Pierre et Marie Curie University,
      75005 Paris, France. cubo@ccr.jussieu.fr
FAU - Ponton, F
AU  - Ponton F
FAU - Laurin, M
AU  - Laurin M
FAU - de Margerie, E
AU  - de Margerie E
FAU - Castanet, J
AU  - Castanet J
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Analysis of Variance
MH  - Animals
MH  - Birds/*anatomy &amp; histology/genetics
MH  - Bone and Bones/*anatomy &amp; histology/blood supply
MH  - Classification/*methods
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Regression Analysis
MH  - Reptiles/*anatomy &amp; histology/genetics
EDAT- 2005/08/09 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/09 09:00
PHST- 2005/08/09 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/09 09:00 [entrez]
AID - V6M87663251G5086 [pii]
AID - 10.1080/10635150591003461 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):562-74. doi: 10.1080/10635150591003461.

PMID- 16085574
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20071114
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Simultaneous statistical multiple alignment and phylogeny reconstruction.
PG  - 548-61
AB  - Although the reconstruction of phylogenetic trees and the computation of multiple
      sequence alignments are highly interdependent, these two areas of research lead
      quite separate lives, the former often making use of stochastic modeling, whereas
      the latter normally does not. Despite the fact that reasonable insertion and
      deletion models for sequence pairs were already introduced more than 10 years
      ago, they have only recently been applied to multiple alignment and only in their
      simplest version. In this paper we present and discuss a strategy based on
      simulated annealing, which makes use of these models to infer a phylogenetic tree
      for a set of DNA or protein sequences together with the sequences'indel history, 
      i.e., their multiple alignment augmented with information about the positioning
      of insertion and deletion events in the tree. Our method is also the first
      application of the TKF2 model in the context of multiple sequence alignment. We
      validate the method via simulations and illustrate it using a data set of primate
      mtDNA.
FAU - Fleissner, Roland
AU  - Fleissner R
AD  - Initiative for Bioinformatics and Evolutionary Studies (IBEST), University of
      Idaho Moscow, Idaho 83844-1103, USA. fleissne@uidaho.edu
FAU - Metzler, Dirk
AU  - Metzler D
FAU - von Haeseler, Arndt
AU  - von Haeseler A
LA  - eng
GR  - P20RR16448/RR/NCRR NIH HHS/United States
GR  - P20RR16454/RR/NCRR NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Animals
MH  - Base Sequence
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - Mutation/genetics
MH  - *Phylogeny
MH  - Primates/genetics
MH  - Sequence Alignment/*methods
EDAT- 2005/08/09 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/09 09:00
PHST- 2005/08/09 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/09 09:00 [entrez]
AID - L126865V16516754 [pii]
AID - 10.1080/10635150590950371 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):548-61. doi: 10.1080/10635150590950371.

PMID- 16085573
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Integration of morphological data sets for phylogenetic analysis of Amniota: the 
      importance of integumentary characters and increased taxonomic sampling.
PG  - 530-47
AB  - Several mutually exclusive hypotheses have been advanced to explain the
      phylogenetic position of turtles among amniotes. Traditional morphology-based
      analyses place turtles among extinct anapsids (reptiles with a solid skull roof),
      whereas more recent studies of both morphological and molecular data support an
      origin of turtles from within Diapsida (reptiles with a doubly fenestrated skull 
      roof). Evaluation of these conflicting hypotheses has been hampered by
      nonoverlapping taxonomic samples and the exclusion of significant taxa from
      published analyses. Furthermore, although data from soft tissues and anatomical
      systems such as the integument may be particularly relevant to this problem, they
      are often excluded from large-scale analyses of morphological systematics. Here, 
      conflicting hypotheses of turtle relationships are tested by (1) combining
      published data into a supermatrix of morphological characters to address issues
      of character conflict and missing data; (2) increasing taxonomic sampling by more
      than doubling the number of operational taxonomic units to test internal
      relationships within suprageneric ingroup taxa; and (3) increasing character
      sampling by approximately 25% by adding new data on the osteology and histology
      of the integument, an anatomical system that has been historically
      underrepresented in morphological systematics. The morphological data set
      assembled here represents the largest yet compiled for Amniota. Reevaluation of
      character data from prior studies of amniote phylogeny favors the hypothesis that
      turtles indeed have diapsid affinities. Addition of new ingroup taxa alone leads 
      to a decrease in overall phylogenetic resolution, indicating that existing
      characters used for amniote phylogeny are insufficient to explain the evolution
      of more highly nested taxa. Incorporation of new data from the soft and osseous
      components of the integument, however, helps resolve relationships among both
      basal and highly nested amniote taxa. Analysis of a data set compiled from
      published sources and data original to this study supports monophyly of Amniota, 
      Synapsida, Reptilia, Parareptilia, Eureptilia, Eosuchia, Diapsida, Neodiapsida,
      Sauria, Lepidosauria, and Archosauriformes, as well as several more highly nested
      divisions within the latter two clades. Turtles are here resolved as the sister
      taxon to a monophyletic Lepidosauria (squamates + Sphenodon), a novel
      phylogenetic position that nevertheless is consistent with recent molecular and
      morphological studies that have hypothesized diapsid affinities for this clade.
FAU - Hill, Robert V
AU  - Hill RV
AD  - Department of Anatomical Sciences, Stony Brook University, Stony Brook, New York 
      11794-8081, USA. rhill01@nyit.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Integumentary System/*anatomy &amp; histology
MH  - *Phylogeny
MH  - *Research Design
MH  - Sample Size
MH  - Turtles/*anatomy &amp; histology/genetics
EDAT- 2005/08/09 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/08/09 09:00
PHST- 2005/08/09 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/08/09 09:00 [entrez]
AID - H6472T3P0V3K3270 [pii]
AID - 10.1080/10635150590950326 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):530-47. doi: 10.1080/10635150590950326.

PMID- 16051588
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Phylogenetic diversity and the greedy algorithm.
PG  - 527-9
AB  - Given a phylogenetic tree with leaves labeled by a collection of species, and
      with weighted edges, the "phylogenetic diversity" of any subset of the species is
      the sum of the edge weights of the minimal subtree connecting the species. This
      measure is relevant in biodiversity conservation where one may wish to compare
      different subsets of species according to how much evolutionary variation they
      encompass. In this note we show that phylogenetic diversity has an attractive
      mathematical property that ensures that we can solve the following problem easily
      by the greedy algorithm: find a subset of the species of any given size k of
      maximal phylogenetic diversity. We also describe an extension of this result that
      also allows weights to be assigned to species.
FAU - Steel, Mike
AU  - Steel M
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, Biomathematics Research 
      Centre, University of Canterbury, Christchurch, New Zealand.
      m.steel@math.canterbury.ac.nz
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - *Models, Theoretical
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2005/07/30 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/07/30 09:00
PHST- 2005/07/30 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/07/30 09:00 [entrez]
AID - R310621741343045 [pii]
AID - 10.1080/10635150590947023 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):527-9. doi: 10.1080/10635150590947023.

PMID- 16051587
OWN - NLM
STAT- MEDLINE
DCOM- 20060120
LR  - 20050729
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 4
DP  - 2005 Aug
TI  - Curious parallels and curious connections--phylogenetic thinking in biology and
      historical linguistics.
PG  - 513-26
AB  - In The Descent of Man (1871), Darwin observed "curious parallels" between the
      processes of biological and linguistic evolution. These parallels mean that
      evolutionary biologists and historical linguists seek answers to similar
      questions and face similar problems. As a result, the theory and methodology of
      the two disciplines have evolved in remarkably similar ways. In addition to
      Darwin's curious parallels of process, there are a number of equally curious
      parallels and connections between the development of methods in biology and
      historical linguistics. Here we briefly review the parallels between biological
      and linguistic evolution and contrast the historical development of phylogenetic 
      methods in the two disciplines. We then look at a number of recent studies that
      have applied phylogenetic methods to language data and outline some current
      problems shared by the two fields.
FAU - Atkinson, Quentin D
AU  - Atkinson QD
AD  - Department of Psychology, University of Auckland, Private Bag 92019, Auckland
      1020, New Zealand.
FAU - Gray, Russell D
AU  - Gray RD
LA  - eng
PT  - Historical Article
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biology/*history/*methods
MH  - Classification/*methods
MH  - History, 16th Century
MH  - History, 18th Century
MH  - History, 19th Century
MH  - History, 20th Century
MH  - History, 21st Century
MH  - History, Ancient
MH  - History, Medieval
MH  - Linguistics/*history/*methods
MH  - *Phylogeny
EDAT- 2005/07/30 09:00
MHDA- 2006/01/21 09:00
CRDT- 2005/07/30 09:00
PHST- 2005/07/30 09:00 [pubmed]
PHST- 2006/01/21 09:00 [medline]
PHST- 2005/07/30 09:00 [entrez]
AID - X14P72450R338642 [pii]
AID - 10.1080/10635150590950317 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Aug;54(4):513-26. doi: 10.1080/10635150590950317.

PMID- 16021696
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 2
DP  - 2005 Apr
TI  - Evidence for multiple reversals of asymmetric mutational constraints during the
      evolution of the mitochondrial genome of metazoa, and consequences for
      phylogenetic inferences.
PG  - 277-98
AB  - Mitochondrial DNA (mtDNA) sequences are comonly used for inferring phylogenetic
      relationships. However, the strand-specific bias in the nucleotide composition of
      the mtDNA, which is thought to reflect assymetric mutational constraints,
      combined with the important compositional heterogeneity among taxa, are known to 
      be highly problematic for phylogenetic analyses. Here, nucleotide composition was
      compared across 49 species of Metazoa (34 arthropods, 2 annelids, 2 molluscs, and
      11 deuterosomes), and analyzed for a mtDNA fragment including six protein-coding 
      genes, i.e., atp6, atp8, cox1, cox2, cox3, and nad2. The analyses show that most 
      metazoan species present a clear strand assymetry, where one strand is biased in 
      favor of A and C, whereas the other strand has reverse bias, i.e. in favor of T
      and G. the origin of this strand bias can be related to assymetric mutational
      constraints involving deaminations of A and C nucleotides during the replication 
      and/or transcription processes. The analyses reveal that six unrelated genera are
      characterized by a reversal of the usual strand bias, i.e., Argiope (Araneae),
      Euscorpius (Scorpiones), Tigrioupus (Maxillopoda), Branchiostoma
      (Cephalochordata) Florometra (Echinodermata), and Katharina (Mollusca). It is
      proposed that assymetric mutational constraints have been independantly reversed 
      in these six genera, through an inversion of the control region, i.e., the region
      that contains most regulatory elements for replication and transcription of the
      mtDNA. We show that reversals of assymetric mutational constraints have dramatic 
      consequences on the phylogenetic analyses, as taxa characterized by reverse
      strand bias tend to group together due to long-branch attraction artifacts. We
      propose a new method for limiting this specific problem in tree reconstruction
      under the Bayesian approach. We apply our method to deal with the question of
      phylogenetic relationships of the major lineages of Arthropoda, This new approach
      provides a better congruence with nuclear analyses based on mtDNA sequences, our 
      data suggest that Chelicerata, Crustacea, Myriapoda, Pancrustacea, and
      Paradoxopoda are monophyletic.
FAU - Hassanin, Alexandre
AU  - Hassanin A
AD  - Museum National d'Histoire Naturelle, Department Systematique et Evolution, Case 
      Postale, Paris, France. hassanin@mnhn.fr
FAU - Leger, Nelly
AU  - Leger N
FAU - Deutsch, Jean
AU  - Deutsch J
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Composition/*genetics
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computational Biology
MH  - DNA, Mitochondrial/*genetics
MH  - *Evolution, Molecular
MH  - Invertebrates/*genetics
MH  - Locus Control Region/genetics
MH  - Models, Genetic
MH  - Mutation/*genetics
MH  - *Phylogeny
EDAT- 2005/07/16 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/16 09:00
PHST- 2005/07/16 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/16 09:00 [entrez]
AID - 10.1080/10635150590947843 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Apr;54(2):277-98. doi: 10.1080/10635150590947843.

PMID- 16012115
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - Estimating divergence times in phylogenetic trees without a molecular clock.
PG  - 500-7
FAU - Britton, Tom
AU  - Britton T
AD  - Department of Mathematics, Stockholm University, SE-106 91, Stockholm, Sweden.
      tom.britton@math.su.se
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Probability
MH  - Time Factors
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - G0WW573682625824 [pii]
AID - 10.1080/10635150590947311 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):500-7. doi: 10.1080/10635150590947311.

PMID- 16012114
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - Choosing the best genes for the job: the case for stationary genes in
      genome-scale phylogenetics.
PG  - 493-500
FAU - Collins, Timothy M
AU  - Collins TM
AD  - Department of Biological Sciences, Florida International University, University
      Park, Miami, FL 33199, USA. collinst@fiu.edu
FAU - Fedrigo, Olivier
AU  - Fedrigo O
FAU - Naylor, Gavin J P
AU  - Naylor GJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
SB  - IM
MH  - Classification/*methods
MH  - Codon/genetics
MH  - Computer Simulation
MH  - *Data Interpretation, Statistical
MH  - Genes/*genetics
MH  - Genetic Variation
MH  - *Genome
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Selection Bias
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - HM7J71847HN21256 [pii]
AID - 10.1080/10635150590947339 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):493-500. doi: 10.1080/10635150590947339.

PMID- 16012113
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20071114
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - Hidden likelihood support in genomic data: can forty-five wrongs make a right?
PG  - 483-92
AB  - Combined analysis of multiple phylogenetic data sets can reveal emergent
      character support that is not evident in separate analyses of individual data
      sets. Previous parsimony analyses have shown that this hidden support often
      accounts for a large percentage of the overall phylogenetic signal in cladistic
      studies. Here, reanalysis of a large comparative genomic data set for yeast
      (genus Saccharomyces) demonstrates that hidden support can be an important factor
      in maximum likelihood analyses of multiple data sets as well. Emergent signal in 
      a concatenation of 106 genes was responsible for up to 64% of the likelihood
      support at a particular node (the difference in log likelihood scores between
      optimal topologies that included and excluded a supported clade). A grouping of
      four yeast species (S. cerevisiae, S. paradoxus, S. mikatae, and S. kudriavzevii)
      was robustly supported by combined analysis of all 106 genes, but separate
      analyses of individual genes suggested numerous conflicts. Forty-eight genes
      strictly contradicted S. cerevisiae + S. paradoxus + S. mikatae + S. kudriavzevii
      in separate analyses, but combined likelihood analyses that included up to 45 of 
      the "wrong" data sets supported this group. Extensive hidden support also emerged
      in a combined likelihood analysis of 41 genes that each recovered the exact same 
      topology in separate analyses of the individual genes. These results show that
      isolated analyses of individual data sets can mask congruence and distort
      interpretations of clade stability, even in strictly model-based phylogenetic
      methods. Consensus and supertree procedures that ignore hidden phylogenetic
      signals are, at best, incomplete.
FAU - Gatesy, John
AU  - Gatesy J
AD  - Department of Biology, University of California-Riverside, Riverside, CA 92521,
      USA. john.gatesy@ucr.edu
FAU - Baker, Richard H
AU  - Baker RH
LA  - eng
GR  - 1F32GM67463-01/GM/NIGMS NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computational Biology
MH  - *Data Interpretation, Statistical
MH  - *Genome, Fungal
MH  - *Likelihood Functions
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Saccharomyces/*genetics
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - G94267T37JV300HP [pii]
AID - 10.1080/10635150590945368 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):483-92. doi: 10.1080/10635150590945368.

PMID- 16012112
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - Analysis and visualization of tree space.
PG  - 471-82
AB  - We explored the use of multidimensional scaling (MDS) of tree-to-tree pairwise
      distances to visualize the relationships among sets of phylogenetic trees. We
      found the technique to be useful for exploring "tree islands" (sets of
      topologically related trees among larger sets of near-optimal trees), for
      comparing sets of trees obtained from bootstrapping and Bayesian sampling, for
      comparing trees obtained from the analysis of several different genes, and for
      comparing multiple Bayesian analyses. The technique was also useful as a teaching
      aid for illustrating the progress of a Bayesian analysis and as an exploratory
      tool for examining large sets of phylogenetic trees. We also identified some
      limitations to the method, including distortions of the multidimensional tree
      space into two dimensions through the MDS technique, and the definition of the
      MDS-defined space based on a limited sample of trees. Nonetheless, the technique 
      is a useful approach for the analysis of large sets of phylogenetic trees.
FAU - Hillis, David M
AU  - Hillis DM
AD  - Section of Integrative Biology and Center for Computational Biology and
      Bioinformatics, The University of Texas at Austin, Austin, TX, 78712, USA.
      dhillis@mail.utexas.edu
FAU - Heath, Tracy A
AU  - Heath TA
FAU - St John, Katherine
AU  - St John K
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - *Data Interpretation, Statistical
MH  - *Phylogeny
MH  - *Software
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - L37747M5V37R177K [pii]
AID - 10.1080/10635150590946961 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):471-82. doi: 10.1080/10635150590946961.

PMID- 16012111
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20071114
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - Branch-length prior influences Bayesian posterior probability of phylogeny.
PG  - 455-70
AB  - The Bayesian method for estimating species phylogenies from molecular sequence
      data provides an attractive alternative to maximum likelihood with nonparametric 
      bootstrap due to the easy interpretation of posterior probabilities for trees and
      to availability of efficient computational algorithms. However, for many data
      sets it produces extremely high posterior probabilities, sometimes for apparently
      incorrect clades. Here we use both computer simulation and empirical data
      analysis to examine the effect of the prior model for internal branch lengths. We
      found that posterior probabilities for trees and clades are sensitive to the
      prior for internal branch lengths, and priors assuming long internal branches
      cause high posterior probabilities for trees. In particular, uniform priors with 
      high upper bounds bias Bayesian clade probabilities in favor of extreme values.
      We discuss possible remedies to the problem, including empirical and full
      Bayesian methods and subjective procedures suggested in Bayesian hypothesis
      testing. Our results also suggest that the bootstrap proportion and Bayesian
      posterior probability are different measures of accuracy, and that the bootstrap 
      proportion, if interpreted as the probability that the clade is true, can be
      either too liberal or too conservative.
FAU - Yang, Ziheng
AU  - Yang Z
AD  - Department of Biology, University College London, Darwin Building, Gower Street
      London WC1E 6BT, United Kingdom. z.yang@ucl.ac.uk
FAU - Rannala, Bruce
AU  - Rannala B
LA  - eng
GR  - HG01988/HG/NHGRI NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Data Interpretation, Statistical
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Plants/genetics
MH  - Probability
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - L278573T10358PQ5 [pii]
AID - 10.1080/10635150590945313 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):455-70. doi: 10.1080/10635150590945313.

PMID- 16012110
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - Placing paleopolyploidy in relation to taxon divergence: a phylogenetic analysis 
      in legumes using 39 gene families.
PG  - 441-54
AB  - Young polyploid events are easily diagnosed by various methods, but older
      polyploid events become increasingly difficult to identify as chromosomal
      rearrangements, tandem gene or partial chromosome duplications, changes in
      substitution rates among duplicated genes, pseudogenization or locus loss, and
      interlocus interactions complicate the means of inferring past genetic events.
      Genomic data have provided valuable information about the polyploid history of
      numerous species, but on their own fail to show whether related species, each
      with a polyploid past, share a particular polyploid event. A phylogenetic
      approach provides a powerful method to determine this but many processes may
      mislead investigators. These processes can affect individual gene trees, but most
      likely will not affect all genes, and almost certainly will not affect all genes 
      in the same way. Thus, a multigene approach, which combines the large-scale
      aspect of genomics with the resolution of phylogenetics, has the power to
      overcome these difficulties and allow us to infer genomic events further into the
      past than would otherwise be possible. Previous work using synonymous distances
      among gene pairs within species has shown evidence for large-scale duplications
      in the legumes Glycine max and Medicago truncatula. We present a case study using
      39 gene families, each with three or four members in G. max and the putative
      orthologues in M. truncatula, rooted using Arabidopsis thaliana. We tested
      whether the gene duplications in these legumes occurred separately in each
      lineage after their divergence (Hypothesis 1), or whether they share a round of
      gene duplications (Hypothesis 2). Many more gene family topologies supported
      Hypothesis 2 over Hypothesis 1 (11 and 2, respectively), even after synonymous
      distance analysis revealed that some topologies were providing misleading
      results. Only ca. 33% of genes examined support either hypothesis, which strongly
      suggests that single gene family approaches may be insufficient when studying
      ancient events with nuclear DNA. Our results suggest that G. max and M.
      truncatula, along with approximately 7000 other legume species from the same
      clade, share an ancient round of gene duplications, either due to polyploidy or
      to some other process.
FAU - Pfeil, B E
AU  - Pfeil BE
AD  - Department of Plant Biology, 228 Plant Sciences Building, Cornell University,
      Ithaca, Ithaca, NY 14853, USA. bep27@cornell.edu
FAU - Schlueter, J A
AU  - Schlueter JA
FAU - Shoemaker, R C
AU  - Shoemaker RC
FAU - Doyle, J J
AU  - Doyle JJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computational Biology
MH  - *Evolution, Molecular
MH  - Expressed Sequence Tags
MH  - Fabaceae/*genetics
MH  - *Gene Duplication
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Multigene Family/*genetics
MH  - *Phylogeny
MH  - *Polyploidy
MH  - Species Specificity
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - H4755N5601267816 [pii]
AID - 10.1080/10635150590945359 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):441-54. doi: 10.1080/10635150590945359.

PMID- 16012109
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - The systematic component of phylogenetic error as a function of taxonomic
      sampling under parsimony.
PG  - 432-40
AB  - The effect of taxonomic sampling on phylogenetic accuracy under parsimony is
      examined by simulating nucleotide sequence evolution. Random error is minimized
      by using very large numbers of simulated characters. This allows estimation of
      the consistency behavior of parsimony, even for trees with up to 100 taxa. Data
      were simulated on 8 distinct 100-taxon model trees and analyzed as stratified
      subsets containing either 25 or 50 taxa, in addition to the full 100-taxon data
      set. Overall accuracy decreased in a majority of cases when taxa were added.
      However, the magnitude of change in the cases in which accuracy increased was
      larger than the magnitude of change in the cases in which accuracy decreased, so,
      on average, overall accuracy increased as more taxa were included. A stratified
      sampling scheme was used to assess accuracy for an initial subsample of 25 taxa. 
      The 25-taxon analyses were compared to 50- and 100-taxon analyses that were
      pruned to include only the original 25 taxa. On average, accuracy for the 25 taxa
      was improved by taxon addition, but there was considerable variation in the
      degree of improvement among the model trees and across different rates of
      substitution.
FAU - Debry, Ronald W
AU  - Debry RW
AD  - Department of Biological Sciences, University of Cincinnati, Box 210006,
      Cincinnati, OH, 45221-0006, USA. ron.debry@uc.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Evolution, Molecular
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - *Research Design
MH  - *Sample Size
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - U65GTX3688558702 [pii]
AID - 10.1080/10635150590946745 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):432-40. doi: 10.1080/10635150590946745.

PMID- 16012108
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - The shape of supertrees to come: tree shape related properties of fourteen
      supertree methods.
PG  - 419-31
AB  - Using a simple example and simulations, we explore the impact of input tree shape
      upon a broad range of supertree methods. We find that input tree shape can affect
      how conflict is resolved by several supertree methods and that input tree shape
      effects may be substantial. Standard and irreversible matrix representation with 
      parsimony (MRP), MinFlip, duplication-only Gene Tree Parsimony (GTP), and an
      implementation of the average consensus method have a tendency to resolve
      conflict in favor of relationships in unbalanced trees. Purvis MRP and the
      average dendrogram method appear to have an opposite tendency. Biases with
      respect to tree shape are correlated with objective functions that are based upon
      unusual asymmetric tree-to-tree distance or fit measures. Split, quartet, and
      triplet fit, most similar supertree, and MinCut methods (provided the latter are 
      interpreted as Adams consensus-like supertrees) are not revealed to have any bias
      with respect to tree shape by our example, but whether this holds more generally 
      is an open problem. Future development and evaluation of supertree methods should
      consider explicitly the undesirable biases and other properties that we
      highlight. In the meantime, use of a single, arbitrarily chosen supertree method 
      is discouraged. Use of multiple methods and/or weighting schemes may allow
      practical assessment of the extent to which inferences from real data depend upon
      methodological biases with respect to input tree shape or size.
FAU - Wilkinson, Mark
AU  - Wilkinson M
AD  - Department of Zoology, The Natural History Museum, London SW7 5BD, United
      Kingdom. marw@nhm.ac.uk
FAU - Cotton, James A
AU  - Cotton JA
FAU - Creevey, Chris
AU  - Creevey C
FAU - Eulenstein, Oliver
AU  - Eulenstein O
FAU - Harris, Simon R
AU  - Harris SR
FAU - Lapointe, Francois-Joseph
AU  - Lapointe FJ
FAU - Levasseur, Claudine
AU  - Levasseur C
FAU - McInerney, James O
AU  - McInerney JO
FAU - Pisani, Davide
AU  - Pisani D
FAU - Thorley, Joseph L
AU  - Thorley JL
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Data Interpretation, Statistical
MH  - *Phylogeny
MH  - Research Design
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - W830404733278661 [pii]
AID - 10.1080/10635150590949832 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):419-31. doi: 10.1080/10635150590949832.

PMID- 16012107
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20071114
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - Joint Bayesian estimation of alignment and phylogeny.
PG  - 401-18
AB  - We describe a novel model and algorithm for simultaneously estimating multiple
      molecular sequence alignments and the phylogenetic trees that relate the
      sequences. Unlike current techniques that base phylogeny estimates on a single
      estimate of the alignment, we take alignment uncertainty into account by
      considering all possible alignments. Furthermore, because the alignment and
      phylogeny are constructed simultaneously, a guide tree is not needed. This
      sidesteps the problem in which alignments created by progressive alignment are
      biased toward the guide tree used to generate them. Joint estimation also allows 
      us to model rate variation between sites when estimating the alignment and to use
      the evidence in shared insertion/deletions (indels) to group sister taxa in the
      phylogeny. Our indel model makes use of affine gap penalties and considers indels
      of multiple letters. We make the simplifying assumption that the indel process is
      identical on all branches. As a result, the probability of a gap is independent
      of branch length. We use a Markov chain Monte Carlo (MCMC) method to sample from 
      the posterior of the joint model, estimating the most probable alignment and tree
      and their support simultaneously. We describe a new MCMC transition kernel that
      improves our algorithm's mixing efficiency, allowing the MCMC chains to converge 
      even when started from arbitrary alignments. Our software implementation can
      estimate alignment uncertainty and we describe a method for summarizing this
      uncertainty in a single plot.
FAU - Redelings, Benjamin D
AU  - Redelings BD
AD  - Department of Biomathematics, David Geffen School of Medicine at UCLA, Los
      Angeles, CA 90095-1766, USA.
FAU - Suchard, Marc A
AU  - Suchard MA
LA  - eng
GR  - CA16042/CA/NCI NIH HHS/United States
GR  - GM008185/GM/NIGMS NIH HHS/United States
GR  - GM068955/GM/NIGMS NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 5S)
SB  - IM
MH  - *Algorithms
MH  - Amino Acid Sequence
MH  - Base Sequence
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - RNA, Ribosomal, 5S/genetics
MH  - Sequence Alignment/*methods
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - G84466WX373376N1 [pii]
AID - 10.1080/10635150590947041 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):401-18. doi: 10.1080/10635150590947041.

PMID- 16012106
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - Mitogenomic perspectives on the origin and phylogeny of living amphibians.
PG  - 391-400
AB  - Establishing the relationships among modern amphibians (lissamphibians) and their
      ancient relatives is necessary for our understanding of early tetrapod evolution.
      However, the phylogeny is still intractable because of the highly specialized
      anatomy and poor fossil record of lissamphibians. Paleobiologists are still not
      sure whether lissamphibians are monophyletic or polyphyletic, and which ancient
      group (temnospondyls or lepospondyls) is most closely related to them. In an
      attempt to address these problems, eight mitochondrial genomes of living
      amphibians were determined and compared with previously published amphibian
      sequences. A comprehensive molecular phylogenetic analysis of nucleotide
      sequences yields a highly resolved tree congruent with the traditional hypotheses
      (Batrachia). By using a molecular clock-independent approach for inferring dating
      information from molecular phylogenies, we present here the first molecular
      timescale for lissamphibian evolution, which suggests that lissamphibians first
      emerged about 330 million years ago. By observing the fit between molecular and
      fossil times, we suggest that the temnospondyl-origin hypothesis for
      lissamphibians is more credible than other hypotheses. Moreover, under this
      timescale, the potential geographic origins of the main living amphibian groups
      are discussed: (i) advanced frogs (neobatrachians) may possess an Africa-India
      origin; (ii) salamanders may have originated in east Asia; (iii) the tropic
      forest of the Triassic Pangaea may be the place of origin for the ancient
      caecilians. An accurate phylogeny with divergence times can be also helpful to
      direct the search for "missing" fossils, and can benefit comparative studies of
      amphibian evolution.
FAU - Zhang, Peng
AU  - Zhang P
AD  - Key Laboratory of Gene Engineering of the Ministry of Education, Biotechnology
      Research Center, Zhongshan University, Guangzhou, 510275, P. R. China.
      lsbrc04@zsu.edu.cn
FAU - Zhou, Hui
AU  - Zhou H
FAU - Chen, Yue-Qin
AU  - Chen YQ
FAU - Liu, Yi-Fei
AU  - Liu YF
FAU - Qu, Liang-Hu
AU  - Qu LH
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Amphibians/*genetics
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Cluster Analysis
MH  - DNA Primers
MH  - DNA, Mitochondrial/*genetics
MH  - *Evolution, Molecular
MH  - Geography
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - L6762K5886Q00XM8 [pii]
AID - 10.1080/10635150590945278 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):391-400. doi: 10.1080/10635150590945278.

PMID- 16012105
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - Partitioned Bayesian analyses, partition choice, and the phylogenetic
      relationships of scincid lizards.
PG  - 373-90
AB  - Partitioned Bayesian analyses of approximately 2.2 kb of nucleotide sequence data
      (mtDNA) were used to elucidate phylogenetic relationships among 30 scincid lizard
      genera. Few partitioned Bayesian analyses exist in the literature, resulting in a
      lack of methods to determine the appropriate number of and identity of
      partitions. Thus, a criterion, based on the Bayes factor, for selecting among
      competing partitioning strategies is proposed and tested. Improvements in both
      mean -lnL and estimated posterior probabilities were observed when specific
      models and parameter estimates were assumed for partitions of the total data set.
      This result is expected given that the 95% credible intervals of model parameter 
      estimates for numerous partitions do not overlap and it reveals that different
      data partitions may evolve quite differently. We further demonstrate that how one
      partitions the data (by gene, codon position, etc.) is shown to be a greater
      concern than simply the overall number of partitions. Using the criterion of the 
      2 ln Bayes factor &gt; 10, the phylogenetic analysis employing the largest number of
      partitions was decisively better than all other strategies. Strategies that
      partitioned the ND1 gene by codon position performed better than other partition 
      strategies, regardless of the overall number of partitions. Scincidae, Acontinae,
      Lygosominae, east Asian and North American "Eumeces" + Neoseps; North African
      Eumeces, Scincus, and Scincopus, and a large group primarily from sub-Saharan
      Africa, Madagascar, and neighboring islands are monophyletic. Feylinia, a
      limbless group of previously uncertain relationships, is nested within a
      "scincine" clade from sub-Saharan Africa. We reject the hypothesis that the
      nearly limbless dibamids are derived from within the Scincidae, but cannot reject
      the hypothesis that they represent the sister taxon to skinks. Amphiglossus,
      Chalcides, the acontines Acontias and Typhlosaurus, and Scincinae are
      paraphyletic. The globally widespread "Eumeces" is polyphyletic and we make
      necessary taxonomic changes.
FAU - Brandley, Matthew C
AU  - Brandley MC
AD  - Department of Biology, San Diego State University, San Diego, CA, 92182-4614,
      USA. brandley@berkeley.edu
FAU - Schmitz, Andreas
AU  - Schmitz A
FAU - Reeder, Tod W
AU  - Reeder TW
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - *Data Interpretation, Statistical
MH  - Lizards/*genetics
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - H33572K2430861M2 [pii]
AID - 10.1080/10635150590946808 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):373-90. doi: 10.1080/10635150590946808.

PMID- 16012104
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - Evaluating intraspecific "network" construction methods using simulated sequence 
      data: do existing algorithms outperform the global maximum parsimony approach?
PG  - 363-72
AB  - In intraspecific studies, reticulated graphs are valuable tools for
      visualization, within a single figure, of alternative genealogical pathways among
      haplotypes. As available software packages implementing the global maximum
      parsimony (MP) approach only give the possibility to merge resulting topologies
      into less-resolved consensus trees, MP has often been neglected as an alternative
      approach to purely algorithmic (i.e., methods defined solely on the basis of an
      algorithm) "network" construction methods. Here, we propose to search tree space 
      using the MP criterion and present a new algorithm for uniting all equally most
      parsimonious trees into a single (possibly reticulated) graph. Using simulated
      sequence data, we compare our method with three purely algorithmic and widely
      used graph construction approaches (minimum-spanning network, statistical
      parsimony, and median-joining network). We demonstrate that the combination of MP
      trees into a single graph provides a good estimate of the true genealogy.
      Moreover, our analyses indicate that, when internal node haplotypes are not
      sampled, the median-joining and MP methods provide the best estimate of the true 
      genealogy whereas the minimum-spanning algorithm shows very poor performances.
FAU - Cassens, Insa
AU  - Cassens I
AD  - Laboratory of Evolutionary Genetics, Institute of Molecular Biology and Medicine,
      Universite Libre de Bruxelles, rue Jeener et Brachet 12, 6041 Gosselies, Belgium.
FAU - Mardulyn, Patrick
AU  - Mardulyn P
FAU - Milinkovitch, Michel C
AU  - Milinkovitch MC
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Base Sequence
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Data Interpretation, Statistical
MH  - Haplotypes/genetics
MH  - *Phylogeny
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - Q512W89524N36102 [pii]
AID - 10.1080/10635150590945377 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):363-72. doi: 10.1080/10635150590945377.

PMID- 16012103
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 3
DP  - 2005 Jun
TI  - Phylogenetic relationships, divergence time estimation, and global biogeographic 
      patterns of calopterygoid damselflies (odonata, zygoptera) inferred from
      ribosomal DNA sequences.
PG  - 347-62
AB  - The calopterygoid superfamily (Calopterygidae + Hetaerinidae) is composed of more
      than twenty genera in two families: the Calopterygidae (at least 17) and the
      Hetaerinidae (at least 4). Here, 62 calopterygoid (ingroup) taxa representing 18 
      genera and 15 outgroup taxa are subjected to phylogenetic analysis using the
      ribosomal 18S and 5.8S genes and internal transcribed spacers (ITS1, ITS2). The
      five other families of calopterid affinity (Polythoridae, Dicteriadidae,
      Amphipterygidae, Euphaeidae, and Chlorocyphidae) are included in the outgroup.
      For phylogenetic inference, we applied maximum parsimony, maximum likelihood, and
      the Bayesian inference methods. A molecular phylogeny combined with a geographic 
      analysis produced a well-supported phylogenetic hypothesis that partly confirms
      the traditional taxonomy and describes distributional patterns. A monophyletic
      origin of the calopterygoids emerges, revealing the Hetaerinid clade as sister
      group to the Calopterygidae sensu strictu. Within Calopterygidae, seven clades of
      subfamily rank are recognized. Phylogenetic dating was performed with
      semiparametric rate smoothing by penalized likelihood, using seven reference
      fossils for calibration. Divergence time based on the ribosomal genes and spacers
      and fossil constraints indicate that Calopteryginae (10 genera, approximately 50%
      of all Calopterygid taxa studied here), Vestalinae (1 genus), and Hetaerinidae (1
      genus out of 4 studied here) started radiating around 65 Mya (K/T boundary). The 
      South American Iridictyon (without distinctive morphology except for wing
      venation) and Southeast Asian Noguchiphaea (with distinctive morphology) are
      older (about 86 My) and may be survivors of old clades with a Gondwanian range
      that went extinct at the K/T boundary. The same reasoning (and an even older age,
      ca. 150 My) applies to the amphipterygids Rimanella and Pentaphlebia (South
      America-Africa). The extant Calopterygidae show particular species and genus
      richness between west China and Japan, with genera originating between the early 
      Oligocene and Pleistocene. Much of that richness probably extended much wider in 
      preglacial times. The Holarctic Calopteryx, of Miocene age, was deeply affected
      by the climatic cooling of the Pliocene and by the Pleistocene glaciations. Its
      North American and Japanese representatives are of Miocene and Pliocene age,
      respectively, but its impoverished Euro-Siberian taxa are late
      Pliocene-Pleistocene, showing reinvasion, speciation, and introgression events.
      The five other calopterid families combine with the Calopterygidae and
      Hetaerinidae to form the monophyletic cohort Caloptera, with Polythoridae,
      Dicteriadidae, and Amphipterygidae sister group to Calopterygoidea. The crown
      node age of the latter three families has an age of about 157 My, but the
      Dicteriadidae and Polythoridae themselves are of Eocene age, and the same is true
      for the Euphaeidae and Chlorocyphidae. The cohort Caloptera itself, with about
      197 My of age, goes back to the early Jurassic.
FAU - Dumont, Henri J
AU  - Dumont HJ
AD  - Department of Biology, Ghent University, Ledeganckstraat, 35, B-9000 Ghent,
      Belgium. HenriDumont@UGent.be
FAU - Vanfleteren, Jacques R
AU  - Vanfleteren JR
FAU - De Jonckheere, Johan F
AU  - De Jonckheere JF
FAU - H Weekers, Peter H
AU  - H Weekers PH
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA Primers
MH  - DNA, Ribosomal/genetics
MH  - *Demography
MH  - *Evolution, Molecular
MH  - Geography
MH  - Insecta/*genetics
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - P202353G3770436X [pii]
AID - 10.1080/10635150590949869 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Jun;54(3):347-62. doi: 10.1080/10635150590949869.

PMID- 16012099
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20091103
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 2
DP  - 2005 Apr
TI  - Molecular phylogeny of the carnivora (mammalia): assessing the impact of
      increased sampling on resolving enigmatic relationships.
PG  - 317-37
AB  - This study analyzed 76 species of Carnivora using a concatenated sequence of 6243
      bp from six genes (nuclear TR-i-I, TBG, and IRBP; mitochondrial ND2, CYTB, and
      12S rRNA), representing the most comprehensive sampling yet undertaken for
      reconstructing the phylogeny of this clade. Maximum parsimony and Bayesian
      methods were remarkably congruent in topologies observed and in nodal support
      measures. We recovered all of the higher level carnivoran clades that had been
      robustly supported in previous analyses (by analyses of morphological and
      molecular data), including the monophyly of Caniformia, Feliformia, Arctoidea,
      Pinnipedia, Musteloidea, Procyonidae + Mustelidae sensu stricto, and a clade of
      (Hyaenidae + (Herpestidae + Malagasy carnivorans)). All of the traditional
      "families," with the exception of Viverridae and Mustelidae, were robustly
      supported as monophyletic groups. We further have determined the relative
      positions of the major lineages within the Caniformia, which previous studies
      could not resolve, including the first robust support for the phylogenetic
      position of marine carnivorans (Pinnipedia) within the Arctoidea (as the
      sister-group to musteloids [sensu lato], with ursids as their sister group).
      Within the pinnipeds, Odobenidae (walrus) was more closely allied with otariids
      (sea lions/fur seals) than with phocids ("true" seals). In addition, we recovered
      a monophyletic clade of skunks and stink badgers (Mephitidae) and resolved the
      topology of musteloid interrelationships as: Ailurus (Mephitidae (Procyonidae,
      Mustelidae [sensu stricto])). This pattern of interrelationships of living
      caniforms suggests a novel inference that large body size may have been the
      primitive condition for Arctoidea, with secondary size reduction evolving later
      in some musteloids. Within Mustelidae, Bayesian analyses are unambiguous in
      supporting otter monophyly (Lutrinae), and in both MP and Bayesian analyses
      Martes is paraphyletic with respect to Gulo and Eira, as has been observed in
      some previous molecular studies. Within Feliformia, we have confirmed that
      Nandinia is the outgroup to all other extant feliforms, and that the Malagasy
      Carnivora are a monophyletic clade closely allied with the mongooses (Herpestidae
      [sensu stricto]). Although the monophyly of each of the three major feliform
      clades (Viverridae sensu stricto, Felidae, and the clade of Hyaenidae +
      (Herpestidae + Malagasy carnivorans)) is robust in all of our analyses, the
      relative phylogenetic positions of these three lineages is not resolvable at
      present. Our analyses document the monophyly of the "social mongooses,"
      strengthening evidence for a single origin of eusociality within the Herpestidae.
      For a single caniform node, the position of pinnipeds relative to Ursidae and
      Musteloidea, parsimony analyses of data for the entire Carnivora did not
      replicate the robust support observed for both parsimony and Bayesian analyses of
      the caniform ingroup alone. More detailed analyses and these results demonstrate 
      that outgroup choice can have a considerable effect on the strength of support
      for a particular topology. Therefore, the use of exemplar taxa as proxies for
      entire clades with diverse evolutionary histories should be approached with
      caution. The Bayesian analysis likelihood functions generally were better able to
      reconstruct phylogenetic relationships (increased resolution and more robust
      support for various nodes) than parsimony analyses when incompletely sampled taxa
      were included. Bayesian analyses were not immune, however, to the effects of
      missing data; lower resolution and support in those analyses likely arise from
      non-overlap of gene sequence data among less well-sampled taxa. These issues are 
      a concern for similar studies, in which different gene sequences are concatenated
      in an effort to increase resolving power.
FAU - Flynn, John J
AU  - Flynn JJ
AD  - Department of Geology, The Field Museum, 1400 S. Lake Shore Drive, Chicago, IL,
      60605, USA. jflynn@amnh.org
FAU - Finarelli, John A
AU  - Finarelli JA
FAU - Zehr, Sarah
AU  - Zehr S
FAU - Hsu, Johnny
AU  - Hsu J
FAU - Nedbal, Michael A
AU  - Nedbal MA
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Eye Proteins)
RN  - 0 (Prealbumin)
RN  - 0 (Retinol-Binding Proteins)
RN  - 0 (Thyroxine-Binding Proteins)
RN  - 0 (interstitial retinol-binding protein)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Carnivora/*genetics
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - Eye Proteins/genetics
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Prealbumin/genetics
MH  - Retinol-Binding Proteins/genetics
MH  - Sample Size
MH  - Sequence Analysis, DNA
MH  - Thyroxine-Binding Proteins/genetics
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - V7716X8Q131184T5 [pii]
AID - 10.1080/10635150590923326 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Apr;54(2):317-37. doi: 10.1080/10635150590923326.

PMID- 16012098
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 2
DP  - 2005 Apr
TI  - Intraspecific variability and timing in ancestral ecology reconstruction: a test 
      case from the cape flora.
PG  - 299-316
AB  - Thamnochortus (ca. 32 species) is an ecologically diverse genus of Restionaceae. 
      Restionaceae comprise a major component of the southern African Cape flora,
      wherein eco-diversification might have been important in the generation of high
      levels of species richness. In an attempt to reconstruct the macroecological
      history of Thamnochortus, it was found that standard procedures for character
      state optimization make two inappropriate assumptions. The first is that
      ancestors are monomorphic (i.e., ecologically uniform) and the second is that
      eco-diversification follows, or is slower than, lineage diversification. We
      demonstrate a variety of coding schemes with which the assumption of monomorphy
      can be avoided. For unordered discrete ecological characters, presence coding and
      generalized frequency coding (GFC) are suboptimal because they occasionally yield
      illogical assignments of no state to ancestors. Polymorphism coding or use of the
      program DIVA are preferable in this respect but are applicable only with
      parsimony. For continuous eco-characters (e.g., a rainfall gradient, where
      individual species occur in ranges), GFC and MaxMin coding provide equally valid 
      solutions to optimizing ranges with parsimony. However, MaxMin can be extended to
      likelihood approaches and is therefore preferable. With respect to rates and
      timing, all algorithms currently employed for ancestral ecology reconstruction
      bias toward slow rates of eco-diversification relative to lineage
      diversification. An alternative to this bias is provided by DIVA, which biases
      toward accelerated rates of eco-diversification and thus inferences of
      ecology-driven speciation. We see no way of choosing between these biases;
      however, phylogeneticists should be aware of them. Applying these methods to
      Thamnochortus, we find there to be important differences in details, yet general 
      congruence, regarding the historical ecology of this clade. We infer the most
      recent common ancestor of Thamnochortus to have been a post-fire resprouting
      species distributed on rocky, well-drained, sandstone-derived soils at
      lower-middle elevations, in regions of moderate levels of yearly (primarily
      winter) rainfall. This species would have been distributed in habitats much like 
      those of the southwestern Cape mountains today. Major ecological trends include
      shifts to lower rainfall regimes and shifts from sandstone to limestone-derived
      alkaline soils at lower altitudes.
FAU - Hardy, Christopher R
AU  - Hardy CR
AD  - Institute of Systematic Botany, Zollikerstrasse 107, Zurich, CH-8008,
      Switzerland. chardy@systbot.unizh.ch
FAU - Linder, H Peter
AU  - Linder HP
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
SB  - IM
MH  - Base Sequence
MH  - Classification/*methods
MH  - Climate
MH  - DNA Primers
MH  - Ecology/methods
MH  - *Environment
MH  - Genes, Plant/genetics
MH  - Magnoliopsida/anatomy &amp; histology/genetics/*physiology
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Plastids/genetics
MH  - Sequence Analysis, DNA
MH  - South Africa
MH  - Species Specificity
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - K55350V144374624 [pii]
AID - 10.1080/10635150590923317 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Apr;54(2):299-316. doi: 10.1080/10635150590923317.

PMID- 16012097
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20071114
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 2
DP  - 2005 Apr
TI  - Gene family content-based phylogeny of prokaryotes: the effect of criteria for
      inferring homology.
PG  - 268-76
AB  - A number of recent papers have suggested that gene family content can be used to 
      resolve phylogenies, particularly in the case of prokaryotes, in which extensive 
      horizontal gene transfer means that individual gene phylogenies may not mirror
      the organismal phylogeny. However, no study has yet examined how sensitive such
      analyses are to the criterion of homology assessment used to assemble multigene
      families. Using data from 99 completely sequenced prokaryotic genomes, we
      examined the effect of homology criteria in phylogenetic analyses wherein
      presence or absence of each family in the genome was used as a cladistic
      character. Different criteria resulted in evidence for contradictory tree
      topologies, sometimes with high bootstrap support. A moderately strict criterion 
      seemed best for assembling multigene families in a biologically meaningful way,
      but it was not necessarily preferable for phylogenetic analysis. Instead, a very 
      strict criterion, which broke up gene families into smaller subfamilies, seemed
      to have advantages for phylogenetic purposes. The poor performance of gene family
      content-based phylogenetic analysis in the case of prokaryotes appears to reflect
      high levels of homoplasy resulting not only from horizontal gene transfer but
      also, more importantly, from extensive parallel loss of gene families in certain 
      bacteria genomes.
FAU - Hughes, Austin L
AU  - Hughes AL
AD  - Department of Biological Sciences, University of South Carolina, Columbia, SC
      29205, USA. austin@biol.sc.edu
FAU - Ekollu, Vikram
AU  - Ekollu V
FAU - Friedman, Robert
AU  - Friedman R
FAU - Rose, John R
AU  - Rose JR
LA  - eng
GR  - GM066710/GM/NIGMS NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Archaea/*genetics
MH  - Bacteria/*genetics
MH  - Classification/*methods
MH  - Computational Biology
MH  - Data Interpretation, Statistical
MH  - Gene Transfer, Horizontal/genetics
MH  - Multigene Family/*genetics
MH  - *Phylogeny
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - Q8H4573848211083 [pii]
AID - 10.1080/10635150590923335 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Apr;54(2):268-76. doi: 10.1080/10635150590923335.

PMID- 16012096
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20091119
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 2
DP  - 2005 Apr
TI  - Phylogenetics of eggshell morphogenesis in Antheraea (lepidoptera: saturniidae): 
      unique origin and repeated reduction of the aeropyle crown.
PG  - 254-67
AB  - Integrated phylogenetic and developmental analyses should enhance our
      understanding of morphological evolution and thereby improve systematists'
      ability to utilize morphological characters, but case studies are few. The
      eggshell (chorion) of Lepidoptera (Insecta) has proven especially tractable
      experimentally for such analyses because its morphogenesis proceeds by
      extracellular assembly of proteins. This study focuses on a morphological
      novelty, the aeropyle crown, that arises at the end of choriogenesis in the wild 
      silkmoth genus Antheraea. Aeropyle crowns are cylindrical projections, ending in 
      prominent prongs, that surround the openings of breathing tubes (aeropyle
      channels) traversing the chorion. They occur over the entire egg surface in some 
      species, are localized to a circumferential band in many others, and in some are 
      missing entirely, thus exhibiting variation typical of discrete characters
      analyzed in morphological phylogenetics. Seeking an integrated
      developmental-phylogenetic view, we first survey aeropyle crown variation broadly
      across Antheraea and related genera. We then map these observations onto a robust
      phylogeny, based on three nuclear genes, to test the adequacy of character
      codings for aeropyle crown variation and to estimate the frequency and direction 
      of change in those characters. Thirdly, we draw on previous studies of
      choriogenesis, supplemented by new data on gene expression, to hypothesize
      developmental-genetic bases for the inferred chorion character transformations.
      Aeropyle crowns are inferred to arise just once, in the ancestor of Antheraea,
      but to undergo four or more subsequent reductions without regain, a pattern
      consistent with Dollo's Law. Spatial distribution shows an analogous trend,
      though less clear-cut, toward reduction of coverage by aeropyle crowns. These
      trends suggest either that there is little or no natural selection on the details
      of the aeropyle crown structure or that evolution toward functional optima is
      ongoing, although no direct evidence exists for either. Genetic, biochemical, and
      microscopy studies point to at least two developmental changes underlying the
      origin of the aeropyle crown, namely, reinitiation of deposition of chorionic
      lamellae after the end of normal choriogenesis (i.e., heterochrony), and sharply 
      increased production of underlying "filler" proteins that push the nascent final 
      lamellae upward to form the crown (i.e., heteroposy). Identification of a unique 
      putative cis-regulatory element shared by unrelated genes involved in aeropyle
      crown formation suggests a possible simple mechanism for repeated evolutionary
      reduction and spatial restriction of aeropyle crowns.
FAU - Regier, Jerome C
AU  - Regier JC
AD  - Center for Biosystems Research, University of Maryland Biotechnology Institute,
      5140 Plant Sciences Building, College Park, MD 20742, USA. regier@umd.edu
FAU - Paukstadt, Ulrich
AU  - Paukstadt U
FAU - Paukstadt, Laela H
AU  - Paukstadt LH
FAU - Mitter, Charles
AU  - Mitter C
FAU - Peigler, Richard S
AU  - Peigler RS
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (Nuclear Proteins)
RN  - 0 (Peptide Elongation Factor 1)
RN  - 0 (Period Circadian Proteins)
RN  - EC 4.1.1.- (Carboxy-Lyases)
RN  - EC 4.1.1.- (L-3,4-dihydroxyphenylalanine decarboxylase, Bombyx mori)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Blotting, Northern
MH  - Carboxy-Lyases/genetics
MH  - Chorion/physiology/*ultrastructure
MH  - DNA Primers
MH  - Gene Expression Regulation, Developmental
MH  - Microscopy, Electron, Transmission
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Morphogenesis
MH  - Moths/*anatomy &amp; histology/genetics/*physiology
MH  - Nuclear Proteins/genetics
MH  - Ovum/*cytology
MH  - Peptide Elongation Factor 1/genetics
MH  - Period Circadian Proteins
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - VH78416654639N4L [pii]
AID - 10.1080/10635150590923281 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Apr;54(2):254-67. doi: 10.1080/10635150590923281.

PMID- 16012095
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 2
DP  - 2005 Apr
TI  - Polytomies and Bayesian phylogenetic inference.
PG  - 241-53
AB  - Bayesian phylogenetic analyses are now very popular in systematics and molecular 
      evolution because they allow the use of much more realistic models than currently
      possible with maximum likelihood methods. There are, however, a growing number of
      examples in which large Bayesian posterior clade probabilities are associated
      with very short branch lengths and low values for non-Bayesian measures of
      support such as nonparametric bootstrapping. For the four-taxon case when the
      true tree is the star phylogeny, Bayesian analyses become increasingly
      unpredictable in their preference for one of the three possible resolved tree
      topologies as data set size increases. This leads to the prediction that hard (or
      near-hard) polytomies in nature will cause unpredictable behavior in Bayesian
      analyses, with arbitrary resolutions of the polytomy receiving very high
      posterior probabilities in some cases. We present a simple solution to this
      problem involving a reversible-jump Markov chain Monte Carlo (MCMC) algorithm
      that allows exploration of all of tree space, including unresolved tree
      topologies with one or more polytomies. The reversible-jump MCMC approach allows 
      prior distributions to place some weight on less-resolved tree topologies, which 
      eliminates misleadingly high posteriors associated with arbitrary resolutions of 
      hard polytomies. Fortunately, assigning some prior probability to polytomous tree
      topologies does not appear to come with a significant cost in terms of the
      ability to assess the level of support for edges that do exist in the true tree. 
      Methods are discussed for applying arbitrary prior distributions to tree
      topologies of varying resolution, and an empirical example showing evidence of
      polytomies is analyzed and discussed.
FAU - Lewis, Paul O
AU  - Lewis PO
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043 (P.O.L.), Storrs, CN 06269-3043, USA.
      paul.lewis@uconn.edu
FAU - Holder, Mark T
AU  - Holder MT
FAU - Holsinger, Kent E
AU  - Holsinger KE
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Chlorophyta/genetics
MH  - Classification/*methods
MH  - *Data Interpretation, Statistical
MH  - Markov Chains
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - X8576658LP734021 [pii]
AID - 10.1080/10635150590924208 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Apr;54(2):241-53. doi: 10.1080/10635150590924208.

PMID- 16012094
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 2
DP  - 2005 Apr
TI  - A new technique for identifying sequence heterochrony.
PG  - 230-40
AB  - Sequence heterochrony (changes in the order in which events occur) is a
      potentially important, but relatively poorly explored, mechanism for the
      evolution of development. In part, this is because of the inherent difficulties
      in inferring sequence heterochrony across species. The event-pairing method,
      developed independently by several workers in the mid-1990s, encodes sequences in
      a way that allows them to be examined in a phylogenetic framework, but the
      results can be difficult to interpret in terms of actual heterochronic changes.
      Here, we describe a new, parsimony-based method to interpret such results. For
      each branch of the tree, it identifies the least number of event movements
      (heterochronies) that will explain all the observed event-pair changes. It has
      the potential to find all alternative, equally parsimonious explanations, and
      generate a consensus, containing the movements that form part of every equally
      most parsimonious explanation. This new technique, which we call Parsimov,
      greatly increases the utility of the event-pair method for inferring instances of
      sequence heterochrony.
FAU - Jeffery, Jonathan E
AU  - Jeffery JE
AD  - Institute of Biology, Leiden University, Kaiserstraat 63, Leiden, 2311GP, The
      Netherlands.
FAU - Bininda-Emonds, Olaf R P
AU  - Bininda-Emonds OR
FAU - Coates, Michael I
AU  - Coates MI
FAU - Richardson, Michael K
AU  - Richardson MK
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Animals
MH  - Classification/*methods
MH  - *Data Interpretation, Statistical
MH  - Morphogenesis/*physiology
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - X4R67R35267316K9 [pii]
AID - 10.1080/10635150590923227 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Apr;54(2):230-40. doi: 10.1080/10635150590923227.

PMID- 16012093
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 2
DP  - 2005 Apr
TI  - Weighted least-squares likelihood ratio test for branch testing in phylogenies
      reconstructed from distance measures.
PG  - 218-29
AB  - A variety of analytical methods is available for branch testing in distance-based
      phylogenies. However, these methods are rarely used, possibly because the
      estimation of some of their statistics, especially the covariances, is not always
      feasible. We show that these difficulties can be overcome if some simplifying
      assumptions are made, namely distance independence. The weighted least-squares
      likelihood ratio test (WLS-LRT) we propose is easy to perform, using only the
      distances and some of their associated variances. If no variances are known, the 
      use of the Felsenstein F-test, also based on weighted least squares, is
      discussed. Using simulated data and a data set of 43 mammalian mitochondrial
      sequences we demonstrate that the WLS-LRT performs as well as the generalized
      least-squares test, and indeed better for a large number of taxa data set. We
      thus show that the assumption of independence does not negatively affect the
      reliability or the accuracy of the least-squares approach. The results of the
      WLS-LRT are no worse than the results of the bootstrap methods, such as the
      Felsenstein bootstrap selection probability test and the Dopazo test. We also
      show that WLS-LRT can be applied in instances where other analytical methods are 
      inappropriate. This point is illustrated by analyzing the relationships between
      human immunodeficiency virus type 1 (HIV-1) sequences isolated from various
      organs of different individuals.
FAU - Sanjuan, Rafael
AU  - Sanjuan R
AD  - Institut Cavanilles de Biodiversitat i Biologia Evolutiva, Universitat de
      Valencia, Edifici d' Instituts de Paterna Apartat 2085 (R.S.), Valencia, 46071,
      Spain. rafael.sanjuan@uv.es
FAU - Wrobel, Borys
AU  - Wrobel B
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Cluster Analysis
MH  - DNA, Mitochondrial/genetics
MH  - *Data Interpretation, Statistical
MH  - *Evolution, Molecular
MH  - HIV-1/genetics
MH  - Likelihood Functions
MH  - Mammals/genetics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - MHXW3237Q3MG6P28 [pii]
AID - 10.1080/10635150590923308 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Apr;54(2):218-29. doi: 10.1080/10635150590923308.

PMID- 16012092
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 2
DP  - 2005 Apr
TI  - Phylogenetic signal in AFLP data sets.
PG  - 197-217
AB  - AFLP markers provide a potential source of phylogenetic information for molecular
      systematic studies. However, there are properties of restriction fragment data
      that limit phylogenetic interpretation of AFLPs. These are (a) possible
      nonindependence of fragments, (b) problems of homology assignment of fragments,
      (c) asymmetry in the probability of losing and gaining fragments, and (d)
      problems in distinguishing heterozygote from homozygote bands. In the present
      study, AFLP data sets of Lactuca s.l. were examined for the presence of
      phylogenetic signal. An indication of this signal was provided by carrying out
      tree length distribution skewness (g1) tests, permutation tail probability (PTP) 
      tests, and relative apparent synapomorphy analysis (RASA). A measure of the
      support for internal branches in the optimal parsimony tree (MPT) was made using 
      bootstrap, jackknife, and decay analysis. Finally, the extent of congruence in
      MPTs for AFLP and internal transcribed spacer (ITS)-1 data sets for the same taxa
      was made using the partition homogeneity test (PHT) and the Templeton test. These
      analytical studies suggested the presence of phylogenetic signal in the AFLP data
      sets, although some incongruence was found between AFLP and ITS MPTs. An
      extensive literature survey undertaken indicated that authors report a general
      congruence of AFLP and ITS tree topologies across a wide range of taxonomic
      groups, suggesting that the present results and conclusions have a general
      bearing. In these earlier studies and those for Lactuca s.l., AFLP markers have
      been found to be informative at somewhat lower taxonomic levels than ITS
      sequences. Tentative estimates are suggested for the levels of ITS sequence
      divergence over which AFLP profiles are likely to be phylogenetically
      informative.
FAU - Koopman, Wim J M
AU  - Koopman WJ
AD  - Biosystematics Group, Nationaal Herbarium Nederland-, Wageningen University
      branch, Wageningen University, Generaal Foulkesweg 37, 6703BL, Wageningen, The
      Netherlands. wim.koopman@wur.nl
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal Spacer)
SB  - IM
MH  - Classification/*methods
MH  - DNA, Ribosomal Spacer/genetics
MH  - *Data Interpretation, Statistical
MH  - Lettuce/genetics
MH  - Nucleic Acid Amplification Techniques
MH  - *Phylogeny
MH  - Polymorphism, Restriction Fragment Length
MH  - Reproducibility of Results
MH  - Research Design
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - X4369UW414846202 [pii]
AID - 10.1080/10635150590924181 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Apr;54(2):197-217. doi: 10.1080/10635150590924181.

PMID- 16012091
OWN - NLM
STAT- MEDLINE
DCOM- 20050818
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 2
DP  - 2005 Apr
TI  - Towards building the tree of life: a simulation study for all angiosperm genera.
PG  - 183-96
AB  - Comprehensive phylogenetic trees are essential tools to better understand
      evolutionary processes. For many groups of organisms or projects aiming to build 
      the Tree of Life, comprehensive phylogenetic analysis implies sampling hundreds
      to thousands of taxa. For the tree of all life this task rises to a highly
      conservative 13 million. Here, we assessed the performances of methods to
      reconstruct large trees using Monte Carlo simulations with parameters inferred
      from four large angiosperm DNA matrices, containing between 141 and 567 taxa. For
      each data set, parameters of the HKY85+G model were estimated and used to
      simulate 20 new matrices for sequence lengths from 100 to 10,000 base pairs.
      Maximum parsimony and neighbor joining were used to analyze each simulated
      matrix. In our simulations, accuracy was measured by counting the number of nodes
      in the model tree that were correctly inferred. The accuracy of the two methods
      increased very quickly with the addition of characters before reaching a plateau 
      around 1000 nucleotides for any sizes of trees simulated. An increase in the
      number of taxa from 141 to 567 did not significantly decrease the accuracy of the
      methods used, despite the increase in the complexity of tree space. Moreover, the
      distribution of branch lengths rather than the rate of evolution was found to be 
      the most important factor for accurately inferring these large trees. Finally, a 
      tree containing 13,000 taxa was created to represent a hypothetical tree of all
      angiosperm genera and the efficiency of phylogenetic reconstructions was tested
      with simulated matrices containing an increasing number of nucleotides up to a
      maximum of 30,000. Even with such a large tree, our simulations suggested that
      simple heuristic searches were able to infer up to 80% of the nodes correctly.
FAU - Salamin, Nicolas
AU  - Salamin N
AD  - Department of Botany, University of Dublin, Trinity College, Dublin 2, Ireland.
      nicolas.salamin@unil.ch
FAU - Hodkinson, Trevor R
AU  - Hodkinson TR
FAU - Savolainen Coates, Vincent
AU  - Savolainen Coates V
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - Classification/*methods
MH  - Cluster Analysis
MH  - Computer Simulation
MH  - Magnoliopsida/*genetics
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Sample Size
EDAT- 2005/07/14 09:00
MHDA- 2005/08/19 09:00
CRDT- 2005/07/14 09:00
PHST- 2005/07/14 09:00 [pubmed]
PHST- 2005/08/19 09:00 [medline]
PHST- 2005/07/14 09:00 [entrez]
AID - PW2TG330K1U06143 [pii]
AID - 10.1080/10635150590923254 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Apr;54(2):183-96. doi: 10.1080/10635150590923254.

PMID- 15823966
OWN - NLM
STAT- MEDLINE
DCOM- 20050505
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 1
DP  - 2005 Feb
TI  - Freshwater fishes, dispersal ability, and nonevidence: "gondwana life rafts" to
      the rescue.
PG  - 158-65
FAU - Sparks, John
AU  - Sparks J
AD  - Department of Ichthyology, Division of Vertebrate Zoology, American Museum of
      Natural History, Central Park West at 79th Street, New York, New York 10024, USA.
      jsparks@amnh.org
FAU - Smith, Wm
AU  - Smith W
LA  - eng
PT  - Comment
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2003 Aug;52(4):548-53. PMID: 12857645
MH  - Adaptation, Biological/physiology
MH  - Animals
MH  - *Demography
MH  - Fishes/*genetics/physiology
MH  - Fresh Water
MH  - Geography
MH  - Geological Phenomena
MH  - *Geology
MH  - *Phylogeny
MH  - Population Dynamics
EDAT- 2005/04/13 09:00
MHDA- 2005/05/06 09:00
CRDT- 2005/04/13 09:00
PHST- 2005/04/13 09:00 [pubmed]
PHST- 2005/05/06 09:00 [medline]
PHST- 2005/04/13 09:00 [entrez]
AID - L55632W36M7M1718 [pii]
AID - 10.1080/10635150590906019 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Feb;54(1):158-65. doi: 10.1080/10635150590906019.

PMID- 15805017
OWN - NLM
STAT- MEDLINE
DCOM- 20050505
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 1
DP  - 2005 Feb
TI  - Morphology's role in phylogeny reconstruction: perspectives from paleontology.
PG  - 166-73
FAU - Smith, Nathan
AU  - Smith N
AD  - Department of Geoscience, University of Iowa, Iowa City, IA 52242, USA.
FAU - Turner, Alan
AU  - Turner A
LA  - eng
PT  - Comment
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2003 Aug;52(4):539-48. PMID: 12857644
MH  - Anatomy, Comparative/*methods
MH  - Classification/*methods
MH  - Evolution, Molecular
MH  - Fossils
MH  - Paleontology/*methods
MH  - *Phylogeny
MH  - Sample Size
EDAT- 2005/04/05 09:00
MHDA- 2005/05/06 09:00
CRDT- 2005/04/05 09:00
PHST- 2005/04/05 09:00 [pubmed]
PHST- 2005/05/06 09:00 [medline]
PHST- 2005/04/05 09:00 [entrez]
AID - L55632W36M7M1718 [pii]
AID - 10.1080/10635150590906000 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Feb;54(1):166-73. doi: 10.1080/10635150590906000.

PMID- 15805016
OWN - NLM
STAT- MEDLINE
DCOM- 20050505
LR  - 20050404
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 1
DP  - 2005 Feb
TI  - Missing the forest for the trees: phylogenetic compression and its implications
      for inferring complex evolutionary histories.
PG  - 146-57
AB  - Phylogenetic tree reconstruction is difficult in the presence of lateral gene
      transfer and other processes generating conflicting signals. We develop a new
      approach to this problem using ideas borrowed from algorithmic information
      theory. It selects the hypothesis that simultaneously minimizes the descriptive
      complexity of the tree(s) plus the data when encoded using those tree(s). In
      practice this is the hypothesis that can compress the data the most. We show not 
      only that phylogenetic compression is an efficient method for encoding most
      phylogenetic data sets and is more efficient than compression schemes designed
      for single sequences, but also that it provides a clear information theoretic
      rule for determining when a collection of conflicting trees is a better
      explanation of the data than a single tree. By casting the parsimony problem in
      this more general framework, we also conclude that the so-called total-evidence
      tree--the tree constructed from all the data simultaneously--is not always the
      most economical explanation of the data.
FAU - Ane, Cecile
AU  - Ane C
AD  - Section of Evolution and Ecology, University of California, Davis, California
      95616, USA. ane@cs.wisc.edu
FAU - Sanderson, Michael
AU  - Sanderson M
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2005/04/05 09:00
MHDA- 2005/05/06 09:00
CRDT- 2005/04/05 09:00
PHST- 2005/04/05 09:00 [pubmed]
PHST- 2005/05/06 09:00 [medline]
PHST- 2005/04/05 09:00 [entrez]
AID - H612622832G18018 [pii]
AID - 10.1080/10635150590905984 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Feb;54(1):146-57. doi: 10.1080/10635150590905984.

PMID- 15805015
OWN - NLM
STAT- MEDLINE
DCOM- 20050505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 1
DP  - 2005 Feb
TI  - The effect of paralogous lineages on the application of reconciliation analysis
      by cophylogeny mapping.
PG  - 127-45
AB  - Paralogy defines similarity caused by duplication rather than common descent and 
      is well known in the case of paralogous gene copies within a single genome. The
      term is here extended to paralogous lineages of associates within a single host. 
      The phylogenies of four genera within the Herpesviridae were reconciled with host
      phylogenies using cophylogenetic mapping. The observed correspondence for each
      pair of phylogenies was evaluated through randomization of the viral phylogeny
      and demonstrated to be greater than expected by chance. A simulation study was
      then carried out to assess the influence of paralogous lineages on the efficacy
      of reconciliation analysis. Combining viral taxa from different genera that
      infected common hosts introduced incongruence into the cophylogenies and reduced 
      both the minimum and maximum observed number of codivergence events relative to
      the initial analysis of orthologous clades. However, at an average sample size
      this did not alter the fundamental significance of observed correspondence. With 
      smaller sample sizes, the number of orthologous taxa selected at random from the 
      pool of taxa was reduced. False-negative results then increased in proportion
      from 0.02 to 0.33. These results demonstrated that reconciliation analysis is
      robust under conditions of paralogy at "normal" sample sizes but is adversely
      affected by a combination of paralogy and low sample size. Consideration of
      phylogenies for Papillomavirus, Atadenovirus, and Mastadenovirus suggest that
      paralogous lineages may be a widespread phenomenon among DNA viruses and that
      duplication irrespective of host speciation is an important cause of viral
      diversification.
FAU - Jackson, Andrew
AU  - Jackson A
AD  - Department of Zoology, University of Oxford, South Parks Road, Oxford OX1 3PS,
      United Kingdom. Andrew.Jackson@zoo.ox.ac.uk
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Databases, Nucleic Acid
MH  - *Evolution, Molecular
MH  - Herpesviridae/*genetics
MH  - Humans
MH  - Likelihood Functions
MH  - Mammals/*genetics/virology
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Sample Size
MH  - Species Specificity
MH  - *Symbiosis
EDAT- 2005/04/05 09:00
MHDA- 2005/05/06 09:00
CRDT- 2005/04/05 09:00
PHST- 2005/04/05 09:00 [pubmed]
PHST- 2005/05/06 09:00 [medline]
PHST- 2005/04/05 09:00 [entrez]
AID - NU3226N6423W4618 [pii]
AID - 10.1080/10635150590905911 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Feb;54(1):127-45. doi: 10.1080/10635150590905911.

PMID- 15805014
OWN - NLM
STAT- MEDLINE
DCOM- 20050505
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 1
DP  - 2005 Feb
TI  - Archaeobatrachian paraphyly and pangaean diversification of crown-group frogs.
PG  - 111-26
AB  - Current models for the early diversification of living frogs inferred from
      morphological, ontogenetic, or DNA sequence data invoke very different scenarios 
      of character evolution and biogeography. To explore central controversies on the 
      phylogeny of Anura, we analyzed nearly 4000 base pairs of mitochondrial and
      nuclear DNA for the major frog lineages. Likelihood-based analyses of this data
      set are congruent with morphological evidence in supporting a paraphyletic
      arrangement of archaeobatrachian frogs, with an (Ascaphus + Leiopelma) clade as
      the sister-group of all other living anurans. The stability of this outcome is
      reinforced by screening for phylogenetic bias resulting from site-specific rate
      variation, homoplasy, or the obligatory use of distantly related outgroups.
      Twenty-one alternative branching and rooting hypotheses were evaluated using a
      nonparametric multicomparison test and parametric bootstrapping. Relaxed
      molecular clock estimates situate the emergence of crown-group anurans in the
      Triassic, approximately 55 million years prior to their first appearance in the
      fossil record. The existence of at least four extant frog lineages on the
      supercontinent Pangaea before its breakup gains support from the estimation that 
      three early splits between Laurasia- and Gondwana-associated families coincide
      with the initial rifting of these landmasses. This observation outlines the
      potential significance of this breakup event in the formation of separate
      Mesozoic faunal assemblages in both hemispheres.
FAU - Roelants, Kim
AU  - Roelants K
AD  - Biology Department, Unit of Ecology and Systematics, Vrije Universiteit Brussel, 
      Pleinlaan 2, B-1050 Brussels, Belgium.
FAU - Bossuyt, Franky
AU  - Bossuyt F
LA  - eng
SI  - GENBANK/AY523683
SI  - GENBANK/AY523684
SI  - GENBANK/AY523685
SI  - GENBANK/AY523686
SI  - GENBANK/AY523687
SI  - GENBANK/AY523688
SI  - GENBANK/AY523689
SI  - GENBANK/AY523690
SI  - GENBANK/AY523691
SI  - GENBANK/AY523692
SI  - GENBANK/AY523693
SI  - GENBANK/AY523694
SI  - GENBANK/AY523695
SI  - GENBANK/AY523696
SI  - GENBANK/AY523697
SI  - GENBANK/AY523698
SI  - GENBANK/AY523699
SI  - GENBANK/AY523700
SI  - GENBANK/AY523701
SI  - GENBANK/AY523702
SI  - GENBANK/AY523703
SI  - GENBANK/AY523704
SI  - GENBANK/AY523705
SI  - GENBANK/AY523706
SI  - GENBANK/AY523707
SI  - GENBANK/AY523708
SI  - GENBANK/AY523709
SI  - GENBANK/AY523710
SI  - GENBANK/AY523711
SI  - GENBANK/AY523712
SI  - GENBANK/AY523713
SI  - GENBANK/AY523714
SI  - GENBANK/AY523715
SI  - GENBANK/AY523716
SI  - GENBANK/AY523717
SI  - GENBANK/AY523718
SI  - GENBANK/AY523719
SI  - GENBANK/AY523720
SI  - GENBANK/AY523721
SI  - GENBANK/AY523722
SI  - GENBANK/AY523723
SI  - GENBANK/AY523724
SI  - GENBANK/AY523725
SI  - GENBANK/AY523726
SI  - GENBANK/AY523727
SI  - GENBANK/AY523728
SI  - GENBANK/AY523729
SI  - GENBANK/AY523730
SI  - GENBANK/AY523731
SI  - GENBANK/AY523732
SI  - GENBANK/AY523733
SI  - GENBANK/AY523734
SI  - GENBANK/AY523735
SI  - GENBANK/AY523736
SI  - GENBANK/AY523737
SI  - GENBANK/AY523738
SI  - GENBANK/AY523739
SI  - GENBANK/AY523740
SI  - GENBANK/AY523741
SI  - GENBANK/AY523742
SI  - GENBANK/AY523743
SI  - GENBANK/AY523744
SI  - GENBANK/AY523745
SI  - GENBANK/AY523746
SI  - GENBANK/AY523747
SI  - GENBANK/AY523748
SI  - GENBANK/AY523749
SI  - GENBANK/AY523750
SI  - GENBANK/AY523751
SI  - GENBANK/AY523752
SI  - GENBANK/AY523753
SI  - GENBANK/AY523754
SI  - GENBANK/AY523755
SI  - GENBANK/AY523756
SI  - GENBANK/AY523757
SI  - GENBANK/AY523758
SI  - GENBANK/AY523759
SI  - GENBANK/AY523760
SI  - GENBANK/AY523761
SI  - GENBANK/AY523762
SI  - GENBANK/AY523763
SI  - GENBANK/AY523764
SI  - GENBANK/AY523765
SI  - GENBANK/AY523766
SI  - GENBANK/AY523767
SI  - GENBANK/AY523768
SI  - GENBANK/AY523769
SI  - GENBANK/AY523770
SI  - GENBANK/AY523771
SI  - GENBANK/AY523772
SI  - GENBANK/AY523773
SI  - GENBANK/AY523774
SI  - GENBANK/AY523775
SI  - GENBANK/AY523776
SI  - GENBANK/AY523777
SI  - GENBANK/AY523778
SI  - GENBANK/AY523779
SI  - GENBANK/AY523780
SI  - GENBANK/AY523781
SI  - GENBANK/AY523782
SI  - GENBANK/AY523783
SI  - GENBANK/AY523784
SI  - GENBANK/AY523785
SI  - GENBANK/AY523786
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (Receptors, Chemokine)
RN  - 0 (Sodium-Calcium Exchanger)
SB  - IM
MH  - Animals
MH  - Anura/*genetics
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA Primers
MH  - *Evolution, Molecular
MH  - Genes, RAG-1/genetics
MH  - Geography
MH  - Geological Phenomena
MH  - *Geology
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Receptors, Chemokine/genetics
MH  - Sequence Analysis, DNA
MH  - Sodium-Calcium Exchanger/genetics
EDAT- 2005/04/05 09:00
MHDA- 2005/05/06 09:00
CRDT- 2005/04/05 09:00
PHST- 2005/04/05 09:00 [pubmed]
PHST- 2005/05/06 09:00 [medline]
PHST- 2005/04/05 09:00 [entrez]
AID - H2K0776381104115 [pii]
AID - 10.1080/10635150590905894 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Feb;54(1):111-26. doi: 10.1080/10635150590905894.

PMID- 15805013
OWN - NLM
STAT- MEDLINE
DCOM- 20050505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 1
DP  - 2005 Feb
TI  - Ontogeny discombobulates phylogeny: paedomorphosis and higher-level salamander
      relationships.
PG  - 91-110
AB  - Evolutionary developmental biology ("evo-devo") has revolutionized evolutionary
      biology but has had relatively little impact on systematics. We show that similar
      large-scale developmental changes in distantly related lineages can dramatically 
      mislead phylogenetic analyses based on morphological data. Salamanders are
      important model systems in many fields of biology and are of special interest in 
      that many species are paedomorphic and thus never complete metamorphosis. A
      recent study of higher-level salamander phylogeny placed most paedomorphic
      families in a single clade based on morphological data. Here, we use new
      molecular and morphological data to show that this result most likely was caused 
      by the misleading effects of paedomorphosis. We also provide a well-supported
      estimate of higher-level salamander relationships based on combined molecular and
      morphological data. Many authors have suggested that paedomorphosis may be
      problematic in studies of salamander phylogeny, but this hypothesis has never
      been tested with a rigorous phylogenetic analysis. We find that the misleading
      effects of paedomorphosis on phylogenetic analysis go beyond the sharing of
      homoplastic larval traits by paedomorphic adults, and the problem therefore is
      not solved by simply excluding suspected paedomorphic characters. Instead, two
      additional factors are critically important in causing paedomorphic species to be
      phylogenetically "misplaced": (1) the absence of clade-specific synapomorphies
      that develop during metamorphosis in nonpaedomorphic taxa and allow their
      "correct" placement and (2) parallel adaptive changes associated with the aquatic
      habitat of the larval stage. Our results suggest that the effects of
      paedomorphosis on phylogenetic analyses may be complex, difficult to detect, and 
      can lead to results that are both wrong and statistically well supported by
      parsimony and Bayesian analyses.
FAU - Wiens, John
AU  - Wiens J
AD  - Department of Ecology and Evolution, Stony Brook University, Stony Brook, New
      York 11794-5245, USA. wiensj@life.bio.sunysb.edu
FAU - Bonett, Ronald
AU  - Bonett R
FAU - Chippindale, Paul
AU  - Chippindale P
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Embryonic Development/*genetics
MH  - Genes, RAG-1/genetics
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - RNA, Ribosomal/genetics
MH  - Research Design
MH  - Sequence Analysis, DNA
MH  - Urodela/*anatomy &amp; histology/embryology/*genetics
EDAT- 2005/04/05 09:00
MHDA- 2005/05/06 09:00
CRDT- 2005/04/05 09:00
PHST- 2005/04/05 09:00 [pubmed]
PHST- 2005/05/06 09:00 [medline]
PHST- 2005/04/05 09:00 [entrez]
AID - WP703888X72H6632 [pii]
AID - 10.1080/10635150590906037 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Feb;54(1):91-110. doi: 10.1080/10635150590906037.

PMID- 15805012
OWN - NLM
STAT- MEDLINE
DCOM- 20050505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 1
DP  - 2005 Feb
TI  - Mitochondrial phylogenetics and evolution of mysticete whales.
PG  - 77-90
AB  - The phylogenetic relationships among baleen whales (Order: Cetacea) remain
      uncertain despite extensive research in cetacean molecular phylogenetics and a
      potential morphological sample size of over 2 million animals harvested.
      Questions remain regarding the number of species and the monophyly of genera, as 
      well as higher order relationships. Here, we approach mysticete phylogeny with
      complete mitochondrial genome sequence analysis. We determined complete mtDNA
      sequences of 10 extant Mysticeti species, inferred their phylogenetic
      relationships, and estimated node divergence times. The mtDNA sequence analysis
      concurs with previous molecular studies in the ordering of the principal
      branches, with Balaenidae (right whales) as sister to all other mysticetes base, 
      followed by Neobalaenidae (pygmy right whale), Eschrichtiidae (gray whale), and
      finally Balaenopteridae (rorquals + humpback whale). The mtDNA analysis further
      suggests that four lineages exist within the clade of Eschrichtiidae +
      Balaenopteridae, including a sister relationship between the humpback and fin
      whales, and a monophyletic group formed by the blue, sei, and Bryde's whales,
      each of which represents a newly recognized phylogenetic relationship in
      Mysticeti. We also estimated the divergence times of all extant mysticete
      species, accounting for evolutionary rate heterogeneity among lineages. When the 
      mtDNA divergence estimates are compared with the mysticete fossil record, several
      lineages have molecular divergence estimates strikingly older than indicated by
      paleontological data. We suggest this discrepancy reflects both a large amount of
      ancestral polymorphism and long generation times of ancestral baleen whale
      populations.
FAU - Sasaki, Takeshi
AU  - Sasaki T
AD  - Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology,
      4259 Nagatsuta-cho, Midori-ku Yokohama, Kanagawa 226-8501, Japan.
FAU - Nikaido, Masato
AU  - Nikaido M
FAU - Hamilton, Healy
AU  - Hamilton H
FAU - Goto, Mutsuo
AU  - Goto M
FAU - Kato, Hidehiro
AU  - Kato H
FAU - Kanda, Naohisa
AU  - Kanda N
FAU - Pastene, Luis
AU  - Pastene L
FAU - Cao, Ying
AU  - Cao Y
FAU - Fordyce, R
AU  - Fordyce R
FAU - Hasegawa, Masami
AU  - Hasegawa M
FAU - Okada, Norihiro
AU  - Okada N
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA Primers
MH  - DNA, Mitochondrial/*genetics
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
MH  - Whales/classification/*genetics
EDAT- 2005/04/05 09:00
MHDA- 2005/05/06 09:00
CRDT- 2005/04/05 09:00
PHST- 2005/04/05 09:00 [pubmed]
PHST- 2005/05/06 09:00 [medline]
PHST- 2005/04/05 09:00 [entrez]
AID - R813631382258535 [pii]
AID - 10.1080/10635150590905939 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Feb;54(1):77-90. doi: 10.1080/10635150590905939.

PMID- 15805011
OWN - NLM
STAT- MEDLINE
DCOM- 20050505
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 1
DP  - 2005 Feb
TI  - Visualizing conflicting evolutionary hypotheses in large collections of trees:
      using consensus networks to study the origins of placentals and hexapods.
PG  - 66-76
AB  - Many phylogenetic methods produce large collections of trees as opposed to a
      single tree, which allows the exploration of support for various evolutionary
      hypotheses. However, to be useful, the information contained in large collections
      of trees should be summarized; frequently this is achieved by constructing a
      consensus tree. Consensus trees display only those signals that are present in a 
      large proportion of the trees. However, by their very nature consensus trees
      require that any conflicts between the trees are necessarily disregarded. We
      present a method that extends the notion of consensus trees to allow the
      visualization of conflicting hypotheses in a consensus network. We demonstrate
      the utility of this method in highlighting differences amongst maximum likelihood
      bootstrap values and Bayesian posterior probabilities in the placental mammal
      phylogeny, and also in comparing the phylogenetic signal contained in amino acid 
      versus nucleotide characters for hexapod monophyly.
FAU - Holland, Barbara
AU  - Holland B
AD  - The Allan Wilson Centre for Molecular Ecology and Evolution, Massey University,
      Private Bag 11222, Palmerston North, New Zealand. b.r.holland@massey.ac.nz
FAU - Delsuc, Frederic
AU  - Delsuc F
FAU - Moulton, Vincent
AU  - Moulton V
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Amino Acid Sequence/genetics
MH  - Animals
MH  - Base Sequence/genetics
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - Insecta/genetics
MH  - Likelihood Functions
MH  - Mammals/genetics
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2005/04/05 09:00
MHDA- 2005/05/06 09:00
CRDT- 2005/04/05 09:00
PHST- 2005/04/05 09:00 [pubmed]
PHST- 2005/05/06 09:00 [medline]
PHST- 2005/04/05 09:00 [entrez]
AID - X024100413841717 [pii]
AID - 10.1080/10635150590906055 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Feb;54(1):66-76. doi: 10.1080/10635150590906055.

PMID- 15805010
OWN - NLM
STAT- MEDLINE
DCOM- 20050505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 1
DP  - 2005 Feb
TI  - Biogeographic interpretation of splits graphs: least squares optimization of
      branch lengths.
PG  - 56-65
AB  - Although most often used to represent phylogenetic uncertainty, network methods
      are also potentially useful for describing the phylogenetic complexity expected
      to characterize recent species radiations. One network method with particular
      advantages in this context is split decomposition. However, in its standard
      implementation this approach is limited by a conservative criterion for branch
      length estimation. Here we extend the utility of split decomposition by
      introducing a least squares optimization technique for correcting branch lengths 
      that may be underestimated by the standard implementation. This optimization of
      branch lengths is generally expected to improve divergence time estimates
      calculated from splits graphs. We illustrate the effect of least squares
      optimization on such estimates using the Australasian Myosotis and the Hawaiian
      silversword alliance as examples. We also discuss the biogeographic
      interpretation and limitations of splits graphs.
FAU - Winkworth, Richard
AU  - Winkworth R
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven,
      Connecticut 06520, USA. Richard.Winkworth@yale.edu
FAU - Bryant, David
AU  - Bryant D
FAU - Lockhart, Peter
AU  - Lockhart P
FAU - Havell, David
AU  - Havell D
FAU - Moulton, Vincent
AU  - Moulton V
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Boraginaceae/genetics
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - Geography
MH  - Least-Squares Analysis
MH  - *Models, Genetic
MH  - New Zealand
MH  - *Phylogeny
MH  - Ranunculus/genetics
MH  - Species Specificity
EDAT- 2005/04/05 09:00
MHDA- 2005/05/06 09:00
CRDT- 2005/04/05 09:00
PHST- 2005/04/05 09:00 [pubmed]
PHST- 2005/05/06 09:00 [medline]
PHST- 2005/04/05 09:00 [entrez]
AID - T774RQM641211301 [pii]
AID - 10.1080/10635150590906046 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Feb;54(1):56-65. doi: 10.1080/10635150590906046.

PMID- 15805009
OWN - NLM
STAT- MEDLINE
DCOM- 20050505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 1
DP  - 2005 Feb
TI  - Is homoplasy or lineage sorting the source of incongruent mtdna and nuclear gene 
      trees in the stiff-tailed ducks (Nomonyx-Oxyura)?
PG  - 35-55
AB  - We evaluated the potential effects of homoplasy, ancestral polymorphism, and
      hybridization as obstacles to resolving phylogenetic relationships within
      Nomonyx-Oxyura stiff-tailed ducks (Oxyurinae; subtribe Oxyurina). Mitochondrial
      DNA (mtDNA) control region sequences from 94 individuals supported monophyly of
      mtDNA haplotypes for each of the six species and provided no evidence of extant
      incomplete lineage sorting or inter-specific hybridization. The ruddy ducks (O.
      j. jamaicensis,O. j. andina, O. j. ferruginea) are each others' closest
      relatives, but the lack of shared haplotypes between O. j. jamaicensis and O. j. 
      ferruginea suggests long-standing historical isolation. In contrast, O. j. andina
      shares haplotypes with O. j. jamaicensis and O. j. ferruginea, which supports
      Todd's (1979) and Fjeldsa's (1986) hypothesis that O. j. andina is an intergrade 
      or hybrid subspecies of O. j. jamaicensis and O. j. ferruginea. Control region
      data and a much larger data set composed of approximately 8800 base pairs of
      mitochondrial and nuclear sequence for each species indicate that the two New
      World species, O. vittata and O. jamaicensis, branch basally within Oxyura. A
      clade of three Old World species (O. australis, O. maccoa, O. leucocephala) is
      well supported, but different loci and also different characters within the mtDNA
      data support three different resolutions of the Old World clade, yielding an
      essentially unresolved trichotomy. Fundamentally different factors limited the
      resolution of the mtDNA and nuclear gene trees. Gene trees for most nuclear loci 
      were unresolved due to slow rates of mutation and a lack of informative
      variation, whereas uncertain resolution of the mtDNA gene tree was due to
      homoplasy. Within the mtDNA, approximately equal numbers of characters supported 
      each of three possible resolutions. Parametric and nonparametric bootstrap
      analyses suggest that resolution of the mtDNA tree based on ~4300 bp per taxon is
      uncertain but that complete mtDNA sequences would yield a fully resolved gene
      tree. A short internode separating O. leucocephala from (O. australis, O. maccoa)
      in the best mtDNA tree combined with long terminal branches and substantial rate 
      variation among nucleotide sites allowed the small number of changes occurring on
      the internode to be obscured by homoplasy in a significant portion of simulated
      data sets. Although most nuclear loci were uninformative, two loci supported a
      resolution of the Old World clade (O. maccoa, O. leucocephala) that is
      incongruent with the best mtDNA tree. Thus, incongruence between nuclear and
      mtDNA trees may be due to random sorting of ancestral lineages during the short
      internode, homoplasy in the mtDNA data, or both. The Oxyura trichotomy represents
      a difficult though likely common problem in molecular systematics. Given a short 
      internode, the mtDNA tree has a greater chance of being congruent with the
      history of speciation because its effective population size (N(e)) is one-quarter
      that of any nuclear locus, but its resolution is more likely to be obscured by
      homoplasy. In contrast, gene trees for more slowly evolving nuclear loci will be 
      difficult to resolve due to a lack of substitutions during the internode, and
      when resolved are more likely to be incongruent with the species history due to
      the stochastic effects of lineage sorting. We suggest that researchers consider
      first whether independent gene trees are adequately resolved and then whether
      those trees are congruent with the species history. In the case of Oxyura, the
      answer to both questions may be no. Complete mtDNA sequences combined with data
      from a very large number of nuclear loci may be the only way to resolve such
      trichotomies.
FAU - McCracken, Kevin
AU  - McCracken K
AD  - Institute of Arctic Biology, Department of Biology and Wildlife,University of
      Alaska Fairbanks, Fairbanks, Alaska 99775 USA. fnkgm@uaf.edu
FAU - Sorenson, Michael
AU  - Sorenson M
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Classification/*methods
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - Ducks/*genetics
MH  - *Evolution, Molecular
MH  - Geography
MH  - Haplotypes/genetics
MH  - Hybridization, Genetic
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Research Design
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2005/04/05 09:00
MHDA- 2005/05/06 09:00
CRDT- 2005/04/05 09:00
PHST- 2005/04/05 09:00 [pubmed]
PHST- 2005/05/06 09:00 [medline]
PHST- 2005/04/05 09:00 [entrez]
AID - WWMQ8U0H67X81756 [pii]
AID - 10.1080/10635150590910249 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Feb;54(1):35-55. doi: 10.1080/10635150590910249.

PMID- 15805008
OWN - NLM
STAT- MEDLINE
DCOM- 20050505
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 1
DP  - 2005 Feb
TI  - Phylogeny and biogeography of exacum (gentianaceae): a disjunctive distribution
      in the Indian ocean basin resulted from long distance dispersal and extensive
      radiation.
PG  - 21-34
AB  - Disjunctive distributions across paleotropical regions in the Indian Ocean Basin 
      (IOB) often invoke dispersal/vicariance debates. Exacum (Gentianaceae, tribe
      Exaceae) species are spread around the IOB, in Africa, Madagascar, Socotra, the
      Arabian peninsula, Sri Lanka, India, the Himalayas, mainland Southeast Asia
      including southern China and Malaysia, and northern Australia. The distribution
      of this genus was suggested to be a typical example of vicariance resulting from 
      the breakup of the Gondwanan supercontinent. The molecular phylogeny of Exacum is
      in principle congruent with morphological conclusions and shows a pattern that
      resembles a vicariance scenario with rapid divergence among lineages, but our
      molecular dating analysis demonstrates that the radiation is too recent to be
      associated with the Gondwanan continental breakup. We used our dating analysis to
      test the results of DIVA and found that the program predicted impossible
      vicariance events. Ancestral area reconstruction suggests that Exacum originated 
      in Madagascar, and divergence dating suggests its origin was not before the
      Eocene. The Madagascan progenitor, the most recent common ancestor of Exacum,
      colonized Sri Lanka and southern India via long-distance dispersals. This
      colonizer underwent an extensive range expansion and spread to Socotra-Arabia,
      northern India, and mainland Southeast Asia in the northern IOB when it was warm 
      and humid in these regions. This widespread common ancestor retreated
      subsequently from most parts of these regions and survived in isolation in
      Socotra-Arabia, southern India-Sri Lanka, and perhaps mainland Southeast Asia,
      possibly as a consequence of drastic climatic changes, particularly the spreading
      drought during the Neogene. Secondary diversification from these surviving
      centers and Madagascar resulted in the extant main lineages of the genus. The
      vicariance-like pattern shown by the phylogeny appears to have resulted from
      long-distance dispersals followed by extensive range expansion and subsequent
      fragmentation. The extant African species E. oldenlandioides is confirmed to be
      recently dispersed from Madagascar.
FAU - Yuan, Yong-Ming
AU  - Yuan YM
AD  - Laboratory of Evolutionary Botany, Institute of Botany, University of Neuchatel, 
      Emile-Argand 11, CH-2007 Neuchatel, Switzerland. yong-ming.yuan@unine.ch
FAU - Wohlhauser, Sebastien
AU  - Wohlhauser S
FAU - Moller, Michael
AU  - Moller M
FAU - Klackenberg, Jens
AU  - Klackenberg J
FAU - Callmander, Martin
AU  - Callmander M
FAU - Kupfer, Philippe
AU  - Kupfer P
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal Spacer)
SB  - IM
MH  - Africa
MH  - Asia
MH  - Base Sequence
MH  - DNA, Ribosomal Spacer/genetics
MH  - *Demography
MH  - *Evolution, Molecular
MH  - Gentianaceae/*genetics
MH  - Geography
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Population Dynamics
MH  - Sequence Analysis, DNA
EDAT- 2005/04/05 09:00
MHDA- 2005/05/06 09:00
CRDT- 2005/04/05 09:00
PHST- 2005/04/05 09:00 [pubmed]
PHST- 2005/05/06 09:00 [medline]
PHST- 2005/04/05 09:00 [entrez]
AID - HK56626MVM43P24R [pii]
AID - 10.1080/10635150590905867 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Feb;54(1):21-34. doi: 10.1080/10635150590905867.

PMID- 15805007
OWN - NLM
STAT- MEDLINE
DCOM- 20050505
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 54
IP  - 1
DP  - 2005 Feb
TI  - Exploring rate variation among and within sites in a densely sampled tree:
      species level phylogenetics of north american tiger beetles (genus cicindela).
PG  - 4-20
AB  - Species-level phylogenetic studies require fast-evolving nucleotide positions to 
      resolve relationships among close relatives, but these sites may be highly
      homoplastic and perhaps uninformative or even misleading deeper in the tree. Here
      we describe a species-level analysis of tiger beetles in the genus Cicindela
      (Coleoptera: Cicindelidae) for 132 terminal taxa and 1897 nucleotide positions
      from three regions of mtDNA, comprising 75% coverage of species occurring in
      North America. Evenly weighted parsimony analysis recovered four major clades
      representing radiations confined to North and Central America. Relationships near
      the tips were well supported but signal was contradictory at deeper nodes. Two
      major categories (3rd positions and all others) can be distinguished in
      likelihood analysis of character variation, of which only the fast-changing 3rd
      position characters were affected by saturation. However, their downweighting
      under a variety of criteria did not improve the tree topology at basal nodes.
      There was weak conflict between 3rd and non-3rd position characters deep in the
      tree, but support levels declined towards the root for all categories, even on
      trees that were reconstructed from 3rd and non-3rd positions separately.
      Statistical analysis of parsimony-based character transitions along branches
      showed a largely homogeneous distribution of change along the root-to-tip axis.
      The comparison of character transitions among the four major portions of the tree
      revealed deviations from stochastic distribution for the non-3rd positions, but
      not for 3rd positions. Hence, variability of functionally constrained non-3rd
      positions differs between clades and may be dependent on the character states at 
      other sites, consistent with the covarion model of molecular evolution. The
      results suggest that some properties of 3rd positions are less problematic for
      phylogenetic reconstruction than other categories despite their high total
      homoplasy. In densely sampled data sets of closely related species, the
      disadvantages of weighting schemes according to homoplasy levels outweigh the
      benefits, showing the difficulty of devising meaningful weighting schemes that
      are applicable universally throughout the tree.
FAU - Vogler, Alfried
AU  - Vogler A
AD  - Department of Entomology, The Natural History Museum, London SW7 5BD, United
      Kingdom. a.vogler@nhm.ac.uk
FAU - Cardoso, Anabela
AU  - Cardoso A
FAU - Barraclough, Timothy
AU  - Barraclough T
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Classification/*methods
MH  - Codon/genetics
MH  - Coleoptera/*genetics
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - North America
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2005/04/05 09:00
MHDA- 2005/05/06 09:00
CRDT- 2005/04/05 09:00
PHST- 2005/04/05 09:00 [pubmed]
PHST- 2005/05/06 09:00 [medline]
PHST- 2005/04/05 09:00 [entrez]
AID - K20H5P59413J12JH [pii]
AID - 10.1080/10635150590906028 [doi]
PST - ppublish
SO  - Syst Biol. 2005 Feb;54(1):4-20. doi: 10.1080/10635150590906028.

PMID- 15764566
OWN - NLM
STAT- MEDLINE
DCOM- 20050405
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 6
DP  - 2004 Dec
TI  - The information content of trees and their matrix representations.
PG  - 989-1001
FAU - Wilkinson, Mark
AU  - Wilkinson M
AD  - Department of Zoology, Natural History Museum, London SW7 5BD, United Kingdom.
FAU - Cotton, James
AU  - Cotton J
FAU - Thorley, Joseph
AU  - Thorley J
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Information Science
MH  - *Models, Theoretical
MH  - *Phylogeny
EDAT- 2005/03/15 09:00
MHDA- 2005/04/06 09:00
CRDT- 2005/03/15 09:00
PHST- 2005/03/15 09:00 [pubmed]
PHST- 2005/04/06 09:00 [medline]
PHST- 2005/03/15 09:00 [entrez]
AID - XW4L6R427D30H9KX [pii]
AID - 10.1080/10635150490522737 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Dec;53(6):989-1001. doi: 10.1080/10635150490522737.

PMID- 15764565
OWN - NLM
STAT- MEDLINE
DCOM- 20050405
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 6
DP  - 2004 Dec
TI  - Identifying and removing fast-evolving sites using compatibility analysis: an
      example from the Arthropoda.
PG  - 978-89
FAU - Pisani, Davide
AU  - Pisani D
AD  - Department of Zoology, The Natural History Museum, United Kingdom.
      d.pisani@nhm.ac.uk
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Arthropods/*genetics
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - *Models, Theoretical
MH  - *Phylogeny
EDAT- 2005/03/15 09:00
MHDA- 2005/04/06 09:00
CRDT- 2005/03/15 09:00
PHST- 2005/03/15 09:00 [pubmed]
PHST- 2005/04/06 09:00 [medline]
PHST- 2005/03/15 09:00 [entrez]
AID - GUV2545YGQ58JMU5 [pii]
AID - 10.1080/10635150490888877 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Dec;53(6):978-89. doi: 10.1080/10635150490888877.

PMID- 15764564
OWN - NLM
STAT- MEDLINE
DCOM- 20050405
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 6
DP  - 2004 Dec
TI  - Areas of endemism: an improved optimality criterion.
PG  - 968-77
AB  - The grid-based method to identify areas of endemism proposed by Szumik et al. is 
      extended. The improvements include the ability to assign scores of endemicity to 
      sets of disjoint areas, and to have each species contribute more to the score of 
      endemicity of an area, or less, according to how well its distribution matches
      the area. The modified method also allows for partially overlapping areas; an
      area partially overlapping with another one of higher score is retained when the 
      set of lower score has a minimum proportion of species endemic to it. Algorithms 
      to evaluate areas of endemism under this criterion are discussed, and implemented
      in a computer program (NDM). The new algorithms allow evaluation of much larger
      data sets.
FAU - Szumik, Claudia
AU  - Szumik C
AD  - Consejo Nacional de Investigaciones Cientificas y Tecnicas, Miguel Lillo 205,
      4000 San Miguel de Tucuman, Tucuman, Argentina.
FAU - Goloboff, Pablo
AU  - Goloboff P
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Animals
MH  - *Environment
MH  - Environmental Monitoring
MH  - Geography
MH  - *Models, Theoretical
MH  - Population Dynamics
EDAT- 2005/03/15 09:00
MHDA- 2005/04/06 09:00
CRDT- 2005/03/15 09:00
PHST- 2005/03/15 09:00 [pubmed]
PHST- 2005/04/06 09:00 [medline]
PHST- 2005/03/15 09:00 [entrez]
AID - GXLDRVH633JWPMB9 [pii]
AID - 10.1080/10635150490888859 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Dec;53(6):968-77. doi: 10.1080/10635150490888859.

PMID- 15764563
OWN - NLM
STAT- MEDLINE
DCOM- 20050405
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 6
DP  - 2004 Dec
TI  - Molecular clock fork phylogenies: closed form analytic maximum likelihood
      solutions.
PG  - 963-7
AB  - Maximum likelihood (ML) is increasingly used as an optimality criterion for
      selecting evolutionary trees, but finding the global optimum is a hard
      computational task. Because no general analytic solution is known, numeric
      techniques such as hill climbing or expectation maximization (EM) are used in
      order to find optimal parameters for a given tree. So far, analytic solutions
      were derived only for the simplest model-three-taxa, two-state characters, under 
      a molecular clock. Quoting Ziheng Yang, who initiated the analytic approach,"this
      seems to be the simplest case, but has many of the conceptual and statistical
      complexities involved in phylogenetic estimation."In this work, we give general
      analytic solutions for a family of trees with four-taxa, two-state characters,
      under a molecular clock. The change from three to four taxa incurs a major
      increase in the complexity of the underlying algebraic system, and requires novel
      techniques and approaches. We start by presenting the general maximum likelihood 
      problem on phylogenetic trees as a constrained optimization problem, and the
      resulting system of polynomial equations. In full generality, it is infeasible to
      solve this system, therefore specialized tools for the molecular clock case are
      developed. Four-taxa rooted trees have two topologies-the fork (two subtrees with
      two leaves each) and the comb (one subtree with three leaves, the other with a
      single leaf). We combine the ultrametric properties of molecular clock fork trees
      with the Hadamard conjugation to derive a number of topology dependent
      identities. Employing these identities, we substantially simplify the system of
      polynomial equations for the fork. We finally employ symbolic algebra software to
      obtain closed formanalytic solutions (expressed parametrically in the input
      data). In general, four-taxa trees can have multiple ML points. In contrast, we
      can now prove that each fork topology has a unique(local and global) ML point.
FAU - Chor, Benny
AU  - Chor B
AD  - School of Computer Science, Tel-Aviv University, Tel-Aviv, Israel.
      benny@cs.tau.ac.il
FAU - Snir, Sagi
AU  - Snir S
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Theoretical
MH  - *Phylogeny
EDAT- 2005/03/15 09:00
MHDA- 2005/04/06 09:00
CRDT- 2005/03/15 09:00
PHST- 2005/03/15 09:00 [pubmed]
PHST- 2005/04/06 09:00 [medline]
PHST- 2005/03/15 09:00 [entrez]
AID - V1M6DHNQ46PGGXW0 [pii]
AID - 10.1080/10635150490522728 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Dec;53(6):963-7. doi: 10.1080/10635150490522728.

PMID- 15764562
OWN - NLM
STAT- MEDLINE
DCOM- 20050405
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 6
DP  - 2004 Dec
TI  - Empirical problems of the hierarchical likelihood ratio test for model selection.
PG  - 949-62
AB  - Advocates of maximum likelihood (ML) approaches to phylogenetics commonly cite as
      one of their primary advantages the use of objective statistical criteria for
      model selection. Currently, a particular implementation of the likelihood ratio
      test (LRT) is the most commonly used model-selection criterion in phylogenetics. 
      This approach requires the choice of a starting point and a parameter addition
      (or removal) sequence that can affect all ML inferences (i.e., topology, model,
      and all evolutionary parameters). Here, several alternative starting points and
      parameter sequences are tested in empirical data sets to assess their influence
      on model selection and optimal topology. In the studied data sets, varying
      model-selection protocols leads to selection of different models that, in some
      cases, lead to different ML trees. Given the sensitivity of the LRT, some
      possible solutions to model selection (within the hypothesis testing approach)
      are outlined, and alternative model-selection criteria are discussed. Some of the
      suggested alternatives seem to lack these problems, although their behavior and
      adequacy for phylogenetics needs to be further explored.
FAU - Pol, Diego
AU  - Pol D
AD  - Division of Paleontology, American Museum of Natural History, New York, NY 10024,
      USA. dpol@amnh.org
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - *Likelihood Functions
MH  - *Models, Theoretical
MH  - *Phylogeny
MH  - Research Design
EDAT- 2005/03/15 09:00
MHDA- 2005/04/06 09:00
CRDT- 2005/03/15 09:00
PHST- 2005/03/15 09:00 [pubmed]
PHST- 2005/04/06 09:00 [medline]
PHST- 2005/03/15 09:00 [entrez]
AID - GC3WHXM4RHXP3QAU [pii]
AID - 10.1080/10635150490888868 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Dec;53(6):949-62. doi: 10.1080/10635150490888868.

PMID- 15764561
OWN - NLM
STAT- MEDLINE
DCOM- 20050405
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 6
DP  - 2004 Dec
TI  - Ecological correlates and evolutionary divergence in the skull of turtles: a
      geometric morphometric assessment.
PG  - 933-48
AB  - Resource use and phylogeny are often correlated with morphological variation.
      Moreover, because biological shapes are often complex and evolve depending on
      several internal constraints, they must be assessed using integrative methods. We
      analyzed the morphological variation of the turtle skull in the context of an
      adaptive radiation. Our focus are turtles of the superfamily Testudinoidea, which
      are remarkably diverse, both in number of species and in ecology. In this study, 
      we depict morphological variation in the turtle skull in three dimensions with
      respect to diet, phylogeny, and habitat using modern geometric morphometrics. Our
      study revealed that morphological specialization was related to both diet and
      habitat. Morphological variation is decomposed in regard of both resource use
      (habitat and diet) and phylogeny. Feeding mode depending on environment was
      suggested as a key factor determining morphological evolution and diversification
      of turtle skulls. Diet (especially durophagy) leads to parallel morphologies in
      different clades. Phylogeny seemed to constrain only localized features of the
      skull and remained of minor influence, because overall morphotypes, closely
      correlated with ecological factors, occurred in both clades. In conclusion, the
      adaptive radiation of the Testudinoidea is revealed to demonstrate a clear
      relationship between the skull shape and life style.
FAU - Claude, Julien
AU  - Claude J
AD  - Faculty of Science, Department of Biology, University of Mahasarakham, Tambon
      Khamriang, Kantarawichai District, Thailand. claude@msu.ac.th
FAU - Pritchard, Peter
AU  - Pritchard P
FAU - Tong, Haiyan
AU  - Tong H
FAU - Paradis, Emmanuel
AU  - Paradis E
FAU - Auffray, Jean-Christophe
AU  - Auffray JC
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Physiological
MH  - Animals
MH  - *Biological Evolution
MH  - Biometry
MH  - Diet
MH  - *Ecology
MH  - Environment
MH  - Female
MH  - Male
MH  - Skull/*anatomy &amp; histology
MH  - Turtles/*anatomy &amp; histology
EDAT- 2005/03/15 09:00
MHDA- 2005/04/06 09:00
CRDT- 2005/03/15 09:00
PHST- 2005/03/15 09:00 [pubmed]
PHST- 2005/04/06 09:00 [medline]
PHST- 2005/03/15 09:00 [entrez]
AID - K3QNKHUPCC3FWTTF [pii]
AID - 10.1080/10635150490889498 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Dec;53(6):933-48. doi: 10.1080/10635150490889498.

PMID- 15764560
OWN - NLM
STAT- MEDLINE
DCOM- 20050405
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 6
DP  - 2004 Dec
TI  - Evolution of a RNA polymerase gene family in Silene (Caryophyllaceae)-incomplete 
      concerted evolution and topological congruence among paralogues.
PG  - 914-32
AB  - Four low-copy nuclear DNA intron regions from the second largest subunits of the 
      RNA polymerase gene family (RPA2, RPB2, RPD2a, and RPD2b), the internal
      transcribed spacers (ITSs) from the nuclear ribosomal regions, and the rps16
      intron from the chloroplast were sequenced and used in a phylogenetic analysis of
      29 species from the tribe Sileneae (Caryophyllaceae). We used a low stringency
      nested polymerase chain reaction (PCR) approach to overcome the difficulties of
      constructing specific primers for amplification of the low copy nuclear DNA
      regions. Maximum parsimony analyses resulted in largely congruent phylogenetic
      trees for all regions. We tested overall model congruence in a likelihood context
      using the software PLATO and found that ITSs, RPA2, and RPB2 deviated from the
      maximum likelihood model for the combined data. The topology parameter was then
      isolated and topological congruence assessed by nonparametric bootstrapping. No
      strong topological incongruence was found. The analysis of the combined data sets
      resolves previously poorly known major relationships within Sileneae. Two
      paralogues of RPD2 were found, and several independent losses and incomplete
      concerted evolution were inferred. The among-site rate variation was
      significantly lower in the RNA polymerase introns than in the rps16 intron and
      ITSs, a property that is attractive in phylogenetic analyses.
FAU - Popp, Magnus
AU  - Popp M
AD  - Department of Systematic Botany, Evolutionary Biology Centre, Uppsala University,
      Norbyvagen 18D, Uppsala, Sweden. magnus.popp@ebc.uu.se
FAU - Oxelman, Bengt
AU  - Oxelman B
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 2.7.7.6 (DNA-Directed RNA Polymerases)
SB  - IM
MH  - DNA-Directed RNA Polymerases/*genetics
MH  - *Evolution, Molecular
MH  - Introns
MH  - *Phylogeny
MH  - Polymerase Chain Reaction
MH  - Silene/*genetics
EDAT- 2005/03/15 09:00
MHDA- 2005/04/06 09:00
CRDT- 2005/03/15 09:00
PHST- 2005/03/15 09:00 [pubmed]
PHST- 2005/04/06 09:00 [medline]
PHST- 2005/03/15 09:00 [entrez]
AID - Y5NQ6XFKP3Q2M1BA [pii]
AID - 10.1080/10635150490888840 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Dec;53(6):914-32. doi: 10.1080/10635150490888840.

PMID- 15764559
OWN - NLM
STAT- MEDLINE
DCOM- 20050405
LR  - 20071114
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 6
DP  - 2004 Dec
TI  - Frequentist properties of Bayesian posterior probabilities of phylogenetic trees 
      under simple and complex substitution models.
PG  - 904-13
AB  - What does the posterior probability of a phylogenetic tree mean?This simulation
      study shows that Bayesian posterior probabilities have the meaning that is
      typically ascribed to them; the posterior probability of a tree is the
      probability that the tree is correct, assuming that the model is correct. At the 
      same time, the Bayesian method can be sensitive to model misspecification, and
      the sensitivity of the Bayesian method appears to be greater than the sensitivity
      of the nonparametric bootstrap method (using maximum likelihood to estimate
      trees). Although the estimates of phylogeny obtained by use of the method of
      maximum likelihood or the Bayesian method are likely to be similar, the
      assessment of the uncertainty of inferred trees via either bootstrapping (for
      maximum likelihood estimates) or posterior probabilities (for Bayesian estimates)
      is not likely to be the same. We suggest that the Bayesian method be implemented 
      with the most complex models of those currently available, as this should reduce 
      the chance that the method will concentrate too much probability on too few
      trees.
FAU - Huelsenbeck, John
AU  - Huelsenbeck J
AD  - Section of Ecology, Behavior and Evolution, Division of Biological Sciences,
      University of California, San Diego, La Jolla, CA 92093-011, USA.
      johnh@biomail.ucsd.edu
FAU - Rannala, Bruce
AU  - Rannala B
LA  - eng
GR  - GM-069801/GM/NIGMS NIH HHS/United States
GR  - HG01988/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - *Models, Theoretical
MH  - *Phylogeny
EDAT- 2005/03/15 09:00
MHDA- 2005/04/06 09:00
CRDT- 2005/03/15 09:00
PHST- 2005/03/15 09:00 [pubmed]
PHST- 2005/04/06 09:00 [medline]
PHST- 2005/03/15 09:00 [entrez]
AID - W9ELDJ3NTGH5RMLE [pii]
AID - 10.1080/10635150490522629 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Dec;53(6):904-13. doi: 10.1080/10635150490522629.

PMID- 15764558
OWN - NLM
STAT- MEDLINE
DCOM- 20050405
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 6
DP  - 2004 Dec
TI  - Trends in morphological evolution in homobasidiomycetes inferred using maximum
      likelihood: a comparison of binary and multistate approaches.
PG  - 889-903
AB  - The homobasidiomycetes is a diverse group of macrofungi that includes mushrooms, 
      puffballs, coral fungi, and other forms. This study used maximum likelihood
      methods to determine if there are general trends (evolutionary tendencies) in the
      evolution of fruiting body forms in homobasidiomycetes, and to estimate the
      ancestral forms of the homobasidiomycetes and euagarics clade. Character
      evolution was modeled using a published 481-species phylogeny under two
      character-coding regimes: additive binary coding, using DISCRETE, and multistate 
      (five-state) coding, using MULTISTATE. Inferences regarding trends in character
      evolution made under binary coding were often in conflict with those made under
      multistate coding, suggesting that the additive binary coding approach cannot
      serve as a surrogate for multistate methods. MULTISTATE was used to develop
      a"minimal"model of fruiting body evolution, in which the 20 parameters that
      specify rates of transformations among character states were grouped into the
      fewest possible rate categories. The minimal model required only four rate
      categories, one of which is approaching zero, and suggests the following
      conclusions regarding trends in evolution of homobasidiomycete fruiting bodies:
      (1) there is an active trend favoring the evolution of pileate-stipitate forms
      (those with a cap and stalk); (2) the hypothesis that the evolution of gasteroid 
      forms (those with internal spore production, such as puffballs) is irreversible
      cannot be rejected; and (3) crustlike resupinate forms are not a particularly
      labile morphology. The latter finding contradicts the conclusions of a previous
      study that used binary character coding. Ancestral state reconstructions under
      binary coding suggest that the ancestor of the homobasidiomycetes was resupinate 
      and the ancestor of the euagarics clade was pileate-stipitate, but ancestral
      state reconstructions under multistate coding did not resolve the ancestral form 
      of either node. The results of this study illustrate the potential sensitivity of
      comparative analyses to character state definitions.
FAU - Hibbett, David
AU  - Hibbett D
AD  - Biology Department, Clark University, Worcester, MA 01610, USA.
      dhibbett@black.clarku.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Basidiomycota/*classification
MH  - *Biological Evolution
MH  - *Models, Theoretical
MH  - *Phylogeny
MH  - Reproduction
EDAT- 2005/03/15 09:00
MHDA- 2005/04/06 09:00
CRDT- 2005/03/15 09:00
PHST- 2005/03/15 09:00 [pubmed]
PHST- 2005/04/06 09:00 [medline]
PHST- 2005/03/15 09:00 [entrez]
AID - Q1WGLMPAQ5K55HKN [pii]
AID - 10.1080/10635150490522610 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Dec;53(6):889-903. doi: 10.1080/10635150490522610.

PMID- 15764557
OWN - NLM
STAT- MEDLINE
DCOM- 20050405
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 6
DP  - 2004 Dec
TI  - Model parameterization, prior distributions, and the general time-reversible
      model in Bayesian phylogenetics.
PG  - 877-88
AB  - Bayesian phylogenetic methods require the selection of prior probability
      distributions for all parameters of the model of evolution. These distributions
      allow one to incorporate prior information into a Bayesian analysis, but even in 
      the absence of meaningful prior information, a prior distribution must be chosen.
      In such situations, researchers typically seek to choose a prior that will have
      little effect on the posterior estimates produced by an analysis, allowing the
      data to dominate. Sometimes a prior that is uniform (assigning equal prior
      probability density to all points within some range) is chosen for this purpose. 
      In reality, the appropriate prior depends on the parameterization chosen for the 
      model of evolution, a choice that is largely arbitrary. There is an extensive
      Bayesian literature on appropriate prior choice, and it has long been appreciated
      that there are parameterizations for which uniform priors can have a strong
      influence on posterior estimates. We here discuss the relationship between model 
      parameterization and prior specification, using the general time-reversible model
      of nucleotide evolution as an example. We present Bayesian analyses of 10
      simulated data sets obtained using a variety of prior distributions and
      parameterizations of the general time-reversible model. Uniform priors can
      produce biased parameter estimates under realistic conditions, and a variety of
      alternative priors avoid this bias.
FAU - Zwickl, Derrick
AU  - Zwickl D
AD  - Section of Integrative Biology, University of Texas, 1 University Station C0930, 
      Austin, TX 78712, USA. zwickl@mail.utexas.edu
FAU - Holder, Mark
AU  - Holder M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - *Models, Theoretical
MH  - *Phylogeny
MH  - Time Factors
EDAT- 2005/03/15 09:00
MHDA- 2005/04/06 09:00
CRDT- 2005/03/15 09:00
PHST- 2005/03/15 09:00 [pubmed]
PHST- 2005/04/06 09:00 [medline]
PHST- 2005/03/15 09:00 [entrez]
AID - L0483QHPUCDQP3P7 [pii]
AID - 10.1080/10635150490522584 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Dec;53(6):877-88. doi: 10.1080/10635150490522584.

PMID- 15764556
OWN - NLM
STAT- MEDLINE
DCOM- 20050405
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 6
DP  - 2004 Dec
TI  - Complex biogeographic patterns in Androsace (Primulaceae) and related genera:
      evidence from phylogenetic analyses of nuclear internal transcribed spacer and
      plastid trnL-F sequences.
PG  - 856-76
AB  - We conducted phylogenetic analyses of Androsace and the closely related genera
      Douglasia, Pomatosace, and Vitaliana using DNA sequences of the nuclear internal 
      transcribed spacer (ITS) and the plastid trnL-F region. Analyses using maximum
      parsimony and Bayesian inference yield congruent relationships among several
      major lineages found. These lineages largely disagree with previously recognized 
      taxonomic groups. Most notably, (1) Androsace sect. Andraspis, comprising the
      short-lived taxa, is highly polyphyletic; (2) Pomatosace constitutes a separate
      phylogenetic lineage within Androsace; and (3) Douglasia and Vitaliana nest
      within Androsace sect. Aretia. Our results suggest multiple origins of the
      short-lived lifeform and a possible reversal from annual or biennial to perennial
      habit at the base of a group that now contains mostly perennial high mountain or 
      arctic taxa. The group containing Androsace sect. Aretia, Douglasia, and
      Vitaliana includes predominantly high alpine and arctic taxa with an
      arctic-alpine distribution, but is not found in the European and northeastern
      American Arctic or in Central and East Asia. This group probably originated in
      Europe in the Pliocene, from where it reached the amphi-Beringian region in the
      Pleistocene or late Pliocene.
FAU - Schneeweiss, Gerald
AU  - Schneeweiss G
AD  - Department of Plant Biogeography, Institute of Botany, University of
      ViennaRennweg 14, A-1030, Vienna, Austria. gerald.schneeweiss@univie.ac.at
FAU - Schonswetter, Peter
AU  - Schonswetter P
FAU - Kelso, Sylvia
AU  - Kelso S
FAU - Niklfeld, Harald
AU  - Niklfeld H
LA  - eng
SI  - GENBANK/AY274936
SI  - GENBANK/AY274937
SI  - GENBANK/AY274938
SI  - GENBANK/AY274939
SI  - GENBANK/AY274940
SI  - GENBANK/AY274941
SI  - GENBANK/AY274942
SI  - GENBANK/AY274943
SI  - GENBANK/AY274944
SI  - GENBANK/AY274945
SI  - GENBANK/AY274946
SI  - GENBANK/AY274947
SI  - GENBANK/AY274948
SI  - GENBANK/AY274949
SI  - GENBANK/AY274950
SI  - GENBANK/AY274951
SI  - GENBANK/AY274952
SI  - GENBANK/AY274953
SI  - GENBANK/AY274954
SI  - GENBANK/AY274955
SI  - GENBANK/AY274956
SI  - GENBANK/AY274957
SI  - GENBANK/AY274958
SI  - GENBANK/AY274959
SI  - GENBANK/AY274960
SI  - GENBANK/AY274961
SI  - GENBANK/AY274962
SI  - GENBANK/AY274963
SI  - GENBANK/AY274964
SI  - GENBANK/AY274965
SI  - GENBANK/AY274966
SI  - GENBANK/AY274967
SI  - GENBANK/AY274968
SI  - GENBANK/AY274969
SI  - GENBANK/AY274970
SI  - GENBANK/AY274971
SI  - GENBANK/AY274972
SI  - GENBANK/AY274973
SI  - GENBANK/AY274974
SI  - GENBANK/AY274975
SI  - GENBANK/AY274976
SI  - GENBANK/AY274977
SI  - GENBANK/AY274978
SI  - GENBANK/AY274979
SI  - GENBANK/AY274980
SI  - GENBANK/AY274981
SI  - GENBANK/AY274982
SI  - GENBANK/AY274983
SI  - GENBANK/AY274984
SI  - GENBANK/AY274985
SI  - GENBANK/AY274986
SI  - GENBANK/AY274987
SI  - GENBANK/AY274988
SI  - GENBANK/AY274989
SI  - GENBANK/AY274990
SI  - GENBANK/AY274991
SI  - GENBANK/AY274992
SI  - GENBANK/AY274993
SI  - GENBANK/AY274994
SI  - GENBANK/AY274995
SI  - GENBANK/AY274996
SI  - GENBANK/AY274997
SI  - GENBANK/AY274998
SI  - GENBANK/AY274999
SI  - GENBANK/AY275000
SI  - GENBANK/AY275001
SI  - GENBANK/AY275002
SI  - GENBANK/AY275003
SI  - GENBANK/AY275004
SI  - GENBANK/AY275005
SI  - GENBANK/AY275006
SI  - GENBANK/AY275007
SI  - GENBANK/AY275008
SI  - GENBANK/AY275009
SI  - GENBANK/AY275010
SI  - GENBANK/AY275011
SI  - GENBANK/AY275012
SI  - GENBANK/AY275013
SI  - GENBANK/AY275014
SI  - GENBANK/AY275015
SI  - GENBANK/AY275016
SI  - GENBANK/AY275017
SI  - GENBANK/AY275018
SI  - GENBANK/AY275019
SI  - GENBANK/AY275020
SI  - GENBANK/AY275021
SI  - GENBANK/AY275022
SI  - GENBANK/AY275023
SI  - GENBANK/AY275024
SI  - GENBANK/AY275025
SI  - GENBANK/AY275026
SI  - GENBANK/AY275027
SI  - GENBANK/AY275028
SI  - GENBANK/AY275029
SI  - GENBANK/AY275030
SI  - GENBANK/AY275031
SI  - GENBANK/AY275032
SI  - GENBANK/AY275033
SI  - GENBANK/AY275034
SI  - GENBANK/AY275035
SI  - GENBANK/AY275036
SI  - GENBANK/AY275037
SI  - GENBANK/AY275038
SI  - GENBANK/AY275039
SI  - GENBANK/AY275040
SI  - GENBANK/AY275041
SI  - GENBANK/AY275042
SI  - GENBANK/AY275043
SI  - GENBANK/AY275044
SI  - GENBANK/AY275045
SI  - GENBANK/AY275046
SI  - GENBANK/AY275047
SI  - GENBANK/AY275048
SI  - GENBANK/AY275049
SI  - GENBANK/AY275050
SI  - GENBANK/AY275051
SI  - GENBANK/AY275052
SI  - GENBANK/AY275053
SI  - GENBANK/AY275054
SI  - GENBANK/AY275055
SI  - GENBANK/AY275056
SI  - GENBANK/AY275057
SI  - GENBANK/AY275058
SI  - GENBANK/AY275059
SI  - GENBANK/AY275060
SI  - GENBANK/AY275061
SI  - GENBANK/AY275062
SI  - GENBANK/AY275063
SI  - GENBANK/AY275064
SI  - GENBANK/AY275065
SI  - GENBANK/AY275066
SI  - GENBANK/AY275067
SI  - GENBANK/AY275068
SI  - GENBANK/AY275069
SI  - GENBANK/AY275070
SI  - GENBANK/AY275071
SI  - GENBANK/AY275072
SI  - GENBANK/AY275073
SI  - GENBANK/AY275074
SI  - GENBANK/AY275075
SI  - GENBANK/AY275076
SI  - GENBANK/AY275077
SI  - GENBANK/AY275078
SI  - GENBANK/AY275079
SI  - GENBANK/AY275080
SI  - GENBANK/AY275081
SI  - GENBANK/AY275082
SI  - GENBANK/AY275083
SI  - GENBANK/AY275084
SI  - GENBANK/AY275085
SI  - GENBANK/AY275086
SI  - GENBANK/AY275087
SI  - GENBANK/AY275088
SI  - GENBANK/AY275089
SI  - GENBANK/AY275090
SI  - GENBANK/AY275091
SI  - GENBANK/AY275092
SI  - GENBANK/AY275093
SI  - GENBANK/AY275094
SI  - GENBANK/AY275095
SI  - GENBANK/AY275096
SI  - GENBANK/AY275097
SI  - GENBANK/AY275098
SI  - GENBANK/AY275099
SI  - GENBANK/AY275100
SI  - GENBANK/AY275101
SI  - GENBANK/AY275102
SI  - GENBANK/AY275103
SI  - GENBANK/AY275104
SI  - GENBANK/AY275105
SI  - GENBANK/AY502123
SI  - GENBANK/AY502124
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Base Sequence
MH  - Biological Evolution
MH  - DNA, Plant/*genetics
MH  - Geography
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Plastids/genetics
MH  - Primulaceae/*genetics
MH  - *Reproduction
MH  - Sequence Analysis, DNA
EDAT- 2005/03/15 09:00
MHDA- 2005/04/06 09:00
CRDT- 2005/03/15 09:00
PHST- 2005/03/15 09:00 [pubmed]
PHST- 2005/04/06 09:00 [medline]
PHST- 2005/03/15 09:00 [entrez]
AID - NJCLQEVAH1BQ2DQQ [pii]
AID - 10.1080/10635150490522566 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Dec;53(6):856-76. doi: 10.1080/10635150490522566.

PMID- 15764555
OWN - NLM
STAT- MEDLINE
DCOM- 20050405
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 6
DP  - 2004 Dec
TI  - Neglected taxonomy of rare desert fishes: congruent evidence for two species of
      leatherside chub.
PG  - 841-55
AB  - Conservation biologists rely heavily on taxonomy to set the scope for biological 
      monitoring and recovery planning of rare or threatened species. Yet, taxonomic
      boundaries are seldom evaluated as falsifiable hypotheses that can be
      statistically tested. Here, we examine species boundaries in leatherside chub
      (Teleostei, Cyprinidae), an imperiled desert fish native to the Bonneville Basin 
      and upper Snake River drainages of western North America. Recent molecular data
      hint that this fish could be composed of two distinct taxa that are
      geographically separated into northern and southern species. To formally test
      this hypothesis, we evaluated leatherside chub using several different categories
      of species concepts, including criteria dependent on phylogenetic, morphological,
      and ecological data. We found that leatherside chub is composed of two
      reciprocally monophyletic clades (candidate species) characterized by numerous
      fixed genetic differences for both mitochondrial and nuclear DNA markers; mtDNA
      sequence divergence between the two clades approached 8%. The candidate species
      also showed significant differences in cranial shape, revealed by morphometric
      analysis. Finally, controlled growth and foraging experiments using
      representative populations from each clade show that candidate species appear to 
      be locally adapted to the thermal environments where they now occur. Combined,
      these three lines of evidence support the hypothesis that leatherside chub is
      composed of two species. Moreover, all lines of evidence place these two species 
      within the genusLepidomeda, a group consisting of three additional species of
      endangered spinedace fishes, and one extinct species, all native to the Colorado 
      River system. Hence, we elevate the two clades of leatherside chub to distinct
      species status (Lepidomeda copeiin the north andL. aliciaein the south), and
      argue that each warrants independent conservation and recovery action.
FAU - Johnson, Jerald
AU  - Johnson J
AD  - Department of Integrative Biology, Brigham Young University, Provo, UT 84602,
      USA. jerry.johnson@byu.edu
FAU - Dowling, Thomas
AU  - Dowling T
FAU - Belk, Mark
AU  - Belk M
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Genetic Markers)
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - *Classification
MH  - Conservation of Natural Resources
MH  - Cyprinidae/*classification/*genetics
MH  - DNA, Mitochondrial/*genetics
MH  - Desert Climate
MH  - *Genetic Markers
MH  - North America
EDAT- 2005/03/15 09:00
MHDA- 2005/04/06 09:00
CRDT- 2005/03/15 09:00
PHST- 2005/03/15 09:00 [pubmed]
PHST- 2005/04/06 09:00 [medline]
PHST- 2005/03/15 09:00 [entrez]
AID - MW2LE260BKB14WMK [pii]
AID - 10.1080/10635150490522557 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Dec;53(6):841-55. doi: 10.1080/10635150490522557.

PMID- 15545259
OWN - NLM
STAT- MEDLINE
DCOM- 20050222
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 5
DP  - 2004 Oct
TI  - Metacanthomorpha: essay on a phylogeny-oriented database for morphology--the
      acanthomorph (teleostei) example.
PG  - 822-34
FAU - Dettai, Agnes
AU  - Dettai A
AD  - Equipe Phylogenie, UMR 7138, Departement Systematique et Evolution, Museum
      National d'Histoire Naturelle, 75231 Paris, France.
FAU - Bailly, Nicolas
AU  - Bailly N
FAU - Vignes-Lebbe, Regine
AU  - Vignes-Lebbe R
FAU - Lecointre, Guillaume
AU  - Lecointre G
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Databases, Factual
MH  - Fishes/*anatomy &amp; histology
MH  - *Phylogeny
EDAT- 2004/11/17 09:00
MHDA- 2005/02/23 09:00
CRDT- 2004/11/17 09:00
PHST- 2004/11/17 09:00 [pubmed]
PHST- 2005/02/23 09:00 [medline]
PHST- 2004/11/17 09:00 [entrez]
AID - XGXVC0R16FY7YTH8 [pii]
AID - 10.1080/10635150490522313 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Oct;53(5):822-34. doi: 10.1080/10635150490522313.

PMID- 15545258
OWN - NLM
STAT- MEDLINE
DCOM- 20050222
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 5
DP  - 2004 Oct
TI  - The phylogenetic definition of reptilia.
PG  - 815-21
FAU - Modesto, Sean P
AU  - Modesto SP
AD  - Department of Biology, University College of Cape Breton, Sydney, Nova Scotia B1P
      6LS, Canada. sean_modesto@uccb.ca
FAU - Anderson, Jason S
AU  - Anderson JS
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Phylogeny
MH  - Reptiles/*classification
MH  - *Terminology as Topic
EDAT- 2004/11/17 09:00
MHDA- 2005/02/23 09:00
CRDT- 2004/11/17 09:00
PHST- 2004/11/17 09:00 [pubmed]
PHST- 2005/02/23 09:00 [medline]
PHST- 2004/11/17 09:00 [entrez]
AID - Y3CCMYB8GV4M6LCL [pii]
AID - 10.1080/10635150490503026 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Oct;53(5):815-21. doi: 10.1080/10635150490503026.

PMID- 15545257
OWN - NLM
STAT- MEDLINE
DCOM- 20050222
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 5
DP  - 2004 Oct
TI  - Effects of tree shape on the accuracy of maximum likelihood-based ancestor
      reconstructions.
PG  - 809-14
FAU - Mooers, Arne O
AU  - Mooers AO
AD  - Department of Biological Sciences, Simon Fraser University, Burnaby, BC V5A 1S6, 
      Canada. amooers@sfu.ca
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - *Likelihood Functions
MH  - Markov Chains
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2004/11/17 09:00
MHDA- 2005/02/23 09:00
CRDT- 2004/11/17 09:00
PHST- 2004/11/17 09:00 [pubmed]
PHST- 2005/02/23 09:00 [medline]
PHST- 2004/11/17 09:00 [entrez]
AID - YB8KM26E3LBR679V [pii]
AID - 10.1080/10635150490502595 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Oct;53(5):809-14. doi: 10.1080/10635150490502595.

PMID- 15545256
OWN - NLM
STAT- MEDLINE
DCOM- 20050222
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 5
DP  - 2004 Oct
TI  - Model selection and model averaging in phylogenetics: advantages of akaike
      information criterion and bayesian approaches over likelihood ratio tests.
PG  - 793-808
AB  - Model selection is a topic of special relevance in molecular phylogenetics that
      affects many, if not all, stages of phylogenetic inference. Here we discuss some 
      fundamental concepts and techniques of model selection in the context of
      phylogenetics. We start by reviewing different aspects of the selection of
      substitution models in phylogenetics from a theoretical, philosophical and
      practical point of view, and summarize this comparison in table format. We argue 
      that the most commonly implemented model selection approach, the hierarchical
      likelihood ratio test, is not the optimal strategy for model selection in
      phylogenetics, and that approaches like the Akaike Information Criterion (AIC)
      and Bayesian methods offer important advantages. In particular, the latter two
      methods are able to simultaneously compare multiple nested or nonnested models,
      assess model selection uncertainty, and allow for the estimation of phylogenies
      and model parameters using all available models (model-averaged inference or
      multimodel inference). We also describe how the relative importance of the
      different parameters included in substitution models can be depicted. To
      illustrate some of these points, we have applied AIC-based model averaging to 37 
      mitochondrial DNA sequences from the subgenus Ohomopterus(genus Carabus) ground
      beetles described by Sota and Vogler (2001).
FAU - Posada, David
AU  - Posada D
AD  - Departamento de Bioquimica, Genetica e Inmunologia, Facultad de Biologia,
      Universidad de Vigo, Vigo 36200, Spain. dposada@uvigo.es
FAU - Buckley, Thomas R
AU  - Buckley TR
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - Coleoptera/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2004/11/17 09:00
MHDA- 2005/02/23 09:00
CRDT- 2004/11/17 09:00
PHST- 2004/11/17 09:00 [pubmed]
PHST- 2005/02/23 09:00 [medline]
PHST- 2004/11/17 09:00 [entrez]
AID - REHE4656RGAP72AK [pii]
AID - 10.1080/10635150490522304 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Oct;53(5):793-808. doi: 10.1080/10635150490522304.

PMID- 15545255
OWN - NLM
STAT- MEDLINE
DCOM- 20050222
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 5
DP  - 2004 Oct
TI  - Testing nested phylogenetic and phylogeographic hypotheses in the Plethodon
      vandykei species group.
PG  - 781-92
AB  - Mesic forests in the North American Pacific Northwest occur in two disjunct
      areas: along the coastal and Cascade ranges of Oregon, Washington, and British
      Columbia as well as the Northern Rocky Mountains of Idaho, Montana, and British
      Columbia. Over 150 species or species complexes have disjunct populations in each
      area, and a priori hypotheses based on phytogeography and geology potentially
      explain the disjunction via either dispersal or vicariance. Here, we test these
      hypotheses in the disjunct salamander complex Plethodon vandykei and P.
      idahoensisby collecting genetic data (669 bp of Cyt b) from 262 individuals.
      Maximum likelihood analysis indicated reciprocal monophyly of these species,
      supporting the ancient vicariance hypothesis, whereas parametric bootstrap and
      Bayesian hypothesis testing allow rejection of the dispersal hypothesis. The
      coalescent estimate of the time since population divergence (estimated using
      MDIV) is 3.75 x 106 years, and the 95%credibility interval of this value overlaps
      with the geological estimate of vicariance, but not the hypothesized dispersal.
      These results are congruent with the pattern seen in other mesic forest amphibian
      lineages and suggest disjunction in amphibians may be a concerted response to a
      geological/climatological event. WithinP. idahoensis, we tested the corollary
      hypothesis of an inland Pleistocene refugium in the Clearwater drainage with
      nested clade analysis and coalescent estimates of population growth rate (g).
      Both analyses support post-Pleistocene expansion from the Clearwater refugium. We
      corroborated this result by calculating Tajima's Dand mismatch distribution
      within each drainage, showing strong evidence for recent population expansion
      within most drainages. This work demonstrates the utility of statistical
      phylogeography and contributes two novel analytical tools: tests of stationarity 
      with respect to topology in the Bayesian estimation, and the use of coalescent
      simulations to test the significance of the population growth-rate parameter.
FAU - Carstens, Bryan C
AU  - Carstens BC
AD  - Department of Biological Sciences, Moscow, Idaho 83844-3051, USA.
      cars5766@uidaho.edu
FAU - Stevenson, Angela L
AU  - Stevenson AL
FAU - Degenhardt, Jeremiah D
AU  - Degenhardt JD
FAU - Sullivan, Jack
AU  - Sullivan J
LA  - eng
GR  - P20 RR16454/RR/NCRR NIH HHS/United States
GR  - P20 RR16488/RR/NCRR NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
SB  - IM
MH  - Analysis of Variance
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - British Columbia
MH  - Computer Simulation
MH  - DNA Primers
MH  - Demography
MH  - *Environment
MH  - *Evolution, Molecular
MH  - *Genetic Variation
MH  - Geography
MH  - Geological Phenomena
MH  - Geology
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Northwestern United States
MH  - *Phylogeny
MH  - Population Dynamics
MH  - Sequence Analysis, DNA
MH  - Species Specificity
MH  - Urodela/*genetics
EDAT- 2004/11/17 09:00
MHDA- 2005/02/23 09:00
CRDT- 2004/11/17 09:00
PHST- 2004/11/17 09:00 [pubmed]
PHST- 2005/02/23 09:00 [medline]
PHST- 2004/11/17 09:00 [entrez]
AID - JF2EJM902WAGM3DP [pii]
AID - 10.1080/10635150490522296 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Oct;53(5):781-92. doi: 10.1080/10635150490522296.

PMID- 15545254
OWN - NLM
STAT- MEDLINE
DCOM- 20050222
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 5
DP  - 2004 Oct
TI  - Molecular systematics and biogeography of the southern South american freshwater 
      "crabs" Aegla (decapoda: Anomura: Aeglidae) using multiple heuristic tree search 
      approaches.
PG  - 767-80
AB  - Recently new heuristic genetic algorithms such as Treefinder and MetaGA have been
      developed to search for optimal trees in a maximum likelihood (ML) framework. In 
      this study we combined these methods with other standard heuristic approaches
      such as ML and maximum parsimony hill-climbing searches and Bayesian inference
      coupled with Markov chain Monte Carlo techniques under homogeneous and mixed
      models of evolution to conduct an extensive phylogenetic analysis of the most
      abundant and widely distributed southern South American freshwater"crab,"the
      Aegla(Anomura: Aeglidae). A total of 167 samples representing 64 Aegla species
      and subspecies were sequenced for one nuclear (28S rDNA) and four mitochondrial
      (12S and 16S rDNA, COI, and COII) genes (5352 bp total). Additionally, six other 
      anomuran species from the genera Munida,Pachycheles, and
      Uroptychus(Galatheoidea), Lithodes(Paguroidea), and Lomis(Lomisoidea) and the
      nuclear 18S rDNA gene (1964 bp) were included in preliminary analyses for rooting
      the Aegla tree. Nonsignificantly different phylogenetic hypotheses resulted from 
      all the different heuristic methods used here, although the best scored
      topologies found under the ML hill-climbing, Bayesian, and MetaGA approaches
      showed considerably better likelihood scores (Delta&gt; 54) than those found under
      the MP and Treefinder approaches. Our trees provided strong support for most of
      the recognized Aegla species except for A. cholchol,A. jarai,A. parana,A.
      marginata, A. platensis, and A. franciscana, which may actually represent
      multiple species. Geographically, the Aegla group was divided into a basal
      western clade (21 species and subspecies) composed of two subclades with
      overlapping distributions, and a more recent central-eastern clade (43 species)
      composed of three subclades with fairly well-recognized distributions. This
      result supports the Pacific-Origin Hypothesis postulated for the group;
      alternative hypotheses of Atlantic or multiple origins were significantly
      rejected by our analyses. Finally, we combined our phylogenetic results with
      previous hypotheses of South American paleodrainages since the Jurassic to
      propose a biogeographical framework of the Aegla radiation.
FAU - Perez-Losada, Marcos
AU  - Perez-Losada M
AD  - Department of Integrative Biology, Brigham Young University, Provo, Utah
      84602-5181, USA. mp323@email.byu.edu
FAU - Bond-Buckup, Georgina
AU  - Bond-Buckup G
FAU - Jara, Carlos G
AU  - Jara CG
FAU - Crandall, Keith A
AU  - Crandall KA
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (RNA, Ribosomal, 28S)
SB  - IM
MH  - Animals
MH  - Anomura/*genetics
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Classification/methods
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - Fresh Water
MH  - Geography
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - RNA, Ribosomal, 28S/genetics
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
MH  - South America
MH  - Species Specificity
EDAT- 2004/11/17 09:00
MHDA- 2005/02/23 09:00
CRDT- 2004/11/17 09:00
PHST- 2004/11/17 09:00 [pubmed]
PHST- 2005/02/23 09:00 [medline]
PHST- 2004/11/17 09:00 [entrez]
AID - YA8DUHM1BGFYYCLC [pii]
AID - 10.1080/10635150490522331 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Oct;53(5):767-80. doi: 10.1080/10635150490522331.

PMID- 15545253
OWN - NLM
STAT- MEDLINE
DCOM- 20050222
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 5
DP  - 2004 Oct
TI  - Explosive radiations and the reliability of molecular clocks: island endemic
      radiations as a test case.
PG  - 758-66
AB  - The reliability of molecular clocks has been questioned for several key
      evolutionary radiations on the basis that the clock might run fast in explosive
      radiations. Molecular date estimates for the radiations of metazoan phyla (the
      Cambrian explosion) and modern orders of mammals and birds are in many cases
      twice as old as the palaeontological evidence would suggest. Could some aspect of
      explosive radiations speed the molecular clock, making molecular date estimates
      too old? Here we use 19 independent instances of recent explosive radiations of
      island endemic taxa as a model system for testing the proposed influence of rapid
      adaptive radiation on the rate of molecular evolution. These radiations are often
      characterized by many of the potential mechanisms for fast rates in explosive
      radiations--such as small population size, elevated speciation rate, rapid rate
      of morphological change, release from previous ecological constraints, and
      adaptation to new niches--and represent a wide variety of species, islands, and
      genes. However, we find no evidence of a consistent increase in rates in island
      taxa compared to their mainland relatives, and therefore find no support for the 
      hypothesis that the molecular clock runs fast in explosive radiations.
FAU - Bromham, Lindell
AU  - Bromham L
AD  - Centre for the Study of Evolution, School of Life Sciences, University of Sussex,
      Falmer, Brighton, UK. lindell@sussex.ac.uk
FAU - Woolfit, Megan
AU  - Woolfit M
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Biological/*genetics
MH  - Animals
MH  - *Evolution, Molecular
MH  - *Geography
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2004/11/17 09:00
MHDA- 2005/02/23 09:00
CRDT- 2004/11/17 09:00
PHST- 2004/11/17 09:00 [pubmed]
PHST- 2005/02/23 09:00 [medline]
PHST- 2004/11/17 09:00 [entrez]
AID - XJQN5F5VLR30QTU4 [pii]
AID - 10.1080/10635150490522278 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Oct;53(5):758-66. doi: 10.1080/10635150490522278.

PMID- 15545252
OWN - NLM
STAT- MEDLINE
DCOM- 20050222
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 5
DP  - 2004 Oct
TI  - Molecular phylogenetics of squamata: the position of snakes, amphisbaenians, and 
      dibamids, and the root of the squamate tree.
PG  - 735-57
AB  - Squamate reptiles (snakes, lizards, and amphisbaenians) serve as model systems
      for evolutionary studies of a variety of morphological and behavioral traits, and
      phylogeny is crucial to many generalizations derived from such studies.
      Specifically, the traditional dichotomy between Iguania (anoles, iguanas,
      chameleons, etc.) and Scleroglossa (skinks, geckos, snakes, etc.) has been
      correlated with major evolutionary shifts within Squamata. We present a molecular
      phylogenetic study of 69 squamate species using approximately 4600 (2876
      parsimony-informative) base pairs (bp) of DNA sequence data from the nuclear
      genes RAG-1(approximately 2750 bp) and c-mos(approximately 360 bp) and the
      mitochondrial ND2 region (approximately 1500 bp), sampling all major clades and
      most major subclades. Under our hypothesis, species previously placed in Iguania,
      Anguimorpha, and almost all recognized squamate families form strongly supported 
      monophyletic groups. However, species previously placed in Scleroglossa,
      Varanoidea, and several other higher taxa do not form monophyletic groups.
      Iguania, the traditional sister group of Scleroglossa, is actually highly nested 
      within Scleroglossa. This unconventional rooting does not seem to be due to
      long-branch attraction, base composition biases among taxa, or convergence caused
      by similar selective forces acting on nonsister taxa. Studies of functional
      tongue morphology and feeding mode have contrasted the similar states found in
      Sphenodon(the nearest outgroup to squamates) and Iguania with those of
      Scleroglossa, but our findings suggest that similar states in Sphenodonand
      Iguania result from homoplasy. Snakes, amphisbaenians, and dibamid lizards,
      limbless forms whose phylogenetic positions historically have been impossible to 
      place with confidence, are not grouped together and appear to have evolved this
      condition independently. Amphisbaenians are the sister group of lacertids, and
      dibamid lizards diverged early in squamate evolutionary history. Snakes are
      grouped with iguanians, lacertiforms, and anguimorphs, but are not nested within 
      anguimorphs.
FAU - Townsend, Ted
AU  - Townsend T
AD  - Department of Biology, Washington University, St. Louis, Missouri 63130, USA.
      townsend@mail.utexas.edu
FAU - Larson, Allan
AU  - Larson A
FAU - Louis, Edward
AU  - Louis E
FAU - Macey, J Robert
AU  - Macey JR
LA  - eng
SI  - GENBANK/AB008539
SI  - GENBANK/AF039474
SI  - GENBANK/AF039475
SI  - GENBANK/AF039476
SI  - GENBANK/AF039478
SI  - GENBANK/AF039479
SI  - GENBANK/AF039482
SI  - GENBANK/AF039483
SI  - GENBANK/AF039484
SI  - GENBANK/AF049864
SI  - GENBANK/AF049865
SI  - GENBANK/AF049866
SI  - GENBANK/AF085603
SI  - GENBANK/AF085604
SI  - GENBANK/AF085606
SI  - GENBANK/AF085607
SI  - GENBANK/AF085625
SI  - GENBANK/AF090846
SI  - GENBANK/AF090850
SI  - GENBANK/AF109208
SI  - GENBANK/AF114248
SI  - GENBANK/AF114249
SI  - GENBANK/AF128463
SI  - GENBANK/AF128476
SI  - GENBANK/AF128478
SI  - GENBANK/AF128482
SI  - GENBANK/AF137525
SI  - GENBANK/AF137527
SI  - GENBANK/AF137529
SI  - GENBANK/AF137531
SI  - GENBANK/AF143724
SI  - GENBANK/AF148703
SI  - GENBANK/AF148711
SI  - GENBANK/AF315386
SI  - GENBANK/AF315387
SI  - GENBANK/AF315388
SI  - GENBANK/AF315390
SI  - GENBANK/AF315391
SI  - GENBANK/AF315394
SI  - GENBANK/AF315396
SI  - GENBANK/AF315400
SI  - GENBANK/AF335080
SI  - GENBANK/AF375640
SI  - GENBANK/AF420861
SI  - GENBANK/AF420863
SI  - GENBANK/AF448734
SI  - GENBANK/AF448761
SI  - GENBANK/AF448780
SI  - GENBANK/AF471096
SI  - GENBANK/AF471133
SI  - GENBANK/AF471163
SI  - GENBANK/AF528747
SI  - GENBANK/AY297486
SI  - GENBANK/AY369016
SI  - GENBANK/AY369024
SI  - GENBANK/AY528719
SI  - GENBANK/AY662533
SI  - GENBANK/AY662537
SI  - GENBANK/AY662538
SI  - GENBANK/AY662539
SI  - GENBANK/AY662540
SI  - GENBANK/AY662541
SI  - GENBANK/AY662542
SI  - GENBANK/AY662543
SI  - GENBANK/AY662546
SI  - GENBANK/AY662547
SI  - GENBANK/AY662548
SI  - GENBANK/AY662549
SI  - GENBANK/AY662550
SI  - GENBANK/AY662551
SI  - GENBANK/AY662552
SI  - GENBANK/AY662553
SI  - GENBANK/AY662554
SI  - GENBANK/AY662555
SI  - GENBANK/AY662556
SI  - GENBANK/AY662557
SI  - GENBANK/AY662558
SI  - GENBANK/AY662559
SI  - GENBANK/AY662560
SI  - GENBANK/AY662561
SI  - GENBANK/AY662562
SI  - GENBANK/AY662563
SI  - GENBANK/AY662564
SI  - GENBANK/AY662565
SI  - GENBANK/AY662566
SI  - GENBANK/AY662567
SI  - GENBANK/AY662568
SI  - GENBANK/AY662569
SI  - GENBANK/AY662570
SI  - GENBANK/AY662571
SI  - GENBANK/AY662572
SI  - GENBANK/AY662573
SI  - GENBANK/AY662574
SI  - GENBANK/AY662576
SI  - GENBANK/AY662577
SI  - GENBANK/AY662578
SI  - GENBANK/AY662579
SI  - GENBANK/AY662580
SI  - GENBANK/AY662581
SI  - GENBANK/AY662582
SI  - GENBANK/AY662583
SI  - GENBANK/AY662584
SI  - GENBANK/AY662585
SI  - GENBANK/AY662586
SI  - GENBANK/AY662587
SI  - GENBANK/AY662588
SI  - GENBANK/AY662589
SI  - GENBANK/AY662590
SI  - GENBANK/AY662591
SI  - GENBANK/AY662592
SI  - GENBANK/AY662593
SI  - GENBANK/AY662594
SI  - GENBANK/AY662595
SI  - GENBANK/AY662596
SI  - GENBANK/AY662597
SI  - GENBANK/AY662598
SI  - GENBANK/AY662599
SI  - GENBANK/AY662600
SI  - GENBANK/AY662601
SI  - GENBANK/AY662602
SI  - GENBANK/AY662603
SI  - GENBANK/AY662604
SI  - GENBANK/AY662605
SI  - GENBANK/AY662606
SI  - GENBANK/AY662607
SI  - GENBANK/AY662608
SI  - GENBANK/AY662609
SI  - GENBANK/AY662610
SI  - GENBANK/AY662611
SI  - GENBANK/AY662612
SI  - GENBANK/AY662613
SI  - GENBANK/AY662614
SI  - GENBANK/AY662615
SI  - GENBANK/AY662616
SI  - GENBANK/AY662617
SI  - GENBANK/AY662618
SI  - GENBANK/AY662619
SI  - GENBANK/AY662620
SI  - GENBANK/AY662621
SI  - GENBANK/AY662622
SI  - GENBANK/AY662623
SI  - GENBANK/AY662624
SI  - GENBANK/AY662625
SI  - GENBANK/AY662626
SI  - GENBANK/AY662627
SI  - GENBANK/AY662628
SI  - GENBANK/AY662629
SI  - GENBANK/AY662630
SI  - GENBANK/AY662631
SI  - GENBANK/AY662632
SI  - GENBANK/AY662633
SI  - GENBANK/AY662634
SI  - GENBANK/AY662635
SI  - GENBANK/AY662636
SI  - GENBANK/AY662637
SI  - GENBANK/AY662638
SI  - GENBANK/AY662639
SI  - GENBANK/AY662640
SI  - GENBANK/AY662641
SI  - GENBANK/AY662642
SI  - GENBANK/AY662643
SI  - GENBANK/AY662644
SI  - GENBANK/AY662645
SI  - GENBANK/AY687907
SI  - GENBANK/M52392
SI  - GENBANK/M58530
SI  - GENBANK/R18178
SI  - GENBANK/R22245
SI  - GENBANK/R29312
SI  - GENBANK/R33417
SI  - GENBANK/R34528
SI  - GENBANK/R36502
SI  - GENBANK/R90194
SI  - GENBANK/R90195
SI  - GENBANK/U71322
SI  - GENBANK/U71325
SI  - GENBANK/U71326
SI  - GENBANK/U71328
SI  - GENBANK/U71330
SI  - GENBANK/U71331
SI  - GENBANK/U71333
SI  - GENBANK/U71334
SI  - GENBANK/U71335
SI  - GENBANK/U82679
SI  - GENBANK/U82680
SI  - GENBANK/U82682
SI  - GENBANK/U82683
SI  - GENBANK/U82684
SI  - GENBANK/U82685
SI  - GENBANK/U82687
SI  - GENBANK/U82689
SI  - GENBANK/U82690
SI  - GENBANK/U82691
SI  - GENBANK/U82692
SI  - GENBANK/X52392
SI  - GENBANK/Y13113
SI  - RefSeq/NC_000886
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Composition
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Computer Simulation
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - Genes, RAG-1/genetics
MH  - Genes, mos/genetics
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Reptiles/*classification/*genetics
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2004/11/17 09:00
MHDA- 2005/02/23 09:00
CRDT- 2004/11/17 09:00
PHST- 2004/11/17 09:00 [pubmed]
PHST- 2005/02/23 09:00 [medline]
PHST- 2004/11/17 09:00 [entrez]
AID - QMRHK814L5GRK9F9 [pii]
AID - 10.1080/10635150490522340 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Oct;53(5):735-57. doi: 10.1080/10635150490522340.

PMID- 15545251
OWN - NLM
STAT- MEDLINE
DCOM- 20050222
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 5
DP  - 2004 Oct
TI  - The origin and radiation of Macaronesian beetles breeding in Euphorbia: the
      relative importance of multiple data partitions and population sampling.
PG  - 711-34
AB  - Species-level phylogenies derived from many independent character sources and
      wide geographical sampling provide a powerful tool in assessing the importance of
      various factors associated with cladogenesis. In this study, we explore the
      relative importance of insular isolation and host plant switching in the
      diversification of a group of bark beetles (Curculionidae: Scolytinae) feeding
      and breeding in woody Euphor biaspurges. All species in the genus Aphanarthrumare
      each associated with only one species group of Euphorbia(succulents or one of
      three different arborescent groups), and the majority of species are endemic to
      one or several of the Macaronesian Islands. Hence, putative mechanisms of
      speciation could be assessed by identifying pairs of sister species in a
      phylogenetic analysis. We used DNA sequences from two nuclear and two
      mitochondrial genes, and morphological characters, to reconstruct the
      genealogical relationships among 92 individuals of 25 species and subspecies of
      Aphanarthrumand related genera. A stable tree topology was highly dependent on
      multiple character sources, but much less so on wide population sampling.
      However, multiple samples per species demonstrated one case of species paraphyly,
      as well as deep coalescence among three putative subspecies pairs. The
      phylogenetic analyses consistently placed the arborescent breeding and West
      African--Lanzarote-distributed species A. armatumin the most basal position in
      Aphanarthrum, rendering this genus paraphyletic with respect to Coleobothrus. Two
      major radiations followed, one predominantly African lineage of succulent feeding
      species, and one island radiation associated with arborescent host plants. Sister
      comparisons showed that most recent divergences occurred in allopatry on closely 
      related hosts, with subsequent expansions obscuring more ancient events. Only 6
      out of 24 cladogenetic events were associated with host switching, rendering
      geographical factors more important in recent diversification.
FAU - Jordal, Bjarte H
AU  - Jordal BH
AD  - School of Biological Sciences, University of East Anglia, Norwich, UK.
      bjarte.jordal@vm.utnu.no
FAU - Hewitt, Godfrey M
AU  - Hewitt GM
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Coleoptera/anatomy &amp; histology/*genetics
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - *Environment
MH  - Euphorbia
MH  - Geography
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2004/11/17 09:00
MHDA- 2005/02/23 09:00
CRDT- 2004/11/17 09:00
PHST- 2004/11/17 09:00 [pubmed]
PHST- 2005/02/23 09:00 [medline]
PHST- 2004/11/17 09:00 [entrez]
AID - R6XXTF9F0Q4JV256 [pii]
AID - 10.1080/10635150490468710 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Oct;53(5):711-34. doi: 10.1080/10635150490468710.

PMID- 15545250
OWN - NLM
STAT- MEDLINE
DCOM- 20050222
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 5
DP  - 2004 Oct
TI  - Multiple data sets, high homoplasy, and the phylogeny of softshell turtles
      (Testudines: Trionychidae).
PG  - 693-710
AB  - We present a phylogenetic hypothesis and novel, rank-free classification for all 
      extant species of softshell turtles (Testudines:Trionychidae). Our data set
      included DNA sequence data from two mitochondrial protein-coding genes and a
      approximately 1-kb nuclear intron for 23 of 26 recognized species, and 59
      previously published morphological characters for a complimentary set of 24
      species. The combined data set provided complete taxonomic coverage for this
      globally distributed clade of turtles, with incomplete data for a few taxa.
      Although our taxonomic sampling is complete, most of the modern taxa are
      representatives of old and very divergent lineages. Thus, due to biological
      realities, our sampling consists of one or a few representatives of several
      ancient lineages across a relatively deep phylogenetic tree. Our analyses of the 
      combined data set converge on a set of well-supported relationships, which is in 
      accord with many aspects of traditional softshell systematics including the
      monophyly of the Cyclanorbinae and Trionychinae. However, our results conflict
      with other aspects of current taxonomy and indicate that most of the currently
      recognized tribes are not monophyletic. We use this strong estimate of the
      phylogeny of softshell turtles for two purposes: (1) as the basis for a novel
      rank-free classification, and (2) to retrospectively examine strategies for
      analyzing highly homoplasious mtDNA data in deep phylogenetic problems where
      increased taxon sampling is not an option. Weeded and weighted parsimony, and
      model-based techniques, generally improved the phylogenetic performance of highly
      homoplasious mtDNA sequences, but no single strategy completely mitigated the
      problems of associated with these highly homoplasious data. Many deep nodes in
      the softshell turtle phylogeny were confidently recovered only after the addition
      of largely nonhomoplasious data from the nuclear intron.
FAU - Engstrom, Tag N
AU  - Engstrom TN
AD  - Center for Population Biology and Section of Evolution and Ecology, University of
      California, Davis, CA 95616, USA. tengstrom@csuchico.edu
FAU - Shaffer, H Bradley
AU  - Shaffer HB
FAU - McCord, William P
AU  - McCord WP
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Composition
MH  - Base Sequence
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
MH  - Turtles/anatomy &amp; histology/*classification/*genetics
EDAT- 2004/11/17 09:00
MHDA- 2005/02/23 09:00
CRDT- 2004/11/17 09:00
PHST- 2004/11/17 09:00 [pubmed]
PHST- 2005/02/23 09:00 [medline]
PHST- 2004/11/17 09:00 [entrez]
AID - WAQW58BTULW9V3BG [pii]
AID - 10.1080/10635150490503053 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Oct;53(5):693-710. doi: 10.1080/10635150490503053.

PMID- 15545249
OWN - NLM
STAT- MEDLINE
DCOM- 20050222
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 5
DP  - 2004 Oct
TI  - Column sorting: rapid calculation of the phylogenetic likelihood function.
PG  - 685-92
AB  - Likelihood applications have become a central approach for molecular evolutionary
      analyses since the first computationally tractable treatment two decades ago.
      Although Felsenstein's original pruning algorithm makes likelihood calculations
      feasible, it is usually possible to take advantage of repetitive structure
      present in the data to arrive at even greater computational reductions. In
      particular, alignment columns with certain similarities have components of the
      likelihood calculation that are identical and need not be recomputed if columns
      are evaluated in an optimal order. We develop an algorithm for exploiting this
      speed improvement via an application of graph theory. The reductions provided by 
      the method depend on both the tree and the data, but typical savings range
      between 15%and 50%. Real-data examples with time reductions of 80%have been
      identified. The overhead costs associated with implementing the algorithm are
      minimal, and they are recovered in all but the smallest data sets. The
      modifications will provide faster likelihood algorithms, which will allow
      likelihood methods to be applied to larger sets of taxa and to include more
      thorough searches of the tree topology space.
FAU - Kosakovsky Pond, Sergei L
AU  - Kosakovsky Pond SL
AD  - Department of Mathematics, University of Arizona, Tucson, Arizona 85271, USA.
FAU - Muse, Spencer V
AU  - Muse SV
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Base Sequence/genetics
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2004/11/17 09:00
MHDA- 2005/02/23 09:00
CRDT- 2004/11/17 09:00
PHST- 2004/11/17 09:00 [pubmed]
PHST- 2005/02/23 09:00 [medline]
PHST- 2004/11/17 09:00 [entrez]
AID - N5QBABBH87BRYJFV [pii]
AID - 10.1080/10635150490522269 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Oct;53(5):685-92. doi: 10.1080/10635150490522269.

PMID- 15545248
OWN - NLM
STAT- MEDLINE
DCOM- 20050222
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 5
DP  - 2004 Oct
TI  - Bayesian estimation of ancestral character states on phylogenies.
PG  - 673-84
AB  - Biologists frequently attempt to infer the character states at ancestral nodes of
      a phylogeny from the distribution of traits observed in contemporary organisms.
      Because phylogenies are normally inferences from data, it is desirable to account
      for the uncertainty in estimates of the tree and its branch lengths when making
      inferences about ancestral states or other comparative parameters. Here we
      present a general Bayesian approach for testing comparative hypotheses across
      statistically justified samples of phylogenies, focusing on the specific issue of
      reconstructing ancestral states. The method uses Markov chain Monte Carlo
      techniques for sampling phylogenetic trees and for investigating the parameters
      of a statistical model of trait evolution. We describe how to combine information
      about the uncertainty of the phylogeny with uncertainty in the estimate of the
      ancestral state. Our approach does not constrain the sample of trees only to
      those that contain the ancestral node or nodes of interest, and we show how to
      reconstruct ancestral states of uncertain nodes using a
      most-recent-common-ancestor approach. We illustrate the methods with data on
      ribonuclease evolution in the Artiodactyla. Software implementing the methods
      (BayesMultiState) is available from the authors.
FAU - Pagel, Mark
AU  - Pagel M
AD  - School of Animal and Microbial Sciences, University of Reading, Whiteknights,
      Reading, England. m.pagel@rdg.ac.uk
FAU - Meade, Andrew
AU  - Meade A
FAU - Barker, Daniel
AU  - Barker D
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 3.1.- (Ribonucleases)
SB  - IM
MH  - Animals
MH  - Artiodactyla/genetics
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - Markov Chains
MH  - *Models, Biological
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Ribonucleases/genetics
EDAT- 2004/11/17 09:00
MHDA- 2005/02/23 09:00
CRDT- 2004/11/17 09:00
PHST- 2004/11/17 09:00 [pubmed]
PHST- 2005/02/23 09:00 [medline]
PHST- 2004/11/17 09:00 [entrez]
AID - VCXNPJMTK6788EJF [pii]
AID - 10.1080/10635150490522232 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Oct;53(5):673-84. doi: 10.1080/10635150490522232.

PMID- 15503677
OWN - NLM
STAT- MEDLINE
DCOM- 20041221
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 3
DP  - 2004 Jun
TI  - Aligned 18S and insect phylogeny.
PG  - 506-14
AB  - The nuclear small subunit rRNA (18S) has played a dominant role in the estimation
      of relationships among insect orders from molecular data. In previous studies,
      18S sequences have been aligned by unadjusted automated approaches (computer
      alignments that are not manually readjusted), most recently with direct
      optimization (simultaneous alignment and tree building using a program called
      "POY"). Parsimony has been the principal optimality criterion. Given the problems
      associated with the alignment of rRNA, and the recent availability of the doublet
      model for the analysis of covarying sites using Bayesian MCMC analysis, a
      different approach is called for in the analysis of these data. In this paper,
      nucleotide sequence data from the 18S small subunit rRNA gene of insects are
      aligned manually with reference to secondary structure, and analyzed under
      Bayesian phylogenetic methods with both GTR+I+G and doublet models in MrBayes. A 
      credible phylogeny of Insecta is recovered that is independent of the
      morphological data and (unlike many other analyses of 18S in insects) not
      contradictory to traditional ideas of insect ordinal relationships based on
      morphology. Hexapoda, including Collembola, are monophyletic. Paraneoptera are
      the sister taxon to a monophyletic Holometabola but weakly supported.
      Ephemeroptera are supported as the sister taxon of Neoptera, and this result is
      interpreted with respect to the evolution of direct sperm transfer and the
      evolution of flight. Many other relationships are well-supported but several taxa
      remain problematic, e.g., there is virtually no support for relationships among
      orthopteroid orders. A website is made available that provides aligned 18S data
      in formats that include structural symbols and Nexus formats.
FAU - Kjer, Karl M
AU  - Kjer KM
AD  - Department of Ecology Evolution and Natural Resources, 14 College Farm Road, Cook
      College, Rutgers University, New Brunswick, NJ 08901, USA. kjer@aesop.rutgers.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal, 18S)
SB  - IM
MH  - Animals
MH  - Base Pairing
MH  - Bayes Theorem
MH  - Insecta/classification/*genetics
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - RNA, Ribosomal, 18S/*genetics
MH  - *Sequence Alignment
EDAT- 2004/10/27 09:00
MHDA- 2004/12/22 09:00
CRDT- 2004/10/27 09:00
PHST- 2004/10/27 09:00 [pubmed]
PHST- 2004/12/22 09:00 [medline]
PHST- 2004/10/27 09:00 [entrez]
AID - 10.1080/10635150490445922 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Jun;53(3):506-14. doi: 10.1080/10635150490445922.

PMID- 15503676
OWN - NLM
STAT- MEDLINE
DCOM- 20041221
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 3
DP  - 2004 Jun
TI  - Molecular phylogenetic dating of asterid flowering plants shows early Cretaceous 
      diversification.
PG  - 496-505
AB  - We present a phylogenetic dating of asterids, based on a 111-taxon tree
      representing all major groups and orders and 83 of the 102 families of asterids, 
      with an underlying data set comprising six chloroplast DNA markers totaling 9914 
      positions. Phylogenetic dating was done with semiparametric rate smoothing by
      penalized likelihood. Confidence intervals were calculated by bootstrapping. Six 
      reference fossils were used for calibration. To explore the effects of various
      sources of error, we repeated the analyses with alternative dating methods
      (nonparametric rate smoothing and the Langley-Fitch clock-based method),
      alternative tree topologies, reduced taxon sampling (22 of the 111 taxa deleted),
      partitioning the data into three genes and three noncoding regions, and
      calibrating with single reference fossils. The analyses with alternative
      topologies, reduced taxon sampling, and coding versus noncoding sequences all
      yielded small or in some cases no deviations. The choice of method influenced the
      age estimates of a few nodes considerably. Calibration with reference fossils is 
      a critical issue, and use of single reference fossils yielded different results
      depending on the fossil. The bootstrap confidence intervals were generally small.
      Our results show that asterids and their major subgroups euasterids, campanulids,
      and lamiids diversified during the Early Cretaceous. Cornales, Ericales, and
      Aquifoliales also have crown node ages from the Early Cretaceous. Dipsacales and 
      Solanales are from the Mid-Cretaceous, the other orders of core campanulids and
      core lamiids from the Late Cretaceous. The considerable diversity exhibited by
      asterids almost from their first appearance in the fossil record also supports an
      origin and first phase of diversification in the Early Cretaceous.
FAU - Bremer, Kare
AU  - Bremer K
AD  - Department of Systematic Botany, Evolutionary Biology Centre, Uppsala University,
      Norbyvagen 18D, SE-752 36 Uppsala, Sweden.
FAU - Friis, Else Marie
AU  - Friis EM
FAU - Bremer, Birgitta
AU  - Bremer B
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
SB  - IM
MH  - DNA, Chloroplast/genetics
MH  - *Evolution, Molecular
MH  - Fossils
MH  - Magnoliopsida/*genetics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Research Design
MH  - Time Factors
EDAT- 2004/10/27 09:00
MHDA- 2004/12/22 09:00
CRDT- 2004/10/27 09:00
PHST- 2004/10/27 09:00 [pubmed]
PHST- 2004/12/22 09:00 [medline]
PHST- 2004/10/27 09:00 [entrez]
AID - 10.1080/10635150490445913 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Jun;53(3):496-505. doi: 10.1080/10635150490445913.

PMID- 15503675
OWN - NLM
STAT- MEDLINE
DCOM- 20041221
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 3
DP  - 2004 Jun
TI  - Modeling compositional heterogeneity.
PG  - 485-95
AB  - Compositional heterogeneity among lineages can compromise phylogenetic analyses, 
      because models in common use assume compositionally homogeneous data. Models that
      can accommodate compositional heterogeneity with few extra parameters are
      described here, and used in two examples where the true tree is known with
      confidence. It is shown using likelihood ratio tests that adequate modeling of
      compositional heterogeneity can be achieved with few composition parameters, that
      the data may not need to be modelled with separate composition parameters for
      each branch in the tree. Tree searching and placement of composition vectors on
      the tree are done in a Bayesian framework using Markov chain Monte Carlo (MCMC)
      methods. Assessment of fit of the model to the data is made in both maximum
      likelihood (ML) and Bayesian frameworks. In an ML framework, overall model fit is
      assessed using the Goldman-Cox test, and the fit of the composition implied by a 
      (possibly heterogeneous) model to the composition of the data is assessed using a
      novel tree-and model-based composition fit test. In a Bayesian framework, overall
      model fit and composition fit are assessed using posterior predictive simulation.
      It is shown that when composition is not accommodated, then the model does not
      fit, and incorrect trees are found; but when composition is accommodated, the
      model then fits, and the known correct phylogenies are obtained.
FAU - Foster, Peter G
AU  - Foster PG
AD  - Department of Zoology, The Natural History Museum, Cromwell Road, London SW7 5BD,
      United Kingdom. p.foster@nhm.ac.uk
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Base Composition
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Likelihood Functions
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
EDAT- 2004/10/27 09:00
MHDA- 2004/12/22 09:00
CRDT- 2004/10/27 09:00
PHST- 2004/10/27 09:00 [pubmed]
PHST- 2004/12/22 09:00 [medline]
PHST- 2004/10/27 09:00 [entrez]
AID - 10.1080/10635150490445779 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Jun;53(3):485-95. doi: 10.1080/10635150490445779.

PMID- 15503674
OWN - NLM
STAT- MEDLINE
DCOM- 20041221
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 3
DP  - 2004 Jun
TI  - Cytogenetics and cladistics.
PG  - 470-84
AB  - Chromosomal data have been underutilized in phylogenetic investigations despite
      the obvious potential that cytogenetic studies have to reveal both structural and
      functional homologies among taxa. In large part this is associated with
      difficulties in scoring conventional and molecular cytogenetic information for
      phylogenetic analysis. The manner in which chromosomal data have been used by
      most authors in the past was often conceptionally flawed in terms of the methods 
      and principles underpinning modern cladistics. We present herein a review of the 
      different methods employed, examine their relative strengths, and then outline a 
      simple approach that considers the chromosomal change as the character, and its
      presence or absence the character state. We test this using one simulated and
      several empirical data sets. Features that are unique to cytogenetic
      investigations, including B-chromosomes, heterochromatic additions/deletions, and
      the location and number of nucleolar organizer regions (NORs), as well as the
      weighting of chromosomal characters, are critically discussed with regard to
      their suitability for phylogenetic reconstruction. We conclude that each of these
      classes of data have inherent problems that limit their usefulness in
      phylogenetic analyses and in most of these instances, inclusion should be subject
      to rigorous appraisal that addresses the criterion of unequivocal homology.
FAU - Dobigny, Gauthier
AU  - Dobigny G
AD  - Museum National d'Histoire Naturelle, Laboratoire Origine, Structure et Evolution
      de la Biodiversite, 55, rue Buffon, F75005 Paris, France. dobigny@sun.ac.za
FAU - Ducroz, Jean-Francois
AU  - Ducroz JF
FAU - Robinson, Terence J
AU  - Robinson TJ
FAU - Volobouev, Vitaly
AU  - Volobouev V
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Heterochromatin)
SB  - IM
MH  - Chromosomes/*genetics
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Cytogenetics/*methods
MH  - Heterochromatin/genetics
MH  - Nucleolus Organizer Region/genetics
MH  - *Phylogeny
MH  - Species Specificity
EDAT- 2004/10/27 09:00
MHDA- 2004/12/22 09:00
CRDT- 2004/10/27 09:00
PHST- 2004/10/27 09:00 [pubmed]
PHST- 2004/12/22 09:00 [medline]
PHST- 2004/10/27 09:00 [entrez]
AID - 10.1080/10635150490445698 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Jun;53(3):470-84. doi: 10.1080/10635150490445698.

PMID- 15503673
OWN - NLM
STAT- MEDLINE
DCOM- 20041221
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 3
DP  - 2004 Jun
TI  - Data partitions and complex models in Bayesian analysis: the phylogeny of
      Gymnophthalmid lizards.
PG  - 448-69
AB  - Phylogenetic studies incorporating multiple loci, and multiple genomes, are
      becoming increasingly common. Coincident with this trend in genetic sampling,
      model-based likelihood techniques including Bayesian phylogenetic methods
      continue to gain popularity. Few studies, however, have examined model fit and
      sensitivity to such potentially heterogeneous data partitions within combined
      data analyses using empirical data. Here we investigate the relative model fit
      and sensitivity of Bayesian phylogenetic methods when alternative site-specific
      partitions of among-site rate variation (with and without autocorrelated rates)
      are considered. Our primary goal in choosing a best-fit model was to employ the
      simplest model that was a good fit to the data while optimizing topology and/or
      Bayesian posterior probabilities. Thus, we were not interested in complex models 
      that did not practically affect our interpretation of the topology under study.
      We applied these alternative models to a four-gene data set including one
      protein-coding nuclear gene (c-mos), one protein-coding mitochondrial gene (ND4),
      and two mitochondrial rRNA genes (12S and 16S) for the diverse yet poorly known
      lizard family Gymnophthalmidae. Our results suggest that the best-fit model
      partitioned among-site rate variation separately among the c-mos, ND4, and 12S + 
      16S gene regions. We found this model yielded identical topologies to those from 
      analyses based on the GTR+I+G model, but significantly changed posterior
      probability estimates of clade support. This partitioned model also produced more
      precise (less variable) estimates of posterior probabilities across generations
      of long Bayesian runs, compared to runs employing a GTR+I+G model estimated for
      the combined data. We use this three-way gamma partitioning in Bayesian analyses 
      to reconstruct a robust phylogenetic hypothesis for the relationships of genera
      within the lizard family Gymnophthalmidae. We then reevaluate the higher-level
      taxonomic arrangement of the Gymnophthalmidae. Based on our findings, we discuss 
      the utility of nontraditional parameters for modeling among-site rate variation
      and the implications and future directions for complex model building and
      testing.
FAU - Castoe, Todd A
AU  - Castoe TA
AD  - Department of Biology, University of Central Florida, 4000 Central Florida
      Boulevard, Orlando, FL 32816-2368, USA. tcastoe@mail.ucf.edu
FAU - Doan, Tiffany M
AU  - Doan TM
FAU - Parkinson, Christopher L
AU  - Parkinson CL
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (RNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - Genes, mos/genetics
MH  - Lizards/classification/*genetics
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - RNA, Ribosomal/genetics
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
MH  - Sequence Homology
EDAT- 2004/10/27 09:00
MHDA- 2004/12/22 09:00
CRDT- 2004/10/27 09:00
PHST- 2004/10/27 09:00 [pubmed]
PHST- 2004/12/22 09:00 [medline]
PHST- 2004/10/27 09:00 [entrez]
AID - 10.1080/10635150490445797 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Jun;53(3):448-69. doi: 10.1080/10635150490445797.

PMID- 15503672
OWN - NLM
STAT- MEDLINE
DCOM- 20041221
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 3
DP  - 2004 Jun
TI  - A molecular supermatrix of the rabbits and hares (Leporidae) allows for the
      identification of five intercontinental exchanges during the Miocene.
PG  - 433-47
AB  - The hares and rabbits belonging to the family Leporidae have a nearly worldwide
      distribution and approximately 72% of the genera have geographically restricted
      distributions. Despite several attempts using morphological, cytogenetic, and
      mitochondrial DNA evidence, a robust phylogeny for the Leporidae remains elusive.
      To provide phylogenetic resolution within this group, a molecular supermatrix was
      constructed for 27 taxa representing all 11 leporid genera. Five nuclear (SPTBN1,
      PRKCI, THY, TG, and MGF) and two mitochondrial (cytochrome b and 12S rRNA) gene
      fragments were analyzed singly and in combination using parsimony, maximum
      likelihood, and Bayesian inference. The analysis of each gene fragment separately
      as well as the combined mtDNA data almost invariably failed to provide strong
      statistical support for intergeneric relationships. In contrast, the combined
      nuclear DNA topology based on 3601 characters greatly increased phylogenetic
      resolution among leporid genera, as was evidenced by the number of topologies in 
      the 95% confidence interval and the number of significantly supported nodes. The 
      final molecular supermatrix contained 5483 genetic characters and analysis
      thereof consistently recovered the same topology across a range of six
      arbitrarily chosen model specifications. Twelve unique insertion-deletions were
      scored and all could be mapped to the tree to provide additional support without 
      introducing any homoplasy. Dispersal-vicariance analyses suggest that the most
      parsimonious solution explaining the current geographic distribution of the group
      involves an Asian or North American origin for the Leporids followed by at least 
      nine dispersals and five vicariance events. Of these dispersals, at least three
      intercontinental exchanges occurred between North America and Asia via the Bering
      Strait and an additional three independent dispersals into Africa could be
      identified. A relaxed Bayesian molecular clock applied to the seven loci used in 
      this study indicated that most of the intercontinental exchanges occurred between
      14 and 9 million years ago and this period is broadly coincidental with the onset
      of major Antarctic expansions causing land bridges to be exposed.
FAU - Matthee, Conrad A
AU  - Matthee CA
AD  - Evolutionary Genomics Group, Department of Zoology, University of Stellenbosch,
      Stellenbosch, 7602, South Africa. cam@sun.ac.za
FAU - van Vuuren, Bettine Jansen
AU  - van Vuuren BJ
FAU - Bell, Diana
AU  - Bell D
FAU - Robinson, Terence J
AU  - Robinson TJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Nuclear Proteins)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - *Demography
MH  - *Evolution, Molecular
MH  - Geography
MH  - Geological Phenomena
MH  - Geology
MH  - Lagomorpha/*genetics
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Nuclear Proteins/genetics
MH  - *Phylogeny
MH  - Population Dynamics
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
EDAT- 2004/10/27 09:00
MHDA- 2004/12/22 09:00
CRDT- 2004/10/27 09:00
PHST- 2004/10/27 09:00 [pubmed]
PHST- 2004/12/22 09:00 [medline]
PHST- 2004/10/27 09:00 [entrez]
AID - 10.1080/10635150490445715 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Jun;53(3):433-47. doi: 10.1080/10635150490445715.

PMID- 15503671
OWN - NLM
STAT- MEDLINE
DCOM- 20041221
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 3
DP  - 2004 Jun
TI  - Biogeography of the Pistia clade (Araceae): based on chloroplast and
      mitochondrial DNA sequences and Bayesian divergence time inference.
PG  - 422-32
AB  - Pistia stratiotes (water lettuce) and Lemna (duckweeds) are the only
      free-floating aquatic Araceae. The geographic origin and phylogenetic placement
      of these unrelated aroids present long-standing problems because of their highly 
      modified reproductive structures and wide geographical distributions. We sampled 
      chloroplast (trnL-trnF and rpl20-rps12 spacers, trnL intron) and mitochondrial
      sequences (nad1 b/c intron) for all genera implicated as close relatives of
      Pistia by morphological, restriction site, and sequencing data, and present a
      hypothesis about its geographic origin based on the consensus of trees obtained
      from the combined data, using Bayesian, maximum likelihood, parsimony, and
      distance analyses. Of the 14 genera closest to Pistia, only Alocasia, Arisaema,
      and Typhonium are species-rich, and the latter two were studied previously,
      facilitating the choice of representatives that span the roots of these genera.
      Results indicate that Pistia and the Seychelles endemic Protarum sechellarum are 
      the basalmost branches in a grade comprising the tribes Colocasieae (Ariopsis,
      Steudnera, Remusatia, Alocasia, Colocasia), Arisaemateae (Arisaema, Pinellia),
      and Areae (Arum, Biarum, Dracunculus, Eminium, Helicodiceros, Theriophonum,
      Typhonium). Unexpectedly, all Areae genera are embedded in Typhonium, which
      throws new light on the geographic history of Areae. A Bayesian analysis of
      divergence times that explores the effects of multiple fossil and geological
      calibration points indicates that the Pistia lineage is 90 to 76 million years
      (my) old. The oldest fossils of the Pistia clade, though not Pistia itself, are
      45-my-old leaves from Germany; the closest outgroup, Peltandreae (comprising a
      few species in Florida, the Mediterranean, and Madagascar), is known from
      60-my-old leaves from Europe, Kazakhstan, North Dakota, and Tennessee. Based on
      the geographic ranges of close relatives, Pistia likely originated in the Tethys 
      region, with Protarum then surviving on the Seychelles, which became isolated
      from Madagascar and India in the Late Cretaceous (85 my ago). Pistia and Protarum
      provide striking examples of ancient lineages that appear to have survived in
      unique or isolated habitats.
FAU - Renner, Susanne S
AU  - Renner SS
AD  - Department of Biology, University of Missouri-St. Louis, 8001 Natural Bridge
      Road, St. Louis, MO 63121, USA. renner@umsl.edu
FAU - Zhang, Li-Bing
AU  - Zhang LB
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Araceae/classification/*genetics
MH  - Base Sequence
MH  - DNA, Chloroplast/genetics
MH  - DNA, Mitochondrial/genetics
MH  - *Demography
MH  - *Evolution, Molecular
MH  - Geography
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2004/10/27 09:00
MHDA- 2004/12/22 09:00
CRDT- 2004/10/27 09:00
PHST- 2004/10/27 09:00 [pubmed]
PHST- 2004/12/22 09:00 [medline]
PHST- 2004/10/27 09:00 [entrez]
AID - 10.1080/10635150490445904 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Jun;53(3):422-32. doi: 10.1080/10635150490445904.

PMID- 15503670
OWN - NLM
STAT- MEDLINE
DCOM- 20041221
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 3
DP  - 2004 Jun
TI  - Molecular phylogenetics and evolution of maternal care in Membracine treehoppers.
PG  - 400-21
AB  - The treehopper subfamily Membracinae (Insecta: Hemiptera: Membracidae) comprises 
      the majority of genera and species diversity in the New World tropics. These
      treehoppers exhibit a wide range of social behaviors, making them an excellent
      group for studying patterns of social evolution in insects. However, to date the 
      tribal and generic relationships have remained unclear. We reconstructed the
      phylogeny of the Membracinae using a combined mitochondrial (COI, COII, tRNA-Leu,
      and 12S) and nuclear (Wg) gene data set. A total of 2608 aligned nucleotide sites
      were obtained for 112 species, representing 25 of 38 currently recognized genera 
      and all four tribes. A strict consensus of five equally parsimonious trees
      recovered the subfamily and three of its four tribes. The majority rule consensus
      tree derived from the Bayesian analyses based on the GTR+I+G and mixed-models
      recovered many clades shared with the parsimony trees and is identical to the
      single best tree inferred from maximum likelihood analysis, aside from the
      rearrangement of one node. A comparison of mitochondrial and nuclear genes
      indicated that Wg provided higher consistency index (CI), data decisiveness (DD),
      partitioned Bremer support (PBS) than any of the mitochondrial genes analyzed.
      The combined mitochondrial and nuclear DNA provide strong support for the
      monophyly of the subfamily and three of its four tribes (Aconophorini,
      Hoplophorionini, and Hypsoprorini). Membracini is paraphyletic with respect to
      Hoplophorionini and contains two lineages, the Membracini sensu strictu and the
      newly resurrected tribe Bolbonotini. Our analyses show that there is a strong
      phylogenetic component to the evolution of maternal care. Given the widespread
      occurrence of maternal care within the subfamily, this trait is estimated to have
      &lt; or = 3 origins, two reversals, and one loss. Our results suggest that the
      evolution of maternal care in insects may not be as evolutionarily labile as
      previously thought.
FAU - Lin, Chung-Ping
AU  - Lin CP
AD  - Department of Entomology, Comstock Hall, Cornell University, Ithaca, NY
      14853-0901, USA. linchung@missouri.edu
FAU - Danforth, Bryan N
AU  - Danforth BN
FAU - Wood, Thomas K
AU  - Wood TK
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Nuclear Proteins)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - Hemiptera/*genetics/physiology
MH  - Likelihood Functions
MH  - Maternal Behavior/*physiology
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Nuclear Proteins/genetics
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2004/10/27 09:00
MHDA- 2004/12/22 09:00
CRDT- 2004/10/27 09:00
PHST- 2004/10/27 09:00 [pubmed]
PHST- 2004/12/22 09:00 [medline]
PHST- 2004/10/27 09:00 [entrez]
AID - 10.1080/10635150490445869 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Jun;53(3):400-21. doi: 10.1080/10635150490445869.

PMID- 15503669
OWN - NLM
STAT- MEDLINE
DCOM- 20041221
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 3
DP  - 2004 Jun
TI  - Testing the phylogenetic position of a parasitic plant (Cuscuta, Convolvulaceae, 
      asteridae): Bayesian inference and the parametric bootstrap on data drawn from
      three genomes.
PG  - 384-99
AB  - Previous findings on structural rearrangements in the chloroplast genome of
      Cuscuta (dodder), the only parasitic genus in the morning-glory family,
      Convolvulaceae, were attributed to its parasitic life style, but without proper
      comparison to related nonparasitic members of the family. Before molecular
      evolutionary questions regarding genome evolution can be answered, the
      phylogenetic problems within the family need to be resolved. However, the
      phylogenetic position of parasitic angiosperms and their precise relationship to 
      nonparasitic relatives are difficult to infer. Problems are encountered with both
      morphological and molecular evidence. Molecular data have been used in numerous
      studies to elucidate relationships of parasitic taxa, despite accelerated rates
      of sequence evolution. To address the question of the position of the genus
      Cuscuta within Convolvulaceae, we generated a new molecular data set consisting
      of mitochondrial (atpA) and nuclear (RPB2) genes, and analyzed these data
      together with an existing chloroplast data matrix (rbcL, atpB, trnL-F, and
      psbE-J), to which an additional chloroplast gene (rpl2) was added. This data set 
      was analyzed with an array of phylogenetic methods, including Bayesian analysis, 
      maximum likelihood, and maximum parsimony. Further exploration of data was done
      by using methods of phylogeny hypothesis testing. At least two nonparasitic
      lineages are shown to diverge within the Convolvulaceae before Cuscuta. However, 
      the exact sister group of Cuscuta could not be ascertained, even though many
      alternatives were rejected with confidence. Caution is therefore warranted when
      interpreting the causes of molecular evolution in Cuscuta. Detailed comparisons
      with nonparasitic Convolvulaceae are necessary before firm conclusions can be
      reached regarding the effects of the parasitic mode of life on patterns of
      molecular evolution in Cuscuta.
FAU - Stefanovic, Sasa
AU  - Stefanovic S
AD  - Department of Botany, University of Washington, Box 355325, Seattle, Washington
      98195-5325, USA. sstefano@bio.indiana.edu
FAU - Olmstead, Richard G
AU  - Olmstead RG
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Chloroplast)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Nuclear Proteins)
SB  - IM
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Convolvulaceae/*genetics
MH  - DNA Primers
MH  - DNA, Chloroplast/genetics
MH  - DNA, Mitochondrial/genetics
MH  - Evolution, Molecular
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Nuclear Proteins/genetics
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
EDAT- 2004/10/27 09:00
MHDA- 2004/12/22 09:00
CRDT- 2004/10/27 09:00
PHST- 2004/10/27 09:00 [pubmed]
PHST- 2004/12/22 09:00 [medline]
PHST- 2004/10/27 09:00 [entrez]
AID - 10.1080/10635150490445896 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Jun;53(3):384-99. doi: 10.1080/10635150490445896.

PMID- 15503668
OWN - NLM
STAT- MEDLINE
DCOM- 20041221
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 3
DP  - 2004 Jun
TI  - Phylogeny of the Nymphalidae (Lepidoptera).
PG  - 363-83
AB  - A generic-level phylogeny for the butterfly family Nymphalidae was produced by
      cladistic analysis of 234 characters from all life stages. The 95 species in the 
      matrix (selected from the 213 studied) represent all important recognized
      lineages within this family. The analysis showed the taxa grouping into six main 
      lineages. The basal branch is the Libytheinae, with the Danainae and Ithomiinae
      on the next branch. The remaining lineages are grouped into two main branches:
      the Heliconiinae-Nymphalinae, primarily flower-visitors (but including the
      fruit-attracted Coeini); and the Limenitidinae (sensu strictu), Biblidinae, and
      the satyroid lineage (Apaturinae, Charaxinae, Biinae, Calinaginae, Morphinae,
      Brassolinae, and Satyrinae), primarily fruit-attracted. Data partitions showed
      that the two data sets (immatures and adults) are very different, and a
      partitioned Bremer support analysis showed that the adult characters are the main
      source of conflict in the nodes of the combined analysis tree. This phylogeny
      includes the widest taxon coverage of any morphological study on Nymphalid
      butterflies to date, and supports the monophyly and relationships of most
      presently recognized subgroups, providing strong evidence for the presently
      accepted phylogenetic scheme.
FAU - Freitas, Andre Victor Lucci
AU  - Freitas AV
AD  - Museu de Historia Natural and Departamento de Zoologia, Instituto de Biologia,
      Universidade Estadual de Campinas, CP 6109, CEP 13083-970, Campinas, SP, Brazil.
FAU - Brown, Keith S Jr
AU  - Brown KS Jr
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Brazil
MH  - Butterflies/*anatomy &amp; histology/classification
MH  - Female
MH  - Genitalia/anatomy &amp; histology
MH  - Larva/anatomy &amp; histology
MH  - Male
MH  - Ovum/cytology
MH  - *Phenotype
MH  - *Phylogeny
MH  - Pupa/anatomy &amp; histology
EDAT- 2004/10/27 09:00
MHDA- 2004/12/22 09:00
CRDT- 2004/10/27 09:00
PHST- 2004/10/27 09:00 [pubmed]
PHST- 2004/12/22 09:00 [medline]
PHST- 2004/10/27 09:00 [entrez]
AID - 10.1080/10635150490445670 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Jun;53(3):363-83. doi: 10.1080/10635150490445670.

PMID- 15371253
OWN - NLM
STAT- MEDLINE
DCOM- 20041201
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 4
DP  - 2004 Aug
TI  - The role of morphological data in phylogeny reconstruction.
PG  - 653-61
FAU - Wiens, John
AU  - Wiens J
AD  - Department of Ecology and Evolution, Stony Brook University, Stony Brook, NY
      11794-5245, USA. wiensj@life.bio.sunysb.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - DNA/genetics
MH  - Fossils
MH  - Genomics
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2004/09/17 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/09/17 05:00
PHST- 2004/09/17 05:00 [pubmed]
PHST- 2004/12/16 09:00 [medline]
PHST- 2004/09/17 05:00 [entrez]
AID - 10.1080/10635150490472959 [doi]
AID - HEEEMFJWL68PRVM3 [pii]
PST - ppublish
SO  - Syst Biol. 2004 Aug;53(4):653-61. doi: 10.1080/10635150490472959.

PMID- 15371252
OWN - NLM
STAT- MEDLINE
DCOM- 20041201
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 4
DP  - 2004 Aug
TI  - The contest between parsimony and likelihood.
PG  - 644-53
FAU - Sober, Elliott
AU  - Sober E
AD  - Philosophy Department, Stanford University, Stanford, CA 94305, USA.
      esober@stanford.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Evolution, Molecular
MH  - Female
MH  - Genotype
MH  - Humans
MH  - Likelihood Functions
MH  - Male
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2004/09/17 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/09/17 05:00
PHST- 2004/09/17 05:00 [pubmed]
PHST- 2004/12/16 09:00 [medline]
PHST- 2004/09/17 05:00 [entrez]
AID - 10.1080/10635150490468657 [doi]
AID - B0YYJGD0N52XH5E2 [pii]
PST - ppublish
SO  - Syst Biol. 2004 Aug;53(4):644-53. doi: 10.1080/10635150490468657.

PMID- 15371251
OWN - NLM
STAT- MEDLINE
DCOM- 20041201
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 4
DP  - 2004 Aug
TI  - The biasing effect of compositional heterogeneity on phylogenetic estimates may
      be underestimated.
PG  - 638-43
FAU - Jermiin, Lars
AU  - Jermiin L
AD  - School of Biological Sciences, University of Sydney, NSW 2006, Australia.
      lsj@bio.usyd.edu.
FAU - Ho, Simon Y
AU  - Ho SY
FAU - Ababneh, Faisal
AU  - Ababneh F
FAU - Robinson, John
AU  - Robinson J
FAU - Larkum, Anthony W
AU  - Larkum AW
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Decision Trees
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Sequence Alignment
MH  - Sequence Homology, Nucleic Acid
EDAT- 2004/09/17 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/09/17 05:00
PHST- 2004/09/17 05:00 [pubmed]
PHST- 2004/12/16 09:00 [medline]
PHST- 2004/09/17 05:00 [entrez]
AID - 10.1080/10635150490468648 [doi]
AID - 34TLNTVHYXCPYNFD [pii]
PST - ppublish
SO  - Syst Biol. 2004 Aug;53(4):638-43. doi: 10.1080/10635150490468648.

PMID- 15371250
OWN - NLM
STAT- MEDLINE
DCOM- 20041201
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 4
DP  - 2004 Aug
TI  - Tracing the decay of the historical signal in biological sequence data.
PG  - 623-37
AB  - Alignments of nucleotide or amino acid sequences may contain a variety of
      different signals, one of which is the historical signal that we often try to
      recover by phylogenetic analysis. Other signals, such as those arising due to
      compositional heterogeneities, among-lineage and among-site rate heterogeneities,
      invariant sites, and covariotides, may interfere adversely with the recovery of
      the historical signal. The effect of the interaction of these signals on
      phylogenetic inference is not well understood and may, in many cases, even be
      underappreciated. In this study, we investigate this matter and present results
      based on Monte Carlo simulations. We explored the success of four phylogenetic
      methods in recovering the true tree from data that had evolved under conditions
      where the equilibrium base frequencies and substitution rates were allowed to
      vary among lineages. Seven scenarios with increasingly complex conditions were
      investigated. All of the methods tested, with the exception of neighbor-joining
      using LogDet distances, were sensitive to compositional convergence in nonsister 
      lineages. Maximum parsimony was also susceptible to attraction between long
      edges. In many cases, however, phylogenetic inference methods can still recover
      the true tree when misleading signals are present, in some instances even when
      the historical signal is no longer dominant. These results highlight the growing 
      need for simple methods to detect violation of the phylogenetic assumptions.
FAU - Ho, Simon Y
AU  - Ho SY
AD  - 1School of Biological Sciences, University of Sydney, NSW 2006, Australia.
FAU - Jermiin, Lars
AU  - Jermiin L
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Amino Acid Sequence
MH  - Amino Acid Substitution
MH  - Animals
MH  - *Base Sequence
MH  - Computer Simulation
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - Phylogeny
MH  - *Sequence Alignment
EDAT- 2004/09/17 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/09/17 05:00
PHST- 2004/09/17 05:00 [pubmed]
PHST- 2004/12/16 09:00 [medline]
PHST- 2004/09/17 05:00 [entrez]
AID - 10.1080/10635150490503035 [doi]
AID - PLPKLVWT3GQJX3T0 [pii]
PST - ppublish
SO  - Syst Biol. 2004 Aug;53(4):623-37. doi: 10.1080/10635150490503035.

PMID- 15371249
OWN - NLM
STAT- MEDLINE
DCOM- 20041201
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 4
DP  - 2004 Aug
TI  - The evolution of body size, Cope's rule and the origin of amniotes.
PG  - 594-622
AB  - The evolution of body size in tetrapods is assessed using a database that
      includes 107 early stegocephalian species ranging in time from the Frasnian
      (Upper Devonian) to the Tatarian (Upper Permian). All analyses use methods that
      incorporate phylogenetic information (topology and branch lengths). In all tests,
      the impact of alternative topologies and branch lengths are assessed. Previous
      reports that raised doubts about the accuracy of squared-change parsimony
      assessment of ancestral character value appear to have used datasets in which
      there was no phylogenetic signal. Hence, squared-change parsimony may be more
      reliable than suggested in recent studies, at least when a phylogenetic signal is
      present in the datasets of interest. Analysis using random taxon reshuffling on
      three reference phylogenies shows that cranial and presacral length include a
      strong phylogenetic signal. Character optimization of body size in
      stegocephalians using squared-change parsimony on a time-calibrated phylogeny
      incorporating branch length information is used to test a previously published
      scenario on the origin of amniotes and of the amniotic egg that implies that the 
      ancestors of amniotes were small (no more than 10 cm in snout-vent length), and
      that their size increased subsequent to the appearance of the amniotic egg. The
      optimization suggests that first amniotes were somewhat larger than previously
      hypothesized; the estimated snout-vent length is about 24 cm, and the lower end
      of the 95% confidence interval of the phylogeny that yields the smallest inferred
      size suggests that no ancestor of amniotes measured less than 12 cm in snout-vent
      length. Character optimization, permutational multiple linear regressions, and
      independent contrast analyses show that Cope's rule of phyletic size increase
      applies to early reptiliomorphs but that it does not apply to early
      stegocephalians globally.
FAU - Laurin, Michel
AU  - Laurin M
AD  - Equipe Formations squelettiques FRE CNRS 2696, Case 7077, Universite Paris 7,
      Paris, France. laurin@ccr.jussieu.fr
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - *Body Constitution
MH  - *Body Size
MH  - Databases, Factual
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2004/09/17 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/09/17 05:00
PHST- 2004/09/17 05:00 [pubmed]
PHST- 2004/12/16 09:00 [medline]
PHST- 2004/09/17 05:00 [entrez]
AID - 10.1080/10635150490445706 [doi]
AID - BE6VPTYUFKVFKLXV [pii]
PST - ppublish
SO  - Syst Biol. 2004 Aug;53(4):594-622. doi: 10.1080/10635150490445706.

PMID- 15371248
OWN - NLM
STAT- MEDLINE
DCOM- 20041201
LR  - 20131121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 4
DP  - 2004 Aug
TI  - Phylogenetic artifacts can be caused by leucine, serine, and arginine codon usage
      heterogeneity: dinoflagellate plastid origins as a case study.
PG  - 582-93
AB  - Phylogenetic analyses of first and second codon positions (DNA1 + 2 analysis) and
      amino acid sequences (protein analysis) are often thought to provide similar
      estimates of deep-level phylogeny. However, here we report a novel artifact
      influencing DNA level phylogenetic inference of protein-coding genes introduced
      by codon usage heterogeneity that causes significant incongruities between DNA1 +
      2 and protein analyses. DNA1 + 2 analyses of plastid-encoded psbA genes (encoding
      of photosystem II D1 proteins) strongly suggest a relationship between haptophyte
      plastids and typical (peridinin-containing) dinoflagellate plastids. The psbA
      genes from haptophytes and a subset of the peridinin-type plastids display
      similar codon usage patterns for Leu, Ser, and Arg, which are each encoded by two
      separated codon sets that differ at first or first plus second codon positions.
      Our detailed analyses clearly indicate that these unusual preferences shared by
      haptophyte and some peridinin-type plastid genes are largely responsible for
      their strong affinity in DNA analyses. In particular, almost all of the support
      from DNA level analyses for the monophyly of haptophyte and peridinin-type
      plastids is lost when the codons corresponding to constant Leu, Ser, and Arg
      amino acids are excluded, suggesting that this signal comes from rapidly evolving
      synonymous substitutions, rather than from substitutions that result in amino
      acid changes. Indeed, protein maximum-likelihood analyses of concatenated PsaA
      and PsbA amino acid sequences indicate that, although 19'
      hexanoyloxyfucoxanthin-type (19' HNOF-type) plastids in dinoflagellates group
      with haptophyte plastids, peridinin-type plastids group weakly with those of
      stramenopiles. Consequently our results cast doubt on the single origin of
      peridinin-type and 19' HNOF-type plastids in dinoflagellates previously suggested
      on the basis of psaA and psbA concatenated gene phylogenetic analyses. We suggest
      that codon usage heterogeneity could be a more general problem for DNA level
      analyses of protein-coding genes, even when third codon positions are excluded.
FAU - Inagaki, Yuji
AU  - Inagaki Y
AD  - Department of Bioscience, Nagahama Institute of Bioscience and Technology,
      Nagahama, Shiga, Japan. yinagai@dal.ca
FAU - Simpson, Alastair
AU  - Simpson A
FAU - Dacks, Joel
AU  - Dacks J
FAU - Roger, Andrew
AU  - Roger A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
RN  - 0 (DNA, Protozoan)
RN  - 452VLY9402 (Serine)
RN  - 94ZLA3W45F (Arginine)
RN  - GMW67QNF9C (Leucine)
SB  - IM
MH  - Animals
MH  - Arginine/genetics
MH  - *Artifacts
MH  - Codon/*genetics
MH  - DNA, Protozoan/chemistry/genetics
MH  - Dinoflagellida/*classification/*genetics
MH  - Leucine/genetics
MH  - *Phylogeny
MH  - Serine/genetics
EDAT- 2004/09/17 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/09/17 05:00
PHST- 2004/09/17 05:00 [pubmed]
PHST- 2004/12/16 09:00 [medline]
PHST- 2004/09/17 05:00 [entrez]
AID - 10.1080/10635150490468756 [doi]
AID - WMFC7X37PFRYAPT0 [pii]
PST - ppublish
SO  - Syst Biol. 2004 Aug;53(4):582-93. doi: 10.1080/10635150490468756.

PMID- 15371247
OWN - NLM
STAT- MEDLINE
DCOM- 20041201
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 4
DP  - 2004 Aug
TI  - A phylogenetic mixture model for detecting pattern-heterogeneity in gene sequence
      or character-state data.
PG  - 571-81
AB  - We describe a general likelihood-based 'mixture model' for inferring phylogenetic
      trees from gene-sequence or other character-state data. The model accommodates
      cases in which different sites in the alignment evolve in qualitatively distinct 
      ways, but does not require prior knowledge of these patterns or partitioning of
      the data. We call this qualitative variability in the pattern of evolution across
      sites "pattern-heterogeneity" to distinguish it from both a homogenous process of
      evolution and from one characterized principally by differences in rates of
      evolution. We present studies to show that the model correctly retrieves the
      signals of pattern-heterogeneity from simulated gene-sequence data, and we apply 
      the method to protein-coding genes and to a ribosomal 12S data set. The mixture
      model outperforms conventional partitioning in both these data sets. We implement
      the mixture model such that it can simultaneously detect rate- and
      pattern-heterogeneity. The model simplifies to a homogeneous model or a
      rate-variability model as special cases, and therefore always performs at least
      as well as these two approaches, and often considerably improves upon them. We
      make the model available within a Bayesian Markov-chain Monte Carlo framework for
      phylogenetic inference, as an easy-to-use computer program.
FAU - Pagel, Mark
AU  - Pagel M
AD  - School of Animal and Microbial Sciences, University of Reading, England.
      m.pagel@rdg.ac.uk
FAU - Meade, Andrew
AU  - Meade A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Codon)
RN  - 0 (RNA, Ribosomal)
RN  - 0 (RNA, ribosomal, 12S)
SB  - IM
MH  - Base Sequence
MH  - Codon/genetics
MH  - Computer Simulation
MH  - *Genes
MH  - Likelihood Functions
MH  - Markov Chains
MH  - *Models, Biological
MH  - Models, Genetic
MH  - Monte Carlo Method
MH  - Pattern Recognition, Automated
MH  - *Phylogeny
MH  - RNA, Ribosomal/chemistry/genetics
EDAT- 2004/09/17 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/09/17 05:00
PHST- 2004/09/17 05:00 [pubmed]
PHST- 2004/12/16 09:00 [medline]
PHST- 2004/09/17 05:00 [entrez]
AID - 10.1080/10635150490468675 [doi]
AID - 3FVXJV27MVYM4BDE [pii]
PST - ppublish
SO  - Syst Biol. 2004 Aug;53(4):571-81. doi: 10.1080/10635150490468675.

PMID- 15371246
OWN - NLM
STAT- MEDLINE
DCOM- 20041201
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 4
DP  - 2004 Aug
TI  - Using exon and intron sequences of the gene Mp20 to resolve basal relationships
      in Cicindela (Coleoptera:Cicindelidae).
PG  - 554-70
AB  - The genus Cicindela (Coleoptera: Cicindelidae) is a species-rich cosmopolitan
      group of tiger beetles useful for comparing clade diversification worldwide.
      Knowledge about relationships of major groups is important for this analysis but 
      basal nodes in Cicindela have been difficult to resolve with standard mtDNA
      markers. Here we developed the Mp20 gene, a single-copy nuclear marker coding for
      a muscle-associated protein in insects, for phylogenetic analysis of basal groups
      of Cicindela. Nearly full-length sequences were obtained for 51 cicindelids,
      including major taxonomic groups from all continents. Sequences of Mp20 were
      between 1.2 and 1.7 kb and spanning three introns. Phylogenetic signal of exon
      and intron sequences was compared with that from four gene regions of mtDNA (COI,
      COIII, Cytb, 16S rRNA; 2.4 kb total). Because introns differed in length,
      sequence alignment was conducted using various procedures of phenetic and
      parsimony-based character coding of indels to assess their phylogenetic
      information content, but major nodes were recovered consistently. Mp20 sequences 
      contributed two thirds of the total support of the combined analysis, with most
      signal from the introns. We found major clades of Cicindela to be geographically 
      largely coincident with continental regions, confined to Australasia, the
      Holarctic, the Indian subcontinent, Africa, and South and Central America. Clock 
      estimates using various maximum-likelihood (ML) branch length calculations
      resulted in roughly similar divergence times whether Mp20 exon, introns, or mtDNA
      were used, and they were not greatly affected by different procedures for coding 
      and optimizing indel characters. Based on existing clock calibrations in
      Cicindela, basal splits of continental lineages occurred in the mid-Miocene,
      placing the radiation of basal groups of Cicindela to a period when their
      open-vegetation habitats expanded globally.
FAU - Pons, Joan
AU  - Pons J
AD  - Department of Entomology, The Natural History Museum, London SW7 5BD, United
      Kingdom.joap@nhm.ac.uk
FAU - Barraclough, Timothy
AU  - Barraclough T
FAU - Theodorides, Kosmas
AU  - Theodorides K
FAU - Cardoso, Anabela
AU  - Cardoso A
FAU - Vogler, Alfried
AU  - Vogler A
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Coleoptera/*classification/*genetics
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - Exons/*genetics
MH  - Gene Amplification
MH  - Geography
MH  - Introns/*genetics
MH  - *Phylogeny
EDAT- 2004/09/17 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/09/17 05:00
PHST- 2004/09/17 05:00 [pubmed]
PHST- 2004/12/16 09:00 [medline]
PHST- 2004/09/17 05:00 [entrez]
AID - 10.1080/10635150490472940 [doi]
AID - YLQ3MPDPCYD6RA19 [pii]
PST - ppublish
SO  - Syst Biol. 2004 Aug;53(4):554-70. doi: 10.1080/10635150490472940.

PMID- 15371245
OWN - NLM
STAT- MEDLINE
DCOM- 20041201
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 4
DP  - 2004 Aug
TI  - Phylogeny and divergence-date estimates of rapid radiations in muroid rodents
      based on multiple nuclear genes.
PG  - 533-53
AB  - The muroid rodents are the largest superfamily of mammals, containing nearly one 
      third of all mammal species. We report on a phylogenetic study comprising 53
      genera sequenced for four nuclear genes, GHR, BRCA1, RAG1, and c-myc, totaling up
      to 6400 nucleotides. Most relationships among the subfamilies are resolved. All
      four genes yield nearly identical phylogenies, differing only in five key
      regions, four of which may represent particularly rapid radiations. Support is
      very strong for a fundamental division of the mole rats of the subfamilies
      Spalacinae and Rhizomyinae from all other muroids. Among the other "core"
      muroids, a rapid radiation led to at least four distinct lineages: Asian
      Calomyscus, an African clade of at least four endemic subfamilies, including the 
      diverse Nesomyinae of Madagascar, a hamster clade with maximum diversity in the
      New World, and an Old World clade including gerbils and the diverse Old World
      mice and rats (Murinae). The Deomyinae, recently removed from the Murinae, is
      well supported as the sister group to the gerbils (Gerbillinae). Four key regions
      appear to represent rapid radiations and, despite a large amount of sequence
      data, remain poorly resolved: the base of the "core" muroids, among the five
      cricetid (hamster) subfamilies, within a large clade of Sigmodontinae endemic to 
      South America, and among major geographic lineages of Old World Murinae. Because 
      of the detailed taxon sampling within the Murinae, we are able to refine the
      fossil calibration of a rate-smoothed molecular clock and apply this clock to
      date key events in muroid evolution. We calculate rate differences among the gene
      regions and relate those differences to relative contribution of each gene to the
      support for various nodes. The among-gene variance in support is greatest for the
      shortest branches. We present a revised classification for this largest but most 
      unsettled mammalian superfamily.
FAU - Steppan, Scott
AU  - Steppan S
AD  - Department of Biological Science, Florida State University, Tallahassee, Florida 
      32306-1100, USA. steppan@bio.fsu.edu
FAU - Adkins, Ronald
AU  - Adkins R
FAU - Anderson, Joel
AU  - Anderson J
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Cell Nucleus/*genetics
MH  - DNA Primers
MH  - Gene Expression
MH  - Genetic Variation
MH  - Models, Genetic
MH  - Muridae/*classification/*genetics
MH  - Phylogeny
EDAT- 2004/09/17 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/09/17 05:00
PHST- 2004/09/17 05:00 [pubmed]
PHST- 2004/12/16 09:00 [medline]
PHST- 2004/09/17 05:00 [entrez]
AID - 10.1080/10635150490468701 [doi]
AID - 148D9JG325A76DWP [pii]
PST - ppublish
SO  - Syst Biol. 2004 Aug;53(4):533-53. doi: 10.1080/10635150490468701.

PMID- 15371244
OWN - NLM
STAT- MEDLINE
DCOM- 20041201
LR  - 20101118
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 4
DP  - 2004 Aug
TI  - A method for estimating the relative importance of characters in cladistic
      analyses.
PG  - 529-32
AB  - The method of character importance ranking (CIR) is proposed here as a means for 
      estimating the relative "importance" of characters in cladistic analyses,
      especially those based on morphological features. CIR uses the weighting variable
      to incrementally remove one character at a time from the analysis, and then
      evaluates the impact of the removal on the shape of the cladogram. The greater
      the impact, the more important the character. The CIR method for determining
      which characters drive the shape of a particular cladogram has several
      applications. It identifies the characters with the strongest (though not
      necessarily most accurate) signal in a cladistic analysis; it permits the
      informed prioritization of characters for further investigation via genetic,
      developmental, and functional approaches; and it highlights characters whose
      definition, scoring, independence, and variation should be reviewed with
      particular care. The application of CIR reveals that at least some cladograms
      depend entirely on a single character.
FAU - DeGusta, David
AU  - DeGusta D
AD  - Department of Anthropological Sciences, Building 360, Stanford University,
      Stanford, CA 94305-2117, USA. degusta@stanford.edu
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Chlorophyta/classification
MH  - Fossils
MH  - Hominidae/classification
MH  - Humans
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2004/09/17 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/09/17 05:00
PHST- 2004/09/17 05:00 [pubmed]
PHST- 2004/12/16 09:00 [medline]
PHST- 2004/09/17 05:00 [entrez]
AID - 10.1080/10635150490470320 [doi]
AID - X291G8RCY99DUWPD [pii]
PST - ppublish
SO  - Syst Biol. 2004 Aug;53(4):529-32. doi: 10.1080/10635150490470320.

PMID- 15371243
OWN - NLM
STAT- MEDLINE
DCOM- 20041201
LR  - 20040916
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 4
DP  - 2004 Aug
TI  - Inconsistency of maximum parsimony revisited.
PG  - 521-8
AB  - Felsenstein (1978, Syst. Zool. 27:401-410) showed that the method of maximum
      parsimony can be inconsistent, i.e., lead to an incorrect result with an infinite
      amount of data. The situation in which this inconsistency occurs is often called 
      the "Felsenstein zone," the phenomenon also known as "long-branch attraction."
      Felsenstein derived a sufficient inconsistency condition from a model for four
      taxa with only two different parameters for the probability of change on the five
      branches connecting the four taxa. In the present paper, his approach is used to 
      derive the inconsistency condition of maximum parsimony from the most general
      model for four taxa, i.e., with five different parameters for the probabilities
      of change on the five branches and, for the first time, for characters with k
      states (k = 2, 3, 4, 5, 6, ...) This is used to determine the factors that can
      cause the inconsistency of maximum parsimony. It is shown that the probability of
      change on all five branches and the number of character states play a role in
      causing inconsistency.
FAU - Schulmeister, Susanne
AU  - Schulmeister S
AD  - Division of Invertebrate Zoology, American Museum of Natural History, Central
      Park West at 79th Street, New York, NY 10024, USA. schulmei@amnh.org
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Models, Biological
EDAT- 2004/09/17 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/09/17 05:00
PHST- 2004/09/17 05:00 [pubmed]
PHST- 2004/12/16 09:00 [medline]
PHST- 2004/09/17 05:00 [entrez]
AID - 10.1080/10635150490445788 [doi]
AID - 8E0094ERMPHMQR65 [pii]
PST - ppublish
SO  - Syst Biol. 2004 Aug;53(4):521-8. doi: 10.1080/10635150490445788.

PMID- 15205059
OWN - NLM
STAT- MEDLINE
DCOM- 20040720
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 2
DP  - 2004 Apr
TI  - Trees versus characters and the supertree/supermatrix "paradox".
PG  - 356-9
FAU - Bininda-Emonds, Olaf R P
AU  - Bininda-Emonds OR
AD  - Lehrstuhl fur Tierzucht, Technical University of Munich, Alte Akademie 12, 85354 
      Freising-Weihenstephan, Germany. Olaf.Bininda@tierzucht.tum.de
LA  - eng
PT  - Comment
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2004 Apr;53(2):342-55. PMID: 15205058
MH  - Classification/*methods
MH  - *Meta-Analysis as Topic
MH  - *Phylogeny
MH  - Research Design
EDAT- 2004/06/19 05:00
MHDA- 2004/07/21 05:00
CRDT- 2004/06/19 05:00
PHST- 2004/06/19 05:00 [pubmed]
PHST- 2004/07/21 05:00 [medline]
PHST- 2004/06/19 05:00 [entrez]
AID - 10.1080/10635150490440396 [doi]
AID - WXQQGW2AAV5GH3AJ [pii]
PST - ppublish
SO  - Syst Biol. 2004 Apr;53(2):356-9. doi: 10.1080/10635150490440396.

PMID- 15205058
OWN - NLM
STAT- MEDLINE
DCOM- 20040720
LR  - 20071115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 2
DP  - 2004 Apr
TI  - Inconsistencies in arguments for the supertree approach: supermatrices versus
      supertrees of Crocodylia.
PG  - 342-55
FAU - Gatesy, John
AU  - Gatesy J
AD  - Department of Biology, University of California, Riverside, California 92521,
      USA. johnga@citrus.ucr.edu
FAU - Baker, Richard H
AU  - Baker RH
FAU - Hayashi, Cheryl
AU  - Hayashi C
LA  - eng
PT  - Comment
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2003 Oct;52(5):724-9. PMID: 14530139
CIN - Syst Biol. 2004 Apr;53(2):356-9. PMID: 15205059
MH  - Alligators and Crocodiles/anatomy &amp; histology/*genetics
MH  - Animals
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - *Meta-Analysis as Topic
MH  - *Phylogeny
MH  - Research Design
EDAT- 2004/06/19 05:00
MHDA- 2004/07/21 05:00
CRDT- 2004/06/19 05:00
PHST- 2004/06/19 05:00 [pubmed]
PHST- 2004/07/21 05:00 [medline]
PHST- 2004/06/19 05:00 [entrez]
AID - 10.1080/10635150490423971 [doi]
AID - G7T40JY8V1W8AHTJ [pii]
PST - ppublish
SO  - Syst Biol. 2004 Apr;53(2):342-55. doi: 10.1080/10635150490423971.

PMID- 15205057
OWN - NLM
STAT- MEDLINE
DCOM- 20040720
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 2
DP  - 2004 Apr
TI  - Accepting partnership by submission? Morphological phylogenetics in a molecular
      millennium.
PG  - 333-42
FAU - Jenner, Ronald A
AU  - Jenner RA
AD  - University Museum of Zoology, University of Cambridge, Downing Street, Cambridge 
      CB2 3EJ, United Kingdom. raj35@cam.ac.uk
LA  - eng
PT  - Comment
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2003 Aug;52(4):539-48. PMID: 12857644
MH  - *Anatomy, Comparative
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - *Research
MH  - Sample Size
EDAT- 2004/06/19 05:00
MHDA- 2004/07/21 05:00
CRDT- 2004/06/19 05:00
PHST- 2004/06/19 05:00 [pubmed]
PHST- 2004/07/21 05:00 [medline]
PHST- 2004/06/19 05:00 [entrez]
AID - 10.1080/10635150490423962 [doi]
AID - LX4MDFXTNE4VUBHM [pii]
PST - ppublish
SO  - Syst Biol. 2004 Apr;53(2):333-42. doi: 10.1080/10635150490423962.

PMID- 15205056
OWN - NLM
STAT- MEDLINE
DCOM- 20040720
LR  - 20040618
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 2
DP  - 2004 Apr
TI  - Distances that perfectly mislead.
PG  - 327-32
AB  - Given a collection of discrete characters (e.g., aligned DNA sites, gene
      adjacencies), a common measure of distance between taxa is the proportion of
      characters for which taxa have different character states. Tree reconstruction
      based on these (uncorrected) distances can be statistically inconsistent and can 
      lead to trees different from those obtained using character-based methods such as
      maximum likelihood or maximum parsimony. However, in these cases the distance
      data often reveal their unreliability by some deviation from additivity, as
      indicated by conflicting support for more than one tree. We describe two results 
      that show how uncorrected (and miscorrected) distance data can be simultaneously 
      perfectly additive and misleading. First, multistate character data can be
      perfectly compatible and define one tree, and yet the uncorrected distances
      derived from these characters are perfectly treelike (and obey a molecular
      clock), only for a completely different tree. Second, under a Markov model of
      character evolution a similar phenomenon can occur; not only is there statistical
      inconsistency using uncorrected distances, but there is no evidence of this
      inconsistency because the distances look perfectly treelike (this does not occur 
      in the classic two-parameter Felsenstein zone). We characterize precisely when
      uncorrected distances are additive on the true (and on a false) tree for four
      taxa. We also extend this result to a more general setting that applies to
      distances corrected according to an incorrect model.
FAU - Huson, Daniel H
AU  - Huson DH
AD  - Center for Bioinformatics, Tubingen University, Tuebingen, Germany.
      huson@informatik.uni-tuebingen.de
FAU - Steel, Mike
AU  - Steel M
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - *Evolution, Molecular
MH  - Markov Chains
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2004/06/19 05:00
MHDA- 2004/07/21 05:00
CRDT- 2004/06/19 05:00
PHST- 2004/06/19 05:00 [pubmed]
PHST- 2004/07/21 05:00 [medline]
PHST- 2004/06/19 05:00 [entrez]
AID - 10.1080/10635150490423809 [doi]
AID - 726PRQEW5BHJNNJN [pii]
PST - ppublish
SO  - Syst Biol. 2004 Apr;53(2):327-32. doi: 10.1080/10635150490423809.

PMID- 15205055
OWN - NLM
STAT- MEDLINE
DCOM- 20040720
LR  - 20081121
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 2
DP  - 2004 Apr
TI  - Single-copy nuclear genes recover cretaceous-age divergences in bees.
PG  - 309-26
AB  - We analyzed the higher level phylogeny of the bee family Halictidae based on the 
      coding regions of three single-copy nuclear genes (long-wavelength [LW] opsin,
      wingless, and elongation factor 1-alpha [EF-1 alpha]). Our combined data set
      consisted of 2,234 aligned nucleotide sites (702 base pairs [bp] for LW opsin,
      405 bp for wingless, and 1,127 bp for EF-1 alpha) and 779 parsimony-informative
      sites. We included 58 species of halictid bees from 33 genera, representing all
      subfamilies and tribes, and rooted the trees using seven outgroups from other bee
      families: Colletidae, Andrenidae, Melittidae, and Apidae. We analyzed the
      separate and combined data sets by a variety of methods, including equal weights 
      parsimony, maximum likelihood, and Bayesian methods. Analysis of the combined
      data set produced a strong phylogenetic signal with high bootstrap and Bremer
      support and high posterior probability well into the base of the tree. The
      phylogeny recovered the monophyly of the Halictidae and of all four subfamilies
      and both tribes, recovered relationships among the subfamilies and tribes
      congruent with morphology, and provided robust support for the relationships
      among the numerous genera in the tribe Halictini, sensu Michener (2000). Using
      our combined nucleotide data set, several recently described halictid fossils
      from the Oligocene and Eocene, and recently developed Bayesian methods, we
      estimated the antiquity of major clades within the family. Our results indicate
      that each of the four subfamilies arose well before the Cretaceous-Tertiary
      boundary and suggest that the early radiation of halictid bees involved
      substantial African-South American interchange roughly coincident with the
      separation of these two continents in the late Cretaceous. This combination of
      single-copy nuclear genes is capable of recovering Cretaceous-age divergences in 
      bees with high levels of support. We propose that LW opsin, wingless, and EF-1
      alpha(F2 copy) may be useful in resolving relationships among bee families and
      other Cretaceous-age insect lineages.
FAU - Danforth, Bryan N
AU  - Danforth BN
AD  - Department of Entomology, Comstock Hall, Cornell University, Ithaca, New York
      14853, USA. bnd1@cornell.edu
FAU - Brady, Sean G
AU  - Brady SG
FAU - Sipes, Sedonia D
AU  - Sipes SD
FAU - Pearson, Adam
AU  - Pearson A
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (Peptide Elongation Factor 1)
RN  - 0 (Rod Opsins)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Bees/classification/*genetics
MH  - DNA Primers
MH  - *Evolution, Molecular
MH  - Gene Components
MH  - Geography
MH  - Geological Phenomena
MH  - Geology
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Peptide Elongation Factor 1/*genetics
MH  - *Phylogeny
MH  - Rod Opsins/*genetics
MH  - Sequence Analysis, DNA
MH  - Time Factors
EDAT- 2004/06/19 05:00
MHDA- 2004/07/21 05:00
CRDT- 2004/06/19 05:00
PHST- 2004/06/19 05:00 [pubmed]
PHST- 2004/07/21 05:00 [medline]
PHST- 2004/06/19 05:00 [entrez]
AID - 10.1080/10635150490423737 [doi]
AID - EHVHUDQB79YD2P7T [pii]
PST - ppublish
SO  - Syst Biol. 2004 Apr;53(2):309-26. doi: 10.1080/10635150490423737.

PMID- 15205054
OWN - NLM
STAT- MEDLINE
DCOM- 20040720
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 2
DP  - 2004 Apr
TI  - Performance of flip supertree construction with a heuristic algorithm.
PG  - 299-308
AB  - Supertree methods are used to assemble separate phylogenetic trees with shared
      taxa into larger trees (supertrees) in an effort to construct more comprehensive 
      phylogenetic hypotheses. In spite of much recent interest in supertrees, there
      are still few methods for supertree construction. The flip supertree problem is
      an error correction approach that seeks to find a minimum number of changes
      (flips) to the matrix representation of the set of input trees to resolve their
      incompatibilities. A previous flip supertree algorithm was limited to finding
      exact solutions and was only feasible for small input trees. We developed a
      heuristic algorithm for the flip supertree problem suitable for much larger input
      trees. We used a series of 48- and 96-taxon simulations to compare supertrees
      constructed with the flip supertree heuristic algorithm with supertrees
      constructed using other approaches, including MinCut (MC), modified MC (MMC), and
      matrix representation with parsimony (MRP). Flip supertrees are generally far
      more accurate than supertrees constructed using MC or MMC algorithms and are at
      least as accurate as supertrees built with MRP. The flip supertree method is
      therefore a viable alternative to other supertree methods when the number of taxa
      is large.
FAU - Eulenstein, Oliver
AU  - Eulenstein O
AD  - Department of Computer Science, Iowa State University, Ames, Iowa 50011, USA.
      oeulenst@cs.iastate.edu
FAU - Chen, Duhong
AU  - Chen D
FAU - Burleigh, J Gordon
AU  - Burleigh JG
FAU - Fernandez-Baca, David
AU  - Fernandez-Baca D
FAU - Sanderson, Michael J
AU  - Sanderson MJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Phylogeny
EDAT- 2004/06/19 05:00
MHDA- 2004/07/21 05:00
CRDT- 2004/06/19 05:00
PHST- 2004/06/19 05:00 [pubmed]
PHST- 2004/07/21 05:00 [medline]
PHST- 2004/06/19 05:00 [entrez]
AID - 10.1080/10635150490423719 [doi]
AID - TDXAHC1A1LCG5DJB [pii]
PST - ppublish
SO  - Syst Biol. 2004 Apr;53(2):299-308. doi: 10.1080/10635150490423719.

PMID- 15205053
OWN - NLM
STAT- MEDLINE
DCOM- 20040720
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 2
DP  - 2004 Apr
TI  - Phylogeny of Agrodiaetus Hubner 1822 (Lepidoptera: Lycaenidae) inferred from
      mtDNA sequences of COI and COII and nuclear sequences of EF1-alpha: karyotype
      diversification and species radiation.
PG  - 278-98
AB  - Butterflies in the large Palearctic genus Agrodiaetus (Lepidoptera: Lycaenidae)
      are extremely uniform and exhibit few distinguishing morphological characters.
      However, these insects are distinctive in one respect: as a group they possess
      among the greatest interspecific karyotype diversity in the animal kingdom, with 
      chromosome numbers (n) ranging from 10 to 125. The monophyly of Agrodiaetus and
      its systematic position relative to other groups within the section Polyommatus
      have been controversial. Characters from the mitochondrial genes for cytochrome
      oxidases I and II and from the nuclear gene for elongation factor 1 alpha were
      used to reconstruct the phylogeny of Agrodiaetus using maximum parsimony and
      Bayesian phylogenetic methods. Ninety-one individuals, encompassing most of the
      taxonomic diversity of Agrodiaetus, and representatives of 14 related genera were
      included in this analysis. Our data indicate that Agrodiaetus is monophyletic.
      Representatives of the genus Polyommatus (sensu stricto) are the closest
      relatives. The sequences of the Agrodiaetus taxa in this analysis are tentatively
      arranged into 12 clades, only 1 of which corresponds to a species group
      traditionally recognized in Agrodiaetus. Heterogeneous substitution rates across 
      a recovered topology were homogenized with a nonparametric rate-smoothing
      algorithm before the application of a molecular clock. Two published estimates of
      substitution rates dated the origin of Agrodiaetus between 2.51 and 3.85 million 
      years ago. During this time, there was heterogeneity in the rate and direction of
      karyotype evolution among lineages within the genus. Karyotype instability has
      evolved independently three times in the section Polyommatus, within the lineages
      Agrodiaetus, Lysandra, and Plebicula. Rapid karyotype diversification may have
      played a significant role in the radiation of the genus Agrodiaetus.
FAU - Kandul, Nikolai P
AU  - Kandul NP
AD  - Department of Organismal and Evolutionary Biology, Harvard University, 26 Oxford 
      Street, Cambridge, Massachusetts 02138, USA. kandul@fas.harvard.edu
FAU - Lukhtanov, Vladimir A
AU  - Lukhtanov VA
FAU - Dantchenko, Alexander V
AU  - Dantchenko AV
FAU - Coleman, James W S
AU  - Coleman JW
FAU - Sekercioglu, Cagan H
AU  - Sekercioglu CH
FAU - Haig, David
AU  - Haig D
FAU - Pierce, Naomi E
AU  - Pierce NE
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Peptide Elongation Factor 1)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Butterflies/*genetics
MH  - Chromosomes/*genetics
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - Karyotyping
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Peptide Elongation Factor 1/genetics
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Time Factors
EDAT- 2004/06/19 05:00
MHDA- 2004/07/21 05:00
CRDT- 2004/06/19 05:00
PHST- 2004/06/19 05:00 [pubmed]
PHST- 2004/07/21 05:00 [medline]
PHST- 2004/06/19 05:00 [entrez]
AID - 10.1080/10635150490423692 [doi]
AID - JP7DXTJH0Y7RWEY3 [pii]
PST - ppublish
SO  - Syst Biol. 2004 Apr;53(2):278-98. doi: 10.1080/10635150490423692.

PMID- 15205052
OWN - NLM
STAT- MEDLINE
DCOM- 20040720
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 2
DP  - 2004 Apr
TI  - The importance of proper model assumption in bayesian phylogenetics.
PG  - 265-77
AB  - We studied the importance of proper model assumption in the context of Bayesian
      phylogenetics by examining &gt;5,000 Bayesian analyses and six nested models of
      nucleotide substitution. Model misspecification can strongly bias bipartition
      posterior probability estimates. These biases were most pronounced when rate
      heterogeneity was ignored. The type of bias seen at a particular bipartition
      appeared to be strongly influenced by the lengths of the branches surrounding
      that bipartition. In the Felsenstein zone, posterior probability estimates of
      bipartitions were biased when the assumed model was underparameterized but were
      unbiased when the assumed model was overparameterized. For the inverse
      Felsenstein zone, however, both underparameterization and overparameterization
      led to biased bipartition posterior probabilities, although the bias caused by
      overparameterization was less pronounced and disappeared with increased sequence 
      length. Model parameter estimates were also affected by model misspecification.
      Underparameterization caused a bias in some parameter estimates, such as branch
      lengths and the gamma shape parameter, whereas overparameterization caused a
      decrease in the precision of some parameter estimates. We caution researchers to 
      assure that the most appropriate model is assumed by employing both a priori
      model choice methods and a posteriori model adequacy tests.
FAU - Lemmon, Alan R
AU  - Lemmon AR
AD  - Section of Integrative Biology, University of Texas, 1 University Station C0930, 
      Austin, Texas 78712, USA. alemmon@evotutor.org
FAU - Moriarty, Emily C
AU  - Moriarty EC
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Computer Simulation
MH  - Evolution, Molecular
MH  - *Models, Genetic
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Research Design
EDAT- 2004/06/19 05:00
MHDA- 2004/07/21 05:00
CRDT- 2004/06/19 05:00
PHST- 2004/06/19 05:00 [pubmed]
PHST- 2004/07/21 05:00 [medline]
PHST- 2004/06/19 05:00 [entrez]
AID - 10.1080/10635150490423520 [doi]
AID - 0V16MQEP6H0V4N55 [pii]
PST - ppublish
SO  - Syst Biol. 2004 Apr;53(2):265-77. doi: 10.1080/10635150490423520.

PMID- 15205051
OWN - NLM
STAT- MEDLINE
DCOM- 20040720
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 2
DP  - 2004 Apr
TI  - Unraveling the evolutionary radiation of the thoracican barnacles using molecular
      and morphological evidence: a comparison of several divergence time estimation
      approaches.
PG  - 244-64
AB  - The Thoracica includes the ordinary barnacles found along the sea shore and is
      the most diverse and well-studied superorder of Cirripedia. However, although the
      literature abounds with scenarios explaining the evolution of these barnacles,
      very few studies have attempted to test these hypotheses in a phylogenetic
      context. The few attempts at phylogenetic analyses have suffered from a lack of
      phylogenetic signal and small numbers of taxa. We collected DNA sequences from
      the nuclear 18S, 28S, and histone H3 genes and the mitochondrial 12S and 16S
      genes (4,871 bp total) and data for 37 adult and 53 larval morphological
      characters from 43 taxa representing all the extant thoracican suborders (except 
      the monospecific Brachylepadomorpha). Four Rhizocephala (highly modified
      parasitic barnacles) taxa and a Rhizocephala + Acrothoracica (burrowing
      barnacles) hypothetical ancestor were used as the outgroup for the molecular and 
      morphological analyses, respectively. We analyzed these data separately and
      combined using maximum likelihood (ML) under "hill-climbing" and genetic
      algorithm heuristic searches, maximum parsimony procedures, and Bayesian
      inference coupled with Markov chain Monte Carlo techniques under mixed and
      homogeneous models of nucleotide substitution. The resulting phylogenetic trees
      answered key questions in barnacle evolution. The four-plated Iblomorpha were
      shown as the most primitive thoracican, and the plateless Heteralepadomorpha were
      placed as the sister group of the Lepadomorpha. These relationships suggest for
      the first time in an invertebrate that exoskeleton biomineralization may have
      evolved from phosphatic to calcitic. Sessilia (nonpedunculate) barnacles were
      depicted as monophyletic and appear to have evolved from a stalked (pedunculate) 
      multiplated (5+) scalpelloidlike ancestor rather than a five-plated lepadomorphan
      ancestor. The Balanomorpha (symmetric sessile barnacles) appear to have the
      following relationship: (Chthamaloidea(Coronuloidea(Tetraclitoidea,
      Balanoidea))). Thoracican divergence times were estimated under ML-based local
      clock, Bayesian, and penalized likelihood approaches using an 18S data set and
      three calibration points: Heteralepadomorpha = 530 million years ago (MYA),
      Scalpellomorpha = 340 MYA, and Verrucomorpha = 120 MYA. Estimated dates varied
      considerably within and between approaches depending on the calibration point.
      Highly parameterized local clock models that assume independent rates (r &gt; or =
      15) for confamilial or congeneric species generated the most congruent estimates 
      among calibrations and agreed more closely with the barnacle fossil record.
      Reasonable estimates were also obtained under the Bayesian procedure of Kishino
      et al. (2001, Mol. Biol. Evol. 18:352-361) but using multiple calibrations. Most 
      of the dates estimated under the Bayesian procedure of Aris-Brosou and Yang
      (2002, Syst. Biol. 51:703-714) and the penalized likelihood method using single
      and/or multiple calibrations were inconsistent among calibrations and did not fit
      the fossil record.
FAU - Perez-Losada, Marcos
AU  - Perez-Losada M
AD  - Department of Integrative Biology, Brigham Young University, Provo, Utah
      84602-5181, USA. mp323@email.byu.edu
FAU - Hoeg, Jens T
AU  - Hoeg JT
FAU - Crandall, Keith A
AU  - Crandall KA
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - 0 (Histones)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA, Ribosomal/genetics
MH  - Databases, Nucleic Acid
MH  - *Evolution, Molecular
MH  - Histones/genetics
MH  - Larva/anatomy &amp; histology
MH  - Likelihood Functions
MH  - Markov Chains
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Thoracica/*anatomy &amp; histology/*genetics/physiology
MH  - Time Factors
EDAT- 2004/06/19 05:00
MHDA- 2004/07/21 05:00
CRDT- 2004/06/19 05:00
PHST- 2004/06/19 05:00 [pubmed]
PHST- 2004/07/21 05:00 [medline]
PHST- 2004/06/19 05:00 [entrez]
AID - 10.1080/10635150490423458 [doi]
AID - E9DCA48FMDAPQGBU [pii]
PST - ppublish
SO  - Syst Biol. 2004 Apr;53(2):244-64. doi: 10.1080/10635150490423458.

PMID- 15205050
OWN - NLM
STAT- MEDLINE
DCOM- 20040720
LR  - 20171116
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 2
DP  - 2004 Apr
TI  - Southern hemisphere biogeography inferred by event-based models: plant versus
      animal patterns.
PG  - 216-43
AB  - The Southern Hemisphere has traditionally been considered as having a
      fundamentally vicariant history. The common trans-Pacific disjunctions are
      usually explained by the sequential breakup of the supercontinent Gondwana during
      the last 165 million years, causing successive division of an ancestral biota.
      However, recent biogeographic studies, based on molecular estimates and more
      accurate paleogeographic reconstructions, indicate that dispersal may have been
      more important than traditionally assumed. We examined the relative roles played 
      by vicariance and dispersal in shaping Southern Hemisphere biotas by analyzing a 
      large data set of 54 animal and 19 plant phylogenies, including marsupials,
      ratites, and southern beeches (1,393 terminals). Parsimony-based tree fitting in 
      conjunction with permutation tests was used to examine to what extent Southern
      Hemisphere biogeographic patterns fit the breakup sequence of Gondwana and to
      identify concordant dispersal patterns. Consistent with other studies, the animal
      data are congruent with the geological sequence of Gondwana breakup: (Africa(New 
      Zealand(southern South America, Australia))). Trans-Antarctic dispersal
      (Australia &lt;--&gt; southern South America) is also significantly more frequent than 
      any other dispersal event in animals, which may be explained by the long period
      of geological contact between Australia and South America via Antarctica. In
      contrast, the dominant pattern in plants, (southern South America(Australia, New 
      Zealand)), is better explained by dispersal, particularly the prevalence of
      trans-Tasman dispersal between New Zealand and Australia. Our results also
      confirm the hybrid origin of the South American biota: there has been
      surprisingly little biotic exchange between the northern tropical and the
      southern temperate regions of South America, especially for animals.
FAU - Sanmartin, Isabel
AU  - Sanmartin I
AD  - Department of Systematic Zoology, Evolutionary Biology Centre, Norbyvagen 18D,
      SE-752 36 Uppsala, Sweden. isabel.sanmartin@ebc.uu.se
FAU - Ronquist, Fredrik
AU  - Ronquist F
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Africa
MH  - Animals
MH  - Antarctic Regions
MH  - Australia
MH  - Demography
MH  - Geography
MH  - Geological Phenomena
MH  - *Geology
MH  - India
MH  - Insecta/*genetics
MH  - *Models, Theoretical
MH  - Pacific Islands
MH  - *Phylogeny
MH  - Plants/*genetics
MH  - South America
MH  - Vertebrates/*genetics
EDAT- 2004/06/19 05:00
MHDA- 2004/07/21 05:00
CRDT- 2004/06/19 05:00
PHST- 2004/06/19 05:00 [pubmed]
PHST- 2004/07/21 05:00 [medline]
PHST- 2004/06/19 05:00 [entrez]
AID - 10.1080/10635150490423430 [doi]
AID - JMGEA89JGWNBCD9Q [pii]
PST - ppublish
SO  - Syst Biol. 2004 Apr;53(2):216-43. doi: 10.1080/10635150490423430.

PMID- 15205049
OWN - NLM
STAT- MEDLINE
DCOM- 20040720
LR  - 20151119
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 2
DP  - 2004 Apr
TI  - Molecular phylogeny, historical biogeography, and divergence time estimates for
      swallowtail butterflies of the genus Papilio (Lepidoptera: Papilionidae).
PG  - 193-215
AB  - Swallowtail butterflies are recognized as model organisms in ecology,
      evolutionary biology, genetics, and conservation biology but present numerous
      unresolved phylogenetic problems. We inferred phylogenetic relationships for 51
      of about 205 species of the genus Papilio (sensu lato) from 3.3-Kilobase (kb)
      sequences of mitochondrial and nuclear DNA (2.3 kb of cytochrome oxidases I and
      II and 1.0 kb of elongation factor 1 alpha). Congruent phylogenetic trees were
      recovered within Papilio from analyses of combined data using maximum likelihood,
      Bayesian analysis, and maximum parsimony bootstrap consensus. Several
      disagreements with the traditional classification of Papilio were found. Five
      major previously hypothesized subdivisions within Papilio were well supported:
      Heraclides, Pterourus, Chilasa, Papilio (sensu stricto), and Eleppone. Further
      studies are required to clarify relationships within traditional "Princeps,"
      which was paraphyletic. Several biologically interesting characteristics of
      Papilio appear to have polyphyletic origins, including mimetic adults, larval
      host associations, and larval morphology. Early diversification within Papilio is
      estimated at 55-65 million years ago based on a combination of biogeographic time
      constraints rather than fossils. This divergence time suggests that Papilio has
      slower apparent substitution rates than do Drosophila and fig-pollinating wasps
      and/or divergences corrected using best-fit substitution models are still being
      consistently underestimated. The amount of sequence divergence between Papilio
      subdivisions is equivalent to divergences between genera in other tribes of the
      Papilionidae, and between genera of moths of the noctuid subfamily Heliothinae.
FAU - Zakharov, Evgueni V
AU  - Zakharov EV
AD  - Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G
      2E9, Canada. zakharov@ualberta.ca
FAU - Caterino, Michael S
AU  - Caterino MS
FAU - Sperling, Felix A H
AU  - Sperling FA
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Butterflies/anatomy &amp; histology/classification/*genetics/physiology
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - Feeding Behavior/physiology
MH  - Geography
MH  - Larva/anatomy &amp; histology/physiology
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Time Factors
MH  - Wings, Animal/anatomy &amp; histology
EDAT- 2004/06/19 05:00
MHDA- 2004/07/21 05:00
CRDT- 2004/06/19 05:00
PHST- 2004/06/19 05:00 [pubmed]
PHST- 2004/07/21 05:00 [medline]
PHST- 2004/06/19 05:00 [entrez]
AID - 10.1080/10635150490423403 [doi]
AID - 30F7FVMWWRQJDQQ3 [pii]
PST - ppublish
SO  - Syst Biol. 2004 Apr;53(2):193-215. doi: 10.1080/10635150490423403.

PMID- 15205048
OWN - NLM
STAT- MEDLINE
DCOM- 20040720
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 2
DP  - 2004 Apr
TI  - Recent origin and phylogenetic utility of divergent ITS putative pseudogenes: a
      case study from Naucleeae (Rubiaceae).
PG  - 177-92
AB  - The internal transcribed spacer (ITS) of nuclear ribosomal DNA has been widely
      used by systematists for reconstructing phylogenies of closely related taxa.
      Although the occurrence of ITS putative pseudogenes is well documented for many
      groups of animals and plants, the potential utility of these pseudogenes in
      phylogenetic analyses has often been underestimated or even ignored in part
      because of deletions that make unambiguous alignment difficult. In addition, long
      branches often can lead to spurious relationships, particularly in parsimony
      analyses. We have discovered unusually high levels of ITS polymorphism (up to
      30%, 40%, and 14%, respectively) in three tropical tree species of the coffee
      family (Rubiaceae), Adinauclea fagifolia, Haldina cordifolia, and Mitragyna
      rubrostipulata. Both secondary structure stability and patterns of nucleotide
      substitutions in a highly conserved region (5.8S gene) were used for
      distinguishing presumed functional sequences from putative pseudogenes. The
      combination of both criteria was the most powerful approach. The sequences from
      A. fagifolia appear to be a mix of functional genes and highly distinct putative 
      pseudogenes, whereas those from H. cordifolia and M. rubrostipulata were
      identified as putative pseudogenes. We explored the potential utility of the
      identified putative pseudogenes in the phylogenetic analyses of Naucleeae sensu
      lato. Both Bayesian and parsimony trees identified the same monophyletic groups
      and indicated that the polymorphisms do not transcend species boundaries,
      implying that they do not predate the divergence of these three species. The
      resulting trees are similar to those produced by previous analyses of chloroplast
      genes. In contrast to results of previous studies therefore, divergent putative
      pseudogenes can be useful for phylogenetic analyses, especially when no sequences
      of their functional counterparts are available. Our studies clearly show that ITS
      polymorphism may not necessarily mislead phylogenetic inference. Despite using
      many different PCR conditions (different primers, higher denaturing temperatures,
      and absence or presence of DMSO and BSA-TMACl), we recovered only a few
      functional ITS copies from A. fagifolia and none from H. cordifolia and M.
      rubrostipulata, which suggests that PCR selection is occurring and/or the
      presumed functional alleles are located at minor loci (with few ribosomal DNA
      copies).
FAU - Razafimandimbison, Sylvain G
AU  - Razafimandimbison SG
AD  - Department of Systemic Biology, Evolutionary Biology Centre, Uppsala University, 
      Sweden. sylvain.razafimandimbison@ebc.uu.se
FAU - Kellogg, Elizabeth A
AU  - Kellogg EA
FAU - Bremer, Birgitta
AU  - Bremer B
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Ribosomal Spacer)
SB  - IM
MH  - Base Composition
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Conserved Sequence/genetics
MH  - DNA Primers
MH  - DNA, Ribosomal Spacer/*genetics
MH  - *Evolution, Molecular
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Mutation/genetics
MH  - *Phylogeny
MH  - Polymerase Chain Reaction/methods
MH  - Polymorphism, Genetic
MH  - Pseudogenes/*genetics
MH  - Rubiaceae/*genetics
MH  - Sequence Analysis, DNA
EDAT- 2004/06/19 05:00
MHDA- 2004/07/21 05:00
CRDT- 2004/06/19 05:00
PHST- 2004/06/19 05:00 [pubmed]
PHST- 2004/07/21 05:00 [medline]
PHST- 2004/06/19 05:00 [entrez]
AID - 10.1080/10635150490423278 [doi]
AID - NJ5D827V0WWMM9RX [pii]
PST - ppublish
SO  - Syst Biol. 2004 Apr;53(2):177-92. doi: 10.1080/10635150490423278.

PMID- 14965911
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Ecology of congruence: past meets present.
PG  - 165-73
AB  - Phylogenetic congruence is governed by various macroevolutionary events,
      including cospeciation, host switching, sorting, duplication, and failure to
      speciate. The relative frequency of these events may be influenced by factors
      that govern the distribution and abundance of the interacting groups; i.e.,
      ecological factors. If so, it may be possible to predict the degree of
      phylogenetic congruence between two groups from information about their ecology. 
      Unfortunately, adequate comparative ecological data are not available for many of
      the systems that have been subjected to cophylogenetic analysis. An exception is 
      provided by chewing lice (Insecta: Phthiraptera), which parasitize birds and
      mammals. For a few genera of these lice, enough data have now been published to
      begin exploring the relationship between ecology and congruence. In general,
      there is a correspondence between important ecological factors and the degree of 
      phylogenetic congruence. Careful comparison of these genera suggests that
      dispersal is a more fundamental barrier to host switching among related hosts
      than is establishment. Transfer experiments show that host-specific lice can
      survive and reproduce on novel hosts that are similar in size to the native host 
      as long as the lice can disperse to these hosts. To date, studies of parasite
      dispersal have been mainly inferential. A better understanding of the role of
      dispersal will require more direct data on dispersal frequency and distances.
FAU - Clayton, Dale H
AU  - Clayton DH
AD  - Department of Biology, University of Utah, Salt Lake City, Utah 84112, USA.
      clayton@biology.utah.edu
FAU - Bush, Sarah E
AU  - Bush SE
FAU - Johnson, Kevin P
AU  - Johnson KP
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Birds/*genetics/parasitology
MH  - *Ecosystem
MH  - Host-Parasite Interactions
MH  - *Movement
MH  - Phthiraptera/*genetics
MH  - *Phylogeny
MH  - Rodentia/genetics/parasitology
MH  - *Symbiosis
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - 87H1QPXWL287QBTA [pii]
AID - 10.1080/10635150490265102 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):165-73. doi: 10.1080/10635150490265102.

PMID- 14965910
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Biogeography explains cophylogenetic patterns in toucan chewing lice.
PG  - 154-64
AB  - Historically, comparisons of host and parasite phylogenies have concentrated on
      cospeciation. However, many of these comparisons have demonstrated that the
      phylogenies of hosts and parasites are seldom completely congruent, suggesting
      that phenomena other than cospeciation play an important role in the evolution of
      host-parasite assemblages. Other coevolutionary phenomena, such as host
      switching, parasite duplication (speciation on the host), sorting (extinction),
      and failure to speciate can also influence host-parasite assemblages. Using
      mitochondrial and nuclear protein-coding DNA sequences, I reconstructed the
      phylogeny of ectoparasitic toucan chewing lice in the Austrophilopterus
      cancellosus subspecies complex and compared this phylogeny with the phylogeny of 
      the hosts, the Ramphastos toucans, to reconstruct the history of coevolutionary
      events in this host-parasite assemblage. Three salient findings emerged. First,
      reconstructions of host and louse phylogenies indicate that they do not branch in
      parallel, and their cophylogenetic history shows little or no significant
      cospeciation. Second, members of monophyletic Austrophilopterus toucan louse
      lineages are not necessarily restricted to monophyletic host lineages. Often,
      closely related lice are found on more distantly related but sympatric toucan
      hosts. Third, the geographic distribution of the hosts apparently plays a role in
      the speciation of these lice. These results suggest that for some louse lineages 
      biogeography may be more important than host associations in structuring louse
      populations and species, particularly when host life history (e.g., hole nesting)
      or parasite life history (e.g., phoresis) might promote frequent host switching
      events between syntopic host species. These findings highlight the importance of 
      integrating biogeographic information into cophylogenetic studies.
FAU - Weckstein, Jason D
AU  - Weckstein JD
AD  - Department of Biological Sciences and Museum of Natural Science, 119 Foster Hall,
      Louisiana State University, Baton Rouge, Louisiana 70803-3216, USA.
      jweckst@inhs.uiuc.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Birds/*genetics/*parasitology
MH  - DNA, Mitochondrial/genetics
MH  - Geography
MH  - Host-Parasite Interactions
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Phthiraptera/*genetics
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - Y9V1F9N5WT50TP6N [pii]
AID - 10.1080/10635150490265085 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):154-64. doi: 10.1080/10635150490265085.

PMID- 14965909
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Clade-limited colonization in brood parasitic finches (Vidua spp.).
PG  - 140-53
AB  - The African brood parasitic finches (Vidua spp.) are host specialists that mimic 
      the songs and nestling mouth markings of their finch hosts (family Estrildidae). 
      Although recent molecular analyses suggest rapid speciation associated with host 
      switches in some members of this group, the association of different Vidua
      lineages with particular host genera suggests the possibility of cospeciation at 
      higher levels in the host and parasite phylogenies. We compared a phylogeny of
      all Vidua species with a phylogeny of their estrildid finch hosts and compared
      divergence time estimates for the two groups. Basal divergences among extant
      members of the Vidulidae and among Vidua species are more recent than those among
      host genera and species, respectively, allowing a model of cospeciation to be
      rejected at most or all levels of the Vidua phylogeny. Nonetheless, some tests
      for cospeciation indicated significant congruence between host and parasite tree 
      topologies. This result may be an artifact of clade-limited colonization. Host
      switches in parasitic finches have most often involved new hosts in the same or a
      closely related genus, an effect that increases the apparent congruence of host
      and parasites trees.
FAU - Sorenson, Michael D
AU  - Sorenson MD
AD  - Department of Biology, Boston University, Boston, Massachusetts 02215, USA.
      msoren@bu.edu
FAU - Balakrishnan, Christopher N
AU  - Balakrishnan CN
FAU - Payne, Robert B
AU  - Payne RB
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - *Models, Biological
MH  - Molecular Sequence Data
MH  - Nesting Behavior/*physiology
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Songbirds/*genetics/physiology
MH  - *Symbiosis
MH  - Time Factors
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - H9WE9D2WQ49E2N03 [pii]
AID - 10.1080/10635150490265021 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):140-53. doi: 10.1080/10635150490265021.

PMID- 14965908
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Correlated evolution in fig pollination.
PG  - 128-39
AB  - This is the first comparative study of correlated evolution between figs (Ficus
      species, Moraceae) and their pollinators (Hymenoptera: Agaoninae) based on
      molecular phylogenies of both lineages. Fig relationships based on the internal
      transcribed spacer region (ITS) of nuclear ribosomal DNA and pollinator
      relationships inferred from mitochondrial cytochrome oxidase I (COI) sequences
      enabled the study of correlated evolution based on molecular phylogenies for the 
      largest set of interacting species ever compared. Comparative methods have been
      applied to tests of adaptation, but the application of these methods in tests of 
      coadaptation, defined as reciprocal evolutionary change in interacting lineages, 
      has received less attention. I have extended tests of correlated evolution
      between two traits along a phylogeny to the case of interacting lineages, where
      two traits may or may not share a common phylogenetic history. Independent
      contrasts and phylogenetic autocorrelation rejected the null hypothesis that
      trait correlations within lineages are stronger than trait correlations between
      interacting lineages. Fig style lengths and pollinator ovipositor lengths, for
      example, were more highly correlated than were pollinator body size and
      ovipositor length. Mutualistic interactions between figs and their pollinators
      illustrate the novel ways in which phylogenies and comparative methods can detect
      patterns of correlated evolution. The most outstanding evidence of correlated
      evolution between these obligate mutualists is that interacting trait
      correlations are stronger than within-lineage allometric relationships.
FAU - Weiblen, George D
AU  - Weiblen GD
AD  - Department of Plant Biology, University of Minnesota, 1445 Gortner Avenue, Saint 
      Paul, Minnesota 55108, USA. gweiblen@umn.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - *Adaptation, Biological
MH  - Animals
MH  - Body Weights and Measures
MH  - DNA, Ribosomal/genetics
MH  - Electron Transport Complex IV/genetics
MH  - Ficus/*anatomy &amp; histology/genetics
MH  - *Phylogeny
MH  - *Symbiosis
MH  - Wasps/*anatomy &amp; histology/genetics
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - F2Q1GN4M4Y59VA9X [pii]
AID - 10.1080/10635150490265012 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):128-39. doi: 10.1080/10635150490265012.

PMID- 14965907
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Plant-insect interactions: double-dating associated insect and plant lineages
      reveals asynchronous radiations.
PG  - 120-7
AB  - An increasing number of plant-insect studies using phylogenetic analysis suggest 
      that cospeciation events are rare in plant-insect systems. Instead, nonrandom
      patterns of phylogenetic congruence are produced by phylogenetically conserved
      host switching (to related plants) or tracking of particular resources or traits 
      (e.g., chemical). The dominance of host switching in many phytophagous insect
      groups may make the detection of genuine cospeciation events difficult. One
      important test of putative cospeciation events is to verify whether reciprocal
      speciation is temporally plausible. We explored techniques for double-dating of
      both plant and insect phylogenies. We use dated molecular phylogenies of a
      psyllid (Hemiptera)-Genisteae (Fabaceae) system, a predominantly monophagous
      insect-plant association widespread on the Atlantic Macaronesian islands.
      Phylogenetic reconciliation analysis suggests high levels of parallel
      cladogenesis between legumes and psyllids. However, dating using molecular clocks
      calibrated on known geological ages of the Macaronesian islands revealed that the
      legume and psyllid radiations were not contemporaneous but sequential. Whereas
      the main plant radiation occurred some 8 million years ago, the insect radiation 
      occurred about 3 million years ago. We estimated that &gt;60% of the psyllid
      speciation has resulted from host switching between related hosts. The only
      evidence for true cospeciation is in the much more recent and localized radiation
      of genistoid legumes in the Canary Islands, where the psyllid and legume
      radiations have been partially contemporaneous. The identification of specific
      cospeciation events over this time period, however, is hindered by the
      phylogenetic uncertainty in both legume and psyllid phylogenies due to the
      apparent rapidity of the species radiations.
FAU - Percy, Diana M
AU  - Percy DM
AD  - Division of Environmental and Evolutionary Biology, Institute of Biomedical and
      Life Sciences, University of Glasgow, Glasgow G12 8QQ, U.K.
      dmpercy@interchange.ubc.ca
FAU - Page, Roderic D M
AU  - Page RD
FAU - Cronk, Quentin C B
AU  - Cronk QC
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Atlantic Islands
MH  - Base Sequence
MH  - *Evolution, Molecular
MH  - Fabaceae/*genetics
MH  - Geography
MH  - Hemiptera/*genetics
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - *Symbiosis
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - RUFM5ARW16YVGBQ1 [pii]
AID - 10.1080/10635150490264996 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):120-7. doi: 10.1080/10635150490264996.

PMID- 14965906
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Evolutionary relationships, cospeciation, and host switching in avian malaria
      parasites.
PG  - 111-9
AB  - We used phylogenetic analyses of cytochrome b sequences of malaria parasites and 
      their avian hosts to assess the coevolutionary relationships between host and
      parasite lineages. Many lineages of avian malaria parasites have broad host
      distributions, which tend to obscure cospeciation events. The hosts of a single
      parasite or of closely related parasites were nonetheless most frequently
      recovered from members of the same host taxonomic family, more so than expected
      by chance. However, global assessments of the relationship between parasite and
      host phylogenetic trees, using Component and ParaFit, failed to detect
      significant cospeciation. The event-based approach employed by TreeFitter
      revealed significant cospeciation and duplication with certain cost assignments
      for these events, but host switching was consistently more prominent in matching 
      the parasite tree to the host tree. The absence of a global cospeciation signal
      despite conservative host distribution most likely reflects relatively frequent
      acquisition of new hosts by individual parasite lineages. Understanding these
      processes will require a more refined species concept for malaria parasites and
      more extensive sampling of parasite distributions across hosts. If parasites can 
      disperse between allopatric host populations through alternative hosts,
      cospeciation may not have a strong influence on the architecture of host-parasite
      relationships. Rather, parasite speciation may happen more often in conjunction
      with the acquisition of new hosts followed by divergent selection between host
      lineages in sympatry. Detailed studies of the phylogeographic distributions of
      hosts and parasites are needed to characterize these events.
FAU - Ricklefs, Robert E
AU  - Ricklefs RE
AD  - Department of Biology, University of Missouri, 8001 Natural Bridge Road, St.
      Louis, Missouri 63121-4499, USA. ricklefs@umsl.edu
FAU - Fallon, Sylvia M
AU  - Fallon SM
FAU - Bermingham, Eldredge
AU  - Bermingham E
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9035-37-4 (Cytochromes b)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Birds/*genetics/*parasitology
MH  - Cytochromes b/genetics
MH  - Haemosporida/*genetics
MH  - Host-Parasite Interactions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - JHWP2NXHNXGL6HET [pii]
AID - 10.1080/10635150490264987 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):111-9. doi: 10.1080/10635150490264987.

PMID- 14965905
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Host-symbiont stability and fast evolutionary rates in an ant-bacterium
      association: cospeciation of camponotus species and their endosymbionts,
      candidatus blochmannia.
PG  - 95-110
AB  - Bacterial endosymbionts are widespread across several insect orders and are
      involved in interactions ranging from obligate mutualism to reproductive
      parasitism. Candidatus Blochmannia gen. nov. (Blochmannia) is an obligate
      bacterial associate of Camponotus and related ant genera (Hymenoptera:
      Formicidae). The occurrence of Blochmannia in all Camponotus species sampled from
      field populations and its maternal transmission to host offspring suggest that
      this bacterium is engaged in a long-term, stable association with its ant hosts. 
      However, evidence for cospeciation in this system is equivocal because previous
      phylogenetic studies were based on limited gene sampling, lacked statistical
      analysis of congruence, and have even suggested host switching. We compared
      phylogenies of host genes (the nuclear EF-1alphaF2 and mitochondrial COI/II) and 
      Blochmannia genes (16S ribosomal DNA [rDNA], groEL, gidA, and rpsB), totaling
      more than 7 kilobases for each of 16 Camponotus species. Each data set was
      analyzed using maximum likelihood and Bayesian phylogenetic reconstruction
      methods. We found minimal conflict among host and symbiont phylogenies, and the
      few areas of discordance occurred at deep nodes that were poorly supported by
      individual data sets. Concatenated protein-coding genes produced a very
      well-resolved tree that, based on the Shimodaira-Hasegawa test, did not conflict 
      with any host or symbiont data set. Correlated rates of synonymous substitution
      (d(S)) along corresponding branches of host and symbiont phylogenies further
      supported the hypothesis of cospeciation. These findings indicate that
      Blochmannia-Camponotus symbiosis has been evolutionarily stable throughout tens
      of millions of years. Based on inferred divergence times among the ant hosts, we 
      estimated rates of sequence evolution of Blochmannia to be approximately 0.0024
      substitutions per site per million years (s/s/MY) for the 16S rDNA gene and
      approximately 0.1094 s/s/MY at synonymous positions of the genes sampled. These
      rates are several-fold higher than those for related bacteria Buchnera aphidicola
      and Escherichia coli. Phylogenetic congruence among Blochmannia genes indicates
      genome stability that typifies primary endosymbionts of insects.
FAU - Degnan, Patrick H
AU  - Degnan PH
AD  - Josephine Bay Paul Center for Comparative Molecular Biology and Evolution, Marine
      Biological Laboratory, 7 MBL Street, Woods Hole, Massachusetts 02543, USA.
FAU - Lazarus, Adam B
AU  - Lazarus AB
FAU - Brock, Chad D
AU  - Brock CD
FAU - Wernegreen, Jennifer J
AU  - Wernegreen JJ
LA  - eng
GR  - R01 GM62626-01/GM/NIGMS NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Ants/*genetics
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - DNA, Ribosomal/genetics
MH  - Enterobacteriaceae/*genetics
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - *Symbiosis
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - JPADCD2N0F198Q0X [pii]
AID - 10.1080/10635150490264842 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):95-110. doi: 10.1080/10635150490264842.

PMID- 14965904
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Untangling coevolutionary history.
PG  - 92-4
FAU - Johnson, Kevin P
AU  - Johnson KP
AD  - Illinois Natural History Survey, Champaign, Illinois 61820, USA.
      kjohnson@inhs.uiuc.edu
FAU - Clayton, Dale H
AU  - Clayton DH
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Ecosystem
MH  - *Phylogeny
MH  - *Symbiosis
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - KEVG8XRCXKCFDXWL [pii]
AID - 10.1080/10635150490264824 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):92-4. doi: 10.1080/10635150490264824.

PMID- 14965902
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Congruence versus phylogenetic accuracy: revisiting the incongruence length
      difference test.
PG  - 81-9
FAU - Hipp, Andrew L
AU  - Hipp AL
AD  - Department of Botany, University of Wisconsin, 430 Lincoln Drive, Madison,
      Wisconsin 53706, USA. alhipp@wisc.edu
FAU - Hall, Jocelyn C
AU  - Hall JC
FAU - Sytsma, Kenneth J
AU  - Sytsma KJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - Lorisidae/*genetics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Research Design
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - 44UF9PBA1YV6KXCU [pii]
AID - 10.1080/10635150490264752 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):81-9. doi: 10.1080/10635150490264752.

PMID- 14965901
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Meaning of the name tetrapoda in the scientific literature: an exchange.
PG  - 68-80
FAU - Laurin, Michel
AU  - Laurin M
AD  - Equipe Formations Squelettiques, UMR CNRS 8570, Case 7077, Universite Paris 7,
      75005 Paris, France. laurin@ccr.jussieu.fr
FAU - Anderson, Jason S
AU  - Anderson JS
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Phylogeny
MH  - *Vertebrates
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - 80CAKC9PNGEFR7X5 [pii]
AID - 10.1080/10635150490264716 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):68-80. doi: 10.1080/10635150490264716.

PMID- 14965900
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Bayesian phylogenetic analysis of combined data.
PG  - 47-67
AB  - The recent development of Bayesian phylogenetic inference using Markov chain
      Monte Carlo (MCMC) techniques has facilitated the exploration of parameter-rich
      evolutionary models. At the same time, stochastic models have become more
      realistic (and complex) and have been extended to new types of data, such as
      morphology. Based on this foundation, we developed a Bayesian MCMC approach to
      the analysis of combined data sets and explored its utility in inferring
      relationships among gall wasps based on data from morphology and four genes
      (nuclear and mitochondrial, ribosomal and protein coding). Examined models range 
      in complexity from those recognizing only a morphological and a molecular
      partition to those having complex substitution models with independent parameters
      for each gene. Bayesian MCMC analysis deals efficiently with complex models:
      convergence occurs faster and more predictably for complex models, mixing is
      adequate for all parameters even under very complex models, and the parameter
      update cycle is virtually unaffected by model partitioning across sites.
      Morphology contributed only 5% of the characters in the data set but nevertheless
      influenced the combined-data tree, supporting the utility of morphological data
      in multigene analyses. We used Bayesian criteria (Bayes factors) to show that
      process heterogeneity across data partitions is a significant model component,
      although not as important as among-site rate variation. More complex evolutionary
      models are associated with more topological uncertainty and less conflict between
      morphology and molecules. Bayes factors sometimes favor simpler models over
      considerably more parameter-rich models, but the best model overall is also the
      most complex and Bayes factors do not support exclusion of apparently weak
      parameters from this model. Thus, Bayes factors appear to be useful for selecting
      among complex models, but it is still unclear whether their use strikes a
      reasonable balance between model complexity and error in parameter estimates.
FAU - Nylander, Johan A A
AU  - Nylander JA
AD  - Department of Systematic Zoology, Evolutionary Biology Centre, Uppsala
      University, Norbyvagen 18 D, SE-752 36 Uppsala, Sweden. johan.nylander@ebc.uu.se
FAU - Ronquist, Fredrik
AU  - Ronquist F
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
FAU - Nieves-Aldrey, Jose Luis
AU  - Nieves-Aldrey JL
LA  - eng
GR  - R01 GM069801/GM/NIGMS NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Bayes Theorem
MH  - Classification/*methods
MH  - Evolution, Molecular
MH  - Markov Chains
MH  - *Models, Biological
MH  - *Models, Genetic
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Research Design
MH  - Wasps/anatomy &amp; histology/genetics
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - C5P30PXY4HAWE7BM [pii]
AID - 10.1080/10635150490264699 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):47-67. doi: 10.1080/10635150490264699.

PMID- 14965899
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Sampling properties of the bootstrap support in molecular phylogeny: influence of
      nonindependence among sites.
PG  - 38-46
AB  - The influence of nonindependence among sites on phylogenetic reconstructions and 
      bootstrap scores was investigated both analytically and empirically. First, the
      sampling properties of the bootstrap support in the four-species case was derived
      for the maximum-parsimony method, assuming either independently or
      nonindependently evolving sites. The influence of various models of departure
      from the independence assumption was quantified. Second, trees and bootstrap
      scores estimated from subsets of consecutive (potentially coevolving) versus
      dispersed (presumably independent) sites of a ribosomal RNA data set were
      contrasted. The two approaches consistently suggest that a departure from the
      assumption of independent sites tends to reduce the amount of phylogenetic
      information contained in the data, but to increase the apparent statistical
      support for reconstructed trees, as measured by the bootstrap. In particular,
      nonindependence can lead to strongly supported wrong internal branches.
FAU - Galtier, Nicolas
AU  - Galtier N
AD  - CNRS UMR 5000, Genome, Populations, Interactions, Universite Montpellier 2, CC
      63, Place E. Bataillon, 34095 Montpellier, France. galtier@univ-montp2.fr
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (RNA, Ribosomal)
SB  - IM
MH  - *Classification/*methods
MH  - Data Interpretation, Statistical
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - RNA, Ribosomal/*genetics
MH  - Selection Bias
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - EJ98AQGFYDAV9EWV [pii]
AID - 10.1080/10635150490264680 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):38-46. doi: 10.1080/10635150490264680.

PMID- 14965898
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Reticulate evolution, introgression, and intertribal gene capture in an
      allohexaploid grass.
PG  - 25-37
AB  - Recent molecular phylogenetic studies of polyploid plants have successfully
      clarified complex patterns of reticulate evolution. In this study of Elymus
      repens, an allohexaploid member of the wheat tribe Triticeae, chloroplast and
      nuclear DNA data reveal an extreme reticulate pattern, revealing at least five
      distinct gene lineages coexisting within the species, acquired through a possible
      combination of allohexaploidy and introgression from both within and beyond the
      Triticeae. Earlier cytogenetic studies of E. repens suggested that Hordeum
      (genome H) and Pseudoroegneria (St) were genome donors to E. repens. Chloroplast 
      DNA data presented here (from the rpoA gene and from the region between trnT and 
      trnF) identify three potential maternal genome donors (Pseudoroegneria,
      Thinopyrum, and Dasypyrum), and information from previous molecular work suggests
      that, of these, Pseudoroegneria is the most likely maternal donor. Nuclear starch
      synthase gene data indicate that both Hordeum and Pseudoroegneria have
      contributed to the nuclear genome of E. repens, in agreement with cytogenetic
      data. However, these data also show unexpected contributions from Taeniatherum,
      and from two additional donors of unknown identity. One of the sequences of
      unknown origin falls within the Triticeae, but is not closely associated with any
      of the sampled diploid genera. The second falls outside of the clade containing
      Triticeae and its outgroup Bromus, suggesting the acquisition of genetic material
      from a surprisingly divergent source. Bias toward the amplification of certain
      starch synthase variants has complicated attempts to thoroughly sample from
      within individuals, but the data clearly indicate a complex pattern of reticulate
      evolution, consistent not only with allohexaploidy, but also with introgression
      from unexpectedly divergent sources.
FAU - Mason-Gamer, Roberta J
AU  - Mason-Gamer RJ
AD  - University of Illinois at Chicago, Department of Biological Sciences, MC 066, 845
      W. Taylor Street, Chicago, Illinois 60607, USA. robie@uic.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Chloroplast)
RN  - EC 2.4.1.21 (Starch Synthase)
SB  - IM
MH  - Base Sequence
MH  - DNA Primers
MH  - DNA, Chloroplast/genetics
MH  - Elymus/*genetics
MH  - *Evolution, Molecular
MH  - *Hybridization, Genetic
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - *Polyploidy
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
MH  - Starch Synthase/genetics
MH  - United States
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - 7MWFMF8QWAHNLQ4R [pii]
AID - 10.1080/10635150490424402 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):25-37. doi: 10.1080/10635150490424402.

PMID- 14965897
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Out of Africa: the slow train to australasia.
PG  - 18-24
AB  - We used mitochondrial DNA (mtDNA) sequences to test biogeographic hypotheses for 
      Patiriella exigua (Asterinidae), one of the world's most widespread coastal sea
      stars. This small intertidal species has an entirely benthic life history and yet
      occurs in southern temperate waters of the Atlantic, Indian, and Pacific oceans. 
      Despite its abundance around southern Africa, southeastern Australia, and several
      oceanic islands, P. exigua is absent from the shores of Western Australia, New
      Zealand, and South America. Phylogenetic analysis of mtDNA sequences (cytochrome 
      oxidase I, control region) indicates that South Africa houses an assemblage of P.
      exigua that is not monophyletic (P = 0.04), whereas Australian and Lord Howe
      Island specimens form an interior monophyletic group. The placement of the root
      in Africa and small genetic divergences between eastern African and Australian
      haplotypes strongly suggest Pleistocene dispersal eastward across the Indian
      Ocean. Dispersal was probably achieved by rafting on wood or macroalgae, which
      was facilitated by the West Wind Drift. Genetic data also support Pleistocene
      colonization of oceanic islands (Lord Howe Island, Amsterdam Island, St. Helena).
      Although many biogeographers have speculated about the role of long-distance
      rafting, this study is one of the first to provide convincing evidence. The
      marked phylogeographic structure evident across small geographic scales in
      Australia and South Africa indicates that gene flow among populations may be
      generally insufficient to prevent the local evolution of monophyly. We suggest
      that P. exigua may rely on passive mechanisms of dispersal.
FAU - Waters, Jonathan M
AU  - Waters JM
AD  - Department of Zoology, University of Otago, P.O. Box 56, Dunedin, New Zealand.
      jonathan.waters@stonebow.otago.ac.nz
FAU - Roy, Michael S
AU  - Roy MS
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - Geography
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - *Movement
MH  - Oceans and Seas
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Starfish/*genetics
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - BR0M9MEL0NM7FULD [pii]
AID - 10.1080/10635150490264671 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):18-24. doi: 10.1080/10635150490264671.

PMID- 14965896
OWN - NLM
STAT- MEDLINE
DCOM- 20040430
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 53
IP  - 1
DP  - 2004 Feb
TI  - Complex data produce better characters.
PG  - 1-17
AB  - Two studies were conducted to explore the use of complex data in character
      description and hybrid identification. In order to determine if complex data
      allow the production of better characters, eight groups of plant systematists
      were given two classes of drawings of plant parts, and asked to divide them into 
      character states (clusters) in two separate experiments. The first class of
      drawings consisted only of cotyledons. The second class consisted of triplets of 
      drawings: a cotyledon, seedling leaf, and inflorescence bract. The triplets were 
      used to simulate complex data such as might be garnered by looking at a plant.
      Each experiment resulted in four characters (groups of clusters), one for each
      group of systematists. Visual and statistical analysis of the data showed that
      the systematists were able to produce smaller, more precisely defined character
      states using the more complex drawings. The character states created with the
      complex drawings also were more consistent across systematists, and agreed more
      closely with an independent assessment of phylogeny. To investigate the utility
      of complex data in an applied task, four observers rated 250 hybrids of Dubautia 
      ciliolata X arborea based on the overall form (Gestalt) of the plants, and took
      measurements of a number of features of the same plants. A composite score of the
      measurements was created using principal components analysis. The correlation
      between the scores on the first principal component and the Gestalt ratings was
      computed. The Gestalt ratings and PC scores were significantly correlated,
      demonstrating that assessments of overall similarity can be as useful as more
      conventional approaches in determining the hybrid status of plants.
FAU - Kirchoff, B K
AU  - Kirchoff BK
AD  - Department of Biology, P. O. Box 26170, University of North Carolina, Greensboro,
      North Carolina 27402, USA. kirchoff@uncg.edu
FAU - Richter, S J
AU  - Richter SJ
FAU - Remington, D L
AU  - Remington DL
FAU - Wisniewski, E
AU  - Wisniewski E
LA  - eng
GR  - 5F32GM20554/GM/NIGMS NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Asteraceae/*anatomy &amp; histology/genetics
MH  - Classification/*methods
MH  - *Hybridization, Genetic
MH  - *Phylogeny
MH  - Plant Components, Aerial/*anatomy &amp; histology
MH  - Principal Component Analysis
MH  - Research Design
EDAT- 2004/02/18 05:00
MHDA- 2004/05/01 05:00
CRDT- 2004/02/18 05:00
PHST- 2004/02/18 05:00 [pubmed]
PHST- 2004/05/01 05:00 [medline]
PHST- 2004/02/18 05:00 [entrez]
AID - FAU7J6EVX4JNNTBC [pii]
AID - 10.1080/10635150490424376 [doi]
PST - ppublish
SO  - Syst Biol. 2004 Feb;53(1):1-17. doi: 10.1080/10635150490424376.

PMID- 14668125
OWN - NLM
STAT- MEDLINE
DCOM- 20040220
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 6
DP  - 2003 Dec
TI  - Matrix representations with parsimony or with distances: two sides of the same
      coin?
PG  - 865-8
FAU - Lapointe, Francois-Joseph
AU  - Lapointe FJ
AD  - Departement de Sciences Biologiques, Universite de Montreal, C.P. 6128, Succ.
      Centre-ville, Montreal, Quebec H3C 3J7, Canada. lapoinf@biol.umontreal.ca
FAU - Wilkinson, Mark
AU  - Wilkinson M
FAU - Bryant, David
AU  - Bryant D
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Evolution, Molecular
MH  - *Models, Theoretical
MH  - *Phylogeny
EDAT- 2003/12/12 05:00
MHDA- 2004/02/21 05:00
CRDT- 2003/12/12 05:00
PHST- 2003/12/12 05:00 [pubmed]
PHST- 2004/02/21 05:00 [medline]
PHST- 2003/12/12 05:00 [entrez]
AID - DYQG25CBFGLNR0B5 [pii]
PST - ppublish
SO  - Syst Biol. 2003 Dec;52(6):865-8.

PMID- 14668123
OWN - NLM
STAT- MEDLINE
DCOM- 20040220
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 6
DP  - 2003 Dec
TI  - Are the dental data really at odds with the molecular data? Morphological
      evidence for whale phylogeny (re)reexamined.
PG  - 853-64; discussion 864-5
FAU - O'Leary, Maureen A
AU  - O'Leary MA
AD  - Department of Anatomical Sciences, Stony Brook University, Stony Brook, New York 
      11794-8081, USA. moleary@mail.som.sunysb.edu
FAU - Gatesy, John
AU  - Gatesy J
FAU - Novacek, Michael J
AU  - Novacek MJ
LA  - eng
PT  - Comment
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2001 Jun;50(3):444-53. PMID: 12116586
MH  - Animals
MH  - Artiodactyla/*anatomy &amp; histology/genetics
MH  - Cetacea/*anatomy &amp; histology/genetics
MH  - Classification/*methods
MH  - Fossils
MH  - Models, Genetic
MH  - *Phylogeny
MH  - *Research Design
MH  - Tooth/anatomy &amp; histology
EDAT- 2003/12/12 05:00
MHDA- 2004/02/21 05:00
CRDT- 2003/12/12 05:00
PHST- 2003/12/12 05:00 [pubmed]
PHST- 2004/02/21 05:00 [medline]
PHST- 2003/12/12 05:00 [entrez]
AID - PG8T9F181X6029KU [pii]
PST - ppublish
SO  - Syst Biol. 2003 Dec;52(6):853-64; discussion 864-5.

PMID- 14668122
OWN - NLM
STAT- MEDLINE
DCOM- 20040220
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 6
DP  - 2003 Dec
TI  - Phylogenetic methods and aetosaur interrelationships: a rejoinder.
PG  - 851-2
FAU - Harris, Simon R
AU  - Harris SR
AD  - Department of Earth Science, University of Bristol, Bristol BS8 1RJ, UK.
      simon.harris@bristol.ac.uk
FAU - Gower, David J
AU  - Gower DJ
FAU - Wilkinson, Mark
AU  - Wilkinson M
LA  - eng
PT  - Comment
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2003 Apr;52(2):253-5. PMID: 12746150
MH  - Alligators and Crocodiles/*anatomy &amp; histology/*classification
MH  - Animals
MH  - Classification/*methods
MH  - Fossils
MH  - *Phylogeny
MH  - Research Design
EDAT- 2003/12/12 05:00
MHDA- 2004/02/21 05:00
CRDT- 2003/12/12 05:00
PHST- 2003/12/12 05:00 [pubmed]
PHST- 2004/02/21 05:00 [medline]
PHST- 2003/12/12 05:00 [entrez]
AID - W2WEGWATHXWW5C7A [pii]
PST - ppublish
SO  - Syst Biol. 2003 Dec;52(6):851-2.

PMID- 14668121
OWN - NLM
STAT- MEDLINE
DCOM- 20040220
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 6
DP  - 2003 Dec
TI  - Branch lengths and support: revisited.
PG  - 849-51
FAU - Morrison, David A
AU  - Morrison DA
AD  - Department of Parasitology, National Veterinary Institute and Swedish University 
      of Agricultural Sciences, 751 89 Uppsala, Sweden. david.morrison@vmm.slu.se
LA  - eng
PT  - Comment
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2003 Feb;52(1):127-30. PMID: 12554447
MH  - Classification/*methods
MH  - *Data Interpretation, Statistical
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2003/12/12 05:00
MHDA- 2004/02/21 05:00
CRDT- 2003/12/12 05:00
PHST- 2003/12/12 05:00 [pubmed]
PHST- 2004/02/21 05:00 [medline]
PHST- 2003/12/12 05:00 [entrez]
AID - HE03Q045FUNAU1G9 [pii]
PST - ppublish
SO  - Syst Biol. 2003 Dec;52(6):849-51.

PMID- 14668120
OWN - NLM
STAT- MEDLINE
DCOM- 20040220
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 6
DP  - 2003 Dec
TI  - Novel versus unsupported clades: assessing the qualitative support for clades in 
      MRP supertrees.
PG  - 839-48
AB  - Matrix representation with parsimony (MRP) supertree construction has been
      criticized because the supertree may specify clades that are contradicted by
      every source tree contributing to it. Such unsupported clades may also occur
      using other supertree methods; however, their incidence is largely unknown. In
      this study, I investigated the frequency of unsupported clades in both simulated 
      and empirical MRP supertrees. Here, I propose a new index, QS, to quantify the
      qualitative support for a supertree and its clades among the set of source trees.
      Results show that unsupported clades are very rare in MRP supertrees, occurring
      most often when there are few source trees that all possess the same set of taxa.
      However, even under these conditions the frequency of unsupported clades was
      &lt;0.2%. Unsupported clades were absent from both the Carnivora and Lagomorpha
      supertrees, reflecting the use of large numbers of source trees for both. The
      proposed QS indices are correlated broadly with another measure of quantitative
      clade support (bootstrap frequencies, as derived from resampling of the MRP
      matrix) but appear to be more sensitive. More importantly, they sample at the
      level of the source trees and thus, unlike the bootstrap, are suitable for
      summarizing the support of MRP supertree clades.
FAU - Bininda-Emonds, Olaf R P
AU  - Bininda-Emonds OR
AD  - Lehrstuhl fur Tierzucht, Technical University of Munich, Alte Akademie 12, 85354 
      Freising-Weihenstephan, Germany. Olaf.Bininda@tierzucht.tum.de
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Carnivora/genetics
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Data Interpretation, Statistical
MH  - *Models, Theoretical
MH  - *Phylogeny
EDAT- 2003/12/12 05:00
MHDA- 2004/02/21 05:00
CRDT- 2003/12/12 05:00
PHST- 2003/12/12 05:00 [pubmed]
PHST- 2004/02/21 05:00 [medline]
PHST- 2003/12/12 05:00 [entrez]
AID - 83KHY5M8Y24XF38T [pii]
PST - ppublish
SO  - Syst Biol. 2003 Dec;52(6):839-48.

PMID- 14668119
OWN - NLM
STAT- MEDLINE
DCOM- 20040220
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 6
DP  - 2003 Dec
TI  - Reconstructing ancestral patterns of colonization and dispersal in the Hawaiian
      understory tree genus Psychotria (Rubiaceae): a comparison of parsimony and
      likelihood approaches.
PG  - 820-38
AB  - Systematic and biogeographical relationships within the Hawaiian clade of the
      pantropical understory shrub genus Psychotria (Rubiaceae) were investigated using
      phylogenetic analysis of 18S-26S ribosomal DNA internal (ITS) and external (ETS) 
      transcribed spacers. Phylogenetic analyses strongly suggest that the Hawaiian
      Psychotria are monophyletic and the result of a single introduction to the
      Hawaiian Islands. The results of phylogenetic analyses of ITS and ETS partitions 
      alone give slightly different topologies among basal lineages of the Hawaiian
      clade; however, such differences are not well supported. Relationships in the
      section Straussia clade in particular are not well resolved because of few
      nucleotide changes on internal branches, suggesting extremely rapid radiation in 
      the lineage. Parsimony and likelihood reconstructions of ancestral geographical
      distributions using the topologies inferred from both parsimony and likelihood
      analysis of combined data and using different combinations of models and branch
      lengths gave highly congruent results. However, for one internal node
      (corresponding to the majority of the "greenwelliae" clade), parsimony
      reconstructions were unable to distinguish between three possible island states, 
      whereas likelihood reconstructions resulted in clear ordering of possible states,
      with the island of Oahu slightly more probable than other islands under all but
      one model and branch length combination considered (the Jukes-Cantor-like model
      with branch lengths inferred under parsimony, under which conditions Maui Nui is 
      more probable). A pattern of colonization from oldest to youngest islands was
      inferred from the phylogeny, using maximum parsimony and maximum likelihood.
      Additionally, a much higher incidence of intraisland versus interisland
      speciation was inferred.
FAU - Nepokroeff, Molly
AU  - Nepokroeff M
AD  - Department of Biology, University of South Dakota, 414 E. Clark Street,
      Vermillion, South Dakota 57069, USA. mnepokro@usd.edu
FAU - Sytsma, Kenneth J
AU  - Sytsma KJ
FAU - Wagner, Warren L
AU  - Wagner WL
FAU - Zimmer, Elizabeth A
AU  - Zimmer EA
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Base Sequence
MH  - DNA Primers
MH  - DNA, Ribosomal/genetics
MH  - *Demography
MH  - *Evolution, Molecular
MH  - *Geography
MH  - Hawaii
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - Psychotria/*genetics
MH  - Sequence Analysis, DNA
EDAT- 2003/12/12 05:00
MHDA- 2004/02/21 05:00
CRDT- 2003/12/12 05:00
PHST- 2003/12/12 05:00 [pubmed]
PHST- 2004/02/21 05:00 [medline]
PHST- 2003/12/12 05:00 [entrez]
AID - YJH99F4F3BTJYETH [pii]
PST - ppublish
SO  - Syst Biol. 2003 Dec;52(6):820-38.

PMID- 14668118
OWN - NLM
STAT- MEDLINE
DCOM- 20040220
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 6
DP  - 2003 Dec
TI  - Phylogenetics of fanged frogs: testing biogeographical hypotheses at the
      interface of the asian and Australian faunal zones.
PG  - 794-819
AB  - The interface of the Asian and Australian faunal zones is defined by a network of
      deep ocean trenches that separate intervening islands of the Philippines and
      Wallacea (Sulawesi, the Lesser Sundas, and the Moluccas). Studies of this region 
      by Wallace marked the genesis of the field of biogeography, yet few workers have 
      used molecular methods to investigate the biogeography of taxa whose distribution
      spans this interface. Some taxa, such as the fanged frogs of the ranid genus
      Limnonectes, have distributions on either side of the zoogeographical lines of
      Wallace and Huxley, offering an opportunity to ask how frequently these purported
      barriers were crossed and by what paths. To examine diversification of
      Limnonectes in Southeast Asia, the Philippines, and Wallacea, we estimated a
      phylogeny from mitochondrial DNA sequences obtained from a robust geographic
      sample. Our analyses suggest that these frogs dispersed from Borneo to the
      Philippines at least twice, from Borneo to Sulawesi once or twice, from Sulawesi 
      to the Philippines once, and from the Philippines to Sulawesi once. Dispersal to 
      the Moluccas occurred from Sulawesi and to the Lesser Sundas from Java/Bali.
      Species distributions are generally concordant with Pleistocene aggregate island 
      complexes of the Philippines and with areas of endemism on Sulawesi. We conclude 
      that the recognition of zoogeographic lines, though insightful, may oversimplify 
      the biogeography of widespread taxa in this region.
FAU - Evans, Ben J
AU  - Evans BJ
AD  - Center for Environmental Research and Conservation and Department of Ecology,
      Evolution and Environmental Biology, Columbia University, New York, NY 10027,
      USA. bje5@columbia.edu
FAU - Brown, Rafe M
AU  - Brown RM
FAU - McGuire, Jimmy A
AU  - McGuire JA
FAU - Supriatna, Jatna
AU  - Supriatna J
FAU - Andayani, Noviar
AU  - Andayani N
FAU - Diesmos, Arvin
AU  - Diesmos A
FAU - Iskandar, Djoko
AU  - Iskandar D
FAU - Melnick, Don J
AU  - Melnick DJ
FAU - Cannatella, David C
AU  - Cannatella DC
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Australia
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - Geography
MH  - Indonesia
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Movement/*physiology
MH  - Philippines
MH  - *Phylogeny
MH  - Ranidae/anatomy &amp; histology/*genetics/physiology
MH  - Sequence Analysis, DNA
EDAT- 2003/12/12 05:00
MHDA- 2004/02/21 05:00
CRDT- 2003/12/12 05:00
PHST- 2003/12/12 05:00 [pubmed]
PHST- 2004/02/21 05:00 [medline]
PHST- 2003/12/12 05:00 [entrez]
AID - T33JWHDN15D6FACB [pii]
PST - ppublish
SO  - Syst Biol. 2003 Dec;52(6):794-819.

PMID- 14668117
OWN - NLM
STAT- MEDLINE
DCOM- 20040220
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 6
DP  - 2003 Dec
TI  - Least and most powerful phylogenetic tests to elucidate the origin of the seed
      plants in the presence of conflicting signals under misspecified models.
PG  - 781-93
AB  - Several tests of molecular phylogenies have been proposed over the last decades, 
      but most of them lead to strikingly different P-values. I propose that such
      discrepancies are principally due to different forms of null hypotheses. To
      support this hypothesis, two new tests are described. Both consider the composite
      null hypothesis that all the topologies are equidistant from the true but unknown
      topology. This composite hypothesis can either be reduced to the simple
      hypothesis at the least favorable distribution (frequentist significance test
      [FST]) or to the maximum likelihood topology (frequentist hypothesis test [FHT]).
      In both cases, the reduced null hypothesis is tested against each topology
      included in the analysis. The tests proposed have an information-theoretic
      justification, and the distribution of their test statistic is estimated by a
      nonparametric bootstrap, adjusting P-values for multiple comparisons. I applied
      the new tests to the reanalysis of two chloroplast genes, psaA and psbB, and
      compared the results with those of previously described tests. As expected, the
      FST and the FHT behaved approximately like the Shimodaira-Hasegawa test and the
      bootstrap, respectively. Although the tests give overconfidence in a wrong tree
      when an overly simple nucleotide substitution model is assumed, more complex
      models incorporating heterogeneity among codon positions resolve some conflicts. 
      To further investigate the influence of the null hypothesis, a power study was
      conducted. Simulations showed that FST and the Shimodaira-Hasegawa test are the
      least powerful and FHT is the most powerful across the parameter space. Although 
      the size of all the tests is affected by misspecification, the two new tests
      appear more robust against misspecification of the model of evolution and
      consistently supported the hypothesis that the Gnetales are nested within
      gymnosperms.
FAU - Aris-Brosou, Stephane
AU  - Aris-Brosou S
AD  - Bioinformatics Research Center, North Carolina State University, Raleigh, North
      Carolina 27695-7566, USA. stephane@statgen.ncsu.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - DNA, Chloroplast/genetics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Plants/*genetics
MH  - Statistics, Nonparametric
EDAT- 2003/12/12 05:00
MHDA- 2004/02/21 05:00
CRDT- 2003/12/12 05:00
PHST- 2003/12/12 05:00 [pubmed]
PHST- 2004/02/21 05:00 [medline]
PHST- 2003/12/12 05:00 [entrez]
AID - 0JMRPA79KF0LY5XH [pii]
PST - ppublish
SO  - Syst Biol. 2003 Dec;52(6):781-93.

PMID- 14668116
OWN - NLM
STAT- MEDLINE
DCOM- 20040220
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 6
DP  - 2003 Dec
TI  - Systematics of the lizard family pygopodidae with implications for the
      diversification of Australian temperate biotas.
PG  - 757-80
AB  - We conducted a phylogenetic study of pygopodid lizards, a group of 38 species
      endemic to Australia and New Guinea, with two major goals: to reconstruct a
      taxonomically complete and robustly supported phylogeny for the group and to use 
      this information to gain insights into the tempo, mode, and timing of the
      pygopodid radiation. Phylogenetic analyses of mitochondrial DNA (mtDNA), nuclear 
      DNA (nDNA), and previously published morphological data using parsimony, maximum 
      likelihood, and Bayesian methods on the independent and combined three data sets 
      yielded trees with similar and largely stable ingroup topologies. However,
      relationships among the six most inclusive and unambiguously supported clades
      (Aprasia, Delma, Lialis, Ophidiocephalus, Pletholax, and Pygopus) varied
      depending on data set analyzed. We used parametric bootstrapping to help us
      understand which of the three-branch schemes linking these six taxa was most
      plausible given our data. We conclude based on our results that the arrangement
      ((((Delma, Lialis)Pygopus)Pletholax)(Aprasia, Ophidiocephalus)) represents the
      best hypothesis of intergeneric relationships. A second major problem to arise in
      our study concerned the inability of our two outgroup taxa (Diplodactylus) to
      root trees properly; three different rooting locations were suggested depending
      upon analysis. This long-branch attraction problem was so severe that the
      outgroup branch also interfered with estimation of ingroup relationships. We
      therefore used the molecular clock method to root the pygopodid tree. Results of 
      two independent molecular clock analyses (mtDNA and nDNA) converged upon the same
      root location (branch leading to Delma). We are confident that we have found the 
      correct root because the possibility of our clock estimates agreeing by chance
      alone is remote given that there are 65 possible root locations (branches) on the
      pygopodid tree (approximately 1 in 65 odds). Our analysis also indicated that
      Delma fraseri is not monophyletic, a result supported by a parametric
      bootstrapping test. We elevated the Western Australian race, Delma f. petersoni, 
      to species status (i.e., Delma petersoni) because hybridization and incomplete
      lineage sorting could be ruled out as potential causes of this paraphyletic gene 
      tree and because D. grayii is broadly sympatric with its sister species D.
      fraseri. Climate changes over the past 23 million years, which transformed
      Australia from a wet, green continent to one that is largely dry and brown, have 
      been suspected as playing a major role in the diversification of Australia's
      temperate biotas. Our phylogenetic analyses of pygopodid speciation and
      biogeography revealed four important findings consistent with this climate change
      diversification model: (1) our fossil-calibrated phylogeny shows that although
      some extant pygopodid lineages predate the onset of aridification, 28 of 33
      pygopodid species included in our study seem to have originated in the last 23
      million years; (2) relative cladogenesis tests suggest that several major clades 
      underwent higher than expected rates of speciation; (3) our findings support
      earlier studies showing that speciation of mesic-adapted biotas in the
      southeastern and southwestern corners of Australia largely occurred within each
      of these regions between 12 and 23 million years ago as opposed to repeated
      dispersal between these regions; and (4) we have identified for the first time
      the existence of several pairs of sympatric sister species of lizards living in
      arid and semiarid ecosystems. These sympatric sister species seem to be younger
      than allopatric or parapatric sister-species pairs, which is not consistent with 
      previous beliefs.
FAU - Jennings, W Bryan
AU  - Jennings WB
AD  - Section of Integrative Biology, University of Texas, Austin, Texas 78712, USA.
      bjennings@oeb.harvard.edu
FAU - Pianka, Eric R
AU  - Pianka ER
FAU - Donnellan, Stephen
AU  - Donnellan S
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Australia
MH  - Base Sequence
MH  - Bayes Theorem
MH  - *Biodiversity
MH  - Climate
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - Fossils
MH  - Geography
MH  - Likelihood Functions
MH  - Lizards/anatomy &amp; histology/*genetics
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2003/12/12 05:00
MHDA- 2004/02/21 05:00
CRDT- 2003/12/12 05:00
PHST- 2003/12/12 05:00 [pubmed]
PHST- 2004/02/21 05:00 [medline]
PHST- 2003/12/12 05:00 [entrez]
AID - 79Q7XNYBXE6DX3PK [pii]
PST - ppublish
SO  - Syst Biol. 2003 Dec;52(6):757-80.

PMID- 14668115
OWN - NLM
STAT- MEDLINE
DCOM- 20040220
LR  - 20061115
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 6
DP  - 2003 Dec
TI  - Time flies, a new molecular time-scale for brachyceran fly evolution without a
      clock.
PG  - 745-56
AB  - The insect order Diptera, the true flies, contains one of the four largest
      Mesozoic insect radiations within its suborder Brachycera. Estimates of
      phylogenetic relationships and divergence dates among the major brachyceran
      lineages have been problematic or vague because of a lack of consistent evidence 
      and the rarity of well-preserved fossils. Here, we combine new evidence from
      nucleotide sequence data, morphological reinterpretations, and fossils to improve
      estimates of brachyceran evolutionary relationships and ages. The 28S ribosomal
      DNA (rDNA) gene was sequenced for a broad diversity of taxa, and the data were
      combined with recently published morphological scorings for a parsimony-based
      phylogenetic analysis. The phylogenetic topology inferred from the combined 28S
      rDNA and morphology data set supports brachyceran monophyly and the monophyly of 
      the four major brachyceran infraorders and suggests relationships largely
      consistent with previous classifications. Weak support was found for a basal
      brachyceran clade comprising the infraorders Stratiomyomorpha (soldier flies and 
      relatives), Xylophagomorpha (xylophagid flies), and Tabanomorpha (horse flies,
      snipe flies, and relatives). This topology and similar alternative arrangements
      were used to obtain Bayesian estimates of divergence times, both with and without
      the assumption of a constant evolutionary rate. The estimated times were
      relatively robust to the choice of prior distributions. Divergence times based on
      the 28S rDNA and several fossil constraints indicate that the Brachycera
      originated in the late Triassic or earliest Mesozoic and that all major lower
      brachyceran fly lineages had near contemporaneous origins in the mid-Jurassic
      prior to the origin of flowering plants (angiosperms). This study provides
      increased resolution of brachyceran phylogeny, and our revised estimates of fly
      ages should improve the temporal context of evolutionary inferences and genomic
      comparisons between fly model organisms.
FAU - Wiegmann, Brian M
AU  - Wiegmann BM
AD  - Department of Entomology, North Carolina State University, Raleigh, North
      Carolina 27695, USA. bwiegman@unity.ncsu.edu
FAU - Yeates, David K
AU  - Yeates DK
FAU - Thorne, Jeffrey L
AU  - Thorne JL
FAU - Kishino, Hirohisa
AU  - Kishino H
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA, Ribosomal/genetics
MH  - Diptera/anatomy &amp; histology/*genetics
MH  - *Evolution, Molecular
MH  - *Fossils
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
EDAT- 2003/12/12 05:00
MHDA- 2004/02/21 05:00
CRDT- 2003/12/12 05:00
PHST- 2003/12/12 05:00 [pubmed]
PHST- 2004/02/21 05:00 [medline]
PHST- 2003/12/12 05:00 [entrez]
AID - 0HWHMF75LPPG7YP6 [pii]
PST - ppublish
SO  - Syst Biol. 2003 Dec;52(6):745-56.

PMID- 14668114
OWN - NLM
STAT- MEDLINE
DCOM- 20040220
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 6
DP  - 2003 Dec
TI  - Phylogenetic systematics and evolution of primate-derived Pneumocystis based on
      mitochondrial or nuclear DNA sequence comparison.
PG  - 735-44
AB  - Previous studies have demonstrated that the agent of Pneumocystis pneumonia
      (PcP), Pneumocystis carinii, is actually a complex of eukaryotic organisms, and
      cophylogeny could explain the distribution of the hosts and parasites. In the
      present work, we tested the hypothesis of cophylogeny between the primate-derived
      Pneumocystis group and their hosts. Specific strains isolated from 20 primate
      species, including humans, were used to produce a phylogeny of the parasites.
      Aligned sequences corresponding to DNA sequences of three genes (DHPS,
      mtSSU-rRNA, and mtLSU-rRNA) were separately analyzed and then combined in a
      single data set. The resulting parasite phylogeny was compared with different
      controversial phylogenies for the hosts. This comparison demonstrated that,
      depending upon which topology is accepted for the hosts, at least 61% and perhaps
      77% of the homologous nodes of the respective cladograms of the hosts and
      parasites may be interpreted as resulting from codivergence events. This finding 
      and the high specificity of these parasites suggests that cophylogeny may be
      considered the dominant pattern of evolution for Pneumocystis organisms,
      representing a new example of parallel evolution between primates and their
      specific parasites. Because the phylogeny of Pneumocystis followed very closely
      the differentiation of their hosts at the species level, the study of the
      parasites could provide valuable information on the phylogeny of their hosts. We 
      used this information to explore controversial hypotheses of the phylogeny of the
      Platyrrhini by comparison with the phylogeny of their specific Pneumocystis
      parasites. If these organisms were closely associated as lung parasites with
      primates through the ages, the hypothesis of the Pneumocystis spp. being new
      pathogenic agents could be refuted. However, these organisms are opportunistic
      symbionts, becoming pathogenic whenever the immunological defences of their hosts
      decline. This study also provides support for the hypothesis that the different
      Pneumocystis species are genetically independent organisms, helping to clarify
      their taxonomic status.
FAU - Hugot, Jean-Pierre
AU  - Hugot JP
AD  - Museum National d'Histoire Naturelle, Departement Systematique et Evolution, UMS 
      Origine, Structure et Evolution de la Biodiversite, 75 231 Paris Cedex 05,
      France. hugot@cimrs1.mnhn.fr
FAU - Demanche, Christine
AU  - Demanche C
FAU - Barriel, Veronique
AU  - Barriel V
FAU - Dei-Cas, Eduardo
AU  - Dei-Cas E
FAU - Guillot, Jacques
AU  - Guillot J
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (RNA, Ribosomal)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - DNA Primers
MH  - DNA, Mitochondrial/genetics
MH  - Humans
MH  - Likelihood Functions
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Pneumocystis carinii/*genetics/*physiology
MH  - Primates/*genetics/*microbiology
MH  - RNA, Ribosomal/genetics
MH  - Sequence Alignment
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2003/12/12 05:00
MHDA- 2004/02/21 05:00
CRDT- 2003/12/12 05:00
PHST- 2003/12/12 05:00 [pubmed]
PHST- 2004/02/21 05:00 [medline]
PHST- 2003/12/12 05:00 [entrez]
AID - 9CDCTNEFXK21YATD [pii]
AID - 10.1080/10635150390250893 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Dec;52(6):735-44. doi: 10.1080/10635150390250893.

PMID- 14530139
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - Supertrees are a necessary not-so-evil: a comment on Gatesy et al.
PG  - 724-9
FAU - Bininda-Emonds, Olaf R P
AU  - Bininda-Emonds OR
AD  - Lehrstuhl fur Tierzucht, Technical University of Munich, Alte Akademie 12, 85354 
      Freising-Weihenstephan, Germany. olaf.bininda@tierzucht.tum.de
FAU - Jones, Kate E
AU  - Jones KE
FAU - Price, Samantha A
AU  - Price SA
FAU - Grenyer, Richard
AU  - Grenyer R
FAU - Cardillo, Marcel
AU  - Cardillo M
FAU - Habib, Michael
AU  - Habib M
FAU - Purvis, Andy
AU  - Purvis A
FAU - Gittleman, John L
AU  - Gittleman JL
LA  - eng
PT  - Comment
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
CON - Syst Biol. 2002 Aug;51(4):652-64. PMID: 12228007
CIN - Syst Biol. 2004 Apr;53(2):342-55. PMID: 15205058
MH  - *Meta-Analysis as Topic
MH  - *Phylogeny
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - 7VHGLGT7C79WE6PQ [pii]
AID - 10.1080/10635150390235647 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):724-9. doi: 10.1080/10635150390235647.

PMID- 14530138
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - Biotic element analysis in biogeography.
PG  - 717-23
AB  - Biotic element analysis is an alternative to the areas-of-endemism approach for
      recognizing the presence or absence of vicariance events in a given region. If an
      ancestral biota was fragmented by vicariance events, biotic elements or clusters 
      of distribution areas should emerge. We propose a statistical test for clustering
      of distribution areas based on a Monte Carlo simulation with a null model that
      considers the spatial autocorrelation in the data. The hypothesis tested is that 
      the observed degree of clustering of ranges can be explained by the range size
      distribution, the varying number of taxa per cell, and the spatial
      autocorrelation of the occurrences of a taxon alone. A method for the
      delimitation of biotic elements which uses model-based Gaussian clustering is
      introduced. We demonstrate our methods and show the importance of grid size by
      means of a case study, an analysis of the distribution patterns of southern
      African species of the weevil genus Scobius. The example highlights the
      difficulties in delimiting areas of endemism if dispersal has occurred and
      illustrates the advantages of the biotic element approach.
FAU - Hausdorf, Bernhard
AU  - Hausdorf B
AD  - Zoologisches Museum der Universitat Hamburg, Martin-Luther-King-Platz 3, 20146
      Hamburg, Germany. hausdorf@zoologie.uni-hamburg.de
FAU - Hennig, Christian
AU  - Hennig C
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biodiversity
MH  - Coleoptera/physiology
MH  - *Geography
MH  - *Models, Theoretical
MH  - Monte Carlo Method
MH  - Normal Distribution
MH  - *Phylogeny
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - YFK9BN4M3FLMCY9E [pii]
AID - 10.1080/10635150390235584 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):717-23. doi: 10.1080/10635150390235584.

PMID- 14530137
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - Comparison of likelihood and Bayesian methods for estimating divergence times
      using multiple gene Loci and calibration points, with application to a radiation 
      of cute-looking mouse lemur species.
PG  - 705-16
AB  - Divergence time and substitution rate are seriously confounded in phylogenetic
      analysis, making it difficult to estimate divergence times when the molecular
      clock (rate constancy among lineages) is violated. This problem can be alleviated
      to some extent by analyzing multiple gene loci simultaneously and by using
      multiple calibration points. While different genes may have different patterns of
      evolutionary rate change, they share the same divergence times. Indeed, the fact 
      that each gene may violate the molecular clock differently leads to the advantage
      of simultaneous analysis of multiple loci. Multiple calibration points provide
      the means for characterizing the local evolutionary rates on the phylogeny. In
      this paper, we extend previous likelihood models of local molecular clock for
      estimating species divergence times to accommodate multiple calibration points
      and multiple genes. Heterogeneity among different genes in evolutionary rate and 
      in substitution process is accounted for by the models. We apply the likelihood
      models to analyze two mitochondrial protein-coding genes, cytochrome oxidase II
      and cytochrome b, to estimate divergence times of Malagasy mouse lemurs and
      related outgroups. The likelihood method is compared with the Bayes method of
      Thorne et al. (1998, Mol. Biol. Evol. 15:1647-1657), which uses a probabilistic
      model to describe the change in evolutionary rate over time and uses the Markov
      chain Monte Carlo procedure to derive the posterior distribution of rates and
      times. Our likelihood implementation has the drawbacks of failing to accommodate 
      uncertainties in fossil calibrations and of requiring the researcher to classify 
      branches on the tree into different rate groups. Both problems are avoided in the
      Bayes method. Despite the differences in the two methods, however, data
      partitions and model assumptions had the greatest impact on date estimation. The 
      three codon positions have very different substitution rates and evolutionary
      dynamics, and assumptions in the substitution model affect date estimation in
      both likelihood and Bayes analyses. The results demonstrate that the separate
      analysis is unreliable, with dates variable among codon positions and between
      methods, and that the combined analysis is much more reliable. When the three
      codon positions were analyzed simultaneously under the most realistic models
      using all available calibration information, the two methods produced similar
      results. The divergence of the mouse lemurs is dated to be around 7-10 million
      years ago, indicating a surprisingly early species radiation for such a
      morphologically uniform group of primates.
FAU - Yang, Ziheng
AU  - Yang Z
AD  - Department of Biology, University College London, Darwin Building, Gower Street, 
      London WC1E 6BT, United Kingdom. z.yang@ucl.ac.uk
FAU - Yoder, Anne D
AU  - Yoder AD
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Bayes Theorem
MH  - Cheirogaleidae/genetics
MH  - *Evolution, Molecular
MH  - *Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Sequence Alignment
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - UKP67HVU98L8MFPE [pii]
AID - 10.1080/10635150390235557 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):705-16. doi: 10.1080/10635150390235557.

PMID- 14530136
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - A simple, fast, and accurate algorithm to estimate large phylogenies by maximum
      likelihood.
PG  - 696-704
AB  - The increase in the number of large data sets and the complexity of current
      probabilistic sequence evolution models necessitates fast and reliable phylogeny 
      reconstruction methods. We describe a new approach, based on the maximum-
      likelihood principle, which clearly satisfies these requirements. The core of
      this method is a simple hill-climbing algorithm that adjusts tree topology and
      branch lengths simultaneously. This algorithm starts from an initial tree built
      by a fast distance-based method and modifies this tree to improve its likelihood 
      at each iteration. Due to this simultaneous adjustment of the topology and branch
      lengths, only a few iterations are sufficient to reach an optimum. We used
      extensive and realistic computer simulations to show that the topological
      accuracy of this new method is at least as high as that of the existing
      maximum-likelihood programs and much higher than the performance of
      distance-based and parsimony approaches. The reduction of computing time is
      dramatic in comparison with other maximum-likelihood packages, while the
      likelihood maximization ability tends to be higher. For example, only 12 min were
      required on a standard personal computer to analyze a data set consisting of 500 
      rbcL sequences with 1,428 base pairs from plant plastids, thus reaching a speed
      of the same order as some popular distance-based and parsimony algorithms. This
      new method is implemented in the PHYML program, which is freely available on our 
      web page: http://www.lirmm.fr/w3ifa/MAAS/.
FAU - Guindon, Stephane
AU  - Guindon S
AD  - LIRMM, CNRS, 161 Rue Ada, 34392, Montpellier Cedex 5, France.
FAU - Gascuel, Olivier
AU  - Gascuel O
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - *Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - 54QHX07WB5K5XCX4 [pii]
AID - 10.1080/10635150390235520 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):696-704. doi: 10.1080/10635150390235520.

PMID- 14530135
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - Canonical phylogenetic ordination.
PG  - 684-95
AB  - A phylogenetic comparative method is proposed for estimating historical effects
      on comparative data using the partitions that compose a cladogram, i.e., its
      monophyletic groups. Two basic matrices, Y and X, are defined in the context of
      an ordinary linear model. Y contains the comparative data measured over t taxa. X
      consists of an initial tree matrix that contains all the xj monophyletic groups
      (each coded separately as a binary indicator variable) of the phylogenetic tree
      available for those taxa. The method seeks to define the subset of groups, i.e., 
      a reduced tree matrix, that best explains the patterns in Y. This definition is
      accomplished via regression or canonical ordination (depending on the
      dimensionality of Y) coupled with Monte Carlo permutations. It is argued here
      that unrestricted permutations (i.e., under an equiprobable model) are valid for 
      testing this specific kind of groupwise hypothesis. Phylogeny is either
      partialled out or, more properly, incorporated into the analysis in the form of
      component variation. Direct extensions allow for testing ecomorphological data
      controlled by phylogeny in a variation partitioning approach. Currently available
      statistical techniques make this method applicable under most
      univariate/multivariate models and metrics; two-way phylogenetic effects can be
      estimated as well. The simplest case (univariate Y), tested with simulations,
      yielded acceptable type I error rates. Applications presented include examples
      from evolutionary ethology, ecology, and ecomorphology. Results showed that the
      new technique detected previously overlooked variation clearly associated with
      phylogeny and that many phylogenetic effects on comparative data may occur at
      particular groups rather than across the entire tree.
FAU - Giannini, Norberto P
AU  - Giannini NP
AD  - Division of Vertebrate Zoology, Department of Mammalogy, American Museum of
      Natural History, Central Park West at 79th Street, New York, New York 10024-5192,
      USA. norberto@amnh.org
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Data Interpretation, Statistical
MH  - Ecosystem
MH  - Lizards/genetics
MH  - *Models, Theoretical
MH  - *Phylogeny
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - U2Q88UUAGT8ULL6L [pii]
AID - 10.1080/10635150390238888 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):684-95. doi: 10.1080/10635150390238888.

PMID- 14530134
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - Performance-based selection of likelihood models for phylogeny estimation.
PG  - 674-83
AB  - Phylogenetic estimation has largely come to rely on explicitly model-based
      methods. This approach requires that a model be chosen and that that choice be
      justified. To date, justification has largely been accomplished through use of
      likelihood-ratio tests (LRTs) to assess the relative fit of a nested series of
      reversible models. While this approach certainly represents an important advance 
      over arbitrary model selection, the best fit of a series of models may not always
      provide the most reliable phylogenetic estimates for finite real data sets, where
      all available models are surely incorrect. Here, we develop a novel approach to
      model selection, which is based on the Bayesian information criterion, but
      incorporates relative branch-length error as a performance measure in a decision 
      theory (DT) framework. This DT method includes a penalty for overfitting, is
      applicable prior to running extensive analyses, and simultaneously compares all
      models being considered and thus does not rely on a series of pairwise
      comparisons of models to traverse model space. We evaluate this method by
      examining four real data sets and by using those data sets to define simulation
      conditions. In the real data sets, the DT method selects the same or simpler
      models than conventional LRTs. In order to lend generality to the simulations,
      codon-based models (with parameters estimated from the real data sets) were used 
      to generate simulated data sets, which are therefore more complex than any of the
      models we evaluate. On average, the DT method selects models that are simpler
      than those chosen by conventional LRTs. Nevertheless, these simpler models
      provide estimates of branch lengths that are more accurate both in terms of
      relative error and absolute error than those derived using the more complex (yet 
      still wrong) models chosen by conventional LRTs. This method is available in a
      program called DT-ModSel.
FAU - Minin, Vladimir
AU  - Minin V
AD  - Department of Methematics, PO Box 441103, University of Idaho, Moscow, Idaho
      83844-1103, USA.
FAU - Abdo, Zaid
AU  - Abdo Z
FAU - Joyce, Paul
AU  - Joyce P
FAU - Sullivan, Jack
AU  - Sullivan J
LA  - eng
GR  - 1P20RR016448-01/RR/NCRR NIH HHS/United States
GR  - GM38737/GM/NIGMS NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Computer Simulation
MH  - *Decision Support Techniques
MH  - *Evolution, Molecular
MH  - *Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - MC94J8EQ777DMDD5 [pii]
AID - 10.1080/10635150390235494 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):674-83. doi: 10.1080/10635150390235494.

PMID- 14530133
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - Reliability of Bayesian posterior probabilities and bootstrap frequencies in
      phylogenetics.
PG  - 665-73
AB  - Many empirical studies have revealed considerable differences between
      nonparametric bootstrapping and Bayesian posterior probabilities in terms of the 
      support values for branches, despite claimed predictions about their approximate 
      equivalence. We investigated this problem by simulating data, which were then
      analyzed by maximum likelihood bootstrapping and Bayesian phylogenetic analysis
      using identical models and reoptimization of parameter values. We show that
      Bayesian posterior probabilities are significantly higher than corresponding
      nonparametric bootstrap frequencies for true clades, but also that erroneous
      conclusions will be made more often. These errors are strongly accentuated when
      the models used for analyses are underparameterized. When data are analyzed under
      the correct model, nonparametric bootstrapping is conservative. Bayesian
      posterior probabilities are also conservative in this respect, but less so.
FAU - Erixon, Per
AU  - Erixon P
AD  - Department of Systematic Botany, Evolutionary Biology Centre, Uppsala University,
      Norbyv. 18D, SE-75236 Uppsala, Sweden. per.erixon@ebc.uu.se
FAU - Svennblad, Bodil
AU  - Svennblad B
FAU - Britton, Tom
AU  - Britton T
FAU - Oxelman, Bengt
AU  - Oxelman B
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Bayes Theorem
MH  - Computer Simulation
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Reproducibility of Results
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - BHHVPETMJCXMX8E4 [pii]
AID - 10.1080/10635150390235485 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):665-73. doi: 10.1080/10635150390235485.

PMID- 14530132
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - Hierarchical phylogenetic models for analyzing multipartite sequence data.
PG  - 649-64
AB  - Debate exists over how to incorporate information from multipartite sequence data
      in phylogenetic analyses. Strict combined-data approaches argue for concatenation
      of all partitions and estimation of one evolutionary history, maximizing the
      explanatory power of the data. Consensus/independence approaches endorse a
      two-step procedure where partitions are analyzed independently and then a
      consensus is determined from the multiple results. Mixtures across the model
      space of a strict combined-data approach and a priori independent parameters are 
      popular methods to integrate these methods. We propose an alternative middle
      ground by constructing a Bayesian hierarchical phylogenetic model. Our
      hierarchical framework enables researchers to pool information across data
      partitions to improve estimate precision in individual partitions while
      permitting estimation and testing of tendencies in across-partition quantities.
      Such across-partition quantities include the distribution from which individual
      topologies relating the sequences within a partition are drawn. We propose
      standard hierarchical priors on continuous evolutionary parameters across
      partitions, while the structure on topologies varies depending on the research
      problem. We illustrate our model with three examples. We first explore the
      evolutionary history of the guinea pig (Cavia porcellus) using alignments of 13
      mitochondrial genes. The hierarchical model returns substantially more precise
      continuous parameter estimates than an independent parameter approach without
      losing the salient features of the data. Second, we analyze the frequency of
      horizontal gene transfer using 50 prokaryotic genes. We assume an unknown
      species-level topology and allow individual gene topologies to differ from this
      with a small estimable probability. Simultaneously inferring the species and
      individual gene topologies returns a transfer frequency of 17%. We also examine
      HIV sequences longitudinally sampled from HIV+ patients. We ask whether
      posttreatment development of CCR5 coreceptor virus represents concerted evolution
      from middisease CXCR4 virus or reemergence of initial infecting CCR5 virus. The
      hierarchical model pools partitions from multiple unrelated patients by assuming 
      that the topology for each patient is drawn from a multinomial distribution with 
      unknown probabilities. Preliminary results suggest evolution and not reemergence.
FAU - Suchard, Marc A
AU  - Suchard MA
AD  - Department of Biomathematics, David Geffen School of Medicine, University of
      California-Los Angeles, Los Angeles, California 90095-1766, USA.
      msuchard@ucla.edu
FAU - Kitchen, Christina M R
AU  - Kitchen CM
FAU - Sinsheimer, Janet S
AU  - Sinsheimer JS
FAU - Weiss, Robert E
AU  - Weiss RE
LA  - eng
GR  - AI28697/AI/NIAID NIH HHS/United States
GR  - CA16042/CA/NCI NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Receptors, CCR5)
SB  - IM
MH  - Animals
MH  - Bacteria/genetics
MH  - *Bayes Theorem
MH  - DNA, Mitochondrial/genetics
MH  - *Evolution, Molecular
MH  - Guinea Pigs
MH  - HIV/genetics
MH  - Humans
MH  - Mammals/genetics
MH  - *Models, Theoretical
MH  - *Phylogeny
MH  - Receptors, CCR5/genetics
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - 4J39QWQNT7JX2R6A [pii]
AID - 10.1080/10635150390238879 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):649-64. doi: 10.1080/10635150390238879.

PMID- 14530131
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - Frequent inconsistency of parsimony under a simple model of cladogenesis.
PG  - 641-8
AB  - Although the conditions under which the parsimony method becomes inconsistent
      have been studied for almost two decades, the probability that the parsimony
      method would encounter conditions causing inconsistency under simple models of
      cladogenesis is unknown. Here, we examine the statistical behavior of the
      parsimony method under a birth-death model of cladogenesis, when the molecular
      clock holds. The parsimony method can become inconsistent a high proportion of
      the time even under this simple model of cladogenesis. When taxon sampling is
      poor or rates of evolution are high, the probability that parsimony will become
      inconsistent increases.
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
AD  - Section of Ecology, Behavior and Evolution, Division of Biological Sciences,
      University of California-San Diego, La Jolla, California 92093-0116, USA.
      johnh@biomail.ucsd.edu
FAU - Lander, Katherine M
AU  - Lander KM
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Evolution, Molecular
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - UT9TG334P4JJRP21 [pii]
AID - 10.1080/10635150390235467 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):641-8. doi: 10.1080/10635150390235467.

PMID- 14530130
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - Phylogenetic relationships among calyptraeid gastropods and their implications
      for the biogeography of marine speciation.
PG  - 618-40
AB  - Although calyptraeid gastropods are not well understood taxonomically, in part
      because their simple plastic shells are the primary taxonomic character, they
      provide an ideal system to examine questions about evolution in the marine
      environment. I conducted a phylogenetic analysis of calyptraeid gastropods using 
      DNA sequence data from mitochondrial cytochrome oxidase I (COI) and 16S genes and
      the nuclear 28S gene. The resultant phylogeny was used to examine the
      biogeographic patterns of speciation in the Calyptraeidae. Parsimony and Bayesian
      analyses of the combined data sets for 94 calyptraeid operational taxonomic units
      and 24 outgroups produced well-resolved phylogenies. Both approaches resulted in 
      identical sister-species relationships, and the few differences in deeper
      topology did not affect biogeographic inferences. The geographic distribution of 
      the species included here demonstrate numerous dispersal events both between the 
      Pacific and Atlantic oceans and across the equator. When parsimony is used to
      reconstruct the movement from the Pacific to the Atlantic oceans on the
      phylogeny, there are 12 transitions between oceans, primarily from the Pacific to
      the Atlantic. When the latitude is coded as north versus south of the equator,
      the most-parsimonious reconstruction gives the origin of calyptraeids in the
      north followed by 15 dispersal events to regions south of the equator and no
      returns to the north. Many clades of the most closely related species are either 
      sympatric or occur along a single coastline. Closely related species can,
      however, occur in such divergent regions as Southern California and South Africa.
      There is little evidence for sister-species pairs or larger clades having been
      split by the Isthmus of Panama or the Benguela upwelling, but the East Pacific
      Barrier appears to separate the most basal taxa from the rest of the family.
FAU - Collin, Rachel
AU  - Collin R
AD  - Committee on Evolutionary Biology, University of Chicago, Culver Hall, 1025 E.
      57th Street, Chicago, Illinois 60637, USA. collinr@naos.si.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - *Geography
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - Movement/physiology
MH  - Oceans and Seas
MH  - *Phylogeny
MH  - Population Dynamics
MH  - Sequence Analysis, DNA
MH  - Snails/*classification/*genetics
MH  - Species Specificity
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - J2B7WGPNWFHV5XLW [pii]
AID - 10.1080/10635150390235430 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):618-40. doi: 10.1080/10635150390235430.

PMID- 14530129
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - Identifying conflicting signal in a multigene analysis reveals a highly resolved 
      tree: the phylogeny of Rodentia (Mammalia).
PG  - 604-17
AB  - Homoplasy among morphological characters has hindered inference of higher level
      rodent phylogeny for over 100 years. Initial molecular studies, based primarily
      on single genes, likewise produced little resolution of the deep relationships
      among rodent families. Two recent molecular studies (Huchon et al., 2002, Mol.
      Biol. Evol. 19:1053-1065; Adkins et al., 2003, Mol. Phylogenet. Evol.
      26:409-420), using larger samples from the nuclear genome, have produced
      phylogenies that are generally concordant with each other, but many of the deep
      superfamilial nodes were still lacking substantial statistical support. Data are 
      presented here for a total of approximately 3,600 base pairs from portions of
      three different nuclear protein-coding genes, CB1, IRBP, and RAG2, from 19
      rodents and three outgroups. Separate analyses, with data partitioned according
      to both genes and codon position, produced conflicting results. Trees obtained
      from all partitions of CB1 and RAG2 and those obtained from the first- plus
      second-position sites of IRBP were generally concordant with each other and the
      trees from the two recent studies, whereas trees obtained from the third-position
      sites of IRBP were not. Although the IRBP third-position sites represent only 1/9
      of the total data set, combined analyses using either parsimony or likelihood
      resulted in trees in agreement with the IRBP third-position sites and in
      disagreement with the remaining 8/9 of the sites from this data set and the two
      recent multigene studies. In contrast, maximum-likelihood analysis using a
      site-specific rates model did recover a tree that is highly congruent with the
      trees in the two recent studies. If the IRBP third-position sites are removed
      from the current data set, then combined likelihood analyses obtain a tree that
      is highly congruent with those of the two recent studies. This analysis also
      provides, for the first time in a study of rodent phylogeny, robust statistical
      support for every bipartition, with just one exception. This tree divides rodents
      into two major clades. The first contains Myodonta (Muroidea plus Dipodidae) and 
      the only unresolved trichotomy, from which descend Geomyoidea, Pedetidae, and
      Castoridae. On the other side of the root is a clade containing Sciuroidea plus
      Gliridae, and Hystricognathi. Some uncertainty remains on the placement of the
      root. Trees on which the Hystricognathi are the basal sister group to Myodonta,
      Geomyoidea, Pedetidae, and Castoridae are also found within a Bayesian 95%
      credible set, as estimated by Metropolis-coupled Markov chain Monte Carlo
      sampling.
FAU - DeBry, Ronald W
AU  - DeBry RW
AD  - Department of Biological Sciences, University of Cincinnati, Box 210006,
      Cincinnati, Ohio 45221-0006, USA. ron.debry@uc.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - DNA Primers
MH  - *Evolution, Molecular
MH  - Likelihood Functions
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Molecular Sequence Data
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Rodentia/*genetics
MH  - Sequence Analysis, DNA
MH  - Species Specificity
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - 4W29J3144M3AFCWC [pii]
AID - 10.1080/10635150390235403 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):604-17. doi: 10.1080/10635150390235403.

PMID- 14530128
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - Estimation of rates-across-sites distributions in phylogenetic substitution
      models.
PG  - 594-603
AB  - Previous work has shown that it is often essential to account for the variation
      in rates at different sites in phylogenetic models in order to avoid phylogenetic
      artifacts such as long branch attraction. In most current models, the gamma
      distribution is used for the rates-across-sites distributions and is implemented 
      as an equal-probability discrete gamma. In this article, we introduce discrete
      distribution estimates with large numbers of equally spaced rate categories
      allowing us to investigate the appropriateness of the gamma model. With large
      numbers of rate categories, these discrete estimates are flexible enough to
      approximate the shape of almost any distribution. Likelihood ratio statistical
      tests and a nonparametric bootstrap confidence-bound estimation procedure based
      on the discrete estimates are presented that can be used to test the fit of a
      parametric family. We applied the methodology to several different protein data
      sets, and found that although the gamma model often provides a good parametric
      model for this type of data, rate estimates from an equal-probability discrete
      gamma model with a small number of categories will tend to underestimate the
      largest rates. In cases when the gamma model assumption is in doubt, rate
      estimates coming from the discrete rate distribution estimate with a large number
      of rate categories provide a robust alternative to gamma estimates. An
      alternative implementation of the gamma distribution is proposed that, for equal 
      numbers of rate categories, is computationally more efficient during optimization
      than the standard gamma implementation and can provide more accurate estimates of
      site rates.
FAU - Susko, Edward
AU  - Susko E
AD  - Department of Mathematics and Statistics, Dalhousie University, Halifax, Nova
      Scotia B3H 3J5, Canada. susko@mathstat.dal.ca
FAU - Field, Chris
AU  - Field C
FAU - Blouin, Christian
AU  - Blouin C
FAU - Roger, Andrew J
AU  - Roger AJ
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (HSP70 Heat-Shock Proteins)
SB  - IM
MH  - *Evolution, Molecular
MH  - HSP70 Heat-Shock Proteins/genetics
MH  - Likelihood Functions
MH  - Markov Chains
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - APHHENJKR7719V6P [pii]
AID - 10.1080/10635150390235395 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):594-603. doi: 10.1080/10635150390235395.

PMID- 14530127
OWN - NLM
STAT- MEDLINE
DCOM- 20031224
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 5
DP  - 2003 Oct
TI  - Type I STS markers are more informative than cytochrome B in phylogenetic
      reconstruction of the Mustelidae (Mammalia: Carnivora).
PG  - 571-93
AB  - We compared the utility of five nuclear gene segments amplified with type I
      sequence-tagged site (STS) primers versus the complete mitochondrial cytochrome b
      (cyt b) gene in resolving phylogenetic relationships within the Mustelidae, a
      large and ecomorphologically diverse family of mammalian carnivores. Maximum
      parsimony and likelihood analyses of separate and combined data sets were used to
      address questions regarding the levels of homoplasy, incongruence, and
      information content within and among loci. All loci showed limited resolution in 
      the separate analyses because of either a low amount of informative variation
      (nuclear genes) or high levels of homoplasy (cyt b). Individually or combined,
      the nuclear gene sequences had less homoplasy, retained more signal, and were
      more decisive, even though cyt b contained more potentially informative variation
      than all the nuclear sequences combined. We obtained a well-resolved and
      supported phylogeny when the nuclear sequences were combined. Maximum likelihood 
      and Bayesian phylogenetic analyses of the total combined data (nuclear and
      mitochondrial DNA sequences) were able to better accommodate the high levels of
      homoplasy in the cyt b data than was an equally weighted maximum parsimony
      analysis. Furthermore, partition Bremer support analyses of the total combined
      tree showed that the relative support of the nuclear and mitochondrial genes
      differed according to whether or not the homoplasy in the cyt b gene was
      downweighted. Although the cyt b gene contributed phylogenetic signal for most
      major groupings, the nuclear gene sequences were more effective in reconstructing
      the deeper nodes of the combined tree in the equally weighted parsimony analysis,
      as judged by the variable-length bootstrap method. The total combined data
      supported the monophyly of the Lutrinae (otters), whereas the Melinae (badgers)
      and Mustelinae (weasels, martens) were both paraphyletic. The American badger,
      Taxidea taxus (Taxidiinae), was the most basal taxon. Because hundreds of type I 
      STS primer sets spanning the complete genomes of the human and mouse have been
      published and thus represent many independently segregating loci, the potential
      utility of these markers for molecular systematics of mammals and other groups is
      enormous.
FAU - Koepfli, Klaus-Peter
AU  - Koepfli KP
AD  - Department of Organismic Biology, Ecology and Evolution, 621 Charles E. Young
      Drive South, University of California, Los Angeles, California 90095-1606, USA.
      klausk@lifesci.ucla.edu
FAU - Wayne, Robert K
AU  - Wayne RK
LA  - eng
SI  - GENBANK/AF498107
SI  - GENBANK/AF498108
SI  - GENBANK/AF498109
SI  - GENBANK/AF498110
SI  - GENBANK/AF498111
SI  - GENBANK/AF498112
SI  - GENBANK/AF498113
SI  - GENBANK/AF498114
SI  - GENBANK/AF498115
SI  - GENBANK/AF498116
SI  - GENBANK/AF498117
SI  - GENBANK/AF498118
SI  - GENBANK/AF498119
SI  - GENBANK/AF498120
SI  - GENBANK/AF498121
SI  - GENBANK/AF498122
SI  - GENBANK/AF498123
SI  - GENBANK/AF498124
SI  - GENBANK/AF498125
SI  - GENBANK/AF498126
SI  - GENBANK/AF498127
SI  - GENBANK/AF498128
SI  - GENBANK/AF498129
SI  - GENBANK/AF498130
SI  - GENBANK/AF498131
SI  - GENBANK/AF498132
SI  - GENBANK/AF498133
SI  - GENBANK/AF498134
SI  - GENBANK/AF498135
SI  - GENBANK/AF498136
SI  - GENBANK/AF498137
SI  - GENBANK/AF498138
SI  - GENBANK/AF498139
SI  - GENBANK/AF498140
SI  - GENBANK/AF498141
SI  - GENBANK/AF498142
SI  - GENBANK/AF498143
SI  - GENBANK/AF498144
SI  - GENBANK/AF498145
SI  - GENBANK/AF498146
SI  - GENBANK/AF498147
SI  - GENBANK/AF498148
SI  - GENBANK/AF498149
SI  - GENBANK/AF498150
SI  - GENBANK/AF498151
SI  - GENBANK/AF498152
SI  - GENBANK/AF498153
SI  - GENBANK/AF498154
SI  - GENBANK/AF498155
SI  - GENBANK/AF498156
SI  - GENBANK/AF498157
SI  - GENBANK/AF498158
SI  - GENBANK/AF498159
SI  - GENBANK/AF498160
SI  - GENBANK/AF498161
SI  - GENBANK/AF498162
SI  - GENBANK/AF498163
SI  - GENBANK/AF498164
SI  - GENBANK/AF498165
SI  - GENBANK/AF498166
SI  - GENBANK/AF498167
SI  - GENBANK/AF498168
SI  - GENBANK/AF498169
SI  - GENBANK/AF498170
SI  - GENBANK/AF498171
SI  - GENBANK/AF498172
SI  - GENBANK/AF498173
SI  - GENBANK/AF498174
SI  - GENBANK/AF498175
SI  - GENBANK/AF498176
SI  - GENBANK/AF498177
SI  - GENBANK/AF498178
SI  - GENBANK/AF498179
SI  - GENBANK/AF498180
SI  - GENBANK/AF498181
SI  - GENBANK/AF498182
SI  - GENBANK/AF498183
SI  - GENBANK/AF498184
SI  - GENBANK/AF498185
SI  - GENBANK/AF498186
SI  - GENBANK/AF498187
SI  - GENBANK/AF498188
SI  - GENBANK/AF498189
SI  - GENBANK/AF498190
SI  - GENBANK/AF498191
SI  - GENBANK/AF498192
SI  - GENBANK/AF498193
SI  - GENBANK/AF498194
SI  - GENBANK/AF498195
SI  - GENBANK/AF498196
SI  - GENBANK/AF498197
SI  - GENBANK/AF498198
SI  - GENBANK/AF498199
SI  - GENBANK/AF498200
SI  - GENBANK/AF498201
SI  - GENBANK/AF498202
SI  - GENBANK/AF498203
SI  - GENBANK/AF498204
SI  - GENBANK/AF498205
SI  - GENBANK/AF498206
SI  - GENBANK/AF498207
SI  - GENBANK/AF498208
SI  - GENBANK/AF498209
SI  - GENBANK/AF498210
SI  - GENBANK/AF498211
SI  - GENBANK/AF498212
SI  - GENBANK/AF498213
SI  - GENBANK/AF498214
SI  - GENBANK/AF498215
SI  - GENBANK/AF498216
SI  - GENBANK/AF498217
SI  - GENBANK/AF498218
SI  - GENBANK/AF498219
SI  - GENBANK/AF498220
SI  - GENBANK/AF498221
SI  - GENBANK/AF498222
SI  - GENBANK/AF498223
SI  - GENBANK/AF498224
SI  - GENBANK/AF498225
SI  - GENBANK/AF498226
SI  - GENBANK/AF498227
SI  - GENBANK/AF498228
SI  - GENBANK/AF498229
SI  - GENBANK/AF498230
SI  - GENBANK/AF498231
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA Primers)
RN  - 9035-37-4 (Cytochromes b)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Bayes Theorem
MH  - Carnivora/*genetics
MH  - Cytochromes b/*genetics
MH  - DNA Primers
MH  - Likelihood Functions
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - *Sequence Tagged Sites
MH  - Species Specificity
EDAT- 2003/10/08 05:00
MHDA- 2003/12/25 05:00
CRDT- 2003/10/08 05:00
PHST- 2003/10/08 05:00 [pubmed]
PHST- 2003/12/25 05:00 [medline]
PHST- 2003/10/08 05:00 [entrez]
AID - GXPX00DCAF903VVC [pii]
AID - 10.1080/10635150390235368 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Oct;52(5):571-93. doi: 10.1080/10635150390235368.

PMID- 12857646
OWN - NLM
STAT- MEDLINE
DCOM- 20030916
LR  - 20190513
IS  - 1063-5157 (Print)
IS  - 1063-5157 (Linking)
VI  - 52
IP  - 4
DP  - 2003 Aug
TI  - Stability in phylogenetic formulations and its relationship to nodal support.
PG  - 554-64
FAU - Giribet, Gonzalo
AU  - Giribet G
AD  - Museum of Comparative Zoology, Department of Organismic &amp; Evolutionary Biology,
      Harvard University, 16 Divinity Avenue, Cambridge, Massachusetts 02138, USA.
      ggiribet@oeb.harvard.edu
LA  - eng
PT  - Comparative Study
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Arthropods/classification
MH  - Data Interpretation, Statistical
MH  - Models, Genetic
MH  - Mollusca/classification
MH  - *Phylogeny
MH  - Research Design
MH  - Sequence Alignment
EDAT- 2003/07/15 05:00
MHDA- 2003/09/17 05:00
CRDT- 2003/07/15 05:00
PHST- 2003/07/15 05:00 [pubmed]
PHST- 2003/09/17 05:00 [medline]
PHST- 2003/07/15 05:00 [entrez]
AID - DTPNBQ6F44Q6DYCC [pii]
AID - 10.1080/10635150390223730 [doi]
PST - ppublish
SO  - Syst Biol. 2003 Aug;52(4):554-64. doi: 10.1080/10635150390223730.
</pre></body></html>