<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0035)https://www.ncbi.nlm.nih.gov/pubmed -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"></head><body><pre>PMID- 28173480
OWN - NLM
STAT- MEDLINE
DCOM- 20180213
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Incomplete Lineage Sorting in Mammalian Phylogenomics.
PG  - 112-120
LID - 10.1093/sysbio/syw082 [doi]
AB  - The impact of incomplete lineage sorting (ILS) on phylogenetic conflicts among
      genes, and the related issue of whether to account for ILS in species tree
      reconstruction, are matters of intense controversy. Here, focusing on full-genome
      data in placental mammals, we empirically test two assumptions underlying current
      usage of tree-building methods that account for ILS. We show that in this data
      set (i) distinct exons from a common gene do not share a common genealogy, and
      (ii) ILS is only a minor determinant of the existing phylogenetic conflict. These
      results shed new light on the relevance and conditions of applicability of
      ILS-aware methods in phylogenomic analyses of protein coding sequences.
FAU - Scornavacca, Celine
AU  - Scornavacca C
AD  - UMR 5554-Institute of Evolutionary Sciences, University Montpellier, CNRS, IRD,
      EPHE, Place E. Bataillon-CC64, Montpellier, France.
FAU - Galtier, Nicolas
AU  - Galtier N
AD  - UMR 5554-Institute of Evolutionary Sciences, University Montpellier, CNRS, IRD,
      EPHE, Place E. Bataillon-CC64, Montpellier, France.
LA  - eng
SI  - Dryad/10.5061/dryad.1m3s2
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Proteins)
SB  - IM
MH  - Animals
MH  - Computer Simulation
MH  - Genome/*genetics
MH  - Genomics/*standards
MH  - Mammals/*classification/genetics
MH  - *Phylogeny
MH  - Proteins/genetics
OTO - NOTNLM
OT  - *Coalescence
OT  - *exon
OT  - *Supertree
OT  - *phylogeny
OT  - *placental
EDAT- 2017/02/09 06:00
MHDA- 2018/02/14 06:00
CRDT- 2017/02/08 06:00
PHST- 2016/01/18 00:00 [received]
PHST- 2016/03/25 00:00 [revised]
PHST- 2016/09/04 00:00 [accepted]
PHST- 2017/02/08 06:00 [entrez]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/14 06:00 [medline]
AID - 2449707 [pii]
AID - 10.1093/sysbio/syw082 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):112-120. doi: 10.1093/sysbio/syw082.

PMID- 28169404
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - Age-Dependent and Lineage-Dependent Speciation and Extinction in the Imbalance of
      Phylogenetic Trees.
PG  - 912-916
LID - 10.1093/sysbio/syx031 [doi]
AB  - It is known that phylogenetic trees are more imbalanced than expected from a
      birth-death model with constant rates of speciation and extinction, and also that
      imbalance can be better fit by allowing the rate of speciation to decrease as the
      age of the parent species increases. If imbalance is measured in more detail, at 
      nodes within trees as a function of the number of species descended from the
      nodes, age-dependent models predict levels of imbalance comparable to real trees 
      for small numbers of descendent species, but predicted imbalance approaches an
      asymptote not found in real trees as the number of descendent species becomes
      large. Age-dependence must therefore be complemented by another process such as
      inheritance of different rates along different lineages, which is known to
      predict insufficient imbalance at nodes with few descendent species, but can
      predict increasing imbalance with increasing numbers of descendent species.
      [Crump-Mode-Jagers process; diversification; macroevolution; taxon sampling; tree
      of life.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Holman, Eric W
AU  - Holman EW
AD  - Department of Psychology, University of California, Los Angeles, CA 90095, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.2q9r7
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Extinction, Biological
MH  - *Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Time
EDAT- 2017/02/09 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/02/08 06:00
PHST- 2016/05/17 00:00 [received]
PHST- 2017/01/25 00:00 [accepted]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/02/08 06:00 [entrez]
AID - 2972845 [pii]
AID - 10.1093/sysbio/syx031 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):912-916. doi: 10.1093/sysbio/syx031.

PMID- 28169402
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Can We "Future-Proof" Consensus Trees?
PG  - 611-619
LID - 10.1093/sysbio/syx030 [doi]
AB  - Consensus methods are widely used for combining phylogenetic trees into a single 
      estimate of the evolutionary tree for a group of species. As more taxa are added,
      the new source trees may begin to tell a different evolutionary story when
      restricted to the original set of taxa. However, if the new trees, restricted to 
      the original set of taxa, were to agree exactly with the earlier trees, then we
      might hope that their consensus would either agree with or resolve the original
      consensus tree. In this article, we ask under what conditions consensus methods
      exist that are "future proof" in this sense. While we show that some methods
      (e.g., Adams consensus) have this property for specific types of input, we also
      establish a rather surprising "no-go" theorem: there is no "reasonable" consensus
      method that satisfies the future-proofing property in general. We then
      investigate a second notion of "future proofing" for consensus methods, in which 
      trees (rather than taxa) are added, and establish some positive and negative
      results. We end with some questions for future work.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Bryant, David
AU  - Bryant D
AD  - Computational Modeling, Department of Mathematics and Statistics, University of
      Otago, Dunedin, New Zealand.
FAU - Francis, Andrew
AU  - Francis A
AD  - Centre for Research in Mathematics, School of Computing, Engineering and
      Mathematics, Western Sydney University, Sydney, Australia.
FAU - Steel, Mike
AU  - Steel M
AD  - Biomathematics Research Centre, Mathematics and Statistics, University of
      Canterbury, Christchurch, New Zealand.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Biological Evolution
MH  - Classification/*methods
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Adams consensus
OT  - *consensus
OT  - *extension stability
OT  - *phylogeny
EDAT- 2017/02/09 06:00
MHDA- 2018/01/30 06:00
CRDT- 2017/02/08 06:00
PHST- 2016/10/30 00:00 [received]
PHST- 2017/01/25 00:00 [accepted]
PHST- 2017/02/09 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2017/02/08 06:00 [entrez]
AID - 2972844 [pii]
AID - 10.1093/sysbio/syx030 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):611-619. doi: 10.1093/sysbio/syx030.

PMID- 28158373
OWN - NLM
STAT- MEDLINE
DCOM- 20180228
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 1
DP  - 2017 Jan 1
TI  - Special Issue: Mathematical and Computational Evolutionary Biology-2015.
PG  - 1-2
LID - 10.1093/sysbio/syw111 [doi]
FAU - Bryant, David
AU  - Bryant D
AD  - Department of Mathematics and Statistics, University of Otago, Dunedin, New
      Zealand.
FAU - Gascuel, Olivier
AU  - Gascuel O
AD  - Unite Bioinformatique Evolutive C3BI, USR 3756 Institut Pasteur - CNRS Paris,
      France.
AD  - Institut de Biologie Computationnelle, LIRMM, UMR 5506, Universite de Montpellier
      - CNRS Montpellier, France.
LA  - eng
PT  - Introductory Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Classification
MH  - *Computational Biology
MH  - Congresses as Topic
PMC - PMC5410973
EDAT- 2017/02/06 06:00
MHDA- 2018/03/01 06:00
CRDT- 2017/02/04 06:00
PHST- 2017/02/04 06:00 [entrez]
PHST- 2017/02/06 06:00 [pubmed]
PHST- 2018/03/01 06:00 [medline]
AID - 2965347 [pii]
AID - 10.1093/sysbio/syw111 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jan 1;66(1):1-2. doi: 10.1093/sysbio/syw111.

PMID- 28123117
OWN - NLM
STAT- MEDLINE
DCOM- 20180130
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 5
DP  - 2017 Sep 1
TI  - Phylogenomics from Whole Genome Sequences Using aTRAM.
PG  - 786-798
LID - 10.1093/sysbio/syw105 [doi]
AB  - Novel sequencing technologies are rapidly expanding the size of data sets that
      can be applied to phylogenetic studies. Currently the most commonly used
      phylogenomic approaches involve some form of genome reduction. While these
      approaches make assembling phylogenomic data sets more economical for organisms
      with large genomes, they reduce the genomic coverage and thereby the long-term
      utility of the data. Currently, for organisms with moderate to small genomes
      ($&lt;$1000 Mbp) it is feasible to sequence the entire genome at modest coverage
      ($10-30\times$). Computational challenges for handling these large data sets can 
      be alleviated by assembling targeted reads, rather than assembling the entire
      genome, to produce a phylogenomic data matrix. Here we demonstrate the use of
      automated Target Restricted Assembly Method (aTRAM) to assemble 1107 single-copy 
      ortholog genes from whole genome sequencing of sucking lice (Anoplura) and
      out-groups. We developed a pipeline to extract exon sequences from the aTRAM
      assemblies by annotating them with respect to the original target protein. We
      aligned these protein sequences with the inferred amino acids and then performed 
      phylogenetic analyses on both the concatenated matrix of genes and on each gene
      separately in a coalescent analysis. Finally, we tested the limits of successful 
      assembly in aTRAM by assembling 100 genes from close- to distantly related taxa
      at high to low levels of coverage.Both the concatenated analysis and the
      coalescent-based analysis produced the same tree topology, which was consistent
      with previously published results and resolved weakly supported nodes. These
      results demonstrate that this approach is successful at developing phylogenomic
      data sets from raw genome sequencing reads. Further, we found that with coverages
      above $5-10\times$, aTRAM was successful at assembling 80-90% of the contigs for 
      both close and distantly related taxa. As sequencing costs continue to decline,
      we expect full genome sequencing will become more feasible for a wider array of
      organisms, and aTRAM will enable mining of these genomic data sets for an
      extensive variety of applications, including phylogenomics. [aTRAM; gene
      assembly; genome sequencing; phylogenomics.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Allen, Julie M
AU  - Allen JM
AD  - Illinois Natural History Survey, University of Illinois at Urbana-Champaign,
      Urbana, IL 61801, USA.
FAU - Boyd, Bret
AU  - Boyd B
AD  - Illinois Natural History Survey, University of Illinois at Urbana-Champaign,
      Urbana, IL 61801, USA.
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL 32611, 
      USA.
FAU - Nguyen, Nam-Phuong
AU  - Nguyen NP
AD  - Carl R. Woese Institute for Genomic Biology, University of Illinois at
      Urbana-Champaign, Urbana, IL 61801, USA.
FAU - Vachaspati, Pranjal
AU  - Vachaspati P
AD  - Department of Computer Science, University of Illinois at Urbana-Champaign,
      Urbana, IL 61801, USA.
FAU - Warnow, Tandy
AU  - Warnow T
AD  - Carl R. Woese Institute for Genomic Biology, University of Illinois at
      Urbana-Champaign, Urbana, IL 61801, USA.
AD  - Department of Computer Science, University of Illinois at Urbana-Champaign,
      Urbana, IL 61801, USA.
AD  - Department of Bioengineering, University of Illinois at Urbana-Champaign, Urbana,
      IL 61801, USA.
FAU - Huang, Daisie I
AU  - Huang DI
AD  - Biodiversity Research Centre, University of British Columbia, Vancouver V6T 1Z4, 
      Canada.
FAU - Grady, Patrick G S
AU  - Grady PGS
AD  - Illinois Natural History Survey, University of Illinois at Urbana-Champaign,
      Urbana, IL 61801, USA.
FAU - Bell, Kayce C
AU  - Bell KC
AD  - Department of Biology and Museum of Southwestern Biology, University of New
      Mexico, Albuquerque, NM 87131, USA.
FAU - Cronk, Quentin C B
AU  - Cronk QCB
AD  - Biodiversity Research Centre, University of British Columbia, Vancouver V6T 1Z4, 
      Canada.
FAU - Mugisha, Lawrence
AU  - Mugisha L
AD  - Conservation &amp; Ecosystem Health Alliance (CEHA), Kampala, Uganda.
AD  - College of Veterinary Medicine, Animal Resources &amp; Biosecurity (COVAB), Makerere 
      University, Uganda.
FAU - Pittendrigh, Barry R
AU  - Pittendrigh BR
AD  - Department of Entomology Michigan State University, East Lansing, MI 48823, USA.
FAU - Leonardi, M Soledad
AU  - Leonardi MS
AD  - Instituto de Biologia de Organismos Marinos, Centro Nacional Patagonico, Puerto
      Madryn, Argentina.
FAU - Reed, David L
AU  - Reed DL
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL 32611, 
      USA.
FAU - Johnson, Kevin P
AU  - Johnson KP
AD  - Illinois Natural History Survey, University of Illinois at Urbana-Champaign,
      Urbana, IL 61801, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.26j38
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Genomics/*methods
MH  - *Phylogeny
MH  - Sequence Analysis
EDAT- 2017/01/27 06:00
MHDA- 2018/01/31 06:00
CRDT- 2017/01/27 06:00
PHST- 2016/06/02 00:00 [received]
PHST- 2016/12/01 00:00 [accepted]
PHST- 2017/01/27 06:00 [pubmed]
PHST- 2018/01/31 06:00 [medline]
PHST- 2017/01/27 06:00 [entrez]
AID - syw105 [pii]
AID - 10.1093/sysbio/syw105 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Sep 1;66(5):786-798. doi: 10.1093/sysbio/syw105.

PMID- 28123116
OWN - NLM
STAT- MEDLINE
DCOM- 20180130
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 5
DP  - 2017 Sep 1
TI  - The First Organ-Based Ontology for Arthropods (Ontology of Arthropod Circulatory 
      Systems - OArCS) and its Integration into a Novel Formalization Scheme for
      Morphological Descriptions.
PG  - 754-768
LID - 10.1093/sysbio/syw108 [doi]
AB  - Morphology, the oldest discipline in the biosciences, is currently experiencing a
      renaissance in the field of comparative phenomics. However,
      morphological/phenotypic research still suffers on various levels from a lack of 
      standards. This shortcoming, first highlighted as the "linguistic problem of
      morphology", concerns the usage of terminology and also the need for
      formalization of morphological descriptions themselves, something of paramount
      importance not only to the field of morphology but also when it comes to the use 
      of phenotypic data in systematics and evolutionary biology. We therefore argue,
      that for morphological descriptions, the basis of all systematic and evolutionary
      interpretations, ontologies need to be utilized which are based exclusively on
      structural qualities/properties and which in no case include statements about
      homology and/or function. Statements about homology and function constitute
      interpretations on a different or higher level. Based on these "anatomy
      ontologies", further ontological dimensions (e.g., referring to functional
      properties or homology) may be exerted for a broad use in evolutionary phenomics.
      To this end we present the first organ-based ontology for the most species-rich
      animal group, the Arthropoda. Our Ontology of Arthropod Circulatory Systems
      (OArCS) contains a comprehensive collection of 383 terms (i.e., labels) tied to
      296 concepts (i.e., definitions) collected from the literature on phenotypic
      aspects of circulatory organ features in arthropods. All of the concepts used in 
      OArCS are based exclusively on structural features, and in the context of the
      ontology are independent of homology and functional assumptions. We cannot rule
      out that in some cases, terms are used which in traditional usage and previous
      accounts might have implied homology and/or function (e.g. heart, sternal
      artery). Concepts are composed of descriptive elements that are used to classify 
      observed instances into the organizational framework of the ontology. That is,
      descriptions in ontologies are only descriptions of individuals if they are
      necessary/and or sufficient representations of attributes (independently)
      observed and recorded for an individual. In addition, we here present for the
      first time an entirely new approach to formalizing phenotypic research, a
      semantic model for the description of a complex organ system in a highly
      disparate taxon, the arthropods. We demonstrate this with a formalized
      morphological description of the hemolymph vascular system in one specimen of the
      European garden spider Araneus diadematus. Our description targets five
      categories of descriptive statement: "position", "spatial relationships",
      "shape", "constituents", and "connections", as the corresponding formalizations
      constitute exemplary patterns useful not only when talking about the circulatory 
      system, but also in descriptions in general. The downstream applications of
      computer-parsable morphological descriptions are widespread, with their core
      utility being the fact that they make it possible to compare collective
      description sets in computational time, that is, very quickly. Among other
      things, this facilitates the identification of phenotypic plasticity and
      variation when single individuals are compared, the identification of those
      traits which correlate between and within taxa, and the identification of links
      between morphological traits and genetic (using GO, Gene Ontology) or
      environmental (using ENVO, Environmental Ontology) factors. [Arthropoda; concept;
      function; hemolymph vascular system; homology; terminology.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Wirkner, Christian S
AU  - Wirkner CS
AD  - Allgemeine &amp; Spezielle Zoologie, Institut fuer Biowissenschaften, Universitaet
      Rostock, 18055 Rostock, Germany.
FAU - Gopel, Torben
AU  - Gopel T
AD  - Allgemeine &amp; Spezielle Zoologie, Institut fuer Biowissenschaften, Universitaet
      Rostock, 18055 Rostock, Germany.
FAU - Runge, Jens
AU  - Runge J
AD  - Allgemeine &amp; Spezielle Zoologie, Institut fuer Biowissenschaften, Universitaet
      Rostock, 18055 Rostock, Germany.
FAU - Keiler, Jonas
AU  - Keiler J
AD  - Allgemeine &amp; Spezielle Zoologie, Institut fuer Biowissenschaften, Universitaet
      Rostock, 18055 Rostock, Germany.
FAU - Klussmann-Fricke, Bastian-Jesper
AU  - Klussmann-Fricke BJ
AD  - Allgemeine &amp; Spezielle Zoologie, Institut fuer Biowissenschaften, Universitaet
      Rostock, 18055 Rostock, Germany.
FAU - Huckstorf, Katarina
AU  - Huckstorf K
AD  - Allgemeine &amp; Spezielle Zoologie, Institut fuer Biowissenschaften, Universitaet
      Rostock, 18055 Rostock, Germany.
FAU - Scholz, Stephan
AU  - Scholz S
AD  - Allgemeine &amp; Spezielle Zoologie, Institut fuer Biowissenschaften, Universitaet
      Rostock, 18055 Rostock, Germany.
FAU - Miko, Istvan
AU  - Miko I
AD  - Department of Entomology, Pennsylvania State University, University Park, PA
      16802, USA.
FAU - J Yoder, Matthew
AU  - J Yoder M
AD  - Illinois Natural History Survey, University of Illinois, Champaign, IL 61820,
      USA.
FAU - Richter, Stefan
AU  - Richter S
AD  - Allgemeine &amp; Spezielle Zoologie, Institut fuer Biowissenschaften, Universitaet
      Rostock, 18055 Rostock, Germany.
LA  - eng
SI  - Dryad/10.5061/dryad.kv62d
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Anatomy/*standards
MH  - Animals
MH  - Arthropods/*anatomy &amp; histology
MH  - Classification/*methods
MH  - Phenotype
MH  - Phylogeny
MH  - *Terminology as Topic
EDAT- 2017/01/27 06:00
MHDA- 2018/01/31 06:00
CRDT- 2017/01/27 06:00
PHST- 2016/05/27 00:00 [received]
PHST- 2016/11/26 00:00 [accepted]
PHST- 2017/01/27 06:00 [pubmed]
PHST- 2018/01/31 06:00 [medline]
PHST- 2017/01/27 06:00 [entrez]
AID - syw108 [pii]
AID - 10.1093/sysbio/syw108 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Sep 1;66(5):754-768. doi: 10.1093/sysbio/syw108.

PMID- 28123115
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Homology-Aware Phylogenomics at Gigabase Scales.
PG  - 590-603
LID - 10.1093/sysbio/syw104 [doi]
AB  - Obstacles to inferring species trees from whole genome data sets range from
      algorithmic and data management challenges to the wholesale discordance in
      evolutionary history found in different parts of a genome. Recent work that
      builds trees directly from genomes by parsing them into sets of small $k$-mer
      strings holds promise to streamline and simplify these efforts, but existing
      approaches do not account well for gene tree discordance. We describe a "seed and
      extend" protocol that finds nearly exact matching sets of orthologous $k$-mers
      and extends them to construct data sets that can properly account for genomic
      heterogeneity. Exploiting an efficient suffix array data structure, sets of whole
      genomes can be parsed and converted into phylogenetic data matrices rapidly, with
      contiguous blocks of $k$-mers from the same chromosome, gene, or scaffold
      concatenated as needed. Phylogenetic trees constructed from highly curated rice
      genome data and a diverse set of six other eukaryotic whole genome,
      transcriptome, and organellar genome data sets recovered trees nearly identical
      to published phylogenomic analyses, in a small fraction of the time, and
      requiring many fewer parameter choices. Our method's ability to retain local
      homology information was demonstrated by using it to characterize gene tree
      discordance across the rice genome, and by its robustness to the high rate of
      interchromosomal gene transfer found in several rice species.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Sanderson, M J
AU  - Sanderson MJ
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA.
FAU - Nicolae, Marius
AU  - Nicolae M
AD  - Department of Computer Science and Engineering, University of Connecticut,
      Storrs, CT 06269, USA.
FAU - McMahon, M M
AU  - McMahon MM
AD  - School of Plant Sciences, University of Arizona, Tucson, AZ 85721, USA.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Gene Transfer, Horizontal/genetics
MH  - *Genomics
MH  - Oryza/classification/genetics
MH  - *Phylogeny
PMC - PMC5790135
OTO - NOTNLM
OT  - *Oryza
OT  - *k-mer
OT  - *lineage sorting
OT  - *phylogenomics
OT  - *suffix array
EDAT- 2017/01/27 06:00
MHDA- 2018/01/30 06:00
CRDT- 2017/01/27 06:00
PHST- 2016/05/19 00:00 [received]
PHST- 2016/11/25 00:00 [accepted]
PHST- 2017/01/27 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2017/01/27 06:00 [entrez]
AID - syw104 [pii]
AID - 10.1093/sysbio/syw104 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):590-603. doi: 10.1093/sysbio/syw104.

PMID- 28123114
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Split Scores: A Tool to Quantify Phylogenetic Signal in Genome-Scale Data.
PG  - 620-636
LID - 10.1093/sysbio/syw103 [doi]
AB  - Detecting variation in the evolutionary process along chromosomes is increasingly
      important as whole-genome data become more widely available. For example, factors
      such as incomplete lineage sorting, horizontal gene transfer, and chromosomal
      inversion are expected to result in changes in the underlying gene trees along a 
      chromosome, while changes in selective pressure and mutational rates for
      different genomic regions may lead to shifts in the underlying mutational
      process. We propose the split score as a general method for quantifying support
      for a particular phylogenetic relationship within a genomic data set. Because the
      split score is based on algebraic properties of a matrix of site pattern
      frequencies, it can be rapidly computed, even for data sets that are large in the
      number of taxa and/or in the length of the alignment, providing an advantage over
      other methods (e.g., maximum likelihood) that are often used to assess such
      support. Using simulation, we explore the properties of the split score,
      including its dependence on sequence length, branch length, size of a split and
      its ability to detect true splits in the underlying tree. Using a sliding window 
      analysis, we show that split scores can be used to detect changes in the
      underlying evolutionary process for genome-scale data from primates, mosquitoes, 
      and viruses in a computationally efficient manner. Computation of the split score
      has been implemented in the software package SplitSup.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Allman, Elizabeth S
AU  - Allman ES
AD  - Department of Mathematics and Statistics, PO Box 756660, University of Alaska
      Fairbanks, Fairbanks, AK 99775-6660, USA.
FAU - Kubatko, Laura S
AU  - Kubatko LS
AD  - Department of Statistics and Department of Evolution, Ecology, and Organismal
      Biology, The Ohio State University, Columbus, OH 43210, USA.
FAU - Rhodes, John A
AU  - Rhodes JA
AD  - Department of Mathematics and Statistics, PO Box 756660, University of Alaska
      Fairbanks, Fairbanks, AK 99775-6660, USA.
LA  - eng
GR  - R01 GM117590/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Culicidae/classification/genetics
MH  - Evolution, Molecular
MH  - Gene Transfer, Horizontal
MH  - Genome/genetics
MH  - *Phylogeny
MH  - Primates/classification/genetics
MH  - Software
MH  - Viruses/classification/genetics
PMC - PMC6075200
OTO - NOTNLM
OT  - *General Markov model
OT  - *genome-scale data analysis
OT  - *matrix flattenings
OT  - *phylogenetic trees
OT  - *singular value decomposition
OT  - *split scores
EDAT- 2017/01/27 06:00
MHDA- 2018/01/30 06:00
CRDT- 2017/01/27 06:00
PHST- 2016/08/02 00:00 [received]
PHST- 2016/10/28 00:00 [accepted]
PHST- 2017/01/27 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2017/01/27 06:00 [entrez]
AID - syw103 [pii]
AID - 10.1093/sysbio/syw103 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):620-636. doi: 10.1093/sysbio/syw103.

PMID- 28123113
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20180226
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Erratum.
PG  - 660
LID - 10.1093/sysbio/syw113 [doi]
LA  - eng
PT  - Journal Article
PT  - Published Erratum
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EFR - Syst Biol. 2017 Jan 1;66(1):e1-e29. PMID: 28173586
PMC - PMC5790131
EDAT- 2017/01/27 06:00
MHDA- 2017/01/27 06:01
CRDT- 2017/01/27 06:00
PHST- 2017/01/27 06:00 [pubmed]
PHST- 2017/01/27 06:01 [medline]
PHST- 2017/01/27 06:00 [entrez]
AID - syw113 [pii]
AID - 10.1093/sysbio/syw113 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):660. doi: 10.1093/sysbio/syw113.

PMID- 28123112
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - How the Aridification of Australia Structured the Biogeography and Influenced the
      Diversification of a Large Lineage of Australian Cicadas.
PG  - 569-589
LID - 10.1093/sysbio/syw078 [doi]
AB  - Over the last 30 million years, Australia's landscape has undergone dramatic
      cooling and drying due to the establishment of the Antarctic Circumpolar Current 
      and change in global CO$_{2}$ levels. Studies have shown that many Australian
      organisms went extinct during these major cooling events, while others
      experienced adaptive radiations and increases in diversification rates as a
      result of exploiting new niches in the arid zone. Despite the many studies on
      diversification and biogeography in Australia, few have been continent-wide and
      none have focused on a group of organisms adapted to feeding on plants. We
      studied 162 species of cicadas in the Australian Pauropsalta complex, a large
      generic lineage within the tribe Cicadettini. We asked whether there were changes
      in the diversification rate of Pauropsalta over time and if so: 1) which clades
      were associated with the rate change? 2) did timing of rate shifts correspond to 
      known periods of dramatic historical climate change, 3) did increases in
      diversification rate along select lineages correspond to adaptive radiations with
      movement into the arid zone? To address these questions, we estimated a molecular
      phylogeny of the Pauropsalta complex using ${\sim}$5300 bp of nucleotide sequence
      data distributed among five loci (one mtDNA locus and four nDNA loci). We found
      that this large group of cicadas did not diversify at a constant rate as they
      spread through Australia; instead the signature of decreasing diversification
      rate changed roughly around the time of the expansion of the east Antarctic ice
      sheets ${\sim}$16 Ma and the glaciation of the northern hemisphere ${\sim}$3 Ma. 
      Unlike other Australian taxa, the Pauropsalta complex did not explosively radiate
      in response to an early invasion of the arid zone. Instead multiple groups
      invaded the arid zone and experienced rates of diversification similar to
      mesic-distributed taxa. We found evidence for relictual groups, located in
      pre-Mesozoic habitat, that have not diversified and continue to reside on mesic
      hosts in isolated "habitat islands". Future work should focus on groups of
      similar ages with similar distribution patterns to determine whether this tempo
      and pattern of diversification and biogeography is consistent with evidence from 
      other phytophagous insects.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Owen, Christopher L
AU  - Owen CL
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, CT 06269-3043, USA.
AD  - Computational Biology Institute, George Washington University, Innovation Hall,
      Suite 305, 45085 University Drive, Ashburn, VA 20147-2766, USA.
FAU - Marshall, David C
AU  - Marshall DC
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, CT 06269-3043, USA.
FAU - Hill, Kathy B R
AU  - Hill KBR
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, CT 06269-3043, USA.
FAU - Simon, Chris
AU  - Simon C
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, CT 06269-3043, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.1580p
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Australia
MH  - *Biodiversity
MH  - Climate Change
MH  - Hemiptera/*classification/genetics
MH  - *Phylogeny
MH  - Phylogeography
OTO - NOTNLM
OT  - *Aridification
OT  - *Australia
OT  - *Pauropsalta
OT  - *cicadas
OT  - *diversification
EDAT- 2017/01/27 06:00
MHDA- 2018/01/30 06:00
CRDT- 2017/01/27 06:00
PHST- 2014/04/03 00:00 [received]
PHST- 2016/08/24 00:00 [accepted]
PHST- 2017/01/27 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2017/01/27 06:00 [entrez]
AID - syw078 [pii]
AID - 10.1093/sysbio/syw078 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):569-589. doi: 10.1093/sysbio/syw078.

PMID- 28108601
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - Phylogenomics using Target-Restricted Assembly Resolves Intrageneric
      Relationships of Parasitic Lice (Phthiraptera: Columbicola).
PG  - 896-911
LID - 10.1093/sysbio/syx027 [doi]
AB  - Parasitic "wing lice" (Phthiraptera: Columbicola) and their dove and pigeon hosts
      are a well-recognized model system for coevolutionary studies at the intersection
      of micro- and macroevolution. Selection on lice in microevolutionary time occurs 
      as pigeons and doves defend themselves against lice by preening. In turn,
      behavioral and morphological adaptations of the lice improve their ability to
      evade host defense. Over macroevolutionary time wing lice tend to cospeciate with
      their hosts; yet, some species of Columbicola have switched to new host species. 
      Understanding the ecological and evolutionary factors that influence coadaptation
      and codiversification in this system will substantially improve our understanding
      of coevolution in general. However, further work is hampered by the lack of a
      robust phylogenetic framework for Columbicola spp. and their hosts. Previous
      attempts to resolve the phylogeny of Columbicola based on sequences from a few
      genes provided limited support. Here, we apply a new approach, target restricted 
      assembly, to assemble 977 orthologous gene sequences from whole-genome sequence
      data generated from very small, ethanol-preserved specimens, representing up to
      61 species of wing lice. Both concatenation and coalescent methods were used to
      estimate the species tree. These two approaches yielded consistent and
      well-supported trees with 90% of all relationships receiving 100% support, which 
      is a substantial improvement over previous studies. We used this new phylogeny to
      show that biogeographic ranges are generally conserved within clades of
      Columbicola wing lice. Limited inconsistencies are probably attributable to
      intercontinental dispersal of hosts, and host switching by some of the lice.
      [aTRAM; coalescent; coevolution; concatenation; species tree.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Boyd, Bret M
AU  - Boyd BM
AD  - Department of Entomology, University of Georgia Athens, 413 Biological Sciences
      Building, Athens, GA 30602, USA.
AD  - Illinois Natural History Survey, Prairie Research Institute, University of
      Illinois Urbana-Champaign, Champaign, IL 61820, USA.
FAU - Allen, Julie M
AU  - Allen JM
AD  - Illinois Natural History Survey, Prairie Research Institute, University of
      Illinois Urbana-Champaign, Champaign, IL 61820, USA.
AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL 32611, 
      USA.
FAU - Nguyen, Nam-Phuong
AU  - Nguyen NP
AD  - Department of Computer Science and Engineering, University of California San
      Diego, La Jolla, CA 92093, USA.
FAU - Sweet, Andrew D
AU  - Sweet AD
AD  - Illinois Natural History Survey, Prairie Research Institute, University of
      Illinois Urbana-Champaign, Champaign, IL 61820, USA.
FAU - Warnow, Tandy
AU  - Warnow T
AD  - Departments of Computer Science and Bioengineering, University of Illinois
      Urbana-Champaign, Urbana, IL 61801, USA.
FAU - Shapiro, Michael D
AU  - Shapiro MD
AD  - Department of Biology, University of Utah, Salt Lake City, UT 84112, USA.
FAU - Villa, Scott M
AU  - Villa SM
AD  - Department of Biology, University of Utah, Salt Lake City, UT 84112, USA.
FAU - Bush, Sarah E
AU  - Bush SE
AD  - Department of Biology, University of Utah, Salt Lake City, UT 84112, USA.
FAU - Clayton, Dale H
AU  - Clayton DH
AD  - Department of Biology, University of Utah, Salt Lake City, UT 84112, USA.
FAU - Johnson, Kevin P
AU  - Johnson KP
AD  - Illinois Natural History Survey, Prairie Research Institute, University of
      Illinois Urbana-Champaign, Champaign, IL 61820, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.4812p
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Columbidae/parasitology
MH  - Genome/*genetics
MH  - Host Specificity
MH  - Host-Parasite Interactions
MH  - Phthiraptera/*classification/*genetics/physiology
MH  - *Phylogeny
PMC - PMC5837638
EDAT- 2017/01/22 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/01/22 06:00
PHST- 2016/09/06 00:00 [received]
PHST- 2017/01/06 00:00 [accepted]
PHST- 2017/01/22 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/01/22 06:00 [entrez]
AID - syx027 [pii]
AID - 10.1093/sysbio/syx027 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):896-911. doi: 10.1093/sysbio/syx027.

PMID- 28057858
OWN - NLM
STAT- MEDLINE
DCOM- 20180215
LR  - 20180808
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 6
DP  - 2017 Nov 1
TI  - ProtASR: An Evolutionary Framework for Ancestral Protein Reconstruction with
      Selection on Folding Stability.
PG  - 1054-1064
LID - 10.1093/sysbio/syw121 [doi]
AB  - The computational reconstruction of ancestral proteins provides information on
      past biological events and has practical implications for biomedicine and
      biotechnology. Currently available tools for ancestral sequence reconstruction
      (ASR) are often based on empirical amino acid substitution models that assume
      that all sites evolve at the same rate and under the same process. However, this 
      assumption is frequently violated because protein evolution is highly
      heterogeneous due to different selective constraints among sites. Here, we
      present ProtASR, a new evolutionary framework to infer ancestral protein
      sequences accounting for selection on protein stability. First, ProtASR generates
      site-specific substitution matrices through the structurally constrained
      mean-field (MF) substitution model, which considers both unfolding and misfolding
      stability. We previously showed that MF models outperform empirical amino acid
      substitution models, as well as other structurally constrained substitution
      models, both in terms of likelihood and correctly inferring amino acid
      distributions across sites. In the second step, ProtASR adapts a well-established
      maximum-likelihood (ML) ASR procedure to infer ancestral proteins under MF
      models. A known bias of ML ASR methods is that they tend to overestimate the
      stability of ancestral proteins by underestimating the frequency of deleterious
      mutations. We compared ProtASR under MF to two empirical substitution models (JTT
      and CAT), reconstructing the ancestral sequences of simulated proteins. ProtASR
      yields reconstructed proteins with less biased stabilities, which are
      significantly closer to those of the simulated proteins. Analysis of extant
      protein families suggests that folding stability evolves through time across
      protein families, potentially reflecting neutral fluctuation. Some families
      exhibit a more constant protein folding stability, while others are more
      variable. ProtASR is freely available from
      https://github.com/miguelarenas/protasr and includes detailed documentation and
      ready-to-use examples. It runs in seconds/minutes depending on protein length and
      alignment size. [Ancestral sequence reconstruction; folding stability; molecular 
      adaptation; phylogenetics; protein evolution; protein structure.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Arenas, Miguel
AU  - Arenas M
AD  - Instituto de Investigacao e Inovacao em Saude (i3S), University of Porto, Porto, 
      Portugal.
AD  - Institute of Molecular Pathology and Immunology of the University of Porto
      (IPATIMUP), Porto, Portugal.
AD  - Centre for Molecular Biology Severo Ochoa (CBMSO), Consejo Superior de
      Investigaciones Cientificas (CSIC), Madrid, Spain.
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Vigo,
      Spain.
FAU - Weber, Claudia C
AU  - Weber CC
AD  - Department of Biology and Center for Computational Genetics and Genomics, Temple 
      University, Philadelphia, PA 19122, USA.
FAU - Liberles, David A
AU  - Liberles DA
AD  - Department of Molecular Biology, University of Wyoming, Laramie, WY 82071, USA.
AD  - Department of Biology and Center for Computational Genetics and Genomics, Temple 
      University, Philadelphia, PA 19122, USA.
FAU - Bastolla, Ugo
AU  - Bastolla U
AD  - Centre for Molecular Biology Severo Ochoa (CBMSO), Consejo Superior de
      Investigaciones Cientificas (CSIC), Madrid, Spain.
LA  - eng
SI  - Dryad/10.5061/dryad.bh213
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ancient)
SB  - IM
MH  - *Algorithms
MH  - Amino Acid Substitution
MH  - Classification/*methods
MH  - DNA, Ancient/chemistry
MH  - Evolution, Molecular
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Protein Folding
MH  - Protein Stability
MH  - *Sequence Analysis, Protein
MH  - Software
EDAT- 2017/01/07 06:00
MHDA- 2018/02/16 06:00
CRDT- 2017/01/07 06:00
PHST- 2016/06/10 00:00 [received]
PHST- 2016/12/28 00:00 [accepted]
PHST- 2017/01/07 06:00 [pubmed]
PHST- 2018/02/16 06:00 [medline]
PHST- 2017/01/07 06:00 [entrez]
AID - syw121 [pii]
AID - 10.1093/sysbio/syw121 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Nov 1;66(6):1054-1064. doi: 10.1093/sysbio/syw121.

PMID- 28053140
OWN - NLM
STAT- MEDLINE
DCOM- 20180130
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 5
DP  - 2017 Sep 1
TI  - Efficient Bayesian Species Tree Inference under the Multispecies Coalescent.
PG  - 823-842
LID - 10.1093/sysbio/syw119 [doi]
AB  - We develop a Bayesian method for inferring the species phylogeny under the
      multispecies coalescent (MSC) model. To improve the mixing properties of the
      Markov chain Monte Carlo (MCMC) algorithm that traverses the space of species
      trees, we implement two efficient MCMC proposals: the first is based on the
      Subtree Pruning and Regrafting (SPR) algorithm and the second is based on a
      node-slider algorithm. Like the Nearest-Neighbor Interchange (NNI) algorithm we
      implemented previously, both new algorithms propose changes to the species tree, 
      while simultaneously altering the gene trees at multiple genetic loci to
      automatically avoid conflicts with the newly proposed species tree. The method
      integrates over gene trees, naturally taking account of the uncertainty of gene
      tree topology and branch lengths given the sequence data. A simulation study was 
      performed to examine the statistical properties of the new method. The method was
      found to show excellent statistical performance, inferring the correct species
      tree with near certainty when 10 loci were included in the dataset. The prior on 
      species trees has some impact, particularly for small numbers of loci. We
      analyzed several previously published datasets (both real and simulated) for
      rattlesnakes and Philippine shrews, in comparison with alternative methods. The
      results suggest that the Bayesian coalescent-based method is statistically more
      efficient than heuristic methods based on summary statistics, and that our
      implementation is computationally more efficient than alternative full-likelihood
      methods under the MSC. Parameter estimates for the rattlesnake data suggest
      drastically different evolutionary dynamics between the nuclear and mitochondrial
      loci, even though they support largely consistent species trees. We discuss the
      different challenges facing the marginal likelihood calculation and transmodel
      MCMC as alternative strategies for estimating posterior probabilities for species
      trees. [Bayes factor; Bayesian inference; MCMC; multispecies coalescent;
      nodeslider; species tree; SPR.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Rannala, Bruce
AU  - Rannala B
AD  - Department of Evolution and Ecology, University of California, Davis, CA 95616,
      USA.
FAU - Yang, Ziheng
AU  - Yang Z
AD  - Department of Genetics, Evolution and Environment, University College London,
      London WC1E 6BT, UK.
LA  - eng
GR  - BB/N000609/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Crotalus/classification/genetics
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Shrews/classification/genetics
EDAT- 2017/01/06 06:00
MHDA- 2018/01/31 06:00
CRDT- 2017/01/06 06:00
PHST- 2015/12/14 00:00 [received]
PHST- 2016/12/10 00:00 [accepted]
PHST- 2017/01/06 06:00 [pubmed]
PHST- 2018/01/31 06:00 [medline]
PHST- 2017/01/06 06:00 [entrez]
AID - syw119 [pii]
AID - 10.1093/sysbio/syw119 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Sep 1;66(5):823-842. doi: 10.1093/sysbio/syw119.

PMID- 28039387
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 3
DP  - 2017 May 1
TI  - Accounting for Uncertainty in Gene Tree Estimation: Summary-Coalescent Species
      Tree Inference in a Challenging Radiation of Australian Lizards.
PG  - 352-366
LID - 10.1093/sysbio/syw089 [doi]
AB  - Accurate gene tree inference is an important aspect of species tree estimation in
      a summary-coalescent framework. Yet, in empirical studies, inferred gene trees
      differ in accuracy due to stochastic variation in phylogenetic signal between
      targeted loci. Empiricists should, therefore, examine the consistency of species 
      tree inference, while accounting for the observed heterogeneity in gene tree
      resolution of phylogenomic data sets. Here, we assess the impact of gene tree
      estimation error on summary-coalescent species tree inference by screening
      ${\sim}2000$ exonic loci based on gene tree resolution prior to phylogenetic
      inference. We focus on a phylogenetically challenging radiation of Australian
      lizards (genus Cryptoblepharus, Scincidae) and explore effects on topology and
      support. We identify a well-supported topology based on all loci and find that a 
      relatively small number of high-resolution gene trees can be sufficient to
      converge on the same topology. Adding gene trees with decreasing resolution
      produced a generally consistent topology, and increased confidence for specific
      bipartitions that were poorly supported when using a small number of informative 
      loci. This corroborates coalescent-based simulation studies that have highlighted
      the need for a large number of loci to confidently resolve challenging
      relationships and refutes the notion that low-resolution gene trees introduce
      phylogenetic noise. Further, our study also highlights the value of quantifying
      changes in nodal support across locus subsets of increasing size (but decreasing 
      gene tree resolution). Such detailed analyses can reveal anomalous fluctuations
      in support at some nodes, suggesting the possibility of model violation. By
      characterizing the heterogeneity in phylogenetic signal among loci, we can
      account for uncertainty in gene tree estimation and assess its effect on the
      consistency of the species tree estimate. We suggest that the evaluation of gene 
      tree resolution should be incorporated in the analysis of empirical phylogenomic 
      data sets. This will ultimately increase our confidence in species tree
      estimation using summary-coalescent methods and enable us to exploit genomic data
      for phylogenetic inference. [Coalescence; concatenation; Cryptoblepharus; exon
      capture; gene tree; phylogenomics; species tree.].
CI  - (c) The authors 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      e-mail: journals.permission@oup.com.
FAU - Blom, Mozes P K
AU  - Blom MPK
AD  - Research School of Biology, Australian National University, Canberra ACT 0200,
      Australia.
FAU - Bragg, Jason G
AU  - Bragg JG
AD  - Research School of Biology, Australian National University, Canberra ACT 0200,
      Australia.
FAU - Potter, Sally
AU  - Potter S
AD  - Research School of Biology, Australian National University, Canberra ACT 0200,
      Australia.
FAU - Moritz, Craig
AU  - Moritz C
AD  - Research School of Biology, Australian National University, Canberra ACT 0200,
      Australia.
LA  - eng
SI  - Dryad/10.5061/dryad.91867
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Australia
MH  - Genome/*genetics
MH  - Lizards/*classification/*genetics
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Uncertainty
EDAT- 2017/01/01 06:00
MHDA- 2018/01/30 06:00
CRDT- 2017/01/01 06:00
PHST- 2015/09/14 00:00 [received]
PHST- 2016/09/27 00:00 [accepted]
PHST- 2017/01/01 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2017/01/01 06:00 [entrez]
AID - syw089 [pii]
AID - 10.1093/sysbio/syw089 [doi]
PST - ppublish
SO  - Syst Biol. 2017 May 1;66(3):352-366. doi: 10.1093/sysbio/syw089.

PMID- 28003535
OWN - NLM
STAT- MEDLINE
DCOM- 20180130
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 5
DP  - 2017 Sep 1
TI  - Species Delimitation with Gene Flow.
PG  - 799-812
LID - 10.1093/sysbio/syw117 [doi]
AB  - Species are commonly thought to be evolutionarily independent in a way that
      populations within a species are not. In recent years, studies that seek to
      identify evolutionarily independent lineages (i.e., to delimit species) using
      genetic data have typically adopted multispecies coalescent approaches that
      assume that evolutionary independence is formed by the differential sorting of
      ancestral alleles due to genetic drift. However, gene flow appears to be common
      among populations and nascent species, and while this process may inhibit lineage
      divergence (and thus independence), it is usually not explicitly considered when 
      delimiting species. In this article, we apply Phylogeographic Inference using
      Approximate Likelihoods (PHRAPL), a recently described method for phylogeographic
      model selection, to species delimitation. We describe an approach to delimiting
      species using PHRAPL that attempts to account for both genetic drift and gene
      flow, and we compare the method's performance to that of a popular delimitation
      approach (BPP) using both simulated and empirical datasets. PHRAPL generally
      infers the correct demographic-delimitation model when the generating model
      includes gene flow between taxa, given a sufficient amount of data. When the
      generating model includes only isolation in the recent past, PHRAPL will in some 
      cases fail to differentiate between gene flow and divergence, leading to model
      misspecification. Nevertheless, the explicit consideration of gene flow by PHRAPL
      is an important complement to existing delimitation approaches, particularly in
      systems where gene flow is likely important. [approximate likelihoods; coalescent
      simulations; genealogical divergence index; Homo sapiens;
      isolation-with-migration; multispecies coalescent; Sarracenia; Scincella.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Jackson, Nathan D
AU  - Jackson ND
AD  - Department of Ecology and Evolutionary Biology, University of Tennessee,
      Knoxville, 442 Hesler Biology Building, Knoxville, TN 37996, USA.
FAU - Carstens, Bryan C
AU  - Carstens BC
AD  - Department of Evolution, Ecology and Organismal Biology, Ohio State University,
      318 W. 12th Avenue, Columbus, OH 43210, USA.
FAU - Morales, Ariadna E
AU  - Morales AE
AD  - Department of Evolution, Ecology and Organismal Biology, Ohio State University,
      318 W. 12th Avenue, Columbus, OH 43210, USA.
FAU - O'Meara, Brian C
AU  - O'Meara BC
AD  - Department of Ecology and Evolutionary Biology, University of Tennessee,
      Knoxville, 442 Hesler Biology Building, Knoxville, TN 37996, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.t8016
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Gene Flow
MH  - *Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Phylogeography
MH  - Species Specificity
EDAT- 2016/12/23 06:00
MHDA- 2018/01/31 06:00
CRDT- 2016/12/23 06:00
PHST- 2015/12/07 00:00 [received]
PHST- 2009/12/13 00:00 [accepted]
PHST- 2016/12/23 06:00 [pubmed]
PHST- 2018/01/31 06:00 [medline]
PHST- 2016/12/23 06:00 [entrez]
AID - syw117 [pii]
AID - 10.1093/sysbio/syw117 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Sep 1;66(5):799-812. doi: 10.1093/sysbio/syw117.

PMID- 28003534
OWN - NLM
STAT- MEDLINE
DCOM- 20180130
LR  - 20190118
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 5
DP  - 2017 Sep 1
TI  - Dental Data Perform Relatively Poorly in Reconstructing Mammal Phylogenies:
      Morphological Partitions Evaluated with Molecular Benchmarks.
PG  - 813-822
LID - 10.1093/sysbio/syw116 [doi]
AB  - Phylogenetic trees underpin reconstructions of evolutionary history and tests of 
      evolutionary hypotheses. They are inferred from both molecular and morphological 
      data, yet the relative value of morphology has been questioned in this context
      due to perceived homoplasy, developmental linkage, and nonindependence of
      characters. Nevertheless, fossil data are limited to incomplete subsets of
      preserved morphology, and different regions are treated as equivalent. Through
      meta-analysis of 40 data sets, we show here that the dental and osteological
      characters of mammals convey significantly different phylogenetic signals, and
      that osteological characters are significantly more compatible with molecular
      trees. Furthermore, the application of simplified paleontological filters
      (retaining only dental data) results in significantly greater loss of
      phylogenetic signal than random character ablation. Although the mammal fossil
      record is largely comprised of teeth, dental data alone are generally found to be
      less reliable for phylogenetic reconstruction given their incongruence with
      osteological and molecular data. These findings highlight the need for rigorous
      meta-analyses of distributions of homoplasy in morphological data. These tests,
      and consequent refinements to phylogenetic analyses that they permit, promise to 
      improve the quality of all macroevolutionary studies that hinge on accurate
      trees. [Homoplasy; Mammalia; morphology; osteology; phylogeny; teeth.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Sansom, Robert S
AU  - Sansom RS
AD  - School of Earth and Environmental Sciences, University of Manchester, Oxford
      Road, Manchester M13 9PT, UK.
FAU - Wills, Matthew Albion
AU  - Wills MA
AD  - Department of Biology and Biochemistry, University of Bath, Claverton Down, Bath 
      BA2 7AY, UK.
FAU - Williams, Tamara
AU  - Williams T
AD  - School of Earth and Environmental Sciences, University of Manchester, Oxford
      Road, Manchester M13 9PT, UK.
LA  - eng
SI  - Dryad/10.5061/dryad.k23mq
GR  - BB/N015827/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
GR  - BB/K015702/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
GR  - BB/K006754/1 /Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Meta-Analysis
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Fossils
MH  - Genetic Markers/genetics
MH  - Mammals/anatomy &amp; histology/*classification/genetics
MH  - *Phylogeny
MH  - Tooth/anatomy &amp; histology
PMC - PMC5790133
EDAT- 2016/12/23 06:00
MHDA- 2018/01/31 06:00
CRDT- 2016/12/23 06:00
PHST- 2016/06/17 00:00 [received]
PHST- 2016/12/20 00:00 [accepted]
PHST- 2016/12/23 06:00 [pubmed]
PHST- 2018/01/31 06:00 [medline]
PHST- 2016/12/23 06:00 [entrez]
AID - syw116 [pii]
AID - 10.1093/sysbio/syw116 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Sep 1;66(5):813-822. doi: 10.1093/sysbio/syw116.

PMID- 28003533
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - A Unifying Comparative Phylogenetic Framework Including Traits Coevolving Across 
      Interacting Lineages.
PG  - 551-568
LID - 10.1093/sysbio/syw115 [doi]
AB  - Models of phenotypic evolution fit to phylogenetic comparative data are widely
      used to make inferences regarding the tempo and mode of trait evolution. A wide
      range of models is already available for this type of analysis, and the field is 
      still under active development. One of the most needed development concerns
      models that better account for the effect of within- and between-clade
      interspecific interactions on trait evolution, which can result from processes as
      diverse as competition, predation, parasitism, or mutualism. Here, we begin by
      developing a very general comparative phylogenetic framework for (multi)-trait
      evolution that can be applied to both ultrametric and nonultrametric trees. This 
      framework not only encapsulates many previous models of continuous univariate and
      multivariate phenotypic evolution, but also paves the way for the consideration
      of a much broader series of models in which lineages coevolve, meaning that trait
      changes in one lineage are influenced by the value of traits in other,
      interacting lineages. Next, we provide a standard way for deriving the
      probabilistic distribution of traits at tip branches under our framework. We show
      that a multivariate normal distribution remains the expected distribution for a
      broad class of models accounting for interspecific interactions. Our derivations 
      allow us to fit various models efficiently, and in particular greatly reduce the 
      computation time needed to fit the recently proposed phenotype matching model.
      Finally, we illustrate the utility of our framework by developing a toy model for
      mutualistic coevolution. Our framework should foster a new era in the study of
      coevolution from comparative data.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Manceau, Marc
AU  - Manceau M
AD  - Museum National d'Histoire Naturelle, 75005 Paris, France.
AD  - Center for Interdisciplinary Research in Biology, College de France, CNRS UMR
      7241, 75005 Paris, France.
AD  - Institut de Biologie, Ecole Normale Superieure, CNRS UMR 8197, 75005 Paris,
      France.
FAU - Lambert, Amaury
AU  - Lambert A
AD  - Laboratoire Probabilites et Modeles Aleatoires, UPMC-University of Paris 06,
      75005 Paris France.
AD  - Center for Interdisciplinary Research in Biology, College de France, CNRS UMR
      7241, 75005 Paris, France.
FAU - Morlon, Helene
AU  - Morlon H
AD  - Institut de Biologie, Ecole Normale Superieure, CNRS UMR 8197, 75005 Paris,
      France.
LA  - eng
SI  - Dryad/10.5061/dryad.52636
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Models, Biological
MH  - Phenotype
MH  - *Phylogeny
MH  - Symbiosis
OTO - NOTNLM
OT  - *Character displacement
OT  - *coevolution
OT  - *comparative phylogenetics
OT  - *interspecific interactions
OT  - *linear stochastic differential equations
OT  - *trait evolution
EDAT- 2016/12/23 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/12/23 06:00
PHST- 2016/05/20 00:00 [received]
PHST- 2016/12/15 00:00 [accepted]
PHST- 2016/12/23 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/12/23 06:00 [entrez]
AID - syw115 [pii]
AID - 10.1093/sysbio/syw115 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):551-568. doi: 10.1093/sysbio/syw115.

PMID- 28003532
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 3
DP  - 2017 May 1
TI  - Uncovering Higher-Taxon Diversification Dynamics from Clade Age and
      Species-Richness Data.
PG  - 367-378
LID - 10.1093/sysbio/syw088 [doi]
AB  - The relationship between clade age and species richness has been increasingly
      used in macroevolutionary studies as evidence for ecologically versus
      time-dependent diversification processes. However, theory suggests that
      phylogenetic structure, age type (crown or stem age), and taxonomic delimitation 
      can affect estimates of the age-richness correlation (ARC) considerably. We
      currently lack an integrative understanding of how these different factors affect
      ARCs, which in turn, obscures further interpretations. To assess its informative 
      breadth, we characterize ARC behavior with simulated and empirical phylogenies,
      considering phylogenetic structure and both crown and stem ages. First, we
      develop a two-state birth-death model to simulate phylogenies including the
      origin of higher taxa and a hierarchical taxonomy to determine ARC expectations
      under ecologically and time-dependent diversification processes. Then, we
      estimate ARCs across various taxonomic ranks of extant amphibians, squamate
      reptiles, mammals, birds, and flowering plants. We find that our model reproduces
      the general ARC trends of a wide range of biological systems despite the
      particularities of taxonomic practice within each, suggesting that the model is
      adequate to establish a framework of ARC null expectations for different
      diversification processes when taxa are defined with a hierarchical taxonomy.
      ARCs estimated with crown ages were positive in all the scenarios we studied,
      including ecologically dependent processes. Negative ARCs were only found at less
      inclusive taxonomic ranks, when considering stem age, and when rates varied among
      clades. This was the case both in ecologically and time-dependent processes.
      Together, our results warn against direct interpretations of single ARC estimates
      and advocate for a more integrative use of ARCs across age types and taxonomic
      ranks in diversification studies. [Birth-Death models; crown age; diversity
      dependence; extinction; phylogenetic structure; speciation; stem age; taxonomy;
      time dependence; tree simulations.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Sanchez-Reyes, Luna L
AU  - Sanchez-Reyes LL
AD  - Instituto de Biologia, Universidad Nacional Autonoma de Mexico, 3er Circuito de
      Ciudad Universitaria, Coyoacan, Ciudad de Mexico 04510, Mexico.
AD  - Posgrado en Ciencias Biologicas, Universidad Nacional Autonoma de Mexico, Av.
      Universidad 3000, Coyoacan, Ciudad de Mexico 04510, Mexico.
FAU - Morlon, Helene
AU  - Morlon H
AD  - Ecole Normale Superieure, UMR 8197 CNRS, 46 rue d'Ulm, 75005, Paris, France.
FAU - Magallon, Susana
AU  - Magallon S
AD  - Instituto de Biologia, Universidad Nacional Autonoma de Mexico, 3er Circuito de
      Ciudad Universitaria, Coyoacan, Ciudad de Mexico 04510, Mexico.
LA  - eng
SI  - Dryad/10.5061/dryad.2b7d5
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biodiversity
MH  - Classification/*methods
MH  - Genetic Speciation
MH  - Magnoliopsida
MH  - *Models, Biological
MH  - *Phylogeny
EDAT- 2016/12/23 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/12/23 06:00
PHST- 2015/09/28 00:00 [received]
PHST- 2016/09/26 00:00 [accepted]
PHST- 2016/12/23 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/12/23 06:00 [entrez]
AID - syw088 [pii]
AID - 10.1093/sysbio/syw088 [doi]
PST - ppublish
SO  - Syst Biol. 2017 May 1;66(3):367-378. doi: 10.1093/sysbio/syw088.

PMID- 28003531
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Bayes Factors Unmask Highly Variable Information Content, Bias, and Extreme
      Influence in Phylogenomic Analyses.
PG  - 517-530
LID - 10.1093/sysbio/syw101 [doi]
AB  - As the application of genomic data in phylogenetics has become routine, a number 
      of cases have arisen where alternative data sets strongly support conflicting
      conclusions. This sensitivity to analytical decisions has prevented firm
      resolution of some of the most recalcitrant nodes in the tree of life. To better 
      understand the causes and nature of this sensitivity, we analyzed several
      phylogenomic data sets using an alternative measure of topological support (the
      Bayes factor) that both demonstrates and averts several limitations of more
      frequently employed support measures (such as Markov chain Monte Carlo estimates 
      of posterior probabilities). Bayes factors reveal important, previously hidden,
      differences across six "phylogenomic" data sets collected to resolve the
      phylogenetic placement of turtles within Amniota. These data sets vary
      substantially in their support for well-established amniote relationships,
      particularly in the proportion of genes that contain extreme amounts of
      information as well as the proportion that strongly reject these uncontroversial 
      relationships. All six data sets contain little information to resolve the
      phylogenetic placement of turtles relative to other amniotes. Bayes factors also 
      reveal that a very small number of extremely influential genes (less than 1% of
      genes in a data set) can fundamentally change significant phylogenetic
      conclusions. In one example, these genes are shown to contain previously
      unrecognized paralogs. This study demonstrates both that the resolution of
      difficult phylogenomic problems remains sensitive to seemingly minor analysis
      details and that Bayes factors are a valuable tool for identifying and solving
      these challenges.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Brown, Jeremy M
AU  - Brown JM
AD  - Department of Biological Sciences and Museum of Natural Science, Louisiana State 
      University, 202 Life Science Building, Baton Rouge, LA 70803, USA.
FAU - Thomson, Robert C
AU  - Thomson RC
AD  - Department of Biology, University of Hawaii at Manoa, 2538 McCarthy Mall,
      Edmondson Hall Rm 216, Honolulu, HI 96822, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.8gm85
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Bias
MH  - Classification/*methods
MH  - Genome
MH  - Markov Chains
MH  - Models, Statistical
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Expressed sequence tags
OT  - *negative constraints
OT  - *ortholog
OT  - *posterior probability
OT  - *ultraconserved elements
EDAT- 2016/12/23 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/12/23 06:00
PHST- 2016/01/19 00:00 [received]
PHST- 2016/10/21 00:00 [accepted]
PHST- 2016/12/23 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/12/23 06:00 [entrez]
AID - syw101 [pii]
AID - 10.1093/sysbio/syw101 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):517-530. doi: 10.1093/sysbio/syw101.

PMID- 27920231
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Bayesian Morphological Clock Methods Resurrect Placoderm Monophyly and Reveal
      Rapid Early Evolution in Jawed Vertebrates.
PG  - 499-516
LID - 10.1093/sysbio/syw107 [doi]
AB  - The phylogeny of early gnathostomes provides an important framework for
      understanding one of the most significant evolutionary events, the origin and
      diversification of jawed vertebrates. A series of recent cladistic analyses have 
      suggested that the placoderms, an extinct group of armoured fish, form a
      paraphyletic group basal to all other jawed vertebrates. We revised and expanded 
      this morphological data set, most notably by sampling autapomorphies in a similar
      way to parsimony-informative traits, thus ensuring this data (unlike most
      existing morphological data sets) satisfied an important assumption of Bayesian
      tip-dated morphological clock approaches. We also found problems with characters 
      supporting placoderm paraphyly, including character correlation and incorrect
      codings. Analysis of this data set reveals that paraphyly and monophyly of core
      placoderms (excluding maxillate forms) are essentially equally parsimonious. The 
      two alternative topologies have different root positions for the jawed
      vertebrates but are otherwise similar. However, analysis using tip-dated clock
      methods reveals strong support for placoderm monophyly, due to this analysis
      favoring trees with more balanced rates of evolution. Furthermore, enforcing
      placoderm paraphyly results in higher levels and unusual patterns of rate
      heterogeneity among branches, similar to that generated from simulated trees
      reconstructed with incorrect root positions. These simulations also show that
      Bayesian tip-dated clock methods outperform parsimony when the outgroup is
      largely uninformative (e.g., due to inapplicable characters), as might be the
      case here. The analysis also reveals that gnathostomes underwent a rapid burst of
      evolution during the Silurian period which declined during the Early Devonian.
      This rapid evolution during a period with few articulated fossils might partly
      explain the difficulty in ascertaining the root position of jawed vertebrates.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - King, Benedict
AU  - King B
AD  - School of Biological Sciences, Flinders University, PO Box 2100, Adelaide, South 
      Australia 5001, Australia.
FAU - Qiao, Tuo
AU  - Qiao T
AD  - Key Laboratory of Vertebrate Evolution and Human Origins of Chinese Academy of
      Sciences, Institute of Vertebrate Paleontology and Paleoanthropology, Chinese
      Academy of Sciences, PO Box 643, Beijing 100044, China.
FAU - Lee, Michael S Y
AU  - Lee MSY
AD  - School of Biological Sciences, Flinders University, PO Box 2100, Adelaide, South 
      Australia 5001, Australia.
AD  - Earth Sciences Section, South Australian Museum, North Terrace, Adelaide 5000,
      Australia.
FAU - Zhu, Min
AU  - Zhu M
AD  - Key Laboratory of Vertebrate Evolution and Human Origins of Chinese Academy of
      Sciences, Institute of Vertebrate Paleontology and Paleoanthropology, Chinese
      Academy of Sciences, PO Box 643, Beijing 100044, China.
FAU - Long, John A
AU  - Long JA
AD  - School of Biological Sciences, Flinders University, PO Box 2100, Adelaide, South 
      Australia 5001, Australia.
LA  - eng
SI  - Dryad/10.5061/dryad.v30f1
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - *Fossils
MH  - *Models, Biological
MH  - Phylogeny
MH  - Vertebrates
OTO - NOTNLM
OT  - *BEAST
OT  - *Bayesian
OT  - *monophyly
OT  - *morphological clock
OT  - *morphology
OT  - *phylogenetics
OT  - *placoderms
OT  - *tip dating
OT  - *tree topology
EDAT- 2016/12/07 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/12/07 06:00
PHST- 2016/07/01 00:00 [received]
PHST- 2016/11/18 00:00 [accepted]
PHST- 2016/12/07 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/12/07 06:00 [entrez]
AID - syw107 [pii]
AID - 10.1093/sysbio/syw107 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):499-516. doi: 10.1093/sysbio/syw107.

PMID- 27892425
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20180226
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - Natural Kinds and Classification in Scientific Practice. - Edited by Catherine
      Kendig.
PG  - 1120-1121
FAU - Ebach, Malte C
AU  - Ebach MC
AD  - Palaeontology, Geobiology and Earth Archives Research Centre, School of
      Biological, Earth and Environmental Sciences, UNSW, Kensington NSW 2052,
      Australia; Email: mcebach@gmail.com.
LA  - eng
PT  - Journal Article
DEP - 20160811
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2016/11/29 06:00
MHDA- 2016/11/29 06:01
CRDT- 2016/11/29 06:00
PHST- 2016/11/29 06:00 [entrez]
PHST- 2016/11/29 06:00 [pubmed]
PHST- 2016/11/29 06:01 [medline]
AID - syw067 [pii]
AID - 10.1093/sysbio/syw067 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):1120-1121. doi: 10.1093/sysbio/syw067. Epub 2016 Aug
      11.

PMID- 27892424
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20180226
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - The Invention of Nature: The Adventures of Alexander von Humboldt, the Lost Hero 
      of Science (UK). The Invention of Nature: Alexander von Humboldt's New World
      (USA). - By Andrea Wulf.
PG  - 1117-1119
FAU - Morrison, David A
AU  - Morrison DA
AD  - Systematic Biology, Evolutionary Biology Centre, Norbyvagen 18D, 752 36 Uppsala, 
      Sweden; E-mail: david.morrison@ebc.uu.se.
LA  - eng
PT  - Journal Article
DEP - 20160802
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2016/11/29 06:00
MHDA- 2016/11/29 06:01
CRDT- 2016/11/29 06:00
PHST- 2016/11/29 06:00 [entrez]
PHST- 2016/11/29 06:00 [pubmed]
PHST- 2016/11/29 06:01 [medline]
AID - syw062 [pii]
AID - 10.1093/sysbio/syw062 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):1117-1119. doi: 10.1093/sysbio/syw062. Epub 2016 Aug 2.

PMID- 27837192
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - New Evidence from Linguistic Phylogenetics Identifies Limits to Punctuational
      Change.
PG  - 604-610
LID - 10.1093/sysbio/syw106 [doi]
AB  - Since the early 1970s, biologists have debated whether evolution is punctuated by
      speciation events with bursts of cladogenetic changes, or whether evolution tends
      to be of a more gradual, anagenetic nature. A similar discussion among linguists 
      has barely begun, but the present results suggest that there is also room for
      controversy over this issue in linguistics. The only previous study correlated
      the number of nodes in linguistic phylogenies with branch lengths and found
      support for punctuated equilibrium. We replicate this result for branch lengths, 
      but find no support for punctuated equilibrium using a different, automated
      measure of linguistic divergence and a much larger data set. With the automated
      measure, segments of trees containing more nodes show no greater divergence from 
      an outgroup than segments containing fewer nodes.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Holman, Eric W
AU  - Holman EW
AD  - Department of Psychology, 502 Portola Plaza, University of California, Los
      Angeles, CA 90095, USA.
FAU - Wichmann, Soren
AU  - Wichmann S
AD  - Leiden University Centre for Linguistics, Leiden University, Van Wijkplaats 4,
      2311 BX Leiden, Netherlands; Humanities Center for Advanced Study "Words, Bones, 
      Genes, Tools", University of Tubingen, Rumelinstrasse 23, 72070 Tubingen,
      Germany; Kazan Federal University, 18 Kremlyovskaya Street, Kazan 420008, Russia.
LA  - eng
SI  - Dryad/10.5061/dryad.4gg07
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Genetic Speciation
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Historical linguistics
OT  - *language evolution
OT  - *lexical similarity
OT  - *phyletic gradualism
EDAT- 2016/11/12 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/11/13 06:00
PHST- 2015/11/20 00:00 [received]
PHST- 2016/11/09 00:00 [accepted]
PHST- 2016/11/12 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/11/13 06:00 [entrez]
AID - syw106 [pii]
AID - 10.1093/sysbio/syw106 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):604-610. doi: 10.1093/sysbio/syw106.

PMID- 27821704
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 3
DP  - 2017 May 1
TI  - Speciation with Gene Flow in North American Myotis Bats.
PG  - 440-452
LID - 10.1093/sysbio/syw100 [doi]
AB  - Growing evidence supports the idea that species can diverge in the presence of
      gene flow. However, most methods of phylogeny estimation do not consider this
      process, despite the fact that ignoring gene flow is known to bias phylogenetic
      inference. Furthermore, studies that do consider divergence-with-gene-flow
      typically do so by estimating rates of gene flow using a isolation-with-migration
      model (IM), rather than evaluating scenarios of gene flow (such as
      divergence-with-gene flow or secondary contact) that represent very different
      types of diversification. In this investigation, we aim to infer the recent
      phylogenetic history of a clade of western long-eared bats while evaluating a
      number of different models that parameterize gene flow in a variety of ways. We
      utilize PHRAPL, a new tool for phylogeographic model selection, to compare the
      fit of a broad set of demographic models that include divergence, migration, or
      both among Myotis evotis, $M$. thysanodes and M. keenii. A genomic data set
      consisting of 808 loci of ultraconserved elements was used to explore such models
      in three steps using an incremental design where each successive set was informed
      by, and thus more focused than, the previous set of models. Specifically, the
      three steps were to (i) assess whether gene flow should be modeled and identify
      the best topologies, (ii) infer directionality of migration using the best
      topologies, and (iii) estimate the timing of gene flow. The best model (AIC model
      weight ${\sim}0.98$) included two divergence events (($M$. evotis, $M$.
      thysanodes), M. keenii) accompanied by gene flow at the initial stages of
      divergence. These results provide a striking example of speciation-with-gene-flow
      in an evolutionary lineage. [Myotis bats; PHRAPL; P2C2M; phylogeographic model
      selection; speciation with gene flow.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Morales, Ariadna E
AU  - Morales AE
AD  - Department of Evolution, Ecology and Organismal Biology, Ohio State University,
      318 W. 12th Avenue, Columbus, OH 43210, USA.
FAU - Jackson, Nathan D
AU  - Jackson ND
AD  - Department of Ecology and Evolutionary Biology, University of Tennessee,
      Knoxville, 442 Hesler Biology Building, Knoxville, TN 37996, USA.
FAU - Dewey, Tanya A
AU  - Dewey TA
AD  - Department of Biology, Colorado State University, 1878 Campus Delivery, Fort
      Collins, CO 80523, USA.
FAU - O'Meara, Brian C
AU  - O'Meara BC
AD  - Department of Ecology and Evolutionary Biology, University of Tennessee,
      Knoxville, 442 Hesler Biology Building, Knoxville, TN 37996, USA.
FAU - Carstens, Bryan C
AU  - Carstens BC
AD  - Department of Evolution, Ecology and Organismal Biology, Ohio State University,
      318 W. 12th Avenue, Columbus, OH 43210, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.b0q2g
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Chiroptera/*classification/*genetics
MH  - *Gene Flow
MH  - *Genetic Speciation
MH  - Models, Biological
MH  - *Phylogeny
MH  - Software
MH  - United States
EDAT- 2016/11/09 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/11/09 06:00
PHST- 2015/11/24 00:00 [received]
PHST- 2016/10/20 00:00 [accepted]
PHST- 2016/11/09 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/11/09 06:00 [entrez]
AID - syw100 [pii]
AID - 10.1093/sysbio/syw100 [doi]
PST - ppublish
SO  - Syst Biol. 2017 May 1;66(3):440-452. doi: 10.1093/sysbio/syw100.

PMID- 27821703
OWN - NLM
STAT- MEDLINE
DCOM- 20180130
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 5
DP  - 2017 Sep 1
TI  - Do Macrophylogenies Yield Stable Macroevolutionary Inferences? An Example from
      Squamate Reptiles.
PG  - 843-856
LID - 10.1093/sysbio/syw102 [doi]
AB  - Advances in the generation, retrieval, and analysis of phylogenetic data have
      enabled researchers to create phylogenies that contain many thousands of taxa.
      These "macrophylogenies"-large trees that typically derive from megaphylogeny,
      supermatrix, or supertree approaches-provide researchers with an unprecedented
      ability to conduct evolutionary analyses across broad phylogenetic scales. Many
      studies have now used these phylogenies to explore the dynamics of speciation,
      extinction, and phenotypic evolution across large swaths of the tree of life.
      These trees are characterized by substantial phylogenetic uncertainty on multiple
      levels, and the stability of macroevolutionary inferences from these data sets
      has not been rigorously explored. As a case study, we tested whether five
      recently published phylogenies for squamate reptiles-each consisting of more than
      4000 species-yield congruent inferences about the processes that underlie
      variation in species richness across replicate evolutionary radiations of
      Australian snakes and lizards. We find discordance across the five focal
      phylogenies with respect to clade age and several diversification rate metrics,
      and in the effects of clade age on species richness. We also find that crown
      clade ages reported in the literature on these Australian groups are in conflict 
      with all of the large phylogenies examined. Macrophylogenies offer an
      unprecedented opportunity to address evolutionary and ecological questions at
      broad phylogenetic scales, but accurately representing the uncertainty that is
      inherent to such analyses remains a critical challenge to our field. [Australia; 
      macroevolution; macrophylogeny; squamates; time calibration.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Title, Pascal O
AU  - Title PO
AD  - Department of Ecology and Evolutionary Biology and Museum of Zoology, University 
      of Michigan, Ann Arbor, MI 48109, USA.
FAU - Rabosky, Daniel L
AU  - Rabosky DL
AD  - Department of Ecology and Evolutionary Biology and Museum of Zoology, University 
      of Michigan, Ann Arbor, MI 48109, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.60js5
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Australia
MH  - Biodiversity
MH  - Biological Evolution
MH  - Ecology
MH  - Lizards/*classification
MH  - *Phylogeny
MH  - Snakes/*classification
EDAT- 2016/11/09 06:00
MHDA- 2018/01/31 06:00
CRDT- 2016/11/09 06:00
PHST- 2015/12/30 00:00 [received]
PHST- 2016/10/27 00:00 [accepted]
PHST- 2016/11/09 06:00 [pubmed]
PHST- 2018/01/31 06:00 [medline]
PHST- 2016/11/09 06:00 [entrez]
AID - syw102 [pii]
AID - 10.1093/sysbio/syw102 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Sep 1;66(5):843-856. doi: 10.1093/sysbio/syw102.

PMID- 27664188
OWN - NLM
STAT- MEDLINE
DCOM- 20180108
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 2
DP  - 2017 Mar 1
TI  - Phylogenomics of Lophotrochozoa with Consideration of Systematic Error.
PG  - 256-282
LID - 10.1093/sysbio/syw079 [doi]
AB  - Phylogenomic studies have improved understanding of deep metazoan phylogeny and
      show promise for resolving incongruences among analyses based on limited numbers 
      of loci. One region of the animal tree that has been especially difficult to
      resolve, even with phylogenomic approaches, is relationships within
      Lophotrochozoa (the animal clade that includes molluscs, annelids, and flatworms 
      among others). Lack of resolution in phylogenomic analyses could be due to
      insufficient phylogenetic signal, limitations in taxon and/or gene sampling, or
      systematic error. Here, we investigated why lophotrochozoan phylogeny has been
      such a difficult question to answer by identifying and reducing sources of
      systematic error. We supplemented existing data with 32 new transcriptomes
      spanning the diversity of Lophotrochozoa and constructed a new set of
      Lophotrochozoa-specific core orthologs. Of these, 638 orthologous groups (OGs)
      passed strict screening for paralogy using a tree-based approach. In order to
      reduce possible sources of systematic error, we calculated branch-length
      heterogeneity, evolutionary rate, percent missing data, compositional bias, and
      saturation for each OG and analyzed increasingly stricter subsets of only the
      most stringent (best) OGs for these five variables. Principal component analysis 
      of the values for each factor examined for each OG revealed that compositional
      heterogeneity and average patristic distance contributed most to the variance
      observed along the first principal component while branch-length heterogeneity
      and, to a lesser extent, saturation contributed most to the variance observed
      along the second. Missing data did not strongly contribute to either. Additional 
      sensitivity analyses examined effects of removing taxa with heterogeneous branch 
      lengths, large amounts of missing data, and compositional heterogeneity. Although
      our analyses do not unambiguously resolve lophotrochozoan phylogeny, we advance
      the field by reducing the list of viable hypotheses. Moreover, our systematic
      approach for dissection of phylogenomic data can be applied to explore sources of
      incongruence and poor support in any phylogenomic data set. [Annelida;
      Brachiopoda; Bryozoa; Entoprocta; Mollusca; Nemertea; Phoronida; Platyzoa;
      Polyzoa; Spiralia; Trochozoa.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Kocot, Kevin M
AU  - Kocot KM
AD  - Department of Biological Sciences, 101 Rouse Life Sciences, Auburn University,
      Auburn, AL 36849, USA.
AD  - Department of Biological Sciences and Alabama Museum of Natural History, 307 Mary
      Harmon Bryant Hall, The University of Alabama, Tuscaloosa, AL 35487, USA.
FAU - Struck, Torsten H
AU  - Struck TH
AD  - Natural History Museum, Department of Research and Collections, University of
      Oslo, PO Box 1172 Blindern, N-0318 Oslo, Norway.
FAU - Merkel, Julia
AU  - Merkel J
AD  - Johannes Gutenberg University, Institute of Zoology, 55099 Mainz, Germany.
FAU - Waits, Damien S
AU  - Waits DS
AD  - Department of Biological Sciences, 101 Rouse Life Sciences, Auburn University,
      Auburn, AL 36849, USA.
FAU - Todt, Christiane
AU  - Todt C
AD  - University Museum of Bergen, The Natural History Collections, University of
      Bergen, Allegaten 41, 5007 Bergen, Norway.
FAU - Brannock, Pamela M
AU  - Brannock PM
AD  - Department of Biological Sciences, 101 Rouse Life Sciences, Auburn University,
      Auburn, AL 36849, USA.
FAU - Weese, David A
AU  - Weese DA
AD  - Department of Biological Sciences, 101 Rouse Life Sciences, Auburn University,
      Auburn, AL 36849, USA.
AD  - Department of Biological and Environmental Sciences, Georgia College and State
      University, Campus Box 81, Milledgeville, GA 31061 USA.
FAU - Cannon, Johanna T
AU  - Cannon JT
AD  - Department of Biological Sciences, 101 Rouse Life Sciences, Auburn University,
      Auburn, AL 36849, USA.
AD  - Department of Zoology, Naturhistoriska riksmuseet, Box 50007, 104 05 Stockholm,
      Sweden.
FAU - Moroz, Leonid L
AU  - Moroz LL
AD  - The Whitney Laboratory for Marine Bioscience, University of Florida, 9505 Ocean
      Shore Blvd, St Augustine, FL 32080, USA.
FAU - Lieb, Bernhard
AU  - Lieb B
AD  - Johannes Gutenberg University, Institute of Zoology, 55099 Mainz, Germany.
FAU - Halanych, Kenneth M
AU  - Halanych KM
AD  - Department of Biological Sciences, 101 Rouse Life Sciences, Auburn University,
      Auburn, AL 36849, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.30k4v
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bryozoa/*classification/*genetics
MH  - Classification/*methods
MH  - Genome/*genetics
MH  - *Phylogeny
EDAT- 2016/11/02 06:00
MHDA- 2018/01/09 06:00
CRDT- 2016/09/25 06:00
PHST- 2015/06/15 00:00 [received]
PHST- 2016/08/24 00:00 [accepted]
PHST- 2016/11/02 06:00 [pubmed]
PHST- 2018/01/09 06:00 [medline]
PHST- 2016/09/25 06:00 [entrez]
AID - syw079 [pii]
AID - 10.1093/sysbio/syw079 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Mar 1;66(2):256-282. doi: 10.1093/sysbio/syw079.

PMID- 27798407
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 3
DP  - 2017 May 1
TI  - Construction of a Species-Level Tree of Life for the Insects and Utility in
      Taxonomic Profiling.
PG  - 426-439
LID - 10.1093/sysbio/syw099 [doi]
AB  - Although comprehensive phylogenies have proven an invaluable tool in ecology and 
      evolution, their construction is made increasingly challenging both by the scale 
      and structure of publically available sequences. The distinct partition between
      gene-rich (genomic) and species-rich (DNA barcode) data is a feature of data that
      has been largely overlooked, yet presents a key obstacle to scaling supermatrix
      analysis. I present a phyloinformatics framework for draft construction of a
      species-level phylogeny of insects (Class Insecta). Matrix-building requires
      separately optimized pipelines for nuclear transcriptomic, mitochondrial genomic,
      and species-rich markers, whereas tree-building requires hierarchical inference
      in order to capture species-breadth while retaining deep-level resolution. The
      phylogeny of insects contains 49,358 species, 13,865 genera, 760 families.
      Deep-level splits largely reflected previous findings for sections of the tree
      that are data rich or unambiguous, such as inter-ordinal Endopterygota and
      Dictyoptera, the recently evolved and relatively homogeneous Lepidoptera,
      Hymenoptera, Brachycera (Diptera), and Cucujiformia (Coleoptera). However,
      analysis of bias, matrix construction and gene-tree variation suggests confidence
      in some relationships (such as in Polyneoptera) is less than has been indicated
      by the matrix bootstrap method. To assess the utility of the insect tree as a
      tool in query profiling several tree-based taxonomic assignment methods are
      compared. Using test data sets with existing taxonomic annotations, a tendency is
      observed for greater accuracy of species-level assignments where using a fixed
      comprehensive tree of life in contrast to methods generating smaller de novo
      reference trees. Described herein is a solution to the discrepancy in the way
      data are fit into supermatrices. The resulting tree facilitates wider studies of 
      insect diversification and application of advanced descriptions of diversity in
      community studies, among other presumed applications. [Data integration; data
      mining; insects; phylogenomics; phyloinformatics; tree of life.].
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Chesters, Douglas
AU  - Chesters D
AD  - Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology,
      Chinese Academy of Sciences, Beijing 100101, China.
LA  - eng
SI  - Dryad/10.5061/dryad.27114
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Insecta/*classification/genetics
MH  - *Phylogeny
PMC - PMC5837528
EDAT- 2016/11/01 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/11/01 06:00
PHST- 2015/09/20 00:00 [received]
PHST- 2016/10/18 00:00 [accepted]
PHST- 2016/11/01 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/11/01 06:00 [entrez]
AID - syw099 [pii]
AID - 10.1093/sysbio/syw099 [doi]
PST - ppublish
SO  - Syst Biol. 2017 May 1;66(3):426-439. doi: 10.1093/sysbio/syw099.

PMID- 27798406
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Biodiversity and the Species Concept-Lineages are not Enough.
PG  - 644-656
LID - 10.1093/sysbio/syw098 [doi]
AB  - The nature and definition of species continue to be matters of debate. Current
      views of species often focus on their nature as lineages-maximal reproductive
      communities through time. Whereas many authors point to the Evolutionary Species 
      Concept as optimal, in its original form it stressed the ecological role of
      species as well as their history as lineages, but most recent authors have
      ignored the role aspect of the concept, making it difficult to apply
      unambiguously in a time-extended way. This trend has been exacerbated by the
      application of methods and concepts emphasizing the notion of monophyly,
      originally applied only at higher levels, to the level of individuals, as well as
      by the current emphasis on molecular data. Hence, some current authors recognize 
      units that are no more than probable exclusive lineages as species. We argue that
      biodiversity is inherently a phenotypic concept and that role, as manifested in
      the organismal extended phenotype, is a necessary component of the species
      concept. Viewing species as historically connected populations with unique role
      brings together the temporal and phenotypic natures of species, providing a clear
      way to view species both in a time-limited and time-extended way. Doing so
      alleviates perceived issues with "paraphyletic species" and returns the focus of 
      species to units that are most relevant for biodiversity.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Freudenstein, John V
AU  - Freudenstein JV
AD  - Department of Evolution, Ecology and Organismal Biology, The Ohio State
      University Herbarium, 1315 Kinnear Road, Columbus, OH 43212, USA.
FAU - Broe, Michael B
AU  - Broe MB
AD  - Department of Evolution, Ecology and Organismal Biology, The Ohio State
      University Herbarium, 1315 Kinnear Road, Columbus, OH 43212, USA.
FAU - Folk, Ryan A
AU  - Folk RA
AD  - Department of Evolution, Ecology and Organismal Biology, The Ohio State
      University Herbarium, 1315 Kinnear Road, Columbus, OH 43212, USA.
AD  - Florida Museum of Natural History, University of Florida, 1659 Museum Road,
      Gainesville, FL 32611, USA.
FAU - Sinn, Brandon T
AU  - Sinn BT
AD  - Department of Evolution, Ecology and Organismal Biology, The Ohio State
      University Herbarium, 1315 Kinnear Road, Columbus, OH 43212, USA.
AD  - New York Botanical Garden, 2900 Southern Blvd., Bronx, NY 10458, USA.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biodiversity
MH  - Biological Evolution
MH  - *Classification
MH  - Ecology
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Coalescence
OT  - *Ecological Species Concept
OT  - *Evolutionary Species Concept
OT  - *General Lineage Concept
OT  - *lineage
OT  - *phenotype
OT  - *species
EDAT- 2016/11/01 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/11/01 06:00
PHST- 2016/04/15 00:00 [received]
PHST- 2016/10/14 00:00 [accepted]
PHST- 2016/11/01 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/11/01 06:00 [entrez]
AID - syw098 [pii]
AID - 10.1093/sysbio/syw098 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):644-656. doi: 10.1093/sysbio/syw098.

PMID- 27798405
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 3
DP  - 2017 May 1
TI  - Possibility and Challenges of Conversion of Current Virus Species Names to
      Linnaean Binomials.
PG  - 463-473
LID - 10.1093/sysbio/syw096 [doi]
AB  - Botanical, mycological, zoological, and prokaryotic species names follow the
      Linnaean format, consisting of an italicized Latinized binomen with a capitalized
      genus name and a lower case species epithet (e.g., Homo sapiens). Virus species
      names, however, do not follow a uniform format, and, even when binomial, are not 
      Linnaean in style. In this thought exercise, we attempted to convert all
      currently official names of species included in the virus family Arenaviridae and
      the virus order Mononegavirales to Linnaean binomials, and to identify and
      address associated challenges and concerns. Surprisingly, this endeavor was not
      as complicated or time-consuming as even the authors of this article expected
      when conceiving the experiment. [Arenaviridae; binomials; ICTV; International
      Committee on Taxonomy of Viruses; Mononegavirales; virus nomenclature; virus
      taxonomy.].
CI  - Published by Oxford University Press on behalf of Society of Systematic
      Biologists 2016. This work is written by a US Government employee and is in the
      public domain in the US.
FAU - Postler, Thomas S
AU  - Postler TS
AD  - Department of Microbiology and Immunology, Columbia University, College of
      Physicians &amp; Surgeons, New York, 10032 NY, USA.
FAU - Clawson, Anna N
AU  - Clawson AN
AD  - Integrated Research Facility at Fort Detrick, National Institute of Allergy and
      Infectious Diseases, National Institutes of Health, Frederick, 21702 MD, USA.
FAU - Amarasinghe, Gaya K
AU  - Amarasinghe GK
AD  - Department of Pathology and Immunology, Washington University School of Medicine,
      St. Louis, 63110 MO, USA.
FAU - Basler, Christopher F
AU  - Basler CF
AD  - Center for Microbial Pathogenesis, Institute for Biomedical Sciences, Georgia
      State University, Atlanta, 30303 GA, USA.
FAU - Bavari, Sbina
AU  - Bavari S
AD  - United States Army Medical Research Institute of Infectious Diseases, Fort
      Detrick, Frederick, 21702 MD, USA.
FAU - Benko, Maria
AU  - Benko M
AD  - Institute for Veterinary Medical Research, Centre for Agricultural Research,
      Hungarian Academy of Sciences, H-1581 Budapest, Hungary.
FAU - Blasdell, Kim R
AU  - Blasdell KR
AD  - Commonwealth Scientific and Industrial Research Organisation (CSIRO) Health and
      Biosecurity, Australian Animal Health Laboratory, Geelong, 3220 Victoria,
      Australia.
FAU - Briese, Thomas
AU  - Briese T
AD  - Center for Infection and Immunity, Mailman School of Public Health, Columbia
      University, New York, 10032 NY, USA.
FAU - Buchmeier, Michael J
AU  - Buchmeier MJ
AD  - Department of Molecular Biology and Biochemistry, University of California,
      Irvine, 92697 CA, USA.
FAU - Bukreyev, Alexander
AU  - Bukreyev A
AD  - University of Texas Medical Branch, TX 77555 Galveston, USA.
FAU - Calisher, Charles H
AU  - Calisher CH
AD  - Arthropod-Borne and Infectious Diseases Laboratory, College of Veterinary
      Medicine and Biomedical Sciences, Colorado State University, Fort Collins, 80523 
      CO, USA.
FAU - Chandran, Kartik
AU  - Chandran K
AD  - Department of Microbiology and Immunology, Albert Einstein College of Medicine,
      Bronx, 10461 NY, USA.
FAU - Charrel, Remi
AU  - Charrel R
AD  - MR "Emergence des Pathologies Virales" (EPV: Aix-Marseille Universite - IRD 190 -
      Inserm 1207 - EHESP), 13284 Marseille, France.
AD  - Institut Hospitalo-Universitaire Mediterranee Infection, APHM Public Hospitals of
      Marseille, 13015 Marseille, France.
FAU - Clegg, Christopher S
AU  - Clegg CS
AD  - Les Mandinaux, 16450 Le Grand-Madieu, France.
FAU - Collins, Peter L
AU  - Collins PL
AD  - Respiratory Viruses Section, Laboratory of Infectious Diseases, National
      Institute of Allergy and Infectious Diseases, National Institutes of Health,
      Bethesda, 20892 MD, USA.
FAU - Juan Carlos, De La Torre
AU  - Juan Carlos T
AD  - Department of Immunology and Microbial Science IMM-6, The Scripps Research
      Institute, La Jolla, 92037 CA, USA.
FAU - Derisi, Joseph L
AU  - Derisi JL
AD  - Departments of Medicine, Biochemistry and Biophysics, and Microbiology,
      University of California, San Francisco, 94158 CA, USA.
FAU - Dietzgen, Ralf G
AU  - Dietzgen RG
AD  - Queensland Alliance for Agriculture and Food Innovation, The University of
      Queensland, St. Lucia, 4072 Queensland, Australia.
FAU - Dolnik, Olga
AU  - Dolnik O
AD  - Institute of Virology, Philipps University Marburg, 35043 Marburg, Germany.
FAU - Durrwald, Ralf
AU  - Durrwald R
AD  - IDT Biologika, 06861 Dessau-Rosslau, Germany.
FAU - Dye, John M
AU  - Dye JM
AD  - United States Army Medical Research Institute of Infectious Diseases, Fort
      Detrick, Frederick, 21702 MD, USA.
FAU - Easton, Andrew J
AU  - Easton AJ
AD  - School of Life Sciences, University of Warwick, CV4 7AL Coventry, UK.
FAU - Emonet, Sebastian
AU  - Emonet S
AD  - Unite de Virologie, IRBA - Echelon Recherche de Lyon, 69007 Lyon, France.
FAU - Formenty, Pierre
AU  - Formenty P
AD  - World Health Organization, CH-1211 Geneva, Switzerland.
FAU - Fouchier, Ron A M
AU  - Fouchier RAM
AD  - Department of Viroscience, Postgraduate School Molecular Medicine, Erasmus
      University Medical Center, 3015 CE Rotterdam, The Netherlands.
FAU - Ghedin, Elodie
AU  - Ghedin E
AD  - Center for Genomics and Systems Biology, Department of Biology, New York
      University, New York, 10003 NY, USA.
FAU - Gonzalez, Jean-Paul
AU  - Gonzalez JP
AD  - Health for Development, Potomac, 20854 MD, USA.
FAU - Harrach, Balazs
AU  - Harrach B
AD  - Institute for Veterinary Medical Research, Centre for Agricultural Research,
      Hungarian Academy of Sciences, H-1581 Budapest, Hungary.
FAU - Hewson, Roger
AU  - Hewson R
AD  - Public Health England, Porton Down, Wiltshire, SP4 0JG Salisbury, UK.
FAU - Horie, Masayuki
AU  - Horie M
AD  - Transboundary Animal Diseases Research Center, Joint Faculty of Veterinary
      Medicine, Kagoshima University, 890-0065 Japan Korimoto Kagoshima, Japan.
FAU - Jiang, Daohong
AU  - Jiang D
AD  - State Key Laboratory of Agricultural Microbiology, The Provincial Key Lab of
      Plant Pathology of Huboi Province, College of Plant Science and Technology,
      Huazhong Agricultural University, Wuhan, 430070 China, China.
FAU - Kobinger, Gary
AU  - Kobinger G
AD  - Department of Microbiology, Immunology and Infectious Diseases, Faculty of
      Medicine, Universite Laval, Quebec, QC G1V 0A6, Canada, Canada.
FAU - Kondo, Hideki
AU  - Kondo H
AD  - Institute of Plant Science and Resources, Okayama University, Kurashiki, 710-0046
      Japan, Japan.
FAU - Kropinski, Andrew M
AU  - Kropinski AM
AD  - Departments of Food Science, Molecular and Cellular Biology, and Pathobiology,
      University of Guelph, Guelph, N1G 2W1 Ontario, Canada.
FAU - Krupovic, Mart
AU  - Krupovic M
AD  - Unite de Biologie Moleculaire du Gene chez les Extremophiles, Department of
      Microbiology, Institut Pasteur, 75015 Paris, France.
FAU - Kurath, Gael
AU  - Kurath G
AD  - US Geological Survey Western Fisheries Research Center, Seattle, 98115 WA, USA.
FAU - Lamb, Robert A
AU  - Lamb RA
AD  - Department of Molecular Biosciences, Northwestern University, Evanston, 60208 IL,
      USA.
AD  - Howard Hughes Medical Institute, Northwestern University, Evanston, 60208 IL,
      USA.
FAU - Leroy, Eric M
AU  - Leroy EM
AD  - Centre International de Recherches Medicales de Franceville, Institut de
      Recherche pour le Developpement, Franceville, Gabon.
FAU - Lukashevich, Igor S
AU  - Lukashevich IS
AD  - Department of Pharmacology and Toxicology, School of Medicine, and the Center for
      Predictive Medicine for Biodefense and Emerging Infectious Diseases, University
      of Louisville, Louisville, 40202 KY, USA.
FAU - Maisner, Andrea
AU  - Maisner A
AD  - Institute of Virology, Philipps University Marburg, 35043 Marburg, Germany.
FAU - Mushegian, Arcady R
AU  - Mushegian AR
AD  - Division of Molecular and Cellular Biosciences, National Science Foundation,
      Arlington, 22230, USA.
FAU - Netesov, Sergey V
AU  - Netesov SV
AD  - Novosibirsk State University, Novosibirsk, Novosibirsk Oblast, 630090 Russia,
      Russia.
FAU - Nowotny, Norbert
AU  - Nowotny N
AD  - Institute of Virology, University of Veterinary Medicine, 1210 Vienna, Austria.
AD  - Department of Basic Medical Sciences, College of Medicine, Mohammed Bin Rashid
      University of Medicine and Health Sciences, Dubai, United Arab Emirates.
FAU - Patterson, Jean L
AU  - Patterson JL
AD  - Department of Virology and Immunology, Texas Biomedical Research Institute, San
      Antonio, 78230 TX, USA.
FAU - Payne, Susan L
AU  - Payne SL
AD  - Department of Veterinary Pathobiology, College of Veterinary Medicine and
      Biomedical Sciences, Texas A&amp;M University, College Station, 77845 TX, USA.
FAU - PaWeska, Janusz T
AU  - PaWeska JT
AD  - Center for Emerging and Zoonotic Diseases, National Institute for Communicable
      Diseases of the National Health Laboratory Service, 2131 Sandringham,
      Johannesburg, Gauteng, South Africa.
FAU - Peters, Clarence J
AU  - Peters CJ
AD  - University of Texas Medical Branch, TX 77555 Galveston, USA.
FAU - Radoshitzky, Sheli R
AU  - Radoshitzky SR
AD  - United States Army Medical Research Institute of Infectious Diseases, Fort
      Detrick, Frederick, 21702 MD, USA.
FAU - Rima, Bertus K
AU  - Rima BK
AD  - Centre for Experimental Medicine, School of Medicine, Dentistry and Biomedical
      Sciences, Queen's University of Belfast, Belfast, Belfast BT7 1NN, Northern
      Ireland, UK.
FAU - Romanowski, Victor
AU  - Romanowski V
AD  - Instituto de Biotecnologia y Biologia Molecular (CONICET-UNLP), Facultad de
      Ciencias Exactas, Universidad Nacional de La Plata, 1900 La Plata, Argentina.
FAU - Rubbenstroth, Dennis
AU  - Rubbenstroth D
AD  - Institute for Virology, Medical Center - University of Freiburg, Faculty of
      Medicine, University of Freiburg, 79104 Freiburg, Germany.
FAU - Sabanadzovic, Sead
AU  - Sabanadzovic S
AD  - Department of Biochemistry, Molecular Biology, Entomology and Plant Pathology,
      Mississippi State University, 39762 MS, USA.
FAU - Sanfacon, Helene
AU  - Sanfacon H
AD  - Summerland Research and Development Centre, Agriculture and Agri-Food Canada,
      Summerland, V0H 1Z0 British Columbia, Canada.
FAU - Salvato, Maria S
AU  - Salvato MS
AD  - Institute of Human Virology, University of Maryland School of Medicine,
      Baltimore, 21201 MD, USA.
FAU - Schwemmle, Martin
AU  - Schwemmle M
AD  - Institute for Virology, Medical Center - University of Freiburg, Faculty of
      Medicine, University of Freiburg, 79104 Freiburg, Germany.
FAU - Smither, Sophie J
AU  - Smither SJ
AD  - Chemical, Biological, and Radiological Division, Defence Science and Technology
      Laboratory, Porton Down, Salisbury, SP4 0JQ Wiltshire, UK.
FAU - Stenglein, Mark D
AU  - Stenglein MD
AD  - Department of Microbiology, Immunology and Pathology, Colorado State University, 
      Fort Collins, 80523 CO, USA.
FAU - Stone, David M
AU  - Stone DM
AD  - Centre for Environment, Fisheries and Aquaculture Science Weymouth, DT4 8UB
      Dorset, UK.
FAU - Takada, Ayato
AU  - Takada A
AD  - Division of Global Epidemiology, Hokkaido University Research Center for Zoonosis
      Control, 001-0020 Sapporo, Japan.
FAU - Tesh, Robert B
AU  - Tesh RB
AD  - University of Texas Medical Branch, TX 77555 Galveston, USA.
FAU - Tomonaga, Keizo
AU  - Tomonaga K
AD  - Institute for Virus Research, Kyoto University, 6068507 Kyoto, Japan.
FAU - Tordo, Noel
AU  - Tordo N
AD  - Institut Pasteur, Unite des Strategies Antivirales, 75015 Paris, France.
AD  - Institut Pasteur de Guinee, Conakry, Guinea.
FAU - Towner, Jonathan S
AU  - Towner JS
AD  - Viral Special Pathogens Branch, Division of High-Consequence Pathogens and
      Pathology, National Center for Emerging and Zoonotic Infectious Diseases, Centers
      for Disease Control and Prevention, Atlanta, 30333 GA, USA.
FAU - Vasilakis, Nikos
AU  - Vasilakis N
AD  - University of Texas Medical Branch, TX 77555 Galveston, USA.
FAU - Volchkov, Viktor E
AU  - Volchkov VE
AD  - Molecular Basis of Viral Pathogenicity, CIRI, INSERM U1111 - CNRS UMR5308,
      Universite de Lyon, Universite Claude Bernard Lyon 1, Ecole Normale Superieure de
      Lyon, 69365 Lyon, France.
FAU - Wahl-Jensen, Victoria
AU  - Wahl-Jensen V
AD  - National Biodefense Analysis and Countermeasures Center, Fort Detrick, Frederick,
      21702 MD, USA.
FAU - Walker, Peter J
AU  - Walker PJ
AD  - Commonwealth Scientific and Industrial Research Organisation (CSIRO) Health and
      Biosecurity, Australian Animal Health Laboratory, Geelong, 3220 Victoria,
      Australia.
FAU - Wang, Lin-Fa
AU  - Wang LF
AD  - Department of Agriculture and Fisheries, Biosecurity Queensland, Brisbane, 4000
      Queensland, Australia.
AD  - Programme in Emerging Infectious Diseases, Duke-NUS Medical School, 1659857
      Singapore, Singapore.
FAU - Varsani, Arvind
AU  - Varsani A
AD  - The Center for Fundamental and Applied Microbiomics, The Biodesign Institute and 
      School of Life Sciences, Arizona State University, Tempe, 85287 AZ, USA.
FAU - Whitfield, Anna E
AU  - Whitfield AE
AD  - Plant Pathology, Kansas State University, Manhattan, 66506 KS, USA.
FAU - Zerbini, F Murilo
AU  - Zerbini FM
AD  - Department de Fitopatologia/BIOAGRO, Universidade Federal de Vicosa, Vicosa - MG,
      36570-900, Brazil, Brazil.
FAU - Kuhn, Jens H
AU  - Kuhn JH
AD  - Integrated Research Facility at Fort Detrick, National Institute of Allergy and
      Infectious Diseases, National Institutes of Health, Frederick, 21702 MD, USA.
LA  - eng
GR  - R01 AI107056/AI/NIAID NIH HHS/United States
GR  - U19 AI109945/AI/NIAID NIH HHS/United States
GR  - R24 AI120942/AI/NIAID NIH HHS/United States
GR  - P01 AI120943/AI/NIAID NIH HHS/United States
GR  - R01 AI114654/AI/NIAID NIH HHS/United States
GR  - 001/World Health Organization/International
GR  - U19 AI109664/AI/NIAID NIH HHS/United States
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Classification
MH  - Terminology as Topic
MH  - *Viruses
PMC - PMC5837305
EDAT- 2016/11/01 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/11/01 06:00
PHST- 2016/08/05 00:00 [received]
PHST- 2016/10/17 00:00 [accepted]
PHST- 2016/11/01 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/11/01 06:00 [entrez]
AID - syw096 [pii]
AID - 10.1093/sysbio/syw096 [doi]
PST - ppublish
SO  - Syst Biol. 2017 May 1;66(3):463-473. doi: 10.1093/sysbio/syw096.

PMID- 27798404
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 3
DP  - 2017 May 1
TI  - The Impact of the Tree Prior on Molecular Dating of Data Sets Containing a
      Mixture of Inter- and Intraspecies Sampling.
PG  - 413-425
LID - 10.1093/sysbio/syw095 [doi]
AB  - In Bayesian phylogenetic analyses of genetic data, prior probability
      distributions need to be specified for the model parameters, including the tree. 
      When Bayesian methods are used for molecular dating, available tree priors
      include those designed for species-level data, such as the pure-birth and
      birth-death priors, and coalescent-based priors designed for population-level
      data. However, molecular dating methods are frequently applied to data sets that 
      include multiple individuals across multiple species. Such data sets violate the 
      assumptions of both the speciation and coalescent-based tree priors, making it
      unclear which should be chosen and whether this choice can affect the estimation 
      of node times. To investigate this problem, we used a simulation approach to
      produce data sets with different proportions of within- and between-species
      sampling under the multispecies coalescent model. These data sets were then
      analyzed under pure-birth, birth-death, constant-size coalescent, and skyline
      coalescent tree priors. We also explored the ability of Bayesian model testing to
      select the best-performing priors. We confirmed the applicability of our results 
      to empirical data sets from cetaceans, phocids, and coregonid whitefish.
      Estimates of node times were generally robust to the choice of tree prior, but
      some combinations of tree priors and sampling schemes led to large differences in
      the age estimates. In particular, the pure-birth tree prior frequently led to
      inaccurate estimates for data sets containing a mixture of inter- and
      intraspecific sampling, whereas the birth-death and skyline coalescent priors
      produced stable results across all scenarios. Model testing provided an adequate 
      means of rejecting inappropriate tree priors. Our results suggest that tree
      priors do not strongly affect Bayesian molecular dating results in most cases,
      even when severely misspecified. However, the choice of tree prior can be
      significant for the accuracy of dating results in the case of data sets with
      mixed inter- and intraspecies sampling. [Bayesian phylogenetic methods; model
      testing; molecular dating; node time; tree prior.].
CI  - (c) The authors 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      e-mail: journals.permission@oup.com.
FAU - Ritchie, Andrew M
AU  - Ritchie AM
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney, NSW
      2006, Australia.
FAU - Lo, Nathan
AU  - Lo N
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney, NSW
      2006, Australia.
FAU - Ho, Simon Y W
AU  - Ho SYW
AD  - School of Life and Environmental Sciences, University of Sydney, Sydney, NSW
      2006, Australia.
LA  - eng
SI  - Dryad/10.5061/dryad.fb1fq
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Databases, Genetic
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Time
EDAT- 2016/11/01 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/11/01 06:00
PHST- 2016/05/22 00:00 [received]
PHST- 2016/10/14 00:00 [accepted]
PHST- 2016/11/01 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/11/01 06:00 [entrez]
AID - syw095 [pii]
AID - 10.1093/sysbio/syw095 [doi]
PST - ppublish
SO  - Syst Biol. 2017 May 1;66(3):413-425. doi: 10.1093/sysbio/syw095.

PMID- 27798403
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 3
DP  - 2017 May 1
TI  - A Relaxed Directional Random Walk Model for Phylogenetic Trait Evolution.
PG  - 299-319
LID - 10.1093/sysbio/syw093 [doi]
AB  - Understanding the processes that give rise to quantitative measurements
      associated with molecular sequence data remains an important issue in statistical
      phylogenetics. Examples of such measurements include geographic coordinates in
      the context of phylogeography and phenotypic traits in the context of comparative
      studies. A popular approach is to model the evolution of continuously varying
      traits as a Brownian diffusion process acting on a phylogenetic tree. However,
      standard Brownian diffusion is quite restrictive and may not accurately
      characterize certain trait evolutionary processes. Here, we relax one of the
      major restrictions of standard Brownian diffusion by incorporating a nontrivial
      estimable mean into the process. We introduce a relaxed directional random walk
      (RDRW) model for the evolution of multivariate continuously varying traits along 
      a phylogenetic tree. Notably, the RDRW model accommodates branch-specific
      variation of directional trends while preserving model identifiability.
      Furthermore, our development of a computationally efficient dynamic programming
      approach to compute the data likelihood enables scaling of our method to large
      data sets frequently encountered in phylogenetic comparative studies and viral
      evolution. We implement the RDRW model in a Bayesian inference framework to
      simultaneously reconstruct the evolutionary histories of molecular sequence data 
      and associated multivariate continuous trait data, and provide tools to visualize
      evolutionary reconstructions. We demonstrate the performance of our model on
      synthetic data, and we illustrate its utility in two viral examples. First, we
      examine the spatiotemporal spread of HIV-1 in central Africa and show that the
      RDRW model uncovers a clearer, more detailed picture of the dynamics of viral
      dispersal than standard Brownian diffusion. Second, we study antigenic evolution 
      in the context of HIV-1 resistance to three broadly neutralizing antibodies. Our 
      analysis reveals evidence of a continuous drift at the HIV-1 population level
      towards enhanced resistance to neutralization by the VRC01 monoclonal antibody
      over the course of the epidemic. [Brownian Motion; Diffusion Processes;
      Phylodynamics; Phylogenetics; Phylogeography; Trait Evolution.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Gill, Mandev S
AU  - Gill MS
AD  - Department of Statistics, Columbia University, New York, NY 10027, USA.
FAU - Tung Ho, Lam Si
AU  - Tung Ho LS
AD  - Department of Biostatistics, Jonathan and Karin Fielding School of Public Health,
      University of California, Los Angeles, CA 90095, USA.
FAU - Baele, Guy
AU  - Baele G
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven,
      Minderbroedersstaat 10, 3000, Leuven, Belgium.
FAU - Lemey, Philippe
AU  - Lemey P
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven,
      Minderbroedersstaat 10, 3000, Leuven, Belgium.
FAU - Suchard, Marc A
AU  - Suchard MA
AD  - Department of Biostatistics, Jonathan and Karin Fielding School of Public Health,
      University of California, Los Angeles, CA 90095, USA.
AD  - Department of Biomathematics, David Geffen School of Medicine at UCLA, University
      of California, Los Angeles, CA 90095, USA.
AD  - Department of Human Genetics, David Geffen School of Medicine at UCLA, Universtiy
      of California, Los Angeles, CA, USA.
LA  - eng
GR  - R01 AI107034/AI/NIAID NIH HHS/United States
GR  - T32 AI007370/AI/NIAID NIH HHS/United States
GR  - R01 HG006139/HG/NHGRI NIH HHS/United States
GR  - R01 LM011247/LM/NLM NIH HHS/United States
GR  - 260864/European Research Council/International
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Africa
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - HIV Infections/epidemiology/immunology/virology
MH  - HIV-1/classification/immunology
MH  - Humans
MH  - *Models, Biological
MH  - Phenotype
MH  - *Phylogeny
PMC - PMC6075548
EDAT- 2016/11/01 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/11/01 06:00
PHST- 2015/12/17 00:00 [received]
PHST- 2016/10/10 00:00 [accepted]
PHST- 2016/11/01 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/11/01 06:00 [entrez]
AID - syw093 [pii]
AID - 10.1093/sysbio/syw093 [doi]
PST - ppublish
SO  - Syst Biol. 2017 May 1;66(3):299-319. doi: 10.1093/sysbio/syw093.

PMID- 27798402
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 3
DP  - 2017 May 1
TI  - Misconceptions on Missing Data in RAD-seq Phylogenetics with a Deep-scale Example
      from Flowering Plants.
PG  - 399-412
LID - 10.1093/sysbio/syw092 [doi]
AB  - Restriction-site associated DNA (RAD) sequencing and related methods rely on the 
      conservation of enzyme recognition sites to isolate homologous DNA fragments for 
      sequencing, with the consequence that mutations disrupting these sites lead to
      missing information. There is thus a clear expectation for how missing data
      should be distributed, with fewer loci recovered between more distantly related
      samples. This observation has led to a related expectation: that RAD-seq data are
      insufficiently informative for resolving deeper scale phylogenetic relationships.
      Here we investigate the relationship between missing information among samples at
      the tips of a tree and information at edges within it. We re-analyze and review
      the distribution of missing data across ten RAD-seq data sets and carry out
      simulations to determine expected patterns of missing information. We also
      present new empirical results for the angiosperm clade Viburnum (Adoxaceae, with 
      a crown age &gt;50 Ma) for which we examine phylogenetic information at different
      depths in the tree and with varied sequencing effort. The total number of loci,
      the proportion that are shared, and phylogenetic informativeness varied
      dramatically across the examined RAD-seq data sets. Insufficient or uneven
      sequencing coverage accounted for similar proportions of missing data as dropout 
      from mutation-disruption. Simulations reveal that mutation-disruption, which
      results in phylogenetically distributed missing data, can be distinguished from
      the more stochastic patterns of missing data caused by low sequencing coverage.
      In Viburnum, doubling sequencing coverage nearly doubled the number of parsimony 
      informative sites, and increased by &gt;10X the number of loci with data shared
      across &gt;40 taxa. Our analysis leads to a set of practical recommendations for
      maximizing phylogenetic information in RAD-seq studies. [hierarchical redundancy;
      phylogenetic informativeness; quartet informativeness; Restriction-site
      associated DNA (RAD) sequencing; sequencing coverage; Viburnum.].
CI  - (c) The authors 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For permissions, please
      e-mail: journals.permission@oup.com.
FAU - Eaton, Deren A R
AU  - Eaton DAR
AD  - Department of Ecology and Evolutionary Biology, Yale University, PO Box 208106,
      New Haven, CT, 06520, USA.
FAU - Spriggs, Elizabeth L
AU  - Spriggs EL
AD  - Department of Ecology and Evolutionary Biology, Yale University, PO Box 208106,
      New Haven, CT, 06520, USA.
FAU - Park, Brian
AU  - Park B
AD  - Department of Ecology and Evolutionary Biology, Yale University, PO Box 208106,
      New Haven, CT, 06520, USA.
FAU - Donoghue, Michael J
AU  - Donoghue MJ
AD  - Department of Ecology and Evolutionary Biology, Yale University, PO Box 208106,
      New Haven, CT, 06520, USA.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Base Sequence
MH  - Computer Simulation
MH  - Magnoliopsida/*classification/*genetics
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
EDAT- 2016/11/01 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/11/01 06:00
PHST- 2016/03/29 00:00 [received]
PHST- 2016/10/10 00:00 [accepted]
PHST- 2016/11/01 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/11/01 06:00 [entrez]
AID - syw092 [pii]
AID - 10.1093/sysbio/syw092 [doi]
PST - ppublish
SO  - Syst Biol. 2017 May 1;66(3):399-412. doi: 10.1093/sysbio/syw092.

PMID- 27780899
OWN - NLM
STAT- MEDLINE
DCOM- 20180108
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 2
DP  - 2017 Mar 1
TI  - Displayed Trees Do Not Determine Distinguishability Under the Network
      Multispecies Coalescent.
PG  - 283-298
LID - 10.1093/sysbio/syw097 [doi]
AB  - Recent work in estimating species relationships from gene trees has included
      inferring networks assuming that past hybridization has occurred between species.
      Probabilistic models using the multispecies coalescent can be used in this
      framework for likelihood-based inference of both network topologies and
      parameters, including branch lengths and hybridization parameters. A difficulty
      for such methods is that it is not always clear whether, or to what extent,
      networks are identifiable-that is whether there could be two distinct networks
      that lead to the same distribution of gene trees. For cases in which incomplete
      lineage sorting occurs in addition to hybridization, we demonstrate a new
      representation of the species network likelihood that expresses the probability
      distribution of the gene tree topologies as a linear combination of gene tree
      distributions given a set of species trees. This representation makes it clear
      that in some cases in which two distinct networks give the same distribution of
      gene trees when sampling one allele per species, the two networks can be
      distinguished theoretically when multiple individuals are sampled per species.
      This result means that network identifiability is not only a function of the
      trees displayed by the networks but also depends on allele sampling within
      species. We additionally give an example in which two networks that display
      exactly the same trees can be distinguished from their gene trees even when there
      is only one lineage sampled per species. [gene tree, hybridization,
      identifiability, maximum likelihood, species tree, phylogeny.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Zhu, Sha
AU  - Zhu S
AD  - Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford OX3 7BN,
      UK.
FAU - Degnan, James H
AU  - Degnan JH
AD  - Department of Mathematics and Statistics, University of New Mexico, Albuquerque, 
      NM 87110, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.t2d38
GR  - R01 GM117590/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Hybridization, Genetic
MH  - Likelihood Functions
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Species Specificity
PMC - PMC5837799
EDAT- 2016/11/01 06:00
MHDA- 2018/01/09 06:00
CRDT- 2016/11/01 06:00
PHST- 2015/03/10 00:00 [received]
PHST- 2016/03/08 00:00 [accepted]
PHST- 2016/11/01 06:00 [pubmed]
PHST- 2018/01/09 06:00 [medline]
PHST- 2016/11/01 06:00 [entrez]
AID - syw097 [pii]
AID - 10.1093/sysbio/syw097 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Mar 1;66(2):283-298. doi: 10.1093/sysbio/syw097.

PMID- 27770056
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - What is Intuitive Taxonomic Practice?
PG  - 637-643
LID - 10.1093/sysbio/syw094 [doi]
AB  - Scotland and Steel (2015) recently explored the idea of character compatibility, 
      examining the issue from the perspective of a particular model, "a simple and
      extreme model in which each character either fits perfectly on some tree, or is
      entirely random $\ldots$,." (Scotland and Steel 2015, p. 492, abstract). They
      suggested that character compatibility, when formalized as a phylogenetic method,
      captured what they believed was the "intuitive taxonomic practice of recognizing 
      taxa based on conserved nonhomoplastic characters" (Scotland and Steel 2015, p.
      493). Although we agree that there is much to be said for compatibility analysis,
      and "it has largely been set aside, initially in favour of maximum parsimony,
      and, more recently, by model-based methods for inferring phylogeny from DNA
      sequence data" (Scotland and Steel 2015, p. 493), their use of the expression
      "intuitive taxonomic practice" attracted our attention. Below we discuss in more 
      detail what "intuitive taxonomic practice" might be and relate that understanding
      to recent progress in the history of biology, specifically the history of
      systematics (taxonomy), and finally sketch out a proposal that might satisfy
      those of us who retain an interest in capturing in a more rigorous way what
      "intuitive taxonomic practice" might have been. [Compatibility methods;
      monothetic taxa; polythetic taxa."History is what we have to struggle to remember
      even when legend [myth] is more pleasing" (Adam Gopnik, The New Yorker, 23
      November 2015).
CI  - The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Williams, David M
AU  - Williams DM
AD  - Department of Life Sciences, the Natural History Museum, London SW7 5BD, London, 
      UK.
FAU - Ebach, Malte C
AU  - Ebach MC
AD  - Palaeontology, Geobiology and Earth Archives Research Centre, School of
      Biological, Earth and Environmental Sciences, UNSW, Kensington NSW 2052,
      Australia.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2017 Jul 1;66(4):662. PMID: 28460129
MH  - *Classification
MH  - Models, Biological
MH  - Phylogeny
EDAT- 2016/11/01 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/11/01 06:00
PHST- 2016/01/25 00:00 [received]
PHST- 2016/10/14 00:00 [accepted]
PHST- 2016/11/01 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/11/01 06:00 [entrez]
AID - syw094 [pii]
AID - 10.1093/sysbio/syw094 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):637-643. doi: 10.1093/sysbio/syw094.

PMID- 27590192
OWN - NLM
STAT- MEDLINE
DCOM- 20180108
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 2
DP  - 2017 Mar 1
TI  - Barrier Displacement on a Neutral Landscape: Toward a Theory of Continental
      Biogeography.
PG  - 167-182
LID - 10.1093/sysbio/syw080 [doi]
AB  - Macroevolutionary theory posits three processes leading to lineage
      diversification and the formation of regional biotas: dispersal (species
      geographic range expansion), speciation (species lineage splitting), and
      extinction (species lineage termination). The Theory of Island Biogeography (TIB)
      predicts species richness values using just two of these processes; dispersal and
      extinction. Yet most species on Earth live on continents or continental shelves, 
      and the dynamics of evolutionary diversification at regional and continental
      scales are qualitatively different from those that govern the formation of
      species richness on biogeographic islands. Certain geomorphological processes
      operating perennially on continental platforms displace barriers to gene flow and
      organismal dispersal, and affect all three terms of macroevolutionary
      diversification. For example, uplift of a dissected landscape and river capture
      both merge and separate portions of adjacent areas, allowing dispersal and larger
      geographic ranges, vicariant speciation and smaller geographic ranges, and
      extinction when range sizes are subdivided below a minimum persistence threshold.
      The TIB also does not predict many biogeographic and phylogenetic patterns widely
      observed in continentally distributed taxa, including: (i) power function-like
      species-area relationships; (ii) log-normal distribution of species geographic
      range sizes, in which most species have restricted ranges (are endemic) and few
      species have broad ranges (are cosmopolitan); (iii) mid-domain effects with more 
      species toward the geographic center, and more early-branching, species-poor
      clades toward the geographic periphery; (iv) exponential rates of net
      diversification with log-linear accumulation of lineages through geological time;
      and (v) power function-like relationships between species-richness and clade
      diversity, in which most clades are species-poor and few clades are species-rich.
      Current theory does not provide a robust mechanistic framework to connect these
      seemingly disparate patterns. Here we present SEAMLESS (Spatially Explicit Area
      Model of Landscape Evolution by SimulationS) that generates clade diversification
      by moving geographic barriers on a continuous, neutral landscape. SEAMLESS is a
      neutral Landscape Evolution Model (LEM) that treats species and barriers as
      functionally equivalent with respect to model parameters. SEAMLESS differs from
      other model-based biogeographic methods (e.g., Lagrange, GeoSSE, BayArea, and
      BioGeoBEARS) by modeling properties of dispersal barriers rather than areas, and 
      by modeling the evolution of species lineages on a continuous landscape, rather
      than the evolution of geographic ranges along branches of a phylogeny. SEAMLESS
      shows how dispersal is required to maintain species richness and avoid clade-wide
      extinction, demonstrates that ancestral range size does not predict species
      richness, and provides a unified explanation for the suite of commonly observed
      biogeographic and phylogenetic patterns listed above. SEAMLESS explains how a
      simple barrier-displacement mechanism affects lineage diversification under
      neutral conditions, and is advanced here toward the formulation of a general
      theory of continental biogeography. [Diversification, extinction, geodispersal,
      macroevolution, river capture, vicariance.].
CI  - Published by Oxford University Press on behalf of Society of Systematic
      Biologists 2016. This work is written by a US Government employee and is in the
      public domain in the US.
FAU - Albert, James S
AU  - Albert JS
AD  - Department of Biology, University of Louisiana at Lafayette, 104 E. University
      Circle, Lafayette, LA 70503, USA.
FAU - Schoolmaster, Donald R Jr
AU  - Schoolmaster DR Jr
AD  - WARC US Geological Survey, 700 Cajundome Blvd Lafayette, LA 70506 USA.
FAU - Tagliacollo, Victor
AU  - Tagliacollo V
AD  - Universidade Federal do Tocantins Avenida NS 15, 109 Norte Palmas, Tocantins
      77001-090, Brazil.
FAU - Duke-Sylvester, Scott M
AU  - Duke-Sylvester SM
AD  - Department of Biology, University of Louisiana at Lafayette, 104 E. University
      Circle, Lafayette, LA 70503, USA.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biodiversity
MH  - *Biological Evolution
MH  - Computer Simulation
MH  - *Genetic Speciation
MH  - *Models, Biological
MH  - Phylogeny
EDAT- 2016/11/01 06:00
MHDA- 2018/01/09 06:00
CRDT- 2016/09/04 06:00
PHST- 2016/04/07 00:00 [received]
PHST- 2016/08/23 00:00 [accepted]
PHST- 2016/11/01 06:00 [pubmed]
PHST- 2018/01/09 06:00 [medline]
PHST- 2016/09/04 06:00 [entrez]
AID - syw080 [pii]
AID - 10.1093/sysbio/syw080 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Mar 1;66(2):167-182. doi: 10.1093/sysbio/syw080.

PMID- 27576546
OWN - NLM
STAT- MEDLINE
DCOM- 20180108
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 2
DP  - 2017 Mar 1
TI  - Efficient Detection of Repeating Sites to Accelerate Phylogenetic Likelihood
      Calculations.
PG  - 205-217
LID - 10.1093/sysbio/syw075 [doi]
AB  - The phylogenetic likelihood function (PLF) is the major computational bottleneck 
      in several applications of evolutionary biology such as phylogenetic inference,
      species delimitation, model selection, and divergence times estimation. Given the
      alignment, a tree and the evolutionary model parameters, the likelihood function 
      computes the conditional likelihood vectors for every node of the tree. Vector
      entries for which all input data are identical result in redundant likelihood
      operations which, in turn, yield identical conditional values. Such operations
      can be omitted for improving run-time and, using appropriate data structures,
      reducing memory usage. We present a fast, novel method for identifying and
      omitting such redundant operations in phylogenetic likelihood calculations, and
      assess the performance improvement and memory savings attained by our method.
      Using empirical and simulated data sets, we show that a prototype implementation 
      of our method yields up to 12-fold speedups and uses up to 78% less memory than
      one of the fastest and most highly tuned implementations of the PLF currently
      available. Our method is generic and can seamlessly be integrated into any
      phylogenetic likelihood implementation. [Algorithms; maximum likelihood;
      phylogenetic likelihood function; phylogenetics].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Kobert, K
AU  - Kobert K
AD  - Scientific Computing Group, Heidelberg Institute for Theoretical Studies,
      Schoss-wolysbronnenweg 35, 69118 Heidelberg, Germany.
FAU - Stamatakis, A
AU  - Stamatakis A
AD  - Scientific Computing Group, Heidelberg Institute for Theoretical Studies,
      Schobeta-wolysbronnenweg 35, 69118 Heidelberg, Germany.
AD  - Karlsruhe Institute of Technology, Institute for Theoretical Informatics,
      Postfach 6980, 76128 Karlsruhe, Germany.
FAU - Flouri, T
AU  - Flouri T
AD  - Scientific Computing Group, Heidelberg Institute for Theoretical Studies,
      Schobeta-wolysbronnenweg 35, 69118 Heidelberg, Germany.
AD  - Karlsruhe Institute of Technology, Institute for Theoretical Informatics,
      Postfach 6980, 76128 Karlsruhe, Germany.
LA  - eng
SI  - Dryad/10.5061/dryad.8f0c8
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Evolution, Molecular
MH  - Likelihood Functions
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Software
PMC - PMC5837535
EDAT- 2016/11/01 06:00
MHDA- 2018/01/09 06:00
CRDT- 2016/09/01 06:00
PHST- 2016/03/16 00:00 [received]
PHST- 2016/08/23 00:00 [accepted]
PHST- 2016/11/01 06:00 [pubmed]
PHST- 2018/01/09 06:00 [medline]
PHST- 2016/09/01 06:00 [entrez]
AID - syw075 [pii]
AID - 10.1093/sysbio/syw075 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Mar 1;66(2):205-217. doi: 10.1093/sysbio/syw075.

PMID- 27694311
OWN - NLM
STAT- MEDLINE
DCOM- 20180108
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 2
DP  - 2017 Mar 1
TI  - Infomap Bioregions: Interactive Mapping of Biogeographical Regions from Species
      Distributions.
PG  - 197-204
LID - 10.1093/sysbio/syw087 [doi]
AB  - Biogeographical regions (bioregions) reveal how different sets of species are
      spatially grouped and therefore are important units for conservation, historical 
      biogeography, ecology, and evolution. Several methods have been developed to
      identify bioregions based on species distribution data rather than expert
      opinion. One approach successfully applies network theory to simplify and
      highlight the underlying structure in species distributions. However, this method
      lacks tools for simple and efficient analysis. Here, we present Infomap
      Bioregions, an interactive web application that inputs species distribution data 
      and generates bioregion maps. Species distributions may be provided as
      georeferenced point occurrences or range maps, and can be of local, regional, or 
      global scale. The application uses a novel adaptive resolution method to make
      best use of often incomplete species distribution data. The results can be
      downloaded as vector graphics, shapefiles, or in table format. We validate the
      tool by processing large data sets of publicly available species distribution
      data of the world's amphibians using species ranges, and mammals using point
      occurrences. We then calculate the fit between the inferred bioregions and WWF
      ecoregions. As examples of applications, researchers can reconstruct ancestral
      ranges in historical biogeography or identify indicator species for targeted
      conservation. [Biogeography; bioregionalization; conservation; mapping].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Edler, Daniel
AU  - Edler D
AD  - Integrated Science Lab, Department of Physics, Umea University, SE-901 87 Umea,
      Sweden.
AD  - Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, SE-405 30 Gothenburg, Sweden.
FAU - Guedes, Thais
AU  - Guedes T
AD  - Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, SE-405 30 Gothenburg, Sweden.
AD  - Federal University of Sao Paulo, 09972-270 Diadema, Brazil.
AD  - Museum of Zoology of University of Sao Paulo, 04263-000 Sao Paulo, Brazil.
FAU - Zizka, Alexander
AU  - Zizka A
AD  - Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, SE-405 30 Gothenburg, Sweden.
FAU - Rosvall, Martin
AU  - Rosvall M
AD  - Integrated Science Lab, Department of Physics, Umea University, SE-901 87 Umea,
      Sweden.
FAU - Antonelli, Alexandre
AU  - Antonelli A
AD  - Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, SE-405 30 Gothenburg, Sweden.
AD  - Gothenburg Botanical Garden, Carl Skottsbergs Gata 22A, 413 19 Gothenburg,
      Sweden.
LA  - eng
SI  - Dryad/10.5061/dryad.2s201
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amphibians/physiology
MH  - *Animal Distribution
MH  - Animals
MH  - Conservation of Natural Resources/*methods
MH  - Ecology/*methods
MH  - Internet
MH  - Mammals/physiology
MH  - Maps as Topic
MH  - Phylogeny
MH  - Phylogeography/*methods
MH  - Software
PMC - PMC5410963
EDAT- 2016/10/04 06:00
MHDA- 2018/01/09 06:00
CRDT- 2016/10/04 06:00
PHST- 2015/12/01 00:00 [received]
PHST- 2016/09/26 00:00 [accepted]
PHST- 2016/10/04 06:00 [pubmed]
PHST- 2018/01/09 06:00 [medline]
PHST- 2016/10/04 06:00 [entrez]
AID - syw087 [pii]
AID - 10.1093/sysbio/syw087 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Mar 1;66(2):197-204. doi: 10.1093/sysbio/syw087.

PMID- 27650175
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 3
DP  - 2017 May 1
TI  - Evaluating the Impact of Genomic Data and Priors on Bayesian Estimates of the
      Angiosperm Evolutionary Timescale.
PG  - 338-351
LID - 10.1093/sysbio/syw086 [doi]
AB  - The evolutionary timescale of angiosperms has long been a key question in
      biology. Molecular estimates of this timescale have shown considerable variation,
      being influenced by differences in taxon sampling, gene sampling, fossil
      calibrations, evolutionary models, and choices of priors. Here, we analyze a data
      set comprising 76 protein-coding genes from the chloroplast genomes of 195 taxa
      spanning 86 families, including novel genome sequences for 11 taxa, to evaluate
      the impact of models, priors, and gene sampling on Bayesian estimates of the
      angiosperm evolutionary timescale. Using a Bayesian relaxed molecular-clock
      method, with a core set of 35 minimum and two maximum fossil constraints, we
      estimated that crown angiosperms arose 221 (251-192) Ma during the Triassic.
      Based on a range of additional sensitivity and subsampling analyses, we found
      that our date estimates were generally robust to large changes in the parameters 
      of the birth-death tree prior and of the model of rate variation across branches.
      We found an exception to this when we implemented fossil calibrations in the form
      of highly informative gamma priors rather than as uniform priors on node ages.
      Under all other calibration schemes, including trials of seven maximum age
      constraints, we consistently found that the earliest divergences of angiosperm
      clades substantially predate the oldest fossils that can be assigned
      unequivocally to their crown group. Overall, our results and experiments with
      genome-scale data suggest that reliable estimates of the angiosperm crown age
      will require increased taxon sampling, significant methodological changes, and
      new information from the fossil record. [Angiospermae, chloroplast, genome,
      molecular dating, Triassic.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Foster, Charles S P
AU  - Foster CSP
AD  - School of Life and Environmental Sciences, Edgeworth David Building A11,
      University of Sydney, Sydney, New South Wales 2006, Australia.
FAU - Sauquet, Herve
AU  - Sauquet H
AD  - Laboratoire ecologie, Systematique, evolution, Universite Paris-Sud, CNRS UMR
      8079, bat. 360, Orsay 91405, France.
FAU - van der Merwe, Marlien
AU  - van der Merwe M
AD  - National Herbarium of New South Wales, Royal Botanic Gardens &amp; Domain Trust, Mrs 
      Macquaries Road, Sydney, New South Wales 2000, Australia.
FAU - McPherson, Hannah
AU  - McPherson H
AD  - National Herbarium of New South Wales, Royal Botanic Gardens &amp; Domain Trust, Mrs 
      Macquaries Road, Sydney, New South Wales 2000, Australia.
FAU - Rossetto, Maurizio
AU  - Rossetto M
AD  - National Herbarium of New South Wales, Royal Botanic Gardens &amp; Domain Trust, Mrs 
      Macquaries Road, Sydney, New South Wales 2000, Australia.
FAU - Ho, Simon Y W
AU  - Ho SYW
AD  - School of Life and Environmental Sciences, Edgeworth David Building A11,
      University of Sydney, Sydney, New South Wales 2006, Australia.
LA  - eng
SI  - Dryad/10.5061/dryad.3vj50
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - Evolution, Molecular
MH  - Fossils
MH  - Genome, Plant/*genetics
MH  - Magnoliopsida/*classification/*genetics
MH  - *Phylogeny
MH  - Time
EDAT- 2016/09/22 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/09/22 06:00
PHST- 2015/08/07 00:00 [received]
PHST- 2016/09/10 00:00 [accepted]
PHST- 2016/09/22 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/09/22 06:00 [entrez]
AID - syw086 [pii]
AID - 10.1093/sysbio/syw086 [doi]
PST - ppublish
SO  - Syst Biol. 2017 May 1;66(3):338-351. doi: 10.1093/sysbio/syw086.

PMID- 27637567
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 3
DP  - 2017 May 1
TI  - Ancestral Gene Flow and Parallel Organellar Genome Capture Result in Extreme
      Phylogenomic Discord in a Lineage of Angiosperms.
PG  - 320-337
LID - 10.1093/sysbio/syw083 [doi]
AB  - While hybridization has recently received a resurgence of attention from
      systematists and evolutionary biologists, there remains a dearth of case studies 
      on ancient, diversified hybrid lineages-clades of organisms that originated
      through reticulation. Studies on these groups are valuable in that they would
      speak to the long-term phylogenetic success of lineages following gene flow
      between species. We present a phylogenomic view of Heuchera, long known for
      frequent hybridization, incorporating all three independent genomes: targeted
      nuclear (~400,000 bp), plastid (~160,000 bp), and mitochondrial (~470,000 bp)
      data. We analyze these data using multiple concatenation and coalescence
      strategies. The nuclear phylogeny is consistent with previous work and with
      morphology, confidently suggesting a monophyletic Heuchera. By contrast, analyses
      of both organellar genomes recover a grossly polyphyletic Heuchera,consisting of 
      three primary clades with relationships extensively rearranged within these as
      well. A minority of nuclear loci also exhibit phylogenetic discord; yet these
      topologies remarkably never resemble the pattern of organellar loci and largely
      present low levels of discord inter alia. Two independent estimates of the
      coalescent branch length of the ancestor of Heuchera using nuclear data suggest
      rare or nonexistent incomplete lineage sorting with related clades, inconsistent 
      with the observed gross polyphyly of organellar genomes (confirmed by simulation 
      of gene trees under the coalescent). These observations, in combination with
      previous work, strongly suggest hybridization as the cause of this phylogenetic
      discord. [Ancient hybridization; chloroplast capture; incongruence;
      phylogenomics; reticulation.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Folk, Ryan A
AU  - Folk RA
AD  - Herbarium, Department of Evolution, Ecology and Organismal Biology, The Ohio
      State University, Columbus, OH 43212, USA and.
FAU - Mandel, Jennifer R
AU  - Mandel JR
AD  - Department of Biological Sciences, University of Memphis, Memphis, TN 38152, USA.
FAU - Freudenstein, John V
AU  - Freudenstein JV
AD  - Herbarium, Department of Evolution, Ecology and Organismal Biology, The Ohio
      State University, Columbus, OH 43212, USA and.
LA  - eng
SI  - Dryad/10.5061/dryad.cd546
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - *Gene Flow
MH  - Genome, Plant/*genetics
MH  - Magnoliopsida/*classification/*genetics
MH  - *Phylogeny
EDAT- 2016/09/18 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/09/18 06:00
PHST- 2015/08/27 00:00 [received]
PHST- 2016/09/04 00:00 [accepted]
PHST- 2016/09/18 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/09/18 06:00 [entrez]
AID - syw083 [pii]
AID - 10.1093/sysbio/syw083 [doi]
PST - ppublish
SO  - Syst Biol. 2017 May 1;66(3):320-337. doi: 10.1093/sysbio/syw083.

PMID- 27633354
OWN - NLM
STAT- MEDLINE
DCOM- 20180108
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 2
DP  - 2017 Mar 1
TI  - Who Let the CAT Out of the Bag? Accurately Dealing with Substitutional
      Heterogeneity in Phylogenomic Analyses.
PG  - 232-255
LID - 10.1093/sysbio/syw084 [doi]
AB  - As phylogenetic datasets have increased in size, site-heterogeneous substitution 
      models such as CAT-F81 and CAT-GTR have been advocated in favor of other models
      because they purportedly suppress long-branch attraction (LBA). These models are 
      two of the most commonly used models in phylogenomics, and they have been applied
      to a variety of taxa, ranging from Drosophila to land plants. However, many
      arguments in favor of CAT models have been based on tenuous assumptions about the
      true phylogeny, rather than rigorous testing with known trees via simulation.
      Moreover, CAT models have not been compared to other approaches for handling
      substitutional heterogeneity such as data partitioning with site-homogeneous
      substitution models. We simulated amino acid sequence datasets with
      substitutional heterogeneity on a variety of tree shapes including those
      susceptible to LBA. Data were analyzed with both CAT models and partitioning to
      explore model performance; in total over 670,000 CPU hours were used, of which
      over 97% was spent running analyses with CAT models. In many cases, all models
      recovered branching patterns that were identical to the known tree. However,
      CAT-F81 consistently performed worse than other models in inferring the correct
      branching patterns, and both CAT models often overestimated substitutional
      heterogeneity. Additionally, reanalysis of two empirical metazoan datasets
      supports the notion that CAT-F81 tends to recover less accurate trees than data
      partitioning and CAT-GTR. Given these results, we conclude that partitioning and 
      CAT-GTR perform similarly in recovering accurate branching patterns. However,
      computation time can be orders of magnitude less for data partitioning, with
      commonly used implementations of CAT-GTR often failing to reach completion in a
      reasonable time frame (i.e., for Bayesian analyses to converge). Practices such
      as removing constant sites and parsimony uninformative characters, or using
      CAT-F81 when CAT-GTR is deemed too computationally expensive, cannot be logically
      justified. Given clear problems with CAT-F81, phylogenies previously inferred
      with this model should be reassessed. [Data partitioning; phylogenomics,
      simulation, site-heterogeneity, substitution models.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Whelan, Nathan V
AU  - Whelan NV
AD  - Department of Biological Sciences, Molette Biology Laboratory for Environmental
      and Climate Change Studies, Auburn University, 101 Life Sciences Building,
      Auburn, AL 36849, USA.
FAU - Halanych, Kenneth M
AU  - Halanych KM
AD  - Department of Biological Sciences, Molette Biology Laboratory for Environmental
      and Climate Change Studies, Auburn University, 101 Life Sciences Building,
      Auburn, AL 36849, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.85b2m
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amino Acid Sequence
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Models, Genetic
MH  - *Phylogeny
EDAT- 2016/09/17 06:00
MHDA- 2018/01/09 06:00
CRDT- 2016/09/17 06:00
PHST- 2015/09/19 00:00 [received]
PHST- 2016/09/04 00:00 [accepted]
PHST- 2016/09/17 06:00 [pubmed]
PHST- 2018/01/09 06:00 [medline]
PHST- 2016/09/17 06:00 [entrez]
AID - syw084 [pii]
AID - 10.1093/sysbio/syw084 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Mar 1;66(2):232-255. doi: 10.1093/sysbio/syw084.

PMID- 27633353
OWN - NLM
STAT- MEDLINE
DCOM- 20180108
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 2
DP  - 2017 Mar 1
TI  - Phylogenetic Tree Estimation With and Without Alignment: New Distance Methods and
      Benchmarking.
PG  - 218-231
LID - 10.1093/sysbio/syw074 [doi]
AB  - Phylogenetic tree inference is a critical component of many systematic and
      evolutionary studies. The majority of these studies are based on the two-step
      process of multiple sequence alignment followed by tree inference, despite
      persistent evidence that the alignment step can lead to biased results. Here we
      present a two-part study that first presents PaHMM-Tree, a novel neighbor
      joining-based method that estimates pairwise distances without assuming a single 
      alignment. We then use simulations to benchmark its performance against a
      wide-range of other phylogenetic tree inference methods, including the first
      comparison of alignment-free distance-based methods against more conventional
      tree estimation methods. Our new method for calculating pairwise distances based 
      on statistical alignment provides distance estimates that are as accurate as
      those obtained using standard methods based on the true alignment. Pairwise
      distance estimates based on the two-step process tend to be substantially less
      accurate. This improved performance carries through to tree inference, where
      PaHMM-Tree provides more accurate tree estimates than all of the pairwise
      distance methods assessed. For close to moderately divergent sequence data we
      find that the two-step methods using statistical inference, where information
      from all sequences is included in the estimation procedure, tend to perform
      better than PaHMM-Tree, particularly full statistical alignment, which
      simultaneously estimates both the tree and the alignment. For deep divergences we
      find the alignment step becomes so prone to error that our distance-based
      PaHMM-Tree outperforms all other methods of tree inference. Finally, we find that
      the accuracy of alignment-free methods tends to decline faster than standard
      two-step methods in the presence of alignment uncertainty, and identify no
      conditions where alignment-free methods are equal to or more accurate than
      standard phylogenetic methods even in the presence of substantial alignment
      error. [Alignment-free; distance-based phylogenetics; pair Hidden Markov Models; 
      phylogenetic inference; statistical alignment.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Bogusz, Marcin
AU  - Bogusz M
AD  - Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala
      University, Norbyvagen 18D, 752 36 Uppsala, Sweden.
FAU - Whelan, Simon
AU  - Whelan S
AD  - Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala
      University, Norbyvagen 18D, 752 36 Uppsala, Sweden.
LA  - eng
SI  - Dryad/10.5061/dryad.n5r49
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Benchmarking
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Evolution, Molecular
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - *Sequence Alignment
EDAT- 2016/09/17 06:00
MHDA- 2018/01/09 06:00
CRDT- 2016/09/17 06:00
PHST- 2016/02/03 00:00 [received]
PHST- 2016/08/23 00:00 [accepted]
PHST- 2016/09/17 06:00 [pubmed]
PHST- 2018/01/09 06:00 [medline]
PHST- 2016/09/17 06:00 [entrez]
AID - syw074 [pii]
AID - 10.1093/sysbio/syw074 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Mar 1;66(2):218-231. doi: 10.1093/sysbio/syw074.

PMID- 27616324
OWN - NLM
STAT- MEDLINE
DCOM- 20180108
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 2
DP  - 2017 Mar 1
TI  - Toward a Self-Updating Platform for Estimating Rates of Speciation and Migration,
      Ages, and Relationships of Taxa.
PG  - 152-166
LID - 10.1093/sysbio/syw066 [doi]
AB  - Rapidly growing biological data-including molecular sequences and fossils-hold an
      unprecedented potential to reveal how evolutionary processes generate and
      maintain biodiversity. However, researchers often have to develop their own
      idiosyncratic workflows to integrate and analyze these data for reconstructing
      time-calibrated phylogenies. In addition, divergence times estimated under
      different methods and assumptions, and based on data of various quality and
      reliability, should not be combined without proper correction. Here we introduce 
      a modular framework termed SUPERSMART (Self-Updating Platform for Estimating
      Rates of Speciation and Migration, Ages, and Relationships of Taxa), and provide 
      a proof of concept for dealing with the moving targets of evolutionary and
      biogeographical research. This framework assembles comprehensive data sets of
      molecular and fossil data for any taxa and infers dated phylogenies using robust 
      species tree methods, also allowing for the inclusion of genomic data produced
      through next-generation sequencing techniques. We exemplify the application of
      our method by presenting phylogenetic and dating analyses for the mammal order
      Primates and for the plant family Arecaceae (palms). We believe that this
      framework will provide a valuable tool for a wide range of hypothesis-driven
      research questions in systematics, biogeography, and evolution. SUPERSMART will
      also accelerate the inference of a "Dated Tree of Life" where all node ages are
      directly comparable. [Bayesian phylogenetics; data mining; divide-and-conquer
      methods; GenBank; multilocus multispecies coalescent; next-generation sequencing;
      palms; primates; tree calibration.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Antonelli, Alexandre
AU  - Antonelli A
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Goteborg, Sweden.
AD  - Gothenburg Botanical Garden, Carl Skottsbergs Gata 22A, SE-41319 Goteborg,
      Sweden.
FAU - Hettling, Hannes
AU  - Hettling H
AD  - Naturalis Biodiversity Center, Darwinweg 4, 2333 CR Leiden, The Netherlands.
FAU - Condamine, Fabien L
AU  - Condamine FL
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Goteborg, Sweden.
AD  - CNRS, UMR 5554 Institut des Sciences de l'Evolution (Universite de Montpellier), 
      Place Eugene Bataillon, 34095 Montpellier, France.
FAU - Vos, Karin
AU  - Vos K
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Goteborg, Sweden.
FAU - Nilsson, R Henrik
AU  - Nilsson RH
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Goteborg, Sweden.
FAU - Sanderson, Michael J
AU  - Sanderson MJ
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, 1041 E.
      Lowell, Tucson, AZ 85721, USA.
FAU - Sauquet, Herve
AU  - Sauquet H
AD  - Universite Paris-Sud, Laboratoire Ecologie, Systematique, Evolution, CNRS UMR
      8079, 91405 Orsay, France.
FAU - Scharn, Ruud
AU  - Scharn R
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Goteborg, Sweden.
FAU - Silvestro, Daniele
AU  - Silvestro D
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Goteborg, Sweden.
AD  - Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne,
      Switzerland.
FAU - Topel, Mats
AU  - Topel M
AD  - Swedish Bioinformatics Infrastructure for Life Sciences, Department of Biological
      and Environmental Sciences, University of Gothenburg, Box 463, SE-405 30,
      Goteborg, Sweden.
AD  - Department of Marine Sciences, University of Gothenburg, Box 460, SE-405 30
      Goteborg, Sweden.
FAU - Bacon, Christine D
AU  - Bacon CD
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Goteborg, Sweden.
FAU - Oxelman, Bengt
AU  - Oxelman B
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Goteborg, Sweden.
FAU - Vos, Rutger A
AU  - Vos RA
AD  - Naturalis Biodiversity Center, Darwinweg 4, 2333 CR Leiden, The Netherlands.
LA  - eng
SI  - Dryad/10.5061/dryad.sk81k
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Age Factors
MH  - Animal Migration
MH  - Animals
MH  - Arecaceae/classification
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - *Fossils
MH  - *Phylogeny
MH  - Primates/classification
MH  - Reproducibility of Results
MH  - Time
PMC - PMC5410925
EDAT- 2016/09/13 06:00
MHDA- 2018/01/09 06:00
CRDT- 2016/09/13 06:00
PHST- 2015/11/30 00:00 [received]
PHST- 2016/07/19 00:00 [accepted]
PHST- 2016/09/13 06:00 [pubmed]
PHST- 2018/01/09 06:00 [medline]
PHST- 2016/09/13 06:00 [entrez]
AID - syw066 [pii]
AID - 10.1093/sysbio/syw066 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Mar 1;66(2):152-166. doi: 10.1093/sysbio/syw066.

PMID- 27616323
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 3
DP  - 2017 May 1
TI  - Species Distributions, Quantum Theory, and the Enhancement of Biodiversity
      Measures.
PG  - 453-462
LID - 10.1093/sysbio/syw072 [doi]
AB  - Species distributions are typically represented by records of their observed
      occurrence at a given spatial and temporal scale. Such records are inevitably
      incomplete and contingent on the spatial-temporal circumstances under which the
      observations were made. Moreover, organisms may respond differently to similar
      environmental conditions at different places or moments, so their distribution
      is, in principle, not completely predictable. We argue that this uncertainty
      exists, and warrants considering species distributions as analogous to coherent
      quantum objects, whose distributions are better described by a wavefunction
      rather than by a set of locations. We use this to extend the existing concept of 
      "dark diversity", which incorporates into biodiversity metrics those species that
      could, but which have not yet been observed to, inhabit a region-thereby
      developing the idea of "potential biodiversity". We show how conceptualizing
      species' distributions in this way could help overcome important weaknesses in
      current biodiversity metrics, both in theory and by using a worked case study of 
      mammal distributions in Spain over the last decade. We propose that considerable 
      theoretical advances could eventually be gained through interdisciplinary
      collaboration between biogeographers and quantum physicists. [Biogeography;
      favorability; physics; predictability; probability; species occurrence;
      uncertainty; wavefunction.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Real, Raimundo
AU  - Real R
AD  - Biogeography, Diversity and Conservation Lab, Departamento de Biologia Animal,
      Facultad de Ciencias, Universidad de Malaga, 29071 Malaga, Spain.
FAU - Barbosa, A Marcia
AU  - Barbosa AM
AD  - CIBIO/InBIO-University of Evora, 7004-516 Evora, Portugal.
FAU - Bull, Joseph W
AU  - Bull JW
AD  - Department of Food and Resource Economics and Center for Macroecology, Evolution 
      and Climate, University of Copenhagen, Rolighedsvej 23, 1958 Copenhagen, Denmark.
LA  - eng
SI  - Dryad/10.5061/dryad.gn6qb
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Animal Distribution
MH  - Animals
MH  - *Biodiversity
MH  - Mammals/classification/physiology
MH  - Phylogeny
MH  - Quantum Theory
MH  - Spain
EDAT- 2016/09/13 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/09/13 06:00
PHST- 2015/11/26 00:00 [received]
PHST- 2016/08/19 00:00 [accepted]
PHST- 2016/09/13 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/09/13 06:00 [entrez]
AID - syw072 [pii]
AID - 10.1093/sysbio/syw072 [doi]
PST - ppublish
SO  - Syst Biol. 2017 May 1;66(3):453-462. doi: 10.1093/sysbio/syw072.

PMID- 27539485
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 4
DP  - 2017 Jul 1
TI  - Disentangling Incomplete Lineage Sorting and Introgression to Refine Species-Tree
      Estimates for Lake Tanganyika Cichlid Fishes.
PG  - 531-550
LID - 10.1093/sysbio/syw069 [doi]
AB  - Adaptive radiation is thought to be responsible for the evolution of a great
      portion of the past and present diversity of life. Instances of adaptive
      radiation, characterized by the rapid emergence of an array of species as a
      consequence to their adaptation to distinct ecological niches, are important
      study systems in evolutionary biology. However, because of the rapid lineage
      formation in these groups, and occasional gene flow between the participating
      species, it is often difficult to reconstruct the phylogenetic history of species
      that underwent an adaptive radiation. In this study, we present a novel approach 
      for species-tree estimation in rapidly diversifying lineages, where introgression
      is known to occur, and apply it to a multimarker data set containing up to 16
      specimens per species for a set of 45 species of East African cichlid fishes (522
      individuals in total), with a main focus on the cichlid species flock of Lake
      Tanganyika. We first identified, using age distributions of most recent common
      ancestors in individual gene trees, those lineages in our data set that show
      strong signatures of past introgression. This led us to formulate three
      hypotheses of introgression between different lineages of Tanganyika cichlids:
      the ancestor of Boulengerochromini (or of Boulengerochromini and Bathybatini)
      received genomic material from the derived H-lineage; the common ancestor of
      Cyprichromini and Perissodini experienced, in turn, introgression from
      Boulengerochromini and/or Bathybatini; and the Lake Tanganyika Haplochromini and 
      closely related riverine lineages received genetic material from Cyphotilapiini. 
      We then applied the multispecies coalescent model to estimate the species tree of
      Lake Tanganyika cichlids, but excluded the lineages involved in these
      introgression events, as the multispecies coalescent model does not incorporate
      introgression. This resulted in a robust species tree, in which the Lamprologini 
      were placed as sister lineage to the H-lineage (including the Eretmodini), and we
      identify a series of rapid splitting events at the base of the H-lineage.
      Divergence ages estimated with the multispecies coalescent model were
      substantially younger than age estimates based on concatenation, and agree with
      the geological history of the Great Lakes of East Africa. Finally, we formally
      tested the three hypotheses of introgression using a likelihood framework, and
      find strong support for introgression between some of the cichlid tribes of Lake 
      Tanganyika.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Meyer, Britta S
AU  - Meyer BS
AD  - Zoological Institute, University of Basel, Basel, Switzerland.
AD  - Evolutionary Ecology of Marine Fishes, GEOMAR Helmholtz Centre for Ocean Research
      Kiel, Kiel, Germany.
FAU - Matschiner, Michael
AU  - Matschiner M
AD  - Zoological Institute, University of Basel, Basel, Switzerland.
AD  - Centre for Ecological and Evolutionary Synthesis (CEES), Department of
      Biosciences, University of Oslo, Oslo, Norway.
FAU - Salzburger, Walter
AU  - Salzburger W
AD  - Zoological Institute, University of Basel, Basel, Switzerland.
AD  - Centre for Ecological and Evolutionary Synthesis (CEES), Department of
      Biosciences, University of Oslo, Oslo, Norway.
LA  - eng
SI  - Dryad/10.5061/dryad.62qn5
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Cichlids/*classification
MH  - Lakes
MH  - Models, Statistical
MH  - *Phylogeny
MH  - Tanzania
OTO - NOTNLM
OT  - *Adaptive radiation
OT  - *Cichlidae
OT  - *Lake Tanganyika
OT  - *introgression
OT  - *species network
EDAT- 2016/08/20 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/08/20 06:00
PHST- 2016/02/10 00:00 [received]
PHST- 2016/07/27 00:00 [accepted]
PHST- 2016/08/20 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/08/20 06:00 [entrez]
AID - syw069 [pii]
AID - 10.1093/sysbio/syw069 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Jul 1;66(4):531-550. doi: 10.1093/sysbio/syw069.

PMID- 27539484
OWN - NLM
STAT- Publisher
LR  - 20160819
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
DP  - 2016 Aug 18
TI  - Retraction.
LID - syw070 [pii]
LA  - eng
PT  - Journal Article
DEP - 20160818
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EDAT- 2016/08/20 06:00
MHDA- 2016/08/20 06:00
CRDT- 2016/08/20 06:00
PHST- 2016/08/20 06:00 [entrez]
PHST- 2016/08/20 06:00 [pubmed]
PHST- 2016/08/20 06:00 [medline]
AID - syw070 [pii]
AID - 10.1093/sysbio/syw070 [doi]
PST - aheadofprint
SO  - Syst Biol. 2016 Aug 18. pii: syw070. doi: 10.1093/sysbio/syw070.

PMID- 27486181
OWN - NLM
STAT- MEDLINE
DCOM- 20180108
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 2
DP  - 2017 Mar 1
TI  - SpeciesGeoCoder: Fast Categorization of Species Occurrences for Analyses of
      Biodiversity, Biogeography, Ecology, and Evolution.
PG  - 145-151
LID - 10.1093/sysbio/syw064 [doi]
AB  - Understanding the patterns and processes underlying the uneven distribution of
      biodiversity across space constitutes a major scientific challenge in systematic 
      biology and biogeography, which largely relies on effectively mapping and making 
      sense of rapidly increasing species occurrence data. There is thus an urgent need
      for making the process of coding species into spatial units faster, automated,
      transparent, and reproducible. Here we present SpeciesGeoCoder, an open-source
      software package written in Python and R, that allows for easy coding of species 
      into user-defined operational units. These units may be of any size and be purely
      spatial (i.e., polygons) such as countries and states, conservation areas,
      biomes, islands, biodiversity hotspots, and areas of endemism, but may also
      include elevation ranges. This flexibility allows scoring species into complex
      categories, such as those encountered in topographically and ecologically
      heterogeneous landscapes. In addition, SpeciesGeoCoder can be used to facilitate 
      sorting and cleaning of occurrence data obtained from online databases, and for
      testing the impact of incorrect identification of specimens on the spatial coding
      of species. The various outputs of SpeciesGeoCoder include quantitative
      biodiversity statistics, global and local distribution maps, and files that can
      be used directly in many phylogeny-based applications for ancestral range
      reconstruction, investigations of biome evolution, and other comparative methods.
      Our simulations indicate that even datasets containing hundreds of millions of
      records can be analyzed in relatively short time using a standard computer. We
      exemplify the use of SpeciesGeoCoder by inferring the historical dispersal of
      birds across the Isthmus of Panama, showing that lowland species crossed the
      Isthmus about twice as frequently as montane species with a marked increase in
      the number of dispersals during the last 10 million years. [ancestral area
      reconstruction; biodiversity patterns; ecology; evolution; point in polygon;
      species distribution data.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Topel, Mats
AU  - Topel M
AD  - Department of Marine Sciences, University of Gothenburg, PO Box 460, SE-405 30
      Goteborg, Sweden.
AD  - Bioinformatics Infrastructure for Life Sciences.
AD  - Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, SE-405 30 Goteborg, Sweden.
FAU - Zizka, Alexander
AU  - Zizka A
AD  - Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, SE-405 30 Goteborg, Sweden.
FAU - Calio, Maria Fernanda
AU  - Calio MF
AD  - Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, SE-405 30 Goteborg, Sweden.
AD  - Universidade de Sao Paulo, Instituto de Biociencias, Departamento de Botanica,
      Rua do Matao, 277, Cidade Universitaria, CEP: 05508-090, Sao Paulo, SP, Brazil.
AD  - Universidade Estadual de Campinas, Instituto de Biologia, Departamento de
      Biologia Vegetal, Rua Monteiro Lobato, 255, Cidade Universitaria, CEP: 13083-862,
      Campinas, SP, Brazil.
FAU - Scharn, Ruud
AU  - Scharn R
AD  - Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, SE-405 30 Goteborg, Sweden.
FAU - Silvestro, Daniele
AU  - Silvestro D
AD  - Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, SE-405 30 Goteborg, Sweden.
FAU - Antonelli, Alexandre
AU  - Antonelli A
AD  - Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, SE-405 30 Goteborg, Sweden.
AD  - Gothenburg Botanical Garden, Carl Skottsbergs gata 22A, SE-41319 Goteborg,
      Sweden.
LA  - eng
SI  - Dryad/10.5061/dryad.tm32k
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biodiversity
MH  - Birds/classification
MH  - Classification/*methods
MH  - Ecology/*methods
MH  - *Phylogeny
MH  - Software
PMC - PMC5410971
EDAT- 2016/08/04 06:00
MHDA- 2018/01/09 06:00
CRDT- 2016/08/04 06:00
PHST- 2015/12/02 00:00 [received]
PHST- 2016/07/08 00:00 [accepted]
PHST- 2016/08/04 06:00 [pubmed]
PHST- 2018/01/09 06:00 [medline]
PHST- 2016/08/04 06:00 [entrez]
AID - syw064 [pii]
AID - 10.1093/sysbio/syw064 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Mar 1;66(2):145-151. doi: 10.1093/sysbio/syw064.

PMID- 27486180
OWN - NLM
STAT- MEDLINE
DCOM- 20180129
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 3
DP  - 2017 May 1
TI  - Maximum Likelihood Implementation of an Isolation-with-Migration Model for Three 
      Species.
PG  - 379-398
LID - 10.1093/sysbio/syw063 [doi]
AB  - We develop a maximum likelihood (ML) method for estimating migration rates
      between species using genomic sequence data. A species tree is used to
      accommodate the phylogenetic relationships among three species, allowing for
      migration between the two sister species, while the third species is used as an
      out-group. A Markov chain characterization of the genealogical process of
      coalescence and migration is used to integrate out the migration histories at
      each locus analytically, whereas Gaussian quadrature is used to integrate over
      the coalescent times on each genealogical tree numerically. This is an extension 
      of our early implementation of the symmetrical isolation-with-migration model for
      three species to accommodate arbitrary loci with two or three sequences per locus
      and to allow asymmetrical migration rates. Our implementation can accommodate
      tens of thousands of loci, making it feasible to analyze genome-scale data sets
      to test for gene flow. We calculate the posterior probabilities of gene trees at 
      individual loci to identify genomic regions that are likely to have been
      transferred between species due to gene flow. We conduct a simulation study to
      examine the statistical properties of the likelihood ratio test for gene flow
      between the two in-group species and of the ML estimates of model parameters such
      as the migration rate. Inclusion of data from a third out-group species is found 
      to increase dramatically the power of the test and the precision of parameter
      estimation. We compiled and analyzed several genomic data sets from the
      Drosophila fruit flies. Our analyses suggest no migration from D. melanogaster to
      D. simulans, and a significant amount of gene flow from D. simulans to D.
      melanogaster, at the rate of ~0.02 migrant individuals per generation. We discuss
      the utility of the multispecies coalescent model for species tree estimation,
      accounting for incomplete lineage sorting and migration.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Dalquen, Daniel A
AU  - Dalquen DA
AD  - Department of Genetics, Evolution and Environment, University College London,
      Darwin Building, Gower Street, London WC1E 6BT, UK.
FAU - Zhu, Tianqi
AU  - Zhu T
AD  - Center for Computational Genomics, Beijing Institute of Genomics, Chinese Academy
      of Sciences, Beijing 100101, China.
FAU - Yang, Ziheng
AU  - Yang Z
AD  - Department of Genetics, Evolution and Environment, University College London,
      Darwin Building, Gower Street, London WC1E 6BT, UK.
AD  - Center for Computational Genomics, Beijing Institute of Genomics, Chinese Academy
      of Sciences, Beijing 100101, China.
LA  - eng
SI  - Dryad/10.5061/dryad.h0h4s
GR  - Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animal Migration
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Drosophila/classification/genetics
MH  - Gene Flow
MH  - Likelihood Functions
MH  - *Models, Biological
MH  - *Phylogeny
OTO - NOTNLM
OT  - *IM model
OT  - *maximum likelihood
OT  - *migration
OT  - *multispecies coalescent
OT  - *speciation
EDAT- 2016/08/04 06:00
MHDA- 2018/01/30 06:00
CRDT- 2016/08/04 06:00
PHST- 2015/09/21 00:00 [received]
PHST- 2016/07/08 00:00 [accepted]
PHST- 2016/08/04 06:00 [pubmed]
PHST- 2018/01/30 06:00 [medline]
PHST- 2016/08/04 06:00 [entrez]
AID - syw063 [pii]
AID - 10.1093/sysbio/syw063 [doi]
PST - ppublish
SO  - Syst Biol. 2017 May 1;66(3):379-398. doi: 10.1093/sysbio/syw063.

PMID- 27444424
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20180226
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Erratum.
PG  - 945
LID - 10.1093/sysbio/syw046 [doi]
LA  - eng
PT  - Journal Article
PT  - Published Erratum
DEP - 20160721
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EFR - Syst Biol. 2016 May;65(3):508-24. PMID: 26880148
EDAT- 2016/07/23 06:00
MHDA- 2016/07/23 06:01
CRDT- 2016/07/23 06:00
PHST- 2016/07/23 06:00 [entrez]
PHST- 2016/07/23 06:00 [pubmed]
PHST- 2016/07/23 06:01 [medline]
AID - syw046 [pii]
AID - 10.1093/sysbio/syw046 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):945. doi: 10.1093/sysbio/syw046. Epub 2016 Jul 21.

PMID- 27444423
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20180226
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Erratum.
PG  - 944
LID - 10.1093/sysbio/syw045 [doi]
LA  - eng
PT  - Journal Article
PT  - Published Erratum
DEP - 20160721
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EFR - Syst Biol. 2016 May;65(3):495-507. PMID: 26658901
EDAT- 2016/07/23 06:00
MHDA- 2016/07/23 06:01
CRDT- 2016/07/23 06:00
PHST- 2016/07/23 06:00 [entrez]
PHST- 2016/07/23 06:00 [pubmed]
PHST- 2016/07/23 06:01 [medline]
AID - syw045 [pii]
AID - 10.1093/sysbio/syw045 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):944. doi: 10.1093/sysbio/syw045. Epub 2016 Jul 21.

PMID- 27425642
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - The Identification of the Closest Living Relative(s) of Tetrapods: Phylogenomic
      Lessons for Resolving Short Ancient Internodes.
PG  - 1057-1075
AB  - Identifying the closest living relative(s) of tetrapods is an important, yet
      still contested question in vertebrate phylogenetics. Three hypotheses are
      possible and ruling out alternatives has proven difficult even with large
      molecular data sets due to weak phylogenetic signal coupled nonphylogenetic noise
      resulting from relatively rapid speciation events that occurred a long time ago
      ([Formula: see text]400 Ma). Here, we revisit the identity of the closest living 
      relative of land vertebrates from a phylogenomic perspective and include new
      genomic data for all extant lungfish genera. RNA-seq proves to be a great
      alternative to genomic sequencing, which currently is technically not feasible in
      lungfishes due to their huge (50-130 Gb) and repetitive genomes. We examined the 
      most important sources of systematic error, namely long-branch attraction (LBA), 
      compositional heterogeneity and distribution of missing data and applied
      different correction techniques. A multispecies coalescent approach is used to
      account for deep coalescence that might come from the short and deep internodes
      separating early sarcopterygian splits. Concatenation methods favored lungfishes 
      as the closest living relatives of tetrapods with strong statistical support.
      Amino acid profile mixture models can unambiguously resolve this difficult
      internode thanks to their ability to avoid systematic error. We assessed the
      performance of different site-heterogeneous models and data partitioning and
      compared the ability of different strategies designed to overcome LBA, including 
      taxon manipulation, reduction of among-lineage rate heterogeneity and removal of 
      fast-evolving or compositionally heterogeneous positions. The identification of
      lungfish as sister group of tetrapods is robust regarding the effects of
      nonstationary composition and distribution of missing data. The multispecies
      coalescent method reconstructed strongly supported topologies that were congruent
      with concatenation, despite pervasive gene tree heterogeneity. We reject
      alternative topologies for early sarcopterygian relationships by increasing the
      signal-to-noise ratio in our alignments. The analytical pipeline outlined here
      combines probabilistic phylogenomic inference with methods for evaluating data
      quality, model adequacy, and assessing systematic error, and thus is likely to
      help resolve similarly difficult internodes in the tree of life. [Coalescence;
      coelacanth; compositional heterogeneity; gene tree; long-branch attraction;
      lungfish; missing data; model misspecification; phylogenomic; species tree;
      systematic error.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Irisarri, Iker
AU  - Irisarri I
AD  - Laboratory for Zoology and Evolutionary Biology, Department of Biology,
      University of Konstanz, 78464 Konstanz, Germany.
FAU - Meyer, Axel
AU  - Meyer A
AD  - Laboratory for Zoology and Evolutionary Biology, Department of Biology,
      University of Konstanz, 78464 Konstanz, Germany.
LA  - eng
SI  - Dryad/10.5061/dryad.gd74v
PT  - Journal Article
DEP - 20160717
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Classification/*methods
MH  - Fishes/classification
MH  - Genome/genetics
MH  - *Phylogeny
MH  - Vertebrates/*classification/*genetics
EDAT- 2016/07/19 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/07/19 06:00
PHST- 2015/11/23 00:00 [received]
PHST- 2016/06/08 00:00 [accepted]
PHST- 2016/07/19 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
PHST- 2016/07/19 06:00 [entrez]
AID - syw057 [pii]
AID - 10.1093/sysbio/syw057 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):1057-1075. doi: 10.1093/sysbio/syw057. Epub 2016 Jul
      17.

PMID- 27401436
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20180222
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - Erratum
PG  - 1123
LID - 10.1093/sysbio/syw061 [doi]
LA  - eng
PT  - Journal Article
PT  - Published Erratum
DEP - 20160708
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EFR - Syst Biol. 2013 Jan 1;62(1):1-21. This article, which was published in its
      accepted format on 8 July 2016, was published in error as a duplicate of
      10.1093/sysbio/sys061 "Integrating Fuzzy Logic and Statistics to Improve the
      Reliable Delimitation of Biogeographic Regions and Transition Z. PMID: 22744774
EDAT- 2016/07/13 06:00
MHDA- 2016/07/13 06:01
CRDT- 2016/07/13 06:00
PHST- 2016/07/13 06:00 [pubmed]
PHST- 2016/07/13 06:01 [medline]
PHST- 2016/07/13 06:00 [entrez]
AID - syw061 [pii]
AID - 10.1093/sysbio/syw061 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):1123. doi: 10.1093/sysbio/syw061. Epub 2016 Jul 8.

PMID- 27368344
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - Understanding Past Population Dynamics: Bayesian Coalescent-Based Modeling with
      Covariates.
PG  - 1041-1056
AB  - Effective population size characterizes the genetic variability in a population
      and is a parameter of paramount importance in population genetics and
      evolutionary biology. Kingman's coalescent process enables inference of past
      population dynamics directly from molecular sequence data, and researchers have
      developed a number of flexible coalescent-based models for Bayesian nonparametric
      estimation of the effective population size as a function of time. Major goals of
      demographic reconstruction include identifying driving factors of effective
      population size, and understanding the association between the effective
      population size and such factors. Building upon Bayesian nonparametric
      coalescent-based approaches, we introduce a flexible framework that incorporates 
      time-varying covariates that exploit Gaussian Markov random fields to achieve
      temporal smoothing of effective population size trajectories. To approximate the 
      posterior distribution, we adapt efficient Markov chain Monte Carlo algorithms
      designed for highly structured Gaussian models. Incorporating covariates into the
      demographic inference framework enables the modeling of associations between the 
      effective population size and covariates while accounting for uncertainty in
      population histories. Furthermore, it can lead to more precise estimates of
      population dynamics. We apply our model to four examples. We reconstruct the
      demographic history of raccoon rabies in North America and find a significant
      association with the spatiotemporal spread of the outbreak. Next, we examine the 
      effective population size trajectory of the DENV-4 virus in Puerto Rico along
      with viral isolate count data and find similar cyclic patterns. We compare the
      population history of the HIV-1 CRF02_AG clade in Cameroon with HIV incidence and
      prevalence data and find that the effective population size is more reflective of
      incidence rate. Finally, we explore the hypothesis that the population dynamics
      of musk ox during the Late Quaternary period were related to climate change.
      [Coalescent; effective population size; Gaussian Markov random fields;
      phylodynamics; phylogenetics; population genetics.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Gill, Mandev S
AU  - Gill MS
AD  - Department of Statistics, Columbia University, New York, NY 10027, USA.
FAU - Lemey, Philippe
AU  - Lemey P
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven,
      Minderbroederstaat 10, 3000 Leuven, Belgium.
FAU - Bennett, Shannon N
AU  - Bennett SN
AD  - Department of Microbiology, California Academy of Sciences, San Francisco, CA
      94118, USA.
FAU - Biek, Roman
AU  - Biek R
AD  - Boyd Orr Centre for Population and Ecosystem Health, Institute of Biodiversity,
      Animal Health and Comparative Medicine, College of Medical, Veterinary and Life
      Sciences, University of Glasgow, Glasgow G12 8QQ, UK.
FAU - Suchard, Marc A
AU  - Suchard MA
AD  - Department of Biomathematics, David Geffen School of Medicine at UCLA, University
      of California, Los Angeles, CA 90095, USA.
AD  - Department of Human Genetics, David Geffen School of Medicine at UCLA, Universtiy
      of California, Los Angeles, CA 90095, USA.
AD  - Department of Biostatistics, Jonathan and Karin Fielding School of Public Health,
      University of California, Los Angeles, CA 90095, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.mj0hn
GR  - R01 AI107034/AI/NIAID NIH HHS/United States
GR  - R01 HG006139/HG/NHGRI NIH HHS/United States
PT  - Journal Article
DEP - 20160701
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Cameroon/epidemiology
MH  - Climate Change
MH  - Genetics, Population
MH  - HIV Infections/epidemiology
MH  - HIV-1/physiology
MH  - Humans
MH  - *Models, Biological
MH  - North America/epidemiology
MH  - Phylogeny
MH  - Population Density
MH  - Population Dynamics
MH  - Puerto Rico/epidemiology
MH  - Rabies/*epidemiology
MH  - Rabies virus/physiology
MH  - Raccoons/virology
PMC - PMC5066065
EDAT- 2016/07/03 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/07/03 06:00
PHST- 2015/12/27 00:00 [received]
PHST- 2016/05/16 00:00 [revised]
PHST- 2016/05/23 00:00 [accepted]
PHST- 2016/07/03 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
PHST- 2016/07/03 06:00 [entrez]
AID - syw050 [pii]
AID - 10.1093/sysbio/syw050 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):1041-1056. doi: 10.1093/sysbio/syw050. Epub 2016 Jul 1.

PMID- 27316673
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Likelihood-Based Parameter Estimation for High-Dimensional Phylogenetic
      Comparative Models: Overcoming the Limitations of "Distance-Based" Methods.
PG  - 852-70
LID - 10.1093/sysbio/syw051 [doi]
AB  - Recently, a suite of distance-based multivariate phylogenetic comparative methods
      has been proposed for studying the evolution of high-dimensional traits, such as 
      morphometric coordinates, gene expression data, and function-valued traits. These
      methods allow for the statistical comparison of evolutionary rates, assessment of
      phylogenetic signal, and tests of correlated high-dimensional trait evolution.
      Simulations reveal that distance-based comparative methods exhibit low
      statistical power and high Type I error under various evolutionary scenarios.
      Distance-based methods are also limited to relatively simple model specification 
      (e.g., Brownian motion evolution) due to the lack of a likelihood function for
      parameter estimation. Here I propose an alternative method for studying
      high-dimensional trait evolution which overcomes some of the statistical
      limitations associated with distance-based methods. This framework, based on
      parametric bootstrapping and maximum pseudolikelihood parameter estimation, opens
      up the ability to estimate alternative evolutionary models, combine multiple
      evolutionary hypotheses, and potentially allow missing data and within-species
      variation. Simulations reveal that pairwise composite likelihood methods
      demonstrate appropriate Type I error and high statistical power, thus providing a
      robust framework for studying high-dimensional trait evolution. These methods are
      implemented in the R package phylocurve [Covariance; distance; evolutionary rate;
      function-valued trait; high-dimensional; morphometric; multivariate; pairwise
      composite likelihood; phylogenetic comparative method; phylogenetic generalized
      least squares; phylogenetic signal.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Goolsby, Eric W
AU  - Goolsby EW
AD  - Arnold Arboretum, Harvard University, Boston, MA 02131, USA and Interdisciplinary
      Toxicology Program, Department of Plant Biology, University of Georgia, Athens,
      GA 30602, USA. eric.goolsby.evolution@gmail.com.
LA  - eng
SI  - Dryad/10.5061/dryad.fh0mc
PT  - Journal Article
DEP - 20160616
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Likelihood Functions
MH  - *Models, Biological
MH  - Phenotype
MH  - *Phylogeny
EDAT- 2016/06/19 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/06/19 06:00
PHST- 2015/11/02 00:00 [received]
PHST- 2016/05/23 00:00 [accepted]
PHST- 2016/06/19 06:00 [entrez]
PHST- 2016/06/19 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
AID - syw051 [pii]
AID - 10.1093/sysbio/syw051 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):852-70. doi: 10.1093/sysbio/syw051. Epub 2016 Jun 16.

PMID- 27288482
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Spatiotemporal Diversification of the True Frogs (Genus Rana): A Historical
      Framework for a Widely Studied Group of Model Organisms.
PG  - 824-42
LID - 10.1093/sysbio/syw055 [doi]
AB  - True frogs of the genus Rana are widely used as model organisms in studies of
      development, genetics, physiology, ecology, behavior, and evolution. Comparative 
      studies among the more than 100 species of Rana rely on an understanding of the
      evolutionary history and patterns of diversification of the group. We estimate a 
      well-resolved, time-calibrated phylogeny from sequences of six nuclear and three 
      mitochondrial loci sampled from most species of Rana, and use that phylogeny to
      clarify the group's diversification and global biogeography. Our analyses
      consistently support an "Out of Asia" pattern with two independent dispersals of 
      Rana from East Asia to North America via Beringian land bridges. The more
      species-rich lineage of New World Rana appears to have experienced a rapid
      radiation following its colonization of the New World, especially with its
      expansion into montane and tropical areas of Mexico, Central America, and South
      America. In contrast, Old World Rana exhibit different trajectories of
      diversification; diversification in the Old World began very slowly and later
      underwent a distinct increase in speciation rate around 29-18 Ma. Net
      diversification is associated with environmental changes and especially intensive
      tectonic movements along the Asian margin from the Oligocene to early Miocene.
      Our phylogeny further suggests that previous classifications were misled by
      morphological homoplasy and plesiomorphic color patterns, as well as a reliance
      primarily on mitochondrial genes. We provide a phylogenetic taxonomy based on
      analyses of multiple nuclear and mitochondrial gene loci. [Amphibians;
      biogeography; diversification rate; Holarctic; transcontinental dispersal.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Yuan, Zhi-Yong
AU  - Yuan ZY
AD  - State Key Laboratory of Genetic Resources and Evolution, and Yunnan Laboratory of
      Molecular Biology of Domestic Animals, Kunming Institute of Zoology, Chinese
      Academy of Sciences, Kunming 650223, Yunnan Kunming College of Life Science,
      University of Chinese Academy of Sciences, Kunming 650204, Yunnan.
FAU - Zhou, Wei-Wei
AU  - Zhou WW
AD  - State Key Laboratory of Genetic Resources and Evolution, and Yunnan Laboratory of
      Molecular Biology of Domestic Animals, Kunming Institute of Zoology, Chinese
      Academy of Sciences, Kunming 650223, Yunnan.
FAU - Chen, Xin
AU  - Chen X
AD  - Department of Biological Sciences, Dartmouth College, Hanover, HN 03755, USA.
FAU - Poyarkov, Nikolay A Jr
AU  - Poyarkov NA Jr
AD  - Department of Vertebrate Zoology, Biological Faculty, Lomonosov Moscow State
      University, Leninskiye Gory, GSP-1, Moscow 119991, Russia.
FAU - Chen, Hong-Man
AU  - Chen HM
AD  - State Key Laboratory of Genetic Resources and Evolution, and Yunnan Laboratory of
      Molecular Biology of Domestic Animals, Kunming Institute of Zoology, Chinese
      Academy of Sciences, Kunming 650223, Yunnan.
FAU - Jang-Liaw, Nian-Hong
AU  - Jang-Liaw NH
AD  - Animal Department, Taipei Zoo, 30 Xinguang Road, Sec. 2, Taipei 11656, Taiwan.
FAU - Chou, Wen-Hao
AU  - Chou WH
AD  - Department of Zoology, National Museum of Natural Science, 1st Kuang-Chien Road, 
      Taichung 40453, Taiwan.
FAU - Matzke, Nicholas J
AU  - Matzke NJ
AD  - Division of Ecology, Evolution, and Genetics, Research School of Biology, The
      Australian National University, ACT 2601, Australia.
FAU - Iizuka, Koji
AU  - Iizuka K
AD  - Kanda-Hitotsubashi JH School, 2-16-14 Hitotsubashi, Chiyoda-ku, Tokyo 101-0003,
      Japan.
FAU - Min, Mi-Sook
AU  - Min MS
AD  - Conservation Genome Resource Bank for Korean Wildlife (CGRB), Research Institute 
      for Veterinary Science, College of Veterinary Medicine, Seoul National
      University, 151-742 Seoul, South Korea.
FAU - Kuzmin, Sergius L
AU  - Kuzmin SL
AD  - Institute of Ecology and Evolution, Russian Academy of Sciences, Moscow 119234,
      Russia.
FAU - Zhang, Ya-Ping
AU  - Zhang YP
AD  - State Key Laboratory of Genetic Resources and Evolution, and Yunnan Laboratory of
      Molecular Biology of Domestic Animals, Kunming Institute of Zoology, Chinese
      Academy of Sciences, Kunming 650223, Yunnan.
FAU - Cannatella, David C
AU  - Cannatella DC
AD  - Department of Integrative Biology, University of Texas at Austin, Austin, TX
      78712, USA.
FAU - Hillis, David M
AU  - Hillis DM
AD  - Department of Integrative Biology, University of Texas at Austin, Austin, TX
      78712, USA.
FAU - Che, Jing
AU  - Che J
AD  - State Key Laboratory of Genetic Resources and Evolution, and Yunnan Laboratory of
      Molecular Biology of Domestic Animals, Kunming Institute of Zoology, Chinese
      Academy of Sciences, Kunming 650223, Yunnan.
LA  - eng
SI  - Dryad/10.5061/dryad.ck1m7
PT  - Journal Article
DEP - 20160610
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Americas
MH  - Animals
MH  - Asia
MH  - Bayes Theorem
MH  - Far East
MH  - *Phylogeny
MH  - Ranidae/*classification/genetics
MH  - Sequence Analysis, DNA
EDAT- 2016/06/12 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/06/12 06:00
PHST- 2015/09/19 00:00 [received]
PHST- 2015/05/31 00:00 [accepted]
PHST- 2016/06/12 06:00 [entrez]
PHST- 2016/06/12 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
AID - syw055 [pii]
AID - 10.1093/sysbio/syw055 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):824-42. doi: 10.1093/sysbio/syw055. Epub 2016 Jun 10.

PMID- 27288480
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - Taxonomic Quality of Species Descriptions Varies over Time and with the Number of
      Authors, but Unevenly among Parasitic Taxa.
PG  - 1107-1116
AB  - Although concerns are being raised about a potential shortage of taxonomists and 
      systematists, recent analyses suggest instead that the number of researchers
      involved in taxonomic descriptions is higher than ever and that the average
      number of new species described annually per taxonomist has declined in recent
      decades. Here, using nine metrics of taxonomic quality, such as the number of
      morphological traits measured, the number of separate line drawings included, and
      whether or not gene sequences are provided, we explore variation in taxonomic
      quality as a function of the number of authors and other potential determinants
      across 2366 descriptions of parasitic helminths published in 1337 articles
      between 1980 and 2014. Taxonomic quality has generally increased over time, but
      unequally among different groups of helminths. For example, the number of
      scanning electron micrographs per description has risen significantly over time
      in cestodes and nematodes, but decreased for digeneans. For most metrics used,
      the greater the number of authors per species description, the higher its
      quality, suggesting not more taxonomists but more collaborations between
      taxonomists and experts from other fields to produce more comprehensive species
      characterizations. Re-descriptions of species were of higher quality than their
      original descriptions, and the improved quality correlated with the number of
      years elapsed between them. However, the extent of this improvement varied among 
      helminth species with different host taxa. Overall, our findings provide a note
      of caution for anyone using trends in the number of species described per author 
      to extrapolate the total number of extant species. They also reveal cultural
      differences among taxonomists working on different groups of parasites that can
      serve to identify areas for potential improvement. [Biodiversity; Cestoda;
      Digenea; integrative taxonomy; Nematoda.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Poulin, Robert
AU  - Poulin R
AD  - Department of Zoology, University of Otago, P.O. Box 56, Dunedin 9054, New
      Zealand.
FAU - Presswell, Bronwen
AU  - Presswell B
AD  - Department of Zoology, University of Otago, P.O. Box 56, Dunedin 9054, New
      Zealand.
LA  - eng
SI  - Dryad/10.5061/dryad.j4j7q
PT  - Journal Article
DEP - 20160610
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Classification
MH  - Helminths/classification
MH  - Periodicals as Topic/*standards/*statistics &amp; numerical data
EDAT- 2016/06/12 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/06/12 06:00
PHST- 2016/04/04 00:00 [received]
PHST- 2016/05/14 00:00 [revised]
PHST- 2016/06/01 00:00 [accepted]
PHST- 2016/06/12 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
PHST- 2016/06/12 06:00 [entrez]
AID - syw053 [pii]
AID - 10.1093/sysbio/syw053 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):1107-1116. doi: 10.1093/sysbio/syw053. Epub 2016 Jun
      10.

PMID- 27288479
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Phylogenetic Stability, Tree Shape, and Character Compatibility: A Case Study
      Using Early Tetrapods.
PG  - 737-58
LID - 10.1093/sysbio/syw049 [doi]
AB  - Phylogenetic tree shape varies as the evolutionary processes affecting a clade
      change over time. In this study, we examined an empirical phylogeny of fossil
      tetrapods during several time intervals, and studied how temporal constraints
      manifested in patterns of tree imbalance and character change. The results
      indicate that the impact of temporal constraints on tree shape is minimal and
      highlights the stability through time of the reference tetrapod phylogeny.
      Unexpected values of imbalance for Mississippian and Pennsylvanian time slices
      strongly support the hypothesis that the Carboniferous was a period of explosive 
      tetrapod radiation. Several significant diversification shifts take place in the 
      Mississippian and underpin increased terrestrialization among the earliest limbed
      vertebrates. Character incompatibility is relatively high at the beginning of
      tetrapod history, but quickly decreases to a relatively stable lower level,
      relative to a null distribution based on constant rates of character change. This
      implies that basal tetrapods had high, but declining, rates of homoplasy early in
      their evolutionary history, although the origin of Lissamphibia is an exception
      to this trend. The time slice approach is a powerful method of phylogenetic
      analysis and a useful tool for assessing the impact of combining extinct and
      extant taxa in phylogenetic analyses of large and speciose clades.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Bernardi, Massimo
AU  - Bernardi M
AD  - MUSE-Museo delle Scienze, Corso del Lavoro e della Scienza, 3, 38122 Trento,
      Italy; School of Earth Sciences, University of Bristol, Wills Memorial Building, 
      Queens Road, Bristol BS8 1RJ, UK; massimo.bernardi@muse.it.
FAU - Angielczyk, Kenneth D
AU  - Angielczyk KD
AD  - Integrative Research Center, Field Museum of Natural History, 1400 South Lake
      Shore Drive, Chicago, IL 60605-2496, USA;
FAU - Mitchell, Jonathan S
AU  - Mitchell JS
AD  - Department of Ecology and Evolutionary Biology, University of Michigan, Ann
      Arbor, MI 48103, USA; and.
FAU - Ruta, Marcello
AU  - Ruta M
AD  - School of Life Sciences, Joseph Banks Laboratories, University of Lincoln, Green 
      Lane, Lincoln LN6 7DL, UK.
LA  - eng
SI  - Dryad/10.5061/dryad.sh8b4
PT  - Journal Article
DEP - 20160610
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amphibians/classification
MH  - Animals
MH  - Biological Evolution
MH  - *Classification
MH  - *Fossils
MH  - Genetic Speciation
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Character compatibility
OT  - *Mesozoic
OT  - *Paleozoic
OT  - *diversification shifts
OT  - *tetrapod terrestrialization
OT  - *tree balance
OT  - *tree distance.
EDAT- 2016/06/12 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/06/12 06:00
PHST- 2014/10/25 00:00 [received]
PHST- 2016/05/19 00:00 [accepted]
PHST- 2016/06/12 06:00 [entrez]
PHST- 2016/06/12 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
AID - syw049 [pii]
AID - 10.1093/sysbio/syw049 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):737-58. doi: 10.1093/sysbio/syw049. Epub 2016 Jun 10.

PMID- 27288478
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - Species-Level Para- and Polyphyly in DNA Barcode Gene Trees: Strong Operational
      Bias in European Lepidoptera.
PG  - 1024-1040
AB  - The proliferation of DNA data is revolutionizing all fields of systematic
      research. DNA barcode sequences, now available for millions of specimens and
      several hundred thousand species, are increasingly used in algorithmic species
      delimitations. This is complicated by occasional incongruences between species
      and gene genealogies, as indicated by situations where conspecific individuals do
      not form a monophyletic cluster in a gene tree. In two previous reviews,
      non-monophyly has been reported as being common in mitochondrial DNA gene trees. 
      We developed a novel web service "Monophylizer" to detect non-monophyly in
      phylogenetic trees and used it to ascertain the incidence of species
      non-monophyly in COI (a.k.a. cox1) barcode sequence data from 4977 species and
      41,583 specimens of European Lepidoptera, the largest data set of DNA barcodes
      analyzed from this regard. Particular attention was paid to accurate species
      identification to ensure data integrity. We investigated the effects of
      tree-building method, sampling effort, and other methodological issues, all of
      which can influence estimates of non-monophyly. We found a 12% incidence of
      non-monophyly, a value significantly lower than that observed in previous
      studies. Neighbor joining (NJ) and maximum likelihood (ML) methods yielded almost
      equal numbers of non-monophyletic species, but 24.1% of these cases of
      non-monophyly were only found by one of these methods. Non-monophyletic species
      tend to show either low genetic distances to their nearest neighbors or
      exceptionally high levels of intraspecific variability. Cases of polyphyly in COI
      trees arising as a result of deep intraspecific divergence are negligible, as the
      detected cases reflected misidentifications or methodological errors. Taking into
      consideration variation in sampling effort, we estimate that the true incidence
      of non-monophyly is approximately 23%, but with operational factors still being
      included. Within the operational factors, we separately assessed the frequency of
      taxonomic limitations (presence of overlooked cryptic and oversplit species) and 
      identification uncertainties. We observed that operational factors are
      potentially present in more than half (58.6%) of the detected cases of
      non-monophyly. Furthermore, we observed that in about 20% of non-monophyletic
      species and entangled species, the lineages involved are either allopatric or
      parapatric-conditions where species delimitation is inherently subjective and
      particularly dependent on the species concept that has been adopted. These
      observations suggest that species-level non-monophyly in COI gene trees is less
      common than previously supposed, with many cases reflecting misidentifications,
      the subjectivity of species delimitation or other operational factors.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Mutanen, Marko
AU  - Mutanen M
AD  - Department of Genetics and Physiology, University of Oulu, Finland;
      marko.mutanen@oulu.fi.
FAU - Kivela, Sami M
AU  - Kivela SM
AD  - Department of Ecology, University of Oulu, Finland.
FAU - Vos, Rutger A
AU  - Vos RA
AD  - Naturalis Biodiversity Center, Leiden, The Netherlands.
FAU - Doorenweerd, Camiel
AU  - Doorenweerd C
AD  - Naturalis Biodiversity Center, Leiden, The Netherlands.
FAU - Ratnasingham, Sujeevan
AU  - Ratnasingham S
AD  - Centre for Biodiversity Genomics, Biodiversity Institute of Ontario, University
      of Guelph, Canada.
FAU - Hausmann, Axel
AU  - Hausmann A
AD  - SNSB - Bavarian State Collection of Zoology, Munich, Germany.
FAU - Huemer, Peter
AU  - Huemer P
AD  - Tiroler Landesmuseen-Betriebsgesellschaft m.b.H., Innsbruck, Austria.
FAU - Dinca, Vlad
AU  - Dinca V
AD  - Centre for Biodiversity Genomics, Biodiversity Institute of Ontario, University
      of Guelph, Canada.
AD  - Institut de Biologia Evolutiva (CSIC-Universitat Pompeu Fabra), Barcelona, Spain.
FAU - van Nieukerken, Erik J
AU  - van Nieukerken EJ
AD  - Institut de Biologia Evolutiva (CSIC-Universitat Pompeu Fabra), Barcelona, Spain.
FAU - Lopez-Vaamonde, Carlos
AU  - Lopez-Vaamonde C
AD  - INRA, UR633 Zoologie Forestiere, 45075 Orleans, France.
AD  - Institut de Recherche sur la Biologie de l'Insecte, CNRS UMR 7261, Universite
      Francois-Rabelais de Tours, UFR Sciences et Techniques, 37200 Tours, France.
FAU - Vila, Roger
AU  - Vila R
AD  - Institut de Biologia Evolutiva (CSIC-Universitat Pompeu Fabra), Barcelona, Spain.
FAU - Aarvik, Leif
AU  - Aarvik L
AD  - Natural History Museum University of Oslo, Norway.
FAU - Decaens, Thibaud
AU  - Decaens T
AD  - Centre d'Ecologie Fonctionnelle et Evolutive, UMR 5175 CNRS / University of
      Montpellier / University of Montpellier 3 / EPHE / SupAgro Montpellier / INRA /
      IRD, 1919 Route de Mende, 34293 Montpellier Cedex 5, France.
FAU - Efetov, Konstantin A
AU  - Efetov KA
AD  - Crimean Federal University, Simferopol, Crimea.
FAU - Hebert, Paul D N
AU  - Hebert PD
AD  - Centre for Biodiversity Genomics, Biodiversity Institute of Ontario, University
      of Guelph, Canada.
FAU - Johnsen, Arild
AU  - Johnsen A
AD  - Natural History Museum University of Oslo, Norway.
FAU - Karsholt, Ole
AU  - Karsholt O
AD  - Zoologisk Museum, Natural History Museum of Denmark, University of Copenhagen,
      Universitetsparken 15, 2100 Copenhagen O, Denmark.
FAU - Pentinsaari, Mikko
AU  - Pentinsaari M
AD  - Department of Genetics and Physiology, University of Oulu, Finland.
FAU - Rougerie, Rodolphe
AU  - Rougerie R
AD  - Departement Systematique et Evolution, Museum National d'Histoire Naturelle,
      Institut de Systematique, Evolution, Biodiversite, ISYEB-UMR 7205 MNHN, CNRS,
      UPMC, EPHE, Sorbonne Universites, Paris, France.
FAU - Segerer, Andreas
AU  - Segerer A
AD  - SNSB - Bavarian State Collection of Zoology, Munich, Germany.
FAU - Tarmann, Gerhard
AU  - Tarmann G
AD  - Tiroler Landesmuseen-Betriebsgesellschaft m.b.H., Innsbruck, Austria.
FAU - Zahiri, Reza
AU  - Zahiri R
AD  - Centre for Biodiversity Genomics, Biodiversity Institute of Ontario, University
      of Guelph, Canada.
AD  - Ottawa Plant Laboratory, Canadian Food Inspection Agency, Canada.
FAU - Godfray, H Charles J
AU  - Godfray HC
AD  - Department of Zoology, University of Oxford, UK.
LA  - eng
SI  - Dryad/10.5061/dryad.k3mr1
PT  - Journal Article
DEP - 20160610
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Bias
MH  - Classification/*methods
MH  - DNA Barcoding, Taxonomic
MH  - DNA, Mitochondrial
MH  - Genes, Mitochondrial
MH  - Lepidoptera/*classification/*genetics
MH  - *Phylogeny
PMC - PMC5066064
OTO - NOTNLM
OT  - *DNA barcoding
OT  - *Lepidoptera
OT  - *gene tree
OT  - *mitochondrial COI
OT  - *mitochondrial cox1
OT  - *paraphyly
OT  - *polyphyly
OT  - *species delimitation
OT  - *species monophyly
EDAT- 2016/06/12 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/06/12 06:00
PHST- 2015/12/14 00:00 [received]
PHST- 2016/04/18 00:00 [revised]
PHST- 2016/04/18 00:00 [accepted]
PHST- 2016/06/12 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
PHST- 2016/06/12 06:00 [entrez]
AID - syw044 [pii]
AID - 10.1093/sysbio/syw044 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):1024-1040. doi: 10.1093/sysbio/syw044. Epub 2016 Jun
      10.

PMID- 27288477
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Sequence Capture versus Restriction Site Associated DNA Sequencing for Shallow
      Systematics.
PG  - 910-24
LID - 10.1093/sysbio/syw036 [doi]
AB  - Sequence capture and restriction site associated DNA sequencing (RAD-Seq) are two
      genomic enrichment strategies for applying next-generation sequencing
      technologies to systematics studies. At shallow timescales, such as within
      species, RAD-Seq has been widely adopted among researchers, although there has
      been little discussion of the potential limitations and benefits of RAD-Seq and
      sequence capture. We discuss a series of issues that may impact the utility of
      sequence capture and RAD-Seq data for shallow systematics in non-model species.
      We review prior studies that used both methods, and investigate differences
      between the methods by re-analyzing existing RAD-Seq and sequence capture data
      sets from a Neotropical bird (Xenops minutus). We suggest that the strengths of
      RAD-Seq data sets for shallow systematics are the wide dispersion of markers
      across the genome, the relative ease and cost of laboratory work, the deep
      coverage and read overlap at recovered loci, and the high overall information
      that results. Sequence capture's benefits include flexibility and repeatability
      in the genomic regions targeted, success using low-quality samples, more
      straightforward read orthology assessment, and higher per-locus information
      content. The utility of a method in systematics, however, rests not only on its
      performance within a study, but on the comparability of data sets and inferences 
      with those of prior work. In RAD-Seq data sets, comparability is compromised by
      low overlap of orthologous markers across species and the sensitivity of genetic 
      diversity in a data set to an interaction between the level of natural
      heterozygosity in the samples examined and the parameters used for orthology
      assessment. In contrast, sequence capture of conserved genomic regions permits
      interrogation of the same loci across divergent species, which is preferable for 
      maintaining comparability among data sets and studies for the purpose of drawing 
      general conclusions about the impact of historical processes across biotas. We
      argue that sequence capture should be given greater attention as a method of
      obtaining data for studies in shallow systematics and comparative phylogeography.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Harvey, Michael G
AU  - Harvey MG
AD  - Museum of Natural Science, Louisiana State University, Baton Rouge, LA 70803,
      USA, Department of Biological Sciences, Louisiana State University, Baton Rouge, 
      LA 70803, USA, mharve9@lsu.edu.
FAU - Smith, Brian Tilston
AU  - Smith BT
AD  - Department of Biological Sciences, Louisiana State University, Baton Rouge, LA
      70803, USA, Department of Ornithology, American Museum of Natural History,
      Central Park West at 79th Street, New York, NY 10024, USA, and.
FAU - Glenn, Travis C
AU  - Glenn TC
AD  - Department of Environmental Health Science, University of Georgia, Athens, GA
      30602, USA.
FAU - Faircloth, Brant C
AU  - Faircloth BC
AD  - Museum of Natural Science, Louisiana State University, Baton Rouge, LA 70803,
      USA, Department of Biological Sciences, Louisiana State University, Baton Rouge, 
      LA 70803, USA.
FAU - Brumfield, Robb T
AU  - Brumfield RT
AD  - Museum of Natural Science, Louisiana State University, Baton Rouge, LA 70803,
      USA, Department of Biological Sciences, Louisiana State University, Baton Rouge, 
      LA 70803, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.604b8
PT  - Journal Article
PT  - Review
DEP - 20160610
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Classification/*methods
MH  - High-Throughput Nucleotide Sequencing
MH  - *Phylogeny
MH  - Phylogeography
MH  - Sequence Analysis, DNA
OTO - NOTNLM
OT  - *Allele frequency spectrum
OT  - *birds
OT  - *coalescent methods; concordance analysis
OT  - *massively parallel sequencing
OT  - *next-generation sequencing
OT  - *ultraconserved elements
EDAT- 2016/06/12 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/06/12 06:00
PHST- 2015/11/20 00:00 [received]
PHST- 2016/04/15 00:00 [accepted]
PHST- 2016/06/12 06:00 [entrez]
PHST- 2016/06/12 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
AID - syw036 [pii]
AID - 10.1093/sysbio/syw036 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):910-24. doi: 10.1093/sysbio/syw036. Epub 2016 Jun 10.

PMID- 27288476
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - Lost in the Other Half: Improving Accuracy in Geometric Morphometric Analyses of 
      One Side of Bilaterally Symmetric Structures.
PG  - 1096-1106
AB  - Systematists and evolutionary biologists have widely adopted Procrustes-based
      geometric morphometrics for measuring size and shape in biology. Many structures,
      and in fact most animals, are bilaterally symmetric with an internal plane of
      symmetry (also called object symmetry). Often, when quantifying asymmetric
      variation is not an aim, only one or the other side is measured and analyzed.
      This approach has been used in hundreds of studies. Its implicit assumption is
      that the information on the other side is redundant and a single side will,
      therefore, produce results mirroring those one would have obtained from the
      analysis of the entire structure with all its left and right landmarks. However, 
      the extent to which this assumption is met has, to my knowledge, never been
      explored. Using two example data sets, I will show that congruence may be high in
      analyses at a macroevolutionary level but much lower at a microevolutionary one, 
      and inaccuracies might especially affect shape. I will discuss some of the other 
      factors that may influence results and will suggest a simple expedient that can
      improve both the visualization and accuracy of shape analyses in one-side-only
      studies.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Cardini, Andrea
AU  - Cardini A
AD  - Dipartimento di Scienze Chimiche e Geologiche, Universita' di Modena e Reggio
      Emilia, Via Campi 103, 41125 Modena, Italy; alcardini@gmail.com cardini@unimo.it.
AD  - Centre for Forensic Science, The University of Western Australia, 35 Stirling
      Highway, Crawley, WA 6009, Australia.
LA  - eng
SI  - Dryad/10.5061/dryad.mr2mh
PT  - Journal Article
DEP - 20160610
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animal Structures/anatomy &amp; histology
MH  - Animals
MH  - *Biological Evolution
MH  - Body Size
MH  - Classification/*methods
MH  - Phylogeny
OTO - NOTNLM
OT  - *Allometry
OT  - *Procrustes
OT  - *landmarks
OT  - *macroevolution
OT  - *microevolution
OT  - *mirror reflection
OT  - *object symmetry
OT  - *shape visualization
EDAT- 2016/06/12 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/06/12 06:00
PHST- 2016/01/27 00:00 [received]
PHST- 2016/04/06 00:00 [revised]
PHST- 2016/05/17 00:00 [accepted]
PHST- 2016/06/12 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
PHST- 2016/06/12 06:00 [entrez]
AID - syw043 [pii]
AID - 10.1093/sysbio/syw043 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):1096-1106. doi: 10.1093/sysbio/syw043. Epub 2016 Jun
      10.

PMID- 27235697
OWN - NLM
STAT- MEDLINE
DCOM- 20180105
LR  - 20190111
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 4
DP  - 2016 Jul
TI  - RevBayes: Bayesian Phylogenetic Inference Using Graphical Models and an
      Interactive Model-Specification Language.
PG  - 726-36
LID - 10.1093/sysbio/syw021 [doi]
AB  - Programs for Bayesian inference of phylogeny currently implement a unique and fi 
      xed suite of models. Consequently, users of these software packages are
      simultaneously forced to use a number of programs for a given study, while also
      lacking the freedom to explore models that have not been implemented by the
      developers of those programs. We developed a new open-source software package,
      RevBayes, to address these problems. RevBayes is entirely based on probabilistic 
      graphical models, a powerful generic framework for specifying and analyzing
      statistical models. Phylogenetic-graphical models can be speci fi ed
      interactively in RevBayes, piece by piece, using a new succinct and intuitive
      language called Rev. Rev is similar to the R language and the BUGS model-speci fi
      cation language, and should be easy to learn for most users. The strength of
      RevBayes is the simplicity with which one can design, specify, and implement new 
      and complex models. Fortunately, this tremendous fl exibility does not come at
      the cost of slower computation; as we demonstrate, RevBayes outperforms competing
      software for several standard analyses. Compared with other programs, RevBayes
      has fewer black-box elements. Users need to explicitly specify each part of the
      model and analysis. Although this explicitness may initially be unfamiliar, we
      are convinced that this transparency will improve understanding of phylogenetic
      models in our fi eld. Moreover, it will motivate the search for improvements to
      existing methods by brazenly exposing the model choices that we make to critical 
      scrutiny. RevBayes is freely available at http://www.RevBayes.com [Bayesian
      inference; Graphical models; MCMC; statistical phylogenetics.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Hohna, Sebastian
AU  - Hohna S
AD  - Department of Integrative Biology; Department of Statistics, University of
      California, Berkeley, CA 94720, USA; Department of Evolution and Ecology,
      University of California, Davis, CA 95616, USA; Department of Mathematics,
      Stockholm University, Stockholm, SE-106 91 Stockholm, Sweden;
      Sebastian.Hoehna@gmail.com.
FAU - Landis, Michael J
AU  - Landis MJ
AD  - Department of Integrative Biology;
FAU - Heath, Tracy A
AU  - Heath TA
AD  - Department of Integrative Biology; Department of Ecology and Evolutionary
      Biology, University of Kansas, Lawrence, KS 66045, USA; Department of Ecology,
      Evolution and Organismal Biology, Iowa State University, Ames, IA 50011, USA;
      Sebastian.Hoehna@gmail.com.
FAU - Boussau, Bastien
AU  - Boussau B
AD  - Department of Integrative Biology; Laboratoire de Biometrie et Biologie
      Evolutive, Centre National de la Recherche Scientifique, Unite Mixte de Recherche
      5558, Universite Lyon 1, F-69622 Villeurbanne, France; and
      Sebastian.Hoehna@gmail.com.
FAU - Lartillot, Nicolas
AU  - Lartillot N
AD  - Laboratoire de Biometrie et Biologie Evolutive, Centre National de la Recherche
      Scientifique, Unite Mixte de Recherche 5558, Universite Lyon 1, F-69622
      Villeurbanne, France; and Sebastian.Hoehna@gmail.com.
FAU - Moore, Brian R
AU  - Moore BR
AD  - Department of Evolution and Ecology, University of California, Davis, CA 95616,
      USA; Sebastian.Hoehna@gmail.com.
FAU - Huelsenbeck, John P
AU  - Huelsenbeck JP
AD  - Department of Integrative Biology; Sebastian.Hoehna@gmail.com.
FAU - Ronquist, Fredrik
AU  - Ronquist F
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History,
      SE-10405 Stockholm, Sweden Sebastian.Hoehna@gmail.com.
LA  - eng
PT  - Journal Article
DEP - 20160528
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - *Models, Biological
MH  - *Phylogeny
MH  - *Software
PMC - PMC4911942
EDAT- 2016/05/29 06:00
MHDA- 2018/01/06 06:00
CRDT- 2016/05/29 06:00
PHST- 2015/04/01 00:00 [received]
PHST- 2015/03/01 00:00 [accepted]
PHST- 2016/05/29 06:00 [entrez]
PHST- 2016/05/29 06:00 [pubmed]
PHST- 2018/01/06 06:00 [medline]
AID - syw021 [pii]
AID - 10.1093/sysbio/syw021 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jul;65(4):726-36. doi: 10.1093/sysbio/syw021. Epub 2016 May 28.

PMID- 27226315
OWN - NLM
STAT- PubMed-not-MEDLINE
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Erratum.
PG  - 943
LID - 10.1093/sysbio/syw008 [doi]
LA  - eng
PT  - Journal Article
PT  - Published Erratum
DEP - 20160524
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EFR - Syst Biol. 2015 May;64(3):369-83. PMID: 25398445
PMC - PMC4997006
EDAT- 2016/05/27 06:00
MHDA- 2016/05/27 06:01
CRDT- 2016/05/27 06:00
PHST- 2016/05/27 06:00 [entrez]
PHST- 2016/05/27 06:00 [pubmed]
PHST- 2016/05/27 06:01 [medline]
AID - syw008 [pii]
AID - 10.1093/sysbio/syw008 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):943. doi: 10.1093/sysbio/syw008. Epub 2016 May 24.

PMID- 27208890
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Testing for Independence between Evolutionary Processes.
PG  - 812-23
LID - 10.1093/sysbio/syw004 [doi]
AB  - Evolutionary events co-occurring along phylogenetic trees usually point to
      complex adaptive phenomena, sometimes implicating epistasis. While a number of
      methods have been developed to account for co-occurrence of events on the same
      internal or external branch of an evolutionary tree, there is a need to account
      for the larger diversity of possible relative positions of events in a tree. Here
      we propose a method to quantify to what extent two or more evolutionary events
      are associated on a phylogenetic tree. The method is applicable to any discrete
      character, like substitutions within a coding sequence or gains/losses of a
      biological function. Our method uses a general approach to statistically test for
      significant associations between events along the tree, which encompasses both
      events inseparable on the same branch, and events genealogically ordered on
      different branches. It assumes that the phylogeny and themapping of branches is
      known without errors. We address this problem from the statistical viewpoint by a
      linear algebra representation of the localization of the evolutionary events on
      the tree.We compute the full probability distribution of the number of paired
      events occurring in the same branch or in different branches of the tree, under a
      null model of independence where each type of event occurs at a constant rate
      uniformly inthephylogenetic tree. The strengths andweaknesses of themethodare
      assessed via simulations;we then apply the method to explore the loss of cell
      motility in intracellular pathogens.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Behdenna, Abdelkader
AU  - Behdenna A
AD  - UMR7138 "Evolution Paris-Seine" UPMC-CNRS Bat A, 4eme etage, porte 414, Case 5 7 
      quai Saint Bernard, 75252 Paris Cedex 05; Atelier de BioInformatique - MNHN Boite
      Courier 50 Batiment 139 - RdC 45 rue Buffon, 75005 Paris; Smile "Stochastic
      Models for the Inference of Life Evolution" Center for Interdisciplinary Research
      in Biology College de France 11, place Marcelin Berthelot 75231 Paris Cedex 05;
      kaderbehdenna@gmail.com.
FAU - Pothier, Joel
AU  - Pothier J
AD  - Atelier de BioInformatique - MNHN Boite Courier 50 Batiment 139 - RdC 45 rue
      Buffon, 75005 Paris; Enzymologie De L'arn Bat B - 2eme etage 7, quai St Bernard
      Boite courrier 60 75252 Paris Cedex 05;
FAU - Abby, Sophie S
AU  - Abby SS
AD  - Institut Pasteur, Microbial Evolutionary Genomics 25-28 Rue du Docteur Roux
      75015, Paris; UMR 3525 - CNRS - Institut Pasteur Genetique des genomes 25-28 rue 
      du Docteur Roux, Site Fernbach 75724 Paris Cedex 15;
FAU - Lambert, Amaury
AU  - Lambert A
AD  - Smile "Stochastic Models for the Inference of Life Evolution" Center for
      Interdisciplinary Research in Biology College de France 11, place Marcelin
      Berthelot 75231 Paris Cedex 05; Laboratoire de Probabilites &amp; Modeles Aleatoires 
      UPMC Univ Paris 06 Case courrier 188 4, Place Jussieu 75252 Paris Cedex 05;
FAU - Achaz, Guillaume
AU  - Achaz G
AD  - UMR7138 "Evolution Paris-Seine" UPMC-CNRS Bat A, 4eme etage, porte 414, Case 5 7 
      quai Saint Bernard, 75252 Paris Cedex 05; Atelier de BioInformatique - MNHN Boite
      Courier 50 Batiment 139 - RdC 45 rue Buffon, 75005 Paris; Smile "Stochastic
      Models for the Inference of Life Evolution" Center for Interdisciplinary Research
      in Biology College de France 11, place Marcelin Berthelot 75231 Paris Cedex 05;
LA  - eng
SI  - Dryad/10.5061/dryad.847vk
PT  - Journal Article
DEP - 20160521
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Data Interpretation, Statistical
MH  - *Phylogeny
MH  - Probability
OTO - NOTNLM
OT  - *Coevolution
OT  - *phylogeny
OT  - *statistical test.
EDAT- 2016/05/22 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/05/22 06:00
PHST- 2015/01/23 00:00 [received]
PHST- 2016/01/21 00:00 [accepted]
PHST- 2016/05/22 06:00 [entrez]
PHST- 2016/05/22 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
AID - syw004 [pii]
AID - 10.1093/sysbio/syw004 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):812-23. doi: 10.1093/sysbio/syw004. Epub 2016 May 21.

PMID- 27162151
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Exploring Phylogenetic Relationships within Myriapoda and the Effects of Matrix
      Composition and Occupancy on Phylogenomic Reconstruction.
PG  - 871-89
LID - 10.1093/sysbio/syw041 [doi]
AB  - Myriapods, including the diverse and familiar centipedes and millipedes, are one 
      of the dominant terrestrial arthropod groups. Although molecular evidence has
      shown that Myriapoda is monophyletic, its internal phylogeny remains contentious 
      and understudied, especially when compared to those of Chelicerata and Hexapoda. 
      Until now, efforts have focused on taxon sampling (e.g., by including a handful
      of genes from many species) or on maximizing matrix size (e.g., by including
      hundreds or thousands of genes in just a few species), but a phylogeny maximizing
      sampling at both levels remains elusive. In this study, we analyzed 40 Illumina
      transcriptomes representing 3 of the 4 myriapod classes (Diplopoda, Chilopoda,
      and Symphyla); 25 transcriptomes were newly sequenced to maximize representation 
      at the ordinal level in Diplopoda and at the family level in Chilopoda. Ten
      supermatrices were constructed to explore the effect of several potential
      phylogenetic biases (e.g., rate of evolution, heterotachy) at 3 levels of gene
      occupancy per taxon (50%, 75%, and 90%). Analyses based on maximum likelihood and
      Bayesian mixture models retrieved monophyly of each myriapod class, and resulted 
      in 2 alternative phylogenetic positions for Symphyla, as sister group to
      Diplopoda + Chilopoda, or closer to Diplopoda, the latter hypothesis having been 
      traditionally supported by morphology. Within centipedes, all orders were well
      supported, but 2 deep nodes remained in conflict in the different analyses
      despite dense taxon sampling at the family level. Relationships among centipede
      orders in all analyses conducted with the most complete matrix (90% occupancy)
      are at odds not only with the sparser but more gene-rich supermatrices (75% and
      50% supermatrices) and with the matrices optimizing phylogenetic informativeness 
      or most conserved genes, but also with previous hypotheses based on morphology,
      development, or other molecular data sets. Our results indicate that a high
      percentage of ribosomal proteins in the most complete matrices, in conjunction
      with distance from the root, can act in concert to compromise the estimated
      relationships within the ingroup. We discuss the implications of these findings
      in the context of the ever more prevalent quest for completeness in phylogenomic 
      studies.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Fernandez, Rosa
AU  - Fernandez R
AD  - Museum of Comparative Zoology &amp; Department of Organismic and Evolutionary
      Biology, Harvard University, 26 Oxford Street, Cambridge, MA 02138, USA
      rfernandezgarcia@g.harvard.edu.
FAU - Edgecombe, Gregory D
AU  - Edgecombe GD
AD  - Department of Earth Sciences, The Natural History Museum, Cromwell Road, London
      SW7 5BD, UK.
FAU - Giribet, Gonzalo
AU  - Giribet G
AD  - Museum of Comparative Zoology &amp; Department of Organismic and Evolutionary
      Biology, Harvard University, 26 Oxford Street, Cambridge, MA 02138, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.8mp17
PT  - Journal Article
DEP - 20160509
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Arthropods/*classification/genetics
MH  - Base Sequence
MH  - Bayes Theorem
MH  - *Phylogeny
PMC - PMC4997009
OTO - NOTNLM
OT  - *Chilopoda
OT  - *Diplopoda
OT  - *Symphyla
OT  - *gene tree
OT  - *missing data
OT  - *node calibration
OT  - *species tree
EDAT- 2016/05/11 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/05/11 06:00
PHST- 2015/12/06 00:00 [received]
PHST- 2016/04/28 00:00 [accepted]
PHST- 2016/05/11 06:00 [entrez]
PHST- 2016/05/11 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
AID - syw041 [pii]
AID - 10.1093/sysbio/syw041 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):871-89. doi: 10.1093/sysbio/syw041. Epub 2016 May 9.

PMID- 27155010
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20190112
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Measuring Stratigraphic Congruence Across Trees, Higher Taxa, and Time.
PG  - 792-811
LID - 10.1093/sysbio/syw039 [doi]
AB  - The congruence between the order of cladistic branching and the first appearance 
      dates of fossil lineages can be quantified using a variety of indices. Good
      matching is a prerequisite for the accurate time calibration of trees, while the 
      distribution of congruence indices across large samples of cladograms has
      underpinned claims about temporal and taxonomic patterns of completeness in the
      fossil record. The most widely used stratigraphic congruence indices are the
      stratigraphic consistency index (SCI), the modified Manhattan stratigraphic
      measure (MSM*), and the gap excess ratio (GER) (plus its derivatives; the
      topological GER and the modified GER). Many factors are believed to variously
      bias these indices, with several empirical and simulation studies addressing some
      subset of the putative interactions. This study combines both approaches to
      quantify the effects (on all five indices) of eight variables reasoned to
      constrain the distribution of possible values (the number of taxa, tree balance, 
      tree resolution, range of first occurrence (FO) dates, center of gravity of FO
      dates, the variability of FO dates, percentage of extant taxa, and percentage of 
      taxa with no fossil record). Our empirical data set comprised 647 published
      animal and plant cladograms spanning the entire Phanerozoic, and for these data
      we also modeled the effects of mean age of FOs (as a proxy for clade age), the
      taxonomic rank of the clade, and the higher taxonomic group to which it belonged.
      The center of gravity of FO dates had not been investigated hitherto, and this
      was found to correlate most strongly with some measures of stratigraphic
      congruence in our empirical study (top-heavy clades had better congruence). The
      modified GER was the index least susceptible to bias. We found significant
      differences across higher taxa for all indices; arthropods had lower congruence
      and tetrapods higher congruence. Stratigraphic congruence-however measured-also
      varied throughout the Phanerozoic, reflecting the taxonomic composition of our
      sample. Notably, periods containing a high proportion of arthropods had poorer
      congruence overall than those with higher proportions of tetrapods. [Fossil
      calibration; gap excess ratio; manhattan stratigraphic metric; molecular clocks; 
      stratigraphic congruence.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - O'Connor, Anne
AU  - O'Connor A
AD  - Milner Centre for Evolution, The University of Bath, The Avenue, Claverton Down, 
      Bath BA2 7AY, UK.
FAU - Wills, Matthew A
AU  - Wills MA
AD  - Milner Centre for Evolution, The University of Bath, The Avenue, Claverton Down, 
      Bath BA2 7AY, UK m.a.wills@bath.ac.uk.
LA  - eng
SI  - Dryad/10.5061/dryad.c19kb
GR  - BB/K006754/1 /Biotechnology and Biological Sciences Research Council/United
      Kingdom
GR  - BB/K015702/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
DEP - 20160506
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Arthropods/classification
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Fossils
MH  - *Phylogeny
MH  - Plants/classification
MH  - Time
PMC - PMC4997008
EDAT- 2016/05/08 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/05/08 06:00
PHST- 2015/08/03 00:00 [received]
PHST- 2016/05/03 00:00 [accepted]
PHST- 2016/05/08 06:00 [entrez]
PHST- 2016/05/08 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
AID - syw039 [pii]
AID - 10.1093/sysbio/syw039 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):792-811. doi: 10.1093/sysbio/syw039. Epub 2016 May 6.

PMID- 27155009
OWN - NLM
STAT- MEDLINE
DCOM- 20180108
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 2
DP  - 2017 Mar 1
TI  - Biogeographic Dating of Speciation Times Using Paleogeographically Informed
      Processes.
PG  - 128-144
LID - 10.1093/sysbio/syw040 [doi]
AB  - Standard models of molecular evolution cannot estimate absolute speciation times 
      alone, and require external calibrations to do so, such as fossils. Because
      fossil calibration methods rely on the incomplete fossil record, a great number
      of nodes in the tree of life cannot be dated precisely. However, many major
      paleogeographical events are dated, and since biogeographic processes depend on
      paleogeographical conditions, biogeographic dating may be used as an alternative 
      or complementary method to fossil dating. I demonstrate how a time-stratified
      biogeographic stochastic process may be used to estimate absolute divergence
      times by conditioning on dated paleogeographical events. Informed by the current 
      paleogeographical literature, I construct an empirical dispersal graph using 25
      areas and 26 epochs for the past 540 Ma of Earth's history. Simulations indicate 
      biogeographic dating performs well so long as paleogeography imposes constraint
      on biogeographic character evolution. To gauge whether biogeographic dating may
      be of practical use, I analyzed the well-studied turtle clade (Testudines) to
      assess how well biogeographic dating fares when compared to fossil-calibrated
      dating estimates reported in the literature. Fossil-free biogeographic dating
      estimated the age of the most recent common ancestor of extant turtles to be from
      the Late Triassic, which is consistent with fossil-based estimates. Dating
      precision improves further when including a root node fossil calibration. The
      described model, paleogeographical dispersal graph, and analysis scripts are
      available for use with RevBayes.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Landis, Michael J
AU  - Landis MJ
AD  - Department of Integrative Biology, University of California, Berkeley, CA 94720, 
      USA.
AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT
      06520, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.dq666
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Calibration
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - Fossils
MH  - *Phylogeny
MH  - Time
MH  - Turtles/classification
PMC - PMC5837510
OTO - NOTNLM
OT  - *Bayesian
OT  - *Testudines
OT  - *biogeography
OT  - *paleogeography
OT  - *phylogenetics
OT  - *tree-dating
EDAT- 2016/05/08 06:00
MHDA- 2018/01/09 06:00
CRDT- 2016/05/08 06:00
PHST- 2015/10/08 00:00 [received]
PHST- 2016/04/28 00:00 [accepted]
PHST- 2016/05/08 06:00 [pubmed]
PHST- 2018/01/09 06:00 [medline]
PHST- 2016/05/08 06:00 [entrez]
AID - syw040 [pii]
AID - 10.1093/sysbio/syw040 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Mar 1;66(2):128-144. doi: 10.1093/sysbio/syw040.

PMID- 27155008
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20190112
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - Estimating Bayesian Phylogenetic Information Content.
PG  - 1009-1023
AB  - Measuring the phylogenetic information content of data has a long history in
      systematics. Here we explore a Bayesian approach to information content
      estimation. The entropy of the posterior distribution compared with the entropy
      of the prior distribution provides a natural way to measure information content. 
      If the data have no information relevant to ranking tree topologies beyond the
      information supplied by the prior, the posterior and prior will be identical.
      Information in data discourages consideration of some hypotheses allowed by the
      prior, resulting in a posterior distribution that is more concentrated (has lower
      entropy) than the prior. We focus on measuring information about tree topology
      using marginal posterior distributions of tree topologies. We show that both the 
      accuracy and the computational efficiency of topological information content
      estimation improve with use of the conditional clade distribution, which also
      allows topological information content to be partitioned by clade. We explore two
      important applications of our method: providing a compelling definition of
      saturation and detecting conflict among data partitions that can negatively
      affect analyses of concatenated data. [Bayesian; concatenation; conditional clade
      distribution; entropy; information; phylogenetics; saturation.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Lewis, Paul O
AU  - Lewis PO
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, CT 06269, USA; paul.lewis@uconn.edu.
FAU - Chen, Ming-Hui
AU  - Chen MH
AD  - Department of Statistics, University of Connecticut, 215 Glenbrook Road, Unit
      4120, Storrs, CT 06269, USA.
FAU - Kuo, Lynn
AU  - Kuo L
AD  - Department of Statistics, University of Connecticut, 215 Glenbrook Road, Unit
      4120, Storrs, CT 06269, USA.
FAU - Lewis, Louise A
AU  - Lewis LA
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, CT 06269, USA.
FAU - Fucikova, Karolina
AU  - Fucikova K
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, CT 06269, USA.
FAU - Neupane, Suman
AU  - Neupane S
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, CT 06269, USA.
FAU - Wang, Yu-Bo
AU  - Wang YB
AD  - Department of Statistics, University of Connecticut, 215 Glenbrook Road, Unit
      4120, Storrs, CT 06269, USA.
FAU - Shi, Daoyuan
AU  - Shi D
AD  - Department of Statistics, University of Connecticut, 215 Glenbrook Road, Unit
      4120, Storrs, CT 06269, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.1dn50
GR  - P01 CA142538/CA/NCI NIH HHS/United States
GR  - P01 CA142538/CA/NCI NIH HHS/United States
PT  - Journal Article
DEP - 20160506
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - *Models, Genetic
MH  - *Phylogeny
PMC - PMC5066063
EDAT- 2016/05/08 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/05/08 06:00
PHST- 2016/01/04 00:00 [received]
PHST- 2016/04/15 00:00 [revised]
PHST- 2016/05/01 00:00 [accepted]
PHST- 2016/05/08 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
PHST- 2016/05/08 06:00 [entrez]
AID - syw042 [pii]
AID - 10.1093/sysbio/syw042 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):1009-1023. doi: 10.1093/sysbio/syw042. Epub 2016 May 6.

PMID- 27151419
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Inconsistency of Species Tree Methods under Gene Flow.
PG  - 843-51
LID - 10.1093/sysbio/syw030 [doi]
AB  - Coalescent-based methods are now broadly used to infer evolutionary relationships
      between groups of organisms under the assumption that incomplete lineage sorting 
      (ILS) is the only source of gene tree discordance. Many of these methods are
      known to consistently estimate the species tree when all their assumptions are
      met. Nonetheless, little work has been done to test the robustness of such
      methods to violations of their assumptions. Here, we study the performance of two
      of the most efficient coalescent-based methods, ASTRAL and NJst, in the presence 
      of gene flow. Gene flow violates the assumption that ILS is the sole source of
      gene tree conflict. We find anomalous gene trees on three-taxon rooted trees and 
      on four-taxon unrooted trees. These anomalous trees do not exist under ILS only, 
      but appear because of gene flow. Our simulations show that species tree methods
      (and concatenation) may reconstruct the wrong evolutionary history, even from a
      very large number of well-reconstructed gene trees. In other words, species tree 
      methods can be inconsistent under gene flow. Our results underline the need for
      methods like PhyloNet, to account simultaneously for ILS and gene flow in a
      unified framework. Although much slower, PhyloNet had better accuracy and
      remained consistent at high levels of gene flow.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Solis-Lemus, Claudia
AU  - Solis-Lemus C
AD  - Department of Statistics, University of Wisconsin, Madison, WI, 53706, USA;
      claudia@stat.wisc.edu.
FAU - Yang, Mengyao
AU  - Yang M
AD  - Department of Statistics, University of Wisconsin, Madison, WI, 53706, USA;
FAU - Ane, Cecile
AU  - Ane C
AD  - Department of Statistics, University of Wisconsin, Madison, WI, 53706, USA;
      Department of Botany, University of Wisconsin, Madison, WI, 53706, USA.
LA  - eng
PT  - Journal Article
DEP - 20160505
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Gene Flow
MH  - Genetic Speciation
MH  - Models, Genetic
MH  - *Phylogeny
OTO - NOTNLM
OT  - *ASTRAL
OT  - *Anomalous gene trees
OT  - *NJst
OT  - *PhyloNet
OT  - *coalescent
OT  - *concatenation
OT  - *hybridization
OT  - *network
EDAT- 2016/05/07 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/05/07 06:00
PHST- 2015/10/20 00:00 [received]
PHST- 2016/04/01 00:00 [accepted]
PHST- 2016/05/07 06:00 [entrez]
PHST- 2016/05/07 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
AID - syw030 [pii]
AID - 10.1093/sysbio/syw030 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):843-51. doi: 10.1093/sysbio/syw030. Epub 2016 May 5.

PMID- 27146045
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - A Return to Linnaeus's Focus on Diagnosis, Not Description: The Use of DNA
      Characters in the Formal Naming of Species.
PG  - 1085-1095
AB  - Descriptions and diagnoses are alternative choices in all Codes of Nomenclature
      because Linnaeus relied on diagnoses, not descriptions, to name ca. 13,400
      animals, plants, and fungi. A diagnosis names characters in which a new taxon
      differs from the most similar known taxon; a description mixes taxonomically
      informative and uninformative features, usually without indicating which is
      which. The first formal diagnoses of new taxa that included DNA-based characters 
      came out in 2001, and by November 2015, at least 98 names of species of acoels,
      lichens, angiosperms, annelids, alveolates, arachnids, centipedes, turtles,
      fishes, butterflies, mollusks, nematodes, and pathogenic fungi have been
      published based on diagnostic mitochondrial, plastid, or nuclear DNA
      substitutions, indels, or rarely genetic distances, with or without additional
      morphological features. Authors have found diverse ways to specify the diagnostic
      traits (all published studies are here tabulated). While descriptions try to
      "cover" within-species variation, a goal rarely accomplished because of (i) the
      stochastic nature of specimen availability (thousands of species are known from
      single collections) and (ii) the subjective circumscription of species, the
      purpose of diagnoses was and is speedy identification. Linnaeus tried to achieve 
      this by citing images, geographic occurrence, and previous literature. The
      renewed attention to sharp diagnosis now coincides with worldwide barcoding
      efforts, may speed up formal naming, and matches the increasing reliance on DNA
      for both classification and identification. I argue for DNA-based diagnoses of
      new species becoming a recommendation in all Codes, not just the bacterial code. 
      [Codes of Nomenclature; description; diagnosis; DNA-based diagnosis; naming new
      species; nomenclature.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Renner, Susanne S
AU  - Renner SS
AD  - Systematic Botany and Mycology, University of Munich (LMU), Menzinger Street 67, 
      80638 Munich, Germany renner@lrz.uni-muenchen.de.
LA  - eng
PT  - Journal Article
DEP - 20160504
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *DNA
MH  - Phylogeny
MH  - Species Specificity
MH  - *Terminology as Topic
EDAT- 2016/05/06 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/05/06 06:00
PHST- 2015/09/12 00:00 [received]
PHST- 2016/04/05 00:00 [accepted]
PHST- 2016/05/06 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
PHST- 2016/05/06 06:00 [entrez]
AID - syw032 [pii]
AID - 10.1093/sysbio/syw032 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):1085-1095. doi: 10.1093/sysbio/syw032. Epub 2016 May 4.

PMID- 27129844
OWN - NLM
STAT- MEDLINE
DCOM- 20180312
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Phylogenomics for Systematic Biology.
PG  - 353-6
LID - 10.1093/sysbio/syw027 [doi]
FAU - Posada, David
AU  - Posada D
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, 36310
      Vigo, Spain.
LA  - eng
PT  - Congress
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *DNA Barcoding, Taxonomic
MH  - Data Mining/*methods
MH  - Databases, Genetic
MH  - *Datasets as Topic
MH  - *Genome
MH  - Genomics/*methods
MH  - *High-Throughput Nucleotide Sequencing
MH  - Humans
MH  - Systems Biology/*methods
EDAT- 2016/05/01 06:00
MHDA- 2018/03/13 06:00
CRDT- 2016/05/01 06:00
PHST- 2016/05/01 06:00 [entrez]
PHST- 2016/05/01 06:00 [pubmed]
PHST- 2018/03/13 06:00 [medline]
AID - syw027 [pii]
AID - 10.1093/sysbio/syw027 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):353-6. doi: 10.1093/sysbio/syw027.

PMID- 27121966
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - Terrace Aware Data Structure for Phylogenomic Inference from Supermatrices.
PG  - 997-1008
AB  - In phylogenomics the analysis of concatenated gene alignments, the so-called
      supermatrix, is commonly accompanied by the assumption of partition models. Under
      such models each gene, or more generally partition, is allowed to evolve under
      its own evolutionary model. Although partition models provide a more
      comprehensive analysis of supermatrices, missing data may hamper the tree search 
      algorithms due to the existence of phylogenetic (partial) terraces. Here, we
      introduce the phylogenetic terrace aware (PTA) data structure for the efficient
      analysis under partition models. In the presence of missing data PTA exploits
      (partial) terraces and induced partition trees to save computation time. We show 
      that an implementation of PTA in IQ-TREE leads to a substantial speedup of up to 
      4.5 and 8 times compared with the standard IQ-TREE and RAxML implementations,
      respectively. PTA is generally applicable to all types of partition models and
      common topological rearrangements thus can be employed by all phylogenomic
      inference software.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Chernomor, Olga
AU  - Chernomor O
AD  - Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories,
      University of Vienna, Medical University of Vienna, A-1030 Vienna, Austria and.
FAU - von Haeseler, Arndt
AU  - von Haeseler A
AD  - Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories,
      University of Vienna, Medical University of Vienna, A-1030 Vienna, Austria and.
AD  - Bioinformatics and Computational Biology, Faculty of Computer Science, University
      of Vienna, A-1090 Vienna, Austria.
FAU - Minh, Bui Quang
AU  - Minh BQ
AD  - Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories,
      University of Vienna, Medical University of Vienna, A-1030 Vienna, Austria and
      minh.bui@univie.ac.at.
LA  - eng
SI  - Dryad/10.5061/dryad.v02t3
PT  - Journal Article
DEP - 20160426
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Software
PMC - PMC5066062
OTO - NOTNLM
OT  - *Maximum likelihood
OT  - *partial terraces
OT  - *partition models
OT  - *phylogenetic terraces
OT  - *phylogenomic inference
EDAT- 2016/04/29 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/04/29 06:00
PHST- 2015/12/09 00:00 [received]
PHST- 2016/04/18 00:00 [revised]
PHST- 2016/04/19 00:00 [accepted]
PHST- 2016/04/29 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
PHST- 2016/04/29 06:00 [entrez]
AID - syw037 [pii]
AID - 10.1093/sysbio/syw037 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):997-1008. doi: 10.1093/sysbio/syw037. Epub 2016 Apr 26.

PMID- 27113475
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Phylogeny and Divergence Times of Lemurs Inferred with Recent and Ancient Fossils
      in the Tree.
PG  - 772-91
LID - 10.1093/sysbio/syw035 [doi]
AB  - Paleontological and neontological systematics seek to answer evolutionary
      questions with different data sets. Phylogenies inferred for combined extant and 
      extinct taxa provide novel insights into the evolutionary history of life.
      Primates have an extensive, diverse fossil record and molecular data for living
      and extinct taxa are rapidly becoming available. We used two models to infer the 
      phylogeny and divergence times for living and fossil primates, the tip-dating
      (TD) and fossilized birth-death process (FBD). We collected new morphological
      data, especially on the living and extinct endemic lemurs of Madagascar. We
      combined the morphological data with published DNA sequences to infer
      near-complete (88% of lemurs) time-calibrated phylogenies. The results suggest
      that primates originated around the Cretaceous-Tertiary boundary, slightly
      earlier than indicated by the fossil record and later than previously inferred
      from molecular data alone. We infer novel relationships among extinct lemurs, and
      strong support for relationships that were previously unresolved. Dates inferred 
      with TD were significantly older than those inferred with FBD, most likely
      related to an assumption of a uniform branching process in the TD compared with a
      birth-death process assumed in the FBD. This is the first study to combine
      morphological and DNA sequence data from extinct and extant primates to infer
      evolutionary relationships and divergence times, and our results shed new light
      on the tempo of lemur evolution and the efficacy of combined phylogenetic
      analyses.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Herrera, James P
AU  - Herrera JP
AD  - Richard Gilder Graduate School; Division of Vertebrate Zoology, American Museum
      of Natural History, Central Park West &amp; 79th Street, New York, NY 10024, USA;
      Interdepartmental Doctoral Program in Anthropological Sciences, Department of
      Anthropology, Stony Brook University, Stony Brook, NY 11794, USA;
      jherrera@amnh.org.
FAU - Davalos, Liliana M
AU  - Davalos LM
AD  - Interdepartmental Doctoral Program in Anthropological Sciences, Department of
      Anthropology, Stony Brook University, Stony Brook, NY 11794, USA; Department of
      Ecology and Evolution, Stony Brook University, Stony Brook, NY 11794, USA; and
      Consortium for Interdisciplinary Environmental Research, Stony Brook University, 
      Stony Brook, NY 11794, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.51f00
PT  - Journal Article
DEP - 20160425
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Fossils
MH  - Lemur/anatomy &amp; histology/*classification/genetics
MH  - Madagascar
MH  - *Phylogeny
MH  - Time
OTO - NOTNLM
OT  - *Bayesian phylogenetics
OT  - *calibration
OT  - *chronogram
OT  - *primatology
OT  - *total evidence
EDAT- 2016/04/27 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/04/27 06:00
PHST- 2015/06/09 00:00 [received]
PHST- 2016/03/30 00:00 [accepted]
PHST- 2016/04/27 06:00 [entrez]
PHST- 2016/04/27 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
AID - syw035 [pii]
AID - 10.1093/sysbio/syw035 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):772-91. doi: 10.1093/sysbio/syw035. Epub 2016 Apr 25.

PMID- 27103169
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Multiple Loci and Complete Taxonomic Sampling Resolve the Phylogeny and
      Biogeographic History of Tenrecs (Mammalia: Tenrecidae) and Reveal Higher
      Speciation Rates in Madagascar's Humid Forests.
PG  - 890-909
LID - 10.1093/sysbio/syw034 [doi]
AB  - The family Tenrecidae (tenrecs) is one of only four extant terrestrial mammal
      lineages to have colonized and diversified on Madagascar. Over the last 15 years,
      several studies have disagreed on relationships among major tenrec lineages,
      resulting in multiple reinterpretations of the number and timing of historical
      transoceanic dispersal events between Africa and Madagascar. We reconstructed the
      phylogeny of Tenrecidae using multiple loci from all recognized extant species
      and estimated divergence timing using six fossil calibrations within Afrotheria. 
      All phylogenetic analyses strongly support monophyly of the Malagasy tenrecs, and
      our divergence timing analysis places their colonization of the island at 30-56
      Ma. Our comprehensive phylogeny supports three important taxonomic revisions that
      reflect the evolutionary history of tenrecs: (1) we formally elevate the African 
      otter shrews to their own family Potamogalidae, thereby rendering extant
      Tenrecidae entirely endemic to Madagascar; (2) we subsume the semiaquatic genus
      Limnogale within the shrew tenrec genus Microgale; and (3) we re-elevate the two 
      largest-bodied shrew tenrecs, Microgale dobsoni and Microgale talazaci, to the
      genus Nesogale Thomas (1918) Finally, we use recently summarized habitat data to 
      test the hypothesis that diversification rates differ between humid and arid
      habitats on Madagascar, and we compare three common methods for ancestral
      biogeographic reconstruction. These analyses suggest higher speciation rates in
      humid habitats and reveal a minimum of three and more likely five independent
      transitions to arid habitats. Our results resolve the relationships among
      previously recalcitrant taxa, illuminate the timing and mechanisms of major
      biogeographic patterns in an extraordinary example of an island radiation, and
      permit the first comprehensive, phylogenetically consistent taxonomy of
      Madagascar's tenrecs.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Everson, Kathryn M
AU  - Everson KM
AD  - University of Alaska Museum, 907 Yukon Drive, Fairbanks, Alaska 99775, USA;
      link.olson@alaska.edu kmeverson@alaska.edu.
FAU - Soarimalala, Voahangy
AU  - Soarimalala V
AD  - Association Vahatra, BP 3972, Antananarivo 101, Madagascar; and.
FAU - Goodman, Steven M
AU  - Goodman SM
AD  - Association Vahatra, BP 3972, Antananarivo 101, Madagascar; and Field Museum of
      Natural History, 1400 South Lake Shore Drive, Chicago, Illinois 60605, USA;
FAU - Olson, Link E
AU  - Olson LE
AD  - University of Alaska Museum, 907 Yukon Drive, Fairbanks, Alaska 99775, USA;
      link.olson@alaska.edu kmeverson@alaska.edu.
LA  - eng
SI  - Dryad/10.5061/dryad.711dc
PT  - Journal Article
DEP - 20160421
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Africa
MH  - Animals
MH  - Ecosystem
MH  - Forests
MH  - Genetic Speciation
MH  - Madagascar
MH  - *Phylogeny
MH  - Shrews/*classification/*genetics
MH  - Time Factors
OTO - NOTNLM
OT  - *Afrotheria
OT  - *Madagascar
OT  - *Tenrecidae
OT  - *biogeography
OT  - *taxonomy
EDAT- 2016/04/23 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/04/23 06:00
PHST- 2015/06/15 00:00 [received]
PHST- 2016/04/07 00:00 [accepted]
PHST- 2016/04/23 06:00 [entrez]
PHST- 2016/04/23 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
AID - syw034 [pii]
AID - 10.1093/sysbio/syw034 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):890-909. doi: 10.1093/sysbio/syw034. Epub 2016 Apr 21.

PMID- 27073251
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - Palaeohistological Evidence for Ancestral High Metabolic Rate in Archosaurs.
PG  - 989-996
AB  - Metabolic heat production in archosaurs has played an important role in their
      evolutionary radiation during the Mesozoic, and their ancestral metabolic
      condition has long been a matter of debate in systematics and palaeontology. The 
      study of fossil bone histology provides crucial information on bone growth rate, 
      which has been used to indirectly investigate the evolution of thermometabolism
      in archosaurs. However, no quantitative estimation of metabolic rate has ever
      been performed on fossils using bone histological features. Moreover, to date, no
      inference model has included phylogenetic information in the form of predictive
      variables. Here we performed statistical predictive modeling using the new method
      of phylogenetic eigenvector maps on a set of bone histological features for a
      sample of extant and extinct vertebrates, to estimate metabolic rates of fossil
      archosauromorphs. This modeling procedure serves as a case study for
      eigenvector-based predictive modeling in a phylogenetic context, as well as an
      investigation of the poorly known evolutionary patterns of metabolic rate in
      archosaurs. Our results show that Mesozoic theropod dinosaurs exhibit metabolic
      rates very close to those found in modern birds, that archosaurs share a higher
      ancestral metabolic rate than that of extant ectotherms, and that this derived
      high metabolic rate was acquired at a much more inclusive level of the
      phylogenetic tree, among non-archosaurian archosauromorphs. These results also
      highlight the difficulties of assigning a given heat production strategy (i.e.,
      endothermy, ectothermy) to an estimated metabolic rate value, and confirm
      findings of previous studies that the definition of the endotherm/ectotherm
      dichotomy may be ambiguous.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Legendre, Lucas J
AU  - Legendre LJ
AD  - Institut des Sciences de la Terre de Paris (ISTeP), Sorbonne Universites -
      Universite Pierre et Marie Curie , 4 Place Jussieu, F-75005 Paris, France;
      lucasjlegendre@gmail.com.
FAU - Guenard, Guillaume
AU  - Guenard G
AD  - Departement de sciences biologiques, Universite de Montreal, CP 6128, Succursale 
      Centre-Ville, Montreal, QC H3C 3J7, Canada.
FAU - Botha-Brink, Jennifer
AU  - Botha-Brink J
AD  - Karoo Palaeontology, National Museum, Bloemfontein 9300, South Africa; and.
AD  - Department of Zoology and Entomology, University of the Free State, Bloemfontein 
      9300, South Africa.
FAU - Cubo, Jorge
AU  - Cubo J
AD  - Institut des Sciences de la Terre de Paris (ISTeP), Sorbonne Universites -
      Universite Pierre et Marie Curie , 4 Place Jussieu, F-75005 Paris, France.
LA  - eng
SI  - Dryad/10.5061/dryad.2853k
PT  - Journal Article
DEP - 20160412
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Basal Metabolism/*physiology
MH  - Biological Evolution
MH  - Birds
MH  - Dinosaurs/*physiology
MH  - *Fossils
MH  - *Models, Biological
MH  - Paleontology
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Archosaurs
OT  - *bone histology
OT  - *phylogenetic comparative methods
OT  - *phylogenetic eigenvectors
OT  - *resting metabolic rate
OT  - *vertebrate palaeontology
EDAT- 2016/04/14 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/04/14 06:00
PHST- 2015/01/23 00:00 [received]
PHST- 2016/04/05 00:00 [revised]
PHST- 2016/04/06 00:00 [accepted]
PHST- 2016/04/14 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
PHST- 2016/04/14 06:00 [entrez]
AID - syw033 [pii]
AID - 10.1093/sysbio/syw033 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):989-996. doi: 10.1093/sysbio/syw033. Epub 2016 Apr 12.

PMID- 27073250
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - Nomenclature for the Nameless: A Proposal for an Integrative Molecular Taxonomy
      of Cryptic Diversity Exemplified by Planktonic Foraminifera.
PG  - 925-40
LID - 10.1093/sysbio/syw031 [doi]
AB  - Investigations of biodiversity, biogeography, and ecological processes rely on
      the identification of "species" as biologically significant, natural units of
      evolution. In this context, morphotaxonomy only provides an adequate level of
      resolution if reproductive isolation matches morphological divergence. In many
      groups of organisms, morphologically defined species often disguise considerable 
      genetic diversity, which may be indicative of the existence of cryptic species.
      The diversity hidden by morphological species can be disentangled through genetic
      surveys, which also provide access to data on the ecological distribution of
      genetically circumscribed units. These units can be identified by unique DNA
      sequence motifs and allow studies of evolutionary and ecological processes at
      different levels of divergence. However, the nomenclature of genetically
      circumscribed units within morphological species is not regulated and lacks
      stability. This represents a major obstacle to efforts to synthesize and
      communicate data on genetic diversity for multiple stakeholders. We have been
      confronted with such an obstacle in our work on planktonic foraminifera, where
      the stakeholder community is particularly diverse, involving geochemists,
      paleoceanographers, paleontologists, and biologists, and the lack of stable
      nomenclature beyond the level of formal morphospecies prevents effective transfer
      of knowledge. To circumvent this problem, we have designed a stable,
      reproducible, and flexible nomenclature system for genetically circumscribed
      units, analogous to the principles of a formal nomenclature system. Our system is
      based on the definition of unique DNA sequence motifs collocated within an
      individual, their typification (in analogy with holotypes), utilization of their 
      hierarchical phylogenetic structure to define levels of divergence below that of 
      the morphospecies, and a set of nomenclature rules assuring stability. The
      resulting molecular operational taxonomic units remain outside the domain of
      current nomenclature codes, but are linked to formal morphospecies as regulated
      by the codes. Subsequently, we show how this system can be applied to classify
      genetically defined units using the SSU rDNA marker in planktonic foraminifera
      and we highlight its potential use for other groups of organisms where similarly 
      high levels of connectivity between molecular and formal taxonomies can be
      achieved.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Morard, Raphael
AU  - Morard R
AD  - MARUM Center for Marine Environmental Sciences, University of Bremen, Leobener
      Strasse, 28359 Bremen, Germany, rmorard@marum.de.
FAU - Escarguel, Gilles
AU  - Escarguel G
AD  - Universite de Lyon; UMR5023 Ecologie des Hydrosystemes Naturels et Anthropises;
      Universitei Lyon 1; ENTPE; CNRS; 6 rue Raphael Dubois, 69622 Villeurbanne,
      France.
FAU - Weiner, Agnes K M
AU  - Weiner AK
AD  - MARUM Center for Marine Environmental Sciences, University of Bremen, Leobener
      Strasse, 28359 Bremen, Germany, Japan Agency for Marine Earth Science and
      Technology (JAMSTEC), 2-15 Natsushima-cho, Yokosuka 237-0061, Kanagawa, Japan.
FAU - Andre, Aurore
AU  - Andre A
AD  - Universite de Reims-Champagne-Ardenne, UFR Sciences Exactes et Naturelles, Campus
      Moulin de la Housse, Batiment 18, 51100 REIMS, France.
FAU - Douady, Christophe J
AU  - Douady CJ
AD  - Universite de Lyon; UMR5023 Ecologie des Hydrosystemes Naturels et Anthropises;
      Universitei Lyon 1; ENTPE; CNRS; 6 rue Raphael Dubois, 69622 Villeurbanne,
      France, Institut Universitaire de France, 103 Boulevard Saint-Michel, 75005
      Paris, France.
FAU - Wade, Christopher M
AU  - Wade CM
AD  - School of Life Sciences, University of Nottingham, University Park, Nottingham
      NG7 2RD, UK.
FAU - Darling, Kate F
AU  - Darling KF
AD  - School of GeoSciences, University of Edinburgh, Edinburgh EH9 3JW, UK, School of 
      Geography and GeoSciences, University of St Andrews, Fife KY16 9AL, UK.
FAU - Ujiie, Yurika
AU  - Ujiie Y
AD  - Department of Biology, Shinshu University, Asahi3-1-1, Matsumoto, Nagano
      390-8621, Japan.
FAU - Seears, Heidi A
AU  - Seears HA
AD  - Department of Biology, Gilmer Hall, University of Virginia, 485 McCormick Road,
      Charlottesville, VA 22904, USA.
FAU - Quillevere, Frederic
AU  - Quillevere F
AD  - Univ Lyon, Universite Lyon 1, ENS de Lyon, CNRS, UMR 5276 LGL-TPE, F-69622
      Villeurbanne, France;
FAU - de Garidel-Thoron, Thibault
AU  - de Garidel-Thoron T
AD  - Centre Europeen de Recherche et d'Enseignement de Geosciences de l'Environnement,
      Centre National de la Recherche Scientifique, et Aix-Marseille Universite,
      Aix-en-Provence, France.
FAU - de Vargas, Colomban
AU  - de Vargas C
AD  - Centre National de la Recherche Scientifique, UMR 7144, EPEP, Station Biologique 
      de Roscoff, 29680 Roscoff, France, and Sorbonne Universites, UPMC Univ Paris 06, 
      UMR 7144, Station Biologique de Roscoff, 29680 Roscoff, France.
FAU - Kucera, Michal
AU  - Kucera M
AD  - MARUM Center for Marine Environmental Sciences, University of Bremen, Leobener
      Strasse, 28359 Bremen, Germany.
LA  - eng
SI  - Dryad/10.5061/dryad.64pg3
PT  - Journal Article
DEP - 20160412
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal)
SB  - IM
MH  - Biodiversity
MH  - Classification/*methods
MH  - DNA, Ribosomal
MH  - Foraminifera/*classification
MH  - Genetic Variation
MH  - *Phylogeny
MH  - Plankton/classification
OTO - NOTNLM
OT  - *Cryptic species
OT  - *MOTUs
OT  - *genetic diversity
OT  - *molecular nomenclature
OT  - *planktonic foraminifera
EDAT- 2016/04/14 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/04/14 06:00
PHST- 2015/10/01 00:00 [received]
PHST- 2016/04/04 00:00 [accepted]
PHST- 2016/04/14 06:00 [entrez]
PHST- 2016/04/14 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
AID - syw031 [pii]
AID - 10.1093/sysbio/syw031 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):925-40. doi: 10.1093/sysbio/syw031. Epub 2016 Apr 12.

PMID- 27055648
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 5
DP  - 2016 Sep
TI  - A Rapid and Scalable Method for Multilocus Species Delimitation Using Bayesian
      Model Comparison and Rooted Triplets.
PG  - 759-71
LID - 10.1093/sysbio/syw028 [doi]
AB  - Multilocus sequence data provide far greater power to resolve species limits than
      the single locus data typically used for broad surveys of clades. However,
      current statistical methods based on a multispecies coalescent framework are
      computationally demanding, because of the number of possible delimitations that
      must be compared and time-consuming likelihood calculations. New methods are
      therefore needed to open up the power of multilocus approaches to larger
      systematic surveys. Here, we present a rapid and scalable method that introduces 
      2 new innovations. First, the method reduces the complexity of likelihood
      calculations by decomposing the tree into rooted triplets. The distribution of
      topologies for a triplet across multiple loci has a uniform trinomial
      distribution when the 3 individuals belong to the same species, but a skewed
      distribution if they belong to separate species with a form that is specified by 
      the multispecies coalescent. A Bayesian model comparison framework was developed 
      and the best delimitation found by comparing the product of posterior
      probabilities of all triplets. The second innovation is a new dynamic programming
      algorithm for finding the optimum delimitation from all those compatible with a
      guide tree by successively analyzing subtrees defined by each node. This
      algorithm removes the need for heuristic searches used by current methods, and
      guarantees that the best solution is found and potentially could be used in other
      systematic applications. We assessed the performance of the method with
      simulated, published, and newly generated data. Analyses of simulated data
      demonstrate that the combined method has favorable statistical properties and
      scalability with increasing sample sizes. Analyses of empirical data from both
      eukaryotes and prokaryotes demonstrate its potential for delimiting species in
      real cases.
CI  - (c) The Author(s) 2016. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - Fujisawa, Tomochika
AU  - Fujisawa T
AD  - Department of Zoology, Kyoto University, Sakyo, Kyoto 606-8502, Japan;
      t.fujisawa05@gmail.com.
FAU - Aswad, Amr
AU  - Aswad A
AD  - Department of Life Sciences, Imperial College London, Silwood Park Campus, Ascot,
      Berkshire SL5 7PY, UK;
FAU - Barraclough, Timothy G
AU  - Barraclough TG
AD  - Department of Life Sciences, Imperial College London, Silwood Park Campus, Ascot,
      Berkshire SL5 7PY, UK;
LA  - eng
SI  - Dryad/10.5061/dryad.3cb25
GR  - BB/G004250/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
DEP - 20160407
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Animals
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Eukaryota/classification
MH  - Markov Chains
MH  - Monte Carlo Method
MH  - *Phylogeny
MH  - Prokaryotic Cells/classification
MH  - Species Specificity
PMC - PMC4997007
OTO - NOTNLM
OT  - *Bacterial species
OT  - *Bayesian model comparison
OT  - *Dynamic programming
OT  - *Multilocus species delimitation
EDAT- 2016/04/09 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/04/09 06:00
PHST- 2015/03/02 00:00 [received]
PHST- 2016/03/30 00:00 [accepted]
PHST- 2016/04/09 06:00 [entrez]
PHST- 2016/04/09 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
AID - syw028 [pii]
AID - 10.1093/sysbio/syw028 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Sep;65(5):759-71. doi: 10.1093/sysbio/syw028. Epub 2016 Apr 7.

PMID- 27048703
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - Testing the Relationships between Diversification, Species Richness, and Trait
      Evolution.
PG  - 975-988
AB  - Understanding which traits drive species diversification is essential for
      macroevolutionary studies and to understand patterns of species richness among
      clades. An important tool for testing if traits influence diversification is to
      estimate rates of net diversification for each clade, and then test for a
      relationship between traits and diversification rates among clades. However, this
      general approach has become very controversial. Numerous papers have now stated
      that it is inappropriate to analyze net diversification rates in groups in which 
      clade richness is not positively correlated with clade age. Similarly, some have 
      stated that variation in net diversification rates does not explain variation in 
      species richness patterns among clades across the Tree of Life. Some authors have
      also suggested that strong correlations between richness and diversification
      rates are a statistical artifact and effectively inevitable. If this latter point
      is true, then correlations between richness and diversification rates would be
      uninformative (or even misleading) for identifying how much variation in species 
      richness among clades is explained by variation in net diversification rates.
      Here, we use simulations (based on empirical data for plethodontid salamanders)
      to address three main questions. First, how is variation in net diversification
      rates among clades related to the relationship between clade age and species
      richness? Second, how accurate are these net diversification rate estimators, and
      does the age-richness relationship have any relevance to their accuracy? Third,
      is a relationship between species richness and diversification rates an
      inevitable, statistical artifact? Our simulations show that strong, positive
      age-richness relationships arise when diversification rates are invariant among
      clades, whereas realistic variation in diversification rates among clades
      frequently disrupts this relationship. Thus, a significant age-richness
      relationship should not be a requirement for utilizing net diversification rates 
      in macroevolutionary studies. Moreover, we find no difference in the accuracy of 
      net diversification rate estimators between conditions in which there are strong,
      positive relationships between clade age and richness and conditions in which
      these strong relationships are absent. We find that net diversification rate
      estimators are reasonably accurate under many conditions (true and estimated
      rates are strongly corrrelated, and typically differ by approximately 10-20%),
      but become more accurate when clades are older and less accurate when using
      incorrect assumptions about extinction. We also find that significant
      relationships between richness and diversification rates fail to arise under many
      conditions, especially when there are faster rates in younger clades. Therefore, 
      a significant relationship between richness and diversification rates is not
      inevitable. Given this latter result, we suggest that relationships between
      richness and diversification should be tested for when attempting to explain the 
      causes of richness patterns, to avoid potential misinterpretations (e.g., high
      diversification rates associated with low-richness clades). Similarly, our
      results also provide some support for previous studies suggesting that variation 
      in diversification rates might explain much of the variation in species richness 
      among major clades, based on strong relationships between clade richness and
      diversification rates.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Kozak, Kenneth H
AU  - Kozak KH
AD  - Bell Museum of Natural History and Department of Fisheries, Wildlife, and
      Conservation Biology, University of Minnesota, St. Paul, MN 55108, USA.
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Department of Ecology and Evolution, University of Arizona, Tucson, AZ 85721,
      USA; wiensj@email.arizona.edu.
LA  - eng
PT  - Journal Article
DEP - 20160404
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biodiversity
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Phenotype
MH  - *Phylogeny
MH  - Time Factors
MH  - Urodela/classification
OTO - NOTNLM
OT  - *Diversification
OT  - *simulations
OT  - *species richness
EDAT- 2016/04/07 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/04/07 06:00
PHST- 2016/03/09 00:00 [received]
PHST- 2016/03/30 00:00 [accepted]
PHST- 2016/04/07 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
PHST- 2016/04/07 06:00 [entrez]
AID - syw029 [pii]
AID - 10.1093/sysbio/syw029 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):975-988. doi: 10.1093/sysbio/syw029. Epub 2016 Apr 4.

PMID- 27037081
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20190112
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - How Well Can We Detect Lineage-Specific Diversification-Rate Shifts? A Simulation
      Study of Sequential AIC Methods.
PG  - 1076-1084
AB  - Evolutionary biologists have long been fascinated by the extreme differences in
      species numbers across branches of the Tree of Life. This has motivated the
      development of statistical methods for detecting shifts in the rate of lineage
      diversification across the branches of phylogenic trees. One of the most
      frequently used methods, MEDUSA, explores a set of diversification-rate models,
      where each model assigns branches of the phylogeny to a set of
      diversification-rate categories. Each model is first fit to the data, and the
      Akaike information criterion (AIC) is then used to identify the optimal
      diversification model. Surprisingly, the statistical behavior of this popular
      method is uncharacterized, which is a concern in light of: (1) the poor
      performance of the AIC as a means of choosing among models in other phylogenetic 
      contexts; (2) the ad hoc algorithm used to visit diversification models, and; (3)
      errors that we reveal in the likelihood function used to fit diversification
      models to the phylogenetic data. Here, we perform an extensive simulation study
      demonstrating that MEDUSA (1) has a high false-discovery rate (on average,
      spurious diversification-rate shifts are identified [Formula: see text] of the
      time), and (2) provides biased estimates of diversification-rate parameters.
      Understanding the statistical behavior of MEDUSA is critical both to empirical
      researchers-in order to clarify whether these methods can make reliable
      inferences from empirical datasets-and to theoretical biologists-in order to
      clarify the specific problems that need to be solved in order to develop more
      reliable approaches for detecting shifts in the rate of lineage diversification. 
      [Akaike information criterion; extinction; lineage-specific diversification
      rates; phylogenetic model selection; speciation.].
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - May, Michael R
AU  - May MR
AD  - Department of Evolution and Ecology, University of California, Davis, Davis, CA
      95616, USA mikeryanmay@gmail.com.
FAU - Moore, Brian R
AU  - Moore BR
AD  - Department of Evolution and Ecology, University of California, Davis, Davis, CA
      95616, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.61q64
PT  - Journal Article
DEP - 20160401
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Biological Evolution
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - Likelihood Functions
MH  - *Phylogeny
MH  - Time Factors
PMC - PMC5066061
EDAT- 2016/04/03 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/04/03 06:00
PHST- 2014/11/18 00:00 [received]
PHST- 2016/03/25 00:00 [accepted]
PHST- 2016/04/03 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
PHST- 2016/04/03 06:00 [entrez]
AID - syw026 [pii]
AID - 10.1093/sysbio/syw026 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):1076-1084. doi: 10.1093/sysbio/syw026. Epub 2016 Apr 1.

PMID- 27016728
OWN - NLM
STAT- MEDLINE
DCOM- 20180105
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 4
DP  - 2016 Jul
TI  - Detecting Hidden Diversification Shifts in Models of Trait-Dependent Speciation
      and Extinction.
PG  - 583-601
LID - 10.1093/sysbio/syw022 [doi]
AB  - The distribution of diversity can vary considerably from clade to clade. Attempts
      to understand these patterns often employ state-dependent speciation and
      extinction models to determine whether the evolution of a particular novel trait 
      has increased speciation rates and/or decreased extinction rates. It is still
      unclear, however, whether these models are uncovering important drivers of
      diversification, or whether they are simply pointing to more complex patterns
      involving many unmeasured and co-distributed factors. Here we describe an
      extension to the popular state-dependent speciation and extinction models that
      specifically accounts for the presence of unmeasured factors that could impact
      diversification rates estimated for the states of any observed trait, addressing 
      at least one major criticism of BiSSE (Binary State Speciation and Extinction)
      methods. Specifically, our model, which we refer to as HiSSE (Hidden State
      Speciation and Extinction), assumes that related to each observed state in the
      model are "hidden" states that exhibit potentially distinct diversification
      dynamics and transition rates than the observed states in isolation. We also
      demonstrate how our model can be used as character-independent diversification
      models that allow for a complex diversification process that is independent of
      the evolution of a character. Under rigorous simulation tests and when applied to
      empirical data, we find that HiSSE performs reasonably well, and can at least
      detect net diversification rate differences between observed and hidden states
      and detect when diversification rate differences do not correlate with the
      observed states. We discuss the remaining issues with state-dependent speciation 
      and extinction models in general, and the important ways in which HiSSE provides 
      a more nuanced understanding of trait-dependent diversification.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Beaulieu, Jeremy M
AU  - Beaulieu JM
AD  - National Institute for Biological and Mathematical Synthesis, University of
      Tennessee, Knoxville, TN 37996, USA Department of Ecology and Evolutionary
      Biology, University of Tennessee, Knoxville, TN 37996-1610, USA
      jbeaulieu@nimbios.org.
FAU - O'Meara, Brian C
AU  - O'Meara BC
AD  - Department of Ecology and Evolutionary Biology, University of Tennessee,
      Knoxville, TN 37996-1610, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.52hp1
PT  - Journal Article
DEP - 20160325
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biodiversity
MH  - Classification/*methods
MH  - *Extinction, Biological
MH  - *Genetic Speciation
MH  - *Models, Biological
MH  - Phenotype
MH  - Phylogeny
OTO - NOTNLM
OT  - *Comparative methods
OT  - *diversification
OT  - *extinction
OT  - *hidden states
OT  - *speciation
EDAT- 2016/03/27 06:00
MHDA- 2018/01/06 06:00
CRDT- 2016/03/27 06:00
PHST- 2015/03/10 00:00 [received]
PHST- 2016/03/08 00:00 [accepted]
PHST- 2016/03/27 06:00 [entrez]
PHST- 2016/03/27 06:00 [pubmed]
PHST- 2018/01/06 06:00 [medline]
AID - syw022 [pii]
AID - 10.1093/sysbio/syw022 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jul;65(4):583-601. doi: 10.1093/sysbio/syw022. Epub 2016 Mar 25.

PMID- 27009895
OWN - NLM
STAT- MEDLINE
DCOM- 20180105
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 4
DP  - 2016 Jul
TI  - Two Influential Primate Classifications Logically Aligned.
PG  - 561-82
LID - 10.1093/sysbio/syw023 [doi]
AB  - Classifications and phylogenies of perceived natural entities change in the light
      of new evidence. Taxonomic changes, translated into Code-compliant names,
      frequently lead to name:meaning dissociations across succeeding treatments.
      Classification standards such as the Mammal Species of the World (MSW) may
      experience significant levels of taxonomic change from one edition to the next,
      with potential costs to long-term, large-scale information integration. This
      circumstance challenges the biodiversity and phylogenetic data communities to
      express taxonomic congruence and incongruence in ways that both humans and
      machines can process, that is, to logically represent taxonomic alignments across
      multiple classifications. We demonstrate that such alignments are feasible for
      two classifications of primates corresponding to the second and third MSW
      editions. Our approach has three main components: (i) use of taxonomic concept
      labels, that is name sec. author (where sec. means according to), to assemble
      each concept hierarchy separately via parent/child relationships; (ii)
      articulation of select concepts across the two hierarchies with user-provided
      Region Connection Calculus (RCC-5) relationships; and (iii) the use of an Answer 
      Set Programming toolkit to infer and visualize logically consistent alignments of
      these input constraints. Our use case entails the Primates sec. Groves (1993;
      MSW2-317 taxonomic concepts; 233 at the species level) and Primates sec. Groves
      (2005; MSW3-483 taxonomic concepts; 376 at the species level). Using 402 RCC-5
      input articulations, the reasoning process yields a single, consistent alignment 
      and 153,111 Maximally Informative Relations that constitute a comprehensive
      meaning resolution map for every concept pair in the Primates sec. MSW2/MSW3. The
      complete alignment, and various partitions thereof, facilitate quantitative
      analyses of name:meaning dissociation, revealing that nearly one in three
      taxonomic names are not reliable across treatments-in the sense of the same name 
      identifying congruent taxonomic meanings. The RCC-5 alignment approach is
      potentially widely applicable in systematics and can achieve scalable, precise
      resolution of semantically evolving name usages in synthetic, next-generation
      biodiversity, and phylogeny data platforms.
CI  - (c) The Author(s) 2016. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - Franz, Nico M
AU  - Franz NM
AD  - School of Life Sciences, PO Box 874501, Arizona State University, Tempe, AZ
      85287, USA; nico.franz@asu.edu.
FAU - Pier, Naomi M
AU  - Pier NM
AD  - School of Life Sciences, PO Box 874501, Arizona State University, Tempe, AZ
      85287, USA;
FAU - Reeder, Deeann M
AU  - Reeder DM
AD  - Department of Biology, Bucknell University, 1 Dent Drive, Lewisburg, PA 17837,
      USA;
FAU - Chen, Mingmin
AU  - Chen M
AD  - Department of Computer Science, 2063 Kemper Hall, 1 Shields Avenue, University of
      California at Davis, CA 95616, USA;
FAU - Yu, Shizhuo
AU  - Yu S
AD  - Department of Computer Science, 2063 Kemper Hall, 1 Shields Avenue, University of
      California at Davis, CA 95616, USA;
FAU - Kianmajd, Parisa
AU  - Kianmajd P
AD  - Department of Computer Science, 2063 Kemper Hall, 1 Shields Avenue, University of
      California at Davis, CA 95616, USA;
FAU - Bowers, Shawn
AU  - Bowers S
AD  - Department of Computer Science, 502 East Boone Avenue, AD Box 26, Gonzaga
      University, Spokane, WA 99258, USA;
FAU - Ludascher, Bertram
AU  - Ludascher B
AD  - Gradate School of Library and Information Science, 510 East Daniel Street,
      University of Illinois at Urbana-Champaign, Champaign, IL 61820.
LA  - eng
SI  - Dryad/10.5061/dryad.6jg71
PT  - Journal Article
DEP - 20160322
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - Classification/*methods
MH  - *Phylogeny
MH  - Primates/*classification
PMC - PMC4911943
OTO - NOTNLM
OT  - *Alignment
OT  - *Primates
OT  - *Region Connection Calculus
OT  - *classification
OT  - *concept taxonomy
OT  - *logic
OT  - *ontology
OT  - *reasoning
EDAT- 2016/03/25 06:00
MHDA- 2018/01/06 06:00
CRDT- 2016/03/25 06:00
PHST- 2015/02/16 00:00 [received]
PHST- 2016/03/17 00:00 [accepted]
PHST- 2016/03/25 06:00 [entrez]
PHST- 2016/03/25 06:00 [pubmed]
PHST- 2018/01/06 06:00 [medline]
AID - syw023 [pii]
AID - 10.1093/sysbio/syw023 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jul;65(4):561-82. doi: 10.1093/sysbio/syw023. Epub 2016 Mar 22.

PMID- 26968785
OWN - NLM
STAT- MEDLINE
DCOM- 20180105
LR  - 20190111
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 4
DP  - 2016 Jul
TI  - Does Gene Tree Discordance Explain the Mismatch between Macroevolutionary Models 
      and Empirical Patterns of Tree Shape and Branching Times?
PG  - 628-39
LID - 10.1093/sysbio/syw019 [doi]
AB  - Classic null models for speciation and extinction give rise to phylogenies that
      differ in distribution from empirical phylogenies. In particular, empirical
      phylogenies are less balanced and have branching times closer to the root
      compared to phylogenies predicted by common null models. This difference might be
      due to null models of the speciation and extinction process being too simplistic,
      or due to the empirical datasets not being representative of random phylogenies. 
      A third possibility arises because phylogenetic reconstruction methods often
      infer gene trees rather than species trees, producing an incongruity between
      models that predict species tree patterns and empirical analyses that consider
      gene trees. We investigate the extent to which the difference between gene trees 
      and species trees under a combined birth-death and multispecies coalescent model 
      can explain the difference in empirical trees and birth-death species trees. We
      simulate gene trees embedded in simulated species trees and investigate their
      difference with respect to tree balance and branching times. We observe that the 
      gene trees are less balanced and typically have branching times closer to the
      root than the species trees. Empirical trees from TreeBase are also less balanced
      than our simulated species trees, and model gene trees can explain an imbalance
      increase of up to 8% compared to species trees. However, we see a much larger
      imbalance increase in empirical trees, about 100%, meaning that additional
      features must also be causing imbalance in empirical trees. This simulation study
      highlights the necessity of revisiting the assumptions made in phylogenetic
      analyses, as these assumptions, such as equating the gene tree with the species
      tree, might lead to a biased conclusion.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Department of Biosystems Science and Engineering, ETH Zurich, Mattenstrasse 26,
      4058 Basel Swiss Institute of Bioinformatics (SIB), 1015 Lausanne, Switzerland
      tanja.stadler@bsse.ethz.ch.
FAU - Degnan, James H
AU  - Degnan JH
AD  - Department of Mathematics and Statistics, University of New Mexico, 311 Terrace
      NE, Albuquerque, NM, 87131, USA;
FAU - Rosenberg, Noah A
AU  - Rosenberg NA
AD  - Department of Biology, Stanford University, 371 Serra Mall, Stanford, CA 94305,
      USA; and.
LA  - eng
SI  - Dryad/10.5061/dryad.8f97r
GR  - R01 GM117590/GM/NIGMS NIH HHS/United States
PT  - Journal Article
DEP - 20160311
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Time
PMC - PMC4911941
OTO - NOTNLM
OT  - *Birth-death process
OT  - *genealogy
OT  - *multispecies coalescent
OT  - *phylogeny
EDAT- 2016/03/13 06:00
MHDA- 2018/01/06 06:00
CRDT- 2016/03/13 06:00
PHST- 2015/06/05 00:00 [received]
PHST- 2016/03/01 00:00 [accepted]
PHST- 2016/03/13 06:00 [entrez]
PHST- 2016/03/13 06:00 [pubmed]
PHST- 2018/01/06 06:00 [medline]
AID - syw019 [pii]
AID - 10.1093/sysbio/syw019 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jul;65(4):628-39. doi: 10.1093/sysbio/syw019. Epub 2016 Mar 11.

PMID- 26966005
OWN - NLM
STAT- MEDLINE
DCOM- 20180105
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 4
DP  - 2016 Jul
TI  - Estimating the Effect of Competition on Trait Evolution Using Maximum Likelihood 
      Inference.
PG  - 700-10
LID - 10.1093/sysbio/syw020 [doi]
AB  - Many classical ecological and evolutionary theoretical frameworks posit that
      competition between species is an important selective force. For example, in
      adaptive radiations, resource competition between evolving lineages plays a role 
      in driving phenotypic diversification and exploration of novel ecological space. 
      Nevertheless, current models of trait evolution fit to phylogenies and
      comparative data sets are not designed to incorporate the effect of competition. 
      The most advanced models in this direction are diversity-dependent models where
      evolutionary rates depend on lineage diversity. However, these models still treat
      changes in traits in one branch as independent of the value of traits on other
      branches, thus ignoring the effect of species similarity on trait evolution.
      Here, we consider a model where the evolutionary dynamics of traits involved in
      interspecific interactions are influenced by species similarity in trait values
      and where we can specify which lineages are in sympatry. We develop a maximum
      likelihood based approach to fit this model to combined phylogenetic and
      phenotypic data. Using simulations, we demonstrate that the approach accurately
      estimates the simulated parameter values across a broad range of parameter space.
      Additionally, we develop tools for specifying the biogeographic context in which 
      trait evolution occurs. In order to compare models, we also apply these
      biogeographic methods to specify which lineages interact sympatrically for two
      diversity-dependent models. Finally, we fit these various models to morphological
      data from a classical adaptive radiation (Greater Antillean Anolis lizards). We
      show that models that account for competition and geography perform better than
      other models. The matching competition model is an important new tool for
      studying the influence of interspecific interactions, in particular competition, 
      on phenotypic evolution. More generally, it constitutes a step toward a better
      integration of interspecific interactions in many ecological and evolutionary
      processes.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Drury, Jonathan
AU  - Drury J
AD  - Institut de Biologie de l'Ecole Normale Superieure (IBENS), CNRS, Inserm, Ecole
      Normale Superieure, PSL Research University, F-75005 Paris, France
      drury@biologie.ens.fr.
FAU - Clavel, Julien
AU  - Clavel J
AD  - Institut de Biologie de l'Ecole Normale Superieure (IBENS), CNRS, Inserm, Ecole
      Normale Superieure, PSL Research University, F-75005 Paris, France.
FAU - Manceau, Marc
AU  - Manceau M
AD  - Institut de Biologie de l'Ecole Normale Superieure (IBENS), CNRS, Inserm, Ecole
      Normale Superieure, PSL Research University, F-75005 Paris, France.
FAU - Morlon, Helene
AU  - Morlon H
AD  - Institut de Biologie de l'Ecole Normale Superieure (IBENS), CNRS, Inserm, Ecole
      Normale Superieure, PSL Research University, F-75005 Paris, France.
LA  - eng
SI  - Dryad/10.5061/dryad.d670p
PT  - Journal Article
DEP - 20160309
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - *Competitive Behavior
MH  - *Computer Simulation
MH  - Geography
MH  - Likelihood Functions
MH  - Lizards/classification/physiology
MH  - *Models, Biological
MH  - Phenotype
MH  - Phylogeny
OTO - NOTNLM
OT  - *Adaptive radiation
OT  - *Anolis
OT  - *community phylogenetics
OT  - *interspecific competition
OT  - *maximum likelihood
OT  - *phylogenetic comparative methods
OT  - *trait evolution
EDAT- 2016/03/12 06:00
MHDA- 2018/01/06 06:00
CRDT- 2016/03/12 06:00
PHST- 2015/07/31 00:00 [received]
PHST- 2016/03/01 00:00 [accepted]
PHST- 2016/03/12 06:00 [entrez]
PHST- 2016/03/12 06:00 [pubmed]
PHST- 2018/01/06 06:00 [medline]
AID - syw020 [pii]
AID - 10.1093/sysbio/syw020 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jul;65(4):700-10. doi: 10.1093/sysbio/syw020. Epub 2016 Mar 9.

PMID- 26927960
OWN - NLM
STAT- MEDLINE
DCOM- 20180105
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 4
DP  - 2016 Jul
TI  - Gene Tree Discordance Causes Apparent Substitution Rate Variation.
PG  - 711-21
LID - 10.1093/sysbio/syw018 [doi]
AB  - Substitution rates are known to be variable among genes, chromosomes, species,
      and lineages due to multifarious biological processes. Here, we consider another 
      source of substitution rate variation due to a technical bias associated with
      gene tree discordance. Discordance has been found to be rampant in genome-wide
      data sets, often due to incomplete lineage sorting (ILS). This apparent
      substitution rate variation is caused when substitutions that occur on discordant
      gene trees are analyzed in the context of a single, fixed species tree. Such
      substitutions have to be resolved by proposing multiple substitutions on the
      species tree, and we therefore refer to this phenomenon as Substitutions Produced
      by ILS (SPILS). We use simulations to demonstrate that SPILS has a larger effect 
      with increasing levels of ILS, and on trees with larger numbers of taxa. Specific
      branches of the species trees are consistently, but erroneously, inferred to be
      longer or shorter, and we show that these branches can be predicted based on
      discordant tree topologies. Moreover, we observe that fixing a species tree
      topology when performing tests of positive selection increases the false positive
      rate, particularly for genes whose discordant topologies are most affected by
      SPILS. Finally, we use data from multiple Drosophila species to show that SPILS
      can be detected in nature. Although the effects of SPILS are modest per gene, it 
      has the potential to affect substitution rate variation whenever high levels of
      ILS are present, particularly in rapid radiations. The problems outlined here
      have implications for character mapping of any type of trait, and for any
      biological process that causes discordance. We discuss possible solutions to
      these problems, and areas in which they are likely to have caused faulty
      inferences of convergence and accelerated evolution.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Mendes, Fabio K
AU  - Mendes FK
AD  - Department of Biology and fkmendes@indiana.edu.
FAU - Hahn, Matthew W
AU  - Hahn MW
AD  - Department of Biology and School of Informatics and Computing, Indiana
      University, Bloomington, IN 47405, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.17k8j
PT  - Journal Article
DEP - 20160228
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amino Acid Substitution
MH  - Animals
MH  - Drosophila/genetics
MH  - *Evolution, Molecular
MH  - *Genome
MH  - *Models, Genetic
MH  - Phylogeny
OTO - NOTNLM
OT  - *Gene tree discordance
OT  - *hemiplasy
OT  - *incomplete lineage sorting
OT  - *substitution rates
EDAT- 2016/03/02 06:00
MHDA- 2018/01/06 06:00
CRDT- 2016/03/02 06:00
PHST- 2015/10/15 00:00 [received]
PHST- 2016/02/23 00:00 [accepted]
PHST- 2016/03/02 06:00 [entrez]
PHST- 2016/03/02 06:00 [pubmed]
PHST- 2018/01/06 06:00 [medline]
AID - syw018 [pii]
AID - 10.1093/sysbio/syw018 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jul;65(4):711-21. doi: 10.1093/sysbio/syw018. Epub 2016 Feb 28.

PMID- 26911152
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Bridging Inter- and Intraspecific Trait Evolution with a Hierarchical Bayesian
      Approach.
PG  - 417-31
LID - 10.1093/sysbio/syw010 [doi]
AB  - The evolution of organisms is crucially dependent on the evolution of
      intraspecific variation. Its interactions with selective agents in the biotic and
      abiotic environments underlie many processes, such as intraspecific competition, 
      resource partitioning and, eventually, species formation. Nevertheless,
      comparative models of trait evolution neither allow explicit testing of
      hypotheses related to the evolution of intraspecific variation nor do they
      simultaneously estimate rates of trait evolution by accounting for both trait
      mean and variance. Here, we present a model of phenotypic trait evolution using a
      hierarchical Bayesian approach that simultaneously incorporates interspecific and
      intraspecific variation. We assume that species-specific trait means evolve under
      a simple Brownian motion process, whereas species-specific trait variances are
      modeled with Brownian or Ornstein-Uhlenbeck processes. After evaluating the power
      of the method through simulations, we examine whether life-history traits impact 
      evolution of intraspecific variation in the Eriogonoideae (buckwheat family,
      Polygonaceae). Our model is readily extendible to more complex scenarios of the
      evolution of inter- and intraspecific variation and presents a step toward more
      comprehensive comparative models for macroevolutionary studies.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Kostikova, Anna
AU  - Kostikova A
AD  - Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne,
      Switzerland; Swiss Institute of Bioinformatics, Quartier Sorge, 1015 Lausanne,
      Swizterland;
FAU - Silvestro, Daniele
AU  - Silvestro D
AD  - Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne,
      Switzerland; Swiss Institute of Bioinformatics, Quartier Sorge, 1015 Lausanne,
      Swizterland; Department of Biological and Environmental Sciences, University of
      Gothenburg, Carl Skottsbergsgata 22B, 413 19 Gothenburg, Sweden;
FAU - Pearman, Peter B
AU  - Pearman PB
AD  - Department of Plant Biology and Ecology, University of the Basque Country
      UPV/EHU, 48940 Leioa, Spain; and IKERBASQUE, Basque Foundation for Science, 48013
      Bilbao, Spain.
FAU - Salamin, Nicolas
AU  - Salamin N
AD  - Department of Ecology and Evolution, University of Lausanne, 1015 Lausanne,
      Switzerland; Swiss Institute of Bioinformatics, Quartier Sorge, 1015 Lausanne,
      Swizterland; nicolas.salamin@unil.ch.
LA  - eng
SI  - Dryad/10.5061/dryad.n4vp0
PT  - Journal Article
DEP - 20160223
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - *Biological Evolution
MH  - Classification/*methods
MH  - Phenotype
MH  - Phylogeny
MH  - Polygonaceae/classification
MH  - Species Specificity
OTO - NOTNLM
OT  - *Comparative phylogenetics
OT  - *Markov Chain Monte Carlo
OT  - *hierarchical Bayesian model
OT  - *intraspecific variance
OT  - *macroevolution
OT  - *niche breadth
EDAT- 2016/02/26 06:00
MHDA- 2018/01/11 06:00
CRDT- 2016/02/26 06:00
PHST- 2014/03/26 00:00 [received]
PHST- 2016/01/11 00:00 [accepted]
PHST- 2016/02/26 06:00 [entrez]
PHST- 2016/02/26 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syw010 [pii]
AID - 10.1093/sysbio/syw010 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):417-31. doi: 10.1093/sysbio/syw010. Epub 2016 Feb 23.

PMID- 26880148
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Integrative Taxonomy Recognizes Evolutionary Units Despite Widespread Mitonuclear
      Discordance: Evidence from a Rotifer Cryptic Species Complex.
PG  - 508-24
LID - 10.1093/sysbio/syw016 [doi]
AB  - Mitonuclear discordance across taxa is increasingly recognized as posing a major 
      challenge to species delimitation based on DNA sequence data. Integrative
      taxonomy has been proposed as a promising framework to help address this problem.
      However, we still lack compelling empirical evidence scrutinizing the efficacy of
      integrative taxonomy in relation to, for instance, complex introgression
      scenarios involving many species. Here, we report remarkably widespread
      mitonuclear discordance between about 15 mitochondrial and 4 nuclear Brachionus
      calyciflorus groups identified using different species delimitation approaches.
      Using coalescent-, Bayesian admixture-, and allele sharing-based methods with DNA
      sequence or microsatellite data, we provide strong evidence in support of
      hybridization as a driver of the observed discordance. We then describe our
      combined molecular, morphological, and ecological approaches to resolving
      phylogenetic conflict and inferring species boundaries. Species delimitations
      based on the ITS1 and 28S nuclear DNA markers proved a more reliable predictor of
      morphological variation than delimitations using the mitochondrial COI gene. A
      short-term competition experiment further revealed systematic differences in the 
      competitive ability between two of the nuclear-delimited species under six
      different growth conditions, independent of COI delimitations; hybrids were also 
      observed. In light of these findings, we discuss the failure of the COI marker to
      estimate morphological stasis and morphological plasticity in the B. calyciflorus
      complex. By using B. calyciflorus as a representative case, we demonstrate the
      potential of integrative taxonomy to guide species delimitation in the presence
      of mitonuclear phylogenetic conflicts.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Papakostas, Spiros
AU  - Papakostas S
AD  - Netherlands Institute of Ecology, Department of Aquatic Ecology, Wageningen,
      Netherlands; Division of Genetics and Physiology, Department of Biology,
      University of Turku, Turku, Finland;
FAU - Michaloudi, Evangelia
AU  - Michaloudi E
AD  - Department of Zoology, School of Biology, Aristotle University of Thessaloniki,
      Thessaloniki, Greece;
FAU - Proios, Konstantinos
AU  - Proios K
AD  - Netherlands Institute of Ecology, Department of Aquatic Ecology, Wageningen,
      Netherlands;
FAU - Brehm, Michaela
AU  - Brehm M
AD  - Netherlands Institute of Ecology, Department of Aquatic Ecology, Wageningen,
      Netherlands;
FAU - Verhage, Laurens
AU  - Verhage L
AD  - Netherlands Institute of Ecology, Department of Aquatic Ecology, Wageningen,
      Netherlands;
FAU - Rota, Jadranka
AU  - Rota J
AD  - Division of Genetics and Physiology, Department of Biology, University of Turku, 
      Turku, Finland; Department of Zoology, School of Biology, Aristotle University of
      Thessaloniki, Thessaloniki, Greece; Department of Biology, Lund University, Lund,
      Sweden;
FAU - Pena, Carlos
AU  - Pena C
AD  - Division of Genetics and Physiology, Department of Biology, University of Turku, 
      Turku, Finland;
FAU - Stamou, Georgia
AU  - Stamou G
AD  - Department of Zoology, School of Biology, Aristotle University of Thessaloniki,
      Thessaloniki, Greece;
FAU - Pritchard, Victoria L
AU  - Pritchard VL
AD  - Division of Genetics and Physiology, Department of Biology, University of Turku, 
      Turku, Finland;
FAU - Fontaneto, Diego
AU  - Fontaneto D
AD  - National Research Council, Institute of Ecosystem Study, Verbania Pallanza,
      Italy.
FAU - Declerck, Steven A J
AU  - Declerck SA
AD  - Netherlands Institute of Ecology, Department of Aquatic Ecology, Wageningen,
      Netherlands;
LA  - eng
SI  - Dryad/10.5061/dryad.8rc4r
PT  - Journal Article
DEP - 20160214
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Ribosomal Spacer)
RN  - 0 (Genetic Markers)
RN  - 0 (RNA, Ribosomal, 28S)
SB  - IM
EIN - Syst Biol. 2016 Sep;65(5):945. PMID: 27444424
MH  - Animals
MH  - Bayes Theorem
MH  - DNA, Ribosomal Spacer/genetics
MH  - Genes, Mitochondrial/*genetics
MH  - Genetic Markers/genetics
MH  - Hybridization, Genetic
MH  - *Phylogeny
MH  - RNA, Ribosomal, 28S/genetics
MH  - Rotifera/*classification/genetics
MH  - Species Specificity
OTO - NOTNLM
OT  - *18S and 28S ribosomal RNA genes
OT  - *DNA barcoding
OT  - *GMYC
OT  - *cyclical parthenogens
OT  - *cytochrome c oxidase subunit I
OT  - *haploweb
OT  - *internal transcribed spacer I
OT  - *reticulate evolution
EDAT- 2016/02/18 06:00
MHDA- 2018/01/11 06:00
CRDT- 2016/02/17 06:00
PHST- 2015/03/24 00:00 [received]
PHST- 2016/02/09 00:00 [accepted]
PHST- 2016/02/17 06:00 [entrez]
PHST- 2016/02/18 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syw016 [pii]
AID - 10.1093/sysbio/syw016 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):508-24. doi: 10.1093/sysbio/syw016. Epub 2016 Feb 14.

PMID- 26880147
OWN - NLM
STAT- MEDLINE
DCOM- 20180105
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 4
DP  - 2016 Jul
TI  - Distinct Processes Drive Diversification in Different Clades of Gesneriaceae.
PG  - 662-84
LID - 10.1093/sysbio/syw012 [doi]
AB  - Using a time-calibrated phylogenetic hypothesis including 768 Gesneriaceae
      species (out of [Formula: see text]3300 species) and more than 29,000 aligned
      bases from 26 gene regions, we test Gesneriaceae for diversification rate shifts 
      and the possible proximal drivers of these shifts: geographic distributions,
      growth forms, and pollination syndromes. Bayesian Analysis of Macroevolutionary
      Mixtures analyses found five significant rate shifts in Beslerieae, core
      Nematanthus, core Columneinae, core Streptocarpus, and Pacific Cyrtandra These
      rate shifts correspond with shifts in diversification rates, as inferred by
      Binary State Speciation and Extinction Model and Geographic State Speciation and 
      Extinction model, associated with hummingbird pollination, epiphytism, unifoliate
      growth, and geographic area. Our results suggest that diversification processes
      are extremely variable across Gesneriaceae clades with different combinations of 
      characters influencing diversification rates in different clades. Diversification
      patterns between New and Old World lineages show dramatic differences, suggesting
      that the processes of diversification in Gesneriaceae are very different in these
      two geographic regions.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Roalson, Eric H
AU  - Roalson EH
AD  - School of Biological Sciences, Washington State University, Pullman, WA 99164,
      USA Molecular Plant Sciences Graduate Program, Washington State University,
      Pullman, WA 99164, USA eric_roalson@wsu.edu.
FAU - Roberts, Wade R
AU  - Roberts WR
AD  - Molecular Plant Sciences Graduate Program, Washington State University, Pullman, 
      WA 99164, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.1br13
PT  - Journal Article
DEP - 20160214
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Biodiversity
MH  - Geography
MH  - Magnoliopsida/*classification/physiology
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Pollination/physiology
OTO - NOTNLM
OT  - *Diversification rates
OT  - *Gesneriaceae
OT  - *Lamiales
OT  - *epiphytism
OT  - *historical biogeography
OT  - *pollination syndrome
EDAT- 2016/02/18 06:00
MHDA- 2018/01/06 06:00
CRDT- 2016/02/17 06:00
PHST- 2015/06/24 00:00 [received]
PHST- 2016/01/02 00:00 [accepted]
PHST- 2016/02/17 06:00 [entrez]
PHST- 2016/02/18 06:00 [pubmed]
PHST- 2018/01/06 06:00 [medline]
AID - syw012 [pii]
AID - 10.1093/sysbio/syw012 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jul;65(4):662-84. doi: 10.1093/sysbio/syw012. Epub 2016 Feb 14.

PMID- 26869489
OWN - NLM
STAT- MEDLINE
DCOM- 20180111
LR  - 20190112
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 6
DP  - 2016 Nov
TI  - Taxonomist's Nightmare ... Evolutionist's Delight : An Integrative Approach
      Resolves Species Limits in Jumping Bristletails Despite Widespread Hybridization 
      and Parthenogenesis.
PG  - 947-974
AB  - Accurate species delimitation is fundamental to biology. Traditionally, species
      were delimited based on morphological characters, sometimes leading to taxonomic 
      uncertainty in morphologically conserved taxa. Recently, multiple taxonomically
      challenging cases have benefited from integrative taxonomy-an approach that
      highlights congruence among different disciplines and invokes evolutionary
      explanations for incongruence, acknowledging that different methods can mirror
      different stages of the speciation continuum. Here, we used a cohesive protocol
      for integrative taxonomy to revise species limits in 20 nominal species and 4
      morphospecies of an ancestrally wingless insect group, the jumping bristletail
      genus Machilis from the European Eastern Alps. Even though morphologically
      conserved, several small-scale endemic species have been described from the
      Eastern Alps based on variation in hypodermal pigmentation patterns-a highly
      questionable character. As valuable as these endemics are for conservation, they 
      have never been verified by alternative methods. Using traditional morphometrics,
      mitochondrial DNA, ribosomal DNA, and amplified fragment-length polymorphism
      markers, we identify six nominal species as taxonomic junior synonyms (Machilis
      alpicola Janetschek, 1953 syn. n. under M. vagans Wygodzinsky, 1941; M. ladensis 
      Janetschek, 1950 syn. n., M. robusta Wygodzinsky, 1941 syn. n., and M. vicina
      Wygodzinsky, 1941 syn. n. under M. inermis Wygodzinsky, 1941; M. aleamaculata
      Wygodzinsky, 1941 syn. n. under M. montana Wygodzinsky, 1941; M. pulchra
      Janetschek, 1950 syn. n. under M. helleri Verhoeff, 1910) and describe two new
      species (Machilis cryptoglacialis sp. n. and Machilis albida sp. n.), one
      uncovered from morphological crypsis and one never sampled before. Building on
      numerous cases of incongruence among data sources, we further shed light on
      complex evolutionary histories including hybrid speciation, historical and recent
      hybridization, and ongoing speciation. We hypothesize that an inherent affinity
      to hybridization, combined with parallel switches to parthenogenesis and repeated
      postglacial colonization events may have boosted endemicity in Eastern Alpine
      Machilis We thus emphasize the importance of integrative taxonomy for rigorous
      species delimitation and its implication for evolutionary research and
      conservation in taxonomically challenging taxa.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Dejaco, Thomas
AU  - Dejaco T
AD  - Molecular Ecology Group, Institute of Ecology, University of Innsbruck,
      Technikerstrasse 25, 6020 Innsbruck, Austria Thomas.Dejaco@gmx.at.
AD  - Museum of Nature South Tyrol, Bindergasse 1, 39100 Bozen/Bolzano, Italy.
FAU - Gassner, Melitta
AU  - Gassner M
AD  - Molecular Ecology Group, Institute of Ecology, University of Innsbruck,
      Technikerstrasse 25, 6020 Innsbruck, Austria.
FAU - Arthofer, Wolfgang
AU  - Arthofer W
AD  - Molecular Ecology Group, Institute of Ecology, University of Innsbruck,
      Technikerstrasse 25, 6020 Innsbruck, Austria.
FAU - Schlick-Steiner, Birgit C
AU  - Schlick-Steiner BC
AD  - Molecular Ecology Group, Institute of Ecology, University of Innsbruck,
      Technikerstrasse 25, 6020 Innsbruck, Austria.
FAU - Steiner, Florian M
AU  - Steiner FM
AD  - Molecular Ecology Group, Institute of Ecology, University of Innsbruck,
      Technikerstrasse 25, 6020 Innsbruck, Austria.
LA  - eng
PT  - Journal Article
DEP - 20160211
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - DNA, Mitochondrial
MH  - Europe, Eastern
MH  - Hybridization, Genetic
MH  - Insecta/anatomy &amp; histology/*classification/genetics
MH  - *Phylogeny
MH  - Species Specificity
PMC - PMC5066060
OTO - NOTNLM
OT  - *Archaeognatha
OT  - *gene tree discordance
OT  - *hybrid speciation
OT  - *hybrid swarm
OT  - *incomplete lineage sorting
OT  - *mitochondrial introgression
OT  - *new species
OT  - *new synonyms
EDAT- 2016/02/13 06:00
MHDA- 2018/01/13 06:00
CRDT- 2016/02/13 06:00
PHST- 2015/07/23 00:00 [received]
PHST- 2015/10/26 00:00 [revised]
PHST- 2016/01/14 00:00 [accepted]
PHST- 2016/02/13 06:00 [pubmed]
PHST- 2018/01/13 06:00 [medline]
PHST- 2016/02/13 06:00 [entrez]
AID - syw003 [pii]
AID - 10.1093/sysbio/syw003 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Nov;65(6):947-974. doi: 10.1093/sysbio/syw003. Epub 2016 Feb 11.

PMID- 26865275
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Bats (Chiroptera: Noctilionoidea) Challenge a Recent Origin of Extant Neotropical
      Diversity.
PG  - 432-48
LID - 10.1093/sysbio/syw011 [doi]
AB  - The mechanisms underlying the high extant biodiversity in the Neotropics have
      been controversial since the 19th century. Support for the influence of
      period-specific changes on diversification often rests on detecting more
      speciation events during a particular period. The timing of speciation events may
      reflect the influence of incomplete taxon sampling, protracted speciation, and
      null processes of lineage accumulation. Here we assess the influence of these
      factors on the timing of speciation with new multilocus data for New World
      noctilionoid bats (Chiroptera: Noctilionoidea). Biogeographic analyses revealed
      the importance of the Neotropics in noctilionoid diversification, and the
      critical role of dispersal. We detected no shift in speciation rate associated
      with the Quaternary or pre-Quaternary periods, and instead found an increase in
      speciation linked to the evolution of the subfamily Stenodermatinae (
      approximately 18 Ma). Simulations modeling constant speciation and extinction
      rates for the phylogeny systematically showed more speciation events in the
      Quaternary. Since recording more divergence events in the Quaternary can result
      from lineage accumulation, the age of extant sister species cannot be interpreted
      as supporting higher speciation rates during this period. Instead, analyzing the 
      factors that influence speciation requires modeling lineage-specific traits and
      environmental, spatial, and ecological drivers of speciation.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Rojas, Danny
AU  - Rojas D
AD  - Department of Biology and Centre for Environmental and Marine Studies, University
      of Aveiro, 3810-193 Aveiro, Portugal; Department of Ecology and Evolution, Stony 
      Brook University, 650 Life Sciences Building Stony Brook, NY 11794, USA;
      rojasmartin.cu@gmail.com.
FAU - Warsi, Omar M
AU  - Warsi OM
AD  - Department of Ecology and Evolution, Stony Brook University, 650 Life Sciences
      Building Stony Brook, NY 11794, USA; Uppsala Biomedicinskt Centrum BMC, Husarg, 3
      751 23 Uppsala, Sweden;
FAU - Davalos, Liliana M
AU  - Davalos LM
AD  - Department of Ecology and Evolution, Stony Brook University, 650 Life Sciences
      Building Stony Brook, NY 11794, USA; Consortium for Inter-Disciplinary
      Environmental Research, School of Marine and Atmospheric Sciences, Stony Brook
      University, 129 Dana Hall, Stony Brook, NY 11794, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.s533p
PT  - Journal Article
DEP - 20160210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2017 Jul 1;66(4):661. PMID: 28204758
MH  - Animals
MH  - *Biodiversity
MH  - Chiroptera/*classification/physiology
MH  - Computer Simulation
MH  - Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Tropical Climate
OTO - NOTNLM
OT  - *Fossils
OT  - *Neotropics
OT  - *Phyllostomidae
OT  - *geographic range evolution
OT  - *speciation
EDAT- 2016/02/13 06:00
MHDA- 2018/01/11 06:00
CRDT- 2016/02/12 06:00
PHST- 2014/08/22 00:00 [received]
PHST- 2016/02/01 00:00 [accepted]
PHST- 2016/02/12 06:00 [entrez]
PHST- 2016/02/13 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syw011 [pii]
AID - 10.1093/sysbio/syw011 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):432-48. doi: 10.1093/sysbio/syw011. Epub 2016 Feb 10.

PMID- 26865274
OWN - NLM
STAT- MEDLINE
DCOM- 20180105
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 4
DP  - 2016 Jul
TI  - Inferring Bounded Evolution in Phenotypic Characters from Phylogenetic
      Comparative Data.
PG  - 651-61
LID - 10.1093/sysbio/syw015 [doi]
AB  - Our understanding of phenotypic evolution over macroevolutionary timescales
      largely relies on the use of stochastic models for the evolution of continuous
      traits over phylogenies. The two most widely used models, Brownian motion and the
      Ornstein-Uhlenbeck (OU) process, differ in that the latter includes constraints
      on the variance that a trait can attain in a clade. The OU model explicitly
      models adaptive evolution toward a trait optimum and has thus been widely used to
      demonstrate the existence of stabilizing selection on a trait. Here we introduce 
      a new model for the evolution of continuous characters on phylogenies: Brownian
      motion between two reflective bounds, or Bounded Brownian Motion (BBM). This
      process also models evolutionary constraints, but of a very different kind. We
      provide analytical expressions for the likelihood of BBM and present a method to 
      calculate the likelihood numerically, as well as the associated R code. Numerical
      simulations show that BBM achieves good performance: parameter estimation is
      generally accurate but more importantly BBM can be very easily discriminated from
      both BM and OU. We then analyze climatic niche evolution in diprotodonts and find
      that BBM best fits this empirical data set, suggesting that the climatic niches
      of diprotodonts are bounded by the climate available in Australia and the
      neighboring islands but probably evolved with little additional constraints. We
      conclude that BBM is a valuable addition to the macroevolutionary toolbox, which 
      should enable researchers to elucidate whether the phenotypic traits they study
      are evolving under hard constraints between bounds.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Boucher, Florian C
AU  - Boucher FC
AD  - Institute of Systematic Botany, University of Zurich, Zurich, Switzerland;
      florian.boucher@systbot.uzh.ch.
FAU - Demery, Vincent
AU  - Demery V
AD  - Laboratoire de Physico-Chimie Theorique, UMR Gulliver 7083, CNRS and
      ESPCI-ParisTech, Paris, France florian.boucher@systbot.uzh.ch.
LA  - eng
SI  - Dryad/10.5061/dryad.k974s
PT  - Journal Article
DEP - 20160210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Australia
MH  - *Biological Evolution
MH  - Marsupialia/classification
MH  - *Models, Biological
MH  - *Phenotype
MH  - *Phylogeny
OTO - NOTNLM
OT  - *BBM; bounds
OT  - *evolutionary constraints
OT  - *macroevolution
OT  - *maximum likelihood estimation
OT  - *phylogenetic comparative data
EDAT- 2016/02/13 06:00
MHDA- 2018/01/06 06:00
CRDT- 2016/02/12 06:00
PHST- 2015/06/15 00:00 [received]
PHST- 2016/01/25 00:00 [accepted]
PHST- 2016/02/12 06:00 [entrez]
PHST- 2016/02/13 06:00 [pubmed]
PHST- 2018/01/06 06:00 [medline]
AID - syw015 [pii]
AID - 10.1093/sysbio/syw015 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jul;65(4):651-61. doi: 10.1093/sysbio/syw015. Epub 2016 Feb 10.

PMID- 26865273
OWN - NLM
STAT- MEDLINE
DCOM- 20180105
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 4
DP  - 2016 Jul
TI  - Analysis of a Rapid Evolutionary Radiation Using Ultraconserved Elements:
      Evidence for a Bias in Some Multispecies Coalescent Methods.
PG  - 612-27
LID - 10.1093/sysbio/syw014 [doi]
AB  - Rapid evolutionary radiations are expected to require large amounts of sequence
      data to resolve. To resolve these types of relationships many systematists
      believe that it will be necessary to collect data by next-generation sequencing
      (NGS) and use multispecies coalescent ("species tree") methods. Ultraconserved
      element (UCE) sequence capture is becoming a popular method to leverage the high 
      throughput of NGS to address problems in vertebrate phylogenetics. Here we
      examine the performance of UCE data for gallopheasants (true pheasants and
      allies), a clade that underwent a rapid radiation 10-15 Ma. Relationships among
      gallopheasant genera have been difficult to establish. We used this rapid
      radiation to assess the performance of species tree methods, using approximately 
      600 kilobases of DNA sequence data from approximately 1500 UCEs. We also
      integrated information from traditional markers (nuclear intron data from 15 loci
      and three mitochondrial gene regions). Species tree methods exhibited troubling
      behavior. Two methods [Maximum Pseudolikelihood for Estimating Species Trees
      (MP-EST) and Accurate Species TRee ALgorithm (ASTRAL)] appeared to perform
      optimally when the set of input gene trees was limited to the most variable UCEs,
      though ASTRAL appeared to be more robust than MP-EST to input trees generated
      using less variable UCEs. In contrast, the rooted triplet consensus method
      implemented in Triplec performed better when the largest set of input gene trees 
      was used. We also found that all three species tree methods exhibited a
      surprising degree of dependence on the program used to estimate input gene trees,
      suggesting that the details of likelihood calculations (e.g., numerical
      optimization) are important for loci with limited phylogenetic information. As an
      alternative to summary species tree methods we explored the performance of
      SuperMatrix Rooted Triple - Maximum Likelihood (SMRT-ML), a concatenation method 
      that is consistent even when gene trees exhibit topological differences due to
      the multispecies coalescent. We found that SMRT-ML performed well for UCE data.
      Our results suggest that UCE data have excellent prospects for the resolution of 
      difficult evolutionary radiations, though specific attention may need to be given
      to the details of the methods used to estimate species trees.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Meiklejohn, Kelly A
AU  - Meiklejohn KA
AD  - Department of Biology, University of Florida, Gainesville, FL;
FAU - Faircloth, Brant C
AU  - Faircloth BC
AD  - Department of Biological Sciences and Museum of Natural Science, Louisiana State 
      University, Baton Rouge, LA;
FAU - Glenn, Travis C
AU  - Glenn TC
AD  - Department of Environmental Health Science, University of Georgia, Athens, GA;
FAU - Kimball, Rebecca T
AU  - Kimball RT
AD  - Department of Biology, University of Florida, Gainesville, FL; Genetics
      Institute, University of Florida, Gainesville, FL.
FAU - Braun, Edward L
AU  - Braun EL
AD  - Department of Biology, University of Florida, Gainesville, FL; Genetics
      Institute, University of Florida, Gainesville, FL.
LA  - eng
SI  - Dryad/10.5061/dryad.p1m52
PT  - Journal Article
DEP - 20160210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - Classification/*methods
MH  - High-Throughput Nucleotide Sequencing
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Probability
OTO - NOTNLM
OT  - *Galliformes, gene tree estimation error
OT  - *Phasianidae
OT  - *Triplec
OT  - *polytomy
OT  - *supermatrix rooted triplets
OT  - *total evidence
EDAT- 2016/02/13 06:00
MHDA- 2018/01/06 06:00
CRDT- 2016/02/12 06:00
PHST- 2015/05/25 00:00 [received]
PHST- 2016/01/25 00:00 [accepted]
PHST- 2016/02/12 06:00 [entrez]
PHST- 2016/02/13 06:00 [pubmed]
PHST- 2018/01/06 06:00 [medline]
AID - syw014 [pii]
AID - 10.1093/sysbio/syw014 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jul;65(4):612-27. doi: 10.1093/sysbio/syw014. Epub 2016 Feb 10.

PMID- 26821913
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20190110
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Computational Performance and Statistical Accuracy of *BEAST and Comparisons with
      Other Methods.
PG  - 381-96
LID - 10.1093/sysbio/syv118 [doi]
AB  - Under the multispecies coalescent model of molecular evolution, gene trees have
      independent evolutionary histories within a shared species tree. In comparison,
      supermatrix concatenation methods assume that gene trees share a single common
      genealogical history, thereby equating gene coalescence with species divergence. 
      The multispecies coalescent is supported by previous studies which found that its
      predicted distributions fit empirical data, and that concatenation is not a
      consistent estimator of the species tree. *BEAST, a fully Bayesian implementation
      of the multispecies coalescent, is popular but computationally intensive, so the 
      increasing size of phylogenetic data sets is both a computational challenge and
      an opportunity for better systematics. Using simulation studies, we characterize 
      the scaling behavior of *BEAST, and enable quantitative prediction of the impact 
      increasing the number of loci has on both computational performance and
      statistical accuracy. Follow-up simulations over a wide range of parameters show 
      that the statistical performance of *BEAST relative to concatenation improves
      both as branch length is reduced and as the number of loci is increased. Finally,
      using simulations based on estimated parameters from two phylogenomic data sets, 
      we compare the performance of a range of species tree and concatenation methods
      to show that using *BEAST with tens of loci can be preferable to using
      concatenation with thousands of loci. Our results provide insight into the
      practicalities of Bayesian species tree estimation, the number of loci required
      to obtain a given level of accuracy and the situations in which supermatrix or
      summary methods will be outperformed by the fully Bayesian multispecies
      coalescent.
CI  - (c) The Author(s) 2016. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - Ogilvie, Huw A
AU  - Ogilvie HA
AD  - Evolution, Ecology and Genetics, Research School of Biology, The Australian
      National University, Canberra, Australia;
FAU - Heled, Joseph
AU  - Heled J
AD  - Department of Computer Science, University of Auckland, Auckland, New Zealand;
      Allan Wilson Centre for Molecular Ecology and Evolution, University of Auckland, 
      Auckland, New Zealand;
FAU - Xie, Dong
AU  - Xie D
AD  - Department of Computer Science, University of Auckland, Auckland, New Zealand;
      Allan Wilson Centre for Molecular Ecology and Evolution, University of Auckland, 
      Auckland, New Zealand;
FAU - Drummond, Alexei J
AU  - Drummond AJ
AD  - Department of Computer Science, University of Auckland, Auckland, New Zealand;
      Allan Wilson Centre for Molecular Ecology and Evolution, University of Auckland, 
      Auckland, New Zealand;
LA  - eng
SI  - Dryad/10.5061/dryad.02tf9
PT  - Journal Article
DEP - 20160128
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Biological Evolution
MH  - Classification/*methods
MH  - Data Interpretation, Statistical
MH  - Evolution, Molecular
MH  - Models, Genetic
MH  - *Phylogeny
MH  - *Software
PMC - PMC4851174
OTO - NOTNLM
OT  - *Bayesian phylogenetics
OT  - *Concatenation
OT  - *Gene tree
OT  - *Multispecies coalescent
OT  - *Phylogenomics
OT  - *Species tree
OT  - *Supermatrix
EDAT- 2016/01/30 06:00
MHDA- 2018/01/11 06:00
CRDT- 2016/01/30 06:00
PHST- 2015/06/09 00:00 [received]
PHST- 2015/12/07 00:00 [accepted]
PHST- 2016/01/30 06:00 [entrez]
PHST- 2016/01/30 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syv118 [pii]
AID - 10.1093/sysbio/syv118 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):381-96. doi: 10.1093/sysbio/syv118. Epub 2016 Jan 28.

PMID- 26821912
OWN - NLM
STAT- MEDLINE
DCOM- 20180105
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 4
DP  - 2016 Jul
TI  - Comparison of Target-Capture and Restriction-Site Associated DNA Sequencing for
      Phylogenomics: A Test in Cardinalid Tanagers (Aves, Genus: Piranga).
PG  - 640-50
LID - 10.1093/sysbio/syw005 [doi]
AB  - Restriction-site associated DNA sequencing (RAD-seq) and target capture of
      specific genomic regions, such as ultraconserved elements (UCEs), are emerging as
      two of the most popular methods for phylogenomics using reduced-representation
      genomic data sets. These two methods were designed to target different
      evolutionary timescales: RAD-seq was designed for population-genomic level
      questions and UCEs for deeper phylogenetics. The utility of both data sets to
      infer phylogenies across a variety of taxonomic levels has not been adequately
      compared within the same taxonomic system. Additionally, the effects of
      uninformative gene trees on species tree analyses (for target capture data) have 
      not been explored. Here, we utilize RAD-seq and UCE data to infer a phylogeny of 
      the bird genus Piranga The group has a range of divergence dates (0.5-6 myr),
      contains 11 recognized species, and lacks a resolved phylogeny. We compared two
      species tree methods for the RAD-seq data and six species tree methods for the
      UCE data. Additionally, in the UCE data, we analyzed a complete matrix as well as
      data sets with only highly informative loci. A complete matrix of 189 UCE loci
      with 10 or more parsimony informative (PI) sites, and an approximately 80%
      complete matrix of 1128 PI single-nucleotide polymorphisms (SNPs) (from RAD-seq) 
      yield the same fully resolved phylogeny of Piranga We inferred non-monophyletic
      relationships of Piranga lutea individuals, with all other a priori species
      identified as monophyletic. Finally, we found that species tree analyses that
      included predominantly uninformative gene trees provided strong support for
      different topologies, with consistent phylogenetic results when limiting species 
      tree analyses to highly informative loci or only using less informative loci with
      concatenation or methods meant for SNPs alone.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Manthey, Joseph D
AU  - Manthey JD
AD  - Biodiversity Institute and Department of Ecology and Evolutionary Biology,
      University of Kansas, Lawrence, KS 66045, USA and.
FAU - Campillo, Luke C
AU  - Campillo LC
AD  - Biodiversity Institute and Department of Ecology and Evolutionary Biology,
      University of Kansas, Lawrence, KS 66045, USA and.
FAU - Burns, Kevin J
AU  - Burns KJ
AD  - Department of Biology, San Diego State University, San Diego, CA 92182, USA.
FAU - Moyle, Robert G
AU  - Moyle RG
AD  - Biodiversity Institute and Department of Ecology and Evolutionary Biology,
      University of Kansas, Lawrence, KS 66045, USA and.
LA  - eng
SI  - Dryad/10.5061/dryad.j5n06
GR  - P20 GM103638/GM/NIGMS NIH HHS/United States
PT  - Journal Article
DEP - 20160128
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Biological Evolution
MH  - Birds/*classification/*genetics
MH  - Classification/*methods
MH  - *Genomics
MH  - *Phylogeny
MH  - Sequence Analysis, DNA/*standards
PMC - PMC5841389
OTO - NOTNLM
OT  - *Birds
OT  - *Piranga
OT  - *phylogenomics
OT  - *restriction-site associated DNA sequencing
OT  - *species tree methods
OT  - *target capture
OT  - *ultraconserved elements
EDAT- 2016/01/30 06:00
MHDA- 2018/01/06 06:00
CRDT- 2016/01/30 06:00
PHST- 2015/06/09 00:00 [received]
PHST- 2016/01/22 00:00 [accepted]
PHST- 2016/01/30 06:00 [entrez]
PHST- 2016/01/30 06:00 [pubmed]
PHST- 2018/01/06 06:00 [medline]
AID - syw005 [pii]
AID - 10.1093/sysbio/syw005 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jul;65(4):640-50. doi: 10.1093/sysbio/syw005. Epub 2016 Jan 28.

PMID- 26797695
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Rarity and Incomplete Sampling in DNA-Based Species Delimitation.
PG  - 478-94
LID - 10.1093/sysbio/syw002 [doi]
AB  - DNA-based species delimitation may be compromised by limited sampling effort and 
      species rarity, including "singleton" representatives of species, which hampers
      estimates of intra- versus interspecies evolutionary processes. In a case study
      of southern African chafers (beetles in the family Scarabaeidae), many species
      and subclades were poorly represented and 48.5% of species were singletons. Using
      cox1 sequences from &gt;500 specimens and approximately 100 species, the Generalized
      Mixed Yule Coalescent (GMYC) analysis as well as various other approaches for
      DNA-based species delimitation (Automatic Barcode Gap Discovery (ABGD), Poisson
      tree processes (PTP), Species Identifier, Statistical Parsimony), frequently
      produced poor results if analyzing a narrow target group only, but the
      performance improved when several subclades were combined. Hence, low sampling
      may be compensated for by "clade addition" of lineages outside of the focal
      group. Similar findings were obtained in reanalysis of published data sets of
      taxonomically poorly known species assemblages of insects from Madagascar. The
      low performance of undersampled trees is not due to high proportions of
      singletons per se, as shown in simulations (with 13%, 40% and 52% singletons).
      However, the GMYC method was highly sensitive to variable effective population
      size ([Formula: see text]), which was exacerbated by variable species abundances 
      in the simulations. Hence, low sampling success and rarity of species affect the 
      power of the GMYC method only if they reflect great differences in [Formula: see 
      text] among species. Potential negative effects of skewed species abundances and 
      prevalence of singletons are ultimately an issue about the variation in [Formula:
      see text] and the degree to which this is correlated with the census population
      size and sampling success. Clade addition beyond a limited study group can
      overcome poor sampling for the GMYC method in particular under variable [Formula:
      see text] This effect was less pronounced for methods of species delimitation not
      based on coalescent models.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Ahrens, Dirk
AU  - Ahrens D
AD  - Centre of Taxonomy and Evolutionary Research, Zoologisches Forschungsmuseum
      Alexander Koenig Bonn, Adenauerallee 160, 53113 Bonn, Germany; Department of Life
      Sciences, Natural History Museum, London SW7 5BD, UK; ahrens.dirk_col@gmx.de.
FAU - Fujisawa, Tomochika
AU  - Fujisawa T
AD  - Department of Zoology, Graduate School of Science, Kyoto University, Sakyo, Kyoto
      606 8502, Japan; Department of Life Sciences, Silwood Park Campus, Imperial
      College London, Ascot SL7 5PY, UK.
FAU - Krammer, Hans-Joachim
AU  - Krammer HJ
AD  - Centre of Taxonomy and Evolutionary Research, Zoologisches Forschungsmuseum
      Alexander Koenig Bonn, Adenauerallee 160, 53113 Bonn, Germany;
FAU - Eberle, Jonas
AU  - Eberle J
AD  - Centre of Taxonomy and Evolutionary Research, Zoologisches Forschungsmuseum
      Alexander Koenig Bonn, Adenauerallee 160, 53113 Bonn, Germany;
FAU - Fabrizi, Silvia
AU  - Fabrizi S
AD  - Centre of Taxonomy and Evolutionary Research, Zoologisches Forschungsmuseum
      Alexander Koenig Bonn, Adenauerallee 160, 53113 Bonn, Germany; Department of Life
      Sciences, Natural History Museum, London SW7 5BD, UK;
FAU - Vogler, Alfried P
AU  - Vogler AP
AD  - Department of Life Sciences, Natural History Museum, London SW7 5BD, UK;
      Department of Life Sciences, Silwood Park Campus, Imperial College London, Ascot 
      SL7 5PY, UK.
LA  - eng
SI  - Dryad/10.5061/dryad.h6q11
PT  - Journal Article
DEP - 20160121
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 9007-49-2 (DNA)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Coleoptera/classification/genetics
MH  - Computer Simulation
MH  - DNA/genetics
MH  - Electron Transport Complex IV/genetics
MH  - Madagascar
MH  - *Phylogeny
MH  - Population Density
MH  - Sample Size
OTO - NOTNLM
OT  - *Coleoptera
OT  - *Sericini
OT  - *effective population size
OT  - *singletons
OT  - *southern Africa
EDAT- 2016/01/23 06:00
MHDA- 2018/01/11 06:00
CRDT- 2016/01/23 06:00
PHST- 2015/03/12 00:00 [received]
PHST- 2016/01/13 00:00 [accepted]
PHST- 2016/01/23 06:00 [entrez]
PHST- 2016/01/23 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syw002 [pii]
AID - 10.1093/sysbio/syw002 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):478-94. doi: 10.1093/sysbio/syw002. Epub 2016 Jan 21.

PMID- 26738927
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20190226
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Detecting the Anomaly Zone in Species Trees and Evidence for a Misleading Signal 
      in Higher-Level Skink Phylogeny (Squamata: Scincidae).
PG  - 465-77
LID - 10.1093/sysbio/syw001 [doi]
AB  - The anomaly zone, defined by the presence of gene tree topologies that are more
      probable than the true species tree, presents a major challenge to the accurate
      resolution of many parts of the Tree of Life. This discrepancy can result from
      consecutive rapid speciation events in the species tree. Similar to the problem
      of long-branch attraction, including more data via loci concatenation will only
      reinforce the support for the incorrect species tree. Empirical phylogenetic
      studies often employ coalescent-based species tree methods to avoid the anomaly
      zone, but to this point these studies have not had a method for providing any
      direct evidence that the species tree is actually in the anomaly zone. In this
      study, we use 16 species of lizards in the family Scincidae to investigate
      whether nodes that are difficult to resolve place the species tree within the
      anomaly zone. We analyze new phylogenomic data (429 loci), using both
      concatenation and coalescent-based species tree estimation, to locate conflicting
      topological signal. We then use the unifying principle of the anomaly zone,
      together with estimates of ancestral population sizes and species persistence
      times, to determine whether the observed phylogenetic conflict is a result of the
      anomaly zone. We identify at least three regions of the Scincidae phylogeny that 
      provide demographic signatures consistent with the anomaly zone, and this new
      information helps reconcile the phylogenetic conflict in previously published
      studies on these lizards. The anomaly zone presents a real problem in
      phylogenetics, and our new framework for identifying anomalous relationships will
      help empiricists leverage their resources appropriately for investigating and
      overcoming this challenge.
CI  - (c) The Author(s) 2016. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Linkem, Charles W
AU  - Linkem CW
AD  - Department of Biology, University of Washington, Seattle WA; cwlinkem@gmail.com.
FAU - Minin, Vladimir N
AU  - Minin VN
AD  - Department of Biology, University of Washington, Seattle WA; Department of
      Statistics, University of Washington, Seattle WA;
FAU - Leache, Adam D
AU  - Leache AD
AD  - Department of Biology, University of Washington, Seattle WA; Burke Museum of
      Natural History and Culture, University of Washington, Seattle, WA, 98195, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.sf6s9
PT  - Journal Article
DEP - 20160106
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Genome/genetics
MH  - Lizards/*classification/*genetics
MH  - *Models, Genetic
MH  - *Phylogeny
PMC - PMC6383586
OTO - NOTNLM
OT  - *Anomalous gene trees
OT  - *IDBA
OT  - *UCE
OT  - *incomplete lineage sorting
OT  - *phylogenetics
OT  - *probes
OT  - *sequence capture
OT  - *ultraconserved elements
EDAT- 2016/01/08 06:00
MHDA- 2018/01/11 06:00
CRDT- 2016/01/08 06:00
PHST- 2014/11/26 00:00 [received]
PHST- 2015/12/29 00:00 [accepted]
PHST- 2016/01/08 06:00 [entrez]
PHST- 2016/01/08 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syw001 [pii]
AID - 10.1093/sysbio/syw001 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):465-77. doi: 10.1093/sysbio/syw001. Epub 2016 Jan 6.

PMID- 28334079
OWN - NLM
STAT- MEDLINE
DCOM- 20180130
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 5
DP  - 2017 Sep 1
TI  - How Do Cold-Adapted Plants Respond to Climatic Cycles? Interglacial Expansion
      Explains Current Distribution and Genomic Diversity in Primula farinosa L.
PG  - 715-736
LID - 10.1093/sysbio/syw114 [doi]
AB  - Understanding the effects of past climatic fluctuations on the distribution and
      population-size dynamics of cold-adapted species is essential for predicting
      their responses to ongoing global climate change. In spite of the heterogeneity
      of cold-adapted species, two main contrasting hypotheses have been proposed to
      explain their responses to Late Quaternary glacial cycles, namely, the
      interglacial contraction versus the interglacial expansion hypotheses. Here, we
      use the cold-adapted plant Primula farinosa to test two demographic models under 
      each of the two alternative hypotheses and a fifth, null model. We first
      approximate the time and extent of demographic contractions and expansions during
      the Late Quaternary by projecting species distribution models across the last 72 
      ka. We also generate genome-wide sequence data using a Reduced Representation
      Library approach to reconstruct the spatial structure, genetic diversity, and
      phylogenetic relationships of lineages within P. farinosa. Finally, by
      integrating the results of climatic and genomic analyses in an Approximate
      Bayesian Computation framework, we propose the most likely model for the extent
      and direction of population-size changes in $P$. farinosa through the Late
      Quaternary. Our results support the interglacial expansion of $P$. farinosa,
      differing from the prevailing paradigm that the observed distribution of
      cold-adapted species currently fragmented in high altitude and latitude regions
      reflects the consequences of postglacial contraction processes.
CI  - (c) The Author(s) 2017. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Theodoridis, Spyros
AU  - Theodoridis S
AD  - Department of Systematic and Evolutionary Botany, University of Zurich, CH-8008
      Zurich, Switzerland.
AD  - Zurich-Basel Plant Science Center, CH-8092 Zurich, Switzerland.
FAU - Randin, Christophe
AU  - Randin C
AD  - Institute of Botany, University of Basel, CH-4056 Basel, Switzerland.
AD  - Department of Ecology &amp; Evolution, University of Lausanne, CH-1015 Lausanne,
      Switzerland.
FAU - Szovenyi, Peter
AU  - Szovenyi P
AD  - Department of Systematic and Evolutionary Botany, University of Zurich, CH-8008
      Zurich, Switzerland.
FAU - Boucher, Florian C
AU  - Boucher FC
AD  - Department of Systematic and Evolutionary Botany, University of Zurich, CH-8008
      Zurich, Switzerland.
AD  - Department of Botany and Zoology, University of Stellenbosch, 7602 Matieland,
      South Africa.
FAU - Patsiou, Theofania S
AU  - Patsiou TS
AD  - Department of Systematic and Evolutionary Botany, University of Zurich, CH-8008
      Zurich, Switzerland.
AD  - Zurich-Basel Plant Science Center, CH-8092 Zurich, Switzerland.
AD  - Institute of Botany, University of Basel, CH-4056 Basel, Switzerland.
FAU - Conti, Elena
AU  - Conti E
AD  - Department of Systematic and Evolutionary Botany, University of Zurich, CH-8008
      Zurich, Switzerland.
AD  - Zurich-Basel Plant Science Center, CH-8092 Zurich, Switzerland.
LA  - eng
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Physiological/*physiology
MH  - Bayes Theorem
MH  - *Climate
MH  - *Cold Temperature
MH  - *Genetic Variation
MH  - Genome, Plant/genetics
MH  - *Ice Cover
MH  - *Phylogeny
MH  - Primula/*genetics/physiology
OTO - NOTNLM
OT  - *Approximate Bayesian computation
OT  - *Late Quaternary glacial cycles
OT  - *Reduced Representation Library
OT  - *climate change
OT  - *hindcasting
OT  - *paleoclimate
OT  - *species distribution models
EDAT- 2016/01/01 00:00
MHDA- 2018/01/31 06:00
CRDT- 2017/03/24 06:00
PHST- 2016/03/08 00:00 [received]
PHST- 2016/12/14 00:00 [accepted]
PHST- 2016/01/01 00:00 [pubmed]
PHST- 2018/01/31 06:00 [medline]
PHST- 2017/03/24 06:00 [entrez]
AID - 2726789 [pii]
AID - 10.1093/sysbio/syw114 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Sep 1;66(5):715-736. doi: 10.1093/sysbio/syw114.

PMID- 26715586
OWN - NLM
STAT- MEDLINE
DCOM- 20180105
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 4
DP  - 2016 Jul
TI  - Modeling Character Change Heterogeneity in Phylogenetic Analyses of Morphology
      through the Use of Priors.
PG  - 602-11
LID - 10.1093/sysbio/syv122 [doi]
AB  - The Mk model was developed for estimating phylogenetic trees from discrete
      morphological data, whether for living or fossil taxa. Like any model, the Mk
      model makes a number of assumptions. One assumption is that transitions between
      character states are symmetric (i.e., the probability of changing from 0 to 1 is 
      the same as 1 to 0). However, some characters in a data matrix may not satisfy
      this assumption. Here, we test methods for relaxing this assumption in a Bayesian
      context. Using empirical data sets, we perform model fitting to illustrate cases 
      in which modeling asymmetric transition rates among characters is preferable to
      the standard Mk model. We use simulated data sets to demonstrate that choosing
      the best-fit model of transition-state symmetry can improve model fit and
      phylogenetic estimation.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Wright, April M
AU  - Wright AM
AD  - Department of Integrative Biology, University of Texas at Austin, Austin, TX
      78712, USA; wright.aprilm@gmail.com.
FAU - Lloyd, Graeme T
AU  - Lloyd GT
AD  - Department of Biological Sciences, Macquarie University, NSW 2109, Australia.
FAU - Hillis, David M
AU  - Hillis DM
AD  - Department of Integrative Biology, University of Texas at Austin, Austin, TX
      78712, USA;
LA  - eng
SI  - Dryad/10.5061/dryad.sb8h1
PT  - Journal Article
DEP - 20151228
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Fossils
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Probability
OTO - NOTNLM
OT  - *Bayesian estimation
OT  - *morphology
OT  - *paleontology
OT  - *phylogeny
OT  - *priors
EDAT- 2015/12/31 06:00
MHDA- 2018/01/06 06:00
CRDT- 2015/12/31 06:00
PHST- 2015/04/30 00:00 [received]
PHST- 2015/12/15 00:00 [accepted]
PHST- 2015/12/31 06:00 [entrez]
PHST- 2015/12/31 06:00 [pubmed]
PHST- 2018/01/06 06:00 [medline]
AID - syv122 [pii]
AID - 10.1093/sysbio/syv122 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jul;65(4):602-11. doi: 10.1093/sysbio/syv122. Epub 2015 Dec 28.

PMID- 26715585
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Machine Learning Biogeographic Processes from Biotic Patterns: A New
      Trait-Dependent Dispersal and Diversification Model with Model Choice By
      Simulation-Trained Discriminant Analysis.
PG  - 525-45
LID - 10.1093/sysbio/syv121 [doi]
AB  - Current statistical biogeographical analysis methods are limited in the ways
      ecology can be related to the processes of diversification and geographical range
      evolution, requiring conflation of geography and ecology, and/or assuming
      ecologies that are uniform across all lineages and invariant in time. This
      precludes the possibility of studying a broad class of macroevolutionary
      biogeographical theories that relate geographical and species histories through
      lineage-specific ecological and evolutionary dynamics, such as taxon cycle
      theory. Here we present a new model that generates phylogenies under a complex of
      superpositioned geographical range evolution, trait evolution, and
      diversification processes that can communicate with each other. We present a
      likelihood-free method of inference under our model using discriminant analysis
      of principal components of summary statistics calculated on phylogenies, with the
      discriminant functions trained on data generated by simulations under our model. 
      This approach of model selection by classification of empirical data with respect
      to data generated under training models is shown to be efficient, robust, and
      performs well over a broad range of parameter space defined by the relative rates
      of dispersal, trait evolution, and diversification processes. We apply our method
      to a case study of the taxon cycle, that is testing for habitat and trophic level
      constraints in the dispersal regimes of the Wallacean avifaunal radiation.
CI  - (c)The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Sukumaran, Jeet
AU  - Sukumaran J
AD  - University of Michigan, Ann Arbor, MI 48109-1079, USA; jeetsukumaran@gmail.com.
FAU - Economo, Evan P
AU  - Economo EP
AD  - University of Michigan, Ann Arbor, MI 48109-1079, USA; Okinawa Institute of
      Science and Technology Graduate University, 1919-1 Tancha, Onna-son,
      Kunigami-gun, Okinawa, Japan 904-0495.
FAU - Lacey Knowles, L
AU  - Lacey Knowles L
AD  - University of Michigan, Ann Arbor, MI 48109-1079, USA;
LA  - eng
SI  - Dryad/10.5061/dryad.s43f8
PT  - Journal Article
DEP - 20151228
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - *Discriminant Analysis
MH  - *Machine Learning
MH  - *Models, Biological
MH  - Phylogeny
MH  - Phylogeography/*methods
OTO - NOTNLM
OT  - *biogeography
OT  - *island radiation
OT  - *machine learning
OT  - *macroevolution
OT  - *model choice
OT  - *phylogenetics
OT  - *taxon cycle
EDAT- 2015/12/31 06:00
MHDA- 2018/01/11 06:00
CRDT- 2015/12/31 06:00
PHST- 2015/04/17 00:00 [received]
PHST- 2015/12/17 00:00 [accepted]
PHST- 2015/12/31 06:00 [entrez]
PHST- 2015/12/31 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syv121 [pii]
AID - 10.1093/sysbio/syv121 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):525-45. doi: 10.1093/sysbio/syv121. Epub 2015 Dec 28.

PMID- 26683588
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Late Glacial Demographic Expansion Motivates a Clock Overhaul for Population
      Genetics.
PG  - 449-64
LID - 10.1093/sysbio/syv120 [doi]
AB  - The molecular clock hypothesis is fundamental in evolutionary biology as by
      assuming constancy of the molecular rate it provides a timeframe for evolution.
      However, increasing evidence shows time dependence of inferred molecular rates
      with inflated values obtained using recent calibrations. As recent demographic
      calibrations are virtually non-existent in most species, older phylogenetic
      calibration points (&gt;1 Ma) are commonly used, which overestimate demographic
      parameters. To obtain more reliable rates of molecular evolution for population
      studies, I propose the calibration of demographic transition (CDT) method, which 
      uses the timing of climatic changes over the late glacial warming period to
      calibrate expansions in various species. Simulation approaches and empirical data
      sets from a diversity of species (from mollusk to humans) confirm that, when
      compared with other genealogy-based calibration methods, the CDT provides a
      robust and broadly applicable clock for population genetics. The resulting CDT
      rates of molecular evolution also confirm rate heterogeneity over time and among 
      taxa. Comparisons of expansion dates with ecological evidence confirm the
      inaccuracy of phylogenetically derived divergence rates when dating
      population-level events. The CDT method opens opportunities for addressing issues
      such as demographic responses to past climate change and the origin of rate
      heterogeneity related to taxa, genes, time, and genetic information content.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Hoareau, Thierry B
AU  - Hoareau TB
AD  - Molecular Ecology and Evolution Programme, Department of Genetics, University of 
      Pretoria, Private bag X20, Hatfield, Pretoria 0028, South Africa
      thoareau@gmail.com.
LA  - eng
SI  - Dryad/10.5061/dryad.3q24t
PT  - Journal Article
DEP - 20151217
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Climate Change
MH  - *Evolution, Molecular
MH  - Genetics, Population/*methods
MH  - Phylogeny
MH  - Population Dynamics
MH  - Time Factors
OTO - NOTNLM
OT  - *Bayesian skyline plots
OT  - *calibration of demographic transition
OT  - *expansion dating
OT  - *molecular calibration
OT  - *molecular rate
OT  - *time dependency
OT  - *two-epoch model
EDAT- 2015/12/20 06:00
MHDA- 2018/01/11 06:00
CRDT- 2015/12/20 06:00
PHST- 2014/08/24 00:00 [received]
PHST- 2015/12/10 00:00 [accepted]
PHST- 2015/12/20 06:00 [entrez]
PHST- 2015/12/20 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syv120 [pii]
AID - 10.1093/sysbio/syv120 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):449-64. doi: 10.1093/sysbio/syv120. Epub 2015 Dec 17.

PMID- 26681696
OWN - NLM
STAT- MEDLINE
DCOM- 20180105
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 4
DP  - 2016 Jul
TI  - The Species versus Subspecies Conundrum: Quantitative Delimitation from
      Integrating Multiple Data Types within a Single Bayesian Approach in Hercules
      Beetles.
PG  - 685-99
LID - 10.1093/sysbio/syv119 [doi]
AB  - With the recent attention and focus on quantitative methods for species
      delimitation, an overlooked but equally important issue regards what has actually
      been delimited. This study investigates the apparent arbitrariness of some
      taxonomic distinctions, and in particular how species and subspecies are
      assigned. Specifically, we use a recently developed Bayesian model-based approach
      to show that in the Hercules beetles (genus Dynastes) there is no statistical
      difference in the probability that putative taxa represent different species,
      irrespective of whether they were given species or subspecies designations. By
      considering multiple data types, as opposed to relying exclusively on genetic
      data alone, we also show that both previously recognized species and subspecies
      represent a variety of points along the speciation spectrum (i.e., previously
      recognized species are not systematically further along the continuum than
      subspecies). For example, based on evolutionary models of divergence, some taxa
      are statistically distinguishable on more than one axis of differentiation (e.g.,
      along both phenotypic and genetic dimensions), whereas other taxa can only be
      delimited statistically from a single data type. Because both phenotypic and
      genetic data are analyzed in a common Bayesian framework, our study provides a
      framework for investigating whether disagreements in species boundaries among
      data types reflect (i) actual discordance with the actual history of lineage
      splitting, or instead (ii) differences among data types in the amount of time
      required for differentiation to become apparent among the delimited taxa. We
      discuss what the answers to these questions imply about what characters are used 
      to delimit species, as well as the diverse processes involved in the origin and
      maintenance of species boundaries. With this in mind, we then reflect more
      generally on how quantitative methods for species delimitation are used to assign
      taxonomic status.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Huang, Jen-Pan
AU  - Huang JP
AD  - Department of Ecology and Evolutionary Biology, 1109 Geddes Avenue, Museum of
      Zoology, University of Michigan, Ann Arbor, MI 48109-1079, USA.
FAU - Knowles, L Lacey
AU  - Knowles LL
AD  - Department of Ecology and Evolutionary Biology, 1109 Geddes Avenue, Museum of
      Zoology, University of Michigan, Ann Arbor, MI 48109-1079, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.8p6m0
PT  - Journal Article
DEP - 20151216
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Coleoptera/*classification
MH  - Data Interpretation, Statistical
MH  - *Phylogeny
MH  - Species Specificity
OTO - NOTNLM
OT  - *Assignment
OT  - *iBPP
OT  - *niche conservatism
OT  - *species delimitation
EDAT- 2015/12/19 06:00
MHDA- 2018/01/06 06:00
CRDT- 2015/12/19 06:00
PHST- 2015/06/30 00:00 [received]
PHST- 2015/12/10 00:00 [accepted]
PHST- 2015/12/19 06:00 [entrez]
PHST- 2015/12/19 06:00 [pubmed]
PHST- 2018/01/06 06:00 [medline]
AID - syv119 [pii]
AID - 10.1093/sysbio/syv119 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jul;65(4):685-99. doi: 10.1093/sysbio/syv119. Epub 2015 Dec 16.

PMID- 26658901
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Characterizing and Comparing Phylogenies from their Laplacian Spectrum.
PG  - 495-507
LID - 10.1093/sysbio/syv116 [doi]
AB  - Phylogenetic trees are central to many areas of biology, ranging from population 
      genetics and epidemiology to microbiology, ecology, and macroevolution. The
      ability to summarize properties of trees, compare different trees, and identify
      distinct modes of division within trees is essential to all these research areas.
      But despite wide-ranging applications, there currently exists no common,
      comprehensive framework for such analyses. Here we present a graph-theoretical
      approach that provides such a framework. We show how to construct the spectral
      density profile of a phylogenetic tree from its Laplacian graph. Using
      ultrametric simulated trees as well as non-ultrametric empirical trees, we
      demonstrate that the spectral density successfully identifies various properties 
      of the trees and clusters them into meaningful groups. Finally, we illustrate how
      the eigengap can identify modes of division within a given tree. As phylogenetic 
      data continue to accumulate and to be integrated into various areas of the life
      sciences, we expect that this spectral graph-theoretical framework to
      phylogenetics will have powerful and long-lasting applications.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Lewitus, Eric
AU  - Lewitus E
AD  - Institut de Biologie (IBENS), Ecole Normale Superieure, Paris, France;
      lewitus@biologie.ens.fr.
FAU - Morlon, Helene
AU  - Morlon H
AD  - Institut de Biologie (IBENS), Ecole Normale Superieure, Paris, France;
LA  - eng
PT  - Journal Article
DEP - 20151212
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2016 Sep;65(5):944. PMID: 27444423
MH  - Biological Science Disciplines/*methods
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Ecology
MH  - Genetics, Population
MH  - *Phylogeny
OTO - NOTNLM
OT  - *Laplacian
OT  - *biodiversity
OT  - *diversification
OT  - *graph theory
OT  - *influenza
OT  - *macroevolution
OT  - *microbiology
OT  - *phylodynamics
OT  - *phylogenetics
EDAT- 2015/12/15 06:00
MHDA- 2018/01/11 06:00
CRDT- 2015/12/15 06:00
PHST- 2015/03/23 00:00 [received]
PHST- 2015/12/04 00:00 [accepted]
PHST- 2015/12/15 06:00 [entrez]
PHST- 2015/12/15 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syv116 [pii]
AID - 10.1093/sysbio/syv116 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):495-507. doi: 10.1093/sysbio/syv116. Epub 2015 Dec 12.

PMID- 26658702
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - Geomolecular Dating and the Origin of Placental Mammals.
PG  - 546-57
LID - 10.1093/sysbio/syv115 [doi]
AB  - In modern evolutionary divergence analysis the role of geological information
      extends beyond providing a timescale, to informing molecular rate variation
      across the tree. Here I consider the implications of this development. I use
      fossil calibrations to test the accuracy of models of molecular rate evolution
      for placental mammals, and reveal substantial misspecification associated with
      life history rate correlates. Adding further calibrations to reduce dating errors
      at specific nodes unfortunately tends to transfer underlying rate errors to
      adjacent branches. Thus, tight calibration across the tree is vital to buffer
      against rate model errors. I argue that this must include allowing maximum bounds
      to be tight when good fossil records permit, otherwise divergences deep in the
      tree will tend to be inflated by the interaction of rate errors and asymmetric
      confidence in minimum and maximum bounds. In the case of placental mammals I
      sought to reduce the potential for transferring calibration and rate model errors
      across the tree by focusing on well-supported calibrations with appropriately
      conservative maximum bounds. The resulting divergence estimates are younger than 
      others published recently, and provide the long-anticipated molecular signature
      for the placental mammal radiation observed in the fossil record near the 66 Ma
      Cretaceous-Paleogene extinction event.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Phillips, Matthew J
AU  - Phillips MJ
AD  - School of Earth, Environmental and Biological Sciences, Queensland University of 
      Technology, Brisbane, QLD 4000, Australia m9.phillips@qut.edu.au.
LA  - eng
SI  - Dryad/10.5061/dryad.s43f8
PT  - Journal Article
DEP - 20151210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Evolution, Molecular
MH  - *Fossils
MH  - Mammals/*classification
MH  - Phylogeny
OTO - NOTNLM
OT  - *Cretaceous-Paleogene boundary
OT  - *Placentalia
OT  - *evolutionary rate
OT  - *fossil calibration
OT  - *life history
OT  - *molecular dating
EDAT- 2015/12/15 06:00
MHDA- 2018/01/11 06:00
CRDT- 2015/12/15 06:00
PHST- 2014/10/21 00:00 [received]
PHST- 2015/12/03 00:00 [accepted]
PHST- 2015/12/15 06:00 [entrez]
PHST- 2015/12/15 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syv115 [pii]
AID - 10.1093/sysbio/syv115 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):546-57. doi: 10.1093/sysbio/syv115. Epub 2015 Dec 10.

PMID- 26615177
OWN - NLM
STAT- MEDLINE
DCOM- 20160728
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 2
DP  - 2016 Mar
TI  - Maximum Likelihood Phylogenetic Inference is Consistent on Multiple Sequence
      Alignments, with or without Gaps.
PG  - 328-33
LID - 10.1093/sysbio/syv089 [doi]
AB  - We prove that maximum likelihood phylogenetic inference is consistent on gapped
      multiple sequence alignments (MSAs) as long as substitution rates across each
      edge are greater than zero, under mild assumptions on the structure of the
      alignment. Under these assumptions, maximum likelihood will asymptotically
      recover the tree with edge lengths corresponding to the mean number of
      substitutions per site on each edge. This refutes Warnow's recent suggestion
      (Warnow 2012) that maximum likelihood phylogenetic inference might be
      statistically inconsistent when gaps are treated as missing data, even if the MSA
      is correct. We also derive a simple new proof of maximum likelihood consistency
      of ungapped alignments.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Truszkowski, Jakub
AU  - Truszkowski J
AD  - European Molecular Biology Laboratory, European Bioinformatics Institute Wellcome
      Genome Campus, Hinxton, CB10 1SD, UK; Cancer Research UK Cambridge Institute,
      University of Cambridge Robinson Way, Cambridge CB2 0RE, UK jakubt@ebi.ac.uk.
FAU - Goldman, Nick
AU  - Goldman N
AD  - European Molecular Biology Laboratory, European Bioinformatics Institute Wellcome
      Genome Campus, Hinxton, CB10 1SD, UK;
LA  - eng
GR  - Cancer Research UK/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151128
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - *Phylogeny
MH  - *Sequence Alignment
PMC - PMC4748752
OTO - NOTNLM
OT  - Kullback-Leibler divergence
OT  - maximum likelihood
OT  - missing data
OT  - multiple sequence alignment
OT  - phylogeny
EDAT- 2015/11/29 06:00
MHDA- 2016/07/29 06:00
CRDT- 2015/11/29 06:00
PHST- 2015/08/05 00:00 [received]
PHST- 2015/11/19 00:00 [accepted]
PHST- 2015/11/29 06:00 [entrez]
PHST- 2015/11/29 06:00 [pubmed]
PHST- 2016/07/29 06:00 [medline]
AID - syv089 [pii]
AID - 10.1093/sysbio/syv089 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Mar;65(2):328-33. doi: 10.1093/sysbio/syv089. Epub 2015 Nov 28.

PMID- 26608965
OWN - NLM
STAT- MEDLINE
DCOM- 20160728
LR  - 20160211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 2
DP  - 2016 Mar
TI  - Cryptic Species or Inadequate Taxonomy? Implementation of 2D Geometric
      Morphometrics Based on Integumental Organs as Landmarks for Delimitation and
      Description of Copepod Taxa.
PG  - 304-27
LID - 10.1093/sysbio/syv088 [doi]
AB  - Discovery of cryptic species using molecular tools has become common in many
      animal groups but it is rarely accompanied by morphological revision, creating
      ongoing problems in taxonomy and conservation. In copepods, cryptic species have 
      been discovered in most groups where fast-evolving molecular markers were
      employed. In this study at Yeelirrie in Western Australia we investigate a
      subterranean species complex belonging to the harpacticoid genus Schizopera Sars,
      1905, using both the barcoding mitochondrial COI gene and landmark-based
      two-dimensional geometric morphometrics. Integumental organs (sensilla and pores)
      are used as landmarks for the first time in any crustacean group. Complete
      congruence between DNA-based species delimitation and relative position of
      integumental organs in two independent morphological structures suggests the
      existence of three distinct evolutionary units. We describe two of them as new
      species, employing a condensed taxonomic format appropriate for cryptic species. 
      We argue that many supposedly cryptic species might not be cryptic if researchers
      focus on analyzing morphological structures with multivariate tools that
      explicitly take into account geometry of the phenotype. A perceived supremacy of 
      molecular methods in detecting cryptic species is in our view a consequence of
      disparity of investment and unexploited recent advancements in morphometrics
      among taxonomists. Our study shows that morphometric data alone could be used to 
      find diagnostic morphological traits and gives hope to anyone studying small
      animals with a hard integument or shell, especially opening the door to assessing
      fossil diversity and rich museum collections. We expect that simultaneous use of 
      molecular tools with geometry-oriented morphometrics may yield faster formal
      description of species. Decrypted species in this study are a good example for
      urgency of formal descriptions, as they display short-range endemism in small
      groundwater calcrete aquifers in a paleochannel, where their conservation may be 
      threatened by proposed mining.
CI  - (c)The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Karanovic, Tomislav
AU  - Karanovic T
AD  - Department of Biological Science, College of Science, Sungkyunkwan University,
      Suwon 440-746, Korea; Institute for Marine and Antarctic Studies, University of
      Tasmania, Hobart, Tasmania 7001, Australia; Tomislav.Karanovic@utas.edu.au.
FAU - Djurakic, Marko
AU  - Djurakic M
AD  - Department of Biology and Ecology, Faculty of Science, University of Novi Sad,
      Trg Dositeja Obradovica 2, Novi Sad 21000, Serbia;
FAU - Eberhard, Stefan M
AU  - Eberhard SM
AD  - Subterranean Ecology Pty Ltd, Coningham, Tasmania 7054, Australia; and Connected 
      Waters Initiative Research Centre, University of New South Wales, Sydney 2052,
      Australia.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151124
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Copepoda/*anatomy &amp; histology/genetics
MH  - Electron Transport Complex IV/genetics
MH  - *Phylogeny
MH  - Species Specificity
MH  - Western Australia
OTO - NOTNLM
OT  - Barcoding
OT  - cuticular pore
OT  - integrative taxonomy
OT  - sensillum
OT  - speciation
OT  - stygofauna
EDAT- 2015/11/27 06:00
MHDA- 2016/07/29 06:00
CRDT- 2015/11/27 06:00
PHST- 2015/06/25 00:00 [received]
PHST- 2015/11/13 00:00 [accepted]
PHST- 2015/11/27 06:00 [entrez]
PHST- 2015/11/27 06:00 [pubmed]
PHST- 2016/07/29 06:00 [medline]
AID - syv088 [pii]
AID - 10.1093/sysbio/syv088 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Mar;65(2):304-27. doi: 10.1093/sysbio/syv088. Epub 2015 Nov 24.

PMID- 26568458
OWN - NLM
STAT- MEDLINE
DCOM- 20160728
LR  - 20160211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 2
DP  - 2016 Mar
TI  - Testing Classical Species Properties with Contemporary Data: How "Bad Species" in
      the Brassy Ringlets (Erebia tyndarus complex, Lepidoptera) Turned Good.
PG  - 292-303
LID - 10.1093/sysbio/syv087 [doi]
AB  - All species concepts are rooted in reproductive, and ultimately genealogical,
      relations. Genetic data are thus the most important source of information for
      species delimitation. Current ease of access to genomic data and recent
      computational advances are blooming a plethora of coalescent-based species
      delimitation methods. Despite their utility as objective approaches to identify
      species boundaries, coalescent-based methods (1) rely on simplified demographic
      models that may fail to capture some attributes of biological species, (2) do not
      make explicit use of the geographic information contained in the data, and (3)
      are often computationally intensive. In this article, we present a case of
      species delimitation in the Erebia tyndarus species complex, a taxon regarded as 
      a classic example of problematic taxonomic resolution. Our approach to species
      delimitation used genomic data to test predictions rooted in the biological
      species concept and in the criterion of coexistence in sympatry. We (1) obtained 
      restriction-site associated DNA (RAD) sequencing data from a carefully designed
      sample, (2) applied two genotype clustering algorithms to identify genetic
      clusters, and (3) performed within-clusters and between-clusters analyses of
      isolation by distance as a test for intrinsic reproductive barriers. Comparison
      of our results with those from a Bayes factor delimitation coalescent-based
      analysis, showed that coalescent-based approaches may lead to overconfident
      splitting of allopatric populations, and indicated that incorrect species
      delimitation is likely to be inferred when an incomplete geographic sample is
      analyzed. While we acknowledge the theoretical justification and practical
      usefulness of coalescent-based species delimitation methods, our results stress
      that, even in the phylogenomic era, the toolkit for species delimitation should
      not dismiss more traditional, biologically grounded, approaches coupling genomic 
      data with geographic information.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Gratton, Paolo
AU  - Gratton P
AD  - Dipartimento di Biologia, Universita di Roma "Tor Vergata", Via della Ricerca
      Scientifica, 00133 Roma Italy; Department of Primatology, Max Planck Institute
      for Evolutionary Anthropology, Deutscher Platz, 6, 04103 Leipzig Germany;
      paolo_gratton@eva.mpg.de.
FAU - Trucchi, Emiliano
AU  - Trucchi E
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis,
      University of Oslo, PO Box 1066, Blindern, Oslo 0316 Norway; and Department of
      Botany and Biodiversity Research, University of Vienna, Rennweg 14, 1030 Vienna, 
      Austria.
FAU - Trasatti, Alessandra
AU  - Trasatti A
AD  - Dipartimento di Biologia, Universita di Roma "Tor Vergata", Via della Ricerca
      Scientifica, 00133 Roma Italy;
FAU - Riccarducci, Giorgio
AU  - Riccarducci G
AD  - Dipartimento di Biologia, Universita di Roma "Tor Vergata", Via della Ricerca
      Scientifica, 00133 Roma Italy;
FAU - Marta, Silvio
AU  - Marta S
AD  - Dipartimento di Biologia, Universita di Roma "Tor Vergata", Via della Ricerca
      Scientifica, 00133 Roma Italy;
FAU - Allegrucci, Giuliana
AU  - Allegrucci G
AD  - Dipartimento di Biologia, Universita di Roma "Tor Vergata", Via della Ricerca
      Scientifica, 00133 Roma Italy;
FAU - Cesaroni, Donatella
AU  - Cesaroni D
AD  - Dipartimento di Biologia, Universita di Roma "Tor Vergata", Via della Ricerca
      Scientifica, 00133 Roma Italy;
FAU - Sbordoni, Valerio
AU  - Sbordoni V
AD  - Dipartimento di Biologia, Universita di Roma "Tor Vergata", Via della Ricerca
      Scientifica, 00133 Roma Italy;
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151114
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Animals
MH  - Butterflies/*classification/genetics
MH  - Classification/*methods
MH  - Genetic Speciation
MH  - Genotype
MH  - Geography
MH  - *Phylogeny
OTO - NOTNLM
OT  - RAD sequencing
OT  - Species delimitation
OT  - alpine butterflies, Erebia
OT  - genotype clustering
OT  - isolation by distance
OT  - next-generation sequencing
OT  - parapatric species
EDAT- 2015/11/17 06:00
MHDA- 2016/07/29 06:00
CRDT- 2015/11/17 06:00
PHST- 2014/12/11 00:00 [received]
PHST- 2015/11/06 00:00 [accepted]
PHST- 2015/11/17 06:00 [entrez]
PHST- 2015/11/17 06:00 [pubmed]
PHST- 2016/07/29 06:00 [medline]
AID - syv087 [pii]
AID - 10.1093/sysbio/syv087 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Mar;65(2):292-303. doi: 10.1093/sysbio/syv087. Epub 2015 Nov 14.

PMID- 26559010
OWN - NLM
STAT- MEDLINE
DCOM- 20160728
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 2
DP  - 2016 Mar
TI  - The Phylogeny of Rickettsia Using Different Evolutionary Signatures: How
      Tree-Like is Bacterial Evolution?
PG  - 265-79
LID - 10.1093/sysbio/syv084 [doi]
AB  - Rickettsia is a genus of intracellular bacteria whose hosts and transmission
      strategies are both impressively diverse, and this is reflected in a highly
      dynamic genome. Some previous studies have described the evolutionary history of 
      Rickettsia as non-tree-like, due to incongruity between phylogenetic
      reconstructions using different portions of the genome. Here, we reconstruct the 
      Rickettsia phylogeny using whole-genome data, including two new genomes from
      previously unsampled host groups. We find that a single topology, which is
      supported by multiple sources of phylogenetic signal, well describes the
      evolutionary history of the core genome. We do observe extensive incongruence
      between individual gene trees, but analyses of simulations over a single topology
      and interspersed partitions of sites show that this is more plausibly attributed 
      to systematic error than to horizontal gene transfer. Some conflicting placements
      also result from phylogenetic analyses of accessory genome content (i.e., gene
      presence/absence), but we argue that these are also due to systematic error,
      stemming from convergent genome reduction, which cannot be accommodated by
      existing phylogenetic methods. Our results show that, even within a single genus,
      tests for gene exchange based on phylogenetic incongruence may be susceptible to 
      false positives.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Murray, Gemma G R
AU  - Murray GG
AD  - Department of Genetics, University of Cambridge, Downing Street, Cambridge CB2
      3EH, UK; and ggrm2@cam.ac.uk.
FAU - Weinert, Lucy A
AU  - Weinert LA
AD  - Department of Veterinary Medicine, University of Cambridge, Madingley Road,
      Cambridge CB3 0ES, UK.
FAU - Rhule, Emma L
AU  - Rhule EL
AD  - Department of Genetics, University of Cambridge, Downing Street, Cambridge CB2
      3EH, UK; and.
FAU - Welch, John J
AU  - Welch JJ
AD  - Department of Genetics, University of Cambridge, Downing Street, Cambridge CB2
      3EH, UK; and.
LA  - eng
GR  - Medical Research Council/United Kingdom
GR  - Wellcome Trust/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151111
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - Classification
MH  - Computer Simulation/*standards
MH  - Genome, Bacterial/*genetics
MH  - *Phylogeny
MH  - Rickettsia/*classification/*genetics
PMC - PMC4748751
OTO - NOTNLM
OT  - Gene tree discordance
OT  - genome reduction
OT  - horizontal gene transfer
OT  - reticulate evolution
OT  - systematic error
EDAT- 2015/11/13 06:00
MHDA- 2016/07/29 06:00
CRDT- 2015/11/13 06:00
PHST- 2015/05/11 00:00 [received]
PHST- 2015/11/04 00:00 [accepted]
PHST- 2015/11/13 06:00 [entrez]
PHST- 2015/11/13 06:00 [pubmed]
PHST- 2016/07/29 06:00 [medline]
AID - syv084 [pii]
AID - 10.1093/sysbio/syv084 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Mar;65(2):265-79. doi: 10.1093/sysbio/syv084. Epub 2015 Nov 11.

PMID- 26559009
OWN - NLM
STAT- MEDLINE
DCOM- 20160728
LR  - 20160211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 2
DP  - 2016 Mar
TI  - Invariant Versus Classical Quartet Inference When Evolution is Heterogeneous
      Across Sites and Lineages.
PG  - 280-91
LID - 10.1093/sysbio/syv086 [doi]
AB  - One reason why classical phylogenetic reconstruction methods fail to correctly
      infer the underlying topology is because they assume oversimplified models. In
      this article, we propose a quartet reconstruction method consistent with the most
      general Markov model of nucleotide substitution, which can also deal with data
      coming from mixtures on the same topology. Our proposed method uses phylogenetic 
      invariants and provides a system of weights that can be used as input for
      quartet-based methods. We study its performance on real data and on a wide range 
      of simulated 4-taxon data (both time-homogeneous and nonhomogeneous, with or
      without among-site rate heterogeneity, and with different branch length
      settings). We compare it to the classical methods of neighbor-joining (with
      paralinear distance), maximum likelihood (with different underlying models), and 
      maximum parsimony. Our results show that this method is accurate and robust, has 
      a similar performance to maximum likelihood when data satisfies the assumptions
      of both methods, and outperform the other methods when these are based on
      inappropriate substitution models. If alignments are long enough, then it also
      outperforms other methods when some of its assumptions are violated.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Fernandez-Sanchez, Jesus
AU  - Fernandez-Sanchez J
AD  - Department of Mathematics, Universitat Politecnica de Catalunya, Barcelona,
      Spain.
FAU - Casanellas, Marta
AU  - Casanellas M
AD  - Department of Mathematics, Universitat Politecnica de Catalunya, Barcelona, Spain
      marta.casanellas@upc.edu.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151111
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Biological Evolution
MH  - Candida albicans/classification/genetics
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Saccharomyces/classification/genetics
OTO - NOTNLM
OT  - General Markov model
OT  - heterogeneity across lineages
OT  - heterogeneity across sites
OT  - phylogenetic invariants
OT  - topology reconstruction
OT  - yeast
EDAT- 2015/11/13 06:00
MHDA- 2016/07/29 06:00
CRDT- 2015/11/13 06:00
PHST- 2014/10/28 00:00 [received]
PHST- 2015/11/06 00:00 [accepted]
PHST- 2015/11/13 06:00 [entrez]
PHST- 2015/11/13 06:00 [pubmed]
PHST- 2016/07/29 06:00 [medline]
AID - syv086 [pii]
AID - 10.1093/sysbio/syv086 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Mar;65(2):280-91. doi: 10.1093/sysbio/syv086. Epub 2015 Nov 11.

PMID- 26546025
OWN - NLM
STAT- PubMed-not-MEDLINE
DCOM- 20160411
LR  - 20151216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 1
DP  - 2016 Jan
TI  - Erratum.
PG  - 180
LID - 10.1093/sysbio/syv074 [doi]
LA  - eng
PT  - Published Erratum
DEP - 20151105
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EFR - Syst Biol. 2015 Jan;64(1):e42-62. PMID: 25070970
PMC - PMC4678254
EDAT- 2015/11/08 06:00
MHDA- 2015/11/08 06:01
CRDT- 2015/11/08 06:00
PHST- 2015/11/08 06:00 [entrez]
PHST- 2015/11/08 06:00 [pubmed]
PHST- 2015/11/08 06:01 [medline]
AID - syv074 [pii]
AID - 10.1093/sysbio/syv074 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jan;65(1):180. doi: 10.1093/sysbio/syv074. Epub 2015 Nov 5.

PMID- 26526428
OWN - NLM
STAT- MEDLINE
DCOM- 20160728
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 2
DP  - 2016 Mar
TI  - Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic
      Uncertainty.
PG  - 250-64
LID - 10.1093/sysbio/syv083 [doi]
AB  - Marginal likelihood estimates to compare models using Bayes factors frequently
      accompany Bayesian phylogenetic inference. Approaches to estimate marginal
      likelihoods have garnered increased attention over the past decade. In
      particular, the introduction of path sampling (PS) and stepping-stone sampling
      (SS) into Bayesian phylogenetics has tremendously improved the accuracy of model 
      selection. These sampling techniques are now used to evaluate complex
      evolutionary and population genetic models on empirical data sets, but
      considerable computational demands hamper their widespread adoption. Further,
      when very diffuse, but proper priors are specified for model parameters,
      numerical issues complicate the exploration of the priors, a necessary step in
      marginal likelihood estimation using PS or SS. To avoid such instabilities,
      generalized SS (GSS) has recently been proposed, introducing the concept of
      "working distributions" to facilitate--or shorten--the integration process that
      underlies marginal likelihood estimation. However, the need to fix the tree
      topology currently limits GSS in a coalescent-based framework. Here, we extend
      GSS by relaxing the fixed underlying tree topology assumption. To this purpose,
      we introduce a "working" distribution on the space of genealogies, which enables 
      estimating marginal likelihoods while accommodating phylogenetic uncertainty. We 
      propose two different "working" distributions that help GSS to outperform PS and 
      SS in terms of accuracy when comparing demographic and evolutionary models
      applied to synthetic data and real-world examples. Further, we show that the use 
      of very diffuse priors can lead to a considerable overestimation in marginal
      likelihood when using PS and SS, while still retrieving the correct marginal
      likelihood using both GSS approaches. The methods used in this article are
      available in BEAST, a powerful user-friendly software package to perform Bayesian
      evolutionary analyses.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Baele, Guy
AU  - Baele G
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven-University
      of Leuven, Leuven, Belgium guy.baele@rega.kuleuven.be.
FAU - Lemey, Philippe
AU  - Lemey P
AD  - Department of Microbiology and Immunology, Rega Institute, KU Leuven-University
      of Leuven, Leuven, Belgium.
FAU - Suchard, Marc A
AU  - Suchard MA
AD  - Department of Biomathematics and Department of Human Genetics, David Geffen
      School of Medicine, University of California, Los Angeles, CA 90095, USA
      Department of Biostatistics, School of Public Health, University of California,
      Los Angeles, CA 90095, USA.
LA  - eng
GR  - 260864/European Research Council/International
GR  - R01 AI107034/AI/NIAID NIH HHS/United States
GR  - LM011827/LM/NLM NIH HHS/United States
GR  - R01 HG006139/HG/NHGRI NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20151101
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Evolution, Molecular
MH  - *Models, Genetic
MH  - *Phylogeny
PMC - PMC5009437
OTO - NOTNLM
OT  - Bayes factor
OT  - Bayesian inference
OT  - MCMC
OT  - Working distribution
OT  - coalescent model
OT  - marginal likelihood
OT  - phylogenetics
EDAT- 2015/11/04 06:00
MHDA- 2016/07/29 06:00
CRDT- 2015/11/04 06:00
PHST- 2015/06/01 00:00 [received]
PHST- 2015/10/28 00:00 [accepted]
PHST- 2015/11/04 06:00 [entrez]
PHST- 2015/11/04 06:00 [pubmed]
PHST- 2016/07/29 06:00 [medline]
AID - syv083 [pii]
AID - 10.1093/sysbio/syv083 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Mar;65(2):250-64. doi: 10.1093/sysbio/syv083. Epub 2015 Nov 1.

PMID- 26526427
OWN - NLM
STAT- MEDLINE
DCOM- 20160728
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 2
DP  - 2016 Mar
TI  - SimPhy: Phylogenomic Simulation of Gene, Locus, and Species Trees.
PG  - 334-44
LID - 10.1093/sysbio/syv082 [doi]
AB  - We present a fast and flexible software package--SimPhy--for the simulation of
      multiple gene families evolving under incomplete lineage sorting, gene
      duplication and loss, horizontal gene transfer--all three potentially leading to 
      species tree/gene tree discordance--and gene conversion. SimPhy implements a
      hierarchical phylogenetic model in which the evolution of species, locus, and
      gene trees is governed by global and local parameters (e.g., genome-wide,
      species-specific, locus-specific), that can be fixed or be sampled from a priori 
      statistical distributions. SimPhy also incorporates comprehensive models of
      substitution rate variation among lineages (uncorrelated relaxed clocks) and the 
      capability of simulating partitioned nucleotide, codon, and protein multilocus
      sequence alignments under a plethora of substitution models using the program
      INDELible. We validate SimPhy's output using theoretical expectations and other
      programs, and show that it scales extremely well with complex models and/or large
      trees, being an order of magnitude faster than the most similar program
      (DLCoal-Sim). In addition, we demonstrate how SimPhy can be useful to understand 
      interactions among different evolutionary processes, conducting a simulation
      study to characterize the systematic overestimation of the duplication time when 
      using standard reconciliation methods. SimPhy is available at
      https://github.com/adamallo/SimPhy, where users can find the source code,
      precompiled executables, a detailed manual and example cases.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Mallo, Diego
AU  - Mallo D
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Vigo
      36310, Spain dmallo@uvigo.es.
FAU - De Oliveira Martins, Leonardo
AU  - De Oliveira Martins L
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Vigo
      36310, Spain.
FAU - Posada, David
AU  - Posada D
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Vigo
      36310, Spain.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151101
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - Genes/genetics
MH  - Genetic Loci/genetics
MH  - Genetic Speciation
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Software/*standards
PMC - PMC4748750
OTO - NOTNLM
OT  - Gene conversion
OT  - gene duplication and loss
OT  - gene family evolution
OT  - horizontal gene transfer
OT  - incomplete lineage sorting
OT  - locus tree
OT  - simulation
OT  - species tree
EDAT- 2015/11/04 06:00
MHDA- 2016/07/29 06:00
CRDT- 2015/11/04 06:00
PHST- 2015/06/30 00:00 [received]
PHST- 2015/10/20 00:00 [accepted]
PHST- 2015/11/04 06:00 [entrez]
PHST- 2015/11/04 06:00 [pubmed]
PHST- 2016/07/29 06:00 [medline]
AID - syv082 [pii]
AID - 10.1093/sysbio/syv082 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Mar;65(2):334-44. doi: 10.1093/sysbio/syv082. Epub 2015 Nov 1.

PMID- 26508768
OWN - NLM
STAT- MEDLINE
DCOM- 20160727
LR  - 20190109
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 1
DP  - 2016 Jan
TI  - Testing for Deperet's Rule (Body Size Increase) in Mammals using Combined Extinct
      and Extant Data.
PG  - 98-108
LID - 10.1093/sysbio/syv075 [doi]
AB  - Whether or not evolutionary lineages in general show a tendency to increase in
      body size has often been discussed. This tendency has been dubbed "Cope's rule"
      but because Cope never hypothesized it, we suggest renaming it after Deperet, who
      formulated it clearly in 1907. Deperet's rule has traditionally been studied
      using fossil data, but more recently a number of studies have used present-day
      species. While several paleontological studies of Cenozoic placental mammals have
      found support for increasing body size, most studies of extant placentals have
      failed to detect such a trend. Here, we present a method to combine information
      from present-day species with fossil data in a Bayesian phylogenetic framework.
      We apply the method to body mass estimates of a large number of extant and
      extinct mammal species, and find strong support for Deperet's rule. The tendency 
      for size increase appears to be driven not by evolution toward larger size in
      established species, but by processes related to the emergence of new species.
      Our analysis shows that complementary data from extant and extinct species can
      greatly improve inference of macroevolutionary processes.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Bokma, Folmer
AU  - Bokma F
AD  - Department of Ecology and Environmental Science, Umea University, SE-901 87 Umea,
      Sweden; folmer.bokma@emg.umu.se.
FAU - Godinot, Marc
AU  - Godinot M
AD  - UMR 7207, CNRS/MNHN/UPMC, Ecole Pratique des Hautes Etudes, Paris, France;
FAU - Maridet, Olivier
AU  - Maridet O
AD  - Jurassica Museum, route de Fontenais 21, 2900 Porrentruy, Switzerland;
FAU - Ladeveze, Sandrine
AU  - Ladeveze S
AD  - UMR 7207, CNRS/MNHN/UPMC, Centre de Recherche sur la Paleodiversite et les
      Paleoenvironments (CR2P), Museum national d'Histoire naturelle, Departement
      Histoire de la Terre, 57 rue Cuvier, F-75231 Paris Cedex 05, France;
FAU - Costeur, Loic
AU  - Costeur L
AD  - Naturhistorisches Museum Basel, Augustinergasse 2, 4001 Basel, Switzerland; and.
FAU - Sole, Floreal
AU  - Sole F
AD  - Department of Palaeontology, Royal Belgian Institute of Natural Sciences, 29, rue
      Vautier, 1000 Brussels, Belgium.
FAU - Gheerbrant, Emmanuel
AU  - Gheerbrant E
AD  - UMR 7207, CNRS/MNHN/UPMC, Centre de Recherche sur la Paleodiversite et les
      Paleoenvironments (CR2P), Museum national d'Histoire naturelle, Departement
      Histoire de la Terre, 57 rue Cuvier, F-75231 Paris Cedex 05, France;
FAU - Peigne, Stephane
AU  - Peigne S
AD  - UMR 7207, CNRS/MNHN/UPMC, Centre de Recherche sur la Paleodiversite et les
      Paleoenvironments (CR2P), Museum national d'Histoire naturelle, Departement
      Histoire de la Terre, 57 rue Cuvier, F-75231 Paris Cedex 05, France;
FAU - Jacques, Florian
AU  - Jacques F
AD  - UMR 7207, CNRS/MNHN/UPMC, Centre de Recherche sur la Paleodiversite et les
      Paleoenvironments (CR2P), Museum national d'Histoire naturelle, Departement
      Histoire de la Terre, 57 rue Cuvier, F-75231 Paris Cedex 05, France;
FAU - Laurin, Michel
AU  - Laurin M
AD  - UMR 7207, CNRS/MNHN/UPMC, Centre de Recherche sur la Paleodiversite et les
      Paleoenvironments (CR2P), Museum national d'Histoire naturelle, Departement
      Histoire de la Terre, 57 rue Cuvier, F-75231 Paris Cedex 05, France;
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151027
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - *Body Size
MH  - Extinction, Biological
MH  - Fossils
MH  - Mammals/*anatomy &amp; histology/*classification
PMC - PMC4678255
OTO - NOTNLM
OT  - Body size
OT  - Cope's rule
OT  - macroevolution
OT  - paleontology
OT  - phylogenetics
EDAT- 2015/10/29 06:00
MHDA- 2016/07/28 06:00
CRDT- 2015/10/29 06:00
PHST- 2014/10/07 00:00 [received]
PHST- 2015/09/23 00:00 [accepted]
PHST- 2015/10/29 06:00 [entrez]
PHST- 2015/10/29 06:00 [pubmed]
PHST- 2016/07/28 06:00 [medline]
AID - syv075 [pii]
AID - 10.1093/sysbio/syv075 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jan;65(1):98-108. doi: 10.1093/sysbio/syv075. Epub 2015 Oct 27.

PMID- 26493828
OWN - NLM
STAT- MEDLINE
DCOM- 20160727
LR  - 20151216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 1
DP  - 2016 Jan
TI  - Inflation of Molecular Clock Rates and Dates: Molecular Phylogenetics,
      Biogeography, and Diversification of a Global Cicada Radiation from Australasia
      (Hemiptera: Cicadidae: Cicadettini).
PG  - 16-34
LID - 10.1093/sysbio/syv069 [doi]
AB  - Dated phylogenetic trees are important for studying mechanisms of
      diversification, and molecular clocks are important tools for studies of
      organisms lacking good fossil records. However, studies have begun to identify
      problems in molecular clock dates caused by uncertainty of the modeled molecular 
      substitution process. Here we explore Bayesian relaxed-clock molecular dating
      while studying the biogeography of ca. 200 species from the global cicada tribe
      Cicadettini. Because the available fossils are few and uninformative, we
      calibrate our trees in part with a cytochrome oxidase I (COI) clock prior
      encompassing a range of literature estimates for arthropods. We show that
      tribe-level analyses calibrated solely with the COI clock recover extremely old
      dates that conflict with published estimates for two well-studied New Zealand
      subclades within Cicadettini. Additional subclade analyses suggest that COI
      relaxed-clock rates and maximum-likelihood branch lengths become inflated
      relative to EF-1[Formula: see text] intron and exon rates and branch lengths as
      clade age increases. We present corrected estimates derived from: (i) an
      extrapolated EF-1[Formula: see text] exon clock derived from COI-calibrated
      analysis within the largest New Zealand subclade; (ii) post hoc scaling of the
      tribe-level chronogram using results from subclade analyses; and (iii)
      exploitation of a geological calibration point associated with New Caledonia. We 
      caution that considerable uncertainty is generated due to dependence of
      substitution estimates on both the taxon sample and the choice of model,
      including gamma category number and the choice of empirical versus estimated base
      frequencies. Our results suggest that diversification of the tribe Cicadettini
      commenced in the early- to mid-Cenozoic and continued with the development of
      open, arid habitats in Australia and worldwide. We find that Cicadettini is a
      rare example of a global terrestrial animal group with an Australasian origin,
      with all non-Australasian genera belonging to two distal clades. Within
      Australia, we show that Cicadettini is more widely distributed than any other
      cicada tribe, diverse in temperate, arid and monsoonal habitats, and nearly
      absent from rainforests. We comment on the taxonomic implications of our findings
      for thirteen cicada genera.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Marshall, David C
AU  - Marshall DC
AD  - Department of Ecology and Evolutionary Biology, 75 N. Eagleville Rd., Storrs, CT 
      06269, USA; david.marshall@uconn.edu.
FAU - Hill, Kathy B R
AU  - Hill KB
AD  - Department of Ecology and Evolutionary Biology, 75 N. Eagleville Rd., Storrs, CT 
      06269, USA;
FAU - Moulds, Max
AU  - Moulds M
AD  - Entomology Department, Australian Museum, 6 College Street, Sydney NSW 2010,
      Australia;
FAU - Vanderpool, Dan
AU  - Vanderpool D
AD  - Department of Ecology and Evolutionary Biology, 75 N. Eagleville Rd., Storrs, CT 
      06269, USA; Division of Biological Sciences, Health Sciences 304, U. Montana,
      Missoula, MT 59812;
FAU - Cooley, John R
AU  - Cooley JR
AD  - Department of Ecology and Evolutionary Biology, 75 N. Eagleville Rd., Storrs, CT 
      06269, USA;
FAU - Mohagan, Alma B
AU  - Mohagan AB
AD  - Central Mindanao University, Sayre Highway, Bukidnon, Philippines.
FAU - Simon, Chris
AU  - Simon C
AD  - Department of Ecology and Evolutionary Biology, 75 N. Eagleville Rd., Storrs, CT 
      06269, USA;
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151022
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Animal Distribution
MH  - Animals
MH  - Australasia
MH  - *Biodiversity
MH  - Electron Transport Complex IV/genetics
MH  - *Evolution, Molecular
MH  - Hemiptera/*classification/genetics
MH  - *Phylogeny
MH  - Time
OTO - NOTNLM
OT  - Aridification
OT  - Australia
OT  - branch lengths
OT  - calibration
OT  - climate change
OT  - gamma distribution
OT  - maximum-likelihood
OT  - relaxed clock
EDAT- 2015/10/24 06:00
MHDA- 2016/07/28 06:00
CRDT- 2015/10/24 06:00
PHST- 2014/04/04 00:00 [received]
PHST- 2015/09/17 00:00 [accepted]
PHST- 2015/10/24 06:00 [entrez]
PHST- 2015/10/24 06:00 [pubmed]
PHST- 2016/07/28 06:00 [medline]
AID - syv069 [pii]
AID - 10.1093/sysbio/syv069 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jan;65(1):16-34. doi: 10.1093/sysbio/syv069. Epub 2015 Oct 22.

PMID- 26493827
OWN - NLM
STAT- MEDLINE
DCOM- 20160728
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 2
DP  - 2016 Mar
TI  - Total-Evidence Dating under the Fossilized Birth-Death Process.
PG  - 228-49
LID - 10.1093/sysbio/syv080 [doi]
AB  - Bayesian total-evidence dating involves the simultaneous analysis of
      morphological data from the fossil record and morphological and sequence data
      from recent organisms, and it accommodates the uncertainty in the placement of
      fossils while dating the phylogenetic tree. Due to the flexibility of the
      Bayesian approach, total-evidence dating can also incorporate additional sources 
      of information. Here, we take advantage of this and expand the analysis to
      include information about fossilization and sampling processes. Our work is based
      on the recently described fossilized birth-death (FBD) process, which has been
      used to model speciation, extinction, and fossilization rates that can vary over 
      time in a piecewise manner. So far, sampling of extant and fossil taxa has been
      assumed to be either complete or uniformly at random, an assumption which is only
      valid for a minority of data sets. We therefore extend the FBD process to
      accommodate diversified sampling of extant taxa, which is standard practice in
      studies of higher-level taxa. We verify the implementation using simulations and 
      apply it to the early radiation of Hymenoptera (wasps, ants, and bees). Previous 
      total-evidence dating analyses of this data set were based on a simple uniform
      tree prior and dated the initial radiation of extant Hymenoptera to the late
      Carboniferous (309 Ma). The analyses using the FBD prior under diversified
      sampling, however, date the radiation to the Triassic and Permian (252 Ma),
      slightly older than the age of the oldest hymenopteran fossils. By exploring a
      variety of FBD model assumptions, we show that it is mainly the accommodation of 
      diversified sampling that causes the push toward more recent divergence times.
      Accounting for diversified sampling thus has the potential to close the
      long-discussed gap between rocks and clocks. We conclude that the explicit
      modeling of fossilization and sampling processes can improve divergence time
      estimates, but only if all important model aspects, including sampling biases,
      are adequately addressed.
CI  - (c)The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Zhang, Chi
AU  - Zhang C
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History,
      SE-104 05 Stockholm, Sweden;
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Department of Biosystems Science and Engineering, Eidgenossische Technische
      Hochschule Zurich, 4053 Basel, Switzerland; Swiss Institute of Bioinformatics
      (SIB), Switzerland;
FAU - Klopfstein, Seraina
AU  - Klopfstein S
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History,
      SE-104 05 Stockholm, Sweden; Department of Invertebrates, Natural History Museum 
      Bern, CH-3005 Bern, Switzerland;
FAU - Heath, Tracy A
AU  - Heath TA
AD  - Department of Integrative Biology, University of California, Berkeley, CA 94720
      USA; Department of Ecology and Evolutionary Biology, University of Kansas,
      Lawrence, KS 66045, USA; Department of Ecology, Evolution &amp; Organismal Biology,
      Iowa State University, Ames, IA 50011, USA.
FAU - Ronquist, Fredrik
AU  - Ronquist F
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History,
      SE-104 05 Stockholm, Sweden; fredrik.ronquist@nrm.se.
LA  - eng
SI  - Dryad/10.5061/dryad.26820
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151022
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biodiversity
MH  - Classification/*methods
MH  - *Fossils
MH  - Genetic Speciation
MH  - Hymenoptera/*classification
MH  - *Models, Biological
MH  - Phylogeny
MH  - Time
PMC - PMC4748749
OTO - NOTNLM
OT  - *Bayesian phylogenetic inference
OT  - *MCMC
OT  - *birth-death process
OT  - *relaxed clock
OT  - *total-evidence dating
OT  - *tree prior
EDAT- 2015/10/24 06:00
MHDA- 2016/07/29 06:00
CRDT- 2015/10/24 06:00
PHST- 2015/02/06 00:00 [received]
PHST- 2015/10/12 00:00 [accepted]
PHST- 2015/10/24 06:00 [entrez]
PHST- 2015/10/24 06:00 [pubmed]
PHST- 2016/07/29 06:00 [medline]
AID - syv080 [pii]
AID - 10.1093/sysbio/syv080 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Mar;65(2):228-49. doi: 10.1093/sysbio/syv080. Epub 2015 Oct 22.

PMID- 26470878
OWN - NLM
STAT- MEDLINE
DCOM- 20160727
LR  - 20151216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 1
DP  - 2016 Jan
TI  - Phylogenies, the Comparative Method, and the Conflation of Tempo and Mode.
PG  - 1-15
LID - 10.1093/sysbio/syv079 [doi]
AB  - The comparison of mathematical models that represent alternative hypotheses about
      the tempo and mode of evolutionary change is a common approach for assessing the 
      evolutionary processes underlying phenotypic diversification. However, because
      model parameters are estimated simultaneously, they are inextricably linked, such
      that changes in tempo, the pace of evolution, and mode, the manner in which
      evolution occurs, may be difficult to assess separately. This may potentially
      complicate biological interpretation, but the extent to which this occurs has not
      yet been determined. In this study, we examined 160 phylogeny x trait empirical
      data sets, and conducted extensive numerical phylogenetic simulations, to
      investigate the efficacy of phylogenetic comparative methods to distinguish
      between models that represent different evolutionary processes in a phylogenetic 
      context. We observed that, in some circumstances, a high uncertainty exists when 
      attempting to distinguish between alternative evolutionary scenarios underlying
      phenotypic variation. When examining data sets simulated under known conditions, 
      we found that evolutionary inference is straightforward when phenotypic patterns 
      are generated by simple evolutionary processes that are represented by modifying 
      a single model parameter at a time. However, inferring the exact nature of the
      evolutionary process that has yielded phenotypic variation when facing complex,
      potentially more realistic, mechanisms is more problematic. A detailed
      investigation of the influence of different model parameters showed that changes 
      in evolutionary rates, marked changes in phylogenetic means, or the existence of 
      a strong selective pull on the data, are all readily recovered by phenotypic
      model comparison. However, under evolutionary processes with a milder restraining
      pull acting on trait values, alternative models representing very different
      evolutionary processes may exhibit similar goodness-of-fit to the data,
      potentially leading to the conflation of interpretations that emphasize tempo and
      mode during empirical evolutionary inference. This is a mathematical and
      conceptual property of the considered models that, while not prohibitive for
      studying phenotypic evolution, should be taken into account and addressed when
      appropriate.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Kaliontzopoulou, Antigoni
AU  - Kaliontzopoulou A
AD  - CIBIO/InBio, Centro de Investigacao em Biodiversidade e Recursos Geneticos,
      Campus Agrario de Vairao, 4485-661 Vairao, Portugal; Department of Ecology,
      Evolution, and Organismal Biology; and antigoni@cibio.up.pt.
FAU - Adams, Dean C
AU  - Adams DC
AD  - Department of Ecology, Evolution, and Organismal Biology; and Department of
      Statistics, Iowa State University, Ames, IA 50011, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151015
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Models, Theoretical
MH  - *Phylogeny
MH  - Reproducibility of Results
OTO - NOTNLM
OT  - Comparative method
OT  - mode
OT  - model fit
OT  - phenotypic evolution
OT  - phylogeny
OT  - tempo
EDAT- 2015/10/17 06:00
MHDA- 2016/07/28 06:00
CRDT- 2015/10/17 06:00
PHST- 2013/07/31 00:00 [received]
PHST- 2015/10/07 00:00 [accepted]
PHST- 2015/10/17 06:00 [entrez]
PHST- 2015/10/17 06:00 [pubmed]
PHST- 2016/07/28 06:00 [medline]
AID - syv079 [pii]
AID - 10.1093/sysbio/syv079 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jan;65(1):1-15. doi: 10.1093/sysbio/syv079. Epub 2015 Oct 15.

PMID- 26454873
OWN - NLM
STAT- MEDLINE
DCOM- 20160727
LR  - 20151216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 1
DP  - 2016 Jan
TI  - Testing Convergence Versus History: Convergence Dominates Phenotypic Evolution
      for over 150 Million Years in Frogs.
PG  - 146-60
LID - 10.1093/sysbio/syv073 [doi]
AB  - Striking evolutionary convergence can lead to similar sets of species in
      different locations, such as in cichlid fishes and Anolis lizards, and suggests
      that evolution can be repeatable and predictable across clades. Yet, most
      examples of convergence involve relatively small temporal and/or spatial scales. 
      Some authors have speculated that at larger scales (e.g., across continents),
      differing evolutionary histories will prevent convergence. However, few studies
      have compared the contrasting roles of convergence and history, and none have
      done so at large scales. Here we develop a two-part approach to test the scale
      over which convergence can occur, comparing the relative importance of
      convergence and history in macroevolution using phylogenetic models of adaptive
      evolution. We apply this approach to data from morphology, ecology, and phylogeny
      from 167 species of anuran amphibians (frogs) from 10 local sites across the
      world, spanning ~160 myr of evolution. Mapping ecology on the phylogeny revealed 
      that similar microhabitat specialists (e.g., aquatic, arboreal) have evolved
      repeatedly across clades and regions, producing many evolutionary replicates for 
      testing for morphological convergence. By comparing morphological optima for
      clades and microhabitat types (our first test), we find that convergence
      associated with microhabitat use dominates frog morphological evolution,
      producing recurrent ecomorphs that together encompass all sampled species in each
      community in each region. However, our second test, which examines whether and
      how much species differ from their inferred optima, shows that convergence is
      incomplete: that is, phenotypes of most species are still somewhat distant from
      the estimated optimum for each microhabitat, seemingly because of insufficient
      time for more complete adaptation (an effect of history). Yet, these effects of
      history are related to past ecologies, and not clade membership. Overall, our
      study elucidates the dominant drivers of morphological evolution across a major
      vertebrate clade and shows that evolution can be repeatable at much greater
      temporal and spatial scales than commonly thought. It also provides an analytical
      framework for testing other potential examples of large-scale convergence.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Moen, Daniel S
AU  - Moen DS
AD  - Institut de Biologie, Ecole Normale Superieure, 75005 Paris, France; Graduate
      Program in Organismic and Evolutionary Biology, University of Massachusetts
      Amherst, Amherst, MA 01003, USA; Department of Integrative Biology, Oklahoma
      State University, Stillwater, OK 74078, USA; daniel.moen@okstate.edu.
FAU - Morlon, Helene
AU  - Morlon H
AD  - Institut de Biologie, Ecole Normale Superieure, 75005 Paris, France;
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151009
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Physiological
MH  - Animals
MH  - Anura/*anatomy &amp; histology/*classification
MH  - *Biological Evolution
MH  - *Ecosystem
MH  - *Models, Biological
MH  - Phenotype
MH  - Phylogeny
MH  - Principal Component Analysis
OTO - NOTNLM
OT  - Amphibians
OT  - Anura
OT  - communities
OT  - constraint
OT  - convergence
OT  - ecomorphology
OT  - macroevolution
EDAT- 2015/10/12 06:00
MHDA- 2016/07/28 06:00
CRDT- 2015/10/12 06:00
PHST- 2014/12/08 00:00 [received]
PHST- 2015/09/22 00:00 [accepted]
PHST- 2015/10/12 06:00 [entrez]
PHST- 2015/10/12 06:00 [pubmed]
PHST- 2016/07/28 06:00 [medline]
AID - syv073 [pii]
AID - 10.1093/sysbio/syv073 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jan;65(1):146-60. doi: 10.1093/sysbio/syv073. Epub 2015 Oct 9.

PMID- 26454872
OWN - NLM
STAT- MEDLINE
DCOM- 20160727
LR  - 20151216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 1
DP  - 2016 Jan
TI  - Detecting Macroevolutionary Self-Destruction from Phylogenies.
PG  - 109-27
LID - 10.1093/sysbio/syv062 [doi]
AB  - Phylogenetic analyses have lent support to the concept of lineage selection: that
      biological lineages can have heritable traits that influence their capacity to
      persist and diversify, and thereby affect their representation in biodiversity.
      While many discussions have focused on "positive" lineage selection, where stably
      heritable properties of lineages enhance their diversification rate, there are
      also intriguing examples that seem to represent "negative" lineage selection,
      where traits reduce the likelihood that a lineage will persist or speciate. In
      this article, we test whether a particular pattern of negative lineage selection 
      is detectable from the distributions of the trait on a phylogeny.
      "Self-destructive" traits are those that arise often but then disappear again
      because they confer either a raised extinction rate or they are prone to a high
      rate of trait loss. For such a trait, the reconstructed origins will tend to be
      dispersed across the tips of the phylogeny, rather than defining large clades of 
      related lineages that all share the trait. We examine the utility of four
      possible measures of "tippiness" as potential indicators of macroevolutionary
      self-destruction, applying them to phylogenies on which trait evolution has been 
      simulated under different combinations of parameters for speciation, extinction, 
      trait gain, and trait loss. We use an efficient simulation approach that starts
      with the required number of tips with and without the trait and uses a model to
      work "backwards" to construct different possible trees that result in that set of
      tips. We then apply these methods to a number of case studies: salt tolerance in 
      grasses, color polymorphism in birds of prey, and selfing in nightshades. We find
      that the relative age of species, measured from tip length, can indicate a
      reduced speciation rate but does not identify traits that increase the extinction
      rate or the trait loss rate. We show that it is possible to detect cases of
      macroevolutionary self-destruction by considering the number of tips with the
      trait that arise from each inferred origin, and the degree to which the trait is 
      scattered across the phylogeny. These metrics, and the methods we present, may be
      useful for testing macroevolutionary hypotheses from phylogenetic patterns.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Bromham, Lindell
AU  - Bromham L
AD  - Centre for Macroevolution and Macroecology, Division of Ecology, Evolution and
      Genetics, Research School of Biology, Australian National University, Canberra,
      ACT 0200, Australia lindell.bromham@anu.edu.au.
FAU - Hua, Xia
AU  - Hua X
AD  - Centre for Macroevolution and Macroecology, Division of Ecology, Evolution and
      Genetics, Research School of Biology, Australian National University, Canberra,
      ACT 0200, Australia.
FAU - Cardillo, Marcel
AU  - Cardillo M
AD  - Centre for Macroevolution and Macroecology, Division of Ecology, Evolution and
      Genetics, Research School of Biology, Australian National University, Canberra,
      ACT 0200, Australia.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151010
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Birds/physiology
MH  - *Extinction, Biological
MH  - *Phylogeny
MH  - Pigmentation/genetics
MH  - Poaceae/classification/genetics/physiology
MH  - Solanum/classification/physiology
MH  - Stress, Physiological
OTO - NOTNLM
OT  - Dead-end
OT  - diversification
OT  - extinction
OT  - lineage selection
OT  - macroevolution
OT  - speciation
EDAT- 2015/10/12 06:00
MHDA- 2016/07/28 06:00
CRDT- 2015/10/12 06:00
PHST- 2014/10/12 00:00 [received]
PHST- 2015/08/18 00:00 [accepted]
PHST- 2015/10/12 06:00 [entrez]
PHST- 2015/10/12 06:00 [pubmed]
PHST- 2016/07/28 06:00 [medline]
AID - syv062 [pii]
AID - 10.1093/sysbio/syv062 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jan;65(1):109-27. doi: 10.1093/sysbio/syv062. Epub 2015 Oct 10.

PMID- 26430060
OWN - NLM
STAT- MEDLINE
DCOM- 20160727
LR  - 20151216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 1
DP  - 2016 Jan
TI  - Rumbling Orchids: How To Assess Divergent Evolution Between Chloroplast
      Endosymbionts and the Nuclear Host.
PG  - 51-65
LID - 10.1093/sysbio/syv070 [doi]
AB  - Phylogenetic relationships inferred from multilocus organellar and nuclear DNA
      data are often difficult to resolve because of evolutionary conflicts among gene 
      trees. However, conflicting or "outlier" associations (i.e., linked pairs of
      "operational terminal units" in two phylogenies) among these data sets often
      provide valuable information on evolutionary processes such as chloroplast
      capture following hybridization, incomplete lineage sorting, and horizontal gene 
      transfer. Statistical tools that to date have been used in cophylogenetic studies
      only also have the potential to test for the degree of topological congruence
      between organellar and nuclear data sets and reliably detect outlier
      associations. Two distance-based methods, namely ParaFit and Procrustean Approach
      to Cophylogeny (PACo), were used in conjunction to detect those outliers
      contributing to conflicting phylogenies independently derived from chloroplast
      and nuclear sequence data. We explored their efficiency of retrieving outlier
      associations, and the impact of input data (unit branch length and additive
      trees) between data sets, by using several simulation approaches. To test their
      performance using real data sets, we additionally inferred the phylogenetic
      relationships within Neotropical Catasetinae (Epidendroideae, Orchidaceae), which
      is a suitable group to investigate phylogenetic incongruence because of
      hybridization processes between some of its constituent species. A comparison
      between trees derived from chloroplast and nuclear sequence data reflected
      strong, well-supported incongruence within Catasetum, Cycnoches, and Mormodes. As
      a result, outliers among chloroplast and nuclear data sets, and in experimental
      simulations, were successfully detected by PACo when using patristic distance
      matrices obtained from phylograms, but not from unit branch length trees. The
      performance of ParaFit was overall inferior compared to PACo, using either
      phylograms or unit branch lengths as input data. Because workflows for applying
      cophylogenetic analyses are not standardized yet, we provide a pipeline for
      executing PACo and ParaFit as well as displaying outlier associations in plots
      and trees by using the software R. The pipeline renders a method to identify
      outliers with high reliability and to assess the combinability of the
      independently derived data sets by means of statistical analyses.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Perez-Escobar, Oscar Alejandro
AU  - Perez-Escobar OA
AD  - Department Biologie, Systematische Botanik und Mykologie, GeoBio-Center,
      Ludwig-Maximilians Universitat, Menzinger Strasse 67, D-80 638 Munich, Germany;
FAU - Balbuena, Juan Antonio
AU  - Balbuena JA
AD  - Cavanilles Institute of Biodiversity and Evolutionary Biology, University of
      Valencia, Official P.O. Box 22085, 46071 Valencia, Spain.
FAU - Gottschling, Marc
AU  - Gottschling M
AD  - Department Biologie, Systematische Botanik und Mykologie, GeoBio-Center,
      Ludwig-Maximilians Universitat, Menzinger Strasse 67, D-80 638 Munich, Germany;
      gottschling@biologie.uni-muenchen.de.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151001
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
RN  - 0 (DNA, Plant)
SB  - IM
MH  - *Biological Evolution
MH  - Chloroplasts/*classification/*genetics
MH  - Classification/*methods
MH  - DNA, Chloroplast/genetics
MH  - DNA, Plant/genetics
MH  - Orchidaceae/*classification/*genetics
MH  - Phylogeny
MH  - Software
MH  - Symbiosis/*genetics
OTO - NOTNLM
OT  - chloroplast capture
OT  - cophylogenetic tool
OT  - hybridization
OT  - orchids
OT  - organelle/host nucleus coevolution
OT  - topological incongruence
EDAT- 2015/10/03 06:00
MHDA- 2016/07/28 06:00
CRDT- 2015/10/03 06:00
PHST- 2014/07/14 00:00 [received]
PHST- 2015/09/15 00:00 [accepted]
PHST- 2015/10/03 06:00 [entrez]
PHST- 2015/10/03 06:00 [pubmed]
PHST- 2016/07/28 06:00 [medline]
AID - syv070 [pii]
AID - 10.1093/sysbio/syv070 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jan;65(1):51-65. doi: 10.1093/sysbio/syv070. Epub 2015 Oct 1.

PMID- 26430059
OWN - NLM
STAT- MEDLINE
DCOM- 20160728
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 2
DP  - 2016 Mar
TI  - Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros,
      Ebenaceae) in New Caledonia, a Biodiversity Hotspot.
PG  - 212-27
LID - 10.1093/sysbio/syv076 [doi]
AB  - Due to its special geological history, the New Caledonian Archipelago is a mosaic
      of soil types, and in combination with climatic conditions this results in a
      heterogeneous environment across relatively small distances. A group of over 20
      endemic species of Diospyros (Ebenaceae) has rapidly and recently radiated on the
      archipelago after a single long-distance dispersal event. Most of the Diospyros
      species in the radiating group are morphologically and ecologically well
      differentiated, but they exhibit low levels of DNA variability. To investigate
      the processes that shaped the diversification of this group we employed
      restriction site associated DNA sequencing (RADseq). Over 8400 filtered SNPs
      generally confirm species delimitations and produce a well-supported phylogenetic
      tree. Our analyses document local introgression, but only a limited potential for
      gene flow over longer distances. The phylogenetic relationships point to an early
      regional clustering among populations and species, indicating that allopatric
      speciation with respect to macrohabitat (i.e., climatic conditions) may have had 
      a role in the initial differentiation within the group. A later, more rapid
      radiation involved divergence with respect to microhabitat (i.e., soil
      preference). Several sister species in the group show a parallel divergence in
      edaphic preference. Searches for genomic regions that are systematically
      differentiated in this replicated phenotypic divergence pointed to loci
      potentially involved in ion binding and cellular transport. These loci appear
      meaningful in the context of adaptations to soil types that differ in heavy-metal
      and mineral content. Identical nucleotide changes affected only two of these
      loci, indicating that introgression may have played a limited role in their
      evolution. Our results suggest that both allopatric diversification and
      (parapatric) ecological divergence shaped successive rounds of speciation in the 
      Diospyros radiation on New Caledonia.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Paun, Ovidiu
AU  - Paun O
AD  - Department of Botany and Biodiversity Research, University of Vienna, 1030
      Vienna, Austria;
FAU - Turner, Barbara
AU  - Turner B
AD  - Department of Botany and Biodiversity Research, University of Vienna, 1030
      Vienna, Austria; barbara.turner@univie.ac.at.
FAU - Trucchi, Emiliano
AU  - Trucchi E
AD  - Department of Botany and Biodiversity Research, University of Vienna, 1030
      Vienna, Austria;
FAU - Munzinger, Jerome
AU  - Munzinger J
AD  - IRD, UMR AMAP, 34398 Montpellier, France;
FAU - Chase, Mark W
AU  - Chase MW
AD  - Jodrell Laboratory, Royal Botanic Gardens, Kew, TW9 3AB Surrey, UK; and School of
      Plant Biology, University of Western Australia, Crawley, WA 6009 Australia.
FAU - Samuel, Rosabelle
AU  - Samuel R
AD  - Department of Botany and Biodiversity Research, University of Vienna, 1030
      Vienna, Austria;
LA  - eng
GR  - P 22159/Austrian Science Fund FWF/Austria
GR  - Y 661/Austrian Science Fund FWF/Austria
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20151001
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
RN  - 0 (Soil)
SB  - IM
MH  - *Adaptation, Physiological
MH  - Biodiversity
MH  - DNA, Plant/genetics
MH  - Diospyros/*classification/*physiology
MH  - *Genetic Speciation
MH  - Genetic Variation
MH  - Genome, Plant/*genetics
MH  - New Caledonia
MH  - Soil/chemistry
MH  - Tropical Climate
PMC - PMC4748748
MID - EMS66300
OID - NLM: EMS66300
OTO - NOTNLM
OT  - Adaptive radiation
OT  - Diospyros
OT  - New Caledonia
OT  - RAD-sequencing
OT  - hybridization
OT  - soil adaptation
EDAT- 2015/10/03 06:00
MHDA- 2016/07/29 06:00
CRDT- 2015/10/03 06:00
PHST- 2015/06/17 00:00 [received]
PHST- 2015/09/25 00:00 [accepted]
PHST- 2015/10/03 06:00 [entrez]
PHST- 2015/10/03 06:00 [pubmed]
PHST- 2016/07/29 06:00 [medline]
AID - syv076 [pii]
AID - 10.1093/sysbio/syv076 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Mar;65(2):212-27. doi: 10.1093/sysbio/syv076. Epub 2015 Oct 1.

PMID- 26424727
OWN - NLM
STAT- MEDLINE
DCOM- 20160727
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 1
DP  - 2016 Jan
TI  - Fast Dating Using Least-Squares Criteria and Algorithms.
PG  - 82-97
LID - 10.1093/sysbio/syv068 [doi]
AB  - Phylogenies provide a useful way to understand the evolutionary history of
      genetic samples, and data sets with more than a thousand taxa are becoming
      increasingly common, notably with viruses (e.g., human immunodeficiency virus
      (HIV)). Dating ancestral events is one of the first, essential goals with such
      data. However, current sophisticated probabilistic approaches struggle to handle 
      data sets of this size. Here, we present very fast dating algorithms, based on a 
      Gaussian model closely related to the Langley-Fitch molecular-clock model. We
      show that this model is robust to uncorrelated violations of the molecular clock.
      Our algorithms apply to serial data, where the tips of the tree have been sampled
      through times. They estimate the substitution rate and the dates of all ancestral
      nodes. When the input tree is unrooted, they can provide an estimate for the root
      position, thus representing a new, practical alternative to the standard rooting 
      methods (e.g., midpoint). Our algorithms exploit the tree (recursive) structure
      of the problem at hand, and the close relationships between least-squares and
      linear algebra. We distinguish between an unconstrained setting and the case
      where the temporal precedence constraint (i.e., an ancestral node must be older
      that its daughter nodes) is accounted for. With rooted trees, the former is
      solved using linear algebra in linear computing time (i.e., proportional to the
      number of taxa), while the resolution of the latter, constrained setting, is
      based on an active-set method that runs in nearly linear time. With unrooted
      trees the computing time becomes (nearly) quadratic (i.e., proportional to the
      square of the number of taxa). In all cases, very large input trees (&gt;10,000
      taxa) can easily be processed and transformed into time-scaled trees. We compare 
      these algorithms to standard methods (root-to-tip, r8s version of Langley-Fitch
      method, and BEAST). Using simulated data, we show that their estimation accuracy 
      is similar to that of the most sophisticated methods, while their computing time 
      is much faster. We apply these algorithms on a large data set comprising 1194
      strains of Influenza virus from the pdm09 H1N1 Human pandemic. Again the results 
      show that these algorithms provide a very fast alternative with results similar
      to those of other computer programs. These algorithms are implemented in the LSD 
      software (least-squares dating), which can be downloaded from
      http://www.atgc-montpellier.fr/LSD/, along with all our data sets and detailed
      results. An Online Appendix, providing additional algorithm descriptions, tables,
      and figures can be found in the Supplementary Material available on Dryad at
      http://dx.doi.org/10.5061/dryad.968t3.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - To, Thu-Hien
AU  - To TH
AD  - Institut de Biologie Computationnelle, LIRMM, UMR 5506 CNRS - Universite de
      Montpellier, France;
FAU - Jung, Matthieu
AU  - Jung M
AD  - Institut de Biologie Computationnelle, LIRMM, UMR 5506 CNRS - Universite de
      Montpellier, France; IGBMC (Institut de Genetique et de Biologie Moleculaire et
      Cellulaire), INSERM, U596, CNRS, UMR7104, Universite de Strasbourg, Illkirch,
      France;
FAU - Lycett, Samantha
AU  - Lycett S
AD  - Institute of Evolutionary Biology, University of Edinburgh, Ashworth
      Laboratories, Edinburgh, UK.
FAU - Gascuel, Olivier
AU  - Gascuel O
AD  - Institut de Biologie Computationnelle, LIRMM, UMR 5506 CNRS - Universite de
      Montpellier, France; gascuel@lirmm.fr.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150930
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Computer Simulation
MH  - Evolution, Molecular
MH  - Influenza A Virus, H1N1 Subtype/*classification/genetics
MH  - Least-Squares Analysis
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Software
PMC - PMC4678253
OTO - NOTNLM
OT  - Active-set method
OT  - algorithms
OT  - computer simulations
OT  - dating
OT  - influenza (H1N1)
OT  - least-squares
OT  - linear algebra
OT  - molecular clock
OT  - serial data
OT  - substitution rate estimation
OT  - temporal precedence constraints
OT  - viruses
EDAT- 2015/10/02 06:00
MHDA- 2016/07/28 06:00
CRDT- 2015/10/02 06:00
PHST- 2014/09/10 00:00 [received]
PHST- 2015/09/16 00:00 [accepted]
PHST- 2015/10/02 06:00 [entrez]
PHST- 2015/10/02 06:00 [pubmed]
PHST- 2016/07/28 06:00 [medline]
AID - syv068 [pii]
AID - 10.1093/sysbio/syv068 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jan;65(1):82-97. doi: 10.1093/sysbio/syv068. Epub 2015 Sep 30.

PMID- 26420142
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20151015
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Species Delimitation's Coming of Age.
PG  - 897-9
LID - 10.1093/sysbio/syv071 [doi]
FAU - Flot, Jean-Francois
AU  - Flot JF
AD  - Department of Genetics, Evolution and Environment, University College London,
      Darwin Building, Gower Street, London WC1E 6BT, United Kingdom Email:
      j.flot@ucl.ac.uk.
LA  - eng
PT  - Introductory Journal Article
DEP - 20150929
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Classification
MH  - *Periodicals as Topic
EDAT- 2015/10/01 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/10/01 06:00
PHST- 2015/10/01 06:00 [entrez]
PHST- 2015/10/01 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv071 [pii]
AID - 10.1093/sysbio/syv071 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):897-9. doi: 10.1093/sysbio/syv071. Epub 2015 Sep 29.

PMID- 26405218
OWN - NLM
STAT- MEDLINE
DCOM- 20160727
LR  - 20190109
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 1
DP  - 2016 Jan
TI  - Quantifying Age-dependent Extinction from Species Phylogenies.
PG  - 35-50
LID - 10.1093/sysbio/syv065 [doi]
AB  - Several ecological factors that could play into species extinction are expected
      to correlate with species age, i.e., time elapsed since the species arose by
      speciation. To date, however, statistical tools to incorporate species age into
      likelihood-based phylogenetic inference have been lacking. We present here a
      computational framework to quantify age-dependent extinction through maximum
      likelihood parameter estimation based on phylogenetic trees, assuming species
      lifetimes are gamma distributed. Testing on simulated trees shows that neglecting
      age dependence can lead to biased estimates of key macroevolutionary parameters. 
      We then apply this method to two real data sets, namely a complete phylogeny of
      birds (class Aves) and a clade of self-compatible and -incompatible nightshades
      (Solanaceae), gaining initial insights into the extent to which age-dependent
      extinction may help explain macroevolutionary patterns. Our methods have been
      added to the R package TreePar.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Alexander, Helen K
AU  - Alexander HK
AD  - Institute for Integrative Biology, ETH Zurich, 8092 Zurich, Switzerland;
      helen.alexander@env.ethz.ch.
FAU - Lambert, Amaury
AU  - Lambert A
AD  - Laboratoire de Probabilites et Modeles Aleatoires CNRS UMR 7599, UPMC Univ Paris 
      06, Paris, France; Center for Interdisciplinary Research in Biology CNRS UMR
      7241, College de France, Paris, France; and.
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Department of Biosystems Science and Engineering, ETH Zurich, 4058 Basel,
      Switzerland;
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150924
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Biological Evolution
MH  - Birds/classification
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Extinction, Biological
MH  - Likelihood Functions
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Solanaceae/classification
MH  - Time
PMC - PMC4678252
OTO - NOTNLM
OT  - Aves
OT  - Solanaceae
OT  - coalescent point process
OT  - diversification
OT  - macroevolution
OT  - maximum likelihood estimation
OT  - phylogenetics
EDAT- 2015/09/26 06:00
MHDA- 2016/07/28 06:00
CRDT- 2015/09/26 06:00
PHST- 2014/04/16 00:00 [received]
PHST- 2015/09/01 00:00 [accepted]
PHST- 2015/09/26 06:00 [entrez]
PHST- 2015/09/26 06:00 [pubmed]
PHST- 2016/07/28 06:00 [medline]
AID - syv065 [pii]
AID - 10.1093/sysbio/syv065 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jan;65(1):35-50. doi: 10.1093/sysbio/syv065. Epub 2015 Sep 24.

PMID- 26396091
OWN - NLM
STAT- MEDLINE
DCOM- 20160728
LR  - 20160211
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 2
DP  - 2016 Mar
TI  - A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification.
PG  - 181-93
LID - 10.1093/sysbio/syv066 [doi]
AB  - Rates of species diversification vary widely across the tree of life and there is
      considerable interest in identifying organismal traits that correlate with rates 
      of speciation and extinction. However, it has been challenging to develop
      methodological frameworks for testing hypotheses about trait-dependent
      diversification that are robust to phylogenetic pseudoreplication and to
      directionally biased rates of character change. We describe a semi-parametric
      test for trait-dependent diversification that explicitly requires replicated
      associations between character states and diversification rates to detect
      effects. To use the method, diversification rates are reconstructed across a
      phylogenetic tree with no consideration of character states. A test statistic is 
      then computed to measure the association between species-level traits and the
      corresponding diversification rate estimates at the tips of the tree. The
      empirical value of the test statistic is compared to a null distribution that is 
      generated by structured permutations of evolutionary rates across the phylogeny. 
      The test is applicable to binary discrete characters as well as continuous-valued
      traits and can accommodate extremely sparse sampling of character states at the
      tips of the tree. We apply the test to several empirical data sets and
      demonstrate that the method has acceptable Type I error rates.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Rabosky, Daniel L
AU  - Rabosky DL
AD  - Department of Ecology and Evolutionary Biology and Museum of Zoology, University 
      of Michigan, 1109 Geddes Avenue, Ann Arbor, MI 48109, USA drabosky@umich.edu.
FAU - Huang, Huateng
AU  - Huang H
AD  - Department of Ecology and Evolutionary Biology and Museum of Zoology, University 
      of Michigan, 1109 Geddes Avenue, Ann Arbor, MI 48109, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150922
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Birds/classification
MH  - Classification/*methods
MH  - Fishes/classification
MH  - *Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
OTO - NOTNLM
OT  - BAMM
OT  - comparative analysis
OT  - extinction
OT  - key innovation
OT  - speciation
OT  - state-dependent diversification
OT  - statistical methodology
EDAT- 2015/09/24 06:00
MHDA- 2016/07/29 06:00
CRDT- 2015/09/24 06:00
PHST- 2015/02/25 00:00 [received]
PHST- 2015/09/09 00:00 [accepted]
PHST- 2015/09/24 06:00 [entrez]
PHST- 2015/09/24 06:00 [pubmed]
PHST- 2016/07/29 06:00 [medline]
AID - syv066 [pii]
AID - 10.1093/sysbio/syv066 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Mar;65(2):181-93. doi: 10.1093/sysbio/syv066. Epub 2015 Sep 22.

PMID- 26385618
OWN - NLM
STAT- MEDLINE
DCOM- 20160727
LR  - 20151216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 1
DP  - 2016 Jan
TI  - Enlarged Multilocus Data set Provides Surprisingly Younger Time of Origin for the
      Plethodontidae, the Largest Family of Salamanders.
PG  - 66-81
LID - 10.1093/sysbio/syv061 [doi]
AB  - Deep phylogenetic relationships of the largest salamander family Plethodontidae
      have been difficult to resolve, probably reflecting a rapid diversification early
      in their evolutionary history. Here, data from 50 independent nuclear markers
      (total 48,582 bp) are used to reconstruct the phylogeny and divergence times for 
      plethodontid salamanders, using both concatenation and coalescence-based species 
      tree analyses. Our results robustly resolve the position of the enigmatic eastern
      North American four-toed salamander (Hemidactylium) as the sister taxon of
      Batrachoseps + Tribe Bolitoglossini, thus settling a long-standing question.
      Furthermore, we statistically reject sister taxon status of Karsenia and
      Hydromantes, the only plethodontids to occur outside the Americas, leading us to 
      new biogeographic hypotheses. Contrary to previous long-standing arguments that
      plethodontid salamanders are an old lineage originating in the Cretaceous (more
      than 90 Ma), our analyses lead to the hypothesis that these salamanders are much 
      younger, arising close to the K-T boundary (~66 Ma). These time estimates are
      highly stable using alternative calibration schemes and dating methods. Our data 
      simulation highlights the potential risk of making strong arguments about
      phylogenetic timing based on inferences from a handful of nuclear genes, a common
      practice. Based on the newly obtained timetree and ancestral area reconstruction 
      results, we argue that (i) the classic "Out of Appalachia" hypothesis of
      plethodontid origins is problematic; (ii) the common ancestor of extant
      plethodontids may have originated in northwestern North America in the early
      Paleocene; (iii) origins of Eurasian plethodontids likely result from two
      separate dispersal events from western North America via Beringia in the late
      Eocene (~42 Ma) and the early Miocene (~23 Ma), respectively.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Shen, Xing-Xing
AU  - Shen XX
AD  - State Key Laboratory of Biocontrol, College of Ecology and Evolution, School of
      Life Sciences, Sun Yat-Sen University, Guangzhou, China and.
FAU - Liang, Dan
AU  - Liang D
AD  - State Key Laboratory of Biocontrol, College of Ecology and Evolution, School of
      Life Sciences, Sun Yat-Sen University, Guangzhou, China and.
FAU - Chen, Meng-Yun
AU  - Chen MY
AD  - State Key Laboratory of Biocontrol, College of Ecology and Evolution, School of
      Life Sciences, Sun Yat-Sen University, Guangzhou, China and.
FAU - Mao, Rong-Li
AU  - Mao RL
AD  - State Key Laboratory of Biocontrol, College of Ecology and Evolution, School of
      Life Sciences, Sun Yat-Sen University, Guangzhou, China and.
FAU - Wake, David B
AU  - Wake DB
AD  - Museum of Vertebrate Zoology and Department of Integrative Biology, 3101 Valley
      Life Sciences Bldg, University of California, Berkeley, CA 94720, USA
      alarzhang@gmail.com wakelab@berkeley.edu.
FAU - Zhang, Peng
AU  - Zhang P
AD  - State Key Laboratory of Biocontrol, College of Ecology and Evolution, School of
      Life Sciences, Sun Yat-Sen University, Guangzhou, China and alarzhang@gmail.com
      wakelab@berkeley.edu.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150918
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animal Distribution
MH  - Animals
MH  - Molecular Sequence Data
MH  - North America
MH  - *Phylogeny
MH  - Sequence Analysis, DNA
MH  - Time
MH  - Urodela/*classification/*genetics
OTO - NOTNLM
OT  - Dispersal
OT  - molecular dating
OT  - paleogeography
OT  - phylogenomics
OT  - species tree
OT  - timetree
EDAT- 2015/09/20 06:00
MHDA- 2016/07/28 06:00
CRDT- 2015/09/20 06:00
PHST- 2014/08/13 00:00 [received]
PHST- 2015/08/15 00:00 [accepted]
PHST- 2015/09/20 06:00 [entrez]
PHST- 2015/09/20 06:00 [pubmed]
PHST- 2016/07/28 06:00 [medline]
AID - syv061 [pii]
AID - 10.1093/sysbio/syv061 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jan;65(1):66-81. doi: 10.1093/sysbio/syv061. Epub 2015 Sep 18.

PMID- 26377442
OWN - NLM
STAT- MEDLINE
DCOM- 20160728
LR  - 20190109
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 2
DP  - 2016 Mar
TI  - Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case
      Example on Limb Development.
PG  - 194-211
LID - 10.1093/sysbio/syv067 [doi]
AB  - How the genotype translates into the phenotype through development is critical to
      fully understand the evolution of phenotypes. We propose a novel approach to
      directly assess how changes in gene expression patterns are associated with
      changes in morphology using the limb as a case example. Our method combines
      molecular biology techniques, such as whole-mount in situ hybridization, with
      image and shape analysis, extending the use of Geometric Morphometrics to the
      analysis of nonanatomical shapes, such as gene expression domains. Elliptical
      Fourier and Procrustes-based semilandmark analyses were used to analyze the
      variation and covariation patterns of the limb bud shape with the expression
      patterns of two relevant genes for limb morphogenesis, Hoxa11 and Hoxa13. We
      devised a multiple thresholding method to semiautomatically segment gene domains 
      at several expression levels in large samples of limb buds from C57Bl6 mouse
      embryos between 10 and 12 postfertilization days. Besides providing an accurate
      phenotyping tool to quantify the spatiotemporal dynamics of gene expression
      patterns within developing structures, our morphometric analyses revealed high,
      non-random, and gene-specific variation undergoing canalization during limb
      development. Our results demonstrate that Hoxa11 and Hoxa13, despite being
      paralogs with analogous functions in limb patterning, show clearly distinct
      dynamic patterns, both in shape and size, and are associated differently with the
      limb bud shape. The correspondence between our results and already
      well-established molecular processes underlying limb development confirms that
      this morphometric approach is a powerful tool to extract features of development 
      regulating morphogenesis. Such multilevel analyses are promising in systems where
      not so much molecular information is available and will advance our understanding
      of the genotype-phenotype map. In systematics, this knowledge will increase our
      ability to infer how evolution modified a common developmental pattern to
      generate a wide diversity of morphologies, as in the vertebrate limb.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Martinez-Abadias, Neus
AU  - Martinez-Abadias N
AD  - EMBL-CRG Systems Biology Program, Centre for Genomic Regulation (CRG), The
      Barcelona Institute of Science and Technology, Dr. Aiguader 88, 08003 Barcelona, 
      Spain; Universitat Pompeu Fabra (UPF), Barcelona, Spain;
FAU - Mateu, Roger
AU  - Mateu R
AD  - EMBL-CRG Systems Biology Program, Centre for Genomic Regulation (CRG), The
      Barcelona Institute of Science and Technology, Dr. Aiguader 88, 08003 Barcelona, 
      Spain; Universitat Pompeu Fabra (UPF), Barcelona, Spain;
FAU - Niksic, Martina
AU  - Niksic M
AD  - EMBL-CRG Systems Biology Program, Centre for Genomic Regulation (CRG), The
      Barcelona Institute of Science and Technology, Dr. Aiguader 88, 08003 Barcelona, 
      Spain; Universitat Pompeu Fabra (UPF), Barcelona, Spain;
FAU - Russo, Lucia
AU  - Russo L
AD  - EMBL-CRG Systems Biology Program, Centre for Genomic Regulation (CRG), The
      Barcelona Institute of Science and Technology, Dr. Aiguader 88, 08003 Barcelona, 
      Spain; Universitat Pompeu Fabra (UPF), Barcelona, Spain;
FAU - Sharpe, James
AU  - Sharpe J
AD  - EMBL-CRG Systems Biology Program, Centre for Genomic Regulation (CRG), The
      Barcelona Institute of Science and Technology, Dr. Aiguader 88, 08003 Barcelona, 
      Spain; Universitat Pompeu Fabra (UPF), Barcelona, Spain; Institucio Catalana de
      Recerca i Estudis Avancats (ICREA), Pg. Lluis Companys 23, 08010 Barcelona, Spain
      james.sharpe@crg.eu.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150916
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Homeodomain Proteins)
RN  - 0 (Hoxa11 protein, mouse)
RN  - 0 (homeobox protein HOXA13)
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Gene Expression Profiling
MH  - *Gene Expression Regulation, Developmental
MH  - Homeodomain Proteins/genetics
MH  - Limb Buds/anatomy &amp; histology/*embryology
MH  - Mice
MH  - *Phenotype
PMC - PMC4748747
OTO - NOTNLM
OT  - Elliptical Fourier analysis
OT  - Hoxa genes
OT  - Procrustes-based semilandmark
OT  - gene domain
OT  - limb morphogenesis
OT  - whole-mount in situ hybridization
EDAT- 2015/09/18 06:00
MHDA- 2016/07/29 06:00
CRDT- 2015/09/18 06:00
PHST- 2015/04/24 00:00 [received]
PHST- 2015/09/11 00:00 [accepted]
PHST- 2015/09/18 06:00 [entrez]
PHST- 2015/09/18 06:00 [pubmed]
PHST- 2016/07/29 06:00 [medline]
AID - syv067 [pii]
AID - 10.1093/sysbio/syv067 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Mar;65(2):194-211. doi: 10.1093/sysbio/syv067. Epub 2015 Sep 16.

PMID- 26370565
OWN - NLM
STAT- MEDLINE
DCOM- 20180108
LR  - 20190112
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 66
IP  - 2
DP  - 2017 Mar 1
TI  - Coordinated Dispersal and Pre-Isthmian Assembly of the Central American
      Ichthyofauna.
PG  - 183-196
LID - 10.1093/sysbio/syv064 [doi]
AB  - We document patterns of coordinated dispersal over evolutionary time frames in
      heroine cichlids and poeciliine live-bearers, the two most species-rich clades of
      freshwater fishes in the Caribbean basin. Observed dispersal rate (DO) values
      were estimated from time-calibrated molecular phylogenies in Lagrange+, a
      modified version of the ML-based parametric biogeographic program Lagrange. DO is
      measured in units of "wallaces" (wa) as the number of biogeographic
      range-expansion events per million years. DO estimates were generated on a
      dynamic paleogeographic landscape of five areas over three time intervals from
      Upper Cretaceous to Recent. Expected dispersal rate (DE) values were generated
      from alternative paleogeographic models, with dispersal rates proportional to
      target area and source-river discharge volume, and inversely proportional to
      paleogeographic distance. Correlations between DO and DE were used to assess the 
      relative contributions of these three biogeographic parameters. DO estimates
      imply a persistent dispersal corridor across the Eastern (Antillean) margin of
      the Caribbean plate, under the influence of prevailing and perennial riverine
      discharge vectors such as the Proto-Orinoco-Amazon river. Ancestral area
      estimation places the earliest colonizations of the Greater Antilles and Central 
      America during the Paleocene-Eocene (ca. 58-45 Ma), potentially during the
      existence of an incomplete Paleogene Arc ( approximately 59 Ma) or Lesser
      Antilles Arc ( approximately 45 Ma), but predating the GAARlandia land bridge (
      approximately 34-33 Ma). Paleogeographic distance is the single best predictor of
      DO. The Western (Central American) plate margin did not serve as a dispersal
      corridor until the Late Neogene (12-0 Ma), and contributed relatively little to
      the formation of modern distributions.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Tagliacollo, Victor A
AU  - Tagliacollo VA
AD  - Biology Department, University of Louisiana at Lafayette, Lafayette, LA
      70504-2451, USA.
AD  - Instituto de Biociencias de Botucatu, Universidade Estadual Paulista-UNESP,
      Botucatu, Sao Paulo 18618-970, Brazil.
FAU - Duke-Sylvester, Scott M
AU  - Duke-Sylvester SM
AD  - Biology Department, University of Louisiana at Lafayette, Lafayette, LA
      70504-2451, USA.
FAU - Matamoros, Wilfredo A
AU  - Matamoros WA
AD  - Facultad de Ciencias Biologicas, Universidad de Ciencias y Artes de Chiapas,
      Libramiento Norte Poniente 1150, Col. Lajas Maciel, C.P. 29039, Tuxtla Gutierrez,
      Chiapas, Mexico; and.
FAU - Chakrabarty, Prosanta
AU  - Chakrabarty P
AD  - Department of Biological Sciences, Museum of Natural Science (Fish Section),
      Louisiana State University, Baton Rouge, LA 70803, USA.
FAU - Albert, James S
AU  - Albert JS
AD  - Biology Department, University of Louisiana at Lafayette, Lafayette, LA
      70504-2451, USA.
LA  - eng
SI  - Dryad/10.5061/dryad.11cv0
PT  - Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Animal Distribution
MH  - Animals
MH  - *Biological Evolution
MH  - Caribbean Region
MH  - Central America
MH  - Fishes/*classification/physiology
MH  - Fresh Water
MH  - Phylogeny
MH  - Sequence Analysis, DNA
MH  - Software
PMC - PMC5410936
OTO - NOTNLM
OT  - *Caribbean plate
OT  - *Central America
OT  - *Cichlidae
OT  - *Greater Antilles
OT  - *Poeciliidae
OT  - *historical biogeography
OT  - *parametric biogeography
EDAT- 2015/09/16 06:00
MHDA- 2018/01/09 06:00
CRDT- 2015/09/16 06:00
PHST- 2015/01/13 00:00 [received]
PHST- 2015/08/27 00:00 [accepted]
PHST- 2015/09/16 06:00 [pubmed]
PHST- 2018/01/09 06:00 [medline]
PHST- 2015/09/16 06:00 [entrez]
AID - syv064 [pii]
AID - 10.1093/sysbio/syv064 [doi]
PST - ppublish
SO  - Syst Biol. 2017 Mar 1;66(2):183-196. doi: 10.1093/sysbio/syv064.

PMID- 26330450
OWN - NLM
STAT- MEDLINE
DCOM- 20160727
LR  - 20151216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 1
DP  - 2016 Jan
TI  - How Should Genes and Taxa be Sampled for Phylogenomic Analyses with Missing Data?
      An Empirical Study in Iguanian Lizards.
PG  - 128-45
LID - 10.1093/sysbio/syv058 [doi]
AB  - Targeted sequence capture is becoming a widespread tool for generating large
      phylogenomic data sets to address difficult phylogenetic problems. However, this 
      methodology often generates data sets in which increasing the number of taxa and 
      loci increases amounts of missing data. Thus, a fundamental (but still
      unresolved) question is whether sampling should be designed to maximize sampling 
      of taxa or genes, or to minimize the inclusion of missing data cells. Here, we
      explore this question for an ancient, rapid radiation of lizards, the pleurodont 
      iguanians. Pleurodonts include many well-known clades (e.g., anoles, basilisks,
      iguanas, and spiny lizards) but relationships among families have proven
      difficult to resolve strongly and consistently using traditional sequencing
      approaches. We generated up to 4921 ultraconserved elements with sampling
      strategies including 16, 29, and 44 taxa, from 1179 to approximately 2.4 million 
      characters per matrix and approximately 30% to 60% total missing data. We then
      compared mean branch support for interfamilial relationships under these 15
      different sampling strategies for both concatenated (maximum likelihood) and
      species tree (NJst) approaches (after showing that mean branch support appears to
      be related to accuracy). We found that both approaches had the highest support
      when including loci with up to 50% missing taxa (matrices with ~40-55% missing
      data overall). Thus, our results show that simply excluding all missing data may 
      be highly problematic as the primary guiding principle for the inclusion or
      exclusion of taxa and genes. The optimal strategy was somewhat different for each
      approach, a pattern that has not been shown previously. For concatenated
      analyses, branch support was maximized when including many taxa (44) but fewer
      characters (1.1 million). For species-tree analyses, branch support was maximized
      with minimal taxon sampling (16) but many loci (4789 of 4921). We also show that 
      the choice of these sampling strategies can be critically important for
      phylogenomic analyses, since some strategies lead to demonstrably incorrect
      inferences (using the same method) that have strong statistical support. Our
      preferred estimate provides strong support for most interfamilial relationships
      in this important but phylogenetically challenging group.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Streicher, Jeffrey W
AU  - Streicher JW
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA; Department of Life Sciences, The Natural History Museum, London SW7
      5BD, UK and j.streicher@nhm.ac.uk.
FAU - Schulte, James A 2nd
AU  - Schulte JA 2nd
AD  - Department of Biology, Clarkson University, Potsdam, NY 13699, USA;
FAU - Wiens, John J
AU  - Wiens JJ
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA;
LA  - eng
PT  - Journal Article
DEP - 20150901
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Genome/genetics
MH  - Lizards/*classification/*genetics
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Sequence Analysis, DNA
OTO - NOTNLM
OT  - Missing data
OT  - Reptilia
OT  - Squamata
OT  - UCEs
OT  - phylogenomics
OT  - species tree
OT  - taxon sampling
EDAT- 2015/09/04 06:00
MHDA- 2016/07/28 06:00
CRDT- 2015/09/03 06:00
PHST- 2014/11/14 00:00 [received]
PHST- 2015/08/04 00:00 [accepted]
PHST- 2015/09/03 06:00 [entrez]
PHST- 2015/09/04 06:00 [pubmed]
PHST- 2016/07/28 06:00 [medline]
AID - syv058 [pii]
AID - 10.1093/sysbio/syv058 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jan;65(1):128-45. doi: 10.1093/sysbio/syv058. Epub 2015 Sep 1.

PMID- 26316424
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Inferring Speciation Processes from Patterns of Natural Variation in Microbial
      Genomes.
PG  - 926-35
LID - 10.1093/sysbio/syv050 [doi]
AB  - Microbial species concepts have long been the focus of contentious debate, fueled
      by technological limitations to the genetic resolution of species, by the
      daunting task of investigating phenotypic variation among individual microscopic 
      organisms, and by a lack of understanding of gene flow in reproductively asexual 
      organisms that are prone to promiscuous horizontal gene transfer. Population
      genomics, the emerging approach of analyzing the complete genomes of a multitude 
      of closely related organisms, is poised to overcome these limitations by
      providing a window into patterns of genome variation revealing the evolutionary
      processes through which species diverge. This new approach is more than just an
      extension of previous multilocus sequencing technologies, in that it provides a
      comprehensive view of interacting evolutionary processes. Here we argue that the 
      application of population genomic tools in a rigorous population genetic
      framework will help to identify the processes of microbial speciation and
      ultimately lead to a general species concept based on the unique biology and
      ecology of microorganisms.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Krause, David J
AU  - Krause DJ
AD  - Department of Microbiology, University of Illinois at Urbana-Champaign, Urbana,
      IL 61801, USA.
FAU - Whitaker, Rachel J
AU  - Whitaker RJ
AD  - Department of Microbiology, University of Illinois at Urbana-Champaign, Urbana,
      IL 61801, USA rwhitaker@life.illinois.edu.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150827
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bacteria/*classification
MH  - *Genetic Speciation
MH  - *Genetic Variation
MH  - Genome, Bacterial/genetics
MH  - Genomics
MH  - Models, Genetic
PMC - PMC4604833
OTO - NOTNLM
OT  - Barriers to gene flow
OT  - differentiation
OT  - recombination
OT  - selection
EDAT- 2015/09/01 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/08/29 06:00
PHST- 2014/04/08 00:00 [received]
PHST- 2015/06/09 00:00 [accepted]
PHST- 2015/08/29 06:00 [entrez]
PHST- 2015/09/01 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv050 [pii]
AID - 10.1093/sysbio/syv050 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):926-35. doi: 10.1093/sysbio/syv050. Epub 2015 Aug 27.

PMID- 26276158
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20151015
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Selecting Question-Specific Genes to Reduce Incongruence in Phylogenomics: A Case
      Study of Jawed Vertebrate Backbone Phylogeny.
PG  - 1104-20
LID - 10.1093/sysbio/syv059 [doi]
AB  - Incongruence between different phylogenomic analyses is the main challenge faced 
      by phylogeneticists in the genomic era. To reduce incongruence, phylogenomic
      studies normally adopt some data filtering approaches, such as reducing missing
      data or using slowly evolving genes, to improve the signal quality of data. Here,
      we assembled a phylogenomic data set of 58 jawed vertebrate taxa and 4682 genes
      to investigate the backbone phylogeny of jawed vertebrates under both
      concatenation and coalescent-based frameworks. To evaluate the efficiency of
      extracting phylogenetic signals among different data filtering methods, we chose 
      six highly intractable internodes within the backbone phylogeny of jawed
      vertebrates as our test questions. We found that our phylogenomic data set
      exhibits substantial conflicting signal among genes for these questions. Our
      analyses showed that non-specific data sets that are generated without bias
      toward specific questions are not sufficient to produce consistent results when
      there are several difficult nodes within a phylogeny. Moreover, phylogenetic
      accuracy based on non-specific data is considerably influenced by the size of
      data and the choice of tree inference methods. To address such incongruences, we 
      selected genes that resolve a given internode but not the entire phylogeny.
      Notably, not only can this strategy yield correct relationships for the question,
      but it also reduces inconsistency associated with data sizes and inference
      methods. Our study highlights the importance of gene selection in phylogenomic
      analyses, suggesting that simply using a large amount of data cannot guarantee
      correct results. Constructing question-specific data sets may be more powerful
      for resolving problematic nodes.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Chen, Meng-Yun
AU  - Chen MY
AD  - State Key Laboratory of Biocontrol, College of Ecology and Evolution, School of
      Life Sciences, Sun Yat-Sen University, Guangzhou 510006, China.
FAU - Liang, Dan
AU  - Liang D
AD  - State Key Laboratory of Biocontrol, College of Ecology and Evolution, School of
      Life Sciences, Sun Yat-Sen University, Guangzhou 510006, China.
FAU - Zhang, Peng
AU  - Zhang P
AD  - State Key Laboratory of Biocontrol, College of Ecology and Evolution, School of
      Life Sciences, Sun Yat-Sen University, Guangzhou 510006, China
      alarzhang@gmail.com zhangp35@mail.sysu.edu.cn.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150813
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Genomics
MH  - *Phylogeny
MH  - Spine/anatomy &amp; histology
MH  - Vertebrates/anatomy &amp; histology/*classification/*genetics
OTO - NOTNLM
OT  - Phylogenomic
OT  - phylogenetic signal
OT  - systematic bias
OT  - vertebrates
EDAT- 2015/08/16 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/08/16 06:00
PHST- 2015/01/12 00:00 [received]
PHST- 2015/08/10 00:00 [accepted]
PHST- 2015/08/16 06:00 [entrez]
PHST- 2015/08/16 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv059 [pii]
AID - 10.1093/sysbio/syv059 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):1104-20. doi: 10.1093/sysbio/syv059. Epub 2015 Aug 13.

PMID- 26272507
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - A Nonstationary Markov Model Detects Directional Evolution in Hymenopteran
      Morphology.
PG  - 1089-103
LID - 10.1093/sysbio/syv052 [doi]
AB  - Directional evolution has played an important role in shaping the morphological, 
      ecological, and molecular diversity of life. However, standard substitution
      models assume stationarity of the evolutionary process over the time scale
      examined, thus impeding the study of directionality. Here we explore a simple,
      nonstationary model of evolution for discrete data, which assumes that the state 
      frequencies at the root differ from the equilibrium frequencies of the
      homogeneous evolutionary process along the rest of the tree (i.e., the process is
      nonstationary, nonreversible, but homogeneous). Within this framework, we develop
      a Bayesian approach for testing directional versus stationary evolution using a
      reversible-jump algorithm. Simulations show that when only data from extant taxa 
      are available, the success in inferring directionality is strongly dependent on
      the evolutionary rate, the shape of the tree, the relative branch lengths, and
      the number of taxa. Given suitable evolutionary rates (0.1-0.5 expected
      substitutions between root and tips), accounting for directionality improves tree
      inference and often allows correct rooting of the tree without the use of an
      outgroup. As an empirical test, we apply our method to study directional
      evolution in hymenopteran morphology. We focus on three character systems: wing
      veins, muscles, and sclerites. We find strong support for a trend toward loss of 
      wing veins and muscles, while stationarity cannot be ruled out for sclerites.
      Adding fossil and time information in a total-evidence dating approach, we show
      that accounting for directionality results in more precise estimates not only of 
      the ancestral state at the root of the tree, but also of the divergence times.
      Our model relaxes the assumption of stationarity and reversibility by adding a
      minimum of additional parameters, and is thus well suited to studying the nature 
      of the evolutionary process in data sets of limited size, such as morphology and 
      ecology.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Klopfstein, Seraina
AU  - Klopfstein S
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History,
      SE-104 05 Stockholm, Sweden; The University of Adelaide, ACEBB, Adelaide SA 5005,
      Australia; Natural History Museum, Department of Invertebrates, CH-3005 Bern,
      Switzerland; klopfstein@nmbe.ch.
FAU - Vilhelmsen, Lars
AU  - Vilhelmsen L
AD  - Biosystematics, Natural History Museum of Denmark, DK-2100 Copenhagen O, Denmark.
FAU - Ronquist, Fredrik
AU  - Ronquist F
AD  - Department of Bioinformatics and Genetics, Swedish Museum of Natural History,
      SE-104 05 Stockholm, Sweden;
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150812
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Computer Simulation
MH  - Hymenoptera/*anatomy &amp; histology/*cytology
MH  - Markov Chains
MH  - *Models, Biological
PMC - PMC4604834
OTO - NOTNLM
OT  - Bayesian inference
OT  - Hymenoptera
OT  - Symphyta
OT  - continuous-time Markov model
OT  - directional selection
OT  - morphology
OT  - neutral evolution
OT  - nonstationary
OT  - positive selection
EDAT- 2015/08/15 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/08/15 06:00
PHST- 2015/01/17 00:00 [received]
PHST- 2015/07/17 00:00 [accepted]
PHST- 2015/08/15 06:00 [entrez]
PHST- 2015/08/15 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv052 [pii]
AID - 10.1093/sysbio/syv052 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):1089-103. doi: 10.1093/sysbio/syv052. Epub 2015 Aug 12.

PMID- 26254671
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20190110
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Historical Biogeography Using Species Geographical Ranges.
PG  - 1059-73
LID - 10.1093/sysbio/syv057 [doi]
AB  - Spatial variation in biodiversity is the result of complex interactions between
      evolutionary history and ecological factors. Methods in historical biogeography
      combine phylogenetic information with current species locations to infer the
      evolutionary history of a clade through space and time. A major limitation of
      most methods for historical biogeographic inference is the requirement of single 
      locations for terminal lineages, reducing contemporary species geographical
      ranges to a point in two-dimensional space. In reality, geographic ranges usually
      show complex geographic patterns, irregular shapes, or discontinuities. In this
      article, we describe a method for phylogeographic analysis using polygonal
      species geographic ranges of arbitrary complexity. By integrating the geographic 
      diversification process across species ranges, we provide a method to infer the
      geographic location of ancestors in a Bayesian framework. By modeling migration
      conditioned on a phylogenetic tree, this approach permits reconstructing the
      geographic location of ancestors through time. We apply this new method to the
      diversification of two neotropical bird genera, Trumpeters (Psophia) and
      Cinclodes ovenbirds. We demonstrate the usefulness of our method (called rase) in
      phylogeographic reconstruction of species ancestral locations and contrast our
      results with previous methods that compel researchers to reduce the distribution 
      of species to one point in space. We discuss model extensions to enable a more
      general, spatially explicit framework for historical biogeographic analysis.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Quintero, Ignacio
AU  - Quintero I
AD  - Department of Ecology and Evolutionary Biology, Yale University, 165 Prospect
      Street, New Haven, CT 06520-8106, USA; ignacio.quintero@yale.edu.
FAU - Keil, Petr
AU  - Keil P
AD  - Department of Ecology and Evolutionary Biology, Yale University, 165 Prospect
      Street, New Haven, CT 06520-8106, USA; Center for Theoretical Study, Charles
      University and Academy of Sciences of the Czech Republic, Jilska 1, 11000 Praha
      1, Czech Republic;
FAU - Jetz, Walter
AU  - Jetz W
AD  - Department of Ecology and Evolutionary Biology, Yale University, 165 Prospect
      Street, New Haven, CT 06520-8106, USA;
FAU - Crawford, Forrest W
AU  - Crawford FW
AD  - Department of Ecology and Evolutionary Biology, Yale University, 165 Prospect
      Street, New Haven, CT 06520-8106, USA; Department of Biostatistics, Yale School
      of Public Health, 60 College Street, New Haven, CT 06510, USA.
LA  - eng
GR  - KL2 TR000140/TR/NCATS NIH HHS/United States
GR  - UL1 TR000142/TR/NCATS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150808
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Animal Distribution
MH  - Animals
MH  - Birds/*classification
MH  - Phylogeny
MH  - Phylogeography/*methods
MH  - South America
PMC - PMC4838013
OTO - NOTNLM
OT  - Bayesian inference
OT  - continuous trait evolution
OT  - diversification
OT  - historical biogeography
OT  - phylogeography
OT  - species distributions
EDAT- 2015/08/10 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/08/10 06:00
PHST- 2015/09/21 00:00 [received]
PHST- 2015/07/24 00:00 [accepted]
PHST- 2015/08/10 06:00 [entrez]
PHST- 2015/08/10 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv057 [pii]
AID - 10.1093/sysbio/syv057 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):1059-73. doi: 10.1093/sysbio/syv057. Epub 2015 Aug 8.

PMID- 26254670
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20151015
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - A Skull Might Lie: Modeling Ancestral Ranges and Diet from Genes and Shape of
      Tree Squirrels.
PG  - 1074-88
LID - 10.1093/sysbio/syv054 [doi]
AB  - Tropical forests of Central and South America represent hotspots of biological
      diversity. Tree squirrels of the tribe Sciurini are an excellent model system for
      the study of tropical biodiversity as these squirrels disperse exceptional
      distances, and after colonizing the tropics of the Central and South America,
      they have diversified rapidly. Here, we compare signals from DNA sequences with
      morphological signals using pictures of skulls and computational simulations.
      Phylogenetic analyses reveal step-wise geographic divergence across the Northern 
      Hemisphere. In Central and South America, tree squirrels form two separate
      clades, which split from a common ancestor. Simulations of ancestral
      distributions show western Amazonia as the epicenter of speciation in South
      America. This finding suggests that wet tropical forests on the foothills of
      Andes possibly served as refugia of squirrel diversification during Pleistocene
      climatic oscillations. Comparison of phylogeny and morphology reveals one major
      discrepancy: Microsciurus species are a single clade morphologically but are
      polyphyletic genetically. Modeling of morphology-diet relationships shows that
      the only group of species with a direct link between skull shape and diet are the
      bark-gleaning insectivorous species of Microsciurus. This finding suggests that
      the current designation of Microsciurus as a genus is based on convergent
      ecologically driven changes in morphology.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Pecnerova, Patricia
AU  - Pecnerova P
AD  - Department of Botany and Zoology, Masaryk University, Kotlarska 2, 602 00 Brno,
      Czech Republic; Institute of Vertebrate Biology, Academy of Sciences of the Czech
      Republic, v.v.i., Kvetna 8, 603 65 Brno, Czech Republic; Department of Zoology,
      Stockholm University, 10691 Stockholm, Sweden; Department of Bioinformatics and
      Genetics, Swedish Museum of Natural History, PO Box 50007, 10405 Stockholm,
      Sweden; patricia.pecnerova@nrm.se.
FAU - Moravec, Jiri C
AU  - Moravec JC
AD  - Institute of Vertebrate Biology, Academy of Sciences of the Czech Republic,
      v.v.i., Kvetna 8, 603 65 Brno, Czech Republic; Institute of Fundamental Sciences,
      Massey University, Private Bag 11 222, Palmerston North, New Zealand; and.
FAU - Martinkova, Natalia
AU  - Martinkova N
AD  - Institute of Vertebrate Biology, Academy of Sciences of the Czech Republic,
      v.v.i., Kvetna 8, 603 65 Brno, Czech Republic; Institute of Biostatistics and
      Analyses, Masaryk University, Kamenice 3, 625 00 Brno, Czech Republic.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150808
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Adaptation, Physiological
MH  - *Animal Distribution
MH  - Animals
MH  - *Computer Simulation
MH  - *Diet
MH  - Phylogeny
MH  - *Sciuridae/anatomy &amp; histology/classification/genetics
MH  - Skull/*anatomy &amp; histology
OTO - NOTNLM
OT  - Ancestral range reconstruction
OT  - Sciurini
OT  - diet modeling
OT  - geometric morphometry
OT  - multilocus phylogeny
OT  - speciation
EDAT- 2015/08/10 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/08/10 06:00
PHST- 2015/01/08 00:00 [received]
PHST- 2015/07/29 00:00 [accepted]
PHST- 2015/08/10 06:00 [entrez]
PHST- 2015/08/10 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv054 [pii]
AID - 10.1093/sysbio/syv054 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):1074-88. doi: 10.1093/sysbio/syv054. Epub 2015 Aug 8.

PMID- 26231183
OWN - NLM
STAT- MEDLINE
DCOM- 20160727
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 1
DP  - 2016 Jan
TI  - An Efficient Independence Sampler for Updating Branches in Bayesian Markov chain 
      Monte Carlo Sampling of Phylogenetic Trees.
PG  - 161-76
LID - 10.1093/sysbio/syv051 [doi]
AB  - Sampling tree space is the most challenging aspect of Bayesian phylogenetic
      inference. The sheer number of alternative topologies is problematic by itself.
      In addition, the complex dependency between branch lengths and topology increases
      the difficulty of moving efficiently among topologies. Current tree proposals are
      fast but sample new trees using primitive transformations or re-mappings of old
      branch lengths. This reduces acceptance rates and presumably slows down
      convergence and mixing. Here, we explore branch proposals that do not rely on old
      branch lengths but instead are based on approximations of the conditional
      posterior. Using a diverse set of empirical data sets, we show that most
      conditional branch posteriors can be accurately approximated via a [Formula: see 
      text] distribution. We empirically determine the relationship between the
      logarithmic conditional posterior density, its derivatives, and the
      characteristics of the branch posterior. We use these relationships to derive an 
      independence sampler for proposing branches with an acceptance ratio of ~90% on
      most data sets. This proposal samples branches between 2x and 3x more efficiently
      than traditional proposals with respect to the effective sample size per unit of 
      runtime. We also compare the performance of standard topology proposals with
      hybrid proposals that use the new independence sampler to update those branches
      that are most affected by the topological change. Our results show that hybrid
      proposals can sometimes noticeably decrease the number of generations necessary
      for topological convergence. Inconsistent performance gains indicate that branch 
      updates are not the limiting factor in improving topological convergence for the 
      currently employed set of proposals. However, our independence sampler might be
      essential for the construction of novel tree proposals that apply more radical
      topology changes.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Aberer, Andre J
AU  - Aberer AJ
AD  - Scientific Computing Group, Heidelberg Institute for Theoretical Studies,
      Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany; andre.aberer@h-its.org.
FAU - Stamatakis, Alexandros
AU  - Stamatakis A
AD  - Scientific Computing Group, Heidelberg Institute for Theoretical Studies,
      Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany; Institute for
      Theoretical Informatics, Karlsruhe Institute of Technology, D-76131 Karlsruhe,
      Germany; and.
FAU - Ronquist, Fredrik
AU  - Ronquist F
AD  - Department of Biodiversity Informatics, Swedish Museum of Natural History, PO Box
      50007, SE-104 05 Stockholm, Sweden.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150730
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Markov Chains
MH  - *Models, Theoretical
MH  - Monte Carlo Method
MH  - *Phylogeny
PMC - PMC4678251
OTO - NOTNLM
OT  - Bayesian inference
OT  - Markov chain Monte Carlo
OT  - Newton-Raphson optimization
OT  - independence sampling
OT  - phylogenetics
OT  - proposal efficiency
OT  - tree topology proposals
EDAT- 2015/08/02 06:00
MHDA- 2016/07/28 06:00
CRDT- 2015/08/02 06:00
PHST- 2015/01/19 00:00 [received]
PHST- 2015/06/09 00:00 [accepted]
PHST- 2015/08/02 06:00 [entrez]
PHST- 2015/08/02 06:00 [pubmed]
PHST- 2016/07/28 06:00 [medline]
AID - syv051 [pii]
AID - 10.1093/sysbio/syv051 [doi]
PST - ppublish
SO  - Syst Biol. 2016 Jan;65(1):161-76. doi: 10.1093/sysbio/syv051. Epub 2015 Jul 30.

PMID- 26231182
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Automation and Evaluation of the SOWH Test with SOWHAT.
PG  - 1048-58
LID - 10.1093/sysbio/syv055 [doi]
AB  - The Swofford-Olsen-Waddell-Hillis (SOWH) test evaluates statistical support for
      incongruent phylogenetic topologies. It is commonly applied to determine if the
      maximum likelihood tree in a phylogenetic analysis is significantly different
      than an alternative hypothesis. The SOWH test compares the observed difference in
      log-likelihood between two topologies to a null distribution of differences in
      log-likelihood generated by parametric resampling. The test is a well-established
      phylogenetic method for topology testing, but it is sensitive to model
      misspecification, it is computationally burdensome to perform, and its
      implementation requires the investigator to make several decisions that each have
      the potential to affect the outcome of the test. We analyzed the effects of
      multiple factors using seven data sets to which the SOWH test was previously
      applied. These factors include a number of sample replicates, likelihood
      software, the introduction of gaps to simulated data, the use of distinct models 
      of evolution for data simulation and likelihood inference, and a suggested test
      correction wherein an unresolved "zero-constrained" tree is used to simulate
      sequence data. To facilitate these analyses and future applications of the SOWH
      test, we wrote SOWHAT, a program that automates the SOWH test. We find that
      inadequate bootstrap sampling can change the outcome of the SOWH test. The
      results also show that using a zero-constrained tree for data simulation can
      result in a wider null distribution and higher p-values, but does not change the 
      outcome of the SOWH test for most of the data sets tested here. These results
      will help others implement and evaluate the SOWH test and allow us to provide
      recommendations for future applications of the SOWH test. SOWHAT is available for
      download from https://github.com/josephryan/SOWHAT.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Church, Samuel H
AU  - Church SH
AD  - Department of Ecology and Evolutionary Biology, Brown University, Providence,
      Rhode Island, USA; church@g.harvard.edu.
FAU - Ryan, Joseph F
AU  - Ryan JF
AD  - Whitney Laboratory for Marine Biosciences, St. Augustine, Florida, USA; and Sars 
      International Centre For Marine Molecular Biology, Bergen, Norway.
FAU - Dunn, Casey W
AU  - Dunn CW
AD  - Department of Ecology and Evolutionary Biology, Brown University, Providence,
      Rhode Island, USA;
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150730
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - Data Interpretation, Statistical
MH  - *Phylogeny
MH  - Primulaceae/*classification/*genetics
MH  - *Software
PMC - PMC4604836
OTO - NOTNLM
OT  - Phylogenetics
OT  - SOWH test
OT  - topology test
EDAT- 2015/08/02 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/08/02 06:00
PHST- 2014/06/18 00:00 [received]
PHST- 2015/07/24 00:00 [accepted]
PHST- 2015/08/02 06:00 [entrez]
PHST- 2015/08/02 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv055 [pii]
AID - 10.1093/sysbio/syv055 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):1048-58. doi: 10.1093/sysbio/syv055. Epub 2015 Jul 30.

PMID- 26227865
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Short Tree, Long Tree, Right Tree, Wrong Tree: New Acquisition Bias Corrections
      for Inferring SNP Phylogenies.
PG  - 1032-47
LID - 10.1093/sysbio/syv053 [doi]
AB  - Single nucleotide polymorphisms (SNPs) are useful markers for phylogenetic
      studies owing in part to their ubiquity throughout the genome and ease of
      collection. Restriction site associated DNA sequencing (RADseq) methods are
      becoming increasingly popular for SNP data collection, but an assessment of the
      best practises for using these data in phylogenetics is lacking. We use computer 
      simulations, and new double digest RADseq (ddRADseq) data for the lizard family
      Phrynosomatidae, to investigate the accuracy of RAD loci for phylogenetic
      inference. We compare the two primary ways RAD loci are used during phylogenetic 
      analysis, including the analysis of full sequences (i.e., SNPs together with
      invariant sites), or the analysis of SNPs on their own after excluding invariant 
      sites. We find that using full sequences rather than just SNPs is preferable from
      the perspectives of branch length and topological accuracy, but not of
      computational time. We introduce two new acquisition bias corrections for dealing
      with alignments composed exclusively of SNPs, a conditional likelihood method and
      a reconstituted DNA approach. The conditional likelihood method conditions on the
      presence of variable characters only (the number of invariant sites that are
      unsampled but known to exist is not considered), while the reconstituted DNA
      approach requires the user to specify the exact number of unsampled invariant
      sites prior to the analysis. Under simulation, branch length biases increase with
      the amount of missing data for both acquisition bias correction methods, but
      branch length accuracy is much improved in the reconstituted DNA approach
      compared to the conditional likelihood approach. Phylogenetic analyses of the
      empirical data using concatenation or a coalescent-based species tree approach
      provide strong support for many of the accepted relationships among phrynosomatid
      lizards, suggesting that RAD loci contain useful phylogenetic signal across a
      range of divergence times despite the presence of missing data. Phylogenetic
      analysis of RAD loci requires careful attention to model assumptions, especially 
      if downstream analyses depend on branch lengths.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Leache, Adam D
AU  - Leache AD
AD  - Department of Biology, University of Washington, Seattle, WA 98195, USA; Burke
      Museum of Natural History and Culture, University of Washington, Seattle, WA
      98195, USA; leache@uw.edu.
FAU - Banbury, Barbara L
AU  - Banbury BL
AD  - Department of Biology, University of Washington, Seattle, WA 98195, USA;
FAU - Felsenstein, Joseph
AU  - Felsenstein J
AD  - Department of Biology, University of Washington, Seattle, WA 98195, USA;
      Department of Genome Sciences, University of Washington, Seattle, WA 98195, USA;
FAU - de Oca, Adrian Nieto-Montes
AU  - de Oca AN
AD  - Departamento de Biologia Evolutiva, Facultad de Ciencias, Universidad Nacional
      Autonoma de Mexico, Ciudad Universitaria, Mexico 04510, Distrito Federal, Mexico;
FAU - Stamatakis, Alexandros
AU  - Stamatakis A
AD  - Exelixis Laboratory, Scientific Computing Group, Heidelberg Institute for
      Theoretical Studies (HITS gGmbH), Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg,
      Germany; and Department of Informatics, Institute for Theoretical Informatics,
      Karlsruhe Institute of Technology, Am Fasanengarten 5, 76131 Karlsruhe, Germany.
LA  - eng
GR  - S10 RR027303/RR/NCRR NIH HHS/United States
GR  - S10 RR029668/RR/NCRR NIH HHS/United States
GR  - S10RR027303/RR/NCRR NIH HHS/United States
GR  - S10RR029668/RR/NCRR NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
DEP - 20150729
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - Lizards/classification/genetics
MH  - *Phylogeny
MH  - Polymorphism, Single Nucleotide/*genetics
PMC - PMC4604835
OTO - NOTNLM
OT  - Conditional likelihood
OT  - Phrynosoma
OT  - Phrynosomatidae
OT  - SVDquartets
OT  - ddRADseq
OT  - maximum likelihood
OT  - reconstituted DNA
EDAT- 2015/08/01 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/08/01 06:00
PHST- 2015/01/08 00:00 [received]
PHST- 2015/07/24 00:00 [accepted]
PHST- 2015/08/01 06:00 [entrez]
PHST- 2015/08/01 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv053 [pii]
AID - 10.1093/sysbio/syv053 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):1032-47. doi: 10.1093/sysbio/syv053. Epub 2015 Jul 29.

PMID- 26209413
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - PoMo: An Allele Frequency-Based Approach for Species Tree Estimation.
PG  - 1018-31
LID - 10.1093/sysbio/syv048 [doi]
AB  - Incomplete lineage sorting can cause incongruencies of the overall species-level 
      phylogenetic tree with the phylogenetic trees for individual genes or genomic
      segments. If these incongruencies are not accounted for, it is possible to incur 
      several biases in species tree estimation. Here, we present a simple maximum
      likelihood approach that accounts for ancestral variation and incomplete lineage 
      sorting. We use a POlymorphisms-aware phylogenetic MOdel (PoMo) that we have
      recently shown to efficiently estimate mutation rates and fixation biases from
      within and between-species variation data. We extend this model to perform
      efficient estimation of species trees. We test the performance of PoMo in several
      different scenarios of incomplete lineage sorting using simulations and compare
      it with existing methods both in accuracy and computational speed. In contrast to
      other approaches, our model does not use coalescent theory but is allele
      frequency based. We show that PoMo is well suited for genome-wide species tree
      estimation and that on such data it is more accurate than previous approaches.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - De Maio, Nicola
AU  - De Maio N
AD  - Institut fur Populationsgenetik, Vetmeduni Vienna, Wien 1210, Austria; Vienna
      Graduate School of Population Genetics, Wien, Austria; and Nuffield Department of
      Clinical Medicine, University of Oxford, Oxford OX3 7BN, UK.
FAU - Schrempf, Dominik
AU  - Schrempf D
AD  - Institut fur Populationsgenetik, Vetmeduni Vienna, Wien 1210, Austria; Vienna
      Graduate School of Population Genetics, Wien, Austria; and.
FAU - Kosiol, Carolin
AU  - Kosiol C
AD  - Institut fur Populationsgenetik, Vetmeduni Vienna, Wien 1210, Austria;
      carolin.kosiol@vetmeduni.ac.at.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150723
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - *Gene Frequency
MH  - Hominidae/classification/genetics
MH  - Mutation
MH  - *Phylogeny
MH  - Polymorphism, Genetic
PMC - PMC4604832
OTO - NOTNLM
OT  - Incomplete lineage sorting
OT  - Phylogenetics
OT  - PoMo
OT  - Species tree
EDAT- 2015/07/26 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/07/26 06:00
PHST- 2014/03/21 00:00 [received]
PHST- 2015/06/11 00:00 [accepted]
PHST- 2015/07/26 06:00 [entrez]
PHST- 2015/07/26 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv048 [pii]
AID - 10.1093/sysbio/syv048 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):1018-31. doi: 10.1093/sysbio/syv048. Epub 2015 Jul 23.

PMID- 26187295
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Resolving Evolutionary Relationships in Closely Related Species with Whole-Genome
      Sequencing Data.
PG  - 1000-17
LID - 10.1093/sysbio/syv045 [doi]
AB  - Using genetic data to resolve the evolutionary relationships of species is of
      major interest in evolutionary and systematic biology. However, reconstructing
      the sequence of speciation events, the so-called species tree, in closely related
      and potentially hybridizing species is very challenging. Processes such as
      incomplete lineage sorting and interspecific gene flow result in local gene
      genealogies that differ in their topology from the species tree, and analyses of 
      few loci with a single sequence per species are likely to produce conflicting or 
      even misleading results. To study these phenomena on a full phylogenomic scale,
      we use whole-genome sequence data from 200 individuals of four black-and-white
      flycatcher species with so far unresolved phylogenetic relationships to infer
      gene tree topologies and visualize genome-wide patterns of gene tree
      incongruence. Using phylogenetic analysis in nonoverlapping 10-kb windows, we
      show that gene tree topologies are extremely diverse and change on a very small
      physical scale. Moreover, we find strong evidence for gene flow among flycatcher 
      species, with distinct patterns of reduced introgression on the Z chromosome. To 
      resolve species relationships on the background of widespread gene tree
      incongruence, we used four complementary coalescent-based methods for species
      tree reconstruction, including complex modeling approaches that incorporate
      post-divergence gene flow among species. This allowed us to infer the most likely
      species tree with high confidence. Based on this finding, we show that regions of
      reduced effective population size, which have been suggested as particularly
      useful for species tree inference, can produce positively misleading species tree
      topologies. Our findings disclose the pitfalls of using loci potentially under
      selection as phylogenetic markers and highlight the potential of modeling
      approaches to disentangle species relationships in systems with large effective
      population sizes and post-divergence gene flow.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Nater, Alexander
AU  - Nater A
AD  - Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala
      University, Uppsala, Sweden alexander.nater@ebc.uu.se.
FAU - Burri, Reto
AU  - Burri R
AD  - Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala
      University, Uppsala, Sweden.
FAU - Kawakami, Takeshi
AU  - Kawakami T
AD  - Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala
      University, Uppsala, Sweden.
FAU - Smeds, Linnea
AU  - Smeds L
AD  - Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala
      University, Uppsala, Sweden.
FAU - Ellegren, Hans
AU  - Ellegren H
AD  - Department of Evolutionary Biology, Evolutionary Biology Centre, Uppsala
      University, Uppsala, Sweden.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150717
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Computer Simulation
MH  - Gene Flow
MH  - Genome/*genetics
MH  - *Phylogeny
MH  - Songbirds/*classification/*genetics
MH  - Species Specificity
PMC - PMC4604831
OTO - NOTNLM
OT  - Approximate Bayesian computation
OT  - demographic modeling
OT  - gene flow
OT  - gene tree
OT  - incomplete lineage sorting
OT  - introgression
OT  - phylogenomics
OT  - species tree
EDAT- 2015/07/19 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/07/19 06:00
PHST- 2015/01/14 00:00 [received]
PHST- 2015/06/24 00:00 [accepted]
PHST- 2015/07/19 06:00 [entrez]
PHST- 2015/07/19 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv045 [pii]
AID - 10.1093/sysbio/syv045 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):1000-17. doi: 10.1093/sysbio/syv045. Epub 2015 Jul 17.

PMID- 26169525
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - Phylogenetic ANOVA: The Expression Variance and Evolution Model for Quantitative 
      Trait Evolution.
PG  - 695-708
LID - 10.1093/sysbio/syv042 [doi]
AB  - A number of methods have been developed for modeling the evolution of a
      quantitative trait on a phylogeny. These methods have received renewed interest
      in the context of genome-wide studies of gene expression, in which the expression
      levels of many genes can be modeled as quantitative traits. We here develop a new
      method for joint analyses of quantitative traits within- and between species, the
      Expression Variance and Evolution (EVE) model. The model parameterizes the ratio 
      of population to evolutionary expression variance, facilitating a wide variety of
      analyses, including a test for lineage-specific shifts in expression level, and a
      phylogenetic ANOVA that can detect genes with increased or decreased ratios of
      expression divergence to diversity, analogous to the famous Hudson Kreitman
      Aguade (HKA) test used to detect selection at the DNA level. We use simulations
      to explore the properties of these tests under a variety of circumstances and
      show that the phylogenetic ANOVA is more accurate than the standard ANOVA (no
      accounting for phylogeny) sometimes used in transcriptomics. We then apply the
      EVE model to a mammalian phylogeny of 15 species typed for expression levels in
      liver tissue. We identify genes with high expression divergence between species
      as candidates for expression level adaptation, and genes with high expression
      diversity within species as candidates for expression level conservation and/or
      plasticity. Using the test for lineage-specific expression shifts, we identify
      several candidate genes for expression level adaptation on the catarrhine and
      human lineages, including genes putatively related to dietary changes in humans. 
      We compare these results to those reported previously using a model which ignores
      expression variance within species, uncovering important differences in
      performance. We demonstrate the necessity for a phylogenetic model in comparative
      expression studies and show the utility of the EVE model to detect expression
      divergence, diversity, and branch-specific shifts.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Rohlfs, Rori V
AU  - Rohlfs RV
AD  - Department of Integrative Biology, University of California Berkeley, CA, USA;
      rrohlfs@berkeley.edu.
FAU - Nielsen, Rasmus
AU  - Nielsen R
AD  - Department of Integrative Biology, University of California Berkeley, CA, USA;
      Center for Bioinformatics, University of Copenhagen, Denmark.
LA  - eng
GR  - 2R14003229-07/PHS HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150713
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Analysis of Variance
MH  - Animals
MH  - Computer Simulation
MH  - Gene Expression Regulation
MH  - Genes/genetics
MH  - Humans
MH  - Liver/metabolism
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Quantitative Trait Loci/*genetics
PMC - PMC4635652
OTO - NOTNLM
OT  - Comparative expression
OT  - Ornstein-Uhlenbeck model
OT  - expression adaptation
OT  - plasticity
OT  - population variance
EDAT- 2015/07/15 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/07/15 06:00
PHST- 2014/08/05 00:00 [received]
PHST- 2015/04/08 00:00 [accepted]
PHST- 2015/07/15 06:00 [entrez]
PHST- 2015/07/15 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv042 [pii]
AID - 10.1093/sysbio/syv042 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):695-708. doi: 10.1093/sysbio/syv042. Epub 2015 Jul 13.

PMID- 26163664
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20190110
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Species Selection Favors Dispersive Life Histories in Sea Slugs, but Higher
      Per-Offspring Investment Drives Shifts to Short-Lived Larvae.
PG  - 983-99
LID - 10.1093/sysbio/syv046 [doi]
AB  - For 40 years, paleontological studies of marine gastropods have suggested that
      species selection favors lineages with short-lived (lecithotrophic) larvae, which
      are less dispersive than long-lived (planktotrophic) larvae. Although
      lecithotrophs appeared to speciate more often and accumulate over time in some
      groups, lecithotrophy also increased extinction rates, and tests for
      state-dependent diversification were never performed. Molecular phylogenies of
      diverse groups instead suggested lecithotrophs accumulate without diversifying
      due to frequent, unidirectional character change. Although lecithotrophy has
      repeatedly originated in most phyla, no adult trait has been correlated with
      shifts in larval type. Thus, both the evolutionary origins of lecithotrophy and
      its consequences for patterns of species richness remain poorly understood. Here,
      we test hypothesized links between development mode and evolutionary rates using 
      likelihood-based methods and a phylogeny of 202 species of gastropod molluscs in 
      Sacoglossa, a clade of herbivorous sea slugs. Evolutionary quantitative genetics 
      modeling and stochastic character mapping supported 27 origins of lecithotrophy. 
      Tests for correlated evolution revealed lecithotrophy evolved more often in
      lineages investing in extra-embryonic yolk, the first adult trait associated with
      shifts in development mode across a group. However, contrary to predictions from 
      paleontological studies, species selection actually favored planktotrophy; most
      extant lecithotrophs originated through recent character change, and did not
      subsequently diversify. Increased offspring provisioning in planktotrophs thus
      favored shifts to short-lived larvae, which led to short-lived lineages over
      macroevolutionary time scales. These findings challenge long-standing assumptions
      about the effects of alternative life histories in the sea. Species selection can
      explain the long-term persistence of planktotrophy, the ancestral state in most
      clades, despite frequent transitions to lecithotrophy.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Krug, Patrick J
AU  - Krug PJ
AD  - Department of Biological Sciences, California State University, Los Angeles, CA
      90032-8201, USA; pkrug@calstatela.edu.
FAU - Vendetti, Jann E
AU  - Vendetti JE
AD  - Department of Biological Sciences, California State University, Los Angeles, CA
      90032-8201, USA;
FAU - Ellingson, Ryan A
AU  - Ellingson RA
AD  - Department of Biological Sciences, California State University, Los Angeles, CA
      90032-8201, USA;
FAU - Trowbridge, Cynthia D
AU  - Trowbridge CD
AD  - Oregon Institute of Marine Biology, University of Oregon, PO Box 5389,
      Charleston, OR 97420, USA;
FAU - Hirano, Yayoi M
AU  - Hirano YM
AD  - Coastal Branch of Natural History Museum and Institute, Chiba, 123 Yoshio,
      Katsuura, 299-5242, Japan;
FAU - Trathen, Danielle Y
AU  - Trathen DY
AD  - Department of Biological Sciences, California State University, Los Angeles, CA
      90032-8201, USA;
FAU - Rodriguez, Albert K
AU  - Rodriguez AK
AD  - Department of Biological Sciences, California State University, Los Angeles, CA
      90032-8201, USA;
FAU - Swennen, Cornelis
AU  - Swennen C
AD  - Faculty of Science and Technology, Prince of Songkla University, Pattani 94000,
      Thailand;
FAU - Wilson, Nerida G
AU  - Wilson NG
AD  - Western Australian Museum, Kew Street, Welshpool, Perth, WA 6106, Australia; and.
FAU - Valdes, Angel A
AU  - Valdes AA
AD  - Department of Biological Sciences, California State Polytechnic University,
      Pomona, CA 91768, USA.
LA  - eng
GR  - GM-61331/GM/NIGMS NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150710
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Aquatic Organisms/classification/growth &amp; development/physiology
MH  - Gastropoda/*classification/genetics/growth &amp; development/physiology
MH  - Larva
MH  - Life Cycle Stages/physiology
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Reproduction
MH  - Selection, Genetic
PMC - PMC4794617
OTO - NOTNLM
OT  - Development mode
OT  - Sacoglossa
OT  - gastropod
OT  - lecithotrophy
OT  - macroevolution
OT  - planktotrophy
OT  - species selection
EDAT- 2015/07/15 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/07/12 06:00
PHST- 2014/07/21 00:00 [received]
PHST- 2015/07/02 00:00 [accepted]
PHST- 2015/07/12 06:00 [entrez]
PHST- 2015/07/15 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv046 [pii]
AID - 10.1093/sysbio/syv046 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):983-99. doi: 10.1093/sysbio/syv046. Epub 2015 Jul 10.

PMID- 26140932
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20150818
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - The Implications of Stratigraphic Compatibility for Character Integration among
      Fossil Taxa.
PG  - 838-52
LID - 10.1093/sysbio/syv040 [doi]
AB  - Two characters are stratigraphically compatible if some phylogenies indicate that
      their combinations (state-pairs) evolved without homoplasy and in an order
      consistent with the fossil record. Simulations assuming independent character
      change indicate that we expect approximately 95% of compatible character pairs to
      also be stratigraphically compatible over a wide range of sampling regimes and
      general evolutionary models. However, two general models of rate heterogeneity
      elevate expected stratigraphic incompatibility: "early burst" models, where rates
      of change are higher among early members of a clade than among later members of
      that clade, and "integration" models, where the evolution of characters is
      correlated in some manner. Both models have important theoretical and
      methodological implications. Therefore, we examine 259 metazoan clades for
      deviations from expected stratigraphic compatibility. We do so first assuming
      independent change with equal rates of character change through time. We then
      repeat the analysis assuming independent change with separate "early" and "late" 
      rates (with "early" = the first third of taxa in a clade), with the early and
      late rates chosen to maximize the probability of the observed compatibility among
      the early taxa and then the whole clade. We single out Cambrian trilobites as a
      possible "control" group because morphometric studies suggest that integration
      patterns are not conserved among closely related species. Even allowing for early
      bursts, we see excess stratigraphic incompatibility (i.e., negative deviations)
      in significantly more clades than expected at 0.50, 0.25, and 0.05 [Formula: see 
      text] values. This pattern is particularly strong in chordates, echinoderms, and 
      arthropods. However, stratigraphic compatibility among Cambrian trilobites
      matches the expectations of integration studies, as they (unlike post-Cambrian
      trilobites) do not deviate from the expectations of independent change with no
      early bursts. Thus, these results suggest that processes such as integration
      strongly affect the data that paleontologists use to study phylogeny, disparity, 
      and rates.
CI  - Published by Oxford University Press on behalf of Society of Systematic
      Biologists 2015. This work is written by US Government employees and is in the
      public domain in the US.
FAU - Wagner, Peter J
AU  - Wagner PJ
AD  - Department of Paleobiology, National Museum of Natural History, Smithsonian
      Institution MRC-121, PO Box 37012, Washington, DC 20013, USA and wagnerpj@si.edu.
FAU - Estabrook, George F
AU  - Estabrook GF
AD  - Department of Ecology and Evolutionary Biology, University of Michigan, 2039
      Kraus Natural Science Building, 830 North University, Ann Arbor, MI 48109, USA.
LA  - eng
PT  - Journal Article
DEP - 20150703
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Fossils
MH  - Invertebrates/*classification
MH  - *Phylogeny
OTO - NOTNLM
OT  - Cambrian
OT  - correlated change
OT  - early bursts
OT  - evolvability
OT  - fossils
OT  - integration
OT  - stratigraphic compatibility
EDAT- 2015/07/05 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/07/05 06:00
PHST- 2015/01/24 00:00 [received]
PHST- 2015/06/09 00:00 [accepted]
PHST- 2015/07/05 06:00 [entrez]
PHST- 2015/07/05 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv040 [pii]
AID - 10.1093/sysbio/syv040 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):838-52. doi: 10.1093/sysbio/syv040. Epub 2015 Jul 3.

PMID- 26130236
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20151015
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Integrating Sequence Evolution into Probabilistic Orthology Analysis.
PG  - 969-82
LID - 10.1093/sysbio/syv044 [doi]
AB  - Orthology analysis, that is, finding out whether a pair of homologous genes are
      orthologs - stemming from a speciation - or paralogs - stemming from a gene
      duplication - is of central importance in computational biology, genome
      annotation, and phylogenetic inference. In particular, an orthologous
      relationship makes functional equivalence of the two genes highly likely. A major
      approach to orthology analysis is to reconcile a gene tree to the corresponding
      species tree, (most commonly performed using the most parsimonious
      reconciliation, MPR). However, most such phylogenetic orthology methods infer the
      gene tree without considering the constraints implied by the species tree and,
      perhaps even more importantly, only allow the gene sequences to influence the
      orthology analysis through the a priori reconstructed gene tree. We propose a
      sound, comprehensive Bayesian Markov chain Monte Carlo-based method,
      DLRSOrthology, to compute orthology probabilities. It efficiently sums over the
      possible gene trees and jointly takes into account the current gene tree, all
      possible reconciliations to the species tree, and the, typically strong, signal
      conveyed by the sequences. We compare our method with PrIME-GEM, a probabilistic 
      orthology approach built on a probabilistic duplication-loss model, and
      MrBayesMPR, a probabilistic orthology approach that is based on conventional
      Bayesian inference coupled with MPR. We find that DLRSOrthology outperforms these
      competing approaches on synthetic data as well as on biological data sets and is 
      robust to incomplete taxon sampling artifacts.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Ullah, Ikram
AU  - Ullah I
AD  - School of Computer Science and Communication, Science for Life Laboratory, KTH
      Royal Institute of Technology, Stockholm, Sweden;
FAU - Sjostrand, Joel
AU  - Sjostrand J
AD  - Department of Numerical Analysis and Computer Science, Science for Life
      Laboratory, Stockholm University, Stockholm, Sweden;
FAU - Andersson, Peter
AU  - Andersson P
AD  - School of Computer Science and Communication, KTH Royal Institute of Technology, 
      Stockholm, Sweden;
FAU - Sennblad, Bengt
AU  - Sennblad B
AD  - Atherosclerosis Research Unit, Dept. of Medicine, Science for Life Laboratory,
      Karolinska Institutet, Solna, Sweden;
FAU - Lagergren, Jens
AU  - Lagergren J
AD  - School of Computer Science and Communication, Science for Life Laboratory,
      Swedish e-Science Research Center (SeRC), KTH Royal Institute of Technology,
      Stockholm, Sweden; jensl@csc.kth.se.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150630
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Computer Simulation
MH  - *Phylogeny
MH  - Sequence Homology
MH  - Software
OTO - NOTNLM
OT  - Comparative genomics
OT  - gene duplication
OT  - gene loss
OT  - orthology
OT  - paralogy
OT  - phylogenetics
OT  - probabilistic modeling
OT  - relaxed molecular clock
OT  - sequence evolution
OT  - tree realization
OT  - tree reconciliation
EDAT- 2015/07/02 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/07/02 06:00
PHST- 2015/06/07 00:00 [received]
PHST- 2015/06/24 00:00 [accepted]
PHST- 2015/07/02 06:00 [entrez]
PHST- 2015/07/02 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv044 [pii]
AID - 10.1093/sysbio/syv044 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):969-82. doi: 10.1093/sysbio/syv044. Epub 2015 Jun 30.

PMID- 26117705
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20150818
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - Exploring Tree-Like and Non-Tree-Like Patterns Using Genome Sequences: An Example
      Using the Inbreeding Plant Species Arabidopsis thaliana (L.) Heynh.
PG  - 809-23
LID - 10.1093/sysbio/syv039 [doi]
AB  - Genome sequence data contain abundant information about genealogical history, but
      methods for extracting and interpreting this information are not yet fully
      developed. We analyzed genome sequences for multiple accessions of the selfing
      plant, Arabidopsis thaliana, with the goal of better understanding its
      genealogical history. As expected from accessions of the same species, we found
      much discordance between nuclear gene trees. Nonetheless, we inferred the optimal
      population tree under the assumption that all discordance is due to incomplete
      lineage sorting. To cope with the size of the data (many genes and many taxa),
      our pipeline is based on parallel computing and divides the problem into
      four-taxon trees. However, just because a population tree can be estimated does
      not mean that the assumptions of the multispecies coalescent model hold.
      Therefore, we implemented a new, nonparametric test to evaluate whether a
      population tree adequately explains the observed quartet frequencies (the
      frequencies of gene trees with each resolution of each four-taxon set). This test
      also considers other models: panmixia and a partially resolved population tree,
      that is, a tree in which some nodes are collapsed into local panmixia. We found
      that a partially resolved population tree provides the best fit to the data,
      providing evidence for tree-like structure within A. thaliana, qualitatively
      similar to what might be expected between different, closely related species.
      Further, we show that the pattern of deviation from expectations can be used to
      identify instances of introgression and detect one clear case of reticulation
      among ecotypes that have come into contact in the United Kingdom. Our study
      illustrates how we can use genome sequence data to evaluate whether phylogenetic 
      relationships are strictly tree-like or reticulating.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Stenz, Noah W M
AU  - Stenz NW
AD  - Department of Botany, University of Wisconsin-Madison, WI, 53706, USA;
FAU - Larget, Bret
AU  - Larget B
AD  - Department of Botany, University of Wisconsin-Madison, WI, 53706, USA; Department
      of of Statistics, University of Wisconsin-Madison, WI, 53706, USA.
FAU - Baum, David A
AU  - Baum DA
AD  - Department of Botany, University of Wisconsin-Madison, WI, 53706, USA;
FAU - Ane, Cecile
AU  - Ane C
AD  - Department of Botany, University of Wisconsin-Madison, WI, 53706, USA; Department
      of of Statistics, University of Wisconsin-Madison, WI, 53706, USA
      cecile.ane@wisc.edu dbaum@wisc.edu.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150627
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Arabidopsis/*classification/*genetics
MH  - Classification/*methods
MH  - Genome, Plant
MH  - Inbreeding
MH  - *Phylogeny
OTO - NOTNLM
OT  - Concordance
OT  - TICR
OT  - gene tree discordance
OT  - quartet
OT  - species concepts
EDAT- 2015/06/29 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/06/29 06:00
PHST- 2014/11/11 00:00 [received]
PHST- 2015/06/04 00:00 [accepted]
PHST- 2015/06/29 06:00 [entrez]
PHST- 2015/06/29 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv039 [pii]
AID - 10.1093/sysbio/syv039 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):809-23. doi: 10.1093/sysbio/syv039. Epub 2015 Jun 27.

PMID- 26115662
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20151015
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Detecting Adaptive Evolution in Phylogenetic Comparative Analysis Using the
      Ornstein-Uhlenbeck Model.
PG  - 953-68
LID - 10.1093/sysbio/syv043 [doi]
AB  - Phylogenetic comparative analysis is an approach to inferring evolutionary
      process from a combination of phylogenetic and phenotypic data. The last few
      years have seen increasingly sophisticated models employed in the evaluation of
      more and more detailed evolutionary hypotheses, including adaptive hypotheses
      with multiple selective optima and hypotheses with rate variation within and
      across lineages. The statistical performance of these sophisticated models has
      received relatively little systematic attention, however. We conducted an
      extensive simulation study to quantify the statistical properties of a class of
      models toward the simpler end of the spectrum that model phenotypic evolution
      using Ornstein-Uhlenbeck processes. We focused on identifying where, how, and why
      these methods break down so that users can apply them with greater understanding 
      of their strengths and weaknesses. Our analysis identifies three key determinants
      of performance: a discriminability ratio, a signal-to-noise ratio, and the number
      of taxa sampled. Interestingly, we find that model-selection power can be high
      even in regions that were previously thought to be difficult, such as when tree
      size is small. On the other hand, we find that model parameters are in many
      circumstances difficult to estimate accurately, indicating a relative paucity of 
      information in the data relative to these parameters. Nevertheless, we note that 
      accurate model selection is often possible when parameters are only weakly
      identified. Our results have implications for more sophisticated methods inasmuch
      as the latter are generalizations of the case we study.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Cressler, Clayton E
AU  - Cressler CE
AD  - Department of Biology, Queen's University, Kingston, ON K7L 3N6, Canada;
      cressler@queensu.ca.
FAU - Butler, Marguerite A
AU  - Butler MA
AD  - Department of Zoology, University of Hawai'i, Honolulu, HI 96822, USA;
FAU - King, Aaron A
AU  - King AA
AD  - Departments of Ecology &amp; Evolutionary Biology and Mathematics, University of
      Michigan, Ann Arbor, MI 48109, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150626
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Computer Simulation
MH  - Lizards/classification
MH  - *Models, Genetic
MH  - *Phylogeny
OTO - NOTNLM
OT  - Adaptation
OT  - Ornstein-Uhlenbeck
OT  - evolutionary model
OT  - model selection
OT  - phylogenetic comparative analysis
EDAT- 2015/06/28 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/06/28 06:00
PHST- 2014/08/18 00:00 [received]
PHST- 2015/02/08 00:00 [accepted]
PHST- 2015/06/28 06:00 [entrez]
PHST- 2015/06/28 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv043 [pii]
AID - 10.1093/sysbio/syv043 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):953-68. doi: 10.1093/sysbio/syv043. Epub 2015 Jun 26.

PMID- 26099258
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20150818
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - Can We Identify Genes with Increased Phylogenetic Reliability?
PG  - 824-37
LID - 10.1093/sysbio/syv041 [doi]
AB  - Topological heterogeneity among gene trees is widely observed in phylogenomic
      analyses and some of this variation is likely caused by systematic error in gene 
      tree estimation. Systematic error can be mitigated by improving models of
      sequence evolution to account for all evolutionary processes relevant to each
      gene or identifying those genes whose evolution best conforms to existing models.
      However, the best method for identifying such genes is not well established.
      Here, we ask if filtering genes according to their clock-likeness or posterior
      predictive effect size (PPES, an inference-based measure of model violation)
      improves phylogenetic reliability and congruence. We compared these approaches to
      each other, and to the common practice of filtering based on rate of evolution,
      using two different metrics. First, we compared gene-tree topologies to accepted 
      reference topologies. Second, we examined topological similarity among gene trees
      in filtered sets. Our results suggest that filtering genes based on
      clock-likeness and PPES can yield a collection of genes with more reliable
      phylogenetic signal. For the two exemplar data sets we explored, from yeast and
      amniotes, clock-likeness and PPES outperformed rate-based filtering in both
      congruence and reliability.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Doyle, Vinson P
AU  - Doyle VP
AD  - Department of Biological Sciences and Department of Plant Pathology and Crop
      Physiology, Louisiana State University, Baton Rouge, LA 70803, USA;
FAU - Young, Randee E
AU  - Young RE
AD  - Department of Biological Sciences and Department of Biology, University of Utah, 
      Salt Lake City, UT 84112, USA;
FAU - Naylor, Gavin J P
AU  - Naylor GJ
AD  - Department of Biology and Hollings Marine Laboratory, College of Charleston,
      Charleston, SC 29424, USA;
FAU - Brown, Jeremy M
AU  - Brown JM
AD  - Department of Biological Sciences and Museum of Natural Science, Louisiana State 
      University, Baton Rouge, LA 70803, USA jembrown@lsu.edu.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150622
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Ascomycota/classification/genetics
MH  - Computer Simulation
MH  - Humans
MH  - *Models, Genetic
MH  - *Phylogeny
OTO - NOTNLM
OT  - molecular clock
OT  - phylogenomics
OT  - posterior prediction
OT  - rate of evolution
OT  - systematic error
EDAT- 2015/06/24 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/06/24 06:00
PHST- 2014/11/26 00:00 [received]
PHST- 2015/06/09 00:00 [accepted]
PHST- 2015/06/24 06:00 [entrez]
PHST- 2015/06/24 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv041 [pii]
AID - 10.1093/sysbio/syv041 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):824-37. doi: 10.1093/sysbio/syv041. Epub 2015 Jun 22.

PMID- 26070685
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - Which Phylogenetic Networks are Merely Trees with Additional Arcs?
PG  - 768-77
LID - 10.1093/sysbio/syv037 [doi]
AB  - A binary phylogenetic network may or may not be obtainable from a tree by the
      addition of directed edges (arcs) between tree arcs. Here, we establish a precise
      and easily tested criterion (based on "2-SAT") that efficiently determines
      whether or not any given network can be realized in this way. Moreover, the proof
      provides a polynomial-time algorithm for finding one or more trees (when they
      exist) on which the network can be based. A number of interesting consequences
      are presented as corollaries; these lead to some further relevant questions and
      observations, which we outline in the conclusion.
CI  - (c) The Author(s) 2015. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - Francis, Andrew R
AU  - Francis AR
AD  - Centre for Research in Mathematics, School of Computing, Engineering and
      Mathematics, University of Western Sydney, Australia;
FAU - Steel, Mike
AU  - Steel M
AD  - Biomathematics Research Centre, University of Canterbury, New Zealand
      mike.steel@canterbury.ac.nz.
LA  - eng
PT  - Journal Article
DEP - 20150611
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - *Classification
MH  - Computer Simulation/standards
MH  - *Phylogeny
PMC - PMC4538883
OTO - NOTNLM
OT  - 2-SAT
OT  - Algorithm
OT  - Antichain
OT  - Phylogenetic network
OT  - phylogenetic tree
OT  - reticulate evolution
EDAT- 2015/06/14 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/06/14 06:00
PHST- 2015/04/13 00:00 [received]
PHST- 2015/05/20 00:00 [accepted]
PHST- 2015/06/14 06:00 [entrez]
PHST- 2015/06/14 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv037 [pii]
AID - 10.1093/sysbio/syv037 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):768-77. doi: 10.1093/sysbio/syv037. Epub 2015 Jun 11.

PMID- 26048340
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - Species-Level Phylogeny and Polyploid Relationships in Hordeum (Poaceae) Inferred
      by Next-Generation Sequencing and In Silico Cloning of Multiple Nuclear Loci.
PG  - 792-808
LID - 10.1093/sysbio/syv035 [doi]
AB  - Polyploidization is an important speciation mechanism in the barley genus
      Hordeum. To analyze evolutionary changes after allopolyploidization, knowledge of
      parental relationships is essential. One chloroplast and 12 nuclear single-copy
      loci were amplified by polymerase chain reaction (PCR) in all Hordeum plus six
      out-group species. Amplicons from each of 96 individuals were pooled, sheared,
      labeled with individual-specific barcodes and sequenced in a single run on a 454 
      platform. Reference sequences were obtained by cloning and Sanger sequencing of
      all loci for nine supplementary individuals. The 454 reads were assembled into
      contigs representing the 13 loci and, for polyploids, also homoeologues.
      Phylogenetic analyses were conducted for all loci separately and for a
      concatenated data matrix of all loci. For diploid taxa, a Bayesian concordance
      analysis and a coalescent-based dated species tree was inferred from all gene
      trees. Chloroplast matK was used to determine the maternal parent in
      allopolyploid taxa. The relative performance of different multilocus analyses in 
      the presence of incomplete lineage sorting and hybridization was also assessed.
      The resulting multilocus phylogeny reveals for the first time species phylogeny
      and progenitor-derivative relationships of all di- and polyploid Hordeum taxa
      within a single analysis. Our study proves that it is possible to obtain a
      multilocus species-level phylogeny for di- and polyploid taxa by combining PCR
      with next-generation sequencing, without cloning and without creating a heavy
      load of sequence data.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Brassac, Jonathan
AU  - Brassac J
AD  - Institute of Plant Genetics and Crop Plant Research (IPK), D-06466 Gatersleben,
      Germany; brassac@ipk-gatersleben.de.
FAU - Blattner, Frank R
AU  - Blattner FR
AD  - Institute of Plant Genetics and Crop Plant Research (IPK), D-06466 Gatersleben,
      Germany; German Centre for Integrative Biodiversity Research (iDiv)
      Halle-Jena-Leipzig, D-04103 Leipzig, Germany.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150605
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
SB  - IM
MH  - Classification/*methods
MH  - DNA, Chloroplast/genetics
MH  - Genes, Plant/genetics
MH  - High-Throughput Nucleotide Sequencing
MH  - Hordeum/*classification/*genetics
MH  - *Phylogeny
MH  - *Polyploidy
PMC - PMC4538882
OTO - NOTNLM
OT  - Evolution
OT  - Hordeum
OT  - in silico cloning
OT  - multispecies coalescent
OT  - nuclear single-copy genes
OT  - phylogeny
OT  - polyploidy
OT  - systematics
EDAT- 2015/06/07 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/06/07 06:00
PHST- 2014/11/18 00:00 [received]
PHST- 2015/06/02 00:00 [accepted]
PHST- 2015/06/07 06:00 [entrez]
PHST- 2015/06/07 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv035 [pii]
AID - 10.1093/sysbio/syv035 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):792-808. doi: 10.1093/sysbio/syv035. Epub 2015 Jun 5.

PMID- 26031838
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - Current Methods for Automated Filtering of Multiple Sequence Alignments
      Frequently Worsen Single-Gene Phylogenetic Inference.
PG  - 778-91
LID - 10.1093/sysbio/syv033 [doi]
AB  - Phylogenetic inference is generally performed on the basis of multiple sequence
      alignments (MSA). Because errors in an alignment can lead to errors in tree
      estimation, there is a strong interest in identifying and removing unreliable
      parts of the alignment. In recent years several automated filtering approaches
      have been proposed, but despite their popularity, a systematic and comprehensive 
      comparison of different alignment filtering methods on real data has been
      lacking. Here, we extend and apply recently introduced phylogenetic tests of
      alignment accuracy on a large number of gene families and contrast the
      performance of unfiltered versus filtered alignments in the context of
      single-gene phylogeny reconstruction. Based on multiple genome-wide empirical and
      simulated data sets, we show that the trees obtained from filtered MSAs are on
      average worse than those obtained from unfiltered MSAs. Furthermore, alignment
      filtering often leads to an increase in the proportion of well-supported branches
      that are actually wrong. We confirm that our findings hold for a wide range of
      parameters and methods. Although our results suggest that light filtering (up to 
      20% of alignment positions) has little impact on tree accuracy and may save some 
      computation time, contrary to widespread practice, we do not generally recommend 
      the use of current alignment filtering methods for phylogenetic inference. By
      providing a way to rigorously and systematically measure the impact of filtering 
      on alignments, the methodology set forth here will guide the development of
      better filtering algorithms.
CI  - (c) The Author(s) 2015. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - Tan, Ge
AU  - Tan G
AD  - Department of Computer Science, ETH Zurich, Universitatstr. 6, 8092 Zurich,
      Switzerland, Department of Molecular Sciences, Institute of Clinical Sciences,
      Faculty of Medicine, Imperial College London, London, UK; MRC Clinical Sciences
      Centre, London W12 0NN, UK;
FAU - Muffato, Matthieu
AU  - Muffato M
AD  - European Molecular Biology Laboratory, European Bioinformatics Institute,
      Hinxton, Cambridge, CB10 1SD, UK;
FAU - Ledergerber, Christian
AU  - Ledergerber C
AD  - Department of Computer Science, ETH Zurich, Universitatstr. 6, 8092 Zurich,
      Switzerland.
FAU - Herrero, Javier
AU  - Herrero J
AD  - European Molecular Biology Laboratory, European Bioinformatics Institute,
      Hinxton, Cambridge, CB10 1SD, UK; University College London, Gower St, London
      WC1E 6BT, UK; and.
FAU - Goldman, Nick
AU  - Goldman N
AD  - European Molecular Biology Laboratory, European Bioinformatics Institute,
      Hinxton, Cambridge, CB10 1SD, UK;
FAU - Gil, Manuel
AU  - Gil M
AD  - Institute of Molecular Life Sciences, University of Zurich, Winterthurerstr. 190 
      , 8057 Zurich, Switzerland; and Swiss Institute of Bioinformatics,
      Universitatstr. 6, 8092 Zurich, Switzerland.
FAU - Dessimoz, Christophe
AU  - Dessimoz C
AD  - University College London, Gower St, London WC1E 6BT, UK; and European Molecular 
      Biology Laboratory, European Bioinformatics Institute, Hinxton, Cambridge, CB10
      1SD, UK; c.dessimoz@ucl.ac.uk.
LA  - eng
GR  - 095908/Wellcome Trust/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150601
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Classification/*methods
MH  - Genome/genetics
MH  - *Phylogeny
MH  - Reproducibility of Results
MH  - Sequence Alignment
PMC - PMC4538881
OTO - NOTNLM
OT  - alignment filtering
OT  - alignment trimming
OT  - molecular phylogeny
OT  - multiple sequence alignment
OT  - phylogenetic inference
OT  - phylogenetics
OT  - phylogeny
EDAT- 2015/06/03 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/06/03 06:00
PHST- 2014/12/03 00:00 [received]
PHST- 2015/05/26 00:00 [accepted]
PHST- 2015/06/03 06:00 [entrez]
PHST- 2015/06/03 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv033 [pii]
AID - 10.1093/sysbio/syv033 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):778-91. doi: 10.1093/sysbio/syv033. Epub 2015 Jun 1.

PMID- 26018570
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Toward Synthesizing Our Knowledge of Morphology: Using Ontologies and Machine
      Reasoning to Extract Presence/Absence Evolutionary Phenotypes across Studies.
PG  - 936-52
LID - 10.1093/sysbio/syv031 [doi]
AB  - The reality of larger and larger molecular databases and the need to integrate
      data scalably have presented a major challenge for the use of phenotypic data.
      Morphology is currently primarily described in discrete publications, entrenched 
      in noncomputer readable text, and requires enormous investments of time and
      resources to integrate across large numbers of taxa and studies. Here we present 
      a new methodology, using ontology-based reasoning systems working with the
      Phenoscape Knowledgebase (KB; kb.phenoscape.org), to automatically integrate
      large amounts of evolutionary character state descriptions into a synthetic
      character matrix of neomorphic (presence/absence) data. Using the KB, which
      includes more than 55 studies of sarcopterygian taxa, we generated a synthetic
      supermatrix of 639 variable characters scored for 1051 taxa, resulting in over
      145,000 populated cells. Of these characters, over 76% were made variable through
      the addition of inferred presence/absence states derived by machine reasoning
      over the formal semantics of the source ontologies. Inferred data reduced the
      missing data in the variable character-subset from 98.5% to 78.2%. Machine
      reasoning also enables the isolation of conflicts in the data, that is, cells
      where both presence and absence are indicated; reports regarding conflicting data
      provenance can be generated automatically. Further, reasoning enables
      quantification and new visualizations of the data, here for example, allowing
      identification of character space that has been undersampled across the
      fin-to-limb transition. The approach and methods demonstrated here to compute
      synthetic presence/absence supermatrices are applicable to any taxonomic and
      phenotypic slice across the tree of life, providing the data are semantically
      annotated. Because such data can also be linked to model organism genetics
      through computational scoring of phenotypic similarity, they open a rich set of
      future research questions into phenotype-to-genome relationships.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Dececchi, T Alexander
AU  - Dececchi TA
AD  - Department of Biology, University of South Dakota, Vermillion, SD 57069, USA;
      alex.dececchi@usd.edu pmabee@usd.edu.
FAU - Balhoff, James P
AU  - Balhoff JP
AD  - National Evolutionary Synthesis Center, Durham, NC 27705, USA; University of
      North Carolina, Chapel Hill, NC 27599, USA;
FAU - Lapp, Hilmar
AU  - Lapp H
AD  - National Evolutionary Synthesis Center, Durham, NC 27705, USA; Center for
      Genomics and Computational Biology, Duke University, Durham, NC 27708, USA.
FAU - Mabee, Paula M
AU  - Mabee PM
AD  - Department of Biology, University of South Dakota, Vermillion, SD 57069, USA;
      alex.dececchi@usd.edu pmabee@usd.edu.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150526
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Amphibians/anatomy &amp; histology/classification
MH  - Animals
MH  - Biological Evolution
MH  - *Biological Ontologies
MH  - Classification
MH  - Computational Biology/*methods
MH  - Data Interpretation, Statistical
MH  - *Phenotype
PMC - PMC4604830
OTO - NOTNLM
OT  - character conflict
OT  - evolutionary mapping
OT  - inference
OT  - missing data
OT  - morphological character
OT  - ontology
OT  - phenotype
OT  - supermatrix
EDAT- 2015/05/29 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/05/29 06:00
PHST- 2014/11/24 00:00 [received]
PHST- 2015/05/20 00:00 [accepted]
PHST- 2015/05/29 06:00 [entrez]
PHST- 2015/05/29 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syv031 [pii]
AID - 10.1093/sysbio/syv031 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):936-52. doi: 10.1093/sysbio/syv031. Epub 2015 May 26.

PMID- 26012872
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20150818
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - Incompatible Ages for Clearwing Butterflies Based on Alternative Secondary
      Calibrations.
PG  - 752-67
LID - 10.1093/sysbio/syv032 [doi]
AB  - The recent publication of a time-tree for the plant family Solanaceae
      (nightshades) provides the opportunity to use independent calibrations to test
      divergence times previously inferred for the diverse Neotropical butterfly tribe 
      Ithomiini. Ithomiini includes clades that are obligate herbivores of Solanaceae, 
      with some genera feeding on only one genus. We used 8 calibrations extracted from
      the plant tree in a new relaxed molecular-clock analysis to produce an
      alternative temporal framework for the diversification of ithomiines. We compared
      the resulting age estimates to: (i) a time-tree obtained using 7 secondary
      calibrations from the Nymphalidae tree of Wahlberg et al. (2009), and (ii)
      Wahlberg et al.'s (2009) original age estimates for the same clades. We found
      that Bayesian clock estimates were rather sensitive to a variety of analytical
      parameters, including taxon sampling. Regardless of this sensitivity however,
      ithomiine divergence times calibrated with the ages of nightshades were always on
      average half the age of previous estimates. Younger dates for ithomiine clades
      appear to fit better with factors long suggested to have promoted diversification
      of the group such as the uplifting of the Andes, in the case of montane genera.
      Alternatively, if ithomiines are as old as previous estimates suggest, the recent
      ages inferred for the diversification of Solanaceae seem likely to be seriously
      underestimated. Our study exemplifies the difficulty of testing hypotheses of
      divergence times and of choosing between alternative dating scenarios, and shows 
      that age estimates based on seemingly plausible calibrations may be grossly
      incongruent.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Garzon-Orduna, Ivonne J
AU  - Garzon-Orduna IJ
AD  - Evolution and Ecology Group, Department of Biology, Middle Tennessee State
      University, Murfreesboro, TN 37132, USA; ivonne.garzon@gmail.com.
FAU - Silva-Brandao, Karina L
AU  - Silva-Brandao KL
AD  - Centro de Energia Nuclear na Agricultura, Universidade de Sao Paulo, Piracicaba, 
      Sao Paulo, Brazil;
FAU - Willmott, Keith R
AU  - Willmott KR
AD  - McGuire Center for Lepidoptera and Biodiversity, Florida Museum of Natural
      History, University of Florida, Gainesville, FL 32611, USA; and.
FAU - Freitas, Andre V L
AU  - Freitas AV
AD  - Departamento de Biologia Animal and Museu de Zoologia Instituto de Biologia,
      Universidade Estadual de Campinas, Campinas, Sao Paulo, Brazil.
FAU - Brower, Andrew V Z
AU  - Brower AV
AD  - Evolution and Ecology Group, Department of Biology, Middle Tennessee State
      University, Murfreesboro, TN 37132, USA;
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150525
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Butterflies/*classification
MH  - Classification/*methods
MH  - Fossils
MH  - *Phylogeny
MH  - Solanaceae/classification
MH  - Time
OTO - NOTNLM
OT  - Danainae
OT  - Ithomiini
OT  - Nymphalidae
OT  - Solanaceae
OT  - asynchronous association
OT  - crown ages
OT  - dating
OT  - herbivory
OT  - hostplant
OT  - molecular clock
OT  - secondary calibrations
EDAT- 2015/05/28 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/05/28 06:00
PHST- 2015/02/20 00:00 [received]
PHST- 2015/05/20 00:00 [accepted]
PHST- 2015/05/28 06:00 [entrez]
PHST- 2015/05/28 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv032 [pii]
AID - 10.1093/sysbio/syv032 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):752-67. doi: 10.1093/sysbio/syv032. Epub 2015 May 25.

PMID- 26012871
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20150818
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - Monte Carlo Strategies for Selecting Parameter Values in Simulation Experiments.
PG  - 741-51
LID - 10.1093/sysbio/syv030 [doi]
AB  - Simulation experiments are used widely throughout evolutionary biology and
      bioinformatics to compare models, promote methods, and test hypotheses. The
      biggest practical constraint on simulation experiments is the computational
      demand, particularly as the number of parameters increases. Given the
      extraordinary success of Monte Carlo methods for conducting inference in
      phylogenetics, and indeed throughout the sciences, we investigate ways in which
      Monte Carlo framework can be used to carry out simulation experiments more
      efficiently. The key idea is to sample parameter values for the experiments,
      rather than iterate through them exhaustively. Exhaustive analyses become
      completely infeasible when the number of parameters gets too large, whereas
      sampled approaches can fare better in higher dimensions. We illustrate the
      framework with applications to phylogenetics and genetic archaeology.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Leigh, Jessica W
AU  - Leigh JW
AD  - Department of Mathematics and Statistics, University of Otago, P.O. Box 56,
      Dunedin 9054, New Zealand jleigh@maths.otago.ac.nz.
FAU - Bryant, David
AU  - Bryant D
AD  - Department of Mathematics and Statistics, University of Otago, P.O. Box 56,
      Dunedin 9054, New Zealand.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150525
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - *Monte Carlo Method
MH  - *Phylogeny
OTO - NOTNLM
OT  - Importance sampling
OT  - Markov chain Monte Carlo
OT  - phylogenetic analysis
OT  - plant domestication
OT  - simulation
EDAT- 2015/05/28 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/05/28 06:00
PHST- 2014/07/27 00:00 [received]
PHST- 2015/05/12 00:00 [accepted]
PHST- 2015/05/28 06:00 [entrez]
PHST- 2015/05/28 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv030 [pii]
AID - 10.1093/sysbio/syv030 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):741-51. doi: 10.1093/sysbio/syv030. Epub 2015 May 25.

PMID- 25999395
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20150818
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - Impacts of Terraces on Phylogenetic Inference.
PG  - 709-26
LID - 10.1093/sysbio/syv024 [doi]
AB  - Terraces are sets of trees with precisely the same likelihood or parsimony score,
      which can be induced by missing sequences in partitioned multi-locus phylogenetic
      data matrices. The potentially large set of trees on a terrace can be
      characterized by enumeration algorithms or consensus methods that exploit the
      pattern of partial taxon coverage in the data, independent of the sequence data
      themselves. Terraces can add ambiguity and complexity to phylogenetic inference, 
      particularly in settings where inference is already challenging: data sets with
      many taxa and relatively few loci. In this article we present five new findings
      about terraces and their impacts on phylogenetic inference. First, we clarify
      assumptions about partitioning scheme model parameters that are necessary for the
      existence of terraces. Second, we explore the dependence of terrace size on
      partitioning scheme and indicate how to find the partitioning scheme associated
      with the largest terrace containing a given tree. Third, we highlight the impact 
      of terrace size on bootstrap estimates of confidence limits in clades, and
      characterize the surprising result that the bootstrap proportion for a clade, as 
      it is usually calculated, can be entirely determined by the frequency of
      bipartitions on a terrace, with some bipartitions receiving high support even
      when incorrect. Fourth, we dissect some effects of prior distributions of edge
      lengths on the computed posterior probabilities of clades on terraces, to
      understand an example in which long edges "attract" each other in Bayesian
      inference. Fifth, we describe how assuming relationships between edge-lengths of 
      different loci, as an attempt to avoid terraces, can also be problematic when
      taxon coverage is partial, specifically when heterotachy is present. Finally, we 
      discuss strategies for remediation of some of these problems. One promising
      approach finds a minimal set of taxa which, when deleted from the data matrix,
      reduces the size of a terrace to a single tree.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Sanderson, Michael J
AU  - Sanderson MJ
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA; sanderm@email.arizona.edu.
FAU - McMahon, Michelle M
AU  - McMahon MM
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA;
FAU - Stamatakis, Alexandros
AU  - Stamatakis A
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA; Scientific Computing Group, Heidelberg Institute for Theoretical
      Studies, Heidelberg 69118, Germany; Institute of Theoretical Informatics,
      Karlsruhe Institute of Technology, Karlsruhe 76131, Germany;
FAU - Zwickl, Derrick J
AU  - Zwickl DJ
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA;
FAU - Steel, Mike
AU  - Steel M
AD  - Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      85721, USA;
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150520
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation/*standards
MH  - Models, Genetic
MH  - *Phylogeny
OTO - NOTNLM
OT  - Bootstrap
OT  - partitioned model
OT  - phylogenetics
OT  - posterior probability
OT  - terrace
EDAT- 2015/05/23 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/05/23 06:00
PHST- 2014/09/26 00:00 [received]
PHST- 2015/04/15 00:00 [accepted]
PHST- 2015/05/23 06:00 [entrez]
PHST- 2015/05/23 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv024 [pii]
AID - 10.1093/sysbio/syv024 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):709-26. doi: 10.1093/sysbio/syv024. Epub 2015 May 20.

PMID- 25995065
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20181202
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - The Root of Flowering Plants and Total Evidence.
PG  - 879-91
LID - 10.1093/sysbio/syv028 [doi]
AB  - Support for Amborella as the sole survivor of an evolutionary lineage that is
      sister to all other angiosperms comes from positions in DNA multiple-sequence
      alignments that have a poor fit to time-reversible substitution models. These
      sites exhibit significant levels of homoplasy, compositional heterogeneity, and
      strong heterotachy. We report phylogenetic analyses with observed, randomized,
      and simulated data which show there is little or no expectation that these sites 
      provide useful information for understanding relationships among basal
      angiosperms. Their inclusion in phylogenetic analyses leads to a long-branch
      attraction artifact that favors Amborella as sister to other angiosperms in
      reconstructed phylogenies. Using parametric simulations, we show that sites in
      chloroplast sequences that exhibit less homoplasy between angiosperms and
      gymnosperms provide more reliable information for inferring basal angiosperm
      relationships. We confirm our earlier findings that the basal angiosperm
      Amborella is most closely related to aquatic herbs. Our current and previously
      reported (Goremykin et al. 2013) analyses highlight an essential aspect of the
      total evidence approach to phylogenetic inference. They suggest that data
      partitioning aimed at identifying components of the data that better fit
      evolutionary models is a more reliable approach to phylogeny reconstruction at
      deep taxonomic levels.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Goremykin, V V
AU  - Goremykin VV
AD  - FEM Research and Innovation Center, Via E. Mach 1, 38010 San Michele all'Adige
      (TN), Italy; Vadim.Goremykin@fmach.it.
FAU - Nikiforova, S V
AU  - Nikiforova SV
AD  - FEM Research and Innovation Center, Via E. Mach 1, 38010 San Michele all'Adige
      (TN), Italy;
FAU - Cavalieri, D
AU  - Cavalieri D
AD  - FEM Research and Innovation Center, Via E. Mach 1, 38010 San Michele all'Adige
      (TN), Italy;
FAU - Pindo, M
AU  - Pindo M
AD  - FEM Research and Innovation Center, Via E. Mach 1, 38010 San Michele all'Adige
      (TN), Italy;
FAU - Lockhart, Peter
AU  - Lockhart P
AD  - Institute of Molecular Biosciences, Massey University, Palmerston North, New
      Zealand; and Institute of Fundamental Sciences, Massey University, Palmerston
      North, New Zealand.
LA  - eng
PT  - Journal Article
PT  - Comment
DEP - 20150520
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Chloroplast)
SB  - IM
CON - Syst Biol. 2013 Jan 1;62(1):50-61. PMID: 22851550
CON - Syst Biol. 2014 May;63(3):368-82. PMID: 24391149
MH  - Computer Simulation
MH  - DNA, Chloroplast/genetics
MH  - Genome, Plant/genetics
MH  - Magnoliopsida/*classification/genetics
MH  - *Phylogeny
OTO - NOTNLM
OT  - Amborella
OT  - angiosperm origins
OT  - model fit
OT  - parametric simulations
OT  - systematic error
OT  - total evidence approach
EDAT- 2015/05/23 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/05/22 06:00
PHST- 2014/10/13 00:00 [received]
PHST- 2015/05/05 00:00 [accepted]
PHST- 2015/05/22 06:00 [entrez]
PHST- 2015/05/23 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv028 [pii]
AID - 10.1093/sysbio/syv028 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):879-91. doi: 10.1093/sysbio/syv028. Epub 2015 May 20.

PMID- 25979143
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20150818
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - The Challenges of Resolving a Rapid, Recent Radiation: Empirical and Simulated
      Phylogenomics of Philippine Shrews.
PG  - 727-40
LID - 10.1093/sysbio/syv029 [doi]
AB  - Phylogenetic relationships in recent, rapid radiations can be difficult to
      resolve due to incomplete lineage sorting and reliance on genetic markers that
      evolve slowly relative to the rate of speciation. By incorporating hundreds to
      thousands of unlinked loci, phylogenomic analyses have the potential to mitigate 
      these difficulties. Here, we attempt to resolve phylogenetic relationships among 
      eight shrew species (genus Crocidura) from the Philippines, a phylogenetic
      problem that has proven intractable with small (&lt; 10 loci) data sets. We
      sequenced hundreds of ultraconserved elements and whole mitochondrial genomes in 
      these species and estimated phylogenies using concatenation, summary coalescent, 
      and hierarchical coalescent methods. The concatenated approach recovered a
      maximally supported and fully resolved tree. In contrast, the coalescent-based
      approaches produced similar topologies, but each had several poorly supported
      nodes. Using simulations, we demonstrate that the concatenated tree could be
      positively misleading. Our simulations also show that the tree shape we tend to
      infer, which involves a series of short internal branches, is difficult to
      resolve, even if substitution models are known and multiple individuals per
      species are sampled. As such, the low support we obtained for backbone
      relationships in our coalescent-based inferences reflects a real and appropriate 
      lack of certainty. Our results illuminate the challenges of estimating a
      bifurcating tree in a rapid and recent radiation, providing a rare empirical
      example of a nearly simultaneous series of speciation events in a terrestrial
      animal lineage as it spreads across an oceanic archipelago.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Giarla, Thomas C
AU  - Giarla TC
AD  - Museum of Natural Science and Department of Biological Sciences, Louisiana State 
      University, Baton Rouge, LA 70803, USA giarla@lsu.edu.
FAU - Esselstyn, Jacob A
AU  - Esselstyn JA
AD  - Museum of Natural Science and Department of Biological Sciences, Louisiana State 
      University, Baton Rouge, LA 70803, USA.
LA  - eng
SI  - GENBANK/KR537873
SI  - GENBANK/KR537874
SI  - GENBANK/KR537875
SI  - GENBANK/KR537876
SI  - GENBANK/KR537877
SI  - GENBANK/KR537878
SI  - GENBANK/KR537879
SI  - GENBANK/KR537880
SI  - GENBANK/KR537881
SI  - GENBANK/KR537882
SI  - GENBANK/KR537883
SI  - GENBANK/KR537884
SI  - GENBANK/KR537885
SI  - GENBANK/KR537886
SI  - GENBANK/KR537887
SI  - GENBANK/KR537888
SI  - GENBANK/KR537889
SI  - GENBANK/KR537890
SI  - GENBANK/KR537891
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150514
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Computer Simulation/*standards
MH  - Conserved Sequence
MH  - Genetic Speciation
MH  - Genome, Mitochondrial/genetics
MH  - Molecular Sequence Data
MH  - *Phylogeny
MH  - Shrews/*classification
OTO - NOTNLM
OT  - Coalescence
OT  - Crocidura
OT  - Philippines
OT  - SNPs
OT  - Soricidae
OT  - concatenation
OT  - species tree
OT  - ultraconserved elements
EDAT- 2015/05/17 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/05/17 06:00
PHST- 2015/02/24 00:00 [received]
PHST- 2015/05/07 00:00 [accepted]
PHST- 2015/05/17 06:00 [entrez]
PHST- 2015/05/17 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv029 [pii]
AID - 10.1093/sysbio/syv029 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):727-40. doi: 10.1093/sysbio/syv029. Epub 2015 May 14.

PMID- 25944476
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - Heterogeneous Rates of Molecular Evolution and Diversification Could Explain the 
      Triassic Age Estimate for Angiosperms.
PG  - 869-78
LID - 10.1093/sysbio/syv027 [doi]
AB  - Dating analyses based on molecular data imply that crown angiosperms existed in
      the Triassic, long before their undisputed appearance in the fossil record in the
      Early Cretaceous. Following a re-analysis of the age of angiosperms using updated
      sequences and fossil calibrations, we use a series of simulations to explore the 
      possibility that the older age estimates are a consequence of (i) major shifts in
      the rate of sequence evolution near the base of the angiosperms and/or (ii) the
      representative taxon sampling strategy employed in such studies. We show that
      both of these factors do tend to yield substantially older age estimates. These
      analyses do not prove that younger age estimates based on the fossil record are
      correct, but they do suggest caution in accepting the older age estimates
      obtained using current relaxed-clock methods. Although we have focused here on
      the angiosperms, we suspect that these results will shed light on dating
      discrepancies in other major clades.
CI  - (c)The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Beaulieu, Jeremy M
AU  - Beaulieu JM
AD  - National Institute for Mathematical and Biological Synthesis (NIMBioS);
      jbeaulieu@nimbios.org.
FAU - O'Meara, Brian C
AU  - O'Meara BC
AD  - Department of Ecology and Evolutionary Biology, University of Tennessee,
      Knoxville, TN 37996-1610, USA;
FAU - Crane, Peter
AU  - Crane P
AD  - School of Forestry and Environmental Studies, Yale University, 195 Prospect
      Street, New Haven, CT 06511, USA; and.
FAU - Donoghue, Michael J
AU  - Donoghue MJ
AD  - Department of Ecology and Evolutionary Biology, Yale University, PO Box 208106,
      New Haven, CT 10620, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150504
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Computer Simulation
MH  - *Evolution, Molecular
MH  - Fossils
MH  - Magnoliopsida/*classification/*genetics
MH  - Models, Genetic
MH  - *Phylogeny
MH  - Time
OTO - NOTNLM
OT  - Angiosperms
OT  - birth-death
OT  - divergence times
OT  - land plants
OT  - rates of molecular evolution
EDAT- 2015/05/07 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/05/07 06:00
PHST- 2015/03/10 00:00 [received]
PHST- 2015/04/28 00:00 [accepted]
PHST- 2015/05/07 06:00 [entrez]
PHST- 2015/05/07 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv027 [pii]
AID - 10.1093/sysbio/syv027 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):869-78. doi: 10.1093/sysbio/syv027. Epub 2015 May 4.

PMID- 25944475
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - A Falsification of the Citation Impediment in the Taxonomic Literature.
PG  - 860-8
LID - 10.1093/sysbio/syv026 [doi]
AB  - Current science evaluation still relies on citation performance, despite
      criticisms of purely bibliometric research assessments. Biological taxonomy
      suffers from a drain of knowledge and manpower, with poor citation performance
      commonly held as one reason for this impediment. But is there really such a
      citation impediment in taxonomy? We compared the citation numbers of 306
      taxonomic and 2291 non-taxonomic research articles (2009-2012) on mosses,
      orchids, ciliates, ants, and snakes, using Web of Science (WoS) and correcting
      for journal visibility. For three of the five taxa, significant differences were 
      absent in citation numbers between taxonomic and non-taxonomic papers. This was
      also true for all taxa combined, although taxonomic papers received more
      citations than non-taxonomic ones. Our results show that, contrary to common
      belief, taxonomic contributions do not generally reduce a journal's citation
      performance and might even increase it. The scope of many journals rarely
      featuring taxonomy would allow editors to encourage a larger number of taxonomic 
      submissions. Moreover, between 1993 and 2012, taxonomic publications accumulated 
      faster than those from all biological fields. However, less than half of the
      taxonomic studies were published in journals in WoS. Thus, editors of highly
      visible journals inviting taxonomic contributions could benefit from taxonomy's
      strong momentum. The taxonomic output could increase even more than at its
      current growth rate if: (i) taxonomists currently publishing on other topics
      returned to taxonomy and (ii) non-taxonomists identifying the need for taxonomic 
      acts started publishing these, possibly in collaboration with taxonomists.
      Finally, considering the high number of taxonomic papers attracted by the journal
      Zootaxa, we expect that the taxonomic community would indeed use increased
      chances of publishing in WoS indexed journals. We conclude that taxonomy's
      standing in the present citation-focused scientific landscape could easily
      improve-if the community becomes aware that there is no citation impediment in
      taxonomy.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Steiner, Florian M
AU  - Steiner FM
AD  - Molecular Ecology Group, Institute of Ecology, University of Innsbruck,
      Technikerstr. 25, 6020 Innsbruck, Austria florian.m.steiner@uibk.ac.at.
FAU - Pautasso, Marco
AU  - Pautasso M
AD  - Forest Pathology and Dendrology, Institute of Integrative Biology, ETHZ,
      Universitatstr. 16, 8092 Zurich, Switzerland Animal and Plant Health Unit,
      European Food Safety Authority, via Carlo Magno 1a, 43126 Parma, Italy.
FAU - Zettel, Herbert
AU  - Zettel H
AD  - 2nd Zoological Department, Natural History Museum Vienna, Burgring 7, 1010
      Vienna, Austria.
FAU - Moder, Karl
AU  - Moder K
AD  - Institute of Applied Statistics and Computing, University of Natural Resources
      and Life Sciences, Peter Jordan-Str. 82, 1180 Vienna, Austria.
FAU - Arthofer, Wolfgang
AU  - Arthofer W
AD  - Molecular Ecology Group, Institute of Ecology, University of Innsbruck,
      Technikerstr. 25, 6020 Innsbruck, Austria.
FAU - Schlick-Steiner, Birgit C
AU  - Schlick-Steiner BC
AD  - Molecular Ecology Group, Institute of Ecology, University of Innsbruck,
      Technikerstr. 25, 6020 Innsbruck, Austria.
LA  - eng
GR  - P 23409/Austrian Science Fund FWF/Austria
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150504
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Classification
MH  - Publishing/ethics/*standards
MH  - Science/*standards
PMC - PMC4538880
OTO - NOTNLM
OT  - Animals
OT  - citations
OT  - impact factor
OT  - microorganisms
OT  - plants
OT  - scientometrics
OT  - taxonomic impediment
OT  - taxonomy
EDAT- 2015/05/07 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/05/07 06:00
PHST- 2014/10/20 00:00 [received]
PHST- 2015/04/27 00:00 [accepted]
PHST- 2015/05/07 06:00 [entrez]
PHST- 2015/05/07 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv026 [pii]
AID - 10.1093/sysbio/syv026 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):860-8. doi: 10.1093/sysbio/syv026. Epub 2015 May 4.

PMID- 25922515
OWN - NLM
STAT- MEDLINE
DCOM- 20151116
LR  - 20170922
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 5
DP  - 2015 Sep
TI  - The Fossil Calibration Database-A New Resource for Divergence Dating.
PG  - 853-9
LID - 10.1093/sysbio/syv025 [doi]
AB  - Fossils provide the principal basis for temporal calibrations, which are critical
      to the accuracy of divergence dating analyses. Translating fossil data into
      minimum and maximum bounds for calibrations is the most important-often least
      appreciated-step of divergence dating. Properly justified calibrations require
      the synthesis of phylogenetic, paleontological, and geological evidence and can
      be difficult for nonspecialists to formulate. The dynamic nature of the fossil
      record (e.g., new discoveries, taxonomic revisions, updates of global or local
      stratigraphy) requires that calibration data be updated continually lest they
      become obsolete. Here, we announce the Fossil Calibration Database
      (http://fossilcalibrations.org), a new open-access resource providing vetted
      fossil calibrations to the scientific community. Calibrations accessioned into
      this database are based on individual fossil specimens and follow best practices 
      for phylogenetic justification and geochronological constraint. The associated
      Fossil Calibration Series, a calibration-themed publication series at
      Palaeontologia Electronica, will serve as a key pipeline for peer-reviewed
      calibrations to enter the database.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Ksepka, Daniel T
AU  - Ksepka DT
AD  - Bruce Museum, 1 Museum Drive, Greenwich, CT 06830, USA; dksepka@brucemuseum.org.
FAU - Parham, James F
AU  - Parham JF
AD  - John D. Cooper Archaeological and Paleontological Center, Department of
      Geological Sciences, California State University, Fullerton, CA 92834, USA;
FAU - Allman, James F
AU  - Allman JF
AD  - Interrobang Digital Media,1034 Pebblebrook Drive, Wake Forest, NC 27587, USA;
FAU - Benton, Michael J
AU  - Benton MJ
AD  - School of Earth Sciences, University of Bristol, Bristol, BS8 1RJ, UK;
FAU - Carrano, Matthew T
AU  - Carrano MT
AD  - Department of Paleobiology, P.O. Box 37012, MRC 121, Smithsonian Institution,
      Washington DC 20013-7012, USA;
FAU - Cranston, Karen A
AU  - Cranston KA
AD  - National Evolutionary Synthesis Center, 2024 West Main Street Suite A200, Durham,
      NC 27705, USA;
FAU - Donoghue, Philip C J
AU  - Donoghue PC
AD  - School of Earth Sciences, University of Bristol, Bristol, BS8 1RJ, UK;
FAU - Head, Jason J
AU  - Head JJ
AD  - Department of Earth and Atmospheric Sciences, University of Nebraska-Lincoln, 228
      Bessey Hall, Lincoln, NE 68588, USA;
FAU - Hermsen, Elizabeth J
AU  - Hermsen EJ
AD  - Department of Environmental and Plant Biology, Ohio University, Athens, OH 45701,
      USA;
FAU - Irmis, Randall B
AU  - Irmis RB
AD  - Natural History Museum of Utah and Department of Geology &amp; Geophysics, University
      of Utah, 301 Wakara Way, Salt Lake City, UT 84108, USA;
FAU - Joyce, Walter G
AU  - Joyce WG
AD  - Department of Geosciences, University of Fribourg, Chemin du Musee 6, 1700
      Fribourg, Switzerland;
FAU - Kohli, Manpreet
AU  - Kohli M
AD  - Department of Biology, Rutgers University, 195 University Ave, Newark, NJ, 07102,
      USA;
FAU - Lamm, Kristin D
AU  - Lamm KD
AD  - Bioinformatics Research Center, North Carolina State University, Raleigh, NC
      27965, USA;
FAU - Leehr, Dan
AU  - Leehr D
AD  - National Evolutionary Synthesis Center, 2024 West Main Street Suite A200, Durham,
      NC 27705, USA; Duke Center for Genomic and Computational Biology, Duke
      University, 101 Science Drive, Durham, NC 27708, USA;
FAU - Patane, Joses L
AU  - Patane JL
AD  - Departamento de Bioquimica, Instituto de Quimica, Universidade de Sao Paulo,
      05508-000, Brazil;
FAU - Polly, P David
AU  - Polly PD
AD  - Department of Geological Sciences, Indiana University, 1001 E. 10th Street,
      Bloomington, IN 47401, USA;
FAU - Phillips, Matthew J
AU  - Phillips MJ
AD  - School of Earth, Environmental and Biological Sciences, Queensland University of 
      Technology, Brisbane 4000, Australia;
FAU - Smith, N Adam
AU  - Smith NA
AD  - Field Museum of Natural History, 1400 S Lake Shore Drive, Chicago, IL 60605, USA;
FAU - Smith, Nathan D
AU  - Smith ND
AD  - Department of Biology, Howard University, 415 College Street NW, Washington, DC, 
      20059, USA;
FAU - Van Tuinen, Marcel
AU  - Van Tuinen M
AD  - Centre of Evolutionary and Ecological Studies, Marine Evolution and Conservation 
      Group, Univerity of Groningen, Nijenborgh 7, 9747 AG Groningen, The Netherlands;
FAU - Ware, Jessica L
AU  - Ware JL
AD  - Department of Biology, Rutgers University, 195 University Ave, Newark, NJ, 07102,
      USA;
FAU - Warnock, Rachel C M
AU  - Warnock RC
AD  - Department of Paleobiology, P.O. Box 37012, MRC 121, Smithsonian Institution,
      Washington DC 20013-7012, USA;
LA  - eng
GR  - BB/G006660/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150427
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Access to Information
MH  - Calibration
MH  - Data Interpretation, Statistical
MH  - Databases, Factual/*standards
MH  - *Fossils
MH  - Internet
MH  - *Phylogeny
MH  - Time
OTO - NOTNLM
OT  - Calibration
OT  - divergence dating
OT  - fossil
EDAT- 2015/04/30 06:00
MHDA- 2015/11/17 06:00
CRDT- 2015/04/30 06:00
PHST- 2014/09/26 00:00 [received]
PHST- 2015/04/22 00:00 [accepted]
PHST- 2015/04/30 06:00 [entrez]
PHST- 2015/04/30 06:00 [pubmed]
PHST- 2015/11/17 06:00 [medline]
AID - syv025 [pii]
AID - 10.1093/sysbio/syv025 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Sep;64(5):853-9. doi: 10.1093/sysbio/syv025. Epub 2015 Apr 27.

PMID- 25888025
OWN - NLM
STAT- MEDLINE
DCOM- 20150921
LR  - 20150617
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 4
DP  - 2015 Jul
TI  - Detection and Polarization of Introgression in a Five-Taxon Phylogeny.
PG  - 651-62
LID - 10.1093/sysbio/syv023 [doi]
AB  - When multiple speciation events occur rapidly in succession, discordant
      genealogies due to incomplete lineage sorting (ILS) can complicate the detection 
      of introgression. A variety of methods, including the [Formula: see
      text]-statistic (a.k.a. the "ABBA-BABA test"), have been proposed to infer
      introgression in the presence of ILS for a four-taxon clade. However, no
      integrated method exists to detect introgression using allelic patterns for more 
      complex phylogenies. Here we explore the issues associated with previous systems 
      of applying [Formula: see text]-statistics to a larger tree topology, and propose
      new [Formula: see text] tests as an integrated framework to infer both the taxa
      involved in and the direction of introgression for a symmetric five-taxon
      phylogeny. Using theory and simulations, we show that the [Formula: see text]
      statistics correctly identify the introgression donor and recipient lineages,
      even at low rates of introgression. [Formula: see text] is also shown to have
      extremely low false-positive rates. The [Formula: see text] tests are
      computationally inexpensive to calculate and can easily be applied to
      phylogenomic data sets, both genome-wide and in windows of the genome. In
      addition, we explore both the principles and problems of introgression detection 
      in even more complex phylogenies.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Pease, James B
AU  - Pease JB
AD  - Department of Biology, Indiana University, 1001 E. Third St., Bloomington, IN
      47405, USA and School of Informatics and Computing, Indiana University,
      Bloomington, IN 47405, USA jbpease@indiana.edu.
FAU - Hahn, Matthew W
AU  - Hahn MW
AD  - Department of Biology, Indiana University, 1001 E. Third St., Bloomington, IN
      47405, USA and School of Informatics and Computing, Indiana University,
      Bloomington, IN 47405, USA Department of Biology, Indiana University, 1001 E.
      Third St., Bloomington, IN 47405, USA and School of Informatics and Computing,
      Indiana University, Bloomington, IN 47405, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150417
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Data Interpretation, Statistical
MH  - Humans
MH  - Neanderthals/classification
MH  - *Phylogeny
OTO - NOTNLM
OT  - ABBA-BABA
OT  - D-statistics
OT  - genomics
OT  - hybridization
OT  - incomplete lineage sorting
OT  - introgression
OT  - phylogenetics
OT  - phylogenomics
EDAT- 2015/04/19 06:00
MHDA- 2015/09/22 06:00
CRDT- 2015/04/19 06:00
PHST- 2015/02/09 00:00 [received]
PHST- 2015/04/10 00:00 [accepted]
PHST- 2015/04/19 06:00 [entrez]
PHST- 2015/04/19 06:00 [pubmed]
PHST- 2015/09/22 06:00 [medline]
AID - syv023 [pii]
AID - 10.1093/sysbio/syv023 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jul;64(4):651-62. doi: 10.1093/sysbio/syv023. Epub 2015 Apr 17.

PMID- 25858352
OWN - NLM
STAT- MEDLINE
DCOM- 20150921
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 4
DP  - 2015 Jul
TI  - A New Hierarchy of Phylogenetic Models Consistent with Heterogeneous Substitution
      Rates.
PG  - 638-50
LID - 10.1093/sysbio/syv021 [doi]
AB  - When the process underlying DNA substitutions varies across evolutionary history,
      some standard Markov models underlying phylogenetic methods are mathematically
      inconsistent. The most prominent example is the general time-reversible model
      (GTR) together with some, but not all, of its submodels. To rectify this
      deficiency, nonhomogeneous Lie Markov models have been identified as the class of
      models that are consistent in the face of a changing process of DNA substitutions
      regardless of taxon sampling. Some well-known models in popular use are within
      this class, but are either overly simplistic (e.g., the Kimura two-parameter
      model) or overly complex (the general Markov model). On a diverse set of
      biological data sets, we test a hierarchy of Lie Markov models spanning the full 
      range of parameter richness. Compared against the benchmark of the ever-popular
      GTR model, we find that as a whole the Lie Markov models perform well, with the
      best performing models having 8-10 parameters and the ability to recognize the
      distinction between purines and pyrimidines.
CI  - (c) The Author(s) 2015. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - Woodhams, Michael D
AU  - Woodhams MD
AD  - School of Physical Sciences, University of Tasmania, Hobart, TAS 7005, Australia 
      and Departament de Matematica Aplicada I, Universitat Politecnica de Catalunya,
      Barcelona, Spain michael.woodhams@utas.edu.au.
FAU - Fernandez-Sanchez, Jesus
AU  - Fernandez-Sanchez J
AD  - School of Physical Sciences, University of Tasmania, Hobart, TAS 7005, Australia 
      and Departament de Matematica Aplicada I, Universitat Politecnica de Catalunya,
      Barcelona, Spain.
FAU - Sumner, Jeremy G
AU  - Sumner JG
AD  - School of Physical Sciences, University of Tasmania, Hobart, TAS 7005, Australia 
      and Departament de Matematica Aplicada I, Universitat Politecnica de Catalunya,
      Barcelona, Spain.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150408
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
RN  - 0 (Nucleotides)
RN  - 9007-49-2 (DNA)
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - DNA/chemistry/genetics
MH  - DNA, Mitochondrial/chemistry/genetics
MH  - Humans
MH  - *Models, Biological
MH  - Nucleotides/genetics/metabolism
MH  - *Phylogeny
MH  - Plants/genetics
PMC - PMC4468350
OTO - NOTNLM
OT  - Lie Markov models
OT  - Model selection
OT  - ModelTest
OT  - multiplicative closure
OT  - phylogenetics
EDAT- 2015/04/11 06:00
MHDA- 2015/09/22 06:00
CRDT- 2015/04/11 06:00
PHST- 2014/12/04 00:00 [received]
PHST- 2015/03/30 00:00 [accepted]
PHST- 2015/04/11 06:00 [entrez]
PHST- 2015/04/11 06:00 [pubmed]
PHST- 2015/09/22 06:00 [medline]
AID - syv021 [pii]
AID - 10.1093/sysbio/syv021 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jul;64(4):638-50. doi: 10.1093/sysbio/syv021. Epub 2015 Apr 8.

PMID- 25841167
OWN - NLM
STAT- MEDLINE
DCOM- 20150921
LR  - 20150617
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 4
DP  - 2015 Jul
TI  - Comparative Analysis of Principal Components Can be Misleading.
PG  - 677-89
LID - 10.1093/sysbio/syv019 [doi]
AB  - Most existing methods for modeling trait evolution are univariate, although
      researchers are often interested in investigating evolutionary patterns and
      processes across multiple traits. Principal components analysis (PCA) is commonly
      used to reduce the dimensionality of multivariate data so that univariate trait
      models can be fit to individual principal components. The problem with using
      standard PCA on phylogenetically structured data has been previously pointed out 
      yet it continues to be widely used in the literature. Here we demonstrate
      precisely how using standard PCA can mislead inferences: The first few principal 
      components of traits evolved under constant-rate multivariate Brownian motion
      will appear to have evolved via an "early burst" process. A phylogenetic PCA
      (pPCA) has been proprosed to alleviate these issues. However, when the true model
      of trait evolution deviates from the model assumed in the calculation of the pPCA
      axes, we find that the use of pPCA suffers from similar artifacts as standard
      PCA. We show that data sets with high effective dimensionality are particularly
      likely to lead to erroneous inferences. Ultimately, all of the problems we report
      stem from the same underlying issue--by considering only the first few principal 
      components as univariate traits, we are effectively examining a biased sample of 
      a multivariate pattern. These results highlight the need for truly multivariate
      phylogenetic comparative methods. As these methods are still being developed, we 
      discuss potential alternative strategies for using and interpreting models fit to
      univariate axes of multivariate data.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Uyeda, Josef C
AU  - Uyeda JC
AD  - Department of Biological Sciences, Institute for Bioinformatics and Evolutionary 
      Studies, University of Idaho, Moscow, ID 83844, USA. josef.uyeda@gmail.com.
FAU - Caetano, Daniel S
AU  - Caetano DS
AD  - Department of Biological Sciences, Institute for Bioinformatics and Evolutionary 
      Studies, University of Idaho, Moscow, ID 83844, USA.
FAU - Pennell, Matthew W
AU  - Pennell MW
AD  - Department of Biological Sciences, Institute for Bioinformatics and Evolutionary 
      Studies, University of Idaho, Moscow, ID 83844, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150403
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Cats/anatomy &amp; histology/classification
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Lizards/anatomy &amp; histology/classification
MH  - *Phylogeny
MH  - Principal Component Analysis
OTO - NOTNLM
OT  - Brownian motion
OT  - Ornstein-Uhlenbeck
OT  - early burst
OT  - multivariate evolution
OT  - phylogenetic comparative methods
OT  - principal components analysis
OT  - quantitative genetics
EDAT- 2015/04/05 06:00
MHDA- 2015/09/22 06:00
CRDT- 2015/04/05 06:00
PHST- 2014/07/21 00:00 [received]
PHST- 2015/03/30 00:00 [accepted]
PHST- 2015/04/05 06:00 [entrez]
PHST- 2015/04/05 06:00 [pubmed]
PHST- 2015/09/22 06:00 [medline]
AID - syv019 [pii]
AID - 10.1093/sysbio/syv019 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jul;64(4):677-89. doi: 10.1093/sysbio/syv019. Epub 2015 Apr 3.

PMID- 25840332
OWN - NLM
STAT- MEDLINE
DCOM- 20150921
LR  - 20150617
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 4
DP  - 2015 Jul
TI  - What's in an Outgroup? The Impact of Outgroup Choice on the Phylogenetic Position
      of Thalattosuchia (Crocodylomorpha) and the Origin of Crocodyliformes.
PG  - 621-37
LID - 10.1093/sysbio/syv020 [doi]
AB  - Outgroup sampling is a central issue in phylogenetic analysis. However, good
      justification is rarely given for outgroup selection in published analyses.
      Recent advances in our understanding of archosaur phylogeny suggest that many
      previous studies of crocodylomorph and crocodyliform relationships have rooted
      trees on outgroup taxa that are only very distantly related to the ingroup (e.g.,
      Gracilisuchus stipanicicorum), or might actually belong within the ingroup.
      Thalattosuchia, a group of Mesozoic marine crocodylomorphs, has a controversial
      phylogenetic position--they are recovered as either the sister group to
      Crocodyliformes, in a basal position within Crocodyliformes, or nested high in
      the crocodyliform tree. Thalattosuchians lack several crocodyliform apomorphies, 
      but share several character states with derived long-snouted forms with a similar
      ecological habit, suggesting their derived position may be the result of
      convergent evolution. Several of these "shared" characters may result from
      ambiguously worded character state definitions--structures that are superficially
      similar but anatomically different in detail are identically coded. A new
      analysis of crocodylomorphs with increased outgroup sampling recovers
      Thalattosuchia as the sister group to Crocodyliformes, distantly related to
      long-snouted crocodyliforms. I also demonstrate that expanding the outgroup
      sampling of previously published matrices results in the recovery of
      thalattosuchians as sister to Crocodyliformes. The exclusion of thalattosuchians 
      from Crocodyliformes has numerous implications for large-scale evolutionary
      trends within the group, including extensive convergence in the evolution of the 
      secondary palate characteristic of the group. These results demonstrate the
      importance of careful outgroup sampling and character construction, and their
      profound effect on the position of labile clades.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Wilberg, Eric W
AU  - Wilberg EW
AD  - Department of Geoscience, University of Iowa, Iowa City, IA, USA and Current
      Address - Department of Geology and Geography, Georgia Southern University,
      Statesboro, GA, USA Department of Geoscience, University of Iowa, Iowa City, IA, 
      USA and Current Address - Department of Geology and Geography, Georgia Southern
      University, Statesboro, GA, USA ewilberg@georgiasouthern.edu.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150402
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Fossils
MH  - *Phylogeny
MH  - Reptiles/*classification
OTO - NOTNLM
OT  - Crocodyliformes
OT  - Crocodylomorpha
OT  - Thalattosuchia
OT  - convergence
OT  - outgroup sampling
EDAT- 2015/04/04 06:00
MHDA- 2015/09/22 06:00
CRDT- 2015/04/04 06:00
PHST- 2014/08/16 00:00 [received]
PHST- 2015/03/30 00:00 [accepted]
PHST- 2015/04/04 06:00 [entrez]
PHST- 2015/04/04 06:00 [pubmed]
PHST- 2015/09/22 06:00 [medline]
AID - syv020 [pii]
AID - 10.1093/sysbio/syv020 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jul;64(4):621-37. doi: 10.1093/sysbio/syv020. Epub 2015 Apr 2.

PMID- 25813358
OWN - NLM
STAT- MEDLINE
DCOM- 20150921
LR  - 20150617
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 4
DP  - 2015 Jul
TI  - On the Robustness to Gene Tree Estimation Error (or lack thereof) of
      Coalescent-Based Species Tree Methods.
PG  - 663-76
LID - 10.1093/sysbio/syv016 [doi]
AB  - The estimation of species trees using multiple loci has become increasingly
      common. Because different loci can have different phylogenetic histories
      (reflected in different gene tree topologies) for multiple biological causes, new
      approaches to species tree estimation have been developed that take gene tree
      heterogeneity into account. Among these multiple causes, incomplete lineage
      sorting (ILS), modeled by the multi-species coalescent, is potentially the most
      common cause of gene tree heterogeneity, and much of the focus of the recent
      literature has been on how to estimate species trees in the presence of ILS.
      Despite progress in developing statistically consistent techniques for estimating
      species trees when gene trees can differ due to ILS, there is substantial
      controversy in the systematics community as to whether to use the new
      coalescent-based methods or the traditional concatenation methods. One of the key
      issues that has been raised is understanding the impact of gene tree estimation
      error on coalescent-based methods that operate by combining gene trees. Here we
      explore the mathematical guarantees of coalescent-based methods when analyzing
      estimated rather than true gene trees. Our results provide some insight into the 
      differences between promise of coalescent-based methods in theory and their
      performance in practice.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Roch, Sebastien
AU  - Roch S
AD  - Department of Mathematics, University of Wisconsin at Madison, 480 Lincoln Dr.,
      Madison, Wisconsin, 53706, USA and Departments of Bioengineering and Computer
      Science, University of Illinois at Urbana-Champaign, Urbana, IL, 61801, USA
      roch@math.wisc.edu.
FAU - Warnow, Tandy
AU  - Warnow T
AD  - Department of Mathematics, University of Wisconsin at Madison, 480 Lincoln Dr.,
      Madison, Wisconsin, 53706, USA and Departments of Bioengineering and Computer
      Science, University of Illinois at Urbana-Champaign, Urbana, IL, 61801, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150325
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Genes/genetics
MH  - *Phylogeny
OTO - NOTNLM
OT  - coalescent-based methods
OT  - gene tree estimation error
OT  - incomplete lineage sorting
OT  - multi-species coalescent
OT  - species tree reconstruction
OT  - statistical consistency
EDAT- 2015/03/31 06:00
MHDA- 2015/09/22 06:00
CRDT- 2015/03/28 06:00
PHST- 2014/12/11 00:00 [received]
PHST- 2015/03/20 00:00 [accepted]
PHST- 2015/03/28 06:00 [entrez]
PHST- 2015/03/31 06:00 [pubmed]
PHST- 2015/09/22 06:00 [medline]
AID - syv016 [pii]
AID - 10.1093/sysbio/syv016 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jul;64(4):663-76. doi: 10.1093/sysbio/syv016. Epub 2015 Mar 25.

PMID- 25805045
OWN - NLM
STAT- MEDLINE
DCOM- 20150921
LR  - 20150617
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 4
DP  - 2015 Jul
TI  - Taxonomy and Phylogeny Can Yield Comparable Results in Comparative
      Paleontological Analyses.
PG  - 608-20
LID - 10.1093/sysbio/syv015 [doi]
AB  - Many extinct taxa with extensive fossil records and mature taxonomic
      classifications have not yet been the subject of formal phylogenetic analysis.
      Here, we test whether the taxonomies available for such groups represent useful
      (i.e., non-misleading) substitutes for trees derived from matrix-based
      phylogenetic analyses. We collected data for 52 animal clades that included
      fossil representatives, and for which a recent cladogram and pre-cladistic
      taxonomy were available. We quantified the difference between the time-scaled
      phylogenies implied by taxonomies and cladograms using the matching cluster
      distance metric. We simulated phenotypic trait values and used them to estimate a
      series of commonly used, phylogenetically explicit measures (phylogenetic signal 
      [Blomberg's [Formula: see text]], phylogenetic generalized least squares [PGLS], 
      mode of evolution [Brownian vs. Ornstein-Uhlenbeck], and phylogenetic clustering 
      of extinction [Fritz and Purvis' [Formula: see text]]) in order to determine the 
      degree to which they co-varied on taxonomic and cladistic trees. With respect to 
      topology taxonomies are good approximations of the underlying evolutionary
      relationships as recorded in inferred cladograms. Detection of phylogenetic
      clustering of extinction could not be properly assessed. For all other
      evolutionary analyses, results from taxonomy-based phylogenies (TBPs) co-varied
      with those from cladogram-based phylogenies (CBPs), but individual comparisons
      could be misleading. The relative length of terminal branches (influenced by
      stratigraphy and sampling rate) is a key control on the shared information
      between, and therefore the relative performance of, TBP and CBP. Collectively
      these results suggest that under particular circumstances and after careful
      consideration some taxonomies, or composite trees that incorporate taxonomic
      information, could be used in place of a formal analytical solution, but workers 
      must be cautious. This opens certain parts of a previously inaccessible section
      of the fossil record to interrogation within an explicit comparative framework,
      which will help to test many classical macroevolutionary hypotheses formulated
      for groups that currently lack formal phylogenetic estimates.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Soul, Laura C
AU  - Soul LC
AD  - Department of Earth Sciences, University of Oxford, South Parks Road, Oxford OX1 
      3AN, UK laura.soul@earth.ox.ac.uk.
FAU - Friedman, Matt
AU  - Friedman M
AD  - Department of Earth Sciences, University of Oxford, South Parks Road, Oxford OX1 
      3AN, UK.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150324
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification
MH  - Computer Simulation
MH  - Fossils
MH  - *Models, Biological
MH  - *Paleontology
MH  - *Phylogeny
OTO - NOTNLM
OT  - Evolutionary mode
OT  - PGLS
OT  - fossil record
OT  - phylogenetic signal
OT  - phylogeny
OT  - taxonomy
EDAT- 2015/03/26 06:00
MHDA- 2015/09/22 06:00
CRDT- 2015/03/26 06:00
PHST- 2013/11/26 00:00 [received]
PHST- 2015/03/18 00:00 [accepted]
PHST- 2015/03/26 06:00 [entrez]
PHST- 2015/03/26 06:00 [pubmed]
PHST- 2015/09/22 06:00 [medline]
AID - syv015 [pii]
AID - 10.1093/sysbio/syv015 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jul;64(4):608-20. doi: 10.1093/sysbio/syv015. Epub 2015 Mar 24.

PMID- 25771083
OWN - NLM
STAT- MEDLINE
DCOM- 20150921
LR  - 20150617
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 4
DP  - 2015 Jul
TI  - How Ecology and Landscape Dynamics Shape Phylogenetic Trees.
PG  - 590-607
LID - 10.1093/sysbio/syv014 [doi]
AB  - Whether biotic or abiotic factors are the dominant drivers of clade
      diversification is a long-standing question in evolutionary biology. The
      ubiquitous patterns of phylogenetic imbalance and branching slowdown have been
      taken as supporting the role of ecological niche filling and spatial
      heterogeneity in ecological features, and thus of biotic processes, in
      diversification. However, a proper theoretical assessment of the relative roles
      of biotic and abiotic factors in macroevolution requires models that integrate
      both types of factors, and such models have been lacking. In this study, we use
      an individual-based model to investigate the temporal patterns of diversification
      driven by ecological speciation in a stochastically fluctuating geographic
      landscape. The model generates phylogenies whose shape evolves as the clade ages.
      Stabilization of tree shape often occurs after ecological saturation, revealing
      species turnover caused by competition and demographic stochasticity. In the
      initial phase of diversification (allopatric radiation into an empty landscape), 
      trees tend to be unbalanced and branching slows down. As diversification proceeds
      further due to landscape dynamics, balance and branching tempo may increase and
      become positive. Three main conclusions follow. First, the phylogenies of
      ecologically saturated clades do not always exhibit branching slowdown. Branching
      slowdown requires that competition be wide or heterogeneous across the landscape,
      or that the characteristics of landscape dynamics vary geographically.
      Conversely, branching acceleration is predicted under narrow competition or
      frequent local catastrophes. Second, ecological heterogeneity does not
      necessarily cause phylogenies to be unbalanced--short time in geographical
      isolation or frequent local catastrophes may lead to balanced trees despite
      spatial heterogeneity. Conversely, unbalanced trees can emerge without spatial
      heterogeneity, notably if competition is wide. Third, short isolation time causes
      a radically different and quite robust pattern of phylogenies that are balanced
      and yet exhibit branching slowdown. In conclusion, biotic factors have a strong
      and diverse influence on the shape of phylogenies of ecologically saturating
      clades and create the evolutionary template in which branching slowdown and tree 
      imbalance may occur. However, the contingency of landscape dynamics and resource 
      distribution can cause wide variation in branching tempo and tree balance.
      Finally, considerable variation in tree shape among simulation replicates calls
      for caution when interpreting variation in the shape of real phylogenies.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Gascuel, Fanny
AU  - Gascuel F
AD  - Center for Interdisciplinary Research in Biology, CNRS UMR 7241, College de
      France, Paris, France; Sorbonne Universites, UPMC Univ Paris 06, CNRS UMR 7625,
      Laboratoire Ecologie et Evolution, Paris, France; Institut de Biologie de l'Ecole
      Normale Superieure, CNRS UMR 8197, Ecole Normale Superieure, Paris, France;
      Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      USA; Laboratoire Evolution et Diversite Biologique, CNRS UMR 5174, Universite
      Paul Sabatier, Toulouse, France; and Sorbonne Universites, UPMC Univ Paris 06,
      CNRS UMR 7599, Laboratoire Probabilites et Modeles Aleatoires, Paris, France
      Center for Interdisciplinary Research in Biology, CNRS UMR 7241, College de
      France, Paris, France; Sorbonne Universites, UPMC Univ Paris 06, CNRS UMR 7625,
      Laboratoire Ecologie et Evolution, Paris, France; Institut de Biologie de l'Ecole
      Normale Superieure, CNRS UMR 8197, Ecole Normale Superieure, Paris, France;
      Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      USA; Laboratoire Evolution et Diversite Biologique, CNRS UMR 5174, Universite
      Paul Sabatier, Toulouse, France; and Sorbonne Universites, UPMC Univ Paris 06,
      CNRS UMR 7599, Laboratoire Probabilites et Modeles Aleatoires, Paris, France
      Center for Interdisciplinary Research in Biology, CNRS UMR 7241, College de
      France, Paris, France; Sorbonne Universites, UPMC Univ Paris 06, CNRS UMR 7625,
      Laboratoire Ecologie et Evolution, Paris, France; Institut de Biologie de l'Ecole
      Normale Superieure, CNRS UMR 8197, Ecole Normale Superieure, Paris, France;
      Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      USA; Laboratoire Evolution et Diversite Biologique, CNRS UMR 5174, Universite
      Paul Sabatier, Toulouse, France; and Sorbonne Universites, UPMC Univ Paris 06,
      CNRS UMR 7599, Laboratoire Probabilites et Modeles Aleatoires, Paris, France
      fanny.gascuel@college-de-france.fr.
FAU - Ferriere, Regis
AU  - Ferriere R
AD  - Center for Interdisciplinary Research in Biology, CNRS UMR 7241, College de
      France, Paris, France; Sorbonne Universites, UPMC Univ Paris 06, CNRS UMR 7625,
      Laboratoire Ecologie et Evolution, Paris, France; Institut de Biologie de l'Ecole
      Normale Superieure, CNRS UMR 8197, Ecole Normale Superieure, Paris, France;
      Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      USA; Laboratoire Evolution et Diversite Biologique, CNRS UMR 5174, Universite
      Paul Sabatier, Toulouse, France; and Sorbonne Universites, UPMC Univ Paris 06,
      CNRS UMR 7599, Laboratoire Probabilites et Modeles Aleatoires, Paris, France
      Center for Interdisciplinary Research in Biology, CNRS UMR 7241, College de
      France, Paris, France; Sorbonne Universites, UPMC Univ Paris 06, CNRS UMR 7625,
      Laboratoire Ecologie et Evolution, Paris, France; Institut de Biologie de l'Ecole
      Normale Superieure, CNRS UMR 8197, Ecole Normale Superieure, Paris, France;
      Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      USA; Laboratoire Evolution et Diversite Biologique, CNRS UMR 5174, Universite
      Paul Sabatier, Toulouse, France; and Sorbonne Universites, UPMC Univ Paris 06,
      CNRS UMR 7599, Laboratoire Probabilites et Modeles Aleatoires, Paris, France.
FAU - Aguilee, Robin
AU  - Aguilee R
AD  - Center for Interdisciplinary Research in Biology, CNRS UMR 7241, College de
      France, Paris, France; Sorbonne Universites, UPMC Univ Paris 06, CNRS UMR 7625,
      Laboratoire Ecologie et Evolution, Paris, France; Institut de Biologie de l'Ecole
      Normale Superieure, CNRS UMR 8197, Ecole Normale Superieure, Paris, France;
      Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      USA; Laboratoire Evolution et Diversite Biologique, CNRS UMR 5174, Universite
      Paul Sabatier, Toulouse, France; and Sorbonne Universites, UPMC Univ Paris 06,
      CNRS UMR 7599, Laboratoire Probabilites et Modeles Aleatoires, Paris, France.
FAU - Lambert, Amaury
AU  - Lambert A
AD  - Center for Interdisciplinary Research in Biology, CNRS UMR 7241, College de
      France, Paris, France; Sorbonne Universites, UPMC Univ Paris 06, CNRS UMR 7625,
      Laboratoire Ecologie et Evolution, Paris, France; Institut de Biologie de l'Ecole
      Normale Superieure, CNRS UMR 8197, Ecole Normale Superieure, Paris, France;
      Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      USA; Laboratoire Evolution et Diversite Biologique, CNRS UMR 5174, Universite
      Paul Sabatier, Toulouse, France; and Sorbonne Universites, UPMC Univ Paris 06,
      CNRS UMR 7599, Laboratoire Probabilites et Modeles Aleatoires, Paris, France
      Center for Interdisciplinary Research in Biology, CNRS UMR 7241, College de
      France, Paris, France; Sorbonne Universites, UPMC Univ Paris 06, CNRS UMR 7625,
      Laboratoire Ecologie et Evolution, Paris, France; Institut de Biologie de l'Ecole
      Normale Superieure, CNRS UMR 8197, Ecole Normale Superieure, Paris, France;
      Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ
      USA; Laboratoire Evolution et Diversite Biologique, CNRS UMR 5174, Universite
      Paul Sabatier, Toulouse, France; and Sorbonne Universites, UPMC Univ Paris 06,
      CNRS UMR 7599, Laboratoire Probabilites et Modeles Aleatoires, Paris, France.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150313
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Computer Simulation
MH  - Ecology
MH  - Geography
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Time Factors
OTO - NOTNLM
OT  - Macroevolution
OT  - adaptive radiation
OT  - allopatric speciation
OT  - competition
OT  - eco-evolutionary feedbacks
OT  - ecological speciation
OT  - geographic isolation
OT  - phylogeny
EDAT- 2015/03/17 06:00
MHDA- 2015/09/22 06:00
CRDT- 2015/03/16 06:00
PHST- 2014/07/23 00:00 [received]
PHST- 2015/03/09 00:00 [accepted]
PHST- 2015/03/16 06:00 [entrez]
PHST- 2015/03/17 06:00 [pubmed]
PHST- 2015/09/22 06:00 [medline]
AID - syv014 [pii]
AID - 10.1093/sysbio/syv014 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jul;64(4):590-607. doi: 10.1093/sysbio/syv014. Epub 2015 Mar 13.

PMID- 25713307
OWN - NLM
STAT- MEDLINE
DCOM- 20150921
LR  - 20150617
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 4
DP  - 2015 Jul
TI  - Approximate Bayesian Computation Reveals the Crucial Role of Oceanic Islands for 
      the Assembly of Continental Biodiversity.
PG  - 579-89
LID - 10.1093/sysbio/syv013 [doi]
AB  - The perceived low levels of genetic diversity, poor interspecific competitive and
      defensive ability, and loss of dispersal capacities of insular lineages have
      driven the view that oceanic islands are evolutionary dead ends. Focusing on the 
      Atlantic bryophyte flora distributed across the archipelagos of the Azores,
      Madeira, the Canary Islands, Western Europe, and northwestern Africa, we used an 
      integrative approach with species distribution modeling and population genetic
      analyses based on approximate Bayesian computation to determine whether this view
      applies to organisms with inherent high dispersal capacities. Genetic diversity
      was found to be higher in island than in continental populations, contributing to
      mounting evidence that, contrary to theoretical expectations, island populations 
      are not necessarily genetically depauperate. Patterns of genetic variation among 
      island and continental populations consistently fitted those simulated under a
      scenario of de novo foundation of continental populations from insular ancestors 
      better than those expected if islands would represent a sink or a refugium of
      continental biodiversity. We, suggest that the northeastern Atlantic archipelagos
      have played a key role as a stepping stone for transoceanic migrants. Our results
      challenge the traditional notion that oceanic islands are the end of the
      colonization road and illustrate the significant role of oceanic islands as
      reservoirs of novel biodiversity for the assembly of continental floras.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Patino, Jairo
AU  - Patino J
AD  - Department of Biology, Ecology and Evolution, University of Liege, Bat. B22,
      Boulevard du Rectorat 27, 4000, Liege, Belgium; Department of Plant Biology,
      University of La Laguna, c/ Astrofisico Francisco Sanches, s/n, 28071, Tenerife, 
      Spain; Azorean Biodiversity Group (GBA, CE3C - Center for Ecology, Evolution and 
      Environmental Changes) and Platform for Enhancing Ecological Research &amp;
      Sustainability (PEERS), Universidade dos Acores, Rua Capitao Joao d'Avila, Pico
      da Urze, 9700-042 Angra do Heroismo, Terceira, Acores, Portugal; Plants Division,
      Department of Life Sciences, The Natural History Museum, Cromwell Road, London,
      SW7 5BD, UK; Evolutionary Biology and Ecology, Universite Libre de Bruxelles
      (ULB), CP 160/12, av FD Roosevelt 50, 1050, Brussels, Belgium; Department of
      Biology, Duke University, Box 90338, Durham, NC 27708, USA; and Department of
      Ecology and Evolution, University of Lausanne, Lausanne, CH-1015, Switzerland
      Department of Biology, Ecology and Evolution, University of Liege, Bat. B22,
      Boulevard du Rectorat 27, 4000, Liege, Belgium; Department of Plant Biology,
      University of La Laguna, c/ Astrofisico Francisco Sanches, s/n, 28071, Tenerife, 
      Spain; Azorean Biodiversity Group (GBA, CE3C - Center for Ecology, Evolution and 
      Environmental Changes) and Platform for Enhancing Ecological Research &amp;
      Sustainability (PEERS), Universidade dos Acores, Rua Capitao Joao d'Avila, Pico
      da Urze, 9700-042 Angra do Heroismo, Terceira, Acores, Portugal; Plants Division,
      Department of Life Sciences, The Natural History Museum, Cromwell Road, London,
      SW7 5BD, UK; Evolutionary Biology and Ecology, Universite Libre de Bruxelles
      (ULB), CP 160/12, av FD Roosevelt 50, 1050, Brussels, Belgium; Department of
      Biology, Duke University, Box 90338, Durham, NC 27708, USA; and Department of
      Ecology and Evolution, University of Lausanne, Lausanne, CH-1015, Switzerland
      Department of Biology, Ecology and Evolution, University of Liege, Bat. B22,
      Boulevard du Rect
FAU - Carine, Mark
AU  - Carine M
AD  - Department of Biology, Ecology and Evolution, University of Liege, Bat. B22,
      Boulevard du Rectorat 27, 4000, Liege, Belgium; Department of Plant Biology,
      University of La Laguna, c/ Astrofisico Francisco Sanches, s/n, 28071, Tenerife, 
      Spain; Azorean Biodiversity Group (GBA, CE3C - Center for Ecology, Evolution and 
      Environmental Changes) and Platform for Enhancing Ecological Research &amp;
      Sustainability (PEERS), Universidade dos Acores, Rua Capitao Joao d'Avila, Pico
      da Urze, 9700-042 Angra do Heroismo, Terceira, Acores, Portugal; Plants Division,
      Department of Life Sciences, The Natural History Museum, Cromwell Road, London,
      SW7 5BD, UK; Evolutionary Biology and Ecology, Universite Libre de Bruxelles
      (ULB), CP 160/12, av FD Roosevelt 50, 1050, Brussels, Belgium; Department of
      Biology, Duke University, Box 90338, Durham, NC 27708, USA; and Department of
      Ecology and Evolution, University of Lausanne, Lausanne, CH-1015, Switzerland.
FAU - Mardulyn, Patrick
AU  - Mardulyn P
AD  - Department of Biology, Ecology and Evolution, University of Liege, Bat. B22,
      Boulevard du Rectorat 27, 4000, Liege, Belgium; Department of Plant Biology,
      University of La Laguna, c/ Astrofisico Francisco Sanches, s/n, 28071, Tenerife, 
      Spain; Azorean Biodiversity Group (GBA, CE3C - Center for Ecology, Evolution and 
      Environmental Changes) and Platform for Enhancing Ecological Research &amp;
      Sustainability (PEERS), Universidade dos Acores, Rua Capitao Joao d'Avila, Pico
      da Urze, 9700-042 Angra do Heroismo, Terceira, Acores, Portugal; Plants Division,
      Department of Life Sciences, The Natural History Museum, Cromwell Road, London,
      SW7 5BD, UK; Evolutionary Biology and Ecology, Universite Libre de Bruxelles
      (ULB), CP 160/12, av FD Roosevelt 50, 1050, Brussels, Belgium; Department of
      Biology, Duke University, Box 90338, Durham, NC 27708, USA; and Department of
      Ecology and Evolution, University of Lausanne, Lausanne, CH-1015, Switzerland.
FAU - Devos, Nicolas
AU  - Devos N
AD  - Department of Biology, Ecology and Evolution, University of Liege, Bat. B22,
      Boulevard du Rectorat 27, 4000, Liege, Belgium; Department of Plant Biology,
      University of La Laguna, c/ Astrofisico Francisco Sanches, s/n, 28071, Tenerife, 
      Spain; Azorean Biodiversity Group (GBA, CE3C - Center for Ecology, Evolution and 
      Environmental Changes) and Platform for Enhancing Ecological Research &amp;
      Sustainability (PEERS), Universidade dos Acores, Rua Capitao Joao d'Avila, Pico
      da Urze, 9700-042 Angra do Heroismo, Terceira, Acores, Portugal; Plants Division,
      Department of Life Sciences, The Natural History Museum, Cromwell Road, London,
      SW7 5BD, UK; Evolutionary Biology and Ecology, Universite Libre de Bruxelles
      (ULB), CP 160/12, av FD Roosevelt 50, 1050, Brussels, Belgium; Department of
      Biology, Duke University, Box 90338, Durham, NC 27708, USA; and Department of
      Ecology and Evolution, University of Lausanne, Lausanne, CH-1015, Switzerland.
FAU - Mateo, Ruben G
AU  - Mateo RG
AD  - Department of Biology, Ecology and Evolution, University of Liege, Bat. B22,
      Boulevard du Rectorat 27, 4000, Liege, Belgium; Department of Plant Biology,
      University of La Laguna, c/ Astrofisico Francisco Sanches, s/n, 28071, Tenerife, 
      Spain; Azorean Biodiversity Group (GBA, CE3C - Center for Ecology, Evolution and 
      Environmental Changes) and Platform for Enhancing Ecological Research &amp;
      Sustainability (PEERS), Universidade dos Acores, Rua Capitao Joao d'Avila, Pico
      da Urze, 9700-042 Angra do Heroismo, Terceira, Acores, Portugal; Plants Division,
      Department of Life Sciences, The Natural History Museum, Cromwell Road, London,
      SW7 5BD, UK; Evolutionary Biology and Ecology, Universite Libre de Bruxelles
      (ULB), CP 160/12, av FD Roosevelt 50, 1050, Brussels, Belgium; Department of
      Biology, Duke University, Box 90338, Durham, NC 27708, USA; and Department of
      Ecology and Evolution, University of Lausanne, Lausanne, CH-1015, Switzerland
      Department of Biology, Ecology and Evolution, University of Liege, Bat. B22,
      Boulevard du Rectorat 27, 4000, Liege, Belgium; Department of Plant Biology,
      University of La Laguna, c/ Astrofisico Francisco Sanches, s/n, 28071, Tenerife, 
      Spain; Azorean Biodiversity Group (GBA, CE3C - Center for Ecology, Evolution and 
      Environmental Changes) and Platform for Enhancing Ecological Research &amp;
      Sustainability (PEERS), Universidade dos Acores, Rua Capitao Joao d'Avila, Pico
      da Urze, 9700-042 Angra do Heroismo, Terceira, Acores, Portugal; Plants Division,
      Department of Life Sciences, The Natural History Museum, Cromwell Road, London,
      SW7 5BD, UK; Evolutionary Biology and Ecology, Universite Libre de Bruxelles
      (ULB), CP 160/12, av FD Roosevelt 50, 1050, Brussels, Belgium; Department of
      Biology, Duke University, Box 90338, Durham, NC 27708, USA; and Department of
      Ecology and Evolution, University of Lausanne, Lausanne, CH-1015, Switzerland.
FAU - Gonzalez-Mancebo, Juana M
AU  - Gonzalez-Mancebo JM
AD  - Department of Biology, Ecology and Evolution, University of Liege, Bat. B22,
      Boulevard du Rectorat 27, 4000, Liege, Belgium; Department of Plant Biology,
      University of La Laguna, c/ Astrofisico Francisco Sanches, s/n, 28071, Tenerife, 
      Spain; Azorean Biodiversity Group (GBA, CE3C - Center for Ecology, Evolution and 
      Environmental Changes) and Platform for Enhancing Ecological Research &amp;
      Sustainability (PEERS), Universidade dos Acores, Rua Capitao Joao d'Avila, Pico
      da Urze, 9700-042 Angra do Heroismo, Terceira, Acores, Portugal; Plants Division,
      Department of Life Sciences, The Natural History Museum, Cromwell Road, London,
      SW7 5BD, UK; Evolutionary Biology and Ecology, Universite Libre de Bruxelles
      (ULB), CP 160/12, av FD Roosevelt 50, 1050, Brussels, Belgium; Department of
      Biology, Duke University, Box 90338, Durham, NC 27708, USA; and Department of
      Ecology and Evolution, University of Lausanne, Lausanne, CH-1015, Switzerland.
FAU - Shaw, A Jonathan
AU  - Shaw AJ
AD  - Department of Biology, Ecology and Evolution, University of Liege, Bat. B22,
      Boulevard du Rectorat 27, 4000, Liege, Belgium; Department of Plant Biology,
      University of La Laguna, c/ Astrofisico Francisco Sanches, s/n, 28071, Tenerife, 
      Spain; Azorean Biodiversity Group (GBA, CE3C - Center for Ecology, Evolution and 
      Environmental Changes) and Platform for Enhancing Ecological Research &amp;
      Sustainability (PEERS), Universidade dos Acores, Rua Capitao Joao d'Avila, Pico
      da Urze, 9700-042 Angra do Heroismo, Terceira, Acores, Portugal; Plants Division,
      Department of Life Sciences, The Natural History Museum, Cromwell Road, London,
      SW7 5BD, UK; Evolutionary Biology and Ecology, Universite Libre de Bruxelles
      (ULB), CP 160/12, av FD Roosevelt 50, 1050, Brussels, Belgium; Department of
      Biology, Duke University, Box 90338, Durham, NC 27708, USA; and Department of
      Ecology and Evolution, University of Lausanne, Lausanne, CH-1015, Switzerland.
FAU - Vanderpoorten, Alain
AU  - Vanderpoorten A
AD  - Department of Biology, Ecology and Evolution, University of Liege, Bat. B22,
      Boulevard du Rectorat 27, 4000, Liege, Belgium; Department of Plant Biology,
      University of La Laguna, c/ Astrofisico Francisco Sanches, s/n, 28071, Tenerife, 
      Spain; Azorean Biodiversity Group (GBA, CE3C - Center for Ecology, Evolution and 
      Environmental Changes) and Platform for Enhancing Ecological Research &amp;
      Sustainability (PEERS), Universidade dos Acores, Rua Capitao Joao d'Avila, Pico
      da Urze, 9700-042 Angra do Heroismo, Terceira, Acores, Portugal; Plants Division,
      Department of Life Sciences, The Natural History Museum, Cromwell Road, London,
      SW7 5BD, UK; Evolutionary Biology and Ecology, Universite Libre de Bruxelles
      (ULB), CP 160/12, av FD Roosevelt 50, 1050, Brussels, Belgium; Department of
      Biology, Duke University, Box 90338, Durham, NC 27708, USA; and Department of
      Ecology and Evolution, University of Lausanne, Lausanne, CH-1015, Switzerland
      Department of Biology, Ecology and Evolution, University of Liege, Bat. B22,
      Boulevard du Rectorat 27, 4000, Liege, Belgium; Department of Plant Biology,
      University of La Laguna, c/ Astrofisico Francisco Sanches, s/n, 28071, Tenerife, 
      Spain; Azorean Biodiversity Group (GBA, CE3C - Center for Ecology, Evolution and 
      Environmental Changes) and Platform for Enhancing Ecological Research &amp;
      Sustainability (PEERS), Universidade dos Acores, Rua Capitao Joao d'Avila, Pico
      da Urze, 9700-042 Angra do Heroismo, Terceira, Acores, Portugal; Plants Division,
      Department of Life Sciences, The Natural History Museum, Cromwell Road, London,
      SW7 5BD, UK; Evolutionary Biology and Ecology, Universite Libre de Bruxelles
      (ULB), CP 160/12, av FD Roosevelt 50, 1050, Brussels, Belgium; Department of
      Biology, Duke University, Box 90338, Durham, NC 27708, USA; and Department of
      Ecology and Evolution, University of Lausanne, Lausanne, CH-1015, Switzerland.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150220
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - *Biodiversity
MH  - Bryophyta/*classification/physiology
MH  - *Islands
MH  - Oceans and Seas
OTO - NOTNLM
OT  - Approximate Bayesian computation
OT  - bryophytes
OT  - colonization
OT  - island biogeography
OT  - last glacial maximum
OT  - long-distance dispersal
EDAT- 2015/02/26 06:00
MHDA- 2015/09/22 06:00
CRDT- 2015/02/26 06:00
PHST- 2014/09/16 00:00 [received]
PHST- 2015/02/14 00:00 [accepted]
PHST- 2015/02/26 06:00 [entrez]
PHST- 2015/02/26 06:00 [pubmed]
PHST- 2015/09/22 06:00 [medline]
AID - syv013 [pii]
AID - 10.1093/sysbio/syv013 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jul;64(4):579-89. doi: 10.1093/sysbio/syv013. Epub 2015 Feb 20.

PMID- 25672902
OWN - NLM
STAT- MEDLINE
DCOM- 20150921
LR  - 20150617
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 4
DP  - 2015 Jul
TI  - Phylogenetic Comparative Methods for Evaluating the Evolutionary History of
      Function-Valued Traits.
PG  - 568-78
LID - 10.1093/sysbio/syv012 [doi]
AB  - Phylogenetic comparative methods offer a suite of tools for studying trait
      evolution. However, most models inherently assume fixed trait values within
      species. Although some methods can incorporate error around species means, few
      are capable of accounting for variation driven by environmental or temporal
      gradients, such as trait responses to abiotic stress or ontogenetic trajectories.
      Such traits, often referred to as function-valued or infinite-dimensional, are
      typically expressed as reaction norms, dose-response curves, or time plots and
      are described by mathematical functions linking independent predictor variables
      to the trait of interest. Here, I introduce a method for extending ancestral
      state reconstruction to incorporate function-valued traits in a phylogenetic
      generalized least squares (PGLS) framework, as well as extensions of this method 
      for testing phylogenetic signal, performing phylogenetic analysis of variance
      (ANOVA), and testing for correlated trait evolution using recently proposed
      multivariate PGLS methods. Statistical power of function-valued comparative
      methods is compared to univariate approaches using data simulations, and the
      assumptions and challenges of each are discussed in detail.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Goolsby, Eric W
AU  - Goolsby EW
AD  - Interdisciplinary Toxicology Program, Department of Plant Biology, University of 
      Georgia, Athens GA, 30602, USA eric.goolsby.evolution@gmail.com.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150210
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Analysis of Variance
MH  - Classification/*methods
MH  - Multivariate Analysis
MH  - *Phylogeny
OTO - NOTNLM
OT  - PGLS
OT  - ancestral state reconstruction
OT  - dose-response curves
OT  - evolution of tolerance
OT  - function-valued traits
OT  - infinite-dimensional traits
OT  - phenotypic plasticity
OT  - phylogenetic comparative method
OT  - phylogenetic signal
OT  - reaction norms
EDAT- 2015/02/13 06:00
MHDA- 2015/09/22 06:00
CRDT- 2015/02/13 06:00
PHST- 2014/11/07 00:00 [received]
PHST- 2015/02/02 00:00 [accepted]
PHST- 2015/02/13 06:00 [entrez]
PHST- 2015/02/13 06:00 [pubmed]
PHST- 2015/09/22 06:00 [medline]
AID - syv012 [pii]
AID - 10.1093/sysbio/syv012 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jul;64(4):568-78. doi: 10.1093/sysbio/syv012. Epub 2015 Feb 10.

PMID- 25649930
OWN - NLM
STAT- MEDLINE
DCOM- 20150921
LR  - 20150617
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 4
DP  - 2015 Jul
TI  - Multilocus Phylogeny of the Afrotropical Freshwater Crab Fauna Reveals Historical
      Drainage Connectivity and Transoceanic Dispersal Since the Eocene.
PG  - 549-67
LID - 10.1093/sysbio/syv011 [doi]
AB  - Phylogenetic reconstruction, divergence time estimations and ancestral range
      estimation were undertaken for 66% of the Afrotropical freshwater crab fauna
      (Potamonautidae) based on four partial DNA loci (12S rRNA, 16S rRNA, cytochrome
      oxidase one [COI], and histone 3). The present study represents the most
      comprehensive taxonomic sampling of any freshwater crab family globally, and
      explores the impact of paleodrainage interconnectivity on cladogenesis among
      freshwater crabs. Phylogenetic analyses of the total evidence data using
      maximum-likelihood (ML), maximum parsimony (MP), and Bayesian inference (BI)
      produced a robust statistically well-supported tree topology that reaffirmed the 
      monophyly of the Afrotropical freshwater crab fauna. The estimated divergence
      times suggest that the Afrotropical Potamonautidae diverged during the Eocene.
      Cladogenesis within and among several genera occurred predominantly during the
      Miocene, which was associated with major tectonic and climatic ameliorations
      throughout the region. Paleodrainage connectivity was observed with specimens
      from the Nilo-Sudan and East African coast proving to be sister to specimens from
      the Upper Guinea Forests in West Africa. In addition, we observed strong sister
      taxon affinity between specimens from East Africa and the Congo basin, including 
      specimens from Lake Tanganyika, while the southern African fauna was retrieved as
      sister to the Angolan taxa. Within the East African clade we observed two
      independent transoceanic dispersal events, one to the Seychelles Archipelago and 
      a second to Madagascar, while we observe a single transoceanic dispersal event
      from West Africa to Sao Tome. The ancestral area estimation suggested a West
      African/East African ancestral range for the family with multiple dispersal
      events between southern Africa and East Africa, and between East Africa and
      Central Africa The taxonomic implications of our results are discussed in light
      of the widespread paraphyly evident among a number of genera.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Daniels, Savel R
AU  - Daniels SR
AD  - Department of Botany and Zoology, Private Bag X1, University of Stellenbosch,
      Matieland 7602, South Africa; Department of Ecology and Evolution, J. W.
      Goethe-University, Biologicum, Frankfurt am Main 60438, Germany; Chengdu
      Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, Peoples
      Republic of China; Department of Animal Ecology and Systematics, Justus Liebig
      University, Giessen 35392, Germany; and Department of Biology, Northern Michigan 
      University, Marquette, MI 49855-5376, USA srd@sun.ac.za.
FAU - Phiri, Ethel E
AU  - Phiri EE
AD  - Department of Botany and Zoology, Private Bag X1, University of Stellenbosch,
      Matieland 7602, South Africa; Department of Ecology and Evolution, J. W.
      Goethe-University, Biologicum, Frankfurt am Main 60438, Germany; Chengdu
      Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, Peoples
      Republic of China; Department of Animal Ecology and Systematics, Justus Liebig
      University, Giessen 35392, Germany; and Department of Biology, Northern Michigan 
      University, Marquette, MI 49855-5376, USA.
FAU - Klaus, Sebastian
AU  - Klaus S
AD  - Department of Botany and Zoology, Private Bag X1, University of Stellenbosch,
      Matieland 7602, South Africa; Department of Ecology and Evolution, J. W.
      Goethe-University, Biologicum, Frankfurt am Main 60438, Germany; Chengdu
      Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, Peoples
      Republic of China; Department of Animal Ecology and Systematics, Justus Liebig
      University, Giessen 35392, Germany; and Department of Biology, Northern Michigan 
      University, Marquette, MI 49855-5376, USA Department of Botany and Zoology,
      Private Bag X1, University of Stellenbosch, Matieland 7602, South Africa;
      Department of Ecology and Evolution, J. W. Goethe-University, Biologicum,
      Frankfurt am Main 60438, Germany; Chengdu Institute of Biology, Chinese Academy
      of Sciences, Chengdu 610041, Peoples Republic of China; Department of Animal
      Ecology and Systematics, Justus Liebig University, Giessen 35392, Germany; and
      Department of Biology, Northern Michigan University, Marquette, MI 49855-5376,
      USA.
FAU - Albrecht, Christian
AU  - Albrecht C
AD  - Department of Botany and Zoology, Private Bag X1, University of Stellenbosch,
      Matieland 7602, South Africa; Department of Ecology and Evolution, J. W.
      Goethe-University, Biologicum, Frankfurt am Main 60438, Germany; Chengdu
      Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, Peoples
      Republic of China; Department of Animal Ecology and Systematics, Justus Liebig
      University, Giessen 35392, Germany; and Department of Biology, Northern Michigan 
      University, Marquette, MI 49855-5376, USA.
FAU - Cumberlidge, Neil
AU  - Cumberlidge N
AD  - Department of Botany and Zoology, Private Bag X1, University of Stellenbosch,
      Matieland 7602, South Africa; Department of Ecology and Evolution, J. W.
      Goethe-University, Biologicum, Frankfurt am Main 60438, Germany; Chengdu
      Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, Peoples
      Republic of China; Department of Animal Ecology and Systematics, Justus Liebig
      University, Giessen 35392, Germany; and Department of Biology, Northern Michigan 
      University, Marquette, MI 49855-5376, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150203
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Histones)
RN  - EC 1.9.3.1 (Electron Transport Complex IV)
SB  - IM
MH  - Africa
MH  - *Animal Distribution
MH  - Animals
MH  - Brachyura/*classification/genetics/*physiology
MH  - Electron Transport Complex IV/genetics
MH  - Fresh Water
MH  - Genes, rRNA/genetics
MH  - Geography
MH  - Histones/genetics
MH  - *Phylogeny
MH  - Time
OTO - NOTNLM
OT  - Eocene
OT  - Potamonautidae
OT  - paleodrainage
OT  - sub-Saharan Africa and associated islands
OT  - zoogeography
EDAT- 2015/02/05 06:00
MHDA- 2015/09/22 06:00
CRDT- 2015/02/05 06:00
PHST- 2014/11/15 00:00 [received]
PHST- 2015/01/28 00:00 [accepted]
PHST- 2015/02/05 06:00 [entrez]
PHST- 2015/02/05 06:00 [pubmed]
PHST- 2015/09/22 06:00 [medline]
AID - syv011 [pii]
AID - 10.1093/sysbio/syv011 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jul;64(4):549-67. doi: 10.1093/sysbio/syv011. Epub 2015 Feb 3.

PMID- 25634098
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Multilocus species trees show the recent adaptive radiation of the mimetic
      heliconius butterflies.
PG  - 505-24
LID - 10.1093/sysbio/syv007 [doi]
AB  - Mullerian mimicry among Neotropical Heliconiini butterflies is an excellent
      example of natural selection, associated with the diversification of a large
      continental-scale radiation. Some of the processes driving the evolution of
      mimicry rings are likely to generate incongruent phylogenetic signals across the 
      assemblage, and thus pose a challenge for systematics. We use a data set of 22
      mitochondrial and nuclear markers from 92% of species in the tribe, obtained by
      Sanger sequencing and de novo assembly of short read data, to re-examine the
      phylogeny of Heliconiini with both supermatrix and multispecies coalescent
      approaches, characterize the patterns of conflicting signal, and compare the
      performance of various methodological approaches to reflect the heterogeneity
      across the data. Despite the large extent of reticulate signal and strong
      conflict between markers, nearly identical topologies are consistently recovered 
      by most of the analyses, although the supermatrix approach failed to reflect the 
      underlying variation in the history of individual loci. However, the supermatrix 
      represents a useful approximation where multiple rare species represented by
      short sequences can be incorporated easily. The first comprehensive,
      time-calibrated phylogeny of this group is used to test the hypotheses of a
      diversification rate increase driven by the dramatic environmental changes in the
      Neotropics over the past 23 myr, or changes caused by diversity-dependent effects
      on the rate of diversification. We find that the rate of diversification has
      increased on the branch leading to the presently most species-rich genus
      Heliconius, but the change occurred gradually and cannot be unequivocally
      attributed to a specific environmental driver. Our study provides comprehensive
      comparison of philosophically distinct species tree reconstruction methods and
      provides insights into the diversification of an important insect radiation in
      the most biodiverse region of the planet.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Kozak, Krzysztof M
AU  - Kozak KM
AD  - Butterfly Genetics Group, Department of Zoology, University of Cambridge, CB2 3EJ
      Cambridge, UK; Laboratory of Genetics, Department of Biology, University of
      Turku, 20014 Turku, Finland; Department of Entomology, The Natural History
      Museum, London SW7 5BD, UK; Department of Biology, University of York, YO10 5DD
      Heslington, York, UK; and Department of Organismic and Evolutionary Biology,
      Harvard University, Cambridge, MA 02138, USA kk443@cam.ac.uk.
FAU - Wahlberg, Niklas
AU  - Wahlberg N
AD  - Butterfly Genetics Group, Department of Zoology, University of Cambridge, CB2 3EJ
      Cambridge, UK; Laboratory of Genetics, Department of Biology, University of
      Turku, 20014 Turku, Finland; Department of Entomology, The Natural History
      Museum, London SW7 5BD, UK; Department of Biology, University of York, YO10 5DD
      Heslington, York, UK; and Department of Organismic and Evolutionary Biology,
      Harvard University, Cambridge, MA 02138, USA.
FAU - Neild, Andrew F E
AU  - Neild AF
AD  - Butterfly Genetics Group, Department of Zoology, University of Cambridge, CB2 3EJ
      Cambridge, UK; Laboratory of Genetics, Department of Biology, University of
      Turku, 20014 Turku, Finland; Department of Entomology, The Natural History
      Museum, London SW7 5BD, UK; Department of Biology, University of York, YO10 5DD
      Heslington, York, UK; and Department of Organismic and Evolutionary Biology,
      Harvard University, Cambridge, MA 02138, USA.
FAU - Dasmahapatra, Kanchon K
AU  - Dasmahapatra KK
AD  - Butterfly Genetics Group, Department of Zoology, University of Cambridge, CB2 3EJ
      Cambridge, UK; Laboratory of Genetics, Department of Biology, University of
      Turku, 20014 Turku, Finland; Department of Entomology, The Natural History
      Museum, London SW7 5BD, UK; Department of Biology, University of York, YO10 5DD
      Heslington, York, UK; and Department of Organismic and Evolutionary Biology,
      Harvard University, Cambridge, MA 02138, USA.
FAU - Mallet, James
AU  - Mallet J
AD  - Butterfly Genetics Group, Department of Zoology, University of Cambridge, CB2 3EJ
      Cambridge, UK; Laboratory of Genetics, Department of Biology, University of
      Turku, 20014 Turku, Finland; Department of Entomology, The Natural History
      Museum, London SW7 5BD, UK; Department of Biology, University of York, YO10 5DD
      Heslington, York, UK; and Department of Organismic and Evolutionary Biology,
      Harvard University, Cambridge, MA 02138, USA.
FAU - Jiggins, Chris D
AU  - Jiggins CD
AD  - Butterfly Genetics Group, Department of Zoology, University of Cambridge, CB2 3EJ
      Cambridge, UK; Laboratory of Genetics, Department of Biology, University of
      Turku, 20014 Turku, Finland; Department of Entomology, The Natural History
      Museum, London SW7 5BD, UK; Department of Biology, University of York, YO10 5DD
      Heslington, York, UK; and Department of Organismic and Evolutionary Biology,
      Harvard University, Cambridge, MA 02138, USA.
LA  - eng
GR  - BB/G006903/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
GR  - H01439X/1/Biotechnology and Biological Sciences Research Council/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150128
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
SB  - IM
MH  - Animals
MH  - Butterflies/*classification/*genetics
MH  - Genetic Markers/genetics
MH  - *Genetic Speciation
MH  - *Phylogeny
MH  - Time
PMC - PMC4395847
OTO - NOTNLM
OT  - Amazonia
OT  - Lepidoptera
OT  - Miocene
OT  - diversification rate
OT  - incongruence
OT  - mimicry
OT  - multispecies coalescent
EDAT- 2015/01/31 06:00
MHDA- 2015/06/20 06:00
CRDT- 2015/01/31 06:00
PHST- 2014/04/08 00:00 [received]
PHST- 2015/01/23 00:00 [accepted]
PHST- 2015/01/31 06:00 [entrez]
PHST- 2015/01/31 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syv007 [pii]
AID - 10.1093/sysbio/syv007 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):505-24. doi: 10.1093/sysbio/syv007. Epub 2015 Jan 28.

PMID- 25634097
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Circumstances in which parsimony but not compatibility will be provably
      misleading.
PG  - 492-504
LID - 10.1093/sysbio/syv008 [doi]
AB  - Phylogenetic methods typically rely on an appropriate model of how data evolved
      in order to infer an accurate phylogenetic tree. For molecular data, standard
      statistical methods have provided an effective strategy for extracting
      phylogenetic information from aligned sequence data when each site (character) is
      subject to a common process. However, for other types of data (e.g.,
      morphological data), characters can be too ambiguous, homoplastic, or saturated
      to develop models that are effective at capturing the underlying process of
      change. To address this, we examine the properties of a classic but neglected
      method for inferring splits in an underlying tree, namely, maximum compatibility.
      By adopting a simple and extreme model in which each character either fits
      perfectly on some tree, or is entirely random (but it is not known which class
      any character belongs to) we are able to derive exact and explicit formulae
      regarding the performance of maximum compatibility. We show that this method is
      able to identify a set of non-trivial homoplasy-free characters, when the number 
      [Formula: see text] of taxa is large, even when the number of random characters
      is large. In contrast, we show that a method that makes more uniform use of all
      the data-maximum parsimony-can provably estimate trees in which none of the
      original homoplasy-free characters support splits.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Scotland, Robert W
AU  - Scotland RW
AD  - Department of Plant Sciences, Oxford University, Oxford OX1 3RB, UK;
FAU - Steel, Mike
AU  - Steel M
AD  - Biomathematics Research Centre, University of Canterbury, Christchurch, New
      Zealand mike.steel@canterbury.ac.nz.
LA  - eng
PT  - Journal Article
DEP - 20150128
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Models, Genetic
MH  - *Phylogeny
PMC - PMC4395848
OTO - NOTNLM
OT  - Character compatibility
OT  - homoplasy
OT  - parsimony
OT  - phylogenetic tree
EDAT- 2015/01/31 06:00
MHDA- 2015/06/20 06:00
CRDT- 2015/01/31 06:00
PHST- 2014/11/29 00:00 [received]
PHST- 2015/01/23 00:00 [accepted]
PHST- 2015/01/31 06:00 [entrez]
PHST- 2015/01/31 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syv008 [pii]
AID - 10.1093/sysbio/syv008 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):492-504. doi: 10.1093/sysbio/syv008. Epub 2015 Jan 28.

PMID- 25631175
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Quantifying MCMC exploration of phylogenetic tree space.
PG  - 472-91
LID - 10.1093/sysbio/syv006 [doi]
AB  - In order to gain an understanding of the effectiveness of phylogenetic Markov
      chain Monte Carlo (MCMC), it is important to understand how quickly the empirical
      distribution of the MCMC converges to the posterior distribution. In this
      article, we investigate this problem on phylogenetic tree topologies with a
      metric that is especially well suited to the task: the subtree prune-and-regraft 
      (SPR) metric. This metric directly corresponds to the minimum number of MCMC
      rearrangements required to move between trees in common phylogenetic MCMC
      implementations. We develop a novel graph-based approach to analyze tree
      posteriors and find that the SPR metric is much more informative than simpler
      metrics that are unrelated to MCMC moves. In doing so, we show conclusively that 
      topological peaks do occur in Bayesian phylogenetic posteriors from real data
      sets as sampled with standard MCMC approaches, investigate the efficiency of
      Metropolis-coupled MCMC (MCMCMC) in traversing the valleys between peaks, and
      show that conditional clade distribution (CCD) can have systematic problems when 
      there are multiple peaks.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Whidden, Chris
AU  - Whidden C
AD  - Program in Computational Biology, Fred Hutchinson Cancer Research Center,
      Seattle, WA 98109, USA cwhidden@fhcrc.org.
FAU - Matsen, Frederick A 4th
AU  - Matsen FA 4th
AD  - Program in Computational Biology, Fred Hutchinson Cancer Research Center,
      Seattle, WA 98109, USA.
LA  - eng
GR  - S10 OD020069/OD/NIH HHS/United States
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150127
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Archaea/classification/genetics
MH  - Bacteria/classification/genetics
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Eukaryota/classification/genetics
MH  - *Markov Chains
MH  - Models, Genetic
MH  - *Monte Carlo Method
MH  - *Phylogeny
PMC - PMC4395846
OTO - NOTNLM
OT  - Markov chain Monte Carlo
OT  - phylogenetic methods
OT  - subtree prune-and-regraft
OT  - topological peaks
OT  - tree space
EDAT- 2015/01/30 06:00
MHDA- 2015/06/20 06:00
CRDT- 2015/01/30 06:00
PHST- 2014/05/09 00:00 [received]
PHST- 2015/01/20 00:00 [accepted]
PHST- 2015/01/30 06:00 [entrez]
PHST- 2015/01/30 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syv006 [pii]
AID - 10.1093/sysbio/syv006 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):472-91. doi: 10.1093/sysbio/syv006. Epub 2015 Jan 27.

PMID- 25616375
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20150415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Ancestral state reconstruction, rate heterogeneity, and the evolution of reptile 
      viviparity.
PG  - 532-44
LID - 10.1093/sysbio/syv005 [doi]
AB  - Virtually all models for reconstructing ancestral states for discrete characters 
      make the crucial assumption that the trait of interest evolves at a uniform rate 
      across the entire tree. However, this assumption is unlikely to hold in many
      situations, particularly as ancestral state reconstructions are being performed
      on increasingly large phylogenies. Here, we show how failure to account for such 
      variable evolutionary rates can cause highly anomalous (and likely incorrect)
      results, while three methods that accommodate rate variability yield the
      opposite, more plausible, and more robust reconstructions. The random local clock
      method, implemented in BEAST, estimates the position and magnitude of rate
      changes on the tree; split BiSSE estimates separate rate parameters for
      pre-specified clades; and the hidden rates model partitions each character state 
      into a number of rate categories. Simulations show the inadequacy of traditional 
      models when characters evolve with both asymmetry (different rates of change
      between states within a character) and heterotachy (different rates of character 
      evolution across different clades). The importance of accounting for rate
      heterogeneity in ancestral state reconstruction is highlighted empirically with a
      new analysis of the evolution of viviparity in squamate reptiles, which reveal a 
      predominance of forward (oviparous-viviparous) transitions and very few
      reversals.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - King, Benedict
AU  - King B
AD  - School of Biological Sciences, Flinders University, PO Box 2100, Adelaide, South 
      Australia 5001; School of Biological Sciences, University of Adelaide, Adelaide, 
      South Australia 5005; and Earth Sciences Section, South Australian Museum, North 
      Terrace, Adelaide 5000, Australia.
FAU - Lee, Michael S Y
AU  - Lee MS
AD  - School of Biological Sciences, Flinders University, PO Box 2100, Adelaide, South 
      Australia 5001; School of Biological Sciences, University of Adelaide, Adelaide, 
      South Australia 5005; and Earth Sciences Section, South Australian Museum, North 
      Terrace, Adelaide 5000, Australia School of Biological Sciences, Flinders
      University, PO Box 2100, Adelaide, South Australia 5001; School of Biological
      Sciences, University of Adelaide, Adelaide, South Australia 5005; and Earth
      Sciences Section, South Australian Museum, North Terrace, Adelaide 5000,
      Australia Mike.Lee@samuseum.sa.gov.au.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150122
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - *Phylogeny
MH  - Reptiles/*classification/genetics/*physiology
MH  - Time
MH  - Viviparity, Nonmammalian/physiology
OTO - NOTNLM
OT  - Ancestral state reconstruction
OT  - BEAST
OT  - BiSSE
OT  - discrete characters
OT  - lizards
OT  - oviparity
OT  - rate heterogeneity
OT  - snakes
OT  - squamates
OT  - viviparity
EDAT- 2015/01/27 06:00
MHDA- 2015/06/20 06:00
CRDT- 2015/01/25 06:00
PHST- 2014/10/29 00:00 [received]
PHST- 2015/01/14 00:00 [accepted]
PHST- 2015/01/25 06:00 [entrez]
PHST- 2015/01/27 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syv005 [pii]
AID - 10.1093/sysbio/syv005 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):532-44. doi: 10.1093/sysbio/syv005. Epub 2015 Jan 22.

PMID- 25604357
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20150415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Assignment of homoeologs to parental genomes in allopolyploids for species tree
      inference, with an example from Fumaria (papaveraceae).
PG  - 448-71
LID - 10.1093/sysbio/syv004 [doi]
AB  - There is a rising awareness that species trees are best inferred from multiple
      loci while taking into account processes affecting individual gene trees, such as
      substitution model error (failure of the model to account for the complexity of
      the data) and coalescent stochasticity (presence of incomplete lineage sorting
      [ILS]). Although most studies have been carried out in the context of dichotomous
      species trees, these processes operate also in more complex evolutionary
      histories involving multiple hybridizations and polyploidy. Recently, methods
      have been developed that accurately handle ILS in allopolyploids, but they are
      thus far restricted to networks of diploids and tetraploids. We propose a
      procedure that improves on this limitation by designing a workflow that assigns
      homoeologs to hypothetical diploid ancestral genomes prior to genome tree
      construction. Conflicting assignment hypotheses are evaluated against
      substitution model error and coalescent stochasticity. Incongruence that cannot
      be explained by stochastic mechanisms needs to be explained by other processes
      (e.g., homoploid hybridization or paralogy). The data can then be filtered to
      build multilabeled genome phylogenies using inference methods that can recover
      species trees, either in the face of substitution model error and coalescent
      stochasticity alone, or while simultaneously accounting for hybridization.
      Methods are already available for folding the resulting multilabeled genome
      phylogeny into a network. We apply the workflow to the reconstruction of the
      reticulate phylogeny of the plant genus Fumaria (Papaveraceae) with ploidal
      levels ranging from 2[Formula: see text] to 14[Formula: see text]. We describe
      the challenges in recovering nuclear NRPB2 homoeologs in high ploidy species
      while combining in vivo cloning and direct sequencing techniques. Using
      parametric bootstrapping simulations we assign nuclear homoeologs and chloroplast
      sequences (four concatenated loci) to their common hypothetical diploid ancestral
      genomes. As these assignments hinge on effective population size assumptions, we 
      investigate how varying these assumptions impacts the recovered multilabeled
      genome phylogeny.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Bertrand, Yann J K
AU  - Bertrand YJ
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Gothenburg, Sweden; and Museum of Archaeology, University of
      Stavanger, NO-4036 Stavanger, Norway yjk_bertrand@ybertrand.org.
FAU - Scheen, Anne-Cathrine
AU  - Scheen AC
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Gothenburg, Sweden; and Museum of Archaeology, University of
      Stavanger, NO-4036 Stavanger, Norway Department of Biological and Environmental
      Sciences, University of Gothenburg, Box 461, SE-405 30 Gothenburg, Sweden; and
      Museum of Archaeology, University of Stavanger, NO-4036 Stavanger, Norway.
FAU - Marcussen, Thomas
AU  - Marcussen T
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Gothenburg, Sweden; and Museum of Archaeology, University of
      Stavanger, NO-4036 Stavanger, Norway.
FAU - Pfeil, Bernard E
AU  - Pfeil BE
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Gothenburg, Sweden; and Museum of Archaeology, University of
      Stavanger, NO-4036 Stavanger, Norway.
FAU - de Sousa, Filipe
AU  - de Sousa F
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Gothenburg, Sweden; and Museum of Archaeology, University of
      Stavanger, NO-4036 Stavanger, Norway.
FAU - Oxelman, Bengt
AU  - Oxelman B
AD  - Department of Biological and Environmental Sciences, University of Gothenburg,
      Box 461, SE-405 30 Gothenburg, Sweden; and Museum of Archaeology, University of
      Stavanger, NO-4036 Stavanger, Norway.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150120
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Chloroplasts/genetics
MH  - Classification/*methods
MH  - Fumaria/*classification/*genetics
MH  - Genome, Plant/*genetics
MH  - *Phylogeny
MH  - *Polyploidy
MH  - Sequence Homology
OTO - NOTNLM
OT  - Allopolyploidy
OT  - Fumaria
OT  - NRDP2
OT  - Papaveraceae
OT  - coalescent simulation
OT  - coalescent stochasticity
OT  - genome tree
OT  - homoeolog assignments
OT  - incomplete lineage sorting
OT  - species phylogeny
OT  - substitution model error
EDAT- 2015/01/22 06:00
MHDA- 2015/06/20 06:00
CRDT- 2015/01/22 06:00
PHST- 2014/02/17 00:00 [received]
PHST- 2015/01/14 00:00 [accepted]
PHST- 2015/01/22 06:00 [entrez]
PHST- 2015/01/22 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syv004 [pii]
AID - 10.1093/sysbio/syv004 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):448-71. doi: 10.1093/sysbio/syv004. Epub 2015 Jan 20.

PMID- 25601944
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20151015
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Delimiting Species-Poor Data Sets using Single Molecular Markers: A Study of
      Barcode Gaps, Haplowebs and GMYC.
PG  - 900-8
LID - 10.1093/sysbio/syu130 [doi]
AB  - Most single-locus molecular approaches to species delimitation available to date 
      have been designed and tested on data sets comprising at least tens of species,
      whereas the opposite case (species-poor data sets for which the hypothesis that
      all individuals are conspecific cannot by rejected beforehand) has rarely been
      the focus of such attempts. Here we compare the performance of barcode gap
      detection, haplowebs and generalized mixed Yule-coalescent (GMYC) models to
      delineate chimpanzees and bonobos using nuclear sequence markers, then apply
      these single-locus species delimitation methods to data sets of one, three, or
      six species simulated under a wide range of population sizes, speciation rates,
      mutation rates and sampling efforts. Our results show that barcode gap detection 
      and GMYC models are unable to delineate species properly in data sets composed of
      one or two species, two situations in which haplowebs outperform them. For data
      sets composed of three or six species, bGMYC and haplowebs outperform the
      single-threshold and multiple-threshold versions of GMYC, whereas a clear barcode
      gap is only observed when population sizes and speciation rates are both small.
      The latter conditions represent a "sweet spot" for molecular taxonomy where all
      the single-locus approaches tested work well; however, the performance of these
      methods decreases strongly when population sizes and speciation rates are high,
      suggesting that multilocus approaches may be necessary to tackle such cases.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Dellicour, Simon
AU  - Dellicour S
AD  - Department of Zoology, University of Oxford, Oxford OX1 3PS, UK and.
FAU - Flot, Jean-Francois
AU  - Flot JF
AD  - Department of Genetics, Evolution and Environment, University College London,
      London WC1E 6BT, UK j.flot@ucl.ac.uk.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150119
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (Genetic Markers)
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - DNA Barcoding, Taxonomic
MH  - *Data Interpretation, Statistical
MH  - Genetic Markers/genetics
MH  - Haplotypes/genetics
OTO - NOTNLM
OT  - Barcode gap
OT  - generalized mixed Yule-coalescent models
OT  - haplowebs
OT  - heterozygosity
OT  - molecular markers
OT  - species delimitation
OT  - systematics
OT  - taxonomy
EDAT- 2015/01/21 06:00
MHDA- 2016/04/08 06:00
CRDT- 2015/01/21 06:00
PHST- 2014/11/27 00:00 [received]
PHST- 2014/12/18 00:00 [accepted]
PHST- 2015/01/21 06:00 [entrez]
PHST- 2015/01/21 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syu130 [pii]
AID - 10.1093/sysbio/syu130 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):900-8. doi: 10.1093/sysbio/syu130. Epub 2015 Jan 19.

PMID- 25601943
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20150217
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - Model inadequacy and mistaken inferences of trait-dependent speciation.
PG  - 340-55
LID - 10.1093/sysbio/syu131 [doi]
AB  - Species richness varies widely across the tree of life, and there is great
      interest in identifying ecological, geographic, and other factors that affect
      rates of species proliferation. Recent methods for explicitly modeling the
      relationships among character states, speciation rates, and extinction rates on
      phylogenetic trees- BiSSE, QuaSSE, GeoSSE, and related models-have been widely
      used to test hypotheses about character state-dependent diversification rates.
      Here, we document the disconcerting ease with which neutral traits are inferred
      to have statistically significant associations with speciation rate. We first
      demonstrate this unfortunate effect for a known model assumption violation:
      shifts in speciation rate associated with a character not included in the model. 
      We further show that for many empirical phylogenies, characters simulated in the 
      absence of state-dependent diversification exhibit an even higher Type I error
      rate, indicating that the method is susceptible to additional, unknown model
      inadequacies. For traits that evolve slowly, the root cause appears to be a
      statistical framework that does not require replicated shifts in character state 
      and diversification. However, spurious associations between character state and
      speciation rate arise even for traits that lack phylogenetic signal, suggesting
      that phylogenetic pseudoreplication alone cannot fully explain the problem. The
      surprising severity of this phenomenon suggests that many trait-diversification
      relationships reported in the literature may not be real. More generally, we
      highlight the need for diagnosing and understanding the consequences of model
      inadequacy in phylogenetic comparative methods.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Rabosky, Daniel L
AU  - Rabosky DL
AD  - Museum of Zoology and Department of Ecology and Evolutionary Biology, University 
      of Michigan, Ann Arbor, MI 48109; and Department of Ecology, Evolution, and
      Behavior, University of Minnesota, Saint Paul, MN 55108, USA drabosky@umich.edu.
FAU - Goldberg, Emma E
AU  - Goldberg EE
AD  - Museum of Zoology and Department of Ecology and Evolutionary Biology, University 
      of Michigan, Ann Arbor, MI 48109; and Department of Ecology, Evolution, and
      Behavior, University of Minnesota, Saint Paul, MN 55108, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20150119
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Body Size
MH  - Classification
MH  - Computer Simulation
MH  - Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Whales/*anatomy &amp; histology/*classification
OTO - NOTNLM
OT  - Character evolution
OT  - extinction
OT  - macroevolution
OT  - speciation
OT  - statistics
EDAT- 2015/01/21 06:00
MHDA- 2015/04/17 06:00
CRDT- 2015/01/21 06:00
PHST- 2015/01/21 06:00 [entrez]
PHST- 2015/01/21 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu131 [pii]
AID - 10.1093/sysbio/syu131 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):340-55. doi: 10.1093/sysbio/syu131. Epub 2015 Jan 19.

PMID- 25595363
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20150415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Deflating trees: improving Bayesian branch-length estimates using informed
      priors.
PG  - 441-7
LID - 10.1093/sysbio/syv003 [doi]
AB  - Prior distributions can have a strong effect on the results of Bayesian analyses.
      However, no general consensus exists for how priors should be set in all
      circumstances. Branch-length priors are of particular interest for phylogenetics,
      because they affect many parameters and biologically relevant inferences have
      been shown to be sensitive to the chosen prior distribution. Here, we explore the
      use of outside information to set informed branch-length priors and compare
      inferences from these informed analyses to those using default settings. For both
      the commonly used exponential and the newly proposed compound Dirichlet prior
      distributions, the incorporation of relevant outside information improves
      inferences for data sets that have produced problematic branch- and tree-length
      estimates under default settings. We suggest that informed priors are worthy of
      further exploration for phylogenetics.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Nelson, Bradley J
AU  - Nelson BJ
AD  - Department of Biological Sciences, Louisiana State University, Baton Rouge, LA
      70803, USA.
FAU - Andersen, John J
AU  - Andersen JJ
AD  - Department of Biological Sciences, Louisiana State University, Baton Rouge, LA
      70803, USA.
FAU - Brown, Jeremy M
AU  - Brown JM
AD  - Department of Biological Sciences, Louisiana State University, Baton Rouge, LA
      70803, USA jembrown@lsu.edu.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150116
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Anura/classification
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - Corbicula/classification
MH  - Data Interpretation, Statistical
MH  - Lizards/classification
MH  - *Phylogeny
MH  - Software
OTO - NOTNLM
OT  - Bayesian phylogenetics
OT  - branch lengths
OT  - prior choice
EDAT- 2015/01/18 06:00
MHDA- 2015/06/20 06:00
CRDT- 2015/01/18 06:00
PHST- 2014/09/11 00:00 [received]
PHST- 2015/01/08 00:00 [accepted]
PHST- 2015/01/18 06:00 [entrez]
PHST- 2015/01/18 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syv003 [pii]
AID - 10.1093/sysbio/syv003 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):441-7. doi: 10.1093/sysbio/syv003. Epub 2015 Jan 16.

PMID- 25577605
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20150415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Phycas: software for Bayesian phylogenetic analysis.
PG  - 525-31
LID - 10.1093/sysbio/syu132 [doi]
AB  - Phycas is open source, freely available Bayesian phylogenetics software written
      primarily in C++ but with a Python interface. Phycas specializes in Bayesian
      model selection for nucleotide sequence data, particularly the estimation of
      marginal likelihoods, central to computing Bayes Factors. Marginal likelihoods
      can be estimated using newer methods (Thermodynamic Integration and Generalized
      Steppingstone) that are more accurate than the widely used Harmonic Mean
      estimator. In addition, Phycas supports two posterior predictive approaches to
      model selection: Gelfand-Ghosh and Conditional Predictive Ordinates. The General 
      Time Reversible family of substitution models, as well as a codon model, are
      available, and data can be partitioned with all parameters unlinked except tree
      topology and edge lengths. Phycas provides for analyses in which the prior on
      tree topologies allows polytomous trees as well as fully resolved trees, and
      provides for several choices for edge length priors, including a hierarchical
      model as well as the recently described compound Dirichlet prior, which helps
      avoid overly informative induced priors on tree length.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Lewis, Paul O
AU  - Lewis PO
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, CT 06269, USA; Department of Ecology and
      Evolution, University of Kansas, 1200 Sunnyside Avenue, Lawrence, KS 66045, USA; 
      and Department of Biology, Box 90338, Duke University, Durham, NC 27708, USA
      paul.lewis@uconn.edu.
FAU - Holder, Mark T
AU  - Holder MT
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, CT 06269, USA; Department of Ecology and
      Evolution, University of Kansas, 1200 Sunnyside Avenue, Lawrence, KS 66045, USA; 
      and Department of Biology, Box 90338, Duke University, Durham, NC 27708, USA.
FAU - Swofford, David L
AU  - Swofford DL
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, Unit 3043, Storrs, CT 06269, USA; Department of Ecology and
      Evolution, University of Kansas, 1200 Sunnyside Avenue, Lawrence, KS 66045, USA; 
      and Department of Biology, Box 90338, Duke University, Durham, NC 27708, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150109
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Bayes Theorem
MH  - Chlorophyta/classification/genetics
MH  - Classification/*methods
MH  - *Phylogeny
MH  - *Software
OTO - NOTNLM
OT  - Bayes Factor
OT  - Bayesian phylogenetics
OT  - conditional predictive ordinates
OT  - data partitioning
OT  - marginal likelihood
OT  - posterior predictive model selection
OT  - steppingstone method
EDAT- 2015/01/13 06:00
MHDA- 2015/06/20 06:00
CRDT- 2015/01/12 06:00
PHST- 2014/08/13 00:00 [received]
PHST- 2014/12/24 00:00 [accepted]
PHST- 2015/01/12 06:00 [entrez]
PHST- 2015/01/13 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syu132 [pii]
AID - 10.1093/sysbio/syu132 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):525-31. doi: 10.1093/sysbio/syu132. Epub 2015 Jan 9.

PMID- 25575504
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Age-dependent speciation can explain the shape of empirical phylogenies.
PG  - 432-40
LID - 10.1093/sysbio/syv001 [doi]
AB  - Tens of thousands of phylogenetic trees, describing the evolutionary
      relationships between hundreds of thousands of taxa, are readily obtainable from 
      various databases. From such trees, inferences can be made about the underlying
      macroevolutionary processes, yet remarkably these processes are still poorly
      understood. Simple and widely used evolutionary null models are problematic:
      Empirical trees show very different imbalance between the sizes of the daughter
      clades of ancestral taxa compared to what models predict. Obtaining a simple
      evolutionary model that is both biologically plausible and produces the imbalance
      seen in empirical trees is a challenging problem, to which none of the existing
      models provide a satisfying answer. Here we propose a simple, biologically
      plausible macroevolutionary model in which the rate of speciation decreases with 
      species age, whereas extinction rates can vary quite generally. We show that this
      model provides a remarkable fit to the thousands of trees stored in the online
      database TreeBase. The biological motivation for the identified age-dependent
      speciation process may be that recently evolved taxa often colonize new regions
      or niches and may initially experience little competition. These new taxa are
      thus more likely to give rise to further new taxa than a taxon that has remained 
      largely unchanged and is, therefore, well adapted to its niche. We show that
      age-dependent speciation may also be the result of different within-species
      populations following the same laws of lineage splitting to produce new species. 
      As the fit of our model to the tree database shows, this simple biological
      motivation provides an explanation for a long standing problem in macroevolution.
CI  - (c) The Author(s) 2015. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Hagen, Oskar
AU  - Hagen O
AD  - Institute of Integrative Biology, ETH Zurich, Universitatsstr. 16, 8092 Zurich,
      Switzerland; Institute for Marine and Antarctic Studies, University of Tasmania, 
      Private Bag 49, Hobart, Tasmania 7001, Australia; Allan Wilson Centre for
      Molecular Ecology and Evolution, Biomathematics Research Centre, University of
      Canterbury, Christchurch 8140, New Zealand; and Department of Biosystems Science 
      and Engineering, ETH Zurich, Mattenstrasse 26, 4058 Basel, Switzerland.
FAU - Hartmann, Klaas
AU  - Hartmann K
AD  - Institute of Integrative Biology, ETH Zurich, Universitatsstr. 16, 8092 Zurich,
      Switzerland; Institute for Marine and Antarctic Studies, University of Tasmania, 
      Private Bag 49, Hobart, Tasmania 7001, Australia; Allan Wilson Centre for
      Molecular Ecology and Evolution, Biomathematics Research Centre, University of
      Canterbury, Christchurch 8140, New Zealand; and Department of Biosystems Science 
      and Engineering, ETH Zurich, Mattenstrasse 26, 4058 Basel, Switzerland.
FAU - Steel, Mike
AU  - Steel M
AD  - Institute of Integrative Biology, ETH Zurich, Universitatsstr. 16, 8092 Zurich,
      Switzerland; Institute for Marine and Antarctic Studies, University of Tasmania, 
      Private Bag 49, Hobart, Tasmania 7001, Australia; Allan Wilson Centre for
      Molecular Ecology and Evolution, Biomathematics Research Centre, University of
      Canterbury, Christchurch 8140, New Zealand; and Department of Biosystems Science 
      and Engineering, ETH Zurich, Mattenstrasse 26, 4058 Basel, Switzerland.
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Institute of Integrative Biology, ETH Zurich, Universitatsstr. 16, 8092 Zurich,
      Switzerland; Institute for Marine and Antarctic Studies, University of Tasmania, 
      Private Bag 49, Hobart, Tasmania 7001, Australia; Allan Wilson Centre for
      Molecular Ecology and Evolution, Biomathematics Research Centre, University of
      Canterbury, Christchurch 8140, New Zealand; and Department of Biosystems Science 
      and Engineering, ETH Zurich, Mattenstrasse 26, 4058 Basel, Switzerland Institute 
      of Integrative Biology, ETH Zurich, Universitatsstr. 16, 8092 Zurich,
      Switzerland; Institute for Marine and Antarctic Studies, University of Tasmania, 
      Private Bag 49, Hobart, Tasmania 7001, Australia; Allan Wilson Centre for
      Molecular Ecology and Evolution, Biomathematics Research Centre, University of
      Canterbury, Christchurch 8140, New Zealand; and Department of Biosystems Science 
      and Engineering, ETH Zurich, Mattenstrasse 26, 4058 Basel, Switzerland
      tanja.stadler@bsse.ethz.ch.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20150108
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2017 Jul 1;66(4):657. PMID: 28180902
MH  - Computer Simulation
MH  - *Genetic Speciation
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Time
PMC - PMC4395845
OTO - NOTNLM
OT  - Birth-death process
OT  - Stochastic models
OT  - diversification
OT  - macroevolution
EDAT- 2015/01/13 06:00
MHDA- 2015/06/20 06:00
CRDT- 2015/01/11 06:00
PHST- 2014/07/08 00:00 [received]
PHST- 2015/01/02 00:00 [accepted]
PHST- 2015/01/11 06:00 [entrez]
PHST- 2015/01/13 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syv001 [pii]
AID - 10.1093/sysbio/syv001 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):432-40. doi: 10.1093/sysbio/syv001. Epub 2015 Jan 8.

PMID- 25540456
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20150217
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - Assessing approaches for inferring species trees from multi-copy genes.
PG  - 325-39
LID - 10.1093/sysbio/syu128 [doi]
AB  - With the availability of genomic sequence data, there is increasing interest in
      using genes with a possible history of duplication and loss for species tree
      inference. Here we assess the performance of both nonprobabilistic and
      probabilistic species tree inference approaches using gene duplication and loss
      and coalescence simulations. We evaluated the performance of gene tree parsimony 
      (GTP) based on duplication (Only-dup), duplication and loss (Dup-loss), and deep 
      coalescence (Deep-c) costs, the NJst distance method, the MulRF supertree method,
      and PHYLDOG, which jointly estimates gene trees and species tree using a
      hierarchical probabilistic model. We examined the effects of gene tree and
      species sampling, gene tree error, and duplication and loss rates on the accuracy
      of phylogenetic estimates. In the 10-taxon duplication and loss simulation
      experiments, MulRF is more accurate than the other methods when the duplication
      and loss rates are low, and Dup-loss is generally the most accurate when the
      duplication and loss rates are high. PHYLDOG performs well in 10-taxon
      duplication and loss simulations, but its run time is prohibitively long on
      larger data sets. In the larger duplication and loss simulation experiments,
      MulRF outperforms all other methods in experiments with at most 100 taxa;
      however, in the larger simulation, Dup-loss generally performs best. In all
      duplication and loss simulation experiments with more than 10 taxa, all methods
      perform better with more gene trees and fewer missing sequences, and they are all
      affected by gene tree error. Our results also highlight high levels of error in
      estimates of duplications and losses from GTP methods and demonstrate the
      usefulness of methods based on generic tree distances for large analyses.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Chaudhary, Ruchi
AU  - Chaudhary R
AD  - Department of Computer Science, Iowa State University, Ames, IA 50011, USA;
      Department of Biology, University of Florida, Gainesville, FL 32611, USA; and
      Universite de Lyon, Universite Lyon 1, CNRS, UMR 5558, Laboratoire de Biometrie
      et Biologie Evolutive, Villeurbanne F-69622, France Department of Computer
      Science, Iowa State University, Ames, IA 50011, USA; Department of Biology,
      University of Florida, Gainesville, FL 32611, USA; and Universite de Lyon,
      Universite Lyon 1, CNRS, UMR 5558, Laboratoire de Biometrie et Biologie
      Evolutive, Villeurbanne F-69622, France ruchic@ufl.edu.
FAU - Boussau, Bastien
AU  - Boussau B
AD  - Department of Computer Science, Iowa State University, Ames, IA 50011, USA;
      Department of Biology, University of Florida, Gainesville, FL 32611, USA; and
      Universite de Lyon, Universite Lyon 1, CNRS, UMR 5558, Laboratoire de Biometrie
      et Biologie Evolutive, Villeurbanne F-69622, France.
FAU - Burleigh, J Gordon
AU  - Burleigh JG
AD  - Department of Computer Science, Iowa State University, Ames, IA 50011, USA;
      Department of Biology, University of Florida, Gainesville, FL 32611, USA; and
      Universite de Lyon, Universite Lyon 1, CNRS, UMR 5558, Laboratoire de Biometrie
      et Biologie Evolutive, Villeurbanne F-69622, France.
FAU - Fernandez-Baca, David
AU  - Fernandez-Baca D
AD  - Department of Computer Science, Iowa State University, Ames, IA 50011, USA;
      Department of Biology, University of Florida, Gainesville, FL 32611, USA; and
      Universite de Lyon, Universite Lyon 1, CNRS, UMR 5558, Laboratoire de Biometrie
      et Biologie Evolutive, Villeurbanne F-69622, France.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20141223
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Gene Deletion
MH  - Gene Duplication
MH  - *Phylogeny
MH  - Sequence Analysis, DNA/*methods/standards
MH  - Software/standards
OTO - NOTNLM
OT  - Deep coalescence
OT  - MulRF
OT  - NJst
OT  - PHYLDOG
OT  - gene duplication
OT  - gene loss
OT  - gene tree parsimony
EDAT- 2014/12/30 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/12/26 06:00
PHST- 2014/12/26 06:00 [entrez]
PHST- 2014/12/30 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu128 [pii]
AID - 10.1093/sysbio/syu128 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):325-39. doi: 10.1093/sysbio/syu128. Epub 2014 Dec 23.

PMID- 25540455
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20150415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Taxon-rich phylogenomic analyses resolve the eukaryotic tree of life and reveal
      the power of subsampling by sites.
PG  - 406-15
LID - 10.1093/sysbio/syu126 [doi]
AB  - Most eukaryotic lineages are microbial, and many have only recently been sampled 
      for phylogenetic studies or remain in the "dark area" of the tree of life where
      there are no molecular data. To assess relationships among eukaryotic lineages,
      we perform a taxon-rich phylogenomic analysis including 232 eukaryotes selected
      to maximize taxonomic diversity and up to 1554 genes chosen as vertically
      inherited based on their broad distribution among eukaryotes. We also include
      sequences from 486 bacteria and 84 archaea to assess the impact of endosymbiotic 
      gene transfer (EGT) from plastids and to detect contamination. Overall, our
      analyses are consistent with other less taxon-rich estimates of the eukaryotic
      tree of life, and we recover strong support for five major clades: Amoebozoa,
      Excavata (without the genus Malawimonas), Opisthokonta, Archaeplastida, and SAR
      (Stramenopila, Alveolata, and Rhizaria). Our analyses also highlight the
      existence of "orphan" lineages, lineages that lack robust placement in the
      eukaryotic tree of life, and indicate the possibility of as yet undiscovered
      diversity. In analyses including bacteria and archaea, we find that approximately
      10% of the 1554 genes, which we choose because they are found in four or five of 
      the five major eukaryotic clades and hence may be more likely to be inherited
      vertically, appear to have been acquired from cyanobacteria through EGT in
      photosynthetic lineages. Removing these EGT genes places the green algae as
      sister to the glaucophytes instead of the red algae, suggesting that unknowingly 
      including genes of plastid origin, and combining them with genes of nuclear
      origin, may mislead phylogenetic estimates. Finally, the large size of our data
      set allows comparative analyses of subsets of data; alignments built from
      randomly sampled sites provide greater support, particularly for deep
      relationships, than do equivalent-sized data sets built from randomly sampled
      genes.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Katz, Laura A
AU  - Katz LA
AD  - Department of Biological Sciences, Smith College, Northampton, MA 01063, USA and 
      Program in Organismic and Evolutionary Biology, UMass-Amherst, Amherst MA 01003, 
      USA Department of Biological Sciences, Smith College, Northampton, MA 01063, USA 
      and Program in Organismic and Evolutionary Biology, UMass-Amherst, Amherst MA
      01003, USA lkatz@smith.edu.
FAU - Grant, Jessica R
AU  - Grant JR
AD  - Department of Biological Sciences, Smith College, Northampton, MA 01063, USA and 
      Program in Organismic and Evolutionary Biology, UMass-Amherst, Amherst MA 01003, 
      USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20141223
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Archaea/genetics
MH  - Bacteria/genetics
MH  - Classification/*methods
MH  - Eukaryota/*classification/*genetics
MH  - *Phylogeny
MH  - Plastids/genetics
OTO - NOTNLM
OT  - Endosymbiotic gene transfer
OT  - eukaryotic phylogeny
OT  - microbial diversity
OT  - phylogenetics
OT  - sampling strategy
OT  - tree of life
EDAT- 2014/12/30 06:00
MHDA- 2015/06/20 06:00
CRDT- 2014/12/26 06:00
PHST- 2014/03/09 00:00 [received]
PHST- 2014/12/15 00:00 [accepted]
PHST- 2014/12/26 06:00 [entrez]
PHST- 2014/12/30 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syu126 [pii]
AID - 10.1093/sysbio/syu126 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):406-15. doi: 10.1093/sysbio/syu126. Epub 2014 Dec 23.

PMID- 25540454
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Cophylogeny reconstruction via an approximate Bayesian computation.
PG  - 416-31
LID - 10.1093/sysbio/syu129 [doi]
AB  - Despite an increasingly vast literature on cophylogenetic reconstructions for
      studying host-parasite associations, understanding the common evolutionary
      history of such systems remains a problem that is far from being solved. Most
      algorithms for host-parasite reconciliation use an event-based model, where the
      events include in general (a subset of) cospeciation, duplication, loss, and host
      switch. All known parsimonious event-based methods then assign a cost to each
      type of event in order to find a reconstruction of minimum cost. The main problem
      with this approach is that the cost of the events strongly influences the
      reconciliation obtained. Some earlier approaches attempt to avoid this problem by
      finding a Pareto set of solutions and hence by considering event costs under some
      minimization constraints. To deal with this problem, we developed an algorithm,
      called Coala, for estimating the frequency of the events based on an approximate 
      Bayesian computation approach. The benefits of this method are 2-fold: (i) it
      provides more confidence in the set of costs to be used in a reconciliation, and 
      (ii) it allows estimation of the frequency of the events in cases where the data 
      set consists of trees with a large number of taxa. We evaluate our method on
      simulated and on biological data sets. We show that in both cases, for the same
      pair of host and parasite trees, different sets of frequencies for the events
      lead to equally probable solutions. Moreover, often these solutions differ
      greatly in terms of the number of inferred events. It appears crucial to take
      this into account before attempting any further biological interpretation of such
      reconciliations. More generally, we also show that the set of frequencies can
      vary widely depending on the input host and parasite trees. Indiscriminately
      applying a standard vector of costs may thus not be a good strategy.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Baudet, C
AU  - Baudet C
AD  - INRIA Grenoble Rhone-Alpes, 38330 Montbonnot Saint-Martin, France; Universite de 
      Lyon, F-69000 Lyon; Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et
      Biologie Evolutive, F-69622 Villeurbanne, France; Universita di Firenze,
      Dipartimento di Sistemi e Informatica, I-50134 Firenze, Italy; and Laboratoire
      Statistique et Genome, UMR CNRS 8071 &amp; USC INRA, Universite d'Evry INRIA Grenoble
      Rhone-Alpes, 38330 Montbonnot Saint-Martin, France; Universite de Lyon, F-69000
      Lyon; Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et Biologie
      Evolutive, F-69622 Villeurbanne, France; Universita di Firenze, Dipartimento di
      Sistemi e Informatica, I-50134 Firenze, Italy; and Laboratoire Statistique et
      Genome, UMR CNRS 8071 &amp; USC INRA, Universite d'Evry.
FAU - Donati, B
AU  - Donati B
AD  - INRIA Grenoble Rhone-Alpes, 38330 Montbonnot Saint-Martin, France; Universite de 
      Lyon, F-69000 Lyon; Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et
      Biologie Evolutive, F-69622 Villeurbanne, France; Universita di Firenze,
      Dipartimento di Sistemi e Informatica, I-50134 Firenze, Italy; and Laboratoire
      Statistique et Genome, UMR CNRS 8071 &amp; USC INRA, Universite d'Evry INRIA Grenoble
      Rhone-Alpes, 38330 Montbonnot Saint-Martin, France; Universite de Lyon, F-69000
      Lyon; Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et Biologie
      Evolutive, F-69622 Villeurbanne, France; Universita di Firenze, Dipartimento di
      Sistemi e Informatica, I-50134 Firenze, Italy; and Laboratoire Statistique et
      Genome, UMR CNRS 8071 &amp; USC INRA, Universite d'Evry INRIA Grenoble Rhone-Alpes,
      38330 Montbonnot Saint-Martin, France; Universite de Lyon, F-69000 Lyon;
      Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et Biologie Evolutive,
      F-69622 Villeurbanne, France; Universita di Firenze, Dipartimento di Sistemi e
      Informatica, I-50134 Firenze, Italy; and Laboratoire Statistique et Genome, UMR
      CNRS 8071 &amp; USC INRA, Universite d'Evry.
FAU - Sinaimeri, B
AU  - Sinaimeri B
AD  - INRIA Grenoble Rhone-Alpes, 38330 Montbonnot Saint-Martin, France; Universite de 
      Lyon, F-69000 Lyon; Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et
      Biologie Evolutive, F-69622 Villeurbanne, France; Universita di Firenze,
      Dipartimento di Sistemi e Informatica, I-50134 Firenze, Italy; and Laboratoire
      Statistique et Genome, UMR CNRS 8071 &amp; USC INRA, Universite d'Evry INRIA Grenoble
      Rhone-Alpes, 38330 Montbonnot Saint-Martin, France; Universite de Lyon, F-69000
      Lyon; Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et Biologie
      Evolutive, F-69622 Villeurbanne, France; Universita di Firenze, Dipartimento di
      Sistemi e Informatica, I-50134 Firenze, Italy; and Laboratoire Statistique et
      Genome, UMR CNRS 8071 &amp; USC INRA, Universite d'Evry.
FAU - Crescenzi, P
AU  - Crescenzi P
AD  - INRIA Grenoble Rhone-Alpes, 38330 Montbonnot Saint-Martin, France; Universite de 
      Lyon, F-69000 Lyon; Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et
      Biologie Evolutive, F-69622 Villeurbanne, France; Universita di Firenze,
      Dipartimento di Sistemi e Informatica, I-50134 Firenze, Italy; and Laboratoire
      Statistique et Genome, UMR CNRS 8071 &amp; USC INRA, Universite d'Evry.
FAU - Gautier, C
AU  - Gautier C
AD  - INRIA Grenoble Rhone-Alpes, 38330 Montbonnot Saint-Martin, France; Universite de 
      Lyon, F-69000 Lyon; Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et
      Biologie Evolutive, F-69622 Villeurbanne, France; Universita di Firenze,
      Dipartimento di Sistemi e Informatica, I-50134 Firenze, Italy; and Laboratoire
      Statistique et Genome, UMR CNRS 8071 &amp; USC INRA, Universite d'Evry INRIA Grenoble
      Rhone-Alpes, 38330 Montbonnot Saint-Martin, France; Universite de Lyon, F-69000
      Lyon; Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et Biologie
      Evolutive, F-69622 Villeurbanne, France; Universita di Firenze, Dipartimento di
      Sistemi e Informatica, I-50134 Firenze, Italy; and Laboratoire Statistique et
      Genome, UMR CNRS 8071 &amp; USC INRA, Universite d'Evry.
FAU - Matias, C
AU  - Matias C
AD  - INRIA Grenoble Rhone-Alpes, 38330 Montbonnot Saint-Martin, France; Universite de 
      Lyon, F-69000 Lyon; Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et
      Biologie Evolutive, F-69622 Villeurbanne, France; Universita di Firenze,
      Dipartimento di Sistemi e Informatica, I-50134 Firenze, Italy; and Laboratoire
      Statistique et Genome, UMR CNRS 8071 &amp; USC INRA, Universite d'Evry.
FAU - Sagot, M-F
AU  - Sagot MF
AD  - INRIA Grenoble Rhone-Alpes, 38330 Montbonnot Saint-Martin, France; Universite de 
      Lyon, F-69000 Lyon; Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et
      Biologie Evolutive, F-69622 Villeurbanne, France; Universita di Firenze,
      Dipartimento di Sistemi e Informatica, I-50134 Firenze, Italy; and Laboratoire
      Statistique et Genome, UMR CNRS 8071 &amp; USC INRA, Universite d'Evry INRIA Grenoble
      Rhone-Alpes, 38330 Montbonnot Saint-Martin, France; Universite de Lyon, F-69000
      Lyon; Universite Lyon 1; CNRS, UMR5558, Laboratoire de Biometrie et Biologie
      Evolutive, F-69622 Villeurbanne, France; Universita di Firenze, Dipartimento di
      Sistemi e Informatica, I-50134 Firenze, Italy; and Laboratoire Statistique et
      Genome, UMR CNRS 8071 &amp; USC INRA, Universite d'Evry marie-france.sagot@inria.fr.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141224
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Algorithms
MH  - Animals
MH  - Arthropods/classification/microbiology
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Host-Parasite Interactions
MH  - *Phylogeny
MH  - Wolbachia/classification/physiology
PMC - PMC4395844
OTO - NOTNLM
OT  - approximate Bayesian computation
OT  - cophylogeny
OT  - host/parasite systems
OT  - likelihood-free inference
EDAT- 2014/12/30 06:00
MHDA- 2015/06/20 06:00
CRDT- 2014/12/26 06:00
PHST- 2014/07/22 00:00 [received]
PHST- 2014/12/18 00:00 [accepted]
PHST- 2014/12/26 06:00 [entrez]
PHST- 2014/12/30 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syu129 [pii]
AID - 10.1093/sysbio/syu129 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):416-31. doi: 10.1093/sysbio/syu129. Epub 2014 Dec 24.

PMID- 25527198
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20150217
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - Among-character rate variation distributions in phylogenetic analysis of discrete
      morphological characters.
PG  - 307-24
LID - 10.1093/sysbio/syu098 [doi]
AB  - Likelihood-based methods are commonplace in phylogenetic systematics. Although
      much effort has been directed toward likelihood-based models for molecular data, 
      comparatively less work has addressed models for discrete morphological character
      (DMC) data. Among-character rate variation (ACRV) may confound phylogenetic
      analysis, but there have been few analyses of the magnitude and distribution of
      rate heterogeneity among DMCs. Using 76 data sets covering a range of plants,
      invertebrate, and vertebrate animals, we used a modified version of MrBayes to
      test equal, gamma-distributed and lognormally distributed models of ACRV,
      integrating across phylogenetic uncertainty using Bayesian model selection. We
      found that in approximately 80% of data sets, unequal-rates models outperformed
      equal-rates models, especially among larger data sets. Moreover, although most
      data sets were equivocal, more data sets favored the lognormal rate distribution 
      relative to the gamma rate distribution, lending some support for more complex
      character correlations than in molecular data. Parsimony estimation of the
      underlying rate distributions in several data sets suggests that the lognormal
      distribution is preferred when there are many slowly evolving characters and
      fewer quickly evolving characters. The commonly adopted four rate category
      discrete approximation used for molecular data was found to be sufficient to
      approximate a gamma rate distribution with discrete characters. However, among
      the two data sets tested that favored a lognormal rate distribution, the
      continuous distribution was better approximated with at least eight discrete rate
      categories. Although the effect of rate model on the estimation of topology was
      difficult to assess across all data sets, it appeared relatively minor between
      the unequal-rates models for the one data set examined carefully. As in molecular
      analyses, we argue that researchers should test and adopt the most appropriate
      model of rate variation for the data set in question. As discrete characters are 
      increasingly used in more sophisticated likelihood-based phylogenetic analyses,
      it is important that these studies be built on the most appropriate and carefully
      selected underlying models of evolution.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Harrison, Luke B
AU  - Harrison LB
AD  - Redpath Museum, McGill University, 859 Sherbrooke Street West Montreal, Quebec,
      Canada H3A 0C4 and Redpath Museum, McGill University, 859 Sherbrooke ST W,
      Montreal, Quebec, Canada H3A 0C4 luke.harrison@mail.mcgill.ca.
FAU - Larsson, Hans C E
AU  - Larsson HC
AD  - Redpath Museum, McGill University, 859 Sherbrooke Street West Montreal, Quebec,
      Canada H3A 0C4 and Redpath Museum, McGill University, 859 Sherbrooke ST W,
      Montreal, Quebec, Canada H3A 0C4.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141218
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bayes Theorem
MH  - Invertebrates/anatomy &amp; histology/*classification/genetics
MH  - Models, Theoretical
MH  - *Phylogeny
MH  - Plants/anatomy &amp; histology/*classification/genetics
MH  - Vertebrates/anatomy &amp; histology/*classification/genetics
OTO - NOTNLM
OT  - Among-character rate variation
OT  - Bayesian model selection
OT  - discrete morphological characters
OT  - morphological phylogenetics
OT  - phylogenetics
OT  - rate heterogeneity
EDAT- 2014/12/21 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/12/21 06:00
PHST- 2014/12/21 06:00 [entrez]
PHST- 2014/12/21 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu098 [pii]
AID - 10.1093/sysbio/syu098 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):307-24. doi: 10.1093/sysbio/syu098. Epub 2014 Dec 18.

PMID- 25516268
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20150217
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - The more, the better: the use of multiple landmark configurations to solve the
      phylogenetic relationships in musteloids.
PG  - 294-306
LID - 10.1093/sysbio/syu107 [doi]
AB  - Although the use of landmark data to study shape changes along a phylogenetic
      tree has become a common practice in evolutionary studies, the role of this sort 
      of data for the inference of phylogenetic relationships remains under debate.
      Theoretical issues aside, the very existence of historical information in
      landmark data has been challenged, since phylogenetic analyses have often shown
      little congruence with alternative sources of evidence. However, most analyses
      conducted in the past were based upon a single landmark configuration, leaving it
      unsettled whether the incorporation of multiple configurations may improve the
      rather poor performance of this data source in most previous phylogenetic
      analyses. In the present study, we present a phylogenetic analysis of landmark
      data that combines information derived from several skeletal structures to derive
      a phylogenetic tree for musteloids. The analysis includes nine configurations
      representing different skeletal structures for 24 species. The resulting tree
      presents several notable concordances with phylogenetic hypotheses derived from
      molecular data. In particular, Mephitidae, Procyonidae, and Lutrinae plus the
      genera Martes, Mustela, Galictis, and Procyon were retrieved as monophyletic. In 
      addition, other groupings were in agreement with molecular phylogenies or
      presented only minor discordances. Complementary analyses have also indicated
      that the results improve substantially when an increasing number of landmark
      configurations are included in the analysis. The results presented here thus
      highlight the importance of combining information from multiple structures to
      derive phylogenetic hypotheses from landmark data.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Catalano, Santiago A
AU  - Catalano SA
AD  - Consejo Nacional de Investigaciones Cientificas y Tecnicas; Unidad Ejecutora
      Lillo (UEL), Miguel Lillo 251, 4000 S.M. de Tucuman, Argentina; Museo Argentino
      de Ciencias Naturales "Bernardino Rivadavia", Division Mastozoologia, Av. Angel
      Gallardo 470. C1405DJ Buenos Aires, Argentina; and Departamento de Ciencias
      Basicas, Universidad Nacional de Lujan, Lujan, Buenos Aires, Argentina Consejo
      Nacional de Investigaciones Cientificas y Tecnicas; Unidad Ejecutora Lillo (UEL),
      Miguel Lillo 251, 4000 S.M. de Tucuman, Argentina; Museo Argentino de Ciencias
      Naturales "Bernardino Rivadavia", Division Mastozoologia, Av. Angel Gallardo 470.
      C1405DJ Buenos Aires, Argentina; and Departamento de Ciencias Basicas,
      Universidad Nacional de Lujan, Lujan, Buenos Aires, Argentina
      sacatalano@gmail.com.
FAU - Ercoli, Marcos D
AU  - Ercoli MD
AD  - Consejo Nacional de Investigaciones Cientificas y Tecnicas; Unidad Ejecutora
      Lillo (UEL), Miguel Lillo 251, 4000 S.M. de Tucuman, Argentina; Museo Argentino
      de Ciencias Naturales "Bernardino Rivadavia", Division Mastozoologia, Av. Angel
      Gallardo 470. C1405DJ Buenos Aires, Argentina; and Departamento de Ciencias
      Basicas, Universidad Nacional de Lujan, Lujan, Buenos Aires, Argentina Consejo
      Nacional de Investigaciones Cientificas y Tecnicas; Unidad Ejecutora Lillo (UEL),
      Miguel Lillo 251, 4000 S.M. de Tucuman, Argentina; Museo Argentino de Ciencias
      Naturales "Bernardino Rivadavia", Division Mastozoologia, Av. Angel Gallardo 470.
      C1405DJ Buenos Aires, Argentina; and Departamento de Ciencias Basicas,
      Universidad Nacional de Lujan, Lujan, Buenos Aires, Argentina.
FAU - Prevosti, Francisco J
AU  - Prevosti FJ
AD  - Consejo Nacional de Investigaciones Cientificas y Tecnicas; Unidad Ejecutora
      Lillo (UEL), Miguel Lillo 251, 4000 S.M. de Tucuman, Argentina; Museo Argentino
      de Ciencias Naturales "Bernardino Rivadavia", Division Mastozoologia, Av. Angel
      Gallardo 470. C1405DJ Buenos Aires, Argentina; and Departamento de Ciencias
      Basicas, Universidad Nacional de Lujan, Lujan, Buenos Aires, Argentina Consejo
      Nacional de Investigaciones Cientificas y Tecnicas; Unidad Ejecutora Lillo (UEL),
      Miguel Lillo 251, 4000 S.M. de Tucuman, Argentina; Museo Argentino de Ciencias
      Naturales "Bernardino Rivadavia", Division Mastozoologia, Av. Angel Gallardo 470.
      C1405DJ Buenos Aires, Argentina; and Departamento de Ciencias Basicas,
      Universidad Nacional de Lujan, Lujan, Buenos Aires, Argentina Consejo Nacional de
      Investigaciones Cientificas y Tecnicas; Unidad Ejecutora Lillo (UEL), Miguel
      Lillo 251, 4000 S.M. de Tucuman, Argentina; Museo Argentino de Ciencias Naturales
      "Bernardino Rivadavia", Division Mastozoologia, Av. Angel Gallardo 470. C1405DJ
      Buenos Aires, Argentina; and Departamento de Ciencias Basicas, Universidad
      Nacional de Lujan, Lujan, Buenos Aires, Argentina.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141215
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bone and Bones/anatomy &amp; histology
MH  - Carnivora/anatomy &amp; histology/*classification
MH  - Classification/*methods
MH  - Humerus/anatomy &amp; histology
MH  - *Phylogeny
OTO - NOTNLM
OT  - Landmark data
OT  - Musteloidea
OT  - multiple configurations
OT  - parsimony
OT  - phylogenetic analysis
OT  - shape characters
EDAT- 2014/12/18 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/12/18 06:00
PHST- 2014/12/18 06:00 [entrez]
PHST- 2014/12/18 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu107 [pii]
AID - 10.1093/sysbio/syu107 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):294-306. doi: 10.1093/sysbio/syu107. Epub 2014 Dec 15.

PMID- 25504792
OWN - NLM
STAT- PubMed-not-MEDLINE
DCOM- 20150529
LR  - 20141216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - Erratum.
PG  - 167
LID - 10.1093/sysbio/syu091 [doi]
LA  - eng
PT  - Published Erratum
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
EFR - Syst Biol. 2014 Nov;63(6):951-70. PMID: 25123369
EDAT- 2014/12/17 06:00
MHDA- 2014/12/17 06:01
CRDT- 2014/12/16 06:00
PHST- 2014/12/16 06:00 [entrez]
PHST- 2014/12/17 06:00 [pubmed]
PHST- 2014/12/17 06:01 [medline]
AID - syu091 [pii]
AID - 10.1093/sysbio/syu091 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):167. doi: 10.1093/sysbio/syu091.

PMID- 25504791
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20141216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - Mathematical and computational evolutionary biology (2013).
PG  - 1-2
LID - 10.1093/sysbio/syu086 [doi]
FAU - Gascuel, Olivier
AU  - Gascuel O
AD  - Institut de Biologie Computationnelle (IBC), LIRMM, UMR5506, CNRS and Universite 
      de Montpellier, France and gascuel@lirmm.fr.
FAU - Stadler, Tanja
AU  - Stadler T
AD  - Department of Biosystems Science and Engineering, Eidgenossische Technische
      Hochschule Zurich, 4058 Basel, Switzerland.
LA  - eng
PT  - Introductory Journal Article
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - *Computational Biology
MH  - Congresses as Topic
MH  - *Models, Theoretical
EDAT- 2014/12/17 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/12/16 06:00
PHST- 2014/12/16 06:00 [entrez]
PHST- 2014/12/17 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu086 [pii]
AID - 10.1093/sysbio/syu086 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):1-2. doi: 10.1093/sysbio/syu086.

PMID- 25503979
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - Characterization of the uncertainty of divergence time estimation under relaxed
      molecular clock models using multiple loci.
PG  - 267-80
LID - 10.1093/sysbio/syu109 [doi]
AB  - Genetic sequence data provide information about the distances between species or 
      branch lengths in a phylogeny, but not about the absolute divergence times or the
      evolutionary rates directly. Bayesian methods for dating species divergences
      estimate times and rates by assigning priors on them. In particular, the prior on
      times (node ages on the phylogeny) incorporates information in the fossil record 
      to calibrate the molecular tree. Because times and rates are confounded, our
      posterior time estimates will not approach point values even if an infinite
      amount of sequence data are used in the analysis. In a previous study we
      developed a finite-sites theory to characterize the uncertainty in Bayesian
      divergence time estimation in analysis of large but finite sequence data sets
      under a strict molecular clock. As most modern clock dating analyses use more
      than one locus and are conducted under relaxed clock models, here we extend the
      theory to the case of relaxed clock analysis of data from multiple loci (site
      partitions). Uncertainty in posterior time estimates is partitioned into three
      sources: Sampling errors in the estimates of branch lengths in the tree for each 
      locus due to limited sequence length, variation of substitution rates among
      lineages and among loci, and uncertainty in fossil calibrations. Using a simple
      but analogous estimation problem involving the multivariate normal distribution, 
      we predict that as the number of loci ([Formula: see text]) goes to infinity, the
      variance in posterior time estimates decreases and approaches the infinite-data
      limit at the rate of 1/[Formula: see text], and the limit is independent of the
      number of sites in the sequence alignment. We then confirmed the predictions by
      using computer simulation on phylogenies of two or three species, and by
      analyzing a real genomic data set for six primate species. Our results suggest
      that with the fossil calibrations fixed, analyzing multiple loci or site
      partitions is the most effective way for improving the precision of posterior
      time estimation. However, even if a huge amount of sequence data is analyzed,
      considerable uncertainty will persist in time estimates.
CI  - (c) The Author(s) 2014. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - Zhu, Tianqi
AU  - Zhu T
AD  - Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing 100101, China
      Department of Genetics, Evolution and Environment, University College London,
      Darwin Building, Gower Street, London WC1E 6BT, UK.
FAU - Dos Reis, Mario
AU  - Dos Reis M
AD  - Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing 100101, China
      Department of Genetics, Evolution and Environment, University College London,
      Darwin Building, Gower Street, London WC1E 6BT, UK.
FAU - Yang, Ziheng
AU  - Yang Z
AD  - Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing 100101, China
      Department of Genetics, Evolution and Environment, University College London,
      Darwin Building, Gower Street, London WC1E 6BT, UK Beijing Institute of Genomics,
      Chinese Academy of Sciences, Beijing 100101, China Department of Genetics,
      Evolution and Environment, University College London, Darwin Building, Gower
      Street, London WC1E 6BT, UK z.yang@ucl.ac.uk.
LA  - eng
GR  - BB/J009709/1/Biotechnology and Biological Sciences Research Council/United
      Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141211
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Fossils
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Primates/classification/genetics
MH  - Sequence Analysis, DNA/*methods
MH  - Time
MH  - Uncertainty
PMC - PMC4380039
OTO - NOTNLM
OT  - Bayesian inference
OT  - divergence time
OT  - finite-sites theory
OT  - infinite-sites theory
OT  - posterior variance
OT  - relaxed clock
EDAT- 2014/12/17 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/12/16 06:00
PHST- 2014/12/16 06:00 [entrez]
PHST- 2014/12/17 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu109 [pii]
AID - 10.1093/sysbio/syu109 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):267-80. doi: 10.1093/sysbio/syu109. Epub 2014 Dec 11.

PMID- 25503772
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - Genetic distance for a general non-stationary markov substitution process.
PG  - 281-93
LID - 10.1093/sysbio/syu106 [doi]
AB  - The genetic distance between biological sequences is a fundamental quantity in
      molecular evolution. It pertains to questions of rates of evolution, existence of
      a molecular clock, and phylogenetic inference. Under the class of continuous-time
      substitution models, the distance is commonly defined as the expected number of
      substitutions at any site in the sequence. We eschew the almost ubiquitous
      assumptions of evolution under stationarity and time-reversible conditions and
      extend the concept of the expected number of substitutions to nonstationary
      Markov models where the only remaining constraint is of time homogeneity between 
      nodes in the tree. Our measure of genetic distance reduces to the standard
      formulation if the data in question are consistent with the stationarity
      assumption. We apply this general model to samples from across the tree of life
      to compare distances so obtained with those from the general time-reversible
      model, with and without rate heterogeneity across sites, and the paralinear
      distance, an empirical pairwise method explicitly designed to address
      nonstationarity. We discover that estimates from both variants of the general
      time-reversible model and the paralinear distance systematically overestimate
      genetic distance and departure from the molecular clock. The magnitude of the
      distance bias is proportional to departure from stationarity, which we
      demonstrate to be associated with longer edge lengths. The marked improvement in 
      consistency between the general nonstationary Markov model and sequence
      alignments leads us to conclude that analyses of evolutionary rates and
      phylogenies will be substantively improved by application of this model.
CI  - (c) The Author(s) 2014. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - Kaehler, Benjamin D
AU  - Kaehler BD
AD  - John Curtin School of Medical Research, Australian National University, Canberra,
      ACT, 2600, Australia; and.
FAU - Yap, Von Bing
AU  - Yap VB
AD  - Department of Statistics and Applied Probability, National University of
      Singapore, Singapore, 117546, Singapore.
FAU - Zhang, Rongli
AU  - Zhang R
AD  - Department of Statistics and Applied Probability, National University of
      Singapore, Singapore, 117546, Singapore.
FAU - Huttley, Gavin A
AU  - Huttley GA
AD  - John Curtin School of Medical Research, Australian National University, Canberra,
      ACT, 2600, Australia; and gavin.huttley@anu.edu.au ben.kaehler@anu.edu.au.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141209
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Evolution, Molecular
MH  - Humans
MH  - Mammals/classification/genetics
MH  - Markov Chains
MH  - *Models, Genetic
MH  - Phylogeny
PMC - PMC4380038
OTO - NOTNLM
OT  - Markov process
OT  - genetic distance
OT  - maximum likelihood
OT  - molecular clock
OT  - nonstationary
EDAT- 2014/12/17 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/12/16 06:00
PHST- 2014/12/16 06:00 [entrez]
PHST- 2014/12/17 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu106 [pii]
AID - 10.1093/sysbio/syu106 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):281-93. doi: 10.1093/sysbio/syu106. Epub 2014 Dec 9.

PMID- 25503771
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20150415
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Using more than the oldest fossils: dating osmundaceae with three Bayesian clock 
      approaches.
PG  - 396-405
LID - 10.1093/sysbio/syu108 [doi]
AB  - A major concern in molecular clock dating is how to use information from the
      fossil record to calibrate genetic distances from DNA sequences. Here we apply
      three Bayesian dating methods that differ in how calibration is achieved-"node
      dating" (ND) in BEAST, "total evidence" (TE) dating in MrBayes, and the
      "fossilized birth-death" (FBD) in FDPPDiv-to infer divergence times in the royal 
      ferns. Osmundaceae have 16-17 species in four genera, two mainly in the Northern 
      Hemisphere and two in South Africa and Australasia; they are the sister clade to 
      the remaining leptosporangiate ferns. Their fossil record consists of at least
      150 species in approximately 17 genera. For ND, we used the five oldest fossils, 
      whereas for TE and FBD dating, which do not require forcing fossils to nodes and 
      thus can use more fossils, we included up to 36 rhizomes and frond
      compression/impression fossils, which for TE dating were scored for 33
      morphological characters. We also subsampled 10%, 25%, and 50% of the 36 fossils 
      to assess model sensitivity. FBD-derived divergence ages were generally greater
      than those inferred from ND; two of seven TE-derived ages agreed with
      FBD-obtained ages, the others were much younger or much older than ND or FBD
      ages. We prefer the FBD-derived ages because they best fit the Osmundales fossil 
      record (including Triassic fossils not used in our study). Under the preferred
      model, the clade encompassing extant Osmundaceae (and many fossils) dates to the 
      latest Paleozoic to Early Triassic; divergences of the extant species occurred
      during the Neogene. Under the assumption of constant speciation and extinction
      rates, the FBD approach yielded speciation and extinction rates that overlapped
      those obtained from just neontological data. However, FBD estimates of speciation
      and extinction are sensitive to violations in the assumption of continuous fossil
      sampling; therefore, these estimates should be treated with caution.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Grimm, Guido W
AU  - Grimm GW
AD  - Department of Palaeobiology, Swedish Museum of Natural History, Svante Arrhenius 
      Vag 7, 10405 Stockholm, Sweden; Department of Palaeontology, University of
      Vienna, Althanstrasse 14, 1090 Vienna, Austria; Natural History Museum of Crete
      and Biology Department, University of Crete, PO Box 2208, 71409 Heraklion, Crete,
      Greece, and Scientific Computing, Heidelberg Institute for Theoretical Studies,
      Schloss-Wolfsbrunnenweg 35. 69118 Heidelberg, Germany; and Institute of
      Systematic Botany and Mycology, University of Munich, Menzinger Str. 67, 80638
      Munich, Germany Department of Palaeobiology, Swedish Museum of Natural History,
      Svante Arrhenius Vag 7, 10405 Stockholm, Sweden; Department of Palaeontology,
      University of Vienna, Althanstrasse 14, 1090 Vienna, Austria; Natural History
      Museum of Crete and Biology Department, University of Crete, PO Box 2208, 71409
      Heraklion, Crete, Greece, and Scientific Computing, Heidelberg Institute for
      Theoretical Studies, Schloss-Wolfsbrunnenweg 35. 69118 Heidelberg, Germany; and
      Institute of Systematic Botany and Mycology, University of Munich, Menzinger Str.
      67, 80638 Munich, Germany.
FAU - Kapli, Paschalia
AU  - Kapli P
AD  - Department of Palaeobiology, Swedish Museum of Natural History, Svante Arrhenius 
      Vag 7, 10405 Stockholm, Sweden; Department of Palaeontology, University of
      Vienna, Althanstrasse 14, 1090 Vienna, Austria; Natural History Museum of Crete
      and Biology Department, University of Crete, PO Box 2208, 71409 Heraklion, Crete,
      Greece, and Scientific Computing, Heidelberg Institute for Theoretical Studies,
      Schloss-Wolfsbrunnenweg 35. 69118 Heidelberg, Germany; and Institute of
      Systematic Botany and Mycology, University of Munich, Menzinger Str. 67, 80638
      Munich, Germany.
FAU - Bomfleur, Benjamin
AU  - Bomfleur B
AD  - Department of Palaeobiology, Swedish Museum of Natural History, Svante Arrhenius 
      Vag 7, 10405 Stockholm, Sweden; Department of Palaeontology, University of
      Vienna, Althanstrasse 14, 1090 Vienna, Austria; Natural History Museum of Crete
      and Biology Department, University of Crete, PO Box 2208, 71409 Heraklion, Crete,
      Greece, and Scientific Computing, Heidelberg Institute for Theoretical Studies,
      Schloss-Wolfsbrunnenweg 35. 69118 Heidelberg, Germany; and Institute of
      Systematic Botany and Mycology, University of Munich, Menzinger Str. 67, 80638
      Munich, Germany.
FAU - McLoughlin, Stephen
AU  - McLoughlin S
AD  - Department of Palaeobiology, Swedish Museum of Natural History, Svante Arrhenius 
      Vag 7, 10405 Stockholm, Sweden; Department of Palaeontology, University of
      Vienna, Althanstrasse 14, 1090 Vienna, Austria; Natural History Museum of Crete
      and Biology Department, University of Crete, PO Box 2208, 71409 Heraklion, Crete,
      Greece, and Scientific Computing, Heidelberg Institute for Theoretical Studies,
      Schloss-Wolfsbrunnenweg 35. 69118 Heidelberg, Germany; and Institute of
      Systematic Botany and Mycology, University of Munich, Menzinger Str. 67, 80638
      Munich, Germany.
FAU - Renner, Susanne S
AU  - Renner SS
AD  - Department of Palaeobiology, Swedish Museum of Natural History, Svante Arrhenius 
      Vag 7, 10405 Stockholm, Sweden; Department of Palaeontology, University of
      Vienna, Althanstrasse 14, 1090 Vienna, Austria; Natural History Museum of Crete
      and Biology Department, University of Crete, PO Box 2208, 71409 Heraklion, Crete,
      Greece, and Scientific Computing, Heidelberg Institute for Theoretical Studies,
      Schloss-Wolfsbrunnenweg 35. 69118 Heidelberg, Germany; and Institute of
      Systematic Botany and Mycology, University of Munich, Menzinger Str. 67, 80638
      Munich, Germany renner@lmu.de.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141209
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Calibration
MH  - Classification/*methods
MH  - Extinction, Biological
MH  - Ferns/*classification
MH  - *Fossils
MH  - Genetic Speciation
MH  - *Phylogeny
MH  - Time
OTO - NOTNLM
OT  - Bayesian inference
OT  - fossilized birth-death dating
OT  - molecular clock calibration
OT  - node dating
OT  - royal ferns
OT  - total evidence dating
EDAT- 2014/12/17 06:00
MHDA- 2015/06/20 06:00
CRDT- 2014/12/16 06:00
PHST- 2014/10/07 00:00 [received]
PHST- 2014/12/01 00:00 [accepted]
PHST- 2014/12/16 06:00 [entrez]
PHST- 2014/12/17 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syu108 [pii]
AID - 10.1093/sysbio/syu108 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):396-405. doi: 10.1093/sysbio/syu108. Epub 2014 Dec 9.

PMID- 25472575
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20190108
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Consensus and confusion in molluscan trees: evaluating morphological and
      molecular phylogenies.
PG  - 384-95
LID - 10.1093/sysbio/syu105 [doi]
AB  - Mollusks are the most morphologically disparate living animal phylum, they have
      diversified into all habitats, and have a deep fossil record. Monophyly and
      identity of their eight living classes is undisputed, but relationships between
      these groups and patterns of their early radiation have remained elusive.
      Arguments about traditional morphological phylogeny focus on a small number of
      topological concepts but often without regard to proximity of the individual
      classes. In contrast, molecular studies have proposed a number of radically
      different, inherently contradictory, and controversial sister relationships.
      Here, we assembled a data set of 42 unique published trees describing molluscan
      interrelationships. We used these data to ask several questions about the state
      of resolution of molluscan phylogeny compared with a null model of the variation 
      possible in random trees constructed from a monophyletic assemblage of eight
      terminals. Although 27 different unique trees have been proposed from
      morphological inference, the majority of these are not statistically different
      from each other. Within the available molecular topologies, only four studies to 
      date have included the deep sea class Monoplacophora; but 36.4% of all trees are 
      not significantly different. We also present supertrees derived from two data
      partitions and three methods, including all available molecular molluscan
      phylogenies, which will form the basis for future hypothesis testing. The
      supertrees presented here were not constructed to provide yet another hypothesis 
      of molluscan relationships, but rather to algorithmically evaluate the
      relationships present in the disparate published topologies. Based on the
      totality of available evidence, certain patterns of relatedness among constituent
      taxa become clear. The internodal distance is consistently short between a few
      taxon pairs, particularly supporting the relatedness of Monoplacophora and the
      chitons, Polyplacophora. Other taxon pairs are rarely or never found in close
      proximity, such as the vermiform Caudofoveata and Bivalvia. Our results have
      specific utility for guiding constructive research planning to better test
      relationships in Mollusca as well as other problematic groups. Taxa with
      consistently proximate relationships should be the focus of a combined approach
      in a concerted assessment of potential genetic and anatomical homology, whereas
      unequivocally distant taxa will make the most constructive choices for exemplar
      selection in higher level phylogenomic analyses.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Sigwart, Julia D
AU  - Sigwart JD
AD  - Marine Laboratory, Queen's University Belfast, BT22 1PF, Northern Ireland, UK;
      and Department of Integrative Biology, Museum of Paleontology and Center for
      Computational Biology, University of California, Berkeley, CA, 94720, USA
      j.sigwart@qub.ac.uk.
FAU - Lindberg, David R
AU  - Lindberg DR
AD  - Marine Laboratory, Queen's University Belfast, BT22 1PF, Northern Ireland, UK;
      and Department of Integrative Biology, Museum of Paleontology and Center for
      Computational Biology, University of California, Berkeley, CA, 94720, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141202
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Mollusca/anatomy &amp; histology/*classification/genetics
MH  - *Phylogeny
PMC - PMC4395843
OTO - NOTNLM
OT  - Aculifera
OT  - Conchifera
OT  - Mollusca
OT  - Serialia
OT  - Testaria
OT  - supertree
EDAT- 2014/12/05 06:00
MHDA- 2015/06/20 06:00
CRDT- 2014/12/05 06:00
PHST- 2013/07/16 00:00 [received]
PHST- 2014/11/21 00:00 [accepted]
PHST- 2014/12/05 06:00 [entrez]
PHST- 2014/12/05 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syu105 [pii]
AID - 10.1093/sysbio/syu105 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):384-95. doi: 10.1093/sysbio/syu105. Epub 2014 Dec 2.

PMID- 25432893
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - Bias and sensitivity in the placement of fossil taxa resulting from
      interpretations of missing data.
PG  - 256-66
LID - 10.1093/sysbio/syu093 [doi]
AB  - The utility of fossils in evolutionary contexts is dependent on their accurate
      placement in phylogenetic frameworks, yet intrinsic and widespread missing data
      make this problematic. The complex taphonomic processes occurring during
      fossilization can make it difficult to distinguish absence from non-preservation,
      especially in the case of exceptionally preserved soft-tissue fossils: is a
      particular morphological character (e.g., appendage, tentacle, or nerve) missing 
      from a fossil because it was never there (phylogenetic absence), or just happened
      to not be preserved (taphonomic loss)? Missing data have not been tested in the
      context of interpretation of non-present anatomy nor in the context of
      directional shifts and biases in affinity. Here, complete taxa, both simulated
      and empirical, are subjected to data loss through the replacement of present
      entries (1s) with either missing (?s) or absent (0s) entries. Both cause taxa to 
      drift down trees, from their original position, toward the root. Absolute
      thresholds at which downshift is significant are extremely low for introduced
      absences (two entries replaced, 6% of present characters). The opposite threshold
      in empirical fossil taxa is also found to be low; two absent entries replaced
      with presences causes fossil taxa to drift up trees. As such, only a few
      instances of non-preserved characters interpreted as absences will cause fossil
      organisms to be erroneously interpreted as more primitive than they were in life.
      This observed sensitivity to coding non-present morphology presents a problem for
      all evolutionary studies that attempt to use fossils to reconstruct rates of
      evolution or unlock sequences of morphological change. Stem-ward slippage,
      whereby fossilization processes cause organisms to appear artificially primitive,
      appears to be a ubiquitous and problematic phenomenon inherent to missing data,
      even when no decay biases exist. Absent characters therefore require explicit
      justification and taphonomic frameworks to support their interpretation.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Sansom, Robert S
AU  - Sansom RS
AD  - Faculty of Life Sciences, University of Manchester, Manchester M13 9PT, UK
      robert.sansom@manchester.ac.uk.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141127
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Bias
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - *Fossils
MH  - Invertebrates/anatomy &amp; histology/classification
MH  - *Phylogeny
PMC - PMC4380037
OTO - NOTNLM
OT  - Missing data
OT  - paleontology
OT  - phylogeny
OT  - soft-bodied
OT  - stem-group
OT  - taphonomy
EDAT- 2014/11/30 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/11/30 06:00
PHST- 2014/11/30 06:00 [entrez]
PHST- 2014/11/30 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu093 [pii]
AID - 10.1093/sysbio/syu093 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):256-66. doi: 10.1093/sysbio/syu093. Epub 2014 Nov 27.

PMID- 25432892
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - Bayesian long branch attraction bias and corrections.
PG  - 243-55
LID - 10.1093/sysbio/syu099 [doi]
AB  - Previous work on the star-tree paradox has shown that Bayesian methods suffer
      from a long branch attraction bias. That work is extended to settings involving
      more taxa and partially resolved trees. The long branch attraction bias is
      confirmed to arise more broadly and an additional source of bias is found. A
      by-product of the analysis is methods that correct for biases toward particular
      topologies. The corrections can be easily calculated using existing Bayesian
      software. Posterior support for a set of two or more trees can thus be
      supplemented with corrected versions to cross-check or replace results.
      Simulations show the corrections to be highly effective.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Susko, Edward
AU  - Susko E
AD  - Department of Mathematics and Statistics, Dalhousie University, Halifax, Nova
      Scotia, canada B3H, 4R2 edward.susko@gmail.com.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141127
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Bias
MH  - *Computer Simulation
MH  - *Models, Statistical
MH  - *Phylogeny
MH  - Software/*standards
OTO - NOTNLM
OT  - Bayesian methods
OT  - bias
OT  - long branch attraction
OT  - posterior probability
OT  - star tree
EDAT- 2014/11/30 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/11/30 06:00
PHST- 2014/11/30 06:00 [entrez]
PHST- 2014/11/30 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu099 [pii]
AID - 10.1093/sysbio/syu099 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):243-55. doi: 10.1093/sysbio/syu099. Epub 2014 Nov 27.

PMID- 25414176
OWN - NLM
STAT- MEDLINE
DCOM- 20160407
LR  - 20151015
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 6
DP  - 2015 Nov
TI  - Historical Species Distribution Models Predict Species Limits in Western
      Plethodon Salamanders.
PG  - 909-25
LID - 10.1093/sysbio/syu090 [doi]
AB  - Allopatry is commonly used to predict boundaries in species delimitation
      investigations under the assumption that currently allopatric distributions are
      indicative of reproductive isolation; however, species ranges are known to change
      over time. Incorporating a temporal perspective of geographic distributions
      should improve species delimitation; to explore this, we investigate three
      species of western Plethodon salamanders that have shifted their ranges since the
      end of the Pleistocene. We generate species distribution models (SDM) of the
      current range, hindcast these models onto a climatic model 21 Ka, and use three
      molecular approaches to delimit species in an integrated fashion. In contrast to 
      expectations based on the current distribution, we detect no independent lineages
      in species with allopatric and patchy distributions (Plethodon vandykei and
      Plethodon larselli). The SDMs indicate that probable habitat is more expansive
      than their current range, especially during the last glacial maximum (LGM) (21
      Ka). However, with a contiguous distribution, two independent lineages were
      detected in Plethodon idahoensis, possibly due to isolation in multiple glacial
      refugia. Results indicate that historical SDMs are a better predictor of species 
      boundaries than current distributions, and strongly imply that researchers should
      incorporate SDM and hindcasting into their investigations and the development of 
      species hypotheses.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Pelletier, Tara A
AU  - Pelletier TA
AD  - Department of Evolution, Ecology and Organismal Biology, Ohio State University,
      Columbus, OH 43201;
FAU - Crisafulli, Charlie
AU  - Crisafulli C
AD  - U.S. Forest Service, Pacific Northwest Research Station, Olympia, WA 98512;
FAU - Wagner, Steve
AU  - Wagner S
AD  - Department of Biological Sciences, Central Washington University, Ellensburg, WA 
      98926; and.
FAU - Zellmer, Amanda J
AU  - Zellmer AJ
AD  - Department of Biology, Occidental College, Los Angeles, CA 90041, USA.
FAU - Carstens, Bryan C
AU  - Carstens BC
AD  - Department of Evolution, Ecology and Organismal Biology, Ohio State University,
      Columbus, OH 43201; carstens.12@osu.edu.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20141119
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Animal Distribution
MH  - Animals
MH  - Climate
MH  - Ecosystem
MH  - *Models, Biological
MH  - Urodela/*physiology
OTO - NOTNLM
OT  - Coalescent
OT  - Pacific Northwest
OT  - niche model
OT  - range expansion
OT  - species delimitation
EDAT- 2014/11/22 06:00
MHDA- 2016/04/08 06:00
CRDT- 2014/11/22 06:00
PHST- 2014/08/13 00:00 [received]
PHST- 2014/11/10 00:00 [accepted]
PHST- 2014/11/22 06:00 [entrez]
PHST- 2014/11/22 06:00 [pubmed]
PHST- 2016/04/08 06:00 [medline]
AID - syu090 [pii]
AID - 10.1093/sysbio/syu090 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Nov;64(6):909-25. doi: 10.1093/sysbio/syu090. Epub 2014 Nov 19.

PMID- 25414175
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20150217
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - Weighted quartets phylogenetics.
PG  - 233-42
LID - 10.1093/sysbio/syu087 [doi]
AB  - Despite impressive technical and theoretical developments, reconstruction of
      phylogenetic trees for enormous quantities of molecular data is still a
      challenging task. A key tool in analyses of large data sets has been the
      construction of separate trees for subsets (e.g., quartets) of sequences, and
      subsequent combination of these subtrees into a single tree for the full set
      (i.e., supertree analysis). Unfortunately, even amalgamating quartets into a
      supertree remains a computationally daunting task. Assigning weights to quartets 
      to indicate importance or reliability was proposed more than a decade ago, but
      handling weighted quartets is even more challenging and has scarcely been
      attempted in the past. In this work, we focus on weighted quartet-based
      approaches. We propose a scheme to assign weights to quartets coming from
      weighted trees and devise a tree similarity measure for weighted trees based on
      weighted quartets. We also extend the quartet MaxCut (QMC algorithm) to handle
      weighted quartets. We evaluate these tools on simulated and real data. Our
      simulated data analysis highlights the additional information that is conveyed
      when using the new weighted tree similarity measure, and shows that extending QMC
      to a weighted setting improves the quality of tree reconstruction. Our analyses
      of a cyanobacterial data set with weighted QMC reinforce previous results
      achieved with other tools.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Avni, Eliran
AU  - Avni E
AD  - Department of Evolutionary Biology, University of Haifa, Haifa 31905, Israel;
      School of Engineering, Kinneret College, 15132, Israel; and Department of
      Evolutionary Biology, University of Haifa, Haifa 31905, Israel;
FAU - Cohen, Reuven
AU  - Cohen R
AD  - Department of Evolutionary Biology, University of Haifa, Haifa 31905, Israel;
      School of Engineering, Kinneret College, 15132, Israel; and Department of
      Evolutionary Biology, University of Haifa, Haifa 31905, Israel;
      ssagi@research.haifa.ac.il.
FAU - Snir, Sagi
AU  - Snir S
AD  - Department of Evolutionary Biology, University of Haifa, Haifa 31905, Israel;
      School of Engineering, Kinneret College, 15132, Israel; and Department of
      Evolutionary Biology, University of Haifa, Haifa 31905, Israel;
      ssagi@research.haifa.ac.il.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141119
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Classification/*methods
MH  - Cyanobacteria/classification
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Software
OTO - NOTNLM
OT  - Phylogenetic reconstruction
OT  - quartet maxcut
OT  - supertree reconstruction
OT  - weighted quartet trees
OT  - weighted tree similarity
EDAT- 2014/11/22 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/11/22 06:00
PHST- 2014/11/22 06:00 [entrez]
PHST- 2014/11/22 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu087 [pii]
AID - 10.1093/sysbio/syu087 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):233-42. doi: 10.1093/sysbio/syu087. Epub 2014 Nov 19.

PMID- 25398445
OWN - NLM
STAT- MEDLINE
DCOM- 20150619
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 3
DP  - 2015 May
TI  - Calibrated birth-death phylogenetic time-tree priors for bayesian inference.
PG  - 369-83
LID - 10.1093/sysbio/syu089 [doi]
AB  - Here we introduce a general class of multiple calibration birth-death tree priors
      for use in Bayesian phylogenetic inference. All tree priors in this class
      separate ancestral node heights into a set of "calibrated nodes" and
      "uncalibrated nodes" such that the marginal distribution of the calibrated nodes 
      is user-specified whereas the density ratio of the birth-death prior is retained 
      for trees with equal values for the calibrated nodes. We describe two
      formulations, one in which the calibration information informs the prior on
      ranked tree topologies, through the (conditional) prior, and the other which
      factorizes the prior on divergence times and ranked topologies, thus allowing
      uniform, or any arbitrary prior distribution on ranked topologies. Although the
      first of these formulations has some attractive properties, the algorithm we
      present for computing its prior density is computationally intensive. However,
      the second formulation is always faster and computationally efficient for up to
      six calibrations. We demonstrate the utility of the new class of
      multiple-calibration tree priors using both small simulations and a real-world
      analysis and compare the results to existing schemes. The two new calibrated tree
      priors described in this article offer greater flexibility and control of prior
      specification in calibrated time-tree inference and divergence time dating, and
      will remove the need for indirect approaches to the assessment of the combined
      effect of calibration densities and tree priors in Bayesian phylogenetic
      inference.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Heled, Joseph
AU  - Heled J
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, New Zealand; Department 
      of Computer Science, The University of Auckland, Auckland, New Zealand
      jheled@gmail.com.
FAU - Drummond, Alexei J
AU  - Drummond AJ
AD  - Allan Wilson Centre for Molecular Ecology and Evolution, New Zealand; Department 
      of Computer Science, The University of Auckland, Auckland, New Zealand Allan
      Wilson Centre for Molecular Ecology and Evolution, New Zealand; Department of
      Computer Science, The University of Auckland, Auckland, New Zealand.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141114
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
EIN - Syst Biol. 2016 Sep;65(5):943. PMID: 27226315
MH  - Algorithms
MH  - Animals
MH  - Anura/classification
MH  - Bayes Theorem
MH  - Calibration
MH  - Classification/*methods
MH  - *Computer Simulation
MH  - *Phylogeny
MH  - Time
MH  - Typhaceae/classification
PMC - PMC4395842
OTO - NOTNLM
OT  - BEAST
OT  - Bayesian inference
OT  - Yule prior
OT  - birth-death tree prior
OT  - fossil calibrations
OT  - multiple calibrations
EDAT- 2014/11/16 06:00
MHDA- 2015/06/20 06:00
CRDT- 2014/11/16 06:00
PHST- 2013/11/19 00:00 [received]
PHST- 2014/11/10 00:00 [accepted]
PHST- 2014/11/16 06:00 [entrez]
PHST- 2014/11/16 06:00 [pubmed]
PHST- 2015/06/20 06:00 [medline]
AID - syu089 [pii]
AID - 10.1093/sysbio/syu089 [doi]
PST - ppublish
SO  - Syst Biol. 2015 May;64(3):369-83. doi: 10.1093/sysbio/syu089. Epub 2014 Nov 14.

PMID- 25398444
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - Integrating fossils, phylogenies, and niche models into biogeography to reveal
      ancient evolutionary history: the case of Hypericum (hypericaceae).
PG  - 215-32
LID - 10.1093/sysbio/syu088 [doi]
AB  - In disciplines such as macroevolution that are not amenable to experimentation,
      scientists usually rely on current observations to test hypotheses about
      historical events, assuming that "the present is the key to the past."
      Biogeographers, for example, used this assumption to reconstruct ancestral ranges
      from the distribution of extant species. Yet, under scenarios of high extinction 
      rates, the biodiversity we observe today might not be representative of the
      historical diversity and this could result in incorrect biogeographic
      reconstructions. Here, we introduce a new approach to incorporate into
      biogeographic inference the temporal, spatial, and environmental information
      provided by the fossil record, as a direct evidence of the extinct biodiversity
      fraction. First, inferences of ancestral ranges for those nodes in the phylogeny 
      calibrated with the fossil record are constrained to include the geographic
      distribution of the fossil. Second, we use fossil distribution and past climate
      data to reconstruct the climatic preferences and potential distribution of
      ancestral lineages over time, and use this information to build a biogeographic
      model that takes into account "ecological connectivity" through time. To show the
      power of this approach, we reconstruct the biogeographic history of the large
      angiosperm genus Hypericum, which has a fossil record extending back to the Early
      Cenozoic. Unlike previous reconstructions based on extant species distributions, 
      our results reveal that Hypericum stem lineages were already distributed in the
      Holarctic before diversification of its crown-group, and that the geographic
      distribution of the genus has been relatively stable throughout the climatic
      oscillations of the Cenozoic. Geographical movement was mediated by the existence
      of climatic corridors, like Beringia, whereas the equatorial tropical belt acted 
      as a climatic barrier, preventing Hypericum lineages to reach the southern
      temperate regions. Our study shows that an integrative approach to historical
      biogeography-that combines sources of evidence as diverse as paleontology,
      ecology, and phylogenetics-could help us obtain more accurate reconstructions of 
      ancient evolutionary history. It also reveals the confounding effect different
      rates of extinction across regions have in biogeography, sometimes leading to
      ancestral areas being erroneously inferred as recent colonization events.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Meseguer, Andrea S
AU  - Meseguer AS
AD  - Department of Biodiversity and Conservation, Real Jardin Botanico, CSIC, Plaza de
      Murillo 2, 28014 Madrid, Spain; INRA, UMR 1062 CBGP Campus International de
      Baillarguet, 34988 Montferrier-sur-Lez, France; Department of Biogeography and
      Global Change, Museo Nacional Ciencias Naturales-CSIC, 28006 Madrid, Spain;
      Department of Botany, Field Museum of Natural History, Chicago, IL 60605, USA and
      Department of Animal and Plant Sciences, University of Sheffield, Sheffield S10
      2TN, UK Department of Biodiversity and Conservation, Real Jardin Botanico, CSIC, 
      Plaza de Murillo 2, 28014 Madrid, Spain; INRA, UMR 1062 CBGP Campus International
      de Baillarguet, 34988 Montferrier-sur-Lez, France; Department of Biogeography and
      Global Change, Museo Nacional Ciencias Naturales-CSIC, 28006 Madrid, Spain;
      Department of Botany, Field Museum of Natural History, Chicago, IL 60605, USA and
      Department of Animal and Plant Sciences, University of Sheffield, Sheffield S10
      2TN, UK asanchezmeseguer@gmail.com isanmartin@rjb.csic.es.
FAU - Lobo, Jorge M
AU  - Lobo JM
AD  - Department of Biodiversity and Conservation, Real Jardin Botanico, CSIC, Plaza de
      Murillo 2, 28014 Madrid, Spain; INRA, UMR 1062 CBGP Campus International de
      Baillarguet, 34988 Montferrier-sur-Lez, France; Department of Biogeography and
      Global Change, Museo Nacional Ciencias Naturales-CSIC, 28006 Madrid, Spain;
      Department of Botany, Field Museum of Natural History, Chicago, IL 60605, USA and
      Department of Animal and Plant Sciences, University of Sheffield, Sheffield S10
      2TN, UK.
FAU - Ree, Richard
AU  - Ree R
AD  - Department of Biodiversity and Conservation, Real Jardin Botanico, CSIC, Plaza de
      Murillo 2, 28014 Madrid, Spain; INRA, UMR 1062 CBGP Campus International de
      Baillarguet, 34988 Montferrier-sur-Lez, France; Department of Biogeography and
      Global Change, Museo Nacional Ciencias Naturales-CSIC, 28006 Madrid, Spain;
      Department of Botany, Field Museum of Natural History, Chicago, IL 60605, USA and
      Department of Animal and Plant Sciences, University of Sheffield, Sheffield S10
      2TN, UK.
FAU - Beerling, David J
AU  - Beerling DJ
AD  - Department of Biodiversity and Conservation, Real Jardin Botanico, CSIC, Plaza de
      Murillo 2, 28014 Madrid, Spain; INRA, UMR 1062 CBGP Campus International de
      Baillarguet, 34988 Montferrier-sur-Lez, France; Department of Biogeography and
      Global Change, Museo Nacional Ciencias Naturales-CSIC, 28006 Madrid, Spain;
      Department of Botany, Field Museum of Natural History, Chicago, IL 60605, USA and
      Department of Animal and Plant Sciences, University of Sheffield, Sheffield S10
      2TN, UK.
FAU - Sanmartin, Isabel
AU  - Sanmartin I
AD  - Department of Biodiversity and Conservation, Real Jardin Botanico, CSIC, Plaza de
      Murillo 2, 28014 Madrid, Spain; INRA, UMR 1062 CBGP Campus International de
      Baillarguet, 34988 Montferrier-sur-Lez, France; Department of Biogeography and
      Global Change, Museo Nacional Ciencias Naturales-CSIC, 28006 Madrid, Spain;
      Department of Botany, Field Museum of Natural History, Chicago, IL 60605, USA and
      Department of Animal and Plant Sciences, University of Sheffield, Sheffield S10
      2TN, UK.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141113
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Extinction, Biological
MH  - *Fossils
MH  - Geography
MH  - Hypericum/anatomy &amp; histology/*classification
MH  - *Models, Biological
MH  - *Phylogeny
MH  - Seeds/anatomy &amp; histology
PMC - PMC4380036
OTO - NOTNLM
OT  - Biogeography
OT  - Cenozoic climate change
OT  - Hypericum
OT  - environmental niche modeling
OT  - extinction
OT  - fossils
OT  - phylogenetics
EDAT- 2014/11/16 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/11/16 06:00
PHST- 2014/11/16 06:00 [entrez]
PHST- 2014/11/16 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu088 [pii]
AID - 10.1093/sysbio/syu088 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):215-32. doi: 10.1093/sysbio/syu088. Epub 2014 Nov 13.

PMID- 25378436
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20150217
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - Practical performance of tree comparison metrics.
PG  - 205-14
LID - 10.1093/sysbio/syu085 [doi]
AB  - The phylogenetic literature contains numerous measures for assessing differences 
      between two phylogenetic trees. Individual measures have been criticized on
      various grounds, but little is known about their comparative performance in
      typical applications. We evaluate the performance of nine tree distance measures 
      on two tasks: 1) distinguishing trees separated by lesser versus greater numbers 
      of recombinations, and 2) distinguishing trees inferred with lower versus higher 
      quality data. We find that when the trees being compared are similar, measures
      that make use of branch lengths are superior, with the branch-length version of
      the Robinson-Foulds metric performing best. In contrast, for dissimilar trees
      topology-only measures are superior, with the Alignment metric of Nye et al.
      performing best. We also apply the measures to a mammalian dataset and observe
      that the best metric depends on whether branch-length information is of interest.
      We give practical recommendations for choosing a tree distance metric in
      different applications.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Kuhner, Mary K
AU  - Kuhner MK
AD  - Department of Genome Sciences, Box 355065, University of Washington, Seattle, WA 
      98195-5065, USA. mkkuhner@uw.edu.
FAU - Yamato, Jon
AU  - Yamato J
AD  - Department of Genome Sciences, Box 355065, University of Washington, Seattle, WA 
      98195-5065, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20141104
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Mammals/classification/genetics
MH  - *Phylogeny
OTO - NOTNLM
OT  - Phylogenetics
OT  - tree comparison
OT  - tree distance metrics
EDAT- 2014/11/08 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/11/08 06:00
PHST- 2014/11/08 06:00 [entrez]
PHST- 2014/11/08 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu085 [pii]
AID - 10.1093/sysbio/syu085 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):205-14. doi: 10.1093/sysbio/syu085. Epub 2014 Nov 4.

PMID- 25358969
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - The phylogenetic likelihood library.
PG  - 356-62
LID - 10.1093/sysbio/syu084 [doi]
AB  - We introduce the Phylogenetic Likelihood Library (PLL), a highly optimized
      application programming interface for developing likelihood-based phylogenetic
      inference and postanalysis software. The PLL implements appropriate data
      structures and functions that allow users to quickly implement common,
      error-prone, and labor-intensive tasks, such as likelihood calculations, model
      parameter as well as branch length optimization, and tree space exploration. The 
      highly optimized and parallelized implementation of the phylogenetic likelihood
      function and a thorough documentation provide a framework for rapid development
      of scalable parallel phylogenetic software. By example of two likelihood-based
      phylogenetic codes we show that the PLL improves the sequential performance of
      current software by a factor of 2-10 while requiring only 1 month of programming 
      time for integration. We show that, when numerical scaling for preventing
      floating point underflow is enabled, the double precision likelihood calculations
      in the PLL are up to 1.9 times faster than those in BEAGLE. On an empirical DNA
      dataset with 2000 taxa the AVX version of PLL is 4 times faster than BEAGLE
      (scaling enabled and required). The PLL is available at http://www.libpll.org
      under the GNU General Public License (GPL).
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Flouri, T
AU  - Flouri T
AD  - Heidelberg Institute for Theoretical Studies, Heidelberg Institute, 69118
      Heidelberg, Germany; Center for Integrative Bioinformatics Vienna, Max F. Perutz 
      Laboratories, University of Vienna, Medical University of Vienna, A-1030 Vienna, 
      Austria; Bioinformatics and Computational Biology, Faculty of Computer Science,
      University of Vienna, A-1090 Vienna, Austria; and Karlsruhe Institute of
      Technology, Institute for Theoretical Informatics, Postfach 6980, 76128
      Karlsruhe, Germany; Tomas.Flouri@h-its.org.
FAU - Izquierdo-Carrasco, F
AU  - Izquierdo-Carrasco F
AD  - Heidelberg Institute for Theoretical Studies, Heidelberg Institute, 69118
      Heidelberg, Germany; Center for Integrative Bioinformatics Vienna, Max F. Perutz 
      Laboratories, University of Vienna, Medical University of Vienna, A-1030 Vienna, 
      Austria; Bioinformatics and Computational Biology, Faculty of Computer Science,
      University of Vienna, A-1090 Vienna, Austria; and Karlsruhe Institute of
      Technology, Institute for Theoretical Informatics, Postfach 6980, 76128
      Karlsruhe, Germany;
FAU - Darriba, D
AU  - Darriba D
AD  - Heidelberg Institute for Theoretical Studies, Heidelberg Institute, 69118
      Heidelberg, Germany; Center for Integrative Bioinformatics Vienna, Max F. Perutz 
      Laboratories, University of Vienna, Medical University of Vienna, A-1030 Vienna, 
      Austria; Bioinformatics and Computational Biology, Faculty of Computer Science,
      University of Vienna, A-1090 Vienna, Austria; and Karlsruhe Institute of
      Technology, Institute for Theoretical Informatics, Postfach 6980, 76128
      Karlsruhe, Germany;
FAU - Aberer, A J
AU  - Aberer AJ
AD  - Heidelberg Institute for Theoretical Studies, Heidelberg Institute, 69118
      Heidelberg, Germany; Center for Integrative Bioinformatics Vienna, Max F. Perutz 
      Laboratories, University of Vienna, Medical University of Vienna, A-1030 Vienna, 
      Austria; Bioinformatics and Computational Biology, Faculty of Computer Science,
      University of Vienna, A-1090 Vienna, Austria; and Karlsruhe Institute of
      Technology, Institute for Theoretical Informatics, Postfach 6980, 76128
      Karlsruhe, Germany;
FAU - Nguyen, L-T
AU  - Nguyen LT
AD  - Heidelberg Institute for Theoretical Studies, Heidelberg Institute, 69118
      Heidelberg, Germany; Center for Integrative Bioinformatics Vienna, Max F. Perutz 
      Laboratories, University of Vienna, Medical University of Vienna, A-1030 Vienna, 
      Austria; Bioinformatics and Computational Biology, Faculty of Computer Science,
      University of Vienna, A-1090 Vienna, Austria; and Karlsruhe Institute of
      Technology, Institute for Theoretical Informatics, Postfach 6980, 76128
      Karlsruhe, Germany; Heidelberg Institute for Theoretical Studies, Heidelberg
      Institute, 69118 Heidelberg, Germany; Center for Integrative Bioinformatics
      Vienna, Max F. Perutz Laboratories, University of Vienna, Medical University of
      Vienna, A-1030 Vienna, Austria; Bioinformatics and Computational Biology, Faculty
      of Computer Science, University of Vienna, A-1090 Vienna, Austria; and Karlsruhe 
      Institute of Technology, Institute for Theoretical Informatics, Postfach 6980,
      76128 Karlsruhe, Germany;
FAU - Minh, B Q
AU  - Minh BQ
AD  - Heidelberg Institute for Theoretical Studies, Heidelberg Institute, 69118
      Heidelberg, Germany; Center for Integrative Bioinformatics Vienna, Max F. Perutz 
      Laboratories, University of Vienna, Medical University of Vienna, A-1030 Vienna, 
      Austria; Bioinformatics and Computational Biology, Faculty of Computer Science,
      University of Vienna, A-1090 Vienna, Austria; and Karlsruhe Institute of
      Technology, Institute for Theoretical Informatics, Postfach 6980, 76128
      Karlsruhe, Germany;
FAU - Von Haeseler, A
AU  - Von Haeseler A
AD  - Heidelberg Institute for Theoretical Studies, Heidelberg Institute, 69118
      Heidelberg, Germany; Center for Integrative Bioinformatics Vienna, Max F. Perutz 
      Laboratories, University of Vienna, Medical University of Vienna, A-1030 Vienna, 
      Austria; Bioinformatics and Computational Biology, Faculty of Computer Science,
      University of Vienna, A-1090 Vienna, Austria; and Karlsruhe Institute of
      Technology, Institute for Theoretical Informatics, Postfach 6980, 76128
      Karlsruhe, Germany; Heidelberg Institute for Theoretical Studies, Heidelberg
      Institute, 69118 Heidelberg, Germany; Center for Integrative Bioinformatics
      Vienna, Max F. Perutz Laboratories, University of Vienna, Medical University of
      Vienna, A-1030 Vienna, Austria; Bioinformatics and Computational Biology, Faculty
      of Computer Science, University of Vienna, A-1090 Vienna, Austria; and Karlsruhe 
      Institute of Technology, Institute for Theoretical Informatics, Postfach 6980,
      76128 Karlsruhe, Germany;
FAU - Stamatakis, A
AU  - Stamatakis A
AD  - Heidelberg Institute for Theoretical Studies, Heidelberg Institute, 69118
      Heidelberg, Germany; Center for Integrative Bioinformatics Vienna, Max F. Perutz 
      Laboratories, University of Vienna, Medical University of Vienna, A-1030 Vienna, 
      Austria; Bioinformatics and Computational Biology, Faculty of Computer Science,
      University of Vienna, A-1090 Vienna, Austria; and Karlsruhe Institute of
      Technology, Institute for Theoretical Informatics, Postfach 6980, 76128
      Karlsruhe, Germany; Heidelberg Institute for Theoretical Studies, Heidelberg
      Institute, 69118 Heidelberg, Germany; Center for Integrative Bioinformatics
      Vienna, Max F. Perutz Laboratories, University of Vienna, Medical University of
      Vienna, A-1030 Vienna, Austria; Bioinformatics and Computational Biology, Faculty
      of Computer Science, University of Vienna, A-1090 Vienna, Austria; and Karlsruhe 
      Institute of Technology, Institute for Theoretical Informatics, Postfach 6980,
      76128 Karlsruhe, Germany;
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141030
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Algorithms
MH  - Classification/*methods
MH  - Libraries, Digital
MH  - *Phylogeny
MH  - *Software/standards
PMC - PMC4380035
OTO - NOTNLM
OT  - Maximum likelihood
OT  - parallel computing
OT  - phylogenetics
EDAT- 2014/11/02 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/11/01 06:00
PHST- 2014/11/01 06:00 [entrez]
PHST- 2014/11/02 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu084 [pii]
AID - 10.1093/sysbio/syu084 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):356-62. doi: 10.1093/sysbio/syu084. Epub 2014 Oct 30.

PMID- 25358968
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20141216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - From integrative taxonomy to species description: one step beyond.
PG  - 152-60
LID - 10.1093/sysbio/syu083 [doi]
FAU - Pante, E
AU  - Pante E
AD  - Littoral, Environnement et Societes (LIENSs) UMR 7266 CNRS, Universite de La
      Rochelle, La Rochelle, France; Fisheries and Oceans Canada, Aquatic animal
      health, 343, University Avenue E1C 9B6 Moncton N.B., Canada; and Museum National 
      d'Histoire Naturelle, Departement Systematique et Evolution, ISyEB (UMR 7205
      CNRS/UPMC/MNHN/EPHE), 43, Rue Cuvier, 75231 Paris, France.
FAU - Schoelinck, C
AU  - Schoelinck C
AD  - Littoral, Environnement et Societes (LIENSs) UMR 7266 CNRS, Universite de La
      Rochelle, La Rochelle, France; Fisheries and Oceans Canada, Aquatic animal
      health, 343, University Avenue E1C 9B6 Moncton N.B., Canada; and Museum National 
      d'Histoire Naturelle, Departement Systematique et Evolution, ISyEB (UMR 7205
      CNRS/UPMC/MNHN/EPHE), 43, Rue Cuvier, 75231 Paris, France.
FAU - Puillandre, N
AU  - Puillandre N
AD  - Littoral, Environnement et Societes (LIENSs) UMR 7266 CNRS, Universite de La
      Rochelle, La Rochelle, France; Fisheries and Oceans Canada, Aquatic animal
      health, 343, University Avenue E1C 9B6 Moncton N.B., Canada; and Museum National 
      d'Histoire Naturelle, Departement Systematique et Evolution, ISyEB (UMR 7205
      CNRS/UPMC/MNHN/EPHE), 43, Rue Cuvier, 75231 Paris, France puillandre@mnhn.fr.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141029
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Journal Impact Factor
MH  - Periodicals as Topic
MH  - Plants/classification
MH  - Research/standards/statistics &amp; numerical data
MH  - Species Specificity
EDAT- 2014/11/02 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/11/01 06:00
PHST- 2014/11/01 06:00 [entrez]
PHST- 2014/11/02 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu083 [pii]
AID - 10.1093/sysbio/syu083 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):152-60. doi: 10.1093/sysbio/syu083. Epub 2014 Oct 29.

PMID- 25305281
OWN - NLM
STAT- MEDLINE
DCOM- 20150416
LR  - 20150217
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 2
DP  - 2015 Mar
TI  - Evolution of neck vertebral shape and neck retraction at the transition to modern
      turtles: an integrated geometric morphometric approach.
PG  - 187-204
LID - 10.1093/sysbio/syu072 [doi]
AB  - The unique ability of modern turtles to retract their head and neck into the
      shell through a side-necked (pleurodiran) or hidden-necked (cryptodiran) motion
      is thought to have evolved independently in crown turtles. The anatomical changes
      that led to the vertebral shapes of modern turtles, however, are still poorly
      understood. Here we present comprehensive geometric morphometric analyses that
      trace turtle vertebral evolution and reconstruct disparity across phylogeny.
      Disparity of vertebral shape was high at the dawn of turtle evolution and
      decreased after the modern groups evolved, reflecting a stabilization of
      morphotypes that correspond to the two retraction modes. Stem turtles, which had 
      a very simple mode of retraction, the lateral head tuck, show increasing
      flexibility of the neck through evolution towards a pleurodiran-like morphotype. 
      The latter was the precondition for evolving pleurodiran and cryptodiran
      vertebrae. There is no correlation between the construction of formed
      articulations in the cervical centra and neck mobility. An increasing mobility
      between vertebrae, associated with changes in vertebral shape, resulted in a more
      advanced ability to retract the neck. In this regard, we hypothesize that the
      lateral tucking retraction of stem turtles was not only the precondition for
      pleurodiran but also of cryptodiran retraction. For the former, a kink in the
      middle third of the neck needed to be acquired, whereas for the latter
      modification was necessary between the eighth cervical vertebra and first
      thoracic vertebra. Our paper highlights the utility of 3D shape data, analyzed in
      a phylogenetic framework, to examine the magnitude and mode of evolutionary
      modifications to vertebral morphology. By reconstructing and visualizing
      ancestral anatomical shapes, we provide insight into the anatomical features
      underlying neck retraction mode, which is a salient component of extant turtle
      classification.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Werneburg, Ingmar
AU  - Werneburg I
AD  - Palantologisches Institut und Museum der Universitat Zurich, Karl-Schmid-Strasse 
      4, 8006 Zurich, Switzerland; Museum fur Naturkunde, Leibniz-Institut fur
      Evolutions- &amp; Biodiversitatsforschung an der Humboldt-Universitat zu Berlin,
      Invalidenstrasse 43, 10115 Berlin, Germany; School of Biological, Earth &amp;
      Environmental Sciences, University of New South Wales, Kensington NSW 2052,
      Australia; Surgical and Orthopaedic Research Laboratories, Prince of Wales
      Clinical School, University of New South Wales, Randwick , NSW 2031, Australia;
      and Department of Geosciences, University of Fribourg, 1700 Fribourg, Switzerland
      Palantologisches Institut und Museum der Universitat Zurich, Karl-Schmid-Strasse 
      4, 8006 Zurich, Switzerland; Museum fur Naturkunde, Leibniz-Institut fur
      Evolutions- &amp; Biodiversitatsforschung an der Humboldt-Universitat zu Berlin,
      Invalidenstrasse 43, 10115 Berlin, Germany; School of Biological, Earth &amp;
      Environmental Sciences, University of New South Wales, Kensington NSW 2052,
      Australia; Surgical and Orthopaedic Research Laboratories, Prince of Wales
      Clinical School, University of New South Wales, Randwick , NSW 2031, Australia;
      and Department of Geosciences, University of Fribourg, 1700 Fribourg, Switzerland
      i.werneburg@gmail.com laura.wilson@unsw.edu.au.
FAU - Wilson, Laura A B
AU  - Wilson LA
AD  - Palantologisches Institut und Museum der Universitat Zurich, Karl-Schmid-Strasse 
      4, 8006 Zurich, Switzerland; Museum fur Naturkunde, Leibniz-Institut fur
      Evolutions- &amp; Biodiversitatsforschung an der Humboldt-Universitat zu Berlin,
      Invalidenstrasse 43, 10115 Berlin, Germany; School of Biological, Earth &amp;
      Environmental Sciences, University of New South Wales, Kensington NSW 2052,
      Australia; Surgical and Orthopaedic Research Laboratories, Prince of Wales
      Clinical School, University of New South Wales, Randwick , NSW 2031, Australia;
      and Department of Geosciences, University of Fribourg, 1700 Fribourg, Switzerland
      i.werneburg@gmail.com laura.wilson@unsw.edu.au.
FAU - Parr, William C H
AU  - Parr WC
AD  - Palantologisches Institut und Museum der Universitat Zurich, Karl-Schmid-Strasse 
      4, 8006 Zurich, Switzerland; Museum fur Naturkunde, Leibniz-Institut fur
      Evolutions- &amp; Biodiversitatsforschung an der Humboldt-Universitat zu Berlin,
      Invalidenstrasse 43, 10115 Berlin, Germany; School of Biological, Earth &amp;
      Environmental Sciences, University of New South Wales, Kensington NSW 2052,
      Australia; Surgical and Orthopaedic Research Laboratories, Prince of Wales
      Clinical School, University of New South Wales, Randwick , NSW 2031, Australia;
      and Department of Geosciences, University of Fribourg, 1700 Fribourg,
      Switzerland.
FAU - Joyce, Walter G
AU  - Joyce WG
AD  - Palantologisches Institut und Museum der Universitat Zurich, Karl-Schmid-Strasse 
      4, 8006 Zurich, Switzerland; Museum fur Naturkunde, Leibniz-Institut fur
      Evolutions- &amp; Biodiversitatsforschung an der Humboldt-Universitat zu Berlin,
      Invalidenstrasse 43, 10115 Berlin, Germany; School of Biological, Earth &amp;
      Environmental Sciences, University of New South Wales, Kensington NSW 2052,
      Australia; Surgical and Orthopaedic Research Laboratories, Prince of Wales
      Clinical School, University of New South Wales, Randwick , NSW 2031, Australia;
      and Department of Geosciences, University of Fribourg, 1700 Fribourg,
      Switzerland.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141009
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - *Biological Evolution
MH  - Cervical Vertebrae/*anatomy &amp; histology
MH  - Neck/*anatomy &amp; histology
MH  - Phylogeny
MH  - Turtles/*anatomy &amp; histology/*classification
OTO - NOTNLM
OT  - 3D warping
OT  - Cryptodira
OT  - Neck mobility
OT  - Pleurodira
OT  - Testudinata
OT  - Testudines
OT  - ancestral shape reconstruction
OT  - neck retraction
OT  - phylomorphospace
EDAT- 2014/10/12 06:00
MHDA- 2015/04/17 06:00
CRDT- 2014/10/12 06:00
PHST- 2014/10/12 06:00 [entrez]
PHST- 2014/10/12 06:00 [pubmed]
PHST- 2015/04/17 06:00 [medline]
AID - syu072 [pii]
AID - 10.1093/sysbio/syu072 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Mar;64(2):187-204. doi: 10.1093/sysbio/syu072. Epub 2014 Oct 9.

PMID- 25293804
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - Cancer evolution: mathematical models and computational inference.
PG  - e1-25
LID - 10.1093/sysbio/syu081 [doi]
AB  - Cancer is a somatic evolutionary process characterized by the accumulation of
      mutations, which contribute to tumor growth, clinical progression, immune escape,
      and drug resistance development. Evolutionary theory can be used to analyze the
      dynamics of tumor cell populations and to make inference about the evolutionary
      history of a tumor from molecular data. We review recent approaches to modeling
      the evolution of cancer, including population dynamics models of tumor initiation
      and progression, phylogenetic methods to model the evolutionary relationship
      between tumor subclones, and probabilistic graphical models to describe
      dependencies among mutations. Evolutionary modeling helps to understand how
      tumors arise and will also play an increasingly important prognostic role in
      predicting disease progression and the outcome of medical interventions, such as 
      targeted therapy.
CI  - (c) The Author(s) 2014. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - Beerenwinkel, Niko
AU  - Beerenwinkel N
AD  - Department of Biosystems Science and Engineering, ETH Zurich, 4058 Basel,
      Switzerland; SIB Swiss Institute of Bioinformatics, 4058 Basel, Switzerland;
      European Molecular Biology Laboratory, European Bioinformatics Institute,
      Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, United Kingdom; 
      Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, CB10 1SA, United
      Kingdom; Cancer Research UK Cambridge Institute, University of Cambridge,
      Cambridge, CB20RE, United Kingdom Department of Biosystems Science and
      Engineering, ETH Zurich, 4058 Basel, Switzerland; SIB Swiss Institute of
      Bioinformatics, 4058 Basel, Switzerland; European Molecular Biology Laboratory,
      European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton,
      Cambridgeshire, CB10 1SA, United Kingdom; Wellcome Trust Sanger Institute,
      Hinxton, Cambridgeshire, CB10 1SA, United Kingdom; Cancer Research UK Cambridge
      Institute, University of Cambridge, Cambridge, CB20RE, United Kingdom
      niko.beerenwinkel@bsse.ethz.ch.
FAU - Schwarz, Roland F
AU  - Schwarz RF
AD  - Department of Biosystems Science and Engineering, ETH Zurich, 4058 Basel,
      Switzerland; SIB Swiss Institute of Bioinformatics, 4058 Basel, Switzerland;
      European Molecular Biology Laboratory, European Bioinformatics Institute,
      Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, United Kingdom; 
      Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, CB10 1SA, United
      Kingdom; Cancer Research UK Cambridge Institute, University of Cambridge,
      Cambridge, CB20RE, United Kingdom.
FAU - Gerstung, Moritz
AU  - Gerstung M
AD  - Department of Biosystems Science and Engineering, ETH Zurich, 4058 Basel,
      Switzerland; SIB Swiss Institute of Bioinformatics, 4058 Basel, Switzerland;
      European Molecular Biology Laboratory, European Bioinformatics Institute,
      Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, United Kingdom; 
      Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, CB10 1SA, United
      Kingdom; Cancer Research UK Cambridge Institute, University of Cambridge,
      Cambridge, CB20RE, United Kingdom.
FAU - Markowetz, Florian
AU  - Markowetz F
AD  - Department of Biosystems Science and Engineering, ETH Zurich, 4058 Basel,
      Switzerland; SIB Swiss Institute of Bioinformatics, 4058 Basel, Switzerland;
      European Molecular Biology Laboratory, European Bioinformatics Institute,
      Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, United Kingdom; 
      Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, CB10 1SA, United
      Kingdom; Cancer Research UK Cambridge Institute, University of Cambridge,
      Cambridge, CB20RE, United Kingdom.
LA  - eng
GR  - 609883/European Research Council/International
GR  - Cancer Research UK/United Kingdom
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Review
DEP - 20141007
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - *Biological Evolution
MH  - Humans
MH  - *Models, Biological
MH  - Mutation
MH  - *Neoplasms
MH  - Phylogeny
PMC - PMC4265145
OTO - NOTNLM
OT  - Cancer
OT  - cancer progression
OT  - evolution
OT  - population genetics
OT  - probabilistic graphical models
EDAT- 2014/10/09 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/10/09 06:00
PHST- 2014/10/09 06:00 [entrez]
PHST- 2014/10/09 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu081 [pii]
AID - 10.1093/sysbio/syu081 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):e1-25. doi: 10.1093/sysbio/syu081. Epub 2014 Oct 7.

PMID- 25281848
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - From gene trees to a dated allopolyploid network: insights from the angiosperm
      genus Viola (Violaceae).
PG  - 84-101
LID - 10.1093/sysbio/syu071 [doi]
AB  - Allopolyploidization accounts for a significant fraction of speciation events in 
      many eukaryotic lineages. However, existing phylogenetic and dating methods
      require tree-like topologies and are unable to handle the network-like
      phylogenetic relationships of lineages containing allopolyploids. No explicit
      framework has so far been established for evaluating competing network
      topologies, and few attempts have been made to date phylogenetic networks. We
      used a four-step approach to generate a dated polyploid species network for the
      cosmopolitan angiosperm genus Viola L. (Violaceae Batch.). The genus contains ca 
      600 species and both recent (neo-) and more ancient (meso-) polyploid lineages
      distributed over 16 sections. First, we obtained DNA sequences of three low-copy 
      nuclear genes and one chloroplast region, from 42 species representing all 16
      sections. Second, we obtained fossil-calibrated chronograms for each nuclear gene
      marker. Third, we determined the most parsimonious multilabeled genome tree and
      its corresponding network, resolved at the section (not the species) level.
      Reconstructing the "correct" network for a set of polyploids depends on
      recovering all homoeologs, i.e., all subgenomes, in these polyploids. Assuming
      the presence of Viola subgenome lineages that were not detected by the nuclear
      gene phylogenies ("ghost subgenome lineages") significantly reduced the number of
      inferred polyploidization events. We identified the most parsimonious network
      topology from a set of five competing scenarios differing in the interpretation
      of homoeolog extinctions and lineage sorting, based on (i) fewest possible ghost 
      subgenome lineages, (ii) fewest possible polyploidization events, and (iii) least
      possible deviation from expected ploidy as inferred from available chromosome
      counts of the involved polyploid taxa. Finally, we estimated the homoploid and
      polyploid speciation times of the most parsimonious network. Homoploid speciation
      times were estimated by coalescent analysis of gene tree node ages. Polyploid
      speciation times were estimated by comparing branch lengths and speciation rates 
      of lineages with and without ploidy shifts. Our analyses recognize Viola as an
      old genus (crown age 31 Ma) whose evolutionary history has been profoundly
      affected by allopolyploidy. Between 16 and 21 allopolyploidizations are necessary
      to explain the diversification of the 16 major lineages (sections) of Viola,
      suggesting that allopolyploidy has accounted for a high percentage-between 67%
      and 88%-of the speciation events at this level. The theoretical and
      methodological approaches presented here for (i) constructing networks and (ii)
      dating speciation events within a network, have general applicability for
      phylogenetic studies of groups where allopolyploidization has occurred. They make
      explicit use of a hitherto underexplored source of ploidy information from
      chromosome counts to help resolve phylogenetic cases where incomplete sequence
      data hampers network inference. Importantly, the coalescent-based method used
      herein circumvents the assumption of tree-like evolution required by most
      techniques for dating speciation events.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Marcussen, Thomas
AU  - Marcussen T
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis
      (CEES), University of Oslo, PO Box 1066 Blindern, NO-0316 Oslo, Norway and
      Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, 405 30 Gothenburg, Sweden Department of Biosciences, Centre for
      Ecological and Evolutionary Synthesis (CEES), University of Oslo, PO Box 1066
      Blindern, NO-0316 Oslo, Norway and Department of Biological and Environmental
      Sciences, University of Gothenburg, PO Box 461, 405 30 Gothenburg, Sweden.
FAU - Heier, Lise
AU  - Heier L
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis
      (CEES), University of Oslo, PO Box 1066 Blindern, NO-0316 Oslo, Norway and
      Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, 405 30 Gothenburg, Sweden.
FAU - Brysting, Anne K
AU  - Brysting AK
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis
      (CEES), University of Oslo, PO Box 1066 Blindern, NO-0316 Oslo, Norway and
      Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, 405 30 Gothenburg, Sweden.
FAU - Oxelman, Bengt
AU  - Oxelman B
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis
      (CEES), University of Oslo, PO Box 1066 Blindern, NO-0316 Oslo, Norway and
      Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, 405 30 Gothenburg, Sweden.
FAU - Jakobsen, Kjetill S
AU  - Jakobsen KS
AD  - Department of Biosciences, Centre for Ecological and Evolutionary Synthesis
      (CEES), University of Oslo, PO Box 1066 Blindern, NO-0316 Oslo, Norway and
      Department of Biological and Environmental Sciences, University of Gothenburg, PO
      Box 461, 405 30 Gothenburg, Sweden k.s.jakobsen@ibv.uio.no.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20141003
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Evolution, Molecular
MH  - Fossils
MH  - *Phylogeny
MH  - Polyploidy
MH  - Time
MH  - Viola/*classification/*genetics
PMC - PMC4265142
OTO - NOTNLM
OT  - Dating
OT  - Viola
OT  - low-copy nuclear gene
OT  - polyploidy
OT  - species network
OT  - violaceae
EDAT- 2014/10/05 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/10/05 06:00
PHST- 2014/10/05 06:00 [entrez]
PHST- 2014/10/05 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu071 [pii]
AID - 10.1093/sysbio/syu071 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):84-101. doi: 10.1093/sysbio/syu071. Epub 2014 Oct 3.

PMID- 25281847
OWN - NLM
STAT- MEDLINE
DCOM- 20180110
LR  - 20190110
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 65
IP  - 3
DP  - 2016 May
TI  - A Bayesian Supertree Model for Genome-Wide Species Tree Reconstruction.
PG  - 397-416
LID - 10.1093/sysbio/syu082 [doi]
AB  - Current phylogenomic data sets highlight the need for species tree methods able
      to deal with several sources of gene tree/species tree incongruence. At the same 
      time, we need to make most use of all available data. Most species tree methods
      deal with single processes of phylogenetic discordance, namely, gene duplication 
      and loss, incomplete lineage sorting (ILS) or horizontal gene transfer. In this
      manuscript, we address the problem of species tree inference from multilocus,
      genome-wide data sets regardless of the presence of gene duplication and loss and
      ILS therefore without the need to identify orthologs or to use a single
      individual per species. We do this by extending the idea of Maximum Likelihood
      (ML) supertrees to a hierarchical Bayesian model where several sources of gene
      tree/species tree disagreement can be accounted for in a modular manner. We
      implemented this model in a computer program called guenomu whose inputs are
      posterior distributions of unrooted gene tree topologies for multiple gene
      families, and whose output is the posterior distribution of rooted species tree
      topologies. We conducted extensive simulations to evaluate the performance of our
      approach in comparison with other species tree approaches able to deal with more 
      than one leaf from the same species. Our method ranked best under simulated data 
      sets, in spite of ignoring branch lengths, and performed well on empirical data, 
      as well as being fast enough to analyze relatively large data sets. Our Bayesian 
      supertree method was also very successful in obtaining better estimates of gene
      trees, by reducing the uncertainty in their distributions. In addition, our
      results show that under complex simulation scenarios, gene tree parsimony is also
      a competitive approach once we consider its speed, in contrast to more
      sophisticated models.
CI  - (c) The Author(s) 2014. Published by Oxford University Press on behalf of the
      Society of Systematic Biologists.
FAU - De Oliveira Martins, Leonardo
AU  - De Oliveira Martins L
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Vigo,
      36310, Spain leomrtns@uvigo.es.
FAU - Mallo, Diego
AU  - Mallo D
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Vigo,
      36310, Spain.
FAU - Posada, David
AU  - Posada D
AD  - Department of Biochemistry, Genetics and Immunology, University of Vigo, Vigo,
      36310, Spain.
LA  - eng
SI  - Dryad/10.5061/dryad.74922
GR  - 203161/European Research Council/International
PT  - Journal Article
DEP - 20141003
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Bayes Theorem
MH  - Classification/*methods
MH  - Computer Simulation
MH  - Genome/genetics
MH  - *Models, Genetic
MH  - *Phylogeny
MH  - Software
PMC - PMC4851173
OTO - NOTNLM
OT  - *hierarchical Bayesian model
OT  - *phylogenomics
OT  - *reconciliation
OT  - *supertree
OT  - *tree distance
EDAT- 2014/10/05 06:00
MHDA- 2018/01/11 06:00
CRDT- 2014/10/05 06:00
PHST- 2014/02/05 00:00 [received]
PHST- 2014/09/30 00:00 [accepted]
PHST- 2014/10/05 06:00 [entrez]
PHST- 2014/10/05 06:00 [pubmed]
PHST- 2018/01/11 06:00 [medline]
AID - syu082 [pii]
AID - 10.1093/sysbio/syu082 [doi]
PST - ppublish
SO  - Syst Biol. 2016 May;65(3):397-416. doi: 10.1093/sysbio/syu082. Epub 2014 Oct 3.

PMID- 25281846
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20171116
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - Methods for the quantitative comparison of molecular estimates of clade age and
      the fossil record.
PG  - 25-41
LID - 10.1093/sysbio/syu068 [doi]
AB  - Approaches quantifying the relative congruence, or incongruence, of molecular
      divergence estimates and the fossil record have been limited. Previously proposed
      methods are largely node specific, assessing incongruence at particular nodes for
      which both fossil data and molecular divergence estimates are available. These
      existing metrics, and other methods that quantify incongruence across topologies 
      including entirely extinct clades, have so far not taken into account uncertainty
      surrounding both the divergence estimates and the ages of fossils. They have also
      treated molecular divergence estimates younger than previously assessed fossil
      minimum estimates of clade age as if they were the same as cases in which they
      were older. However, these cases are not the same. Recovered divergence dates
      younger than compared oldest known occurrences require prior hypotheses regarding
      the phylogenetic position of the compared fossil record and standard assumptions 
      about the relative timing of morphological and molecular change to be incorrect. 
      Older molecular dates, by contrast, are consistent with an incomplete fossil
      record and do not require prior assessments of the fossil record to be unreliable
      in some way. Here, we compare previous approaches and introduce two new
      descriptive metrics. Both metrics explicitly incorporate information on
      uncertainty by utilizing the 95% confidence intervals on estimated divergence
      dates and data on stratigraphic uncertainty concerning the age of the compared
      fossils. Metric scores are maximized when these ranges are overlapping. MDI
      (minimum divergence incongruence) discriminates between situations where
      molecular estimates are younger or older than known fossils reporting both
      absolute fit values and a number score for incompatible nodes. DIG range
      (divergence implied gap range) allows quantification of the minimum increase in
      implied missing fossil record induced by enforcing a given set of molecular-based
      estimates. These metrics are used together to describe the relationship between
      time trees and a set of fossil data, which we recommend be phylogenetically
      vetted and referred on the basis of apomorphy. Differences from previously
      proposed metrics and the utility of MDI and DIG range are illustrated in three
      empirical case studies from angiosperms, ostracods, and birds. These case studies
      also illustrate the ways in which MDI and DIG range may be used to assess time
      trees resultant from analyses varying in calibration regime, divergence dating
      approach or molecular sequence data analyzed.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Clarke, Julia A
AU  - Clarke JA
AD  - Jackson School of Geosciences, The University of Texas at Austin, 1 University
      Station C1100, Austin, TX 78712; Department of Geology and Geological
      Engineering, South Dakota School of Mines and Technology, 501 East Saint Joseph
      Street, Rapid City, SD 57701, USA Julia_Clarke@jsg.utexas.edu.
FAU - Boyd, Clint A
AU  - Boyd CA
AD  - Jackson School of Geosciences, The University of Texas at Austin, 1 University
      Station C1100, Austin, TX 78712; Department of Geology and Geological
      Engineering, South Dakota School of Mines and Technology, 501 East Saint Joseph
      Street, Rapid City, SD 57701, USA.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20141003
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
SB  - IM
MH  - Animals
MH  - Classification/*methods
MH  - Crustacea/classification/genetics
MH  - Evolution, Molecular
MH  - *Fossils
MH  - Genes, Mitochondrial/genetics
MH  - Magnoliopsida/classification/genetics
MH  - *Phylogeny
MH  - Spheniscidae/classification/genetics
MH  - Time
OTO - NOTNLM
OT  - Time tree
OT  - angiosperm
OT  - calibration
OT  - divergence dating
OT  - ostracod
OT  - penguin
OT  - stratigraphic consistency metrics
EDAT- 2014/10/05 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/10/05 06:00
PHST- 2014/10/05 06:00 [entrez]
PHST- 2014/10/05 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu068 [pii]
AID - 10.1093/sysbio/syu068 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):25-41. doi: 10.1093/sysbio/syu068. Epub 2014 Oct 3.

PMID- 25261464
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20181113
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - Genomic repeat abundances contain phylogenetic signal.
PG  - 112-26
LID - 10.1093/sysbio/syu080 [doi]
AB  - A large proportion of genomic information, particularly repetitive elements, is
      usually ignored when researchers are using next-generation sequencing. Here we
      demonstrate the usefulness of this repetitive fraction in phylogenetic analyses, 
      utilizing comparative graph-based clustering of next-generation sequence reads,
      which results in abundance estimates of different classes of genomic repeats.
      Phylogenetic trees are then inferred based on the genome-wide abundance of
      different repeat types treated as continuously varying characters; such repeats
      are scattered across chromosomes and in angiosperms can constitute a majority of 
      nuclear genomic DNA. In six diverse examples, five angiosperms and one insect,
      this method provides generally well-supported relationships at interspecific and 
      intergeneric levels that agree with results from more standard phylogenetic
      analyses of commonly used markers. We propose that this methodology may prove
      especially useful in groups where there is little genetic differentiation in
      standard phylogenetic markers. At the same time as providing data for
      phylogenetic inference, this method additionally yields a wealth of data for
      comparative studies of genome evolution.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists.
FAU - Dodsworth, Steven
AU  - Dodsworth S
AD  - School of Biological and Chemical Sciences, Queen Mary University of London, Mile
      End Road, London E1 4NS, UK; Jodrell Laboratory, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3DS, UK; School of Plant Biology, The University of Western 
      Australia, Crawley WA 6009, Australia; Institute of Plant Molecular Biology,
      Biology Centre ASCR, Branisovska 31, Ceske Budejovice, CZ-37005, Czech Republic; 
      Systematic Botany and Mycology, University of Munich (LMU), Menzinger Strasse 67,
      80638 Munchen, Germany; and Department of Systematic and Evolutionary Botany,
      University of Vienna, Rennweg 14, A-1030 Vienna, Austria School of Biological and
      Chemical Sciences, Queen Mary University of London, Mile End Road, London E1 4NS,
      UK; Jodrell Laboratory, Royal Botanic Gardens, Kew, Richmond, Surrey TW9 3DS, UK;
      School of Plant Biology, The University of Western Australia, Crawley WA 6009,
      Australia; Institute of Plant Molecular Biology, Biology Centre ASCR, Branisovska
      31, Ceske Budejovice, CZ-37005, Czech Republic; Systematic Botany and Mycology,
      University of Munich (LMU), Menzinger Strasse 67, 80638 Munchen, Germany; and
      Department of Systematic and Evolutionary Botany, University of Vienna, Rennweg
      14, A-1030 Vienna, Austria.
FAU - Chase, Mark W
AU  - Chase MW
AD  - School of Biological and Chemical Sciences, Queen Mary University of London, Mile
      End Road, London E1 4NS, UK; Jodrell Laboratory, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3DS, UK; School of Plant Biology, The University of Western 
      Australia, Crawley WA 6009, Australia; Institute of Plant Molecular Biology,
      Biology Centre ASCR, Branisovska 31, Ceske Budejovice, CZ-37005, Czech Republic; 
      Systematic Botany and Mycology, University of Munich (LMU), Menzinger Strasse 67,
      80638 Munchen, Germany; and Department of Systematic and Evolutionary Botany,
      University of Vienna, Rennweg 14, A-1030 Vienna, Austria School of Biological and
      Chemical Sciences, Queen Mary University of London, Mile End Road, London E1 4NS,
      UK; Jodrell Laboratory, Royal Botanic Gardens, Kew, Richmond, Surrey TW9 3DS, UK;
      School of Plant Biology, The University of Western Australia, Crawley WA 6009,
      Australia; Institute of Plant Molecular Biology, Biology Centre ASCR, Branisovska
      31, Ceske Budejovice, CZ-37005, Czech Republic; Systematic Botany and Mycology,
      University of Munich (LMU), Menzinger Strasse 67, 80638 Munchen, Germany; and
      Department of Systematic and Evolutionary Botany, University of Vienna, Rennweg
      14, A-1030 Vienna, Austria.
FAU - Kelly, Laura J
AU  - Kelly LJ
AD  - School of Biological and Chemical Sciences, Queen Mary University of London, Mile
      End Road, London E1 4NS, UK; Jodrell Laboratory, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3DS, UK; School of Plant Biology, The University of Western 
      Australia, Crawley WA 6009, Australia; Institute of Plant Molecular Biology,
      Biology Centre ASCR, Branisovska 31, Ceske Budejovice, CZ-37005, Czech Republic; 
      Systematic Botany and Mycology, University of Munich (LMU), Menzinger Strasse 67,
      80638 Munchen, Germany; and Department of Systematic and Evolutionary Botany,
      University of Vienna, Rennweg 14, A-1030 Vienna, Austria School of Biological and
      Chemical Sciences, Queen Mary University of London, Mile End Road, London E1 4NS,
      UK; Jodrell Laboratory, Royal Botanic Gardens, Kew, Richmond, Surrey TW9 3DS, UK;
      School of Plant Biology, The University of Western Australia, Crawley WA 6009,
      Australia; Institute of Plant Molecular Biology, Biology Centre ASCR, Branisovska
      31, Ceske Budejovice, CZ-37005, Czech Republic; Systematic Botany and Mycology,
      University of Munich (LMU), Menzinger Strasse 67, 80638 Munchen, Germany; and
      Department of Systematic and Evolutionary Botany, University of Vienna, Rennweg
      14, A-1030 Vienna, Austria.
FAU - Leitch, Ilia J
AU  - Leitch IJ
AD  - School of Biological and Chemical Sciences, Queen Mary University of London, Mile
      End Road, London E1 4NS, UK; Jodrell Laboratory, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3DS, UK; School of Plant Biology, The University of Western 
      Australia, Crawley WA 6009, Australia; Institute of Plant Molecular Biology,
      Biology Centre ASCR, Branisovska 31, Ceske Budejovice, CZ-37005, Czech Republic; 
      Systematic Botany and Mycology, University of Munich (LMU), Menzinger Strasse 67,
      80638 Munchen, Germany; and Department of Systematic and Evolutionary Botany,
      University of Vienna, Rennweg 14, A-1030 Vienna, Austria.
FAU - Macas, Jiri
AU  - Macas J
AD  - School of Biological and Chemical Sciences, Queen Mary University of London, Mile
      End Road, London E1 4NS, UK; Jodrell Laboratory, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3DS, UK; School of Plant Biology, The University of Western 
      Australia, Crawley WA 6009, Australia; Institute of Plant Molecular Biology,
      Biology Centre ASCR, Branisovska 31, Ceske Budejovice, CZ-37005, Czech Republic; 
      Systematic Botany and Mycology, University of Munich (LMU), Menzinger Strasse 67,
      80638 Munchen, Germany; and Department of Systematic and Evolutionary Botany,
      University of Vienna, Rennweg 14, A-1030 Vienna, Austria.
FAU - Novak, Petr
AU  - Novak P
AD  - School of Biological and Chemical Sciences, Queen Mary University of London, Mile
      End Road, London E1 4NS, UK; Jodrell Laboratory, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3DS, UK; School of Plant Biology, The University of Western 
      Australia, Crawley WA 6009, Australia; Institute of Plant Molecular Biology,
      Biology Centre ASCR, Branisovska 31, Ceske Budejovice, CZ-37005, Czech Republic; 
      Systematic Botany and Mycology, University of Munich (LMU), Menzinger Strasse 67,
      80638 Munchen, Germany; and Department of Systematic and Evolutionary Botany,
      University of Vienna, Rennweg 14, A-1030 Vienna, Austria.
FAU - Piednoel, Mathieu
AU  - Piednoel M
AD  - School of Biological and Chemical Sciences, Queen Mary University of London, Mile
      End Road, London E1 4NS, UK; Jodrell Laboratory, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3DS, UK; School of Plant Biology, The University of Western 
      Australia, Crawley WA 6009, Australia; Institute of Plant Molecular Biology,
      Biology Centre ASCR, Branisovska 31, Ceske Budejovice, CZ-37005, Czech Republic; 
      Systematic Botany and Mycology, University of Munich (LMU), Menzinger Strasse 67,
      80638 Munchen, Germany; and Department of Systematic and Evolutionary Botany,
      University of Vienna, Rennweg 14, A-1030 Vienna, Austria.
FAU - Weiss-Schneeweiss, Hanna
AU  - Weiss-Schneeweiss H
AD  - School of Biological and Chemical Sciences, Queen Mary University of London, Mile
      End Road, London E1 4NS, UK; Jodrell Laboratory, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3DS, UK; School of Plant Biology, The University of Western 
      Australia, Crawley WA 6009, Australia; Institute of Plant Molecular Biology,
      Biology Centre ASCR, Branisovska 31, Ceske Budejovice, CZ-37005, Czech Republic; 
      Systematic Botany and Mycology, University of Munich (LMU), Menzinger Strasse 67,
      80638 Munchen, Germany; and Department of Systematic and Evolutionary Botany,
      University of Vienna, Rennweg 14, A-1030 Vienna, Austria.
FAU - Leitch, Andrew R
AU  - Leitch AR
AD  - School of Biological and Chemical Sciences, Queen Mary University of London, Mile
      End Road, London E1 4NS, UK; Jodrell Laboratory, Royal Botanic Gardens, Kew,
      Richmond, Surrey TW9 3DS, UK; School of Plant Biology, The University of Western 
      Australia, Crawley WA 6009, Australia; Institute of Plant Molecular Biology,
      Biology Centre ASCR, Branisovska 31, Ceske Budejovice, CZ-37005, Czech Republic; 
      Systematic Botany and Mycology, University of Munich (LMU), Menzinger Strasse 67,
      80638 Munchen, Germany; and Department of Systematic and Evolutionary Botany,
      University of Vienna, Rennweg 14, A-1030 Vienna, Austria a.r.leitch@qmul.ac.uk.
LA  - eng
GR  - P 21440/Austrian Science Fund FWF/Austria
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20140925
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Plant)
SB  - IM
MH  - Animals
MH  - Cluster Analysis
MH  - DNA, Plant/genetics
MH  - Drosophila/classification/genetics
MH  - Genes, Insect/genetics
MH  - Genome/*genetics
MH  - Magnoliopsida/genetics
MH  - *Phylogeny
MH  - Repetitive Sequences, Nucleic Acid/genetics
PMC - PMC4265144
OTO - NOTNLM
OT  - Repetitive DNA
OT  - continuous characters
OT  - genomics
OT  - molecular systematics
OT  - next-generation sequencing
OT  - phylogenetics
EDAT- 2014/09/28 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/09/28 06:00
PHST- 2014/09/28 06:00 [entrez]
PHST- 2014/09/28 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu080 [pii]
AID - 10.1093/sysbio/syu080 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):112-26. doi: 10.1093/sysbio/syu080. Epub 2014 Sep 25.

PMID- 25246662
OWN - NLM
STAT- MEDLINE
DCOM- 20150604
LR  - 20141216
IS  - 1076-836X (Electronic)
IS  - 1063-5157 (Linking)
VI  - 64
IP  - 1
DP  - 2015 Jan
TI  - The effects of inference method, population sampling, and gene sampling on
      species tree inferences: an empirical study in slender salamanders
      (Plethodontidae: Batrachoseps).
PG  - 66-83
LID - 10.1093/sysbio/syu078 [doi]
AB  - Species tree methods are now widely used to infer the relationships among species
      from multilocus data sets. Many methods have been developed, which differ in
      whether gene and species trees are estimated simultaneously or sequentially, and 
      in how gene trees are used to infer the species tree. While these methods perform
      well on simulated data, less is known about what impacts their performance on
      empirical data. We used a data set including five nuclear genes and one
      mitochondrial gene for 22 species of Batrachoseps to compare the effects of
      method of analysis, within-species sampling and gene sampling on species tree
      inferences. For this data set, the choice of inference method had the largest
      effect on the species tree topology. Exclusion of individual loci had large
      effects in *BEAST and STEM, but not in MP-EST. Different loci carried the
      greatest leverage in these different methods, showing that the causes of their
      disproportionate effects differ. Even though substantial information was present 
      in the nuclear loci, the mitochondrial gene dominated the *BEAST species tree.
      This leverage is inherent to the mtDNA locus and results from its high variation 
      and lower assumed ploidy. This mtDNA leverage may be problematic when mtDNA has
      undergone introgression, as is likely in this data set. By contrast, the leverage
      of RAG1 in STEM analyses does not reflect properties inherent to the locus, but
      rather results from a gene tree that is strongly discordant with all others, and 
      is best explained by introgression between distantly related species.
      Within-species sampling was also important, especially in *BEAST analyses, as
      shown by differences in tree topology across 100 subsampled data sets. Despite
      the sensitivity of the species tree methods to multiple factors, five species
      groups, the relationships among these, and some relationships within them, are
      generally consistently resolved for Batrachoseps.
CI  - (c) The Author(s) 2014. Published by Oxford University Press, on behalf of the
      Society of Systematic Biologists. All rights reserved. For Permissions, please
      email: journals.permissions@oup.com.
FAU - Jockusch, Elizabeth L
AU  - Jockusch EL
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, U-3043, Storrs, CT 06269-3043, USA; and CIBIO-InBIO, Centro de
      Investigacao em Biodiversidade e Recursos Geneticos, Campus Agrario de Vairao,
      Universidade do Porto, 4485-661 Vairao, Portugal elizabeth.jockusch@uconn.edu.
FAU - Martinez-Solano, Inigo
AU  - Martinez-Solano I
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, U-3043, Storrs, CT 06269-3043, USA; and CIBIO-InBIO, Centro de
      Investigacao em Biodiversidade e Recursos Geneticos, Campus Agrario de Vairao,
      Universidade do Porto, 4485-661 Vairao, Portugal Department of Ecology and
      Evolutionary Biology, University of Connecticut, 75 N. Eagleville Road, U-3043,
      Storrs, CT 06269-3043, USA; and CIBIO-InBIO, Centro de Investigacao em
      Biodiversidade e Recursos Geneticos, Campus Agrario de Vairao, Universidade do
      Porto, 4485-661 Vairao, Portugal.
FAU - Timpe, Elizabeth K
AU  - Timpe EK
AD  - Department of Ecology and Evolutionary Biology, University of Connecticut, 75 N. 
      Eagleville Road, U-3043, Storrs, CT 06269-3043, USA; and CIBIO-InBIO, Centro de
      Investigacao em Biodiversidade e Recursos Geneticos, Campus Agrario de Vairao,
      Universidade do Porto, 4485-661 Vairao, Portugal.
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
PT  - Research Support, U.S. Gov't, Non-P.H.S.
DEP - 20140922
PL  - England
TA  - Syst Biol
JT  - Systematic biology
JID - 9302532
RN  - 0 (DNA, Mitochondrial)
SB  - IM
MH  - Animals
MH  - Classification
MH  - DNA, Mitochondrial/genetics
MH  - *Phylogeny
MH  - Urodela/*classification/*genetics
OTO - NOTNLM
OT  - *BEAST
OT  - MP-EST
OT  - STEM
OT  - gene sampling
OT  - introgression
OT  - species tree inference
OT  - within-species sampling
EDAT- 2014/09/24 06:00
MHDA- 2015/06/05 06:00
CRDT- 2014/09/24 06:00
PHST- 2014/09/24 06:00 [entrez]
PHST- 2014/09/24 06:00 [pubmed]
PHST- 2015/06/05 06:00 [medline]
AID - syu078 [pii]
AID - 10.1093/sysbio/syu078 [doi]
PST - ppublish
SO  - Syst Biol. 2015 Jan;64(1):66-83. doi: 10.1093/sysbio/syu078. Epub 2014 Sep 22.
</pre></body></html>